
<html lang="en"     class="pb-page"  data-request-id="dc8273f7-fcea-4f37-aab8-9f141fe03a38"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2016.59.issue-9;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.5b02008"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Docking Screens for Novel Ligands Conferring New Biology" /></meta><meta name="dc.Creator" content="John J.  Irwin" /></meta><meta name="dc.Creator" content="Brian K.  Shoichet" /></meta><meta name="dc.Description" content="It is now plausible to dock libraries of 10 million molecules against targets over several days or weeks. When the molecules screened are commercially available, they may be rapidly tested to find ..." /></meta><meta name="Description" content="It is now plausible to dock libraries of 10 million molecules against targets over several days or weeks. When the molecules screened are commercially available, they may be rapidly tested to find ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 15, 2016" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b02008" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b02008" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b02008" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b02008" /></link>
        
    
    

<title>Docking Screens for Novel Ligands Conferring New Biology | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b02008" /></meta><meta property="og:title" content="Docking Screens for Novel Ligands Conferring New Biology" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0003.jpeg" /></meta><meta property="og:description" content="It is now plausible to dock libraries of 10 million molecules against targets over several days or weeks. When the molecules screened are commercially available, they may be rapidly tested to find new leads. Although docking retains important liabilities (it cannot calculate affinities accurately nor even reliably rank order high-scoring molecules), it can often can distinguish likely from unlikely ligands, often with hit rates above 10%. Here we summarize the improvements in libraries, target quality, and methods that have supported these advances, and the open access resources that make docking accessible. Recent docking screens for new ligands are sketched, as are the binding, crystallographic, and in vivo assays that support them. Like any technique, controls are crucial, and key experimental ones are reviewed. With such controls, docking campaigns can find ligands with new chemotypes, often revealing the new biology that may be docking’s greatest impact over the next few years." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b02008"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b02008">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b02008&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b02008&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b02008&amp;href=/doi/10.1021/acs.jmedchem.5b02008" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 4103-4120</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b01875" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.6b00078" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Docking Screens for Novel Ligands Conferring New Biology</span></h1><h2 class="article_header-subtitle"><span class="hlFld-SubTitle">Miniperspective</span></h2><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=John+J.++Irwin">John J. Irwin</a></span><span class="author-xref-symbol "><sup>*</sup></span></span></li><span class="comma-separator"> and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Brian+K.++Shoichet">Brian K. Shoichet</a></span><span class="author-xref-symbol "><sup>*</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text">Department of Pharmaceutical Chemistry and QB3 Institute, University of California—San Francisco, San Francisco, California 94158, United States</span></div><div class="corresp-info"><strong>*</strong>J.J.I.: phone, 415-514-4127; e-mail, <a href="/cdn-cgi/l/email-protection#375d5e4504050577505a565e5b1954585a"><span class="__cf_email__" data-cfemail="8be1e2f9b8b9b9cbece6eae2e7a5e8e4e6">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>B.K.S.; phone, 415-514-4126; e-mail, <a href="/cdn-cgi/l/email-protection#81e3f2e9eee8e2e9e4f5c1e6ece0e8edafe2eeec"><span class="__cf_email__" data-cfemail="84e6f7ecebede7ece1f0c4e3e9e5ede8aae7ebe9">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b02008&amp;href=/doi/10.1021%2Facs.jmedchem.5b02008" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 4103–4120</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 25, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>25 December 2015</li><li><span class="item_label"><b>Published</b> online</span>15 March 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 May 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b02008" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b02008</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4103%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJohn%2BJ.%2BIrwin%252C%2BBrian%2BK.%2BShoichet%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D9%26contentID%3Dacs.jmedchem.5b02008%26title%3DDocking%2BScreens%2Bfor%2BNovel%2BLigands%2BConferring%2BNew%2BBiology%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4120%26publicationDate%3DMay%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b02008"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">5265</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">129</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b02008" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Docking Screens for Novel Ligands Conferring New Biology&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;J. Irwin&quot;},{&quot;first_name&quot;:&quot;Brian&quot;,&quot;last_name&quot;:&quot;K. Shoichet&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;4103-4120&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b02008&quot;},&quot;abstract&quot;:&quot;It is now plausible to dock libraries of 10 million molecules against targets over several days or weeks. When the molecules screened are commercially available, they may be rapidly tested to find new leads. Although docking retains important liabilities (it cannot calculate affinities accurately nor even reliably rank order high-scoring molecules), it can often can distinguish likely from unlikely ligands, often with hit rates above 10%. Here we summarize the improvements in libraries, target quality, and methods that have supported these advances, and the open access resources that make docking accessible. Recent docking screens for new ligands are sketched, as are the binding, crystallographic, and in vivo assays that support them. Like any technique, controls are crucial, and key experimental ones are reviewed. With such controls, docking campaigns can find ligands with new chemotypes, often revealing the new biology that may be docking’s greatest impact over the next few years.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b02008&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b02008" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b02008&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b02008" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b02008&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b02008" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b02008&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b02008&amp;href=/doi/10.1021/acs.jmedchem.5b02008" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b02008" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b02008" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b02008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b02008%26sid%3Dliteratum%253Aachs%26pmid%3D26913380%26genre%3Darticle%26aulast%3DIrwin%26date%3D2016%26atitle%3DDocking%2BScreens%2Bfor%2BNovel%2BLigands%2BConferring%2BNew%2BBiology%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D9%26spage%3D4103%26epage%3D4120%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/jmcmar.2016.59.issue-9/20160512/jmcmar.2016.59.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/medium/jm-2015-020088_0003.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02008&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">It is now plausible to dock libraries of 10 million molecules against targets over several days or weeks. When the molecules screened are commercially available, they may be rapidly tested to find new leads. Although docking retains important liabilities (it cannot calculate affinities accurately nor even reliably rank order high-scoring molecules), it can often can distinguish likely from unlikely ligands, often with hit rates above 10%. Here we summarize the improvements in libraries, target quality, and methods that have supported these advances, and the open access resources that make docking accessible. Recent docking screens for new ligands are sketched, as are the binding, crystallographic, and in vivo assays that support them. Like any technique, controls are crucial, and key experimental ones are reviewed. With such controls, docking campaigns can find ligands with new chemotypes, often revealing the new biology that may be docking’s greatest impact over the next few years.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar/59/9"><issue-title>Computational Methods for Medicinal Chemistry</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Structure-Based Ligand Discovery</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66366" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66366" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Beginning in the 1970s, simulations of protein structures were promoted for structure-based or “rational” drug design.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> The techniques struggled to meet their initial promise, and early discovery continues to be dominated by empirical methods. Still, in the past decade, drugs for which structure and computation were genuinely pivotal have begun to appear; close to 20 drugs, with clear links to structure-based discovery or design, are now in clinical use.<a onclick="showRef(event, 'ref3 ref4 ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5 ref6 ref7">(3-7)</a> As few as they are, they may exceed those deriving from another widely heralded approach, high-throughput screening (HTS).<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Docking and Its Discontents</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21612" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21612" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">A central technique in structure-based discovery is molecular docking.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> In docking screens, libraries of about 10<sup>7</sup> molecules can be interrogated for those that complement a protein structure (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). For each molecule in the library, hundreds to thousands of orientations may be sampled in the protein site, and for every orientation there are hundreds of conformations; increasingly, alternative protein conformations are also considered. Overall, 10<sup>12</sup>–10<sup>13</sup> ligand complexes may be calculated in a large library screen, each ranked using scoring functions that consider several polar, nonpolar, and solvent-dependent scoring terms, all approximate.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/medium/jm-2015-020088_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0005.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Docking for new chemotypes from large libraries.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02008&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">There are several first-principle reasons why docking screens might never work. Among them, docking’s emphasis on throughput ensures that it undersamples states, principally protein conformations and ordered water networks. Even for ligand conformations, which are heavily sampled, the energy weighting among them is poorly treated or ignored. Docking complexes are rarely fully relaxed, leading to what amounts to irreversible work. Ligand and protein desolvation energies are at best approximate and are often entangled with docking interaction energy scores. These scores ignore important terms (among others: cation−π interactions and polarizability), and even the terms that are included are rarely well-parametrized for the diverse molecules in docking screening decks. The docking scoring energies are calculated all-at-once rather than via small perturbations that are central to techniques, such as free energy perturbation and alchemical thermodynamic integration, that can at least pretend to the calculation of relative and absolute affinities of binding. Over a calculation that samples 10<sup>7</sup> candidate ligands in 10<sup>13</sup> configurations, any one of these terms could ensure the failure of a docking screen for plausible ligands.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Pragmatic Success: Docking as a Screening Technique</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52515" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52515" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Despite what would seem to be eviscerating weaknesses, docking screens have been successfully prosecuted, discovering new ligands (<a class="ref internalNav" href="#tbl1" aria-label="Table 01">Table 01</a>)<a onclick="showRef(event, 'ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26 ref27 ref28 ref29 ref30 ref31 ref32 ref33 ref34 ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42 ref43 ref44 ref45 ref46 ref47 ref48 ref49 ref50 ref51 ref52 ref53 ref54 ref55 ref56 ref57 ref58 ref59 ref60 ref61 ref62 ref63 ref64 ref65'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26 ref27 ref28 ref29 ref30 ref31 ref32 ref33 ref34 ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42 ref43 ref44 ref45 ref46 ref47 ref48 ref49 ref50 ref51 ref52 ref53 ref54 ref55 ref56 ref57 ref58 ref59 ref60 ref61 ref62 ref63 ref64 ref65">(10-65)</a> that are increasingly confirmed with detailed biophysics, from concentration–response curves to <i>K</i><sub>i</sub> and <i>K</i><sub>d</sub> values of binding, to the comparison of subsequent crystal structures to the docking predictions (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). In blind, prospective comparisons with HTS against enzymes like HSP90,<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> PTP-1B,<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> β-lactamase,<a onclick="showRef(event, 'ref68 ref48'); return false;" href="javascript:void(0);" class="ref ref68 ref48">(68, 48)</a> and cruzain,<a onclick="showRef(event, 'ref50 ref69'); return false;" href="javascript:void(0);" class="ref ref50 ref69">(50, 69)</a> the docking “hit rates” (actives/tested) were 2–3 orders of magnitude better than for the empirical screen, and while HTS found docking false negatives, docking in its turn found false negatives from HTS.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/medium/jm-2015-020088_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0006.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Superposition of docking-predicted and subsequently determined crystal structures. (A) A 9 nM inverse agonist of the β2 adrenergic receptor discovered by docking (orange) superposed on the crystallographic result (green).<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> (B) Docking-predicted structure of eticlopride (light blue) superposed on the crystallographic result (yellow) in the dopamine D<sub>3</sub> receptor. Reproduced with permission from  <cite>Nature Chemical Biology</cite> (<contrib-group>Carlsson, J.</contrib-group>; Ligand discovery from a dopamine D3 receptor homology model and crystal structure; <span class="NLM_year">2011</span>, <em>7</em>, 769–778);<a onclick="showRef(event, 'ref213'); return false;" href="javascript:void(0);" class="ref ref213">(213)</a> Copyright 2011 Macmillan Publishers Ltd. (C) A 30 μM inhibitor of β-lactamase discovered by docking (purple) superposed on its crystallographic structure (white).<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> (D, E) Two crystal structures (gray carbons, electron density in wire mesh) superposed on the docking predicted ligands (yellow carbons) and poses refined by postdocking rescoring programs AMBERDOCK (cyan carbons) and PLOP (magenta carbons) for the nonpolar T4 lysozyme L99A and L99A/M102Q model cavities. Reproduced with permission from  <cite>Journal of Molecular Biology</cite> (<contrib-group>Graves, A. P.</contrib-group>; Rescoring docking hit lists for model cavity sites: predictions and experimental testing; <span class="NLM_year">2008</span>, <em>377</em>, 914–934);<a onclick="showRef(event, 'ref245'); return false;" href="javascript:void(0);" class="ref ref245">(245)</a> Copyright 2008 Elsevier, Ltd. (F) A docked ligand superposed on the crystallographic result from a screen against the model anion cavity in cytochrome <i>c</i> peroxidase W191G/gateless. Reproduced with permission from  <cite>Nature Chemistry</cite> (<contrib-group>Fischer, M.</contrib-group>; Incorporation of protein flexibility and conformational energy penalties in docking screens to improve ligand discovery; <span class="NLM_year">2014</span>, <em>6</em>, 575–583);<a onclick="showRef(event, 'ref243'); return false;" href="javascript:void(0);" class="ref ref243">(243)</a> Copyright 2014 Macmillan Publishers Ltd. (G) A 40 nM boronic acid inhibitor of AmpC predicted by covalent docking (cyan carbons) superposed on its crystal structure with the enzyme.<a onclick="showRef(event, 'ref175'); return false;" href="javascript:void(0);" class="ref ref175">(175)</a> (H) A 370 nM inhibitor of RSK2 kinase predicted by covalent docking (magenta carbons) superposed on its crystal structure with the enzyme (yellow carbons, with <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density shown in green mesh.<a onclick="showRef(event, 'ref175'); return false;" href="javascript:void(0);" class="ref ref175">(175)</a> (I) The docked structure (green carbons) of the high-energy intermediate of <i>S</i>-adenosyl homocysteine superposed on the crystallographic structure of the product, <i>S</i>-inosyl homocysteine (red carbons) in complex with Tm0936. Reproduced with permission from  <cite>Nature</cite> (<contrib-group>Hermann, J. C.</contrib-group>; Structure-based activity prediction for an enzyme of unknown function; <span class="NLM_year">2007</span>, <em>448</em>, 775–779);<a onclick="showRef(event, 'ref198'); return false;" href="javascript:void(0);" class="ref ref198">(198)</a> Copyright 2007 Macmillan Publishers Ltd. (J) An 8 μM inhibitor of PDK1 showing the docking pose (yellow carbons) superposed on the subsequently determined crystal structure (magenta carbons).<a onclick="showRef(event, 'ref234'); return false;" href="javascript:void(0);" class="ref ref234">(234)</a> (K) Modeled binding mode of a 10.6 μM fragment (orange carbons) superimposed with the dominant binding mode determined crystallographically (green carbon atoms).<a onclick="showRef(event, 'ref251'); return false;" href="javascript:void(0);" class="ref ref251">(251)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02008&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Selected Examples of Ligand Discovery by Molecular Docking</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">target</th><th class="colsep0 rowsep0" align="center">hit rate (actives/tested)</th><th class="colsep0 rowsep0" align="center">best hit (μM)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">structure determined?</th><th class="colsep0 rowsep0" align="center">biology investigated?</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">carbonic anhydrase II<a onclick="showRef(event, 'ref203'); return false;" href="javascript:void(0);" class="ref ref203">(203)</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.0008</td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">H1 histamine GPCR<a onclick="showRef(event, 'ref204'); return false;" href="javascript:void(0);" class="ref ref204">(204)</a></td><td class="colsep0 rowsep0" align="left">73% (19/26)</td><td class="colsep0 rowsep0" align="left">0.006</td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">β2 adrenergic<a onclick="showRef(event, 'ref49 ref205'); return false;" href="javascript:void(0);" class="ref ref49 ref205">(49, 205)</a></td><td class="colsep0 rowsep0" align="left">24% (6/25)</td><td class="colsep0 rowsep0" align="left">0.009</td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">A2a adenosine GPCR<a onclick="showRef(event, 'ref206'); return false;" href="javascript:void(0);" class="ref ref206">(206)</a></td><td class="colsep0 rowsep0" align="left">45% (9/20)</td><td class="colsep0 rowsep0" align="left">0.015</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SLC1A5<a onclick="showRef(event, 'ref207'); return false;" href="javascript:void(0);" class="ref ref207">(207)</a></td><td class="colsep0 rowsep0" align="left">50% (7/14)</td><td class="colsep0 rowsep0" align="left">0.02</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">cell culture</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">A2a adenosine GPCR<a onclick="showRef(event, 'ref208 ref209'); return false;" href="javascript:void(0);" class="ref ref208 ref209">(208, 209)</a></td><td class="colsep0 rowsep0" align="left">41% (23/56)</td><td class="colsep0 rowsep0" align="left">0.032</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PTPσ<a onclick="showRef(event, 'ref210'); return false;" href="javascript:void(0);" class="ref ref210">(210)</a></td><td class="colsep0 rowsep0" align="left">4.7% (7/147)</td><td class="colsep0 rowsep0" align="left">0.1</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CD40-TRAF6<a onclick="showRef(event, 'ref211'); return false;" href="javascript:void(0);" class="ref ref211">(211)</a></td><td class="colsep0 rowsep0" align="left">4.5% (7/150)</td><td class="colsep0 rowsep0" align="left">0.1</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">mouse</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DDX3 helicase<a onclick="showRef(event, 'ref212'); return false;" href="javascript:void(0);" class="ref ref212">(212)</a></td><td class="colsep0 rowsep0" align="left">40% (10/25)</td><td class="colsep0 rowsep0" align="left">0.2</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dopamine D3 GPCR X-ray<a onclick="showRef(event, 'ref213'); return false;" href="javascript:void(0);" class="ref ref213">(213)</a></td><td class="colsep0 rowsep0" align="left">20% (5/25)</td><td class="colsep0 rowsep0" align="left">0.3</td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dopamine D3 GPCR model<a onclick="showRef(event, 'ref213'); return false;" href="javascript:void(0);" class="ref ref213">(213)</a></td><td class="colsep0 rowsep0" align="left">23% (6/26)</td><td class="colsep0 rowsep0" align="left">0.2</td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LSD1<a onclick="showRef(event, 'ref214'); return false;" href="javascript:void(0);" class="ref ref214">(214)</a></td><td class="colsep0 rowsep0" align="left">7% (9/121)</td><td class="colsep0 rowsep0" align="left">0.2</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">cell culture</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MELK<a onclick="showRef(event, 'ref215'); return false;" href="javascript:void(0);" class="ref ref215">(215)</a></td><td class="colsep0 rowsep0" align="left">19% (3/16)</td><td class="colsep0 rowsep0" align="left">0.37</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NQO2<a onclick="showRef(event, 'ref216'); return false;" href="javascript:void(0);" class="ref ref216">(216)</a></td><td class="colsep0 rowsep0" align="left">14% (35/250)</td><td class="colsep0 rowsep0" align="left">0.4</td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="left">mouse</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NRPs<a onclick="showRef(event, 'ref217'); return false;" href="javascript:void(0);" class="ref ref217">(217)</a></td><td class="colsep0 rowsep0" align="left">0.2% (56/3000)</td><td class="colsep0 rowsep0" align="left">0.6</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">mouse</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P38 MAPK<a onclick="showRef(event, 'ref218'); return false;" href="javascript:void(0);" class="ref ref218">(218)</a></td><td class="colsep0 rowsep0" align="left">6% (6/98)</td><td class="colsep0 rowsep0" align="left">0.7</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NEDD8<a onclick="showRef(event, 'ref185'); return false;" href="javascript:void(0);" class="ref ref185">(185)</a></td><td class="colsep0 rowsep0" align="left">37% (3/8)</td><td class="colsep0 rowsep0" align="left">0.85</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PA-Nter<a onclick="showRef(event, 'ref219'); return false;" href="javascript:void(0);" class="ref ref219">(219)</a></td><td class="colsep0 rowsep0" align="left">20% (3/15)</td><td class="colsep0 rowsep0" align="left">0.94</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">influenza virus</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">G4<a onclick="showRef(event, 'ref220'); return false;" href="javascript:void(0);" class="ref ref220">(220)</a></td><td class="colsep0 rowsep0" align="left">16.6% (3/18)</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">cell culture</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DYRK1A<a onclick="showRef(event, 'ref221'); return false;" href="javascript:void(0);" class="ref ref221">(221)</a></td><td class="colsep0 rowsep0" align="left">3.5% (6/168)</td><td class="colsep0 rowsep0" align="left">1.5</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">cell culture</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLR<a onclick="showRef(event, 'ref222'); return false;" href="javascript:void(0);" class="ref ref222">(222)</a></td><td class="colsep0 rowsep0" align="left">8.5% (2/23)</td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">cells/functional</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Crm1<a onclick="showRef(event, 'ref223'); return false;" href="javascript:void(0);" class="ref ref223">(223)</a></td><td class="colsep0 rowsep0" align="left">8% (33/402)</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HCoV-OC43<a onclick="showRef(event, 'ref224'); return false;" href="javascript:void(0);" class="ref ref224">(224)</a></td><td class="colsep0 rowsep0" align="left">12% (1/8)</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HisG<a onclick="showRef(event, 'ref225'); return false;" href="javascript:void(0);" class="ref ref225">(225)</a></td><td class="colsep0 rowsep0" align="left">14% (7/50)</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><i>Mycobacterium</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AR<a onclick="showRef(event, 'ref226'); return false;" href="javascript:void(0);" class="ref ref226">(226)</a></td><td class="colsep0 rowsep0" align="left">11% (1/9)</td><td class="colsep0 rowsep0" align="left">2.7</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">cell culture</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cruzain<a onclick="showRef(event, 'ref227'); return false;" href="javascript:void(0);" class="ref ref227">(227)</a></td><td class="colsep0 rowsep0" align="left">52% (12/23)</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">trypanosomes</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">VHZP<a onclick="showRef(event, 'ref228'); return false;" href="javascript:void(0);" class="ref ref228">(228)</a></td><td class="colsep0 rowsep0" align="left">5% (7/142)</td><td class="colsep0 rowsep0" align="left">3.5</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SET7<a onclick="showRef(event, 'ref229'); return false;" href="javascript:void(0);" class="ref ref229">(229)</a></td><td class="colsep0 rowsep0" align="left">5% (7/127)</td><td class="colsep0 rowsep0" align="left">4.9</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GPB<a onclick="showRef(event, 'ref230'); return false;" href="javascript:void(0);" class="ref ref230">(230)</a></td><td class="colsep0 rowsep0" align="left">57% (4/7)</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="left">hepatocytes</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DPP-IV<a onclick="showRef(event, 'ref231'); return false;" href="javascript:void(0);" class="ref ref231">(231)</a></td><td class="colsep0 rowsep0" align="left">15% (15/99)</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">11β-HSD1<a onclick="showRef(event, 'ref232'); return false;" href="javascript:void(0);" class="ref ref232">(232)</a></td><td class="colsep0 rowsep0" align="left">10% (4/39)</td><td class="colsep0 rowsep0" align="left">5.3</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RSV<a onclick="showRef(event, 'ref233'); return false;" href="javascript:void(0);" class="ref ref233">(233)</a></td><td class="colsep0 rowsep0" align="left">66% (20/30)</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDK1<a onclick="showRef(event, 'ref234'); return false;" href="javascript:void(0);" class="ref ref234">(234)</a></td><td class="colsep0 rowsep0" align="left">20% (3/15)</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TGT<a onclick="showRef(event, 'ref235'); return false;" href="javascript:void(0);" class="ref ref235">(235)</a></td><td class="colsep0 rowsep0" align="left">50% (3/6)</td><td class="colsep0 rowsep0" align="left">8.3<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">EGFR ED<a onclick="showRef(event, 'ref236'); return false;" href="javascript:void(0);" class="ref ref236">(236)</a></td><td class="colsep0 rowsep0" align="left">13% (14/109)</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">cell culture</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PKR1<a onclick="showRef(event, 'ref237'); return false;" href="javascript:void(0);" class="ref ref237">(237)</a></td><td class="colsep0 rowsep0" align="left">10% (1/10)</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">mouse</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mGlu1<a onclick="showRef(event, 'ref238'); return false;" href="javascript:void(0);" class="ref ref238">(238)</a></td><td class="colsep0 rowsep0" align="left">14% (5/35)</td><td class="colsep0 rowsep0" align="left">10.2</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3CLpro<a onclick="showRef(event, 'ref239'); return false;" href="javascript:void(0);" class="ref ref239">(239)</a></td><td class="colsep0 rowsep0" align="left">4% (3/74)</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4(1)<a onclick="showRef(event, 'ref240'); return false;" href="javascript:void(0);" class="ref ref240">(240)</a></td><td class="colsep0 rowsep0" align="left">3.2% (5/153)</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RSV<a onclick="showRef(event, 'ref241'); return false;" href="javascript:void(0);" class="ref ref241">(241)</a></td><td class="colsep0 rowsep0" align="left">10% (5/50)</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="left">virus</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AmpC β-lactamase<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></td><td class="colsep0 rowsep0" align="left">5% (3/5)</td><td class="colsep0 rowsep0" align="left">26<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CDK4<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></td><td class="colsep0 rowsep0" align="left">5% (18/382)</td><td class="colsep0 rowsep0" align="left">0.04</td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HIVgp41<a onclick="showRef(event, 'ref135'); return false;" href="javascript:void(0);" class="ref ref135">(135)</a></td><td class="colsep0 rowsep0" align="left">1.7% (2/120)</td><td class="colsep0 rowsep0" align="left">56</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">cell fusion</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CcP W191G cavity<a onclick="showRef(event, 'ref242'); return false;" href="javascript:void(0);" class="ref ref242">(242)</a></td><td class="colsep0 rowsep0" align="left">94%(15/16)</td><td class="colsep0 rowsep0" align="left">20<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CcP W191G/gateless cavity<a onclick="showRef(event, 'ref243'); return false;" href="javascript:void(0);" class="ref ref243">(243)</a></td><td class="colsep0 rowsep0" align="left">60% (9/15)</td><td class="colsep0 rowsep0" align="left">7<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">T4 lysozyme L99A cavity<a onclick="showRef(event, 'ref244'); return false;" href="javascript:void(0);" class="ref ref244">(244)</a></td><td class="colsep0 rowsep0" align="left">100% (7/7)</td><td class="colsep0 rowsep0" align="left">56<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">L99A/M102Q cavity<a onclick="showRef(event, 'ref245'); return false;" href="javascript:void(0);" class="ref ref245">(245)</a></td><td class="colsep0 rowsep0" align="left">70% (23/33)</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">L99A/M102H cavity<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a></td><td class="colsep0 rowsep0" align="left">74% (21/29)</td><td class="colsep0 rowsep0" align="left">3<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DPP-IV<a onclick="showRef(event, 'ref246'); return false;" href="javascript:void(0);" class="ref ref246">(246)</a></td><td class="colsep0 rowsep0" align="left">50% (3/6)</td><td class="colsep0 rowsep0" align="left">0.72</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">rat</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MOR42-3<a onclick="showRef(event, 'ref247'); return false;" href="javascript:void(0);" class="ref ref247">(247)</a></td><td class="colsep0 rowsep0" align="left">55% (22/40)</td><td class="colsep0 rowsep0" align="left">100<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">frog</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TLR2<a onclick="showRef(event, 'ref248'); return false;" href="javascript:void(0);" class="ref ref248">(248)</a></td><td class="colsep0 rowsep0" align="left">20% (1/5)</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">cell culture</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">pgMDD<a onclick="showRef(event, 'ref249'); return false;" href="javascript:void(0);" class="ref ref249">(249)</a></td><td class="colsep0 rowsep0" align="left">36% (4/11)</td><td class="colsep0 rowsep0" align="left">157<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">cell culture</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RSK kinase (covalent)<a onclick="showRef(event, 'ref175'); return false;" href="javascript:void(0);" class="ref ref175">(175)</a></td><td class="colsep0 rowsep0" align="left">62% (5/8)</td><td class="colsep0 rowsep0" align="left">0.43</td><td class="colsep0 rowsep0" align="left">yes</td><td class="colsep0 rowsep0" align="left">cell culture</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">iNOS<a onclick="showRef(event, 'ref250'); return false;" href="javascript:void(0);" class="ref ref250">(250)</a></td><td class="colsep0 rowsep0" align="left">12.5% (1/8)</td><td class="colsep0 rowsep0" align="left">2.5</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">zebrafish</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">Fragment docking.</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last">ND = not determined (binding, no <i>K</i><sub>d</sub> values).</p></div></div></div><div class="NLM_p">How can these successes be reconciled with the weaknesses of the approach? Like other large-scale screens, docking seeks new observations, in its case new ligands, unanticipated from previous series, with the hope of conferring new biology. It emphasizes throughput and the polling of many possibilities and tolerates both false positives and false negatives as long as enough experimentally confirmed positives are found. Whereas docking can fail for any of the reasons sketched above, it happens that, crude as they are, the docking scores successfully reduce the many millions of molecules screened to a relatively small number of plausible candidates. As these candidates are available commercially or, for industrial users, from an in-house library, the cost of failure is cheap. Even a 10% success rate is often more than sufficient to justify the enterprise.</div><div class="NLM_p last">Here we describe practical strategies for successful docking campaigns and the resources that allow for their prosecution by a wide community, not just specialists. Because the enterprise depends on a close integration of the docking screen and the experimental testing of a few candidates, we will consider strategies to pick hits for testing and for the critical evaluation of the experimental tests. We will only summarize recent methods developments in docking, which have been well reviewed elsewhere.<a onclick="showRef(event, 'ref70 ref71 ref72 ref73 ref74'); return false;" href="javascript:void(0);" class="ref ref70 ref71 ref72 ref73 ref74">(70-74)</a> We will close with a discussion of the major contribution of docking to early discovery in medicinal chemistry and chemical biology, which is the illumination of genuinely new chemotypes for targets of active biological interest.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i5">Steps in a Docking Campaign</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10516" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10516" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Docking is increasingly enabled for nonspecialists by Web tools (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>) and available compound databases (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). Still, the approach integrates disparate methods, and there are several places it can go wrong if attention is not paid to maddening details. Among these are the protein structures on which docking depends, whether from X-ray crystallography, NMR, comparative modeling,<a onclick="showRef(event, 'ref75 ref76 ref77'); return false;" href="javascript:void(0);" class="ref ref75 ref76 ref77">(75-77)</a> and soon electron microscopy. Binding sites can have incomplete, flipped, or ambiguous side chains, and some residues may occupy more than one position in the structure. Some binding site metals will lack parameters. Structural waters,<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> which can play a critical role in recognition, must be treated as part of the site, taken out, or treated as displaceable if the method can do so.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Web Resources Widely Used for Docking Screens</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">name</th><th class="colsep0 rowsep0" align="center">Web site</th><th class="colsep0 rowsep0" align="center">compounds and cost</th><th class="colsep0 rowsep0" align="center">used for</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ChemNavigator</td><td class="colsep0 rowsep0" align="left"><a href="http://www.chemnavigator.com" class="extLink">http://www.chemnavigator.com</a></td><td class="colsep0 rowsep0" align="left">9.2 × 10<sup>7</sup>, commercial</td><td class="colsep0 rowsep0" align="left">Procurement service</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ChemSpider</td><td class="colsep0 rowsep0" align="left"><a href="http://chemspider.com" class="extLink">http://chemspider.com</a></td><td class="colsep0 rowsep0" align="left">3.5 × 10<sup>7</sup>, free</td><td class="colsep0 rowsep0" align="left">Chemical structure database with integrated information</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DOCK Blaster</td><td class="colsep0 rowsep0" align="left"><a href="http://blaster.docking.org" class="extLink">http://blaster.docking.org</a></td><td class="colsep0 rowsep0" align="left">Free</td><td class="colsep0 rowsep0" align="left">Noncovalent docking</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Docking Server</td><td class="colsep0 rowsep0" align="left"><a href="http://www.dockingserver.com" class="extLink">http://www.dockingserver.com</a></td><td class="colsep0 rowsep0" align="left">Free and commercial</td><td class="colsep0 rowsep0" align="left">Noncovalent docking</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DOCKovalent</td><td class="colsep0 rowsep0" align="left"><a href="http://covalent.docking.org" class="extLink">http://covalent.docking.org</a></td><td class="colsep0 rowsep0" align="left">Free</td><td class="colsep0 rowsep0" align="left">Covalent docking</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DUDE</td><td class="colsep0 rowsep0" align="left"><a href="http://dude.docking.org" class="extLink">http://dude.docking.org</a></td><td class="colsep0 rowsep0" align="left">Free</td><td class="colsep0 rowsep0" align="left">Decoy database and builder</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">eMolecules</td><td class="colsep0 rowsep0" align="left"><a href="http://emolecules.com" class="extLink">http://emolecules.com</a></td><td class="colsep0 rowsep0" align="left">7 × 10<sup>7</sup>, free/commercial</td><td class="colsep0 rowsep0" align="left">Procurement service</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">iScreen</td><td class="colsep0 rowsep0" align="left"><a href="http://iscreen.cmu.edu.tw" class="extLink">http://iscreen.cmu.edu.tw</a></td><td class="colsep0 rowsep0" align="left">Free</td><td class="colsep0 rowsep0" align="left">Noncovalent docking with a TCM focus</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Modbase</td><td class="colsep0 rowsep0" align="left"><a href="http://salilab.org" class="extLink">http://salilab.org</a></td><td class="colsep0 rowsep0" align="left">3.5 × 10<sup>7</sup>, free</td><td class="colsep0 rowsep0" align="left">Database of comparative models for docking</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MolPort</td><td class="colsep0 rowsep0" align="left"><a href="http://molport.com" class="extLink">http://molport.com</a></td><td class="colsep0 rowsep0" align="left">2.6 × 10<sup>7</sup>, free</td><td class="colsep0 rowsep0" align="left">Procurement service</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDB</td><td class="colsep0 rowsep0" align="left"><a href="http://www.pdb.org" class="extLink">http://www.pdb.org</a></td><td class="colsep0 rowsep0" align="left">1.1 × 10<sup>5</sup>, free</td><td class="colsep0 rowsep0" align="left">Database of crystal structures for docking</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ZINC</td><td class="colsep0 rowsep0" align="left"><a href="http://zinc15.docking.org" class="extLink">http://zinc15.docking.org</a></td><td class="colsep0 rowsep0" align="left">1.3 × 10<sup>8</sup>, free</td><td class="colsep0 rowsep0" align="left">Commercially available compounds for docking</td></tr></tbody></table></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Databases Widely Used for Docking Screens</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">name</th><th class="colsep0 rowsep0" align="center">Web site</th><th class="colsep0 rowsep0" align="center">compounds and cost</th><th class="colsep0 rowsep0" align="center">used for</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ACD</td><td class="colsep0 rowsep0" align="left"><a href="http://accelrys.com" class="extLink">http://accelrys.com</a></td><td class="colsep0 rowsep0" align="left">7 × 10<sup>6</sup>, commercial</td><td class="colsep0 rowsep0" align="left">Screening and building blocks</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ChEMBL</td><td class="colsep0 rowsep0" align="left"><a href="http://www.ebi.ac.uk/chembldb" class="extLink">http://www.ebi.ac.uk/chembldb</a></td><td class="colsep0 rowsep0" align="left">1.3 × 10<sup>6</sup>, free</td><td class="colsep0 rowsep0" align="left">The curated medicinal chemistry literature</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MDDR</td><td class="colsep0 rowsep0" align="left"><a href="http://accelrys.com" class="extLink">http://accelrys.com</a></td><td class="colsep0 rowsep0" align="left">2 × 10<sup>5</sup>, commercial</td><td class="colsep0 rowsep0" align="left">Biological active compounds compiled from patents</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Open NCI</td><td class="colsep0 rowsep0" align="left"><a href="http://cactus.nci.nih.gov" class="extLink">http://cactus.nci.nih.gov</a></td><td class="colsep0 rowsep0" align="left">3 × 10<sup>5</sup>, free<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">Free samples for testing in cancer research<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="5004131d1004313927313e">[email protected]</a></td><td class="colsep0 rowsep0" align="left"><a href="http://tcm.cmu.edu.tw" class="extLink">http://tcm.cmu.edu.tw</a></td><td class="colsep0 rowsep0" align="left">3.7 × 10<sup>4</sup>, free</td><td class="colsep0 rowsep0" align="left">Compounds from Traditional Chinese Medicine</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">WOMBAT</td><td class="colsep0 rowsep0" align="left"><a href="http://www.sunsetmolecular.com" class="extLink">http://www.sunsetmolecular.com</a></td><td class="colsep0 rowsep0" align="left">3 × 10<sup>5</sup>, commercial</td><td class="colsep0 rowsep0" align="left">Bioactivity databases for lead and drug discovery</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ZINC</td><td class="colsep0 rowsep0" align="left"><a href="http://zinc15.docking.org" class="extLink">http://zinc15.docking.org</a></td><td class="colsep0 rowsep0" align="left">1.3 × 10<sup>8</sup>, free</td><td class="colsep0 rowsep0" align="left">Commercially available compounds for docking</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">Free net of shipping, available for cancer-focused projects only.</p></div></div></div><div class="NLM_p">We find it useful to conduct retrospective sanity checks before undertaking a new, prospective docking campaign. When possible, annotated ligands may be gathered as positive controls, including any ligand-bound crystal structures if available. Large ligand sets may often be found in ChEMBL, and if so, they are likely also to be in ZINC, where they exist in a ready-to-dock form (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). These sanity check calculations work best if the ligands are combined with property-matched decoys, which physically resemble the ligands but are topologically dissimilar.<a onclick="showRef(event, 'ref79 ref80'); return false;" href="javascript:void(0);" class="ref ref79 ref80">(79, 80)</a> We typically use about 30–50 decoy molecules for each ligand in these sanity-check screens, asking if the calculation can enrich the ligands at the top of the docking-ranked list, in sensible poses, compared to the decoys. Parameters may be modified to improve ligand enrichment, but overfitting will bias the ultimate prospective screens toward precedented chemotypes, defeating the purpose. Decoys can be calculated readily using DUD-E<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>).</div><div class="NLM_p last">In prospective screens, large libraries of molecules are docked for those that complement the binding site. A focus on commercially available molecules (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>), for academic laboratories or on in-house libraries, for pharmaceutical research organizations, ensures that the molecules may be readily acquired and tested, which is crucial for the success of an enterprise that anticipates a 90% failure rate. ZINC incorporates many of these, amounting to over 100 million commercially available “druglike” compounds represented in precalculated, ready-to-dock formats. ZINC is easily subdivided by physical properties corresponding to current opinion in the field, such as “fragment-like”<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a> and “lead-like”<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a> subsets. Especially for those programs optimized for the parallel execution that docking allows and not limited by seat licenses, a full screen may be completed in a week or two of wall time on a typical academic cluster. What emerges at the end of the calculation is a list of molecules, rank-ordered by complementarity to the binding site by the docking scoring function in one or more high-scoring configurations in the binding site. These may be visualized individually and prioritized for purchasing and testing.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">Hit Picking Parties</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71277" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71277" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Docking ignores or approximates key energy terms and over a screen of 10<sup>7</sup> compounds simply misrepresents molecules; strained conformations of ligands may be docked, p<i>K</i><sub>a</sub> values may be miscalculated or high-energy tautomers sampled, among other potential problems. Even if these errors affect only a small percentage of the docked library, the scum often rises to the top (for instance, high-energy ligand conformations and mischarged molecules can make interactions that low-energy states cannot, resulting in artificially favorable scores). Like other high-throughput methods, rapid filters for artifacts are crucial, as are chemoinformatic controls for similarity to known chemotypes. Since the purpose of a docking screen is often the discovery of new chemical matter, complementing a binding site but dissimilar to molecules that could be found by QSAR, for instance, it is sensible to eliminate those docking hits that would have been found by topological similarity alone (this control is insisted on by some journals, including this one).</div><div class="NLM_p">In addition to automated, typically postdocking filters, visual evaluation of docked, high-ranking candidates can bring an integrated knowledge of the system and of physical chemistry. As an evaluation of HTS campaigns,<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> “hit-picking parties”, involving two to eight scientists from a broader team, can help select a final set of high-ranking molecules to acquire and physically test; these molecules, though always from the very top scoring molecules from a screen, are rarely the top ranked by raw docking score. Rather, the molecules may be selected for diversity and for favored interactions, for instance, the prioritization of a key hydrogen bond. More important than prioritizing favored interactions, which after all can bias the ligands toward known chemotypes, is the deprioritization of ligands that have unfavorable features for which the docking scoring function does not properly penalize. As mentioned above, common examples include ligand conformational strain, improper tautomerization, and improper protonation of ionizable groups. Each of these can be a challenging problem in molecular representation, requiring relatively high level calculations for any given molecule,<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> and it is little surprise that in over 10 million molecules, errors are made. Sometimes errors that slip through the automated library creation pipeline can be caught by an experienced medicinal chemist or biophysicist and deprioritized. Thus, it is often the case that a molecule that is ranked below 250th will be prioritized over a molecule that is ranked better than 10th. These hit-picking parties have a secondary benefit of training a wider team in molecular interactions, medicinal chemistry, and target biology.</div><div class="NLM_p last">With docking’s current accuracy, we often find that acquiring and testing 25–50 new molecules is sufficient to find two to five new ligand families. Sourcing this many compounds from commercial vendors is typically economically feasible even for academic laboratories (many more may be considered by pharmaceutical research organizations), and this number of compounds is often suited to the low throughput, careful assays in which docking hits are tested. For targets with better binding sites, like aminergic G protein coupled receptors (GPCRs), the number of compounds to acquire might be lower, while for targets with more solvent accessible sites, like proteases or β-lactamase, the number of compounds to acquire will be higher because the success rate will be lower. Improvements in docking methods, compound accessibility, and ease of testing will change these numbers, but the logic of testing enough molecules to ensure a sufficient number of plausible new lead chemotypes, given an expected hit rate, will remain valid.</div></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Critical Evaluation of Experimental Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10300" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10300" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Whereas there remain docking campaigns where no molecules are tested whatsoever,<a onclick="showRef(event, 'ref86 ref87 ref88 ref89 ref90 ref91 ref92 ref93 ref94 ref95 ref96 ref97 ref98 ref99 ref100 ref101 ref102 ref103 ref104 ref105 ref106 ref107 ref108 ref109 ref110 ref111 ref112 ref113 ref114 ref115 ref116 ref117 ref118'); return false;" href="javascript:void(0);" class="ref ref86 ref87 ref88 ref89 ref90 ref91 ref92 ref93 ref94 ref95 ref96 ref97 ref98 ref99 ref100 ref101 ref102 ref103 ref104 ref105 ref106 ref107 ref108 ref109 ref110 ref111 ref112 ref113 ref114 ref115 ref116 ref117 ref118">(86-118)</a> the most interesting are those that lead to experiments. To the delight (and sometimes the surprise) of the dockers, they often reveal “hits”, molecules that are active in a binding or functional assay. As with other screening techniques, these initial experimental hits must initially be viewed with suspicion, as they can be pock-marked with artifact.</div><div class="NLM_p">The origins of these experimental artifacts have been extensively reviewed:<a onclick="showRef(event, 'ref119 ref120 ref121 ref122'); return false;" href="javascript:void(0);" class="ref ref119 ref120 ref121 ref122">(119-122)</a> they can be broadly divided into molecules with pan assay interference (PAINS) chemotypes, which are promiscuous hitters,<a onclick="showRef(event, 'ref120'); return false;" href="javascript:void(0);" class="ref ref120">(120)</a> covalent acting molecules<a onclick="showRef(event, 'ref123'); return false;" href="javascript:void(0);" class="ref ref123">(123)</a> (but see below), molecules that interfere with assay spectroscopy,<a onclick="showRef(event, 'ref124 ref125 ref126'); return false;" href="javascript:void(0);" class="ref ref124 ref125 ref126">(124-126)</a> and molecules that form colloidal aggregates<a onclick="showRef(event, 'ref127 ref128 ref129'); return false;" href="javascript:void(0);" class="ref ref127 ref128 ref129">(127-129)</a> (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>). Hits with PAINS chemotypes<a onclick="showRef(event, 'ref130 ref131 ref132 ref133'); return false;" href="javascript:void(0);" class="ref ref130 ref131 ref132 ref133">(130-133)</a> are not always pathological (over 60 FDA-approved and worldwide drugs contain PAINS chemotypes) but often are and should be flagged for detailed follow-up if not simply discarded. Many PAINS chemotypes can be rapidly identified using publicly available filters (e.g., <a href="http://zinc15.docking.org/patterns/home" class="extLink">http://zinc15.docking.org/patterns/home</a>, <a class="ref internalNav" href="#app1" aria-label="Appendix 1">Appendix 1</a>). Sharing no single mechanism, they fall to no single set of controls but can be interrogated for classical dose–response curves, lack of incubation effects, imperviousness to mild reductants, and specificity versus counterscreening enzymes (all favorable features, arguing against artifactual activity). Covalent and spectroscopic interference molecules do have specific physical mechanisms for which they can be controlled (<a class="ref internalNav" href="#app1" aria-label="Appendix 1">Appendix 1</a>). By far the most common artifact from virtual and high-throughput screening campaigns is colloidal aggregation. Between 1% and 3% of the molecules in most screening libraries will aggregate at relevant concentrations, and the colloids that they form sequester and artifactually inhibit,<a onclick="showRef(event, 'ref68 ref69 ref129'); return false;" href="javascript:void(0);" class="ref ref68 ref69 ref129">(68, 69, 129)</a> and occasionally activate,<a onclick="showRef(event, 'ref134'); return false;" href="javascript:void(0);" class="ref ref134">(134)</a> proteins. Fortunately, this mechanism is readily controlled (<a class="ref internalNav" href="#app1" aria-label="Appendix 1">Appendix 1</a>). Studies that omit controls for these experimental artifacts may waste person years of effort as hopeless compounds are progressed.<a onclick="showRef(event, 'ref135 ref136 ref137 ref138 ref133 ref139'); return false;" href="javascript:void(0);" class="ref ref135 ref136 ref137 ref138 ref133 ref139">(135-138, 133, 139)</a></div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/medium/jm-2015-020088_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0007.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Aggregators (first row) and PAINS<a onclick="showRef(event, 'ref120'); return false;" href="javascript:void(0);" class="ref ref120">(120)</a> (second row) identified by docking: (A) 0.5 μM inhibitor of β-lactamase;<a onclick="showRef(event, 'ref127'); return false;" href="javascript:void(0);" class="ref ref127">(127)</a> (B) 7 μM inhibitor of DHFR;<a onclick="showRef(event, 'ref252'); return false;" href="javascript:void(0);" class="ref ref252">(252)</a> (C) 5 μM inhibitor of VEGF (and 10 μM IGF-1);<a onclick="showRef(event, 'ref253'); return false;" href="javascript:void(0);" class="ref ref253">(253)</a> (D) 14 μM inhibitor of tyrosine phosphatase;<a onclick="showRef(event, 'ref131'); return false;" href="javascript:void(0);" class="ref ref131">(131)</a> (E) 30 μM inhibitor of PTPRQ;<a onclick="showRef(event, 'ref130'); return false;" href="javascript:void(0);" class="ref ref130">(130)</a> (F) 3 μM inhibitor of DHFR.<a onclick="showRef(event, 'ref254'); return false;" href="javascript:void(0);" class="ref ref254">(254)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02008&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Detailed biophysical testing of new inhibitors for mechanism is always illuminating, irrespective of whether one suspects artifacts. There is an understandable tendency to fall in love with early hits, but hard experience<a onclick="showRef(event, 'ref127'); return false;" href="javascript:void(0);" class="ref ref127">(127)</a> suggests that doing so can obscure well-behaved, optimizable molecules that one would find if not distracted by the false positives. Measuring, and publishing, full concentration–response curves is a simple but crucial step toward this; much can be learned from the steepness of the curve and how well it is sampled (we recommend sampling every half-log in concentration and low and high enough in concentration to establish upper and lower baselines). A step further is to measure the full binding constant, through enzymological <i>K</i><sub>i</sub>, radioligand displacement or by reporter-free methods such as isothermal titration calorimetry, surface plasmon resonance, or related techniques. Here too, full curves should be measured and reported.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/medium/jm-2015-020088_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0001.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Classical and aggregation-based concentration–response curves. (a) The well-behaved, competitive inhibitor BZBTH2B (■, red curve) and the aggregator rottlerin (●, black curve), against the enzyme β-lactamase (adapted from ref <a onclick="showRef(event, 'ref255'); return false;" href="javascript:void(0);" class="ref ref255">255</a>, used with permission). (b) As the concentration of β-lactamase is increased, the inhibition curve shifts right and steepens for colloidal aggregators like rottlerin.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02008&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec7" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Improvements in Docking</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33448" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33448" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Investigators have long sought fundamental breakthroughs that would address docking’s core liabilities. Perhaps because of its entangled approximations, docking has resisted dramatic improvements and has had to be content with incremental progress. Still, over time, these have made noticeable improvements. Methods have added new polar interaction terms, flexible waters,<a onclick="showRef(event, 'ref78 ref146 ref147 ref148 ref149 ref150 ref151'); return false;" href="javascript:void(0);" class="ref ref78 ref146 ref147 ref148 ref149 ref150 ref151">(78, 146-151)</a> metal coordination,<a onclick="showRef(event, 'ref152 ref153'); return false;" href="javascript:void(0);" class="ref ref152 ref153">(152, 153)</a> protein preparation tools,<a onclick="showRef(event, 'ref154'); return false;" href="javascript:void(0);" class="ref ref154">(154)</a> parallelization,<a onclick="showRef(event, 'ref155 ref156 ref157'); return false;" href="javascript:void(0);" class="ref ref155 ref156 ref157">(155-157)</a> combinatorial library docking,<a onclick="showRef(event, 'ref158'); return false;" href="javascript:void(0);" class="ref ref158">(158)</a> new sampling methods, and treatments ordered waters, often using a version of inhomogeneous solvation theory.<a onclick="showRef(event, 'ref159 ref160 ref161'); return false;" href="javascript:void(0);" class="ref ref159 ref160 ref161">(159-161)</a> Just as important as these improvements to docking itself have been improvements in the quality and extent of the chemical libraries. Among the most widely used is the ZINC database, which has grown from about 750 000 commercially available molecules a decade ago<a onclick="showRef(event, 'ref162'); return false;" href="javascript:void(0);" class="ref ref162">(162)</a> to over 100 million for which three-dimensional, parametrized, dockable structures are now available.<a onclick="showRef(event, 'ref163'); return false;" href="javascript:void(0);" class="ref ref163">(163)</a> The quality of the molecules represented has also improved, drawing on improved treatments of molecular protonation and tautomerization,<a onclick="showRef(event, 'ref164 ref165'); return false;" href="javascript:void(0);" class="ref ref164 ref165">(164, 165)</a> as well as conformational libraries that are more likely to sample the experimentally observed form.<a onclick="showRef(event, 'ref166'); return false;" href="javascript:void(0);" class="ref ref166">(166)</a> Finally, the druggability of docking targets has improved. Over 60% of drugs act on membrane proteins, but 10 years ago the structures of few pharmacologically relevant membrane proteins had been determined, something that has changed dramatically in the past decade. These receptors, with their well-formed ligand binding sites, are often well-suited to complementarity-based approaches like docking and have afforded hit rates that are almost a log-order better than those experienced with the more open sites typical of soluble proteins, and hits that can be 2–3 log-orders better in affinity. As unglamorous as it seems, incremental improvements in docking methods, libraries, and target selection will continue to be central to progress in the field over the next few years.</div><div class="NLM_p last">What would a big breakthrough in docking look like? If one could accurately calculate binding free energies or even accurately rank-order among the top 0.1% of docking-prioritized hit-list (about 10 000 molecules), the impact would be enormous and would justify substantial investment. In principle, the biophysical bases for calculating accurate binding constants are in hand via alchemical methods and thermodynamic integration,<a onclick="showRef(event, 'ref167 ref168'); return false;" href="javascript:void(0);" class="ref ref167 ref168">(167, 168)</a> though these methods are only beginning to be tested prospectively.<a onclick="showRef(event, 'ref169 ref170'); return false;" href="javascript:void(0);" class="ref ref169 ref170">(169, 170)</a> Even with recent advances in long molecular dynamics simulations,<a onclick="showRef(event, 'ref171 ref172 ref173'); return false;" href="javascript:void(0);" class="ref ref171 ref172 ref173">(171-173)</a> however, these approaches remain far too slow to be practical on the scale of a large library screen. Calculating the <i>K</i><sub>d</sub> values of a 10 million molecule, diverse library would take decades, even with the increases in computer power likely in the next several years. Meanwhile, lack of proper parametrization of the library molecules ensures large errors, even for these high-end methods. Where the more accurate biophysical methods may have impact on a large scale screen, in the next few years, is in rescoring a top tranche of a docking hit-list, helping to prioritize the final few that will get tested. They can also improve the accuracy of docked geometries, which is crucial for the enterprise and is where docking can add the most once campaigns have progressed from initial discovery to optimization. Such goals for the high-end biophysical methods may seem modest, but if successful, they would have great impact.</div></div><div id="sec8" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Fragment and Covalent Docking</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02288" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02288" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Fragment-based discovery addresses the chemical space problem by screening molecules typically smaller than ∼17 non-hydrogen atoms (about 250 amu). Just as chemotype possibilities combinatorially explode as molecules grow in size, they collapse as they shrink, and fragment screens span a far greater percentage of chemotypes than screens of the druglike (up to 500 amu) or even leadlike (up to 350 amu) molecules. As powerful as fragment screens have been for chemotype discovery, empirical fragment libraries are rarely more than 10 000 molecules, owing to the low throughput of the biophysical methods used to assay them. This is less than needed to cover even known biorelevant chemotypes,<a onclick="showRef(event, 'ref174'); return false;" href="javascript:void(0);" class="ref ref174">(174)</a> and the empirical libraries are dwarfed by the 2.4 million fragments that are simply commercially available. Screening the commercially available fragments is beyond biophysical screening methods used in fragment-based discovery but would take a few days on a decent academic cluster by docking.</div><div class="NLM_p">The concern in fragment docking has been that the methods have been optimized for larger molecules, and docking would lead to promiscuous prediction of fragment poses. This worry has merit, as newly discovered docked fragments often have less fidelity to X-ray structures than do those of larger molecules; many in the field believe that fragment docking remains risky. Still, many predicted fragment geometries are fairly accurate, and the hit-rates in fragment docking can be a log-order better than leadlike or druglike screens.<a onclick="showRef(event, 'ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref51 ref52">(51, 52)</a> With all their potential liabilities, fragment docking can fill chemotype holes in empirical libraries, sometimes leading to hits with much higher affinity and ligand efficiency than empirical screens alone.<a onclick="showRef(event, 'ref174'); return false;" href="javascript:void(0);" class="ref ref174">(174)</a></div><div class="NLM_p last">The gap in our libraries for covalent probes is more compelling still. From bitter experience with false positives, most empirical screening libraries have been scrubbed clean of hot electrophiles, making it hard to intentionally screen a large library for covalent ligands. Conversely, electrophiles remain common among commercial libraries; among leadlike molecules in ZINC, over 600 000 can act covalently as Michael acceptors, epoxides, boronic acids, nitriles, and other warheads<a onclick="showRef(event, 'ref175'); return false;" href="javascript:void(0);" class="ref ref175">(175)</a> (<a href="http://covalent.docking.org" class="extLink">http://covalent.docking.org</a>). Recently, several methods have been introduced to intentionally screen such libraries for covalent ligands<a onclick="showRef(event, 'ref175 ref176 ref177 ref178 ref179 ref180 ref181'); return false;" href="javascript:void(0);" class="ref ref175 ref176 ref177 ref178 ref179 ref180 ref181">(175-181)</a> typically by modifying an existing noncovalent scoring function to sample the close bond approaches and to discount the van der Waals clashes implied by the formation of the covalent adduct. In favorable circumstances, these methods predict poses with high fidelity to subsequent crystal structures and have prospective hit rates substantially higher than experienced with noncovalent docking, while retaining decent specificity against off-targets.<a onclick="showRef(event, 'ref175'); return false;" href="javascript:void(0);" class="ref ref175">(175)</a> This is an area where new tools are rapidly being introduced<a onclick="showRef(event, 'ref175 ref176 ref177 ref182 ref183 ref184 ref185'); return false;" href="javascript:void(0);" class="ref ref175 ref176 ref177 ref182 ref183 ref184 ref185">(175-177, 182-185)</a> and in which there is widespread interest.<a onclick="showRef(event, 'ref98 ref186 ref187 ref188'); return false;" href="javascript:void(0);" class="ref ref98 ref186 ref187 ref188">(98, 186-188)</a></div></div><div id="sec9" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Expansion to Comparative Models</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45041" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45041" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">With all the errors in docking, targeting homology models might seem risky. On the other hand, the number of targets with experimental structures is outstripped by several log-orders by those for which plausible comparative models may be calculated. Modeling from templates with sufficient sequence identity and, where possible, fortified by other constraints,<a onclick="showRef(event, 'ref77 ref189'); return false;" href="javascript:void(0);" class="ref ref77 ref189">(77, 189)</a> the technique can provide useful structures for ligand discovery.<a onclick="showRef(event, 'ref190 ref191'); return false;" href="javascript:void(0);" class="ref ref190 ref191">(190, 191)</a> Indeed, in a prospective, head-to-head study of docking against a comparative model and then, subsequently, a crystal structure of the same receptor, about the same hit rate (20–23%) and hit affinity (0.2–3 μM) was returned for both structures. Multiple docking screens against homology models have been reported for ligand, substrate,<a onclick="showRef(event, 'ref192 ref193 ref194'); return false;" href="javascript:void(0);" class="ref ref192 ref193 ref194">(192-194)</a> and deorphanization studies,<a onclick="showRef(event, 'ref188'); return false;" href="javascript:void(0);" class="ref ref188">(188)</a> and indeed for one of the approved drugs where structure-based design played a pivotal role, aliskiren, homology modeling of lead complexes long predated crystal structures.<a onclick="showRef(event, 'ref195'); return false;" href="javascript:void(0);" class="ref ref195">(195)</a> Even with all the errors in docking and protein structure modeling, targeting comparative models can greatly expand dockable targets.</div></div><div id="sec10" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">New Chemotypes from Docking</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65808" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65808" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">What structure-based screens can reliably return are novel chemotypes unrelated to previously known ligands for a target. There are multiple ways a protein structure can recognize any given ligand chemotype,<a onclick="showRef(event, 'ref196'); return false;" href="javascript:void(0);" class="ref ref196">(196)</a> and a particular binding site can recognize multiple, unrelated scaffolds. Even for well-precedented targets, structure-based screens can find new ligands without antecedents in previous medicinal chemistry. Though there is no strong logic for why new ligand chemistry should lead to new biology, it often does. This ability to break out of medicinal chemistry boxes and to discover new chemotypes is where docking screens may find some of their most important contributions over the next few years. This may be illustrated by example.</div><div id="sec10_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> New β-Lactamase Inhibitors That Do Not Up-Regulate Enzyme Expression</h3><div class="NLM_p">Class C β-lactamases are bacterial enzymes that are the major source of resistance to β-lactam antibiotics like penicillins and cephalosporins in the clinic. Whereas β-lactam-based inhibitors, such as clavulanate, are well-known, they have grave liabilities for the class C enzymes: their IC<sub>50</sub> values are modest, and worse still they up-regulate the expression of the very enzyme that they are meant to inhibit, actually increasing resistance. A docking screen for new inhibitors of the class C β-lactamase, AmpC, discovered a family of thiophene-carboxylate sulfonamides. These inhibitors were dissimilar to the known β-lactams, including clavulanate, but they were found by crystallography to bind in the same site<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> and to inhibit the enzyme with about 1 μM potency.<a onclick="showRef(event, 'ref197'); return false;" href="javascript:void(0);" class="ref ref197">(197)</a> Unlike clavulanate, the new inhibitors simply inhibited the enzyme in bacterial cell culture; they did not up-regulate its expression. Thus, they, unlike classic β-lactamase inhibitors, were synergistic, not antagonistic, with primary β-lactam antibiotics such as ceftazidime and piperacillin (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>a).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/medium/jm-2015-020088_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0002.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. New functions conferred by new chemotypes discovered by docking. (A) The β-lactam inhibitor clavulanic acid up-regulates the expression of β-lactamase in <i>E. cloacae</i>, antagonizing the efficacy of the primary β-lactam antibiotics ceftazidime (CAZ) and piperacillin (PIP) in a disk diffusion assay (left).<a onclick="showRef(event, 'ref47 ref197'); return false;" href="javascript:void(0);" class="ref ref47 ref197">(47, 197)</a> In contrast, the docking-discovered inhibitors simply inhibit the enzyme without up-regulating it, synergizing with the same primary antibiotics (right). (B) Lorazepam (left) and ogerin (middle) are both modeled to bind at the same site on GPR68. Ogerin, however, is more potent and specific than lorazepam. In wild-type mice, ogerin reduces fear conditioning in a contextual freezing assay compared to knockout mice, where it has no measurable effect (right). Reproduced with permission from  <cite>Nature</cite> (<contrib-group>Huang, X. P.</contrib-group>; Allosteric ligands for the pharmacologically dark receptors GPR68 and GPR65; <span class="NLM_year">2015</span>, <em>527</em>, 477–483);<a onclick="showRef(event, 'ref256'); return false;" href="javascript:void(0);" class="ref ref256">(256)</a> Copyright 2015 Macmillan Publishers Ltd. (C) Inhibition of visceral hypersensitivity in IBS rat model. Therapeutic effects of compound <b>15</b> in IBS rats. The pain threshold decreased in response to CRD (at score 3 (left) and score 4 (right)).<a onclick="showRef(event, 'ref201'); return false;" href="javascript:void(0);" class="ref ref201">(201)</a> (D) Left panel. Xenograft tumor growth of human lung cancer cell line H2122, 50 mg kg<sup>–1</sup> day<sup>–1</sup> RBC8 initiated 24 h after inoculation inhibited tumor growth. Typical tumor appearance at 21 days is shown. Right panel. Effect of RBC8 on H358 xenograft models. RBC8 treatment (50 mg kg<sup>–1</sup> day<sup>–1</sup>) initiated 24 h after inoculation inhibited xenograft tumor growth of human lung cancer cell line H358. Data represent the mean ± SEM of six mice. Tumor volume in the treatment group was statistically different from controls as determined by the Student’s <i>t</i> test. Reproduced with permission from  <cite>Nature</cite> (<contrib-group>Yan, C.</contrib-group>; Discovery and characterization of small molecules that target the GTPase Ral; <span class="NLM_year">2014</span>, <em>515</em>, 443–447);<a onclick="showRef(event, 'ref202'); return false;" href="javascript:void(0);" class="ref ref202">(202)</a> Copyright 2014 Macmillan Publishers Ltd.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02008&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec10_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Receptor Deorphanization</h3><div class="NLM_p">Tm0936 was an orphan enzyme from <i>Thermatoga maritime</i> whose structure had been determined as part of a structural genomics initiative. Whereas by sequence and structure Tm0936 could be readily assigned to the amidohyrolase superfamily, its function was unknown and automated assignment misannotated it as a cytosine deaminase. A docking screen of 14 000 metabolites, represented as high-energy intermediates, against the enzyme’s structure revealed several analogs of adenosine as top-ranked hits, suggesting that they went through a deamination reaction on the purine ring (revealed by the high-energy intermediate form docked). Among these was <i>S</i>-adenosyl homocysteine, which on testing was found to be an excellent substrate for Tm0936, with a second order rate constant of 10<sup>5</sup> mol<sup>–1</sup> s<sup>–1</sup>. Subsequent crystallography confirmed the product of the predicted deamination, <i>S</i>-inosyl homocysteine, to superpose well with the docked pose (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>I). The conversion of <i>S</i>-adenosine homocysteine to the inosyl derivative suggested a previously unknown <i>S</i>-adenosyl homocysteine degradation pathway in bacteria.<a onclick="showRef(event, 'ref198'); return false;" href="javascript:void(0);" class="ref ref198">(198)</a></div><div class="NLM_p last">A similar strategy has recently illuminated the function of the orphan GPR68 (also known as OGR1) in learning and memory.<a onclick="showRef(event, 'ref256'); return false;" href="javascript:void(0);" class="ref ref256">(256)</a> This GPCR was screened in general function assays against a small drug library, and the benzodiazepine lorazepam was found to be a positive allosteric modulator (PAM) of it. Because of its binding to its therapeutic target, the GABA ion channel, lorazepam itself was not useful as a biological probe for GPR68’s function. Accordingly, thousands of 3D models of GPR68 were constructed using MODELLER<a onclick="showRef(event, 'ref199 ref200'); return false;" href="javascript:void(0);" class="ref ref199 ref200">(199, 200)</a> and prioritized by their ability to highly rank lorazepam versus the known decoy molecules from the initial empirical screen. A subsequent series of docking screens, testing, and optimization led to a PAM with about 30-fold improved efficacy over lorazepam. Unlike lorazepam, the new molecule had no measurable activity against the GABA channel and was specific for GPR68 versus sequence-related orphan receptors, such as GPR4 and GPR65. This molecule, dubbed ogerin (for OGR1 ligand), was advanced into in vivo functional studies. In a screen of mouse behavioral assays, ogerin reduced fear-conditioning in a context-based learning assay, consistent with its high expression in the hippocampus, which is associated with this modality. Conversely, ogerin had no effect on cue-based learning, associated with the amygdala, nor did it have any effect on even context-based learning in GPR68 knockout mice. A close analog of ogerin that had little effect in vitro also had little effect in vivo. These observations support the on-target effect of ogerin on GPR68 in vivo and by extension a function of this formally “dark” receptor in learning and memory.</div></div><div id="sec10_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Targeting the Serotonin Receptor</h3><div class="NLM_p last">Recent studies have suggested that the 5-HT<sub>2B</sub> GPCR plays a role in the development of irritable bowel syndrome and may be a target for new therapeutics. Currently, only a few selective antagonists of the receptor have been reported, due to the high degree of homology with its closely related protein family members. Huang and co-workers recently used a crystal structure of 5-HT<sub>2B</sub> with an agonist bound to discover novel and selective antagonist chemotypes for the receptor.<a onclick="showRef(event, 'ref201'); return false;" href="javascript:void(0);" class="ref ref201">(201)</a> They created induced-fit models using antagonists and docked an in-house collection of over 100 000 diverse compounds against it. Several new compounds emerged including a chemotype that contained a protonated triazine ring instead of the conventional tertiary aliphatic amine. This novel chemotype led to subtype selective compounds for the 5-HT<sub>2B</sub> receptor. Tested in vivo, these selective antagonists substantially attenuated visceral hypersensitivity in irritable bowel syndrome rat models. This scaffold represents a new avenue for the development of selective 5-HT<sub>2B</sub> antagonists for the treatment of 5-HT<sub>2B</sub>-related diseases.</div></div><div id="sec10_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Targeting Ras-like Kinases</h3><div class="NLM_p last">Ras and related soluble GTP-based kinases are perhaps the proteins most implicated in cancer but have long resisted modulation because of their high affinity of GDP and GTP. Theodorescu and Meroueh used molecular docking screens to discover inhibitors to modulate the GDP-form of the Ras-like kinases RalA and RalB.<a onclick="showRef(event, 'ref202'); return false;" href="javascript:void(0);" class="ref ref202">(202)</a> These inhibitors were active not only in vitro but also in Ral-mediated cell spreading in murine fibroblasts and anchorage-independent growth of human cancer cell lines. Two of the compounds were selective for Ral versus Ras or Rho kinases and inhibited tumor growth in a xenograph mouse model at a level similar to depletion of Ral by siRNA. These results illustrate the ability of a structure-based approach to discover chemotypes that have long eluded purely empirical approaches.</div></div></div><div id="sec11" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Forward Looking Guidance</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67115" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67115" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In the past decade docking has advanced by incremental improvement, not least in the size and quality of docking libraries. A true breakthrough in the rapid calculation of binding affinity or the ranking of diverse compounds, even if occurring postdocking, on a list of 1000–10 000 prioritized molecules would change the field. Even without such a breakthrough, steady optimization has brought the field to where it often can find new lead matter for biologically relevant targets. A great opportunity for docking is its application to the discovery of genuinely new molecules for both precedented and unprecedented targets. Whereas there is no strong reason why new chemotypes should lead to new biology, they often do, even for well-studied targets. The opportunity to use docking screens for new chemotype discovery is increasingly available not only to specialists but to the community, via open access tools (<a class="ref internalNav" href="#tbl2" aria-label="Tables 2">Tables 2</a>–<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">04</a>). The size of the chemical libraries routinely used by these structure-based screens has, moreover, grown steadily since they were first introduced 20 years ago, from 55 000 in the early 1990s to 250 000 in the early 2000s, to 750 000 by 2005, to now approaching 10 million and with another log-order expansion in sight. They far exceed most empirical libraries except perhaps those that are DNA-encoded. These days, a conservative “leadlike” docking campaign might screen six million molecules, most available from vendors within a couple of weeks. Were one willing to wait a couple of months, this number would grow to 60 million leadlike molecules. Naturally, such large libraries are interesting only if the structure-based methods can prioritize the tiny percentage of plausible ligands from among the vast number of decoys, and there is always a concern that adding more library molecules will simply drown the signal in noise. Still, results over the past several years, on multiple targets from multiple laboratories, suggest that docking, for all its problems, can distinguish plausible from implausible molecules, at least for well-formed binding sites. It also suggests that larger libraries have led to more novel and often more potent chemotypes, though admittedly this has not been investigated in a controlled way. In the next several years, docking’s greatest impact may be in its pragmatic application to biomedical problems where new chemical matter can confer new biology.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Widely Available Docking Software</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">docking program</th><th class="colsep0 rowsep0" align="center">Web site</th><th class="colsep0 rowsep0" align="center">terms</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AutoDock</td><td class="colsep0 rowsep0" align="left"><a href="http://autodock.scripps.edu" class="extLink">http://autodock.scripps.edu</a></td><td class="colsep0 rowsep0" align="left">Free</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AutoDock Vina</td><td class="colsep0 rowsep0" align="left"><a href="http://vina.scripps.edu" class="extLink">http://vina.scripps.edu</a></td><td class="colsep0 rowsep0" align="left">Free</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DiscoveryStudio</td><td class="colsep0 rowsep0" align="left"><a href="http://www.accelrys.com" class="extLink">http://www.accelrys.com</a></td><td class="colsep0 rowsep0" align="left">Commercial</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DOCK</td><td class="colsep0 rowsep0" align="left"><a href="http://dock.compbio.ucsf.edu" class="extLink">http://dock.compbio.ucsf.edu</a></td><td class="colsep0 rowsep0" align="left">Free to Academics</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FlexX</td><td class="colsep0 rowsep0" align="left"><a href="http://www.biosolveit.de/FlexX/" class="extLink">http://www.biosolveit.de/FlexX/</a></td><td class="colsep0 rowsep0" align="left">Commercial</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FRED</td><td class="colsep0 rowsep0" align="left"><a href="http://www.eyesopen.com" class="extLink">http://www.eyesopen.com</a></td><td class="colsep0 rowsep0" align="left">Free to qualified academics</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Glide XP/SP</td><td class="colsep0 rowsep0" align="left"><a href="http://www.schrodinger.com" class="extLink">http://www.schrodinger.com</a></td><td class="colsep0 rowsep0" align="left">Commercial</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GOLD</td><td class="colsep0 rowsep0" align="left"><a href="http://www.ccdc.cam.ac.uk" class="extLink">http://www.ccdc.cam.ac.uk</a></td><td class="colsep0 rowsep0" align="left">Commercial</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ICM-Pro</td><td class="colsep0 rowsep0" align="left"><a href="http://www.molsoft.com" class="extLink">http://www.molsoft.com</a></td><td class="colsep0 rowsep0" align="left">Commercial</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MOE</td><td class="colsep0 rowsep0" align="left"><a href="https://www.chemcomp.com" class="extLink">https://www.chemcomp.com</a></td><td class="colsep0 rowsep0" align="left">Commercial</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rDOCK</td><td class="colsep0 rowsep0" align="left"><a href="http://rdock.sourceforge.net" class="extLink">http://rdock.sourceforge.net</a></td><td class="colsep0 rowsep0" align="left">Free</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Surflex-Dock/Sybyl-X</td><td class="colsep0 rowsep0" align="left"><a href="http://www.tripos.com/surflex" class="extLink">http://www.tripos.com/surflex</a></td><td class="colsep0 rowsep0" align="left">Commercial</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CovDOCK</td><td class="colsep0 rowsep0" align="left"><a href="http://www.schrodinger.com" class="extLink">http://www.schrodinger.com</a></td><td class="colsep0 rowsep0" align="left">Commercial</td></tr></tbody></table></div></div></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.5b02008" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37431" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37431" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John J. Irwin</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical
Chemistry and QB3 Institute, University
of California—San Francisco, San Francisco, California 94158, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a9c3c0db9a9b9be9cec4c8c0c587cac6c4"><span class="__cf_email__" data-cfemail="80eae9f2b3b2b2c0e7ede1e9ecaee3efed">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Brian K. Shoichet</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical
Chemistry and QB3 Institute, University
of California—San Francisco, San Francisco, California 94158, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a0c2d3c8cfc9c3c8c5d4e0c7cdc1c9cc8ec3cfcd"><span class="__cf_email__" data-cfemail="f09283989f9993989584b0979d91999cde939f9d">[email protected]</span></a></span></div></li></ul></li><li><ul></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): The authors are founders of Blue Dolphin Lead Discovery LLC, which conducts fee-for-service molecular docking screens.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03555" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03555" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">John J. Irwin</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02008&amp;id=BIO-d77e2744-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>John J. Irwin</b> trained in organic chemistry and crystallography with Jack Dunitz at ETH Zurich (1988–1991) and in macromolecular crystallography software development with Gerard Bricgone at the MRC Laboratory of Molecular Biology (1994–1998). He developed the ZINC database and is part of the team behind the DUD and DUD-E docking benchmarks, the DOCK Blaster Web tool for public access molecular docking, and the similarity ensemble approach for associating proteins by the ligands they bind. His work is supported by the NIH and the FDA.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Brian K. Shoichet</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02008&amp;id=BIO-d77e2749-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Brian K. Shoichet</b> trained in molecular biophysics with Tack Kuntz at University of California—San Francisco (1986–1992), where he developed new docking methods and applied them to the first screens of readily available molecules. In his postdoctoral work he trained in protein structure with Brian Matthews at the Institute of Molecular Biology (1993–1996), where he investigated trade-offs between protein stability and ligand recognition. A theme of his lab has been a cycle of methods development and experimental testing. This has led to the discovery of colloidal aggregation, chemoinformatics methods for target identification, structure-based methods for ligand discovery, and the application of both to GPCRs. The work is supported by the NIH and by the FDA.</p></figure></div><div class="ack" id="ACK-d77e2754-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18231" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18231" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The work is supported by U.S. NIH Grants GM71896 and GM71630. We thank Gerhard Klebe, Ruth Brenk, Ruben Abagyan, Woody Sherman, Colin Groom, Ruth Thomas, Jason Cole, and Niu Huang for alerting us to recent publications. We thank Trent Balius, Marcus Fischer, and Matt O’Meara for reading the manuscript.</p></div><div class="NLM_app-group"><div id="app1" class="NLM_sec NLM_sec-type_appendix"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Appendix 1: Controls for Artifactual Assay Activity</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32740" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32740" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><b>(a) Irreversible Inhibitors.</b> Some noncovalent docking hits act as covalent artifacts, often via a reactive center present on the docked molecule or on an impurity. A rapid counterscreen for irreversible inhibition is to incubate the target protein and the hit at 5× their apparent IC<sub>50</sub> and then dilute them 10-fold (other IC<sub>50</sub> ratios may of course be chosen). If inhibition is rapidly reversible, the inhibition on dilution should drop to about 33% of full inhibition at this ratio. If dilution changes the inhibition little, it supports covalent activity. Legitimate, slow off-rate inhibition is another alternative, but such molecules are rare among initial screening hits. This experiment will only work for soluble proteins, but related experiments to measure off-rate may be adapted for membrane proteins.</div><div class="NLM_p"><b>(b) PAINS Molecules.</b> The chemotypes represented in these molecules often occur among promiscuous molecules that fail to progress. Several tools are available to rapidly detect these chemotypes, including at <a href="http://zinc15.docking.org/patterns/home" class="extLink">http://zinc15.docking.org/patterns/home</a>. Just because a hit has PAINS functionality does not invalidate it, but it does make careful testing for mechanism important. We recommend counterscreening the molecule against unrelated targets, determining whether it competes with a ligand known for the site and whether its concentration–response curves are well-behaved (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>a, red curve).</div><div class="NLM_p"><b>(c) Spectroscopic Interference Compounds.</b> Compounds that are absorb light or that fluoresce in a region used to measure activity can look like hits via assay interference,<a onclick="showRef(event, 'ref124 ref125 ref126 ref140'); return false;" href="javascript:void(0);" class="ref ref124 ref125 ref126 ref140">(124-126, 140)</a> as can molecules that inhibit a reporter enzyme, like luciferase.<a onclick="showRef(event, 'ref141'); return false;" href="javascript:void(0);" class="ref ref141">(141)</a> Spectroscopic interference should change linearly with concentration, following Beer’s law, rather than log-linearly as in a single site isotherm. Inhibitors of the reporter may be counterscreened against it.</div><div class="NLM_p"><b>(d) Active Impurities.</b> Sometimes compounds are not what they pretend to be or contain reactive impurities. Active molecules should be tested for purity. Anything below 95% should be further purified. ACS journals often request purity of 98% or better.</div><div class="NLM_p"><b>(e) Colloidal Aggregation.</b> Perhaps the largest single source of artifacts in early discovery is colloidal aggregation by small molecules.<a onclick="showRef(event, 'ref68 ref69 ref127'); return false;" href="javascript:void(0);" class="ref ref68 ref69 ref127">(68, 69, 127)</a> These particles, between 50 and 1000 nm in radius, adsorb protein without specificity, partially denaturing them. About 1–2% of molecules in a typical screening deck will aggregate at relevant concentrations, ensuring hits reflecting colloid-formation dominate screens, virtual and empirical, that do not control for them. Fortunately, this mechanism of inhibition may be readily controlled:</div><div class="NLM_p">(i) If inhibition can be attenuated by small amounts of nonionic detergent, the compound is likely an aggregator. We typically use 0.01% v/v freshly prepared Triton-X 100 or 0.025% v/v Tween-80<a onclick="showRef(event, 'ref142 ref143 ref144'); return false;" href="javascript:void(0);" class="ref ref142 ref143 ref144">(142-144)</a> for membrane or cell assays.</div><div class="NLM_p">(ii) Direct observation of particles in the 50–1000 nm size range by dynamic light scattering (DLS). Formation of particles does not guarantee promiscuous inhibition, but it is a worrying sign.</div><div class="NLM_p">(iii) For cell-based assays, colloidal particles can be precipitated by centrifugation before the assay is run, from assay media. If the compound is much more effective before spin down than after, it suggests colloidal aggregation.</div><div class="NLM_p">(iv) Noncompetitive inhibition with high Hill slopes. There are classical reasons for noncompetitive inhibition and for cooperative binding, but the latter is rare in early discovery and the two together suggest aggregation.</div><div class="NLM_p">(v) Attenuation of inhibition by increasing target concentration. Except when the receptor concentration to <i>K</i><sub>i</sub> ratio is high,<a onclick="showRef(event, 'ref145'); return false;" href="javascript:void(0);" class="ref ref145">(145)</a> increasing receptor concentration should not affect inhibition for well-behaved inhibitors, but it will be much reduced for colloidal aggregators and increase the steepness of the response curve (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>b). This experiment is only accessible for soluble proteins.</div><div class="NLM_p last">Investigators who worry that they have aggregators but are unable to perform these controls (e.g., DLS) can contact the authors who may be able to do so on their behalf (supported by NIH Grant GM71630).</div></div></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i24" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i24"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i25" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i25"> Abbreviations Used</h2><tr><td class="NLM_term">3CLpro</td><td class="NLM_def"><p class="first last">SARS-CoV 3-chymotrypsin-like protease</p></td></tr><tr><td class="NLM_term">5-HT<sub>2B</sub></td><td class="NLM_def"><p class="first last">serotonin G protein coupled receptor 2B</p></td></tr><tr><td class="NLM_term">ACS</td><td class="NLM_def"><p class="first last">American Chemical Society</p></td></tr><tr><td class="NLM_term">AR</td><td class="NLM_def"><p class="first last">androgen receptor</p></td></tr><tr><td class="NLM_term">ChEMBL</td><td class="NLM_def"><p class="first last">The EMBL Medicinal Chemistry database</p></td></tr><tr><td class="NLM_term">DLS</td><td class="NLM_def"><p class="first last">dynamic light scattering</p></td></tr><tr><td class="NLM_term">DUD-E</td><td class="NLM_def"><p class="first last">A Directory of Useful Decoys—Enhanced</p></td></tr><tr><td class="NLM_term">EGFR ED</td><td class="NLM_def"><p class="first last">epidermal growth factor receptor extracellular domain</p></td></tr><tr><td class="NLM_term">FDA</td><td class="NLM_def"><p class="first last">Food and Drug Administration</p></td></tr><tr><td class="NLM_term">G4</td><td class="NLM_def"><p class="first last">G-quadruplex</p></td></tr><tr><td class="NLM_term">GABA</td><td class="NLM_def"><p class="first last">γ-aminobutyric acid</p></td></tr><tr><td class="NLM_term">GDP</td><td class="NLM_def"><p class="first last">guanosine diphosphate</p></td></tr><tr><td class="NLM_term">GLR</td><td class="NLM_def"><p class="first last">glucagon receptor</p></td></tr><tr><td class="NLM_term">GPCR</td><td class="NLM_def"><p class="first last">G protein coupled receptor</p></td></tr><tr><td class="NLM_term">GPB</td><td class="NLM_def"><p class="first last">glycogen phosphorylase B</p></td></tr><tr><td class="NLM_term">GPR4</td><td class="NLM_def"><p class="first last">G protein coupled receptor 4</p></td></tr><tr><td class="NLM_term">GPR65</td><td class="NLM_def"><p class="first last">psychosine receptor</p></td></tr><tr><td class="NLM_term">GPR68</td><td class="NLM_def"><p class="first last">ovarian cancer G-protein coupled receptor 1</p></td></tr><tr><td class="NLM_term">GTP</td><td class="NLM_def"><p class="first last">guanosine triphosphate</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screening</p></td></tr><tr><td class="NLM_term">HCoV-OC43</td><td class="NLM_def"><p class="first last">human coronavirus nucleocapsid protein</p></td></tr><tr><td class="NLM_term">HisG</td><td class="NLM_def"><p class="first last"><i>Mycobacterium tuberculosis</i> ATP phosphoribosyl transferase</p></td></tr><tr><td class="NLM_term">HSP90</td><td class="NLM_def"><p class="first last">heat shock protein 90</p></td></tr><tr><td class="NLM_term">OGR1</td><td class="NLM_def"><p class="first last">ovarian cancer G-protein coupled receptor 1</p></td></tr><tr><td class="NLM_term">MELK</td><td class="NLM_def"><p class="first last">maternal embryonic leucine zipper kinase</p></td></tr><tr><td class="NLM_term">NRPS</td><td class="NLM_def"><p class="first last">nonribosomal peptide synthetase</p></td></tr><tr><td class="NLM_term">PAINS</td><td class="NLM_def"><p class="first last">pan assay interference</p></td></tr><tr><td class="NLM_term">PAM</td><td class="NLM_def"><p class="first last">positive allosteric modulator</p></td></tr><tr><td class="NLM_term">pgMDD</td><td class="NLM_def"><p class="first last">meso-2,6-diaminopimelate dehydrogenase</p></td></tr><tr><td class="NLM_term">PKR1</td><td class="NLM_def"><p class="first last">prokineticin receptor 1</p></td></tr><tr><td class="NLM_term">PTP-1B</td><td class="NLM_def"><p class="first last">tyrosine-protein phosphatase 1B</p></td></tr><tr><td class="NLM_term">PA-Nter</td><td class="NLM_def"><p class="first last">influenza virus PA endonuclease</p></td></tr><tr><td class="NLM_term">PTPσ</td><td class="NLM_def"><p class="first last">protein tyrosine phosphatase σ</p></td></tr><tr><td class="NLM_term">RalA</td><td class="NLM_def"><p class="first last">Ras-related protein Ral-Al</p></td></tr><tr><td class="NLM_term">RalB</td><td class="NLM_def"><p class="first last">Ras-related protein Ral-B</p></td></tr><tr><td class="NLM_term">TGT</td><td class="NLM_def"><p class="first last">tRNA-guanine transglycosylase</p></td></tr><tr><td class="NLM_term">siRNA</td><td class="NLM_def"><p class="first last">small interfering RNA</p></td></tr><tr><td class="NLM_term">SLC1A5</td><td class="NLM_def"><p class="first last">alanine–serine–cysteine transporter ASCT2</p></td></tr><tr><td class="NLM_term">VHZP</td><td class="NLM_def"><p class="first last">VHZ phosphatase</p></td></tr><tr><td class="NLM_term">ZINC</td><td class="NLM_def"><p class="first last">“ZINC is not commercial”, a database of compounds for virtual screening</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56816" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56816" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 256 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Beddell, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodford, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norrington, F. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilkinson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wootton, R.</span><span> </span><span class="NLM_article-title">Compounds designed to fit a site of known structure in human haemoglobin</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1976</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">201</span><span class="NLM_x">–</span> <span class="NLM_lpage">209</span><span class="refDoi"> DOI: 10.1111/j.1476-5381.1976.tb07468.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1111%2Fj.1476-5381.1976.tb07468.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=938794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADyaE28XlsFGhtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1976&pages=201-209&author=C.+R.+Beddellauthor=P.+J.+Goodfordauthor=F.+E.+Norringtonauthor=S.+Wilkinsonauthor=R.+Wootton&title=Compounds+designed+to+fit+a+site+of+known+structure+in+human+haemoglobin&doi=10.1111%2Fj.1476-5381.1976.tb07468.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Compounds designed to fit a site of known structure in human hemoglobin</span></div><div class="casAuthors">Beddell, C. R.; Goodford, P. J.; Norrington, F. E.; Wilkinson, S.; Wootton, R.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">201-9</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    </div><div class="casAbstract">Three 2,3-diphosphoglycerate (I) analogs, e.g. 4,4'-diformyl-2-bibenzyloxyacetic acid (II), bound to human deoxy-Hb at the I-binding site.  They promoted O liberation in the predicted sequence as assessed by sigmoidal dose-response curves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoG1ShdM6H3e7Vg90H21EOLACvtfcHk0litxTBNy6Okyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XlsFGhtLk%253D&md5=7f53a69f78a5930b75956c30aa03b720</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.1976.tb07468.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.1976.tb07468.x%26sid%3Dliteratum%253Aachs%26aulast%3DBeddell%26aufirst%3DC.%2BR.%26aulast%3DGoodford%26aufirst%3DP.%2BJ.%26aulast%3DNorrington%26aufirst%3DF.%2BE.%26aulast%3DWilkinson%26aufirst%3DS.%26aulast%3DWootton%26aufirst%3DR.%26atitle%3DCompounds%2520designed%2520to%2520fit%2520a%2520site%2520of%2520known%2520structure%2520in%2520human%2520haemoglobin%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D1976%26volume%3D57%26spage%3D201%26epage%3D209%26doi%3D10.1111%2Fj.1476-5381.1976.tb07468.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Cohen, S. S.</span><span> </span><span class="NLM_article-title">A strategy for the chemotherapy of infectious disease</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">197</span><span class="NLM_x">, </span> <span class="NLM_fpage">431</span><span class="NLM_x">–</span> <span class="NLM_lpage">432</span><span class="refDoi"> DOI: 10.1126/science.195340</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1126%2Fscience.195340" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=197&publication_year=1977&pages=431-432&author=S.+S.+Cohen&title=A+strategy+for+the+chemotherapy+of+infectious+disease&doi=10.1126%2Fscience.195340"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.195340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.195340%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DS.%2BS.%26atitle%3DA%2520strategy%2520for%2520the%2520chemotherapy%2520of%2520infectious%2520disease%26jtitle%3DScience%26date%3D1977%26volume%3D197%26spage%3D431%26epage%3D432%26doi%3D10.1126%2Fscience.195340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span> </span><span class="NLM_article-title">Structure-based drug screening for G-protein-coupled receptors</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">268</span><span class="NLM_x">–</span> <span class="NLM_lpage">272</span><span class="refDoi"> DOI: 10.1016/j.tips.2012.03.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.tips.2012.03.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=22503476" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFGit7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=268-272&author=B.+K.+Shoichetauthor=B.+K.+Kobilka&title=Structure-based+drug+screening+for+G-protein-coupled+receptors&doi=10.1016%2Fj.tips.2012.03.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based drug screening for G-protein-coupled receptors</span></div><div class="casAuthors">Shoichet, Brian K.; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">268-272</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  G-protein-coupled receptors (GPCRs) represent a large family of signaling proteins that includes many therapeutic targets; however, progress in identifying new small mol. drugs has been disappointing.  The past 4 years have seen remarkable progress in the structural biol. of GPCRs, raising the possibility of applying structure-based approaches to GPCR drug discovery efforts.  Of the various structure-based approaches that have been applied to sol. protein targets, such as proteases and kinases, in silico docking is among the most ready applicable to GPCRs.  Early studies suggest that GPCR binding pockets are well suited to docking, and docking screens have identified potent and novel compds. for these targets.  This review will focus on the current state of in silico docking for GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Eh6VhLHhH7Vg90H21EOLACvtfcHk0litxTBNy6Okyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFGit7Y%253D&md5=30f26646db71f5c0c8448e9d85908060</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2012.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2012.03.007%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520drug%2520screening%2520for%2520G-protein-coupled%2520receptors%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26spage%3D268%26epage%3D272%26doi%3D10.1016%2Fj.tips.2012.03.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Mittl, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grutter, M. G.</span><span> </span><span class="NLM_article-title">Opportunities for structure-based design of protease-directed drugs</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">769</span><span class="NLM_x">–</span> <span class="NLM_lpage">775</span><span class="refDoi"> DOI: 10.1016/j.sbi.2006.10.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.sbi.2006.10.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=17112720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1OhtL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=769-775&author=P.+R.+Mittlauthor=M.+G.+Grutter&title=Opportunities+for+structure-based+design+of+protease-directed+drugs&doi=10.1016%2Fj.sbi.2006.10.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Opportunities for structure-based design of protease-directed drugs</span></div><div class="casAuthors">Mittl, Peer RE; Gruetter, Markus G.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">769-775</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  As a result of the recent enormous technol. progress, exptl. structure detn. has become an integral part of the development of drugs against disease-related target proteins.  The post-translational modification of proteins is an important regulatory process in living organisms; one such example is lytic processing by peptidases.  Many different peptidases represent disease targets and are being used in structure-based drug design approaches.  The development of drugs such as aliskiren and tipranavir, which inhibit renin and HIV protease, resp., testifies to the success of this approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcGxJhLhiHbrVg90H21EOLACvtfcHk0liKY6BuQzcwGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1OhtL%252FN&md5=905b47627a5799b80be34fa0ed9d749d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2006.10.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2006.10.014%26sid%3Dliteratum%253Aachs%26aulast%3DMittl%26aufirst%3DP.%2BR.%26aulast%3DGrutter%26aufirst%3DM.%2BG.%26atitle%3DOpportunities%2520for%2520structure-based%2520design%2520of%2520protease-directed%2520drugs%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2006%26volume%3D16%26spage%3D769%26epage%3D775%26doi%3D10.1016%2Fj.sbi.2006.10.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">von Itzstein, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, W.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kok, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pegg, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dyason, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, T. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smythe, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliver, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colman, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varghese, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryan, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woods, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bethell, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hotham, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Penn, C. R.</span><span> </span><span class="NLM_article-title">Rational design of potent sialidase-based inhibitors of influenza virus replication</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">363</span><span class="NLM_x">, </span> <span class="NLM_fpage">418</span><span class="NLM_x">–</span> <span class="NLM_lpage">423</span><span class="refDoi"> DOI: 10.1038/363418a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2F363418a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=8502295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADyaK3sXkvVGntb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=363&publication_year=1993&pages=418-423&author=M.+von+Itzsteinauthor=W.-Y.+Wuauthor=G.+B.+Kokauthor=M.+S.+Peggauthor=J.+C.+Dyasonauthor=B.+Jinauthor=T.+V.+Phanauthor=M.+L.+Smytheauthor=H.+F.+Whiteauthor=S.+W.+Oliverauthor=P.+M.+Colmanauthor=J.+N.+Vargheseauthor=D.+M.+Ryanauthor=J.+M.+Woodsauthor=R.+C.+Bethellauthor=V.+J.+Hothamauthor=J.+M.+Cameronauthor=C.+R.+Penn&title=Rational+design+of+potent+sialidase-based+inhibitors+of+influenza+virus+replication&doi=10.1038%2F363418a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Rational design of potent sialidase-based inhibitors of influenza virus replication</span></div><div class="casAuthors">von Itzstein, Mark; Wu, Wen Yang; Kok, Gaik B.; Pegg, Michael S.; Dyason, Jeffrey C.; Jin, Betty; Phan Tho Van; Smythe, Mark L.; White, Hume F.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">363</span>
        (<span class="NLM_cas:issue">6428</span>),
    <span class="NLM_cas:pages">418-23</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Two potent inhibitors based on the crystal structure of influenza virus sialidase have been designed.  These compds. are effective inhibitors not only on the enzyme, but also of the virus in cell culture and in animal models.  The results provide an example of the power of rational, computer-assisted drug design, as well as indicating significant progress in the development of a new therapeutic or prophylactic treatment for influenza infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz-zoSWSTF3bVg90H21EOLACvtfcHk0liKY6BuQzcwGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXkvVGntb0%253D&md5=dc1e83fa24f50f54941924a43770468e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2F363418a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F363418a0%26sid%3Dliteratum%253Aachs%26aulast%3Dvon%2BItzstein%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DW.-Y.%26aulast%3DKok%26aufirst%3DG.%2BB.%26aulast%3DPegg%26aufirst%3DM.%2BS.%26aulast%3DDyason%26aufirst%3DJ.%2BC.%26aulast%3DJin%26aufirst%3DB.%26aulast%3DPhan%26aufirst%3DT.%2BV.%26aulast%3DSmythe%26aufirst%3DM.%2BL.%26aulast%3DWhite%26aufirst%3DH.%2BF.%26aulast%3DOliver%26aufirst%3DS.%2BW.%26aulast%3DColman%26aufirst%3DP.%2BM.%26aulast%3DVarghese%26aufirst%3DJ.%2BN.%26aulast%3DRyan%26aufirst%3DD.%2BM.%26aulast%3DWoods%26aufirst%3DJ.%2BM.%26aulast%3DBethell%26aufirst%3DR.%2BC.%26aulast%3DHotham%26aufirst%3DV.%2BJ.%26aulast%3DCameron%26aufirst%3DJ.%2BM.%26aulast%3DPenn%26aufirst%3DC.%2BR.%26atitle%3DRational%2520design%2520of%2520potent%2520sialidase-based%2520inhibitors%2520of%2520influenza%2520virus%2520replication%26jtitle%3DNature%26date%3D1993%26volume%3D363%26spage%3D418%26epage%3D423%26doi%3D10.1038%2F363418a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Varney, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marzoni, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmer, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deal, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webber, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Welsh, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bacquet, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morse, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Booth, C. L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrmann, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howland, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, J.</span><span> </span><span class="NLM_article-title">Crystal-structure-based design and synthesis of Benz[cd]indole-containing inhibitors of thymidylate synthase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">663</span><span class="NLM_x">–</span> <span class="NLM_lpage">676</span><span class="refDoi"> DOI: 10.1021/jm00082a006</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00082a006" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADyaK38Xht1ymtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1992&pages=663-676&author=M.+D.+Varneyauthor=G.+P.+Marzoniauthor=C.+L.+Palmerauthor=J.+G.+Dealauthor=S.+Webberauthor=K.+M.+Welshauthor=R.+J.+Bacquetauthor=C.+A.+Morseauthor=C.+L.+J.+Boothauthor=S.+M.+Herrmannauthor=E.+F.+Howlandauthor=R.+W.+Wardauthor=J.+White&title=Crystal-structure-based+design+and+synthesis+of+Benz%5Bcd%5Dindole-containing+inhibitors+of+thymidylate+synthase&doi=10.1021%2Fjm00082a006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal-structure-based design and synthesis of benz[cd]indole-containing inhibitors of thymidylate synthase</span></div><div class="casAuthors">Varney, Michael D.; Marzoni, Gifford P.; Palmer, Cindy L.; Deal, Judith G.; Webber, Stephanie; Welsh, Katherine M.; Bacquet, Russell J.; Bartlett, Charlotte A.; Morse, Catharine A.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">663-76</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Benz[cd]indoles I [R1 = H, Me, Et, Pr, CHMe2, CH2CH2OH, CH2CH2CH2OH, CH2SMe, R2 = H, Me, Cl, SMe, R3 = piperazinyl, morpholino, (un)substituted Ph; X = NH, NNH2, NOH, NMe, O, (SMe)2] were prepd. as inhibitors of thymidylate synthase (TS).  The structure-activity of the lead compd. was studied by conceptually dividing the mol. into four regions and independently optimizing the substituents for each region.  Combination of favored substituents for each region led to inhibitors with Kis against the human enzyme in the range of 10-20 nM.  Thymidine shift expts. suggested that the cytotoxic properties of the best enzyme inhibitors were due to TS targeting in cells.  The inhibitors were synthesized from substituted 6-aminobenz[cd]indol-2(1H)-ones by alkylation with both a simple alkyl group and a substituted benzylic portion.  The 2,6-diaminobenz[cd]indoles were prepd. from the corresponding lactams by conversion to the thiolactam, alkylation to the methylated thiolactam, and then displacement with a substituted or unsubstituted amine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrosbvRQ2rQbVg90H21EOLACvtfcHk0lhG0YruYXL6vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xht1ymtLg%253D&md5=b1059e47866ee0111117ca66b043eac7</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm00082a006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00082a006%26sid%3Dliteratum%253Aachs%26aulast%3DVarney%26aufirst%3DM.%2BD.%26aulast%3DMarzoni%26aufirst%3DG.%2BP.%26aulast%3DPalmer%26aufirst%3DC.%2BL.%26aulast%3DDeal%26aufirst%3DJ.%2BG.%26aulast%3DWebber%26aufirst%3DS.%26aulast%3DWelsh%26aufirst%3DK.%2BM.%26aulast%3DBacquet%26aufirst%3DR.%2BJ.%26aulast%3DMorse%26aufirst%3DC.%2BA.%26aulast%3DBooth%26aufirst%3DC.%2BL.%2BJ.%26aulast%3DHerrmann%26aufirst%3DS.%2BM.%26aulast%3DHowland%26aufirst%3DE.%2BF.%26aulast%3DWard%26aufirst%3DR.%2BW.%26aulast%3DWhite%26aufirst%3DJ.%26atitle%3DCrystal-structure-based%2520design%2520and%2520synthesis%2520of%2520Benz%255Bcd%255Dindole-containing%2520inhibitors%2520of%2520thymidylate%2520synthase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1992%26volume%3D35%26spage%3D663%26epage%3D676%26doi%3D10.1021%2Fjm00082a006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Feng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallace, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stafford, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaldor, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kassel, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Navre, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skene, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asakawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeuchi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gwaltney, S. L.,  2nd</span><span> </span><span class="NLM_article-title">Discovery of alogliptin: a potent, selective, bioavailable, and efficacious inhibitor of dipeptidyl peptidase IV</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2297</span><span class="NLM_x">–</span> <span class="NLM_lpage">2300</span><span class="refDoi"> DOI: 10.1021/jm070104l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm070104l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkt1Oqtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2297-2300&author=J.+Fengauthor=Z.+Zhangauthor=M.+B.+Wallaceauthor=J.+A.+Staffordauthor=S.+W.+Kaldorauthor=D.+B.+Kasselauthor=M.+Navreauthor=L.+Shiauthor=R.+J.+Skeneauthor=T.+Asakawaauthor=K.+Takeuchiauthor=R.+Xuauthor=D.+R.+Webbauthor=S.+L.+Gwaltney&title=Discovery+of+alogliptin%3A+a+potent%2C+selective%2C+bioavailable%2C+and+efficacious+inhibitor+of+dipeptidyl+peptidase+IV&doi=10.1021%2Fjm070104l"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Alogliptin: A Potent, Selective, Bioavailable, and Efficacious Inhibitor of Dipeptidyl Peptidase IV</span></div><div class="casAuthors">Feng, Jun; Zhang, Zhiyuan; Wallace, Michael B.; Stafford, Jeffrey A.; Kaldor, Stephen W.; Kassel, Daniel B.; Navre, Marc; Shi, Lihong; Skene, Robert J.; Asakawa, Tomoko; Takeuchi, Koji; Xu, Rongda; Webb, David R.; Gwaltney, Stephen L., II</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2297-2300</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Alogliptin is a potent, selective inhibitor of the serine protease dipeptidyl peptidase IV (DPP-4).  Herein, the authors describe the structure-based design and optimization of alogliptin and related quinazolinone-based DPP-4 inhibitors.  Following an oral dose, these noncovalent inhibitors provide sustained redn. of plasma DPP-4 activity and a lowering of blood glucose in animal models of diabetes.  Alogliptin is currently undergoing phase III trials in patients with type 2 diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRXyHbCH4hrLVg90H21EOLACvtfcHk0lhG0YruYXL6vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkt1Oqtrs%253D&md5=807ee0a245580988c22a02832a13084e</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm070104l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070104l%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DWallace%26aufirst%3DM.%2BB.%26aulast%3DStafford%26aufirst%3DJ.%2BA.%26aulast%3DKaldor%26aufirst%3DS.%2BW.%26aulast%3DKassel%26aufirst%3DD.%2BB.%26aulast%3DNavre%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DSkene%26aufirst%3DR.%2BJ.%26aulast%3DAsakawa%26aufirst%3DT.%26aulast%3DTakeuchi%26aufirst%3DK.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DWebb%26aufirst%3DD.%2BR.%26aulast%3DGwaltney%26aufirst%3DS.%2BL.%26atitle%3DDiscovery%2520of%2520alogliptin%253A%2520a%2520potent%252C%2520selective%252C%2520bioavailable%252C%2520and%2520efficacious%2520inhibitor%2520of%2520dipeptidyl%2520peptidase%2520IV%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D2297%26epage%3D2300%26doi%3D10.1021%2Fjm070104l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Sicheri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moarefi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span> </span><span class="NLM_article-title">Crystal structure of the Src family tyrosine kinase Hck</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">385</span><span class="NLM_x">, </span> <span class="NLM_fpage">602</span><span class="NLM_x">–</span> <span class="NLM_lpage">609</span><span class="refDoi"> DOI: 10.1038/385602a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2F385602a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=9024658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADyaK2sXht1ymtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=385&publication_year=1997&pages=602-609&author=F.+Sicheriauthor=I.+Moarefiauthor=J.+Kuriyan&title=Crystal+structure+of+the+Src+family+tyrosine+kinase+Hck&doi=10.1038%2F385602a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the Src family tyrosine kinase Hck</span></div><div class="casAuthors">Sicheri, Frank; Moarefi, Ismail; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">385</span>
        (<span class="NLM_cas:issue">6617</span>),
    <span class="NLM_cas:pages">602-609</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">The crystal structure of the hematopoietic cell kinase Hck has been detd. at 2.6/2.9 Å resoln.  Inhibition of enzymic activity is a consequence of intramol. interactions of the enzyme's Src-homol. domains SH2 and SH3, with concomitant displacement of elements of the catalytic domain.  The conformation of the active site has similarities with that of inactive cyclin-dependent protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotdo3t4KV6jLVg90H21EOLACvtfcHk0lhG0YruYXL6vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXht1ymtLs%253D&md5=5703b01ea960313345d43e0f8047ffcf</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2F385602a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F385602a0%26sid%3Dliteratum%253Aachs%26aulast%3DSicheri%26aufirst%3DF.%26aulast%3DMoarefi%26aufirst%3DI.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DCrystal%2520structure%2520of%2520the%2520Src%2520family%2520tyrosine%2520kinase%2520Hck%26jtitle%3DNature%26date%3D1997%26volume%3D385%26spage%3D602%26epage%3D609%26doi%3D10.1038%2F385602a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Proteome-scale docking: myth and reality</span> <span class="citation_source-journal">Drug Discovery Today: Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e403</span><span class="NLM_x">–</span> <span class="NLM_lpage">409</span><span class="refDoi"> DOI: 10.1016/j.ddtec.2013.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.ddtec.2013.01.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=e403-409&author=D.+Rognan&title=Proteome-scale+docking%3A+myth+and+reality&doi=10.1016%2Fj.ddtec.2013.01.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2013.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2013.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DProteome-scale%2520docking%253A%2520myth%2520and%2520reality%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2013%26volume%3D10%26spage%3De403%26epage%3D409%26doi%3D10.1016%2Fj.ddtec.2013.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Richter, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sieghart, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varagic, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morzinger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecker, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernst, M.</span><span> </span><span class="NLM_article-title">Diazepam-bound GABAA receptor models identify new benzodiazepine binding-site ligands</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">455</span><span class="NLM_x">–</span> <span class="NLM_lpage">464</span><span class="refDoi"> DOI: 10.1038/nchembio.917</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnchembio.917" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=22446838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XksVegs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=455-464&author=L.+Richterauthor=C.+de+Graafauthor=W.+Sieghartauthor=Z.+Varagicauthor=M.+Morzingerauthor=I.+J.+de+Eschauthor=G.+F.+Eckerauthor=M.+Ernst&title=Diazepam-bound+GABAA+receptor+models+identify+new+benzodiazepine+binding-site+ligands&doi=10.1038%2Fnchembio.917"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Diazepam-bound GABAA receptor models identify new benzodiazepine binding-site ligands</span></div><div class="casAuthors">Richter, Lars; de Graaf, Chris; Sieghart, Werner; Varagic, Zdravko; Moerzinger, Martina; de Esch, Iwan J. P.; Ecker, Gerhard F.; Ernst, Margot</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">455-464</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Benzodiazepines exert their anxiolytic, anticonvulsant, muscle-relaxant and sedative-hypnotic properties by allosterically enhancing the action of GABA at GABAA receptors via their benzodiazepine-binding site.  Although these drugs have been used clin. since 1960, the mol. basis of this interaction is still not known.  By using multiple homol. models and an unbiased docking protocol, we identified a binding hypothesis for the diazepam-bound structure of the benzodiazepine site, which was confirmed by exptl. evidence.  Moreover, two independent virtual screening approaches based on this structure identified known benzodiazepine-site ligands from different structural classes and predicted potential new ligands for this site.  Receptor-binding assays and electrophysiol. studies on recombinant receptors confirmed these predictions and thus identified new chemotypes for the benzodiazepine-binding site.  Our results support the validity of the diazepam-bound structure of the benzodiazepine-binding pocket, demonstrate its suitability for drug discovery and pave the way for structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprfKzTBHZvLbVg90H21EOLACvtfcHk0liEPGiJJkf-aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XksVegs7c%253D&md5=1fe4a778d6a7c585bcd3d44bcf22d631</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.917&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.917%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DL.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26aulast%3DSieghart%26aufirst%3DW.%26aulast%3DVaragic%26aufirst%3DZ.%26aulast%3DMorzinger%26aufirst%3DM.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26aulast%3DErnst%26aufirst%3DM.%26atitle%3DDiazepam-bound%2520GABAA%2520receptor%2520models%2520identify%2520new%2520benzodiazepine%2520binding-site%2520ligands%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D455%26epage%3D464%26doi%3D10.1038%2Fnchembio.917" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Min, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, Z.</span><span> </span><span class="NLM_article-title">Structure-based virtual screening of novel inhibitors of the uridyltransferase activity of Xanthomonas oryzae pv. oryzae GlmU</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">53C</span><span class="NLM_x">, </span> <span class="NLM_fpage">150</span><span class="NLM_x">–</span> <span class="NLM_lpage">158</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2012.03.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.ejmech.2012.03.051" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53C&publication_year=2012&pages=150-158&author=J.+Minauthor=D.+Linauthor=Q.+Zhangauthor=J.+Zhangauthor=Z.+Yu&title=Structure-based+virtual+screening+of+novel+inhibitors+of+the+uridyltransferase+activity+of+Xanthomonas+oryzae+pv.+oryzae+GlmU&doi=10.1016%2Fj.ejmech.2012.03.051"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2012.03.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2012.03.051%26sid%3Dliteratum%253Aachs%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DZ.%26atitle%3DStructure-based%2520virtual%2520screening%2520of%2520novel%2520inhibitors%2520of%2520the%2520uridyltransferase%2520activity%2520of%2520Xanthomonas%2520oryzae%2520pv.%2520oryzae%2520GlmU%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2012%26volume%3D53C%26spage%3D150%26epage%3D158%26doi%3D10.1016%2Fj.ejmech.2012.03.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Lavecchia, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Giovanni, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pesapane, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montuori, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ragno, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martucci, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masullo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vendittis, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novellino, E.</span><span> </span><span class="NLM_article-title">Discovery of new inhibitors of Cdc25B dual specificity phosphatases by structure-based virtual screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">4142</span><span class="NLM_x">–</span> <span class="NLM_lpage">4158</span><span class="refDoi"> DOI: 10.1021/jm201624h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201624h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvV2ksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=4142-4158&author=A.+Lavecchiaauthor=C.+Di+Giovanniauthor=A.+Pesapaneauthor=N.+Montuoriauthor=P.+Ragnoauthor=N.+M.+Martucciauthor=M.+Masulloauthor=E.+De+Vendittisauthor=E.+Novellino&title=Discovery+of+new+inhibitors+of+Cdc25B+dual+specificity+phosphatases+by+structure-based+virtual+screening&doi=10.1021%2Fjm201624h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of New Inhibitors of Cdc25B Dual Specificity Phosphatases by Structure-Based Virtual Screening</span></div><div class="casAuthors">Lavecchia, Antonio; Di Giovanni, Carmen; Pesapane, Ada; Montuori, Nunzia; Ragno, Pia; Martucci, Nicola Massimiliano; Masullo, Mariorosario; De Vendittis, Emmanuele; Novellino, Ettore</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4142-4158</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cell division cycle 25 (Cdc25) proteins are highly conserved dual specificity phosphatases that regulate cyclin-dependent kinases and represent attractive drug targets for anticancer therapies.  To discover more potent and diverse inhibitors of Cdc25 biol. activity, virtual screening was performed by docking 2.1 million compds. into the Cdc25B active site.  An initial subset of top-ranked compds. was selected and assayed, and 15 were found to have enzyme inhibition activity at micromolar concn.  Among these, four structurally diverse inhibitors with a different inhibition profile were found to inhibit human MCF-7, PC-3, and K562 cancer cell proliferation and significantly affect the cell cycle progression.  A subsequent hierarchical similarity search with the most active reversible Cdc25B inhibitor found led to the identification of an addnl. set of 19 ligands, three of which were confirmed as Cdc25B inhibitors with IC50 values of 7.9, 4.2, and 9.9 μM, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGo8BLCiMiC7Vg90H21EOLACvtfcHk0liEPGiJJkf-aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvV2ksr4%253D&md5=5e44438a537b9f82c54767b87d0135ea</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm201624h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201624h%26sid%3Dliteratum%253Aachs%26aulast%3DLavecchia%26aufirst%3DA.%26aulast%3DDi%2BGiovanni%26aufirst%3DC.%26aulast%3DPesapane%26aufirst%3DA.%26aulast%3DMontuori%26aufirst%3DN.%26aulast%3DRagno%26aufirst%3DP.%26aulast%3DMartucci%26aufirst%3DN.%2BM.%26aulast%3DMasullo%26aufirst%3DM.%26aulast%3DDe%2BVendittis%26aufirst%3DE.%26aulast%3DNovellino%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520new%2520inhibitors%2520of%2520Cdc25B%2520dual%2520specificity%2520phosphatases%2520by%2520structure-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D4142%26epage%3D4158%26doi%3D10.1021%2Fjm201624h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">da Silva, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">dos Santos, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campos, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mafud, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polikarpov, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figueira, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nascimento, A. S.</span><span> </span><span class="NLM_article-title">Structure-based identification of novel PPAR gamma ligands</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">5795</span><span class="NLM_x">–</span> <span class="NLM_lpage">5802</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2013.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2013.09.010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=5795-5802&author=F.+M.+da+Silvaauthor=J.+C.+dos+Santosauthor=J.+L.+Camposauthor=A.+C.+Mafudauthor=I.+Polikarpovauthor=A.+C.+Figueiraauthor=A.+S.+Nascimento&title=Structure-based+identification+of+novel+PPAR+gamma+ligands&doi=10.1016%2Fj.bmcl.2013.09.010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.09.010%26sid%3Dliteratum%253Aachs%26aulast%3Dda%2BSilva%26aufirst%3DF.%2BM.%26aulast%3Ddos%2BSantos%26aufirst%3DJ.%2BC.%26aulast%3DCampos%26aufirst%3DJ.%2BL.%26aulast%3DMafud%26aufirst%3DA.%2BC.%26aulast%3DPolikarpov%26aufirst%3DI.%26aulast%3DFigueira%26aufirst%3DA.%2BC.%26aulast%3DNascimento%26aufirst%3DA.%2BS.%26atitle%3DStructure-based%2520identification%2520of%2520novel%2520PPAR%2520gamma%2520ligands%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D5795%26epage%3D5802%26doi%3D10.1016%2Fj.bmcl.2013.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Lee, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baday, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guibao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, J. J.</span><span> </span><span class="NLM_article-title">Structure-based discovery of novel small molecule Wnt signaling inhibitors by targeting the cysteine-rich domain of frizzled</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">290</span><span class="NLM_x">, </span> <span class="NLM_fpage">30596</span><span class="NLM_x">–</span> <span class="NLM_lpage">30606</span><span class="refDoi"> DOI: 10.1074/jbc.M115.673202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1074%2Fjbc.M115.673202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26504084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVGhtrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=30596-30606&author=H.+J.+Leeauthor=J.+Baoauthor=A.+Millerauthor=C.+Zhangauthor=J.+Wuauthor=Y.+C.+Badayauthor=C.+Guibaoauthor=L.+Liauthor=D.+Wuauthor=J.+J.+Zheng&title=Structure-based+discovery+of+novel+small+molecule+Wnt+signaling+inhibitors+by+targeting+the+cysteine-rich+domain+of+frizzled&doi=10.1074%2Fjbc.M115.673202"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of novel small molecule Wnt signaling inhibitors by targeting the cysteine-rich domain of Frizzled</span></div><div class="casAuthors">Lee, Ho-Jin; Bao, Ju; Miller, Ami; Zhang, Chi; Wu, Jibo; Baday, Yiressy C.; Guibao, Cristina; Li, Lin; Wu, Dianqing; Zheng, Jie J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">30596-30606</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Frizzled is the earliest discovered glycosylated Wnt protein receptor and is crit. for the initiation of Wnt signaling.  Antagonizing Frizzled is effective in inhibiting the growth of multiple tumor types.  The extracellular N-terminus of Frizzled contains a conserved Cys-rich domain that directly interacts with Wnt ligands.  Structure-based virtual screening and cell-based assays were used to identify 5 small mols. that could inhibit canonical Wnt signaling and have low IC50 values in the micromolar range.  NMR expts. confirmed that these compds. specifically bound to the Wnt binding site on the Frizzled8 Cys-rich domain with submicromolar dissocn. consts.  This study confirmed the feasibility of targeting the Frizzled Cys-rich domain as an effective way of regulating canonical Wnt signaling.  These small mols. could be further optimized into more potent therapeutic agents for regulating abnormal Wnt signaling by targeting Frizzled.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYJRelDBXuWLVg90H21EOLACvtfcHk0lhM7Ktw8SVkEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVGhtrrP&md5=4ceed0d6226dcd648a925e5d6392caa2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.673202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.673202%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%2BJ.%26aulast%3DBao%26aufirst%3DJ.%26aulast%3DMiller%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DBaday%26aufirst%3DY.%2BC.%26aulast%3DGuibao%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DZheng%26aufirst%3DJ.%2BJ.%26atitle%3DStructure-based%2520discovery%2520of%2520novel%2520small%2520molecule%2520Wnt%2520signaling%2520inhibitors%2520by%2520targeting%2520the%2520cysteine-rich%2520domain%2520of%2520frizzled%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D30596%26epage%3D30606%26doi%3D10.1074%2Fjbc.M115.673202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Harikishore, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoon, H. S.</span><span> </span><span class="NLM_article-title">Combination of pharmacophore hypothesis and molecular docking to identify novel inhibitors of HCV NS5B polymerase</span> <span class="citation_source-journal">Mol. Diversity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">529</span><span class="NLM_x">–</span> <span class="NLM_lpage">539</span><span class="refDoi"> DOI: 10.1007/s11030-015-9591-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs11030-015-9591-5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2015&pages=529-539&author=A.+Harikishoreauthor=E.+Liauthor=J.+J.+Leeauthor=N.+J.+Choauthor=H.+S.+Yoon&title=Combination+of+pharmacophore+hypothesis+and+molecular+docking+to+identify+novel+inhibitors+of+HCV+NS5B+polymerase&doi=10.1007%2Fs11030-015-9591-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1007%2Fs11030-015-9591-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11030-015-9591-5%26sid%3Dliteratum%253Aachs%26aulast%3DHarikishore%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DLee%26aufirst%3DJ.%2BJ.%26aulast%3DCho%26aufirst%3DN.%2BJ.%26aulast%3DYoon%26aufirst%3DH.%2BS.%26atitle%3DCombination%2520of%2520pharmacophore%2520hypothesis%2520and%2520molecular%2520docking%2520to%2520identify%2520novel%2520inhibitors%2520of%2520HCV%2520NS5B%2520polymerase%26jtitle%3DMol.%2520Diversity%26date%3D2015%26volume%3D19%26spage%3D529%26epage%3D539%26doi%3D10.1007%2Fs11030-015-9591-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mishra, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schiltz, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Makanji, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheidt, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazar, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodruff, T. K.</span><span> </span><span class="NLM_article-title">Virtual high-throughput screening to identify novel activin antagonists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">5637</span><span class="NLM_x">–</span> <span class="NLM_lpage">5648</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00753</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00753" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5637-5648&author=J.+Zhuauthor=R.+K.+Mishraauthor=G.+E.+Schiltzauthor=Y.+Makanjiauthor=K.+A.+Scheidtauthor=A.+P.+Mazarauthor=T.+K.+Woodruff&title=Virtual+high-throughput+screening+to+identify+novel+activin+antagonists&doi=10.1021%2Facs.jmedchem.5b00753"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00753%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DMishra%26aufirst%3DR.%2BK.%26aulast%3DSchiltz%26aufirst%3DG.%2BE.%26aulast%3DMakanji%26aufirst%3DY.%26aulast%3DScheidt%26aufirst%3DK.%2BA.%26aulast%3DMazar%26aufirst%3DA.%2BP.%26aulast%3DWoodruff%26aufirst%3DT.%2BK.%26atitle%3DVirtual%2520high-throughput%2520screening%2520to%2520identify%2520novel%2520activin%2520antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5637%26epage%3D5648%26doi%3D10.1021%2Facs.jmedchem.5b00753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Soni, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suryadevara, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sriram, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Consortium, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nandicoori, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yogeeswari, P.</span><span> </span><span class="NLM_article-title">Structure-based design of diverse inhibitors of Mycobacterium tuberculosis N-acetylglucosamine-1-phosphate uridyltransferase: combined molecular docking, dynamic simulation, and biological activity</span> <span class="citation_source-journal">J. Mol. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">174</span><span class="refDoi"> DOI: 10.1007/s00894-015-2704-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs00894-015-2704-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26078037" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BC2MbktVSktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=174&author=V.+Soniauthor=P.+Suryadevaraauthor=D.+Sriramauthor=O.+Consortiumauthor=S.+Kumarauthor=V.+K.+Nandicooriauthor=P.+Yogeeswari&title=Structure-based+design+of+diverse+inhibitors+of+Mycobacterium+tuberculosis+N-acetylglucosamine-1-phosphate+uridyltransferase%3A+combined+molecular+docking%2C+dynamic+simulation%2C+and+biological+activity&doi=10.1007%2Fs00894-015-2704-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of diverse inhibitors of Mycobacterium tuberculosis N-acetylglucosamine-1-phosphate uridyltransferase: combined molecular docking, dynamic simulation, and biological activity</span></div><div class="casAuthors">Soni Vijay; Suryadevara Priyanka; Sriram Dharmarajan; Kumar Santhosh; Nandicoori Vinay Kumar; Yogeeswari Perumal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">174</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Persistent nature of Mycobacterium tuberculosis is one of the major factors which make the drug development process monotonous against this organism.  The highly lipophilic cell wall, which constituting outer mycolic acid and inner peptidoglycan layers, acts as a barrier for the drugs to enter the bacteria.  The rigidity of the cell wall is imparted by the peptidoglycan layer, which is covalently linked to mycolic acid by arabinogalactan.  Uridine diphosphate-N-acetylglucosamine (UDP-GlcNAc) serves as the starting material in the biosynthesis of this peptidoglycan layers.  This UDP-GlcNAc is synthesized by N-acetylglucosamine-1-phosphate uridyltransferase (GlmU(Mtb)), a bi-functional enzyme with two functional sites, acetyltransferase site and uridyltransferase site.  Here, we report design and screening of nine inhibitors against UTP and NAcGlc-1-P of uridyltransferase active site of glmU(Mtb).  Compound 4 was showing good inhibition and was selected for further analysis.  The isothermal titration calorimetry (ITC) experiments showed the binding energy pattern of compound 4 to the uridyltransferase active site is similar to that of substrate UTP.  In silico molecular dynamics (MD) simulation studies, for compound 4, carried out for 10 ns showed the protein-compound complex to be stable throughout the simulation with relative rmsd in acceptable range.  Hence, these compounds can serve as a starting point in the drug discovery processes against Mycobacterium tuberculosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTxgQSOJ7Hiq-Lt9HypzAgQfW6udTcc2eaFuackBhN_rbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MbktVSktQ%253D%253D&md5=f7a7f8e427f2d5cfe01c8b63f95a4369</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1007%2Fs00894-015-2704-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00894-015-2704-3%26sid%3Dliteratum%253Aachs%26aulast%3DSoni%26aufirst%3DV.%26aulast%3DSuryadevara%26aufirst%3DP.%26aulast%3DSriram%26aufirst%3DD.%26aulast%3DConsortium%26aufirst%3DO.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DNandicoori%26aufirst%3DV.%2BK.%26aulast%3DYogeeswari%26aufirst%3DP.%26atitle%3DStructure-based%2520design%2520of%2520diverse%2520inhibitors%2520of%2520Mycobacterium%2520tuberculosis%2520N-acetylglucosamine-1-phosphate%2520uridyltransferase%253A%2520combined%2520molecular%2520docking%252C%2520dynamic%2520simulation%252C%2520and%2520biological%2520activity%26jtitle%3DJ.%2520Mol.%2520Model.%26date%3D2015%26volume%3D21%26spage%3D174%26doi%3D10.1007%2Fs00894-015-2704-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Ruggeri, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drinkwater, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sivaraman, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamert, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGowan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paiardini, A.</span><span> </span><span class="NLM_article-title">Identification and validation of a potent dual inhibitor of the P. falciparum M1 and M17 aminopeptidases using virtual screening</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0138957</span><span class="refDoi"> DOI: 10.1371/journal.pone.0138957</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pone.0138957" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0138957&author=C.+Ruggeriauthor=N.+Drinkwaterauthor=K.+K.+Sivaramanauthor=R.+S.+Bamertauthor=S.+McGowanauthor=A.+Paiardini&title=Identification+and+validation+of+a+potent+dual+inhibitor+of+the+P.+falciparum+M1+and+M17+aminopeptidases+using+virtual+screening&doi=10.1371%2Fjournal.pone.0138957"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0138957&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0138957%26sid%3Dliteratum%253Aachs%26aulast%3DRuggeri%26aufirst%3DC.%26aulast%3DDrinkwater%26aufirst%3DN.%26aulast%3DSivaraman%26aufirst%3DK.%2BK.%26aulast%3DBamert%26aufirst%3DR.%2BS.%26aulast%3DMcGowan%26aufirst%3DS.%26aulast%3DPaiardini%26aufirst%3DA.%26atitle%3DIdentification%2520and%2520validation%2520of%2520a%2520potent%2520dual%2520inhibitor%2520of%2520the%2520P.%2520falciparum%2520M1%2520and%2520M17%2520aminopeptidases%2520using%2520virtual%2520screening%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0138957%26doi%3D10.1371%2Fjournal.pone.0138957" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Schmidt, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernat, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brox, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tschammer, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolb, P.</span><span> </span><span class="NLM_article-title">Identifying modulators of CXC receptors 3 and 4 with tailored selectivity using multi-target docking</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">715</span><span class="NLM_x">–</span> <span class="NLM_lpage">724</span><span class="refDoi"> DOI: 10.1021/cb500577j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500577j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFWqsb%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=715-724&author=D.+Schmidtauthor=V.+Bernatauthor=R.+Broxauthor=N.+Tschammerauthor=P.+Kolb&title=Identifying+modulators+of+CXC+receptors+3+and+4+with+tailored+selectivity+using+multi-target+docking&doi=10.1021%2Fcb500577j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Modulators of CXC Receptors 3 and 4 with Tailored Selectivity Using Multi-Target Docking</span></div><div class="casAuthors">Schmidt, Denis; Bernat, Viachaslau; Brox, Regine; Tschammer, Nuska; Kolb, Peter</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">715-724</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The G protein-coupled receptors of the C-X-C subfamily form a group among the chemokine receptors whose endogenous ligands are peptides with a common Cys-X-Cys motif.  The CXC chemokine receptors 3 and 4 (CXCR3, CXCR4), which are investigated in this study, are linked to severe diseases such as cancer, multiple sclerosis, and HIV infections.  Of particular interest, this receptor pair potentially forms a target for a polypharmacol. drug treatment.  Considering known ligands from public databases, such dual binders have not been identified yet.  We therefore applied large-scale docking to the structure of CXCR4 and a homol. model of CXCR3 with the goal to predict such dual binders, as well as compds. selective for either one of the receptors.  Using signaling and biochem. assays, we showed that more than 50% of these predictions were correct in each category, yielding ligands with excellent binding efficiencies.  These results highlight that docking is a suitable tool for the identification of ligands with tailored binding profiles to GPCRs, even when using homol. models.  More importantly, we present novel CXCR3-CXCR4 dual modulators that might pave the road to understanding the mechanisms of polypharmacol. inhibition of these receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6Px6d1pbV3rVg90H21EOLACvtfcHk0lhCjEhMeJ8_rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFWqsb%252FM&md5=9259d56e4c28e68360f3ac214ea62ea4</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fcb500577j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500577j%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DD.%26aulast%3DBernat%26aufirst%3DV.%26aulast%3DBrox%26aufirst%3DR.%26aulast%3DTschammer%26aufirst%3DN.%26aulast%3DKolb%26aufirst%3DP.%26atitle%3DIdentifying%2520modulators%2520of%2520CXC%2520receptors%25203%2520and%25204%2520with%2520tailored%2520selectivity%2520using%2520multi-target%2520docking%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D715%26epage%3D724%26doi%3D10.1021%2Fcb500577j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Perryman, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ekins, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freundlich, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonge, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">A virtual screen discovers novel, fragment-sized inhibitors of Mycobacterium tuberculosis InhA</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">645</span><span class="NLM_x">–</span> <span class="NLM_lpage">659</span><span class="refDoi"> DOI: 10.1021/ci500672v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500672v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVehsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=645-659&author=A.+L.+Perrymanauthor=W.+Yuauthor=X.+Wangauthor=S.+Ekinsauthor=S.+Forliauthor=S.+G.+Liauthor=J.+S.+Freundlichauthor=P.+J.+Tongeauthor=A.+J.+Olson&title=A+virtual+screen+discovers+novel%2C+fragment-sized+inhibitors+of+Mycobacterium+tuberculosis+InhA&doi=10.1021%2Fci500672v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">A Virtual Screen Discovers Novel, Fragment-Sized Inhibitors of Mycobacterium tuberculosis InhA</span></div><div class="casAuthors">Perryman, Alexander L.; Yu, Weixuan; Wang, Xin; Ekins, Sean; Forli, Stefano; Li, Shao-Gang; Freundlich, Joel S.; Tonge, Peter J.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">645-659</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Isoniazid (INH) is usually administered to treat latent Mycobacterium tuberculosis (Mtb) infections and is used in combination therapy to treat active tuberculosis (TB).  Unfortunately, resistance to this drug is hampering its clin. effectiveness.  INH is a prodrug that must be activated by Mtb catalase-peroxidase (KatG) before it can inhibit InhA (Mtb enoyl-acyl-carrier-protein reductase).  Isoniazid-resistant cases of TB found in clin. settings usually involve mutations in or deletion of katG, which abrogate INH activation.  Compds. that inhibit InhA without requiring prior activation by KatG would not be affected by this resistance mechanism and hence would display continued potency against these drug-resistant isolates of Mtb.  Virtual screening expts. vs. InhA in the GO Fight Against Malaria (GO FAM) project were designed to discover new scaffolds that display base-stacking interactions with the NAD cofactor.  GO FAM expts. included targets from other pathogens, including Mtb, when they had structural similarity to a malaria target.  Eight of the 16 sol. compds. identified by docking against InhA plus visual inspection were modest inhibitors and did not require prior activation by KatG.  The best two inhibitors discovered are both fragment-sized compds. and displayed Ki values of 54 and 59 μM, resp.  Importantly, the novel inhibitors discovered have low structural similarity to known InhA inhibitors and thus help expand the no. of chemotypes on which future medicinal chem. efforts can be focused.  These new fragment hits could eventually help advance the fight against INH-resistant Mtb strains, which pose a significant global health threat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9fifQHkgzX7Vg90H21EOLACvtfcHk0lhCjEhMeJ8_rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVehsrc%253D&md5=f68a7af63dca612e3017260c32593858</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fci500672v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500672v%26sid%3Dliteratum%253Aachs%26aulast%3DPerryman%26aufirst%3DA.%2BL.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DEkins%26aufirst%3DS.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DS.%2BG.%26aulast%3DFreundlich%26aufirst%3DJ.%2BS.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DA%2520virtual%2520screen%2520discovers%2520novel%252C%2520fragment-sized%2520inhibitors%2520of%2520Mycobacterium%2520tuberculosis%2520InhA%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D645%26epage%3D659%26doi%3D10.1021%2Fci500672v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span> </span><span class="NLM_article-title">Identification of novel small molecules as inhibitors of hepatitis C virus by structure-based virtual screening</span> <span class="citation_source-journal">Int. J. Mol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">22845</span><span class="NLM_x">–</span> <span class="NLM_lpage">22856</span><span class="refDoi"> DOI: 10.3390/ijms141122845</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.3390%2Fijms141122845" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24264035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltlOgur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=22845-22856&author=J.+Liauthor=X.+Liuauthor=S.+Liauthor=Y.+Wangauthor=N.+Zhouauthor=C.+Luoauthor=X.+Luoauthor=M.+Zhengauthor=H.+Jiangauthor=K.+Chen&title=Identification+of+novel+small+molecules+as+inhibitors+of+hepatitis+C+virus+by+structure-based+virtual+screening&doi=10.3390%2Fijms141122845"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel small molecules as inhibitors of hepatitis C virus by structure-based virtual screening</span></div><div class="casAuthors">Li, Jing; Liu, Xian; Li, Shanshan; Wang, Yulan; Zhou, Nannan; Luo, Cheng; Luo, Xiaomin; Zheng, Mingyue; Jiang, Hualiang; Chen, Kaixian</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">22845-22856, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Hepatitis C virus (HCV) NS3/NS4A serine protease is essential for viral replication, which is regarded as a promising drug target for developing direct-acting anti-HCV agents.  In this study, sixteen novel compds. with cell-based HCV replicon activity ranging from 3.0 to 28.2 μM (IC50) were successfully identified by means of structure-based virtual screening.  Compd. 5 and compd. 11, with an IC50 of 3.0 μM and 5.1 μM, resp., are the two most potent mols. with low cytotoxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYDkmJf8CQ3bVg90H21EOLACvtfcHk0lhCjEhMeJ8_rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltlOgur0%253D&md5=eab2e3f26ed0b67aeb3e60cc74197488</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.3390%2Fijms141122845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms141122845%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DN.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26atitle%3DIdentification%2520of%2520novel%2520small%2520molecules%2520as%2520inhibitors%2520of%2520hepatitis%2520C%2520virus%2520by%2520structure-based%2520virtual%2520screening%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2013%26volume%3D14%26spage%3D22845%26epage%3D22856%26doi%3D10.3390%2Fijms141122845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Soelaiman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, B. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mrksich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, W. J.</span><span> </span><span class="NLM_article-title">Structure-based inhibitor discovery against adenylyl cyclase toxins from pathogenic bacteria that cause anthrax and whooping cough</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">25990</span><span class="NLM_x">–</span> <span class="NLM_lpage">25997</span><span class="refDoi"> DOI: 10.1074/jbc.M301232200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1074%2Fjbc.M301232200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=12676933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlt1Cltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=25990-25997&author=S.+Soelaimanauthor=B.+Q.+Weiauthor=P.+Bergsonauthor=Y.+S.+Leeauthor=Y.+Shenauthor=M.+Mrksichauthor=B.+K.+Shoichetauthor=W.+J.+Tang&title=Structure-based+inhibitor+discovery+against+adenylyl+cyclase+toxins+from+pathogenic+bacteria+that+cause+anthrax+and+whooping+cough&doi=10.1074%2Fjbc.M301232200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based Inhibitor Discovery against Adenylyl Cyclase Toxins from Pathogenic Bacteria That Cause Anthrax and Whooping Cough</span></div><div class="casAuthors">Soelaiman, Sandriyana; Wei, Binqing Q.; Bergson, Pamela; Lee, Young-Sam; Shen, Yuequan; Mrksich, Milan; Shoichet, Brian K.; Tang, Wei-Jen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">25990-25997</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Edema factor (EF) and CyaA are adenylyl cyclase toxins secreted by pathogenic bacteria that cause anthrax and whooping cough, resp.  Using the structure of the catalytic site of EF, we screened a data base of com. available, small mol. wt. chems. for those that could specifically inhibit adenylyl cyclase activity of EF.  From 24 compds. tested, we have identified one quinazoline compd., Et 5-aminopyrazolo[1,5-a]quinazoline-3-carboxylate, that specifically inhibits adenylyl cyclase activity of EF and CyaA with ∼20 μM Ki.  This compd. neither affects the activity of host resident adenylyl cyclases type I, II, and V nor exhibits promiscuous inhibition.  The compd. is a competitive inhibitor, consistent with the prediction that it binds to the adenine portion of the ATP binding site on EF.  EF is activated by the host calcium sensor, calmodulin.  Surface plasmon resonance spectroscopic anal. shows that this compd. does not affect the binding of calmodulin to EF.  This compd. is dissimilar from a previously described, non-nucleoside inhibitor of host adenylyl cyclase.  It may serve as a lead to design antitoxins to address the role of adenylyl cyclase toxins in bacterial pathogenesis and to fight against anthrax and whooping cough.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFnvSIcMfQNbVg90H21EOLACvtfcHk0ljZgwgnLCCKTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlt1Cltro%253D&md5=2b0dbb955395039f3b50c78ca947036f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M301232200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M301232200%26sid%3Dliteratum%253Aachs%26aulast%3DSoelaiman%26aufirst%3DS.%26aulast%3DWei%26aufirst%3DB.%2BQ.%26aulast%3DBergson%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DY.%2BS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DMrksich%26aufirst%3DM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DTang%26aufirst%3DW.%2BJ.%26atitle%3DStructure-based%2520inhibitor%2520discovery%2520against%2520adenylyl%2520cyclase%2520toxins%2520from%2520pathogenic%2520bacteria%2520that%2520cause%2520anthrax%2520and%2520whooping%2520cough%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D25990%26epage%3D25997%26doi%3D10.1074%2Fjbc.M301232200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Iwata, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arisawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamada, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kita, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizutani, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomioka, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyamoto, S.</span><span> </span><span class="NLM_article-title">Discovery of novel aldose reductase inhibitors using a protein structure-based approach: 3D-database search followed by design and synthesis</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1718</span><span class="NLM_x">–</span> <span class="NLM_lpage">1728</span><span class="refDoi"> DOI: 10.1021/jm000483h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm000483h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3MXivFynu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=1718-1728&author=Y.+Iwataauthor=M.+Arisawaauthor=R.+Hamadaauthor=Y.+Kitaauthor=M.+Y.+Mizutaniauthor=N.+Tomiokaauthor=A.+Itaiauthor=S.+Miyamoto&title=Discovery+of+novel+aldose+reductase+inhibitors+using+a+protein+structure-based+approach%3A+3D-database+search+followed+by+design+and+synthesis&doi=10.1021%2Fjm000483h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Aldose Reductase Inhibitors Using a Protein Structure-Based Approach: 3D-Database Search Followed by Design and Synthesis</span></div><div class="casAuthors">Iwata, Yoriko; Arisawa, Mitsuhiro; Hamada, Ryuji; Kita, Yasuyuki; Mizutani, Miho Y.; Tomioka, Nobuo; Itai, Akiko; Miyamoto, Shuichi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1718-1728</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aldose reductase (AR) has been implicated in the etiol. of diabetic complications.  Due to the limited no. of currently available drugs for the treatment of diabetic complications, we have carried out structure-based drug design and synthesis to find new types of AR inhibitors.  With the ADAM&EVE program, a three-dimensional database (ACD3D) was searched using the ligand binding site of the AR crystal structure.  Out of 179 compds. selected through this search followed by visual inspection, 36 compds. were purchased and subjected to a biol. assay.  Ten compds. showed more than 40% inhibition of AR at a 15 μg/mL concn.  In a subsequent lead optimization, a series of analogs of the most active compd. were synthesized based on the docking mode derived by ADAM&EVE.  Many of these congeners exhibited higher activities compared to the mother compd. Indeed, the most potent, synthesized compd. showed a ∼20-fold increase in inhibitory activity (IC50 = 0.21 vs. 4.3 μM).  Furthermore, a hydrophobic subsite was newly inferred, which would be useful for the design of inhibitors with improved affinity for AR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6c8cdbn1oQLVg90H21EOLACvtfcHk0ljZgwgnLCCKTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXivFynu7g%253D&md5=6eff99eefff1e4c4531df078969c73d7</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm000483h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000483h%26sid%3Dliteratum%253Aachs%26aulast%3DIwata%26aufirst%3DY.%26aulast%3DArisawa%26aufirst%3DM.%26aulast%3DHamada%26aufirst%3DR.%26aulast%3DKita%26aufirst%3DY.%26aulast%3DMizutani%26aufirst%3DM.%2BY.%26aulast%3DTomioka%26aufirst%3DN.%26aulast%3DItai%26aufirst%3DA.%26aulast%3DMiyamoto%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520novel%2520aldose%2520reductase%2520inhibitors%2520using%2520a%2520protein%2520structure-based%2520approach%253A%25203D-database%2520search%2520followed%2520by%2520design%2520and%2520synthesis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D1718%26epage%3D1728%26doi%3D10.1021%2Fjm000483h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Enyedy, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ling, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nacro, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomita, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, Y. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roller, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Discovery of small-molecule inhibitors of Bcl-2 through structure-based computer screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">4313</span><span class="NLM_x">–</span> <span class="NLM_lpage">4324</span><span class="refDoi"> DOI: 10.1021/jm010016f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm010016f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnvFaks70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=4313-4324&author=I.+J.+Enyedyauthor=Y.+Lingauthor=K.+Nacroauthor=Y.+Tomitaauthor=X.+Wuauthor=Y.+Caoauthor=R.+Guoauthor=B.+Liauthor=X.+Zhuauthor=Y.+Huangauthor=Y.+Q.+Longauthor=P.+P.+Rollerauthor=D.+Yangauthor=S.+Wang&title=Discovery+of+small-molecule+inhibitors+of+Bcl-2+through+structure-based+computer+screening&doi=10.1021%2Fjm010016f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small-Molecule Inhibitors of Bcl-2 through Structure-Based Computer Screening</span></div><div class="casAuthors">Enyedy, Istvan J.; Ling, Yan; Nacro, Kassoum; Tomita, York; Wu, Xihan; Cao, Yeyu; Guo, Ribo; Li, Bihua; Zhu, Xiaofeng; Huang, Ying; Long, Ya-Qiu; Roller, Peter P.; Yang, Dajun; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4313-4324</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bcl-2 belongs to a growing family of proteins which regulates programmed cell death (apoptosis).  Overexpression of Bcl-2 has been obsd. in 70% of breast cancer, 30-60% of prostate cancer, 80% of B-cell lymphomas, 90% of colorectal adenocarcinomas, and many other forms of cancer. Thereby, Bcl-2 is an attractive new anti-cancer target. Herein, we describe the discovery of novel classes of small-mol. inhibitors targeted at the BH3 binding pocket in Bcl-2.  The three-dimensional (3D) structure of Bcl-2 has been modeled on the basis of a high-resoln. NMR soln. structure of Bcl-XL, which shares a high sequence homol. with Bcl-2.  A structure-based computer screening approach has been employed to search the National Cancer Institute 3D database of 206 876 org. compds. to identify potential Bcl-2 small-mol. inhibitors that bind to the BH3 binding site of Bcl-2.  These potential Bcl-2 small-mol. inhibitors were first tested in an in vitro binding assay for their potency in inhibition of the binding of a Bak BH3 peptide to Bcl-2.  Thirty-five potential inhibitors were tested in this binding assay, and seven of them were found to have a binding affinity (IC50 value) from 1.6 to 14.0 M.  The anti-proliferative activity of these seven active compds. has been tested using a human myeloid leukemia cell line, HL-60, which expresses the highest level of Bcl-2 protein among all the cancer cell lines examd.  The most potent compd. had an IC50 value of 4 μM in inhibition of cell growth using the 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide assay.  Five other compds. had moderate activity in inhibition of cell growth.  The most active compd. was further evaluated for its ability to induce apoptosis in cancer cells.  It was found that this compd. induces apoptosis in cancer cells with high Bcl-2 expression and its potency correlates with the Bcl-2 expression level in cancer cells.  Furthermore, using NMR methods, we conclusively demonstrated that this most active compd. binds to the BH3 binding site in Bcl-XL.  Our results showed that small-mol. inhibitors of Bcl-2 modulate the biol. function of Bcl-2, and induce apoptosis in cancer cells with high Bcl-2 expression, while they have little effect on cancer cells with low or undetectable levels of Bcl-2 expression.  Therefore, this active compd. can be used as a valuable pharmacol. tool to elucidate the function of Bcl-2 and also serves as a novel lead compd. for further design and optimization.  Our results suggest that the structure-based computer screening strategy employed in the study is effective for identifying novel, structurally diverse, nonpeptide small-mol. inhibitors that target the BH3 binding site of Bcl-2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVlg73stNG2LVg90H21EOLACvtfcHk0ljZgwgnLCCKTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnvFaks70%253D&md5=b02822b211ecab923903ff08d8a219c8</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm010016f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm010016f%26sid%3Dliteratum%253Aachs%26aulast%3DEnyedy%26aufirst%3DI.%2BJ.%26aulast%3DLing%26aufirst%3DY.%26aulast%3DNacro%26aufirst%3DK.%26aulast%3DTomita%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DLong%26aufirst%3DY.%2BQ.%26aulast%3DRoller%26aufirst%3DP.%2BP.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520small-molecule%2520inhibitors%2520of%2520Bcl-2%2520through%2520structure-based%2520computer%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D4313%26epage%3D4324%26doi%3D10.1021%2Fjm010016f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Pang, Y. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollmeyer, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perola, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGrath, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangel, W. F.</span><span> </span><span class="NLM_article-title">Discovery of a new inhibitor lead of adenovirus proteinase: steps toward selective, irreversible inhibitors of cysteine proteinases</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">502</span><span class="NLM_x">, </span> <span class="NLM_fpage">93</span><span class="NLM_x">–</span> <span class="NLM_lpage">97</span><span class="refDoi"> DOI: 10.1016/S0014-5793(01)02672-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2FS0014-5793%2801%2902672-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=502&publication_year=2001&pages=93-97&author=Y.+P.+Pangauthor=K.+Xuauthor=T.+M.+Kollmeyerauthor=E.+Perolaauthor=W.+J.+McGrathauthor=D.+T.+Greenauthor=W.+F.+Mangel&title=Discovery+of+a+new+inhibitor+lead+of+adenovirus+proteinase%3A+steps+toward+selective%2C+irreversible+inhibitors+of+cysteine+proteinases&doi=10.1016%2FS0014-5793%2801%2902672-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2801%2902672-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252801%252902672-2%26sid%3Dliteratum%253Aachs%26aulast%3DPang%26aufirst%3DY.%2BP.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DKollmeyer%26aufirst%3DT.%2BM.%26aulast%3DPerola%26aufirst%3DE.%26aulast%3DMcGrath%26aufirst%3DW.%2BJ.%26aulast%3DGreen%26aufirst%3DD.%2BT.%26aulast%3DMangel%26aufirst%3DW.%2BF.%26atitle%3DDiscovery%2520of%2520a%2520new%2520inhibitor%2520lead%2520of%2520adenovirus%2520proteinase%253A%2520steps%2520toward%2520selective%252C%2520irreversible%2520inhibitors%2520of%2520cysteine%2520proteinases%26jtitle%3DFEBS%2520Lett.%26date%3D2001%26volume%3D502%26spage%3D93%26epage%3D97%26doi%3D10.1016%2FS0014-5793%2801%2902672-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Honma, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aoyama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hashimoto, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machida, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukasawa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikeura, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikuta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki-Takahashi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwasawa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishimura, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morishima, H.</span><span> </span><span class="NLM_article-title">Structure-based generation of a new class of potent Cdk4 inhibitors: new de novo design strategy and library design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">4615</span><span class="NLM_x">–</span> <span class="NLM_lpage">4627</span><span class="refDoi"> DOI: 10.1021/jm0103256</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0103256" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3MXovFagtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=4615-4627&author=T.+Honmaauthor=K.+Hayashiauthor=T.+Aoyamaauthor=N.+Hashimotoauthor=T.+Machidaauthor=K.+Fukasawaauthor=T.+Iwamaauthor=C.+Ikeuraauthor=M.+Ikutaauthor=I.+Suzuki-Takahashiauthor=Y.+Iwasawaauthor=T.+Hayamaauthor=S.+Nishimuraauthor=H.+Morishima&title=Structure-based+generation+of+a+new+class+of+potent+Cdk4+inhibitors%3A+new+de+novo+design+strategy+and+library+design&doi=10.1021%2Fjm0103256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Generation of a New Class of Potent Cdk4 Inhibitors: New de Novo Design Strategy and Library Design</span></div><div class="casAuthors">Honma, Teruki; Hayashi, Kyoko; Aoyama, Tetsuya; Hashimoto, Noriaki; Machida, Takumitsu; Fukasawa, Kazuhiro; Iwama, Toshiharu; Ikeura, Chinatsu; Ikuta, Mari; Suzuki-Takahashi, Ikuko; Iwasawa, Yoshikazu; Hayama, Takashi; Nishimura, Susumu; Morishima, Hajime</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">4615-4627</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As a first step in structure-based design of highly selective and potent Cdk4 inhibitors, we performed structure-based generation of a novel series of Cdk4 inhibitors.  A Cdk4 homol. model was constructed according to x-ray anal. of an activated form of Cdk2.  Using this model, we applied a new de novo design strategy which combined the de novo design program LEGEND with our inhouse structure selection supporting system SEEDS to generate new scaffold candidates.  In this way, four classes of scaffold candidates including diarylurea were identified.  By constructing diarylurea informer libraries based on the structural requirements of Cdk inhibitors in the ATP binding pocket of the Cdk4 model, we were able to identify a potent Cdk4 inhibitor N-(9-oxo-9H-fluoren-4-yl)-N'-pyridin-2-ylurea (I) with IC50 = 0.10 μM, together with preliminary SAR.  We performed a docking study between I and the Cdk4 model and selected a reasonable binding mode which is consistent with the SAR.  Further modification based on the proposed binding mode provided a more potent compd., N-[(9bR)-5-oxo- 2,3,5,9b-tetrahydro-1H-pyrrolo[2,1-a]isoindol-9-yl]-N'-pyridin-2-ylurea (II) with IC50 = 0.042 μM, x-ray anal. of which was accomplished by the soaking method.  The predicted binding mode of I in Cdk4 was validated by x-ray anal. of the Cdk2-II complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvx3J3Bqyg9LVg90H21EOLACvtfcHk0ljgcXQG5zeZ_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXovFagtrs%253D&md5=0c4458fa88d41c148a672863b3a6daa0</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm0103256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0103256%26sid%3Dliteratum%253Aachs%26aulast%3DHonma%26aufirst%3DT.%26aulast%3DHayashi%26aufirst%3DK.%26aulast%3DAoyama%26aufirst%3DT.%26aulast%3DHashimoto%26aufirst%3DN.%26aulast%3DMachida%26aufirst%3DT.%26aulast%3DFukasawa%26aufirst%3DK.%26aulast%3DIwama%26aufirst%3DT.%26aulast%3DIkeura%26aufirst%3DC.%26aulast%3DIkuta%26aufirst%3DM.%26aulast%3DSuzuki-Takahashi%26aufirst%3DI.%26aulast%3DIwasawa%26aufirst%3DY.%26aulast%3DHayama%26aufirst%3DT.%26aulast%3DNishimura%26aufirst%3DS.%26aulast%3DMorishima%26aufirst%3DH.%26atitle%3DStructure-based%2520generation%2520of%2520a%2520new%2520class%2520of%2520potent%2520Cdk4%2520inhibitors%253A%2520new%2520de%2520novo%2520design%2520strategy%2520and%2520library%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D4615%26epage%3D4627%26doi%3D10.1021%2Fjm0103256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raaka, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samuels, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">In silico discovery of novel Retinoic Acid Receptor agonist structures</span> <span class="citation_source-journal">BMC Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="refDoi"> DOI: 10.1186/1472-6807-1-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1186%2F1472-6807-1-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=11405897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BD3srgs1Crsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2001&pages=1&author=M.+Schapiraauthor=B.+M.+Raakaauthor=H.+H.+Samuelsauthor=R.+Abagyan&title=In+silico+discovery+of+novel+Retinoic+Acid+Receptor+agonist+structures&doi=10.1186%2F1472-6807-1-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">In silico discovery of novel retinoic acid receptor agonist structures</span></div><div class="casAuthors">Schapira M; Raaka B M; Samuels H H; Abagyan R</div><div class="citationInfo"><span class="NLM_cas:title">BMC structural biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Several Retinoic Acid Receptors (RAR) agonists have therapeutic activity against a variety of cancer types; however, unacceptable toxicity profiles have hindered the development of drugs.  RAR agonists presenting novel structural and chemical features could therefore open new avenues for the discovery of leads against breast, lung and prostate cancer or leukemia.  RESULTS:  We have analysed the induced fit of the active site residues upon binding of a known ligand.  The derived binding site models were used to dock over 150,000 molecules in silico (or virtually) to the structure of the receptor with the Internal Coordinates Mechanics (ICM) program.  Thirty ligand candidates were tested in vitro.  CONCLUSIONS:  Two novel agonists resulting from the predicted receptor model were active at 50 nM.  One of them displays novel structural features which may translate into the development of new ligands for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRGuB_7wOJu3Wx8TEN9K18QfW6udTcc2ea2AWZp9TCsxLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3srgs1Crsg%253D%253D&md5=ea9be764e620d3e7823f0d183005f4d3</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1186%2F1472-6807-1-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1472-6807-1-1%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DRaaka%26aufirst%3DB.%2BM.%26aulast%3DSamuels%26aufirst%3DH.%2BH.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DIn%2520silico%2520discovery%2520of%2520novel%2520Retinoic%2520Acid%2520Receptor%2520agonist%2520structures%26jtitle%3DBMC%2520Struct.%2520Biol.%26date%3D2001%26volume%3D1%26spage%3D1%26doi%3D10.1186%2F1472-6807-1-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Freymann, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wenck, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engel, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Focia, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eakin, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craig, S. P.</span><span> </span><span class="NLM_article-title">Efficient identification of inhibitors targeting the closed active site conformation of the HPRT from Trypanosoma cruzi</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">957</span><span class="NLM_x">–</span> <span class="NLM_lpage">968</span><span class="refDoi"> DOI: 10.1016/S1074-5521(00)00045-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2FS1074-5521%2800%2900045-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=11137818" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltFegtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2000&pages=957-968&author=D.+M.+Freymannauthor=M.+A.+Wenckauthor=J.+C.+Engelauthor=J.+Fengauthor=P.+J.+Fociaauthor=A.+E.+Eakinauthor=S.+P.+Craig&title=Efficient+identification+of+inhibitors+targeting+the+closed+active+site+conformation+of+the+HPRT+from+Trypanosoma+cruzi&doi=10.1016%2FS1074-5521%2800%2900045-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient identification of inhibitors targeting the closed active site conformation of the HPRT from Trypanosoma cruzi</span></div><div class="casAuthors">Freymann, Douglas M.; Wenck, Mary Anne; Engel, Juan C.; Feng, Jun; Focia, Pamela J.; Eakin, Ann E.; Craig, Sydney P, III</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">957-968</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Background: Currently, only two drugs are recommended for treatment of infection with Trypanosoma cruzi, the etiol. agent of Chagas' disease.  These compds. kill the trypomastigote forms of the parasite circulating in the bloodstream, but are relatively ineffective against the intracellular stage of the parasite life cycle.  Neither drug is approved by the FDA for use in the US.  The hypoxanthine phosphoribosyltransferase (HPRT) from T. cruzi is a possible new target for antiparasite chemotherapy.  The crystal structure of the HPRT in a conformation approximating the transition state reveals a closed active site that provides a well-defined target for computational structure-based drug discovery.  Results: A flexible ligand docking program incorporating a desolvation correction was used to screen the Available Chems. Directory for inhibitors targeted to the closed conformation of the trypanosomal HPRT.  Of 22 potential inhibitors identified, acquired and tested, 16 yielded Ki's between 0.5 and 17 μM vs. the substrate phosphoribosylpyrophosphate.  Surprisingly, three of eight compds. tested were effective in inhibiting the growth of parasites in infected mammalian cells.  Conclusions: This structure-based docking method provided a remarkably efficient path for the identification of inhibitors targeting the closed conformation of the trypanosomal HPRT.  The inhibition consts. of the lead inhibitors identified are unusually favorable, and the trypanostatic activity of three of the compds. in cell culture suggests that they may provide useful starting points for drug design for the treatment of Chagas' disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBVYimL73xQLVg90H21EOLACvtfcHk0ljfJXaZEyb-Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltFegtQ%253D%253D&md5=da1b73a30a1ebc176191e88eaeffb245</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2800%2900045-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252800%252900045-4%26sid%3Dliteratum%253Aachs%26aulast%3DFreymann%26aufirst%3DD.%2BM.%26aulast%3DWenck%26aufirst%3DM.%2BA.%26aulast%3DEngel%26aufirst%3DJ.%2BC.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DFocia%26aufirst%3DP.%2BJ.%26aulast%3DEakin%26aufirst%3DA.%2BE.%26aulast%3DCraig%26aufirst%3DS.%2BP.%26atitle%3DEfficient%2520identification%2520of%2520inhibitors%2520targeting%2520the%2520closed%2520active%2520site%2520conformation%2520of%2520the%2520HPRT%2520from%2520Trypanosoma%2520cruzi%26jtitle%3DChem.%2520Biol.%26date%3D2000%26volume%3D7%26spage%3D957%26epage%3D968%26doi%3D10.1016%2FS1074-5521%2800%2900045-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Paiva, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanderwall, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blanchard, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozarich, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williamson, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, T. M.</span><span> </span><span class="NLM_article-title">Inhibitors of dihydrodipicolinate reductase, a key enzyme of the diaminopimelate pathway of Mycobacterium tuberculosis</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Protein Struct. Mol. Enzymol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">1545</span><span class="NLM_x">, </span> <span class="NLM_fpage">67</span><span class="NLM_x">–</span> <span class="NLM_lpage">77</span><span class="refDoi"> DOI: 10.1016/S0167-4838(00)00262-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2FS0167-4838%2800%2900262-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=11342032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3MXpsFKhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1545&publication_year=2001&pages=67-77&author=A.+M.+Paivaauthor=D.+E.+Vanderwallauthor=J.+S.+Blanchardauthor=J.+W.+Kozarichauthor=J.+M.+Williamsonauthor=T.+M.+Kelly&title=Inhibitors+of+dihydrodipicolinate+reductase%2C+a+key+enzyme+of+the+diaminopimelate+pathway+of+Mycobacterium+tuberculosis&doi=10.1016%2FS0167-4838%2800%2900262-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of dihydrodipicolinate reductase, a key enzyme of the diaminopimelate pathway of Mycobacterium tuberculosis</span></div><div class="casAuthors">Paiva, A. M.; Vanderwall, D. E.; Blanchard, J. S.; Kozarich, J. W.; Williamson, J. M.; Kelly, T. M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Protein Structure and Molecular Enzymology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">1545</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">67-77</span>CODEN:
                <span class="NLM_cas:coden">BBAEDZ</span>;
        ISSN:<span class="NLM_cas:issn">0167-4838</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Tuberculosis (TB) remains a leading cause of infectious disease in the world today and therapies developed over the last forty years are becoming increasingly ineffective against resistant strains of Mycobacterium tuberculosis.  In an effort to explore new mechanisms for drug development, we have investigated the enzymes of the diaminopimelate biosynthetic pathway as potential targets.  Specifically, dihydrodipicolinate reductase, the essential gene product of dapB, was screened for novel inhibitors.  Inhibitors were identified both by a mol. modeling approach which utilized the available crystal structure of the enzyme with an inhibitor bound at the active site as well as by more conventional screening strategies.  The resulting compds. contain a no. of structural motifs and were all competitive with respect to the DHDP substrate.  The Ki values for the inhibitors range from 10 to 90 μM. The mol. modeling approach was very effective in identifying novel inhibitors of the enzyme.  These compds. were obtained at a higher frequency based on the no. of compds. analyzed than those inhibitors discovered via conventional screening.  However, conventional screening proved beneficial in identifying compds. with greater structural diversity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmPtGdeRnJ2LVg90H21EOLACvtfcHk0ljfJXaZEyb-Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXpsFKhtw%253D%253D&md5=26da0a9a71513d545e4c45ae4b48b853</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2FS0167-4838%2800%2900262-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0167-4838%252800%252900262-4%26sid%3Dliteratum%253Aachs%26aulast%3DPaiva%26aufirst%3DA.%2BM.%26aulast%3DVanderwall%26aufirst%3DD.%2BE.%26aulast%3DBlanchard%26aufirst%3DJ.%2BS.%26aulast%3DKozarich%26aufirst%3DJ.%2BW.%26aulast%3DWilliamson%26aufirst%3DJ.%2BM.%26aulast%3DKelly%26aufirst%3DT.%2BM.%26atitle%3DInhibitors%2520of%2520dihydrodipicolinate%2520reductase%252C%2520a%2520key%2520enzyme%2520of%2520the%2520diaminopimelate%2520pathway%2520of%2520Mycobacterium%2520tuberculosis%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Protein%2520Struct.%2520Mol.%2520Enzymol.%26date%3D2001%26volume%3D1545%26spage%3D67%26epage%3D77%26doi%3D10.1016%2FS0167-4838%2800%2900262-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Ragno, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Massa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cerbara, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bottoni, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scatena, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jesacher, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loidl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span> </span><span class="NLM_article-title">3-(4-Aroyl-1-methyl-1H-pyrrol-2-yl)-N-hydroxy-2-propenamides as a new class of synthetic histone deacetylase inhibitors. 3. Discovery of novel lead compounds through structure-based drug design and docking studies</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1351</span><span class="NLM_x">–</span> <span class="NLM_lpage">1359</span><span class="refDoi"> DOI: 10.1021/jm031036f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm031036f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1351-1359&author=R.+Ragnoauthor=A.+Maiauthor=S.+Massaauthor=I.+Cerbaraauthor=S.+Valenteauthor=P.+Bottoniauthor=R.+Scatenaauthor=F.+Jesacherauthor=P.+Loidlauthor=G.+Brosch&title=3-%284-Aroyl-1-methyl-1H-pyrrol-2-yl%29-N-hydroxy-2-propenamides+as+a+new+class+of+synthetic+histone+deacetylase+inhibitors.+3.+Discovery+of+novel+lead+compounds+through+structure-based+drug+design+and+docking+studies&doi=10.1021%2Fjm031036f"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm031036f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm031036f%26sid%3Dliteratum%253Aachs%26aulast%3DRagno%26aufirst%3DR.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DMassa%26aufirst%3DS.%26aulast%3DCerbara%26aufirst%3DI.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DBottoni%26aufirst%3DP.%26aulast%3DScatena%26aufirst%3DR.%26aulast%3DJesacher%26aufirst%3DF.%26aulast%3DLoidl%26aufirst%3DP.%26aulast%3DBrosch%26aufirst%3DG.%26atitle%3D3-%25284-Aroyl-1-methyl-1H-pyrrol-2-yl%2529-N-hydroxy-2-propenamides%2520as%2520a%2520new%2520class%2520of%2520synthetic%2520histone%2520deacetylase%2520inhibitors.%25203.%2520Discovery%2520of%2520novel%2520lead%2520compounds%2520through%2520structure-based%2520drug%2520design%2520and%2520docking%2520studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1351%26epage%3D1359%26doi%3D10.1021%2Fjm031036f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Park, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hwang, K. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oh, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K.</span><span> </span><span class="NLM_article-title">Discovery of novel alpha-glucosidase inhibitors based on the virtual screening with the homology-modeled protein structure</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">284</span><span class="NLM_x">–</span> <span class="NLM_lpage">292</span><span class="refDoi"> DOI: 10.1016/j.bmc.2007.09.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmc.2007.09.036" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=284-292&author=H.+Parkauthor=K.+Y.+Hwangauthor=K.+H.+Ohauthor=Y.+H.+Kimauthor=J.+Y.+Leeauthor=K.+Kim&title=Discovery+of+novel+alpha-glucosidase+inhibitors+based+on+the+virtual+screening+with+the+homology-modeled+protein+structure&doi=10.1016%2Fj.bmc.2007.09.036"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2007.09.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2007.09.036%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DH.%26aulast%3DHwang%26aufirst%3DK.%2BY.%26aulast%3DOh%26aufirst%3DK.%2BH.%26aulast%3DKim%26aufirst%3DY.%2BH.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DKim%26aufirst%3DK.%26atitle%3DDiscovery%2520of%2520novel%2520alpha-glucosidase%2520inhibitors%2520based%2520on%2520the%2520virtual%2520screening%2520with%2520the%2520homology-modeled%2520protein%2520structure%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2008%26volume%3D16%26spage%3D284%26epage%3D292%26doi%3D10.1016%2Fj.bmc.2007.09.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Ostrov, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hernandez Prada, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corsino, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finton, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowe, T. C.</span><span> </span><span class="NLM_article-title">Discovery of novel DNA gyrase inhibitors by high-throughput virtual screening</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">3688</span><span class="NLM_x">–</span> <span class="NLM_lpage">3698</span><span class="refDoi"> DOI: 10.1128/AAC.00392-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1128%2FAAC.00392-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=17682095" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGhtbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2007&pages=3688-3698&author=D.+A.+Ostrovauthor=J.+A.+Hernandez+Pradaauthor=P.+E.+Corsinoauthor=K.+A.+Fintonauthor=N.+Leauthor=T.+C.+Rowe&title=Discovery+of+novel+DNA+gyrase+inhibitors+by+high-throughput+virtual+screening&doi=10.1128%2FAAC.00392-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel DNA gyrase inhibitors by high-throughput virtual screening</span></div><div class="casAuthors">Ostrov, David A.; Prada, Jose A. Hernandez; Corsino, Patrick E.; Finton, Kathryn A.; Le, Nhan; Rowe, Thomas C.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3688-3698</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The bacterial type II topoisomerases DNA gyrase and topoisomerase IV are validated targets for clin. useful quinolone antimicrobial drugs.  A significant limitation to widely utilized quinolone inhibitors is the emergence of drug-resistant bacteria due to an altered DNA gyrase.  To address this problem, we have used structure-based mol. docking to identify novel drug-like small mols. that target sites distinct from those targeted by quinolone inhibitors.  A chem. ligand database contg. approx. 140,000 small mols. (mol. wt., <500) was molecularly docked onto two sites of Escherichia coli DNA gyrase targeting (i) a previously unexplored structural pocket formed at the dimer interface of subunit A and (ii) a small region of the ATP binding pocket on subunit B overlapping the site targeted by coumarin and cyclothialidine drugs.  This approach identified several small-mol. compds. that inhibited the DNA supercoiling activity of purified E. coli DNA gyrase.  These compds. are structurally unrelated to previously identified gyrase inhibitors and represent potential scaffolds for the optimization of novel antibacterial agents that act on fluoroquinolone-resistant strains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7z3236rWKm7Vg90H21EOLACvtfcHk0lhtszfmHOFFaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGhtbnI&md5=4bb6ab8cffb1e923e01d64a49a8ed759</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1128%2FAAC.00392-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00392-07%26sid%3Dliteratum%253Aachs%26aulast%3DOstrov%26aufirst%3DD.%2BA.%26aulast%3DHernandez%2BPrada%26aufirst%3DJ.%2BA.%26aulast%3DCorsino%26aufirst%3DP.%2BE.%26aulast%3DFinton%26aufirst%3DK.%2BA.%26aulast%3DLe%26aufirst%3DN.%26aulast%3DRowe%26aufirst%3DT.%2BC.%26atitle%3DDiscovery%2520of%2520novel%2520DNA%2520gyrase%2520inhibitors%2520by%2520high-throughput%2520virtual%2520screening%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2007%26volume%3D51%26spage%3D3688%26epage%3D3698%26doi%3D10.1128%2FAAC.00392-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Raboisson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desjarlais, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reed, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lattanze, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaikin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manthey, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomczuk, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marugan, J. J.</span><span> </span><span class="NLM_article-title">Identification of novel short chain 4-substituted indoles as potent alphavbeta3 antagonist using structure-based drug design</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">334</span><span class="NLM_x">–</span> <span class="NLM_lpage">343</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2006.10.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.ejmech.2006.10.015" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2007&pages=334-343&author=P.+Raboissonauthor=R.+L.+Desjarlaisauthor=R.+Reedauthor=J.+Lattanzeauthor=M.+Chaikinauthor=C.+L.+Mantheyauthor=B.+E.+Tomczukauthor=J.+J.+Marugan&title=Identification+of+novel+short+chain+4-substituted+indoles+as+potent+alphavbeta3+antagonist+using+structure-based+drug+design&doi=10.1016%2Fj.ejmech.2006.10.015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2006.10.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2006.10.015%26sid%3Dliteratum%253Aachs%26aulast%3DRaboisson%26aufirst%3DP.%26aulast%3DDesjarlais%26aufirst%3DR.%2BL.%26aulast%3DReed%26aufirst%3DR.%26aulast%3DLattanze%26aufirst%3DJ.%26aulast%3DChaikin%26aufirst%3DM.%26aulast%3DManthey%26aufirst%3DC.%2BL.%26aulast%3DTomczuk%26aufirst%3DB.%2BE.%26aulast%3DMarugan%26aufirst%3DJ.%2BJ.%26atitle%3DIdentification%2520of%2520novel%2520short%2520chain%25204-substituted%2520indoles%2520as%2520potent%2520alphavbeta3%2520antagonist%2520using%2520structure-based%2520drug%2520design%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2007%26volume%3D42%26spage%3D334%26epage%3D343%26doi%3D10.1016%2Fj.ejmech.2006.10.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Wang, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z. M.</span><span> </span><span class="NLM_article-title">Identification of some novel AHAS inhibitors via molecular docking and virtual screening approach</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">374</span><span class="NLM_x">–</span> <span class="NLM_lpage">380</span><span class="refDoi"> DOI: 10.1016/j.bmc.2006.09.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmc.2006.09.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=17049866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Cmt77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2007&pages=374-380&author=J.+G.+Wangauthor=Y.+J.+Xiaoauthor=Y.+H.+Liauthor=Y.+Maauthor=Z.+M.+Li&title=Identification+of+some+novel+AHAS+inhibitors+via+molecular+docking+and+virtual+screening+approach&doi=10.1016%2Fj.bmc.2006.09.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of some novel AHAS inhibitors via molecular docking and virtual screening approach</span></div><div class="casAuthors">Wang, Jian-Guo; Xiao, Yong-Jun; Li, Yong-Hong; Ma, Yi; Li, Zheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">374-380</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Acetohydroxyacid synthase (AHAS; EC 2.2.1.6) catalyzes the first common step in branched-chain amino acid biosynthesis.  This enzyme is an important target for the design of environmental-benign herbicides.  Based on the crystal structure of AHAS/sulfonylurea complex, we have carried out computational screening of the ACD-3D database in order to look for novel non-sulfonylurea inhibitors of AHAS for the first time.  Three novel compds. were found to inhibit plant AHAS in vitro among 14 procured compds.  One compd. showed promising activity in vivo for rape root growth inhibition bioassay.  This research provided useful clues for further design and discovery of AHAS inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAnfTtWRvu77Vg90H21EOLACvtfcHk0lhtszfmHOFFaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Cmt77I&md5=3b61a8b7e87697d734c13d014e06bf3f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2006.09.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2006.09.050%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%2BG.%26aulast%3DXiao%26aufirst%3DY.%2BJ.%26aulast%3DLi%26aufirst%3DY.%2BH.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DZ.%2BM.%26atitle%3DIdentification%2520of%2520some%2520novel%2520AHAS%2520inhibitors%2520via%2520molecular%2520docking%2520and%2520virtual%2520screening%2520approach%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2007%26volume%3D15%26spage%3D374%26epage%3D380%26doi%3D10.1016%2Fj.bmc.2006.09.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Foloppe, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fisher, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howes, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potter, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surgenor, A. E.</span><span> </span><span class="NLM_article-title">Identification of chemically diverse Chk1 inhibitors by receptor-based virtual screening</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">4792</span><span class="NLM_x">–</span> <span class="NLM_lpage">4802</span><span class="refDoi"> DOI: 10.1016/j.bmc.2006.03.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmc.2006.03.021" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2006&pages=4792-4802&author=N.+Foloppeauthor=L.+M.+Fisherauthor=R.+Howesauthor=A.+Potterauthor=A.+G.+Robertsonauthor=A.+E.+Surgenor&title=Identification+of+chemically+diverse+Chk1+inhibitors+by+receptor-based+virtual+screening&doi=10.1016%2Fj.bmc.2006.03.021"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2006.03.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2006.03.021%26sid%3Dliteratum%253Aachs%26aulast%3DFoloppe%26aufirst%3DN.%26aulast%3DFisher%26aufirst%3DL.%2BM.%26aulast%3DHowes%26aufirst%3DR.%26aulast%3DPotter%26aufirst%3DA.%26aulast%3DRobertson%26aufirst%3DA.%2BG.%26aulast%3DSurgenor%26aufirst%3DA.%2BE.%26atitle%3DIdentification%2520of%2520chemically%2520diverse%2520Chk1%2520inhibitors%2520by%2520receptor-based%2520virtual%2520screening%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2006%26volume%3D14%26spage%3D4792%26epage%3D4802%26doi%3D10.1016%2Fj.bmc.2006.03.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Cherkasov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fallahi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammond, G. L.</span><span> </span><span class="NLM_article-title">Successful in silico discovery of novel nonsteroidal ligands for human sex hormone binding globulin</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">3203</span><span class="NLM_x">–</span> <span class="NLM_lpage">3213</span><span class="refDoi"> DOI: 10.1021/jm049087f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049087f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=3203-3213&author=A.+Cherkasovauthor=Z.+Shiauthor=M.+Fallahiauthor=G.+L.+Hammond&title=Successful+in+silico+discovery+of+novel+nonsteroidal+ligands+for+human+sex+hormone+binding+globulin&doi=10.1021%2Fjm049087f"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm049087f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049087f%26sid%3Dliteratum%253Aachs%26aulast%3DCherkasov%26aufirst%3DA.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DFallahi%26aufirst%3DM.%26aulast%3DHammond%26aufirst%3DG.%2BL.%26atitle%3DSuccessful%2520in%2520silico%2520discovery%2520of%2520novel%2520nonsteroidal%2520ligands%2520for%2520human%2520sex%2520hormone%2520binding%2520globulin%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D3203%26epage%3D3213%26doi%3D10.1021%2Fjm049087f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Huentelman, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zubcevic, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hernandez Prada, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dimitrov, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raizada, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostrov, D. A.</span><span> </span><span class="NLM_article-title">Structure-based discovery of a novel angiotensin-converting enzyme 2 inhibitor</span> <span class="citation_source-journal">Hypertension</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">903</span><span class="NLM_x">–</span> <span class="NLM_lpage">906</span><span class="refDoi"> DOI: 10.1161/01.HYP.0000146120.29648.36</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1161%2F01.HYP.0000146120.29648.36" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=15492138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpvVGmurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=903-906&author=M.+J.+Huentelmanauthor=J.+Zubcevicauthor=J.+A.+Hernandez+Pradaauthor=X.+Xiaoauthor=D.+S.+Dimitrovauthor=M.+K.+Raizadaauthor=D.+A.+Ostrov&title=Structure-based+discovery+of+a+novel+angiotensin-converting+enzyme+2+inhibitor&doi=10.1161%2F01.HYP.0000146120.29648.36"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of a novel angiotensin-converting enzyme 2 inhibitor</span></div><div class="casAuthors">Huentelman, Matthew J.; Zubcevic, Jasenka; Hernandez Prada, Jose A.; Xiao, Xiaodong; Dimitrov, Dimiter S.; Raizada, Mohan K.; Ostrov, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Hypertension</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">903-906</span>CODEN:
                <span class="NLM_cas:coden">HPRTDN</span>;
        ISSN:<span class="NLM_cas:issn">0194-911X</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Angiotensin-converting enzyme 2 (ACE2) is considered an important therapeutic target for controlling cardiovascular diseases and severe acute respiratory syndrome (SARS) outbreaks.  Recently solved high-resoln. crystal structures of the apo-bound and inhibitor-bound forms of ACE2 have provided the basis for a novel mol. docking approach in an attempt to identify ACE2 inhibitors and compds. that block SARS coronavirus spike protein-mediated cell fusion.  In this study, ≈140 000 small mols. were screened by in silico mol. docking.  In this structure-activity relation study, the mols. with the highest predicted binding scores were identified and assayed for ACE2 enzymic inhibitory activity and for their ability to inhibit SARS coronavirus spike protein-mediated cell fusion.  This approach identified N-(2-aminoethyl)-1 aziridine-ethanamine as a novel ACE2 inhibitor that also is effective in blocking the SARS coronavirus spike protein-mediated cell fusion.  Thus, the mol. docking approach resulting in the inhibitory capacity of N-(2-aminoethyl)-1 aziridine-ethanamine provides an attractive small mol. lead compd. on which the development of more effective therapeutic agents could be developed to modulate hypertension and for controlling SARS infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHyw2ny8yWH7Vg90H21EOLACvtfcHk0lhMupCv1IQD1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpvVGmurc%253D&md5=4bfdeab803ff822a69c07acf97ed4897</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1161%2F01.HYP.0000146120.29648.36&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252F01.HYP.0000146120.29648.36%26sid%3Dliteratum%253Aachs%26aulast%3DHuentelman%26aufirst%3DM.%2BJ.%26aulast%3DZubcevic%26aufirst%3DJ.%26aulast%3DHernandez%2BPrada%26aufirst%3DJ.%2BA.%26aulast%3DXiao%26aufirst%3DX.%26aulast%3DDimitrov%26aufirst%3DD.%2BS.%26aulast%3DRaizada%26aufirst%3DM.%2BK.%26aulast%3DOstrov%26aufirst%3DD.%2BA.%26atitle%3DStructure-based%2520discovery%2520of%2520a%2520novel%2520angiotensin-converting%2520enzyme%25202%2520inhibitor%26jtitle%3DHypertension%26date%3D2004%26volume%3D44%26spage%3D903%26epage%3D906%26doi%3D10.1161%2F01.HYP.0000146120.29648.36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Mizutani, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itai, A.</span><span> </span><span class="NLM_article-title">Efficient method for high-throughput virtual screening based on flexible docking: discovery of novel acetylcholinesterase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">4818</span><span class="NLM_x">–</span> <span class="NLM_lpage">4828</span><span class="refDoi"> DOI: 10.1021/jm030605g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030605g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=4818-4828&author=M.+Y.+Mizutaniauthor=A.+Itai&title=Efficient+method+for+high-throughput+virtual+screening+based+on+flexible+docking%3A+discovery+of+novel+acetylcholinesterase+inhibitors&doi=10.1021%2Fjm030605g"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm030605g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030605g%26sid%3Dliteratum%253Aachs%26aulast%3DMizutani%26aufirst%3DM.%2BY.%26aulast%3DItai%26aufirst%3DA.%26atitle%3DEfficient%2520method%2520for%2520high-throughput%2520virtual%2520screening%2520based%2520on%2520flexible%2520docking%253A%2520discovery%2520of%2520novel%2520acetylcholinesterase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D4818%26epage%3D4828%26doi%3D10.1021%2Fjm030605g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Chandra, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhagavat, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sreekanthreddy, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somasundaram, K.</span><span> </span><span class="NLM_article-title">Virtual screening, identification and experimental testing of novel inhibitors of PBEF1/Visfatin/NMPRTase for glioma therapy</span> <span class="citation_source-journal">J. Clin. Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="refDoi"> DOI: 10.1186/2043-9113-1-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1186%2F2043-9113-1-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=21884623" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFShur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2011&pages=5&author=N.+Chandraauthor=R.+Bhagavatauthor=E.+Sharmaauthor=P.+Sreekanthreddyauthor=K.+Somasundaram&title=Virtual+screening%2C+identification+and+experimental+testing+of+novel+inhibitors+of+PBEF1%2FVisfatin%2FNMPRTase+for+glioma+therapy&doi=10.1186%2F2043-9113-1-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening, identification and experimental testing of novel inhibitors of PBEF1/visfatin/NMPRTase for glioma therapy</span></div><div class="casAuthors">Chandra, Nagasuma; Bhagavat, Raghu; Sharma, Eshita; Sreekanthreddy, P.; Somasundaram, Kumaravel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Bioinformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5</span>CODEN:
                <span class="NLM_cas:coden">JCBOBR</span>;
        ISSN:<span class="NLM_cas:issn">2043-9113</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Pre-B-cell colony enhancing factor 1 gene (PBEF1) encodes nicotinamide phosphoribosyltransferase (NMPRTase), which catalyzes the rate limiting step in the salvage pathway of NAD+ metab. in mammalian cells.  PBEF1 transcript and protein levels have been shown to be elevated in glioblastoma and a chem. inhibitor of NMPRTase has been shown to specifically inhibit cancer cells.  Methods: Virtual screening using docking was used to screen a library of more than 13,000 chem. compds.  A shortlisted set of compds. were tested for their inhibition activity in vitro by an NMPRTase enzyme assay.  Further, the ability of the compds. to inhibit glioma cell proliferation was carried out.  Results: Virtual screening resulted in short listing of 34 possible ligands, of which six were tested exptl., using the NMPRTase enzyme inhibition assay and further with the glioma cell viability assays.  Of these, two compds. were found to be significantly efficacious in inhibiting the conversion of nicotinamide to NAD+, and out of which, one compd., 3-amino-2-benzyl-7-nitro-4-(2-quinolyl-)-1,2-dihydroisoquinolin-1-one, was found to inhibit the growth of a PBEF1 over expressing glioma derived cell line U87 as well.  Conclusions: Thus, a novel inhibitor has been identified through a structure based drug discovery approach and is further supported by exptl. evidence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpJpygBbJb1LVg90H21EOLACvtfcHk0lhMupCv1IQD1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFShur4%253D&md5=b39873fb098f5ffcf1c31d9a44ed35f7</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1186%2F2043-9113-1-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F2043-9113-1-5%26sid%3Dliteratum%253Aachs%26aulast%3DChandra%26aufirst%3DN.%26aulast%3DBhagavat%26aufirst%3DR.%26aulast%3DSharma%26aufirst%3DE.%26aulast%3DSreekanthreddy%26aufirst%3DP.%26aulast%3DSomasundaram%26aufirst%3DK.%26atitle%3DVirtual%2520screening%252C%2520identification%2520and%2520experimental%2520testing%2520of%2520novel%2520inhibitors%2520of%2520PBEF1%252FVisfatin%252FNMPRTase%2520for%2520glioma%2520therapy%26jtitle%3DJ.%2520Clin.%2520Bioinf.%26date%3D2011%26volume%3D1%26spage%3D5%26doi%3D10.1186%2F2043-9113-1-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Xu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Unzue, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spiliotopoulos, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nevado, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Discovery of CREBBP bromodomain inhibitors by high-throughput docking and hit optimization guided by molecular dynamics</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">1340</span><span class="NLM_x">–</span> <span class="NLM_lpage">1349</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00171</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00171" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVyitLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1340-1349&author=M.+Xuauthor=A.+Unzueauthor=J.+Dongauthor=D.+Spiliotopoulosauthor=C.+Nevadoauthor=A.+Caflisch&title=Discovery+of+CREBBP+bromodomain+inhibitors+by+high-throughput+docking+and+hit+optimization+guided+by+molecular+dynamics&doi=10.1021%2Facs.jmedchem.5b00171"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of CREBBP Bromodomain Inhibitors by High-Throughput Docking and Hit Optimization Guided by Molecular Dynamics</span></div><div class="casAuthors">Xu, Min; Unzue, Andrea; Dong, Jing; Spiliotopoulos, Dimitrios; Nevado, Cristina; Caflisch, Amedeo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1340-1349</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors have identified two chemotypes of CREBBP bromodomain ligands by fragment-based high-throughput docking.  Only 17 mols. from the original library of two-million compds. were tested in vitro.  Optimization of the two low-micromolar hits, the 4-acylpyrrole (III) and acylbenzene (IV), was driven by mol. dynamics results which suggested improvement of the polar interactions with the Arg1173 side chain at the rim of the binding site.  The synthesis of only two derivs. of III yielded the 4-acylpyrrole I which shows a single-digit micromolar affinity for the CREBBP bromodomain and a ligand efficiency of 0.34 kcal/mol per nonhydrogen atom.  Optimization of the acylbenzene hit IV resulted in a series of derivs. with nanomolar potencies, good ligand efficiency and selectivity.  The in silico predicted binding mode of the acylbenzene deriv. II was validated by solving the structure of the complex with the CREBBP bromodomain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq100fCa86MT7Vg90H21EOLACvtfcHk0lhOKCr-od_eyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVyitLrP&md5=e401c14fc119d2d1c39b74b5b41f4d3a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00171&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00171%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DM.%26aulast%3DUnzue%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DJ.%26aulast%3DSpiliotopoulos%26aufirst%3DD.%26aulast%3DNevado%26aufirst%3DC.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520CREBBP%2520bromodomain%2520inhibitors%2520by%2520high-throughput%2520docking%2520and%2520hit%2520optimization%2520guided%2520by%2520molecular%2520dynamics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1340%26epage%3D1349%26doi%3D10.1021%2Facs.jmedchem.5b00171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, W.</span><span> </span><span class="NLM_article-title">Discovery of new potent inhibitors for carbonic anhydrase IX by structure-based virtual screening</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">3496</span><span class="NLM_x">–</span> <span class="NLM_lpage">3499</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2013.04.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2013.04.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=23664215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsVynsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=3496-3499&author=L.+Wangauthor=C.+Yangauthor=W.+Luauthor=L.+Liuauthor=R.+Gaoauthor=S.+Liaoauthor=Z.+Zhaoauthor=L.+Zhuauthor=Y.+Xuauthor=H.+Liauthor=J.+Huangauthor=W.+Zhu&title=Discovery+of+new+potent+inhibitors+for+carbonic+anhydrase+IX+by+structure-based+virtual+screening&doi=10.1016%2Fj.bmcl.2013.04.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of new potent inhibitors for carbonic anhydrase IX by structure-based virtual screening</span></div><div class="casAuthors">Wang, Liyan; Yang, Chunmei; Lu, Weiqiang; Liu, Li; Gao, Rui; Liao, Sha; Zhao, Zhenjiang; Zhu, Lili; Xu, Yufang; Li, Honglin; Huang, Jin; Zhu, Weiping</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3496-3499</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Through structure-based virtual screening, some dozen of benzene sulfonamides with novel scaffolds are identified as potent inhibitors against carbonic anhydrase (CA) IX with IC50 values ranging from 2.86 to 588.34 nM.  Among them, compds. 1 and 9 show high selectivity against tumor-target CA IX over CA II (the selectivity ratios are 21.3 and 136.6, resp.).  The possible binding poses of hit compds. are also explored and the selectivity is elucidated by mol. docking simulations.  The hit compds. discovered in this work would provide novel scaffolds for further hit-to-lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRat2J1LShn7Vg90H21EOLACvtfcHk0lhOKCr-od_eyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsVynsbc%253D&md5=193b016905803f30f055f354dcdd59c3</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.04.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.04.048%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DR.%26aulast%3DLiao%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DW.%26atitle%3DDiscovery%2520of%2520new%2520potent%2520inhibitors%2520for%2520carbonic%2520anhydrase%2520IX%2520by%2520structure-based%2520virtual%2520screening%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D3496%26epage%3D3499%26doi%3D10.1016%2Fj.bmcl.2013.04.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Heusser, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borghese, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cullins, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broemstrup, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, U. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindahl, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, R. A.</span><span> </span><span class="NLM_article-title">Functional validation of virtual screening for novel agents with general anesthetic action at ligand-gated ion channels</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">670</span><span class="NLM_x">–</span> <span class="NLM_lpage">678</span><span class="refDoi"> DOI: 10.1124/mol.113.087692</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1124%2Fmol.113.087692" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2013&pages=670-678&author=S.+A.+Heusserauthor=R.+J.+Howardauthor=C.+M.+Borgheseauthor=M.+A.+Cullinsauthor=T.+Broemstrupauthor=U.+S.+Leeauthor=E.+Lindahlauthor=J.+Carlssonauthor=R.+A.+Harris&title=Functional+validation+of+virtual+screening+for+novel+agents+with+general+anesthetic+action+at+ligand-gated+ion+channels&doi=10.1124%2Fmol.113.087692"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1124%2Fmol.113.087692&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.113.087692%26sid%3Dliteratum%253Aachs%26aulast%3DHeusser%26aufirst%3DS.%2BA.%26aulast%3DHoward%26aufirst%3DR.%2BJ.%26aulast%3DBorghese%26aufirst%3DC.%2BM.%26aulast%3DCullins%26aufirst%3DM.%2BA.%26aulast%3DBroemstrup%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DU.%2BS.%26aulast%3DLindahl%26aufirst%3DE.%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DHarris%26aufirst%3DR.%2BA.%26atitle%3DFunctional%2520validation%2520of%2520virtual%2520screening%2520for%2520novel%2520agents%2520with%2520general%2520anesthetic%2520action%2520at%2520ligand-gated%2520ion%2520channels%26jtitle%3DMol.%2520Pharmacol.%26date%3D2013%26volume%3D84%26spage%3D670%26epage%3D678%26doi%3D10.1124%2Fmol.113.087692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Zhao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gartenmann, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spiliotopoulos, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Discovery of BRD4 bromodomain inhibitors by fragment-based high-throughput docking</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">2493</span><span class="NLM_x">–</span> <span class="NLM_lpage">2496</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2014.04.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2014.04.017" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=2493-2496&author=H.+Zhaoauthor=L.+Gartenmannauthor=J.+Dongauthor=D.+Spiliotopoulosauthor=A.+Caflisch&title=Discovery+of+BRD4+bromodomain+inhibitors+by+fragment-based+high-throughput+docking&doi=10.1016%2Fj.bmcl.2014.04.017"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.04.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.04.017%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DGartenmann%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DJ.%26aulast%3DSpiliotopoulos%26aufirst%3DD.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520BRD4%2520bromodomain%2520inhibitors%2520by%2520fragment-based%2520high-throughput%2520docking%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D2493%26epage%3D2496%26doi%3D10.1016%2Fj.bmcl.2014.04.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Kouskoumvekaki, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petersen, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fratev, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taboureau, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oprea, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sonne, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flindt, E. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jonsdottir, S. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristiansen, K.</span><span> </span><span class="NLM_article-title">Discovery of a novel selective PPARgamma ligand with partial agonist binding properties by integrated in silico/in vitro work flow</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">923</span><span class="NLM_x">–</span> <span class="NLM_lpage">937</span><span class="refDoi"> DOI: 10.1021/ci3006148</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3006148" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivFCns7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=923-937&author=I.+Kouskoumvekakiauthor=R.+K.+Petersenauthor=F.+Fratevauthor=O.+Taboureauauthor=T.+E.+Nielsenauthor=T.+I.+Opreaauthor=S.+B.+Sonneauthor=E.+N.+Flindtauthor=S.+O.+Jonsdottirauthor=K.+Kristiansen&title=Discovery+of+a+novel+selective+PPARgamma+ligand+with+partial+agonist+binding+properties+by+integrated+in+silico%2Fin+vitro+work+flow&doi=10.1021%2Fci3006148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Novel Selective PPARγ Ligand with Partial Agonist Binding Properties by Integrated in Silico/in Vitro Work Flow</span></div><div class="casAuthors">Kouskoumvekaki, Irene; Petersen, Rasmus K.; Fratev, Filip; Taboureau, Olivier; Nielsen, Thomas E.; Oprea, Tudor I.; Sonne, Si B.; Flindt, Esben N.; Jonsdottir, Svava Osk; Kristiansen, Karsten</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">923-937</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Full agonists to the peroxisome proliferator-activated receptor (PPAR)γ, such as Rosiglitazone, have been assocd. with a series of undesired side effects, such as wt. gain, fluid retention, cardiac hypertrophy, and hepatotoxicity.  Nevertheless, PPARγ is involved in the expression of genes that control glucose and lipid metab. and is an important target for drugs against type 2 diabetes, dyslipidemia, atherosclerosis, and cardiovascular disease.  In an effort to identify novel PPARγ ligands with an improved pharmacol. profile, emphasis has shifted to selective ligands with partial agonist binding properties.  Toward this end we applied an integrated in silico/in vitro workflow, based on pharmacophore- and structure-based virtual screening of the ZINC library, coupled with competitive binding and transactivation assays, and adipocyte differentiation and gene expression studies.  Hit compd. 9 was identified as the most potent ligand (IC50 = 0.3 μM) and a relatively poor inducer of adipocyte differentiation.  The binding mode of compd. 9 was confirmed by mol. dynamics simulation, and the calcd. free energy of binding was -8.4 kcal/mol.  A novel functional group, the carbonitrile group, was identified to be a key substituent in the ligand-protein interactions.  Further studies on the transcriptional regulation properties of compd. 9 revealed a gene regulatory profile that was to a large extent unique, however functionally closer to that of a partial agonist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2ghUTeDUWrVg90H21EOLACvtfcHk0lhOKCr-od_eyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivFCns7w%253D&md5=c0f5a929bf9712e43d958a974fef4b91</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fci3006148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3006148%26sid%3Dliteratum%253Aachs%26aulast%3DKouskoumvekaki%26aufirst%3DI.%26aulast%3DPetersen%26aufirst%3DR.%2BK.%26aulast%3DFratev%26aufirst%3DF.%26aulast%3DTaboureau%26aufirst%3DO.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DSonne%26aufirst%3DS.%2BB.%26aulast%3DFlindt%26aufirst%3DE.%2BN.%26aulast%3DJonsdottir%26aufirst%3DS.%2BO.%26aulast%3DKristiansen%26aufirst%3DK.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520selective%2520PPARgamma%2520ligand%2520with%2520partial%2520agonist%2520binding%2520properties%2520by%2520integrated%2520in%2520silico%252Fin%2520vitro%2520work%2520flow%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D923%26epage%3D937%26doi%3D10.1021%2Fci3006148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Xu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, I. A.</span><span> </span><span class="NLM_article-title">Crystal structure of avian aminoimidazole-4-carboxamide ribonucleotide transformylase in complex with a novel non-folate inhibitor identified by virtual ligand screening</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">279</span><span class="NLM_x">, </span> <span class="NLM_fpage">50555</span><span class="NLM_x">–</span> <span class="NLM_lpage">50565</span><span class="refDoi"> DOI: 10.1074/jbc.M406801200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1074%2Fjbc.M406801200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=50555-50565&author=L.+Xuauthor=C.+L.+Liauthor=A.+J.+Olsonauthor=I.+A.+Wilson&title=Crystal+structure+of+avian+aminoimidazole-4-carboxamide+ribonucleotide+transformylase+in+complex+with+a+novel+non-folate+inhibitor+identified+by+virtual+ligand+screening&doi=10.1074%2Fjbc.M406801200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M406801200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M406801200%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DC.%2BL.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26aulast%3DWilson%26aufirst%3DI.%2BA.%26atitle%3DCrystal%2520structure%2520of%2520avian%2520aminoimidazole-4-carboxamide%2520ribonucleotide%2520transformylase%2520in%2520complex%2520with%2520a%2520novel%2520non-folate%2520inhibitor%2520identified%2520by%2520virtual%2520ligand%2520screening%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D50555%26epage%3D50565%26doi%3D10.1074%2Fjbc.M406801200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Pierce, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stuver-Moody, C.</span><span> </span><span class="NLM_article-title">Docking study yields four novel inhibitors of the protooncogene Pim-1 kinase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1972</span><span class="NLM_x">–</span> <span class="NLM_lpage">1975</span><span class="refDoi"> DOI: 10.1021/jm701248t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701248t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitlOmtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=1972-1975&author=A.+C.+Pierceauthor=M.+Jacobsauthor=C.+Stuver-Moody&title=Docking+study+yields+four+novel+inhibitors+of+the+protooncogene+Pim-1+kinase&doi=10.1021%2Fjm701248t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Study Yields Four Novel Inhibitors of the Protooncogene Pim-1 Kinase</span></div><div class="casAuthors">Pierce, Albert C.; Jacobs, Marc; Stuver-Moody, Cameron</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1972-1975</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To supplement the hits from a high throughput screen, docking was performed against Pim-1 kinase.  Glide docking was augmented with a filter to require traditional or arom. CH··O hydrogen bonds to the kinase hinge.  Four diverse actives, of 96 mols. assayed, had Ki values between 0.091 and 4.5 μM.  This gives a 14-fold enrichment over the earlier HTS run, and the two crystal structures solved confirmed the binding modes predicted by docking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoH6GwwgMwC27Vg90H21EOLACvtfcHk0lhjCDBvBkDEsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitlOmtLY%253D&md5=fafba52528730d4e250d489ad24a6cd8</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm701248t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701248t%26sid%3Dliteratum%253Aachs%26aulast%3DPierce%26aufirst%3DA.%2BC.%26aulast%3DJacobs%26aufirst%3DM.%26aulast%3DStuver-Moody%26aufirst%3DC.%26atitle%3DDocking%2520study%2520yields%2520four%2520novel%2520inhibitors%2520of%2520the%2520protooncogene%2520Pim-1%2520kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D1972%26epage%3D1975%26doi%3D10.1021%2Fjm701248t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Powers, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morandi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Structure-based discovery of a novel, noncovalent inhibitor of AmpC beta-lactamase</span> <span class="citation_source-journal">Structure (Oxford, U. K.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1013</span><span class="NLM_x">–</span> <span class="NLM_lpage">1023</span><span class="refDoi"> DOI: 10.1016/S0969-2126(02)00799-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2FS0969-2126%2802%2900799-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2002&pages=1013-1023&author=R.+A.+Powersauthor=F.+Morandiauthor=B.+K.+Shoichet&title=Structure-based+discovery+of+a+novel%2C+noncovalent+inhibitor+of+AmpC+beta-lactamase&doi=10.1016%2FS0969-2126%2802%2900799-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2802%2900799-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252802%252900799-2%26sid%3Dliteratum%253Aachs%26aulast%3DPowers%26aufirst%3DR.%2BA.%26aulast%3DMorandi%26aufirst%3DF.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520discovery%2520of%2520a%2520novel%252C%2520noncovalent%2520inhibitor%2520of%2520AmpC%2520beta-lactamase%26jtitle%3DStructure%2520%2528Oxford%252C%2520U.%2520K.%2529%26date%3D2002%26volume%3D10%26spage%3D1013%26epage%3D1023%26doi%3D10.1016%2FS0969-2126%2802%2900799-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cancian, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maltby, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Comprehensive mechanistic analysis of hits from high-throughput and docking screens against beta-lactamase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2502</span><span class="NLM_x">–</span> <span class="NLM_lpage">2511</span><span class="refDoi"> DOI: 10.1021/jm701500e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701500e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtFSksb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2502-2511&author=K.+Babaogluauthor=A.+Simeonovauthor=J.+J.+Irwinauthor=M.+E.+Nelsonauthor=B.+Fengauthor=C.+J.+Thomasauthor=L.+Cancianauthor=M.+P.+Costiauthor=D.+A.+Maltbyauthor=A.+Jadhavauthor=J.+Ingleseauthor=C.+P.+Austinauthor=B.+K.+Shoichet&title=Comprehensive+mechanistic+analysis+of+hits+from+high-throughput+and+docking+screens+against+beta-lactamase&doi=10.1021%2Fjm701500e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive Mechanistic Analysis of Hits from High-Throughput and Docking Screens against β-Lactamase</span></div><div class="casAuthors">Babaoglu, Kerim; Simeonov, Anton; Irwin, John J.; Nelson, Michael E.; Feng, Brian; Thomas, Craig J.; Cancian, Laura; Costi, M. Paola; Maltby, David A.; Jadhav, Ajit; Inglese, James; Austin, Christopher P.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2502-2511</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) is widely used in drug discovery.  Esp. for screens of unbiased libraries, false positives can dominate "hit lists"; their origins are much debated.  Here we det. the mechanism of every active hit from a screen of 70,563 unbiased mols. against β-lactamase using quant. HTS (qHTS).  Of the 1274 initial inhibitors, 95% were detergent-sensitive and were classified as aggregators.  Among the 70 remaining were 25 potent, covalent-acting β-lactams.  Mass spectra, counter-screens, and crystallog. identified 12 as promiscuous covalent inhibitors.  The remaining 33 were either aggregators or irreproducible.  No specific reversible inhibitors were found.  We turned to mol. docking to prioritize mols. from the same library for testing at higher concns.  Of 16 tested, 2 were modest inhibitors.  Subsequent X-ray structures corresponded to the docking prediction.  Analog synthesis improved affinity to 8 μM.  These results suggest that it may be the phys. behavior of org. mols., not their reactivity, that accounts for most screening artifacts.  Structure-based methods may prioritize weak-but-novel chemotypes in unbiased library screens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNtnaCegS8aLVg90H21EOLACvtfcHk0lhjCDBvBkDEsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtFSksb0%253D&md5=7736429aeafc6b17a5d0182e24df0dd4</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm701500e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701500e%26sid%3Dliteratum%253Aachs%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DNelson%26aufirst%3DM.%2BE.%26aulast%3DFeng%26aufirst%3DB.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DCancian%26aufirst%3DL.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26aulast%3DMaltby%26aufirst%3DD.%2BA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DComprehensive%2520mechanistic%2520analysis%2520of%2520hits%2520from%2520high-throughput%2520and%2520docking%2520screens%2520against%2520beta-lactamase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2502%26epage%3D2511%26doi%3D10.1021%2Fjm701500e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Wacker, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fenalti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katritch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span> </span><span class="NLM_article-title">Conserved binding mode of human beta2 adrenergic receptor inverse agonists and antagonist revealed by X-ray crystallography</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">11443</span><span class="NLM_x">–</span> <span class="NLM_lpage">11445</span><span class="refDoi"> DOI: 10.1021/ja105108q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja105108q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=11443-11445&author=D.+Wackerauthor=G.+Fenaltiauthor=M.+A.+Brownauthor=V.+Katritchauthor=R.+Abagyanauthor=V.+Cherezovauthor=R.+C.+Stevens&title=Conserved+binding+mode+of+human+beta2+adrenergic+receptor+inverse+agonists+and+antagonist+revealed+by+X-ray+crystallography&doi=10.1021%2Fja105108q"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fja105108q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja105108q%26sid%3Dliteratum%253Aachs%26aulast%3DWacker%26aufirst%3DD.%26aulast%3DFenalti%26aufirst%3DG.%26aulast%3DBrown%26aufirst%3DM.%2BA.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DConserved%2520binding%2520mode%2520of%2520human%2520beta2%2520adrenergic%2520receptor%2520inverse%2520agonists%2520and%2520antagonist%2520revealed%2520by%2520X-ray%2520crystallography%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D11443%26epage%3D11445%26doi%3D10.1021%2Fja105108q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Ferreira, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eidam, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mott, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKerrow, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maloney, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Complementarity between a docking and a high-throughput screen in discovering new cruzain inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">4891</span><span class="NLM_x">–</span> <span class="NLM_lpage">4905</span><span class="refDoi"> DOI: 10.1021/jm100488w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100488w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntlWrurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=4891-4905&author=R.+S.+Ferreiraauthor=A.+Simeonovauthor=A.+Jadhavauthor=O.+Eidamauthor=B.+T.+Mottauthor=M.+J.+Keiserauthor=J.+H.+McKerrowauthor=D.+J.+Maloneyauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Complementarity+between+a+docking+and+a+high-throughput+screen+in+discovering+new+cruzain+inhibitors&doi=10.1021%2Fjm100488w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Complementarity Between a Docking and a High-Throughput Screen in Discovering New Cruzain Inhibitors</span></div><div class="casAuthors">Ferreira, Rafaela S.; Simeonov, Anton; Jadhav, Ajit; Eidam, Oliv; Mott, Bryan T.; Keiser, Michael J.; McKerrow, James H.; Maloney, David J.; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4891-4905</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual and high-throughput screens (HTS) should have complementary strengths and weaknesses, but studies that prospectively and comprehensively compare them are rare.  We undertook a parallel docking and HTS screen of 197861 compds. against cruzain, a thiol protease target for Chagas disease, looking for reversible, competitive inhibitors.  On workup, 99% of the hits were eliminated as false positives, yielding 146 well-behaved, competitive ligands.  These fell into five chemotypes: two were prioritized by scoring among the top 0.1% of the docking-ranked library, two were prioritized by behavior in the HTS and by clustering, and one chemotype was prioritized by both approaches.  Detn. of an inhibitor/cruzain crystal structure and comparison of the high-scoring docking hits to expt. illuminated the origins of docking false-negatives and false-positives.  Prioritizing mols. that are both predicted by docking and are HTS-active yields well-behaved mols., relatively unobscured by the false-positives to which both techniques are individually prone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmmoDO-1mm8bVg90H21EOLACvtfcHk0lg8IjMiPd75XQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntlWrurk%253D&md5=0661663ff0802bf981a1997eeca9fa05</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm100488w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100488w%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%26aufirst%3DR.%2BS.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DEidam%26aufirst%3DO.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DMcKerrow%26aufirst%3DJ.%2BH.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DComplementarity%2520between%2520a%2520docking%2520and%2520a%2520high-throughput%2520screen%2520in%2520discovering%2520new%2520cruzain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D4891%26epage%3D4905%26doi%3D10.1021%2Fjm100488w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Molecular docking and ligand specificity in fragment-based inhibitor discovery</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">358</span><span class="NLM_x">–</span> <span class="NLM_lpage">364</span><span class="refDoi"> DOI: 10.1038/nchembio.155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnchembio.155" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=19305397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsVCqt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=358-364&author=Y.+Chenauthor=B.+K.+Shoichet&title=Molecular+docking+and+ligand+specificity+in+fragment-based+inhibitor+discovery&doi=10.1038%2Fnchembio.155"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular docking and ligand specificity in fragment-based inhibitor discovery</span></div><div class="casAuthors">Chen, Yu; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">358-364</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fragment screens have successfully identified new scaffolds in drug discovery, often with relatively high hit rates (5%) using small screening libraries (1,000-10,000 compds.).  This raises two questions: would other noteworthy chemotypes be found were one to screen all com. available fragments (>300,000), and does the success rate imply low specificity of fragments.  We used mol. docking to screen large libraries of fragments against CTX-M β-lactamase.  We identified ten millimolar-range inhibitors from the 69 compds. tested.  The docking poses corresponded closely to the crystallog. structures subsequently detd.  Notably, these initial low-affinity hits showed little specificity between CTX-M and an unrelated β-lactamase, AmpC, which is unusual among β-lactamase inhibitors.  This is consistent with the idea that the high hit rates among fragments correlate to a low initial specificity.  As the inhibitors were progressed, both specificity and affinity rose together, yielding to our knowledge the first micromolar-range noncovalent inhibitors against a class A β-lactamase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO8zTkq9kMAbVg90H21EOLACvtfcHk0lg8IjMiPd75XQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsVCqt7k%253D&md5=6ee8dd19d0b99599ea74eaf735483b1d</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.155%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DMolecular%2520docking%2520and%2520ligand%2520specificity%2520in%2520fragment-based%2520inhibitor%2520discovery%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D358%26epage%3D364%26doi%3D10.1038%2Fnchembio.155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Teotico, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocklin, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferreira, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giannetti, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Docking for fragment inhibitors of AmpC beta-lactamase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">7455</span><span class="NLM_x">–</span> <span class="NLM_lpage">7460</span><span class="refDoi"> DOI: 10.1073/pnas.0813029106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1073%2Fpnas.0813029106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=19416920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmt1Khtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=7455-7460&author=D.+G.+Teoticoauthor=K.+Babaogluauthor=G.+J.+Rocklinauthor=R.+S.+Ferreiraauthor=A.+M.+Giannettiauthor=B.+K.+Shoichet&title=Docking+for+fragment+inhibitors+of+AmpC+beta-lactamase&doi=10.1073%2Fpnas.0813029106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Docking for fragment inhibitors of AmpC 83-lactamase</span></div><div class="casAuthors">Teotico, Denise G.; Babaoglu, Kerim; Rocklin, Gabriel J.; Ferreira, Rafaela S.; Giannetti, Anthony M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7455-7460</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Fragment screens for new ligands have had wide success, not withstanding their constraint to libraries of 1000-10,000 mols.  Larger libraries would be addressable were mol. docking reliable for fragment screens, but this has not been widely accepted.  To investigate docking's ability to prioritize fragments, a library of >137,000 such mols. were docked against the structure of β-lactamase.  Forty-eight fragments highly ranked by docking were acquired and tested; 23 had Ki values ranging from 0.7 to 9.2 mM.  X-ray crystal structures of the enzyme-bound complexes were detd. for 8 of the fragments.  For 4, the correspondence between the predicted and exptl. structures was high (RMSD between 1.2 and 1.4 Å), whereas for another 2, the fidelity was lower but retained most key interactions (RMSD 2.4-2.6 Å).  Two of the 8 fragments adopted very different poses in the active site owing to enzyme conformational changes.  The 48% hit rate of the fragment docking compares very favorably with "lead-like" docking and high-throughput screening against the same enzyme.  To understand this, we investigated the occurrence of the fragment scaffolds among larger, lead-like mols.  Approx. 1% of com. available fragments contain these inhibitors whereas only 10-7% of lead-like mols. do.  This suggests that many more chemotypes and combinations of chemotypes are present among fragments than are available among lead-like mols., contributing to the higher hit rates.  The ability of docking to prioritize these fragments suggests that the technique can be used to exploit the better chemotype coverage that exists at the fragment level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprpafzXv4ydrVg90H21EOLACvtfcHk0lg8IjMiPd75XQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmt1Khtrg%253D&md5=d2df01432d5381c226e8aa7e008457b5</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0813029106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0813029106%26sid%3Dliteratum%253Aachs%26aulast%3DTeotico%26aufirst%3DD.%2BG.%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DRocklin%26aufirst%3DG.%2BJ.%26aulast%3DFerreira%26aufirst%3DR.%2BS.%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDocking%2520for%2520fragment%2520inhibitors%2520of%2520AmpC%2520beta-lactamase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D7455%26epage%3D7460%26doi%3D10.1073%2Fpnas.0813029106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Merski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">The impact of introducing a histidine into an apolar cavity site on docking and ligand recognition</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2874</span><span class="NLM_x">–</span> <span class="NLM_lpage">2884</span><span class="refDoi"> DOI: 10.1021/jm301823g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301823g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2874-2884&author=M.+Merskiauthor=B.+K.+Shoichet&title=The+impact+of+introducing+a+histidine+into+an+apolar+cavity+site+on+docking+and+ligand+recognition&doi=10.1021%2Fjm301823g"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm301823g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301823g%26sid%3Dliteratum%253Aachs%26aulast%3DMerski%26aufirst%3DM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DThe%2520impact%2520of%2520introducing%2520a%2520histidine%2520into%2520an%2520apolar%2520cavity%2520site%2520on%2520docking%2520and%2520ligand%2520recognition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2874%26epage%3D2884%26doi%3D10.1021%2Fjm301823g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Barelier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyce, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodin, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Roles for ordered and bulk solvent in ligand recognition and docking in two related cavities</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e69153</span><span class="refDoi"> DOI: 10.1371/journal.pone.0069153</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pone.0069153" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=23874896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1WmtLbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e69153&author=S.+Barelierauthor=S.+E.+Boyceauthor=I.+Fishauthor=M.+Fischerauthor=D.+B.+Goodinauthor=B.+K.+Shoichet&title=Roles+for+ordered+and+bulk+solvent+in+ligand+recognition+and+docking+in+two+related+cavities&doi=10.1371%2Fjournal.pone.0069153"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Roles for ordered and bulk solvent in ligand recognition and docking in two related cavities</span></div><div class="casAuthors">Barelier, Sarah; Boyce, Sarah E.; Fish, Inbar; Fischer, Marcus; Goodin, David B.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e69153</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">A key challenge in structure-based discovery is accounting for modulation of protein-ligand interactions by ordered and bulk solvent.  To investigate this, we compared ligand binding to a buried cavity in Cytochrome c Peroxidase (CcP), where affinity is dominated by a single ionic interaction, vs. a cavity variant partly opened to solvent by loop deletion.  This opening had unexpected effects on ligand orientation, affinity and ordered water structure.  Some ligands lost over ten-fold in affinity and reoriented in the cavity, while others retained their geometries, formed new interactions with water networks and improved affinity.  To test our ability to discover new ligands against this opened site prospectively, a 534,000 fragment library was docked against the open cavity using two models of ligand solvation.  Using an older solvation model that prioritized many neutral mols., three such uncharged docking hits were tested, none of which was obsd. to bind; these mols. were not highly ranked by the new, context-dependent solvation score.  Using this new method, another 15 highly-ranked mols. were tested for binding.  In contrast to the previous result, 14 of these bound detectably, with affinities ranging from 8 μM to 2 mM.  In crystal structures, four of these new ligands superposed well with the docking predictions but two did not, reflecting unanticipated interactions with newly ordered waters mols.  Comparing recognition between this open cavity and its buried analog begins to isolate the roles of ordered solvent in a system that lends itself readily to prospective testing and that may be broadly useful to the community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojbQG1fpsJIrVg90H21EOLACvtfcHk0lj20yJjqaQm-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1WmtLbK&md5=5ace2d9d19690ddbe591ee6abad9c9bf</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0069153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0069153%26sid%3Dliteratum%253Aachs%26aulast%3DBarelier%26aufirst%3DS.%26aulast%3DBoyce%26aufirst%3DS.%2BE.%26aulast%3DFish%26aufirst%3DI.%26aulast%3DFischer%26aufirst%3DM.%26aulast%3DGoodin%26aufirst%3DD.%2BB.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRoles%2520for%2520ordered%2520and%2520bulk%2520solvent%2520in%2520ligand%2520recognition%2520and%2520docking%2520in%2520two%2520related%2520cavities%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De69153%26doi%3D10.1371%2Fjournal.pone.0069153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lubet, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bode, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, Z.</span><span> </span><span class="NLM_article-title">Discovery of novel checkpoint kinase 1 inhibitors by virtual screening based on multiple crystal structures</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2904</span><span class="NLM_x">–</span> <span class="NLM_lpage">2914</span><span class="refDoi"> DOI: 10.1021/ci200257b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200257b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht12ktrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2904-2914&author=Y.+Liauthor=D.+J.+Kimauthor=W.+Maauthor=R.+A.+Lubetauthor=A.+M.+Bodeauthor=Z.+Dong&title=Discovery+of+novel+checkpoint+kinase+1+inhibitors+by+virtual+screening+based+on+multiple+crystal+structures&doi=10.1021%2Fci200257b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Checkpoint Kinase 1 Inhibitors by Virtual Screening Based on Multiple Crystal Structures</span></div><div class="casAuthors">Li, Yan; Kim, Dong-Joon; Ma, Wei-Ya; Lubet, Ronald A.; Bode, Ann M.; Dong, Zi-Gang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2904-2914</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Incorporating receptor flexibility is considered crucial for improvement of docking-based virtual screening.  With an abundance of crystallog. structures freely available, docking with multiple crystal structures is believed to be a practical approach to cope with protein flexibility.  Here we describe a successful application of the docking of multiple structures to discover novel and potent Chk1 inhibitors.  Forty-six Chk1 structures were first compared in single structure docking by predicting the binding mode and recovering known ligands.  Combinations of different protein structures were then compared by recovery of known ligands and an optimal ensemble of Chk1 structures were selected.  The chosen structures were used in the virtual screening of over 60000 diverse compds. for Chk1 inhibitors.  Six novel compds. ranked at the top of the hits list were tested exptl., and two of these compds. inhibited Chk1 activity-the best with an IC50 value of 9.6 μM.  Further study indicated that achieving a better enrichment and identifying more diverse compds. was more likely using multiple structures than using only a single structure even when protein structures were randomly selected.  Taking into account conformational energy difference did not help to improve enrichment in the top ranked list.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKXlCGjr_Om7Vg90H21EOLACvtfcHk0lj20yJjqaQm-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht12ktrrL&md5=14ea06cd98ce6159ddc68bd8ddf7db05</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fci200257b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200257b%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DD.%2BJ.%26aulast%3DMa%26aufirst%3DW.%26aulast%3DLubet%26aufirst%3DR.%2BA.%26aulast%3DBode%26aufirst%3DA.%2BM.%26aulast%3DDong%26aufirst%3DZ.%26atitle%3DDiscovery%2520of%2520novel%2520checkpoint%2520kinase%25201%2520inhibitors%2520by%2520virtual%2520screening%2520based%2520on%2520multiple%2520crystal%2520structures%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2904%26epage%3D2914%26doi%3D10.1021%2Fci200257b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Akula, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, F. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, N.</span><span> </span><span class="NLM_article-title">Discovery of novel SecA inhibitors of Candidatus Liberibacter asiaticus by structure based design</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">4183</span><span class="NLM_x">–</span> <span class="NLM_lpage">4188</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2011.05.086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2011.05.086" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=4183-4188&author=N.+Akulaauthor=H.+Zhengauthor=F.+Q.+Hanauthor=N.+Wang&title=Discovery+of+novel+SecA+inhibitors+of+Candidatus+Liberibacter+asiaticus+by+structure+based+design&doi=10.1016%2Fj.bmcl.2011.05.086"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.05.086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.05.086%26sid%3Dliteratum%253Aachs%26aulast%3DAkula%26aufirst%3DN.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DHan%26aufirst%3DF.%2BQ.%26aulast%3DWang%26aufirst%3DN.%26atitle%3DDiscovery%2520of%2520novel%2520SecA%2520inhibitors%2520of%2520Candidatus%2520Liberibacter%2520asiaticus%2520by%2520structure%2520based%2520design%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D4183%26epage%3D4188%26doi%3D10.1016%2Fj.bmcl.2011.05.086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lehmann, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">Discovery of a novel acetylcholinesterase inhibitor by structure-based virtual screening techniques</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">3181</span><span class="NLM_x">–</span> <span class="NLM_lpage">3187</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2012.03.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2012.03.046" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=3181-3187&author=Y.+Chenauthor=L.+Fangauthor=S.+Pengauthor=H.+Liaoauthor=J.+Lehmannauthor=Y.+Zhang&title=Discovery+of+a+novel+acetylcholinesterase+inhibitor+by+structure-based+virtual+screening+techniques&doi=10.1016%2Fj.bmcl.2012.03.046"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.03.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.03.046%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DFang%26aufirst%3DL.%26aulast%3DPeng%26aufirst%3DS.%26aulast%3DLiao%26aufirst%3DH.%26aulast%3DLehmann%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520acetylcholinesterase%2520inhibitor%2520by%2520structure-based%2520virtual%2520screening%2520techniques%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D3181%26epage%3D3187%26doi%3D10.1016%2Fj.bmcl.2012.03.046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Ji, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, M.</span><span> </span><span class="NLM_article-title">Virtual screening of novel reversible inhibitors for marine alkaline protease MP</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">131</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2013.10.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.jmgm.2013.10.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsl2murzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2013&pages=125-131&author=X.+Jiauthor=Y.+Zhengauthor=W.+Wangauthor=J.+Shengauthor=J.+Haoauthor=M.+Sun&title=Virtual+screening+of+novel+reversible+inhibitors+for+marine+alkaline+protease+MP&doi=10.1016%2Fj.jmgm.2013.10.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening of novel reversible inhibitors for marine alkaline protease MP</span></div><div class="casAuthors">Ji, Xiaofeng; Zheng, Yuan; Wang, Wei; Sheng, Jun; Hao, Jianhua; Sun, Mi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">125-131</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Marine alk. protease (MP,2MP: the marine alk. protease, PDB id: 3U1R. accession no. ACY25898) is produced by a marine bacterium strain isolated from Yellow Sea sediment in China.  Previous research has shown that this protease is a cold-adapted enzyme with antioxidant activity that could be used as a detergent additive.  Owing to its instability in the liq. state, MP's application in liq. detergents was limited.  Therefore, the discovery of reversible MP inhibitors to stabilize the protease was imperative.  Here, we used the X-ray structure of MP and recompiled AutoDock 4.2 with refined Zn2+ characters to screen the free chem. database ZINC.  After completing the docking procedure, we applied strategies including the "initial filter", consensus scoring and pharmocophore model to accelerate the process and improve the virtual screening success rate.  The "initial filter" was built based on the docking results of boronic acid derivs. validated as reversible inhibitors of MP by our previous studies.  Finally, ten compds. were purchased or synthesized to test their binding affinity for MP.  Three of the compds. could reversibly inhibit MP with apparent Ki values of 0.8-1.2 mmol.  These active compds. and their binding modes provide useful information for understanding the mol. mechanism of reversible MP inhibition.  The results may also serve as the foundation for further screening and design of reversible MP inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohwfyjUL2g87Vg90H21EOLACvtfcHk0lhQ8R1YAxBzKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsl2murzE&md5=d95f6a7ed789e8dd89891dcc62d374e7</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2013.10.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2013.10.004%26sid%3Dliteratum%253Aachs%26aulast%3DJi%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DSheng%26aufirst%3DJ.%26aulast%3DHao%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DM.%26atitle%3DVirtual%2520screening%2520of%2520novel%2520reversible%2520inhibitors%2520for%2520marine%2520alkaline%2520protease%2520MP%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2013%26volume%3D46%26spage%3D125%26epage%3D131%26doi%3D10.1016%2Fj.jmgm.2013.10.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Kang, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, H. J.</span><span> </span><span class="NLM_article-title">In silico identification of novel ligands for G-quadruplex in the c-MYC promoter</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">339</span><span class="NLM_x">–</span> <span class="NLM_lpage">348</span><span class="refDoi"> DOI: 10.1007/s10822-014-9826-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs10822-014-9826-z" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=339-348&author=H.+J.+Kangauthor=H.+J.+Park&title=In+silico+identification+of+novel+ligands+for+G-quadruplex+in+the+c-MYC+promoter&doi=10.1007%2Fs10822-014-9826-z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9826-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9826-z%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DH.%2BJ.%26aulast%3DPark%26aufirst%3DH.%2BJ.%26atitle%3DIn%2520silico%2520identification%2520of%2520novel%2520ligands%2520for%2520G-quadruplex%2520in%2520the%2520c-MYC%2520promoter%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D339%26epage%3D348%26doi%3D10.1007%2Fs10822-014-9826-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Zhao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khadka, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, W. J.</span><span> </span><span class="NLM_article-title">Computer-aided discovery of aminopyridines as novel JAK2 inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">985</span><span class="NLM_x">–</span> <span class="NLM_lpage">995</span><span class="refDoi"> DOI: 10.1016/j.bmc.2015.01.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmc.2015.01.016" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=985-995&author=C.+Zhaoauthor=S.+H.+Yangauthor=D.+B.+Khadkaauthor=Y.+Jinauthor=K.+T.+Leeauthor=W.+J.+Cho&title=Computer-aided+discovery+of+aminopyridines+as+novel+JAK2+inhibitors&doi=10.1016%2Fj.bmc.2015.01.016"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.01.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.01.016%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DS.%2BH.%26aulast%3DKhadka%26aufirst%3DD.%2BB.%26aulast%3DJin%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DK.%2BT.%26aulast%3DCho%26aufirst%3DW.%2BJ.%26atitle%3DComputer-aided%2520discovery%2520of%2520aminopyridines%2520as%2520novel%2520JAK2%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D985%26epage%3D995%26doi%3D10.1016%2Fj.bmc.2015.01.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Lindert, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tallorin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, Q. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burkart, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">In silico screening for Plasmodium falciparum enoyl-ACP reductase inhibitors</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">79</span><span class="NLM_x">–</span> <span class="NLM_lpage">87</span><span class="refDoi"> DOI: 10.1007/s10822-014-9806-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs10822-014-9806-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=79-87&author=S.+Lindertauthor=L.+Tallorinauthor=Q.+G.+Nguyenauthor=M.+D.+Burkartauthor=J.+A.+McCammon&title=In+silico+screening+for+Plasmodium+falciparum+enoyl-ACP+reductase+inhibitors&doi=10.1007%2Fs10822-014-9806-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9806-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9806-3%26sid%3Dliteratum%253Aachs%26aulast%3DLindert%26aufirst%3DS.%26aulast%3DTallorin%26aufirst%3DL.%26aulast%3DNguyen%26aufirst%3DQ.%2BG.%26aulast%3DBurkart%26aufirst%3DM.%2BD.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DIn%2520silico%2520screening%2520for%2520Plasmodium%2520falciparum%2520enoyl-ACP%2520reductase%2520inhibitors%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D79%26epage%3D87%26doi%3D10.1007%2Fs10822-014-9806-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, M.</span><span> </span><span class="NLM_article-title">Discovery of novel EGFR tyrosine kinase inhibitors by structure-based virtual screening</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">4004</span><span class="NLM_x">–</span> <span class="NLM_lpage">4009</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2012.04.092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2012.04.092" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=4004-4009&author=S.+Liauthor=X.+Sunauthor=H.+Zhaoauthor=Y.+Tangauthor=M.+Lan&title=Discovery+of+novel+EGFR+tyrosine+kinase+inhibitors+by+structure-based+virtual+screening&doi=10.1016%2Fj.bmcl.2012.04.092"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.04.092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.04.092%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520novel%2520EGFR%2520tyrosine%2520kinase%2520inhibitors%2520by%2520structure-based%2520virtual%2520screening%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D4004%26epage%3D4009%26doi%3D10.1016%2Fj.bmcl.2012.04.092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Fang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, G. H.</span><span> </span><span class="NLM_article-title">Consensus models for CDK5 inhibitors in silico and their application to inhibitor discovery</span> <span class="citation_source-journal">Mol. Diversity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">149</span><span class="NLM_x">–</span> <span class="NLM_lpage">162</span><span class="refDoi"> DOI: 10.1007/s11030-014-9561-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs11030-014-9561-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2015&pages=149-162&author=J.+Fangauthor=R.+Yangauthor=L.+Gaoauthor=S.+Yangauthor=X.+Pangauthor=C.+Liauthor=Y.+Heauthor=A.+L.+Liuauthor=G.+H.+Du&title=Consensus+models+for+CDK5+inhibitors+in+silico+and+their+application+to+inhibitor+discovery&doi=10.1007%2Fs11030-014-9561-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1007%2Fs11030-014-9561-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11030-014-9561-3%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DPang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DA.%2BL.%26aulast%3DDu%26aufirst%3DG.%2BH.%26atitle%3DConsensus%2520models%2520for%2520CDK5%2520inhibitors%2520in%2520silico%2520and%2520their%2520application%2520to%2520inhibitor%2520discovery%26jtitle%3DMol.%2520Diversity%26date%3D2015%26volume%3D19%26spage%3D149%26epage%3D162%26doi%3D10.1007%2Fs11030-014-9561-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J.</span><span> </span><span class="NLM_article-title">Discovery of novel human acrosin inhibitors by virtual screening</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">977</span><span class="NLM_x">–</span> <span class="NLM_lpage">985</span><span class="refDoi"> DOI: 10.1007/s10822-011-9476-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs10822-011-9476-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=977-985&author=X.+Liuauthor=G.+Dongauthor=J.+Zhangauthor=J.+Qiauthor=C.+Zhengauthor=Y.+Zhouauthor=J.+Zhuauthor=C.+Shengauthor=J.+Lu&title=Discovery+of+novel+human+acrosin+inhibitors+by+virtual+screening&doi=10.1007%2Fs10822-011-9476-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9476-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9476-3%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DSheng%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520novel%2520human%2520acrosin%2520inhibitors%2520by%2520virtual%2520screening%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2011%26volume%3D25%26spage%3D977%26epage%3D985%26doi%3D10.1007%2Fs10822-011-9476-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Kokkonen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kokkola, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suuronen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poso, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarho, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lahtela-Kakkonen, M.</span><span> </span><span class="NLM_article-title">Virtual screening approach of sirtuin inhibitors results in two new scaffolds</span> <span class="citation_source-journal">Eur. J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">76</span><span class="NLM_x">, </span> <span class="NLM_fpage">27</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1016/j.ejps.2015.04.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.ejps.2015.04.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25936698" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsl2qtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2015&pages=27-32&author=P.+Kokkonenauthor=T.+Kokkolaauthor=T.+Suuronenauthor=A.+Posoauthor=E.+Jarhoauthor=M.+Lahtela-Kakkonen&title=Virtual+screening+approach+of+sirtuin+inhibitors+results+in+two+new+scaffolds&doi=10.1016%2Fj.ejps.2015.04.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening approach of sirtuin inhibitors results in two new scaffolds</span></div><div class="casAuthors">Kokkonen, Piia; Kokkola, Tarja; Suuronen, Tiina; Poso, Antti; Jarho, Elina; Lahtela-Kakkonen, Maija</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">27-32</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Sirtuins (SIRT1-SIRT7) are NAD dependent deacetylases and intriguing drug targets in for example neurodegenerative diseases and cancer.  Virtual screening has been shown to be a fast and efficient method of discovering new sirtuin inhibitors.  In this study, a new putative binding site on the zinc binding domain of sirtuins was utilized to screen the ZINC database virtually in order to discover new sirtuin inhibiting scaffolds.  Altogether 26 compds. were tested in vitro and initially 15 inhibitors displayed >30% SIRT3 inhibition.  However, the evaluation of raw data from in vitro assay revealed that many of the compds. had intrinsic property to interfere with the fluorescence signal at the assay wavelengths resulting in false positives.  All compds. with over 30% SIRT3 inhibition were studied more closely for their behavior in the assay and eventually, three compds. were identified as novel sirtuin inhibitors.  They displayed 32-40% SIRT3 and 21-60% SIRT2 inhibition.  The inhibitors display two new scaffolds, the smaller of which can be considered as a promising fragment, which offers a base for structural optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLEtTL9egZ0LVg90H21EOLACvtfcHk0liMOt7b10SksQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsl2qtLg%253D&md5=060b22ac9ee10687aa32f5cc8da4b58f</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.ejps.2015.04.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejps.2015.04.025%26sid%3Dliteratum%253Aachs%26aulast%3DKokkonen%26aufirst%3DP.%26aulast%3DKokkola%26aufirst%3DT.%26aulast%3DSuuronen%26aufirst%3DT.%26aulast%3DPoso%26aufirst%3DA.%26aulast%3DJarho%26aufirst%3DE.%26aulast%3DLahtela-Kakkonen%26aufirst%3DM.%26atitle%3DVirtual%2520screening%2520approach%2520of%2520sirtuin%2520inhibitors%2520results%2520in%2520two%2520new%2520scaffolds%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2015%26volume%3D76%26spage%3D27%26epage%3D32%26doi%3D10.1016%2Fj.ejps.2015.04.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Roughley, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span> </span><span class="NLM_article-title">How well can fragments explore accessed chemical space? A case study from heat shock protein 90</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">3989</span><span class="NLM_x">–</span> <span class="NLM_lpage">4005</span><span class="refDoi"> DOI: 10.1021/jm200350g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200350g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmsFymurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3989-4005&author=S.+D.+Roughleyauthor=R.+E.+Hubbard&title=How+well+can+fragments+explore+accessed+chemical+space%3F+A+case+study+from+heat+shock+protein+90&doi=10.1021%2Fjm200350g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">How Well Can Fragments Explore Accessed Chemical Space? A Case Study from Heat Shock Protein 90</span></div><div class="casAuthors">Roughley, Stephen D.; Hubbard, Roderick E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3989-4005</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  This article summerizes the published literature on the generation of inhibitors and design of drug against the mol. chaperone, heat shock protein 90 (Hsp90), as potential therapeutics to treat cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocGw-54Mo8SrVg90H21EOLACvtfcHk0liMOt7b10SksQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmsFymurs%253D&md5=90c2c8028cb460c0c6fdc956e7daae28</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Fjm200350g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200350g%26sid%3Dliteratum%253Aachs%26aulast%3DRoughley%26aufirst%3DS.%2BD.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DHow%2520well%2520can%2520fragments%2520explore%2520accessed%2520chemical%2520space%253F%2520A%2520case%2520study%2520from%2520heat%2520shock%2520protein%252090%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D3989%26epage%3D4005%26doi%3D10.1021%2Fjm200350g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Doman, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGovern, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witherbee, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasten, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurumbail, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stallings, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Connolly, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Molecular docking and high-throughput screening for novel inhibitors of protein tyrosine phosphatase-1B</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">2213</span><span class="NLM_x">–</span> <span class="NLM_lpage">2221</span><span class="refDoi"> DOI: 10.1021/jm010548w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm010548w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD38XivFCgtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2213-2221&author=T.+N.+Domanauthor=S.+L.+McGovernauthor=B.+J.+Witherbeeauthor=T.+P.+Kastenauthor=R.+Kurumbailauthor=W.+C.+Stallingsauthor=D.+T.+Connollyauthor=B.+K.+Shoichet&title=Molecular+docking+and+high-throughput+screening+for+novel+inhibitors+of+protein+tyrosine+phosphatase-1B&doi=10.1021%2Fjm010548w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular docking and high-throughput screening for novel inhibitors of protein tyrosine phosphatase-1B</span></div><div class="casAuthors">Doman, Thompson N.; McGovern, Susan L.; Witherbee, Bryan J.; Kasten, Thomas P.; Kurumbail, Ravi; Stallings, William C.; Connolly, Daniel T.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2213-2221</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) of compd. libraries is used to discover novel leads for drug development.  When a structure is available for the target, computer-based screening using mol. docking may also be considered.  The two techniques have rarely been used together on the same target.  The opportunity to do so presented itself in a project to discover novel inhibitors for the enzyme protein tyrosine phosphatase-1B (PTP1B), a tyrosine phosphatase that has been implicated as a key target for type II diabetes.  A corporate library of approx. 400 000 compds. was screened using high-throughput exptl. techniques for compds. that inhibited PTP1B.  Concurrently, mol. docking was used to screen approx. 235 000 com. available compds. against the X-ray crystallog. structure of PTP1B, and 365 high-scoring mols. were tested as inhibitors of the enzyme.  Of approx. 400 000 mols. tested in the high-throughput exptl. assay, 85 (0.021%) inhibited the enzyme with IC50 values less than 100 μM; the most active had an IC50 value of 4.2 μM.  Of the 365 mols. suggested by mol. docking, 127 (34.8%) inhibited PTP1B with IC50 values less than 100 μM; the most active of these had an IC50 of 1.7 μM.  Structure-based docking therefore enriched the hit rate by 1700-fold over random screening.  The hits from both the high-throughput and docking screens were dissimilar from phosphotyrosine, the canonical substrate group for PTP1B; the two hit lists were also very different from each other.  Surprisingly, the docking hits were judged to be more druglike than the HTS hits.  The diversity of both hit lists and their dissimilarity from each other suggest that docking and HTS may be complementary techniques for lead discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAMecsrqOs0LVg90H21EOLACvtfcHk0liMOt7b10SksQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XivFCgtbg%253D&md5=63e0730d9748e0dacbad1e9cdba93ca2</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fjm010548w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm010548w%26sid%3Dliteratum%253Aachs%26aulast%3DDoman%26aufirst%3DT.%2BN.%26aulast%3DMcGovern%26aufirst%3DS.%2BL.%26aulast%3DWitherbee%26aufirst%3DB.%2BJ.%26aulast%3DKasten%26aufirst%3DT.%2BP.%26aulast%3DKurumbail%26aufirst%3DR.%26aulast%3DStallings%26aufirst%3DW.%2BC.%26aulast%3DConnolly%26aufirst%3DD.%2BT.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DMolecular%2520docking%2520and%2520high-throughput%2520screening%2520for%2520novel%2520inhibitors%2520of%2520protein%2520tyrosine%2520phosphatase-1B%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D2213%26epage%3D2221%26doi%3D10.1021%2Fjm010548w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Feng, B. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C. P.</span><span> </span><span class="NLM_article-title">A high-throughput screen for aggregation-based inhibition in a large compound library</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2385</span><span class="NLM_x">–</span> <span class="NLM_lpage">2390</span><span class="refDoi"> DOI: 10.1021/jm061317y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm061317y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksVSlsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2385-2390&author=B.+Y.+Fengauthor=A.+Simeonovauthor=A.+Jadhavauthor=K.+Babaogluauthor=J.+Ingleseauthor=B.+K.+Shoichetauthor=C.+P.+Austin&title=A+high-throughput+screen+for+aggregation-based+inhibition+in+a+large+compound+library&doi=10.1021%2Fjm061317y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">A High-Throughput Screen for Aggregation-Based Inhibition in a Large Compound Library</span></div><div class="casAuthors">Feng, Brian Y.; Simeonov, Anton; Jadhav, Ajit; Babaoglu, Kerim; Inglese, James; Shoichet, Brian K.; Austin, Christopher P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2385-2390</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) is the primary technique for new lead identification in drug discovery and chem. biol.  Unfortunately, it is susceptible to false-pos. hits.  One common mechanism for such false-positives is the congregation of org. mols. into colloidal aggregates, which nonspecifically inhibit enzymes.  To both evaluate the feasibility of large-scale identification of aggregate-based inhibition and quantify its prevalence among screening hits, the authors tested 70 563 mols. from the National Institutes of Health Chem. Genomics Center (NCGC) library for detergent-sensitive inhibition.  Each mol. was screened in at least seven concns., such that dose-response curves were obtained for all mols. in the library.  There were 1274 inhibitors identified in total, of which 1204 were unambiguously detergent-sensitive.  The authors identified these as aggregate-based inhibitors.  Thirty-one library mols. were independently purchased and retested in secondary low-throughput expts.; 29 of these were confirmed as either aggregators or nonaggregators, as appropriate.  Finally, with the dose-response information collected for every compd., the authors could examine the correlation between aggregate-based inhibition and steep dose-response curves.  Three key results emerge from this study: first, detergent-dependent identification of aggregate-based inhibition is feasible on the large scale.  Second, 95% of the actives obtained in this screen are aggregate-based inhibitors.  Third, aggregate-based inhibition is correlated with steep dose-response curves, although not absolutely.  The results of this screen are being released publicly via the PubChem database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn9eDfL7UswbVg90H21EOLACvtfcHk0liH5vnLfyGCRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksVSlsrw%253D&md5=33399bcd4998faf830f54a6f0d1f636a</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fjm061317y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061317y%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DB.%2BY.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26atitle%3DA%2520high-throughput%2520screen%2520for%2520aggregation-based%2520inhibition%2520in%2520a%2520large%2520compound%2520library%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D2385%26epage%3D2390%26doi%3D10.1021%2Fjm061317y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferreira, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klumpp, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mott, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maloney, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span> </span><span class="NLM_article-title">Quantitative analyses of aggregation, autofluorescence, and reactivity artifacts in a screen for inhibitors of a thiol protease</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">37</span><span class="NLM_x">–</span> <span class="NLM_lpage">51</span><span class="refDoi"> DOI: 10.1021/jm901070c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901070c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVSgtLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=37-51&author=A.+Jadhavauthor=R.+S.+Ferreiraauthor=C.+Klumppauthor=B.+T.+Mottauthor=C.+P.+Austinauthor=J.+Ingleseauthor=C.+J.+Thomasauthor=D.+J.+Maloneyauthor=B.+K.+Shoichetauthor=A.+Simeonov&title=Quantitative+analyses+of+aggregation%2C+autofluorescence%2C+and+reactivity+artifacts+in+a+screen+for+inhibitors+of+a+thiol+protease&doi=10.1021%2Fjm901070c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative Analyses of Aggregation, Autofluorescence, and Reactivity Artifacts in a Screen for Inhibitors of a Thiol Protease</span></div><div class="casAuthors">Jadhav, Ajit; Ferreira, Rafaela S.; Klumpp, Carleen; Mott, Bryan T.; Austin, Christopher P.; Inglese, James; Thomas, Craig J.; Maloney, David J.; Shoichet, Brian K.; Simeonov, Anton</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-51</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The perceived and actual burden of false positives in high-throughput screening has received considerable attention; however, few studies exist on the contributions of distinct mechanisms of nonspecific effects like chem. reactivity, assay signal interference, and colloidal aggregation.  Here, we analyze the outcome of a screen of 197861 diverse compds. in a concn.-response format against the cysteine protease cruzain, a target expected to be particularly sensitive to reactive compds., and using an assay format with light detection in the short-wavelength region where significant compd. autofluorescence is typically encountered.  Approx. 1.9% of all compds. screened were detergent-sensitive inhibitors.  The contribution from autofluorescence and compds. bearing reactive functionalities was dramatically lower: of all hits, only 1.8% were autofluorescent and 1.5% contained reactive or undesired functional groups.  The distribution of false positives was relatively const. across library sources.  The simple step of including detergent in the assay buffer suppressed the nonspecific effect of approx. 93% of the original hits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0icdibXhgLbVg90H21EOLACvtfcHk0liH5vnLfyGCRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVSgtLjL&md5=c2a46f1632b2b7d79655c43cbae8fdba</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fjm901070c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901070c%26sid%3Dliteratum%253Aachs%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DFerreira%26aufirst%3DR.%2BS.%26aulast%3DKlumpp%26aufirst%3DC.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DQuantitative%2520analyses%2520of%2520aggregation%252C%2520autofluorescence%252C%2520and%2520reactivity%2520artifacts%2520in%2520a%2520screen%2520for%2520inhibitors%2520of%2520a%2520thiol%2520protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D37%26epage%3D51%26doi%3D10.1021%2Fjm901070c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Forli, S.</span><span> </span><span class="NLM_article-title">Charting a Path to Success in Virtual Screening</span> <span class="citation_source-journal">Molecules</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">18732</span><span class="NLM_x">–</span> <span class="NLM_lpage">18758</span><span class="refDoi"> DOI: 10.3390/molecules201018732</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.3390%2Fmolecules201018732" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26501243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslartLfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=18732-18758&author=S.+Forli&title=Charting+a+Path+to+Success+in+Virtual+Screening&doi=10.3390%2Fmolecules201018732"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Charting a path to success in virtual screening</span></div><div class="casAuthors">Forli, Stefano</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">18732-18758</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Docking is commonly applied to drug design efforts, esp. high-throughput virtual screenings of small mols., to identify new compds. that bind to a given target.  Despite great advances and successful applications in recent years, a no. of issues remain unsolved.  Most of the challenges and problems faced when running docking expts. are independent of the specific software used, and can be ascribed to either improper input prepn. or to the simplified approaches applied to achieve high-throughput speed.  Being aware of approxns. and limitations of such methods is essential to prevent errors, deal with misleading results, and increase the success rate of virtual screening campaigns.  In this review, best practices and most common issues of docking and virtual screening will be discussed, covering the journey from the design of the virtual expt. to the hit identification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRNqBR6AUIubVg90H21EOLACvtfcHk0liH5vnLfyGCRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslartLfL&md5=991965e4159b6616ecfe45357ef595f7</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.3390%2Fmolecules201018732&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules201018732%26sid%3Dliteratum%253Aachs%26aulast%3DForli%26aufirst%3DS.%26atitle%3DCharting%2520a%2520Path%2520to%2520Success%2520in%2520Virtual%2520Screening%26jtitle%3DMolecules%26date%3D2015%26volume%3D20%26spage%3D18732%26epage%3D18758%26doi%3D10.3390%2Fmolecules201018732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Spyrakis, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavasotto, C. N.</span><span> </span><span class="NLM_article-title">Open challenges in structure-based virtual screening: Receptor modeling, target flexibility consideration and active site water molecules description</span> <span class="citation_source-journal">Arch. Biochem. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">583</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">119</span><span class="refDoi"> DOI: 10.1016/j.abb.2015.08.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.abb.2015.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26271444" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFWmtb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=583&publication_year=2015&pages=105-119&author=F.+Spyrakisauthor=C.+N.+Cavasotto&title=Open+challenges+in+structure-based+virtual+screening%3A+Receptor+modeling%2C+target+flexibility+consideration+and+active+site+water+molecules+description&doi=10.1016%2Fj.abb.2015.08.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Open challenges in structure-based virtual screening: Receptor modeling, target flexibility consideration and active site water molecules description</span></div><div class="casAuthors">Spyrakis, Francesca; Cavasotto, Claudio N.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">583</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">105-119</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Structure-based virtual screening is currently an established tool in drug lead discovery projects.  Although in the last years the field saw an impressive progress in terms of algorithm development, computational performance, and retrospective and prospective applications in ligand identification, there are still long-standing challenges where further improvement is needed.  In this review, we consider the conceptual frame, state-of-the-art and recent developments of three crit. "structural" issues in structure-based drug lead discovery: the use of homol. modeling to accurately model the binding site when no exptl. structures are available, the necessity of accounting for the dynamics of intrinsically flexible systems as proteins, and the importance of considering active site water mols. in lead identification and optimization campaigns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmOGftaXjkYrVg90H21EOLACvtfcHk0lg9Q_WeHprN0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFWmtb%252FK&md5=b8818c43713169365741420862ff4448</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2015.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2015.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DSpyrakis%26aufirst%3DF.%26aulast%3DCavasotto%26aufirst%3DC.%2BN.%26atitle%3DOpen%2520challenges%2520in%2520structure-based%2520virtual%2520screening%253A%2520Receptor%2520modeling%252C%2520target%2520flexibility%2520consideration%2520and%2520active%2520site%2520water%2520molecules%2520description%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2015%26volume%3D583%26spage%3D105%26epage%3D119%26doi%3D10.1016%2Fj.abb.2015.08.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">Ferreira, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dos Santos, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliva, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andricopulo, A. D.</span><span> </span><span class="NLM_article-title">Molecular docking and structure-based drug design strategies</span> <span class="citation_source-journal">Molecules</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">13384</span><span class="NLM_x">–</span> <span class="NLM_lpage">13421</span><span class="refDoi"> DOI: 10.3390/molecules200713384</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.3390%2Fmolecules200713384" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26205061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1OjtLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=13384-13421&author=L.+G.+Ferreiraauthor=R.+N.+Dos+Santosauthor=G.+Olivaauthor=A.+D.+Andricopulo&title=Molecular+docking+and+structure-based+drug+design+strategies&doi=10.3390%2Fmolecules200713384"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular docking and structure-based drug design strategies</span></div><div class="casAuthors">Ferreira, Leonardo G.; dos Santos, Ricardo N.; Oliva, Glaucius; Andricopulo, Adriano D.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">13384-13421</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Pharmaceutical research has successfully incorporated a wealth of mol. modeling methods, within a variety of drug discovery programs, to study complex biol. and chem. systems.  The integration of computational and exptl. strategies has been of great value in the identification and development of novel promising compds.  Broadly used in modern drug design, mol. docking methods explore the ligand conformations adopted within the binding sites of macromol. targets.  This approach also ests. the ligand-receptor binding free energy by evaluating crit. phenomena involved in the intermol. recognition process.  Today, as a variety of docking algorithms are available, an understanding of the advantages and limitations of each method is of fundamental importance in the development of effective strategies and the generation of relevant results.  The purpose of this review is to examine current mol. docking strategies used in drug discovery and medicinal chem., exploring the advances in the field and the role played by the integration of structure- and ligand-based methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf5La9X0Wng7Vg90H21EOLACvtfcHk0lg9Q_WeHprN0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1OjtLzJ&md5=b5f70b90e45098d0f0252a2926f9671e</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.3390%2Fmolecules200713384&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules200713384%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%26aufirst%3DL.%2BG.%26aulast%3DDos%2BSantos%26aufirst%3DR.%2BN.%26aulast%3DOliva%26aufirst%3DG.%26aulast%3DAndricopulo%26aufirst%3DA.%2BD.%26atitle%3DMolecular%2520docking%2520and%2520structure-based%2520drug%2520design%2520strategies%26jtitle%3DMolecules%26date%3D2015%26volume%3D20%26spage%3D13384%26epage%3D13421%26doi%3D10.3390%2Fmolecules200713384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group">Cerqueira, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gesto, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliveira, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santos-Martins, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bras, N. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sousa, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandes, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramos, M. J.</span><span> </span><span class="NLM_article-title">Receptor-based virtual screening protocol for drug discovery</span> <span class="citation_source-journal">Arch. Biochem. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">582</span><span class="NLM_x">, </span> <span class="NLM_fpage">56</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.1016/j.abb.2015.05.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.abb.2015.05.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26045247" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpsVyls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=582&publication_year=2015&pages=56-67&author=N.+M.+Cerqueiraauthor=D.+Gestoauthor=E.+F.+Oliveiraauthor=D.+Santos-Martinsauthor=N.+F.+Brasauthor=S.+F.+Sousaauthor=P.+A.+Fernandesauthor=M.+J.+Ramos&title=Receptor-based+virtual+screening+protocol+for+drug+discovery&doi=10.1016%2Fj.abb.2015.05.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Receptor-based virtual screening protocol for drug discovery</span></div><div class="casAuthors">Cerqueira, Nuno M. F. S. A.; Gesto, Diana; Oliveira, Eduardo F.; Santos-Martins, Diogo; Bras, Natercia F.; Sousa, Sergio F.; Fernandes, Pedro A.; Ramos, Maria J.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">582</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">56-67</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Computational aided drug design (CADD) is presently a key component in the process of drug discovery and development as it offers great promise to drastically reduce cost and time requirements.In the pharmaceutical arena, virtual screening is normally regarded as the top CADD tool to screen large libraries of chem. structures and reduce them to a key set of likely drug candidates regarding a specific protein target.  This chapter provides a comprehensive overview of the receptor-based virtual screening process and of its importance in the present drug discovery and development paradigm.  Following a focused contextualization on the subject, the main stages of a virtual screening campaign, including its strengths and limitations, are the subject of particular attention in this review.  In all of these stages special consideration will be given to practical issues that are normally the Achilles heel of the virtual screening process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDV14aAOZb_LVg90H21EOLACvtfcHk0lg9Q_WeHprN0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpsVyls78%253D&md5=c6f861d3150b3e76fb3a9b5b905d195a</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2015.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2015.05.011%26sid%3Dliteratum%253Aachs%26aulast%3DCerqueira%26aufirst%3DN.%2BM.%26aulast%3DGesto%26aufirst%3DD.%26aulast%3DOliveira%26aufirst%3DE.%2BF.%26aulast%3DSantos-Martins%26aufirst%3DD.%26aulast%3DBras%26aufirst%3DN.%2BF.%26aulast%3DSousa%26aufirst%3DS.%2BF.%26aulast%3DFernandes%26aufirst%3DP.%2BA.%26aulast%3DRamos%26aufirst%3DM.%2BJ.%26atitle%3DReceptor-based%2520virtual%2520screening%2520protocol%2520for%2520drug%2520discovery%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2015%26volume%3D582%26spage%3D56%26epage%3D67%26doi%3D10.1016%2Fj.abb.2015.05.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group">Kumalo, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhakat, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soliman, M. E.</span><span> </span><span class="NLM_article-title">Theory and applications of covalent docking in drug discovery: merits and pitfalls</span> <span class="citation_source-journal">Molecules</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1984</span><span class="NLM_x">–</span> <span class="NLM_lpage">2000</span><span class="refDoi"> DOI: 10.3390/molecules20021984</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.3390%2Fmolecules20021984" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25633330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitF2gsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=1984-2000&author=H.+M.+Kumaloauthor=S.+Bhakatauthor=M.+E.+Soliman&title=Theory+and+applications+of+covalent+docking+in+drug+discovery%3A+merits+and+pitfalls&doi=10.3390%2Fmolecules20021984"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Theory and applications of covalent docking in drug discovery: merits and pitfalls</span></div><div class="casAuthors">Kumalo, Hezekiel Mathambo; Bhakat, Soumendranath; Soliman, Mahmoud E. S.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1984-2000, 17 pp.</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  The present art of drug discovery and design of new drugs is based on suicidal irreversible inhibitors.  Covalent inhibition is the strategy that is used to achieve irreversible inhibition.  Irreversible inhibitors interact with their targets in a time-dependent fashion, and the reaction proceeds to completion rather than to equil.  Covalent inhibitors possessed some significant advantages over non-covalent inhibitors such as covalent warheads can target rare, non-conserved residue of a particular target protein and thus led to development of highly selective inhibitors, covalent inhibitors can be effective in targeting proteins with shallow binding cleavage which will led to development of novel inhibitors with increased potency than non-covalent inhibitors.  Several computational approaches have been developed to simulate covalent interactions; however, this is still a challenging area to explore.  Covalent mol. docking has been recently implemented in the computer-aided drug design workflows to describe covalent interactions between inhibitors and biol. targets.  In this review we highlight: (i) covalent interactions in biomol. systems; (ii) the math. framework of covalent mol. docking; (iii) implementation of covalent docking protocol in drug design workflows; (iv) applications covalent docking: case studies and (v) shortcomings and future perspectives of covalent docking.  To the best of our knowledge; this review is the first account that highlights different aspects of covalent docking with its merits and pitfalls.  We believe that the method and applications highlighted in this study will help future efforts towards the design of irreversible inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyAF4rJ5aS7rVg90H21EOLACvtfcHk0lhUH-dqmQmv2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitF2gsrg%253D&md5=4c47470306282d34bab152867dc522e3</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.3390%2Fmolecules20021984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules20021984%26sid%3Dliteratum%253Aachs%26aulast%3DKumalo%26aufirst%3DH.%2BM.%26aulast%3DBhakat%26aufirst%3DS.%26aulast%3DSoliman%26aufirst%3DM.%2BE.%26atitle%3DTheory%2520and%2520applications%2520of%2520covalent%2520docking%2520in%2520drug%2520discovery%253A%2520merits%2520and%2520pitfalls%26jtitle%3DMolecules%26date%3D2015%26volume%3D20%26spage%3D1984%26epage%3D2000%26doi%3D10.3390%2Fmolecules20021984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group">Webb, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Protein structure modeling with MODELLER</span> <span class="citation_source-journal">Methods Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1137</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">15</span><span class="refDoi"> DOI: 10.1007/978-1-4939-0366-5_1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2F978-1-4939-0366-5_1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24573470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns12isrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1137&publication_year=2014&pages=1-15&author=B.+Webbauthor=A.+Sali&title=Protein+structure+modeling+with+MODELLER&doi=10.1007%2F978-1-4939-0366-5_1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Structure Modeling with MODELLER</span></div><div class="casAuthors">Webb, Benjamin; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1137</span>
        (<span class="NLM_cas:issue">Protein Structure Prediction</span>),
    <span class="NLM_cas:pages">1-15</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Genome sequencing projects have resulted in a rapid increase in the no. of known protein sequences.  In contrast, only about one-hundredth of these sequences have been characterized at at. resoln. using exptl. structure detn. methods.  Computational protein structure modeling techniques have the potential to bridge this sequence-structure gap.  In this chapter, we present an example that illustrates the use of MODELLER to construct a comparative model for a protein with unknown structure.  Automation of a similar protocol has resulted in models of useful accuracy for domains in more than half of all known protein sequences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs6xvMHHyK0bVg90H21EOLACvtfcHk0lhUH-dqmQmv2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns12isrc%253D&md5=19709a3ffefe33fd6e5d8506258bd6f0</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-0366-5_1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-0366-5_1%26sid%3Dliteratum%253Aachs%26aulast%3DWebb%26aufirst%3DB.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DProtein%2520structure%2520modeling%2520with%2520MODELLER%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2014%26volume%3D1137%26spage%3D1%26epage%3D15%26doi%3D10.1007%2F978-1-4939-0366-5_1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group">Pieper, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, G. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneidman-Duhovny, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuri, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spill, Y. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinkam, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tainer, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilges, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">ModBase, a database of annotated comparative protein structure models and associated resources</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">D336</span><span class="NLM_x">–</span> <span class="NLM_lpage">346</span><span class="refDoi"> DOI: 10.1093/nar/gkt1144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1093%2Fnar%2Fgkt1144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24271400" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1Gj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D336-346&author=U.+Pieperauthor=B.+M.+Webbauthor=G.+Q.+Dongauthor=D.+Schneidman-Duhovnyauthor=H.+Fanauthor=S.+J.+Kimauthor=N.+Khuriauthor=Y.+G.+Spillauthor=P.+Weinkamauthor=M.+Hammelauthor=J.+A.+Tainerauthor=M.+Nilgesauthor=A.+Sali&title=ModBase%2C+a+database+of+annotated+comparative+protein+structure+models+and+associated+resources&doi=10.1093%2Fnar%2Fgkt1144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">ModBase, a database of annotated comparative protein structure models and associated resources</span></div><div class="casAuthors">Pieper, Ursula; Webb, Benjamin M.; Dong, Guang Qiang; Schneidman-Duhovny, Dina; Fan, Hao; Kim, Seung Joong; Khuri, Natalia; Spill, Yannick G.; Weinkam, Patrick; Hammel, Michal; Tainer, John A.; Nilges, Michael; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D336-D346</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ModBase http://salilab.org/modbase is a database of annotated comparative protein structure models.  The models are calcd. by ModPipe, an automated modeling pipeline that relies primarily on Modeller for fold assignment, sequence-structure alignment, model building and model assessment http://salilab.org/modeller/.  ModBase currently contains almost 30 million reliable models for domains in 4.7 million unique protein sequences.  ModBase allows users to compute or update comparative models on demand, through an interface to the ModWeb modeling server http://salilab.org/modweb.  ModBase models are also available through the Protein Model Portal http://www.proteinmodelportal.org/.  Recently developed assocd. resources include the AllosMod server for modeling ligand-induced protein dynamics http://salilab.org/allosmod, the AllosMod-FoXS server for predicting a structural ensemble that fits an SAXS profile http://salilab.org/allosmod-foxs, the FoXSDock server for protein-protein docking filtered by an SAXS profile http://salilab.org/foxsdock, the SAXS Merge server for automatic merging of SAXS profiles http://salilab.org/saxsmerge and the Pose & Rank server for scoring protein-ligand complexes http://salilab.org/poseandrank. In this update, we also highlight two applications of ModBase: a PSI:Biol. initiative to maximize the structural coverage of the human alpha-helical transmembrane proteome and a detn. of structural determinants of human immunodeficiency virus-1 protease specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomlF6VKj_t4bVg90H21EOLACvtfcHk0lhUH-dqmQmv2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1Gj&md5=1a0809c46145a0a783ad5d5b2a6e3326</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1144%26sid%3Dliteratum%253Aachs%26aulast%3DPieper%26aufirst%3DU.%26aulast%3DWebb%26aufirst%3DB.%2BM.%26aulast%3DDong%26aufirst%3DG.%2BQ.%26aulast%3DSchneidman-Duhovny%26aufirst%3DD.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DS.%2BJ.%26aulast%3DKhuri%26aufirst%3DN.%26aulast%3DSpill%26aufirst%3DY.%2BG.%26aulast%3DWeinkam%26aufirst%3DP.%26aulast%3DHammel%26aufirst%3DM.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26aulast%3DNilges%26aufirst%3DM.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DModBase%252C%2520a%2520database%2520of%2520annotated%2520comparative%2520protein%2520structure%2520models%2520and%2520associated%2520resources%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DD336%26epage%3D346%26doi%3D10.1093%2Fnar%2Fgkt1144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Virtual ligand screening against comparative protein structure models</span> <span class="citation_source-journal">Methods Mol. Biol. (N. Y., NY, U. S.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">819</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">126</span><span class="refDoi"> DOI: 10.1007/978-1-61779-465-0_8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2F978-1-61779-465-0_8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=22183533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BC38%252Fos1Cjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=819&publication_year=2012&pages=105-126&author=H.+Fanauthor=J.+J.+Irwinauthor=A.+Sali&title=Virtual+ligand+screening+against+comparative+protein+structure+models&doi=10.1007%2F978-1-61779-465-0_8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual ligand screening against comparative protein structure models</span></div><div class="casAuthors">Fan Hao; Irwin John J; Sali Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Methods in molecular biology (Clifton, N.J.)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">819</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">105-26</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Virtual ligand screening uses computation to discover new ligands of a protein by screening one or more of its structural models against a database of potential ligands.  Comparative protein structure modeling extends the applicability of virtual screening beyond the atomic structures determined by X-ray crystallography or NMR spectroscopy.  Here, we describe an integrated modeling and docking protocol, combining comparative modeling by MODELLER and virtual ligand screening by DOCK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS3KPL-W_yq5JagfuraLLzSfW6udTcc2eZBNya1mVfGeLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38%252Fos1Cjuw%253D%253D&md5=e94c5ee89cbf4ffcd3e3b0b39358c908</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-465-0_8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-465-0_8%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DH.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DVirtual%2520ligand%2520screening%2520against%2520comparative%2520protein%2520structure%2520models%26jtitle%3DMethods%2520Mol.%2520Biol.%2520%2528N.%2520Y.%252C%2520NY%252C%2520U.%2520S.%2529%26date%3D2012%26volume%3D819%26spage%3D105%26epage%3D126%26doi%3D10.1007%2F978-1-61779-465-0_8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Exploiting ordered waters in molecular docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">4862</span><span class="NLM_x">–</span> <span class="NLM_lpage">4865</span><span class="refDoi"> DOI: 10.1021/jm8006239</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8006239" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptl2hs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=4862-4865&author=N.+Huangauthor=B.+K.+Shoichet&title=Exploiting+ordered+waters+in+molecular+docking&doi=10.1021%2Fjm8006239"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting Ordered Waters in Molecular Docking</span></div><div class="casAuthors">Huang, Niu; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4862-4865</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A current weakness in docking is the treatment of water-mediated protein-ligand interactions.  The authors explore switching ordered water mols. "on" and "off" during docking screens of a large library.  The method assumes additivity and scales linearly with the no. of waters sampled despite the exponential growth in configurations.  It is tested for ligand enrichment against 24 targets, exploring up to 256 water configurations.  Water inclusion increased enrichment substantially for 12 targets, while most others were largely unaffected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0tyvmJzfbCbVg90H21EOLACvtfcHk0lhpmPcoIH5mJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptl2hs7o%253D&md5=2f0430f9a71bb76e68977fcdff800366</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1021%2Fjm8006239&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8006239%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DExploiting%2520ordered%2520waters%2520in%2520molecular%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D4862%26epage%3D4865%26doi%3D10.1021%2Fjm8006239" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Virtual screening in lead discovery and optimization</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">396</span><span class="NLM_x">–</span> <span class="NLM_lpage">403</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=15338948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpvFOmsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2004&pages=396-403&author=A.+N.+Jain&title=Virtual+screening+in+lead+discovery+and+optimization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening in lead discovery and optimization</span></div><div class="casAuthors">Jain, Ajay N.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">396-403</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Thomson Scientific</span>)
        </div><div class="casAbstract">A review.  Virtual screening by mol. docking, using a protein with an exptl. detd. structure as a target, has become an established method for lead discovery and for enhancing efficiency in lead optimization.  Generalizations of the quant. structure-activity relationship concept have led to approaches for virtual screening in the absence of a protein target structure, instead relying upon ligand-based models as surrogates of protein active sites.  Recently reported methods for ligand-based virtual screening can achieve similar enrichment rates to those obtained using mol. docking.  This review will discuss recent advances in both domains of virtual screening, including theor. and practical advances and the implications for their application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeVT5-ZGuZn7Vg90H21EOLACvtfcHk0lhpmPcoIH5mJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpvFOmsrs%253D&md5=7fddac472fc1eda64682db69a7fbb843</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DVirtual%2520screening%2520in%2520lead%2520discovery%2520and%2520optimization%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2004%26volume%3D7%26spage%3D396%26epage%3D403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartshorn, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nissink, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span> </span><span class="NLM_article-title">Modeling water molecules in protein-ligand docking using GOLD</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">6504</span><span class="NLM_x">–</span> <span class="NLM_lpage">6515</span><span class="refDoi"> DOI: 10.1021/jm050543p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050543p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFCnu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6504-6515&author=M.+L.+Verdonkauthor=G.+Chessariauthor=J.+C.+Coleauthor=M.+J.+Hartshornauthor=C.+W.+Murrayauthor=J.+W.+Nissinkauthor=R.+D.+Taylorauthor=R.+Taylor&title=Modeling+water+molecules+in+protein-ligand+docking+using+GOLD&doi=10.1021%2Fjm050543p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling Water Molecules in Protein-Ligand Docking Using GOLD</span></div><div class="casAuthors">Verdonk, Marcel L.; Chessari, Gianni; Cole, Jason C.; Hartshorn, Michael J.; Murray, Christopher W.; Nissink, J. Willem M.; Taylor, Richard D.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6504-6515</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We implemented a novel approach to score water mediation and displacement in the protein-ligand docking program GOLD.  The method allows water mols. to switch on and off and to rotate around their three principal axes.  A const. penalty, σp, representing the loss of rigid-body entropy, is added for water mols. that are switched on, hence rewarding water displacement.  We tested the methodol. in an extensive validation study.  First, σp is optimized against a training set of 58 protein-ligand complexes.  For this training set, our algorithm correctly predicts water mediation/displacement in ∼92% of the cases.  We obsd. small improvements in the quality of the predicted binding modes for water-mediated complexes.  In the second part of this work, an entirely independent set of 225 complexes is used.  For this test set, our algorithm correctly predicts water mediation/displacement in ∼93% of the cases.  Improvements in binding mode quality were obsd. for individual water-mediated complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKBGOf-rFSarVg90H21EOLACvtfcHk0lhpmPcoIH5mJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFCnu78%253D&md5=78b60f794d8af7266f0a37f6e2aea6b1</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1021%2Fjm050543p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050543p%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DModeling%2520water%2520molecules%2520in%2520protein-ligand%2520docking%2520using%2520GOLD%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6504%26epage%3D6515%26doi%3D10.1021%2Fjm050543p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carchia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Directory of useful decoys, enhanced (DUD-E): better ligands and decoys for better benchmarking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">6582</span><span class="NLM_x">–</span> <span class="NLM_lpage">6594</span><span class="refDoi"> DOI: 10.1021/jm300687e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300687e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFaku7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6582-6594&author=M.+M.+Mysingerauthor=M.+Carchiaauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Directory+of+useful+decoys%2C+enhanced+%28DUD-E%29%3A+better+ligands+and+decoys+for+better+benchmarking&doi=10.1021%2Fjm300687e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Directory of Useful Decoys, Enhanced (DUD-E): Better Ligands and Decoys for Better Benchmarking</span></div><div class="casAuthors">Mysinger, Michael M.; Carchia, Michael; Irwin, John. J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6582-6594</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A key metric to assess mol. docking remains ligand enrichment against challenging decoys.  Whereas the directory of useful decoys (DUD) has been widely used, clear areas for optimization have emerged.  Here we describe an improved benchmarking set that includes more diverse targets such as GPCRs and ion channels, totaling 102 proteins with 22886 clustered ligands drawn from ChEMBL, each with 50 property-matched decoys drawn from ZINC.  To ensure chemotype diversity, we cluster each target's ligands by their Bemis-Murcko at. frameworks.  We add net charge to the matched physicochem. properties and include only the most dissimilar decoys, by topol., from the ligands.  An online automated tool (http://decoys.docking.org) generates these improved matched decoys for user-supplied ligands.  We test this data set by docking all 102 targets, using the results to improve the balance between ligand desolvation and electrostatics in DOCK 3.6.  The complete DUD-E benchmarking set is freely available at http://dude.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowquULXHTtJLVg90H21EOLACvtfcHk0lgB3pNYmfhI2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFaku7c%253D&md5=23ba4049662b388a880d4a48dde3769c</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1021%2Fjm300687e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300687e%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DCarchia%26aufirst%3DM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDirectory%2520of%2520useful%2520decoys%252C%2520enhanced%2520%2528DUD-E%2529%253A%2520better%2520ligands%2520and%2520decoys%2520for%2520better%2520benchmarking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6582%26epage%3D6594%26doi%3D10.1021%2Fjm300687e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group">Carr, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery: leads by design</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">987</span><span class="NLM_x">–</span> <span class="NLM_lpage">992</span><span class="refDoi"> DOI: 10.1016/S1359-6446(05)03511-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2FS1359-6446%2805%2903511-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=16023057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmt1Whtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=987-992&author=R.+A.+Carrauthor=M.+Congreveauthor=C.+W.+Murrayauthor=D.+C.+Rees&title=Fragment-based+lead+discovery%3A+leads+by+design&doi=10.1016%2FS1359-6446%2805%2903511-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery: leads by design</span></div><div class="casAuthors">Carr, Robin A. E.; Congreve, Miles; Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">987-992</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Fragment-based lead discovery (also referred to as needles, shapes, binding elements, seed templates or scaffolds) is a new lead discovery approach in which much lower mol. wt. (120-250Da) compds. are screened relative to HTS campaigns.  Fragment-based hits are typically weak inhibitors (10μM-mM), and therefore need to be screened at higher concn. using very sensitive biophys. detection techniques such as protein crystallog. and NMR as the primary screening techniques, rather than bioassays.  Compared with HTS hits, these fragments are simpler, less functionalized compds. with correspondingly lower affinity.  However, fragment hits typically possess high ligand efficiency' (binding affinity per heavy atom) and so are highly suitable for optimization into clin. candidates with good drug-like properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq15FNhjT6Qw7Vg90H21EOLACvtfcHk0lgB3pNYmfhI2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmt1Whtr0%253D&md5=1bf7dd97ec056af9ed5143f96b34bb53</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2805%2903511-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252805%252903511-7%26sid%3Dliteratum%253Aachs%26aulast%3DCarr%26aufirst%3DR.%2BA.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DFragment-based%2520lead%2520discovery%253A%2520leads%2520by%2520design%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26spage%3D987%26epage%3D992%26doi%3D10.1016%2FS1359-6446%2805%2903511-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group">Oprea, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teague, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leeson, P. D.</span><span> </span><span class="NLM_article-title">Is there a difference between leads and drugs? A historical perspective</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">1308</span><span class="NLM_x">–</span> <span class="NLM_lpage">1315</span><span class="refDoi"> DOI: 10.1021/ci010366a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci010366a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlt1Okt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2001&pages=1308-1315&author=T.+I.+Opreaauthor=A.+M.+Davisauthor=S.+J.+Teagueauthor=P.+D.+Leeson&title=Is+there+a+difference+between+leads+and+drugs%3F+A+historical+perspective&doi=10.1021%2Fci010366a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Is There a Difference between Leads and Drugs? A Historical Perspective</span></div><div class="casAuthors">Oprea, Tudor I.; Davis, Andrew M.; Teague, Simon J.; Leeson, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1308-1315</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To be considered for further development, lead structures should display the following properties: (1) simple chem. features, amenable for chem. optimization; (2) membership to an established SAR series; (3) favorable patent situation; and (4) good absorption, distribution, metab., and excretion (ADME) properties.  There are two distinct categories of leads: those that lack any therapeutic use (i.e., "pure" leads), and those that are marketed drugs themselves but have been altered to yield novel drugs.  We have previously analyzed the design of leadlike combinatorial libraries starting from 18 lead and drug pairs of structures (S. J. Teague et al. Angew. Chem., Int. Ed. Engl. 1999, 38, 3743-3748).  Here, we report results based on an extended dataset of 96 lead-drug pairs, of which 62 are lead structures that are not marketed as drugs, and 75 are drugs that are not presumably used as leads.  We examd. the following properties: MW (mol. wt.), CMR (the calcd. mol. refractivity), RNG (the no. of rings), RTB (the no. of rotatable bonds), the no. of hydrogen bond donors (HDO) and acceptors (HAC), the calcd. logarithm of the n-octanol/water partition (CLogP), the calcd. logarithm of the distribution coeff. at pH 7.4 (LogD74), the Daylight-fingerprint druglike score (DFPS), and the property and pharmacophore features score (PPFS).  The following differences were obsd. between the medians of drugs and leads: ΔMW = 69; ΔCMR = 1.8; ΔRNG = ΔHAC =1; ΔRTB = 2; ΔCLogP = 0.43; ΔLogD74 = 0.97; ΔHDO = 0; ΔDFPS = 0.15; ΔPPFS = 0.12.  Lead structures exhibit, on the av., less mol. complexity (less MW, less no. of rings and rotatable bonds), are less hydrophobic (lower CLogP and LogD74), and less druglike (lower druglike scores).  These findings indicate that the process of optimizing a lead into a drug results in more complex structures.  This information should be used in the design of novel combinatorial libraries that are aimed at lead discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrG4E0R8n7tLrVg90H21EOLACvtfcHk0lgB3pNYmfhI2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlt1Okt7s%253D&md5=422b9cf937168d9c2f3d1c8429648ea0</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Fci010366a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci010366a%26sid%3Dliteratum%253Aachs%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DDavis%26aufirst%3DA.%2BM.%26aulast%3DTeague%26aufirst%3DS.%2BJ.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26atitle%3DIs%2520there%2520a%2520difference%2520between%2520leads%2520and%2520drugs%253F%2520A%2520historical%2520perspective%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2001%26volume%3D41%26spage%3D1308%26epage%3D1315%26doi%3D10.1021%2Fci010366a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group">Milletti, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hermann, J. C.</span><span> </span><span class="NLM_article-title">Targeted kinase selectivity from kinase profiling data</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">383</span><span class="NLM_x">–</span> <span class="NLM_lpage">386</span><span class="refDoi"> DOI: 10.1021/ml300012r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml300012r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=383-386&author=F.+Millettiauthor=J.+C.+Hermann&title=Targeted+kinase+selectivity+from+kinase+profiling+data&doi=10.1021%2Fml300012r"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1021%2Fml300012r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml300012r%26sid%3Dliteratum%253Aachs%26aulast%3DMilletti%26aufirst%3DF.%26aulast%3DHermann%26aufirst%3DJ.%2BC.%26atitle%3DTargeted%2520kinase%2520selectivity%2520from%2520kinase%2520profiling%2520data%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D383%26epage%3D386%26doi%3D10.1021%2Fml300012r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Contribution of conformer focusing to the uncertainty in predicting free energies for protein-ligand binding</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">5880</span><span class="NLM_x">–</span> <span class="NLM_lpage">5884</span><span class="refDoi"> DOI: 10.1021/jm060763i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060763i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD28Xptl2hsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=5880-5884&author=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Contribution+of+conformer+focusing+to+the+uncertainty+in+predicting+free+energies+for+protein-ligand+binding&doi=10.1021%2Fjm060763i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Contribution of Conformer Focusing to the Uncertainty in Predicting Free Energies for Protein-Ligand Binding</span></div><div class="casAuthors">Tirado-Rives, Julian; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5880-5884</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">When a ligand binds to a protein, it is typically not in the lowest-energy conformation for the unbound ligand and there is also a loss of conformational degrees of freedom.  The free-energy change for this "conformer focusing" is addressed here formally, and the assocd. errors with its estn. or neglect are considered in the context of scoring functions for protein-ligand docking and computation of abs. free energies of binding.  Specific applications for inhibition of HIV-1 reverse transcriptase are reported.  It is concluded that the uncertainties from this source alone are sufficient to preclude the viability of current docking methodol. for rank-ordering of diverse compds. in high-throughput virtual screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS33v6gg6p27Vg90H21EOLACvtfcHk0ligflrrI4c8rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xptl2hsLk%253D&md5=8a24236944e39caece2bd5dde96a7ca8</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1021%2Fjm060763i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060763i%26sid%3Dliteratum%253Aachs%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DContribution%2520of%2520conformer%2520focusing%2520to%2520the%2520uncertainty%2520in%2520predicting%2520free%2520energies%2520for%2520protein-ligand%2520binding%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D5880%26epage%3D5884%26doi%3D10.1021%2Fjm060763i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group">Afzal, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Firoz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaggi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bawa, S.</span><span> </span><span class="NLM_article-title">Docking based virtual screening and molecular dynamics study to identify potential monoacylglycerol lipase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">3986</span><span class="NLM_x">–</span> <span class="NLM_lpage">3996</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2014.06.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2014.06.029" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=3986-3996&author=O.+Afzalauthor=S.+Kumarauthor=R.+Kumarauthor=A.+Firozauthor=M.+Jaggiauthor=S.+Bawa&title=Docking+based+virtual+screening+and+molecular+dynamics+study+to+identify+potential+monoacylglycerol+lipase+inhibitors&doi=10.1016%2Fj.bmcl.2014.06.029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.06.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.06.029%26sid%3Dliteratum%253Aachs%26aulast%3DAfzal%26aufirst%3DO.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DKumar%26aufirst%3DR.%26aulast%3DFiroz%26aufirst%3DA.%26aulast%3DJaggi%26aufirst%3DM.%26aulast%3DBawa%26aufirst%3DS.%26atitle%3DDocking%2520based%2520virtual%2520screening%2520and%2520molecular%2520dynamics%2520study%2520to%2520identify%2520potential%2520monoacylglycerol%2520lipase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D3986%26epage%3D3996%26doi%3D10.1016%2Fj.bmcl.2014.06.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group">Vyas, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghate, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goel, A.</span><span> </span><span class="NLM_article-title">Pharmacophore modeling, virtual screening, docking and in silico ADMET analysis of protein kinase B (PKB beta) inhibitors</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">17</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2013.01.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.jmgm.2013.01.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsVykur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2013&pages=17-25&author=V.+K.+Vyasauthor=M.+Ghateauthor=A.+Goel&title=Pharmacophore+modeling%2C+virtual+screening%2C+docking+and+in+silico+ADMET+analysis+of+protein+kinase+B+%28PKB+beta%29+inhibitors&doi=10.1016%2Fj.jmgm.2013.01.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacophore modeling, virtual screening, docking and in silico ADMET analysis of protein kinase B (PKB β) inhibitors</span></div><div class="casAuthors">Vyas, Vivek K.; Ghate, Manjunath; Goel, Ashutosh</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">17-25</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Protein kinase B (PKB) is a key mediator of proliferation and survival pathways that are crit. for cancer growth.  Therefore, inhibitors of PKB are useful agents for the treatment of cancer.  Herein, we describe pharmacophore-based virtual screening combined with docking study as a rational strategy for identification of novel hits or leads.  Pharmacophore models of PKB β inhibitors were established using the DISCOtech and refined with GASP from compds. with IC50 values ranging from 2.2 to 246 nM.  The best pharmacophore model consists of one hydrogen bond acceptor (HBA), one hydrogen bond donor (HBD) site and two hydrophobic (HY) features.  The pharmacophore models were validated through receiver operating characteristic (ROC) and Guener-Henry (GH) scoring methods indicated that the model-3 was statistically valuable and reliable in identifying PKB β inhibitors.  Pharmacophore model as a 3D search query was searched against NCI database.  Several compds. with different structures (scaffolds) were retrieved as hits.  Mols. with a Qfit value of more than 95 and three other known inhibitors were docked in the active site of PKB to further explore the binding mode of these compds.  Finally in silico pharmacokinetic and toxicities were predicted for active hit mols.  The hits reported here showed good potential to be PKB β inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbLeReas0BxbVg90H21EOLACvtfcHk0ligflrrI4c8rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsVykur0%253D&md5=c1059c14a682402903c98f04f5ae65df</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2013.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2013.01.010%26sid%3Dliteratum%253Aachs%26aulast%3DVyas%26aufirst%3DV.%2BK.%26aulast%3DGhate%26aufirst%3DM.%26aulast%3DGoel%26aufirst%3DA.%26atitle%3DPharmacophore%2520modeling%252C%2520virtual%2520screening%252C%2520docking%2520and%2520in%2520silico%2520ADMET%2520analysis%2520of%2520protein%2520kinase%2520B%2520%2528PKB%2520beta%2529%2520inhibitors%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2013%26volume%3D42%26spage%3D17%26epage%3D25%26doi%3D10.1016%2Fj.jmgm.2013.01.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group">Xiao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, Y.</span><span> </span><span class="NLM_article-title">Effective virtual screening strategy focusing on the identification of novel Bruton’s tyrosine kinase inhibitors</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">142</span><span class="NLM_x">–</span> <span class="NLM_lpage">154</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2015.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.jmgm.2015.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXos1Olu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2015&pages=142-154&author=J.+Xiaoauthor=S.+Zhangauthor=M.+Luoauthor=Y.+Zouauthor=Y.+Zhangauthor=Y.+Lai&title=Effective+virtual+screening+strategy+focusing+on+the+identification+of+novel+Bruton%E2%80%99s+tyrosine+kinase+inhibitors&doi=10.1016%2Fj.jmgm.2015.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Effective virtual screening strategy focusing on the identification of novel Bruton's tyrosine kinase inhibitors</span></div><div class="casAuthors">Xiao, Jianhu; Zhang, Shengping; Luo, Minghao; Zou, Yi; Zhang, Yihua; Lai, Yisheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">142-154</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Dysregulation of the B-cell receptor (BCR) signaling pathway plays a vital role in the pathogenesis and development of B-cell malignancies.  Bruton's tyrosine kinase (BTK), a key component in the BCR signaling, has been validated as a valuable target for the treatment of B-cell malignancies.  In an attempt to find novel and potent BTK inhibitors, both ligand- and structure-based pharmacophore models were generated using Discovery Studio 2.5 and Ligandscout 3.11 with the aim of screening the ChemBridge database.  The resulting hits were then subjected to sequential docking expts. using two independent docking programs, CDOCKER and Glide.  Mols. displaying high glide scores and H-bond interactions with the key residue Met477 in both of the docking programs were retained.  Drug-like criteria including Lipinski's rule of five and ADMET properties filters were employed for further refinement of the retrieved hits.  By clustering, eight promising compds. with novel chem. scaffolds were finally selected and the top two ranking compds. were evaluated by mol. dynamics simulation.  We believe that these compds. are of great potential in BTK inhibition and will be used for further investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohhNhLGO9zSLVg90H21EOLACvtfcHk0ligflrrI4c8rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXos1Olu74%253D&md5=77035d3cf6b0bba7db7e104ce596c859</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2015.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2015.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DZou%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLai%26aufirst%3DY.%26atitle%3DEffective%2520virtual%2520screening%2520strategy%2520focusing%2520on%2520the%2520identification%2520of%2520novel%2520Bruton%25E2%2580%2599s%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2015%26volume%3D60%26spage%3D142%26epage%3D154%26doi%3D10.1016%2Fj.jmgm.2015.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group">Wilsey, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurka, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toth, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franco, J.</span><span> </span><span class="NLM_article-title">A large scale virtual screen of DprE1</span> <span class="citation_source-journal">Comput. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">121</span><span class="NLM_x">–</span> <span class="NLM_lpage">125</span><span class="refDoi"> DOI: 10.1016/j.compbiolchem.2013.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.compbiolchem.2013.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24055764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVehtrfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2013&pages=121-125&author=C.+Wilseyauthor=J.+Gurkaauthor=D.+Tothauthor=J.+Franco&title=A+large+scale+virtual+screen+of+DprE1&doi=10.1016%2Fj.compbiolchem.2013.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">A large scale virtual screen of DprE1</span></div><div class="casAuthors">Wilsey, Claire; Gurka, Jessica; Toth, David; Franco, Jimmy</div><div class="citationInfo"><span class="NLM_cas:title">Computational Biology and Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">121-125</span>CODEN:
                <span class="NLM_cas:coden">CBCOCH</span>;
        ISSN:<span class="NLM_cas:issn">1476-9271</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Tuberculosis continues to plague the world with the World Health Organization estg. that about one third of the world's population is infected.  Due to the emergence of MDR and XDR strains of TB, the need for novel therapeutics has become increasing urgent.  Herein the authors report the results of a virtual screen of 4.1 million compds. against a promising drug target, DrpE1.  The virtual compds. were obtained from the Zinc docking site and screened using the mol. docking program, AutoDock Vina.  The computational hits have led to the identification of several promising lead compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC9CN2V_DDirVg90H21EOLACvtfcHk0liQl5xQjM7-Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVehtrfF&md5=1b2a306a2e3c40a8b94c0340cdca352e</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1016%2Fj.compbiolchem.2013.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.compbiolchem.2013.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DWilsey%26aufirst%3DC.%26aulast%3DGurka%26aufirst%3DJ.%26aulast%3DToth%26aufirst%3DD.%26aulast%3DFranco%26aufirst%3DJ.%26atitle%3DA%2520large%2520scale%2520virtual%2520screen%2520of%2520DprE1%26jtitle%3DComput.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D47%26spage%3D121%26epage%3D125%26doi%3D10.1016%2Fj.compbiolchem.2013.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group">Messaoudi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belguith, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ben Hamida, J.</span><span> </span><span class="NLM_article-title">Homology modeling and virtual screening approaches to identify potent inhibitors of VEB-1 beta-lactamase</span> <span class="citation_source-journal">Theor. Biol. Med. Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">22</span><span class="refDoi"> DOI: 10.1186/1742-4682-10-22</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1186%2F1742-4682-10-22" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=23547944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvFWms70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=22&author=A.+Messaoudiauthor=H.+Belguithauthor=J.+Ben+Hamida&title=Homology+modeling+and+virtual+screening+approaches+to+identify+potent+inhibitors+of+VEB-1+beta-lactamase&doi=10.1186%2F1742-4682-10-22"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Homology modeling and virtual screening approaches to identify potent inhibitors of VEB-1 β-lactamase</span></div><div class="casAuthors">Messaoudi, Abdelmonaem; Belguith, Hatem; Ben Hamida, Jeannette</div><div class="citationInfo"><span class="NLM_cas:title">Theoretical Biology and Medical Modelling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">22</span>CODEN:
                <span class="NLM_cas:coden">TBMMA8</span>;
        ISSN:<span class="NLM_cas:issn">1742-4682</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: blaVEB-1 is an integron-located extended-spectrum β-lactamase gene initially detected in Escherichia coli and Pseudomonas aeruginosa strains from South-East Asia.  Several recent studies have reported that VEB-1-pos. strains are highly resistant to ceftazidime, cefotaxime and aztreonam antibiotics.  One strategy to overcome resistance involves administering antibiotics together with β-lactamase inhibitors during the treatment of infectious diseases.  During this study, four VEB-1 β-lactamase inhibitors were identified using computer-aided drug design.  Methods: The SWISS-MODEL tool was utilized to generate three dimensional structures of VEB-1 β-lactamase, and the 3D model VEB-1 was verified using PROCHECK, ERRAT and VERIFY 3D programs.  Virtual screening was performed by docking inhibitors obtained from the ZINC Database to the active site of the VEB-1 protein using AutoDock Vina software.  Results and Conclusion: Homol. modeling studies were performed to obtain a three-dimensional structure of VEB-1 β-lactamase.  The generated model was validated, and virtual screening of a large chem. ligand library with docking simulations was performed using AutoDock software with the ZINC database.  On the basis of the dock-score, four mols. were subjected to ADME/TOX anal., with ZINC4085364 emerging as the most potent inhibitor of the VEB-1 β-lactamase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCOH7dOHnC6rVg90H21EOLACvtfcHk0lh002nKbz_LwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvFWms70%253D&md5=4ededf14f6224c5629aa7d9ca7075631</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1186%2F1742-4682-10-22&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1742-4682-10-22%26sid%3Dliteratum%253Aachs%26aulast%3DMessaoudi%26aufirst%3DA.%26aulast%3DBelguith%26aufirst%3DH.%26aulast%3DBen%2BHamida%26aufirst%3DJ.%26atitle%3DHomology%2520modeling%2520and%2520virtual%2520screening%2520approaches%2520to%2520identify%2520potent%2520inhibitors%2520of%2520VEB-1%2520beta-lactamase%26jtitle%3DTheor.%2520Biol.%2520Med.%2520Modell.%26date%3D2013%26volume%3D10%26spage%3D22%26doi%3D10.1186%2F1742-4682-10-22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group">Nogara, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saraiva, R.d.A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caeran Bueno, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lissner, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lenz Dalla Corte, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braga, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosemberg, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocha, J. B.</span><span> </span><span class="NLM_article-title">Virtual screening of acetylcholinesterase inhibitors using the Lipinski’s rule of five and ZINC databank</span> <span class="citation_source-journal">BioMed Res. Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">2015</span><span class="NLM_x">, </span> <span class="NLM_fpage">870389</span><span class="refDoi"> DOI: 10.1155/2015/870389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1155%2F2015%2F870389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25685814" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BC2MrmsVyhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2015&publication_year=2015&pages=870389&author=P.+A.+Nogaraauthor=R.d.A.+Saraivaauthor=D.+Caeran+Buenoauthor=L.+J.+Lissnerauthor=C.+Lenz+Dalla+Corteauthor=M.+M.+Bragaauthor=D.+B.+Rosembergauthor=J.+B.+Rocha&title=Virtual+screening+of+acetylcholinesterase+inhibitors+using+the+Lipinski%E2%80%99s+rule+of+five+and+ZINC+databank&doi=10.1155%2F2015%2F870389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening of acetylcholinesterase inhibitors using the Lipinski's rule of five and ZINC databank</span></div><div class="casAuthors">Nogara Pablo Andrei; Caeran Bueno Diones; Lissner Lilian Juliana; Lenz Dalla Corte Cristiane; Braga Marcos M; Rosemberg Denis Broock; Rocha Joao Batista Teixeira; Saraiva Rogerio de Aquino</div><div class="citationInfo"><span class="NLM_cas:title">BioMed research international</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">2015</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">870389</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Alzheimer's disease (AD) is a progressive and neurodegenerative pathology that can affect people over 65 years of age.  It causes several complications, such as behavioral changes, language deficits, depression, and memory impairments.  One of the methods used to treat AD is the increase of acetylcholine (ACh) in the brain by using acetylcholinesterase inhibitors (AChEIs).  In this study, we used the ZINC databank and the Lipinski's rule of five to perform a virtual screening and a molecular docking (using Auto Dock Vina 1.1.1) aiming to select possible compounds that have quaternary ammonium atom able to inhibit acetylcholinesterase (AChE) activity.  The molecules were obtained by screening and further in vitro assays were performed to analyze the most potent inhibitors through the IC50 value and also to describe the interaction models between inhibitors and enzyme by molecular docking.  The results showed that compound D inhibited AChE activity from different vertebrate sources and butyrylcholinesterase (BChE) from Equus ferus (EfBChE), with IC50 ranging from 1.69 ± 0.46 to 5.64 ± 2.47 μM.  Compound D interacted with the peripheral anionic subsite in both enzymes, blocking substrate entrance to the active site.  In contrast, compound C had higher specificity as inhibitor of EfBChE.  In conclusion, the screening was effective in finding inhibitors of AChE and BuChE from different organisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTBBryFJmsYVNTiMd2aNQVUfW6udTcc2eYKRgbVwHFb6Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MrmsVyhsw%253D%253D&md5=6f41ff4cd81fe9d0e71edc03e9933fc1</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1155%2F2015%2F870389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2015%252F870389%26sid%3Dliteratum%253Aachs%26aulast%3DNogara%26aufirst%3DP.%2BA.%26aulast%3DSaraiva%26aufirst%3DR.d.A.%26aulast%3DCaeran%2BBueno%26aufirst%3DD.%26aulast%3DLissner%26aufirst%3DL.%2BJ.%26aulast%3DLenz%2BDalla%2BCorte%26aufirst%3DC.%26aulast%3DBraga%26aufirst%3DM.%2BM.%26aulast%3DRosemberg%26aufirst%3DD.%2BB.%26aulast%3DRocha%26aufirst%3DJ.%2BB.%26atitle%3DVirtual%2520screening%2520of%2520acetylcholinesterase%2520inhibitors%2520using%2520the%2520Lipinski%25E2%2580%2599s%2520rule%2520of%2520five%2520and%2520ZINC%2520databank%26jtitle%3DBioMed%2520Res.%2520Int.%26date%3D2015%26volume%3D2015%26spage%3D870389%26doi%3D10.1155%2F2015%2F870389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group">Regon, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gogoi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rai, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bordoloi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bezbaruah, R. L.</span><span> </span><span class="NLM_article-title">Design of inhibitors of the HIV-1 integrase core domain using virtual screening</span> <span class="citation_source-journal">Bioinformation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">76</span><span class="NLM_x">–</span> <span class="NLM_lpage">80</span><span class="refDoi"> DOI: 10.6026/97320630010076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.6026%2F97320630010076" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=76-80&author=P.+Regonauthor=D.+Gogoiauthor=A.+K.+Raiauthor=M.+Bordoloiauthor=R.+L.+Bezbaruah&title=Design+of+inhibitors+of+the+HIV-1+integrase+core+domain+using+virtual+screening&doi=10.6026%2F97320630010076"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.6026%2F97320630010076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.6026%252F97320630010076%26sid%3Dliteratum%253Aachs%26aulast%3DRegon%26aufirst%3DP.%26aulast%3DGogoi%26aufirst%3DD.%26aulast%3DRai%26aufirst%3DA.%2BK.%26aulast%3DBordoloi%26aufirst%3DM.%26aulast%3DBezbaruah%26aufirst%3DR.%2BL.%26atitle%3DDesign%2520of%2520inhibitors%2520of%2520the%2520HIV-1%2520integrase%2520core%2520domain%2520using%2520virtual%2520screening%26jtitle%3DBioinformation%26date%3D2014%26volume%3D10%26spage%3D76%26epage%3D80%26doi%3D10.6026%2F97320630010076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group">Saraswat, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nehra, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaudhary, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, C. V.</span><span> </span><span class="NLM_article-title">In-silico screening and in-vitro validation of vascular endothelial growth factor receptor-2 (VEGFR-2) inhibitors</span> <span class="citation_source-journal">Bioinformation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">273</span><span class="NLM_x">–</span> <span class="NLM_lpage">280</span><span class="refDoi"> DOI: 10.6026/97320630010273</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.6026%2F97320630010273" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=273-280&author=D.+Saraswatauthor=S.+Nehraauthor=K.+K.+Chaudharyauthor=C.+V.+Prasad&title=In-silico+screening+and+in-vitro+validation+of+vascular+endothelial+growth+factor+receptor-2+%28VEGFR-2%29+inhibitors&doi=10.6026%2F97320630010273"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.6026%2F97320630010273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.6026%252F97320630010273%26sid%3Dliteratum%253Aachs%26aulast%3DSaraswat%26aufirst%3DD.%26aulast%3DNehra%26aufirst%3DS.%26aulast%3DChaudhary%26aufirst%3DK.%2BK.%26aulast%3DPrasad%26aufirst%3DC.%2BV.%26atitle%3DIn-silico%2520screening%2520and%2520in-vitro%2520validation%2520of%2520vascular%2520endothelial%2520growth%2520factor%2520receptor-2%2520%2528VEGFR-2%2529%2520inhibitors%26jtitle%3DBioinformation%26date%3D2014%26volume%3D10%26spage%3D273%26epage%3D280%26doi%3D10.6026%2F97320630010273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group">Giri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taj, G.</span><span> </span><span class="NLM_article-title">In silico-prediction of downstream MYB interacting partners of MAPK3 in Arabidopsis</span> <span class="citation_source-journal">Bioinformation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">721</span><span class="NLM_x">–</span> <span class="NLM_lpage">725</span><span class="refDoi"> DOI: 10.6026/97320630010721</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.6026%2F97320630010721" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=721-725&author=P.+Giriauthor=A.+Kumarauthor=G.+Taj&title=In+silico-prediction+of+downstream+MYB+interacting+partners+of+MAPK3+in+Arabidopsis&doi=10.6026%2F97320630010721"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.6026%2F97320630010721&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.6026%252F97320630010721%26sid%3Dliteratum%253Aachs%26aulast%3DGiri%26aufirst%3DP.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DTaj%26aufirst%3DG.%26atitle%3DIn%2520silico-prediction%2520of%2520downstream%2520MYB%2520interacting%2520partners%2520of%2520MAPK3%2520in%2520Arabidopsis%26jtitle%3DBioinformation%26date%3D2014%26volume%3D10%26spage%3D721%26epage%3D725%26doi%3D10.6026%2F97320630010721" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group">Alhazmi, M. I.</span><span> </span><span class="NLM_article-title">Molecular docking of selected phytocompounds with H1N1 Proteins</span> <span class="citation_source-journal">Bioinformation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">196</span><span class="NLM_x">–</span> <span class="NLM_lpage">202</span><span class="refDoi"> DOI: 10.6026/97320630011196</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.6026%2F97320630011196" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=196-202&author=M.+I.+Alhazmi&title=Molecular+docking+of+selected+phytocompounds+with+H1N1+Proteins&doi=10.6026%2F97320630011196"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.6026%2F97320630011196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.6026%252F97320630011196%26sid%3Dliteratum%253Aachs%26aulast%3DAlhazmi%26aufirst%3DM.%2BI.%26atitle%3DMolecular%2520docking%2520of%2520selected%2520phytocompounds%2520with%2520H1N1%2520Proteins%26jtitle%3DBioinformation%26date%3D2015%26volume%3D11%26spage%3D196%26epage%3D202%26doi%3D10.6026%2F97320630011196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group">Shao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span> </span><span class="NLM_article-title">Dual-inhibitors of STAT5 and STAT3: studies from molecular docking and molecular dynamics simulations</span> <span class="citation_source-journal">J. Mol. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">2399</span><span class="refDoi"> DOI: 10.1007/s00894-014-2399-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs00894-014-2399-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25098340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BC2cbps1elsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=2399&author=S.+Shaoauthor=R.+Yuauthor=Y.+Yuauthor=Y.+Li&title=Dual-inhibitors+of+STAT5+and+STAT3%3A+studies+from+molecular+docking+and+molecular+dynamics+simulations&doi=10.1007%2Fs00894-014-2399-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Dual-inhibitors of STAT5 and STAT3: studies from molecular docking and molecular dynamics simulations</span></div><div class="casAuthors">Shao Shengjuan; Yu Rilei; Yu Yanqing; Li Yanni</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2399</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although molecularly targeted therapy with imatinib has improved treatments of chronic myeloid leukemia (CML), clinical resistance gradually develops in patients with accelerated or blast phase CML.  The inability of imatinib to cure CML suggests that inactivation of BCR-ABL kinase activity alone is not sufficient to control the disease.  Aberrant STAT signaling and constitutive STAT5 or STAT3 activation are frequently found in both acute and chronic leukemia.  Constitutive activation of STAT5 and STAT3 are associated with imatinib resistance on leukemia cells.  Development of drugs targeting SH2 domains of STAT5 and STAT3 provides a novel strategy for the treatment of the imatinib-resistant CML.  Here, molecular docking and molecular dynamics simulations were used to investigate the interactions of the drugs targeting STAT3 and STAT5 receptors at molecular level.  The calculated binding free energies are consistent with the ranking of the experimental affinities and our simulations also explained their differences in binding energy.  Then virtual screening based on molecular docking and molecular dynamics was applied to screen a set of ~1500 compounds for dual inhibitors of the SH2 domains of STAT5 and STAT3.  Three top score compounds obtained in virtual screening were compound 660, 304, and 561.  Results show that the three predicted dual-inhibitors are well fitted within the two binding domains and are predicted to present improved STAT5 and STAT3 SH2 inhibitory activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRDm_Hf0VLBp7P3FVUi_gf8fW6udTcc2ebf65om848U6bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cbps1elsg%253D%253D&md5=54b44aed17241d682d5302cfb79eb314</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1007%2Fs00894-014-2399-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00894-014-2399-x%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26atitle%3DDual-inhibitors%2520of%2520STAT5%2520and%2520STAT3%253A%2520studies%2520from%2520molecular%2520docking%2520and%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Mol.%2520Model.%26date%3D2014%26volume%3D20%26spage%3D2399%26doi%3D10.1007%2Fs00894-014-2399-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group">Sun, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bao, J.</span><span> </span><span class="NLM_article-title">Screening of novel inhibitors targeting lactate dehydrogenase A via four molecular docking strategies and dynamics simulations</span> <span class="citation_source-journal">J. Mol. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">133</span><span class="refDoi"> DOI: 10.1007/s00894-015-2675-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs00894-015-2675-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25934158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BC2MfgtFylsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=133&author=R.+Sunauthor=X.+Liauthor=Y.+Liauthor=X.+Zhangauthor=X.+Liauthor=X.+Liauthor=Z.+Shiauthor=J.+Bao&title=Screening+of+novel+inhibitors+targeting+lactate+dehydrogenase+A+via+four+molecular+docking+strategies+and+dynamics+simulations&doi=10.1007%2Fs00894-015-2675-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Screening of novel inhibitors targeting lactate dehydrogenase A via four molecular docking strategies and dynamics simulations</span></div><div class="casAuthors">Sun Rong; Li Xin; Li Yuanyuan; Zhang Xun; Li Xinru; Li Xiaoyu; Shi Zheng; Bao Jinku</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">133</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Lactate dehydrogenase A (LDHA) is a metabolic enzyme which catalyzes the interconversion of lactate and pyruvate in the glycolysis pathway, thus playing key roles in aerobic glycolysis.  The inhibition of LDHA by small molecules has become an attractive strategy for anticancer therapy in recent years.  However, very few LDHA inhibitors have been reported, even though a great deal of effort has directed into identifying LDHA inhibitors using structure-based approaches.  Therefore, high-throughput and high-accuracy screening approaches are still urgently needed in order to target LDHA effectively.  In the present work, after establishing that our docking strategies performed well using test datasets, we screened 32791 Specs products for their docking scores with the substrate-binding pocket and, separately, the cofactor-binding pocket of LDHA.  We subsequently identified 76 hits (i.e., ligands that show low docking scores) for the cofactor-binding pocket and 27 hits for the substrate-binding pocket.  Two representative compounds, ZINC20036549 and ZINC19369718, were then chosen for further MD simulation analysis, and we found that these compounds maintained their inhibitory activity during the MD simulations.  Meanwhile, we found that ZINC19369718 interacts with a novel binding site close to the active site, and that this interaction may inhibit the catalytic activity of LDHA.  Together, these results offer not only a new paradigm for identifying Specs drug-like products for novel therapeutic use but they also provide further opportunity to adopt LDHA inhibition as a strategy for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcROAp0PJSifc_IW7hznt0zpfW6udTcc2ebf65om848U6bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MfgtFylsQ%253D%253D&md5=9dc35b7875893ee56d95090473065512</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1007%2Fs00894-015-2675-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00894-015-2675-4%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DBao%26aufirst%3DJ.%26atitle%3DScreening%2520of%2520novel%2520inhibitors%2520targeting%2520lactate%2520dehydrogenase%2520A%2520via%2520four%2520molecular%2520docking%2520strategies%2520and%2520dynamics%2520simulations%26jtitle%3DJ.%2520Mol.%2520Model.%26date%3D2015%26volume%3D21%26spage%3D133%26doi%3D10.1007%2Fs00894-015-2675-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group">Sgrignani, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novati, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colombo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grazioso, G.</span><span> </span><span class="NLM_article-title">Covalent docking of selected boron-based serine beta-lactamase inhibitors</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">441</span><span class="NLM_x">–</span> <span class="NLM_lpage">450</span><span class="refDoi"> DOI: 10.1007/s10822-015-9834-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs10822-015-9834-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=441-450&author=J.+Sgrignaniauthor=B.+Novatiauthor=G.+Colomboauthor=G.+Grazioso&title=Covalent+docking+of+selected+boron-based+serine+beta-lactamase+inhibitors&doi=10.1007%2Fs10822-015-9834-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9834-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9834-7%26sid%3Dliteratum%253Aachs%26aulast%3DSgrignani%26aufirst%3DJ.%26aulast%3DNovati%26aufirst%3DB.%26aulast%3DColombo%26aufirst%3DG.%26aulast%3DGrazioso%26aufirst%3DG.%26atitle%3DCovalent%2520docking%2520of%2520selected%2520boron-based%2520serine%2520beta-lactamase%2520inhibitors%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D441%26epage%3D450%26doi%3D10.1007%2Fs10822-015-9834-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group">Ogrizek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lesnik, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sosic, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodoscek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirkovic, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kos, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janezic, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gobec, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konc, J.</span><span> </span><span class="NLM_article-title">Molecular dynamics to enhance structure-based virtual screening on cathepsin B</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">707</span><span class="NLM_x">–</span> <span class="NLM_lpage">712</span><span class="refDoi"> DOI: 10.1007/s10822-015-9847-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs10822-015-9847-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25947277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnvFChsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=707-712&author=M.+Ogrizekauthor=S.+Turkauthor=S.+Lesnikauthor=I.+Sosicauthor=M.+Hodoscekauthor=B.+Mirkovicauthor=J.+Kosauthor=D.+Janezicauthor=S.+Gobecauthor=J.+Konc&title=Molecular+dynamics+to+enhance+structure-based+virtual+screening+on+cathepsin+B&doi=10.1007%2Fs10822-015-9847-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics to enhance structure-based virtual screening on cathepsin B</span></div><div class="casAuthors">Ogrizek, Mitja; Turk, Samo; Lesnik, Samo; Sosic, Izidor; Hodoscek, Milan; Mirkovic, Bojana; Kos, Janko; Janezic, Dusanka; Gobec, Stanislav; Konc, Janez</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">707-712</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Mol. dynamics (MD) and mol. docking are commonly used to study mol. interactions in drug discovery.  Most docking approaches consider proteins as rigid, which can decrease the accuracy of predicted docked poses.  Therefore MD simulations can be used prior to docking to add flexibility to proteins.  We evaluated the contribution of using MD together with docking in a docking study on human cathepsin B, a well-studied protein involved in numerous pathol. processes.  Using CHARMM biomol. simulation program and AutoDock Vina mol. docking program, we found, that short MD simulations significantly improved mol. docking.  Our results, expressed with the area under the receiver operating characteristic curves, show an increase in discriminatory power i.e. the ability to discriminate active from inactive compds. of mol. docking, when docking is performed to selected snapshots from MD simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfD2hUeaLu7rVg90H21EOLACvtfcHk0ljBVV_0Q5Zn8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnvFChsb8%253D&md5=140a6ee2d68d50a93f78291a30ed0c4d</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9847-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9847-2%26sid%3Dliteratum%253Aachs%26aulast%3DOgrizek%26aufirst%3DM.%26aulast%3DTurk%26aufirst%3DS.%26aulast%3DLesnik%26aufirst%3DS.%26aulast%3DSosic%26aufirst%3DI.%26aulast%3DHodoscek%26aufirst%3DM.%26aulast%3DMirkovic%26aufirst%3DB.%26aulast%3DKos%26aufirst%3DJ.%26aulast%3DJanezic%26aufirst%3DD.%26aulast%3DGobec%26aufirst%3DS.%26aulast%3DKonc%26aufirst%3DJ.%26atitle%3DMolecular%2520dynamics%2520to%2520enhance%2520structure-based%2520virtual%2520screening%2520on%2520cathepsin%2520B%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D707%26epage%3D712%26doi%3D10.1007%2Fs10822-015-9847-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group">Singh, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pandey, P. N.</span><span> </span><span class="NLM_article-title">Molecular docking and molecular dynamics study on SmHDAC1 to identify potential lead compounds against Schistosomiasis</span> <span class="citation_source-journal">Mol. Biol. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">689</span><span class="NLM_x">–</span> <span class="NLM_lpage">698</span><span class="refDoi"> DOI: 10.1007/s11033-014-3816-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs11033-014-3816-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25663090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitlaqtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2015&pages=689-698&author=R.+Singhauthor=P.+N.+Pandey&title=Molecular+docking+and+molecular+dynamics+study+on+SmHDAC1+to+identify+potential+lead+compounds+against+Schistosomiasis&doi=10.1007%2Fs11033-014-3816-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular docking and molecular dynamics study on SmHDAC1 to identify potential lead compounds against Schistosomiasis</span></div><div class="casAuthors">Singh, Raghvendra; Pandey, Paras Nath</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">689-698</span>CODEN:
                <span class="NLM_cas:coden">MLBRBU</span>;
        ISSN:<span class="NLM_cas:issn">0301-4851</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Schistosomiasis, a disease caused by helminth parasites of genus Schistosoma.  Its treatment intensively depends on single drug, praziquantel which increases the risk of development of drug-resistant parasite.  Inhibitors of human HDAC are profoundly reported as novel anti-cancer drugs and used as new anit-parasitic agents.  Schistosomamonsoni class I HDACs are expressed in all stages of life cycle and indicating that this enzyme is most likely a major target for the designing specific inhibitors.  In order to find novel target for the treatment of Schistosomiasis, three dimensional structure of SmHDAC1 was generated, using homol. modeling.  Features of the generated structure, was then deduced with respect to conformation of peptide backbone, local compatibility of the generated structure in terms of energy and mol. dynamics study.  Considering these features of the generated structure, we selected all the class 1 inhibitors reported so far, which showed interactions with HDACs.  Virtual screening was done using reported inhibitors (70) and using SmHDAC1 and HsHDAC1 as the targets.  On the basis of binding affinity and IC50 value, 24th ligand was selected for the mol. docking purpose.  In this study, out of all the reported inhibitors, 24th inhibitor (N,8-dihydroxy-8-(naphthalen-2-yl) octanamide zinc id- ZINC13474421) showed better binding with SmHDAC1 (-8.1 kcal/mol) as compared to HsHDAC1 (-6.4 kcal/mol) in terms of binding energy and supported by IC50 value.  This paper throws light on the reliable model for further structure based drug designing, concerning SmHDAC1 of S. mansoni.  Mol. docking studies highlighted advantages of comparative in silico interaction studies of SmHDAC1 and HsHDAC1.  N,8-dihydroxy-8-(naphthalen-2-yl) octanamide can further use for the clin. trial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqumvOBY0VSiLVg90H21EOLACvtfcHk0ljCRHseKtt4yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitlaqtLc%253D&md5=eed4a7eaf7bfbb7ee97fe3599fc56c99</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1007%2Fs11033-014-3816-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11033-014-3816-z%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DR.%26aulast%3DPandey%26aufirst%3DP.%2BN.%26atitle%3DMolecular%2520docking%2520and%2520molecular%2520dynamics%2520study%2520on%2520SmHDAC1%2520to%2520identify%2520potential%2520lead%2520compounds%2520against%2520Schistosomiasis%26jtitle%3DMol.%2520Biol.%2520Rep.%26date%3D2015%26volume%3D42%26spage%3D689%26epage%3D698%26doi%3D10.1007%2Fs11033-014-3816-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group">Pinheiro, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duarte, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alves, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Molfetta, F. A.</span><span> </span><span class="NLM_article-title">Virtual screening and molecular dynamics simulations from a bank of molecules of the Amazon region against functional NS3-4A protease-helicase enzyme of hepatitis C virus</span> <span class="citation_source-journal">Appl. Biochem. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">176</span><span class="NLM_x">, </span> <span class="NLM_fpage">1709</span><span class="NLM_x">–</span> <span class="NLM_lpage">1721</span><span class="refDoi"> DOI: 10.1007/s12010-015-1672-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs12010-015-1672-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26009474" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovFOgsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=176&publication_year=2015&pages=1709-1721&author=A.+S.+Pinheiroauthor=J.+B.+Duarteauthor=C.+N.+Alvesauthor=F.+A.+de+Molfetta&title=Virtual+screening+and+molecular+dynamics+simulations+from+a+bank+of+molecules+of+the+Amazon+region+against+functional+NS3-4A+protease-helicase+enzyme+of+hepatitis+C+virus&doi=10.1007%2Fs12010-015-1672-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Screening and Molecular Dynamics Simulations from a Bank of Molecules of the Amazon Region Against Functional NS3-4A Protease-Helicase Enzyme of Hepatitis C Virus</span></div><div class="casAuthors">Pinheiro, Alan Sena; Duarte, Jaqueline Bianca Carvalho; Alves, Claudio Nahum; Alberto de Molfetta, Fabio</div><div class="citationInfo"><span class="NLM_cas:title">Applied Biochemistry and Biotechnology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">176</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1709-1721</span>CODEN:
                <span class="NLM_cas:coden">ABIBDL</span>;
        ISSN:<span class="NLM_cas:issn">0273-2289</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Hepatitis C virus (HCV) infection is a disease that affects approx. 3 % of the global population and requires new therapeutic agents without the inconvenience assocd. with current anti-HCV treatment.  This paper reports on a study of a virtual screening and a mol. dynamics simulation of compds. derived from natural products from the Amazon region that are potentially effective against the NS3-4A enzyme of HCV, which plays an important role in the replication process of this virus.  According to the results of the mol. docking calcns. and subsequent consensual anal., the best scored compds. showed interactions between hydrogen and residues of the catalytic triad as well as interactions with residues that guide ligands to the active site of the enzyme.  They also showed stability in the mol. dynamics simulation, as the structures preserved important interactions at the active site of the enzyme.  The root mean square deviation (RMSD) values were stabilized at the end of the simulation time.  Such compds. are considered promising as novel therapies against HCV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwbaCUHstharVg90H21EOLACvtfcHk0ljCRHseKtt4yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovFOgsLw%253D&md5=6fc53bd1f1c8ec97bc98d4855c15a508</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1007%2Fs12010-015-1672-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12010-015-1672-5%26sid%3Dliteratum%253Aachs%26aulast%3DPinheiro%26aufirst%3DA.%2BS.%26aulast%3DDuarte%26aufirst%3DJ.%2BB.%26aulast%3DAlves%26aufirst%3DC.%2BN.%26aulast%3Dde%2BMolfetta%26aufirst%3DF.%2BA.%26atitle%3DVirtual%2520screening%2520and%2520molecular%2520dynamics%2520simulations%2520from%2520a%2520bank%2520of%2520molecules%2520of%2520the%2520Amazon%2520region%2520against%2520functional%2520NS3-4A%2520protease-helicase%2520enzyme%2520of%2520hepatitis%2520C%2520virus%26jtitle%3DAppl.%2520Biochem.%2520Biotechnol.%26date%3D2015%26volume%3D176%26spage%3D1709%26epage%3D1721%26doi%3D10.1007%2Fs12010-015-1672-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group">Wang, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xue, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, F.</span><span> </span><span class="NLM_article-title">Identification of dual active agents targeting 5-HT1A and SERT by combinatorial virtual screening methods</span> <span class="citation_source-journal">Bio-Med. Mater. Eng.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">2233</span><span class="NLM_x">–</span> <span class="NLM_lpage">2239</span><span class="refDoi"> DOI: 10.3233/BME-151529</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.3233%2FBME-151529" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlyjsrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2015&pages=2233-2239&issue=Suppl.+1&author=P.+Wangauthor=F.+Yangauthor=H.+Yangauthor=X.+Xuauthor=D.+Liuauthor=W.+Xueauthor=F.+Zhu&title=Identification+of+dual+active+agents+targeting+5-HT1A+and+SERT+by+combinatorial+virtual+screening+methods&doi=10.3233%2FBME-151529"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of dual active agents targeting 5-HT1A and SERT by combinatorial virtual screening methods</span></div><div class="casAuthors">Liu, Feng; Lee, Dong-Hoon; Lagoa, Ricardo; Kumar, Sandeep; Wang, Panpan; Yang, Fengyuan; Yang, Hong; Xu, Xiaofei; Liu, Duo; Xue, Weiwei; Zhu, Feng</div><div class="citationInfo"><span class="NLM_cas:title">Bio-Medical Materials and Engineering</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">s1</span>),
    <span class="NLM_cas:pages">S2233-S2239</span>CODEN:
                <span class="NLM_cas:coden">BMENEO</span>;
        ISSN:<span class="NLM_cas:issn">0959-2989</span>.
    
            (<span class="NLM_cas:orgname">IOS Press</span>)
        </div><div class="casAbstract">Selective serotonin reuptake inhibitors (SSRIs) are most adopted therapeutics marketed for major depression, and the efficacy of which are greatly reduced by their delayed onset of action and undesirable side effects. 5-HT1A receptor partial agonist and SERT inhibitor (SPARI) was proposed as a novel strategy to overcome the shortage of efficacy by a neg. feedback control of 5-HT1A receptor.  However, only one SPARI (vilazodone) has been approved for clin. use, and none is currently in clin. trial, which demonstrates a strong need for searching more novel SPARIs to facilitate antidepressants discovery.  This work applied a combinatorial virtual screening method (CVSM) by integrating multiple tools.  Statistic anal. reveals that CVSM surpasses single virtual screening methods in terms of hit rates and enrichment factors.  By adopting optimized CVSM, 91 promising dual target leads form 15 scaffolds were identified, and 40% of these scaffolds have already been reported to show antidepressant related therapeutic effects.  In sum, CVSM is capable in identifying novel SPARIs from large chem. libraries with extremely low false hit rate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr073ag0IyfrLVg90H21EOLACvtfcHk0ljCRHseKtt4yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlyjsrjL&md5=0910120e940bbb1043a920b58be5798b</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.3233%2FBME-151529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3233%252FBME-151529%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DXue%26aufirst%3DW.%26aulast%3DZhu%26aufirst%3DF.%26atitle%3DIdentification%2520of%2520dual%2520active%2520agents%2520targeting%25205-HT1A%2520and%2520SERT%2520by%2520combinatorial%2520virtual%2520screening%2520methods%26jtitle%3DBio-Med.%2520Mater.%2520Eng.%26date%3D2015%26volume%3D26%26issue%3DSuppl.%25201%26spage%3D2233%26epage%3D2239%26doi%3D10.3233%2FBME-151529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group">Dhanjal, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sreenidhi, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bafna, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katiyar, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goyal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grover, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundar, D.</span><span> </span><span class="NLM_article-title">Computational structure-based de novo design of hypothetical inhibitors against the anti- inflammatory target COX-2</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0134691</span><span class="refDoi"> DOI: 10.1371/journal.pone.0134691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pone.0134691" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0134691&author=J.+K.+Dhanjalauthor=A.+K.+Sreenidhiauthor=K.+Bafnaauthor=S.+P.+Katiyarauthor=S.+Goyalauthor=A.+Groverauthor=D.+Sundar&title=Computational+structure-based+de+novo+design+of+hypothetical+inhibitors+against+the+anti-+inflammatory+target+COX-2&doi=10.1371%2Fjournal.pone.0134691"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0134691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0134691%26sid%3Dliteratum%253Aachs%26aulast%3DDhanjal%26aufirst%3DJ.%2BK.%26aulast%3DSreenidhi%26aufirst%3DA.%2BK.%26aulast%3DBafna%26aufirst%3DK.%26aulast%3DKatiyar%26aufirst%3DS.%2BP.%26aulast%3DGoyal%26aufirst%3DS.%26aulast%3DGrover%26aufirst%3DA.%26aulast%3DSundar%26aufirst%3DD.%26atitle%3DComputational%2520structure-based%2520de%2520novo%2520design%2520of%2520hypothetical%2520inhibitors%2520against%2520the%2520anti-%2520inflammatory%2520target%2520COX-2%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0134691%26doi%3D10.1371%2Fjournal.pone.0134691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group">He, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, G.</span><span> </span><span class="NLM_article-title">Discovery of potential drugs for human-infecting H7N9 virus containing R294K mutation</span> <span class="citation_source-journal">Drug Des., Dev. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2377</span><span class="NLM_x">–</span> <span class="NLM_lpage">2390</span><span class="refDoi"> DOI: 10.2147/DDDT.S74061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.2147%2FDDDT.S74061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25489236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFGhsb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=2377-2390&author=J.+Y.+Heauthor=C.+Liauthor=G.+Wu&title=Discovery+of+potential+drugs+for+human-infecting+H7N9+virus+containing+R294K+mutation&doi=10.2147%2FDDDT.S74061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of potential drugs for human-infecting H7N9 virus containing R294K mutation</span></div><div class="casAuthors">He, Jiao-Yu; Li, Cheng; Wu, Guo</div><div class="citationInfo"><span class="NLM_cas:title">Drug Design, Development and Therapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2377-2390</span>CODEN:
                <span class="NLM_cas:coden">DDDTAQ</span>;
        ISSN:<span class="NLM_cas:issn">1177-8881</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">Background: After the first epidemic wave from Feb. through May 2013, the influenza A (H7N9) virus emerged and has followed a second epidemic wave since June 2013.  As of June 27, 2014, the outbreak of H7N9 had caused 450 confirmed cases of human infection, with 165 deaths included.  The case-fatality rate of all confirmed cases is about 36%, making the H7N9 virus a significant threat to people's health.  At present, neuraminidase inhibitors are the only licensed antiviral medications available to treat H7N9 infections in humans.  Oseltamivir is the most commonly used inhibitor, and it is also a front-line drug for the threatening H7N9.  Unfortunately, it has been reported that patients treated with oseltamivir can induce R294K (Arg294Lys) substitution in the H7N9 virus, which is a rare mutation and can reduce the antiviral efficacy of inhibitors.  Even worse, deaths caused by such mutation after oseltamivir treatment have already been reported, indicating that the need to find substitutive neuraminidase inhibitors for currently available drugs to treat drug-resistant H7N9 is really pressing.  Materials and methods: First, the structure of H7N9 contg. the R294K substitution was downloaded from the Protein Data Bank, and structural information of approved drugs was downloaded from the ZINC (ZINC Is Not Com.) database.  Taking oseltamivir carboxylate as a ref. drug, we then filtered these mols. through virtual screening to find out potential inhibitors targeting the mutated H7N9 virus.  For further evaluation, we carried out a 14 ns mol. dynamic simulation for each H7N9-drug complex and calcd. the binding energy for each candidate drug.  Results: We found five inhibitors that could be candidate drugs for treating the mutated H7N9 virus.  Docking poses showed these drugs could bind to the virus effectively, with the contribution of hydrogen bonds and hydrophobic interactions.  With regard to the mol. dynamic simulations, receptor-ligand complexes formed by these candidate drugs were more stable than the one formed by oseltamivir carboxylate.  The binding energy of oseltamivir carboxylate was -122.4 kJ/mol, while those for these potential inhibitors were -417.5, -404.7, -372.2, -304.3, and -289.9 kJ/mol, much better than the ref. drug.  Conclusion: Given the current and future threat of the mutated H7N9 virus, it is urgent that potent drugs and effective antiviral therapeutics be found.  Our study therefore is able to complement currently available drugs for influenza A infectors and helps to prevent the ongoing threat of H7N9 virus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFU5IR7X5QELVg90H21EOLACvtfcHk0liUFlKiwGiqww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFGhsb%252FF&md5=d253ca7cd12d234b39bec9f4d2cc3655</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.2147%2FDDDT.S74061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FDDDT.S74061%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DJ.%2BY.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520potential%2520drugs%2520for%2520human-infecting%2520H7N9%2520virus%2520containing%2520R294K%2520mutation%26jtitle%3DDrug%2520Des.%252C%2520Dev.%2520Ther.%26date%3D2014%26volume%3D8%26spage%3D2377%26epage%3D2390%26doi%3D10.2147%2FDDDT.S74061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group">Balakrishnan, N.; Raj, J. S.; Kandakatla, N.</span><span> </span><span class="NLM_article-title">Discovery of novel GSK-3beta inhibitors using pharmacophore and virtual screening studies</span>.  <span class="citation_source-journal">Interdiscip. Sci.: Comput. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span>DOI: <span class="refDoi"> DOI: 10.1007/s12539-015-0100-4</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs12539-015-0100-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=N.+Balakrishnan&author=J.+S.+Raj&author=N.+Kandakatla&title=Discovery+of+novel+GSK-3beta+inhibitors+using+pharmacophore+and+virtual+screening+studies&doi=10.1007%2Fs12539-015-0100-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1007%2Fs12539-015-0100-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12539-015-0100-4%26sid%3Dliteratum%253Aachs%26aulast%3DBalakrishnan%26aufirst%3DN.%26atitle%3DDiscovery%2520of%2520novel%2520GSK-3beta%2520inhibitors%2520using%2520pharmacophore%2520and%2520virtual%2520screening%2520studies%26jtitle%3DInterdiscip.%2520Sci.%253A%2520Comput.%2520Life%2520Sci.%26date%3D2015%26doi%3D10.1007%2Fs12539-015-0100-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group">Thai, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngo, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, T. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, N. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le, M. T.</span><span> </span><span class="NLM_article-title">Virtual screening for novel Staphylococcus Aureus NorA efflux pump inhibitors from natural products</span> <span class="citation_source-journal">Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">135</span><span class="NLM_x">–</span> <span class="NLM_lpage">155</span><span class="refDoi"> DOI: 10.2174/1573406410666140902110903</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.2174%2F1573406410666140902110903" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25181985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXislemtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=135-155&author=K.+M.+Thaiauthor=T.+D.+Ngoauthor=T.+V.+Phanauthor=T.+D.+Tranauthor=N.+V.+Nguyenauthor=T.+H.+Nguyenauthor=M.+T.+Le&title=Virtual+screening+for+novel+Staphylococcus+Aureus+NorA+efflux+pump+inhibitors+from+natural+products&doi=10.2174%2F1573406410666140902110903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Screening for Novel Staphylococcus Aureus NorA Efflux Pump Inhibitors From Natural Products</span></div><div class="casAuthors">Thai, Khac-Minh; Ngo, Trieu-Du; Phan, Thien-Vy; Tran, Thanh-Dao; Nguyen, Ngoc-Vinh; Nguyen, Thien-Hai; Le, Minh-Tri</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">135-155</span>CODEN:
                <span class="NLM_cas:coden">MCEHAJ</span>;
        ISSN:<span class="NLM_cas:issn">1573-4064</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">NorA is a member of the Major Facilitator Superfamily (MFS) drug efflux pumps that have been shown to mediate antibiotic resistance in Staphylococcus aureus (SA).  In this study, QSAR anal., virtual screening and mol. docking were implemented in an effort to discover novel SA NorA efflux pump inhibitors.  Originally, a set of 47 structurally diverse compds. compiled from the literature was used to develop linear QSAR models and another set of 15 different compds. were chosen for extra validation.  The final model which was estd. by statistical values for the full data set (n = 45, Q2 = 0.80, RMSE = 0.20) and for the external test set (n = 15, R2 = 0.60, |res|max = 0.75, |res|min = 0.02) was applied on the collection of 182 flavonoides and the traditional Chinese medicine (TCM) database to screen for novel NorA inhibitors.  Finally, 33 lead compds. that met the Lipinski's rules of five/three and had good predicted pIC50 values from in silico screening process were employed to analyze the binding ability by docking studies on NorA homol. model in place of its unavailable crystal structures at two active sites, the central channel and the Walker B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHpK-LeL0byrVg90H21EOLACvtfcHk0liUFlKiwGiqww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXislemtbs%253D&md5=e86c28e478370ed34ade2576080e5e18</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.2174%2F1573406410666140902110903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1573406410666140902110903%26sid%3Dliteratum%253Aachs%26aulast%3DThai%26aufirst%3DK.%2BM.%26aulast%3DNgo%26aufirst%3DT.%2BD.%26aulast%3DPhan%26aufirst%3DT.%2BV.%26aulast%3DTran%26aufirst%3DT.%2BD.%26aulast%3DNguyen%26aufirst%3DN.%2BV.%26aulast%3DNguyen%26aufirst%3DT.%2BH.%26aulast%3DLe%26aufirst%3DM.%2BT.%26atitle%3DVirtual%2520screening%2520for%2520novel%2520Staphylococcus%2520Aureus%2520NorA%2520efflux%2520pump%2520inhibitors%2520from%2520natural%2520products%26jtitle%3DMed.%2520Chem.%26date%3D2015%26volume%3D11%26spage%3D135%26epage%3D155%26doi%3D10.2174%2F1573406410666140902110903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span><span class="NLM_contrib-group">Tran, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le, L.</span><span> </span><span class="NLM_article-title">Identification of novel compounds against an R294K substitution of influenza A (H7N9) virus using ensemble based drug virtual screening</span> <span class="citation_source-journal">Int. J. Med. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">163</span><span class="NLM_x">–</span> <span class="NLM_lpage">176</span><span class="refDoi"> DOI: 10.7150/ijms.10826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.7150%2Fijms.10826" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25589893" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BC2Mvms1ejsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=163-176&author=N.+Tranauthor=T.+Vanauthor=H.+Nguyenauthor=L.+Le&title=Identification+of+novel+compounds+against+an+R294K+substitution+of+influenza+A+%28H7N9%29+virus+using+ensemble+based+drug+virtual+screening&doi=10.7150%2Fijms.10826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel compounds against an R294K substitution of influenza A (H7N9) virus using ensemble based drug virtual screening</span></div><div class="casAuthors">Tran Nhut; Van Thanh; Le Ly; Nguyen Hieu</div><div class="citationInfo"><span class="NLM_cas:title">International journal of medical sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">163-76</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Influenza virus H7N9 foremost emerged in China in 2013 and killed hundreds of people in Asia since they possessed all mutations that enable them to resist to all existing influenza drugs, resulting in high mortality to human.  In the effort to identify novel inhibitors combat resistant strains of influenza virus H7N9; we performed virtual screening targeting the Neuraminidase (NA) protein against natural compounds of traditional Chinese medicine database (TCM) and ZINC natural products.  Compounds expressed high binding affinity to the target protein was then evaluated for molecular properties to determine drug-like molecules. 4 compounds showed their binding energy less than -11 Kcal/mol were selected for molecular dynamics (MD) simulation to capture intermolecular interactions of ligand-protein complexes.  The molecular mechanics/Poisson-Boltzmann surface area (MM/PBSA) method was utilized to estimate binding free energy of the complex.  In term of stability, NA-7181 (IUPAC namely {9-Hydroxy-10-[3-(trifluoromrthyl) cyclohexyl]-4.8-diazatricyclo [6.4.0.02,6]dodec-4-yl}(perhydro-1H-inden-5-yl)formaldehyde) achieved stable conformation after 20 ns and 27 ns for ligand and protein root mean square deviation, respectively.  In term of binding free energy, 7181 gave the negative value of -30.031 (KJ/mol) indicating the compound obtained a favourable state in the active site of the protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRbjj8ucg3x6q91pGlMMoT5fW6udTcc2ebGRkjCeP2aYLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mvms1ejsw%253D%253D&md5=ac04691ad0fdab05e75eab9a81f6c3ce</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.7150%2Fijms.10826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fijms.10826%26sid%3Dliteratum%253Aachs%26aulast%3DTran%26aufirst%3DN.%26aulast%3DVan%26aufirst%3DT.%26aulast%3DNguyen%26aufirst%3DH.%26aulast%3DLe%26aufirst%3DL.%26atitle%3DIdentification%2520of%2520novel%2520compounds%2520against%2520an%2520R294K%2520substitution%2520of%2520influenza%2520A%2520%2528H7N9%2529%2520virus%2520using%2520ensemble%2520based%2520drug%2520virtual%2520screening%26jtitle%3DInt.%2520J.%2520Med.%2520Sci.%26date%3D2015%26volume%3D12%26spage%3D163%26epage%3D176%26doi%3D10.7150%2Fijms.10826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref108"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref108'); return false;" data-citation="" class="refNumLink">108</a></strong><div class="NLM_citation" id="cit108"><span><span class="NLM_contrib-group">Gupta, C. L.; Akhtar, S.; Kumar, N.; Ali, J.; Pathak, N.; Bajpai, P.</span><span> </span><span class="NLM_article-title">In silico elucidation and inhibition studies of selected phytoligands against mitogen-activated protein kinases of protozoan parasites</span>.  <span class="citation_source-journal">Interdiscip. Sci.: Comput. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span>DOI: <span class="refDoi"> DOI: 10.1007/s12539-015-0269-6</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs12539-015-0269-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=C.+L.+Gupta&author=S.+Akhtar&author=N.+Kumar&author=J.+Ali&author=N.+Pathak&author=P.+Bajpai&title=In+silico+elucidation+and+inhibition+studies+of+selected+phytoligands+against+mitogen-activated+protein+kinases+of+protozoan+parasites&doi=10.1007%2Fs12539-015-0269-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1007%2Fs12539-015-0269-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12539-015-0269-6%26sid%3Dliteratum%253Aachs%26aulast%3DGupta%26aufirst%3DC.%2BL.%26atitle%3DIn%2520silico%2520elucidation%2520and%2520inhibition%2520studies%2520of%2520selected%2520phytoligands%2520against%2520mitogen-activated%2520protein%2520kinases%2520of%2520protozoan%2520parasites%26jtitle%3DInterdiscip.%2520Sci.%253A%2520Comput.%2520Life%2520Sci.%26date%3D2015%26doi%3D10.1007%2Fs12539-015-0269-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref109"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref109'); return false;" data-citation="" class="refNumLink">109</a></strong><div class="NLM_citation" id="cit109"><span><span class="NLM_contrib-group">Mitra, J.; Narad, P.; Sengupta, A.; Sharma, P. D.; Paul, P. K.</span> In <span> </span><span class="NLM_article-title">silico identification of ergosterol as a novel fungal metabolite enhancing RuBisCO activity in Lycopersicum esculentum</span>.  <span class="citation_source-journal">Interdiscip. Sci.: Comput. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span>DOI: <span class="refDoi"> DOI: 10.1007/s12539-015-0105-z</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs12539-015-0105-z" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=J.+Mitra&author=P.+Narad&author=A.+Sengupta&author=P.+D.+Sharma&author=P.+K.+Paul&title=silico+identification+of+ergosterol+as+a+novel+fungal+metabolite+enhancing+RuBisCO+activity+in+Lycopersicum+esculentum&doi=10.1007%2Fs12539-015-0105-z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1007%2Fs12539-015-0105-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12539-015-0105-z%26sid%3Dliteratum%253Aachs%26aulast%3DMitra%26aufirst%3DJ.%26atitle%3Dsilico%2520identification%2520of%2520ergosterol%2520as%2520a%2520novel%2520fungal%2520metabolite%2520enhancing%2520RuBisCO%2520activity%2520in%2520Lycopersicum%2520esculentum%26jtitle%3DInterdiscip.%2520Sci.%253A%2520Comput.%2520Life%2520Sci.%26date%3D2015%26doi%3D10.1007%2Fs12539-015-0105-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref110"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref110'); return false;" data-citation="" class="refNumLink">110</a></strong><div class="NLM_citation" id="cit110"><span><span class="NLM_contrib-group">Sbongile, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soliman, M. E.</span><span> </span><span class="NLM_article-title">In silico identification of irreversible cathepsin B inhibitors as anti- cancer agents: virtual screening, covalent docking analysis and molecular dynamics simulations</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">399</span><span class="NLM_x">–</span> <span class="NLM_lpage">410</span><span class="refDoi"> DOI: 10.2174/1386207318666150305154621</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.2174%2F1386207318666150305154621" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25747438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVGmtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2015&pages=399-410&author=M.+Sbongileauthor=M.+E.+Soliman&title=In+silico+identification+of+irreversible+cathepsin+B+inhibitors+as+anti-+cancer+agents%3A+virtual+screening%2C+covalent+docking+analysis+and+molecular+dynamics+simulations&doi=10.2174%2F1386207318666150305154621"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">In Silico Identification of Irreversible Cathepsin B Inhibitors as Anti- Cancer Agents: Virtual Screening, Covalent Docking Analysis and Molecular Dynamics Simulations</span></div><div class="casAuthors">Sbongile, Mbatha; Soliman, Mahmoud E. S.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry & High Throughput Screening</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">399-410</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Cathepsin B is a cysteine protease that belongs to the papain superfamily.  Malfunctions related to cathepsin B can lead to inflammation and cancer.  Via an integrated in silico approach, this study is aimed to identify novel Michael acceptors-type compds. that can irreversibly inhibit cathepsin B enzyme via covalent bond formation with the active site cysteine residue.  Here, we report the first account of covalent docking approach incorporated into a hybrid ligand/structure-based virtual screening to est. the binding affinities of various compds. from chem. databases against the cathepsin B protein.  For validation, compds. with exptl. detd. anti-cathepsin B activity from PubChem bioassay database were also screened and covalently docked to the enzyme target.  Interestingly, four novel compds. exhibited better covalent binding affinity when compared against the exptl. detd. prototypes.  Mol. dynamics simulations were performed to ensure the stability of the docked complexes and to allow further anal. on the MD av. structures.  Perresidue interaction decompn. anal. was carried out to provide deeper insight into the interaction themes of discovered hits with the active site residues.  It is found that polar and hydrophobic interactions contributed the most towards drug binding.  The hybrid computational methods applied in this study should serve as a powerful tool in the drug design and development process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGZtQeMsmhn7Vg90H21EOLACvtfcHk0lgfazgYU4kntQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVGmtrk%253D&md5=ac91293e5e902541f9f8b70eca39deab</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.2174%2F1386207318666150305154621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1386207318666150305154621%26sid%3Dliteratum%253Aachs%26aulast%3DSbongile%26aufirst%3DM.%26aulast%3DSoliman%26aufirst%3DM.%2BE.%26atitle%3DIn%2520silico%2520identification%2520of%2520irreversible%2520cathepsin%2520B%2520inhibitors%2520as%2520anti-%2520cancer%2520agents%253A%2520virtual%2520screening%252C%2520covalent%2520docking%2520analysis%2520and%2520molecular%2520dynamics%2520simulations%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2015%26volume%3D18%26spage%3D399%26epage%3D410%26doi%3D10.2174%2F1386207318666150305154621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref111"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref111'); return false;" data-citation="" class="refNumLink">111</a></strong><div class="NLM_citation" id="cit111"><span><span class="NLM_contrib-group">Batool, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khan, Z. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamal, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mushtaq, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamal, M. A.</span><span> </span><span class="NLM_article-title">In silico screening for identification of novel anti-malarial inhibitors by molecular docking, pharmacophore modeling and virtual screening</span> <span class="citation_source-journal">Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">687</span><span class="NLM_x">–</span> <span class="NLM_lpage">700</span><span class="refDoi"> DOI: 10.2174/1573406411666150305113533</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.2174%2F1573406411666150305113533" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25741881" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2rsrjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=687-700&author=S.+Batoolauthor=Z.+A.+Khanauthor=W.+Kamalauthor=G.+Mushtaqauthor=M.+A.+Kamal&title=In+silico+screening+for+identification+of+novel+anti-malarial+inhibitors+by+molecular+docking%2C+pharmacophore+modeling+and+virtual+screening&doi=10.2174%2F1573406411666150305113533"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">In silico Screening for Identification of Novel Anti-malarial Inhibitors by Molecular Docking, Pharmacophore Modeling and Virtual Screening</span></div><div class="casAuthors">Batool, Sidra; Khan, Zeshan Aslam; Kamal, Warda; Mushtaq, Gohar; Kamal, Mohammad Amjad</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">687-700</span>CODEN:
                <span class="NLM_cas:coden">MCEHAJ</span>;
        ISSN:<span class="NLM_cas:issn">1573-4064</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Objective: Drug resistance from affordable drugs has increased the no. of deaths from malaria globally.  This problem has raised the requirement to design new drugs against multidrugresistant Plasmodium falciparum parasite.  Methods: In the current project, we have focused on four important proteins of Plasmodium falciparum and used them as receptors against a dataset of four anti-malarial drugs.  In silico anal. of these receptors and ligand dataset was carried out using Autodock 4.2.  A pharmacophore model was also established using Ligandscout.  Results: Anal. of docking expts. showed that all ligands bind efficiently to four proteins of Plasmodium falciparum.  We have used ligand-based pharmacophore modeling and developed a pharmacophore model that has three hydrophobic regions, two arom. rings, one hydrogen acceptor and one hydrogen donor.  Using this pharmacophore model, we have screened a library of 50,000 compds.  The compds. that shared features of our pharmacophore model and exhibited interactions with the four proteins of our receptors dataset are short-listed.  Conclusion: As there is a need of more anti-malarial drugs, therefore, this research will be helpful in identifying novel anti-malarial drugs that exhibited bindings with four important proteins of Plasmodium falciparum.  The hits obtained in this study can be considered as useful leads in anti-malarial drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7UeYnQfB4bLVg90H21EOLACvtfcHk0lgfazgYU4kntQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2rsrjI&md5=01edcb27067a7529826bb7d2575f0a18</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.2174%2F1573406411666150305113533&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1573406411666150305113533%26sid%3Dliteratum%253Aachs%26aulast%3DBatool%26aufirst%3DS.%26aulast%3DKhan%26aufirst%3DZ.%2BA.%26aulast%3DKamal%26aufirst%3DW.%26aulast%3DMushtaq%26aufirst%3DG.%26aulast%3DKamal%26aufirst%3DM.%2BA.%26atitle%3DIn%2520silico%2520screening%2520for%2520identification%2520of%2520novel%2520anti-malarial%2520inhibitors%2520by%2520molecular%2520docking%252C%2520pharmacophore%2520modeling%2520and%2520virtual%2520screening%26jtitle%3DMed.%2520Chem.%26date%3D2015%26volume%3D11%26spage%3D687%26epage%3D700%26doi%3D10.2174%2F1573406411666150305113533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref112"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref112'); return false;" data-citation="" class="refNumLink">112</a></strong><div class="NLM_citation" id="cit112"><span><span class="NLM_contrib-group">Kumar, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raj, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varadwaj, P. K.</span><span> </span><span class="NLM_article-title">In-silico identification of inhibitors against mutated BCR-ABL protein of chronic myeloid leukemia: a virtual screening and molecular dynamics simulation study</span> <span class="citation_source-journal">J. Biomol. Struct. Dyn.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">13</span><span class="refDoi"> DOI: 10.1080/07391102.2015.1110046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1080%2F07391102.2015.1110046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BC28voslWntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=1-13&author=H.+Kumarauthor=U.+Rajauthor=S.+Guptaauthor=P.+K.+Varadwaj&title=In-silico+identification+of+inhibitors+against+mutated+BCR-ABL+protein+of+chronic+myeloid+leukemia%3A+a+virtual+screening+and+molecular+dynamics+simulation+study&doi=10.1080%2F07391102.2015.1110046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Azadirachtin(A) Distinctively Modulates Subdomain 2 of Actin - Novel Mechanism to Induce Depolymerization revealed by Molecular Dynamics Study</span></div><div class="casAuthors">Kumar R Pravin; Kumar R Pravin; Roopa L; Mohammed M M Sudheer; Kulkarni Naveen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular structure & dynamics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-39</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Azadirachtin(A) (AZA) a potential insecticide from neem binds to actin and induces depolymerization in Drosophila.  AZA binds to the pocket same as that of Latrunculin A (LAT) but LAT inhibits actin polymerization by stiffening the actin structure and affects the ADP-ATP exchange.  The mechanism by which AZA induces actin depolymerization is not clearly understood.  Therefore, different computational experiments were conducted to delineate the precise mechanism of AZA induced actin depolymerization.  Molecular dynamics studies showed that AZA strongly interacted with subdomain 2 and destabilised the interactions between subdomain 2 of one actin and subdomain 1 & 4 of the adjacent actin, causing the separation of actin subunits.  The separation was observed between subdomain 3 of subunit n and subdomain 4 of subunit n+2.  However, the specific triggering point for the separation of the subunits was the destabilisation of direct interactions between subdomain 2 of subunit n (Arg39, Val45, Gly46 and Arg62) and subdomain 4 of subunit n+2 (Asp286, Ile287, Asp288, Ile289, Asp244 and Lys291).  These results reveal a unique mechanism of an actin filament modulator that induces depolymerization.  This mechanism of AZA can be used to design similar molecules against mammalian actins for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR9OyCfOzN4fXELvajU9pjPfW6udTcc2eYV4MyhLWHeKbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28voslWntA%253D%253D&md5=49be334e5f4d7ad5bdfde38c2f89ff70</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1080%2F07391102.2015.1110046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F07391102.2015.1110046%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DH.%26aulast%3DRaj%26aufirst%3DU.%26aulast%3DGupta%26aufirst%3DS.%26aulast%3DVaradwaj%26aufirst%3DP.%2BK.%26atitle%3DIn-silico%2520identification%2520of%2520inhibitors%2520against%2520mutated%2520BCR-ABL%2520protein%2520of%2520chronic%2520myeloid%2520leukemia%253A%2520a%2520virtual%2520screening%2520and%2520molecular%2520dynamics%2520simulation%2520study%26jtitle%3DJ.%2520Biomol.%2520Struct.%2520Dyn.%26date%3D2015%26spage%3D1%26epage%3D13%26doi%3D10.1080%2F07391102.2015.1110046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref113"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref113'); return false;" data-citation="" class="refNumLink">113</a></strong><div class="NLM_citation" id="cit113"><span><span class="NLM_contrib-group">Shafique, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bibi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rashid, S.</span><span> </span><span class="NLM_article-title">In silico identification of putative bifunctional Plk1 inhibitors by integrative virtual screening and structural dynamics approach</span> <span class="citation_source-journal">J. Theor. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">388</span><span class="NLM_x">, </span> <span class="NLM_fpage">72</span><span class="NLM_x">–</span> <span class="NLM_lpage">84</span><span class="refDoi"> DOI: 10.1016/j.jtbi.2015.10.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.jtbi.2015.10.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=388&publication_year=2016&pages=72-84&author=S.+Shafiqueauthor=N.+Bibiauthor=S.+Rashid&title=In+silico+identification+of+putative+bifunctional+Plk1+inhibitors+by+integrative+virtual+screening+and+structural+dynamics+approach&doi=10.1016%2Fj.jtbi.2015.10.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1016%2Fj.jtbi.2015.10.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jtbi.2015.10.006%26sid%3Dliteratum%253Aachs%26aulast%3DShafique%26aufirst%3DS.%26aulast%3DBibi%26aufirst%3DN.%26aulast%3DRashid%26aufirst%3DS.%26atitle%3DIn%2520silico%2520identification%2520of%2520putative%2520bifunctional%2520Plk1%2520inhibitors%2520by%2520integrative%2520virtual%2520screening%2520and%2520structural%2520dynamics%2520approach%26jtitle%3DJ.%2520Theor.%2520Biol.%26date%3D2016%26volume%3D388%26spage%3D72%26epage%3D84%26doi%3D10.1016%2Fj.jtbi.2015.10.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref114"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref114'); return false;" data-citation="" class="refNumLink">114</a></strong><div class="NLM_citation" id="cit114"><span><span class="NLM_contrib-group">Pei, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.</span><span> </span><span class="NLM_article-title">Enrichment assessment of multiple virtual screening strategies for Toll-Like Receptor 8 agonists based on a maximal unbiased benchmarking data set</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">86</span><span class="NLM_x">, </span> <span class="NLM_fpage">1226</span><span class="NLM_x">–</span> <span class="NLM_lpage">1241</span><span class="refDoi"> DOI: 10.1111/cbdd.12590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1111%2Fcbdd.12590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26017460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFelsL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2015&pages=1226-1241&author=F.+Peiauthor=H.+Jinauthor=X.+Zhouauthor=J.+Xiaauthor=L.+Sunauthor=Z.+Liuauthor=L.+Zhang&title=Enrichment+assessment+of+multiple+virtual+screening+strategies+for+Toll-Like+Receptor+8+agonists+based+on+a+maximal+unbiased+benchmarking+data+set&doi=10.1111%2Fcbdd.12590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Enrichment Assessment of Multiple Virtual Screening Strategies for Toll-Like Receptor 8 Agonists Based on a Maximal Unbiased Benchmarking Data Set</span></div><div class="casAuthors">Pei, Fen; Jin, Hongwei; Zhou, Xin; Xia, Jie; Sun, Lidan; Liu, Zhenming; Zhang, Liangren</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1226-1241</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Toll-like receptor 8 agonists, which activate adaptive immune responses by inducing robust prodn. of T-helper 1-polarizing cytokines, are promising candidates for vaccine adjuvants.  As the binding site of toll-like receptor 8 is large and highly flexible, virtual screening by individual method has inevitable limitations; thus, a comprehensive comparison of different methods may provide insights into seeking effective strategy for the discovery of novel toll-like receptor 8 agonists.  In this study, the performance of knowledge-based pharmacophore, shape-based 3D screening, and combined strategies was assessed against a max. unbiased benchmarking data set contg. 13 actives and 1302 decoys specialized for toll-like receptor 8 agonists.  Prior structure-activity relationship knowledge was involved in knowledge-based pharmacophore generation, and a set of antagonists was innovatively used to verify the selectivity of the selected knowledge-based pharmacophore.  The benchmarking data set was generated from our recently developed 'MUBD-DECOYMAKER' protocol.  The enrichment assessment demonstrated a considerable performance through our selected three-layer virtual screening strategy: knowledge-based pharmacophore (Phar1) screening, shape-based 3D similarity search (Q4_combo), and then a Gold docking screening.  This virtual screening strategy could be further employed to perform large-scale database screening and to discover novel toll-like receptor 8 agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo80grcbCzo4bVg90H21EOLACvtfcHk0lhDL4TMBSY0YQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFelsL%252FF&md5=15e493183f6992ba27d549b32ce45147</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12590%26sid%3Dliteratum%253Aachs%26aulast%3DPei%26aufirst%3DF.%26aulast%3DJin%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DXia%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.%26atitle%3DEnrichment%2520assessment%2520of%2520multiple%2520virtual%2520screening%2520strategies%2520for%2520Toll-Like%2520Receptor%25208%2520agonists%2520based%2520on%2520a%2520maximal%2520unbiased%2520benchmarking%2520data%2520set%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2015%26volume%3D86%26spage%3D1226%26epage%3D1241%26doi%3D10.1111%2Fcbdd.12590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref115"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref115'); return false;" data-citation="" class="refNumLink">115</a></strong><div class="NLM_citation" id="cit115"><span><span class="NLM_contrib-group">Kumar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chauhan, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raghava, G. P.</span><span> </span><span class="NLM_article-title">In silico designing and screening of antagonists against cancer drug target XIAP</span> <span class="citation_source-journal">Curr. Cancer Drug Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">836</span><span class="NLM_x">–</span> <span class="NLM_lpage">846</span><span class="refDoi"> DOI: 10.2174/1568009615666150706103537</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.2174%2F1568009615666150706103537" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26143944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVGnsrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=836-846&author=R.+Kumarauthor=J.+S.+Chauhanauthor=G.+P.+Raghava&title=In+silico+designing+and+screening+of+antagonists+against+cancer+drug+target+XIAP&doi=10.2174%2F1568009615666150706103537"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">In Silico Designing and Screening of Antagonists against Cancer Drug Target XIAP</span></div><div class="casAuthors">Kumar, Rahul; Chauhan, Jagat Singh; Singh Raghava, Gajendra Pal</div><div class="citationInfo"><span class="NLM_cas:title">Current Cancer Drug Targets</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">836-846</span>CODEN:
                <span class="NLM_cas:coden">CCDTB9</span>;
        ISSN:<span class="NLM_cas:issn">1568-0096</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">X-linked inhibitor of apoptosis (XIAP) is a member of inhibitor of apoptosis (IAP) family and involved in the suppression of apoptosis in cancer cells.  This property makes it a therapeutic target for the cancer therapy.  In the present study, we have developed QSAR models using chem. descriptors, fingerprints, principal components, docking energy parameters and similarity-based approach against XIAP.  We have achieved correlation (R) of 0.803 with R2 value of 0.645 at 10-fold cross validation using SMOreg algorithm.  We have evaluated these models on independent dataset to ascertain its robustness and achieved correlation (R) of 0.793 with R2 value of 0.628.  Further, we have used these models for the screening of FDA approved drugs and drug-like mols. from ZINC database and prioritized them on the basis of their predicted pIC50 values.  Docking studies of top hits with XIAP-BIR3 domain shows that Iodixanol (DB01249) and ZINC68678304 have higher binding affinities than well-known tetrapeptide inhibitor, AVPI.  We have integrated these models in a web server named as "XIAPin".  We hope that this web server will contribute in the designing of nifty antagonists against XIAP.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrK-w4YQIpLBrVg90H21EOLACvtfcHk0lhDL4TMBSY0YQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVGnsrnN&md5=c0e189903d7946e10667d07e347a6883</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.2174%2F1568009615666150706103537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568009615666150706103537%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DR.%26aulast%3DChauhan%26aufirst%3DJ.%2BS.%26aulast%3DRaghava%26aufirst%3DG.%2BP.%26atitle%3DIn%2520silico%2520designing%2520and%2520screening%2520of%2520antagonists%2520against%2520cancer%2520drug%2520target%2520XIAP%26jtitle%3DCurr.%2520Cancer%2520Drug%2520Targets%26date%3D2015%26volume%3D15%26spage%3D836%26epage%3D846%26doi%3D10.2174%2F1568009615666150706103537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref116"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref116'); return false;" data-citation="" class="refNumLink">116</a></strong><div class="NLM_citation" id="cit116"><span><span class="NLM_contrib-group">Mathew, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fatima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fatmi, M. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Archunan, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ilyas, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Begum, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azhar, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damanhouri, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qadri, I.</span><span> </span><span class="NLM_article-title">Computational docking study of p7 ion channel from HCV genotype 3 and genotype 4 and its interaction with natural compounds</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0126510</span><span class="refDoi"> DOI: 10.1371/journal.pone.0126510</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pone.0126510" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0126510&author=S.+Mathewauthor=K.+Fatimaauthor=M.+Q.+Fatmiauthor=G.+Archunanauthor=M.+Ilyasauthor=N.+Begumauthor=E.+Azharauthor=G.+Damanhouriauthor=I.+Qadri&title=Computational+docking+study+of+p7+ion+channel+from+HCV+genotype+3+and+genotype+4+and+its+interaction+with+natural+compounds&doi=10.1371%2Fjournal.pone.0126510"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0126510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0126510%26sid%3Dliteratum%253Aachs%26aulast%3DMathew%26aufirst%3DS.%26aulast%3DFatima%26aufirst%3DK.%26aulast%3DFatmi%26aufirst%3DM.%2BQ.%26aulast%3DArchunan%26aufirst%3DG.%26aulast%3DIlyas%26aufirst%3DM.%26aulast%3DBegum%26aufirst%3DN.%26aulast%3DAzhar%26aufirst%3DE.%26aulast%3DDamanhouri%26aufirst%3DG.%26aulast%3DQadri%26aufirst%3DI.%26atitle%3DComputational%2520docking%2520study%2520of%2520p7%2520ion%2520channel%2520from%2520HCV%2520genotype%25203%2520and%2520genotype%25204%2520and%2520its%2520interaction%2520with%2520natural%2520compounds%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0126510%26doi%3D10.1371%2Fjournal.pone.0126510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref117"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref117'); return false;" data-citation="" class="refNumLink">117</a></strong><div class="NLM_citation" id="cit117"><span><span class="NLM_contrib-group">Lee, T. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arcus, V. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lott, J. S.</span><span> </span><span class="NLM_article-title">Prediction of the substrate for nonribosomal peptide synthetase (NRPS) adenylation domains by virtual screening</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">2052</span><span class="NLM_x">–</span> <span class="NLM_lpage">2066</span><span class="refDoi"> DOI: 10.1002/prot.24922</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fprot.24922" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2015&pages=2052-2066&author=T.+V.+Leeauthor=R.+D.+Johnsonauthor=V.+L.+Arcusauthor=J.+S.+Lott&title=Prediction+of+the+substrate+for+nonribosomal+peptide+synthetase+%28NRPS%29+adenylation+domains+by+virtual+screening&doi=10.1002%2Fprot.24922"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1002%2Fprot.24922&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.24922%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DT.%2BV.%26aulast%3DJohnson%26aufirst%3DR.%2BD.%26aulast%3DArcus%26aufirst%3DV.%2BL.%26aulast%3DLott%26aufirst%3DJ.%2BS.%26atitle%3DPrediction%2520of%2520the%2520substrate%2520for%2520nonribosomal%2520peptide%2520synthetase%2520%2528NRPS%2529%2520adenylation%2520domains%2520by%2520virtual%2520screening%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2015%26volume%3D83%26spage%3D2052%26epage%3D2066%26doi%3D10.1002%2Fprot.24922" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref118"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref118'); return false;" data-citation="" class="refNumLink">118</a></strong><div class="NLM_citation" id="cit118"><span><span class="NLM_contrib-group">Xiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span> </span><span class="NLM_article-title">Topomer CoMFA and virtual screening studies of azaindole class renin inhibitors</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">458</span><span class="NLM_x">–</span> <span class="NLM_lpage">472</span><span class="refDoi"> DOI: 10.2174/1386207317666140107094708</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.2174%2F1386207317666140107094708" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24392830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVOnsLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2014&pages=458-472&author=Y.+Xiangauthor=J.+Songauthor=Z.+Zhang&title=Topomer+CoMFA+and+virtual+screening+studies+of+azaindole+class+renin+inhibitors&doi=10.2174%2F1386207317666140107094708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Topomer CoMFA and Virtual Screening Studies of Azaindole Class Renin Inhibitors</span></div><div class="casAuthors">Xiang, Yuhong; Song, Jia; Zhang, Zhuoyong</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry & High Throughput Screening</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">458-472</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Direct renin inhibitors (DRIs) have increasingly shown a significant advantage in the treatment of hypertension and protection of target organs.  In this paper, a series of azaindole class renin inhibitors were subjected to 3D-QSAR study using Topomer CoMFA.  Five kinds of splitting mode for different fragment cutting and 6 different training and test sets grouping were attempted to build consensus models.  The results indicated that 6 consensus models had similar predictability (q2 and r2pred) and stability (r2).  The best model showed good stability and predictability (q2 of 0.616, r2 of 0.908).  The r2pred value of the external test set was 0.70, which means that the model had an excellent external predictive ability, and the robustness of the developed model was assessed by the Y-randomization test.  This study also adopted the methodol. of fragment-based drug design (FBDD) to virtual screen new renin inhibitors by using Topomer Search technol.  The R1-group of the compd. No. 13 with the highest activity was chosen as the basic scaffold, and its remaining R2-group acted as a query to screen 142,025 mols. of ZINC database for similar fragments.  The obtained 30 fragments with the highest R2-group contribution values were added to the basic scaffold resp.  Finally 30 new azaindole compds. with potent high activities were obtained.  Further the binding modes were studied by using Surflex- Dock.  The docking results showed good binding interactions of the designed compds. with the renin protein, thus the rationality of this design was further verified from the perspective of the renin receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgmCzCRZwX_bVg90H21EOLACvtfcHk0lgbSQ1VBScyHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVOnsLrK&md5=eb17c6c0bfece3dcc70dad6f1bc36bfc</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.2174%2F1386207317666140107094708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1386207317666140107094708%26sid%3Dliteratum%253Aachs%26aulast%3DXiang%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DZ.%26atitle%3DTopomer%2520CoMFA%2520and%2520virtual%2520screening%2520studies%2520of%2520azaindole%2520class%2520renin%2520inhibitors%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2014%26volume%3D17%26spage%3D458%26epage%3D472%26doi%3D10.2174%2F1386207317666140107094708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref119"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref119'); return false;" data-citation="" class="refNumLink">119</a></strong><div class="NLM_citation" id="cit119"><span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blagg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buser-Doepner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravatt, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frederiksen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimshaw, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hepworth, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howe, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotz, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mader, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller-Fahrnow, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Hagan, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundstrom, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taunton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toledo-Sherman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walpole, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walters, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willson, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuercher, W. J.</span><span> </span><span class="NLM_article-title">The promise and peril of chemical probes</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">536</span><span class="NLM_x">–</span> <span class="NLM_lpage">541</span><span class="refDoi"> DOI: 10.1038/nchembio.1867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnchembio.1867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26196764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=536-541&author=C.+H.+Arrowsmithauthor=J.+E.+Audiaauthor=C.+Austinauthor=J.+Baellauthor=J.+Bennettauthor=J.+Blaggauthor=C.+Bountraauthor=P.+E.+Brennanauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=C.+Buser-Doepnerauthor=R.+M.+Campbellauthor=A.+J.+Carterauthor=P.+Cohenauthor=R.+A.+Copelandauthor=B.+Cravattauthor=J.+L.+Dahlinauthor=D.+Dhanakauthor=A.+M.+Edwardsauthor=M.+Frederiksenauthor=S.+V.+Fryeauthor=N.+Grayauthor=C.+E.+Grimshawauthor=D.+Hepworthauthor=T.+Howeauthor=K.+V.+Huberauthor=J.+Jinauthor=S.+Knappauthor=J.+D.+Kotzauthor=R.+G.+Krugerauthor=D.+Loweauthor=M.+M.+Maderauthor=B.+Marsdenauthor=A.+Mueller-Fahrnowauthor=S.+Mullerauthor=R.+C.+O%E2%80%99Haganauthor=J.+P.+Overingtonauthor=D.+R.+Owenauthor=S.+H.+Rosenbergauthor=B.+Rothauthor=R.+Rossauthor=M.+Schapiraauthor=S.+L.+Schreiberauthor=B.+Shoichetauthor=M.+Sundstromauthor=G.+Superti-Furgaauthor=J.+Tauntonauthor=L.+Toledo-Shermanauthor=C.+Walpoleauthor=M.+A.+Waltersauthor=T.+M.+Willsonauthor=P.+Workmanauthor=R.+N.+Youngauthor=W.+J.+Zuercher&title=The+promise+and+peril+of+chemical+probes&doi=10.1038%2Fnchembio.1867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">The promise and peril of chemical probes</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Audia, James E.; Austin, Christopher; Baell, Jonathan; Bennett, Jonathan; Blagg, Julian; Bountra, Chas; Brennan, Paul E.; Brown, Peter J.; Bunnage, Mark E.; Buser-Doepner, Carolyn; Campbell, Robert M.; Carter, Adrian J.; Cohen, Philip; Copeland, Robert A.; Cravatt, Ben; Dahlin, Jayme L.; Dhanak, Dashyant; Edwards, Aled M.; Frye, Stephen V.; Gray, Nathanael; Grimshaw, Charles E.; Hepworth, David; Howe, Trevor; Huber, Kilian V. M.; Jin, Jian; Knapp, Stefan; Kotz, Joanne D.; Kruger, Ryan G.; Lowe, Derek; Mader, Mary M.; Marsden, Brian; Mueller-Fahrnow, Anke; Muller, Susanne; O'Hagan, Ronan C.; Overington, John P.; Owen, Dafydd R.; Rosenberg, Saul H.; Roth, Brian; Ross, Ruth; Schapira, Matthieu; Schreiber, Stuart L.; Shoichet, Brian; Sundstrom, Michael; Superti-Furga, Giulio; Taunton, Jack; Toledo-Sherman, Leticia; Walpole, Chris; Walters, Michael A.; Willson, Timothy M.; Workman, Paul; Young, Robert N.; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">536-541</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes are powerful reagents with increasing impacts on biomedical research.  However, probes of poor quality or that are used incorrectly generate misleading results.  To help address these shortcomings, we will create a community-driven wiki resource to improve quality and convey current best practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-li4dxq487rVg90H21EOLACvtfcHk0lgbSQ1VBScyHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO&md5=d8c7a7af94921ab4cdcc871487cceadd</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1867%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DCarter%26aufirst%3DA.%2BJ.%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DHuber%26aufirst%3DK.%2BV.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKotz%26aufirst%3DJ.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DLowe%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Hagan%26aufirst%3DR.%2BC.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DRoth%26aufirst%3DB.%26aulast%3DRoss%26aufirst%3DR.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DSundstrom%26aufirst%3DM.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DToledo-Sherman%26aufirst%3DL.%26aulast%3DWalpole%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DR.%2BN.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26atitle%3DThe%2520promise%2520and%2520peril%2520of%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D536%26epage%3D541%26doi%3D10.1038%2Fnchembio.1867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref120"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref120'); return false;" data-citation="" class="refNumLink">120</a></strong><div class="NLM_citation" id="cit120"><span><span class="NLM_contrib-group">Baell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, G. A.</span><span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2719</span><span class="NLM_x">–</span> <span class="NLM_lpage">2740</span><span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lhocKMrIuActA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref121"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref121'); return false;" data-citation="" class="refNumLink">121</a></strong><div class="NLM_citation" id="cit121"><span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Screening in a spirit haunted world</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">607</span><span class="NLM_x">–</span> <span class="NLM_lpage">615</span><span class="refDoi"> DOI: 10.1016/j.drudis.2006.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.drudis.2006.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=16793529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=607-615&author=B.+K.+Shoichet&title=Screening+in+a+spirit+haunted+world&doi=10.1016%2Fj.drudis.2006.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit121R"><div class="casContent"><span class="casTitleNuber">121</span><div class="casTitle"><span class="NLM_cas:atitle">Screening in a spirit haunted world</span></div><div class="casAuthors">Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">13 & 14</span>),
    <span class="NLM_cas:pages">607-615</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  High-throughput screening (HTS) campaigns can be dominated by hits that ultimately turn out to be non-drug-like.  These 'nuisance' compds. often behave strangely, with steep dose-response curves, absence of clear structure-activity relationships, and high sensitivity to assay conditions.  Several mechanisms contribute to these artifacts, including chem. reactive mols., those that absorb light in assays and those that affect redox conditions.  One of the most common mechanisms behind artifactual inhibition is discussed in this review: at micromolar concns. org. mols. can aggregate to form particles in aq. buffers, and these aggregates can sequester and thereby inhibit protein targets.  Aggregation-based inhibition is baffling from a chem. perspective, but viewed biophys. such behavior is expected.  The range of mols. that behave this way, their rapid detection in a screening environment and their possible biol. implications will be considered here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzoEkwI2izy7Vg90H21EOLACvtfcHk0lhocKMrIuActA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLk%253D&md5=08d22b6b06215ac435824b7664b3674c</span></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DScreening%2520in%2520a%2520spirit%2520haunted%2520world%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26spage%3D607%26epage%3D615%26doi%3D10.1016%2Fj.drudis.2006.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref122"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref122'); return false;" data-citation="" class="refNumLink">122</a></strong><div class="NLM_citation" id="cit122"><span><span class="NLM_contrib-group">Walters, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Namchuk, M.</span><span> </span><span class="NLM_article-title">Designing screens: how to make your hits a hit</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">259</span><span class="NLM_x">–</span> <span class="NLM_lpage">266</span><span class="refDoi"> DOI: 10.1038/nrd1063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnrd1063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=12669025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3sXisFWktrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=259-266&author=W.+P.+Waltersauthor=M.+Namchuk&title=Designing+screens%3A+how+to+make+your+hits+a+hit&doi=10.1038%2Fnrd1063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">A guide to drug discovery: Designing screens: how to make your hits a hit</span></div><div class="casAuthors">Walters, W. Patrick; Namchuk, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">259-266</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The basic goal of small-mol. screening is the identification of chem. interesting starting points for elaboration towards a drug.  A no. of innovative approaches for pursuing this goal have evolved, and the right approach is dictated by the target class being pursued and the capabilities of the organization involved.hood.  A recent trend in high-throughput screening has been to place less emphasis on the no. of data points that can be produced, and to focus instead on the quality of the data obtained.  Several computational and technol. advances have aided in the selection of compds. for screening and widened the variety of assay formats available for screening.  The effect on the efficiency of the screening process is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ5E12ik7BrbVg90H21EOLACvtfcHk0lhocKMrIuActA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXisFWktrk%253D&md5=6bec4547b8dce69ab84ce226cde28f8a</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1038%2Fnrd1063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1063%26sid%3Dliteratum%253Aachs%26aulast%3DWalters%26aufirst%3DW.%2BP.%26aulast%3DNamchuk%26aufirst%3DM.%26atitle%3DDesigning%2520screens%253A%2520how%2520to%2520make%2520your%2520hits%2520a%2520hit%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D259%26epage%3D266%26doi%3D10.1038%2Fnrd1063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref123"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref123'); return false;" data-citation="" class="refNumLink">123</a></strong><div class="NLM_citation" id="cit123"><span><span class="NLM_contrib-group">Rishton, G. M.</span><span> </span><span class="NLM_article-title">Nonleadlikeness and leadlikeness in biochemical screening</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">86</span><span class="NLM_x">–</span> <span class="NLM_lpage">96</span><span class="refDoi"> DOI: 10.1016/S1359644602025722</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2FS1359644602025722" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=12565011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BD3s%252Fls1SjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=86-96&author=G.+M.+Rishton&title=Nonleadlikeness+and+leadlikeness+in+biochemical+screening&doi=10.1016%2FS1359644602025722"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">Nonleadlikeness and leadlikeness in biochemical screening</span></div><div class="casAuthors">Rishton Gilbert M</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">86-96</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">Biochemical assays have largely supplanted functional biological assays as drug screening tools in the early stages of drug discovery.  The de-selection of compounds that are 'nonleadlike' binders (and bonders) and the proactive selection of those compounds that are 'leadlike' in their binding to the target are vital components of the screening effort.  The physiochemical properties of leadlikeness and the surprising differences between those properties and the now classical definitions of druglikeness are becoming apparent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSFoWSt5LTTmsrRqEXsZufAfW6udTcc2ebCJO2wLtGKobntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3s%252Fls1SjtQ%253D%253D&md5=5e34cece6f751fa727f949c3526aa95c</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1016%2FS1359644602025722&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359644602025722%26sid%3Dliteratum%253Aachs%26aulast%3DRishton%26aufirst%3DG.%2BM.%26atitle%3DNonleadlikeness%2520and%2520leadlikeness%2520in%2520biochemical%2520screening%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2003%26volume%3D8%26spage%3D86%26epage%3D96%26doi%3D10.1016%2FS1359644602025722" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref124"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref124'); return false;" data-citation="" class="refNumLink">124</a></strong><div class="NLM_citation" id="cit124"><span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Southall, N. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinn, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auld, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span> </span><span class="NLM_article-title">Fluorescence spectroscopic profiling of compound libraries</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2363</span><span class="NLM_x">–</span> <span class="NLM_lpage">2371</span><span class="refDoi"> DOI: 10.1021/jm701301m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701301m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjslSns7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2363-2371&author=A.+Simeonovauthor=A.+Jadhavauthor=C.+J.+Thomasauthor=Y.+Wangauthor=R.+Huangauthor=N.+T.+Southallauthor=P.+Shinnauthor=J.+Smithauthor=C.+P.+Austinauthor=D.+S.+Auldauthor=J.+Inglese&title=Fluorescence+spectroscopic+profiling+of+compound+libraries&doi=10.1021%2Fjm701301m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit124R"><div class="casContent"><span class="casTitleNuber">124</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence Spectroscopic Profiling of Compound Libraries</span></div><div class="casAuthors">Simeonov, Anton; Jadhav, Ajit; Thomas, Craig J.; Wang, Yuhong; Huang, Ruili; Southall, Noel T.; Shinn, Paul; Smith, Jeremy; Austin, Christopher P.; Auld, Douglas S.; Inglese, James</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2363-2371</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chromo/fluorophoric properties often accompany the heterocyclic scaffolds and impurities that comprise libraries used for high-throughput screening (HTS).  These properties affect assay outputs obtained with optical detection, thus complicating anal. and leading to false positives and negatives.  Here, we report the fluorescence profile of more than 70 000 samples across spectral regions commonly utilized in HTS.  The quant. HTS paradigm was utilized to test each sample at seven or more concns. over a 4-log range in 1536-well format.  Raw fluorescence was compared with fluorophore stds. to compute a normalized response as a function of concn. and spectral region.  More than 5% of library members were brighter than the equiv. of 10 nM 4-Me umbelliferone, a common UV-active probe.  Red-shifting the spectral window by as little as 100 nm was accompanied by a dramatic decrease in autofluorescence.  Native compd. fluorescence, fluorescent impurities, novel fluorescent compds., and the utilization of fluorescence profiling data are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreZBF1CG9forVg90H21EOLACvtfcHk0lh7FvSZ5oiJlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjslSns7Y%253D&md5=4a7b8ee758fce4be3586aa5b573a3fc1</span></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1021%2Fjm701301m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701301m%26sid%3Dliteratum%253Aachs%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DSouthall%26aufirst%3DN.%2BT.%26aulast%3DShinn%26aufirst%3DP.%26aulast%3DSmith%26aufirst%3DJ.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DInglese%26aufirst%3DJ.%26atitle%3DFluorescence%2520spectroscopic%2520profiling%2520of%2520compound%2520libraries%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2363%26epage%3D2371%26doi%3D10.1021%2Fjm701301m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref125"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref125'); return false;" data-citation="" class="refNumLink">125</a></strong><div class="NLM_citation" id="cit125"><span><span class="NLM_contrib-group">Lea, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span> </span><span class="NLM_article-title">Fluorescence polarization assays in small molecule screening</span> <span class="citation_source-journal">Expert Opin. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">17</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1517/17460441.2011.537322</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1517%2F17460441.2011.537322" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=22328899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SrtbvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=17-32&author=W.+A.+Leaauthor=A.+Simeonov&title=Fluorescence+polarization+assays+in+small+molecule+screening&doi=10.1517%2F17460441.2011.537322"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence polarization assays in small molecule screening</span></div><div class="casAuthors">Lea, Wendy A.; Simeonov, Anton</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17-32</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Importance of the field: Fluorescence polarization (FP) is a homogeneous method that allows rapid and quant. anal. of diverse mol. interactions and enzyme activities.  This technique has been widely utilized in clin. and biomedical settings, including the diagnosis of certain diseases and monitoring therapeutic drug levels in body fluids.  Recent developments in the field have been symbolized by the facile adoption of FP in high-throughput screening and small mol. drug discovery of an increasing range of target classes.  Areas covered in this review: The article provides a brief overview of the theor. foundation of FP, followed by updates on recent advancements in its application for various drug target classes, including GPCRs, enzymes and protein-protein interactions.  The strengths and weaknesses of this method, practical considerations in assay design, novel applications and future directions are also discussed.  What the reader will gain: The reader is informed of the most recent advancements and future directions of FP application to small mol. screening.  Take home message: In addn. to its continued utilization in high-throughput screening, FP has expanded into new disease and target areas and has been marked by increased use of labeled small mol. ligands for receptor-binding studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqudLso-a4wR7Vg90H21EOLACvtfcHk0lh7FvSZ5oiJlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SrtbvP&md5=6f102f40b56fd7a13c4fb59b64a3f883</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1517%2F17460441.2011.537322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2011.537322%26sid%3Dliteratum%253Aachs%26aulast%3DLea%26aufirst%3DW.%2BA.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DFluorescence%2520polarization%2520assays%2520in%2520small%2520molecule%2520screening%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2011%26volume%3D6%26spage%3D17%26epage%3D32%26doi%3D10.1517%2F17460441.2011.537322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref126"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref126'); return false;" data-citation="" class="refNumLink">126</a></strong><div class="NLM_citation" id="cit126"><span><span class="NLM_contrib-group">Thorne, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auld, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span> </span><span class="NLM_article-title">Apparent activity in high-throughput screening: origins of compound-dependent assay interference</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">315</span><span class="NLM_x">–</span> <span class="NLM_lpage">324</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2010.03.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.cbpa.2010.03.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=20417149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=315-324&author=N.+Thorneauthor=D.+S.+Auldauthor=J.+Inglese&title=Apparent+activity+in+high-throughput+screening%3A+origins+of+compound-dependent+assay+interference&doi=10.1016%2Fj.cbpa.2010.03.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">Apparent activity in high-throughput screening: origins of compound-dependent assay interference</span></div><div class="casAuthors">Thorne, Natasha; Auld, Douglas S.; Inglese, James</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">315-324</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Expansive compd. collections made up of structurally heterogeneous chems., the activities of which are largely undefined, present challenging problems for high-throughput screening (HTS).  Foremost is differentiating whether the activity for a given compd. in an assay is directed against the targeted biol., or is the result of surreptitious compd. activity involving the assay detection system.  Such compd. interference can be esp. difficult to identify if it is reproducible and concn.-dependent - characteristics generally attributed to compds. with genuine activity.  While reactive chem. groups on compds. were once thought to be the primary source of compd. interference in assays used in HTS, recent work suggests that other factors, such as compd. aggregation, may play a more significant role in many assay formats.  Considerable progress has been made to profile representative compd. libraries in an effort to identify chem. classes susceptible to producing compd. interference, such as compds. commonly found to inhibit the reporter enzyme firefly luciferase.  Such work has also led to the development of practices that have the potential to significantly reduce compd. interference, for example, through the addn. of non-ionic detergent to assay buffer to reduce aggregation-based inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBawORe7gR97Vg90H21EOLACvtfcHk0lh7FvSZ5oiJlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsL8%253D&md5=4d3dd71f66ce937002874106edaff12e</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.03.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.03.020%26sid%3Dliteratum%253Aachs%26aulast%3DThorne%26aufirst%3DN.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DInglese%26aufirst%3DJ.%26atitle%3DApparent%2520activity%2520in%2520high-throughput%2520screening%253A%2520origins%2520of%2520compound-dependent%2520assay%2520interference%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26spage%3D315%26epage%3D324%26doi%3D10.1016%2Fj.cbpa.2010.03.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref127"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref127'); return false;" data-citation="" class="refNumLink">127</a></strong><div class="NLM_citation" id="cit127"><span><span class="NLM_contrib-group">McGovern, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caselli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grigorieff, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">A common mechanism underlying promiscuous inhibitors from virtual and high-throughput screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">1712</span><span class="NLM_x">–</span> <span class="NLM_lpage">1722</span><span class="refDoi"> DOI: 10.1021/jm010533y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm010533y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD38Xhslegtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=1712-1722&author=S.+L.+McGovernauthor=E.+Caselliauthor=N.+Grigorieffauthor=B.+K.+Shoichet&title=A+common+mechanism+underlying+promiscuous+inhibitors+from+virtual+and+high-throughput+screening&doi=10.1021%2Fjm010533y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">A Common Mechanism Underlying Promiscuous Inhibitors from Virtual and High-Throughput Screening</span></div><div class="casAuthors">McGovern, Susan L.; Caselli, Emilia; Grigorieff, Nikolaus; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1712-1722</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput and virtual screening are widely used to discover novel leads for drug design.  On examn., many screening hits appear non-drug-like: they act noncompetitively, show little relation between structure and activity, and have poor selectivity.  Attempts to develop these peculiar mols. into viable leads are often futile, and much time can be wasted on the characterization of these "phony" hits.  Despite their common occurrence, the mechanism of action of these promiscuous mols. remains unknown.  To investigate this problem, 45 diverse screening hits were studied.  Fifteen of these were previously reported as inhibitors of various receptors, including β-lactamase, malarial protease, dihydrofolate reductase, HIV Tar RNA, thymidylate synthase, kinesin, insulin receptor, tyrosine kinases, farnesyltransferase, gyrase, prions, triosephosphate isomerase, nitric oxide synthase, phosphoinositide 3-kinase, and integrase; 30 were from an inhouse screening library of a major pharmaceutical company.  In addn. to their original targets, 35 of these 45 compds. were shown to inhibit several unrelated model enzymes.  These 35 screening hits included compds., such as fullerenes, dyes, and quercetin, that have repeatedly shown activity against diverse targets.  When tested against the model enzymes, the compds. showed time-dependent but reversible inhibition that was dramatically attenuated by albumin, guanidinium, or urea. Surprisingly, increasing the concn. of the model enzymes 10-fold largely eliminated inhibition, despite a 1000-fold excess of inhibitor; a well-behaved competitive inhibitor did not show this behavior.  One model to explain these observations was that the active form of the promiscuous inhibitors was an aggregate of many individual mols.  To test this hypothesis, light scattering and electron microscopy expts. were performed.  The nonspecific inhibitors were obsd. to form particles of 30-400 nm diam. by both techniques.  In control expts., a well-behaved competitive inhibitor and an inactive dye-like mol. were not obsd. to form aggregates.  Consistent with the hypothesis that the aggregates are the inhibitory species, the particle size and IC50 values of the promiscuous inhibitors varied monotonically with ionic strength; a competitive inhibitor was unaffected by changes in ionic strength. Unexpectedly, aggregate formation appears to explain the activity of many nonspecific inhibitors and may account for the activity of many promiscuous screening hits.  Mols. acting via this mechanism may be widespread in drug discovery screening databases.  Recognition of these compds. may improve screening results in many areas of pharmaceutical interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYFB0NNK2CIbVg90H21EOLACvtfcHk0li7wSeiG6Pz9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xhslegtrk%253D&md5=b9e6c29f6a1da588a6452859061edfbb</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1021%2Fjm010533y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm010533y%26sid%3Dliteratum%253Aachs%26aulast%3DMcGovern%26aufirst%3DS.%2BL.%26aulast%3DCaselli%26aufirst%3DE.%26aulast%3DGrigorieff%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DA%2520common%2520mechanism%2520underlying%2520promiscuous%2520inhibitors%2520from%2520virtual%2520and%2520high-throughput%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D1712%26epage%3D1722%26doi%3D10.1021%2Fjm010533y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref128"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref128'); return false;" data-citation="" class="refNumLink">128</a></strong><div class="NLM_citation" id="cit128"><span><span class="NLM_contrib-group">Feng, B. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelat, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doman, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guy, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">High-throughput assays for promiscuous inhibitors</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">–</span> <span class="NLM_lpage">148</span><span class="refDoi"> DOI: 10.1038/nchembio718</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnchembio718" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=16408018" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmt1yhu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2005&pages=146-148&author=B.+Y.+Fengauthor=A.+Shelatauthor=T.+N.+Domanauthor=R.+K.+Guyauthor=B.+K.+Shoichet&title=High-throughput+assays+for+promiscuous+inhibitors&doi=10.1038%2Fnchembio718"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput assays for promiscuous inhibitors</span></div><div class="casAuthors">Feng, Brian Y.; Shelat, Anang; Doman, Thompson N.; Guy, R. Kip; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">146-148</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) searches large libraries of chem. compds. for those that can modulate the activity of a particular biol. target; it is the dominant technique used in early-stage drug discovery.  A key problem in HTS is the prevalence of nonspecific or 'promiscuous' inhibitors.  These mols. have peculiar properties, act on unrelated targets and can dominate the results from screening campaigns.  Several explanations have been proposed to account for promiscuous inhibitors, including chem. reactivity, interference in assay read-out, high mol. flexibility and hydrophobicity.  The diversity of these models reflects the apparently unrelated mols. whose behaviors they seek to explain.  However, a single mechanism may explain the effects of many promiscuous inhibitors: some org. mols. form large colloid-like aggregates that sequester and thereby inhibit enzymes.  Hits from HTS, leads for drug discovery and even several drugs appear to act through this mechanism at micromolar concns.  Here, we report two rapid assays for detecting promiscuous aggregates that we tested against 1,030 'drug-like' mols.  The results from these assays were used to test two preliminary computational models of this phenomenon and as benchmarks to develop new models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCy0Rj4jKNOLVg90H21EOLACvtfcHk0li7wSeiG6Pz9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmt1yhu7k%253D&md5=3414328ac1d56eb2cce7c59f18a784d0</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1038%2Fnchembio718&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio718%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DB.%2BY.%26aulast%3DShelat%26aufirst%3DA.%26aulast%3DDoman%26aufirst%3DT.%2BN.%26aulast%3DGuy%26aufirst%3DR.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DHigh-throughput%2520assays%2520for%2520promiscuous%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2005%26volume%3D1%26spage%3D146%26epage%3D148%26doi%3D10.1038%2Fnchembio718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref129"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref129'); return false;" data-citation="" class="refNumLink">129</a></strong><div class="NLM_citation" id="cit129"><span><span class="NLM_contrib-group">Coan, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Stoichiometry and physical chemistry of promiscuous aggregate-based inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">9606</span><span class="NLM_x">–</span> <span class="NLM_lpage">9612</span><span class="refDoi"> DOI: 10.1021/ja802977h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja802977h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvF2gsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=9606-9612&author=K.+E.+Coanauthor=B.+K.+Shoichet&title=Stoichiometry+and+physical+chemistry+of+promiscuous+aggregate-based+inhibitors&doi=10.1021%2Fja802977h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit129R"><div class="casContent"><span class="casTitleNuber">129</span><div class="casTitle"><span class="NLM_cas:atitle">Stoichiometry and Physical Chemistry of Promiscuous Aggregate-Based Inhibitors</span></div><div class="casAuthors">Coan, Kristin E. D.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">9606-9612</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many false positives in early drug discovery owe to nonspecific inhibition by colloid-like aggregates of org. mols.  Despite their prevalence, little is known about aggregate concn., structure, or dynamic equil.; the binding mechanism, stoichiometry with, and affinity for enzymes remain uncertain.  To investigate the elementary question of concn., we counted aggregate particles using flow cytometry.  For seven aggregate-forming mols., aggregates were not obsd. until the concn. of monomer crossed a threshold, indicating a "crit. aggregation concn." (CAC).  Above the CAC, aggregate count increased linearly with added org. material, while the particles dispersed when dild. below the CAC.  The concn. of monomeric org. mol. is const. above the CAC, as is the size of the aggregate particles.  For two compds. that form large aggregates, nicardipine and miconazole, we measured particle nos. directly by flow cytometry, detg. that the aggregate concn. just above the CAC ranged from 5 to 30 fM.  By correlating inhibition of an enzyme with aggregate count for these two drugs, we detd. that the stoichiometry of binding is about 10 000 enzyme mols. per aggregate particle.  Using measured vols. for nicardipine and miconazole aggregate particles (2.1 × 1011 and 4.7 × 1010 Å3, resp.), computed monomer vols., and the observation that past the CAC all addnl. monomer forms aggregate particles, we find that aggregates are densely packed particles.  Finally, given their size and enzyme stoichiometry, all sequestered enzyme can be comfortably accommodated on the surface of the aggregate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXzKuuN6PwebVg90H21EOLACvtfcHk0li7wSeiG6Pz9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvF2gsL8%253D&md5=6bb7b0d190db00bd872a2ba86a5d56ef</span></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1021%2Fja802977h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja802977h%26sid%3Dliteratum%253Aachs%26aulast%3DCoan%26aufirst%3DK.%2BE.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStoichiometry%2520and%2520physical%2520chemistry%2520of%2520promiscuous%2520aggregate-based%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D9606%26epage%3D9612%26doi%3D10.1021%2Fja802977h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref130"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref130'); return false;" data-citation="" class="refNumLink">130</a></strong><div class="NLM_citation" id="cit130"><span><span class="NLM_contrib-group">Park, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ku, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, B. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. J.</span><span> </span><span class="NLM_article-title">Identification of novel PTPRQ phosphatase inhibitors based on the virtual screening with docking simulations</span> <span class="citation_source-journal">Theor. Biol. Med. Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">49</span><span class="refDoi"> DOI: 10.1186/1742-4682-10-49</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1186%2F1742-4682-10-49" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=23981594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXis1Kisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=49&author=H.+Parkauthor=K.+R.+Yuauthor=B.+Kuauthor=B.+Y.+Kimauthor=S.+J.+Kim&title=Identification+of+novel+PTPRQ+phosphatase+inhibitors+based+on+the+virtual+screening+with+docking+simulations&doi=10.1186%2F1742-4682-10-49"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit130R"><div class="casContent"><span class="casTitleNuber">130</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel PTPRQ phosphatase inhibitors based on the virtual screening with docking simulations</span></div><div class="casAuthors">Park, Hwangseo; Yu, Keum Ran; Ku, Bonsu; Kim, Bo Yeon; Kim, Seung Jun</div><div class="citationInfo"><span class="NLM_cas:title">Theoretical Biology and Medical Modelling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">49/1-49/10, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">TBMMA8</span>;
        ISSN:<span class="NLM_cas:issn">1742-4682</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Protein tyrosine phosphatase receptor type Q (PTPRQ) is an unusual PTP that has intrinsic dephosphorylating activity for various phosphatidyl inositides instead of phospho-tyrosine substrates.  Although PTPRQ was known to be involved in the pathogenesis of obesity, no small-mol. inhibitor has been reported so far.  Here we report six novel PTPRQ inhibitors identified with computer-aided drug design protocol involving the virtual screening with docking simulations and enzyme inhibition assay.  These inhibitors exhibit moderate potencies against PTPRQ with the assocd. IC50 values ranging from 29 to 86 μM.  Because the newly discovered inhibitors were also computationally screened for having desirable physicochem. properties as a drug candidate, they deserve consideration for further development by structure-activity relationship studies to optimize the antiobestic activities.  Structural features relevant to the stabilization of the inhibitors in the active site of PTPRQ are addressed in detail.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmk6jn7B36uLVg90H21EOLACvtfcHk0lixdMGby2Yk-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXis1Kisr8%253D&md5=27f8ae52fff042eba62b3ab816d0b303</span></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1186%2F1742-4682-10-49&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1742-4682-10-49%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DK.%2BR.%26aulast%3DKu%26aufirst%3DB.%26aulast%3DKim%26aufirst%3DB.%2BY.%26aulast%3DKim%26aufirst%3DS.%2BJ.%26atitle%3DIdentification%2520of%2520novel%2520PTPRQ%2520phosphatase%2520inhibitors%2520based%2520on%2520the%2520virtual%2520screening%2520with%2520docking%2520simulations%26jtitle%3DTheor.%2520Biol.%2520Med.%2520Modell.%26date%3D2013%26volume%3D10%26spage%3D49%26doi%3D10.1186%2F1742-4682-10-49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref131"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref131'); return false;" data-citation="" class="refNumLink">131</a></strong><div class="NLM_citation" id="cit131"><span><span class="NLM_contrib-group">Hou, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, H.</span><span> </span><span class="NLM_article-title">Protein flexibility in docking-based virtual screening: discovery of novel lymphoid-specific tyrosine phosphatase inhibitors using multiple crystal structures</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">1973</span><span class="NLM_x">–</span> <span class="NLM_lpage">1983</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00344</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00344" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyntbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=1973-1983&author=X.+Houauthor=K.+Liauthor=X.+Yuauthor=J.+P.+Sunauthor=H.+Fang&title=Protein+flexibility+in+docking-based+virtual+screening%3A+discovery+of+novel+lymphoid-specific+tyrosine+phosphatase+inhibitors+using+multiple+crystal+structures&doi=10.1021%2Facs.jcim.5b00344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit131R"><div class="casContent"><span class="casTitleNuber">131</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Flexibility in Docking-Based Virtual Screening: Discovery of Novel Lymphoid-Specific Tyrosine Phosphatase Inhibitors Using Multiple Crystal Structures</span></div><div class="casAuthors">Hou, Xuben; Li, Kangshuai; Yu, Xiao; Sun, Jin-peng; Fang, Hao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1973-1983</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Incorporating protein flexibility is a major challenge for docking-based virtual screening.  With an increasing no. of available crystal structures, ensemble docking with multiple protein structures is an efficient approach to deal with protein flexibility.  Herein, the authors report the successful application of a docking-based virtual screen using multiple crystal structures to discover novel inhibitors of lymphoid-specific tyrosine phosphatase (LYP), a potential drug target for autoimmune diseases.  The appropriate use of multiple protein structures allowed a better enrichment than a single structure in the recovery of known inhibitors.  Subsequently, an optimal ensemble of LYP structures was selected and used in docking-based virtual screening.  Eight novel LYP inhibitors (IC50 ranging from 7.95 to 56.6 μM) were identified among 23 hit compds.  Further studies demonstrated that the most active compd. B15 possessed some selectivity over other protein phosphatases and could effectively up-regulate TCR (T cell receptor)-mediated signaling in Jurkat T cells.  These novel hits not only provided good starting points for the development of therapeutic agents useful in autoimmune diseases but also demonstrated the advantages of choosing an appropriate ensemble of protein structures in docking-based virtual screening over using a single protein conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiDcC42tLbKLVg90H21EOLACvtfcHk0lixdMGby2Yk-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyntbvM&md5=9296da92e46d98f8363906ff1c14fd70</span></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00344%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DJ.%2BP.%26aulast%3DFang%26aufirst%3DH.%26atitle%3DProtein%2520flexibility%2520in%2520docking-based%2520virtual%2520screening%253A%2520discovery%2520of%2520novel%2520lymphoid-specific%2520tyrosine%2520phosphatase%2520inhibitors%2520using%2520multiple%2520crystal%2520structures%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D1973%26epage%3D1983%26doi%3D10.1021%2Facs.jcim.5b00344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref132"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref132'); return false;" data-citation="" class="refNumLink">132</a></strong><div class="NLM_citation" id="cit132"><span><span class="NLM_contrib-group">Petersen, G. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saxena, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renuka, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yogeeswari, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santos, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bizarro, C. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sriram, D.</span><span> </span><span class="NLM_article-title">Structure-based virtual screening as a tool for the identification of novel inhibitors against Mycobacterium tuberculosis 3-dehydroquinate dehydratase</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">124</span><span class="NLM_x">–</span> <span class="NLM_lpage">131</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2015.05.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.jmgm.2015.05.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVaksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2015&pages=124-131&author=G.+O.+Petersenauthor=S.+Saxenaauthor=J.+Renukaauthor=V.+Soniauthor=P.+Yogeeswariauthor=D.+S.+Santosauthor=C.+V.+Bizarroauthor=D.+Sriram&title=Structure-based+virtual+screening+as+a+tool+for+the+identification+of+novel+inhibitors+against+Mycobacterium+tuberculosis+3-dehydroquinate+dehydratase&doi=10.1016%2Fj.jmgm.2015.05.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit132R"><div class="casContent"><span class="casTitleNuber">132</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based virtual screening as a tool for the identification of novel inhibitors against Mycobacterium tuberculosis 3-dehydroquinate dehydratase</span></div><div class="casAuthors">Petersen, Guilherme O.; Saxena, Shalini; Renuka, Janupally; Soni, Vijay; Yogeeswari, Perumal; Santos, Diogenes S.; Bizarro, Cristiano V.; Sriram, Dharmarajan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">124-131</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">3-Dehydroquinate dehydratase (DHQase), the third enzyme of the shikimate pathway, catalyzes the reversible reaction of 3-dehydroquinate into 3-dehydroshikimate.  The aim of the present study was to identify new drug-like mols. as inhibitors for Mycobacterium tuberculosis DHQase employing structure-based pharmacophore modeling technique using an in house database consisting of about 2500 small mols.  Further the pharmacophore models were validated using enrichment calcns., and finally three models were employed for high-throughput virtual screening and docking to identify novel small mols. as DHQase inhibitors.  Five compds. were identified, out of which, one mol. (Lead 1) showed 58% inhibition at 50 μM concn. in the Mtb DHQase assay.  Chem. derivs. of the Lead 1 when tested evolved top two hits with IC50s of 17.1 and 31.5 μM as well as MIC values of 25 and 6.25 μg/mL resp. and no cytotoxicity up to 100 μM concn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAcR8epVnSz7Vg90H21EOLACvtfcHk0lixdMGby2Yk-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVaksr4%253D&md5=318eb834c417191820854cd944ccb385</span></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2015.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2015.05.001%26sid%3Dliteratum%253Aachs%26aulast%3DPetersen%26aufirst%3DG.%2BO.%26aulast%3DSaxena%26aufirst%3DS.%26aulast%3DRenuka%26aufirst%3DJ.%26aulast%3DSoni%26aufirst%3DV.%26aulast%3DYogeeswari%26aufirst%3DP.%26aulast%3DSantos%26aufirst%3DD.%2BS.%26aulast%3DBizarro%26aufirst%3DC.%2BV.%26aulast%3DSriram%26aufirst%3DD.%26atitle%3DStructure-based%2520virtual%2520screening%2520as%2520a%2520tool%2520for%2520the%2520identification%2520of%2520novel%2520inhibitors%2520against%2520Mycobacterium%2520tuberculosis%25203-dehydroquinate%2520dehydratase%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2015%26volume%3D60%26spage%3D124%26epage%3D131%26doi%3D10.1016%2Fj.jmgm.2015.05.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref133"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref133'); return false;" data-citation="" class="refNumLink">133</a></strong><div class="NLM_citation" id="cit133"><span><span class="NLM_contrib-group">Park, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kyung, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoon, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryu, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeong, D. G.</span><span> </span><span class="NLM_article-title">Structure-based virtual screening approach to the discovery of novel PTPMT1 phosphatase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1271</span><span class="NLM_x">–</span> <span class="NLM_lpage">1275</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2011.10.083</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2011.10.083" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=1271-1275&author=H.+Parkauthor=S.+Y.+Kimauthor=A.+Kyungauthor=T.+S.+Yoonauthor=S.+E.+Ryuauthor=D.+G.+Jeong&title=Structure-based+virtual+screening+approach+to+the+discovery+of+novel+PTPMT1+phosphatase+inhibitors&doi=10.1016%2Fj.bmcl.2011.10.083"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.10.083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.10.083%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DS.%2BY.%26aulast%3DKyung%26aufirst%3DA.%26aulast%3DYoon%26aufirst%3DT.%2BS.%26aulast%3DRyu%26aufirst%3DS.%2BE.%26aulast%3DJeong%26aufirst%3DD.%2BG.%26atitle%3DStructure-based%2520virtual%2520screening%2520approach%2520to%2520the%2520discovery%2520of%2520novel%2520PTPMT1%2520phosphatase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D1271%26epage%3D1275%26doi%3D10.1016%2Fj.bmcl.2011.10.083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref134"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref134'); return false;" data-citation="" class="refNumLink">134</a></strong><div class="NLM_citation" id="cit134"><span><span class="NLM_contrib-group">Wolan, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zorn, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, J. A.</span><span> </span><span class="NLM_article-title">Small-molecule activators of a proenzyme</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">326</span><span class="NLM_x">, </span> <span class="NLM_fpage">853</span><span class="NLM_x">–</span> <span class="NLM_lpage">858</span><span class="refDoi"> DOI: 10.1126/science.1177585</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1126%2Fscience.1177585" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=19892984" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGhs7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=326&publication_year=2009&pages=853-858&author=D.+W.+Wolanauthor=J.+A.+Zornauthor=D.+C.+Grayauthor=J.+A.+Wells&title=Small-molecule+activators+of+a+proenzyme&doi=10.1126%2Fscience.1177585"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit134R"><div class="casContent"><span class="casTitleNuber">134</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Activators of a Proenzyme</span></div><div class="casAuthors">Wolan, Dennis W.; Zorn, Julie A.; Gray, Daniel C.; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">326</span>
        (<span class="NLM_cas:issue">5954</span>),
    <span class="NLM_cas:pages">853-858</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Virtually all of the 560 human proteases are stored as inactive proenzymes and are strictly regulated.  We report the identification and characterization of the first small mols. that directly activate proenzymes, the apoptotic procaspases-3 and -6.  It is surprising that these compds. induce autoproteolytic activation by stabilizing a conformation that is both more active and more susceptible to intermol. proteolysis.  These procaspase activators bypass the normal upstream proapoptotic signaling cascades and induce rapid apoptosis in a variety of cell lines.  Systematic biochem. and biophys. analyses identified a cluster of mutations in procaspase-3 that resist small-mol. activation both in vitro and in cells.  Compds. that induce gain of function are rare, and the activators reported here will enable direct control of the executioner caspases in apoptosis and in cellular differentiation.  More generally, these studies presage the discovery of other proenzyme activators to explore fundamental processes of proenzyme activation and their fate-detg. roles in biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrq87GHAd7ZRrVg90H21EOLACvtfcHk0lje_TNsVokM8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGhs7%252FK&md5=cdc123b08c2c8b84da15687638657c3a</span></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1126%2Fscience.1177585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1177585%26sid%3Dliteratum%253Aachs%26aulast%3DWolan%26aufirst%3DD.%2BW.%26aulast%3DZorn%26aufirst%3DJ.%2BA.%26aulast%3DGray%26aufirst%3DD.%2BC.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DSmall-molecule%2520activators%2520of%2520a%2520proenzyme%26jtitle%3DScience%26date%3D2009%26volume%3D326%26spage%3D853%26epage%3D858%26doi%3D10.1126%2Fscience.1177585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref135"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref135'); return false;" data-citation="" class="refNumLink">135</a></strong><div class="NLM_citation" id="cit135"><span><span class="NLM_contrib-group">Allen, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gochin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizzo, R. C.</span><span> </span><span class="NLM_article-title">Small molecule inhibitors of HIVgp41 N-heptad repeat trimer formation</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">2853</span><span class="NLM_x">–</span> <span class="NLM_lpage">2859</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2015.04.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2015.04.067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26013847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsFyiurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=2853-2859&author=W.+J.+Allenauthor=H.+A.+Yiauthor=M.+Gochinauthor=A.+Jacobsauthor=R.+C.+Rizzo&title=Small+molecule+inhibitors+of+HIVgp41+N-heptad+repeat+trimer+formation&doi=10.1016%2Fj.bmcl.2015.04.067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit135R"><div class="casContent"><span class="casTitleNuber">135</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibitors of HIVgp41 N-heptad repeat trimer formation</span></div><div class="casAuthors">Allen, William J.; Yi, Hyun Ah; Gochin, Miriam; Jacobs, Amy; Rizzo, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2853-2859</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Identification of mechanistically novel anti-HIV fusion inhibitors was accomplished using a computer-aided structure-based design approach with the goal of blocking the formation of the N-heptad repeat (NHR) trimer of the viral protein gp41.  A virtual screening strategy that included per-residue interaction patterns (footprints) was employed to identify small mols. compatible with putative binding pockets at the internal interface of the NHR helixes at the core native viral six-helix bundle.  From a screen of ∼2.8 million compds. using the DOCK program, 120 with favorable energetic and footprint overlap characteristics were purchased and exptl. tested leading to two compds. with favorable cell-cell fusion (IC50) and cytotoxicity profiles.  Importantly, both hits were identified on the basis of scores contg. footprint overlap terms and would not have been identified using the std. DOCK energy function alone.  To our knowledge, these compds. represent the first reported small mols. that inhibit viral entry via the proposed NHR-trimer obstruction mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwtzEm7EC52bVg90H21EOLACvtfcHk0lje_TNsVokM8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsFyiurs%253D&md5=a46287d045987ac50e65263481f263ba</span></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.04.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.04.067%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DW.%2BJ.%26aulast%3DYi%26aufirst%3DH.%2BA.%26aulast%3DGochin%26aufirst%3DM.%26aulast%3DJacobs%26aufirst%3DA.%26aulast%3DRizzo%26aufirst%3DR.%2BC.%26atitle%3DSmall%2520molecule%2520inhibitors%2520of%2520HIVgp41%2520N-heptad%2520repeat%2520trimer%2520formation%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D2853%26epage%3D2859%26doi%3D10.1016%2Fj.bmcl.2015.04.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref136"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref136'); return false;" data-citation="" class="refNumLink">136</a></strong><div class="NLM_citation" id="cit136"><span><span class="NLM_contrib-group">Dong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jing, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Y.</span><span> </span><span class="NLM_article-title">Pharmacophore identification, virtual screening and biological evaluation of prenylated flavonoids derivatives as PKB/Akt1 inhibitors</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">5949</span><span class="NLM_x">–</span> <span class="NLM_lpage">5958</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2011.10.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.ejmech.2011.10.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=22027102" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVKqtLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2011&pages=5949-5958&author=X.+Dongauthor=X.+Zhouauthor=H.+Jingauthor=J.+Chenauthor=T.+Liuauthor=B.+Yangauthor=Q.+Heauthor=Y.+Hu&title=Pharmacophore+identification%2C+virtual+screening+and+biological+evaluation+of+prenylated+flavonoids+derivatives+as+PKB%2FAkt1+inhibitors&doi=10.1016%2Fj.ejmech.2011.10.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit136R"><div class="casContent"><span class="casTitleNuber">136</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacophore identification, virtual screening and biological evaluation of prenylated flavonoids derivatives as PKB/Akt1 inhibitors</span></div><div class="casAuthors">Dong, Xiaowu; Zhou, Xinglu; Jing, Hui; Chen, Jianzhong; Liu, Tao; Yang, Bo; He, Qiaojun; Hu, Yongzhou</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5949-5958</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A total of 24 well-defined PKB/Akt1 inhibitors were used to generate pharmacophore models applying Catalyst/HypoGen program.  The best ranked model (Hypo_1) was then validated by cost anal., prediction capability, Cat-Scramble and receiver operating characteristic (ROC) studies.  Then, pharmacophore-based virtual screening combined with docking study was performed to search an inhouse compd. database.  Nine preferable hits 75-80, HTS-02143, BTB-14740 and HTS-08006 were prepd. and biol. evaluated.  Several compds. were identified as good PKB/Akt1 inhibitors, suggesting that Hypo_1 would be reliable and useful in virtual screening.  Flow cytometric and western blotting anal. on compds. 79 and 80 further demonstrated that the inhibition of phosphorylation of PKB/Akt1 and its substrates (such as GSK3β) was responsible for their cytotoxic activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXTEc-nFDC_rVg90H21EOLACvtfcHk0lje_TNsVokM8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVKqtLnE&md5=e563fb1a3ea5bbbc7c1ebff8753f38cf</span></div><a href="/servlet/linkout?suffix=cit136&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2011.10.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2011.10.006%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DJing%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DHe%26aufirst%3DQ.%26aulast%3DHu%26aufirst%3DY.%26atitle%3DPharmacophore%2520identification%252C%2520virtual%2520screening%2520and%2520biological%2520evaluation%2520of%2520prenylated%2520flavonoids%2520derivatives%2520as%2520PKB%252FAkt1%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2011%26volume%3D46%26spage%3D5949%26epage%3D5958%26doi%3D10.1016%2Fj.ejmech.2011.10.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref137"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref137'); return false;" data-citation="" class="refNumLink">137</a></strong><div class="NLM_citation" id="cit137"><span><span class="NLM_contrib-group">Zhong, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, Z. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span> </span><span class="NLM_article-title">Structure-based discovery of an immunomodulatory inhibitor of TLR1-TLR2 heterodimerization from a natural product-like database</span> <span class="citation_source-journal">Chem. Commun. (Cambridge, U. K.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">11178</span><span class="NLM_x">–</span> <span class="NLM_lpage">11181</span><span class="refDoi"> DOI: 10.1039/C5CC02728D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1039%2FC5CC02728D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26051605" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXoslOnurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2015&pages=11178-11181&author=Z.+Zhongauthor=L.+J.+Liuauthor=Z.+Q.+Dongauthor=L.+Luauthor=M.+Wangauthor=C.+H.+Leungauthor=D.+L.+Maauthor=Y.+Wang&title=Structure-based+discovery+of+an+immunomodulatory+inhibitor+of+TLR1-TLR2+heterodimerization+from+a+natural+product-like+database&doi=10.1039%2FC5CC02728D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit137R"><div class="casContent"><span class="casTitleNuber">137</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of an immunomodulatory inhibitor of TLR1-TLR2 heterodimerization from a natural product-like database</span></div><div class="casAuthors">Zhong, Zhangfeng; Liu, Li-Juan; Dong, Zhi-Qiang; Lu, Lihua; Wang, Modi; Leung, Chung-Hang; Ma, Dik-Lung; Wang, Yitao</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">56</span>),
    <span class="NLM_cas:pages">11178-11181</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">We report herein the identification of an immunomodulatory natural product-like compd. 1 as a direct inhibitor of TLR1-TLR2 heterodimerization.  Compd. 1 suppressed TNF-α and IL-6 secretion in Pam3CSK4-induced macrophages.  Moreover, compd. 1 inhibited the phagocytic activity of macrophages, presumably through modulation of TLR1-TLR2 signaling and inactivation of NF-κB.  Mol. docking revealed that compd. 1 bound to the interface region of TLR1-TLR2 by forming two hydrogen bonds with residues lining the binding site.  To our knowledge, compd. 1 has been only the second inhibitor overall of TLR1-TLR2 heterodimerization reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqurqIe4Y09JLVg90H21EOLACvtfcHk0liF6zPU2Cnx0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXoslOnurk%253D&md5=89503b243b836322a73a541d22c56e39</span></div><a href="/servlet/linkout?suffix=cit137&amp;dbid=16384&amp;doi=10.1039%2FC5CC02728D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5CC02728D%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DL.%2BJ.%26aulast%3DDong%26aufirst%3DZ.%2BQ.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLeung%26aufirst%3DC.%2BH.%26aulast%3DMa%26aufirst%3DD.%2BL.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DStructure-based%2520discovery%2520of%2520an%2520immunomodulatory%2520inhibitor%2520of%2520TLR1-TLR2%2520heterodimerization%2520from%2520a%2520natural%2520product-like%2520database%26jtitle%3DChem.%2520Commun.%2520%2528Cambridge%252C%2520U.%2520K.%2529%26date%3D2015%26volume%3D51%26spage%3D11178%26epage%3D11181%26doi%3D10.1039%2FC5CC02728D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref138"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref138'); return false;" data-citation="" class="refNumLink">138</a></strong><div class="NLM_citation" id="cit138"><span><span class="NLM_contrib-group">Mohammed, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Numair, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balakrishnan, A.</span><span> </span><span class="NLM_article-title">Docking studies on the interaction of flavonoids with fat mass and obesity associated protein</span> <span class="citation_source-journal">Pak. J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1647</span><span class="NLM_x">–</span> <span class="NLM_lpage">1653</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26408884" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BC283mvFKjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=1647-1653&author=A.+Mohammedauthor=K.+S.+Al-Numairauthor=A.+Balakrishnan&title=Docking+studies+on+the+interaction+of+flavonoids+with+fat+mass+and+obesity+associated+protein"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit138R"><div class="casContent"><span class="casTitleNuber">138</span><div class="casTitle"><span class="NLM_cas:atitle">Docking studies on the interaction of flavonoids with fat mass and obesity associated protein</span></div><div class="casAuthors">Mohammed Alamri; Al-Numair Khalid Suliman; Balakrishnan Aristatile</div><div class="citationInfo"><span class="NLM_cas:title">Pakistan journal of pharmaceutical sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1647-53</span>
        ISSN:<span class="NLM_cas:issn">1011-601X</span>.
    </div><div class="casAbstract">Obesity is the excessive fat accumulation in human body leading to increases a risk of various chronic diseases such as diabetes, cardiovascular diseases, cancer and osteoarthritis.  Several flavonoids are known to have lipolytic activity influencing lipolysis and adipogenesis in adipose cells.  To explore mechanism of the association of flavonoids in obesity and obesity associated protein (FTO), molecular docking studies were done for FTO with flavonoids, with orlistat (antiobesity drug) as a control.  Autodock tools were used for docking flavonoids and orlistat with FTO.  The results were visualized by PyMol and Discovery studio visualizer.  Upon docking simulation, it was observed that flavonoid quercetin showed highest binding affinity (most negative δG), whereas daidzein was least affinity towards FTO.  The binding affinity of other flavonoids was in the order of Exemestane >Kaempherol >Letrozole >Rutin.  This study concludes that flavonoids primarily, quercetin ameliorates obesity by establishing a physical interaction with FTO.  Interactions were also observed between FTO and other flavonoids and were of not greater inhibition compared to quercetin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRUkpRKZyo6CTdblLE_E9PWfW6udTcc2eYQ8KdVqc-_8rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC283mvFKjuw%253D%253D&md5=f261e8d6e1caaa58a9f7923281adbb61</span></div><a href="/servlet/linkout?suffix=cit138&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMohammed%26aufirst%3DA.%26aulast%3DAl-Numair%26aufirst%3DK.%2BS.%26aulast%3DBalakrishnan%26aufirst%3DA.%26atitle%3DDocking%2520studies%2520on%2520the%2520interaction%2520of%2520flavonoids%2520with%2520fat%2520mass%2520and%2520obesity%2520associated%2520protein%26jtitle%3DPak.%2520J.%2520Pharm.%2520Sci.%26date%3D2015%26volume%3D28%26spage%3D1647%26epage%3D1653" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref139"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref139'); return false;" data-citation="" class="refNumLink">139</a></strong><div class="NLM_citation" id="cit139"><span><span class="NLM_contrib-group">Brecher, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banavali, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kramer, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span> </span><span class="NLM_article-title">Novel broad spectrum inhibitors targeting the flavivirus methyltransferase</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0130062</span><span class="refDoi"> DOI: 10.1371/journal.pone.0130062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pone.0130062" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0130062&author=M.+Brecherauthor=H.+Chenauthor=B.+Liuauthor=N.+K.+Banavaliauthor=S.+A.+Jonesauthor=J.+Zhangauthor=Z.+Liauthor=L.+D.+Kramerauthor=H.+Li&title=Novel+broad+spectrum+inhibitors+targeting+the+flavivirus+methyltransferase&doi=10.1371%2Fjournal.pone.0130062"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit139&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0130062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0130062%26sid%3Dliteratum%253Aachs%26aulast%3DBrecher%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DBanavali%26aufirst%3DN.%2BK.%26aulast%3DJones%26aufirst%3DS.%2BA.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DKramer%26aufirst%3DL.%2BD.%26aulast%3DLi%26aufirst%3DH.%26atitle%3DNovel%2520broad%2520spectrum%2520inhibitors%2520targeting%2520the%2520flavivirus%2520methyltransferase%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0130062%26doi%3D10.1371%2Fjournal.pone.0130062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref140"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref140'); return false;" data-citation="" class="refNumLink">140</a></strong><div class="NLM_citation" id="cit140"><span><span class="NLM_contrib-group">Walters, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Namchuk, M.</span><span> </span><span class="NLM_article-title">Designing screens: how to make your hits a hit</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">259</span><span class="NLM_x">–</span> <span class="NLM_lpage">266</span><span class="refDoi"> DOI: 10.1038/nrd1063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnrd1063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=12669025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3sXisFWktrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=259-266&author=W.+Waltersauthor=M.+Namchuk&title=Designing+screens%3A+how+to+make+your+hits+a+hit&doi=10.1038%2Fnrd1063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit140R"><div class="casContent"><span class="casTitleNuber">140</span><div class="casTitle"><span class="NLM_cas:atitle">A guide to drug discovery: Designing screens: how to make your hits a hit</span></div><div class="casAuthors">Walters, W. Patrick; Namchuk, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">259-266</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The basic goal of small-mol. screening is the identification of chem. interesting starting points for elaboration towards a drug.  A no. of innovative approaches for pursuing this goal have evolved, and the right approach is dictated by the target class being pursued and the capabilities of the organization involved.hood.  A recent trend in high-throughput screening has been to place less emphasis on the no. of data points that can be produced, and to focus instead on the quality of the data obtained.  Several computational and technol. advances have aided in the selection of compds. for screening and widened the variety of assay formats available for screening.  The effect on the efficiency of the screening process is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ5E12ik7BrbVg90H21EOLACvtfcHk0liF6zPU2Cnx0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXisFWktrk%253D&md5=6bec4547b8dce69ab84ce226cde28f8a</span></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.1038%2Fnrd1063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1063%26sid%3Dliteratum%253Aachs%26aulast%3DWalters%26aufirst%3DW.%26aulast%3DNamchuk%26aufirst%3DM.%26atitle%3DDesigning%2520screens%253A%2520how%2520to%2520make%2520your%2520hits%2520a%2520hit%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D259%26epage%3D266%26doi%3D10.1038%2Fnrd1063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref141"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref141'); return false;" data-citation="" class="refNumLink">141</a></strong><div class="NLM_citation" id="cit141"><span><span class="NLM_contrib-group">Thorne, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lea, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lovell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auld, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span> </span><span class="NLM_article-title">Firefly luciferase in chemical biology: a compendium of inhibitors, mechanistic evaluation of chemotypes, and suggested use as a reporter</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1060</span><span class="NLM_x">–</span> <span class="NLM_lpage">1072</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2012.07.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.chembiol.2012.07.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=22921073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1KkurjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1060-1072&author=N.+Thorneauthor=M.+Shenauthor=W.+A.+Leaauthor=A.+Simeonovauthor=S.+Lovellauthor=D.+S.+Auldauthor=J.+Inglese&title=Firefly+luciferase+in+chemical+biology%3A+a+compendium+of+inhibitors%2C+mechanistic+evaluation+of+chemotypes%2C+and+suggested+use+as+a+reporter&doi=10.1016%2Fj.chembiol.2012.07.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit141R"><div class="casContent"><span class="casTitleNuber">141</span><div class="casTitle"><span class="NLM_cas:atitle">Firefly Luciferase in Chemical Biology: A Compendium of Inhibitors, Mechanistic Evaluation of Chemotypes, and Suggested Use As a Reporter</span></div><div class="casAuthors">Thorne, Natasha; Shen, Min; Lea, Wendy A.; Simeonov, Anton; Lovell, Scott; Auld, Douglas S.; Inglese, James</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1060-1072</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Firefly luciferase (FLuc) is frequently used as a reporter in high-throughput screening assays, owing to the exceptional sensitivity, dynamic range, and rapid measurement that bioluminescence affords.  However, interaction of small mols. with FLuc has, to some extent, confounded its use in chem. biol. and drug discovery.  To identify and characterize chemotypes interacting with FLuc, we detd. potency values for 360,864 compds. found in the NIH Mol. Libraries Small Mol. Repository, available in PubChem.  FLuc inhibitory activity was obsd. for 12% of this library with discernible SAR.  Characterization of 151 inhibitors demonstrated a variety of inhibition modes, including FLuc-catalyzed formation of multisubstrate adduct enzyme inhibitor complexes.  As in some cell-based FLuc reporter assays, compds. acting as FLuc inhibitors yield paradoxical luminescence increases, thus data on compds. acquired from FLuc-dependent assays require careful anal. as described here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWw4FbLlFcILVg90H21EOLACvtfcHk0liF6zPU2Cnx0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1KkurjM&md5=5ab0f756ab0a3ece1ea28ffac52bbd5f</span></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.07.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.07.015%26sid%3Dliteratum%253Aachs%26aulast%3DThorne%26aufirst%3DN.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DLea%26aufirst%3DW.%2BA.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DInglese%26aufirst%3DJ.%26atitle%3DFirefly%2520luciferase%2520in%2520chemical%2520biology%253A%2520a%2520compendium%2520of%2520inhibitors%252C%2520mechanistic%2520evaluation%2520of%2520chemotypes%252C%2520and%2520suggested%2520use%2520as%2520a%2520reporter%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D1060%26epage%3D1072%26doi%3D10.1016%2Fj.chembiol.2012.07.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref142"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref142'); return false;" data-citation="" class="refNumLink">142</a></strong><div class="NLM_citation" id="cit142"><span><span class="NLM_contrib-group">Feng, B. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">A detergent-based assay for the detection of promiscuous inhibitors</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">550</span><span class="NLM_x">–</span> <span class="NLM_lpage">553</span><span class="refDoi"> DOI: 10.1038/nprot.2006.77</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnprot.2006.77" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=17191086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFOitLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=550-553&author=B.+Y.+Fengauthor=B.+K.+Shoichet&title=A+detergent-based+assay+for+the+detection+of+promiscuous+inhibitors&doi=10.1038%2Fnprot.2006.77"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit142R"><div class="casContent"><span class="casTitleNuber">142</span><div class="casTitle"><span class="NLM_cas:atitle">A detergent-based assay for the detection of promiscuous inhibitors</span></div><div class="casAuthors">Feng, Brian Y.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">550-553</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">At micromolar concns., many small mols. self-assoc. into colloidal aggregates that non-specifically inhibit enzymes and other proteins.  Here the authors describe a protocol for identifying aggregate-based inhibitors and distinguishing them from small mols. that inhibit via specific mechanisms.  As a convenient proxy for promiscuous, aggregate-based inhibition, the authors monitor inhibition of β-lactamase in the absence and presence of detergent.  Inhibition that is attenuated in the presence of detergent is characteristic of an aggregate-based mechanism.  In the 96-well-format assay described here, about 200 mols. can be tested, in duplicate, per h for detergent-dependent sensitivity.  Furthermore, the authors also describe simple expts. that can offer addnl. confirmation of aggregate-based inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplGQSBvMgM-7Vg90H21EOLACvtfcHk0liHjHh9uZ9hYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFOitLjO&md5=8aa7562b5f69c896af7e133ecc35e6de</span></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2006.77&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2006.77%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DB.%2BY.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DA%2520detergent-based%2520assay%2520for%2520the%2520detection%2520of%2520promiscuous%2520inhibitors%26jtitle%3DNat.%2520Protoc.%26date%3D2006%26volume%3D1%26spage%3D550%26epage%3D553%26doi%3D10.1038%2Fnprot.2006.77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref143"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref143'); return false;" data-citation="" class="refNumLink">143</a></strong><div class="NLM_citation" id="cit143"><span><span class="NLM_contrib-group">Sassano, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doak, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Colloidal aggregation causes inhibition of G protein-coupled receptors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2406</span><span class="NLM_x">–</span> <span class="NLM_lpage">2414</span><span class="refDoi"> DOI: 10.1021/jm301749y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301749y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVCgt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2406-2414&author=M.+F.+Sassanoauthor=A.+K.+Doakauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Colloidal+aggregation+causes+inhibition+of+G+protein-coupled+receptors&doi=10.1021%2Fjm301749y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit143R"><div class="casContent"><span class="casTitleNuber">143</span><div class="casTitle"><span class="NLM_cas:atitle">Colloidal aggregation causes inhibition of G protein-coupled receptors</span></div><div class="casAuthors">Sassano, Maria F.; Doak, Allison K.; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2406-2414</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Colloidal aggregation is the dominant mechanism for artifactual inhibition of sol. proteins, and controls against it are now widely deployed.  Conversely, investigating this mechanism for membrane-bound receptors has proven difficult.  Here, the authors investigated the activity of 4 well-characterized aggregators against 3 G protein-coupled receptors (GPCRs) recognizing peptide and protein ligands.  Each of the aggregators was active at micromolar concns. against the 3 GPCRs in cell-based assays.  This activity could be attenuated by either centrifugation of the inhibitor stock soln. or by the addn. of Tween-80 detergent.  In the absence of agonist, the aggregators acted as inverse agonists, consistent with a direct receptor interaction.  Meanwhile, several literature GPCR ligands that resembled aggregators themselves formed colloids, by both phys. and enzymol. tests.  These observations suggest that some GPCRs may be artifactually antagonized by colloidal aggregates, an effect that merits the attention of investigators in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojG9CcuOYoXrVg90H21EOLACvtfcHk0liHjHh9uZ9hYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVCgt7s%253D&md5=84fa59f9635330739bf836f8085be5e0</span></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.1021%2Fjm301749y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301749y%26sid%3Dliteratum%253Aachs%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DColloidal%2520aggregation%2520causes%2520inhibition%2520of%2520G%2520protein-coupled%2520receptors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2406%26epage%3D2414%26doi%3D10.1021%2Fjm301749y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref144"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref144'); return false;" data-citation="" class="refNumLink">144</a></strong><div class="NLM_citation" id="cit144"><span><span class="NLM_contrib-group">Owen, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doak, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wassam, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Colloidal aggregation affects the efficacy of anticancer drugs in cell culture</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1429</span><span class="NLM_x">–</span> <span class="NLM_lpage">1435</span><span class="refDoi"> DOI: 10.1021/cb300189b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300189b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsFOmsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1429-1435&author=S.+C.+Owenauthor=A.+K.+Doakauthor=P.+Wassamauthor=M.+S.+Shoichetauthor=B.+K.+Shoichet&title=Colloidal+aggregation+affects+the+efficacy+of+anticancer+drugs+in+cell+culture&doi=10.1021%2Fcb300189b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit144R"><div class="casContent"><span class="casTitleNuber">144</span><div class="casTitle"><span class="NLM_cas:atitle">Colloidal Aggregation Affects the Efficacy of Anticancer Drugs in Cell Culture</span></div><div class="casAuthors">Owen, Shawn C.; Doak, Allison K.; Wassam, Pascal; Shoichet, Molly S.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1429-1435</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many small mols., including bioactive mols. and approved drugs, spontaneously form colloidal aggregates in aq. soln. at micromolar concns.  Though it is widely accepted that aggregation leads to artifacts in screens for ligands of sol. proteins, the effects of colloid formation in cell-based assays have not been studied.  Here, seven anticancer drugs and one diagnostic reagent were found to form colloids in both biochem. buffer and in cell culture media.  In cell-based assays, the antiproliferative activities of three of the drugs were substantially reduced when in colloidal form as compared to monomeric form; a new formulation method ensured the presence of drug colloids vs. drug monomers in soln.  We also found that Evans Blue, a dye classically used to measure vascular permeability and to demonstrate the "enhanced permeability and retention (EPR) effect" in solid tumors, forms colloids that adsorb albumin, as opposed to older literature that suggested the reverse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN47RUN9WrPrVg90H21EOLACvtfcHk0liHjHh9uZ9hYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsFOmsrs%253D&md5=207a6f05e2ee718392562b28d2c52f89</span></div><a href="/servlet/linkout?suffix=cit144&amp;dbid=16384&amp;doi=10.1021%2Fcb300189b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300189b%26sid%3Dliteratum%253Aachs%26aulast%3DOwen%26aufirst%3DS.%2BC.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DWassam%26aufirst%3DP.%26aulast%3DShoichet%26aufirst%3DM.%2BS.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DColloidal%2520aggregation%2520affects%2520the%2520efficacy%2520of%2520anticancer%2520drugs%2520in%2520cell%2520culture%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1429%26epage%3D1435%26doi%3D10.1021%2Fcb300189b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref145"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref145'); return false;" data-citation="" class="refNumLink">145</a></strong><div class="NLM_citation" id="cit145"><span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Interpreting steep dose-response curves in early inhibitor discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">7274</span><span class="NLM_x">–</span> <span class="NLM_lpage">7277</span><span class="refDoi"> DOI: 10.1021/jm061103g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm061103g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtF2it7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=7274-7277&author=B.+K.+Shoichet&title=Interpreting+steep+dose-response+curves+in+early+inhibitor+discovery&doi=10.1021%2Fjm061103g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit145R"><div class="casContent"><span class="casTitleNuber">145</span><div class="casTitle"><span class="NLM_cas:atitle">Interpreting Steep Dose-Response Curves in Early Inhibitor Discovery</span></div><div class="casAuthors">Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">7274-7277</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many screening hits inhibit enzymes with steep dose-response curves, which are considered pathol.  Three models might explain these curves: multisite binding, an inhibitor phase transition, or stoichiometric inhibition caused by a high enzyme to Kd ratio.  Expts. with promiscuous aggregators, for which steep curves are common, suggest that these curves owe to stoichiometric inhibition, which predicts that IC50 should vary linearly with enzyme concn.  Most steep dose-response curves in screening may be due to this effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDnPkuWbZ-YLVg90H21EOLACvtfcHk0lhTnU-vhirkRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtF2it7fF&md5=97975678ee10f97457dccde7d15173ee</span></div><a href="/servlet/linkout?suffix=cit145&amp;dbid=16384&amp;doi=10.1021%2Fjm061103g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061103g%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DInterpreting%2520steep%2520dose-response%2520curves%2520in%2520early%2520inhibitor%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D7274%26epage%3D7277%26doi%3D10.1021%2Fjm061103g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref146"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref146'); return false;" data-citation="" class="refNumLink">146</a></strong><div class="NLM_citation" id="cit146"><span><span class="NLM_contrib-group">Fukunishi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, H.</span><span> </span><span class="NLM_article-title">Improved estimation of protein-ligand binding free energy by using the ligand-entropy and mobility of water molecules</span> <span class="citation_source-journal">Pharmaceuticals</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">604</span><span class="NLM_x">–</span> <span class="NLM_lpage">622</span><span class="refDoi"> DOI: 10.3390/ph6050604</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.3390%2Fph6050604" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2013&pages=604-622&author=Y.+Fukunishiauthor=H.+Nakamura&title=Improved+estimation+of+protein-ligand+binding+free+energy+by+using+the+ligand-entropy+and+mobility+of+water+molecules&doi=10.3390%2Fph6050604"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit146&amp;dbid=16384&amp;doi=10.3390%2Fph6050604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fph6050604%26sid%3Dliteratum%253Aachs%26aulast%3DFukunishi%26aufirst%3DY.%26aulast%3DNakamura%26aufirst%3DH.%26atitle%3DImproved%2520estimation%2520of%2520protein-ligand%2520binding%2520free%2520energy%2520by%2520using%2520the%2520ligand-entropy%2520and%2520mobility%2520of%2520water%2520molecules%26jtitle%3DPharmaceuticals%26date%3D2013%26volume%3D6%26spage%3D604%26epage%3D622%26doi%3D10.3390%2Fph6050604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref147"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref147'); return false;" data-citation="" class="refNumLink">147</a></strong><div class="NLM_citation" id="cit147"><span><span class="NLM_contrib-group">Barillari, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Essex, J. W.</span><span> </span><span class="NLM_article-title">Classification of water molecules in protein binding sites</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">2577</span><span class="NLM_x">–</span> <span class="NLM_lpage">2587</span><span class="refDoi"> DOI: 10.1021/ja066980q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja066980q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsFGjt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=2577-2587&author=C.+Barillariauthor=J.+Taylorauthor=R.+Vinerauthor=J.+W.+Essex&title=Classification+of+water+molecules+in+protein+binding+sites&doi=10.1021%2Fja066980q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit147R"><div class="casContent"><span class="casTitleNuber">147</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of Water Molecules in Protein Binding Sites</span></div><div class="casAuthors">Barillari, Caterina; Taylor, Justine; Viner, Russell; Essex, Jonathan W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2577-2587</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Water mols. play a crucial role in mediating the interaction between a ligand and a macromol. receptor.  An understanding of the nature and role of each water mol. in the active site of a protein could greatly increase the efficiency of rational drug design approaches: if the propensity of a water mol. for displacement can be detd., then synthetic effort may be most profitably applied to the design of specific ligands with the displacement of this water mol. in mind.  In this paper, a thermodn. anal. of water mols. in the binding sites of six proteins, each complexed with a no. of inhibitors, is presented.  Two classes of water mols. were identified: those conserved and not displaced by any of the ligands, and those that are displaced by some ligands.  The abs. binding free energies of 54 water mols. were calcd. using the double decoupling method, with replica exchange thermodn. integration in Monte Carlo simulations.  It was found that conserved water mols. are on av. more tightly bound than displaced water mols.  In addn., Bayesian statistics is used to calc. the probability that a particular water mol. may be displaced by an appropriately designed ligand, given the calcd. binding free energy of the water mol.  This approach therefore allows the numerical assessment of whether or not a given water mol. should be targeted for displacement as part of a rational drug design strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr38ETxUkD-NrVg90H21EOLACvtfcHk0lhTnU-vhirkRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsFGjt7o%253D&md5=e74664f19993a9e9c087dbb9c1d54c3e</span></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1021%2Fja066980q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja066980q%26sid%3Dliteratum%253Aachs%26aulast%3DBarillari%26aufirst%3DC.%26aulast%3DTaylor%26aufirst%3DJ.%26aulast%3DViner%26aufirst%3DR.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26atitle%3DClassification%2520of%2520water%2520molecules%2520in%2520protein%2520binding%2520sites%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D2577%26epage%3D2587%26doi%3D10.1021%2Fja066980q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref148"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref148'); return false;" data-citation="" class="refNumLink">148</a></strong><div class="NLM_citation" id="cit148"><span><span class="NLM_contrib-group">Michel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Prediction of the water content in protein binding sites</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">113</span><span class="NLM_x">, </span> <span class="NLM_fpage">13337</span><span class="NLM_x">–</span> <span class="NLM_lpage">13346</span><span class="refDoi"> DOI: 10.1021/jp9047456</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp9047456" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFegsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=13337-13346&author=J.+Michelauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Prediction+of+the+water+content+in+protein+binding+sites&doi=10.1021%2Fjp9047456"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit148R"><div class="casContent"><span class="casTitleNuber">148</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of the Water Content in Protein Binding Sites</span></div><div class="casAuthors">Michel, Julien; Tirado-Rives, Julian; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">13337-13346</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An efficient mol. simulation methodol. has been developed to det. the positioning of water mols. in the binding site of a protein or protein-ligand complex.  Occupancies and abs. binding free energies of water mols. are computed using a statistical thermodn. approach.  The methodol., referred to as Just Add Water Mols. (JAWS), features "θ-water" mols. that can appear and disappear on a binding-site grid.  Key approxns. render the technique far more efficient than conventional free energy simulations.  Testing of JAWS on five diverse examples (neuraminidase, scytalone dehydratase, major urinary protein 1, β-lactoglobulin, and COX-2) demonstrates its accuracy in locating hydration sites in comparison to results from high-resoln. crystal structures.  Possible applications include aid in refinement of protein crystal structures, drug lead optimization, setup of docking calcns., and simulations of protein-ligand complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC6L_AGV7e7LVg90H21EOLACvtfcHk0lhTnU-vhirkRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFegsr7I&md5=bc36a74922d2b1ce1d15e53b9e144afe</span></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1021%2Fjp9047456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp9047456%26sid%3Dliteratum%253Aachs%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DPrediction%2520of%2520the%2520water%2520content%2520in%2520protein%2520binding%2520sites%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2009%26volume%3D113%26spage%3D13337%26epage%3D13346%26doi%3D10.1021%2Fjp9047456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref149"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref149'); return false;" data-citation="" class="refNumLink">149</a></strong><div class="NLM_citation" id="cit149"><span><span class="NLM_contrib-group">Abel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salam, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farid, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span> </span><span class="NLM_article-title">Contribution of explicit solvent effects to the binding affinity of small-molecule inhibitors in blood coagulation factor serine proteases</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1049</span><span class="NLM_x">–</span> <span class="NLM_lpage">1066</span><span class="refDoi"> DOI: 10.1002/cmdc.201000533</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fcmdc.201000533" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=21506273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmsFGjs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=1049-1066&author=R.+Abelauthor=N.+K.+Salamauthor=J.+Shelleyauthor=R.+Faridauthor=R.+A.+Friesnerauthor=W.+Sherman&title=Contribution+of+explicit+solvent+effects+to+the+binding+affinity+of+small-molecule+inhibitors+in+blood+coagulation+factor+serine+proteases&doi=10.1002%2Fcmdc.201000533"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit149R"><div class="casContent"><span class="casTitleNuber">149</span><div class="casTitle"><span class="NLM_cas:atitle">Contribution of Explicit Solvent Effects to the Binding Affinity of Small-Molecule Inhibitors in Blood Coagulation Factor Serine Proteases</span></div><div class="casAuthors">Abel, Robert; Salam, Noeris K.; Shelley, John; Farid, Ramy; Friesner, Richard A.; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1049-1066</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The prevention of blood coagulation is important in treating thromboembolic disorders, and several serine proteases involved in the coagulation cascade have been classified as pharmaceutically relevant.  Whereas structure-based drug design has contributed to the development of some serine protease inhibitors, traditional computational methods have not been able to fully describe structure-activity relationships (SAR).  Here, we study the SAR for a no. of serine proteases by using a method that calcs. the thermodn. properties (enthalpy and entropy) of the water that solvates the active site.  We show that the displacement of water from specific subpockets (such as S1-4 and the ester binding pocket) of the active site by the ligand can govern potency, esp. for cases in which small chem. changes (i.e., a Me group or halogen) result in a substantial increase in potency.  Furthermore, we describe how relative binding free energies can be estd. by combining the water displacement energy with complementary terms from an implicit solvent mol. mechanics description binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrr_5CTLjkC3LVg90H21EOLACvtfcHk0lgb2vig4D9POg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmsFGjs7c%253D&md5=d432b1e0f07de0702e2320284e131a9c</span></div><a href="/servlet/linkout?suffix=cit149&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201000533&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201000533%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DSalam%26aufirst%3DN.%2BK.%26aulast%3DShelley%26aufirst%3DJ.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DContribution%2520of%2520explicit%2520solvent%2520effects%2520to%2520the%2520binding%2520affinity%2520of%2520small-molecule%2520inhibitors%2520in%2520blood%2520coagulation%2520factor%2520serine%2520proteases%26jtitle%3DChemMedChem%26date%3D2011%26volume%3D6%26spage%3D1049%26epage%3D1066%26doi%3D10.1002%2Fcmdc.201000533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref150"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref150'); return false;" data-citation="" class="refNumLink">150</a></strong><div class="NLM_citation" id="cit150"><span><span class="NLM_contrib-group">Zheng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, J.</span><span> </span><span class="NLM_article-title">Water PMF for predicting the properties of water molecules in protein binding site</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">583</span><span class="NLM_x">–</span> <span class="NLM_lpage">592</span><span class="refDoi"> DOI: 10.1002/jcc.23170</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fjcc.23170" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=23114863" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1WjsrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2013&pages=583-592&author=M.+Zhengauthor=Y.+Liauthor=B.+Xiongauthor=H.+Jiangauthor=J.+Shen&title=Water+PMF+for+predicting+the+properties+of+water+molecules+in+protein+binding+site&doi=10.1002%2Fjcc.23170"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit150R"><div class="casContent"><span class="casTitleNuber">150</span><div class="casTitle"><span class="NLM_cas:atitle">Water PMF for predicting the properties of water molecules in protein binding site</span></div><div class="casAuthors">Zheng, Mingyue; Li, Yanlian; Xiong, Bing; Jiang, Hualiang; Shen, Jingkang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">583-592</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Water is an important component in living systems and deserves better understanding in chem. and biol.  However, due to the difficulty of investigating the water functions in protein structures, it is usually ignored in computational modeling, esp. in the field of computer-aided drug design.  Here, using the potential of mean forces (PMFs) approach, we constructed a water PMF (wPMF) based on 3946 non-redundant high resoln. crystal structures.  The extd. wPMF potential was first used to investigate the structure pattern of water and analyze the residue hydrophilicity.  Then, the relationship between wPMF score and the B factor value of crystal waters was studied.  It was found that wPMF agrees well with some previously reported exptl. observations.  In addn., the wPMF score was also tested in parallel with 3D-RISM to measure the ability of retrieving exptl. obsd. waters, and showed comparable performance but with much less computational cost.  In the end, we proposed a grid-based clustering scheme together with a distance weighted wPMF score to further extend wPMF to predict the potential hydration sites of protein structure.  From the test, this approach can predict the hydration site at the accuracy about 80% when the calcd. score lower than -4.0.  It also allows the assessment of whether or not a given water mol. should be targeted for displacement in ligand design.  Overall, the wPMF presented here provides an optional soln. to many water related computational modeling problems, some of which can be highly valuable as part of a rational drug design strategy. © 2012 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvOEcbTf2Xj7Vg90H21EOLACvtfcHk0lgb2vig4D9POg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1WjsrjL&md5=298ec577c4a193a3c5d7f0bedb7859f9</span></div><a href="/servlet/linkout?suffix=cit150&amp;dbid=16384&amp;doi=10.1002%2Fjcc.23170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.23170%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DXiong%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DShen%26aufirst%3DJ.%26atitle%3DWater%2520PMF%2520for%2520predicting%2520the%2520properties%2520of%2520water%2520molecules%2520in%2520protein%2520binding%2520site%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2013%26volume%3D34%26spage%3D583%26epage%3D592%26doi%3D10.1002%2Fjcc.23170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref151"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref151'); return false;" data-citation="" class="refNumLink">151</a></strong><div class="NLM_citation" id="cit151"><span><span class="NLM_contrib-group">Forli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">A force field with discrete displaceable waters and desolvation entropy for hydrated ligand docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">623</span><span class="NLM_x">–</span> <span class="NLM_lpage">638</span><span class="refDoi"> DOI: 10.1021/jm2005145</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2005145" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Smur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=623-638&author=S.+Forliauthor=A.+J.+Olson&title=A+force+field+with+discrete+displaceable+waters+and+desolvation+entropy+for+hydrated+ligand+docking&doi=10.1021%2Fjm2005145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit151R"><div class="casContent"><span class="casTitleNuber">151</span><div class="casTitle"><span class="NLM_cas:atitle">A Force Field with Discrete Displaceable Waters and Desolvation Entropy for Hydrated Ligand Docking</span></div><div class="casAuthors">Forli, Stefano; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">623-638</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In modeling ligand-protein interactions, the representation and role of water are of great importance.  We introduce a force field and hydration docking method that enables the automated prediction of waters mediating the binding of ligands with target proteins.  The method presumes no prior knowledge of the apo or holo protein hydration state and is potentially useful in the process of structure-based drug discovery.  The hydration force field accounts for the entropic and enthalpic contributions of discrete waters to ligand binding, improving energy estn. accuracy and docking performance.  The force field has been calibrated and validated on a total of 417 complexes (197 training set; 220 test set), then tested in cross-docking expts., for a total of 1649 ligand-protein complexes evaluated.  The method is computationally efficient and was used to model up to 35 waters during docking.  The method was implemented and tested using unaltered AutoDock4 with new force field tables.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoARuEGWgy1RbVg90H21EOLACvtfcHk0lgb2vig4D9POg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Smur3O&md5=dd45a13de7c8433b8eaf168fadfcd1e7</span></div><a href="/servlet/linkout?suffix=cit151&amp;dbid=16384&amp;doi=10.1021%2Fjm2005145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2005145%26sid%3Dliteratum%253Aachs%26aulast%3DForli%26aufirst%3DS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DA%2520force%2520field%2520with%2520discrete%2520displaceable%2520waters%2520and%2520desolvation%2520entropy%2520for%2520hydrated%2520ligand%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D623%26epage%3D638%26doi%3D10.1021%2Fjm2005145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref152"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref152'); return false;" data-citation="" class="refNumLink">152</a></strong><div class="NLM_citation" id="cit152"><span><span class="NLM_contrib-group">Pottel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therrien, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gleason, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moitessier, N.</span><span> </span><span class="NLM_article-title">Docking ligands into flexible and solvated macromolecules. 6. Development and application to the docking of HDACs and other zinc metalloenzymes inhibitors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">254</span><span class="NLM_x">–</span> <span class="NLM_lpage">265</span><span class="refDoi"> DOI: 10.1021/ci400550m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400550m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFyms7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=254-265&author=J.+Pottelauthor=E.+Therrienauthor=J.+L.+Gleasonauthor=N.+Moitessier&title=Docking+ligands+into+flexible+and+solvated+macromolecules.+6.+Development+and+application+to+the+docking+of+HDACs+and+other+zinc+metalloenzymes+inhibitors&doi=10.1021%2Fci400550m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit152R"><div class="casContent"><span class="casTitleNuber">152</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Ligands into Flexible and Solvated Macromolecules. 6. Development and Application to the Docking of HDACs and other Zinc Metalloenzymes Inhibitors</span></div><div class="casAuthors">Pottel, Joshua; Therrien, Eric; Gleason, James L.; Moitessier, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">254-265</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Metalloenzymes are ubiquitous proteins which feature one or more metal ions either directly involved in the enzymic activity and/or structural properties (i.e., zinc fingers).  Several members of this class take advantage of the Lewis acidic properties of zinc ions to carry out their various catalytic transformations including isomerization or amide cleavage.  These enzymes have been validated as drug targets for a no. of diseases including cancer; however, despite their pharmaceutical relevance and the availability of crystal structures, structure-based drug design methods have been poorly and indirectly parametrized for these classes of enzymes.  More specifically, the metal coordination component and proton transfers of the process of drugs binding to metalloenzymes have been inadequately modeled by current docking programs, if at all.  In addn., several known issues, such as coordination geometry, at. charge variability, and a potential proton transfer from small mols. to a neighboring basic residue, have often been ignored.  We report herein the development of specific functions and parameters to account for zinc-drug coordination focusing on the above-listed phenomena and their impact on docking to zinc metalloenzymes.  These atom-type-dependent but at. charge-independent functions implemented into Fitted 3.1 enable the simulation of drug binding to metalloenzymes, considering an acid-base reaction with a neighboring residue when necessary with good accuracy.  The ability of docking to predict angiotensin converting enzyme (ACE) inhibitory dipeptide sequences was assessed using AutoDock Vina.  All potential dipeptides and phospho-dipeptides were docked and scored.  Peptide intestinal stability was assessed using a prediction amino acid clustering model.  Selected dipeptides, having AutoDock Vina scores ≤ -8.1 and predicted to be 'stable' intestinally, were characterized, using LIGPLOT and for ACE-inhibitory potency.  Two newly identified ACE-inhibitory dipeptides, Asp-Trp and Trp-Pro, having Vina scores of -8.3 and -8.6 gave IC50 values of 258 ± 4.23 and 217 ± 15.7 μM, resp.  LIGPLOT anal. indicated no zinc interaction for these dipeptides.  Phospho-dipeptides were predicted to have a good affinity for ACE.  However, the exptl. detd. IC50 results did not correlate since, for example, Trp-pThr and Pro-pTyr, having Vina scores of -8.5 and -8.1, resp., displayed IC50 values of >500 μM.  While docking allowed identification of new ACE inhibitory dipeptides, it may not be a fully reliable predictive tool in all cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpP7uZp-NLQXrVg90H21EOLACvtfcHk0lhmHABsRdTwKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFyms7fK&md5=6ce5b06aaede542219621d7a5df3b23e</span></div><a href="/servlet/linkout?suffix=cit152&amp;dbid=16384&amp;doi=10.1021%2Fci400550m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400550m%26sid%3Dliteratum%253Aachs%26aulast%3DPottel%26aufirst%3DJ.%26aulast%3DTherrien%26aufirst%3DE.%26aulast%3DGleason%26aufirst%3DJ.%2BL.%26aulast%3DMoitessier%26aufirst%3DN.%26atitle%3DDocking%2520ligands%2520into%2520flexible%2520and%2520solvated%2520macromolecules.%25206.%2520Development%2520and%2520application%2520to%2520the%2520docking%2520of%2520HDACs%2520and%2520other%2520zinc%2520metalloenzymes%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D254%26epage%3D265%26doi%3D10.1021%2Fci400550m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref153"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref153'); return false;" data-citation="" class="refNumLink">153</a></strong><div class="NLM_citation" id="cit153"><span><span class="NLM_contrib-group">Adeniyi, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ajibade, P. A.</span><span> </span><span class="NLM_article-title">Comparing the suitability of autodock, gold and glide for the docking and predicting the possible targets of Ru(II)-based complexes as anticancer agents</span> <span class="citation_source-journal">Molecules</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">3760</span><span class="NLM_x">–</span> <span class="NLM_lpage">3778</span><span class="refDoi"> DOI: 10.3390/molecules18043760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.3390%2Fmolecules18043760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=23529035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlslCmsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=3760-3778&author=A.+A.+Adeniyiauthor=P.+A.+Ajibade&title=Comparing+the+suitability+of+autodock%2C+gold+and+glide+for+the+docking+and+predicting+the+possible+targets+of+Ru%28II%29-based+complexes+as+anticancer+agents&doi=10.3390%2Fmolecules18043760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit153R"><div class="casContent"><span class="casTitleNuber">153</span><div class="casTitle"><span class="NLM_cas:atitle">Comparing the suitability of Autodock, Gold and Glide for the docking and predicting the possible targets of Ru(II)-based complexes as anticancer agents</span></div><div class="casAuthors">Adeniyi, Adebayo A.; Ajibade, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3760-3778</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">In cancer chemotherapy, metal-based complexes have been recognized as the most promising means of inhibiting cancer growth due to the successful application of cis-platin and its derivs. above many of the existing org. anticancer agents.  The limitations in their rational design can be traced to the complexity of the mechanism of their operations, lack of proper knowledge of their targets and lack of force fields in docking packages to appropriately define the metal center of the organometallic complexes.  In this paper, some of the promising anticancer complexes of Ru(II) such as the rapta-based complexes formulated as [Ru(η6-p-cymene)L2(pta)] and those with unusual ligands are considered.  CatB and kinases which have been exptl. confirmed as possible targets of the complexes are also predicted by the three methods as one of the most targeted receptors while TopII and HDAC7 are predicted by two and one of the methods as best targets.  The interesting features of the binding of the complexes show that some of the complexes preferentially target specific macromols. than the others, which is an indication of their specificity and possibility of their therapeutic combination without severe side effects that may come from competition for the same target.  Also, introduction of unusual ligands is found to significantly improve the activities of most of the complexes studied.  Strong correlations are obsd. for the predicted binding sites and the orientation of the complexes within the binding site by the three methods of docking.  However there are disparities in the ranking of the complexes by the three method of docking, esp. that of Glide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkdpV-_FnD1rVg90H21EOLACvtfcHk0lhmHABsRdTwKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlslCmsrw%253D&md5=750e7ed12bc2d8827e4b00fba9ddd5c5</span></div><a href="/servlet/linkout?suffix=cit153&amp;dbid=16384&amp;doi=10.3390%2Fmolecules18043760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules18043760%26sid%3Dliteratum%253Aachs%26aulast%3DAdeniyi%26aufirst%3DA.%2BA.%26aulast%3DAjibade%26aufirst%3DP.%2BA.%26atitle%3DComparing%2520the%2520suitability%2520of%2520autodock%252C%2520gold%2520and%2520glide%2520for%2520the%2520docking%2520and%2520predicting%2520the%2520possible%2520targets%2520of%2520Ru%2528II%2529-based%2520complexes%2520as%2520anticancer%2520agents%26jtitle%3DMolecules%26date%3D2013%26volume%3D18%26spage%3D3760%26epage%3D3778%26doi%3D10.3390%2Fmolecules18043760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref154"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref154'); return false;" data-citation="" class="refNumLink">154</a></strong><div class="NLM_citation" id="cit154"><span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">Assignment of polar states for protein amino acid residues using an interaction cluster decomposition algorithm and its application to high resolution protein structure modeling</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">824</span><span class="NLM_x">–</span> <span class="NLM_lpage">837</span><span class="refDoi"> DOI: 10.1002/prot.21125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fprot.21125" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2007&pages=824-837&author=X.+Liauthor=M.+P.+Jacobsonauthor=K.+Zhuauthor=S.+Zhaoauthor=R.+A.+Friesner&title=Assignment+of+polar+states+for+protein+amino+acid+residues+using+an+interaction+cluster+decomposition+algorithm+and+its+application+to+high+resolution+protein+structure+modeling&doi=10.1002%2Fprot.21125"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit154&amp;dbid=16384&amp;doi=10.1002%2Fprot.21125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21125%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DAssignment%2520of%2520polar%2520states%2520for%2520protein%2520amino%2520acid%2520residues%2520using%2520an%2520interaction%2520cluster%2520decomposition%2520algorithm%2520and%2520its%2520application%2520to%2520high%2520resolution%2520protein%2520structure%2520modeling%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2007%26volume%3D66%26spage%3D824%26epage%3D837%26doi%3D10.1002%2Fprot.21125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref155"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref155'); return false;" data-citation="" class="refNumLink">155</a></strong><div class="NLM_citation" id="cit155"><span><span class="NLM_contrib-group">Kruger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grunzke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herres-Pawlis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffmann, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de la Garza, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kohlbacher, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagel, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gesing, S.</span><span> </span><span class="NLM_article-title">Performance studies on distributed virtual screening</span> <span class="citation_source-journal">BioMed Res. Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">2014</span><span class="NLM_x">, </span> <span class="NLM_fpage">624024</span><span class="refDoi"> DOI: 10.1155/2014/624024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1155%2F2014%2F624024" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2014&publication_year=2014&pages=624024&author=J.+Krugerauthor=R.+Grunzkeauthor=S.+Herres-Pawlisauthor=A.+Hoffmannauthor=L.+de+la+Garzaauthor=O.+Kohlbacherauthor=W.+E.+Nagelauthor=S.+Gesing&title=Performance+studies+on+distributed+virtual+screening&doi=10.1155%2F2014%2F624024"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit155&amp;dbid=16384&amp;doi=10.1155%2F2014%2F624024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2014%252F624024%26sid%3Dliteratum%253Aachs%26aulast%3DKruger%26aufirst%3DJ.%26aulast%3DGrunzke%26aufirst%3DR.%26aulast%3DHerres-Pawlis%26aufirst%3DS.%26aulast%3DHoffmann%26aufirst%3DA.%26aulast%3Dde%2Bla%2BGarza%26aufirst%3DL.%26aulast%3DKohlbacher%26aufirst%3DO.%26aulast%3DNagel%26aufirst%3DW.%2BE.%26aulast%3DGesing%26aufirst%3DS.%26atitle%3DPerformance%2520studies%2520on%2520distributed%2520virtual%2520screening%26jtitle%3DBioMed%2520Res.%2520Int.%26date%3D2014%26volume%3D2014%26spage%3D624024%26doi%3D10.1155%2F2014%2F624024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref156"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref156'); return false;" data-citation="" class="refNumLink">156</a></strong><div class="NLM_citation" id="cit156"><span><span class="NLM_contrib-group">Trott, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">455</span><span class="NLM_x">–</span> <span class="NLM_lpage">461</span><span class="refDoi"> DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-461&author=O.+Trottauthor=A.+J.+Olson&title=AutoDock+Vina%3A+improving+the+speed+and+accuracy+of+docking+with+a+new+scoring+function%2C+efficient+optimization%2C+and+multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit156R"><div class="casContent"><span class="casTitleNuber">156</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0lhmHABsRdTwKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit156&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock%2520Vina%253A%2520improving%2520the%2520speed%2520and%2520accuracy%2520of%2520docking%2520with%2520a%2520new%2520scoring%2520function%252C%2520efficient%2520optimization%252C%2520and%2520multithreading%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D461%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref157"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref157'); return false;" data-citation="" class="refNumLink">157</a></strong><div class="NLM_citation" id="cit157"><span><span class="NLM_contrib-group">Kasam, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salzemann, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botha, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dacosta, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Degliesposti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isea, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maass, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenyon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rastelli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hofmann-Apitius, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breton, V.</span><span> </span><span class="NLM_article-title">WISDOM-II: screening against multiple targets implicated in malaria using computational grid infrastructures</span> <span class="citation_source-journal">Malar. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">88</span><span class="refDoi"> DOI: 10.1186/1475-2875-8-88</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1186%2F1475-2875-8-88" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=19409081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BD1MzpvVeitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=88&author=V.+Kasamauthor=J.+Salzemannauthor=M.+Bothaauthor=A.+Dacostaauthor=G.+Degliespostiauthor=R.+Iseaauthor=D.+Kimauthor=A.+Maassauthor=C.+Kenyonauthor=G.+Rastelliauthor=M.+Hofmann-Apitiusauthor=V.+Breton&title=WISDOM-II%3A+screening+against+multiple+targets+implicated+in+malaria+using+computational+grid+infrastructures&doi=10.1186%2F1475-2875-8-88"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit157R"><div class="casContent"><span class="casTitleNuber">157</span><div class="casTitle"><span class="NLM_cas:atitle">WISDOM-II: screening against multiple targets implicated in malaria using computational grid infrastructures</span></div><div class="casAuthors">Kasam Vinod; Salzemann Jean; Botha Marli; Dacosta Ana; Degliesposti Gianluca; Isea Raul; Kim Doman; Maass Astrid; Kenyon Colin; Rastelli Giulio; Hofmann-Apitius Martin; Breton Vincent</div><div class="citationInfo"><span class="NLM_cas:title">Malaria journal</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">88</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Despite continuous efforts of the international community to reduce the impact of malaria on developing countries, no significant progress has been made in the recent years and the discovery of new drugs is more than ever needed.  Out of the many proteins involved in the metabolic activities of the Plasmodium parasite, some are promising targets to carry out rational drug discovery.  MOTIVATION:  Recent years have witnessed the emergence of grids, which are highly distributed computing infrastructures particularly well fitted for embarrassingly parallel computations like docking.  In 2005, a first attempt at using grids for large-scale virtual screening focused on plasmepsins and ended up in the identification of previously unknown scaffolds, which were confirmed in vitro to be active plasmepsin inhibitors.  Following this success, a second deployment took place in the fall of 2006 focussing on one well known target, dihydrofolate reductase (DHFR), and on a new promising one, glutathione-S-transferase.  METHODS:  In silico drug design, especially vHTS is a widely and well-accepted technology in lead identification and lead optimization.  This approach, therefore builds, upon the progress made in computational chemistry to achieve more accurate in silico docking and in information technology to design and operate large scale grid infrastructures.  RESULTS:  On the computational side, a sustained infrastructure has been developed: docking at large scale, using different strategies in result analysis, storing of the results on the fly into MySQL databases and application of molecular dynamics refinement are MM-PBSA and MM-GBSA rescoring.  The modeling results obtained are very promising.  Based on the modeling results, In vitro results are underway for all the targets against which screening is performed.  CONCLUSION:  The current paper describes the rational drug discovery activity at large scale, especially molecular docking using FlexX software on computational grids in finding hits against three different targets (PfGST, PfDHFR, PvDHFR (wild type and mutant forms) implicated in malaria.  Grid-enabled virtual screening approach is proposed to produce focus compound libraries for other biological targets relevant to fight the infectious diseases of the developing world.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTs3wyAqQCw3UEgZCMqtRBifW6udTcc2ea46PwTBkYXULntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MzpvVeitw%253D%253D&md5=b3d674ddc73d104e96924752d34a499b</span></div><a href="/servlet/linkout?suffix=cit157&amp;dbid=16384&amp;doi=10.1186%2F1475-2875-8-88&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1475-2875-8-88%26sid%3Dliteratum%253Aachs%26aulast%3DKasam%26aufirst%3DV.%26aulast%3DSalzemann%26aufirst%3DJ.%26aulast%3DBotha%26aufirst%3DM.%26aulast%3DDacosta%26aufirst%3DA.%26aulast%3DDegliesposti%26aufirst%3DG.%26aulast%3DIsea%26aufirst%3DR.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DMaass%26aufirst%3DA.%26aulast%3DKenyon%26aufirst%3DC.%26aulast%3DRastelli%26aufirst%3DG.%26aulast%3DHofmann-Apitius%26aufirst%3DM.%26aulast%3DBreton%26aufirst%3DV.%26atitle%3DWISDOM-II%253A%2520screening%2520against%2520multiple%2520targets%2520implicated%2520in%2520malaria%2520using%2520computational%2520grid%2520infrastructures%26jtitle%3DMalar.%2520J.%26date%3D2009%26volume%3D8%26spage%3D88%26doi%3D10.1186%2F1475-2875-8-88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref158"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref158'); return false;" data-citation="" class="refNumLink">158</a></strong><div class="NLM_citation" id="cit158"><span>CombiGlide; <span class="NLM_publisher-name">Schrodinger</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=CombiGlide%3B+Schrodinger%3A+New+York%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit158&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DCombiGlide%26pub%3DSchrodinger%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref159"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref159'); return false;" data-citation="" class="refNumLink">159</a></strong><div class="NLM_citation" id="cit159"><span><span class="NLM_contrib-group">Lazaridis, T.</span><span> </span><span class="NLM_article-title">Inhomogeneous fluid approach to solvation thermodynamics. 2. Applications to simple fluids</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">3542</span><span class="NLM_x">–</span> <span class="NLM_lpage">3550</span><span class="refDoi"> DOI: 10.1021/jp972358w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp972358w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADyaK1cXivVOltrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=1998&pages=3542-3550&author=T.+Lazaridis&title=Inhomogeneous+fluid+approach+to+solvation+thermodynamics.+2.+Applications+to+simple+fluids&doi=10.1021%2Fjp972358w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit159R"><div class="casContent"><span class="casTitleNuber">159</span><div class="casTitle"><span class="NLM_cas:atitle">Inhomogeneous Fluid Approach to Solvation Thermodynamics. 2. Applications to Simple Fluids</span></div><div class="casAuthors">Lazaridis, Themis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3542-3550</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1089-5647</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the previous paper expressions for the partial molar energy and entropy at infinite diln. have been derived based on the inhomogeneous forms of the energy equation and the correlation expansion for the entropy.  These expressions are here applied to a series of solutes of varying size in dense hard-sphere and Lennard-Jones solvents, some of which serve as ref. systems for comparison with water.  Numerical results are obtained under the assumption that the inhomogeneous solvent-solvent pair correlation function in the mixt. is equal to the bulk solvent radial distribution function (Kirkwood superposition approxn.).  The correlation functions required are obtained by both integral equation theory (Percus-Yevick approxn.) and Monte Carlo simulations.  The thermodn. results are compared with equation of state, integral equation, and free energy simulation results for the same systems.  For hard-sphere systems the excess entropies are in good agreement with equation-of-state results but in many Lennard-Jones systems the calcd. partial molar energies and entropies are lower than the expected values.  This is attributable to overestimation of the structure of the bulk triplet correlation function by the superposition approxn.  The decompn. of the chem. potential shows that similar solvation free energies can have entirely different phys. origins.  Specifically, in solvents of high cohesive energy d. the chem. potential is dominated by the breakup of solvent-solvent interactions locally around the solute.  In solvents of low cohesive energy d. it is dominated by the pressure-vol. term.  Increase in solvent-solvent interaction strength leads to increase in the chem. potential of the solute due to the higher solvent reorganization energy, which is insufficiently compensated by an increase in solvent reorganization entropy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdwkuoqtHxPLVg90H21EOLACvtfcHk0lhZtbOHqOvLdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXivVOltrw%253D&md5=63ce8d44f2f6c8d50f81303dba75924f</span></div><a href="/servlet/linkout?suffix=cit159&amp;dbid=16384&amp;doi=10.1021%2Fjp972358w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp972358w%26sid%3Dliteratum%253Aachs%26aulast%3DLazaridis%26aufirst%3DT.%26atitle%3DInhomogeneous%2520fluid%2520approach%2520to%2520solvation%2520thermodynamics.%25202.%2520Applications%2520to%2520simple%2520fluids%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1998%26volume%3D102%26spage%3D3542%26epage%3D3550%26doi%3D10.1021%2Fjp972358w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref160"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref160'); return false;" data-citation="" class="refNumLink">160</a></strong><div class="NLM_citation" id="cit160"><span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tubert-Brohman, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">Docking performance of the glide program as evaluated on the Astex and DUD datasets: a complete set of glide SP results and selected results for a new scoring function integrating WaterMap and glide</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">787</span><span class="NLM_x">–</span> <span class="NLM_lpage">799</span><span class="refDoi"> DOI: 10.1007/s10822-012-9575-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs10822-012-9575-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=22576241" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVelu7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=787-799&author=M.+P.+Repaskyauthor=R.+B.+Murphyauthor=J.+L.+Banksauthor=J.+R.+Greenwoodauthor=I.+Tubert-Brohmanauthor=S.+Bhatauthor=R.+A.+Friesner&title=Docking+performance+of+the+glide+program+as+evaluated+on+the+Astex+and+DUD+datasets%3A+a+complete+set+of+glide+SP+results+and+selected+results+for+a+new+scoring+function+integrating+WaterMap+and+glide&doi=10.1007%2Fs10822-012-9575-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit160R"><div class="casContent"><span class="casTitleNuber">160</span><div class="casTitle"><span class="NLM_cas:atitle">Docking performance of the glide program as evaluated on the Astex and DUD datasets: a complete set of glide SP results and selected results for a new scoring function integrating WaterMap and glide</span></div><div class="casAuthors">Repasky, Matthew P.; Murphy, Robert B.; Banks, Jay L.; Greenwood, Jeremy R.; Tubert-Brohman, Ivan; Bhat, Sathesh; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">787-799</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Glide SP mode enrichment results for two prepns. of the DUD dataset and native ligand docking RMSDs for two prepns. of the Astex dataset are presented.  Following a best-practices prepn. scheme, an av. RMSD of 1.140 Å for native ligand docking with Glide SP is computed.  Following the same best-practices prepn. scheme for the DUD dataset an av. area under the ROC curve (AUC) of 0.80 and av. early enrichment via the ROC (0.1 %) metric of 0.12 were obsd. 74 and 56 % of the 39 best-practices prepd. targets showed AUC over 0.7 and 0.8, resp.  Av. AUC was greater than 0.7 for all best-practices protein families demonstrating consistent enrichment performance across a broad range of proteins and ligand chemotypes.  In both Astex and DUD datasets, docking performance is significantly improved employing a best-practices prepn. scheme over using minimally-prepd. structures from the PDB.  Enrichment results for WScore, a new scoring function and sampling methodol. integrating WaterMap and Glide, are presented for four DUD targets, hivrt, hsp90, cdk2, and fxa.  WScore performance in early enrichment is consistently strong and all systems examd. show AUC > 0.9 and superior early enrichment to DUD best-practices Glide SP results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHCCMQs2lTrLVg90H21EOLACvtfcHk0lhZtbOHqOvLdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVelu7nI&md5=9eb2c6feb24177c81bb5c06b947893fc</span></div><a href="/servlet/linkout?suffix=cit160&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9575-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9575-9%26sid%3Dliteratum%253Aachs%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DTubert-Brohman%26aufirst%3DI.%26aulast%3DBhat%26aufirst%3DS.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DDocking%2520performance%2520of%2520the%2520glide%2520program%2520as%2520evaluated%2520on%2520the%2520Astex%2520and%2520DUD%2520datasets%253A%2520a%2520complete%2520set%2520of%2520glide%2520SP%2520results%2520and%2520selected%2520results%2520for%2520a%2520new%2520scoring%2520function%2520integrating%2520WaterMap%2520and%2520glide%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D787%26epage%3D799%26doi%3D10.1007%2Fs10822-012-9575-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref161"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref161'); return false;" data-citation="" class="refNumLink">161</a></strong><div class="NLM_citation" id="cit161"><span><span class="NLM_contrib-group">Nguyen, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">Grid inhomogeneous solvation theory: hydration structure and thermodynamics of the miniature receptor cucurbit[7]uril</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">137</span><span class="NLM_x">, </span> <span class="NLM_fpage">044101</span><span class="refDoi"> DOI: 10.1063/1.4733951</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1063%2F1.4733951" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=22852591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVOgurzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2012&pages=044101&author=C.+N.+Nguyenauthor=T.+K.+Youngauthor=M.+K.+Gilson&title=Grid+inhomogeneous+solvation+theory%3A+hydration+structure+and+thermodynamics+of+the+miniature+receptor+cucurbit%5B7%5Duril&doi=10.1063%2F1.4733951"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit161R"><div class="casContent"><span class="casTitleNuber">161</span><div class="casTitle"><span class="NLM_cas:atitle">Grid inhomogeneous solvation theory: Hydration structure and thermodynamics of the miniature receptor cucurbit[7]uril</span></div><div class="casAuthors">Nguyen, Crystal N.; Kurtzman Young, Tom; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">044101/1-044101/17</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The displacement of perturbed water upon binding is believed to play a crit. role in the thermodn. of biomol. recognition, but it is nontrivial to unambiguously define and answer questions about this process.  We address this issue by introducing grid inhomogeneous solvation theory (GIST), which discretizes the equations of inhomogeneous solvation theory (IST) onto a three-dimensional grid situated in the region of interest around a solute mol. or complex.  Snapshots from explicit solvent simulations are used to est. localized solvation entropies, energies, and free energies assocd. with the grid boxes, or voxels, and properly summing these thermodn. quantities over voxels yields information about hydration thermodn.  GIST thus provides a smoothly varying representation of water properties as a function of position, rather than focusing on hydration sites where solvent is present at high d.  It therefore accounts for full or partial displacement of water from sites that are highly occupied by water, as well as for partly occupied and water-depleted regions around the solute.  GIST can also provide a well-defined est. of the solvation free energy and therefore enables a rigorous end-states anal. of binding.  For example, one may not only use a first GIST calcn. to project the thermodn. consequences of displacing water from the surface of a receptor by a ligand, but also account, in a second GIST calcn., for the thermodn. of subsequent solvent reorganization around the bound complex.  In the present study, a first GIST anal. of the mol. host cucurbit7uril is found to yield a rich picture of hydration structure and thermodn. in and around this miniature receptor.  One of the most striking results is the observation of a toroidal region of high water d. at the center of the host's nonpolar cavity.  Despite its high d., the water in this toroidal region is disfavored energetically and entropically, and hence may contribute to the known ability of this small receptor to bind guest mols. with unusually high affinities.  Interestingly, the toroidal region of high water d. persists even when all partial charges of the receptor are set to zero.  Thus, localized regions of high solvent d. can be generated in a binding site without strong, attractive solute-solvent interactions.  (c) 2012 American Institute of Physics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkKZ05RaZJbbVg90H21EOLACvtfcHk0lieGVeo3LA1iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVOgurzJ&md5=d8eceb021e903950784786a656fdfb40</span></div><a href="/servlet/linkout?suffix=cit161&amp;dbid=16384&amp;doi=10.1063%2F1.4733951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.4733951%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DC.%2BN.%26aulast%3DYoung%26aufirst%3DT.%2BK.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DGrid%2520inhomogeneous%2520solvation%2520theory%253A%2520hydration%2520structure%2520and%2520thermodynamics%2520of%2520the%2520miniature%2520receptor%2520cucurbit%255B7%255Duril%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2012%26volume%3D137%26spage%3D044101%26doi%3D10.1063%2F1.4733951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref162"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref162'); return false;" data-citation="" class="refNumLink">162</a></strong><div class="NLM_citation" id="cit162"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">ZINC--A free database of commercially available compounds for virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span><span class="refDoi"> DOI: 10.1021/ci049714+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049714%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=177-182&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=ZINC%2D%2DA+free+database+of+commercially+available+compounds+for+virtual+screening&doi=10.1021%2Fci049714%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit162R"><div class="casContent"><span class="casTitleNuber">162</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC--a free database of commercially available compounds for virtual screening</span></div><div class="casAuthors">Irwin John J; Shoichet Brian K</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-82</span>
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    </div><div class="casAbstract">A critical barrier to entry into structure-based virtual screening is the lack of a suitable, easy to access database of purchasable compounds.  We have therefore prepared a library of 727,842 molecules, each with 3D structure, using catalogs of compounds from vendors (the size of this library continues to grow).  The molecules have been assigned biologically relevant protonation states and are annotated with properties such as molecular weight, calculated LogP, and number of rotatable bonds.  Each molecule in the library contains vendor and purchasing information and is ready for docking using a number of popular docking programs.  Within certain limits, the molecules are prepared in multiple protonation states and multiple tautomeric forms.  In one format, multiple conformations are available for the molecules.  This database is available for free download (http://zinc.docking.org) in several common file formats including SMILES, mol2, 3D SDF, and DOCK flexibase format.  A Web-based query tool incorporating a molecular drawing interface enables the database to be searched and browsed and subsets to be created.  Users can process their own molecules by uploading them to a server.  Our hope is that this database will bring virtual screening libraries to a wide community of structural biologists and medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQVi_-excgTX-_Q9URG6Dx8fW6udTcc2eb0W1ePpcp3Frntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D&md5=9f2e3ec06d53276884fc17b9aad47e61</span></div><a href="/servlet/linkout?suffix=cit162&amp;dbid=16384&amp;doi=10.1021%2Fci049714%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049714%252B%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DZINC--A%2520free%2520database%2520of%2520commercially%2520available%2520compounds%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D177%26epage%3D182%26doi%3D10.1021%2Fci049714%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref163"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref163'); return false;" data-citation="" class="refNumLink">163</a></strong><div class="NLM_citation" id="cit163"><span><span class="NLM_contrib-group">Sterling, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span> </span><span class="NLM_article-title">ZINC 15 - Ligand discovery for everyone</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">2324</span><span class="NLM_x">–</span> <span class="NLM_lpage">2337</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00559</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00559" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1OhurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2324-2337&author=T.+Sterlingauthor=J.+J.+Irwin&title=ZINC+15+-+Ligand+discovery+for+everyone&doi=10.1021%2Facs.jcim.5b00559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit163R"><div class="casContent"><span class="casTitleNuber">163</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC 15 - Ligand Discovery for Everyone</span></div><div class="casAuthors">Sterling, Teague; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2324-2337</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many questions about the biol. activity and availability of small mols. remain inaccessible to investigators who could most benefit from their answers.  To narrow the gap between chemoinformatics and biol., we have developed a suite of ligand annotation, purchasability, target, and biol. assocn. tools, incorporated into ZINC and meant for investigators who are not computer specialists.  The new version contains over 120 million purchasable "drug-like" compds. - effectively all org. mols. that are for sale - a quarter of which are available for immediate delivery.  ZINC connects purchasable compds. to high-value ones such as metabolites, drugs, natural products, and annotated compds. from the literature.  Compds. may be accessed by the genes for which they are annotated as well as the major and minor target classes to which those genes belong.  It offers new anal. tools that are easy for nonspecialists yet with few limitations for experts.  ZINC retains its original 3D roots - all mols. are available in biol. relevant, ready-to-dock formats.  ZINC is freely available at http://zinc15.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWfkDnal3D3bVg90H21EOLACvtfcHk0lieGVeo3LA1iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1OhurbF&md5=e767c26c1d0aff7432c403c59a5d2b8b</span></div><a href="/servlet/linkout?suffix=cit163&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00559%26sid%3Dliteratum%253Aachs%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DZINC%252015%2520-%2520Ligand%2520discovery%2520for%2520everyone%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2324%26epage%3D2337%26doi%3D10.1021%2Facs.jcim.5b00559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref164"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref164'); return false;" data-citation="" class="refNumLink">164</a></strong><div class="NLM_citation" id="cit164"><span><span class="NLM_contrib-group">ten Brink, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Exner, T. E.</span><span> </span><span class="NLM_article-title">pK(a) based protonation states and microspecies for protein-ligand docking</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">935</span><span class="NLM_x">–</span> <span class="NLM_lpage">942</span><span class="refDoi"> DOI: 10.1007/s10822-010-9385-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs10822-010-9385-x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=935-942&author=T.+ten+Brinkauthor=T.+E.+Exner&title=pK%28a%29+based+protonation+states+and+microspecies+for+protein-ligand+docking&doi=10.1007%2Fs10822-010-9385-x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit164&amp;dbid=16384&amp;doi=10.1007%2Fs10822-010-9385-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-010-9385-x%26sid%3Dliteratum%253Aachs%26aulast%3Dten%2BBrink%26aufirst%3DT.%26aulast%3DExner%26aufirst%3DT.%2BE.%26atitle%3DpK%2528a%2529%2520based%2520protonation%2520states%2520and%2520microspecies%2520for%2520protein-ligand%2520docking%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2010%26volume%3D24%26spage%3D935%26epage%3D942%26doi%3D10.1007%2Fs10822-010-9385-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref165"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref165'); return false;" data-citation="" class="refNumLink">165</a></strong><div class="NLM_citation" id="cit165"><span><span class="NLM_contrib-group">Kim, M. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">Computation of pH-dependent binding free energies</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">43</span><span class="NLM_x">–</span> <span class="NLM_lpage">49</span><span class="refDoi"> DOI: 10.1002/bip.22702</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fbip.22702" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26202905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslSqsrzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2016&pages=43-49&author=M.+O.+Kimauthor=J.+A.+McCammon&title=Computation+of+pH-dependent+binding+free+energies&doi=10.1002%2Fbip.22702"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit165R"><div class="casContent"><span class="casTitleNuber">165</span><div class="casTitle"><span class="NLM_cas:atitle">Computation of pH-dependent binding free energies</span></div><div class="casAuthors">Kim, M. Olivia; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-49</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Protein-ligand binding accompanies changes in the surrounding electrostatic environments of the two binding partners and may lead to changes in protonation upon binding.  In cases where the complex formation results in a net transfer of protons, the binding process is pH-dependent.  However, conventional free energy computations or mol. docking protocols typically employ fixed protonation states for the titratable groups in both binding partners set a priori, which are identical for the free and bound states.  In this review, we draw attention to these important yet largely ignored binding-induced protonation changes in protein-ligand assocn. by outlining phys. origins and prevalence of the protonation changes upon binding.  Following a summary of various theor. methods for pKa prediction, we discuss the theor. framework to examine the pH dependence of protein-ligand binding processes.  © 2015 Wiley Periodicals, Inc.  Biopolymers 105: 43-49, 2015.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS2LkIfxVqrrVg90H21EOLACvtfcHk0ljlQ3tIwlf7mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslSqsrzO&md5=8edc165134617308d9fef26620974012</span></div><a href="/servlet/linkout?suffix=cit165&amp;dbid=16384&amp;doi=10.1002%2Fbip.22702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.22702%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DM.%2BO.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DComputation%2520of%2520pH-dependent%2520binding%2520free%2520energies%26jtitle%3DBiopolymers%26date%3D2016%26volume%3D105%26spage%3D43%26epage%3D49%26doi%3D10.1002%2Fbip.22702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref166"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref166'); return false;" data-citation="" class="refNumLink">166</a></strong><div class="NLM_citation" id="cit166"><span><span class="NLM_contrib-group">Hawkins, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholls, A.</span><span> </span><span class="NLM_article-title">Conformer generation with OMEGA: learning from the data set and the analysis of failures</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2919</span><span class="NLM_x">–</span> <span class="NLM_lpage">2936</span><span class="refDoi"> DOI: 10.1021/ci300314k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300314k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFekurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2919-2936&author=P.+C.+Hawkinsauthor=A.+Nicholls&title=Conformer+generation+with+OMEGA%3A+learning+from+the+data+set+and+the+analysis+of+failures&doi=10.1021%2Fci300314k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit166R"><div class="casContent"><span class="casTitleNuber">166</span><div class="casTitle"><span class="NLM_cas:atitle">Conformer Generation with OMEGA: Learning from the Data Set and the Analysis of Failures</span></div><div class="casAuthors">Hawkins, Paul C. D.; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2919-2936</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We recently published a high quality validation set for testing conformer generators, consisting of structures from both the PDB and the CSD and tested the performance of the authors' conformer generator, OMEGA, on these sets.  In the present publication, we focus on understanding the suitability of those data sets for validation and identifying and learning from OMEGA's failures.  We compare, for the first time we are aware of, the coverage of the applicable property spaces between the validation data sets we used and the parent compd. sets to det. if our data sets adequately sample these property spaces.  We also introduce the concept of torsion fingerprinting and compare this method of dissimilation to the more traditional graph-centric diversification methods we used in our previous publication.  To improve our ability to programmatically identify cases where the crystallog. conformation is not well reproduced computationally, we introduce a new metric to compare conformations, RMSTanimoto.  This new metric is used alongside those from our previous publication to efficiently identify reprodn. failures.  We find RMSTanimoto to be particularly effective in identifying failures for the smallest mols. in our data sets.  Anal. of the nature of these failures, particularly those for the CSD, sheds further light on the issue of strain in crystallog. structures.  Some of the residual failure cases not resolved by simple changes in OMEGA's defaults present significant challenges to conformer generation engines like OMEGA and are a source of new avenues to further improve their performance, while others illustrate the pitfalls of validating against crystallog. ligand conformations, particularly those from the PDB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTlHYy7GV9RbVg90H21EOLACvtfcHk0ljlQ3tIwlf7mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFekurzN&md5=852c44d8587480e69709cefeee9206ac</span></div><a href="/servlet/linkout?suffix=cit166&amp;dbid=16384&amp;doi=10.1021%2Fci300314k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300314k%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DConformer%2520generation%2520with%2520OMEGA%253A%2520learning%2520from%2520the%2520data%2520set%2520and%2520the%2520analysis%2520of%2520failures%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D2919%26epage%3D2936%26doi%3D10.1021%2Fci300314k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref167"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref167'); return false;" data-citation="" class="refNumLink">167</a></strong><div class="NLM_citation" id="cit167"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, L. L.</span><span> </span><span class="NLM_article-title">Perspective on free-energy perturbation calculations for chemical equilibria</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">869</span><span class="NLM_x">–</span> <span class="NLM_lpage">876</span><span class="refDoi"> DOI: 10.1021/ct800011m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct800011m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlslOisLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=869-876&author=W.+L.+Jorgensenauthor=L.+L.+Thomas&title=Perspective+on+free-energy+perturbation+calculations+for+chemical+equilibria&doi=10.1021%2Fct800011m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit167R"><div class="casContent"><span class="casTitleNuber">167</span><div class="casTitle"><span class="NLM_cas:atitle">Perspective on Free-Energy Perturbation Calculations for Chemical Equilibria</span></div><div class="casAuthors">Jorgensen, William L.; Thomas, Laura L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">869-876</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An overview has been provided on the history of free-energy perturbation (FEP) calcns. and their application to computation of relative free energies of solvation.  In view of the expanding role of FEP calcns. for pursuing a wide range of problems in chem. and biochem., including mol. design, it is important to establish reliable protocols for their execution.  For 30 interconversions of substituted benzenes, which represent prototypical cases for mol. design, alternative sampling procedures and nos. of FEP windows were considered.  Simple overlap sampling with 11 or 15 windows and double-wide sampling with 14 windows emerged as viable procedures for such routine FEP calcns.; the present protocols with these sampling methods yield statistical uncertainties of 0.2 kcal/mol for free-energy differences.  The slightly greater accuracy obtainable with 21 to 25 windows does not clearly justify the increased costs, while overlap sampling with 6 or 8 windows may be useful for preliminary exploration of large nos. of mol. changes.  Application of the power-series expansion of the Zwanzig equation was also tested.  Use of the first-order term is generally reliable, while inclusion of the slowly convergent, second-order fluctuation term causes deterioration in the results for strongly hydrogen-bonded solutes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqukDgvYtLny7Vg90H21EOLACvtfcHk0ljlQ3tIwlf7mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlslOisLw%253D&md5=4f6cd47142dbbcd51e94aedd2d878538</span></div><a href="/servlet/linkout?suffix=cit167&amp;dbid=16384&amp;doi=10.1021%2Fct800011m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct800011m%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DThomas%26aufirst%3DL.%2BL.%26atitle%3DPerspective%2520on%2520free-energy%2520perturbation%2520calculations%2520for%2520chemical%2520equilibria%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2008%26volume%3D4%26spage%3D869%26epage%3D876%26doi%3D10.1021%2Fct800011m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref168"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref168'); return false;" data-citation="" class="refNumLink">168</a></strong><div class="NLM_citation" id="cit168"><span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span> </span><span class="NLM_article-title">Binding of small-molecule ligands to proteins: "what you see" is not always "what you get"</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">489</span><span class="NLM_x">–</span> <span class="NLM_lpage">498</span><span class="refDoi"> DOI: 10.1016/j.str.2009.02.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.str.2009.02.010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=489-498&author=D.+L.+Mobleyauthor=K.+A.+Dill&title=Binding+of+small-molecule+ligands+to+proteins%3A+%22what+you+see%22+is+not+always+%22what+you+get%22&doi=10.1016%2Fj.str.2009.02.010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit168&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2009.02.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2009.02.010%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DBinding%2520of%2520small-molecule%2520ligands%2520to%2520proteins%253A%2520%2522what%2520you%2520see%2522%2520is%2520not%2520always%2520%2522what%2520you%2520get%2522%26jtitle%3DStructure%26date%3D2009%26volume%3D17%26spage%3D489%26epage%3D498%26doi%3D10.1016%2Fj.str.2009.02.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref169"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref169'); return false;" data-citation="" class="refNumLink">169</a></strong><div class="NLM_citation" id="cit169"><span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McReynolds, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span> </span><span class="NLM_article-title">Predicting absolute ligand binding free energies to a simple model site</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">371</span><span class="NLM_x">, </span> <span class="NLM_fpage">1118</span><span class="NLM_x">–</span> <span class="NLM_lpage">1134</span><span class="refDoi"> DOI: 10.1016/j.jmb.2007.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.jmb.2007.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=17599350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2sXot12rt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=2007&pages=1118-1134&author=D.+L.+Mobleyauthor=A.+P.+Gravesauthor=J.+D.+Choderaauthor=A.+C.+McReynoldsauthor=B.+K.+Shoichetauthor=K.+A.+Dill&title=Predicting+absolute+ligand+binding+free+energies+to+a+simple+model+site&doi=10.1016%2Fj.jmb.2007.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit169R"><div class="casContent"><span class="casTitleNuber">169</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Absolute Ligand Binding Free Energies to a Simple Model Site</span></div><div class="casAuthors">Mobley, David L.; Graves, Alan P.; Chodera, John D.; McReynolds, Andrea C.; Shoichet, Brian K.; Dill, Ken A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1118-1134</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A central challenge in structure-based ligand design is the accurate prediction of binding free energies.  Here the authors apply alchem. free energy calcns. in explicit solvent to predict ligand binding in a model cavity in T4 lysozyme.  Even in this simple site, there are challenges.  The authors made systematic improvements, beginning with single poses from docking, then including multiple poses, addnl. protein conformational changes, and using an improved charge model.  Computed abs. binding free energies had an RMS error of 1.9 kcal/mol relative to previously detd. exptl. values.  In blind prospective tests, the methods correctly discriminated between several true ligands and decoys in a set of putative binders identified by docking.  In these prospective tests, the RMS error in predicted binding free energies relative to those subsequently detd. exptl. was only 0.6 kcal/mol.  X-ray crystal structures of the new ligands bound in the cavity corresponded closely to predictions from the free energy calcns., but sometimes differed from those predicted by docking.  Finally, the authors examd. the impact of holding the protein rigid, as in docking, with a view to learning how approxns. made in docking affect accuracy and how they may be improved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD1-EYEODQ47Vg90H21EOLACvtfcHk0liFOgxCOlZCUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXot12rt7o%253D&md5=2480139daa5a0857c59e821d2ecbce18</span></div><a href="/servlet/linkout?suffix=cit169&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2007.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2007.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DMcReynolds%26aufirst%3DA.%2BC.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DPredicting%2520absolute%2520ligand%2520binding%2520free%2520energies%2520to%2520a%2520simple%2520model%2520site%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D371%26spage%3D1118%26epage%3D1134%26doi%3D10.1016%2Fj.jmb.2007.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref170"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref170'); return false;" data-citation="" class="refNumLink">170</a></strong><div class="NLM_citation" id="cit170"><span><span class="NLM_contrib-group">Rocklin, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyce, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span> </span><span class="NLM_article-title">Blind prediction of charged ligand binding affinities in a model binding site</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">425</span><span class="NLM_x">, </span> <span class="NLM_fpage">4569</span><span class="NLM_x">–</span> <span class="NLM_lpage">4583</span><span class="refDoi"> DOI: 10.1016/j.jmb.2013.07.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.jmb.2013.07.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=23896298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1OjtrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=425&publication_year=2013&pages=4569-4583&author=G.+J.+Rocklinauthor=S.+E.+Boyceauthor=M.+Fischerauthor=I.+Fishauthor=D.+L.+Mobleyauthor=B.+K.+Shoichetauthor=K.+A.+Dill&title=Blind+prediction+of+charged+ligand+binding+affinities+in+a+model+binding+site&doi=10.1016%2Fj.jmb.2013.07.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit170R"><div class="casContent"><span class="casTitleNuber">170</span><div class="casTitle"><span class="NLM_cas:atitle">Blind Prediction of Charged Ligand Binding Affinities in a Model Binding Site</span></div><div class="casAuthors">Rocklin, Gabriel J.; Boyce, Sarah E.; Fischer, Marcus; Fish, Inbar; Mobley, David L.; Shoichet, Brian K.; Dill, Ken A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">425</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">4569-4583</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Predicting abs. protein-ligand binding affinities remains a frontier challenge in ligand discovery and design.  This becomes more difficult when ionic interactions are involved because of the large opposing solvation and electrostatic attraction energies.  In a blind test, we examd. whether alchem. free-energy calcns. could predict binding affinities of 14 charged and 5 neutral compds. previously untested as ligands for a cavity binding site in cytochrome c peroxidase.  In this simplified site, polar and cationic ligands compete with solvent to interact with a buried aspartate.  Predictions were tested by calorimetry, spectroscopy, and crystallog.  Of the 15 compds. predicted to bind, 13 were exptl. confirmed, while 4 compds. were false neg. predictions.  Predictions had a root-mean-square error of 1.95 kcal/mol to the exptl. affinities, and predicted poses had an av. RMSD of 1.7 Å to the crystallog. poses.  This test serves as a benchmark for these thermodynamically rigorous calcns. at predicting binding affinities for charged compds. and gives insights into the existing sources of error, which are primarily electrostatic interactions inside proteins.  Our expts. also provide a useful set of ionic binding affinities in a simplified system for testing new affinity prediction methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZ_zM-ANClXLVg90H21EOLACvtfcHk0liFOgxCOlZCUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1OjtrnO&md5=d83561c4373ce7a24c7c040a00a0ea22</span></div><a href="/servlet/linkout?suffix=cit170&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2013.07.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2013.07.030%26sid%3Dliteratum%253Aachs%26aulast%3DRocklin%26aufirst%3DG.%2BJ.%26aulast%3DBoyce%26aufirst%3DS.%2BE.%26aulast%3DFischer%26aufirst%3DM.%26aulast%3DFish%26aufirst%3DI.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DBlind%2520prediction%2520of%2520charged%2520ligand%2520binding%2520affinities%2520in%2520a%2520model%2520binding%2520site%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2013%26volume%3D425%26spage%3D4569%26epage%3D4583%26doi%3D10.1016%2Fj.jmb.2013.07.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref171"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref171'); return false;" data-citation="" class="refNumLink">171</a></strong><div class="NLM_citation" id="cit171"><span><span class="NLM_contrib-group">Lippert, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Predescu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ierardi, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackenzie, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eastwood, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">Accurate and efficient integration for molecular dynamics simulations at constant temperature and pressure</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">139</span><span class="NLM_x">, </span> <span class="NLM_fpage">164106</span><span class="refDoi"> DOI: 10.1063/1.4825247</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1063%2F1.4825247" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24182003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Kisb%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2013&pages=164106&author=R.+A.+Lippertauthor=C.+Predescuauthor=D.+J.+Ierardiauthor=K.+M.+Mackenzieauthor=M.+P.+Eastwoodauthor=R.+O.+Drorauthor=D.+E.+Shaw&title=Accurate+and+efficient+integration+for+molecular+dynamics+simulations+at+constant+temperature+and+pressure&doi=10.1063%2F1.4825247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit171R"><div class="casContent"><span class="casTitleNuber">171</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate and efficient integration for molecular dynamics simulations at constant temperature and pressure</span></div><div class="casAuthors">Lippert, Ross A.; Predescu, Cristian; Ierardi, Douglas J.; MacKenzie, Kenneth M.; Eastwood, Michael P.; Dror, Ron O.; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">164106/1-164106/11</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">In mol. dynamics simulations, control over temp. and pressure is typically achieved by augmenting the original system with addnl. dynamical variables to create a thermostat and a barostat, resp.  These variables generally evolve on timescales much longer than those of particle motion, but typical integrator implementations update the addnl. variables along with the particle positions and momenta at each time step.  We present a framework that replaces the traditional integration procedure with sep. barostat, thermostat, and Newtonian particle motion updates, allowing thermostat and barostat updates to be applied infrequently.  Such infrequent updates provide a particularly substantial performance advantage for simulations parallelized across many computer processors, because thermostat and barostat updates typically require communication among all processors.  Infrequent updates can also improve accuracy by alleviating certain sources of error assocd. with limited-precision arithmetic.  In addn., sepg. the barostat, thermostat, and particle motion update steps reduces certain truncation errors, bringing the time-av. pressure closer to its target value.  Finally, this framework, which we have implemented on both general-purpose and special-purpose hardware, reduces software complexity and improves software modularity.  (c) 2013 American Institute of Physics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW51ZULlZFvrVg90H21EOLACvtfcHk0liFOgxCOlZCUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Kisb%252FL&md5=0bd59b4b73b6999b9311306f4c32f3c1</span></div><a href="/servlet/linkout?suffix=cit171&amp;dbid=16384&amp;doi=10.1063%2F1.4825247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.4825247%26sid%3Dliteratum%253Aachs%26aulast%3DLippert%26aufirst%3DR.%2BA.%26aulast%3DPredescu%26aufirst%3DC.%26aulast%3DIerardi%26aufirst%3DD.%2BJ.%26aulast%3DMackenzie%26aufirst%3DK.%2BM.%26aulast%3DEastwood%26aufirst%3DM.%2BP.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DAccurate%2520and%2520efficient%2520integration%2520for%2520molecular%2520dynamics%2520simulations%2520at%2520constant%2520temperature%2520and%2520pressure%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2013%26volume%3D139%26spage%3D164106%26doi%3D10.1063%2F1.4825247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref172"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref172'); return false;" data-citation="" class="refNumLink">172</a></strong><div class="NLM_citation" id="cit172"><span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valant, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borhani, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valcourt, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arlow, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canals, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahmani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sexton, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christopoulos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">Structural basis for modulation of a G-protein-coupled receptor by allosteric drugs</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">503</span><span class="NLM_x">, </span> <span class="NLM_fpage">295</span><span class="NLM_x">–</span> <span class="NLM_lpage">299</span><span class="refDoi"> DOI: 10.1038/nature12595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnature12595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24121438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslSns7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2013&pages=295-299&author=R.+O.+Drorauthor=H.+F.+Greenauthor=C.+Valantauthor=D.+W.+Borhaniauthor=J.+R.+Valcourtauthor=A.+C.+Panauthor=D.+H.+Arlowauthor=M.+Canalsauthor=J.+R.+Laneauthor=R.+Rahmaniauthor=J.+B.+Baellauthor=P.+M.+Sextonauthor=A.+Christopoulosauthor=D.+E.+Shaw&title=Structural+basis+for+modulation+of+a+G-protein-coupled+receptor+by+allosteric+drugs&doi=10.1038%2Fnature12595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit172R"><div class="casContent"><span class="casTitleNuber">172</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for modulation of a G-protein-coupled receptor by allosteric drugs</span></div><div class="casAuthors">Dror, Ron O.; Green, Hillary F.; Valant, Celine; Borhani, David W.; Valcourt, James R.; Pan, Albert C.; Arlow, Daniel H.; Canals, Meritxell; Lane, J. Robert; Rahmani, Raphael; Baell, Jonathan B.; Sexton, Patrick M.; Christopoulos, Arthur; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">503</span>
        (<span class="NLM_cas:issue">7475</span>),
    <span class="NLM_cas:pages">295-299</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The design of G-protein-coupled receptor (GPCR) allosteric modulators, an active area of modern pharmaceutical research, has proved challenging because neither the binding modes nor the mol. mechanisms of such drugs are known.  Here we det. binding sites, bound conformations and specific drug-receptor interactions for several allosteric modulators of the M2 muscarinic acetylcholine receptor (M2 receptor), a prototypical family A GPCR, using at.-level simulations in which the modulators spontaneously assoc. with the receptor.  Despite substantial structural diversity, all modulators form cation-π interactions with clusters of arom. residues in the receptor extracellular vestibule, approx. 15 Å from the classical, 'orthosteric' ligand-binding site.  We validate the obsd. modulator binding modes through radioligand binding expts. on receptor mutants designed, on the basis of our simulations, either to increase or to decrease modulator affinity.  Simulations also revealed mechanisms that contribute to pos. and neg. allosteric modulation of classical ligand binding, including coupled conformational changes of the two binding sites and electrostatic interactions between ligands in these sites.  These observations enabled the design of chem. modifications that substantially alter a modulator's allosteric effects.  Our findings thus provide a structural basis for the rational design of allosteric modulators targeting muscarinic and possibly other GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTlvAcaBf-1rVg90H21EOLACvtfcHk0li-JUALm8JK5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslSns7nN&md5=2fa40ecd46ce62125ce4a12c18270b1f</span></div><a href="/servlet/linkout?suffix=cit172&amp;dbid=16384&amp;doi=10.1038%2Fnature12595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12595%26sid%3Dliteratum%253Aachs%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DGreen%26aufirst%3DH.%2BF.%26aulast%3DValant%26aufirst%3DC.%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DValcourt%26aufirst%3DJ.%2BR.%26aulast%3DPan%26aufirst%3DA.%2BC.%26aulast%3DArlow%26aufirst%3DD.%2BH.%26aulast%3DCanals%26aufirst%3DM.%26aulast%3DLane%26aufirst%3DJ.%2BR.%26aulast%3DRahmani%26aufirst%3DR.%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DStructural%2520basis%2520for%2520modulation%2520of%2520a%2520G-protein-coupled%2520receptor%2520by%2520allosteric%2520drugs%26jtitle%3DNature%26date%3D2013%26volume%3D503%26spage%3D295%26epage%3D299%26doi%3D10.1038%2Fnature12595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref173"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref173'); return false;" data-citation="" class="refNumLink">173</a></strong><div class="NLM_citation" id="cit173"><span><span class="NLM_contrib-group">Shukla, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hernandez, C. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span> </span><span class="NLM_article-title">Markov state models provide insights into dynamic modulation of protein function</span> <span class="citation_source-journal">Acc. Chem. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">414</span><span class="NLM_x">–</span> <span class="NLM_lpage">422</span><span class="refDoi"> DOI: 10.1021/ar5002999</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar5002999" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXks1Oksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2015&pages=414-422&author=D.+Shuklaauthor=C.+X.+Hernandezauthor=J.+K.+Weberauthor=V.+S.+Pande&title=Markov+state+models+provide+insights+into+dynamic+modulation+of+protein+function&doi=10.1021%2Far5002999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit173R"><div class="casContent"><span class="casTitleNuber">173</span><div class="casTitle"><span class="NLM_cas:atitle">Markov state models provide insights into dynamic modulation of protein function</span></div><div class="casAuthors">Shukla, Diwakar; Hernandez, Carlos X.; Weber, Jeffrey K.; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">414-422</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Protein function is inextricably linked to protein dynamics.  As one moves from a static structural picture to a dynamic ensemble view of protein structure and function, novel computational paradigms are required for observing and understanding conformational dynamics of proteins and its functional implications.  In principle, mol. dynamics simulations can provide the time evolution of atomistic models of proteins, but the long time scales assocd. with functional dynamics make it difficult to observe rare dynamical transitions.  The issue of extg. essential functional components of protein dynamics from noisy simulation data presents another set of challenges in obtaining an unbiased understanding of protein motions.  Therefore, a methodol. that provides a statistical framework for efficient sampling and a human-readable view of the key aspects of functional dynamics from data anal. is required.  The Markov state model (MSM), which has recently become popular worldwide for studying protein dynamics, is an example of such a framework.  Here, the authors review the use of MSMs for efficient sampling of the hierarchy of time scales assocd. with protein dynamics, automatic identification of key conformational states, and the degrees of freedom assocd. with slow dynamical processes.  Applications of MSMs for studying long time scale phenomena such as activation mechanisms of cellular signaling proteins has yielded novel insights into protein function.  In particular, from MSMs built using large-scale simulations of GPCRs and kinases, the authors have shown that complex conformational changes in proteins can be described in terms of structural changes in key structural motifs or "mol. switches" within the protein, that the transitions between functionally active and inactive states of proteins proceed via multiple pathways, and that ligand or substrate binding modulates the flux through these pathways.  Finally, MSMs also provide a theor. toolbox for studying the effect of nonequil. perturbations on conformational dynamics.  Considering that protein dynamics in vivo occur under nonequil. conditions, MSMs coupled with nonequil. statistical mechanics provide a way to connect cellular components to their functional environments.  Nonequil. perturbations of protein folding MSMs reveal the presence of dynamically frozen glass-like states in their conformational landscape.  These frozen states are also obsd. to be rich in β-sheets, which indicates their possible role in the nucleation of β-sheet rich aggregates such as those obsd. in amyloid-fibril formation.  Finally, the authors describe how MSMs have been used to understand the dynamical behavior of intrinsically disordered proteins such as amyloid-β, human islet amyloid polypeptide, and p53.  While certainly not a panacea for studying functional dynamics, MSMs provide a rigorous theor. foundation for understanding complex entropically dominated processes and a convenient lens for viewing protein motions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_eCbE2nT1prVg90H21EOLACvtfcHk0li-JUALm8JK5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXks1Oksw%253D%253D&md5=d7d0042973fdc6a4f28d6259729dcebe</span></div><a href="/servlet/linkout?suffix=cit173&amp;dbid=16384&amp;doi=10.1021%2Far5002999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far5002999%26sid%3Dliteratum%253Aachs%26aulast%3DShukla%26aufirst%3DD.%26aulast%3DHernandez%26aufirst%3DC.%2BX.%26aulast%3DWeber%26aufirst%3DJ.%2BK.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DMarkov%2520state%2520models%2520provide%2520insights%2520into%2520dynamic%2520modulation%2520of%2520protein%2520function%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2015%26volume%3D48%26spage%3D414%26epage%3D422%26doi%3D10.1021%2Far5002999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref174"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref174'); return false;" data-citation="" class="refNumLink">174</a></strong><div class="NLM_citation" id="cit174"><span><span class="NLM_contrib-group">Barelier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eidam, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollander, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figaroa, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nachane, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siegal, G.</span><span> </span><span class="NLM_article-title">Increasing chemical space coverage by combining empirical and computational fragment screens</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1528</span><span class="NLM_x">–</span> <span class="NLM_lpage">1535</span><span class="refDoi"> DOI: 10.1021/cb5001636</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb5001636" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnsFSksbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1528-1535&author=S.+Barelierauthor=O.+Eidamauthor=I.+Fishauthor=J.+Hollanderauthor=F.+Figaroaauthor=R.+Nachaneauthor=J.+J.+Irwinauthor=B.+K.+Shoichetauthor=G.+Siegal&title=Increasing+chemical+space+coverage+by+combining+empirical+and+computational+fragment+screens&doi=10.1021%2Fcb5001636"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit174R"><div class="casContent"><span class="casTitleNuber">174</span><div class="casTitle"><span class="NLM_cas:atitle">Increasing Chemical Space Coverage by Combining Empirical and Computational Fragment Screens</span></div><div class="casAuthors">Barelier, Sarah; Eidam, Oliv; Fish, Inbar; Hollander, Johan; Figaroa, Francis; Nachane, Ruta; Irwin, John J.; Shoichet, Brian K.; Siegal, Gregg</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1528-1535</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Most libraries for fragment-based drug discovery are restricted to 1,000-10,000 compds., but over 500,000 fragments are com. available and potentially accessible by virtual screening.  Whether this larger set would increase chemotype coverage, and whether a computational screen can pragmatically prioritize them, is debated.  To investigate this question, a 1281-fragment library was screened by NMR (NMR) against AmpC β-lactamase, and hits were confirmed by surface plasmon resonance (SPR).  Nine hits with novel chemotypes were confirmed biochem. with KI values from 0.2 to low mM.  We also computationally docked 290,000 purchasable fragments with chemotypes unrepresented in the empirical library, finding 10 that had KI values from 0.03 to low mM.  Though less novel than those discovered by NMR, the docking-derived fragments filled chemotype holes from the empirical library.  Crystal structures of nine of the fragments in complex with AmpC β-lactamase revealed new binding sites and explained the relatively high affinity of the docking-derived fragments.  The existence of chemotype holes is likely a general feature of fragment libraries, as calcn. suggests that to represent the fragment substructures of even known biogenic mols. would demand a library of minimally over 32,000 fragments.  Combining computational and empirical fragment screens enables the discovery of unexpected chemotypes, here by the NMR screen, while capturing chemotypes missing from the empirical library and tailored to the target, with little extra cost in resources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5DNyK9V50nLVg90H21EOLACvtfcHk0li-JUALm8JK5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnsFSksbg%253D&md5=7c75e304e70fec6d9209c21a87a06a17</span></div><a href="/servlet/linkout?suffix=cit174&amp;dbid=16384&amp;doi=10.1021%2Fcb5001636&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb5001636%26sid%3Dliteratum%253Aachs%26aulast%3DBarelier%26aufirst%3DS.%26aulast%3DEidam%26aufirst%3DO.%26aulast%3DFish%26aufirst%3DI.%26aulast%3DHollander%26aufirst%3DJ.%26aulast%3DFigaroa%26aufirst%3DF.%26aulast%3DNachane%26aufirst%3DR.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DSiegal%26aufirst%3DG.%26atitle%3DIncreasing%2520chemical%2520space%2520coverage%2520by%2520combining%2520empirical%2520and%2520computational%2520fragment%2520screens%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1528%26epage%3D1535%26doi%3D10.1021%2Fcb5001636" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref175"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref175'); return false;" data-citation="" class="refNumLink">175</a></strong><div class="NLM_citation" id="cit175"><span><span class="NLM_contrib-group">London, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krishnan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uchida, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eidam, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibold, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cimermancic, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taunton, J.</span><span> </span><span class="NLM_article-title">Covalent docking of large libraries for the discovery of chemical probes</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1066</span><span class="NLM_x">–</span> <span class="NLM_lpage">1072</span><span class="refDoi"> DOI: 10.1038/nchembio.1666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnchembio.1666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25344815" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVSqsbjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=1066-1072&author=N.+Londonauthor=R.+M.+Millerauthor=S.+Krishnanauthor=K.+Uchidaauthor=J.+J.+Irwinauthor=O.+Eidamauthor=L.+Giboldauthor=P.+Cimermancicauthor=R.+Bonnetauthor=B.+K.+Shoichetauthor=J.+Taunton&title=Covalent+docking+of+large+libraries+for+the+discovery+of+chemical+probes&doi=10.1038%2Fnchembio.1666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit175R"><div class="casContent"><span class="casTitleNuber">175</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent docking of large libraries for the discovery of chemical probes</span></div><div class="casAuthors">London, Nir; Miller, Rand M.; Krishnan, Shyam; Uchida, Kenji; Irwin, John J.; Eidam, Oliv; Gibold, Lucie; Cimermancic, Peter; Bonnet, Richard; Shoichet, Brian K.; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1066-1072</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes that form a covalent bond with a protein target often show enhanced selectivity, potency and utility for biol. studies.  Despite these advantages, protein-reactive compds. are usually avoided in high-throughput screening campaigns.  Here we describe a general method (DOCKovalent) for screening large virtual libraries of electrophilic small mols.  We apply this method prospectively to discover reversible covalent fragments that target distinct protein nucleophiles, including the catalytic serine of AmpC β-lactamase and noncatalytic cysteines in RSK2, MSK1 and JAK3 kinases.  We identify submicromolar to low-nanomolar hits with high ligand efficiency, cellular activity and selectivity, including what are to our knowledge the first reported reversible covalent inhibitors of JAK3.  Crystal structures of inhibitor complexes with AmpC and RSK2 confirm the docking predictions and guide further optimization.  As covalent virtual screening may have broad utility for the rapid discovery of chem. probes, we have made the method freely available through an automated web server (http://covalent.docking.org/).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpngQdIYqslNLVg90H21EOLACvtfcHk0liYC6W5M6S3-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVSqsbjE&md5=4e5fbeb8f67cb82d464d57069ba0c93a</span></div><a href="/servlet/linkout?suffix=cit175&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1666%26sid%3Dliteratum%253Aachs%26aulast%3DLondon%26aufirst%3DN.%26aulast%3DMiller%26aufirst%3DR.%2BM.%26aulast%3DKrishnan%26aufirst%3DS.%26aulast%3DUchida%26aufirst%3DK.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DEidam%26aufirst%3DO.%26aulast%3DGibold%26aufirst%3DL.%26aulast%3DCimermancic%26aufirst%3DP.%26aulast%3DBonnet%26aufirst%3DR.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DCovalent%2520docking%2520of%2520large%2520libraries%2520for%2520the%2520discovery%2520of%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D1066%26epage%3D1072%26doi%3D10.1038%2Fnchembio.1666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref176"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref176'); return false;" data-citation="" class="refNumLink">176</a></strong><div class="NLM_citation" id="cit176"><span><span class="NLM_contrib-group">Bianco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodsell, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">Covalent docking using autodock: Two-point attractor and flexible side chain methods</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">295</span><span class="NLM_x">–</span> <span class="NLM_lpage">301</span><span class="refDoi"> DOI: 10.1002/pro.2733</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fpro.2733" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26103917" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFehs7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2016&pages=295-301&author=G.+Biancoauthor=S.+Forliauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=Covalent+docking+using+autodock%3A+Two-point+attractor+and+flexible+side+chain+methods&doi=10.1002%2Fpro.2733"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit176R"><div class="casContent"><span class="casTitleNuber">176</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent docking using autodock: Two-point attractor and flexible side chain methods</span></div><div class="casAuthors">Bianco, Giulia; Forli, Stefano; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">295-301</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">We describe two methods of automated covalent docking using Autodock4: the two-point attractor method and the flexible side chain method.  Both methods were applied to a training set of 20 diverse protein-ligand covalent complexes, evaluating their reliability in predicting the crystallog. pose of the ligands.  The flexible side chain method performed best, recovering the pose in 75% of cases, with failures for the largest inhibitors tested.  Both methods are freely available at the AutoDock website ().</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbgwvwS_NTKrVg90H21EOLACvtfcHk0liYC6W5M6S3-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFehs7rI&md5=4e380f340983e78e149f48934ccc59fd</span></div><a href="/servlet/linkout?suffix=cit176&amp;dbid=16384&amp;doi=10.1002%2Fpro.2733&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.2733%26sid%3Dliteratum%253Aachs%26aulast%3DBianco%26aufirst%3DG.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DCovalent%2520docking%2520using%2520autodock%253A%2520Two-point%2520attractor%2520and%2520flexible%2520side%2520chain%2520methods%26jtitle%3DProtein%2520Sci.%26date%3D2016%26volume%3D25%26spage%3D295%26epage%3D301%26doi%3D10.1002%2Fpro.2733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref177"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref177'); return false;" data-citation="" class="refNumLink">177</a></strong><div class="NLM_citation" id="cit177"><span><span class="NLM_contrib-group">Zhu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borrelli, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farid, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harder, E.</span><span> </span><span class="NLM_article-title">Docking covalent inhibitors: a parameter free approach to pose prediction and scoring</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1932</span><span class="NLM_x">–</span> <span class="NLM_lpage">1940</span><span class="refDoi"> DOI: 10.1021/ci500118s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500118s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpsVKksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1932-1940&author=K.+Zhuauthor=K.+W.+Borrelliauthor=J.+R.+Greenwoodauthor=T.+Dayauthor=R.+Abelauthor=R.+S.+Faridauthor=E.+Harder&title=Docking+covalent+inhibitors%3A+a+parameter+free+approach+to+pose+prediction+and+scoring&doi=10.1021%2Fci500118s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit177R"><div class="casContent"><span class="casTitleNuber">177</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Covalent Inhibitors: A Parameter Free Approach To Pose Prediction and Scoring</span></div><div class="casAuthors">Zhu, Kai; Borrelli, Kenneth W.; Greenwood, Jeremy R.; Day, Tyler; Abel, Robert; Farid, Ramy S.; Harder, Edward</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1932-1940</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Although many popular docking programs include a facility to account for covalent ligands, large-scale systematic docking validation studies of covalent inhibitors have been sparse.  In this paper, we present the development and validation of a novel approach for docking and scoring covalent inhibitors, which consists of conventional noncovalent docking, heuristic formation of the covalent attachment point, and structural refinement of the protein-ligand complex.  This approach combines the strengths of the docking program Glide and the protein structure modeling program Prime and does not require any parameter fitting for the study of addnl. covalent reaction types.  We first test this method by predicting the native binding geometry of 38 covalently bound complexes.  The av. RMSD of the predicted poses is 1.52 Å, and 76% of test set inhibitors have an RMSD of less than 2.0 Å.  In addn., the apparent affinity score constructed herein is tested on a virtual screening study and the characterization of the SAR properties of two different series of congeneric compds. with satisfactory success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbtjEb3EqzQ7Vg90H21EOLACvtfcHk0liYC6W5M6S3-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpsVKksLs%253D&md5=3633c29fa5dc29817b178fd10a4c7b15</span></div><a href="/servlet/linkout?suffix=cit177&amp;dbid=16384&amp;doi=10.1021%2Fci500118s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500118s%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DBorrelli%26aufirst%3DK.%2BW.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DFarid%26aufirst%3DR.%2BS.%26aulast%3DHarder%26aufirst%3DE.%26atitle%3DDocking%2520covalent%2520inhibitors%253A%2520a%2520parameter%2520free%2520approach%2520to%2520pose%2520prediction%2520and%2520scoring%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1932%26epage%3D1940%26doi%3D10.1021%2Fci500118s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref178"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref178'); return false;" data-citation="" class="refNumLink">178</a></strong><div class="NLM_citation" id="cit178"><span><span class="NLM_contrib-group">Ouyang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ge, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwoh, C. K.</span><span> </span><span class="NLM_article-title">CovalentDock Cloud: a web server for automated covalent docking</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">W329</span><span class="NLM_x">–</span> <span class="NLM_lpage">332</span><span class="refDoi"> DOI: 10.1093/nar/gkt406</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1093%2Fnar%2Fgkt406" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=W329-332&author=X.+Ouyangauthor=S.+Zhouauthor=Z.+Geauthor=R.+Liauthor=C.+K.+Kwoh&title=CovalentDock+Cloud%3A+a+web+server+for+automated+covalent+docking&doi=10.1093%2Fnar%2Fgkt406"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit178&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt406%26sid%3Dliteratum%253Aachs%26aulast%3DOuyang%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DGe%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DKwoh%26aufirst%3DC.%2BK.%26atitle%3DCovalentDock%2520Cloud%253A%2520a%2520web%2520server%2520for%2520automated%2520covalent%2520docking%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3DW329%26epage%3D332%26doi%3D10.1093%2Fnar%2Fgkt406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref179"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref179'); return false;" data-citation="" class="refNumLink">179</a></strong><div class="NLM_citation" id="cit179"><span><span class="NLM_contrib-group">Ouyang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ge, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwoh, C. K.</span><span> </span><span class="NLM_article-title">CovalentDock: automated covalent docking with parameterized covalent linkage energy estimation and molecular geometry constraints</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">326</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1002/jcc.23136</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fjcc.23136" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=23034731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVKksLbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2013&pages=326-336&author=X.+Ouyangauthor=S.+Zhouauthor=C.+T.+Suauthor=Z.+Geauthor=R.+Liauthor=C.+K.+Kwoh&title=CovalentDock%3A+automated+covalent+docking+with+parameterized+covalent+linkage+energy+estimation+and+molecular+geometry+constraints&doi=10.1002%2Fjcc.23136"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit179R"><div class="casContent"><span class="casTitleNuber">179</span><div class="casTitle"><span class="NLM_cas:atitle">CovalentDock: Automated covalent docking with parameterized covalent linkage energy estimation and molecular geometry constrains</span></div><div class="casAuthors">Ouyang, Xuchang; Zhou, Shuo; Su, Chinh Tran To; Ge, Zemei; Li, Runtao; Kwoh, Chee Keong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">326-336</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Covalent linkage formation is a very important mechanism for many covalent drugs to work.  However, partly due to the limitations of proper computational tools for covalent docking, most covalent drugs are not discovered systematically.  In this article, we present a new covalent docking package, the CovalentDock, built on the top of the source code of Autodock.  We developed an empirical model of free energy change estn. for covalent linkage formation, which is compatible with existing scoring functions used in docking, while handling the mol. geometry constrains of the covalent linkage with special atom types and directional grid maps.  Integrated prepn. scripts are also written for the automation of the whole covalent docking workflow.  The result tested on existing crystal structures with covalent linkage shows that CovalentDock can reproduce the native covalent complexes with significant improved accuracy when compared with the default covalent docking method in Autodock.  Expts. also suggest that CovalentDock is capable of covalent virtual screening with satisfactory enrichment performance.  In addn., the investigation on the results also shows that the chirality and target selectivity along with the mol. geometry constrains are well preserved by CovalentDock, showing great capability of this method in the application for covalent drug discovery. © 2012 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1kpnxsCasoLVg90H21EOLACvtfcHk0lhSwgVW9THrIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVKksLbJ&md5=5f548c115755f2e1470848ba6f6d701b</span></div><a href="/servlet/linkout?suffix=cit179&amp;dbid=16384&amp;doi=10.1002%2Fjcc.23136&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.23136%26sid%3Dliteratum%253Aachs%26aulast%3DOuyang%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DSu%26aufirst%3DC.%2BT.%26aulast%3DGe%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DKwoh%26aufirst%3DC.%2BK.%26atitle%3DCovalentDock%253A%2520automated%2520covalent%2520docking%2520with%2520parameterized%2520covalent%2520linkage%2520energy%2520estimation%2520and%2520molecular%2520geometry%2520constraints%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2013%26volume%3D34%26spage%3D326%26epage%3D336%26doi%3D10.1002%2Fjcc.23136" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref180"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref180'); return false;" data-citation="" class="refNumLink">180</a></strong><div class="NLM_citation" id="cit180"><span><span class="NLM_contrib-group">Toledo Warshaviak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golan, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borrelli, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalid, O.</span><span> </span><span class="NLM_article-title">Structure-based virtual screening approach for discovery of covalently bound ligands</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1941</span><span class="NLM_x">–</span> <span class="NLM_lpage">1950</span><span class="refDoi"> DOI: 10.1021/ci500175r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500175r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslegtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1941-1950&author=D.+Toledo+Warshaviakauthor=G.+Golanauthor=K.+W.+Borrelliauthor=K.+Zhuauthor=O.+Kalid&title=Structure-based+virtual+screening+approach+for+discovery+of+covalently+bound+ligands&doi=10.1021%2Fci500175r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit180R"><div class="casContent"><span class="casTitleNuber">180</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Virtual Screening Approach for Discovery of Covalently Bound Ligands</span></div><div class="casAuthors">Toledo Warshaviak, Dora; Golan, Gali; Borrelli, Kenneth W.; Zhu, Kai; Kalid, Ori</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1941-1950</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a fast and effective covalent docking approach suitable for large-scale virtual screening (VS).  We applied this method to four targets (HCV NS3 protease, Cathepsin K, EGFR, and XPO1) with known crystal structures and known covalent inhibitors.  We implemented a customized "VS mode" of the Schrodinger Covalent Docking algorithm (CovDock), which we refer to as CovDock-VS.  Known actives and target-specific sets of decoys were docked to selected X-ray structures, and poses were filtered based on noncovalent protein-ligand interactions known to be important for activity.  We were able to retrieve 71%, 72%, and 77% of the known actives for Cathepsin K, HCV NS3 protease, and EGFR within 5% of the decoy library, resp.  With the more challenging XPO1 target, where no specific interactions with the protein could be used for postprocessing of the docking results, we were able to retrieve 95% of the actives within 30% of the decoy library and achieved an early enrichment factor (EF1%) of 33.  The poses of the known actives bound to existing crystal structures of 4 targets were predicted with an av. RMSD of 1.9 Å.  To the best of our knowledge, CovDock-VS is the first fully automated tool for efficient virtual screening of covalent inhibitors.  Importantly, CovDock-VS can handle multiple chem. reactions within the same library, only requiring a generic SMARTS-based predefinition of the reaction.  CovDock-VS provides a fast and accurate way of differentiating actives from decoys without significantly deteriorating the accuracy of the predicted poses for covalent protein-ligand complexes.  Therefore, we propose CovDock-VS as an efficient structure-based virtual screening method for discovery of novel and diverse covalent ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-DH45VZl-w7Vg90H21EOLACvtfcHk0lhSwgVW9THrIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslegtbw%253D&md5=0ff8b6e682473abd2e2dadb49d95539a</span></div><a href="/servlet/linkout?suffix=cit180&amp;dbid=16384&amp;doi=10.1021%2Fci500175r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500175r%26sid%3Dliteratum%253Aachs%26aulast%3DToledo%2BWarshaviak%26aufirst%3DD.%26aulast%3DGolan%26aufirst%3DG.%26aulast%3DBorrelli%26aufirst%3DK.%2BW.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DKalid%26aufirst%3DO.%26atitle%3DStructure-based%2520virtual%2520screening%2520approach%2520for%2520discovery%2520of%2520covalently%2520bound%2520ligands%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1941%26epage%3D1950%26doi%3D10.1021%2Fci500175r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref181"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref181'); return false;" data-citation="" class="refNumLink">181</a></strong><div class="NLM_citation" id="cit181"><span><span class="NLM_contrib-group">Munoz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sospedra, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gomara, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mestres, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haro, I.</span><span> </span><span class="NLM_article-title">The covalent coupling of HAV-VP3 (110–121) synthetic peptide to liposomes: physicochemical studies</span> <span class="citation_source-journal">Int. J. Pharm.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">269</span><span class="NLM_x">, </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">184</span><span class="refDoi"> DOI: 10.1016/j.ijpharm.2003.09.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref181/cit181&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.ijpharm.2003.09.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref181/cit181&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=14698589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref181/cit181&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvFektLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=269&publication_year=2004&pages=177-184&author=M.+Munozauthor=P.+Sospedraauthor=M.+J.+Gomaraauthor=C.+Mestresauthor=I.+Haro&title=The+covalent+coupling+of+HAV-VP3+%28110%E2%80%93121%29+synthetic+peptide+to+liposomes%3A+physicochemical+studies&doi=10.1016%2Fj.ijpharm.2003.09.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit181R"><div class="casContent"><span class="casTitleNuber">181</span><div class="casTitle"><span class="NLM_cas:atitle">The covalent coupling of HAV-VP3 (110-121) synthetic peptide to liposomes: physicochemical studies</span></div><div class="casAuthors">Munoz, M.; Sospedra, P.; Gomara, M. J.; Mestres, C.; Haro, I.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Pharmaceutics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">269</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-184</span>CODEN:
                <span class="NLM_cas:coden">IJPHDE</span>;
        ISSN:<span class="NLM_cas:issn">0378-5173</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">In this work we have studied the conjugation of the immunogenic peptide sequence (110-121) belonging to the VP3 capsid protein of hepatitis A virus to the surface of preformed liposomes by means of an amide bond between the vesicles and the synthetic peptide.  The surface activity of the conjugate at air/water interface was detd.  Moreover, the interaction of the conjugate with lipids was also studied recording the pressure increases produced after the injection of the liposome-peptide prepn. under dipalmitoylphosphatidylcholine (DPPC), dipalmitoylphosphatidylglycerol (DPPG) and stearylamine (SA) monolayers at different initial surface pressures.  As expected, due to the neg. net charge of the liposome-peptide complex, the higher interaction was found with pos. charge monolayers (SA).  However, the conjugate was also able to incorporate to zwitterionic and anionic lipids.  This behavior was also confirmed performing compression isotherms of monolayers of these lipids spread on subphases contg. the conjugate.These results suggest that the coupling of VP3 (110-121) to liposomes does not influence its ability to interact with membrane lipids such as DPPC and DPPG.  Then it can be assumed that its immunogenicity will be preserved or even increased after this modification.  All these results are also useful in the prepn. of liposome-based synthetic peptide vaccines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoszv87GCrVVLVg90H21EOLACvtfcHk0lhSwgVW9THrIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpvFektLo%253D&md5=41ee4ea1d9fff23d66094262cb320449</span></div><a href="/servlet/linkout?suffix=cit181&amp;dbid=16384&amp;doi=10.1016%2Fj.ijpharm.2003.09.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijpharm.2003.09.014%26sid%3Dliteratum%253Aachs%26aulast%3DMunoz%26aufirst%3DM.%26aulast%3DSospedra%26aufirst%3DP.%26aulast%3DGomara%26aufirst%3DM.%2BJ.%26aulast%3DMestres%26aufirst%3DC.%26aulast%3DHaro%26aufirst%3DI.%26atitle%3DThe%2520covalent%2520coupling%2520of%2520HAV-VP3%2520%2528110%25E2%2580%2593121%2529%2520synthetic%2520peptide%2520to%2520liposomes%253A%2520physicochemical%2520studies%26jtitle%3DInt.%2520J.%2520Pharm.%26date%3D2004%26volume%3D269%26spage%3D177%26epage%3D184%26doi%3D10.1016%2Fj.ijpharm.2003.09.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref182"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref182'); return false;" data-citation="" class="refNumLink">182</a></strong><div class="NLM_citation" id="cit182"><span><span class="NLM_contrib-group">Scholz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knorr, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamacher, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, B.</span><span> </span><span class="NLM_article-title">DOCKTITE-a highly versatile step-by-step workflow for covalent docking and virtual screening in the molecular operating environment</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">398</span><span class="NLM_x">–</span> <span class="NLM_lpage">406</span><span class="refDoi"> DOI: 10.1021/ci500681r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500681r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFGrsrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=398-406&author=C.+Scholzauthor=S.+Knorrauthor=K.+Hamacherauthor=B.+Schmidt&title=DOCKTITE-a+highly+versatile+step-by-step+workflow+for+covalent+docking+and+virtual+screening+in+the+molecular+operating+environment&doi=10.1021%2Fci500681r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit182R"><div class="casContent"><span class="casTitleNuber">182</span><div class="casTitle"><span class="NLM_cas:atitle">DOCKTITE-A Highly Versatile Step-by-Step Workflow for Covalent Docking and Virtual Screening in the Molecular Operating Environment</span></div><div class="casAuthors">Scholz, Christoph; Knorr, Sabine; Hamacher, Kay; Schmidt, Boris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">398-406</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The formation of a covalent bond with the target is essential for a no. of successful drugs, yet tools for covalent docking without significant restrictions regarding warhead or receptor classes are rare and limited in use.  In this work we present DOCKTITE, a highly versatile workflow for covalent docking in the Mol. Operating Environment (MOE) combining automated warhead screening, nucleophilic side chain attachment, pharmacophore-based docking, and a novel consensus scoring approach.  The comprehensive validation study includes pose predictions of 35 protein/ligand complexes which resulted in a mean RMSD of 1.74 Å and a prediction rate of 71.4% with an RMSD below 2 Å, a virtual screening with an area under the curve (AUC) for the receiver operating characteristics (ROC) of 0.81, and a significant correlation between predicted and exptl. binding affinities (ρ = 0.806, R2 = 0.649, p < 0.005).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkYai-iGLMS7Vg90H21EOLACvtfcHk0lh3VWOcm0o2PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFGrsrrK&md5=738e69cfc93c5ce0f053b81837f632d7</span></div><a href="/servlet/linkout?suffix=cit182&amp;dbid=16384&amp;doi=10.1021%2Fci500681r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500681r%26sid%3Dliteratum%253Aachs%26aulast%3DScholz%26aufirst%3DC.%26aulast%3DKnorr%26aufirst%3DS.%26aulast%3DHamacher%26aufirst%3DK.%26aulast%3DSchmidt%26aufirst%3DB.%26atitle%3DDOCKTITE-a%2520highly%2520versatile%2520step-by-step%2520workflow%2520for%2520covalent%2520docking%2520and%2520virtual%2520screening%2520in%2520the%2520molecular%2520operating%2520environment%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D398%26epage%3D406%26doi%3D10.1021%2Fci500681r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref183"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref183'); return false;" data-citation="" class="refNumLink">183</a></strong><div class="NLM_citation" id="cit183"><span><span class="NLM_contrib-group">Schmidt, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Welker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rieger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sahu, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sotriffer, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schirmeister, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engels, B.</span><span> </span><span class="NLM_article-title">Protocol for rational design of covalently interacting inhibitors</span> <span class="citation_source-journal">ChemPhysChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">3226</span><span class="NLM_x">–</span> <span class="NLM_lpage">3235</span><span class="refDoi"> DOI: 10.1002/cphc.201402542</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fcphc.201402542" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25251382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslCisLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=3226-3235&author=T.+C.+Schmidtauthor=A.+Welkerauthor=M.+Riegerauthor=P.+K.+Sahuauthor=C.+A.+Sotrifferauthor=T.+Schirmeisterauthor=B.+Engels&title=Protocol+for+rational+design+of+covalently+interacting+inhibitors&doi=10.1002%2Fcphc.201402542"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit183R"><div class="casContent"><span class="casTitleNuber">183</span><div class="casTitle"><span class="NLM_cas:atitle">Protocol for Rational Design of Covalently Interacting Inhibitors</span></div><div class="casAuthors">Schmidt, Thomas C.; Welker, Armin; Rieger, Max; Sahu, Prabhat K.; Sotriffer, Christoph A.; Schirmeister, Tanja; Engels, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">ChemPhysChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3226-3235</span>CODEN:
                <span class="NLM_cas:coden">CPCHFT</span>;
        ISSN:<span class="NLM_cas:issn">1439-4235</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The inhibition potencies of covalent inhibitors mainly result from the formation of a covalent bond to the enzyme during the inhibition mechanism.  This class of inhibitors has essentially been ignored in previous target-directed drug discovery projects because of concerns about possible side effects.  However, their advantages, such as higher binding energies and longer drug-target residence times moved them into the focus of recent investigations.  While the rational design of non-covalent inhibitors became std. the corresponding design of covalent inhibitors is still in its early stages.  Potent covalent inhibitors can be retrieved from large compd. libraries by covalent docking approaches but protocols are missing that can reliably predict the influence of variations in the substitution pattern on the affinity and/or reactivity of a given covalent inhibitor.  Hence, the wanted property profile can only be obtained from trial-and-error proceedings.  This paper presents an appropriate protocol which is able to predict improved covalent inhibitors.  It uses hybrid approaches, which mix quantum mech. (QM) and mol. mech. (MM) methods to predict variations in the reactivity of the inhibitor.  They are also used to compute the required information about the non-covalent enzyme-inhibitor complex.  Docking tools are employed to improve the inhibitor with respect to the non-covalent interactions formed in the binding site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodO7ngSMXNtLVg90H21EOLACvtfcHk0lh3VWOcm0o2PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslCisLnP&md5=8d5389e92ef83e2d7f90c48a4bea5103</span></div><a href="/servlet/linkout?suffix=cit183&amp;dbid=16384&amp;doi=10.1002%2Fcphc.201402542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcphc.201402542%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DT.%2BC.%26aulast%3DWelker%26aufirst%3DA.%26aulast%3DRieger%26aufirst%3DM.%26aulast%3DSahu%26aufirst%3DP.%2BK.%26aulast%3DSotriffer%26aufirst%3DC.%2BA.%26aulast%3DSchirmeister%26aufirst%3DT.%26aulast%3DEngels%26aufirst%3DB.%26atitle%3DProtocol%2520for%2520rational%2520design%2520of%2520covalently%2520interacting%2520inhibitors%26jtitle%3DChemPhysChem%26date%3D2014%26volume%3D15%26spage%3D3226%26epage%3D3235%26doi%3D10.1002%2Fcphc.201402542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref184"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref184'); return false;" data-citation="" class="refNumLink">184</a></strong><div class="NLM_citation" id="cit184"><span><span class="NLM_contrib-group">Dong, G. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calhoun, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalyanaraman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Branch, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mashiyama, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">London, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babbitt, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Prediction of substrates for glutathione transferases by covalent docking</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1687</span><span class="NLM_x">–</span> <span class="NLM_lpage">1699</span><span class="refDoi"> DOI: 10.1021/ci5001554</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci5001554" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref184/cit184&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnsVehuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1687-1699&author=G.+Q.+Dongauthor=S.+Calhounauthor=H.+Fanauthor=C.+Kalyanaramanauthor=M.+C.+Branchauthor=S.+T.+Mashiyamaauthor=N.+Londonauthor=M.+P.+Jacobsonauthor=P.+C.+Babbittauthor=B.+K.+Shoichetauthor=R.+N.+Armstrongauthor=A.+Sali&title=Prediction+of+substrates+for+glutathione+transferases+by+covalent+docking&doi=10.1021%2Fci5001554"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit184R"><div class="casContent"><span class="casTitleNuber">184</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of Substrates for Glutathione Transferases by Covalent Docking</span></div><div class="casAuthors">Dong, Guang Qiang; Calhoun, Sara; Fan, Hao; Kalyanaraman, Chakrapani; Branch, Megan C.; Mashiyama, Susan T.; London, Nir; Jacobson, Matthew P.; Babbitt, Patricia C.; Shoichet, Brian K.; Armstrong, Richard N.; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1687-1699</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzymes in the glutathione transferase (GST) superfamily catalyze the conjugation of glutathione (GSH) to electrophilic substrates.  As a consequence they are involved in a no. of key biol. processes, including protection of cells against chem. damage, steroid and prostaglandin biosynthesis, tyrosine catabolism, and cell apoptosis.  Although virtual screening has been used widely to discover substrates by docking potential noncovalent ligands into active site clefts of enzymes, docking has been rarely constrained by a covalent bond between the enzyme and ligand.  In this study, we investigate the accuracy of docking poses and substrate discovery in the GST superfamily, by docking 6738 potential ligands from the KEGG and MetaCyc compd. libraries into 14 representative GST enzymes with known structures and substrates using the PLOP program [Jacobson Proteins 2004, 55, 351].  For X-ray structures as receptors, one of the top 3 ranked models is within 3 Å all-atom root mean square deviation (RMSD) of the native complex in 11 of the 14 cases; the enrichment LogAUC value is better than random in all cases, and better than 25 in 7 of 11 cases.  For comparative models as receptors, near-native ligand-enzyme configurations are often sampled but difficult to rank highly.  For models based on templates with the highest sequence identity, the enrichment LogAUC is better than 25 in 5 of 11 cases, not significantly different from the crystal structures.  In conclusion, we show that covalent docking can be a useful tool for substrate discovery and point out specific challenges for future method improvement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1kupr30YhIbVg90H21EOLACvtfcHk0lh3VWOcm0o2PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnsVehuro%253D&md5=b24a9f0d0a51fcd65d66797e3f9688aa</span></div><a href="/servlet/linkout?suffix=cit184&amp;dbid=16384&amp;doi=10.1021%2Fci5001554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci5001554%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DG.%2BQ.%26aulast%3DCalhoun%26aufirst%3DS.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DKalyanaraman%26aufirst%3DC.%26aulast%3DBranch%26aufirst%3DM.%2BC.%26aulast%3DMashiyama%26aufirst%3DS.%2BT.%26aulast%3DLondon%26aufirst%3DN.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DBabbitt%26aufirst%3DP.%2BC.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DArmstrong%26aufirst%3DR.%2BN.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DPrediction%2520of%2520substrates%2520for%2520glutathione%2520transferases%2520by%2520covalent%2520docking%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1687%26epage%3D1699%26doi%3D10.1021%2Fci5001554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref185"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref185'); return false;" data-citation="" class="refNumLink">185</a></strong><div class="NLM_citation" id="cit185"><span><span class="NLM_contrib-group">Zhang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ji, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, Y.</span><span> </span><span class="NLM_article-title">Effective virtual screening strategy toward covalent ligands: identification of novel NEDD8-activating enzyme inhibitors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1785</span><span class="NLM_x">–</span> <span class="NLM_lpage">1797</span><span class="refDoi"> DOI: 10.1021/ci5002058</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci5002058" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXosFaitLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1785-1797&author=S.+Zhangauthor=J.+Tanauthor=Z.+Laiauthor=Y.+Liauthor=J.+Pangauthor=J.+Xiaoauthor=Z.+Huangauthor=Y.+Zhangauthor=H.+Jiauthor=Y.+Lai&title=Effective+virtual+screening+strategy+toward+covalent+ligands%3A+identification+of+novel+NEDD8-activating+enzyme+inhibitors&doi=10.1021%2Fci5002058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit185R"><div class="casContent"><span class="casTitleNuber">185</span><div class="casTitle"><span class="NLM_cas:atitle">Effective Virtual Screening Strategy toward Covalent Ligands: Identification of Novel NEDD8-Activating Enzyme Inhibitors</span></div><div class="casAuthors">Zhang, Shengping; Tan, Jiani; Lai, Zhonghui; Li, Ying; Pang, Junxia; Xiao, Jianhu; Huang, Zhangjian; Zhang, Yihua; Ji, Hui; Lai, Yisheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1785-1797</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The NEDD8-activating enzyme (NAE) is an emerging target for cancer therapy, which regulates the degrdn. and turnover of a variety of cancer-related proteins by activating the cullin-RING E3 ubiquitin ligases.  Among a limited no. of known NAE inhibitors, the covalent inhibitors have demonstrated the most potent efficacy through their covalently linked adducts with NEDD8.  Inspired by this unique mechanism, in this study, a novel combined strategy of virtual screening (VS) was adopted with the aim to identify diverse covalent inhibitors of NAE.  To be specific, a docking-enabled pharmacophore model was first built from the possible active conformations of chosen covalent inhibitors.  Meanwhile, a dynamic structure-based phamacophore was also established based on the snapshots derived from mol. dynamic simulation.  Subsequent screening of a focused ZINC database using these pharmacophore models combined with covalent docking discovered three novel active compds.  Among them, compd. LZ3 exhibited the most potent NAE inhibitory activity with an IC50 value of 1.06 ± 0.18 μM.  Furthermore, a cell-based washout expt. proved the proposed covalent binding mechanism for compd. LZ3, which confirmed the successful application of our combined VS strategy, indicating it may provide a viable soln. to systematically discover novel covalent ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhRJSduwx3CrVg90H21EOLACvtfcHk0li-9DPb9sXJcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXosFaitLw%253D&md5=07a5cf406764b37d2f68b8aba55649eb</span></div><a href="/servlet/linkout?suffix=cit185&amp;dbid=16384&amp;doi=10.1021%2Fci5002058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci5002058%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DTan%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DPang%26aufirst%3DJ.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJi%26aufirst%3DH.%26aulast%3DLai%26aufirst%3DY.%26atitle%3DEffective%2520virtual%2520screening%2520strategy%2520toward%2520covalent%2520ligands%253A%2520identification%2520of%2520novel%2520NEDD8-activating%2520enzyme%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1785%26epage%3D1797%26doi%3D10.1021%2Fci5002058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref186"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref186'); return false;" data-citation="" class="refNumLink">186</a></strong><div class="NLM_citation" id="cit186"><span><span class="NLM_contrib-group">Fakhouri, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Elimat, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurst, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reggio, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearce, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oberlies, N. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Croatt, M. P.</span><span> </span><span class="NLM_article-title">Isolation, semisynthesis, covalent docking and transforming growth factor beta-activated kinase 1 (TAK1)-inhibitory activities of (5Z)-7-oxozeaenol analogues</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">6993</span><span class="NLM_x">–</span> <span class="NLM_lpage">6999</span><span class="refDoi"> DOI: 10.1016/j.bmc.2015.09.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref186/cit186&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmc.2015.09.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref186/cit186&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26481152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref186/cit186&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Wjs7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=6993-6999&author=L.+Fakhouriauthor=T.+El-Elimatauthor=D.+P.+Hurstauthor=P.+H.+Reggioauthor=C.+J.+Pearceauthor=N.+H.+Oberliesauthor=M.+P.+Croatt&title=Isolation%2C+semisynthesis%2C+covalent+docking+and+transforming+growth+factor+beta-activated+kinase+1+%28TAK1%29-inhibitory+activities+of+%285Z%29-7-oxozeaenol+analogues&doi=10.1016%2Fj.bmc.2015.09.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit186R"><div class="casContent"><span class="casTitleNuber">186</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation, semisynthesis, covalent docking and transforming growth factor beta-activated kinase 1 (TAK1)-inhibitory activities of (5Z)-7-oxozeaenol analogues</span></div><div class="casAuthors">Fakhouri, Lara; El-Elimat, Tamam; Hurst, Dow P.; Reggio, Patricia H.; Pearce, Cedric J.; Oberlies, Nicholas H.; Croatt, Mitchell P.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6993-6999</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">(5Z)-7-Oxozeanol and related analogs were isolated and screened to explore their activity as TAK1 inhibitors.  Seven analogs were synthesized and more than a score of natural products isolated that examd. the role that different areas of the mol. contribute to TAK1 inhibition.  A novel nonarom. difluoro-deriv. was synthesized that had similar potency compared to the lead.  This is the first example of a nonarom. compd. in this class to have TAK1 inhibition.  Covalent docking for the isolated and synthesized analogs was carried out and a strong correlation between the obsd. activities and the calcd. binding was found.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJxKqASB5kirVg90H21EOLACvtfcHk0li-9DPb9sXJcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Wjs7vO&md5=b5b32bf90eeb6594dabfa738e874e26a</span></div><a href="/servlet/linkout?suffix=cit186&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.09.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.09.037%26sid%3Dliteratum%253Aachs%26aulast%3DFakhouri%26aufirst%3DL.%26aulast%3DEl-Elimat%26aufirst%3DT.%26aulast%3DHurst%26aufirst%3DD.%2BP.%26aulast%3DReggio%26aufirst%3DP.%2BH.%26aulast%3DPearce%26aufirst%3DC.%2BJ.%26aulast%3DOberlies%26aufirst%3DN.%2BH.%26aulast%3DCroatt%26aufirst%3DM.%2BP.%26atitle%3DIsolation%252C%2520semisynthesis%252C%2520covalent%2520docking%2520and%2520transforming%2520growth%2520factor%2520beta-activated%2520kinase%25201%2520%2528TAK1%2529-inhibitory%2520activities%2520of%2520%25285Z%2529-7-oxozeaenol%2520analogues%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D6993%26epage%3D6999%26doi%3D10.1016%2Fj.bmc.2015.09.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref187"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref187'); return false;" data-citation="" class="refNumLink">187</a></strong><div class="NLM_citation" id="cit187"><span><span class="NLM_contrib-group">Li, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">You, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span> </span><span class="NLM_article-title">Discovery of novel covalent proteasome inhibitors through a combination of pharmacophore screening, covalent docking, and molecular dynamics simulations</span> <span class="citation_source-journal">J. Mol. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">2515</span><span class="refDoi"> DOI: 10.1007/s00894-014-2515-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref187/cit187&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs00894-014-2515-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref187/cit187&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25394401" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref187/cit187&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BC2M3ot1WmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=2515&author=A.+Liauthor=H.+Sunauthor=L.+Duauthor=X.+Wuauthor=J.+Caoauthor=Q.+Youauthor=Y.+Li&title=Discovery+of+novel+covalent+proteasome+inhibitors+through+a+combination+of+pharmacophore+screening%2C+covalent+docking%2C+and+molecular+dynamics+simulations&doi=10.1007%2Fs00894-014-2515-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit187R"><div class="casContent"><span class="casTitleNuber">187</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel covalent proteasome inhibitors through a combination of pharmacophore screening, covalent docking, and molecular dynamics simulations</span></div><div class="casAuthors">Li Aibo; Sun Haopeng; Du Lei; Wu Xiaoxin; Cao Jianqin; You Qidong; Li Yuyan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2515</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The ubiquitin-proteasome pathway plays a pivotal role in the regulation of cellular protein processing and degradation.  Proteasome inhibitors (PIs) have enormous potential to treat multiple myeloma, solid tumors, parasites, inflammation, and immune diseases, which is spurring the development of new types of PIs with enhanced efficacy, fewer side effects, and reduced drug resistance.  Nevertheless, virtual screening for covalent PIs has rarely been reported because calculating the covalent binding energy is a challenging task.  The aim of this study was to discover new covalent inhibitors of the 20S proteasome.  The structures of PIs were manually divided into two parts: a noncovalent binding part resulting from virtual screening, and an epoxyketone group that was pre-selected as a covalent binding part.  The SPECS database was screened by noncovalent docking and a pharmacophore model built with the 20S proteasome.  After validating the covalent conjugation, 88 hits with epoxyketone were covalently docked into the 20S proteasome to analyze the intermolecular interactions.  Four compounds were selected after multiple filtration and validations.  Molecular dynamics simulations were performed to check the stability of the noncovalent and covalent docked ligand-enzyme complexes and investigate the interaction patterns of the screened inhibitors.  Finally, two compounds with novel aromatic backbones, reasonable interactions, and stable covalent binding modes were retained.  These compounds can serve as potential hits for further biological evaluation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRD0fkfwqKVeVHFYivNCdPRfW6udTcc2ebmIHsmt0mWprntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3ot1WmtA%253D%253D&md5=975a5c6ef3c6511c0c863a736fd05b4a</span></div><a href="/servlet/linkout?suffix=cit187&amp;dbid=16384&amp;doi=10.1007%2Fs00894-014-2515-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00894-014-2515-y%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DA.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DDu%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DYou%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520novel%2520covalent%2520proteasome%2520inhibitors%2520through%2520a%2520combination%2520of%2520pharmacophore%2520screening%252C%2520covalent%2520docking%252C%2520and%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Mol.%2520Model.%26date%3D2014%26volume%3D20%26spage%3D2515%26doi%3D10.1007%2Fs00894-014-2515-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref188"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref188'); return false;" data-citation="" class="refNumLink">188</a></strong><div class="NLM_citation" id="cit188"><span><span class="NLM_contrib-group">London, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farelli, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korczynska, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Obaidi, N. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babbitt, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almo, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Covalent docking predicts substrates for haloalkanoate dehalogenase superfamily phosphatases</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">528</span><span class="NLM_x">–</span> <span class="NLM_lpage">537</span><span class="refDoi"> DOI: 10.1021/bi501140k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi501140k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=528-537&author=N.+Londonauthor=J.+D.+Farelliauthor=S.+D.+Brownauthor=C.+Liuauthor=H.+Huangauthor=M.+Korczynskaauthor=N.+F.+Al-Obaidiauthor=P.+C.+Babbittauthor=S.+C.+Almoauthor=K.+N.+Allenauthor=B.+K.+Shoichet&title=Covalent+docking+predicts+substrates+for+haloalkanoate+dehalogenase+superfamily+phosphatases&doi=10.1021%2Fbi501140k"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit188&amp;dbid=16384&amp;doi=10.1021%2Fbi501140k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi501140k%26sid%3Dliteratum%253Aachs%26aulast%3DLondon%26aufirst%3DN.%26aulast%3DFarelli%26aufirst%3DJ.%2BD.%26aulast%3DBrown%26aufirst%3DS.%2BD.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DKorczynska%26aufirst%3DM.%26aulast%3DAl-Obaidi%26aufirst%3DN.%2BF.%26aulast%3DBabbitt%26aufirst%3DP.%2BC.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DAllen%26aufirst%3DK.%2BN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DCovalent%2520docking%2520predicts%2520substrates%2520for%2520haloalkanoate%2520dehalogenase%2520superfamily%2520phosphatases%26jtitle%3DBiochemistry%26date%3D2015%26volume%3D54%26spage%3D528%26epage%3D537%26doi%3D10.1021%2Fbi501140k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref189"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref189'); return false;" data-citation="" class="refNumLink">189</a></strong><div class="NLM_citation" id="cit189"><span><span class="NLM_contrib-group">Webb, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lasker, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneidman-Duhovny, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tjioe, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velazquez-Muriel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Modeling of proteins and their assemblies with the integrative modeling platform</span> <span class="citation_source-journal">Methods Mol. Biol. (N. Y., NY, U. S.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">781</span><span class="NLM_x">, </span> <span class="NLM_fpage">377</span><span class="NLM_x">–</span> <span class="NLM_lpage">397</span><span class="refDoi"> DOI: 10.1007/978-1-61779-276-2_19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref189/cit189&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2F978-1-61779-276-2_19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref189/cit189&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=21877292" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref189/cit189&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVClsbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=781&publication_year=2011&pages=377-397&author=B.+Webbauthor=K.+Laskerauthor=D.+Schneidman-Duhovnyauthor=E.+Tjioeauthor=J.+Phillipsauthor=S.+J.+Kimauthor=J.+Velazquez-Murielauthor=D.+Russelauthor=A.+Sali&title=Modeling+of+proteins+and+their+assemblies+with+the+integrative+modeling+platform&doi=10.1007%2F978-1-61779-276-2_19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit189R"><div class="casContent"><span class="casTitleNuber">189</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling of proteins and their assemblies with the integrative modeling platform</span></div><div class="casAuthors">Webb, Benjamin; Lasker, Keren; Schneidman-Duhovny, Dina; Tjioe, Elina; Phillips, Jeremy; Kim, Seung Joong; Velazquez-Muriel, Javier; Russel, Daniel; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">781</span>
        (<span class="NLM_cas:issue">Network Biology</span>),
    <span class="NLM_cas:pages">377-397</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">To understand the workings of the living cell, we need to characterize protein assemblies that constitute the cell (for example, the ribosome, 26S proteasome, and the nuclear pore complex).  A reliable high-resoln. structural characterization of these assemblies is frequently beyond the reach of current exptl. methods, such as X-ray crystallog., NMR spectroscopy, electron microscopy, footprinting, chem. crosslinking, FRET spectroscopy, small-angle X-ray scattering, and proteomics.  However, the information garnered from different methods can be combined and used to build computational models of the assembly structures that are consistent with all of the available datasets.  Here, we describe a protocol for this integration, whereby the information is converted to a set of spatial restraints and a variety of optimization procedures can be used to generate models that satisfy the restraints as much as possible.  These generated models can then potentially inform about the precision and accuracy of structure detn., the accuracy of the input datasets, and further data generation.  We also demonstrate the Integrative Modeling Platform (IMP) software, which provides the necessary computational framework to implement this protocol, and several applications for specific-use cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoOF9UyKyz3rVg90H21EOLACvtfcHk0liHQkBjZsCiMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVClsbrM&md5=84ee7542a032775ef6591dcdf7d43ef4</span></div><a href="/servlet/linkout?suffix=cit189&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-276-2_19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-276-2_19%26sid%3Dliteratum%253Aachs%26aulast%3DWebb%26aufirst%3DB.%26aulast%3DLasker%26aufirst%3DK.%26aulast%3DSchneidman-Duhovny%26aufirst%3DD.%26aulast%3DTjioe%26aufirst%3DE.%26aulast%3DPhillips%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.%2BJ.%26aulast%3DVelazquez-Muriel%26aufirst%3DJ.%26aulast%3DRussel%26aufirst%3DD.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DModeling%2520of%2520proteins%2520and%2520their%2520assemblies%2520with%2520the%2520integrative%2520modeling%2520platform%26jtitle%3DMethods%2520Mol.%2520Biol.%2520%2528N.%2520Y.%252C%2520NY%252C%2520U.%2520S.%2529%26date%3D2011%26volume%3D781%26spage%3D377%26epage%3D397%26doi%3D10.1007%2F978-1-61779-276-2_19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref190"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref190'); return false;" data-citation="" class="refNumLink">190</a></strong><div class="NLM_citation" id="cit190"><span><span class="NLM_contrib-group">Evers, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Successful virtual screening for a submicromolar antagonist of the neurokinin-1 receptor based on a ligand-supported homology model</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">5381</span><span class="NLM_x">–</span> <span class="NLM_lpage">5392</span><span class="refDoi"> DOI: 10.1021/jm0311487</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0311487" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5381-5392&author=A.+Eversauthor=G.+Klebe&title=Successful+virtual+screening+for+a+submicromolar+antagonist+of+the+neurokinin-1+receptor+based+on+a+ligand-supported+homology+model&doi=10.1021%2Fjm0311487"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit190&amp;dbid=16384&amp;doi=10.1021%2Fjm0311487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0311487%26sid%3Dliteratum%253Aachs%26aulast%3DEvers%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DSuccessful%2520virtual%2520screening%2520for%2520a%2520submicromolar%2520antagonist%2520of%2520the%2520neurokinin-1%2520receptor%2520based%2520on%2520a%2520ligand-supported%2520homology%2520model%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5381%26epage%3D5392%26doi%3D10.1021%2Fjm0311487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref191"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref191'); return false;" data-citation="" class="refNumLink">191</a></strong><div class="NLM_citation" id="cit191"><span><span class="NLM_contrib-group">Evers, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Ligand-supported homology modeling of g-protein-coupled receptor sites: models sufficient for successful virtual screening</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">248</span><span class="NLM_x">–</span> <span class="NLM_lpage">251</span><span class="refDoi"> DOI: 10.1002/anie.200352776</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref191/cit191&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fanie.200352776" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2004&pages=248-251&author=A.+Eversauthor=G.+Klebe&title=Ligand-supported+homology+modeling+of+g-protein-coupled+receptor+sites%3A+models+sufficient+for+successful+virtual+screening&doi=10.1002%2Fanie.200352776"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit191&amp;dbid=16384&amp;doi=10.1002%2Fanie.200352776&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200352776%26sid%3Dliteratum%253Aachs%26aulast%3DEvers%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DLigand-supported%2520homology%2520modeling%2520of%2520g-protein-coupled%2520receptor%2520sites%253A%2520models%2520sufficient%2520for%2520successful%2520virtual%2520screening%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2004%26volume%3D43%26spage%3D248%26epage%3D251%26doi%3D10.1002%2Fanie.200352776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref192"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref192'); return false;" data-citation="" class="refNumLink">192</a></strong><div class="NLM_citation" id="cit192"><span><span class="NLM_contrib-group">Zhao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vetting, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonanno, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillerich, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seidel, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babbitt, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almo, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweedler, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlt, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cronan, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Discovery of new enzymes and metabolic pathways by using structure and genome context</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">502</span><span class="NLM_x">, </span> <span class="NLM_fpage">698</span><span class="NLM_x">–</span> <span class="NLM_lpage">702</span><span class="refDoi"> DOI: 10.1038/nature12576</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnature12576" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24056934" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFSls73J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=502&publication_year=2013&pages=698-702&author=S.+Zhaoauthor=R.+Kumarauthor=A.+Sakaiauthor=M.+W.+Vettingauthor=B.+M.+Woodauthor=S.+Brownauthor=J.+B.+Bonannoauthor=B.+S.+Hillerichauthor=R.+D.+Seidelauthor=P.+C.+Babbittauthor=S.+C.+Almoauthor=J.+V.+Sweedlerauthor=J.+A.+Gerltauthor=J.+E.+Cronanauthor=M.+P.+Jacobson&title=Discovery+of+new+enzymes+and+metabolic+pathways+by+using+structure+and+genome+context&doi=10.1038%2Fnature12576"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit192R"><div class="casContent"><span class="casTitleNuber">192</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of new enzymes and metabolic pathways by using structure and genome context</span></div><div class="casAuthors">Zhao, Suwen; Kumar, Ritesh; Sakai, Ayano; Vetting, Matthew W.; Wood, B. McKay; Brown, Shoshana; Bonanno, Jeffery B.; Hillerich, Brandan S.; Seidel, Ronald D.; Babbitt, Patricia C.; Almo, Steven C.; Sweedler, Jonathan V.; Gerlt, John A.; Cronan, John E.; Jacobson, Matthew P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">502</span>
        (<span class="NLM_cas:issue">7473</span>),
    <span class="NLM_cas:pages">698-702</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Assigning valid functions to proteins identified in genome projects is challenging: overprediction and database annotation errors are the principal concerns.  We and others are developing computation-guided strategies for functional discovery with metabolite docking' to exptl. derived or homol.-based three-dimensional structures.  Bacterial metabolic pathways often are encoded by genome neighborhoods' (gene clusters and/or operons), which can provide important clues for functional assignment.  We recently demonstrated the synergy of docking and pathway context by predicting' the intermediates in the glycolytic pathway in Escherichia coli.  Metabolite docking to multiple binding proteins and enzymes in the same pathway increases the reliability of in silico predictions of substrate specificities because the pathway intermediates are structurally similar.  Here we report that structure-guided approaches for predicting the substrate specificities of several enzymes encoded by a bacterial gene cluster allowed the correct prediction of the in vitro activity of a structurally characterized enzyme of unknown function (PDB 2PMQ), 2-epimerization of trans-4-hydroxy-L-proline betaine (tHyp-B) and cis-4-hydroxy-D-proline betaine (cHyp-B), and also the correct identification of the catabolic pathway in which Hyp-B 2-epimerase participates.  The substrate-liganded pose predicted by virtual library screening (docking) was confirmed exptl.  The enzymic activities in the predicted pathway were confirmed by in vitro assays and genetic analyses; the intermediates were identified by metabolomics; and repression of the genes encoding the pathway by high salt concns. was established by transcriptomics, confirming the osmolyte role of tHyp-B.  This study establishes the utility of structure-guided functional predictions to enable the discovery of new metabolic pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzzoHQvsXyDbVg90H21EOLACvtfcHk0liHQkBjZsCiMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFSls73J&md5=225ec9508efaab563b2c5d1dd8552c0a</span></div><a href="/servlet/linkout?suffix=cit192&amp;dbid=16384&amp;doi=10.1038%2Fnature12576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12576%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DKumar%26aufirst%3DR.%26aulast%3DSakai%26aufirst%3DA.%26aulast%3DVetting%26aufirst%3DM.%2BW.%26aulast%3DWood%26aufirst%3DB.%2BM.%26aulast%3DBrown%26aufirst%3DS.%26aulast%3DBonanno%26aufirst%3DJ.%2BB.%26aulast%3DHillerich%26aufirst%3DB.%2BS.%26aulast%3DSeidel%26aufirst%3DR.%2BD.%26aulast%3DBabbitt%26aufirst%3DP.%2BC.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DSweedler%26aufirst%3DJ.%2BV.%26aulast%3DGerlt%26aufirst%3DJ.%2BA.%26aulast%3DCronan%26aufirst%3DJ.%2BE.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DDiscovery%2520of%2520new%2520enzymes%2520and%2520metabolic%2520pathways%2520by%2520using%2520structure%2520and%2520genome%2520context%26jtitle%3DNature%26date%3D2013%26volume%3D502%26spage%3D698%26epage%3D702%26doi%3D10.1038%2Fnature12576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref193"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref193'); return false;" data-citation="" class="refNumLink">193</a></strong><div class="NLM_citation" id="cit193"><span><span class="NLM_contrib-group">Kalyanaraman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imker, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, E. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glasner, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babbitt, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almo, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlt, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Discovery of a dipeptide epimerase enzymatic function guided by homology modeling and virtual screening</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1668</span><span class="NLM_x">–</span> <span class="NLM_lpage">1677</span><span class="refDoi"> DOI: 10.1016/j.str.2008.08.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref193/cit193&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.str.2008.08.015" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=1668-1677&author=C.+Kalyanaramanauthor=H.+J.+Imkerauthor=A.+A.+Fedorovauthor=E.+V.+Fedorovauthor=M.+E.+Glasnerauthor=P.+C.+Babbittauthor=S.+C.+Almoauthor=J.+A.+Gerltauthor=M.+P.+Jacobson&title=Discovery+of+a+dipeptide+epimerase+enzymatic+function+guided+by+homology+modeling+and+virtual+screening&doi=10.1016%2Fj.str.2008.08.015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit193&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2008.08.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2008.08.015%26sid%3Dliteratum%253Aachs%26aulast%3DKalyanaraman%26aufirst%3DC.%26aulast%3DImker%26aufirst%3DH.%2BJ.%26aulast%3DFedorov%26aufirst%3DA.%2BA.%26aulast%3DFedorov%26aufirst%3DE.%2BV.%26aulast%3DGlasner%26aufirst%3DM.%2BE.%26aulast%3DBabbitt%26aufirst%3DP.%2BC.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DGerlt%26aufirst%3DJ.%2BA.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DDiscovery%2520of%2520a%2520dipeptide%2520epimerase%2520enzymatic%2520function%2520guided%2520by%2520homology%2520modeling%2520and%2520virtual%2520screening%26jtitle%3DStructure%26date%3D2008%26volume%3D16%26spage%3D1668%26epage%3D1677%26doi%3D10.1016%2Fj.str.2008.08.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref194"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref194'); return false;" data-citation="" class="refNumLink">194</a></strong><div class="NLM_citation" id="cit194"><span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitchcock, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seidel, R. D.,  2nd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillerich, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almo, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raushel, F. M.</span><span> </span><span class="NLM_article-title">Assignment of pterin deaminase activity to an enzyme of unknown function guided by homology modeling and docking</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">795</span><span class="NLM_x">–</span> <span class="NLM_lpage">803</span><span class="refDoi"> DOI: 10.1021/ja309680b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja309680b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref194/cit194&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVyiurjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=795-803&author=H.+Fanauthor=D.+S.+Hitchcockauthor=R.+D.+Seidelauthor=B.+Hillerichauthor=H.+Linauthor=S.+C.+Almoauthor=A.+Saliauthor=B.+K.+Shoichetauthor=F.+M.+Raushel&title=Assignment+of+pterin+deaminase+activity+to+an+enzyme+of+unknown+function+guided+by+homology+modeling+and+docking&doi=10.1021%2Fja309680b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit194R"><div class="casContent"><span class="casTitleNuber">194</span><div class="casTitle"><span class="NLM_cas:atitle">Assignment of Pterin Deaminase Activity to an Enzyme of Unknown Function Guided by Homology Modeling and Docking</span></div><div class="casAuthors">Fan, Hao; Hitchcock, Daniel S.; Seidel, Ronald D.; Hillerich, Brandan; Lin, Henry; Almo, Steven C.; Sali, Andrej; Shoichet, Brian K.; Raushel, Frank M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">795-803</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Of the over 22 million protein sequences in the nonredundant TrEMBL database, fewer than 1% have exptl. confirmed functions.  Structure-based methods have been used to predict enzyme activities from exptl. detd. structures; however, for the vast majority of proteins, no such structures are available.  Here, homol. models of a functionally uncharacterized amidohydrolase from Agrobacterium radiobacter K84 (Arad3529) were computed on the basis of a remote template structure.  The protein backbone of two loops near the active site was remodeled, resulting in four distinct active site conformations.  Substrates of Arad3529 were predicted by docking of 57 672 high-energy intermediate (HEI) forms of 6440 metabolites against these four homol. models.  On the basis of docking ranks and geometries, a set of modified pterins were suggested as candidate substrates for Arad3529.  The predictions were tested by enzymol. expts., and Arad3529 deaminated many pterin metabolites (substrate, kcat/Km [M-1 s-1]): formylpterin, 5.2 × 106; pterin-6-carboxylate, 4.0 × 106; pterin-7-carboxylate, 3.7 × 106; pterin, 3.3 × 106; hydroxymethylpterin, 1.2 × 106; biopterin, 1.0 × 106; d-(+)-neopterin, 3.1 × 105; isoxanthopterin, 2.8 × 105; sepiapterin, 1.3 × 105; folate, 1.3 × 105, xanthopterin, 1.17 × 105; and 7,8-dihydrohydroxymethylpterin, 3.3 × 104.  While pterin is a ubiquitous oxidative product of folate degrdn., genomic anal. suggests that the first step of an undescribed pterin degrdn. pathway is catalyzed by Arad3529.  Homol. model-based virtual screening, esp. with modeling of protein backbone flexibility, may be broadly useful for enzyme function annotation and discovering new pathways and drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkH7Qij21G67Vg90H21EOLACvtfcHk0lh8S_RJFWNusw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVyiurjJ&md5=7e500cb2222fa90af3a94d83527af5e6</span></div><a href="/servlet/linkout?suffix=cit194&amp;dbid=16384&amp;doi=10.1021%2Fja309680b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja309680b%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DH.%26aulast%3DHitchcock%26aufirst%3DD.%2BS.%26aulast%3DSeidel%26aufirst%3DR.%2BD.%26aulast%3DHillerich%26aufirst%3DB.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DRaushel%26aufirst%3DF.%2BM.%26atitle%3DAssignment%2520of%2520pterin%2520deaminase%2520activity%2520to%2520an%2520enzyme%2520of%2520unknown%2520function%2520guided%2520by%2520homology%2520modeling%2520and%2520docking%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D795%26epage%3D803%26doi%3D10.1021%2Fja309680b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref195"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref195'); return false;" data-citation="" class="refNumLink">195</a></strong><div class="NLM_citation" id="cit195"><span><span class="NLM_contrib-group">Jensen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunner, H. R.</span><span> </span><span class="NLM_article-title">Aliskiren: the first renin inhibitor for clinical treatment</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">399</span><span class="NLM_x">–</span> <span class="NLM_lpage">410</span><span class="refDoi"> DOI: 10.1038/nrd2550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref195/cit195&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnrd2550" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref195/cit195&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=18340340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref195/cit195&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlt1agtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=399-410&author=C.+Jensenauthor=P.+Heroldauthor=H.+R.+Brunner&title=Aliskiren%3A+the+first+renin+inhibitor+for+clinical+treatment&doi=10.1038%2Fnrd2550"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit195R"><div class="casContent"><span class="casTitleNuber">195</span><div class="casTitle"><span class="NLM_cas:atitle">Aliskiren: the first renin inhibitor for clinical treatment</span></div><div class="casAuthors">Jensen, Chris; Herold, Peter; Brunner, Hans Rudolf</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">399-410</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The renin-angiotensin system, which is a key regulator of blood pressure, has been successfully targeted by several classes of drugs for the treatment of cardiovascular disease.  However, renin - the protease at the top of the cascade - although highly attractive as a target, has also proved to be challenging.  Here, we describe how these challenges were tackled in the discovery and development of aliskiren, the first renin inhibitor to be approved for clin. use.  The first evidence of the existence of renin was presented over 100 years ago.  However, the importance of renin and the renin-angiotensin system in the pathogenesis of cardiovascular disease was only fully realized in the 1970s.  It was another 20 years before the first inhibitors of renin were available for clin. research.  Here, we describe the discovery and development of aliskiren, an orally active renin inhibitor, which became the first drug in its class to receive regulatory approval.  In 2007, it was approved for the treatment of hypertension by the US Food and Drug Administration and the European Medicines Agency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwW8kGNQazBbVg90H21EOLACvtfcHk0lh8S_RJFWNusw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlt1agtbg%253D&md5=891d7f8266f9b03954ae40e028a8cd42</span></div><a href="/servlet/linkout?suffix=cit195&amp;dbid=16384&amp;doi=10.1038%2Fnrd2550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2550%26sid%3Dliteratum%253Aachs%26aulast%3DJensen%26aufirst%3DC.%26aulast%3DHerold%26aufirst%3DP.%26aulast%3DBrunner%26aufirst%3DH.%2BR.%26atitle%3DAliskiren%253A%2520the%2520first%2520renin%2520inhibitor%2520for%2520clinical%2520treatment%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D399%26epage%3D410%26doi%3D10.1038%2Fnrd2550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref196"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref196'); return false;" data-citation="" class="refNumLink">196</a></strong><div class="NLM_citation" id="cit196"><span><span class="NLM_contrib-group">Barelier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sterling, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Meara, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">The recognition of identical ligands by unrelated proteins</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">2772</span><span class="NLM_x">–</span> <span class="NLM_lpage">2784</span><span class="refDoi"> DOI: 10.1021/acschembio.5b00683</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00683" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref196/cit196&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFyitr7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=2772-2784&author=S.+Barelierauthor=T.+Sterlingauthor=M.+J.+O%E2%80%99Mearaauthor=B.+K.+Shoichet&title=The+recognition+of+identical+ligands+by+unrelated+proteins&doi=10.1021%2Facschembio.5b00683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit196R"><div class="casContent"><span class="casTitleNuber">196</span><div class="casTitle"><span class="NLM_cas:atitle">The recognition of identical ligands by unrelated proteins</span></div><div class="casAuthors">Barelier, Sarah; Sterling, Teague; O'Meara, Matthew J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2772-2784</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding of drugs and reagents to off-targets is well-known.  Whereas many off-targets are related to the primary target by sequence and fold, many ligands bind to unrelated pairs of proteins, and these are harder to anticipate.  If the binding site in the off-target can be related to that of the primary target, this challenge resolves into aligning the 2 pockets.  However, other cases are possible: the ligand might interact with entirely different residues and environments in the off-target, or wholly different ligand atoms may be implicated in the 2 complexes.  To investigate these scenarios at at. resoln., the structures of 59 ligands in 116 complexes (62 pairs in total), where the protein pairs were unrelated by fold but bound an identical ligand, were examd.  In almost half of the pairs, the ligand interacted with unrelated residues in the 2 proteins (29 pairs), and in 14 of the pairs wholly different ligand moieties were implicated in each complex.  Even in those 19 pairs of complexes that presented similar environments to the ligand, ligand superposition rarely resulted in the overlap of related residues.  There appears to be no single pattern-matching "code" for identifying binding sites in unrelated proteins that bind identical ligands, although modeling suggests that there might be a limited no. of different patterns that suffice to recognize different ligand functional groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa1_lQJDNMCrVg90H21EOLACvtfcHk0liSNv0u90iZ_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFyitr7J&md5=9d6ac31c2a58bc85f77e7eb34649c195</span></div><a href="/servlet/linkout?suffix=cit196&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00683%26sid%3Dliteratum%253Aachs%26aulast%3DBarelier%26aufirst%3DS.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DThe%2520recognition%2520of%2520identical%2520ligands%2520by%2520unrelated%2520proteins%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D2772%26epage%3D2784%26doi%3D10.1021%2Facschembio.5b00683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref197"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref197'); return false;" data-citation="" class="refNumLink">197</a></strong><div class="NLM_citation" id="cit197"><span><span class="NLM_contrib-group">Tondi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morandi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Structure-based optimization of a non-beta-lactam lead results in inhibitors that do not up-regulate beta-lactamase expression in cell culture</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">127</span><span class="NLM_x">, </span> <span class="NLM_fpage">4632</span><span class="NLM_x">–</span> <span class="NLM_lpage">4639</span><span class="refDoi"> DOI: 10.1021/ja042984o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja042984o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2005&pages=4632-4639&author=D.+Tondiauthor=F.+Morandiauthor=R.+Bonnetauthor=M.+P.+Costiauthor=B.+K.+Shoichet&title=Structure-based+optimization+of+a+non-beta-lactam+lead+results+in+inhibitors+that+do+not+up-regulate+beta-lactamase+expression+in+cell+culture&doi=10.1021%2Fja042984o"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit197&amp;dbid=16384&amp;doi=10.1021%2Fja042984o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja042984o%26sid%3Dliteratum%253Aachs%26aulast%3DTondi%26aufirst%3DD.%26aulast%3DMorandi%26aufirst%3DF.%26aulast%3DBonnet%26aufirst%3DR.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520optimization%2520of%2520a%2520non-beta-lactam%2520lead%2520results%2520in%2520inhibitors%2520that%2520do%2520not%2520up-regulate%2520beta-lactamase%2520expression%2520in%2520cell%2520culture%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D127%26spage%3D4632%26epage%3D4639%26doi%3D10.1021%2Fja042984o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref198"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref198'); return false;" data-citation="" class="refNumLink">198</a></strong><div class="NLM_citation" id="cit198"><span><span class="NLM_contrib-group">Hermann, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marti-Arbona, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almo, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raushel, F. M.</span><span> </span><span class="NLM_article-title">Structure-based activity prediction for an enzyme of unknown function</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">448</span><span class="NLM_x">, </span> <span class="NLM_fpage">775</span><span class="NLM_x">–</span> <span class="NLM_lpage">779</span><span class="refDoi"> DOI: 10.1038/nature05981</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnature05981" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=17603473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2sXptVGrs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=448&publication_year=2007&pages=775-779&author=J.+C.+Hermannauthor=R.+Marti-Arbonaauthor=A.+A.+Fedorovauthor=E.+Fedorovauthor=S.+C.+Almoauthor=B.+K.+Shoichetauthor=F.+M.+Raushel&title=Structure-based+activity+prediction+for+an+enzyme+of+unknown+function&doi=10.1038%2Fnature05981"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit198R"><div class="casContent"><span class="casTitleNuber">198</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based activity prediction for an enzyme of unknown function</span></div><div class="casAuthors">Hermann, Johannes C.; Marti-Arbona, Ricardo; Fedorov, Alexander A.; Fedorov, Elena; Almo, Steven C.; Shoichet, Brian K.; Raushel, Frank M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">448</span>
        (<span class="NLM_cas:issue">7155</span>),
    <span class="NLM_cas:pages">775-779</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">With many genomes sequenced, a pressing challenge in biol. is predicting the function of the proteins that the genes encode.  When proteins are unrelated to others of known activity, bioinformatics inference for function becomes problematic.  It would thus be useful to interrogate protein structures for function directly.  Here, we predict the function of an enzyme of unknown activity, Tm0936 from Thermotoga maritima, by docking high-energy intermediate forms of thousands of candidate metabolites.  The docking hit list was dominated by adenine analogs, which appeared to undergo C6-deamination.  Four of these, including 5-methylthioadenosine and S-adenosylhomocysteine (SAH), were tested as substrates, and three had substantial catalytic rate consts. (105 M-1 s-1).  The x-ray crystal structure of the complex between Tm0936 and the product resulting from the deamination of SAH, S-inosylhomocysteine, was detd., and it corresponded closely to the predicted structure.  The deaminated products can be further metabolized by T. maritima in a previously uncharacterized SAH degrdn. pathway.  Structure-based docking with high-energy forms of potential substrates may be a useful tool to annotate enzymes for function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomC_L0C9hsf7Vg90H21EOLACvtfcHk0liSNv0u90iZ_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXptVGrs7w%253D&md5=b0bd6612147fb848f49f8b9ccfd8384e</span></div><a href="/servlet/linkout?suffix=cit198&amp;dbid=16384&amp;doi=10.1038%2Fnature05981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05981%26sid%3Dliteratum%253Aachs%26aulast%3DHermann%26aufirst%3DJ.%2BC.%26aulast%3DMarti-Arbona%26aufirst%3DR.%26aulast%3DFedorov%26aufirst%3DA.%2BA.%26aulast%3DFedorov%26aufirst%3DE.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DRaushel%26aufirst%3DF.%2BM.%26atitle%3DStructure-based%2520activity%2520prediction%2520for%2520an%2520enzyme%2520of%2520unknown%2520function%26jtitle%3DNature%26date%3D2007%26volume%3D448%26spage%3D775%26epage%3D779%26doi%3D10.1038%2Fnature05981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref199"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref199'); return false;" data-citation="" class="refNumLink">199</a></strong><div class="NLM_citation" id="cit199"><span><span class="NLM_contrib-group">Sali, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bycroft, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fersht, A. R.</span><span> </span><span class="NLM_article-title">Stabilization of protein structure by interaction of alpha-helix dipole with a charged side chain</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">335</span><span class="NLM_x">, </span> <span class="NLM_fpage">740</span><span class="NLM_x">–</span> <span class="NLM_lpage">743</span><span class="refDoi"> DOI: 10.1038/335740a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref199/cit199&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2F335740a0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=335&publication_year=1988&pages=740-743&author=D.+Saliauthor=M.+Bycroftauthor=A.+R.+Fersht&title=Stabilization+of+protein+structure+by+interaction+of+alpha-helix+dipole+with+a+charged+side+chain&doi=10.1038%2F335740a0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit199&amp;dbid=16384&amp;doi=10.1038%2F335740a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F335740a0%26sid%3Dliteratum%253Aachs%26aulast%3DSali%26aufirst%3DD.%26aulast%3DBycroft%26aufirst%3DM.%26aulast%3DFersht%26aufirst%3DA.%2BR.%26atitle%3DStabilization%2520of%2520protein%2520structure%2520by%2520interaction%2520of%2520alpha-helix%2520dipole%2520with%2520a%2520charged%2520side%2520chain%26jtitle%3DNature%26date%3D1988%26volume%3D335%26spage%3D740%26epage%3D743%26doi%3D10.1038%2F335740a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref200"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref200'); return false;" data-citation="" class="refNumLink">200</a></strong><div class="NLM_citation" id="cit200"><span><span class="NLM_contrib-group">Eswar, N.; Sali, A.</span><span> </span><span class="NLM_article-title">Comparative modeling of drug target proteins</span>. In  <span class="citation_source-book">Comprehensive Medicinal Chemistry II</span>; <span class="NLM_contrib-group">Taylor, J. B.; Triggle, D. J.</span>, Eds.; <span class="NLM_publisher-name">Elsevier</span>,<span class="NLM_x"> </span><span class="NLM_year">2007</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">4</span>, pp  <span class="NLM_fpage">216</span><span class="NLM_x">–</span> <span class="NLM_lpage">231</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref200/cit200&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2FB0-08-045044-X%2F00251-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&pages=216-231&author=N.+Eswar&author=A.+Saliauthor=J.+B.+Taylor&author=D.+J.+Triggle&title=Comprehensive+Medicinal+Chemistry+II"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit200&amp;dbid=16384&amp;doi=10.1016%2FB0-08-045044-X%2F00251-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB0-08-045044-X%252F00251-0%26sid%3Dliteratum%253Aachs%26aulast%3DEswar%26aufirst%3DN.%26atitle%3DComparative%2520modeling%2520of%2520drug%2520target%2520proteins%26btitle%3DComprehensive%2520Medicinal%2520Chemistry%2520II%26aulast%3DTaylor%26aufirst%3DJ.%2BB.%26pub%3DElsevier%26date%3D2007%26volume%3D4%26spage%3D216%26epage%3D231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref201"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref201'); return false;" data-citation="" class="refNumLink">201</a></strong><div class="NLM_citation" id="cit201"><span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, N.</span><span> </span><span class="NLM_article-title">Structure-based discovery of novel and selective 5-hydroxytryptamine 2B receptor antagonists for the treatment of irritable bowel syndrome</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">707</span><span class="NLM_x">–</span> <span class="NLM_lpage">720</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01631</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01631" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref201/cit201&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVygsb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=707-720&author=Y.+Zhouauthor=J.+Maauthor=X.+Linauthor=X.+P.+Huangauthor=K.+Wuauthor=N.+Huang&title=Structure-based+discovery+of+novel+and+selective+5-hydroxytryptamine+2B+receptor+antagonists+for+the+treatment+of+irritable+bowel+syndrome&doi=10.1021%2Facs.jmedchem.5b01631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit201R"><div class="casContent"><span class="casTitleNuber">201</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of Novel and Selective 5-Hydroxytryptamine 2B Receptor Antagonists for the Treatment of Irritable Bowel Syndrome</span></div><div class="casAuthors">Zhou, Yu; Ma, Jing; Lin, Xingyu; Huang, Xi-Ping; Wu, Kaichun; Huang, Niu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">707-720</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here the authors employed structure-based ligand discovery techniques to explore a recently detd. crystal structure of the 5-hydroxytryptamine 2B (5-HT2B) receptor.  Ten compds. contg. a novel chem. scaffold were identified; among them, seven mols. were active in cellular function assays with the most potent one exhibiting an IC50 value of 27.3 nM.  The authors then systematically probed the binding characteristics of this scaffold by designing, synthesizing, and testing a series of structural modifications.  The structure-activity relationship studies strongly support the authors' predicted binding model.  The binding profiling across a panel of 11 5-HT receptors indicated that these compds. are highly selective for the 5-HT2B receptor.  Oral administration of N4-(3-bromophenyl)-1,3,5-triazaspiro[5.5]undeca-1,3-diene-2,4-diamine (compd. 15) (30 mg/kg) produced significant attenuation of visceral hypersensitivity in a rat model of irritable bowel syndrome (IBS).  The authors expect this novel scaffold will serve as the foundation for the development of 5-HT2B antagonists for the treatment of IBS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTmaQmcj_24LVg90H21EOLACvtfcHk0ljgI_w8hlWalQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVygsb%252FE&md5=5ca671401f4f8664fe55153121f2bb55</span></div><a href="/servlet/linkout?suffix=cit201&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01631%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DWu%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DN.%26atitle%3DStructure-based%2520discovery%2520of%2520novel%2520and%2520selective%25205-hydroxytryptamine%25202B%2520receptor%2520antagonists%2520for%2520the%2520treatment%2520of%2520irritable%2520bowel%2520syndrome%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D707%26epage%3D720%26doi%3D10.1021%2Facs.jmedchem.5b01631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref202"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref202'); return false;" data-citation="" class="refNumLink">202</a></strong><div class="NLM_citation" id="cit202"><span><span class="NLM_contrib-group">Yan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wempe, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khanna, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meier, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffman, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owens, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wysoczynski, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nitz, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knabe, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmed, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brautigan, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paschal, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meroueh, S. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theodorescu, D.</span><span> </span><span class="NLM_article-title">Discovery and characterization of small molecules that target the GTPase Ral</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">515</span><span class="NLM_x">, </span> <span class="NLM_fpage">443</span><span class="NLM_x">–</span> <span class="NLM_lpage">447</span><span class="refDoi"> DOI: 10.1038/nature13713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref202/cit202&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnature13713" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref202/cit202&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25219851" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref202/cit202&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVemtrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=515&publication_year=2014&pages=443-447&author=C.+Yanauthor=D.+Liuauthor=L.+Liauthor=M.+F.+Wempeauthor=S.+Guinauthor=M.+Khannaauthor=J.+Meierauthor=B.+Hoffmanauthor=C.+Owensauthor=C.+L.+Wysoczynskiauthor=M.+D.+Nitzauthor=W.+E.+Knabeauthor=M.+Ahmedauthor=D.+L.+Brautiganauthor=B.+M.+Paschalauthor=M.+A.+Schwartzauthor=D.+N.+Jonesauthor=D.+Rossauthor=S.+O.+Merouehauthor=D.+Theodorescu&title=Discovery+and+characterization+of+small+molecules+that+target+the+GTPase+Ral&doi=10.1038%2Fnature13713"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit202R"><div class="casContent"><span class="casTitleNuber">202</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of small molecules that target the GTPase Ral</span></div><div class="casAuthors">Yan, Chao; Liu, Degang; Li, Liwei; Wempe, Michael F.; Guin, Sunny; Khanna, May; Meier, Jeremy; Hoffman, Brenton; Owens, Charles; Wysoczynski, Christina L.; Nitz, Matthew D.; Knabe, Eric W.; Brautigan, David L.; Paschal, Bryce M.; Schwartz, Martin A.; Jones, David N. M.; Ross, David; Meroueh, Samy O.; Theodorescu, Dan</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">515</span>
        (<span class="NLM_cas:issue">7527</span>),
    <span class="NLM_cas:pages">443-447</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The Ras-like GTPases RalA and RalB are important drivers of tumor growth and metastasis.  Chems. that block Ral function would be valuable as research tools and for cancer therapeutics.  Here we used protein structure anal. and virtual screening to identify drug-like mols. that bind to a site on the GDP-bound form of Ral.  The compds. RBC6, RBC8 and RBC10 inhibited the binding of Ral to its effector RALBP1, as well as inhibiting Ral-mediated cell spreading of murine embryonic fibroblasts and anchorage-independent growth of human cancer cell lines.  The binding of the RBC8 deriv. BQU57 to RalB was confirmed by isothermal titrn. calorimetry, surface plasmon resonance and 1H-15N transverse relaxation-optimized spectroscopy (TROSY) NMR spectroscopy.  RBC8 and BQU57 show selectivity for Ral relative to the GTPases Ras and RhoA and inhibit tumor xenograft growth to a similar extent to the depletion of Ral using RNA interference.  Our results show the utility of structure-based discovery for the development of therapeutics for Ral-dependent cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFuvI2j20IarVg90H21EOLACvtfcHk0ljgI_w8hlWalQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVemtrjN&md5=9d2f67e08379fffa98fe88e2c49597fd</span></div><a href="/servlet/linkout?suffix=cit202&amp;dbid=16384&amp;doi=10.1038%2Fnature13713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13713%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DWempe%26aufirst%3DM.%2BF.%26aulast%3DGuin%26aufirst%3DS.%26aulast%3DKhanna%26aufirst%3DM.%26aulast%3DMeier%26aufirst%3DJ.%26aulast%3DHoffman%26aufirst%3DB.%26aulast%3DOwens%26aufirst%3DC.%26aulast%3DWysoczynski%26aufirst%3DC.%2BL.%26aulast%3DNitz%26aufirst%3DM.%2BD.%26aulast%3DKnabe%26aufirst%3DW.%2BE.%26aulast%3DAhmed%26aufirst%3DM.%26aulast%3DBrautigan%26aufirst%3DD.%2BL.%26aulast%3DPaschal%26aufirst%3DB.%2BM.%26aulast%3DSchwartz%26aufirst%3DM.%2BA.%26aulast%3DJones%26aufirst%3DD.%2BN.%26aulast%3DRoss%26aufirst%3DD.%26aulast%3DMeroueh%26aufirst%3DS.%2BO.%26aulast%3DTheodorescu%26aufirst%3DD.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520small%2520molecules%2520that%2520target%2520the%2520GTPase%2520Ral%26jtitle%3DNature%26date%3D2014%26volume%3D515%26spage%3D443%26epage%3D447%26doi%3D10.1038%2Fnature13713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref203"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref203'); return false;" data-citation="" class="refNumLink">203</a></strong><div class="NLM_citation" id="cit203"><span><span class="NLM_contrib-group">Gruneberg, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wendt, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Subnanomolar inhibitors from computer screening: a model study using human carbonic anhydrase II</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">389</span><span class="NLM_x">–</span> <span class="NLM_lpage">393</span><span class="refDoi"> DOI: 10.1002/1521-3773(20010119)40:2<389::AID-ANIE389>3.0.CO;2-#</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref203/cit203&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2F1521-3773%2820010119%2940%3A2%3C389%3A%3AAID-ANIE389%3E3.0.CO%3B2-%23" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=389-393&author=S.+Grunebergauthor=B.+Wendtauthor=G.+Klebe&title=Subnanomolar+inhibitors+from+computer+screening%3A+a+model+study+using+human+carbonic+anhydrase+II&doi=10.1002%2F1521-3773%2820010119%2940%3A2%3C389%3A%3AAID-ANIE389%3E3.0.CO%3B2-%23"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit203&amp;dbid=16384&amp;doi=10.1002%2F1521-3773%2820010119%2940%3A2%3C389%3A%3AAID-ANIE389%3E3.0.CO%3B2-%23&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1521-3773%252820010119%252940%253A2%253C389%253A%253AAID-ANIE389%253E3.0.CO%253B2-%2523%26sid%3Dliteratum%253Aachs%26aulast%3DGruneberg%26aufirst%3DS.%26aulast%3DWendt%26aufirst%3DB.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DSubnanomolar%2520inhibitors%2520from%2520computer%2520screening%253A%2520a%2520model%2520study%2520using%2520human%2520carbonic%2520anhydrase%2520II%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2001%26volume%3D40%26spage%3D389%26epage%3D393%26doi%3D10.1002%2F1521-3773%2820010119%2940%3A2%3C389%3A%3AAID-ANIE389%3E3.0.CO%3B2-%23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref204"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref204'); return false;" data-citation="" class="refNumLink">204</a></strong><div class="NLM_citation" id="cit204"><span><span class="NLM_contrib-group">de Graaf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kooistra, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vischer, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katritch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuijer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiroishi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwata, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimamura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span> </span><span class="NLM_article-title">Crystal structure-based virtual screening for fragment-like ligands of the human histamine H(1) receptor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">8195</span><span class="NLM_x">–</span> <span class="NLM_lpage">8206</span><span class="refDoi"> DOI: 10.1021/jm2011589</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2011589" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref204/cit204&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlaltbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8195-8206&author=C.+de+Graafauthor=A.+J.+Kooistraauthor=H.+F.+Vischerauthor=V.+Katritchauthor=M.+Kuijerauthor=M.+Shiroishiauthor=S.+Iwataauthor=T.+Shimamuraauthor=R.+C.+Stevensauthor=I.+J.+de+Eschauthor=R.+Leurs&title=Crystal+structure-based+virtual+screening+for+fragment-like+ligands+of+the+human+histamine+H%281%29+receptor&doi=10.1021%2Fjm2011589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit204R"><div class="casContent"><span class="casTitleNuber">204</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure-Based Virtual Screening for Fragment-like Ligands of the Human Histamine H1 Receptor</span></div><div class="casAuthors">de Graaf, Chris; Kooistra, Albert J.; Vischer, Henry F.; Katritch, Vsevolod; Kuijer, Martien; Shiroishi, Mitsunori; Iwata, So; Shimamura, Tatsuro; Stevens, Raymond C.; de Esch, Iwan J. P.; Leurs, Rob</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8195-8206</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent crystal structure detns. of druggable class A G protein-coupled receptors (GPCRs) have opened up excellent opportunities in structure-based ligand discovery for this pharmaceutically important protein family.  We have developed and validated a customized structure-based virtual fragment screening protocol against the recently detd. human histamine H1 receptor (H1R) crystal structure.  The method combines mol. docking simulations with a protein-ligand interaction fingerprint (IFP) scoring method.  The optimized in silico screening approach was successfully applied to identify a chem. diverse set of novel fragment-like (≤22 heavy atoms) H1R ligands with an exceptionally high hit rate of 73%.  Of the 26 tested fragments, 19 compds. had affinities ranging from 10 μM to 6 nM.  The current study shows the potential of in silico screening against GPCR crystal structures to explore novel, fragment-like GPCR ligand space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbwo3rozOuPbVg90H21EOLACvtfcHk0ljk9BBpLFiLlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlaltbnL&md5=e45d22d293667ff443f6d62ecef90f9d</span></div><a href="/servlet/linkout?suffix=cit204&amp;dbid=16384&amp;doi=10.1021%2Fjm2011589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2011589%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DVischer%26aufirst%3DH.%2BF.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DKuijer%26aufirst%3DM.%26aulast%3DShiroishi%26aufirst%3DM.%26aulast%3DIwata%26aufirst%3DS.%26aulast%3DShimamura%26aufirst%3DT.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26atitle%3DCrystal%2520structure-based%2520virtual%2520screening%2520for%2520fragment-like%2520ligands%2520of%2520the%2520human%2520histamine%2520H%25281%2529%2520receptor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8195%26epage%3D8206%26doi%3D10.1021%2Fjm2011589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref205"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref205'); return false;" data-citation="" class="refNumLink">205</a></strong><div class="NLM_citation" id="cit205"><span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenbaum, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Structure-based discovery of beta2-adrenergic receptor ligands</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">6843</span><span class="NLM_x">–</span> <span class="NLM_lpage">6848</span><span class="refDoi"> DOI: 10.1073/pnas.0812657106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref205/cit205&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1073%2Fpnas.0812657106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref205/cit205&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=19342484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref205/cit205&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsV2qsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=6843-6848&author=P.+Kolbauthor=D.+M.+Rosenbaumauthor=J.+J.+Irwinauthor=J.+J.+Fungauthor=B.+K.+Kobilkaauthor=B.+K.+Shoichet&title=Structure-based+discovery+of+beta2-adrenergic+receptor+ligands&doi=10.1073%2Fpnas.0812657106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit205R"><div class="casContent"><span class="casTitleNuber">205</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of β2-adrenergic receptor ligands</span></div><div class="casAuthors">Kolb, Peter; Rosenbaum, Daniel M.; Irwin, John J.; Fung, Juan Jose; Kobilka, Brian K.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6843-6848</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Aminergic G protein-coupled receptors (GPCRs) have been a major focus of pharmaceutical research for many years.  Due partly to the lack of reliable receptor structures, drug discovery efforts have been largely ligand-based.  The recently detd. X-ray structure of the β2-adrenergic receptor offers an opportunity to investigate the advantages and limitations inherent in a structure-based approach to ligand discovery against this and related GPCR targets.  Approx. 1 million com. available, "lead-like" mols. were docked against the β2-adrenergic receptor structure.  On testing of 25 high-ranking mols., 6 were active with binding affinities <4 μM, with the best mol. binding with a Ki of 9 nM (95% confidence interval 7-10 nM).  Five of these mols. were inverse agonists.  The high hit rate, the high affinity of the most potent mol., the discovery of unprecedented chemotypes among the new inhibitors, and the apparent bias toward inverse agonists among the docking hits, have implications for structure-based approaches against GPCRs that recognize small org. mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo89kM84Li_H7Vg90H21EOLACvtfcHk0ljk9BBpLFiLlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsV2qsro%253D&md5=4d1a4cb2aa3925aa4c99c6b0496417a7</span></div><a href="/servlet/linkout?suffix=cit205&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0812657106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0812657106%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFung%26aufirst%3DJ.%2BJ.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520discovery%2520of%2520beta2-adrenergic%2520receptor%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D6843%26epage%3D6848%26doi%3D10.1073%2Fpnas.0812657106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref206"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref206'); return false;" data-citation="" class="refNumLink">206</a></strong><div class="NLM_citation" id="cit206"><span><span class="NLM_contrib-group">Rodriguez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, Z. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moss, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlsson, J.</span><span> </span><span class="NLM_article-title">Molecular docking screening using agonist-bound GPCR structures: probing the A2A adenosine receptor</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">550</span><span class="NLM_x">–</span> <span class="NLM_lpage">563</span><span class="refDoi"> DOI: 10.1021/ci500639g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500639g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref206/cit206&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVWgtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=550-563&author=D.+Rodriguezauthor=Z.+G.+Gaoauthor=S.+M.+Mossauthor=K.+A.+Jacobsonauthor=J.+Carlsson&title=Molecular+docking+screening+using+agonist-bound+GPCR+structures%3A+probing+the+A2A+adenosine+receptor&doi=10.1021%2Fci500639g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit206R"><div class="casContent"><span class="casTitleNuber">206</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Docking Screening Using Agonist-Bound GPCR Structures: Probing the A2A Adenosine Receptor</span></div><div class="casAuthors">Rodriguez, David; Gao, Zhang-Guo; Moss, Steven M.; Jacobson, Kenneth A.; Carlsson, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">550-563</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Crystal structures of G protein-coupled receptors (GPCRs) have recently revealed the mol. basis of ligand binding and activation, which has provided exciting opportunities for structure-based drug design.  The A2A adenosine receptor (A2AAR) is a promising therapeutic target for cardiovascular diseases, but progress in this area is limited by the lack of novel agonist scaffolds.  The authors carried out docking screens of 6.7 million com. available mols. against active-like conformations of the A2AAR to investigate whether these structures could guide the discovery of agonists.  Nine out of the 20 predicted agonists were confirmed to be A2AAR ligands, but none of these activated the ARs.  The difficulties in discovering AR agonists using structure-based methods originated from limited at.-level understanding of the activation mechanism and a chem. bias toward antagonists in the screened library.  In particular, the compn. of the screened library was found to strongly reduce the likelihood of identifying AR agonists, which reflected the high ligand complexity required for receptor activation.  Extension of this anal. to other pharmaceutically relevant GPCRs suggested that library screening may not be suitable for targets requiring a complex receptor-ligand interaction network.  The authors' results provide specific directions for the future development of novel A2AAR agonists and general strategies for structure-based drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp89cADSkVJDLVg90H21EOLACvtfcHk0lj9_0zj82BkMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVWgtbs%253D&md5=0faec5c3d810a9ef32e5f374b6f9b447</span></div><a href="/servlet/linkout?suffix=cit206&amp;dbid=16384&amp;doi=10.1021%2Fci500639g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500639g%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez%26aufirst%3DD.%26aulast%3DGao%26aufirst%3DZ.%2BG.%26aulast%3DMoss%26aufirst%3DS.%2BM.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DCarlsson%26aufirst%3DJ.%26atitle%3DMolecular%2520docking%2520screening%2520using%2520agonist-bound%2520GPCR%2520structures%253A%2520probing%2520the%2520A2A%2520adenosine%2520receptor%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D550%26epage%3D563%26doi%3D10.1021%2Fci500639g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref207"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref207'); return false;" data-citation="" class="refNumLink">207</a></strong><div class="NLM_citation" id="cit207"><span><span class="NLM_contrib-group">Colas, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grewer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otte, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gameiro, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albers, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shere, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonomi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holst, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, A.</span><span> </span><span class="NLM_article-title">Ligand discovery for the alanine-serine-cysteine transporter (ASCT2, SLC1A5) from homology modeling and virtual screening</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1004477</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1004477</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref207/cit207&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pcbi.1004477" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=e1004477&author=C.+Colasauthor=C.+Grewerauthor=N.+J.+Otteauthor=A.+Gameiroauthor=T.+Albersauthor=K.+Singhauthor=H.+Shereauthor=M.+Bonomiauthor=J.+Holstauthor=A.+Schlessinger&title=Ligand+discovery+for+the+alanine-serine-cysteine+transporter+%28ASCT2%2C+SLC1A5%29+from+homology+modeling+and+virtual+screening&doi=10.1371%2Fjournal.pcbi.1004477"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit207&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1004477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1004477%26sid%3Dliteratum%253Aachs%26aulast%3DColas%26aufirst%3DC.%26aulast%3DGrewer%26aufirst%3DC.%26aulast%3DOtte%26aufirst%3DN.%2BJ.%26aulast%3DGameiro%26aufirst%3DA.%26aulast%3DAlbers%26aufirst%3DT.%26aulast%3DSingh%26aufirst%3DK.%26aulast%3DShere%26aufirst%3DH.%26aulast%3DBonomi%26aufirst%3DM.%26aulast%3DHolst%26aufirst%3DJ.%26aulast%3DSchlessinger%26aufirst%3DA.%26atitle%3DLigand%2520discovery%2520for%2520the%2520alanine-serine-cysteine%2520transporter%2520%2528ASCT2%252C%2520SLC1A5%2529%2520from%2520homology%2520modeling%2520and%2520virtual%2520screening%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3De1004477%26doi%3D10.1371%2Fjournal.pcbi.1004477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref208"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref208'); return false;" data-citation="" class="refNumLink">208</a></strong><div class="NLM_citation" id="cit208"><span><span class="NLM_contrib-group">Katritch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaakola, V. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ijzerman, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeager, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kufareva, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Structure-based discovery of novel chemotypes for adenosine A(2A) receptor antagonists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">1799</span><span class="NLM_x">–</span> <span class="NLM_lpage">1809</span><span class="refDoi"> DOI: 10.1021/jm901647p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901647p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=1799-1809&author=V.+Katritchauthor=V.+P.+Jaakolaauthor=J.+R.+Laneauthor=J.+Linauthor=A.+P.+Ijzermanauthor=M.+Yeagerauthor=I.+Kufarevaauthor=R.+C.+Stevensauthor=R.+Abagyan&title=Structure-based+discovery+of+novel+chemotypes+for+adenosine+A%282A%29+receptor+antagonists&doi=10.1021%2Fjm901647p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit208&amp;dbid=16384&amp;doi=10.1021%2Fjm901647p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901647p%26sid%3Dliteratum%253Aachs%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DJaakola%26aufirst%3DV.%2BP.%26aulast%3DLane%26aufirst%3DJ.%2BR.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26aulast%3DYeager%26aufirst%3DM.%26aulast%3DKufareva%26aufirst%3DI.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DStructure-based%2520discovery%2520of%2520novel%2520chemotypes%2520for%2520adenosine%2520A%25282A%2529%2520receptor%2520antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D1799%26epage%3D1809%26doi%3D10.1021%2Fjm901647p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref209"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref209'); return false;" data-citation="" class="refNumLink">209</a></strong><div class="NLM_citation" id="cit209"><span><span class="NLM_contrib-group">Carlsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, Z. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, K. A.</span><span> </span><span class="NLM_article-title">Structure-based discovery of A2A adenosine receptor ligands</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">3748</span><span class="NLM_x">–</span> <span class="NLM_lpage">3755</span><span class="refDoi"> DOI: 10.1021/jm100240h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100240h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref209/cit209&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvFaqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=3748-3755&author=J.+Carlssonauthor=L.+Yooauthor=Z.+G.+Gaoauthor=J.+J.+Irwinauthor=B.+K.+Shoichetauthor=K.+A.+Jacobson&title=Structure-based+discovery+of+A2A+adenosine+receptor+ligands&doi=10.1021%2Fjm100240h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit209R"><div class="casContent"><span class="casTitleNuber">209</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of A2A Adenosine Receptor Ligands</span></div><div class="casAuthors">Carlsson, Jens; Yoo, Lena; Gao, Zhan-Guo; Irwin, John J.; Shoichet, Brian K.; Jacobson, Kenneth A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3748-3755</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent detn. of X-ray structures of pharmacol. relevant GPCRs has made these targets accessible to structure-based ligand discovery.  Here we explore whether novel chemotypes may be discovered for the A2A adenosine receptor, based on complementarity to its recently detd. structure.  The A2A adenosine receptor signals in the periphery and the CNS, with agonists explored as anti-inflammatory drugs and antagonists explored for neurodegenerative diseases.  We used mol. docking to screen a 1.4 million compd. database against the X-ray structure computationally and tested 20 high-ranking, previously unknown mols. exptl.  Of these 35% showed substantial activity with affinities between 200 nM and 9 μM.  For the most potent of these new inhibitors, over 50-fold specificity was obsd. for the A2A vs. the related A1 and A3 subtypes.  These high hit rates and affinities at least partly reflect the bias of com. libraries toward GPCR-like chemotypes, an issue that we attempt to investigate quant.  Despite this bias, many of the most potent new ligands were novel, dissimilar from known ligands, providing new lead structures for modulation of this medically important target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzvS6-9T9bd7Vg90H21EOLACvtfcHk0lj9_0zj82BkMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvFaqsL8%253D&md5=c36c941d52d2cec06387d79c4c423d46</span></div><a href="/servlet/linkout?suffix=cit209&amp;dbid=16384&amp;doi=10.1021%2Fjm100240h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100240h%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DYoo%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DZ.%2BG.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DStructure-based%2520discovery%2520of%2520A2A%2520adenosine%2520receptor%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D3748%26epage%3D3755%26doi%3D10.1021%2Fjm100240h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref210"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref210'); return false;" data-citation="" class="refNumLink">210</a></strong><div class="NLM_citation" id="cit210"><span><span class="NLM_contrib-group">Park, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chien, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryu, S. E.</span><span> </span><span class="NLM_article-title">Discovery of potent inhibitors of receptor protein tyrosine phosphatase sigma through the structure-based virtual screening</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">6333</span><span class="NLM_x">–</span> <span class="NLM_lpage">6337</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2012.08.081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref210/cit210&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2012.08.081" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=6333-6337&author=H.+Parkauthor=P.+N.+Chienauthor=S.+E.+Ryu&title=Discovery+of+potent+inhibitors+of+receptor+protein+tyrosine+phosphatase+sigma+through+the+structure-based+virtual+screening&doi=10.1016%2Fj.bmcl.2012.08.081"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit210&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.08.081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.08.081%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DH.%26aulast%3DChien%26aufirst%3DP.%2BN.%26aulast%3DRyu%26aufirst%3DS.%2BE.%26atitle%3DDiscovery%2520of%2520potent%2520inhibitors%2520of%2520receptor%2520protein%2520tyrosine%2520phosphatase%2520sigma%2520through%2520the%2520structure-based%2520virtual%2520screening%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D6333%26epage%3D6337%26doi%3D10.1016%2Fj.bmcl.2012.08.081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref211"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref211'); return false;" data-citation="" class="refNumLink">211</a></strong><div class="NLM_citation" id="cit211"><span><span class="NLM_contrib-group">Zarzycka, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seijkens, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nabuurs, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritschel, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grommes, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soehnlein, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schrijver, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Tiel, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hackeng, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giehler, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kieser, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lutgens, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vriend, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicolaes, G. A.</span><span> </span><span class="NLM_article-title">Discovery of small molecule CD40-TRAF6 inhibitors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">294</span><span class="NLM_x">–</span> <span class="NLM_lpage">307</span><span class="refDoi"> DOI: 10.1021/ci500631e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500631e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref211/cit211&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2gsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=294-307&author=B.+Zarzyckaauthor=T.+Seijkensauthor=S.+B.+Nabuursauthor=T.+Ritschelauthor=J.+Grommesauthor=O.+Soehnleinauthor=R.+Schrijverauthor=C.+M.+van+Tielauthor=T.+M.+Hackengauthor=C.+Weberauthor=F.+Giehlerauthor=A.+Kieserauthor=E.+Lutgensauthor=G.+Vriendauthor=G.+A.+Nicolaes&title=Discovery+of+small+molecule+CD40-TRAF6+inhibitors&doi=10.1021%2Fci500631e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit211R"><div class="casContent"><span class="casTitleNuber">211</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Molecule CD40-TRAF6 Inhibitors</span></div><div class="casAuthors">Zarzycka, Barbara; Seijkens, Tom; Nabuurs, Sander B.; Ritschel, Tina; Grommes, Jochen; Soehnlein, Oliver; Schrijver, Roy; van Tiel, Claudia M.; Hackeng, Tilman M.; Weber, Christian; Giehler, Fabian; Kieser, Arnd; Lutgens, Esther; Vriend, Gert; Nicolaes, Gerry A. F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">294-307</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The CD154-CD40 receptor complex plays a pivotal role in several inflammatory pathways.  Attempts to inhibit the formation of this complex have resulted in systemic side effects.  Downstream inhibition of the CD40 signaling pathway therefore seems a better way to ameliorate inflammatory disease.  To relay a signal, the CD40 receptor recruits adapter proteins called tumor necrosis factor receptor-assocd. factors (TRAFs).  CD40-TRAF6 interactions are known to play an essential role in several inflammatory diseases.  We used in silico, in vitro, and in vivo expts. to identify and characterize compds. that block CD40-TRAF6 interactions.  We present in detail our drug docking and optimization pipeline and show how we used it to find lead compds. that reduce inflammation in models of peritonitis and sepsis.  These compds. appear to be good leads for drug development, given the obsd. absence of side effects and their demonstrated efficacy for peritonitis and sepsis in mouse models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMWtO3zA6Qk7Vg90H21EOLACvtfcHk0liBEvqIROgAEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2gsLk%253D&md5=2879de87f371f19a9097461d76e28038</span></div><a href="/servlet/linkout?suffix=cit211&amp;dbid=16384&amp;doi=10.1021%2Fci500631e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500631e%26sid%3Dliteratum%253Aachs%26aulast%3DZarzycka%26aufirst%3DB.%26aulast%3DSeijkens%26aufirst%3DT.%26aulast%3DNabuurs%26aufirst%3DS.%2BB.%26aulast%3DRitschel%26aufirst%3DT.%26aulast%3DGrommes%26aufirst%3DJ.%26aulast%3DSoehnlein%26aufirst%3DO.%26aulast%3DSchrijver%26aufirst%3DR.%26aulast%3Dvan%2BTiel%26aufirst%3DC.%2BM.%26aulast%3DHackeng%26aufirst%3DT.%2BM.%26aulast%3DWeber%26aufirst%3DC.%26aulast%3DGiehler%26aufirst%3DF.%26aulast%3DKieser%26aufirst%3DA.%26aulast%3DLutgens%26aufirst%3DE.%26aulast%3DVriend%26aufirst%3DG.%26aulast%3DNicolaes%26aufirst%3DG.%2BA.%26atitle%3DDiscovery%2520of%2520small%2520molecule%2520CD40-TRAF6%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D294%26epage%3D307%26doi%3D10.1021%2Fci500631e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref212"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref212'); return false;" data-citation="" class="refNumLink">212</a></strong><div class="NLM_citation" id="cit212"><span><span class="NLM_contrib-group">Fazi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tintori, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botta, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Selvaraj, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garbelli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maga, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botta, M.</span><span> </span><span class="NLM_article-title">Homology Model-Based Virtual Screening for the Identification of Human Helicase DDX3 Inhibitors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">2443</span><span class="NLM_x">–</span> <span class="NLM_lpage">2454</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00419</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00419" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref212/cit212&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVent7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2443-2454&author=R.+Faziauthor=C.+Tintoriauthor=A.+Braiauthor=L.+Bottaauthor=M.+Selvarajauthor=A.+Garbelliauthor=G.+Magaauthor=M.+Botta&title=Homology+Model-Based+Virtual+Screening+for+the+Identification+of+Human+Helicase+DDX3+Inhibitors&doi=10.1021%2Facs.jcim.5b00419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit212R"><div class="casContent"><span class="casTitleNuber">212</span><div class="casTitle"><span class="NLM_cas:atitle">Homology Model-Based Virtual Screening for the Identification of Human Helicase DDX3 Inhibitors</span></div><div class="casAuthors">Fazi, Roberta; Tintori, Cristina; Brai, Annalaura; Botta, Lorenzo; Selvaraj, Manikandan; Garbelli, Anna; Maga, Giovanni; Botta, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2443-2454</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting cellular cofactors instead of viral enzymes represents a new strategy to combat infectious diseases, which should help to overcome the problem of viral resistance.  Recently, it has been revealed that the cellular ATPase/RNA helicase X-linked DEAD-box polypeptide 3 (DDX3) is an essential host factor for the replication of several viruses such as HIV, HCV, JEV, Dengue, and West Nile.  Accordingly, a drug targeting DDX3 could theor. inhibit all viruses that are dependent on this host factor.  Herein, for the first time, a model of hDDX3 in its closed conformation, which binds the viral RNA was developed by using the homol. module of Prime through the Maestro interface of Schrodinger.  Next, a structure-based virtual screening protocol was applied to identify DDX3 small mol. inhibitors targeting the RNA binding pocket.  As a result, an impressive hit rate of 40% was obtained with the identification of 10 active compds. out of the 25 tested small mols.  The best poses of the active ligands highlighted the crucial residues to be targeted for the inhibition of the helicase activity of DDX3.  The obtained results confirm the reliability of the constructed DDX3/RNA model and the proposed computational strategy for investigating novel DDX3 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO1Sy7ZFr2kbVg90H21EOLACvtfcHk0liBEvqIROgAEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVent7fN&md5=0c739457f73969394c6a31c113617bb6</span></div><a href="/servlet/linkout?suffix=cit212&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00419%26sid%3Dliteratum%253Aachs%26aulast%3DFazi%26aufirst%3DR.%26aulast%3DTintori%26aufirst%3DC.%26aulast%3DBrai%26aufirst%3DA.%26aulast%3DBotta%26aufirst%3DL.%26aulast%3DSelvaraj%26aufirst%3DM.%26aulast%3DGarbelli%26aufirst%3DA.%26aulast%3DMaga%26aufirst%3DG.%26aulast%3DBotta%26aufirst%3DM.%26atitle%3DHomology%2520Model-Based%2520Virtual%2520Screening%2520for%2520the%2520Identification%2520of%2520Human%2520Helicase%2520DDX3%2520Inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2443%26epage%3D2454%26doi%3D10.1021%2Facs.jcim.5b00419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref213"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref213'); return false;" data-citation="" class="refNumLink">213</a></strong><div class="NLM_citation" id="cit213"><span><span class="NLM_contrib-group">Carlsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Ligand discovery from a dopamine D3 receptor homology model and crystal structure</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">769</span><span class="NLM_x">–</span> <span class="NLM_lpage">778</span><span class="refDoi"> DOI: 10.1038/nchembio.662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref213/cit213&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnchembio.662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref213/cit213&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=21926995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref213/cit213&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=769-778&author=J.+Carlssonauthor=R.+G.+Colemanauthor=V.+Setolaauthor=J.+J.+Irwinauthor=H.+Fanauthor=A.+Schlessingerauthor=A.+Saliauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Ligand+discovery+from+a+dopamine+D3+receptor+homology+model+and+crystal+structure&doi=10.1038%2Fnchembio.662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit213R"><div class="casContent"><span class="casTitleNuber">213</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand discovery from a dopamine D3 receptor homology model and crystal structure</span></div><div class="casAuthors">Carlsson, Jens; Coleman, Ryan G.; Setola, Vincent; Irwin, John J.; Fan, Hao; Schlessinger, Avner; Sali, Andrej; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">769-778</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are intensely studied as drug targets and for their role in signaling.  With the detn. of the first crystal structures, interest in structure-based ligand discovery increased.  Unfortunately, for most GPCRs no exptl. structures are available.  The detn. of the D3 receptor structure and the challenge to the community to predict it enabled a fully prospective comparison of ligand discovery from a modeled structure vs. that of the subsequently released crystal structure.  Over 3.3 million mols. were docked against a homol. model, and 26 of the highest ranking were tested for binding.  Six had affinities ranging from 0.2 to 3.1 μM.  Subsequently, the crystal structure was released and the docking screen repeated.  Of the 25 compds. selected, five had affinities ranging from 0.3 to 3.0 μM.  One of the new ligands from the homol. model screen was optimized for affinity to 81 nM.  The feasibility of docking screens against modeled GPCRs more generally is considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz6gBx3Olbn7Vg90H21EOLACvtfcHk0liBEvqIROgAEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP&md5=7242aa512079a348678b309d352ecdb9</span></div><a href="/servlet/linkout?suffix=cit213&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.662%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DLigand%2520discovery%2520from%2520a%2520dopamine%2520D3%2520receptor%2520homology%2520model%2520and%2520crystal%2520structure%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D769%26epage%3D778%26doi%3D10.1038%2Fnchembio.662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref214"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref214'); return false;" data-citation="" class="refNumLink">214</a></strong><div class="NLM_citation" id="cit214"><span><span class="NLM_contrib-group">Sorna, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theisen, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stephens, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warner, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vankayalapati, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span> </span><span class="NLM_article-title">High-throughput virtual screening identifies novel N′-(1-phenylethylidene)-benzohydrazides as potent, specific, and reversible LSD1 inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">9496</span><span class="NLM_x">–</span> <span class="NLM_lpage">9508</span><span class="refDoi"> DOI: 10.1021/jm400870h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400870h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref214/cit214&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslyks7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9496-9508&author=V.+Sornaauthor=E.+R.+Theisenauthor=B.+Stephensauthor=S.+L.+Warnerauthor=D.+J.+Bearssauthor=H.+Vankayalapatiauthor=S.+Sharma&title=High-throughput+virtual+screening+identifies+novel+N%E2%80%B2-%281-phenylethylidene%29-benzohydrazides+as+potent%2C+specific%2C+and+reversible+LSD1+inhibitors&doi=10.1021%2Fjm400870h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit214R"><div class="casContent"><span class="casTitleNuber">214</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Virtual Screening Identifies Novel N'-(1-Phenylethylidene)-benzohydrazides as Potent, Specific, and Reversible LSD1 Inhibitors</span></div><div class="casAuthors">Sorna, Venkataswamy; Theisen, Emily R.; Stephens, Bret; Warner, Steven L.; Bearss, David J.; Vankayalapati, Hariprasad; Sharma, Sunil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9496-9508</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) plays an important role in regulating histone lysine methylation at residues K4 and K9 on histone H3 and is an attractive therapeutic target in multiple malignancies.  Here we report a structure-based virtual screen of a compd. library contg. ∼2 million small mol. entities.  Computational docking and scoring followed by biochem. screening led to the identification of a novel N'-(1-phenylethylidene)-benzohydrazide series of LSD1 inhibitors with hits showing biochem. IC50s in the 200-400 nM range.  Hit-to-lead optimization and structure-activity relation studies aided in the discovery of compd. (I), with a Ki of 31 nM.  Compd. I is reversible and specific for LSD1 as compared to the monoamine oxidases shows minimal inhibition of CYPs and hERG and inhibits proliferation and survival in several cancer cell lines, including breast and colorectal cancer.  Compd. I may be used to probe LSD1's biol. role in these cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp97P3ef8ur6rVg90H21EOLACvtfcHk0lhi3_h1bHAbGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslyks7fF&md5=47293fd790c5bc9fd944943ff4b1a58a</span></div><a href="/servlet/linkout?suffix=cit214&amp;dbid=16384&amp;doi=10.1021%2Fjm400870h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400870h%26sid%3Dliteratum%253Aachs%26aulast%3DSorna%26aufirst%3DV.%26aulast%3DTheisen%26aufirst%3DE.%2BR.%26aulast%3DStephens%26aufirst%3DB.%26aulast%3DWarner%26aufirst%3DS.%2BL.%26aulast%3DBearss%26aufirst%3DD.%2BJ.%26aulast%3DVankayalapati%26aufirst%3DH.%26aulast%3DSharma%26aufirst%3DS.%26atitle%3DHigh-throughput%2520virtual%2520screening%2520identifies%2520novel%2520N%25E2%2580%25B2-%25281-phenylethylidene%2529-benzohydrazides%2520as%2520potent%252C%2520specific%252C%2520and%2520reversible%2520LSD1%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9496%26epage%3D9508%26doi%3D10.1021%2Fjm400870h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref215"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref215'); return false;" data-citation="" class="refNumLink">215</a></strong><div class="NLM_citation" id="cit215"><span><span class="NLM_contrib-group">Mahasenan, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span> </span><span class="NLM_article-title">Novel inhibitor discovery through virtual screening against multiple protein conformations generated via ligand-directed modeling: a maternal embryonic leucine zipper kinase example</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1345</span><span class="NLM_x">–</span> <span class="NLM_lpage">1355</span><span class="refDoi"> DOI: 10.1021/ci300040c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300040c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref215/cit215&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmt1Sisr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1345-1355&author=K.+V.+Mahasenanauthor=C.+Li&title=Novel+inhibitor+discovery+through+virtual+screening+against+multiple+protein+conformations+generated+via+ligand-directed+modeling%3A+a+maternal+embryonic+leucine+zipper+kinase+example&doi=10.1021%2Fci300040c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit215R"><div class="casContent"><span class="casTitleNuber">215</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Inhibitor Discovery through Virtual Screening against Multiple Protein Conformations Generated via Ligand-Directed Modeling: A Maternal Embryonic Leucine Zipper Kinase Example</span></div><div class="casAuthors">Mahasenan, Kiran V.; Li, Chenglong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1345-1355</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Kinase targets have been demonstrated to undergo major conformational reorganization upon ligand binding.  Such protein conformational plasticity remains a significant challenge in structure-based virtual screening methodol. and may be approximated by screening against an ensemble of diverse protein conformations.  Maternal embryonic leucine zipper kinase (MELK), a member of serine-threonine kinase family, has been recently found to be involved in the tumerogenic state of glioblastoma, breast, ovarian, and colon cancers.  We therefore modeled several conformers of MELK utilizing the available chemogenomic and crystallog. data of homologous kinases.  We carried out docking pose prediction and virtual screening enrichment studies with these conformers.  The performances of the ensembles were evaluated by their ability to reproduce known inhibitor bioactive conformations and to efficiently recover known active compds. early in the virtual screen when seeded with decoy sets.  A few of the individual MELK conformers performed satisfactorily in reproducing the native protein-ligand pharmacophoric interactions up to 50% of the cases.  By selecting an ensemble of a few representative conformational states, most of the known inhibitor binding poses could be rationalized.  For example, a four conformer ensemble is able to recover 95% of the studied actives, esp. with imperfect scoring function(s).  The virtual screening enrichment varied considerably among different MELK conformers.  Enrichment appears to improve by selection of a proper protein conformation.  For example, several holo and unliganded active conformations are better to accommodate diverse chemotypes than ATP-bound conformer.  These results prove that using an ensemble of diverse conformations could give a better performance.  Applying this approach, we were able to screen a com. available library of half a million compds. against three conformers to discover three novel inhibitors of MELK, one from each template.  Among the three compds. validated via exptl. enzyme inhibition assays, one is relatively potent (15; Kd = 0.37 μM), one moderately active (12; Kd = 3.2 μM), and one weak but very selective (9; Kd = 18 μM).  These novel hits may be utilized to assist in the development of small mol. therapeutic agents useful in diseases caused by deregulated MELK, and perhaps more importantly, the approach demonstrates the advantages of choosing an appropriate ensemble of a few conformers in pursuing compd. potency, selectivity, and novel chemotypes over using single target conformation for structure-based drug design in general.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGranNM-bXItZbVg90H21EOLACvtfcHk0lhi3_h1bHAbGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmt1Sisr0%253D&md5=a17bcbfa52439be4f076182f405460e6</span></div><a href="/servlet/linkout?suffix=cit215&amp;dbid=16384&amp;doi=10.1021%2Fci300040c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300040c%26sid%3Dliteratum%253Aachs%26aulast%3DMahasenan%26aufirst%3DK.%2BV.%26aulast%3DLi%26aufirst%3DC.%26atitle%3DNovel%2520inhibitor%2520discovery%2520through%2520virtual%2520screening%2520against%2520multiple%2520protein%2520conformations%2520generated%2520via%2520ligand-directed%2520modeling%253A%2520a%2520maternal%2520embryonic%2520leucine%2520zipper%2520kinase%2520example%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1345%26epage%3D1355%26doi%3D10.1021%2Fci300040c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref216"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref216'); return false;" data-citation="" class="refNumLink">216</a></strong><div class="NLM_citation" id="cit216"><span><span class="NLM_contrib-group">Nolan, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunstan, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caraher, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratford, I. J.</span><span> </span><span class="NLM_article-title">In silico screening reveals structurally diverse, nanomolar inhibitors of NQO2 that are functionally active in cells and can modulate NF-kappaB signaling</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">194</span><span class="NLM_x">–</span> <span class="NLM_lpage">203</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-11-0543</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref216/cit216&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1158%2F1535-7163.MCT-11-0543" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref216/cit216&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=22090421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref216/cit216&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVWnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=194-203&author=K.+A.+Nolanauthor=M.+S.+Dunstanauthor=M.+C.+Caraherauthor=K.+A.+Scottauthor=D.+Leysauthor=I.+J.+Stratford&title=In+silico+screening+reveals+structurally+diverse%2C+nanomolar+inhibitors+of+NQO2+that+are+functionally+active+in+cells+and+can+modulate+NF-kappaB+signaling&doi=10.1158%2F1535-7163.MCT-11-0543"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit216R"><div class="casContent"><span class="casTitleNuber">216</span><div class="casTitle"><span class="NLM_cas:atitle">In Silico Screening Reveals Structurally Diverse, Nanomolar Inhibitors of NQO2 That Are Functionally Active in Cells and Can Modulate NF-κB Signaling</span></div><div class="casAuthors">Nolan, Karen A.; Dunstan, Mark S.; Caraher, Mary C.; Scott, Katherine A.; Leys, David; Stratford, Ian J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">194-203</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The National Cancer Institute chem. database has been screened using in silico docking to identify novel nanomolar inhibitors of NRH:quinone oxidoreductase 2 (NQO2).  The inhibitors identified from the screen exhibit a diverse range of scaffolds and the structure of one of the inhibitors, NSC13000 cocrystalized with NQO2, has been solved.  This has been used to aid the generation of a structure-activity relationship between the computationally derived binding affinity and exptl. measured enzyme inhibitory potency.  Many of the compds. are functionally active as inhibitors of NQO2 in cells at nontoxic concns.  To show this, advantage was taken of the NQO2-mediated toxicity of the chemotherapeutic drug CB1954.  The toxicity of this drug is substantially reduced when the function of NQO2 is inhibited, and many of the compds. achieve this in cells at nanomolar concns.  The NQO2 inhibitors also attenuated TNFα-mediated, NF-[kcy]B-driven transcriptional activity.  The link between NQO2 and the regulation of NF-[kcy]B was confirmed by using short interfering RNA to NQO2 and by the observation that NRH, the cofactor for NQO2 enzyme activity, could regulate NF-[kcy]B activity in an NQO2-dependent manner.  NF-[kcy]B is a potential therapeutic target and this study reveals an underlying mechanism that may be usable for developing new anticancer drugs.  Mol Cancer Ther; 11(1); 194-203.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8u03ajpk-crVg90H21EOLACvtfcHk0lhi3_h1bHAbGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVWnsQ%253D%253D&md5=4418803505ef49def2b9d44944d3cc62</span></div><a href="/servlet/linkout?suffix=cit216&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-11-0543&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-11-0543%26sid%3Dliteratum%253Aachs%26aulast%3DNolan%26aufirst%3DK.%2BA.%26aulast%3DDunstan%26aufirst%3DM.%2BS.%26aulast%3DCaraher%26aufirst%3DM.%2BC.%26aulast%3DScott%26aufirst%3DK.%2BA.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DStratford%26aufirst%3DI.%2BJ.%26atitle%3DIn%2520silico%2520screening%2520reveals%2520structurally%2520diverse%252C%2520nanomolar%2520inhibitors%2520of%2520NQO2%2520that%2520are%2520functionally%2520active%2520in%2520cells%2520and%2520can%2520modulate%2520NF-kappaB%2520signaling%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2012%26volume%3D11%26spage%3D194%26epage%3D203%26doi%3D10.1158%2F1535-7163.MCT-11-0543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref217"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref217'); return false;" data-citation="" class="refNumLink">217</a></strong><div class="NLM_citation" id="cit217"><span><span class="NLM_contrib-group">Borriello, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montes, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lepelletier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leforban, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, W. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demange, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delhomme, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pavoni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarray, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boucher, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dufour, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hermine, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garbay, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hadj-Slimane, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raynaud, F.</span><span> </span><span class="NLM_article-title">Structure-based discovery of a small non-peptidic Neuropilins antagonist exerting in vitro and in vivo anti-tumor activity on breast cancer model</span> <span class="citation_source-journal">Cancer Lett. (N. Y., NY, U. S.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">349</span><span class="NLM_x">, </span> <span class="NLM_fpage">120</span><span class="NLM_x">–</span> <span class="NLM_lpage">127</span><span class="refDoi"> DOI: 10.1016/j.canlet.2014.04.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref217/cit217&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.canlet.2014.04.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref217/cit217&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24752068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref217/cit217&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnvFSltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=349&publication_year=2014&pages=120-127&author=L.+Borrielloauthor=M.+Montesauthor=Y.+Lepelletierauthor=B.+Leforbanauthor=W.+Q.+Liuauthor=L.+Demangeauthor=B.+Delhommeauthor=S.+Pavoniauthor=R.+Jarrayauthor=J.+L.+Boucherauthor=S.+Dufourauthor=O.+Hermineauthor=C.+Garbayauthor=R.+Hadj-Slimaneauthor=F.+Raynaud&title=Structure-based+discovery+of+a+small+non-peptidic+Neuropilins+antagonist+exerting+in+vitro+and+in+vivo+anti-tumor+activity+on+breast+cancer+model&doi=10.1016%2Fj.canlet.2014.04.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit217R"><div class="casContent"><span class="casTitleNuber">217</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of a small non-peptidic Neuropilins antagonist exerting in vitro and in vivo anti-tumor activity on breast cancer model</span></div><div class="casAuthors">Borriello, Lucia; Montes, Matthieu; Lepelletier, Yves; Leforban, Bertrand; Liu, Wang-Qing; Demange, Luc; Delhomme, Brigitte; Pavoni, Serena; Jarray, Rafika; Boucher, Jean Luc; Dufour, Sylvie; Hermine, Olivier; Garbay, Christiane; Hadj-Slimane, Reda; Raynaud, Francoise</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">349</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">120-127</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Neuropilin-1/-2 (+33 NRPs), VEGF-A165 co-receptors, are over-expressed during cancer progression.  Thus, NRPs targeted drug development is challenged using a multistep in silico/in vitro screening procedure.  The first fully non-peptidic VEGF-A165/NRPs protein-protein interaction antagonist (IC50 = 34 μM) without effect on pro-angiogenic kinases has been identified (compd.-1).  This hit showed breast cancer cells anti-proliferative activity (IC50 = 0.60 μM).  Compd.-1 treated NOG-xenografted mice significantly exerted tumor growth inhibition, which is correlated with Ki-67low expression and apoptosis.  Furthermore, CD31+/CD34+ vessels are reduced in accordance with HUVEC-tube formation inhibition (IC50 = 0.20 μM).  Taking together, compd.-1 is the first fully org. inhibitor targeting NRPs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIJJGk2lBSDrVg90H21EOLACvtfcHk0li4EVMEav64FA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnvFSltL0%253D&md5=4a719d3c60463a3c4d7b1e28b3ab4393</span></div><a href="/servlet/linkout?suffix=cit217&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2014.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2014.04.004%26sid%3Dliteratum%253Aachs%26aulast%3DBorriello%26aufirst%3DL.%26aulast%3DMontes%26aufirst%3DM.%26aulast%3DLepelletier%26aufirst%3DY.%26aulast%3DLeforban%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DW.%2BQ.%26aulast%3DDemange%26aufirst%3DL.%26aulast%3DDelhomme%26aufirst%3DB.%26aulast%3DPavoni%26aufirst%3DS.%26aulast%3DJarray%26aufirst%3DR.%26aulast%3DBoucher%26aufirst%3DJ.%2BL.%26aulast%3DDufour%26aufirst%3DS.%26aulast%3DHermine%26aufirst%3DO.%26aulast%3DGarbay%26aufirst%3DC.%26aulast%3DHadj-Slimane%26aufirst%3DR.%26aulast%3DRaynaud%26aufirst%3DF.%26atitle%3DStructure-based%2520discovery%2520of%2520a%2520small%2520non-peptidic%2520Neuropilins%2520antagonist%2520exerting%2520in%2520vitro%2520and%2520in%2520vivo%2520anti-tumor%2520activity%2520on%2520breast%2520cancer%2520model%26jtitle%3DCancer%2520Lett.%2520%2528N.%2520Y.%252C%2520NY%252C%2520U.%2520S.%2529%26date%3D2014%26volume%3D349%26spage%3D120%26epage%3D127%26doi%3D10.1016%2Fj.canlet.2014.04.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref218"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref218'); return false;" data-citation="" class="refNumLink">218</a></strong><div class="NLM_citation" id="cit218"><span><span class="NLM_contrib-group">Choi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ko, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, S.</span><span> </span><span class="NLM_article-title">Structure-based virtual screening approach to the discovery of p38 MAP kinase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">2195</span><span class="NLM_x">–</span> <span class="NLM_lpage">2199</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2012.01.104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref218/cit218&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2012.01.104" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=2195-2199&author=H.+Choiauthor=H.+J.+Parkauthor=J.+C.+Shinauthor=H.+S.+Koauthor=J.+K.+Leeauthor=S.+Leeauthor=H.+Parkauthor=S.+Hong&title=Structure-based+virtual+screening+approach+to+the+discovery+of+p38+MAP+kinase+inhibitors&doi=10.1016%2Fj.bmcl.2012.01.104"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit218&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.01.104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.01.104%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DH.%26aulast%3DPark%26aufirst%3DH.%2BJ.%26aulast%3DShin%26aufirst%3DJ.%2BC.%26aulast%3DKo%26aufirst%3DH.%2BS.%26aulast%3DLee%26aufirst%3DJ.%2BK.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DHong%26aufirst%3DS.%26atitle%3DStructure-based%2520virtual%2520screening%2520approach%2520to%2520the%2520discovery%2520of%2520p38%2520MAP%2520kinase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D2195%26epage%3D2199%26doi%3D10.1016%2Fj.bmcl.2012.01.104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref219"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref219'); return false;" data-citation="" class="refNumLink">219</a></strong><div class="NLM_citation" id="cit219"><span><span class="NLM_contrib-group">Pala, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevaert, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dallocchio, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dessi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogolino, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carcelli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanna, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sechi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naesens, L.</span><span> </span><span class="NLM_article-title">Virtual screening and biological validation of novel influenza virus PA endonuclease Inhibitors</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">866</span><span class="NLM_x">–</span> <span class="NLM_lpage">871</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00109</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00109" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref219/cit219&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVagu7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=866-871&author=N.+Palaauthor=A.+Stevaertauthor=R.+Dallocchioauthor=A.+Dessiauthor=D.+Rogolinoauthor=M.+Carcelliauthor=V.+Sannaauthor=M.+Sechiauthor=L.+Naesens&title=Virtual+screening+and+biological+validation+of+novel+influenza+virus+PA+endonuclease+Inhibitors&doi=10.1021%2Facsmedchemlett.5b00109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit219R"><div class="casContent"><span class="casTitleNuber">219</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Screening and Biological Validation of Novel Influenza Virus PA Endonuclease Inhibitors</span></div><div class="casAuthors">Pala, Nicolino; Stevaert, Annelies; Dallocchio, Roberto; Dessi, Alessandro; Rogolino, Dominga; Carcelli, Mauro; Sanna, Vanna; Sechi, Mario; Naesens, Lieve</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">866-871</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The influenza virus RNA-dependent RNA polymerase complex (RdRp), a heterotrimeric protein complex responsible for viral RNA transcription and replication, represents a primary target for antiviral drug development.  One particularly attractive approach is interference with the endonucleolytic "cap-snatching" reaction by the RdRp subunit PA, more precisely by inhibiting its metal-dependent catalytic activity which resides in the N-terminal part of PA (PA-Nter).  Almost all PA inhibitors (PAIs) thus far discovered bear pharmacophoric fragments with chelating motifs able to bind the bivalent metal ions in the catalytic core of PA-Nter.  More recently, the availability of crystallog. structures of PA-Nter has enabled rational design of original PAIs with improved binding properties and antiviral potency.  We here present a coupled pharmacophore/docking virtual screening approach that allowed us to identify PAIs with interesting inhibitory activity in a PA-Nter enzymic assay.  Moreover, antiviral activity in the low micromolar range was obsd. in cell-based influenza virus assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKH888RsSFzrVg90H21EOLACvtfcHk0li4EVMEav64FA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVagu7bJ&md5=81f9b315013d32ad71fbc7f6c8ce2e62</span></div><a href="/servlet/linkout?suffix=cit219&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00109%26sid%3Dliteratum%253Aachs%26aulast%3DPala%26aufirst%3DN.%26aulast%3DStevaert%26aufirst%3DA.%26aulast%3DDallocchio%26aufirst%3DR.%26aulast%3DDessi%26aufirst%3DA.%26aulast%3DRogolino%26aufirst%3DD.%26aulast%3DCarcelli%26aufirst%3DM.%26aulast%3DSanna%26aufirst%3DV.%26aulast%3DSechi%26aufirst%3DM.%26aulast%3DNaesens%26aufirst%3DL.%26atitle%3DVirtual%2520screening%2520and%2520biological%2520validation%2520of%2520novel%2520influenza%2520virus%2520PA%2520endonuclease%2520Inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D866%26epage%3D871%26doi%3D10.1021%2Facsmedchemlett.5b00109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref220"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref220'); return false;" data-citation="" class="refNumLink">220</a></strong><div class="NLM_citation" id="cit220"><span><span class="NLM_contrib-group">Di Leva, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zizza, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cingolani, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">D’Angelo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pagano, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amato, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salvati, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sissi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pinato, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marinelli, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosconati, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novellino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Randazzo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biroccio, A.</span><span> </span><span class="NLM_article-title">Exploring the chemical space of G-quadruplex binders: discovery of a novel chemotype targeting the human telomeric sequence</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">9646</span><span class="NLM_x">–</span> <span class="NLM_lpage">9654</span><span class="refDoi"> DOI: 10.1021/jm401185b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401185b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref220/cit220&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVajtbvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9646-9654&author=F.+S.+Di+Levaauthor=P.+Zizzaauthor=C.+Cingolaniauthor=C.+D%E2%80%99Angeloauthor=B.+Paganoauthor=J.+Amatoauthor=E.+Salvatiauthor=C.+Sissiauthor=O.+Pinatoauthor=L.+Marinelliauthor=A.+Cavalliauthor=S.+Cosconatiauthor=E.+Novellinoauthor=A.+Randazzoauthor=A.+Biroccio&title=Exploring+the+chemical+space+of+G-quadruplex+binders%3A+discovery+of+a+novel+chemotype+targeting+the+human+telomeric+sequence&doi=10.1021%2Fjm401185b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit220R"><div class="casContent"><span class="casTitleNuber">220</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring the Chemical Space of G-Quadruplex Binders: Discovery of a Novel Chemotype Targeting the Human Telomeric Sequence</span></div><div class="casAuthors">Di Leva, Francesco Saverio; Zizza, Pasquale; Cingolani, Chiara; D'Angelo, Carmen; Pagano, Bruno; Amato, Jussara; Salvati, Erica; Sissi, Claudia; Pinato, Odra; Marinelli, Luciana; Cavalli, Andrea; Cosconati, Sandro; Novellino, Ettore; Randazzo, Antonio; Biroccio, Annamaria</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9646-9654</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recent findings have unambiguously demonstrated that DNA G-rich sequences can adopt a G-quadruplex folding in living cells, thus further validating them as crucial targets for anticancer therapy.  Herein, to identify new potent G4 binders as antitumor drug candidates, the authors have targeted a 24-nt G4-forming telomeric sequence employing a receptor-based virtual screening approach.  Among the best candidates, in vitro binding expts. allowed identification of three novel G4 ligands.  Among them, the best compd. features an unprecedented binding selectivity for the human telomeric DNA G-quadruplex with no detectable binding for other G4-forming sequences present at different genomic sites.  This behavior correlates with the detected ability to generate DNA damage response in tumor cells at the telomeric level and efficient antiproliferative effect on different tumor cell lines at low micromolar concns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolkWbQ7-ptEbVg90H21EOLACvtfcHk0lgN2FCm4ciDgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVajtbvK&md5=05c07f98198d5cdc1da2de690182184e</span></div><a href="/servlet/linkout?suffix=cit220&amp;dbid=16384&amp;doi=10.1021%2Fjm401185b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401185b%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BLeva%26aufirst%3DF.%2BS.%26aulast%3DZizza%26aufirst%3DP.%26aulast%3DCingolani%26aufirst%3DC.%26aulast%3DD%25E2%2580%2599Angelo%26aufirst%3DC.%26aulast%3DPagano%26aufirst%3DB.%26aulast%3DAmato%26aufirst%3DJ.%26aulast%3DSalvati%26aufirst%3DE.%26aulast%3DSissi%26aufirst%3DC.%26aulast%3DPinato%26aufirst%3DO.%26aulast%3DMarinelli%26aufirst%3DL.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DCosconati%26aufirst%3DS.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DRandazzo%26aufirst%3DA.%26aulast%3DBiroccio%26aufirst%3DA.%26atitle%3DExploring%2520the%2520chemical%2520space%2520of%2520G-quadruplex%2520binders%253A%2520discovery%2520of%2520a%2520novel%2520chemotype%2520targeting%2520the%2520human%2520telomeric%2520sequence%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9646%26epage%3D9654%26doi%3D10.1021%2Fjm401185b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref221"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref221'); return false;" data-citation="" class="refNumLink">221</a></strong><div class="NLM_citation" id="cit221"><span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span> </span><span class="NLM_article-title">Discovery of potent small molecule inhibitors of DYRK1A by structure-based virtual screening and bioassay</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">168</span><span class="NLM_x">–</span> <span class="NLM_lpage">171</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2011.11.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref221/cit221&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2011.11.043" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=168-171&author=D.+Wangauthor=F.+Wangauthor=Y.+Tanauthor=L.+Dongauthor=L.+Chenauthor=W.+Zhuauthor=H.+Wang&title=Discovery+of+potent+small+molecule+inhibitors+of+DYRK1A+by+structure-based+virtual+screening+and+bioassay&doi=10.1016%2Fj.bmcl.2011.11.043"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit221&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.11.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.11.043%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DTan%26aufirst%3DY.%26aulast%3DDong%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DH.%26atitle%3DDiscovery%2520of%2520potent%2520small%2520molecule%2520inhibitors%2520of%2520DYRK1A%2520by%2520structure-based%2520virtual%2520screening%2520and%2520bioassay%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D168%26epage%3D171%26doi%3D10.1016%2Fj.bmcl.2011.11.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref222"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref222'); return false;" data-citation="" class="refNumLink">222</a></strong><div class="NLM_citation" id="cit222"><span><span class="NLM_contrib-group">de Graaf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piwnica, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giordanetto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Structure-based discovery of allosteric modulators of two related class B G-protein-coupled receptors</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">2159</span><span class="NLM_x">–</span> <span class="NLM_lpage">2169</span><span class="refDoi"> DOI: 10.1002/cmdc.201100317</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref222/cit222&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fcmdc.201100317" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref222/cit222&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=21994134" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref222/cit222&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht12qsbfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=2159-2169&author=C.+de+Graafauthor=C.+Reinauthor=D.+Piwnicaauthor=F.+Giordanettoauthor=D.+Rognan&title=Structure-based+discovery+of+allosteric+modulators+of+two+related+class+B+G-protein-coupled+receptors&doi=10.1002%2Fcmdc.201100317"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit222R"><div class="casContent"><span class="casTitleNuber">222</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of Allosteric Modulators of Two Related Class B G-Protein-Coupled Receptors</span></div><div class="casAuthors">de Graaf, Chris; Rein, Chantal; Piwnica, David; Giordanetto, Fabrizio; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2159-2169</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Despite the availability of X-ray crystal structure data for several members of the G-protein-coupled receptor (GPCR) superfamily, structure-based discovery of GPCR ligands has been exclusively restricted to class A (rhodopsin-like) receptors.  Herein we report the identification, by a docking-based virtual screening approach, of noncompetitive ligands for two related class B (secretin-like) GPCRs: the glucagon receptor (GLR) and the glucagon-like peptide 1 receptor (GLP-1R).  Starting from a knowledge-based three-dimensional model of the GLR, a database of 1.9 million com. available drug-like compds. was screened for chem. similarity to existing GLR noncompetitive antagonists and docked to the transmembrane cavity of the GLR; 23 compds. were then selected based on protein-ligand interaction fingerprints, and were then purchased and evaluated for in vitro binding to GLR and modulation of glucagon-induced cAMP release.  Two of the 23 compds. inhibited the effect of glucagon in a dose-dependent manner, with one inhibitor exhibiting the same potency as L-168 049, a ref. noncompetitive GLR antagonist, in a whole-cell-based functional assay.  Interestingly, one virtual hit that was inactive at the GLR was shown to bind to GLP-1R and potentiate the response to the endogenous GLP-1 ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLnGlSRCe5d7Vg90H21EOLACvtfcHk0lgN2FCm4ciDgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht12qsbfP&md5=ea40e9f9394d32b7a084dcf5048f2b32</span></div><a href="/servlet/linkout?suffix=cit222&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201100317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201100317%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26aulast%3DRein%26aufirst%3DC.%26aulast%3DPiwnica%26aufirst%3DD.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DStructure-based%2520discovery%2520of%2520allosteric%2520modulators%2520of%2520two%2520related%2520class%2520B%2520G-protein-coupled%2520receptors%26jtitle%3DChemMedChem%26date%3D2011%26volume%3D6%26spage%3D2159%26epage%3D2169%26doi%3D10.1002%2Fcmdc.201100317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref223"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref223'); return false;" data-citation="" class="refNumLink">223</a></strong><div class="NLM_citation" id="cit223"><span><span class="NLM_contrib-group">Kalid, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toledo Warshaviak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shechter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shacham, S.</span><span> </span><span class="NLM_article-title">Consensus Induced Fit Docking (cIFD): methodology, validation, and application to the discovery of novel Crm1 inhibitors</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1217</span><span class="NLM_x">–</span> <span class="NLM_lpage">1228</span><span class="refDoi"> DOI: 10.1007/s10822-012-9611-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref223/cit223&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs10822-012-9611-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=1217-1228&author=O.+Kalidauthor=D.+Toledo+Warshaviakauthor=S.+Shechterauthor=W.+Shermanauthor=S.+Shacham&title=Consensus+Induced+Fit+Docking+%28cIFD%29%3A+methodology%2C+validation%2C+and+application+to+the+discovery+of+novel+Crm1+inhibitors&doi=10.1007%2Fs10822-012-9611-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit223&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9611-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9611-9%26sid%3Dliteratum%253Aachs%26aulast%3DKalid%26aufirst%3DO.%26aulast%3DToledo%2BWarshaviak%26aufirst%3DD.%26aulast%3DShechter%26aufirst%3DS.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DShacham%26aufirst%3DS.%26atitle%3DConsensus%2520Induced%2520Fit%2520Docking%2520%2528cIFD%2529%253A%2520methodology%252C%2520validation%252C%2520and%2520application%2520to%2520the%2520discovery%2520of%2520novel%2520Crm1%2520inhibitors%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D1217%26epage%3D1228%26doi%3D10.1007%2Fs10822-012-9611-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref224"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref224'); return false;" data-citation="" class="refNumLink">224</a></strong><div class="NLM_citation" id="cit224"><span><span class="NLM_contrib-group">Chang, C. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeyachandran, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, M. H.</span><span> </span><span class="NLM_article-title">Structure-based virtual screening and experimental validation of the discovery of inhibitors targeted towards the human coronavirus nucleocapsid protein</span> <span class="citation_source-journal">Mol. BioSyst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">59</span><span class="NLM_x">–</span> <span class="NLM_lpage">66</span><span class="refDoi"> DOI: 10.1039/C5MB00582E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref224/cit224&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1039%2FC5MB00582E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref224/cit224&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26542199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref224/cit224&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1yktrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=59-66&author=C.+K.+Changauthor=S.+Jeyachandranauthor=N.+J.+Huauthor=C.+L.+Liuauthor=S.+Y.+Linauthor=Y.+S.+Wangauthor=Y.+M.+Changauthor=M.+H.+Hou&title=Structure-based+virtual+screening+and+experimental+validation+of+the+discovery+of+inhibitors+targeted+towards+the+human+coronavirus+nucleocapsid+protein&doi=10.1039%2FC5MB00582E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit224R"><div class="casContent"><span class="casTitleNuber">224</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based virtual screening and experimental validation of the discovery of inhibitors targeted towards the human coronavirus nucleocapsid protein</span></div><div class="casAuthors">Chang, Chung-ke; Jeyachandran, Sivakamavalli; Hu, Nien-Jen; Liu, Chia-Ling; Lin, Shing-Yen; Wang, Yong-Sheng; Chang, Yu-Ming; Hou, Ming-Hon</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">59-66</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Nucleocapsid protein (NP), an essential RNA-binding viral protein in human coronavirus (CoV)-infected cells, is required for the replication and transcription of viral RNA.  Recent studies suggested that human CoV NP is a valid target for antiviral drug development.  Based on this aspect, structure-based virtual screening targeting nucleocapsid protein (NP) was performed to identify good chem. starting points for medicinal chem.  The present study utilized structure-based virtual screening against human CoV-OC43 using the Zinc database, which is performed through docking with varying precisions and computational intensities to identify eight potential compds.  The chosen potential leads were further validated exptl. using biophys. means.  Surface plasmon resonance (SPR) anal. indicated that one among the potential leads, 6-chloro-7-(2-morpholin-4-yl-ethylamino) quinoxaline-5,8-dione (small-compd. H3), exhibited a significant decrease of RNA-binding capacity of NP by more than 20%.  The loss of binding activity was manifested as a 20% decrease in the min. on-rate accompanied with a 70% increase in the max. off-rate.  Fluorescence titrn. and X-ray crystallog. studies indicated that H3 antagonizes the binding between HCoV-OC43 NP and RNA by interacting with the N-terminal domain of the NP.  Our findings provide insight into the development of new therapeutics that disrupt the interaction between RNA and viral NP in the HCoV.  The discovery of the new compd. would be an impetus to design novel NP inhibitors against human CoV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrR4yjm6xLe7rVg90H21EOLACvtfcHk0ljxd9ez0Quy9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1yktrzL&md5=31ef327deaa11323822157670a539092</span></div><a href="/servlet/linkout?suffix=cit224&amp;dbid=16384&amp;doi=10.1039%2FC5MB00582E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5MB00582E%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DC.%2BK.%26aulast%3DJeyachandran%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DN.%2BJ.%26aulast%3DLiu%26aufirst%3DC.%2BL.%26aulast%3DLin%26aufirst%3DS.%2BY.%26aulast%3DWang%26aufirst%3DY.%2BS.%26aulast%3DChang%26aufirst%3DY.%2BM.%26aulast%3DHou%26aufirst%3DM.%2BH.%26atitle%3DStructure-based%2520virtual%2520screening%2520and%2520experimental%2520validation%2520of%2520the%2520discovery%2520of%2520inhibitors%2520targeted%2520towards%2520the%2520human%2520coronavirus%2520nucleocapsid%2520protein%26jtitle%3DMol.%2520BioSyst.%26date%3D2016%26volume%3D12%26spage%3D59%26epage%3D66%26doi%3D10.1039%2FC5MB00582E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref225"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref225'); return false;" data-citation="" class="refNumLink">225</a></strong><div class="NLM_citation" id="cit225"><span><span class="NLM_contrib-group">Cho, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ioerger, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sacchettini, J. C.</span><span> </span><span class="NLM_article-title">Discovery of novel nitrobenzothiazole inhibitors for Mycobacterium tuberculosis ATP phosphoribosyl transferase (HisG) through virtual screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">5984</span><span class="NLM_x">–</span> <span class="NLM_lpage">5992</span><span class="refDoi"> DOI: 10.1021/jm800328v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800328v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref225/cit225&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVymtb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=5984-5992&author=Y.+Choauthor=T.+R.+Ioergerauthor=J.+C.+Sacchettini&title=Discovery+of+novel+nitrobenzothiazole+inhibitors+for+Mycobacterium+tuberculosis+ATP+phosphoribosyl+transferase+%28HisG%29+through+virtual+screening&doi=10.1021%2Fjm800328v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit225R"><div class="casContent"><span class="casTitleNuber">225</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Nitrobenzothiazole Inhibitors for Mycobacterium tuberculosis ATP Phosphoribosyl Transferase (HisG) through Virtual Screening</span></div><div class="casAuthors">Cho, Yoonsang; Ioerger, Thomas R.; Sacchettini, James C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5984-5992</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HisG is an ATP-phosphoribosyl transferase (ATPPRTase) that catalyzes the first step in the biosynthetic pathway for histidine.  Among the enzymes in this pathway, only HisG represents a potential drug target for tuberculosis.  Only a few inhibitors with limited potency for HisG are currently known.  To discover more potent and diverse inhibitors, virtual screening was performed.  The crystal structure of M. tuberculosis HisG has been solved and reveals a large, solvent-exposed active site with subsites for ATP and PRPP substrates.  Two docking algorithms, GOLD and FLEXX, were used to screen two large libraries, Chembridge and NCI, contg. over 500000 compds. combined.  An initial subset of top-ranked compds. were selected and assayed, and seven were found to have enzyme inhibition activity at micromolar concns.  Several of the hits contained a nitrobenzothiazole fragment, which was predicted to dock into the monophosphate-binding loop, and this binding mode was confirmed by crystallog. evidence.  A secondary screen was performed to identify compds. with similar structures.  Several of these also exhibited micromolar inhibition.  Furthermore, two of the compds. showed bacteriocidal activity in a whole-cell assay against Mycobacterium smegmatis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbgXaEKz2UCrVg90H21EOLACvtfcHk0ljxd9ez0Quy9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVymtb3L&md5=dfec06f998fffff9c121e027616c1dbd</span></div><a href="/servlet/linkout?suffix=cit225&amp;dbid=16384&amp;doi=10.1021%2Fjm800328v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800328v%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DY.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26atitle%3DDiscovery%2520of%2520novel%2520nitrobenzothiazole%2520inhibitors%2520for%2520Mycobacterium%2520tuberculosis%2520ATP%2520phosphoribosyl%2520transferase%2520%2528HisG%2529%2520through%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D5984%26epage%3D5992%26doi%3D10.1021%2Fjm800328v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref226"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref226'); return false;" data-citation="" class="refNumLink">226</a></strong><div class="NLM_citation" id="cit226"><span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jing, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, G.</span><span> </span><span class="NLM_article-title">Discovery of novel androgen receptor antagonists: a hybrid approach of pharmacophore-based and docking-based virtual screening</span> <span class="citation_source-journal">Anti-Cancer Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">747</span><span class="NLM_x">–</span> <span class="NLM_lpage">753</span><span class="refDoi"> DOI: 10.1097/CAD.0000000000000245</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref226/cit226&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1097%2FCAD.0000000000000245" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2015&pages=747-753&author=J.+Liuauthor=B.+Liuauthor=G.+Guoauthor=Y.+Jingauthor=G.+Zhao&title=Discovery+of+novel+androgen+receptor+antagonists%3A+a+hybrid+approach+of+pharmacophore-based+and+docking-based+virtual+screening&doi=10.1097%2FCAD.0000000000000245"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit226&amp;dbid=16384&amp;doi=10.1097%2FCAD.0000000000000245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FCAD.0000000000000245%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DGuo%26aufirst%3DG.%26aulast%3DJing%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520novel%2520androgen%2520receptor%2520antagonists%253A%2520a%2520hybrid%2520approach%2520of%2520pharmacophore-based%2520and%2520docking-based%2520virtual%2520screening%26jtitle%3DAnti-Cancer%2520Drugs%26date%3D2015%26volume%3D26%26spage%3D747%26epage%3D753%26doi%3D10.1097%2FCAD.0000000000000245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref227"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref227'); return false;" data-citation="" class="refNumLink">227</a></strong><div class="NLM_citation" id="cit227"><span><span class="NLM_contrib-group">Wiggers, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocha, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandes, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sesti-Costa, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carneiro, Z. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheleski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">da Silva, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Juliano, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cezari, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silva, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKerrow, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montanari, C. A.</span><span> </span><span class="NLM_article-title">Non-peptidic cruzain inhibitors with trypanocidal activity discovered by virtual screening and in vitro assay</span> <span class="citation_source-journal">PLoS Neglected Trop. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e2370</span><span class="refDoi"> DOI: 10.1371/journal.pntd.0002370</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref227/cit227&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pntd.0002370" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref227/cit227&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=23991231" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref227/cit227&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVOrtrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2013&pages=e2370&author=H.+J.+Wiggersauthor=J.+R.+Rochaauthor=W.+B.+Fernandesauthor=R.+Sesti-Costaauthor=Z.+A.+Carneiroauthor=J.+Cheleskiauthor=A.+B.+da+Silvaauthor=L.+Julianoauthor=M.+H.+Cezariauthor=J.+S.+Silvaauthor=J.+H.+McKerrowauthor=C.+A.+Montanari&title=Non-peptidic+cruzain+inhibitors+with+trypanocidal+activity+discovered+by+virtual+screening+and+in+vitro+assay&doi=10.1371%2Fjournal.pntd.0002370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit227R"><div class="casContent"><span class="casTitleNuber">227</span><div class="casTitle"><span class="NLM_cas:atitle">Non-peptidic cruzain inhibitors with trypanocidal activity discovered by virtual screening and in vitro assay</span></div><div class="casAuthors">Wiggers, Helton J.; Rocha, Josmar R.; Fernandes, William B.; Sesti-Costa, Renata; Carneiro, Zumira A.; Cheleski, Juliana; da Silva, Alberico B. F.; Juliano, Luiz; Cezari, Maria H. S.; Silva, Joao S.; McKerrow, James H.; Montanari, Carlos A.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Neglected Tropical Diseases</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e2370, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">PNTDAM</span>;
        ISSN:<span class="NLM_cas:issn">1935-2735</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">A multi-step cascade strategy using integrated ligand- and target-based virtual screening methods was developed to select a small no. of compds. from the ZINC database to be evaluated for trypanocidal activity.  Winnowing the database to 23 selected compds., 12 non-covalent binding cruzain inhibitors with affinity values (Ki) in the low micromolar range (3-60 μM) acting through a competitive inhibition mechanism were identified.  This mechanism has been confirmed by detg. the binding mode of the cruzain inhibitor Nequimed176 through X-ray crystallog. studies.  Cruzain, a validated therapeutic target for new chemotherapy for Chagas disease, also shares high similarity with the mammalian homolog cathepsin L.  Because increased activity of cathepsin L is related to invasive properties and has been linked to metastatic cancer cells, cruzain inhibitors from the same library were assayed against it.  Affinity values were in a similar range (4-80 μM), yielding poor selectivity towards cruzain but raising the possibility of investigating such inhibitors for their effect on cell proliferation.  In order to select the most promising enzyme inhibitors retaining trypanocidal activity for structure-activity relationship (SAR) studies, the most potent cruzain inhibitors were assayed against T. cruzi-infected cells.  Two compds. were found to have trypanocidal activity.  Using compd. Nequimed42 as precursor, an SAR was established in which the 2-acetamidothiophene-3-carboxamide group was identified as essential for enzyme and parasite inhibition activities.  The IC50 value for compd. Nequimed42 acting against the trypomastigote form of the Tulahuen lacZ strain was found to be 10.6 ± 0.1 μM, tenfold lower than that obtained for benznidazole, which was taken as pos. control.  In addn., by employing the strategy of mol. simplification, a smaller compd. derived from Nequimed42 with a ligand efficiency (LE) of 0.33 kcal mol-1 atom-1 (compd. Nequimed176) is highlighted as a novel non-peptidic, non-covalent cruzain inhibitor as a trypanocidal agent candidate for optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomw90Q6T0IA7Vg90H21EOLACvtfcHk0ljxd9ez0Quy9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVOrtrfO&md5=60b21e0f11e9a35cbd73387a9011dea0</span></div><a href="/servlet/linkout?suffix=cit227&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pntd.0002370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pntd.0002370%26sid%3Dliteratum%253Aachs%26aulast%3DWiggers%26aufirst%3DH.%2BJ.%26aulast%3DRocha%26aufirst%3DJ.%2BR.%26aulast%3DFernandes%26aufirst%3DW.%2BB.%26aulast%3DSesti-Costa%26aufirst%3DR.%26aulast%3DCarneiro%26aufirst%3DZ.%2BA.%26aulast%3DCheleski%26aufirst%3DJ.%26aulast%3Dda%2BSilva%26aufirst%3DA.%2BB.%26aulast%3DJuliano%26aufirst%3DL.%26aulast%3DCezari%26aufirst%3DM.%2BH.%26aulast%3DSilva%26aufirst%3DJ.%2BS.%26aulast%3DMcKerrow%26aufirst%3DJ.%2BH.%26aulast%3DMontanari%26aufirst%3DC.%2BA.%26atitle%3DNon-peptidic%2520cruzain%2520inhibitors%2520with%2520trypanocidal%2520activity%2520discovered%2520by%2520virtual%2520screening%2520and%2520in%2520vitro%2520assay%26jtitle%3DPLoS%2520Neglected%2520Trop.%2520Dis.%26date%3D2013%26volume%3D7%26spage%3De2370%26doi%3D10.1371%2Fjournal.pntd.0002370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref228"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref228'); return false;" data-citation="" class="refNumLink">228</a></strong><div class="NLM_citation" id="cit228"><span><span class="NLM_contrib-group">Park, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oh, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryu, S. E.</span><span> </span><span class="NLM_article-title">Identification of potent VHZ phosphatase inhibitors with structure-based virtual screening</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">226</span><span class="NLM_x">–</span> <span class="NLM_lpage">231</span><span class="refDoi"> DOI: 10.1177/1087057112463067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref228/cit228&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1177%2F1087057112463067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref228/cit228&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=23042075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref228/cit228&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Okt77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=226-231&author=H.+Parkauthor=S.+Y.+Parkauthor=J.+J.+Ohauthor=S.+E.+Ryu&title=Identification+of+potent+VHZ+phosphatase+inhibitors+with+structure-based+virtual+screening&doi=10.1177%2F1087057112463067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit228R"><div class="casContent"><span class="casTitleNuber">228</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of potent VHZ phosphatase inhibitors with structure-based virtual screening</span></div><div class="casAuthors">Park, Hwangseo; Park, So Ya; Oh, Jung Jin; Ryu, Seong Eon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">226-231, 6 pp.</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">VH1-like phosphatase Z (VHZ) has proved to be a promising target for the development of therapeutics for the treatment of human cancers.  Here, we report the first example for a successful application of structure-based virtual screening to identify the novel small-mol. inhibitors of VHZ.  These inhibitors revealed high potencies with the assocd. IC50 values ranging from 3 to 20 μM and were also screened for having desirable physicochem. properties as a drug candidate.  Therefore, they deserve consideration for further development by structure-activity relationship studies to optimize inhibitory and anticancer activities.  Structural features relevant to the stabilization of the newly identified inhibitors in the active site of VHZ are discussed in detail.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqShMNo_EGLu7Vg90H21EOLACvtfcHk0lhSYIGMr_BJdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Okt77J&md5=a4189c62904a6955b69b9546ede0ffb9</span></div><a href="/servlet/linkout?suffix=cit228&amp;dbid=16384&amp;doi=10.1177%2F1087057112463067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057112463067%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DH.%26aulast%3DPark%26aufirst%3DS.%2BY.%26aulast%3DOh%26aufirst%3DJ.%2BJ.%26aulast%3DRyu%26aufirst%3DS.%2BE.%26atitle%3DIdentification%2520of%2520potent%2520VHZ%2520phosphatase%2520inhibitors%2520with%2520structure-based%2520virtual%2520screening%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2013%26volume%3D18%26spage%3D226%26epage%3D231%26doi%3D10.1177%2F1087057112463067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref229"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref229'); return false;" data-citation="" class="refNumLink">229</a></strong><div class="NLM_citation" id="cit229"><span><span class="NLM_contrib-group">Meng, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Discovery and optimization of novel, selective histone methyltransferase SET7 inhibitors by pharmacophore- and docking-based virtual screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">8166</span><span class="NLM_x">–</span> <span class="NLM_lpage">8181</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01154</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01154" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=8166-8181&author=F.+Mengauthor=S.+Chengauthor=H.+Dingauthor=S.+Liuauthor=Y.+Liuauthor=K.+Zhuauthor=S.+Chenauthor=J.+Luauthor=Y.+Xieauthor=L.+Liauthor=R.+Liuauthor=Z.+Shiauthor=Y.+Zhouauthor=Y.+C.+Liuauthor=M.+Zhengauthor=H.+Jiangauthor=W.+Luauthor=H.+Liuauthor=C.+Luo&title=Discovery+and+optimization+of+novel%2C+selective+histone+methyltransferase+SET7+inhibitors+by+pharmacophore-+and+docking-based+virtual+screening&doi=10.1021%2Facs.jmedchem.5b01154"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit229&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01154%26sid%3Dliteratum%253Aachs%26aulast%3DMeng%26aufirst%3DF.%26aulast%3DCheng%26aufirst%3DS.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520novel%252C%2520selective%2520histone%2520methyltransferase%2520SET7%2520inhibitors%2520by%2520pharmacophore-%2520and%2520docking-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D8166%26epage%3D8181%26doi%3D10.1021%2Facs.jmedchem.5b01154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref230"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref230'); return false;" data-citation="" class="refNumLink">230</a></strong><div class="NLM_citation" id="cit230"><span><span class="NLM_contrib-group">Parmenopoulou, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kantsadi, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsirkone, V. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chatzileontiadou, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manta, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zographos, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Molfeta, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Archontis, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agius, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayes, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leonidas, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Komiotis, D.</span><span> </span><span class="NLM_article-title">Structure based inhibitor design targeting glycogen phosphorylase B. Virtual screening, synthesis, biochemical and biological assessment of novel N-acyl-beta-d-glucopyranosylamines</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">4810</span><span class="NLM_x">–</span> <span class="NLM_lpage">4825</span><span class="refDoi"> DOI: 10.1016/j.bmc.2014.06.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref230/cit230&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmc.2014.06.058" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=4810-4825&author=V.+Parmenopoulouauthor=A.+L.+Kantsadiauthor=V.+G.+Tsirkoneauthor=D.+S.+Chatzileontiadouauthor=S.+Mantaauthor=S.+E.+Zographosauthor=C.+Molfetaauthor=G.+Archontisauthor=L.+Agiusauthor=J.+M.+Hayesauthor=D.+D.+Leonidasauthor=D.+Komiotis&title=Structure+based+inhibitor+design+targeting+glycogen+phosphorylase+B.+Virtual+screening%2C+synthesis%2C+biochemical+and+biological+assessment+of+novel+N-acyl-beta-d-glucopyranosylamines&doi=10.1016%2Fj.bmc.2014.06.058"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit230&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2014.06.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2014.06.058%26sid%3Dliteratum%253Aachs%26aulast%3DParmenopoulou%26aufirst%3DV.%26aulast%3DKantsadi%26aufirst%3DA.%2BL.%26aulast%3DTsirkone%26aufirst%3DV.%2BG.%26aulast%3DChatzileontiadou%26aufirst%3DD.%2BS.%26aulast%3DManta%26aufirst%3DS.%26aulast%3DZographos%26aufirst%3DS.%2BE.%26aulast%3DMolfeta%26aufirst%3DC.%26aulast%3DArchontis%26aufirst%3DG.%26aulast%3DAgius%26aufirst%3DL.%26aulast%3DHayes%26aufirst%3DJ.%2BM.%26aulast%3DLeonidas%26aufirst%3DD.%2BD.%26aulast%3DKomiotis%26aufirst%3DD.%26atitle%3DStructure%2520based%2520inhibitor%2520design%2520targeting%2520glycogen%2520phosphorylase%2520B.%2520Virtual%2520screening%252C%2520synthesis%252C%2520biochemical%2520and%2520biological%2520assessment%2520of%2520novel%2520N-acyl-beta-d-glucopyranosylamines%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2014%26volume%3D22%26spage%3D4810%26epage%3D4825%26doi%3D10.1016%2Fj.bmc.2014.06.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref231"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref231'); return false;" data-citation="" class="refNumLink">231</a></strong><div class="NLM_citation" id="cit231"><span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y.</span><span> </span><span class="NLM_article-title">Identification of diverse dipeptidyl peptidase IV inhibitors via structure-based virtual screening</span> <span class="citation_source-journal">J. Mol. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">4033</span><span class="NLM_x">–</span> <span class="NLM_lpage">4042</span><span class="refDoi"> DOI: 10.1007/s00894-012-1394-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref231/cit231&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs00894-012-1394-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref231/cit231&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=22460522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref231/cit231&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1OrtbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=4033-4042&author=C.+Liauthor=W.+Luauthor=C.+Luauthor=W.+Xiaoauthor=X.+Shenauthor=J.+Huangauthor=G.+Liuauthor=Y.+Tang&title=Identification+of+diverse+dipeptidyl+peptidase+IV+inhibitors+via+structure-based+virtual+screening&doi=10.1007%2Fs00894-012-1394-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit231R"><div class="casContent"><span class="casTitleNuber">231</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of diverse dipeptidyl peptidase IV inhibitors via structure-based virtual screening</span></div><div class="casAuthors">Li, Cui; Lu, Weiqiang; Lu, Chunhua; Xiao, Wen; Shen, Xu; Huang, Jin; Liu, Guixia; Tang, Yun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4033-4042</span>CODEN:
                <span class="NLM_cas:coden">JMMOFK</span>;
        ISSN:<span class="NLM_cas:issn">0948-5023</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Dipeptidyl peptidase IV (DPP4) is an important target for the treatment of type II diabetes mellitus.  Inhibition of DPP4 will improve glycemic control in such patients by preventing the rapid breakdown and thereby prolonging the physiol. actions of incretin hormones.  Known DPP4 inhibitors (including marketed drugs and those drug candidates) appear to share similar structural features: the cyanopyrrolidine moieties, the xanthenes/pyrimidine parts and amino-like linkages.  In this study, a multi-step virtual screening strategy including both rigid and flexible docking was employed to search for novel structures with DPP4 inhibition.  From SPECS database, consisting of over 190,000 com. available compds., 99 virtual hits were picked up and 15 of them were eventually identified to have DPP4 inhibitory activities at 5 ∼ 50 μM.  Diverse structures of our compds. were out of usual structural categories.  Hence a pharmacophore model was built to further explore their common binding features on the enzyme.  The results provided a new pathway for the discovery of DPP4 inhibitors and would be helpful for further optimization of DPP4 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0SD-Ww74h5rVg90H21EOLACvtfcHk0ljX1pk6ot7rDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1OrtbbL&md5=7a34282901e72ea605ad4a18d11f0e1b</span></div><a href="/servlet/linkout?suffix=cit231&amp;dbid=16384&amp;doi=10.1007%2Fs00894-012-1394-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00894-012-1394-3%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DXiao%26aufirst%3DW.%26aulast%3DShen%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DTang%26aufirst%3DY.%26atitle%3DIdentification%2520of%2520diverse%2520dipeptidyl%2520peptidase%2520IV%2520inhibitors%2520via%2520structure-based%2520virtual%2520screening%26jtitle%3DJ.%2520Mol.%2520Model.%26date%3D2012%26volume%3D18%26spage%3D4033%26epage%3D4042%26doi%3D10.1007%2Fs00894-012-1394-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref232"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref232'); return false;" data-citation="" class="refNumLink">232</a></strong><div class="NLM_citation" id="cit232"><span><span class="NLM_contrib-group">Lagos, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vecchiola, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allende, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuentes, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tichauer, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valdivia, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campino, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tapia-Castillo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baudrand, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villarroel, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cifuentes, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carvajal, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fardella, C. E.</span><span> </span><span class="NLM_article-title">Identification of novel 11beta-HSD1 inhibitors by combined ligand- and structure-based virtual screening</span> <span class="citation_source-journal">Mol. Cell. Endocrinol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">384</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span><span class="refDoi"> DOI: 10.1016/j.mce.2014.01.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref232/cit232&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.mce.2014.01.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref232/cit232&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24447464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref232/cit232&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjslOqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=2014&pages=71-82&author=C.+F.+Lagosauthor=A.+Vecchiolaauthor=F.+Allendeauthor=C.+A.+Fuentesauthor=J.+E.+Tichauerauthor=C.+Valdiviaauthor=S.+Solariauthor=C.+Campinoauthor=A.+Tapia-Castilloauthor=R.+Baudrandauthor=P.+Villarroelauthor=M.+Cifuentesauthor=G.+I.+Owenauthor=C.+A.+Carvajalauthor=C.+E.+Fardella&title=Identification+of+novel+11beta-HSD1+inhibitors+by+combined+ligand-+and+structure-based+virtual+screening&doi=10.1016%2Fj.mce.2014.01.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit232R"><div class="casContent"><span class="casTitleNuber">232</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel 11β-HSD1 inhibitors by combined ligand- and structure-based virtual screening</span></div><div class="casAuthors">Lagos, Carlos F.; Vecchiola, Andrea; Allende, Fidel; Fuentes, Cristobal A.; Tichauer, Juan E.; Valdivia, Carolina; Solari, Sandra; Campino, Carmen; Tapia-Castillo, Alejandra; Baudrand, Rene; Villarroel, Pia; Cifuentes, Mariana; Owen, Gareth I.; Carvajal, Cristian A.; Fardella, Carlos E.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Endocrinology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">384</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">71-82</span>CODEN:
                <span class="NLM_cas:coden">MCEND6</span>;
        ISSN:<span class="NLM_cas:issn">0303-7207</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">11 Beta-hydroxysteroid dehydrogenase type 1 (11β-HSD1) converts cortisone to cortisol in a NADPH dependent manner.  Overexpression of 11β-HSD1 in key metabolic tissues is related to the development of type 2 diabetes, obesity, hypertension and metabolic syndrome.  Using crystal structures of human 11β-HSD1 in complex with inhibitors as source of structural information, a combined ligand and structure-based virtual screening approach was implemented to identify novel 11β-HSD1 inhibitors.  A selected group of compds. was identified in silico and further evaluated in cell-based assays for cytotoxicity and 11β-HSD1 mediated cortisol prodn. inhibitory capacity.  The expression of 11β-HSD1 and 11β-HSD2 in human LS14 adipocytes was assessed during differentiation.  Biol. evaluation of 39 compds. in adipocytes and steroids quantification by HPLC-MS/MS identify 4 compds. that exhibit 11β-HSD1 mediated cortisol prodn. inhibitory activity with potencies in the micromolar range.  Two compds. showed to be selective for the 11β-HSD1 reductase activity and over 11β-HSD2 isoform, and thus represent novel leads for the development of more active derivs. with higher efficacies targeting intracellular cortisol levels in type 2 diabetes and metabolic syndrome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXQ1cIMu4wa7Vg90H21EOLACvtfcHk0ljX1pk6ot7rDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjslOqsLs%253D&md5=91793b8b6ab3a5bf39ddc358f680d52f</span></div><a href="/servlet/linkout?suffix=cit232&amp;dbid=16384&amp;doi=10.1016%2Fj.mce.2014.01.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mce.2014.01.011%26sid%3Dliteratum%253Aachs%26aulast%3DLagos%26aufirst%3DC.%2BF.%26aulast%3DVecchiola%26aufirst%3DA.%26aulast%3DAllende%26aufirst%3DF.%26aulast%3DFuentes%26aufirst%3DC.%2BA.%26aulast%3DTichauer%26aufirst%3DJ.%2BE.%26aulast%3DValdivia%26aufirst%3DC.%26aulast%3DSolari%26aufirst%3DS.%26aulast%3DCampino%26aufirst%3DC.%26aulast%3DTapia-Castillo%26aufirst%3DA.%26aulast%3DBaudrand%26aufirst%3DR.%26aulast%3DVillarroel%26aufirst%3DP.%26aulast%3DCifuentes%26aufirst%3DM.%26aulast%3DOwen%26aufirst%3DG.%2BI.%26aulast%3DCarvajal%26aufirst%3DC.%2BA.%26aulast%3DFardella%26aufirst%3DC.%2BE.%26atitle%3DIdentification%2520of%2520novel%252011beta-HSD1%2520inhibitors%2520by%2520combined%2520ligand-%2520and%2520structure-based%2520virtual%2520screening%26jtitle%3DMol.%2520Cell.%2520Endocrinol.%26date%3D2014%26volume%3D384%26spage%3D71%26epage%3D82%26doi%3D10.1016%2Fj.mce.2014.01.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref233"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref233'); return false;" data-citation="" class="refNumLink">233</a></strong><div class="NLM_citation" id="cit233"><span><span class="NLM_contrib-group">Cancellieri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bassetto, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Widjaja, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Kuppeveld, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Haan, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brancale, A.</span><span> </span><span class="NLM_article-title">In silico structure-based design and synthesis of novel anti-RSV compounds</span> <span class="citation_source-journal">Antiviral Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">122</span><span class="NLM_x">, </span> <span class="NLM_fpage">46</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1016/j.antiviral.2015.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref233/cit233&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.antiviral.2015.08.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2015&pages=46-50&author=M.+Cancellieriauthor=M.+Bassettoauthor=I.+Widjajaauthor=F.+van+Kuppeveldauthor=C.+A.+de+Haanauthor=A.+Brancale&title=In+silico+structure-based+design+and+synthesis+of+novel+anti-RSV+compounds&doi=10.1016%2Fj.antiviral.2015.08.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit233&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2015.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2015.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DCancellieri%26aufirst%3DM.%26aulast%3DBassetto%26aufirst%3DM.%26aulast%3DWidjaja%26aufirst%3DI.%26aulast%3Dvan%2BKuppeveld%26aufirst%3DF.%26aulast%3Dde%2BHaan%26aufirst%3DC.%2BA.%26aulast%3DBrancale%26aufirst%3DA.%26atitle%3DIn%2520silico%2520structure-based%2520design%2520and%2520synthesis%2520of%2520novel%2520anti-RSV%2520compounds%26jtitle%3DAntiviral%2520Res.%26date%3D2015%26volume%3D122%26spage%3D46%26epage%3D50%26doi%3D10.1016%2Fj.antiviral.2015.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref234"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref234'); return false;" data-citation="" class="refNumLink">234</a></strong><div class="NLM_citation" id="cit234"><span><span class="NLM_contrib-group">Rettenmaier, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karpiak, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doak, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, J. A.</span><span> </span><span class="NLM_article-title">Small-molecule allosteric modulators of the protein kinase PDK1 from structure-based docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">8285</span><span class="NLM_x">–</span> <span class="NLM_lpage">8291</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref234/cit234&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Wls77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=8285-8291&author=T.+J.+Rettenmaierauthor=H.+Fanauthor=J.+Karpiakauthor=A.+Doakauthor=A.+Saliauthor=B.+K.+Shoichetauthor=J.+A.+Wells&title=Small-molecule+allosteric+modulators+of+the+protein+kinase+PDK1+from+structure-based+docking&doi=10.1021%2Facs.jmedchem.5b01216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit234R"><div class="casContent"><span class="casTitleNuber">234</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Allosteric Modulators of the Protein Kinase PDK1 from Structure-Based Docking</span></div><div class="casAuthors">Rettenmaier, T. Justin; Fan, Hao; Karpiak, Joel; Doak, Allison; Sali, Andrej; Shoichet, Brian K.; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8285-8291</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Finding small mols. that target allosteric sites remains a grand challenge for ligand discovery.  In the protein kinase field, only a handful of highly selective allosteric modulators have been found.  Thus, more general methods are needed to discover allosteric modulators for addnl. kinases.  Here, the authors use virtual screening against an ensemble of both crystal structures and comparative models to identify ligands for an allosteric peptide-binding site on the protein kinase PDK1 (the PIF pocket).  The authors optimized these ligands through an analog-by-catalog search that yielded compd. (I), which binds to PDK1 with 8 μM affinity.  The authors confirmed the docking poses by detg. a crystal structure of PDK1 in complex with (I).  Because the PIF pocket appears to be a recurring structural feature of the kinase fold, known generally as the helix αC patch, this approach may enable the discovery of allosteric modulators for other kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDLc4nMZELgLVg90H21EOLACvtfcHk0ljX1pk6ot7rDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Wls77P&md5=2afb6d39150cf74a80bf649c94df607c</span></div><a href="/servlet/linkout?suffix=cit234&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01216%26sid%3Dliteratum%253Aachs%26aulast%3DRettenmaier%26aufirst%3DT.%2BJ.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DKarpiak%26aufirst%3DJ.%26aulast%3DDoak%26aufirst%3DA.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DSmall-molecule%2520allosteric%2520modulators%2520of%2520the%2520protein%2520kinase%2520PDK1%2520from%2520structure-based%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D8285%26epage%3D8291%26doi%3D10.1021%2Facs.jmedchem.5b01216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref235"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref235'); return false;" data-citation="" class="refNumLink">235</a></strong><div class="NLM_citation" id="cit235"><span><span class="NLM_contrib-group">Gradler, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerber, H. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodenough-Lashua, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ficner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reuter, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stubbs, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">A new target for shigellosis: rational design and crystallographic studies of inhibitors of tRNA-guanine transglycosylase</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">306</span><span class="NLM_x">, </span> <span class="NLM_fpage">455</span><span class="NLM_x">–</span> <span class="NLM_lpage">467</span><span class="refDoi"> DOI: 10.1006/jmbi.2000.4256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref235/cit235&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1006%2Fjmbi.2000.4256" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=306&publication_year=2001&pages=455-467&author=U.+Gradlerauthor=H.+D.+Gerberauthor=D.+M.+Goodenough-Lashuaauthor=G.+A.+Garciaauthor=R.+Ficnerauthor=K.+Reuterauthor=M.+T.+Stubbsauthor=G.+Klebe&title=A+new+target+for+shigellosis%3A+rational+design+and+crystallographic+studies+of+inhibitors+of+tRNA-guanine+transglycosylase&doi=10.1006%2Fjmbi.2000.4256"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit235&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.2000.4256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.2000.4256%26sid%3Dliteratum%253Aachs%26aulast%3DGradler%26aufirst%3DU.%26aulast%3DGerber%26aufirst%3DH.%2BD.%26aulast%3DGoodenough-Lashua%26aufirst%3DD.%2BM.%26aulast%3DGarcia%26aufirst%3DG.%2BA.%26aulast%3DFicner%26aufirst%3DR.%26aulast%3DReuter%26aufirst%3DK.%26aulast%3DStubbs%26aufirst%3DM.%2BT.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DA%2520new%2520target%2520for%2520shigellosis%253A%2520rational%2520design%2520and%2520crystallographic%2520studies%2520of%2520inhibitors%2520of%2520tRNA-guanine%2520transglycosylase%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2001%26volume%3D306%26spage%3D455%26epage%3D467%26doi%3D10.1006%2Fjmbi.2000.4256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref236"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref236'); return false;" data-citation="" class="refNumLink">236</a></strong><div class="NLM_citation" id="cit236"><span><span class="NLM_contrib-group">Ramirez, U. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nikonova, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pecherskaya, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawrence, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serebriiskii, I. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Einarson, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golemis, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaffe, E. K.</span><span> </span><span class="NLM_article-title">Compounds identified by virtual docking to a tetrameric EGFR extracellular domain can modulate Grb2 internalization</span> <span class="citation_source-journal">BMC Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">436</span><span class="refDoi"> DOI: 10.1186/s12885-015-1415-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref236/cit236&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1186%2Fs12885-015-1415-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=436&author=U.+D.+Ramirezauthor=A.+S.+Nikonovaauthor=H.+Liuauthor=A.+Pecherskayaauthor=S.+H.+Lawrenceauthor=I.+G.+Serebriiskiiauthor=Y.+Zhouauthor=M.+K.+Robinsonauthor=M.+B.+Einarsonauthor=E.+A.+Golemisauthor=E.+K.+Jaffe&title=Compounds+identified+by+virtual+docking+to+a+tetrameric+EGFR+extracellular+domain+can+modulate+Grb2+internalization&doi=10.1186%2Fs12885-015-1415-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit236&amp;dbid=16384&amp;doi=10.1186%2Fs12885-015-1415-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12885-015-1415-6%26sid%3Dliteratum%253Aachs%26aulast%3DRamirez%26aufirst%3DU.%2BD.%26aulast%3DNikonova%26aufirst%3DA.%2BS.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DPecherskaya%26aufirst%3DA.%26aulast%3DLawrence%26aufirst%3DS.%2BH.%26aulast%3DSerebriiskii%26aufirst%3DI.%2BG.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DRobinson%26aufirst%3DM.%2BK.%26aulast%3DEinarson%26aufirst%3DM.%2BB.%26aulast%3DGolemis%26aufirst%3DE.%2BA.%26aulast%3DJaffe%26aufirst%3DE.%2BK.%26atitle%3DCompounds%2520identified%2520by%2520virtual%2520docking%2520to%2520a%2520tetrameric%2520EGFR%2520extracellular%2520domain%2520can%2520modulate%2520Grb2%2520internalization%26jtitle%3DBMC%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D436%26doi%3D10.1186%2Fs12885-015-1415-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref237"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref237'); return false;" data-citation="" class="refNumLink">237</a></strong><div class="NLM_citation" id="cit237"><span><span class="NLM_contrib-group">Gasser, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brogi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urayama, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurose, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tafi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ribeiro, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desaubry, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nebigil, C. G.</span><span> </span><span class="NLM_article-title">Discovery and cardioprotective effects of the first non-peptide agonists of the G protein-coupled prokineticin receptor-1</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0121027</span><span class="refDoi"> DOI: 10.1371/journal.pone.0121027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref237/cit237&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pone.0121027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref237/cit237&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25831128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref237/cit237&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1KhtLjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0121027&author=A.+Gasserauthor=S.+Brogiauthor=K.+Urayamaauthor=T.+Nishiauthor=H.+Kuroseauthor=A.+Tafiauthor=N.+Ribeiroauthor=L.+Desaubryauthor=C.+G.+Nebigil&title=Discovery+and+cardioprotective+effects+of+the+first+non-peptide+agonists+of+the+G+protein-coupled+prokineticin+receptor-1&doi=10.1371%2Fjournal.pone.0121027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit237R"><div class="casContent"><span class="casTitleNuber">237</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and cardioprotective effects of the first non-peptide agonists of the G protein-coupled prokineticin receptor-1</span></div><div class="casAuthors">Gasser, Adeline; Brogi, Simone; Urayama, Kyoji; Nishi, Toshishide; Kurose, Hitoshi; Tafi, Andrea; Ribeiro, Nigel; Desaubry, Laurent; Nebigil, Canan G.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e0121027/1-e0121027/20</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Prokineticins are angiogenic hormones that activate two G protein-coupled receptors: PKR1 and PKR2.  PKR1 has emerged as a crit. mediator of cardiovascular homeostasis and cardioprotection.  Identification of non-peptide PKR1 agonists that contribute to myocardial repair and collateral vessel growth hold promises for treatment of heart diseases.  Through a combination of in silico studies, medicinal chem., and pharmacol. profiling approaches, we designed, synthesized, and characterized the first PKR1 agonists, demonstrating their cardioprotective activity against myocardial infarction (MI) in mice.  Based on high throughput docking protocol, 250,000 compds. were computationally screened for putative PKR1 agonistic activity, using a homol. model, and 10 virtual hits were pharmacol. evaluated.  One hit internalizes PKR1, increases calcium release and activates ERK and Akt kinases.  Among the 30 derivs. of the hit compd., the most potent deriv., IS20, was confirmed for its selectivity and specificity through genetic gain- and loss-of-function of PKR1.  Importantly, IS20 prevented cardiac lesion formation and improved cardiac function after MI in mice, promoting proliferation of cardiac progenitor cells and neovasculogenesis.  The preclin. investigation of the first PKR1 agonists provides a novel approach to promote cardiac neovasculogenesis after MI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdwKeAR30Fy7Vg90H21EOLACvtfcHk0liw82HF-VNMBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1KhtLjE&md5=57ee5a56af6d324b0a8875f249532f4f</span></div><a href="/servlet/linkout?suffix=cit237&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0121027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0121027%26sid%3Dliteratum%253Aachs%26aulast%3DGasser%26aufirst%3DA.%26aulast%3DBrogi%26aufirst%3DS.%26aulast%3DUrayama%26aufirst%3DK.%26aulast%3DNishi%26aufirst%3DT.%26aulast%3DKurose%26aufirst%3DH.%26aulast%3DTafi%26aufirst%3DA.%26aulast%3DRibeiro%26aufirst%3DN.%26aulast%3DDesaubry%26aufirst%3DL.%26aulast%3DNebigil%26aufirst%3DC.%2BG.%26atitle%3DDiscovery%2520and%2520cardioprotective%2520effects%2520of%2520the%2520first%2520non-peptide%2520agonists%2520of%2520the%2520G%2520protein-coupled%2520prokineticin%2520receptor-1%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0121027%26doi%3D10.1371%2Fjournal.pone.0121027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref238"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref238'); return false;" data-citation="" class="refNumLink">238</a></strong><div class="NLM_citation" id="cit238"><span><span class="NLM_contrib-group">Jang, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seo, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">No, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pae, A. N.</span><span> </span><span class="NLM_article-title">Novel scaffold identification of mGlu1 receptor negative allosteric modulators using a hierarchical virtual screening approach</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">87</span><span class="NLM_x">, </span> <span class="NLM_fpage">239</span><span class="NLM_x">–</span> <span class="NLM_lpage">256</span><span class="refDoi"> DOI: 10.1111/cbdd.12654</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref238/cit238&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1111%2Fcbdd.12654" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref238/cit238&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26343933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref238/cit238&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1aksbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2016&pages=239-256&author=J.+W.+Jangauthor=N.+C.+Choauthor=S.+J.+Minauthor=Y.+S.+Choauthor=K.+D.+Parkauthor=S.+H.+Seoauthor=K.+T.+Noauthor=A.+N.+Pae&title=Novel+scaffold+identification+of+mGlu1+receptor+negative+allosteric+modulators+using+a+hierarchical+virtual+screening+approach&doi=10.1111%2Fcbdd.12654"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit238R"><div class="casContent"><span class="casTitleNuber">238</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Scaffold Identification of mGlu1 Receptor Negative Allosteric Modulators Using a Hierarchical Virtual Screening Approach</span></div><div class="casAuthors">Jang, Jae Wan; Cho, Nam-Chul; Min, Sun-Joon; Cho, Yong Seo; Park, Ki Duk; Seo, Seon Hee; No, Kyoung Tai; Pae, Ae Nim</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">239-256</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Metabotropic glutamate receptor 1 (mGluR1) is considered as an attractive drug target for neuropathic pain treatments.  The hierarchical virtual screening approach for identifying novel scaffolds of mGluR1 allosteric modulators was performed using a homol. model built with the dopamine D3 crystal structure as template.  The mGluR1 mutagenesis data, conserved amino acid sequences across class A and class C GPCRs, and previously reported multiple sequence alignments of class C GPCRs to the rhodopsin template, were employed for the sequence alignment to overcome difficulties of model generation with low sequence identity of mGluR1 and dopamine D3.  The structures refined by mol. dynamics simulations were employed for docking of Asinex com. libraries after hierarchical virtual screening with pharmacophore and naive Bayesian models.  Five of 35 compds. exptl. evaluated using a calcium mobilization assay exhibited micromolar activities (IC50) with chemotype novelty that demonstrated the validity of our methods.  A hierarchical structure and ligand-based virtual screening approach with homol. model of class C GPCR based on dopamine D3 class A GPCR structure was successfully performed and applied to discover novel neg. mGluR1 allosteric modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXRRdozrIIkLVg90H21EOLACvtfcHk0lhpiv59thCOKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1aksbbK&md5=8d268d09b46f5311cf48e62f830cb3cf</span></div><a href="/servlet/linkout?suffix=cit238&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12654%26sid%3Dliteratum%253Aachs%26aulast%3DJang%26aufirst%3DJ.%2BW.%26aulast%3DCho%26aufirst%3DN.%2BC.%26aulast%3DMin%26aufirst%3DS.%2BJ.%26aulast%3DCho%26aufirst%3DY.%2BS.%26aulast%3DPark%26aufirst%3DK.%2BD.%26aulast%3DSeo%26aufirst%3DS.%2BH.%26aulast%3DNo%26aufirst%3DK.%2BT.%26aulast%3DPae%26aufirst%3DA.%2BN.%26atitle%3DNovel%2520scaffold%2520identification%2520of%2520mGlu1%2520receptor%2520negative%2520allosteric%2520modulators%2520using%2520a%2520hierarchical%2520virtual%2520screening%2520approach%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2016%26volume%3D87%26spage%3D239%26epage%3D256%26doi%3D10.1111%2Fcbdd.12654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref239"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref239'); return false;" data-citation="" class="refNumLink">239</a></strong><div class="NLM_citation" id="cit239"><span><span class="NLM_contrib-group">Lee, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mittal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gatuz, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Truong, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Torres, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulhearn, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, M. E.</span><span> </span><span class="NLM_article-title">Identification of novel drug scaffolds for inhibition of SARS-CoV 3-chymotrypsin-like protease using virtual and high-throughput screenings</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">167</span><span class="NLM_x">–</span> <span class="NLM_lpage">177</span><span class="refDoi"> DOI: 10.1016/j.bmc.2013.11.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref239/cit239&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmc.2013.11.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref239/cit239&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24332657" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref239/cit239&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOhsbfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=167-177&author=H.+Leeauthor=A.+Mittalauthor=K.+Patelauthor=J.+L.+Gatuzauthor=L.+Truongauthor=J.+Torresauthor=D.+C.+Mulhearnauthor=M.+E.+Johnson&title=Identification+of+novel+drug+scaffolds+for+inhibition+of+SARS-CoV+3-chymotrypsin-like+protease+using+virtual+and+high-throughput+screenings&doi=10.1016%2Fj.bmc.2013.11.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit239R"><div class="casContent"><span class="casTitleNuber">239</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel drug scaffolds for inhibition of SARS-CoV 3-Chymotrypsin-like protease using virtual and high-throughput screenings</span></div><div class="casAuthors">Lee, Hyun; Mittal, Anuradha; Patel, Kavankumar; Gatuz, Joseph L.; Truong, Lena; Torres, Jaime; Mulhearn, Debbie C.; Johnson, Michael E.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">167-177</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We have used a combination of virtual screening (VS) and high-throughput screening (HTS) techniques to identify novel, non-peptidic small mol. inhibitors against human SARS-CoV 3CLpro.  A structure-based VS approach integrating docking and pharmacophore based methods was employed to computationally screen 621,000 compds. from the ZINC library.  The screening protocol was validated using known 3CLpro inhibitors and was optimized for speed, improved selectivity, and for accommodating receptor flexibility.  Subsequently, a fluorescence-based enzymic HTS assay was developed and optimized to exptl. screen approx. 41,000 compds. from four structurally diverse libraries chosen mainly based on the VS results.  False positives from initial HTS hits were eliminated by a secondary orthogonal binding anal. using surface plasmon resonance (SPR).  The campaign identified a reversible small mol. inhibitor exhibiting mixed-type inhibition with a Ki value of 11.1 μM.  Together, these results validate our protocols as suitable approaches to screen virtual and chem. libraries, and the newly identified compd. reported in our study represents a promising structural scaffold to pursue for further SARS-CoV 3CLpro inhibitor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrK4pEI5rpdq7Vg90H21EOLACvtfcHk0lhpiv59thCOKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOhsbfL&md5=8b8f5d24fe89cbf963c0175fb81b03b0</span></div><a href="/servlet/linkout?suffix=cit239&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.11.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.11.041%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%26aulast%3DMittal%26aufirst%3DA.%26aulast%3DPatel%26aufirst%3DK.%26aulast%3DGatuz%26aufirst%3DJ.%2BL.%26aulast%3DTruong%26aufirst%3DL.%26aulast%3DTorres%26aufirst%3DJ.%26aulast%3DMulhearn%26aufirst%3DD.%2BC.%26aulast%3DJohnson%26aufirst%3DM.%2BE.%26atitle%3DIdentification%2520of%2520novel%2520drug%2520scaffolds%2520for%2520inhibition%2520of%2520SARS-CoV%25203-chymotrypsin-like%2520protease%2520using%2520virtual%2520and%2520high-throughput%2520screenings%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2014%26volume%3D22%26spage%3D167%26epage%3D177%26doi%3D10.1016%2Fj.bmc.2013.11.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref240"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref240'); return false;" data-citation="" class="refNumLink">240</a></strong><div class="NLM_citation" id="cit240"><span><span class="NLM_contrib-group">Duffy, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsia, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kharenko, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norek, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gesner, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLure, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lakshminarasimhan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suto, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, H. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitchen, D. B.</span><span> </span><span class="NLM_article-title">Discovery of a new chemical series of BRD4(1) inhibitors using protein-ligand docking and structure-guided design</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">2818</span><span class="NLM_x">–</span> <span class="NLM_lpage">2823</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2015.04.107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref240/cit240&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2015.04.107" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=2818-2823&author=B.+C.+Duffyauthor=S.+Liuauthor=G.+S.+Martinauthor=R.+Wangauthor=M.+M.+Hsiaauthor=H.+Zhaoauthor=C.+Guoauthor=M.+Ellisauthor=J.+F.+Quinnauthor=O.+A.+Kharenkoauthor=K.+Norekauthor=E.+M.+Gesnerauthor=P.+R.+Youngauthor=K.+G.+McLureauthor=G.+S.+Wagnerauthor=D.+Lakshminarasimhanauthor=A.+Whiteauthor=R.+K.+Sutoauthor=H.+C.+Hansenauthor=D.+B.+Kitchen&title=Discovery+of+a+new+chemical+series+of+BRD4%281%29+inhibitors+using+protein-ligand+docking+and+structure-guided+design&doi=10.1016%2Fj.bmcl.2015.04.107"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit240&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.04.107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.04.107%26sid%3Dliteratum%253Aachs%26aulast%3DDuffy%26aufirst%3DB.%2BC.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DHsia%26aufirst%3DM.%2BM.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DGuo%26aufirst%3DC.%26aulast%3DEllis%26aufirst%3DM.%26aulast%3DQuinn%26aufirst%3DJ.%2BF.%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DNorek%26aufirst%3DK.%26aulast%3DGesner%26aufirst%3DE.%2BM.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DWagner%26aufirst%3DG.%2BS.%26aulast%3DLakshminarasimhan%26aufirst%3DD.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DSuto%26aufirst%3DR.%2BK.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3DKitchen%26aufirst%3DD.%2BB.%26atitle%3DDiscovery%2520of%2520a%2520new%2520chemical%2520series%2520of%2520BRD4%25281%2529%2520inhibitors%2520using%2520protein-ligand%2520docking%2520and%2520structure-guided%2520design%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D2818%26epage%3D2823%26doi%3D10.1016%2Fj.bmcl.2015.04.107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref241"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref241'); return false;" data-citation="" class="refNumLink">241</a></strong><div class="NLM_citation" id="cit241"><span><span class="NLM_contrib-group">Ouizougun-Oubari, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pereira, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarus, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galloux, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lassoued, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fix, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tortorici, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoos, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baron, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">England, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desmaele, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couvreur, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bontems, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rey, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eleouet, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sizun, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slama-Schwok, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquerroy, S.</span><span> </span><span class="NLM_article-title">A druggable pocket at the nucleocapsid/phosphoprotein interaction site of human respiratory syncytial virus</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">89</span><span class="NLM_x">, </span> <span class="NLM_fpage">11129</span><span class="NLM_x">–</span> <span class="NLM_lpage">11143</span><span class="refDoi"> DOI: 10.1128/JVI.01612-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref241/cit241&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1128%2FJVI.01612-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref241/cit241&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26246564" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref241/cit241&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslymur7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2015&pages=11129-11143&author=M.+Ouizougun-Oubariauthor=N.+Pereiraauthor=B.+Tarusauthor=M.+Gallouxauthor=S.+Lassouedauthor=J.+Fixauthor=M.+A.+Tortoriciauthor=S.+Hoosauthor=B.+Baronauthor=P.+Englandauthor=D.+Desmaeleauthor=P.+Couvreurauthor=F.+Bontemsauthor=F.+A.+Reyauthor=J.+F.+Eleouetauthor=C.+Sizunauthor=A.+Slama-Schwokauthor=S.+Duquerroy&title=A+druggable+pocket+at+the+nucleocapsid%2Fphosphoprotein+interaction+site+of+human+respiratory+syncytial+virus&doi=10.1128%2FJVI.01612-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit241R"><div class="casContent"><span class="casTitleNuber">241</span><div class="casTitle"><span class="NLM_cas:atitle">A druggable pocket at the nucleocapsid/phosphoprotein interaction site of human respiratory syncytial virus</span></div><div class="casAuthors">Ouizougun-Oubari, Mohamed; Pereira, Nelson; Tarus, Bogdan; Galloux, Marie; Lassoued, Safa; Fix, Jenna; Tortorici, M. Alejandra; Hoos, Sylviane; Baron, Bruno; England, Patrick; Desmaele, Didier; Couvreur, Patrick; Bontems, Francois; Rey, Felix A.; Eleouet, Jean-Francois; Sizun, Christina; Slama-Schwok, Anny; Duquerroy, Stephane</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11129-11143</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Presently, respiratory syncytial virus (RSV), the main cause of severe respiratory infections in infants, cannot be treated efficiently with antivirals.  However, its RNA-dependent polymerase complex offers potential targets for RSV-specific drugs.  This includes the recognition of its template, the ribonucleoprotein complex (RNP), consisting of genomic RNA encapsidated by the RSV nucleoprotein, N.  This recognition proceeds via interaction between the phosphoprotein P, which is the main polymerase cofactor, and N.  The determinant role of the C terminus of P, and more particularly of the last residue, F241, in RNP binding and viral RNA synthesis has been assessed previously.  Here, we provide detailed structural insight into this crucial interaction for RSV polymerase activity.  We solved the crystallog. structures of complexes between the N-terminal domain of N (N-NTD) and C-terminal peptides of P and characterized binding by biophys. approaches.  Our results provide a rationale for the pivotal role of F241, which inserts into a well-defined N-NTD pocket.  This primary binding site is completed by transient contacts with upstream P residues outside the pocket.  Based on the structural information of the N-NTD:P complex, we identified inhibitors of this interaction, selected by in silico screening of small compds., that efficiently bind to N and compete with P in vitro.  One of the compds. displayed inhibitory activity on RSV replication, thereby strengthening the relevance of N-NTD for structure-based design of RSV-specific antivirals.  Structure factors and coordinates have been deposited in the Protein Data Bank under accession codes 4UC6 and 4UC7 for the Nat1 and Nat2 structures of the N-NTD; 4UC8, 4UC9, 4UCA, and 4UCB for the N-NTD in complex with P1, P2, P3, and P7 P-CTD peptides; and 4UCC, 4UCD, and 4UCE for the N-NTD in complex with M76, M81, and M72 compds., resp.  Backbone chem. shift assignments for the N-NTD have been deposited in the BioMagResBank (http://www.bmrb.wisc.edu) under accession no. 25705.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9DPzJ81oHGLVg90H21EOLACvtfcHk0ljQDzRrbIeKYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslymur7F&md5=a373390a4652cbf4255d66b62752ab8a</span></div><a href="/servlet/linkout?suffix=cit241&amp;dbid=16384&amp;doi=10.1128%2FJVI.01612-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.01612-15%26sid%3Dliteratum%253Aachs%26aulast%3DOuizougun-Oubari%26aufirst%3DM.%26aulast%3DPereira%26aufirst%3DN.%26aulast%3DTarus%26aufirst%3DB.%26aulast%3DGalloux%26aufirst%3DM.%26aulast%3DLassoued%26aufirst%3DS.%26aulast%3DFix%26aufirst%3DJ.%26aulast%3DTortorici%26aufirst%3DM.%2BA.%26aulast%3DHoos%26aufirst%3DS.%26aulast%3DBaron%26aufirst%3DB.%26aulast%3DEngland%26aufirst%3DP.%26aulast%3DDesmaele%26aufirst%3DD.%26aulast%3DCouvreur%26aufirst%3DP.%26aulast%3DBontems%26aufirst%3DF.%26aulast%3DRey%26aufirst%3DF.%2BA.%26aulast%3DEleouet%26aufirst%3DJ.%2BF.%26aulast%3DSizun%26aufirst%3DC.%26aulast%3DSlama-Schwok%26aufirst%3DA.%26aulast%3DDuquerroy%26aufirst%3DS.%26atitle%3DA%2520druggable%2520pocket%2520at%2520the%2520nucleocapsid%252Fphosphoprotein%2520interaction%2520site%2520of%2520human%2520respiratory%2520syncytial%2520virus%26jtitle%3DJ.%2520Virol.%26date%3D2015%26volume%3D89%26spage%3D11129%26epage%3D11143%26doi%3D10.1128%2FJVI.01612-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref242"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref242'); return false;" data-citation="" class="refNumLink">242</a></strong><div class="NLM_citation" id="cit242"><span><span class="NLM_contrib-group">Brenk, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vetter, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyce, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodin, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Probing molecular docking in a charged model binding site</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">357</span><span class="NLM_x">, </span> <span class="NLM_fpage">1449</span><span class="NLM_x">–</span> <span class="NLM_lpage">1470</span><span class="refDoi"> DOI: 10.1016/j.jmb.2006.01.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref242/cit242&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.jmb.2006.01.034" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=357&publication_year=2006&pages=1449-1470&author=R.+Brenkauthor=S.+W.+Vetterauthor=S.+E.+Boyceauthor=D.+B.+Goodinauthor=B.+K.+Shoichet&title=Probing+molecular+docking+in+a+charged+model+binding+site&doi=10.1016%2Fj.jmb.2006.01.034"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit242&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2006.01.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2006.01.034%26sid%3Dliteratum%253Aachs%26aulast%3DBrenk%26aufirst%3DR.%26aulast%3DVetter%26aufirst%3DS.%2BW.%26aulast%3DBoyce%26aufirst%3DS.%2BE.%26aulast%3DGoodin%26aufirst%3DD.%2BB.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DProbing%2520molecular%2520docking%2520in%2520a%2520charged%2520model%2520binding%2520site%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D357%26spage%3D1449%26epage%3D1470%26doi%3D10.1016%2Fj.jmb.2006.01.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref243"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref243'); return false;" data-citation="" class="refNumLink">243</a></strong><div class="NLM_citation" id="cit243"><span><span class="NLM_contrib-group">Fischer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraser, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Incorporation of protein flexibility and conformational energy penalties in docking screens to improve ligand discovery</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">575</span><span class="NLM_x">–</span> <span class="NLM_lpage">583</span><span class="refDoi"> DOI: 10.1038/nchem.1954</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref243/cit243&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnchem.1954" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref243/cit243&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24950326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref243/cit243&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1eqsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=575-583&author=M.+Fischerauthor=R.+G.+Colemanauthor=J.+S.+Fraserauthor=B.+K.+Shoichet&title=Incorporation+of+protein+flexibility+and+conformational+energy+penalties+in+docking+screens+to+improve+ligand+discovery&doi=10.1038%2Fnchem.1954"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit243R"><div class="casContent"><span class="casTitleNuber">243</span><div class="casTitle"><span class="NLM_cas:atitle">Incorporation of protein flexibility and conformational energy penalties in docking screens to improve ligand discovery</span></div><div class="casAuthors">Fischer, Marcus; Coleman, Ryan G.; Fraser, James S.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">575-583</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteins fluctuate between alternative conformations, which presents a challenge for ligand discovery because such flexibility is difficult to treat computationally owing to problems with conformational sampling and energy weighting.  Here we describe a flexible docking method that samples and wts. protein conformations using exptl. derived conformations as a guide.  The crystallog. refined occupancies of these conformations, which are observable in an apo receptor structure, define energy penalties for docking.  In a large prospective library screen, we identified new ligands that target specific receptor conformations of a cavity in cytochrome c peroxidase, and we confirm both ligand pose and assocd. receptor conformation predictions by crystallog.  The inclusion of receptor flexibility led to ligands with new chemotypes and phys. properties.  By exploiting exptl. measures of loop and side-chain flexibility, this method can be extended to the discovery of new ligands for hundreds of targets in the Protein Data Bank for which similar exptl. information is available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9q4phmwMLHbVg90H21EOLACvtfcHk0ljQDzRrbIeKYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1eqsLw%253D&md5=141d6f5238664dabb0890cc5f764b99d</span></div><a href="/servlet/linkout?suffix=cit243&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1954&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1954%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DM.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26aulast%3DFraser%26aufirst%3DJ.%2BS.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DIncorporation%2520of%2520protein%2520flexibility%2520and%2520conformational%2520energy%2520penalties%2520in%2520docking%2520screens%2520to%2520improve%2520ligand%2520discovery%26jtitle%3DNat.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D575%26epage%3D583%26doi%3D10.1038%2Fnchem.1954" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref244"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref244'); return false;" data-citation="" class="refNumLink">244</a></strong><div class="NLM_citation" id="cit244"><span><span class="NLM_contrib-group">Wei, B. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baase, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weaver, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthews, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">A model binding site for testing scoring functions in molecular docking</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">322</span><span class="NLM_x">, </span> <span class="NLM_fpage">339</span><span class="NLM_x">–</span> <span class="NLM_lpage">355</span><span class="refDoi"> DOI: 10.1016/S0022-2836(02)00777-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref244/cit244&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2FS0022-2836%2802%2900777-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref244/cit244&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=12217695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref244/cit244&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD38Xms1Wqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=322&publication_year=2002&pages=339-355&author=B.+Q.+Weiauthor=W.+A.+Baaseauthor=L.+H.+Weaverauthor=B.+W.+Matthewsauthor=B.+K.+Shoichet&title=A+model+binding+site+for+testing+scoring+functions+in+molecular+docking&doi=10.1016%2FS0022-2836%2802%2900777-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit244R"><div class="casContent"><span class="casTitleNuber">244</span><div class="casTitle"><span class="NLM_cas:atitle">A Model Binding Site for Testing Scoring Functions in Molecular Docking</span></div><div class="casAuthors">Wei, Binqing Q.; Baase, Walter A.; Weaver, Larry H.; Matthews, Brian W.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">322</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">339-355</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Prediction of interaction energies between ligands and their receptors remains a major challenge for structure-based inhibitor discovery.  Much effort has been devoted to developing scoring schemes that can successfully rank the affinities of a diverse set of possible ligands to a binding site for which the structure is known.  To test these scoring functions, well-characterized exptl. systems can be very useful.  Here, mutation-created binding sites in T4 lysozyme were used to investigate how the quality of at. charges and solvation energies affects mol. docking.  At. charges and solvation energies were calcd. for 172,118 mols. in the Available Chems.  Directory using a semi-empirical quantum mech. approach by the program AMSOL.  The database was first screened against the apolar cavity site created by the mutation Leu99Ala (L99A).  Compared to the electronegativity-based charges that are widely used, the new charges and desolvation energies improved ranking of known apolar ligands, and better distinguished them from more polar isosteres that are not obsd. to bind.  To investigate whether the new charges had predictive value, the non-polar residue Met102, which forms part of the binding site, was changed to the polar residue glutamine.  The structure of the resulting Leu99 Ala and Met102 Gln double mutant of T4 lysozyme (L99A/M102Q) was detd. and the docking calcn. was repeated for the new site.  Seven representative polar mols. that preferentially docked to the polar vs. the apolar binding site were tested exptl.  All seven bind to the polar cavity (L99A/M102Q) but do not detectably bind to the apolar cavity (L99A).  Five ligand-bound structures of L99A/M102Q were detd. by X-ray crystallog.  Docking predictions corresponded to the crystallog. results to within 0.4 A RMSD.  Improved treatment of partial at. charges and desolvation energies in database docking appears feasible and leads to better distinction of true ligands.  Simple model binding sites, such as L99A and its more polar variants, may find broad use in the development and testing of docking algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGral4WDhJW6jbVg90H21EOLACvtfcHk0ljQDzRrbIeKYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xms1Wqsrs%253D&md5=040e009b33f125ab4abbb322cf32d2c9</span></div><a href="/servlet/linkout?suffix=cit244&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2900777-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252900777-5%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DB.%2BQ.%26aulast%3DBaase%26aufirst%3DW.%2BA.%26aulast%3DWeaver%26aufirst%3DL.%2BH.%26aulast%3DMatthews%26aufirst%3DB.%2BW.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DA%2520model%2520binding%2520site%2520for%2520testing%2520scoring%2520functions%2520in%2520molecular%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D322%26spage%3D339%26epage%3D355%26doi%3D10.1016%2FS0022-2836%2802%2900777-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref245"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref245'); return false;" data-citation="" class="refNumLink">245</a></strong><div class="NLM_citation" id="cit245"><span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shivakumar, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyce, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Rescoring docking hit lists for model cavity sites: predictions and experimental testing</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">377</span><span class="NLM_x">, </span> <span class="NLM_fpage">914</span><span class="NLM_x">–</span> <span class="NLM_lpage">934</span><span class="refDoi"> DOI: 10.1016/j.jmb.2008.01.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref245/cit245&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.jmb.2008.01.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref245/cit245&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=18280498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref245/cit245&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsVSltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=377&publication_year=2008&pages=914-934&author=A.+P.+Gravesauthor=D.+M.+Shivakumarauthor=S.+E.+Boyceauthor=M.+P.+Jacobsonauthor=D.+A.+Caseauthor=B.+K.+Shoichet&title=Rescoring+docking+hit+lists+for+model+cavity+sites%3A+predictions+and+experimental+testing&doi=10.1016%2Fj.jmb.2008.01.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit245R"><div class="casContent"><span class="casTitleNuber">245</span><div class="casTitle"><span class="NLM_cas:atitle">Rescoring Docking Hit Lists for Model Cavity Sites: Predictions and Experimental Testing</span></div><div class="casAuthors">Graves, Alan P.; Shivakumar, Devleena M.; Boyce, Sarah E.; Jacobson, Matthew P.; Case, David A.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">377</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">914-934</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Mol. docking computationally screens thousands to millions of org. mols. against protein structures, looking for those with complementary fits.  Many approxns. are made, often resulting in low "hit rates.".  A strategy to overcome these approxns. is to rescore top-ranked docked mols. using a better but slower method.  One such is afforded by mol. mechanics-generalized Born surface area (MM-GBSA) techniques.  These more phys. realistic methods have improved models for solvation and electrostatic interactions and conformational change compared to most docking programs.  To investigate MM-GBSA rescoring, the authors reranked docking hit lists in three small buried sites: a hydrophobic cavity that binds apolar ligands, a slightly polar cavity that binds aryl and hydrogen-bonding ligands, and an anionic cavity that binds cationic ligands.  These sites are simple; consequently, incorrect predictions can be attributed to particular errors in the method, and many likely ligands may actually be tested.  In retrospective calcns., MM-GBSA techniques with binding-site minimization better distinguished the known ligands for each cavity from the known decoys compared to the docking calcn. alone.  This encouraged us to test rescoring prospectively on mols. that ranked poorly by docking but that ranked well when rescored by MM-GBSA.  A total of 33 mols. highly ranked by MM-GBSA for the three cavities were tested exptl.  Of these, 23 were obsd. to bind-these are docking false negatives rescued by rescoring.  The 10 remaining mols. are true negatives by docking and false positives by MM-GBSA.  X-ray crystal structures were detd. for 21 of these 23 mols.  In many cases, the geometry prediction by MM-GBSA improved the initial docking pose and more closely resembled the crystallog. result; yet in several cases, the rescored geometry failed to capture large conformational changes in the protein.  Intriguingly, rescoring not only rescued docking false positives, but also introduced several new false positives into the top-ranking mols.  The authors consider the origins of the successes and failures in MM-GBSA rescoring in these model cavity sites and the prospects for rescoring in biol. relevant targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVMe23LJZDCbVg90H21EOLACvtfcHk0ljB1G4k53pQxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsVSltL0%253D&md5=83bfde66d8b2f032b72b092506f84662</span></div><a href="/servlet/linkout?suffix=cit245&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2008.01.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2008.01.049%26sid%3Dliteratum%253Aachs%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DShivakumar%26aufirst%3DD.%2BM.%26aulast%3DBoyce%26aufirst%3DS.%2BE.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRescoring%2520docking%2520hit%2520lists%2520for%2520model%2520cavity%2520sites%253A%2520predictions%2520and%2520experimental%2520testing%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D377%26spage%3D914%26epage%3D934%26doi%3D10.1016%2Fj.jmb.2008.01.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref246"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref246'); return false;" data-citation="" class="refNumLink">246</a></strong><div class="NLM_citation" id="cit246"><span><span class="NLM_contrib-group">Tanwar, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanwar, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaquiquzzaman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alam, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akhter, M.</span><span> </span><span class="NLM_article-title">Structure based virtual screening of MDPI database: discovery of structurally diverse and novel DPP-IV inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">3447</span><span class="NLM_x">–</span> <span class="NLM_lpage">3451</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2014.05.076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref246/cit246&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2014.05.076" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=3447-3451&author=O.+Tanwarauthor=L.+Tanwarauthor=M.+Shaquiquzzamanauthor=M.+M.+Alamauthor=M.+Akhter&title=Structure+based+virtual+screening+of+MDPI+database%3A+discovery+of+structurally+diverse+and+novel+DPP-IV+inhibitors&doi=10.1016%2Fj.bmcl.2014.05.076"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit246&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.05.076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.05.076%26sid%3Dliteratum%253Aachs%26aulast%3DTanwar%26aufirst%3DO.%26aulast%3DTanwar%26aufirst%3DL.%26aulast%3DShaquiquzzaman%26aufirst%3DM.%26aulast%3DAlam%26aufirst%3DM.%2BM.%26aulast%3DAkhter%26aufirst%3DM.%26atitle%3DStructure%2520based%2520virtual%2520screening%2520of%2520MDPI%2520database%253A%2520discovery%2520of%2520structurally%2520diverse%2520and%2520novel%2520DPP-IV%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D3447%26epage%3D3451%26doi%3D10.1016%2Fj.bmcl.2014.05.076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref247"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref247'); return false;" data-citation="" class="refNumLink">247</a></strong><div class="NLM_citation" id="cit247"><span><span class="NLM_contrib-group">Bavan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luetje, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abaffy, T.</span><span> </span><span class="NLM_article-title">Discovery of novel ligands for mouse olfactory receptor MOR42–3 using an in silico screening approach and in vitro validation</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e92064</span><span class="refDoi"> DOI: 10.1371/journal.pone.0092064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref247/cit247&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pone.0092064" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e92064&author=S.+Bavanauthor=B.+Shermanauthor=C.+W.+Luetjeauthor=T.+Abaffy&title=Discovery+of+novel+ligands+for+mouse+olfactory+receptor+MOR42%E2%80%933+using+an+in+silico+screening+approach+and+in+vitro+validation&doi=10.1371%2Fjournal.pone.0092064"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit247&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0092064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0092064%26sid%3Dliteratum%253Aachs%26aulast%3DBavan%26aufirst%3DS.%26aulast%3DSherman%26aufirst%3DB.%26aulast%3DLuetje%26aufirst%3DC.%2BW.%26aulast%3DAbaffy%26aufirst%3DT.%26atitle%3DDiscovery%2520of%2520novel%2520ligands%2520for%2520mouse%2520olfactory%2520receptor%2520MOR42%25E2%2580%25933%2520using%2520an%2520in%2520silico%2520screening%2520approach%2520and%2520in%2520vitro%2520validation%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26spage%3De92064%26doi%3D10.1371%2Fjournal.pone.0092064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref248"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref248'); return false;" data-citation="" class="refNumLink">248</a></strong><div class="NLM_citation" id="cit248"><span><span class="NLM_contrib-group">Murgueitio, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henneke, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glossmann, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santos-Sierra, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolber, G.</span><span> </span><span class="NLM_article-title">Prospective virtual screening in a sparse data scenario: design of small-molecule TLR2 antagonists</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">813</span><span class="NLM_x">–</span> <span class="NLM_lpage">822</span><span class="refDoi"> DOI: 10.1002/cmdc.201300445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref248/cit248&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fcmdc.201300445" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=813-822&author=M.+S.+Murgueitioauthor=P.+Hennekeauthor=H.+Glossmannauthor=S.+Santos-Sierraauthor=G.+Wolber&title=Prospective+virtual+screening+in+a+sparse+data+scenario%3A+design+of+small-molecule+TLR2+antagonists&doi=10.1002%2Fcmdc.201300445"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit248&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300445%26sid%3Dliteratum%253Aachs%26aulast%3DMurgueitio%26aufirst%3DM.%2BS.%26aulast%3DHenneke%26aufirst%3DP.%26aulast%3DGlossmann%26aufirst%3DH.%26aulast%3DSantos-Sierra%26aufirst%3DS.%26aulast%3DWolber%26aufirst%3DG.%26atitle%3DProspective%2520virtual%2520screening%2520in%2520a%2520sparse%2520data%2520scenario%253A%2520design%2520of%2520small-molecule%2520TLR2%2520antagonists%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D813%26epage%3D822%26doi%3D10.1002%2Fcmdc.201300445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref249"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref249'); return false;" data-citation="" class="refNumLink">249</a></strong><div class="NLM_citation" id="cit249"><span><span class="NLM_contrib-group">Stone, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parikh, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Rami, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ge, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellogg, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, P.</span><span> </span><span class="NLM_article-title">Identification of small-molecule inhibitors against meso-2, 6-diaminopimelate dehydrogenase from Porphyromonas gingivalis</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0141126</span><span class="refDoi"> DOI: 10.1371/journal.pone.0141126</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref249/cit249&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pone.0141126" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0141126&author=V.+N.+Stoneauthor=H.+I.+Parikhauthor=F.+El-Ramiauthor=X.+Geauthor=W.+Chenauthor=Y.+Zhangauthor=G.+E.+Kelloggauthor=P.+Xu&title=Identification+of+small-molecule+inhibitors+against+meso-2%2C+6-diaminopimelate+dehydrogenase+from+Porphyromonas+gingivalis&doi=10.1371%2Fjournal.pone.0141126"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit249&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0141126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0141126%26sid%3Dliteratum%253Aachs%26aulast%3DStone%26aufirst%3DV.%2BN.%26aulast%3DParikh%26aufirst%3DH.%2BI.%26aulast%3DEl-Rami%26aufirst%3DF.%26aulast%3DGe%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DKellogg%26aufirst%3DG.%2BE.%26aulast%3DXu%26aufirst%3DP.%26atitle%3DIdentification%2520of%2520small-molecule%2520inhibitors%2520against%2520meso-2%252C%25206-diaminopimelate%2520dehydrogenase%2520from%2520Porphyromonas%2520gingivalis%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0141126%26doi%3D10.1371%2Fjournal.pone.0141126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref250"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref250'); return false;" data-citation="" class="refNumLink">250</a></strong><div class="NLM_citation" id="cit250"><span><span class="NLM_contrib-group">Zhong, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chong, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, C. H.</span><span> </span><span class="NLM_article-title">Discovery of a natural product-like iNOS inhibitor by molecular docking with potential neuroprotective effects in vivo</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e92905</span><span class="refDoi"> DOI: 10.1371/journal.pone.0092905</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref250/cit250&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pone.0092905" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e92905&author=H.+J.+Zhongauthor=L.+J.+Liuauthor=C.+M.+Chongauthor=L.+Luauthor=M.+Wangauthor=D.+S.+Chanauthor=P.+W.+Chanauthor=S.+M.+Leeauthor=D.+L.+Maauthor=C.+H.+Leung&title=Discovery+of+a+natural+product-like+iNOS+inhibitor+by+molecular+docking+with+potential+neuroprotective+effects+in+vivo&doi=10.1371%2Fjournal.pone.0092905"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit250&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0092905&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0092905%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DH.%2BJ.%26aulast%3DLiu%26aufirst%3DL.%2BJ.%26aulast%3DChong%26aufirst%3DC.%2BM.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DD.%2BS.%26aulast%3DChan%26aufirst%3DP.%2BW.%26aulast%3DLee%26aufirst%3DS.%2BM.%26aulast%3DMa%26aufirst%3DD.%2BL.%26aulast%3DLeung%26aufirst%3DC.%2BH.%26atitle%3DDiscovery%2520of%2520a%2520natural%2520product-like%2520iNOS%2520inhibitor%2520by%2520molecular%2520docking%2520with%2520potential%2520neuroprotective%2520effects%2520in%2520vivo%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26spage%3De92905%26doi%3D10.1371%2Fjournal.pone.0092905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref251"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref251'); return false;" data-citation="" class="refNumLink">251</a></strong><div class="NLM_citation" id="cit251"><span><span class="NLM_contrib-group">Mpamhanga, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spinks, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tulloch, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shanks, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collie, I. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairlamb, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frearson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, W. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, I. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brenk, R.</span><span> </span><span class="NLM_article-title">One scaffold, three binding modes: novel and selective pteridine reductase 1 inhibitors derived from fragment hits discovered by virtual screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">4454</span><span class="NLM_x">–</span> <span class="NLM_lpage">4465</span><span class="refDoi"> DOI: 10.1021/jm900414x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm900414x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=4454-4465&author=C.+P.+Mpamhangaauthor=D.+Spinksauthor=L.+B.+Tullochauthor=E.+J.+Shanksauthor=D.+A.+Robinsonauthor=I.+T.+Collieauthor=A.+H.+Fairlambauthor=P.+G.+Wyattauthor=J.+A.+Frearsonauthor=W.+N.+Hunterauthor=I.+H.+Gilbertauthor=R.+Brenk&title=One+scaffold%2C+three+binding+modes%3A+novel+and+selective+pteridine+reductase+1+inhibitors+derived+from+fragment+hits+discovered+by+virtual+screening&doi=10.1021%2Fjm900414x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit251&amp;dbid=16384&amp;doi=10.1021%2Fjm900414x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900414x%26sid%3Dliteratum%253Aachs%26aulast%3DMpamhanga%26aufirst%3DC.%2BP.%26aulast%3DSpinks%26aufirst%3DD.%26aulast%3DTulloch%26aufirst%3DL.%2BB.%26aulast%3DShanks%26aufirst%3DE.%2BJ.%26aulast%3DRobinson%26aufirst%3DD.%2BA.%26aulast%3DCollie%26aufirst%3DI.%2BT.%26aulast%3DFairlamb%26aufirst%3DA.%2BH.%26aulast%3DWyatt%26aufirst%3DP.%2BG.%26aulast%3DFrearson%26aufirst%3DJ.%2BA.%26aulast%3DHunter%26aufirst%3DW.%2BN.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DBrenk%26aufirst%3DR.%26atitle%3DOne%2520scaffold%252C%2520three%2520binding%2520modes%253A%2520novel%2520and%2520selective%2520pteridine%2520reductase%25201%2520inhibitors%2520derived%2520from%2520fragment%2520hits%2520discovered%2520by%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D4454%26epage%3D4465%26doi%3D10.1021%2Fjm900414x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref252"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref252'); return false;" data-citation="" class="refNumLink">252</a></strong><div class="NLM_citation" id="cit252"><span><span class="NLM_contrib-group">Gschwend, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sirawaraporn, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, D. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, I. D.</span><span> </span><span class="NLM_article-title">Specificity in structure-based drug design: identification of a novel, selective inhibitor of Pneumocystis carinii dihydrofolate reductase</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">59</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.1002/(SICI)1097-0134(199709)29:1<59::AID-PROT4>3.0.CO;2-A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref252/cit252&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2F%28SICI%291097-0134%28199709%2929%3A1%3C59%3A%3AAID-PROT4%3E3.0.CO%3B2-A" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1997&pages=59-67&author=D.+A.+Gschwendauthor=W.+Sirawarapornauthor=D.+V.+Santiauthor=I.+D.+Kuntz&title=Specificity+in+structure-based+drug+design%3A+identification+of+a+novel%2C+selective+inhibitor+of+Pneumocystis+carinii+dihydrofolate+reductase&doi=10.1002%2F%28SICI%291097-0134%28199709%2929%3A1%3C59%3A%3AAID-PROT4%3E3.0.CO%3B2-A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit252&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-0134%28199709%2929%3A1%3C59%3A%3AAID-PROT4%3E3.0.CO%3B2-A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-0134%2528199709%252929%253A1%253C59%253A%253AAID-PROT4%253E3.0.CO%253B2-A%26sid%3Dliteratum%253Aachs%26aulast%3DGschwend%26aufirst%3DD.%2BA.%26aulast%3DSirawaraporn%26aufirst%3DW.%26aulast%3DSanti%26aufirst%3DD.%2BV.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26atitle%3DSpecificity%2520in%2520structure-based%2520drug%2520design%253A%2520identification%2520of%2520a%2520novel%252C%2520selective%2520inhibitor%2520of%2520Pneumocystis%2520carinii%2520dihydrofolate%2520reductase%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D1997%26volume%3D29%26spage%3D59%26epage%3D67%26doi%3D10.1002%2F%28SICI%291097-0134%28199709%2929%3A1%3C59%3A%3AAID-PROT4%3E3.0.CO%3B2-A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref253"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref253'); return false;" data-citation="" class="refNumLink">253</a></strong><div class="NLM_citation" id="cit253"><span><span class="NLM_contrib-group">Sun, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">App, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirth, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMahon, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, C.</span><span> </span><span class="NLM_article-title">Synthesis and biological evaluations of 3-substituted indolin-2-ones: a novel class of tyrosine kinase inhibitors that exhibit selectivity toward particular receptor tyrosine kinases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">2588</span><span class="NLM_x">–</span> <span class="NLM_lpage">2603</span><span class="refDoi"> DOI: 10.1021/jm980123i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm980123i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref253/cit253&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADyaK1cXkt12jtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=2588-2603&author=L.+Sunauthor=N.+Tranauthor=F.+Tangauthor=H.+Appauthor=P.+Hirthauthor=G.+McMahonauthor=C.+Tang&title=Synthesis+and+biological+evaluations+of+3-substituted+indolin-2-ones%3A+a+novel+class+of+tyrosine+kinase+inhibitors+that+exhibit+selectivity+toward+particular+receptor+tyrosine+kinases&doi=10.1021%2Fjm980123i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit253R"><div class="casContent"><span class="casTitleNuber">253</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Biological Evaluations of 3-Substituted Indolin-2-ones: A Novel Class of Tyrosine Kinase Inhibitors That Exhibit Selectivity toward Particular Receptor Tyrosine Kinases</span></div><div class="casAuthors">Sun, Li; Tran, Ngoc; Tang, Flora; App, Harald; Hirth, Peter; McMahon, Gerald; Tang, Cho</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2588-2603</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">3-Substituted indolin-2-ones have been designed and synthesized as a novel class of tyrosine kinase inhibitors which exhibit selectivity toward different receptor tyrosine kinases (RTKs).  These compds. have been evaluated for their relative inhibitory properties against a panel of RTKs in intact cells.  By modifying the 3-substituted indolin-2-ones, we have identified compds. which showed selective inhibition of the ligand-dependent autophosphorylation of various RTKs at submicromolar levels in cells.  Structure-activity anal. for these compds. and their relative potency and selectivity to inhibit particular RTKs has detd. that (1) 3-[(five-membered heteroaryl ring)methylidenyl]indolin-2-ones are highly specific against the VEGF (Flk-1) RTK activity, (2) 3-(substituted benzylidenyl)indolin-2-ones contg. bulky group(s) in the Ph ring at the C-3 position of indolin-2-ones showed high selectivity toward the EGF and Her-2 RTKs, and (3) the compd. contg. an extended side chain at the C-3 position of the indolin-2-one exhibited high potency and selectivity when tested against the PDGF and VEGF (Flk-1) RTKs.  Recent published crystallog. data for two of these 3-substituted indolin-2-ones provides a rationale to suggest that these compds. may bind in the ATP binding pocket of RTKs.  The structure-activity anal. supports the use of subsets of these compds. as specific chem. leads for the development of RTK-specific drugs with broad application for the treatment of human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJoPixq8Am5rVg90H21EOLACvtfcHk0lhSbeuYmiaYdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXkt12jtrw%253D&md5=536302b717170a630008dac14832204c</span></div><a href="/servlet/linkout?suffix=cit253&amp;dbid=16384&amp;doi=10.1021%2Fjm980123i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm980123i%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DL.%26aulast%3DTran%26aufirst%3DN.%26aulast%3DTang%26aufirst%3DF.%26aulast%3DApp%26aufirst%3DH.%26aulast%3DHirth%26aufirst%3DP.%26aulast%3DMcMahon%26aufirst%3DG.%26aulast%3DTang%26aufirst%3DC.%26atitle%3DSynthesis%2520and%2520biological%2520evaluations%2520of%25203-substituted%2520indolin-2-ones%253A%2520a%2520novel%2520class%2520of%2520tyrosine%2520kinase%2520inhibitors%2520that%2520exhibit%2520selectivity%2520toward%2520particular%2520receptor%2520tyrosine%2520kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D2588%26epage%3D2603%26doi%3D10.1021%2Fjm980123i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref254"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref254'); return false;" data-citation="" class="refNumLink">254</a></strong><div class="NLM_citation" id="cit254"><span><span class="NLM_contrib-group">Kobayashi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinjo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koseki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrow, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aoki, S.</span><span> </span><span class="NLM_article-title">Identification of novel potential antibiotics against Staphylococcus using structure-based drug screening targeting dihydrofolate reductase</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1242</span><span class="NLM_x">–</span> <span class="NLM_lpage">1253</span><span class="refDoi"> DOI: 10.1021/ci400686d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400686d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref254/cit254&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXks1yrsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1242-1253&author=M.+Kobayashiauthor=T.+Kinjoauthor=Y.+Kosekiauthor=C.+R.+Bourneauthor=W.+W.+Barrowauthor=S.+Aoki&title=Identification+of+novel+potential+antibiotics+against+Staphylococcus+using+structure-based+drug+screening+targeting+dihydrofolate+reductase&doi=10.1021%2Fci400686d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit254R"><div class="casContent"><span class="casTitleNuber">254</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel potential antibiotics against Staphylococcus using structure-based drug screening targeting dihydrofolate reductase</span></div><div class="casAuthors">Kobayashi, Maiko; Kinjo, Tomohiro; Koseki, Yuji; Bourne, Christina R.; Barrow, William W.; Aoki, Shunsuke</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1242-1253</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The emergence of multidrug-resistant Staphylococcus aureus (S. aureus) makes the treatment of infectious diseases in hospitals more difficult and increases the mortality of the patients.  In this study, we attempted to identify novel potent antibiotic candidate compds. against S. aureus dihydrofolate reductase (saDHFR).  The authors performed three-step in silico structure-based drug screening (SBDS) based on the crystal structure of saDHFR using a 154,118 chem. compd. library.  They subsequently evaluated whether candidate chem. compds. exhibited inhibitory effects on the growth of the model bacterium: Staphylococcus epidermidis.  Compd. KB1 showed a strong inhibitory effect on the growth of S. epidermidis.  The authors rescreened chem. structures similar to KB1 from a 461,397 chem. compd. library.  Three of the four KB1 analogs (KBS1, KBS3, and KBS4) showed inhibitory effects on the growth of S. epidermidis and enzyme inhibitory effects on saDHFR.  The authors performed structure-activity relationship (SAR) anal. of active chem. compds. and obsd. a correlative relationship among the IC50 values, interaction residues, and structure scaffolds.  In addn., the active chem. compds. (KB1, KBS3, and KBS4) had no inhibitory effects on the growth of model enterobacteria (Escherichia coli BL21 and JM109 strains) and no toxic effects on cultured mammalian cells (MDCK cells).  The results obtained from Protein Ligand Interaction Fingerprint (PLIF) and Ligand Interaction (LI) analyses suggested that all of the active compds. exhibited potential inhibitory effects on mutated saDHFR of the drug-resistant strains.  The structural and exptl. information concerning these novel chem. compds. will likely contribute to the development of new antibiotics for both wild-type and drug-resistant S. aureus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg4TaA44xcErVg90H21EOLACvtfcHk0lhSbeuYmiaYdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXks1yrsL8%253D&md5=a6da2186254d39cf533b4e3cae44bfa1</span></div><a href="/servlet/linkout?suffix=cit254&amp;dbid=16384&amp;doi=10.1021%2Fci400686d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400686d%26sid%3Dliteratum%253Aachs%26aulast%3DKobayashi%26aufirst%3DM.%26aulast%3DKinjo%26aufirst%3DT.%26aulast%3DKoseki%26aufirst%3DY.%26aulast%3DBourne%26aufirst%3DC.%2BR.%26aulast%3DBarrow%26aufirst%3DW.%2BW.%26aulast%3DAoki%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520novel%2520potential%2520antibiotics%2520against%2520Staphylococcus%2520using%2520structure-based%2520drug%2520screening%2520targeting%2520dihydrofolate%2520reductase%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1242%26epage%3D1253%26doi%3D10.1021%2Fci400686d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref255"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref255'); return false;" data-citation="" class="refNumLink">255</a></strong><div class="NLM_citation" id="cit255"><span><span class="NLM_contrib-group">McGovern, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Kinase inhibitors: not just for kinases anymore</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">1478</span><span class="NLM_x">–</span> <span class="NLM_lpage">1483</span><span class="refDoi"> DOI: 10.1021/jm020427b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020427b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref255/cit255&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhvFeksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=1478-1483&author=S.+L.+McGovernauthor=B.+K.+Shoichet&title=Kinase+inhibitors%3A+not+just+for+kinases+anymore&doi=10.1021%2Fjm020427b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit255R"><div class="casContent"><span class="casTitleNuber">255</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase Inhibitors: Not Just for Kinases Anymore</span></div><div class="casAuthors">McGovern, Susan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1478-1483</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Kinase inhibitors are widely employed as biol. reagents and as leads for drug design.  Their use is often complicated by their lack of specificity.  Although binding conserved ATP sites accounts for some of their nonspecificity, some compds. inhibit proteins not known to bind ATP.  It has been found that promiscuous hits from high-throughput screening may act as aggregates.  To explore whether this mechanism might explain the action of widely used nonspecific kinase inhibitors, 15 such compds. were studied.  Eight of these, rottlerin, quercetin, K-252c, bisindolylmaleimide I, bisindolylmaleimide IX, U0126, indirubin, and indigo, inhibited three diverse non-kinase enzymes.  Inhibition was time-dependent and sensitive to enzyme concn.; by light scattering, the compds. formed particles of 100-1000 nm diam.  These observations suggest that these eight kinase inhibitors, at least at micromolar concns., are promiscuous and act as aggregates.  Results obtained from the use of these compds. at micromolar or higher concns. against individual enzymes should be interpreted cautiously.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-JSJ7FPf7e7Vg90H21EOLACvtfcHk0lgm5bGS7NzsLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhvFeksLs%253D&md5=1d7e4e99a27be5ab279744aff92de26d</span></div><a href="/servlet/linkout?suffix=cit255&amp;dbid=16384&amp;doi=10.1021%2Fjm020427b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020427b%26sid%3Dliteratum%253Aachs%26aulast%3DMcGovern%26aufirst%3DS.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DKinase%2520inhibitors%253A%2520not%2520just%2520for%2520kinases%2520anymore%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D1478%26epage%3D1483%26doi%3D10.1021%2Fjm020427b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref256"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref256'); return false;" data-citation="" class="refNumLink">256</a></strong><div class="NLM_citation" id="cit256"><span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karpiak, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kroeze, W. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moy, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saddoris, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nikolova, V. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farrell, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangano, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deshpande, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Penn, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koller, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenakin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span> </span><span class="NLM_article-title">Allosteric ligands for the pharmacologically dark receptors GPR68 and GPR65</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">527</span><span class="NLM_x">, </span> <span class="NLM_fpage">477</span><span class="NLM_x">–</span> <span class="NLM_lpage">483</span><span class="refDoi"> DOI: 10.1038/nature15699</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref256/cit256&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnature15699" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref256/cit256&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26550826" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref256/cit256&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVWmsrjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=527&publication_year=2015&pages=477-483&author=X.+P.+Huangauthor=J.+Karpiakauthor=W.+K.+Kroezeauthor=H.+Zhuauthor=X.+Chenauthor=S.+S.+Moyauthor=K.+A.+Saddorisauthor=V.+D.+Nikolovaauthor=M.+S.+Farrellauthor=S.+Wangauthor=T.+J.+Manganoauthor=D.+A.+Deshpandeauthor=A.+Jiangauthor=R.+B.+Pennauthor=J.+Jinauthor=B.+H.+Kollerauthor=T.+Kenakinauthor=B.+K.+Shoichetauthor=B.+L.+Roth&title=Allosteric+ligands+for+the+pharmacologically+dark+receptors+GPR68+and+GPR65&doi=10.1038%2Fnature15699"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit256R"><div class="casContent"><span class="casTitleNuber">256</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric ligands for the pharmacologically dark receptors GPR68 and GPR65</span></div><div class="casAuthors">Huang, Xi-Ping; Karpiak, Joel; Kroeze, Wesley K.; Zhu, Hu; Chen, Xin; Moy, Sheryl S.; Saddoris, Kara A.; Nikolova, Viktoriya D.; Farrell, Martilias S.; Wang, Sheng; Mangano, Thomas J.; Deshpande, Deepak A.; Jiang, Alice; Penn, Raymond B.; Jin, Jian; Koller, Beverly H.; Kenakin, Terry; Shoichet, Brian K.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">527</span>
        (<span class="NLM_cas:issue">7579</span>),
    <span class="NLM_cas:pages">477-483</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">At least 120 non-olfactory G-protein-coupled receptors in the human genome are 'orphans' for which endogenous ligands are unknown, and many have no selective ligands, hindering the detn. of their biol. functions and clin. relevance.  Among these is GPR68, a proton receptor that lacks small mol. modulators for probing its biol.  Using yeast-based screens against GPR68, here the authors identify the benzodiazepine drug lorazepam as a non-selective GPR68 pos. allosteric modulator.  More than 3000 GPR68 homol. models were refined to recognize lorazepam in a putative allosteric site.  Docking 3.1 million mols. predicted new GPR68 modulators, many of which were confirmed in functional assays.  One potent GPR68 modulator, ogerin, suppressed recall in fear conditioning in wild-type but not in GPR68-knockout mice.  The same approach led to the discovery of allosteric agonists and neg. allosteric modulators for GPR65.  Combining phys. and structure-based screening may be broadly useful for ligand discovery for understudied and orphan GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDB_IiCoSwVrVg90H21EOLACvtfcHk0lgm5bGS7NzsLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVWmsrjJ&md5=a23d2d68c06b96cfbe7b53f2b17f3895</span></div><a href="/servlet/linkout?suffix=cit256&amp;dbid=16384&amp;doi=10.1038%2Fnature15699&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature15699%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DKarpiak%26aufirst%3DJ.%26aulast%3DKroeze%26aufirst%3DW.%2BK.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DMoy%26aufirst%3DS.%2BS.%26aulast%3DSaddoris%26aufirst%3DK.%2BA.%26aulast%3DNikolova%26aufirst%3DV.%2BD.%26aulast%3DFarrell%26aufirst%3DM.%2BS.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DMangano%26aufirst%3DT.%2BJ.%26aulast%3DDeshpande%26aufirst%3DD.%2BA.%26aulast%3DJiang%26aufirst%3DA.%26aulast%3DPenn%26aufirst%3DR.%2BB.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKoller%26aufirst%3DB.%2BH.%26aulast%3DKenakin%26aufirst%3DT.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DAllosteric%2520ligands%2520for%2520the%2520pharmacologically%2520dark%2520receptors%2520GPR68%2520and%2520GPR65%26jtitle%3DNature%26date%3D2015%26volume%3D527%26spage%3D477%26epage%3D483%26doi%3D10.1038%2Fnature15699" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107'],'ref108':['cit108'],'ref109':['cit109'],'ref110':['cit110'],'ref111':['cit111'],'ref112':['cit112'],'ref113':['cit113'],'ref114':['cit114'],'ref115':['cit115'],'ref116':['cit116'],'ref117':['cit117'],'ref118':['cit118'],'ref119':['cit119'],'ref120':['cit120'],'ref121':['cit121'],'ref122':['cit122'],'ref123':['cit123'],'ref124':['cit124'],'ref125':['cit125'],'ref126':['cit126'],'ref127':['cit127'],'ref128':['cit128'],'ref129':['cit129'],'ref130':['cit130'],'ref131':['cit131'],'ref132':['cit132'],'ref133':['cit133'],'ref134':['cit134'],'ref135':['cit135'],'ref136':['cit136'],'ref137':['cit137'],'ref138':['cit138'],'ref139':['cit139'],'ref140':['cit140'],'ref141':['cit141'],'ref142':['cit142'],'ref143':['cit143'],'ref144':['cit144'],'ref145':['cit145'],'ref146':['cit146'],'ref147':['cit147'],'ref148':['cit148'],'ref149':['cit149'],'ref150':['cit150'],'ref151':['cit151'],'ref152':['cit152'],'ref153':['cit153'],'ref154':['cit154'],'ref155':['cit155'],'ref156':['cit156'],'ref157':['cit157'],'ref158':['cit158'],'ref159':['cit159'],'ref160':['cit160'],'ref161':['cit161'],'ref162':['cit162'],'ref163':['cit163'],'ref164':['cit164'],'ref165':['cit165'],'ref166':['cit166'],'ref167':['cit167'],'ref168':['cit168'],'ref169':['cit169'],'ref170':['cit170'],'ref171':['cit171'],'ref172':['cit172'],'ref173':['cit173'],'ref174':['cit174'],'ref175':['cit175'],'ref176':['cit176'],'ref177':['cit177'],'ref178':['cit178'],'ref179':['cit179'],'ref180':['cit180'],'ref181':['cit181'],'ref182':['cit182'],'ref183':['cit183'],'ref184':['cit184'],'ref185':['cit185'],'ref186':['cit186'],'ref187':['cit187'],'ref188':['cit188'],'ref189':['cit189'],'ref190':['cit190'],'ref191':['cit191'],'ref192':['cit192'],'ref193':['cit193'],'ref194':['cit194'],'ref195':['cit195'],'ref196':['cit196'],'ref197':['cit197'],'ref198':['cit198'],'ref199':['cit199'],'ref200':['cit200'],'ref201':['cit201'],'ref202':['cit202'],'ref203':['cit203'],'ref204':['cit204'],'ref205':['cit205'],'ref206':['cit206'],'ref207':['cit207'],'ref208':['cit208'],'ref209':['cit209'],'ref210':['cit210'],'ref211':['cit211'],'ref212':['cit212'],'ref213':['cit213'],'ref214':['cit214'],'ref215':['cit215'],'ref216':['cit216'],'ref217':['cit217'],'ref218':['cit218'],'ref219':['cit219'],'ref220':['cit220'],'ref221':['cit221'],'ref222':['cit222'],'ref223':['cit223'],'ref224':['cit224'],'ref225':['cit225'],'ref226':['cit226'],'ref227':['cit227'],'ref228':['cit228'],'ref229':['cit229'],'ref230':['cit230'],'ref231':['cit231'],'ref232':['cit232'],'ref233':['cit233'],'ref234':['cit234'],'ref235':['cit235'],'ref236':['cit236'],'ref237':['cit237'],'ref238':['cit238'],'ref239':['cit239'],'ref240':['cit240'],'ref241':['cit241'],'ref242':['cit242'],'ref243':['cit243'],'ref244':['cit244'],'ref245':['cit245'],'ref246':['cit246'],'ref247':['cit247'],'ref248':['cit248'],'ref249':['cit249'],'ref250':['cit250'],'ref251':['cit251'],'ref252':['cit252'],'ref253':['cit253'],'ref254':['cit254'],'ref255':['cit255'],'ref256':['cit256']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 129 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Francois Berenger, Ashutosh Kumar, Kam Y. J. Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Yoshihiro Yamanishi</span>. </span><span class="cited-content_cbyCitation_article-title">Lean-Docking: Exploiting Ligands’ Predicted Docking Scores to Accelerate Molecular Docking. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (5)
                                     , 2341-2352. <a href="https://doi.org/10.1021/acs.jcim.0c01452" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c01452</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c01452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c01452%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DLean-Docking%25253A%252BExploiting%252BLigands%2525E2%252580%252599%252BPredicted%252BDocking%252BScores%252Bto%252BAccelerate%252BMolecular%252BDocking%26aulast%3DBerenger%26aufirst%3DFrancois%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D17122020%26date%3D16042021%26volume%3D61%26issue%3D5%26spage%3D2341%26epage%3D2352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Reed M. Stein, Ying Yang, Trent E. Balius, Matt J. O’Meara, Jiankun Lyu, Jennifer Young, Khanh Tang, Brian K. Shoichet, <span class="NLM_string-name hlFld-ContribAuthor">John J. Irwin</span>. </span><span class="cited-content_cbyCitation_article-title">Property-Unmatched Decoys in Docking Benchmarks. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (2)
                                     , 699-714. <a href="https://doi.org/10.1021/acs.jcim.0c00598" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00598</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00598%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DProperty-Unmatched%252BDecoys%252Bin%252BDocking%252BBenchmarks%26aulast%3DStein%26aufirst%3DReed%2BM.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D29052020%26date%3D25012021%26volume%3D61%26issue%3D2%26spage%3D699%26epage%3D714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hao Liang, Hongbo Liu, Yi Kuang, Limin Chen, Min Ye, <span class="NLM_string-name hlFld-ContribAuthor">Luhua Lai</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Targeted Covalent Natural Products against PLK1 by Herb-Based Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (9)
                                     , 4350-4358. <a href="https://doi.org/10.1021/acs.jcim.0c00074" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00074</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00074%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDiscovery%252Bof%252BTargeted%252BCovalent%252BNatural%252BProducts%252Bagainst%252BPLK1%252Bby%252BHerb-Based%252BScreening%26aulast%3DLiang%26aufirst%3DHao%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D22012020%26date%3D28052020%26date%3D14052020%26volume%3D60%26issue%3D9%26spage%3D4350%26epage%3D4358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mariia Radaeva, Xuesen Dong, <span class="NLM_string-name hlFld-ContribAuthor">Artem Cherkasov</span>. </span><span class="cited-content_cbyCitation_article-title">The Use of Methods of Computer-Aided Drug Discovery in the Development of Topoisomerase II Inhibitors: Applications and Future Directions. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (8)
                                     , 3703-3721. <a href="https://doi.org/10.1021/acs.jcim.0c00325" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00325</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00325&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00325%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DThe%252BUse%252Bof%252BMethods%252Bof%252BComputer-Aided%252BDrug%252BDiscovery%252Bin%252Bthe%252BDevelopment%252Bof%252BTopoisomerase%252BII%252BInhibitors%25253A%252BApplications%252Band%252BFuture%252BDirections%26aulast%3DRadaeva%26aufirst%3DMariia%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D03042020%26date%3D03082020%26date%3D20072020%26volume%3D60%26issue%3D8%26spage%3D3703%26epage%3D3721" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Daniel A. Greenfield, Hayden R. Schmidt, Meredith A. Skiba, Michael D. Mandler, Jacob R. Anderson, Piotr Sliz, <span class="NLM_string-name hlFld-ContribAuthor">Andrew C. Kruse</span>. </span><span class="cited-content_cbyCitation_article-title">Virtual Screening for Ligand Discovery at the σ1 Receptor. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (8)
                                     , 1555-1561. <a href="https://doi.org/10.1021/acsmedchemlett.9b00314" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00314</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00314%26sid%3Dliteratum%253Aachs%26jtitle%3DJ.%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D127%26issue%3D1%26spage%3D17%26epage%3D29%26jtitle%3DBehav.%2520Pharmacol.%26date%3D2016%26volume%3D27%26issue%3D2%25E2%2580%25933%2520Spec%2520Issue%26spage%3D100%26epage%3D15%26jtitle%3DHandb.%2520Exp.%2520Pharmacol.%26date%3D2017%26volume%3D244%26spage%3D237%26epage%3D308%26jtitle%3DNeural%2520Regener.%2520Res.%26date%3D2018%26volume%3D13%26issue%3D5%26spage%3D775%26epage%3D778%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DVirtual%252BScreening%252Bfor%252BLigand%252BDiscovery%252Bat%252Bthe%252B%2525CF%2525831%252BReceptor%26aulast%3DGreenfield%26aufirst%3DDaniel%2BA.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D20042020%26date%3D27072020%26date%3D31072020%26date%3D27072020%26volume%3D11%26issue%3D8%26spage%3D1555%26epage%3D1561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Flavio Ballante, Axel Rudling, Alexey Zeifman, Andreas Luttens, Duy Duc Vo, John J. Irwin, Jan Kihlberg, Jose Brea, Maria Isabel Loza, <span class="NLM_string-name hlFld-ContribAuthor">Jens Carlsson</span>. </span><span class="cited-content_cbyCitation_article-title">Docking Finds GPCR Ligands in Dark Chemical Matter. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (2)
                                     , 613-620. <a href="https://doi.org/10.1021/acs.jmedchem.9b01560" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01560</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01560%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDocking%252BFinds%252BGPCR%252BLigands%252Bin%252BDark%252BChemical%252BMatter%26aulast%3DBallante%26aufirst%3DFlavio%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D18092019%26date%3D13012020%26date%3D17122019%26volume%3D63%26issue%3D2%26spage%3D613%26epage%3D620" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xin-Qiu Yao, <span class="NLM_string-name hlFld-ContribAuthor">Donald Hamelberg</span>. </span><span class="cited-content_cbyCitation_article-title">Detecting Functional Dynamics in Proteins with Comparative Perturbed-Ensembles Analysis. </span><span class="cited-content_cbyCitation_journal-name">Accounts of Chemical Research</span><span> <strong>2019,</strong> <em>52 </em>
                                    (12)
                                     , 3455-3464. <a href="https://doi.org/10.1021/acs.accounts.9b00485" title="DOI URL">https://doi.org/10.1021/acs.accounts.9b00485</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.accounts.9b00485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.accounts.9b00485%26sid%3Dliteratum%253Aachs%26jtitle%3DAccounts%2520of%2520Chemical%2520Research%26atitle%3DDetecting%252BFunctional%252BDynamics%252Bin%252BProteins%252Bwith%252BComparative%252BPerturbed-Ensembles%252BAnalysis%26aulast%3DYao%26aufirst%3DXin-Qiu%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D12092019%26date%3D03122019%26volume%3D52%26issue%3D12%26spage%3D3455%26epage%3D3464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jianing Lu, Xuben Hou, Cheng Wang, <span class="NLM_string-name hlFld-ContribAuthor">Yingkai Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Incorporating Explicit Water Molecules and Ligand Conformation Stability in Machine-Learning Scoring Functions. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (11)
                                     , 4540-4549. <a href="https://doi.org/10.1021/acs.jcim.9b00645" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00645</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00645%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DIncorporating%252BExplicit%252BWater%252BMolecules%252Band%252BLigand%252BConformation%252BStability%252Bin%252BMachine-Learning%252BScoring%252BFunctions%26aulast%3DLu%26aufirst%3DJianing%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D01082019%26date%3D31102019%26date%3D22102019%26volume%3D59%26issue%3D11%26spage%3D4540%26epage%3D4549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">W. Patrick Walters, , <span class="NLM_string-name hlFld-ContribAuthor">Renxiao Wang</span>. </span><span class="cited-content_cbyCitation_article-title">New Trends in Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (9)
                                     , 3603-3604. <a href="https://doi.org/10.1021/acs.jcim.9b00728" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00728</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00728&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00728%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DNew%252BTrends%252Bin%252BVirtual%252BScreening%26aulast%3DWalters%26aufirst%3DW.%2BPatrick%26date%3D2019%26date%3D2019%26date%3D10092019%26volume%3D59%26issue%3D9%26spage%3D3603%26epage%3D3604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Kelly
L. Damm-Ganamet, Nidhi Arora, Stephane Becart, James P. Edwards, Alec D. Lebsack, Heather M. McAllister, Marina I. Nelen, Navin L. Rao, Lori Westover, John J. M. Wiener, <span class="NLM_string-name hlFld-ContribAuthor">Taraneh Mirzadegan</span>. </span><span class="cited-content_cbyCitation_article-title">Accelerating Lead Identification by High Throughput Virtual Screening: Prospective Case Studies from the Pharmaceutical Industry. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (5)
                                     , 2046-2062. <a href="https://doi.org/10.1021/acs.jcim.8b00941" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00941</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00941&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00941%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAccelerating%252BLead%252BIdentification%252Bby%252BHigh%252BThroughput%252BVirtual%252BScreening%25253A%252BProspective%252BCase%252BStudies%252Bfrom%252Bthe%252BPharmaceutical%252BIndustry%26aulast%3DDamm-Ganamet%26aufirst%3DKelly%2BL.%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D20122018%26date%3D13032019%26date%3D28022019%26volume%3D59%26issue%3D5%26spage%3D2046%26epage%3D2062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Natalie
J. Tatum, Fernanda Duarte, Shina C. L. Kamerlin, <span class="NLM_string-name hlFld-ContribAuthor">Ehmke Pohl</span>. </span><span class="cited-content_cbyCitation_article-title">Relative Binding Energies Predict Crystallographic Binding Modes of Ethionamide Booster Lead Compounds. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry Letters</span><span> <strong>2019,</strong> <em>10 </em>
                                    (9)
                                     , 2244-2249. <a href="https://doi.org/10.1021/acs.jpclett.9b00741" title="DOI URL">https://doi.org/10.1021/acs.jpclett.9b00741</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpclett.9b00741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpclett.9b00741%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520Letters%26atitle%3DRelative%252BBinding%252BEnergies%252BPredict%252BCrystallographic%252BBinding%252BModes%252Bof%252BEthionamide%252BBooster%252BLead%252BCompounds%26aulast%3DTatum%26aufirst%3DNatalie%2BJ.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D14032019%26date%3D09042019%26date%3D23042019%26date%3D09042019%26volume%3D10%26issue%3D9%26spage%3D2244%26epage%3D2249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Andrea Basciu, Giuliano Malloci, Fabio Pietrucci, Alexandre M. J.
J. Bonvin, <span class="NLM_string-name hlFld-ContribAuthor">Attilio V. Vargiu</span>. </span><span class="cited-content_cbyCitation_article-title">Holo-like and Druggable Protein Conformations from Enhanced Sampling of Binding Pocket Volume and Shape. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (4)
                                     , 1515-1528. <a href="https://doi.org/10.1021/acs.jcim.8b00730" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00730</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00730&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00730%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DHolo-like%252Band%252BDruggable%252BProtein%252BConformations%252Bfrom%252BEnhanced%252BSampling%252Bof%252BBinding%252BPocket%252BVolume%252Band%252BShape%26aulast%3DBasciu%26aufirst%3DAndrea%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D19102018%26date%3D28032019%26date%3D18032019%26volume%3D59%26issue%3D4%26spage%3D1515%26epage%3D1528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Youtian Hu, Lu Wang, Xu Han, Yueyang Zhou, Tonghui Zhang, Li Wang, Ting Hong, Wei Zhang, Xun-Xiang Guo, Jielin Sun, Yingxin Qi, Jing Yu, Hong Liu, <span class="NLM_string-name hlFld-ContribAuthor">Fang Wu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Bioactive Inhibitor with a New Scaffold for Cystathionine γ-Lyase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (3)
                                     , 1677-1683. <a href="https://doi.org/10.1021/acs.jmedchem.8b01720" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01720</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01720&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01720%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BBioactive%252BInhibitor%252Bwith%252Ba%252BNew%252BScaffold%252Bfor%252BCystathionine%252B%2525CE%2525B3-Lyase%26aulast%3DHu%26aufirst%3DYoutian%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D03112018%26date%3D31122018%26date%3D18122018%26volume%3D62%26issue%3D3%26spage%3D1677%26epage%3D1683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jiaye Guo, Stephen Collins, W. Todd Miller, <span class="NLM_string-name hlFld-ContribAuthor">Robert C. Rizzo</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of a Water-Coordinating HER2 Inhibitor by Virtual Screening Using Similarity-Based Scoring. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2018,</strong> <em>57 </em>
                                    (32)
                                     , 4934-4951. <a href="https://doi.org/10.1021/acs.biochem.8b00524" title="DOI URL">https://doi.org/10.1021/acs.biochem.8b00524</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.8b00524&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.8b00524%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DIdentification%252Bof%252Ba%252BWater-Coordinating%252BHER2%252BInhibitor%252Bby%252BVirtual%252BScreening%252BUsing%252BSimilarity-Based%252BScoring%26aulast%3DGuo%26aufirst%3DJiaye%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D07052018%26date%3D22062018%26date%3D30072018%26date%3D05072018%26volume%3D57%26issue%3D32%26spage%3D4934%26epage%3D4951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dahlia
R. Weiss, Joel Karpiak, Xi-Ping Huang, Maria F. Sassano, Jiankun Lyu, Bryan L. Roth, <span class="NLM_string-name hlFld-ContribAuthor">Brian K. Shoichet</span>. </span><span class="cited-content_cbyCitation_article-title">Selectivity Challenges in Docking Screens for GPCR Targets and Antitargets. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (15)
                                     , 6830-6845. <a href="https://doi.org/10.1021/acs.jmedchem.8b00718" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00718</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00718&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00718%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSelectivity%252BChallenges%252Bin%252BDocking%252BScreens%252Bfor%252BGPCR%252BTargets%252Band%252BAntitargets%26aulast%3DWeiss%26aufirst%3DDahlia%2BR.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D04052018%26date%3D24072018%26date%3D10072018%26volume%3D61%26issue%3D15%26spage%3D6830%26epage%3D6845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Andrey
V. Bogolyubsky, Olena Savych, Anton V. Zhemera, Sergey E. Pipko, Alexander V. Grishchenko, Anzhelika I. Konovets, Roman O. Doroshchuk, Dmytro N. Khomenko, Volodymyr S. Brovarets, Yurii S. Moroz, <span class="NLM_string-name hlFld-ContribAuthor">Mykhailo Vybornyi</span>. </span><span class="cited-content_cbyCitation_article-title">Facile One-Pot Parallel Synthesis of 3-Amino-1,2,4-triazoles. </span><span class="cited-content_cbyCitation_journal-name">ACS Combinatorial Science</span><span> <strong>2018,</strong> <em>20 </em>
                                    (7)
                                     , 461-466. <a href="https://doi.org/10.1021/acscombsci.8b00060" title="DOI URL">https://doi.org/10.1021/acscombsci.8b00060</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscombsci.8b00060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscombsci.8b00060%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Combinatorial%2520Science%26atitle%3DFacile%252BOne-Pot%252BParallel%252BSynthesis%252Bof%252B3-Amino-1%25252C2%25252C4-triazoles%26aulast%3DBogolyubsky%26aufirst%3DAndrey%2BV.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D17042018%26date%3D01062018%26date%3D18062018%26date%3D06062018%26volume%3D20%26issue%3D7%26spage%3D461%26epage%3D466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mariama Jaiteh, Alexey Zeifman, Marcus Saarinen, Per Svenningsson, Jose Bréa, Maria Isabel Loza, <span class="NLM_string-name hlFld-ContribAuthor">Jens Carlsson</span>. </span><span class="cited-content_cbyCitation_article-title">Docking Screens for Dual Inhibitors of Disparate Drug Targets for Parkinson’s Disease. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (12)
                                     , 5269-5278. <a href="https://doi.org/10.1021/acs.jmedchem.8b00204" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00204</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00204%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDocking%252BScreens%252Bfor%252BDual%252BInhibitors%252Bof%252BDisparate%252BDrug%252BTargets%252Bfor%252BParkinson%2525E2%252580%252599s%252BDisease%26aulast%3DJaiteh%26aufirst%3DMariama%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D06022018%26date%3D15062018%26date%3D24052018%26volume%3D61%26issue%3D12%26spage%3D5269%26epage%3D5278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ryo  Kunimoto</span> and <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>  . </span><span class="cited-content_cbyCitation_article-title">Combining Similarity Searching and Network Analysis for the Identification of Active Compounds. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2018,</strong> <em>3 </em>
                                    (4)
                                     , 3768-3777. <a href="https://doi.org/10.1021/acsomega.8b00344" title="DOI URL">https://doi.org/10.1021/acsomega.8b00344</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.8b00344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.8b00344%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DCombining%252BSimilarity%252BSearching%252Band%252BNetwork%252BAnalysis%252Bfor%252Bthe%252BIdentification%252Bof%252BActive%252BCompounds%26aulast%3DKunimoto%26aufirst%3DRyo%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D26022018%26date%3D22032018%26date%3D30042018%26volume%3D3%26issue%3D4%26spage%3D3768%26epage%3D3777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John J.  Irwin</span>, <span class="hlFld-ContribAuthor ">Garrett  Gaskins</span>, <span class="hlFld-ContribAuthor ">Teague  Sterling</span>, <span class="hlFld-ContribAuthor ">Michael M.  Mysinger</span>, and <span class="hlFld-ContribAuthor ">Michael J.  Keiser</span>  . </span><span class="cited-content_cbyCitation_article-title">Predicted Biological Activity of Purchasable Chemical Space. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (1)
                                     , 148-164. <a href="https://doi.org/10.1021/acs.jcim.7b00316" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00316</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00316%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPredicted%252BBiological%252BActivity%252Bof%252BPurchasable%252BChemical%252BSpace%26aulast%3DIrwin%26aufirst%3DJohn%2BJ.%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D29052017%26date%3D29122017%26date%3D22012018%26date%3D01122017%26volume%3D58%26issue%3D1%26spage%3D148%26epage%3D164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jean-Rémy  Marchand</span>, <span class="hlFld-ContribAuthor ">Andrea  Dalle Vedove</span>, <span class="hlFld-ContribAuthor ">Graziano  Lolli</span>, and <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of Inhibitors of Four Bromodomains by Fragment-Anchored Ligand Docking. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (10)
                                     , 2584-2597. <a href="https://doi.org/10.1021/acs.jcim.7b00336" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00336</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00336%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDiscovery%252Bof%252BInhibitors%252Bof%252BFour%252BBromodomains%252Bby%252BFragment-Anchored%252BLigand%252BDocking%26aulast%3DMarchand%26aufirst%3DJean-R%25C3%25A9my%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D06062017%26date%3D18092017%26date%3D23102017%26date%3D01092017%26volume%3D57%26issue%3D10%26spage%3D2584%26epage%3D2597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Haresh  Ajani</span>, <span class="hlFld-ContribAuthor ">Adam  Pecina</span>, <span class="hlFld-ContribAuthor ">Saltuk M.  Eyrilmez</span>, <span class="hlFld-ContribAuthor ">Jindřich  Fanfrlík</span>, <span class="hlFld-ContribAuthor ">Susanta  Haldar</span>, <span class="hlFld-ContribAuthor ">Jan  Řezáč</span>, <span class="hlFld-ContribAuthor ">Pavel  Hobza</span>, and <span class="hlFld-ContribAuthor ">Martin  Lepšík</span>  . </span><span class="cited-content_cbyCitation_article-title">Superior Performance of the SQM/COSMO Scoring Functions in Native Pose Recognition of Diverse Protein–Ligand Complexes in Cognate Docking. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2017,</strong> <em>2 </em>
                                    (7)
                                     , 4022-4029. <a href="https://doi.org/10.1021/acsomega.7b00503" title="DOI URL">https://doi.org/10.1021/acsomega.7b00503</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.7b00503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.7b00503%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DSuperior%252BPerformance%252Bof%252Bthe%252BSQM%25252FCOSMO%252BScoring%252BFunctions%252Bin%252BNative%252BPose%252BRecognition%252Bof%252BDiverse%252BProtein%2525E2%252580%252593Ligand%252BComplexes%252Bin%252BCognate%252BDocking%26aulast%3DAjani%26aufirst%3DHaresh%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D24042017%26date%3D18072017%26date%3D31072017%26volume%3D2%26issue%3D7%26spage%3D4022%26epage%3D4029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrew  Anighoro</span> and <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>  . </span><span class="cited-content_cbyCitation_article-title">Compound Ranking Based on Fuzzy Three-Dimensional Similarity Improves the Performance of Docking into Homology Models of G-Protein-Coupled Receptors. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2017,</strong> <em>2 </em>
                                    (6)
                                     , 2583-2592. <a href="https://doi.org/10.1021/acsomega.7b00330" title="DOI URL">https://doi.org/10.1021/acsomega.7b00330</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.7b00330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.7b00330%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DCompound%252BRanking%252BBased%252Bon%252BFuzzy%252BThree-Dimensional%252BSimilarity%252BImproves%252Bthe%252BPerformance%252Bof%252BDocking%252Binto%252BHomology%252BModels%252Bof%252BG-Protein-Coupled%252BReceptors%26aulast%3DAnighoro%26aufirst%3DAndrew%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D20032017%26date%3D31052017%26date%3D30062017%26volume%3D2%26issue%3D6%26spage%3D2583%26epage%3D2592" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fuqiang  Ban</span>, <span class="hlFld-ContribAuthor ">Kush  Dalal</span>, <span class="hlFld-ContribAuthor ">Huifang  Li</span>, <span class="hlFld-ContribAuthor ">Eric  LeBlanc</span>, <span class="hlFld-ContribAuthor ">Paul S.  Rennie</span>, and <span class="hlFld-ContribAuthor ">Artem  Cherkasov</span>  . </span><span class="cited-content_cbyCitation_article-title">Best Practices of Computer-Aided Drug Discovery: Lessons Learned from the Development of a Preclinical Candidate for Prostate Cancer with a New Mechanism of Action. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (5)
                                     , 1018-1028. <a href="https://doi.org/10.1021/acs.jcim.7b00137" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00137</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00137%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DBest%252BPractices%252Bof%252BComputer-Aided%252BDrug%252BDiscovery%25253A%252BLessons%252BLearned%252Bfrom%252Bthe%252BDevelopment%252Bof%252Ba%252BPreclinical%252BCandidate%252Bfor%252BProstate%252BCancer%252Bwith%252Ba%252BNew%252BMechanism%252Bof%252BAction%26aulast%3DBan%26aufirst%3DFuqiang%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D07032017%26date%3D04052017%26date%3D22052017%26date%3D25042017%26volume%3D57%26issue%3D5%26spage%3D1018%26epage%3D1028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Weilin  Zhang</span>, <span class="hlFld-ContribAuthor ">Jianfeng  Pei</span>, and <span class="hlFld-ContribAuthor ">Luhua  Lai</span>  . </span><span class="cited-content_cbyCitation_article-title">Computational Multitarget Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (3)
                                     , 403-412. <a href="https://doi.org/10.1021/acs.jcim.6b00491" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00491</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00491&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00491%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DComputational%252BMultitarget%252BDrug%252BDesign%26aulast%3DZhang%26aufirst%3DWeilin%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D19082016%26date%3D23022017%26date%3D27032017%26date%3D06022017%26volume%3D57%26issue%3D3%26spage%3D403%26epage%3D412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joshua  Pottel</span> and <span class="hlFld-ContribAuthor ">Nicolas  Moitessier</span>  . </span><span class="cited-content_cbyCitation_article-title">Customizable Generation of Synthetically Accessible, Local Chemical Subspaces. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (3)
                                     , 454-467. <a href="https://doi.org/10.1021/acs.jcim.6b00648" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00648</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00648%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DCustomizable%252BGeneration%252Bof%252BSynthetically%252BAccessible%25252C%252BLocal%252BChemical%252BSubspaces%26aulast%3DPottel%26aufirst%3DJoshua%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D24102016%26date%3D09032017%26date%3D27032017%26date%3D24022017%26volume%3D57%26issue%3D3%26spage%3D454%26epage%3D467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adam  Pecina</span>, <span class="hlFld-ContribAuthor ">Susanta  Haldar</span>, <span class="hlFld-ContribAuthor ">Jindřich  Fanfrlík</span>, <span class="hlFld-ContribAuthor ">René  Meier</span>, <span class="hlFld-ContribAuthor ">Jan  Řezáč</span>, <span class="hlFld-ContribAuthor ">Martin  Lepšík</span>, and <span class="hlFld-ContribAuthor ">Pavel  Hobza</span>  . </span><span class="cited-content_cbyCitation_article-title">SQM/COSMO Scoring Function at the DFTB3-D3H4 Level: Unique Identification of Native Protein–Ligand Poses. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (2)
                                     , 127-132. <a href="https://doi.org/10.1021/acs.jcim.6b00513" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00513</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00513&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00513%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DSQM%25252FCOSMO%252BScoring%252BFunction%252Bat%252Bthe%252BDFTB3-D3H4%252BLevel%25253A%252BUnique%252BIdentification%252Bof%252BNative%252BProtein%2525E2%252580%252593Ligand%252BPoses%26aulast%3DPecina%26aufirst%3DAdam%26date%3D2017%26date%3D2017%26date%3D03012017%26date%3D17012017%26date%3D27022017%26date%3D03012017%26volume%3D57%26issue%3D2%26spage%3D127%26epage%3D132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Manish  Sud</span>  . </span><span class="cited-content_cbyCitation_article-title">MayaChemTools: An Open Source Package for Computational Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2016,</strong> <em>56 </em>
                                    (12)
                                     , 2292-2297. <a href="https://doi.org/10.1021/acs.jcim.6b00505" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00505</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00505&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00505%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMayaChemTools%25253A%252BAn%252BOpen%252BSource%252BPackage%252Bfor%252BComputational%252BDrug%252BDiscovery%26aulast%3DSud%26aufirst%3DManish%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D27082016%26date%3D16112016%26date%3D27122016%26date%3D04112016%26volume%3D56%26issue%3D12%26spage%3D2292%26epage%3D2297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eelke B.  Lenselink</span>, <span class="hlFld-ContribAuthor ">Willem  Jespers</span>, <span class="hlFld-ContribAuthor ">Herman W. T.  van Vlijmen</span>, <span class="hlFld-ContribAuthor ">Adriaan P.  IJzerman</span>, and <span class="hlFld-ContribAuthor ">Gerard J. P.  van Westen</span>  . </span><span class="cited-content_cbyCitation_article-title">Interacting with GPCRs: Using Interaction Fingerprints for Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2016,</strong> <em>56 </em>
                                    (10)
                                     , 2053-2060. <a href="https://doi.org/10.1021/acs.jcim.6b00314" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00314</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00314%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DInteracting%252Bwith%252BGPCRs%25253A%252BUsing%252BInteraction%252BFingerprints%252Bfor%252BVirtual%252BScreening%26aulast%3DLenselink%26aufirst%3DEelke%2BB.%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D02062016%26date%3D27092016%26date%3D24102016%26date%3D14092016%26volume%3D56%26issue%3D10%26spage%3D2053%26epage%3D2060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrey  Tolmachev</span>, <span class="hlFld-ContribAuthor ">Andrey V.  Bogolubsky</span>, <span class="hlFld-ContribAuthor ">Sergey E.  Pipko</span>, <span class="hlFld-ContribAuthor ">Alexander V.  Grishchenko</span>, <span class="hlFld-ContribAuthor ">Dmytro V.  Ushakov</span>, <span class="hlFld-ContribAuthor ">Anton V.  Zhemera</span>, <span class="hlFld-ContribAuthor ">Oleksandr O.  Viniychuk</span>, <span class="hlFld-ContribAuthor ">Anzhelika I.  Konovets</span>, <span class="hlFld-ContribAuthor ">Olga A.  Zaporozhets</span>, <span class="hlFld-ContribAuthor ">Pavel K.  Mykhailiuk</span>, and <span class="hlFld-ContribAuthor ">Yurii S.  Moroz</span>  . </span><span class="cited-content_cbyCitation_article-title">Expanding Synthesizable Space of Disubstituted 1,2,4-Oxadiazoles. </span><span class="cited-content_cbyCitation_journal-name">ACS Combinatorial Science</span><span> <strong>2016,</strong> <em>18 </em>
                                    (10)
                                     , 616-624. <a href="https://doi.org/10.1021/acscombsci.6b00103" title="DOI URL">https://doi.org/10.1021/acscombsci.6b00103</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscombsci.6b00103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscombsci.6b00103%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Combinatorial%2520Science%26atitle%3DExpanding%252BSynthesizable%252BSpace%252Bof%252BDisubstituted%252B1%25252C2%25252C4-Oxadiazoles%26aulast%3DTolmachev%26aufirst%3DAndrey%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D12072016%26date%3D18082016%26date%3D07092016%26date%3D10102016%26date%3D22082016%26volume%3D18%26issue%3D10%26spage%3D616%26epage%3D624" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kelly L.  Damm-Ganamet</span>, <span class="hlFld-ContribAuthor ">Scott D.  Bembenek</span>, <span class="hlFld-ContribAuthor ">Jennifer W.  Venable</span>, <span class="hlFld-ContribAuthor ">Glenda G.  Castro</span>, <span class="hlFld-ContribAuthor ">Lieve  Mangelschots</span>, <span class="hlFld-ContribAuthor ">Daniëlle C. G.  Peeters</span>, <span class="hlFld-ContribAuthor ">Heather M.  Mcallister</span>, <span class="hlFld-ContribAuthor ">James P.  Edwards</span>, <span class="hlFld-ContribAuthor ">Daniel  Disepio</span>, and <span class="hlFld-ContribAuthor ">Taraneh  Mirzadegan</span>  . </span><span class="cited-content_cbyCitation_article-title">A Prospective Virtual Screening Study: Enriching Hit Rates and Designing Focus Libraries To Find Inhibitors of PI3Kδ and PI3Kγ. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (9)
                                     , 4302-4313. <a href="https://doi.org/10.1021/acs.jmedchem.5b01974" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01974</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01974%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252BProspective%252BVirtual%252BScreening%252BStudy%25253A%252BEnriching%252BHit%252BRates%252Band%252BDesigning%252BFocus%252BLibraries%252BTo%252BFind%252BInhibitors%252Bof%252BPI3K%2525CE%2525B4%252Band%252BPI3K%2525CE%2525B3%26aulast%3DDamm-Ganamet%26aufirst%3DKelly%2BL.%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D21122015%26date%3D28042016%26date%3D12052016%26date%3D04042016%26volume%3D59%26issue%3D9%26spage%3D4302%26epage%3D4313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ghita  Ghislat</span>, <span class="hlFld-ContribAuthor ">Taufiq  Rahman</span>, <span class="hlFld-ContribAuthor ">Pedro J.  Ballester</span>. </span><span class="cited-content_cbyCitation_article-title">Recent progress on the prospective application of machine learning to structure-based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2021,</strong> <em>65 </em>, 28-34. <a href="https://doi.org/10.1016/j.cbpa.2021.04.009" title="DOI URL">https://doi.org/10.1016/j.cbpa.2021.04.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2021.04.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2021.04.009%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DRecent%252Bprogress%252Bon%252Bthe%252Bprospective%252Bapplication%252Bof%252Bmachine%252Blearning%252Bto%252Bstructure-based%252Bvirtual%252Bscreening%26aulast%3DGhislat%26aufirst%3DGhita%26date%3D2021%26volume%3D65%26spage%3D28%26epage%3D34" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David E.  Graff</span>, <span class="hlFld-ContribAuthor ">Eugene I.  Shakhnovich</span>, <span class="hlFld-ContribAuthor ">Connor W.  Coley</span>. </span><span class="cited-content_cbyCitation_article-title">Accelerating high-throughput virtual screening through molecular pool-based active learning. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2021,</strong> <em>12 </em>
                                    (22)
                                     , 7866-7881. <a href="https://doi.org/10.1039/D0SC06805E" title="DOI URL">https://doi.org/10.1039/D0SC06805E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0SC06805E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0SC06805E%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DAccelerating%252Bhigh-throughput%252Bvirtual%252Bscreening%252Bthrough%252Bmolecular%252Bpool-based%252Bactive%252Blearning%26aulast%3DGraff%26aufirst%3DDavid%2BE.%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D22%26spage%3D7866%26epage%3D7881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Julio  Caballero</span>. </span><span class="cited-content_cbyCitation_article-title">The latest automated docking technologies for novel drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2021,</strong> <em>16 </em>
                                    (6)
                                     , 625-645. <a href="https://doi.org/10.1080/17460441.2021.1858793" title="DOI URL">https://doi.org/10.1080/17460441.2021.1858793</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2021.1858793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2021.1858793%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DThe%252Blatest%252Bautomated%252Bdocking%252Btechnologies%252Bfor%252Bnovel%252Bdrug%252Bdiscovery%26aulast%3DCaballero%26aufirst%3DJulio%26date%3D2021%26date%3D2020%26volume%3D16%26issue%3D6%26spage%3D625%26epage%3D645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jacopo  Manigrasso</span>, <span class="hlFld-ContribAuthor ">Marco  Marcia</span>, <span class="hlFld-ContribAuthor ">Marco  De Vivo</span>. </span><span class="cited-content_cbyCitation_article-title">Computer-aided design of RNA-targeted small molecules: A growing need in drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Chem</span><span> <strong>2021,</strong> <em>44 </em><a href="https://doi.org/10.1016/j.chempr.2021.05.021" title="DOI URL">https://doi.org/10.1016/j.chempr.2021.05.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chempr.2021.05.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chempr.2021.05.021%26sid%3Dliteratum%253Aachs%26jtitle%3DChem%26atitle%3DComputer-aided%252Bdesign%252Bof%252BRNA-targeted%252Bsmall%252Bmolecules%25253A%252BA%252Bgrowing%252Bneed%252Bin%252Bdrug%252Bdiscovery%26aulast%3DManigrasso%26aufirst%3DJacopo%26date%3D2021%26volume%3D44" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jon  Kapla</span>, <span class="hlFld-ContribAuthor ">Ismael  Rodríguez-Espigares</span>, <span class="hlFld-ContribAuthor ">Flavio  Ballante</span>, <span class="hlFld-ContribAuthor ">Jana  Selent</span>, <span class="hlFld-ContribAuthor ">Jens  Carlsson</span>, . </span><span class="cited-content_cbyCitation_article-title">Can molecular dynamics simulations improve the structural accuracy and virtual screening performance of GPCR models?. </span><span class="cited-content_cbyCitation_journal-name">PLOS Computational Biology</span><span> <strong>2021,</strong> <em>17 </em>
                                    (5)
                                     , e1008936. <a href="https://doi.org/10.1371/journal.pcbi.1008936" title="DOI URL">https://doi.org/10.1371/journal.pcbi.1008936</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pcbi.1008936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pcbi.1008936%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520Computational%2520Biology%26atitle%3DCan%252Bmolecular%252Bdynamics%252Bsimulations%252Bimprove%252Bthe%252Bstructural%252Baccuracy%252Band%252Bvirtual%252Bscreening%252Bperformance%252Bof%252BGPCR%252Bmodels%25253F%26aulast%3DKapla%26aufirst%3DJon%26date%3D2021%26date%3D2021%26volume%3D17%26issue%3D5%26spage%3De1008936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pinku  Halder</span>, <span class="hlFld-ContribAuthor ">Upamanyu  Pal</span>, <span class="hlFld-ContribAuthor ">Pranab  Paladhi</span>, <span class="hlFld-ContribAuthor ">Saurav  Dutta</span>, <span class="hlFld-ContribAuthor ">Pallab  Paul</span>, <span class="hlFld-ContribAuthor ">Samudra  Pal</span>, <span class="hlFld-ContribAuthor ">Debasmita  Das</span>, <span class="hlFld-ContribAuthor ">Agnish  Ganguly</span>, <span class="hlFld-ContribAuthor ">IshitaDutta</span>, <span class="hlFld-ContribAuthor ">SayarneelMandal</span>, <span class="hlFld-ContribAuthor ">Anirban  Ray</span>, <span class="hlFld-ContribAuthor ">Sujay  Ghosh</span>. </span><span class="cited-content_cbyCitation_article-title">Evaluation of potency of the selected bioactive molecules from Indian medicinal plants with MPro of SARS-CoV-2 through in silico analysis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Ayurveda and Integrative Medicine</span><span> <strong>2021,</strong> <em>426 </em><a href="https://doi.org/10.1016/j.jaim.2021.05.003" title="DOI URL">https://doi.org/10.1016/j.jaim.2021.05.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jaim.2021.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jaim.2021.05.003%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Ayurveda%2520and%2520Integrative%2520Medicine%26atitle%3DEvaluation%252Bof%252Bpotency%252Bof%252Bthe%252Bselected%252Bbioactive%252Bmolecules%252Bfrom%252BIndian%252Bmedicinal%252Bplants%252Bwith%252BMPro%252Bof%252BSARS-CoV-2%252Bthrough%252Bin%252Bsilico%252Banalysis%26aulast%3DHalder%26aufirst%3DPinku%26date%3D2021%26volume%3D426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Francesca  Spyrakis</span>, <span class="hlFld-ContribAuthor ">Aurijit  Sarkar</span>, <span class="hlFld-ContribAuthor ">Glen E.  Kellogg</span>. </span><span class="cited-content_cbyCitation_article-title">Docking, Scoring, and Virtual Screening in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 1-102. <a href="https://doi.org/10.1002/0471266949.bmc301" title="DOI URL">https://doi.org/10.1002/0471266949.bmc301</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/0471266949.bmc301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F0471266949.bmc301%26sid%3Dliteratum%253Aachs%26atitle%3DDocking%25252C%252BScoring%25252C%252Band%252BVirtual%252BScreening%252Bin%252BDrug%252BDiscovery%26aulast%3DSpyrakis%26aufirst%3DFrancesca%26date%3D2021%26date%3D2021%26spage%3D1%26epage%3D102%26pub%3DWiley%26atitle%3DBurger%252527s%252BMedicinal%252BChemistry%252Band%252BDrug%252BDiscovery%26aulast%3DAbraham%26aufirst%3DDonald%2BJ.%26date%3D2003%26date%3D2003%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David S.  Goodsell</span>, <span class="hlFld-ContribAuthor ">Michel F.  Sanner</span>, <span class="hlFld-ContribAuthor ">Arthur J.  Olson</span>, <span class="hlFld-ContribAuthor ">Stefano  Forli</span>. </span><span class="cited-content_cbyCitation_article-title">The
              AutoDock
              suite at 30. </span><span class="cited-content_cbyCitation_journal-name">Protein Science</span><span> <strong>2021,</strong> <em>30 </em>
                                    (1)
                                     , 31-43. <a href="https://doi.org/10.1002/pro.3934" title="DOI URL">https://doi.org/10.1002/pro.3934</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/pro.3934&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fpro.3934%26sid%3Dliteratum%253Aachs%26jtitle%3DProtein%2520Science%26atitle%3DThe%252BAutoDock%252Bsuite%252Bat%252B30%26aulast%3DGoodsell%26aufirst%3DDavid%2BS.%26date%3D2021%26date%3D2020%26volume%3D30%26issue%3D1%26spage%3D31%26epage%3D43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongjian  Li</span>, <span class="hlFld-ContribAuthor ">Kam‐Heung  Sze</span>, <span class="hlFld-ContribAuthor ">Gang  Lu</span>, <span class="hlFld-ContribAuthor ">Pedro J.  Ballester</span>. </span><span class="cited-content_cbyCitation_article-title">Machine‐learning scoring functions for structure‐based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">WIREs Computational Molecular Science</span><span> <strong>2021,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1002/wcms.1478" title="DOI URL">https://doi.org/10.1002/wcms.1478</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/wcms.1478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fwcms.1478%26sid%3Dliteratum%253Aachs%26jtitle%3DWIREs%2520Computational%2520Molecular%2520Science%26atitle%3DMachine%2525E2%252580%252590learning%252Bscoring%252Bfunctions%252Bfor%252Bstructure%2525E2%252580%252590based%252Bvirtual%252Bscreening%26aulast%3DLi%26aufirst%3DHongjian%26date%3D2021%26date%3D2020%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrea  Scarpino</span>, <span class="hlFld-ContribAuthor ">György G.  Ferenczy</span>, <span class="hlFld-ContribAuthor ">György M.  Keserű</span>. </span><span class="cited-content_cbyCitation_article-title">Binding Mode Prediction and Virtual Screening Applications by Covalent Docking. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 73-88. <a href="https://doi.org/10.1007/978-1-0716-1209-5_4" title="DOI URL">https://doi.org/10.1007/978-1-0716-1209-5_4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-0716-1209-5_4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-0716-1209-5_4%26sid%3Dliteratum%253Aachs%26atitle%3DBinding%252BMode%252BPrediction%252Band%252BVirtual%252BScreening%252BApplications%252Bby%252BCovalent%252BDocking%26aulast%3DScarpino%26aufirst%3DAndrea%26date%3D2021%26date%3D2021%26spage%3D73%26epage%3D88%26pub%3DSpringer%2520US%26atitle%3DProtein-Ligand%252BInteractions%252Band%252BDrug%252BDesign%26aulast%3DBallante%26aufirst%3DFlavio%26date%3D2021%26volume%3D2266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Agostino  Bruno</span>, <span class="hlFld-ContribAuthor ">Gabriele  Costantino</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular docking studies and virtual drug screening of chemosensitizers. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 169-183. <a href="https://doi.org/10.1016/B978-0-12-820701-7.00013-0" title="DOI URL">https://doi.org/10.1016/B978-0-12-820701-7.00013-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-820701-7.00013-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-820701-7.00013-0%26sid%3Dliteratum%253Aachs%26atitle%3DMolecular%252Bdocking%252Bstudies%252Band%252Bvirtual%252Bdrug%252Bscreening%252Bof%252Bchemosensitizers%26aulast%3DBruno%26aufirst%3DAgostino%26date%3D2021%26spage%3D169%26epage%3D183%26pub%3DElsevier%26atitle%3DpH-Interfering%252BAgents%252Bas%252BChemosensitizers%252Bin%252BCancer%252BTherapy%26date%3D2021%26volume%3D14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Helene J.  Bustad</span>, <span class="hlFld-ContribAuthor ">Juha P.  Kallio</span>, <span class="hlFld-ContribAuthor ">Marta  Vorland</span>, <span class="hlFld-ContribAuthor ">Valeria  Fiorentino</span>, <span class="hlFld-ContribAuthor ">Sverre  Sandberg</span>, <span class="hlFld-ContribAuthor ">Caroline  Schmitt</span>, <span class="hlFld-ContribAuthor ">Aasne K.  Aarsand</span>, <span class="hlFld-ContribAuthor ">Aurora  Martinez</span>. </span><span class="cited-content_cbyCitation_article-title">Acute Intermittent Porphyria: An Overview of Therapy Developments and Future Perspectives Focusing on Stabilisation of HMBS and Proteostasis Regulators. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (2)
                                     , 675. <a href="https://doi.org/10.3390/ijms22020675" title="DOI URL">https://doi.org/10.3390/ijms22020675</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22020675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22020675%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DAcute%252BIntermittent%252BPorphyria%25253A%252BAn%252BOverview%252Bof%252BTherapy%252BDevelopments%252Band%252BFuture%252BPerspectives%252BFocusing%252Bon%252BStabilisation%252Bof%252BHMBS%252Band%252BProteostasis%252BRegulators%26aulast%3DBustad%26aufirst%3DHelene%2BJ.%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D2%26spage%3D675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohammad Rejaur  Rahman</span>, <span class="hlFld-ContribAuthor ">Anik  Banik</span>, <span class="hlFld-ContribAuthor ">Ishtiak Malique  Chowdhury</span>, <span class="hlFld-ContribAuthor ">Emran Hossain  Sajib</span>, <span class="hlFld-ContribAuthor ">Sanchita  Sarkar</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of potential antivirals against SARS-CoV-2 using virtual screening method. </span><span class="cited-content_cbyCitation_journal-name">Informatics in Medicine Unlocked</span><span> <strong>2021,</strong> <em>23 </em>, 100531. <a href="https://doi.org/10.1016/j.imu.2021.100531" title="DOI URL">https://doi.org/10.1016/j.imu.2021.100531</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.imu.2021.100531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.imu.2021.100531%26sid%3Dliteratum%253Aachs%26jtitle%3DInformatics%2520in%2520Medicine%2520Unlocked%26atitle%3DIdentification%252Bof%252Bpotential%252Bantivirals%252Bagainst%252BSARS-CoV-2%252Busing%252Bvirtual%252Bscreening%252Bmethod%26aulast%3DRahman%26aufirst%3DMohammad%2BRejaur%26date%3D2021%26volume%3D23%26spage%3D100531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giulia  Bianco</span>, <span class="hlFld-ContribAuthor ">David S.  Goodsell</span>, <span class="hlFld-ContribAuthor ">Stefano  Forli</span>. </span><span class="cited-content_cbyCitation_article-title">Selective and Effective: Current Progress in Computational Structure-Based Drug Discovery of Targeted Covalent Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Trends in Pharmacological Sciences</span><span> <strong>2020,</strong> <em>41 </em>
                                    (12)
                                     , 1038-1049. <a href="https://doi.org/10.1016/j.tips.2020.10.005" title="DOI URL">https://doi.org/10.1016/j.tips.2020.10.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tips.2020.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tips.2020.10.005%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Pharmacological%2520Sciences%26atitle%3DSelective%252Band%252BEffective%25253A%252BCurrent%252BProgress%252Bin%252BComputational%252BStructure-Based%252BDrug%252BDiscovery%252Bof%252BTargeted%252BCovalent%252BInhibitors%26aulast%3DBianco%26aufirst%3DGiulia%26date%3D2020%26volume%3D41%26issue%3D12%26spage%3D1038%26epage%3D1049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adam  Pecina</span>, <span class="hlFld-ContribAuthor ">Saltuk M.  Eyrilmez</span>, <span class="hlFld-ContribAuthor ">Cemal  Köprülüoğlu</span>, <span class="hlFld-ContribAuthor ">Vijay Madhav  Miriyala</span>, <span class="hlFld-ContribAuthor ">Martin  Lepšík</span>, <span class="hlFld-ContribAuthor ">Jindřich  Fanfrlík</span>, <span class="hlFld-ContribAuthor ">Jan  Řezáč</span>, <span class="hlFld-ContribAuthor ">Pavel  Hobza</span>. </span><span class="cited-content_cbyCitation_article-title">SQM/COSMO Scoring Function: Reliable Quantum‐Mechanical Tool for Sampling and Ranking in Structure‐Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name">ChemPlusChem</span><span> <strong>2020,</strong> <em>85 </em>
                                    (11)
                                     , 2362-2371. <a href="https://doi.org/10.1002/cplu.202000120" title="DOI URL">https://doi.org/10.1002/cplu.202000120</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cplu.202000120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcplu.202000120%26sid%3Dliteratum%253Aachs%26jtitle%3DChemPlusChem%26atitle%3DSQM%25252FCOSMO%252BScoring%252BFunction%25253A%252BReliable%252BQuantum%2525E2%252580%252590Mechanical%252BTool%252Bfor%252BSampling%252Band%252BRanking%252Bin%252BStructure%2525E2%252580%252590Based%252BDrug%252BDesign%26aulast%3DPecina%26aufirst%3DAdam%26date%3D2020%26date%3D2020%26volume%3D85%26issue%3D11%26spage%3D2362%26epage%3D2371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yongbeom  Kwon</span>, <span class="hlFld-ContribAuthor ">Woong-Hee  Shin</span>, <span class="hlFld-ContribAuthor ">Junsu  Ko</span>, <span class="hlFld-ContribAuthor ">Juyong  Lee</span>. </span><span class="cited-content_cbyCitation_article-title">AK-Score: Accurate Protein-Ligand Binding Affinity Prediction Using an Ensemble of 3D-Convolutional Neural Networks. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (22)
                                     , 8424. <a href="https://doi.org/10.3390/ijms21228424" title="DOI URL">https://doi.org/10.3390/ijms21228424</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21228424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21228424%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DAK-Score%25253A%252BAccurate%252BProtein-Ligand%252BBinding%252BAffinity%252BPrediction%252BUsing%252Ban%252BEnsemble%252Bof%252B3D-Convolutional%252BNeural%252BNetworks%26aulast%3DKwon%26aufirst%3DYongbeom%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D22%26spage%3D8424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kishore Reddy Venkata  Thappeta</span>, <span class="hlFld-ContribAuthor ">Li Na  Zhao</span>, <span class="hlFld-ContribAuthor ">Choy Eng  Nge</span>, <span class="hlFld-ContribAuthor ">Sharon  Crasta</span>, <span class="hlFld-ContribAuthor ">Chung Yan  Leong</span>, <span class="hlFld-ContribAuthor ">Veronica  Ng</span>, <span class="hlFld-ContribAuthor ">Yoganathan  Kanagasundaram</span>, <span class="hlFld-ContribAuthor ">Hao  Fan</span>, <span class="hlFld-ContribAuthor ">Siew Bee  Ng</span>. </span><span class="cited-content_cbyCitation_article-title">In-Silico Identified New Natural Sortase A Inhibitors Disrupt S. aureus Biofilm Formation. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (22)
                                     , 8601. <a href="https://doi.org/10.3390/ijms21228601" title="DOI URL">https://doi.org/10.3390/ijms21228601</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21228601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21228601%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DIn-Silico%252BIdentified%252BNew%252BNatural%252BSortase%252BA%252BInhibitors%252BDisrupt%252BS.%252Baureus%252BBiofilm%252BFormation%26aulast%3DThappeta%26aufirst%3DKishore%2BReddy%2BVenkata%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D22%26spage%3D8601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ido  Rippin</span>, <span class="hlFld-ContribAuthor ">Netaly  Khazanov</span>, <span class="hlFld-ContribAuthor ">Shirley  Ben Joseph</span>, <span class="hlFld-ContribAuthor ">Tania  Kudinov</span>, <span class="hlFld-ContribAuthor ">Eva  Berent</span>, <span class="hlFld-ContribAuthor ">Sara Melisa  Arciniegas Ruiz</span>, <span class="hlFld-ContribAuthor ">Daniele  Marciano</span>, <span class="hlFld-ContribAuthor ">Laura  Levy</span>, <span class="hlFld-ContribAuthor ">Arie  Gruzman</span>, <span class="hlFld-ContribAuthor ">Hanoch  Senderowitz</span>, <span class="hlFld-ContribAuthor ">Hagit  Eldar-Finkelman</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and Design of Novel Small Molecule GSK-3 Inhibitors Targeting the Substrate Binding Site. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (22)
                                     , 8709. <a href="https://doi.org/10.3390/ijms21228709" title="DOI URL">https://doi.org/10.3390/ijms21228709</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21228709&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21228709%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DDiscovery%252Band%252BDesign%252Bof%252BNovel%252BSmall%252BMolecule%252BGSK-3%252BInhibitors%252BTargeting%252Bthe%252BSubstrate%252BBinding%252BSite%26aulast%3DRippin%26aufirst%3DIdo%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D22%26spage%3D8709" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stelios  Mylonas</span>, <span class="hlFld-ContribAuthor ">Apostolos  Axenopoulos</span>, <span class="hlFld-ContribAuthor ">Sotiris  Katsamakas</span>, <span class="hlFld-ContribAuthor ">Ioannis  Gkekas</span>, <span class="hlFld-ContribAuthor ">Kostas  Stamatopoulos</span>, <span class="hlFld-ContribAuthor ">Spyros  Petrakis</span>, <span class="hlFld-ContribAuthor ">Petros  Daras</span>. </span><span class="cited-content_cbyCitation_article-title">Deep learning-assisted pipeline for Virtual Screening of ligand compound databases: Application on inhibiting the entry of SARS-CoV-2 into human cells. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 132-139. <a href="https://doi.org/10.1109/BIBE50027.2020.00030" title="DOI URL">https://doi.org/10.1109/BIBE50027.2020.00030</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1109/BIBE50027.2020.00030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1109%2FBIBE50027.2020.00030%26sid%3Dliteratum%253Aachs%26atitle%3DDeep%252Blearning-assisted%252Bpipeline%252Bfor%252BVirtual%252BScreening%252Bof%252Bligand%252Bcompound%252Bdatabases%25253A%252BApplication%252Bon%252Binhibiting%252Bthe%252Bentry%252Bof%252BSARS-CoV-2%252Binto%252Bhuman%252Bcells%26aulast%3DMylonas%26aufirst%3DStelios%26date%3D2020%26spage%3D132%26epage%3D139%26pub%3DIEEE%26atitle%3D2020%252BIEEE%252B20th%252BInternational%252BConference%252Bon%252BBioinformatics%252Band%252BBioengineering%252B%252528BIBE%252529%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giuseppe  Marchetti</span>, <span class="hlFld-ContribAuthor ">Alessandro  Dessì</span>, <span class="hlFld-ContribAuthor ">Roberto  Dallocchio</span>, <span class="hlFld-ContribAuthor ">Ioannis  Tsamesidis</span>, <span class="hlFld-ContribAuthor ">Maria Carmina  Pau</span>, <span class="hlFld-ContribAuthor ">Francesco Michelangelo  Turrini</span>, <span class="hlFld-ContribAuthor ">Antonella  Pantaleo</span>. </span><span class="cited-content_cbyCitation_article-title">Syk Inhibitors: New Computational Insights into Their Intraerythrocytic Action in Plasmodium falciparum Malaria. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (19)
                                     , 7009. <a href="https://doi.org/10.3390/ijms21197009" title="DOI URL">https://doi.org/10.3390/ijms21197009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21197009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21197009%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DSyk%252BInhibitors%25253A%252BNew%252BComputational%252BInsights%252Binto%252BTheir%252BIntraerythrocytic%252BAction%252Bin%252BPlasmodium%252Bfalciparum%252BMalaria%26aulast%3DMarchetti%26aufirst%3DGiuseppe%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D19%26spage%3D7009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yusuf O.  Adeshina</span>, <span class="hlFld-ContribAuthor ">Eric J.  Deeds</span>, <span class="hlFld-ContribAuthor ">John  Karanicolas</span>. </span><span class="cited-content_cbyCitation_article-title">Machine learning classification can reduce false positives in structure-based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2020,</strong> <em>117 </em>
                                    (31)
                                     , 18477-18488. <a href="https://doi.org/10.1073/pnas.2000585117" title="DOI URL">https://doi.org/10.1073/pnas.2000585117</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.2000585117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.2000585117%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DMachine%252Blearning%252Bclassification%252Bcan%252Breduce%252Bfalse%252Bpositives%252Bin%252Bstructure-based%252Bvirtual%252Bscreening%26aulast%3DAdeshina%26aufirst%3DYusuf%2BO.%26date%3D2020%26date%3D2020%26volume%3D117%26issue%3D31%26spage%3D18477%26epage%3D18488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giulio  Poli</span>, <span class="hlFld-ContribAuthor ">Tiziano  Tuccinardi</span>. </span><span class="cited-content_cbyCitation_article-title">Consensus Docking in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Current Bioactive Compounds</span><span> <strong>2020,</strong> <em>16 </em>
                                    (3)
                                     , 182-190. <a href="https://doi.org/10.2174/1573407214666181023114820" title="DOI URL">https://doi.org/10.2174/1573407214666181023114820</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1573407214666181023114820&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1573407214666181023114820%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Bioactive%2520Compounds%26atitle%3DConsensus%252BDocking%252Bin%252BDrug%252BDiscovery%26aulast%3DPoli%26aufirst%3DGiulio%26date%3D2020%26volume%3D16%26issue%3D3%26spage%3D182%26epage%3D190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yaron  Ben Shoshan-Galeczki</span>, <span class="hlFld-ContribAuthor ">Masha Y.  Niv</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based screening for discovery of sweet compounds. </span><span class="cited-content_cbyCitation_journal-name">Food Chemistry</span><span> <strong>2020,</strong> <em>315 </em>, 126286. <a href="https://doi.org/10.1016/j.foodchem.2020.126286" title="DOI URL">https://doi.org/10.1016/j.foodchem.2020.126286</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.foodchem.2020.126286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.foodchem.2020.126286%26sid%3Dliteratum%253Aachs%26jtitle%3DFood%2520Chemistry%26atitle%3DStructure-based%252Bscreening%252Bfor%252Bdiscovery%252Bof%252Bsweet%252Bcompounds%26aulast%3DBen%2BShoshan-Galeczki%26aufirst%3DYaron%26date%3D2020%26volume%3D315%26spage%3D126286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alessandro  Dessì</span>, <span class="hlFld-ContribAuthor ">Paola  Peluso</span>, <span class="hlFld-ContribAuthor ">Roberto  Dallocchio</span>, <span class="hlFld-ContribAuthor ">Robin  Weiss</span>, <span class="hlFld-ContribAuthor ">Giuseppina  Andreotti</span>, <span class="hlFld-ContribAuthor ">Mariateresa  Allocca</span>, <span class="hlFld-ContribAuthor ">Emmanuel  Aubert</span>, <span class="hlFld-ContribAuthor ">Patrick  Pale</span>, <span class="hlFld-ContribAuthor ">Victor  Mamane</span>, <span class="hlFld-ContribAuthor ">Sergio  Cossu</span>. </span><span class="cited-content_cbyCitation_article-title">Rational Design, Synthesis, Characterization and Evaluation of Iodinated 4,4′-Bipyridines as New Transthyretin Fibrillogenesis Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (9)
                                     , 2213. <a href="https://doi.org/10.3390/molecules25092213" title="DOI URL">https://doi.org/10.3390/molecules25092213</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25092213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25092213%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DRational%252BDesign%25252C%252BSynthesis%25252C%252BCharacterization%252Band%252BEvaluation%252Bof%252BIodinated%252B4%25252C4%2525E2%252580%2525B2-Bipyridines%252Bas%252BNew%252BTransthyretin%252BFibrillogenesis%252BInhibitors%26aulast%3DDess%25C3%25AC%26aufirst%3DAlessandro%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D9%26spage%3D2213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vasiliki  Pardali</span>, <span class="hlFld-ContribAuthor ">Erofili  Giannakopoulou</span>, <span class="hlFld-ContribAuthor ">Dimitrios-Ilias  Balourdas</span>, <span class="hlFld-ContribAuthor ">Vassilios  Myrianthopoulos</span>, <span class="hlFld-ContribAuthor ">Martin C.  Taylor</span>, <span class="hlFld-ContribAuthor ">Marina  Šekutor</span>, <span class="hlFld-ContribAuthor ">Kata  Mlinarić-Majerski</span>, <span class="hlFld-ContribAuthor ">John M.  Kelly</span>, <span class="hlFld-ContribAuthor ">Grigoris  Zoidis</span>. </span><span class="cited-content_cbyCitation_article-title">Lipophilic Guanylhydrazone Analogues as Promising Trypanocidal Agents: An Extended SAR Study. </span><span class="cited-content_cbyCitation_journal-name">Current Pharmaceutical Design</span><span> <strong>2020,</strong> <em>26 </em>
                                    (8)
                                     , 838-866. <a href="https://doi.org/10.2174/1381612826666200210150127" title="DOI URL">https://doi.org/10.2174/1381612826666200210150127</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1381612826666200210150127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1381612826666200210150127%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Pharmaceutical%2520Design%26atitle%3DLipophilic%252BGuanylhydrazone%252BAnalogues%252Bas%252BPromising%252BTrypanocidal%252BAgents%25253A%252BAn%252BExtended%252BSAR%252BStudy%26aulast%3DPardali%26aufirst%3DVasiliki%26date%3D2020%26volume%3D26%26issue%3D8%26spage%3D838%26epage%3D866" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ningning  Fan</span>, <span class="hlFld-ContribAuthor ">Christoph A.  Bauer</span>, <span class="hlFld-ContribAuthor ">Conrad  Stork</span>, <span class="hlFld-ContribAuthor ">Christina  Bruyn Kops</span>, <span class="hlFld-ContribAuthor ">Johannes  Kirchmair</span>. </span><span class="cited-content_cbyCitation_article-title">ALADDIN: Docking Approach Augmented by Machine Learning for Protein Structure Selection Yields Superior Virtual Screening Performance. </span><span class="cited-content_cbyCitation_journal-name">Molecular Informatics</span><span> <strong>2020,</strong> <em>39 </em>
                                    (4)
                                     , 1900103. <a href="https://doi.org/10.1002/minf.201900103" title="DOI URL">https://doi.org/10.1002/minf.201900103</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/minf.201900103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fminf.201900103%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Informatics%26atitle%3DALADDIN%25253A%252BDocking%252BApproach%252BAugmented%252Bby%252BMachine%252BLearning%252Bfor%252BProtein%252BStructure%252BSelection%252BYields%252BSuperior%252BVirtual%252BScreening%252BPerformance%26aulast%3DFan%26aufirst%3DNingning%26date%3D2020%26volume%3D39%26issue%3D4%26spage%3D1900103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Reed M.  Stein</span>, <span class="hlFld-ContribAuthor ">Hye Jin  Kang</span>, <span class="hlFld-ContribAuthor ">John D.  McCorvy</span>, <span class="hlFld-ContribAuthor ">Grant C.  Glatfelter</span>, <span class="hlFld-ContribAuthor ">Anthony J.  Jones</span>, <span class="hlFld-ContribAuthor ">Tao  Che</span>, <span class="hlFld-ContribAuthor ">Samuel  Slocum</span>, <span class="hlFld-ContribAuthor ">Xi-Ping  Huang</span>, <span class="hlFld-ContribAuthor ">Olena  Savych</span>, <span class="hlFld-ContribAuthor ">Yurii S.  Moroz</span>, <span class="hlFld-ContribAuthor ">Benjamin  Stauch</span>, <span class="hlFld-ContribAuthor ">Linda C.  Johansson</span>, <span class="hlFld-ContribAuthor ">Vadim  Cherezov</span>, <span class="hlFld-ContribAuthor ">Terry  Kenakin</span>, <span class="hlFld-ContribAuthor ">John J.  Irwin</span>, <span class="hlFld-ContribAuthor ">Brian K.  Shoichet</span>, <span class="hlFld-ContribAuthor ">Bryan L.  Roth</span>, <span class="hlFld-ContribAuthor ">Margarita L.  Dubocovich</span>. </span><span class="cited-content_cbyCitation_article-title">Virtual discovery of melatonin receptor ligands to modulate circadian rhythms. </span><span class="cited-content_cbyCitation_journal-name">Nature</span><span> <strong>2020,</strong> <em>579 </em>
                                    (7800)
                                     , 609-614. <a href="https://doi.org/10.1038/s41586-020-2027-0" title="DOI URL">https://doi.org/10.1038/s41586-020-2027-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41586-020-2027-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41586-020-2027-0%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%26atitle%3DVirtual%252Bdiscovery%252Bof%252Bmelatonin%252Breceptor%252Bligands%252Bto%252Bmodulate%252Bcircadian%252Brhythms%26aulast%3DStein%26aufirst%3DReed%2BM.%26date%3D2020%26date%3D2020%26volume%3D579%26issue%3D7800%26spage%3D609%26epage%3D614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kelly L.  Damm-Ganamet</span>, <span class="hlFld-ContribAuthor ">Marie-Laure  Rives</span>, <span class="hlFld-ContribAuthor ">Alan D.  Wickenden</span>, <span class="hlFld-ContribAuthor ">Heather M.  McAllister</span>, <span class="hlFld-ContribAuthor ">Taraneh  Mirzadegan</span>. </span><span class="cited-content_cbyCitation_article-title">A computational approach yields selective inhibitors of human excitatory amino acid transporter 2 (EAAT2). </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2020,</strong> <em>295 </em>
                                    (13)
                                     , 4359-4366. <a href="https://doi.org/10.1074/jbc.AC119.011190" title="DOI URL">https://doi.org/10.1074/jbc.AC119.011190</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.AC119.011190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.AC119.011190%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DA%252Bcomputational%252Bapproach%252Byields%252Bselective%252Binhibitors%252Bof%252Bhuman%252Bexcitatory%252Bamino%252Bacid%252Btransporter%252B2%252B%252528EAAT2%252529%26aulast%3DDamm-Ganamet%26aufirst%3DKelly%2BL.%26date%3D2020%26volume%3D295%26issue%3D13%26spage%3D4359%26epage%3D4366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jincai  Yang</span>, <span class="hlFld-ContribAuthor ">Cheng  Shen</span>, <span class="hlFld-ContribAuthor ">Niu  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting or Pretending: Artificial Intelligence for Protein-Ligand Interactions Lack of Sufficiently Large and Unbiased Datasets. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2020,</strong> <em>11 </em><a href="https://doi.org/10.3389/fphar.2020.00069" title="DOI URL">https://doi.org/10.3389/fphar.2020.00069</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2020.00069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2020.00069%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DPredicting%252Bor%252BPretending%25253A%252BArtificial%252BIntelligence%252Bfor%252BProtein-Ligand%252BInteractions%252BLack%252Bof%252BSufficiently%252BLarge%252Band%252BUnbiased%252BDatasets%26aulast%3DYang%26aufirst%3DJincai%26date%3D2020%26date%3D2020%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Conor D.  Parks</span>, <span class="hlFld-ContribAuthor ">Zied  Gaieb</span>, <span class="hlFld-ContribAuthor ">Michael  Chiu</span>, <span class="hlFld-ContribAuthor ">Huanwang  Yang</span>, <span class="hlFld-ContribAuthor ">Chenghua  Shao</span>, <span class="hlFld-ContribAuthor ">W. Patrick  Walters</span>, <span class="hlFld-ContribAuthor ">Johanna M.  Jansen</span>, <span class="hlFld-ContribAuthor ">Georgia  McGaughey</span>, <span class="hlFld-ContribAuthor ">Richard A.  Lewis</span>, <span class="hlFld-ContribAuthor ">Scott D.  Bembenek</span>, <span class="hlFld-ContribAuthor ">Michael K.  Ameriks</span>, <span class="hlFld-ContribAuthor ">Tara  Mirzadegan</span>, <span class="hlFld-ContribAuthor ">Stephen K.  Burley</span>, <span class="hlFld-ContribAuthor ">Rommie E.  Amaro</span>, <span class="hlFld-ContribAuthor ">Michael K.  Gilson</span>. </span><span class="cited-content_cbyCitation_article-title">D3R grand challenge 4: blind prediction of protein–ligand poses, affinity rankings, and relative binding free energies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2020,</strong> <em>34 </em>
                                    (2)
                                     , 99-119. <a href="https://doi.org/10.1007/s10822-020-00289-y" title="DOI URL">https://doi.org/10.1007/s10822-020-00289-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-020-00289-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-020-00289-y%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DD3R%252Bgrand%252Bchallenge%252B4%25253A%252Bblind%252Bprediction%252Bof%252Bprotein%2525E2%252580%252593ligand%252Bposes%25252C%252Baffinity%252Brankings%25252C%252Band%252Brelative%252Bbinding%252Bfree%252Benergies%26aulast%3DParks%26aufirst%3DConor%2BD.%26date%3D2020%26date%3D2020%26volume%3D34%26issue%3D2%26spage%3D99%26epage%3D119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">P.  Srivastava</span>, <span class="hlFld-ContribAuthor ">P.  Kumar</span>, <span class="hlFld-ContribAuthor ">Anjani Kumar  Tiwari</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, and In Silico Evaluation of Methyl 2-(2-(5-Bromo/chloro-2-oxobenzoxazol-3(2H)-yl)-acetamido)-3-phenylpropanoate for TSPO Targeting. </span><span class="cited-content_cbyCitation_journal-name">Radiochemistry</span><span> <strong>2020,</strong> <em>62 </em>
                                    (1)
                                     , 107-118. <a href="https://doi.org/10.1134/S1066362220010142" title="DOI URL">https://doi.org/10.1134/S1066362220010142</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S1066362220010142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS1066362220010142%26sid%3Dliteratum%253Aachs%26jtitle%3DRadiochemistry%26atitle%3DDesign%25252C%252BSynthesis%25252C%252Band%252BIn%252BSilico%252BEvaluation%252Bof%252BMethyl%252B2-%2525282-%2525285-Bromo%25252Fchloro-2-oxobenzoxazol-3%2525282H%252529-yl%252529-acetamido%252529-3-phenylpropanoate%252Bfor%252BTSPO%252BTargeting%26aulast%3DSrivastava%26aufirst%3DP.%26date%3D2020%26date%3D2020%26volume%3D62%26issue%3D1%26spage%3D107%26epage%3D118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David K.  Brown</span>, <span class="hlFld-ContribAuthor ">Özlem  Tastan Bishop</span>. </span><span class="cited-content_cbyCitation_article-title">Role of Structural Bioinformatics in Drug Discovery by Computational SNP
                        Analysis. </span><span class="cited-content_cbyCitation_journal-name">Global Heart</span><span> <strong>2019,</strong> <em>12 </em>
                                    (2)
                                     , 151. <a href="https://doi.org/10.1016/j.gheart.2017.01.009" title="DOI URL">https://doi.org/10.1016/j.gheart.2017.01.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.gheart.2017.01.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.gheart.2017.01.009%26sid%3Dliteratum%253Aachs%26jtitle%3DGlobal%2520Heart%26atitle%3DRole%252Bof%252BStructural%252BBioinformatics%252Bin%252BDrug%252BDiscovery%252Bby%252BComputational%252BSNP%252BAnalysis%26aulast%3DBrown%26aufirst%3DDavid%2BK.%26date%3D2019%26date%3D2017%26volume%3D12%26issue%3D2%26spage%3D151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ashutosh  Kumar</span>, <span class="hlFld-ContribAuthor ">Kam Y. J.  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Improving ligand 3D shape similarity-based pose prediction with a continuum solvent model. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2019,</strong> <em>33 </em>
                                    (12)
                                     , 1045-1055. <a href="https://doi.org/10.1007/s10822-019-00220-0" title="DOI URL">https://doi.org/10.1007/s10822-019-00220-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-019-00220-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-019-00220-0%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DImproving%252Bligand%252B3D%252Bshape%252Bsimilarity-based%252Bpose%252Bprediction%252Bwith%252Ba%252Bcontinuum%252Bsolvent%252Bmodel%26aulast%3DKumar%26aufirst%3DAshutosh%26date%3D2019%26date%3D2019%26volume%3D33%26issue%3D12%26spage%3D1045%26epage%3D1055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vipul  Gupta</span>, <span class="hlFld-ContribAuthor ">Alina  Crudu</span>, <span class="hlFld-ContribAuthor ">Yukiko  Matsuoka</span>, <span class="hlFld-ContribAuthor ">Samik  Ghosh</span>, <span class="hlFld-ContribAuthor ">Roger  Rozot</span>, <span class="hlFld-ContribAuthor ">Xavier  Marat</span>, <span class="hlFld-ContribAuthor ">Sibylle  Jäger</span>, <span class="hlFld-ContribAuthor ">Hiroaki  Kitano</span>, <span class="hlFld-ContribAuthor ">Lionel  Breton</span>. </span><span class="cited-content_cbyCitation_article-title">Multi-dimensional computational pipeline for large-scale deep screening of compound effect assessment: an in silico case study on ageing-related compounds. </span><span class="cited-content_cbyCitation_journal-name">npj Systems Biology and Applications</span><span> <strong>2019,</strong> <em>5 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41540-019-0119-y" title="DOI URL">https://doi.org/10.1038/s41540-019-0119-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41540-019-0119-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41540-019-0119-y%26sid%3Dliteratum%253Aachs%26jtitle%3Dnpj%2520Systems%2520Biology%2520and%2520Applications%26atitle%3DMulti-dimensional%252Bcomputational%252Bpipeline%252Bfor%252Blarge-scale%252Bdeep%252Bscreening%252Bof%252Bcompound%252Beffect%252Bassessment%25253A%252Ban%252Bin%252Bsilico%252Bcase%252Bstudy%252Bon%252Bageing-related%252Bcompounds%26aulast%3DGupta%26aufirst%3DVipul%26date%3D2019%26date%3D2019%26volume%3D5%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sheisi F.L.  da Silva Rocha</span>, <span class="hlFld-ContribAuthor ">Carolina G.  Olanda</span>, <span class="hlFld-ContribAuthor ">Harold H.  Fokoue</span>, <span class="hlFld-ContribAuthor ">Carlos M.R.  Sant'Anna</span>. </span><span class="cited-content_cbyCitation_article-title">Virtual Screening Techniques in Drug Discovery: Review and Recent Applications. </span><span class="cited-content_cbyCitation_journal-name">Current Topics in Medicinal Chemistry</span><span> <strong>2019,</strong> <em>19 </em>
                                    (19)
                                     , 1751-1767. <a href="https://doi.org/10.2174/1568026619666190816101948" title="DOI URL">https://doi.org/10.2174/1568026619666190816101948</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1568026619666190816101948&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1568026619666190816101948%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Topics%2520in%2520Medicinal%2520Chemistry%26atitle%3DVirtual%252BScreening%252BTechniques%252Bin%252BDrug%252BDiscovery%25253A%252BReview%252Band%252BRecent%252BApplications%26aulast%3Dda%2BSilva%2BRocha%26aufirst%3DSheisi%2BF.L.%26date%3D2019%26volume%3D19%26issue%3D19%26spage%3D1751%26epage%3D1767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rachel-Ann A.  Garibsingh</span>, <span class="hlFld-ContribAuthor ">Avner  Schlessinger</span>. </span><span class="cited-content_cbyCitation_article-title">Advances and Challenges in Rational Drug Design for SLCs. </span><span class="cited-content_cbyCitation_journal-name">Trends in Pharmacological Sciences</span><span> <strong>2019,</strong> <em>40 </em>
                                    (10)
                                     , 790-800. <a href="https://doi.org/10.1016/j.tips.2019.08.006" title="DOI URL">https://doi.org/10.1016/j.tips.2019.08.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tips.2019.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tips.2019.08.006%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Pharmacological%2520Sciences%26atitle%3DAdvances%252Band%252BChallenges%252Bin%252BRational%252BDrug%252BDesign%252Bfor%252BSLCs%26aulast%3DGaribsingh%26aufirst%3DRachel-Ann%2BA.%26date%3D2019%26volume%3D40%26issue%3D10%26spage%3D790%26epage%3D800" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Concetta  Russo Spena</span>, <span class="hlFld-ContribAuthor ">Lucia  De Stefano</span>, <span class="hlFld-ContribAuthor ">Giulio  Poli</span>, <span class="hlFld-ContribAuthor ">Carlotta  Granchi</span>, <span class="hlFld-ContribAuthor ">Maguie  El Boustani</span>, <span class="hlFld-ContribAuthor ">Fabrizio  Ecca</span>, <span class="hlFld-ContribAuthor ">Gabriele  Grassi</span>, <span class="hlFld-ContribAuthor ">Mario  Grassi</span>, <span class="hlFld-ContribAuthor ">Vincenzo  Canzonieri</span>, <span class="hlFld-ContribAuthor ">Antonio  Giordano</span>, <span class="hlFld-ContribAuthor ">Tiziano  Tuccinardi</span>, <span class="hlFld-ContribAuthor ">Isabella  Caligiuri</span>, <span class="hlFld-ContribAuthor ">Flavio  Rizzolio</span>. </span><span class="cited-content_cbyCitation_article-title">Virtual screening identifies a PIN1 inhibitor with possible antiovarian cancer effects. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cellular Physiology</span><span> <strong>2019,</strong> <em>234 </em>
                                    (9)
                                     , 15708-15716. <a href="https://doi.org/10.1002/jcp.28224" title="DOI URL">https://doi.org/10.1002/jcp.28224</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jcp.28224&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjcp.28224%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cellular%2520Physiology%26atitle%3DVirtual%252Bscreening%252Bidentifies%252Ba%252BPIN1%252Binhibitor%252Bwith%252Bpossible%252Bantiovarian%252Bcancer%252Beffects%26aulast%3DRusso%2BSpena%26aufirst%3DConcetta%26date%3D2019%26date%3D2019%26volume%3D234%26issue%3D9%26spage%3D15708%26epage%3D15716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Courtney D.  Singleton</span>, <span class="hlFld-ContribAuthor ">Monica S.  Humby</span>, <span class="hlFld-ContribAuthor ">Hyun Ah  Yi</span>, <span class="hlFld-ContribAuthor ">Robert C.  Rizzo</span>, <span class="hlFld-ContribAuthor ">Amy  Jacobs</span>, . </span><span class="cited-content_cbyCitation_article-title">Identification of Ebola Virus Inhibitors Targeting GP2 Using Principles of Molecular Mimicry. </span><span class="cited-content_cbyCitation_journal-name">Journal of Virology</span><span> <strong>2019,</strong> <em>93 </em>
                                    (15)
                                     <a href="https://doi.org/10.1128/JVI.00676-19" title="DOI URL">https://doi.org/10.1128/JVI.00676-19</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1128/JVI.00676-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1128%2FJVI.00676-19%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Virology%26atitle%3DIdentification%252Bof%252BEbola%252BVirus%252BInhibitors%252BTargeting%252BGP2%252BUsing%252BPrinciples%252Bof%252BMolecular%252BMimicry%26aulast%3DSingleton%26aufirst%3DCourtney%2BD.%26date%3D2019%26volume%3D93%26issue%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giulio  Rastelli</span>, <span class="hlFld-ContribAuthor ">Luca  Pinzi</span>. </span><span class="cited-content_cbyCitation_article-title">Refinement and Rescoring of Virtual Screening Results. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2019,</strong> <em>7 </em><a href="https://doi.org/10.3389/fchem.2019.00498" title="DOI URL">https://doi.org/10.3389/fchem.2019.00498</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2019.00498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2019.00498%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DRefinement%252Band%252BRescoring%252Bof%252BVirtual%252BScreening%252BResults%26aulast%3DRastelli%26aufirst%3DGiulio%26date%3D2019%26date%3D2019%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yorley  Duarte</span>, <span class="hlFld-ContribAuthor ">Valeria  Márquez‐Miranda</span>, <span class="hlFld-ContribAuthor ">Matthieu J.  Miossec</span>, <span class="hlFld-ContribAuthor ">Fernando  González‐Nilo</span>. </span><span class="cited-content_cbyCitation_article-title">Integration of target discovery, drug discovery and drug delivery: A review on computational strategies. </span><span class="cited-content_cbyCitation_journal-name">Wiley Interdisciplinary Reviews: Nanomedicine and Nanobiotechnology</span><span> <strong>2019,</strong> <em>11 </em>
                                    (4)
                                     <a href="https://doi.org/10.1002/wnan.1554" title="DOI URL">https://doi.org/10.1002/wnan.1554</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/wnan.1554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fwnan.1554%26sid%3Dliteratum%253Aachs%26jtitle%3DWiley%2520Interdisciplinary%2520Reviews%253A%2520Nanomedicine%2520and%2520Nanobiotechnology%26atitle%3DIntegration%252Bof%252Btarget%252Bdiscovery%25252C%252Bdrug%252Bdiscovery%252Band%252Bdrug%252Bdelivery%25253A%252BA%252Breview%252Bon%252Bcomputational%252Bstrategies%26aulast%3DDuarte%26aufirst%3DYorley%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bryan L.  Roth</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular pharmacology of metabotropic receptors targeted by neuropsychiatric drugs. </span><span class="cited-content_cbyCitation_journal-name">Nature Structural & Molecular Biology</span><span> <strong>2019,</strong> <em>26 </em>
                                    (7)
                                     , 535-544. <a href="https://doi.org/10.1038/s41594-019-0252-8" title="DOI URL">https://doi.org/10.1038/s41594-019-0252-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41594-019-0252-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41594-019-0252-8%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Structural%2520%2526%2520Molecular%2520Biology%26atitle%3DMolecular%252Bpharmacology%252Bof%252Bmetabotropic%252Breceptors%252Btargeted%252Bby%252Bneuropsychiatric%252Bdrugs%26aulast%3DRoth%26aufirst%3DBryan%2BL.%26date%3D2019%26date%3D2019%26volume%3D26%26issue%3D7%26spage%3D535%26epage%3D544" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel  Reker</span>, <span class="hlFld-ContribAuthor ">Gonçalo J. L.  Bernardes</span>, <span class="hlFld-ContribAuthor ">Tiago  Rodrigues</span>. </span><span class="cited-content_cbyCitation_article-title">Computational advances in combating colloidal aggregation in drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Nature Chemistry</span><span> <strong>2019,</strong> <em>11 </em>
                                    (5)
                                     , 402-418. <a href="https://doi.org/10.1038/s41557-019-0234-9" title="DOI URL">https://doi.org/10.1038/s41557-019-0234-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41557-019-0234-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41557-019-0234-9%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Chemistry%26atitle%3DComputational%252Badvances%252Bin%252Bcombating%252Bcolloidal%252Baggregation%252Bin%252Bdrug%252Bdiscovery%26aulast%3DReker%26aufirst%3DDaniel%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D5%26spage%3D402%26epage%3D418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peter M. U.  Ung</span>, <span class="hlFld-ContribAuthor ">Masahiro  Sonoshita</span>, <span class="hlFld-ContribAuthor ">Alex P.  Scopton</span>, <span class="hlFld-ContribAuthor ">Arvin C.  Dar</span>, <span class="hlFld-ContribAuthor ">Ross L.  Cagan</span>, <span class="hlFld-ContribAuthor ">Avner  Schlessinger</span>, . </span><span class="cited-content_cbyCitation_article-title">Integrated computational and Drosophila cancer model platform captures previously unappreciated chemicals perturbing a kinase network. </span><span class="cited-content_cbyCitation_journal-name">PLOS Computational Biology</span><span> <strong>2019,</strong> <em>15 </em>
                                    (4)
                                     , e1006878. <a href="https://doi.org/10.1371/journal.pcbi.1006878" title="DOI URL">https://doi.org/10.1371/journal.pcbi.1006878</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pcbi.1006878&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pcbi.1006878%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520Computational%2520Biology%26atitle%3DIntegrated%252Bcomputational%252Band%252BDrosophila%252Bcancer%252Bmodel%252Bplatform%252Bcaptures%252Bpreviously%252Bunappreciated%252Bchemicals%252Bperturbing%252Ba%252Bkinase%252Bnetwork%26aulast%3DUng%26aufirst%3DPeter%2BM.%2BU.%26date%3D2019%26date%3D2019%26volume%3D15%26issue%3D4%26spage%3De1006878" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhihua  Wang</span>, <span class="hlFld-ContribAuthor ">Hao  Liang</span>, <span class="hlFld-ContribAuthor ">Haijie  Cao</span>, <span class="hlFld-ContribAuthor ">Bingjie  Zhang</span>, <span class="hlFld-ContribAuthor ">Jun  Li</span>, <span class="hlFld-ContribAuthor ">Wenqiong  Wang</span>, <span class="hlFld-ContribAuthor ">Shanshan  Qin</span>, <span class="hlFld-ContribAuthor ">Yuefei  Wang</span>, <span class="hlFld-ContribAuthor ">Lijiang  Xuan</span>, <span class="hlFld-ContribAuthor ">Luhua  Lai</span>, <span class="hlFld-ContribAuthor ">Wenqing  Shui</span>. </span><span class="cited-content_cbyCitation_article-title">Efficient ligand discovery from natural herbs by integrating virtual screening, affinity mass spectrometry and targeted metabolomics. </span><span class="cited-content_cbyCitation_journal-name">The Analyst</span><span> <strong>2019,</strong> <em>144 </em>
                                    (9)
                                     , 2881-2890. <a href="https://doi.org/10.1039/C8AN02482K" title="DOI URL">https://doi.org/10.1039/C8AN02482K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8AN02482K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8AN02482K%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Analyst%26atitle%3DEfficient%252Bligand%252Bdiscovery%252Bfrom%252Bnatural%252Bherbs%252Bby%252Bintegrating%252Bvirtual%252Bscreening%25252C%252Baffinity%252Bmass%252Bspectrometry%252Band%252Btargeted%252Bmetabolomics%26aulast%3DWang%26aufirst%3DZhihua%26date%3D2019%26date%3D2019%26volume%3D144%26issue%3D9%26spage%3D2881%26epage%3D2890" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Valerie J.  Gillet</span>. </span><span class="cited-content_cbyCitation_article-title">Applications of Chemoinformatics in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 17-36. <a href="https://doi.org/10.1002/9781119483977.ch2" title="DOI URL">https://doi.org/10.1002/9781119483977.ch2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9781119483977.ch2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9781119483977.ch2%26sid%3Dliteratum%253Aachs%26atitle%3DApplications%252Bof%252BChemoinformatics%252Bin%252BDrug%252BDiscovery%26aulast%3DGillet%26aufirst%3DValerie%2BJ.%26date%3D2019%26date%3D2019%26spage%3D17%26epage%3D36%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Ltd%26atitle%3DBiomolecular%252Band%252BBioanalytical%252BTechniques%26aulast%3DRamesh%26aufirst%3DVasudevan%26date%3D2019%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aleix  Gimeno</span>, <span class="hlFld-ContribAuthor ">María  Ojeda-Montes</span>, <span class="hlFld-ContribAuthor ">Sarah  Tomás-Hernández</span>, <span class="hlFld-ContribAuthor ">Adrià  Cereto-Massagué</span>, <span class="hlFld-ContribAuthor ">Raúl  Beltrán-Debón</span>, <span class="hlFld-ContribAuthor ">Miquel  Mulero</span>, <span class="hlFld-ContribAuthor ">Gerard  Pujadas</span>, <span class="hlFld-ContribAuthor ">Santiago  Garcia-Vallvé</span>. </span><span class="cited-content_cbyCitation_article-title">The Light and Dark Sides of Virtual Screening: What Is There to Know?. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2019,</strong> <em>20 </em>
                                    (6)
                                     , 1375. <a href="https://doi.org/10.3390/ijms20061375" title="DOI URL">https://doi.org/10.3390/ijms20061375</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms20061375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms20061375%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DThe%252BLight%252Band%252BDark%252BSides%252Bof%252BVirtual%252BScreening%25253A%252BWhat%252BIs%252BThere%252Bto%252BKnow%25253F%26aulast%3DGimeno%26aufirst%3DAleix%26date%3D2019%26date%3D2019%26volume%3D20%26issue%3D6%26spage%3D1375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiankun  Lyu</span>, <span class="hlFld-ContribAuthor ">Sheng  Wang</span>, <span class="hlFld-ContribAuthor ">Trent E.  Balius</span>, <span class="hlFld-ContribAuthor ">Isha  Singh</span>, <span class="hlFld-ContribAuthor ">Anat  Levit</span>, <span class="hlFld-ContribAuthor ">Yurii S.  Moroz</span>, <span class="hlFld-ContribAuthor ">Matthew J.  O’Meara</span>, <span class="hlFld-ContribAuthor ">Tao  Che</span>, <span class="hlFld-ContribAuthor ">Enkhjargal  Algaa</span>, <span class="hlFld-ContribAuthor ">Kateryna  Tolmachova</span>, <span class="hlFld-ContribAuthor ">Andrey A.  Tolmachev</span>, <span class="hlFld-ContribAuthor ">Brian K.  Shoichet</span>, <span class="hlFld-ContribAuthor ">Bryan L.  Roth</span>, <span class="hlFld-ContribAuthor ">John J.  Irwin</span>. </span><span class="cited-content_cbyCitation_article-title">Ultra-large library docking for discovering new chemotypes. </span><span class="cited-content_cbyCitation_journal-name">Nature</span><span> <strong>2019,</strong> <em>566 </em>
                                    (7743)
                                     , 224-229. <a href="https://doi.org/10.1038/s41586-019-0917-9" title="DOI URL">https://doi.org/10.1038/s41586-019-0917-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41586-019-0917-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41586-019-0917-9%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%26atitle%3DUltra-large%252Blibrary%252Bdocking%252Bfor%252Bdiscovering%252Bnew%252Bchemotypes%26aulast%3DLyu%26aufirst%3DJiankun%26date%3D2019%26date%3D2019%26volume%3D566%26issue%3D7743%26spage%3D224%26epage%3D229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexey  Aleksandrov</span>, <span class="hlFld-ContribAuthor ">Hannu  Myllykallio</span>. </span><span class="cited-content_cbyCitation_article-title">Advances and challenges in drug design against tuberculosis: application of in silico approaches. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2019,</strong> <em>14 </em>
                                    (1)
                                     , 35-46. <a href="https://doi.org/10.1080/17460441.2019.1550482" title="DOI URL">https://doi.org/10.1080/17460441.2019.1550482</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2019.1550482&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2019.1550482%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DAdvances%252Band%252Bchallenges%252Bin%252Bdrug%252Bdesign%252Bagainst%252Btuberculosis%25253A%252Bapplication%252Bof%252Bin%252Bsilico%252Bapproaches%26aulast%3DAleksandrov%26aufirst%3DAlexey%26date%3D2019%26date%3D2018%26volume%3D14%26issue%3D1%26spage%3D35%26epage%3D46" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Evgeny  Ivashko</span>, <span class="hlFld-ContribAuthor ">Natalia  Nikitina</span>. </span><span class="cited-content_cbyCitation_article-title">Use of a Desktop Grid to Effectively Discover Hits in Virtual Drug Screening. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 735-743. <a href="https://doi.org/10.1007/978-3-030-36592-9_60" title="DOI URL">https://doi.org/10.1007/978-3-030-36592-9_60</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-36592-9_60&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-36592-9_60%26sid%3Dliteratum%253Aachs%26atitle%3DUse%252Bof%252Ba%252BDesktop%252BGrid%252Bto%252BEffectively%252BDiscover%252BHits%252Bin%252BVirtual%252BDrug%252BScreening%26aulast%3DIvashko%26aufirst%3DEvgeny%26date%3D2019%26date%3D2019%26spage%3D735%26epage%3D743%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DSupercomputing%26aulast%3DVoevodin%26aufirst%3DVladimir%26date%3D2019%26volume%3D1129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zied  Gaieb</span>, <span class="hlFld-ContribAuthor ">Conor D.  Parks</span>, <span class="hlFld-ContribAuthor ">Michael  Chiu</span>, <span class="hlFld-ContribAuthor ">Huanwang  Yang</span>, <span class="hlFld-ContribAuthor ">Chenghua  Shao</span>, <span class="hlFld-ContribAuthor ">W. Patrick  Walters</span>, <span class="hlFld-ContribAuthor ">Millard H.  Lambert</span>, <span class="hlFld-ContribAuthor ">Neysa  Nevins</span>, <span class="hlFld-ContribAuthor ">Scott D.  Bembenek</span>, <span class="hlFld-ContribAuthor ">Michael K.  Ameriks</span>, <span class="hlFld-ContribAuthor ">Tara  Mirzadegan</span>, <span class="hlFld-ContribAuthor ">Stephen K.  Burley</span>, <span class="hlFld-ContribAuthor ">Rommie E.  Amaro</span>, <span class="hlFld-ContribAuthor ">Michael K.  Gilson</span>. </span><span class="cited-content_cbyCitation_article-title">D3R Grand Challenge 3: blind prediction of protein–ligand poses and affinity rankings. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2019,</strong> <em>33 </em>
                                    (1)
                                     , 1-18. <a href="https://doi.org/10.1007/s10822-018-0180-4" title="DOI URL">https://doi.org/10.1007/s10822-018-0180-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-018-0180-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-018-0180-4%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DD3R%252BGrand%252BChallenge%252B3%25253A%252Bblind%252Bprediction%252Bof%252Bprotein%2525E2%252580%252593ligand%252Bposes%252Band%252Baffinity%252Brankings%26aulast%3DGaieb%26aufirst%3DZied%26date%3D2019%26date%3D2019%26volume%3D33%26issue%3D1%26spage%3D1%26epage%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Luciano A.  Abriata</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Tools for Structural Analysis of Proteins. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 539-549. <a href="https://doi.org/10.1016/B978-0-12-809633-8.12366-0" title="DOI URL">https://doi.org/10.1016/B978-0-12-809633-8.12366-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-809633-8.12366-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-809633-8.12366-0%26sid%3Dliteratum%253Aachs%26atitle%3DComputational%252BTools%252Bfor%252BStructural%252BAnalysis%252Bof%252BProteins%26aulast%3DAbriata%26aufirst%3DLuciano%2BA.%26date%3D2019%26spage%3D539%26epage%3D549%26pub%3DElsevier%26atitle%3DEncyclopedia%252Bof%252BBioinformatics%252Band%252BComputational%252BBiology%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christina  Athanasiou</span>, <span class="hlFld-ContribAuthor ">Zoe  Cournia</span>. </span><span class="cited-content_cbyCitation_article-title">From Computers to Bedside: Computational Chemistry Contributing to FDA Approval. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 163-203. <a href="https://doi.org/10.1002/9783527806836.ch7" title="DOI URL">https://doi.org/10.1002/9783527806836.ch7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527806836.ch7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527806836.ch7%26sid%3Dliteratum%253Aachs%26atitle%3DFrom%252BComputers%252Bto%252BBedside%25253A%252BComputational%252BChemistry%252BContributing%252Bto%252BFDA%252BApproval%26aulast%3DAthanasiou%26aufirst%3DChristina%26date%3D2018%26date%3D2018%26spage%3D163%26epage%3D203%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DBiomolecular%252BSimulations%252Bin%252BStructure-Based%252BDrug%252BDiscovery%26aulast%3DGervasio%26aufirst%3DFrancesco%2BL.%26date%3D2018%26volume%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hyun-Hwi  Kim</span>, <span class="hlFld-ContribAuthor ">Ja-Shil  Hyun</span>, <span class="hlFld-ContribAuthor ">Joonhyeok  Choi</span>, <span class="hlFld-ContribAuthor ">Kwang-Eun  Choi</span>, <span class="hlFld-ContribAuthor ">Jun-Goo  Jee</span>, <span class="hlFld-ContribAuthor ">Sung Jean  Park</span>. </span><span class="cited-content_cbyCitation_article-title">Structural ensemble-based docking simulation and biophysical studies discovered new inhibitors of Hsp90 N-terminal domain. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2018,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-017-18332-8" title="DOI URL">https://doi.org/10.1038/s41598-017-18332-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-017-18332-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-017-18332-8%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DStructural%252Bensemble-based%252Bdocking%252Bsimulation%252Band%252Bbiophysical%252Bstudies%252Bdiscovered%252Bnew%252Binhibitors%252Bof%252BHsp90%252BN-terminal%252Bdomain%26aulast%3DKim%26aufirst%3DHyun-Hwi%26date%3D2018%26date%3D2018%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xuyang  Liu</span>, <span class="hlFld-ContribAuthor ">Zhan-Guo  Gao</span>, <span class="hlFld-ContribAuthor ">Yiran  Wu</span>, <span class="hlFld-ContribAuthor ">Raymond C.  Stevens</span>, <span class="hlFld-ContribAuthor ">Kenneth A.  Jacobson</span>, <span class="hlFld-ContribAuthor ">Suwen  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Salvianolic acids from antithrombotic Traditional Chinese Medicine Danshen are antagonists of human P2Y1 and P2Y12 receptors. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2018,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-018-26577-0" title="DOI URL">https://doi.org/10.1038/s41598-018-26577-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-018-26577-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-018-26577-0%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DSalvianolic%252Bacids%252Bfrom%252Bantithrombotic%252BTraditional%252BChinese%252BMedicine%252BDanshen%252Bare%252Bantagonists%252Bof%252Bhuman%252BP2Y1%252Band%252BP2Y12%252Breceptors%26aulast%3DLiu%26aufirst%3DXuyang%26date%3D2018%26date%3D2018%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Avner  Schlessinger</span>, <span class="hlFld-ContribAuthor ">Matthew A.  Welch</span>, <span class="hlFld-ContribAuthor ">Herman  van Vlijmen</span>, <span class="hlFld-ContribAuthor ">Ken  Korzekwa</span>, <span class="hlFld-ContribAuthor ">Peter W.  Swaan</span>, <span class="hlFld-ContribAuthor ">Pär  Matsson</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Modeling of Drug-Transporter Interactions-An International Transporter Consortium Perspective. </span><span class="cited-content_cbyCitation_journal-name">Clinical Pharmacology & Therapeutics</span><span> <strong>2018,</strong> <em>104 </em>
                                    (5)
                                     , 818-835. <a href="https://doi.org/10.1002/cpt.1174" title="DOI URL">https://doi.org/10.1002/cpt.1174</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cpt.1174&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcpt.1174%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Pharmacology%2520%2526%2520Therapeutics%26atitle%3DMolecular%252BModeling%252Bof%252BDrug-Transporter%252BInteractions-An%252BInternational%252BTransporter%252BConsortium%252BPerspective%26aulast%3DSchlessinger%26aufirst%3DAvner%26date%3D2018%26date%3D2018%26volume%3D104%26issue%3D5%26spage%3D818%26epage%3D835" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">L.  Dettori</span>, <span class="hlFld-ContribAuthor ">C.  Jelsch</span>, <span class="hlFld-ContribAuthor ">Y.  Guiavarc’h</span>, <span class="hlFld-ContribAuthor ">S.  Delaunay</span>, <span class="hlFld-ContribAuthor ">X.  Framboisier</span>, <span class="hlFld-ContribAuthor ">I.  Chevalot</span>, <span class="hlFld-ContribAuthor ">C.  Humeau</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular rules for selectivity in lipase-catalysed acylation of lysine. </span><span class="cited-content_cbyCitation_journal-name">Process Biochemistry</span><span> <strong>2018,</strong> <em>74 </em>, 50-60. <a href="https://doi.org/10.1016/j.procbio.2018.07.021" title="DOI URL">https://doi.org/10.1016/j.procbio.2018.07.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.procbio.2018.07.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.procbio.2018.07.021%26sid%3Dliteratum%253Aachs%26jtitle%3DProcess%2520Biochemistry%26atitle%3DMolecular%252Brules%252Bfor%252Bselectivity%252Bin%252Blipase-catalysed%252Bacylation%252Bof%252Blysine%26aulast%3DDettori%26aufirst%3DL.%26date%3D2018%26volume%3D74%26spage%3D50%26epage%3D60" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jia  Wang</span>, <span class="hlFld-ContribAuthor ">Grace Qun  Gong</span>, <span class="hlFld-ContribAuthor ">Yan  Zhou</span>, <span class="hlFld-ContribAuthor ">Woo-Jeong  Lee</span>, <span class="hlFld-ContribAuthor ">Christina Maree  Buchanan</span>, <span class="hlFld-ContribAuthor ">William Alexander  Denny</span>, <span class="hlFld-ContribAuthor ">Gordon William  Rewcastle</span>, <span class="hlFld-ContribAuthor ">Jackie Diane  Kendall</span>, <span class="hlFld-ContribAuthor ">James Michael Jeremy  Dickson</span>, <span class="hlFld-ContribAuthor ">Jack Urquhart  Flanagan</span>, <span class="hlFld-ContribAuthor ">Peter Robin  Shepherd</span>, <span class="hlFld-ContribAuthor ">De-Hua  Yang</span>, <span class="hlFld-ContribAuthor ">Ming-Wei  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">High-throughput screening campaigns against a PI3Kα isoform bearing the H1047R mutation identified potential inhibitors with novel scaffolds. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmacologica Sinica</span><span> <strong>2018,</strong> <em>39 </em>
                                    (11)
                                     , 1816-1822. <a href="https://doi.org/10.1038/s41401-018-0057-z" title="DOI URL">https://doi.org/10.1038/s41401-018-0057-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41401-018-0057-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41401-018-0057-z%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmacologica%2520Sinica%26atitle%3DHigh-throughput%252Bscreening%252Bcampaigns%252Bagainst%252Ba%252BPI3K%2525CE%2525B1%252Bisoform%252Bbearing%252Bthe%252BH1047R%252Bmutation%252Bidentified%252Bpotential%252Binhibitors%252Bwith%252Bnovel%252Bscaffolds%26aulast%3DWang%26aufirst%3DJia%26date%3D2018%26date%3D2018%26volume%3D39%26issue%3D11%26spage%3D1816%26epage%3D1822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chao  Shen</span>, <span class="hlFld-ContribAuthor ">Zhe  Wang</span>, <span class="hlFld-ContribAuthor ">Xiaojun  Yao</span>, <span class="hlFld-ContribAuthor ">Youyong  Li</span>, <span class="hlFld-ContribAuthor ">Tailong  Lei</span>, <span class="hlFld-ContribAuthor ">Ercheng  Wang</span>, <span class="hlFld-ContribAuthor ">Lei  Xu</span>, <span class="hlFld-ContribAuthor ">Feng  Zhu</span>, <span class="hlFld-ContribAuthor ">Dan  Li</span>, <span class="hlFld-ContribAuthor ">Tingjun  Hou</span>. </span><span class="cited-content_cbyCitation_article-title">Comprehensive assessment of nine docking programs on type II kinase inhibitors: prediction accuracy of sampling power, scoring power and screening power. </span><span class="cited-content_cbyCitation_journal-name">Briefings in Bioinformatics</span><span> <strong>2018,</strong> <em>17 </em><a href="https://doi.org/10.1093/bib/bby103" title="DOI URL">https://doi.org/10.1093/bib/bby103</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bib/bby103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbib%2Fbby103%26sid%3Dliteratum%253Aachs%26jtitle%3DBriefings%2520in%2520Bioinformatics%26atitle%3DComprehensive%252Bassessment%252Bof%252Bnine%252Bdocking%252Bprograms%252Bon%252Btype%252BII%252Bkinase%252Binhibitors%25253A%252Bprediction%252Baccuracy%252Bof%252Bsampling%252Bpower%25252C%252Bscoring%252Bpower%252Band%252Bscreening%252Bpower%26aulast%3DShen%26aufirst%3DChao%26date%3D2018%26date%3D2018%26volume%3D17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Meenakshi  Gupta</span>, <span class="hlFld-ContribAuthor ">Ruchika  Sharma</span>, <span class="hlFld-ContribAuthor ">Anoop  Kumar</span>. </span><span class="cited-content_cbyCitation_article-title">Docking techniques in pharmacology: How much promising?. </span><span class="cited-content_cbyCitation_journal-name">Computational Biology and Chemistry</span><span> <strong>2018,</strong> <em>76 </em>, 210-217. <a href="https://doi.org/10.1016/j.compbiolchem.2018.06.005" title="DOI URL">https://doi.org/10.1016/j.compbiolchem.2018.06.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.compbiolchem.2018.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.compbiolchem.2018.06.005%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520Biology%2520and%2520Chemistry%26atitle%3DDocking%252Btechniques%252Bin%252Bpharmacology%25253A%252BHow%252Bmuch%252Bpromising%25253F%26aulast%3DGupta%26aufirst%3DMeenakshi%26date%3D2018%26volume%3D76%26spage%3D210%26epage%3D217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christoph  Sotriffer</span>. </span><span class="cited-content_cbyCitation_article-title">Docking of Covalent Ligands: Challenges and Approaches. </span><span class="cited-content_cbyCitation_journal-name">Molecular Informatics</span><span> <strong>2018,</strong> <em>37 </em>
                                    (9-10)
                                     , 1800062. <a href="https://doi.org/10.1002/minf.201800062" title="DOI URL">https://doi.org/10.1002/minf.201800062</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/minf.201800062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fminf.201800062%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Informatics%26atitle%3DDocking%252Bof%252BCovalent%252BLigands%25253A%252BChallenges%252Band%252BApproaches%26aulast%3DSotriffer%26aufirst%3DChristoph%26date%3D2018%26date%3D2018%26volume%3D37%26issue%3D9-10%26spage%3D1800062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jean-Rémy  Marchand</span>, <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">In silico fragment-based drug design with SEED. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>156 </em>, 907-917. <a href="https://doi.org/10.1016/j.ejmech.2018.07.042" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.07.042</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.07.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.07.042%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIn%252Bsilico%252Bfragment-based%252Bdrug%252Bdesign%252Bwith%252BSEED%26aulast%3DMarchand%26aufirst%3DJean-R%25C3%25A9my%26date%3D2018%26volume%3D156%26spage%3D907%26epage%3D917" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaojing  Yuan</span>, <span class="hlFld-ContribAuthor ">Yechun  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Trends and Applications of Molecular Modeling in GPCR–Ligand Recognition and Structure-Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2018,</strong> <em>19 </em>
                                    (7)
                                     , 2105. <a href="https://doi.org/10.3390/ijms19072105" title="DOI URL">https://doi.org/10.3390/ijms19072105</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms19072105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms19072105%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DRecent%252BTrends%252Band%252BApplications%252Bof%252BMolecular%252BModeling%252Bin%252BGPCR%2525E2%252580%252593Ligand%252BRecognition%252Band%252BStructure-Based%252BDrug%252BDesign%26aulast%3DYuan%26aufirst%3DXiaojing%26date%3D2018%26date%3D2018%26volume%3D19%26issue%3D7%26spage%3D2105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adam  Pecina</span>, <span class="hlFld-ContribAuthor ">Jiří  Brynda</span>, <span class="hlFld-ContribAuthor ">Lukáš  Vrzal</span>, <span class="hlFld-ContribAuthor ">Ramachandran  Gnanasekaran</span>, <span class="hlFld-ContribAuthor ">Magdalena  Hořejší</span>, <span class="hlFld-ContribAuthor ">Saltuk M.  Eyrilmez</span>, <span class="hlFld-ContribAuthor ">Jan  Řezáč</span>, <span class="hlFld-ContribAuthor ">Martin  Lepšík</span>, <span class="hlFld-ContribAuthor ">Pavlína  Řezáčová</span>, <span class="hlFld-ContribAuthor ">Pavel  Hobza</span>, <span class="hlFld-ContribAuthor ">Pavel  Majer</span>, <span class="hlFld-ContribAuthor ">Václav  Veverka</span>, <span class="hlFld-ContribAuthor ">Jindřich  Fanfrlík</span>. </span><span class="cited-content_cbyCitation_article-title">Ranking Power of the SQM/COSMO Scoring Function on Carbonic Anhydrase II–Inhibitor Complexes. </span><span class="cited-content_cbyCitation_journal-name">ChemPhysChem</span><span> <strong>2018,</strong> <em>19 </em>
                                    (7)
                                     , 873-879. <a href="https://doi.org/10.1002/cphc.201701104" title="DOI URL">https://doi.org/10.1002/cphc.201701104</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cphc.201701104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcphc.201701104%26sid%3Dliteratum%253Aachs%26jtitle%3DChemPhysChem%26atitle%3DRanking%252BPower%252Bof%252Bthe%252BSQM%25252FCOSMO%252BScoring%252BFunction%252Bon%252BCarbonic%252BAnhydrase%2525E2%252580%252585II%2525E2%252580%252593Inhibitor%252BComplexes%26aulast%3DPecina%26aufirst%3DAdam%26date%3D2018%26date%3D2018%26volume%3D19%26issue%3D7%26spage%3D873%26epage%3D879" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Magdalena  Korczynska</span>, <span class="hlFld-ContribAuthor ">Mary J.  Clark</span>, <span class="hlFld-ContribAuthor ">Celine  Valant</span>, <span class="hlFld-ContribAuthor ">Jun  Xu</span>, <span class="hlFld-ContribAuthor ">Ee Von  Moo</span>, <span class="hlFld-ContribAuthor ">Sabine  Albold</span>, <span class="hlFld-ContribAuthor ">Dahlia R.  Weiss</span>, <span class="hlFld-ContribAuthor ">Hayarpi  Torosyan</span>, <span class="hlFld-ContribAuthor ">Weijiao  Huang</span>, <span class="hlFld-ContribAuthor ">Andrew C.  Kruse</span>, <span class="hlFld-ContribAuthor ">Brent R.  Lyda</span>, <span class="hlFld-ContribAuthor ">Lauren T.  May</span>, <span class="hlFld-ContribAuthor ">Jo-Anne  Baltos</span>, <span class="hlFld-ContribAuthor ">Patrick M.  Sexton</span>, <span class="hlFld-ContribAuthor ">Brian K.  Kobilka</span>, <span class="hlFld-ContribAuthor ">Arthur  Christopoulos</span>, <span class="hlFld-ContribAuthor ">Brian K.  Shoichet</span>, <span class="hlFld-ContribAuthor ">Roger K.  Sunahara</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based discovery of selective positive allosteric modulators of antagonists for the M
              2
              muscarinic acetylcholine receptor. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2018,</strong> <em>115 </em>
                                    (10)
                                     , E2419-E2428. <a href="https://doi.org/10.1073/pnas.1718037115" title="DOI URL">https://doi.org/10.1073/pnas.1718037115</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.1718037115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.1718037115%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DStructure-based%252Bdiscovery%252Bof%252Bselective%252Bpositive%252Ballosteric%252Bmodulators%252Bof%252Bantagonists%252Bfor%252Bthe%252BM%252B2%252Bmuscarinic%252Bacetylcholine%252Breceptor%26aulast%3DKorczynska%26aufirst%3DMagdalena%26date%3D2018%26date%3D2018%26volume%3D115%26issue%3D10%26spage%3DE2419%26epage%3DE2428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natalia  Nikitina</span>, <span class="hlFld-ContribAuthor ">Evgeny  Ivashko</span>, <span class="hlFld-ContribAuthor ">Andrei  Tchernykh</span>. </span><span class="cited-content_cbyCitation_article-title">Congestion game scheduling for virtual drug screening optimization. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2018,</strong> <em>32 </em>
                                    (2)
                                     , 363-374. <a href="https://doi.org/10.1007/s10822-017-0093-7" title="DOI URL">https://doi.org/10.1007/s10822-017-0093-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-017-0093-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-017-0093-7%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DCongestion%252Bgame%252Bscheduling%252Bfor%252Bvirtual%252Bdrug%252Bscreening%252Boptimization%26aulast%3DNikitina%26aufirst%3DNatalia%26date%3D2018%26date%3D2017%26volume%3D32%26issue%3D2%26spage%3D363%26epage%3D374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Malkeet Singh  Bahia</span>, <span class="hlFld-ContribAuthor ">Ido  Nissim</span>, <span class="hlFld-ContribAuthor ">Masha Y.  Niv</span>. </span><span class="cited-content_cbyCitation_article-title">Bitterness prediction in-silico : A step towards better drugs. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Pharmaceutics</span><span> <strong>2018,</strong> <em>536 </em>
                                    (2)
                                     , 526-529. <a href="https://doi.org/10.1016/j.ijpharm.2017.03.076" title="DOI URL">https://doi.org/10.1016/j.ijpharm.2017.03.076</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ijpharm.2017.03.076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ijpharm.2017.03.076%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Pharmaceutics%26atitle%3DBitterness%252Bprediction%252Bin-silico%252B%25253A%252BA%252Bstep%252Btowards%252Bbetter%252Bdrugs%26aulast%3DBahia%26aufirst%3DMalkeet%2BSingh%26date%3D2018%26volume%3D536%26issue%3D2%26spage%3D526%26epage%3D529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gaurava  Srivastava</span>, <span class="hlFld-ContribAuthor ">Ashish  Tiwari</span>, <span class="hlFld-ContribAuthor ">Ashok  Sharma</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Methods for Multi-Target Drug Designing Against Mycobacterium tuberculosis. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 459-483. <a href="https://doi.org/10.1007/7653_2018_19" title="DOI URL">https://doi.org/10.1007/7653_2018_19</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7653_2018_19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7653_2018_19%26sid%3Dliteratum%253Aachs%26atitle%3DComputational%252BMethods%252Bfor%252BMulti-Target%252BDrug%252BDesigning%252BAgainst%252BMycobacterium%252Btuberculosis%26aulast%3DSrivastava%26aufirst%3DGaurava%26date%3D2018%26date%3D2018%26spage%3D459%26epage%3D483%26pub%3DSpringer%2520New%2520York%26atitle%3DMulti-Target%252BDrug%252BDesign%252BUsing%252BChem-Bioinformatic%252BApproaches%26aulast%3DRoy%26aufirst%3DKunal%26date%3D2019%26volume%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vassilios  Myrianthopoulos</span>, <span class="hlFld-ContribAuthor ">George  Lambrinidis</span>, <span class="hlFld-ContribAuthor ">Emmanuel  Mikros</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Screening of Compound Libraries Using a Consensus of Orthogonal Methodologies. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 261-277. <a href="https://doi.org/10.1007/978-1-4939-8630-9_15" title="DOI URL">https://doi.org/10.1007/978-1-4939-8630-9_15</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4939-8630-9_15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4939-8630-9_15%26sid%3Dliteratum%253Aachs%26atitle%3DIn%252BSilico%252BScreening%252Bof%252BCompound%252BLibraries%252BUsing%252Ba%252BConsensus%252Bof%252BOrthogonal%252BMethodologies%26aulast%3DMyrianthopoulos%26aufirst%3DVassilios%26date%3D2018%26date%3D2018%26spage%3D261%26epage%3D277%26pub%3DSpringer%2520New%2520York%26atitle%3DRational%252BDrug%252BDesign%26aulast%3DMavromoustakos%26aufirst%3DThomas%26date%3D2018%26volume%3D1824" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrew  Anighoro</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">A Hybrid Virtual Screening Protocol Based on Binding Mode Similarity. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 165-175. <a href="https://doi.org/10.1007/978-1-4939-8630-9_9" title="DOI URL">https://doi.org/10.1007/978-1-4939-8630-9_9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4939-8630-9_9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4939-8630-9_9%26sid%3Dliteratum%253Aachs%26atitle%3DA%252BHybrid%252BVirtual%252BScreening%252BProtocol%252BBased%252Bon%252BBinding%252BMode%252BSimilarity%26aulast%3DAnighoro%26aufirst%3DAndrew%26date%3D2018%26date%3D2018%26spage%3D165%26epage%3D175%26pub%3DSpringer%2520New%2520York%26atitle%3DRational%252BDrug%252BDesign%26aulast%3DMavromoustakos%26aufirst%3DThomas%26date%3D2018%26volume%3D1824" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">A.  Sabitha Rani</span>, <span class="hlFld-ContribAuthor ">G.  Neelima</span>, <span class="hlFld-ContribAuthor ">Rupasree  Mukhopadhyay</span>, <span class="hlFld-ContribAuthor ">K. S. N.  Jyothi</span>, <span class="hlFld-ContribAuthor ">G.  Sulakshana</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Characterization of Plant Secondary Metabolites. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 251-273. <a href="https://doi.org/10.1007/978-981-13-0347-0_15" title="DOI URL">https://doi.org/10.1007/978-981-13-0347-0_15</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-981-13-0347-0_15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-981-13-0347-0_15%26sid%3Dliteratum%253Aachs%26atitle%3DIn%252BSilico%252BCharacterization%252Bof%252BPlant%252BSecondary%252BMetabolites%26aulast%3DSabitha%2BRani%26aufirst%3DA.%26date%3D2018%26date%3D2018%26spage%3D251%26epage%3D273%26pub%3DSpringer%2520Singapore%26atitle%3DIn%252BSilico%252BApproach%252Bfor%252BSustainable%252BAgriculture%26aulast%3DChoudhary%26aufirst%3DDevendra%2BK.%26date%3D2018%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div><a href="#" data-source="/pb/widgets/cited-by?doi=10.1021%2Facs.jmedchem.5b02008&amp;widgetId=9cf770f4-a9a2-46e0-bd29-3da15bcf4ae0&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-9%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b02008&amp;citedByCount=129&amp;pagesCount=2&amp;pageNumber=-1" title="Load all citations" class="button_primary float-right cited-by__see-more mb-5"><i aria-hidden="”true”" class="icon-angle-double-down"></i><span>Load all citations</span></a></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/medium/jm-2015-020088_0003.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02008&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/medium/jm-2015-020088_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0005.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Docking for new chemotypes from large libraries.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02008&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/medium/jm-2015-020088_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0006.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Superposition of docking-predicted and subsequently determined crystal structures. (A) A 9 nM inverse agonist of the β2 adrenergic receptor discovered by docking (orange) superposed on the crystallographic result (green).<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> (B) Docking-predicted structure of eticlopride (light blue) superposed on the crystallographic result (yellow) in the dopamine D<sub>3</sub> receptor. Reproduced with permission from  <cite>Nature Chemical Biology</cite> (<contrib-group>Carlsson, J.</contrib-group>; Ligand discovery from a dopamine D3 receptor homology model and crystal structure; <span class="NLM_year">2011</span>, <em>7</em>, 769–778);<a onclick="showRef(event, 'ref213'); return false;" href="javascript:void(0);" class="ref ref213">(213)</a> Copyright 2011 Macmillan Publishers Ltd. (C) A 30 μM inhibitor of β-lactamase discovered by docking (purple) superposed on its crystallographic structure (white).<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> (D, E) Two crystal structures (gray carbons, electron density in wire mesh) superposed on the docking predicted ligands (yellow carbons) and poses refined by postdocking rescoring programs AMBERDOCK (cyan carbons) and PLOP (magenta carbons) for the nonpolar T4 lysozyme L99A and L99A/M102Q model cavities. Reproduced with permission from  <cite>Journal of Molecular Biology</cite> (<contrib-group>Graves, A. P.</contrib-group>; Rescoring docking hit lists for model cavity sites: predictions and experimental testing; <span class="NLM_year">2008</span>, <em>377</em>, 914–934);<a onclick="showRef(event, 'ref245'); return false;" href="javascript:void(0);" class="ref ref245">(245)</a> Copyright 2008 Elsevier, Ltd. (F) A docked ligand superposed on the crystallographic result from a screen against the model anion cavity in cytochrome <i>c</i> peroxidase W191G/gateless. Reproduced with permission from  <cite>Nature Chemistry</cite> (<contrib-group>Fischer, M.</contrib-group>; Incorporation of protein flexibility and conformational energy penalties in docking screens to improve ligand discovery; <span class="NLM_year">2014</span>, <em>6</em>, 575–583);<a onclick="showRef(event, 'ref243'); return false;" href="javascript:void(0);" class="ref ref243">(243)</a> Copyright 2014 Macmillan Publishers Ltd. (G) A 40 nM boronic acid inhibitor of AmpC predicted by covalent docking (cyan carbons) superposed on its crystal structure with the enzyme.<a onclick="showRef(event, 'ref175'); return false;" href="javascript:void(0);" class="ref ref175">(175)</a> (H) A 370 nM inhibitor of RSK2 kinase predicted by covalent docking (magenta carbons) superposed on its crystal structure with the enzyme (yellow carbons, with <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density shown in green mesh.<a onclick="showRef(event, 'ref175'); return false;" href="javascript:void(0);" class="ref ref175">(175)</a> (I) The docked structure (green carbons) of the high-energy intermediate of <i>S</i>-adenosyl homocysteine superposed on the crystallographic structure of the product, <i>S</i>-inosyl homocysteine (red carbons) in complex with Tm0936. Reproduced with permission from  <cite>Nature</cite> (<contrib-group>Hermann, J. C.</contrib-group>; Structure-based activity prediction for an enzyme of unknown function; <span class="NLM_year">2007</span>, <em>448</em>, 775–779);<a onclick="showRef(event, 'ref198'); return false;" href="javascript:void(0);" class="ref ref198">(198)</a> Copyright 2007 Macmillan Publishers Ltd. (J) An 8 μM inhibitor of PDK1 showing the docking pose (yellow carbons) superposed on the subsequently determined crystal structure (magenta carbons).<a onclick="showRef(event, 'ref234'); return false;" href="javascript:void(0);" class="ref ref234">(234)</a> (K) Modeled binding mode of a 10.6 μM fragment (orange carbons) superimposed with the dominant binding mode determined crystallographically (green carbon atoms).<a onclick="showRef(event, 'ref251'); return false;" href="javascript:void(0);" class="ref ref251">(251)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02008&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/medium/jm-2015-020088_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0007.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Aggregators (first row) and PAINS<a onclick="showRef(event, 'ref120'); return false;" href="javascript:void(0);" class="ref ref120">(120)</a> (second row) identified by docking: (A) 0.5 μM inhibitor of β-lactamase;<a onclick="showRef(event, 'ref127'); return false;" href="javascript:void(0);" class="ref ref127">(127)</a> (B) 7 μM inhibitor of DHFR;<a onclick="showRef(event, 'ref252'); return false;" href="javascript:void(0);" class="ref ref252">(252)</a> (C) 5 μM inhibitor of VEGF (and 10 μM IGF-1);<a onclick="showRef(event, 'ref253'); return false;" href="javascript:void(0);" class="ref ref253">(253)</a> (D) 14 μM inhibitor of tyrosine phosphatase;<a onclick="showRef(event, 'ref131'); return false;" href="javascript:void(0);" class="ref ref131">(131)</a> (E) 30 μM inhibitor of PTPRQ;<a onclick="showRef(event, 'ref130'); return false;" href="javascript:void(0);" class="ref ref130">(130)</a> (F) 3 μM inhibitor of DHFR.<a onclick="showRef(event, 'ref254'); return false;" href="javascript:void(0);" class="ref ref254">(254)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02008&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/medium/jm-2015-020088_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0001.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Classical and aggregation-based concentration–response curves. (a) The well-behaved, competitive inhibitor BZBTH2B (■, red curve) and the aggregator rottlerin (●, black curve), against the enzyme β-lactamase (adapted from ref <a onclick="showRef(event, 'ref255'); return false;" href="javascript:void(0);" class="ref ref255">255</a>, used with permission). (b) As the concentration of β-lactamase is increased, the inhibition curve shifts right and steepens for colloidal aggregators like rottlerin.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02008&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/medium/jm-2015-020088_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0002.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. New functions conferred by new chemotypes discovered by docking. (A) The β-lactam inhibitor clavulanic acid up-regulates the expression of β-lactamase in <i>E. cloacae</i>, antagonizing the efficacy of the primary β-lactam antibiotics ceftazidime (CAZ) and piperacillin (PIP) in a disk diffusion assay (left).<a onclick="showRef(event, 'ref47 ref197'); return false;" href="javascript:void(0);" class="ref ref47 ref197">(47, 197)</a> In contrast, the docking-discovered inhibitors simply inhibit the enzyme without up-regulating it, synergizing with the same primary antibiotics (right). (B) Lorazepam (left) and ogerin (middle) are both modeled to bind at the same site on GPR68. Ogerin, however, is more potent and specific than lorazepam. In wild-type mice, ogerin reduces fear conditioning in a contextual freezing assay compared to knockout mice, where it has no measurable effect (right). Reproduced with permission from  <cite>Nature</cite> (<contrib-group>Huang, X. P.</contrib-group>; Allosteric ligands for the pharmacologically dark receptors GPR68 and GPR65; <span class="NLM_year">2015</span>, <em>527</em>, 477–483);<a onclick="showRef(event, 'ref256'); return false;" href="javascript:void(0);" class="ref ref256">(256)</a> Copyright 2015 Macmillan Publishers Ltd. (C) Inhibition of visceral hypersensitivity in IBS rat model. Therapeutic effects of compound <b>15</b> in IBS rats. The pain threshold decreased in response to CRD (at score 3 (left) and score 4 (right)).<a onclick="showRef(event, 'ref201'); return false;" href="javascript:void(0);" class="ref ref201">(201)</a> (D) Left panel. Xenograft tumor growth of human lung cancer cell line H2122, 50 mg kg<sup>–1</sup> day<sup>–1</sup> RBC8 initiated 24 h after inoculation inhibited tumor growth. Typical tumor appearance at 21 days is shown. Right panel. Effect of RBC8 on H358 xenograft models. RBC8 treatment (50 mg kg<sup>–1</sup> day<sup>–1</sup>) initiated 24 h after inoculation inhibited xenograft tumor growth of human lung cancer cell line H358. Data represent the mean ± SEM of six mice. Tumor volume in the treatment group was statistically different from controls as determined by the Student’s <i>t</i> test. Reproduced with permission from  <cite>Nature</cite> (<contrib-group>Yan, C.</contrib-group>; Discovery and characterization of small molecules that target the GTPase Ral; <span class="NLM_year">2014</span>, <em>515</em>, 443–447);<a onclick="showRef(event, 'ref202'); return false;" href="javascript:void(0);" class="ref ref202">(202)</a> Copyright 2014 Macmillan Publishers Ltd.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b02008/20160506/images/large/jm-2015-020088_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b02008&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i26">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66283" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66283" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 256 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Beddell, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodford, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norrington, F. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilkinson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wootton, R.</span><span> </span><span class="NLM_article-title">Compounds designed to fit a site of known structure in human haemoglobin</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1976</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">201</span><span class="NLM_x">–</span> <span class="NLM_lpage">209</span><span class="refDoi"> DOI: 10.1111/j.1476-5381.1976.tb07468.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1111%2Fj.1476-5381.1976.tb07468.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=938794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADyaE28XlsFGhtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1976&pages=201-209&author=C.+R.+Beddellauthor=P.+J.+Goodfordauthor=F.+E.+Norringtonauthor=S.+Wilkinsonauthor=R.+Wootton&title=Compounds+designed+to+fit+a+site+of+known+structure+in+human+haemoglobin&doi=10.1111%2Fj.1476-5381.1976.tb07468.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Compounds designed to fit a site of known structure in human hemoglobin</span></div><div class="casAuthors">Beddell, C. R.; Goodford, P. J.; Norrington, F. E.; Wilkinson, S.; Wootton, R.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">201-9</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    </div><div class="casAbstract">Three 2,3-diphosphoglycerate (I) analogs, e.g. 4,4'-diformyl-2-bibenzyloxyacetic acid (II), bound to human deoxy-Hb at the I-binding site.  They promoted O liberation in the predicted sequence as assessed by sigmoidal dose-response curves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoG1ShdM6H3e7Vg90H21EOLACvtfcHk0lj8fKpWctY1WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XlsFGhtLk%253D&md5=7f53a69f78a5930b75956c30aa03b720</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.1976.tb07468.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.1976.tb07468.x%26sid%3Dliteratum%253Aachs%26aulast%3DBeddell%26aufirst%3DC.%2BR.%26aulast%3DGoodford%26aufirst%3DP.%2BJ.%26aulast%3DNorrington%26aufirst%3DF.%2BE.%26aulast%3DWilkinson%26aufirst%3DS.%26aulast%3DWootton%26aufirst%3DR.%26atitle%3DCompounds%2520designed%2520to%2520fit%2520a%2520site%2520of%2520known%2520structure%2520in%2520human%2520haemoglobin%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D1976%26volume%3D57%26spage%3D201%26epage%3D209%26doi%3D10.1111%2Fj.1476-5381.1976.tb07468.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Cohen, S. S.</span><span> </span><span class="NLM_article-title">A strategy for the chemotherapy of infectious disease</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">197</span><span class="NLM_x">, </span> <span class="NLM_fpage">431</span><span class="NLM_x">–</span> <span class="NLM_lpage">432</span><span class="refDoi"> DOI: 10.1126/science.195340</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1126%2Fscience.195340" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=197&publication_year=1977&pages=431-432&author=S.+S.+Cohen&title=A+strategy+for+the+chemotherapy+of+infectious+disease&doi=10.1126%2Fscience.195340"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.195340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.195340%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DS.%2BS.%26atitle%3DA%2520strategy%2520for%2520the%2520chemotherapy%2520of%2520infectious%2520disease%26jtitle%3DScience%26date%3D1977%26volume%3D197%26spage%3D431%26epage%3D432%26doi%3D10.1126%2Fscience.195340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span> </span><span class="NLM_article-title">Structure-based drug screening for G-protein-coupled receptors</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">268</span><span class="NLM_x">–</span> <span class="NLM_lpage">272</span><span class="refDoi"> DOI: 10.1016/j.tips.2012.03.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.tips.2012.03.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=22503476" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFGit7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=268-272&author=B.+K.+Shoichetauthor=B.+K.+Kobilka&title=Structure-based+drug+screening+for+G-protein-coupled+receptors&doi=10.1016%2Fj.tips.2012.03.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based drug screening for G-protein-coupled receptors</span></div><div class="casAuthors">Shoichet, Brian K.; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">268-272</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  G-protein-coupled receptors (GPCRs) represent a large family of signaling proteins that includes many therapeutic targets; however, progress in identifying new small mol. drugs has been disappointing.  The past 4 years have seen remarkable progress in the structural biol. of GPCRs, raising the possibility of applying structure-based approaches to GPCR drug discovery efforts.  Of the various structure-based approaches that have been applied to sol. protein targets, such as proteases and kinases, in silico docking is among the most ready applicable to GPCRs.  Early studies suggest that GPCR binding pockets are well suited to docking, and docking screens have identified potent and novel compds. for these targets.  This review will focus on the current state of in silico docking for GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Eh6VhLHhH7Vg90H21EOLACvtfcHk0lj8fKpWctY1WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFGit7Y%253D&md5=30f26646db71f5c0c8448e9d85908060</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2012.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2012.03.007%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520drug%2520screening%2520for%2520G-protein-coupled%2520receptors%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26spage%3D268%26epage%3D272%26doi%3D10.1016%2Fj.tips.2012.03.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Mittl, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grutter, M. G.</span><span> </span><span class="NLM_article-title">Opportunities for structure-based design of protease-directed drugs</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">769</span><span class="NLM_x">–</span> <span class="NLM_lpage">775</span><span class="refDoi"> DOI: 10.1016/j.sbi.2006.10.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.sbi.2006.10.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=17112720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1OhtL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=769-775&author=P.+R.+Mittlauthor=M.+G.+Grutter&title=Opportunities+for+structure-based+design+of+protease-directed+drugs&doi=10.1016%2Fj.sbi.2006.10.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Opportunities for structure-based design of protease-directed drugs</span></div><div class="casAuthors">Mittl, Peer RE; Gruetter, Markus G.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">769-775</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  As a result of the recent enormous technol. progress, exptl. structure detn. has become an integral part of the development of drugs against disease-related target proteins.  The post-translational modification of proteins is an important regulatory process in living organisms; one such example is lytic processing by peptidases.  Many different peptidases represent disease targets and are being used in structure-based drug design approaches.  The development of drugs such as aliskiren and tipranavir, which inhibit renin and HIV protease, resp., testifies to the success of this approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcGxJhLhiHbrVg90H21EOLACvtfcHk0lgq8Df4iyWzkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1OhtL%252FN&md5=905b47627a5799b80be34fa0ed9d749d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2006.10.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2006.10.014%26sid%3Dliteratum%253Aachs%26aulast%3DMittl%26aufirst%3DP.%2BR.%26aulast%3DGrutter%26aufirst%3DM.%2BG.%26atitle%3DOpportunities%2520for%2520structure-based%2520design%2520of%2520protease-directed%2520drugs%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2006%26volume%3D16%26spage%3D769%26epage%3D775%26doi%3D10.1016%2Fj.sbi.2006.10.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">von Itzstein, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, W.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kok, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pegg, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dyason, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, T. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smythe, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliver, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colman, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varghese, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryan, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woods, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bethell, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hotham, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Penn, C. R.</span><span> </span><span class="NLM_article-title">Rational design of potent sialidase-based inhibitors of influenza virus replication</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">363</span><span class="NLM_x">, </span> <span class="NLM_fpage">418</span><span class="NLM_x">–</span> <span class="NLM_lpage">423</span><span class="refDoi"> DOI: 10.1038/363418a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2F363418a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=8502295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADyaK3sXkvVGntb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=363&publication_year=1993&pages=418-423&author=M.+von+Itzsteinauthor=W.-Y.+Wuauthor=G.+B.+Kokauthor=M.+S.+Peggauthor=J.+C.+Dyasonauthor=B.+Jinauthor=T.+V.+Phanauthor=M.+L.+Smytheauthor=H.+F.+Whiteauthor=S.+W.+Oliverauthor=P.+M.+Colmanauthor=J.+N.+Vargheseauthor=D.+M.+Ryanauthor=J.+M.+Woodsauthor=R.+C.+Bethellauthor=V.+J.+Hothamauthor=J.+M.+Cameronauthor=C.+R.+Penn&title=Rational+design+of+potent+sialidase-based+inhibitors+of+influenza+virus+replication&doi=10.1038%2F363418a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Rational design of potent sialidase-based inhibitors of influenza virus replication</span></div><div class="casAuthors">von Itzstein, Mark; Wu, Wen Yang; Kok, Gaik B.; Pegg, Michael S.; Dyason, Jeffrey C.; Jin, Betty; Phan Tho Van; Smythe, Mark L.; White, Hume F.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">363</span>
        (<span class="NLM_cas:issue">6428</span>),
    <span class="NLM_cas:pages">418-23</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Two potent inhibitors based on the crystal structure of influenza virus sialidase have been designed.  These compds. are effective inhibitors not only on the enzyme, but also of the virus in cell culture and in animal models.  The results provide an example of the power of rational, computer-assisted drug design, as well as indicating significant progress in the development of a new therapeutic or prophylactic treatment for influenza infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz-zoSWSTF3bVg90H21EOLACvtfcHk0lgq8Df4iyWzkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXkvVGntb0%253D&md5=dc1e83fa24f50f54941924a43770468e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2F363418a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F363418a0%26sid%3Dliteratum%253Aachs%26aulast%3Dvon%2BItzstein%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DW.-Y.%26aulast%3DKok%26aufirst%3DG.%2BB.%26aulast%3DPegg%26aufirst%3DM.%2BS.%26aulast%3DDyason%26aufirst%3DJ.%2BC.%26aulast%3DJin%26aufirst%3DB.%26aulast%3DPhan%26aufirst%3DT.%2BV.%26aulast%3DSmythe%26aufirst%3DM.%2BL.%26aulast%3DWhite%26aufirst%3DH.%2BF.%26aulast%3DOliver%26aufirst%3DS.%2BW.%26aulast%3DColman%26aufirst%3DP.%2BM.%26aulast%3DVarghese%26aufirst%3DJ.%2BN.%26aulast%3DRyan%26aufirst%3DD.%2BM.%26aulast%3DWoods%26aufirst%3DJ.%2BM.%26aulast%3DBethell%26aufirst%3DR.%2BC.%26aulast%3DHotham%26aufirst%3DV.%2BJ.%26aulast%3DCameron%26aufirst%3DJ.%2BM.%26aulast%3DPenn%26aufirst%3DC.%2BR.%26atitle%3DRational%2520design%2520of%2520potent%2520sialidase-based%2520inhibitors%2520of%2520influenza%2520virus%2520replication%26jtitle%3DNature%26date%3D1993%26volume%3D363%26spage%3D418%26epage%3D423%26doi%3D10.1038%2F363418a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Varney, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marzoni, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmer, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deal, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webber, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Welsh, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bacquet, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morse, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Booth, C. L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrmann, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howland, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, J.</span><span> </span><span class="NLM_article-title">Crystal-structure-based design and synthesis of Benz[cd]indole-containing inhibitors of thymidylate synthase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">663</span><span class="NLM_x">–</span> <span class="NLM_lpage">676</span><span class="refDoi"> DOI: 10.1021/jm00082a006</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00082a006" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADyaK38Xht1ymtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1992&pages=663-676&author=M.+D.+Varneyauthor=G.+P.+Marzoniauthor=C.+L.+Palmerauthor=J.+G.+Dealauthor=S.+Webberauthor=K.+M.+Welshauthor=R.+J.+Bacquetauthor=C.+A.+Morseauthor=C.+L.+J.+Boothauthor=S.+M.+Herrmannauthor=E.+F.+Howlandauthor=R.+W.+Wardauthor=J.+White&title=Crystal-structure-based+design+and+synthesis+of+Benz%5Bcd%5Dindole-containing+inhibitors+of+thymidylate+synthase&doi=10.1021%2Fjm00082a006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal-structure-based design and synthesis of benz[cd]indole-containing inhibitors of thymidylate synthase</span></div><div class="casAuthors">Varney, Michael D.; Marzoni, Gifford P.; Palmer, Cindy L.; Deal, Judith G.; Webber, Stephanie; Welsh, Katherine M.; Bacquet, Russell J.; Bartlett, Charlotte A.; Morse, Catharine A.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">663-76</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Benz[cd]indoles I [R1 = H, Me, Et, Pr, CHMe2, CH2CH2OH, CH2CH2CH2OH, CH2SMe, R2 = H, Me, Cl, SMe, R3 = piperazinyl, morpholino, (un)substituted Ph; X = NH, NNH2, NOH, NMe, O, (SMe)2] were prepd. as inhibitors of thymidylate synthase (TS).  The structure-activity of the lead compd. was studied by conceptually dividing the mol. into four regions and independently optimizing the substituents for each region.  Combination of favored substituents for each region led to inhibitors with Kis against the human enzyme in the range of 10-20 nM.  Thymidine shift expts. suggested that the cytotoxic properties of the best enzyme inhibitors were due to TS targeting in cells.  The inhibitors were synthesized from substituted 6-aminobenz[cd]indol-2(1H)-ones by alkylation with both a simple alkyl group and a substituted benzylic portion.  The 2,6-diaminobenz[cd]indoles were prepd. from the corresponding lactams by conversion to the thiolactam, alkylation to the methylated thiolactam, and then displacement with a substituted or unsubstituted amine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrosbvRQ2rQbVg90H21EOLACvtfcHk0ljO3F3Crp0_HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xht1ymtLg%253D&md5=b1059e47866ee0111117ca66b043eac7</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm00082a006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00082a006%26sid%3Dliteratum%253Aachs%26aulast%3DVarney%26aufirst%3DM.%2BD.%26aulast%3DMarzoni%26aufirst%3DG.%2BP.%26aulast%3DPalmer%26aufirst%3DC.%2BL.%26aulast%3DDeal%26aufirst%3DJ.%2BG.%26aulast%3DWebber%26aufirst%3DS.%26aulast%3DWelsh%26aufirst%3DK.%2BM.%26aulast%3DBacquet%26aufirst%3DR.%2BJ.%26aulast%3DMorse%26aufirst%3DC.%2BA.%26aulast%3DBooth%26aufirst%3DC.%2BL.%2BJ.%26aulast%3DHerrmann%26aufirst%3DS.%2BM.%26aulast%3DHowland%26aufirst%3DE.%2BF.%26aulast%3DWard%26aufirst%3DR.%2BW.%26aulast%3DWhite%26aufirst%3DJ.%26atitle%3DCrystal-structure-based%2520design%2520and%2520synthesis%2520of%2520Benz%255Bcd%255Dindole-containing%2520inhibitors%2520of%2520thymidylate%2520synthase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1992%26volume%3D35%26spage%3D663%26epage%3D676%26doi%3D10.1021%2Fjm00082a006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Feng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallace, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stafford, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaldor, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kassel, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Navre, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skene, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asakawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeuchi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gwaltney, S. L.,  2nd</span><span> </span><span class="NLM_article-title">Discovery of alogliptin: a potent, selective, bioavailable, and efficacious inhibitor of dipeptidyl peptidase IV</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2297</span><span class="NLM_x">–</span> <span class="NLM_lpage">2300</span><span class="refDoi"> DOI: 10.1021/jm070104l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm070104l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkt1Oqtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2297-2300&author=J.+Fengauthor=Z.+Zhangauthor=M.+B.+Wallaceauthor=J.+A.+Staffordauthor=S.+W.+Kaldorauthor=D.+B.+Kasselauthor=M.+Navreauthor=L.+Shiauthor=R.+J.+Skeneauthor=T.+Asakawaauthor=K.+Takeuchiauthor=R.+Xuauthor=D.+R.+Webbauthor=S.+L.+Gwaltney&title=Discovery+of+alogliptin%3A+a+potent%2C+selective%2C+bioavailable%2C+and+efficacious+inhibitor+of+dipeptidyl+peptidase+IV&doi=10.1021%2Fjm070104l"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Alogliptin: A Potent, Selective, Bioavailable, and Efficacious Inhibitor of Dipeptidyl Peptidase IV</span></div><div class="casAuthors">Feng, Jun; Zhang, Zhiyuan; Wallace, Michael B.; Stafford, Jeffrey A.; Kaldor, Stephen W.; Kassel, Daniel B.; Navre, Marc; Shi, Lihong; Skene, Robert J.; Asakawa, Tomoko; Takeuchi, Koji; Xu, Rongda; Webb, David R.; Gwaltney, Stephen L., II</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2297-2300</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Alogliptin is a potent, selective inhibitor of the serine protease dipeptidyl peptidase IV (DPP-4).  Herein, the authors describe the structure-based design and optimization of alogliptin and related quinazolinone-based DPP-4 inhibitors.  Following an oral dose, these noncovalent inhibitors provide sustained redn. of plasma DPP-4 activity and a lowering of blood glucose in animal models of diabetes.  Alogliptin is currently undergoing phase III trials in patients with type 2 diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRXyHbCH4hrLVg90H21EOLACvtfcHk0ljO3F3Crp0_HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkt1Oqtrs%253D&md5=807ee0a245580988c22a02832a13084e</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm070104l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070104l%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DWallace%26aufirst%3DM.%2BB.%26aulast%3DStafford%26aufirst%3DJ.%2BA.%26aulast%3DKaldor%26aufirst%3DS.%2BW.%26aulast%3DKassel%26aufirst%3DD.%2BB.%26aulast%3DNavre%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DSkene%26aufirst%3DR.%2BJ.%26aulast%3DAsakawa%26aufirst%3DT.%26aulast%3DTakeuchi%26aufirst%3DK.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DWebb%26aufirst%3DD.%2BR.%26aulast%3DGwaltney%26aufirst%3DS.%2BL.%26atitle%3DDiscovery%2520of%2520alogliptin%253A%2520a%2520potent%252C%2520selective%252C%2520bioavailable%252C%2520and%2520efficacious%2520inhibitor%2520of%2520dipeptidyl%2520peptidase%2520IV%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D2297%26epage%3D2300%26doi%3D10.1021%2Fjm070104l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Sicheri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moarefi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span> </span><span class="NLM_article-title">Crystal structure of the Src family tyrosine kinase Hck</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">385</span><span class="NLM_x">, </span> <span class="NLM_fpage">602</span><span class="NLM_x">–</span> <span class="NLM_lpage">609</span><span class="refDoi"> DOI: 10.1038/385602a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2F385602a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=9024658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADyaK2sXht1ymtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=385&publication_year=1997&pages=602-609&author=F.+Sicheriauthor=I.+Moarefiauthor=J.+Kuriyan&title=Crystal+structure+of+the+Src+family+tyrosine+kinase+Hck&doi=10.1038%2F385602a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the Src family tyrosine kinase Hck</span></div><div class="casAuthors">Sicheri, Frank; Moarefi, Ismail; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">385</span>
        (<span class="NLM_cas:issue">6617</span>),
    <span class="NLM_cas:pages">602-609</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">The crystal structure of the hematopoietic cell kinase Hck has been detd. at 2.6/2.9 Å resoln.  Inhibition of enzymic activity is a consequence of intramol. interactions of the enzyme's Src-homol. domains SH2 and SH3, with concomitant displacement of elements of the catalytic domain.  The conformation of the active site has similarities with that of inactive cyclin-dependent protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotdo3t4KV6jLVg90H21EOLACvtfcHk0ljO3F3Crp0_HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXht1ymtLs%253D&md5=5703b01ea960313345d43e0f8047ffcf</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2F385602a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F385602a0%26sid%3Dliteratum%253Aachs%26aulast%3DSicheri%26aufirst%3DF.%26aulast%3DMoarefi%26aufirst%3DI.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DCrystal%2520structure%2520of%2520the%2520Src%2520family%2520tyrosine%2520kinase%2520Hck%26jtitle%3DNature%26date%3D1997%26volume%3D385%26spage%3D602%26epage%3D609%26doi%3D10.1038%2F385602a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Proteome-scale docking: myth and reality</span> <span class="citation_source-journal">Drug Discovery Today: Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e403</span><span class="NLM_x">–</span> <span class="NLM_lpage">409</span><span class="refDoi"> DOI: 10.1016/j.ddtec.2013.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.ddtec.2013.01.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=e403-409&author=D.+Rognan&title=Proteome-scale+docking%3A+myth+and+reality&doi=10.1016%2Fj.ddtec.2013.01.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2013.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2013.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DProteome-scale%2520docking%253A%2520myth%2520and%2520reality%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2013%26volume%3D10%26spage%3De403%26epage%3D409%26doi%3D10.1016%2Fj.ddtec.2013.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Richter, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sieghart, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varagic, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morzinger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecker, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernst, M.</span><span> </span><span class="NLM_article-title">Diazepam-bound GABAA receptor models identify new benzodiazepine binding-site ligands</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">455</span><span class="NLM_x">–</span> <span class="NLM_lpage">464</span><span class="refDoi"> DOI: 10.1038/nchembio.917</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnchembio.917" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=22446838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XksVegs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=455-464&author=L.+Richterauthor=C.+de+Graafauthor=W.+Sieghartauthor=Z.+Varagicauthor=M.+Morzingerauthor=I.+J.+de+Eschauthor=G.+F.+Eckerauthor=M.+Ernst&title=Diazepam-bound+GABAA+receptor+models+identify+new+benzodiazepine+binding-site+ligands&doi=10.1038%2Fnchembio.917"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Diazepam-bound GABAA receptor models identify new benzodiazepine binding-site ligands</span></div><div class="casAuthors">Richter, Lars; de Graaf, Chris; Sieghart, Werner; Varagic, Zdravko; Moerzinger, Martina; de Esch, Iwan J. P.; Ecker, Gerhard F.; Ernst, Margot</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">455-464</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Benzodiazepines exert their anxiolytic, anticonvulsant, muscle-relaxant and sedative-hypnotic properties by allosterically enhancing the action of GABA at GABAA receptors via their benzodiazepine-binding site.  Although these drugs have been used clin. since 1960, the mol. basis of this interaction is still not known.  By using multiple homol. models and an unbiased docking protocol, we identified a binding hypothesis for the diazepam-bound structure of the benzodiazepine site, which was confirmed by exptl. evidence.  Moreover, two independent virtual screening approaches based on this structure identified known benzodiazepine-site ligands from different structural classes and predicted potential new ligands for this site.  Receptor-binding assays and electrophysiol. studies on recombinant receptors confirmed these predictions and thus identified new chemotypes for the benzodiazepine-binding site.  Our results support the validity of the diazepam-bound structure of the benzodiazepine-binding pocket, demonstrate its suitability for drug discovery and pave the way for structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprfKzTBHZvLbVg90H21EOLACvtfcHk0lgPjqXAHX5p_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XksVegs7c%253D&md5=1fe4a778d6a7c585bcd3d44bcf22d631</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.917&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.917%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DL.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26aulast%3DSieghart%26aufirst%3DW.%26aulast%3DVaragic%26aufirst%3DZ.%26aulast%3DMorzinger%26aufirst%3DM.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26aulast%3DErnst%26aufirst%3DM.%26atitle%3DDiazepam-bound%2520GABAA%2520receptor%2520models%2520identify%2520new%2520benzodiazepine%2520binding-site%2520ligands%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D455%26epage%3D464%26doi%3D10.1038%2Fnchembio.917" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Min, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, Z.</span><span> </span><span class="NLM_article-title">Structure-based virtual screening of novel inhibitors of the uridyltransferase activity of Xanthomonas oryzae pv. oryzae GlmU</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">53C</span><span class="NLM_x">, </span> <span class="NLM_fpage">150</span><span class="NLM_x">–</span> <span class="NLM_lpage">158</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2012.03.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.ejmech.2012.03.051" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53C&publication_year=2012&pages=150-158&author=J.+Minauthor=D.+Linauthor=Q.+Zhangauthor=J.+Zhangauthor=Z.+Yu&title=Structure-based+virtual+screening+of+novel+inhibitors+of+the+uridyltransferase+activity+of+Xanthomonas+oryzae+pv.+oryzae+GlmU&doi=10.1016%2Fj.ejmech.2012.03.051"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2012.03.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2012.03.051%26sid%3Dliteratum%253Aachs%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DZ.%26atitle%3DStructure-based%2520virtual%2520screening%2520of%2520novel%2520inhibitors%2520of%2520the%2520uridyltransferase%2520activity%2520of%2520Xanthomonas%2520oryzae%2520pv.%2520oryzae%2520GlmU%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2012%26volume%3D53C%26spage%3D150%26epage%3D158%26doi%3D10.1016%2Fj.ejmech.2012.03.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Lavecchia, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Giovanni, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pesapane, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montuori, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ragno, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martucci, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masullo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Vendittis, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novellino, E.</span><span> </span><span class="NLM_article-title">Discovery of new inhibitors of Cdc25B dual specificity phosphatases by structure-based virtual screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">4142</span><span class="NLM_x">–</span> <span class="NLM_lpage">4158</span><span class="refDoi"> DOI: 10.1021/jm201624h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201624h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvV2ksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=4142-4158&author=A.+Lavecchiaauthor=C.+Di+Giovanniauthor=A.+Pesapaneauthor=N.+Montuoriauthor=P.+Ragnoauthor=N.+M.+Martucciauthor=M.+Masulloauthor=E.+De+Vendittisauthor=E.+Novellino&title=Discovery+of+new+inhibitors+of+Cdc25B+dual+specificity+phosphatases+by+structure-based+virtual+screening&doi=10.1021%2Fjm201624h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of New Inhibitors of Cdc25B Dual Specificity Phosphatases by Structure-Based Virtual Screening</span></div><div class="casAuthors">Lavecchia, Antonio; Di Giovanni, Carmen; Pesapane, Ada; Montuori, Nunzia; Ragno, Pia; Martucci, Nicola Massimiliano; Masullo, Mariorosario; De Vendittis, Emmanuele; Novellino, Ettore</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4142-4158</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cell division cycle 25 (Cdc25) proteins are highly conserved dual specificity phosphatases that regulate cyclin-dependent kinases and represent attractive drug targets for anticancer therapies.  To discover more potent and diverse inhibitors of Cdc25 biol. activity, virtual screening was performed by docking 2.1 million compds. into the Cdc25B active site.  An initial subset of top-ranked compds. was selected and assayed, and 15 were found to have enzyme inhibition activity at micromolar concn.  Among these, four structurally diverse inhibitors with a different inhibition profile were found to inhibit human MCF-7, PC-3, and K562 cancer cell proliferation and significantly affect the cell cycle progression.  A subsequent hierarchical similarity search with the most active reversible Cdc25B inhibitor found led to the identification of an addnl. set of 19 ligands, three of which were confirmed as Cdc25B inhibitors with IC50 values of 7.9, 4.2, and 9.9 μM, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGo8BLCiMiC7Vg90H21EOLACvtfcHk0lgPjqXAHX5p_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvV2ksr4%253D&md5=5e44438a537b9f82c54767b87d0135ea</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm201624h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201624h%26sid%3Dliteratum%253Aachs%26aulast%3DLavecchia%26aufirst%3DA.%26aulast%3DDi%2BGiovanni%26aufirst%3DC.%26aulast%3DPesapane%26aufirst%3DA.%26aulast%3DMontuori%26aufirst%3DN.%26aulast%3DRagno%26aufirst%3DP.%26aulast%3DMartucci%26aufirst%3DN.%2BM.%26aulast%3DMasullo%26aufirst%3DM.%26aulast%3DDe%2BVendittis%26aufirst%3DE.%26aulast%3DNovellino%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520new%2520inhibitors%2520of%2520Cdc25B%2520dual%2520specificity%2520phosphatases%2520by%2520structure-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D4142%26epage%3D4158%26doi%3D10.1021%2Fjm201624h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">da Silva, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">dos Santos, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campos, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mafud, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polikarpov, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figueira, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nascimento, A. S.</span><span> </span><span class="NLM_article-title">Structure-based identification of novel PPAR gamma ligands</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">5795</span><span class="NLM_x">–</span> <span class="NLM_lpage">5802</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2013.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2013.09.010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=5795-5802&author=F.+M.+da+Silvaauthor=J.+C.+dos+Santosauthor=J.+L.+Camposauthor=A.+C.+Mafudauthor=I.+Polikarpovauthor=A.+C.+Figueiraauthor=A.+S.+Nascimento&title=Structure-based+identification+of+novel+PPAR+gamma+ligands&doi=10.1016%2Fj.bmcl.2013.09.010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.09.010%26sid%3Dliteratum%253Aachs%26aulast%3Dda%2BSilva%26aufirst%3DF.%2BM.%26aulast%3Ddos%2BSantos%26aufirst%3DJ.%2BC.%26aulast%3DCampos%26aufirst%3DJ.%2BL.%26aulast%3DMafud%26aufirst%3DA.%2BC.%26aulast%3DPolikarpov%26aufirst%3DI.%26aulast%3DFigueira%26aufirst%3DA.%2BC.%26aulast%3DNascimento%26aufirst%3DA.%2BS.%26atitle%3DStructure-based%2520identification%2520of%2520novel%2520PPAR%2520gamma%2520ligands%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D5795%26epage%3D5802%26doi%3D10.1016%2Fj.bmcl.2013.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Lee, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baday, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guibao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, J. J.</span><span> </span><span class="NLM_article-title">Structure-based discovery of novel small molecule Wnt signaling inhibitors by targeting the cysteine-rich domain of frizzled</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">290</span><span class="NLM_x">, </span> <span class="NLM_fpage">30596</span><span class="NLM_x">–</span> <span class="NLM_lpage">30606</span><span class="refDoi"> DOI: 10.1074/jbc.M115.673202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1074%2Fjbc.M115.673202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26504084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVGhtrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=30596-30606&author=H.+J.+Leeauthor=J.+Baoauthor=A.+Millerauthor=C.+Zhangauthor=J.+Wuauthor=Y.+C.+Badayauthor=C.+Guibaoauthor=L.+Liauthor=D.+Wuauthor=J.+J.+Zheng&title=Structure-based+discovery+of+novel+small+molecule+Wnt+signaling+inhibitors+by+targeting+the+cysteine-rich+domain+of+frizzled&doi=10.1074%2Fjbc.M115.673202"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of novel small molecule Wnt signaling inhibitors by targeting the cysteine-rich domain of Frizzled</span></div><div class="casAuthors">Lee, Ho-Jin; Bao, Ju; Miller, Ami; Zhang, Chi; Wu, Jibo; Baday, Yiressy C.; Guibao, Cristina; Li, Lin; Wu, Dianqing; Zheng, Jie J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">30596-30606</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Frizzled is the earliest discovered glycosylated Wnt protein receptor and is crit. for the initiation of Wnt signaling.  Antagonizing Frizzled is effective in inhibiting the growth of multiple tumor types.  The extracellular N-terminus of Frizzled contains a conserved Cys-rich domain that directly interacts with Wnt ligands.  Structure-based virtual screening and cell-based assays were used to identify 5 small mols. that could inhibit canonical Wnt signaling and have low IC50 values in the micromolar range.  NMR expts. confirmed that these compds. specifically bound to the Wnt binding site on the Frizzled8 Cys-rich domain with submicromolar dissocn. consts.  This study confirmed the feasibility of targeting the Frizzled Cys-rich domain as an effective way of regulating canonical Wnt signaling.  These small mols. could be further optimized into more potent therapeutic agents for regulating abnormal Wnt signaling by targeting Frizzled.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYJRelDBXuWLVg90H21EOLACvtfcHk0libSaqVQuYUig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVGhtrrP&md5=4ceed0d6226dcd648a925e5d6392caa2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.673202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.673202%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%2BJ.%26aulast%3DBao%26aufirst%3DJ.%26aulast%3DMiller%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DBaday%26aufirst%3DY.%2BC.%26aulast%3DGuibao%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DZheng%26aufirst%3DJ.%2BJ.%26atitle%3DStructure-based%2520discovery%2520of%2520novel%2520small%2520molecule%2520Wnt%2520signaling%2520inhibitors%2520by%2520targeting%2520the%2520cysteine-rich%2520domain%2520of%2520frizzled%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D30596%26epage%3D30606%26doi%3D10.1074%2Fjbc.M115.673202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Harikishore, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoon, H. S.</span><span> </span><span class="NLM_article-title">Combination of pharmacophore hypothesis and molecular docking to identify novel inhibitors of HCV NS5B polymerase</span> <span class="citation_source-journal">Mol. Diversity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">529</span><span class="NLM_x">–</span> <span class="NLM_lpage">539</span><span class="refDoi"> DOI: 10.1007/s11030-015-9591-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs11030-015-9591-5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2015&pages=529-539&author=A.+Harikishoreauthor=E.+Liauthor=J.+J.+Leeauthor=N.+J.+Choauthor=H.+S.+Yoon&title=Combination+of+pharmacophore+hypothesis+and+molecular+docking+to+identify+novel+inhibitors+of+HCV+NS5B+polymerase&doi=10.1007%2Fs11030-015-9591-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1007%2Fs11030-015-9591-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11030-015-9591-5%26sid%3Dliteratum%253Aachs%26aulast%3DHarikishore%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DLee%26aufirst%3DJ.%2BJ.%26aulast%3DCho%26aufirst%3DN.%2BJ.%26aulast%3DYoon%26aufirst%3DH.%2BS.%26atitle%3DCombination%2520of%2520pharmacophore%2520hypothesis%2520and%2520molecular%2520docking%2520to%2520identify%2520novel%2520inhibitors%2520of%2520HCV%2520NS5B%2520polymerase%26jtitle%3DMol.%2520Diversity%26date%3D2015%26volume%3D19%26spage%3D529%26epage%3D539%26doi%3D10.1007%2Fs11030-015-9591-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mishra, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schiltz, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Makanji, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheidt, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazar, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodruff, T. K.</span><span> </span><span class="NLM_article-title">Virtual high-throughput screening to identify novel activin antagonists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">5637</span><span class="NLM_x">–</span> <span class="NLM_lpage">5648</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00753</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00753" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5637-5648&author=J.+Zhuauthor=R.+K.+Mishraauthor=G.+E.+Schiltzauthor=Y.+Makanjiauthor=K.+A.+Scheidtauthor=A.+P.+Mazarauthor=T.+K.+Woodruff&title=Virtual+high-throughput+screening+to+identify+novel+activin+antagonists&doi=10.1021%2Facs.jmedchem.5b00753"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00753%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DMishra%26aufirst%3DR.%2BK.%26aulast%3DSchiltz%26aufirst%3DG.%2BE.%26aulast%3DMakanji%26aufirst%3DY.%26aulast%3DScheidt%26aufirst%3DK.%2BA.%26aulast%3DMazar%26aufirst%3DA.%2BP.%26aulast%3DWoodruff%26aufirst%3DT.%2BK.%26atitle%3DVirtual%2520high-throughput%2520screening%2520to%2520identify%2520novel%2520activin%2520antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5637%26epage%3D5648%26doi%3D10.1021%2Facs.jmedchem.5b00753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Soni, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suryadevara, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sriram, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Consortium, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nandicoori, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yogeeswari, P.</span><span> </span><span class="NLM_article-title">Structure-based design of diverse inhibitors of Mycobacterium tuberculosis N-acetylglucosamine-1-phosphate uridyltransferase: combined molecular docking, dynamic simulation, and biological activity</span> <span class="citation_source-journal">J. Mol. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">174</span><span class="refDoi"> DOI: 10.1007/s00894-015-2704-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs00894-015-2704-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26078037" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BC2MbktVSktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=174&author=V.+Soniauthor=P.+Suryadevaraauthor=D.+Sriramauthor=O.+Consortiumauthor=S.+Kumarauthor=V.+K.+Nandicooriauthor=P.+Yogeeswari&title=Structure-based+design+of+diverse+inhibitors+of+Mycobacterium+tuberculosis+N-acetylglucosamine-1-phosphate+uridyltransferase%3A+combined+molecular+docking%2C+dynamic+simulation%2C+and+biological+activity&doi=10.1007%2Fs00894-015-2704-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of diverse inhibitors of Mycobacterium tuberculosis N-acetylglucosamine-1-phosphate uridyltransferase: combined molecular docking, dynamic simulation, and biological activity</span></div><div class="casAuthors">Soni Vijay; Suryadevara Priyanka; Sriram Dharmarajan; Kumar Santhosh; Nandicoori Vinay Kumar; Yogeeswari Perumal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">174</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Persistent nature of Mycobacterium tuberculosis is one of the major factors which make the drug development process monotonous against this organism.  The highly lipophilic cell wall, which constituting outer mycolic acid and inner peptidoglycan layers, acts as a barrier for the drugs to enter the bacteria.  The rigidity of the cell wall is imparted by the peptidoglycan layer, which is covalently linked to mycolic acid by arabinogalactan.  Uridine diphosphate-N-acetylglucosamine (UDP-GlcNAc) serves as the starting material in the biosynthesis of this peptidoglycan layers.  This UDP-GlcNAc is synthesized by N-acetylglucosamine-1-phosphate uridyltransferase (GlmU(Mtb)), a bi-functional enzyme with two functional sites, acetyltransferase site and uridyltransferase site.  Here, we report design and screening of nine inhibitors against UTP and NAcGlc-1-P of uridyltransferase active site of glmU(Mtb).  Compound 4 was showing good inhibition and was selected for further analysis.  The isothermal titration calorimetry (ITC) experiments showed the binding energy pattern of compound 4 to the uridyltransferase active site is similar to that of substrate UTP.  In silico molecular dynamics (MD) simulation studies, for compound 4, carried out for 10 ns showed the protein-compound complex to be stable throughout the simulation with relative rmsd in acceptable range.  Hence, these compounds can serve as a starting point in the drug discovery processes against Mycobacterium tuberculosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTxgQSOJ7Hiq-Lt9HypzAgQfW6udTcc2eZ7AXeLk-N_Vbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MbktVSktQ%253D%253D&md5=f7a7f8e427f2d5cfe01c8b63f95a4369</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1007%2Fs00894-015-2704-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00894-015-2704-3%26sid%3Dliteratum%253Aachs%26aulast%3DSoni%26aufirst%3DV.%26aulast%3DSuryadevara%26aufirst%3DP.%26aulast%3DSriram%26aufirst%3DD.%26aulast%3DConsortium%26aufirst%3DO.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DNandicoori%26aufirst%3DV.%2BK.%26aulast%3DYogeeswari%26aufirst%3DP.%26atitle%3DStructure-based%2520design%2520of%2520diverse%2520inhibitors%2520of%2520Mycobacterium%2520tuberculosis%2520N-acetylglucosamine-1-phosphate%2520uridyltransferase%253A%2520combined%2520molecular%2520docking%252C%2520dynamic%2520simulation%252C%2520and%2520biological%2520activity%26jtitle%3DJ.%2520Mol.%2520Model.%26date%3D2015%26volume%3D21%26spage%3D174%26doi%3D10.1007%2Fs00894-015-2704-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Ruggeri, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drinkwater, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sivaraman, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamert, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGowan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paiardini, A.</span><span> </span><span class="NLM_article-title">Identification and validation of a potent dual inhibitor of the P. falciparum M1 and M17 aminopeptidases using virtual screening</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0138957</span><span class="refDoi"> DOI: 10.1371/journal.pone.0138957</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pone.0138957" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0138957&author=C.+Ruggeriauthor=N.+Drinkwaterauthor=K.+K.+Sivaramanauthor=R.+S.+Bamertauthor=S.+McGowanauthor=A.+Paiardini&title=Identification+and+validation+of+a+potent+dual+inhibitor+of+the+P.+falciparum+M1+and+M17+aminopeptidases+using+virtual+screening&doi=10.1371%2Fjournal.pone.0138957"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0138957&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0138957%26sid%3Dliteratum%253Aachs%26aulast%3DRuggeri%26aufirst%3DC.%26aulast%3DDrinkwater%26aufirst%3DN.%26aulast%3DSivaraman%26aufirst%3DK.%2BK.%26aulast%3DBamert%26aufirst%3DR.%2BS.%26aulast%3DMcGowan%26aufirst%3DS.%26aulast%3DPaiardini%26aufirst%3DA.%26atitle%3DIdentification%2520and%2520validation%2520of%2520a%2520potent%2520dual%2520inhibitor%2520of%2520the%2520P.%2520falciparum%2520M1%2520and%2520M17%2520aminopeptidases%2520using%2520virtual%2520screening%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0138957%26doi%3D10.1371%2Fjournal.pone.0138957" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Schmidt, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernat, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brox, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tschammer, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolb, P.</span><span> </span><span class="NLM_article-title">Identifying modulators of CXC receptors 3 and 4 with tailored selectivity using multi-target docking</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">715</span><span class="NLM_x">–</span> <span class="NLM_lpage">724</span><span class="refDoi"> DOI: 10.1021/cb500577j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500577j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFWqsb%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=715-724&author=D.+Schmidtauthor=V.+Bernatauthor=R.+Broxauthor=N.+Tschammerauthor=P.+Kolb&title=Identifying+modulators+of+CXC+receptors+3+and+4+with+tailored+selectivity+using+multi-target+docking&doi=10.1021%2Fcb500577j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Modulators of CXC Receptors 3 and 4 with Tailored Selectivity Using Multi-Target Docking</span></div><div class="casAuthors">Schmidt, Denis; Bernat, Viachaslau; Brox, Regine; Tschammer, Nuska; Kolb, Peter</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">715-724</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The G protein-coupled receptors of the C-X-C subfamily form a group among the chemokine receptors whose endogenous ligands are peptides with a common Cys-X-Cys motif.  The CXC chemokine receptors 3 and 4 (CXCR3, CXCR4), which are investigated in this study, are linked to severe diseases such as cancer, multiple sclerosis, and HIV infections.  Of particular interest, this receptor pair potentially forms a target for a polypharmacol. drug treatment.  Considering known ligands from public databases, such dual binders have not been identified yet.  We therefore applied large-scale docking to the structure of CXCR4 and a homol. model of CXCR3 with the goal to predict such dual binders, as well as compds. selective for either one of the receptors.  Using signaling and biochem. assays, we showed that more than 50% of these predictions were correct in each category, yielding ligands with excellent binding efficiencies.  These results highlight that docking is a suitable tool for the identification of ligands with tailored binding profiles to GPCRs, even when using homol. models.  More importantly, we present novel CXCR3-CXCR4 dual modulators that might pave the road to understanding the mechanisms of polypharmacol. inhibition of these receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6Px6d1pbV3rVg90H21EOLACvtfcHk0licGXmCgbTueg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFWqsb%252FM&md5=9259d56e4c28e68360f3ac214ea62ea4</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fcb500577j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500577j%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DD.%26aulast%3DBernat%26aufirst%3DV.%26aulast%3DBrox%26aufirst%3DR.%26aulast%3DTschammer%26aufirst%3DN.%26aulast%3DKolb%26aufirst%3DP.%26atitle%3DIdentifying%2520modulators%2520of%2520CXC%2520receptors%25203%2520and%25204%2520with%2520tailored%2520selectivity%2520using%2520multi-target%2520docking%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D715%26epage%3D724%26doi%3D10.1021%2Fcb500577j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Perryman, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ekins, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freundlich, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonge, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">A virtual screen discovers novel, fragment-sized inhibitors of Mycobacterium tuberculosis InhA</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">645</span><span class="NLM_x">–</span> <span class="NLM_lpage">659</span><span class="refDoi"> DOI: 10.1021/ci500672v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500672v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVehsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=645-659&author=A.+L.+Perrymanauthor=W.+Yuauthor=X.+Wangauthor=S.+Ekinsauthor=S.+Forliauthor=S.+G.+Liauthor=J.+S.+Freundlichauthor=P.+J.+Tongeauthor=A.+J.+Olson&title=A+virtual+screen+discovers+novel%2C+fragment-sized+inhibitors+of+Mycobacterium+tuberculosis+InhA&doi=10.1021%2Fci500672v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">A Virtual Screen Discovers Novel, Fragment-Sized Inhibitors of Mycobacterium tuberculosis InhA</span></div><div class="casAuthors">Perryman, Alexander L.; Yu, Weixuan; Wang, Xin; Ekins, Sean; Forli, Stefano; Li, Shao-Gang; Freundlich, Joel S.; Tonge, Peter J.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">645-659</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Isoniazid (INH) is usually administered to treat latent Mycobacterium tuberculosis (Mtb) infections and is used in combination therapy to treat active tuberculosis (TB).  Unfortunately, resistance to this drug is hampering its clin. effectiveness.  INH is a prodrug that must be activated by Mtb catalase-peroxidase (KatG) before it can inhibit InhA (Mtb enoyl-acyl-carrier-protein reductase).  Isoniazid-resistant cases of TB found in clin. settings usually involve mutations in or deletion of katG, which abrogate INH activation.  Compds. that inhibit InhA without requiring prior activation by KatG would not be affected by this resistance mechanism and hence would display continued potency against these drug-resistant isolates of Mtb.  Virtual screening expts. vs. InhA in the GO Fight Against Malaria (GO FAM) project were designed to discover new scaffolds that display base-stacking interactions with the NAD cofactor.  GO FAM expts. included targets from other pathogens, including Mtb, when they had structural similarity to a malaria target.  Eight of the 16 sol. compds. identified by docking against InhA plus visual inspection were modest inhibitors and did not require prior activation by KatG.  The best two inhibitors discovered are both fragment-sized compds. and displayed Ki values of 54 and 59 μM, resp.  Importantly, the novel inhibitors discovered have low structural similarity to known InhA inhibitors and thus help expand the no. of chemotypes on which future medicinal chem. efforts can be focused.  These new fragment hits could eventually help advance the fight against INH-resistant Mtb strains, which pose a significant global health threat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9fifQHkgzX7Vg90H21EOLACvtfcHk0licGXmCgbTueg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVehsrc%253D&md5=f68a7af63dca612e3017260c32593858</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fci500672v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500672v%26sid%3Dliteratum%253Aachs%26aulast%3DPerryman%26aufirst%3DA.%2BL.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DEkins%26aufirst%3DS.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DS.%2BG.%26aulast%3DFreundlich%26aufirst%3DJ.%2BS.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DA%2520virtual%2520screen%2520discovers%2520novel%252C%2520fragment-sized%2520inhibitors%2520of%2520Mycobacterium%2520tuberculosis%2520InhA%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D645%26epage%3D659%26doi%3D10.1021%2Fci500672v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span> </span><span class="NLM_article-title">Identification of novel small molecules as inhibitors of hepatitis C virus by structure-based virtual screening</span> <span class="citation_source-journal">Int. J. Mol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">22845</span><span class="NLM_x">–</span> <span class="NLM_lpage">22856</span><span class="refDoi"> DOI: 10.3390/ijms141122845</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.3390%2Fijms141122845" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24264035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltlOgur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=22845-22856&author=J.+Liauthor=X.+Liuauthor=S.+Liauthor=Y.+Wangauthor=N.+Zhouauthor=C.+Luoauthor=X.+Luoauthor=M.+Zhengauthor=H.+Jiangauthor=K.+Chen&title=Identification+of+novel+small+molecules+as+inhibitors+of+hepatitis+C+virus+by+structure-based+virtual+screening&doi=10.3390%2Fijms141122845"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel small molecules as inhibitors of hepatitis C virus by structure-based virtual screening</span></div><div class="casAuthors">Li, Jing; Liu, Xian; Li, Shanshan; Wang, Yulan; Zhou, Nannan; Luo, Cheng; Luo, Xiaomin; Zheng, Mingyue; Jiang, Hualiang; Chen, Kaixian</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">22845-22856, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Hepatitis C virus (HCV) NS3/NS4A serine protease is essential for viral replication, which is regarded as a promising drug target for developing direct-acting anti-HCV agents.  In this study, sixteen novel compds. with cell-based HCV replicon activity ranging from 3.0 to 28.2 μM (IC50) were successfully identified by means of structure-based virtual screening.  Compd. 5 and compd. 11, with an IC50 of 3.0 μM and 5.1 μM, resp., are the two most potent mols. with low cytotoxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYDkmJf8CQ3bVg90H21EOLACvtfcHk0licGXmCgbTueg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltlOgur0%253D&md5=eab2e3f26ed0b67aeb3e60cc74197488</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.3390%2Fijms141122845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms141122845%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DN.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26atitle%3DIdentification%2520of%2520novel%2520small%2520molecules%2520as%2520inhibitors%2520of%2520hepatitis%2520C%2520virus%2520by%2520structure-based%2520virtual%2520screening%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2013%26volume%3D14%26spage%3D22845%26epage%3D22856%26doi%3D10.3390%2Fijms141122845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Soelaiman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, B. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mrksich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, W. J.</span><span> </span><span class="NLM_article-title">Structure-based inhibitor discovery against adenylyl cyclase toxins from pathogenic bacteria that cause anthrax and whooping cough</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">25990</span><span class="NLM_x">–</span> <span class="NLM_lpage">25997</span><span class="refDoi"> DOI: 10.1074/jbc.M301232200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1074%2Fjbc.M301232200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=12676933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlt1Cltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=25990-25997&author=S.+Soelaimanauthor=B.+Q.+Weiauthor=P.+Bergsonauthor=Y.+S.+Leeauthor=Y.+Shenauthor=M.+Mrksichauthor=B.+K.+Shoichetauthor=W.+J.+Tang&title=Structure-based+inhibitor+discovery+against+adenylyl+cyclase+toxins+from+pathogenic+bacteria+that+cause+anthrax+and+whooping+cough&doi=10.1074%2Fjbc.M301232200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based Inhibitor Discovery against Adenylyl Cyclase Toxins from Pathogenic Bacteria That Cause Anthrax and Whooping Cough</span></div><div class="casAuthors">Soelaiman, Sandriyana; Wei, Binqing Q.; Bergson, Pamela; Lee, Young-Sam; Shen, Yuequan; Mrksich, Milan; Shoichet, Brian K.; Tang, Wei-Jen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">25990-25997</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Edema factor (EF) and CyaA are adenylyl cyclase toxins secreted by pathogenic bacteria that cause anthrax and whooping cough, resp.  Using the structure of the catalytic site of EF, we screened a data base of com. available, small mol. wt. chems. for those that could specifically inhibit adenylyl cyclase activity of EF.  From 24 compds. tested, we have identified one quinazoline compd., Et 5-aminopyrazolo[1,5-a]quinazoline-3-carboxylate, that specifically inhibits adenylyl cyclase activity of EF and CyaA with ∼20 μM Ki.  This compd. neither affects the activity of host resident adenylyl cyclases type I, II, and V nor exhibits promiscuous inhibition.  The compd. is a competitive inhibitor, consistent with the prediction that it binds to the adenine portion of the ATP binding site on EF.  EF is activated by the host calcium sensor, calmodulin.  Surface plasmon resonance spectroscopic anal. shows that this compd. does not affect the binding of calmodulin to EF.  This compd. is dissimilar from a previously described, non-nucleoside inhibitor of host adenylyl cyclase.  It may serve as a lead to design antitoxins to address the role of adenylyl cyclase toxins in bacterial pathogenesis and to fight against anthrax and whooping cough.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFnvSIcMfQNbVg90H21EOLACvtfcHk0liTndu1RIa_lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlt1Cltro%253D&md5=2b0dbb955395039f3b50c78ca947036f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M301232200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M301232200%26sid%3Dliteratum%253Aachs%26aulast%3DSoelaiman%26aufirst%3DS.%26aulast%3DWei%26aufirst%3DB.%2BQ.%26aulast%3DBergson%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DY.%2BS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DMrksich%26aufirst%3DM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DTang%26aufirst%3DW.%2BJ.%26atitle%3DStructure-based%2520inhibitor%2520discovery%2520against%2520adenylyl%2520cyclase%2520toxins%2520from%2520pathogenic%2520bacteria%2520that%2520cause%2520anthrax%2520and%2520whooping%2520cough%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D25990%26epage%3D25997%26doi%3D10.1074%2Fjbc.M301232200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Iwata, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arisawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamada, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kita, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizutani, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomioka, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyamoto, S.</span><span> </span><span class="NLM_article-title">Discovery of novel aldose reductase inhibitors using a protein structure-based approach: 3D-database search followed by design and synthesis</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1718</span><span class="NLM_x">–</span> <span class="NLM_lpage">1728</span><span class="refDoi"> DOI: 10.1021/jm000483h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm000483h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3MXivFynu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=1718-1728&author=Y.+Iwataauthor=M.+Arisawaauthor=R.+Hamadaauthor=Y.+Kitaauthor=M.+Y.+Mizutaniauthor=N.+Tomiokaauthor=A.+Itaiauthor=S.+Miyamoto&title=Discovery+of+novel+aldose+reductase+inhibitors+using+a+protein+structure-based+approach%3A+3D-database+search+followed+by+design+and+synthesis&doi=10.1021%2Fjm000483h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Aldose Reductase Inhibitors Using a Protein Structure-Based Approach: 3D-Database Search Followed by Design and Synthesis</span></div><div class="casAuthors">Iwata, Yoriko; Arisawa, Mitsuhiro; Hamada, Ryuji; Kita, Yasuyuki; Mizutani, Miho Y.; Tomioka, Nobuo; Itai, Akiko; Miyamoto, Shuichi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1718-1728</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aldose reductase (AR) has been implicated in the etiol. of diabetic complications.  Due to the limited no. of currently available drugs for the treatment of diabetic complications, we have carried out structure-based drug design and synthesis to find new types of AR inhibitors.  With the ADAM&EVE program, a three-dimensional database (ACD3D) was searched using the ligand binding site of the AR crystal structure.  Out of 179 compds. selected through this search followed by visual inspection, 36 compds. were purchased and subjected to a biol. assay.  Ten compds. showed more than 40% inhibition of AR at a 15 μg/mL concn.  In a subsequent lead optimization, a series of analogs of the most active compd. were synthesized based on the docking mode derived by ADAM&EVE.  Many of these congeners exhibited higher activities compared to the mother compd. Indeed, the most potent, synthesized compd. showed a ∼20-fold increase in inhibitory activity (IC50 = 0.21 vs. 4.3 μM).  Furthermore, a hydrophobic subsite was newly inferred, which would be useful for the design of inhibitors with improved affinity for AR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6c8cdbn1oQLVg90H21EOLACvtfcHk0liTndu1RIa_lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXivFynu7g%253D&md5=6eff99eefff1e4c4531df078969c73d7</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm000483h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000483h%26sid%3Dliteratum%253Aachs%26aulast%3DIwata%26aufirst%3DY.%26aulast%3DArisawa%26aufirst%3DM.%26aulast%3DHamada%26aufirst%3DR.%26aulast%3DKita%26aufirst%3DY.%26aulast%3DMizutani%26aufirst%3DM.%2BY.%26aulast%3DTomioka%26aufirst%3DN.%26aulast%3DItai%26aufirst%3DA.%26aulast%3DMiyamoto%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520novel%2520aldose%2520reductase%2520inhibitors%2520using%2520a%2520protein%2520structure-based%2520approach%253A%25203D-database%2520search%2520followed%2520by%2520design%2520and%2520synthesis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D1718%26epage%3D1728%26doi%3D10.1021%2Fjm000483h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Enyedy, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ling, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nacro, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomita, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, Y. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roller, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Discovery of small-molecule inhibitors of Bcl-2 through structure-based computer screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">4313</span><span class="NLM_x">–</span> <span class="NLM_lpage">4324</span><span class="refDoi"> DOI: 10.1021/jm010016f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm010016f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnvFaks70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=4313-4324&author=I.+J.+Enyedyauthor=Y.+Lingauthor=K.+Nacroauthor=Y.+Tomitaauthor=X.+Wuauthor=Y.+Caoauthor=R.+Guoauthor=B.+Liauthor=X.+Zhuauthor=Y.+Huangauthor=Y.+Q.+Longauthor=P.+P.+Rollerauthor=D.+Yangauthor=S.+Wang&title=Discovery+of+small-molecule+inhibitors+of+Bcl-2+through+structure-based+computer+screening&doi=10.1021%2Fjm010016f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small-Molecule Inhibitors of Bcl-2 through Structure-Based Computer Screening</span></div><div class="casAuthors">Enyedy, Istvan J.; Ling, Yan; Nacro, Kassoum; Tomita, York; Wu, Xihan; Cao, Yeyu; Guo, Ribo; Li, Bihua; Zhu, Xiaofeng; Huang, Ying; Long, Ya-Qiu; Roller, Peter P.; Yang, Dajun; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4313-4324</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bcl-2 belongs to a growing family of proteins which regulates programmed cell death (apoptosis).  Overexpression of Bcl-2 has been obsd. in 70% of breast cancer, 30-60% of prostate cancer, 80% of B-cell lymphomas, 90% of colorectal adenocarcinomas, and many other forms of cancer. Thereby, Bcl-2 is an attractive new anti-cancer target. Herein, we describe the discovery of novel classes of small-mol. inhibitors targeted at the BH3 binding pocket in Bcl-2.  The three-dimensional (3D) structure of Bcl-2 has been modeled on the basis of a high-resoln. NMR soln. structure of Bcl-XL, which shares a high sequence homol. with Bcl-2.  A structure-based computer screening approach has been employed to search the National Cancer Institute 3D database of 206 876 org. compds. to identify potential Bcl-2 small-mol. inhibitors that bind to the BH3 binding site of Bcl-2.  These potential Bcl-2 small-mol. inhibitors were first tested in an in vitro binding assay for their potency in inhibition of the binding of a Bak BH3 peptide to Bcl-2.  Thirty-five potential inhibitors were tested in this binding assay, and seven of them were found to have a binding affinity (IC50 value) from 1.6 to 14.0 M.  The anti-proliferative activity of these seven active compds. has been tested using a human myeloid leukemia cell line, HL-60, which expresses the highest level of Bcl-2 protein among all the cancer cell lines examd.  The most potent compd. had an IC50 value of 4 μM in inhibition of cell growth using the 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide assay.  Five other compds. had moderate activity in inhibition of cell growth.  The most active compd. was further evaluated for its ability to induce apoptosis in cancer cells.  It was found that this compd. induces apoptosis in cancer cells with high Bcl-2 expression and its potency correlates with the Bcl-2 expression level in cancer cells.  Furthermore, using NMR methods, we conclusively demonstrated that this most active compd. binds to the BH3 binding site in Bcl-XL.  Our results showed that small-mol. inhibitors of Bcl-2 modulate the biol. function of Bcl-2, and induce apoptosis in cancer cells with high Bcl-2 expression, while they have little effect on cancer cells with low or undetectable levels of Bcl-2 expression.  Therefore, this active compd. can be used as a valuable pharmacol. tool to elucidate the function of Bcl-2 and also serves as a novel lead compd. for further design and optimization.  Our results suggest that the structure-based computer screening strategy employed in the study is effective for identifying novel, structurally diverse, nonpeptide small-mol. inhibitors that target the BH3 binding site of Bcl-2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVlg73stNG2LVg90H21EOLACvtfcHk0liTndu1RIa_lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnvFaks70%253D&md5=b02822b211ecab923903ff08d8a219c8</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm010016f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm010016f%26sid%3Dliteratum%253Aachs%26aulast%3DEnyedy%26aufirst%3DI.%2BJ.%26aulast%3DLing%26aufirst%3DY.%26aulast%3DNacro%26aufirst%3DK.%26aulast%3DTomita%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DLong%26aufirst%3DY.%2BQ.%26aulast%3DRoller%26aufirst%3DP.%2BP.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520small-molecule%2520inhibitors%2520of%2520Bcl-2%2520through%2520structure-based%2520computer%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D4313%26epage%3D4324%26doi%3D10.1021%2Fjm010016f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Pang, Y. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollmeyer, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perola, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGrath, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangel, W. F.</span><span> </span><span class="NLM_article-title">Discovery of a new inhibitor lead of adenovirus proteinase: steps toward selective, irreversible inhibitors of cysteine proteinases</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">502</span><span class="NLM_x">, </span> <span class="NLM_fpage">93</span><span class="NLM_x">–</span> <span class="NLM_lpage">97</span><span class="refDoi"> DOI: 10.1016/S0014-5793(01)02672-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2FS0014-5793%2801%2902672-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=502&publication_year=2001&pages=93-97&author=Y.+P.+Pangauthor=K.+Xuauthor=T.+M.+Kollmeyerauthor=E.+Perolaauthor=W.+J.+McGrathauthor=D.+T.+Greenauthor=W.+F.+Mangel&title=Discovery+of+a+new+inhibitor+lead+of+adenovirus+proteinase%3A+steps+toward+selective%2C+irreversible+inhibitors+of+cysteine+proteinases&doi=10.1016%2FS0014-5793%2801%2902672-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2801%2902672-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252801%252902672-2%26sid%3Dliteratum%253Aachs%26aulast%3DPang%26aufirst%3DY.%2BP.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DKollmeyer%26aufirst%3DT.%2BM.%26aulast%3DPerola%26aufirst%3DE.%26aulast%3DMcGrath%26aufirst%3DW.%2BJ.%26aulast%3DGreen%26aufirst%3DD.%2BT.%26aulast%3DMangel%26aufirst%3DW.%2BF.%26atitle%3DDiscovery%2520of%2520a%2520new%2520inhibitor%2520lead%2520of%2520adenovirus%2520proteinase%253A%2520steps%2520toward%2520selective%252C%2520irreversible%2520inhibitors%2520of%2520cysteine%2520proteinases%26jtitle%3DFEBS%2520Lett.%26date%3D2001%26volume%3D502%26spage%3D93%26epage%3D97%26doi%3D10.1016%2FS0014-5793%2801%2902672-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Honma, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aoyama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hashimoto, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machida, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukasawa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikeura, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikuta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki-Takahashi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwasawa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishimura, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morishima, H.</span><span> </span><span class="NLM_article-title">Structure-based generation of a new class of potent Cdk4 inhibitors: new de novo design strategy and library design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">4615</span><span class="NLM_x">–</span> <span class="NLM_lpage">4627</span><span class="refDoi"> DOI: 10.1021/jm0103256</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0103256" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3MXovFagtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=4615-4627&author=T.+Honmaauthor=K.+Hayashiauthor=T.+Aoyamaauthor=N.+Hashimotoauthor=T.+Machidaauthor=K.+Fukasawaauthor=T.+Iwamaauthor=C.+Ikeuraauthor=M.+Ikutaauthor=I.+Suzuki-Takahashiauthor=Y.+Iwasawaauthor=T.+Hayamaauthor=S.+Nishimuraauthor=H.+Morishima&title=Structure-based+generation+of+a+new+class+of+potent+Cdk4+inhibitors%3A+new+de+novo+design+strategy+and+library+design&doi=10.1021%2Fjm0103256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Generation of a New Class of Potent Cdk4 Inhibitors: New de Novo Design Strategy and Library Design</span></div><div class="casAuthors">Honma, Teruki; Hayashi, Kyoko; Aoyama, Tetsuya; Hashimoto, Noriaki; Machida, Takumitsu; Fukasawa, Kazuhiro; Iwama, Toshiharu; Ikeura, Chinatsu; Ikuta, Mari; Suzuki-Takahashi, Ikuko; Iwasawa, Yoshikazu; Hayama, Takashi; Nishimura, Susumu; Morishima, Hajime</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">4615-4627</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As a first step in structure-based design of highly selective and potent Cdk4 inhibitors, we performed structure-based generation of a novel series of Cdk4 inhibitors.  A Cdk4 homol. model was constructed according to x-ray anal. of an activated form of Cdk2.  Using this model, we applied a new de novo design strategy which combined the de novo design program LEGEND with our inhouse structure selection supporting system SEEDS to generate new scaffold candidates.  In this way, four classes of scaffold candidates including diarylurea were identified.  By constructing diarylurea informer libraries based on the structural requirements of Cdk inhibitors in the ATP binding pocket of the Cdk4 model, we were able to identify a potent Cdk4 inhibitor N-(9-oxo-9H-fluoren-4-yl)-N'-pyridin-2-ylurea (I) with IC50 = 0.10 μM, together with preliminary SAR.  We performed a docking study between I and the Cdk4 model and selected a reasonable binding mode which is consistent with the SAR.  Further modification based on the proposed binding mode provided a more potent compd., N-[(9bR)-5-oxo- 2,3,5,9b-tetrahydro-1H-pyrrolo[2,1-a]isoindol-9-yl]-N'-pyridin-2-ylurea (II) with IC50 = 0.042 μM, x-ray anal. of which was accomplished by the soaking method.  The predicted binding mode of I in Cdk4 was validated by x-ray anal. of the Cdk2-II complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvx3J3Bqyg9LVg90H21EOLACvtfcHk0lgzHaQJy7UyVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXovFagtrs%253D&md5=0c4458fa88d41c148a672863b3a6daa0</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm0103256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0103256%26sid%3Dliteratum%253Aachs%26aulast%3DHonma%26aufirst%3DT.%26aulast%3DHayashi%26aufirst%3DK.%26aulast%3DAoyama%26aufirst%3DT.%26aulast%3DHashimoto%26aufirst%3DN.%26aulast%3DMachida%26aufirst%3DT.%26aulast%3DFukasawa%26aufirst%3DK.%26aulast%3DIwama%26aufirst%3DT.%26aulast%3DIkeura%26aufirst%3DC.%26aulast%3DIkuta%26aufirst%3DM.%26aulast%3DSuzuki-Takahashi%26aufirst%3DI.%26aulast%3DIwasawa%26aufirst%3DY.%26aulast%3DHayama%26aufirst%3DT.%26aulast%3DNishimura%26aufirst%3DS.%26aulast%3DMorishima%26aufirst%3DH.%26atitle%3DStructure-based%2520generation%2520of%2520a%2520new%2520class%2520of%2520potent%2520Cdk4%2520inhibitors%253A%2520new%2520de%2520novo%2520design%2520strategy%2520and%2520library%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D4615%26epage%3D4627%26doi%3D10.1021%2Fjm0103256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raaka, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samuels, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">In silico discovery of novel Retinoic Acid Receptor agonist structures</span> <span class="citation_source-journal">BMC Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="refDoi"> DOI: 10.1186/1472-6807-1-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1186%2F1472-6807-1-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=11405897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BD3srgs1Crsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2001&pages=1&author=M.+Schapiraauthor=B.+M.+Raakaauthor=H.+H.+Samuelsauthor=R.+Abagyan&title=In+silico+discovery+of+novel+Retinoic+Acid+Receptor+agonist+structures&doi=10.1186%2F1472-6807-1-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">In silico discovery of novel retinoic acid receptor agonist structures</span></div><div class="casAuthors">Schapira M; Raaka B M; Samuels H H; Abagyan R</div><div class="citationInfo"><span class="NLM_cas:title">BMC structural biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Several Retinoic Acid Receptors (RAR) agonists have therapeutic activity against a variety of cancer types; however, unacceptable toxicity profiles have hindered the development of drugs.  RAR agonists presenting novel structural and chemical features could therefore open new avenues for the discovery of leads against breast, lung and prostate cancer or leukemia.  RESULTS:  We have analysed the induced fit of the active site residues upon binding of a known ligand.  The derived binding site models were used to dock over 150,000 molecules in silico (or virtually) to the structure of the receptor with the Internal Coordinates Mechanics (ICM) program.  Thirty ligand candidates were tested in vitro.  CONCLUSIONS:  Two novel agonists resulting from the predicted receptor model were active at 50 nM.  One of them displays novel structural features which may translate into the development of new ligands for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRGuB_7wOJu3Wx8TEN9K18QfW6udTcc2eb59q70MXm6Sbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3srgs1Crsg%253D%253D&md5=ea9be764e620d3e7823f0d183005f4d3</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1186%2F1472-6807-1-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1472-6807-1-1%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DRaaka%26aufirst%3DB.%2BM.%26aulast%3DSamuels%26aufirst%3DH.%2BH.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DIn%2520silico%2520discovery%2520of%2520novel%2520Retinoic%2520Acid%2520Receptor%2520agonist%2520structures%26jtitle%3DBMC%2520Struct.%2520Biol.%26date%3D2001%26volume%3D1%26spage%3D1%26doi%3D10.1186%2F1472-6807-1-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Freymann, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wenck, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engel, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Focia, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eakin, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craig, S. P.</span><span> </span><span class="NLM_article-title">Efficient identification of inhibitors targeting the closed active site conformation of the HPRT from Trypanosoma cruzi</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">957</span><span class="NLM_x">–</span> <span class="NLM_lpage">968</span><span class="refDoi"> DOI: 10.1016/S1074-5521(00)00045-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2FS1074-5521%2800%2900045-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=11137818" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltFegtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2000&pages=957-968&author=D.+M.+Freymannauthor=M.+A.+Wenckauthor=J.+C.+Engelauthor=J.+Fengauthor=P.+J.+Fociaauthor=A.+E.+Eakinauthor=S.+P.+Craig&title=Efficient+identification+of+inhibitors+targeting+the+closed+active+site+conformation+of+the+HPRT+from+Trypanosoma+cruzi&doi=10.1016%2FS1074-5521%2800%2900045-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient identification of inhibitors targeting the closed active site conformation of the HPRT from Trypanosoma cruzi</span></div><div class="casAuthors">Freymann, Douglas M.; Wenck, Mary Anne; Engel, Juan C.; Feng, Jun; Focia, Pamela J.; Eakin, Ann E.; Craig, Sydney P, III</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">957-968</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Background: Currently, only two drugs are recommended for treatment of infection with Trypanosoma cruzi, the etiol. agent of Chagas' disease.  These compds. kill the trypomastigote forms of the parasite circulating in the bloodstream, but are relatively ineffective against the intracellular stage of the parasite life cycle.  Neither drug is approved by the FDA for use in the US.  The hypoxanthine phosphoribosyltransferase (HPRT) from T. cruzi is a possible new target for antiparasite chemotherapy.  The crystal structure of the HPRT in a conformation approximating the transition state reveals a closed active site that provides a well-defined target for computational structure-based drug discovery.  Results: A flexible ligand docking program incorporating a desolvation correction was used to screen the Available Chems. Directory for inhibitors targeted to the closed conformation of the trypanosomal HPRT.  Of 22 potential inhibitors identified, acquired and tested, 16 yielded Ki's between 0.5 and 17 μM vs. the substrate phosphoribosylpyrophosphate.  Surprisingly, three of eight compds. tested were effective in inhibiting the growth of parasites in infected mammalian cells.  Conclusions: This structure-based docking method provided a remarkably efficient path for the identification of inhibitors targeting the closed conformation of the trypanosomal HPRT.  The inhibition consts. of the lead inhibitors identified are unusually favorable, and the trypanostatic activity of three of the compds. in cell culture suggests that they may provide useful starting points for drug design for the treatment of Chagas' disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBVYimL73xQLVg90H21EOLACvtfcHk0lhJZdCD_rN-6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltFegtQ%253D%253D&md5=da1b73a30a1ebc176191e88eaeffb245</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2800%2900045-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252800%252900045-4%26sid%3Dliteratum%253Aachs%26aulast%3DFreymann%26aufirst%3DD.%2BM.%26aulast%3DWenck%26aufirst%3DM.%2BA.%26aulast%3DEngel%26aufirst%3DJ.%2BC.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DFocia%26aufirst%3DP.%2BJ.%26aulast%3DEakin%26aufirst%3DA.%2BE.%26aulast%3DCraig%26aufirst%3DS.%2BP.%26atitle%3DEfficient%2520identification%2520of%2520inhibitors%2520targeting%2520the%2520closed%2520active%2520site%2520conformation%2520of%2520the%2520HPRT%2520from%2520Trypanosoma%2520cruzi%26jtitle%3DChem.%2520Biol.%26date%3D2000%26volume%3D7%26spage%3D957%26epage%3D968%26doi%3D10.1016%2FS1074-5521%2800%2900045-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Paiva, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanderwall, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blanchard, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozarich, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williamson, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, T. M.</span><span> </span><span class="NLM_article-title">Inhibitors of dihydrodipicolinate reductase, a key enzyme of the diaminopimelate pathway of Mycobacterium tuberculosis</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Protein Struct. Mol. Enzymol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">1545</span><span class="NLM_x">, </span> <span class="NLM_fpage">67</span><span class="NLM_x">–</span> <span class="NLM_lpage">77</span><span class="refDoi"> DOI: 10.1016/S0167-4838(00)00262-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2FS0167-4838%2800%2900262-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=11342032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3MXpsFKhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1545&publication_year=2001&pages=67-77&author=A.+M.+Paivaauthor=D.+E.+Vanderwallauthor=J.+S.+Blanchardauthor=J.+W.+Kozarichauthor=J.+M.+Williamsonauthor=T.+M.+Kelly&title=Inhibitors+of+dihydrodipicolinate+reductase%2C+a+key+enzyme+of+the+diaminopimelate+pathway+of+Mycobacterium+tuberculosis&doi=10.1016%2FS0167-4838%2800%2900262-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of dihydrodipicolinate reductase, a key enzyme of the diaminopimelate pathway of Mycobacterium tuberculosis</span></div><div class="casAuthors">Paiva, A. M.; Vanderwall, D. E.; Blanchard, J. S.; Kozarich, J. W.; Williamson, J. M.; Kelly, T. M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Protein Structure and Molecular Enzymology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">1545</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">67-77</span>CODEN:
                <span class="NLM_cas:coden">BBAEDZ</span>;
        ISSN:<span class="NLM_cas:issn">0167-4838</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Tuberculosis (TB) remains a leading cause of infectious disease in the world today and therapies developed over the last forty years are becoming increasingly ineffective against resistant strains of Mycobacterium tuberculosis.  In an effort to explore new mechanisms for drug development, we have investigated the enzymes of the diaminopimelate biosynthetic pathway as potential targets.  Specifically, dihydrodipicolinate reductase, the essential gene product of dapB, was screened for novel inhibitors.  Inhibitors were identified both by a mol. modeling approach which utilized the available crystal structure of the enzyme with an inhibitor bound at the active site as well as by more conventional screening strategies.  The resulting compds. contain a no. of structural motifs and were all competitive with respect to the DHDP substrate.  The Ki values for the inhibitors range from 10 to 90 μM. The mol. modeling approach was very effective in identifying novel inhibitors of the enzyme.  These compds. were obtained at a higher frequency based on the no. of compds. analyzed than those inhibitors discovered via conventional screening.  However, conventional screening proved beneficial in identifying compds. with greater structural diversity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmPtGdeRnJ2LVg90H21EOLACvtfcHk0lhJZdCD_rN-6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXpsFKhtw%253D%253D&md5=26da0a9a71513d545e4c45ae4b48b853</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2FS0167-4838%2800%2900262-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0167-4838%252800%252900262-4%26sid%3Dliteratum%253Aachs%26aulast%3DPaiva%26aufirst%3DA.%2BM.%26aulast%3DVanderwall%26aufirst%3DD.%2BE.%26aulast%3DBlanchard%26aufirst%3DJ.%2BS.%26aulast%3DKozarich%26aufirst%3DJ.%2BW.%26aulast%3DWilliamson%26aufirst%3DJ.%2BM.%26aulast%3DKelly%26aufirst%3DT.%2BM.%26atitle%3DInhibitors%2520of%2520dihydrodipicolinate%2520reductase%252C%2520a%2520key%2520enzyme%2520of%2520the%2520diaminopimelate%2520pathway%2520of%2520Mycobacterium%2520tuberculosis%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Protein%2520Struct.%2520Mol.%2520Enzymol.%26date%3D2001%26volume%3D1545%26spage%3D67%26epage%3D77%26doi%3D10.1016%2FS0167-4838%2800%2900262-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Ragno, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Massa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cerbara, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bottoni, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scatena, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jesacher, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loidl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span> </span><span class="NLM_article-title">3-(4-Aroyl-1-methyl-1H-pyrrol-2-yl)-N-hydroxy-2-propenamides as a new class of synthetic histone deacetylase inhibitors. 3. Discovery of novel lead compounds through structure-based drug design and docking studies</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1351</span><span class="NLM_x">–</span> <span class="NLM_lpage">1359</span><span class="refDoi"> DOI: 10.1021/jm031036f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm031036f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1351-1359&author=R.+Ragnoauthor=A.+Maiauthor=S.+Massaauthor=I.+Cerbaraauthor=S.+Valenteauthor=P.+Bottoniauthor=R.+Scatenaauthor=F.+Jesacherauthor=P.+Loidlauthor=G.+Brosch&title=3-%284-Aroyl-1-methyl-1H-pyrrol-2-yl%29-N-hydroxy-2-propenamides+as+a+new+class+of+synthetic+histone+deacetylase+inhibitors.+3.+Discovery+of+novel+lead+compounds+through+structure-based+drug+design+and+docking+studies&doi=10.1021%2Fjm031036f"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm031036f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm031036f%26sid%3Dliteratum%253Aachs%26aulast%3DRagno%26aufirst%3DR.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DMassa%26aufirst%3DS.%26aulast%3DCerbara%26aufirst%3DI.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DBottoni%26aufirst%3DP.%26aulast%3DScatena%26aufirst%3DR.%26aulast%3DJesacher%26aufirst%3DF.%26aulast%3DLoidl%26aufirst%3DP.%26aulast%3DBrosch%26aufirst%3DG.%26atitle%3D3-%25284-Aroyl-1-methyl-1H-pyrrol-2-yl%2529-N-hydroxy-2-propenamides%2520as%2520a%2520new%2520class%2520of%2520synthetic%2520histone%2520deacetylase%2520inhibitors.%25203.%2520Discovery%2520of%2520novel%2520lead%2520compounds%2520through%2520structure-based%2520drug%2520design%2520and%2520docking%2520studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1351%26epage%3D1359%26doi%3D10.1021%2Fjm031036f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Park, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hwang, K. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oh, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K.</span><span> </span><span class="NLM_article-title">Discovery of novel alpha-glucosidase inhibitors based on the virtual screening with the homology-modeled protein structure</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">284</span><span class="NLM_x">–</span> <span class="NLM_lpage">292</span><span class="refDoi"> DOI: 10.1016/j.bmc.2007.09.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmc.2007.09.036" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=284-292&author=H.+Parkauthor=K.+Y.+Hwangauthor=K.+H.+Ohauthor=Y.+H.+Kimauthor=J.+Y.+Leeauthor=K.+Kim&title=Discovery+of+novel+alpha-glucosidase+inhibitors+based+on+the+virtual+screening+with+the+homology-modeled+protein+structure&doi=10.1016%2Fj.bmc.2007.09.036"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2007.09.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2007.09.036%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DH.%26aulast%3DHwang%26aufirst%3DK.%2BY.%26aulast%3DOh%26aufirst%3DK.%2BH.%26aulast%3DKim%26aufirst%3DY.%2BH.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DKim%26aufirst%3DK.%26atitle%3DDiscovery%2520of%2520novel%2520alpha-glucosidase%2520inhibitors%2520based%2520on%2520the%2520virtual%2520screening%2520with%2520the%2520homology-modeled%2520protein%2520structure%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2008%26volume%3D16%26spage%3D284%26epage%3D292%26doi%3D10.1016%2Fj.bmc.2007.09.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Ostrov, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hernandez Prada, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corsino, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finton, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowe, T. C.</span><span> </span><span class="NLM_article-title">Discovery of novel DNA gyrase inhibitors by high-throughput virtual screening</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">3688</span><span class="NLM_x">–</span> <span class="NLM_lpage">3698</span><span class="refDoi"> DOI: 10.1128/AAC.00392-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1128%2FAAC.00392-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=17682095" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGhtbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2007&pages=3688-3698&author=D.+A.+Ostrovauthor=J.+A.+Hernandez+Pradaauthor=P.+E.+Corsinoauthor=K.+A.+Fintonauthor=N.+Leauthor=T.+C.+Rowe&title=Discovery+of+novel+DNA+gyrase+inhibitors+by+high-throughput+virtual+screening&doi=10.1128%2FAAC.00392-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel DNA gyrase inhibitors by high-throughput virtual screening</span></div><div class="casAuthors">Ostrov, David A.; Prada, Jose A. Hernandez; Corsino, Patrick E.; Finton, Kathryn A.; Le, Nhan; Rowe, Thomas C.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3688-3698</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The bacterial type II topoisomerases DNA gyrase and topoisomerase IV are validated targets for clin. useful quinolone antimicrobial drugs.  A significant limitation to widely utilized quinolone inhibitors is the emergence of drug-resistant bacteria due to an altered DNA gyrase.  To address this problem, we have used structure-based mol. docking to identify novel drug-like small mols. that target sites distinct from those targeted by quinolone inhibitors.  A chem. ligand database contg. approx. 140,000 small mols. (mol. wt., <500) was molecularly docked onto two sites of Escherichia coli DNA gyrase targeting (i) a previously unexplored structural pocket formed at the dimer interface of subunit A and (ii) a small region of the ATP binding pocket on subunit B overlapping the site targeted by coumarin and cyclothialidine drugs.  This approach identified several small-mol. compds. that inhibited the DNA supercoiling activity of purified E. coli DNA gyrase.  These compds. are structurally unrelated to previously identified gyrase inhibitors and represent potential scaffolds for the optimization of novel antibacterial agents that act on fluoroquinolone-resistant strains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7z3236rWKm7Vg90H21EOLACvtfcHk0lgg7YX7aYPGsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGhtbnI&md5=4bb6ab8cffb1e923e01d64a49a8ed759</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1128%2FAAC.00392-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00392-07%26sid%3Dliteratum%253Aachs%26aulast%3DOstrov%26aufirst%3DD.%2BA.%26aulast%3DHernandez%2BPrada%26aufirst%3DJ.%2BA.%26aulast%3DCorsino%26aufirst%3DP.%2BE.%26aulast%3DFinton%26aufirst%3DK.%2BA.%26aulast%3DLe%26aufirst%3DN.%26aulast%3DRowe%26aufirst%3DT.%2BC.%26atitle%3DDiscovery%2520of%2520novel%2520DNA%2520gyrase%2520inhibitors%2520by%2520high-throughput%2520virtual%2520screening%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2007%26volume%3D51%26spage%3D3688%26epage%3D3698%26doi%3D10.1128%2FAAC.00392-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Raboisson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desjarlais, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reed, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lattanze, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaikin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manthey, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomczuk, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marugan, J. J.</span><span> </span><span class="NLM_article-title">Identification of novel short chain 4-substituted indoles as potent alphavbeta3 antagonist using structure-based drug design</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">334</span><span class="NLM_x">–</span> <span class="NLM_lpage">343</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2006.10.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.ejmech.2006.10.015" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2007&pages=334-343&author=P.+Raboissonauthor=R.+L.+Desjarlaisauthor=R.+Reedauthor=J.+Lattanzeauthor=M.+Chaikinauthor=C.+L.+Mantheyauthor=B.+E.+Tomczukauthor=J.+J.+Marugan&title=Identification+of+novel+short+chain+4-substituted+indoles+as+potent+alphavbeta3+antagonist+using+structure-based+drug+design&doi=10.1016%2Fj.ejmech.2006.10.015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2006.10.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2006.10.015%26sid%3Dliteratum%253Aachs%26aulast%3DRaboisson%26aufirst%3DP.%26aulast%3DDesjarlais%26aufirst%3DR.%2BL.%26aulast%3DReed%26aufirst%3DR.%26aulast%3DLattanze%26aufirst%3DJ.%26aulast%3DChaikin%26aufirst%3DM.%26aulast%3DManthey%26aufirst%3DC.%2BL.%26aulast%3DTomczuk%26aufirst%3DB.%2BE.%26aulast%3DMarugan%26aufirst%3DJ.%2BJ.%26atitle%3DIdentification%2520of%2520novel%2520short%2520chain%25204-substituted%2520indoles%2520as%2520potent%2520alphavbeta3%2520antagonist%2520using%2520structure-based%2520drug%2520design%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2007%26volume%3D42%26spage%3D334%26epage%3D343%26doi%3D10.1016%2Fj.ejmech.2006.10.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Wang, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z. M.</span><span> </span><span class="NLM_article-title">Identification of some novel AHAS inhibitors via molecular docking and virtual screening approach</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">374</span><span class="NLM_x">–</span> <span class="NLM_lpage">380</span><span class="refDoi"> DOI: 10.1016/j.bmc.2006.09.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmc.2006.09.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=17049866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Cmt77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2007&pages=374-380&author=J.+G.+Wangauthor=Y.+J.+Xiaoauthor=Y.+H.+Liauthor=Y.+Maauthor=Z.+M.+Li&title=Identification+of+some+novel+AHAS+inhibitors+via+molecular+docking+and+virtual+screening+approach&doi=10.1016%2Fj.bmc.2006.09.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of some novel AHAS inhibitors via molecular docking and virtual screening approach</span></div><div class="casAuthors">Wang, Jian-Guo; Xiao, Yong-Jun; Li, Yong-Hong; Ma, Yi; Li, Zheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">374-380</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Acetohydroxyacid synthase (AHAS; EC 2.2.1.6) catalyzes the first common step in branched-chain amino acid biosynthesis.  This enzyme is an important target for the design of environmental-benign herbicides.  Based on the crystal structure of AHAS/sulfonylurea complex, we have carried out computational screening of the ACD-3D database in order to look for novel non-sulfonylurea inhibitors of AHAS for the first time.  Three novel compds. were found to inhibit plant AHAS in vitro among 14 procured compds.  One compd. showed promising activity in vivo for rape root growth inhibition bioassay.  This research provided useful clues for further design and discovery of AHAS inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAnfTtWRvu77Vg90H21EOLACvtfcHk0lgg7YX7aYPGsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Cmt77I&md5=3b61a8b7e87697d734c13d014e06bf3f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2006.09.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2006.09.050%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%2BG.%26aulast%3DXiao%26aufirst%3DY.%2BJ.%26aulast%3DLi%26aufirst%3DY.%2BH.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DZ.%2BM.%26atitle%3DIdentification%2520of%2520some%2520novel%2520AHAS%2520inhibitors%2520via%2520molecular%2520docking%2520and%2520virtual%2520screening%2520approach%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2007%26volume%3D15%26spage%3D374%26epage%3D380%26doi%3D10.1016%2Fj.bmc.2006.09.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Foloppe, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fisher, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howes, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potter, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surgenor, A. E.</span><span> </span><span class="NLM_article-title">Identification of chemically diverse Chk1 inhibitors by receptor-based virtual screening</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">4792</span><span class="NLM_x">–</span> <span class="NLM_lpage">4802</span><span class="refDoi"> DOI: 10.1016/j.bmc.2006.03.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmc.2006.03.021" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2006&pages=4792-4802&author=N.+Foloppeauthor=L.+M.+Fisherauthor=R.+Howesauthor=A.+Potterauthor=A.+G.+Robertsonauthor=A.+E.+Surgenor&title=Identification+of+chemically+diverse+Chk1+inhibitors+by+receptor-based+virtual+screening&doi=10.1016%2Fj.bmc.2006.03.021"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2006.03.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2006.03.021%26sid%3Dliteratum%253Aachs%26aulast%3DFoloppe%26aufirst%3DN.%26aulast%3DFisher%26aufirst%3DL.%2BM.%26aulast%3DHowes%26aufirst%3DR.%26aulast%3DPotter%26aufirst%3DA.%26aulast%3DRobertson%26aufirst%3DA.%2BG.%26aulast%3DSurgenor%26aufirst%3DA.%2BE.%26atitle%3DIdentification%2520of%2520chemically%2520diverse%2520Chk1%2520inhibitors%2520by%2520receptor-based%2520virtual%2520screening%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2006%26volume%3D14%26spage%3D4792%26epage%3D4802%26doi%3D10.1016%2Fj.bmc.2006.03.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Cherkasov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fallahi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammond, G. L.</span><span> </span><span class="NLM_article-title">Successful in silico discovery of novel nonsteroidal ligands for human sex hormone binding globulin</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">3203</span><span class="NLM_x">–</span> <span class="NLM_lpage">3213</span><span class="refDoi"> DOI: 10.1021/jm049087f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049087f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=3203-3213&author=A.+Cherkasovauthor=Z.+Shiauthor=M.+Fallahiauthor=G.+L.+Hammond&title=Successful+in+silico+discovery+of+novel+nonsteroidal+ligands+for+human+sex+hormone+binding+globulin&doi=10.1021%2Fjm049087f"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm049087f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049087f%26sid%3Dliteratum%253Aachs%26aulast%3DCherkasov%26aufirst%3DA.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DFallahi%26aufirst%3DM.%26aulast%3DHammond%26aufirst%3DG.%2BL.%26atitle%3DSuccessful%2520in%2520silico%2520discovery%2520of%2520novel%2520nonsteroidal%2520ligands%2520for%2520human%2520sex%2520hormone%2520binding%2520globulin%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D3203%26epage%3D3213%26doi%3D10.1021%2Fjm049087f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Huentelman, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zubcevic, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hernandez Prada, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dimitrov, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raizada, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostrov, D. A.</span><span> </span><span class="NLM_article-title">Structure-based discovery of a novel angiotensin-converting enzyme 2 inhibitor</span> <span class="citation_source-journal">Hypertension</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">903</span><span class="NLM_x">–</span> <span class="NLM_lpage">906</span><span class="refDoi"> DOI: 10.1161/01.HYP.0000146120.29648.36</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1161%2F01.HYP.0000146120.29648.36" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=15492138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpvVGmurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=903-906&author=M.+J.+Huentelmanauthor=J.+Zubcevicauthor=J.+A.+Hernandez+Pradaauthor=X.+Xiaoauthor=D.+S.+Dimitrovauthor=M.+K.+Raizadaauthor=D.+A.+Ostrov&title=Structure-based+discovery+of+a+novel+angiotensin-converting+enzyme+2+inhibitor&doi=10.1161%2F01.HYP.0000146120.29648.36"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of a novel angiotensin-converting enzyme 2 inhibitor</span></div><div class="casAuthors">Huentelman, Matthew J.; Zubcevic, Jasenka; Hernandez Prada, Jose A.; Xiao, Xiaodong; Dimitrov, Dimiter S.; Raizada, Mohan K.; Ostrov, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Hypertension</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">903-906</span>CODEN:
                <span class="NLM_cas:coden">HPRTDN</span>;
        ISSN:<span class="NLM_cas:issn">0194-911X</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Angiotensin-converting enzyme 2 (ACE2) is considered an important therapeutic target for controlling cardiovascular diseases and severe acute respiratory syndrome (SARS) outbreaks.  Recently solved high-resoln. crystal structures of the apo-bound and inhibitor-bound forms of ACE2 have provided the basis for a novel mol. docking approach in an attempt to identify ACE2 inhibitors and compds. that block SARS coronavirus spike protein-mediated cell fusion.  In this study, ≈140 000 small mols. were screened by in silico mol. docking.  In this structure-activity relation study, the mols. with the highest predicted binding scores were identified and assayed for ACE2 enzymic inhibitory activity and for their ability to inhibit SARS coronavirus spike protein-mediated cell fusion.  This approach identified N-(2-aminoethyl)-1 aziridine-ethanamine as a novel ACE2 inhibitor that also is effective in blocking the SARS coronavirus spike protein-mediated cell fusion.  Thus, the mol. docking approach resulting in the inhibitory capacity of N-(2-aminoethyl)-1 aziridine-ethanamine provides an attractive small mol. lead compd. on which the development of more effective therapeutic agents could be developed to modulate hypertension and for controlling SARS infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHyw2ny8yWH7Vg90H21EOLACvtfcHk0li5GH-xcAVl_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpvVGmurc%253D&md5=4bfdeab803ff822a69c07acf97ed4897</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1161%2F01.HYP.0000146120.29648.36&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252F01.HYP.0000146120.29648.36%26sid%3Dliteratum%253Aachs%26aulast%3DHuentelman%26aufirst%3DM.%2BJ.%26aulast%3DZubcevic%26aufirst%3DJ.%26aulast%3DHernandez%2BPrada%26aufirst%3DJ.%2BA.%26aulast%3DXiao%26aufirst%3DX.%26aulast%3DDimitrov%26aufirst%3DD.%2BS.%26aulast%3DRaizada%26aufirst%3DM.%2BK.%26aulast%3DOstrov%26aufirst%3DD.%2BA.%26atitle%3DStructure-based%2520discovery%2520of%2520a%2520novel%2520angiotensin-converting%2520enzyme%25202%2520inhibitor%26jtitle%3DHypertension%26date%3D2004%26volume%3D44%26spage%3D903%26epage%3D906%26doi%3D10.1161%2F01.HYP.0000146120.29648.36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Mizutani, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itai, A.</span><span> </span><span class="NLM_article-title">Efficient method for high-throughput virtual screening based on flexible docking: discovery of novel acetylcholinesterase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">4818</span><span class="NLM_x">–</span> <span class="NLM_lpage">4828</span><span class="refDoi"> DOI: 10.1021/jm030605g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030605g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=4818-4828&author=M.+Y.+Mizutaniauthor=A.+Itai&title=Efficient+method+for+high-throughput+virtual+screening+based+on+flexible+docking%3A+discovery+of+novel+acetylcholinesterase+inhibitors&doi=10.1021%2Fjm030605g"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm030605g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030605g%26sid%3Dliteratum%253Aachs%26aulast%3DMizutani%26aufirst%3DM.%2BY.%26aulast%3DItai%26aufirst%3DA.%26atitle%3DEfficient%2520method%2520for%2520high-throughput%2520virtual%2520screening%2520based%2520on%2520flexible%2520docking%253A%2520discovery%2520of%2520novel%2520acetylcholinesterase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D4818%26epage%3D4828%26doi%3D10.1021%2Fjm030605g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Chandra, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhagavat, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sreekanthreddy, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somasundaram, K.</span><span> </span><span class="NLM_article-title">Virtual screening, identification and experimental testing of novel inhibitors of PBEF1/Visfatin/NMPRTase for glioma therapy</span> <span class="citation_source-journal">J. Clin. Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="refDoi"> DOI: 10.1186/2043-9113-1-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1186%2F2043-9113-1-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=21884623" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFShur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2011&pages=5&author=N.+Chandraauthor=R.+Bhagavatauthor=E.+Sharmaauthor=P.+Sreekanthreddyauthor=K.+Somasundaram&title=Virtual+screening%2C+identification+and+experimental+testing+of+novel+inhibitors+of+PBEF1%2FVisfatin%2FNMPRTase+for+glioma+therapy&doi=10.1186%2F2043-9113-1-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening, identification and experimental testing of novel inhibitors of PBEF1/visfatin/NMPRTase for glioma therapy</span></div><div class="casAuthors">Chandra, Nagasuma; Bhagavat, Raghu; Sharma, Eshita; Sreekanthreddy, P.; Somasundaram, Kumaravel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Bioinformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5</span>CODEN:
                <span class="NLM_cas:coden">JCBOBR</span>;
        ISSN:<span class="NLM_cas:issn">2043-9113</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Pre-B-cell colony enhancing factor 1 gene (PBEF1) encodes nicotinamide phosphoribosyltransferase (NMPRTase), which catalyzes the rate limiting step in the salvage pathway of NAD+ metab. in mammalian cells.  PBEF1 transcript and protein levels have been shown to be elevated in glioblastoma and a chem. inhibitor of NMPRTase has been shown to specifically inhibit cancer cells.  Methods: Virtual screening using docking was used to screen a library of more than 13,000 chem. compds.  A shortlisted set of compds. were tested for their inhibition activity in vitro by an NMPRTase enzyme assay.  Further, the ability of the compds. to inhibit glioma cell proliferation was carried out.  Results: Virtual screening resulted in short listing of 34 possible ligands, of which six were tested exptl., using the NMPRTase enzyme inhibition assay and further with the glioma cell viability assays.  Of these, two compds. were found to be significantly efficacious in inhibiting the conversion of nicotinamide to NAD+, and out of which, one compd., 3-amino-2-benzyl-7-nitro-4-(2-quinolyl-)-1,2-dihydroisoquinolin-1-one, was found to inhibit the growth of a PBEF1 over expressing glioma derived cell line U87 as well.  Conclusions: Thus, a novel inhibitor has been identified through a structure based drug discovery approach and is further supported by exptl. evidence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpJpygBbJb1LVg90H21EOLACvtfcHk0li5GH-xcAVl_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFShur4%253D&md5=b39873fb098f5ffcf1c31d9a44ed35f7</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1186%2F2043-9113-1-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F2043-9113-1-5%26sid%3Dliteratum%253Aachs%26aulast%3DChandra%26aufirst%3DN.%26aulast%3DBhagavat%26aufirst%3DR.%26aulast%3DSharma%26aufirst%3DE.%26aulast%3DSreekanthreddy%26aufirst%3DP.%26aulast%3DSomasundaram%26aufirst%3DK.%26atitle%3DVirtual%2520screening%252C%2520identification%2520and%2520experimental%2520testing%2520of%2520novel%2520inhibitors%2520of%2520PBEF1%252FVisfatin%252FNMPRTase%2520for%2520glioma%2520therapy%26jtitle%3DJ.%2520Clin.%2520Bioinf.%26date%3D2011%26volume%3D1%26spage%3D5%26doi%3D10.1186%2F2043-9113-1-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Xu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Unzue, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spiliotopoulos, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nevado, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Discovery of CREBBP bromodomain inhibitors by high-throughput docking and hit optimization guided by molecular dynamics</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">1340</span><span class="NLM_x">–</span> <span class="NLM_lpage">1349</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00171</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00171" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVyitLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1340-1349&author=M.+Xuauthor=A.+Unzueauthor=J.+Dongauthor=D.+Spiliotopoulosauthor=C.+Nevadoauthor=A.+Caflisch&title=Discovery+of+CREBBP+bromodomain+inhibitors+by+high-throughput+docking+and+hit+optimization+guided+by+molecular+dynamics&doi=10.1021%2Facs.jmedchem.5b00171"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of CREBBP Bromodomain Inhibitors by High-Throughput Docking and Hit Optimization Guided by Molecular Dynamics</span></div><div class="casAuthors">Xu, Min; Unzue, Andrea; Dong, Jing; Spiliotopoulos, Dimitrios; Nevado, Cristina; Caflisch, Amedeo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1340-1349</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors have identified two chemotypes of CREBBP bromodomain ligands by fragment-based high-throughput docking.  Only 17 mols. from the original library of two-million compds. were tested in vitro.  Optimization of the two low-micromolar hits, the 4-acylpyrrole (III) and acylbenzene (IV), was driven by mol. dynamics results which suggested improvement of the polar interactions with the Arg1173 side chain at the rim of the binding site.  The synthesis of only two derivs. of III yielded the 4-acylpyrrole I which shows a single-digit micromolar affinity for the CREBBP bromodomain and a ligand efficiency of 0.34 kcal/mol per nonhydrogen atom.  Optimization of the acylbenzene hit IV resulted in a series of derivs. with nanomolar potencies, good ligand efficiency and selectivity.  The in silico predicted binding mode of the acylbenzene deriv. II was validated by solving the structure of the complex with the CREBBP bromodomain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq100fCa86MT7Vg90H21EOLACvtfcHk0li5GH-xcAVl_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVyitLrP&md5=e401c14fc119d2d1c39b74b5b41f4d3a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00171&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00171%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DM.%26aulast%3DUnzue%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DJ.%26aulast%3DSpiliotopoulos%26aufirst%3DD.%26aulast%3DNevado%26aufirst%3DC.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520CREBBP%2520bromodomain%2520inhibitors%2520by%2520high-throughput%2520docking%2520and%2520hit%2520optimization%2520guided%2520by%2520molecular%2520dynamics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1340%26epage%3D1349%26doi%3D10.1021%2Facs.jmedchem.5b00171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, W.</span><span> </span><span class="NLM_article-title">Discovery of new potent inhibitors for carbonic anhydrase IX by structure-based virtual screening</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">3496</span><span class="NLM_x">–</span> <span class="NLM_lpage">3499</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2013.04.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2013.04.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=23664215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsVynsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=3496-3499&author=L.+Wangauthor=C.+Yangauthor=W.+Luauthor=L.+Liuauthor=R.+Gaoauthor=S.+Liaoauthor=Z.+Zhaoauthor=L.+Zhuauthor=Y.+Xuauthor=H.+Liauthor=J.+Huangauthor=W.+Zhu&title=Discovery+of+new+potent+inhibitors+for+carbonic+anhydrase+IX+by+structure-based+virtual+screening&doi=10.1016%2Fj.bmcl.2013.04.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of new potent inhibitors for carbonic anhydrase IX by structure-based virtual screening</span></div><div class="casAuthors">Wang, Liyan; Yang, Chunmei; Lu, Weiqiang; Liu, Li; Gao, Rui; Liao, Sha; Zhao, Zhenjiang; Zhu, Lili; Xu, Yufang; Li, Honglin; Huang, Jin; Zhu, Weiping</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3496-3499</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Through structure-based virtual screening, some dozen of benzene sulfonamides with novel scaffolds are identified as potent inhibitors against carbonic anhydrase (CA) IX with IC50 values ranging from 2.86 to 588.34 nM.  Among them, compds. 1 and 9 show high selectivity against tumor-target CA IX over CA II (the selectivity ratios are 21.3 and 136.6, resp.).  The possible binding poses of hit compds. are also explored and the selectivity is elucidated by mol. docking simulations.  The hit compds. discovered in this work would provide novel scaffolds for further hit-to-lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRat2J1LShn7Vg90H21EOLACvtfcHk0liz7qCuxyh_-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsVynsbc%253D&md5=193b016905803f30f055f354dcdd59c3</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.04.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.04.048%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DR.%26aulast%3DLiao%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DW.%26atitle%3DDiscovery%2520of%2520new%2520potent%2520inhibitors%2520for%2520carbonic%2520anhydrase%2520IX%2520by%2520structure-based%2520virtual%2520screening%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D3496%26epage%3D3499%26doi%3D10.1016%2Fj.bmcl.2013.04.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Heusser, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borghese, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cullins, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broemstrup, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, U. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindahl, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, R. A.</span><span> </span><span class="NLM_article-title">Functional validation of virtual screening for novel agents with general anesthetic action at ligand-gated ion channels</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">670</span><span class="NLM_x">–</span> <span class="NLM_lpage">678</span><span class="refDoi"> DOI: 10.1124/mol.113.087692</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1124%2Fmol.113.087692" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2013&pages=670-678&author=S.+A.+Heusserauthor=R.+J.+Howardauthor=C.+M.+Borgheseauthor=M.+A.+Cullinsauthor=T.+Broemstrupauthor=U.+S.+Leeauthor=E.+Lindahlauthor=J.+Carlssonauthor=R.+A.+Harris&title=Functional+validation+of+virtual+screening+for+novel+agents+with+general+anesthetic+action+at+ligand-gated+ion+channels&doi=10.1124%2Fmol.113.087692"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1124%2Fmol.113.087692&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.113.087692%26sid%3Dliteratum%253Aachs%26aulast%3DHeusser%26aufirst%3DS.%2BA.%26aulast%3DHoward%26aufirst%3DR.%2BJ.%26aulast%3DBorghese%26aufirst%3DC.%2BM.%26aulast%3DCullins%26aufirst%3DM.%2BA.%26aulast%3DBroemstrup%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DU.%2BS.%26aulast%3DLindahl%26aufirst%3DE.%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DHarris%26aufirst%3DR.%2BA.%26atitle%3DFunctional%2520validation%2520of%2520virtual%2520screening%2520for%2520novel%2520agents%2520with%2520general%2520anesthetic%2520action%2520at%2520ligand-gated%2520ion%2520channels%26jtitle%3DMol.%2520Pharmacol.%26date%3D2013%26volume%3D84%26spage%3D670%26epage%3D678%26doi%3D10.1124%2Fmol.113.087692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Zhao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gartenmann, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spiliotopoulos, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Discovery of BRD4 bromodomain inhibitors by fragment-based high-throughput docking</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">2493</span><span class="NLM_x">–</span> <span class="NLM_lpage">2496</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2014.04.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2014.04.017" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=2493-2496&author=H.+Zhaoauthor=L.+Gartenmannauthor=J.+Dongauthor=D.+Spiliotopoulosauthor=A.+Caflisch&title=Discovery+of+BRD4+bromodomain+inhibitors+by+fragment-based+high-throughput+docking&doi=10.1016%2Fj.bmcl.2014.04.017"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.04.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.04.017%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DGartenmann%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DJ.%26aulast%3DSpiliotopoulos%26aufirst%3DD.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520BRD4%2520bromodomain%2520inhibitors%2520by%2520fragment-based%2520high-throughput%2520docking%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D2493%26epage%3D2496%26doi%3D10.1016%2Fj.bmcl.2014.04.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Kouskoumvekaki, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petersen, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fratev, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taboureau, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oprea, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sonne, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flindt, E. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jonsdottir, S. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristiansen, K.</span><span> </span><span class="NLM_article-title">Discovery of a novel selective PPARgamma ligand with partial agonist binding properties by integrated in silico/in vitro work flow</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">923</span><span class="NLM_x">–</span> <span class="NLM_lpage">937</span><span class="refDoi"> DOI: 10.1021/ci3006148</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3006148" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivFCns7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=923-937&author=I.+Kouskoumvekakiauthor=R.+K.+Petersenauthor=F.+Fratevauthor=O.+Taboureauauthor=T.+E.+Nielsenauthor=T.+I.+Opreaauthor=S.+B.+Sonneauthor=E.+N.+Flindtauthor=S.+O.+Jonsdottirauthor=K.+Kristiansen&title=Discovery+of+a+novel+selective+PPARgamma+ligand+with+partial+agonist+binding+properties+by+integrated+in+silico%2Fin+vitro+work+flow&doi=10.1021%2Fci3006148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Novel Selective PPARγ Ligand with Partial Agonist Binding Properties by Integrated in Silico/in Vitro Work Flow</span></div><div class="casAuthors">Kouskoumvekaki, Irene; Petersen, Rasmus K.; Fratev, Filip; Taboureau, Olivier; Nielsen, Thomas E.; Oprea, Tudor I.; Sonne, Si B.; Flindt, Esben N.; Jonsdottir, Svava Osk; Kristiansen, Karsten</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">923-937</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Full agonists to the peroxisome proliferator-activated receptor (PPAR)γ, such as Rosiglitazone, have been assocd. with a series of undesired side effects, such as wt. gain, fluid retention, cardiac hypertrophy, and hepatotoxicity.  Nevertheless, PPARγ is involved in the expression of genes that control glucose and lipid metab. and is an important target for drugs against type 2 diabetes, dyslipidemia, atherosclerosis, and cardiovascular disease.  In an effort to identify novel PPARγ ligands with an improved pharmacol. profile, emphasis has shifted to selective ligands with partial agonist binding properties.  Toward this end we applied an integrated in silico/in vitro workflow, based on pharmacophore- and structure-based virtual screening of the ZINC library, coupled with competitive binding and transactivation assays, and adipocyte differentiation and gene expression studies.  Hit compd. 9 was identified as the most potent ligand (IC50 = 0.3 μM) and a relatively poor inducer of adipocyte differentiation.  The binding mode of compd. 9 was confirmed by mol. dynamics simulation, and the calcd. free energy of binding was -8.4 kcal/mol.  A novel functional group, the carbonitrile group, was identified to be a key substituent in the ligand-protein interactions.  Further studies on the transcriptional regulation properties of compd. 9 revealed a gene regulatory profile that was to a large extent unique, however functionally closer to that of a partial agonist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2ghUTeDUWrVg90H21EOLACvtfcHk0liz7qCuxyh_-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivFCns7w%253D&md5=c0f5a929bf9712e43d958a974fef4b91</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fci3006148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3006148%26sid%3Dliteratum%253Aachs%26aulast%3DKouskoumvekaki%26aufirst%3DI.%26aulast%3DPetersen%26aufirst%3DR.%2BK.%26aulast%3DFratev%26aufirst%3DF.%26aulast%3DTaboureau%26aufirst%3DO.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DSonne%26aufirst%3DS.%2BB.%26aulast%3DFlindt%26aufirst%3DE.%2BN.%26aulast%3DJonsdottir%26aufirst%3DS.%2BO.%26aulast%3DKristiansen%26aufirst%3DK.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520selective%2520PPARgamma%2520ligand%2520with%2520partial%2520agonist%2520binding%2520properties%2520by%2520integrated%2520in%2520silico%252Fin%2520vitro%2520work%2520flow%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D923%26epage%3D937%26doi%3D10.1021%2Fci3006148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Xu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, I. A.</span><span> </span><span class="NLM_article-title">Crystal structure of avian aminoimidazole-4-carboxamide ribonucleotide transformylase in complex with a novel non-folate inhibitor identified by virtual ligand screening</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">279</span><span class="NLM_x">, </span> <span class="NLM_fpage">50555</span><span class="NLM_x">–</span> <span class="NLM_lpage">50565</span><span class="refDoi"> DOI: 10.1074/jbc.M406801200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1074%2Fjbc.M406801200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=50555-50565&author=L.+Xuauthor=C.+L.+Liauthor=A.+J.+Olsonauthor=I.+A.+Wilson&title=Crystal+structure+of+avian+aminoimidazole-4-carboxamide+ribonucleotide+transformylase+in+complex+with+a+novel+non-folate+inhibitor+identified+by+virtual+ligand+screening&doi=10.1074%2Fjbc.M406801200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M406801200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M406801200%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DC.%2BL.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26aulast%3DWilson%26aufirst%3DI.%2BA.%26atitle%3DCrystal%2520structure%2520of%2520avian%2520aminoimidazole-4-carboxamide%2520ribonucleotide%2520transformylase%2520in%2520complex%2520with%2520a%2520novel%2520non-folate%2520inhibitor%2520identified%2520by%2520virtual%2520ligand%2520screening%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D50555%26epage%3D50565%26doi%3D10.1074%2Fjbc.M406801200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Pierce, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stuver-Moody, C.</span><span> </span><span class="NLM_article-title">Docking study yields four novel inhibitors of the protooncogene Pim-1 kinase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1972</span><span class="NLM_x">–</span> <span class="NLM_lpage">1975</span><span class="refDoi"> DOI: 10.1021/jm701248t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701248t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitlOmtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=1972-1975&author=A.+C.+Pierceauthor=M.+Jacobsauthor=C.+Stuver-Moody&title=Docking+study+yields+four+novel+inhibitors+of+the+protooncogene+Pim-1+kinase&doi=10.1021%2Fjm701248t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Study Yields Four Novel Inhibitors of the Protooncogene Pim-1 Kinase</span></div><div class="casAuthors">Pierce, Albert C.; Jacobs, Marc; Stuver-Moody, Cameron</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1972-1975</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To supplement the hits from a high throughput screen, docking was performed against Pim-1 kinase.  Glide docking was augmented with a filter to require traditional or arom. CH··O hydrogen bonds to the kinase hinge.  Four diverse actives, of 96 mols. assayed, had Ki values between 0.091 and 4.5 μM.  This gives a 14-fold enrichment over the earlier HTS run, and the two crystal structures solved confirmed the binding modes predicted by docking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoH6GwwgMwC27Vg90H21EOLACvtfcHk0li36ohISbrFMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitlOmtLY%253D&md5=fafba52528730d4e250d489ad24a6cd8</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm701248t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701248t%26sid%3Dliteratum%253Aachs%26aulast%3DPierce%26aufirst%3DA.%2BC.%26aulast%3DJacobs%26aufirst%3DM.%26aulast%3DStuver-Moody%26aufirst%3DC.%26atitle%3DDocking%2520study%2520yields%2520four%2520novel%2520inhibitors%2520of%2520the%2520protooncogene%2520Pim-1%2520kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D1972%26epage%3D1975%26doi%3D10.1021%2Fjm701248t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Powers, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morandi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Structure-based discovery of a novel, noncovalent inhibitor of AmpC beta-lactamase</span> <span class="citation_source-journal">Structure (Oxford, U. K.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1013</span><span class="NLM_x">–</span> <span class="NLM_lpage">1023</span><span class="refDoi"> DOI: 10.1016/S0969-2126(02)00799-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2FS0969-2126%2802%2900799-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2002&pages=1013-1023&author=R.+A.+Powersauthor=F.+Morandiauthor=B.+K.+Shoichet&title=Structure-based+discovery+of+a+novel%2C+noncovalent+inhibitor+of+AmpC+beta-lactamase&doi=10.1016%2FS0969-2126%2802%2900799-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2802%2900799-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252802%252900799-2%26sid%3Dliteratum%253Aachs%26aulast%3DPowers%26aufirst%3DR.%2BA.%26aulast%3DMorandi%26aufirst%3DF.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520discovery%2520of%2520a%2520novel%252C%2520noncovalent%2520inhibitor%2520of%2520AmpC%2520beta-lactamase%26jtitle%3DStructure%2520%2528Oxford%252C%2520U.%2520K.%2529%26date%3D2002%26volume%3D10%26spage%3D1013%26epage%3D1023%26doi%3D10.1016%2FS0969-2126%2802%2900799-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cancian, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maltby, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Comprehensive mechanistic analysis of hits from high-throughput and docking screens against beta-lactamase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2502</span><span class="NLM_x">–</span> <span class="NLM_lpage">2511</span><span class="refDoi"> DOI: 10.1021/jm701500e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701500e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtFSksb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2502-2511&author=K.+Babaogluauthor=A.+Simeonovauthor=J.+J.+Irwinauthor=M.+E.+Nelsonauthor=B.+Fengauthor=C.+J.+Thomasauthor=L.+Cancianauthor=M.+P.+Costiauthor=D.+A.+Maltbyauthor=A.+Jadhavauthor=J.+Ingleseauthor=C.+P.+Austinauthor=B.+K.+Shoichet&title=Comprehensive+mechanistic+analysis+of+hits+from+high-throughput+and+docking+screens+against+beta-lactamase&doi=10.1021%2Fjm701500e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive Mechanistic Analysis of Hits from High-Throughput and Docking Screens against β-Lactamase</span></div><div class="casAuthors">Babaoglu, Kerim; Simeonov, Anton; Irwin, John J.; Nelson, Michael E.; Feng, Brian; Thomas, Craig J.; Cancian, Laura; Costi, M. Paola; Maltby, David A.; Jadhav, Ajit; Inglese, James; Austin, Christopher P.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2502-2511</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) is widely used in drug discovery.  Esp. for screens of unbiased libraries, false positives can dominate "hit lists"; their origins are much debated.  Here we det. the mechanism of every active hit from a screen of 70,563 unbiased mols. against β-lactamase using quant. HTS (qHTS).  Of the 1274 initial inhibitors, 95% were detergent-sensitive and were classified as aggregators.  Among the 70 remaining were 25 potent, covalent-acting β-lactams.  Mass spectra, counter-screens, and crystallog. identified 12 as promiscuous covalent inhibitors.  The remaining 33 were either aggregators or irreproducible.  No specific reversible inhibitors were found.  We turned to mol. docking to prioritize mols. from the same library for testing at higher concns.  Of 16 tested, 2 were modest inhibitors.  Subsequent X-ray structures corresponded to the docking prediction.  Analog synthesis improved affinity to 8 μM.  These results suggest that it may be the phys. behavior of org. mols., not their reactivity, that accounts for most screening artifacts.  Structure-based methods may prioritize weak-but-novel chemotypes in unbiased library screens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNtnaCegS8aLVg90H21EOLACvtfcHk0li36ohISbrFMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtFSksb0%253D&md5=7736429aeafc6b17a5d0182e24df0dd4</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm701500e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701500e%26sid%3Dliteratum%253Aachs%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DNelson%26aufirst%3DM.%2BE.%26aulast%3DFeng%26aufirst%3DB.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DCancian%26aufirst%3DL.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26aulast%3DMaltby%26aufirst%3DD.%2BA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DComprehensive%2520mechanistic%2520analysis%2520of%2520hits%2520from%2520high-throughput%2520and%2520docking%2520screens%2520against%2520beta-lactamase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2502%26epage%3D2511%26doi%3D10.1021%2Fjm701500e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Wacker, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fenalti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katritch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span> </span><span class="NLM_article-title">Conserved binding mode of human beta2 adrenergic receptor inverse agonists and antagonist revealed by X-ray crystallography</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">11443</span><span class="NLM_x">–</span> <span class="NLM_lpage">11445</span><span class="refDoi"> DOI: 10.1021/ja105108q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja105108q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=11443-11445&author=D.+Wackerauthor=G.+Fenaltiauthor=M.+A.+Brownauthor=V.+Katritchauthor=R.+Abagyanauthor=V.+Cherezovauthor=R.+C.+Stevens&title=Conserved+binding+mode+of+human+beta2+adrenergic+receptor+inverse+agonists+and+antagonist+revealed+by+X-ray+crystallography&doi=10.1021%2Fja105108q"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fja105108q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja105108q%26sid%3Dliteratum%253Aachs%26aulast%3DWacker%26aufirst%3DD.%26aulast%3DFenalti%26aufirst%3DG.%26aulast%3DBrown%26aufirst%3DM.%2BA.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DConserved%2520binding%2520mode%2520of%2520human%2520beta2%2520adrenergic%2520receptor%2520inverse%2520agonists%2520and%2520antagonist%2520revealed%2520by%2520X-ray%2520crystallography%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D11443%26epage%3D11445%26doi%3D10.1021%2Fja105108q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Ferreira, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eidam, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mott, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKerrow, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maloney, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Complementarity between a docking and a high-throughput screen in discovering new cruzain inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">4891</span><span class="NLM_x">–</span> <span class="NLM_lpage">4905</span><span class="refDoi"> DOI: 10.1021/jm100488w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100488w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntlWrurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=4891-4905&author=R.+S.+Ferreiraauthor=A.+Simeonovauthor=A.+Jadhavauthor=O.+Eidamauthor=B.+T.+Mottauthor=M.+J.+Keiserauthor=J.+H.+McKerrowauthor=D.+J.+Maloneyauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Complementarity+between+a+docking+and+a+high-throughput+screen+in+discovering+new+cruzain+inhibitors&doi=10.1021%2Fjm100488w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Complementarity Between a Docking and a High-Throughput Screen in Discovering New Cruzain Inhibitors</span></div><div class="casAuthors">Ferreira, Rafaela S.; Simeonov, Anton; Jadhav, Ajit; Eidam, Oliv; Mott, Bryan T.; Keiser, Michael J.; McKerrow, James H.; Maloney, David J.; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4891-4905</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual and high-throughput screens (HTS) should have complementary strengths and weaknesses, but studies that prospectively and comprehensively compare them are rare.  We undertook a parallel docking and HTS screen of 197861 compds. against cruzain, a thiol protease target for Chagas disease, looking for reversible, competitive inhibitors.  On workup, 99% of the hits were eliminated as false positives, yielding 146 well-behaved, competitive ligands.  These fell into five chemotypes: two were prioritized by scoring among the top 0.1% of the docking-ranked library, two were prioritized by behavior in the HTS and by clustering, and one chemotype was prioritized by both approaches.  Detn. of an inhibitor/cruzain crystal structure and comparison of the high-scoring docking hits to expt. illuminated the origins of docking false-negatives and false-positives.  Prioritizing mols. that are both predicted by docking and are HTS-active yields well-behaved mols., relatively unobscured by the false-positives to which both techniques are individually prone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmmoDO-1mm8bVg90H21EOLACvtfcHk0lgLLAJwKVhzOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntlWrurk%253D&md5=0661663ff0802bf981a1997eeca9fa05</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm100488w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100488w%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%26aufirst%3DR.%2BS.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DEidam%26aufirst%3DO.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DMcKerrow%26aufirst%3DJ.%2BH.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DComplementarity%2520between%2520a%2520docking%2520and%2520a%2520high-throughput%2520screen%2520in%2520discovering%2520new%2520cruzain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D4891%26epage%3D4905%26doi%3D10.1021%2Fjm100488w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Molecular docking and ligand specificity in fragment-based inhibitor discovery</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">358</span><span class="NLM_x">–</span> <span class="NLM_lpage">364</span><span class="refDoi"> DOI: 10.1038/nchembio.155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnchembio.155" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=19305397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsVCqt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=358-364&author=Y.+Chenauthor=B.+K.+Shoichet&title=Molecular+docking+and+ligand+specificity+in+fragment-based+inhibitor+discovery&doi=10.1038%2Fnchembio.155"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular docking and ligand specificity in fragment-based inhibitor discovery</span></div><div class="casAuthors">Chen, Yu; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">358-364</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fragment screens have successfully identified new scaffolds in drug discovery, often with relatively high hit rates (5%) using small screening libraries (1,000-10,000 compds.).  This raises two questions: would other noteworthy chemotypes be found were one to screen all com. available fragments (>300,000), and does the success rate imply low specificity of fragments.  We used mol. docking to screen large libraries of fragments against CTX-M β-lactamase.  We identified ten millimolar-range inhibitors from the 69 compds. tested.  The docking poses corresponded closely to the crystallog. structures subsequently detd.  Notably, these initial low-affinity hits showed little specificity between CTX-M and an unrelated β-lactamase, AmpC, which is unusual among β-lactamase inhibitors.  This is consistent with the idea that the high hit rates among fragments correlate to a low initial specificity.  As the inhibitors were progressed, both specificity and affinity rose together, yielding to our knowledge the first micromolar-range noncovalent inhibitors against a class A β-lactamase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO8zTkq9kMAbVg90H21EOLACvtfcHk0lgLLAJwKVhzOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsVCqt7k%253D&md5=6ee8dd19d0b99599ea74eaf735483b1d</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.155%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DMolecular%2520docking%2520and%2520ligand%2520specificity%2520in%2520fragment-based%2520inhibitor%2520discovery%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D358%26epage%3D364%26doi%3D10.1038%2Fnchembio.155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Teotico, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocklin, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferreira, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giannetti, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Docking for fragment inhibitors of AmpC beta-lactamase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">7455</span><span class="NLM_x">–</span> <span class="NLM_lpage">7460</span><span class="refDoi"> DOI: 10.1073/pnas.0813029106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1073%2Fpnas.0813029106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=19416920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmt1Khtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=7455-7460&author=D.+G.+Teoticoauthor=K.+Babaogluauthor=G.+J.+Rocklinauthor=R.+S.+Ferreiraauthor=A.+M.+Giannettiauthor=B.+K.+Shoichet&title=Docking+for+fragment+inhibitors+of+AmpC+beta-lactamase&doi=10.1073%2Fpnas.0813029106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Docking for fragment inhibitors of AmpC 83-lactamase</span></div><div class="casAuthors">Teotico, Denise G.; Babaoglu, Kerim; Rocklin, Gabriel J.; Ferreira, Rafaela S.; Giannetti, Anthony M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7455-7460</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Fragment screens for new ligands have had wide success, not withstanding their constraint to libraries of 1000-10,000 mols.  Larger libraries would be addressable were mol. docking reliable for fragment screens, but this has not been widely accepted.  To investigate docking's ability to prioritize fragments, a library of >137,000 such mols. were docked against the structure of β-lactamase.  Forty-eight fragments highly ranked by docking were acquired and tested; 23 had Ki values ranging from 0.7 to 9.2 mM.  X-ray crystal structures of the enzyme-bound complexes were detd. for 8 of the fragments.  For 4, the correspondence between the predicted and exptl. structures was high (RMSD between 1.2 and 1.4 Å), whereas for another 2, the fidelity was lower but retained most key interactions (RMSD 2.4-2.6 Å).  Two of the 8 fragments adopted very different poses in the active site owing to enzyme conformational changes.  The 48% hit rate of the fragment docking compares very favorably with "lead-like" docking and high-throughput screening against the same enzyme.  To understand this, we investigated the occurrence of the fragment scaffolds among larger, lead-like mols.  Approx. 1% of com. available fragments contain these inhibitors whereas only 10-7% of lead-like mols. do.  This suggests that many more chemotypes and combinations of chemotypes are present among fragments than are available among lead-like mols., contributing to the higher hit rates.  The ability of docking to prioritize these fragments suggests that the technique can be used to exploit the better chemotype coverage that exists at the fragment level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprpafzXv4ydrVg90H21EOLACvtfcHk0lgLLAJwKVhzOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmt1Khtrg%253D&md5=d2df01432d5381c226e8aa7e008457b5</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0813029106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0813029106%26sid%3Dliteratum%253Aachs%26aulast%3DTeotico%26aufirst%3DD.%2BG.%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DRocklin%26aufirst%3DG.%2BJ.%26aulast%3DFerreira%26aufirst%3DR.%2BS.%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDocking%2520for%2520fragment%2520inhibitors%2520of%2520AmpC%2520beta-lactamase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D7455%26epage%3D7460%26doi%3D10.1073%2Fpnas.0813029106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Merski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">The impact of introducing a histidine into an apolar cavity site on docking and ligand recognition</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2874</span><span class="NLM_x">–</span> <span class="NLM_lpage">2884</span><span class="refDoi"> DOI: 10.1021/jm301823g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301823g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2874-2884&author=M.+Merskiauthor=B.+K.+Shoichet&title=The+impact+of+introducing+a+histidine+into+an+apolar+cavity+site+on+docking+and+ligand+recognition&doi=10.1021%2Fjm301823g"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm301823g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301823g%26sid%3Dliteratum%253Aachs%26aulast%3DMerski%26aufirst%3DM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DThe%2520impact%2520of%2520introducing%2520a%2520histidine%2520into%2520an%2520apolar%2520cavity%2520site%2520on%2520docking%2520and%2520ligand%2520recognition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2874%26epage%3D2884%26doi%3D10.1021%2Fjm301823g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Barelier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyce, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodin, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Roles for ordered and bulk solvent in ligand recognition and docking in two related cavities</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e69153</span><span class="refDoi"> DOI: 10.1371/journal.pone.0069153</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pone.0069153" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=23874896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1WmtLbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e69153&author=S.+Barelierauthor=S.+E.+Boyceauthor=I.+Fishauthor=M.+Fischerauthor=D.+B.+Goodinauthor=B.+K.+Shoichet&title=Roles+for+ordered+and+bulk+solvent+in+ligand+recognition+and+docking+in+two+related+cavities&doi=10.1371%2Fjournal.pone.0069153"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Roles for ordered and bulk solvent in ligand recognition and docking in two related cavities</span></div><div class="casAuthors">Barelier, Sarah; Boyce, Sarah E.; Fish, Inbar; Fischer, Marcus; Goodin, David B.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e69153</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">A key challenge in structure-based discovery is accounting for modulation of protein-ligand interactions by ordered and bulk solvent.  To investigate this, we compared ligand binding to a buried cavity in Cytochrome c Peroxidase (CcP), where affinity is dominated by a single ionic interaction, vs. a cavity variant partly opened to solvent by loop deletion.  This opening had unexpected effects on ligand orientation, affinity and ordered water structure.  Some ligands lost over ten-fold in affinity and reoriented in the cavity, while others retained their geometries, formed new interactions with water networks and improved affinity.  To test our ability to discover new ligands against this opened site prospectively, a 534,000 fragment library was docked against the open cavity using two models of ligand solvation.  Using an older solvation model that prioritized many neutral mols., three such uncharged docking hits were tested, none of which was obsd. to bind; these mols. were not highly ranked by the new, context-dependent solvation score.  Using this new method, another 15 highly-ranked mols. were tested for binding.  In contrast to the previous result, 14 of these bound detectably, with affinities ranging from 8 μM to 2 mM.  In crystal structures, four of these new ligands superposed well with the docking predictions but two did not, reflecting unanticipated interactions with newly ordered waters mols.  Comparing recognition between this open cavity and its buried analog begins to isolate the roles of ordered solvent in a system that lends itself readily to prospective testing and that may be broadly useful to the community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojbQG1fpsJIrVg90H21EOLACvtfcHk0liG5uJ3eZzCvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1WmtLbK&md5=5ace2d9d19690ddbe591ee6abad9c9bf</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0069153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0069153%26sid%3Dliteratum%253Aachs%26aulast%3DBarelier%26aufirst%3DS.%26aulast%3DBoyce%26aufirst%3DS.%2BE.%26aulast%3DFish%26aufirst%3DI.%26aulast%3DFischer%26aufirst%3DM.%26aulast%3DGoodin%26aufirst%3DD.%2BB.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRoles%2520for%2520ordered%2520and%2520bulk%2520solvent%2520in%2520ligand%2520recognition%2520and%2520docking%2520in%2520two%2520related%2520cavities%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De69153%26doi%3D10.1371%2Fjournal.pone.0069153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lubet, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bode, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, Z.</span><span> </span><span class="NLM_article-title">Discovery of novel checkpoint kinase 1 inhibitors by virtual screening based on multiple crystal structures</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2904</span><span class="NLM_x">–</span> <span class="NLM_lpage">2914</span><span class="refDoi"> DOI: 10.1021/ci200257b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200257b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht12ktrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2904-2914&author=Y.+Liauthor=D.+J.+Kimauthor=W.+Maauthor=R.+A.+Lubetauthor=A.+M.+Bodeauthor=Z.+Dong&title=Discovery+of+novel+checkpoint+kinase+1+inhibitors+by+virtual+screening+based+on+multiple+crystal+structures&doi=10.1021%2Fci200257b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Checkpoint Kinase 1 Inhibitors by Virtual Screening Based on Multiple Crystal Structures</span></div><div class="casAuthors">Li, Yan; Kim, Dong-Joon; Ma, Wei-Ya; Lubet, Ronald A.; Bode, Ann M.; Dong, Zi-Gang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2904-2914</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Incorporating receptor flexibility is considered crucial for improvement of docking-based virtual screening.  With an abundance of crystallog. structures freely available, docking with multiple crystal structures is believed to be a practical approach to cope with protein flexibility.  Here we describe a successful application of the docking of multiple structures to discover novel and potent Chk1 inhibitors.  Forty-six Chk1 structures were first compared in single structure docking by predicting the binding mode and recovering known ligands.  Combinations of different protein structures were then compared by recovery of known ligands and an optimal ensemble of Chk1 structures were selected.  The chosen structures were used in the virtual screening of over 60000 diverse compds. for Chk1 inhibitors.  Six novel compds. ranked at the top of the hits list were tested exptl., and two of these compds. inhibited Chk1 activity-the best with an IC50 value of 9.6 μM.  Further study indicated that achieving a better enrichment and identifying more diverse compds. was more likely using multiple structures than using only a single structure even when protein structures were randomly selected.  Taking into account conformational energy difference did not help to improve enrichment in the top ranked list.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKXlCGjr_Om7Vg90H21EOLACvtfcHk0liG5uJ3eZzCvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht12ktrrL&md5=14ea06cd98ce6159ddc68bd8ddf7db05</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fci200257b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200257b%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DD.%2BJ.%26aulast%3DMa%26aufirst%3DW.%26aulast%3DLubet%26aufirst%3DR.%2BA.%26aulast%3DBode%26aufirst%3DA.%2BM.%26aulast%3DDong%26aufirst%3DZ.%26atitle%3DDiscovery%2520of%2520novel%2520checkpoint%2520kinase%25201%2520inhibitors%2520by%2520virtual%2520screening%2520based%2520on%2520multiple%2520crystal%2520structures%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2904%26epage%3D2914%26doi%3D10.1021%2Fci200257b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Akula, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, F. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, N.</span><span> </span><span class="NLM_article-title">Discovery of novel SecA inhibitors of Candidatus Liberibacter asiaticus by structure based design</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">4183</span><span class="NLM_x">–</span> <span class="NLM_lpage">4188</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2011.05.086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2011.05.086" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=4183-4188&author=N.+Akulaauthor=H.+Zhengauthor=F.+Q.+Hanauthor=N.+Wang&title=Discovery+of+novel+SecA+inhibitors+of+Candidatus+Liberibacter+asiaticus+by+structure+based+design&doi=10.1016%2Fj.bmcl.2011.05.086"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.05.086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.05.086%26sid%3Dliteratum%253Aachs%26aulast%3DAkula%26aufirst%3DN.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DHan%26aufirst%3DF.%2BQ.%26aulast%3DWang%26aufirst%3DN.%26atitle%3DDiscovery%2520of%2520novel%2520SecA%2520inhibitors%2520of%2520Candidatus%2520Liberibacter%2520asiaticus%2520by%2520structure%2520based%2520design%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D4183%26epage%3D4188%26doi%3D10.1016%2Fj.bmcl.2011.05.086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lehmann, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">Discovery of a novel acetylcholinesterase inhibitor by structure-based virtual screening techniques</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">3181</span><span class="NLM_x">–</span> <span class="NLM_lpage">3187</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2012.03.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2012.03.046" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=3181-3187&author=Y.+Chenauthor=L.+Fangauthor=S.+Pengauthor=H.+Liaoauthor=J.+Lehmannauthor=Y.+Zhang&title=Discovery+of+a+novel+acetylcholinesterase+inhibitor+by+structure-based+virtual+screening+techniques&doi=10.1016%2Fj.bmcl.2012.03.046"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.03.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.03.046%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DFang%26aufirst%3DL.%26aulast%3DPeng%26aufirst%3DS.%26aulast%3DLiao%26aufirst%3DH.%26aulast%3DLehmann%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520acetylcholinesterase%2520inhibitor%2520by%2520structure-based%2520virtual%2520screening%2520techniques%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D3181%26epage%3D3187%26doi%3D10.1016%2Fj.bmcl.2012.03.046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Ji, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, M.</span><span> </span><span class="NLM_article-title">Virtual screening of novel reversible inhibitors for marine alkaline protease MP</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">131</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2013.10.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.jmgm.2013.10.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsl2murzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2013&pages=125-131&author=X.+Jiauthor=Y.+Zhengauthor=W.+Wangauthor=J.+Shengauthor=J.+Haoauthor=M.+Sun&title=Virtual+screening+of+novel+reversible+inhibitors+for+marine+alkaline+protease+MP&doi=10.1016%2Fj.jmgm.2013.10.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening of novel reversible inhibitors for marine alkaline protease MP</span></div><div class="casAuthors">Ji, Xiaofeng; Zheng, Yuan; Wang, Wei; Sheng, Jun; Hao, Jianhua; Sun, Mi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">125-131</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Marine alk. protease (MP,2MP: the marine alk. protease, PDB id: 3U1R. accession no. ACY25898) is produced by a marine bacterium strain isolated from Yellow Sea sediment in China.  Previous research has shown that this protease is a cold-adapted enzyme with antioxidant activity that could be used as a detergent additive.  Owing to its instability in the liq. state, MP's application in liq. detergents was limited.  Therefore, the discovery of reversible MP inhibitors to stabilize the protease was imperative.  Here, we used the X-ray structure of MP and recompiled AutoDock 4.2 with refined Zn2+ characters to screen the free chem. database ZINC.  After completing the docking procedure, we applied strategies including the "initial filter", consensus scoring and pharmocophore model to accelerate the process and improve the virtual screening success rate.  The "initial filter" was built based on the docking results of boronic acid derivs. validated as reversible inhibitors of MP by our previous studies.  Finally, ten compds. were purchased or synthesized to test their binding affinity for MP.  Three of the compds. could reversibly inhibit MP with apparent Ki values of 0.8-1.2 mmol.  These active compds. and their binding modes provide useful information for understanding the mol. mechanism of reversible MP inhibition.  The results may also serve as the foundation for further screening and design of reversible MP inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohwfyjUL2g87Vg90H21EOLACvtfcHk0ljiBgKT2U65_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsl2murzE&md5=d95f6a7ed789e8dd89891dcc62d374e7</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2013.10.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2013.10.004%26sid%3Dliteratum%253Aachs%26aulast%3DJi%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DSheng%26aufirst%3DJ.%26aulast%3DHao%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DM.%26atitle%3DVirtual%2520screening%2520of%2520novel%2520reversible%2520inhibitors%2520for%2520marine%2520alkaline%2520protease%2520MP%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2013%26volume%3D46%26spage%3D125%26epage%3D131%26doi%3D10.1016%2Fj.jmgm.2013.10.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Kang, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, H. J.</span><span> </span><span class="NLM_article-title">In silico identification of novel ligands for G-quadruplex in the c-MYC promoter</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">339</span><span class="NLM_x">–</span> <span class="NLM_lpage">348</span><span class="refDoi"> DOI: 10.1007/s10822-014-9826-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs10822-014-9826-z" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=339-348&author=H.+J.+Kangauthor=H.+J.+Park&title=In+silico+identification+of+novel+ligands+for+G-quadruplex+in+the+c-MYC+promoter&doi=10.1007%2Fs10822-014-9826-z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9826-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9826-z%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DH.%2BJ.%26aulast%3DPark%26aufirst%3DH.%2BJ.%26atitle%3DIn%2520silico%2520identification%2520of%2520novel%2520ligands%2520for%2520G-quadruplex%2520in%2520the%2520c-MYC%2520promoter%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D339%26epage%3D348%26doi%3D10.1007%2Fs10822-014-9826-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Zhao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khadka, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, W. J.</span><span> </span><span class="NLM_article-title">Computer-aided discovery of aminopyridines as novel JAK2 inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">985</span><span class="NLM_x">–</span> <span class="NLM_lpage">995</span><span class="refDoi"> DOI: 10.1016/j.bmc.2015.01.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmc.2015.01.016" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=985-995&author=C.+Zhaoauthor=S.+H.+Yangauthor=D.+B.+Khadkaauthor=Y.+Jinauthor=K.+T.+Leeauthor=W.+J.+Cho&title=Computer-aided+discovery+of+aminopyridines+as+novel+JAK2+inhibitors&doi=10.1016%2Fj.bmc.2015.01.016"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.01.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.01.016%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DS.%2BH.%26aulast%3DKhadka%26aufirst%3DD.%2BB.%26aulast%3DJin%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DK.%2BT.%26aulast%3DCho%26aufirst%3DW.%2BJ.%26atitle%3DComputer-aided%2520discovery%2520of%2520aminopyridines%2520as%2520novel%2520JAK2%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D985%26epage%3D995%26doi%3D10.1016%2Fj.bmc.2015.01.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Lindert, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tallorin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, Q. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burkart, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">In silico screening for Plasmodium falciparum enoyl-ACP reductase inhibitors</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">79</span><span class="NLM_x">–</span> <span class="NLM_lpage">87</span><span class="refDoi"> DOI: 10.1007/s10822-014-9806-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs10822-014-9806-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=79-87&author=S.+Lindertauthor=L.+Tallorinauthor=Q.+G.+Nguyenauthor=M.+D.+Burkartauthor=J.+A.+McCammon&title=In+silico+screening+for+Plasmodium+falciparum+enoyl-ACP+reductase+inhibitors&doi=10.1007%2Fs10822-014-9806-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9806-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9806-3%26sid%3Dliteratum%253Aachs%26aulast%3DLindert%26aufirst%3DS.%26aulast%3DTallorin%26aufirst%3DL.%26aulast%3DNguyen%26aufirst%3DQ.%2BG.%26aulast%3DBurkart%26aufirst%3DM.%2BD.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DIn%2520silico%2520screening%2520for%2520Plasmodium%2520falciparum%2520enoyl-ACP%2520reductase%2520inhibitors%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D79%26epage%3D87%26doi%3D10.1007%2Fs10822-014-9806-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, M.</span><span> </span><span class="NLM_article-title">Discovery of novel EGFR tyrosine kinase inhibitors by structure-based virtual screening</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">4004</span><span class="NLM_x">–</span> <span class="NLM_lpage">4009</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2012.04.092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2012.04.092" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=4004-4009&author=S.+Liauthor=X.+Sunauthor=H.+Zhaoauthor=Y.+Tangauthor=M.+Lan&title=Discovery+of+novel+EGFR+tyrosine+kinase+inhibitors+by+structure-based+virtual+screening&doi=10.1016%2Fj.bmcl.2012.04.092"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.04.092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.04.092%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520novel%2520EGFR%2520tyrosine%2520kinase%2520inhibitors%2520by%2520structure-based%2520virtual%2520screening%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D4004%26epage%3D4009%26doi%3D10.1016%2Fj.bmcl.2012.04.092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Fang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, G. H.</span><span> </span><span class="NLM_article-title">Consensus models for CDK5 inhibitors in silico and their application to inhibitor discovery</span> <span class="citation_source-journal">Mol. Diversity</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">149</span><span class="NLM_x">–</span> <span class="NLM_lpage">162</span><span class="refDoi"> DOI: 10.1007/s11030-014-9561-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs11030-014-9561-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2015&pages=149-162&author=J.+Fangauthor=R.+Yangauthor=L.+Gaoauthor=S.+Yangauthor=X.+Pangauthor=C.+Liauthor=Y.+Heauthor=A.+L.+Liuauthor=G.+H.+Du&title=Consensus+models+for+CDK5+inhibitors+in+silico+and+their+application+to+inhibitor+discovery&doi=10.1007%2Fs11030-014-9561-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1007%2Fs11030-014-9561-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11030-014-9561-3%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DPang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DA.%2BL.%26aulast%3DDu%26aufirst%3DG.%2BH.%26atitle%3DConsensus%2520models%2520for%2520CDK5%2520inhibitors%2520in%2520silico%2520and%2520their%2520application%2520to%2520inhibitor%2520discovery%26jtitle%3DMol.%2520Diversity%26date%3D2015%26volume%3D19%26spage%3D149%26epage%3D162%26doi%3D10.1007%2Fs11030-014-9561-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J.</span><span> </span><span class="NLM_article-title">Discovery of novel human acrosin inhibitors by virtual screening</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">977</span><span class="NLM_x">–</span> <span class="NLM_lpage">985</span><span class="refDoi"> DOI: 10.1007/s10822-011-9476-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs10822-011-9476-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=977-985&author=X.+Liuauthor=G.+Dongauthor=J.+Zhangauthor=J.+Qiauthor=C.+Zhengauthor=Y.+Zhouauthor=J.+Zhuauthor=C.+Shengauthor=J.+Lu&title=Discovery+of+novel+human+acrosin+inhibitors+by+virtual+screening&doi=10.1007%2Fs10822-011-9476-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9476-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9476-3%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DSheng%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520novel%2520human%2520acrosin%2520inhibitors%2520by%2520virtual%2520screening%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2011%26volume%3D25%26spage%3D977%26epage%3D985%26doi%3D10.1007%2Fs10822-011-9476-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Kokkonen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kokkola, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suuronen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poso, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarho, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lahtela-Kakkonen, M.</span><span> </span><span class="NLM_article-title">Virtual screening approach of sirtuin inhibitors results in two new scaffolds</span> <span class="citation_source-journal">Eur. J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">76</span><span class="NLM_x">, </span> <span class="NLM_fpage">27</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1016/j.ejps.2015.04.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.ejps.2015.04.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25936698" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsl2qtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2015&pages=27-32&author=P.+Kokkonenauthor=T.+Kokkolaauthor=T.+Suuronenauthor=A.+Posoauthor=E.+Jarhoauthor=M.+Lahtela-Kakkonen&title=Virtual+screening+approach+of+sirtuin+inhibitors+results+in+two+new+scaffolds&doi=10.1016%2Fj.ejps.2015.04.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening approach of sirtuin inhibitors results in two new scaffolds</span></div><div class="casAuthors">Kokkonen, Piia; Kokkola, Tarja; Suuronen, Tiina; Poso, Antti; Jarho, Elina; Lahtela-Kakkonen, Maija</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">27-32</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Sirtuins (SIRT1-SIRT7) are NAD dependent deacetylases and intriguing drug targets in for example neurodegenerative diseases and cancer.  Virtual screening has been shown to be a fast and efficient method of discovering new sirtuin inhibitors.  In this study, a new putative binding site on the zinc binding domain of sirtuins was utilized to screen the ZINC database virtually in order to discover new sirtuin inhibiting scaffolds.  Altogether 26 compds. were tested in vitro and initially 15 inhibitors displayed >30% SIRT3 inhibition.  However, the evaluation of raw data from in vitro assay revealed that many of the compds. had intrinsic property to interfere with the fluorescence signal at the assay wavelengths resulting in false positives.  All compds. with over 30% SIRT3 inhibition were studied more closely for their behavior in the assay and eventually, three compds. were identified as novel sirtuin inhibitors.  They displayed 32-40% SIRT3 and 21-60% SIRT2 inhibition.  The inhibitors display two new scaffolds, the smaller of which can be considered as a promising fragment, which offers a base for structural optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLEtTL9egZ0LVg90H21EOLACvtfcHk0li54W_QnvoBzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsl2qtLg%253D&md5=060b22ac9ee10687aa32f5cc8da4b58f</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.ejps.2015.04.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejps.2015.04.025%26sid%3Dliteratum%253Aachs%26aulast%3DKokkonen%26aufirst%3DP.%26aulast%3DKokkola%26aufirst%3DT.%26aulast%3DSuuronen%26aufirst%3DT.%26aulast%3DPoso%26aufirst%3DA.%26aulast%3DJarho%26aufirst%3DE.%26aulast%3DLahtela-Kakkonen%26aufirst%3DM.%26atitle%3DVirtual%2520screening%2520approach%2520of%2520sirtuin%2520inhibitors%2520results%2520in%2520two%2520new%2520scaffolds%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2015%26volume%3D76%26spage%3D27%26epage%3D32%26doi%3D10.1016%2Fj.ejps.2015.04.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Roughley, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span> </span><span class="NLM_article-title">How well can fragments explore accessed chemical space? A case study from heat shock protein 90</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">3989</span><span class="NLM_x">–</span> <span class="NLM_lpage">4005</span><span class="refDoi"> DOI: 10.1021/jm200350g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200350g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmsFymurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3989-4005&author=S.+D.+Roughleyauthor=R.+E.+Hubbard&title=How+well+can+fragments+explore+accessed+chemical+space%3F+A+case+study+from+heat+shock+protein+90&doi=10.1021%2Fjm200350g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">How Well Can Fragments Explore Accessed Chemical Space? A Case Study from Heat Shock Protein 90</span></div><div class="casAuthors">Roughley, Stephen D.; Hubbard, Roderick E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3989-4005</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  This article summerizes the published literature on the generation of inhibitors and design of drug against the mol. chaperone, heat shock protein 90 (Hsp90), as potential therapeutics to treat cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocGw-54Mo8SrVg90H21EOLACvtfcHk0li54W_QnvoBzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmsFymurs%253D&md5=90c2c8028cb460c0c6fdc956e7daae28</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Fjm200350g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200350g%26sid%3Dliteratum%253Aachs%26aulast%3DRoughley%26aufirst%3DS.%2BD.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DHow%2520well%2520can%2520fragments%2520explore%2520accessed%2520chemical%2520space%253F%2520A%2520case%2520study%2520from%2520heat%2520shock%2520protein%252090%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D3989%26epage%3D4005%26doi%3D10.1021%2Fjm200350g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Doman, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGovern, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witherbee, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasten, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurumbail, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stallings, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Connolly, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Molecular docking and high-throughput screening for novel inhibitors of protein tyrosine phosphatase-1B</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">2213</span><span class="NLM_x">–</span> <span class="NLM_lpage">2221</span><span class="refDoi"> DOI: 10.1021/jm010548w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm010548w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD38XivFCgtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2213-2221&author=T.+N.+Domanauthor=S.+L.+McGovernauthor=B.+J.+Witherbeeauthor=T.+P.+Kastenauthor=R.+Kurumbailauthor=W.+C.+Stallingsauthor=D.+T.+Connollyauthor=B.+K.+Shoichet&title=Molecular+docking+and+high-throughput+screening+for+novel+inhibitors+of+protein+tyrosine+phosphatase-1B&doi=10.1021%2Fjm010548w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular docking and high-throughput screening for novel inhibitors of protein tyrosine phosphatase-1B</span></div><div class="casAuthors">Doman, Thompson N.; McGovern, Susan L.; Witherbee, Bryan J.; Kasten, Thomas P.; Kurumbail, Ravi; Stallings, William C.; Connolly, Daniel T.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2213-2221</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) of compd. libraries is used to discover novel leads for drug development.  When a structure is available for the target, computer-based screening using mol. docking may also be considered.  The two techniques have rarely been used together on the same target.  The opportunity to do so presented itself in a project to discover novel inhibitors for the enzyme protein tyrosine phosphatase-1B (PTP1B), a tyrosine phosphatase that has been implicated as a key target for type II diabetes.  A corporate library of approx. 400 000 compds. was screened using high-throughput exptl. techniques for compds. that inhibited PTP1B.  Concurrently, mol. docking was used to screen approx. 235 000 com. available compds. against the X-ray crystallog. structure of PTP1B, and 365 high-scoring mols. were tested as inhibitors of the enzyme.  Of approx. 400 000 mols. tested in the high-throughput exptl. assay, 85 (0.021%) inhibited the enzyme with IC50 values less than 100 μM; the most active had an IC50 value of 4.2 μM.  Of the 365 mols. suggested by mol. docking, 127 (34.8%) inhibited PTP1B with IC50 values less than 100 μM; the most active of these had an IC50 of 1.7 μM.  Structure-based docking therefore enriched the hit rate by 1700-fold over random screening.  The hits from both the high-throughput and docking screens were dissimilar from phosphotyrosine, the canonical substrate group for PTP1B; the two hit lists were also very different from each other.  Surprisingly, the docking hits were judged to be more druglike than the HTS hits.  The diversity of both hit lists and their dissimilarity from each other suggest that docking and HTS may be complementary techniques for lead discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAMecsrqOs0LVg90H21EOLACvtfcHk0liiEAHB1XZpuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XivFCgtbg%253D&md5=63e0730d9748e0dacbad1e9cdba93ca2</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fjm010548w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm010548w%26sid%3Dliteratum%253Aachs%26aulast%3DDoman%26aufirst%3DT.%2BN.%26aulast%3DMcGovern%26aufirst%3DS.%2BL.%26aulast%3DWitherbee%26aufirst%3DB.%2BJ.%26aulast%3DKasten%26aufirst%3DT.%2BP.%26aulast%3DKurumbail%26aufirst%3DR.%26aulast%3DStallings%26aufirst%3DW.%2BC.%26aulast%3DConnolly%26aufirst%3DD.%2BT.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DMolecular%2520docking%2520and%2520high-throughput%2520screening%2520for%2520novel%2520inhibitors%2520of%2520protein%2520tyrosine%2520phosphatase-1B%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D2213%26epage%3D2221%26doi%3D10.1021%2Fjm010548w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Feng, B. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C. P.</span><span> </span><span class="NLM_article-title">A high-throughput screen for aggregation-based inhibition in a large compound library</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2385</span><span class="NLM_x">–</span> <span class="NLM_lpage">2390</span><span class="refDoi"> DOI: 10.1021/jm061317y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm061317y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksVSlsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2385-2390&author=B.+Y.+Fengauthor=A.+Simeonovauthor=A.+Jadhavauthor=K.+Babaogluauthor=J.+Ingleseauthor=B.+K.+Shoichetauthor=C.+P.+Austin&title=A+high-throughput+screen+for+aggregation-based+inhibition+in+a+large+compound+library&doi=10.1021%2Fjm061317y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">A High-Throughput Screen for Aggregation-Based Inhibition in a Large Compound Library</span></div><div class="casAuthors">Feng, Brian Y.; Simeonov, Anton; Jadhav, Ajit; Babaoglu, Kerim; Inglese, James; Shoichet, Brian K.; Austin, Christopher P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2385-2390</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) is the primary technique for new lead identification in drug discovery and chem. biol.  Unfortunately, it is susceptible to false-pos. hits.  One common mechanism for such false-positives is the congregation of org. mols. into colloidal aggregates, which nonspecifically inhibit enzymes.  To both evaluate the feasibility of large-scale identification of aggregate-based inhibition and quantify its prevalence among screening hits, the authors tested 70 563 mols. from the National Institutes of Health Chem. Genomics Center (NCGC) library for detergent-sensitive inhibition.  Each mol. was screened in at least seven concns., such that dose-response curves were obtained for all mols. in the library.  There were 1274 inhibitors identified in total, of which 1204 were unambiguously detergent-sensitive.  The authors identified these as aggregate-based inhibitors.  Thirty-one library mols. were independently purchased and retested in secondary low-throughput expts.; 29 of these were confirmed as either aggregators or nonaggregators, as appropriate.  Finally, with the dose-response information collected for every compd., the authors could examine the correlation between aggregate-based inhibition and steep dose-response curves.  Three key results emerge from this study: first, detergent-dependent identification of aggregate-based inhibition is feasible on the large scale.  Second, 95% of the actives obtained in this screen are aggregate-based inhibitors.  Third, aggregate-based inhibition is correlated with steep dose-response curves, although not absolutely.  The results of this screen are being released publicly via the PubChem database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn9eDfL7UswbVg90H21EOLACvtfcHk0liiEAHB1XZpuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksVSlsrw%253D&md5=33399bcd4998faf830f54a6f0d1f636a</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fjm061317y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061317y%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DB.%2BY.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26atitle%3DA%2520high-throughput%2520screen%2520for%2520aggregation-based%2520inhibition%2520in%2520a%2520large%2520compound%2520library%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D2385%26epage%3D2390%26doi%3D10.1021%2Fjm061317y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferreira, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klumpp, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mott, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maloney, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span> </span><span class="NLM_article-title">Quantitative analyses of aggregation, autofluorescence, and reactivity artifacts in a screen for inhibitors of a thiol protease</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">37</span><span class="NLM_x">–</span> <span class="NLM_lpage">51</span><span class="refDoi"> DOI: 10.1021/jm901070c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901070c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVSgtLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=37-51&author=A.+Jadhavauthor=R.+S.+Ferreiraauthor=C.+Klumppauthor=B.+T.+Mottauthor=C.+P.+Austinauthor=J.+Ingleseauthor=C.+J.+Thomasauthor=D.+J.+Maloneyauthor=B.+K.+Shoichetauthor=A.+Simeonov&title=Quantitative+analyses+of+aggregation%2C+autofluorescence%2C+and+reactivity+artifacts+in+a+screen+for+inhibitors+of+a+thiol+protease&doi=10.1021%2Fjm901070c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative Analyses of Aggregation, Autofluorescence, and Reactivity Artifacts in a Screen for Inhibitors of a Thiol Protease</span></div><div class="casAuthors">Jadhav, Ajit; Ferreira, Rafaela S.; Klumpp, Carleen; Mott, Bryan T.; Austin, Christopher P.; Inglese, James; Thomas, Craig J.; Maloney, David J.; Shoichet, Brian K.; Simeonov, Anton</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-51</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The perceived and actual burden of false positives in high-throughput screening has received considerable attention; however, few studies exist on the contributions of distinct mechanisms of nonspecific effects like chem. reactivity, assay signal interference, and colloidal aggregation.  Here, we analyze the outcome of a screen of 197861 diverse compds. in a concn.-response format against the cysteine protease cruzain, a target expected to be particularly sensitive to reactive compds., and using an assay format with light detection in the short-wavelength region where significant compd. autofluorescence is typically encountered.  Approx. 1.9% of all compds. screened were detergent-sensitive inhibitors.  The contribution from autofluorescence and compds. bearing reactive functionalities was dramatically lower: of all hits, only 1.8% were autofluorescent and 1.5% contained reactive or undesired functional groups.  The distribution of false positives was relatively const. across library sources.  The simple step of including detergent in the assay buffer suppressed the nonspecific effect of approx. 93% of the original hits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0icdibXhgLbVg90H21EOLACvtfcHk0liiEAHB1XZpuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVSgtLjL&md5=c2a46f1632b2b7d79655c43cbae8fdba</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fjm901070c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901070c%26sid%3Dliteratum%253Aachs%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DFerreira%26aufirst%3DR.%2BS.%26aulast%3DKlumpp%26aufirst%3DC.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DQuantitative%2520analyses%2520of%2520aggregation%252C%2520autofluorescence%252C%2520and%2520reactivity%2520artifacts%2520in%2520a%2520screen%2520for%2520inhibitors%2520of%2520a%2520thiol%2520protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D37%26epage%3D51%26doi%3D10.1021%2Fjm901070c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Forli, S.</span><span> </span><span class="NLM_article-title">Charting a Path to Success in Virtual Screening</span> <span class="citation_source-journal">Molecules</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">18732</span><span class="NLM_x">–</span> <span class="NLM_lpage">18758</span><span class="refDoi"> DOI: 10.3390/molecules201018732</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.3390%2Fmolecules201018732" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26501243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslartLfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=18732-18758&author=S.+Forli&title=Charting+a+Path+to+Success+in+Virtual+Screening&doi=10.3390%2Fmolecules201018732"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Charting a path to success in virtual screening</span></div><div class="casAuthors">Forli, Stefano</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">18732-18758</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Docking is commonly applied to drug design efforts, esp. high-throughput virtual screenings of small mols., to identify new compds. that bind to a given target.  Despite great advances and successful applications in recent years, a no. of issues remain unsolved.  Most of the challenges and problems faced when running docking expts. are independent of the specific software used, and can be ascribed to either improper input prepn. or to the simplified approaches applied to achieve high-throughput speed.  Being aware of approxns. and limitations of such methods is essential to prevent errors, deal with misleading results, and increase the success rate of virtual screening campaigns.  In this review, best practices and most common issues of docking and virtual screening will be discussed, covering the journey from the design of the virtual expt. to the hit identification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRNqBR6AUIubVg90H21EOLACvtfcHk0lglzMIeKfw3SA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslartLfL&md5=991965e4159b6616ecfe45357ef595f7</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.3390%2Fmolecules201018732&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules201018732%26sid%3Dliteratum%253Aachs%26aulast%3DForli%26aufirst%3DS.%26atitle%3DCharting%2520a%2520Path%2520to%2520Success%2520in%2520Virtual%2520Screening%26jtitle%3DMolecules%26date%3D2015%26volume%3D20%26spage%3D18732%26epage%3D18758%26doi%3D10.3390%2Fmolecules201018732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Spyrakis, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavasotto, C. N.</span><span> </span><span class="NLM_article-title">Open challenges in structure-based virtual screening: Receptor modeling, target flexibility consideration and active site water molecules description</span> <span class="citation_source-journal">Arch. Biochem. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">583</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">119</span><span class="refDoi"> DOI: 10.1016/j.abb.2015.08.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.abb.2015.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26271444" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFWmtb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=583&publication_year=2015&pages=105-119&author=F.+Spyrakisauthor=C.+N.+Cavasotto&title=Open+challenges+in+structure-based+virtual+screening%3A+Receptor+modeling%2C+target+flexibility+consideration+and+active+site+water+molecules+description&doi=10.1016%2Fj.abb.2015.08.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Open challenges in structure-based virtual screening: Receptor modeling, target flexibility consideration and active site water molecules description</span></div><div class="casAuthors">Spyrakis, Francesca; Cavasotto, Claudio N.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">583</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">105-119</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Structure-based virtual screening is currently an established tool in drug lead discovery projects.  Although in the last years the field saw an impressive progress in terms of algorithm development, computational performance, and retrospective and prospective applications in ligand identification, there are still long-standing challenges where further improvement is needed.  In this review, we consider the conceptual frame, state-of-the-art and recent developments of three crit. "structural" issues in structure-based drug lead discovery: the use of homol. modeling to accurately model the binding site when no exptl. structures are available, the necessity of accounting for the dynamics of intrinsically flexible systems as proteins, and the importance of considering active site water mols. in lead identification and optimization campaigns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmOGftaXjkYrVg90H21EOLACvtfcHk0lglzMIeKfw3SA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFWmtb%252FK&md5=b8818c43713169365741420862ff4448</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2015.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2015.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DSpyrakis%26aufirst%3DF.%26aulast%3DCavasotto%26aufirst%3DC.%2BN.%26atitle%3DOpen%2520challenges%2520in%2520structure-based%2520virtual%2520screening%253A%2520Receptor%2520modeling%252C%2520target%2520flexibility%2520consideration%2520and%2520active%2520site%2520water%2520molecules%2520description%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2015%26volume%3D583%26spage%3D105%26epage%3D119%26doi%3D10.1016%2Fj.abb.2015.08.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">Ferreira, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dos Santos, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliva, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andricopulo, A. D.</span><span> </span><span class="NLM_article-title">Molecular docking and structure-based drug design strategies</span> <span class="citation_source-journal">Molecules</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">13384</span><span class="NLM_x">–</span> <span class="NLM_lpage">13421</span><span class="refDoi"> DOI: 10.3390/molecules200713384</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.3390%2Fmolecules200713384" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26205061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1OjtLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=13384-13421&author=L.+G.+Ferreiraauthor=R.+N.+Dos+Santosauthor=G.+Olivaauthor=A.+D.+Andricopulo&title=Molecular+docking+and+structure-based+drug+design+strategies&doi=10.3390%2Fmolecules200713384"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular docking and structure-based drug design strategies</span></div><div class="casAuthors">Ferreira, Leonardo G.; dos Santos, Ricardo N.; Oliva, Glaucius; Andricopulo, Adriano D.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">13384-13421</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Pharmaceutical research has successfully incorporated a wealth of mol. modeling methods, within a variety of drug discovery programs, to study complex biol. and chem. systems.  The integration of computational and exptl. strategies has been of great value in the identification and development of novel promising compds.  Broadly used in modern drug design, mol. docking methods explore the ligand conformations adopted within the binding sites of macromol. targets.  This approach also ests. the ligand-receptor binding free energy by evaluating crit. phenomena involved in the intermol. recognition process.  Today, as a variety of docking algorithms are available, an understanding of the advantages and limitations of each method is of fundamental importance in the development of effective strategies and the generation of relevant results.  The purpose of this review is to examine current mol. docking strategies used in drug discovery and medicinal chem., exploring the advances in the field and the role played by the integration of structure- and ligand-based methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf5La9X0Wng7Vg90H21EOLACvtfcHk0lglzMIeKfw3SA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1OjtLzJ&md5=b5f70b90e45098d0f0252a2926f9671e</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.3390%2Fmolecules200713384&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules200713384%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%26aufirst%3DL.%2BG.%26aulast%3DDos%2BSantos%26aufirst%3DR.%2BN.%26aulast%3DOliva%26aufirst%3DG.%26aulast%3DAndricopulo%26aufirst%3DA.%2BD.%26atitle%3DMolecular%2520docking%2520and%2520structure-based%2520drug%2520design%2520strategies%26jtitle%3DMolecules%26date%3D2015%26volume%3D20%26spage%3D13384%26epage%3D13421%26doi%3D10.3390%2Fmolecules200713384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group">Cerqueira, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gesto, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliveira, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santos-Martins, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bras, N. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sousa, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandes, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramos, M. J.</span><span> </span><span class="NLM_article-title">Receptor-based virtual screening protocol for drug discovery</span> <span class="citation_source-journal">Arch. Biochem. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">582</span><span class="NLM_x">, </span> <span class="NLM_fpage">56</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.1016/j.abb.2015.05.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.abb.2015.05.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26045247" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpsVyls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=582&publication_year=2015&pages=56-67&author=N.+M.+Cerqueiraauthor=D.+Gestoauthor=E.+F.+Oliveiraauthor=D.+Santos-Martinsauthor=N.+F.+Brasauthor=S.+F.+Sousaauthor=P.+A.+Fernandesauthor=M.+J.+Ramos&title=Receptor-based+virtual+screening+protocol+for+drug+discovery&doi=10.1016%2Fj.abb.2015.05.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Receptor-based virtual screening protocol for drug discovery</span></div><div class="casAuthors">Cerqueira, Nuno M. F. S. A.; Gesto, Diana; Oliveira, Eduardo F.; Santos-Martins, Diogo; Bras, Natercia F.; Sousa, Sergio F.; Fernandes, Pedro A.; Ramos, Maria J.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">582</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">56-67</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Computational aided drug design (CADD) is presently a key component in the process of drug discovery and development as it offers great promise to drastically reduce cost and time requirements.In the pharmaceutical arena, virtual screening is normally regarded as the top CADD tool to screen large libraries of chem. structures and reduce them to a key set of likely drug candidates regarding a specific protein target.  This chapter provides a comprehensive overview of the receptor-based virtual screening process and of its importance in the present drug discovery and development paradigm.  Following a focused contextualization on the subject, the main stages of a virtual screening campaign, including its strengths and limitations, are the subject of particular attention in this review.  In all of these stages special consideration will be given to practical issues that are normally the Achilles heel of the virtual screening process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDV14aAOZb_LVg90H21EOLACvtfcHk0lii0YyDK880Hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpsVyls78%253D&md5=c6f861d3150b3e76fb3a9b5b905d195a</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2015.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2015.05.011%26sid%3Dliteratum%253Aachs%26aulast%3DCerqueira%26aufirst%3DN.%2BM.%26aulast%3DGesto%26aufirst%3DD.%26aulast%3DOliveira%26aufirst%3DE.%2BF.%26aulast%3DSantos-Martins%26aufirst%3DD.%26aulast%3DBras%26aufirst%3DN.%2BF.%26aulast%3DSousa%26aufirst%3DS.%2BF.%26aulast%3DFernandes%26aufirst%3DP.%2BA.%26aulast%3DRamos%26aufirst%3DM.%2BJ.%26atitle%3DReceptor-based%2520virtual%2520screening%2520protocol%2520for%2520drug%2520discovery%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2015%26volume%3D582%26spage%3D56%26epage%3D67%26doi%3D10.1016%2Fj.abb.2015.05.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group">Kumalo, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhakat, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soliman, M. E.</span><span> </span><span class="NLM_article-title">Theory and applications of covalent docking in drug discovery: merits and pitfalls</span> <span class="citation_source-journal">Molecules</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1984</span><span class="NLM_x">–</span> <span class="NLM_lpage">2000</span><span class="refDoi"> DOI: 10.3390/molecules20021984</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.3390%2Fmolecules20021984" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25633330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitF2gsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=1984-2000&author=H.+M.+Kumaloauthor=S.+Bhakatauthor=M.+E.+Soliman&title=Theory+and+applications+of+covalent+docking+in+drug+discovery%3A+merits+and+pitfalls&doi=10.3390%2Fmolecules20021984"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Theory and applications of covalent docking in drug discovery: merits and pitfalls</span></div><div class="casAuthors">Kumalo, Hezekiel Mathambo; Bhakat, Soumendranath; Soliman, Mahmoud E. S.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1984-2000, 17 pp.</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  The present art of drug discovery and design of new drugs is based on suicidal irreversible inhibitors.  Covalent inhibition is the strategy that is used to achieve irreversible inhibition.  Irreversible inhibitors interact with their targets in a time-dependent fashion, and the reaction proceeds to completion rather than to equil.  Covalent inhibitors possessed some significant advantages over non-covalent inhibitors such as covalent warheads can target rare, non-conserved residue of a particular target protein and thus led to development of highly selective inhibitors, covalent inhibitors can be effective in targeting proteins with shallow binding cleavage which will led to development of novel inhibitors with increased potency than non-covalent inhibitors.  Several computational approaches have been developed to simulate covalent interactions; however, this is still a challenging area to explore.  Covalent mol. docking has been recently implemented in the computer-aided drug design workflows to describe covalent interactions between inhibitors and biol. targets.  In this review we highlight: (i) covalent interactions in biomol. systems; (ii) the math. framework of covalent mol. docking; (iii) implementation of covalent docking protocol in drug design workflows; (iv) applications covalent docking: case studies and (v) shortcomings and future perspectives of covalent docking.  To the best of our knowledge; this review is the first account that highlights different aspects of covalent docking with its merits and pitfalls.  We believe that the method and applications highlighted in this study will help future efforts towards the design of irreversible inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyAF4rJ5aS7rVg90H21EOLACvtfcHk0lii0YyDK880Hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitF2gsrg%253D&md5=4c47470306282d34bab152867dc522e3</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.3390%2Fmolecules20021984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules20021984%26sid%3Dliteratum%253Aachs%26aulast%3DKumalo%26aufirst%3DH.%2BM.%26aulast%3DBhakat%26aufirst%3DS.%26aulast%3DSoliman%26aufirst%3DM.%2BE.%26atitle%3DTheory%2520and%2520applications%2520of%2520covalent%2520docking%2520in%2520drug%2520discovery%253A%2520merits%2520and%2520pitfalls%26jtitle%3DMolecules%26date%3D2015%26volume%3D20%26spage%3D1984%26epage%3D2000%26doi%3D10.3390%2Fmolecules20021984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group">Webb, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Protein structure modeling with MODELLER</span> <span class="citation_source-journal">Methods Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1137</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">15</span><span class="refDoi"> DOI: 10.1007/978-1-4939-0366-5_1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2F978-1-4939-0366-5_1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24573470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns12isrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1137&publication_year=2014&pages=1-15&author=B.+Webbauthor=A.+Sali&title=Protein+structure+modeling+with+MODELLER&doi=10.1007%2F978-1-4939-0366-5_1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Structure Modeling with MODELLER</span></div><div class="casAuthors">Webb, Benjamin; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1137</span>
        (<span class="NLM_cas:issue">Protein Structure Prediction</span>),
    <span class="NLM_cas:pages">1-15</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Genome sequencing projects have resulted in a rapid increase in the no. of known protein sequences.  In contrast, only about one-hundredth of these sequences have been characterized at at. resoln. using exptl. structure detn. methods.  Computational protein structure modeling techniques have the potential to bridge this sequence-structure gap.  In this chapter, we present an example that illustrates the use of MODELLER to construct a comparative model for a protein with unknown structure.  Automation of a similar protocol has resulted in models of useful accuracy for domains in more than half of all known protein sequences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs6xvMHHyK0bVg90H21EOLACvtfcHk0lii0YyDK880Hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns12isrc%253D&md5=19709a3ffefe33fd6e5d8506258bd6f0</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-0366-5_1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-0366-5_1%26sid%3Dliteratum%253Aachs%26aulast%3DWebb%26aufirst%3DB.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DProtein%2520structure%2520modeling%2520with%2520MODELLER%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2014%26volume%3D1137%26spage%3D1%26epage%3D15%26doi%3D10.1007%2F978-1-4939-0366-5_1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group">Pieper, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, G. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneidman-Duhovny, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuri, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spill, Y. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinkam, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tainer, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilges, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">ModBase, a database of annotated comparative protein structure models and associated resources</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">D336</span><span class="NLM_x">–</span> <span class="NLM_lpage">346</span><span class="refDoi"> DOI: 10.1093/nar/gkt1144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1093%2Fnar%2Fgkt1144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24271400" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1Gj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D336-346&author=U.+Pieperauthor=B.+M.+Webbauthor=G.+Q.+Dongauthor=D.+Schneidman-Duhovnyauthor=H.+Fanauthor=S.+J.+Kimauthor=N.+Khuriauthor=Y.+G.+Spillauthor=P.+Weinkamauthor=M.+Hammelauthor=J.+A.+Tainerauthor=M.+Nilgesauthor=A.+Sali&title=ModBase%2C+a+database+of+annotated+comparative+protein+structure+models+and+associated+resources&doi=10.1093%2Fnar%2Fgkt1144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">ModBase, a database of annotated comparative protein structure models and associated resources</span></div><div class="casAuthors">Pieper, Ursula; Webb, Benjamin M.; Dong, Guang Qiang; Schneidman-Duhovny, Dina; Fan, Hao; Kim, Seung Joong; Khuri, Natalia; Spill, Yannick G.; Weinkam, Patrick; Hammel, Michal; Tainer, John A.; Nilges, Michael; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D336-D346</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ModBase http://salilab.org/modbase is a database of annotated comparative protein structure models.  The models are calcd. by ModPipe, an automated modeling pipeline that relies primarily on Modeller for fold assignment, sequence-structure alignment, model building and model assessment http://salilab.org/modeller/.  ModBase currently contains almost 30 million reliable models for domains in 4.7 million unique protein sequences.  ModBase allows users to compute or update comparative models on demand, through an interface to the ModWeb modeling server http://salilab.org/modweb.  ModBase models are also available through the Protein Model Portal http://www.proteinmodelportal.org/.  Recently developed assocd. resources include the AllosMod server for modeling ligand-induced protein dynamics http://salilab.org/allosmod, the AllosMod-FoXS server for predicting a structural ensemble that fits an SAXS profile http://salilab.org/allosmod-foxs, the FoXSDock server for protein-protein docking filtered by an SAXS profile http://salilab.org/foxsdock, the SAXS Merge server for automatic merging of SAXS profiles http://salilab.org/saxsmerge and the Pose & Rank server for scoring protein-ligand complexes http://salilab.org/poseandrank. In this update, we also highlight two applications of ModBase: a PSI:Biol. initiative to maximize the structural coverage of the human alpha-helical transmembrane proteome and a detn. of structural determinants of human immunodeficiency virus-1 protease specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomlF6VKj_t4bVg90H21EOLACvtfcHk0liIa_UgALKcZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1Gj&md5=1a0809c46145a0a783ad5d5b2a6e3326</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1144%26sid%3Dliteratum%253Aachs%26aulast%3DPieper%26aufirst%3DU.%26aulast%3DWebb%26aufirst%3DB.%2BM.%26aulast%3DDong%26aufirst%3DG.%2BQ.%26aulast%3DSchneidman-Duhovny%26aufirst%3DD.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DS.%2BJ.%26aulast%3DKhuri%26aufirst%3DN.%26aulast%3DSpill%26aufirst%3DY.%2BG.%26aulast%3DWeinkam%26aufirst%3DP.%26aulast%3DHammel%26aufirst%3DM.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26aulast%3DNilges%26aufirst%3DM.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DModBase%252C%2520a%2520database%2520of%2520annotated%2520comparative%2520protein%2520structure%2520models%2520and%2520associated%2520resources%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DD336%26epage%3D346%26doi%3D10.1093%2Fnar%2Fgkt1144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Virtual ligand screening against comparative protein structure models</span> <span class="citation_source-journal">Methods Mol. Biol. (N. Y., NY, U. S.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">819</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">126</span><span class="refDoi"> DOI: 10.1007/978-1-61779-465-0_8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2F978-1-61779-465-0_8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=22183533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BC38%252Fos1Cjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=819&publication_year=2012&pages=105-126&author=H.+Fanauthor=J.+J.+Irwinauthor=A.+Sali&title=Virtual+ligand+screening+against+comparative+protein+structure+models&doi=10.1007%2F978-1-61779-465-0_8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual ligand screening against comparative protein structure models</span></div><div class="casAuthors">Fan Hao; Irwin John J; Sali Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Methods in molecular biology (Clifton, N.J.)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">819</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">105-26</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Virtual ligand screening uses computation to discover new ligands of a protein by screening one or more of its structural models against a database of potential ligands.  Comparative protein structure modeling extends the applicability of virtual screening beyond the atomic structures determined by X-ray crystallography or NMR spectroscopy.  Here, we describe an integrated modeling and docking protocol, combining comparative modeling by MODELLER and virtual ligand screening by DOCK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS3KPL-W_yq5JagfuraLLzSfW6udTcc2eborCe9m2kQvbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38%252Fos1Cjuw%253D%253D&md5=e94c5ee89cbf4ffcd3e3b0b39358c908</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-465-0_8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-465-0_8%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DH.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DVirtual%2520ligand%2520screening%2520against%2520comparative%2520protein%2520structure%2520models%26jtitle%3DMethods%2520Mol.%2520Biol.%2520%2528N.%2520Y.%252C%2520NY%252C%2520U.%2520S.%2529%26date%3D2012%26volume%3D819%26spage%3D105%26epage%3D126%26doi%3D10.1007%2F978-1-61779-465-0_8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Exploiting ordered waters in molecular docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">4862</span><span class="NLM_x">–</span> <span class="NLM_lpage">4865</span><span class="refDoi"> DOI: 10.1021/jm8006239</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8006239" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptl2hs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=4862-4865&author=N.+Huangauthor=B.+K.+Shoichet&title=Exploiting+ordered+waters+in+molecular+docking&doi=10.1021%2Fjm8006239"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting Ordered Waters in Molecular Docking</span></div><div class="casAuthors">Huang, Niu; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4862-4865</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A current weakness in docking is the treatment of water-mediated protein-ligand interactions.  The authors explore switching ordered water mols. "on" and "off" during docking screens of a large library.  The method assumes additivity and scales linearly with the no. of waters sampled despite the exponential growth in configurations.  It is tested for ligand enrichment against 24 targets, exploring up to 256 water configurations.  Water inclusion increased enrichment substantially for 12 targets, while most others were largely unaffected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0tyvmJzfbCbVg90H21EOLACvtfcHk0liIa_UgALKcZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptl2hs7o%253D&md5=2f0430f9a71bb76e68977fcdff800366</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1021%2Fjm8006239&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8006239%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DExploiting%2520ordered%2520waters%2520in%2520molecular%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D4862%26epage%3D4865%26doi%3D10.1021%2Fjm8006239" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Virtual screening in lead discovery and optimization</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">396</span><span class="NLM_x">–</span> <span class="NLM_lpage">403</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=15338948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpvFOmsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2004&pages=396-403&author=A.+N.+Jain&title=Virtual+screening+in+lead+discovery+and+optimization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening in lead discovery and optimization</span></div><div class="casAuthors">Jain, Ajay N.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">396-403</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Thomson Scientific</span>)
        </div><div class="casAbstract">A review.  Virtual screening by mol. docking, using a protein with an exptl. detd. structure as a target, has become an established method for lead discovery and for enhancing efficiency in lead optimization.  Generalizations of the quant. structure-activity relationship concept have led to approaches for virtual screening in the absence of a protein target structure, instead relying upon ligand-based models as surrogates of protein active sites.  Recently reported methods for ligand-based virtual screening can achieve similar enrichment rates to those obtained using mol. docking.  This review will discuss recent advances in both domains of virtual screening, including theor. and practical advances and the implications for their application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeVT5-ZGuZn7Vg90H21EOLACvtfcHk0liIa_UgALKcZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpvFOmsrs%253D&md5=7fddac472fc1eda64682db69a7fbb843</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DVirtual%2520screening%2520in%2520lead%2520discovery%2520and%2520optimization%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2004%26volume%3D7%26spage%3D396%26epage%3D403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartshorn, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nissink, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span> </span><span class="NLM_article-title">Modeling water molecules in protein-ligand docking using GOLD</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">6504</span><span class="NLM_x">–</span> <span class="NLM_lpage">6515</span><span class="refDoi"> DOI: 10.1021/jm050543p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050543p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFCnu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6504-6515&author=M.+L.+Verdonkauthor=G.+Chessariauthor=J.+C.+Coleauthor=M.+J.+Hartshornauthor=C.+W.+Murrayauthor=J.+W.+Nissinkauthor=R.+D.+Taylorauthor=R.+Taylor&title=Modeling+water+molecules+in+protein-ligand+docking+using+GOLD&doi=10.1021%2Fjm050543p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling Water Molecules in Protein-Ligand Docking Using GOLD</span></div><div class="casAuthors">Verdonk, Marcel L.; Chessari, Gianni; Cole, Jason C.; Hartshorn, Michael J.; Murray, Christopher W.; Nissink, J. Willem M.; Taylor, Richard D.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6504-6515</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We implemented a novel approach to score water mediation and displacement in the protein-ligand docking program GOLD.  The method allows water mols. to switch on and off and to rotate around their three principal axes.  A const. penalty, σp, representing the loss of rigid-body entropy, is added for water mols. that are switched on, hence rewarding water displacement.  We tested the methodol. in an extensive validation study.  First, σp is optimized against a training set of 58 protein-ligand complexes.  For this training set, our algorithm correctly predicts water mediation/displacement in ∼92% of the cases.  We obsd. small improvements in the quality of the predicted binding modes for water-mediated complexes.  In the second part of this work, an entirely independent set of 225 complexes is used.  For this test set, our algorithm correctly predicts water mediation/displacement in ∼93% of the cases.  Improvements in binding mode quality were obsd. for individual water-mediated complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKBGOf-rFSarVg90H21EOLACvtfcHk0ljgztpcJ5Z4DA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFCnu78%253D&md5=78b60f794d8af7266f0a37f6e2aea6b1</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1021%2Fjm050543p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050543p%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DModeling%2520water%2520molecules%2520in%2520protein-ligand%2520docking%2520using%2520GOLD%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6504%26epage%3D6515%26doi%3D10.1021%2Fjm050543p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carchia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Directory of useful decoys, enhanced (DUD-E): better ligands and decoys for better benchmarking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">6582</span><span class="NLM_x">–</span> <span class="NLM_lpage">6594</span><span class="refDoi"> DOI: 10.1021/jm300687e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300687e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFaku7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6582-6594&author=M.+M.+Mysingerauthor=M.+Carchiaauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Directory+of+useful+decoys%2C+enhanced+%28DUD-E%29%3A+better+ligands+and+decoys+for+better+benchmarking&doi=10.1021%2Fjm300687e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Directory of Useful Decoys, Enhanced (DUD-E): Better Ligands and Decoys for Better Benchmarking</span></div><div class="casAuthors">Mysinger, Michael M.; Carchia, Michael; Irwin, John. J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6582-6594</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A key metric to assess mol. docking remains ligand enrichment against challenging decoys.  Whereas the directory of useful decoys (DUD) has been widely used, clear areas for optimization have emerged.  Here we describe an improved benchmarking set that includes more diverse targets such as GPCRs and ion channels, totaling 102 proteins with 22886 clustered ligands drawn from ChEMBL, each with 50 property-matched decoys drawn from ZINC.  To ensure chemotype diversity, we cluster each target's ligands by their Bemis-Murcko at. frameworks.  We add net charge to the matched physicochem. properties and include only the most dissimilar decoys, by topol., from the ligands.  An online automated tool (http://decoys.docking.org) generates these improved matched decoys for user-supplied ligands.  We test this data set by docking all 102 targets, using the results to improve the balance between ligand desolvation and electrostatics in DOCK 3.6.  The complete DUD-E benchmarking set is freely available at http://dude.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowquULXHTtJLVg90H21EOLACvtfcHk0ljgztpcJ5Z4DA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFaku7c%253D&md5=23ba4049662b388a880d4a48dde3769c</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1021%2Fjm300687e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300687e%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DCarchia%26aufirst%3DM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDirectory%2520of%2520useful%2520decoys%252C%2520enhanced%2520%2528DUD-E%2529%253A%2520better%2520ligands%2520and%2520decoys%2520for%2520better%2520benchmarking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6582%26epage%3D6594%26doi%3D10.1021%2Fjm300687e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group">Carr, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery: leads by design</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">987</span><span class="NLM_x">–</span> <span class="NLM_lpage">992</span><span class="refDoi"> DOI: 10.1016/S1359-6446(05)03511-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2FS1359-6446%2805%2903511-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=16023057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmt1Whtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=987-992&author=R.+A.+Carrauthor=M.+Congreveauthor=C.+W.+Murrayauthor=D.+C.+Rees&title=Fragment-based+lead+discovery%3A+leads+by+design&doi=10.1016%2FS1359-6446%2805%2903511-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery: leads by design</span></div><div class="casAuthors">Carr, Robin A. E.; Congreve, Miles; Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">987-992</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Fragment-based lead discovery (also referred to as needles, shapes, binding elements, seed templates or scaffolds) is a new lead discovery approach in which much lower mol. wt. (120-250Da) compds. are screened relative to HTS campaigns.  Fragment-based hits are typically weak inhibitors (10μM-mM), and therefore need to be screened at higher concn. using very sensitive biophys. detection techniques such as protein crystallog. and NMR as the primary screening techniques, rather than bioassays.  Compared with HTS hits, these fragments are simpler, less functionalized compds. with correspondingly lower affinity.  However, fragment hits typically possess high ligand efficiency' (binding affinity per heavy atom) and so are highly suitable for optimization into clin. candidates with good drug-like properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq15FNhjT6Qw7Vg90H21EOLACvtfcHk0ljgztpcJ5Z4DA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmt1Whtr0%253D&md5=1bf7dd97ec056af9ed5143f96b34bb53</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2805%2903511-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252805%252903511-7%26sid%3Dliteratum%253Aachs%26aulast%3DCarr%26aufirst%3DR.%2BA.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DFragment-based%2520lead%2520discovery%253A%2520leads%2520by%2520design%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26spage%3D987%26epage%3D992%26doi%3D10.1016%2FS1359-6446%2805%2903511-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group">Oprea, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teague, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leeson, P. D.</span><span> </span><span class="NLM_article-title">Is there a difference between leads and drugs? A historical perspective</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">1308</span><span class="NLM_x">–</span> <span class="NLM_lpage">1315</span><span class="refDoi"> DOI: 10.1021/ci010366a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci010366a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlt1Okt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2001&pages=1308-1315&author=T.+I.+Opreaauthor=A.+M.+Davisauthor=S.+J.+Teagueauthor=P.+D.+Leeson&title=Is+there+a+difference+between+leads+and+drugs%3F+A+historical+perspective&doi=10.1021%2Fci010366a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Is There a Difference between Leads and Drugs? A Historical Perspective</span></div><div class="casAuthors">Oprea, Tudor I.; Davis, Andrew M.; Teague, Simon J.; Leeson, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1308-1315</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To be considered for further development, lead structures should display the following properties: (1) simple chem. features, amenable for chem. optimization; (2) membership to an established SAR series; (3) favorable patent situation; and (4) good absorption, distribution, metab., and excretion (ADME) properties.  There are two distinct categories of leads: those that lack any therapeutic use (i.e., "pure" leads), and those that are marketed drugs themselves but have been altered to yield novel drugs.  We have previously analyzed the design of leadlike combinatorial libraries starting from 18 lead and drug pairs of structures (S. J. Teague et al. Angew. Chem., Int. Ed. Engl. 1999, 38, 3743-3748).  Here, we report results based on an extended dataset of 96 lead-drug pairs, of which 62 are lead structures that are not marketed as drugs, and 75 are drugs that are not presumably used as leads.  We examd. the following properties: MW (mol. wt.), CMR (the calcd. mol. refractivity), RNG (the no. of rings), RTB (the no. of rotatable bonds), the no. of hydrogen bond donors (HDO) and acceptors (HAC), the calcd. logarithm of the n-octanol/water partition (CLogP), the calcd. logarithm of the distribution coeff. at pH 7.4 (LogD74), the Daylight-fingerprint druglike score (DFPS), and the property and pharmacophore features score (PPFS).  The following differences were obsd. between the medians of drugs and leads: ΔMW = 69; ΔCMR = 1.8; ΔRNG = ΔHAC =1; ΔRTB = 2; ΔCLogP = 0.43; ΔLogD74 = 0.97; ΔHDO = 0; ΔDFPS = 0.15; ΔPPFS = 0.12.  Lead structures exhibit, on the av., less mol. complexity (less MW, less no. of rings and rotatable bonds), are less hydrophobic (lower CLogP and LogD74), and less druglike (lower druglike scores).  These findings indicate that the process of optimizing a lead into a drug results in more complex structures.  This information should be used in the design of novel combinatorial libraries that are aimed at lead discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrG4E0R8n7tLrVg90H21EOLACvtfcHk0ljArnYSK06DuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlt1Okt7s%253D&md5=422b9cf937168d9c2f3d1c8429648ea0</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Fci010366a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci010366a%26sid%3Dliteratum%253Aachs%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DDavis%26aufirst%3DA.%2BM.%26aulast%3DTeague%26aufirst%3DS.%2BJ.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26atitle%3DIs%2520there%2520a%2520difference%2520between%2520leads%2520and%2520drugs%253F%2520A%2520historical%2520perspective%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2001%26volume%3D41%26spage%3D1308%26epage%3D1315%26doi%3D10.1021%2Fci010366a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group">Milletti, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hermann, J. C.</span><span> </span><span class="NLM_article-title">Targeted kinase selectivity from kinase profiling data</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">383</span><span class="NLM_x">–</span> <span class="NLM_lpage">386</span><span class="refDoi"> DOI: 10.1021/ml300012r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml300012r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=383-386&author=F.+Millettiauthor=J.+C.+Hermann&title=Targeted+kinase+selectivity+from+kinase+profiling+data&doi=10.1021%2Fml300012r"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1021%2Fml300012r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml300012r%26sid%3Dliteratum%253Aachs%26aulast%3DMilletti%26aufirst%3DF.%26aulast%3DHermann%26aufirst%3DJ.%2BC.%26atitle%3DTargeted%2520kinase%2520selectivity%2520from%2520kinase%2520profiling%2520data%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D383%26epage%3D386%26doi%3D10.1021%2Fml300012r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Contribution of conformer focusing to the uncertainty in predicting free energies for protein-ligand binding</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">5880</span><span class="NLM_x">–</span> <span class="NLM_lpage">5884</span><span class="refDoi"> DOI: 10.1021/jm060763i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060763i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD28Xptl2hsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=5880-5884&author=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Contribution+of+conformer+focusing+to+the+uncertainty+in+predicting+free+energies+for+protein-ligand+binding&doi=10.1021%2Fjm060763i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Contribution of Conformer Focusing to the Uncertainty in Predicting Free Energies for Protein-Ligand Binding</span></div><div class="casAuthors">Tirado-Rives, Julian; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5880-5884</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">When a ligand binds to a protein, it is typically not in the lowest-energy conformation for the unbound ligand and there is also a loss of conformational degrees of freedom.  The free-energy change for this "conformer focusing" is addressed here formally, and the assocd. errors with its estn. or neglect are considered in the context of scoring functions for protein-ligand docking and computation of abs. free energies of binding.  Specific applications for inhibition of HIV-1 reverse transcriptase are reported.  It is concluded that the uncertainties from this source alone are sufficient to preclude the viability of current docking methodol. for rank-ordering of diverse compds. in high-throughput virtual screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS33v6gg6p27Vg90H21EOLACvtfcHk0ljArnYSK06DuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xptl2hsLk%253D&md5=8a24236944e39caece2bd5dde96a7ca8</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1021%2Fjm060763i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060763i%26sid%3Dliteratum%253Aachs%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DContribution%2520of%2520conformer%2520focusing%2520to%2520the%2520uncertainty%2520in%2520predicting%2520free%2520energies%2520for%2520protein-ligand%2520binding%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D5880%26epage%3D5884%26doi%3D10.1021%2Fjm060763i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group">Afzal, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Firoz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaggi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bawa, S.</span><span> </span><span class="NLM_article-title">Docking based virtual screening and molecular dynamics study to identify potential monoacylglycerol lipase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">3986</span><span class="NLM_x">–</span> <span class="NLM_lpage">3996</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2014.06.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2014.06.029" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=3986-3996&author=O.+Afzalauthor=S.+Kumarauthor=R.+Kumarauthor=A.+Firozauthor=M.+Jaggiauthor=S.+Bawa&title=Docking+based+virtual+screening+and+molecular+dynamics+study+to+identify+potential+monoacylglycerol+lipase+inhibitors&doi=10.1016%2Fj.bmcl.2014.06.029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.06.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.06.029%26sid%3Dliteratum%253Aachs%26aulast%3DAfzal%26aufirst%3DO.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DKumar%26aufirst%3DR.%26aulast%3DFiroz%26aufirst%3DA.%26aulast%3DJaggi%26aufirst%3DM.%26aulast%3DBawa%26aufirst%3DS.%26atitle%3DDocking%2520based%2520virtual%2520screening%2520and%2520molecular%2520dynamics%2520study%2520to%2520identify%2520potential%2520monoacylglycerol%2520lipase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D3986%26epage%3D3996%26doi%3D10.1016%2Fj.bmcl.2014.06.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group">Vyas, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghate, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goel, A.</span><span> </span><span class="NLM_article-title">Pharmacophore modeling, virtual screening, docking and in silico ADMET analysis of protein kinase B (PKB beta) inhibitors</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">17</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2013.01.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.jmgm.2013.01.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsVykur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2013&pages=17-25&author=V.+K.+Vyasauthor=M.+Ghateauthor=A.+Goel&title=Pharmacophore+modeling%2C+virtual+screening%2C+docking+and+in+silico+ADMET+analysis+of+protein+kinase+B+%28PKB+beta%29+inhibitors&doi=10.1016%2Fj.jmgm.2013.01.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacophore modeling, virtual screening, docking and in silico ADMET analysis of protein kinase B (PKB β) inhibitors</span></div><div class="casAuthors">Vyas, Vivek K.; Ghate, Manjunath; Goel, Ashutosh</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">17-25</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Protein kinase B (PKB) is a key mediator of proliferation and survival pathways that are crit. for cancer growth.  Therefore, inhibitors of PKB are useful agents for the treatment of cancer.  Herein, we describe pharmacophore-based virtual screening combined with docking study as a rational strategy for identification of novel hits or leads.  Pharmacophore models of PKB β inhibitors were established using the DISCOtech and refined with GASP from compds. with IC50 values ranging from 2.2 to 246 nM.  The best pharmacophore model consists of one hydrogen bond acceptor (HBA), one hydrogen bond donor (HBD) site and two hydrophobic (HY) features.  The pharmacophore models were validated through receiver operating characteristic (ROC) and Guener-Henry (GH) scoring methods indicated that the model-3 was statistically valuable and reliable in identifying PKB β inhibitors.  Pharmacophore model as a 3D search query was searched against NCI database.  Several compds. with different structures (scaffolds) were retrieved as hits.  Mols. with a Qfit value of more than 95 and three other known inhibitors were docked in the active site of PKB to further explore the binding mode of these compds.  Finally in silico pharmacokinetic and toxicities were predicted for active hit mols.  The hits reported here showed good potential to be PKB β inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbLeReas0BxbVg90H21EOLACvtfcHk0ljArnYSK06DuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsVykur0%253D&md5=c1059c14a682402903c98f04f5ae65df</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2013.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2013.01.010%26sid%3Dliteratum%253Aachs%26aulast%3DVyas%26aufirst%3DV.%2BK.%26aulast%3DGhate%26aufirst%3DM.%26aulast%3DGoel%26aufirst%3DA.%26atitle%3DPharmacophore%2520modeling%252C%2520virtual%2520screening%252C%2520docking%2520and%2520in%2520silico%2520ADMET%2520analysis%2520of%2520protein%2520kinase%2520B%2520%2528PKB%2520beta%2529%2520inhibitors%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2013%26volume%3D42%26spage%3D17%26epage%3D25%26doi%3D10.1016%2Fj.jmgm.2013.01.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group">Xiao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, Y.</span><span> </span><span class="NLM_article-title">Effective virtual screening strategy focusing on the identification of novel Bruton’s tyrosine kinase inhibitors</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">142</span><span class="NLM_x">–</span> <span class="NLM_lpage">154</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2015.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.jmgm.2015.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXos1Olu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2015&pages=142-154&author=J.+Xiaoauthor=S.+Zhangauthor=M.+Luoauthor=Y.+Zouauthor=Y.+Zhangauthor=Y.+Lai&title=Effective+virtual+screening+strategy+focusing+on+the+identification+of+novel+Bruton%E2%80%99s+tyrosine+kinase+inhibitors&doi=10.1016%2Fj.jmgm.2015.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Effective virtual screening strategy focusing on the identification of novel Bruton's tyrosine kinase inhibitors</span></div><div class="casAuthors">Xiao, Jianhu; Zhang, Shengping; Luo, Minghao; Zou, Yi; Zhang, Yihua; Lai, Yisheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">142-154</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Dysregulation of the B-cell receptor (BCR) signaling pathway plays a vital role in the pathogenesis and development of B-cell malignancies.  Bruton's tyrosine kinase (BTK), a key component in the BCR signaling, has been validated as a valuable target for the treatment of B-cell malignancies.  In an attempt to find novel and potent BTK inhibitors, both ligand- and structure-based pharmacophore models were generated using Discovery Studio 2.5 and Ligandscout 3.11 with the aim of screening the ChemBridge database.  The resulting hits were then subjected to sequential docking expts. using two independent docking programs, CDOCKER and Glide.  Mols. displaying high glide scores and H-bond interactions with the key residue Met477 in both of the docking programs were retained.  Drug-like criteria including Lipinski's rule of five and ADMET properties filters were employed for further refinement of the retrieved hits.  By clustering, eight promising compds. with novel chem. scaffolds were finally selected and the top two ranking compds. were evaluated by mol. dynamics simulation.  We believe that these compds. are of great potential in BTK inhibition and will be used for further investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohhNhLGO9zSLVg90H21EOLACvtfcHk0ljxsSt5qmQviw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXos1Olu74%253D&md5=77035d3cf6b0bba7db7e104ce596c859</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2015.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2015.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DZou%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLai%26aufirst%3DY.%26atitle%3DEffective%2520virtual%2520screening%2520strategy%2520focusing%2520on%2520the%2520identification%2520of%2520novel%2520Bruton%25E2%2580%2599s%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2015%26volume%3D60%26spage%3D142%26epage%3D154%26doi%3D10.1016%2Fj.jmgm.2015.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group">Wilsey, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurka, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toth, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franco, J.</span><span> </span><span class="NLM_article-title">A large scale virtual screen of DprE1</span> <span class="citation_source-journal">Comput. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">121</span><span class="NLM_x">–</span> <span class="NLM_lpage">125</span><span class="refDoi"> DOI: 10.1016/j.compbiolchem.2013.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.compbiolchem.2013.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24055764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVehtrfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2013&pages=121-125&author=C.+Wilseyauthor=J.+Gurkaauthor=D.+Tothauthor=J.+Franco&title=A+large+scale+virtual+screen+of+DprE1&doi=10.1016%2Fj.compbiolchem.2013.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">A large scale virtual screen of DprE1</span></div><div class="casAuthors">Wilsey, Claire; Gurka, Jessica; Toth, David; Franco, Jimmy</div><div class="citationInfo"><span class="NLM_cas:title">Computational Biology and Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">121-125</span>CODEN:
                <span class="NLM_cas:coden">CBCOCH</span>;
        ISSN:<span class="NLM_cas:issn">1476-9271</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Tuberculosis continues to plague the world with the World Health Organization estg. that about one third of the world's population is infected.  Due to the emergence of MDR and XDR strains of TB, the need for novel therapeutics has become increasing urgent.  Herein the authors report the results of a virtual screen of 4.1 million compds. against a promising drug target, DrpE1.  The virtual compds. were obtained from the Zinc docking site and screened using the mol. docking program, AutoDock Vina.  The computational hits have led to the identification of several promising lead compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC9CN2V_DDirVg90H21EOLACvtfcHk0ljxsSt5qmQviw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVehtrfF&md5=1b2a306a2e3c40a8b94c0340cdca352e</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1016%2Fj.compbiolchem.2013.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.compbiolchem.2013.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DWilsey%26aufirst%3DC.%26aulast%3DGurka%26aufirst%3DJ.%26aulast%3DToth%26aufirst%3DD.%26aulast%3DFranco%26aufirst%3DJ.%26atitle%3DA%2520large%2520scale%2520virtual%2520screen%2520of%2520DprE1%26jtitle%3DComput.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D47%26spage%3D121%26epage%3D125%26doi%3D10.1016%2Fj.compbiolchem.2013.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group">Messaoudi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belguith, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ben Hamida, J.</span><span> </span><span class="NLM_article-title">Homology modeling and virtual screening approaches to identify potent inhibitors of VEB-1 beta-lactamase</span> <span class="citation_source-journal">Theor. Biol. Med. Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">22</span><span class="refDoi"> DOI: 10.1186/1742-4682-10-22</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1186%2F1742-4682-10-22" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=23547944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvFWms70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=22&author=A.+Messaoudiauthor=H.+Belguithauthor=J.+Ben+Hamida&title=Homology+modeling+and+virtual+screening+approaches+to+identify+potent+inhibitors+of+VEB-1+beta-lactamase&doi=10.1186%2F1742-4682-10-22"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Homology modeling and virtual screening approaches to identify potent inhibitors of VEB-1 β-lactamase</span></div><div class="casAuthors">Messaoudi, Abdelmonaem; Belguith, Hatem; Ben Hamida, Jeannette</div><div class="citationInfo"><span class="NLM_cas:title">Theoretical Biology and Medical Modelling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">22</span>CODEN:
                <span class="NLM_cas:coden">TBMMA8</span>;
        ISSN:<span class="NLM_cas:issn">1742-4682</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: blaVEB-1 is an integron-located extended-spectrum β-lactamase gene initially detected in Escherichia coli and Pseudomonas aeruginosa strains from South-East Asia.  Several recent studies have reported that VEB-1-pos. strains are highly resistant to ceftazidime, cefotaxime and aztreonam antibiotics.  One strategy to overcome resistance involves administering antibiotics together with β-lactamase inhibitors during the treatment of infectious diseases.  During this study, four VEB-1 β-lactamase inhibitors were identified using computer-aided drug design.  Methods: The SWISS-MODEL tool was utilized to generate three dimensional structures of VEB-1 β-lactamase, and the 3D model VEB-1 was verified using PROCHECK, ERRAT and VERIFY 3D programs.  Virtual screening was performed by docking inhibitors obtained from the ZINC Database to the active site of the VEB-1 protein using AutoDock Vina software.  Results and Conclusion: Homol. modeling studies were performed to obtain a three-dimensional structure of VEB-1 β-lactamase.  The generated model was validated, and virtual screening of a large chem. ligand library with docking simulations was performed using AutoDock software with the ZINC database.  On the basis of the dock-score, four mols. were subjected to ADME/TOX anal., with ZINC4085364 emerging as the most potent inhibitor of the VEB-1 β-lactamase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCOH7dOHnC6rVg90H21EOLACvtfcHk0ljxsSt5qmQviw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvFWms70%253D&md5=4ededf14f6224c5629aa7d9ca7075631</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1186%2F1742-4682-10-22&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1742-4682-10-22%26sid%3Dliteratum%253Aachs%26aulast%3DMessaoudi%26aufirst%3DA.%26aulast%3DBelguith%26aufirst%3DH.%26aulast%3DBen%2BHamida%26aufirst%3DJ.%26atitle%3DHomology%2520modeling%2520and%2520virtual%2520screening%2520approaches%2520to%2520identify%2520potent%2520inhibitors%2520of%2520VEB-1%2520beta-lactamase%26jtitle%3DTheor.%2520Biol.%2520Med.%2520Modell.%26date%3D2013%26volume%3D10%26spage%3D22%26doi%3D10.1186%2F1742-4682-10-22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group">Nogara, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saraiva, R.d.A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caeran Bueno, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lissner, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lenz Dalla Corte, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braga, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosemberg, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocha, J. B.</span><span> </span><span class="NLM_article-title">Virtual screening of acetylcholinesterase inhibitors using the Lipinski’s rule of five and ZINC databank</span> <span class="citation_source-journal">BioMed Res. Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">2015</span><span class="NLM_x">, </span> <span class="NLM_fpage">870389</span><span class="refDoi"> DOI: 10.1155/2015/870389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1155%2F2015%2F870389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25685814" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BC2MrmsVyhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2015&publication_year=2015&pages=870389&author=P.+A.+Nogaraauthor=R.d.A.+Saraivaauthor=D.+Caeran+Buenoauthor=L.+J.+Lissnerauthor=C.+Lenz+Dalla+Corteauthor=M.+M.+Bragaauthor=D.+B.+Rosembergauthor=J.+B.+Rocha&title=Virtual+screening+of+acetylcholinesterase+inhibitors+using+the+Lipinski%E2%80%99s+rule+of+five+and+ZINC+databank&doi=10.1155%2F2015%2F870389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening of acetylcholinesterase inhibitors using the Lipinski's rule of five and ZINC databank</span></div><div class="casAuthors">Nogara Pablo Andrei; Caeran Bueno Diones; Lissner Lilian Juliana; Lenz Dalla Corte Cristiane; Braga Marcos M; Rosemberg Denis Broock; Rocha Joao Batista Teixeira; Saraiva Rogerio de Aquino</div><div class="citationInfo"><span class="NLM_cas:title">BioMed research international</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">2015</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">870389</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Alzheimer's disease (AD) is a progressive and neurodegenerative pathology that can affect people over 65 years of age.  It causes several complications, such as behavioral changes, language deficits, depression, and memory impairments.  One of the methods used to treat AD is the increase of acetylcholine (ACh) in the brain by using acetylcholinesterase inhibitors (AChEIs).  In this study, we used the ZINC databank and the Lipinski's rule of five to perform a virtual screening and a molecular docking (using Auto Dock Vina 1.1.1) aiming to select possible compounds that have quaternary ammonium atom able to inhibit acetylcholinesterase (AChE) activity.  The molecules were obtained by screening and further in vitro assays were performed to analyze the most potent inhibitors through the IC50 value and also to describe the interaction models between inhibitors and enzyme by molecular docking.  The results showed that compound D inhibited AChE activity from different vertebrate sources and butyrylcholinesterase (BChE) from Equus ferus (EfBChE), with IC50 ranging from 1.69 ± 0.46 to 5.64 ± 2.47 μM.  Compound D interacted with the peripheral anionic subsite in both enzymes, blocking substrate entrance to the active site.  In contrast, compound C had higher specificity as inhibitor of EfBChE.  In conclusion, the screening was effective in finding inhibitors of AChE and BuChE from different organisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTBBryFJmsYVNTiMd2aNQVUfW6udTcc2ebR053w804RHLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MrmsVyhsw%253D%253D&md5=6f41ff4cd81fe9d0e71edc03e9933fc1</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1155%2F2015%2F870389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2015%252F870389%26sid%3Dliteratum%253Aachs%26aulast%3DNogara%26aufirst%3DP.%2BA.%26aulast%3DSaraiva%26aufirst%3DR.d.A.%26aulast%3DCaeran%2BBueno%26aufirst%3DD.%26aulast%3DLissner%26aufirst%3DL.%2BJ.%26aulast%3DLenz%2BDalla%2BCorte%26aufirst%3DC.%26aulast%3DBraga%26aufirst%3DM.%2BM.%26aulast%3DRosemberg%26aufirst%3DD.%2BB.%26aulast%3DRocha%26aufirst%3DJ.%2BB.%26atitle%3DVirtual%2520screening%2520of%2520acetylcholinesterase%2520inhibitors%2520using%2520the%2520Lipinski%25E2%2580%2599s%2520rule%2520of%2520five%2520and%2520ZINC%2520databank%26jtitle%3DBioMed%2520Res.%2520Int.%26date%3D2015%26volume%3D2015%26spage%3D870389%26doi%3D10.1155%2F2015%2F870389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group">Regon, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gogoi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rai, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bordoloi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bezbaruah, R. L.</span><span> </span><span class="NLM_article-title">Design of inhibitors of the HIV-1 integrase core domain using virtual screening</span> <span class="citation_source-journal">Bioinformation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">76</span><span class="NLM_x">–</span> <span class="NLM_lpage">80</span><span class="refDoi"> DOI: 10.6026/97320630010076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.6026%2F97320630010076" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=76-80&author=P.+Regonauthor=D.+Gogoiauthor=A.+K.+Raiauthor=M.+Bordoloiauthor=R.+L.+Bezbaruah&title=Design+of+inhibitors+of+the+HIV-1+integrase+core+domain+using+virtual+screening&doi=10.6026%2F97320630010076"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.6026%2F97320630010076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.6026%252F97320630010076%26sid%3Dliteratum%253Aachs%26aulast%3DRegon%26aufirst%3DP.%26aulast%3DGogoi%26aufirst%3DD.%26aulast%3DRai%26aufirst%3DA.%2BK.%26aulast%3DBordoloi%26aufirst%3DM.%26aulast%3DBezbaruah%26aufirst%3DR.%2BL.%26atitle%3DDesign%2520of%2520inhibitors%2520of%2520the%2520HIV-1%2520integrase%2520core%2520domain%2520using%2520virtual%2520screening%26jtitle%3DBioinformation%26date%3D2014%26volume%3D10%26spage%3D76%26epage%3D80%26doi%3D10.6026%2F97320630010076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group">Saraswat, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nehra, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaudhary, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, C. V.</span><span> </span><span class="NLM_article-title">In-silico screening and in-vitro validation of vascular endothelial growth factor receptor-2 (VEGFR-2) inhibitors</span> <span class="citation_source-journal">Bioinformation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">273</span><span class="NLM_x">–</span> <span class="NLM_lpage">280</span><span class="refDoi"> DOI: 10.6026/97320630010273</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.6026%2F97320630010273" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=273-280&author=D.+Saraswatauthor=S.+Nehraauthor=K.+K.+Chaudharyauthor=C.+V.+Prasad&title=In-silico+screening+and+in-vitro+validation+of+vascular+endothelial+growth+factor+receptor-2+%28VEGFR-2%29+inhibitors&doi=10.6026%2F97320630010273"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.6026%2F97320630010273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.6026%252F97320630010273%26sid%3Dliteratum%253Aachs%26aulast%3DSaraswat%26aufirst%3DD.%26aulast%3DNehra%26aufirst%3DS.%26aulast%3DChaudhary%26aufirst%3DK.%2BK.%26aulast%3DPrasad%26aufirst%3DC.%2BV.%26atitle%3DIn-silico%2520screening%2520and%2520in-vitro%2520validation%2520of%2520vascular%2520endothelial%2520growth%2520factor%2520receptor-2%2520%2528VEGFR-2%2529%2520inhibitors%26jtitle%3DBioinformation%26date%3D2014%26volume%3D10%26spage%3D273%26epage%3D280%26doi%3D10.6026%2F97320630010273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group">Giri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taj, G.</span><span> </span><span class="NLM_article-title">In silico-prediction of downstream MYB interacting partners of MAPK3 in Arabidopsis</span> <span class="citation_source-journal">Bioinformation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">721</span><span class="NLM_x">–</span> <span class="NLM_lpage">725</span><span class="refDoi"> DOI: 10.6026/97320630010721</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.6026%2F97320630010721" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=721-725&author=P.+Giriauthor=A.+Kumarauthor=G.+Taj&title=In+silico-prediction+of+downstream+MYB+interacting+partners+of+MAPK3+in+Arabidopsis&doi=10.6026%2F97320630010721"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.6026%2F97320630010721&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.6026%252F97320630010721%26sid%3Dliteratum%253Aachs%26aulast%3DGiri%26aufirst%3DP.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DTaj%26aufirst%3DG.%26atitle%3DIn%2520silico-prediction%2520of%2520downstream%2520MYB%2520interacting%2520partners%2520of%2520MAPK3%2520in%2520Arabidopsis%26jtitle%3DBioinformation%26date%3D2014%26volume%3D10%26spage%3D721%26epage%3D725%26doi%3D10.6026%2F97320630010721" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group">Alhazmi, M. I.</span><span> </span><span class="NLM_article-title">Molecular docking of selected phytocompounds with H1N1 Proteins</span> <span class="citation_source-journal">Bioinformation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">196</span><span class="NLM_x">–</span> <span class="NLM_lpage">202</span><span class="refDoi"> DOI: 10.6026/97320630011196</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.6026%2F97320630011196" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=196-202&author=M.+I.+Alhazmi&title=Molecular+docking+of+selected+phytocompounds+with+H1N1+Proteins&doi=10.6026%2F97320630011196"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.6026%2F97320630011196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.6026%252F97320630011196%26sid%3Dliteratum%253Aachs%26aulast%3DAlhazmi%26aufirst%3DM.%2BI.%26atitle%3DMolecular%2520docking%2520of%2520selected%2520phytocompounds%2520with%2520H1N1%2520Proteins%26jtitle%3DBioinformation%26date%3D2015%26volume%3D11%26spage%3D196%26epage%3D202%26doi%3D10.6026%2F97320630011196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group">Shao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span> </span><span class="NLM_article-title">Dual-inhibitors of STAT5 and STAT3: studies from molecular docking and molecular dynamics simulations</span> <span class="citation_source-journal">J. Mol. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">2399</span><span class="refDoi"> DOI: 10.1007/s00894-014-2399-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs00894-014-2399-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25098340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BC2cbps1elsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=2399&author=S.+Shaoauthor=R.+Yuauthor=Y.+Yuauthor=Y.+Li&title=Dual-inhibitors+of+STAT5+and+STAT3%3A+studies+from+molecular+docking+and+molecular+dynamics+simulations&doi=10.1007%2Fs00894-014-2399-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Dual-inhibitors of STAT5 and STAT3: studies from molecular docking and molecular dynamics simulations</span></div><div class="casAuthors">Shao Shengjuan; Yu Rilei; Yu Yanqing; Li Yanni</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2399</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although molecularly targeted therapy with imatinib has improved treatments of chronic myeloid leukemia (CML), clinical resistance gradually develops in patients with accelerated or blast phase CML.  The inability of imatinib to cure CML suggests that inactivation of BCR-ABL kinase activity alone is not sufficient to control the disease.  Aberrant STAT signaling and constitutive STAT5 or STAT3 activation are frequently found in both acute and chronic leukemia.  Constitutive activation of STAT5 and STAT3 are associated with imatinib resistance on leukemia cells.  Development of drugs targeting SH2 domains of STAT5 and STAT3 provides a novel strategy for the treatment of the imatinib-resistant CML.  Here, molecular docking and molecular dynamics simulations were used to investigate the interactions of the drugs targeting STAT3 and STAT5 receptors at molecular level.  The calculated binding free energies are consistent with the ranking of the experimental affinities and our simulations also explained their differences in binding energy.  Then virtual screening based on molecular docking and molecular dynamics was applied to screen a set of ~1500 compounds for dual inhibitors of the SH2 domains of STAT5 and STAT3.  Three top score compounds obtained in virtual screening were compound 660, 304, and 561.  Results show that the three predicted dual-inhibitors are well fitted within the two binding domains and are predicted to present improved STAT5 and STAT3 SH2 inhibitory activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRDm_Hf0VLBp7P3FVUi_gf8fW6udTcc2ebR053w804RHLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cbps1elsg%253D%253D&md5=54b44aed17241d682d5302cfb79eb314</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1007%2Fs00894-014-2399-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00894-014-2399-x%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26atitle%3DDual-inhibitors%2520of%2520STAT5%2520and%2520STAT3%253A%2520studies%2520from%2520molecular%2520docking%2520and%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Mol.%2520Model.%26date%3D2014%26volume%3D20%26spage%3D2399%26doi%3D10.1007%2Fs00894-014-2399-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group">Sun, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bao, J.</span><span> </span><span class="NLM_article-title">Screening of novel inhibitors targeting lactate dehydrogenase A via four molecular docking strategies and dynamics simulations</span> <span class="citation_source-journal">J. Mol. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">133</span><span class="refDoi"> DOI: 10.1007/s00894-015-2675-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs00894-015-2675-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25934158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BC2MfgtFylsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=133&author=R.+Sunauthor=X.+Liauthor=Y.+Liauthor=X.+Zhangauthor=X.+Liauthor=X.+Liauthor=Z.+Shiauthor=J.+Bao&title=Screening+of+novel+inhibitors+targeting+lactate+dehydrogenase+A+via+four+molecular+docking+strategies+and+dynamics+simulations&doi=10.1007%2Fs00894-015-2675-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Screening of novel inhibitors targeting lactate dehydrogenase A via four molecular docking strategies and dynamics simulations</span></div><div class="casAuthors">Sun Rong; Li Xin; Li Yuanyuan; Zhang Xun; Li Xinru; Li Xiaoyu; Shi Zheng; Bao Jinku</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">133</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Lactate dehydrogenase A (LDHA) is a metabolic enzyme which catalyzes the interconversion of lactate and pyruvate in the glycolysis pathway, thus playing key roles in aerobic glycolysis.  The inhibition of LDHA by small molecules has become an attractive strategy for anticancer therapy in recent years.  However, very few LDHA inhibitors have been reported, even though a great deal of effort has directed into identifying LDHA inhibitors using structure-based approaches.  Therefore, high-throughput and high-accuracy screening approaches are still urgently needed in order to target LDHA effectively.  In the present work, after establishing that our docking strategies performed well using test datasets, we screened 32791 Specs products for their docking scores with the substrate-binding pocket and, separately, the cofactor-binding pocket of LDHA.  We subsequently identified 76 hits (i.e., ligands that show low docking scores) for the cofactor-binding pocket and 27 hits for the substrate-binding pocket.  Two representative compounds, ZINC20036549 and ZINC19369718, were then chosen for further MD simulation analysis, and we found that these compounds maintained their inhibitory activity during the MD simulations.  Meanwhile, we found that ZINC19369718 interacts with a novel binding site close to the active site, and that this interaction may inhibit the catalytic activity of LDHA.  Together, these results offer not only a new paradigm for identifying Specs drug-like products for novel therapeutic use but they also provide further opportunity to adopt LDHA inhibition as a strategy for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcROAp0PJSifc_IW7hznt0zpfW6udTcc2eYmJGPphGdZ97ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MfgtFylsQ%253D%253D&md5=9dc35b7875893ee56d95090473065512</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1007%2Fs00894-015-2675-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00894-015-2675-4%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DBao%26aufirst%3DJ.%26atitle%3DScreening%2520of%2520novel%2520inhibitors%2520targeting%2520lactate%2520dehydrogenase%2520A%2520via%2520four%2520molecular%2520docking%2520strategies%2520and%2520dynamics%2520simulations%26jtitle%3DJ.%2520Mol.%2520Model.%26date%3D2015%26volume%3D21%26spage%3D133%26doi%3D10.1007%2Fs00894-015-2675-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group">Sgrignani, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novati, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colombo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grazioso, G.</span><span> </span><span class="NLM_article-title">Covalent docking of selected boron-based serine beta-lactamase inhibitors</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">441</span><span class="NLM_x">–</span> <span class="NLM_lpage">450</span><span class="refDoi"> DOI: 10.1007/s10822-015-9834-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs10822-015-9834-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=441-450&author=J.+Sgrignaniauthor=B.+Novatiauthor=G.+Colomboauthor=G.+Grazioso&title=Covalent+docking+of+selected+boron-based+serine+beta-lactamase+inhibitors&doi=10.1007%2Fs10822-015-9834-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9834-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9834-7%26sid%3Dliteratum%253Aachs%26aulast%3DSgrignani%26aufirst%3DJ.%26aulast%3DNovati%26aufirst%3DB.%26aulast%3DColombo%26aufirst%3DG.%26aulast%3DGrazioso%26aufirst%3DG.%26atitle%3DCovalent%2520docking%2520of%2520selected%2520boron-based%2520serine%2520beta-lactamase%2520inhibitors%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D441%26epage%3D450%26doi%3D10.1007%2Fs10822-015-9834-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group">Ogrizek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lesnik, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sosic, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodoscek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirkovic, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kos, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janezic, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gobec, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konc, J.</span><span> </span><span class="NLM_article-title">Molecular dynamics to enhance structure-based virtual screening on cathepsin B</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">707</span><span class="NLM_x">–</span> <span class="NLM_lpage">712</span><span class="refDoi"> DOI: 10.1007/s10822-015-9847-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs10822-015-9847-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25947277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnvFChsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=707-712&author=M.+Ogrizekauthor=S.+Turkauthor=S.+Lesnikauthor=I.+Sosicauthor=M.+Hodoscekauthor=B.+Mirkovicauthor=J.+Kosauthor=D.+Janezicauthor=S.+Gobecauthor=J.+Konc&title=Molecular+dynamics+to+enhance+structure-based+virtual+screening+on+cathepsin+B&doi=10.1007%2Fs10822-015-9847-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics to enhance structure-based virtual screening on cathepsin B</span></div><div class="casAuthors">Ogrizek, Mitja; Turk, Samo; Lesnik, Samo; Sosic, Izidor; Hodoscek, Milan; Mirkovic, Bojana; Kos, Janko; Janezic, Dusanka; Gobec, Stanislav; Konc, Janez</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">707-712</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Mol. dynamics (MD) and mol. docking are commonly used to study mol. interactions in drug discovery.  Most docking approaches consider proteins as rigid, which can decrease the accuracy of predicted docked poses.  Therefore MD simulations can be used prior to docking to add flexibility to proteins.  We evaluated the contribution of using MD together with docking in a docking study on human cathepsin B, a well-studied protein involved in numerous pathol. processes.  Using CHARMM biomol. simulation program and AutoDock Vina mol. docking program, we found, that short MD simulations significantly improved mol. docking.  Our results, expressed with the area under the receiver operating characteristic curves, show an increase in discriminatory power i.e. the ability to discriminate active from inactive compds. of mol. docking, when docking is performed to selected snapshots from MD simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfD2hUeaLu7rVg90H21EOLACvtfcHk0ljYYv5A_pMiCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnvFChsb8%253D&md5=140a6ee2d68d50a93f78291a30ed0c4d</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9847-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9847-2%26sid%3Dliteratum%253Aachs%26aulast%3DOgrizek%26aufirst%3DM.%26aulast%3DTurk%26aufirst%3DS.%26aulast%3DLesnik%26aufirst%3DS.%26aulast%3DSosic%26aufirst%3DI.%26aulast%3DHodoscek%26aufirst%3DM.%26aulast%3DMirkovic%26aufirst%3DB.%26aulast%3DKos%26aufirst%3DJ.%26aulast%3DJanezic%26aufirst%3DD.%26aulast%3DGobec%26aufirst%3DS.%26aulast%3DKonc%26aufirst%3DJ.%26atitle%3DMolecular%2520dynamics%2520to%2520enhance%2520structure-based%2520virtual%2520screening%2520on%2520cathepsin%2520B%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D707%26epage%3D712%26doi%3D10.1007%2Fs10822-015-9847-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group">Singh, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pandey, P. N.</span><span> </span><span class="NLM_article-title">Molecular docking and molecular dynamics study on SmHDAC1 to identify potential lead compounds against Schistosomiasis</span> <span class="citation_source-journal">Mol. Biol. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">689</span><span class="NLM_x">–</span> <span class="NLM_lpage">698</span><span class="refDoi"> DOI: 10.1007/s11033-014-3816-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs11033-014-3816-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25663090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitlaqtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2015&pages=689-698&author=R.+Singhauthor=P.+N.+Pandey&title=Molecular+docking+and+molecular+dynamics+study+on+SmHDAC1+to+identify+potential+lead+compounds+against+Schistosomiasis&doi=10.1007%2Fs11033-014-3816-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular docking and molecular dynamics study on SmHDAC1 to identify potential lead compounds against Schistosomiasis</span></div><div class="casAuthors">Singh, Raghvendra; Pandey, Paras Nath</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">689-698</span>CODEN:
                <span class="NLM_cas:coden">MLBRBU</span>;
        ISSN:<span class="NLM_cas:issn">0301-4851</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Schistosomiasis, a disease caused by helminth parasites of genus Schistosoma.  Its treatment intensively depends on single drug, praziquantel which increases the risk of development of drug-resistant parasite.  Inhibitors of human HDAC are profoundly reported as novel anti-cancer drugs and used as new anit-parasitic agents.  Schistosomamonsoni class I HDACs are expressed in all stages of life cycle and indicating that this enzyme is most likely a major target for the designing specific inhibitors.  In order to find novel target for the treatment of Schistosomiasis, three dimensional structure of SmHDAC1 was generated, using homol. modeling.  Features of the generated structure, was then deduced with respect to conformation of peptide backbone, local compatibility of the generated structure in terms of energy and mol. dynamics study.  Considering these features of the generated structure, we selected all the class 1 inhibitors reported so far, which showed interactions with HDACs.  Virtual screening was done using reported inhibitors (70) and using SmHDAC1 and HsHDAC1 as the targets.  On the basis of binding affinity and IC50 value, 24th ligand was selected for the mol. docking purpose.  In this study, out of all the reported inhibitors, 24th inhibitor (N,8-dihydroxy-8-(naphthalen-2-yl) octanamide zinc id- ZINC13474421) showed better binding with SmHDAC1 (-8.1 kcal/mol) as compared to HsHDAC1 (-6.4 kcal/mol) in terms of binding energy and supported by IC50 value.  This paper throws light on the reliable model for further structure based drug designing, concerning SmHDAC1 of S. mansoni.  Mol. docking studies highlighted advantages of comparative in silico interaction studies of SmHDAC1 and HsHDAC1.  N,8-dihydroxy-8-(naphthalen-2-yl) octanamide can further use for the clin. trial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqumvOBY0VSiLVg90H21EOLACvtfcHk0ljAYZPU_5mTyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitlaqtLc%253D&md5=eed4a7eaf7bfbb7ee97fe3599fc56c99</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1007%2Fs11033-014-3816-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11033-014-3816-z%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DR.%26aulast%3DPandey%26aufirst%3DP.%2BN.%26atitle%3DMolecular%2520docking%2520and%2520molecular%2520dynamics%2520study%2520on%2520SmHDAC1%2520to%2520identify%2520potential%2520lead%2520compounds%2520against%2520Schistosomiasis%26jtitle%3DMol.%2520Biol.%2520Rep.%26date%3D2015%26volume%3D42%26spage%3D689%26epage%3D698%26doi%3D10.1007%2Fs11033-014-3816-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group">Pinheiro, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duarte, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alves, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Molfetta, F. A.</span><span> </span><span class="NLM_article-title">Virtual screening and molecular dynamics simulations from a bank of molecules of the Amazon region against functional NS3-4A protease-helicase enzyme of hepatitis C virus</span> <span class="citation_source-journal">Appl. Biochem. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">176</span><span class="NLM_x">, </span> <span class="NLM_fpage">1709</span><span class="NLM_x">–</span> <span class="NLM_lpage">1721</span><span class="refDoi"> DOI: 10.1007/s12010-015-1672-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs12010-015-1672-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26009474" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovFOgsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=176&publication_year=2015&pages=1709-1721&author=A.+S.+Pinheiroauthor=J.+B.+Duarteauthor=C.+N.+Alvesauthor=F.+A.+de+Molfetta&title=Virtual+screening+and+molecular+dynamics+simulations+from+a+bank+of+molecules+of+the+Amazon+region+against+functional+NS3-4A+protease-helicase+enzyme+of+hepatitis+C+virus&doi=10.1007%2Fs12010-015-1672-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Screening and Molecular Dynamics Simulations from a Bank of Molecules of the Amazon Region Against Functional NS3-4A Protease-Helicase Enzyme of Hepatitis C Virus</span></div><div class="casAuthors">Pinheiro, Alan Sena; Duarte, Jaqueline Bianca Carvalho; Alves, Claudio Nahum; Alberto de Molfetta, Fabio</div><div class="citationInfo"><span class="NLM_cas:title">Applied Biochemistry and Biotechnology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">176</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1709-1721</span>CODEN:
                <span class="NLM_cas:coden">ABIBDL</span>;
        ISSN:<span class="NLM_cas:issn">0273-2289</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Hepatitis C virus (HCV) infection is a disease that affects approx. 3 % of the global population and requires new therapeutic agents without the inconvenience assocd. with current anti-HCV treatment.  This paper reports on a study of a virtual screening and a mol. dynamics simulation of compds. derived from natural products from the Amazon region that are potentially effective against the NS3-4A enzyme of HCV, which plays an important role in the replication process of this virus.  According to the results of the mol. docking calcns. and subsequent consensual anal., the best scored compds. showed interactions between hydrogen and residues of the catalytic triad as well as interactions with residues that guide ligands to the active site of the enzyme.  They also showed stability in the mol. dynamics simulation, as the structures preserved important interactions at the active site of the enzyme.  The root mean square deviation (RMSD) values were stabilized at the end of the simulation time.  Such compds. are considered promising as novel therapies against HCV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwbaCUHstharVg90H21EOLACvtfcHk0ljAYZPU_5mTyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovFOgsLw%253D&md5=6fc53bd1f1c8ec97bc98d4855c15a508</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1007%2Fs12010-015-1672-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12010-015-1672-5%26sid%3Dliteratum%253Aachs%26aulast%3DPinheiro%26aufirst%3DA.%2BS.%26aulast%3DDuarte%26aufirst%3DJ.%2BB.%26aulast%3DAlves%26aufirst%3DC.%2BN.%26aulast%3Dde%2BMolfetta%26aufirst%3DF.%2BA.%26atitle%3DVirtual%2520screening%2520and%2520molecular%2520dynamics%2520simulations%2520from%2520a%2520bank%2520of%2520molecules%2520of%2520the%2520Amazon%2520region%2520against%2520functional%2520NS3-4A%2520protease-helicase%2520enzyme%2520of%2520hepatitis%2520C%2520virus%26jtitle%3DAppl.%2520Biochem.%2520Biotechnol.%26date%3D2015%26volume%3D176%26spage%3D1709%26epage%3D1721%26doi%3D10.1007%2Fs12010-015-1672-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group">Wang, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xue, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, F.</span><span> </span><span class="NLM_article-title">Identification of dual active agents targeting 5-HT1A and SERT by combinatorial virtual screening methods</span> <span class="citation_source-journal">Bio-Med. Mater. Eng.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">2233</span><span class="NLM_x">–</span> <span class="NLM_lpage">2239</span><span class="refDoi"> DOI: 10.3233/BME-151529</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.3233%2FBME-151529" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlyjsrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2015&pages=2233-2239&issue=Suppl.+1&author=P.+Wangauthor=F.+Yangauthor=H.+Yangauthor=X.+Xuauthor=D.+Liuauthor=W.+Xueauthor=F.+Zhu&title=Identification+of+dual+active+agents+targeting+5-HT1A+and+SERT+by+combinatorial+virtual+screening+methods&doi=10.3233%2FBME-151529"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of dual active agents targeting 5-HT1A and SERT by combinatorial virtual screening methods</span></div><div class="casAuthors">Liu, Feng; Lee, Dong-Hoon; Lagoa, Ricardo; Kumar, Sandeep; Wang, Panpan; Yang, Fengyuan; Yang, Hong; Xu, Xiaofei; Liu, Duo; Xue, Weiwei; Zhu, Feng</div><div class="citationInfo"><span class="NLM_cas:title">Bio-Medical Materials and Engineering</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">s1</span>),
    <span class="NLM_cas:pages">S2233-S2239</span>CODEN:
                <span class="NLM_cas:coden">BMENEO</span>;
        ISSN:<span class="NLM_cas:issn">0959-2989</span>.
    
            (<span class="NLM_cas:orgname">IOS Press</span>)
        </div><div class="casAbstract">Selective serotonin reuptake inhibitors (SSRIs) are most adopted therapeutics marketed for major depression, and the efficacy of which are greatly reduced by their delayed onset of action and undesirable side effects. 5-HT1A receptor partial agonist and SERT inhibitor (SPARI) was proposed as a novel strategy to overcome the shortage of efficacy by a neg. feedback control of 5-HT1A receptor.  However, only one SPARI (vilazodone) has been approved for clin. use, and none is currently in clin. trial, which demonstrates a strong need for searching more novel SPARIs to facilitate antidepressants discovery.  This work applied a combinatorial virtual screening method (CVSM) by integrating multiple tools.  Statistic anal. reveals that CVSM surpasses single virtual screening methods in terms of hit rates and enrichment factors.  By adopting optimized CVSM, 91 promising dual target leads form 15 scaffolds were identified, and 40% of these scaffolds have already been reported to show antidepressant related therapeutic effects.  In sum, CVSM is capable in identifying novel SPARIs from large chem. libraries with extremely low false hit rate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr073ag0IyfrLVg90H21EOLACvtfcHk0ljAYZPU_5mTyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlyjsrjL&md5=0910120e940bbb1043a920b58be5798b</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.3233%2FBME-151529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3233%252FBME-151529%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DXue%26aufirst%3DW.%26aulast%3DZhu%26aufirst%3DF.%26atitle%3DIdentification%2520of%2520dual%2520active%2520agents%2520targeting%25205-HT1A%2520and%2520SERT%2520by%2520combinatorial%2520virtual%2520screening%2520methods%26jtitle%3DBio-Med.%2520Mater.%2520Eng.%26date%3D2015%26volume%3D26%26issue%3DSuppl.%25201%26spage%3D2233%26epage%3D2239%26doi%3D10.3233%2FBME-151529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group">Dhanjal, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sreenidhi, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bafna, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katiyar, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goyal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grover, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundar, D.</span><span> </span><span class="NLM_article-title">Computational structure-based de novo design of hypothetical inhibitors against the anti- inflammatory target COX-2</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0134691</span><span class="refDoi"> DOI: 10.1371/journal.pone.0134691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pone.0134691" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0134691&author=J.+K.+Dhanjalauthor=A.+K.+Sreenidhiauthor=K.+Bafnaauthor=S.+P.+Katiyarauthor=S.+Goyalauthor=A.+Groverauthor=D.+Sundar&title=Computational+structure-based+de+novo+design+of+hypothetical+inhibitors+against+the+anti-+inflammatory+target+COX-2&doi=10.1371%2Fjournal.pone.0134691"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0134691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0134691%26sid%3Dliteratum%253Aachs%26aulast%3DDhanjal%26aufirst%3DJ.%2BK.%26aulast%3DSreenidhi%26aufirst%3DA.%2BK.%26aulast%3DBafna%26aufirst%3DK.%26aulast%3DKatiyar%26aufirst%3DS.%2BP.%26aulast%3DGoyal%26aufirst%3DS.%26aulast%3DGrover%26aufirst%3DA.%26aulast%3DSundar%26aufirst%3DD.%26atitle%3DComputational%2520structure-based%2520de%2520novo%2520design%2520of%2520hypothetical%2520inhibitors%2520against%2520the%2520anti-%2520inflammatory%2520target%2520COX-2%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0134691%26doi%3D10.1371%2Fjournal.pone.0134691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group">He, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, G.</span><span> </span><span class="NLM_article-title">Discovery of potential drugs for human-infecting H7N9 virus containing R294K mutation</span> <span class="citation_source-journal">Drug Des., Dev. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2377</span><span class="NLM_x">–</span> <span class="NLM_lpage">2390</span><span class="refDoi"> DOI: 10.2147/DDDT.S74061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.2147%2FDDDT.S74061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25489236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFGhsb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=2377-2390&author=J.+Y.+Heauthor=C.+Liauthor=G.+Wu&title=Discovery+of+potential+drugs+for+human-infecting+H7N9+virus+containing+R294K+mutation&doi=10.2147%2FDDDT.S74061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of potential drugs for human-infecting H7N9 virus containing R294K mutation</span></div><div class="casAuthors">He, Jiao-Yu; Li, Cheng; Wu, Guo</div><div class="citationInfo"><span class="NLM_cas:title">Drug Design, Development and Therapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2377-2390</span>CODEN:
                <span class="NLM_cas:coden">DDDTAQ</span>;
        ISSN:<span class="NLM_cas:issn">1177-8881</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">Background: After the first epidemic wave from Feb. through May 2013, the influenza A (H7N9) virus emerged and has followed a second epidemic wave since June 2013.  As of June 27, 2014, the outbreak of H7N9 had caused 450 confirmed cases of human infection, with 165 deaths included.  The case-fatality rate of all confirmed cases is about 36%, making the H7N9 virus a significant threat to people's health.  At present, neuraminidase inhibitors are the only licensed antiviral medications available to treat H7N9 infections in humans.  Oseltamivir is the most commonly used inhibitor, and it is also a front-line drug for the threatening H7N9.  Unfortunately, it has been reported that patients treated with oseltamivir can induce R294K (Arg294Lys) substitution in the H7N9 virus, which is a rare mutation and can reduce the antiviral efficacy of inhibitors.  Even worse, deaths caused by such mutation after oseltamivir treatment have already been reported, indicating that the need to find substitutive neuraminidase inhibitors for currently available drugs to treat drug-resistant H7N9 is really pressing.  Materials and methods: First, the structure of H7N9 contg. the R294K substitution was downloaded from the Protein Data Bank, and structural information of approved drugs was downloaded from the ZINC (ZINC Is Not Com.) database.  Taking oseltamivir carboxylate as a ref. drug, we then filtered these mols. through virtual screening to find out potential inhibitors targeting the mutated H7N9 virus.  For further evaluation, we carried out a 14 ns mol. dynamic simulation for each H7N9-drug complex and calcd. the binding energy for each candidate drug.  Results: We found five inhibitors that could be candidate drugs for treating the mutated H7N9 virus.  Docking poses showed these drugs could bind to the virus effectively, with the contribution of hydrogen bonds and hydrophobic interactions.  With regard to the mol. dynamic simulations, receptor-ligand complexes formed by these candidate drugs were more stable than the one formed by oseltamivir carboxylate.  The binding energy of oseltamivir carboxylate was -122.4 kJ/mol, while those for these potential inhibitors were -417.5, -404.7, -372.2, -304.3, and -289.9 kJ/mol, much better than the ref. drug.  Conclusion: Given the current and future threat of the mutated H7N9 virus, it is urgent that potent drugs and effective antiviral therapeutics be found.  Our study therefore is able to complement currently available drugs for influenza A infectors and helps to prevent the ongoing threat of H7N9 virus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFU5IR7X5QELVg90H21EOLACvtfcHk0ljWqj3RLJL-tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFGhsb%252FF&md5=d253ca7cd12d234b39bec9f4d2cc3655</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.2147%2FDDDT.S74061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FDDDT.S74061%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DJ.%2BY.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520potential%2520drugs%2520for%2520human-infecting%2520H7N9%2520virus%2520containing%2520R294K%2520mutation%26jtitle%3DDrug%2520Des.%252C%2520Dev.%2520Ther.%26date%3D2014%26volume%3D8%26spage%3D2377%26epage%3D2390%26doi%3D10.2147%2FDDDT.S74061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group">Balakrishnan, N.; Raj, J. S.; Kandakatla, N.</span><span> </span><span class="NLM_article-title">Discovery of novel GSK-3beta inhibitors using pharmacophore and virtual screening studies</span>.  <span class="citation_source-journal">Interdiscip. Sci.: Comput. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span>DOI: <span class="refDoi"> DOI: 10.1007/s12539-015-0100-4</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs12539-015-0100-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=N.+Balakrishnan&author=J.+S.+Raj&author=N.+Kandakatla&title=Discovery+of+novel+GSK-3beta+inhibitors+using+pharmacophore+and+virtual+screening+studies&doi=10.1007%2Fs12539-015-0100-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1007%2Fs12539-015-0100-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12539-015-0100-4%26sid%3Dliteratum%253Aachs%26aulast%3DBalakrishnan%26aufirst%3DN.%26atitle%3DDiscovery%2520of%2520novel%2520GSK-3beta%2520inhibitors%2520using%2520pharmacophore%2520and%2520virtual%2520screening%2520studies%26jtitle%3DInterdiscip.%2520Sci.%253A%2520Comput.%2520Life%2520Sci.%26date%3D2015%26doi%3D10.1007%2Fs12539-015-0100-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group">Thai, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngo, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, T. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, N. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le, M. T.</span><span> </span><span class="NLM_article-title">Virtual screening for novel Staphylococcus Aureus NorA efflux pump inhibitors from natural products</span> <span class="citation_source-journal">Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">135</span><span class="NLM_x">–</span> <span class="NLM_lpage">155</span><span class="refDoi"> DOI: 10.2174/1573406410666140902110903</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.2174%2F1573406410666140902110903" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25181985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXislemtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=135-155&author=K.+M.+Thaiauthor=T.+D.+Ngoauthor=T.+V.+Phanauthor=T.+D.+Tranauthor=N.+V.+Nguyenauthor=T.+H.+Nguyenauthor=M.+T.+Le&title=Virtual+screening+for+novel+Staphylococcus+Aureus+NorA+efflux+pump+inhibitors+from+natural+products&doi=10.2174%2F1573406410666140902110903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Screening for Novel Staphylococcus Aureus NorA Efflux Pump Inhibitors From Natural Products</span></div><div class="casAuthors">Thai, Khac-Minh; Ngo, Trieu-Du; Phan, Thien-Vy; Tran, Thanh-Dao; Nguyen, Ngoc-Vinh; Nguyen, Thien-Hai; Le, Minh-Tri</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">135-155</span>CODEN:
                <span class="NLM_cas:coden">MCEHAJ</span>;
        ISSN:<span class="NLM_cas:issn">1573-4064</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">NorA is a member of the Major Facilitator Superfamily (MFS) drug efflux pumps that have been shown to mediate antibiotic resistance in Staphylococcus aureus (SA).  In this study, QSAR anal., virtual screening and mol. docking were implemented in an effort to discover novel SA NorA efflux pump inhibitors.  Originally, a set of 47 structurally diverse compds. compiled from the literature was used to develop linear QSAR models and another set of 15 different compds. were chosen for extra validation.  The final model which was estd. by statistical values for the full data set (n = 45, Q2 = 0.80, RMSE = 0.20) and for the external test set (n = 15, R2 = 0.60, |res|max = 0.75, |res|min = 0.02) was applied on the collection of 182 flavonoides and the traditional Chinese medicine (TCM) database to screen for novel NorA inhibitors.  Finally, 33 lead compds. that met the Lipinski's rules of five/three and had good predicted pIC50 values from in silico screening process were employed to analyze the binding ability by docking studies on NorA homol. model in place of its unavailable crystal structures at two active sites, the central channel and the Walker B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHpK-LeL0byrVg90H21EOLACvtfcHk0ljWqj3RLJL-tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXislemtbs%253D&md5=e86c28e478370ed34ade2576080e5e18</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.2174%2F1573406410666140902110903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1573406410666140902110903%26sid%3Dliteratum%253Aachs%26aulast%3DThai%26aufirst%3DK.%2BM.%26aulast%3DNgo%26aufirst%3DT.%2BD.%26aulast%3DPhan%26aufirst%3DT.%2BV.%26aulast%3DTran%26aufirst%3DT.%2BD.%26aulast%3DNguyen%26aufirst%3DN.%2BV.%26aulast%3DNguyen%26aufirst%3DT.%2BH.%26aulast%3DLe%26aufirst%3DM.%2BT.%26atitle%3DVirtual%2520screening%2520for%2520novel%2520Staphylococcus%2520Aureus%2520NorA%2520efflux%2520pump%2520inhibitors%2520from%2520natural%2520products%26jtitle%3DMed.%2520Chem.%26date%3D2015%26volume%3D11%26spage%3D135%26epage%3D155%26doi%3D10.2174%2F1573406410666140902110903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span><span class="NLM_contrib-group">Tran, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le, L.</span><span> </span><span class="NLM_article-title">Identification of novel compounds against an R294K substitution of influenza A (H7N9) virus using ensemble based drug virtual screening</span> <span class="citation_source-journal">Int. J. Med. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">163</span><span class="NLM_x">–</span> <span class="NLM_lpage">176</span><span class="refDoi"> DOI: 10.7150/ijms.10826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.7150%2Fijms.10826" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25589893" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BC2Mvms1ejsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=163-176&author=N.+Tranauthor=T.+Vanauthor=H.+Nguyenauthor=L.+Le&title=Identification+of+novel+compounds+against+an+R294K+substitution+of+influenza+A+%28H7N9%29+virus+using+ensemble+based+drug+virtual+screening&doi=10.7150%2Fijms.10826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel compounds against an R294K substitution of influenza A (H7N9) virus using ensemble based drug virtual screening</span></div><div class="casAuthors">Tran Nhut; Van Thanh; Le Ly; Nguyen Hieu</div><div class="citationInfo"><span class="NLM_cas:title">International journal of medical sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">163-76</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Influenza virus H7N9 foremost emerged in China in 2013 and killed hundreds of people in Asia since they possessed all mutations that enable them to resist to all existing influenza drugs, resulting in high mortality to human.  In the effort to identify novel inhibitors combat resistant strains of influenza virus H7N9; we performed virtual screening targeting the Neuraminidase (NA) protein against natural compounds of traditional Chinese medicine database (TCM) and ZINC natural products.  Compounds expressed high binding affinity to the target protein was then evaluated for molecular properties to determine drug-like molecules. 4 compounds showed their binding energy less than -11 Kcal/mol were selected for molecular dynamics (MD) simulation to capture intermolecular interactions of ligand-protein complexes.  The molecular mechanics/Poisson-Boltzmann surface area (MM/PBSA) method was utilized to estimate binding free energy of the complex.  In term of stability, NA-7181 (IUPAC namely {9-Hydroxy-10-[3-(trifluoromrthyl) cyclohexyl]-4.8-diazatricyclo [6.4.0.02,6]dodec-4-yl}(perhydro-1H-inden-5-yl)formaldehyde) achieved stable conformation after 20 ns and 27 ns for ligand and protein root mean square deviation, respectively.  In term of binding free energy, 7181 gave the negative value of -30.031 (KJ/mol) indicating the compound obtained a favourable state in the active site of the protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRbjj8ucg3x6q91pGlMMoT5fW6udTcc2eYMUkuv6oIb6Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mvms1ejsw%253D%253D&md5=ac04691ad0fdab05e75eab9a81f6c3ce</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.7150%2Fijms.10826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fijms.10826%26sid%3Dliteratum%253Aachs%26aulast%3DTran%26aufirst%3DN.%26aulast%3DVan%26aufirst%3DT.%26aulast%3DNguyen%26aufirst%3DH.%26aulast%3DLe%26aufirst%3DL.%26atitle%3DIdentification%2520of%2520novel%2520compounds%2520against%2520an%2520R294K%2520substitution%2520of%2520influenza%2520A%2520%2528H7N9%2529%2520virus%2520using%2520ensemble%2520based%2520drug%2520virtual%2520screening%26jtitle%3DInt.%2520J.%2520Med.%2520Sci.%26date%3D2015%26volume%3D12%26spage%3D163%26epage%3D176%26doi%3D10.7150%2Fijms.10826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref108"><div class="reference"><strong class="refLabel"><a href="#ref108" class="rightTabRefNumLink">108</a></strong><div class="NLM_citation" id="rightTab-cit108"><span><span class="NLM_contrib-group">Gupta, C. L.; Akhtar, S.; Kumar, N.; Ali, J.; Pathak, N.; Bajpai, P.</span><span> </span><span class="NLM_article-title">In silico elucidation and inhibition studies of selected phytoligands against mitogen-activated protein kinases of protozoan parasites</span>.  <span class="citation_source-journal">Interdiscip. Sci.: Comput. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span>DOI: <span class="refDoi"> DOI: 10.1007/s12539-015-0269-6</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs12539-015-0269-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=C.+L.+Gupta&author=S.+Akhtar&author=N.+Kumar&author=J.+Ali&author=N.+Pathak&author=P.+Bajpai&title=In+silico+elucidation+and+inhibition+studies+of+selected+phytoligands+against+mitogen-activated+protein+kinases+of+protozoan+parasites&doi=10.1007%2Fs12539-015-0269-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1007%2Fs12539-015-0269-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12539-015-0269-6%26sid%3Dliteratum%253Aachs%26aulast%3DGupta%26aufirst%3DC.%2BL.%26atitle%3DIn%2520silico%2520elucidation%2520and%2520inhibition%2520studies%2520of%2520selected%2520phytoligands%2520against%2520mitogen-activated%2520protein%2520kinases%2520of%2520protozoan%2520parasites%26jtitle%3DInterdiscip.%2520Sci.%253A%2520Comput.%2520Life%2520Sci.%26date%3D2015%26doi%3D10.1007%2Fs12539-015-0269-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref109"><div class="reference"><strong class="refLabel"><a href="#ref109" class="rightTabRefNumLink">109</a></strong><div class="NLM_citation" id="rightTab-cit109"><span><span class="NLM_contrib-group">Mitra, J.; Narad, P.; Sengupta, A.; Sharma, P. D.; Paul, P. K.</span> In <span> </span><span class="NLM_article-title">silico identification of ergosterol as a novel fungal metabolite enhancing RuBisCO activity in Lycopersicum esculentum</span>.  <span class="citation_source-journal">Interdiscip. Sci.: Comput. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span>DOI: <span class="refDoi"> DOI: 10.1007/s12539-015-0105-z</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs12539-015-0105-z" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=J.+Mitra&author=P.+Narad&author=A.+Sengupta&author=P.+D.+Sharma&author=P.+K.+Paul&title=silico+identification+of+ergosterol+as+a+novel+fungal+metabolite+enhancing+RuBisCO+activity+in+Lycopersicum+esculentum&doi=10.1007%2Fs12539-015-0105-z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1007%2Fs12539-015-0105-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12539-015-0105-z%26sid%3Dliteratum%253Aachs%26aulast%3DMitra%26aufirst%3DJ.%26atitle%3Dsilico%2520identification%2520of%2520ergosterol%2520as%2520a%2520novel%2520fungal%2520metabolite%2520enhancing%2520RuBisCO%2520activity%2520in%2520Lycopersicum%2520esculentum%26jtitle%3DInterdiscip.%2520Sci.%253A%2520Comput.%2520Life%2520Sci.%26date%3D2015%26doi%3D10.1007%2Fs12539-015-0105-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref110"><div class="reference"><strong class="refLabel"><a href="#ref110" class="rightTabRefNumLink">110</a></strong><div class="NLM_citation" id="rightTab-cit110"><span><span class="NLM_contrib-group">Sbongile, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soliman, M. E.</span><span> </span><span class="NLM_article-title">In silico identification of irreversible cathepsin B inhibitors as anti- cancer agents: virtual screening, covalent docking analysis and molecular dynamics simulations</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">399</span><span class="NLM_x">–</span> <span class="NLM_lpage">410</span><span class="refDoi"> DOI: 10.2174/1386207318666150305154621</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.2174%2F1386207318666150305154621" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25747438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVGmtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2015&pages=399-410&author=M.+Sbongileauthor=M.+E.+Soliman&title=In+silico+identification+of+irreversible+cathepsin+B+inhibitors+as+anti-+cancer+agents%3A+virtual+screening%2C+covalent+docking+analysis+and+molecular+dynamics+simulations&doi=10.2174%2F1386207318666150305154621"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">In Silico Identification of Irreversible Cathepsin B Inhibitors as Anti- Cancer Agents: Virtual Screening, Covalent Docking Analysis and Molecular Dynamics Simulations</span></div><div class="casAuthors">Sbongile, Mbatha; Soliman, Mahmoud E. S.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry & High Throughput Screening</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">399-410</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Cathepsin B is a cysteine protease that belongs to the papain superfamily.  Malfunctions related to cathepsin B can lead to inflammation and cancer.  Via an integrated in silico approach, this study is aimed to identify novel Michael acceptors-type compds. that can irreversibly inhibit cathepsin B enzyme via covalent bond formation with the active site cysteine residue.  Here, we report the first account of covalent docking approach incorporated into a hybrid ligand/structure-based virtual screening to est. the binding affinities of various compds. from chem. databases against the cathepsin B protein.  For validation, compds. with exptl. detd. anti-cathepsin B activity from PubChem bioassay database were also screened and covalently docked to the enzyme target.  Interestingly, four novel compds. exhibited better covalent binding affinity when compared against the exptl. detd. prototypes.  Mol. dynamics simulations were performed to ensure the stability of the docked complexes and to allow further anal. on the MD av. structures.  Perresidue interaction decompn. anal. was carried out to provide deeper insight into the interaction themes of discovered hits with the active site residues.  It is found that polar and hydrophobic interactions contributed the most towards drug binding.  The hybrid computational methods applied in this study should serve as a powerful tool in the drug design and development process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGZtQeMsmhn7Vg90H21EOLACvtfcHk0ljxz-rYp7Llrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVGmtrk%253D&md5=ac91293e5e902541f9f8b70eca39deab</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.2174%2F1386207318666150305154621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1386207318666150305154621%26sid%3Dliteratum%253Aachs%26aulast%3DSbongile%26aufirst%3DM.%26aulast%3DSoliman%26aufirst%3DM.%2BE.%26atitle%3DIn%2520silico%2520identification%2520of%2520irreversible%2520cathepsin%2520B%2520inhibitors%2520as%2520anti-%2520cancer%2520agents%253A%2520virtual%2520screening%252C%2520covalent%2520docking%2520analysis%2520and%2520molecular%2520dynamics%2520simulations%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2015%26volume%3D18%26spage%3D399%26epage%3D410%26doi%3D10.2174%2F1386207318666150305154621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref111"><div class="reference"><strong class="refLabel"><a href="#ref111" class="rightTabRefNumLink">111</a></strong><div class="NLM_citation" id="rightTab-cit111"><span><span class="NLM_contrib-group">Batool, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khan, Z. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamal, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mushtaq, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamal, M. A.</span><span> </span><span class="NLM_article-title">In silico screening for identification of novel anti-malarial inhibitors by molecular docking, pharmacophore modeling and virtual screening</span> <span class="citation_source-journal">Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">687</span><span class="NLM_x">–</span> <span class="NLM_lpage">700</span><span class="refDoi"> DOI: 10.2174/1573406411666150305113533</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.2174%2F1573406411666150305113533" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25741881" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2rsrjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=687-700&author=S.+Batoolauthor=Z.+A.+Khanauthor=W.+Kamalauthor=G.+Mushtaqauthor=M.+A.+Kamal&title=In+silico+screening+for+identification+of+novel+anti-malarial+inhibitors+by+molecular+docking%2C+pharmacophore+modeling+and+virtual+screening&doi=10.2174%2F1573406411666150305113533"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">In silico Screening for Identification of Novel Anti-malarial Inhibitors by Molecular Docking, Pharmacophore Modeling and Virtual Screening</span></div><div class="casAuthors">Batool, Sidra; Khan, Zeshan Aslam; Kamal, Warda; Mushtaq, Gohar; Kamal, Mohammad Amjad</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">687-700</span>CODEN:
                <span class="NLM_cas:coden">MCEHAJ</span>;
        ISSN:<span class="NLM_cas:issn">1573-4064</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Objective: Drug resistance from affordable drugs has increased the no. of deaths from malaria globally.  This problem has raised the requirement to design new drugs against multidrugresistant Plasmodium falciparum parasite.  Methods: In the current project, we have focused on four important proteins of Plasmodium falciparum and used them as receptors against a dataset of four anti-malarial drugs.  In silico anal. of these receptors and ligand dataset was carried out using Autodock 4.2.  A pharmacophore model was also established using Ligandscout.  Results: Anal. of docking expts. showed that all ligands bind efficiently to four proteins of Plasmodium falciparum.  We have used ligand-based pharmacophore modeling and developed a pharmacophore model that has three hydrophobic regions, two arom. rings, one hydrogen acceptor and one hydrogen donor.  Using this pharmacophore model, we have screened a library of 50,000 compds.  The compds. that shared features of our pharmacophore model and exhibited interactions with the four proteins of our receptors dataset are short-listed.  Conclusion: As there is a need of more anti-malarial drugs, therefore, this research will be helpful in identifying novel anti-malarial drugs that exhibited bindings with four important proteins of Plasmodium falciparum.  The hits obtained in this study can be considered as useful leads in anti-malarial drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7UeYnQfB4bLVg90H21EOLACvtfcHk0ljxz-rYp7Llrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2rsrjI&md5=01edcb27067a7529826bb7d2575f0a18</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.2174%2F1573406411666150305113533&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1573406411666150305113533%26sid%3Dliteratum%253Aachs%26aulast%3DBatool%26aufirst%3DS.%26aulast%3DKhan%26aufirst%3DZ.%2BA.%26aulast%3DKamal%26aufirst%3DW.%26aulast%3DMushtaq%26aufirst%3DG.%26aulast%3DKamal%26aufirst%3DM.%2BA.%26atitle%3DIn%2520silico%2520screening%2520for%2520identification%2520of%2520novel%2520anti-malarial%2520inhibitors%2520by%2520molecular%2520docking%252C%2520pharmacophore%2520modeling%2520and%2520virtual%2520screening%26jtitle%3DMed.%2520Chem.%26date%3D2015%26volume%3D11%26spage%3D687%26epage%3D700%26doi%3D10.2174%2F1573406411666150305113533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref112"><div class="reference"><strong class="refLabel"><a href="#ref112" class="rightTabRefNumLink">112</a></strong><div class="NLM_citation" id="rightTab-cit112"><span><span class="NLM_contrib-group">Kumar, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raj, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varadwaj, P. K.</span><span> </span><span class="NLM_article-title">In-silico identification of inhibitors against mutated BCR-ABL protein of chronic myeloid leukemia: a virtual screening and molecular dynamics simulation study</span> <span class="citation_source-journal">J. Biomol. Struct. Dyn.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">13</span><span class="refDoi"> DOI: 10.1080/07391102.2015.1110046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1080%2F07391102.2015.1110046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BC28voslWntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=1-13&author=H.+Kumarauthor=U.+Rajauthor=S.+Guptaauthor=P.+K.+Varadwaj&title=In-silico+identification+of+inhibitors+against+mutated+BCR-ABL+protein+of+chronic+myeloid+leukemia%3A+a+virtual+screening+and+molecular+dynamics+simulation+study&doi=10.1080%2F07391102.2015.1110046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Azadirachtin(A) Distinctively Modulates Subdomain 2 of Actin - Novel Mechanism to Induce Depolymerization revealed by Molecular Dynamics Study</span></div><div class="casAuthors">Kumar R Pravin; Kumar R Pravin; Roopa L; Mohammed M M Sudheer; Kulkarni Naveen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular structure & dynamics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-39</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Azadirachtin(A) (AZA) a potential insecticide from neem binds to actin and induces depolymerization in Drosophila.  AZA binds to the pocket same as that of Latrunculin A (LAT) but LAT inhibits actin polymerization by stiffening the actin structure and affects the ADP-ATP exchange.  The mechanism by which AZA induces actin depolymerization is not clearly understood.  Therefore, different computational experiments were conducted to delineate the precise mechanism of AZA induced actin depolymerization.  Molecular dynamics studies showed that AZA strongly interacted with subdomain 2 and destabilised the interactions between subdomain 2 of one actin and subdomain 1 & 4 of the adjacent actin, causing the separation of actin subunits.  The separation was observed between subdomain 3 of subunit n and subdomain 4 of subunit n+2.  However, the specific triggering point for the separation of the subunits was the destabilisation of direct interactions between subdomain 2 of subunit n (Arg39, Val45, Gly46 and Arg62) and subdomain 4 of subunit n+2 (Asp286, Ile287, Asp288, Ile289, Asp244 and Lys291).  These results reveal a unique mechanism of an actin filament modulator that induces depolymerization.  This mechanism of AZA can be used to design similar molecules against mammalian actins for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR9OyCfOzN4fXELvajU9pjPfW6udTcc2eYKLR4Jj_QnY7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28voslWntA%253D%253D&md5=49be334e5f4d7ad5bdfde38c2f89ff70</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1080%2F07391102.2015.1110046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F07391102.2015.1110046%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DH.%26aulast%3DRaj%26aufirst%3DU.%26aulast%3DGupta%26aufirst%3DS.%26aulast%3DVaradwaj%26aufirst%3DP.%2BK.%26atitle%3DIn-silico%2520identification%2520of%2520inhibitors%2520against%2520mutated%2520BCR-ABL%2520protein%2520of%2520chronic%2520myeloid%2520leukemia%253A%2520a%2520virtual%2520screening%2520and%2520molecular%2520dynamics%2520simulation%2520study%26jtitle%3DJ.%2520Biomol.%2520Struct.%2520Dyn.%26date%3D2015%26spage%3D1%26epage%3D13%26doi%3D10.1080%2F07391102.2015.1110046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref113"><div class="reference"><strong class="refLabel"><a href="#ref113" class="rightTabRefNumLink">113</a></strong><div class="NLM_citation" id="rightTab-cit113"><span><span class="NLM_contrib-group">Shafique, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bibi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rashid, S.</span><span> </span><span class="NLM_article-title">In silico identification of putative bifunctional Plk1 inhibitors by integrative virtual screening and structural dynamics approach</span> <span class="citation_source-journal">J. Theor. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">388</span><span class="NLM_x">, </span> <span class="NLM_fpage">72</span><span class="NLM_x">–</span> <span class="NLM_lpage">84</span><span class="refDoi"> DOI: 10.1016/j.jtbi.2015.10.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.jtbi.2015.10.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=388&publication_year=2016&pages=72-84&author=S.+Shafiqueauthor=N.+Bibiauthor=S.+Rashid&title=In+silico+identification+of+putative+bifunctional+Plk1+inhibitors+by+integrative+virtual+screening+and+structural+dynamics+approach&doi=10.1016%2Fj.jtbi.2015.10.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1016%2Fj.jtbi.2015.10.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jtbi.2015.10.006%26sid%3Dliteratum%253Aachs%26aulast%3DShafique%26aufirst%3DS.%26aulast%3DBibi%26aufirst%3DN.%26aulast%3DRashid%26aufirst%3DS.%26atitle%3DIn%2520silico%2520identification%2520of%2520putative%2520bifunctional%2520Plk1%2520inhibitors%2520by%2520integrative%2520virtual%2520screening%2520and%2520structural%2520dynamics%2520approach%26jtitle%3DJ.%2520Theor.%2520Biol.%26date%3D2016%26volume%3D388%26spage%3D72%26epage%3D84%26doi%3D10.1016%2Fj.jtbi.2015.10.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref114"><div class="reference"><strong class="refLabel"><a href="#ref114" class="rightTabRefNumLink">114</a></strong><div class="NLM_citation" id="rightTab-cit114"><span><span class="NLM_contrib-group">Pei, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.</span><span> </span><span class="NLM_article-title">Enrichment assessment of multiple virtual screening strategies for Toll-Like Receptor 8 agonists based on a maximal unbiased benchmarking data set</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">86</span><span class="NLM_x">, </span> <span class="NLM_fpage">1226</span><span class="NLM_x">–</span> <span class="NLM_lpage">1241</span><span class="refDoi"> DOI: 10.1111/cbdd.12590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1111%2Fcbdd.12590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26017460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFelsL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2015&pages=1226-1241&author=F.+Peiauthor=H.+Jinauthor=X.+Zhouauthor=J.+Xiaauthor=L.+Sunauthor=Z.+Liuauthor=L.+Zhang&title=Enrichment+assessment+of+multiple+virtual+screening+strategies+for+Toll-Like+Receptor+8+agonists+based+on+a+maximal+unbiased+benchmarking+data+set&doi=10.1111%2Fcbdd.12590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Enrichment Assessment of Multiple Virtual Screening Strategies for Toll-Like Receptor 8 Agonists Based on a Maximal Unbiased Benchmarking Data Set</span></div><div class="casAuthors">Pei, Fen; Jin, Hongwei; Zhou, Xin; Xia, Jie; Sun, Lidan; Liu, Zhenming; Zhang, Liangren</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1226-1241</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Toll-like receptor 8 agonists, which activate adaptive immune responses by inducing robust prodn. of T-helper 1-polarizing cytokines, are promising candidates for vaccine adjuvants.  As the binding site of toll-like receptor 8 is large and highly flexible, virtual screening by individual method has inevitable limitations; thus, a comprehensive comparison of different methods may provide insights into seeking effective strategy for the discovery of novel toll-like receptor 8 agonists.  In this study, the performance of knowledge-based pharmacophore, shape-based 3D screening, and combined strategies was assessed against a max. unbiased benchmarking data set contg. 13 actives and 1302 decoys specialized for toll-like receptor 8 agonists.  Prior structure-activity relationship knowledge was involved in knowledge-based pharmacophore generation, and a set of antagonists was innovatively used to verify the selectivity of the selected knowledge-based pharmacophore.  The benchmarking data set was generated from our recently developed 'MUBD-DECOYMAKER' protocol.  The enrichment assessment demonstrated a considerable performance through our selected three-layer virtual screening strategy: knowledge-based pharmacophore (Phar1) screening, shape-based 3D similarity search (Q4_combo), and then a Gold docking screening.  This virtual screening strategy could be further employed to perform large-scale database screening and to discover novel toll-like receptor 8 agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo80grcbCzo4bVg90H21EOLACvtfcHk0ljch0aHzscCgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFelsL%252FF&md5=15e493183f6992ba27d549b32ce45147</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12590%26sid%3Dliteratum%253Aachs%26aulast%3DPei%26aufirst%3DF.%26aulast%3DJin%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DXia%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.%26atitle%3DEnrichment%2520assessment%2520of%2520multiple%2520virtual%2520screening%2520strategies%2520for%2520Toll-Like%2520Receptor%25208%2520agonists%2520based%2520on%2520a%2520maximal%2520unbiased%2520benchmarking%2520data%2520set%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2015%26volume%3D86%26spage%3D1226%26epage%3D1241%26doi%3D10.1111%2Fcbdd.12590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref115"><div class="reference"><strong class="refLabel"><a href="#ref115" class="rightTabRefNumLink">115</a></strong><div class="NLM_citation" id="rightTab-cit115"><span><span class="NLM_contrib-group">Kumar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chauhan, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raghava, G. P.</span><span> </span><span class="NLM_article-title">In silico designing and screening of antagonists against cancer drug target XIAP</span> <span class="citation_source-journal">Curr. Cancer Drug Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">836</span><span class="NLM_x">–</span> <span class="NLM_lpage">846</span><span class="refDoi"> DOI: 10.2174/1568009615666150706103537</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.2174%2F1568009615666150706103537" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26143944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVGnsrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=836-846&author=R.+Kumarauthor=J.+S.+Chauhanauthor=G.+P.+Raghava&title=In+silico+designing+and+screening+of+antagonists+against+cancer+drug+target+XIAP&doi=10.2174%2F1568009615666150706103537"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">In Silico Designing and Screening of Antagonists against Cancer Drug Target XIAP</span></div><div class="casAuthors">Kumar, Rahul; Chauhan, Jagat Singh; Singh Raghava, Gajendra Pal</div><div class="citationInfo"><span class="NLM_cas:title">Current Cancer Drug Targets</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">836-846</span>CODEN:
                <span class="NLM_cas:coden">CCDTB9</span>;
        ISSN:<span class="NLM_cas:issn">1568-0096</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">X-linked inhibitor of apoptosis (XIAP) is a member of inhibitor of apoptosis (IAP) family and involved in the suppression of apoptosis in cancer cells.  This property makes it a therapeutic target for the cancer therapy.  In the present study, we have developed QSAR models using chem. descriptors, fingerprints, principal components, docking energy parameters and similarity-based approach against XIAP.  We have achieved correlation (R) of 0.803 with R2 value of 0.645 at 10-fold cross validation using SMOreg algorithm.  We have evaluated these models on independent dataset to ascertain its robustness and achieved correlation (R) of 0.793 with R2 value of 0.628.  Further, we have used these models for the screening of FDA approved drugs and drug-like mols. from ZINC database and prioritized them on the basis of their predicted pIC50 values.  Docking studies of top hits with XIAP-BIR3 domain shows that Iodixanol (DB01249) and ZINC68678304 have higher binding affinities than well-known tetrapeptide inhibitor, AVPI.  We have integrated these models in a web server named as "XIAPin".  We hope that this web server will contribute in the designing of nifty antagonists against XIAP.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrK-w4YQIpLBrVg90H21EOLACvtfcHk0ljch0aHzscCgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVGnsrnN&md5=c0e189903d7946e10667d07e347a6883</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.2174%2F1568009615666150706103537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568009615666150706103537%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DR.%26aulast%3DChauhan%26aufirst%3DJ.%2BS.%26aulast%3DRaghava%26aufirst%3DG.%2BP.%26atitle%3DIn%2520silico%2520designing%2520and%2520screening%2520of%2520antagonists%2520against%2520cancer%2520drug%2520target%2520XIAP%26jtitle%3DCurr.%2520Cancer%2520Drug%2520Targets%26date%3D2015%26volume%3D15%26spage%3D836%26epage%3D846%26doi%3D10.2174%2F1568009615666150706103537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref116"><div class="reference"><strong class="refLabel"><a href="#ref116" class="rightTabRefNumLink">116</a></strong><div class="NLM_citation" id="rightTab-cit116"><span><span class="NLM_contrib-group">Mathew, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fatima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fatmi, M. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Archunan, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ilyas, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Begum, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azhar, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damanhouri, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qadri, I.</span><span> </span><span class="NLM_article-title">Computational docking study of p7 ion channel from HCV genotype 3 and genotype 4 and its interaction with natural compounds</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0126510</span><span class="refDoi"> DOI: 10.1371/journal.pone.0126510</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pone.0126510" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0126510&author=S.+Mathewauthor=K.+Fatimaauthor=M.+Q.+Fatmiauthor=G.+Archunanauthor=M.+Ilyasauthor=N.+Begumauthor=E.+Azharauthor=G.+Damanhouriauthor=I.+Qadri&title=Computational+docking+study+of+p7+ion+channel+from+HCV+genotype+3+and+genotype+4+and+its+interaction+with+natural+compounds&doi=10.1371%2Fjournal.pone.0126510"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0126510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0126510%26sid%3Dliteratum%253Aachs%26aulast%3DMathew%26aufirst%3DS.%26aulast%3DFatima%26aufirst%3DK.%26aulast%3DFatmi%26aufirst%3DM.%2BQ.%26aulast%3DArchunan%26aufirst%3DG.%26aulast%3DIlyas%26aufirst%3DM.%26aulast%3DBegum%26aufirst%3DN.%26aulast%3DAzhar%26aufirst%3DE.%26aulast%3DDamanhouri%26aufirst%3DG.%26aulast%3DQadri%26aufirst%3DI.%26atitle%3DComputational%2520docking%2520study%2520of%2520p7%2520ion%2520channel%2520from%2520HCV%2520genotype%25203%2520and%2520genotype%25204%2520and%2520its%2520interaction%2520with%2520natural%2520compounds%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0126510%26doi%3D10.1371%2Fjournal.pone.0126510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref117"><div class="reference"><strong class="refLabel"><a href="#ref117" class="rightTabRefNumLink">117</a></strong><div class="NLM_citation" id="rightTab-cit117"><span><span class="NLM_contrib-group">Lee, T. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arcus, V. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lott, J. S.</span><span> </span><span class="NLM_article-title">Prediction of the substrate for nonribosomal peptide synthetase (NRPS) adenylation domains by virtual screening</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">2052</span><span class="NLM_x">–</span> <span class="NLM_lpage">2066</span><span class="refDoi"> DOI: 10.1002/prot.24922</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fprot.24922" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2015&pages=2052-2066&author=T.+V.+Leeauthor=R.+D.+Johnsonauthor=V.+L.+Arcusauthor=J.+S.+Lott&title=Prediction+of+the+substrate+for+nonribosomal+peptide+synthetase+%28NRPS%29+adenylation+domains+by+virtual+screening&doi=10.1002%2Fprot.24922"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1002%2Fprot.24922&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.24922%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DT.%2BV.%26aulast%3DJohnson%26aufirst%3DR.%2BD.%26aulast%3DArcus%26aufirst%3DV.%2BL.%26aulast%3DLott%26aufirst%3DJ.%2BS.%26atitle%3DPrediction%2520of%2520the%2520substrate%2520for%2520nonribosomal%2520peptide%2520synthetase%2520%2528NRPS%2529%2520adenylation%2520domains%2520by%2520virtual%2520screening%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2015%26volume%3D83%26spage%3D2052%26epage%3D2066%26doi%3D10.1002%2Fprot.24922" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref118"><div class="reference"><strong class="refLabel"><a href="#ref118" class="rightTabRefNumLink">118</a></strong><div class="NLM_citation" id="rightTab-cit118"><span><span class="NLM_contrib-group">Xiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span> </span><span class="NLM_article-title">Topomer CoMFA and virtual screening studies of azaindole class renin inhibitors</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">458</span><span class="NLM_x">–</span> <span class="NLM_lpage">472</span><span class="refDoi"> DOI: 10.2174/1386207317666140107094708</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.2174%2F1386207317666140107094708" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24392830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVOnsLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2014&pages=458-472&author=Y.+Xiangauthor=J.+Songauthor=Z.+Zhang&title=Topomer+CoMFA+and+virtual+screening+studies+of+azaindole+class+renin+inhibitors&doi=10.2174%2F1386207317666140107094708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Topomer CoMFA and Virtual Screening Studies of Azaindole Class Renin Inhibitors</span></div><div class="casAuthors">Xiang, Yuhong; Song, Jia; Zhang, Zhuoyong</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry & High Throughput Screening</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">458-472</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Direct renin inhibitors (DRIs) have increasingly shown a significant advantage in the treatment of hypertension and protection of target organs.  In this paper, a series of azaindole class renin inhibitors were subjected to 3D-QSAR study using Topomer CoMFA.  Five kinds of splitting mode for different fragment cutting and 6 different training and test sets grouping were attempted to build consensus models.  The results indicated that 6 consensus models had similar predictability (q2 and r2pred) and stability (r2).  The best model showed good stability and predictability (q2 of 0.616, r2 of 0.908).  The r2pred value of the external test set was 0.70, which means that the model had an excellent external predictive ability, and the robustness of the developed model was assessed by the Y-randomization test.  This study also adopted the methodol. of fragment-based drug design (FBDD) to virtual screen new renin inhibitors by using Topomer Search technol.  The R1-group of the compd. No. 13 with the highest activity was chosen as the basic scaffold, and its remaining R2-group acted as a query to screen 142,025 mols. of ZINC database for similar fragments.  The obtained 30 fragments with the highest R2-group contribution values were added to the basic scaffold resp.  Finally 30 new azaindole compds. with potent high activities were obtained.  Further the binding modes were studied by using Surflex- Dock.  The docking results showed good binding interactions of the designed compds. with the renin protein, thus the rationality of this design was further verified from the perspective of the renin receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgmCzCRZwX_bVg90H21EOLACvtfcHk0lg-8SmHnVQ3Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVOnsLrK&md5=eb17c6c0bfece3dcc70dad6f1bc36bfc</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.2174%2F1386207317666140107094708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1386207317666140107094708%26sid%3Dliteratum%253Aachs%26aulast%3DXiang%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DZ.%26atitle%3DTopomer%2520CoMFA%2520and%2520virtual%2520screening%2520studies%2520of%2520azaindole%2520class%2520renin%2520inhibitors%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2014%26volume%3D17%26spage%3D458%26epage%3D472%26doi%3D10.2174%2F1386207317666140107094708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref119"><div class="reference"><strong class="refLabel"><a href="#ref119" class="rightTabRefNumLink">119</a></strong><div class="NLM_citation" id="rightTab-cit119"><span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blagg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buser-Doepner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravatt, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frederiksen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimshaw, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hepworth, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howe, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotz, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mader, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller-Fahrnow, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Hagan, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundstrom, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taunton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toledo-Sherman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walpole, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walters, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willson, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuercher, W. J.</span><span> </span><span class="NLM_article-title">The promise and peril of chemical probes</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">536</span><span class="NLM_x">–</span> <span class="NLM_lpage">541</span><span class="refDoi"> DOI: 10.1038/nchembio.1867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnchembio.1867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26196764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=536-541&author=C.+H.+Arrowsmithauthor=J.+E.+Audiaauthor=C.+Austinauthor=J.+Baellauthor=J.+Bennettauthor=J.+Blaggauthor=C.+Bountraauthor=P.+E.+Brennanauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=C.+Buser-Doepnerauthor=R.+M.+Campbellauthor=A.+J.+Carterauthor=P.+Cohenauthor=R.+A.+Copelandauthor=B.+Cravattauthor=J.+L.+Dahlinauthor=D.+Dhanakauthor=A.+M.+Edwardsauthor=M.+Frederiksenauthor=S.+V.+Fryeauthor=N.+Grayauthor=C.+E.+Grimshawauthor=D.+Hepworthauthor=T.+Howeauthor=K.+V.+Huberauthor=J.+Jinauthor=S.+Knappauthor=J.+D.+Kotzauthor=R.+G.+Krugerauthor=D.+Loweauthor=M.+M.+Maderauthor=B.+Marsdenauthor=A.+Mueller-Fahrnowauthor=S.+Mullerauthor=R.+C.+O%E2%80%99Haganauthor=J.+P.+Overingtonauthor=D.+R.+Owenauthor=S.+H.+Rosenbergauthor=B.+Rothauthor=R.+Rossauthor=M.+Schapiraauthor=S.+L.+Schreiberauthor=B.+Shoichetauthor=M.+Sundstromauthor=G.+Superti-Furgaauthor=J.+Tauntonauthor=L.+Toledo-Shermanauthor=C.+Walpoleauthor=M.+A.+Waltersauthor=T.+M.+Willsonauthor=P.+Workmanauthor=R.+N.+Youngauthor=W.+J.+Zuercher&title=The+promise+and+peril+of+chemical+probes&doi=10.1038%2Fnchembio.1867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">The promise and peril of chemical probes</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Audia, James E.; Austin, Christopher; Baell, Jonathan; Bennett, Jonathan; Blagg, Julian; Bountra, Chas; Brennan, Paul E.; Brown, Peter J.; Bunnage, Mark E.; Buser-Doepner, Carolyn; Campbell, Robert M.; Carter, Adrian J.; Cohen, Philip; Copeland, Robert A.; Cravatt, Ben; Dahlin, Jayme L.; Dhanak, Dashyant; Edwards, Aled M.; Frye, Stephen V.; Gray, Nathanael; Grimshaw, Charles E.; Hepworth, David; Howe, Trevor; Huber, Kilian V. M.; Jin, Jian; Knapp, Stefan; Kotz, Joanne D.; Kruger, Ryan G.; Lowe, Derek; Mader, Mary M.; Marsden, Brian; Mueller-Fahrnow, Anke; Muller, Susanne; O'Hagan, Ronan C.; Overington, John P.; Owen, Dafydd R.; Rosenberg, Saul H.; Roth, Brian; Ross, Ruth; Schapira, Matthieu; Schreiber, Stuart L.; Shoichet, Brian; Sundstrom, Michael; Superti-Furga, Giulio; Taunton, Jack; Toledo-Sherman, Leticia; Walpole, Chris; Walters, Michael A.; Willson, Timothy M.; Workman, Paul; Young, Robert N.; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">536-541</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes are powerful reagents with increasing impacts on biomedical research.  However, probes of poor quality or that are used incorrectly generate misleading results.  To help address these shortcomings, we will create a community-driven wiki resource to improve quality and convey current best practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-li4dxq487rVg90H21EOLACvtfcHk0lg-8SmHnVQ3Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO&md5=d8c7a7af94921ab4cdcc871487cceadd</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1867%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DCarter%26aufirst%3DA.%2BJ.%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DHuber%26aufirst%3DK.%2BV.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKotz%26aufirst%3DJ.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DLowe%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Hagan%26aufirst%3DR.%2BC.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DRoth%26aufirst%3DB.%26aulast%3DRoss%26aufirst%3DR.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DSundstrom%26aufirst%3DM.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DToledo-Sherman%26aufirst%3DL.%26aulast%3DWalpole%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DR.%2BN.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26atitle%3DThe%2520promise%2520and%2520peril%2520of%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D536%26epage%3D541%26doi%3D10.1038%2Fnchembio.1867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref120"><div class="reference"><strong class="refLabel"><a href="#ref120" class="rightTabRefNumLink">120</a></strong><div class="NLM_citation" id="rightTab-cit120"><span><span class="NLM_contrib-group">Baell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, G. A.</span><span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2719</span><span class="NLM_x">–</span> <span class="NLM_lpage">2740</span><span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0li5jDZ8NmtxOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref121"><div class="reference"><strong class="refLabel"><a href="#ref121" class="rightTabRefNumLink">121</a></strong><div class="NLM_citation" id="rightTab-cit121"><span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Screening in a spirit haunted world</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">607</span><span class="NLM_x">–</span> <span class="NLM_lpage">615</span><span class="refDoi"> DOI: 10.1016/j.drudis.2006.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.drudis.2006.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=16793529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=607-615&author=B.+K.+Shoichet&title=Screening+in+a+spirit+haunted+world&doi=10.1016%2Fj.drudis.2006.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit121R"><div class="casContent"><span class="casTitleNuber">121</span><div class="casTitle"><span class="NLM_cas:atitle">Screening in a spirit haunted world</span></div><div class="casAuthors">Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">13 & 14</span>),
    <span class="NLM_cas:pages">607-615</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  High-throughput screening (HTS) campaigns can be dominated by hits that ultimately turn out to be non-drug-like.  These 'nuisance' compds. often behave strangely, with steep dose-response curves, absence of clear structure-activity relationships, and high sensitivity to assay conditions.  Several mechanisms contribute to these artifacts, including chem. reactive mols., those that absorb light in assays and those that affect redox conditions.  One of the most common mechanisms behind artifactual inhibition is discussed in this review: at micromolar concns. org. mols. can aggregate to form particles in aq. buffers, and these aggregates can sequester and thereby inhibit protein targets.  Aggregation-based inhibition is baffling from a chem. perspective, but viewed biophys. such behavior is expected.  The range of mols. that behave this way, their rapid detection in a screening environment and their possible biol. implications will be considered here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzoEkwI2izy7Vg90H21EOLACvtfcHk0li5jDZ8NmtxOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLk%253D&md5=08d22b6b06215ac435824b7664b3674c</span></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DScreening%2520in%2520a%2520spirit%2520haunted%2520world%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26spage%3D607%26epage%3D615%26doi%3D10.1016%2Fj.drudis.2006.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref122"><div class="reference"><strong class="refLabel"><a href="#ref122" class="rightTabRefNumLink">122</a></strong><div class="NLM_citation" id="rightTab-cit122"><span><span class="NLM_contrib-group">Walters, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Namchuk, M.</span><span> </span><span class="NLM_article-title">Designing screens: how to make your hits a hit</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">259</span><span class="NLM_x">–</span> <span class="NLM_lpage">266</span><span class="refDoi"> DOI: 10.1038/nrd1063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnrd1063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=12669025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3sXisFWktrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=259-266&author=W.+P.+Waltersauthor=M.+Namchuk&title=Designing+screens%3A+how+to+make+your+hits+a+hit&doi=10.1038%2Fnrd1063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">A guide to drug discovery: Designing screens: how to make your hits a hit</span></div><div class="casAuthors">Walters, W. Patrick; Namchuk, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">259-266</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The basic goal of small-mol. screening is the identification of chem. interesting starting points for elaboration towards a drug.  A no. of innovative approaches for pursuing this goal have evolved, and the right approach is dictated by the target class being pursued and the capabilities of the organization involved.hood.  A recent trend in high-throughput screening has been to place less emphasis on the no. of data points that can be produced, and to focus instead on the quality of the data obtained.  Several computational and technol. advances have aided in the selection of compds. for screening and widened the variety of assay formats available for screening.  The effect on the efficiency of the screening process is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ5E12ik7BrbVg90H21EOLACvtfcHk0li5jDZ8NmtxOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXisFWktrk%253D&md5=6bec4547b8dce69ab84ce226cde28f8a</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1038%2Fnrd1063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1063%26sid%3Dliteratum%253Aachs%26aulast%3DWalters%26aufirst%3DW.%2BP.%26aulast%3DNamchuk%26aufirst%3DM.%26atitle%3DDesigning%2520screens%253A%2520how%2520to%2520make%2520your%2520hits%2520a%2520hit%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D259%26epage%3D266%26doi%3D10.1038%2Fnrd1063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref123"><div class="reference"><strong class="refLabel"><a href="#ref123" class="rightTabRefNumLink">123</a></strong><div class="NLM_citation" id="rightTab-cit123"><span><span class="NLM_contrib-group">Rishton, G. M.</span><span> </span><span class="NLM_article-title">Nonleadlikeness and leadlikeness in biochemical screening</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">86</span><span class="NLM_x">–</span> <span class="NLM_lpage">96</span><span class="refDoi"> DOI: 10.1016/S1359644602025722</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2FS1359644602025722" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=12565011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BD3s%252Fls1SjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=86-96&author=G.+M.+Rishton&title=Nonleadlikeness+and+leadlikeness+in+biochemical+screening&doi=10.1016%2FS1359644602025722"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">Nonleadlikeness and leadlikeness in biochemical screening</span></div><div class="casAuthors">Rishton Gilbert M</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">86-96</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">Biochemical assays have largely supplanted functional biological assays as drug screening tools in the early stages of drug discovery.  The de-selection of compounds that are 'nonleadlike' binders (and bonders) and the proactive selection of those compounds that are 'leadlike' in their binding to the target are vital components of the screening effort.  The physiochemical properties of leadlikeness and the surprising differences between those properties and the now classical definitions of druglikeness are becoming apparent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSFoWSt5LTTmsrRqEXsZufAfW6udTcc2eaClCb5Bb3pQrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3s%252Fls1SjtQ%253D%253D&md5=5e34cece6f751fa727f949c3526aa95c</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1016%2FS1359644602025722&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359644602025722%26sid%3Dliteratum%253Aachs%26aulast%3DRishton%26aufirst%3DG.%2BM.%26atitle%3DNonleadlikeness%2520and%2520leadlikeness%2520in%2520biochemical%2520screening%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2003%26volume%3D8%26spage%3D86%26epage%3D96%26doi%3D10.1016%2FS1359644602025722" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref124"><div class="reference"><strong class="refLabel"><a href="#ref124" class="rightTabRefNumLink">124</a></strong><div class="NLM_citation" id="rightTab-cit124"><span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Southall, N. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinn, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auld, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span> </span><span class="NLM_article-title">Fluorescence spectroscopic profiling of compound libraries</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2363</span><span class="NLM_x">–</span> <span class="NLM_lpage">2371</span><span class="refDoi"> DOI: 10.1021/jm701301m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701301m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjslSns7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2363-2371&author=A.+Simeonovauthor=A.+Jadhavauthor=C.+J.+Thomasauthor=Y.+Wangauthor=R.+Huangauthor=N.+T.+Southallauthor=P.+Shinnauthor=J.+Smithauthor=C.+P.+Austinauthor=D.+S.+Auldauthor=J.+Inglese&title=Fluorescence+spectroscopic+profiling+of+compound+libraries&doi=10.1021%2Fjm701301m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit124R"><div class="casContent"><span class="casTitleNuber">124</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence Spectroscopic Profiling of Compound Libraries</span></div><div class="casAuthors">Simeonov, Anton; Jadhav, Ajit; Thomas, Craig J.; Wang, Yuhong; Huang, Ruili; Southall, Noel T.; Shinn, Paul; Smith, Jeremy; Austin, Christopher P.; Auld, Douglas S.; Inglese, James</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2363-2371</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chromo/fluorophoric properties often accompany the heterocyclic scaffolds and impurities that comprise libraries used for high-throughput screening (HTS).  These properties affect assay outputs obtained with optical detection, thus complicating anal. and leading to false positives and negatives.  Here, we report the fluorescence profile of more than 70 000 samples across spectral regions commonly utilized in HTS.  The quant. HTS paradigm was utilized to test each sample at seven or more concns. over a 4-log range in 1536-well format.  Raw fluorescence was compared with fluorophore stds. to compute a normalized response as a function of concn. and spectral region.  More than 5% of library members were brighter than the equiv. of 10 nM 4-Me umbelliferone, a common UV-active probe.  Red-shifting the spectral window by as little as 100 nm was accompanied by a dramatic decrease in autofluorescence.  Native compd. fluorescence, fluorescent impurities, novel fluorescent compds., and the utilization of fluorescence profiling data are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreZBF1CG9forVg90H21EOLACvtfcHk0ljoaLmGPD5-dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjslSns7Y%253D&md5=4a7b8ee758fce4be3586aa5b573a3fc1</span></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1021%2Fjm701301m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701301m%26sid%3Dliteratum%253Aachs%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DSouthall%26aufirst%3DN.%2BT.%26aulast%3DShinn%26aufirst%3DP.%26aulast%3DSmith%26aufirst%3DJ.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DInglese%26aufirst%3DJ.%26atitle%3DFluorescence%2520spectroscopic%2520profiling%2520of%2520compound%2520libraries%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2363%26epage%3D2371%26doi%3D10.1021%2Fjm701301m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref125"><div class="reference"><strong class="refLabel"><a href="#ref125" class="rightTabRefNumLink">125</a></strong><div class="NLM_citation" id="rightTab-cit125"><span><span class="NLM_contrib-group">Lea, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span> </span><span class="NLM_article-title">Fluorescence polarization assays in small molecule screening</span> <span class="citation_source-journal">Expert Opin. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">17</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1517/17460441.2011.537322</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1517%2F17460441.2011.537322" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=22328899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SrtbvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=17-32&author=W.+A.+Leaauthor=A.+Simeonov&title=Fluorescence+polarization+assays+in+small+molecule+screening&doi=10.1517%2F17460441.2011.537322"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence polarization assays in small molecule screening</span></div><div class="casAuthors">Lea, Wendy A.; Simeonov, Anton</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17-32</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Importance of the field: Fluorescence polarization (FP) is a homogeneous method that allows rapid and quant. anal. of diverse mol. interactions and enzyme activities.  This technique has been widely utilized in clin. and biomedical settings, including the diagnosis of certain diseases and monitoring therapeutic drug levels in body fluids.  Recent developments in the field have been symbolized by the facile adoption of FP in high-throughput screening and small mol. drug discovery of an increasing range of target classes.  Areas covered in this review: The article provides a brief overview of the theor. foundation of FP, followed by updates on recent advancements in its application for various drug target classes, including GPCRs, enzymes and protein-protein interactions.  The strengths and weaknesses of this method, practical considerations in assay design, novel applications and future directions are also discussed.  What the reader will gain: The reader is informed of the most recent advancements and future directions of FP application to small mol. screening.  Take home message: In addn. to its continued utilization in high-throughput screening, FP has expanded into new disease and target areas and has been marked by increased use of labeled small mol. ligands for receptor-binding studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqudLso-a4wR7Vg90H21EOLACvtfcHk0ljoaLmGPD5-dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SrtbvP&md5=6f102f40b56fd7a13c4fb59b64a3f883</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1517%2F17460441.2011.537322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2011.537322%26sid%3Dliteratum%253Aachs%26aulast%3DLea%26aufirst%3DW.%2BA.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DFluorescence%2520polarization%2520assays%2520in%2520small%2520molecule%2520screening%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2011%26volume%3D6%26spage%3D17%26epage%3D32%26doi%3D10.1517%2F17460441.2011.537322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref126"><div class="reference"><strong class="refLabel"><a href="#ref126" class="rightTabRefNumLink">126</a></strong><div class="NLM_citation" id="rightTab-cit126"><span><span class="NLM_contrib-group">Thorne, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auld, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span> </span><span class="NLM_article-title">Apparent activity in high-throughput screening: origins of compound-dependent assay interference</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">315</span><span class="NLM_x">–</span> <span class="NLM_lpage">324</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2010.03.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.cbpa.2010.03.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=20417149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=315-324&author=N.+Thorneauthor=D.+S.+Auldauthor=J.+Inglese&title=Apparent+activity+in+high-throughput+screening%3A+origins+of+compound-dependent+assay+interference&doi=10.1016%2Fj.cbpa.2010.03.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">Apparent activity in high-throughput screening: origins of compound-dependent assay interference</span></div><div class="casAuthors">Thorne, Natasha; Auld, Douglas S.; Inglese, James</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">315-324</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Expansive compd. collections made up of structurally heterogeneous chems., the activities of which are largely undefined, present challenging problems for high-throughput screening (HTS).  Foremost is differentiating whether the activity for a given compd. in an assay is directed against the targeted biol., or is the result of surreptitious compd. activity involving the assay detection system.  Such compd. interference can be esp. difficult to identify if it is reproducible and concn.-dependent - characteristics generally attributed to compds. with genuine activity.  While reactive chem. groups on compds. were once thought to be the primary source of compd. interference in assays used in HTS, recent work suggests that other factors, such as compd. aggregation, may play a more significant role in many assay formats.  Considerable progress has been made to profile representative compd. libraries in an effort to identify chem. classes susceptible to producing compd. interference, such as compds. commonly found to inhibit the reporter enzyme firefly luciferase.  Such work has also led to the development of practices that have the potential to significantly reduce compd. interference, for example, through the addn. of non-ionic detergent to assay buffer to reduce aggregation-based inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBawORe7gR97Vg90H21EOLACvtfcHk0liLdFrFPow7Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsL8%253D&md5=4d3dd71f66ce937002874106edaff12e</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.03.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.03.020%26sid%3Dliteratum%253Aachs%26aulast%3DThorne%26aufirst%3DN.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DInglese%26aufirst%3DJ.%26atitle%3DApparent%2520activity%2520in%2520high-throughput%2520screening%253A%2520origins%2520of%2520compound-dependent%2520assay%2520interference%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26spage%3D315%26epage%3D324%26doi%3D10.1016%2Fj.cbpa.2010.03.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref127"><div class="reference"><strong class="refLabel"><a href="#ref127" class="rightTabRefNumLink">127</a></strong><div class="NLM_citation" id="rightTab-cit127"><span><span class="NLM_contrib-group">McGovern, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caselli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grigorieff, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">A common mechanism underlying promiscuous inhibitors from virtual and high-throughput screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">1712</span><span class="NLM_x">–</span> <span class="NLM_lpage">1722</span><span class="refDoi"> DOI: 10.1021/jm010533y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm010533y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD38Xhslegtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=1712-1722&author=S.+L.+McGovernauthor=E.+Caselliauthor=N.+Grigorieffauthor=B.+K.+Shoichet&title=A+common+mechanism+underlying+promiscuous+inhibitors+from+virtual+and+high-throughput+screening&doi=10.1021%2Fjm010533y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">A Common Mechanism Underlying Promiscuous Inhibitors from Virtual and High-Throughput Screening</span></div><div class="casAuthors">McGovern, Susan L.; Caselli, Emilia; Grigorieff, Nikolaus; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1712-1722</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput and virtual screening are widely used to discover novel leads for drug design.  On examn., many screening hits appear non-drug-like: they act noncompetitively, show little relation between structure and activity, and have poor selectivity.  Attempts to develop these peculiar mols. into viable leads are often futile, and much time can be wasted on the characterization of these "phony" hits.  Despite their common occurrence, the mechanism of action of these promiscuous mols. remains unknown.  To investigate this problem, 45 diverse screening hits were studied.  Fifteen of these were previously reported as inhibitors of various receptors, including β-lactamase, malarial protease, dihydrofolate reductase, HIV Tar RNA, thymidylate synthase, kinesin, insulin receptor, tyrosine kinases, farnesyltransferase, gyrase, prions, triosephosphate isomerase, nitric oxide synthase, phosphoinositide 3-kinase, and integrase; 30 were from an inhouse screening library of a major pharmaceutical company.  In addn. to their original targets, 35 of these 45 compds. were shown to inhibit several unrelated model enzymes.  These 35 screening hits included compds., such as fullerenes, dyes, and quercetin, that have repeatedly shown activity against diverse targets.  When tested against the model enzymes, the compds. showed time-dependent but reversible inhibition that was dramatically attenuated by albumin, guanidinium, or urea. Surprisingly, increasing the concn. of the model enzymes 10-fold largely eliminated inhibition, despite a 1000-fold excess of inhibitor; a well-behaved competitive inhibitor did not show this behavior.  One model to explain these observations was that the active form of the promiscuous inhibitors was an aggregate of many individual mols.  To test this hypothesis, light scattering and electron microscopy expts. were performed.  The nonspecific inhibitors were obsd. to form particles of 30-400 nm diam. by both techniques.  In control expts., a well-behaved competitive inhibitor and an inactive dye-like mol. were not obsd. to form aggregates.  Consistent with the hypothesis that the aggregates are the inhibitory species, the particle size and IC50 values of the promiscuous inhibitors varied monotonically with ionic strength; a competitive inhibitor was unaffected by changes in ionic strength. Unexpectedly, aggregate formation appears to explain the activity of many nonspecific inhibitors and may account for the activity of many promiscuous screening hits.  Mols. acting via this mechanism may be widespread in drug discovery screening databases.  Recognition of these compds. may improve screening results in many areas of pharmaceutical interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYFB0NNK2CIbVg90H21EOLACvtfcHk0liLdFrFPow7Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xhslegtrk%253D&md5=b9e6c29f6a1da588a6452859061edfbb</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1021%2Fjm010533y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm010533y%26sid%3Dliteratum%253Aachs%26aulast%3DMcGovern%26aufirst%3DS.%2BL.%26aulast%3DCaselli%26aufirst%3DE.%26aulast%3DGrigorieff%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DA%2520common%2520mechanism%2520underlying%2520promiscuous%2520inhibitors%2520from%2520virtual%2520and%2520high-throughput%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D1712%26epage%3D1722%26doi%3D10.1021%2Fjm010533y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref128"><div class="reference"><strong class="refLabel"><a href="#ref128" class="rightTabRefNumLink">128</a></strong><div class="NLM_citation" id="rightTab-cit128"><span><span class="NLM_contrib-group">Feng, B. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelat, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doman, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guy, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">High-throughput assays for promiscuous inhibitors</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">–</span> <span class="NLM_lpage">148</span><span class="refDoi"> DOI: 10.1038/nchembio718</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnchembio718" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=16408018" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmt1yhu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2005&pages=146-148&author=B.+Y.+Fengauthor=A.+Shelatauthor=T.+N.+Domanauthor=R.+K.+Guyauthor=B.+K.+Shoichet&title=High-throughput+assays+for+promiscuous+inhibitors&doi=10.1038%2Fnchembio718"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput assays for promiscuous inhibitors</span></div><div class="casAuthors">Feng, Brian Y.; Shelat, Anang; Doman, Thompson N.; Guy, R. Kip; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">146-148</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) searches large libraries of chem. compds. for those that can modulate the activity of a particular biol. target; it is the dominant technique used in early-stage drug discovery.  A key problem in HTS is the prevalence of nonspecific or 'promiscuous' inhibitors.  These mols. have peculiar properties, act on unrelated targets and can dominate the results from screening campaigns.  Several explanations have been proposed to account for promiscuous inhibitors, including chem. reactivity, interference in assay read-out, high mol. flexibility and hydrophobicity.  The diversity of these models reflects the apparently unrelated mols. whose behaviors they seek to explain.  However, a single mechanism may explain the effects of many promiscuous inhibitors: some org. mols. form large colloid-like aggregates that sequester and thereby inhibit enzymes.  Hits from HTS, leads for drug discovery and even several drugs appear to act through this mechanism at micromolar concns.  Here, we report two rapid assays for detecting promiscuous aggregates that we tested against 1,030 'drug-like' mols.  The results from these assays were used to test two preliminary computational models of this phenomenon and as benchmarks to develop new models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCy0Rj4jKNOLVg90H21EOLACvtfcHk0liLdFrFPow7Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmt1yhu7k%253D&md5=3414328ac1d56eb2cce7c59f18a784d0</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1038%2Fnchembio718&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio718%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DB.%2BY.%26aulast%3DShelat%26aufirst%3DA.%26aulast%3DDoman%26aufirst%3DT.%2BN.%26aulast%3DGuy%26aufirst%3DR.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DHigh-throughput%2520assays%2520for%2520promiscuous%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2005%26volume%3D1%26spage%3D146%26epage%3D148%26doi%3D10.1038%2Fnchembio718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref129"><div class="reference"><strong class="refLabel"><a href="#ref129" class="rightTabRefNumLink">129</a></strong><div class="NLM_citation" id="rightTab-cit129"><span><span class="NLM_contrib-group">Coan, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Stoichiometry and physical chemistry of promiscuous aggregate-based inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">9606</span><span class="NLM_x">–</span> <span class="NLM_lpage">9612</span><span class="refDoi"> DOI: 10.1021/ja802977h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja802977h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvF2gsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=9606-9612&author=K.+E.+Coanauthor=B.+K.+Shoichet&title=Stoichiometry+and+physical+chemistry+of+promiscuous+aggregate-based+inhibitors&doi=10.1021%2Fja802977h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit129R"><div class="casContent"><span class="casTitleNuber">129</span><div class="casTitle"><span class="NLM_cas:atitle">Stoichiometry and Physical Chemistry of Promiscuous Aggregate-Based Inhibitors</span></div><div class="casAuthors">Coan, Kristin E. D.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">9606-9612</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many false positives in early drug discovery owe to nonspecific inhibition by colloid-like aggregates of org. mols.  Despite their prevalence, little is known about aggregate concn., structure, or dynamic equil.; the binding mechanism, stoichiometry with, and affinity for enzymes remain uncertain.  To investigate the elementary question of concn., we counted aggregate particles using flow cytometry.  For seven aggregate-forming mols., aggregates were not obsd. until the concn. of monomer crossed a threshold, indicating a "crit. aggregation concn." (CAC).  Above the CAC, aggregate count increased linearly with added org. material, while the particles dispersed when dild. below the CAC.  The concn. of monomeric org. mol. is const. above the CAC, as is the size of the aggregate particles.  For two compds. that form large aggregates, nicardipine and miconazole, we measured particle nos. directly by flow cytometry, detg. that the aggregate concn. just above the CAC ranged from 5 to 30 fM.  By correlating inhibition of an enzyme with aggregate count for these two drugs, we detd. that the stoichiometry of binding is about 10 000 enzyme mols. per aggregate particle.  Using measured vols. for nicardipine and miconazole aggregate particles (2.1 × 1011 and 4.7 × 1010 Å3, resp.), computed monomer vols., and the observation that past the CAC all addnl. monomer forms aggregate particles, we find that aggregates are densely packed particles.  Finally, given their size and enzyme stoichiometry, all sequestered enzyme can be comfortably accommodated on the surface of the aggregate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXzKuuN6PwebVg90H21EOLACvtfcHk0lhNZonEucmTjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvF2gsL8%253D&md5=6bb7b0d190db00bd872a2ba86a5d56ef</span></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1021%2Fja802977h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja802977h%26sid%3Dliteratum%253Aachs%26aulast%3DCoan%26aufirst%3DK.%2BE.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStoichiometry%2520and%2520physical%2520chemistry%2520of%2520promiscuous%2520aggregate-based%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D9606%26epage%3D9612%26doi%3D10.1021%2Fja802977h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref130"><div class="reference"><strong class="refLabel"><a href="#ref130" class="rightTabRefNumLink">130</a></strong><div class="NLM_citation" id="rightTab-cit130"><span><span class="NLM_contrib-group">Park, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ku, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, B. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. J.</span><span> </span><span class="NLM_article-title">Identification of novel PTPRQ phosphatase inhibitors based on the virtual screening with docking simulations</span> <span class="citation_source-journal">Theor. Biol. Med. Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">49</span><span class="refDoi"> DOI: 10.1186/1742-4682-10-49</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1186%2F1742-4682-10-49" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=23981594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXis1Kisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=49&author=H.+Parkauthor=K.+R.+Yuauthor=B.+Kuauthor=B.+Y.+Kimauthor=S.+J.+Kim&title=Identification+of+novel+PTPRQ+phosphatase+inhibitors+based+on+the+virtual+screening+with+docking+simulations&doi=10.1186%2F1742-4682-10-49"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit130R"><div class="casContent"><span class="casTitleNuber">130</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel PTPRQ phosphatase inhibitors based on the virtual screening with docking simulations</span></div><div class="casAuthors">Park, Hwangseo; Yu, Keum Ran; Ku, Bonsu; Kim, Bo Yeon; Kim, Seung Jun</div><div class="citationInfo"><span class="NLM_cas:title">Theoretical Biology and Medical Modelling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">49/1-49/10, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">TBMMA8</span>;
        ISSN:<span class="NLM_cas:issn">1742-4682</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Protein tyrosine phosphatase receptor type Q (PTPRQ) is an unusual PTP that has intrinsic dephosphorylating activity for various phosphatidyl inositides instead of phospho-tyrosine substrates.  Although PTPRQ was known to be involved in the pathogenesis of obesity, no small-mol. inhibitor has been reported so far.  Here we report six novel PTPRQ inhibitors identified with computer-aided drug design protocol involving the virtual screening with docking simulations and enzyme inhibition assay.  These inhibitors exhibit moderate potencies against PTPRQ with the assocd. IC50 values ranging from 29 to 86 μM.  Because the newly discovered inhibitors were also computationally screened for having desirable physicochem. properties as a drug candidate, they deserve consideration for further development by structure-activity relationship studies to optimize the antiobestic activities.  Structural features relevant to the stabilization of the inhibitors in the active site of PTPRQ are addressed in detail.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmk6jn7B36uLVg90H21EOLACvtfcHk0lhNZonEucmTjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXis1Kisr8%253D&md5=27f8ae52fff042eba62b3ab816d0b303</span></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1186%2F1742-4682-10-49&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1742-4682-10-49%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DK.%2BR.%26aulast%3DKu%26aufirst%3DB.%26aulast%3DKim%26aufirst%3DB.%2BY.%26aulast%3DKim%26aufirst%3DS.%2BJ.%26atitle%3DIdentification%2520of%2520novel%2520PTPRQ%2520phosphatase%2520inhibitors%2520based%2520on%2520the%2520virtual%2520screening%2520with%2520docking%2520simulations%26jtitle%3DTheor.%2520Biol.%2520Med.%2520Modell.%26date%3D2013%26volume%3D10%26spage%3D49%26doi%3D10.1186%2F1742-4682-10-49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref131"><div class="reference"><strong class="refLabel"><a href="#ref131" class="rightTabRefNumLink">131</a></strong><div class="NLM_citation" id="rightTab-cit131"><span><span class="NLM_contrib-group">Hou, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, H.</span><span> </span><span class="NLM_article-title">Protein flexibility in docking-based virtual screening: discovery of novel lymphoid-specific tyrosine phosphatase inhibitors using multiple crystal structures</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">1973</span><span class="NLM_x">–</span> <span class="NLM_lpage">1983</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00344</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00344" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyntbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=1973-1983&author=X.+Houauthor=K.+Liauthor=X.+Yuauthor=J.+P.+Sunauthor=H.+Fang&title=Protein+flexibility+in+docking-based+virtual+screening%3A+discovery+of+novel+lymphoid-specific+tyrosine+phosphatase+inhibitors+using+multiple+crystal+structures&doi=10.1021%2Facs.jcim.5b00344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit131R"><div class="casContent"><span class="casTitleNuber">131</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Flexibility in Docking-Based Virtual Screening: Discovery of Novel Lymphoid-Specific Tyrosine Phosphatase Inhibitors Using Multiple Crystal Structures</span></div><div class="casAuthors">Hou, Xuben; Li, Kangshuai; Yu, Xiao; Sun, Jin-peng; Fang, Hao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1973-1983</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Incorporating protein flexibility is a major challenge for docking-based virtual screening.  With an increasing no. of available crystal structures, ensemble docking with multiple protein structures is an efficient approach to deal with protein flexibility.  Herein, the authors report the successful application of a docking-based virtual screen using multiple crystal structures to discover novel inhibitors of lymphoid-specific tyrosine phosphatase (LYP), a potential drug target for autoimmune diseases.  The appropriate use of multiple protein structures allowed a better enrichment than a single structure in the recovery of known inhibitors.  Subsequently, an optimal ensemble of LYP structures was selected and used in docking-based virtual screening.  Eight novel LYP inhibitors (IC50 ranging from 7.95 to 56.6 μM) were identified among 23 hit compds.  Further studies demonstrated that the most active compd. B15 possessed some selectivity over other protein phosphatases and could effectively up-regulate TCR (T cell receptor)-mediated signaling in Jurkat T cells.  These novel hits not only provided good starting points for the development of therapeutic agents useful in autoimmune diseases but also demonstrated the advantages of choosing an appropriate ensemble of protein structures in docking-based virtual screening over using a single protein conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiDcC42tLbKLVg90H21EOLACvtfcHk0lhNZonEucmTjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyntbvM&md5=9296da92e46d98f8363906ff1c14fd70</span></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00344%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DJ.%2BP.%26aulast%3DFang%26aufirst%3DH.%26atitle%3DProtein%2520flexibility%2520in%2520docking-based%2520virtual%2520screening%253A%2520discovery%2520of%2520novel%2520lymphoid-specific%2520tyrosine%2520phosphatase%2520inhibitors%2520using%2520multiple%2520crystal%2520structures%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D1973%26epage%3D1983%26doi%3D10.1021%2Facs.jcim.5b00344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref132"><div class="reference"><strong class="refLabel"><a href="#ref132" class="rightTabRefNumLink">132</a></strong><div class="NLM_citation" id="rightTab-cit132"><span><span class="NLM_contrib-group">Petersen, G. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saxena, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renuka, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yogeeswari, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santos, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bizarro, C. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sriram, D.</span><span> </span><span class="NLM_article-title">Structure-based virtual screening as a tool for the identification of novel inhibitors against Mycobacterium tuberculosis 3-dehydroquinate dehydratase</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">124</span><span class="NLM_x">–</span> <span class="NLM_lpage">131</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2015.05.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.jmgm.2015.05.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVaksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2015&pages=124-131&author=G.+O.+Petersenauthor=S.+Saxenaauthor=J.+Renukaauthor=V.+Soniauthor=P.+Yogeeswariauthor=D.+S.+Santosauthor=C.+V.+Bizarroauthor=D.+Sriram&title=Structure-based+virtual+screening+as+a+tool+for+the+identification+of+novel+inhibitors+against+Mycobacterium+tuberculosis+3-dehydroquinate+dehydratase&doi=10.1016%2Fj.jmgm.2015.05.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit132R"><div class="casContent"><span class="casTitleNuber">132</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based virtual screening as a tool for the identification of novel inhibitors against Mycobacterium tuberculosis 3-dehydroquinate dehydratase</span></div><div class="casAuthors">Petersen, Guilherme O.; Saxena, Shalini; Renuka, Janupally; Soni, Vijay; Yogeeswari, Perumal; Santos, Diogenes S.; Bizarro, Cristiano V.; Sriram, Dharmarajan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">124-131</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">3-Dehydroquinate dehydratase (DHQase), the third enzyme of the shikimate pathway, catalyzes the reversible reaction of 3-dehydroquinate into 3-dehydroshikimate.  The aim of the present study was to identify new drug-like mols. as inhibitors for Mycobacterium tuberculosis DHQase employing structure-based pharmacophore modeling technique using an in house database consisting of about 2500 small mols.  Further the pharmacophore models were validated using enrichment calcns., and finally three models were employed for high-throughput virtual screening and docking to identify novel small mols. as DHQase inhibitors.  Five compds. were identified, out of which, one mol. (Lead 1) showed 58% inhibition at 50 μM concn. in the Mtb DHQase assay.  Chem. derivs. of the Lead 1 when tested evolved top two hits with IC50s of 17.1 and 31.5 μM as well as MIC values of 25 and 6.25 μg/mL resp. and no cytotoxicity up to 100 μM concn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAcR8epVnSz7Vg90H21EOLACvtfcHk0lgwG9_w726vig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVaksr4%253D&md5=318eb834c417191820854cd944ccb385</span></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2015.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2015.05.001%26sid%3Dliteratum%253Aachs%26aulast%3DPetersen%26aufirst%3DG.%2BO.%26aulast%3DSaxena%26aufirst%3DS.%26aulast%3DRenuka%26aufirst%3DJ.%26aulast%3DSoni%26aufirst%3DV.%26aulast%3DYogeeswari%26aufirst%3DP.%26aulast%3DSantos%26aufirst%3DD.%2BS.%26aulast%3DBizarro%26aufirst%3DC.%2BV.%26aulast%3DSriram%26aufirst%3DD.%26atitle%3DStructure-based%2520virtual%2520screening%2520as%2520a%2520tool%2520for%2520the%2520identification%2520of%2520novel%2520inhibitors%2520against%2520Mycobacterium%2520tuberculosis%25203-dehydroquinate%2520dehydratase%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2015%26volume%3D60%26spage%3D124%26epage%3D131%26doi%3D10.1016%2Fj.jmgm.2015.05.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref133"><div class="reference"><strong class="refLabel"><a href="#ref133" class="rightTabRefNumLink">133</a></strong><div class="NLM_citation" id="rightTab-cit133"><span><span class="NLM_contrib-group">Park, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kyung, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoon, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryu, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeong, D. G.</span><span> </span><span class="NLM_article-title">Structure-based virtual screening approach to the discovery of novel PTPMT1 phosphatase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1271</span><span class="NLM_x">–</span> <span class="NLM_lpage">1275</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2011.10.083</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2011.10.083" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=1271-1275&author=H.+Parkauthor=S.+Y.+Kimauthor=A.+Kyungauthor=T.+S.+Yoonauthor=S.+E.+Ryuauthor=D.+G.+Jeong&title=Structure-based+virtual+screening+approach+to+the+discovery+of+novel+PTPMT1+phosphatase+inhibitors&doi=10.1016%2Fj.bmcl.2011.10.083"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.10.083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.10.083%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DS.%2BY.%26aulast%3DKyung%26aufirst%3DA.%26aulast%3DYoon%26aufirst%3DT.%2BS.%26aulast%3DRyu%26aufirst%3DS.%2BE.%26aulast%3DJeong%26aufirst%3DD.%2BG.%26atitle%3DStructure-based%2520virtual%2520screening%2520approach%2520to%2520the%2520discovery%2520of%2520novel%2520PTPMT1%2520phosphatase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D1271%26epage%3D1275%26doi%3D10.1016%2Fj.bmcl.2011.10.083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref134"><div class="reference"><strong class="refLabel"><a href="#ref134" class="rightTabRefNumLink">134</a></strong><div class="NLM_citation" id="rightTab-cit134"><span><span class="NLM_contrib-group">Wolan, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zorn, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, J. A.</span><span> </span><span class="NLM_article-title">Small-molecule activators of a proenzyme</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">326</span><span class="NLM_x">, </span> <span class="NLM_fpage">853</span><span class="NLM_x">–</span> <span class="NLM_lpage">858</span><span class="refDoi"> DOI: 10.1126/science.1177585</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1126%2Fscience.1177585" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=19892984" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGhs7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=326&publication_year=2009&pages=853-858&author=D.+W.+Wolanauthor=J.+A.+Zornauthor=D.+C.+Grayauthor=J.+A.+Wells&title=Small-molecule+activators+of+a+proenzyme&doi=10.1126%2Fscience.1177585"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit134R"><div class="casContent"><span class="casTitleNuber">134</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Activators of a Proenzyme</span></div><div class="casAuthors">Wolan, Dennis W.; Zorn, Julie A.; Gray, Daniel C.; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">326</span>
        (<span class="NLM_cas:issue">5954</span>),
    <span class="NLM_cas:pages">853-858</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Virtually all of the 560 human proteases are stored as inactive proenzymes and are strictly regulated.  We report the identification and characterization of the first small mols. that directly activate proenzymes, the apoptotic procaspases-3 and -6.  It is surprising that these compds. induce autoproteolytic activation by stabilizing a conformation that is both more active and more susceptible to intermol. proteolysis.  These procaspase activators bypass the normal upstream proapoptotic signaling cascades and induce rapid apoptosis in a variety of cell lines.  Systematic biochem. and biophys. analyses identified a cluster of mutations in procaspase-3 that resist small-mol. activation both in vitro and in cells.  Compds. that induce gain of function are rare, and the activators reported here will enable direct control of the executioner caspases in apoptosis and in cellular differentiation.  More generally, these studies presage the discovery of other proenzyme activators to explore fundamental processes of proenzyme activation and their fate-detg. roles in biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrq87GHAd7ZRrVg90H21EOLACvtfcHk0lgKKSlcC0qTig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGhs7%252FK&md5=cdc123b08c2c8b84da15687638657c3a</span></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1126%2Fscience.1177585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1177585%26sid%3Dliteratum%253Aachs%26aulast%3DWolan%26aufirst%3DD.%2BW.%26aulast%3DZorn%26aufirst%3DJ.%2BA.%26aulast%3DGray%26aufirst%3DD.%2BC.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DSmall-molecule%2520activators%2520of%2520a%2520proenzyme%26jtitle%3DScience%26date%3D2009%26volume%3D326%26spage%3D853%26epage%3D858%26doi%3D10.1126%2Fscience.1177585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref135"><div class="reference"><strong class="refLabel"><a href="#ref135" class="rightTabRefNumLink">135</a></strong><div class="NLM_citation" id="rightTab-cit135"><span><span class="NLM_contrib-group">Allen, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gochin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizzo, R. C.</span><span> </span><span class="NLM_article-title">Small molecule inhibitors of HIVgp41 N-heptad repeat trimer formation</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">2853</span><span class="NLM_x">–</span> <span class="NLM_lpage">2859</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2015.04.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2015.04.067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26013847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsFyiurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=2853-2859&author=W.+J.+Allenauthor=H.+A.+Yiauthor=M.+Gochinauthor=A.+Jacobsauthor=R.+C.+Rizzo&title=Small+molecule+inhibitors+of+HIVgp41+N-heptad+repeat+trimer+formation&doi=10.1016%2Fj.bmcl.2015.04.067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit135R"><div class="casContent"><span class="casTitleNuber">135</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibitors of HIVgp41 N-heptad repeat trimer formation</span></div><div class="casAuthors">Allen, William J.; Yi, Hyun Ah; Gochin, Miriam; Jacobs, Amy; Rizzo, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2853-2859</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Identification of mechanistically novel anti-HIV fusion inhibitors was accomplished using a computer-aided structure-based design approach with the goal of blocking the formation of the N-heptad repeat (NHR) trimer of the viral protein gp41.  A virtual screening strategy that included per-residue interaction patterns (footprints) was employed to identify small mols. compatible with putative binding pockets at the internal interface of the NHR helixes at the core native viral six-helix bundle.  From a screen of ∼2.8 million compds. using the DOCK program, 120 with favorable energetic and footprint overlap characteristics were purchased and exptl. tested leading to two compds. with favorable cell-cell fusion (IC50) and cytotoxicity profiles.  Importantly, both hits were identified on the basis of scores contg. footprint overlap terms and would not have been identified using the std. DOCK energy function alone.  To our knowledge, these compds. represent the first reported small mols. that inhibit viral entry via the proposed NHR-trimer obstruction mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwtzEm7EC52bVg90H21EOLACvtfcHk0lgKKSlcC0qTig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsFyiurs%253D&md5=a46287d045987ac50e65263481f263ba</span></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.04.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.04.067%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DW.%2BJ.%26aulast%3DYi%26aufirst%3DH.%2BA.%26aulast%3DGochin%26aufirst%3DM.%26aulast%3DJacobs%26aufirst%3DA.%26aulast%3DRizzo%26aufirst%3DR.%2BC.%26atitle%3DSmall%2520molecule%2520inhibitors%2520of%2520HIVgp41%2520N-heptad%2520repeat%2520trimer%2520formation%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D2853%26epage%3D2859%26doi%3D10.1016%2Fj.bmcl.2015.04.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref136"><div class="reference"><strong class="refLabel"><a href="#ref136" class="rightTabRefNumLink">136</a></strong><div class="NLM_citation" id="rightTab-cit136"><span><span class="NLM_contrib-group">Dong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jing, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Y.</span><span> </span><span class="NLM_article-title">Pharmacophore identification, virtual screening and biological evaluation of prenylated flavonoids derivatives as PKB/Akt1 inhibitors</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">5949</span><span class="NLM_x">–</span> <span class="NLM_lpage">5958</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2011.10.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.ejmech.2011.10.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=22027102" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVKqtLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2011&pages=5949-5958&author=X.+Dongauthor=X.+Zhouauthor=H.+Jingauthor=J.+Chenauthor=T.+Liuauthor=B.+Yangauthor=Q.+Heauthor=Y.+Hu&title=Pharmacophore+identification%2C+virtual+screening+and+biological+evaluation+of+prenylated+flavonoids+derivatives+as+PKB%2FAkt1+inhibitors&doi=10.1016%2Fj.ejmech.2011.10.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit136R"><div class="casContent"><span class="casTitleNuber">136</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacophore identification, virtual screening and biological evaluation of prenylated flavonoids derivatives as PKB/Akt1 inhibitors</span></div><div class="casAuthors">Dong, Xiaowu; Zhou, Xinglu; Jing, Hui; Chen, Jianzhong; Liu, Tao; Yang, Bo; He, Qiaojun; Hu, Yongzhou</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5949-5958</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A total of 24 well-defined PKB/Akt1 inhibitors were used to generate pharmacophore models applying Catalyst/HypoGen program.  The best ranked model (Hypo_1) was then validated by cost anal., prediction capability, Cat-Scramble and receiver operating characteristic (ROC) studies.  Then, pharmacophore-based virtual screening combined with docking study was performed to search an inhouse compd. database.  Nine preferable hits 75-80, HTS-02143, BTB-14740 and HTS-08006 were prepd. and biol. evaluated.  Several compds. were identified as good PKB/Akt1 inhibitors, suggesting that Hypo_1 would be reliable and useful in virtual screening.  Flow cytometric and western blotting anal. on compds. 79 and 80 further demonstrated that the inhibition of phosphorylation of PKB/Akt1 and its substrates (such as GSK3β) was responsible for their cytotoxic activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXTEc-nFDC_rVg90H21EOLACvtfcHk0lgKKSlcC0qTig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVKqtLnE&md5=e563fb1a3ea5bbbc7c1ebff8753f38cf</span></div><a href="/servlet/linkout?suffix=cit136&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2011.10.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2011.10.006%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DJing%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DHe%26aufirst%3DQ.%26aulast%3DHu%26aufirst%3DY.%26atitle%3DPharmacophore%2520identification%252C%2520virtual%2520screening%2520and%2520biological%2520evaluation%2520of%2520prenylated%2520flavonoids%2520derivatives%2520as%2520PKB%252FAkt1%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2011%26volume%3D46%26spage%3D5949%26epage%3D5958%26doi%3D10.1016%2Fj.ejmech.2011.10.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref137"><div class="reference"><strong class="refLabel"><a href="#ref137" class="rightTabRefNumLink">137</a></strong><div class="NLM_citation" id="rightTab-cit137"><span><span class="NLM_contrib-group">Zhong, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, Z. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span> </span><span class="NLM_article-title">Structure-based discovery of an immunomodulatory inhibitor of TLR1-TLR2 heterodimerization from a natural product-like database</span> <span class="citation_source-journal">Chem. Commun. (Cambridge, U. K.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">11178</span><span class="NLM_x">–</span> <span class="NLM_lpage">11181</span><span class="refDoi"> DOI: 10.1039/C5CC02728D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1039%2FC5CC02728D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26051605" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXoslOnurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2015&pages=11178-11181&author=Z.+Zhongauthor=L.+J.+Liuauthor=Z.+Q.+Dongauthor=L.+Luauthor=M.+Wangauthor=C.+H.+Leungauthor=D.+L.+Maauthor=Y.+Wang&title=Structure-based+discovery+of+an+immunomodulatory+inhibitor+of+TLR1-TLR2+heterodimerization+from+a+natural+product-like+database&doi=10.1039%2FC5CC02728D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit137R"><div class="casContent"><span class="casTitleNuber">137</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of an immunomodulatory inhibitor of TLR1-TLR2 heterodimerization from a natural product-like database</span></div><div class="casAuthors">Zhong, Zhangfeng; Liu, Li-Juan; Dong, Zhi-Qiang; Lu, Lihua; Wang, Modi; Leung, Chung-Hang; Ma, Dik-Lung; Wang, Yitao</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">56</span>),
    <span class="NLM_cas:pages">11178-11181</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">We report herein the identification of an immunomodulatory natural product-like compd. 1 as a direct inhibitor of TLR1-TLR2 heterodimerization.  Compd. 1 suppressed TNF-α and IL-6 secretion in Pam3CSK4-induced macrophages.  Moreover, compd. 1 inhibited the phagocytic activity of macrophages, presumably through modulation of TLR1-TLR2 signaling and inactivation of NF-κB.  Mol. docking revealed that compd. 1 bound to the interface region of TLR1-TLR2 by forming two hydrogen bonds with residues lining the binding site.  To our knowledge, compd. 1 has been only the second inhibitor overall of TLR1-TLR2 heterodimerization reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqurqIe4Y09JLVg90H21EOLACvtfcHk0lhqp_241_D8Ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXoslOnurk%253D&md5=89503b243b836322a73a541d22c56e39</span></div><a href="/servlet/linkout?suffix=cit137&amp;dbid=16384&amp;doi=10.1039%2FC5CC02728D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5CC02728D%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DL.%2BJ.%26aulast%3DDong%26aufirst%3DZ.%2BQ.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLeung%26aufirst%3DC.%2BH.%26aulast%3DMa%26aufirst%3DD.%2BL.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DStructure-based%2520discovery%2520of%2520an%2520immunomodulatory%2520inhibitor%2520of%2520TLR1-TLR2%2520heterodimerization%2520from%2520a%2520natural%2520product-like%2520database%26jtitle%3DChem.%2520Commun.%2520%2528Cambridge%252C%2520U.%2520K.%2529%26date%3D2015%26volume%3D51%26spage%3D11178%26epage%3D11181%26doi%3D10.1039%2FC5CC02728D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref138"><div class="reference"><strong class="refLabel"><a href="#ref138" class="rightTabRefNumLink">138</a></strong><div class="NLM_citation" id="rightTab-cit138"><span><span class="NLM_contrib-group">Mohammed, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Numair, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balakrishnan, A.</span><span> </span><span class="NLM_article-title">Docking studies on the interaction of flavonoids with fat mass and obesity associated protein</span> <span class="citation_source-journal">Pak. J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1647</span><span class="NLM_x">–</span> <span class="NLM_lpage">1653</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26408884" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BC283mvFKjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=1647-1653&author=A.+Mohammedauthor=K.+S.+Al-Numairauthor=A.+Balakrishnan&title=Docking+studies+on+the+interaction+of+flavonoids+with+fat+mass+and+obesity+associated+protein"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit138R"><div class="casContent"><span class="casTitleNuber">138</span><div class="casTitle"><span class="NLM_cas:atitle">Docking studies on the interaction of flavonoids with fat mass and obesity associated protein</span></div><div class="casAuthors">Mohammed Alamri; Al-Numair Khalid Suliman; Balakrishnan Aristatile</div><div class="citationInfo"><span class="NLM_cas:title">Pakistan journal of pharmaceutical sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1647-53</span>
        ISSN:<span class="NLM_cas:issn">1011-601X</span>.
    </div><div class="casAbstract">Obesity is the excessive fat accumulation in human body leading to increases a risk of various chronic diseases such as diabetes, cardiovascular diseases, cancer and osteoarthritis.  Several flavonoids are known to have lipolytic activity influencing lipolysis and adipogenesis in adipose cells.  To explore mechanism of the association of flavonoids in obesity and obesity associated protein (FTO), molecular docking studies were done for FTO with flavonoids, with orlistat (antiobesity drug) as a control.  Autodock tools were used for docking flavonoids and orlistat with FTO.  The results were visualized by PyMol and Discovery studio visualizer.  Upon docking simulation, it was observed that flavonoid quercetin showed highest binding affinity (most negative δG), whereas daidzein was least affinity towards FTO.  The binding affinity of other flavonoids was in the order of Exemestane >Kaempherol >Letrozole >Rutin.  This study concludes that flavonoids primarily, quercetin ameliorates obesity by establishing a physical interaction with FTO.  Interactions were also observed between FTO and other flavonoids and were of not greater inhibition compared to quercetin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRUkpRKZyo6CTdblLE_E9PWfW6udTcc2ebG5yiodVMZQbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC283mvFKjuw%253D%253D&md5=f261e8d6e1caaa58a9f7923281adbb61</span></div><a href="/servlet/linkout?suffix=cit138&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMohammed%26aufirst%3DA.%26aulast%3DAl-Numair%26aufirst%3DK.%2BS.%26aulast%3DBalakrishnan%26aufirst%3DA.%26atitle%3DDocking%2520studies%2520on%2520the%2520interaction%2520of%2520flavonoids%2520with%2520fat%2520mass%2520and%2520obesity%2520associated%2520protein%26jtitle%3DPak.%2520J.%2520Pharm.%2520Sci.%26date%3D2015%26volume%3D28%26spage%3D1647%26epage%3D1653" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref139"><div class="reference"><strong class="refLabel"><a href="#ref139" class="rightTabRefNumLink">139</a></strong><div class="NLM_citation" id="rightTab-cit139"><span><span class="NLM_contrib-group">Brecher, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banavali, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kramer, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span> </span><span class="NLM_article-title">Novel broad spectrum inhibitors targeting the flavivirus methyltransferase</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0130062</span><span class="refDoi"> DOI: 10.1371/journal.pone.0130062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pone.0130062" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0130062&author=M.+Brecherauthor=H.+Chenauthor=B.+Liuauthor=N.+K.+Banavaliauthor=S.+A.+Jonesauthor=J.+Zhangauthor=Z.+Liauthor=L.+D.+Kramerauthor=H.+Li&title=Novel+broad+spectrum+inhibitors+targeting+the+flavivirus+methyltransferase&doi=10.1371%2Fjournal.pone.0130062"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit139&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0130062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0130062%26sid%3Dliteratum%253Aachs%26aulast%3DBrecher%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DBanavali%26aufirst%3DN.%2BK.%26aulast%3DJones%26aufirst%3DS.%2BA.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DKramer%26aufirst%3DL.%2BD.%26aulast%3DLi%26aufirst%3DH.%26atitle%3DNovel%2520broad%2520spectrum%2520inhibitors%2520targeting%2520the%2520flavivirus%2520methyltransferase%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0130062%26doi%3D10.1371%2Fjournal.pone.0130062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref140"><div class="reference"><strong class="refLabel"><a href="#ref140" class="rightTabRefNumLink">140</a></strong><div class="NLM_citation" id="rightTab-cit140"><span><span class="NLM_contrib-group">Walters, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Namchuk, M.</span><span> </span><span class="NLM_article-title">Designing screens: how to make your hits a hit</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">259</span><span class="NLM_x">–</span> <span class="NLM_lpage">266</span><span class="refDoi"> DOI: 10.1038/nrd1063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnrd1063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=12669025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3sXisFWktrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=259-266&author=W.+Waltersauthor=M.+Namchuk&title=Designing+screens%3A+how+to+make+your+hits+a+hit&doi=10.1038%2Fnrd1063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit140R"><div class="casContent"><span class="casTitleNuber">140</span><div class="casTitle"><span class="NLM_cas:atitle">A guide to drug discovery: Designing screens: how to make your hits a hit</span></div><div class="casAuthors">Walters, W. Patrick; Namchuk, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">259-266</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The basic goal of small-mol. screening is the identification of chem. interesting starting points for elaboration towards a drug.  A no. of innovative approaches for pursuing this goal have evolved, and the right approach is dictated by the target class being pursued and the capabilities of the organization involved.hood.  A recent trend in high-throughput screening has been to place less emphasis on the no. of data points that can be produced, and to focus instead on the quality of the data obtained.  Several computational and technol. advances have aided in the selection of compds. for screening and widened the variety of assay formats available for screening.  The effect on the efficiency of the screening process is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ5E12ik7BrbVg90H21EOLACvtfcHk0lhqp_241_D8Ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXisFWktrk%253D&md5=6bec4547b8dce69ab84ce226cde28f8a</span></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.1038%2Fnrd1063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1063%26sid%3Dliteratum%253Aachs%26aulast%3DWalters%26aufirst%3DW.%26aulast%3DNamchuk%26aufirst%3DM.%26atitle%3DDesigning%2520screens%253A%2520how%2520to%2520make%2520your%2520hits%2520a%2520hit%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D259%26epage%3D266%26doi%3D10.1038%2Fnrd1063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref141"><div class="reference"><strong class="refLabel"><a href="#ref141" class="rightTabRefNumLink">141</a></strong><div class="NLM_citation" id="rightTab-cit141"><span><span class="NLM_contrib-group">Thorne, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lea, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lovell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auld, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span> </span><span class="NLM_article-title">Firefly luciferase in chemical biology: a compendium of inhibitors, mechanistic evaluation of chemotypes, and suggested use as a reporter</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1060</span><span class="NLM_x">–</span> <span class="NLM_lpage">1072</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2012.07.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.chembiol.2012.07.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=22921073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1KkurjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1060-1072&author=N.+Thorneauthor=M.+Shenauthor=W.+A.+Leaauthor=A.+Simeonovauthor=S.+Lovellauthor=D.+S.+Auldauthor=J.+Inglese&title=Firefly+luciferase+in+chemical+biology%3A+a+compendium+of+inhibitors%2C+mechanistic+evaluation+of+chemotypes%2C+and+suggested+use+as+a+reporter&doi=10.1016%2Fj.chembiol.2012.07.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit141R"><div class="casContent"><span class="casTitleNuber">141</span><div class="casTitle"><span class="NLM_cas:atitle">Firefly Luciferase in Chemical Biology: A Compendium of Inhibitors, Mechanistic Evaluation of Chemotypes, and Suggested Use As a Reporter</span></div><div class="casAuthors">Thorne, Natasha; Shen, Min; Lea, Wendy A.; Simeonov, Anton; Lovell, Scott; Auld, Douglas S.; Inglese, James</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1060-1072</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Firefly luciferase (FLuc) is frequently used as a reporter in high-throughput screening assays, owing to the exceptional sensitivity, dynamic range, and rapid measurement that bioluminescence affords.  However, interaction of small mols. with FLuc has, to some extent, confounded its use in chem. biol. and drug discovery.  To identify and characterize chemotypes interacting with FLuc, we detd. potency values for 360,864 compds. found in the NIH Mol. Libraries Small Mol. Repository, available in PubChem.  FLuc inhibitory activity was obsd. for 12% of this library with discernible SAR.  Characterization of 151 inhibitors demonstrated a variety of inhibition modes, including FLuc-catalyzed formation of multisubstrate adduct enzyme inhibitor complexes.  As in some cell-based FLuc reporter assays, compds. acting as FLuc inhibitors yield paradoxical luminescence increases, thus data on compds. acquired from FLuc-dependent assays require careful anal. as described here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWw4FbLlFcILVg90H21EOLACvtfcHk0ljT052la8nSiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1KkurjM&md5=5ab0f756ab0a3ece1ea28ffac52bbd5f</span></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.07.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.07.015%26sid%3Dliteratum%253Aachs%26aulast%3DThorne%26aufirst%3DN.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DLea%26aufirst%3DW.%2BA.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DInglese%26aufirst%3DJ.%26atitle%3DFirefly%2520luciferase%2520in%2520chemical%2520biology%253A%2520a%2520compendium%2520of%2520inhibitors%252C%2520mechanistic%2520evaluation%2520of%2520chemotypes%252C%2520and%2520suggested%2520use%2520as%2520a%2520reporter%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D1060%26epage%3D1072%26doi%3D10.1016%2Fj.chembiol.2012.07.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref142"><div class="reference"><strong class="refLabel"><a href="#ref142" class="rightTabRefNumLink">142</a></strong><div class="NLM_citation" id="rightTab-cit142"><span><span class="NLM_contrib-group">Feng, B. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">A detergent-based assay for the detection of promiscuous inhibitors</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">550</span><span class="NLM_x">–</span> <span class="NLM_lpage">553</span><span class="refDoi"> DOI: 10.1038/nprot.2006.77</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnprot.2006.77" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=17191086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFOitLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=550-553&author=B.+Y.+Fengauthor=B.+K.+Shoichet&title=A+detergent-based+assay+for+the+detection+of+promiscuous+inhibitors&doi=10.1038%2Fnprot.2006.77"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit142R"><div class="casContent"><span class="casTitleNuber">142</span><div class="casTitle"><span class="NLM_cas:atitle">A detergent-based assay for the detection of promiscuous inhibitors</span></div><div class="casAuthors">Feng, Brian Y.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">550-553</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">At micromolar concns., many small mols. self-assoc. into colloidal aggregates that non-specifically inhibit enzymes and other proteins.  Here the authors describe a protocol for identifying aggregate-based inhibitors and distinguishing them from small mols. that inhibit via specific mechanisms.  As a convenient proxy for promiscuous, aggregate-based inhibition, the authors monitor inhibition of β-lactamase in the absence and presence of detergent.  Inhibition that is attenuated in the presence of detergent is characteristic of an aggregate-based mechanism.  In the 96-well-format assay described here, about 200 mols. can be tested, in duplicate, per h for detergent-dependent sensitivity.  Furthermore, the authors also describe simple expts. that can offer addnl. confirmation of aggregate-based inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplGQSBvMgM-7Vg90H21EOLACvtfcHk0ljT052la8nSiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFOitLjO&md5=8aa7562b5f69c896af7e133ecc35e6de</span></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2006.77&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2006.77%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DB.%2BY.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DA%2520detergent-based%2520assay%2520for%2520the%2520detection%2520of%2520promiscuous%2520inhibitors%26jtitle%3DNat.%2520Protoc.%26date%3D2006%26volume%3D1%26spage%3D550%26epage%3D553%26doi%3D10.1038%2Fnprot.2006.77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref143"><div class="reference"><strong class="refLabel"><a href="#ref143" class="rightTabRefNumLink">143</a></strong><div class="NLM_citation" id="rightTab-cit143"><span><span class="NLM_contrib-group">Sassano, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doak, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Colloidal aggregation causes inhibition of G protein-coupled receptors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2406</span><span class="NLM_x">–</span> <span class="NLM_lpage">2414</span><span class="refDoi"> DOI: 10.1021/jm301749y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301749y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVCgt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2406-2414&author=M.+F.+Sassanoauthor=A.+K.+Doakauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Colloidal+aggregation+causes+inhibition+of+G+protein-coupled+receptors&doi=10.1021%2Fjm301749y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit143R"><div class="casContent"><span class="casTitleNuber">143</span><div class="casTitle"><span class="NLM_cas:atitle">Colloidal aggregation causes inhibition of G protein-coupled receptors</span></div><div class="casAuthors">Sassano, Maria F.; Doak, Allison K.; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2406-2414</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Colloidal aggregation is the dominant mechanism for artifactual inhibition of sol. proteins, and controls against it are now widely deployed.  Conversely, investigating this mechanism for membrane-bound receptors has proven difficult.  Here, the authors investigated the activity of 4 well-characterized aggregators against 3 G protein-coupled receptors (GPCRs) recognizing peptide and protein ligands.  Each of the aggregators was active at micromolar concns. against the 3 GPCRs in cell-based assays.  This activity could be attenuated by either centrifugation of the inhibitor stock soln. or by the addn. of Tween-80 detergent.  In the absence of agonist, the aggregators acted as inverse agonists, consistent with a direct receptor interaction.  Meanwhile, several literature GPCR ligands that resembled aggregators themselves formed colloids, by both phys. and enzymol. tests.  These observations suggest that some GPCRs may be artifactually antagonized by colloidal aggregates, an effect that merits the attention of investigators in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojG9CcuOYoXrVg90H21EOLACvtfcHk0ljT052la8nSiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVCgt7s%253D&md5=84fa59f9635330739bf836f8085be5e0</span></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.1021%2Fjm301749y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301749y%26sid%3Dliteratum%253Aachs%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DColloidal%2520aggregation%2520causes%2520inhibition%2520of%2520G%2520protein-coupled%2520receptors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2406%26epage%3D2414%26doi%3D10.1021%2Fjm301749y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref144"><div class="reference"><strong class="refLabel"><a href="#ref144" class="rightTabRefNumLink">144</a></strong><div class="NLM_citation" id="rightTab-cit144"><span><span class="NLM_contrib-group">Owen, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doak, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wassam, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Colloidal aggregation affects the efficacy of anticancer drugs in cell culture</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1429</span><span class="NLM_x">–</span> <span class="NLM_lpage">1435</span><span class="refDoi"> DOI: 10.1021/cb300189b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300189b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsFOmsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1429-1435&author=S.+C.+Owenauthor=A.+K.+Doakauthor=P.+Wassamauthor=M.+S.+Shoichetauthor=B.+K.+Shoichet&title=Colloidal+aggregation+affects+the+efficacy+of+anticancer+drugs+in+cell+culture&doi=10.1021%2Fcb300189b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit144R"><div class="casContent"><span class="casTitleNuber">144</span><div class="casTitle"><span class="NLM_cas:atitle">Colloidal Aggregation Affects the Efficacy of Anticancer Drugs in Cell Culture</span></div><div class="casAuthors">Owen, Shawn C.; Doak, Allison K.; Wassam, Pascal; Shoichet, Molly S.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1429-1435</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many small mols., including bioactive mols. and approved drugs, spontaneously form colloidal aggregates in aq. soln. at micromolar concns.  Though it is widely accepted that aggregation leads to artifacts in screens for ligands of sol. proteins, the effects of colloid formation in cell-based assays have not been studied.  Here, seven anticancer drugs and one diagnostic reagent were found to form colloids in both biochem. buffer and in cell culture media.  In cell-based assays, the antiproliferative activities of three of the drugs were substantially reduced when in colloidal form as compared to monomeric form; a new formulation method ensured the presence of drug colloids vs. drug monomers in soln.  We also found that Evans Blue, a dye classically used to measure vascular permeability and to demonstrate the "enhanced permeability and retention (EPR) effect" in solid tumors, forms colloids that adsorb albumin, as opposed to older literature that suggested the reverse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN47RUN9WrPrVg90H21EOLACvtfcHk0ljT052la8nSiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsFOmsrs%253D&md5=207a6f05e2ee718392562b28d2c52f89</span></div><a href="/servlet/linkout?suffix=cit144&amp;dbid=16384&amp;doi=10.1021%2Fcb300189b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300189b%26sid%3Dliteratum%253Aachs%26aulast%3DOwen%26aufirst%3DS.%2BC.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DWassam%26aufirst%3DP.%26aulast%3DShoichet%26aufirst%3DM.%2BS.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DColloidal%2520aggregation%2520affects%2520the%2520efficacy%2520of%2520anticancer%2520drugs%2520in%2520cell%2520culture%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1429%26epage%3D1435%26doi%3D10.1021%2Fcb300189b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref145"><div class="reference"><strong class="refLabel"><a href="#ref145" class="rightTabRefNumLink">145</a></strong><div class="NLM_citation" id="rightTab-cit145"><span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Interpreting steep dose-response curves in early inhibitor discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">7274</span><span class="NLM_x">–</span> <span class="NLM_lpage">7277</span><span class="refDoi"> DOI: 10.1021/jm061103g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm061103g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtF2it7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=7274-7277&author=B.+K.+Shoichet&title=Interpreting+steep+dose-response+curves+in+early+inhibitor+discovery&doi=10.1021%2Fjm061103g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit145R"><div class="casContent"><span class="casTitleNuber">145</span><div class="casTitle"><span class="NLM_cas:atitle">Interpreting Steep Dose-Response Curves in Early Inhibitor Discovery</span></div><div class="casAuthors">Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">7274-7277</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many screening hits inhibit enzymes with steep dose-response curves, which are considered pathol.  Three models might explain these curves: multisite binding, an inhibitor phase transition, or stoichiometric inhibition caused by a high enzyme to Kd ratio.  Expts. with promiscuous aggregators, for which steep curves are common, suggest that these curves owe to stoichiometric inhibition, which predicts that IC50 should vary linearly with enzyme concn.  Most steep dose-response curves in screening may be due to this effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDnPkuWbZ-YLVg90H21EOLACvtfcHk0lgv2TcB5OZSZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtF2it7fF&md5=97975678ee10f97457dccde7d15173ee</span></div><a href="/servlet/linkout?suffix=cit145&amp;dbid=16384&amp;doi=10.1021%2Fjm061103g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061103g%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DInterpreting%2520steep%2520dose-response%2520curves%2520in%2520early%2520inhibitor%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D7274%26epage%3D7277%26doi%3D10.1021%2Fjm061103g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref146"><div class="reference"><strong class="refLabel"><a href="#ref146" class="rightTabRefNumLink">146</a></strong><div class="NLM_citation" id="rightTab-cit146"><span><span class="NLM_contrib-group">Fukunishi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, H.</span><span> </span><span class="NLM_article-title">Improved estimation of protein-ligand binding free energy by using the ligand-entropy and mobility of water molecules</span> <span class="citation_source-journal">Pharmaceuticals</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">604</span><span class="NLM_x">–</span> <span class="NLM_lpage">622</span><span class="refDoi"> DOI: 10.3390/ph6050604</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.3390%2Fph6050604" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2013&pages=604-622&author=Y.+Fukunishiauthor=H.+Nakamura&title=Improved+estimation+of+protein-ligand+binding+free+energy+by+using+the+ligand-entropy+and+mobility+of+water+molecules&doi=10.3390%2Fph6050604"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit146&amp;dbid=16384&amp;doi=10.3390%2Fph6050604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fph6050604%26sid%3Dliteratum%253Aachs%26aulast%3DFukunishi%26aufirst%3DY.%26aulast%3DNakamura%26aufirst%3DH.%26atitle%3DImproved%2520estimation%2520of%2520protein-ligand%2520binding%2520free%2520energy%2520by%2520using%2520the%2520ligand-entropy%2520and%2520mobility%2520of%2520water%2520molecules%26jtitle%3DPharmaceuticals%26date%3D2013%26volume%3D6%26spage%3D604%26epage%3D622%26doi%3D10.3390%2Fph6050604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref147"><div class="reference"><strong class="refLabel"><a href="#ref147" class="rightTabRefNumLink">147</a></strong><div class="NLM_citation" id="rightTab-cit147"><span><span class="NLM_contrib-group">Barillari, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Essex, J. W.</span><span> </span><span class="NLM_article-title">Classification of water molecules in protein binding sites</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">2577</span><span class="NLM_x">–</span> <span class="NLM_lpage">2587</span><span class="refDoi"> DOI: 10.1021/ja066980q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja066980q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsFGjt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=2577-2587&author=C.+Barillariauthor=J.+Taylorauthor=R.+Vinerauthor=J.+W.+Essex&title=Classification+of+water+molecules+in+protein+binding+sites&doi=10.1021%2Fja066980q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit147R"><div class="casContent"><span class="casTitleNuber">147</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of Water Molecules in Protein Binding Sites</span></div><div class="casAuthors">Barillari, Caterina; Taylor, Justine; Viner, Russell; Essex, Jonathan W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2577-2587</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Water mols. play a crucial role in mediating the interaction between a ligand and a macromol. receptor.  An understanding of the nature and role of each water mol. in the active site of a protein could greatly increase the efficiency of rational drug design approaches: if the propensity of a water mol. for displacement can be detd., then synthetic effort may be most profitably applied to the design of specific ligands with the displacement of this water mol. in mind.  In this paper, a thermodn. anal. of water mols. in the binding sites of six proteins, each complexed with a no. of inhibitors, is presented.  Two classes of water mols. were identified: those conserved and not displaced by any of the ligands, and those that are displaced by some ligands.  The abs. binding free energies of 54 water mols. were calcd. using the double decoupling method, with replica exchange thermodn. integration in Monte Carlo simulations.  It was found that conserved water mols. are on av. more tightly bound than displaced water mols.  In addn., Bayesian statistics is used to calc. the probability that a particular water mol. may be displaced by an appropriately designed ligand, given the calcd. binding free energy of the water mol.  This approach therefore allows the numerical assessment of whether or not a given water mol. should be targeted for displacement as part of a rational drug design strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr38ETxUkD-NrVg90H21EOLACvtfcHk0lgv2TcB5OZSZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsFGjt7o%253D&md5=e74664f19993a9e9c087dbb9c1d54c3e</span></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1021%2Fja066980q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja066980q%26sid%3Dliteratum%253Aachs%26aulast%3DBarillari%26aufirst%3DC.%26aulast%3DTaylor%26aufirst%3DJ.%26aulast%3DViner%26aufirst%3DR.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26atitle%3DClassification%2520of%2520water%2520molecules%2520in%2520protein%2520binding%2520sites%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D2577%26epage%3D2587%26doi%3D10.1021%2Fja066980q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref148"><div class="reference"><strong class="refLabel"><a href="#ref148" class="rightTabRefNumLink">148</a></strong><div class="NLM_citation" id="rightTab-cit148"><span><span class="NLM_contrib-group">Michel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Prediction of the water content in protein binding sites</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">113</span><span class="NLM_x">, </span> <span class="NLM_fpage">13337</span><span class="NLM_x">–</span> <span class="NLM_lpage">13346</span><span class="refDoi"> DOI: 10.1021/jp9047456</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp9047456" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFegsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=13337-13346&author=J.+Michelauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Prediction+of+the+water+content+in+protein+binding+sites&doi=10.1021%2Fjp9047456"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit148R"><div class="casContent"><span class="casTitleNuber">148</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of the Water Content in Protein Binding Sites</span></div><div class="casAuthors">Michel, Julien; Tirado-Rives, Julian; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">13337-13346</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An efficient mol. simulation methodol. has been developed to det. the positioning of water mols. in the binding site of a protein or protein-ligand complex.  Occupancies and abs. binding free energies of water mols. are computed using a statistical thermodn. approach.  The methodol., referred to as Just Add Water Mols. (JAWS), features "θ-water" mols. that can appear and disappear on a binding-site grid.  Key approxns. render the technique far more efficient than conventional free energy simulations.  Testing of JAWS on five diverse examples (neuraminidase, scytalone dehydratase, major urinary protein 1, β-lactoglobulin, and COX-2) demonstrates its accuracy in locating hydration sites in comparison to results from high-resoln. crystal structures.  Possible applications include aid in refinement of protein crystal structures, drug lead optimization, setup of docking calcns., and simulations of protein-ligand complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC6L_AGV7e7LVg90H21EOLACvtfcHk0lgv2TcB5OZSZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFegsr7I&md5=bc36a74922d2b1ce1d15e53b9e144afe</span></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1021%2Fjp9047456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp9047456%26sid%3Dliteratum%253Aachs%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DPrediction%2520of%2520the%2520water%2520content%2520in%2520protein%2520binding%2520sites%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2009%26volume%3D113%26spage%3D13337%26epage%3D13346%26doi%3D10.1021%2Fjp9047456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref149"><div class="reference"><strong class="refLabel"><a href="#ref149" class="rightTabRefNumLink">149</a></strong><div class="NLM_citation" id="rightTab-cit149"><span><span class="NLM_contrib-group">Abel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salam, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farid, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span> </span><span class="NLM_article-title">Contribution of explicit solvent effects to the binding affinity of small-molecule inhibitors in blood coagulation factor serine proteases</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1049</span><span class="NLM_x">–</span> <span class="NLM_lpage">1066</span><span class="refDoi"> DOI: 10.1002/cmdc.201000533</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fcmdc.201000533" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=21506273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmsFGjs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=1049-1066&author=R.+Abelauthor=N.+K.+Salamauthor=J.+Shelleyauthor=R.+Faridauthor=R.+A.+Friesnerauthor=W.+Sherman&title=Contribution+of+explicit+solvent+effects+to+the+binding+affinity+of+small-molecule+inhibitors+in+blood+coagulation+factor+serine+proteases&doi=10.1002%2Fcmdc.201000533"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit149R"><div class="casContent"><span class="casTitleNuber">149</span><div class="casTitle"><span class="NLM_cas:atitle">Contribution of Explicit Solvent Effects to the Binding Affinity of Small-Molecule Inhibitors in Blood Coagulation Factor Serine Proteases</span></div><div class="casAuthors">Abel, Robert; Salam, Noeris K.; Shelley, John; Farid, Ramy; Friesner, Richard A.; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1049-1066</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The prevention of blood coagulation is important in treating thromboembolic disorders, and several serine proteases involved in the coagulation cascade have been classified as pharmaceutically relevant.  Whereas structure-based drug design has contributed to the development of some serine protease inhibitors, traditional computational methods have not been able to fully describe structure-activity relationships (SAR).  Here, we study the SAR for a no. of serine proteases by using a method that calcs. the thermodn. properties (enthalpy and entropy) of the water that solvates the active site.  We show that the displacement of water from specific subpockets (such as S1-4 and the ester binding pocket) of the active site by the ligand can govern potency, esp. for cases in which small chem. changes (i.e., a Me group or halogen) result in a substantial increase in potency.  Furthermore, we describe how relative binding free energies can be estd. by combining the water displacement energy with complementary terms from an implicit solvent mol. mechanics description binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrr_5CTLjkC3LVg90H21EOLACvtfcHk0lhwMJEY-0qIZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmsFGjs7c%253D&md5=d432b1e0f07de0702e2320284e131a9c</span></div><a href="/servlet/linkout?suffix=cit149&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201000533&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201000533%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DSalam%26aufirst%3DN.%2BK.%26aulast%3DShelley%26aufirst%3DJ.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DContribution%2520of%2520explicit%2520solvent%2520effects%2520to%2520the%2520binding%2520affinity%2520of%2520small-molecule%2520inhibitors%2520in%2520blood%2520coagulation%2520factor%2520serine%2520proteases%26jtitle%3DChemMedChem%26date%3D2011%26volume%3D6%26spage%3D1049%26epage%3D1066%26doi%3D10.1002%2Fcmdc.201000533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref150"><div class="reference"><strong class="refLabel"><a href="#ref150" class="rightTabRefNumLink">150</a></strong><div class="NLM_citation" id="rightTab-cit150"><span><span class="NLM_contrib-group">Zheng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, J.</span><span> </span><span class="NLM_article-title">Water PMF for predicting the properties of water molecules in protein binding site</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">583</span><span class="NLM_x">–</span> <span class="NLM_lpage">592</span><span class="refDoi"> DOI: 10.1002/jcc.23170</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fjcc.23170" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=23114863" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1WjsrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2013&pages=583-592&author=M.+Zhengauthor=Y.+Liauthor=B.+Xiongauthor=H.+Jiangauthor=J.+Shen&title=Water+PMF+for+predicting+the+properties+of+water+molecules+in+protein+binding+site&doi=10.1002%2Fjcc.23170"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit150R"><div class="casContent"><span class="casTitleNuber">150</span><div class="casTitle"><span class="NLM_cas:atitle">Water PMF for predicting the properties of water molecules in protein binding site</span></div><div class="casAuthors">Zheng, Mingyue; Li, Yanlian; Xiong, Bing; Jiang, Hualiang; Shen, Jingkang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">583-592</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Water is an important component in living systems and deserves better understanding in chem. and biol.  However, due to the difficulty of investigating the water functions in protein structures, it is usually ignored in computational modeling, esp. in the field of computer-aided drug design.  Here, using the potential of mean forces (PMFs) approach, we constructed a water PMF (wPMF) based on 3946 non-redundant high resoln. crystal structures.  The extd. wPMF potential was first used to investigate the structure pattern of water and analyze the residue hydrophilicity.  Then, the relationship between wPMF score and the B factor value of crystal waters was studied.  It was found that wPMF agrees well with some previously reported exptl. observations.  In addn., the wPMF score was also tested in parallel with 3D-RISM to measure the ability of retrieving exptl. obsd. waters, and showed comparable performance but with much less computational cost.  In the end, we proposed a grid-based clustering scheme together with a distance weighted wPMF score to further extend wPMF to predict the potential hydration sites of protein structure.  From the test, this approach can predict the hydration site at the accuracy about 80% when the calcd. score lower than -4.0.  It also allows the assessment of whether or not a given water mol. should be targeted for displacement in ligand design.  Overall, the wPMF presented here provides an optional soln. to many water related computational modeling problems, some of which can be highly valuable as part of a rational drug design strategy. © 2012 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvOEcbTf2Xj7Vg90H21EOLACvtfcHk0lhwMJEY-0qIZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1WjsrjL&md5=298ec577c4a193a3c5d7f0bedb7859f9</span></div><a href="/servlet/linkout?suffix=cit150&amp;dbid=16384&amp;doi=10.1002%2Fjcc.23170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.23170%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DXiong%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DShen%26aufirst%3DJ.%26atitle%3DWater%2520PMF%2520for%2520predicting%2520the%2520properties%2520of%2520water%2520molecules%2520in%2520protein%2520binding%2520site%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2013%26volume%3D34%26spage%3D583%26epage%3D592%26doi%3D10.1002%2Fjcc.23170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref151"><div class="reference"><strong class="refLabel"><a href="#ref151" class="rightTabRefNumLink">151</a></strong><div class="NLM_citation" id="rightTab-cit151"><span><span class="NLM_contrib-group">Forli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">A force field with discrete displaceable waters and desolvation entropy for hydrated ligand docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">623</span><span class="NLM_x">–</span> <span class="NLM_lpage">638</span><span class="refDoi"> DOI: 10.1021/jm2005145</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2005145" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Smur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=623-638&author=S.+Forliauthor=A.+J.+Olson&title=A+force+field+with+discrete+displaceable+waters+and+desolvation+entropy+for+hydrated+ligand+docking&doi=10.1021%2Fjm2005145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit151R"><div class="casContent"><span class="casTitleNuber">151</span><div class="casTitle"><span class="NLM_cas:atitle">A Force Field with Discrete Displaceable Waters and Desolvation Entropy for Hydrated Ligand Docking</span></div><div class="casAuthors">Forli, Stefano; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">623-638</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In modeling ligand-protein interactions, the representation and role of water are of great importance.  We introduce a force field and hydration docking method that enables the automated prediction of waters mediating the binding of ligands with target proteins.  The method presumes no prior knowledge of the apo or holo protein hydration state and is potentially useful in the process of structure-based drug discovery.  The hydration force field accounts for the entropic and enthalpic contributions of discrete waters to ligand binding, improving energy estn. accuracy and docking performance.  The force field has been calibrated and validated on a total of 417 complexes (197 training set; 220 test set), then tested in cross-docking expts., for a total of 1649 ligand-protein complexes evaluated.  The method is computationally efficient and was used to model up to 35 waters during docking.  The method was implemented and tested using unaltered AutoDock4 with new force field tables.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoARuEGWgy1RbVg90H21EOLACvtfcHk0lhwMJEY-0qIZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Smur3O&md5=dd45a13de7c8433b8eaf168fadfcd1e7</span></div><a href="/servlet/linkout?suffix=cit151&amp;dbid=16384&amp;doi=10.1021%2Fjm2005145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2005145%26sid%3Dliteratum%253Aachs%26aulast%3DForli%26aufirst%3DS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DA%2520force%2520field%2520with%2520discrete%2520displaceable%2520waters%2520and%2520desolvation%2520entropy%2520for%2520hydrated%2520ligand%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D623%26epage%3D638%26doi%3D10.1021%2Fjm2005145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref152"><div class="reference"><strong class="refLabel"><a href="#ref152" class="rightTabRefNumLink">152</a></strong><div class="NLM_citation" id="rightTab-cit152"><span><span class="NLM_contrib-group">Pottel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therrien, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gleason, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moitessier, N.</span><span> </span><span class="NLM_article-title">Docking ligands into flexible and solvated macromolecules. 6. Development and application to the docking of HDACs and other zinc metalloenzymes inhibitors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">254</span><span class="NLM_x">–</span> <span class="NLM_lpage">265</span><span class="refDoi"> DOI: 10.1021/ci400550m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400550m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFyms7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=254-265&author=J.+Pottelauthor=E.+Therrienauthor=J.+L.+Gleasonauthor=N.+Moitessier&title=Docking+ligands+into+flexible+and+solvated+macromolecules.+6.+Development+and+application+to+the+docking+of+HDACs+and+other+zinc+metalloenzymes+inhibitors&doi=10.1021%2Fci400550m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit152R"><div class="casContent"><span class="casTitleNuber">152</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Ligands into Flexible and Solvated Macromolecules. 6. Development and Application to the Docking of HDACs and other Zinc Metalloenzymes Inhibitors</span></div><div class="casAuthors">Pottel, Joshua; Therrien, Eric; Gleason, James L.; Moitessier, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">254-265</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Metalloenzymes are ubiquitous proteins which feature one or more metal ions either directly involved in the enzymic activity and/or structural properties (i.e., zinc fingers).  Several members of this class take advantage of the Lewis acidic properties of zinc ions to carry out their various catalytic transformations including isomerization or amide cleavage.  These enzymes have been validated as drug targets for a no. of diseases including cancer; however, despite their pharmaceutical relevance and the availability of crystal structures, structure-based drug design methods have been poorly and indirectly parametrized for these classes of enzymes.  More specifically, the metal coordination component and proton transfers of the process of drugs binding to metalloenzymes have been inadequately modeled by current docking programs, if at all.  In addn., several known issues, such as coordination geometry, at. charge variability, and a potential proton transfer from small mols. to a neighboring basic residue, have often been ignored.  We report herein the development of specific functions and parameters to account for zinc-drug coordination focusing on the above-listed phenomena and their impact on docking to zinc metalloenzymes.  These atom-type-dependent but at. charge-independent functions implemented into Fitted 3.1 enable the simulation of drug binding to metalloenzymes, considering an acid-base reaction with a neighboring residue when necessary with good accuracy.  The ability of docking to predict angiotensin converting enzyme (ACE) inhibitory dipeptide sequences was assessed using AutoDock Vina.  All potential dipeptides and phospho-dipeptides were docked and scored.  Peptide intestinal stability was assessed using a prediction amino acid clustering model.  Selected dipeptides, having AutoDock Vina scores ≤ -8.1 and predicted to be 'stable' intestinally, were characterized, using LIGPLOT and for ACE-inhibitory potency.  Two newly identified ACE-inhibitory dipeptides, Asp-Trp and Trp-Pro, having Vina scores of -8.3 and -8.6 gave IC50 values of 258 ± 4.23 and 217 ± 15.7 μM, resp.  LIGPLOT anal. indicated no zinc interaction for these dipeptides.  Phospho-dipeptides were predicted to have a good affinity for ACE.  However, the exptl. detd. IC50 results did not correlate since, for example, Trp-pThr and Pro-pTyr, having Vina scores of -8.5 and -8.1, resp., displayed IC50 values of >500 μM.  While docking allowed identification of new ACE inhibitory dipeptides, it may not be a fully reliable predictive tool in all cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpP7uZp-NLQXrVg90H21EOLACvtfcHk0li0LpTcwt9Oog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFyms7fK&md5=6ce5b06aaede542219621d7a5df3b23e</span></div><a href="/servlet/linkout?suffix=cit152&amp;dbid=16384&amp;doi=10.1021%2Fci400550m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400550m%26sid%3Dliteratum%253Aachs%26aulast%3DPottel%26aufirst%3DJ.%26aulast%3DTherrien%26aufirst%3DE.%26aulast%3DGleason%26aufirst%3DJ.%2BL.%26aulast%3DMoitessier%26aufirst%3DN.%26atitle%3DDocking%2520ligands%2520into%2520flexible%2520and%2520solvated%2520macromolecules.%25206.%2520Development%2520and%2520application%2520to%2520the%2520docking%2520of%2520HDACs%2520and%2520other%2520zinc%2520metalloenzymes%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D254%26epage%3D265%26doi%3D10.1021%2Fci400550m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref153"><div class="reference"><strong class="refLabel"><a href="#ref153" class="rightTabRefNumLink">153</a></strong><div class="NLM_citation" id="rightTab-cit153"><span><span class="NLM_contrib-group">Adeniyi, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ajibade, P. A.</span><span> </span><span class="NLM_article-title">Comparing the suitability of autodock, gold and glide for the docking and predicting the possible targets of Ru(II)-based complexes as anticancer agents</span> <span class="citation_source-journal">Molecules</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">3760</span><span class="NLM_x">–</span> <span class="NLM_lpage">3778</span><span class="refDoi"> DOI: 10.3390/molecules18043760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.3390%2Fmolecules18043760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=23529035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlslCmsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=3760-3778&author=A.+A.+Adeniyiauthor=P.+A.+Ajibade&title=Comparing+the+suitability+of+autodock%2C+gold+and+glide+for+the+docking+and+predicting+the+possible+targets+of+Ru%28II%29-based+complexes+as+anticancer+agents&doi=10.3390%2Fmolecules18043760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit153R"><div class="casContent"><span class="casTitleNuber">153</span><div class="casTitle"><span class="NLM_cas:atitle">Comparing the suitability of Autodock, Gold and Glide for the docking and predicting the possible targets of Ru(II)-based complexes as anticancer agents</span></div><div class="casAuthors">Adeniyi, Adebayo A.; Ajibade, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3760-3778</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">In cancer chemotherapy, metal-based complexes have been recognized as the most promising means of inhibiting cancer growth due to the successful application of cis-platin and its derivs. above many of the existing org. anticancer agents.  The limitations in their rational design can be traced to the complexity of the mechanism of their operations, lack of proper knowledge of their targets and lack of force fields in docking packages to appropriately define the metal center of the organometallic complexes.  In this paper, some of the promising anticancer complexes of Ru(II) such as the rapta-based complexes formulated as [Ru(η6-p-cymene)L2(pta)] and those with unusual ligands are considered.  CatB and kinases which have been exptl. confirmed as possible targets of the complexes are also predicted by the three methods as one of the most targeted receptors while TopII and HDAC7 are predicted by two and one of the methods as best targets.  The interesting features of the binding of the complexes show that some of the complexes preferentially target specific macromols. than the others, which is an indication of their specificity and possibility of their therapeutic combination without severe side effects that may come from competition for the same target.  Also, introduction of unusual ligands is found to significantly improve the activities of most of the complexes studied.  Strong correlations are obsd. for the predicted binding sites and the orientation of the complexes within the binding site by the three methods of docking.  However there are disparities in the ranking of the complexes by the three method of docking, esp. that of Glide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkdpV-_FnD1rVg90H21EOLACvtfcHk0li0LpTcwt9Oog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlslCmsrw%253D&md5=750e7ed12bc2d8827e4b00fba9ddd5c5</span></div><a href="/servlet/linkout?suffix=cit153&amp;dbid=16384&amp;doi=10.3390%2Fmolecules18043760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules18043760%26sid%3Dliteratum%253Aachs%26aulast%3DAdeniyi%26aufirst%3DA.%2BA.%26aulast%3DAjibade%26aufirst%3DP.%2BA.%26atitle%3DComparing%2520the%2520suitability%2520of%2520autodock%252C%2520gold%2520and%2520glide%2520for%2520the%2520docking%2520and%2520predicting%2520the%2520possible%2520targets%2520of%2520Ru%2528II%2529-based%2520complexes%2520as%2520anticancer%2520agents%26jtitle%3DMolecules%26date%3D2013%26volume%3D18%26spage%3D3760%26epage%3D3778%26doi%3D10.3390%2Fmolecules18043760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref154"><div class="reference"><strong class="refLabel"><a href="#ref154" class="rightTabRefNumLink">154</a></strong><div class="NLM_citation" id="rightTab-cit154"><span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">Assignment of polar states for protein amino acid residues using an interaction cluster decomposition algorithm and its application to high resolution protein structure modeling</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">824</span><span class="NLM_x">–</span> <span class="NLM_lpage">837</span><span class="refDoi"> DOI: 10.1002/prot.21125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fprot.21125" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2007&pages=824-837&author=X.+Liauthor=M.+P.+Jacobsonauthor=K.+Zhuauthor=S.+Zhaoauthor=R.+A.+Friesner&title=Assignment+of+polar+states+for+protein+amino+acid+residues+using+an+interaction+cluster+decomposition+algorithm+and+its+application+to+high+resolution+protein+structure+modeling&doi=10.1002%2Fprot.21125"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit154&amp;dbid=16384&amp;doi=10.1002%2Fprot.21125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21125%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DAssignment%2520of%2520polar%2520states%2520for%2520protein%2520amino%2520acid%2520residues%2520using%2520an%2520interaction%2520cluster%2520decomposition%2520algorithm%2520and%2520its%2520application%2520to%2520high%2520resolution%2520protein%2520structure%2520modeling%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2007%26volume%3D66%26spage%3D824%26epage%3D837%26doi%3D10.1002%2Fprot.21125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref155"><div class="reference"><strong class="refLabel"><a href="#ref155" class="rightTabRefNumLink">155</a></strong><div class="NLM_citation" id="rightTab-cit155"><span><span class="NLM_contrib-group">Kruger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grunzke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herres-Pawlis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffmann, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de la Garza, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kohlbacher, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagel, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gesing, S.</span><span> </span><span class="NLM_article-title">Performance studies on distributed virtual screening</span> <span class="citation_source-journal">BioMed Res. Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">2014</span><span class="NLM_x">, </span> <span class="NLM_fpage">624024</span><span class="refDoi"> DOI: 10.1155/2014/624024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1155%2F2014%2F624024" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2014&publication_year=2014&pages=624024&author=J.+Krugerauthor=R.+Grunzkeauthor=S.+Herres-Pawlisauthor=A.+Hoffmannauthor=L.+de+la+Garzaauthor=O.+Kohlbacherauthor=W.+E.+Nagelauthor=S.+Gesing&title=Performance+studies+on+distributed+virtual+screening&doi=10.1155%2F2014%2F624024"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit155&amp;dbid=16384&amp;doi=10.1155%2F2014%2F624024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2014%252F624024%26sid%3Dliteratum%253Aachs%26aulast%3DKruger%26aufirst%3DJ.%26aulast%3DGrunzke%26aufirst%3DR.%26aulast%3DHerres-Pawlis%26aufirst%3DS.%26aulast%3DHoffmann%26aufirst%3DA.%26aulast%3Dde%2Bla%2BGarza%26aufirst%3DL.%26aulast%3DKohlbacher%26aufirst%3DO.%26aulast%3DNagel%26aufirst%3DW.%2BE.%26aulast%3DGesing%26aufirst%3DS.%26atitle%3DPerformance%2520studies%2520on%2520distributed%2520virtual%2520screening%26jtitle%3DBioMed%2520Res.%2520Int.%26date%3D2014%26volume%3D2014%26spage%3D624024%26doi%3D10.1155%2F2014%2F624024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref156"><div class="reference"><strong class="refLabel"><a href="#ref156" class="rightTabRefNumLink">156</a></strong><div class="NLM_citation" id="rightTab-cit156"><span><span class="NLM_contrib-group">Trott, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">455</span><span class="NLM_x">–</span> <span class="NLM_lpage">461</span><span class="refDoi"> DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-461&author=O.+Trottauthor=A.+J.+Olson&title=AutoDock+Vina%3A+improving+the+speed+and+accuracy+of+docking+with+a+new+scoring+function%2C+efficient+optimization%2C+and+multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit156R"><div class="casContent"><span class="casTitleNuber">156</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0li0LpTcwt9Oog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit156&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock%2520Vina%253A%2520improving%2520the%2520speed%2520and%2520accuracy%2520of%2520docking%2520with%2520a%2520new%2520scoring%2520function%252C%2520efficient%2520optimization%252C%2520and%2520multithreading%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D461%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref157"><div class="reference"><strong class="refLabel"><a href="#ref157" class="rightTabRefNumLink">157</a></strong><div class="NLM_citation" id="rightTab-cit157"><span><span class="NLM_contrib-group">Kasam, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salzemann, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botha, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dacosta, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Degliesposti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isea, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maass, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenyon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rastelli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hofmann-Apitius, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breton, V.</span><span> </span><span class="NLM_article-title">WISDOM-II: screening against multiple targets implicated in malaria using computational grid infrastructures</span> <span class="citation_source-journal">Malar. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">88</span><span class="refDoi"> DOI: 10.1186/1475-2875-8-88</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1186%2F1475-2875-8-88" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=19409081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BD1MzpvVeitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=88&author=V.+Kasamauthor=J.+Salzemannauthor=M.+Bothaauthor=A.+Dacostaauthor=G.+Degliespostiauthor=R.+Iseaauthor=D.+Kimauthor=A.+Maassauthor=C.+Kenyonauthor=G.+Rastelliauthor=M.+Hofmann-Apitiusauthor=V.+Breton&title=WISDOM-II%3A+screening+against+multiple+targets+implicated+in+malaria+using+computational+grid+infrastructures&doi=10.1186%2F1475-2875-8-88"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit157R"><div class="casContent"><span class="casTitleNuber">157</span><div class="casTitle"><span class="NLM_cas:atitle">WISDOM-II: screening against multiple targets implicated in malaria using computational grid infrastructures</span></div><div class="casAuthors">Kasam Vinod; Salzemann Jean; Botha Marli; Dacosta Ana; Degliesposti Gianluca; Isea Raul; Kim Doman; Maass Astrid; Kenyon Colin; Rastelli Giulio; Hofmann-Apitius Martin; Breton Vincent</div><div class="citationInfo"><span class="NLM_cas:title">Malaria journal</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">88</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Despite continuous efforts of the international community to reduce the impact of malaria on developing countries, no significant progress has been made in the recent years and the discovery of new drugs is more than ever needed.  Out of the many proteins involved in the metabolic activities of the Plasmodium parasite, some are promising targets to carry out rational drug discovery.  MOTIVATION:  Recent years have witnessed the emergence of grids, which are highly distributed computing infrastructures particularly well fitted for embarrassingly parallel computations like docking.  In 2005, a first attempt at using grids for large-scale virtual screening focused on plasmepsins and ended up in the identification of previously unknown scaffolds, which were confirmed in vitro to be active plasmepsin inhibitors.  Following this success, a second deployment took place in the fall of 2006 focussing on one well known target, dihydrofolate reductase (DHFR), and on a new promising one, glutathione-S-transferase.  METHODS:  In silico drug design, especially vHTS is a widely and well-accepted technology in lead identification and lead optimization.  This approach, therefore builds, upon the progress made in computational chemistry to achieve more accurate in silico docking and in information technology to design and operate large scale grid infrastructures.  RESULTS:  On the computational side, a sustained infrastructure has been developed: docking at large scale, using different strategies in result analysis, storing of the results on the fly into MySQL databases and application of molecular dynamics refinement are MM-PBSA and MM-GBSA rescoring.  The modeling results obtained are very promising.  Based on the modeling results, In vitro results are underway for all the targets against which screening is performed.  CONCLUSION:  The current paper describes the rational drug discovery activity at large scale, especially molecular docking using FlexX software on computational grids in finding hits against three different targets (PfGST, PfDHFR, PvDHFR (wild type and mutant forms) implicated in malaria.  Grid-enabled virtual screening approach is proposed to produce focus compound libraries for other biological targets relevant to fight the infectious diseases of the developing world.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTs3wyAqQCw3UEgZCMqtRBifW6udTcc2eZpe7IaAbH8F7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MzpvVeitw%253D%253D&md5=b3d674ddc73d104e96924752d34a499b</span></div><a href="/servlet/linkout?suffix=cit157&amp;dbid=16384&amp;doi=10.1186%2F1475-2875-8-88&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1475-2875-8-88%26sid%3Dliteratum%253Aachs%26aulast%3DKasam%26aufirst%3DV.%26aulast%3DSalzemann%26aufirst%3DJ.%26aulast%3DBotha%26aufirst%3DM.%26aulast%3DDacosta%26aufirst%3DA.%26aulast%3DDegliesposti%26aufirst%3DG.%26aulast%3DIsea%26aufirst%3DR.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DMaass%26aufirst%3DA.%26aulast%3DKenyon%26aufirst%3DC.%26aulast%3DRastelli%26aufirst%3DG.%26aulast%3DHofmann-Apitius%26aufirst%3DM.%26aulast%3DBreton%26aufirst%3DV.%26atitle%3DWISDOM-II%253A%2520screening%2520against%2520multiple%2520targets%2520implicated%2520in%2520malaria%2520using%2520computational%2520grid%2520infrastructures%26jtitle%3DMalar.%2520J.%26date%3D2009%26volume%3D8%26spage%3D88%26doi%3D10.1186%2F1475-2875-8-88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref158"><div class="reference"><strong class="refLabel"><a href="#ref158" class="rightTabRefNumLink">158</a></strong><div class="NLM_citation" id="rightTab-cit158"><span>CombiGlide; <span class="NLM_publisher-name">Schrodinger</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=CombiGlide%3B+Schrodinger%3A+New+York%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit158&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DCombiGlide%26pub%3DSchrodinger%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref159"><div class="reference"><strong class="refLabel"><a href="#ref159" class="rightTabRefNumLink">159</a></strong><div class="NLM_citation" id="rightTab-cit159"><span><span class="NLM_contrib-group">Lazaridis, T.</span><span> </span><span class="NLM_article-title">Inhomogeneous fluid approach to solvation thermodynamics. 2. Applications to simple fluids</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">3542</span><span class="NLM_x">–</span> <span class="NLM_lpage">3550</span><span class="refDoi"> DOI: 10.1021/jp972358w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp972358w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADyaK1cXivVOltrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=1998&pages=3542-3550&author=T.+Lazaridis&title=Inhomogeneous+fluid+approach+to+solvation+thermodynamics.+2.+Applications+to+simple+fluids&doi=10.1021%2Fjp972358w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit159R"><div class="casContent"><span class="casTitleNuber">159</span><div class="casTitle"><span class="NLM_cas:atitle">Inhomogeneous Fluid Approach to Solvation Thermodynamics. 2. Applications to Simple Fluids</span></div><div class="casAuthors">Lazaridis, Themis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3542-3550</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1089-5647</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the previous paper expressions for the partial molar energy and entropy at infinite diln. have been derived based on the inhomogeneous forms of the energy equation and the correlation expansion for the entropy.  These expressions are here applied to a series of solutes of varying size in dense hard-sphere and Lennard-Jones solvents, some of which serve as ref. systems for comparison with water.  Numerical results are obtained under the assumption that the inhomogeneous solvent-solvent pair correlation function in the mixt. is equal to the bulk solvent radial distribution function (Kirkwood superposition approxn.).  The correlation functions required are obtained by both integral equation theory (Percus-Yevick approxn.) and Monte Carlo simulations.  The thermodn. results are compared with equation of state, integral equation, and free energy simulation results for the same systems.  For hard-sphere systems the excess entropies are in good agreement with equation-of-state results but in many Lennard-Jones systems the calcd. partial molar energies and entropies are lower than the expected values.  This is attributable to overestimation of the structure of the bulk triplet correlation function by the superposition approxn.  The decompn. of the chem. potential shows that similar solvation free energies can have entirely different phys. origins.  Specifically, in solvents of high cohesive energy d. the chem. potential is dominated by the breakup of solvent-solvent interactions locally around the solute.  In solvents of low cohesive energy d. it is dominated by the pressure-vol. term.  Increase in solvent-solvent interaction strength leads to increase in the chem. potential of the solute due to the higher solvent reorganization energy, which is insufficiently compensated by an increase in solvent reorganization entropy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdwkuoqtHxPLVg90H21EOLACvtfcHk0ljQ5loM6CD9vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXivVOltrw%253D&md5=63ce8d44f2f6c8d50f81303dba75924f</span></div><a href="/servlet/linkout?suffix=cit159&amp;dbid=16384&amp;doi=10.1021%2Fjp972358w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp972358w%26sid%3Dliteratum%253Aachs%26aulast%3DLazaridis%26aufirst%3DT.%26atitle%3DInhomogeneous%2520fluid%2520approach%2520to%2520solvation%2520thermodynamics.%25202.%2520Applications%2520to%2520simple%2520fluids%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1998%26volume%3D102%26spage%3D3542%26epage%3D3550%26doi%3D10.1021%2Fjp972358w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref160"><div class="reference"><strong class="refLabel"><a href="#ref160" class="rightTabRefNumLink">160</a></strong><div class="NLM_citation" id="rightTab-cit160"><span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tubert-Brohman, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">Docking performance of the glide program as evaluated on the Astex and DUD datasets: a complete set of glide SP results and selected results for a new scoring function integrating WaterMap and glide</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">787</span><span class="NLM_x">–</span> <span class="NLM_lpage">799</span><span class="refDoi"> DOI: 10.1007/s10822-012-9575-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs10822-012-9575-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=22576241" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVelu7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=787-799&author=M.+P.+Repaskyauthor=R.+B.+Murphyauthor=J.+L.+Banksauthor=J.+R.+Greenwoodauthor=I.+Tubert-Brohmanauthor=S.+Bhatauthor=R.+A.+Friesner&title=Docking+performance+of+the+glide+program+as+evaluated+on+the+Astex+and+DUD+datasets%3A+a+complete+set+of+glide+SP+results+and+selected+results+for+a+new+scoring+function+integrating+WaterMap+and+glide&doi=10.1007%2Fs10822-012-9575-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit160R"><div class="casContent"><span class="casTitleNuber">160</span><div class="casTitle"><span class="NLM_cas:atitle">Docking performance of the glide program as evaluated on the Astex and DUD datasets: a complete set of glide SP results and selected results for a new scoring function integrating WaterMap and glide</span></div><div class="casAuthors">Repasky, Matthew P.; Murphy, Robert B.; Banks, Jay L.; Greenwood, Jeremy R.; Tubert-Brohman, Ivan; Bhat, Sathesh; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">787-799</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Glide SP mode enrichment results for two prepns. of the DUD dataset and native ligand docking RMSDs for two prepns. of the Astex dataset are presented.  Following a best-practices prepn. scheme, an av. RMSD of 1.140 Å for native ligand docking with Glide SP is computed.  Following the same best-practices prepn. scheme for the DUD dataset an av. area under the ROC curve (AUC) of 0.80 and av. early enrichment via the ROC (0.1 %) metric of 0.12 were obsd. 74 and 56 % of the 39 best-practices prepd. targets showed AUC over 0.7 and 0.8, resp.  Av. AUC was greater than 0.7 for all best-practices protein families demonstrating consistent enrichment performance across a broad range of proteins and ligand chemotypes.  In both Astex and DUD datasets, docking performance is significantly improved employing a best-practices prepn. scheme over using minimally-prepd. structures from the PDB.  Enrichment results for WScore, a new scoring function and sampling methodol. integrating WaterMap and Glide, are presented for four DUD targets, hivrt, hsp90, cdk2, and fxa.  WScore performance in early enrichment is consistently strong and all systems examd. show AUC > 0.9 and superior early enrichment to DUD best-practices Glide SP results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHCCMQs2lTrLVg90H21EOLACvtfcHk0ljVgvGVM4dvwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVelu7nI&md5=9eb2c6feb24177c81bb5c06b947893fc</span></div><a href="/servlet/linkout?suffix=cit160&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9575-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9575-9%26sid%3Dliteratum%253Aachs%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DTubert-Brohman%26aufirst%3DI.%26aulast%3DBhat%26aufirst%3DS.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DDocking%2520performance%2520of%2520the%2520glide%2520program%2520as%2520evaluated%2520on%2520the%2520Astex%2520and%2520DUD%2520datasets%253A%2520a%2520complete%2520set%2520of%2520glide%2520SP%2520results%2520and%2520selected%2520results%2520for%2520a%2520new%2520scoring%2520function%2520integrating%2520WaterMap%2520and%2520glide%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D787%26epage%3D799%26doi%3D10.1007%2Fs10822-012-9575-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref161"><div class="reference"><strong class="refLabel"><a href="#ref161" class="rightTabRefNumLink">161</a></strong><div class="NLM_citation" id="rightTab-cit161"><span><span class="NLM_contrib-group">Nguyen, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">Grid inhomogeneous solvation theory: hydration structure and thermodynamics of the miniature receptor cucurbit[7]uril</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">137</span><span class="NLM_x">, </span> <span class="NLM_fpage">044101</span><span class="refDoi"> DOI: 10.1063/1.4733951</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1063%2F1.4733951" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=22852591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVOgurzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2012&pages=044101&author=C.+N.+Nguyenauthor=T.+K.+Youngauthor=M.+K.+Gilson&title=Grid+inhomogeneous+solvation+theory%3A+hydration+structure+and+thermodynamics+of+the+miniature+receptor+cucurbit%5B7%5Duril&doi=10.1063%2F1.4733951"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit161R"><div class="casContent"><span class="casTitleNuber">161</span><div class="casTitle"><span class="NLM_cas:atitle">Grid inhomogeneous solvation theory: Hydration structure and thermodynamics of the miniature receptor cucurbit[7]uril</span></div><div class="casAuthors">Nguyen, Crystal N.; Kurtzman Young, Tom; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">044101/1-044101/17</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The displacement of perturbed water upon binding is believed to play a crit. role in the thermodn. of biomol. recognition, but it is nontrivial to unambiguously define and answer questions about this process.  We address this issue by introducing grid inhomogeneous solvation theory (GIST), which discretizes the equations of inhomogeneous solvation theory (IST) onto a three-dimensional grid situated in the region of interest around a solute mol. or complex.  Snapshots from explicit solvent simulations are used to est. localized solvation entropies, energies, and free energies assocd. with the grid boxes, or voxels, and properly summing these thermodn. quantities over voxels yields information about hydration thermodn.  GIST thus provides a smoothly varying representation of water properties as a function of position, rather than focusing on hydration sites where solvent is present at high d.  It therefore accounts for full or partial displacement of water from sites that are highly occupied by water, as well as for partly occupied and water-depleted regions around the solute.  GIST can also provide a well-defined est. of the solvation free energy and therefore enables a rigorous end-states anal. of binding.  For example, one may not only use a first GIST calcn. to project the thermodn. consequences of displacing water from the surface of a receptor by a ligand, but also account, in a second GIST calcn., for the thermodn. of subsequent solvent reorganization around the bound complex.  In the present study, a first GIST anal. of the mol. host cucurbit7uril is found to yield a rich picture of hydration structure and thermodn. in and around this miniature receptor.  One of the most striking results is the observation of a toroidal region of high water d. at the center of the host's nonpolar cavity.  Despite its high d., the water in this toroidal region is disfavored energetically and entropically, and hence may contribute to the known ability of this small receptor to bind guest mols. with unusually high affinities.  Interestingly, the toroidal region of high water d. persists even when all partial charges of the receptor are set to zero.  Thus, localized regions of high solvent d. can be generated in a binding site without strong, attractive solute-solvent interactions.  (c) 2012 American Institute of Physics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkKZ05RaZJbbVg90H21EOLACvtfcHk0ljVgvGVM4dvwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVOgurzJ&md5=d8eceb021e903950784786a656fdfb40</span></div><a href="/servlet/linkout?suffix=cit161&amp;dbid=16384&amp;doi=10.1063%2F1.4733951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.4733951%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DC.%2BN.%26aulast%3DYoung%26aufirst%3DT.%2BK.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DGrid%2520inhomogeneous%2520solvation%2520theory%253A%2520hydration%2520structure%2520and%2520thermodynamics%2520of%2520the%2520miniature%2520receptor%2520cucurbit%255B7%255Duril%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2012%26volume%3D137%26spage%3D044101%26doi%3D10.1063%2F1.4733951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref162"><div class="reference"><strong class="refLabel"><a href="#ref162" class="rightTabRefNumLink">162</a></strong><div class="NLM_citation" id="rightTab-cit162"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">ZINC--A free database of commercially available compounds for virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span><span class="refDoi"> DOI: 10.1021/ci049714+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049714%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=177-182&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=ZINC%2D%2DA+free+database+of+commercially+available+compounds+for+virtual+screening&doi=10.1021%2Fci049714%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit162R"><div class="casContent"><span class="casTitleNuber">162</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC--a free database of commercially available compounds for virtual screening</span></div><div class="casAuthors">Irwin John J; Shoichet Brian K</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-82</span>
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    </div><div class="casAbstract">A critical barrier to entry into structure-based virtual screening is the lack of a suitable, easy to access database of purchasable compounds.  We have therefore prepared a library of 727,842 molecules, each with 3D structure, using catalogs of compounds from vendors (the size of this library continues to grow).  The molecules have been assigned biologically relevant protonation states and are annotated with properties such as molecular weight, calculated LogP, and number of rotatable bonds.  Each molecule in the library contains vendor and purchasing information and is ready for docking using a number of popular docking programs.  Within certain limits, the molecules are prepared in multiple protonation states and multiple tautomeric forms.  In one format, multiple conformations are available for the molecules.  This database is available for free download (http://zinc.docking.org) in several common file formats including SMILES, mol2, 3D SDF, and DOCK flexibase format.  A Web-based query tool incorporating a molecular drawing interface enables the database to be searched and browsed and subsets to be created.  Users can process their own molecules by uploading them to a server.  Our hope is that this database will bring virtual screening libraries to a wide community of structural biologists and medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQVi_-excgTX-_Q9URG6Dx8fW6udTcc2eaOn0X2bFGBdbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D&md5=9f2e3ec06d53276884fc17b9aad47e61</span></div><a href="/servlet/linkout?suffix=cit162&amp;dbid=16384&amp;doi=10.1021%2Fci049714%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049714%252B%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DZINC--A%2520free%2520database%2520of%2520commercially%2520available%2520compounds%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D177%26epage%3D182%26doi%3D10.1021%2Fci049714%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref163"><div class="reference"><strong class="refLabel"><a href="#ref163" class="rightTabRefNumLink">163</a></strong><div class="NLM_citation" id="rightTab-cit163"><span><span class="NLM_contrib-group">Sterling, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span> </span><span class="NLM_article-title">ZINC 15 - Ligand discovery for everyone</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">2324</span><span class="NLM_x">–</span> <span class="NLM_lpage">2337</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00559</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00559" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1OhurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2324-2337&author=T.+Sterlingauthor=J.+J.+Irwin&title=ZINC+15+-+Ligand+discovery+for+everyone&doi=10.1021%2Facs.jcim.5b00559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit163R"><div class="casContent"><span class="casTitleNuber">163</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC 15 - Ligand Discovery for Everyone</span></div><div class="casAuthors">Sterling, Teague; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2324-2337</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many questions about the biol. activity and availability of small mols. remain inaccessible to investigators who could most benefit from their answers.  To narrow the gap between chemoinformatics and biol., we have developed a suite of ligand annotation, purchasability, target, and biol. assocn. tools, incorporated into ZINC and meant for investigators who are not computer specialists.  The new version contains over 120 million purchasable "drug-like" compds. - effectively all org. mols. that are for sale - a quarter of which are available for immediate delivery.  ZINC connects purchasable compds. to high-value ones such as metabolites, drugs, natural products, and annotated compds. from the literature.  Compds. may be accessed by the genes for which they are annotated as well as the major and minor target classes to which those genes belong.  It offers new anal. tools that are easy for nonspecialists yet with few limitations for experts.  ZINC retains its original 3D roots - all mols. are available in biol. relevant, ready-to-dock formats.  ZINC is freely available at http://zinc15.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWfkDnal3D3bVg90H21EOLACvtfcHk0ljVgvGVM4dvwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1OhurbF&md5=e767c26c1d0aff7432c403c59a5d2b8b</span></div><a href="/servlet/linkout?suffix=cit163&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00559%26sid%3Dliteratum%253Aachs%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DZINC%252015%2520-%2520Ligand%2520discovery%2520for%2520everyone%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2324%26epage%3D2337%26doi%3D10.1021%2Facs.jcim.5b00559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref164"><div class="reference"><strong class="refLabel"><a href="#ref164" class="rightTabRefNumLink">164</a></strong><div class="NLM_citation" id="rightTab-cit164"><span><span class="NLM_contrib-group">ten Brink, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Exner, T. E.</span><span> </span><span class="NLM_article-title">pK(a) based protonation states and microspecies for protein-ligand docking</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">935</span><span class="NLM_x">–</span> <span class="NLM_lpage">942</span><span class="refDoi"> DOI: 10.1007/s10822-010-9385-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs10822-010-9385-x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=935-942&author=T.+ten+Brinkauthor=T.+E.+Exner&title=pK%28a%29+based+protonation+states+and+microspecies+for+protein-ligand+docking&doi=10.1007%2Fs10822-010-9385-x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit164&amp;dbid=16384&amp;doi=10.1007%2Fs10822-010-9385-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-010-9385-x%26sid%3Dliteratum%253Aachs%26aulast%3Dten%2BBrink%26aufirst%3DT.%26aulast%3DExner%26aufirst%3DT.%2BE.%26atitle%3DpK%2528a%2529%2520based%2520protonation%2520states%2520and%2520microspecies%2520for%2520protein-ligand%2520docking%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2010%26volume%3D24%26spage%3D935%26epage%3D942%26doi%3D10.1007%2Fs10822-010-9385-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref165"><div class="reference"><strong class="refLabel"><a href="#ref165" class="rightTabRefNumLink">165</a></strong><div class="NLM_citation" id="rightTab-cit165"><span><span class="NLM_contrib-group">Kim, M. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J. A.</span><span> </span><span class="NLM_article-title">Computation of pH-dependent binding free energies</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">43</span><span class="NLM_x">–</span> <span class="NLM_lpage">49</span><span class="refDoi"> DOI: 10.1002/bip.22702</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fbip.22702" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26202905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslSqsrzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2016&pages=43-49&author=M.+O.+Kimauthor=J.+A.+McCammon&title=Computation+of+pH-dependent+binding+free+energies&doi=10.1002%2Fbip.22702"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit165R"><div class="casContent"><span class="casTitleNuber">165</span><div class="casTitle"><span class="NLM_cas:atitle">Computation of pH-dependent binding free energies</span></div><div class="casAuthors">Kim, M. Olivia; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-49</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Protein-ligand binding accompanies changes in the surrounding electrostatic environments of the two binding partners and may lead to changes in protonation upon binding.  In cases where the complex formation results in a net transfer of protons, the binding process is pH-dependent.  However, conventional free energy computations or mol. docking protocols typically employ fixed protonation states for the titratable groups in both binding partners set a priori, which are identical for the free and bound states.  In this review, we draw attention to these important yet largely ignored binding-induced protonation changes in protein-ligand assocn. by outlining phys. origins and prevalence of the protonation changes upon binding.  Following a summary of various theor. methods for pKa prediction, we discuss the theor. framework to examine the pH dependence of protein-ligand binding processes.  © 2015 Wiley Periodicals, Inc.  Biopolymers 105: 43-49, 2015.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS2LkIfxVqrrVg90H21EOLACvtfcHk0lhDd5ijd5E2jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslSqsrzO&md5=8edc165134617308d9fef26620974012</span></div><a href="/servlet/linkout?suffix=cit165&amp;dbid=16384&amp;doi=10.1002%2Fbip.22702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.22702%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DM.%2BO.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DComputation%2520of%2520pH-dependent%2520binding%2520free%2520energies%26jtitle%3DBiopolymers%26date%3D2016%26volume%3D105%26spage%3D43%26epage%3D49%26doi%3D10.1002%2Fbip.22702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref166"><div class="reference"><strong class="refLabel"><a href="#ref166" class="rightTabRefNumLink">166</a></strong><div class="NLM_citation" id="rightTab-cit166"><span><span class="NLM_contrib-group">Hawkins, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholls, A.</span><span> </span><span class="NLM_article-title">Conformer generation with OMEGA: learning from the data set and the analysis of failures</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2919</span><span class="NLM_x">–</span> <span class="NLM_lpage">2936</span><span class="refDoi"> DOI: 10.1021/ci300314k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300314k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFekurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2919-2936&author=P.+C.+Hawkinsauthor=A.+Nicholls&title=Conformer+generation+with+OMEGA%3A+learning+from+the+data+set+and+the+analysis+of+failures&doi=10.1021%2Fci300314k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit166R"><div class="casContent"><span class="casTitleNuber">166</span><div class="casTitle"><span class="NLM_cas:atitle">Conformer Generation with OMEGA: Learning from the Data Set and the Analysis of Failures</span></div><div class="casAuthors">Hawkins, Paul C. D.; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2919-2936</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We recently published a high quality validation set for testing conformer generators, consisting of structures from both the PDB and the CSD and tested the performance of the authors' conformer generator, OMEGA, on these sets.  In the present publication, we focus on understanding the suitability of those data sets for validation and identifying and learning from OMEGA's failures.  We compare, for the first time we are aware of, the coverage of the applicable property spaces between the validation data sets we used and the parent compd. sets to det. if our data sets adequately sample these property spaces.  We also introduce the concept of torsion fingerprinting and compare this method of dissimilation to the more traditional graph-centric diversification methods we used in our previous publication.  To improve our ability to programmatically identify cases where the crystallog. conformation is not well reproduced computationally, we introduce a new metric to compare conformations, RMSTanimoto.  This new metric is used alongside those from our previous publication to efficiently identify reprodn. failures.  We find RMSTanimoto to be particularly effective in identifying failures for the smallest mols. in our data sets.  Anal. of the nature of these failures, particularly those for the CSD, sheds further light on the issue of strain in crystallog. structures.  Some of the residual failure cases not resolved by simple changes in OMEGA's defaults present significant challenges to conformer generation engines like OMEGA and are a source of new avenues to further improve their performance, while others illustrate the pitfalls of validating against crystallog. ligand conformations, particularly those from the PDB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTlHYy7GV9RbVg90H21EOLACvtfcHk0lhDd5ijd5E2jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFekurzN&md5=852c44d8587480e69709cefeee9206ac</span></div><a href="/servlet/linkout?suffix=cit166&amp;dbid=16384&amp;doi=10.1021%2Fci300314k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300314k%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DConformer%2520generation%2520with%2520OMEGA%253A%2520learning%2520from%2520the%2520data%2520set%2520and%2520the%2520analysis%2520of%2520failures%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D2919%26epage%3D2936%26doi%3D10.1021%2Fci300314k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref167"><div class="reference"><strong class="refLabel"><a href="#ref167" class="rightTabRefNumLink">167</a></strong><div class="NLM_citation" id="rightTab-cit167"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, L. L.</span><span> </span><span class="NLM_article-title">Perspective on free-energy perturbation calculations for chemical equilibria</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">869</span><span class="NLM_x">–</span> <span class="NLM_lpage">876</span><span class="refDoi"> DOI: 10.1021/ct800011m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct800011m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlslOisLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=869-876&author=W.+L.+Jorgensenauthor=L.+L.+Thomas&title=Perspective+on+free-energy+perturbation+calculations+for+chemical+equilibria&doi=10.1021%2Fct800011m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit167R"><div class="casContent"><span class="casTitleNuber">167</span><div class="casTitle"><span class="NLM_cas:atitle">Perspective on Free-Energy Perturbation Calculations for Chemical Equilibria</span></div><div class="casAuthors">Jorgensen, William L.; Thomas, Laura L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">869-876</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An overview has been provided on the history of free-energy perturbation (FEP) calcns. and their application to computation of relative free energies of solvation.  In view of the expanding role of FEP calcns. for pursuing a wide range of problems in chem. and biochem., including mol. design, it is important to establish reliable protocols for their execution.  For 30 interconversions of substituted benzenes, which represent prototypical cases for mol. design, alternative sampling procedures and nos. of FEP windows were considered.  Simple overlap sampling with 11 or 15 windows and double-wide sampling with 14 windows emerged as viable procedures for such routine FEP calcns.; the present protocols with these sampling methods yield statistical uncertainties of 0.2 kcal/mol for free-energy differences.  The slightly greater accuracy obtainable with 21 to 25 windows does not clearly justify the increased costs, while overlap sampling with 6 or 8 windows may be useful for preliminary exploration of large nos. of mol. changes.  Application of the power-series expansion of the Zwanzig equation was also tested.  Use of the first-order term is generally reliable, while inclusion of the slowly convergent, second-order fluctuation term causes deterioration in the results for strongly hydrogen-bonded solutes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqukDgvYtLny7Vg90H21EOLACvtfcHk0lhDd5ijd5E2jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlslOisLw%253D&md5=4f6cd47142dbbcd51e94aedd2d878538</span></div><a href="/servlet/linkout?suffix=cit167&amp;dbid=16384&amp;doi=10.1021%2Fct800011m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct800011m%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DThomas%26aufirst%3DL.%2BL.%26atitle%3DPerspective%2520on%2520free-energy%2520perturbation%2520calculations%2520for%2520chemical%2520equilibria%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2008%26volume%3D4%26spage%3D869%26epage%3D876%26doi%3D10.1021%2Fct800011m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref168"><div class="reference"><strong class="refLabel"><a href="#ref168" class="rightTabRefNumLink">168</a></strong><div class="NLM_citation" id="rightTab-cit168"><span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span> </span><span class="NLM_article-title">Binding of small-molecule ligands to proteins: "what you see" is not always "what you get"</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">489</span><span class="NLM_x">–</span> <span class="NLM_lpage">498</span><span class="refDoi"> DOI: 10.1016/j.str.2009.02.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.str.2009.02.010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=489-498&author=D.+L.+Mobleyauthor=K.+A.+Dill&title=Binding+of+small-molecule+ligands+to+proteins%3A+%22what+you+see%22+is+not+always+%22what+you+get%22&doi=10.1016%2Fj.str.2009.02.010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit168&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2009.02.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2009.02.010%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DBinding%2520of%2520small-molecule%2520ligands%2520to%2520proteins%253A%2520%2522what%2520you%2520see%2522%2520is%2520not%2520always%2520%2522what%2520you%2520get%2522%26jtitle%3DStructure%26date%3D2009%26volume%3D17%26spage%3D489%26epage%3D498%26doi%3D10.1016%2Fj.str.2009.02.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref169"><div class="reference"><strong class="refLabel"><a href="#ref169" class="rightTabRefNumLink">169</a></strong><div class="NLM_citation" id="rightTab-cit169"><span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McReynolds, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span> </span><span class="NLM_article-title">Predicting absolute ligand binding free energies to a simple model site</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">371</span><span class="NLM_x">, </span> <span class="NLM_fpage">1118</span><span class="NLM_x">–</span> <span class="NLM_lpage">1134</span><span class="refDoi"> DOI: 10.1016/j.jmb.2007.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.jmb.2007.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=17599350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2sXot12rt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=2007&pages=1118-1134&author=D.+L.+Mobleyauthor=A.+P.+Gravesauthor=J.+D.+Choderaauthor=A.+C.+McReynoldsauthor=B.+K.+Shoichetauthor=K.+A.+Dill&title=Predicting+absolute+ligand+binding+free+energies+to+a+simple+model+site&doi=10.1016%2Fj.jmb.2007.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit169R"><div class="casContent"><span class="casTitleNuber">169</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Absolute Ligand Binding Free Energies to a Simple Model Site</span></div><div class="casAuthors">Mobley, David L.; Graves, Alan P.; Chodera, John D.; McReynolds, Andrea C.; Shoichet, Brian K.; Dill, Ken A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1118-1134</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A central challenge in structure-based ligand design is the accurate prediction of binding free energies.  Here the authors apply alchem. free energy calcns. in explicit solvent to predict ligand binding in a model cavity in T4 lysozyme.  Even in this simple site, there are challenges.  The authors made systematic improvements, beginning with single poses from docking, then including multiple poses, addnl. protein conformational changes, and using an improved charge model.  Computed abs. binding free energies had an RMS error of 1.9 kcal/mol relative to previously detd. exptl. values.  In blind prospective tests, the methods correctly discriminated between several true ligands and decoys in a set of putative binders identified by docking.  In these prospective tests, the RMS error in predicted binding free energies relative to those subsequently detd. exptl. was only 0.6 kcal/mol.  X-ray crystal structures of the new ligands bound in the cavity corresponded closely to predictions from the free energy calcns., but sometimes differed from those predicted by docking.  Finally, the authors examd. the impact of holding the protein rigid, as in docking, with a view to learning how approxns. made in docking affect accuracy and how they may be improved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD1-EYEODQ47Vg90H21EOLACvtfcHk0lhPeYVRInwGww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXot12rt7o%253D&md5=2480139daa5a0857c59e821d2ecbce18</span></div><a href="/servlet/linkout?suffix=cit169&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2007.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2007.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DMcReynolds%26aufirst%3DA.%2BC.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DPredicting%2520absolute%2520ligand%2520binding%2520free%2520energies%2520to%2520a%2520simple%2520model%2520site%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D371%26spage%3D1118%26epage%3D1134%26doi%3D10.1016%2Fj.jmb.2007.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref170"><div class="reference"><strong class="refLabel"><a href="#ref170" class="rightTabRefNumLink">170</a></strong><div class="NLM_citation" id="rightTab-cit170"><span><span class="NLM_contrib-group">Rocklin, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyce, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span> </span><span class="NLM_article-title">Blind prediction of charged ligand binding affinities in a model binding site</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">425</span><span class="NLM_x">, </span> <span class="NLM_fpage">4569</span><span class="NLM_x">–</span> <span class="NLM_lpage">4583</span><span class="refDoi"> DOI: 10.1016/j.jmb.2013.07.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.jmb.2013.07.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=23896298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1OjtrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=425&publication_year=2013&pages=4569-4583&author=G.+J.+Rocklinauthor=S.+E.+Boyceauthor=M.+Fischerauthor=I.+Fishauthor=D.+L.+Mobleyauthor=B.+K.+Shoichetauthor=K.+A.+Dill&title=Blind+prediction+of+charged+ligand+binding+affinities+in+a+model+binding+site&doi=10.1016%2Fj.jmb.2013.07.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit170R"><div class="casContent"><span class="casTitleNuber">170</span><div class="casTitle"><span class="NLM_cas:atitle">Blind Prediction of Charged Ligand Binding Affinities in a Model Binding Site</span></div><div class="casAuthors">Rocklin, Gabriel J.; Boyce, Sarah E.; Fischer, Marcus; Fish, Inbar; Mobley, David L.; Shoichet, Brian K.; Dill, Ken A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">425</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">4569-4583</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Predicting abs. protein-ligand binding affinities remains a frontier challenge in ligand discovery and design.  This becomes more difficult when ionic interactions are involved because of the large opposing solvation and electrostatic attraction energies.  In a blind test, we examd. whether alchem. free-energy calcns. could predict binding affinities of 14 charged and 5 neutral compds. previously untested as ligands for a cavity binding site in cytochrome c peroxidase.  In this simplified site, polar and cationic ligands compete with solvent to interact with a buried aspartate.  Predictions were tested by calorimetry, spectroscopy, and crystallog.  Of the 15 compds. predicted to bind, 13 were exptl. confirmed, while 4 compds. were false neg. predictions.  Predictions had a root-mean-square error of 1.95 kcal/mol to the exptl. affinities, and predicted poses had an av. RMSD of 1.7 Å to the crystallog. poses.  This test serves as a benchmark for these thermodynamically rigorous calcns. at predicting binding affinities for charged compds. and gives insights into the existing sources of error, which are primarily electrostatic interactions inside proteins.  Our expts. also provide a useful set of ionic binding affinities in a simplified system for testing new affinity prediction methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZ_zM-ANClXLVg90H21EOLACvtfcHk0lhPeYVRInwGww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1OjtrnO&md5=d83561c4373ce7a24c7c040a00a0ea22</span></div><a href="/servlet/linkout?suffix=cit170&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2013.07.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2013.07.030%26sid%3Dliteratum%253Aachs%26aulast%3DRocklin%26aufirst%3DG.%2BJ.%26aulast%3DBoyce%26aufirst%3DS.%2BE.%26aulast%3DFischer%26aufirst%3DM.%26aulast%3DFish%26aufirst%3DI.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DBlind%2520prediction%2520of%2520charged%2520ligand%2520binding%2520affinities%2520in%2520a%2520model%2520binding%2520site%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2013%26volume%3D425%26spage%3D4569%26epage%3D4583%26doi%3D10.1016%2Fj.jmb.2013.07.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref171"><div class="reference"><strong class="refLabel"><a href="#ref171" class="rightTabRefNumLink">171</a></strong><div class="NLM_citation" id="rightTab-cit171"><span><span class="NLM_contrib-group">Lippert, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Predescu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ierardi, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackenzie, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eastwood, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">Accurate and efficient integration for molecular dynamics simulations at constant temperature and pressure</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">139</span><span class="NLM_x">, </span> <span class="NLM_fpage">164106</span><span class="refDoi"> DOI: 10.1063/1.4825247</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1063%2F1.4825247" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24182003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Kisb%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2013&pages=164106&author=R.+A.+Lippertauthor=C.+Predescuauthor=D.+J.+Ierardiauthor=K.+M.+Mackenzieauthor=M.+P.+Eastwoodauthor=R.+O.+Drorauthor=D.+E.+Shaw&title=Accurate+and+efficient+integration+for+molecular+dynamics+simulations+at+constant+temperature+and+pressure&doi=10.1063%2F1.4825247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit171R"><div class="casContent"><span class="casTitleNuber">171</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate and efficient integration for molecular dynamics simulations at constant temperature and pressure</span></div><div class="casAuthors">Lippert, Ross A.; Predescu, Cristian; Ierardi, Douglas J.; MacKenzie, Kenneth M.; Eastwood, Michael P.; Dror, Ron O.; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">164106/1-164106/11</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">In mol. dynamics simulations, control over temp. and pressure is typically achieved by augmenting the original system with addnl. dynamical variables to create a thermostat and a barostat, resp.  These variables generally evolve on timescales much longer than those of particle motion, but typical integrator implementations update the addnl. variables along with the particle positions and momenta at each time step.  We present a framework that replaces the traditional integration procedure with sep. barostat, thermostat, and Newtonian particle motion updates, allowing thermostat and barostat updates to be applied infrequently.  Such infrequent updates provide a particularly substantial performance advantage for simulations parallelized across many computer processors, because thermostat and barostat updates typically require communication among all processors.  Infrequent updates can also improve accuracy by alleviating certain sources of error assocd. with limited-precision arithmetic.  In addn., sepg. the barostat, thermostat, and particle motion update steps reduces certain truncation errors, bringing the time-av. pressure closer to its target value.  Finally, this framework, which we have implemented on both general-purpose and special-purpose hardware, reduces software complexity and improves software modularity.  (c) 2013 American Institute of Physics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW51ZULlZFvrVg90H21EOLACvtfcHk0lhPeYVRInwGww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Kisb%252FL&md5=0bd59b4b73b6999b9311306f4c32f3c1</span></div><a href="/servlet/linkout?suffix=cit171&amp;dbid=16384&amp;doi=10.1063%2F1.4825247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.4825247%26sid%3Dliteratum%253Aachs%26aulast%3DLippert%26aufirst%3DR.%2BA.%26aulast%3DPredescu%26aufirst%3DC.%26aulast%3DIerardi%26aufirst%3DD.%2BJ.%26aulast%3DMackenzie%26aufirst%3DK.%2BM.%26aulast%3DEastwood%26aufirst%3DM.%2BP.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DAccurate%2520and%2520efficient%2520integration%2520for%2520molecular%2520dynamics%2520simulations%2520at%2520constant%2520temperature%2520and%2520pressure%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2013%26volume%3D139%26spage%3D164106%26doi%3D10.1063%2F1.4825247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref172"><div class="reference"><strong class="refLabel"><a href="#ref172" class="rightTabRefNumLink">172</a></strong><div class="NLM_citation" id="rightTab-cit172"><span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valant, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borhani, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valcourt, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arlow, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canals, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahmani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sexton, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christopoulos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">Structural basis for modulation of a G-protein-coupled receptor by allosteric drugs</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">503</span><span class="NLM_x">, </span> <span class="NLM_fpage">295</span><span class="NLM_x">–</span> <span class="NLM_lpage">299</span><span class="refDoi"> DOI: 10.1038/nature12595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnature12595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24121438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslSns7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2013&pages=295-299&author=R.+O.+Drorauthor=H.+F.+Greenauthor=C.+Valantauthor=D.+W.+Borhaniauthor=J.+R.+Valcourtauthor=A.+C.+Panauthor=D.+H.+Arlowauthor=M.+Canalsauthor=J.+R.+Laneauthor=R.+Rahmaniauthor=J.+B.+Baellauthor=P.+M.+Sextonauthor=A.+Christopoulosauthor=D.+E.+Shaw&title=Structural+basis+for+modulation+of+a+G-protein-coupled+receptor+by+allosteric+drugs&doi=10.1038%2Fnature12595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit172R"><div class="casContent"><span class="casTitleNuber">172</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for modulation of a G-protein-coupled receptor by allosteric drugs</span></div><div class="casAuthors">Dror, Ron O.; Green, Hillary F.; Valant, Celine; Borhani, David W.; Valcourt, James R.; Pan, Albert C.; Arlow, Daniel H.; Canals, Meritxell; Lane, J. Robert; Rahmani, Raphael; Baell, Jonathan B.; Sexton, Patrick M.; Christopoulos, Arthur; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">503</span>
        (<span class="NLM_cas:issue">7475</span>),
    <span class="NLM_cas:pages">295-299</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The design of G-protein-coupled receptor (GPCR) allosteric modulators, an active area of modern pharmaceutical research, has proved challenging because neither the binding modes nor the mol. mechanisms of such drugs are known.  Here we det. binding sites, bound conformations and specific drug-receptor interactions for several allosteric modulators of the M2 muscarinic acetylcholine receptor (M2 receptor), a prototypical family A GPCR, using at.-level simulations in which the modulators spontaneously assoc. with the receptor.  Despite substantial structural diversity, all modulators form cation-π interactions with clusters of arom. residues in the receptor extracellular vestibule, approx. 15 Å from the classical, 'orthosteric' ligand-binding site.  We validate the obsd. modulator binding modes through radioligand binding expts. on receptor mutants designed, on the basis of our simulations, either to increase or to decrease modulator affinity.  Simulations also revealed mechanisms that contribute to pos. and neg. allosteric modulation of classical ligand binding, including coupled conformational changes of the two binding sites and electrostatic interactions between ligands in these sites.  These observations enabled the design of chem. modifications that substantially alter a modulator's allosteric effects.  Our findings thus provide a structural basis for the rational design of allosteric modulators targeting muscarinic and possibly other GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTlvAcaBf-1rVg90H21EOLACvtfcHk0liLVUT7AfpsLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslSns7nN&md5=2fa40ecd46ce62125ce4a12c18270b1f</span></div><a href="/servlet/linkout?suffix=cit172&amp;dbid=16384&amp;doi=10.1038%2Fnature12595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12595%26sid%3Dliteratum%253Aachs%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DGreen%26aufirst%3DH.%2BF.%26aulast%3DValant%26aufirst%3DC.%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DValcourt%26aufirst%3DJ.%2BR.%26aulast%3DPan%26aufirst%3DA.%2BC.%26aulast%3DArlow%26aufirst%3DD.%2BH.%26aulast%3DCanals%26aufirst%3DM.%26aulast%3DLane%26aufirst%3DJ.%2BR.%26aulast%3DRahmani%26aufirst%3DR.%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DStructural%2520basis%2520for%2520modulation%2520of%2520a%2520G-protein-coupled%2520receptor%2520by%2520allosteric%2520drugs%26jtitle%3DNature%26date%3D2013%26volume%3D503%26spage%3D295%26epage%3D299%26doi%3D10.1038%2Fnature12595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref173"><div class="reference"><strong class="refLabel"><a href="#ref173" class="rightTabRefNumLink">173</a></strong><div class="NLM_citation" id="rightTab-cit173"><span><span class="NLM_contrib-group">Shukla, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hernandez, C. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span> </span><span class="NLM_article-title">Markov state models provide insights into dynamic modulation of protein function</span> <span class="citation_source-journal">Acc. Chem. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">414</span><span class="NLM_x">–</span> <span class="NLM_lpage">422</span><span class="refDoi"> DOI: 10.1021/ar5002999</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar5002999" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXks1Oksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2015&pages=414-422&author=D.+Shuklaauthor=C.+X.+Hernandezauthor=J.+K.+Weberauthor=V.+S.+Pande&title=Markov+state+models+provide+insights+into+dynamic+modulation+of+protein+function&doi=10.1021%2Far5002999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit173R"><div class="casContent"><span class="casTitleNuber">173</span><div class="casTitle"><span class="NLM_cas:atitle">Markov state models provide insights into dynamic modulation of protein function</span></div><div class="casAuthors">Shukla, Diwakar; Hernandez, Carlos X.; Weber, Jeffrey K.; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">414-422</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Protein function is inextricably linked to protein dynamics.  As one moves from a static structural picture to a dynamic ensemble view of protein structure and function, novel computational paradigms are required for observing and understanding conformational dynamics of proteins and its functional implications.  In principle, mol. dynamics simulations can provide the time evolution of atomistic models of proteins, but the long time scales assocd. with functional dynamics make it difficult to observe rare dynamical transitions.  The issue of extg. essential functional components of protein dynamics from noisy simulation data presents another set of challenges in obtaining an unbiased understanding of protein motions.  Therefore, a methodol. that provides a statistical framework for efficient sampling and a human-readable view of the key aspects of functional dynamics from data anal. is required.  The Markov state model (MSM), which has recently become popular worldwide for studying protein dynamics, is an example of such a framework.  Here, the authors review the use of MSMs for efficient sampling of the hierarchy of time scales assocd. with protein dynamics, automatic identification of key conformational states, and the degrees of freedom assocd. with slow dynamical processes.  Applications of MSMs for studying long time scale phenomena such as activation mechanisms of cellular signaling proteins has yielded novel insights into protein function.  In particular, from MSMs built using large-scale simulations of GPCRs and kinases, the authors have shown that complex conformational changes in proteins can be described in terms of structural changes in key structural motifs or "mol. switches" within the protein, that the transitions between functionally active and inactive states of proteins proceed via multiple pathways, and that ligand or substrate binding modulates the flux through these pathways.  Finally, MSMs also provide a theor. toolbox for studying the effect of nonequil. perturbations on conformational dynamics.  Considering that protein dynamics in vivo occur under nonequil. conditions, MSMs coupled with nonequil. statistical mechanics provide a way to connect cellular components to their functional environments.  Nonequil. perturbations of protein folding MSMs reveal the presence of dynamically frozen glass-like states in their conformational landscape.  These frozen states are also obsd. to be rich in β-sheets, which indicates their possible role in the nucleation of β-sheet rich aggregates such as those obsd. in amyloid-fibril formation.  Finally, the authors describe how MSMs have been used to understand the dynamical behavior of intrinsically disordered proteins such as amyloid-β, human islet amyloid polypeptide, and p53.  While certainly not a panacea for studying functional dynamics, MSMs provide a rigorous theor. foundation for understanding complex entropically dominated processes and a convenient lens for viewing protein motions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_eCbE2nT1prVg90H21EOLACvtfcHk0liLVUT7AfpsLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXks1Oksw%253D%253D&md5=d7d0042973fdc6a4f28d6259729dcebe</span></div><a href="/servlet/linkout?suffix=cit173&amp;dbid=16384&amp;doi=10.1021%2Far5002999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far5002999%26sid%3Dliteratum%253Aachs%26aulast%3DShukla%26aufirst%3DD.%26aulast%3DHernandez%26aufirst%3DC.%2BX.%26aulast%3DWeber%26aufirst%3DJ.%2BK.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DMarkov%2520state%2520models%2520provide%2520insights%2520into%2520dynamic%2520modulation%2520of%2520protein%2520function%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2015%26volume%3D48%26spage%3D414%26epage%3D422%26doi%3D10.1021%2Far5002999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref174"><div class="reference"><strong class="refLabel"><a href="#ref174" class="rightTabRefNumLink">174</a></strong><div class="NLM_citation" id="rightTab-cit174"><span><span class="NLM_contrib-group">Barelier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eidam, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollander, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figaroa, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nachane, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siegal, G.</span><span> </span><span class="NLM_article-title">Increasing chemical space coverage by combining empirical and computational fragment screens</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1528</span><span class="NLM_x">–</span> <span class="NLM_lpage">1535</span><span class="refDoi"> DOI: 10.1021/cb5001636</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb5001636" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnsFSksbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1528-1535&author=S.+Barelierauthor=O.+Eidamauthor=I.+Fishauthor=J.+Hollanderauthor=F.+Figaroaauthor=R.+Nachaneauthor=J.+J.+Irwinauthor=B.+K.+Shoichetauthor=G.+Siegal&title=Increasing+chemical+space+coverage+by+combining+empirical+and+computational+fragment+screens&doi=10.1021%2Fcb5001636"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit174R"><div class="casContent"><span class="casTitleNuber">174</span><div class="casTitle"><span class="NLM_cas:atitle">Increasing Chemical Space Coverage by Combining Empirical and Computational Fragment Screens</span></div><div class="casAuthors">Barelier, Sarah; Eidam, Oliv; Fish, Inbar; Hollander, Johan; Figaroa, Francis; Nachane, Ruta; Irwin, John J.; Shoichet, Brian K.; Siegal, Gregg</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1528-1535</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Most libraries for fragment-based drug discovery are restricted to 1,000-10,000 compds., but over 500,000 fragments are com. available and potentially accessible by virtual screening.  Whether this larger set would increase chemotype coverage, and whether a computational screen can pragmatically prioritize them, is debated.  To investigate this question, a 1281-fragment library was screened by NMR (NMR) against AmpC β-lactamase, and hits were confirmed by surface plasmon resonance (SPR).  Nine hits with novel chemotypes were confirmed biochem. with KI values from 0.2 to low mM.  We also computationally docked 290,000 purchasable fragments with chemotypes unrepresented in the empirical library, finding 10 that had KI values from 0.03 to low mM.  Though less novel than those discovered by NMR, the docking-derived fragments filled chemotype holes from the empirical library.  Crystal structures of nine of the fragments in complex with AmpC β-lactamase revealed new binding sites and explained the relatively high affinity of the docking-derived fragments.  The existence of chemotype holes is likely a general feature of fragment libraries, as calcn. suggests that to represent the fragment substructures of even known biogenic mols. would demand a library of minimally over 32,000 fragments.  Combining computational and empirical fragment screens enables the discovery of unexpected chemotypes, here by the NMR screen, while capturing chemotypes missing from the empirical library and tailored to the target, with little extra cost in resources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5DNyK9V50nLVg90H21EOLACvtfcHk0liLVUT7AfpsLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnsFSksbg%253D&md5=7c75e304e70fec6d9209c21a87a06a17</span></div><a href="/servlet/linkout?suffix=cit174&amp;dbid=16384&amp;doi=10.1021%2Fcb5001636&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb5001636%26sid%3Dliteratum%253Aachs%26aulast%3DBarelier%26aufirst%3DS.%26aulast%3DEidam%26aufirst%3DO.%26aulast%3DFish%26aufirst%3DI.%26aulast%3DHollander%26aufirst%3DJ.%26aulast%3DFigaroa%26aufirst%3DF.%26aulast%3DNachane%26aufirst%3DR.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DSiegal%26aufirst%3DG.%26atitle%3DIncreasing%2520chemical%2520space%2520coverage%2520by%2520combining%2520empirical%2520and%2520computational%2520fragment%2520screens%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1528%26epage%3D1535%26doi%3D10.1021%2Fcb5001636" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref175"><div class="reference"><strong class="refLabel"><a href="#ref175" class="rightTabRefNumLink">175</a></strong><div class="NLM_citation" id="rightTab-cit175"><span><span class="NLM_contrib-group">London, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krishnan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uchida, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eidam, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibold, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cimermancic, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taunton, J.</span><span> </span><span class="NLM_article-title">Covalent docking of large libraries for the discovery of chemical probes</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1066</span><span class="NLM_x">–</span> <span class="NLM_lpage">1072</span><span class="refDoi"> DOI: 10.1038/nchembio.1666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnchembio.1666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25344815" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVSqsbjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=1066-1072&author=N.+Londonauthor=R.+M.+Millerauthor=S.+Krishnanauthor=K.+Uchidaauthor=J.+J.+Irwinauthor=O.+Eidamauthor=L.+Giboldauthor=P.+Cimermancicauthor=R.+Bonnetauthor=B.+K.+Shoichetauthor=J.+Taunton&title=Covalent+docking+of+large+libraries+for+the+discovery+of+chemical+probes&doi=10.1038%2Fnchembio.1666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit175R"><div class="casContent"><span class="casTitleNuber">175</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent docking of large libraries for the discovery of chemical probes</span></div><div class="casAuthors">London, Nir; Miller, Rand M.; Krishnan, Shyam; Uchida, Kenji; Irwin, John J.; Eidam, Oliv; Gibold, Lucie; Cimermancic, Peter; Bonnet, Richard; Shoichet, Brian K.; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1066-1072</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes that form a covalent bond with a protein target often show enhanced selectivity, potency and utility for biol. studies.  Despite these advantages, protein-reactive compds. are usually avoided in high-throughput screening campaigns.  Here we describe a general method (DOCKovalent) for screening large virtual libraries of electrophilic small mols.  We apply this method prospectively to discover reversible covalent fragments that target distinct protein nucleophiles, including the catalytic serine of AmpC β-lactamase and noncatalytic cysteines in RSK2, MSK1 and JAK3 kinases.  We identify submicromolar to low-nanomolar hits with high ligand efficiency, cellular activity and selectivity, including what are to our knowledge the first reported reversible covalent inhibitors of JAK3.  Crystal structures of inhibitor complexes with AmpC and RSK2 confirm the docking predictions and guide further optimization.  As covalent virtual screening may have broad utility for the rapid discovery of chem. probes, we have made the method freely available through an automated web server (http://covalent.docking.org/).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpngQdIYqslNLVg90H21EOLACvtfcHk0liKUUWKMrHB5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVSqsbjE&md5=4e5fbeb8f67cb82d464d57069ba0c93a</span></div><a href="/servlet/linkout?suffix=cit175&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1666%26sid%3Dliteratum%253Aachs%26aulast%3DLondon%26aufirst%3DN.%26aulast%3DMiller%26aufirst%3DR.%2BM.%26aulast%3DKrishnan%26aufirst%3DS.%26aulast%3DUchida%26aufirst%3DK.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DEidam%26aufirst%3DO.%26aulast%3DGibold%26aufirst%3DL.%26aulast%3DCimermancic%26aufirst%3DP.%26aulast%3DBonnet%26aufirst%3DR.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DCovalent%2520docking%2520of%2520large%2520libraries%2520for%2520the%2520discovery%2520of%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D1066%26epage%3D1072%26doi%3D10.1038%2Fnchembio.1666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref176"><div class="reference"><strong class="refLabel"><a href="#ref176" class="rightTabRefNumLink">176</a></strong><div class="NLM_citation" id="rightTab-cit176"><span><span class="NLM_contrib-group">Bianco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodsell, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">Covalent docking using autodock: Two-point attractor and flexible side chain methods</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">295</span><span class="NLM_x">–</span> <span class="NLM_lpage">301</span><span class="refDoi"> DOI: 10.1002/pro.2733</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fpro.2733" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26103917" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFehs7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2016&pages=295-301&author=G.+Biancoauthor=S.+Forliauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=Covalent+docking+using+autodock%3A+Two-point+attractor+and+flexible+side+chain+methods&doi=10.1002%2Fpro.2733"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit176R"><div class="casContent"><span class="casTitleNuber">176</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent docking using autodock: Two-point attractor and flexible side chain methods</span></div><div class="casAuthors">Bianco, Giulia; Forli, Stefano; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">295-301</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">We describe two methods of automated covalent docking using Autodock4: the two-point attractor method and the flexible side chain method.  Both methods were applied to a training set of 20 diverse protein-ligand covalent complexes, evaluating their reliability in predicting the crystallog. pose of the ligands.  The flexible side chain method performed best, recovering the pose in 75% of cases, with failures for the largest inhibitors tested.  Both methods are freely available at the AutoDock website ().</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbgwvwS_NTKrVg90H21EOLACvtfcHk0liKUUWKMrHB5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFehs7rI&md5=4e380f340983e78e149f48934ccc59fd</span></div><a href="/servlet/linkout?suffix=cit176&amp;dbid=16384&amp;doi=10.1002%2Fpro.2733&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.2733%26sid%3Dliteratum%253Aachs%26aulast%3DBianco%26aufirst%3DG.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DCovalent%2520docking%2520using%2520autodock%253A%2520Two-point%2520attractor%2520and%2520flexible%2520side%2520chain%2520methods%26jtitle%3DProtein%2520Sci.%26date%3D2016%26volume%3D25%26spage%3D295%26epage%3D301%26doi%3D10.1002%2Fpro.2733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref177"><div class="reference"><strong class="refLabel"><a href="#ref177" class="rightTabRefNumLink">177</a></strong><div class="NLM_citation" id="rightTab-cit177"><span><span class="NLM_contrib-group">Zhu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borrelli, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farid, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harder, E.</span><span> </span><span class="NLM_article-title">Docking covalent inhibitors: a parameter free approach to pose prediction and scoring</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1932</span><span class="NLM_x">–</span> <span class="NLM_lpage">1940</span><span class="refDoi"> DOI: 10.1021/ci500118s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500118s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpsVKksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1932-1940&author=K.+Zhuauthor=K.+W.+Borrelliauthor=J.+R.+Greenwoodauthor=T.+Dayauthor=R.+Abelauthor=R.+S.+Faridauthor=E.+Harder&title=Docking+covalent+inhibitors%3A+a+parameter+free+approach+to+pose+prediction+and+scoring&doi=10.1021%2Fci500118s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit177R"><div class="casContent"><span class="casTitleNuber">177</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Covalent Inhibitors: A Parameter Free Approach To Pose Prediction and Scoring</span></div><div class="casAuthors">Zhu, Kai; Borrelli, Kenneth W.; Greenwood, Jeremy R.; Day, Tyler; Abel, Robert; Farid, Ramy S.; Harder, Edward</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1932-1940</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Although many popular docking programs include a facility to account for covalent ligands, large-scale systematic docking validation studies of covalent inhibitors have been sparse.  In this paper, we present the development and validation of a novel approach for docking and scoring covalent inhibitors, which consists of conventional noncovalent docking, heuristic formation of the covalent attachment point, and structural refinement of the protein-ligand complex.  This approach combines the strengths of the docking program Glide and the protein structure modeling program Prime and does not require any parameter fitting for the study of addnl. covalent reaction types.  We first test this method by predicting the native binding geometry of 38 covalently bound complexes.  The av. RMSD of the predicted poses is 1.52 Å, and 76% of test set inhibitors have an RMSD of less than 2.0 Å.  In addn., the apparent affinity score constructed herein is tested on a virtual screening study and the characterization of the SAR properties of two different series of congeneric compds. with satisfactory success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbtjEb3EqzQ7Vg90H21EOLACvtfcHk0liKUUWKMrHB5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpsVKksLs%253D&md5=3633c29fa5dc29817b178fd10a4c7b15</span></div><a href="/servlet/linkout?suffix=cit177&amp;dbid=16384&amp;doi=10.1021%2Fci500118s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500118s%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DBorrelli%26aufirst%3DK.%2BW.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DFarid%26aufirst%3DR.%2BS.%26aulast%3DHarder%26aufirst%3DE.%26atitle%3DDocking%2520covalent%2520inhibitors%253A%2520a%2520parameter%2520free%2520approach%2520to%2520pose%2520prediction%2520and%2520scoring%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1932%26epage%3D1940%26doi%3D10.1021%2Fci500118s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref178"><div class="reference"><strong class="refLabel"><a href="#ref178" class="rightTabRefNumLink">178</a></strong><div class="NLM_citation" id="rightTab-cit178"><span><span class="NLM_contrib-group">Ouyang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ge, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwoh, C. K.</span><span> </span><span class="NLM_article-title">CovalentDock Cloud: a web server for automated covalent docking</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">W329</span><span class="NLM_x">–</span> <span class="NLM_lpage">332</span><span class="refDoi"> DOI: 10.1093/nar/gkt406</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1093%2Fnar%2Fgkt406" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=W329-332&author=X.+Ouyangauthor=S.+Zhouauthor=Z.+Geauthor=R.+Liauthor=C.+K.+Kwoh&title=CovalentDock+Cloud%3A+a+web+server+for+automated+covalent+docking&doi=10.1093%2Fnar%2Fgkt406"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit178&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt406%26sid%3Dliteratum%253Aachs%26aulast%3DOuyang%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DGe%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DKwoh%26aufirst%3DC.%2BK.%26atitle%3DCovalentDock%2520Cloud%253A%2520a%2520web%2520server%2520for%2520automated%2520covalent%2520docking%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3DW329%26epage%3D332%26doi%3D10.1093%2Fnar%2Fgkt406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref179"><div class="reference"><strong class="refLabel"><a href="#ref179" class="rightTabRefNumLink">179</a></strong><div class="NLM_citation" id="rightTab-cit179"><span><span class="NLM_contrib-group">Ouyang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ge, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwoh, C. K.</span><span> </span><span class="NLM_article-title">CovalentDock: automated covalent docking with parameterized covalent linkage energy estimation and molecular geometry constraints</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">326</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1002/jcc.23136</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fjcc.23136" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=23034731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVKksLbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2013&pages=326-336&author=X.+Ouyangauthor=S.+Zhouauthor=C.+T.+Suauthor=Z.+Geauthor=R.+Liauthor=C.+K.+Kwoh&title=CovalentDock%3A+automated+covalent+docking+with+parameterized+covalent+linkage+energy+estimation+and+molecular+geometry+constraints&doi=10.1002%2Fjcc.23136"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit179R"><div class="casContent"><span class="casTitleNuber">179</span><div class="casTitle"><span class="NLM_cas:atitle">CovalentDock: Automated covalent docking with parameterized covalent linkage energy estimation and molecular geometry constrains</span></div><div class="casAuthors">Ouyang, Xuchang; Zhou, Shuo; Su, Chinh Tran To; Ge, Zemei; Li, Runtao; Kwoh, Chee Keong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">326-336</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Covalent linkage formation is a very important mechanism for many covalent drugs to work.  However, partly due to the limitations of proper computational tools for covalent docking, most covalent drugs are not discovered systematically.  In this article, we present a new covalent docking package, the CovalentDock, built on the top of the source code of Autodock.  We developed an empirical model of free energy change estn. for covalent linkage formation, which is compatible with existing scoring functions used in docking, while handling the mol. geometry constrains of the covalent linkage with special atom types and directional grid maps.  Integrated prepn. scripts are also written for the automation of the whole covalent docking workflow.  The result tested on existing crystal structures with covalent linkage shows that CovalentDock can reproduce the native covalent complexes with significant improved accuracy when compared with the default covalent docking method in Autodock.  Expts. also suggest that CovalentDock is capable of covalent virtual screening with satisfactory enrichment performance.  In addn., the investigation on the results also shows that the chirality and target selectivity along with the mol. geometry constrains are well preserved by CovalentDock, showing great capability of this method in the application for covalent drug discovery. © 2012 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1kpnxsCasoLVg90H21EOLACvtfcHk0lhKqOqmQO8CXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVKksLbJ&md5=5f548c115755f2e1470848ba6f6d701b</span></div><a href="/servlet/linkout?suffix=cit179&amp;dbid=16384&amp;doi=10.1002%2Fjcc.23136&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.23136%26sid%3Dliteratum%253Aachs%26aulast%3DOuyang%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DSu%26aufirst%3DC.%2BT.%26aulast%3DGe%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DKwoh%26aufirst%3DC.%2BK.%26atitle%3DCovalentDock%253A%2520automated%2520covalent%2520docking%2520with%2520parameterized%2520covalent%2520linkage%2520energy%2520estimation%2520and%2520molecular%2520geometry%2520constraints%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2013%26volume%3D34%26spage%3D326%26epage%3D336%26doi%3D10.1002%2Fjcc.23136" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref180"><div class="reference"><strong class="refLabel"><a href="#ref180" class="rightTabRefNumLink">180</a></strong><div class="NLM_citation" id="rightTab-cit180"><span><span class="NLM_contrib-group">Toledo Warshaviak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golan, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borrelli, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalid, O.</span><span> </span><span class="NLM_article-title">Structure-based virtual screening approach for discovery of covalently bound ligands</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1941</span><span class="NLM_x">–</span> <span class="NLM_lpage">1950</span><span class="refDoi"> DOI: 10.1021/ci500175r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500175r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslegtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1941-1950&author=D.+Toledo+Warshaviakauthor=G.+Golanauthor=K.+W.+Borrelliauthor=K.+Zhuauthor=O.+Kalid&title=Structure-based+virtual+screening+approach+for+discovery+of+covalently+bound+ligands&doi=10.1021%2Fci500175r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit180R"><div class="casContent"><span class="casTitleNuber">180</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Virtual Screening Approach for Discovery of Covalently Bound Ligands</span></div><div class="casAuthors">Toledo Warshaviak, Dora; Golan, Gali; Borrelli, Kenneth W.; Zhu, Kai; Kalid, Ori</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1941-1950</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a fast and effective covalent docking approach suitable for large-scale virtual screening (VS).  We applied this method to four targets (HCV NS3 protease, Cathepsin K, EGFR, and XPO1) with known crystal structures and known covalent inhibitors.  We implemented a customized "VS mode" of the Schrodinger Covalent Docking algorithm (CovDock), which we refer to as CovDock-VS.  Known actives and target-specific sets of decoys were docked to selected X-ray structures, and poses were filtered based on noncovalent protein-ligand interactions known to be important for activity.  We were able to retrieve 71%, 72%, and 77% of the known actives for Cathepsin K, HCV NS3 protease, and EGFR within 5% of the decoy library, resp.  With the more challenging XPO1 target, where no specific interactions with the protein could be used for postprocessing of the docking results, we were able to retrieve 95% of the actives within 30% of the decoy library and achieved an early enrichment factor (EF1%) of 33.  The poses of the known actives bound to existing crystal structures of 4 targets were predicted with an av. RMSD of 1.9 Å.  To the best of our knowledge, CovDock-VS is the first fully automated tool for efficient virtual screening of covalent inhibitors.  Importantly, CovDock-VS can handle multiple chem. reactions within the same library, only requiring a generic SMARTS-based predefinition of the reaction.  CovDock-VS provides a fast and accurate way of differentiating actives from decoys without significantly deteriorating the accuracy of the predicted poses for covalent protein-ligand complexes.  Therefore, we propose CovDock-VS as an efficient structure-based virtual screening method for discovery of novel and diverse covalent ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-DH45VZl-w7Vg90H21EOLACvtfcHk0lhKqOqmQO8CXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslegtbw%253D&md5=0ff8b6e682473abd2e2dadb49d95539a</span></div><a href="/servlet/linkout?suffix=cit180&amp;dbid=16384&amp;doi=10.1021%2Fci500175r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500175r%26sid%3Dliteratum%253Aachs%26aulast%3DToledo%2BWarshaviak%26aufirst%3DD.%26aulast%3DGolan%26aufirst%3DG.%26aulast%3DBorrelli%26aufirst%3DK.%2BW.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DKalid%26aufirst%3DO.%26atitle%3DStructure-based%2520virtual%2520screening%2520approach%2520for%2520discovery%2520of%2520covalently%2520bound%2520ligands%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1941%26epage%3D1950%26doi%3D10.1021%2Fci500175r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref181"><div class="reference"><strong class="refLabel"><a href="#ref181" class="rightTabRefNumLink">181</a></strong><div class="NLM_citation" id="rightTab-cit181"><span><span class="NLM_contrib-group">Munoz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sospedra, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gomara, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mestres, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haro, I.</span><span> </span><span class="NLM_article-title">The covalent coupling of HAV-VP3 (110–121) synthetic peptide to liposomes: physicochemical studies</span> <span class="citation_source-journal">Int. J. Pharm.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">269</span><span class="NLM_x">, </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">184</span><span class="refDoi"> DOI: 10.1016/j.ijpharm.2003.09.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref181/cit181&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.ijpharm.2003.09.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref181/cit181&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=14698589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref181/cit181&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvFektLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=269&publication_year=2004&pages=177-184&author=M.+Munozauthor=P.+Sospedraauthor=M.+J.+Gomaraauthor=C.+Mestresauthor=I.+Haro&title=The+covalent+coupling+of+HAV-VP3+%28110%E2%80%93121%29+synthetic+peptide+to+liposomes%3A+physicochemical+studies&doi=10.1016%2Fj.ijpharm.2003.09.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit181R"><div class="casContent"><span class="casTitleNuber">181</span><div class="casTitle"><span class="NLM_cas:atitle">The covalent coupling of HAV-VP3 (110-121) synthetic peptide to liposomes: physicochemical studies</span></div><div class="casAuthors">Munoz, M.; Sospedra, P.; Gomara, M. J.; Mestres, C.; Haro, I.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Pharmaceutics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">269</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-184</span>CODEN:
                <span class="NLM_cas:coden">IJPHDE</span>;
        ISSN:<span class="NLM_cas:issn">0378-5173</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">In this work we have studied the conjugation of the immunogenic peptide sequence (110-121) belonging to the VP3 capsid protein of hepatitis A virus to the surface of preformed liposomes by means of an amide bond between the vesicles and the synthetic peptide.  The surface activity of the conjugate at air/water interface was detd.  Moreover, the interaction of the conjugate with lipids was also studied recording the pressure increases produced after the injection of the liposome-peptide prepn. under dipalmitoylphosphatidylcholine (DPPC), dipalmitoylphosphatidylglycerol (DPPG) and stearylamine (SA) monolayers at different initial surface pressures.  As expected, due to the neg. net charge of the liposome-peptide complex, the higher interaction was found with pos. charge monolayers (SA).  However, the conjugate was also able to incorporate to zwitterionic and anionic lipids.  This behavior was also confirmed performing compression isotherms of monolayers of these lipids spread on subphases contg. the conjugate.These results suggest that the coupling of VP3 (110-121) to liposomes does not influence its ability to interact with membrane lipids such as DPPC and DPPG.  Then it can be assumed that its immunogenicity will be preserved or even increased after this modification.  All these results are also useful in the prepn. of liposome-based synthetic peptide vaccines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoszv87GCrVVLVg90H21EOLACvtfcHk0lhKqOqmQO8CXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpvFektLo%253D&md5=41ee4ea1d9fff23d66094262cb320449</span></div><a href="/servlet/linkout?suffix=cit181&amp;dbid=16384&amp;doi=10.1016%2Fj.ijpharm.2003.09.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijpharm.2003.09.014%26sid%3Dliteratum%253Aachs%26aulast%3DMunoz%26aufirst%3DM.%26aulast%3DSospedra%26aufirst%3DP.%26aulast%3DGomara%26aufirst%3DM.%2BJ.%26aulast%3DMestres%26aufirst%3DC.%26aulast%3DHaro%26aufirst%3DI.%26atitle%3DThe%2520covalent%2520coupling%2520of%2520HAV-VP3%2520%2528110%25E2%2580%2593121%2529%2520synthetic%2520peptide%2520to%2520liposomes%253A%2520physicochemical%2520studies%26jtitle%3DInt.%2520J.%2520Pharm.%26date%3D2004%26volume%3D269%26spage%3D177%26epage%3D184%26doi%3D10.1016%2Fj.ijpharm.2003.09.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref182"><div class="reference"><strong class="refLabel"><a href="#ref182" class="rightTabRefNumLink">182</a></strong><div class="NLM_citation" id="rightTab-cit182"><span><span class="NLM_contrib-group">Scholz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knorr, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamacher, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, B.</span><span> </span><span class="NLM_article-title">DOCKTITE-a highly versatile step-by-step workflow for covalent docking and virtual screening in the molecular operating environment</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">398</span><span class="NLM_x">–</span> <span class="NLM_lpage">406</span><span class="refDoi"> DOI: 10.1021/ci500681r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500681r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFGrsrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=398-406&author=C.+Scholzauthor=S.+Knorrauthor=K.+Hamacherauthor=B.+Schmidt&title=DOCKTITE-a+highly+versatile+step-by-step+workflow+for+covalent+docking+and+virtual+screening+in+the+molecular+operating+environment&doi=10.1021%2Fci500681r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit182R"><div class="casContent"><span class="casTitleNuber">182</span><div class="casTitle"><span class="NLM_cas:atitle">DOCKTITE-A Highly Versatile Step-by-Step Workflow for Covalent Docking and Virtual Screening in the Molecular Operating Environment</span></div><div class="casAuthors">Scholz, Christoph; Knorr, Sabine; Hamacher, Kay; Schmidt, Boris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">398-406</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The formation of a covalent bond with the target is essential for a no. of successful drugs, yet tools for covalent docking without significant restrictions regarding warhead or receptor classes are rare and limited in use.  In this work we present DOCKTITE, a highly versatile workflow for covalent docking in the Mol. Operating Environment (MOE) combining automated warhead screening, nucleophilic side chain attachment, pharmacophore-based docking, and a novel consensus scoring approach.  The comprehensive validation study includes pose predictions of 35 protein/ligand complexes which resulted in a mean RMSD of 1.74 Å and a prediction rate of 71.4% with an RMSD below 2 Å, a virtual screening with an area under the curve (AUC) for the receiver operating characteristics (ROC) of 0.81, and a significant correlation between predicted and exptl. binding affinities (ρ = 0.806, R2 = 0.649, p < 0.005).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkYai-iGLMS7Vg90H21EOLACvtfcHk0lj6BlDtKp1Ylw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFGrsrrK&md5=738e69cfc93c5ce0f053b81837f632d7</span></div><a href="/servlet/linkout?suffix=cit182&amp;dbid=16384&amp;doi=10.1021%2Fci500681r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500681r%26sid%3Dliteratum%253Aachs%26aulast%3DScholz%26aufirst%3DC.%26aulast%3DKnorr%26aufirst%3DS.%26aulast%3DHamacher%26aufirst%3DK.%26aulast%3DSchmidt%26aufirst%3DB.%26atitle%3DDOCKTITE-a%2520highly%2520versatile%2520step-by-step%2520workflow%2520for%2520covalent%2520docking%2520and%2520virtual%2520screening%2520in%2520the%2520molecular%2520operating%2520environment%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D398%26epage%3D406%26doi%3D10.1021%2Fci500681r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref183"><div class="reference"><strong class="refLabel"><a href="#ref183" class="rightTabRefNumLink">183</a></strong><div class="NLM_citation" id="rightTab-cit183"><span><span class="NLM_contrib-group">Schmidt, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Welker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rieger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sahu, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sotriffer, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schirmeister, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engels, B.</span><span> </span><span class="NLM_article-title">Protocol for rational design of covalently interacting inhibitors</span> <span class="citation_source-journal">ChemPhysChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">3226</span><span class="NLM_x">–</span> <span class="NLM_lpage">3235</span><span class="refDoi"> DOI: 10.1002/cphc.201402542</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fcphc.201402542" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25251382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslCisLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=3226-3235&author=T.+C.+Schmidtauthor=A.+Welkerauthor=M.+Riegerauthor=P.+K.+Sahuauthor=C.+A.+Sotrifferauthor=T.+Schirmeisterauthor=B.+Engels&title=Protocol+for+rational+design+of+covalently+interacting+inhibitors&doi=10.1002%2Fcphc.201402542"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit183R"><div class="casContent"><span class="casTitleNuber">183</span><div class="casTitle"><span class="NLM_cas:atitle">Protocol for Rational Design of Covalently Interacting Inhibitors</span></div><div class="casAuthors">Schmidt, Thomas C.; Welker, Armin; Rieger, Max; Sahu, Prabhat K.; Sotriffer, Christoph A.; Schirmeister, Tanja; Engels, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">ChemPhysChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3226-3235</span>CODEN:
                <span class="NLM_cas:coden">CPCHFT</span>;
        ISSN:<span class="NLM_cas:issn">1439-4235</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The inhibition potencies of covalent inhibitors mainly result from the formation of a covalent bond to the enzyme during the inhibition mechanism.  This class of inhibitors has essentially been ignored in previous target-directed drug discovery projects because of concerns about possible side effects.  However, their advantages, such as higher binding energies and longer drug-target residence times moved them into the focus of recent investigations.  While the rational design of non-covalent inhibitors became std. the corresponding design of covalent inhibitors is still in its early stages.  Potent covalent inhibitors can be retrieved from large compd. libraries by covalent docking approaches but protocols are missing that can reliably predict the influence of variations in the substitution pattern on the affinity and/or reactivity of a given covalent inhibitor.  Hence, the wanted property profile can only be obtained from trial-and-error proceedings.  This paper presents an appropriate protocol which is able to predict improved covalent inhibitors.  It uses hybrid approaches, which mix quantum mech. (QM) and mol. mech. (MM) methods to predict variations in the reactivity of the inhibitor.  They are also used to compute the required information about the non-covalent enzyme-inhibitor complex.  Docking tools are employed to improve the inhibitor with respect to the non-covalent interactions formed in the binding site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodO7ngSMXNtLVg90H21EOLACvtfcHk0lj6BlDtKp1Ylw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslCisLnP&md5=8d5389e92ef83e2d7f90c48a4bea5103</span></div><a href="/servlet/linkout?suffix=cit183&amp;dbid=16384&amp;doi=10.1002%2Fcphc.201402542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcphc.201402542%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DT.%2BC.%26aulast%3DWelker%26aufirst%3DA.%26aulast%3DRieger%26aufirst%3DM.%26aulast%3DSahu%26aufirst%3DP.%2BK.%26aulast%3DSotriffer%26aufirst%3DC.%2BA.%26aulast%3DSchirmeister%26aufirst%3DT.%26aulast%3DEngels%26aufirst%3DB.%26atitle%3DProtocol%2520for%2520rational%2520design%2520of%2520covalently%2520interacting%2520inhibitors%26jtitle%3DChemPhysChem%26date%3D2014%26volume%3D15%26spage%3D3226%26epage%3D3235%26doi%3D10.1002%2Fcphc.201402542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref184"><div class="reference"><strong class="refLabel"><a href="#ref184" class="rightTabRefNumLink">184</a></strong><div class="NLM_citation" id="rightTab-cit184"><span><span class="NLM_contrib-group">Dong, G. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calhoun, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalyanaraman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Branch, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mashiyama, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">London, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babbitt, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Prediction of substrates for glutathione transferases by covalent docking</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1687</span><span class="NLM_x">–</span> <span class="NLM_lpage">1699</span><span class="refDoi"> DOI: 10.1021/ci5001554</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci5001554" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref184/cit184&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnsVehuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1687-1699&author=G.+Q.+Dongauthor=S.+Calhounauthor=H.+Fanauthor=C.+Kalyanaramanauthor=M.+C.+Branchauthor=S.+T.+Mashiyamaauthor=N.+Londonauthor=M.+P.+Jacobsonauthor=P.+C.+Babbittauthor=B.+K.+Shoichetauthor=R.+N.+Armstrongauthor=A.+Sali&title=Prediction+of+substrates+for+glutathione+transferases+by+covalent+docking&doi=10.1021%2Fci5001554"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit184R"><div class="casContent"><span class="casTitleNuber">184</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of Substrates for Glutathione Transferases by Covalent Docking</span></div><div class="casAuthors">Dong, Guang Qiang; Calhoun, Sara; Fan, Hao; Kalyanaraman, Chakrapani; Branch, Megan C.; Mashiyama, Susan T.; London, Nir; Jacobson, Matthew P.; Babbitt, Patricia C.; Shoichet, Brian K.; Armstrong, Richard N.; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1687-1699</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzymes in the glutathione transferase (GST) superfamily catalyze the conjugation of glutathione (GSH) to electrophilic substrates.  As a consequence they are involved in a no. of key biol. processes, including protection of cells against chem. damage, steroid and prostaglandin biosynthesis, tyrosine catabolism, and cell apoptosis.  Although virtual screening has been used widely to discover substrates by docking potential noncovalent ligands into active site clefts of enzymes, docking has been rarely constrained by a covalent bond between the enzyme and ligand.  In this study, we investigate the accuracy of docking poses and substrate discovery in the GST superfamily, by docking 6738 potential ligands from the KEGG and MetaCyc compd. libraries into 14 representative GST enzymes with known structures and substrates using the PLOP program [Jacobson Proteins 2004, 55, 351].  For X-ray structures as receptors, one of the top 3 ranked models is within 3 Å all-atom root mean square deviation (RMSD) of the native complex in 11 of the 14 cases; the enrichment LogAUC value is better than random in all cases, and better than 25 in 7 of 11 cases.  For comparative models as receptors, near-native ligand-enzyme configurations are often sampled but difficult to rank highly.  For models based on templates with the highest sequence identity, the enrichment LogAUC is better than 25 in 5 of 11 cases, not significantly different from the crystal structures.  In conclusion, we show that covalent docking can be a useful tool for substrate discovery and point out specific challenges for future method improvement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1kupr30YhIbVg90H21EOLACvtfcHk0lj6BlDtKp1Ylw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnsVehuro%253D&md5=b24a9f0d0a51fcd65d66797e3f9688aa</span></div><a href="/servlet/linkout?suffix=cit184&amp;dbid=16384&amp;doi=10.1021%2Fci5001554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci5001554%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DG.%2BQ.%26aulast%3DCalhoun%26aufirst%3DS.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DKalyanaraman%26aufirst%3DC.%26aulast%3DBranch%26aufirst%3DM.%2BC.%26aulast%3DMashiyama%26aufirst%3DS.%2BT.%26aulast%3DLondon%26aufirst%3DN.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DBabbitt%26aufirst%3DP.%2BC.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DArmstrong%26aufirst%3DR.%2BN.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DPrediction%2520of%2520substrates%2520for%2520glutathione%2520transferases%2520by%2520covalent%2520docking%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1687%26epage%3D1699%26doi%3D10.1021%2Fci5001554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref185"><div class="reference"><strong class="refLabel"><a href="#ref185" class="rightTabRefNumLink">185</a></strong><div class="NLM_citation" id="rightTab-cit185"><span><span class="NLM_contrib-group">Zhang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ji, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, Y.</span><span> </span><span class="NLM_article-title">Effective virtual screening strategy toward covalent ligands: identification of novel NEDD8-activating enzyme inhibitors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1785</span><span class="NLM_x">–</span> <span class="NLM_lpage">1797</span><span class="refDoi"> DOI: 10.1021/ci5002058</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci5002058" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXosFaitLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1785-1797&author=S.+Zhangauthor=J.+Tanauthor=Z.+Laiauthor=Y.+Liauthor=J.+Pangauthor=J.+Xiaoauthor=Z.+Huangauthor=Y.+Zhangauthor=H.+Jiauthor=Y.+Lai&title=Effective+virtual+screening+strategy+toward+covalent+ligands%3A+identification+of+novel+NEDD8-activating+enzyme+inhibitors&doi=10.1021%2Fci5002058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit185R"><div class="casContent"><span class="casTitleNuber">185</span><div class="casTitle"><span class="NLM_cas:atitle">Effective Virtual Screening Strategy toward Covalent Ligands: Identification of Novel NEDD8-Activating Enzyme Inhibitors</span></div><div class="casAuthors">Zhang, Shengping; Tan, Jiani; Lai, Zhonghui; Li, Ying; Pang, Junxia; Xiao, Jianhu; Huang, Zhangjian; Zhang, Yihua; Ji, Hui; Lai, Yisheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1785-1797</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The NEDD8-activating enzyme (NAE) is an emerging target for cancer therapy, which regulates the degrdn. and turnover of a variety of cancer-related proteins by activating the cullin-RING E3 ubiquitin ligases.  Among a limited no. of known NAE inhibitors, the covalent inhibitors have demonstrated the most potent efficacy through their covalently linked adducts with NEDD8.  Inspired by this unique mechanism, in this study, a novel combined strategy of virtual screening (VS) was adopted with the aim to identify diverse covalent inhibitors of NAE.  To be specific, a docking-enabled pharmacophore model was first built from the possible active conformations of chosen covalent inhibitors.  Meanwhile, a dynamic structure-based phamacophore was also established based on the snapshots derived from mol. dynamic simulation.  Subsequent screening of a focused ZINC database using these pharmacophore models combined with covalent docking discovered three novel active compds.  Among them, compd. LZ3 exhibited the most potent NAE inhibitory activity with an IC50 value of 1.06 ± 0.18 μM.  Furthermore, a cell-based washout expt. proved the proposed covalent binding mechanism for compd. LZ3, which confirmed the successful application of our combined VS strategy, indicating it may provide a viable soln. to systematically discover novel covalent ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhRJSduwx3CrVg90H21EOLACvtfcHk0liIoecPFsxhJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXosFaitLw%253D&md5=07a5cf406764b37d2f68b8aba55649eb</span></div><a href="/servlet/linkout?suffix=cit185&amp;dbid=16384&amp;doi=10.1021%2Fci5002058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci5002058%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DTan%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DPang%26aufirst%3DJ.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJi%26aufirst%3DH.%26aulast%3DLai%26aufirst%3DY.%26atitle%3DEffective%2520virtual%2520screening%2520strategy%2520toward%2520covalent%2520ligands%253A%2520identification%2520of%2520novel%2520NEDD8-activating%2520enzyme%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1785%26epage%3D1797%26doi%3D10.1021%2Fci5002058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref186"><div class="reference"><strong class="refLabel"><a href="#ref186" class="rightTabRefNumLink">186</a></strong><div class="NLM_citation" id="rightTab-cit186"><span><span class="NLM_contrib-group">Fakhouri, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Elimat, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurst, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reggio, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearce, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oberlies, N. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Croatt, M. P.</span><span> </span><span class="NLM_article-title">Isolation, semisynthesis, covalent docking and transforming growth factor beta-activated kinase 1 (TAK1)-inhibitory activities of (5Z)-7-oxozeaenol analogues</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">6993</span><span class="NLM_x">–</span> <span class="NLM_lpage">6999</span><span class="refDoi"> DOI: 10.1016/j.bmc.2015.09.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref186/cit186&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmc.2015.09.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref186/cit186&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26481152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref186/cit186&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Wjs7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=6993-6999&author=L.+Fakhouriauthor=T.+El-Elimatauthor=D.+P.+Hurstauthor=P.+H.+Reggioauthor=C.+J.+Pearceauthor=N.+H.+Oberliesauthor=M.+P.+Croatt&title=Isolation%2C+semisynthesis%2C+covalent+docking+and+transforming+growth+factor+beta-activated+kinase+1+%28TAK1%29-inhibitory+activities+of+%285Z%29-7-oxozeaenol+analogues&doi=10.1016%2Fj.bmc.2015.09.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit186R"><div class="casContent"><span class="casTitleNuber">186</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation, semisynthesis, covalent docking and transforming growth factor beta-activated kinase 1 (TAK1)-inhibitory activities of (5Z)-7-oxozeaenol analogues</span></div><div class="casAuthors">Fakhouri, Lara; El-Elimat, Tamam; Hurst, Dow P.; Reggio, Patricia H.; Pearce, Cedric J.; Oberlies, Nicholas H.; Croatt, Mitchell P.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6993-6999</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">(5Z)-7-Oxozeanol and related analogs were isolated and screened to explore their activity as TAK1 inhibitors.  Seven analogs were synthesized and more than a score of natural products isolated that examd. the role that different areas of the mol. contribute to TAK1 inhibition.  A novel nonarom. difluoro-deriv. was synthesized that had similar potency compared to the lead.  This is the first example of a nonarom. compd. in this class to have TAK1 inhibition.  Covalent docking for the isolated and synthesized analogs was carried out and a strong correlation between the obsd. activities and the calcd. binding was found.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJxKqASB5kirVg90H21EOLACvtfcHk0liIoecPFsxhJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Wjs7vO&md5=b5b32bf90eeb6594dabfa738e874e26a</span></div><a href="/servlet/linkout?suffix=cit186&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.09.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.09.037%26sid%3Dliteratum%253Aachs%26aulast%3DFakhouri%26aufirst%3DL.%26aulast%3DEl-Elimat%26aufirst%3DT.%26aulast%3DHurst%26aufirst%3DD.%2BP.%26aulast%3DReggio%26aufirst%3DP.%2BH.%26aulast%3DPearce%26aufirst%3DC.%2BJ.%26aulast%3DOberlies%26aufirst%3DN.%2BH.%26aulast%3DCroatt%26aufirst%3DM.%2BP.%26atitle%3DIsolation%252C%2520semisynthesis%252C%2520covalent%2520docking%2520and%2520transforming%2520growth%2520factor%2520beta-activated%2520kinase%25201%2520%2528TAK1%2529-inhibitory%2520activities%2520of%2520%25285Z%2529-7-oxozeaenol%2520analogues%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D6993%26epage%3D6999%26doi%3D10.1016%2Fj.bmc.2015.09.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref187"><div class="reference"><strong class="refLabel"><a href="#ref187" class="rightTabRefNumLink">187</a></strong><div class="NLM_citation" id="rightTab-cit187"><span><span class="NLM_contrib-group">Li, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">You, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span> </span><span class="NLM_article-title">Discovery of novel covalent proteasome inhibitors through a combination of pharmacophore screening, covalent docking, and molecular dynamics simulations</span> <span class="citation_source-journal">J. Mol. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">2515</span><span class="refDoi"> DOI: 10.1007/s00894-014-2515-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref187/cit187&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs00894-014-2515-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref187/cit187&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25394401" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref187/cit187&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A280%3ADC%252BC2M3ot1WmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=2515&author=A.+Liauthor=H.+Sunauthor=L.+Duauthor=X.+Wuauthor=J.+Caoauthor=Q.+Youauthor=Y.+Li&title=Discovery+of+novel+covalent+proteasome+inhibitors+through+a+combination+of+pharmacophore+screening%2C+covalent+docking%2C+and+molecular+dynamics+simulations&doi=10.1007%2Fs00894-014-2515-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit187R"><div class="casContent"><span class="casTitleNuber">187</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel covalent proteasome inhibitors through a combination of pharmacophore screening, covalent docking, and molecular dynamics simulations</span></div><div class="casAuthors">Li Aibo; Sun Haopeng; Du Lei; Wu Xiaoxin; Cao Jianqin; You Qidong; Li Yuyan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2515</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The ubiquitin-proteasome pathway plays a pivotal role in the regulation of cellular protein processing and degradation.  Proteasome inhibitors (PIs) have enormous potential to treat multiple myeloma, solid tumors, parasites, inflammation, and immune diseases, which is spurring the development of new types of PIs with enhanced efficacy, fewer side effects, and reduced drug resistance.  Nevertheless, virtual screening for covalent PIs has rarely been reported because calculating the covalent binding energy is a challenging task.  The aim of this study was to discover new covalent inhibitors of the 20S proteasome.  The structures of PIs were manually divided into two parts: a noncovalent binding part resulting from virtual screening, and an epoxyketone group that was pre-selected as a covalent binding part.  The SPECS database was screened by noncovalent docking and a pharmacophore model built with the 20S proteasome.  After validating the covalent conjugation, 88 hits with epoxyketone were covalently docked into the 20S proteasome to analyze the intermolecular interactions.  Four compounds were selected after multiple filtration and validations.  Molecular dynamics simulations were performed to check the stability of the noncovalent and covalent docked ligand-enzyme complexes and investigate the interaction patterns of the screened inhibitors.  Finally, two compounds with novel aromatic backbones, reasonable interactions, and stable covalent binding modes were retained.  These compounds can serve as potential hits for further biological evaluation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRD0fkfwqKVeVHFYivNCdPRfW6udTcc2eZitHN8qgIvlLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3ot1WmtA%253D%253D&md5=975a5c6ef3c6511c0c863a736fd05b4a</span></div><a href="/servlet/linkout?suffix=cit187&amp;dbid=16384&amp;doi=10.1007%2Fs00894-014-2515-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00894-014-2515-y%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DA.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DDu%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DYou%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520novel%2520covalent%2520proteasome%2520inhibitors%2520through%2520a%2520combination%2520of%2520pharmacophore%2520screening%252C%2520covalent%2520docking%252C%2520and%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Mol.%2520Model.%26date%3D2014%26volume%3D20%26spage%3D2515%26doi%3D10.1007%2Fs00894-014-2515-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref188"><div class="reference"><strong class="refLabel"><a href="#ref188" class="rightTabRefNumLink">188</a></strong><div class="NLM_citation" id="rightTab-cit188"><span><span class="NLM_contrib-group">London, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farelli, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korczynska, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Obaidi, N. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babbitt, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almo, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Covalent docking predicts substrates for haloalkanoate dehalogenase superfamily phosphatases</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">528</span><span class="NLM_x">–</span> <span class="NLM_lpage">537</span><span class="refDoi"> DOI: 10.1021/bi501140k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi501140k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=528-537&author=N.+Londonauthor=J.+D.+Farelliauthor=S.+D.+Brownauthor=C.+Liuauthor=H.+Huangauthor=M.+Korczynskaauthor=N.+F.+Al-Obaidiauthor=P.+C.+Babbittauthor=S.+C.+Almoauthor=K.+N.+Allenauthor=B.+K.+Shoichet&title=Covalent+docking+predicts+substrates+for+haloalkanoate+dehalogenase+superfamily+phosphatases&doi=10.1021%2Fbi501140k"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit188&amp;dbid=16384&amp;doi=10.1021%2Fbi501140k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi501140k%26sid%3Dliteratum%253Aachs%26aulast%3DLondon%26aufirst%3DN.%26aulast%3DFarelli%26aufirst%3DJ.%2BD.%26aulast%3DBrown%26aufirst%3DS.%2BD.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DKorczynska%26aufirst%3DM.%26aulast%3DAl-Obaidi%26aufirst%3DN.%2BF.%26aulast%3DBabbitt%26aufirst%3DP.%2BC.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DAllen%26aufirst%3DK.%2BN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DCovalent%2520docking%2520predicts%2520substrates%2520for%2520haloalkanoate%2520dehalogenase%2520superfamily%2520phosphatases%26jtitle%3DBiochemistry%26date%3D2015%26volume%3D54%26spage%3D528%26epage%3D537%26doi%3D10.1021%2Fbi501140k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref189"><div class="reference"><strong class="refLabel"><a href="#ref189" class="rightTabRefNumLink">189</a></strong><div class="NLM_citation" id="rightTab-cit189"><span><span class="NLM_contrib-group">Webb, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lasker, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneidman-Duhovny, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tjioe, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velazquez-Muriel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Modeling of proteins and their assemblies with the integrative modeling platform</span> <span class="citation_source-journal">Methods Mol. Biol. (N. Y., NY, U. S.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">781</span><span class="NLM_x">, </span> <span class="NLM_fpage">377</span><span class="NLM_x">–</span> <span class="NLM_lpage">397</span><span class="refDoi"> DOI: 10.1007/978-1-61779-276-2_19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref189/cit189&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2F978-1-61779-276-2_19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref189/cit189&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=21877292" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref189/cit189&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVClsbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=781&publication_year=2011&pages=377-397&author=B.+Webbauthor=K.+Laskerauthor=D.+Schneidman-Duhovnyauthor=E.+Tjioeauthor=J.+Phillipsauthor=S.+J.+Kimauthor=J.+Velazquez-Murielauthor=D.+Russelauthor=A.+Sali&title=Modeling+of+proteins+and+their+assemblies+with+the+integrative+modeling+platform&doi=10.1007%2F978-1-61779-276-2_19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit189R"><div class="casContent"><span class="casTitleNuber">189</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling of proteins and their assemblies with the integrative modeling platform</span></div><div class="casAuthors">Webb, Benjamin; Lasker, Keren; Schneidman-Duhovny, Dina; Tjioe, Elina; Phillips, Jeremy; Kim, Seung Joong; Velazquez-Muriel, Javier; Russel, Daniel; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">781</span>
        (<span class="NLM_cas:issue">Network Biology</span>),
    <span class="NLM_cas:pages">377-397</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">To understand the workings of the living cell, we need to characterize protein assemblies that constitute the cell (for example, the ribosome, 26S proteasome, and the nuclear pore complex).  A reliable high-resoln. structural characterization of these assemblies is frequently beyond the reach of current exptl. methods, such as X-ray crystallog., NMR spectroscopy, electron microscopy, footprinting, chem. crosslinking, FRET spectroscopy, small-angle X-ray scattering, and proteomics.  However, the information garnered from different methods can be combined and used to build computational models of the assembly structures that are consistent with all of the available datasets.  Here, we describe a protocol for this integration, whereby the information is converted to a set of spatial restraints and a variety of optimization procedures can be used to generate models that satisfy the restraints as much as possible.  These generated models can then potentially inform about the precision and accuracy of structure detn., the accuracy of the input datasets, and further data generation.  We also demonstrate the Integrative Modeling Platform (IMP) software, which provides the necessary computational framework to implement this protocol, and several applications for specific-use cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoOF9UyKyz3rVg90H21EOLACvtfcHk0lgzOOizSG4DWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVClsbrM&md5=84ee7542a032775ef6591dcdf7d43ef4</span></div><a href="/servlet/linkout?suffix=cit189&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-276-2_19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-276-2_19%26sid%3Dliteratum%253Aachs%26aulast%3DWebb%26aufirst%3DB.%26aulast%3DLasker%26aufirst%3DK.%26aulast%3DSchneidman-Duhovny%26aufirst%3DD.%26aulast%3DTjioe%26aufirst%3DE.%26aulast%3DPhillips%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.%2BJ.%26aulast%3DVelazquez-Muriel%26aufirst%3DJ.%26aulast%3DRussel%26aufirst%3DD.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DModeling%2520of%2520proteins%2520and%2520their%2520assemblies%2520with%2520the%2520integrative%2520modeling%2520platform%26jtitle%3DMethods%2520Mol.%2520Biol.%2520%2528N.%2520Y.%252C%2520NY%252C%2520U.%2520S.%2529%26date%3D2011%26volume%3D781%26spage%3D377%26epage%3D397%26doi%3D10.1007%2F978-1-61779-276-2_19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref190"><div class="reference"><strong class="refLabel"><a href="#ref190" class="rightTabRefNumLink">190</a></strong><div class="NLM_citation" id="rightTab-cit190"><span><span class="NLM_contrib-group">Evers, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Successful virtual screening for a submicromolar antagonist of the neurokinin-1 receptor based on a ligand-supported homology model</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">5381</span><span class="NLM_x">–</span> <span class="NLM_lpage">5392</span><span class="refDoi"> DOI: 10.1021/jm0311487</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0311487" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5381-5392&author=A.+Eversauthor=G.+Klebe&title=Successful+virtual+screening+for+a+submicromolar+antagonist+of+the+neurokinin-1+receptor+based+on+a+ligand-supported+homology+model&doi=10.1021%2Fjm0311487"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit190&amp;dbid=16384&amp;doi=10.1021%2Fjm0311487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0311487%26sid%3Dliteratum%253Aachs%26aulast%3DEvers%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DSuccessful%2520virtual%2520screening%2520for%2520a%2520submicromolar%2520antagonist%2520of%2520the%2520neurokinin-1%2520receptor%2520based%2520on%2520a%2520ligand-supported%2520homology%2520model%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5381%26epage%3D5392%26doi%3D10.1021%2Fjm0311487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref191"><div class="reference"><strong class="refLabel"><a href="#ref191" class="rightTabRefNumLink">191</a></strong><div class="NLM_citation" id="rightTab-cit191"><span><span class="NLM_contrib-group">Evers, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Ligand-supported homology modeling of g-protein-coupled receptor sites: models sufficient for successful virtual screening</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">248</span><span class="NLM_x">–</span> <span class="NLM_lpage">251</span><span class="refDoi"> DOI: 10.1002/anie.200352776</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref191/cit191&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fanie.200352776" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2004&pages=248-251&author=A.+Eversauthor=G.+Klebe&title=Ligand-supported+homology+modeling+of+g-protein-coupled+receptor+sites%3A+models+sufficient+for+successful+virtual+screening&doi=10.1002%2Fanie.200352776"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit191&amp;dbid=16384&amp;doi=10.1002%2Fanie.200352776&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200352776%26sid%3Dliteratum%253Aachs%26aulast%3DEvers%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DLigand-supported%2520homology%2520modeling%2520of%2520g-protein-coupled%2520receptor%2520sites%253A%2520models%2520sufficient%2520for%2520successful%2520virtual%2520screening%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2004%26volume%3D43%26spage%3D248%26epage%3D251%26doi%3D10.1002%2Fanie.200352776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref192"><div class="reference"><strong class="refLabel"><a href="#ref192" class="rightTabRefNumLink">192</a></strong><div class="NLM_citation" id="rightTab-cit192"><span><span class="NLM_contrib-group">Zhao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vetting, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonanno, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillerich, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seidel, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babbitt, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almo, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweedler, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlt, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cronan, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Discovery of new enzymes and metabolic pathways by using structure and genome context</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">502</span><span class="NLM_x">, </span> <span class="NLM_fpage">698</span><span class="NLM_x">–</span> <span class="NLM_lpage">702</span><span class="refDoi"> DOI: 10.1038/nature12576</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnature12576" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24056934" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFSls73J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=502&publication_year=2013&pages=698-702&author=S.+Zhaoauthor=R.+Kumarauthor=A.+Sakaiauthor=M.+W.+Vettingauthor=B.+M.+Woodauthor=S.+Brownauthor=J.+B.+Bonannoauthor=B.+S.+Hillerichauthor=R.+D.+Seidelauthor=P.+C.+Babbittauthor=S.+C.+Almoauthor=J.+V.+Sweedlerauthor=J.+A.+Gerltauthor=J.+E.+Cronanauthor=M.+P.+Jacobson&title=Discovery+of+new+enzymes+and+metabolic+pathways+by+using+structure+and+genome+context&doi=10.1038%2Fnature12576"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit192R"><div class="casContent"><span class="casTitleNuber">192</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of new enzymes and metabolic pathways by using structure and genome context</span></div><div class="casAuthors">Zhao, Suwen; Kumar, Ritesh; Sakai, Ayano; Vetting, Matthew W.; Wood, B. McKay; Brown, Shoshana; Bonanno, Jeffery B.; Hillerich, Brandan S.; Seidel, Ronald D.; Babbitt, Patricia C.; Almo, Steven C.; Sweedler, Jonathan V.; Gerlt, John A.; Cronan, John E.; Jacobson, Matthew P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">502</span>
        (<span class="NLM_cas:issue">7473</span>),
    <span class="NLM_cas:pages">698-702</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Assigning valid functions to proteins identified in genome projects is challenging: overprediction and database annotation errors are the principal concerns.  We and others are developing computation-guided strategies for functional discovery with metabolite docking' to exptl. derived or homol.-based three-dimensional structures.  Bacterial metabolic pathways often are encoded by genome neighborhoods' (gene clusters and/or operons), which can provide important clues for functional assignment.  We recently demonstrated the synergy of docking and pathway context by predicting' the intermediates in the glycolytic pathway in Escherichia coli.  Metabolite docking to multiple binding proteins and enzymes in the same pathway increases the reliability of in silico predictions of substrate specificities because the pathway intermediates are structurally similar.  Here we report that structure-guided approaches for predicting the substrate specificities of several enzymes encoded by a bacterial gene cluster allowed the correct prediction of the in vitro activity of a structurally characterized enzyme of unknown function (PDB 2PMQ), 2-epimerization of trans-4-hydroxy-L-proline betaine (tHyp-B) and cis-4-hydroxy-D-proline betaine (cHyp-B), and also the correct identification of the catabolic pathway in which Hyp-B 2-epimerase participates.  The substrate-liganded pose predicted by virtual library screening (docking) was confirmed exptl.  The enzymic activities in the predicted pathway were confirmed by in vitro assays and genetic analyses; the intermediates were identified by metabolomics; and repression of the genes encoding the pathway by high salt concns. was established by transcriptomics, confirming the osmolyte role of tHyp-B.  This study establishes the utility of structure-guided functional predictions to enable the discovery of new metabolic pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzzoHQvsXyDbVg90H21EOLACvtfcHk0lgzOOizSG4DWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFSls73J&md5=225ec9508efaab563b2c5d1dd8552c0a</span></div><a href="/servlet/linkout?suffix=cit192&amp;dbid=16384&amp;doi=10.1038%2Fnature12576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12576%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DKumar%26aufirst%3DR.%26aulast%3DSakai%26aufirst%3DA.%26aulast%3DVetting%26aufirst%3DM.%2BW.%26aulast%3DWood%26aufirst%3DB.%2BM.%26aulast%3DBrown%26aufirst%3DS.%26aulast%3DBonanno%26aufirst%3DJ.%2BB.%26aulast%3DHillerich%26aufirst%3DB.%2BS.%26aulast%3DSeidel%26aufirst%3DR.%2BD.%26aulast%3DBabbitt%26aufirst%3DP.%2BC.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DSweedler%26aufirst%3DJ.%2BV.%26aulast%3DGerlt%26aufirst%3DJ.%2BA.%26aulast%3DCronan%26aufirst%3DJ.%2BE.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DDiscovery%2520of%2520new%2520enzymes%2520and%2520metabolic%2520pathways%2520by%2520using%2520structure%2520and%2520genome%2520context%26jtitle%3DNature%26date%3D2013%26volume%3D502%26spage%3D698%26epage%3D702%26doi%3D10.1038%2Fnature12576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref193"><div class="reference"><strong class="refLabel"><a href="#ref193" class="rightTabRefNumLink">193</a></strong><div class="NLM_citation" id="rightTab-cit193"><span><span class="NLM_contrib-group">Kalyanaraman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imker, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, E. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glasner, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babbitt, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almo, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlt, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Discovery of a dipeptide epimerase enzymatic function guided by homology modeling and virtual screening</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1668</span><span class="NLM_x">–</span> <span class="NLM_lpage">1677</span><span class="refDoi"> DOI: 10.1016/j.str.2008.08.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref193/cit193&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.str.2008.08.015" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=1668-1677&author=C.+Kalyanaramanauthor=H.+J.+Imkerauthor=A.+A.+Fedorovauthor=E.+V.+Fedorovauthor=M.+E.+Glasnerauthor=P.+C.+Babbittauthor=S.+C.+Almoauthor=J.+A.+Gerltauthor=M.+P.+Jacobson&title=Discovery+of+a+dipeptide+epimerase+enzymatic+function+guided+by+homology+modeling+and+virtual+screening&doi=10.1016%2Fj.str.2008.08.015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit193&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2008.08.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2008.08.015%26sid%3Dliteratum%253Aachs%26aulast%3DKalyanaraman%26aufirst%3DC.%26aulast%3DImker%26aufirst%3DH.%2BJ.%26aulast%3DFedorov%26aufirst%3DA.%2BA.%26aulast%3DFedorov%26aufirst%3DE.%2BV.%26aulast%3DGlasner%26aufirst%3DM.%2BE.%26aulast%3DBabbitt%26aufirst%3DP.%2BC.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DGerlt%26aufirst%3DJ.%2BA.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DDiscovery%2520of%2520a%2520dipeptide%2520epimerase%2520enzymatic%2520function%2520guided%2520by%2520homology%2520modeling%2520and%2520virtual%2520screening%26jtitle%3DStructure%26date%3D2008%26volume%3D16%26spage%3D1668%26epage%3D1677%26doi%3D10.1016%2Fj.str.2008.08.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref194"><div class="reference"><strong class="refLabel"><a href="#ref194" class="rightTabRefNumLink">194</a></strong><div class="NLM_citation" id="rightTab-cit194"><span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitchcock, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seidel, R. D.,  2nd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillerich, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almo, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raushel, F. M.</span><span> </span><span class="NLM_article-title">Assignment of pterin deaminase activity to an enzyme of unknown function guided by homology modeling and docking</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">795</span><span class="NLM_x">–</span> <span class="NLM_lpage">803</span><span class="refDoi"> DOI: 10.1021/ja309680b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja309680b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref194/cit194&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVyiurjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=795-803&author=H.+Fanauthor=D.+S.+Hitchcockauthor=R.+D.+Seidelauthor=B.+Hillerichauthor=H.+Linauthor=S.+C.+Almoauthor=A.+Saliauthor=B.+K.+Shoichetauthor=F.+M.+Raushel&title=Assignment+of+pterin+deaminase+activity+to+an+enzyme+of+unknown+function+guided+by+homology+modeling+and+docking&doi=10.1021%2Fja309680b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit194R"><div class="casContent"><span class="casTitleNuber">194</span><div class="casTitle"><span class="NLM_cas:atitle">Assignment of Pterin Deaminase Activity to an Enzyme of Unknown Function Guided by Homology Modeling and Docking</span></div><div class="casAuthors">Fan, Hao; Hitchcock, Daniel S.; Seidel, Ronald D.; Hillerich, Brandan; Lin, Henry; Almo, Steven C.; Sali, Andrej; Shoichet, Brian K.; Raushel, Frank M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">795-803</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Of the over 22 million protein sequences in the nonredundant TrEMBL database, fewer than 1% have exptl. confirmed functions.  Structure-based methods have been used to predict enzyme activities from exptl. detd. structures; however, for the vast majority of proteins, no such structures are available.  Here, homol. models of a functionally uncharacterized amidohydrolase from Agrobacterium radiobacter K84 (Arad3529) were computed on the basis of a remote template structure.  The protein backbone of two loops near the active site was remodeled, resulting in four distinct active site conformations.  Substrates of Arad3529 were predicted by docking of 57 672 high-energy intermediate (HEI) forms of 6440 metabolites against these four homol. models.  On the basis of docking ranks and geometries, a set of modified pterins were suggested as candidate substrates for Arad3529.  The predictions were tested by enzymol. expts., and Arad3529 deaminated many pterin metabolites (substrate, kcat/Km [M-1 s-1]): formylpterin, 5.2 × 106; pterin-6-carboxylate, 4.0 × 106; pterin-7-carboxylate, 3.7 × 106; pterin, 3.3 × 106; hydroxymethylpterin, 1.2 × 106; biopterin, 1.0 × 106; d-(+)-neopterin, 3.1 × 105; isoxanthopterin, 2.8 × 105; sepiapterin, 1.3 × 105; folate, 1.3 × 105, xanthopterin, 1.17 × 105; and 7,8-dihydrohydroxymethylpterin, 3.3 × 104.  While pterin is a ubiquitous oxidative product of folate degrdn., genomic anal. suggests that the first step of an undescribed pterin degrdn. pathway is catalyzed by Arad3529.  Homol. model-based virtual screening, esp. with modeling of protein backbone flexibility, may be broadly useful for enzyme function annotation and discovering new pathways and drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkH7Qij21G67Vg90H21EOLACvtfcHk0liFBqlNSPPwDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVyiurjJ&md5=7e500cb2222fa90af3a94d83527af5e6</span></div><a href="/servlet/linkout?suffix=cit194&amp;dbid=16384&amp;doi=10.1021%2Fja309680b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja309680b%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DH.%26aulast%3DHitchcock%26aufirst%3DD.%2BS.%26aulast%3DSeidel%26aufirst%3DR.%2BD.%26aulast%3DHillerich%26aufirst%3DB.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DRaushel%26aufirst%3DF.%2BM.%26atitle%3DAssignment%2520of%2520pterin%2520deaminase%2520activity%2520to%2520an%2520enzyme%2520of%2520unknown%2520function%2520guided%2520by%2520homology%2520modeling%2520and%2520docking%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D795%26epage%3D803%26doi%3D10.1021%2Fja309680b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref195"><div class="reference"><strong class="refLabel"><a href="#ref195" class="rightTabRefNumLink">195</a></strong><div class="NLM_citation" id="rightTab-cit195"><span><span class="NLM_contrib-group">Jensen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunner, H. R.</span><span> </span><span class="NLM_article-title">Aliskiren: the first renin inhibitor for clinical treatment</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">399</span><span class="NLM_x">–</span> <span class="NLM_lpage">410</span><span class="refDoi"> DOI: 10.1038/nrd2550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref195/cit195&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnrd2550" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref195/cit195&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=18340340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref195/cit195&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlt1agtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=399-410&author=C.+Jensenauthor=P.+Heroldauthor=H.+R.+Brunner&title=Aliskiren%3A+the+first+renin+inhibitor+for+clinical+treatment&doi=10.1038%2Fnrd2550"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit195R"><div class="casContent"><span class="casTitleNuber">195</span><div class="casTitle"><span class="NLM_cas:atitle">Aliskiren: the first renin inhibitor for clinical treatment</span></div><div class="casAuthors">Jensen, Chris; Herold, Peter; Brunner, Hans Rudolf</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">399-410</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The renin-angiotensin system, which is a key regulator of blood pressure, has been successfully targeted by several classes of drugs for the treatment of cardiovascular disease.  However, renin - the protease at the top of the cascade - although highly attractive as a target, has also proved to be challenging.  Here, we describe how these challenges were tackled in the discovery and development of aliskiren, the first renin inhibitor to be approved for clin. use.  The first evidence of the existence of renin was presented over 100 years ago.  However, the importance of renin and the renin-angiotensin system in the pathogenesis of cardiovascular disease was only fully realized in the 1970s.  It was another 20 years before the first inhibitors of renin were available for clin. research.  Here, we describe the discovery and development of aliskiren, an orally active renin inhibitor, which became the first drug in its class to receive regulatory approval.  In 2007, it was approved for the treatment of hypertension by the US Food and Drug Administration and the European Medicines Agency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwW8kGNQazBbVg90H21EOLACvtfcHk0liFBqlNSPPwDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlt1agtbg%253D&md5=891d7f8266f9b03954ae40e028a8cd42</span></div><a href="/servlet/linkout?suffix=cit195&amp;dbid=16384&amp;doi=10.1038%2Fnrd2550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2550%26sid%3Dliteratum%253Aachs%26aulast%3DJensen%26aufirst%3DC.%26aulast%3DHerold%26aufirst%3DP.%26aulast%3DBrunner%26aufirst%3DH.%2BR.%26atitle%3DAliskiren%253A%2520the%2520first%2520renin%2520inhibitor%2520for%2520clinical%2520treatment%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D399%26epage%3D410%26doi%3D10.1038%2Fnrd2550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref196"><div class="reference"><strong class="refLabel"><a href="#ref196" class="rightTabRefNumLink">196</a></strong><div class="NLM_citation" id="rightTab-cit196"><span><span class="NLM_contrib-group">Barelier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sterling, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Meara, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">The recognition of identical ligands by unrelated proteins</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">2772</span><span class="NLM_x">–</span> <span class="NLM_lpage">2784</span><span class="refDoi"> DOI: 10.1021/acschembio.5b00683</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00683" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref196/cit196&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFyitr7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=2772-2784&author=S.+Barelierauthor=T.+Sterlingauthor=M.+J.+O%E2%80%99Mearaauthor=B.+K.+Shoichet&title=The+recognition+of+identical+ligands+by+unrelated+proteins&doi=10.1021%2Facschembio.5b00683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit196R"><div class="casContent"><span class="casTitleNuber">196</span><div class="casTitle"><span class="NLM_cas:atitle">The recognition of identical ligands by unrelated proteins</span></div><div class="casAuthors">Barelier, Sarah; Sterling, Teague; O'Meara, Matthew J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2772-2784</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding of drugs and reagents to off-targets is well-known.  Whereas many off-targets are related to the primary target by sequence and fold, many ligands bind to unrelated pairs of proteins, and these are harder to anticipate.  If the binding site in the off-target can be related to that of the primary target, this challenge resolves into aligning the 2 pockets.  However, other cases are possible: the ligand might interact with entirely different residues and environments in the off-target, or wholly different ligand atoms may be implicated in the 2 complexes.  To investigate these scenarios at at. resoln., the structures of 59 ligands in 116 complexes (62 pairs in total), where the protein pairs were unrelated by fold but bound an identical ligand, were examd.  In almost half of the pairs, the ligand interacted with unrelated residues in the 2 proteins (29 pairs), and in 14 of the pairs wholly different ligand moieties were implicated in each complex.  Even in those 19 pairs of complexes that presented similar environments to the ligand, ligand superposition rarely resulted in the overlap of related residues.  There appears to be no single pattern-matching "code" for identifying binding sites in unrelated proteins that bind identical ligands, although modeling suggests that there might be a limited no. of different patterns that suffice to recognize different ligand functional groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa1_lQJDNMCrVg90H21EOLACvtfcHk0lhzl03YAm5NSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFyitr7J&md5=9d6ac31c2a58bc85f77e7eb34649c195</span></div><a href="/servlet/linkout?suffix=cit196&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00683%26sid%3Dliteratum%253Aachs%26aulast%3DBarelier%26aufirst%3DS.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DThe%2520recognition%2520of%2520identical%2520ligands%2520by%2520unrelated%2520proteins%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D2772%26epage%3D2784%26doi%3D10.1021%2Facschembio.5b00683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref197"><div class="reference"><strong class="refLabel"><a href="#ref197" class="rightTabRefNumLink">197</a></strong><div class="NLM_citation" id="rightTab-cit197"><span><span class="NLM_contrib-group">Tondi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morandi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Structure-based optimization of a non-beta-lactam lead results in inhibitors that do not up-regulate beta-lactamase expression in cell culture</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">127</span><span class="NLM_x">, </span> <span class="NLM_fpage">4632</span><span class="NLM_x">–</span> <span class="NLM_lpage">4639</span><span class="refDoi"> DOI: 10.1021/ja042984o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja042984o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2005&pages=4632-4639&author=D.+Tondiauthor=F.+Morandiauthor=R.+Bonnetauthor=M.+P.+Costiauthor=B.+K.+Shoichet&title=Structure-based+optimization+of+a+non-beta-lactam+lead+results+in+inhibitors+that+do+not+up-regulate+beta-lactamase+expression+in+cell+culture&doi=10.1021%2Fja042984o"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit197&amp;dbid=16384&amp;doi=10.1021%2Fja042984o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja042984o%26sid%3Dliteratum%253Aachs%26aulast%3DTondi%26aufirst%3DD.%26aulast%3DMorandi%26aufirst%3DF.%26aulast%3DBonnet%26aufirst%3DR.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520optimization%2520of%2520a%2520non-beta-lactam%2520lead%2520results%2520in%2520inhibitors%2520that%2520do%2520not%2520up-regulate%2520beta-lactamase%2520expression%2520in%2520cell%2520culture%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D127%26spage%3D4632%26epage%3D4639%26doi%3D10.1021%2Fja042984o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref198"><div class="reference"><strong class="refLabel"><a href="#ref198" class="rightTabRefNumLink">198</a></strong><div class="NLM_citation" id="rightTab-cit198"><span><span class="NLM_contrib-group">Hermann, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marti-Arbona, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almo, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raushel, F. M.</span><span> </span><span class="NLM_article-title">Structure-based activity prediction for an enzyme of unknown function</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">448</span><span class="NLM_x">, </span> <span class="NLM_fpage">775</span><span class="NLM_x">–</span> <span class="NLM_lpage">779</span><span class="refDoi"> DOI: 10.1038/nature05981</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnature05981" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=17603473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD2sXptVGrs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=448&publication_year=2007&pages=775-779&author=J.+C.+Hermannauthor=R.+Marti-Arbonaauthor=A.+A.+Fedorovauthor=E.+Fedorovauthor=S.+C.+Almoauthor=B.+K.+Shoichetauthor=F.+M.+Raushel&title=Structure-based+activity+prediction+for+an+enzyme+of+unknown+function&doi=10.1038%2Fnature05981"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit198R"><div class="casContent"><span class="casTitleNuber">198</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based activity prediction for an enzyme of unknown function</span></div><div class="casAuthors">Hermann, Johannes C.; Marti-Arbona, Ricardo; Fedorov, Alexander A.; Fedorov, Elena; Almo, Steven C.; Shoichet, Brian K.; Raushel, Frank M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">448</span>
        (<span class="NLM_cas:issue">7155</span>),
    <span class="NLM_cas:pages">775-779</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">With many genomes sequenced, a pressing challenge in biol. is predicting the function of the proteins that the genes encode.  When proteins are unrelated to others of known activity, bioinformatics inference for function becomes problematic.  It would thus be useful to interrogate protein structures for function directly.  Here, we predict the function of an enzyme of unknown activity, Tm0936 from Thermotoga maritima, by docking high-energy intermediate forms of thousands of candidate metabolites.  The docking hit list was dominated by adenine analogs, which appeared to undergo C6-deamination.  Four of these, including 5-methylthioadenosine and S-adenosylhomocysteine (SAH), were tested as substrates, and three had substantial catalytic rate consts. (105 M-1 s-1).  The x-ray crystal structure of the complex between Tm0936 and the product resulting from the deamination of SAH, S-inosylhomocysteine, was detd., and it corresponded closely to the predicted structure.  The deaminated products can be further metabolized by T. maritima in a previously uncharacterized SAH degrdn. pathway.  Structure-based docking with high-energy forms of potential substrates may be a useful tool to annotate enzymes for function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomC_L0C9hsf7Vg90H21EOLACvtfcHk0lhzl03YAm5NSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXptVGrs7w%253D&md5=b0bd6612147fb848f49f8b9ccfd8384e</span></div><a href="/servlet/linkout?suffix=cit198&amp;dbid=16384&amp;doi=10.1038%2Fnature05981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05981%26sid%3Dliteratum%253Aachs%26aulast%3DHermann%26aufirst%3DJ.%2BC.%26aulast%3DMarti-Arbona%26aufirst%3DR.%26aulast%3DFedorov%26aufirst%3DA.%2BA.%26aulast%3DFedorov%26aufirst%3DE.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DRaushel%26aufirst%3DF.%2BM.%26atitle%3DStructure-based%2520activity%2520prediction%2520for%2520an%2520enzyme%2520of%2520unknown%2520function%26jtitle%3DNature%26date%3D2007%26volume%3D448%26spage%3D775%26epage%3D779%26doi%3D10.1038%2Fnature05981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref199"><div class="reference"><strong class="refLabel"><a href="#ref199" class="rightTabRefNumLink">199</a></strong><div class="NLM_citation" id="rightTab-cit199"><span><span class="NLM_contrib-group">Sali, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bycroft, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fersht, A. R.</span><span> </span><span class="NLM_article-title">Stabilization of protein structure by interaction of alpha-helix dipole with a charged side chain</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">335</span><span class="NLM_x">, </span> <span class="NLM_fpage">740</span><span class="NLM_x">–</span> <span class="NLM_lpage">743</span><span class="refDoi"> DOI: 10.1038/335740a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref199/cit199&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2F335740a0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=335&publication_year=1988&pages=740-743&author=D.+Saliauthor=M.+Bycroftauthor=A.+R.+Fersht&title=Stabilization+of+protein+structure+by+interaction+of+alpha-helix+dipole+with+a+charged+side+chain&doi=10.1038%2F335740a0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit199&amp;dbid=16384&amp;doi=10.1038%2F335740a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F335740a0%26sid%3Dliteratum%253Aachs%26aulast%3DSali%26aufirst%3DD.%26aulast%3DBycroft%26aufirst%3DM.%26aulast%3DFersht%26aufirst%3DA.%2BR.%26atitle%3DStabilization%2520of%2520protein%2520structure%2520by%2520interaction%2520of%2520alpha-helix%2520dipole%2520with%2520a%2520charged%2520side%2520chain%26jtitle%3DNature%26date%3D1988%26volume%3D335%26spage%3D740%26epage%3D743%26doi%3D10.1038%2F335740a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref200"><div class="reference"><strong class="refLabel"><a href="#ref200" class="rightTabRefNumLink">200</a></strong><div class="NLM_citation" id="rightTab-cit200"><span><span class="NLM_contrib-group">Eswar, N.; Sali, A.</span><span> </span><span class="NLM_article-title">Comparative modeling of drug target proteins</span>. In  <span class="citation_source-book">Comprehensive Medicinal Chemistry II</span>; <span class="NLM_contrib-group">Taylor, J. B.; Triggle, D. J.</span>, Eds.; <span class="NLM_publisher-name">Elsevier</span>,<span class="NLM_x"> </span><span class="NLM_year">2007</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">4</span>, pp  <span class="NLM_fpage">216</span><span class="NLM_x">–</span> <span class="NLM_lpage">231</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref200/cit200&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2FB0-08-045044-X%2F00251-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&pages=216-231&author=N.+Eswar&author=A.+Saliauthor=J.+B.+Taylor&author=D.+J.+Triggle&title=Comprehensive+Medicinal+Chemistry+II"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit200&amp;dbid=16384&amp;doi=10.1016%2FB0-08-045044-X%2F00251-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB0-08-045044-X%252F00251-0%26sid%3Dliteratum%253Aachs%26aulast%3DEswar%26aufirst%3DN.%26atitle%3DComparative%2520modeling%2520of%2520drug%2520target%2520proteins%26btitle%3DComprehensive%2520Medicinal%2520Chemistry%2520II%26aulast%3DTaylor%26aufirst%3DJ.%2BB.%26pub%3DElsevier%26date%3D2007%26volume%3D4%26spage%3D216%26epage%3D231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref201"><div class="reference"><strong class="refLabel"><a href="#ref201" class="rightTabRefNumLink">201</a></strong><div class="NLM_citation" id="rightTab-cit201"><span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, N.</span><span> </span><span class="NLM_article-title">Structure-based discovery of novel and selective 5-hydroxytryptamine 2B receptor antagonists for the treatment of irritable bowel syndrome</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">707</span><span class="NLM_x">–</span> <span class="NLM_lpage">720</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01631</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01631" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref201/cit201&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVygsb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=707-720&author=Y.+Zhouauthor=J.+Maauthor=X.+Linauthor=X.+P.+Huangauthor=K.+Wuauthor=N.+Huang&title=Structure-based+discovery+of+novel+and+selective+5-hydroxytryptamine+2B+receptor+antagonists+for+the+treatment+of+irritable+bowel+syndrome&doi=10.1021%2Facs.jmedchem.5b01631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit201R"><div class="casContent"><span class="casTitleNuber">201</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of Novel and Selective 5-Hydroxytryptamine 2B Receptor Antagonists for the Treatment of Irritable Bowel Syndrome</span></div><div class="casAuthors">Zhou, Yu; Ma, Jing; Lin, Xingyu; Huang, Xi-Ping; Wu, Kaichun; Huang, Niu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">707-720</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here the authors employed structure-based ligand discovery techniques to explore a recently detd. crystal structure of the 5-hydroxytryptamine 2B (5-HT2B) receptor.  Ten compds. contg. a novel chem. scaffold were identified; among them, seven mols. were active in cellular function assays with the most potent one exhibiting an IC50 value of 27.3 nM.  The authors then systematically probed the binding characteristics of this scaffold by designing, synthesizing, and testing a series of structural modifications.  The structure-activity relationship studies strongly support the authors' predicted binding model.  The binding profiling across a panel of 11 5-HT receptors indicated that these compds. are highly selective for the 5-HT2B receptor.  Oral administration of N4-(3-bromophenyl)-1,3,5-triazaspiro[5.5]undeca-1,3-diene-2,4-diamine (compd. 15) (30 mg/kg) produced significant attenuation of visceral hypersensitivity in a rat model of irritable bowel syndrome (IBS).  The authors expect this novel scaffold will serve as the foundation for the development of 5-HT2B antagonists for the treatment of IBS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTmaQmcj_24LVg90H21EOLACvtfcHk0lgK7NRXiELRdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVygsb%252FE&md5=5ca671401f4f8664fe55153121f2bb55</span></div><a href="/servlet/linkout?suffix=cit201&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01631%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DWu%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DN.%26atitle%3DStructure-based%2520discovery%2520of%2520novel%2520and%2520selective%25205-hydroxytryptamine%25202B%2520receptor%2520antagonists%2520for%2520the%2520treatment%2520of%2520irritable%2520bowel%2520syndrome%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D707%26epage%3D720%26doi%3D10.1021%2Facs.jmedchem.5b01631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref202"><div class="reference"><strong class="refLabel"><a href="#ref202" class="rightTabRefNumLink">202</a></strong><div class="NLM_citation" id="rightTab-cit202"><span><span class="NLM_contrib-group">Yan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wempe, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khanna, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meier, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffman, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owens, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wysoczynski, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nitz, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knabe, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmed, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brautigan, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paschal, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meroueh, S. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theodorescu, D.</span><span> </span><span class="NLM_article-title">Discovery and characterization of small molecules that target the GTPase Ral</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">515</span><span class="NLM_x">, </span> <span class="NLM_fpage">443</span><span class="NLM_x">–</span> <span class="NLM_lpage">447</span><span class="refDoi"> DOI: 10.1038/nature13713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref202/cit202&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnature13713" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref202/cit202&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25219851" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref202/cit202&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVemtrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=515&publication_year=2014&pages=443-447&author=C.+Yanauthor=D.+Liuauthor=L.+Liauthor=M.+F.+Wempeauthor=S.+Guinauthor=M.+Khannaauthor=J.+Meierauthor=B.+Hoffmanauthor=C.+Owensauthor=C.+L.+Wysoczynskiauthor=M.+D.+Nitzauthor=W.+E.+Knabeauthor=M.+Ahmedauthor=D.+L.+Brautiganauthor=B.+M.+Paschalauthor=M.+A.+Schwartzauthor=D.+N.+Jonesauthor=D.+Rossauthor=S.+O.+Merouehauthor=D.+Theodorescu&title=Discovery+and+characterization+of+small+molecules+that+target+the+GTPase+Ral&doi=10.1038%2Fnature13713"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit202R"><div class="casContent"><span class="casTitleNuber">202</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of small molecules that target the GTPase Ral</span></div><div class="casAuthors">Yan, Chao; Liu, Degang; Li, Liwei; Wempe, Michael F.; Guin, Sunny; Khanna, May; Meier, Jeremy; Hoffman, Brenton; Owens, Charles; Wysoczynski, Christina L.; Nitz, Matthew D.; Knabe, Eric W.; Brautigan, David L.; Paschal, Bryce M.; Schwartz, Martin A.; Jones, David N. M.; Ross, David; Meroueh, Samy O.; Theodorescu, Dan</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">515</span>
        (<span class="NLM_cas:issue">7527</span>),
    <span class="NLM_cas:pages">443-447</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The Ras-like GTPases RalA and RalB are important drivers of tumor growth and metastasis.  Chems. that block Ral function would be valuable as research tools and for cancer therapeutics.  Here we used protein structure anal. and virtual screening to identify drug-like mols. that bind to a site on the GDP-bound form of Ral.  The compds. RBC6, RBC8 and RBC10 inhibited the binding of Ral to its effector RALBP1, as well as inhibiting Ral-mediated cell spreading of murine embryonic fibroblasts and anchorage-independent growth of human cancer cell lines.  The binding of the RBC8 deriv. BQU57 to RalB was confirmed by isothermal titrn. calorimetry, surface plasmon resonance and 1H-15N transverse relaxation-optimized spectroscopy (TROSY) NMR spectroscopy.  RBC8 and BQU57 show selectivity for Ral relative to the GTPases Ras and RhoA and inhibit tumor xenograft growth to a similar extent to the depletion of Ral using RNA interference.  Our results show the utility of structure-based discovery for the development of therapeutics for Ral-dependent cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFuvI2j20IarVg90H21EOLACvtfcHk0lgK7NRXiELRdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVemtrjN&md5=9d2f67e08379fffa98fe88e2c49597fd</span></div><a href="/servlet/linkout?suffix=cit202&amp;dbid=16384&amp;doi=10.1038%2Fnature13713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13713%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DWempe%26aufirst%3DM.%2BF.%26aulast%3DGuin%26aufirst%3DS.%26aulast%3DKhanna%26aufirst%3DM.%26aulast%3DMeier%26aufirst%3DJ.%26aulast%3DHoffman%26aufirst%3DB.%26aulast%3DOwens%26aufirst%3DC.%26aulast%3DWysoczynski%26aufirst%3DC.%2BL.%26aulast%3DNitz%26aufirst%3DM.%2BD.%26aulast%3DKnabe%26aufirst%3DW.%2BE.%26aulast%3DAhmed%26aufirst%3DM.%26aulast%3DBrautigan%26aufirst%3DD.%2BL.%26aulast%3DPaschal%26aufirst%3DB.%2BM.%26aulast%3DSchwartz%26aufirst%3DM.%2BA.%26aulast%3DJones%26aufirst%3DD.%2BN.%26aulast%3DRoss%26aufirst%3DD.%26aulast%3DMeroueh%26aufirst%3DS.%2BO.%26aulast%3DTheodorescu%26aufirst%3DD.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520small%2520molecules%2520that%2520target%2520the%2520GTPase%2520Ral%26jtitle%3DNature%26date%3D2014%26volume%3D515%26spage%3D443%26epage%3D447%26doi%3D10.1038%2Fnature13713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref203"><div class="reference"><strong class="refLabel"><a href="#ref203" class="rightTabRefNumLink">203</a></strong><div class="NLM_citation" id="rightTab-cit203"><span><span class="NLM_contrib-group">Gruneberg, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wendt, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Subnanomolar inhibitors from computer screening: a model study using human carbonic anhydrase II</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">389</span><span class="NLM_x">–</span> <span class="NLM_lpage">393</span><span class="refDoi"> DOI: 10.1002/1521-3773(20010119)40:2<389::AID-ANIE389>3.0.CO;2-#</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref203/cit203&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2F1521-3773%2820010119%2940%3A2%3C389%3A%3AAID-ANIE389%3E3.0.CO%3B2-%23" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=389-393&author=S.+Grunebergauthor=B.+Wendtauthor=G.+Klebe&title=Subnanomolar+inhibitors+from+computer+screening%3A+a+model+study+using+human+carbonic+anhydrase+II&doi=10.1002%2F1521-3773%2820010119%2940%3A2%3C389%3A%3AAID-ANIE389%3E3.0.CO%3B2-%23"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit203&amp;dbid=16384&amp;doi=10.1002%2F1521-3773%2820010119%2940%3A2%3C389%3A%3AAID-ANIE389%3E3.0.CO%3B2-%23&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1521-3773%252820010119%252940%253A2%253C389%253A%253AAID-ANIE389%253E3.0.CO%253B2-%2523%26sid%3Dliteratum%253Aachs%26aulast%3DGruneberg%26aufirst%3DS.%26aulast%3DWendt%26aufirst%3DB.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DSubnanomolar%2520inhibitors%2520from%2520computer%2520screening%253A%2520a%2520model%2520study%2520using%2520human%2520carbonic%2520anhydrase%2520II%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2001%26volume%3D40%26spage%3D389%26epage%3D393%26doi%3D10.1002%2F1521-3773%2820010119%2940%3A2%3C389%3A%3AAID-ANIE389%3E3.0.CO%3B2-%23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref204"><div class="reference"><strong class="refLabel"><a href="#ref204" class="rightTabRefNumLink">204</a></strong><div class="NLM_citation" id="rightTab-cit204"><span><span class="NLM_contrib-group">de Graaf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kooistra, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vischer, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katritch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuijer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiroishi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwata, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimamura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span> </span><span class="NLM_article-title">Crystal structure-based virtual screening for fragment-like ligands of the human histamine H(1) receptor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">8195</span><span class="NLM_x">–</span> <span class="NLM_lpage">8206</span><span class="refDoi"> DOI: 10.1021/jm2011589</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2011589" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref204/cit204&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlaltbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8195-8206&author=C.+de+Graafauthor=A.+J.+Kooistraauthor=H.+F.+Vischerauthor=V.+Katritchauthor=M.+Kuijerauthor=M.+Shiroishiauthor=S.+Iwataauthor=T.+Shimamuraauthor=R.+C.+Stevensauthor=I.+J.+de+Eschauthor=R.+Leurs&title=Crystal+structure-based+virtual+screening+for+fragment-like+ligands+of+the+human+histamine+H%281%29+receptor&doi=10.1021%2Fjm2011589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit204R"><div class="casContent"><span class="casTitleNuber">204</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure-Based Virtual Screening for Fragment-like Ligands of the Human Histamine H1 Receptor</span></div><div class="casAuthors">de Graaf, Chris; Kooistra, Albert J.; Vischer, Henry F.; Katritch, Vsevolod; Kuijer, Martien; Shiroishi, Mitsunori; Iwata, So; Shimamura, Tatsuro; Stevens, Raymond C.; de Esch, Iwan J. P.; Leurs, Rob</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8195-8206</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent crystal structure detns. of druggable class A G protein-coupled receptors (GPCRs) have opened up excellent opportunities in structure-based ligand discovery for this pharmaceutically important protein family.  We have developed and validated a customized structure-based virtual fragment screening protocol against the recently detd. human histamine H1 receptor (H1R) crystal structure.  The method combines mol. docking simulations with a protein-ligand interaction fingerprint (IFP) scoring method.  The optimized in silico screening approach was successfully applied to identify a chem. diverse set of novel fragment-like (≤22 heavy atoms) H1R ligands with an exceptionally high hit rate of 73%.  Of the 26 tested fragments, 19 compds. had affinities ranging from 10 μM to 6 nM.  The current study shows the potential of in silico screening against GPCR crystal structures to explore novel, fragment-like GPCR ligand space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbwo3rozOuPbVg90H21EOLACvtfcHk0lihWOpCSamm1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlaltbnL&md5=e45d22d293667ff443f6d62ecef90f9d</span></div><a href="/servlet/linkout?suffix=cit204&amp;dbid=16384&amp;doi=10.1021%2Fjm2011589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2011589%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DVischer%26aufirst%3DH.%2BF.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DKuijer%26aufirst%3DM.%26aulast%3DShiroishi%26aufirst%3DM.%26aulast%3DIwata%26aufirst%3DS.%26aulast%3DShimamura%26aufirst%3DT.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26atitle%3DCrystal%2520structure-based%2520virtual%2520screening%2520for%2520fragment-like%2520ligands%2520of%2520the%2520human%2520histamine%2520H%25281%2529%2520receptor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8195%26epage%3D8206%26doi%3D10.1021%2Fjm2011589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref205"><div class="reference"><strong class="refLabel"><a href="#ref205" class="rightTabRefNumLink">205</a></strong><div class="NLM_citation" id="rightTab-cit205"><span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenbaum, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Structure-based discovery of beta2-adrenergic receptor ligands</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">6843</span><span class="NLM_x">–</span> <span class="NLM_lpage">6848</span><span class="refDoi"> DOI: 10.1073/pnas.0812657106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref205/cit205&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1073%2Fpnas.0812657106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref205/cit205&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=19342484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref205/cit205&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsV2qsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=6843-6848&author=P.+Kolbauthor=D.+M.+Rosenbaumauthor=J.+J.+Irwinauthor=J.+J.+Fungauthor=B.+K.+Kobilkaauthor=B.+K.+Shoichet&title=Structure-based+discovery+of+beta2-adrenergic+receptor+ligands&doi=10.1073%2Fpnas.0812657106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit205R"><div class="casContent"><span class="casTitleNuber">205</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of β2-adrenergic receptor ligands</span></div><div class="casAuthors">Kolb, Peter; Rosenbaum, Daniel M.; Irwin, John J.; Fung, Juan Jose; Kobilka, Brian K.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6843-6848</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Aminergic G protein-coupled receptors (GPCRs) have been a major focus of pharmaceutical research for many years.  Due partly to the lack of reliable receptor structures, drug discovery efforts have been largely ligand-based.  The recently detd. X-ray structure of the β2-adrenergic receptor offers an opportunity to investigate the advantages and limitations inherent in a structure-based approach to ligand discovery against this and related GPCR targets.  Approx. 1 million com. available, "lead-like" mols. were docked against the β2-adrenergic receptor structure.  On testing of 25 high-ranking mols., 6 were active with binding affinities <4 μM, with the best mol. binding with a Ki of 9 nM (95% confidence interval 7-10 nM).  Five of these mols. were inverse agonists.  The high hit rate, the high affinity of the most potent mol., the discovery of unprecedented chemotypes among the new inhibitors, and the apparent bias toward inverse agonists among the docking hits, have implications for structure-based approaches against GPCRs that recognize small org. mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo89kM84Li_H7Vg90H21EOLACvtfcHk0lihWOpCSamm1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsV2qsro%253D&md5=4d1a4cb2aa3925aa4c99c6b0496417a7</span></div><a href="/servlet/linkout?suffix=cit205&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0812657106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0812657106%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFung%26aufirst%3DJ.%2BJ.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520discovery%2520of%2520beta2-adrenergic%2520receptor%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D6843%26epage%3D6848%26doi%3D10.1073%2Fpnas.0812657106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref206"><div class="reference"><strong class="refLabel"><a href="#ref206" class="rightTabRefNumLink">206</a></strong><div class="NLM_citation" id="rightTab-cit206"><span><span class="NLM_contrib-group">Rodriguez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, Z. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moss, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlsson, J.</span><span> </span><span class="NLM_article-title">Molecular docking screening using agonist-bound GPCR structures: probing the A2A adenosine receptor</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">550</span><span class="NLM_x">–</span> <span class="NLM_lpage">563</span><span class="refDoi"> DOI: 10.1021/ci500639g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500639g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref206/cit206&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVWgtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=550-563&author=D.+Rodriguezauthor=Z.+G.+Gaoauthor=S.+M.+Mossauthor=K.+A.+Jacobsonauthor=J.+Carlsson&title=Molecular+docking+screening+using+agonist-bound+GPCR+structures%3A+probing+the+A2A+adenosine+receptor&doi=10.1021%2Fci500639g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit206R"><div class="casContent"><span class="casTitleNuber">206</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Docking Screening Using Agonist-Bound GPCR Structures: Probing the A2A Adenosine Receptor</span></div><div class="casAuthors">Rodriguez, David; Gao, Zhang-Guo; Moss, Steven M.; Jacobson, Kenneth A.; Carlsson, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">550-563</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Crystal structures of G protein-coupled receptors (GPCRs) have recently revealed the mol. basis of ligand binding and activation, which has provided exciting opportunities for structure-based drug design.  The A2A adenosine receptor (A2AAR) is a promising therapeutic target for cardiovascular diseases, but progress in this area is limited by the lack of novel agonist scaffolds.  The authors carried out docking screens of 6.7 million com. available mols. against active-like conformations of the A2AAR to investigate whether these structures could guide the discovery of agonists.  Nine out of the 20 predicted agonists were confirmed to be A2AAR ligands, but none of these activated the ARs.  The difficulties in discovering AR agonists using structure-based methods originated from limited at.-level understanding of the activation mechanism and a chem. bias toward antagonists in the screened library.  In particular, the compn. of the screened library was found to strongly reduce the likelihood of identifying AR agonists, which reflected the high ligand complexity required for receptor activation.  Extension of this anal. to other pharmaceutically relevant GPCRs suggested that library screening may not be suitable for targets requiring a complex receptor-ligand interaction network.  The authors' results provide specific directions for the future development of novel A2AAR agonists and general strategies for structure-based drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp89cADSkVJDLVg90H21EOLACvtfcHk0lihWOpCSamm1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVWgtbs%253D&md5=0faec5c3d810a9ef32e5f374b6f9b447</span></div><a href="/servlet/linkout?suffix=cit206&amp;dbid=16384&amp;doi=10.1021%2Fci500639g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500639g%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez%26aufirst%3DD.%26aulast%3DGao%26aufirst%3DZ.%2BG.%26aulast%3DMoss%26aufirst%3DS.%2BM.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DCarlsson%26aufirst%3DJ.%26atitle%3DMolecular%2520docking%2520screening%2520using%2520agonist-bound%2520GPCR%2520structures%253A%2520probing%2520the%2520A2A%2520adenosine%2520receptor%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D550%26epage%3D563%26doi%3D10.1021%2Fci500639g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref207"><div class="reference"><strong class="refLabel"><a href="#ref207" class="rightTabRefNumLink">207</a></strong><div class="NLM_citation" id="rightTab-cit207"><span><span class="NLM_contrib-group">Colas, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grewer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otte, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gameiro, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albers, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shere, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonomi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holst, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, A.</span><span> </span><span class="NLM_article-title">Ligand discovery for the alanine-serine-cysteine transporter (ASCT2, SLC1A5) from homology modeling and virtual screening</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1004477</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1004477</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref207/cit207&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pcbi.1004477" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=e1004477&author=C.+Colasauthor=C.+Grewerauthor=N.+J.+Otteauthor=A.+Gameiroauthor=T.+Albersauthor=K.+Singhauthor=H.+Shereauthor=M.+Bonomiauthor=J.+Holstauthor=A.+Schlessinger&title=Ligand+discovery+for+the+alanine-serine-cysteine+transporter+%28ASCT2%2C+SLC1A5%29+from+homology+modeling+and+virtual+screening&doi=10.1371%2Fjournal.pcbi.1004477"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit207&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1004477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1004477%26sid%3Dliteratum%253Aachs%26aulast%3DColas%26aufirst%3DC.%26aulast%3DGrewer%26aufirst%3DC.%26aulast%3DOtte%26aufirst%3DN.%2BJ.%26aulast%3DGameiro%26aufirst%3DA.%26aulast%3DAlbers%26aufirst%3DT.%26aulast%3DSingh%26aufirst%3DK.%26aulast%3DShere%26aufirst%3DH.%26aulast%3DBonomi%26aufirst%3DM.%26aulast%3DHolst%26aufirst%3DJ.%26aulast%3DSchlessinger%26aufirst%3DA.%26atitle%3DLigand%2520discovery%2520for%2520the%2520alanine-serine-cysteine%2520transporter%2520%2528ASCT2%252C%2520SLC1A5%2529%2520from%2520homology%2520modeling%2520and%2520virtual%2520screening%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3De1004477%26doi%3D10.1371%2Fjournal.pcbi.1004477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref208"><div class="reference"><strong class="refLabel"><a href="#ref208" class="rightTabRefNumLink">208</a></strong><div class="NLM_citation" id="rightTab-cit208"><span><span class="NLM_contrib-group">Katritch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaakola, V. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ijzerman, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeager, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kufareva, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Structure-based discovery of novel chemotypes for adenosine A(2A) receptor antagonists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">1799</span><span class="NLM_x">–</span> <span class="NLM_lpage">1809</span><span class="refDoi"> DOI: 10.1021/jm901647p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901647p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=1799-1809&author=V.+Katritchauthor=V.+P.+Jaakolaauthor=J.+R.+Laneauthor=J.+Linauthor=A.+P.+Ijzermanauthor=M.+Yeagerauthor=I.+Kufarevaauthor=R.+C.+Stevensauthor=R.+Abagyan&title=Structure-based+discovery+of+novel+chemotypes+for+adenosine+A%282A%29+receptor+antagonists&doi=10.1021%2Fjm901647p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit208&amp;dbid=16384&amp;doi=10.1021%2Fjm901647p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901647p%26sid%3Dliteratum%253Aachs%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DJaakola%26aufirst%3DV.%2BP.%26aulast%3DLane%26aufirst%3DJ.%2BR.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26aulast%3DYeager%26aufirst%3DM.%26aulast%3DKufareva%26aufirst%3DI.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DStructure-based%2520discovery%2520of%2520novel%2520chemotypes%2520for%2520adenosine%2520A%25282A%2529%2520receptor%2520antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D1799%26epage%3D1809%26doi%3D10.1021%2Fjm901647p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref209"><div class="reference"><strong class="refLabel"><a href="#ref209" class="rightTabRefNumLink">209</a></strong><div class="NLM_citation" id="rightTab-cit209"><span><span class="NLM_contrib-group">Carlsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, Z. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, K. A.</span><span> </span><span class="NLM_article-title">Structure-based discovery of A2A adenosine receptor ligands</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">3748</span><span class="NLM_x">–</span> <span class="NLM_lpage">3755</span><span class="refDoi"> DOI: 10.1021/jm100240h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100240h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref209/cit209&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvFaqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=3748-3755&author=J.+Carlssonauthor=L.+Yooauthor=Z.+G.+Gaoauthor=J.+J.+Irwinauthor=B.+K.+Shoichetauthor=K.+A.+Jacobson&title=Structure-based+discovery+of+A2A+adenosine+receptor+ligands&doi=10.1021%2Fjm100240h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit209R"><div class="casContent"><span class="casTitleNuber">209</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of A2A Adenosine Receptor Ligands</span></div><div class="casAuthors">Carlsson, Jens; Yoo, Lena; Gao, Zhan-Guo; Irwin, John J.; Shoichet, Brian K.; Jacobson, Kenneth A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3748-3755</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent detn. of X-ray structures of pharmacol. relevant GPCRs has made these targets accessible to structure-based ligand discovery.  Here we explore whether novel chemotypes may be discovered for the A2A adenosine receptor, based on complementarity to its recently detd. structure.  The A2A adenosine receptor signals in the periphery and the CNS, with agonists explored as anti-inflammatory drugs and antagonists explored for neurodegenerative diseases.  We used mol. docking to screen a 1.4 million compd. database against the X-ray structure computationally and tested 20 high-ranking, previously unknown mols. exptl.  Of these 35% showed substantial activity with affinities between 200 nM and 9 μM.  For the most potent of these new inhibitors, over 50-fold specificity was obsd. for the A2A vs. the related A1 and A3 subtypes.  These high hit rates and affinities at least partly reflect the bias of com. libraries toward GPCR-like chemotypes, an issue that we attempt to investigate quant.  Despite this bias, many of the most potent new ligands were novel, dissimilar from known ligands, providing new lead structures for modulation of this medically important target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzvS6-9T9bd7Vg90H21EOLACvtfcHk0lghNIryeD9uKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvFaqsL8%253D&md5=c36c941d52d2cec06387d79c4c423d46</span></div><a href="/servlet/linkout?suffix=cit209&amp;dbid=16384&amp;doi=10.1021%2Fjm100240h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100240h%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DYoo%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DZ.%2BG.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DStructure-based%2520discovery%2520of%2520A2A%2520adenosine%2520receptor%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D3748%26epage%3D3755%26doi%3D10.1021%2Fjm100240h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref210"><div class="reference"><strong class="refLabel"><a href="#ref210" class="rightTabRefNumLink">210</a></strong><div class="NLM_citation" id="rightTab-cit210"><span><span class="NLM_contrib-group">Park, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chien, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryu, S. E.</span><span> </span><span class="NLM_article-title">Discovery of potent inhibitors of receptor protein tyrosine phosphatase sigma through the structure-based virtual screening</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">6333</span><span class="NLM_x">–</span> <span class="NLM_lpage">6337</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2012.08.081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref210/cit210&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2012.08.081" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=6333-6337&author=H.+Parkauthor=P.+N.+Chienauthor=S.+E.+Ryu&title=Discovery+of+potent+inhibitors+of+receptor+protein+tyrosine+phosphatase+sigma+through+the+structure-based+virtual+screening&doi=10.1016%2Fj.bmcl.2012.08.081"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit210&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.08.081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.08.081%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DH.%26aulast%3DChien%26aufirst%3DP.%2BN.%26aulast%3DRyu%26aufirst%3DS.%2BE.%26atitle%3DDiscovery%2520of%2520potent%2520inhibitors%2520of%2520receptor%2520protein%2520tyrosine%2520phosphatase%2520sigma%2520through%2520the%2520structure-based%2520virtual%2520screening%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D6333%26epage%3D6337%26doi%3D10.1016%2Fj.bmcl.2012.08.081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref211"><div class="reference"><strong class="refLabel"><a href="#ref211" class="rightTabRefNumLink">211</a></strong><div class="NLM_citation" id="rightTab-cit211"><span><span class="NLM_contrib-group">Zarzycka, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seijkens, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nabuurs, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritschel, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grommes, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soehnlein, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schrijver, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Tiel, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hackeng, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giehler, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kieser, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lutgens, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vriend, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicolaes, G. A.</span><span> </span><span class="NLM_article-title">Discovery of small molecule CD40-TRAF6 inhibitors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">294</span><span class="NLM_x">–</span> <span class="NLM_lpage">307</span><span class="refDoi"> DOI: 10.1021/ci500631e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500631e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref211/cit211&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2gsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=294-307&author=B.+Zarzyckaauthor=T.+Seijkensauthor=S.+B.+Nabuursauthor=T.+Ritschelauthor=J.+Grommesauthor=O.+Soehnleinauthor=R.+Schrijverauthor=C.+M.+van+Tielauthor=T.+M.+Hackengauthor=C.+Weberauthor=F.+Giehlerauthor=A.+Kieserauthor=E.+Lutgensauthor=G.+Vriendauthor=G.+A.+Nicolaes&title=Discovery+of+small+molecule+CD40-TRAF6+inhibitors&doi=10.1021%2Fci500631e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit211R"><div class="casContent"><span class="casTitleNuber">211</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Molecule CD40-TRAF6 Inhibitors</span></div><div class="casAuthors">Zarzycka, Barbara; Seijkens, Tom; Nabuurs, Sander B.; Ritschel, Tina; Grommes, Jochen; Soehnlein, Oliver; Schrijver, Roy; van Tiel, Claudia M.; Hackeng, Tilman M.; Weber, Christian; Giehler, Fabian; Kieser, Arnd; Lutgens, Esther; Vriend, Gert; Nicolaes, Gerry A. F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">294-307</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The CD154-CD40 receptor complex plays a pivotal role in several inflammatory pathways.  Attempts to inhibit the formation of this complex have resulted in systemic side effects.  Downstream inhibition of the CD40 signaling pathway therefore seems a better way to ameliorate inflammatory disease.  To relay a signal, the CD40 receptor recruits adapter proteins called tumor necrosis factor receptor-assocd. factors (TRAFs).  CD40-TRAF6 interactions are known to play an essential role in several inflammatory diseases.  We used in silico, in vitro, and in vivo expts. to identify and characterize compds. that block CD40-TRAF6 interactions.  We present in detail our drug docking and optimization pipeline and show how we used it to find lead compds. that reduce inflammation in models of peritonitis and sepsis.  These compds. appear to be good leads for drug development, given the obsd. absence of side effects and their demonstrated efficacy for peritonitis and sepsis in mouse models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMWtO3zA6Qk7Vg90H21EOLACvtfcHk0lghNIryeD9uKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2gsLk%253D&md5=2879de87f371f19a9097461d76e28038</span></div><a href="/servlet/linkout?suffix=cit211&amp;dbid=16384&amp;doi=10.1021%2Fci500631e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500631e%26sid%3Dliteratum%253Aachs%26aulast%3DZarzycka%26aufirst%3DB.%26aulast%3DSeijkens%26aufirst%3DT.%26aulast%3DNabuurs%26aufirst%3DS.%2BB.%26aulast%3DRitschel%26aufirst%3DT.%26aulast%3DGrommes%26aufirst%3DJ.%26aulast%3DSoehnlein%26aufirst%3DO.%26aulast%3DSchrijver%26aufirst%3DR.%26aulast%3Dvan%2BTiel%26aufirst%3DC.%2BM.%26aulast%3DHackeng%26aufirst%3DT.%2BM.%26aulast%3DWeber%26aufirst%3DC.%26aulast%3DGiehler%26aufirst%3DF.%26aulast%3DKieser%26aufirst%3DA.%26aulast%3DLutgens%26aufirst%3DE.%26aulast%3DVriend%26aufirst%3DG.%26aulast%3DNicolaes%26aufirst%3DG.%2BA.%26atitle%3DDiscovery%2520of%2520small%2520molecule%2520CD40-TRAF6%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D294%26epage%3D307%26doi%3D10.1021%2Fci500631e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref212"><div class="reference"><strong class="refLabel"><a href="#ref212" class="rightTabRefNumLink">212</a></strong><div class="NLM_citation" id="rightTab-cit212"><span><span class="NLM_contrib-group">Fazi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tintori, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botta, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Selvaraj, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garbelli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maga, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botta, M.</span><span> </span><span class="NLM_article-title">Homology Model-Based Virtual Screening for the Identification of Human Helicase DDX3 Inhibitors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">2443</span><span class="NLM_x">–</span> <span class="NLM_lpage">2454</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00419</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00419" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref212/cit212&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVent7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2443-2454&author=R.+Faziauthor=C.+Tintoriauthor=A.+Braiauthor=L.+Bottaauthor=M.+Selvarajauthor=A.+Garbelliauthor=G.+Magaauthor=M.+Botta&title=Homology+Model-Based+Virtual+Screening+for+the+Identification+of+Human+Helicase+DDX3+Inhibitors&doi=10.1021%2Facs.jcim.5b00419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit212R"><div class="casContent"><span class="casTitleNuber">212</span><div class="casTitle"><span class="NLM_cas:atitle">Homology Model-Based Virtual Screening for the Identification of Human Helicase DDX3 Inhibitors</span></div><div class="casAuthors">Fazi, Roberta; Tintori, Cristina; Brai, Annalaura; Botta, Lorenzo; Selvaraj, Manikandan; Garbelli, Anna; Maga, Giovanni; Botta, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2443-2454</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting cellular cofactors instead of viral enzymes represents a new strategy to combat infectious diseases, which should help to overcome the problem of viral resistance.  Recently, it has been revealed that the cellular ATPase/RNA helicase X-linked DEAD-box polypeptide 3 (DDX3) is an essential host factor for the replication of several viruses such as HIV, HCV, JEV, Dengue, and West Nile.  Accordingly, a drug targeting DDX3 could theor. inhibit all viruses that are dependent on this host factor.  Herein, for the first time, a model of hDDX3 in its closed conformation, which binds the viral RNA was developed by using the homol. module of Prime through the Maestro interface of Schrodinger.  Next, a structure-based virtual screening protocol was applied to identify DDX3 small mol. inhibitors targeting the RNA binding pocket.  As a result, an impressive hit rate of 40% was obtained with the identification of 10 active compds. out of the 25 tested small mols.  The best poses of the active ligands highlighted the crucial residues to be targeted for the inhibition of the helicase activity of DDX3.  The obtained results confirm the reliability of the constructed DDX3/RNA model and the proposed computational strategy for investigating novel DDX3 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO1Sy7ZFr2kbVg90H21EOLACvtfcHk0lgxi8wGXXev-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVent7fN&md5=0c739457f73969394c6a31c113617bb6</span></div><a href="/servlet/linkout?suffix=cit212&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00419%26sid%3Dliteratum%253Aachs%26aulast%3DFazi%26aufirst%3DR.%26aulast%3DTintori%26aufirst%3DC.%26aulast%3DBrai%26aufirst%3DA.%26aulast%3DBotta%26aufirst%3DL.%26aulast%3DSelvaraj%26aufirst%3DM.%26aulast%3DGarbelli%26aufirst%3DA.%26aulast%3DMaga%26aufirst%3DG.%26aulast%3DBotta%26aufirst%3DM.%26atitle%3DHomology%2520Model-Based%2520Virtual%2520Screening%2520for%2520the%2520Identification%2520of%2520Human%2520Helicase%2520DDX3%2520Inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2443%26epage%3D2454%26doi%3D10.1021%2Facs.jcim.5b00419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref213"><div class="reference"><strong class="refLabel"><a href="#ref213" class="rightTabRefNumLink">213</a></strong><div class="NLM_citation" id="rightTab-cit213"><span><span class="NLM_contrib-group">Carlsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Ligand discovery from a dopamine D3 receptor homology model and crystal structure</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">769</span><span class="NLM_x">–</span> <span class="NLM_lpage">778</span><span class="refDoi"> DOI: 10.1038/nchembio.662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref213/cit213&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnchembio.662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref213/cit213&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=21926995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref213/cit213&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=769-778&author=J.+Carlssonauthor=R.+G.+Colemanauthor=V.+Setolaauthor=J.+J.+Irwinauthor=H.+Fanauthor=A.+Schlessingerauthor=A.+Saliauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Ligand+discovery+from+a+dopamine+D3+receptor+homology+model+and+crystal+structure&doi=10.1038%2Fnchembio.662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit213R"><div class="casContent"><span class="casTitleNuber">213</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand discovery from a dopamine D3 receptor homology model and crystal structure</span></div><div class="casAuthors">Carlsson, Jens; Coleman, Ryan G.; Setola, Vincent; Irwin, John J.; Fan, Hao; Schlessinger, Avner; Sali, Andrej; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">769-778</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are intensely studied as drug targets and for their role in signaling.  With the detn. of the first crystal structures, interest in structure-based ligand discovery increased.  Unfortunately, for most GPCRs no exptl. structures are available.  The detn. of the D3 receptor structure and the challenge to the community to predict it enabled a fully prospective comparison of ligand discovery from a modeled structure vs. that of the subsequently released crystal structure.  Over 3.3 million mols. were docked against a homol. model, and 26 of the highest ranking were tested for binding.  Six had affinities ranging from 0.2 to 3.1 μM.  Subsequently, the crystal structure was released and the docking screen repeated.  Of the 25 compds. selected, five had affinities ranging from 0.3 to 3.0 μM.  One of the new ligands from the homol. model screen was optimized for affinity to 81 nM.  The feasibility of docking screens against modeled GPCRs more generally is considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz6gBx3Olbn7Vg90H21EOLACvtfcHk0lgxi8wGXXev-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP&md5=7242aa512079a348678b309d352ecdb9</span></div><a href="/servlet/linkout?suffix=cit213&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.662%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DLigand%2520discovery%2520from%2520a%2520dopamine%2520D3%2520receptor%2520homology%2520model%2520and%2520crystal%2520structure%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D769%26epage%3D778%26doi%3D10.1038%2Fnchembio.662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref214"><div class="reference"><strong class="refLabel"><a href="#ref214" class="rightTabRefNumLink">214</a></strong><div class="NLM_citation" id="rightTab-cit214"><span><span class="NLM_contrib-group">Sorna, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theisen, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stephens, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warner, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vankayalapati, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span> </span><span class="NLM_article-title">High-throughput virtual screening identifies novel N′-(1-phenylethylidene)-benzohydrazides as potent, specific, and reversible LSD1 inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">9496</span><span class="NLM_x">–</span> <span class="NLM_lpage">9508</span><span class="refDoi"> DOI: 10.1021/jm400870h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400870h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref214/cit214&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslyks7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9496-9508&author=V.+Sornaauthor=E.+R.+Theisenauthor=B.+Stephensauthor=S.+L.+Warnerauthor=D.+J.+Bearssauthor=H.+Vankayalapatiauthor=S.+Sharma&title=High-throughput+virtual+screening+identifies+novel+N%E2%80%B2-%281-phenylethylidene%29-benzohydrazides+as+potent%2C+specific%2C+and+reversible+LSD1+inhibitors&doi=10.1021%2Fjm400870h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit214R"><div class="casContent"><span class="casTitleNuber">214</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Virtual Screening Identifies Novel N'-(1-Phenylethylidene)-benzohydrazides as Potent, Specific, and Reversible LSD1 Inhibitors</span></div><div class="casAuthors">Sorna, Venkataswamy; Theisen, Emily R.; Stephens, Bret; Warner, Steven L.; Bearss, David J.; Vankayalapati, Hariprasad; Sharma, Sunil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9496-9508</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) plays an important role in regulating histone lysine methylation at residues K4 and K9 on histone H3 and is an attractive therapeutic target in multiple malignancies.  Here we report a structure-based virtual screen of a compd. library contg. ∼2 million small mol. entities.  Computational docking and scoring followed by biochem. screening led to the identification of a novel N'-(1-phenylethylidene)-benzohydrazide series of LSD1 inhibitors with hits showing biochem. IC50s in the 200-400 nM range.  Hit-to-lead optimization and structure-activity relation studies aided in the discovery of compd. (I), with a Ki of 31 nM.  Compd. I is reversible and specific for LSD1 as compared to the monoamine oxidases shows minimal inhibition of CYPs and hERG and inhibits proliferation and survival in several cancer cell lines, including breast and colorectal cancer.  Compd. I may be used to probe LSD1's biol. role in these cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp97P3ef8ur6rVg90H21EOLACvtfcHk0lhhgygIRVP6SA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslyks7fF&md5=47293fd790c5bc9fd944943ff4b1a58a</span></div><a href="/servlet/linkout?suffix=cit214&amp;dbid=16384&amp;doi=10.1021%2Fjm400870h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400870h%26sid%3Dliteratum%253Aachs%26aulast%3DSorna%26aufirst%3DV.%26aulast%3DTheisen%26aufirst%3DE.%2BR.%26aulast%3DStephens%26aufirst%3DB.%26aulast%3DWarner%26aufirst%3DS.%2BL.%26aulast%3DBearss%26aufirst%3DD.%2BJ.%26aulast%3DVankayalapati%26aufirst%3DH.%26aulast%3DSharma%26aufirst%3DS.%26atitle%3DHigh-throughput%2520virtual%2520screening%2520identifies%2520novel%2520N%25E2%2580%25B2-%25281-phenylethylidene%2529-benzohydrazides%2520as%2520potent%252C%2520specific%252C%2520and%2520reversible%2520LSD1%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9496%26epage%3D9508%26doi%3D10.1021%2Fjm400870h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref215"><div class="reference"><strong class="refLabel"><a href="#ref215" class="rightTabRefNumLink">215</a></strong><div class="NLM_citation" id="rightTab-cit215"><span><span class="NLM_contrib-group">Mahasenan, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span> </span><span class="NLM_article-title">Novel inhibitor discovery through virtual screening against multiple protein conformations generated via ligand-directed modeling: a maternal embryonic leucine zipper kinase example</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1345</span><span class="NLM_x">–</span> <span class="NLM_lpage">1355</span><span class="refDoi"> DOI: 10.1021/ci300040c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300040c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref215/cit215&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmt1Sisr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1345-1355&author=K.+V.+Mahasenanauthor=C.+Li&title=Novel+inhibitor+discovery+through+virtual+screening+against+multiple+protein+conformations+generated+via+ligand-directed+modeling%3A+a+maternal+embryonic+leucine+zipper+kinase+example&doi=10.1021%2Fci300040c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit215R"><div class="casContent"><span class="casTitleNuber">215</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Inhibitor Discovery through Virtual Screening against Multiple Protein Conformations Generated via Ligand-Directed Modeling: A Maternal Embryonic Leucine Zipper Kinase Example</span></div><div class="casAuthors">Mahasenan, Kiran V.; Li, Chenglong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1345-1355</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Kinase targets have been demonstrated to undergo major conformational reorganization upon ligand binding.  Such protein conformational plasticity remains a significant challenge in structure-based virtual screening methodol. and may be approximated by screening against an ensemble of diverse protein conformations.  Maternal embryonic leucine zipper kinase (MELK), a member of serine-threonine kinase family, has been recently found to be involved in the tumerogenic state of glioblastoma, breast, ovarian, and colon cancers.  We therefore modeled several conformers of MELK utilizing the available chemogenomic and crystallog. data of homologous kinases.  We carried out docking pose prediction and virtual screening enrichment studies with these conformers.  The performances of the ensembles were evaluated by their ability to reproduce known inhibitor bioactive conformations and to efficiently recover known active compds. early in the virtual screen when seeded with decoy sets.  A few of the individual MELK conformers performed satisfactorily in reproducing the native protein-ligand pharmacophoric interactions up to 50% of the cases.  By selecting an ensemble of a few representative conformational states, most of the known inhibitor binding poses could be rationalized.  For example, a four conformer ensemble is able to recover 95% of the studied actives, esp. with imperfect scoring function(s).  The virtual screening enrichment varied considerably among different MELK conformers.  Enrichment appears to improve by selection of a proper protein conformation.  For example, several holo and unliganded active conformations are better to accommodate diverse chemotypes than ATP-bound conformer.  These results prove that using an ensemble of diverse conformations could give a better performance.  Applying this approach, we were able to screen a com. available library of half a million compds. against three conformers to discover three novel inhibitors of MELK, one from each template.  Among the three compds. validated via exptl. enzyme inhibition assays, one is relatively potent (15; Kd = 0.37 μM), one moderately active (12; Kd = 3.2 μM), and one weak but very selective (9; Kd = 18 μM).  These novel hits may be utilized to assist in the development of small mol. therapeutic agents useful in diseases caused by deregulated MELK, and perhaps more importantly, the approach demonstrates the advantages of choosing an appropriate ensemble of a few conformers in pursuing compd. potency, selectivity, and novel chemotypes over using single target conformation for structure-based drug design in general.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGranNM-bXItZbVg90H21EOLACvtfcHk0lhhgygIRVP6SA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmt1Sisr0%253D&md5=a17bcbfa52439be4f076182f405460e6</span></div><a href="/servlet/linkout?suffix=cit215&amp;dbid=16384&amp;doi=10.1021%2Fci300040c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300040c%26sid%3Dliteratum%253Aachs%26aulast%3DMahasenan%26aufirst%3DK.%2BV.%26aulast%3DLi%26aufirst%3DC.%26atitle%3DNovel%2520inhibitor%2520discovery%2520through%2520virtual%2520screening%2520against%2520multiple%2520protein%2520conformations%2520generated%2520via%2520ligand-directed%2520modeling%253A%2520a%2520maternal%2520embryonic%2520leucine%2520zipper%2520kinase%2520example%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1345%26epage%3D1355%26doi%3D10.1021%2Fci300040c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref216"><div class="reference"><strong class="refLabel"><a href="#ref216" class="rightTabRefNumLink">216</a></strong><div class="NLM_citation" id="rightTab-cit216"><span><span class="NLM_contrib-group">Nolan, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunstan, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caraher, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratford, I. J.</span><span> </span><span class="NLM_article-title">In silico screening reveals structurally diverse, nanomolar inhibitors of NQO2 that are functionally active in cells and can modulate NF-kappaB signaling</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">194</span><span class="NLM_x">–</span> <span class="NLM_lpage">203</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-11-0543</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref216/cit216&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1158%2F1535-7163.MCT-11-0543" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref216/cit216&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=22090421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref216/cit216&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVWnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=194-203&author=K.+A.+Nolanauthor=M.+S.+Dunstanauthor=M.+C.+Caraherauthor=K.+A.+Scottauthor=D.+Leysauthor=I.+J.+Stratford&title=In+silico+screening+reveals+structurally+diverse%2C+nanomolar+inhibitors+of+NQO2+that+are+functionally+active+in+cells+and+can+modulate+NF-kappaB+signaling&doi=10.1158%2F1535-7163.MCT-11-0543"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit216R"><div class="casContent"><span class="casTitleNuber">216</span><div class="casTitle"><span class="NLM_cas:atitle">In Silico Screening Reveals Structurally Diverse, Nanomolar Inhibitors of NQO2 That Are Functionally Active in Cells and Can Modulate NF-κB Signaling</span></div><div class="casAuthors">Nolan, Karen A.; Dunstan, Mark S.; Caraher, Mary C.; Scott, Katherine A.; Leys, David; Stratford, Ian J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">194-203</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The National Cancer Institute chem. database has been screened using in silico docking to identify novel nanomolar inhibitors of NRH:quinone oxidoreductase 2 (NQO2).  The inhibitors identified from the screen exhibit a diverse range of scaffolds and the structure of one of the inhibitors, NSC13000 cocrystalized with NQO2, has been solved.  This has been used to aid the generation of a structure-activity relationship between the computationally derived binding affinity and exptl. measured enzyme inhibitory potency.  Many of the compds. are functionally active as inhibitors of NQO2 in cells at nontoxic concns.  To show this, advantage was taken of the NQO2-mediated toxicity of the chemotherapeutic drug CB1954.  The toxicity of this drug is substantially reduced when the function of NQO2 is inhibited, and many of the compds. achieve this in cells at nanomolar concns.  The NQO2 inhibitors also attenuated TNFα-mediated, NF-[kcy]B-driven transcriptional activity.  The link between NQO2 and the regulation of NF-[kcy]B was confirmed by using short interfering RNA to NQO2 and by the observation that NRH, the cofactor for NQO2 enzyme activity, could regulate NF-[kcy]B activity in an NQO2-dependent manner.  NF-[kcy]B is a potential therapeutic target and this study reveals an underlying mechanism that may be usable for developing new anticancer drugs.  Mol Cancer Ther; 11(1); 194-203.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8u03ajpk-crVg90H21EOLACvtfcHk0lhhgygIRVP6SA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVWnsQ%253D%253D&md5=4418803505ef49def2b9d44944d3cc62</span></div><a href="/servlet/linkout?suffix=cit216&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-11-0543&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-11-0543%26sid%3Dliteratum%253Aachs%26aulast%3DNolan%26aufirst%3DK.%2BA.%26aulast%3DDunstan%26aufirst%3DM.%2BS.%26aulast%3DCaraher%26aufirst%3DM.%2BC.%26aulast%3DScott%26aufirst%3DK.%2BA.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DStratford%26aufirst%3DI.%2BJ.%26atitle%3DIn%2520silico%2520screening%2520reveals%2520structurally%2520diverse%252C%2520nanomolar%2520inhibitors%2520of%2520NQO2%2520that%2520are%2520functionally%2520active%2520in%2520cells%2520and%2520can%2520modulate%2520NF-kappaB%2520signaling%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2012%26volume%3D11%26spage%3D194%26epage%3D203%26doi%3D10.1158%2F1535-7163.MCT-11-0543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref217"><div class="reference"><strong class="refLabel"><a href="#ref217" class="rightTabRefNumLink">217</a></strong><div class="NLM_citation" id="rightTab-cit217"><span><span class="NLM_contrib-group">Borriello, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montes, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lepelletier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leforban, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, W. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demange, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delhomme, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pavoni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarray, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boucher, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dufour, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hermine, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garbay, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hadj-Slimane, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raynaud, F.</span><span> </span><span class="NLM_article-title">Structure-based discovery of a small non-peptidic Neuropilins antagonist exerting in vitro and in vivo anti-tumor activity on breast cancer model</span> <span class="citation_source-journal">Cancer Lett. (N. Y., NY, U. S.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">349</span><span class="NLM_x">, </span> <span class="NLM_fpage">120</span><span class="NLM_x">–</span> <span class="NLM_lpage">127</span><span class="refDoi"> DOI: 10.1016/j.canlet.2014.04.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref217/cit217&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.canlet.2014.04.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref217/cit217&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24752068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref217/cit217&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnvFSltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=349&publication_year=2014&pages=120-127&author=L.+Borrielloauthor=M.+Montesauthor=Y.+Lepelletierauthor=B.+Leforbanauthor=W.+Q.+Liuauthor=L.+Demangeauthor=B.+Delhommeauthor=S.+Pavoniauthor=R.+Jarrayauthor=J.+L.+Boucherauthor=S.+Dufourauthor=O.+Hermineauthor=C.+Garbayauthor=R.+Hadj-Slimaneauthor=F.+Raynaud&title=Structure-based+discovery+of+a+small+non-peptidic+Neuropilins+antagonist+exerting+in+vitro+and+in+vivo+anti-tumor+activity+on+breast+cancer+model&doi=10.1016%2Fj.canlet.2014.04.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit217R"><div class="casContent"><span class="casTitleNuber">217</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of a small non-peptidic Neuropilins antagonist exerting in vitro and in vivo anti-tumor activity on breast cancer model</span></div><div class="casAuthors">Borriello, Lucia; Montes, Matthieu; Lepelletier, Yves; Leforban, Bertrand; Liu, Wang-Qing; Demange, Luc; Delhomme, Brigitte; Pavoni, Serena; Jarray, Rafika; Boucher, Jean Luc; Dufour, Sylvie; Hermine, Olivier; Garbay, Christiane; Hadj-Slimane, Reda; Raynaud, Francoise</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">349</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">120-127</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Neuropilin-1/-2 (+33 NRPs), VEGF-A165 co-receptors, are over-expressed during cancer progression.  Thus, NRPs targeted drug development is challenged using a multistep in silico/in vitro screening procedure.  The first fully non-peptidic VEGF-A165/NRPs protein-protein interaction antagonist (IC50 = 34 μM) without effect on pro-angiogenic kinases has been identified (compd.-1).  This hit showed breast cancer cells anti-proliferative activity (IC50 = 0.60 μM).  Compd.-1 treated NOG-xenografted mice significantly exerted tumor growth inhibition, which is correlated with Ki-67low expression and apoptosis.  Furthermore, CD31+/CD34+ vessels are reduced in accordance with HUVEC-tube formation inhibition (IC50 = 0.20 μM).  Taking together, compd.-1 is the first fully org. inhibitor targeting NRPs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIJJGk2lBSDrVg90H21EOLACvtfcHk0lh-0iCzHgF9pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnvFSltL0%253D&md5=4a719d3c60463a3c4d7b1e28b3ab4393</span></div><a href="/servlet/linkout?suffix=cit217&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2014.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2014.04.004%26sid%3Dliteratum%253Aachs%26aulast%3DBorriello%26aufirst%3DL.%26aulast%3DMontes%26aufirst%3DM.%26aulast%3DLepelletier%26aufirst%3DY.%26aulast%3DLeforban%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DW.%2BQ.%26aulast%3DDemange%26aufirst%3DL.%26aulast%3DDelhomme%26aufirst%3DB.%26aulast%3DPavoni%26aufirst%3DS.%26aulast%3DJarray%26aufirst%3DR.%26aulast%3DBoucher%26aufirst%3DJ.%2BL.%26aulast%3DDufour%26aufirst%3DS.%26aulast%3DHermine%26aufirst%3DO.%26aulast%3DGarbay%26aufirst%3DC.%26aulast%3DHadj-Slimane%26aufirst%3DR.%26aulast%3DRaynaud%26aufirst%3DF.%26atitle%3DStructure-based%2520discovery%2520of%2520a%2520small%2520non-peptidic%2520Neuropilins%2520antagonist%2520exerting%2520in%2520vitro%2520and%2520in%2520vivo%2520anti-tumor%2520activity%2520on%2520breast%2520cancer%2520model%26jtitle%3DCancer%2520Lett.%2520%2528N.%2520Y.%252C%2520NY%252C%2520U.%2520S.%2529%26date%3D2014%26volume%3D349%26spage%3D120%26epage%3D127%26doi%3D10.1016%2Fj.canlet.2014.04.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref218"><div class="reference"><strong class="refLabel"><a href="#ref218" class="rightTabRefNumLink">218</a></strong><div class="NLM_citation" id="rightTab-cit218"><span><span class="NLM_contrib-group">Choi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ko, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, S.</span><span> </span><span class="NLM_article-title">Structure-based virtual screening approach to the discovery of p38 MAP kinase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">2195</span><span class="NLM_x">–</span> <span class="NLM_lpage">2199</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2012.01.104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref218/cit218&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2012.01.104" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=2195-2199&author=H.+Choiauthor=H.+J.+Parkauthor=J.+C.+Shinauthor=H.+S.+Koauthor=J.+K.+Leeauthor=S.+Leeauthor=H.+Parkauthor=S.+Hong&title=Structure-based+virtual+screening+approach+to+the+discovery+of+p38+MAP+kinase+inhibitors&doi=10.1016%2Fj.bmcl.2012.01.104"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit218&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.01.104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.01.104%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DH.%26aulast%3DPark%26aufirst%3DH.%2BJ.%26aulast%3DShin%26aufirst%3DJ.%2BC.%26aulast%3DKo%26aufirst%3DH.%2BS.%26aulast%3DLee%26aufirst%3DJ.%2BK.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DHong%26aufirst%3DS.%26atitle%3DStructure-based%2520virtual%2520screening%2520approach%2520to%2520the%2520discovery%2520of%2520p38%2520MAP%2520kinase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D2195%26epage%3D2199%26doi%3D10.1016%2Fj.bmcl.2012.01.104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref219"><div class="reference"><strong class="refLabel"><a href="#ref219" class="rightTabRefNumLink">219</a></strong><div class="NLM_citation" id="rightTab-cit219"><span><span class="NLM_contrib-group">Pala, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevaert, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dallocchio, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dessi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogolino, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carcelli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanna, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sechi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naesens, L.</span><span> </span><span class="NLM_article-title">Virtual screening and biological validation of novel influenza virus PA endonuclease Inhibitors</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">866</span><span class="NLM_x">–</span> <span class="NLM_lpage">871</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00109</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00109" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref219/cit219&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVagu7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=866-871&author=N.+Palaauthor=A.+Stevaertauthor=R.+Dallocchioauthor=A.+Dessiauthor=D.+Rogolinoauthor=M.+Carcelliauthor=V.+Sannaauthor=M.+Sechiauthor=L.+Naesens&title=Virtual+screening+and+biological+validation+of+novel+influenza+virus+PA+endonuclease+Inhibitors&doi=10.1021%2Facsmedchemlett.5b00109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit219R"><div class="casContent"><span class="casTitleNuber">219</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Screening and Biological Validation of Novel Influenza Virus PA Endonuclease Inhibitors</span></div><div class="casAuthors">Pala, Nicolino; Stevaert, Annelies; Dallocchio, Roberto; Dessi, Alessandro; Rogolino, Dominga; Carcelli, Mauro; Sanna, Vanna; Sechi, Mario; Naesens, Lieve</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">866-871</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The influenza virus RNA-dependent RNA polymerase complex (RdRp), a heterotrimeric protein complex responsible for viral RNA transcription and replication, represents a primary target for antiviral drug development.  One particularly attractive approach is interference with the endonucleolytic "cap-snatching" reaction by the RdRp subunit PA, more precisely by inhibiting its metal-dependent catalytic activity which resides in the N-terminal part of PA (PA-Nter).  Almost all PA inhibitors (PAIs) thus far discovered bear pharmacophoric fragments with chelating motifs able to bind the bivalent metal ions in the catalytic core of PA-Nter.  More recently, the availability of crystallog. structures of PA-Nter has enabled rational design of original PAIs with improved binding properties and antiviral potency.  We here present a coupled pharmacophore/docking virtual screening approach that allowed us to identify PAIs with interesting inhibitory activity in a PA-Nter enzymic assay.  Moreover, antiviral activity in the low micromolar range was obsd. in cell-based influenza virus assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKH888RsSFzrVg90H21EOLACvtfcHk0lh-0iCzHgF9pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVagu7bJ&md5=81f9b315013d32ad71fbc7f6c8ce2e62</span></div><a href="/servlet/linkout?suffix=cit219&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00109%26sid%3Dliteratum%253Aachs%26aulast%3DPala%26aufirst%3DN.%26aulast%3DStevaert%26aufirst%3DA.%26aulast%3DDallocchio%26aufirst%3DR.%26aulast%3DDessi%26aufirst%3DA.%26aulast%3DRogolino%26aufirst%3DD.%26aulast%3DCarcelli%26aufirst%3DM.%26aulast%3DSanna%26aufirst%3DV.%26aulast%3DSechi%26aufirst%3DM.%26aulast%3DNaesens%26aufirst%3DL.%26atitle%3DVirtual%2520screening%2520and%2520biological%2520validation%2520of%2520novel%2520influenza%2520virus%2520PA%2520endonuclease%2520Inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D866%26epage%3D871%26doi%3D10.1021%2Facsmedchemlett.5b00109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref220"><div class="reference"><strong class="refLabel"><a href="#ref220" class="rightTabRefNumLink">220</a></strong><div class="NLM_citation" id="rightTab-cit220"><span><span class="NLM_contrib-group">Di Leva, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zizza, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cingolani, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">D’Angelo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pagano, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amato, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salvati, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sissi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pinato, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marinelli, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosconati, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novellino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Randazzo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biroccio, A.</span><span> </span><span class="NLM_article-title">Exploring the chemical space of G-quadruplex binders: discovery of a novel chemotype targeting the human telomeric sequence</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">9646</span><span class="NLM_x">–</span> <span class="NLM_lpage">9654</span><span class="refDoi"> DOI: 10.1021/jm401185b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401185b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref220/cit220&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVajtbvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9646-9654&author=F.+S.+Di+Levaauthor=P.+Zizzaauthor=C.+Cingolaniauthor=C.+D%E2%80%99Angeloauthor=B.+Paganoauthor=J.+Amatoauthor=E.+Salvatiauthor=C.+Sissiauthor=O.+Pinatoauthor=L.+Marinelliauthor=A.+Cavalliauthor=S.+Cosconatiauthor=E.+Novellinoauthor=A.+Randazzoauthor=A.+Biroccio&title=Exploring+the+chemical+space+of+G-quadruplex+binders%3A+discovery+of+a+novel+chemotype+targeting+the+human+telomeric+sequence&doi=10.1021%2Fjm401185b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit220R"><div class="casContent"><span class="casTitleNuber">220</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring the Chemical Space of G-Quadruplex Binders: Discovery of a Novel Chemotype Targeting the Human Telomeric Sequence</span></div><div class="casAuthors">Di Leva, Francesco Saverio; Zizza, Pasquale; Cingolani, Chiara; D'Angelo, Carmen; Pagano, Bruno; Amato, Jussara; Salvati, Erica; Sissi, Claudia; Pinato, Odra; Marinelli, Luciana; Cavalli, Andrea; Cosconati, Sandro; Novellino, Ettore; Randazzo, Antonio; Biroccio, Annamaria</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9646-9654</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recent findings have unambiguously demonstrated that DNA G-rich sequences can adopt a G-quadruplex folding in living cells, thus further validating them as crucial targets for anticancer therapy.  Herein, to identify new potent G4 binders as antitumor drug candidates, the authors have targeted a 24-nt G4-forming telomeric sequence employing a receptor-based virtual screening approach.  Among the best candidates, in vitro binding expts. allowed identification of three novel G4 ligands.  Among them, the best compd. features an unprecedented binding selectivity for the human telomeric DNA G-quadruplex with no detectable binding for other G4-forming sequences present at different genomic sites.  This behavior correlates with the detected ability to generate DNA damage response in tumor cells at the telomeric level and efficient antiproliferative effect on different tumor cell lines at low micromolar concns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolkWbQ7-ptEbVg90H21EOLACvtfcHk0lh-0iCzHgF9pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVajtbvK&md5=05c07f98198d5cdc1da2de690182184e</span></div><a href="/servlet/linkout?suffix=cit220&amp;dbid=16384&amp;doi=10.1021%2Fjm401185b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401185b%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BLeva%26aufirst%3DF.%2BS.%26aulast%3DZizza%26aufirst%3DP.%26aulast%3DCingolani%26aufirst%3DC.%26aulast%3DD%25E2%2580%2599Angelo%26aufirst%3DC.%26aulast%3DPagano%26aufirst%3DB.%26aulast%3DAmato%26aufirst%3DJ.%26aulast%3DSalvati%26aufirst%3DE.%26aulast%3DSissi%26aufirst%3DC.%26aulast%3DPinato%26aufirst%3DO.%26aulast%3DMarinelli%26aufirst%3DL.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DCosconati%26aufirst%3DS.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DRandazzo%26aufirst%3DA.%26aulast%3DBiroccio%26aufirst%3DA.%26atitle%3DExploring%2520the%2520chemical%2520space%2520of%2520G-quadruplex%2520binders%253A%2520discovery%2520of%2520a%2520novel%2520chemotype%2520targeting%2520the%2520human%2520telomeric%2520sequence%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9646%26epage%3D9654%26doi%3D10.1021%2Fjm401185b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref221"><div class="reference"><strong class="refLabel"><a href="#ref221" class="rightTabRefNumLink">221</a></strong><div class="NLM_citation" id="rightTab-cit221"><span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span> </span><span class="NLM_article-title">Discovery of potent small molecule inhibitors of DYRK1A by structure-based virtual screening and bioassay</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">168</span><span class="NLM_x">–</span> <span class="NLM_lpage">171</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2011.11.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref221/cit221&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2011.11.043" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=168-171&author=D.+Wangauthor=F.+Wangauthor=Y.+Tanauthor=L.+Dongauthor=L.+Chenauthor=W.+Zhuauthor=H.+Wang&title=Discovery+of+potent+small+molecule+inhibitors+of+DYRK1A+by+structure-based+virtual+screening+and+bioassay&doi=10.1016%2Fj.bmcl.2011.11.043"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit221&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.11.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.11.043%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DTan%26aufirst%3DY.%26aulast%3DDong%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DH.%26atitle%3DDiscovery%2520of%2520potent%2520small%2520molecule%2520inhibitors%2520of%2520DYRK1A%2520by%2520structure-based%2520virtual%2520screening%2520and%2520bioassay%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D168%26epage%3D171%26doi%3D10.1016%2Fj.bmcl.2011.11.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref222"><div class="reference"><strong class="refLabel"><a href="#ref222" class="rightTabRefNumLink">222</a></strong><div class="NLM_citation" id="rightTab-cit222"><span><span class="NLM_contrib-group">de Graaf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piwnica, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giordanetto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Structure-based discovery of allosteric modulators of two related class B G-protein-coupled receptors</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">2159</span><span class="NLM_x">–</span> <span class="NLM_lpage">2169</span><span class="refDoi"> DOI: 10.1002/cmdc.201100317</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref222/cit222&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fcmdc.201100317" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref222/cit222&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=21994134" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref222/cit222&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht12qsbfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=2159-2169&author=C.+de+Graafauthor=C.+Reinauthor=D.+Piwnicaauthor=F.+Giordanettoauthor=D.+Rognan&title=Structure-based+discovery+of+allosteric+modulators+of+two+related+class+B+G-protein-coupled+receptors&doi=10.1002%2Fcmdc.201100317"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit222R"><div class="casContent"><span class="casTitleNuber">222</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of Allosteric Modulators of Two Related Class B G-Protein-Coupled Receptors</span></div><div class="casAuthors">de Graaf, Chris; Rein, Chantal; Piwnica, David; Giordanetto, Fabrizio; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2159-2169</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Despite the availability of X-ray crystal structure data for several members of the G-protein-coupled receptor (GPCR) superfamily, structure-based discovery of GPCR ligands has been exclusively restricted to class A (rhodopsin-like) receptors.  Herein we report the identification, by a docking-based virtual screening approach, of noncompetitive ligands for two related class B (secretin-like) GPCRs: the glucagon receptor (GLR) and the glucagon-like peptide 1 receptor (GLP-1R).  Starting from a knowledge-based three-dimensional model of the GLR, a database of 1.9 million com. available drug-like compds. was screened for chem. similarity to existing GLR noncompetitive antagonists and docked to the transmembrane cavity of the GLR; 23 compds. were then selected based on protein-ligand interaction fingerprints, and were then purchased and evaluated for in vitro binding to GLR and modulation of glucagon-induced cAMP release.  Two of the 23 compds. inhibited the effect of glucagon in a dose-dependent manner, with one inhibitor exhibiting the same potency as L-168 049, a ref. noncompetitive GLR antagonist, in a whole-cell-based functional assay.  Interestingly, one virtual hit that was inactive at the GLR was shown to bind to GLP-1R and potentiate the response to the endogenous GLP-1 ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLnGlSRCe5d7Vg90H21EOLACvtfcHk0lioqitrsqdt0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht12qsbfP&md5=ea40e9f9394d32b7a084dcf5048f2b32</span></div><a href="/servlet/linkout?suffix=cit222&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201100317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201100317%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26aulast%3DRein%26aufirst%3DC.%26aulast%3DPiwnica%26aufirst%3DD.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DStructure-based%2520discovery%2520of%2520allosteric%2520modulators%2520of%2520two%2520related%2520class%2520B%2520G-protein-coupled%2520receptors%26jtitle%3DChemMedChem%26date%3D2011%26volume%3D6%26spage%3D2159%26epage%3D2169%26doi%3D10.1002%2Fcmdc.201100317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref223"><div class="reference"><strong class="refLabel"><a href="#ref223" class="rightTabRefNumLink">223</a></strong><div class="NLM_citation" id="rightTab-cit223"><span><span class="NLM_contrib-group">Kalid, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toledo Warshaviak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shechter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shacham, S.</span><span> </span><span class="NLM_article-title">Consensus Induced Fit Docking (cIFD): methodology, validation, and application to the discovery of novel Crm1 inhibitors</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1217</span><span class="NLM_x">–</span> <span class="NLM_lpage">1228</span><span class="refDoi"> DOI: 10.1007/s10822-012-9611-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref223/cit223&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs10822-012-9611-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=1217-1228&author=O.+Kalidauthor=D.+Toledo+Warshaviakauthor=S.+Shechterauthor=W.+Shermanauthor=S.+Shacham&title=Consensus+Induced+Fit+Docking+%28cIFD%29%3A+methodology%2C+validation%2C+and+application+to+the+discovery+of+novel+Crm1+inhibitors&doi=10.1007%2Fs10822-012-9611-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit223&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9611-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9611-9%26sid%3Dliteratum%253Aachs%26aulast%3DKalid%26aufirst%3DO.%26aulast%3DToledo%2BWarshaviak%26aufirst%3DD.%26aulast%3DShechter%26aufirst%3DS.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DShacham%26aufirst%3DS.%26atitle%3DConsensus%2520Induced%2520Fit%2520Docking%2520%2528cIFD%2529%253A%2520methodology%252C%2520validation%252C%2520and%2520application%2520to%2520the%2520discovery%2520of%2520novel%2520Crm1%2520inhibitors%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D1217%26epage%3D1228%26doi%3D10.1007%2Fs10822-012-9611-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref224"><div class="reference"><strong class="refLabel"><a href="#ref224" class="rightTabRefNumLink">224</a></strong><div class="NLM_citation" id="rightTab-cit224"><span><span class="NLM_contrib-group">Chang, C. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeyachandran, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, M. H.</span><span> </span><span class="NLM_article-title">Structure-based virtual screening and experimental validation of the discovery of inhibitors targeted towards the human coronavirus nucleocapsid protein</span> <span class="citation_source-journal">Mol. BioSyst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">59</span><span class="NLM_x">–</span> <span class="NLM_lpage">66</span><span class="refDoi"> DOI: 10.1039/C5MB00582E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref224/cit224&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1039%2FC5MB00582E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref224/cit224&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26542199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref224/cit224&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1yktrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=59-66&author=C.+K.+Changauthor=S.+Jeyachandranauthor=N.+J.+Huauthor=C.+L.+Liuauthor=S.+Y.+Linauthor=Y.+S.+Wangauthor=Y.+M.+Changauthor=M.+H.+Hou&title=Structure-based+virtual+screening+and+experimental+validation+of+the+discovery+of+inhibitors+targeted+towards+the+human+coronavirus+nucleocapsid+protein&doi=10.1039%2FC5MB00582E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit224R"><div class="casContent"><span class="casTitleNuber">224</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based virtual screening and experimental validation of the discovery of inhibitors targeted towards the human coronavirus nucleocapsid protein</span></div><div class="casAuthors">Chang, Chung-ke; Jeyachandran, Sivakamavalli; Hu, Nien-Jen; Liu, Chia-Ling; Lin, Shing-Yen; Wang, Yong-Sheng; Chang, Yu-Ming; Hou, Ming-Hon</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">59-66</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Nucleocapsid protein (NP), an essential RNA-binding viral protein in human coronavirus (CoV)-infected cells, is required for the replication and transcription of viral RNA.  Recent studies suggested that human CoV NP is a valid target for antiviral drug development.  Based on this aspect, structure-based virtual screening targeting nucleocapsid protein (NP) was performed to identify good chem. starting points for medicinal chem.  The present study utilized structure-based virtual screening against human CoV-OC43 using the Zinc database, which is performed through docking with varying precisions and computational intensities to identify eight potential compds.  The chosen potential leads were further validated exptl. using biophys. means.  Surface plasmon resonance (SPR) anal. indicated that one among the potential leads, 6-chloro-7-(2-morpholin-4-yl-ethylamino) quinoxaline-5,8-dione (small-compd. H3), exhibited a significant decrease of RNA-binding capacity of NP by more than 20%.  The loss of binding activity was manifested as a 20% decrease in the min. on-rate accompanied with a 70% increase in the max. off-rate.  Fluorescence titrn. and X-ray crystallog. studies indicated that H3 antagonizes the binding between HCoV-OC43 NP and RNA by interacting with the N-terminal domain of the NP.  Our findings provide insight into the development of new therapeutics that disrupt the interaction between RNA and viral NP in the HCoV.  The discovery of the new compd. would be an impetus to design novel NP inhibitors against human CoV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrR4yjm6xLe7rVg90H21EOLACvtfcHk0lioqitrsqdt0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1yktrzL&md5=31ef327deaa11323822157670a539092</span></div><a href="/servlet/linkout?suffix=cit224&amp;dbid=16384&amp;doi=10.1039%2FC5MB00582E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5MB00582E%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DC.%2BK.%26aulast%3DJeyachandran%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DN.%2BJ.%26aulast%3DLiu%26aufirst%3DC.%2BL.%26aulast%3DLin%26aufirst%3DS.%2BY.%26aulast%3DWang%26aufirst%3DY.%2BS.%26aulast%3DChang%26aufirst%3DY.%2BM.%26aulast%3DHou%26aufirst%3DM.%2BH.%26atitle%3DStructure-based%2520virtual%2520screening%2520and%2520experimental%2520validation%2520of%2520the%2520discovery%2520of%2520inhibitors%2520targeted%2520towards%2520the%2520human%2520coronavirus%2520nucleocapsid%2520protein%26jtitle%3DMol.%2520BioSyst.%26date%3D2016%26volume%3D12%26spage%3D59%26epage%3D66%26doi%3D10.1039%2FC5MB00582E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref225"><div class="reference"><strong class="refLabel"><a href="#ref225" class="rightTabRefNumLink">225</a></strong><div class="NLM_citation" id="rightTab-cit225"><span><span class="NLM_contrib-group">Cho, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ioerger, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sacchettini, J. C.</span><span> </span><span class="NLM_article-title">Discovery of novel nitrobenzothiazole inhibitors for Mycobacterium tuberculosis ATP phosphoribosyl transferase (HisG) through virtual screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">5984</span><span class="NLM_x">–</span> <span class="NLM_lpage">5992</span><span class="refDoi"> DOI: 10.1021/jm800328v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800328v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref225/cit225&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVymtb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=5984-5992&author=Y.+Choauthor=T.+R.+Ioergerauthor=J.+C.+Sacchettini&title=Discovery+of+novel+nitrobenzothiazole+inhibitors+for+Mycobacterium+tuberculosis+ATP+phosphoribosyl+transferase+%28HisG%29+through+virtual+screening&doi=10.1021%2Fjm800328v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit225R"><div class="casContent"><span class="casTitleNuber">225</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Nitrobenzothiazole Inhibitors for Mycobacterium tuberculosis ATP Phosphoribosyl Transferase (HisG) through Virtual Screening</span></div><div class="casAuthors">Cho, Yoonsang; Ioerger, Thomas R.; Sacchettini, James C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5984-5992</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HisG is an ATP-phosphoribosyl transferase (ATPPRTase) that catalyzes the first step in the biosynthetic pathway for histidine.  Among the enzymes in this pathway, only HisG represents a potential drug target for tuberculosis.  Only a few inhibitors with limited potency for HisG are currently known.  To discover more potent and diverse inhibitors, virtual screening was performed.  The crystal structure of M. tuberculosis HisG has been solved and reveals a large, solvent-exposed active site with subsites for ATP and PRPP substrates.  Two docking algorithms, GOLD and FLEXX, were used to screen two large libraries, Chembridge and NCI, contg. over 500000 compds. combined.  An initial subset of top-ranked compds. were selected and assayed, and seven were found to have enzyme inhibition activity at micromolar concns.  Several of the hits contained a nitrobenzothiazole fragment, which was predicted to dock into the monophosphate-binding loop, and this binding mode was confirmed by crystallog. evidence.  A secondary screen was performed to identify compds. with similar structures.  Several of these also exhibited micromolar inhibition.  Furthermore, two of the compds. showed bacteriocidal activity in a whole-cell assay against Mycobacterium smegmatis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbgXaEKz2UCrVg90H21EOLACvtfcHk0liVtaIcp_FUfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVymtb3L&md5=dfec06f998fffff9c121e027616c1dbd</span></div><a href="/servlet/linkout?suffix=cit225&amp;dbid=16384&amp;doi=10.1021%2Fjm800328v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800328v%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DY.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26atitle%3DDiscovery%2520of%2520novel%2520nitrobenzothiazole%2520inhibitors%2520for%2520Mycobacterium%2520tuberculosis%2520ATP%2520phosphoribosyl%2520transferase%2520%2528HisG%2529%2520through%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D5984%26epage%3D5992%26doi%3D10.1021%2Fjm800328v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref226"><div class="reference"><strong class="refLabel"><a href="#ref226" class="rightTabRefNumLink">226</a></strong><div class="NLM_citation" id="rightTab-cit226"><span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jing, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, G.</span><span> </span><span class="NLM_article-title">Discovery of novel androgen receptor antagonists: a hybrid approach of pharmacophore-based and docking-based virtual screening</span> <span class="citation_source-journal">Anti-Cancer Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">747</span><span class="NLM_x">–</span> <span class="NLM_lpage">753</span><span class="refDoi"> DOI: 10.1097/CAD.0000000000000245</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref226/cit226&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1097%2FCAD.0000000000000245" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2015&pages=747-753&author=J.+Liuauthor=B.+Liuauthor=G.+Guoauthor=Y.+Jingauthor=G.+Zhao&title=Discovery+of+novel+androgen+receptor+antagonists%3A+a+hybrid+approach+of+pharmacophore-based+and+docking-based+virtual+screening&doi=10.1097%2FCAD.0000000000000245"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit226&amp;dbid=16384&amp;doi=10.1097%2FCAD.0000000000000245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FCAD.0000000000000245%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DGuo%26aufirst%3DG.%26aulast%3DJing%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520novel%2520androgen%2520receptor%2520antagonists%253A%2520a%2520hybrid%2520approach%2520of%2520pharmacophore-based%2520and%2520docking-based%2520virtual%2520screening%26jtitle%3DAnti-Cancer%2520Drugs%26date%3D2015%26volume%3D26%26spage%3D747%26epage%3D753%26doi%3D10.1097%2FCAD.0000000000000245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref227"><div class="reference"><strong class="refLabel"><a href="#ref227" class="rightTabRefNumLink">227</a></strong><div class="NLM_citation" id="rightTab-cit227"><span><span class="NLM_contrib-group">Wiggers, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocha, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandes, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sesti-Costa, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carneiro, Z. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheleski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">da Silva, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Juliano, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cezari, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silva, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKerrow, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montanari, C. A.</span><span> </span><span class="NLM_article-title">Non-peptidic cruzain inhibitors with trypanocidal activity discovered by virtual screening and in vitro assay</span> <span class="citation_source-journal">PLoS Neglected Trop. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e2370</span><span class="refDoi"> DOI: 10.1371/journal.pntd.0002370</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref227/cit227&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pntd.0002370" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref227/cit227&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=23991231" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref227/cit227&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVOrtrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2013&pages=e2370&author=H.+J.+Wiggersauthor=J.+R.+Rochaauthor=W.+B.+Fernandesauthor=R.+Sesti-Costaauthor=Z.+A.+Carneiroauthor=J.+Cheleskiauthor=A.+B.+da+Silvaauthor=L.+Julianoauthor=M.+H.+Cezariauthor=J.+S.+Silvaauthor=J.+H.+McKerrowauthor=C.+A.+Montanari&title=Non-peptidic+cruzain+inhibitors+with+trypanocidal+activity+discovered+by+virtual+screening+and+in+vitro+assay&doi=10.1371%2Fjournal.pntd.0002370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit227R"><div class="casContent"><span class="casTitleNuber">227</span><div class="casTitle"><span class="NLM_cas:atitle">Non-peptidic cruzain inhibitors with trypanocidal activity discovered by virtual screening and in vitro assay</span></div><div class="casAuthors">Wiggers, Helton J.; Rocha, Josmar R.; Fernandes, William B.; Sesti-Costa, Renata; Carneiro, Zumira A.; Cheleski, Juliana; da Silva, Alberico B. F.; Juliano, Luiz; Cezari, Maria H. S.; Silva, Joao S.; McKerrow, James H.; Montanari, Carlos A.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Neglected Tropical Diseases</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e2370, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">PNTDAM</span>;
        ISSN:<span class="NLM_cas:issn">1935-2735</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">A multi-step cascade strategy using integrated ligand- and target-based virtual screening methods was developed to select a small no. of compds. from the ZINC database to be evaluated for trypanocidal activity.  Winnowing the database to 23 selected compds., 12 non-covalent binding cruzain inhibitors with affinity values (Ki) in the low micromolar range (3-60 μM) acting through a competitive inhibition mechanism were identified.  This mechanism has been confirmed by detg. the binding mode of the cruzain inhibitor Nequimed176 through X-ray crystallog. studies.  Cruzain, a validated therapeutic target for new chemotherapy for Chagas disease, also shares high similarity with the mammalian homolog cathepsin L.  Because increased activity of cathepsin L is related to invasive properties and has been linked to metastatic cancer cells, cruzain inhibitors from the same library were assayed against it.  Affinity values were in a similar range (4-80 μM), yielding poor selectivity towards cruzain but raising the possibility of investigating such inhibitors for their effect on cell proliferation.  In order to select the most promising enzyme inhibitors retaining trypanocidal activity for structure-activity relationship (SAR) studies, the most potent cruzain inhibitors were assayed against T. cruzi-infected cells.  Two compds. were found to have trypanocidal activity.  Using compd. Nequimed42 as precursor, an SAR was established in which the 2-acetamidothiophene-3-carboxamide group was identified as essential for enzyme and parasite inhibition activities.  The IC50 value for compd. Nequimed42 acting against the trypomastigote form of the Tulahuen lacZ strain was found to be 10.6 ± 0.1 μM, tenfold lower than that obtained for benznidazole, which was taken as pos. control.  In addn., by employing the strategy of mol. simplification, a smaller compd. derived from Nequimed42 with a ligand efficiency (LE) of 0.33 kcal mol-1 atom-1 (compd. Nequimed176) is highlighted as a novel non-peptidic, non-covalent cruzain inhibitor as a trypanocidal agent candidate for optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomw90Q6T0IA7Vg90H21EOLACvtfcHk0liVtaIcp_FUfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVOrtrfO&md5=60b21e0f11e9a35cbd73387a9011dea0</span></div><a href="/servlet/linkout?suffix=cit227&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pntd.0002370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pntd.0002370%26sid%3Dliteratum%253Aachs%26aulast%3DWiggers%26aufirst%3DH.%2BJ.%26aulast%3DRocha%26aufirst%3DJ.%2BR.%26aulast%3DFernandes%26aufirst%3DW.%2BB.%26aulast%3DSesti-Costa%26aufirst%3DR.%26aulast%3DCarneiro%26aufirst%3DZ.%2BA.%26aulast%3DCheleski%26aufirst%3DJ.%26aulast%3Dda%2BSilva%26aufirst%3DA.%2BB.%26aulast%3DJuliano%26aufirst%3DL.%26aulast%3DCezari%26aufirst%3DM.%2BH.%26aulast%3DSilva%26aufirst%3DJ.%2BS.%26aulast%3DMcKerrow%26aufirst%3DJ.%2BH.%26aulast%3DMontanari%26aufirst%3DC.%2BA.%26atitle%3DNon-peptidic%2520cruzain%2520inhibitors%2520with%2520trypanocidal%2520activity%2520discovered%2520by%2520virtual%2520screening%2520and%2520in%2520vitro%2520assay%26jtitle%3DPLoS%2520Neglected%2520Trop.%2520Dis.%26date%3D2013%26volume%3D7%26spage%3De2370%26doi%3D10.1371%2Fjournal.pntd.0002370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref228"><div class="reference"><strong class="refLabel"><a href="#ref228" class="rightTabRefNumLink">228</a></strong><div class="NLM_citation" id="rightTab-cit228"><span><span class="NLM_contrib-group">Park, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oh, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryu, S. E.</span><span> </span><span class="NLM_article-title">Identification of potent VHZ phosphatase inhibitors with structure-based virtual screening</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">226</span><span class="NLM_x">–</span> <span class="NLM_lpage">231</span><span class="refDoi"> DOI: 10.1177/1087057112463067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref228/cit228&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1177%2F1087057112463067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref228/cit228&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=23042075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref228/cit228&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Okt77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=226-231&author=H.+Parkauthor=S.+Y.+Parkauthor=J.+J.+Ohauthor=S.+E.+Ryu&title=Identification+of+potent+VHZ+phosphatase+inhibitors+with+structure-based+virtual+screening&doi=10.1177%2F1087057112463067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit228R"><div class="casContent"><span class="casTitleNuber">228</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of potent VHZ phosphatase inhibitors with structure-based virtual screening</span></div><div class="casAuthors">Park, Hwangseo; Park, So Ya; Oh, Jung Jin; Ryu, Seong Eon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">226-231, 6 pp.</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">VH1-like phosphatase Z (VHZ) has proved to be a promising target for the development of therapeutics for the treatment of human cancers.  Here, we report the first example for a successful application of structure-based virtual screening to identify the novel small-mol. inhibitors of VHZ.  These inhibitors revealed high potencies with the assocd. IC50 values ranging from 3 to 20 μM and were also screened for having desirable physicochem. properties as a drug candidate.  Therefore, they deserve consideration for further development by structure-activity relationship studies to optimize inhibitory and anticancer activities.  Structural features relevant to the stabilization of the newly identified inhibitors in the active site of VHZ are discussed in detail.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqShMNo_EGLu7Vg90H21EOLACvtfcHk0li1smftsQatEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Okt77J&md5=a4189c62904a6955b69b9546ede0ffb9</span></div><a href="/servlet/linkout?suffix=cit228&amp;dbid=16384&amp;doi=10.1177%2F1087057112463067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057112463067%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DH.%26aulast%3DPark%26aufirst%3DS.%2BY.%26aulast%3DOh%26aufirst%3DJ.%2BJ.%26aulast%3DRyu%26aufirst%3DS.%2BE.%26atitle%3DIdentification%2520of%2520potent%2520VHZ%2520phosphatase%2520inhibitors%2520with%2520structure-based%2520virtual%2520screening%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2013%26volume%3D18%26spage%3D226%26epage%3D231%26doi%3D10.1177%2F1087057112463067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref229"><div class="reference"><strong class="refLabel"><a href="#ref229" class="rightTabRefNumLink">229</a></strong><div class="NLM_citation" id="rightTab-cit229"><span><span class="NLM_contrib-group">Meng, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Discovery and optimization of novel, selective histone methyltransferase SET7 inhibitors by pharmacophore- and docking-based virtual screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">8166</span><span class="NLM_x">–</span> <span class="NLM_lpage">8181</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01154</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01154" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=8166-8181&author=F.+Mengauthor=S.+Chengauthor=H.+Dingauthor=S.+Liuauthor=Y.+Liuauthor=K.+Zhuauthor=S.+Chenauthor=J.+Luauthor=Y.+Xieauthor=L.+Liauthor=R.+Liuauthor=Z.+Shiauthor=Y.+Zhouauthor=Y.+C.+Liuauthor=M.+Zhengauthor=H.+Jiangauthor=W.+Luauthor=H.+Liuauthor=C.+Luo&title=Discovery+and+optimization+of+novel%2C+selective+histone+methyltransferase+SET7+inhibitors+by+pharmacophore-+and+docking-based+virtual+screening&doi=10.1021%2Facs.jmedchem.5b01154"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit229&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01154%26sid%3Dliteratum%253Aachs%26aulast%3DMeng%26aufirst%3DF.%26aulast%3DCheng%26aufirst%3DS.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520novel%252C%2520selective%2520histone%2520methyltransferase%2520SET7%2520inhibitors%2520by%2520pharmacophore-%2520and%2520docking-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D8166%26epage%3D8181%26doi%3D10.1021%2Facs.jmedchem.5b01154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref230"><div class="reference"><strong class="refLabel"><a href="#ref230" class="rightTabRefNumLink">230</a></strong><div class="NLM_citation" id="rightTab-cit230"><span><span class="NLM_contrib-group">Parmenopoulou, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kantsadi, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsirkone, V. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chatzileontiadou, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manta, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zographos, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Molfeta, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Archontis, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agius, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayes, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leonidas, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Komiotis, D.</span><span> </span><span class="NLM_article-title">Structure based inhibitor design targeting glycogen phosphorylase B. Virtual screening, synthesis, biochemical and biological assessment of novel N-acyl-beta-d-glucopyranosylamines</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">4810</span><span class="NLM_x">–</span> <span class="NLM_lpage">4825</span><span class="refDoi"> DOI: 10.1016/j.bmc.2014.06.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref230/cit230&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmc.2014.06.058" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=4810-4825&author=V.+Parmenopoulouauthor=A.+L.+Kantsadiauthor=V.+G.+Tsirkoneauthor=D.+S.+Chatzileontiadouauthor=S.+Mantaauthor=S.+E.+Zographosauthor=C.+Molfetaauthor=G.+Archontisauthor=L.+Agiusauthor=J.+M.+Hayesauthor=D.+D.+Leonidasauthor=D.+Komiotis&title=Structure+based+inhibitor+design+targeting+glycogen+phosphorylase+B.+Virtual+screening%2C+synthesis%2C+biochemical+and+biological+assessment+of+novel+N-acyl-beta-d-glucopyranosylamines&doi=10.1016%2Fj.bmc.2014.06.058"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit230&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2014.06.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2014.06.058%26sid%3Dliteratum%253Aachs%26aulast%3DParmenopoulou%26aufirst%3DV.%26aulast%3DKantsadi%26aufirst%3DA.%2BL.%26aulast%3DTsirkone%26aufirst%3DV.%2BG.%26aulast%3DChatzileontiadou%26aufirst%3DD.%2BS.%26aulast%3DManta%26aufirst%3DS.%26aulast%3DZographos%26aufirst%3DS.%2BE.%26aulast%3DMolfeta%26aufirst%3DC.%26aulast%3DArchontis%26aufirst%3DG.%26aulast%3DAgius%26aufirst%3DL.%26aulast%3DHayes%26aufirst%3DJ.%2BM.%26aulast%3DLeonidas%26aufirst%3DD.%2BD.%26aulast%3DKomiotis%26aufirst%3DD.%26atitle%3DStructure%2520based%2520inhibitor%2520design%2520targeting%2520glycogen%2520phosphorylase%2520B.%2520Virtual%2520screening%252C%2520synthesis%252C%2520biochemical%2520and%2520biological%2520assessment%2520of%2520novel%2520N-acyl-beta-d-glucopyranosylamines%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2014%26volume%3D22%26spage%3D4810%26epage%3D4825%26doi%3D10.1016%2Fj.bmc.2014.06.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref231"><div class="reference"><strong class="refLabel"><a href="#ref231" class="rightTabRefNumLink">231</a></strong><div class="NLM_citation" id="rightTab-cit231"><span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y.</span><span> </span><span class="NLM_article-title">Identification of diverse dipeptidyl peptidase IV inhibitors via structure-based virtual screening</span> <span class="citation_source-journal">J. Mol. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">4033</span><span class="NLM_x">–</span> <span class="NLM_lpage">4042</span><span class="refDoi"> DOI: 10.1007/s00894-012-1394-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref231/cit231&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1007%2Fs00894-012-1394-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref231/cit231&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=22460522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref231/cit231&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1OrtbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=4033-4042&author=C.+Liauthor=W.+Luauthor=C.+Luauthor=W.+Xiaoauthor=X.+Shenauthor=J.+Huangauthor=G.+Liuauthor=Y.+Tang&title=Identification+of+diverse+dipeptidyl+peptidase+IV+inhibitors+via+structure-based+virtual+screening&doi=10.1007%2Fs00894-012-1394-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit231R"><div class="casContent"><span class="casTitleNuber">231</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of diverse dipeptidyl peptidase IV inhibitors via structure-based virtual screening</span></div><div class="casAuthors">Li, Cui; Lu, Weiqiang; Lu, Chunhua; Xiao, Wen; Shen, Xu; Huang, Jin; Liu, Guixia; Tang, Yun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4033-4042</span>CODEN:
                <span class="NLM_cas:coden">JMMOFK</span>;
        ISSN:<span class="NLM_cas:issn">0948-5023</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Dipeptidyl peptidase IV (DPP4) is an important target for the treatment of type II diabetes mellitus.  Inhibition of DPP4 will improve glycemic control in such patients by preventing the rapid breakdown and thereby prolonging the physiol. actions of incretin hormones.  Known DPP4 inhibitors (including marketed drugs and those drug candidates) appear to share similar structural features: the cyanopyrrolidine moieties, the xanthenes/pyrimidine parts and amino-like linkages.  In this study, a multi-step virtual screening strategy including both rigid and flexible docking was employed to search for novel structures with DPP4 inhibition.  From SPECS database, consisting of over 190,000 com. available compds., 99 virtual hits were picked up and 15 of them were eventually identified to have DPP4 inhibitory activities at 5 ∼ 50 μM.  Diverse structures of our compds. were out of usual structural categories.  Hence a pharmacophore model was built to further explore their common binding features on the enzyme.  The results provided a new pathway for the discovery of DPP4 inhibitors and would be helpful for further optimization of DPP4 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0SD-Ww74h5rVg90H21EOLACvtfcHk0li1smftsQatEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1OrtbbL&md5=7a34282901e72ea605ad4a18d11f0e1b</span></div><a href="/servlet/linkout?suffix=cit231&amp;dbid=16384&amp;doi=10.1007%2Fs00894-012-1394-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00894-012-1394-3%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DXiao%26aufirst%3DW.%26aulast%3DShen%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DTang%26aufirst%3DY.%26atitle%3DIdentification%2520of%2520diverse%2520dipeptidyl%2520peptidase%2520IV%2520inhibitors%2520via%2520structure-based%2520virtual%2520screening%26jtitle%3DJ.%2520Mol.%2520Model.%26date%3D2012%26volume%3D18%26spage%3D4033%26epage%3D4042%26doi%3D10.1007%2Fs00894-012-1394-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref232"><div class="reference"><strong class="refLabel"><a href="#ref232" class="rightTabRefNumLink">232</a></strong><div class="NLM_citation" id="rightTab-cit232"><span><span class="NLM_contrib-group">Lagos, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vecchiola, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allende, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuentes, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tichauer, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valdivia, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campino, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tapia-Castillo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baudrand, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villarroel, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cifuentes, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carvajal, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fardella, C. E.</span><span> </span><span class="NLM_article-title">Identification of novel 11beta-HSD1 inhibitors by combined ligand- and structure-based virtual screening</span> <span class="citation_source-journal">Mol. Cell. Endocrinol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">384</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span><span class="refDoi"> DOI: 10.1016/j.mce.2014.01.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref232/cit232&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.mce.2014.01.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref232/cit232&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24447464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref232/cit232&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjslOqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=2014&pages=71-82&author=C.+F.+Lagosauthor=A.+Vecchiolaauthor=F.+Allendeauthor=C.+A.+Fuentesauthor=J.+E.+Tichauerauthor=C.+Valdiviaauthor=S.+Solariauthor=C.+Campinoauthor=A.+Tapia-Castilloauthor=R.+Baudrandauthor=P.+Villarroelauthor=M.+Cifuentesauthor=G.+I.+Owenauthor=C.+A.+Carvajalauthor=C.+E.+Fardella&title=Identification+of+novel+11beta-HSD1+inhibitors+by+combined+ligand-+and+structure-based+virtual+screening&doi=10.1016%2Fj.mce.2014.01.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit232R"><div class="casContent"><span class="casTitleNuber">232</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel 11β-HSD1 inhibitors by combined ligand- and structure-based virtual screening</span></div><div class="casAuthors">Lagos, Carlos F.; Vecchiola, Andrea; Allende, Fidel; Fuentes, Cristobal A.; Tichauer, Juan E.; Valdivia, Carolina; Solari, Sandra; Campino, Carmen; Tapia-Castillo, Alejandra; Baudrand, Rene; Villarroel, Pia; Cifuentes, Mariana; Owen, Gareth I.; Carvajal, Cristian A.; Fardella, Carlos E.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Endocrinology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">384</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">71-82</span>CODEN:
                <span class="NLM_cas:coden">MCEND6</span>;
        ISSN:<span class="NLM_cas:issn">0303-7207</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">11 Beta-hydroxysteroid dehydrogenase type 1 (11β-HSD1) converts cortisone to cortisol in a NADPH dependent manner.  Overexpression of 11β-HSD1 in key metabolic tissues is related to the development of type 2 diabetes, obesity, hypertension and metabolic syndrome.  Using crystal structures of human 11β-HSD1 in complex with inhibitors as source of structural information, a combined ligand and structure-based virtual screening approach was implemented to identify novel 11β-HSD1 inhibitors.  A selected group of compds. was identified in silico and further evaluated in cell-based assays for cytotoxicity and 11β-HSD1 mediated cortisol prodn. inhibitory capacity.  The expression of 11β-HSD1 and 11β-HSD2 in human LS14 adipocytes was assessed during differentiation.  Biol. evaluation of 39 compds. in adipocytes and steroids quantification by HPLC-MS/MS identify 4 compds. that exhibit 11β-HSD1 mediated cortisol prodn. inhibitory activity with potencies in the micromolar range.  Two compds. showed to be selective for the 11β-HSD1 reductase activity and over 11β-HSD2 isoform, and thus represent novel leads for the development of more active derivs. with higher efficacies targeting intracellular cortisol levels in type 2 diabetes and metabolic syndrome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXQ1cIMu4wa7Vg90H21EOLACvtfcHk0lj3CIw48Ae_Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjslOqsLs%253D&md5=91793b8b6ab3a5bf39ddc358f680d52f</span></div><a href="/servlet/linkout?suffix=cit232&amp;dbid=16384&amp;doi=10.1016%2Fj.mce.2014.01.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mce.2014.01.011%26sid%3Dliteratum%253Aachs%26aulast%3DLagos%26aufirst%3DC.%2BF.%26aulast%3DVecchiola%26aufirst%3DA.%26aulast%3DAllende%26aufirst%3DF.%26aulast%3DFuentes%26aufirst%3DC.%2BA.%26aulast%3DTichauer%26aufirst%3DJ.%2BE.%26aulast%3DValdivia%26aufirst%3DC.%26aulast%3DSolari%26aufirst%3DS.%26aulast%3DCampino%26aufirst%3DC.%26aulast%3DTapia-Castillo%26aufirst%3DA.%26aulast%3DBaudrand%26aufirst%3DR.%26aulast%3DVillarroel%26aufirst%3DP.%26aulast%3DCifuentes%26aufirst%3DM.%26aulast%3DOwen%26aufirst%3DG.%2BI.%26aulast%3DCarvajal%26aufirst%3DC.%2BA.%26aulast%3DFardella%26aufirst%3DC.%2BE.%26atitle%3DIdentification%2520of%2520novel%252011beta-HSD1%2520inhibitors%2520by%2520combined%2520ligand-%2520and%2520structure-based%2520virtual%2520screening%26jtitle%3DMol.%2520Cell.%2520Endocrinol.%26date%3D2014%26volume%3D384%26spage%3D71%26epage%3D82%26doi%3D10.1016%2Fj.mce.2014.01.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref233"><div class="reference"><strong class="refLabel"><a href="#ref233" class="rightTabRefNumLink">233</a></strong><div class="NLM_citation" id="rightTab-cit233"><span><span class="NLM_contrib-group">Cancellieri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bassetto, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Widjaja, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Kuppeveld, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Haan, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brancale, A.</span><span> </span><span class="NLM_article-title">In silico structure-based design and synthesis of novel anti-RSV compounds</span> <span class="citation_source-journal">Antiviral Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">122</span><span class="NLM_x">, </span> <span class="NLM_fpage">46</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1016/j.antiviral.2015.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref233/cit233&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.antiviral.2015.08.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2015&pages=46-50&author=M.+Cancellieriauthor=M.+Bassettoauthor=I.+Widjajaauthor=F.+van+Kuppeveldauthor=C.+A.+de+Haanauthor=A.+Brancale&title=In+silico+structure-based+design+and+synthesis+of+novel+anti-RSV+compounds&doi=10.1016%2Fj.antiviral.2015.08.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit233&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2015.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2015.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DCancellieri%26aufirst%3DM.%26aulast%3DBassetto%26aufirst%3DM.%26aulast%3DWidjaja%26aufirst%3DI.%26aulast%3Dvan%2BKuppeveld%26aufirst%3DF.%26aulast%3Dde%2BHaan%26aufirst%3DC.%2BA.%26aulast%3DBrancale%26aufirst%3DA.%26atitle%3DIn%2520silico%2520structure-based%2520design%2520and%2520synthesis%2520of%2520novel%2520anti-RSV%2520compounds%26jtitle%3DAntiviral%2520Res.%26date%3D2015%26volume%3D122%26spage%3D46%26epage%3D50%26doi%3D10.1016%2Fj.antiviral.2015.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref234"><div class="reference"><strong class="refLabel"><a href="#ref234" class="rightTabRefNumLink">234</a></strong><div class="NLM_citation" id="rightTab-cit234"><span><span class="NLM_contrib-group">Rettenmaier, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karpiak, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doak, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, J. A.</span><span> </span><span class="NLM_article-title">Small-molecule allosteric modulators of the protein kinase PDK1 from structure-based docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">8285</span><span class="NLM_x">–</span> <span class="NLM_lpage">8291</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref234/cit234&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Wls77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=8285-8291&author=T.+J.+Rettenmaierauthor=H.+Fanauthor=J.+Karpiakauthor=A.+Doakauthor=A.+Saliauthor=B.+K.+Shoichetauthor=J.+A.+Wells&title=Small-molecule+allosteric+modulators+of+the+protein+kinase+PDK1+from+structure-based+docking&doi=10.1021%2Facs.jmedchem.5b01216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit234R"><div class="casContent"><span class="casTitleNuber">234</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Allosteric Modulators of the Protein Kinase PDK1 from Structure-Based Docking</span></div><div class="casAuthors">Rettenmaier, T. Justin; Fan, Hao; Karpiak, Joel; Doak, Allison; Sali, Andrej; Shoichet, Brian K.; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8285-8291</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Finding small mols. that target allosteric sites remains a grand challenge for ligand discovery.  In the protein kinase field, only a handful of highly selective allosteric modulators have been found.  Thus, more general methods are needed to discover allosteric modulators for addnl. kinases.  Here, the authors use virtual screening against an ensemble of both crystal structures and comparative models to identify ligands for an allosteric peptide-binding site on the protein kinase PDK1 (the PIF pocket).  The authors optimized these ligands through an analog-by-catalog search that yielded compd. (I), which binds to PDK1 with 8 μM affinity.  The authors confirmed the docking poses by detg. a crystal structure of PDK1 in complex with (I).  Because the PIF pocket appears to be a recurring structural feature of the kinase fold, known generally as the helix αC patch, this approach may enable the discovery of allosteric modulators for other kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDLc4nMZELgLVg90H21EOLACvtfcHk0lj3CIw48Ae_Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Wls77P&md5=2afb6d39150cf74a80bf649c94df607c</span></div><a href="/servlet/linkout?suffix=cit234&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01216%26sid%3Dliteratum%253Aachs%26aulast%3DRettenmaier%26aufirst%3DT.%2BJ.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DKarpiak%26aufirst%3DJ.%26aulast%3DDoak%26aufirst%3DA.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DSmall-molecule%2520allosteric%2520modulators%2520of%2520the%2520protein%2520kinase%2520PDK1%2520from%2520structure-based%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D8285%26epage%3D8291%26doi%3D10.1021%2Facs.jmedchem.5b01216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref235"><div class="reference"><strong class="refLabel"><a href="#ref235" class="rightTabRefNumLink">235</a></strong><div class="NLM_citation" id="rightTab-cit235"><span><span class="NLM_contrib-group">Gradler, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerber, H. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodenough-Lashua, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ficner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reuter, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stubbs, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">A new target for shigellosis: rational design and crystallographic studies of inhibitors of tRNA-guanine transglycosylase</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">306</span><span class="NLM_x">, </span> <span class="NLM_fpage">455</span><span class="NLM_x">–</span> <span class="NLM_lpage">467</span><span class="refDoi"> DOI: 10.1006/jmbi.2000.4256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref235/cit235&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1006%2Fjmbi.2000.4256" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=306&publication_year=2001&pages=455-467&author=U.+Gradlerauthor=H.+D.+Gerberauthor=D.+M.+Goodenough-Lashuaauthor=G.+A.+Garciaauthor=R.+Ficnerauthor=K.+Reuterauthor=M.+T.+Stubbsauthor=G.+Klebe&title=A+new+target+for+shigellosis%3A+rational+design+and+crystallographic+studies+of+inhibitors+of+tRNA-guanine+transglycosylase&doi=10.1006%2Fjmbi.2000.4256"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit235&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.2000.4256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.2000.4256%26sid%3Dliteratum%253Aachs%26aulast%3DGradler%26aufirst%3DU.%26aulast%3DGerber%26aufirst%3DH.%2BD.%26aulast%3DGoodenough-Lashua%26aufirst%3DD.%2BM.%26aulast%3DGarcia%26aufirst%3DG.%2BA.%26aulast%3DFicner%26aufirst%3DR.%26aulast%3DReuter%26aufirst%3DK.%26aulast%3DStubbs%26aufirst%3DM.%2BT.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DA%2520new%2520target%2520for%2520shigellosis%253A%2520rational%2520design%2520and%2520crystallographic%2520studies%2520of%2520inhibitors%2520of%2520tRNA-guanine%2520transglycosylase%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2001%26volume%3D306%26spage%3D455%26epage%3D467%26doi%3D10.1006%2Fjmbi.2000.4256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref236"><div class="reference"><strong class="refLabel"><a href="#ref236" class="rightTabRefNumLink">236</a></strong><div class="NLM_citation" id="rightTab-cit236"><span><span class="NLM_contrib-group">Ramirez, U. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nikonova, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pecherskaya, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawrence, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serebriiskii, I. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Einarson, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golemis, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaffe, E. K.</span><span> </span><span class="NLM_article-title">Compounds identified by virtual docking to a tetrameric EGFR extracellular domain can modulate Grb2 internalization</span> <span class="citation_source-journal">BMC Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">436</span><span class="refDoi"> DOI: 10.1186/s12885-015-1415-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref236/cit236&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1186%2Fs12885-015-1415-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=436&author=U.+D.+Ramirezauthor=A.+S.+Nikonovaauthor=H.+Liuauthor=A.+Pecherskayaauthor=S.+H.+Lawrenceauthor=I.+G.+Serebriiskiiauthor=Y.+Zhouauthor=M.+K.+Robinsonauthor=M.+B.+Einarsonauthor=E.+A.+Golemisauthor=E.+K.+Jaffe&title=Compounds+identified+by+virtual+docking+to+a+tetrameric+EGFR+extracellular+domain+can+modulate+Grb2+internalization&doi=10.1186%2Fs12885-015-1415-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit236&amp;dbid=16384&amp;doi=10.1186%2Fs12885-015-1415-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12885-015-1415-6%26sid%3Dliteratum%253Aachs%26aulast%3DRamirez%26aufirst%3DU.%2BD.%26aulast%3DNikonova%26aufirst%3DA.%2BS.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DPecherskaya%26aufirst%3DA.%26aulast%3DLawrence%26aufirst%3DS.%2BH.%26aulast%3DSerebriiskii%26aufirst%3DI.%2BG.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DRobinson%26aufirst%3DM.%2BK.%26aulast%3DEinarson%26aufirst%3DM.%2BB.%26aulast%3DGolemis%26aufirst%3DE.%2BA.%26aulast%3DJaffe%26aufirst%3DE.%2BK.%26atitle%3DCompounds%2520identified%2520by%2520virtual%2520docking%2520to%2520a%2520tetrameric%2520EGFR%2520extracellular%2520domain%2520can%2520modulate%2520Grb2%2520internalization%26jtitle%3DBMC%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D436%26doi%3D10.1186%2Fs12885-015-1415-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref237"><div class="reference"><strong class="refLabel"><a href="#ref237" class="rightTabRefNumLink">237</a></strong><div class="NLM_citation" id="rightTab-cit237"><span><span class="NLM_contrib-group">Gasser, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brogi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urayama, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurose, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tafi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ribeiro, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desaubry, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nebigil, C. G.</span><span> </span><span class="NLM_article-title">Discovery and cardioprotective effects of the first non-peptide agonists of the G protein-coupled prokineticin receptor-1</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0121027</span><span class="refDoi"> DOI: 10.1371/journal.pone.0121027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref237/cit237&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pone.0121027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref237/cit237&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=25831128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref237/cit237&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1KhtLjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0121027&author=A.+Gasserauthor=S.+Brogiauthor=K.+Urayamaauthor=T.+Nishiauthor=H.+Kuroseauthor=A.+Tafiauthor=N.+Ribeiroauthor=L.+Desaubryauthor=C.+G.+Nebigil&title=Discovery+and+cardioprotective+effects+of+the+first+non-peptide+agonists+of+the+G+protein-coupled+prokineticin+receptor-1&doi=10.1371%2Fjournal.pone.0121027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit237R"><div class="casContent"><span class="casTitleNuber">237</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and cardioprotective effects of the first non-peptide agonists of the G protein-coupled prokineticin receptor-1</span></div><div class="casAuthors">Gasser, Adeline; Brogi, Simone; Urayama, Kyoji; Nishi, Toshishide; Kurose, Hitoshi; Tafi, Andrea; Ribeiro, Nigel; Desaubry, Laurent; Nebigil, Canan G.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e0121027/1-e0121027/20</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Prokineticins are angiogenic hormones that activate two G protein-coupled receptors: PKR1 and PKR2.  PKR1 has emerged as a crit. mediator of cardiovascular homeostasis and cardioprotection.  Identification of non-peptide PKR1 agonists that contribute to myocardial repair and collateral vessel growth hold promises for treatment of heart diseases.  Through a combination of in silico studies, medicinal chem., and pharmacol. profiling approaches, we designed, synthesized, and characterized the first PKR1 agonists, demonstrating their cardioprotective activity against myocardial infarction (MI) in mice.  Based on high throughput docking protocol, 250,000 compds. were computationally screened for putative PKR1 agonistic activity, using a homol. model, and 10 virtual hits were pharmacol. evaluated.  One hit internalizes PKR1, increases calcium release and activates ERK and Akt kinases.  Among the 30 derivs. of the hit compd., the most potent deriv., IS20, was confirmed for its selectivity and specificity through genetic gain- and loss-of-function of PKR1.  Importantly, IS20 prevented cardiac lesion formation and improved cardiac function after MI in mice, promoting proliferation of cardiac progenitor cells and neovasculogenesis.  The preclin. investigation of the first PKR1 agonists provides a novel approach to promote cardiac neovasculogenesis after MI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdwKeAR30Fy7Vg90H21EOLACvtfcHk0lin13Z1O70uhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1KhtLjE&md5=57ee5a56af6d324b0a8875f249532f4f</span></div><a href="/servlet/linkout?suffix=cit237&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0121027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0121027%26sid%3Dliteratum%253Aachs%26aulast%3DGasser%26aufirst%3DA.%26aulast%3DBrogi%26aufirst%3DS.%26aulast%3DUrayama%26aufirst%3DK.%26aulast%3DNishi%26aufirst%3DT.%26aulast%3DKurose%26aufirst%3DH.%26aulast%3DTafi%26aufirst%3DA.%26aulast%3DRibeiro%26aufirst%3DN.%26aulast%3DDesaubry%26aufirst%3DL.%26aulast%3DNebigil%26aufirst%3DC.%2BG.%26atitle%3DDiscovery%2520and%2520cardioprotective%2520effects%2520of%2520the%2520first%2520non-peptide%2520agonists%2520of%2520the%2520G%2520protein-coupled%2520prokineticin%2520receptor-1%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0121027%26doi%3D10.1371%2Fjournal.pone.0121027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref238"><div class="reference"><strong class="refLabel"><a href="#ref238" class="rightTabRefNumLink">238</a></strong><div class="NLM_citation" id="rightTab-cit238"><span><span class="NLM_contrib-group">Jang, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seo, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">No, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pae, A. N.</span><span> </span><span class="NLM_article-title">Novel scaffold identification of mGlu1 receptor negative allosteric modulators using a hierarchical virtual screening approach</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">87</span><span class="NLM_x">, </span> <span class="NLM_fpage">239</span><span class="NLM_x">–</span> <span class="NLM_lpage">256</span><span class="refDoi"> DOI: 10.1111/cbdd.12654</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref238/cit238&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1111%2Fcbdd.12654" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref238/cit238&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26343933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref238/cit238&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1aksbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2016&pages=239-256&author=J.+W.+Jangauthor=N.+C.+Choauthor=S.+J.+Minauthor=Y.+S.+Choauthor=K.+D.+Parkauthor=S.+H.+Seoauthor=K.+T.+Noauthor=A.+N.+Pae&title=Novel+scaffold+identification+of+mGlu1+receptor+negative+allosteric+modulators+using+a+hierarchical+virtual+screening+approach&doi=10.1111%2Fcbdd.12654"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit238R"><div class="casContent"><span class="casTitleNuber">238</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Scaffold Identification of mGlu1 Receptor Negative Allosteric Modulators Using a Hierarchical Virtual Screening Approach</span></div><div class="casAuthors">Jang, Jae Wan; Cho, Nam-Chul; Min, Sun-Joon; Cho, Yong Seo; Park, Ki Duk; Seo, Seon Hee; No, Kyoung Tai; Pae, Ae Nim</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">239-256</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Metabotropic glutamate receptor 1 (mGluR1) is considered as an attractive drug target for neuropathic pain treatments.  The hierarchical virtual screening approach for identifying novel scaffolds of mGluR1 allosteric modulators was performed using a homol. model built with the dopamine D3 crystal structure as template.  The mGluR1 mutagenesis data, conserved amino acid sequences across class A and class C GPCRs, and previously reported multiple sequence alignments of class C GPCRs to the rhodopsin template, were employed for the sequence alignment to overcome difficulties of model generation with low sequence identity of mGluR1 and dopamine D3.  The structures refined by mol. dynamics simulations were employed for docking of Asinex com. libraries after hierarchical virtual screening with pharmacophore and naive Bayesian models.  Five of 35 compds. exptl. evaluated using a calcium mobilization assay exhibited micromolar activities (IC50) with chemotype novelty that demonstrated the validity of our methods.  A hierarchical structure and ligand-based virtual screening approach with homol. model of class C GPCR based on dopamine D3 class A GPCR structure was successfully performed and applied to discover novel neg. mGluR1 allosteric modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXRRdozrIIkLVg90H21EOLACvtfcHk0lin13Z1O70uhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1aksbbK&md5=8d268d09b46f5311cf48e62f830cb3cf</span></div><a href="/servlet/linkout?suffix=cit238&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12654%26sid%3Dliteratum%253Aachs%26aulast%3DJang%26aufirst%3DJ.%2BW.%26aulast%3DCho%26aufirst%3DN.%2BC.%26aulast%3DMin%26aufirst%3DS.%2BJ.%26aulast%3DCho%26aufirst%3DY.%2BS.%26aulast%3DPark%26aufirst%3DK.%2BD.%26aulast%3DSeo%26aufirst%3DS.%2BH.%26aulast%3DNo%26aufirst%3DK.%2BT.%26aulast%3DPae%26aufirst%3DA.%2BN.%26atitle%3DNovel%2520scaffold%2520identification%2520of%2520mGlu1%2520receptor%2520negative%2520allosteric%2520modulators%2520using%2520a%2520hierarchical%2520virtual%2520screening%2520approach%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2016%26volume%3D87%26spage%3D239%26epage%3D256%26doi%3D10.1111%2Fcbdd.12654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref239"><div class="reference"><strong class="refLabel"><a href="#ref239" class="rightTabRefNumLink">239</a></strong><div class="NLM_citation" id="rightTab-cit239"><span><span class="NLM_contrib-group">Lee, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mittal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gatuz, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Truong, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Torres, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulhearn, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, M. E.</span><span> </span><span class="NLM_article-title">Identification of novel drug scaffolds for inhibition of SARS-CoV 3-chymotrypsin-like protease using virtual and high-throughput screenings</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">167</span><span class="NLM_x">–</span> <span class="NLM_lpage">177</span><span class="refDoi"> DOI: 10.1016/j.bmc.2013.11.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref239/cit239&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmc.2013.11.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref239/cit239&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24332657" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref239/cit239&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOhsbfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=167-177&author=H.+Leeauthor=A.+Mittalauthor=K.+Patelauthor=J.+L.+Gatuzauthor=L.+Truongauthor=J.+Torresauthor=D.+C.+Mulhearnauthor=M.+E.+Johnson&title=Identification+of+novel+drug+scaffolds+for+inhibition+of+SARS-CoV+3-chymotrypsin-like+protease+using+virtual+and+high-throughput+screenings&doi=10.1016%2Fj.bmc.2013.11.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit239R"><div class="casContent"><span class="casTitleNuber">239</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel drug scaffolds for inhibition of SARS-CoV 3-Chymotrypsin-like protease using virtual and high-throughput screenings</span></div><div class="casAuthors">Lee, Hyun; Mittal, Anuradha; Patel, Kavankumar; Gatuz, Joseph L.; Truong, Lena; Torres, Jaime; Mulhearn, Debbie C.; Johnson, Michael E.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">167-177</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We have used a combination of virtual screening (VS) and high-throughput screening (HTS) techniques to identify novel, non-peptidic small mol. inhibitors against human SARS-CoV 3CLpro.  A structure-based VS approach integrating docking and pharmacophore based methods was employed to computationally screen 621,000 compds. from the ZINC library.  The screening protocol was validated using known 3CLpro inhibitors and was optimized for speed, improved selectivity, and for accommodating receptor flexibility.  Subsequently, a fluorescence-based enzymic HTS assay was developed and optimized to exptl. screen approx. 41,000 compds. from four structurally diverse libraries chosen mainly based on the VS results.  False positives from initial HTS hits were eliminated by a secondary orthogonal binding anal. using surface plasmon resonance (SPR).  The campaign identified a reversible small mol. inhibitor exhibiting mixed-type inhibition with a Ki value of 11.1 μM.  Together, these results validate our protocols as suitable approaches to screen virtual and chem. libraries, and the newly identified compd. reported in our study represents a promising structural scaffold to pursue for further SARS-CoV 3CLpro inhibitor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrK4pEI5rpdq7Vg90H21EOLACvtfcHk0lgfKrrD_5zLzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOhsbfL&md5=8b8f5d24fe89cbf963c0175fb81b03b0</span></div><a href="/servlet/linkout?suffix=cit239&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.11.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.11.041%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%26aulast%3DMittal%26aufirst%3DA.%26aulast%3DPatel%26aufirst%3DK.%26aulast%3DGatuz%26aufirst%3DJ.%2BL.%26aulast%3DTruong%26aufirst%3DL.%26aulast%3DTorres%26aufirst%3DJ.%26aulast%3DMulhearn%26aufirst%3DD.%2BC.%26aulast%3DJohnson%26aufirst%3DM.%2BE.%26atitle%3DIdentification%2520of%2520novel%2520drug%2520scaffolds%2520for%2520inhibition%2520of%2520SARS-CoV%25203-chymotrypsin-like%2520protease%2520using%2520virtual%2520and%2520high-throughput%2520screenings%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2014%26volume%3D22%26spage%3D167%26epage%3D177%26doi%3D10.1016%2Fj.bmc.2013.11.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref240"><div class="reference"><strong class="refLabel"><a href="#ref240" class="rightTabRefNumLink">240</a></strong><div class="NLM_citation" id="rightTab-cit240"><span><span class="NLM_contrib-group">Duffy, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsia, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kharenko, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norek, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gesner, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLure, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lakshminarasimhan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suto, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, H. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitchen, D. B.</span><span> </span><span class="NLM_article-title">Discovery of a new chemical series of BRD4(1) inhibitors using protein-ligand docking and structure-guided design</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">2818</span><span class="NLM_x">–</span> <span class="NLM_lpage">2823</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2015.04.107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref240/cit240&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2015.04.107" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=2818-2823&author=B.+C.+Duffyauthor=S.+Liuauthor=G.+S.+Martinauthor=R.+Wangauthor=M.+M.+Hsiaauthor=H.+Zhaoauthor=C.+Guoauthor=M.+Ellisauthor=J.+F.+Quinnauthor=O.+A.+Kharenkoauthor=K.+Norekauthor=E.+M.+Gesnerauthor=P.+R.+Youngauthor=K.+G.+McLureauthor=G.+S.+Wagnerauthor=D.+Lakshminarasimhanauthor=A.+Whiteauthor=R.+K.+Sutoauthor=H.+C.+Hansenauthor=D.+B.+Kitchen&title=Discovery+of+a+new+chemical+series+of+BRD4%281%29+inhibitors+using+protein-ligand+docking+and+structure-guided+design&doi=10.1016%2Fj.bmcl.2015.04.107"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit240&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.04.107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.04.107%26sid%3Dliteratum%253Aachs%26aulast%3DDuffy%26aufirst%3DB.%2BC.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DHsia%26aufirst%3DM.%2BM.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DGuo%26aufirst%3DC.%26aulast%3DEllis%26aufirst%3DM.%26aulast%3DQuinn%26aufirst%3DJ.%2BF.%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DNorek%26aufirst%3DK.%26aulast%3DGesner%26aufirst%3DE.%2BM.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DWagner%26aufirst%3DG.%2BS.%26aulast%3DLakshminarasimhan%26aufirst%3DD.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DSuto%26aufirst%3DR.%2BK.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3DKitchen%26aufirst%3DD.%2BB.%26atitle%3DDiscovery%2520of%2520a%2520new%2520chemical%2520series%2520of%2520BRD4%25281%2529%2520inhibitors%2520using%2520protein-ligand%2520docking%2520and%2520structure-guided%2520design%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D2818%26epage%3D2823%26doi%3D10.1016%2Fj.bmcl.2015.04.107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref241"><div class="reference"><strong class="refLabel"><a href="#ref241" class="rightTabRefNumLink">241</a></strong><div class="NLM_citation" id="rightTab-cit241"><span><span class="NLM_contrib-group">Ouizougun-Oubari, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pereira, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarus, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galloux, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lassoued, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fix, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tortorici, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoos, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baron, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">England, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desmaele, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couvreur, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bontems, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rey, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eleouet, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sizun, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slama-Schwok, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquerroy, S.</span><span> </span><span class="NLM_article-title">A druggable pocket at the nucleocapsid/phosphoprotein interaction site of human respiratory syncytial virus</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">89</span><span class="NLM_x">, </span> <span class="NLM_fpage">11129</span><span class="NLM_x">–</span> <span class="NLM_lpage">11143</span><span class="refDoi"> DOI: 10.1128/JVI.01612-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref241/cit241&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1128%2FJVI.01612-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref241/cit241&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26246564" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref241/cit241&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslymur7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2015&pages=11129-11143&author=M.+Ouizougun-Oubariauthor=N.+Pereiraauthor=B.+Tarusauthor=M.+Gallouxauthor=S.+Lassouedauthor=J.+Fixauthor=M.+A.+Tortoriciauthor=S.+Hoosauthor=B.+Baronauthor=P.+Englandauthor=D.+Desmaeleauthor=P.+Couvreurauthor=F.+Bontemsauthor=F.+A.+Reyauthor=J.+F.+Eleouetauthor=C.+Sizunauthor=A.+Slama-Schwokauthor=S.+Duquerroy&title=A+druggable+pocket+at+the+nucleocapsid%2Fphosphoprotein+interaction+site+of+human+respiratory+syncytial+virus&doi=10.1128%2FJVI.01612-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit241R"><div class="casContent"><span class="casTitleNuber">241</span><div class="casTitle"><span class="NLM_cas:atitle">A druggable pocket at the nucleocapsid/phosphoprotein interaction site of human respiratory syncytial virus</span></div><div class="casAuthors">Ouizougun-Oubari, Mohamed; Pereira, Nelson; Tarus, Bogdan; Galloux, Marie; Lassoued, Safa; Fix, Jenna; Tortorici, M. Alejandra; Hoos, Sylviane; Baron, Bruno; England, Patrick; Desmaele, Didier; Couvreur, Patrick; Bontems, Francois; Rey, Felix A.; Eleouet, Jean-Francois; Sizun, Christina; Slama-Schwok, Anny; Duquerroy, Stephane</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11129-11143</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Presently, respiratory syncytial virus (RSV), the main cause of severe respiratory infections in infants, cannot be treated efficiently with antivirals.  However, its RNA-dependent polymerase complex offers potential targets for RSV-specific drugs.  This includes the recognition of its template, the ribonucleoprotein complex (RNP), consisting of genomic RNA encapsidated by the RSV nucleoprotein, N.  This recognition proceeds via interaction between the phosphoprotein P, which is the main polymerase cofactor, and N.  The determinant role of the C terminus of P, and more particularly of the last residue, F241, in RNP binding and viral RNA synthesis has been assessed previously.  Here, we provide detailed structural insight into this crucial interaction for RSV polymerase activity.  We solved the crystallog. structures of complexes between the N-terminal domain of N (N-NTD) and C-terminal peptides of P and characterized binding by biophys. approaches.  Our results provide a rationale for the pivotal role of F241, which inserts into a well-defined N-NTD pocket.  This primary binding site is completed by transient contacts with upstream P residues outside the pocket.  Based on the structural information of the N-NTD:P complex, we identified inhibitors of this interaction, selected by in silico screening of small compds., that efficiently bind to N and compete with P in vitro.  One of the compds. displayed inhibitory activity on RSV replication, thereby strengthening the relevance of N-NTD for structure-based design of RSV-specific antivirals.  Structure factors and coordinates have been deposited in the Protein Data Bank under accession codes 4UC6 and 4UC7 for the Nat1 and Nat2 structures of the N-NTD; 4UC8, 4UC9, 4UCA, and 4UCB for the N-NTD in complex with P1, P2, P3, and P7 P-CTD peptides; and 4UCC, 4UCD, and 4UCE for the N-NTD in complex with M76, M81, and M72 compds., resp.  Backbone chem. shift assignments for the N-NTD have been deposited in the BioMagResBank (http://www.bmrb.wisc.edu) under accession no. 25705.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9DPzJ81oHGLVg90H21EOLACvtfcHk0lgfKrrD_5zLzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslymur7F&md5=a373390a4652cbf4255d66b62752ab8a</span></div><a href="/servlet/linkout?suffix=cit241&amp;dbid=16384&amp;doi=10.1128%2FJVI.01612-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.01612-15%26sid%3Dliteratum%253Aachs%26aulast%3DOuizougun-Oubari%26aufirst%3DM.%26aulast%3DPereira%26aufirst%3DN.%26aulast%3DTarus%26aufirst%3DB.%26aulast%3DGalloux%26aufirst%3DM.%26aulast%3DLassoued%26aufirst%3DS.%26aulast%3DFix%26aufirst%3DJ.%26aulast%3DTortorici%26aufirst%3DM.%2BA.%26aulast%3DHoos%26aufirst%3DS.%26aulast%3DBaron%26aufirst%3DB.%26aulast%3DEngland%26aufirst%3DP.%26aulast%3DDesmaele%26aufirst%3DD.%26aulast%3DCouvreur%26aufirst%3DP.%26aulast%3DBontems%26aufirst%3DF.%26aulast%3DRey%26aufirst%3DF.%2BA.%26aulast%3DEleouet%26aufirst%3DJ.%2BF.%26aulast%3DSizun%26aufirst%3DC.%26aulast%3DSlama-Schwok%26aufirst%3DA.%26aulast%3DDuquerroy%26aufirst%3DS.%26atitle%3DA%2520druggable%2520pocket%2520at%2520the%2520nucleocapsid%252Fphosphoprotein%2520interaction%2520site%2520of%2520human%2520respiratory%2520syncytial%2520virus%26jtitle%3DJ.%2520Virol.%26date%3D2015%26volume%3D89%26spage%3D11129%26epage%3D11143%26doi%3D10.1128%2FJVI.01612-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref242"><div class="reference"><strong class="refLabel"><a href="#ref242" class="rightTabRefNumLink">242</a></strong><div class="NLM_citation" id="rightTab-cit242"><span><span class="NLM_contrib-group">Brenk, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vetter, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyce, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodin, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Probing molecular docking in a charged model binding site</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">357</span><span class="NLM_x">, </span> <span class="NLM_fpage">1449</span><span class="NLM_x">–</span> <span class="NLM_lpage">1470</span><span class="refDoi"> DOI: 10.1016/j.jmb.2006.01.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref242/cit242&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.jmb.2006.01.034" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=357&publication_year=2006&pages=1449-1470&author=R.+Brenkauthor=S.+W.+Vetterauthor=S.+E.+Boyceauthor=D.+B.+Goodinauthor=B.+K.+Shoichet&title=Probing+molecular+docking+in+a+charged+model+binding+site&doi=10.1016%2Fj.jmb.2006.01.034"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit242&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2006.01.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2006.01.034%26sid%3Dliteratum%253Aachs%26aulast%3DBrenk%26aufirst%3DR.%26aulast%3DVetter%26aufirst%3DS.%2BW.%26aulast%3DBoyce%26aufirst%3DS.%2BE.%26aulast%3DGoodin%26aufirst%3DD.%2BB.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DProbing%2520molecular%2520docking%2520in%2520a%2520charged%2520model%2520binding%2520site%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D357%26spage%3D1449%26epage%3D1470%26doi%3D10.1016%2Fj.jmb.2006.01.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref243"><div class="reference"><strong class="refLabel"><a href="#ref243" class="rightTabRefNumLink">243</a></strong><div class="NLM_citation" id="rightTab-cit243"><span><span class="NLM_contrib-group">Fischer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraser, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Incorporation of protein flexibility and conformational energy penalties in docking screens to improve ligand discovery</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">575</span><span class="NLM_x">–</span> <span class="NLM_lpage">583</span><span class="refDoi"> DOI: 10.1038/nchem.1954</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref243/cit243&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnchem.1954" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref243/cit243&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=24950326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref243/cit243&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1eqsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=575-583&author=M.+Fischerauthor=R.+G.+Colemanauthor=J.+S.+Fraserauthor=B.+K.+Shoichet&title=Incorporation+of+protein+flexibility+and+conformational+energy+penalties+in+docking+screens+to+improve+ligand+discovery&doi=10.1038%2Fnchem.1954"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit243R"><div class="casContent"><span class="casTitleNuber">243</span><div class="casTitle"><span class="NLM_cas:atitle">Incorporation of protein flexibility and conformational energy penalties in docking screens to improve ligand discovery</span></div><div class="casAuthors">Fischer, Marcus; Coleman, Ryan G.; Fraser, James S.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">575-583</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteins fluctuate between alternative conformations, which presents a challenge for ligand discovery because such flexibility is difficult to treat computationally owing to problems with conformational sampling and energy weighting.  Here we describe a flexible docking method that samples and wts. protein conformations using exptl. derived conformations as a guide.  The crystallog. refined occupancies of these conformations, which are observable in an apo receptor structure, define energy penalties for docking.  In a large prospective library screen, we identified new ligands that target specific receptor conformations of a cavity in cytochrome c peroxidase, and we confirm both ligand pose and assocd. receptor conformation predictions by crystallog.  The inclusion of receptor flexibility led to ligands with new chemotypes and phys. properties.  By exploiting exptl. measures of loop and side-chain flexibility, this method can be extended to the discovery of new ligands for hundreds of targets in the Protein Data Bank for which similar exptl. information is available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9q4phmwMLHbVg90H21EOLACvtfcHk0lisguEttCVyxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1eqsLw%253D&md5=141d6f5238664dabb0890cc5f764b99d</span></div><a href="/servlet/linkout?suffix=cit243&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1954&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1954%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DM.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26aulast%3DFraser%26aufirst%3DJ.%2BS.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DIncorporation%2520of%2520protein%2520flexibility%2520and%2520conformational%2520energy%2520penalties%2520in%2520docking%2520screens%2520to%2520improve%2520ligand%2520discovery%26jtitle%3DNat.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D575%26epage%3D583%26doi%3D10.1038%2Fnchem.1954" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref244"><div class="reference"><strong class="refLabel"><a href="#ref244" class="rightTabRefNumLink">244</a></strong><div class="NLM_citation" id="rightTab-cit244"><span><span class="NLM_contrib-group">Wei, B. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baase, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weaver, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthews, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">A model binding site for testing scoring functions in molecular docking</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">322</span><span class="NLM_x">, </span> <span class="NLM_fpage">339</span><span class="NLM_x">–</span> <span class="NLM_lpage">355</span><span class="refDoi"> DOI: 10.1016/S0022-2836(02)00777-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref244/cit244&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2FS0022-2836%2802%2900777-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref244/cit244&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=12217695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref244/cit244&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD38Xms1Wqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=322&publication_year=2002&pages=339-355&author=B.+Q.+Weiauthor=W.+A.+Baaseauthor=L.+H.+Weaverauthor=B.+W.+Matthewsauthor=B.+K.+Shoichet&title=A+model+binding+site+for+testing+scoring+functions+in+molecular+docking&doi=10.1016%2FS0022-2836%2802%2900777-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit244R"><div class="casContent"><span class="casTitleNuber">244</span><div class="casTitle"><span class="NLM_cas:atitle">A Model Binding Site for Testing Scoring Functions in Molecular Docking</span></div><div class="casAuthors">Wei, Binqing Q.; Baase, Walter A.; Weaver, Larry H.; Matthews, Brian W.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">322</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">339-355</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Prediction of interaction energies between ligands and their receptors remains a major challenge for structure-based inhibitor discovery.  Much effort has been devoted to developing scoring schemes that can successfully rank the affinities of a diverse set of possible ligands to a binding site for which the structure is known.  To test these scoring functions, well-characterized exptl. systems can be very useful.  Here, mutation-created binding sites in T4 lysozyme were used to investigate how the quality of at. charges and solvation energies affects mol. docking.  At. charges and solvation energies were calcd. for 172,118 mols. in the Available Chems.  Directory using a semi-empirical quantum mech. approach by the program AMSOL.  The database was first screened against the apolar cavity site created by the mutation Leu99Ala (L99A).  Compared to the electronegativity-based charges that are widely used, the new charges and desolvation energies improved ranking of known apolar ligands, and better distinguished them from more polar isosteres that are not obsd. to bind.  To investigate whether the new charges had predictive value, the non-polar residue Met102, which forms part of the binding site, was changed to the polar residue glutamine.  The structure of the resulting Leu99 Ala and Met102 Gln double mutant of T4 lysozyme (L99A/M102Q) was detd. and the docking calcn. was repeated for the new site.  Seven representative polar mols. that preferentially docked to the polar vs. the apolar binding site were tested exptl.  All seven bind to the polar cavity (L99A/M102Q) but do not detectably bind to the apolar cavity (L99A).  Five ligand-bound structures of L99A/M102Q were detd. by X-ray crystallog.  Docking predictions corresponded to the crystallog. results to within 0.4 A RMSD.  Improved treatment of partial at. charges and desolvation energies in database docking appears feasible and leads to better distinction of true ligands.  Simple model binding sites, such as L99A and its more polar variants, may find broad use in the development and testing of docking algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGral4WDhJW6jbVg90H21EOLACvtfcHk0lisguEttCVyxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xms1Wqsrs%253D&md5=040e009b33f125ab4abbb322cf32d2c9</span></div><a href="/servlet/linkout?suffix=cit244&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2900777-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252900777-5%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DB.%2BQ.%26aulast%3DBaase%26aufirst%3DW.%2BA.%26aulast%3DWeaver%26aufirst%3DL.%2BH.%26aulast%3DMatthews%26aufirst%3DB.%2BW.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DA%2520model%2520binding%2520site%2520for%2520testing%2520scoring%2520functions%2520in%2520molecular%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D322%26spage%3D339%26epage%3D355%26doi%3D10.1016%2FS0022-2836%2802%2900777-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref245"><div class="reference"><strong class="refLabel"><a href="#ref245" class="rightTabRefNumLink">245</a></strong><div class="NLM_citation" id="rightTab-cit245"><span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shivakumar, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyce, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Rescoring docking hit lists for model cavity sites: predictions and experimental testing</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">377</span><span class="NLM_x">, </span> <span class="NLM_fpage">914</span><span class="NLM_x">–</span> <span class="NLM_lpage">934</span><span class="refDoi"> DOI: 10.1016/j.jmb.2008.01.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref245/cit245&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.jmb.2008.01.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref245/cit245&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=18280498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref245/cit245&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsVSltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=377&publication_year=2008&pages=914-934&author=A.+P.+Gravesauthor=D.+M.+Shivakumarauthor=S.+E.+Boyceauthor=M.+P.+Jacobsonauthor=D.+A.+Caseauthor=B.+K.+Shoichet&title=Rescoring+docking+hit+lists+for+model+cavity+sites%3A+predictions+and+experimental+testing&doi=10.1016%2Fj.jmb.2008.01.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit245R"><div class="casContent"><span class="casTitleNuber">245</span><div class="casTitle"><span class="NLM_cas:atitle">Rescoring Docking Hit Lists for Model Cavity Sites: Predictions and Experimental Testing</span></div><div class="casAuthors">Graves, Alan P.; Shivakumar, Devleena M.; Boyce, Sarah E.; Jacobson, Matthew P.; Case, David A.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">377</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">914-934</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Mol. docking computationally screens thousands to millions of org. mols. against protein structures, looking for those with complementary fits.  Many approxns. are made, often resulting in low "hit rates.".  A strategy to overcome these approxns. is to rescore top-ranked docked mols. using a better but slower method.  One such is afforded by mol. mechanics-generalized Born surface area (MM-GBSA) techniques.  These more phys. realistic methods have improved models for solvation and electrostatic interactions and conformational change compared to most docking programs.  To investigate MM-GBSA rescoring, the authors reranked docking hit lists in three small buried sites: a hydrophobic cavity that binds apolar ligands, a slightly polar cavity that binds aryl and hydrogen-bonding ligands, and an anionic cavity that binds cationic ligands.  These sites are simple; consequently, incorrect predictions can be attributed to particular errors in the method, and many likely ligands may actually be tested.  In retrospective calcns., MM-GBSA techniques with binding-site minimization better distinguished the known ligands for each cavity from the known decoys compared to the docking calcn. alone.  This encouraged us to test rescoring prospectively on mols. that ranked poorly by docking but that ranked well when rescored by MM-GBSA.  A total of 33 mols. highly ranked by MM-GBSA for the three cavities were tested exptl.  Of these, 23 were obsd. to bind-these are docking false negatives rescued by rescoring.  The 10 remaining mols. are true negatives by docking and false positives by MM-GBSA.  X-ray crystal structures were detd. for 21 of these 23 mols.  In many cases, the geometry prediction by MM-GBSA improved the initial docking pose and more closely resembled the crystallog. result; yet in several cases, the rescored geometry failed to capture large conformational changes in the protein.  Intriguingly, rescoring not only rescued docking false positives, but also introduced several new false positives into the top-ranking mols.  The authors consider the origins of the successes and failures in MM-GBSA rescoring in these model cavity sites and the prospects for rescoring in biol. relevant targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVMe23LJZDCbVg90H21EOLACvtfcHk0lgRcFLopmibSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsVSltL0%253D&md5=83bfde66d8b2f032b72b092506f84662</span></div><a href="/servlet/linkout?suffix=cit245&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2008.01.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2008.01.049%26sid%3Dliteratum%253Aachs%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DShivakumar%26aufirst%3DD.%2BM.%26aulast%3DBoyce%26aufirst%3DS.%2BE.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRescoring%2520docking%2520hit%2520lists%2520for%2520model%2520cavity%2520sites%253A%2520predictions%2520and%2520experimental%2520testing%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D377%26spage%3D914%26epage%3D934%26doi%3D10.1016%2Fj.jmb.2008.01.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref246"><div class="reference"><strong class="refLabel"><a href="#ref246" class="rightTabRefNumLink">246</a></strong><div class="NLM_citation" id="rightTab-cit246"><span><span class="NLM_contrib-group">Tanwar, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanwar, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaquiquzzaman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alam, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akhter, M.</span><span> </span><span class="NLM_article-title">Structure based virtual screening of MDPI database: discovery of structurally diverse and novel DPP-IV inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">3447</span><span class="NLM_x">–</span> <span class="NLM_lpage">3451</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2014.05.076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref246/cit246&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1016%2Fj.bmcl.2014.05.076" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=3447-3451&author=O.+Tanwarauthor=L.+Tanwarauthor=M.+Shaquiquzzamanauthor=M.+M.+Alamauthor=M.+Akhter&title=Structure+based+virtual+screening+of+MDPI+database%3A+discovery+of+structurally+diverse+and+novel+DPP-IV+inhibitors&doi=10.1016%2Fj.bmcl.2014.05.076"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit246&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.05.076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.05.076%26sid%3Dliteratum%253Aachs%26aulast%3DTanwar%26aufirst%3DO.%26aulast%3DTanwar%26aufirst%3DL.%26aulast%3DShaquiquzzaman%26aufirst%3DM.%26aulast%3DAlam%26aufirst%3DM.%2BM.%26aulast%3DAkhter%26aufirst%3DM.%26atitle%3DStructure%2520based%2520virtual%2520screening%2520of%2520MDPI%2520database%253A%2520discovery%2520of%2520structurally%2520diverse%2520and%2520novel%2520DPP-IV%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D3447%26epage%3D3451%26doi%3D10.1016%2Fj.bmcl.2014.05.076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref247"><div class="reference"><strong class="refLabel"><a href="#ref247" class="rightTabRefNumLink">247</a></strong><div class="NLM_citation" id="rightTab-cit247"><span><span class="NLM_contrib-group">Bavan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luetje, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abaffy, T.</span><span> </span><span class="NLM_article-title">Discovery of novel ligands for mouse olfactory receptor MOR42–3 using an in silico screening approach and in vitro validation</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e92064</span><span class="refDoi"> DOI: 10.1371/journal.pone.0092064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref247/cit247&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pone.0092064" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e92064&author=S.+Bavanauthor=B.+Shermanauthor=C.+W.+Luetjeauthor=T.+Abaffy&title=Discovery+of+novel+ligands+for+mouse+olfactory+receptor+MOR42%E2%80%933+using+an+in+silico+screening+approach+and+in+vitro+validation&doi=10.1371%2Fjournal.pone.0092064"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit247&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0092064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0092064%26sid%3Dliteratum%253Aachs%26aulast%3DBavan%26aufirst%3DS.%26aulast%3DSherman%26aufirst%3DB.%26aulast%3DLuetje%26aufirst%3DC.%2BW.%26aulast%3DAbaffy%26aufirst%3DT.%26atitle%3DDiscovery%2520of%2520novel%2520ligands%2520for%2520mouse%2520olfactory%2520receptor%2520MOR42%25E2%2580%25933%2520using%2520an%2520in%2520silico%2520screening%2520approach%2520and%2520in%2520vitro%2520validation%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26spage%3De92064%26doi%3D10.1371%2Fjournal.pone.0092064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref248"><div class="reference"><strong class="refLabel"><a href="#ref248" class="rightTabRefNumLink">248</a></strong><div class="NLM_citation" id="rightTab-cit248"><span><span class="NLM_contrib-group">Murgueitio, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henneke, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glossmann, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santos-Sierra, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolber, G.</span><span> </span><span class="NLM_article-title">Prospective virtual screening in a sparse data scenario: design of small-molecule TLR2 antagonists</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">813</span><span class="NLM_x">–</span> <span class="NLM_lpage">822</span><span class="refDoi"> DOI: 10.1002/cmdc.201300445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref248/cit248&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2Fcmdc.201300445" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=813-822&author=M.+S.+Murgueitioauthor=P.+Hennekeauthor=H.+Glossmannauthor=S.+Santos-Sierraauthor=G.+Wolber&title=Prospective+virtual+screening+in+a+sparse+data+scenario%3A+design+of+small-molecule+TLR2+antagonists&doi=10.1002%2Fcmdc.201300445"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit248&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300445%26sid%3Dliteratum%253Aachs%26aulast%3DMurgueitio%26aufirst%3DM.%2BS.%26aulast%3DHenneke%26aufirst%3DP.%26aulast%3DGlossmann%26aufirst%3DH.%26aulast%3DSantos-Sierra%26aufirst%3DS.%26aulast%3DWolber%26aufirst%3DG.%26atitle%3DProspective%2520virtual%2520screening%2520in%2520a%2520sparse%2520data%2520scenario%253A%2520design%2520of%2520small-molecule%2520TLR2%2520antagonists%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D813%26epage%3D822%26doi%3D10.1002%2Fcmdc.201300445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref249"><div class="reference"><strong class="refLabel"><a href="#ref249" class="rightTabRefNumLink">249</a></strong><div class="NLM_citation" id="rightTab-cit249"><span><span class="NLM_contrib-group">Stone, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parikh, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Rami, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ge, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellogg, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, P.</span><span> </span><span class="NLM_article-title">Identification of small-molecule inhibitors against meso-2, 6-diaminopimelate dehydrogenase from Porphyromonas gingivalis</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0141126</span><span class="refDoi"> DOI: 10.1371/journal.pone.0141126</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref249/cit249&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pone.0141126" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0141126&author=V.+N.+Stoneauthor=H.+I.+Parikhauthor=F.+El-Ramiauthor=X.+Geauthor=W.+Chenauthor=Y.+Zhangauthor=G.+E.+Kelloggauthor=P.+Xu&title=Identification+of+small-molecule+inhibitors+against+meso-2%2C+6-diaminopimelate+dehydrogenase+from+Porphyromonas+gingivalis&doi=10.1371%2Fjournal.pone.0141126"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit249&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0141126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0141126%26sid%3Dliteratum%253Aachs%26aulast%3DStone%26aufirst%3DV.%2BN.%26aulast%3DParikh%26aufirst%3DH.%2BI.%26aulast%3DEl-Rami%26aufirst%3DF.%26aulast%3DGe%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DKellogg%26aufirst%3DG.%2BE.%26aulast%3DXu%26aufirst%3DP.%26atitle%3DIdentification%2520of%2520small-molecule%2520inhibitors%2520against%2520meso-2%252C%25206-diaminopimelate%2520dehydrogenase%2520from%2520Porphyromonas%2520gingivalis%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0141126%26doi%3D10.1371%2Fjournal.pone.0141126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref250"><div class="reference"><strong class="refLabel"><a href="#ref250" class="rightTabRefNumLink">250</a></strong><div class="NLM_citation" id="rightTab-cit250"><span><span class="NLM_contrib-group">Zhong, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chong, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, C. H.</span><span> </span><span class="NLM_article-title">Discovery of a natural product-like iNOS inhibitor by molecular docking with potential neuroprotective effects in vivo</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e92905</span><span class="refDoi"> DOI: 10.1371/journal.pone.0092905</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref250/cit250&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1371%2Fjournal.pone.0092905" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e92905&author=H.+J.+Zhongauthor=L.+J.+Liuauthor=C.+M.+Chongauthor=L.+Luauthor=M.+Wangauthor=D.+S.+Chanauthor=P.+W.+Chanauthor=S.+M.+Leeauthor=D.+L.+Maauthor=C.+H.+Leung&title=Discovery+of+a+natural+product-like+iNOS+inhibitor+by+molecular+docking+with+potential+neuroprotective+effects+in+vivo&doi=10.1371%2Fjournal.pone.0092905"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit250&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0092905&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0092905%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DH.%2BJ.%26aulast%3DLiu%26aufirst%3DL.%2BJ.%26aulast%3DChong%26aufirst%3DC.%2BM.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DD.%2BS.%26aulast%3DChan%26aufirst%3DP.%2BW.%26aulast%3DLee%26aufirst%3DS.%2BM.%26aulast%3DMa%26aufirst%3DD.%2BL.%26aulast%3DLeung%26aufirst%3DC.%2BH.%26atitle%3DDiscovery%2520of%2520a%2520natural%2520product-like%2520iNOS%2520inhibitor%2520by%2520molecular%2520docking%2520with%2520potential%2520neuroprotective%2520effects%2520in%2520vivo%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26spage%3De92905%26doi%3D10.1371%2Fjournal.pone.0092905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref251"><div class="reference"><strong class="refLabel"><a href="#ref251" class="rightTabRefNumLink">251</a></strong><div class="NLM_citation" id="rightTab-cit251"><span><span class="NLM_contrib-group">Mpamhanga, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spinks, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tulloch, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shanks, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collie, I. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairlamb, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frearson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, W. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, I. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brenk, R.</span><span> </span><span class="NLM_article-title">One scaffold, three binding modes: novel and selective pteridine reductase 1 inhibitors derived from fragment hits discovered by virtual screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">4454</span><span class="NLM_x">–</span> <span class="NLM_lpage">4465</span><span class="refDoi"> DOI: 10.1021/jm900414x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm900414x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=4454-4465&author=C.+P.+Mpamhangaauthor=D.+Spinksauthor=L.+B.+Tullochauthor=E.+J.+Shanksauthor=D.+A.+Robinsonauthor=I.+T.+Collieauthor=A.+H.+Fairlambauthor=P.+G.+Wyattauthor=J.+A.+Frearsonauthor=W.+N.+Hunterauthor=I.+H.+Gilbertauthor=R.+Brenk&title=One+scaffold%2C+three+binding+modes%3A+novel+and+selective+pteridine+reductase+1+inhibitors+derived+from+fragment+hits+discovered+by+virtual+screening&doi=10.1021%2Fjm900414x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit251&amp;dbid=16384&amp;doi=10.1021%2Fjm900414x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900414x%26sid%3Dliteratum%253Aachs%26aulast%3DMpamhanga%26aufirst%3DC.%2BP.%26aulast%3DSpinks%26aufirst%3DD.%26aulast%3DTulloch%26aufirst%3DL.%2BB.%26aulast%3DShanks%26aufirst%3DE.%2BJ.%26aulast%3DRobinson%26aufirst%3DD.%2BA.%26aulast%3DCollie%26aufirst%3DI.%2BT.%26aulast%3DFairlamb%26aufirst%3DA.%2BH.%26aulast%3DWyatt%26aufirst%3DP.%2BG.%26aulast%3DFrearson%26aufirst%3DJ.%2BA.%26aulast%3DHunter%26aufirst%3DW.%2BN.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DBrenk%26aufirst%3DR.%26atitle%3DOne%2520scaffold%252C%2520three%2520binding%2520modes%253A%2520novel%2520and%2520selective%2520pteridine%2520reductase%25201%2520inhibitors%2520derived%2520from%2520fragment%2520hits%2520discovered%2520by%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D4454%26epage%3D4465%26doi%3D10.1021%2Fjm900414x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref252"><div class="reference"><strong class="refLabel"><a href="#ref252" class="rightTabRefNumLink">252</a></strong><div class="NLM_citation" id="rightTab-cit252"><span><span class="NLM_contrib-group">Gschwend, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sirawaraporn, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, D. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, I. D.</span><span> </span><span class="NLM_article-title">Specificity in structure-based drug design: identification of a novel, selective inhibitor of Pneumocystis carinii dihydrofolate reductase</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">59</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.1002/(SICI)1097-0134(199709)29:1<59::AID-PROT4>3.0.CO;2-A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref252/cit252&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1002%2F%28SICI%291097-0134%28199709%2929%3A1%3C59%3A%3AAID-PROT4%3E3.0.CO%3B2-A" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1997&pages=59-67&author=D.+A.+Gschwendauthor=W.+Sirawarapornauthor=D.+V.+Santiauthor=I.+D.+Kuntz&title=Specificity+in+structure-based+drug+design%3A+identification+of+a+novel%2C+selective+inhibitor+of+Pneumocystis+carinii+dihydrofolate+reductase&doi=10.1002%2F%28SICI%291097-0134%28199709%2929%3A1%3C59%3A%3AAID-PROT4%3E3.0.CO%3B2-A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit252&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-0134%28199709%2929%3A1%3C59%3A%3AAID-PROT4%3E3.0.CO%3B2-A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-0134%2528199709%252929%253A1%253C59%253A%253AAID-PROT4%253E3.0.CO%253B2-A%26sid%3Dliteratum%253Aachs%26aulast%3DGschwend%26aufirst%3DD.%2BA.%26aulast%3DSirawaraporn%26aufirst%3DW.%26aulast%3DSanti%26aufirst%3DD.%2BV.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26atitle%3DSpecificity%2520in%2520structure-based%2520drug%2520design%253A%2520identification%2520of%2520a%2520novel%252C%2520selective%2520inhibitor%2520of%2520Pneumocystis%2520carinii%2520dihydrofolate%2520reductase%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D1997%26volume%3D29%26spage%3D59%26epage%3D67%26doi%3D10.1002%2F%28SICI%291097-0134%28199709%2929%3A1%3C59%3A%3AAID-PROT4%3E3.0.CO%3B2-A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref253"><div class="reference"><strong class="refLabel"><a href="#ref253" class="rightTabRefNumLink">253</a></strong><div class="NLM_citation" id="rightTab-cit253"><span><span class="NLM_contrib-group">Sun, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">App, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirth, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMahon, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, C.</span><span> </span><span class="NLM_article-title">Synthesis and biological evaluations of 3-substituted indolin-2-ones: a novel class of tyrosine kinase inhibitors that exhibit selectivity toward particular receptor tyrosine kinases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">2588</span><span class="NLM_x">–</span> <span class="NLM_lpage">2603</span><span class="refDoi"> DOI: 10.1021/jm980123i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm980123i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref253/cit253&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADyaK1cXkt12jtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=2588-2603&author=L.+Sunauthor=N.+Tranauthor=F.+Tangauthor=H.+Appauthor=P.+Hirthauthor=G.+McMahonauthor=C.+Tang&title=Synthesis+and+biological+evaluations+of+3-substituted+indolin-2-ones%3A+a+novel+class+of+tyrosine+kinase+inhibitors+that+exhibit+selectivity+toward+particular+receptor+tyrosine+kinases&doi=10.1021%2Fjm980123i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit253R"><div class="casContent"><span class="casTitleNuber">253</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Biological Evaluations of 3-Substituted Indolin-2-ones: A Novel Class of Tyrosine Kinase Inhibitors That Exhibit Selectivity toward Particular Receptor Tyrosine Kinases</span></div><div class="casAuthors">Sun, Li; Tran, Ngoc; Tang, Flora; App, Harald; Hirth, Peter; McMahon, Gerald; Tang, Cho</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2588-2603</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">3-Substituted indolin-2-ones have been designed and synthesized as a novel class of tyrosine kinase inhibitors which exhibit selectivity toward different receptor tyrosine kinases (RTKs).  These compds. have been evaluated for their relative inhibitory properties against a panel of RTKs in intact cells.  By modifying the 3-substituted indolin-2-ones, we have identified compds. which showed selective inhibition of the ligand-dependent autophosphorylation of various RTKs at submicromolar levels in cells.  Structure-activity anal. for these compds. and their relative potency and selectivity to inhibit particular RTKs has detd. that (1) 3-[(five-membered heteroaryl ring)methylidenyl]indolin-2-ones are highly specific against the VEGF (Flk-1) RTK activity, (2) 3-(substituted benzylidenyl)indolin-2-ones contg. bulky group(s) in the Ph ring at the C-3 position of indolin-2-ones showed high selectivity toward the EGF and Her-2 RTKs, and (3) the compd. contg. an extended side chain at the C-3 position of the indolin-2-one exhibited high potency and selectivity when tested against the PDGF and VEGF (Flk-1) RTKs.  Recent published crystallog. data for two of these 3-substituted indolin-2-ones provides a rationale to suggest that these compds. may bind in the ATP binding pocket of RTKs.  The structure-activity anal. supports the use of subsets of these compds. as specific chem. leads for the development of RTK-specific drugs with broad application for the treatment of human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJoPixq8Am5rVg90H21EOLACvtfcHk0ljyhRmMPZMU3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXkt12jtrw%253D&md5=536302b717170a630008dac14832204c</span></div><a href="/servlet/linkout?suffix=cit253&amp;dbid=16384&amp;doi=10.1021%2Fjm980123i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm980123i%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DL.%26aulast%3DTran%26aufirst%3DN.%26aulast%3DTang%26aufirst%3DF.%26aulast%3DApp%26aufirst%3DH.%26aulast%3DHirth%26aufirst%3DP.%26aulast%3DMcMahon%26aufirst%3DG.%26aulast%3DTang%26aufirst%3DC.%26atitle%3DSynthesis%2520and%2520biological%2520evaluations%2520of%25203-substituted%2520indolin-2-ones%253A%2520a%2520novel%2520class%2520of%2520tyrosine%2520kinase%2520inhibitors%2520that%2520exhibit%2520selectivity%2520toward%2520particular%2520receptor%2520tyrosine%2520kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D2588%26epage%3D2603%26doi%3D10.1021%2Fjm980123i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref254"><div class="reference"><strong class="refLabel"><a href="#ref254" class="rightTabRefNumLink">254</a></strong><div class="NLM_citation" id="rightTab-cit254"><span><span class="NLM_contrib-group">Kobayashi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinjo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koseki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrow, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aoki, S.</span><span> </span><span class="NLM_article-title">Identification of novel potential antibiotics against Staphylococcus using structure-based drug screening targeting dihydrofolate reductase</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1242</span><span class="NLM_x">–</span> <span class="NLM_lpage">1253</span><span class="refDoi"> DOI: 10.1021/ci400686d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400686d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref254/cit254&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2cXks1yrsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1242-1253&author=M.+Kobayashiauthor=T.+Kinjoauthor=Y.+Kosekiauthor=C.+R.+Bourneauthor=W.+W.+Barrowauthor=S.+Aoki&title=Identification+of+novel+potential+antibiotics+against+Staphylococcus+using+structure-based+drug+screening+targeting+dihydrofolate+reductase&doi=10.1021%2Fci400686d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit254R"><div class="casContent"><span class="casTitleNuber">254</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel potential antibiotics against Staphylococcus using structure-based drug screening targeting dihydrofolate reductase</span></div><div class="casAuthors">Kobayashi, Maiko; Kinjo, Tomohiro; Koseki, Yuji; Bourne, Christina R.; Barrow, William W.; Aoki, Shunsuke</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1242-1253</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The emergence of multidrug-resistant Staphylococcus aureus (S. aureus) makes the treatment of infectious diseases in hospitals more difficult and increases the mortality of the patients.  In this study, we attempted to identify novel potent antibiotic candidate compds. against S. aureus dihydrofolate reductase (saDHFR).  The authors performed three-step in silico structure-based drug screening (SBDS) based on the crystal structure of saDHFR using a 154,118 chem. compd. library.  They subsequently evaluated whether candidate chem. compds. exhibited inhibitory effects on the growth of the model bacterium: Staphylococcus epidermidis.  Compd. KB1 showed a strong inhibitory effect on the growth of S. epidermidis.  The authors rescreened chem. structures similar to KB1 from a 461,397 chem. compd. library.  Three of the four KB1 analogs (KBS1, KBS3, and KBS4) showed inhibitory effects on the growth of S. epidermidis and enzyme inhibitory effects on saDHFR.  The authors performed structure-activity relationship (SAR) anal. of active chem. compds. and obsd. a correlative relationship among the IC50 values, interaction residues, and structure scaffolds.  In addn., the active chem. compds. (KB1, KBS3, and KBS4) had no inhibitory effects on the growth of model enterobacteria (Escherichia coli BL21 and JM109 strains) and no toxic effects on cultured mammalian cells (MDCK cells).  The results obtained from Protein Ligand Interaction Fingerprint (PLIF) and Ligand Interaction (LI) analyses suggested that all of the active compds. exhibited potential inhibitory effects on mutated saDHFR of the drug-resistant strains.  The structural and exptl. information concerning these novel chem. compds. will likely contribute to the development of new antibiotics for both wild-type and drug-resistant S. aureus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg4TaA44xcErVg90H21EOLACvtfcHk0ljyhRmMPZMU3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXks1yrsL8%253D&md5=a6da2186254d39cf533b4e3cae44bfa1</span></div><a href="/servlet/linkout?suffix=cit254&amp;dbid=16384&amp;doi=10.1021%2Fci400686d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400686d%26sid%3Dliteratum%253Aachs%26aulast%3DKobayashi%26aufirst%3DM.%26aulast%3DKinjo%26aufirst%3DT.%26aulast%3DKoseki%26aufirst%3DY.%26aulast%3DBourne%26aufirst%3DC.%2BR.%26aulast%3DBarrow%26aufirst%3DW.%2BW.%26aulast%3DAoki%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520novel%2520potential%2520antibiotics%2520against%2520Staphylococcus%2520using%2520structure-based%2520drug%2520screening%2520targeting%2520dihydrofolate%2520reductase%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1242%26epage%3D1253%26doi%3D10.1021%2Fci400686d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref255"><div class="reference"><strong class="refLabel"><a href="#ref255" class="rightTabRefNumLink">255</a></strong><div class="NLM_citation" id="rightTab-cit255"><span><span class="NLM_contrib-group">McGovern, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Kinase inhibitors: not just for kinases anymore</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">1478</span><span class="NLM_x">–</span> <span class="NLM_lpage">1483</span><span class="refDoi"> DOI: 10.1021/jm020427b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020427b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref255/cit255&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhvFeksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=1478-1483&author=S.+L.+McGovernauthor=B.+K.+Shoichet&title=Kinase+inhibitors%3A+not+just+for+kinases+anymore&doi=10.1021%2Fjm020427b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit255R"><div class="casContent"><span class="casTitleNuber">255</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase Inhibitors: Not Just for Kinases Anymore</span></div><div class="casAuthors">McGovern, Susan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1478-1483</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Kinase inhibitors are widely employed as biol. reagents and as leads for drug design.  Their use is often complicated by their lack of specificity.  Although binding conserved ATP sites accounts for some of their nonspecificity, some compds. inhibit proteins not known to bind ATP.  It has been found that promiscuous hits from high-throughput screening may act as aggregates.  To explore whether this mechanism might explain the action of widely used nonspecific kinase inhibitors, 15 such compds. were studied.  Eight of these, rottlerin, quercetin, K-252c, bisindolylmaleimide I, bisindolylmaleimide IX, U0126, indirubin, and indigo, inhibited three diverse non-kinase enzymes.  Inhibition was time-dependent and sensitive to enzyme concn.; by light scattering, the compds. formed particles of 100-1000 nm diam.  These observations suggest that these eight kinase inhibitors, at least at micromolar concns., are promiscuous and act as aggregates.  Results obtained from the use of these compds. at micromolar or higher concns. against individual enzymes should be interpreted cautiously.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-JSJ7FPf7e7Vg90H21EOLACvtfcHk0lj3uMcneYeEBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhvFeksLs%253D&md5=1d7e4e99a27be5ab279744aff92de26d</span></div><a href="/servlet/linkout?suffix=cit255&amp;dbid=16384&amp;doi=10.1021%2Fjm020427b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020427b%26sid%3Dliteratum%253Aachs%26aulast%3DMcGovern%26aufirst%3DS.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DKinase%2520inhibitors%253A%2520not%2520just%2520for%2520kinases%2520anymore%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D1478%26epage%3D1483%26doi%3D10.1021%2Fjm020427b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref256"><div class="reference"><strong class="refLabel"><a href="#ref256" class="rightTabRefNumLink">256</a></strong><div class="NLM_citation" id="rightTab-cit256"><span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karpiak, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kroeze, W. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moy, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saddoris, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nikolova, V. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farrell, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangano, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deshpande, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Penn, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koller, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenakin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span> </span><span class="NLM_article-title">Allosteric ligands for the pharmacologically dark receptors GPR68 and GPR65</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">527</span><span class="NLM_x">, </span> <span class="NLM_fpage">477</span><span class="NLM_x">–</span> <span class="NLM_lpage">483</span><span class="refDoi"> DOI: 10.1038/nature15699</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref256/cit256&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=10.1038%2Fnature15699" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref256/cit256&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=26550826" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref256/cit256&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVWmsrjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=527&publication_year=2015&pages=477-483&author=X.+P.+Huangauthor=J.+Karpiakauthor=W.+K.+Kroezeauthor=H.+Zhuauthor=X.+Chenauthor=S.+S.+Moyauthor=K.+A.+Saddorisauthor=V.+D.+Nikolovaauthor=M.+S.+Farrellauthor=S.+Wangauthor=T.+J.+Manganoauthor=D.+A.+Deshpandeauthor=A.+Jiangauthor=R.+B.+Pennauthor=J.+Jinauthor=B.+H.+Kollerauthor=T.+Kenakinauthor=B.+K.+Shoichetauthor=B.+L.+Roth&title=Allosteric+ligands+for+the+pharmacologically+dark+receptors+GPR68+and+GPR65&doi=10.1038%2Fnature15699"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit256R"><div class="casContent"><span class="casTitleNuber">256</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric ligands for the pharmacologically dark receptors GPR68 and GPR65</span></div><div class="casAuthors">Huang, Xi-Ping; Karpiak, Joel; Kroeze, Wesley K.; Zhu, Hu; Chen, Xin; Moy, Sheryl S.; Saddoris, Kara A.; Nikolova, Viktoriya D.; Farrell, Martilias S.; Wang, Sheng; Mangano, Thomas J.; Deshpande, Deepak A.; Jiang, Alice; Penn, Raymond B.; Jin, Jian; Koller, Beverly H.; Kenakin, Terry; Shoichet, Brian K.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">527</span>
        (<span class="NLM_cas:issue">7579</span>),
    <span class="NLM_cas:pages">477-483</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">At least 120 non-olfactory G-protein-coupled receptors in the human genome are 'orphans' for which endogenous ligands are unknown, and many have no selective ligands, hindering the detn. of their biol. functions and clin. relevance.  Among these is GPR68, a proton receptor that lacks small mol. modulators for probing its biol.  Using yeast-based screens against GPR68, here the authors identify the benzodiazepine drug lorazepam as a non-selective GPR68 pos. allosteric modulator.  More than 3000 GPR68 homol. models were refined to recognize lorazepam in a putative allosteric site.  Docking 3.1 million mols. predicted new GPR68 modulators, many of which were confirmed in functional assays.  One potent GPR68 modulator, ogerin, suppressed recall in fear conditioning in wild-type but not in GPR68-knockout mice.  The same approach led to the discovery of allosteric agonists and neg. allosteric modulators for GPR65.  Combining phys. and structure-based screening may be broadly useful for ligand discovery for understudied and orphan GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDB_IiCoSwVrVg90H21EOLACvtfcHk0lj3uMcneYeEBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVWmsrjJ&md5=a23d2d68c06b96cfbe7b53f2b17f3895</span></div><a href="/servlet/linkout?suffix=cit256&amp;dbid=16384&amp;doi=10.1038%2Fnature15699&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature15699%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DKarpiak%26aufirst%3DJ.%26aulast%3DKroeze%26aufirst%3DW.%2BK.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DMoy%26aufirst%3DS.%2BS.%26aulast%3DSaddoris%26aufirst%3DK.%2BA.%26aulast%3DNikolova%26aufirst%3DV.%2BD.%26aulast%3DFarrell%26aufirst%3DM.%2BS.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DMangano%26aufirst%3DT.%2BJ.%26aulast%3DDeshpande%26aufirst%3DD.%2BA.%26aulast%3DJiang%26aufirst%3DA.%26aulast%3DPenn%26aufirst%3DR.%2BB.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKoller%26aufirst%3DB.%2BH.%26aulast%3DKenakin%26aufirst%3DT.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DAllosteric%2520ligands%2520for%2520the%2520pharmacologically%2520dark%2520receptors%2520GPR68%2520and%2520GPR65%26jtitle%3DNature%26date%3D2015%26volume%3D527%26spage%3D477%26epage%3D483%26doi%3D10.1038%2Fnature15699" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-9%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b02008" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b02008" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677b17f88fa6231a","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
