
<html lang="en"     class="pb-page"  data-request-id="1f7ed4ec-1f24-49c2-8ff1-98b55277e914"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.0c01111;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2020.63.issue-17;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of First-in-Class Protein Arginine Methyltransferase 5 (PRMT5) Degraders" /></meta><meta name="dc.Creator" content="Yudao  Shen" /></meta><meta name="dc.Creator" content="Guozhen  Gao" /></meta><meta name="dc.Creator" content="Xufen  Yu" /></meta><meta name="dc.Creator" content="Huensuk  Kim" /></meta><meta name="dc.Creator" content="Li  Wang" /></meta><meta name="dc.Creator" content="Ling  Xie" /></meta><meta name="dc.Creator" content="Megan  Schwarz" /></meta><meta name="dc.Creator" content="Xian  Chen" /></meta><meta name="dc.Creator" content="Ernesto  Guccione" /></meta><meta name="dc.Creator" content="Jing  Liu" /></meta><meta name="dc.Creator" content="Mark T.  Bedford" /></meta><meta name="dc.Creator" content="Jian  Jin" /></meta><meta name="dc.Description" content="The aberrant expression of protein arginine methyltransferase 5 (PRMT5) has been associated with multiple cancers. Using the proteolysis targeting chimera technology, we discovered a first-in-class..." /></meta><meta name="Description" content="The aberrant expression of protein arginine methyltransferase 5 (PRMT5) has been associated with multiple cancers. Using the proteolysis targeting chimera technology, we discovered a first-in-class..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 11, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01111" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01111" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01111" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01111" /></link>
        
    
    

<title>Discovery of First-in-Class Protein Arginine Methyltransferase 5 (PRMT5) Degraders | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01111" /></meta><meta property="og:title" content="Discovery of First-in-Class Protein Arginine Methyltransferase 5 (PRMT5) Degraders" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0015.jpeg" /></meta><meta property="og:description" content="The aberrant expression of protein arginine methyltransferase 5 (PRMT5) has been associated with multiple cancers. Using the proteolysis targeting chimera technology, we discovered a first-in-class PRMT5 degrader 15 (MS4322). Here, we report the design, synthesis, and characterization of compound 15 and two structurally similar controls 17 (MS4370) and 21 (MS4369), with impaired binding to the von Hippel-Lindau E3 ligase and PRMT5, respectively. Compound 15, but not 17 and 21, effectively reduced the PRMT5 protein level in MCF-7 cells. Our mechanism studies indicate that compound 15 degraded PRMT5 in an E3 ligase- and proteasome-dependent manner. Compound 15 also effectively reduced the PRMT5 protein level and inhibited growth in multiple cancer cell lines. Moreover, compound 15 was highly selective for PRMT5 in a global proteomic study and exhibited good plasma exposure in mice. Collectively, compound 15 and its two controls 17 and 21 are valuable chemical tools for exploring the PRMT5 functions in health and disease." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01111"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01111">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01111&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01111&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01111&amp;href=/doi/10.1021/acs.jmedchem.0c01111" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 17</span><span class="cit-fg-pageRange">, 9977-9989</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/17" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01088" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01126" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of First-in-Class Protein Arginine Methyltransferase 5 (PRMT5) Degraders</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Yudao Shen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yudao Shen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yudao++Shen">Yudao Shen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guozhen Gao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guozhen Gao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Epigenetics and Molecular Carcinogenesis, The University of Texas MD Anderson Cancer Center, Smithville, Texas 78957, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guozhen++Gao">Guozhen Gao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xufen Yu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xufen Yu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xufen++Yu">Xufen Yu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Huensuk Kim</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Huensuk Kim</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Huensuk++Kim">Huensuk Kim</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Li Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Li Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Li++Wang">Li Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ling Xie</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ling Xie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ling++Xie">Ling Xie</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Megan Schwarz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Megan Schwarz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Megan++Schwarz">Megan Schwarz</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xian Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xian Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xian++Chen">Xian Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ernesto Guccione</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ernesto Guccione</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ernesto++Guccione">Ernesto Guccione</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jing Liu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jing Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#88e2e1e6efa6e4e1fdc8e5fbfbe5a6edecfd"><span class="__cf_email__" data-cfemail="7b1112151c5517120e3b16080816551e1f0e">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Liu">Jing Liu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4740-243X" title="Orcid link">http://orcid.org/0000-0003-4740-243X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mark T. Bedford</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mark T. Bedford</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Epigenetics and Molecular Carcinogenesis, The University of Texas MD Anderson Cancer Center, Smithville, Texas 78957, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#066b7264636260697462466b62676862637475696828697461"><span class="__cf_email__" data-cfemail="86ebf2e4e3e2e0e9f4e2c6ebe2e7e8e2e3f4f5e9e8a8e9f4e1">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mark+T.++Bedford">Mark T. Bedford</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jian Jin</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jian Jin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#a9c3c0c8c787c3c0c7e9c4dadac487cccddc"><span class="__cf_email__" data-cfemail="442e2d252a6a2e2d2a04293737296a212031">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Jin">Jian Jin</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link">http://orcid.org/0000-0002-2387-3862</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01111&amp;href=/doi/10.1021%2Facs.jmedchem.0c01111" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 17</span><span class="cit-pageRange">, 9977–9989</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 11, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>29 June 2020</li><li><span class="item_label"><b>Published</b> online</span>11 August 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 September 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01111" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01111</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9977%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYudao%2BShen%252C%2BGuozhen%2BGao%252C%2BXufen%2BYu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D17%26contentID%3Dacs.jmedchem.0c01111%26title%3DDiscovery%2Bof%2BFirst-in-Class%2BProtein%2BArginine%2BMethyltransferase%2B5%2B%2528PRMT5%2529%2BDegraders%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9989%26publicationDate%3DSeptember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01111"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2699</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">8</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01111" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of First-in-Class Protein Arginine Methyltransferase 5 (PRMT5) Degraders&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yudao&quot;,&quot;last_name&quot;:&quot;Shen&quot;},{&quot;first_name&quot;:&quot;Guozhen&quot;,&quot;last_name&quot;:&quot;Gao&quot;},{&quot;first_name&quot;:&quot;Xufen&quot;,&quot;last_name&quot;:&quot;Yu&quot;},{&quot;first_name&quot;:&quot;Huensuk&quot;,&quot;last_name&quot;:&quot;Kim&quot;},{&quot;first_name&quot;:&quot;Li&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Ling&quot;,&quot;last_name&quot;:&quot;Xie&quot;},{&quot;first_name&quot;:&quot;Megan&quot;,&quot;last_name&quot;:&quot;Schwarz&quot;},{&quot;first_name&quot;:&quot;Xian&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Ernesto&quot;,&quot;last_name&quot;:&quot;Guccione&quot;},{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Mark&quot;,&quot;last_name&quot;:&quot;T. Bedford&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Jin&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;11&quot;,&quot;issue&quot;:&quot;17&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;9977-9989&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01111&quot;},&quot;abstract&quot;:&quot;The aberrant expression of protein arginine methyltransferase 5 (PRMT5) has been associated with multiple cancers. Using the proteolysis targeting chimera technology, we discovered a first-in-class PRMT5 degrader 15 (MS4322). Here, we report the design, synthesis, and characterization of compound 15 and two structurally similar controls 17 (MS4370) and 21 (MS4369), with impaired binding to the von Hippel-Lindau E3 ligase and PRMT5, respectively. Compound 15, but not 17 and 21, effectively reduced the PRMT5 protein level in MCF-7 cells. Our mechanism studies indicate that compound 15 degraded PRMT5 in an E3 ligase- and proteasome-dependent manner. Compound 15 also effectively reduced the PRMT5 protein level and inhibited growth in multiple cancer cell lines. Moreover, compound 15 was highly selective for PRMT5 in a global proteomic study and exhibited good plasma exposure in mice. Collectively, compound 15 and its two controls 17 and 21 are valuable chemical tools for exploring the PRMT5 functions in health an&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01111&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01111" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01111&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01111" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01111&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01111" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01111&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01111&amp;href=/doi/10.1021/acs.jmedchem.0c01111" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01111" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01111" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01111%26sid%3Dliteratum%253Aachs%26pmid%3D32787082%26genre%3Darticle%26aulast%3DShen%26date%3D2020%26atitle%3DDiscovery%2Bof%2BFirst-in-Class%2BProtein%2BArginine%2BMethyltransferase%2B5%2B%2528PRMT5%2529%2BDegraders%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D17%26spage%3D9977%26epage%3D9989%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=291011" title="High-performance liquid chromatography">High-performance liquid chromatography</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/17" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/jmcmar.2020.63.issue-17/20200910/jmcmar.2020.63.issue-17.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The aberrant expression of protein arginine methyltransferase 5 (PRMT5) has been associated with multiple cancers. Using the proteolysis targeting chimera technology, we discovered a first-in-class PRMT5 degrader <b>15</b> (MS4322). Here, we report the design, synthesis, and characterization of compound <b>15</b> and two structurally similar controls <b>17</b> (MS4370) and <b>21</b> (MS4369), with impaired binding to the von Hippel-Lindau E3 ligase and PRMT5, respectively. Compound <b>15</b>, but not <b>17</b> and <b>21</b>, effectively reduced the PRMT5 protein level in MCF-7 cells. Our mechanism studies indicate that compound <b>15</b> degraded PRMT5 in an E3 ligase- and proteasome-dependent manner. Compound <b>15</b> also effectively reduced the PRMT5 protein level and inhibited growth in multiple cancer cell lines. Moreover, compound <b>15</b> was highly selective for PRMT5 in a global proteomic study and exhibited good plasma exposure in mice. Collectively, compound <b>15</b> and its two controls <b>17</b> and <b>21</b> are valuable chemical tools for exploring the PRMT5 functions in health and disease.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81100" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81100" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As the predominant member of type II protein arginine methyltransferases (PRMTs), protein arginine methyltransferase 5 (PRMT5) catalyzes monomethylation and symmetric dimethylation of arginine residues of its histone substrates, including H3R2, H3R8, and H4R3, and its nonhistone substrates, including p53, EGFR, N-MYC, SmD3, and RNA polymerase II, when interacting with its binding partner MEP50.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5">(1−5)</a> Through methylation of these substrates, PRMT5 regulates multiple biological processes, such as chromatin remodeling, gene expression, mRNA splicing, DNA replication and repair, and cell cycle regulation.<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6−8)</a> The aberrant expression of PRMT5 has been associated with infectious disease, heart disease, and cancers, including breast cancer, lung cancer, and hepatocellular cancer (HCC).<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9−11)</a> Genetic knockdown of PRMT5 was effective to suppress tumor growth in vitro and <i>in vivo</i>.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> For example, PRMT5 knockdown decreased the proliferation, invasion, and migration of HCC cell lines.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The knockdown of PRMT5 in AR-positive LNCaP prostate cancer cells completely suppressed tumor growth in xenograft mouse models.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> In addition, the knockdown of PRMT5 drastically prolonged the survival in a murine model of BCR-ABL-driven chronic myelogenous leukemia<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> and of MYC-driven B cell lymphoma.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Moreover, the PRMT5 knockdown effectively suppressed the growth of patient-derived xenograft glioblastoma (GBM) tumors <i>in vivo</i>.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Collectively, these findings suggest that PRMT5 downregulation may provide a potential therapeutic strategy for treating cancer.</div><div class="NLM_p">Multiple small-molecule PRMT5 inhibitors have been reported.<a onclick="showRef(event, 'ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26 ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26 ref27 ref28 ref29">(19−29)</a> Among them, GSK3326595 and JNJ64619178 are currently being evaluated in the clinic. These compounds potently inhibited the methyltransferase activity of PRMT5, and some of them displayed promising tumor growth inhibition effects in animal models. However, different from PRMT5 genetic knockdown, these catalytic inhibitors cannot eliminate the potential scaffolding functions of PRMT5.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Recently, proteolysis targeting chimera (PROTAC) has emerged as a powerful technology to reduce the protein levels of the target of interest by hijacking the cellular ubiquitin–proteasome system.<a onclick="showRef(event, 'ref30 ref31 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32 ref33 ref34">(30−34)</a> PROTACs are expected to phenocopy the effects of genetic knockdown or knockout as they not only inhibit the target proteins as traditional inhibitors do but also more importantly eliminate the other functions (e.g., scaffolding function) of the protein targets. Another potential advantage of PROTACs over occupancy-driven small-molecule inhibitors is that PROTACs may potentially overcome the resistance to small-molecule inhibitor treatments.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> To the best of our knowledge, PRMT5 PROTAC degraders have not been reported to date.</div><div class="NLM_p last">In this study, we report the design, synthesis, and characterization of first-in-class PRMT5 degraders. Through a brief structure–activity relationship (SAR) study, a potent PRMT5 degrader, compound <b>15</b> (MS4322), was discovered by linking the PRMT5 inhibitor EPZ015666<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> to a von Hippel-Lindau (VHL) E3 ligase ligand, (<i>S</i>,<i>R</i>,<i>S</i>)-AHPC-Me (VHL-2). Compound <b>15</b> effectively reduced PRMT5 protein levels in multiple cell lines. In particular, it reduced the PRMT5 protein level in MCF7 cells in a concentration-, time-, VHL- and proteasome-dependent manner. In addition, compound <b>15</b> inhibited cell growth at least as effectively as EPZ015666 in multiple cancer cell lines and exhibited good plasma exposure in a mouse pharmacokinetic (PK) study. Furthermore, the results from our mass spectrometry (MS)-based global proteomics analysis indicate that compound <b>15</b> is a highly selective PRMT5 degrader. To the best of our knowledge, compound <b>15</b> is also the first degrader of any PRMTs, a class of important epigenetic enzymes. Moreover, we developed two structurally similar control compounds, <b>21</b> (MS4369) and <b>17</b> (MS4370), which displayed an impaired binding to PRMT5 and VHL, respectively, and could not reduce PRMT5 protein levels in cells. Overall, we provide compound <b>15</b> and its two control compounds <b>17</b> and <b>21</b> to the research community as useful chemical tools for further investigating the physiological and pathophysiological functions of PRMT5.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42263" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42263" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design, Synthesis, and Evaluation of an Initial Set of Putative PRMT5 Degraders</h3><div class="NLM_p">EPZ015666 is the first potent and selective PRMT5 inhibitor, and the X-ray cocrystal structure of PRMT5:MEP50 in complex with EPZ015666 and the cofactor S-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine (SAM) revealed that EPZ015666 binds in the substrate-binding site of PRMT5.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Importantly, the oxetane moiety of EPZ015666 is solvent-exposed and does not interact with any PRMT5 residues (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). We therefore hypothesized that the addition of a linker to this moiety would have a minimal effect on the binding of the resulting compounds to PRMT5. To facilitate the installation of a linker, we replaced the oxetane group with an azetidine group, leading to compound <b>3</b>. Five putative PRMT5 degraders <b>9</b>–<b>13</b> were designed by linking compound <b>3</b> to the E3 ligase VHL binder VHL-1 with five different linkers (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Design of PRMT5 putative degraders <b>9</b>, <b>10</b>, <b>11</b>, <b>12</b>, and <b>13</b>. (A) Crystal structure of PRMT5:MEP50 (gray) in complex with SAM (magenta) and EPZ015666 (green) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X61">4X61</a>). The highlighted oxetane moiety of EPZ015666 is solvent-exposed. Key H-bond interactions between the EPZ015666 and PRMT5 residues are highlighted by red dashed lines. (B) Chemical structures of EPZ015666, compound <b>3</b>, and the five putative PRMT5 degraders <b>9</b>–<b>13</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We developed a concise synthetic route to prepare bivalent compounds <b>9</b>–<b>13</b>. Commercially available 1,2,3,4-tetrahydroisoquinoline and <i>tert</i>-butyl (<i>S</i>)-(oxiran-2-ylmethyl)carbamate underwent epoxide opening, deprotection, and amide coupling with 6-chloropyrimidine-4-carboxylic acid, yielding compound <b>1</b> but not compound <b>1a</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). This is most likely due to the fact that the chloro group on the pyrimidine ring was substituted by 1-hydroxy-7-aza-benzo-triazole (HOAt) to yield <b>1</b> under the amide coupling reaction condition. Fortunately, compound <b>1</b> can be readily converted to <b>2</b> using <i>tert</i>-butyl 3-aminoazetidine-1-carboxylate as a nucleophilic substitution reagent. Deprotection of intermediate <b>2</b> under the acidic condition afforded intermediate <b>3</b>. The amide coupling reactions between the intermediate <b>3</b> and linker-attached VHL-1 derivatives <b>4</b>–<b>8</b> resulted in the desired compounds <b>9</b>–<b>13</b>.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0013.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of PRMT5 Putative Degraders <b>9</b>–<b>13</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>i</i>-PrOH, reflux; (b) TFA, DCM; (c) EDCI·HCl, HOAt, NMM, 6-chloropyrimidine-4-carboxylic acid, DMSO, 18% over three steps; (d) <i>tert</i>-butyl 3-aminoazetidine-1-carboxylate, NMP, 90%; (e) dicarboxylic acids, EDCI·HCl, HOAt, NMM, DMSO, 54–78%; (f) <b>3</b>, EDCI·HCl, HOAt, NMM, DMSO, 36–60%.</p></p></figure><div class="NLM_p">Previous studies have shown that MCF-7 cells, an estrogen receptor (ER) + breast cancer cell line, have sufficient PRMT5 and MEP50 protein levels.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> In addition, MCF-7 cells were successfully used for assessing the VHL-recruiting ER degrader ERD-308.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> These results indicate that MCF-7 cells are suitable for evaluating the effect of our putative PRMT5 degraders on reducing the PRMT5 protein level. We therefore treated MCF-7 cells with compounds <b>9</b>–<b>13</b> at three different concentrations (0.2, 1, and 5 μM). As illustrated by <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, although compounds <b>9</b>–<b>11</b>, which contain relatively short polyethylene glycol (PEG) linkers, and compound <b>12</b>, which contains an all carbon linker, did not significantly reduce PRMT5 protein levels, compound <b>13</b>, which contains a longer PEG linker, reduced the PRMT5 protein level at 1 μM and more effectively at 5 μM.</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Effects of compounds <b>9</b>–<b>13</b> on reducing the PRMT5 protein level in MCF-7 cells. MCF-7 cells were treated with compounds <b>9</b>–<b>13</b> at the indicated concentrations (0.2, 1, and 5 μM) for 6 d. Cell lysates were collected, and the PRMT5 protein levels were detected by western blots. The results are representative of at least two independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Design and Synthesis of PRMT5 Degrader <b>15</b> and Its Control Compounds <b>17</b> and <b>21</b></h3><div class="NLM_p">It has been reported that the VHL binder (<i>S</i>,<i>R</i>,<i>S</i>)-AHPC-Me (VHL-2) (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>) can result in more effective degraders than the ones derived from VHL-1.<a onclick="showRef(event, 'ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref38 ref39">(38,39)</a> For example, compared with VHL-1-based MEK1/2 degraders, VHL-2-based degraders displayed a higher potency in reducing the MEK1/2 protein levels.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Therefore, we designed compound <b>15</b> by replacing the VHL-1 moiety of degrader <b>13</b>, the most effective compound from our initial SAR study, with VHL-2 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). In addition, we designed two close analogues, compounds <b>17</b> and <b>21</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), as controls for compound <b>15</b>. Compound <b>17</b> is a diastereoisomer of compound <b>15</b> and was designed to diminish its binding to the E3 ligase VHL, based on the previous reports that the diastereoisomer of VHL-2 led to impaired binding to VHL,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> while possessing the same linker and PRMT5 binding moiety, thus maintaining a similar inhibitory potency for PRMT5 as compound <b>15</b>. It was previously reported that the replacement of the 1,2,3,4-tetrahydroisoquinoline moiety of EPZ015666 with the piperidine group led to a significantly diminished PRMT5 inhibition activity.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> We therefore designed the control compound <b>21</b> by replacing the 1,2,3,4-tetrahydroisoquinoline moiety of compound <b>15</b> with the piperidine group to reduce its inhibitory potency for PRMT5, while maintaining the same binding affinity to VHL as compound <b>15</b>. Because of their high structural similarity, compounds <b>15</b>, <b>17</b>, and <b>21</b> are expected to have similar physicochemical properties.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Design of the PRMT5 degrader <b>15</b> and its two control compounds <b>17</b> and <b>21</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0014.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>15</b>, <b>17</b>, and <b>21</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) EDCI·HCl, HOAt, NMM, DMSO, 50%; (b) <b>3</b>, EDCI·HCl, HOAt, NMM, DMSO, 35–53%; (c) <i>i</i>-PrOH, reflux; (d) TFA, DCM; (e) EDCI·HCl, HOAt, NMM, 6-chloropyrimidine-4-carboxylic acid, DMSO; (f) <i>tert</i>-butyl 3-aminoazetidine-1-carboxylate, NMP, 13% over four steps; (g) <b>14</b>, EDCI·HCl, HOAt, NMM, DMSO, 49%.</p></p></figure><div class="NLM_p last">Compounds <b>15</b>, <b>17</b>, and <b>21</b> were prepared using the synthetic routes outlined in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Starting from VHL-2<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> and commercially available 3,6,9,12,15-pentaoxaheptadecanedioic acid, intermediate <b>14</b> was produced under an amide coupling condition. Under the same amide coupling condition, degrader <b>15</b> was prepared from carboxylic acid <b>14</b> and amine <b>3</b>. Using the same synthetic sequence, compound <b>17</b> was synthesized from a diastereoisomer of VHL-2, which was prepared according to the published procedures.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> To synthesize compound <b>21</b>, intermediate <b>19</b> was prepared using the same synthetic route as preparing intermediate <b>3</b> from piperidine and <i>tert</i>-butyl (<i>S</i>)-(oxiran-2-ylmethyl)carbamate, via epoxide opening, deprotection, amide coupling, and nucleophilic aromatic substitution (SN<sub>Ar</sub>) reactions. Deprotection of intermediate <b>19</b> provided intermediate <b>20</b>. An amide coupling reaction between amine <b>20</b> and acid <b>14</b> yielded compound <b>21</b>.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Inhibition of PRMT5 Methyltransferase Activity in Biochemical Assays</h3><div class="NLM_p">We first evaluated the effect of compounds <b>15</b>, <b>17</b>, and <b>21</b> on inhibiting the methyltransferase activity of PRMT5 in a human PRMT5/MEP50 radioactive biochemical assay using tritiated SAM (<sup>3</sup>H-SAM) as the methyl donor. Compared with the PRMT5 inhibitor EPZ015666 (IC<sub>50</sub> = 30 ± 3 nM), compound <b>15</b> (IC<sub>50</sub> = 18 ± 1 nM), which contains a linked VHL-2 moiety, and compound <b>17</b> (IC<sub>50</sub> = 12 ± 1 nM), which contains a diastereoisomer of VHL-2, maintained high potency in inhibiting the PRMT5 enzymatic activity (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). As expected, compound <b>21</b>, which comprises a piperidinyl group instead of the tetrahydroisoquinolinyl group on the PRMT5 binding moiety, indeed displayed much reduced potency in inhibiting the PRMT5 methyltransferase activity.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effects of compounds <b>15</b>, <b>17</b>, and <b>21</b> on inhibiting the PRMT5 methyltransferase activity in a radioactive biochemical assay. Substrate: H4 (<b>1–15</b>)-biotin (400 nM). Cofactor: <sup>3</sup>H-SAM (1 μM). EPZ015666 was used as a positive control. IC<sub>50</sub> determination experiments were performed in duplicate, and the values are presented as mean ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> PRMT5 Degradation and Mechanism of Action Studies in MCF-7 Cells</h3><div class="NLM_p">We next assessed the PRMT5 degradation effects of compounds <b>15</b>, <b>17</b>, and <b>21</b> in MCF-7 cells. We were pleased to find that compound <b>15</b> effectively reduced the PRMT5 protein level in a concentration-dependent manner, with a DC<sub>50</sub> value of 1.1 ± 0.6 μM and <i>D</i><sub>max</sub> = 74 ± 10% (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A,B), and compound <b>15</b> was more effective than compound <b>13</b> at degrading PRMT5 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). In addition, the two control compounds, <b>17</b> and <b>21</b>, and the PRMT5 inhibitor EPZ015666 did not reduce PRMT5 protein levels at 5 μM (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). Thus, compounds <b>17</b> and <b>21</b> are good negative controls for the PRMT5 degradation activity of compound <b>15</b>. Next, we conducted a time course study for determining the effect of compound <b>15</b> on PRMT5 degradation. MCF-7 cells were treated with compound <b>15</b> at 5 μM for 0–8 d. Significant PRMT5 degradation was observed after a 2 d treatment, and the degradation effect was gradually increased to the maximum after a 6 to 8 d treatment (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). It is worth to note that the degradation kinetics by compound <b>15</b> is significantly slower than that by typical degraders targeting kinases.<a onclick="showRef(event, 'ref39 ref40 ref41 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref39 ref40 ref41 ref42 ref43">(39−43)</a> It is unclear why PRMT5 degradation induced by compound <b>15</b> is slower. This warrants further investigation. We also compared the effect of compound <b>15</b> and EPZ015666 on inhibiting global arginine symmetric dimethylation (SDMA) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D). In MCF-7 cells treated with 0.1, 0.3, 1, and 3.0 μM of these two compounds for 6 d, both the PRMT5 degrader <b>15</b> and the inhibitor EPZ015666 concentration-dependently decreased global SDMA, and compound <b>15</b> appeared slightly more effective than EPZ015666.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Effects of compound <b>15</b> on reducing the PRMT5 protein level and inhibiting SDMA in MCF-7 cells. (A,B) Compound <b>15</b> concentration-dependently reduced the PRMT5 protein level, whereas compounds <b>17</b> and <b>21</b> and EPZ015666 did not. MCF-7 cells were treated with the indicated concentration of <b>15</b>, <b>17</b>, <b>21</b>, or EPZ015666 for 6 d. Cell lysates were collected, and the protein levels of PRMT5 were detected by western blotting. The relative strength of the PRMT5 signal on the western blots was measured by densitometry. DC<sub>50</sub> and <i>D</i><sub>max</sub> values are presented as mean ± SD from three independent experiments. (C) Compound <b>15</b> reduced the PRMT5 protein level in a time-dependent manner. MCF-7 cells were treated with 5 μM of compound <b>15</b> for 0, 2, 4, 6, or 8 d. The PRMT5 protein levels were detected by western blotting. (D) Compound <b>15</b> and EPZ015666 inhibited SDMA. MCF-7 cells were treated with the indicated concentration of EPZ015666 or compound <b>15</b> for 6 d. The SDMA levels were detected by western blotting. Western blot results are representative of at least two independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To demonstrate that the observed PRMT5 degradation induced by compound <b>15</b> is dependent on the ubiquitin proteasome system, we perform a set of rescue experiments. We co-treated MCF-7 cells with 5 μM of compound <b>15</b> and 30 μM of PRMT5 inhibitor EPZ015666, 100 μM of VHL E3 ligase ligand VH-298,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> 2 μM of neddylation inhibitor MLN4924,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> or 30 μM of proteasome inhibitor MG-132. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A, the co-treatment with EPZ015666, which competes with compound <b>15</b> at the PRMT5 substrate binding site, or VH-298, which competes with compound <b>15</b> at the VHL binding site, significantly rescued the reduction of the PRMT5 protein level compared to the dimethyl sulfoxide (DMSO) control (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A, lane 2). Similarly, co-treatment with the proteasome inhibitor MG-132 or the cullin-RING E3 ubiquitin ligase (CRL) neddylation inhibitor MLN4924 also partially restored the PRMT5 protein level in MCF-7 cells. These results, together with the lack of degradation effects by the control compounds <b>17</b> and <b>21</b>, indicate that compound <b>15</b> degrades the PRMT5 protein through a PRMT5-, VHL-, and proteasome-dependent mechanism.</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Compound <b>15</b> reduced the PRMT5 protein level in MCF-7 cells in a PRMT5-, E3 ligase VHL-, and proteasome-dependent manner. MCF-7 cells were treated with 5 μM of compound <b>15</b> for 7 d. During the last 24 h, the cells were co-treated with 100 μM VH-298, 2 μM MLN4924, 30 μM MG-132, or 30 μM EPZ015666. The protein levels of PRMT5 were detected by western blotting. (B) PRMT5 degradation induced by compound <b>15</b> was reversible. MCF-7 cells were treated with 5 μM of compound <b>15</b> for 6 d, and compound <b>15</b> was then washed out. The PRMT5 protein levels were detected by western blotting at 0, 12, 24, and 48 h post the washout.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">It has been reported that PROTAC-induced protein degradation does not impair the resynthesis of the protein.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> We therefore evaluated whether the PRMT5 degradation induced by compound <b>15</b> can be rescued by using a washout assay (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). We treated MCF-7 cells with 5 μM of compound <b>15</b> for 6 d and then washed out compound <b>15</b> and found that the PRMT5 protein level was gradually recovered over 12, 24, and 48 h. These results suggest that the cellular PRMT5 protein level can be recovered after the removal of compound <b>15</b>.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Effects of PRMT5 Degrader <b>15</b> on Cell Growth Inhibition</h3><div class="NLM_p">We next evaluated the antiproliferative effect of degrader <b>15</b> in MCF-7 cells. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>, compound <b>15</b> potently inhibited MCF-7 cell proliferation in a concentration-dependent manner. It was at least as effective as the positive control EPZ015666. This result is consistent with the effect of compound <b>15</b> and EPZ015666 on inhibiting global SDMA (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D). As expected, the negative control compound <b>21</b>, with much reduced PRMT5 inhibitory activity (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>), did not display any antiproliferative effect on MCF-7 cells. Importantly, the control compound <b>17</b>, which exhibited similar PRMT5 inhibitory activity as compound <b>15</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>) but did not degrade PRMT5 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A), displayed much weaker antiproliferative activity than compound <b>15</b>. Collectively, these results suggest that the observed antiproliferative effect of compound <b>15</b> is mainly because of its PRMT5 degradation activity but not its PRMT5 inhibitory activity.</div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Effects of compounds <b>15</b>, <b>17</b>, <b>21</b>, and EPZ015666 on inhibiting the proliferation of MCF-7 cells. MCF-7 cells were treated with the indicated concentration of compounds <b>15</b>, <b>17</b>, <b>21</b>, or EPZ015666 for 6 d. The relative cell viabilities to the untreated group are shown as mean ± SD (<i>n</i> = 3).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Compound <b>15</b> is a Highly Selective PRMT5 Degrader</h3><div class="NLM_p">To assess the selectivity of compound <b>15</b> across the proteome, we performed MS-based label-free quantitative (LFQ) global proteomics analysis of MCF-7 cells treated with compound <b>15</b> or DMSO (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). The samples for MS analysis were prepared by treating MCF-7 cells with compound <b>15</b> (at 5 μM) or DMSO for 5 d. The compound concentration and treatment time were selected based on the above concentration range and time course studies (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A–C). Out of the more than 5000 proteins identified, PRMT5, its binding partner WDR77 (MEP50), and AGRN (an unrelated protein) are the only proteins whose levels were reduced by compound <b>15</b>, as indicated by the volcano plot (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). These results strongly suggest that compound <b>15</b> is a highly selective PRMT5 degrader.</div><figure id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Global proteomic analysis indicates that compound <b>15</b> is a highly selective PRMT5 degrader. MCF-7 cells were treated with compound <b>15</b> at 5 μM or DMSO for 5 d and then harvested and lysed for liquid chromatography–MS (LC–MS) analysis. LFQ was used to calculate the peptide intensity; the relative abundance of PRMT5 (LFQ intensity value) between the compound <b>15</b>-treated and DMSO-treated groups is shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We next evaluated the effects of compound <b>15</b> on PRMT5 degradation and cell growth inhibition in other cancer cell lines. We were pleased to find that the PRMT5 degradation induced by compound <b>15</b> is not limited to MCF-7 cells. The PRMT5 protein levels in Hela (cervical cancer), A549 (lung adenocarcinoma), A172 (GBM), and Jurkat (leukemia) cells were significantly reduced after the cells were treated with compound <b>15</b> at 5 μM for 6 d (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>). Furthermore, compound <b>15</b> concentration-dependently inhibited the proliferation of Hela, A549, A172, and Jurkat cells, with a similar potency as the PRMT5 inhibitor EPZ015666 in Hela, A549, and Jurkat cells, and a slightly better potency than EPZ015666 in A172 cells (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>).</div><figure id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Compound <b>15</b> reduced the PRMT5 protein levels in HeLa, A549, A172, and Jurkat cells. The cells were treated with 5 μM of compound <b>15</b> for 6 d. The protein levels of PRMT5 were detected by western blotting. The western blotting results are representative of two independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig10" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Compound <b>15</b> and EPZ015666 inhibited the proliferation of HeLa (A), A549 (B), A172 (C), and Jurkat (D) cells. The cells were treated with <b>15</b> or EPZ015666 at the indicated concentrations for 6 d. The relative cell viabilities to the untreated group are shown as mean ± SD (<i>n</i> = 3).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> <i>In Vivo</i> Mouse PK Study</h3><div class="NLM_p">Lastly, we assessed the <i>in vivo</i> mouse PK properties of compound <b>15</b>. Compound concentrations in plasma from male Swiss albino mice were determined following a single intraperitoneal (IP) administration of compound <b>15</b> at 150 mg/kg. We were pleased to find that a single IP injection resulted in good exposure levels in plasma (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>). In particular, a very high plasma concentration (14 ± 2 μM) was achieved at 2 h post dosing. Even after 12 h, the concentration of compound <b>15</b> was still above 100 nM. In addition, compound 15 was well tolerated by the treated mice, and no clinical signs were observed in the PK study. Taken together, these results suggest that compound <b>15</b> could be a valuable chemical tool for investigating the effects of PRMT5 degradation <i>in vivo</i>.</div><figure id="fig11" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Plasma concentrations of compound <b>15</b> over 12 h, following a single 150 mg/kg IP injection in male Swiss albino mice. The compound concentration shown at each time point is the mean ± SD from three test mice.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56654" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56654" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Although the PROTAC technology has been successfully applied to the generation of numerous small-molecule degraders for a number of kinases,<a onclick="showRef(event, 'ref32 ref40 ref41 ref47'); return false;" href="javascript:void(0);" class="ref ref32 ref40 ref41 ref47">(32,40,41,47)</a> including ALK, MEK/1/2, and EGFR degraders recently discovered by us,<a onclick="showRef(event, 'ref39 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref39 ref42 ref43">(39,42,43)</a> very few small-molecule degraders of histone methyltransferases (HMTs, also known as protein methyltransferases) have been reported.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Through a brief SAR study, we discovered a first-in-class PRMT5 degrader, compound <b>15</b>, which is also the first degrader of any PRMTs, an important subclass of HMTs. Compound <b>15</b> is an E3 ligase VHL-recruiting bifunctional small molecule, effectively reducing the PRMT5 protein level in MCF-7 cells in a concentration-, time-, PRMT5-, VHL-, and proteasome-dependent manner. The results from our MS-based quantitative global proteomics analysis indicate that compound <b>15</b> is a highly selective PRMT5 degrader. Compound <b>15</b> was at least as effective as the parent PRMT5 inhibitor EPZ015666 at inhibiting global SDMA and proliferation in MCF-7 cells. In addition, compound <b>15</b> was able to significantly reduce PRMT5 protein levels in other cancer lines such as Hela, A549, A172, and Jurkat cells and inhibit the proliferation of these cells. Furthermore, compound <b>15</b> displayed good plasma exposure in a mouse PK study. Moreover, we developed two structurally similar control compounds, <b>17</b>, which exhibited similar PRMT5 inhibitory activity as compound <b>15</b> but did not degrade PRMT5, and <b>21</b>, which displayed much reduced inhibitory activity for PRMT5 and did not degrade PRMT5. Collectively, compound <b>15</b> and its controls <b>17</b> and <b>21</b> are a set of valuable chemical tools for further exploring PRMT5 functions in physiology and pathophysiology.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88184" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88184" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Chemistry General Procedure</h3><div class="NLM_p">All commercially available chemical reagents were directly used without further purification. A Teledyne ISCO CombiFlash Rf<sup>+</sup> instrument equipped with a variable wavelength UV detector and a fraction collector was used to conduct flash column chromatography. Redi<i>SepR</i>f HP C18 and flash silica columns were used for purification. High-performnace liquid chromatography (HPLC) spectra for all compounds were acquired using an Agilent 1200 series system with a diode array detector. Chromatography was performed on a 2.1 × 150 mm Zorbax 300SB-C18 5 μm column with water containing 0.1% formic acid as solvent A and acetonitrile containing 0.1% formic acid as solvent B at a flow rate of 0.4 mL/min. The gradient program was as follows: 1% B (0–1 min), 1–99% B (1–4 min), and 99% B (4–8 min). High-resolution mass spectrometry (HRMS) data were acquired in positive ion mode using an Agilent G1969A API-TOF system with an electrospray ionization (ESI) source. Nuclear magnetic resonance (NMR) spectra were acquired on either a Bruker DRX-600 spectrometer (800, 600 MHz <sup>1</sup>H NMR) or a Bruker Avance-III 800 MHz spectrometer (201 MHz <sup>13</sup>C NMR). Chemical shifts are reported in ppm (δ). Preparative HPLC was performed on Agilent Prep 1200 series with a UV detector set to 254 nm or 220 nM. Samples were injected into a Phenomenex Luna 5 μM C<sub>18</sub> column, with dimensions of 250 × 30 mm, at room temperature. The flow rate was 40 mL/min. A linear gradient was used with 10% of MeOH (A) in H<sub>2</sub>O (with 0.1% trifluoroacetic acid (TFA)) (B) to 100% of MeOH (A). HPLC was used to establish the purity of target compounds. All final compounds had >95% purity using the HPLC methods described above. All final compounds are characterized by the salt form of trifluoroacetic acid.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> (<i>S</i>)-6-((3<i>H</i>-[1,2,3]Triazolo[4,5-<i>b</i>]pyridin-3-yl)oxy)-<i>N</i>-(3-(3,4-dihydroisoquinolin-2(1<i>H</i>)-yl)-2-hydroxypropyl)pyrimidine-4-carboxamide (<b>1</b>)</h4><div class="NLM_p last">To the solution of <i>tert</i>-butyl (<i>S</i>)-(oxiran-2-ylmethyl)carbamate (1 g, 5.77 mmol) in isopropanol (10 mL) was added 1,2,3,4-tetrahydroisoquinoline (770 mg, 5.77 mmol). The resulting solution was heated at reflux for 6 h, before the volatile was removed under reduced pressure. The resulting residue was dissolved in dichloromethane (DCM, 5 mL). After the resulting solution was treated with TFA (5 mL) for 1 h, the volatiles were evaporated under reduced pressure. The resulting brown oil was used in the next step without further purification. The residue was added to a solution of 6-chloropyrimidine-4-carboxylic acid (951 mg, 6 mmol), 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide (EDCI, 1.66 g, 8.67 mmol), HOAt (1.9 g, 14.45 mmol), and <i>N</i>-methylmorpholine (NMM, 1.12 g, 11.12 mmol) in DMSO (30 mL), and the solution was stirred for 6 h. Reverse-phase chromatography was used to purify the mixture without additional workup to give intermediate <b>1</b> (580 mg, 18%). <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.75 (dd, <i>J</i> = 4.4, 1.5 Hz, 1H), 8.61 (dd, <i>J</i> = 8.5, 1.5 Hz, 1H), 8.37 (d, <i>J</i> = 1.2 Hz, 1H), 7.99 (d, <i>J</i> = 1.2 Hz, 1H), 7.61 (dd, <i>J</i> = 8.5, 4.5 Hz, 1H), 7.10–7.02 (m, 3H), 6.98 (d, <i>J</i> = 7.3 Hz, 1H), 4.17–4.08 (m, 1H), 3.73 (s, 2H), 3.65–3.51 (m, 2H), 2.97–2.80 (m, 4H), 2.75–2.65 (m, 2H). MS (ESI) <i>m</i>/<i>z</i>: 447.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> <i>tert</i>-Butyl (<i>S</i>)-3-((6-((3-(3,4-Dihydroisoquinolin-2(1<i>H</i>)-yl)-2-hydroxypropyl)carbamoyl) pyrimidin-4-yl)amino)azetidine-1-carboxylate (<b>2</b>)</h4><div class="NLM_p last">To a solution of <b>1</b> (580 mg, 1 mmol) in <i>N</i>-methyl-2-pyrrolidone (NMP, 10 mL) was added <i>tert</i>-butyl 3-aminoazetidine-1-carboxylate (1 g, 5.81 mmol). The resulting mixture was stirred overnight before being quenched with water (50 mL). The mixture was extracted with ethyl acetate (3 × 50 mL). The combined organic phase was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The resulting residue was purified by silica gel flash chromatography (methanol/DCM, 0–10%) to afford intermediate <b>2</b> (434 mg, yield 90%). <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.25 (s, 1H), 7.14–7.02 (m, 4H), 6.99 (d, <i>J</i> = 7.3 Hz, 1H), 4.71–4.62 (m, 1H), 4.33–4.20 (m, 2H), 4.10–4.03 (m, 1H), 3.86–3.76 (m, 2H), 3.71 (s, 2H), 3.56–3.44 (m, 2H), 2.92 (t, <i>J</i> = 5.9 Hz, 2H), 2.88–2.76 (m, 2H), 2.65 (d, <i>J</i> = 6.1 Hz, 2H), 1.44 (s, 9H). MS (ESI) <i>m</i>/<i>z</i>: 483.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> (<i>S</i>)-6-(Azetidin-3-ylamino)-<i>N</i>-(3-(3,4-dihydroisoquinolin-2(1<i>H</i>)-yl)-2-hydroxypropyl)pyrimidine-4-carboxamide (<b>3</b>)</h4><div class="NLM_p last">To a solution of <b>2</b> (434 mg, 0.9 mmol) in DCM (5 mL) was added TFA (5 mL). After the reaction was stirred for 30 min, the volatiles were removed under reduced pressure to give intermediate <b>3</b> as a TFA salt (560 mg, yield 100%). <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.60 (s, 1H), 7.36–7.14 (m, 5H), 4.67–4.57 (m, 1H), 4.46–4.31 (m, 4H), 4.27–4.19 (m, 2H), 3.94–3.80 (m, 1H), 3.61–3.07 (m, 8H). MS (ESI) <i>m</i>/<i>z</i>: 383.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Synthesis of VHL-1 PEG Linkers (<b>4–8</b>)</h4><div class="NLM_p last">To a solution of diacid (4 mmol) in dimethylformamide (10 mL) and DCM (250 mL) was added NMM (10 mmol), VHL-1 (2 mmol), HOAt (2.4 mmol), and EDCI (2.4 mmol) at 0 °C. The resulting reaction solution was stirred at 0 °C for 6 h before being stirred at room temperature (rt) overnight. The progress of the reaction was monitored by LC–MS. After VHL-1 was totally consumed, the reaction was concentrated. The resulting residue was purified by reverse-phase chromatography to yield the desired product.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> 2-(2-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-Hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl) pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-2-oxoethoxy)acetic Acid (<b>4</b>)</h4><div class="NLM_p last">(810 mg, yield 69%) as white solid. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 8.97 (s, 1H), 7.47 (d, <i>J</i> = 8.2 Hz, 2H), 7.43 (d, <i>J</i> = 8.1 Hz, 2H), 4.69 (s, 1H), 4.60–4.47 (m, 3H), 4.36 (d, <i>J</i> = 15.5 Hz, 1H), 4.27–4.17 (m, 2H), 4.16–4.07 (m, 2H), 3.89 (d, <i>J</i> = 11.0 Hz, 1H), 3.81 (dd, <i>J</i> = 11.0, 3.8 Hz, 1H), 2.48 (s, 3H), 2.22 (dd, <i>J</i> = 13.1, 7.6 Hz, 1H), 2.13–2.06 (m, 1H), 1.05 (s, 9H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>26</sub>H<sub>35</sub>N<sub>4</sub>O<sub>7</sub>S, 547.2221; found, 547.2230.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 3-(2-(3-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-Hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl) carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-3-oxopropoxy)ethoxy) propanoic Acid (<b>5</b>)</h4><div class="NLM_p last">(680 mg, yield 64%) as white solid. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 8.98 (d, <i>J</i> = 20.1 Hz, 1H), 7.48 (d, <i>J</i> = 8.0 Hz, 2H), 7.43 (d, <i>J</i> = 8.1 Hz, 2H), 4.64 (s, 1H), 4.59–4.51 (m, 2H), 4.49 (s, 1H), 4.35 (d, <i>J</i> = 15.5 Hz, 1H), 3.89 (d, <i>J</i> = 11.0 Hz, 1H), 3.80 (dd, <i>J</i> = 10.9, 3.8 Hz, 1H), 3.76–3.67 (m, 4H), 3.63–3.55 (m, 4H), 2.60–2.43 (m, 7H), 2.21 (dd, <i>J</i> = 13.1, 7.6 Hz, 1H), 2.08 (ddd, <i>J</i> = 13.2, 9.1, 4.5 Hz, 1H), 1.04 (s, 9H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>43</sub>N<sub>4</sub>O<sub>8</sub>S, 619.2796; found, 619.2800.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> (<i>S</i>)-15-((2<i>S</i>,4<i>R</i>)-4-Hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl) pyrrolidine-1-carbonyl)-16,16-dimethyl-13-oxo-4,7,10-trioxa-14-azaheptadecanoic Acid (<b>6</b>)</h4><div class="NLM_p last">(677 mg, yield 57%) as white solid. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 8.95 (s, 1H), 7.47 (d, <i>J</i> = 8.1 Hz, 2H), 7.42 (d, <i>J</i> = 8.1 Hz, 2H), 4.65 (s, 1H), 4.59–4.51 (m, 2H), 4.49 (s, 1H), 4.35 (d, <i>J</i> = 15.5 Hz, 1H), 3.89 (d, <i>J</i> = 11.1 Hz, 1H), 3.80 (dd, <i>J</i> = 10.9, 3.9 Hz, 1H), 3.76–3.67 (m, 4H), 3.66–3.54 (m, 8H), 2.60–2.50 (m, 3H), 2.50–2.43 (m, 4H), 2.21 (dd, <i>J</i> = 13.1, 7.6 Hz, 1H), 2.13–2.08 (m, 1H), 1.04 (s, 9H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>32</sub>H<sub>47</sub>N<sub>4</sub>O<sub>9</sub>S, 663.3058; found, 663.3059.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 11-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-Hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl) pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-11-oxoundecanoic Acid (<b>7</b>)</h4><div class="NLM_p last">(930 mg, yield 78%) as white solid. <sup>1</sup>H NMR (600 MHz CD<sub>3</sub>OD): δ 8.95 (s, 1H), 7.49 (d, <i>J</i> = 8.4 Hz, 2H), 7.44 (d, <i>J</i> = 7.8 Hz, 2H), 4.66 (s, 1H), 4.61–4.50 (m, 3H), 4.38 (d, <i>J</i> = 15.6 Hz, 1H), 3.93 (d, <i>J</i> = 9.6 Hz, 1H), 3.82 (dd, <i>J</i> = 11.4, 3.6 Hz, 1H), 2.50 (s, 3H), 2.35–2.21 (m, 5H), 2.12–2.07 (m, 1H), 1.66–1.57 (m, 4H),1.37–1.29 (m, 10H), 1.06 (s, 9H). HRMS (ESI-TOF) calcd for C<sub>33</sub>H<sub>49</sub>N<sub>4</sub>O<sub>6</sub>S, 629.3367; found, 629.3368.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> (<i>S</i>)-19-((2<i>S</i>,4<i>R</i>)-4-Hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl) pyrrolidine-1-carbonyl)-20,20-dimethyl-17-oxo-3,6,9,12,15-pentaoxa-18-azahenicosanoic Acid (<b>8</b>)</h4><div class="NLM_p last">(496 mg, yield 54%) as white solid. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 8.89 (s, 1H), 7.47 (d, <i>J</i> = 8.1 Hz, 2H), 7.42 (d, <i>J</i> = 8.1 Hz, 2H), 4.69 (s, 1H), 4.59–4.46 (m, 3H), 4.36 (d, <i>J</i> = 15.5 Hz, 1H), 4.16–4.00 (m, 4H), 3.87 (d, <i>J</i> = 11.0 Hz, 1H), 3.80 (dd, <i>J</i> = 11.0, 3.7 Hz, 1H), 3.76–3.53 (m, 16H), 2.48 (s, 3H), 2.22 (dd, <i>J</i> = 13.1, 7.6 Hz, 1H), 2.12–2.07 (m, 1H), 1.04 (s, 9H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>34</sub>H<sub>51</sub>N<sub>4</sub>O<sub>11</sub>S, 723.3270; found, 723.3269.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> <i>N</i>-((<i>S</i>)-3-(3,4-Dihydroisoquinolin-2(1<i>H</i>)-yl)-2-hydroxypropyl)-6-((1-(2-(2-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-2-oxoethoxy)acetyl)azetidin-3-yl)amino)pyrimidine-4-carboxamide (<b>9</b>)</h4><div class="NLM_p last">To a solution of <b>3</b> (10 mg, 0.016 mmol), <b>4</b> (10 mg, 0.02 mmol), EDCI (6 mg, 0.03 mmol), and HOAt (4 mg, 0.03 mmol) in DMSO (1 mL) was added NMM (15 mg, 0.15 mmol). After stirring overnight at room temperature, the resulting mixture was purified by preparative HPLC (10–100% methanol/0.1% TFA in H<sub>2</sub>O) to afford <b>9</b> as a white solid (6 mg, yield 36%). <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.96 (s, 1H), 8.52 (s, 1H), 7.48–7.41 (m, 4H), 7.34–7.16 (m, 5H), 4.71–4.47 (m, 4H), 4.44–4.33 (m, 4H), 4.25–4.05 (m, 9H), 3.98–3.95 (m, 1H), 3.91–3.76 (m, 5H), 3.57–3.49 (m, 2H), 3.44–3.35 (m, 2H), 2.47 (s, 3H), 2.26–2.21 (m, 1H), 2.12–2.06 (m, 1H), 1.04 (s, 9H). <sup>13</sup>C NMR (151 MHz, MeOD): δ 172.96, 170.44, 169.89, 169.63, 157.07, 151.71, 147.17, 146.45, 139.01, 130.65, 129.87, 128.98, 128.50, 128.11, 127.60, 126.91, 126.45, 69.91, 69.68, 68.55, 64.24, 59.47, 57.47, 57.07, 56.73, 54.98, 43.16, 42.31, 41.75, 39.09, 37.55, 36.26, 35.65, 35.59, 25.57, 24.53, 14.22. HPLC 95% pure, <i>t</i><sub>R</sub> = 3.97 min; HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> for C<sub>46</sub>H<sub>59</sub>N<sub>10</sub>O<sub>8</sub>S<sup>+</sup>, 911.4233; found, 911.4233.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> <i>N</i>-((<i>S</i>)-3-(3,4-Dihydroisoquinolin-2(1<i>H</i>)-yl)-2-hydroxypropyl)-6-((1-(3-(2-(3-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-3-oxopropoxy)ethoxy)propanoyl)azetidin-3-yl)amino)pyrimidine-4-carboxamide (<b>10</b>)</h4><div class="NLM_p last">To a solution of <b>3</b> (10 mg, 0.016 mmol), <b>5</b> (10 mg, 0.02 mmol), EDCI (6 mg, 0.03 mmol), and HOAt (4 mg, 0.03 mmol) in DMSO (1 mL) was added NMM (15 mg, 0.15 mmol). After stirring overnight at rt, the resulting mixture was purified by preparative HPLC (10–100% methanol/0.1% TFA in H<sub>2</sub>O) to afford <b>10</b> as a white solid (8 mg, yield 48%). <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.95 (s, 1H), 8.54 (s, 1H), 7.49–7.34 (m, 4H), 7.33–7.14 (m, 5H), 4.79–4.74 (m, 1H), 4.66–4.44 (m, 7H), 4.44–4.28 (m, 5H), 4.16–4.10 (m, 1H), 3.90–3.77 (m, 5H), 3.75–3.67 (m, 4H), 3.61–3.49 (m, 6H), 3.44–3.37 (m, 2H), 2.56–2.38 (m, 7H), 2.25–2.20 (m, 1H), 2.11–2.03 (m, 1H), 1.03 (s, 9H). <sup>13</sup>C NMR (151 MHz, MeOD): δ 173.11, 172.51, 172.35, 170.73, 156.98, 151.75, 147.12, 139.07, 132.33, 130.65, 129.84, 128.98, 128.50, 128.11, 127.63, 126.91, 126.45, 70.12, 70.10, 70.02, 69.69, 66.91, 66.68, 59.46, 57.53, 57.29, 56.62, 54.26, 43.17, 42.29, 42.18, 40.56, 37.57, 35.98, 35.39, 31.90, 25.59, 24.53, 14.21. HPLC 95% pure, <i>t</i><sub>R</sub> = 3.98 min; HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> for C<sub>50</sub>H<sub>67</sub>N<sub>10</sub>O<sub>9</sub>S<sup>+</sup>, 983.4808; found, 983.4795.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> <i>N</i>-((<i>S</i>)-3-(3,4-Dihydroisoquinolin-2(1<i>H</i>)-yl)-2-hydroxypropyl)-6-((1-((<i>S</i>)-15-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidine-1-carbonyl)-16,16-dimethyl-13-oxo-4,7,10-trioxa-14-azaheptadecanoyl)azetidin-3-yl)amino)pyrimidine-4-carboxamide (<b>11</b>)</h4><div class="NLM_p last">To a solution of <b>3</b> (10 mg, 0.016 mmol), <b>6</b> (10 mg, 0.016 mmol), EDCI (6 mg, 0.03 mmol), and HOAt (4 mg, 0.03 mmol) in DMSO (1 mL) was added NMM (15 mg, 0.15 mmol). After stirring overnight at rt, the resulting mixture was purified by preparative HPLC (10–100% methanol/0.1% TFA in H<sub>2</sub>O) to afford <b>11</b> as a white solid (10 mg, yield 60%). <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.96 (s, 1H), 8.55 (s, 1H), 7.50–7.39 (m, 4H), 7.33–7.15 (m, 5H), 4.81–4.76 (m, 1H), 4.66–4.48 (m, 7H), 4.44–4.31 (m, 5H), 4.17–4.14 (m, 1H), 3.91–3.78 (m, 5H), 3.73–3.69 (m, 4H), 3.64–3.48 (m, 10H), 3.41–3.35 (m, 2H), 2.58–2.38 (m, 7H), 2.25–2.20 (m, 1H), 2.11–2.07 (m, 1H), 1.03 (s, 9H). <sup>13</sup>C NMR (151 MHz, MeOD): δ 173.09, 172.52, 172.34, 170.74, 151.78, 147.10, 139.09, 132.36, 130.66, 129.82, 128.97, 128.50, 128.11, 127.63, 126.91, 126.46, 70.13, 70.10, 70.06, 70.04, 70.02, 69.96, 69.82, 69.69, 69.68, 66.88, 66.61, 59.45, 57.56, 57.20, 56.60, 54.29, 43.20, 42.90, 42.29, 40.60, 39.03, 37.57, 36.09, 35.94, 35.87, 35.39, 31.88, 25.65, 25.64, 25.60, 24.53, 14.20. HPLC 95% pure, <i>t</i><sub>R</sub> = 4.00 min; HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> for C<sub>52</sub>H<sub>71</sub>N<sub>10</sub>O<sub>10</sub>S<sup>+</sup>, 1027.5070; found, 127.5066.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> <i>N</i>-((<i>S</i>)-3-(3,4-Dihydroisoquinolin-2(1<i>H</i>)-yl)-2-hydroxypropyl)-6-((1-(11-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-11-oxoundecanoyl)azetidin-3-yl)amino)pyrimidine-4-carboxamide (<b>12</b>)</h4><div class="NLM_p last">To a solution of <b>3</b> (20 mg, 0.032 mmol), <b>7</b> (16 mg, 0.025 mmol), EDCI (12 mg, 0.06 mmol), and HOAt (8 mg, 0.06 mmol) in DMSO (1 mL) was added NMM (30 mg, 0.30 mmol). After stirring overnight at rt, the resulting mixture was purified by preparative HPLC (10–100% methanol/0.1% TFA in H<sub>2</sub>O) to afford <b>12</b> as a white solid (11 mg, yield 39%). <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.92 (s, 1H), 8.53 (s, 1H), 7.49–7.39 (m, 4H), 7.34–7.18 (m, 5H), 4.79–4.73 (m, 1H), 4.65–4.48 (m, 7H), 4.43–4.32 (m, 5H), 4.09–4.05 (m, 1H), 3.92–3.79 (m, 5H), 3.55–3.50 (m, 2H), 3.42–3.37 (m, 2H), 2.47 (s, 3H), 2.26–2.08 (m, 6H), 1.59 (s, 4H), 1.32 (s, 10H), 1.03 (s, 9H). <sup>13</sup>C NMR (151 MHz, MeOD): δ 174.66, 174.56, 173.08, 170.95, 151.68, 139.04, 130.65, 129.90, 128.97, 128.50, 128.11, 127.61, 126.92, 126.45, 69.68, 59.43, 57.55, 57.10, 56.62, 54.29, 43.14, 42.29, 40.53, 37.53, 35.25, 35.18, 30.87, 29.00, 28.94, 28.89, 28.84, 25.63, 25.59, 25.59, 24.54, 24.52, 14.23. HPLC 95% pure, <i>t</i><sub>R</sub> = 4.00 min; HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> for C<sub>53</sub>H<sub>73</sub>N<sub>10</sub>O<sub>7</sub>S<sup>+</sup>, 993.5379; found, 993.5378.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <i>N</i>-((<i>S</i>)-3-(3,4-Dihydroisoquinolin-2(1<i>H</i>)-yl)-2-hydroxypropyl)-6-((1-((<i>S</i>)-19-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidine-1-carbonyl)-20,20-dimethyl-17-oxo-3,6,9,12,15-pentaoxa-18-azahenicosanoyl)azetidin-3-yl)amino)pyrimidine-4-carboxamide (<b>13</b>)</h4><div class="NLM_p last">To a solution of <b>3</b> (20 mg, 0.032 mmol), <b>8</b> (16 mg, 0.02 mmol), EDCI (6 mg, 0.03 mmol), and HOAt (4 mg, 0.03 mmol) in DMSO (1 mL) was added NMM (30 mg, 0.30 mmol). After stirring overnight at rt, the resulting mixture was purified by preparative HPLC (10–100% methanol/0.1% TFA in H<sub>2</sub>O) to afford <b>13</b> as a white solid (10 mg, yield 38%). <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.90 (s, 1H), 8.52 (s, 1H), 7.48–7.41 (m, 4H), 7.35–7.16 (m, 5H), 4.81–4.66 (m, 6H), 4.62–4.46 (m, 6H), 4.42–4.32 (m, 5H), 4.25–4.21 (m, 1H), 4.10–4.02 (m, 4H), 3.89–3.67 (m, 4H), 3.70–3.43 (m, 17H), 2.47 (s, 3H), 2.26–2.21 (m, 1H), 2.08 (t, <i>J</i> = 11.2 Hz, 1H), 1.03 (s, 9H). <sup>13</sup>C NMR (151 MHz, MeOD): δ 172.99, 170.70, 170.50, 170.30, 157.14, 151.71, 147.22, 139.02, 132.26, 130.66, 129.90, 129.13, 128.99, 128.50, 128.11, 127.61, 126.91, 126.45, 70.84, 70.32, 70.21, 70.11, 70.04, 70.02, 69.67, 69.23, 59.43, 57.93, 56.77, 56.72, 54.97, 43.15, 42.31, 41.78, 37.60, 35.70, 25.58, 25.55, 24.54, 14.26. HPLC 95% pure, <i>t</i><sub>R</sub> = 3.78 min; HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> for C<sub>54</sub>H<sub>75</sub>N<sub>10</sub>O<sub>12</sub>S, 1087.5281; found, 1087.5261.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> (<i>S</i>)-19-((2<i>S</i>,4<i>R</i>)-4-Hydroxy-2-(((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)pyrrolidine-1-carbonyl)-20,20-dimethyl-17-oxo-3,6,9,12,15-pentaoxa-18-azahenicosanoic Acid (<b>14</b>)</h4><div class="NLM_p last">To a solution of (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-amino-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (VHL-2) (33.6 mg, 0.06 mmol), 3,6,9,12,15-pentaoxaheptadecanedioic acid (31.03 mg, 0.1 mmol), EDCI (12 mg, 0.06 mmol), and HOAt ((8 mg, 0.06 mmol) in DMSO (2 mL) was added NMM (30 mg, 0.3 mmol). After being stirred overnight at rt, the resulting mixture was purified by preparative HPLC (10–100% methanol/0.1% TFA in H<sub>2</sub>O) to afford <b>14</b> as an oil (22 mg, yield 50%). <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.25 (s, 1H), 7.51–7.38 (m, 4H), 5.04–4.97 (m, 2H), 4.68 (s, 1H), 4.60–4.54 (m, 1H), 4.44 (s, 1H), 4.13 (s, 1H), 4.06–4.04 (m, 1H), 3.85 (d, <i>J</i> = 11.1 Hz, 1H), 3.76–3.61 (m, 17H), 3.35–3.30 (m, 1H), 2.52 (s, 3H), 2.24–2.20 (m, 1H), 1.99–1.93 (m, 1H), 1.51 (d, <i>J</i> = 7.0 Hz, 3H), 1.05 (s, 9H). MS (ESI) <i>m</i>/<i>z</i>: 737.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> <i>N</i>-((<i>S</i>)-3-(3,4-Dihydroisoquinolin-2(1<i>H</i>)-yl)-2-hydroxypropyl)-6-((1-((<i>S</i>)-19-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-(((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)pyrrolidine-1-carbonyl)-20,20-dimethyl-17-oxo-3,6,9,12,15-pentaoxa-18-azahenicosanoyl)azetidin-3-yl)amino)pyrimidine-4-carboxamide (<b>15</b>)</h4><div class="NLM_p last">To a solution of <b>3</b> (10 mg, 0.016 mmol), <b>14</b> (10 mg, 0.014 mmol), EDCI (6 mg, 0.03 mmol), and HOAt (4 mg, 0.03 mmol) in DMSO (1 mL) was added NMM (15 mg, 0.15 mmol). After being stirred overnight at rt, the resulting mixture was purified by preparative HPLC (10–100% methanol/0.1% TFA in H<sub>2</sub>O) to afford <b>15</b> as a white solid (6 mg, yield 35%). <sup>1</sup>H NMR (800 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.03 (s, 1H), 8.59 (s, 1H), 7.50–7.43 (m, 4H), 7.34 (t, <i>J</i> = 7.4 Hz, 1H), 7.31–7.27 (m, 2H), 7.26–7.20 (m, 2H), 5.07–5.01 (m, 1H), 4.75 (t, <i>J</i> = 8.8 Hz, 1H), 4.70 (s, 1H), 4.68–4.57 (m, 2H), 4.49–4.34 (m, 4H), 4.30–4.23 (m, 1H), 4.18–4.11 (m, 2H), 4.11–4.03 (m, 2H), 4.00–3.95 (m, 1H), 3.87 (d, <i>J</i> = 11.0 Hz, 2H), 3.78–3.61 (m, 18H), 3.61–3.39 (m, 5H), 3.30–3.11 (m, 2H), 2.51 (s, 3H), 2.24 (dd, <i>J</i> = 13.2, 7.6 Hz, 1H), 2.04–1.92 (m, 1H), 1.53 (d, <i>J</i> = 7.2 Hz, 3H), 1.07 (s, 9H). <sup>13</sup>C NMR (201 MHz, MeOD): δ 171.76, 170.68, 170.51, 170.29, 164.14, 163.01, 156.76, 151.88, 146.93, 144.48, 132.41, 130.66, 129.75, 129.11, 128.50, 128.12, 126.92, 126.45, 126.30, 126.10, 104.05, 70.81, 70.32, 70.20, 70.10, 70.04, 69.67, 69.57, 69.17, 64.26, 59.23, 57.81, 56.83, 56.69, 54.96, 48.77, 42.84, 41.88, 37.46, 35.66, 25.60, 24.53, 21.00, 14.08. HPLC 95% pure, <i>t</i><sub>R</sub> = 3.83 min; HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> for C<sub>55</sub>H<sub>77</sub>N<sub>10</sub>O<sub>12</sub>S<sup>+</sup>, 1101.5438; found, 1101.5427.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (<i>S</i>)-19-((2<i>R</i>,4<i>S</i>)-4-Hydroxy-2-(((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl) pyrrolidine-1-carbonyl)-20,20-dimethyl-17-oxo-3,6,9,12,15-pentaoxa-18-azahenicosanoic Acid (<b>16</b>)</h4><div class="NLM_p last">To a solution of (2<i>R</i>,4<i>S</i>)-1-((<i>S</i>)-2-amino-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (100 mg, 0.2 mmol), 3,6,9,12,15-pentaoxaheptadecanedioic acid (100 mg, 0.3 mmol), EDCI (76 mg, 0.4 mmol), and HOAt (53 mg, 0.4 mmol) in DMSO (2 mL) was added NMM (101 mg, 1 mmol). After being stirred overnight at rt, the resulting mixture was purified by preparative HPLC (10–100% methanol/0.1% TFA in H<sub>2</sub>O) to afford <b>16</b> as an oil (66 mg, yield 50%). <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.34 (s, 1H), 7.58 (d, <i>J</i> = 8.2 Hz, 2H), 7.52 (d, <i>J</i> = 8.2 Hz, 2H), 5.04 (q, <i>J</i> = 7.1 Hz, 1H), 4.66–4.62 (m, 1H), 4.59 (t, <i>J</i> = 7.5 Hz, 1H), 4.53–4.46 (m, 1H), 4.23–3.98 (m, 4H), 3.97–3.92 (m, 1H), 3.79–3.55 (m, 17H), 2.57 (s, 3H), 2.29–2.19 (m, 1H), 2.17–2.09 (m, 1H), 1.48 (d, <i>J</i> = 7.1 Hz, 3H), 1.08 (s, 9H). MS (ESI) <i>m</i>/z: 737.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> <i>N</i>-((<i>S</i>)-3-(3,4-Dihydroisoquinolin-2(1<i>H</i>)-yl)-2-hydroxypropyl)-6-((1-((<i>S</i>)-19-((2<i>R</i>,4<i>S</i>)-4-hydroxy-2-(((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)pyrrolidine-1-carbonyl)-20,20-dimethyl-17-oxo-3,6,9,12,15-pentaoxa-18-azahenicosanoyl)azetidin-3-yl)amino)pyrimidine-4-carboxamide (<b>17</b>)</h4><div class="NLM_p last">To a solution of <b>3</b> (10 mg, 0.016 mmol), <b>16</b> (10 mg, 0.014 mmol), EDCI (6 mg, 0.03 mmol), and HOAt (4 mg, 0.03 mmol) in DMSO (1 mL) was added NMM (15 mg, 0.15 mmol). After being stirred overnight at rt, the resulting mixture was purified by preparative HPLC (10–100% methanol/0.1% TFA in H<sub>2</sub>O) to afford <b>17</b> as a white solid (9 mg, yield 53%). <sup>1</sup>H NMR (800 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.13 (s, 1H), 8.58 (d, <i>J</i> = 9.5 Hz, 1H), 7.59–7.41 (m, 4H), 7.38–7.26 (m, 3H), 7.25–7.14 (m, 2H), 5.07–5.01 (m, 1H), 4.79–4.72 (m, 1H), 4.71–4.56 (m, 3H), 4.55–4.35 (m, 4H), 4.33–4.24 (m, 2H), 4.13 (s, 2H), 4.06–3.84 (m, 5H), 3.78–3.40 (m, 23H), 3.33–3.08 (m, 1H), 2.54 (s, 3H), 2.30–2.19 (m, 1H), 2.19–2.10 (m, 1H), 1.48 (d, <i>J</i> = 6.8 Hz, 3H), 1.09 (s, 9H). <sup>13</sup>C NMR (201 MHz, MeOD): δ 172.14, 171.01, 170.52, 170.21, 163.07, 159.78, 151.75, 146.63, 144.43, 132.62, 130.69, 129.44, 129.29, 129.02, 128.51, 128.11, 126.91, 126.47, 104.13, 70.77, 70.36, 70.16, 70.13, 70.05, 69.98, 69.94, 69.66, 69.28, 69.19, 64.23, 59.39, 57.80, 57.19, 55.66, 54.81, 48.59, 43.38, 42.04, 37.66, 34.74, 25.65, 25.45, 24.53, 21.22, 14.13. HPLC 95% pure, <i>t</i><sub>R</sub> = 3.78 min; HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> for C<sub>55</sub>H<sub>77</sub>N<sub>10</sub>O<sub>12</sub>S<sup>+</sup>, 1101.5438; found, 1101.5425.</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> <i>tert</i>-Butyl (<i>S</i>)-3-((6-((2-hydroxy-3-(piperidin-1-yl)propyl)carbamoyl)pyrimidin-4-yl)amino)azetidine-1-carboxylate (<b>19</b>)</h4><div class="NLM_p last">Intermediate <b>19</b> (220 mg, yield 13% over four steps) was prepared according to the same procedure used for preparing intermediate <b>2</b> from piperidine (255 mg, 3 mmol) and <i>tert</i>-butyl (<i>S</i>)-(oxiran-2-ylmethyl)carbamate (519 mg, 3 mmol). <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.57 (s, 1H), 7.22 (s, 1H), 4.50–4.13 (m, 4H), 3.89–3.81 (m, 2H), 3.66–3.39 (m, 4H), 3.29–2.89 (m, 4H), 2.12–1.70 (m, 5H), 1.67–1.23 (m, 10H). MS (ESI) <i>m</i>/z: 435.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> (<i>S</i>)-6-(Azetidin-3-ylamino)-<i>N</i>-(2-hydroxy-3-(piperidin-1-yl)propyl)pyrimidine-4-carboxamide (<b>20</b>)</h4><div class="NLM_p last">A solution of <b>19</b> (220 mg, 0.4 mmol) in DCM (2 mL) was treated with TFA (2 mL) for 30 min. The volatiles were removed under reduced pressure to give compound <b>20</b> (260 mg, yield 100%). <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.91 (s, 1H), 7.66 (s, 1H), 5.06–4.95 (m, 1H), 4.72 (dd, <i>J</i> = 12.9, 6.9 Hz, 1H), 4.43–4.36 (m, 1H), 4.24 (s, 2H), 3.64–3.37 (m, 4H), 3.25–3.18 (m, 4H), 3.12–2.99 (m, 2H), 2.98–2.88 (m, 1H), 1.98–1.63 (m, 4H), 1.61–1.42 (m, 1H). MS (ESI) <i>m</i>/<i>z</i>: 335.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 6-((1-((<i>S</i>)-19-((2<i>S</i>,4<i>R</i>)-4-Hydroxy-2-(((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)pyrrolidine-1-carbonyl)-20,20-dimethyl-17-oxo-3,6,9,12,15-pentaoxa-18-azahenicosanoyl)azetidin-3-yl)amino)-<i>N</i>-((<i>S</i>)-2-hydroxy-3-(piperidin-1-yl)propyl)pyrimidine-4-carboxamide (<b>21</b>)</h4><div class="NLM_p last">To a solution of <b>20</b> (10 mg, 0.02 mmol), <b>14</b> (10 mg, 0.014 mmol), EDCI (6 mg, 0.03 mmol), and HOAt (4 mg, 0.03 mmol) in DMSO (1 mL) was added NMM (15 mg, 0.15 mmol). After being stirred overnight at rt, the resulting mixture was purified by preparative HPLC (10–100% methanol/0.1% TFA in H<sub>2</sub>O) to afford <b>21</b> as a white solid (8 mg, 49%). <sup>1</sup>H NMR (800 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.08 (s, 1H), 8.60 (s, 1H), 7.54–7.42 (m, 4H), 7.25 (s, 1H), 5.06–5.01 (m, 1H), 4.78–4.72 (m, 1H), 4.71 (s, 1H), 4.65–4.56 (m, 1H), 4.51–4.39 (m, 2H), 4.34–4.24 (m, 2H), 4.19–4.12 (m, 2H), 4.11–4.03 (m, 2H), 4.02–3.94 (m, 1H), 3.87 (d, <i>J</i> = 11.0 Hz, 1H), 3.83–3.45 (m, 23H), 3.27–3.20 (m, 1H), 3.12 (dd, <i>J</i> = 13.3, 10.5 Hz, 1H), 3.05 (td, <i>J</i> = 12.4, 3.5 Hz, 1H), 2.98 (td, <i>J</i> = 12.3, 3.2 Hz, 1H), 2.52 (s, 3H), 2.24 (dd, <i>J</i> = 13.3, 7.7 Hz, 1H), 2.02–1.75 (m, 6H), 1.54 (d, <i>J</i> = 7.1 Hz, 3H), 1.07 (s, 9H). <sup>13</sup>C NMR (201 MHz, methanol-<i>d</i><sub>4</sub>): δ 171.77, 170.68, 170.53, 170.29, 163.81, 163.05, 160.20, 159.43, 152.02, 151.44, 146.75, 144.56, 132.58, 129.63, 129.12, 126.32, 126.12, 104.07, 70.85, 70.35, 70.27, 70.23, 70.15, 70.13, 70.07, 70.05, 69.69, 69.57, 69.25, 63.91, 59.34, 59.23, 57.80, 56.82, 56.70, 54.93, 54.80, 51.89, 48.78, 43.30, 41.95, 37.44, 35.66, 25.60, 22.42, 21.24, 21.00, 14.00. HPLC 95% pure, <i>t</i><sub>R</sub> = 3.76 min; HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> for C<sub>51</sub>H<sub>77</sub>N<sub>10</sub>O<sub>12</sub>S<sup>+</sup>, 1053.5438; found, 1053.5429.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Cell Culture and Cell Viability Assay</h3><div class="NLM_p last">MCF-7, HeLa, A549, and A172 cells were cultured in Dulbecco’s modified Eagle medium, supplemented with 10% fetal bovine serum (FBS), in 5% CO<sub>2</sub> at 37 °C. Jurkat cells were cultured in RPMI 1640 medium, supplemented with 10% FBS, in 5% CO<sub>2</sub> at 37 °C. A certain number of different cells were seeded and then treated with the indicated compounds for certain days (the medium plus compounds was replaced every other day). The cells were then harvested using Promega CellTiter-Glo (Promega) to count the cell titer, according to the manufacturer’s instructions. The ratio of the cell titer of the experiment group to that of the control group (DMSO-treated) was calculated as the normalized viability.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Western Blot and Antibodies</h3><div class="NLM_p last">Western blotting was performed with a standard protocol. Briefly, cells were harvested and washed with ice-cold phosphate-buffered saline (PBS) and lysed with radioimmunoprecipitation assay buffer (50 mM Tris-HCl, pH 7.5, 150 mM NaCl, 1% NP-40, 0.1% sodium dodecyl sulfate, 1% sodium deoxycholate, 5 mM ethylenediaminetetraacetic acid), supplemented with protease inhibitor cocktail (Roche). After being loaded for polyacrylamide gel electrophoresis, the total proteins were transferred to the polyvinylidene difluoride membrane and sequentially incubated in 5% fat-free milk in PBST (PBS + 0.05% Tween 20), primary antibody diluted in PBST, and secondary antibody diluted in PBST. Then, the membrane was incubated with enhanced chemiluminescent reagents and developed with X-ray films. The antibodies used in this study included anti-PRMT5 (active Motif), anti-SDMA (Cell Signaling Technologies), and anti-β-actin (Sigma).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> PRMT5/MEP50 Methyltransferase Inhibition Assay</h3><div class="NLM_p last">In this biochemical assay (FlashPlate), which monitors the transfer of a <sup>3</sup>H-labeled methyl group from the cofactor SAM to the substrate H4 (<b>1–15</b>), human PRMT5/MEP50 was used as the enzyme (5 nM). The concentration of the substrate core histone was 0.04 μM, and the concentration of the cofactor SAM was 1  μM. Four compounds were tested in a 10-concentration IC<sub>50</sub> mode with threefold serial dilution, in duplicate, starting at 5 μM. Compound pre-incubation was performed for 20 min at rt in a mixture of enzyme and substrate.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> MS Assay</h3><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> Sample Preparation</h4><div class="NLM_p last">MCF7 cells were treated by compound <b>15</b> and DMSO, respectively, at a concentration of 5 μM for 5 d. The medium was replaced by a fresh medium with compound <b>15</b> and DMSO every 2 d (48 h). Each proteomics experiment result was obtained from two biological replicates, and each sample subjected to LS–MS was validated by repetitive western blot experiments with two different PRMT5 antibodies (Santacruz; A-11; CST 2252S). The harvested cells were lysed in a lysis buffer (8 M urea, 50 mM Tris-HCl pH 8.0). Sonication (5 s on 5 s off, 2 × 30 s) was performed to shear genomic DNA. The lysates were centrifugated for 30 min at 10,000<i>g</i> at 4 °C, and the supernatant was transferred to a clean tube. The protein concentration was determined ( bicinchoninic acid assay), and the protein was reduced with 5 mM dithiothreitol, alkylated with 15 mM iodoacetamide in the dark, and then diluted with 3× volume of 25 mM Tris (pH 8.0) buffer and 1 mM CaCl<sub>2</sub>. The final urea concentration is 2 M. Trypsin was added into the protein solution with 1:100 ratio (trypsin/protein) and digested in 12–16 h or overnight at rt.</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> MS Analysis</h4><div class="NLM_p last">Peptides were cleaned up by C18 stage tips, and the concentration was determined (peptide assay, Thermo 23275). The cleaned peptides were dissolved in 0.1% formic acid and analyzed on a Q-Exactive HF-X system coupled with an Easy nanoLC 1200 (Thermo Fisher Scientific, San Jose, CA). Peptides (0.5 μg) were loaded on to an Acclain PepMap RSLC C18 column (250 mm × 75 μm ID, C18, 2 μm, Thermo Fisher). The analytical separation of all peptides was achieved with a 130 min gradient. A linear gradient of 5–30% buffer B over 110 min was executed at a 300 nL/min flow rate followed by a ramp to 100% B in 5 min, and 15 min wash with 100% B, where buffer A was aqueous 0.1% formic acid and buffer B was 80% acetonitrile and 0.1% formic acid. LC–MS experiments were performed in a data-dependent mode with full MS (externally calibrated to a mass accuracy of <5 ppm and a resolution of 60,000 at <i>m</i>/<i>z</i> 200), followed by high-energy collision-activated dissociation-tandem mass spectrometry (MS/MS) of the top 20 most intense ions with a resolution of 15,000 at <i>m</i>/<i>z</i> 200. High-energy collision-activated dissociation-MS/MS was used to dissociate peptides at a normalized collision energy of 27 eV in the presence of nitrogen bath gas atoms. Dynamic exclusion was 30.0 s. There were two biological replicates for one treatment, and each sample was subjected to two technical LC–MS replicates.</div></div><div id="sec4_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> MS Data Analysis</h4><div class="NLM_p last">Mass spectra processing and peptide identification were performed on the Andromeda search engine in MaxQuant software (version 1.6.10.43) against a human UniProt database (UP000005640). All searches were conducted with a defined modification of cysteine carbamidomethylation, with methionine oxidation and protein amino-terminal acetylation as dynamic modifications. Peptides were confidently identified using a target-decoy approach, with a peptide false discovery rate (FDR) of 1% and a protein FDR of 1%. A minimum peptide length of 7 amino acids was required; maximally two missed cleavages were allowed; the initial mass deviation for the precursor ion was up to 7 ppm, and the maximum allowed mass deviation for fragment ions was 0.5 Da. Data processing and statistical analysis were performed on Perseus (version 1.6.10.50). Protein quantitation was performed on biological replicate runs, and a two-sample <i>t</i> test statistics was used to report statistically significant expression fold changes.</div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> PK Assay</h3><div class="NLM_p last">Six male Swiss albino mice were administered intraperitoneally, with <b>15</b> formulated in normal saline at a single 150 mg/kg dose. Blood samples (approximately 60 μL) were collected under light isoflurane anesthesia at 0.25, 0.5, 1, 2, 6, and 12 h. At each time point, blood samples were collected from three mice. Immediately after collection, plasma was harvested by centrifugation and stored at −70 °C until analysis. All samples were processed for analysis by protein precipitation using acetonitrile and analyzed with the fit-for-purpose LC–MS/MS method (LLOQ = 9.25 ng/mL for plasma).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i45"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01111" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43818" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43818" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01111?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01111</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline"><sup>1</sup>H NMR, <sup>13</sup>C NMR, and HPLC spectra of compounds <b>15</b>, <b>17</b>, and <b>21</b>; raw data of compounds <b>15</b>, <b>17</b>, <b>21</b>, and EPZ015666 inhibiting the PRMT5 methyltransferase activity in the PRMT5 biochemical assay; raw data of compounds <b>15</b>, <b>17</b>, <b>21</b>, and EPZ015666 in cell viability assays (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01111/suppl_file/jm0c01111_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and biological data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01111/suppl_file/jm0c01111_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01111/suppl_file/jm0c01111_si_001.pdf">jm0c01111_si_001.pdf (538.14 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01111/suppl_file/jm0c01111_si_002.csv">jm0c01111_si_002.csv (3.35 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01111" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33374" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33374" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Liu</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4740-243X" title="Orcid link">http://orcid.org/0000-0003-4740-243X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#5e343730397032372b1e332d2d33703b3a2b"><span class="__cf_email__" data-cfemail="83e9eaede4adefeaf6c3eef0f0eeade6e7f6">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mark T. Bedford</span> - <span class="hlFld-Affiliation affiliation">Department
of Epigenetics and Molecular Carcinogenesis, The University of Texas MD Anderson Cancer Center, Smithville, Texas 78957, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#87eaf3e5e2e3e1e8f5e3c7eae3e6e9e3e2f5f4e8e9a9e8f5e0"><span class="__cf_email__" data-cfemail="81ecf5e3e4e5e7eef3e5c1ece5e0efe5e4f3f2eeefafeef3e6">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Jin</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link">http://orcid.org/0000-0002-2387-3862</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#c4aeada5aaeaaeadaa84a9b7b7a9eaa1a0b1"><span class="__cf_email__" data-cfemail="eb81828a85c5818285ab86989886c58e8f9e">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yudao Shen</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guozhen Gao</span> - <span class="hlFld-Affiliation affiliation">Department
of Epigenetics and Molecular Carcinogenesis, The University of Texas MD Anderson Cancer Center, Smithville, Texas 78957, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xufen Yu</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Huensuk Kim</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Li Wang</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Biophysics, University
of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ling Xie</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Biophysics, University
of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Megan Schwarz</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xian Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Biophysics, University
of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ernesto Guccione</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Y.S. and G.G. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i47">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24377" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24377" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The research described here was supported in part by an endowed professorship (to J.J.) from the Icahn School of Medicine at Mount Sinai and the grants R01GM122749 (to J.J.) and R01GM126421 (to M.T.B.) from the U.S. National Institutes of Health.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">PRMT5</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase 5</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last">S-adenosylmethionine</p></td></tr><tr><td class="NLM_term">MTA</td><td class="NLM_def"><p class="first last">methylthioadenosine</p></td></tr><tr><td class="NLM_term">MTAP</td><td class="NLM_def"><p class="first last">methylthioadenosine phosphorylase</p></td></tr><tr><td class="NLM_term">VHL</td><td class="NLM_def"><p class="first last">von Hippel-Lindau</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">VHL-2</td><td class="NLM_def"><p class="first last">(<i>S</i>,<i>R</i>,<i>S</i>)-AHPC-Me</p></td></tr><tr><td class="NLM_term">PEG</td><td class="NLM_def"><p class="first last">poly(ethylene glycol)</p></td></tr><tr><td class="NLM_term">SDMA</td><td class="NLM_def"><p class="first last">arginine symmetric dimethylation</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetic</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">EDCI</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3-dimethylaminopropyl)carbodiimide</p></td></tr><tr><td class="NLM_term">HOAt</td><td class="NLM_def"><p class="first last">1-hydroxy-7-aza-benzo-triazole</p></td></tr><tr><td class="NLM_term">NMM</td><td class="NLM_def"><p class="first last"><i>N</i>-methylmorpholine</p></td></tr><tr><td class="NLM_term">NMP</td><td class="NLM_def"><p class="first last"><i>N</i>-methyl-2-pyrrolidone</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">ESI</td><td class="NLM_def"><p class="first last">electrospray ionization</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i49">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75110" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75110" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 48 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Branscombe, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.-h.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pestka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S.</span></span> <span> </span><span class="NLM_article-title">PRMT5 (Janus kinase-binding protein 1) catalyzes the formation of symmetric dimethylarginine residues in proteins</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">32971</span>– <span class="NLM_lpage">32976</span>, <span class="refDoi"> DOI: 10.1074/jbc.m105412200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1074%2Fjbc.M105412200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=11413150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFCmtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=32971-32976&author=T.+L.+Branscombeauthor=A.+Frankelauthor=J.-H.+Leeauthor=J.+R.+Cookauthor=Z.-h.+Yangauthor=S.+Pestkaauthor=S.+Clarke&title=PRMT5+%28Janus+kinase-binding+protein+1%29+catalyzes+the+formation+of+symmetric+dimethylarginine+residues+in+proteins&doi=10.1074%2Fjbc.m105412200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT5 (Janus kinase-binding protein 1) catalyzes the formation of symmetric dimethylarginine residues in proteins</span></div><div class="casAuthors">Branscombe, Tina L.; Frankel, Adam; Lee, Jin-Hyung; Cook, Jeffry R.; Yang, Zhi-Hong; Pestka, Sidney; Clarke, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">32971-32976</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">We have identified a new mammalian protein arginine N-methyltransferase, PRMT5, formerly designated Janus kinase-binding protein 1, that can catalyze the formation of ω-NG-monomethylarginine and sym. ω-NG,NG'-dimethylarginine in a variety of proteins.  A hemagglutinin peptide-tagged PRMT5 complex purified from human HeLa cells catalyzes the S-adenosyl-L-[methyl-3H]methionine-dependent in vitro methylation of myelin basic protein.  When the radiolabeled myelin basic protein was acid-hydrolyzed to free amino acids, and the products were sepd. by high-resoln. cation exchange chromatog., we were able to detect two tritiated species.  One species co-migrated with a ω-NG-monomethylarginine std., and the other co-chromatographed with a sym. ω-NG,NG'-dimethylarginine std.  Upon base treatment, this second species formed methylamine, a breakdown product characteristic of sym. ω-NG,NG'-dimethylarginine.  Further anal. of these two species by thin layer chromatog. confirmed their identification as ω-NG-monomethylarginine and sym. ω-NG,NG'-dimethylarginine.  The hemagglutinin-PRMT5 complex was also able to monomethylate and sym. dimethylate bovine histone H2A and a glutathione S-transferase-fibrillarin (amino acids 1-148) fusion protein (glutathione S-transferase-GAR).  A mutation introduced into the S-adenosyl-L-methionine-binding motif I of a myc-tagged PRMT5 construct in COS-1 cells led to a near complete loss of obsd. enzymic activity.  PRMT5 is the first example of a catalytic chain for a type II protein arginine N-methyltransferase that can result in the formation of sym. dimethylarginine residues as obsd. previously in myelin basic protein, Sm small nuclear ribonucleoproteins, and other polypeptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZJzrYzZ_ze7Vg90H21EOLACvtfcHk0liI3avM5DLBrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFCmtbw%253D&md5=12e2a0db9d4747c4e7b06668b9fd4d2b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M105412200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M105412200%26sid%3Dliteratum%253Aachs%26aulast%3DBranscombe%26aufirst%3DT.%2BL.%26aulast%3DFrankel%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DJ.-H.%26aulast%3DCook%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DZ.-h.%26aulast%3DPestka%26aufirst%3DS.%26aulast%3DClarke%26aufirst%3DS.%26atitle%3DPRMT5%2520%2528Janus%2520kinase-binding%2520protein%25201%2529%2520catalyzes%2520the%2520formation%2520of%2520symmetric%2520dimethylarginine%2520residues%2520in%2520proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D32971%26epage%3D32976%26doi%3D10.1074%2Fjbc.m105412200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishwanath, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdjument-Bromage, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sif, S.</span></span> <span> </span><span class="NLM_article-title">Human SWI/SNF-associated PRMT5 methylates histone H3 arginine 8 and negatively regulates expression of ST7 and NM23 tumor suppressor genes</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">9630</span>– <span class="NLM_lpage">9645</span>, <span class="refDoi"> DOI: 10.1128/mcb.24.21.9630-9645.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1128%2FMCB.24.21.9630-9645.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=15485929" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltl2htQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=9630-9645&author=S.+Palauthor=S.+N.+Vishwanathauthor=H.+Erdjument-Bromageauthor=P.+Tempstauthor=S.+Sif&title=Human+SWI%2FSNF-associated+PRMT5+methylates+histone+H3+arginine+8+and+negatively+regulates+expression+of+ST7+and+NM23+tumor+suppressor+genes&doi=10.1128%2Fmcb.24.21.9630-9645.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Human SWI/SNF-associated PRMT5 methylates histone H3 arginine 8 and negatively regulates expression of ST7 and NM23 tumor suppressor genes</span></div><div class="casAuthors">Pal, Sharmistha; Vishwanath, Sheethal N.; Erdjument-Bromage, Hediye; Tempst, Paul; Sif, Said</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9630-9645</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) have been implicated in transcriptional activation and repression, but their role in controlling cell growth and proliferation remains obscure.  We have recently shown that PRMT5 can interact with flag-tagged BRG1- and hBRM-based hSWI/SNF chromatin remodelers and that both complexes can specifically methylate histones H3 and H4.  Here we report that PRMT5 can be found in assocn. with endogenous hSWI/SNF complexes, which can methylate H3 and H4 N-terminal tails, and show that H3 arginine 8 and H4 arginine 3 are preferred sites of methylation by recombinant and hSWI/SNF-assocd. PRMT5.  To elucidate the role played by PRMT5 in gene regulation, we have established a PRMT5 antisense cell line and detd. by microarray anal. that more genes are derepressed when PRMT5 levels are reduced.  Among the affected genes, we show that suppressor of tumorigenicity 7 (ST7) and nonmetastatic 23 (NM23) are direct targets of PRMT5-contg. BRG1 and hBRM complexes.  Furthermore, we demonstrate that expression of ST7 and NM23 is reduced in a cell line that overexpresses PRMT5 and that this decrease in expression correlates with H3R8 methylation, H3K9 deacetylation, and increased transformation of NIH 3T3 cells.  These findings suggest that the BRG1- and hBRM-assocd. PRMT5 regulates cell growth and proliferation by controlling expression of genes involved in tumor suppression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHC4UpBSGWILVg90H21EOLACvtfcHk0liI3avM5DLBrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltl2htQ%253D%253D&md5=663371d517c0e9e3b709483e320869f7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.21.9630-9645.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.21.9630-9645.2004%26sid%3Dliteratum%253Aachs%26aulast%3DPal%26aufirst%3DS.%26aulast%3DVishwanath%26aufirst%3DS.%2BN.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DSif%26aufirst%3DS.%26atitle%3DHuman%2520SWI%252FSNF-associated%2520PRMT5%2520methylates%2520histone%2520H3%2520arginine%25208%2520and%2520negatively%2520regulates%2520expression%2520of%2520ST7%2520and%2520NM23%2520tumor%2520suppressor%2520genes%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D9630%26epage%3D9645%26doi%3D10.1128%2Fmcb.24.21.9630-9645.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrd, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grever, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sif, S.</span></span> <span> </span><span class="NLM_article-title">Low levels of miR-92b/96 induce PRMT5 translation and H3R8/H4R3 methylation in mantle cell lymphoma</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">3558</span>– <span class="NLM_lpage">3569</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7601794</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fsj.emboj.7601794" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=17627275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BD2sXos1eqtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=3558-3569&author=S.+Palauthor=R.+A.+Baiocchiauthor=J.+C.+Byrdauthor=M.+R.+Greverauthor=S.+T.+Jacobauthor=S.+Sif&title=Low+levels+of+miR-92b%2F96+induce+PRMT5+translation+and+H3R8%2FH4R3+methylation+in+mantle+cell+lymphoma&doi=10.1038%2Fsj.emboj.7601794"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Low levels of miR-92b/96 induce PRMT5 translation and H3R8/H4R3 methylation in mantle cell lymphoma</span></div><div class="casAuthors">Pal, Sharmistha; Baiocchi, Robert A.; Byrd, John C.; Grever, Michael R.; Jacob, Samson T.; Sif, Said</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3558-3569</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase PRMT5 interacts with human SWI/SNF complexes and methylates histones H3R8 and H4R3.  To elucidate the role of PRMT5 in human cancer, we analyzed PRMT5 expression in normal human B lymphocytes and a panel of lymphoid cancer cell lines as well as mantle cell lymphoma (MCL) clin. samples.  We show that PRMT5 protein levels are elevated in all cancer cells, including clin. samples examd. despite its low rate of transcription and mRNA stability.  Remarkably, polysome profiling revealed that PRMT5 mRNA is translated more efficiently in Mino and JeKo MCL cells than in normal B cells, and that decreased miR-92b and miR-96 expression augments PRMT5 translation.  Consequently, global methylation of H3R8 and H4R3 is increased and is accompanied by repression of suppressor of tumorigenicity 7 (ST7) in lymphoid cancer cells.  Furthermore, knockdown of PRMT5 expression reduces proliferation of transformed JeKo and Raji cells.  Thus, our studies indicate that aberrant expression of PRMT5 leads to altered epigenetic modification of chromatin, which in turn impacts transcriptional performance of anti-cancer genes and growth of transformed lymphoid cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiSE5d420ctLVg90H21EOLACvtfcHk0ljrl2V1dZUAhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXos1eqtbg%253D&md5=f2b153393a175b1427873a9db0f52a4a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7601794&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7601794%26sid%3Dliteratum%253Aachs%26aulast%3DPal%26aufirst%3DS.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DGrever%26aufirst%3DM.%2BR.%26aulast%3DJacob%26aufirst%3DS.%2BT.%26aulast%3DSif%26aufirst%3DS.%26atitle%3DLow%2520levels%2520of%2520miR-92b%252F96%2520induce%2520PRMT5%2520translation%2520and%2520H3R8%252FH4R3%2520methylation%2520in%2520mantle%2520cell%2520lymphoma%26jtitle%3DEMBO%2520J.%26date%3D2007%26volume%3D26%26spage%3D3558%26epage%3D3569%26doi%3D10.1038%2Fsj.emboj.7601794" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Migliori, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phalke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bezzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mok, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahu, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunaratne, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capasso, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecatiello, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Marco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackstock, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amati, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mapelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span> <span> </span><span class="NLM_article-title">Symmetric dimethylation of H3R2 is a newly identified histone mark that supports euchromatin maintenance</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">136</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fnsmb.2209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=22231400" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC38Xls1ShtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=136-144&author=V.+Miglioriauthor=J.+M%C3%BCllerauthor=S.+Phalkeauthor=D.+Lowauthor=M.+Bezziauthor=W.+C.+Mokauthor=S.+K.+Sahuauthor=J.+Gunaratneauthor=P.+Capassoauthor=C.+Bassiauthor=V.+Cecatielloauthor=A.+De+Marcoauthor=W.+Blackstockauthor=V.+Kuznetsovauthor=B.+Amatiauthor=M.+Mapelliauthor=E.+Guccione&title=Symmetric+dimethylation+of+H3R2+is+a+newly+identified+histone+mark+that+supports+euchromatin+maintenance&doi=10.1038%2Fnsmb.2209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Symmetric dimethylation of H3R2 is a newly identified histone mark that supports euchromatin maintenance</span></div><div class="casAuthors">Migliori, Valentina; Muller, Julius; Phalke, Sameer; Low, Diana; Bezzi, Marco; Mok, Wei Chuen; Sahu, Sanjeeb Kumar; Gunaratne, Jayantha; Capasso, Paola; Bassi, Christian; Cecatiello, Valentina; De Marco, Ario; Blackstock, Walter; Kuznetsov, Vladimir; Amati, Bruno; Mapelli, Marina; Guccione, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">136-144</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The asym. dimethylation of histone H3 arginine 2 (H3R2me2a) acts as a repressive mark that antagonizes trimethylation of H3 lysine 4.  Here we report that H3R2 is also sym. dimethylated (H3R2me2s) by PRMT5 and PRMT7 and present in euchromatic regions.  Profiling of H3-tail interactors by SILAC MS revealed that H3R2me2s excludes binding of RBBP7, a central component of co-repressor complexes Sin3a, NURD and PRC2.  Conversely H3R2me2s enhances binding of WDR5, a common component of the coactivator complexes MLL, SET1A, SET1B, NLS1 and ATAC.  The interaction of histone H3 with WDR5 distinguishes H3R2me2s from H3R2me2a, which impedes the recruitment of WDR5 to chromatin.  The crystallog. structure of WDR5 and the H3R2me2s peptide elucidates the mol. determinants of this high affinity interaction.  Our findings identify H3R2me2s as a previously unknown mark that keeps genes poised in euchromatin for transcriptional activation upon cell-cycle withdrawal and differentiation in human cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDe73jpv41ObVg90H21EOLACvtfcHk0ljrl2V1dZUAhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xls1ShtA%253D%253D&md5=c7b15ed7ee8f4819b45d488171e89ac4</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2209%26sid%3Dliteratum%253Aachs%26aulast%3DMigliori%26aufirst%3DV.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%26aulast%3DPhalke%26aufirst%3DS.%26aulast%3DLow%26aufirst%3DD.%26aulast%3DBezzi%26aufirst%3DM.%26aulast%3DMok%26aufirst%3DW.%2BC.%26aulast%3DSahu%26aufirst%3DS.%2BK.%26aulast%3DGunaratne%26aufirst%3DJ.%26aulast%3DCapasso%26aufirst%3DP.%26aulast%3DBassi%26aufirst%3DC.%26aulast%3DCecatiello%26aufirst%3DV.%26aulast%3DDe%2BMarco%26aufirst%3DA.%26aulast%3DBlackstock%26aufirst%3DW.%26aulast%3DKuznetsov%26aufirst%3DV.%26aulast%3DAmati%26aufirst%3DB.%26aulast%3DMapelli%26aufirst%3DM.%26aulast%3DGuccione%26aufirst%3DE.%26atitle%3DSymmetric%2520dimethylation%2520of%2520H3R2%2520is%2520a%2520newly%2520identified%2520histone%2520mark%2520that%2520supports%2520euchromatin%2520maintenance%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D136%26epage%3D144%26doi%3D10.1038%2Fnsmb.2209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Musiani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bok, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massignani, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabaglio, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ippolito, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuomo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozbek, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zorgati, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghoshdastider, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonaldi, T.</span></span> <span> </span><span class="NLM_article-title">Proteomics profiling of arginine methylation defines PRMT5 substrate specificity</span>. <i>Sci. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">eaat8388</span> <span class="refDoi"> DOI: 10.1126/scisignal.aat8388</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1126%2Fscisignal.aat8388" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=30940768" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2019&author=D.+Musianiauthor=J.+Bokauthor=E.+Massignaniauthor=L.+Wuauthor=T.+Tabaglioauthor=M.+R.+Ippolitoauthor=A.+Cuomoauthor=U.+Ozbekauthor=H.+Zorgatiauthor=U.+Ghoshdastiderauthor=R.+C.+Robinsonauthor=E.+Guccioneauthor=T.+Bonaldi&title=Proteomics+profiling+of+arginine+methylation+defines+PRMT5+substrate+specificity&doi=10.1126%2Fscisignal.aat8388"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscisignal.aat8388&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscisignal.aat8388%26sid%3Dliteratum%253Aachs%26aulast%3DMusiani%26aufirst%3DD.%26aulast%3DBok%26aufirst%3DJ.%26aulast%3DMassignani%26aufirst%3DE.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DTabaglio%26aufirst%3DT.%26aulast%3DIppolito%26aufirst%3DM.%2BR.%26aulast%3DCuomo%26aufirst%3DA.%26aulast%3DOzbek%26aufirst%3DU.%26aulast%3DZorgati%26aufirst%3DH.%26aulast%3DGhoshdastider%26aufirst%3DU.%26aulast%3DRobinson%26aufirst%3DR.%2BC.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DBonaldi%26aufirst%3DT.%26atitle%3DProteomics%2520profiling%2520of%2520arginine%2520methylation%2520defines%2520PRMT5%2520substrate%2520specificity%26jtitle%3DSci.%2520Signaling%26date%3D2019%26volume%3D12%26doi%3D10.1126%2Fscisignal.aat8388" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karkhanis, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbalzano, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sif, S.</span></span> <span> </span><span class="NLM_article-title">Versatility of PRMT5-induced methylation in growth control and development</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">633</span>– <span class="NLM_lpage">641</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2011.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1016%2Fj.tibs.2011.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=21975038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFejurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2011&pages=633-641&author=V.+Karkhanisauthor=Y.-J.+Huauthor=R.+A.+Baiocchiauthor=A.+N.+Imbalzanoauthor=S.+Sif&title=Versatility+of+PRMT5-induced+methylation+in+growth+control+and+development&doi=10.1016%2Fj.tibs.2011.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Versatility of PRMT5-induced methylation in growth control and development</span></div><div class="casAuthors">Karkhanis, Vrajesh; Hu, Yu-Jie; Baiocchi, Robert A.; Imbalzano, Anthony N.; Sif, Said</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">633-641</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Arginine methylation governs important cellular processes that impact growth and proliferation, as well as differentiation and development.  Through their ability to catalyze sym. or asym. methylation of histone and non-histone proteins, members of the protein arginine methyltransferase (PRMT) family regulate chromatin structure and expression of a wide spectrum of target genes.  Unlike other PRMTs, PRMT5 works in concert with a variety of cellular proteins including ATP-dependent chromatin remodelers and co-repressors to induce epigenetic silencing.  Recent work also implicates PRMT5 in the control of growth-promoting and pro-survival pathways, which demonstrates its versatility as an enzyme involved in both epigenetic regulation of anti-cancer target genes and organelle biogenesis.  These studies not only provide insight into the mol. mechanisms by which PRMT5 contributes to growth control, but also justify therapeutic targeting of PRMT5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD1--MutXQ2bVg90H21EOLACvtfcHk0ljLjp-YLC1now"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFejurrJ&md5=e5cb228750165f39636118ddee2a0ee8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2011.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2011.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DKarkhanis%26aufirst%3DV.%26aulast%3DHu%26aufirst%3DY.-J.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DImbalzano%26aufirst%3DA.%2BN.%26aulast%3DSif%26aufirst%3DS.%26atitle%3DVersatility%2520of%2520PRMT5-induced%2520methylation%2520in%2520growth%2520control%2520and%2520development%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2011%26volume%3D36%26spage%3D633%26epage%3D641%26doi%3D10.1016%2Fj.tibs.2011.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blanc, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, S.</span></span> <span> </span><span class="NLM_article-title">Arginine methylation: the coming of age</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2016.11.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1016%2Fj.molcel.2016.11.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=28061334" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmsVKmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2017&pages=8-24&author=R.+S.+Blancauthor=S.+Richard&title=Arginine+methylation%3A+the+coming+of+age&doi=10.1016%2Fj.molcel.2016.11.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine Methylation: The Coming of Age</span></div><div class="casAuthors">Blanc, Romeo S.; Richard, Stephane</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8-24</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Arginine methylation is a common post-translational modification functioning as an epigenetic regulator of transcription and playing key roles in pre-mRNA splicing, DNA damage signaling, mRNA translation, cell signaling, and cell fate decision.  Recently, a wealth of studies using transgenic mouse models and selective PRMT inhibitors helped define physiol. roles for protein arginine methyltransferases (PRMTs) linking them to diseases such as cancer and metabolic, neurodegenerative, and muscular disorders.  This review describes the recent mol. advances that have been uncovered in normal and diseased mammalian cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqT2OQsTa2UnbVg90H21EOLACvtfcHk0ljLjp-YLC1now"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmsVKmtA%253D%253D&md5=7ca3a6b7f0914c0034534ab8ca908506</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2016.11.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2016.11.003%26sid%3Dliteratum%253Aachs%26aulast%3DBlanc%26aufirst%3DR.%2BS.%26aulast%3DRichard%26aufirst%3DS.%26atitle%3DArginine%2520methylation%253A%2520the%2520coming%2520of%2520age%26jtitle%3DMol.%2520Cell%26date%3D2017%26volume%3D65%26spage%3D8%26epage%3D24%26doi%3D10.1016%2Fj.molcel.2016.11.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fong, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pignata, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goy, P.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawabata, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koh, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musiani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massignani, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotini, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wun, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rialdi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wollmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mzoughi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luciani, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wouters, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delwel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papapetrou, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minden, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonaldi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Wahab, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span> <span> </span><span class="NLM_article-title">Therapeutic targeting of RNA splicing catalysis through inhibition of protein arginine methylation</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">194</span>– <span class="NLM_lpage">209</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2019.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1016%2Fj.ccell.2019.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=31408619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFyltLbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2019&pages=194-209&author=J.+Y.+Fongauthor=L.+Pignataauthor=P.-A.+Goyauthor=K.+C.+Kawabataauthor=S.+C.-W.+Leeauthor=C.+M.+Kohauthor=D.+Musianiauthor=E.+Massignaniauthor=A.+G.+Kotiniauthor=A.+Pensonauthor=C.+M.+Wunauthor=Y.+Shenauthor=M.+Schwarzauthor=D.+H.+Lowauthor=A.+Rialdiauthor=M.+Kiauthor=H.+Wollmannauthor=S.+Mzoughiauthor=F.+Gayauthor=C.+Thompsonauthor=T.+Hartauthor=O.+Barbashauthor=G.+M.+Lucianiauthor=M.+M.+Szewczykauthor=B.+J.+Woutersauthor=R.+Delwelauthor=E.+P.+Papapetrouauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=M.+D.+Mindenauthor=J.+Jinauthor=A.+Melnickauthor=T.+Bonaldiauthor=O.+Abdel-Wahabauthor=E.+Guccione&title=Therapeutic+targeting+of+RNA+splicing+catalysis+through+inhibition+of+protein+arginine+methylation&doi=10.1016%2Fj.ccell.2019.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic Targeting of RNA Splicing Catalysis through Inhibition of Protein Arginine Methylation</span></div><div class="casAuthors">Fong, Jia Yi; Pignata, Luca; Goy, Pierre-Alexis; Kawabata, Kimihito Cojin; Lee, Stanley Chun-Wei; Koh, Cheryl M.; Musiani, Daniele; Massignani, Enrico; Kotini, Andriana G.; Penson, Alex; Wun, Cheng Mun; Shen, Yudao; Schwarz, Megan; Low, Diana H. P.; Rialdi, Alexander; Ki, Michelle; Wollmann, Heike; Mzoughi, Slim; Gay, Florence; Thompson, Christine; Hart, Timothy; Barbash, Olena; Luciani, Genna M.; Szewczyk, Magdalena M.; Wouters, Bas J.; Delwel, Ruud; Papapetrou, Eirini P.; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Minden, Mark D.; Jin, Jian; Melnick, Ari; Bonaldi, Tiziana; Abdel-Wahab, Omar; Guccione, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">194-209.e9</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Cancer-assocd. mutations in genes encoding RNA splicing factors (SFs) commonly occur in leukemias, as well as in a variety of solid tumors, and confer dependence on wild-type splicing.  These observations have led to clin. efforts to directly inhibit the spliceosome in patients with refractory leukemias.  Here, we identify that inhibiting sym. or asym. dimethylation of arginine, mediated by PRMT5 and type I protein arginine methyltransferases (PRMTs), resp., reduces splicing fidelity and results in preferential killing of SF-mutant leukemias over wild-type counterparts.  These data identify genetic subsets of cancer most likely to respond to PRMT inhibition, synergistic effects of combined PRMT5 and type I PRMT inhibition, and a mechanistic basis for the therapeutic efficacy of PRMT inhibition in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7gcqR_ejFw7Vg90H21EOLACvtfcHk0ljLjp-YLC1now"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFyltLbO&md5=0d124279e457ee519f878f49cdc5b36b</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DFong%26aufirst%3DJ.%2BY.%26aulast%3DPignata%26aufirst%3DL.%26aulast%3DGoy%26aufirst%3DP.-A.%26aulast%3DKawabata%26aufirst%3DK.%2BC.%26aulast%3DLee%26aufirst%3DS.%2BC.-W.%26aulast%3DKoh%26aufirst%3DC.%2BM.%26aulast%3DMusiani%26aufirst%3DD.%26aulast%3DMassignani%26aufirst%3DE.%26aulast%3DKotini%26aufirst%3DA.%2BG.%26aulast%3DPenson%26aufirst%3DA.%26aulast%3DWun%26aufirst%3DC.%2BM.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSchwarz%26aufirst%3DM.%26aulast%3DLow%26aufirst%3DD.%2BH.%26aulast%3DRialdi%26aufirst%3DA.%26aulast%3DKi%26aufirst%3DM.%26aulast%3DWollmann%26aufirst%3DH.%26aulast%3DMzoughi%26aufirst%3DS.%26aulast%3DGay%26aufirst%3DF.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DHart%26aufirst%3DT.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DLuciani%26aufirst%3DG.%2BM.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DWouters%26aufirst%3DB.%2BJ.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DPapapetrou%26aufirst%3DE.%2BP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DMinden%26aufirst%3DM.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DBonaldi%26aufirst%3DT.%26aulast%3DAbdel-Wahab%26aufirst%3DO.%26aulast%3DGuccione%26aufirst%3DE.%26atitle%3DTherapeutic%2520targeting%2520of%2520RNA%2520splicing%2520catalysis%2520through%2520inhibition%2520of%2520protein%2520arginine%2520methylation%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26spage%3D194%26epage%3D209%26doi%3D10.1016%2Fj.ccell.2019.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferases and cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1038/nrc3409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fnrc3409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=23235912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVajur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=37-50&author=Y.+Yangauthor=M.+T.+Bedford&title=Protein+arginine+methyltransferases+and+cancer&doi=10.1038%2Fnrc3409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferases and cancer</span></div><div class="casAuthors">Yang, Yanzhong; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-50</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There are nine protein arginine methyltransferases (PRMTs) encoded in mammalian genomes, the protein products of which catalyze three types of arginine methylation - monomethylation and two types of dimethylation.  Protein arginine methylation is an abundant modification that has been implicated in signal transduction, gene transcription, DNA repair and mRNA splicing, among others.  Studies have only recently linked this modification to carcinogenesis and metastasis.  Sequencing studies have not generally found alterations to the PRMTs; however, overexpression of these enzymes is often assocd. with various cancers, which might make some of them viable targets for therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnb4O4wtn5fLVg90H21EOLACvtfcHk0ljxmAPIYH_ofA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVajur7I&md5=9df3b58d92e7932e214e6de26a10fa46</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrc3409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3409%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DProtein%2520arginine%2520methyltransferases%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2013%26volume%3D13%26spage%3D37%26epage%3D50%26doi%3D10.1038%2Fnrc3409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stopa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krebs, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shechter, D.</span></span> <span> </span><span class="NLM_article-title">The PRMT5 arginine methyltransferase: many roles in development, cancer and beyond</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">2041</span>– <span class="NLM_lpage">2059</span>, <span class="refDoi"> DOI: 10.1007/s00018-015-1847-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1007%2Fs00018-015-1847-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=25662273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitlOrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2015&pages=2041-2059&author=N.+Stopaauthor=J.+E.+Krebsauthor=D.+Shechter&title=The+PRMT5+arginine+methyltransferase%3A+many+roles+in+development%2C+cancer+and+beyond&doi=10.1007%2Fs00018-015-1847-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The PRMT5 arginine methyltransferase- many roles in development, cancer and beyond</span></div><div class="casAuthors">Stopa, Nicole; Krebs, Jocelyn E.; Shechter, David</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2041-2059</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">Post-translational arginine methylation is responsible for regulation of many biol. processes.  The protein arginine methyltransferase 5 (PRMT5, also known as Hsl7, Jbp1, Skb1, Capsuleen, or Dart5) is the major enzyme responsible for mono- and sym. dimethylation of arginine.  An expanding literature demonstrates its crit. biol. function in a wide range of cellular processes.  Histone and other protein methylation by PRMT5 regulate genome organization, transcription, stem cells, primordial germ cells, differentiation, the cell cycle, and spliceosome assembly.  Metazoan PRMT5 is found in complex with the WD-repeat protein MEP50 (also known as Wdr77, androgen receptor coactivator p44, or Valois).  PRMT5 also directly assocs. with a range of other protein factors, including pICln, Menin, CoPR5 and RioK1 that may alter its subcellular localization and protein substrate selection.  Protein substrate and PRMT5-MEP50 post-translation modifications induce crosstalk to regulate PRMT5 activity.  Crystal structures of C. elegans PRMT5 and human and frog PRMT5-MEP50 complexes provide substantial insight into the mechanisms of substrate recognition and procession to dimethylation.  Enzymol. studies of PRMT5 have uncovered compelling insights essential for future development of specific PRMT5 inhibitors.  In addn., newly accumulating evidence implicates PRMT5 and MEP50 expression levels and their methyltransferase activity in cancer tumorigenesis, and, significantly, as markers of poor clin. outcome, marking them as potential oncogenes.  Here, we review the substantial new literature on PRMT5 and its partners to highlight the significance of understanding this essential enzyme in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGZo-Bwo40_7Vg90H21EOLACvtfcHk0ljxmAPIYH_ofA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitlOrtr0%253D&md5=2173fff7825f142cde0ddde7f7b879f5</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs00018-015-1847-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-015-1847-9%26sid%3Dliteratum%253Aachs%26aulast%3DStopa%26aufirst%3DN.%26aulast%3DKrebs%26aufirst%3DJ.%2BE.%26aulast%3DShechter%26aufirst%3DD.%26atitle%3DThe%2520PRMT5%2520arginine%2520methyltransferase%253A%2520many%2520roles%2520in%2520development%252C%2520cancer%2520and%2520beyond%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2015%26volume%3D72%26spage%3D2041%26epage%3D2059%26doi%3D10.1007%2Fs00018-015-1847-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richters, A.</span></span> <span> </span><span class="NLM_article-title">Targeting protein arginine methyltransferase 5 in disease</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2081</span>– <span class="NLM_lpage">2098</span>, <span class="refDoi"> DOI: 10.4155/fmc-2017-0089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.4155%2Ffmc-2017-0089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=29076773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOit7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=2081-2098&author=A.+Richters&title=Targeting+protein+arginine+methyltransferase+5+in+disease&doi=10.4155%2Ffmc-2017-0089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein arginine methyltransferase 5 in disease</span></div><div class="casAuthors">Richters, Andre</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2081-2098</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">PRMT5 catalyzes the mono- and sym. dimethylation of the arginine N-guanidine group of a wide variety of target proteins including histones, transcriptional elongation factors, kinases and tumor suppressors by utilizing the essential co-factor S-adenosylmethionine as Me source.  PRMT5 overexpression has been linked to the progression of various diseases, including cancer, and is oftentimes assocd. with a poor prognosis.  Therefore, PRMT5 is promoted as a valuable target for drug discovery approaches and was a subject matter in recent endeavors aiming for the development of specific PRMT5 inhibitors.  This review will embrace the significance of PRMT5 as therapeutic target with respect to its mol. interdependencies in disease states as well as its implication in drug development approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqverMGPRhuJbVg90H21EOLACvtfcHk0ljxmAPIYH_ofA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOit7fN&md5=6f8f3e27b9d7389e5b89ff084e6b691e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2017-0089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2017-0089%26sid%3Dliteratum%253Aachs%26aulast%3DRichters%26aufirst%3DA.%26atitle%3DTargeting%2520protein%2520arginine%2520methyltransferase%25205%2520in%2520disease%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D2081%26epage%3D2098%26doi%3D10.4155%2Ffmc-2017-0089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCorvy, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wacker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agegnehu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lansu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tribo, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, R. H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span> <span> </span><span class="NLM_article-title">Structural determinants of 5-HT2B receptor activation and biased agonism</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">787</span>– <span class="NLM_lpage">796</span>, <span class="refDoi"> DOI: 10.1038/s41594-018-0116-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fs41594-018-0116-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=30127358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFGjtLjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=787-796&author=J.+D.+McCorvyauthor=D.+Wackerauthor=S.+Wangauthor=B.+Agegnehuauthor=J.+Liuauthor=K.+Lansuauthor=A.+R.+Triboauthor=R.+H.+J.+Olsenauthor=T.+Cheauthor=J.+Jinauthor=B.+L.+Roth&title=Structural+determinants+of+5-HT2B+receptor+activation+and+biased+agonism&doi=10.1038%2Fs41594-018-0116-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structural determinants of 5-HT2B receptor activation and biased agonism</span></div><div class="casAuthors">McCorvy, John D.; Wacker, Daniel; Wang, Sheng; Agegnehu, Bemnat; Liu, Jing; Lansu, Katherine; Tribo, Alexandra R.; Olsen, Reid H. J.; Che, Tao; Jin, Jian; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">787-796</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Serotonin (5-hydroxytryptamine; 5-HT) receptors modulate a variety of physiol. processes ranging from perception, cognition and emotion to vascular and smooth muscle contraction, platelet aggregation, gastrointestinal function and reprodn.  Drugs that interact with 5-HT receptors effectively treat diseases as diverse as migraine headaches, depression and obesity.  Here we present four structures of a prototypical serotonin receptor-the human 5-HT2B receptor-in complex with chem. and pharmacol. diverse drugs, including methysergide, methylergonovine, lisuride and LY266097.  A detailed anal. of these structures complemented by comprehensive interrogation of signaling illuminated key structural determinants essential for activation.  Addnl. structure-guided mutagenesis expts. revealed binding pocket residues that were essential for agonist-mediated biased signaling and β-arrestin2 translocation.  Given the importance of 5-HT receptors for a large no. of therapeutic indications, insights derived from these studies should accelerate the design of safer and more effective medications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwC5QTOdEZhLVg90H21EOLACvtfcHk0ljxmAPIYH_ofA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFGjtLjM&md5=93be9401bc680249499fa3af5d40f395</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fs41594-018-0116-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41594-018-0116-7%26sid%3Dliteratum%253Aachs%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DWacker%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DAgegnehu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLansu%26aufirst%3DK.%26aulast%3DTribo%26aufirst%3DA.%2BR.%26aulast%3DOlsen%26aufirst%3DR.%2BH.%2BJ.%26aulast%3DChe%26aufirst%3DT.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DStructural%2520determinants%2520of%25205-HT2B%2520receptor%2520activation%2520and%2520biased%2520agonism%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D787%26epage%3D796%26doi%3D10.1038%2Fs41594-018-0116-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shimizu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takami, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koike, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodera, Y.</span></span> <span> </span><span class="NLM_article-title">The protein arginine methyltransferase 5 promotes malignant phenotype of hepatocellular carcinoma cells and is associated with adverse patient outcomes after curative hepatectomy</span>. <i>Int. J. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">381</span>– <span class="NLM_lpage">386</span>, <span class="refDoi"> DOI: 10.3892/ijo.2017.3833</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.3892%2Fijo.2017.3833" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=28101581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitleqtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2017&pages=381-386&author=D.+Shimizuauthor=M.+Kandaauthor=H.+Sugimotoauthor=M.+Shibataauthor=H.+Tanakaauthor=H.+Takamiauthor=N.+Iwataauthor=M.+Hayashiauthor=C.+Tanakaauthor=D.+Kobayashiauthor=S.+Yamadaauthor=G.+Nakayamaauthor=M.+Koikeauthor=M.+Fujiwaraauthor=T.+Fujiiauthor=Y.+Kodera&title=The+protein+arginine+methyltransferase+5+promotes+malignant+phenotype+of+hepatocellular+carcinoma+cells+and+is+associated+with+adverse+patient+outcomes+after+curative+hepatectomy&doi=10.3892%2Fijo.2017.3833"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The protein arginine methyltransferase 5 promotes malignant phenotype of hepatocellular carcinoma cells and is associated with adverse patient outcomes after curative hepatectomy</span></div><div class="casAuthors">Shimizu, Dai; Kanda, Mitsuro; Sugimoto, Hiroyuki; Shibata, Masahiro; Tanaka, Haruyoshi; Takami, Hideki; Iwata, Naoki; Hayashi, Masamichi; Tanaka, Chie; Kobayashi, Daisuke; Yamada, Suguru; Nakayama, Goro; Koike, Masahiko; Fujiwara, Michitaka; Fujii, Tsutomu; Ko, Yasuhiro</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">381-386</span>CODEN:
                <span class="NLM_cas:coden">IJONES</span>;
        ISSN:<span class="NLM_cas:issn">1791-2423</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">The prognosis of advanced hepatocellular carcinoma (HCC) is dismal.  Novel mol. targets for diagnosis and therapy is urgently required.  This study evaluated expression and functions of the protein arginine methyltransferase 5 (PRMT5) in HCC.  Using HCC cell lines, the expression levels of PRMT5 mRNA were detd. using the quant. real-time reverse-transcription polymerase chain reaction, and the effect of a small interfering PRMT5-siRNA on cell phenotype was evaluated.  Further, PRMT5 expression was detd. in 144 pairs of resected liver tissues to evaluate its clin. significance.  Regardless of their differentiated phenotypes, nine HCC cell lines expressed different levels of PRMT5 mRNA.  Inhibition of PRMT5 expression significantly decreased the proliferation, invasion, and migration of HCC cell lines.  Although the level of PRMT5 mRNA was not influenced by patient's background liver status, it was significantly higher in HCC tissues than in the corresponding noncancerous tissues.  High levels of PRMT5 mRNA in HCC tissues were significantly assocd. with advanced disease stage and adverse prognosis.  In conclusion, our results indicate that PRMT5 may act as a putative oncogene in HCC and that the levels of PRMT5 mRNA represent a promising prognostic marker and a potential target of mol. therapy for HCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgqfuS0Y9NMLVg90H21EOLACvtfcHk0lgIFp4qqjJ1nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitleqtbc%253D&md5=080eb5763b9c1b1a61ab981d3e422971</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.3892%2Fijo.2017.3833&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijo.2017.3833%26sid%3Dliteratum%253Aachs%26aulast%3DShimizu%26aufirst%3DD.%26aulast%3DKanda%26aufirst%3DM.%26aulast%3DSugimoto%26aufirst%3DH.%26aulast%3DShibata%26aufirst%3DM.%26aulast%3DTanaka%26aufirst%3DH.%26aulast%3DTakami%26aufirst%3DH.%26aulast%3DIwata%26aufirst%3DN.%26aulast%3DHayashi%26aufirst%3DM.%26aulast%3DTanaka%26aufirst%3DC.%26aulast%3DKobayashi%26aufirst%3DD.%26aulast%3DYamada%26aufirst%3DS.%26aulast%3DNakayama%26aufirst%3DG.%26aulast%3DKoike%26aufirst%3DM.%26aulast%3DFujiwara%26aufirst%3DM.%26aulast%3DFujii%26aufirst%3DT.%26aulast%3DKodera%26aufirst%3DY.%26atitle%3DThe%2520protein%2520arginine%2520methyltransferase%25205%2520promotes%2520malignant%2520phenotype%2520of%2520hepatocellular%2520carcinoma%2520cells%2520and%2520is%2520associated%2520with%2520adverse%2520patient%2520outcomes%2520after%2520curative%2520hepatectomy%26jtitle%3DInt.%2520J.%2520Oncol.%26date%3D2017%26volume%3D50%26spage%3D381%26epage%3D386%26doi%3D10.3892%2Fijo.2017.3833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elzey, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pili, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratliff, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, C.-D.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferase 5 functions as an epigenetic activator of the androgen receptor to promote prostate cancer cell growth</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1223</span>– <span class="NLM_lpage">1231</span>, <span class="refDoi"> DOI: 10.1038/onc.2016.287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fonc.2016.287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=27546619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlyisLvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=1223-1231&author=X.+Dengauthor=G.+Shaoauthor=H.-T.+Zhangauthor=C.+Liauthor=D.+Zhangauthor=L.+Chengauthor=B.+D.+Elzeyauthor=R.+Piliauthor=T.+L.+Ratliffauthor=J.+Huangauthor=C.-D.+Hu&title=Protein+arginine+methyltransferase+5+functions+as+an+epigenetic+activator+of+the+androgen+receptor+to+promote+prostate+cancer+cell+growth&doi=10.1038%2Fonc.2016.287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferase 5 functions as an epigenetic activator of the androgen receptor to promote prostate cancer cell growth</span></div><div class="casAuthors">Deng, X.; Shao, G.; Zhang, H.-T.; Li, C.; Zhang, D.; Cheng, L.; Elzey, B. D.; Pili, R.; Ratliff, T. L.; Huang, J.; Hu, C.-D.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1223-1231</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is an emerging epigenetic enzyme that mainly represses transcription of target genes via sym. dimethylation of arginine residues on histones H4R3, H3R8 and H2AR3.  Accumulating evidence suggests that PRMT5 may function as an oncogene to drive cancer cell growth by epigenetic inactivation of several tumor suppressors.  Here, we provide evidence that PRMT5 promotes prostate cancer cell growth by epigenetically activating transcription of the androgen receptor (AR) in prostate cancer cells.  Knockdown of PRMT5 or inhibition of PRMT5 by a specific inhibitor reduces the expression of AR and suppresses the growth of multiple AR-pos., but not AR-neg., prostate cancer cells.  Significantly, knockdown of PRMT5 in AR-pos. LNCaP cells completely suppresses the growth of xenograft tumors in mice.  Mol. anal. reveals that PRMT5 binds to the proximal promoter region of the AR gene and contributes mainly to the enriched sym. dimethylation of H4R3 in the same region.  Mechanistically, PRMT5 is recruited to the AR promoter by its interaction with Sp1, the major transcription factor responsible for AR transcription, and forms a complex with Brg1, an ATP-dependent chromatin remodeler, on the proximal promoter region of the AR gene.  Furthermore, PRMT5 expression in prostate cancer tissues is significantly higher than that in benign prostatic hyperplasia tissues, and PRMT5 expression correlates pos. with AR expression at both the protein and mRNA levels.  Taken together, our results identify PRMT5 as a novel epigenetic activator of AR in prostate cancer.  Given that inhibiting AR transcriptional activity or androgen synthesis remains the major mechanism of action for most existing anti-androgen agents, our findings also raise an interesting possibility that targeting PRMT5 may represent a novel approach for prostate cancer treatment by eliminating AR expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6wjniAjJYjbVg90H21EOLACvtfcHk0lgIFp4qqjJ1nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlyisLvK&md5=75aff420e6c8a819bd41fd45cec7dec9</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fonc.2016.287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2016.287%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DShao%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DH.-T.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DCheng%26aufirst%3DL.%26aulast%3DElzey%26aufirst%3DB.%2BD.%26aulast%3DPili%26aufirst%3DR.%26aulast%3DRatliff%26aufirst%3DT.%2BL.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DC.-D.%26atitle%3DProtein%2520arginine%2520methyltransferase%25205%2520functions%2520as%2520an%2520epigenetic%2520activator%2520of%2520the%2520androgen%2520receptor%2520to%2520promote%2520prostate%2520cancer%2520cell%2520growth%26jtitle%3DOncogene%26date%3D2017%26volume%3D36%26spage%3D1223%26epage%3D1231%26doi%3D10.1038%2Fonc.2016.287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, J.</span></span> <span> </span><span class="NLM_article-title">Targeting methyltransferase PRMT5 eliminates leukemia stem cells in chronic myelogenous leukemia</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">3961</span>– <span class="NLM_lpage">3980</span>, <span class="refDoi"> DOI: 10.1172/jci85239</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1172%2FJCI85239" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=27643437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A280%3ADC%252BC2svhs12juw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2016&pages=3961-3980&author=Y.+Jinauthor=J.+Zhouauthor=F.+Xuauthor=B.+Jinauthor=L.+Cuiauthor=Y.+Wangauthor=X.+Duauthor=J.+Liauthor=P.+Liauthor=R.+Renauthor=J.+Pan&title=Targeting+methyltransferase+PRMT5+eliminates+leukemia+stem+cells+in+chronic+myelogenous+leukemia&doi=10.1172%2Fjci85239"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting methyltransferase PRMT5 eliminates leukemia stem cells in chronic myelogenous leukemia</span></div><div class="casAuthors">Jin Yanli; Zhou Jingfeng; Xu Fang; Jin Bei; Cui Lijing; Wang Yun; Du Xin; Li Juan; Li Peng; Ren Ruibao; Pan Jingxuan</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3961-3980</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Imatinib-insensitive leukemia stem cells (LSCs) are believed to be responsible for resistance to BCR-ABL tyrosine kinase inhibitors and relapse of chronic myelogenous leukemia (CML).  Identifying therapeutic targets to eradicate CML LSCs may be a strategy to cure CML.  In the present study, we discovered a positive feedback loop between BCR-ABL and protein arginine methyltransferase 5 (PRMT5) in CML cells.  Overexpression of PRMT5 was observed in human CML LSCs.  Silencing PRMT5 with shRNA or blocking PRMT5 methyltransferase activity with the small-molecule inhibitor PJ-68 reduced survival, serial replating capacity, and long-term culture-initiating cells (LTC-ICs) in LSCs from CML patients.  Further, PRMT5 knockdown or PJ-68 treatment dramatically prolonged survival in a murine model of retroviral BCR-ABL-driven CML and impaired the in vivo self-renewal capacity of transplanted CML LSCs.  PJ-68 also inhibited long-term engraftment of human CML CD34+ cells in immunodeficient mice.  Moreover, inhibition of PRMT5 abrogated the Wnt/β-catenin pathway in CML CD34+ cells by depleting dishevelled homolog 3 (DVL3).  This study suggests that epigenetic methylation modification on histone protein arginine residues is a regulatory mechanism to control self-renewal of LSCs and indicates that PRMT5 may represent a potential therapeutic target against LSCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTTlS-7r3kmkRJ5E29YNGvQfW6udTcc2eb0MmtVuBx18Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2svhs12juw%253D%253D&md5=906aacd5086992e34c0a08f271e3e56a</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1172%2FJCI85239&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI85239%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DJin%26aufirst%3DB.%26aulast%3DCui%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DRen%26aufirst%3DR.%26aulast%3DPan%26aufirst%3DJ.%26atitle%3DTargeting%2520methyltransferase%2520PRMT5%2520eliminates%2520leukemia%2520stem%2520cells%2520in%2520chronic%2520myelogenous%2520leukemia%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2016%26volume%3D126%26spage%3D3961%26epage%3D3980%26doi%3D10.1172%2Fjci85239" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomie, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seibold, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumball, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capitini, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rui, L.</span></span> <span> </span><span class="NLM_article-title">PRMT5 is upregulated by B-cell receptor signaling and forms a positive-feedback loop with PI3K/AKT in lymphoma cells</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">2898</span>– <span class="NLM_lpage">2911</span>, <span class="refDoi"> DOI: 10.1038/s41375-019-0489-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fs41375-019-0489-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=31123343" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVyqs7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=2898-2911&author=F.+Zhuauthor=H.+Guoauthor=P.+D.+Batesauthor=S.+Zhangauthor=H.+Zhangauthor=K.+J.+Nomieauthor=Y.+Liauthor=L.+Luauthor=K.+R.+Seiboldauthor=F.+Wangauthor=I.+Rumballauthor=H.+Cameronauthor=N.+M.+Hoangauthor=D.+T.+Yangauthor=W.+Xuauthor=L.+Zhangauthor=M.+Wangauthor=C.+M.+Capitiniauthor=L.+Rui&title=PRMT5+is+upregulated+by+B-cell+receptor+signaling+and+forms+a+positive-feedback+loop+with+PI3K%2FAKT+in+lymphoma+cells&doi=10.1038%2Fs41375-019-0489-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT5 is upregulated by B-cell receptor signaling and forms a positive-feedback loop with PI3K/AKT in lymphoma cells</span></div><div class="casAuthors">Zhu, Fen; Guo, Hui; Bates, Paul D.; Zhang, Shanxiang; Zhang, Hui; Nomie, Krystle J.; Li, Yangguang; Lu, Li; Seibold, Kaitlyn R.; Wang, Fangyu; Rumball, Ian; Cameron, Hunter; Hoang, Nguyet M.; Yang, David T.; Xu, Wei; Zhang, Liang; Wang, Michael; Capitini, Christian M.; Rui, Lixin</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2898-2911</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">PRMT5, which regulates gene expression by sym. dimethylation of histones and non-histone target proteins, is overexpressed and plays a pathogenic role in many cancers.  In diffuse large B cell lymphoma (DLBCL), the mechanisms of PRMT5 dysregulation and its role in lymphomagenesis remain largely unknown.  Here we demonstrate that B cell receptor (BCR) signaling regulates PRMT5 expression in DLBCL cells.  Immunohistochem. anal. reveals elevated levels of PRMT5 expression in DLBCL cases and in germinal center (GC) B cells when compared to naive B cells.  PRMT5 can be induced in naive B cells by BCR stimulation.  We discovered that BTK-NF-κB signaling induces PRMT5 transcription in activated B cell-like (ABC) DLBCL cells while BCR downstream PI3K-AKT-MYC signaling upregulates PRMT5 expression in both ABC and GCB DLBCL cells.  PRMT5 inhibition inhibits the growth of DLBCL cells in vitro and patient derived xenografts.  Genomic and biochem. anal. demonstrate that PRMT5 promotes cell cycle progression and activates PI3K-AKT signaling, suggesting a feedback regulatory mechanism to enhance cell survival and proliferation.  Co-targeting PRMT5 and AKT by their specific inhibitors is lethal to DLBCL cell lines and primary cancer cells.  Therefore, this study provides a mechanistic rationale for clin. trials to evaluate PRMT5 and AKT inhibitors for DLBCL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3qt1S5OemMLVg90H21EOLACvtfcHk0lizstGhtt-SuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVyqs7nN&md5=b4ad5f3eb84cf8b8fa9a05828db51128</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fs41375-019-0489-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41375-019-0489-6%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DF.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DBates%26aufirst%3DP.%2BD.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DNomie%26aufirst%3DK.%2BJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DSeibold%26aufirst%3DK.%2BR.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DRumball%26aufirst%3DI.%26aulast%3DCameron%26aufirst%3DH.%26aulast%3DHoang%26aufirst%3DN.%2BM.%26aulast%3DYang%26aufirst%3DD.%2BT.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DCapitini%26aufirst%3DC.%2BM.%26aulast%3DRui%26aufirst%3DL.%26atitle%3DPRMT5%2520is%2520upregulated%2520by%2520B-cell%2520receptor%2520signaling%2520and%2520forms%2520a%2520positive-feedback%2520loop%2520with%2520PI3K%252FAKT%2520in%2520lymphoma%2520cells%26jtitle%3DLeukemia%26date%3D2019%26volume%3D33%26spage%3D2898%26epage%3D2911%26doi%3D10.1038%2Fs41375-019-0489-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koh, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bezzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, D. H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ang, W. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teo, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, F. P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Haddawi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabò, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amati, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wee, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span> <span> </span><span class="NLM_article-title">MYC regulates the core pre-mRNA splicing machinery as an essential step in lymphomagenesis</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>523</i></span>,  <span class="NLM_fpage">96</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1038/nature14351</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fnature14351" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=25970242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFaitrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=523&publication_year=2015&pages=96-100&author=C.+M.+Kohauthor=M.+Bezziauthor=D.+H.+P.+Lowauthor=W.+X.+Angauthor=S.+X.+Teoauthor=F.+P.+H.+Gayauthor=M.+Al-Haddawiauthor=S.+Y.+Tanauthor=M.+Osatoauthor=A.+Sab%C3%B2author=B.+Amatiauthor=K.+B.+Weeauthor=E.+Guccione&title=MYC+regulates+the+core+pre-mRNA+splicing+machinery+as+an+essential+step+in+lymphomagenesis&doi=10.1038%2Fnature14351"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">MYC regulates the core pre-mRNA splicing machinery as an essential step in lymphomagenesis</span></div><div class="casAuthors">Koh, Cheryl M.; Bezzi, Marco; Low, Diana H. P.; Ang, Wei Xia; Teo, Shun Xie; Gay, Florence P. H.; Al-Haddawi, Muthafar; Tan, Soo Yong; Osato, Motomi; Sabo, Arianna; Amati, Bruno; Wee, Keng Boon; Guccione, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">523</span>
        (<span class="NLM_cas:issue">7558</span>),
    <span class="NLM_cas:pages">96-100</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Deregulated expression of the MYC transcription factor occurs in most human cancers and correlates with high proliferation, reprogrammed cellular metab. and poor prognosis.  Overexpressed MYC binds to virtually all active promoters within a cell, although with different binding affinities, and modulates the expression of distinct subsets of genes.  However, the crit. effectors of MYC in tumorigenesis remain largely unknown.  Here the authors show that during lymphomagenesis in Eμ-myc transgenic mice, MYC directly upregulates the transcription of the core small nuclear ribonucleoprotein particle assembly genes, including Prmt5, an arginine methyltransferase that methylates Sm proteins.  This coordinated regulatory effect is crit. for the core biogenesis of small nuclear ribonucleoprotein particles, effective pre-messenger-RNA splicing, cell survival and proliferation.  The authors' results demonstrate that MYC maintains the splicing fidelity of exons with a weak 5' donor site.  Addnl., the authors identify pre-messenger-RNAs that are particularly sensitive to the perturbation of the MYC-PRMT5 axis, resulting in either intron retention (for example, Dvl1) or exon skipping (for example, Atr, Ep400).  Using antisense oligonucleotides, the authors demonstrate the contribution of these splicing defects to the anti-proliferative/apoptotic phenotype obsd. in PRMT5-depleted Eμ-myc B cells.  The authors conclude that, in addn. to its well-documented oncogenic functions in transcription and translation, MYC also safeguards proper pre-messenger-RNA splicing as an essential step in lymphomagenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprbG6Q1B6f8rVg90H21EOLACvtfcHk0lizstGhtt-SuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFaitrrO&md5=4fd14f9728e96c73f6ded4a0e0b1d0fc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnature14351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14351%26sid%3Dliteratum%253Aachs%26aulast%3DKoh%26aufirst%3DC.%2BM.%26aulast%3DBezzi%26aufirst%3DM.%26aulast%3DLow%26aufirst%3DD.%2BH.%2BP.%26aulast%3DAng%26aufirst%3DW.%2BX.%26aulast%3DTeo%26aufirst%3DS.%2BX.%26aulast%3DGay%26aufirst%3DF.%2BP.%2BH.%26aulast%3DAl-Haddawi%26aufirst%3DM.%26aulast%3DTan%26aufirst%3DS.%2BY.%26aulast%3DOsato%26aufirst%3DM.%26aulast%3DSab%25C3%25B2%26aufirst%3DA.%26aulast%3DAmati%26aufirst%3DB.%26aulast%3DWee%26aufirst%3DK.%2BB.%26aulast%3DGuccione%26aufirst%3DE.%26atitle%3DMYC%2520regulates%2520the%2520core%2520pre-mRNA%2520splicing%2520machinery%2520as%2520an%2520essential%2520step%2520in%2520lymphomagenesis%26jtitle%3DNature%26date%3D2015%26volume%3D523%26spage%3D96%26epage%3D100%26doi%3D10.1038%2Fnature14351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alinari, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lustberg, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katherine Martin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordero-Nieves, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banasavadi-Siddegowda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Virk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnholtz-Sloan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakravarti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowicki, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapalombella, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Datta, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haseley, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patton, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcucci, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuovo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrd, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiocca, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sif, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span> <span> </span><span class="NLM_article-title">Genetic validation of the protein arginine methyltransferase PRMT5 as a candidate therapeutic target in glioblastoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">1752</span>– <span class="NLM_lpage">1765</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-13-0884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1158%2F0008-5472.CAN-13-0884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=24453002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktF2murY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=1752-1765&author=F.+Yanauthor=L.+Alinariauthor=M.+E.+Lustbergauthor=L.+Katherine+Martinauthor=H.+M.+Cordero-Nievesauthor=Y.+Banasavadi-Siddegowdaauthor=S.+Virkauthor=J.+Barnholtz-Sloanauthor=E.+H.+Bellauthor=J.+Wojtonauthor=N.+K.+Jacobauthor=A.+Chakravartiauthor=M.+O.+Nowickiauthor=X.+Wuauthor=R.+Lapalombellaauthor=J.+Dattaauthor=B.+Yuauthor=K.+Gordonauthor=A.+Haseleyauthor=J.+T.+Pattonauthor=P.+L.+Smithauthor=J.+Ryuauthor=X.+Zhangauthor=X.+Moauthor=G.+Marcucciauthor=G.+Nuovoauthor=C.-H.+Kwonauthor=J.+C.+Byrdauthor=E.+A.+Chioccaauthor=C.+Liauthor=S.+Sifauthor=S.+Jacobauthor=S.+Lawlerauthor=B.+Kaurauthor=R.+A.+Baiocchi&title=Genetic+validation+of+the+protein+arginine+methyltransferase+PRMT5+as+a+candidate+therapeutic+target+in+glioblastoma&doi=10.1158%2F0008-5472.can-13-0884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic Validation of the Protein Arginine Methyltransferase PRMT5 as a Candidate Therapeutic Target in Glioblastoma</span></div><div class="casAuthors">Yan, Fengting; Alinari, Lapo; Lustberg, Mark E.; Martin, Ludmila Katherine; Cordero-Nieves, Hector M.; Banasavadi-Siddegowda, Yeshavanth; Virk, Selene; Barnholtz-Sloan, Jill; Bell, Erica Hlavin; Wojton, Jeffrey; Jacob, Naduparambil K.; Chakravarti, Arnab; Nowicki, Michal O.; Wu, Xin; Lapalombella, Rosa; Datta, Jharna; Yu, Bo; Gordon, Kate; Haseley, Amy; Patton, John T.; Smith, Porsha L.; Ryu, John; Zhang, Xiaoli; Mo, Xiaokui; Marcucci, Guido; Nuovo, Gerard; Kwon, Chang-Hyuk; Byrd, John C.; Chiocca, E. Antonio; Li, Chenglong; Sif, Said; Jacob, Samson; Lawler, Sean; Kaur, Balveen; Baiocchi, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1752-1765</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Glioblastoma is the most common and aggressive histol. subtype of brain cancer with poor outcomes and limited treatment options.  Here, we report the selective overexpression of the protein arginine methyltransferase PRMT5 as a novel candidate theranostic target in this disease.  PRMT5 silences the transcription of regulatory genes by catalyzing sym. dimethylation of arginine residues on histone tails.  PRMT5 overexpression in patient-derived primary tumors and cell lines correlated with cell line growth rate and inversely with overall patient survival.  Genetic attenuation of PRMT5 led to cell-cycle arrest, apoptosis, and loss of cell migratory activity.  Cell death was p53-independent but caspase-dependent and enhanced with temozolomide, a chemotherapeutic agent used as a present std. of care.  Global gene profiling and chromatin immunopptn. identified the tumor suppressor ST7 as a key gene silenced by PRMT5.  Diminished ST7 expression was assocd. with reduced patient survival.  PRMT5 attenuation limited PRMT5 recruitment to the ST7 promoter, led to restored expression of ST7 and cell growth inhibition.  Finally, PRMT5 attenuation enhanced glioblastoma cell survival in a mouse xenograft model of aggressive glioblastoma.  Together, our findings defined PRMT5 as a candidate prognostic factor and therapeutic target in glioblastoma, offering a preclin. justification for targeting PRMT5-driven oncogenic pathways in this deadly disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtN9wQGNGjE7Vg90H21EOLACvtfcHk0lietOtjkSYBOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktF2murY%253D&md5=c38d456f2a4a7ffeab43a6fe5341e63c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-0884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-0884%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DF.%26aulast%3DAlinari%26aufirst%3DL.%26aulast%3DLustberg%26aufirst%3DM.%2BE.%26aulast%3DKatherine%2BMartin%26aufirst%3DL.%26aulast%3DCordero-Nieves%26aufirst%3DH.%2BM.%26aulast%3DBanasavadi-Siddegowda%26aufirst%3DY.%26aulast%3DVirk%26aufirst%3DS.%26aulast%3DBarnholtz-Sloan%26aufirst%3DJ.%26aulast%3DBell%26aufirst%3DE.%2BH.%26aulast%3DWojton%26aufirst%3DJ.%26aulast%3DJacob%26aufirst%3DN.%2BK.%26aulast%3DChakravarti%26aufirst%3DA.%26aulast%3DNowicki%26aufirst%3DM.%2BO.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DLapalombella%26aufirst%3DR.%26aulast%3DDatta%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DGordon%26aufirst%3DK.%26aulast%3DHaseley%26aufirst%3DA.%26aulast%3DPatton%26aufirst%3DJ.%2BT.%26aulast%3DSmith%26aufirst%3DP.%2BL.%26aulast%3DRyu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DMo%26aufirst%3DX.%26aulast%3DMarcucci%26aufirst%3DG.%26aulast%3DNuovo%26aufirst%3DG.%26aulast%3DKwon%26aufirst%3DC.-H.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DChiocca%26aufirst%3DE.%2BA.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DSif%26aufirst%3DS.%26aulast%3DJacob%26aufirst%3DS.%26aulast%3DLawler%26aufirst%3DS.%26aulast%3DKaur%26aufirst%3DB.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26atitle%3DGenetic%2520validation%2520of%2520the%2520protein%2520arginine%2520methyltransferase%2520PRMT5%2520as%2520a%2520candidate%2520therapeutic%2520target%2520in%2520glioblastoma%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26spage%3D1752%26epage%3D1765%26doi%3D10.1158%2F0008-5472.can-13-0884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan-Penebre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuplast, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stickland, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurse, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oza, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K. W.</span></span> <span> </span><span class="NLM_article-title">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">432</span>– <span class="NLM_lpage">437</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1810</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fnchembio.1810" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=25915199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=432-437&author=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=C.+R.+Majerauthor=P.+A.+Boriack-Sjodinauthor=T.+J.+Wigleauthor=L.+D.+Johnstonauthor=N.+Riouxauthor=M.+J.+Munchhofauthor=L.+Jinauthor=S.+L.+Jacquesauthor=K.+A.+Westauthor=T.+Lingarajauthor=K.+Sticklandauthor=S.+A.+Ribichauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+M.+Pollockauthor=J.+J.+Smithauthor=O.+Barbashauthor=M.+Pappalardiauthor=T.+F.+Hoauthor=K.+Nurseauthor=K.+P.+Ozaauthor=K.+T.+Gallagherauthor=R.+Krugerauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworthauthor=K.+W.+Duncan&title=A+selective+inhibitor+of+PRMT5+with+in+vivo+and+in+vitro+potency+in+MCL+models&doi=10.1038%2Fnchembio.1810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span></div><div class="casAuthors">Chan-Penebre, Elayne; Kuplast, Kristy G.; Majer, Christina R.; Boriack-Sjodin, P. Ann; Wigle, Tim J.; Johnston, L. Danielle; Rioux, Nathalie; Munchhof, Michael J.; Jin, Lei; Jacques, Suzanne L.; West, Kip A.; Lingaraj, Trupti; Stickland, Kimberly; Ribich, Scott A.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Pollock, Roy M.; Smith, Jesse J.; Barbash, Olena; Pappalardi, Melissa; Ho, Thau F.; Nurse, Kelvin; Oza, Khyati P.; Gallagher, Kathleen T.; Kruger, Ryan; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard; Duncan, Kenneth W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">432-437</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase-5 (PRMT5) is reported to have a role in diverse cellular processes, including tumorigenesis, and its overexpression is obsd. in cell lines and primary patient samples derived from lymphomas, particularly mantle cell lymphoma (MCL).  Here we describe the identification and characterization of a potent and selective inhibitor of PRMT5 with antiproliferative effects in both in vitro and in vivo models of MCL.  EPZ015666 (GSK3235025) is an orally available inhibitor of PRMT5 enzymic activity in biochem. assays with a half-maximal inhibitory concn. (IC50) of 22 nM and broad selectivity against a panel of other histone methyltransferases.  Treatment of MCL cell lines with EPZ015666 led to inhibition of SmD3 methylation and cell death, with IC50 values in the nanomolar range.  Oral dosing with EPZ015666 demonstrated dose-dependent antitumor activity in multiple MCL xenograft models.  EPZ015666 represents a validated chem. probe for further study of PRMT5 biol. and arginine methylation in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTtcvcqvOJLVg90H21EOLACvtfcHk0lietOtjkSYBOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D&md5=bf0686e1c2d9d96ac5deef22c0bf80b7</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1810%26sid%3Dliteratum%253Aachs%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%2BA.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DStickland%26aufirst%3DK.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DPappalardi%26aufirst%3DM.%26aulast%3DHo%26aufirst%3DT.%2BF.%26aulast%3DNurse%26aufirst%3DK.%26aulast%3DOza%26aufirst%3DK.%2BP.%26aulast%3DGallagher%26aufirst%3DK.%2BT.%26aulast%3DKruger%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520PRMT5%2520with%2520in%2520vivo%2520and%2520in%2520vitro%2520potency%2520in%2520MCL%2520models%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D432%26epage%3D437%26doi%3D10.1038%2Fnchembio.1810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Potent, selective, and cell active protein arginine methyltransferase 5 (PRMT5) inhibitor developed by structure-based virtual screening and hit optimization</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">6289</span>– <span class="NLM_lpage">6304</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00587</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00587" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKitr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=6289-6304&author=R.+Maoauthor=J.+Shaoauthor=K.+Zhuauthor=Y.+Zhangauthor=H.+Dingauthor=C.+Zhangauthor=Z.+Shiauthor=H.+Jiangauthor=D.+Sunauthor=W.+Duanauthor=C.+Luo&title=Potent%2C+selective%2C+and+cell+active+protein+arginine+methyltransferase+5+%28PRMT5%29+inhibitor+developed+by+structure-based+virtual+screening+and+hit+optimization&doi=10.1021%2Facs.jmedchem.7b00587"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Potent, Selective, and Cell Active Protein Arginine Methyltransferase 5 (PRMT5) Inhibitor Developed by Structure-Based Virtual Screening and Hit Optimization</span></div><div class="casAuthors">Mao, Ruifeng; Shao, Jingwei; Zhu, Kongkai; Zhang, Yuanyuan; Ding, Hong; Zhang, Chenhua; Shi, Zhe; Jiang, Hualiang; Sun, Dequn; Duan, Wenhu; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6289-6304</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Abstr. PRMT5 plays important roles in diverse cellular processes and is upregulated in several human malignancies.  Besides, PRMT5 has been validated as anticancer target in mantle cell lymphoma.  In this study, the authors found a potent and selective PRMT5 inhibitor by performing structure-based virtual screening and hit optimization.  The identified compd. I (IC50 = 0.33 μM) exhibited a broad selectivity against a panel of other methyltransferases.  The direct binding of I to PRMT5 was validated by surface plasmon resonance expts., with a Kd of 0.987 μM.  Kinetic expts. indicated that I was a SAM competitive inhibitor other than substrate.  In addn., I showed selective antiproliferative effects against MV4-11 cells, and further studies indicated that the mechanism of cellular antitumor activity was due to inhibition of PRMT5 mediated SmD3 methylation. I may represent a promising lead compd. to understand more about PRMT5 and potentially assist the development of treatments for leukemia indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI1jnM_kEhRrVg90H21EOLACvtfcHk0lgRNk4F7Vmaog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKitr7E&md5=5e3687166e2e6b5558409806e3c4b9e5</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00587%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DR.%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DDuan%26aufirst%3DW.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DPotent%252C%2520selective%252C%2520and%2520cell%2520active%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520inhibitor%2520developed%2520by%2520structure-based%2520virtual%2520screening%2520and%2520hit%2520optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D6289%26epage%3D6304%26doi%3D10.1021%2Facs.jmedchem.7b00587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of selective inhibitors of protein arginine methyltransferase 5 by docking-based virtual screening</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">3648</span>– <span class="NLM_lpage">3661</span>, <span class="refDoi"> DOI: 10.1039/c7ob00070g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1039%2FC7OB00070G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=28397890" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltl2mtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=3648-3661&author=Y.+Yeauthor=B.+Zhangauthor=R.+Maoauthor=C.+Zhangauthor=Y.+Wangauthor=J.+Xingauthor=Y.-C.+Liuauthor=X.+Luoauthor=H.+Dingauthor=Y.+Yangauthor=B.+Zhouauthor=H.+Jiangauthor=K.+Chenauthor=C.+Luoauthor=M.+Zheng&title=Discovery+and+optimization+of+selective+inhibitors+of+protein+arginine+methyltransferase+5+by+docking-based+virtual+screening&doi=10.1039%2Fc7ob00070g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and optimization of selective inhibitors of protein arginine methyltransferase 5 by docking-based virtual screening</span></div><div class="casAuthors">Ye, Yan; Zhang, Bidong; Mao, Ruifeng; Zhang, Chenhua; Wang, Yulan; Xing, Jing; Liu, Yu-Chih; Luo, Xiaomin; Ding, Hong; Yang, Yaxi; Zhou, Bing; Jiang, Hualiang; Chen, Kaixian; Luo, Cheng; Zheng, Mingyue</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3648-3661</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is a type II PRMT enzyme crit. for diverse cellular processes and different types of cancers.  Many efforts have been made to discover novel scaffold PRMT5 inhibitors.  Herein, the authors report the discovery of DC_P33 as a hit compd. of PRMT5 inhibitor, identified by mol. docking based virtual screening and 3H-labeled radioactive methylation assays.  Structure-activity relationship (SAR) anal. was performed on the analogs of DC_P33 and then structural modifications were done to improve its activity.  Among the derivs., the compd. DC_C01 displayed an IC50 value of 2.8 μM, and good selectivity toward PRMT1, EZH2 and DNMT3A.  Moreover, DC_C01 exhibited anti-proliferation activities against Z-138, Maver-1, and Jeko-1 cancer cells with EC50 values of 12 μM, 12 μM, and 10.5 μM, resp.  Taken together, these results contribute to the development of specific inhibitors against PRMT5 and cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkWfdAd-COtbVg90H21EOLACvtfcHk0lgRNk4F7Vmaog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltl2mtr0%253D&md5=ead3b89fa64875e2dbf47c3a99714564</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1039%2FC7OB00070G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7OB00070G%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DMao%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXing%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DY.-C.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DZheng%26aufirst%3DM.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520selective%2520inhibitors%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520by%2520docking-based%2520virtual%2520screening%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26spage%3D3648%26epage%3D3661%26doi%3D10.1039%2Fc7ob00070g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan-Penebre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuplast, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter
Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span> <span> </span><span class="NLM_article-title">Structure and property guided design in the identification of PRMT5 tool compound EPZ015666</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">162</span>– <span class="NLM_lpage">166</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00380</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00380" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGnsr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=162-166&author=K.+W.+Duncanauthor=N.+Riouxauthor=P.+A.+Boriack-Sjodinauthor=M.+J.+Munchhofauthor=L.+A.+Reiterauthor=C.+R.+Majerauthor=L.+Jinauthor=L.+D.+Johnstonauthor=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=M.+Porter%0AScottauthor=R.+M.+Pollockauthor=N.+J.+Watersauthor=J.+J.+Smithauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworth&title=Structure+and+property+guided+design+in+the+identification+of+PRMT5+tool+compound+EPZ015666&doi=10.1021%2Facsmedchemlett.5b00380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and Property Guided Design in the Identification of PRMT5 Tool Compound EPZ015666</span></div><div class="casAuthors">Duncan, Kenneth W.; Rioux, Nathalie; Boriack-Sjodin, P. Ann; Munchhof, Michael J.; Reiter, Lawrence A.; Majer, Christina R.; Jin, Lei; Johnston, L. Danielle; Chan-Penebre, Elayne; Kuplast, Kristy G.; Porter Scott, Margaret; Pollock, Roy M.; Waters, Nigel J.; Smith, Jesse J.; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">162-166</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent publication of a potent and selective inhibitor of protein methyltransferase 5 (PRMT5) provides the scientific community with in vivo-active tool compd. I (EPZ015666, GSK3235025) to probe the underlying pharmacol. of this key enzyme.  Herein, the authors report the design and optimization strategies employed on an initial hit compd. with poor in vitro clearance to yield in vivo tool compd. I and an addnl. potent in vitro tool mol. II (GSK3203591).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2V7LiQgVZjrVg90H21EOLACvtfcHk0lgRNk4F7Vmaog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGnsr3L&md5=e7ac92c2d985da4994e9bb959adededf</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00380%26sid%3Dliteratum%253Aachs%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DReiter%26aufirst%3DL.%2BA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DPorter%2BScott%26aufirst%3DM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26atitle%3DStructure%2520and%2520property%2520guided%2520design%2520in%2520the%2520identification%2520of%2520PRMT5%2520tool%2520compound%2520EPZ015666%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D162%26epage%3D166%26doi%3D10.1021%2Facsmedchemlett.5b00380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a dual PRMT5-PRMT7 inhibitor</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">408</span>– <span class="NLM_lpage">412</span>, <span class="refDoi"> DOI: 10.1021/ml500467h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500467h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjs1Ggu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=408-412&author=D.+Smilauthor=M.+S.+Eramauthor=F.+Liauthor=S.+Kennedyauthor=M.+M.+Szewczykauthor=P.+J.+Brownauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=M.+Schapira&title=Discovery+of+a+dual+PRMT5-PRMT7+inhibitor&doi=10.1021%2Fml500467h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Dual PRMT5-PRMT7 Inhibitor</span></div><div class="casAuthors">Smil, David; Eram, Mohammad S.; Li, Fengling; Kennedy, Steven; Szewczyk, Magdalena M.; Brown, Peter J.; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Vedadi, Masoud; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">408-412</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The protein arginine methyltransferases PRMT7 and PRMT5, resp., monomethylate and sym. dimethylate arginine side-chains of proteins involved in diverse cellular mechanisms, including chromatin-mediated control of gene transcription, splicing, and the RAS to ERK transduction cascade.  It is believed that PRMT5 and PRMT7 act in conjunction to methylate their substrates, and genetic deletions support the notion that these enzymes derepress cell proliferation and migration in cancer.  Using available structures of PRMT5, we designed DS-437, a PRMT5 inhibitor with an IC50 value of 6 μM against both PRMT5 and PRMT7 that is inactive against 29 other human protein-, DNA-, and RNA-methyltransferases and inhibits sym. dimethylation of PRMT5 substrates in cells.  This compd. behaves as a cofactor competitor and represents a valid scaffold to interrogate the potential of the PRMT5-PRMT7 axis as a target for therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9J078gEcjRLVg90H21EOLACvtfcHk0ljNJr0kjfZa6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjs1Ggu7k%253D&md5=8eefe93068418c61881850d3b4d39676</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fml500467h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500467h%26sid%3Dliteratum%253Aachs%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520dual%2520PRMT5-PRMT7%2520inhibitor%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D408%26epage%3D412%26doi%3D10.1021%2Fml500467h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prabhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demir, Ö.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaro, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neil, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span> <span> </span><span class="NLM_article-title">Development of an AlphaLISA high throughput technique to screen for small molecule inhibitors targeting protein arginine methyltransferases</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2509</span>– <span class="NLM_lpage">2520</span>, <span class="refDoi"> DOI: 10.1039/c7mb00391a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1039%2FC7MB00391A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=29099132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslagurvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=2509-2520&author=L.+Prabhuauthor=L.+Chenauthor=H.+Weiauthor=%C3%96.+Demirauthor=A.+Safaauthor=L.+Zengauthor=R.+E.+Amaroauthor=B.+H.+O%E2%80%99Neilauthor=Z.-Y.+Zhangauthor=T.+Lu&title=Development+of+an+AlphaLISA+high+throughput+technique+to+screen+for+small+molecule+inhibitors+targeting+protein+arginine+methyltransferases&doi=10.1039%2Fc7mb00391a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Development of an AlphaLISA high throughput technique to screen for small molecule inhibitors targeting protein arginine methyltransferases</span></div><div class="casAuthors">Prabhu, Lakshmi; Chen, Lan; Wei, Han; Demir, Ozlem; Safa, Ahmad; Zeng, Lifan; Amaro, Rommie E.; O.bxsolid.Neil, Bert H.; Zhang, Zhon-Yin; Lu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2509-2520</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The protein arginine methyltransferase (PRMT) family of enzymes comprises nine family members in mammals.  They catalyze arginine methylation, either monomethylation or sym./asym. dimethylation of histone and non-histone proteins.  PRMT methylation of its substrate proteins modulates cellular processes such as signal transduction, transcription, and mRNA splicing.  Recent studies have linked overexpression of PRMT5, a member of the PRMT superfamily, to oncogenesis, making it a potential target for cancer therapy.  In this study, we developed a highly sensitive (Z' score = 0.7) robotic high throughput screening (HTS) platform to discover small mol. inhibitors of PRMT5 by adapting the AlphaLISA technol.  Using biotinylated histone H4 as a substrate, and S-adenosyl-L-methionine as a Me donor, PRMT5 sym. dimethylated H4 at arginine (R) 3.  Highly specific acceptor beads for sym. dimethylated H4R3 and streptavidin-coated donor beads bound the substrate, emitting a signal that is proportional to the methyltransferase activity.  Using this powerful approach, we identified specific PRMT5 inhibitors P1608K04 and P1618J22, and further validated their efficacy and specificity for inhibiting PRMT5.  Importantly, these two compds. exhibited much more potent efficacy than the com. PRMT5 inhibitor EPZ015666 in both pancreatic and colorectal cancer cells.  Overall, our work highlights a novel, powerful, and sensitive approach to identify specific PRMT5 inhibitors.  The general principle of this HTS screening method can not only be applied to PRMT5 and the PRMT superfamily, but may also be extended to other epigenetic targets.  This approach allows us to identify compds. that inhibit the activity of their resp. targets, and screening hits like P1608K04 and P1618J22 may serve as the basis for novel drug development to treat cancer and/or other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhtJJg4OpI_bVg90H21EOLACvtfcHk0lind14Ql85DcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslagurvE&md5=6cd641ae47a2d2014da4463709746a51</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2FC7MB00391A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7MB00391A%26sid%3Dliteratum%253Aachs%26aulast%3DPrabhu%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DH.%26aulast%3DDemir%26aufirst%3D%25C3%2596.%26aulast%3DSafa%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DAmaro%26aufirst%3DR.%2BE.%26aulast%3DO%25E2%2580%2599Neil%26aufirst%3DB.%2BH.%26aulast%3DZhang%26aufirst%3DZ.-Y.%26aulast%3DLu%26aufirst%3DT.%26atitle%3DDevelopment%2520of%2520an%2520AlphaLISA%2520high%2520throughput%2520technique%2520to%2520screen%2520for%2520small%2520molecule%2520inhibitors%2520targeting%2520protein%2520arginine%2520methyltransferases%26jtitle%3DMol.%2520BioSyst.%26date%3D2017%26volume%3D13%26spage%3D2509%26epage%3D2520%26doi%3D10.1039%2Fc7mb00391a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kong, G.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Bryant, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span> <span> </span><span class="NLM_article-title">Selective small-chemical inhibitors of protein arginine methyltransferase 5 with anti-lung cancer activity</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0181601</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0181601</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1371%2Fjournal.pone.0181601" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=28806746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtV2rtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=G.-M.+Kongauthor=M.+Yuauthor=Z.+Guauthor=Z.+Chenauthor=R.-M.+Xuauthor=D.+O%E2%80%99Bryantauthor=Z.+Wang&title=Selective+small-chemical+inhibitors+of+protein+arginine+methyltransferase+5+with+anti-lung+cancer+activity&doi=10.1371%2Fjournal.pone.0181601"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Selective small-chemical inhibitors of protein arginine methyltransferase 5 with anti-lung cancer activity</span></div><div class="casAuthors">Kong, Gui-Mei; Yu, Min; Gu, Zhongping; Chen, Zhi; Xu, Rui-Ming; O'Bryant, Deon; Wang, Zhengxin</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e0181601/1-e0181601/23</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) plays crit. roles in a wide variety of biol. processes, including tumorigenesis.  By screening a library of small chem. compds., we identified eight compds. that selectively inhibit the PRMT5 enzymic activity, with IC50 values ranging from 0.1 to 6 μM.  Mol. docking simulation and site directed mutagenesis indicated that identified compds. target the substrate-binding site in PRMT5.  Treatment of lung cancer cells with identified inhibitors led to inhibition of the sym. arginine methylation of SmD3 and histones and the cellular proliferation.  Oral administration of the inhibitor demonstrated antitumor activity in a lung tumor xenograft model.  Thus, identified PRMT5-specific small-mol. inhibitors would help elucidate the biol. roles of PRMT5 and serve as lead compds. for future drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHrmbn8I9YMLVg90H21EOLACvtfcHk0lind14Ql85DcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtV2rtbk%253D&md5=0306ed1351aac8a04d286df255c6e432</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0181601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0181601%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DG.-M.%26aulast%3DYu%26aufirst%3DM.%26aulast%3DGu%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DR.-M.%26aulast%3DO%25E2%2580%2599Bryant%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DZ.%26atitle%3DSelective%2520small-chemical%2520inhibitors%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520with%2520anti-lung%2520cancer%2520activity%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0181601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonday, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortez, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grogan, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonysamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weichert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocchinfuso, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span> <span> </span><span class="NLM_article-title">LLY-283, a potent and selective inhibitor of arginine methyltransferase 5, PRMT5, with antitumor activity</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">612</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00014</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotFarurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=612-617&author=Z.+Q.+Bondayauthor=G.+S.+Cortezauthor=M.+J.+Groganauthor=S.+Antonysamyauthor=K.+Weichertauthor=W.+P.+Bocchinfusoauthor=F.+Liauthor=S.+Kennedyauthor=B.+Liauthor=M.+M.+Maderauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+S.+Eramauthor=M.+M.+Szewczykauthor=D.+Barsyte-Lovejoyauthor=M.+Vedadiauthor=E.+Guccioneauthor=R.+M.+Campbell&title=LLY-283%2C+a+potent+and+selective+inhibitor+of+arginine+methyltransferase+5%2C+PRMT5%2C+with+antitumor+activity&doi=10.1021%2Facsmedchemlett.8b00014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">LLY-283, a Potent and Selective Inhibitor of Arginine Methyltransferase 5, PRMT5, with Antitumor Activity</span></div><div class="casAuthors">Bonday, Zahid Q.; Cortez, Guillermo S.; Grogan, Michael J.; Antonysamy, Stephen; Weichert, Ken; Bocchinfuso, Wayne P.; Li, Fengling; Kennedy, Steven; Li, Binghui; Mader, Mary M.; Arrowsmith, Cheryl H.; Brown, Peter J.; Eram, Mohammad S.; Szewczyk, Magdalena M.; Barsyte-Lovejoy, Dalia; Vedadi, Masoud; Guccione, Ernesto; Campbell, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">612-617</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is a type II arginine methyltransferase that catalyzes the formation of sym. dimethylarginine in a no. of nuclear and cytoplasmic proteins.  Although the cellular functions of PRMT5 have not been fully unraveled, it has been implicated in a no. of cellular processes like RNA processing, signal transduction and transcriptional regulation.  PRMT5 is ubiquitously expressed in most tissues and its expression has been shown to be elevated in several cancers including breast cancer, gastric cancer, glioblastoma and lymphoma.  Here we describe the identification and characterization of a novel and selective PRMT5 inhibitor with potent in vitro and in vivo activity.  Compd. 1 (also called LLY-283) inhibited PRMT5 enzymic activity in vitro and in cells with IC50 of 22 ± 3 nM and 25 ± 1 nM, resp. while its diastereomer, compd. 2 (also called LLY-284) was much less active.  Compd. 1 also showed antitumor activity in mouse xenografts when dosed orally and can serve as an excellent probe mol. for understanding the biol. function of PRMT5 in normal and cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYIq2FC-EskrVg90H21EOLACvtfcHk0lind14Ql85DcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotFarurc%253D&md5=93517c34cf9f99d76730a538f594ddd7</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00014%26sid%3Dliteratum%253Aachs%26aulast%3DBonday%26aufirst%3DZ.%2BQ.%26aulast%3DCortez%26aufirst%3DG.%2BS.%26aulast%3DGrogan%26aufirst%3DM.%2BJ.%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DWeichert%26aufirst%3DK.%26aulast%3DBocchinfuso%26aufirst%3DW.%2BP.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26atitle%3DLLY-283%252C%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520arginine%2520methyltransferase%25205%252C%2520PRMT5%252C%2520with%2520antitumor%2520activity%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D612%26epage%3D617%26doi%3D10.1021%2Facsmedchemlett.8b00014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Z.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, C.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span> <span> </span><span class="NLM_article-title">Discovery of new potent protein arginine methyltransferase 5 (PRMT5) inhibitors by assembly of key pharmacophores from known inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">3693</span>– <span class="NLM_lpage">3699</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.10.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1016%2Fj.bmcl.2018.10.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=30366617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVSlsLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=3693-3699&author=K.+Zhuauthor=J.-L.+Songauthor=H.-R.+Taoauthor=Z.-Q.+Chengauthor=C.-S.+Jiangauthor=H.+Zhang&title=Discovery+of+new+potent+protein+arginine+methyltransferase+5+%28PRMT5%29+inhibitors+by+assembly+of+key+pharmacophores+from+known+inhibitors&doi=10.1016%2Fj.bmcl.2018.10.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of new potent protein arginine methyltransferase 5 (PRMT5) inhibitors by assembly of key pharmacophores from known inhibitors</span></div><div class="casAuthors">Zhu, Kongkai; Song, Jia-Li; Tao, Hong-Rui; Cheng, Zhi-Qiang; Jiang, Cheng-Shi; Zhang, Hua</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">23-24</span>),
    <span class="NLM_cas:pages">3693-3699</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is an epigenetics related enzyme that has been validated as a promising therapeutic target for human cancer.  Up to now, two small mol. PRMT5 inhibitors has been put into phase I clin. trial.  In the present study, a series of candidate mols. were designed by combining key pharmacophores of formerly reported PRMT5 inhibitors.  The in vitro PRMT5 inhibitory testing of compd. 4b14 revealed an IC50 of 2.71 μM, exhibiting high selectivity over PRMT1 and PRMT4 (>70-fold selective).  As expected, 4b14 exhibited potent anti-proliferative activity against a panel of leukemia and lymphoma cells, including MV4-11, Pfeiffer, SU-DHL-4 and KARPAS-422.  Besides, 4b14 showed significant cell cycle arrest and apoptosis-inducing effects, as well as reduced the cellular sym. arginine dimethylation level of SmD3 protein.  Finally, affinity profiling anal. indicated that hydrophobic interactions, π-π stacking and cation-π actions made the major contributions to the overall binding affinity.  This scaffold provides a new chem. template for further development of better lead compds. targeting PRMT5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUiq5gq-S5uLVg90H21EOLACvtfcHk0lgTIpOJyYYM8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVSlsLrP&md5=d397612b9684719d6696dff6362a8806</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.10.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.10.026%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DSong%26aufirst%3DJ.-L.%26aulast%3DTao%26aufirst%3DH.-R.%26aulast%3DCheng%26aufirst%3DZ.-Q.%26aulast%3DJiang%26aufirst%3DC.-S.%26aulast%3DZhang%26aufirst%3DH.%26atitle%3DDiscovery%2520of%2520new%2520potent%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520inhibitors%2520by%2520assembly%2520of%2520key%2520pharmacophores%2520from%2520known%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26spage%3D3693%26epage%3D3699%26doi%3D10.1016%2Fj.bmcl.2018.10.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, C.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span> <span> </span><span class="NLM_article-title">Identification of 5-benzylidene-2-phenylthiazolones as potent PRMT5 inhibitors by virtual screening, structural optimization and biological evaluations</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">298</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2018.08.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1016%2Fj.bioorg.2018.08.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=30172110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1WmsrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2018&pages=289-298&author=K.+Zhuauthor=H.+Taoauthor=J.-L.+Songauthor=L.+Jinauthor=Y.+Zhangauthor=J.+Liuauthor=Z.+Chenauthor=C.-S.+Jiangauthor=C.+Luoauthor=H.+Zhang&title=Identification+of+5-benzylidene-2-phenylthiazolones+as+potent+PRMT5+inhibitors+by+virtual+screening%2C+structural+optimization+and+biological+evaluations&doi=10.1016%2Fj.bioorg.2018.08.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of 5-benzylidene-2-phenylthiazolones as potent PRMT5 inhibitors by virtual screening, structural optimization and biological evaluations</span></div><div class="casAuthors">Zhu, Kongkai; Tao, Hongrui; Song, Jia-Li; Jin, Lu; Zhang, Yuanyuan; Liu, Jingqiu; Chen, Zhifeng; Jiang, Cheng-Shi; Luo, Cheng; Zhang, Hua</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">289-298</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is an epigenetics related enzyme that has been validated as an important therapeutic target for glioblastoma and mantel cell lymphoma.  In the present study, 11 novel PRMT5 inhibitors with 5-benzylidene-2-phenylthiazolone scaffold were identified by mol. docking-based virtual screening and structural optimization.  Their IC50 values against PRMT5 at enzymic level were ranging from 0.77 to 23 μM.  As expected, the top two active hits (5 and 19) showed potent anti-proliferative activity against MV4-11 cells with EC50 values lower than 10 μM and reduced the cellular sym. arginine dimethylation levels of SmD3 protein.  Besides, 5 and 19 demonstrated the mechanism of cell killing in cell cycle arrest and apoptotic effect.  The probable binding modes of the two compds. were explored and further verified by mol. dynamics simulation.  The structure-activity relationship (SAR) of this class of structures was also discussed and further demonstrated by mol. docking simulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1K2TslS6a5bVg90H21EOLACvtfcHk0lgTIpOJyYYM8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1WmsrrK&md5=734c196a4312bac7c377ad1ca1ec4d81</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2018.08.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2018.08.021%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DTao%26aufirst%3DH.%26aulast%3DSong%26aufirst%3DJ.-L.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DC.-S.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DH.%26atitle%3DIdentification%2520of%25205-benzylidene-2-phenylthiazolones%2520as%2520potent%2520PRMT5%2520inhibitors%2520by%2520virtual%2520screening%252C%2520structural%2520optimization%2520and%2520biological%2520evaluations%26jtitle%3DBioorg.%2520Chem.%26date%3D2018%26volume%3D81%26spage%3D289%26epage%3D298%26doi%3D10.1016%2Fj.bioorg.2018.08.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leal, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shetty, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaddi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luengo, J. I.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent and selective covalent protein arginine methyltransferase 5 (PRMT5) inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1033</span>– <span class="NLM_lpage">1038</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00074</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00074" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A280%3ADC%252BB3MzotVeltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1033-1038&author=H.+Linauthor=M.+Wangauthor=Y.+W.+Zhangauthor=S.+Tongauthor=R.+A.+Lealauthor=R.+Shettyauthor=K.+Vaddiauthor=J.+I.+Luengo&title=Discovery+of+potent+and+selective+covalent+protein+arginine+methyltransferase+5+%28PRMT5%29+inhibitors&doi=10.1021%2Facsmedchemlett.9b00074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent and Selective Covalent Protein Arginine Methyltransferase 5 (PRMT5) Inhibitors</span></div><div class="casAuthors">Lin Hong; Wang Min; Zhang Yang W; Leal Raul A; Shetty Rupa; Vaddi Kris; Luengo Juan I; Tong Shuilong</div><div class="citationInfo"><span class="NLM_cas:title">ACS medicinal chemistry letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1033-1038</span>
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is known to symmetrically dimethylate numerous cytosolic and nuclear proteins that are involved in a variety of cellular processes.  Recent findings have revealed its potential as a cancer therapeutic target.  PRMT5 possesses a cysteine (C449) in the active site, unique to PRMT5.  Therefore, covalent PRMT5 inhibition is an attractive chemical approach.  Herein, we report an exciting discovery of a series of novel hemiaminals that under physiological conditions can be converted to aldehydes and react with C449 to form covalent adducts, which presumably undergo an unprecedented elimination to form the thiol-vinyl ethers, as indicated by electron density in the co-crystal structure of the PRMT5/MEP50 complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTdNDVSKB6iezW1zwg2yUK5fW6udTcc2eYPjPnF3-BksLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MzotVeltQ%253D%253D&md5=4b5b08dc3e3615144ece564f70bf3bd0</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00074%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DY.%2BW.%26aulast%3DTong%26aufirst%3DS.%26aulast%3DLeal%26aufirst%3DR.%2BA.%26aulast%3DShetty%26aufirst%3DR.%26aulast%3DVaddi%26aufirst%3DK.%26aulast%3DLuengo%26aufirst%3DJ.%2BI.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520selective%2520covalent%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D1033%26epage%3D1038%26doi%3D10.1021%2Facsmedchemlett.9b00074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTACs: great opportunities for academia and industry</span>. <i>Signal Transduction Targeted Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">64</span>, <span class="refDoi"> DOI: 10.1038/s41392-019-0101-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fs41392-019-0101-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=31885879" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&pages=64&author=X.+Sunauthor=H.+Gaoauthor=Y.+Yangauthor=M.+Heauthor=Y.+Wuauthor=Y.+Songauthor=Y.+Tongauthor=Y.+Rao&title=PROTACs%3A+great+opportunities+for+academia+and+industry&doi=10.1038%2Fs41392-019-0101-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">PROTACs: great opportunities for academia and industry</span></div><div class="casAuthors">Sun Xiuyun; Gao Hongying; Yang Yiqing; He Ming; Wu Yue; Song Yugang; Tong Yan; Rao Yu; Sun Xiuyun; Gao Hongying; Yang Yiqing; Rao Yu</div><div class="citationInfo"><span class="NLM_cas:title">Signal transduction and targeted therapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">64</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although many kinds of therapies are applied in the clinic, drug-resistance is a major and unavoidable problem.  Another disturbing statistic is the limited number of drug targets, which are presently only 20-25% of all protein targets that are currently being studied.  Moreover, the focus of current explorations of targets are their enzymatic functions, which ignores the functions from their scaffold moiety.  As a promising and appealing technology, PROteolysis TArgeting Chimeras (PROTACs) have attracted great attention both from academia and industry for finding available approaches to solve the above problems.  PROTACs regulate protein function by degrading target proteins instead of inhibiting them, providing more sensitivity to drug-resistant targets and a greater chance to affect the nonenzymatic functions.  PROTACs have been proven to show better selectivity compared to classic inhibitors.  PROTACs can be described as a chemical knockdown approach with rapidity and reversibility, which presents new and different biology compared to other gene editing tools by avoiding misinterpretations that arise from potential genetic compensation and/or spontaneous mutations.  PRTOACs have been widely explored throughout the world and have outperformed not only in cancer diseases, but also in immune disorders, viral infections and neurodegenerative diseases.  Although PROTACs present a very promising and powerful approach for crossing the hurdles of present drug discovery and tool development in biology, more efforts are needed to gain to get deeper insight into the efficacy and safety of PROTACs in the clinic.  More target binders and more E3 ligases applicable for developing PROTACs are waiting for exploration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQretvERU2xT-_Q9URG6Dx8fW6udTcc2ebS69b-qOiL6rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D&md5=7d9545a21e3a78cededf2977a31e8f7d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fs41392-019-0101-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-019-0101-6%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTACs%253A%2520great%2520opportunities%2520for%2520academia%2520and%2520industry%26jtitle%3DSignal%2520Transduction%2520Targeted%2520Ther.%26date%3D2019%26volume%3D4%26spage%3D64%26doi%3D10.1038%2Fs41392-019-0101-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lhcN9VJ7FwckQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: expanding the toolbox</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">949</span>– <span class="NLM_lpage">963</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0047-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fs41573-019-0047-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=31666732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=949-963&author=M.+Schapiraauthor=M.+F.+Calabreseauthor=A.+N.+Bullockauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+expanding+the+toolbox&doi=10.1038%2Fs41573-019-0047-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation: expanding the toolbox</span></div><div class="casAuthors">Schapira, Matthieu; Calabrese, Matthew F.; Bullock, Alex N.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">949-963</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review. Proteolysis-targeting chimeras (PROTACs) and related mols. that induce targeted protein degrdn. by the ubiquitin-proteasome system represent a new therapeutic modality and are the focus of great interest, owing to potential advantages over traditional occupancy-based inhibitors with respect to dosing, side effects, drug resistance and modulating 'undruggable' targets.  However, the technol. is still maturing, and the design elements for successful PROTAC-based drugs are currently being elucidated.  Importantly, fewer than 10 of the more than 600 E3 ubiquitin ligases have so far been exploited for targeted protein degrdn., and expansion of knowledge in this area is a key opportunity.  Here, we briefly discuss lessons learned about targeted protein degrdn. in chem. biol. and drug discovery and systematically review the expression profile, domain architecture and chem. tractability of human E3 ligases that could expand the toolbox for PROTAC discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwUvM27MU3WrVg90H21EOLACvtfcHk0lhcN9VJ7FwckQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM&md5=9bfd48c6c47fc1cce15129495fea7079</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0047-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0047-y%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520expanding%2520the%2520toolbox%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D949%26epage%3D963%26doi%3D10.1038%2Fs41573-019-0047-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span> <span> </span><span class="NLM_article-title">Protein knockdown technology: application of ubiquitin ligase to cancer therapy</span>. <i>Curr. Cancer Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">136</span>– <span class="NLM_lpage">146</span>, <span class="refDoi"> DOI: 10.2174/1568009616666151112122502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.2174%2F1568009616666151112122502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=26560118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFOqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=136-146&author=N.+Ohokaauthor=N.+Shibataauthor=T.+Hattoriauthor=M.+Naito&title=Protein+knockdown+technology%3A+application+of+ubiquitin+ligase+to+cancer+therapy&doi=10.2174%2F1568009616666151112122502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Knockdown Technology: Application of Ubiquitin Ligase to Cancer Therapy</span></div><div class="casAuthors">Ohoka, Nobumichi; Shibata, Norihito; Hattori, Takayuki; Naito, Mikihiko</div><div class="citationInfo"><span class="NLM_cas:title">Current Cancer Drug Targets</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">136-146</span>CODEN:
                <span class="NLM_cas:coden">CCDTB9</span>;
        ISSN:<span class="NLM_cas:issn">1568-0096</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Selective degrdn. of pathogenic proteins by small mols. in cells is a novel approach for development of therapeutic agents against various diseases, including cancer.  We and others have developed a protein knockdown technol. with a series of hybrid small compds., called SNIPERs (Specific and Nongenetic IAP-dependent Protein ERasers); and peptidic chimeric mols., called PROTACs (proteolysis-targeting chimeric mols.), which induce selective degrdn. of target proteins via the ubiquitin-proteasome pathway.  These compds. include two different ligands connected by a linker; one is a ligand for a ubiquitin ligase and the other is a ligand for the target protein, which are expected to crosslink these proteins in cells.  Theor., any cytosolic protein can be targeted for degrdn. by this technol.  To date, several SNIPERs and PROTACs against various oncogenic proteins have been developed, which specifically induce polyubiquitylation and proteasomal degrdn. of the oncogenic proteins, resulting in cell death, growth arrest, or impaired migration of cancer cells.  Thus, this protein knockdown technol. has a great potential for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmS_kGOWfeuLVg90H21EOLACvtfcHk0li4kFMhzIXjKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFOqtw%253D%253D&md5=d78f1ae70ea89843d9f946454cc95058</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.2174%2F1568009616666151112122502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568009616666151112122502%26sid%3Dliteratum%253Aachs%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DProtein%2520knockdown%2520technology%253A%2520application%2520of%2520ubiquitin%2520ligase%2520to%2520cancer%2520therapy%26jtitle%3DCurr.%2520Cancer%2520Drug%2520Targets%26date%3D2016%26volume%3D16%26spage%3D136%26epage%3D146%26doi%3D10.2174%2F1568009616666151112122502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Protac-induced protein degradation in drug discovery: breaking the rules or just making new ones?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=444-452&author=I.+Churcher&title=Protac-induced+protein+degradation+in+drug+discovery%3A+breaking+the+rules+or+just+making+new+ones%3F&doi=10.1021%2Facs.jmedchem.7b01272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span></div><div class="casAuthors">Churcher, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-452</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn., using bifunctional small mols. (Protacs) to remove specific proteins from within cells, has emerged as a novel drug discovery strategy with the potential to offer therapeutic interventions not achievable with existing approaches.  In this perspective, the brief history of the field is surveyed from a drug discovery perspective with a focus on the key advances in knowledge which have led to the definition and exemplification of protein degrdn. concepts, and their resulting applications to medicine discovery.  The approach has the potential to bring disruptive change to drug discovery - the many potential advantages and outstanding challenges which lie ahead of this technol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n8I_0MhtkbVg90H21EOLACvtfcHk0li4kFMhzIXjKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ&md5=e60f1777de8bd860c5001137d8583f1f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01272%26sid%3Dliteratum%253Aachs%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DProtac-induced%2520protein%2520degradation%2520in%2520drug%2520discovery%253A%2520breaking%2520the%2520rules%2520or%2520just%2520making%2520new%2520ones%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D444%26epage%3D452%26doi%3D10.1021%2Facs.jmedchem.7b01272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTAC technology: opportunities and challenges</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00597</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00597" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A280%3ADC%252BB383kslaitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=237-240&author=H.+Gaoauthor=X.+Sunauthor=Y.+Rao&title=PROTAC+technology%3A+opportunities+and+challenges&doi=10.1021%2Facsmedchemlett.9b00597"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC Technology: Opportunities and Challenges</span></div><div class="casAuthors">Gao Hongying; Sun Xiuyun; Rao Yu; Gao Hongying; Sun Xiuyun</div><div class="citationInfo"><span class="NLM_cas:title">ACS medicinal chemistry letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">237-240</span>
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    </div><div class="casAbstract">PROTACs-induced targeted protein degradation has emerged as a novel therapeutic strategy in drug development and attracted the favor of academic institutions, large pharmaceutical enterprises (e.g., AstraZeneca, Bayer, Novartis, Amgen, Pfizer, GlaxoSmithKline, Merck, and Boehringer Ingelheim, etc.), and biotechnology companies.  PROTACs opened a new chapter for novel drug development.  However, any new technology will face many new problems and challenges.  Perspectives on the potential opportunities and challenges of PROTACs will contribute to the research and development of new protein degradation drugs and degrader tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSj8qlWczrRKmx8TEN9K18QfW6udTcc2eY_rHFp6M0Uhrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB383kslaitA%253D%253D&md5=68988d6477e83f4ebc26d11e3b8ce5b1</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00597%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTAC%2520technology%253A%2520opportunities%2520and%2520challenges%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D237%26epage%3D240%26doi%3D10.1021%2Facsmedchemlett.9b00597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zielinska, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaaban, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Bailon, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarrold, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farnie, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smalley, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, C. C.</span></span> <span> </span><span class="NLM_article-title">PRMT5 is a critical regulator of breast cancer stem cell function via histone methylation and FOXP1 expression</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3498</span>– <span class="NLM_lpage">3513</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.11.096</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1016%2Fj.celrep.2017.11.096" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=29262329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVWlsrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2017&pages=3498-3513&author=K.+Chiangauthor=A.+E.+Zielinskaauthor=A.+M.+Shaabanauthor=M.+P.+Sanchez-Bailonauthor=J.+Jarroldauthor=T.+L.+Clarkeauthor=J.+Zhangauthor=A.+Francisauthor=L.+J.+Jonesauthor=S.+Smithauthor=O.+Barbashauthor=E.+Guccioneauthor=G.+Farnieauthor=M.+J.+Smalleyauthor=C.+C.+Davies&title=PRMT5+is+a+critical+regulator+of+breast+cancer+stem+cell+function+via+histone+methylation+and+FOXP1+expression&doi=10.1016%2Fj.celrep.2017.11.096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT5 Is a Critical Regulator of Breast Cancer Stem Cell Function via Histone Methylation and FOXP1 Expression</span></div><div class="casAuthors">Chiang, Kelly; Zielinska, Agnieszka E.; Shaaban, Abeer M.; Sanchez-Bailon, Maria Pilar; Jarrold, James; Clarke, Thomas L.; Zhang, Jingxian; Francis, Adele; Jones, Louise J.; Smith, Sally; Barbash, Olena; Guccione, Ernesto; Farnie, Gillian; Smalley, Matthew J.; Davies, Clare C.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3498-3513</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Breast cancer progression, treatment resistance, and relapse are thought to originate from a small population of tumor cells, breast cancer stem cells (BCSCs).  Identification of factors crit. for BCSC function is therefore vital for the development of therapies.  Here, we identify the arginine methyltransferase PRMT5 as a key in vitro and in vivo regulator of BCSC proliferation and self-renewal and establish FOXP1, a winged helix/forkhead transcription factor, as a crit. effector of PRMT5-induced BCSC function.  Mechanistically, PRMT5 recruitment to the FOXP1 promoter facilitates H3R2me2s, SET1 recruitment, H3K4me3, and gene expression.  Our findings are clin. significant, as PRMT5 depletion within established tumor xenografts or treatment of patient-derived BCSCs with a pre-clin. PRMT5 inhibitor substantially reduces BCSC nos.  Together, our findings highlight the importance of PRMT5 in BCSC maintenance and suggest that small-mol. inhibitors of PRMT5 or downstream targets could be an effective strategy eliminating this cancer-causing population.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiZ-akPD0aZLVg90H21EOLACvtfcHk0lhvW8q5wIw2lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVWlsrfK&md5=28b81dc86c2ea7c8c93d562723129945</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.11.096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.11.096%26sid%3Dliteratum%253Aachs%26aulast%3DChiang%26aufirst%3DK.%26aulast%3DZielinska%26aufirst%3DA.%2BE.%26aulast%3DShaaban%26aufirst%3DA.%2BM.%26aulast%3DSanchez-Bailon%26aufirst%3DM.%2BP.%26aulast%3DJarrold%26aufirst%3DJ.%26aulast%3DClarke%26aufirst%3DT.%2BL.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DFrancis%26aufirst%3DA.%26aulast%3DJones%26aufirst%3DL.%2BJ.%26aulast%3DSmith%26aufirst%3DS.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DFarnie%26aufirst%3DG.%26aulast%3DSmalley%26aufirst%3DM.%2BJ.%26aulast%3DDavies%26aufirst%3DC.%2BC.%26atitle%3DPRMT5%2520is%2520a%2520critical%2520regulator%2520of%2520breast%2520cancer%2520stem%2520cell%2520function%2520via%2520histone%2520methylation%2520and%2520FOXP1%2520expression%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D21%26spage%3D3498%26epage%3D3513%26doi%3D10.1016%2Fj.celrep.2017.11.096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1420</span>– <span class="NLM_lpage">1442</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01572</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01572" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1420-1442&author=J.+Huauthor=B.+Huauthor=M.+Wangauthor=F.+Xuauthor=B.+Miaoauthor=C.-Y.+Yangauthor=M.+Wangauthor=Z.+Liuauthor=D.+F.+Hayesauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ERD-308+as+a+highly+potent+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+estrogen+receptor+%28ER%29&doi=10.1021%2Facs.jmedchem.8b01572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span></div><div class="casAuthors">Hu, Jiantao; Hu, Biao; Wang, Mingliang; Xu, Fuming; Miao, Bukeyan; Yang, Chao-Yie; Wang, Mi; Liu, Zhaomin; Hayes, Daniel F.; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1420-1442</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The estrogen receptor (ER) is a validated target for the treatment of estrogen receptor-pos. (ER+) breast cancer.  Here, we describe the design, synthesis, and extensive structure-activity relationship (SAR) studies of small-mol. ERα degraders based on the proteolysis targeting chimeras (PROTAC) concept.  Our efforts have resulted in the discovery of highly potent and effective PROTAC ER degraders, as exemplified by ERD-308 I.  I achieves DC50 (concn. causing 50% of protein degrdn.) values of 0.17 and 0.43 nM in MCF-7 and T47D ER+ breast cancer cell lines, resp., and induces >95% of ER degrdn. at concns. as low as 5 nM in both cell lines.  Significantly, I induces more complete ER degrdn. than fulvestrant, the only approved selective ER degrader (SERD), and is more effective in inhibition of cell proliferation than fulvestrant in MCF-7 cells.  Further optimization of I may lead to a new therapy for advanced ER+ breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHRlcISYyFXLVg90H21EOLACvtfcHk0lhvW8q5wIw2lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D&md5=678e82dd460c065e2054ba18ea8af18a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01572%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DMiao%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ERD-308%2520as%2520a%2520highly%2520potent%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520estrogen%2520receptor%2520%2528ER%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1420%26epage%3D1442%26doi%3D10.1021%2Facs.jmedchem.8b01572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+K.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0lhvW8q5wIw2lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%2BK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class mitogen-activated protein kinase kinase 1/2 degrader</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">10897</span>– <span class="NLM_lpage">10911</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01528</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01528" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFOmt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=10897-10911&author=J.+Weiauthor=J.+Huauthor=L.+Wangauthor=L.+Xieauthor=M.+S.+Jinauthor=X.+Chenauthor=J.+Liuauthor=J.+Jin&title=Discovery+of+a+first-in-class+mitogen-activated+protein+kinase+kinase+1%2F2+degrader&doi=10.1021%2Facs.jmedchem.9b01528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a First-in-Class Mitogen-Activated Protein Kinase Kinase 1/2 Degrader</span></div><div class="casAuthors">Wei, Jieli; Hu, Jianping; Wang, Li; Xie, Ling; Jin, Margaret S.; Chen, Xian; Liu, Jing; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10897-10911</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MEK1 and MEK2 (also known as MAP2K1 and MAP2K2) are the "gatekeepers" of the ERK signaling output with redundant roles in controlling ERK activity.  Numerous inhibitors targeting MEK1/2 have been developed including three FDA-approved drugs.  However, acquired resistance to MEK1/2 inhibitors has been obsd. in patients and new therapeutic strategies are needed to overcome the resistance.  Here, the authors report a first-in-class degrader of MEK1/2, MS432 (I), which potently and selectively degraded MEK1 and MEK2 in a VHL E3 ligase- and proteasome-dependent manner and suppressed ERK phosphorylation in cells.  It inhibited colorectal cancer and melanoma cell proliferation much more effectively than its neg. control MS432N and its effect was phenocopied by MEK1/2 knockdown.  Compd. I was highly selective for MEK1/2 in global proteomic profiling studies.  It was also bioavailable in mice and can be used for in vivo efficacy studies.  The authors provide two well-characterized chem. tools to the biomedical community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1e5sg33oDYrVg90H21EOLACvtfcHk0lin7l3nmPF92Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFOmt7bK&md5=2c293b6b87990d6b814ec1dc88034f19</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01528%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DJin%26aufirst%3DM.%2BS.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520mitogen-activated%2520protein%2520kinase%2520kinase%25201%252F2%2520degrader%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D10897%26epage%3D10911%26doi%3D10.1021%2Facs.jmedchem.9b01528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandarlapaty, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">Potent and preferential degradation of CDK6 via proteolysis targeting chimera degraders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">7575</span>– <span class="NLM_lpage">7582</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00871</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00871" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVWls7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=7575-7582&author=S.+Suauthor=Z.+Yangauthor=H.+Gaoauthor=H.+Yangauthor=S.+Zhuauthor=Z.+Anauthor=J.+Wangauthor=Q.+Liauthor=S.+Chandarlapatyauthor=H.+Dengauthor=W.+Wuauthor=Y.+Rao&title=Potent+and+preferential+degradation+of+CDK6+via+proteolysis+targeting+chimera+degraders&doi=10.1021%2Facs.jmedchem.9b00871"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and Preferential Degradation of CDK6 via Proteolysis Targeting Chimera Degraders</span></div><div class="casAuthors">Su, Shang; Yang, Zimo; Gao, Hongying; Yang, Haiyan; Zhu, Songbiao; An, Zixuan; Wang, Juanjuan; Li, Qing; Chandarlapaty, Sarat; Deng, Haiteng; Wu, Wei; Rao, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7575-7582</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A focused PROTAC library hijacking cancer therapeutic target CDK6 was developed.  A design principle as "match/mismatch" was proposed for understanding the degrdn. profile differences in these PROTACs.  Notably, potent PROTACs with specific and remarkable CDK6 degrdn. potential were generated by linking CDK6 inhibitor palbociclib and E3 ligase CRBN recruiter pomalidomide.  The PROTAC strongly inhibited proliferation of hematopoietic cancer cells including multiple myeloma and robustly degraded copy-amplified/mutated forms of CDK6, indicating future potential clin. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQhCa_z8uvA7Vg90H21EOLACvtfcHk0lin7l3nmPF92Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVWls7zF&md5=ed100530898c2e4c1754ca48e7295134</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00871&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00871%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DAn%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DChandarlapaty%26aufirst%3DS.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPotent%2520and%2520preferential%2520degradation%2520of%2520CDK6%2520via%2520proteolysis%2520targeting%2520chimera%2520degraders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D7575%26epage%3D7582%26doi%3D10.1021%2Facs.jmedchem.9b00871" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span> <span> </span><span class="NLM_article-title">Homolog-selective degradation as a strategy to probe the function of CDK6 in AML</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">300</span>– <span class="NLM_lpage">306</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1016%2Fj.chembiol.2018.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=30595531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsVSj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=300-306&author=M.+Brandauthor=B.+Jiangauthor=S.+Bauerauthor=K.+A.+Donovanauthor=Y.+Liangauthor=E.+S.+Wangauthor=R.+P.+Nowakauthor=J.+C.+Yuanauthor=T.+Zhangauthor=N.+Kwiatkowskiauthor=A.+C.+M%C3%BCllerauthor=E.+S.+Fischerauthor=N.+S.+Grayauthor=G.+E.+Winter&title=Homolog-selective+degradation+as+a+strategy+to+probe+the+function+of+CDK6+in+AML&doi=10.1016%2Fj.chembiol.2018.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Homolog-Selective Degradation as a Strategy to Probe the Function of CDK6 in AML</span></div><div class="casAuthors">Brand, Matthias; Jiang, Baishan; Bauer, Sophie; Donovan, Katherine A.; Liang, Yanke; Wang, Eric S.; Nowak, Radoslaw P.; Yuan, Jingting C.; Zhang, Tinghu; Kwiatkowski, Nicholas; Mueller, Andre C.; Fischer, Eric S.; Gray, Nathanael S.; Winter, Georg E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">300-306.e9</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The design of selective small mols. is often stymied by similar ligand binding pockets.  Here, we report BSJ-03-123, a phthalimide-based degrader that exploits protein-interface determinants to achieve proteome-wide selectivity for the degrdn. of cyclin-dependent kinase 6 (CDK6).  Pharmacol. CDK6 degrdn. targets a selective dependency of acute myeloid leukemia cells, and transcriptomics and phosphoproteomics profiling of acute degrdn. of CDK6 enabled dynamic mapping of its immediate role in coordinating signaling and transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGognoz8FFoR9LVg90H21EOLACvtfcHk0lin7l3nmPF92Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsVSj&md5=f8c1c3520c6c0218763d8769c59b4342</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DE.%2BS.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DYuan%26aufirst%3DJ.%2BC.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DA.%2BC.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26atitle%3DHomolog-selective%2520degradation%2520as%2520a%2520strategy%2520to%2520probe%2520the%2520function%2520of%2520CDK6%2520in%2520AML%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D300%26epage%3D306%26doi%3D10.1016%2Fj.chembiol.2018.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) of anaplastic lymphoma kinase (ALK)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1016%2Fj.ejmech.2018.03.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=29627725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=304-314&author=C.+Zhangauthor=X.-R.+Hanauthor=X.+Yangauthor=B.+Jiangauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Proteolysis+targeting+chimeras+%28PROTACs%29+of+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1016%2Fj.ejmech.2018.03.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Zhang, Chengwei; Han, Xiao-Ran; Yang, Xiaobao; Jiang, Biao; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-314</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Anaplastic lymphoma kinase (ALK) activation has been assocd. with many types of human cancer.  Significant efforts have been devoted to the development of ALK inhibitors to antagonize the kinase activity of ALK.  Four ALK inhibitors have been approved by the FDA to date for treating patients with ALK-pos. nonsmall cell lung cancers (NSCLC).  However, drug resistance has been obsd. in the majority of patients treated with these inhibitors.  New therapeutic strategies (e.g., compds. with novel mechanisms of action) are needed to overcome the drug resistance issue.  The emerging PROTAC (Proteolysis Targeting Chimera) technol. has been successfully applied to selective degrdn. of multiple protein targets, but not ALK.  Since ALK protein levels are not important for viability in mammals, ALK PROTACs could lead to novel therapeutics with minimal toxicity.  Here the authors report the design, synthesis and biol. evaluation of novel PROTACs (degraders) of ALK.  MS4077 (I) and MS4078 (II) potently decreased cellular levels of oncogenic active ALK fusion proteins in a concn.- and time-dependent manner in SU-DHL-1 lymphoma and NCI-H2228 lung cancer cells.  The ALK protein degrdn. induced by compds. I and II was cereblon and proteasome dependent.  In addn., compds. I and II potently inhibited proliferation of SU-DHL-1 cells.  Furthermore, compd. II displayed good plasma exposure in a mouse pharmacokinetic study, thus is suitable for in vivo efficacy studies.  The authors also developed MS4748 and MS4740 , very close analogs of I and II resp., which are incapable to degrade the ALK fusion proteins, as neg. controls.  All 4 compds. are valuable chem. tools for investigating effects of ALK pharmacol. degrdn.  The study paved the way for developing the next generation of ALK PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPGM3aAZZ6h7Vg90H21EOLACvtfcHk0lh5IMlmDA1ObA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D&md5=804324b95034f38b46e45f89e18e55be</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.071%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DX.-R.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D304%26epage%3D314%26doi%3D10.1016%2Fj.ejmech.2018.03.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent and selective epidermal growth factor receptor (EGFR) bifunctional small-molecule degraders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">1216</span>– <span class="NLM_lpage">1232</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01566</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01566" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisFWksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=1216-1232&author=M.+Chengauthor=X.+Yuauthor=K.+Luauthor=L.+Xieauthor=L.+Wangauthor=F.+Mengauthor=X.+Hanauthor=X.+Chenauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Discovery+of+potent+and+selective+epidermal+growth+factor+receptor+%28EGFR%29+bifunctional+small-molecule+degraders&doi=10.1021%2Facs.jmedchem.9b01566"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent and Selective Epidermal Growth Factor Receptor (EGFR) Bifunctional Small-Molecule Degraders</span></div><div class="casAuthors">Cheng, Meng; Yu, Xufen; Lu, Kaylene; Xie, Ling; Wang, Li; Meng, Fanye; Han, Xiaoran; Chen, Xian; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1216-1232</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Several epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors have been developed and approved by Food and Drug Administration for the treatment of non-small-cell lung cancers, but their efficacy can be compromised by acquired drug resistance conferred by EGFR-mutant variants.  Here, we described the discovery of a novel E3 ligase von Hippel-Lindau-recruiting EGFR degrader, MS39 (compd. 6), and a first-in-class E3 ligase cereblon-recruiting EGFR degrader, MS154 (compd. 10), using the proteolysis targeting chimera technol.  These compds. potently induced the degrdn. of mutant but not wild-type EGFR in an E3 ligase-dependent manner in cancer cell lines and effectively suppressed the growth of lung cancer cells compared with the corresponding neg. controls.  The global proteomic analyses revealed that the compds. were highly selective for EGFR.  Furthermore, both compds. were bioavailable in mouse pharmacokinetic studies, and compd. 6 is the first EGFR degrader suitable for in vivo efficacy studies.  Overall, we provide a set of well-characterized chem. tools to the research community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraFLSjUGE-6rVg90H21EOLACvtfcHk0lh5IMlmDA1ObA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisFWksA%253D%253D&md5=ec831897a4258f1968e254aa433426c3</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01566%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DM.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DK.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMeng%26aufirst%3DF.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520selective%2520epidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520bifunctional%2520small-molecule%2520degraders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D1216%26epage%3D1232%26doi%3D10.1021%2Facs.jmedchem.9b01566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzes, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantrell, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Potent and selective chemical probe of hypoxic signalling downstream of HIF-alpha hydroxylation via VHL inhibition</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13312</span>, <span class="refDoi"> DOI: 10.1038/ncomms13312</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fncomms13312" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=27811928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVSitLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=13312&author=J.+Frostauthor=C.+Galdeanoauthor=P.+Soaresauthor=M.+S.+Gaddauthor=K.+M.+Grzesauthor=L.+Ellisauthor=O.+Epemoluauthor=S.+Shimamuraauthor=M.+Bantscheffauthor=P.+Grandiauthor=K.+D.+Readauthor=D.+A.+Cantrellauthor=S.+Rochaauthor=A.+Ciulli&title=Potent+and+selective+chemical+probe+of+hypoxic+signalling+downstream+of+HIF-alpha+hydroxylation+via+VHL+inhibition&doi=10.1038%2Fncomms13312"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective chemical probe of hypoxic signalling downstream of HIF-α hydroxylation via VHL inhibition</span></div><div class="casAuthors">Frost, Julianty; Galdeano, Carles; Soares, Pedro; Gadd, Morgan S.; Grzes, Katarzyna M.; Ellis, Lucy; Epemolu, Ola; Shimamura, Satoko; Bantscheff, Marcus; Grandi, Paola; Read, Kevin D.; Cantrell, Doreen A.; Rocha, Sonia; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13312</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. strategies to using small mols. to stimulate hypoxia inducible factors (HIFs) activity and trigger a hypoxic response under normoxic conditions, such as iron chelators and inhibitors of prolyl hydroxylase domain (PHD) enzymes, have broad-spectrum activities and off-target effects.  Here we disclose VH298, a potent VHL inhibitor that stabilizes HIF-α and elicits a hypoxic response via a different mechanism, that is the blockade of the VHL:HIF-α protein-protein interaction downstream of HIF-α hydroxylation by PHD enzymes.  We show that VH298 engages with high affinity and specificity with VHL as its only major cellular target, leading to selective on-target accumulation of hydroxylated HIF-α in a concn.- and time-dependent fashion in different cell lines, with subsequent upregulation of HIF-target genes at both mRNA and protein levels.  VH298 represents a high-quality chem. probe of the HIF signalling cascade and an attractive starting point to the development of potential new therapeutics targeting hypoxia signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLB1WvVF4EFLVg90H21EOLACvtfcHk0lh5IMlmDA1ObA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVSitLfJ&md5=7a884b6193b3ca5e119fecea2eb31d3e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fncomms13312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms13312%26sid%3Dliteratum%253Aachs%26aulast%3DFrost%26aufirst%3DJ.%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DGrzes%26aufirst%3DK.%2BM.%26aulast%3DEllis%26aufirst%3DL.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCantrell%26aufirst%3DD.%2BA.%26aulast%3DRocha%26aufirst%3DS.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DPotent%2520and%2520selective%2520chemical%2520probe%2520of%2520hypoxic%2520signalling%2520downstream%2520of%2520HIF-alpha%2520hydroxylation%2520via%2520VHL%2520inhibition%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D13312%26doi%3D10.1038%2Fncomms13312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, W.</span></span> <span> </span><span class="NLM_article-title">MLN4924 (Pevonedistat), a protein neddylation inhibitor, suppresses proliferation and migration of human clear cell renal cell carcinoma</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">5599</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-06098-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fs41598-017-06098-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=28717191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A280%3ADC%252BC1cjosFWksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=5599&author=S.+Tongauthor=Y.+Siauthor=H.+Yuauthor=L.+Zhangauthor=P.+Xieauthor=W.+Jiang&title=MLN4924+%28Pevonedistat%29%2C+a+protein+neddylation+inhibitor%2C+suppresses+proliferation+and+migration+of+human+clear+cell+renal+cell+carcinoma&doi=10.1038%2Fs41598-017-06098-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">MLN4924 (Pevonedistat), a protein neddylation inhibitor, suppresses proliferation and migration of human clear cell renal cell carcinoma</span></div><div class="casAuthors">Tong Shuai; Si Yang; Yu Hefen; Xie Ping; Jiang Wenguo; Tong Shuai; Si Yang; Yu Hefen; Xie Ping; Jiang Wenguo; Zhang Lingqiang; Jiang Wenguo</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5599</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Neddylation is a post-translational protein modification associated with cancer development.  MLN4924 is a neddylation inhibitor currently under investigation in multiple phase I studies on various malignancies, and its clincal name is Pevonedistat.  It has been documented that MLN4924 blocks Cullins neddylation and inactivates CRLs and, in turn, triggers cell-cycle arrest, apoptosis, senescence and autophagy in many cancer cells.  In this study, we investigated the anti-tumor effect of MLN4924 in human clear cell renal carcinoma (ccRCC).  Levels of both Nedd8 activating enzyme E1 and Nedd8-conjugating enzyme E2 were higher in ccRCC tissues and RCC cancer cells than in normal.  Moreover, MLN4924 treatment led to rapid inhibition of Cullin1 neddylation and notably suppressed growth and survival as well as migration in a dose-and time-dependent manner.  Mechanistic studies revealed that MLN4924 induced the accumulation of a number of CRL substrates, including p21, p27 and Wee1 to trigger DNA damage and induce growth arrest at the G2/M phase.  MLN4924 also induced anti-migration and anti-invasion by activating E-cadherin and repressing Vimentin.  Taken together, this study provides the first evidence that neddylation pathway is overactive in ccRCC and that MLN4924 induces dose-dependent anti-proliferation, anti-migration, anti-invasion in ccRCC cells.  The study thus indicates that MLN4924 has potential therapeutic value for the clinical treatment of renal cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQrdwMzHGpO0rrIbt-tGZmxfW6udTcc2eYJFKpBYkNa2rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cjosFWksg%253D%253D&md5=1e0a80b533adac1674de53b9b68750a7</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-06098-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-06098-y%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DS.%26aulast%3DSi%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DP.%26aulast%3DJiang%26aufirst%3DW.%26atitle%3DMLN4924%2520%2528Pevonedistat%2529%252C%2520a%2520protein%2520neddylation%2520inhibitor%252C%2520suppresses%2520proliferation%2520and%2520migration%2520of%2520human%2520clear%2520cell%2520renal%2520cell%2520carcinoma%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D5599%26doi%3D10.1038%2Fs41598-017-06098-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faelth-Savitski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casillas, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Catalytic in vivo protein knockdown by small-molecule PROTACs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+D.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+protein+knockdown+by+small-molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0lgQIC6OjfTsOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520protein%2520knockdown%2520by%2520small-molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tovell, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shpiro, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crafter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alessi, D. R.</span></span> <span> </span><span class="NLM_article-title">Design and characterization of SGK3-PROTAC1, an isoform specific SGK3 kinase PROTAC degrader</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2024</span>– <span class="NLM_lpage">2034</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00505</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00505" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1KksLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=2024-2034&author=H.+Tovellauthor=A.+Testaauthor=H.+Zhouauthor=N.+Shpiroauthor=C.+Crafterauthor=A.+Ciulliauthor=D.+R.+Alessi&title=Design+and+characterization+of+SGK3-PROTAC1%2C+an+isoform+specific+SGK3+kinase+PROTAC+degrader&doi=10.1021%2Facschembio.9b00505"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Characterization of SGK3-PROTAC1, an Isoform Specific SGK3 Kinase PROTAC Degrader</span></div><div class="casAuthors">Tovell, Hannah; Testa, Andrea; Zhou, Houjiang; Shpiro, Natalia; Crafter, Claire; Ciulli, Alessio; Alessi, Dario R.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2024-2034</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SGK3 is a PX domain contg. protein kinase activated at endosomes downstream of Class 1 and 3 PI3K family members by growth factors and oncogenic mutations.  SGK3 plays a key role in mediating resistance of breast cancer cells to Class 1 PI3K or Akt inhibitors, by substituting for loss of Akt activity and restoring proliferative pathways such as mTORC1 signaling.  It is therefore crit. to develop tools to potently target SGK3, and obstruct its role in inhibitor resistance.  Here we describe the development of SGK3-PROTAC1 (I), a PROTAC conjugate of the 308-R SGK inhibitor with the VH032 VHL binding ligand, targeting SGK3 for degrdn.  0.3 μM I induced 50% degrdn. of endogenous SGK3 within 2 h, with maximal 80% degrdn. obsd. within 8 h, accompanied by a loss of phosphorylation of NDRG1, an SGK3 substrate.  I did not degrade closely related SGK1 and SGK2 isoforms that are nevertheless engaged and inhibited by 308-R.  Proteomic anal. revealed that SGK3 was the only cellular protein whose cellular levels were significantly reduced following treatment with I.  Low doses of I (0.1-0.3 μM) restored sensitivity of SGK3 dependent ZR-75-1 and CAMA-1 breast cancer cells to Akt (AZD5363) and PI3K (GDC0941) inhibitors, whereas the cis epimer analog incapable of binding to the VHL E3 ligase had no impact.  I suppressed proliferation of ZR-75-1 and CAMA-1 cancer cell lines treated with a PI3K inhibitor (GDC0941) more effectively than could be achieved by a conventional SGK isoform inhibitor (14H).  This work underscores the benefit of the PROTAC approach in targeting protein kinase signaling pathways with greater efficacy and selectivity than can be achieved with conventional inhibitors.  I will be an important reagent to explore the roles of the SGK3 pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCZDBO0-RuDrVg90H21EOLACvtfcHk0lgKyqNw_CefiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1KksLnN&md5=83f84ea9aba235a129c4cc4919888c6f</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00505&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00505%26sid%3Dliteratum%253Aachs%26aulast%3DTovell%26aufirst%3DH.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DShpiro%26aufirst%3DN.%26aulast%3DCrafter%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26atitle%3DDesign%2520and%2520characterization%2520of%2520SGK3-PROTAC1%252C%2520an%2520isoform%2520specific%2520SGK3%2520kinase%2520PROTAC%2520degrader%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D2024%26epage%3D2034%26doi%3D10.1021%2Facschembio.9b00505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratikopoulos, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, K.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leshchenko, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rialdi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dale, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagana, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parekh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class EZH2 selective degrader</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">222</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0421-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fs41589-019-0421-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=31819273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlCht7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=214-222&author=A.+Maauthor=E.+Stratikopoulosauthor=K.-S.+Parkauthor=J.+Weiauthor=T.+C.+Martinauthor=X.+Yangauthor=M.+Schwarzauthor=V.+Leshchenkoauthor=A.+Rialdiauthor=B.+Daleauthor=A.+Laganaauthor=E.+Guccioneauthor=S.+Parekhauthor=R.+Parsonsauthor=J.+Jin&title=Discovery+of+a+first-in-class+EZH2+selective+degrader&doi=10.1038%2Fs41589-019-0421-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a first-in-class EZH2 selective degrader</span></div><div class="casAuthors">Ma, Anqi; Stratikopoulos, Elias; Park, Kwang-Su; Wei, Jieli; Martin, Tiphaine C.; Yang, Xiaobao; Schwarz, Megan; Leshchenko, Violetta; Rialdi, Alexander; Dale, Brandon; Lagana, Alessandro; Guccione, Ernesto; Parekh, Samir; Parsons, Ramon; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">214-222</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The enhancer of zeste homolog 2 (EZH2) is the main enzymic subunit of the PRC2 complex, which catalyzes trimethylation of histone H3 lysine 27 (H3K27me3) to promote transcriptional silencing.  EZH2 is overexpressed in multiple types of cancer including triple-neg. breast cancer (TNBC), and high expression levels correlate with poor prognosis.  Several EZH2 inhibitors, which inhibit the methyltransferase activity of EZH2, have shown promise in treating sarcoma and follicular lymphoma in clinics.  However, EZH2 inhibitors are ineffective at blocking proliferation of TNBC cells, even though they effectively reduce the H3K27me3 mark.  Using a hydrophobic tagging approach, we generated MS1943, a first-in-class EZH2 selective degrader that effectively reduces EZH2 levels in cells.  Importantly, MS1943 has a profound cytotoxic effect in multiple TNBC cells, while sparing normal cells, and is efficacious in vivo, suggesting that pharmacol. degrdn. of EZH2 can be advantageous for treating the cancers that are dependent on EZH2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPb6bK3v_gQrVg90H21EOLACvtfcHk0lgKyqNw_CefiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlCht7zJ&md5=e1ccae457eba9b298f8b0e583bbaa833</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0421-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0421-4%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DA.%26aulast%3DStratikopoulos%26aufirst%3DE.%26aulast%3DPark%26aufirst%3DK.-S.%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DMartin%26aufirst%3DT.%2BC.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DSchwarz%26aufirst%3DM.%26aulast%3DLeshchenko%26aufirst%3DV.%26aulast%3DRialdi%26aufirst%3DA.%26aulast%3DDale%26aufirst%3DB.%26aulast%3DLagana%26aufirst%3DA.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DParekh%26aufirst%3DS.%26aulast%3DParsons%26aufirst%3DR.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520EZH2%2520selective%2520degrader%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D16%26spage%3D214%26epage%3D222%26doi%3D10.1038%2Fs41589-019-0421-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 8 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Ryan V. Quiroz, Michael H. Reutershan, Sebastian E. Schneider, David Sloman, Brian M. Lacey, Brooke M. Swalm, Charles S. Yeung, Craig Gibeau, Daniel S. Spellman, Danica A. Rankic, Dapeng Chen, David Witter, Doug Linn, Erik Munsell, Guo Feng, Haiyan Xu, Jonathan M. E. Hughes, Jongwon Lim, Josep Saurí, Kristin Geddes, Murray Wan, My Sam Mansueto, Nicole E. Follmer, Patrick S. Fier, Phieng Siliphaivanh, Pierre Daublain, Rachel L. Palte, Robert P. Hayes, Sandra Lee, Shuhei Kawamura, Steven Silverman, Sulagna Sanyal, Timothy J. Henderson, Yingchun Ye, Yuanwei Gao, Benjamin Nicholson, <span class="NLM_string-name hlFld-ContribAuthor">Michelle R. Machacek</span>. </span><span class="cited-content_cbyCitation_article-title">The Discovery of Two Novel Classes of 5,5-Bicyclic Nucleoside-Derived PRMT5 Inhibitors for the Treatment of Cancer. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (7)
                                     , 3911-3939. <a href="https://doi.org/10.1021/acs.jmedchem.0c02083" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02083</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02083%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThe%252BDiscovery%252Bof%252BTwo%252BNovel%252BClasses%252Bof%252B5%25252C5-Bicyclic%252BNucleoside-Derived%252BPRMT5%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BCancer%26aulast%3DQuiroz%26aufirst%3DRyan%2BV.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D01122020%26date%3D23032021%26volume%3D64%26issue%3D7%26spage%3D3911%26epage%3D3939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ernesto  Guccione</span>, <span class="hlFld-ContribAuthor ">Megan  Schwarz</span>, <span class="hlFld-ContribAuthor ">Federico  Di Tullio</span>, <span class="hlFld-ContribAuthor ">Slim  Mzoughi</span>. </span><span class="cited-content_cbyCitation_article-title">Cancer synthetic vulnerabilities to protein arginine methyltransferase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Pharmacology</span><span> <strong>2021,</strong> <em>59 </em>, 33-42. <a href="https://doi.org/10.1016/j.coph.2021.04.004" title="DOI URL">https://doi.org/10.1016/j.coph.2021.04.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.coph.2021.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.coph.2021.04.004%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Pharmacology%26atitle%3DCancer%252Bsynthetic%252Bvulnerabilities%252Bto%252Bprotein%252Barginine%252Bmethyltransferase%252Binhibitors%26aulast%3DGuccione%26aufirst%3DErnesto%26date%3D2021%26volume%3D59%26spage%3D33%26epage%3D42" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qin  Wu</span>, <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>. </span><span class="cited-content_cbyCitation_article-title">Protein arginine methylation: from enigmatic functions to therapeutic targeting. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2021,</strong> <em>20 </em>
                                    (7)
                                     , 509-530. <a href="https://doi.org/10.1038/s41573-021-00159-8" title="DOI URL">https://doi.org/10.1038/s41573-021-00159-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41573-021-00159-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41573-021-00159-8%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DProtein%252Barginine%252Bmethylation%25253A%252Bfrom%252Benigmatic%252Bfunctions%252Bto%252Btherapeutic%252Btargeting%26aulast%3DWu%26aufirst%3DQin%26date%3D2021%26date%3D2021%26volume%3D20%26issue%3D7%26spage%3D509%26epage%3D530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ting  Cai</span>, <span class="hlFld-ContribAuthor ">Zhenbao  Yu</span>, <span class="hlFld-ContribAuthor ">Zhen  Wang</span>, <span class="hlFld-ContribAuthor ">Chen  Liang</span>, <span class="hlFld-ContribAuthor ">Stéphane  Richard</span>. </span><span class="cited-content_cbyCitation_article-title">Arginine methylation of SARS-Cov-2 nucleocapsid protein regulates RNA binding, its ability to suppress stress granule formation, and viral replication. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2021,</strong> <em>297 </em>
                                    (1)
                                     , 100821. <a href="https://doi.org/10.1016/j.jbc.2021.100821" title="DOI URL">https://doi.org/10.1016/j.jbc.2021.100821</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jbc.2021.100821&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jbc.2021.100821%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DArginine%252Bmethylation%252Bof%252BSARS-Cov-2%252Bnucleocapsid%252Bprotein%252Bregulates%252BRNA%252Bbinding%25252C%252Bits%252Bability%252Bto%252Bsuppress%252Bstress%252Bgranule%252Bformation%25252C%252Band%252Bviral%252Breplication%26aulast%3DCai%26aufirst%3DTing%26date%3D2021%26volume%3D297%26issue%3D1%26spage%3D100821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marton  Siklos</span>, <span class="hlFld-ContribAuthor ">Stefan  Kubicek</span>. </span><span class="cited-content_cbyCitation_article-title">T
              herapeutic targeting of chromatin: status and opportunities. </span><span class="cited-content_cbyCitation_journal-name">The FEBS Journal</span><span> <strong>2021,</strong> <em>17 </em><a href="https://doi.org/10.1111/febs.15966" title="DOI URL">https://doi.org/10.1111/febs.15966</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/febs.15966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Ffebs.15966%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520FEBS%2520Journal%26atitle%3DT%252Bherapeutic%252Btargeting%252Bof%252Bchromatin%25253A%252Bstatus%252Band%252Bopportunities%26aulast%3DSiklos%26aufirst%3DMarton%26date%3D2021%26date%3D2021%26volume%3D17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brandon  Dale</span>, <span class="hlFld-ContribAuthor ">Meng  Cheng</span>, <span class="hlFld-ContribAuthor ">Kwang-Su  Park</span>, <span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Yue  Xiong</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Advancing targeted protein degradation for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Cancer</span><span> <strong>2021,</strong> <em>170 </em><a href="https://doi.org/10.1038/s41568-021-00365-x" title="DOI URL">https://doi.org/10.1038/s41568-021-00365-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41568-021-00365-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41568-021-00365-x%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Cancer%26atitle%3DAdvancing%252Btargeted%252Bprotein%252Bdegradation%252Bfor%252Bcancer%252Btherapy%26aulast%3DDale%26aufirst%3DBrandon%26date%3D2021%26date%3D2021%26volume%3D170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jee Won  Hwang</span>, <span class="hlFld-ContribAuthor ">Yena  Cho</span>, <span class="hlFld-ContribAuthor ">Gyu-Un  Bae</span>, <span class="hlFld-ContribAuthor ">Su-Nam  Kim</span>, <span class="hlFld-ContribAuthor ">Yong Kee  Kim</span>. </span><span class="cited-content_cbyCitation_article-title">Protein arginine methyltransferases: promising targets for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Experimental & Molecular Medicine</span><span> <strong>2021,</strong> <em>53 </em>
                                    (5)
                                     , 788-808. <a href="https://doi.org/10.1038/s12276-021-00613-y" title="DOI URL">https://doi.org/10.1038/s12276-021-00613-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s12276-021-00613-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs12276-021-00613-y%26sid%3Dliteratum%253Aachs%26jtitle%3DExperimental%2520%2526%2520Molecular%2520Medicine%26atitle%3DProtein%252Barginine%252Bmethyltransferases%25253A%252Bpromising%252Btargets%252Bfor%252Bcancer%252Btherapy%26aulast%3DHwang%26aufirst%3DJee%2BWon%26date%3D2021%26date%3D2021%26volume%3D53%26issue%3D5%26spage%3D788%26epage%3D808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tanner  Wright</span>, <span class="hlFld-ContribAuthor ">Yalong  Wang</span>, <span class="hlFld-ContribAuthor ">Mark T.  Bedford</span>. </span><span class="cited-content_cbyCitation_article-title">The Role of the PRMT5–SND1 Axis in Hepatocellular Carcinoma. </span><span class="cited-content_cbyCitation_journal-name">Epigenomes</span><span> <strong>2021,</strong> <em>5 </em>
                                    (1)
                                     , 2. <a href="https://doi.org/10.3390/epigenomes5010002" title="DOI URL">https://doi.org/10.3390/epigenomes5010002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/epigenomes5010002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fepigenomes5010002%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenomes%26atitle%3DThe%252BRole%252Bof%252Bthe%252BPRMT5%2525E2%252580%252593SND1%252BAxis%252Bin%252BHepatocellular%252BCarcinoma%26aulast%3DWright%26aufirst%3DTanner%26date%3D2021%26date%3D2021%26volume%3D5%26issue%3D1%26spage%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Design of PRMT5 putative degraders <b>9</b>, <b>10</b>, <b>11</b>, <b>12</b>, and <b>13</b>. (A) Crystal structure of PRMT5:MEP50 (gray) in complex with SAM (magenta) and EPZ015666 (green) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X61">4X61</a>). The highlighted oxetane moiety of EPZ015666 is solvent-exposed. Key H-bond interactions between the EPZ015666 and PRMT5 residues are highlighted by red dashed lines. (B) Chemical structures of EPZ015666, compound <b>3</b>, and the five putative PRMT5 degraders <b>9</b>–<b>13</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0013.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of PRMT5 Putative Degraders <b>9</b>–<b>13</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>i</i>-PrOH, reflux; (b) TFA, DCM; (c) EDCI·HCl, HOAt, NMM, 6-chloropyrimidine-4-carboxylic acid, DMSO, 18% over three steps; (d) <i>tert</i>-butyl 3-aminoazetidine-1-carboxylate, NMP, 90%; (e) dicarboxylic acids, EDCI·HCl, HOAt, NMM, DMSO, 54–78%; (f) <b>3</b>, EDCI·HCl, HOAt, NMM, DMSO, 36–60%.</p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Effects of compounds <b>9</b>–<b>13</b> on reducing the PRMT5 protein level in MCF-7 cells. MCF-7 cells were treated with compounds <b>9</b>–<b>13</b> at the indicated concentrations (0.2, 1, and 5 μM) for 6 d. Cell lysates were collected, and the PRMT5 protein levels were detected by western blots. The results are representative of at least two independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Design of the PRMT5 degrader <b>15</b> and its two control compounds <b>17</b> and <b>21</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0014.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>15</b>, <b>17</b>, and <b>21</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) EDCI·HCl, HOAt, NMM, DMSO, 50%; (b) <b>3</b>, EDCI·HCl, HOAt, NMM, DMSO, 35–53%; (c) <i>i</i>-PrOH, reflux; (d) TFA, DCM; (e) EDCI·HCl, HOAt, NMM, 6-chloropyrimidine-4-carboxylic acid, DMSO; (f) <i>tert</i>-butyl 3-aminoazetidine-1-carboxylate, NMP, 13% over four steps; (g) <b>14</b>, EDCI·HCl, HOAt, NMM, DMSO, 49%.</p></p></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effects of compounds <b>15</b>, <b>17</b>, and <b>21</b> on inhibiting the PRMT5 methyltransferase activity in a radioactive biochemical assay. Substrate: H4 (<b>1–15</b>)-biotin (400 nM). Cofactor: <sup>3</sup>H-SAM (1 μM). EPZ015666 was used as a positive control. IC<sub>50</sub> determination experiments were performed in duplicate, and the values are presented as mean ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Effects of compound <b>15</b> on reducing the PRMT5 protein level and inhibiting SDMA in MCF-7 cells. (A,B) Compound <b>15</b> concentration-dependently reduced the PRMT5 protein level, whereas compounds <b>17</b> and <b>21</b> and EPZ015666 did not. MCF-7 cells were treated with the indicated concentration of <b>15</b>, <b>17</b>, <b>21</b>, or EPZ015666 for 6 d. Cell lysates were collected, and the protein levels of PRMT5 were detected by western blotting. The relative strength of the PRMT5 signal on the western blots was measured by densitometry. DC<sub>50</sub> and <i>D</i><sub>max</sub> values are presented as mean ± SD from three independent experiments. (C) Compound <b>15</b> reduced the PRMT5 protein level in a time-dependent manner. MCF-7 cells were treated with 5 μM of compound <b>15</b> for 0, 2, 4, 6, or 8 d. The PRMT5 protein levels were detected by western blotting. (D) Compound <b>15</b> and EPZ015666 inhibited SDMA. MCF-7 cells were treated with the indicated concentration of EPZ015666 or compound <b>15</b> for 6 d. The SDMA levels were detected by western blotting. Western blot results are representative of at least two independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Compound <b>15</b> reduced the PRMT5 protein level in MCF-7 cells in a PRMT5-, E3 ligase VHL-, and proteasome-dependent manner. MCF-7 cells were treated with 5 μM of compound <b>15</b> for 7 d. During the last 24 h, the cells were co-treated with 100 μM VH-298, 2 μM MLN4924, 30 μM MG-132, or 30 μM EPZ015666. The protein levels of PRMT5 were detected by western blotting. (B) PRMT5 degradation induced by compound <b>15</b> was reversible. MCF-7 cells were treated with 5 μM of compound <b>15</b> for 6 d, and compound <b>15</b> was then washed out. The PRMT5 protein levels were detected by western blotting at 0, 12, 24, and 48 h post the washout.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Effects of compounds <b>15</b>, <b>17</b>, <b>21</b>, and EPZ015666 on inhibiting the proliferation of MCF-7 cells. MCF-7 cells were treated with the indicated concentration of compounds <b>15</b>, <b>17</b>, <b>21</b>, or EPZ015666 for 6 d. The relative cell viabilities to the untreated group are shown as mean ± SD (<i>n</i> = 3).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Global proteomic analysis indicates that compound <b>15</b> is a highly selective PRMT5 degrader. MCF-7 cells were treated with compound <b>15</b> at 5 μM or DMSO for 5 d and then harvested and lysed for liquid chromatography–MS (LC–MS) analysis. LFQ was used to calculate the peptide intensity; the relative abundance of PRMT5 (LFQ intensity value) between the compound <b>15</b>-treated and DMSO-treated groups is shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Compound <b>15</b> reduced the PRMT5 protein levels in HeLa, A549, A172, and Jurkat cells. The cells were treated with 5 μM of compound <b>15</b> for 6 d. The protein levels of PRMT5 were detected by western blotting. The western blotting results are representative of two independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Compound <b>15</b> and EPZ015666 inhibited the proliferation of HeLa (A), A549 (B), A172 (C), and Jurkat (D) cells. The cells were treated with <b>15</b> or EPZ015666 at the indicated concentrations for 6 d. The relative cell viabilities to the untreated group are shown as mean ± SD (<i>n</i> = 3).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/medium/jm0c01111_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Plasma concentrations of compound <b>15</b> over 12 h, following a single 150 mg/kg IP injection in male Swiss albino mice. The compound concentration shown at each time point is the mean ± SD from three test mice.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c01111/20200903/images/large/jm0c01111_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01111&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i49">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16872" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16872" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 48 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Branscombe, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.-h.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pestka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S.</span></span> <span> </span><span class="NLM_article-title">PRMT5 (Janus kinase-binding protein 1) catalyzes the formation of symmetric dimethylarginine residues in proteins</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">32971</span>– <span class="NLM_lpage">32976</span>, <span class="refDoi"> DOI: 10.1074/jbc.m105412200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1074%2Fjbc.M105412200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=11413150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFCmtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=32971-32976&author=T.+L.+Branscombeauthor=A.+Frankelauthor=J.-H.+Leeauthor=J.+R.+Cookauthor=Z.-h.+Yangauthor=S.+Pestkaauthor=S.+Clarke&title=PRMT5+%28Janus+kinase-binding+protein+1%29+catalyzes+the+formation+of+symmetric+dimethylarginine+residues+in+proteins&doi=10.1074%2Fjbc.m105412200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT5 (Janus kinase-binding protein 1) catalyzes the formation of symmetric dimethylarginine residues in proteins</span></div><div class="casAuthors">Branscombe, Tina L.; Frankel, Adam; Lee, Jin-Hyung; Cook, Jeffry R.; Yang, Zhi-Hong; Pestka, Sidney; Clarke, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">32971-32976</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">We have identified a new mammalian protein arginine N-methyltransferase, PRMT5, formerly designated Janus kinase-binding protein 1, that can catalyze the formation of ω-NG-monomethylarginine and sym. ω-NG,NG'-dimethylarginine in a variety of proteins.  A hemagglutinin peptide-tagged PRMT5 complex purified from human HeLa cells catalyzes the S-adenosyl-L-[methyl-3H]methionine-dependent in vitro methylation of myelin basic protein.  When the radiolabeled myelin basic protein was acid-hydrolyzed to free amino acids, and the products were sepd. by high-resoln. cation exchange chromatog., we were able to detect two tritiated species.  One species co-migrated with a ω-NG-monomethylarginine std., and the other co-chromatographed with a sym. ω-NG,NG'-dimethylarginine std.  Upon base treatment, this second species formed methylamine, a breakdown product characteristic of sym. ω-NG,NG'-dimethylarginine.  Further anal. of these two species by thin layer chromatog. confirmed their identification as ω-NG-monomethylarginine and sym. ω-NG,NG'-dimethylarginine.  The hemagglutinin-PRMT5 complex was also able to monomethylate and sym. dimethylate bovine histone H2A and a glutathione S-transferase-fibrillarin (amino acids 1-148) fusion protein (glutathione S-transferase-GAR).  A mutation introduced into the S-adenosyl-L-methionine-binding motif I of a myc-tagged PRMT5 construct in COS-1 cells led to a near complete loss of obsd. enzymic activity.  PRMT5 is the first example of a catalytic chain for a type II protein arginine N-methyltransferase that can result in the formation of sym. dimethylarginine residues as obsd. previously in myelin basic protein, Sm small nuclear ribonucleoproteins, and other polypeptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZJzrYzZ_ze7Vg90H21EOLACvtfcHk0lg1QQs19zNCeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFCmtbw%253D&md5=12e2a0db9d4747c4e7b06668b9fd4d2b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M105412200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M105412200%26sid%3Dliteratum%253Aachs%26aulast%3DBranscombe%26aufirst%3DT.%2BL.%26aulast%3DFrankel%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DJ.-H.%26aulast%3DCook%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DZ.-h.%26aulast%3DPestka%26aufirst%3DS.%26aulast%3DClarke%26aufirst%3DS.%26atitle%3DPRMT5%2520%2528Janus%2520kinase-binding%2520protein%25201%2529%2520catalyzes%2520the%2520formation%2520of%2520symmetric%2520dimethylarginine%2520residues%2520in%2520proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D32971%26epage%3D32976%26doi%3D10.1074%2Fjbc.m105412200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishwanath, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdjument-Bromage, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sif, S.</span></span> <span> </span><span class="NLM_article-title">Human SWI/SNF-associated PRMT5 methylates histone H3 arginine 8 and negatively regulates expression of ST7 and NM23 tumor suppressor genes</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">9630</span>– <span class="NLM_lpage">9645</span>, <span class="refDoi"> DOI: 10.1128/mcb.24.21.9630-9645.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1128%2FMCB.24.21.9630-9645.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=15485929" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltl2htQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=9630-9645&author=S.+Palauthor=S.+N.+Vishwanathauthor=H.+Erdjument-Bromageauthor=P.+Tempstauthor=S.+Sif&title=Human+SWI%2FSNF-associated+PRMT5+methylates+histone+H3+arginine+8+and+negatively+regulates+expression+of+ST7+and+NM23+tumor+suppressor+genes&doi=10.1128%2Fmcb.24.21.9630-9645.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Human SWI/SNF-associated PRMT5 methylates histone H3 arginine 8 and negatively regulates expression of ST7 and NM23 tumor suppressor genes</span></div><div class="casAuthors">Pal, Sharmistha; Vishwanath, Sheethal N.; Erdjument-Bromage, Hediye; Tempst, Paul; Sif, Said</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9630-9645</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) have been implicated in transcriptional activation and repression, but their role in controlling cell growth and proliferation remains obscure.  We have recently shown that PRMT5 can interact with flag-tagged BRG1- and hBRM-based hSWI/SNF chromatin remodelers and that both complexes can specifically methylate histones H3 and H4.  Here we report that PRMT5 can be found in assocn. with endogenous hSWI/SNF complexes, which can methylate H3 and H4 N-terminal tails, and show that H3 arginine 8 and H4 arginine 3 are preferred sites of methylation by recombinant and hSWI/SNF-assocd. PRMT5.  To elucidate the role played by PRMT5 in gene regulation, we have established a PRMT5 antisense cell line and detd. by microarray anal. that more genes are derepressed when PRMT5 levels are reduced.  Among the affected genes, we show that suppressor of tumorigenicity 7 (ST7) and nonmetastatic 23 (NM23) are direct targets of PRMT5-contg. BRG1 and hBRM complexes.  Furthermore, we demonstrate that expression of ST7 and NM23 is reduced in a cell line that overexpresses PRMT5 and that this decrease in expression correlates with H3R8 methylation, H3K9 deacetylation, and increased transformation of NIH 3T3 cells.  These findings suggest that the BRG1- and hBRM-assocd. PRMT5 regulates cell growth and proliferation by controlling expression of genes involved in tumor suppression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHC4UpBSGWILVg90H21EOLACvtfcHk0lg1QQs19zNCeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltl2htQ%253D%253D&md5=663371d517c0e9e3b709483e320869f7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1128%2FMCB.24.21.9630-9645.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.24.21.9630-9645.2004%26sid%3Dliteratum%253Aachs%26aulast%3DPal%26aufirst%3DS.%26aulast%3DVishwanath%26aufirst%3DS.%2BN.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DSif%26aufirst%3DS.%26atitle%3DHuman%2520SWI%252FSNF-associated%2520PRMT5%2520methylates%2520histone%2520H3%2520arginine%25208%2520and%2520negatively%2520regulates%2520expression%2520of%2520ST7%2520and%2520NM23%2520tumor%2520suppressor%2520genes%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D9630%26epage%3D9645%26doi%3D10.1128%2Fmcb.24.21.9630-9645.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrd, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grever, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sif, S.</span></span> <span> </span><span class="NLM_article-title">Low levels of miR-92b/96 induce PRMT5 translation and H3R8/H4R3 methylation in mantle cell lymphoma</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">3558</span>– <span class="NLM_lpage">3569</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7601794</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fsj.emboj.7601794" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=17627275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BD2sXos1eqtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=3558-3569&author=S.+Palauthor=R.+A.+Baiocchiauthor=J.+C.+Byrdauthor=M.+R.+Greverauthor=S.+T.+Jacobauthor=S.+Sif&title=Low+levels+of+miR-92b%2F96+induce+PRMT5+translation+and+H3R8%2FH4R3+methylation+in+mantle+cell+lymphoma&doi=10.1038%2Fsj.emboj.7601794"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Low levels of miR-92b/96 induce PRMT5 translation and H3R8/H4R3 methylation in mantle cell lymphoma</span></div><div class="casAuthors">Pal, Sharmistha; Baiocchi, Robert A.; Byrd, John C.; Grever, Michael R.; Jacob, Samson T.; Sif, Said</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3558-3569</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase PRMT5 interacts with human SWI/SNF complexes and methylates histones H3R8 and H4R3.  To elucidate the role of PRMT5 in human cancer, we analyzed PRMT5 expression in normal human B lymphocytes and a panel of lymphoid cancer cell lines as well as mantle cell lymphoma (MCL) clin. samples.  We show that PRMT5 protein levels are elevated in all cancer cells, including clin. samples examd. despite its low rate of transcription and mRNA stability.  Remarkably, polysome profiling revealed that PRMT5 mRNA is translated more efficiently in Mino and JeKo MCL cells than in normal B cells, and that decreased miR-92b and miR-96 expression augments PRMT5 translation.  Consequently, global methylation of H3R8 and H4R3 is increased and is accompanied by repression of suppressor of tumorigenicity 7 (ST7) in lymphoid cancer cells.  Furthermore, knockdown of PRMT5 expression reduces proliferation of transformed JeKo and Raji cells.  Thus, our studies indicate that aberrant expression of PRMT5 leads to altered epigenetic modification of chromatin, which in turn impacts transcriptional performance of anti-cancer genes and growth of transformed lymphoid cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiSE5d420ctLVg90H21EOLACvtfcHk0lixmcla46x2Pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXos1eqtbg%253D&md5=f2b153393a175b1427873a9db0f52a4a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7601794&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7601794%26sid%3Dliteratum%253Aachs%26aulast%3DPal%26aufirst%3DS.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DGrever%26aufirst%3DM.%2BR.%26aulast%3DJacob%26aufirst%3DS.%2BT.%26aulast%3DSif%26aufirst%3DS.%26atitle%3DLow%2520levels%2520of%2520miR-92b%252F96%2520induce%2520PRMT5%2520translation%2520and%2520H3R8%252FH4R3%2520methylation%2520in%2520mantle%2520cell%2520lymphoma%26jtitle%3DEMBO%2520J.%26date%3D2007%26volume%3D26%26spage%3D3558%26epage%3D3569%26doi%3D10.1038%2Fsj.emboj.7601794" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Migliori, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phalke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bezzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mok, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahu, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunaratne, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capasso, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecatiello, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Marco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackstock, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amati, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mapelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span> <span> </span><span class="NLM_article-title">Symmetric dimethylation of H3R2 is a newly identified histone mark that supports euchromatin maintenance</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">136</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fnsmb.2209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=22231400" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC38Xls1ShtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=136-144&author=V.+Miglioriauthor=J.+M%C3%BCllerauthor=S.+Phalkeauthor=D.+Lowauthor=M.+Bezziauthor=W.+C.+Mokauthor=S.+K.+Sahuauthor=J.+Gunaratneauthor=P.+Capassoauthor=C.+Bassiauthor=V.+Cecatielloauthor=A.+De+Marcoauthor=W.+Blackstockauthor=V.+Kuznetsovauthor=B.+Amatiauthor=M.+Mapelliauthor=E.+Guccione&title=Symmetric+dimethylation+of+H3R2+is+a+newly+identified+histone+mark+that+supports+euchromatin+maintenance&doi=10.1038%2Fnsmb.2209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Symmetric dimethylation of H3R2 is a newly identified histone mark that supports euchromatin maintenance</span></div><div class="casAuthors">Migliori, Valentina; Muller, Julius; Phalke, Sameer; Low, Diana; Bezzi, Marco; Mok, Wei Chuen; Sahu, Sanjeeb Kumar; Gunaratne, Jayantha; Capasso, Paola; Bassi, Christian; Cecatiello, Valentina; De Marco, Ario; Blackstock, Walter; Kuznetsov, Vladimir; Amati, Bruno; Mapelli, Marina; Guccione, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">136-144</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The asym. dimethylation of histone H3 arginine 2 (H3R2me2a) acts as a repressive mark that antagonizes trimethylation of H3 lysine 4.  Here we report that H3R2 is also sym. dimethylated (H3R2me2s) by PRMT5 and PRMT7 and present in euchromatic regions.  Profiling of H3-tail interactors by SILAC MS revealed that H3R2me2s excludes binding of RBBP7, a central component of co-repressor complexes Sin3a, NURD and PRC2.  Conversely H3R2me2s enhances binding of WDR5, a common component of the coactivator complexes MLL, SET1A, SET1B, NLS1 and ATAC.  The interaction of histone H3 with WDR5 distinguishes H3R2me2s from H3R2me2a, which impedes the recruitment of WDR5 to chromatin.  The crystallog. structure of WDR5 and the H3R2me2s peptide elucidates the mol. determinants of this high affinity interaction.  Our findings identify H3R2me2s as a previously unknown mark that keeps genes poised in euchromatin for transcriptional activation upon cell-cycle withdrawal and differentiation in human cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDe73jpv41ObVg90H21EOLACvtfcHk0lixmcla46x2Pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xls1ShtA%253D%253D&md5=c7b15ed7ee8f4819b45d488171e89ac4</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2209%26sid%3Dliteratum%253Aachs%26aulast%3DMigliori%26aufirst%3DV.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%26aulast%3DPhalke%26aufirst%3DS.%26aulast%3DLow%26aufirst%3DD.%26aulast%3DBezzi%26aufirst%3DM.%26aulast%3DMok%26aufirst%3DW.%2BC.%26aulast%3DSahu%26aufirst%3DS.%2BK.%26aulast%3DGunaratne%26aufirst%3DJ.%26aulast%3DCapasso%26aufirst%3DP.%26aulast%3DBassi%26aufirst%3DC.%26aulast%3DCecatiello%26aufirst%3DV.%26aulast%3DDe%2BMarco%26aufirst%3DA.%26aulast%3DBlackstock%26aufirst%3DW.%26aulast%3DKuznetsov%26aufirst%3DV.%26aulast%3DAmati%26aufirst%3DB.%26aulast%3DMapelli%26aufirst%3DM.%26aulast%3DGuccione%26aufirst%3DE.%26atitle%3DSymmetric%2520dimethylation%2520of%2520H3R2%2520is%2520a%2520newly%2520identified%2520histone%2520mark%2520that%2520supports%2520euchromatin%2520maintenance%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D136%26epage%3D144%26doi%3D10.1038%2Fnsmb.2209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Musiani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bok, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massignani, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabaglio, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ippolito, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuomo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozbek, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zorgati, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghoshdastider, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonaldi, T.</span></span> <span> </span><span class="NLM_article-title">Proteomics profiling of arginine methylation defines PRMT5 substrate specificity</span>. <i>Sci. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">eaat8388</span> <span class="refDoi"> DOI: 10.1126/scisignal.aat8388</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1126%2Fscisignal.aat8388" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=30940768" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2019&author=D.+Musianiauthor=J.+Bokauthor=E.+Massignaniauthor=L.+Wuauthor=T.+Tabaglioauthor=M.+R.+Ippolitoauthor=A.+Cuomoauthor=U.+Ozbekauthor=H.+Zorgatiauthor=U.+Ghoshdastiderauthor=R.+C.+Robinsonauthor=E.+Guccioneauthor=T.+Bonaldi&title=Proteomics+profiling+of+arginine+methylation+defines+PRMT5+substrate+specificity&doi=10.1126%2Fscisignal.aat8388"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscisignal.aat8388&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscisignal.aat8388%26sid%3Dliteratum%253Aachs%26aulast%3DMusiani%26aufirst%3DD.%26aulast%3DBok%26aufirst%3DJ.%26aulast%3DMassignani%26aufirst%3DE.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DTabaglio%26aufirst%3DT.%26aulast%3DIppolito%26aufirst%3DM.%2BR.%26aulast%3DCuomo%26aufirst%3DA.%26aulast%3DOzbek%26aufirst%3DU.%26aulast%3DZorgati%26aufirst%3DH.%26aulast%3DGhoshdastider%26aufirst%3DU.%26aulast%3DRobinson%26aufirst%3DR.%2BC.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DBonaldi%26aufirst%3DT.%26atitle%3DProteomics%2520profiling%2520of%2520arginine%2520methylation%2520defines%2520PRMT5%2520substrate%2520specificity%26jtitle%3DSci.%2520Signaling%26date%3D2019%26volume%3D12%26doi%3D10.1126%2Fscisignal.aat8388" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karkhanis, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbalzano, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sif, S.</span></span> <span> </span><span class="NLM_article-title">Versatility of PRMT5-induced methylation in growth control and development</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">633</span>– <span class="NLM_lpage">641</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2011.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1016%2Fj.tibs.2011.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=21975038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFejurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2011&pages=633-641&author=V.+Karkhanisauthor=Y.-J.+Huauthor=R.+A.+Baiocchiauthor=A.+N.+Imbalzanoauthor=S.+Sif&title=Versatility+of+PRMT5-induced+methylation+in+growth+control+and+development&doi=10.1016%2Fj.tibs.2011.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Versatility of PRMT5-induced methylation in growth control and development</span></div><div class="casAuthors">Karkhanis, Vrajesh; Hu, Yu-Jie; Baiocchi, Robert A.; Imbalzano, Anthony N.; Sif, Said</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">633-641</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Arginine methylation governs important cellular processes that impact growth and proliferation, as well as differentiation and development.  Through their ability to catalyze sym. or asym. methylation of histone and non-histone proteins, members of the protein arginine methyltransferase (PRMT) family regulate chromatin structure and expression of a wide spectrum of target genes.  Unlike other PRMTs, PRMT5 works in concert with a variety of cellular proteins including ATP-dependent chromatin remodelers and co-repressors to induce epigenetic silencing.  Recent work also implicates PRMT5 in the control of growth-promoting and pro-survival pathways, which demonstrates its versatility as an enzyme involved in both epigenetic regulation of anti-cancer target genes and organelle biogenesis.  These studies not only provide insight into the mol. mechanisms by which PRMT5 contributes to growth control, but also justify therapeutic targeting of PRMT5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD1--MutXQ2bVg90H21EOLACvtfcHk0lg07hrHHlYxLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFejurrJ&md5=e5cb228750165f39636118ddee2a0ee8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2011.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2011.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DKarkhanis%26aufirst%3DV.%26aulast%3DHu%26aufirst%3DY.-J.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DImbalzano%26aufirst%3DA.%2BN.%26aulast%3DSif%26aufirst%3DS.%26atitle%3DVersatility%2520of%2520PRMT5-induced%2520methylation%2520in%2520growth%2520control%2520and%2520development%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2011%26volume%3D36%26spage%3D633%26epage%3D641%26doi%3D10.1016%2Fj.tibs.2011.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blanc, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, S.</span></span> <span> </span><span class="NLM_article-title">Arginine methylation: the coming of age</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2016.11.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1016%2Fj.molcel.2016.11.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=28061334" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmsVKmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2017&pages=8-24&author=R.+S.+Blancauthor=S.+Richard&title=Arginine+methylation%3A+the+coming+of+age&doi=10.1016%2Fj.molcel.2016.11.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine Methylation: The Coming of Age</span></div><div class="casAuthors">Blanc, Romeo S.; Richard, Stephane</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8-24</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Arginine methylation is a common post-translational modification functioning as an epigenetic regulator of transcription and playing key roles in pre-mRNA splicing, DNA damage signaling, mRNA translation, cell signaling, and cell fate decision.  Recently, a wealth of studies using transgenic mouse models and selective PRMT inhibitors helped define physiol. roles for protein arginine methyltransferases (PRMTs) linking them to diseases such as cancer and metabolic, neurodegenerative, and muscular disorders.  This review describes the recent mol. advances that have been uncovered in normal and diseased mammalian cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqT2OQsTa2UnbVg90H21EOLACvtfcHk0lg07hrHHlYxLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmsVKmtA%253D%253D&md5=7ca3a6b7f0914c0034534ab8ca908506</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2016.11.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2016.11.003%26sid%3Dliteratum%253Aachs%26aulast%3DBlanc%26aufirst%3DR.%2BS.%26aulast%3DRichard%26aufirst%3DS.%26atitle%3DArginine%2520methylation%253A%2520the%2520coming%2520of%2520age%26jtitle%3DMol.%2520Cell%26date%3D2017%26volume%3D65%26spage%3D8%26epage%3D24%26doi%3D10.1016%2Fj.molcel.2016.11.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fong, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pignata, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goy, P.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawabata, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koh, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musiani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massignani, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotini, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wun, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rialdi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wollmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mzoughi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luciani, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wouters, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delwel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papapetrou, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minden, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonaldi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Wahab, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span> <span> </span><span class="NLM_article-title">Therapeutic targeting of RNA splicing catalysis through inhibition of protein arginine methylation</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">194</span>– <span class="NLM_lpage">209</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2019.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1016%2Fj.ccell.2019.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=31408619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFyltLbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2019&pages=194-209&author=J.+Y.+Fongauthor=L.+Pignataauthor=P.-A.+Goyauthor=K.+C.+Kawabataauthor=S.+C.-W.+Leeauthor=C.+M.+Kohauthor=D.+Musianiauthor=E.+Massignaniauthor=A.+G.+Kotiniauthor=A.+Pensonauthor=C.+M.+Wunauthor=Y.+Shenauthor=M.+Schwarzauthor=D.+H.+Lowauthor=A.+Rialdiauthor=M.+Kiauthor=H.+Wollmannauthor=S.+Mzoughiauthor=F.+Gayauthor=C.+Thompsonauthor=T.+Hartauthor=O.+Barbashauthor=G.+M.+Lucianiauthor=M.+M.+Szewczykauthor=B.+J.+Woutersauthor=R.+Delwelauthor=E.+P.+Papapetrouauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=M.+D.+Mindenauthor=J.+Jinauthor=A.+Melnickauthor=T.+Bonaldiauthor=O.+Abdel-Wahabauthor=E.+Guccione&title=Therapeutic+targeting+of+RNA+splicing+catalysis+through+inhibition+of+protein+arginine+methylation&doi=10.1016%2Fj.ccell.2019.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic Targeting of RNA Splicing Catalysis through Inhibition of Protein Arginine Methylation</span></div><div class="casAuthors">Fong, Jia Yi; Pignata, Luca; Goy, Pierre-Alexis; Kawabata, Kimihito Cojin; Lee, Stanley Chun-Wei; Koh, Cheryl M.; Musiani, Daniele; Massignani, Enrico; Kotini, Andriana G.; Penson, Alex; Wun, Cheng Mun; Shen, Yudao; Schwarz, Megan; Low, Diana H. P.; Rialdi, Alexander; Ki, Michelle; Wollmann, Heike; Mzoughi, Slim; Gay, Florence; Thompson, Christine; Hart, Timothy; Barbash, Olena; Luciani, Genna M.; Szewczyk, Magdalena M.; Wouters, Bas J.; Delwel, Ruud; Papapetrou, Eirini P.; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Minden, Mark D.; Jin, Jian; Melnick, Ari; Bonaldi, Tiziana; Abdel-Wahab, Omar; Guccione, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">194-209.e9</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Cancer-assocd. mutations in genes encoding RNA splicing factors (SFs) commonly occur in leukemias, as well as in a variety of solid tumors, and confer dependence on wild-type splicing.  These observations have led to clin. efforts to directly inhibit the spliceosome in patients with refractory leukemias.  Here, we identify that inhibiting sym. or asym. dimethylation of arginine, mediated by PRMT5 and type I protein arginine methyltransferases (PRMTs), resp., reduces splicing fidelity and results in preferential killing of SF-mutant leukemias over wild-type counterparts.  These data identify genetic subsets of cancer most likely to respond to PRMT inhibition, synergistic effects of combined PRMT5 and type I PRMT inhibition, and a mechanistic basis for the therapeutic efficacy of PRMT inhibition in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7gcqR_ejFw7Vg90H21EOLACvtfcHk0lg07hrHHlYxLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFyltLbO&md5=0d124279e457ee519f878f49cdc5b36b</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DFong%26aufirst%3DJ.%2BY.%26aulast%3DPignata%26aufirst%3DL.%26aulast%3DGoy%26aufirst%3DP.-A.%26aulast%3DKawabata%26aufirst%3DK.%2BC.%26aulast%3DLee%26aufirst%3DS.%2BC.-W.%26aulast%3DKoh%26aufirst%3DC.%2BM.%26aulast%3DMusiani%26aufirst%3DD.%26aulast%3DMassignani%26aufirst%3DE.%26aulast%3DKotini%26aufirst%3DA.%2BG.%26aulast%3DPenson%26aufirst%3DA.%26aulast%3DWun%26aufirst%3DC.%2BM.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSchwarz%26aufirst%3DM.%26aulast%3DLow%26aufirst%3DD.%2BH.%26aulast%3DRialdi%26aufirst%3DA.%26aulast%3DKi%26aufirst%3DM.%26aulast%3DWollmann%26aufirst%3DH.%26aulast%3DMzoughi%26aufirst%3DS.%26aulast%3DGay%26aufirst%3DF.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DHart%26aufirst%3DT.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DLuciani%26aufirst%3DG.%2BM.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DWouters%26aufirst%3DB.%2BJ.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DPapapetrou%26aufirst%3DE.%2BP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DMinden%26aufirst%3DM.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DBonaldi%26aufirst%3DT.%26aulast%3DAbdel-Wahab%26aufirst%3DO.%26aulast%3DGuccione%26aufirst%3DE.%26atitle%3DTherapeutic%2520targeting%2520of%2520RNA%2520splicing%2520catalysis%2520through%2520inhibition%2520of%2520protein%2520arginine%2520methylation%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26spage%3D194%26epage%3D209%26doi%3D10.1016%2Fj.ccell.2019.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferases and cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1038/nrc3409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fnrc3409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=23235912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVajur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=37-50&author=Y.+Yangauthor=M.+T.+Bedford&title=Protein+arginine+methyltransferases+and+cancer&doi=10.1038%2Fnrc3409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferases and cancer</span></div><div class="casAuthors">Yang, Yanzhong; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-50</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There are nine protein arginine methyltransferases (PRMTs) encoded in mammalian genomes, the protein products of which catalyze three types of arginine methylation - monomethylation and two types of dimethylation.  Protein arginine methylation is an abundant modification that has been implicated in signal transduction, gene transcription, DNA repair and mRNA splicing, among others.  Studies have only recently linked this modification to carcinogenesis and metastasis.  Sequencing studies have not generally found alterations to the PRMTs; however, overexpression of these enzymes is often assocd. with various cancers, which might make some of them viable targets for therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnb4O4wtn5fLVg90H21EOLACvtfcHk0lgCRKrnu8GZJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVajur7I&md5=9df3b58d92e7932e214e6de26a10fa46</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrc3409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3409%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DProtein%2520arginine%2520methyltransferases%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2013%26volume%3D13%26spage%3D37%26epage%3D50%26doi%3D10.1038%2Fnrc3409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stopa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krebs, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shechter, D.</span></span> <span> </span><span class="NLM_article-title">The PRMT5 arginine methyltransferase: many roles in development, cancer and beyond</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">2041</span>– <span class="NLM_lpage">2059</span>, <span class="refDoi"> DOI: 10.1007/s00018-015-1847-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1007%2Fs00018-015-1847-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=25662273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitlOrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2015&pages=2041-2059&author=N.+Stopaauthor=J.+E.+Krebsauthor=D.+Shechter&title=The+PRMT5+arginine+methyltransferase%3A+many+roles+in+development%2C+cancer+and+beyond&doi=10.1007%2Fs00018-015-1847-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The PRMT5 arginine methyltransferase- many roles in development, cancer and beyond</span></div><div class="casAuthors">Stopa, Nicole; Krebs, Jocelyn E.; Shechter, David</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2041-2059</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">Post-translational arginine methylation is responsible for regulation of many biol. processes.  The protein arginine methyltransferase 5 (PRMT5, also known as Hsl7, Jbp1, Skb1, Capsuleen, or Dart5) is the major enzyme responsible for mono- and sym. dimethylation of arginine.  An expanding literature demonstrates its crit. biol. function in a wide range of cellular processes.  Histone and other protein methylation by PRMT5 regulate genome organization, transcription, stem cells, primordial germ cells, differentiation, the cell cycle, and spliceosome assembly.  Metazoan PRMT5 is found in complex with the WD-repeat protein MEP50 (also known as Wdr77, androgen receptor coactivator p44, or Valois).  PRMT5 also directly assocs. with a range of other protein factors, including pICln, Menin, CoPR5 and RioK1 that may alter its subcellular localization and protein substrate selection.  Protein substrate and PRMT5-MEP50 post-translation modifications induce crosstalk to regulate PRMT5 activity.  Crystal structures of C. elegans PRMT5 and human and frog PRMT5-MEP50 complexes provide substantial insight into the mechanisms of substrate recognition and procession to dimethylation.  Enzymol. studies of PRMT5 have uncovered compelling insights essential for future development of specific PRMT5 inhibitors.  In addn., newly accumulating evidence implicates PRMT5 and MEP50 expression levels and their methyltransferase activity in cancer tumorigenesis, and, significantly, as markers of poor clin. outcome, marking them as potential oncogenes.  Here, we review the substantial new literature on PRMT5 and its partners to highlight the significance of understanding this essential enzyme in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGZo-Bwo40_7Vg90H21EOLACvtfcHk0lgCRKrnu8GZJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitlOrtr0%253D&md5=2173fff7825f142cde0ddde7f7b879f5</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs00018-015-1847-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-015-1847-9%26sid%3Dliteratum%253Aachs%26aulast%3DStopa%26aufirst%3DN.%26aulast%3DKrebs%26aufirst%3DJ.%2BE.%26aulast%3DShechter%26aufirst%3DD.%26atitle%3DThe%2520PRMT5%2520arginine%2520methyltransferase%253A%2520many%2520roles%2520in%2520development%252C%2520cancer%2520and%2520beyond%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2015%26volume%3D72%26spage%3D2041%26epage%3D2059%26doi%3D10.1007%2Fs00018-015-1847-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richters, A.</span></span> <span> </span><span class="NLM_article-title">Targeting protein arginine methyltransferase 5 in disease</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2081</span>– <span class="NLM_lpage">2098</span>, <span class="refDoi"> DOI: 10.4155/fmc-2017-0089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.4155%2Ffmc-2017-0089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=29076773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOit7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=2081-2098&author=A.+Richters&title=Targeting+protein+arginine+methyltransferase+5+in+disease&doi=10.4155%2Ffmc-2017-0089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein arginine methyltransferase 5 in disease</span></div><div class="casAuthors">Richters, Andre</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2081-2098</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">PRMT5 catalyzes the mono- and sym. dimethylation of the arginine N-guanidine group of a wide variety of target proteins including histones, transcriptional elongation factors, kinases and tumor suppressors by utilizing the essential co-factor S-adenosylmethionine as Me source.  PRMT5 overexpression has been linked to the progression of various diseases, including cancer, and is oftentimes assocd. with a poor prognosis.  Therefore, PRMT5 is promoted as a valuable target for drug discovery approaches and was a subject matter in recent endeavors aiming for the development of specific PRMT5 inhibitors.  This review will embrace the significance of PRMT5 as therapeutic target with respect to its mol. interdependencies in disease states as well as its implication in drug development approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqverMGPRhuJbVg90H21EOLACvtfcHk0lgCRKrnu8GZJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOit7fN&md5=6f8f3e27b9d7389e5b89ff084e6b691e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2017-0089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2017-0089%26sid%3Dliteratum%253Aachs%26aulast%3DRichters%26aufirst%3DA.%26atitle%3DTargeting%2520protein%2520arginine%2520methyltransferase%25205%2520in%2520disease%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D2081%26epage%3D2098%26doi%3D10.4155%2Ffmc-2017-0089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCorvy, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wacker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agegnehu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lansu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tribo, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, R. H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span> <span> </span><span class="NLM_article-title">Structural determinants of 5-HT2B receptor activation and biased agonism</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">787</span>– <span class="NLM_lpage">796</span>, <span class="refDoi"> DOI: 10.1038/s41594-018-0116-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fs41594-018-0116-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=30127358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFGjtLjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=787-796&author=J.+D.+McCorvyauthor=D.+Wackerauthor=S.+Wangauthor=B.+Agegnehuauthor=J.+Liuauthor=K.+Lansuauthor=A.+R.+Triboauthor=R.+H.+J.+Olsenauthor=T.+Cheauthor=J.+Jinauthor=B.+L.+Roth&title=Structural+determinants+of+5-HT2B+receptor+activation+and+biased+agonism&doi=10.1038%2Fs41594-018-0116-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structural determinants of 5-HT2B receptor activation and biased agonism</span></div><div class="casAuthors">McCorvy, John D.; Wacker, Daniel; Wang, Sheng; Agegnehu, Bemnat; Liu, Jing; Lansu, Katherine; Tribo, Alexandra R.; Olsen, Reid H. J.; Che, Tao; Jin, Jian; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">787-796</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Serotonin (5-hydroxytryptamine; 5-HT) receptors modulate a variety of physiol. processes ranging from perception, cognition and emotion to vascular and smooth muscle contraction, platelet aggregation, gastrointestinal function and reprodn.  Drugs that interact with 5-HT receptors effectively treat diseases as diverse as migraine headaches, depression and obesity.  Here we present four structures of a prototypical serotonin receptor-the human 5-HT2B receptor-in complex with chem. and pharmacol. diverse drugs, including methysergide, methylergonovine, lisuride and LY266097.  A detailed anal. of these structures complemented by comprehensive interrogation of signaling illuminated key structural determinants essential for activation.  Addnl. structure-guided mutagenesis expts. revealed binding pocket residues that were essential for agonist-mediated biased signaling and β-arrestin2 translocation.  Given the importance of 5-HT receptors for a large no. of therapeutic indications, insights derived from these studies should accelerate the design of safer and more effective medications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwC5QTOdEZhLVg90H21EOLACvtfcHk0lgCRKrnu8GZJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFGjtLjM&md5=93be9401bc680249499fa3af5d40f395</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fs41594-018-0116-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41594-018-0116-7%26sid%3Dliteratum%253Aachs%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DWacker%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DAgegnehu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLansu%26aufirst%3DK.%26aulast%3DTribo%26aufirst%3DA.%2BR.%26aulast%3DOlsen%26aufirst%3DR.%2BH.%2BJ.%26aulast%3DChe%26aufirst%3DT.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DStructural%2520determinants%2520of%25205-HT2B%2520receptor%2520activation%2520and%2520biased%2520agonism%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D787%26epage%3D796%26doi%3D10.1038%2Fs41594-018-0116-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shimizu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takami, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koike, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodera, Y.</span></span> <span> </span><span class="NLM_article-title">The protein arginine methyltransferase 5 promotes malignant phenotype of hepatocellular carcinoma cells and is associated with adverse patient outcomes after curative hepatectomy</span>. <i>Int. J. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">381</span>– <span class="NLM_lpage">386</span>, <span class="refDoi"> DOI: 10.3892/ijo.2017.3833</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.3892%2Fijo.2017.3833" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=28101581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitleqtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2017&pages=381-386&author=D.+Shimizuauthor=M.+Kandaauthor=H.+Sugimotoauthor=M.+Shibataauthor=H.+Tanakaauthor=H.+Takamiauthor=N.+Iwataauthor=M.+Hayashiauthor=C.+Tanakaauthor=D.+Kobayashiauthor=S.+Yamadaauthor=G.+Nakayamaauthor=M.+Koikeauthor=M.+Fujiwaraauthor=T.+Fujiiauthor=Y.+Kodera&title=The+protein+arginine+methyltransferase+5+promotes+malignant+phenotype+of+hepatocellular+carcinoma+cells+and+is+associated+with+adverse+patient+outcomes+after+curative+hepatectomy&doi=10.3892%2Fijo.2017.3833"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The protein arginine methyltransferase 5 promotes malignant phenotype of hepatocellular carcinoma cells and is associated with adverse patient outcomes after curative hepatectomy</span></div><div class="casAuthors">Shimizu, Dai; Kanda, Mitsuro; Sugimoto, Hiroyuki; Shibata, Masahiro; Tanaka, Haruyoshi; Takami, Hideki; Iwata, Naoki; Hayashi, Masamichi; Tanaka, Chie; Kobayashi, Daisuke; Yamada, Suguru; Nakayama, Goro; Koike, Masahiko; Fujiwara, Michitaka; Fujii, Tsutomu; Ko, Yasuhiro</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">381-386</span>CODEN:
                <span class="NLM_cas:coden">IJONES</span>;
        ISSN:<span class="NLM_cas:issn">1791-2423</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">The prognosis of advanced hepatocellular carcinoma (HCC) is dismal.  Novel mol. targets for diagnosis and therapy is urgently required.  This study evaluated expression and functions of the protein arginine methyltransferase 5 (PRMT5) in HCC.  Using HCC cell lines, the expression levels of PRMT5 mRNA were detd. using the quant. real-time reverse-transcription polymerase chain reaction, and the effect of a small interfering PRMT5-siRNA on cell phenotype was evaluated.  Further, PRMT5 expression was detd. in 144 pairs of resected liver tissues to evaluate its clin. significance.  Regardless of their differentiated phenotypes, nine HCC cell lines expressed different levels of PRMT5 mRNA.  Inhibition of PRMT5 expression significantly decreased the proliferation, invasion, and migration of HCC cell lines.  Although the level of PRMT5 mRNA was not influenced by patient's background liver status, it was significantly higher in HCC tissues than in the corresponding noncancerous tissues.  High levels of PRMT5 mRNA in HCC tissues were significantly assocd. with advanced disease stage and adverse prognosis.  In conclusion, our results indicate that PRMT5 may act as a putative oncogene in HCC and that the levels of PRMT5 mRNA represent a promising prognostic marker and a potential target of mol. therapy for HCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgqfuS0Y9NMLVg90H21EOLACvtfcHk0lgLLdsxauZlhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitleqtbc%253D&md5=080eb5763b9c1b1a61ab981d3e422971</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.3892%2Fijo.2017.3833&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijo.2017.3833%26sid%3Dliteratum%253Aachs%26aulast%3DShimizu%26aufirst%3DD.%26aulast%3DKanda%26aufirst%3DM.%26aulast%3DSugimoto%26aufirst%3DH.%26aulast%3DShibata%26aufirst%3DM.%26aulast%3DTanaka%26aufirst%3DH.%26aulast%3DTakami%26aufirst%3DH.%26aulast%3DIwata%26aufirst%3DN.%26aulast%3DHayashi%26aufirst%3DM.%26aulast%3DTanaka%26aufirst%3DC.%26aulast%3DKobayashi%26aufirst%3DD.%26aulast%3DYamada%26aufirst%3DS.%26aulast%3DNakayama%26aufirst%3DG.%26aulast%3DKoike%26aufirst%3DM.%26aulast%3DFujiwara%26aufirst%3DM.%26aulast%3DFujii%26aufirst%3DT.%26aulast%3DKodera%26aufirst%3DY.%26atitle%3DThe%2520protein%2520arginine%2520methyltransferase%25205%2520promotes%2520malignant%2520phenotype%2520of%2520hepatocellular%2520carcinoma%2520cells%2520and%2520is%2520associated%2520with%2520adverse%2520patient%2520outcomes%2520after%2520curative%2520hepatectomy%26jtitle%3DInt.%2520J.%2520Oncol.%26date%3D2017%26volume%3D50%26spage%3D381%26epage%3D386%26doi%3D10.3892%2Fijo.2017.3833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elzey, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pili, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratliff, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, C.-D.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferase 5 functions as an epigenetic activator of the androgen receptor to promote prostate cancer cell growth</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1223</span>– <span class="NLM_lpage">1231</span>, <span class="refDoi"> DOI: 10.1038/onc.2016.287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fonc.2016.287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=27546619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlyisLvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=1223-1231&author=X.+Dengauthor=G.+Shaoauthor=H.-T.+Zhangauthor=C.+Liauthor=D.+Zhangauthor=L.+Chengauthor=B.+D.+Elzeyauthor=R.+Piliauthor=T.+L.+Ratliffauthor=J.+Huangauthor=C.-D.+Hu&title=Protein+arginine+methyltransferase+5+functions+as+an+epigenetic+activator+of+the+androgen+receptor+to+promote+prostate+cancer+cell+growth&doi=10.1038%2Fonc.2016.287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferase 5 functions as an epigenetic activator of the androgen receptor to promote prostate cancer cell growth</span></div><div class="casAuthors">Deng, X.; Shao, G.; Zhang, H.-T.; Li, C.; Zhang, D.; Cheng, L.; Elzey, B. D.; Pili, R.; Ratliff, T. L.; Huang, J.; Hu, C.-D.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1223-1231</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is an emerging epigenetic enzyme that mainly represses transcription of target genes via sym. dimethylation of arginine residues on histones H4R3, H3R8 and H2AR3.  Accumulating evidence suggests that PRMT5 may function as an oncogene to drive cancer cell growth by epigenetic inactivation of several tumor suppressors.  Here, we provide evidence that PRMT5 promotes prostate cancer cell growth by epigenetically activating transcription of the androgen receptor (AR) in prostate cancer cells.  Knockdown of PRMT5 or inhibition of PRMT5 by a specific inhibitor reduces the expression of AR and suppresses the growth of multiple AR-pos., but not AR-neg., prostate cancer cells.  Significantly, knockdown of PRMT5 in AR-pos. LNCaP cells completely suppresses the growth of xenograft tumors in mice.  Mol. anal. reveals that PRMT5 binds to the proximal promoter region of the AR gene and contributes mainly to the enriched sym. dimethylation of H4R3 in the same region.  Mechanistically, PRMT5 is recruited to the AR promoter by its interaction with Sp1, the major transcription factor responsible for AR transcription, and forms a complex with Brg1, an ATP-dependent chromatin remodeler, on the proximal promoter region of the AR gene.  Furthermore, PRMT5 expression in prostate cancer tissues is significantly higher than that in benign prostatic hyperplasia tissues, and PRMT5 expression correlates pos. with AR expression at both the protein and mRNA levels.  Taken together, our results identify PRMT5 as a novel epigenetic activator of AR in prostate cancer.  Given that inhibiting AR transcriptional activity or androgen synthesis remains the major mechanism of action for most existing anti-androgen agents, our findings also raise an interesting possibility that targeting PRMT5 may represent a novel approach for prostate cancer treatment by eliminating AR expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6wjniAjJYjbVg90H21EOLACvtfcHk0lgLLdsxauZlhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlyisLvK&md5=75aff420e6c8a819bd41fd45cec7dec9</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fonc.2016.287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2016.287%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DShao%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DH.-T.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DCheng%26aufirst%3DL.%26aulast%3DElzey%26aufirst%3DB.%2BD.%26aulast%3DPili%26aufirst%3DR.%26aulast%3DRatliff%26aufirst%3DT.%2BL.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DC.-D.%26atitle%3DProtein%2520arginine%2520methyltransferase%25205%2520functions%2520as%2520an%2520epigenetic%2520activator%2520of%2520the%2520androgen%2520receptor%2520to%2520promote%2520prostate%2520cancer%2520cell%2520growth%26jtitle%3DOncogene%26date%3D2017%26volume%3D36%26spage%3D1223%26epage%3D1231%26doi%3D10.1038%2Fonc.2016.287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, J.</span></span> <span> </span><span class="NLM_article-title">Targeting methyltransferase PRMT5 eliminates leukemia stem cells in chronic myelogenous leukemia</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">3961</span>– <span class="NLM_lpage">3980</span>, <span class="refDoi"> DOI: 10.1172/jci85239</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1172%2FJCI85239" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=27643437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A280%3ADC%252BC2svhs12juw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2016&pages=3961-3980&author=Y.+Jinauthor=J.+Zhouauthor=F.+Xuauthor=B.+Jinauthor=L.+Cuiauthor=Y.+Wangauthor=X.+Duauthor=J.+Liauthor=P.+Liauthor=R.+Renauthor=J.+Pan&title=Targeting+methyltransferase+PRMT5+eliminates+leukemia+stem+cells+in+chronic+myelogenous+leukemia&doi=10.1172%2Fjci85239"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting methyltransferase PRMT5 eliminates leukemia stem cells in chronic myelogenous leukemia</span></div><div class="casAuthors">Jin Yanli; Zhou Jingfeng; Xu Fang; Jin Bei; Cui Lijing; Wang Yun; Du Xin; Li Juan; Li Peng; Ren Ruibao; Pan Jingxuan</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3961-3980</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Imatinib-insensitive leukemia stem cells (LSCs) are believed to be responsible for resistance to BCR-ABL tyrosine kinase inhibitors and relapse of chronic myelogenous leukemia (CML).  Identifying therapeutic targets to eradicate CML LSCs may be a strategy to cure CML.  In the present study, we discovered a positive feedback loop between BCR-ABL and protein arginine methyltransferase 5 (PRMT5) in CML cells.  Overexpression of PRMT5 was observed in human CML LSCs.  Silencing PRMT5 with shRNA or blocking PRMT5 methyltransferase activity with the small-molecule inhibitor PJ-68 reduced survival, serial replating capacity, and long-term culture-initiating cells (LTC-ICs) in LSCs from CML patients.  Further, PRMT5 knockdown or PJ-68 treatment dramatically prolonged survival in a murine model of retroviral BCR-ABL-driven CML and impaired the in vivo self-renewal capacity of transplanted CML LSCs.  PJ-68 also inhibited long-term engraftment of human CML CD34+ cells in immunodeficient mice.  Moreover, inhibition of PRMT5 abrogated the Wnt/β-catenin pathway in CML CD34+ cells by depleting dishevelled homolog 3 (DVL3).  This study suggests that epigenetic methylation modification on histone protein arginine residues is a regulatory mechanism to control self-renewal of LSCs and indicates that PRMT5 may represent a potential therapeutic target against LSCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTTlS-7r3kmkRJ5E29YNGvQfW6udTcc2eZ3J3Eks85zi7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2svhs12juw%253D%253D&md5=906aacd5086992e34c0a08f271e3e56a</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1172%2FJCI85239&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI85239%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DJin%26aufirst%3DB.%26aulast%3DCui%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DRen%26aufirst%3DR.%26aulast%3DPan%26aufirst%3DJ.%26atitle%3DTargeting%2520methyltransferase%2520PRMT5%2520eliminates%2520leukemia%2520stem%2520cells%2520in%2520chronic%2520myelogenous%2520leukemia%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2016%26volume%3D126%26spage%3D3961%26epage%3D3980%26doi%3D10.1172%2Fjci85239" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomie, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seibold, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumball, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capitini, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rui, L.</span></span> <span> </span><span class="NLM_article-title">PRMT5 is upregulated by B-cell receptor signaling and forms a positive-feedback loop with PI3K/AKT in lymphoma cells</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">2898</span>– <span class="NLM_lpage">2911</span>, <span class="refDoi"> DOI: 10.1038/s41375-019-0489-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fs41375-019-0489-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=31123343" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVyqs7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=2898-2911&author=F.+Zhuauthor=H.+Guoauthor=P.+D.+Batesauthor=S.+Zhangauthor=H.+Zhangauthor=K.+J.+Nomieauthor=Y.+Liauthor=L.+Luauthor=K.+R.+Seiboldauthor=F.+Wangauthor=I.+Rumballauthor=H.+Cameronauthor=N.+M.+Hoangauthor=D.+T.+Yangauthor=W.+Xuauthor=L.+Zhangauthor=M.+Wangauthor=C.+M.+Capitiniauthor=L.+Rui&title=PRMT5+is+upregulated+by+B-cell+receptor+signaling+and+forms+a+positive-feedback+loop+with+PI3K%2FAKT+in+lymphoma+cells&doi=10.1038%2Fs41375-019-0489-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT5 is upregulated by B-cell receptor signaling and forms a positive-feedback loop with PI3K/AKT in lymphoma cells</span></div><div class="casAuthors">Zhu, Fen; Guo, Hui; Bates, Paul D.; Zhang, Shanxiang; Zhang, Hui; Nomie, Krystle J.; Li, Yangguang; Lu, Li; Seibold, Kaitlyn R.; Wang, Fangyu; Rumball, Ian; Cameron, Hunter; Hoang, Nguyet M.; Yang, David T.; Xu, Wei; Zhang, Liang; Wang, Michael; Capitini, Christian M.; Rui, Lixin</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2898-2911</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">PRMT5, which regulates gene expression by sym. dimethylation of histones and non-histone target proteins, is overexpressed and plays a pathogenic role in many cancers.  In diffuse large B cell lymphoma (DLBCL), the mechanisms of PRMT5 dysregulation and its role in lymphomagenesis remain largely unknown.  Here we demonstrate that B cell receptor (BCR) signaling regulates PRMT5 expression in DLBCL cells.  Immunohistochem. anal. reveals elevated levels of PRMT5 expression in DLBCL cases and in germinal center (GC) B cells when compared to naive B cells.  PRMT5 can be induced in naive B cells by BCR stimulation.  We discovered that BTK-NF-κB signaling induces PRMT5 transcription in activated B cell-like (ABC) DLBCL cells while BCR downstream PI3K-AKT-MYC signaling upregulates PRMT5 expression in both ABC and GCB DLBCL cells.  PRMT5 inhibition inhibits the growth of DLBCL cells in vitro and patient derived xenografts.  Genomic and biochem. anal. demonstrate that PRMT5 promotes cell cycle progression and activates PI3K-AKT signaling, suggesting a feedback regulatory mechanism to enhance cell survival and proliferation.  Co-targeting PRMT5 and AKT by their specific inhibitors is lethal to DLBCL cell lines and primary cancer cells.  Therefore, this study provides a mechanistic rationale for clin. trials to evaluate PRMT5 and AKT inhibitors for DLBCL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3qt1S5OemMLVg90H21EOLACvtfcHk0liTVS6HEZSb3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVyqs7nN&md5=b4ad5f3eb84cf8b8fa9a05828db51128</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fs41375-019-0489-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41375-019-0489-6%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DF.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DBates%26aufirst%3DP.%2BD.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DNomie%26aufirst%3DK.%2BJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DSeibold%26aufirst%3DK.%2BR.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DRumball%26aufirst%3DI.%26aulast%3DCameron%26aufirst%3DH.%26aulast%3DHoang%26aufirst%3DN.%2BM.%26aulast%3DYang%26aufirst%3DD.%2BT.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DCapitini%26aufirst%3DC.%2BM.%26aulast%3DRui%26aufirst%3DL.%26atitle%3DPRMT5%2520is%2520upregulated%2520by%2520B-cell%2520receptor%2520signaling%2520and%2520forms%2520a%2520positive-feedback%2520loop%2520with%2520PI3K%252FAKT%2520in%2520lymphoma%2520cells%26jtitle%3DLeukemia%26date%3D2019%26volume%3D33%26spage%3D2898%26epage%3D2911%26doi%3D10.1038%2Fs41375-019-0489-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koh, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bezzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, D. H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ang, W. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teo, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, F. P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Haddawi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabò, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amati, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wee, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span> <span> </span><span class="NLM_article-title">MYC regulates the core pre-mRNA splicing machinery as an essential step in lymphomagenesis</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>523</i></span>,  <span class="NLM_fpage">96</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1038/nature14351</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fnature14351" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=25970242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFaitrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=523&publication_year=2015&pages=96-100&author=C.+M.+Kohauthor=M.+Bezziauthor=D.+H.+P.+Lowauthor=W.+X.+Angauthor=S.+X.+Teoauthor=F.+P.+H.+Gayauthor=M.+Al-Haddawiauthor=S.+Y.+Tanauthor=M.+Osatoauthor=A.+Sab%C3%B2author=B.+Amatiauthor=K.+B.+Weeauthor=E.+Guccione&title=MYC+regulates+the+core+pre-mRNA+splicing+machinery+as+an+essential+step+in+lymphomagenesis&doi=10.1038%2Fnature14351"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">MYC regulates the core pre-mRNA splicing machinery as an essential step in lymphomagenesis</span></div><div class="casAuthors">Koh, Cheryl M.; Bezzi, Marco; Low, Diana H. P.; Ang, Wei Xia; Teo, Shun Xie; Gay, Florence P. H.; Al-Haddawi, Muthafar; Tan, Soo Yong; Osato, Motomi; Sabo, Arianna; Amati, Bruno; Wee, Keng Boon; Guccione, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">523</span>
        (<span class="NLM_cas:issue">7558</span>),
    <span class="NLM_cas:pages">96-100</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Deregulated expression of the MYC transcription factor occurs in most human cancers and correlates with high proliferation, reprogrammed cellular metab. and poor prognosis.  Overexpressed MYC binds to virtually all active promoters within a cell, although with different binding affinities, and modulates the expression of distinct subsets of genes.  However, the crit. effectors of MYC in tumorigenesis remain largely unknown.  Here the authors show that during lymphomagenesis in Eμ-myc transgenic mice, MYC directly upregulates the transcription of the core small nuclear ribonucleoprotein particle assembly genes, including Prmt5, an arginine methyltransferase that methylates Sm proteins.  This coordinated regulatory effect is crit. for the core biogenesis of small nuclear ribonucleoprotein particles, effective pre-messenger-RNA splicing, cell survival and proliferation.  The authors' results demonstrate that MYC maintains the splicing fidelity of exons with a weak 5' donor site.  Addnl., the authors identify pre-messenger-RNAs that are particularly sensitive to the perturbation of the MYC-PRMT5 axis, resulting in either intron retention (for example, Dvl1) or exon skipping (for example, Atr, Ep400).  Using antisense oligonucleotides, the authors demonstrate the contribution of these splicing defects to the anti-proliferative/apoptotic phenotype obsd. in PRMT5-depleted Eμ-myc B cells.  The authors conclude that, in addn. to its well-documented oncogenic functions in transcription and translation, MYC also safeguards proper pre-messenger-RNA splicing as an essential step in lymphomagenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprbG6Q1B6f8rVg90H21EOLACvtfcHk0liTVS6HEZSb3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFaitrrO&md5=4fd14f9728e96c73f6ded4a0e0b1d0fc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnature14351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14351%26sid%3Dliteratum%253Aachs%26aulast%3DKoh%26aufirst%3DC.%2BM.%26aulast%3DBezzi%26aufirst%3DM.%26aulast%3DLow%26aufirst%3DD.%2BH.%2BP.%26aulast%3DAng%26aufirst%3DW.%2BX.%26aulast%3DTeo%26aufirst%3DS.%2BX.%26aulast%3DGay%26aufirst%3DF.%2BP.%2BH.%26aulast%3DAl-Haddawi%26aufirst%3DM.%26aulast%3DTan%26aufirst%3DS.%2BY.%26aulast%3DOsato%26aufirst%3DM.%26aulast%3DSab%25C3%25B2%26aufirst%3DA.%26aulast%3DAmati%26aufirst%3DB.%26aulast%3DWee%26aufirst%3DK.%2BB.%26aulast%3DGuccione%26aufirst%3DE.%26atitle%3DMYC%2520regulates%2520the%2520core%2520pre-mRNA%2520splicing%2520machinery%2520as%2520an%2520essential%2520step%2520in%2520lymphomagenesis%26jtitle%3DNature%26date%3D2015%26volume%3D523%26spage%3D96%26epage%3D100%26doi%3D10.1038%2Fnature14351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alinari, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lustberg, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katherine Martin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordero-Nieves, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banasavadi-Siddegowda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Virk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnholtz-Sloan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakravarti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowicki, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapalombella, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Datta, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haseley, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patton, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcucci, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuovo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrd, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiocca, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sif, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span> <span> </span><span class="NLM_article-title">Genetic validation of the protein arginine methyltransferase PRMT5 as a candidate therapeutic target in glioblastoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">1752</span>– <span class="NLM_lpage">1765</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-13-0884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1158%2F0008-5472.CAN-13-0884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=24453002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktF2murY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=1752-1765&author=F.+Yanauthor=L.+Alinariauthor=M.+E.+Lustbergauthor=L.+Katherine+Martinauthor=H.+M.+Cordero-Nievesauthor=Y.+Banasavadi-Siddegowdaauthor=S.+Virkauthor=J.+Barnholtz-Sloanauthor=E.+H.+Bellauthor=J.+Wojtonauthor=N.+K.+Jacobauthor=A.+Chakravartiauthor=M.+O.+Nowickiauthor=X.+Wuauthor=R.+Lapalombellaauthor=J.+Dattaauthor=B.+Yuauthor=K.+Gordonauthor=A.+Haseleyauthor=J.+T.+Pattonauthor=P.+L.+Smithauthor=J.+Ryuauthor=X.+Zhangauthor=X.+Moauthor=G.+Marcucciauthor=G.+Nuovoauthor=C.-H.+Kwonauthor=J.+C.+Byrdauthor=E.+A.+Chioccaauthor=C.+Liauthor=S.+Sifauthor=S.+Jacobauthor=S.+Lawlerauthor=B.+Kaurauthor=R.+A.+Baiocchi&title=Genetic+validation+of+the+protein+arginine+methyltransferase+PRMT5+as+a+candidate+therapeutic+target+in+glioblastoma&doi=10.1158%2F0008-5472.can-13-0884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic Validation of the Protein Arginine Methyltransferase PRMT5 as a Candidate Therapeutic Target in Glioblastoma</span></div><div class="casAuthors">Yan, Fengting; Alinari, Lapo; Lustberg, Mark E.; Martin, Ludmila Katherine; Cordero-Nieves, Hector M.; Banasavadi-Siddegowda, Yeshavanth; Virk, Selene; Barnholtz-Sloan, Jill; Bell, Erica Hlavin; Wojton, Jeffrey; Jacob, Naduparambil K.; Chakravarti, Arnab; Nowicki, Michal O.; Wu, Xin; Lapalombella, Rosa; Datta, Jharna; Yu, Bo; Gordon, Kate; Haseley, Amy; Patton, John T.; Smith, Porsha L.; Ryu, John; Zhang, Xiaoli; Mo, Xiaokui; Marcucci, Guido; Nuovo, Gerard; Kwon, Chang-Hyuk; Byrd, John C.; Chiocca, E. Antonio; Li, Chenglong; Sif, Said; Jacob, Samson; Lawler, Sean; Kaur, Balveen; Baiocchi, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1752-1765</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Glioblastoma is the most common and aggressive histol. subtype of brain cancer with poor outcomes and limited treatment options.  Here, we report the selective overexpression of the protein arginine methyltransferase PRMT5 as a novel candidate theranostic target in this disease.  PRMT5 silences the transcription of regulatory genes by catalyzing sym. dimethylation of arginine residues on histone tails.  PRMT5 overexpression in patient-derived primary tumors and cell lines correlated with cell line growth rate and inversely with overall patient survival.  Genetic attenuation of PRMT5 led to cell-cycle arrest, apoptosis, and loss of cell migratory activity.  Cell death was p53-independent but caspase-dependent and enhanced with temozolomide, a chemotherapeutic agent used as a present std. of care.  Global gene profiling and chromatin immunopptn. identified the tumor suppressor ST7 as a key gene silenced by PRMT5.  Diminished ST7 expression was assocd. with reduced patient survival.  PRMT5 attenuation limited PRMT5 recruitment to the ST7 promoter, led to restored expression of ST7 and cell growth inhibition.  Finally, PRMT5 attenuation enhanced glioblastoma cell survival in a mouse xenograft model of aggressive glioblastoma.  Together, our findings defined PRMT5 as a candidate prognostic factor and therapeutic target in glioblastoma, offering a preclin. justification for targeting PRMT5-driven oncogenic pathways in this deadly disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtN9wQGNGjE7Vg90H21EOLACvtfcHk0liTVS6HEZSb3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktF2murY%253D&md5=c38d456f2a4a7ffeab43a6fe5341e63c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-0884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-0884%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DF.%26aulast%3DAlinari%26aufirst%3DL.%26aulast%3DLustberg%26aufirst%3DM.%2BE.%26aulast%3DKatherine%2BMartin%26aufirst%3DL.%26aulast%3DCordero-Nieves%26aufirst%3DH.%2BM.%26aulast%3DBanasavadi-Siddegowda%26aufirst%3DY.%26aulast%3DVirk%26aufirst%3DS.%26aulast%3DBarnholtz-Sloan%26aufirst%3DJ.%26aulast%3DBell%26aufirst%3DE.%2BH.%26aulast%3DWojton%26aufirst%3DJ.%26aulast%3DJacob%26aufirst%3DN.%2BK.%26aulast%3DChakravarti%26aufirst%3DA.%26aulast%3DNowicki%26aufirst%3DM.%2BO.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DLapalombella%26aufirst%3DR.%26aulast%3DDatta%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DGordon%26aufirst%3DK.%26aulast%3DHaseley%26aufirst%3DA.%26aulast%3DPatton%26aufirst%3DJ.%2BT.%26aulast%3DSmith%26aufirst%3DP.%2BL.%26aulast%3DRyu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DMo%26aufirst%3DX.%26aulast%3DMarcucci%26aufirst%3DG.%26aulast%3DNuovo%26aufirst%3DG.%26aulast%3DKwon%26aufirst%3DC.-H.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DChiocca%26aufirst%3DE.%2BA.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DSif%26aufirst%3DS.%26aulast%3DJacob%26aufirst%3DS.%26aulast%3DLawler%26aufirst%3DS.%26aulast%3DKaur%26aufirst%3DB.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26atitle%3DGenetic%2520validation%2520of%2520the%2520protein%2520arginine%2520methyltransferase%2520PRMT5%2520as%2520a%2520candidate%2520therapeutic%2520target%2520in%2520glioblastoma%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26spage%3D1752%26epage%3D1765%26doi%3D10.1158%2F0008-5472.can-13-0884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan-Penebre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuplast, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stickland, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurse, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oza, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K. W.</span></span> <span> </span><span class="NLM_article-title">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">432</span>– <span class="NLM_lpage">437</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1810</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fnchembio.1810" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=25915199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=432-437&author=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=C.+R.+Majerauthor=P.+A.+Boriack-Sjodinauthor=T.+J.+Wigleauthor=L.+D.+Johnstonauthor=N.+Riouxauthor=M.+J.+Munchhofauthor=L.+Jinauthor=S.+L.+Jacquesauthor=K.+A.+Westauthor=T.+Lingarajauthor=K.+Sticklandauthor=S.+A.+Ribichauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+M.+Pollockauthor=J.+J.+Smithauthor=O.+Barbashauthor=M.+Pappalardiauthor=T.+F.+Hoauthor=K.+Nurseauthor=K.+P.+Ozaauthor=K.+T.+Gallagherauthor=R.+Krugerauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworthauthor=K.+W.+Duncan&title=A+selective+inhibitor+of+PRMT5+with+in+vivo+and+in+vitro+potency+in+MCL+models&doi=10.1038%2Fnchembio.1810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span></div><div class="casAuthors">Chan-Penebre, Elayne; Kuplast, Kristy G.; Majer, Christina R.; Boriack-Sjodin, P. Ann; Wigle, Tim J.; Johnston, L. Danielle; Rioux, Nathalie; Munchhof, Michael J.; Jin, Lei; Jacques, Suzanne L.; West, Kip A.; Lingaraj, Trupti; Stickland, Kimberly; Ribich, Scott A.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Pollock, Roy M.; Smith, Jesse J.; Barbash, Olena; Pappalardi, Melissa; Ho, Thau F.; Nurse, Kelvin; Oza, Khyati P.; Gallagher, Kathleen T.; Kruger, Ryan; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard; Duncan, Kenneth W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">432-437</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase-5 (PRMT5) is reported to have a role in diverse cellular processes, including tumorigenesis, and its overexpression is obsd. in cell lines and primary patient samples derived from lymphomas, particularly mantle cell lymphoma (MCL).  Here we describe the identification and characterization of a potent and selective inhibitor of PRMT5 with antiproliferative effects in both in vitro and in vivo models of MCL.  EPZ015666 (GSK3235025) is an orally available inhibitor of PRMT5 enzymic activity in biochem. assays with a half-maximal inhibitory concn. (IC50) of 22 nM and broad selectivity against a panel of other histone methyltransferases.  Treatment of MCL cell lines with EPZ015666 led to inhibition of SmD3 methylation and cell death, with IC50 values in the nanomolar range.  Oral dosing with EPZ015666 demonstrated dose-dependent antitumor activity in multiple MCL xenograft models.  EPZ015666 represents a validated chem. probe for further study of PRMT5 biol. and arginine methylation in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTtcvcqvOJLVg90H21EOLACvtfcHk0lgUozW8mBTPzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D&md5=bf0686e1c2d9d96ac5deef22c0bf80b7</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1810%26sid%3Dliteratum%253Aachs%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%2BA.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DStickland%26aufirst%3DK.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DPappalardi%26aufirst%3DM.%26aulast%3DHo%26aufirst%3DT.%2BF.%26aulast%3DNurse%26aufirst%3DK.%26aulast%3DOza%26aufirst%3DK.%2BP.%26aulast%3DGallagher%26aufirst%3DK.%2BT.%26aulast%3DKruger%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520PRMT5%2520with%2520in%2520vivo%2520and%2520in%2520vitro%2520potency%2520in%2520MCL%2520models%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D432%26epage%3D437%26doi%3D10.1038%2Fnchembio.1810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Potent, selective, and cell active protein arginine methyltransferase 5 (PRMT5) inhibitor developed by structure-based virtual screening and hit optimization</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">6289</span>– <span class="NLM_lpage">6304</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00587</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00587" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKitr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=6289-6304&author=R.+Maoauthor=J.+Shaoauthor=K.+Zhuauthor=Y.+Zhangauthor=H.+Dingauthor=C.+Zhangauthor=Z.+Shiauthor=H.+Jiangauthor=D.+Sunauthor=W.+Duanauthor=C.+Luo&title=Potent%2C+selective%2C+and+cell+active+protein+arginine+methyltransferase+5+%28PRMT5%29+inhibitor+developed+by+structure-based+virtual+screening+and+hit+optimization&doi=10.1021%2Facs.jmedchem.7b00587"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Potent, Selective, and Cell Active Protein Arginine Methyltransferase 5 (PRMT5) Inhibitor Developed by Structure-Based Virtual Screening and Hit Optimization</span></div><div class="casAuthors">Mao, Ruifeng; Shao, Jingwei; Zhu, Kongkai; Zhang, Yuanyuan; Ding, Hong; Zhang, Chenhua; Shi, Zhe; Jiang, Hualiang; Sun, Dequn; Duan, Wenhu; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6289-6304</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Abstr. PRMT5 plays important roles in diverse cellular processes and is upregulated in several human malignancies.  Besides, PRMT5 has been validated as anticancer target in mantle cell lymphoma.  In this study, the authors found a potent and selective PRMT5 inhibitor by performing structure-based virtual screening and hit optimization.  The identified compd. I (IC50 = 0.33 μM) exhibited a broad selectivity against a panel of other methyltransferases.  The direct binding of I to PRMT5 was validated by surface plasmon resonance expts., with a Kd of 0.987 μM.  Kinetic expts. indicated that I was a SAM competitive inhibitor other than substrate.  In addn., I showed selective antiproliferative effects against MV4-11 cells, and further studies indicated that the mechanism of cellular antitumor activity was due to inhibition of PRMT5 mediated SmD3 methylation. I may represent a promising lead compd. to understand more about PRMT5 and potentially assist the development of treatments for leukemia indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI1jnM_kEhRrVg90H21EOLACvtfcHk0lgUozW8mBTPzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKitr7E&md5=5e3687166e2e6b5558409806e3c4b9e5</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00587%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DR.%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DDuan%26aufirst%3DW.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DPotent%252C%2520selective%252C%2520and%2520cell%2520active%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520inhibitor%2520developed%2520by%2520structure-based%2520virtual%2520screening%2520and%2520hit%2520optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D6289%26epage%3D6304%26doi%3D10.1021%2Facs.jmedchem.7b00587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of selective inhibitors of protein arginine methyltransferase 5 by docking-based virtual screening</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">3648</span>– <span class="NLM_lpage">3661</span>, <span class="refDoi"> DOI: 10.1039/c7ob00070g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1039%2FC7OB00070G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=28397890" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltl2mtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=3648-3661&author=Y.+Yeauthor=B.+Zhangauthor=R.+Maoauthor=C.+Zhangauthor=Y.+Wangauthor=J.+Xingauthor=Y.-C.+Liuauthor=X.+Luoauthor=H.+Dingauthor=Y.+Yangauthor=B.+Zhouauthor=H.+Jiangauthor=K.+Chenauthor=C.+Luoauthor=M.+Zheng&title=Discovery+and+optimization+of+selective+inhibitors+of+protein+arginine+methyltransferase+5+by+docking-based+virtual+screening&doi=10.1039%2Fc7ob00070g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and optimization of selective inhibitors of protein arginine methyltransferase 5 by docking-based virtual screening</span></div><div class="casAuthors">Ye, Yan; Zhang, Bidong; Mao, Ruifeng; Zhang, Chenhua; Wang, Yulan; Xing, Jing; Liu, Yu-Chih; Luo, Xiaomin; Ding, Hong; Yang, Yaxi; Zhou, Bing; Jiang, Hualiang; Chen, Kaixian; Luo, Cheng; Zheng, Mingyue</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3648-3661</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is a type II PRMT enzyme crit. for diverse cellular processes and different types of cancers.  Many efforts have been made to discover novel scaffold PRMT5 inhibitors.  Herein, the authors report the discovery of DC_P33 as a hit compd. of PRMT5 inhibitor, identified by mol. docking based virtual screening and 3H-labeled radioactive methylation assays.  Structure-activity relationship (SAR) anal. was performed on the analogs of DC_P33 and then structural modifications were done to improve its activity.  Among the derivs., the compd. DC_C01 displayed an IC50 value of 2.8 μM, and good selectivity toward PRMT1, EZH2 and DNMT3A.  Moreover, DC_C01 exhibited anti-proliferation activities against Z-138, Maver-1, and Jeko-1 cancer cells with EC50 values of 12 μM, 12 μM, and 10.5 μM, resp.  Taken together, these results contribute to the development of specific inhibitors against PRMT5 and cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkWfdAd-COtbVg90H21EOLACvtfcHk0lg4xuukKicMVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltl2mtr0%253D&md5=ead3b89fa64875e2dbf47c3a99714564</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1039%2FC7OB00070G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7OB00070G%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DMao%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXing%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DY.-C.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DZheng%26aufirst%3DM.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520selective%2520inhibitors%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520by%2520docking-based%2520virtual%2520screening%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26spage%3D3648%26epage%3D3661%26doi%3D10.1039%2Fc7ob00070g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan-Penebre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuplast, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter
Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span> <span> </span><span class="NLM_article-title">Structure and property guided design in the identification of PRMT5 tool compound EPZ015666</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">162</span>– <span class="NLM_lpage">166</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00380</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00380" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGnsr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=162-166&author=K.+W.+Duncanauthor=N.+Riouxauthor=P.+A.+Boriack-Sjodinauthor=M.+J.+Munchhofauthor=L.+A.+Reiterauthor=C.+R.+Majerauthor=L.+Jinauthor=L.+D.+Johnstonauthor=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=M.+Porter%0AScottauthor=R.+M.+Pollockauthor=N.+J.+Watersauthor=J.+J.+Smithauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworth&title=Structure+and+property+guided+design+in+the+identification+of+PRMT5+tool+compound+EPZ015666&doi=10.1021%2Facsmedchemlett.5b00380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and Property Guided Design in the Identification of PRMT5 Tool Compound EPZ015666</span></div><div class="casAuthors">Duncan, Kenneth W.; Rioux, Nathalie; Boriack-Sjodin, P. Ann; Munchhof, Michael J.; Reiter, Lawrence A.; Majer, Christina R.; Jin, Lei; Johnston, L. Danielle; Chan-Penebre, Elayne; Kuplast, Kristy G.; Porter Scott, Margaret; Pollock, Roy M.; Waters, Nigel J.; Smith, Jesse J.; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">162-166</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent publication of a potent and selective inhibitor of protein methyltransferase 5 (PRMT5) provides the scientific community with in vivo-active tool compd. I (EPZ015666, GSK3235025) to probe the underlying pharmacol. of this key enzyme.  Herein, the authors report the design and optimization strategies employed on an initial hit compd. with poor in vitro clearance to yield in vivo tool compd. I and an addnl. potent in vitro tool mol. II (GSK3203591).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2V7LiQgVZjrVg90H21EOLACvtfcHk0lg4xuukKicMVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGnsr3L&md5=e7ac92c2d985da4994e9bb959adededf</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00380%26sid%3Dliteratum%253Aachs%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DReiter%26aufirst%3DL.%2BA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DPorter%2BScott%26aufirst%3DM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26atitle%3DStructure%2520and%2520property%2520guided%2520design%2520in%2520the%2520identification%2520of%2520PRMT5%2520tool%2520compound%2520EPZ015666%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D162%26epage%3D166%26doi%3D10.1021%2Facsmedchemlett.5b00380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a dual PRMT5-PRMT7 inhibitor</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">408</span>– <span class="NLM_lpage">412</span>, <span class="refDoi"> DOI: 10.1021/ml500467h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500467h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjs1Ggu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=408-412&author=D.+Smilauthor=M.+S.+Eramauthor=F.+Liauthor=S.+Kennedyauthor=M.+M.+Szewczykauthor=P.+J.+Brownauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=M.+Schapira&title=Discovery+of+a+dual+PRMT5-PRMT7+inhibitor&doi=10.1021%2Fml500467h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Dual PRMT5-PRMT7 Inhibitor</span></div><div class="casAuthors">Smil, David; Eram, Mohammad S.; Li, Fengling; Kennedy, Steven; Szewczyk, Magdalena M.; Brown, Peter J.; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Vedadi, Masoud; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">408-412</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The protein arginine methyltransferases PRMT7 and PRMT5, resp., monomethylate and sym. dimethylate arginine side-chains of proteins involved in diverse cellular mechanisms, including chromatin-mediated control of gene transcription, splicing, and the RAS to ERK transduction cascade.  It is believed that PRMT5 and PRMT7 act in conjunction to methylate their substrates, and genetic deletions support the notion that these enzymes derepress cell proliferation and migration in cancer.  Using available structures of PRMT5, we designed DS-437, a PRMT5 inhibitor with an IC50 value of 6 μM against both PRMT5 and PRMT7 that is inactive against 29 other human protein-, DNA-, and RNA-methyltransferases and inhibits sym. dimethylation of PRMT5 substrates in cells.  This compd. behaves as a cofactor competitor and represents a valid scaffold to interrogate the potential of the PRMT5-PRMT7 axis as a target for therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9J078gEcjRLVg90H21EOLACvtfcHk0lgw1UniyvOSTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjs1Ggu7k%253D&md5=8eefe93068418c61881850d3b4d39676</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fml500467h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500467h%26sid%3Dliteratum%253Aachs%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520dual%2520PRMT5-PRMT7%2520inhibitor%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D408%26epage%3D412%26doi%3D10.1021%2Fml500467h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prabhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demir, Ö.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaro, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neil, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span> <span> </span><span class="NLM_article-title">Development of an AlphaLISA high throughput technique to screen for small molecule inhibitors targeting protein arginine methyltransferases</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2509</span>– <span class="NLM_lpage">2520</span>, <span class="refDoi"> DOI: 10.1039/c7mb00391a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1039%2FC7MB00391A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=29099132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslagurvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=2509-2520&author=L.+Prabhuauthor=L.+Chenauthor=H.+Weiauthor=%C3%96.+Demirauthor=A.+Safaauthor=L.+Zengauthor=R.+E.+Amaroauthor=B.+H.+O%E2%80%99Neilauthor=Z.-Y.+Zhangauthor=T.+Lu&title=Development+of+an+AlphaLISA+high+throughput+technique+to+screen+for+small+molecule+inhibitors+targeting+protein+arginine+methyltransferases&doi=10.1039%2Fc7mb00391a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Development of an AlphaLISA high throughput technique to screen for small molecule inhibitors targeting protein arginine methyltransferases</span></div><div class="casAuthors">Prabhu, Lakshmi; Chen, Lan; Wei, Han; Demir, Ozlem; Safa, Ahmad; Zeng, Lifan; Amaro, Rommie E.; O.bxsolid.Neil, Bert H.; Zhang, Zhon-Yin; Lu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2509-2520</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The protein arginine methyltransferase (PRMT) family of enzymes comprises nine family members in mammals.  They catalyze arginine methylation, either monomethylation or sym./asym. dimethylation of histone and non-histone proteins.  PRMT methylation of its substrate proteins modulates cellular processes such as signal transduction, transcription, and mRNA splicing.  Recent studies have linked overexpression of PRMT5, a member of the PRMT superfamily, to oncogenesis, making it a potential target for cancer therapy.  In this study, we developed a highly sensitive (Z' score = 0.7) robotic high throughput screening (HTS) platform to discover small mol. inhibitors of PRMT5 by adapting the AlphaLISA technol.  Using biotinylated histone H4 as a substrate, and S-adenosyl-L-methionine as a Me donor, PRMT5 sym. dimethylated H4 at arginine (R) 3.  Highly specific acceptor beads for sym. dimethylated H4R3 and streptavidin-coated donor beads bound the substrate, emitting a signal that is proportional to the methyltransferase activity.  Using this powerful approach, we identified specific PRMT5 inhibitors P1608K04 and P1618J22, and further validated their efficacy and specificity for inhibiting PRMT5.  Importantly, these two compds. exhibited much more potent efficacy than the com. PRMT5 inhibitor EPZ015666 in both pancreatic and colorectal cancer cells.  Overall, our work highlights a novel, powerful, and sensitive approach to identify specific PRMT5 inhibitors.  The general principle of this HTS screening method can not only be applied to PRMT5 and the PRMT superfamily, but may also be extended to other epigenetic targets.  This approach allows us to identify compds. that inhibit the activity of their resp. targets, and screening hits like P1608K04 and P1618J22 may serve as the basis for novel drug development to treat cancer and/or other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhtJJg4OpI_bVg90H21EOLACvtfcHk0lgw1UniyvOSTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslagurvE&md5=6cd641ae47a2d2014da4463709746a51</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2FC7MB00391A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7MB00391A%26sid%3Dliteratum%253Aachs%26aulast%3DPrabhu%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DH.%26aulast%3DDemir%26aufirst%3D%25C3%2596.%26aulast%3DSafa%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DAmaro%26aufirst%3DR.%2BE.%26aulast%3DO%25E2%2580%2599Neil%26aufirst%3DB.%2BH.%26aulast%3DZhang%26aufirst%3DZ.-Y.%26aulast%3DLu%26aufirst%3DT.%26atitle%3DDevelopment%2520of%2520an%2520AlphaLISA%2520high%2520throughput%2520technique%2520to%2520screen%2520for%2520small%2520molecule%2520inhibitors%2520targeting%2520protein%2520arginine%2520methyltransferases%26jtitle%3DMol.%2520BioSyst.%26date%3D2017%26volume%3D13%26spage%3D2509%26epage%3D2520%26doi%3D10.1039%2Fc7mb00391a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kong, G.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Bryant, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span> <span> </span><span class="NLM_article-title">Selective small-chemical inhibitors of protein arginine methyltransferase 5 with anti-lung cancer activity</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0181601</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0181601</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1371%2Fjournal.pone.0181601" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=28806746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtV2rtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=G.-M.+Kongauthor=M.+Yuauthor=Z.+Guauthor=Z.+Chenauthor=R.-M.+Xuauthor=D.+O%E2%80%99Bryantauthor=Z.+Wang&title=Selective+small-chemical+inhibitors+of+protein+arginine+methyltransferase+5+with+anti-lung+cancer+activity&doi=10.1371%2Fjournal.pone.0181601"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Selective small-chemical inhibitors of protein arginine methyltransferase 5 with anti-lung cancer activity</span></div><div class="casAuthors">Kong, Gui-Mei; Yu, Min; Gu, Zhongping; Chen, Zhi; Xu, Rui-Ming; O'Bryant, Deon; Wang, Zhengxin</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e0181601/1-e0181601/23</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) plays crit. roles in a wide variety of biol. processes, including tumorigenesis.  By screening a library of small chem. compds., we identified eight compds. that selectively inhibit the PRMT5 enzymic activity, with IC50 values ranging from 0.1 to 6 μM.  Mol. docking simulation and site directed mutagenesis indicated that identified compds. target the substrate-binding site in PRMT5.  Treatment of lung cancer cells with identified inhibitors led to inhibition of the sym. arginine methylation of SmD3 and histones and the cellular proliferation.  Oral administration of the inhibitor demonstrated antitumor activity in a lung tumor xenograft model.  Thus, identified PRMT5-specific small-mol. inhibitors would help elucidate the biol. roles of PRMT5 and serve as lead compds. for future drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHrmbn8I9YMLVg90H21EOLACvtfcHk0lgw1UniyvOSTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtV2rtbk%253D&md5=0306ed1351aac8a04d286df255c6e432</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0181601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0181601%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DG.-M.%26aulast%3DYu%26aufirst%3DM.%26aulast%3DGu%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DR.-M.%26aulast%3DO%25E2%2580%2599Bryant%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DZ.%26atitle%3DSelective%2520small-chemical%2520inhibitors%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520with%2520anti-lung%2520cancer%2520activity%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0181601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonday, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortez, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grogan, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonysamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weichert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocchinfuso, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span> <span> </span><span class="NLM_article-title">LLY-283, a potent and selective inhibitor of arginine methyltransferase 5, PRMT5, with antitumor activity</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">612</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00014</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotFarurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=612-617&author=Z.+Q.+Bondayauthor=G.+S.+Cortezauthor=M.+J.+Groganauthor=S.+Antonysamyauthor=K.+Weichertauthor=W.+P.+Bocchinfusoauthor=F.+Liauthor=S.+Kennedyauthor=B.+Liauthor=M.+M.+Maderauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+S.+Eramauthor=M.+M.+Szewczykauthor=D.+Barsyte-Lovejoyauthor=M.+Vedadiauthor=E.+Guccioneauthor=R.+M.+Campbell&title=LLY-283%2C+a+potent+and+selective+inhibitor+of+arginine+methyltransferase+5%2C+PRMT5%2C+with+antitumor+activity&doi=10.1021%2Facsmedchemlett.8b00014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">LLY-283, a Potent and Selective Inhibitor of Arginine Methyltransferase 5, PRMT5, with Antitumor Activity</span></div><div class="casAuthors">Bonday, Zahid Q.; Cortez, Guillermo S.; Grogan, Michael J.; Antonysamy, Stephen; Weichert, Ken; Bocchinfuso, Wayne P.; Li, Fengling; Kennedy, Steven; Li, Binghui; Mader, Mary M.; Arrowsmith, Cheryl H.; Brown, Peter J.; Eram, Mohammad S.; Szewczyk, Magdalena M.; Barsyte-Lovejoy, Dalia; Vedadi, Masoud; Guccione, Ernesto; Campbell, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">612-617</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is a type II arginine methyltransferase that catalyzes the formation of sym. dimethylarginine in a no. of nuclear and cytoplasmic proteins.  Although the cellular functions of PRMT5 have not been fully unraveled, it has been implicated in a no. of cellular processes like RNA processing, signal transduction and transcriptional regulation.  PRMT5 is ubiquitously expressed in most tissues and its expression has been shown to be elevated in several cancers including breast cancer, gastric cancer, glioblastoma and lymphoma.  Here we describe the identification and characterization of a novel and selective PRMT5 inhibitor with potent in vitro and in vivo activity.  Compd. 1 (also called LLY-283) inhibited PRMT5 enzymic activity in vitro and in cells with IC50 of 22 ± 3 nM and 25 ± 1 nM, resp. while its diastereomer, compd. 2 (also called LLY-284) was much less active.  Compd. 1 also showed antitumor activity in mouse xenografts when dosed orally and can serve as an excellent probe mol. for understanding the biol. function of PRMT5 in normal and cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYIq2FC-EskrVg90H21EOLACvtfcHk0liquFPEDbZDHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotFarurc%253D&md5=93517c34cf9f99d76730a538f594ddd7</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00014%26sid%3Dliteratum%253Aachs%26aulast%3DBonday%26aufirst%3DZ.%2BQ.%26aulast%3DCortez%26aufirst%3DG.%2BS.%26aulast%3DGrogan%26aufirst%3DM.%2BJ.%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DWeichert%26aufirst%3DK.%26aulast%3DBocchinfuso%26aufirst%3DW.%2BP.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26atitle%3DLLY-283%252C%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520arginine%2520methyltransferase%25205%252C%2520PRMT5%252C%2520with%2520antitumor%2520activity%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D612%26epage%3D617%26doi%3D10.1021%2Facsmedchemlett.8b00014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Z.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, C.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span> <span> </span><span class="NLM_article-title">Discovery of new potent protein arginine methyltransferase 5 (PRMT5) inhibitors by assembly of key pharmacophores from known inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">3693</span>– <span class="NLM_lpage">3699</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.10.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1016%2Fj.bmcl.2018.10.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=30366617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVSlsLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=3693-3699&author=K.+Zhuauthor=J.-L.+Songauthor=H.-R.+Taoauthor=Z.-Q.+Chengauthor=C.-S.+Jiangauthor=H.+Zhang&title=Discovery+of+new+potent+protein+arginine+methyltransferase+5+%28PRMT5%29+inhibitors+by+assembly+of+key+pharmacophores+from+known+inhibitors&doi=10.1016%2Fj.bmcl.2018.10.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of new potent protein arginine methyltransferase 5 (PRMT5) inhibitors by assembly of key pharmacophores from known inhibitors</span></div><div class="casAuthors">Zhu, Kongkai; Song, Jia-Li; Tao, Hong-Rui; Cheng, Zhi-Qiang; Jiang, Cheng-Shi; Zhang, Hua</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">23-24</span>),
    <span class="NLM_cas:pages">3693-3699</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is an epigenetics related enzyme that has been validated as a promising therapeutic target for human cancer.  Up to now, two small mol. PRMT5 inhibitors has been put into phase I clin. trial.  In the present study, a series of candidate mols. were designed by combining key pharmacophores of formerly reported PRMT5 inhibitors.  The in vitro PRMT5 inhibitory testing of compd. 4b14 revealed an IC50 of 2.71 μM, exhibiting high selectivity over PRMT1 and PRMT4 (>70-fold selective).  As expected, 4b14 exhibited potent anti-proliferative activity against a panel of leukemia and lymphoma cells, including MV4-11, Pfeiffer, SU-DHL-4 and KARPAS-422.  Besides, 4b14 showed significant cell cycle arrest and apoptosis-inducing effects, as well as reduced the cellular sym. arginine dimethylation level of SmD3 protein.  Finally, affinity profiling anal. indicated that hydrophobic interactions, π-π stacking and cation-π actions made the major contributions to the overall binding affinity.  This scaffold provides a new chem. template for further development of better lead compds. targeting PRMT5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUiq5gq-S5uLVg90H21EOLACvtfcHk0liquFPEDbZDHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVSlsLrP&md5=d397612b9684719d6696dff6362a8806</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.10.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.10.026%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DSong%26aufirst%3DJ.-L.%26aulast%3DTao%26aufirst%3DH.-R.%26aulast%3DCheng%26aufirst%3DZ.-Q.%26aulast%3DJiang%26aufirst%3DC.-S.%26aulast%3DZhang%26aufirst%3DH.%26atitle%3DDiscovery%2520of%2520new%2520potent%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520inhibitors%2520by%2520assembly%2520of%2520key%2520pharmacophores%2520from%2520known%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26spage%3D3693%26epage%3D3699%26doi%3D10.1016%2Fj.bmcl.2018.10.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, C.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span> <span> </span><span class="NLM_article-title">Identification of 5-benzylidene-2-phenylthiazolones as potent PRMT5 inhibitors by virtual screening, structural optimization and biological evaluations</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">298</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2018.08.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1016%2Fj.bioorg.2018.08.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=30172110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1WmsrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2018&pages=289-298&author=K.+Zhuauthor=H.+Taoauthor=J.-L.+Songauthor=L.+Jinauthor=Y.+Zhangauthor=J.+Liuauthor=Z.+Chenauthor=C.-S.+Jiangauthor=C.+Luoauthor=H.+Zhang&title=Identification+of+5-benzylidene-2-phenylthiazolones+as+potent+PRMT5+inhibitors+by+virtual+screening%2C+structural+optimization+and+biological+evaluations&doi=10.1016%2Fj.bioorg.2018.08.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of 5-benzylidene-2-phenylthiazolones as potent PRMT5 inhibitors by virtual screening, structural optimization and biological evaluations</span></div><div class="casAuthors">Zhu, Kongkai; Tao, Hongrui; Song, Jia-Li; Jin, Lu; Zhang, Yuanyuan; Liu, Jingqiu; Chen, Zhifeng; Jiang, Cheng-Shi; Luo, Cheng; Zhang, Hua</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">289-298</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is an epigenetics related enzyme that has been validated as an important therapeutic target for glioblastoma and mantel cell lymphoma.  In the present study, 11 novel PRMT5 inhibitors with 5-benzylidene-2-phenylthiazolone scaffold were identified by mol. docking-based virtual screening and structural optimization.  Their IC50 values against PRMT5 at enzymic level were ranging from 0.77 to 23 μM.  As expected, the top two active hits (5 and 19) showed potent anti-proliferative activity against MV4-11 cells with EC50 values lower than 10 μM and reduced the cellular sym. arginine dimethylation levels of SmD3 protein.  Besides, 5 and 19 demonstrated the mechanism of cell killing in cell cycle arrest and apoptotic effect.  The probable binding modes of the two compds. were explored and further verified by mol. dynamics simulation.  The structure-activity relationship (SAR) of this class of structures was also discussed and further demonstrated by mol. docking simulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1K2TslS6a5bVg90H21EOLACvtfcHk0liquFPEDbZDHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1WmsrrK&md5=734c196a4312bac7c377ad1ca1ec4d81</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2018.08.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2018.08.021%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DTao%26aufirst%3DH.%26aulast%3DSong%26aufirst%3DJ.-L.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DC.-S.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DH.%26atitle%3DIdentification%2520of%25205-benzylidene-2-phenylthiazolones%2520as%2520potent%2520PRMT5%2520inhibitors%2520by%2520virtual%2520screening%252C%2520structural%2520optimization%2520and%2520biological%2520evaluations%26jtitle%3DBioorg.%2520Chem.%26date%3D2018%26volume%3D81%26spage%3D289%26epage%3D298%26doi%3D10.1016%2Fj.bioorg.2018.08.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leal, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shetty, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaddi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luengo, J. I.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent and selective covalent protein arginine methyltransferase 5 (PRMT5) inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1033</span>– <span class="NLM_lpage">1038</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00074</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00074" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A280%3ADC%252BB3MzotVeltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1033-1038&author=H.+Linauthor=M.+Wangauthor=Y.+W.+Zhangauthor=S.+Tongauthor=R.+A.+Lealauthor=R.+Shettyauthor=K.+Vaddiauthor=J.+I.+Luengo&title=Discovery+of+potent+and+selective+covalent+protein+arginine+methyltransferase+5+%28PRMT5%29+inhibitors&doi=10.1021%2Facsmedchemlett.9b00074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent and Selective Covalent Protein Arginine Methyltransferase 5 (PRMT5) Inhibitors</span></div><div class="casAuthors">Lin Hong; Wang Min; Zhang Yang W; Leal Raul A; Shetty Rupa; Vaddi Kris; Luengo Juan I; Tong Shuilong</div><div class="citationInfo"><span class="NLM_cas:title">ACS medicinal chemistry letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1033-1038</span>
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is known to symmetrically dimethylate numerous cytosolic and nuclear proteins that are involved in a variety of cellular processes.  Recent findings have revealed its potential as a cancer therapeutic target.  PRMT5 possesses a cysteine (C449) in the active site, unique to PRMT5.  Therefore, covalent PRMT5 inhibition is an attractive chemical approach.  Herein, we report an exciting discovery of a series of novel hemiaminals that under physiological conditions can be converted to aldehydes and react with C449 to form covalent adducts, which presumably undergo an unprecedented elimination to form the thiol-vinyl ethers, as indicated by electron density in the co-crystal structure of the PRMT5/MEP50 complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTdNDVSKB6iezW1zwg2yUK5fW6udTcc2eYD7Mh4VIshtbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MzotVeltQ%253D%253D&md5=4b5b08dc3e3615144ece564f70bf3bd0</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00074%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DY.%2BW.%26aulast%3DTong%26aufirst%3DS.%26aulast%3DLeal%26aufirst%3DR.%2BA.%26aulast%3DShetty%26aufirst%3DR.%26aulast%3DVaddi%26aufirst%3DK.%26aulast%3DLuengo%26aufirst%3DJ.%2BI.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520selective%2520covalent%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D1033%26epage%3D1038%26doi%3D10.1021%2Facsmedchemlett.9b00074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTACs: great opportunities for academia and industry</span>. <i>Signal Transduction Targeted Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">64</span>, <span class="refDoi"> DOI: 10.1038/s41392-019-0101-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fs41392-019-0101-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=31885879" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&pages=64&author=X.+Sunauthor=H.+Gaoauthor=Y.+Yangauthor=M.+Heauthor=Y.+Wuauthor=Y.+Songauthor=Y.+Tongauthor=Y.+Rao&title=PROTACs%3A+great+opportunities+for+academia+and+industry&doi=10.1038%2Fs41392-019-0101-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">PROTACs: great opportunities for academia and industry</span></div><div class="casAuthors">Sun Xiuyun; Gao Hongying; Yang Yiqing; He Ming; Wu Yue; Song Yugang; Tong Yan; Rao Yu; Sun Xiuyun; Gao Hongying; Yang Yiqing; Rao Yu</div><div class="citationInfo"><span class="NLM_cas:title">Signal transduction and targeted therapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">64</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although many kinds of therapies are applied in the clinic, drug-resistance is a major and unavoidable problem.  Another disturbing statistic is the limited number of drug targets, which are presently only 20-25% of all protein targets that are currently being studied.  Moreover, the focus of current explorations of targets are their enzymatic functions, which ignores the functions from their scaffold moiety.  As a promising and appealing technology, PROteolysis TArgeting Chimeras (PROTACs) have attracted great attention both from academia and industry for finding available approaches to solve the above problems.  PROTACs regulate protein function by degrading target proteins instead of inhibiting them, providing more sensitivity to drug-resistant targets and a greater chance to affect the nonenzymatic functions.  PROTACs have been proven to show better selectivity compared to classic inhibitors.  PROTACs can be described as a chemical knockdown approach with rapidity and reversibility, which presents new and different biology compared to other gene editing tools by avoiding misinterpretations that arise from potential genetic compensation and/or spontaneous mutations.  PRTOACs have been widely explored throughout the world and have outperformed not only in cancer diseases, but also in immune disorders, viral infections and neurodegenerative diseases.  Although PROTACs present a very promising and powerful approach for crossing the hurdles of present drug discovery and tool development in biology, more efforts are needed to gain to get deeper insight into the efficacy and safety of PROTACs in the clinic.  More target binders and more E3 ligases applicable for developing PROTACs are waiting for exploration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQretvERU2xT-_Q9URG6Dx8fW6udTcc2eYD7Mh4VIshtbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D&md5=7d9545a21e3a78cededf2977a31e8f7d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fs41392-019-0101-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-019-0101-6%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTACs%253A%2520great%2520opportunities%2520for%2520academia%2520and%2520industry%26jtitle%3DSignal%2520Transduction%2520Targeted%2520Ther.%26date%3D2019%26volume%3D4%26spage%3D64%26doi%3D10.1038%2Fs41392-019-0101-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lj3rK0iRrhIqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: expanding the toolbox</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">949</span>– <span class="NLM_lpage">963</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0047-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fs41573-019-0047-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=31666732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=949-963&author=M.+Schapiraauthor=M.+F.+Calabreseauthor=A.+N.+Bullockauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+expanding+the+toolbox&doi=10.1038%2Fs41573-019-0047-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation: expanding the toolbox</span></div><div class="casAuthors">Schapira, Matthieu; Calabrese, Matthew F.; Bullock, Alex N.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">949-963</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review. Proteolysis-targeting chimeras (PROTACs) and related mols. that induce targeted protein degrdn. by the ubiquitin-proteasome system represent a new therapeutic modality and are the focus of great interest, owing to potential advantages over traditional occupancy-based inhibitors with respect to dosing, side effects, drug resistance and modulating 'undruggable' targets.  However, the technol. is still maturing, and the design elements for successful PROTAC-based drugs are currently being elucidated.  Importantly, fewer than 10 of the more than 600 E3 ubiquitin ligases have so far been exploited for targeted protein degrdn., and expansion of knowledge in this area is a key opportunity.  Here, we briefly discuss lessons learned about targeted protein degrdn. in chem. biol. and drug discovery and systematically review the expression profile, domain architecture and chem. tractability of human E3 ligases that could expand the toolbox for PROTAC discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwUvM27MU3WrVg90H21EOLACvtfcHk0lj3rK0iRrhIqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM&md5=9bfd48c6c47fc1cce15129495fea7079</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0047-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0047-y%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520expanding%2520the%2520toolbox%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D949%26epage%3D963%26doi%3D10.1038%2Fs41573-019-0047-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span> <span> </span><span class="NLM_article-title">Protein knockdown technology: application of ubiquitin ligase to cancer therapy</span>. <i>Curr. Cancer Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">136</span>– <span class="NLM_lpage">146</span>, <span class="refDoi"> DOI: 10.2174/1568009616666151112122502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.2174%2F1568009616666151112122502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=26560118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFOqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=136-146&author=N.+Ohokaauthor=N.+Shibataauthor=T.+Hattoriauthor=M.+Naito&title=Protein+knockdown+technology%3A+application+of+ubiquitin+ligase+to+cancer+therapy&doi=10.2174%2F1568009616666151112122502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Knockdown Technology: Application of Ubiquitin Ligase to Cancer Therapy</span></div><div class="casAuthors">Ohoka, Nobumichi; Shibata, Norihito; Hattori, Takayuki; Naito, Mikihiko</div><div class="citationInfo"><span class="NLM_cas:title">Current Cancer Drug Targets</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">136-146</span>CODEN:
                <span class="NLM_cas:coden">CCDTB9</span>;
        ISSN:<span class="NLM_cas:issn">1568-0096</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Selective degrdn. of pathogenic proteins by small mols. in cells is a novel approach for development of therapeutic agents against various diseases, including cancer.  We and others have developed a protein knockdown technol. with a series of hybrid small compds., called SNIPERs (Specific and Nongenetic IAP-dependent Protein ERasers); and peptidic chimeric mols., called PROTACs (proteolysis-targeting chimeric mols.), which induce selective degrdn. of target proteins via the ubiquitin-proteasome pathway.  These compds. include two different ligands connected by a linker; one is a ligand for a ubiquitin ligase and the other is a ligand for the target protein, which are expected to crosslink these proteins in cells.  Theor., any cytosolic protein can be targeted for degrdn. by this technol.  To date, several SNIPERs and PROTACs against various oncogenic proteins have been developed, which specifically induce polyubiquitylation and proteasomal degrdn. of the oncogenic proteins, resulting in cell death, growth arrest, or impaired migration of cancer cells.  Thus, this protein knockdown technol. has a great potential for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmS_kGOWfeuLVg90H21EOLACvtfcHk0ljSUnB4K-IWnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFOqtw%253D%253D&md5=d78f1ae70ea89843d9f946454cc95058</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.2174%2F1568009616666151112122502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568009616666151112122502%26sid%3Dliteratum%253Aachs%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DProtein%2520knockdown%2520technology%253A%2520application%2520of%2520ubiquitin%2520ligase%2520to%2520cancer%2520therapy%26jtitle%3DCurr.%2520Cancer%2520Drug%2520Targets%26date%3D2016%26volume%3D16%26spage%3D136%26epage%3D146%26doi%3D10.2174%2F1568009616666151112122502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Protac-induced protein degradation in drug discovery: breaking the rules or just making new ones?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=444-452&author=I.+Churcher&title=Protac-induced+protein+degradation+in+drug+discovery%3A+breaking+the+rules+or+just+making+new+ones%3F&doi=10.1021%2Facs.jmedchem.7b01272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span></div><div class="casAuthors">Churcher, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-452</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn., using bifunctional small mols. (Protacs) to remove specific proteins from within cells, has emerged as a novel drug discovery strategy with the potential to offer therapeutic interventions not achievable with existing approaches.  In this perspective, the brief history of the field is surveyed from a drug discovery perspective with a focus on the key advances in knowledge which have led to the definition and exemplification of protein degrdn. concepts, and their resulting applications to medicine discovery.  The approach has the potential to bring disruptive change to drug discovery - the many potential advantages and outstanding challenges which lie ahead of this technol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n8I_0MhtkbVg90H21EOLACvtfcHk0ljSUnB4K-IWnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ&md5=e60f1777de8bd860c5001137d8583f1f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01272%26sid%3Dliteratum%253Aachs%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DProtac-induced%2520protein%2520degradation%2520in%2520drug%2520discovery%253A%2520breaking%2520the%2520rules%2520or%2520just%2520making%2520new%2520ones%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D444%26epage%3D452%26doi%3D10.1021%2Facs.jmedchem.7b01272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTAC technology: opportunities and challenges</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00597</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00597" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A280%3ADC%252BB383kslaitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=237-240&author=H.+Gaoauthor=X.+Sunauthor=Y.+Rao&title=PROTAC+technology%3A+opportunities+and+challenges&doi=10.1021%2Facsmedchemlett.9b00597"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC Technology: Opportunities and Challenges</span></div><div class="casAuthors">Gao Hongying; Sun Xiuyun; Rao Yu; Gao Hongying; Sun Xiuyun</div><div class="citationInfo"><span class="NLM_cas:title">ACS medicinal chemistry letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">237-240</span>
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    </div><div class="casAbstract">PROTACs-induced targeted protein degradation has emerged as a novel therapeutic strategy in drug development and attracted the favor of academic institutions, large pharmaceutical enterprises (e.g., AstraZeneca, Bayer, Novartis, Amgen, Pfizer, GlaxoSmithKline, Merck, and Boehringer Ingelheim, etc.), and biotechnology companies.  PROTACs opened a new chapter for novel drug development.  However, any new technology will face many new problems and challenges.  Perspectives on the potential opportunities and challenges of PROTACs will contribute to the research and development of new protein degradation drugs and degrader tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSj8qlWczrRKmx8TEN9K18QfW6udTcc2eaEQQj-5rWdcLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB383kslaitA%253D%253D&md5=68988d6477e83f4ebc26d11e3b8ce5b1</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00597%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTAC%2520technology%253A%2520opportunities%2520and%2520challenges%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D237%26epage%3D240%26doi%3D10.1021%2Facsmedchemlett.9b00597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zielinska, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaaban, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Bailon, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarrold, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farnie, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smalley, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, C. C.</span></span> <span> </span><span class="NLM_article-title">PRMT5 is a critical regulator of breast cancer stem cell function via histone methylation and FOXP1 expression</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3498</span>– <span class="NLM_lpage">3513</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.11.096</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1016%2Fj.celrep.2017.11.096" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=29262329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVWlsrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2017&pages=3498-3513&author=K.+Chiangauthor=A.+E.+Zielinskaauthor=A.+M.+Shaabanauthor=M.+P.+Sanchez-Bailonauthor=J.+Jarroldauthor=T.+L.+Clarkeauthor=J.+Zhangauthor=A.+Francisauthor=L.+J.+Jonesauthor=S.+Smithauthor=O.+Barbashauthor=E.+Guccioneauthor=G.+Farnieauthor=M.+J.+Smalleyauthor=C.+C.+Davies&title=PRMT5+is+a+critical+regulator+of+breast+cancer+stem+cell+function+via+histone+methylation+and+FOXP1+expression&doi=10.1016%2Fj.celrep.2017.11.096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT5 Is a Critical Regulator of Breast Cancer Stem Cell Function via Histone Methylation and FOXP1 Expression</span></div><div class="casAuthors">Chiang, Kelly; Zielinska, Agnieszka E.; Shaaban, Abeer M.; Sanchez-Bailon, Maria Pilar; Jarrold, James; Clarke, Thomas L.; Zhang, Jingxian; Francis, Adele; Jones, Louise J.; Smith, Sally; Barbash, Olena; Guccione, Ernesto; Farnie, Gillian; Smalley, Matthew J.; Davies, Clare C.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3498-3513</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Breast cancer progression, treatment resistance, and relapse are thought to originate from a small population of tumor cells, breast cancer stem cells (BCSCs).  Identification of factors crit. for BCSC function is therefore vital for the development of therapies.  Here, we identify the arginine methyltransferase PRMT5 as a key in vitro and in vivo regulator of BCSC proliferation and self-renewal and establish FOXP1, a winged helix/forkhead transcription factor, as a crit. effector of PRMT5-induced BCSC function.  Mechanistically, PRMT5 recruitment to the FOXP1 promoter facilitates H3R2me2s, SET1 recruitment, H3K4me3, and gene expression.  Our findings are clin. significant, as PRMT5 depletion within established tumor xenografts or treatment of patient-derived BCSCs with a pre-clin. PRMT5 inhibitor substantially reduces BCSC nos.  Together, our findings highlight the importance of PRMT5 in BCSC maintenance and suggest that small-mol. inhibitors of PRMT5 or downstream targets could be an effective strategy eliminating this cancer-causing population.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiZ-akPD0aZLVg90H21EOLACvtfcHk0li1jIbgZdlqEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVWlsrfK&md5=28b81dc86c2ea7c8c93d562723129945</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.11.096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.11.096%26sid%3Dliteratum%253Aachs%26aulast%3DChiang%26aufirst%3DK.%26aulast%3DZielinska%26aufirst%3DA.%2BE.%26aulast%3DShaaban%26aufirst%3DA.%2BM.%26aulast%3DSanchez-Bailon%26aufirst%3DM.%2BP.%26aulast%3DJarrold%26aufirst%3DJ.%26aulast%3DClarke%26aufirst%3DT.%2BL.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DFrancis%26aufirst%3DA.%26aulast%3DJones%26aufirst%3DL.%2BJ.%26aulast%3DSmith%26aufirst%3DS.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DFarnie%26aufirst%3DG.%26aulast%3DSmalley%26aufirst%3DM.%2BJ.%26aulast%3DDavies%26aufirst%3DC.%2BC.%26atitle%3DPRMT5%2520is%2520a%2520critical%2520regulator%2520of%2520breast%2520cancer%2520stem%2520cell%2520function%2520via%2520histone%2520methylation%2520and%2520FOXP1%2520expression%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D21%26spage%3D3498%26epage%3D3513%26doi%3D10.1016%2Fj.celrep.2017.11.096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1420</span>– <span class="NLM_lpage">1442</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01572</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01572" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1420-1442&author=J.+Huauthor=B.+Huauthor=M.+Wangauthor=F.+Xuauthor=B.+Miaoauthor=C.-Y.+Yangauthor=M.+Wangauthor=Z.+Liuauthor=D.+F.+Hayesauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ERD-308+as+a+highly+potent+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+estrogen+receptor+%28ER%29&doi=10.1021%2Facs.jmedchem.8b01572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span></div><div class="casAuthors">Hu, Jiantao; Hu, Biao; Wang, Mingliang; Xu, Fuming; Miao, Bukeyan; Yang, Chao-Yie; Wang, Mi; Liu, Zhaomin; Hayes, Daniel F.; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1420-1442</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The estrogen receptor (ER) is a validated target for the treatment of estrogen receptor-pos. (ER+) breast cancer.  Here, we describe the design, synthesis, and extensive structure-activity relationship (SAR) studies of small-mol. ERα degraders based on the proteolysis targeting chimeras (PROTAC) concept.  Our efforts have resulted in the discovery of highly potent and effective PROTAC ER degraders, as exemplified by ERD-308 I.  I achieves DC50 (concn. causing 50% of protein degrdn.) values of 0.17 and 0.43 nM in MCF-7 and T47D ER+ breast cancer cell lines, resp., and induces >95% of ER degrdn. at concns. as low as 5 nM in both cell lines.  Significantly, I induces more complete ER degrdn. than fulvestrant, the only approved selective ER degrader (SERD), and is more effective in inhibition of cell proliferation than fulvestrant in MCF-7 cells.  Further optimization of I may lead to a new therapy for advanced ER+ breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHRlcISYyFXLVg90H21EOLACvtfcHk0li1jIbgZdlqEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D&md5=678e82dd460c065e2054ba18ea8af18a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01572%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DMiao%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ERD-308%2520as%2520a%2520highly%2520potent%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520estrogen%2520receptor%2520%2528ER%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1420%26epage%3D1442%26doi%3D10.1021%2Facs.jmedchem.8b01572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+K.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0li1jIbgZdlqEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%2BK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class mitogen-activated protein kinase kinase 1/2 degrader</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">10897</span>– <span class="NLM_lpage">10911</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01528</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01528" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFOmt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=10897-10911&author=J.+Weiauthor=J.+Huauthor=L.+Wangauthor=L.+Xieauthor=M.+S.+Jinauthor=X.+Chenauthor=J.+Liuauthor=J.+Jin&title=Discovery+of+a+first-in-class+mitogen-activated+protein+kinase+kinase+1%2F2+degrader&doi=10.1021%2Facs.jmedchem.9b01528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a First-in-Class Mitogen-Activated Protein Kinase Kinase 1/2 Degrader</span></div><div class="casAuthors">Wei, Jieli; Hu, Jianping; Wang, Li; Xie, Ling; Jin, Margaret S.; Chen, Xian; Liu, Jing; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10897-10911</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MEK1 and MEK2 (also known as MAP2K1 and MAP2K2) are the "gatekeepers" of the ERK signaling output with redundant roles in controlling ERK activity.  Numerous inhibitors targeting MEK1/2 have been developed including three FDA-approved drugs.  However, acquired resistance to MEK1/2 inhibitors has been obsd. in patients and new therapeutic strategies are needed to overcome the resistance.  Here, the authors report a first-in-class degrader of MEK1/2, MS432 (I), which potently and selectively degraded MEK1 and MEK2 in a VHL E3 ligase- and proteasome-dependent manner and suppressed ERK phosphorylation in cells.  It inhibited colorectal cancer and melanoma cell proliferation much more effectively than its neg. control MS432N and its effect was phenocopied by MEK1/2 knockdown.  Compd. I was highly selective for MEK1/2 in global proteomic profiling studies.  It was also bioavailable in mice and can be used for in vivo efficacy studies.  The authors provide two well-characterized chem. tools to the biomedical community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1e5sg33oDYrVg90H21EOLACvtfcHk0lgCrAaU8-Ct5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFOmt7bK&md5=2c293b6b87990d6b814ec1dc88034f19</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01528%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DJin%26aufirst%3DM.%2BS.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520mitogen-activated%2520protein%2520kinase%2520kinase%25201%252F2%2520degrader%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D10897%26epage%3D10911%26doi%3D10.1021%2Facs.jmedchem.9b01528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandarlapaty, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">Potent and preferential degradation of CDK6 via proteolysis targeting chimera degraders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">7575</span>– <span class="NLM_lpage">7582</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00871</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00871" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVWls7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=7575-7582&author=S.+Suauthor=Z.+Yangauthor=H.+Gaoauthor=H.+Yangauthor=S.+Zhuauthor=Z.+Anauthor=J.+Wangauthor=Q.+Liauthor=S.+Chandarlapatyauthor=H.+Dengauthor=W.+Wuauthor=Y.+Rao&title=Potent+and+preferential+degradation+of+CDK6+via+proteolysis+targeting+chimera+degraders&doi=10.1021%2Facs.jmedchem.9b00871"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and Preferential Degradation of CDK6 via Proteolysis Targeting Chimera Degraders</span></div><div class="casAuthors">Su, Shang; Yang, Zimo; Gao, Hongying; Yang, Haiyan; Zhu, Songbiao; An, Zixuan; Wang, Juanjuan; Li, Qing; Chandarlapaty, Sarat; Deng, Haiteng; Wu, Wei; Rao, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7575-7582</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A focused PROTAC library hijacking cancer therapeutic target CDK6 was developed.  A design principle as "match/mismatch" was proposed for understanding the degrdn. profile differences in these PROTACs.  Notably, potent PROTACs with specific and remarkable CDK6 degrdn. potential were generated by linking CDK6 inhibitor palbociclib and E3 ligase CRBN recruiter pomalidomide.  The PROTAC strongly inhibited proliferation of hematopoietic cancer cells including multiple myeloma and robustly degraded copy-amplified/mutated forms of CDK6, indicating future potential clin. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQhCa_z8uvA7Vg90H21EOLACvtfcHk0lgCrAaU8-Ct5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVWls7zF&md5=ed100530898c2e4c1754ca48e7295134</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00871&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00871%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DAn%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DChandarlapaty%26aufirst%3DS.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPotent%2520and%2520preferential%2520degradation%2520of%2520CDK6%2520via%2520proteolysis%2520targeting%2520chimera%2520degraders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D7575%26epage%3D7582%26doi%3D10.1021%2Facs.jmedchem.9b00871" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span> <span> </span><span class="NLM_article-title">Homolog-selective degradation as a strategy to probe the function of CDK6 in AML</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">300</span>– <span class="NLM_lpage">306</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1016%2Fj.chembiol.2018.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=30595531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsVSj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=300-306&author=M.+Brandauthor=B.+Jiangauthor=S.+Bauerauthor=K.+A.+Donovanauthor=Y.+Liangauthor=E.+S.+Wangauthor=R.+P.+Nowakauthor=J.+C.+Yuanauthor=T.+Zhangauthor=N.+Kwiatkowskiauthor=A.+C.+M%C3%BCllerauthor=E.+S.+Fischerauthor=N.+S.+Grayauthor=G.+E.+Winter&title=Homolog-selective+degradation+as+a+strategy+to+probe+the+function+of+CDK6+in+AML&doi=10.1016%2Fj.chembiol.2018.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Homolog-Selective Degradation as a Strategy to Probe the Function of CDK6 in AML</span></div><div class="casAuthors">Brand, Matthias; Jiang, Baishan; Bauer, Sophie; Donovan, Katherine A.; Liang, Yanke; Wang, Eric S.; Nowak, Radoslaw P.; Yuan, Jingting C.; Zhang, Tinghu; Kwiatkowski, Nicholas; Mueller, Andre C.; Fischer, Eric S.; Gray, Nathanael S.; Winter, Georg E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">300-306.e9</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The design of selective small mols. is often stymied by similar ligand binding pockets.  Here, we report BSJ-03-123, a phthalimide-based degrader that exploits protein-interface determinants to achieve proteome-wide selectivity for the degrdn. of cyclin-dependent kinase 6 (CDK6).  Pharmacol. CDK6 degrdn. targets a selective dependency of acute myeloid leukemia cells, and transcriptomics and phosphoproteomics profiling of acute degrdn. of CDK6 enabled dynamic mapping of its immediate role in coordinating signaling and transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGognoz8FFoR9LVg90H21EOLACvtfcHk0lgCrAaU8-Ct5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsVSj&md5=f8c1c3520c6c0218763d8769c59b4342</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DE.%2BS.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DYuan%26aufirst%3DJ.%2BC.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DA.%2BC.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26atitle%3DHomolog-selective%2520degradation%2520as%2520a%2520strategy%2520to%2520probe%2520the%2520function%2520of%2520CDK6%2520in%2520AML%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D300%26epage%3D306%26doi%3D10.1016%2Fj.chembiol.2018.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) of anaplastic lymphoma kinase (ALK)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1016%2Fj.ejmech.2018.03.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=29627725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=304-314&author=C.+Zhangauthor=X.-R.+Hanauthor=X.+Yangauthor=B.+Jiangauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Proteolysis+targeting+chimeras+%28PROTACs%29+of+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1016%2Fj.ejmech.2018.03.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Zhang, Chengwei; Han, Xiao-Ran; Yang, Xiaobao; Jiang, Biao; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-314</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Anaplastic lymphoma kinase (ALK) activation has been assocd. with many types of human cancer.  Significant efforts have been devoted to the development of ALK inhibitors to antagonize the kinase activity of ALK.  Four ALK inhibitors have been approved by the FDA to date for treating patients with ALK-pos. nonsmall cell lung cancers (NSCLC).  However, drug resistance has been obsd. in the majority of patients treated with these inhibitors.  New therapeutic strategies (e.g., compds. with novel mechanisms of action) are needed to overcome the drug resistance issue.  The emerging PROTAC (Proteolysis Targeting Chimera) technol. has been successfully applied to selective degrdn. of multiple protein targets, but not ALK.  Since ALK protein levels are not important for viability in mammals, ALK PROTACs could lead to novel therapeutics with minimal toxicity.  Here the authors report the design, synthesis and biol. evaluation of novel PROTACs (degraders) of ALK.  MS4077 (I) and MS4078 (II) potently decreased cellular levels of oncogenic active ALK fusion proteins in a concn.- and time-dependent manner in SU-DHL-1 lymphoma and NCI-H2228 lung cancer cells.  The ALK protein degrdn. induced by compds. I and II was cereblon and proteasome dependent.  In addn., compds. I and II potently inhibited proliferation of SU-DHL-1 cells.  Furthermore, compd. II displayed good plasma exposure in a mouse pharmacokinetic study, thus is suitable for in vivo efficacy studies.  The authors also developed MS4748 and MS4740 , very close analogs of I and II resp., which are incapable to degrade the ALK fusion proteins, as neg. controls.  All 4 compds. are valuable chem. tools for investigating effects of ALK pharmacol. degrdn.  The study paved the way for developing the next generation of ALK PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPGM3aAZZ6h7Vg90H21EOLACvtfcHk0lijVZUo2sPb6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D&md5=804324b95034f38b46e45f89e18e55be</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.071%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DX.-R.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D304%26epage%3D314%26doi%3D10.1016%2Fj.ejmech.2018.03.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent and selective epidermal growth factor receptor (EGFR) bifunctional small-molecule degraders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">1216</span>– <span class="NLM_lpage">1232</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01566</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01566" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisFWksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=1216-1232&author=M.+Chengauthor=X.+Yuauthor=K.+Luauthor=L.+Xieauthor=L.+Wangauthor=F.+Mengauthor=X.+Hanauthor=X.+Chenauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Discovery+of+potent+and+selective+epidermal+growth+factor+receptor+%28EGFR%29+bifunctional+small-molecule+degraders&doi=10.1021%2Facs.jmedchem.9b01566"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent and Selective Epidermal Growth Factor Receptor (EGFR) Bifunctional Small-Molecule Degraders</span></div><div class="casAuthors">Cheng, Meng; Yu, Xufen; Lu, Kaylene; Xie, Ling; Wang, Li; Meng, Fanye; Han, Xiaoran; Chen, Xian; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1216-1232</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Several epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors have been developed and approved by Food and Drug Administration for the treatment of non-small-cell lung cancers, but their efficacy can be compromised by acquired drug resistance conferred by EGFR-mutant variants.  Here, we described the discovery of a novel E3 ligase von Hippel-Lindau-recruiting EGFR degrader, MS39 (compd. 6), and a first-in-class E3 ligase cereblon-recruiting EGFR degrader, MS154 (compd. 10), using the proteolysis targeting chimera technol.  These compds. potently induced the degrdn. of mutant but not wild-type EGFR in an E3 ligase-dependent manner in cancer cell lines and effectively suppressed the growth of lung cancer cells compared with the corresponding neg. controls.  The global proteomic analyses revealed that the compds. were highly selective for EGFR.  Furthermore, both compds. were bioavailable in mouse pharmacokinetic studies, and compd. 6 is the first EGFR degrader suitable for in vivo efficacy studies.  Overall, we provide a set of well-characterized chem. tools to the research community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraFLSjUGE-6rVg90H21EOLACvtfcHk0lijVZUo2sPb6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisFWksA%253D%253D&md5=ec831897a4258f1968e254aa433426c3</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01566%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DM.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DK.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMeng%26aufirst%3DF.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520selective%2520epidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520bifunctional%2520small-molecule%2520degraders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D1216%26epage%3D1232%26doi%3D10.1021%2Facs.jmedchem.9b01566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzes, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantrell, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Potent and selective chemical probe of hypoxic signalling downstream of HIF-alpha hydroxylation via VHL inhibition</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13312</span>, <span class="refDoi"> DOI: 10.1038/ncomms13312</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fncomms13312" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=27811928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVSitLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=13312&author=J.+Frostauthor=C.+Galdeanoauthor=P.+Soaresauthor=M.+S.+Gaddauthor=K.+M.+Grzesauthor=L.+Ellisauthor=O.+Epemoluauthor=S.+Shimamuraauthor=M.+Bantscheffauthor=P.+Grandiauthor=K.+D.+Readauthor=D.+A.+Cantrellauthor=S.+Rochaauthor=A.+Ciulli&title=Potent+and+selective+chemical+probe+of+hypoxic+signalling+downstream+of+HIF-alpha+hydroxylation+via+VHL+inhibition&doi=10.1038%2Fncomms13312"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective chemical probe of hypoxic signalling downstream of HIF-α hydroxylation via VHL inhibition</span></div><div class="casAuthors">Frost, Julianty; Galdeano, Carles; Soares, Pedro; Gadd, Morgan S.; Grzes, Katarzyna M.; Ellis, Lucy; Epemolu, Ola; Shimamura, Satoko; Bantscheff, Marcus; Grandi, Paola; Read, Kevin D.; Cantrell, Doreen A.; Rocha, Sonia; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13312</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. strategies to using small mols. to stimulate hypoxia inducible factors (HIFs) activity and trigger a hypoxic response under normoxic conditions, such as iron chelators and inhibitors of prolyl hydroxylase domain (PHD) enzymes, have broad-spectrum activities and off-target effects.  Here we disclose VH298, a potent VHL inhibitor that stabilizes HIF-α and elicits a hypoxic response via a different mechanism, that is the blockade of the VHL:HIF-α protein-protein interaction downstream of HIF-α hydroxylation by PHD enzymes.  We show that VH298 engages with high affinity and specificity with VHL as its only major cellular target, leading to selective on-target accumulation of hydroxylated HIF-α in a concn.- and time-dependent fashion in different cell lines, with subsequent upregulation of HIF-target genes at both mRNA and protein levels.  VH298 represents a high-quality chem. probe of the HIF signalling cascade and an attractive starting point to the development of potential new therapeutics targeting hypoxia signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLB1WvVF4EFLVg90H21EOLACvtfcHk0lgWlgi59rzrxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVSitLfJ&md5=7a884b6193b3ca5e119fecea2eb31d3e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fncomms13312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms13312%26sid%3Dliteratum%253Aachs%26aulast%3DFrost%26aufirst%3DJ.%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DGrzes%26aufirst%3DK.%2BM.%26aulast%3DEllis%26aufirst%3DL.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCantrell%26aufirst%3DD.%2BA.%26aulast%3DRocha%26aufirst%3DS.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DPotent%2520and%2520selective%2520chemical%2520probe%2520of%2520hypoxic%2520signalling%2520downstream%2520of%2520HIF-alpha%2520hydroxylation%2520via%2520VHL%2520inhibition%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D13312%26doi%3D10.1038%2Fncomms13312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, W.</span></span> <span> </span><span class="NLM_article-title">MLN4924 (Pevonedistat), a protein neddylation inhibitor, suppresses proliferation and migration of human clear cell renal cell carcinoma</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">5599</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-06098-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fs41598-017-06098-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=28717191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A280%3ADC%252BC1cjosFWksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=5599&author=S.+Tongauthor=Y.+Siauthor=H.+Yuauthor=L.+Zhangauthor=P.+Xieauthor=W.+Jiang&title=MLN4924+%28Pevonedistat%29%2C+a+protein+neddylation+inhibitor%2C+suppresses+proliferation+and+migration+of+human+clear+cell+renal+cell+carcinoma&doi=10.1038%2Fs41598-017-06098-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">MLN4924 (Pevonedistat), a protein neddylation inhibitor, suppresses proliferation and migration of human clear cell renal cell carcinoma</span></div><div class="casAuthors">Tong Shuai; Si Yang; Yu Hefen; Xie Ping; Jiang Wenguo; Tong Shuai; Si Yang; Yu Hefen; Xie Ping; Jiang Wenguo; Zhang Lingqiang; Jiang Wenguo</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5599</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Neddylation is a post-translational protein modification associated with cancer development.  MLN4924 is a neddylation inhibitor currently under investigation in multiple phase I studies on various malignancies, and its clincal name is Pevonedistat.  It has been documented that MLN4924 blocks Cullins neddylation and inactivates CRLs and, in turn, triggers cell-cycle arrest, apoptosis, senescence and autophagy in many cancer cells.  In this study, we investigated the anti-tumor effect of MLN4924 in human clear cell renal carcinoma (ccRCC).  Levels of both Nedd8 activating enzyme E1 and Nedd8-conjugating enzyme E2 were higher in ccRCC tissues and RCC cancer cells than in normal.  Moreover, MLN4924 treatment led to rapid inhibition of Cullin1 neddylation and notably suppressed growth and survival as well as migration in a dose-and time-dependent manner.  Mechanistic studies revealed that MLN4924 induced the accumulation of a number of CRL substrates, including p21, p27 and Wee1 to trigger DNA damage and induce growth arrest at the G2/M phase.  MLN4924 also induced anti-migration and anti-invasion by activating E-cadherin and repressing Vimentin.  Taken together, this study provides the first evidence that neddylation pathway is overactive in ccRCC and that MLN4924 induces dose-dependent anti-proliferation, anti-migration, anti-invasion in ccRCC cells.  The study thus indicates that MLN4924 has potential therapeutic value for the clinical treatment of renal cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQrdwMzHGpO0rrIbt-tGZmxfW6udTcc2eZLcMZl68ExP7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cjosFWksg%253D%253D&md5=1e0a80b533adac1674de53b9b68750a7</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-06098-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-06098-y%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DS.%26aulast%3DSi%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DP.%26aulast%3DJiang%26aufirst%3DW.%26atitle%3DMLN4924%2520%2528Pevonedistat%2529%252C%2520a%2520protein%2520neddylation%2520inhibitor%252C%2520suppresses%2520proliferation%2520and%2520migration%2520of%2520human%2520clear%2520cell%2520renal%2520cell%2520carcinoma%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D5599%26doi%3D10.1038%2Fs41598-017-06098-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faelth-Savitski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casillas, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Catalytic in vivo protein knockdown by small-molecule PROTACs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+D.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+protein+knockdown+by+small-molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0liJ4yl699ryeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520protein%2520knockdown%2520by%2520small-molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tovell, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shpiro, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crafter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alessi, D. R.</span></span> <span> </span><span class="NLM_article-title">Design and characterization of SGK3-PROTAC1, an isoform specific SGK3 kinase PROTAC degrader</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2024</span>– <span class="NLM_lpage">2034</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00505</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00505" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1KksLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=2024-2034&author=H.+Tovellauthor=A.+Testaauthor=H.+Zhouauthor=N.+Shpiroauthor=C.+Crafterauthor=A.+Ciulliauthor=D.+R.+Alessi&title=Design+and+characterization+of+SGK3-PROTAC1%2C+an+isoform+specific+SGK3+kinase+PROTAC+degrader&doi=10.1021%2Facschembio.9b00505"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Characterization of SGK3-PROTAC1, an Isoform Specific SGK3 Kinase PROTAC Degrader</span></div><div class="casAuthors">Tovell, Hannah; Testa, Andrea; Zhou, Houjiang; Shpiro, Natalia; Crafter, Claire; Ciulli, Alessio; Alessi, Dario R.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2024-2034</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SGK3 is a PX domain contg. protein kinase activated at endosomes downstream of Class 1 and 3 PI3K family members by growth factors and oncogenic mutations.  SGK3 plays a key role in mediating resistance of breast cancer cells to Class 1 PI3K or Akt inhibitors, by substituting for loss of Akt activity and restoring proliferative pathways such as mTORC1 signaling.  It is therefore crit. to develop tools to potently target SGK3, and obstruct its role in inhibitor resistance.  Here we describe the development of SGK3-PROTAC1 (I), a PROTAC conjugate of the 308-R SGK inhibitor with the VH032 VHL binding ligand, targeting SGK3 for degrdn.  0.3 μM I induced 50% degrdn. of endogenous SGK3 within 2 h, with maximal 80% degrdn. obsd. within 8 h, accompanied by a loss of phosphorylation of NDRG1, an SGK3 substrate.  I did not degrade closely related SGK1 and SGK2 isoforms that are nevertheless engaged and inhibited by 308-R.  Proteomic anal. revealed that SGK3 was the only cellular protein whose cellular levels were significantly reduced following treatment with I.  Low doses of I (0.1-0.3 μM) restored sensitivity of SGK3 dependent ZR-75-1 and CAMA-1 breast cancer cells to Akt (AZD5363) and PI3K (GDC0941) inhibitors, whereas the cis epimer analog incapable of binding to the VHL E3 ligase had no impact.  I suppressed proliferation of ZR-75-1 and CAMA-1 cancer cell lines treated with a PI3K inhibitor (GDC0941) more effectively than could be achieved by a conventional SGK isoform inhibitor (14H).  This work underscores the benefit of the PROTAC approach in targeting protein kinase signaling pathways with greater efficacy and selectivity than can be achieved with conventional inhibitors.  I will be an important reagent to explore the roles of the SGK3 pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCZDBO0-RuDrVg90H21EOLACvtfcHk0liJ4yl699ryeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1KksLnN&md5=83f84ea9aba235a129c4cc4919888c6f</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00505&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00505%26sid%3Dliteratum%253Aachs%26aulast%3DTovell%26aufirst%3DH.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DShpiro%26aufirst%3DN.%26aulast%3DCrafter%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26atitle%3DDesign%2520and%2520characterization%2520of%2520SGK3-PROTAC1%252C%2520an%2520isoform%2520specific%2520SGK3%2520kinase%2520PROTAC%2520degrader%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D2024%26epage%3D2034%26doi%3D10.1021%2Facschembio.9b00505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratikopoulos, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, K.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leshchenko, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rialdi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dale, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagana, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parekh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class EZH2 selective degrader</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">222</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0421-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=10.1038%2Fs41589-019-0421-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=31819273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlCht7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=214-222&author=A.+Maauthor=E.+Stratikopoulosauthor=K.-S.+Parkauthor=J.+Weiauthor=T.+C.+Martinauthor=X.+Yangauthor=M.+Schwarzauthor=V.+Leshchenkoauthor=A.+Rialdiauthor=B.+Daleauthor=A.+Laganaauthor=E.+Guccioneauthor=S.+Parekhauthor=R.+Parsonsauthor=J.+Jin&title=Discovery+of+a+first-in-class+EZH2+selective+degrader&doi=10.1038%2Fs41589-019-0421-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a first-in-class EZH2 selective degrader</span></div><div class="casAuthors">Ma, Anqi; Stratikopoulos, Elias; Park, Kwang-Su; Wei, Jieli; Martin, Tiphaine C.; Yang, Xiaobao; Schwarz, Megan; Leshchenko, Violetta; Rialdi, Alexander; Dale, Brandon; Lagana, Alessandro; Guccione, Ernesto; Parekh, Samir; Parsons, Ramon; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">214-222</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The enhancer of zeste homolog 2 (EZH2) is the main enzymic subunit of the PRC2 complex, which catalyzes trimethylation of histone H3 lysine 27 (H3K27me3) to promote transcriptional silencing.  EZH2 is overexpressed in multiple types of cancer including triple-neg. breast cancer (TNBC), and high expression levels correlate with poor prognosis.  Several EZH2 inhibitors, which inhibit the methyltransferase activity of EZH2, have shown promise in treating sarcoma and follicular lymphoma in clinics.  However, EZH2 inhibitors are ineffective at blocking proliferation of TNBC cells, even though they effectively reduce the H3K27me3 mark.  Using a hydrophobic tagging approach, we generated MS1943, a first-in-class EZH2 selective degrader that effectively reduces EZH2 levels in cells.  Importantly, MS1943 has a profound cytotoxic effect in multiple TNBC cells, while sparing normal cells, and is efficacious in vivo, suggesting that pharmacol. degrdn. of EZH2 can be advantageous for treating the cancers that are dependent on EZH2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPb6bK3v_gQrVg90H21EOLACvtfcHk0licqgXEQ6n0og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlCht7zJ&md5=e1ccae457eba9b298f8b0e583bbaa833</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0421-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0421-4%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DA.%26aulast%3DStratikopoulos%26aufirst%3DE.%26aulast%3DPark%26aufirst%3DK.-S.%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DMartin%26aufirst%3DT.%2BC.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DSchwarz%26aufirst%3DM.%26aulast%3DLeshchenko%26aufirst%3DV.%26aulast%3DRialdi%26aufirst%3DA.%26aulast%3DDale%26aufirst%3DB.%26aulast%3DLagana%26aufirst%3DA.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DParekh%26aufirst%3DS.%26aulast%3DParsons%26aufirst%3DR.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520EZH2%2520selective%2520degrader%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D16%26spage%3D214%26epage%3D222%26doi%3D10.1038%2Fs41589-019-0421-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X61" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X61','PDB','4X61'); return false;">PDB: 4X61</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i45"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01111">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_66018"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01111?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01111</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline"><sup>1</sup>H NMR, <sup>13</sup>C NMR, and HPLC spectra of compounds <b>15</b>, <b>17</b>, and <b>21</b>; raw data of compounds <b>15</b>, <b>17</b>, <b>21</b>, and EPZ015666 inhibiting the PRMT5 methyltransferase activity in the PRMT5 biochemical assay; raw data of compounds <b>15</b>, <b>17</b>, <b>21</b>, and EPZ015666 in cell viability assays (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01111/suppl_file/jm0c01111_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and biological data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01111/suppl_file/jm0c01111_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01111/suppl_file/jm0c01111_si_001.pdf">jm0c01111_si_001.pdf (538.14 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01111/suppl_file/jm0c01111_si_002.csv">jm0c01111_si_002.csv (3.35 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01111%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-17%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01111" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677aeb0869f6d1f3","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
