
<html lang="en"     class="pb-page"  data-request-id="a3f10f14-7c28-427d-9bcd-1809ade1ae26"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2016.59.issue-9;website:website:acspubs;pageGroup:string:Publication Pages;article:article:10.1021/acs.jmedchem.5b01756;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Interactive and Versatile Navigation of Structural Databases" /></meta><meta name="dc.Creator" content="Oliver  Korb" /></meta><meta name="dc.Creator" content="Bernd  Kuhn" /></meta><meta name="dc.Creator" content="Jérôme  Hert" /></meta><meta name="dc.Creator" content="Neil  Taylor" /></meta><meta name="dc.Creator" content="Jason  Cole" /></meta><meta name="dc.Creator" content="Colin  Groom" /></meta><meta name="dc.Creator" content="Martin  Stahl" /></meta><meta name="dc.Description" content="We present CSD-CrossMiner, a novel tool for pharmacophore-based searches in crystal structure databases. Intuitive pharmacophore queries describing, among others, protein–ligand interaction pattern..." /></meta><meta name="Description" content="We present CSD-CrossMiner, a novel tool for pharmacophore-based searches in crystal structure databases. Intuitive pharmacophore queries describing, among others, protein–ligand interaction pattern..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 8, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b01756" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01756" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b01756" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01756" /></link>
        
    
    

<title>Interactive and Versatile Navigation of Structural Databases | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b01756" /></meta><meta property="og:title" content="Interactive and Versatile Navigation of Structural Databases" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0014.jpeg" /></meta><meta property="og:description" content="We present CSD-CrossMiner, a novel tool for pharmacophore-based searches in crystal structure databases. Intuitive pharmacophore queries describing, among others, protein–ligand interaction patterns, ligand scaffolds, or protein environments can be built and modified interactively. Matching crystal structures are overlaid onto the query and visualized as soon as they are available, enabling the researcher to quickly modify a hypothesis on the fly. We exemplify the utility of the approach by showing applications relevant to real-world drug discovery projects, including the identification of novel fragments for a specific protein environment or scaffold hopping. The ability to concurrently search protein–ligand binding sites extracted from the Protein Data Bank (PDB) and small organic molecules from the Cambridge Structural Database (CSD) using the same pharmacophore query further emphasizes the flexibility of CSD-CrossMiner. We believe that CSD-CrossMiner closes an important gap in mining structural data and will allow users to extract more value from the growing number of available crystal structures." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b01756"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b01756">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b01756&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b01756&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b01756&amp;href=/doi/10.1021/acs.jmedchem.5b01756" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 4257-4266</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b01643" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.5b01921" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Interactive and Versatile Navigation of Structural Databases</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Oliver++Korb">Oliver Korb</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Bernd++Kuhn">Bernd Kuhn</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=J%C3%A9r%C3%B4me++Hert">Jérôme Hert</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Neil++Taylor">Neil Taylor</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jason++Cole">Jason Cole</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Colin++Groom">Colin Groom</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Martin++Stahl">Martin Stahl</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Cambridge Crystallographic Data Centre, 12 Union Road, Cambridge CB2 1EZ, U.K.</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Roche Pharma Research and Early Development, Roche Innovation Center Basel, F. Hoffmann-La Roche Ltd., Grenzacherstrasse 124, CH-4070 Basel, Switzerland</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Desert Scientific Software Pty Ltd., Level 5 Nexus Building, Norwest Business Park, 4 Columbia Court, Baulkham Hills, NSW 2153, Australia</span></div><div class="corresp-info"><strong>*</strong>O.K.: e-mail, <a href="/cdn-cgi/l/email-protection#355e5a475775565651561b5654581b54561b405e"><span class="__cf_email__" data-cfemail="aec5c1dccceecdcdcacd80cdcfc380cfcd80dbc5">[email protected]</span></a>; phone, +44 1223 763923.</div><div class="corresp-info"><strong>*</strong>B.K.: e-mail, <a href="/cdn-cgi/l/email-protection#036166716d672d68766b6d43716c606b662d606c6e"><span class="__cf_email__" data-cfemail="4d2f283f232963263825230d3f222e2528632e2220">[email protected]</span></a>; phone, +41 61 6889773.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01756&amp;href=/doi/10.1021%2Facs.jmedchem.5b01756" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 4257–4266</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 8, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>10 November 2015</li><li><span class="item_label"><b>Published</b> online</span>8 January 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 May 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b01756" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01756</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4257%26pageCount%3D10%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DOliver%2BKorb%252C%2BBernd%2BKuhn%252C%2BJ%25C3%25A9r%25C3%25B4me%2BHert%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D9%26contentID%3Dacs.jmedchem.5b01756%26title%3DInteractive%2Band%2BVersatile%2BNavigation%2Bof%2BStructural%2BDatabases%26numPages%3D10%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4266%26publicationDate%3DMay%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b01756"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2260</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">16</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b01756" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Interactive and Versatile Navigation of Structural Databases&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Oliver&quot;,&quot;last_name&quot;:&quot;Korb&quot;},{&quot;first_name&quot;:&quot;Bernd&quot;,&quot;last_name&quot;:&quot;Kuhn&quot;},{&quot;first_name&quot;:&quot;Jérôme&quot;,&quot;last_name&quot;:&quot;Hert&quot;},{&quot;first_name&quot;:&quot;Neil&quot;,&quot;last_name&quot;:&quot;Taylor&quot;},{&quot;first_name&quot;:&quot;Jason&quot;,&quot;last_name&quot;:&quot;Cole&quot;},{&quot;first_name&quot;:&quot;Colin&quot;,&quot;last_name&quot;:&quot;Groom&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;Stahl&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;08&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;4257-4266&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b01756&quot;},&quot;abstract&quot;:&quot;We present CSD-CrossMiner, a novel tool for pharmacophore-based searches in crystal structure databases. Intuitive pharmacophore queries describing, among others, protein–ligand interaction patterns, ligand scaffolds, or protein environments can be built and modified interactively. Matching crystal structures are overlaid onto the query and visualized as soon as they are available, enabling the researcher to quickly modify a hypothesis on the fly. We exemplify the utility of the approach by showing applications relevant to real-world drug discovery projects, including the identification of novel fragments for a specific protein environment or scaffold hopping. The ability to concurrently search protein–ligand binding sites extracted from the Protein Data Bank (PDB) and small organic molecules from the Cambridge Structural Database (CSD) using the same pharmacophore query further emphasizes the flexibility of CSD-CrossMiner. We believe that CSD-CrossMiner closes an important gap in mining structural data and w&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01756&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01756" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01756&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01756" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01756&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01756" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01756&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01756&amp;href=/doi/10.1021/acs.jmedchem.5b01756" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b01756" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b01756" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b01756%26sid%3Dliteratum%253Aachs%26pmid%3D26745458%26genre%3Darticle%26aulast%3DKorb%26date%3D2016%26atitle%3DInteractive%2Band%2BVersatile%2BNavigation%2Bof%2BStructural%2BDatabases%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D9%26spage%3D4257%26epage%3D4266%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290663" title="Pharmacophores">Pharmacophores</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=292201" title="Scaffolds">Scaffolds</a>,</li><li><a href="/action/doSearch?ConceptID=291763" title="Noncovalent interactions">Noncovalent interactions</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/jmcmar.2016.59.issue-9/20160512/jmcmar.2016.59.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">We present CSD-CrossMiner, a novel tool for pharmacophore-based searches in crystal structure databases. Intuitive pharmacophore queries describing, among others, protein–ligand interaction patterns, ligand scaffolds, or protein environments can be built and modified interactively. Matching crystal structures are overlaid onto the query and visualized as soon as they are available, enabling the researcher to quickly modify a hypothesis on the fly. We exemplify the utility of the approach by showing applications relevant to real-world drug discovery projects, including the identification of novel fragments for a specific protein environment or scaffold hopping. The ability to concurrently search protein–ligand binding sites extracted from the Protein Data Bank (PDB) and small organic molecules from the Cambridge Structural Database (CSD) using the same pharmacophore query further emphasizes the flexibility of CSD-CrossMiner. We believe that CSD-CrossMiner closes an important gap in mining structural data and will allow users to extract more value from the growing number of available crystal structures.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar/59/9"><issue-title>Computational Methods for Medicinal Chemistry</issue-title></a> special issue.</p></div><div class="extra-info-sec fnGroup"><div class="fnGroupItem" id="csd"><span><h4></h4></span><span><p class="last">For more information about CSD-CrossMiner or if you are interested in a free trial of this prototype application, please contact <a href="/cdn-cgi/l/email-protection#5233363f3b3c12313136317c31333f7c33317c2739"><span class="__cf_email__" data-cfemail="7716131a1e1937141413145914161a59161459021c">[email protected]</span></a>.</p></span></div></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31053" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31053" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The two major crystallographic databases of the world, the Cambridge Structural Database<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> (CSD) and the Protein Data Bank<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2, 3)</a> (PDB), have long reached a size and coverage that make them invaluable resources for structure-based design. Exploitation of this structural data has led to the generation of practically useful empirical conformation rules<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4, 5)</a> and interaction preferences.<a onclick="showRef(event, 'ref6 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8 ref9">(6-9)</a> More and more refined analyses are being carried out today, complementing first-principles approaches<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and supporting various aspects of small molecule drug discovery.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></div><div class="NLM_p">It is surprising, then, that the arsenal of methods to query the CSD and PDB is still limited in functionality: first, query engines are specific for either database, ConQuest<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> being the gold standard for CSD queries. A unified search engine does not yet exist, yet both databases often yield complementary results, whether comparing atom contact statistics, searching for interaction motifs, or generating ideas for a specific structure-based design project. Second, a broad array of tools have been developed to query the PDB,<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13, 14)</a> but these tools remain highly segregated and are based on alternate annotation<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and simplification models.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Substructure-based interaction queries<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> cannot be combined with more fuzzy queries comparing protein cavities,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> with scaffold hopping<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> or with queries based on protein motifs.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20, 21)</a> Lastly, the lack of intuitive query editors and the lack of interactive query response are impediments to iterative refinement of complex 3D searches.</div><div class="NLM_p last">In view of these gaps, we have set out to design and implement a versatile method to expand the possibilities of 3D searching. Our interest in such an expansion is 2-fold: we wish to provide a tool that enables an exploration of protein–ligand complexes beyond the classic concept of pairwise interactions and that can serve as an idea generator for structure-based projects. To this end, we chose to use the well-established technique of 3D pharmacophore searching as our basis, due to its conceptual clarity, because it is amenable to 3D query design and because it is scalable in terms of specificity and may be combined with substructure queries. An efficient database and query scheme in conjunction with careful interface design enable fast and highly interactive queries.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74358" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74358" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Structural Databases</h3><div class="NLM_p">The method has been designed to operate on any database of molecular structures as long as 3D coordinates for the atoms are available. Here we specifically analyze small molecule structures stored in the CSD and protein–ligand binding sites extracted from the PDB. For the CSD (version 5.36 including the November 2014 update) a subset has been created containing structures that (a) are not organometallic, (b) have an <i>R</i>-factor of at maximum 10%, (c) for which 3D coordinates have been determined, (d) have no disorder, and (e) are not polymeric. This resulted in a database containing around 234 400 structures. Protein binding sites without hydrogen atoms were extracted in MOL2 format from Proasis,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> a curated version of the PDB, and stored in an SQLite<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> database. Database entries were generated for all HET molecules, except metals and commonly found small ions, as of June 16, 2015. Those HET groups with less than 5 or more than 100 atoms were removed. All molecules within a cutoff radius of 6 Å around the selected HET group were included in the respective database entry. This database consists of 210 200 binding sites derived from around 58 300 PDB entries.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Overview of the Different Feature Types Used in This Work</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0013.gif" alt="" id="fx1" /></img><div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Feature Annotation</h3><div class="NLM_p last">Each entry in a structural database is annotated with a set of feature points which are used to perform the actual 3D search. The method distinguishes between <i>base</i> and <i>virtual</i> feature points, and a specific feature type may consist of a single or multiple of these feature points. Similar to other published methods,<a onclick="showRef(event, 'ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26">(24-26)</a> substructure-based features are defined by a hierarchy of SMARTS<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> patterns which can be tailored by the user. For molecular fragments from the PDB, for which we do not use explicit hydrogen atoms, we define hydrogen bond donors and acceptors based on expert knowledge about predominant protonation and tautomer states. The SMARTS list was defined to cover nitrogen heterocycles<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and functional groups occurring in a set of protein–ligand binding sites.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> It was then extended wherever an assignment was considered incorrect in our test applications. Abstract feature representations are derived from the substructures by applying a point generation rule. Point generation rules define how the feature points will be determined from a set of atomic coordinates. For example, virtual hydrogen, lone pair, or centroid positions can be assigned to represent donor, acceptor, and ring features, respectively. While in this work exclusively substructure-based features are used, the approach can deal with other point-based features such as molecular surfaces represented by surface points and corresponding projected points along their surface normal. All features assigned to a database entry are preindexed and stored in a feature database which is stored in memory at run-time. <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> illustrates the features used in the application examples (additional features and the full list of SMARTS patterns can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b01756/suppl_file/jm5b01756_si_001.pdf" class="ext-link">Supporting Information</a>).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Fingerprint Generation</h3><div class="NLM_p">In order to quickly decide whether a database entry has to be considered for the 3D search step, a <i>feature presence</i> fingerprint is precalculated. For each database entry, this fingerprint stores the presence or absence of feature types in a small molecule, a protein component, or any of the two.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (a) Base features are shown as wireframe spheres, while virtual points defining directionality are shown as solid spheres. A nondirectional acceptor is shown in red, a directional donor in blue, and an exit vector in white. (b) A pharmacophore query defining two directional planar ring features. In the top pharmacophore no constraints are applied, so in a hit the ring systems can be in the same or in different molecules. Intermolecular and intramolecular constraints (dashed lines) are applied in the middle and bottom pharmacophore, respectively, so only hits where the rings are in different molecules (middle) or in the same molecule are retrieved (bottom). (c) Examples of assigning a small molecule (S) or protein (P) constraint to a pharmacophore defining two directional planar ring features. The small molecule-only example translates to, for example, ligand-cofactor searches in the PDB or small molecule interaction searches in the CSD. The protein–ligand query allows for protein–ligand interaction searches in the PDB, and with a protein–protein query, interactions within protein components can be analyzed.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Pharmacophore Query</h3><div class="NLM_p last">The locations of features to be searched for are specified by tolerance spheres (see <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>a). Since feature types may be represented by multiple feature points, e.g., for a directional feature such as a hydrogen bond donor, the same number of tolerance spheres is used to specify the query. The radius of each tolerance sphere reflects the uncertainty in the exact location of the feature and can thus be used to control the specificity of a query. Different types of constraints can be applied to the query (see <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>b). For pairs of query features, an intra- or intermolecular constraint can be defined, which requires the respective feature points in a matching structure to be located in the same or in different molecular structures. If no such constraint is defined, hits of both kinds can be found. For databases containing protein and small molecule annotations, each query feature can additionally be constrained to be part of a protein, a small molecule component, or any of the two. Combinations of the <i>intra-</i>/<i>inter</i>molecular constraints with the protein/small molecule annotations allow for highly tailored interrogation of nonbonded interactions (see <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>c). Excluded volumes are not explicitly supported but can be emulated. For example, an excluded volume for a ligand in a protein–ligand binding site can be defined by a protein “heavy_atom” feature at the desired excluded position.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Searching</h3><div class="NLM_p last">The database search starts with the conversion of the pharmacophore query defined in Cartesian space by tolerance spheres into a distance space representation, thereby eliminating translational and rotational degrees of freedom. For each pair of tolerance spheres (neglecting pairs that form part of the same feature), a distance constraint is derived by measuring the distance between the two sphere centers and adding, as well as subtracting, the sum of the sphere radii to arrive at upper and lower bounds, respectively (if the spheres overlap, the lower bound will be set to zero). Additionally, each distance constraint stores whether an intra- or intermolecular constraint has been defined for the respective pair of features and whether a feature is constrained to be part of a protein or small molecule component. Given this information, the fingerprint described above can also be generated for the pharmacophore query. For each feature type present in the query, the respective bits of the <i>feature presence</i> fingerprint are set taking the respective molecule components into account. The query fingerprint can then be compared to the respective fingerprint of any database entry. Each bit set in the query fingerprint is required to be present in the database entry fingerprint. Only if the fingerprint comparison passes, a database entry is subjected to a 3D search using the precalculated feature points. A constraint-aware backtracking algorithm similar to the one described by Chisholm et al.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> is used to search for matching structures in distance space. Given a specific database entry and the query, a match protocol is generated by sorting the pharmacophore features in the query with respect to (a) a decreasing number of constraints, (b) an increasing number of feature points available in the database entry, and (c) increasing smallest widths of any of the distance constraint ranges a pharmacophore feature is involved in (for a run-time comparison see the <a href="/doi/suppl/10.1021/acs.jmedchem.5b01756/suppl_file/jm5b01756_si_001.pdf" class="ext-link">Supporting Information</a>). A <i>depth-first-search</i> (DFS) is then started from the feature involved in the highest number of constraints. Every time the current match is expanded to include a feature point of the next feature in the match protocol, all distance constraints (including the check for <i>intra</i>-/<i>intermolecular</i>) to all feature points already part of the current match are evaluated and only if all constraints pass, the match is expanded. For small molecule crystal structures the search engine additionally takes symmetry-related copies of the feature points into account and by default considers 26 unit cells around the central unit cell (unless the query is <i>intramolecular</i> only). Full matches are recorded if (a) they are not symmetry-related to previous matches and (b) they pass a Cartesian space overlay check. The latter check is necessary, since not all matches in distance space correspond to matches of the original query in Cartesian space. For example, the mirror images of a tetrahedral four-point pharmacophore cannot be distinguished in distance space. Thus, a gradient-based overlay technique has been developed which checks if a match in distance space can be overlaid onto the pharmacophore in Cartesian space (see <a href="/doi/suppl/10.1021/acs.jmedchem.5b01756/suppl_file/jm5b01756_si_001.pdf" class="ext-link">Supporting Information</a>). If this overlay is possible, i.e., a rigid-body transformation can be found that overlays a given set of points onto the pharmacophore spheres such that all points from the match lie in their corresponding pharmacophore spheres, the match is recorded.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Match Processing</h3><div class="NLM_p last">Since for a single database entry multiple matches may have been identified, for each match the Kabsch algorithm<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> is used to calculate the minimum overlay root-mean-square deviation (rmsd) of the point coordinates in the match, with respect to the pharmacophore sphere centers. This is used to obtain a unique ranking of the matches. Although the search method will have checked that all matched points could lie inside their corresponding tolerance spheres, they may not necessarily do so after the application of the Kabsch algorithm. Nevertheless, the ranking obtained by this method follows the user’s intuition in most cases. Additionally, the rmsd-based ranking allows the user to, for example, only keep the top <i>n</i> matches per structure. The molecular structures corresponding to a feature database entry are loaded from the structural database (residing on disk), and for each pharmacophore match, the matched structures are overlaid onto the pharmacophore using the transformation obtained by the Kabsch method. By default, a bounding sphere is calculated for the pharmacophore and only the parts of the molecular structures inside the bounding sphere are kept. Ring systems (plus attached terminal substituents) are always kept as intact units if at least one ring atom is inside the bounding sphere. For bonds which have one atom located inside and the other one located outside the bounding sphere, the latter atom will be replaced by an R-group symbol.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Diagrams</h3><div class="NLM_p">A force-field based diagram generation method is used to create a 2D representation of each pharmacophore match. First, all base features of the pharmacophore query are projected into the 2D plane. The generated 2D coordinates are then used as a template in the diagram generation process to place the atoms of a 3D pharmacophore match in the same region in the 2D layout. Since the same template is used for all matches, the resulting diagrams are aligned in the same way. An example is presented in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Example of a hydrogen bonding pharmacophore query (directional donors and acceptors are shown in blue and red, respectively) overlaid with CSD entry DOPLIF. Light green atoms represent nonterminal R-groups which are not inside the bounding-sphere fitted around the pharmacophore. (b) 2D representation of the pharmacophore and the small molecule hit shown in (a).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Interactive User Interface</h3><div class="NLM_p">The design of an interactive user interface was a key component of making the above-described functionality available in an intuitive and easily accessible fashion. In this context, the meaning of <i>interactive</i> is not to be mistaken for <i>real time</i>. While in the interactive interface, the user can interact with the search query at any time; complete search results will not necessarily be available instantaneously. Once a feature database has been created for a given structural database, this features database is stored in memory. The user then starts building a search query from scratch or based on a reference structure, which is automatically annotated with the feature types stored in the database. Tolerance sphere positions and radii can be modified using the mouse, and as soon as the interactive editing mode is activated, any change in the topology or the geometry of the pharmacophore query triggers a new database search. Any pharmacophore match found will be immediately displayed in the 3D view, giving the user instant feedback on the kind of hits to be expected from the current query setup. In the background, all hits of a search (up to a user-defined rmsd limit) are collected and up to a user-defined number of structures (typically around 100–500) are displayed in the 3D view as well as in a results browser showing the 2D diagrams along with the structure identifier and overlay rmsd. While the hits are generally sorted according to the overlay rmsd, they can also be clustered on-the-fly with respect to a molecular fingerprint<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> at a user-specified Tanimoto threshold. In order to speed up the database search for large databases, which may contain several hundred thousand structures, the user can dedicate several CPU cores to the search if available. An overview of the overall approach is presented in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Overview of the overall query workflow. Starting from a pharmacophore query (top left), the feature database residing in memory is filtered to only return the database entries that contain the feature types used in the pharmacophore. These database entries are then split up across the number of CPU cores assigned to perform the search. For each database entry, first the 3D search is performed to identify matching structures only using the feature points stored in memory. For each match, the corresponding molecular structures are then fetched from disk. The resulting hits are overlaid onto the pharmacophore query as soon as they are available. On the basis of the types of hits retrieved and presented in the 3D and 2D displays, a user may decide to modify the query by changing the geometry of the pharmacophore (sphere positions and radii) or by adding/deleting features. Every time the query is modified, a new search is started initiating the next iteration of the interactive feedback cycle.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01805" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01805" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">CSD-CrossMiner<a class="ref internalNav" href="#csd" aria-label="a">a</a> was designed to query structural databases in an efficient, intuitive, and highly flexible manner. Due to the generality of the approach, we foresee applications in small molecule drug discovery, in protein design, and in materials science. Here we focus on examples of relevance to medicinal chemistry. Several of these involve the identification of molecular fragments binding to closely related protein environments. A second set of examples illustrates the use of CSD-CrossMiner as a scaffold hopping tool, either with intramolecular constraints only or in combination with a protein environment. All case studies involve targets of previous drug discovery projects. The detailed pharmacophore queries used in this section can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b01756/suppl_file/jm5b01756_si_001.pdf" class="ext-link">Supporting Information</a>.</div><div class="NLM_p">The following examples show results from both PDB and CSD searches. To be able to query both databases with the same program, our algorithm has to correctly assign feature types for CSD structures, which typically contain explicit hydrogen atoms, as well as for PDB structures, which are stored without this information in our database. We provide an extended set of SMARTS-based feature definitions for both worlds depending on whether hydrogen atoms are present or not in a molecule. For structures without hydrogens feature type assignment is less obvious, and expert knowledge about preferred protonation and tautomer states is used to classify substructures as hydrogen bond donors and acceptors. It is also noteworthy that our definitions cover nonclassical interaction partners, such as polarized C–H groups acting as hydrogen bond donors. The full list of feature type definitions can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b01756/suppl_file/jm5b01756_si_001.pdf" class="ext-link">Supporting Information</a>.</div><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Fragments Binding to Related Protein Environments</h3><div class="NLM_p">A standard task in structure-based drug design is “pocket filling”, i.e., the identification of ligand moieties that are complementary in shape and interaction types to a given protein environment. Common questions are “What does ligand group X interact with?” and “Which other ligand group X could be suitable in this protein environment?” Often we do not want to limit ourselves to a certain interaction type for group X. There are several features that make CSD-CrossMiner particularly suitable to address such questions. First, molecules are annotated as being protein or small molecule enabling protein–ligand interaction searches. Second, a special feature type “heavy_atom” exists which matches all non-hydrogen atoms without further restriction. This allows for queries in which one interaction partner is kept very general. Third, the query can be set up directly in 3D space by means of imported reference structures. This is a big advantage over 2D query builders implemented in other programs, which require complex geometric definitions to represent 3D relationships within a binding pocket. In the following, we will illustrate the power of CSD-CrossMiner with three different queries in which one interaction partner is defined while the other is completely open.</div><div class="NLM_p">Inhibitors of the cysteine protease cathepsin L typically form a covalent bond with the active site Cys 25 and gain additional binding affinity by occupying the S2 and S3 pockets.<a onclick="showRef(event, 'ref32 ref33 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref32 ref33 ref34 ref35">(32-35)</a> The S3 pocket is characterized by a planar amide backbone fragment involving Gly 67–68 at its base, while the carbonyl group of Gly 61 exposes a hydrogen bond acceptor into the binding site (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>). This pocket has been used previously as a model system to study stacking effects on amide groups<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> as well as halogen bonding<a onclick="showRef(event, 'ref33 ref37'); return false;" href="javascript:void(0);" class="ref ref33 ref37">(33, 37)</a> in detail. With CSD-CrossMiner we can look for ligand fragments that are known to bind to similar protein environments. <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a> illustrates how we define a CSD-CrossMiner query based on the key residues Gly 61, Gly 67–68, and Leu 69, which define the shape of the S3 cavity. We did not include the side chain of Glu 63 due to its high flexibility as observed in crystal structure overlays. To ensure that solutions are enriched with ligand fragments interacting with all specified protein residues, we included two ligand “heavy_atom” features into our intermolecular query.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Left: S3 pocket of human cathepsin L where protein residues used to define the CSD-CrossMiner pharmacophore are highlighted as green sticks (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2xu1">2xu1</a>). The pyrrolidine inhibitor scaffold is shown in cyan, without the chlorophenyl fragment of the S3 pocket. Right: CSD-CrossMiner pharmacophore model with directional hydrogen bond donor (blue) and acceptor (red) as well as hydrophobic (green) features for the protein. To avoid restricting the choice of ligand motifs, the “heavy_atom” feature (lig_heavy, orange) is used for the small molecule component. Intramolecular constraints are set between the prot_acc and prot_don features derived from the amide and between the two lig_heavy features.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As similar S3 pockets are present in related cysteine proteases such as cathepsin K or S, complexes with these proteins turn up at the top of the hit list. Clustering the solution space in real time helps to remove redundant binding site information and allows the user to quickly grasp the diversity in ligand topology and protein–ligand interactions. <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a> shows three solutions to our query that appear in the top 25 entries of the hit list after clustering with a Tanimoto threshold of 0.7 and removing all cathepsin L, K, and S structures. Interestingly, our pharmacophore motif defined for the S3 pocket in cathepsin L also matches the S1 pocket of serine proteases, e.g., factor Xa structure 2ei7, a relationship we were not aware of before. Here, the indole heterocycle stacks on one of the backbone amides which form the wall of the S1 pocket, while the indole NH donor interacts with a backbone carbonyl oxygen acceptor. Other matches from non-cathepsin binding pockets are a chlorophenyl fragment in HIV-RT (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3dm2">3dm2</a>) and a triazole heterocycle in cruzain (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4klb">4klb</a>). The chlorophenyl is a motif that is already known to interact very strongly when attached to the pyrrolidine scaffold displayed in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Compared to a methyl group it provides a ∼500-fold affinity gain. This was rationalized by favorable stacking of the phenyl group to the amide backbone and a close to ideal halogen bond interaction of the chlorine atom with the carbonyl oxygen atom of Gly 61. In HIV-RT, too, a halogen bond with the carbonyl backbone of Val 179 can be identified while the phenyl interacts with the amide of the Asn 103 side chain in a more perpendicular fashion.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Examples of PDB solutions for the pharmacophore query defined in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>. All three binding sites appear in the top 25 of the hit list after clustering with a Tanimoto threshold of 0.7 and removal of all cathepsin L, K, and S structures. The ligand is shown in magenta, while protein residues are colored in gray.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Another site for molecular recognition is the S4 pocket of the serine protease factor Xa. It is composed of the side chains of Tyr 99, Phe 174, and Trp 215, providing an aromatic cage with opportunities for both edge-to-face and stacking interactions. Strong gains in binding affinity can be achieved by interacting with this pocket, and the vast majority of potent factor Xa inhibitors extend into this region (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>).<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Using the S4 pocket as reference for a CSD-CrossMiner query, we can see that such aromatic cages are not unique to factor Xa but occur in a larger number of functionally unrelated proteins as recognition motif for small molecule fragments. Among them is choline-binding protein L which binds a tetraalkylammonium moiety in a box of two Trp and one Tyr residues. In independent studies of cation−π interactions, the Diederich group identified quarternary onium ions as very efficient binders to the S4 pocket of factor Xa.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Other examples with aromatic cages are the first tudor-like domain of spindlin 1 and a back-pocket in the DFG-out binding site of the protein kinase lok (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Top: CSD-CrossMiner pharmacophore query with three “planar_ring_projected” features (green) in the protein and one small molecule “heavy_atom” feature (orange) as derived from the factor Xa complex structure 2w26. Bottom: Selected PDB solutions from the top 25 hit list after clustering with a Tanimoto threshold of 0.7 and removal of all factor Xa structures: 4cnl (choline-binding protein L from streptococcus pneumonia), 4h75 (human spindlin 1), 4aot (human lok kinase). Ligand and protein are colored in magenta and gray, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">β sheets have pleated patterns of laterally connected β strands with shallow pockets about 7 Å wide. These pockets can form part of ligand binding sites. In p38 MAP kinase, the lipophilic back pocket separated from the hinge region by the gatekeeper residue is partially formed by two β strands. Which ligand motifs interact with this environment? <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a> shows the CSD-CrossMiner query for this question using the donor/acceptor pattern of the protein backbone and a single “heavy_atom” ligand probe. Hits show a propensity for halogen binding, small alkyl groups, hydroxyl functionalities, and (hetero)aromatic rings. In the latter case, the aromatic plane is oriented almost symmetrically between two adjacent faces of the β sheet with angles of 60–70°. The ligand substructures that are displayed in <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a> occur in at least 100 matches of our query. While this is not a solid statistical analysis, it still indicates a trend for favorable interaction with β sheet pockets. This is often enough for idea generation in a medicinal chemistry program. The binding pockets that match our query are mostly from the kinase family (e.g., p38, abl, braf, egfr), but also other targets (e.g., HIV-RT) have cavities in which ligand atoms interact with β sheets.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Top: CSD-CrossMiner pharmacophore query with four “donor_projected” (blue) and two “acceptor_projected” (red) features representing a protein β sheet as well as a small molecule “heavy_atom” feature (orange). Intramolecular constraints were set for all features within each of the two β strands. The p38 MAP kinase structure 3roc is used as reference. Bottom: Substructures found in more than 100 matches of the pharmacophore query together with PDB codes of representative complexes. The orange circle highlights the atom occupying the ligand “heavy_atom” feature. “A” stands for carbon or nitrogen atoms.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Scaffold Hopping</h3><div class="NLM_p">Scaffold hopping is defined as replacing an active central element of a molecule with a novel one, with the aim of identifying patentable structures or improving molecular properties.<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40, 41)</a> An integral part of 3D scaffold hopping algorithms is the matching of substructures to exit vectors that connect the core with the substituents of the template molecule.<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42, 43)</a> In CSD-CrossMiner we have the possibility to define an exit vector pharmacophore which allows the user to combine scaffold hopping with other powerful features of our tool. The two examples below, from 11β-hydroxysteroid dehydrogenase 1 (11β-HSD1) and from kinase drug discovery, illustrate the concept.</div><div class="NLM_p">11β-HSD1 is a central enzyme in the regulation of intracellular cortisol concentrations and has been targeted with the aim to treat the metabolic syndrome. It reduces the carbonyl group of the substrate cortisone to the hydroxyl analogue cortisol by means of NADPH as a cofactor. Some of the first inhibitors of 11β-HSD1 used an amide carbonyl group to mimic the cortisone carbonyl interactions with protein residues Ser150 and Tyr163, which are important for preorganizing the substrate for catalysis (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>).<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> While the carbonyl oxygen serves as a central anchor point, the amide exit vectors appropriately position two hydrophobic residues in the steroid binding cavity of 11β-HSD1. We were wondering about other small molecule motifs that could engage in two intermolecular hydrogen bond interactions while preserving at least one of the exit vectors and constructed the pharmacophore query displayed in <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>. The same query can be used for searching the PDB and the CSD, which extends the search space significantly and which is to our knowledge not possible with any other tool. The results of the CSD-based search returned several interesting hits summarized in <a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>. Apart from the secondary amide used in the template, sulfonamides and aliphatic hydroxyl groups fulfill our query definitions. Most interestingly, a tetrazole motif is identified from the search in which the two intermolecular hydrogen bonds are formed by two adjacent nitrogen atoms with exposed lone pairs instead of a single oxygen atom. This structural insight opens up a plethora of heteroaromatic rings as scaffold hopping possibilities. Indeed, a triazole was used as central scaffold in 11β-HSD1 inhibitors from Amgen,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> and the complex structure (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>d) showed the same hydrogen bond arrangement as found in our search.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. CSD-CrossMiner pharmacophore model for 11β-HSD1 scaffold hopping with “donor_projected” (blue) and “acceptor_projected” (red) as well as “exit_vector” (white) features. The cocrystal structure with PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2irw">2irw</a> is taken as reference where protein residues Ser150 and Tyr163 are shown in gray and the inhibitor is shown in magenta. An intramolecular constraint is set between the acceptor and exit vector features derived from the ligand.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (a–c) Selected CSD solutions for the pharmacophore query defined in <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>. All three examples appear in the top 75 of the hit list after clustering with a Tanimoto threshold of 0.7. (d) Cutout of a 11β-HSD1 complex with a triazole inhibitor (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3d5q">3d5q</a>). The ligand is shown in magenta, and the protein residues are in gray.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Scaffold hopping through design of a “pseudo ring” formed by an intramolecular hydrogen bond is an elegant approach to broaden the intellectual property space for kinase hinge binding motifs. An example of this is the replacement of the pyrido[2,3-<i>d</i>]pyrimidin-7-one core displayed in <a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a>a by a single aromatic ring with an additional intramolecular hydrogen bond from a pyrimidin-4-ylurea motif yielding equipotent tyrosine kinase inhibitors.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> We aimed at reproducing this finding by running a pharmacophore search in the CSD involving planar ring, acceptor, and exit vector features (<a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a>). Results of this search are summarized in <a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a>b−d and show that solutions with an internal hydrogen bond are retrieved among top-ranking matches.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. (a) Reference ligand taken from the ABL kinase complex structure 2hzi and CSD-CrossMiner small molecule pharmacophore model. Pharmacophore features are colored as green (“planar_ring_projected”), red (hydrogen bond “acceptor_projected”), and white (“exit_vector”). Intramolecular constraints were set between all features. (b–d) Examples of solutions within the top 100 of the hit list after clustering with a Tanimoto threshold of 0.7. Different flavors of pseudo-ring scaffold hopping using an intramolecular hydrogen bond are shown in (c) and (d).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We can reuse this query and ask which kinase hinge binding scaffolds are known in the PDB that contain such a ligand pharmacophore. To this end, only two protein features for the hinge hydrogen bond donor and acceptor involved in interactions with the reference ligand of <a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a> need to be added. <a class="ref internalNav" href="#fig11" aria-label="Figure 11">Figure 11</a> contains representative examples of kinase scaffolds that match this pharmacophore, illustrating the diversity of ligand topologies that can be used.</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Results from a PDB search of the pharmacophore model of <a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a> augmented with two protein features (hydrogen bond donor and acceptor from the hinge backbone of reference structure 2hzi). Top: Core structure from 2hzi. Bottom: Other known kinase scaffolds that match the query: (a) 4ehe, braf, (b) 3tt0, fgfr1, (c) 4c3f, lck. Pharmacophore features are colored as green (“planar_ring_projected”), red (hydrogen bond “acceptor_projected”), blue (hydrogen bond “donor_accepted”), and white (“exit_vector”). Intramolecular constraints are set between all small molecule and protein features, respectively. Ligands are shown in magenta and protein residues in gray.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusions and Outlook</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36271" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36271" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We have presented an approach that enables interactive navigation of the CSD and the PDB through the same interface. Essentially, the approach combines well-known concepts (fingerprints, 3D pharmacophores, SMARTS-based substructure definitions) with each other to achieve an unprecedented degree of interactivity and usability. High query precision can be achieved by substructure-based feature definition, while the spherical 3D constraints allow for a user-definable amount of fuzziness. Search results are fed back immediately such that iterative query refinement becomes a normal way of working. Molecular fingerprint-based clustering and structural annotations, e.g., information on the drug target, serve to narrow down potentially large lists of hit structures. The optional specification of intra- and intermolecular constraints between pharmacophore features and the ability to restrict features to be part of a protein or a small molecule enable a plethora of different query types: applications may range from the retrieval of sets of overlaid structures sharing common binding patterns to scaffold hopping, cavity comparisons, and the analysis of complex multicomponent nonbonded networks. Most interaction queries are immediately applicable to the PDB and the CSD, substituting a protein interaction partner by the crystal environment. We believe that this flexibility sets the stage for broad applications for structure-based design and solid state modeling alike.</div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01756">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40018" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40018" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b01756" class="ext-link">10.1021/acs.jmedchem.5b01756</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Description of the Cartesian point onto sphere overlay method, assessment of the pharmacophore search algorithm, feature definitions, and the pharmacophore models presented in this manuscript (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01756/suppl_file/jm5b01756_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01756/suppl_file/jm5b01756_si_001.pdf">jm5b01756_si_001.pdf (837.51 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.5b01756" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73213" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73213" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Oliver Korb</span> - <span class="hlFld-Affiliation affiliation">Cambridge
Crystallographic Data Centre, 12 Union Road, Cambridge CB2 1EZ, U.K.</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#305b5f425270535354531e53515d1e51531e455b"><span class="__cf_email__" data-cfemail="600b0f120220030304034e03010d4e01034e150b">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bernd Kuhn</span> - <span class="hlFld-Affiliation affiliation">Roche
Pharma Research and Early Development, Roche Innovation Center Basel, F. Hoffmann-La Roche Ltd., Grenzacherstrasse 124, CH-4070 Basel, Switzerland</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#8defe8ffe3e9a3e6f8e5e3cdffe2eee5e8a3eee2e0"><span class="__cf_email__" data-cfemail="482a2d3a262c66233d2026083a272b202d662b2725">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jérôme Hert</span> - <span class="hlFld-Affiliation affiliation">Roche
Pharma Research and Early Development, Roche Innovation Center Basel, F. Hoffmann-La Roche Ltd., Grenzacherstrasse 124, CH-4070 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Neil Taylor</span> - <span class="hlFld-Affiliation affiliation">Desert
Scientific Software Pty Ltd., Level 5 Nexus Building, Norwest Business Park, 4 Columbia Court, Baulkham Hills, NSW 2153, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jason Cole</span> - <span class="hlFld-Affiliation affiliation">Cambridge
Crystallographic Data Centre, 12 Union Road, Cambridge CB2 1EZ, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Colin Groom</span> - <span class="hlFld-Affiliation affiliation">Cambridge
Crystallographic Data Centre, 12 Union Road, Cambridge CB2 1EZ, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin Stahl</span> - <span class="hlFld-Affiliation affiliation">Roche
Pharma Research and Early Development, Roche Innovation Center Basel, F. Hoffmann-La Roche Ltd., Grenzacherstrasse 124, CH-4070 Basel, Switzerland</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>O.K. and B.K. contributed equally. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d290e695-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04564" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04564" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank Dr. Fabian Dey, Dr. Wolfgang Guba, Dr. Christian Kramer, Dr. Michael Reutlinger, Dr. Lisha Wang, and Dr. Nicolas Zorn for testing the presented approach in drug discovery projects and for providing scientific feedback. Dr. Peter Galek is acknowledged for helpful discussions on 3D search algorithms and in-memory databases.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i19" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i19"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i20" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i20"> Abbreviations Used</h2><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">Protein Data Bank</p></td></tr><tr><td class="NLM_term">CSD</td><td class="NLM_def"><p class="first last">Cambridge Structural Database</p></td></tr><tr><td class="NLM_term">rmsd</td><td class="NLM_def"><p class="first last">root-mean-square deviation</p></td></tr><tr><td class="NLM_term">SMARTS</td><td class="NLM_def"><p class="first last">SMiles ARbitrary Target Specification</p></td></tr><tr><td class="NLM_term">HIV-RT</td><td class="NLM_def"><p class="first last">human immunodeficiency virus reverse transcriptase</p></td></tr><tr><td class="NLM_term">lok</td><td class="NLM_def"><p class="first last">lymphocyte-oriented kinase</p></td></tr><tr><td class="NLM_term">p38 MAP</td><td class="NLM_def"><p class="first last">p38 mitogen-activated protein kinase</p></td></tr><tr><td class="NLM_term">abl</td><td class="NLM_def"><p class="first last">Abelson tyrosine-protein kinase</p></td></tr><tr><td class="NLM_term">braf</td><td class="NLM_def"><p class="first last">B-rapidly accelerated fibrosarcoma kinase</p></td></tr><tr><td class="NLM_term">egfr</td><td class="NLM_def"><p class="first last">epidermal growth factor receptor</p></td></tr><tr><td class="NLM_term">11β-HSD1</td><td class="NLM_def"><p class="first last">11β-hydroxysteroid dehydrogenase</p></td></tr><tr><td class="NLM_term">NADPH</td><td class="NLM_def"><p class="first last">reduced form of nicotinamide adenine dinucleotide phosphate</p></td></tr><tr><td class="NLM_term">fgfr1</td><td class="NLM_def"><p class="first last">fibroblast growth factor receptor 1</p></td></tr><tr><td class="NLM_term">lck</td><td class="NLM_def"><p class="first last">lymphocyte cell-specific protein-tyrosine kinase</p></td></tr><tr><td class="NLM_term">DFG</td><td class="NLM_def"><p class="first last">aspartate–phenylalanine–glycine sequence motif</p></td></tr></table></div></div><div class="NLM_fn" id="csd">a <p class="last">For more information about CSD-CrossMiner or if you are interested in a free trial of this prototype application, please contact <a href="/cdn-cgi/l/email-protection#1170757c787f51727275723f72707c3f70723f647a"><span class="__cf_email__" data-cfemail="cfaeaba2a6a18facacabace1acaea2e1aeace1baa4">[email protected]</span></a>.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75244" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75244" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 46 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Allen, F. H.</span><span> </span><span class="NLM_article-title">The Cambridge Structural Database: A Quarter of a Million Crystal Structures and Rising</span> <span class="citation_source-journal">Acta Crystallogr., Sect. B: Struct. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">380</span><span class="NLM_x">–</span> <span class="NLM_lpage">388</span><span class="refDoi"> DOI: 10.1107/S0108768102003890</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1107%2FS0108768102003890" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=12037359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BD38XktVOqu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=380-388&author=F.+H.+Allen&title=The+Cambridge+Structural+Database%3A+A+Quarter+of+a+Million+Crystal+Structures+and+Rising&doi=10.1107%2FS0108768102003890"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Cambridge Structural Database: a quarter of a million crystal structures and rising</span></div><div class="casAuthors">Allen, Frank H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section B: Structural Science</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">B58</span>
        (<span class="NLM_cas:issue">3, No. 1</span>),
    <span class="NLM_cas:pages">380-388</span>CODEN:
                <span class="NLM_cas:coden">ASBSDK</span>;
        ISSN:<span class="NLM_cas:issn">0108-7681</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">The Cambridge Structural Database (CSD) now contains data for more than a quarter of a million small-mol. crystal structures.  The information content of the CSD, together with methods for data acquisition, processing and validation, are summarized, with particular emphasis on the chem. information added by CSD editors.  Nearly 80% of new structural data arrives electronically, mostly in CIF format, and the CCDC acts as the official crystal structure data depository for 51 major journals.  The CCDC now maintains both a CIF archive (more than 73000 CIFs dating from 1996), as well as the distributed binary CSD archive; the availability of data in both archives is discussed.  A statistical survey of the CSD is also presented and projections concerning future accession rates indicate that the CSD will contain at least 500000 crystal structures by the year 2010.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_Uk3GetkiibVg90H21EOLACvtfcHk0lgfTxxZ9fpawQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XktVOqu74%253D&md5=406cd0df6ea9035a0ebf8dd9eccbd1f8</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1107%2FS0108768102003890&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0108768102003890%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DF.%2BH.%26atitle%3DThe%2520Cambridge%2520Structural%2520Database%253A%2520A%2520Quarter%2520of%2520a%2520Million%2520Crystal%2520Structures%2520and%2520Rising%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520B%253A%2520Struct.%2520Sci.%26date%3D2002%26volume%3D58%26spage%3D380%26epage%3D388%26doi%3D10.1107%2FS0108768102003890" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battistuz, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bluhm, W. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burkhardt, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iype, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fagan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marvin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padilla, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ravichandran, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thanki, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zardecki, C.</span><span> </span><span class="NLM_article-title">The Protein Data Bank</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">899</span><span class="NLM_x">–</span> <span class="NLM_lpage">907</span><span class="refDoi"> DOI: 10.1107/S0907444902003451</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1107%2FS0907444902003451" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=12037327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BD38XktVKhtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=899-907&author=H.+M.+Bermanauthor=T.+Battistuzauthor=T.+N.+Bhatauthor=W.+F.+Bluhmauthor=P.+E.+Bourneauthor=K.+Burkhardtauthor=Z.+Fengauthor=G.+L.+Gillilandauthor=L.+Iypeauthor=S.+Jainauthor=P.+Faganauthor=J.+Marvinauthor=D.+Padillaauthor=V.+Ravichandranauthor=B.+Schneiderauthor=N.+Thankiauthor=H.+Weissigauthor=J.+D.+Westbrookauthor=C.+Zardecki&title=The+Protein+Data+Bank&doi=10.1107%2FS0907444902003451"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Battistuz, Tammy; Bhat, T. N.; Bluhm, Wolfgang F.; Bourne, Philip E.; Burkhardt, Kyle; Feng, Zukang; Gilliland, Gary L.; Iype, Lisa; Jain, Shri; Fagan, Phoebe; Marvin, Jessica; Padilla, David; Ravichandran, Veerasamy; Schneider, Bohdan; Thanki, Narmada; Weissig, Helge; Westbrook, John D.; Zardecki, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">6, No. 1</span>),
    <span class="NLM_cas:pages">899-907</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">The Protein Data Bank [PDB; Berman, Westbrook et al. (2000), Nucleic Acids Res. 28, 235-242; http://www.pdb.org/] is the single worldwide archive of primary structural data of biol. macromols.  Many secondary sources of information are derived from PDB data.  It is the starting point for studies in structural bioinformatics.  This article describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information and plans for the future development of the resource.  The reader should come away with an understanding of the scope of the PDB and what is provided by the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsUFcMyys-PrVg90H21EOLACvtfcHk0lgfTxxZ9fpawQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XktVKhtLg%253D&md5=d7594ad00ec8e5bacc413d38bceaa360</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1107%2FS0907444902003451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444902003451%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DBattistuz%26aufirst%3DT.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DBluhm%26aufirst%3DW.%2BF.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DBurkhardt%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%2BL.%26aulast%3DIype%26aufirst%3DL.%26aulast%3DJain%26aufirst%3DS.%26aulast%3DFagan%26aufirst%3DP.%26aulast%3DMarvin%26aufirst%3DJ.%26aulast%3DPadilla%26aufirst%3DD.%26aulast%3DRavichandran%26aufirst%3DV.%26aulast%3DSchneider%26aufirst%3DB.%26aulast%3DThanki%26aufirst%3DN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DWestbrook%26aufirst%3DJ.%2BD.%26aulast%3DZardecki%26aufirst%3DC.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2002%26volume%3D58%26spage%3D899%26epage%3D907%26doi%3D10.1107%2FS0907444902003451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Bernstein, F. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koetzle, T. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, G. J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, E. F.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brice, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodgers, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennard, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimanouchi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tasumi, M.</span><span> </span><span class="NLM_article-title">The Protein Data Bank: A Computer-Based Archival File for Macromolecular Structures</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">535</span><span class="NLM_x">–</span> <span class="NLM_lpage">542</span><span class="refDoi"> DOI: 10.1016/S0022-2836(77)80200-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1016%2FS0022-2836%2877%2980200-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=875032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADyaE2sXkvVert7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=1977&pages=535-542&author=F.+C.+Bernsteinauthor=T.+F.+Koetzleauthor=G.+J.+B.+Williamsauthor=E.+F.+Meyerauthor=M.+D.+Briceauthor=J.+R.+Rodgersauthor=O.+Kennardauthor=T.+Shimanouchiauthor=M.+Tasumi&title=The+Protein+Data+Bank%3A+A+Computer-Based+Archival+File+for+Macromolecular+Structures&doi=10.1016%2FS0022-2836%2877%2980200-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank:  a computer-based archival file for macromolecular structures</span></div><div class="casAuthors">Bernstein, Frances C.; Koetzle, Thomas F.; Williams, Graheme J. B.; Meyer, Edgar F., Jr.; Brice, Michael D.; Rodgers, John R.; Kennard, Olga; Shimanouchi, Takehiko; Tasumi, Mitsuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">535-42</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">The Protein Data Bank is a computer-based archival file for macromol. structures.  The Bank stores in a uniform format at. coordinates and partial bond connectivities, as derived from crystallog. studies.  Text included in each data entry gives pertinent information for the structure at hand (e.g., species from which the mol. was obtained, resoln. of diffraction data, literature citations, and specifications of secondary structure).  In addn., the Protein Data Bank stores structure factors and phases.  Input of data and general maintenance functions are at Brookhaven National Lab.  All data are available on magnetic tape for public distribution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9tv93dkDhxrVg90H21EOLACvtfcHk0lj3dYZITmH03A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXkvVert7o%253D&md5=07736506e495f157c600cf51a253a0a0</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2877%2980200-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252877%252980200-3%26sid%3Dliteratum%253Aachs%26aulast%3DBernstein%26aufirst%3DF.%2BC.%26aulast%3DKoetzle%26aufirst%3DT.%2BF.%26aulast%3DWilliams%26aufirst%3DG.%2BJ.%2BB.%26aulast%3DMeyer%26aufirst%3DE.%2BF.%26aulast%3DBrice%26aufirst%3DM.%2BD.%26aulast%3DRodgers%26aufirst%3DJ.%2BR.%26aulast%3DKennard%26aufirst%3DO.%26aulast%3DShimanouchi%26aufirst%3DT.%26aulast%3DTasumi%26aufirst%3DM.%26atitle%3DThe%2520Protein%2520Data%2520Bank%253A%2520A%2520Computer-Based%2520Archival%2520File%2520for%2520Macromolecular%2520Structures%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1977%26volume%3D112%26spage%3D535%26epage%3D542%26doi%3D10.1016%2FS0022-2836%2877%2980200-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Schärfer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulz-Gasch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ehrlich, H.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guba, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span> </span><span class="NLM_article-title">Torsion Angle Preferences in Druglike Chemical Space: A Comprehensive Guide</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2016</span><span class="NLM_x">–</span> <span class="NLM_lpage">2028</span><span class="refDoi"> DOI: 10.1021/jm3016816</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3016816" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A280%3ADC%252BC3szltlGitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2016-2028&author=C.+Sch%C3%A4rferauthor=T.+Schulz-Gaschauthor=H.-C.+Ehrlichauthor=W.+Gubaauthor=M.+Rareyauthor=M.+Stahl&title=Torsion+Angle+Preferences+in+Druglike+Chemical+Space%3A+A+Comprehensive+Guide&doi=10.1021%2Fjm3016816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Torsion angle preferences in druglike chemical space: a comprehensive guide</span></div><div class="casAuthors">Scharfer Christin; Schulz-Gasch Tanja; Ehrlich Hans-Christian; Guba Wolfgang; Rarey Matthias; Stahl Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2016-28</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Crystal structure databases offer ample opportunities to derive small molecule conformation preferences, but the derived knowledge is not systematically applied in drug discovery research.  We address this gap by a comprehensive and extendable expert system enabling quick assessment of the probability of a given conformation to occur.  It is based on a hierarchical system of torsion patterns that cover a large part of druglike chemical space.  Each torsion pattern has associated frequency histograms generated from CSD and PDB data and, derived from the histograms, traffic-light rules for frequently observed, rare, and highly unlikely torsion ranges.  Structures imported into the corresponding software are annotated according to these rules.  We present the concept behind the tree of torsion patterns, the design of an intuitive user interface for the management and usage of the torsion library, and we illustrate how the system helps analyze and understand conformation properties of substructures widely used in medicinal chemistry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR2PDX1V9jtEkEgZCMqtRBifW6udTcc2eZ_TOVCb43GE7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3szltlGitw%253D%253D&md5=7d0c2f9177cef90f0d7376836a8cefb2</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm3016816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3016816%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25A4rfer%26aufirst%3DC.%26aulast%3DSchulz-Gasch%26aufirst%3DT.%26aulast%3DEhrlich%26aufirst%3DH.-C.%26aulast%3DGuba%26aufirst%3DW.%26aulast%3DRarey%26aufirst%3DM.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DTorsion%2520Angle%2520Preferences%2520in%2520Druglike%2520Chemical%2520Space%253A%2520A%2520Comprehensive%2520Guide%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2016%26epage%3D2028%26doi%3D10.1021%2Fjm3016816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Kuhn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohr, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span> </span><span class="NLM_article-title">Intramolecular Hydrogen Bonding in Medicinal Chemistry</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2601</span><span class="NLM_x">–</span> <span class="NLM_lpage">2611</span><span class="refDoi"> DOI: 10.1021/jm100087s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100087s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Clur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2601-2611&author=B.+Kuhnauthor=P.+Mohrauthor=M.+Stahl&title=Intramolecular+Hydrogen+Bonding+in+Medicinal+Chemistry&doi=10.1021%2Fjm100087s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Intramolecular Hydrogen Bonding in Medicinal Chemistry</span></div><div class="casAuthors">Kuhn, Bernd; Mohr, Peter; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2601-2611</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The formation of intramol. hydrogen bonds has a very pronounced effect on mol. structure and properties.  We study both aspects in detail with the aim of enabling a more rational use of this class of interactions in medicinal chem.  On the basis of exhaustive searches in crystal structure databases, we derive propensities for intramol. hydrogen bond formation of five- to eight-membered ring systems of relevance in drug discovery.  A no. of motifs, several of which are clearly underutilized in drug discovery, are analyzed in more detail by comparing small mol. and protein-ligand X-ray structures.  To investigate effects on physicochem. properties, sets of closely related structures with and without the ability to form intramol. hydrogen bonds were designed, synthesized, and characterized with respect to membrane permeability, water soly., and lipophilicity.  We find that changes in these properties depend on a subtle balance between the strength of the hydrogen bond interaction, geometry of the newly formed ring system, and the relative energies of the open and closed conformations in polar and unpolar environments.  A no. of general guidelines for medicinal chemists emerge from this study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdaV7tFRU4k7Vg90H21EOLACvtfcHk0lj3dYZITmH03A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Clur8%253D&md5=b9b6865dfcc9bdf7796d15218f6e8ff2</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm100087s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100087s%26sid%3Dliteratum%253Aachs%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DMohr%26aufirst%3DP.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DIntramolecular%2520Hydrogen%2520Bonding%2520in%2520Medicinal%2520Chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2601%26epage%3D2611%26doi%3D10.1021%2Fjm100087s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Bissantz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span> </span><span class="NLM_article-title">A Medicinal Chemist’s Guide to Molecular Interactions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5061</span><span class="NLM_x">–</span> <span class="NLM_lpage">5084</span><span class="refDoi"> DOI: 10.1021/jm100112j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100112j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5061-5084&author=C.+Bissantzauthor=B.+Kuhnauthor=M.+Stahl&title=A+Medicinal+Chemist%E2%80%99s+Guide+to+Molecular+Interactions&doi=10.1021%2Fjm100112j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A Medicinal Chemist's Guide to Molecular Interactions</span></div><div class="casAuthors">Bissantz, Caterina; Kuhn, Bernd; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5061-5084</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraV8HPbwsihbVg90H21EOLACvtfcHk0lgGvxh3cYVKfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D&md5=f39cf8700267cdc311c73de4d433bab0</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fjm100112j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100112j%26sid%3Dliteratum%253Aachs%26aulast%3DBissantz%26aufirst%3DC.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DA%2520Medicinal%2520Chemist%25E2%2580%2599s%2520Guide%2520to%2520Molecular%2520Interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5061%26epage%3D5084%26doi%3D10.1021%2Fjm100112j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span> </span><span class="NLM_article-title">SuperStar: A Knowledge-Based Approach for Identifying Interaction Sites in Proteins</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">1093</span><span class="NLM_x">–</span> <span class="NLM_lpage">1108</span><span class="refDoi"> DOI: 10.1006/jmbi.1999.2809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1006%2Fjmbi.1999.2809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10369784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADyaK1MXjslKhtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=1999&pages=1093-1108&author=M.+L.+Verdonkauthor=J.+C.+Coleauthor=R.+Taylor&title=SuperStar%3A+A+Knowledge-Based+Approach+for+Identifying+Interaction+Sites+in+Proteins&doi=10.1006%2Fjmbi.1999.2809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">SuperStar: A Knowledge-based Approach for Identifying Interaction Sites in Proteins</span></div><div class="casAuthors">Verdonk, Marcel L.; Cole, Jason C.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1093-1108</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">An empirical method for identifying interaction sites in proteins is described and validated.  The method is based entirely on exptl. information about non-bonded interactions occurring in small-mol. crystal structures.  These data are used in the form of scatterplots that show the exptl. obsd. distribution of one functional group (the "contact group" or "probe") around another.  A template mol. (e.g. a protein binding site) is broken down into structure fragments and the scatterplots, showing the distribution of a chosen probe around these structure fragments, are superimposed on the corresponding parts of the template.  The scatterplots are then translated into a three-dimensional map that shows the propensity of the probe at different positions around the template mol.  The method is illustrated for L-arabinose-binding protein, complexed with L-arabinose and with D-fucose, and for dihydrofolate reductase complexed with methotrexate.  The method is validated on 122 x-ray structures of protein-ligand complexes.  For all the binding sites of these proteins, propensity maps are generated for four different probes: a charged NH+3 nitrogen, a carbonyl oxygen, a hydroxyl oxygen and a Me carbon atom.  Next, the maps are compared with the exptl. obsd. positions of ligand atoms of these types.  For 74% of these ligand atoms (84% of the solvent-inaccessible ones) the calcd. propensity of the matching probe at the exptl. positions is higher than expected by chance.  For 68% of the atoms (82% of the solvent-inaccessible ones) the propensity of the matching probe is higher than that of the other three probes.  These results indicate that the approach generally gives good predictions for protein-ligand interactions.  The potential applications of the propensity maps range from an aid in manual docking and structure-based drug design to their use in pharmacophore development.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9QVkFGAwFt7Vg90H21EOLACvtfcHk0lgGvxh3cYVKfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjslKhtrs%253D&md5=72e09c54c9a631a8271fbe72f2f431b1</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1999.2809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1999.2809%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DSuperStar%253A%2520A%2520Knowledge-Based%2520Approach%2520for%2520Identifying%2520Interaction%2520Sites%2520in%2520Proteins%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1999%26volume%3D289%26spage%3D1093%26epage%3D1108%26doi%3D10.1006%2Fjmbi.1999.2809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Bruno, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lommerse, J. P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowland, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span> </span><span class="NLM_article-title">IsoStar: A Library of Information about Nonbonded Interactions</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">525</span><span class="NLM_x">–</span> <span class="NLM_lpage">537</span><span class="refDoi"> DOI: 10.1023/A:1007934413448</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1023%2FA%3A1007934413448" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=9491345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADyaK1cXht1Olu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1997&pages=525-537&author=I.+J.+Brunoauthor=J.+C.+Coleauthor=J.+P.+M.+Lommerseauthor=R.+S.+Rowlandauthor=R.+Taylorauthor=M.+L.+Verdonk&title=IsoStar%3A+A+Library+of+Information+about+Nonbonded+Interactions&doi=10.1023%2FA%3A1007934413448"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Isostar: a library of information about nonbonded interactions</span></div><div class="casAuthors">Bruno, Ian J.; Cole, Jason C.; Lommerse, Jos P. M.; Rowland, R. Scott; Taylor, Robin; Verdonk, Marcel L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">525-537</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">Crystallog. and theor. (ab initio) data on intermol. nonbonded interactions were gathered together in a computerized library ('IsoStar').  The library contains information about the nonbonded contacts formed by some 250 chem. groupings.  The data can be displayed visually and used to aid protein-ligand docking or the identification of bioisosteric replacements.  Data from the library show that there is great variability in the geometrical preferences of different types of hydrogen bonds, although in general there is a tendency for H-bonds to form along lone-pair directions.  The H-bond acceptor abilities of oxygen and sulfur atoms are highly dependent on intramol. environments.  The nonbonded contacts formed by many hydrophobic groups show surprisingly strong directional preferences.  Many unusual nonbonded interactions are to be found in the library and are of potential value for designing novel biol. active mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcIzn6KO_NVbVg90H21EOLACvtfcHk0lgGvxh3cYVKfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXht1Olu7s%253D&md5=ec168252a1dc7bddc341235db0f5ccfb</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1023%2FA%3A1007934413448&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1007934413448%26sid%3Dliteratum%253Aachs%26aulast%3DBruno%26aufirst%3DI.%2BJ.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DLommerse%26aufirst%3DJ.%2BP.%2BM.%26aulast%3DRowland%26aufirst%3DR.%2BS.%26aulast%3DTaylor%26aufirst%3DR.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26atitle%3DIsoStar%253A%2520A%2520Library%2520of%2520Information%2520about%2520Nonbonded%2520Interactions%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1997%26volume%3D11%26spage%3D525%26epage%3D537%26doi%3D10.1023%2FA%3A1007934413448" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Kuhn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuchs, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reutlinger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, N. R.</span><span> </span><span class="NLM_article-title">Rationalizing Tight Ligand Binding through Cooperative Interaction Networks</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">3180</span><span class="NLM_x">–</span> <span class="NLM_lpage">3198</span><span class="refDoi"> DOI: 10.1021/ci200319e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200319e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVOlt7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=3180-3198&author=B.+Kuhnauthor=J.+E.+Fuchsauthor=M.+Reutlingerauthor=M.+Stahlauthor=N.+R.+Taylor&title=Rationalizing+Tight+Ligand+Binding+through+Cooperative+Interaction+Networks&doi=10.1021%2Fci200319e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Rationalizing Tight Ligand Binding through Cooperative Interaction Networks</span></div><div class="casAuthors">Kuhn, Bernd; Fuchs, Julian E.; Reutlinger, Michael; Stahl, Martin; Taylor, Neil R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3180-3198</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small modifications of the mol. structure of a ligand sometimes cause strong gains in binding affinity to a protein target, rendering a weakly active chem. series suddenly attractive for further optimization.  Our goal in this study is to better rationalize and predict the occurrence of such interaction hot-spots in receptor binding sites.  To this end, we introduce two new concepts into the computational description of mol. recognition.  First, we take a broader view of noncovalent interactions and describe protein-ligand binding with a comprehensive set of favorable and unfavorable contact types, including for example halogen bonding and orthogonal multipolar interactions.  Second, we go beyond the commonly used pairwise additive treatment of at. interactions and use a small world network approach to describe how interactions are modulated by their environment.  This approach allows us to capture local cooperativity effects and considerably improves the performance of a newly derived empirical scoring function, Scorpions-core.  More importantly, however, we demonstrate how an intuitive visualization of key intermol. interactions, interaction networks, and binding hot-spots supports the identification and rationalization of tight ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCzUUIjVW8P7Vg90H21EOLACvtfcHk0liao33kvBapPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVOlt7rI&md5=e8e1bdc3bad1cfdfc11253de5837e4c6</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fci200319e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200319e%26sid%3Dliteratum%253Aachs%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3DReutlinger%26aufirst%3DM.%26aulast%3DStahl%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DN.%2BR.%26atitle%3DRationalizing%2520Tight%2520Ligand%2520Binding%2520through%2520Cooperative%2520Interaction%2520Networks%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D3180%26epage%3D3198%26doi%3D10.1021%2Fci200319e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Quantum Mechanical Methods for Drug Design</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span><span class="NLM_x">–</span> <span class="NLM_lpage">45</span><span class="refDoi"> DOI: 10.2174/156802610790232242</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.2174%2F156802610790232242" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=19929831" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFWqu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=33-45&author=T.+Zhouauthor=D.+Huangauthor=A.+Caflisch&title=Quantum+Mechanical+Methods+for+Drug+Design&doi=10.2174%2F156802610790232242"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Quantum mechanical methods for drug design</span></div><div class="casAuthors">Zhou, Ting; Huang, Danzhi; Caflisch, Amedeo</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-45</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Quantum mech. (QM) methods are becoming popular in computational drug design and development mainly because high accuracy is required to est. (relative) binding affinities.  For low- to medium-throughput in silico screening, (e.g., scoring and prioritizing a series of inhibitors sharing the same mol. scaffold) efficient approxns. have been developed in the past decade, like linear scaling QM in which the computation time scales almost linearly with the no. of basis functions.  Furthermore, QM-based procedures have been used recently for detg. protonation states of ionizable groups, evaluating energies, and optimizing mol. structures.  For high-throughput in silico screening QM approaches have been employed to derive robust quant. structure-activity relation models.  It is expected that the use of QM methods will keep growing in all phases of computer-aided drug design and development.  However, extensive sampling of conformational space and treatment of soln. of macromols. are still limiting factors for the broad application of QM in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocoQjA2ielFbVg90H21EOLACvtfcHk0liao33kvBapPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFWqu7o%253D&md5=735ac208504a7e563e6ca0eb3c1cd17b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.2174%2F156802610790232242&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802610790232242%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DQuantum%2520Mechanical%2520Methods%2520for%2520Drug%2520Design%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2010%26volume%3D10%26spage%3D33%26epage%3D45%26doi%3D10.2174%2F156802610790232242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Groom, C.</span><span> </span><span class="NLM_article-title">Small Molecule Crystal Structures in Drug Discovery</span>. In  <span class="citation_source-book">Multifaceted Roles of Crystallography in Modern Drug Discovery</span>; <span class="NLM_contrib-group">Scapin, G.; Patel, D.; Arnold, E.</span>, Eds.; NATO Science for Peace and Security Series A: Chemistry and Biology; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">The Netherlands</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">107</span><span class="NLM_x">–</span> <span class="NLM_lpage">114</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1007%2F978-94-017-9719-1_9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=107-114&author=C.+Groomauthor=G.+Scapin&author=D.+Patel&author=E.+Arnold&title=Multifaceted+Roles+of+Crystallography+in+Modern+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1007%2F978-94-017-9719-1_9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-94-017-9719-1_9%26sid%3Dliteratum%253Aachs%26aulast%3DGroom%26aufirst%3DC.%26atitle%3DSmall%2520Molecule%2520Crystal%2520Structures%2520in%2520Drug%2520Discovery%26btitle%3DMultifaceted%2520Roles%2520of%2520Crystallography%2520in%2520Modern%2520Drug%2520Discovery%26aulast%3DScapin%26aufirst%3DG.%26pub%3DSpringer%26date%3D2015%26spage%3D107%26epage%3D114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Bruno, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edgington, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macrae, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span> </span><span class="NLM_article-title">New Software for Searching the Cambridge Structural Database and Visualizing Crystal Structures</span> <span class="citation_source-journal">Acta Crystallogr., Sect. B: Struct. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">389</span><span class="NLM_x">–</span> <span class="NLM_lpage">397</span><span class="refDoi"> DOI: 10.1107/S0108768102003324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1107%2FS0108768102003324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=12037360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BD38XktVOqu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=389-397&author=I.+J.+Brunoauthor=J.+C.+Coleauthor=P.+R.+Edgingtonauthor=M.+Kesslerauthor=C.+F.+Macraeauthor=P.+McCabeauthor=J.+Pearsonauthor=R.+Taylor&title=New+Software+for+Searching+the+Cambridge+Structural+Database+and+Visualizing+Crystal+Structures&doi=10.1107%2FS0108768102003324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">New software for searching the Cambridge Structural Database and visualizing crystal structures</span></div><div class="casAuthors">Bruno, Ian J.; Cole, Jason C.; Edgington, Paul R.; Kessler, Magnus; Macrae, Clare F.; McCabe, Patrick; Pearson, Jonathan; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section B: Structural Science</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">B58</span>
        (<span class="NLM_cas:issue">3, No. 1</span>),
    <span class="NLM_cas:pages">389-397</span>CODEN:
                <span class="NLM_cas:coden">ASBSDK</span>;
        ISSN:<span class="NLM_cas:issn">0108-7681</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">Two new programs were developed for searching the Cambridge Structural Database (CSD) and visualizing database entries: ConQuest and Mercury.  The former is a new search interface to the CSD, the latter is a high-performance crystal-structure visualizer with extensive facilities for exploring networks of intermol. contacts.  Particular emphasis was placed on making the programs as intuitive as possible.  Both ConQuest and Mercury run under Windows and various types of Unix, including Linux.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIZFqT_ocomLVg90H21EOLACvtfcHk0liC_kC_GMW95w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XktVOqu78%253D&md5=b8cd5dddcd43067010fef6d60e37b3c2</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1107%2FS0108768102003324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0108768102003324%26sid%3Dliteratum%253Aachs%26aulast%3DBruno%26aufirst%3DI.%2BJ.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DEdgington%26aufirst%3DP.%2BR.%26aulast%3DKessler%26aufirst%3DM.%26aulast%3DMacrae%26aufirst%3DC.%2BF.%26aulast%3DMcCabe%26aufirst%3DP.%26aulast%3DPearson%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DNew%2520Software%2520for%2520Searching%2520the%2520Cambridge%2520Structural%2520Database%2520and%2520Visualizing%2520Crystal%2520Structures%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520B%253A%2520Struct.%2520Sci.%26date%3D2002%26volume%3D58%26spage%3D389%26epage%3D397%26doi%3D10.1107%2FS0108768102003324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Hendlich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Günther, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Relibase: Design and Development of a Database for Comprehensive Analysis of Protein-Ligand Interactions</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">326</span><span class="NLM_x">, </span> <span class="NLM_fpage">607</span><span class="NLM_x">–</span> <span class="NLM_lpage">620</span><span class="refDoi"> DOI: 10.1016/S0022-2836(02)01408-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1016%2FS0022-2836%2802%2901408-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=326&publication_year=2003&pages=607-620&author=M.+Hendlichauthor=A.+Bergnerauthor=J.+G%C3%BCntherauthor=G.+Klebe&title=Relibase%3A+Design+and+Development+of+a+Database+for+Comprehensive+Analysis+of+Protein-Ligand+Interactions&doi=10.1016%2FS0022-2836%2802%2901408-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2901408-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252901408-0%26sid%3Dliteratum%253Aachs%26aulast%3DHendlich%26aufirst%3DM.%26aulast%3DBergner%26aufirst%3DA.%26aulast%3DG%25C3%25BCnther%26aufirst%3DJ.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DRelibase%253A%2520Design%2520and%2520Development%2520of%2520a%2520Database%2520for%2520Comprehensive%2520Analysis%2520of%2520Protein-Ligand%2520Interactions%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2003%26volume%3D326%26spage%3D607%26epage%3D620%26doi%3D10.1016%2FS0022-2836%2802%2901408-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Inhester, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span> </span><span class="NLM_article-title">Protein–ligand Interaction Databases: Advanced Tools to Mine Activity Data and Interactions on a Structural Level</span> <span class="citation_source-journal">Wiley Interdiscip. Rev. Comput. Mol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">562</span><span class="NLM_x">–</span> <span class="NLM_lpage">575</span><span class="refDoi"> DOI: 10.1002/wcms.1192</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1002%2Fwcms.1192" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVelu7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=562-575&author=T.+Inhesterauthor=M.+Rarey&title=Protein%E2%80%93ligand+Interaction+Databases%3A+Advanced+Tools+to+Mine+Activity+Data+and+Interactions+on+a+Structural+Level&doi=10.1002%2Fwcms.1192"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-ligand interaction databases: advanced tools to mine activity data and interactions on a structural level</span></div><div class="casAuthors">Inhester, Therese; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Wiley Interdisciplinary Reviews: Computational Molecular Science</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">562-575</span>CODEN:
                <span class="NLM_cas:coden">WIRCAH</span>;
        ISSN:<span class="NLM_cas:issn">1759-0884</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The formation of mol. complexes between proteins and small org. substances is a fundamental concept of life.  Biochem. expts. from X-ray crystallog. to isothermal titrn. calorimetry (ITC) are applied in large-scale providing data for the anal. of the structural foundations of binding affinity.  In recent years, several, mostly publically available databases emerged contg. affinity data and structural information.  These databases are central for the construction of complex models describing interaction geometries and correlate structural features to the strength of binding.  Binding affinity databases reflect the knowledge of affinity measurements from many sources, mostly scientific and patent literature.  A crit. aspect is the data quality, which is affected by transcription errors during database construction as well as exptl. uncertainties.  The Protein Data Bank (PDB) is the central resource for macromol. biol. structures contg. nearly 100,000 data entries today.  Sophisticated geometric databases have been constructed based on this allowing for complex queries about the spatial arrangement of functional groups and their interactions.  For scientists working in mol. design like medicinal chemists, access to this information can substantially support the process of creating new mol. entities specifically interacting with proteins of interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVnCd3Xg79fLVg90H21EOLACvtfcHk0liC_kC_GMW95w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVelu7jL&md5=7bce4b6447299944bd7598eaf0d3e7e0</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2Fwcms.1192&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwcms.1192%26sid%3Dliteratum%253Aachs%26aulast%3DInhester%26aufirst%3DT.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DProtein%25E2%2580%2593ligand%2520Interaction%2520Databases%253A%2520Advanced%2520Tools%2520to%2520Mine%2520Activity%2520Data%2520and%2520Interactions%2520on%2520a%2520Structural%2520Level%26jtitle%3DWiley%2520Interdiscip.%2520Rev.%2520Comput.%2520Mol.%2520Sci.%26date%3D2014%26volume%3D4%26spage%3D562%26epage%3D575%26doi%3D10.1002%2Fwcms.1192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Schreyer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T.</span><span> </span><span class="NLM_article-title">CREDO: A Protein–Ligand Interaction Database for Drug Discovery</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">157</span><span class="NLM_x">–</span> <span class="NLM_lpage">167</span><span class="refDoi"> DOI: 10.1111/j.1747-0285.2008.00762.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1111%2Fj.1747-0285.2008.00762.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=19207418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitlGit7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2009&pages=157-167&author=A.+Schreyerauthor=T.+Blundell&title=CREDO%3A+A+Protein%E2%80%93Ligand+Interaction+Database+for+Drug+Discovery&doi=10.1111%2Fj.1747-0285.2008.00762.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">CREDO: a protein-ligand interaction database for drug discovery</span></div><div class="casAuthors">Schreyer, Adrian; Blundell, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-167</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Harnessing data from the growing no. of protein-ligand complexes in the Protein Data Bank is an important task in drug discovery.  In order to benefit from the abundance of three-dimensional structures, structural data must be integrated with sequence as well as chem. data and the protein-small mol. interactions characterized structurally at the inter-at. level.  In this study, we present CREDO, a new publicly available database of protein-ligand interactions, which represents contacts as structural interaction fingerprints, implements novel features and is completely scriptable through its application programming interface.  Features of CREDO include implementation of mol. shape descriptors with ultrafast shape recognition, fragmentation of ligands in the Protein Data Bank, sequence-to-structure mapping and the identification of approved drugs.  Selected analyses of these key features are presented to highlight a range of potential applications of CREDO.  The CREDO dataset has been released into the public domain together with the application programming interface under a Creative Commons license at http://www-cryst.bioc.cam.ac.uk/credo.  We believe that the free availability and numerous features of CREDO database will be useful not only for com. but also for academia-driven drug discovery programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqun6w6sZJKzrVg90H21EOLACvtfcHk0liC_kC_GMW95w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitlGit7s%253D&md5=15a45aa365edc83d294848b637ae31d0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2008.00762.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2008.00762.x%26sid%3Dliteratum%253Aachs%26aulast%3DSchreyer%26aufirst%3DA.%26aulast%3DBlundell%26aufirst%3DT.%26atitle%3DCREDO%253A%2520A%2520Protein%25E2%2580%2593Ligand%2520Interaction%2520Database%2520for%2520Drug%2520Discovery%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2009%26volume%3D73%26spage%3D157%26epage%3D167%26doi%3D10.1111%2Fj.1747-0285.2008.00762.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Weisel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bitter, H.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diederich, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">So, W. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kondru, R.</span><span> </span><span class="NLM_article-title">PROLIX: Rapid Mining of Protein-Ligand Interactions in Large Crystal Structure Databases</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1450</span><span class="NLM_x">–</span> <span class="NLM_lpage">1461</span><span class="refDoi"> DOI: 10.1021/ci300034x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300034x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvV2nsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1450-1461&author=M.+Weiselauthor=H.-M.+Bitterauthor=F.+Diederichauthor=W.+V.+Soauthor=R.+Kondru&title=PROLIX%3A+Rapid+Mining+of+Protein-Ligand+Interactions+in+Large+Crystal+Structure+Databases&doi=10.1021%2Fci300034x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">PROLIX: Rapid Mining of Protein-Ligand Interactions in Large Crystal Structure Databases</span></div><div class="casAuthors">Weisel, Martin; Bitter, Hans-Marcus; Diederich, Francois; So, W. Venus; Kondru, Rama</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1450-1461</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A central problem in structure-based drug design is understanding protein-ligand interactions quant. and qual.  Several recent studies have highlighted from a qual. perspective the nature of these interactions and their utility in drug discovery.  However, a common limitation is a lack of adequate tools to mine these interactions comprehensively, since exhaustive searches of the protein data bank are time-consuming and difficult to perform.  Consequently, fundamental questions remain unanswered: How unique or how common are the protein-ligand interactions obsd. in a given drug design project when compared to all complexed structures in the protein data bank.  Which interaction patterns might explain the affinity of a tool compd. toward unwanted targets.  To answer these questions and to enable the systematic and comprehensive study of protein-ligand interactions, we introduce PROLIX (Protein Ligand Interaction Explorer), a tool that uses sophisticated fingerprint representations of protein-ligand interaction patterns for rapid data mining in large crystal structure databases.  Our implementation strategy pursues a branch-and-bound technique that enables mining against thousands of complexes within a few seconds.  Key elements of PROLIX include (i) an intuitive interface that enables users to formulate complex queries easily, (ii) exceptional speed for results retrieval, and (iii) a sophisticated results summarization.  Herein we describe the algorithms developed to enable complex queries and fast retrieval of search results, as well as the intuitive aspects of the user interface and summarization viewer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi4_gNpv_-8rVg90H21EOLACvtfcHk0liC_kC_GMW95w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvV2nsro%253D&md5=dfcfea5c25b13d2cc9543735a9d11b15</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fci300034x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300034x%26sid%3Dliteratum%253Aachs%26aulast%3DWeisel%26aufirst%3DM.%26aulast%3DBitter%26aufirst%3DH.-M.%26aulast%3DDiederich%26aufirst%3DF.%26aulast%3DSo%26aufirst%3DW.%2BV.%26aulast%3DKondru%26aufirst%3DR.%26atitle%3DPROLIX%253A%2520Rapid%2520Mining%2520of%2520Protein-Ligand%2520Interactions%2520in%2520Large%2520Crystal%2520Structure%2520Databases%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1450%26epage%3D1461%26doi%3D10.1021%2Fci300034x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Shirvanyants, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alexandrova, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dokholyan, N. V.</span><span> </span><span class="NLM_article-title">Rigid Substructure Search</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">1327</span><span class="NLM_x">–</span> <span class="NLM_lpage">1329</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btr129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1093%2Fbioinformatics%2Fbtr129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=21460026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkvFajt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=1327-1329&author=D.+Shirvanyantsauthor=A.+N.+Alexandrovaauthor=N.+V.+Dokholyan&title=Rigid+Substructure+Search&doi=10.1093%2Fbioinformatics%2Fbtr129"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Rigid substructure search</span></div><div class="casAuthors">Shirvanyants, David; Alexandrova, Anastassia N.; Dokholyan, Nikolay V.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1327-1329</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Identifying the location of binding sites on proteins is of fundamental importance for a wide range of applications, including mol. docking, de novo drug design, structure identification and comparison of functional sites.  Here we present Erebus, a web server that searches the entire Protein Data Bank for a given substructure defined by a set of atoms of interest, such as the binding scaffolds for small mols.  The identified substructure contains atoms having the same names, belonging to same amino acids and sepd. by the same distances (within a given tolerance) as the atoms of the query structure.  The accuracy of a match is measured by the root-mean-square deviation or by the normal wt. with a given variance.  Tests show that our approach can reliably locate rigid binding scaffolds of drugs and metal ions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxJlxMIQf27LVg90H21EOLACvtfcHk0lgoXFk8ImqLJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkvFajt7k%253D&md5=d650bf655e40b5d129278c8b7e5d6570</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtr129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtr129%26sid%3Dliteratum%253Aachs%26aulast%3DShirvanyants%26aufirst%3DD.%26aulast%3DAlexandrova%26aufirst%3DA.%2BN.%26aulast%3DDokholyan%26aufirst%3DN.%2BV.%26atitle%3DRigid%2520Substructure%2520Search%26jtitle%3DBioinformatics%26date%3D2011%26volume%3D27%26spage%3D1327%26epage%3D1329%26doi%3D10.1093%2Fbioinformatics%2Fbtr129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Schmitt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">A New Method to Detect Related Function among Proteins Independent of Sequence and Fold Homology</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">323</span><span class="NLM_x">, </span> <span class="NLM_fpage">387</span><span class="NLM_x">–</span> <span class="NLM_lpage">406</span><span class="refDoi"> DOI: 10.1016/S0022-2836(02)00811-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1016%2FS0022-2836%2802%2900811-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=12381328" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BD38XnslGqsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=323&publication_year=2002&pages=387-406&author=S.+Schmittauthor=D.+Kuhnauthor=G.+Klebe&title=A+New+Method+to+Detect+Related+Function+among+Proteins+Independent+of+Sequence+and+Fold+Homology&doi=10.1016%2FS0022-2836%2802%2900811-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A new method to detect related function among proteins independent of sequence and fold homology</span></div><div class="casAuthors">Schmitt, Stefan; Kuhn, Daniel; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">323</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">387-406</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A new method has been developed to detect functional relationships among proteins independent of a given sequence or fold homol.  It is based on the idea that protein function is intimately related to the recognition and subsequent response to the binding of a substrate or an endogenous ligand in a well-characterized binding pocket.  Thus, recognition of similar ligands, supposedly linked to similar function, requires conserved recognition features exposed in terms of common physicochem. interaction properties via the functional groups of the residues flanking a particular binding cavity.  Following a technique commonly used in the comparison of small mol. ligands, generic pseudocenters coding for possible interaction properties were assigned for a large sample set of cavities extd. from the entire PDB and stored in the database Cavbase.  Using a particular query cavity a series of related cavities of decreasing similarity is detected based on a clique detection algorithm.  The detected similarity is ranked according to property-based surface patches shared in common by the different clique solns.  The approach either retrieves protein cavities accommodating the same (e.g. co-factors) or closely related ligands or it exts. proteins exhibiting similar function in terms of a related catalytic mechanism.  Finally the new method has strong potential to suggest alternative mol. skeletons in de novo design.  The retrieval of mol. building blocks accommodated in a particular sub-pocket that shares similarity with the pocket in a protein studied by drug design can inspire the discovery of novel ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRCdsfEkyF6rVg90H21EOLACvtfcHk0lgoXFk8ImqLJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnslGqsrY%253D&md5=192ad6a44a2d4d5406ca47cecb65bd50</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2900811-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252900811-2%26sid%3Dliteratum%253Aachs%26aulast%3DSchmitt%26aufirst%3DS.%26aulast%3DKuhn%26aufirst%3DD.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DA%2520New%2520Method%2520to%2520Detect%2520Related%2520Function%2520among%2520Proteins%2520Independent%2520of%2520Sequence%2520and%2520Fold%2520Homology%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D323%26spage%3D387%26epage%3D406%26doi%3D10.1016%2FS0022-2836%2802%2900811-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Desaphy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Sc-PDB-Frag: A Database of Protein–Ligand Interaction Patterns for Bioisosteric Replacements</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1908</span><span class="NLM_x">–</span> <span class="NLM_lpage">1918</span><span class="refDoi"> DOI: 10.1021/ci500282c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500282c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFShsrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1908-1918&author=J.+Desaphyauthor=D.+Rognan&title=Sc-PDB-Frag%3A+A+Database+of+Protein%E2%80%93Ligand+Interaction+Patterns+for+Bioisosteric+Replacements&doi=10.1021%2Fci500282c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">sc-PDB-Frag: A Database of Protein-Ligand Interaction Patterns for Bioisosteric Replacements</span></div><div class="casAuthors">Desaphy, Jeremy; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1908-1918</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bioisosteric replacement plays an important role in medicinal chem. by keeping the biol. activity of a mol. while changing either its core scaffold or substituents, thereby facilitating lead optimization and patenting.  Bioisosteres are classically chosen in order to keep the main pharmacophoric moieties of the substructure to replace.  However, notably when changing a scaffold, no attention is usually paid as whether all atoms of the ref. scaffold are equally important for binding to the desired target.  We herewith propose a novel database for bioisosteric replacement (scPDBFrag), capitalizing on our recently published structure-based approach to scaffold hopping, focusing on interaction pattern graphs.  Protein-bound ligands are first fragmented and the interaction of the corresponding fragments with their protein environment computed-on-the-fly.  Using an inhouse developed graph alignment tool, interaction patterns graphs can be compared, aligned, and sorted by decreasing similarity to any ref.  In the herein presented s.c.-PDB-Frag database (http://bioinfo-pharma.u-strasbg.fr/scPDBFrag), fragments, interaction patterns, alignments, and pairwise similarity scores have been extd. from the s.c.-PDB database of 8077 druggable protein-ligand complexes and further stored in a relational database.  We herewith present the database, its Web implementation, and procedures for identifying true bioisosteric replacements based on conserved interaction patterns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTtQN9l6kql7Vg90H21EOLACvtfcHk0lgoXFk8ImqLJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFShsrfK&md5=db24b1007d32c27af5d076d133aa5bcb</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fci500282c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500282c%26sid%3Dliteratum%253Aachs%26aulast%3DDesaphy%26aufirst%3DJ.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DSc-PDB-Frag%253A%2520A%2520Database%2520of%2520Protein%25E2%2580%2593Ligand%2520Interaction%2520Patterns%2520for%2520Bioisosteric%2520Replacements%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1908%26epage%3D1918%26doi%3D10.1021%2Fci500282c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Golovin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henrick, K.</span><span> </span><span class="NLM_article-title">MSDmotif: Exploring Protein Sites and Motifs</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">312</span><span class="refDoi"> DOI: 10.1186/1471-2105-9-312</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1186%2F1471-2105-9-312" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=18637174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A280%3ADC%252BD1cvos1Glug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=312&author=A.+Golovinauthor=K.+Henrick&title=MSDmotif%3A+Exploring+Protein+Sites+and+Motifs&doi=10.1186%2F1471-2105-9-312"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">MSDmotif: exploring protein sites and motifs</span></div><div class="casAuthors">Golovin Adel; Henrick Kim</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">312</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Protein structures have conserved features - motifs, which have a sufficient influence on the protein function.  These motifs can be found in sequence as well as in 3D space.  Understanding of these fragments is essential for 3D structure prediction, modelling and drug-design.  The Protein Data Bank (PDB) is the source of this information however present search tools have limited 3D options to integrate protein sequence with its 3D structure.  RESULTS:  We describe here a web application for querying the PDB for ligands, binding sites, small 3D structural and sequence motifs and the underlying database.  Novel algorithms for chemical fragments, 3D motifs, phi/psi sequences, super-secondary structure motifs and for small 3D structural motif associations searches are incorporated.  The interface provides functionality for visualization, search criteria creation, sequence and 3D multiple alignment options.  MSDmotif is an integrated system where a results page is also a search form.  A set of motif statistics is available for analysis.  This set includes molecule and motif binding statistics, distribution of motif sequences, occurrence of an amino-acid within a motif, correlation of amino-acids side-chain charges within a motif and Ramachandran plots for each residue.  The binding statistics are presented in association with properties that include a ligand fragment library.  Access is also provided through the distributed Annotation System (DAS) protocol.  An additional entry point facilitates XML requests with XML responses.  CONCLUSION:  MSDmotif is unique by combining chemical, sequence and 3D data in a single search engine with a range of search and visualisation options.  It provides multiple views of data found in the PDB archive for exploring protein structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQNtCwFapSCF8AKB7Fa-xrUfW6udTcc2eYoDfduuYI2_rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1cvos1Glug%253D%253D&md5=23056f227d4293f9b1a756972403faa0</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-9-312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-9-312%26sid%3Dliteratum%253Aachs%26aulast%3DGolovin%26aufirst%3DA.%26aulast%3DHenrick%26aufirst%3DK.%26atitle%3DMSDmotif%253A%2520Exploring%2520Protein%2520Sites%2520and%2520Motifs%26jtitle%3DBMC%2520Bioinf.%26date%3D2008%26volume%3D9%26spage%3D312%26doi%3D10.1186%2F1471-2105-9-312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Gonzalez, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hannigan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeGrado, W. F.</span><span> </span><span class="NLM_article-title">A Real-Time All-Atom Structural Search Engine for Proteins</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003750</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1003750</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1371%2Fjournal.pcbi.1003750" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=25079944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1ersLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1003750&author=G.+Gonzalezauthor=B.+Hanniganauthor=W.+F.+DeGrado&title=A+Real-Time+All-Atom+Structural+Search+Engine+for+Proteins&doi=10.1371%2Fjournal.pcbi.1003750"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A real-time all-atom structural search engine for proteins</span></div><div class="casAuthors">Gonzalez, Gabriel; Hannigan, Brett; DeGrado, William F.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e1003750/1-e1003750/8, 8 pp.</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein designers use a wide variety of software tools for de novo design, yet their repertoire still lacks a fast and interactive all-atom search engine.  To solve this, we have built the Suns program: a real-time, at. search engine integrated into the PyMOL mol. visualization system.  Users build at.-level structural search queries within PyMOL and receive a stream of search results aligned to their query within a few seconds.  This instant feedback cycle enables a new "designability"-inspired approach to protein design where the designer searches for and interactively incorporates native-like fragments from proven protein structures.  We demonstrate the use of Suns to interactively build protein motifs, tertiary interactions, and to identify scaffolds compatible with hot-spot residues.  The official web site and installer are available at online.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9e-aJDstg5rVg90H21EOLACvtfcHk0ljXWEkpcmH_MQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1ersLfN&md5=18c14d2b95bc3ec62ebe56fac2a30ba0</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003750%26sid%3Dliteratum%253Aachs%26aulast%3DGonzalez%26aufirst%3DG.%26aulast%3DHannigan%26aufirst%3DB.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3DA%2520Real-Time%2520All-Atom%2520Structural%2520Search%2520Engine%2520for%2520Proteins%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3De1003750%26doi%3D10.1371%2Fjournal.pcbi.1003750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span>Desert Scientific Software. <a href="http://www.desertsci.com/" class="extLink">http://www.desertsci.com/</a> (accessed Aug 26,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Desert+Scientific+Software.+http%3A%2F%2Fwww.desertsci.com%2F+%28accessed+Aug+26%2C+2015%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span>SQLite Home Page. <a href="http://www.sqlite.org/" class="extLink">http://www.sqlite.org/</a> (accessed Oct 20,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=SQLite+Home+Page.+http%3A%2F%2Fwww.sqlite.org%2F+%28accessed+Oct+20%2C+2015%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Van Drie, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weininger, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, Y. C.</span><span> </span><span class="NLM_article-title">ALADDIN: An Integrated Tool for Computer-Assisted Molecular Design and Pharmacophore Recognition from Geometric, Steric, and Substructure Searching of Three-Dimensional Molecular Structures</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">225</span><span class="NLM_x">–</span> <span class="NLM_lpage">251</span><span class="refDoi"> DOI: 10.1007/BF01533070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1007%2FBF01533070" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1989&pages=225-251&author=J.+H.+Van+Drieauthor=D.+Weiningerauthor=Y.+C.+Martin&title=ALADDIN%3A+An+Integrated+Tool+for+Computer-Assisted+Molecular+Design+and+Pharmacophore+Recognition+from+Geometric%2C+Steric%2C+and+Substructure+Searching+of+Three-Dimensional+Molecular+Structures&doi=10.1007%2FBF01533070"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2FBF01533070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01533070%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDrie%26aufirst%3DJ.%2BH.%26aulast%3DWeininger%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DY.%2BC.%26atitle%3DALADDIN%253A%2520An%2520Integrated%2520Tool%2520for%2520Computer-Assisted%2520Molecular%2520Design%2520and%2520Pharmacophore%2520Recognition%2520from%2520Geometric%252C%2520Steric%252C%2520and%2520Substructure%2520Searching%2520of%2520Three-Dimensional%2520Molecular%2520Structures%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1989%26volume%3D3%26spage%3D225%26epage%3D251%26doi%3D10.1007%2FBF01533070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Sastry, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span> </span><span class="NLM_article-title">Rapid Shape-Based Ligand Alignment and Virtual Screening Method Based on Atom/Feature-Pair Similarities and Volume Overlap Scoring</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2455</span><span class="NLM_x">–</span> <span class="NLM_lpage">2466</span><span class="refDoi"> DOI: 10.1021/ci2002704</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci2002704" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFKnt73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2455-2466&author=G.+M.+Sastryauthor=S.+L.+Dixonauthor=W.+Sherman&title=Rapid+Shape-Based+Ligand+Alignment+and+Virtual+Screening+Method+Based+on+Atom%2FFeature-Pair+Similarities+and+Volume+Overlap+Scoring&doi=10.1021%2Fci2002704"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Shape-Based Ligand Alignment and Virtual Screening Method Based on Atom/Feature-Pair Similarities and Volume Overlap Scoring</span></div><div class="casAuthors">Sastry, G. Madhavi; Dixon, Steven L.; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2455-2466</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Shape-based methods for aligning and scoring ligands have proven to be valuable in the field of computer-aided drug design.  Here, we describe a new shape-based flexible ligand superposition and virtual screening method, Phase Shape, which is shown to rapidly produce accurate 3D ligand alignments and efficiently enrich actives in virtual screening.  We describe the methodol., which is based on the principle of atom distribution triplets to rapidly define trial alignments, followed by refinement of top alignments to maximize the vol. overlap.  The method can be run in a shape-only mode or it can include atom types or pharmacophore feature encoding, the latter consistently producing the best results for database screening.  We apply Phase Shape to flexibly align mols. that bind to the same target and show that the method consistently produces correct alignments when compared with crystal structures.  We then illustrate the effectiveness of the method for identifying active compds. in virtual screening of eleven diverse targets.  Multiple parameters are explored, including atom typing, query structure conformation, and the database conformer generation protocol.  We show that Phase Shape performs well in database screening calcns. when compared with other shape-based methods using a common set of actives and decoys from the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM-u23DvDpQ7Vg90H21EOLACvtfcHk0ljXWEkpcmH_MQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFKnt73K&md5=4234abb2dd7e6364c377e92047b65aba</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fci2002704&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci2002704%26sid%3Dliteratum%253Aachs%26aulast%3DSastry%26aufirst%3DG.%2BM.%26aulast%3DDixon%26aufirst%3DS.%2BL.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DRapid%2520Shape-Based%2520Ligand%2520Alignment%2520and%2520Virtual%2520Screening%2520Method%2520Based%2520on%2520Atom%252FFeature-Pair%2520Similarities%2520and%2520Volume%2520Overlap%2520Scoring%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2455%26epage%3D2466%26doi%3D10.1021%2Fci2002704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Wolber, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langer, T.</span><span> </span><span class="NLM_article-title">LigandScout: 3-D Pharmacophores Derived from Protein-Bound Ligands and Their Use as Virtual Screening Filters</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">160</span><span class="NLM_x">–</span> <span class="NLM_lpage">169</span><span class="refDoi"> DOI: 10.1021/ci049885e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049885e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=160-169&author=G.+Wolberauthor=T.+Langer&title=LigandScout%3A+3-D+Pharmacophores+Derived+from+Protein-Bound+Ligands+and+Their+Use+as+Virtual+Screening+Filters&doi=10.1021%2Fci049885e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">LigandScout: 3-D pharmacophores derived from protein-bound ligands and their use as virtual screening filters</span></div><div class="casAuthors">Wolber Gerhard; Langer Thierry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">160-9</span>
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    </div><div class="casAbstract">From the historically grown archive of protein-ligand complexes in the Protein Data Bank small organic ligands are extracted and interpreted in terms of their chemical characteristics and features.  Subsequently, pharmacophores representing ligand-receptor interaction are derived from each of these small molecules and its surrounding amino acids.  Based on a defined set of only six types of chemical features and volume constraints, three-dimensional pharmacophore models are constructed, which are sufficiently selective to identify the described binding mode and are thus a useful tool for in-silico screening of large compound databases.  The algorithms for ligand extraction and interpretation as well as the pharmacophore creation technique from the automatically interpreted data are presented and applied to a rhinovirus capsid complex as application example.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQVi_-excgTX2BAiYWy6n3SfW6udTcc2eZn7DJluS53Y7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtg%253D%253D&md5=31f8ff7a2fa4c9411d54b001b7e1da50</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fci049885e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049885e%26sid%3Dliteratum%253Aachs%26aulast%3DWolber%26aufirst%3DG.%26aulast%3DLanger%26aufirst%3DT.%26atitle%3DLigandScout%253A%25203-D%2520Pharmacophores%2520Derived%2520from%2520Protein-Bound%2520Ligands%2520and%2520Their%2520Use%2520as%2520Virtual%2520Screening%2520Filters%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D160%26epage%3D169%26doi%3D10.1021%2Fci049885e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span>Daylight Theory: SMARTS—A Language for Describing Molecular Patterns. <a href="http://www.daylight.com/dayhtml/doc/theory/theory.smarts.html" class="extLink">http://www.daylight.com/dayhtml/doc/theory/theory.smarts.html</a> (accessed Jun 1,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Daylight+Theory%3A+SMARTS%E2%80%94A+Language+for+Describing+Molecular+Patterns.+http%3A%2F%2Fwww.daylight.com%2Fdayhtml%2Fdoc%2Ftheory%2Ftheory.smarts.html+%28accessed+Jun+1%2C+2015%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Vitaku, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Njardarson, J. T.</span><span> </span><span class="NLM_article-title">Analysis of the Structural Diversity, Substitution Patterns, and Frequency of Nitrogen Heterocycles among U.S. FDA Approved Pharmaceuticals</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">10257</span><span class="NLM_x">–</span> <span class="NLM_lpage">10274</span><span class="refDoi"> DOI: 10.1021/jm501100b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501100b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlu7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=10257-10274&author=E.+Vitakuauthor=D.+T.+Smithauthor=J.+T.+Njardarson&title=Analysis+of+the+Structural+Diversity%2C+Substitution+Patterns%2C+and+Frequency+of+Nitrogen+Heterocycles+among+U.S.+FDA+Approved+Pharmaceuticals&doi=10.1021%2Fjm501100b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of the Structural Diversity, Substitution Patterns, and Frequency of Nitrogen Heterocycles among U.S. FDA Approved Pharmaceuticals</span></div><div class="casAuthors">Vitaku, Edon; Smith, David T.; Njardarson, Jon T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10257-10274</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Nitrogen heterocycles are among the most significant structural components of pharmaceuticals.  Anal. of our database of U.S. FDA approved drugs reveals that 59% of unique small-mol. drugs contain a nitrogen heterocycle.  In this review we report on the top 25 most commonly utilized nitrogen heterocycles found in pharmaceuticals.  The main part of our anal. is divided into seven sections: (1) three- and four-membered heterocycles, (2) five-, (3) six-, and (4) seven- and eight-membered heterocycles, as well as (5) fused, (6) bridged bicyclic, and (7) macrocyclic nitrogen heterocycles.  Each section reveals the top nitrogen heterocyclic structures and their relative impact for that ring type.  For the most commonly used nitrogen heterocycles, we report detailed substitution patterns, highlight common architectural cores, and discuss unusual or rare structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5cUk9KxUSkbVg90H21EOLACvtfcHk0lhjhMUS6hC67Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlu7%252FP&md5=7065b3b2fc6f69cede0f87479c7cf472</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm501100b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501100b%26sid%3Dliteratum%253Aachs%26aulast%3DVitaku%26aufirst%3DE.%26aulast%3DSmith%26aufirst%3DD.%2BT.%26aulast%3DNjardarson%26aufirst%3DJ.%2BT.%26atitle%3DAnalysis%2520of%2520the%2520Structural%2520Diversity%252C%2520Substitution%2520Patterns%252C%2520and%2520Frequency%2520of%2520Nitrogen%2520Heterocycles%2520among%2520U.S.%2520FDA%2520Approved%2520Pharmaceuticals%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D10257%26epage%3D10274%26doi%3D10.1021%2Fjm501100b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Hartshorn, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brewerton, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mooij, W. T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mortenson, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span> </span><span class="NLM_article-title">Diverse, High-Quality Test Set for the Validation of Protein–Ligand Docking Performance</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">726</span><span class="NLM_x">–</span> <span class="NLM_lpage">741</span><span class="refDoi"> DOI: 10.1021/jm061277y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm061277y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmsVymtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=726-741&author=M.+J.+Hartshornauthor=M.+L.+Verdonkauthor=G.+Chessariauthor=S.+C.+Brewertonauthor=W.+T.+M.+Mooijauthor=P.+N.+Mortensonauthor=C.+W.+Murray&title=Diverse%2C+High-Quality+Test+Set+for+the+Validation+of+Protein%E2%80%93Ligand+Docking+Performance&doi=10.1021%2Fjm061277y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Diverse, High-Quality Test Set for the Validation of Protein-Ligand Docking Performance</span></div><div class="casAuthors">Hartshorn, Michael J.; Verdonk, Marcel L.; Chessari, Gianni; Brewerton, Suzanne C.; Mooij, Wijnand T. M.; Mortenson, Paul N.; Murray, Christopher W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">726-741</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A procedure for analyzing and classifying publicly available crystal structures has been developed.  It has been used to identify high-resoln. protein-ligand complexes that can be assessed by reconstructing the electron d. for the ligand using the deposited structure factors.  The complexes have been clustered according to the protein sequences, and clusters have been discarded if they do not represent proteins thought to be of direct interest to the pharmaceutical or agrochem. industry.  Rules have been used to exclude complexes contg. nondrug-like ligands.  One complex from each cluster has been selected where a structure of sufficient quality was available.  The final Astex diverse set contains 85 diverse, relevant protein-ligand complexes, which have been prepd. in a format suitable for docking and are to be made freely available to the entire research community (http://www.ccdc.cam.ac.uk).  The performance of the docking program GOLD against the new set is assessed using a variety of protocols.  Relatively unbiased protocols give success rates of approx. 80% for redocking into native structures, but it is possible to get success rates of over 90% with some protocols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFW8sJ7rQ0abVg90H21EOLACvtfcHk0lhjhMUS6hC67Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmsVymtg%253D%253D&md5=8a5ae20c581f8a242945cced94709443</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm061277y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061277y%26sid%3Dliteratum%253Aachs%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DBrewerton%26aufirst%3DS.%2BC.%26aulast%3DMooij%26aufirst%3DW.%2BT.%2BM.%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26atitle%3DDiverse%252C%2520High-Quality%2520Test%2520Set%2520for%2520the%2520Validation%2520of%2520Protein%25E2%2580%2593Ligand%2520Docking%2520Performance%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D726%26epage%3D741%26doi%3D10.1021%2Fjm061277y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Chisholm, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motherwell, S.</span><span> </span><span class="NLM_article-title">A New Algorithm for Performing Three-Dimensional Searches of the Cambridge Structural Database</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">331</span><span class="NLM_x">–</span> <span class="NLM_lpage">334</span><span class="refDoi"> DOI: 10.1107/S0021889804002924</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1107%2FS0021889804002924" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitVagtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2004&pages=331-334&author=J.+A.+Chisholmauthor=S.+Motherwell&title=A+New+Algorithm+for+Performing+Three-Dimensional+Searches+of+the+Cambridge+Structural+Database&doi=10.1107%2FS0021889804002924"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A new algorithm for performing three-dimensional searches of the Cambridge Structural Database</span></div><div class="casAuthors">Chisholm, James A.; Motherwell, Sam</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">331-334</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A search algorithm, 3DSEARCH, is presented that can readily identify challenging extended chem. queries from three-dimensional mol. crystal structure information.  The program combines substructure search and distance search techniques within a depth-first backtracking algorithm.  Performance metrics are presented for example searches composed of several substructures and several intermol. connections.  Such searches, which are outside the capabilities of current search engines, can now be performed on the entire Cambridge Structural Database with search times of around half an hour.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouEqCGY4cEjbVg90H21EOLACvtfcHk0lgBHy3GnEJOtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitVagtbY%253D&md5=5b2c4002d3a36708dad10326d1010c71</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0021889804002924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889804002924%26sid%3Dliteratum%253Aachs%26aulast%3DChisholm%26aufirst%3DJ.%2BA.%26aulast%3DMotherwell%26aufirst%3DS.%26atitle%3DA%2520New%2520Algorithm%2520for%2520Performing%2520Three-Dimensional%2520Searches%2520of%2520the%2520Cambridge%2520Structural%2520Database%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2004%26volume%3D37%26spage%3D331%26epage%3D334%26doi%3D10.1107%2FS0021889804002924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">A Solution for the Best Rotation to Relate Two Sets of Vectors</span> <span class="citation_source-journal">Acta Crystallogr., Sect. A: Cryst. Phys., Diffr., Theor. Gen. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1976</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">922</span><span class="NLM_x">–</span> <span class="NLM_lpage">923</span><span class="refDoi"> DOI: 10.1107/S0567739476001873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1107%2FS0567739476001873" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1976&pages=922-923&author=W.+Kabsch&title=A+Solution+for+the+Best+Rotation+to+Relate+Two+Sets+of+Vectors&doi=10.1107%2FS0567739476001873"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0567739476001873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0567739476001873%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DA%2520Solution%2520for%2520the%2520Best%2520Rotation%2520to%2520Relate%2520Two%2520Sets%2520of%2520Vectors%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520A%253A%2520Cryst.%2520Phys.%252C%2520Diffr.%252C%2520Theor.%2520Gen.%2520Crystallogr.%26date%3D1976%26volume%3D32%26spage%3D922%26epage%3D923%26doi%3D10.1107%2FS0567739476001873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Ehmke, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winkler, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banner, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haap, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schweizer, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rottmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaiser, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freymond, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schirmeister, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diederich, F.</span><span> </span><span class="NLM_article-title">Optimization of Triazine Nitriles as Rhodesain Inhibitors: Structure–Activity Relationships, Bioisosteric Imidazopyridine Nitriles, and X-Ray Crystal Structure Analysis with Human Cathepsin L</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">967</span><span class="NLM_x">–</span> <span class="NLM_lpage">975</span><span class="refDoi"> DOI: 10.1002/cmdc.201300112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1002%2Fcmdc.201300112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=23658062" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntlSru7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=967-975&author=V.+Ehmkeauthor=E.+Winklerauthor=D.+W.+Bannerauthor=W.+Haapauthor=W.+B.+Schweizerauthor=M.+Rottmannauthor=M.+Kaiserauthor=C.+Freymondauthor=T.+Schirmeisterauthor=F.+Diederich&title=Optimization+of+Triazine+Nitriles+as+Rhodesain+Inhibitors%3A+Structure%E2%80%93Activity+Relationships%2C+Bioisosteric+Imidazopyridine+Nitriles%2C+and+X-Ray+Crystal+Structure+Analysis+with+Human+Cathepsin+L&doi=10.1002%2Fcmdc.201300112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of Triazine Nitriles as Rhodesain Inhibitors: Structure-Activity Relationships, Bioisosteric Imidazopyridine Nitriles, and X-ray Crystal Structure Analysis with Human Cathepsin L</span></div><div class="casAuthors">Ehmke, Veronika; Winkler, Edwin; Banner, David W.; Haap, Wolfgang; Schweizer, W. Bernd; Rottmann, Matthias; Kaiser, Marcel; Freymond, Celine; Schirmeister, Tanja; Diederich, Francois</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">967-975</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The cysteine protease rhodesain of Trypanosoma brucei parasites causing African sleeping sickness has emerged as a target for the development of new drug candidates.  Based on a triazine nitrile moiety as electrophilic headgroup, optimization studies on the substituents for the S1, S2, and S3 pockets of the enzyme were performed using structure-based design and resulted in inhibitors with inhibition consts. in the single-digit nanomolar range.  Comprehensive structure-activity relationships clarified the binding preferences of the individual pockets of the active site.  The S1 pocket tolerates various substituents with a preference for flexible and basic side chains.  Variation of the S2 substituent led to high-affinity ligands with inhibition consts. down to 2 nM for compds. bearing cyclohexyl substituents.  Systematic investigations on the S3 pocket revealed its potential to achieve high activities with arom. vectors that undergo stacking interactions with the planar peptide backbone forming part of the pocket.  X-ray crystal structure anal. with the structurally related enzyme human cathepsin L confirmed the binding mode of the triazine ligand series as proposed by mol. modeling.  Sub-micromolar inhibition of the proliferation of cultured parasites was achieved for ligands decorated with the best substituents identified through the optimization cycles.  In cell-based assays, the introduction of a basic side chain on the inhibitors resulted in a 35-fold increase in antitrypanosomal activity.  Finally, bioisosteric imidazopyridine nitriles were studied to prevent off-target effects with unselective nucleophiles by decreasing the inherent electrophilicity of the triazine nitrile headgroup.  Using this ligand, the stabilization by intramol. hydrogen bonding of the thioimidate intermediate, formed upon attack of the catalytic cysteine residue, compensates for the lower reactivity of the headgroup.  The imidazopyridine nitrile ligand showed excellent stability toward the thiol nucleophile glutathione in a quant. in vitro assay and fourfold lower cytotoxicity than the parent triazine nitrile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGwTj6sYRPi7Vg90H21EOLACvtfcHk0lgBHy3GnEJOtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntlSru7k%253D&md5=0db889bf80bc6ea9fa3c16bfed80f614</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300112%26sid%3Dliteratum%253Aachs%26aulast%3DEhmke%26aufirst%3DV.%26aulast%3DWinkler%26aufirst%3DE.%26aulast%3DBanner%26aufirst%3DD.%2BW.%26aulast%3DHaap%26aufirst%3DW.%26aulast%3DSchweizer%26aufirst%3DW.%2BB.%26aulast%3DRottmann%26aufirst%3DM.%26aulast%3DKaiser%26aufirst%3DM.%26aulast%3DFreymond%26aufirst%3DC.%26aulast%3DSchirmeister%26aufirst%3DT.%26aulast%3DDiederich%26aufirst%3DF.%26atitle%3DOptimization%2520of%2520Triazine%2520Nitriles%2520as%2520Rhodesain%2520Inhibitors%253A%2520Structure%25E2%2580%2593Activity%2520Relationships%252C%2520Bioisosteric%2520Imidazopyridine%2520Nitriles%252C%2520and%2520X-Ray%2520Crystal%2520Structure%2520Analysis%2520with%2520Human%2520Cathepsin%2520L%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26spage%3D967%26epage%3D975%26doi%3D10.1002%2Fcmdc.201300112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Hardegger, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spinnler, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anselm, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecabert, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stihle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gsell, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thoma, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plancher, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartmann, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banner, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haap, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diederich, F.</span><span> </span><span class="NLM_article-title">Systematic Investigation of Halogen Bonding in Protein–Ligand Interactions</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">314</span><span class="NLM_x">–</span> <span class="NLM_lpage">318</span><span class="refDoi"> DOI: 10.1002/anie.201006781</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1002%2Fanie.201006781" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1alu73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=314-318&author=L.+A.+Hardeggerauthor=B.+Kuhnauthor=B.+Spinnlerauthor=L.+Anselmauthor=R.+Ecabertauthor=M.+Stihleauthor=B.+Gsellauthor=R.+Thomaauthor=J.+Diezauthor=J.+Benzauthor=J.-M.+Plancherauthor=G.+Hartmannauthor=D.+W.+Bannerauthor=W.+Haapauthor=F.+Diederich&title=Systematic+Investigation+of+Halogen+Bonding+in+Protein%E2%80%93Ligand+Interactions&doi=10.1002%2Fanie.201006781"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic Investigation of Halogen Bonding in Protein-Ligand Interactions</span></div><div class="casAuthors">Hardegger, Leo A.; Kuhn, Bernd; Spinnler, Beat; Anselm, Lilli; Ecabert, Robert; Stihle, Martine; Gsell, Bernard; Thoma, Ralf; Diez, Joachim; Benz, Joerg; Plancher, Jean-Marc; Hartmann, Guido; Banner, David W.; Haap, Wolfgang; Diederich, Francois</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">314-318, S314/1-S314/145</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Halogen bonding triggers activity.  Increasing binding affinity was obsd. for a series of covalent human Cathepsin L inhibitors by exchanging an aryl ring H atom with Cl, Br, and I, which undergo halogen bonding with the C=O group of Gly61 in the S3 pocket of the enzyme.  Fluorine, in contrast, strongly avoids halogen bonding.  The strong distance and angle dependence of halogen bonding was confirmed for biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbsbCNMQhVu7Vg90H21EOLACvtfcHk0lgBHy3GnEJOtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1alu73N&md5=1a7f2983a386fd133be0663657fc486b</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fanie.201006781&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201006781%26sid%3Dliteratum%253Aachs%26aulast%3DHardegger%26aufirst%3DL.%2BA.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DSpinnler%26aufirst%3DB.%26aulast%3DAnselm%26aufirst%3DL.%26aulast%3DEcabert%26aufirst%3DR.%26aulast%3DStihle%26aufirst%3DM.%26aulast%3DGsell%26aufirst%3DB.%26aulast%3DThoma%26aufirst%3DR.%26aulast%3DDiez%26aufirst%3DJ.%26aulast%3DBenz%26aufirst%3DJ.%26aulast%3DPlancher%26aufirst%3DJ.-M.%26aulast%3DHartmann%26aufirst%3DG.%26aulast%3DBanner%26aufirst%3DD.%2BW.%26aulast%3DHaap%26aufirst%3DW.%26aulast%3DDiederich%26aufirst%3DF.%26atitle%3DSystematic%2520Investigation%2520of%2520Halogen%2520Bonding%2520in%2520Protein%25E2%2580%2593Ligand%2520Interactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2011%26volume%3D50%26spage%3D314%26epage%3D318%26doi%3D10.1002%2Fanie.201006781" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Shenoy, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sivaraman, J.</span><span> </span><span class="NLM_article-title">Structural Basis for Reversible and Irreversible Inhibition of Human Cathepsin L by Their Respective Dipeptidyl Glyoxal and Diazomethylketone Inhibitors</span> <span class="citation_source-journal">J. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">173</span><span class="NLM_x">, </span> <span class="NLM_fpage">14</span><span class="NLM_x">–</span> <span class="NLM_lpage">19</span><span class="refDoi"> DOI: 10.1016/j.jsb.2010.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1016%2Fj.jsb.2010.09.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2011&pages=14-19&author=R.+T.+Shenoyauthor=J.+Sivaraman&title=Structural+Basis+for+Reversible+and+Irreversible+Inhibition+of+Human+Cathepsin+L+by+Their+Respective+Dipeptidyl+Glyoxal+and+Diazomethylketone+Inhibitors&doi=10.1016%2Fj.jsb.2010.09.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.jsb.2010.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsb.2010.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DShenoy%26aufirst%3DR.%2BT.%26aulast%3DSivaraman%26aufirst%3DJ.%26atitle%3DStructural%2520Basis%2520for%2520Reversible%2520and%2520Irreversible%2520Inhibition%2520of%2520Human%2520Cathepsin%2520L%2520by%2520Their%2520Respective%2520Dipeptidyl%2520Glyoxal%2520and%2520Diazomethylketone%2520Inhibitors%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D2011%26volume%3D173%26spage%3D14%26epage%3D19%26doi%3D10.1016%2Fj.jsb.2010.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Asaad, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bethel, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coulson, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dawson, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerhardt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grist, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamlin, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, E. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karoutchi, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenny, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morley, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oldham, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rankine, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Augustin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simader, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinbacher, S.</span><span> </span><span class="NLM_article-title">Dipeptidyl Nitrile Inhibitors of Cathepsin L</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">4280</span><span class="NLM_x">–</span> <span class="NLM_lpage">4283</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.05.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1016%2Fj.bmcl.2009.05.071" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=4280-4283&author=N.+Asaadauthor=P.+A.+Bethelauthor=M.+D.+Coulsonauthor=J.+E.+Dawsonauthor=S.+J.+Fordauthor=S.+Gerhardtauthor=M.+Gristauthor=G.+A.+Hamlinauthor=M.+J.+Jamesauthor=E.+V.+Jonesauthor=G.+I.+Karoutchiauthor=P.+W.+Kennyauthor=A.+D.+Morleyauthor=K.+Oldhamauthor=N.+Rankineauthor=D.+Ryanauthor=S.+L.+Wellsauthor=L.+Woodauthor=M.+Augustinauthor=S.+Krappauthor=H.+Simaderauthor=S.+Steinbacher&title=Dipeptidyl+Nitrile+Inhibitors+of+Cathepsin+L&doi=10.1016%2Fj.bmcl.2009.05.071"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.05.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.05.071%26sid%3Dliteratum%253Aachs%26aulast%3DAsaad%26aufirst%3DN.%26aulast%3DBethel%26aufirst%3DP.%2BA.%26aulast%3DCoulson%26aufirst%3DM.%2BD.%26aulast%3DDawson%26aufirst%3DJ.%2BE.%26aulast%3DFord%26aufirst%3DS.%2BJ.%26aulast%3DGerhardt%26aufirst%3DS.%26aulast%3DGrist%26aufirst%3DM.%26aulast%3DHamlin%26aufirst%3DG.%2BA.%26aulast%3DJames%26aufirst%3DM.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BV.%26aulast%3DKaroutchi%26aufirst%3DG.%2BI.%26aulast%3DKenny%26aufirst%3DP.%2BW.%26aulast%3DMorley%26aufirst%3DA.%2BD.%26aulast%3DOldham%26aufirst%3DK.%26aulast%3DRankine%26aufirst%3DN.%26aulast%3DRyan%26aufirst%3DD.%26aulast%3DWells%26aufirst%3DS.%2BL.%26aulast%3DWood%26aufirst%3DL.%26aulast%3DAugustin%26aufirst%3DM.%26aulast%3DKrapp%26aufirst%3DS.%26aulast%3DSimader%26aufirst%3DH.%26aulast%3DSteinbacher%26aufirst%3DS.%26atitle%3DDipeptidyl%2520Nitrile%2520Inhibitors%2520of%2520Cathepsin%2520L%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D4280%26epage%3D4283%26doi%3D10.1016%2Fj.bmcl.2009.05.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Harder, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diederich, F.</span><span> </span><span class="NLM_article-title">Efficient Stacking on Protein Amide Fragments</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">397</span><span class="NLM_x">–</span> <span class="NLM_lpage">404</span><span class="refDoi"> DOI: 10.1002/cmdc.201200512</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1002%2Fcmdc.201200512" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=23355480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVymsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=397-404&author=M.+Harderauthor=B.+Kuhnauthor=F.+Diederich&title=Efficient+Stacking+on+Protein+Amide+Fragments&doi=10.1002%2Fcmdc.201200512"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Stacking on Protein Amide Fragments</span></div><div class="casAuthors">Harder, Michael; Kuhn, Bernd; Diederich, Francois</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">397-404</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The less polar π-surface of protein amide groups is exposed in many receptor binding sites, either as part of the backbone or in Gln/Asn side chains.  Using quantum chem. calcns. and Protein Data Bank (PDB) searches on model systems, we investigate the energetics and geometric preferences for the stacking on amide groups of a large no. of heteroarenes that are relevant to medicinal chem.  From this study, we discern that the stacking energy of an arom. ligand substituent can be improved by: (1) orienting the fragment dipole vector such that it is aligned in an antiparallel fashion with the dipole of the interacting protein amide group, (2) increasing its dipole moment, and (3) decreasing its π-electron d.  These guidelines should be helpful to more rationally exploit this interaction type in future structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEY-6adf4ZHLVg90H21EOLACvtfcHk0lgVFWtpFuZg0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVymsLw%253D&md5=4dab202cc8cefb347d0d4bb3a55a941c</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201200512&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201200512%26sid%3Dliteratum%253Aachs%26aulast%3DHarder%26aufirst%3DM.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DDiederich%26aufirst%3DF.%26atitle%3DEfficient%2520Stacking%2520on%2520Protein%2520Amide%2520Fragments%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26spage%3D397%26epage%3D404%26doi%3D10.1002%2Fcmdc.201200512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Hardegger, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spinnler, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anselm, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecabert, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stihle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gsell, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thoma, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plancher, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartmann, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isshiki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikami, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimma, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haap, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banner, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diederich, F.</span><span> </span><span class="NLM_article-title">Halogen Bonding at the Active Sites of Human Cathepsin L and MEK1 Kinase: Efficient Interactions in Different Environments</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">2048</span><span class="NLM_x">–</span> <span class="NLM_lpage">2054</span><span class="refDoi"> DOI: 10.1002/cmdc.201100353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1002%2Fcmdc.201100353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=21898833" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFajtr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=2048-2054&author=L.+A.+Hardeggerauthor=B.+Kuhnauthor=B.+Spinnlerauthor=L.+Anselmauthor=R.+Ecabertauthor=M.+Stihleauthor=B.+Gsellauthor=R.+Thomaauthor=J.+Diezauthor=J.+Benzauthor=J.-M.+Plancherauthor=G.+Hartmannauthor=Y.+Isshikiauthor=K.+Morikamiauthor=N.+Shimmaauthor=W.+Haapauthor=D.+W.+Bannerauthor=F.+Diederich&title=Halogen+Bonding+at+the+Active+Sites+of+Human+Cathepsin+L+and+MEK1+Kinase%3A+Efficient+Interactions+in+Different+Environments&doi=10.1002%2Fcmdc.201100353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Halogen Bonding at the Active Sites of Human Cathepsin L and MEK1 Kinase: Efficient Interactions in Different Environments</span></div><div class="casAuthors">Hardegger, Leo A.; Kuhn, Bernd; Spinnler, Beat; Anselm, Lilli; Ecabert, Robert; Stihle, Martine; Gsell, Bernard; Thoma, Ralf; Diez, Joachim; Benz, Joerg; Plancher, Jean-Marc; Hartmann, Guido; Isshiki, Yoshiaki; Morikami, Kenji; Shimma, Nobuo; Haap, Wolfgang; Banner, David W.; Diederich, Francois</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2048-2054</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">In two series of small-mol. ligands, one inhibiting human cathepsin L (hcatL) and the other MEK1 kinase, biol. affinities were found to strongly increase when an aryl ring of the inhibitors is substituted with the larger halogens Cl, Br, and I, but to decrease upon F substitution.  X-ray co-crystal structure analyses revealed that the higher halides engage in halogen bonding (XB) with a backbone C=O in the S3 pocket of hcatL and in a back pocket of MEK1.  While the S3 pocket is located at the surface of the enzyme, which provides a polar environment, the back pocket in MEK1 is deeply buried in the protein and is of pronounced apolar character.  This study analyzes environmental effects on XB in protein-ligand complexes.  It is hypothesized that energetic gains by XB are predominantly not due to water replacements but originate from direct interactions between the XB donor (Caryl-X) and the XB acceptor (C=O) in the correct geometry.  New x-ray co-crystal structures in the same crystal form (space group P212121) were obtained for aryl chloride, bromide, and iodide ligands bound to hcatL.  These high-resoln. structures reveal that the backbone C=O group of Gly61 in most hcatL co-crystal structures maintains water solvation while engaging in XB.  An aryl-CF3-substituted ligand of hcatL with an unexpectedly high affinity was found to adopt the same binding geometry as the aryl halides, with the CF3 group pointing to the C=O group of Gly61 in the S3 pocket.  In this case, a repulsive F2C-F···O=C contact apparently is energetically overcompensated by other favorable protein-ligand contacts established by the CF3 group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnaNYbZ3KJUbVg90H21EOLACvtfcHk0lgVFWtpFuZg0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFajtr7E&md5=12c7cc93b65ba017acee37796811309d</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201100353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201100353%26sid%3Dliteratum%253Aachs%26aulast%3DHardegger%26aufirst%3DL.%2BA.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DSpinnler%26aufirst%3DB.%26aulast%3DAnselm%26aufirst%3DL.%26aulast%3DEcabert%26aufirst%3DR.%26aulast%3DStihle%26aufirst%3DM.%26aulast%3DGsell%26aufirst%3DB.%26aulast%3DThoma%26aufirst%3DR.%26aulast%3DDiez%26aufirst%3DJ.%26aulast%3DBenz%26aufirst%3DJ.%26aulast%3DPlancher%26aufirst%3DJ.-M.%26aulast%3DHartmann%26aufirst%3DG.%26aulast%3DIsshiki%26aufirst%3DY.%26aulast%3DMorikami%26aufirst%3DK.%26aulast%3DShimma%26aufirst%3DN.%26aulast%3DHaap%26aufirst%3DW.%26aulast%3DBanner%26aufirst%3DD.%2BW.%26aulast%3DDiederich%26aufirst%3DF.%26atitle%3DHalogen%2520Bonding%2520at%2520the%2520Active%2520Sites%2520of%2520Human%2520Cathepsin%2520L%2520and%2520MEK1%2520Kinase%253A%2520Efficient%2520Interactions%2520in%2520Different%2520Environments%26jtitle%3DChemMedChem%26date%3D2011%26volume%3D6%26spage%3D2048%26epage%3D2054%26doi%3D10.1002%2Fcmdc.201100353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Roehrig, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Straub, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pohlmann, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lampe, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pernerstorfer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlemmer, K.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinemer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perzborn, E.</span><span> </span><span class="NLM_article-title">Discovery of the Novel Antithrombotic Agent 5-Chloro-N-({(5S)-2-Oxo-3- [4-(3-Oxomorpholin-4-Yl)phenyl]-1,3-Oxazolidin-5-Yl}methyl)thiophene- 2-Carboxamide (BAY 59–7939): An Oral, Direct Factor Xa Inhibitor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">5900</span><span class="NLM_x">–</span> <span class="NLM_lpage">5908</span><span class="refDoi"> DOI: 10.1021/jm050101d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050101d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=5900-5908&author=S.+Roehrigauthor=A.+Straubauthor=J.+Pohlmannauthor=T.+Lampeauthor=J.+Pernerstorferauthor=K.-H.+Schlemmerauthor=P.+Reinemerauthor=E.+Perzborn&title=Discovery+of+the+Novel+Antithrombotic+Agent+5-Chloro-N-%28%7B%285S%29-2-Oxo-3-+%5B4-%283-Oxomorpholin-4-Yl%29phenyl%5D-1%2C3-Oxazolidin-5-Yl%7Dmethyl%29thiophene-+2-Carboxamide+%28BAY+59%E2%80%937939%29%3A+An+Oral%2C+Direct+Factor+Xa+Inhibitor&doi=10.1021%2Fjm050101d"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm050101d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050101d%26sid%3Dliteratum%253Aachs%26aulast%3DRoehrig%26aufirst%3DS.%26aulast%3DStraub%26aufirst%3DA.%26aulast%3DPohlmann%26aufirst%3DJ.%26aulast%3DLampe%26aufirst%3DT.%26aulast%3DPernerstorfer%26aufirst%3DJ.%26aulast%3DSchlemmer%26aufirst%3DK.-H.%26aulast%3DReinemer%26aufirst%3DP.%26aulast%3DPerzborn%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520the%2520Novel%2520Antithrombotic%2520Agent%25205-Chloro-N-%2528%257B%25285S%2529-2-Oxo-3-%2520%255B4-%25283-Oxomorpholin-4-Yl%2529phenyl%255D-1%252C3-Oxazolidin-5-Yl%257Dmethyl%2529thiophene-%25202-Carboxamide%2520%2528BAY%252059%25E2%2580%25937939%2529%253A%2520An%2520Oral%252C%2520Direct%2520Factor%2520Xa%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D5900%26epage%3D5908%26doi%3D10.1021%2Fjm050101d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Salonen, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bucher, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banner, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haap, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mary, J.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuster, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seiler, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schweizer, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diederich, F.</span><span> </span><span class="NLM_article-title">Cation−π Interactions at the Active Site of Factor Xa: Dramatic Enhancement upon Stepwise N-Alkylation of Ammonium Ions</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">811</span><span class="NLM_x">–</span> <span class="NLM_lpage">814</span><span class="refDoi"> DOI: 10.1002/anie.200804695</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1002%2Fanie.200804695" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=811-814&author=L.+M.+Salonenauthor=C.+Bucherauthor=D.+W.+Bannerauthor=W.+Haapauthor=J.-L.+Maryauthor=J.+Benzauthor=O.+Kusterauthor=P.+Seilerauthor=W.+B.+Schweizerauthor=F.+Diederich&title=Cation%E2%88%92%CF%80+Interactions+at+the+Active+Site+of+Factor+Xa%3A+Dramatic+Enhancement+upon+Stepwise+N-Alkylation+of+Ammonium+Ions&doi=10.1002%2Fanie.200804695"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fanie.200804695&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200804695%26sid%3Dliteratum%253Aachs%26aulast%3DSalonen%26aufirst%3DL.%2BM.%26aulast%3DBucher%26aufirst%3DC.%26aulast%3DBanner%26aufirst%3DD.%2BW.%26aulast%3DHaap%26aufirst%3DW.%26aulast%3DMary%26aufirst%3DJ.-L.%26aulast%3DBenz%26aufirst%3DJ.%26aulast%3DKuster%26aufirst%3DO.%26aulast%3DSeiler%26aufirst%3DP.%26aulast%3DSchweizer%26aufirst%3DW.%2BB.%26aulast%3DDiederich%26aufirst%3DF.%26atitle%3DCation%25E2%2588%2592%25CF%2580%2520Interactions%2520at%2520the%2520Active%2520Site%2520of%2520Factor%2520Xa%253A%2520Dramatic%2520Enhancement%2520upon%2520Stepwise%2520N-Alkylation%2520of%2520Ammonium%2520Ions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2009%26volume%3D48%26spage%3D811%26epage%3D814%26doi%3D10.1002%2Fanie.200804695" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Böhm, H.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flohr, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span> </span><span class="NLM_article-title">Scaffold Hopping</span> <span class="citation_source-journal">Drug Discovery Today: Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">217</span><span class="NLM_x">–</span> <span class="NLM_lpage">224</span><span class="refDoi"> DOI: 10.1016/j.ddtec.2004.10.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1016%2Fj.ddtec.2004.10.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=24981488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A280%3ADC%252BC2cfosVaktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2004&pages=217-224&author=H.-J.+B%C3%B6hmauthor=A.+Flohrauthor=M.+Stahl&title=Scaffold+Hopping&doi=10.1016%2Fj.ddtec.2004.10.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Scaffold hopping</span></div><div class="casAuthors">Bohm Hans-Joachim; Flohr Alexander; Stahl Martin</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">217-24</span>
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    </div><div class="casAbstract">The aim of scaffold hopping is to discover structurally novel compounds starting from known active compounds by modifying the central core structure of the molecule.  Scaffold hopping is a central task of modern medicinal chemistry requiring a multitude of techniques, which are discussed in this article.  Their application has led to several molecules with chemically completely different core structures, and yet binding to the same receptor.  Computational approaches for scaffold hopping highlight the challenges of the field that are still unsolved.:</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSkmF4guWxLWsKL0FlyGHDofW6udTcc2eb2lpHWBpVVr7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cfosVaktQ%253D%253D&md5=9a90f37dd2accf66bfd379f9ec602e03</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2004.10.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2004.10.009%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25B6hm%26aufirst%3DH.-J.%26aulast%3DFlohr%26aufirst%3DA.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DScaffold%2520Hopping%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2004%26volume%3D1%26spage%3D217%26epage%3D224%26doi%3D10.1016%2Fj.ddtec.2004.10.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Sun, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tawa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallqvist, A.</span><span> </span><span class="NLM_article-title">Classification of Scaffold-Hopping Approaches</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">310</span><span class="NLM_x">–</span> <span class="NLM_lpage">324</span><span class="refDoi"> DOI: 10.1016/j.drudis.2011.10.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1016%2Fj.drudis.2011.10.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=22056715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFyrt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=310-324&author=H.+Sunauthor=G.+Tawaauthor=A.+Wallqvist&title=Classification+of+Scaffold-Hopping+Approaches&doi=10.1016%2Fj.drudis.2011.10.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of scaffold-hopping approaches</span></div><div class="casAuthors">Sun, Hongmao; Tawa, Gregory; Wallqvist, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">7-8</span>),
    <span class="NLM_cas:pages">310-324</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The general goal of drug discovery is to identify novel compds. that are active against a preselected biol. target with acceptable pharmacol. properties defined by marketed drugs.  Scaffold hopping has been widely applied by medicinal chemists to discover equipotent compds. with novel backbones that have improved properties.  In this article we classify scaffold hopping into four major categories, namely heterocycle replacements, ring opening or closure, peptidomimetics and topol.-based hopping.  We review the structural diversity of original and final scaffolds with respect to each category.  We discuss the advantages and limitations of small, medium and large-step scaffold hopping.  Finally, we summarize software that is frequently used to facilitate different kinds of scaffold-hopping methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAONaJ8Be0x7Vg90H21EOLACvtfcHk0lhAfpdkN3GdoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFyrt7k%253D&md5=5f1143cc255c81b88ea4f83ac9f8efc6</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.10.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.10.024%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DH.%26aulast%3DTawa%26aufirst%3DG.%26aulast%3DWallqvist%26aufirst%3DA.%26atitle%3DClassification%2520of%2520Scaffold-Hopping%2520Approaches%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2012%26volume%3D17%26spage%3D310%26epage%3D324%26doi%3D10.1016%2Fj.drudis.2011.10.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Maass, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulz-Gasch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span> </span><span class="NLM_article-title">Recore: A Fast and Versatile Method for Scaffold Hopping Based on Small Molecule Crystal Structure Conformations</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">390</span><span class="NLM_x">–</span> <span class="NLM_lpage">399</span><span class="refDoi"> DOI: 10.1021/ci060094h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci060094h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhvVWjtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=390-399&author=P.+Maassauthor=T.+Schulz-Gaschauthor=M.+Stahlauthor=M.+Rarey&title=Recore%3A+A+Fast+and+Versatile+Method+for+Scaffold+Hopping+Based+on+Small+Molecule+Crystal+Structure+Conformations&doi=10.1021%2Fci060094h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Recore: A Fast and Versatile Method for Scaffold Hopping Based on Small Molecule Crystal Structure Conformations</span></div><div class="casAuthors">Maass, Patrick; Schulz-Gasch, Tanja; Stahl, Martin; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">390-399</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Replacing central elements of known active structures is a common procedure to enter new compd. classes.  Different computational methods have already been developed to help with this task, varying in the description of possible replacements, the query input, and the similarity measure used.  In this paper, a novel approach for scaffold replacement and a corresponding software tool, called Recore, is introduced.  In contrast to prior methods, the authors main objective was to combine the following three properties in one tool: to avoid structures with strained conformations, to enable the exploration of large search spaces, and to allow interactive use through short response times.  The authors introduce a new technique employing 3D fragments generated by combinatorial enumeration of cuts.  It allows focusing on fragments suitable for scaffold replacement while retaining conformational information of the corresponding crystal structures.  Based on this idea, the authors present an algorithm utilizing a geometric rank searching approach.  Given a geometric arrangement of two or three exit vectors and addnl. pharmacophore features, the algorithm finds fragments fulfilling all these constraints ordered by increasing deviation from the query constraints.  For the validation of the approach, three different drug design scenarios have been used.  The results obtained show that the authors approach is able to propose new valid scaffold topologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpphxap_w-jZbVg90H21EOLACvtfcHk0lhAfpdkN3GdoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhvVWjtbk%253D&md5=e9d4e277da49f4ffe2c21472990c2387</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fci060094h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci060094h%26sid%3Dliteratum%253Aachs%26aulast%3DMaass%26aufirst%3DP.%26aulast%3DSchulz-Gasch%26aufirst%3DT.%26aulast%3DStahl%26aufirst%3DM.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DRecore%253A%2520A%2520Fast%2520and%2520Versatile%2520Method%2520for%2520Scaffold%2520Hopping%2520Based%2520on%2520Small%2520Molecule%2520Crystal%2520Structure%2520Conformations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D390%26epage%3D399%26doi%3D10.1021%2Fci060094h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Lauri, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartlett, P. A.</span><span> </span><span class="NLM_article-title">CAVEAT: A Program to Facilitate the Design of Organic Molecules</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">51</span><span class="NLM_x">–</span> <span class="NLM_lpage">66</span><span class="refDoi"> DOI: 10.1007/BF00124349</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1007%2FBF00124349" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1994&pages=51-66&author=G.+Lauriauthor=P.+A.+Bartlett&title=CAVEAT%3A+A+Program+to+Facilitate+the+Design+of+Organic+Molecules&doi=10.1007%2FBF00124349"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1007%2FBF00124349&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00124349%26sid%3Dliteratum%253Aachs%26aulast%3DLauri%26aufirst%3DG.%26aulast%3DBartlett%26aufirst%3DP.%2BA.%26atitle%3DCAVEAT%253A%2520A%2520Program%2520to%2520Facilitate%2520the%2520Design%2520of%2520Organic%2520Molecules%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1994%26volume%3D8%26spage%3D51%26epage%3D66%26doi%3D10.1007%2FBF00124349" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Patel, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shuai, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinges, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pliushchev, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monzon, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagaw, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engstrom, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerdesky, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Longenecker, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Judge, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qin, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imade, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stolarik, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beno, D. W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brune, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chovan, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sham, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Link, J. T.</span><span> </span><span class="NLM_article-title">Discovery of Adamantane Ethers as Inhibitors of 11β-HSD-1: Synthesis and Biological Evaluation</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">750</span><span class="NLM_x">–</span> <span class="NLM_lpage">755</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2006.10.074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1016%2Fj.bmcl.2006.10.074" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=750-755&author=J.+R.+Patelauthor=Q.+Shuaiauthor=J.+Dingesauthor=M.+Winnauthor=M.+Pliushchevauthor=S.+Fungauthor=K.+Monzonauthor=W.+Chiouauthor=J.+Wangauthor=L.+Panauthor=S.+Wagawauthor=K.+Engstromauthor=F.+A.+Kerdeskyauthor=K.+Longeneckerauthor=R.+Judgeauthor=W.+Qinauthor=H.+M.+Imadeauthor=D.+Stolarikauthor=D.+W.+A.+Benoauthor=M.+Bruneauthor=L.+E.+Chovanauthor=H.+L.+Shamauthor=P.+Jacobsonauthor=J.+T.+Link&title=Discovery+of+Adamantane+Ethers+as+Inhibitors+of+11%CE%B2-HSD-1%3A+Synthesis+and+Biological+Evaluation&doi=10.1016%2Fj.bmcl.2006.10.074"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2006.10.074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2006.10.074%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DJ.%2BR.%26aulast%3DShuai%26aufirst%3DQ.%26aulast%3DDinges%26aufirst%3DJ.%26aulast%3DWinn%26aufirst%3DM.%26aulast%3DPliushchev%26aufirst%3DM.%26aulast%3DFung%26aufirst%3DS.%26aulast%3DMonzon%26aufirst%3DK.%26aulast%3DChiou%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DPan%26aufirst%3DL.%26aulast%3DWagaw%26aufirst%3DS.%26aulast%3DEngstrom%26aufirst%3DK.%26aulast%3DKerdesky%26aufirst%3DF.%2BA.%26aulast%3DLongenecker%26aufirst%3DK.%26aulast%3DJudge%26aufirst%3DR.%26aulast%3DQin%26aufirst%3DW.%26aulast%3DImade%26aufirst%3DH.%2BM.%26aulast%3DStolarik%26aufirst%3DD.%26aulast%3DBeno%26aufirst%3DD.%2BW.%2BA.%26aulast%3DBrune%26aufirst%3DM.%26aulast%3DChovan%26aufirst%3DL.%2BE.%26aulast%3DSham%26aufirst%3DH.%2BL.%26aulast%3DJacobson%26aufirst%3DP.%26aulast%3DLink%26aufirst%3DJ.%2BT.%26atitle%3DDiscovery%2520of%2520Adamantane%2520Ethers%2520as%2520Inhibitors%2520of%252011%25CE%25B2-HSD-1%253A%2520Synthesis%2520and%2520Biological%2520Evaluation%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D750%26epage%3D755%26doi%3D10.1016%2Fj.bmcl.2006.10.074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Tu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powers, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ursu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudom, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meininger, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeGraffenreid, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaen, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labelle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamamoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, N. P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span> </span><span class="NLM_article-title">Distinctive Molecular Inhibition Mechanisms for Selective Inhibitors of Human 11β-Hydroxysteroid Dehydrogenase Type 1</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">8922</span><span class="NLM_x">–</span> <span class="NLM_lpage">8931</span><span class="refDoi"> DOI: 10.1016/j.bmc.2008.08.065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1016%2Fj.bmc.2008.08.065" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=8922-8931&author=H.+Tuauthor=J.+P.+Powersauthor=J.+Liuauthor=S.+Ursuauthor=A.+Sudomauthor=X.+Yanauthor=H.+Xuauthor=D.+Meiningerauthor=M.+DeGraffenreidauthor=X.+Heauthor=J.+C.+Jaenauthor=D.+Sunauthor=M.+Labelleauthor=H.+Yamamotoauthor=B.+Shanauthor=N.+P.+C.+Walkerauthor=Z.+Wang&title=Distinctive+Molecular+Inhibition+Mechanisms+for+Selective+Inhibitors+of+Human+11%CE%B2-Hydroxysteroid+Dehydrogenase+Type+1&doi=10.1016%2Fj.bmc.2008.08.065"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2008.08.065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2008.08.065%26sid%3Dliteratum%253Aachs%26aulast%3DTu%26aufirst%3DH.%26aulast%3DPowers%26aufirst%3DJ.%2BP.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DUrsu%26aufirst%3DS.%26aulast%3DSudom%26aufirst%3DA.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DMeininger%26aufirst%3DD.%26aulast%3DDeGraffenreid%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DX.%26aulast%3DJaen%26aufirst%3DJ.%2BC.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DLabelle%26aufirst%3DM.%26aulast%3DYamamoto%26aufirst%3DH.%26aulast%3DShan%26aufirst%3DB.%26aulast%3DWalker%26aufirst%3DN.%2BP.%2BC.%26aulast%3DWang%26aufirst%3DZ.%26atitle%3DDistinctive%2520Molecular%2520Inhibition%2520Mechanisms%2520for%2520Selective%2520Inhibitors%2520of%2520Human%252011%25CE%25B2-Hydroxysteroid%2520Dehydrogenase%2520Type%25201%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2008%26volume%3D16%26spage%3D8922%26epage%3D8931%26doi%3D10.1016%2Fj.bmc.2008.08.065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Furet, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caravatti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guagnano, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schoepfer, J.</span><span> </span><span class="NLM_article-title">Entry into a New Class of Protein Kinase Inhibitors by Pseudo Ring Design</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">897</span><span class="NLM_x">–</span> <span class="NLM_lpage">900</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2007.12.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1016%2Fj.bmcl.2007.12.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=18248988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFWksb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=897-900&author=P.+Furetauthor=G.+Caravattiauthor=V.+Guagnanoauthor=M.+Langauthor=T.+Meyerauthor=J.+Schoepfer&title=Entry+into+a+New+Class+of+Protein+Kinase+Inhibitors+by+Pseudo+Ring+Design&doi=10.1016%2Fj.bmcl.2007.12.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Entry into a new class of protein kinase inhibitors by pseudo ring design</span></div><div class="casAuthors">Furet, Pascal; Caravatti, Giorgio; Guagnano, Vito; Lang, Marc; Meyer, Thomas; Schoepfer, Joseph</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">897-900</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A pyrimidin-4-yl-urea motif forming a pseudo ring by intramol. hydrogen bonding has been designed to mimic the pyrido[2,3-d]pyrimidin-7-one core structure of a well-established class of protein kinase inhibitors.  Potent inhibition of a no. of protein kinases was obtained with the first prototype compd. synthesized to probe the design concept.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5MZZKS4I03LVg90H21EOLACvtfcHk0liUQ_KFX8h4Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFWksb0%253D&md5=989b4bb7f1d5177be45d4edf31290de5</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.12.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.12.041%26sid%3Dliteratum%253Aachs%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DCaravatti%26aufirst%3DG.%26aulast%3DGuagnano%26aufirst%3DV.%26aulast%3DLang%26aufirst%3DM.%26aulast%3DMeyer%26aufirst%3DT.%26aulast%3DSchoepfer%26aufirst%3DJ.%26atitle%3DEntry%2520into%2520a%2520New%2520Class%2520of%2520Protein%2520Kinase%2520Inhibitors%2520by%2520Pseudo%2520Ring%2520Design%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D897%26epage%3D900%26doi%3D10.1016%2Fj.bmcl.2007.12.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit6'],'ref6':['cit5'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 16 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Robin Taylor, <span class="NLM_string-name hlFld-ContribAuthor">Peter A. Wood</span>. </span><span class="cited-content_cbyCitation_article-title">A Million Crystal Structures: The Whole Is Greater than the Sum of Its Parts. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2019,</strong> <em>119 </em>
                                    (16)
                                     , 9427-9477. <a href="https://doi.org/10.1021/acs.chemrev.9b00155" title="DOI URL">https://doi.org/10.1021/acs.chemrev.9b00155</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.9b00155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.9b00155%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DA%252BMillion%252BCrystal%252BStructures%25253A%252BThe%252BWhole%252BIs%252BGreater%252Bthan%252Bthe%252BSum%252Bof%252BIts%252BParts%26aulast%3DTaylor%26aufirst%3DRobin%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D07032019%26date%3D17062019%26volume%3D119%26issue%3D16%26spage%3D9427%26epage%3D9477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Maxwell D. Cummings, <span class="NLM_string-name hlFld-ContribAuthor">Sivakumar Sekharan</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Macrocycle Design in Small-Molecule Drug Discovery and Simple Metrics To Identify Opportunities for Macrocyclization of Small-Molecule Ligands. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (15)
                                     , 6843-6853. <a href="https://doi.org/10.1021/acs.jmedchem.8b01985" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01985</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01985%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BMacrocycle%252BDesign%252Bin%252BSmall-Molecule%252BDrug%252BDiscovery%252Band%252BSimple%252BMetrics%252BTo%252BIdentify%252BOpportunities%252Bfor%252BMacrocyclization%252Bof%252BSmall-Molecule%252BLigands%26aulast%3DCummings%26aufirst%3DMaxwell%2BD.%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D18122018%26date%3D22032019%26date%3D12032019%26volume%3D62%26issue%3D15%26spage%3D6843%26epage%3D6853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fabrizio Giordanetto, Chentian Jin, Lindsay Willmore, Miklos Feher, <span class="NLM_string-name hlFld-ContribAuthor">David E. Shaw</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment Hits: What do They Look Like and How do They Bind?. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (7)
                                     , 3381-3394. <a href="https://doi.org/10.1021/acs.jmedchem.8b01855" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01855</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01855%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFragment%252BHits%25253A%252BWhat%252Bdo%252BThey%252BLook%252BLike%252Band%252BHow%252Bdo%252BThey%252BBind%25253F%26aulast%3DGiordanetto%26aufirst%3DFabrizio%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D27112018%26date%3D01042019%26date%3D15032019%26volume%3D62%26issue%3D7%26spage%3D3381%26epage%3D3394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mahendra  Awale</span>, <span class="hlFld-ContribAuthor ">Daniel  Probst</span>, and <span class="hlFld-ContribAuthor ">Jean-Louis  Reymond</span>  . </span><span class="cited-content_cbyCitation_article-title">WebMolCS: A Web-Based Interface for Visualizing Molecules in Three-Dimensional Chemical Spaces. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (4)
                                     , 643-649. <a href="https://doi.org/10.1021/acs.jcim.6b00690" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00690</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00690&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00690%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DWebMolCS%25253A%252BA%252BWeb-Based%252BInterface%252Bfor%252BVisualizing%252BMolecules%252Bin%252BThree-Dimensional%252BChemical%252BSpaces%26aulast%3DAwale%26aufirst%3DMahendra%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D11112016%26date%3D30032017%26date%3D24042017%26date%3D19032017%26volume%3D57%26issue%3D4%26spage%3D643%26epage%3D649" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Therese  Inhester</span>, <span class="hlFld-ContribAuthor ">Stefan  Bietz</span>, <span class="hlFld-ContribAuthor ">Matthias  Hilbig</span>, <span class="hlFld-ContribAuthor ">Robert  Schmidt</span>, and <span class="hlFld-ContribAuthor ">Matthias  Rarey</span>  . </span><span class="cited-content_cbyCitation_article-title">Index-Based Searching of Interaction Patterns in Large Collections of Protein–Ligand Interfaces. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (2)
                                     , 148-158. <a href="https://doi.org/10.1021/acs.jcim.6b00561" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00561</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00561%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DIndex-Based%252BSearching%252Bof%252BInteraction%252BPatterns%252Bin%252BLarge%252BCollections%252Bof%252BProtein%2525E2%252580%252593Ligand%252BInterfaces%26aulast%3DInhester%26aufirst%3DTherese%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D16092016%26date%3D27012017%26date%3D27022017%26volume%3D57%26issue%3D2%26spage%3D148%26epage%3D158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michail N.  Elinson</span>, <span class="hlFld-ContribAuthor ">Yuliya E.  Ryzhkova</span>, <span class="hlFld-ContribAuthor ">Anatoly N.  Vereshchagin</span>, <span class="hlFld-ContribAuthor ">Fedor V.  Ryzhkov</span>, <span class="hlFld-ContribAuthor ">Mikhail P.  Egorov</span>. </span><span class="cited-content_cbyCitation_article-title">Electrocatalytic multicomponent one‐pot approach to tetrahydro‐2′
              H
              ,
              
                4
                H
              
              ‐spiro[benzofuran‐2,5′‐pyrimidine] scaffold. </span><span class="cited-content_cbyCitation_journal-name">Journal of Heterocyclic Chemistry</span><span> <strong>2021,</strong> <em>58 </em>
                                    (7)
                                     , 1484-1495. <a href="https://doi.org/10.1002/jhet.4274" title="DOI URL">https://doi.org/10.1002/jhet.4274</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jhet.4274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjhet.4274%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Heterocyclic%2520Chemistry%26atitle%3DElectrocatalytic%252Bmulticomponent%252Bone%2525E2%252580%252590pot%252Bapproach%252Bto%252Btetrahydro%2525E2%252580%2525902%2525E2%252580%2525B2%252BH%252B%25252C%252B4%252BH%252B%2525E2%252580%252590spiro%25255Bbenzofuran%2525E2%252580%2525902%25252C5%2525E2%252580%2525B2%2525E2%252580%252590pyrimidine%25255D%252Bscaffold%26aulast%3DElinson%26aufirst%3DMichail%2BN.%26date%3D2021%26date%3D2021%26volume%3D58%26issue%3D7%26spage%3D1484%26epage%3D1495" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Konrad  Diedrich</span>, <span class="hlFld-ContribAuthor ">Joel  Graef</span>, <span class="hlFld-ContribAuthor ">Katrin  Schöning-Stierand</span>, <span class="hlFld-ContribAuthor ">Matthias  Rarey</span>, . </span><span class="cited-content_cbyCitation_article-title">GeoMine: interactive pattern mining of protein–ligand interfaces in the Protein Data Bank. </span><span class="cited-content_cbyCitation_journal-name">Bioinformatics</span><span> <strong>2021,</strong> <em>37 </em>
                                    (3)
                                     , 424-425. <a href="https://doi.org/10.1093/bioinformatics/btaa693" title="DOI URL">https://doi.org/10.1093/bioinformatics/btaa693</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bioinformatics/btaa693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbioinformatics%2Fbtaa693%26sid%3Dliteratum%253Aachs%26jtitle%3DBioinformatics%26atitle%3DGeoMine%25253A%252Binteractive%252Bpattern%252Bmining%252Bof%252Bprotein%2525E2%252580%252593ligand%252Binterfaces%252Bin%252Bthe%252BProtein%252BData%252BBank%26aulast%3DDiedrich%26aufirst%3DKonrad%26date%3D2021%26date%3D2020%26volume%3D37%26issue%3D3%26spage%3D424%26epage%3D425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Markella  Konstantinidou</span>, <span class="hlFld-ContribAuthor ">Zlata  Boiarska</span>, <span class="hlFld-ContribAuthor ">Roberto  Butera</span>, <span class="hlFld-ContribAuthor ">Constantinos G.  Neochoritis</span>, <span class="hlFld-ContribAuthor ">Katarzyna  Kurpiewska</span>, <span class="hlFld-ContribAuthor ">Justyna  Kalinowska‐Tłuscik</span>, <span class="hlFld-ContribAuthor ">Alexander  Dömling</span>. </span><span class="cited-content_cbyCitation_article-title">Diaminoimidazopyrimidines: Access via the Groebke–Blackburn–Bienaymé Reaction and Structural Data Mining. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Organic Chemistry</span><span> <strong>2020,</strong> <em>2020 </em>
                                    (34)
                                     , 5601-5605. <a href="https://doi.org/10.1002/ejoc.202000933" title="DOI URL">https://doi.org/10.1002/ejoc.202000933</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ejoc.202000933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fejoc.202000933%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DDiaminoimidazopyrimidines%25253A%252BAccess%252Bvia%252Bthe%252BGroebke%2525E2%252580%252593Blackburn%2525E2%252580%252593Bienaym%2525C3%2525A9%252BReaction%252Band%252BStructural%252BData%252BMining%26aulast%3DKonstantinidou%26aufirst%3DMarkella%26date%3D2020%26date%3D2020%26volume%3D2020%26issue%3D34%26spage%3D5601%26epage%3D5605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Simon J.  Coles</span>, <span class="hlFld-ContribAuthor ">David R.  Allan</span>, <span class="hlFld-ContribAuthor ">Christine M.  Beavers</span>, <span class="hlFld-ContribAuthor ">Simon J.  Teat</span>, <span class="hlFld-ContribAuthor ">Stephen J. W.  Holgate</span>, <span class="hlFld-ContribAuthor ">Clare A.  Tovee</span>. </span><span class="cited-content_cbyCitation_article-title">Leading Edge Chemical Crystallography Service Provision and Its Impact on Crystallographic Data Science in the Twenty-First Century. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 69-140. <a href="https://doi.org/10.1007/430_2020_63" title="DOI URL">https://doi.org/10.1007/430_2020_63</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/430_2020_63&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F430_2020_63%26sid%3Dliteratum%253Aachs%26atitle%3DLeading%252BEdge%252BChemical%252BCrystallography%252BService%252BProvision%252Band%252BIts%252BImpact%252Bon%252BCrystallographic%252BData%252BScience%252Bin%252Bthe%252BTwenty-First%252BCentury%26aulast%3DColes%26aufirst%3DSimon%2BJ.%26date%3D2020%26date%3D2020%26spage%3D69%26epage%3D140%26pub%3DSpringer%2520International%2520Publishing%26atitle%3D21st%252BCentury%252BChallenges%252Bin%252BChemical%252BCrystallography%252BI%26aulast%3DMingos%26aufirst%3DD.%2BMichael%2BP.%26date%3D2020%26volume%3D185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joseph  Mitchell</span>, <span class="hlFld-ContribAuthor ">Jennifer H.  Robertson</span>, <span class="hlFld-ContribAuthor ">Paul R.  Raithby</span>. </span><span class="cited-content_cbyCitation_article-title">Cambridge Structural Database. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,<a href="https://doi.org/10.1016/B978-0-12-409547-2.14829-2" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.14829-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.14829-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.14829-2%26sid%3Dliteratum%253Aachs%26atitle%3DCambridge%252BStructural%252BDatabase%26aulast%3DMitchell%26aufirst%3DJoseph%26date%3D2020%26pub%3DElsevier%26atitle%3DReference%252BModule%252Bin%252BChemistry%25252C%252BMolecular%252BSciences%252Band%252BChemical%252BEngineering%26date%3D2020%26volume%3D332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jason C.  Cole</span>, <span class="hlFld-ContribAuthor ">Seth  Wiggin</span>, <span class="hlFld-ContribAuthor ">Francesca  Stanzione</span>. </span><span class="cited-content_cbyCitation_article-title">New insights and innovation from a million crystal structures in the Cambridge Structural Database. </span><span class="cited-content_cbyCitation_journal-name">Structural Dynamics</span><span> <strong>2019,</strong> <em>6 </em>
                                    (5)
                                     , 054301. <a href="https://doi.org/10.1063/1.5116878" title="DOI URL">https://doi.org/10.1063/1.5116878</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1063/1.5116878&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1063%2F1.5116878%26sid%3Dliteratum%253Aachs%26jtitle%3DStructural%2520Dynamics%26atitle%3DNew%252Binsights%252Band%252Binnovation%252Bfrom%252Ba%252Bmillion%252Bcrystal%252Bstructures%252Bin%252Bthe%252BCambridge%252BStructural%252BDatabase%26aulast%3DCole%26aufirst%3DJason%2BC.%26date%3D2019%26volume%3D6%26issue%3D5%26spage%3D054301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yajun  Zheng</span>, <span class="hlFld-ContribAuthor ">Colin M.  Tice</span>, <span class="hlFld-ContribAuthor ">Suresh B.  Singh</span>. </span><span class="cited-content_cbyCitation_article-title">Conformational control in structure-based drug design. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>27 </em>
                                    (13)
                                     , 2825-2837. <a href="https://doi.org/10.1016/j.bmcl.2017.04.079" title="DOI URL">https://doi.org/10.1016/j.bmcl.2017.04.079</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2017.04.079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2017.04.079%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DConformational%252Bcontrol%252Bin%252Bstructure-based%252Bdrug%252Bdesign%26aulast%3DZheng%26aufirst%3DYajun%26date%3D2017%26volume%3D27%26issue%3D13%26spage%3D2825%26epage%3D2837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Upadhyayula Surya  Raghavender</span>. </span><span class="cited-content_cbyCitation_article-title">Analysis of residue conformations in peptides in Cambridge structural database and protein-peptide structural complexes. </span><span class="cited-content_cbyCitation_journal-name">Chemical Biology & Drug Design</span><span> <strong>2017,</strong> <em>89 </em>
                                    (3)
                                     , 428-442. <a href="https://doi.org/10.1111/cbdd.12862" title="DOI URL">https://doi.org/10.1111/cbdd.12862</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cbdd.12862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcbdd.12862%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Biology%2520%2526%2520Drug%2520Design%26atitle%3DAnalysis%252Bof%252Bresidue%252Bconformations%252Bin%252Bpeptides%252Bin%252BCambridge%252Bstructural%252Bdatabase%252Band%252Bprotein-peptide%252Bstructural%252Bcomplexes%26aulast%3DRaghavender%26aufirst%3DUpadhyayula%2BSurya%26date%3D2017%26date%3D2016%26volume%3D89%26issue%3D3%26spage%3D428%26epage%3D442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maude  Giroud</span>, <span class="hlFld-ContribAuthor ">Jakov  Ivkovic</span>, <span class="hlFld-ContribAuthor ">Mara  Martignoni</span>, <span class="hlFld-ContribAuthor ">Marianne  Fleuti</span>, <span class="hlFld-ContribAuthor ">Nils  Trapp</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Haap</span>, <span class="hlFld-ContribAuthor ">Andreas  Kuglstatter</span>, <span class="hlFld-ContribAuthor ">Jörg  Benz</span>, <span class="hlFld-ContribAuthor ">Bernd  Kuhn</span>, <span class="hlFld-ContribAuthor ">Tanja  Schirmeister</span>, <span class="hlFld-ContribAuthor ">François  Diederich</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibition of the Cysteine Protease Human Cathepsin L by Triazine Nitriles: Amide⋅⋅⋅Heteroarene π-Stacking Interactions and Chalcogen Bonding in the S3 Pocket. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2017,</strong> <em>12 </em>
                                    (3)
                                     , 257-270. <a href="https://doi.org/10.1002/cmdc.201600563" title="DOI URL">https://doi.org/10.1002/cmdc.201600563</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201600563&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201600563%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DInhibition%252Bof%252Bthe%252BCysteine%252BProtease%252BHuman%252BCathepsin%2525E2%252580%252585L%252Bby%252BTriazine%252BNitriles%25253A%252BAmide%2525E2%25258B%252585%2525E2%25258B%252585%2525E2%25258B%252585Heteroarene%252B%2525CF%252580-Stacking%252BInteractions%252Band%252BChalcogen%252BBonding%252Bin%252Bthe%252BS3%252BPocket%26aulast%3DGiroud%26aufirst%3DMaude%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D3%26spage%3D257%26epage%3D270" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">R.A.  Lewis</span>, <span class="hlFld-ContribAuthor ">F.  Sirockin</span>. </span><span class="cited-content_cbyCitation_article-title">2/3D Pharmacophore Definitions and Their Application. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 189-220. <a href="https://doi.org/10.1016/B978-0-12-409547-2.12346-7" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.12346-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.12346-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.12346-7%26sid%3Dliteratum%253Aachs%26atitle%3D2%25252F3D%252BPharmacophore%252BDefinitions%252Band%252BTheir%252BApplication%26aulast%3DLewis%26aufirst%3DR.A.%26date%3D2017%26spage%3D189%26epage%3D220%26pub%3DElsevier%26atitle%3DComprehensive%252BMedicinal%252BChemistry%252BIII%26date%3D2017%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mahendra  Awale</span>, <span class="hlFld-ContribAuthor ">Jean-Louis  Reymond</span>. </span><span class="cited-content_cbyCitation_article-title">Web-based 3D-visualization of the DrugBank chemical space. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2016,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-016-0138-2" title="DOI URL">https://doi.org/10.1186/s13321-016-0138-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-016-0138-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-016-0138-2%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DWeb-based%252B3D-visualization%252Bof%252Bthe%252BDrugBank%252Bchemical%252Bspace%26aulast%3DAwale%26aufirst%3DMahendra%26date%3D2016%26date%3D2016%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (a) Base features are shown as wireframe spheres, while virtual points defining directionality are shown as solid spheres. A nondirectional acceptor is shown in red, a directional donor in blue, and an exit vector in white. (b) A pharmacophore query defining two directional planar ring features. In the top pharmacophore no constraints are applied, so in a hit the ring systems can be in the same or in different molecules. Intermolecular and intramolecular constraints (dashed lines) are applied in the middle and bottom pharmacophore, respectively, so only hits where the rings are in different molecules (middle) or in the same molecule are retrieved (bottom). (c) Examples of assigning a small molecule (S) or protein (P) constraint to a pharmacophore defining two directional planar ring features. The small molecule-only example translates to, for example, ligand-cofactor searches in the PDB or small molecule interaction searches in the CSD. The protein–ligand query allows for protein–ligand interaction searches in the PDB, and with a protein–protein query, interactions within protein components can be analyzed.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Example of a hydrogen bonding pharmacophore query (directional donors and acceptors are shown in blue and red, respectively) overlaid with CSD entry DOPLIF. Light green atoms represent nonterminal R-groups which are not inside the bounding-sphere fitted around the pharmacophore. (b) 2D representation of the pharmacophore and the small molecule hit shown in (a).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Overview of the overall query workflow. Starting from a pharmacophore query (top left), the feature database residing in memory is filtered to only return the database entries that contain the feature types used in the pharmacophore. These database entries are then split up across the number of CPU cores assigned to perform the search. For each database entry, first the 3D search is performed to identify matching structures only using the feature points stored in memory. For each match, the corresponding molecular structures are then fetched from disk. The resulting hits are overlaid onto the pharmacophore query as soon as they are available. On the basis of the types of hits retrieved and presented in the 3D and 2D displays, a user may decide to modify the query by changing the geometry of the pharmacophore (sphere positions and radii) or by adding/deleting features. Every time the query is modified, a new search is started initiating the next iteration of the interactive feedback cycle.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Left: S3 pocket of human cathepsin L where protein residues used to define the CSD-CrossMiner pharmacophore are highlighted as green sticks (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2xu1">2xu1</a>). The pyrrolidine inhibitor scaffold is shown in cyan, without the chlorophenyl fragment of the S3 pocket. Right: CSD-CrossMiner pharmacophore model with directional hydrogen bond donor (blue) and acceptor (red) as well as hydrophobic (green) features for the protein. To avoid restricting the choice of ligand motifs, the “heavy_atom” feature (lig_heavy, orange) is used for the small molecule component. Intramolecular constraints are set between the prot_acc and prot_don features derived from the amide and between the two lig_heavy features.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Examples of PDB solutions for the pharmacophore query defined in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>. All three binding sites appear in the top 25 of the hit list after clustering with a Tanimoto threshold of 0.7 and removal of all cathepsin L, K, and S structures. The ligand is shown in magenta, while protein residues are colored in gray.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Top: CSD-CrossMiner pharmacophore query with three “planar_ring_projected” features (green) in the protein and one small molecule “heavy_atom” feature (orange) as derived from the factor Xa complex structure 2w26. Bottom: Selected PDB solutions from the top 25 hit list after clustering with a Tanimoto threshold of 0.7 and removal of all factor Xa structures: 4cnl (choline-binding protein L from streptococcus pneumonia), 4h75 (human spindlin 1), 4aot (human lok kinase). Ligand and protein are colored in magenta and gray, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Top: CSD-CrossMiner pharmacophore query with four “donor_projected” (blue) and two “acceptor_projected” (red) features representing a protein β sheet as well as a small molecule “heavy_atom” feature (orange). Intramolecular constraints were set for all features within each of the two β strands. The p38 MAP kinase structure 3roc is used as reference. Bottom: Substructures found in more than 100 matches of the pharmacophore query together with PDB codes of representative complexes. The orange circle highlights the atom occupying the ligand “heavy_atom” feature. “A” stands for carbon or nitrogen atoms.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. CSD-CrossMiner pharmacophore model for 11β-HSD1 scaffold hopping with “donor_projected” (blue) and “acceptor_projected” (red) as well as “exit_vector” (white) features. The cocrystal structure with PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2irw">2irw</a> is taken as reference where protein residues Ser150 and Tyr163 are shown in gray and the inhibitor is shown in magenta. An intramolecular constraint is set between the acceptor and exit vector features derived from the ligand.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (a–c) Selected CSD solutions for the pharmacophore query defined in <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>. All three examples appear in the top 75 of the hit list after clustering with a Tanimoto threshold of 0.7. (d) Cutout of a 11β-HSD1 complex with a triazole inhibitor (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3d5q">3d5q</a>). The ligand is shown in magenta, and the protein residues are in gray.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. (a) Reference ligand taken from the ABL kinase complex structure 2hzi and CSD-CrossMiner small molecule pharmacophore model. Pharmacophore features are colored as green (“planar_ring_projected”), red (hydrogen bond “acceptor_projected”), and white (“exit_vector”). Intramolecular constraints were set between all features. (b–d) Examples of solutions within the top 100 of the hit list after clustering with a Tanimoto threshold of 0.7. Different flavors of pseudo-ring scaffold hopping using an intramolecular hydrogen bond are shown in (c) and (d).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/medium/jm-2015-01756e_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Results from a PDB search of the pharmacophore model of <a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a> augmented with two protein features (hydrogen bond donor and acceptor from the hinge backbone of reference structure 2hzi). Top: Core structure from 2hzi. Bottom: Other known kinase scaffolds that match the query: (a) 4ehe, braf, (b) 3tt0, fgfr1, (c) 4c3f, lck. Pharmacophore features are colored as green (“planar_ring_projected”), red (hydrogen bond “acceptor_projected”), blue (hydrogen bond “donor_accepted”), and white (“exit_vector”). Intramolecular constraints are set between all small molecule and protein features, respectively. Ligands are shown in magenta and protein residues in gray.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-9/acs.jmedchem.5b01756/20160506/images/large/jm-2015-01756e_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01756&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i21">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58588" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58588" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 46 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Allen, F. H.</span><span> </span><span class="NLM_article-title">The Cambridge Structural Database: A Quarter of a Million Crystal Structures and Rising</span> <span class="citation_source-journal">Acta Crystallogr., Sect. B: Struct. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">380</span><span class="NLM_x">–</span> <span class="NLM_lpage">388</span><span class="refDoi"> DOI: 10.1107/S0108768102003890</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1107%2FS0108768102003890" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=12037359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BD38XktVOqu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=380-388&author=F.+H.+Allen&title=The+Cambridge+Structural+Database%3A+A+Quarter+of+a+Million+Crystal+Structures+and+Rising&doi=10.1107%2FS0108768102003890"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Cambridge Structural Database: a quarter of a million crystal structures and rising</span></div><div class="casAuthors">Allen, Frank H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section B: Structural Science</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">B58</span>
        (<span class="NLM_cas:issue">3, No. 1</span>),
    <span class="NLM_cas:pages">380-388</span>CODEN:
                <span class="NLM_cas:coden">ASBSDK</span>;
        ISSN:<span class="NLM_cas:issn">0108-7681</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">The Cambridge Structural Database (CSD) now contains data for more than a quarter of a million small-mol. crystal structures.  The information content of the CSD, together with methods for data acquisition, processing and validation, are summarized, with particular emphasis on the chem. information added by CSD editors.  Nearly 80% of new structural data arrives electronically, mostly in CIF format, and the CCDC acts as the official crystal structure data depository for 51 major journals.  The CCDC now maintains both a CIF archive (more than 73000 CIFs dating from 1996), as well as the distributed binary CSD archive; the availability of data in both archives is discussed.  A statistical survey of the CSD is also presented and projections concerning future accession rates indicate that the CSD will contain at least 500000 crystal structures by the year 2010.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_Uk3GetkiibVg90H21EOLACvtfcHk0liHiCV_lDRyRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XktVOqu74%253D&md5=406cd0df6ea9035a0ebf8dd9eccbd1f8</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1107%2FS0108768102003890&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0108768102003890%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DF.%2BH.%26atitle%3DThe%2520Cambridge%2520Structural%2520Database%253A%2520A%2520Quarter%2520of%2520a%2520Million%2520Crystal%2520Structures%2520and%2520Rising%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520B%253A%2520Struct.%2520Sci.%26date%3D2002%26volume%3D58%26spage%3D380%26epage%3D388%26doi%3D10.1107%2FS0108768102003890" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battistuz, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bluhm, W. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burkhardt, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iype, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fagan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marvin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padilla, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ravichandran, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thanki, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zardecki, C.</span><span> </span><span class="NLM_article-title">The Protein Data Bank</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">899</span><span class="NLM_x">–</span> <span class="NLM_lpage">907</span><span class="refDoi"> DOI: 10.1107/S0907444902003451</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1107%2FS0907444902003451" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=12037327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BD38XktVKhtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=899-907&author=H.+M.+Bermanauthor=T.+Battistuzauthor=T.+N.+Bhatauthor=W.+F.+Bluhmauthor=P.+E.+Bourneauthor=K.+Burkhardtauthor=Z.+Fengauthor=G.+L.+Gillilandauthor=L.+Iypeauthor=S.+Jainauthor=P.+Faganauthor=J.+Marvinauthor=D.+Padillaauthor=V.+Ravichandranauthor=B.+Schneiderauthor=N.+Thankiauthor=H.+Weissigauthor=J.+D.+Westbrookauthor=C.+Zardecki&title=The+Protein+Data+Bank&doi=10.1107%2FS0907444902003451"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Battistuz, Tammy; Bhat, T. N.; Bluhm, Wolfgang F.; Bourne, Philip E.; Burkhardt, Kyle; Feng, Zukang; Gilliland, Gary L.; Iype, Lisa; Jain, Shri; Fagan, Phoebe; Marvin, Jessica; Padilla, David; Ravichandran, Veerasamy; Schneider, Bohdan; Thanki, Narmada; Weissig, Helge; Westbrook, John D.; Zardecki, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">6, No. 1</span>),
    <span class="NLM_cas:pages">899-907</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">The Protein Data Bank [PDB; Berman, Westbrook et al. (2000), Nucleic Acids Res. 28, 235-242; http://www.pdb.org/] is the single worldwide archive of primary structural data of biol. macromols.  Many secondary sources of information are derived from PDB data.  It is the starting point for studies in structural bioinformatics.  This article describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information and plans for the future development of the resource.  The reader should come away with an understanding of the scope of the PDB and what is provided by the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsUFcMyys-PrVg90H21EOLACvtfcHk0liHiCV_lDRyRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XktVKhtLg%253D&md5=d7594ad00ec8e5bacc413d38bceaa360</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1107%2FS0907444902003451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444902003451%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DBattistuz%26aufirst%3DT.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DBluhm%26aufirst%3DW.%2BF.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DBurkhardt%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%2BL.%26aulast%3DIype%26aufirst%3DL.%26aulast%3DJain%26aufirst%3DS.%26aulast%3DFagan%26aufirst%3DP.%26aulast%3DMarvin%26aufirst%3DJ.%26aulast%3DPadilla%26aufirst%3DD.%26aulast%3DRavichandran%26aufirst%3DV.%26aulast%3DSchneider%26aufirst%3DB.%26aulast%3DThanki%26aufirst%3DN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DWestbrook%26aufirst%3DJ.%2BD.%26aulast%3DZardecki%26aufirst%3DC.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2002%26volume%3D58%26spage%3D899%26epage%3D907%26doi%3D10.1107%2FS0907444902003451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Bernstein, F. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koetzle, T. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, G. J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, E. F.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brice, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodgers, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennard, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimanouchi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tasumi, M.</span><span> </span><span class="NLM_article-title">The Protein Data Bank: A Computer-Based Archival File for Macromolecular Structures</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">535</span><span class="NLM_x">–</span> <span class="NLM_lpage">542</span><span class="refDoi"> DOI: 10.1016/S0022-2836(77)80200-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1016%2FS0022-2836%2877%2980200-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=875032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADyaE2sXkvVert7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=1977&pages=535-542&author=F.+C.+Bernsteinauthor=T.+F.+Koetzleauthor=G.+J.+B.+Williamsauthor=E.+F.+Meyerauthor=M.+D.+Briceauthor=J.+R.+Rodgersauthor=O.+Kennardauthor=T.+Shimanouchiauthor=M.+Tasumi&title=The+Protein+Data+Bank%3A+A+Computer-Based+Archival+File+for+Macromolecular+Structures&doi=10.1016%2FS0022-2836%2877%2980200-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank:  a computer-based archival file for macromolecular structures</span></div><div class="casAuthors">Bernstein, Frances C.; Koetzle, Thomas F.; Williams, Graheme J. B.; Meyer, Edgar F., Jr.; Brice, Michael D.; Rodgers, John R.; Kennard, Olga; Shimanouchi, Takehiko; Tasumi, Mitsuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">535-42</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">The Protein Data Bank is a computer-based archival file for macromol. structures.  The Bank stores in a uniform format at. coordinates and partial bond connectivities, as derived from crystallog. studies.  Text included in each data entry gives pertinent information for the structure at hand (e.g., species from which the mol. was obtained, resoln. of diffraction data, literature citations, and specifications of secondary structure).  In addn., the Protein Data Bank stores structure factors and phases.  Input of data and general maintenance functions are at Brookhaven National Lab.  All data are available on magnetic tape for public distribution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9tv93dkDhxrVg90H21EOLACvtfcHk0lglTwSCPtrXMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXkvVert7o%253D&md5=07736506e495f157c600cf51a253a0a0</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2877%2980200-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252877%252980200-3%26sid%3Dliteratum%253Aachs%26aulast%3DBernstein%26aufirst%3DF.%2BC.%26aulast%3DKoetzle%26aufirst%3DT.%2BF.%26aulast%3DWilliams%26aufirst%3DG.%2BJ.%2BB.%26aulast%3DMeyer%26aufirst%3DE.%2BF.%26aulast%3DBrice%26aufirst%3DM.%2BD.%26aulast%3DRodgers%26aufirst%3DJ.%2BR.%26aulast%3DKennard%26aufirst%3DO.%26aulast%3DShimanouchi%26aufirst%3DT.%26aulast%3DTasumi%26aufirst%3DM.%26atitle%3DThe%2520Protein%2520Data%2520Bank%253A%2520A%2520Computer-Based%2520Archival%2520File%2520for%2520Macromolecular%2520Structures%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1977%26volume%3D112%26spage%3D535%26epage%3D542%26doi%3D10.1016%2FS0022-2836%2877%2980200-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Schärfer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulz-Gasch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ehrlich, H.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guba, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span> </span><span class="NLM_article-title">Torsion Angle Preferences in Druglike Chemical Space: A Comprehensive Guide</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2016</span><span class="NLM_x">–</span> <span class="NLM_lpage">2028</span><span class="refDoi"> DOI: 10.1021/jm3016816</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3016816" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A280%3ADC%252BC3szltlGitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2016-2028&author=C.+Sch%C3%A4rferauthor=T.+Schulz-Gaschauthor=H.-C.+Ehrlichauthor=W.+Gubaauthor=M.+Rareyauthor=M.+Stahl&title=Torsion+Angle+Preferences+in+Druglike+Chemical+Space%3A+A+Comprehensive+Guide&doi=10.1021%2Fjm3016816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Torsion angle preferences in druglike chemical space: a comprehensive guide</span></div><div class="casAuthors">Scharfer Christin; Schulz-Gasch Tanja; Ehrlich Hans-Christian; Guba Wolfgang; Rarey Matthias; Stahl Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2016-28</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Crystal structure databases offer ample opportunities to derive small molecule conformation preferences, but the derived knowledge is not systematically applied in drug discovery research.  We address this gap by a comprehensive and extendable expert system enabling quick assessment of the probability of a given conformation to occur.  It is based on a hierarchical system of torsion patterns that cover a large part of druglike chemical space.  Each torsion pattern has associated frequency histograms generated from CSD and PDB data and, derived from the histograms, traffic-light rules for frequently observed, rare, and highly unlikely torsion ranges.  Structures imported into the corresponding software are annotated according to these rules.  We present the concept behind the tree of torsion patterns, the design of an intuitive user interface for the management and usage of the torsion library, and we illustrate how the system helps analyze and understand conformation properties of substructures widely used in medicinal chemistry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR2PDX1V9jtEkEgZCMqtRBifW6udTcc2eYkP9PJLewKNLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3szltlGitw%253D%253D&md5=7d0c2f9177cef90f0d7376836a8cefb2</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm3016816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3016816%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25A4rfer%26aufirst%3DC.%26aulast%3DSchulz-Gasch%26aufirst%3DT.%26aulast%3DEhrlich%26aufirst%3DH.-C.%26aulast%3DGuba%26aufirst%3DW.%26aulast%3DRarey%26aufirst%3DM.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DTorsion%2520Angle%2520Preferences%2520in%2520Druglike%2520Chemical%2520Space%253A%2520A%2520Comprehensive%2520Guide%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2016%26epage%3D2028%26doi%3D10.1021%2Fjm3016816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Kuhn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohr, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span> </span><span class="NLM_article-title">Intramolecular Hydrogen Bonding in Medicinal Chemistry</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2601</span><span class="NLM_x">–</span> <span class="NLM_lpage">2611</span><span class="refDoi"> DOI: 10.1021/jm100087s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100087s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Clur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2601-2611&author=B.+Kuhnauthor=P.+Mohrauthor=M.+Stahl&title=Intramolecular+Hydrogen+Bonding+in+Medicinal+Chemistry&doi=10.1021%2Fjm100087s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Intramolecular Hydrogen Bonding in Medicinal Chemistry</span></div><div class="casAuthors">Kuhn, Bernd; Mohr, Peter; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2601-2611</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The formation of intramol. hydrogen bonds has a very pronounced effect on mol. structure and properties.  We study both aspects in detail with the aim of enabling a more rational use of this class of interactions in medicinal chem.  On the basis of exhaustive searches in crystal structure databases, we derive propensities for intramol. hydrogen bond formation of five- to eight-membered ring systems of relevance in drug discovery.  A no. of motifs, several of which are clearly underutilized in drug discovery, are analyzed in more detail by comparing small mol. and protein-ligand X-ray structures.  To investigate effects on physicochem. properties, sets of closely related structures with and without the ability to form intramol. hydrogen bonds were designed, synthesized, and characterized with respect to membrane permeability, water soly., and lipophilicity.  We find that changes in these properties depend on a subtle balance between the strength of the hydrogen bond interaction, geometry of the newly formed ring system, and the relative energies of the open and closed conformations in polar and unpolar environments.  A no. of general guidelines for medicinal chemists emerge from this study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdaV7tFRU4k7Vg90H21EOLACvtfcHk0lglTwSCPtrXMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Clur8%253D&md5=b9b6865dfcc9bdf7796d15218f6e8ff2</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm100087s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100087s%26sid%3Dliteratum%253Aachs%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DMohr%26aufirst%3DP.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DIntramolecular%2520Hydrogen%2520Bonding%2520in%2520Medicinal%2520Chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2601%26epage%3D2611%26doi%3D10.1021%2Fjm100087s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Bissantz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span> </span><span class="NLM_article-title">A Medicinal Chemist’s Guide to Molecular Interactions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5061</span><span class="NLM_x">–</span> <span class="NLM_lpage">5084</span><span class="refDoi"> DOI: 10.1021/jm100112j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100112j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5061-5084&author=C.+Bissantzauthor=B.+Kuhnauthor=M.+Stahl&title=A+Medicinal+Chemist%E2%80%99s+Guide+to+Molecular+Interactions&doi=10.1021%2Fjm100112j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A Medicinal Chemist's Guide to Molecular Interactions</span></div><div class="casAuthors">Bissantz, Caterina; Kuhn, Bernd; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5061-5084</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraV8HPbwsihbVg90H21EOLACvtfcHk0livpiVm0LtoLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D&md5=f39cf8700267cdc311c73de4d433bab0</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fjm100112j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100112j%26sid%3Dliteratum%253Aachs%26aulast%3DBissantz%26aufirst%3DC.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DA%2520Medicinal%2520Chemist%25E2%2580%2599s%2520Guide%2520to%2520Molecular%2520Interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5061%26epage%3D5084%26doi%3D10.1021%2Fjm100112j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span> </span><span class="NLM_article-title">SuperStar: A Knowledge-Based Approach for Identifying Interaction Sites in Proteins</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">1093</span><span class="NLM_x">–</span> <span class="NLM_lpage">1108</span><span class="refDoi"> DOI: 10.1006/jmbi.1999.2809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1006%2Fjmbi.1999.2809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10369784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADyaK1MXjslKhtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=1999&pages=1093-1108&author=M.+L.+Verdonkauthor=J.+C.+Coleauthor=R.+Taylor&title=SuperStar%3A+A+Knowledge-Based+Approach+for+Identifying+Interaction+Sites+in+Proteins&doi=10.1006%2Fjmbi.1999.2809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">SuperStar: A Knowledge-based Approach for Identifying Interaction Sites in Proteins</span></div><div class="casAuthors">Verdonk, Marcel L.; Cole, Jason C.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1093-1108</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">An empirical method for identifying interaction sites in proteins is described and validated.  The method is based entirely on exptl. information about non-bonded interactions occurring in small-mol. crystal structures.  These data are used in the form of scatterplots that show the exptl. obsd. distribution of one functional group (the "contact group" or "probe") around another.  A template mol. (e.g. a protein binding site) is broken down into structure fragments and the scatterplots, showing the distribution of a chosen probe around these structure fragments, are superimposed on the corresponding parts of the template.  The scatterplots are then translated into a three-dimensional map that shows the propensity of the probe at different positions around the template mol.  The method is illustrated for L-arabinose-binding protein, complexed with L-arabinose and with D-fucose, and for dihydrofolate reductase complexed with methotrexate.  The method is validated on 122 x-ray structures of protein-ligand complexes.  For all the binding sites of these proteins, propensity maps are generated for four different probes: a charged NH+3 nitrogen, a carbonyl oxygen, a hydroxyl oxygen and a Me carbon atom.  Next, the maps are compared with the exptl. obsd. positions of ligand atoms of these types.  For 74% of these ligand atoms (84% of the solvent-inaccessible ones) the calcd. propensity of the matching probe at the exptl. positions is higher than expected by chance.  For 68% of the atoms (82% of the solvent-inaccessible ones) the propensity of the matching probe is higher than that of the other three probes.  These results indicate that the approach generally gives good predictions for protein-ligand interactions.  The potential applications of the propensity maps range from an aid in manual docking and structure-based drug design to their use in pharmacophore development.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9QVkFGAwFt7Vg90H21EOLACvtfcHk0livpiVm0LtoLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjslKhtrs%253D&md5=72e09c54c9a631a8271fbe72f2f431b1</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1999.2809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1999.2809%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DSuperStar%253A%2520A%2520Knowledge-Based%2520Approach%2520for%2520Identifying%2520Interaction%2520Sites%2520in%2520Proteins%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1999%26volume%3D289%26spage%3D1093%26epage%3D1108%26doi%3D10.1006%2Fjmbi.1999.2809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Bruno, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lommerse, J. P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowland, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span> </span><span class="NLM_article-title">IsoStar: A Library of Information about Nonbonded Interactions</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">525</span><span class="NLM_x">–</span> <span class="NLM_lpage">537</span><span class="refDoi"> DOI: 10.1023/A:1007934413448</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1023%2FA%3A1007934413448" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=9491345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADyaK1cXht1Olu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1997&pages=525-537&author=I.+J.+Brunoauthor=J.+C.+Coleauthor=J.+P.+M.+Lommerseauthor=R.+S.+Rowlandauthor=R.+Taylorauthor=M.+L.+Verdonk&title=IsoStar%3A+A+Library+of+Information+about+Nonbonded+Interactions&doi=10.1023%2FA%3A1007934413448"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Isostar: a library of information about nonbonded interactions</span></div><div class="casAuthors">Bruno, Ian J.; Cole, Jason C.; Lommerse, Jos P. M.; Rowland, R. Scott; Taylor, Robin; Verdonk, Marcel L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">525-537</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">Crystallog. and theor. (ab initio) data on intermol. nonbonded interactions were gathered together in a computerized library ('IsoStar').  The library contains information about the nonbonded contacts formed by some 250 chem. groupings.  The data can be displayed visually and used to aid protein-ligand docking or the identification of bioisosteric replacements.  Data from the library show that there is great variability in the geometrical preferences of different types of hydrogen bonds, although in general there is a tendency for H-bonds to form along lone-pair directions.  The H-bond acceptor abilities of oxygen and sulfur atoms are highly dependent on intramol. environments.  The nonbonded contacts formed by many hydrophobic groups show surprisingly strong directional preferences.  Many unusual nonbonded interactions are to be found in the library and are of potential value for designing novel biol. active mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcIzn6KO_NVbVg90H21EOLACvtfcHk0livpiVm0LtoLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXht1Olu7s%253D&md5=ec168252a1dc7bddc341235db0f5ccfb</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1023%2FA%3A1007934413448&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1007934413448%26sid%3Dliteratum%253Aachs%26aulast%3DBruno%26aufirst%3DI.%2BJ.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DLommerse%26aufirst%3DJ.%2BP.%2BM.%26aulast%3DRowland%26aufirst%3DR.%2BS.%26aulast%3DTaylor%26aufirst%3DR.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26atitle%3DIsoStar%253A%2520A%2520Library%2520of%2520Information%2520about%2520Nonbonded%2520Interactions%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1997%26volume%3D11%26spage%3D525%26epage%3D537%26doi%3D10.1023%2FA%3A1007934413448" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Kuhn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuchs, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reutlinger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, N. R.</span><span> </span><span class="NLM_article-title">Rationalizing Tight Ligand Binding through Cooperative Interaction Networks</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">3180</span><span class="NLM_x">–</span> <span class="NLM_lpage">3198</span><span class="refDoi"> DOI: 10.1021/ci200319e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200319e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVOlt7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=3180-3198&author=B.+Kuhnauthor=J.+E.+Fuchsauthor=M.+Reutlingerauthor=M.+Stahlauthor=N.+R.+Taylor&title=Rationalizing+Tight+Ligand+Binding+through+Cooperative+Interaction+Networks&doi=10.1021%2Fci200319e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Rationalizing Tight Ligand Binding through Cooperative Interaction Networks</span></div><div class="casAuthors">Kuhn, Bernd; Fuchs, Julian E.; Reutlinger, Michael; Stahl, Martin; Taylor, Neil R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3180-3198</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small modifications of the mol. structure of a ligand sometimes cause strong gains in binding affinity to a protein target, rendering a weakly active chem. series suddenly attractive for further optimization.  Our goal in this study is to better rationalize and predict the occurrence of such interaction hot-spots in receptor binding sites.  To this end, we introduce two new concepts into the computational description of mol. recognition.  First, we take a broader view of noncovalent interactions and describe protein-ligand binding with a comprehensive set of favorable and unfavorable contact types, including for example halogen bonding and orthogonal multipolar interactions.  Second, we go beyond the commonly used pairwise additive treatment of at. interactions and use a small world network approach to describe how interactions are modulated by their environment.  This approach allows us to capture local cooperativity effects and considerably improves the performance of a newly derived empirical scoring function, Scorpions-core.  More importantly, however, we demonstrate how an intuitive visualization of key intermol. interactions, interaction networks, and binding hot-spots supports the identification and rationalization of tight ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCzUUIjVW8P7Vg90H21EOLACvtfcHk0livpiVm0LtoLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVOlt7rI&md5=e8e1bdc3bad1cfdfc11253de5837e4c6</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fci200319e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200319e%26sid%3Dliteratum%253Aachs%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3DReutlinger%26aufirst%3DM.%26aulast%3DStahl%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DN.%2BR.%26atitle%3DRationalizing%2520Tight%2520Ligand%2520Binding%2520through%2520Cooperative%2520Interaction%2520Networks%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D3180%26epage%3D3198%26doi%3D10.1021%2Fci200319e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span> </span><span class="NLM_article-title">Quantum Mechanical Methods for Drug Design</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span><span class="NLM_x">–</span> <span class="NLM_lpage">45</span><span class="refDoi"> DOI: 10.2174/156802610790232242</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.2174%2F156802610790232242" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=19929831" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFWqu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=33-45&author=T.+Zhouauthor=D.+Huangauthor=A.+Caflisch&title=Quantum+Mechanical+Methods+for+Drug+Design&doi=10.2174%2F156802610790232242"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Quantum mechanical methods for drug design</span></div><div class="casAuthors">Zhou, Ting; Huang, Danzhi; Caflisch, Amedeo</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-45</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Quantum mech. (QM) methods are becoming popular in computational drug design and development mainly because high accuracy is required to est. (relative) binding affinities.  For low- to medium-throughput in silico screening, (e.g., scoring and prioritizing a series of inhibitors sharing the same mol. scaffold) efficient approxns. have been developed in the past decade, like linear scaling QM in which the computation time scales almost linearly with the no. of basis functions.  Furthermore, QM-based procedures have been used recently for detg. protonation states of ionizable groups, evaluating energies, and optimizing mol. structures.  For high-throughput in silico screening QM approaches have been employed to derive robust quant. structure-activity relation models.  It is expected that the use of QM methods will keep growing in all phases of computer-aided drug design and development.  However, extensive sampling of conformational space and treatment of soln. of macromols. are still limiting factors for the broad application of QM in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocoQjA2ielFbVg90H21EOLACvtfcHk0lhBVU0Murn5yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFWqu7o%253D&md5=735ac208504a7e563e6ca0eb3c1cd17b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.2174%2F156802610790232242&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802610790232242%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DQuantum%2520Mechanical%2520Methods%2520for%2520Drug%2520Design%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2010%26volume%3D10%26spage%3D33%26epage%3D45%26doi%3D10.2174%2F156802610790232242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Groom, C.</span><span> </span><span class="NLM_article-title">Small Molecule Crystal Structures in Drug Discovery</span>. In  <span class="citation_source-book">Multifaceted Roles of Crystallography in Modern Drug Discovery</span>; <span class="NLM_contrib-group">Scapin, G.; Patel, D.; Arnold, E.</span>, Eds.; NATO Science for Peace and Security Series A: Chemistry and Biology; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">The Netherlands</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">107</span><span class="NLM_x">–</span> <span class="NLM_lpage">114</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1007%2F978-94-017-9719-1_9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=107-114&author=C.+Groomauthor=G.+Scapin&author=D.+Patel&author=E.+Arnold&title=Multifaceted+Roles+of+Crystallography+in+Modern+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1007%2F978-94-017-9719-1_9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-94-017-9719-1_9%26sid%3Dliteratum%253Aachs%26aulast%3DGroom%26aufirst%3DC.%26atitle%3DSmall%2520Molecule%2520Crystal%2520Structures%2520in%2520Drug%2520Discovery%26btitle%3DMultifaceted%2520Roles%2520of%2520Crystallography%2520in%2520Modern%2520Drug%2520Discovery%26aulast%3DScapin%26aufirst%3DG.%26pub%3DSpringer%26date%3D2015%26spage%3D107%26epage%3D114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Bruno, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edgington, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macrae, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span> </span><span class="NLM_article-title">New Software for Searching the Cambridge Structural Database and Visualizing Crystal Structures</span> <span class="citation_source-journal">Acta Crystallogr., Sect. B: Struct. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">389</span><span class="NLM_x">–</span> <span class="NLM_lpage">397</span><span class="refDoi"> DOI: 10.1107/S0108768102003324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1107%2FS0108768102003324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=12037360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BD38XktVOqu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=389-397&author=I.+J.+Brunoauthor=J.+C.+Coleauthor=P.+R.+Edgingtonauthor=M.+Kesslerauthor=C.+F.+Macraeauthor=P.+McCabeauthor=J.+Pearsonauthor=R.+Taylor&title=New+Software+for+Searching+the+Cambridge+Structural+Database+and+Visualizing+Crystal+Structures&doi=10.1107%2FS0108768102003324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">New software for searching the Cambridge Structural Database and visualizing crystal structures</span></div><div class="casAuthors">Bruno, Ian J.; Cole, Jason C.; Edgington, Paul R.; Kessler, Magnus; Macrae, Clare F.; McCabe, Patrick; Pearson, Jonathan; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section B: Structural Science</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">B58</span>
        (<span class="NLM_cas:issue">3, No. 1</span>),
    <span class="NLM_cas:pages">389-397</span>CODEN:
                <span class="NLM_cas:coden">ASBSDK</span>;
        ISSN:<span class="NLM_cas:issn">0108-7681</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">Two new programs were developed for searching the Cambridge Structural Database (CSD) and visualizing database entries: ConQuest and Mercury.  The former is a new search interface to the CSD, the latter is a high-performance crystal-structure visualizer with extensive facilities for exploring networks of intermol. contacts.  Particular emphasis was placed on making the programs as intuitive as possible.  Both ConQuest and Mercury run under Windows and various types of Unix, including Linux.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIZFqT_ocomLVg90H21EOLACvtfcHk0lhBVU0Murn5yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XktVOqu78%253D&md5=b8cd5dddcd43067010fef6d60e37b3c2</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1107%2FS0108768102003324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0108768102003324%26sid%3Dliteratum%253Aachs%26aulast%3DBruno%26aufirst%3DI.%2BJ.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DEdgington%26aufirst%3DP.%2BR.%26aulast%3DKessler%26aufirst%3DM.%26aulast%3DMacrae%26aufirst%3DC.%2BF.%26aulast%3DMcCabe%26aufirst%3DP.%26aulast%3DPearson%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DNew%2520Software%2520for%2520Searching%2520the%2520Cambridge%2520Structural%2520Database%2520and%2520Visualizing%2520Crystal%2520Structures%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520B%253A%2520Struct.%2520Sci.%26date%3D2002%26volume%3D58%26spage%3D389%26epage%3D397%26doi%3D10.1107%2FS0108768102003324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Hendlich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Günther, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Relibase: Design and Development of a Database for Comprehensive Analysis of Protein-Ligand Interactions</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">326</span><span class="NLM_x">, </span> <span class="NLM_fpage">607</span><span class="NLM_x">–</span> <span class="NLM_lpage">620</span><span class="refDoi"> DOI: 10.1016/S0022-2836(02)01408-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1016%2FS0022-2836%2802%2901408-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=326&publication_year=2003&pages=607-620&author=M.+Hendlichauthor=A.+Bergnerauthor=J.+G%C3%BCntherauthor=G.+Klebe&title=Relibase%3A+Design+and+Development+of+a+Database+for+Comprehensive+Analysis+of+Protein-Ligand+Interactions&doi=10.1016%2FS0022-2836%2802%2901408-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2901408-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252901408-0%26sid%3Dliteratum%253Aachs%26aulast%3DHendlich%26aufirst%3DM.%26aulast%3DBergner%26aufirst%3DA.%26aulast%3DG%25C3%25BCnther%26aufirst%3DJ.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DRelibase%253A%2520Design%2520and%2520Development%2520of%2520a%2520Database%2520for%2520Comprehensive%2520Analysis%2520of%2520Protein-Ligand%2520Interactions%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2003%26volume%3D326%26spage%3D607%26epage%3D620%26doi%3D10.1016%2FS0022-2836%2802%2901408-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Inhester, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span> </span><span class="NLM_article-title">Protein–ligand Interaction Databases: Advanced Tools to Mine Activity Data and Interactions on a Structural Level</span> <span class="citation_source-journal">Wiley Interdiscip. Rev. Comput. Mol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">562</span><span class="NLM_x">–</span> <span class="NLM_lpage">575</span><span class="refDoi"> DOI: 10.1002/wcms.1192</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1002%2Fwcms.1192" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVelu7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=562-575&author=T.+Inhesterauthor=M.+Rarey&title=Protein%E2%80%93ligand+Interaction+Databases%3A+Advanced+Tools+to+Mine+Activity+Data+and+Interactions+on+a+Structural+Level&doi=10.1002%2Fwcms.1192"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-ligand interaction databases: advanced tools to mine activity data and interactions on a structural level</span></div><div class="casAuthors">Inhester, Therese; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Wiley Interdisciplinary Reviews: Computational Molecular Science</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">562-575</span>CODEN:
                <span class="NLM_cas:coden">WIRCAH</span>;
        ISSN:<span class="NLM_cas:issn">1759-0884</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The formation of mol. complexes between proteins and small org. substances is a fundamental concept of life.  Biochem. expts. from X-ray crystallog. to isothermal titrn. calorimetry (ITC) are applied in large-scale providing data for the anal. of the structural foundations of binding affinity.  In recent years, several, mostly publically available databases emerged contg. affinity data and structural information.  These databases are central for the construction of complex models describing interaction geometries and correlate structural features to the strength of binding.  Binding affinity databases reflect the knowledge of affinity measurements from many sources, mostly scientific and patent literature.  A crit. aspect is the data quality, which is affected by transcription errors during database construction as well as exptl. uncertainties.  The Protein Data Bank (PDB) is the central resource for macromol. biol. structures contg. nearly 100,000 data entries today.  Sophisticated geometric databases have been constructed based on this allowing for complex queries about the spatial arrangement of functional groups and their interactions.  For scientists working in mol. design like medicinal chemists, access to this information can substantially support the process of creating new mol. entities specifically interacting with proteins of interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVnCd3Xg79fLVg90H21EOLACvtfcHk0lglVsuMmOCn5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVelu7jL&md5=7bce4b6447299944bd7598eaf0d3e7e0</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2Fwcms.1192&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwcms.1192%26sid%3Dliteratum%253Aachs%26aulast%3DInhester%26aufirst%3DT.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DProtein%25E2%2580%2593ligand%2520Interaction%2520Databases%253A%2520Advanced%2520Tools%2520to%2520Mine%2520Activity%2520Data%2520and%2520Interactions%2520on%2520a%2520Structural%2520Level%26jtitle%3DWiley%2520Interdiscip.%2520Rev.%2520Comput.%2520Mol.%2520Sci.%26date%3D2014%26volume%3D4%26spage%3D562%26epage%3D575%26doi%3D10.1002%2Fwcms.1192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Schreyer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T.</span><span> </span><span class="NLM_article-title">CREDO: A Protein–Ligand Interaction Database for Drug Discovery</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">157</span><span class="NLM_x">–</span> <span class="NLM_lpage">167</span><span class="refDoi"> DOI: 10.1111/j.1747-0285.2008.00762.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1111%2Fj.1747-0285.2008.00762.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=19207418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitlGit7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2009&pages=157-167&author=A.+Schreyerauthor=T.+Blundell&title=CREDO%3A+A+Protein%E2%80%93Ligand+Interaction+Database+for+Drug+Discovery&doi=10.1111%2Fj.1747-0285.2008.00762.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">CREDO: a protein-ligand interaction database for drug discovery</span></div><div class="casAuthors">Schreyer, Adrian; Blundell, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-167</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Harnessing data from the growing no. of protein-ligand complexes in the Protein Data Bank is an important task in drug discovery.  In order to benefit from the abundance of three-dimensional structures, structural data must be integrated with sequence as well as chem. data and the protein-small mol. interactions characterized structurally at the inter-at. level.  In this study, we present CREDO, a new publicly available database of protein-ligand interactions, which represents contacts as structural interaction fingerprints, implements novel features and is completely scriptable through its application programming interface.  Features of CREDO include implementation of mol. shape descriptors with ultrafast shape recognition, fragmentation of ligands in the Protein Data Bank, sequence-to-structure mapping and the identification of approved drugs.  Selected analyses of these key features are presented to highlight a range of potential applications of CREDO.  The CREDO dataset has been released into the public domain together with the application programming interface under a Creative Commons license at http://www-cryst.bioc.cam.ac.uk/credo.  We believe that the free availability and numerous features of CREDO database will be useful not only for com. but also for academia-driven drug discovery programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqun6w6sZJKzrVg90H21EOLACvtfcHk0lglVsuMmOCn5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitlGit7s%253D&md5=15a45aa365edc83d294848b637ae31d0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2008.00762.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2008.00762.x%26sid%3Dliteratum%253Aachs%26aulast%3DSchreyer%26aufirst%3DA.%26aulast%3DBlundell%26aufirst%3DT.%26atitle%3DCREDO%253A%2520A%2520Protein%25E2%2580%2593Ligand%2520Interaction%2520Database%2520for%2520Drug%2520Discovery%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2009%26volume%3D73%26spage%3D157%26epage%3D167%26doi%3D10.1111%2Fj.1747-0285.2008.00762.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Weisel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bitter, H.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diederich, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">So, W. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kondru, R.</span><span> </span><span class="NLM_article-title">PROLIX: Rapid Mining of Protein-Ligand Interactions in Large Crystal Structure Databases</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1450</span><span class="NLM_x">–</span> <span class="NLM_lpage">1461</span><span class="refDoi"> DOI: 10.1021/ci300034x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300034x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvV2nsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1450-1461&author=M.+Weiselauthor=H.-M.+Bitterauthor=F.+Diederichauthor=W.+V.+Soauthor=R.+Kondru&title=PROLIX%3A+Rapid+Mining+of+Protein-Ligand+Interactions+in+Large+Crystal+Structure+Databases&doi=10.1021%2Fci300034x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">PROLIX: Rapid Mining of Protein-Ligand Interactions in Large Crystal Structure Databases</span></div><div class="casAuthors">Weisel, Martin; Bitter, Hans-Marcus; Diederich, Francois; So, W. Venus; Kondru, Rama</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1450-1461</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A central problem in structure-based drug design is understanding protein-ligand interactions quant. and qual.  Several recent studies have highlighted from a qual. perspective the nature of these interactions and their utility in drug discovery.  However, a common limitation is a lack of adequate tools to mine these interactions comprehensively, since exhaustive searches of the protein data bank are time-consuming and difficult to perform.  Consequently, fundamental questions remain unanswered: How unique or how common are the protein-ligand interactions obsd. in a given drug design project when compared to all complexed structures in the protein data bank.  Which interaction patterns might explain the affinity of a tool compd. toward unwanted targets.  To answer these questions and to enable the systematic and comprehensive study of protein-ligand interactions, we introduce PROLIX (Protein Ligand Interaction Explorer), a tool that uses sophisticated fingerprint representations of protein-ligand interaction patterns for rapid data mining in large crystal structure databases.  Our implementation strategy pursues a branch-and-bound technique that enables mining against thousands of complexes within a few seconds.  Key elements of PROLIX include (i) an intuitive interface that enables users to formulate complex queries easily, (ii) exceptional speed for results retrieval, and (iii) a sophisticated results summarization.  Herein we describe the algorithms developed to enable complex queries and fast retrieval of search results, as well as the intuitive aspects of the user interface and summarization viewer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi4_gNpv_-8rVg90H21EOLACvtfcHk0lglVsuMmOCn5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvV2nsro%253D&md5=dfcfea5c25b13d2cc9543735a9d11b15</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fci300034x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300034x%26sid%3Dliteratum%253Aachs%26aulast%3DWeisel%26aufirst%3DM.%26aulast%3DBitter%26aufirst%3DH.-M.%26aulast%3DDiederich%26aufirst%3DF.%26aulast%3DSo%26aufirst%3DW.%2BV.%26aulast%3DKondru%26aufirst%3DR.%26atitle%3DPROLIX%253A%2520Rapid%2520Mining%2520of%2520Protein-Ligand%2520Interactions%2520in%2520Large%2520Crystal%2520Structure%2520Databases%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1450%26epage%3D1461%26doi%3D10.1021%2Fci300034x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Shirvanyants, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alexandrova, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dokholyan, N. V.</span><span> </span><span class="NLM_article-title">Rigid Substructure Search</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">1327</span><span class="NLM_x">–</span> <span class="NLM_lpage">1329</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btr129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1093%2Fbioinformatics%2Fbtr129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=21460026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkvFajt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=1327-1329&author=D.+Shirvanyantsauthor=A.+N.+Alexandrovaauthor=N.+V.+Dokholyan&title=Rigid+Substructure+Search&doi=10.1093%2Fbioinformatics%2Fbtr129"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Rigid substructure search</span></div><div class="casAuthors">Shirvanyants, David; Alexandrova, Anastassia N.; Dokholyan, Nikolay V.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1327-1329</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Identifying the location of binding sites on proteins is of fundamental importance for a wide range of applications, including mol. docking, de novo drug design, structure identification and comparison of functional sites.  Here we present Erebus, a web server that searches the entire Protein Data Bank for a given substructure defined by a set of atoms of interest, such as the binding scaffolds for small mols.  The identified substructure contains atoms having the same names, belonging to same amino acids and sepd. by the same distances (within a given tolerance) as the atoms of the query structure.  The accuracy of a match is measured by the root-mean-square deviation or by the normal wt. with a given variance.  Tests show that our approach can reliably locate rigid binding scaffolds of drugs and metal ions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxJlxMIQf27LVg90H21EOLACvtfcHk0lhz-EPefu6BSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkvFajt7k%253D&md5=d650bf655e40b5d129278c8b7e5d6570</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtr129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtr129%26sid%3Dliteratum%253Aachs%26aulast%3DShirvanyants%26aufirst%3DD.%26aulast%3DAlexandrova%26aufirst%3DA.%2BN.%26aulast%3DDokholyan%26aufirst%3DN.%2BV.%26atitle%3DRigid%2520Substructure%2520Search%26jtitle%3DBioinformatics%26date%3D2011%26volume%3D27%26spage%3D1327%26epage%3D1329%26doi%3D10.1093%2Fbioinformatics%2Fbtr129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Schmitt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">A New Method to Detect Related Function among Proteins Independent of Sequence and Fold Homology</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">323</span><span class="NLM_x">, </span> <span class="NLM_fpage">387</span><span class="NLM_x">–</span> <span class="NLM_lpage">406</span><span class="refDoi"> DOI: 10.1016/S0022-2836(02)00811-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1016%2FS0022-2836%2802%2900811-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=12381328" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BD38XnslGqsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=323&publication_year=2002&pages=387-406&author=S.+Schmittauthor=D.+Kuhnauthor=G.+Klebe&title=A+New+Method+to+Detect+Related+Function+among+Proteins+Independent+of+Sequence+and+Fold+Homology&doi=10.1016%2FS0022-2836%2802%2900811-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A new method to detect related function among proteins independent of sequence and fold homology</span></div><div class="casAuthors">Schmitt, Stefan; Kuhn, Daniel; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">323</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">387-406</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A new method has been developed to detect functional relationships among proteins independent of a given sequence or fold homol.  It is based on the idea that protein function is intimately related to the recognition and subsequent response to the binding of a substrate or an endogenous ligand in a well-characterized binding pocket.  Thus, recognition of similar ligands, supposedly linked to similar function, requires conserved recognition features exposed in terms of common physicochem. interaction properties via the functional groups of the residues flanking a particular binding cavity.  Following a technique commonly used in the comparison of small mol. ligands, generic pseudocenters coding for possible interaction properties were assigned for a large sample set of cavities extd. from the entire PDB and stored in the database Cavbase.  Using a particular query cavity a series of related cavities of decreasing similarity is detected based on a clique detection algorithm.  The detected similarity is ranked according to property-based surface patches shared in common by the different clique solns.  The approach either retrieves protein cavities accommodating the same (e.g. co-factors) or closely related ligands or it exts. proteins exhibiting similar function in terms of a related catalytic mechanism.  Finally the new method has strong potential to suggest alternative mol. skeletons in de novo design.  The retrieval of mol. building blocks accommodated in a particular sub-pocket that shares similarity with the pocket in a protein studied by drug design can inspire the discovery of novel ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRCdsfEkyF6rVg90H21EOLACvtfcHk0lhz-EPefu6BSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnslGqsrY%253D&md5=192ad6a44a2d4d5406ca47cecb65bd50</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2900811-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252900811-2%26sid%3Dliteratum%253Aachs%26aulast%3DSchmitt%26aufirst%3DS.%26aulast%3DKuhn%26aufirst%3DD.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DA%2520New%2520Method%2520to%2520Detect%2520Related%2520Function%2520among%2520Proteins%2520Independent%2520of%2520Sequence%2520and%2520Fold%2520Homology%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D323%26spage%3D387%26epage%3D406%26doi%3D10.1016%2FS0022-2836%2802%2900811-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Desaphy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Sc-PDB-Frag: A Database of Protein–Ligand Interaction Patterns for Bioisosteric Replacements</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1908</span><span class="NLM_x">–</span> <span class="NLM_lpage">1918</span><span class="refDoi"> DOI: 10.1021/ci500282c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500282c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFShsrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1908-1918&author=J.+Desaphyauthor=D.+Rognan&title=Sc-PDB-Frag%3A+A+Database+of+Protein%E2%80%93Ligand+Interaction+Patterns+for+Bioisosteric+Replacements&doi=10.1021%2Fci500282c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">sc-PDB-Frag: A Database of Protein-Ligand Interaction Patterns for Bioisosteric Replacements</span></div><div class="casAuthors">Desaphy, Jeremy; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1908-1918</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bioisosteric replacement plays an important role in medicinal chem. by keeping the biol. activity of a mol. while changing either its core scaffold or substituents, thereby facilitating lead optimization and patenting.  Bioisosteres are classically chosen in order to keep the main pharmacophoric moieties of the substructure to replace.  However, notably when changing a scaffold, no attention is usually paid as whether all atoms of the ref. scaffold are equally important for binding to the desired target.  We herewith propose a novel database for bioisosteric replacement (scPDBFrag), capitalizing on our recently published structure-based approach to scaffold hopping, focusing on interaction pattern graphs.  Protein-bound ligands are first fragmented and the interaction of the corresponding fragments with their protein environment computed-on-the-fly.  Using an inhouse developed graph alignment tool, interaction patterns graphs can be compared, aligned, and sorted by decreasing similarity to any ref.  In the herein presented s.c.-PDB-Frag database (http://bioinfo-pharma.u-strasbg.fr/scPDBFrag), fragments, interaction patterns, alignments, and pairwise similarity scores have been extd. from the s.c.-PDB database of 8077 druggable protein-ligand complexes and further stored in a relational database.  We herewith present the database, its Web implementation, and procedures for identifying true bioisosteric replacements based on conserved interaction patterns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTtQN9l6kql7Vg90H21EOLACvtfcHk0lhz-EPefu6BSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFShsrfK&md5=db24b1007d32c27af5d076d133aa5bcb</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fci500282c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500282c%26sid%3Dliteratum%253Aachs%26aulast%3DDesaphy%26aufirst%3DJ.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DSc-PDB-Frag%253A%2520A%2520Database%2520of%2520Protein%25E2%2580%2593Ligand%2520Interaction%2520Patterns%2520for%2520Bioisosteric%2520Replacements%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1908%26epage%3D1918%26doi%3D10.1021%2Fci500282c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Golovin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henrick, K.</span><span> </span><span class="NLM_article-title">MSDmotif: Exploring Protein Sites and Motifs</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">312</span><span class="refDoi"> DOI: 10.1186/1471-2105-9-312</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1186%2F1471-2105-9-312" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=18637174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A280%3ADC%252BD1cvos1Glug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=312&author=A.+Golovinauthor=K.+Henrick&title=MSDmotif%3A+Exploring+Protein+Sites+and+Motifs&doi=10.1186%2F1471-2105-9-312"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">MSDmotif: exploring protein sites and motifs</span></div><div class="casAuthors">Golovin Adel; Henrick Kim</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">312</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Protein structures have conserved features - motifs, which have a sufficient influence on the protein function.  These motifs can be found in sequence as well as in 3D space.  Understanding of these fragments is essential for 3D structure prediction, modelling and drug-design.  The Protein Data Bank (PDB) is the source of this information however present search tools have limited 3D options to integrate protein sequence with its 3D structure.  RESULTS:  We describe here a web application for querying the PDB for ligands, binding sites, small 3D structural and sequence motifs and the underlying database.  Novel algorithms for chemical fragments, 3D motifs, phi/psi sequences, super-secondary structure motifs and for small 3D structural motif associations searches are incorporated.  The interface provides functionality for visualization, search criteria creation, sequence and 3D multiple alignment options.  MSDmotif is an integrated system where a results page is also a search form.  A set of motif statistics is available for analysis.  This set includes molecule and motif binding statistics, distribution of motif sequences, occurrence of an amino-acid within a motif, correlation of amino-acids side-chain charges within a motif and Ramachandran plots for each residue.  The binding statistics are presented in association with properties that include a ligand fragment library.  Access is also provided through the distributed Annotation System (DAS) protocol.  An additional entry point facilitates XML requests with XML responses.  CONCLUSION:  MSDmotif is unique by combining chemical, sequence and 3D data in a single search engine with a range of search and visualisation options.  It provides multiple views of data found in the PDB archive for exploring protein structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQNtCwFapSCF8AKB7Fa-xrUfW6udTcc2eYyIEHGODVfKLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1cvos1Glug%253D%253D&md5=23056f227d4293f9b1a756972403faa0</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-9-312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-9-312%26sid%3Dliteratum%253Aachs%26aulast%3DGolovin%26aufirst%3DA.%26aulast%3DHenrick%26aufirst%3DK.%26atitle%3DMSDmotif%253A%2520Exploring%2520Protein%2520Sites%2520and%2520Motifs%26jtitle%3DBMC%2520Bioinf.%26date%3D2008%26volume%3D9%26spage%3D312%26doi%3D10.1186%2F1471-2105-9-312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Gonzalez, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hannigan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeGrado, W. F.</span><span> </span><span class="NLM_article-title">A Real-Time All-Atom Structural Search Engine for Proteins</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003750</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1003750</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1371%2Fjournal.pcbi.1003750" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=25079944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1ersLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1003750&author=G.+Gonzalezauthor=B.+Hanniganauthor=W.+F.+DeGrado&title=A+Real-Time+All-Atom+Structural+Search+Engine+for+Proteins&doi=10.1371%2Fjournal.pcbi.1003750"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A real-time all-atom structural search engine for proteins</span></div><div class="casAuthors">Gonzalez, Gabriel; Hannigan, Brett; DeGrado, William F.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e1003750/1-e1003750/8, 8 pp.</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein designers use a wide variety of software tools for de novo design, yet their repertoire still lacks a fast and interactive all-atom search engine.  To solve this, we have built the Suns program: a real-time, at. search engine integrated into the PyMOL mol. visualization system.  Users build at.-level structural search queries within PyMOL and receive a stream of search results aligned to their query within a few seconds.  This instant feedback cycle enables a new "designability"-inspired approach to protein design where the designer searches for and interactively incorporates native-like fragments from proven protein structures.  We demonstrate the use of Suns to interactively build protein motifs, tertiary interactions, and to identify scaffolds compatible with hot-spot residues.  The official web site and installer are available at online.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9e-aJDstg5rVg90H21EOLACvtfcHk0ljBiUQUzBpp-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1ersLfN&md5=18c14d2b95bc3ec62ebe56fac2a30ba0</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003750%26sid%3Dliteratum%253Aachs%26aulast%3DGonzalez%26aufirst%3DG.%26aulast%3DHannigan%26aufirst%3DB.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3DA%2520Real-Time%2520All-Atom%2520Structural%2520Search%2520Engine%2520for%2520Proteins%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3De1003750%26doi%3D10.1371%2Fjournal.pcbi.1003750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span>Desert Scientific Software. <a href="http://www.desertsci.com/" class="extLink">http://www.desertsci.com/</a> (accessed Aug 26,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Desert+Scientific+Software.+http%3A%2F%2Fwww.desertsci.com%2F+%28accessed+Aug+26%2C+2015%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span>SQLite Home Page. <a href="http://www.sqlite.org/" class="extLink">http://www.sqlite.org/</a> (accessed Oct 20,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=SQLite+Home+Page.+http%3A%2F%2Fwww.sqlite.org%2F+%28accessed+Oct+20%2C+2015%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Van Drie, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weininger, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, Y. C.</span><span> </span><span class="NLM_article-title">ALADDIN: An Integrated Tool for Computer-Assisted Molecular Design and Pharmacophore Recognition from Geometric, Steric, and Substructure Searching of Three-Dimensional Molecular Structures</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">225</span><span class="NLM_x">–</span> <span class="NLM_lpage">251</span><span class="refDoi"> DOI: 10.1007/BF01533070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1007%2FBF01533070" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1989&pages=225-251&author=J.+H.+Van+Drieauthor=D.+Weiningerauthor=Y.+C.+Martin&title=ALADDIN%3A+An+Integrated+Tool+for+Computer-Assisted+Molecular+Design+and+Pharmacophore+Recognition+from+Geometric%2C+Steric%2C+and+Substructure+Searching+of+Three-Dimensional+Molecular+Structures&doi=10.1007%2FBF01533070"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2FBF01533070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01533070%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDrie%26aufirst%3DJ.%2BH.%26aulast%3DWeininger%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DY.%2BC.%26atitle%3DALADDIN%253A%2520An%2520Integrated%2520Tool%2520for%2520Computer-Assisted%2520Molecular%2520Design%2520and%2520Pharmacophore%2520Recognition%2520from%2520Geometric%252C%2520Steric%252C%2520and%2520Substructure%2520Searching%2520of%2520Three-Dimensional%2520Molecular%2520Structures%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1989%26volume%3D3%26spage%3D225%26epage%3D251%26doi%3D10.1007%2FBF01533070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Sastry, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span> </span><span class="NLM_article-title">Rapid Shape-Based Ligand Alignment and Virtual Screening Method Based on Atom/Feature-Pair Similarities and Volume Overlap Scoring</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2455</span><span class="NLM_x">–</span> <span class="NLM_lpage">2466</span><span class="refDoi"> DOI: 10.1021/ci2002704</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci2002704" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFKnt73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2455-2466&author=G.+M.+Sastryauthor=S.+L.+Dixonauthor=W.+Sherman&title=Rapid+Shape-Based+Ligand+Alignment+and+Virtual+Screening+Method+Based+on+Atom%2FFeature-Pair+Similarities+and+Volume+Overlap+Scoring&doi=10.1021%2Fci2002704"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Shape-Based Ligand Alignment and Virtual Screening Method Based on Atom/Feature-Pair Similarities and Volume Overlap Scoring</span></div><div class="casAuthors">Sastry, G. Madhavi; Dixon, Steven L.; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2455-2466</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Shape-based methods for aligning and scoring ligands have proven to be valuable in the field of computer-aided drug design.  Here, we describe a new shape-based flexible ligand superposition and virtual screening method, Phase Shape, which is shown to rapidly produce accurate 3D ligand alignments and efficiently enrich actives in virtual screening.  We describe the methodol., which is based on the principle of atom distribution triplets to rapidly define trial alignments, followed by refinement of top alignments to maximize the vol. overlap.  The method can be run in a shape-only mode or it can include atom types or pharmacophore feature encoding, the latter consistently producing the best results for database screening.  We apply Phase Shape to flexibly align mols. that bind to the same target and show that the method consistently produces correct alignments when compared with crystal structures.  We then illustrate the effectiveness of the method for identifying active compds. in virtual screening of eleven diverse targets.  Multiple parameters are explored, including atom typing, query structure conformation, and the database conformer generation protocol.  We show that Phase Shape performs well in database screening calcns. when compared with other shape-based methods using a common set of actives and decoys from the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM-u23DvDpQ7Vg90H21EOLACvtfcHk0ljBiUQUzBpp-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFKnt73K&md5=4234abb2dd7e6364c377e92047b65aba</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fci2002704&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci2002704%26sid%3Dliteratum%253Aachs%26aulast%3DSastry%26aufirst%3DG.%2BM.%26aulast%3DDixon%26aufirst%3DS.%2BL.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DRapid%2520Shape-Based%2520Ligand%2520Alignment%2520and%2520Virtual%2520Screening%2520Method%2520Based%2520on%2520Atom%252FFeature-Pair%2520Similarities%2520and%2520Volume%2520Overlap%2520Scoring%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2455%26epage%3D2466%26doi%3D10.1021%2Fci2002704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Wolber, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langer, T.</span><span> </span><span class="NLM_article-title">LigandScout: 3-D Pharmacophores Derived from Protein-Bound Ligands and Their Use as Virtual Screening Filters</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">160</span><span class="NLM_x">–</span> <span class="NLM_lpage">169</span><span class="refDoi"> DOI: 10.1021/ci049885e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049885e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=160-169&author=G.+Wolberauthor=T.+Langer&title=LigandScout%3A+3-D+Pharmacophores+Derived+from+Protein-Bound+Ligands+and+Their+Use+as+Virtual+Screening+Filters&doi=10.1021%2Fci049885e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">LigandScout: 3-D pharmacophores derived from protein-bound ligands and their use as virtual screening filters</span></div><div class="casAuthors">Wolber Gerhard; Langer Thierry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">160-9</span>
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    </div><div class="casAbstract">From the historically grown archive of protein-ligand complexes in the Protein Data Bank small organic ligands are extracted and interpreted in terms of their chemical characteristics and features.  Subsequently, pharmacophores representing ligand-receptor interaction are derived from each of these small molecules and its surrounding amino acids.  Based on a defined set of only six types of chemical features and volume constraints, three-dimensional pharmacophore models are constructed, which are sufficiently selective to identify the described binding mode and are thus a useful tool for in-silico screening of large compound databases.  The algorithms for ligand extraction and interpretation as well as the pharmacophore creation technique from the automatically interpreted data are presented and applied to a rhinovirus capsid complex as application example.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQVi_-excgTX2BAiYWy6n3SfW6udTcc2ebpyLwHq0mRHLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtg%253D%253D&md5=31f8ff7a2fa4c9411d54b001b7e1da50</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fci049885e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049885e%26sid%3Dliteratum%253Aachs%26aulast%3DWolber%26aufirst%3DG.%26aulast%3DLanger%26aufirst%3DT.%26atitle%3DLigandScout%253A%25203-D%2520Pharmacophores%2520Derived%2520from%2520Protein-Bound%2520Ligands%2520and%2520Their%2520Use%2520as%2520Virtual%2520Screening%2520Filters%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D160%26epage%3D169%26doi%3D10.1021%2Fci049885e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span>Daylight Theory: SMARTS—A Language for Describing Molecular Patterns. <a href="http://www.daylight.com/dayhtml/doc/theory/theory.smarts.html" class="extLink">http://www.daylight.com/dayhtml/doc/theory/theory.smarts.html</a> (accessed Jun 1,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Daylight+Theory%3A+SMARTS%E2%80%94A+Language+for+Describing+Molecular+Patterns.+http%3A%2F%2Fwww.daylight.com%2Fdayhtml%2Fdoc%2Ftheory%2Ftheory.smarts.html+%28accessed+Jun+1%2C+2015%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Vitaku, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Njardarson, J. T.</span><span> </span><span class="NLM_article-title">Analysis of the Structural Diversity, Substitution Patterns, and Frequency of Nitrogen Heterocycles among U.S. FDA Approved Pharmaceuticals</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">10257</span><span class="NLM_x">–</span> <span class="NLM_lpage">10274</span><span class="refDoi"> DOI: 10.1021/jm501100b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501100b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlu7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=10257-10274&author=E.+Vitakuauthor=D.+T.+Smithauthor=J.+T.+Njardarson&title=Analysis+of+the+Structural+Diversity%2C+Substitution+Patterns%2C+and+Frequency+of+Nitrogen+Heterocycles+among+U.S.+FDA+Approved+Pharmaceuticals&doi=10.1021%2Fjm501100b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of the Structural Diversity, Substitution Patterns, and Frequency of Nitrogen Heterocycles among U.S. FDA Approved Pharmaceuticals</span></div><div class="casAuthors">Vitaku, Edon; Smith, David T.; Njardarson, Jon T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10257-10274</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Nitrogen heterocycles are among the most significant structural components of pharmaceuticals.  Anal. of our database of U.S. FDA approved drugs reveals that 59% of unique small-mol. drugs contain a nitrogen heterocycle.  In this review we report on the top 25 most commonly utilized nitrogen heterocycles found in pharmaceuticals.  The main part of our anal. is divided into seven sections: (1) three- and four-membered heterocycles, (2) five-, (3) six-, and (4) seven- and eight-membered heterocycles, as well as (5) fused, (6) bridged bicyclic, and (7) macrocyclic nitrogen heterocycles.  Each section reveals the top nitrogen heterocyclic structures and their relative impact for that ring type.  For the most commonly used nitrogen heterocycles, we report detailed substitution patterns, highlight common architectural cores, and discuss unusual or rare structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5cUk9KxUSkbVg90H21EOLACvtfcHk0lhh2H3ugO3Pjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlu7%252FP&md5=7065b3b2fc6f69cede0f87479c7cf472</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm501100b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501100b%26sid%3Dliteratum%253Aachs%26aulast%3DVitaku%26aufirst%3DE.%26aulast%3DSmith%26aufirst%3DD.%2BT.%26aulast%3DNjardarson%26aufirst%3DJ.%2BT.%26atitle%3DAnalysis%2520of%2520the%2520Structural%2520Diversity%252C%2520Substitution%2520Patterns%252C%2520and%2520Frequency%2520of%2520Nitrogen%2520Heterocycles%2520among%2520U.S.%2520FDA%2520Approved%2520Pharmaceuticals%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D10257%26epage%3D10274%26doi%3D10.1021%2Fjm501100b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Hartshorn, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brewerton, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mooij, W. T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mortenson, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span> </span><span class="NLM_article-title">Diverse, High-Quality Test Set for the Validation of Protein–Ligand Docking Performance</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">726</span><span class="NLM_x">–</span> <span class="NLM_lpage">741</span><span class="refDoi"> DOI: 10.1021/jm061277y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm061277y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmsVymtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=726-741&author=M.+J.+Hartshornauthor=M.+L.+Verdonkauthor=G.+Chessariauthor=S.+C.+Brewertonauthor=W.+T.+M.+Mooijauthor=P.+N.+Mortensonauthor=C.+W.+Murray&title=Diverse%2C+High-Quality+Test+Set+for+the+Validation+of+Protein%E2%80%93Ligand+Docking+Performance&doi=10.1021%2Fjm061277y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Diverse, High-Quality Test Set for the Validation of Protein-Ligand Docking Performance</span></div><div class="casAuthors">Hartshorn, Michael J.; Verdonk, Marcel L.; Chessari, Gianni; Brewerton, Suzanne C.; Mooij, Wijnand T. M.; Mortenson, Paul N.; Murray, Christopher W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">726-741</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A procedure for analyzing and classifying publicly available crystal structures has been developed.  It has been used to identify high-resoln. protein-ligand complexes that can be assessed by reconstructing the electron d. for the ligand using the deposited structure factors.  The complexes have been clustered according to the protein sequences, and clusters have been discarded if they do not represent proteins thought to be of direct interest to the pharmaceutical or agrochem. industry.  Rules have been used to exclude complexes contg. nondrug-like ligands.  One complex from each cluster has been selected where a structure of sufficient quality was available.  The final Astex diverse set contains 85 diverse, relevant protein-ligand complexes, which have been prepd. in a format suitable for docking and are to be made freely available to the entire research community (http://www.ccdc.cam.ac.uk).  The performance of the docking program GOLD against the new set is assessed using a variety of protocols.  Relatively unbiased protocols give success rates of approx. 80% for redocking into native structures, but it is possible to get success rates of over 90% with some protocols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFW8sJ7rQ0abVg90H21EOLACvtfcHk0lhh2H3ugO3Pjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmsVymtg%253D%253D&md5=8a5ae20c581f8a242945cced94709443</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm061277y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061277y%26sid%3Dliteratum%253Aachs%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DBrewerton%26aufirst%3DS.%2BC.%26aulast%3DMooij%26aufirst%3DW.%2BT.%2BM.%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26atitle%3DDiverse%252C%2520High-Quality%2520Test%2520Set%2520for%2520the%2520Validation%2520of%2520Protein%25E2%2580%2593Ligand%2520Docking%2520Performance%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D726%26epage%3D741%26doi%3D10.1021%2Fjm061277y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Chisholm, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motherwell, S.</span><span> </span><span class="NLM_article-title">A New Algorithm for Performing Three-Dimensional Searches of the Cambridge Structural Database</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">331</span><span class="NLM_x">–</span> <span class="NLM_lpage">334</span><span class="refDoi"> DOI: 10.1107/S0021889804002924</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1107%2FS0021889804002924" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitVagtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2004&pages=331-334&author=J.+A.+Chisholmauthor=S.+Motherwell&title=A+New+Algorithm+for+Performing+Three-Dimensional+Searches+of+the+Cambridge+Structural+Database&doi=10.1107%2FS0021889804002924"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A new algorithm for performing three-dimensional searches of the Cambridge Structural Database</span></div><div class="casAuthors">Chisholm, James A.; Motherwell, Sam</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">331-334</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A search algorithm, 3DSEARCH, is presented that can readily identify challenging extended chem. queries from three-dimensional mol. crystal structure information.  The program combines substructure search and distance search techniques within a depth-first backtracking algorithm.  Performance metrics are presented for example searches composed of several substructures and several intermol. connections.  Such searches, which are outside the capabilities of current search engines, can now be performed on the entire Cambridge Structural Database with search times of around half an hour.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouEqCGY4cEjbVg90H21EOLACvtfcHk0lhh2H3ugO3Pjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitVagtbY%253D&md5=5b2c4002d3a36708dad10326d1010c71</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0021889804002924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889804002924%26sid%3Dliteratum%253Aachs%26aulast%3DChisholm%26aufirst%3DJ.%2BA.%26aulast%3DMotherwell%26aufirst%3DS.%26atitle%3DA%2520New%2520Algorithm%2520for%2520Performing%2520Three-Dimensional%2520Searches%2520of%2520the%2520Cambridge%2520Structural%2520Database%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2004%26volume%3D37%26spage%3D331%26epage%3D334%26doi%3D10.1107%2FS0021889804002924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">A Solution for the Best Rotation to Relate Two Sets of Vectors</span> <span class="citation_source-journal">Acta Crystallogr., Sect. A: Cryst. Phys., Diffr., Theor. Gen. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1976</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">922</span><span class="NLM_x">–</span> <span class="NLM_lpage">923</span><span class="refDoi"> DOI: 10.1107/S0567739476001873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1107%2FS0567739476001873" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1976&pages=922-923&author=W.+Kabsch&title=A+Solution+for+the+Best+Rotation+to+Relate+Two+Sets+of+Vectors&doi=10.1107%2FS0567739476001873"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0567739476001873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0567739476001873%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DA%2520Solution%2520for%2520the%2520Best%2520Rotation%2520to%2520Relate%2520Two%2520Sets%2520of%2520Vectors%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520A%253A%2520Cryst.%2520Phys.%252C%2520Diffr.%252C%2520Theor.%2520Gen.%2520Crystallogr.%26date%3D1976%26volume%3D32%26spage%3D922%26epage%3D923%26doi%3D10.1107%2FS0567739476001873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Ehmke, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winkler, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banner, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haap, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schweizer, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rottmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaiser, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freymond, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schirmeister, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diederich, F.</span><span> </span><span class="NLM_article-title">Optimization of Triazine Nitriles as Rhodesain Inhibitors: Structure–Activity Relationships, Bioisosteric Imidazopyridine Nitriles, and X-Ray Crystal Structure Analysis with Human Cathepsin L</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">967</span><span class="NLM_x">–</span> <span class="NLM_lpage">975</span><span class="refDoi"> DOI: 10.1002/cmdc.201300112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1002%2Fcmdc.201300112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=23658062" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntlSru7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=967-975&author=V.+Ehmkeauthor=E.+Winklerauthor=D.+W.+Bannerauthor=W.+Haapauthor=W.+B.+Schweizerauthor=M.+Rottmannauthor=M.+Kaiserauthor=C.+Freymondauthor=T.+Schirmeisterauthor=F.+Diederich&title=Optimization+of+Triazine+Nitriles+as+Rhodesain+Inhibitors%3A+Structure%E2%80%93Activity+Relationships%2C+Bioisosteric+Imidazopyridine+Nitriles%2C+and+X-Ray+Crystal+Structure+Analysis+with+Human+Cathepsin+L&doi=10.1002%2Fcmdc.201300112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of Triazine Nitriles as Rhodesain Inhibitors: Structure-Activity Relationships, Bioisosteric Imidazopyridine Nitriles, and X-ray Crystal Structure Analysis with Human Cathepsin L</span></div><div class="casAuthors">Ehmke, Veronika; Winkler, Edwin; Banner, David W.; Haap, Wolfgang; Schweizer, W. Bernd; Rottmann, Matthias; Kaiser, Marcel; Freymond, Celine; Schirmeister, Tanja; Diederich, Francois</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">967-975</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The cysteine protease rhodesain of Trypanosoma brucei parasites causing African sleeping sickness has emerged as a target for the development of new drug candidates.  Based on a triazine nitrile moiety as electrophilic headgroup, optimization studies on the substituents for the S1, S2, and S3 pockets of the enzyme were performed using structure-based design and resulted in inhibitors with inhibition consts. in the single-digit nanomolar range.  Comprehensive structure-activity relationships clarified the binding preferences of the individual pockets of the active site.  The S1 pocket tolerates various substituents with a preference for flexible and basic side chains.  Variation of the S2 substituent led to high-affinity ligands with inhibition consts. down to 2 nM for compds. bearing cyclohexyl substituents.  Systematic investigations on the S3 pocket revealed its potential to achieve high activities with arom. vectors that undergo stacking interactions with the planar peptide backbone forming part of the pocket.  X-ray crystal structure anal. with the structurally related enzyme human cathepsin L confirmed the binding mode of the triazine ligand series as proposed by mol. modeling.  Sub-micromolar inhibition of the proliferation of cultured parasites was achieved for ligands decorated with the best substituents identified through the optimization cycles.  In cell-based assays, the introduction of a basic side chain on the inhibitors resulted in a 35-fold increase in antitrypanosomal activity.  Finally, bioisosteric imidazopyridine nitriles were studied to prevent off-target effects with unselective nucleophiles by decreasing the inherent electrophilicity of the triazine nitrile headgroup.  Using this ligand, the stabilization by intramol. hydrogen bonding of the thioimidate intermediate, formed upon attack of the catalytic cysteine residue, compensates for the lower reactivity of the headgroup.  The imidazopyridine nitrile ligand showed excellent stability toward the thiol nucleophile glutathione in a quant. in vitro assay and fourfold lower cytotoxicity than the parent triazine nitrile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGwTj6sYRPi7Vg90H21EOLACvtfcHk0lh5wYUhy1oLWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntlSru7k%253D&md5=0db889bf80bc6ea9fa3c16bfed80f614</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300112%26sid%3Dliteratum%253Aachs%26aulast%3DEhmke%26aufirst%3DV.%26aulast%3DWinkler%26aufirst%3DE.%26aulast%3DBanner%26aufirst%3DD.%2BW.%26aulast%3DHaap%26aufirst%3DW.%26aulast%3DSchweizer%26aufirst%3DW.%2BB.%26aulast%3DRottmann%26aufirst%3DM.%26aulast%3DKaiser%26aufirst%3DM.%26aulast%3DFreymond%26aufirst%3DC.%26aulast%3DSchirmeister%26aufirst%3DT.%26aulast%3DDiederich%26aufirst%3DF.%26atitle%3DOptimization%2520of%2520Triazine%2520Nitriles%2520as%2520Rhodesain%2520Inhibitors%253A%2520Structure%25E2%2580%2593Activity%2520Relationships%252C%2520Bioisosteric%2520Imidazopyridine%2520Nitriles%252C%2520and%2520X-Ray%2520Crystal%2520Structure%2520Analysis%2520with%2520Human%2520Cathepsin%2520L%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26spage%3D967%26epage%3D975%26doi%3D10.1002%2Fcmdc.201300112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Hardegger, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spinnler, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anselm, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecabert, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stihle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gsell, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thoma, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plancher, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartmann, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banner, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haap, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diederich, F.</span><span> </span><span class="NLM_article-title">Systematic Investigation of Halogen Bonding in Protein–Ligand Interactions</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">314</span><span class="NLM_x">–</span> <span class="NLM_lpage">318</span><span class="refDoi"> DOI: 10.1002/anie.201006781</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1002%2Fanie.201006781" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1alu73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=314-318&author=L.+A.+Hardeggerauthor=B.+Kuhnauthor=B.+Spinnlerauthor=L.+Anselmauthor=R.+Ecabertauthor=M.+Stihleauthor=B.+Gsellauthor=R.+Thomaauthor=J.+Diezauthor=J.+Benzauthor=J.-M.+Plancherauthor=G.+Hartmannauthor=D.+W.+Bannerauthor=W.+Haapauthor=F.+Diederich&title=Systematic+Investigation+of+Halogen+Bonding+in+Protein%E2%80%93Ligand+Interactions&doi=10.1002%2Fanie.201006781"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic Investigation of Halogen Bonding in Protein-Ligand Interactions</span></div><div class="casAuthors">Hardegger, Leo A.; Kuhn, Bernd; Spinnler, Beat; Anselm, Lilli; Ecabert, Robert; Stihle, Martine; Gsell, Bernard; Thoma, Ralf; Diez, Joachim; Benz, Joerg; Plancher, Jean-Marc; Hartmann, Guido; Banner, David W.; Haap, Wolfgang; Diederich, Francois</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">314-318, S314/1-S314/145</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Halogen bonding triggers activity.  Increasing binding affinity was obsd. for a series of covalent human Cathepsin L inhibitors by exchanging an aryl ring H atom with Cl, Br, and I, which undergo halogen bonding with the C=O group of Gly61 in the S3 pocket of the enzyme.  Fluorine, in contrast, strongly avoids halogen bonding.  The strong distance and angle dependence of halogen bonding was confirmed for biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbsbCNMQhVu7Vg90H21EOLACvtfcHk0lh5wYUhy1oLWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1alu73N&md5=1a7f2983a386fd133be0663657fc486b</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fanie.201006781&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201006781%26sid%3Dliteratum%253Aachs%26aulast%3DHardegger%26aufirst%3DL.%2BA.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DSpinnler%26aufirst%3DB.%26aulast%3DAnselm%26aufirst%3DL.%26aulast%3DEcabert%26aufirst%3DR.%26aulast%3DStihle%26aufirst%3DM.%26aulast%3DGsell%26aufirst%3DB.%26aulast%3DThoma%26aufirst%3DR.%26aulast%3DDiez%26aufirst%3DJ.%26aulast%3DBenz%26aufirst%3DJ.%26aulast%3DPlancher%26aufirst%3DJ.-M.%26aulast%3DHartmann%26aufirst%3DG.%26aulast%3DBanner%26aufirst%3DD.%2BW.%26aulast%3DHaap%26aufirst%3DW.%26aulast%3DDiederich%26aufirst%3DF.%26atitle%3DSystematic%2520Investigation%2520of%2520Halogen%2520Bonding%2520in%2520Protein%25E2%2580%2593Ligand%2520Interactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2011%26volume%3D50%26spage%3D314%26epage%3D318%26doi%3D10.1002%2Fanie.201006781" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Shenoy, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sivaraman, J.</span><span> </span><span class="NLM_article-title">Structural Basis for Reversible and Irreversible Inhibition of Human Cathepsin L by Their Respective Dipeptidyl Glyoxal and Diazomethylketone Inhibitors</span> <span class="citation_source-journal">J. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">173</span><span class="NLM_x">, </span> <span class="NLM_fpage">14</span><span class="NLM_x">–</span> <span class="NLM_lpage">19</span><span class="refDoi"> DOI: 10.1016/j.jsb.2010.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1016%2Fj.jsb.2010.09.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2011&pages=14-19&author=R.+T.+Shenoyauthor=J.+Sivaraman&title=Structural+Basis+for+Reversible+and+Irreversible+Inhibition+of+Human+Cathepsin+L+by+Their+Respective+Dipeptidyl+Glyoxal+and+Diazomethylketone+Inhibitors&doi=10.1016%2Fj.jsb.2010.09.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.jsb.2010.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsb.2010.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DShenoy%26aufirst%3DR.%2BT.%26aulast%3DSivaraman%26aufirst%3DJ.%26atitle%3DStructural%2520Basis%2520for%2520Reversible%2520and%2520Irreversible%2520Inhibition%2520of%2520Human%2520Cathepsin%2520L%2520by%2520Their%2520Respective%2520Dipeptidyl%2520Glyoxal%2520and%2520Diazomethylketone%2520Inhibitors%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D2011%26volume%3D173%26spage%3D14%26epage%3D19%26doi%3D10.1016%2Fj.jsb.2010.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Asaad, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bethel, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coulson, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dawson, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerhardt, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grist, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamlin, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, E. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karoutchi, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenny, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morley, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oldham, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rankine, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Augustin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simader, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinbacher, S.</span><span> </span><span class="NLM_article-title">Dipeptidyl Nitrile Inhibitors of Cathepsin L</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">4280</span><span class="NLM_x">–</span> <span class="NLM_lpage">4283</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.05.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1016%2Fj.bmcl.2009.05.071" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=4280-4283&author=N.+Asaadauthor=P.+A.+Bethelauthor=M.+D.+Coulsonauthor=J.+E.+Dawsonauthor=S.+J.+Fordauthor=S.+Gerhardtauthor=M.+Gristauthor=G.+A.+Hamlinauthor=M.+J.+Jamesauthor=E.+V.+Jonesauthor=G.+I.+Karoutchiauthor=P.+W.+Kennyauthor=A.+D.+Morleyauthor=K.+Oldhamauthor=N.+Rankineauthor=D.+Ryanauthor=S.+L.+Wellsauthor=L.+Woodauthor=M.+Augustinauthor=S.+Krappauthor=H.+Simaderauthor=S.+Steinbacher&title=Dipeptidyl+Nitrile+Inhibitors+of+Cathepsin+L&doi=10.1016%2Fj.bmcl.2009.05.071"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.05.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.05.071%26sid%3Dliteratum%253Aachs%26aulast%3DAsaad%26aufirst%3DN.%26aulast%3DBethel%26aufirst%3DP.%2BA.%26aulast%3DCoulson%26aufirst%3DM.%2BD.%26aulast%3DDawson%26aufirst%3DJ.%2BE.%26aulast%3DFord%26aufirst%3DS.%2BJ.%26aulast%3DGerhardt%26aufirst%3DS.%26aulast%3DGrist%26aufirst%3DM.%26aulast%3DHamlin%26aufirst%3DG.%2BA.%26aulast%3DJames%26aufirst%3DM.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BV.%26aulast%3DKaroutchi%26aufirst%3DG.%2BI.%26aulast%3DKenny%26aufirst%3DP.%2BW.%26aulast%3DMorley%26aufirst%3DA.%2BD.%26aulast%3DOldham%26aufirst%3DK.%26aulast%3DRankine%26aufirst%3DN.%26aulast%3DRyan%26aufirst%3DD.%26aulast%3DWells%26aufirst%3DS.%2BL.%26aulast%3DWood%26aufirst%3DL.%26aulast%3DAugustin%26aufirst%3DM.%26aulast%3DKrapp%26aufirst%3DS.%26aulast%3DSimader%26aufirst%3DH.%26aulast%3DSteinbacher%26aufirst%3DS.%26atitle%3DDipeptidyl%2520Nitrile%2520Inhibitors%2520of%2520Cathepsin%2520L%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D4280%26epage%3D4283%26doi%3D10.1016%2Fj.bmcl.2009.05.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Harder, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diederich, F.</span><span> </span><span class="NLM_article-title">Efficient Stacking on Protein Amide Fragments</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">397</span><span class="NLM_x">–</span> <span class="NLM_lpage">404</span><span class="refDoi"> DOI: 10.1002/cmdc.201200512</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1002%2Fcmdc.201200512" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=23355480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVymsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=397-404&author=M.+Harderauthor=B.+Kuhnauthor=F.+Diederich&title=Efficient+Stacking+on+Protein+Amide+Fragments&doi=10.1002%2Fcmdc.201200512"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Stacking on Protein Amide Fragments</span></div><div class="casAuthors">Harder, Michael; Kuhn, Bernd; Diederich, Francois</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">397-404</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The less polar π-surface of protein amide groups is exposed in many receptor binding sites, either as part of the backbone or in Gln/Asn side chains.  Using quantum chem. calcns. and Protein Data Bank (PDB) searches on model systems, we investigate the energetics and geometric preferences for the stacking on amide groups of a large no. of heteroarenes that are relevant to medicinal chem.  From this study, we discern that the stacking energy of an arom. ligand substituent can be improved by: (1) orienting the fragment dipole vector such that it is aligned in an antiparallel fashion with the dipole of the interacting protein amide group, (2) increasing its dipole moment, and (3) decreasing its π-electron d.  These guidelines should be helpful to more rationally exploit this interaction type in future structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEY-6adf4ZHLVg90H21EOLACvtfcHk0li6i9CDBvzv8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVymsLw%253D&md5=4dab202cc8cefb347d0d4bb3a55a941c</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201200512&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201200512%26sid%3Dliteratum%253Aachs%26aulast%3DHarder%26aufirst%3DM.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DDiederich%26aufirst%3DF.%26atitle%3DEfficient%2520Stacking%2520on%2520Protein%2520Amide%2520Fragments%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26spage%3D397%26epage%3D404%26doi%3D10.1002%2Fcmdc.201200512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Hardegger, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spinnler, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anselm, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecabert, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stihle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gsell, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thoma, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plancher, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartmann, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isshiki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikami, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimma, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haap, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banner, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diederich, F.</span><span> </span><span class="NLM_article-title">Halogen Bonding at the Active Sites of Human Cathepsin L and MEK1 Kinase: Efficient Interactions in Different Environments</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">2048</span><span class="NLM_x">–</span> <span class="NLM_lpage">2054</span><span class="refDoi"> DOI: 10.1002/cmdc.201100353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1002%2Fcmdc.201100353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=21898833" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFajtr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=2048-2054&author=L.+A.+Hardeggerauthor=B.+Kuhnauthor=B.+Spinnlerauthor=L.+Anselmauthor=R.+Ecabertauthor=M.+Stihleauthor=B.+Gsellauthor=R.+Thomaauthor=J.+Diezauthor=J.+Benzauthor=J.-M.+Plancherauthor=G.+Hartmannauthor=Y.+Isshikiauthor=K.+Morikamiauthor=N.+Shimmaauthor=W.+Haapauthor=D.+W.+Bannerauthor=F.+Diederich&title=Halogen+Bonding+at+the+Active+Sites+of+Human+Cathepsin+L+and+MEK1+Kinase%3A+Efficient+Interactions+in+Different+Environments&doi=10.1002%2Fcmdc.201100353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Halogen Bonding at the Active Sites of Human Cathepsin L and MEK1 Kinase: Efficient Interactions in Different Environments</span></div><div class="casAuthors">Hardegger, Leo A.; Kuhn, Bernd; Spinnler, Beat; Anselm, Lilli; Ecabert, Robert; Stihle, Martine; Gsell, Bernard; Thoma, Ralf; Diez, Joachim; Benz, Joerg; Plancher, Jean-Marc; Hartmann, Guido; Isshiki, Yoshiaki; Morikami, Kenji; Shimma, Nobuo; Haap, Wolfgang; Banner, David W.; Diederich, Francois</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2048-2054</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">In two series of small-mol. ligands, one inhibiting human cathepsin L (hcatL) and the other MEK1 kinase, biol. affinities were found to strongly increase when an aryl ring of the inhibitors is substituted with the larger halogens Cl, Br, and I, but to decrease upon F substitution.  X-ray co-crystal structure analyses revealed that the higher halides engage in halogen bonding (XB) with a backbone C=O in the S3 pocket of hcatL and in a back pocket of MEK1.  While the S3 pocket is located at the surface of the enzyme, which provides a polar environment, the back pocket in MEK1 is deeply buried in the protein and is of pronounced apolar character.  This study analyzes environmental effects on XB in protein-ligand complexes.  It is hypothesized that energetic gains by XB are predominantly not due to water replacements but originate from direct interactions between the XB donor (Caryl-X) and the XB acceptor (C=O) in the correct geometry.  New x-ray co-crystal structures in the same crystal form (space group P212121) were obtained for aryl chloride, bromide, and iodide ligands bound to hcatL.  These high-resoln. structures reveal that the backbone C=O group of Gly61 in most hcatL co-crystal structures maintains water solvation while engaging in XB.  An aryl-CF3-substituted ligand of hcatL with an unexpectedly high affinity was found to adopt the same binding geometry as the aryl halides, with the CF3 group pointing to the C=O group of Gly61 in the S3 pocket.  In this case, a repulsive F2C-F···O=C contact apparently is energetically overcompensated by other favorable protein-ligand contacts established by the CF3 group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnaNYbZ3KJUbVg90H21EOLACvtfcHk0li6i9CDBvzv8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFajtr7E&md5=12c7cc93b65ba017acee37796811309d</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201100353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201100353%26sid%3Dliteratum%253Aachs%26aulast%3DHardegger%26aufirst%3DL.%2BA.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DSpinnler%26aufirst%3DB.%26aulast%3DAnselm%26aufirst%3DL.%26aulast%3DEcabert%26aufirst%3DR.%26aulast%3DStihle%26aufirst%3DM.%26aulast%3DGsell%26aufirst%3DB.%26aulast%3DThoma%26aufirst%3DR.%26aulast%3DDiez%26aufirst%3DJ.%26aulast%3DBenz%26aufirst%3DJ.%26aulast%3DPlancher%26aufirst%3DJ.-M.%26aulast%3DHartmann%26aufirst%3DG.%26aulast%3DIsshiki%26aufirst%3DY.%26aulast%3DMorikami%26aufirst%3DK.%26aulast%3DShimma%26aufirst%3DN.%26aulast%3DHaap%26aufirst%3DW.%26aulast%3DBanner%26aufirst%3DD.%2BW.%26aulast%3DDiederich%26aufirst%3DF.%26atitle%3DHalogen%2520Bonding%2520at%2520the%2520Active%2520Sites%2520of%2520Human%2520Cathepsin%2520L%2520and%2520MEK1%2520Kinase%253A%2520Efficient%2520Interactions%2520in%2520Different%2520Environments%26jtitle%3DChemMedChem%26date%3D2011%26volume%3D6%26spage%3D2048%26epage%3D2054%26doi%3D10.1002%2Fcmdc.201100353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Roehrig, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Straub, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pohlmann, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lampe, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pernerstorfer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlemmer, K.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinemer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perzborn, E.</span><span> </span><span class="NLM_article-title">Discovery of the Novel Antithrombotic Agent 5-Chloro-N-({(5S)-2-Oxo-3- [4-(3-Oxomorpholin-4-Yl)phenyl]-1,3-Oxazolidin-5-Yl}methyl)thiophene- 2-Carboxamide (BAY 59–7939): An Oral, Direct Factor Xa Inhibitor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">5900</span><span class="NLM_x">–</span> <span class="NLM_lpage">5908</span><span class="refDoi"> DOI: 10.1021/jm050101d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050101d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=5900-5908&author=S.+Roehrigauthor=A.+Straubauthor=J.+Pohlmannauthor=T.+Lampeauthor=J.+Pernerstorferauthor=K.-H.+Schlemmerauthor=P.+Reinemerauthor=E.+Perzborn&title=Discovery+of+the+Novel+Antithrombotic+Agent+5-Chloro-N-%28%7B%285S%29-2-Oxo-3-+%5B4-%283-Oxomorpholin-4-Yl%29phenyl%5D-1%2C3-Oxazolidin-5-Yl%7Dmethyl%29thiophene-+2-Carboxamide+%28BAY+59%E2%80%937939%29%3A+An+Oral%2C+Direct+Factor+Xa+Inhibitor&doi=10.1021%2Fjm050101d"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm050101d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050101d%26sid%3Dliteratum%253Aachs%26aulast%3DRoehrig%26aufirst%3DS.%26aulast%3DStraub%26aufirst%3DA.%26aulast%3DPohlmann%26aufirst%3DJ.%26aulast%3DLampe%26aufirst%3DT.%26aulast%3DPernerstorfer%26aufirst%3DJ.%26aulast%3DSchlemmer%26aufirst%3DK.-H.%26aulast%3DReinemer%26aufirst%3DP.%26aulast%3DPerzborn%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520the%2520Novel%2520Antithrombotic%2520Agent%25205-Chloro-N-%2528%257B%25285S%2529-2-Oxo-3-%2520%255B4-%25283-Oxomorpholin-4-Yl%2529phenyl%255D-1%252C3-Oxazolidin-5-Yl%257Dmethyl%2529thiophene-%25202-Carboxamide%2520%2528BAY%252059%25E2%2580%25937939%2529%253A%2520An%2520Oral%252C%2520Direct%2520Factor%2520Xa%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D5900%26epage%3D5908%26doi%3D10.1021%2Fjm050101d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Salonen, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bucher, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banner, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haap, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mary, J.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuster, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seiler, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schweizer, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diederich, F.</span><span> </span><span class="NLM_article-title">Cation−π Interactions at the Active Site of Factor Xa: Dramatic Enhancement upon Stepwise N-Alkylation of Ammonium Ions</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">811</span><span class="NLM_x">–</span> <span class="NLM_lpage">814</span><span class="refDoi"> DOI: 10.1002/anie.200804695</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1002%2Fanie.200804695" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=811-814&author=L.+M.+Salonenauthor=C.+Bucherauthor=D.+W.+Bannerauthor=W.+Haapauthor=J.-L.+Maryauthor=J.+Benzauthor=O.+Kusterauthor=P.+Seilerauthor=W.+B.+Schweizerauthor=F.+Diederich&title=Cation%E2%88%92%CF%80+Interactions+at+the+Active+Site+of+Factor+Xa%3A+Dramatic+Enhancement+upon+Stepwise+N-Alkylation+of+Ammonium+Ions&doi=10.1002%2Fanie.200804695"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fanie.200804695&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200804695%26sid%3Dliteratum%253Aachs%26aulast%3DSalonen%26aufirst%3DL.%2BM.%26aulast%3DBucher%26aufirst%3DC.%26aulast%3DBanner%26aufirst%3DD.%2BW.%26aulast%3DHaap%26aufirst%3DW.%26aulast%3DMary%26aufirst%3DJ.-L.%26aulast%3DBenz%26aufirst%3DJ.%26aulast%3DKuster%26aufirst%3DO.%26aulast%3DSeiler%26aufirst%3DP.%26aulast%3DSchweizer%26aufirst%3DW.%2BB.%26aulast%3DDiederich%26aufirst%3DF.%26atitle%3DCation%25E2%2588%2592%25CF%2580%2520Interactions%2520at%2520the%2520Active%2520Site%2520of%2520Factor%2520Xa%253A%2520Dramatic%2520Enhancement%2520upon%2520Stepwise%2520N-Alkylation%2520of%2520Ammonium%2520Ions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2009%26volume%3D48%26spage%3D811%26epage%3D814%26doi%3D10.1002%2Fanie.200804695" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Böhm, H.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flohr, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span> </span><span class="NLM_article-title">Scaffold Hopping</span> <span class="citation_source-journal">Drug Discovery Today: Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">217</span><span class="NLM_x">–</span> <span class="NLM_lpage">224</span><span class="refDoi"> DOI: 10.1016/j.ddtec.2004.10.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1016%2Fj.ddtec.2004.10.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=24981488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A280%3ADC%252BC2cfosVaktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2004&pages=217-224&author=H.-J.+B%C3%B6hmauthor=A.+Flohrauthor=M.+Stahl&title=Scaffold+Hopping&doi=10.1016%2Fj.ddtec.2004.10.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Scaffold hopping</span></div><div class="casAuthors">Bohm Hans-Joachim; Flohr Alexander; Stahl Martin</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">217-24</span>
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    </div><div class="casAbstract">The aim of scaffold hopping is to discover structurally novel compounds starting from known active compounds by modifying the central core structure of the molecule.  Scaffold hopping is a central task of modern medicinal chemistry requiring a multitude of techniques, which are discussed in this article.  Their application has led to several molecules with chemically completely different core structures, and yet binding to the same receptor.  Computational approaches for scaffold hopping highlight the challenges of the field that are still unsolved.:</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSkmF4guWxLWsKL0FlyGHDofW6udTcc2eYIjHlryRWOjrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cfosVaktQ%253D%253D&md5=9a90f37dd2accf66bfd379f9ec602e03</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2004.10.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2004.10.009%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25B6hm%26aufirst%3DH.-J.%26aulast%3DFlohr%26aufirst%3DA.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DScaffold%2520Hopping%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2004%26volume%3D1%26spage%3D217%26epage%3D224%26doi%3D10.1016%2Fj.ddtec.2004.10.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Sun, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tawa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallqvist, A.</span><span> </span><span class="NLM_article-title">Classification of Scaffold-Hopping Approaches</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">310</span><span class="NLM_x">–</span> <span class="NLM_lpage">324</span><span class="refDoi"> DOI: 10.1016/j.drudis.2011.10.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1016%2Fj.drudis.2011.10.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=22056715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFyrt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=310-324&author=H.+Sunauthor=G.+Tawaauthor=A.+Wallqvist&title=Classification+of+Scaffold-Hopping+Approaches&doi=10.1016%2Fj.drudis.2011.10.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of scaffold-hopping approaches</span></div><div class="casAuthors">Sun, Hongmao; Tawa, Gregory; Wallqvist, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">7-8</span>),
    <span class="NLM_cas:pages">310-324</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The general goal of drug discovery is to identify novel compds. that are active against a preselected biol. target with acceptable pharmacol. properties defined by marketed drugs.  Scaffold hopping has been widely applied by medicinal chemists to discover equipotent compds. with novel backbones that have improved properties.  In this article we classify scaffold hopping into four major categories, namely heterocycle replacements, ring opening or closure, peptidomimetics and topol.-based hopping.  We review the structural diversity of original and final scaffolds with respect to each category.  We discuss the advantages and limitations of small, medium and large-step scaffold hopping.  Finally, we summarize software that is frequently used to facilitate different kinds of scaffold-hopping methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAONaJ8Be0x7Vg90H21EOLACvtfcHk0lgeFXt1ZrwsLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFyrt7k%253D&md5=5f1143cc255c81b88ea4f83ac9f8efc6</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.10.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.10.024%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DH.%26aulast%3DTawa%26aufirst%3DG.%26aulast%3DWallqvist%26aufirst%3DA.%26atitle%3DClassification%2520of%2520Scaffold-Hopping%2520Approaches%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2012%26volume%3D17%26spage%3D310%26epage%3D324%26doi%3D10.1016%2Fj.drudis.2011.10.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Maass, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulz-Gasch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span> </span><span class="NLM_article-title">Recore: A Fast and Versatile Method for Scaffold Hopping Based on Small Molecule Crystal Structure Conformations</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">390</span><span class="NLM_x">–</span> <span class="NLM_lpage">399</span><span class="refDoi"> DOI: 10.1021/ci060094h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci060094h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhvVWjtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=390-399&author=P.+Maassauthor=T.+Schulz-Gaschauthor=M.+Stahlauthor=M.+Rarey&title=Recore%3A+A+Fast+and+Versatile+Method+for+Scaffold+Hopping+Based+on+Small+Molecule+Crystal+Structure+Conformations&doi=10.1021%2Fci060094h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Recore: A Fast and Versatile Method for Scaffold Hopping Based on Small Molecule Crystal Structure Conformations</span></div><div class="casAuthors">Maass, Patrick; Schulz-Gasch, Tanja; Stahl, Martin; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">390-399</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Replacing central elements of known active structures is a common procedure to enter new compd. classes.  Different computational methods have already been developed to help with this task, varying in the description of possible replacements, the query input, and the similarity measure used.  In this paper, a novel approach for scaffold replacement and a corresponding software tool, called Recore, is introduced.  In contrast to prior methods, the authors main objective was to combine the following three properties in one tool: to avoid structures with strained conformations, to enable the exploration of large search spaces, and to allow interactive use through short response times.  The authors introduce a new technique employing 3D fragments generated by combinatorial enumeration of cuts.  It allows focusing on fragments suitable for scaffold replacement while retaining conformational information of the corresponding crystal structures.  Based on this idea, the authors present an algorithm utilizing a geometric rank searching approach.  Given a geometric arrangement of two or three exit vectors and addnl. pharmacophore features, the algorithm finds fragments fulfilling all these constraints ordered by increasing deviation from the query constraints.  For the validation of the approach, three different drug design scenarios have been used.  The results obtained show that the authors approach is able to propose new valid scaffold topologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpphxap_w-jZbVg90H21EOLACvtfcHk0liTVLRDUrgoAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhvVWjtbk%253D&md5=e9d4e277da49f4ffe2c21472990c2387</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fci060094h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci060094h%26sid%3Dliteratum%253Aachs%26aulast%3DMaass%26aufirst%3DP.%26aulast%3DSchulz-Gasch%26aufirst%3DT.%26aulast%3DStahl%26aufirst%3DM.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DRecore%253A%2520A%2520Fast%2520and%2520Versatile%2520Method%2520for%2520Scaffold%2520Hopping%2520Based%2520on%2520Small%2520Molecule%2520Crystal%2520Structure%2520Conformations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D390%26epage%3D399%26doi%3D10.1021%2Fci060094h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Lauri, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartlett, P. A.</span><span> </span><span class="NLM_article-title">CAVEAT: A Program to Facilitate the Design of Organic Molecules</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">51</span><span class="NLM_x">–</span> <span class="NLM_lpage">66</span><span class="refDoi"> DOI: 10.1007/BF00124349</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1007%2FBF00124349" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1994&pages=51-66&author=G.+Lauriauthor=P.+A.+Bartlett&title=CAVEAT%3A+A+Program+to+Facilitate+the+Design+of+Organic+Molecules&doi=10.1007%2FBF00124349"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1007%2FBF00124349&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00124349%26sid%3Dliteratum%253Aachs%26aulast%3DLauri%26aufirst%3DG.%26aulast%3DBartlett%26aufirst%3DP.%2BA.%26atitle%3DCAVEAT%253A%2520A%2520Program%2520to%2520Facilitate%2520the%2520Design%2520of%2520Organic%2520Molecules%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1994%26volume%3D8%26spage%3D51%26epage%3D66%26doi%3D10.1007%2FBF00124349" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Patel, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shuai, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinges, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pliushchev, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monzon, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagaw, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engstrom, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerdesky, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Longenecker, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Judge, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qin, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imade, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stolarik, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beno, D. W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brune, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chovan, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sham, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Link, J. T.</span><span> </span><span class="NLM_article-title">Discovery of Adamantane Ethers as Inhibitors of 11β-HSD-1: Synthesis and Biological Evaluation</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">750</span><span class="NLM_x">–</span> <span class="NLM_lpage">755</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2006.10.074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1016%2Fj.bmcl.2006.10.074" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=750-755&author=J.+R.+Patelauthor=Q.+Shuaiauthor=J.+Dingesauthor=M.+Winnauthor=M.+Pliushchevauthor=S.+Fungauthor=K.+Monzonauthor=W.+Chiouauthor=J.+Wangauthor=L.+Panauthor=S.+Wagawauthor=K.+Engstromauthor=F.+A.+Kerdeskyauthor=K.+Longeneckerauthor=R.+Judgeauthor=W.+Qinauthor=H.+M.+Imadeauthor=D.+Stolarikauthor=D.+W.+A.+Benoauthor=M.+Bruneauthor=L.+E.+Chovanauthor=H.+L.+Shamauthor=P.+Jacobsonauthor=J.+T.+Link&title=Discovery+of+Adamantane+Ethers+as+Inhibitors+of+11%CE%B2-HSD-1%3A+Synthesis+and+Biological+Evaluation&doi=10.1016%2Fj.bmcl.2006.10.074"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2006.10.074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2006.10.074%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DJ.%2BR.%26aulast%3DShuai%26aufirst%3DQ.%26aulast%3DDinges%26aufirst%3DJ.%26aulast%3DWinn%26aufirst%3DM.%26aulast%3DPliushchev%26aufirst%3DM.%26aulast%3DFung%26aufirst%3DS.%26aulast%3DMonzon%26aufirst%3DK.%26aulast%3DChiou%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DPan%26aufirst%3DL.%26aulast%3DWagaw%26aufirst%3DS.%26aulast%3DEngstrom%26aufirst%3DK.%26aulast%3DKerdesky%26aufirst%3DF.%2BA.%26aulast%3DLongenecker%26aufirst%3DK.%26aulast%3DJudge%26aufirst%3DR.%26aulast%3DQin%26aufirst%3DW.%26aulast%3DImade%26aufirst%3DH.%2BM.%26aulast%3DStolarik%26aufirst%3DD.%26aulast%3DBeno%26aufirst%3DD.%2BW.%2BA.%26aulast%3DBrune%26aufirst%3DM.%26aulast%3DChovan%26aufirst%3DL.%2BE.%26aulast%3DSham%26aufirst%3DH.%2BL.%26aulast%3DJacobson%26aufirst%3DP.%26aulast%3DLink%26aufirst%3DJ.%2BT.%26atitle%3DDiscovery%2520of%2520Adamantane%2520Ethers%2520as%2520Inhibitors%2520of%252011%25CE%25B2-HSD-1%253A%2520Synthesis%2520and%2520Biological%2520Evaluation%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D750%26epage%3D755%26doi%3D10.1016%2Fj.bmcl.2006.10.074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Tu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powers, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ursu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudom, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meininger, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeGraffenreid, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaen, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labelle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamamoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, N. P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span> </span><span class="NLM_article-title">Distinctive Molecular Inhibition Mechanisms for Selective Inhibitors of Human 11β-Hydroxysteroid Dehydrogenase Type 1</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">8922</span><span class="NLM_x">–</span> <span class="NLM_lpage">8931</span><span class="refDoi"> DOI: 10.1016/j.bmc.2008.08.065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1016%2Fj.bmc.2008.08.065" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=8922-8931&author=H.+Tuauthor=J.+P.+Powersauthor=J.+Liuauthor=S.+Ursuauthor=A.+Sudomauthor=X.+Yanauthor=H.+Xuauthor=D.+Meiningerauthor=M.+DeGraffenreidauthor=X.+Heauthor=J.+C.+Jaenauthor=D.+Sunauthor=M.+Labelleauthor=H.+Yamamotoauthor=B.+Shanauthor=N.+P.+C.+Walkerauthor=Z.+Wang&title=Distinctive+Molecular+Inhibition+Mechanisms+for+Selective+Inhibitors+of+Human+11%CE%B2-Hydroxysteroid+Dehydrogenase+Type+1&doi=10.1016%2Fj.bmc.2008.08.065"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2008.08.065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2008.08.065%26sid%3Dliteratum%253Aachs%26aulast%3DTu%26aufirst%3DH.%26aulast%3DPowers%26aufirst%3DJ.%2BP.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DUrsu%26aufirst%3DS.%26aulast%3DSudom%26aufirst%3DA.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DMeininger%26aufirst%3DD.%26aulast%3DDeGraffenreid%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DX.%26aulast%3DJaen%26aufirst%3DJ.%2BC.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DLabelle%26aufirst%3DM.%26aulast%3DYamamoto%26aufirst%3DH.%26aulast%3DShan%26aufirst%3DB.%26aulast%3DWalker%26aufirst%3DN.%2BP.%2BC.%26aulast%3DWang%26aufirst%3DZ.%26atitle%3DDistinctive%2520Molecular%2520Inhibition%2520Mechanisms%2520for%2520Selective%2520Inhibitors%2520of%2520Human%252011%25CE%25B2-Hydroxysteroid%2520Dehydrogenase%2520Type%25201%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2008%26volume%3D16%26spage%3D8922%26epage%3D8931%26doi%3D10.1016%2Fj.bmc.2008.08.065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Furet, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caravatti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guagnano, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schoepfer, J.</span><span> </span><span class="NLM_article-title">Entry into a New Class of Protein Kinase Inhibitors by Pseudo Ring Design</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">897</span><span class="NLM_x">–</span> <span class="NLM_lpage">900</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2007.12.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=10.1016%2Fj.bmcl.2007.12.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=18248988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFWksb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=897-900&author=P.+Furetauthor=G.+Caravattiauthor=V.+Guagnanoauthor=M.+Langauthor=T.+Meyerauthor=J.+Schoepfer&title=Entry+into+a+New+Class+of+Protein+Kinase+Inhibitors+by+Pseudo+Ring+Design&doi=10.1016%2Fj.bmcl.2007.12.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Entry into a new class of protein kinase inhibitors by pseudo ring design</span></div><div class="casAuthors">Furet, Pascal; Caravatti, Giorgio; Guagnano, Vito; Lang, Marc; Meyer, Thomas; Schoepfer, Joseph</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">897-900</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A pyrimidin-4-yl-urea motif forming a pseudo ring by intramol. hydrogen bonding has been designed to mimic the pyrido[2,3-d]pyrimidin-7-one core structure of a well-established class of protein kinase inhibitors.  Potent inhibition of a no. of protein kinases was obtained with the first prototype compd. synthesized to probe the design concept.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5MZZKS4I03LVg90H21EOLACvtfcHk0lgMwv_1TiuyBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFWksb0%253D&md5=989b4bb7f1d5177be45d4edf31290de5</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.12.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.12.041%26sid%3Dliteratum%253Aachs%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DCaravatti%26aufirst%3DG.%26aulast%3DGuagnano%26aufirst%3DV.%26aulast%3DLang%26aufirst%3DM.%26aulast%3DMeyer%26aufirst%3DT.%26aulast%3DSchoepfer%26aufirst%3DJ.%26atitle%3DEntry%2520into%2520a%2520New%2520Class%2520of%2520Protein%2520Kinase%2520Inhibitors%2520by%2520Pseudo%2520Ring%2520Design%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D897%26epage%3D900%26doi%3D10.1016%2Fj.bmcl.2007.12.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2xu1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2xu1','PDB','2xu1'); return false;">PDB: 2xu1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3dm2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3dm2','PDB','3dm2'); return false;">PDB: 3dm2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4klb" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4klb','PDB','4klb'); return false;">PDB: 4klb</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2irw" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2irw','PDB','2irw'); return false;">PDB: 2irw</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3d5q" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3d5q','PDB','3d5q'); return false;">PDB: 3d5q</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i16"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01756">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_54687"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b01756">10.1021/acs.jmedchem.5b01756</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Description of the Cartesian point onto sphere overlay method, assessment of the pharmacophore search algorithm, feature definitions, and the pharmacophore models presented in this manuscript (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01756/suppl_file/jm5b01756_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01756/suppl_file/jm5b01756_si_001.pdf">jm5b01756_si_001.pdf (837.51 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-9%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01756%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b01756" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677b110749b23de3","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
