
<html lang="en"     class="pb-page"  data-request-id="74d00528-92ca-499a-9df9-d5b1e9f1f29f"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2021.64.issue-5;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.0c02227"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Decision Making in Structure-Based Drug Discovery: Visual Inspection of Docking Results" /></meta><meta name="dc.Creator" content="André  Fischer" /></meta><meta name="dc.Creator" content="Martin  Smieško" /></meta><meta name="dc.Creator" content="Manuel  Sellner" /></meta><meta name="dc.Creator" content="Markus A.  Lill" /></meta><meta name="dc.Description" content="Molecular docking is a computational method widely used in drug discovery. Due to the inherent inaccuracies of molecular docking, visual inspection of binding modes is a crucial routine in the deci..." /></meta><meta name="Description" content="Molecular docking is a computational method widely used in drug discovery. Due to the inherent inaccuracies of molecular docking, visual inspection of binding modes is a crucial routine in the deci..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 22, 2021" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02227" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02227" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02227" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02227" /></link>
        
    
    

<title>Decision Making in Structure-Based Drug Discovery: Visual Inspection of Docking Results | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02227" /></meta><meta property="og:title" content="Decision Making in Structure-Based Drug Discovery: Visual Inspection of Docking Results" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/large/jm0c02227_0005.jpeg" /></meta><meta property="og:description" content="Molecular docking is a computational method widely used in drug discovery. Due to the inherent inaccuracies of molecular docking, visual inspection of binding modes is a crucial routine in the decision making process of computational medicinal chemists. Despite its apparent importance for medicinal chemistry projects, guidelines for the visual docking pose assessment have been hardly discussed in the literature. Here, we review the medicinal chemistry literature with the aim of identifying consistent principles for visual inspection, highlighting cases of its successful application, and discussing its limitations. In this context, we conducted a survey reaching experts in both academia and the pharmaceutical industry, which also included a challenge to distinguish native from incorrect poses. We were able to collect 93 expert opinions that offer valuable insights into visually supported decision-making processes. This perspective shall motivate discussions among experienced computational medicinal chemists and guide young scientists new to the field to stratify their compounds." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02227"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02227">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02227&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02227&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02227&amp;href=/doi/10.1021/acs.jmedchem.0c02227" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 2489-2500</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c02176" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.0c01491" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Decision Making in Structure-Based Drug Discovery: Visual Inspection of Docking Results</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">André Fischer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">André Fischer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Computational Pharmacy, Department of Pharmaceutical Sciences, University of Basel, Klingelbergstrasse 61, 4056 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andr%C3%A9++Fischer">André Fischer</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath1">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Martin Smieško</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martin Smieško</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Computational Pharmacy, Department of Pharmaceutical Sciences, University of Basel, Klingelbergstrasse 61, 4056 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martin++Smie%C5%A1ko">Martin Smieško</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath2">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2758-2680" title="Orcid link">http://orcid.org/0000-0003-2758-2680</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Manuel Sellner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Manuel Sellner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Computational Pharmacy, Department of Pharmaceutical Sciences, University of Basel, Klingelbergstrasse 61, 4056 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Manuel++Sellner">Manuel Sellner</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath3">View Biography</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Markus A. Lill</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Markus A. Lill</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Computational Pharmacy, Department of Pharmaceutical Sciences, University of Basel, Klingelbergstrasse 61, 4056 Basel, Switzerland</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#acc1cddec7d9df82c0c5c0c0ecd9c2c5cecddf82cfc4"><span class="__cf_email__" data-cfemail="e08d81928b9593ce8c898c8ca0958e89828193ce8388">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Markus+A.++Lill">Markus A. Lill</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath4">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3023-5188" title="Orcid link">http://orcid.org/0000-0003-3023-5188</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02227&amp;href=/doi/10.1021%2Facs.jmedchem.0c02227" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 2489–2500</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 22, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>23 December 2020</li><li><span class="item_label"><b>Published</b> online</span>22 February 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 March 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02227" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02227</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2489%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DAndr%25C3%25A9%2BFischer%252C%2BMartin%2BSmie%25C5%25A1ko%252C%2BManuel%2BSellner%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D5%26contentID%3Dacs.jmedchem.0c02227%26title%3DDecision%2BMaking%2Bin%2BStructure-Based%2BDrug%2BDiscovery%253A%2BVisual%2BInspection%2Bof%2BDocking%2BResults%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2500%26publicationDate%3DMarch%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02227"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">7984</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">3</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02227" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Decision Making in Structure-Based Drug Discovery: Visual Inspection of Docking Results&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;André&quot;,&quot;last_name&quot;:&quot;Fischer&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;Smieško&quot;},{&quot;first_name&quot;:&quot;Manuel&quot;,&quot;last_name&quot;:&quot;Sellner&quot;},{&quot;first_name&quot;:&quot;Markus&quot;,&quot;last_name&quot;:&quot;A. Lill&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;22&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;2489-2500&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02227&quot;},&quot;abstract&quot;:&quot;Molecular docking is a computational method widely used in drug discovery. Due to the inherent inaccuracies of molecular docking, visual inspection of binding modes is a crucial routine in the decision making process of computational medicinal chemists. Despite its apparent importance for medicinal chemistry projects, guidelines for the visual docking pose assessment have been hardly discussed in the literature. Here, we review the medicinal chemistry literature with the aim of identifying consistent principles for visual inspection, highlighting cases of its successful application, and discussing its limitations. In this context, we conducted a survey reaching experts in both academia and the pharmaceutical industry, which also included a challenge to distinguish native from incorrect poses. We were able to collect 93 expert opinions that offer valuable insights into visually supported decision-making processes. This perspective shall motivate discussions among experienced computational medicinal chemists an&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02227&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02227" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02227&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02227" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02227&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02227" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02227&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02227&amp;href=/doi/10.1021/acs.jmedchem.0c02227" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02227" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02227" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02227%26sid%3Dliteratum%253Aachs%26pmid%3D33617246%26genre%3Darticle%26aulast%3DFischer%26date%3D2021%26atitle%3DDecision%2BMaking%2Bin%2BStructure-Based%2BDrug%2BDiscovery%253A%2BVisual%2BInspection%2Bof%2BDocking%2BResults%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D5%26spage%3D2489%26epage%3D2500%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292580" title="Binding modes">Binding modes</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291984" title="Molecular mechanics">Molecular mechanics</a>,</li><li><a href="/action/doSearch?ConceptID=291763" title="Noncovalent interactions">Noncovalent interactions</a>,</li><li><a href="/action/doSearch?ConceptID=292474" title="Structural bioinformatics">Structural bioinformatics</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/jmcmar.2021.64.issue-5/20210311/jmcmar.2021.64.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/medium/jm0c02227_0005.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/large/jm0c02227_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02227&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Molecular docking is a computational method widely used in drug discovery. Due to the inherent inaccuracies of molecular docking, visual inspection of binding modes is a crucial routine in the decision making process of computational medicinal chemists. Despite its apparent importance for medicinal chemistry projects, guidelines for the visual docking pose assessment have been hardly discussed in the literature. Here, we review the medicinal chemistry literature with the aim of identifying consistent principles for visual inspection, highlighting cases of its successful application, and discussing its limitations. In this context, we conducted a survey reaching experts in both academia and the pharmaceutical industry, which also included a challenge to distinguish native from incorrect poses. We were able to collect 93 expert opinions that offer valuable insights into visually supported decision-making processes. This perspective shall motivate discussions among experienced computational medicinal chemists and guide young scientists new to the field to stratify their compounds.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32542" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32542" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Computational procedures such as structure-based virtual screening of large digital compound libraries and the evaluation of ligand binding modes have become an essential component in modern drug discovery.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> They may not only provide insights into ligands that potentially bind to a particular target but also assist in the interpretation of experimental observations such as structure–activity relationships.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Thus, structure-based computational methods can be applied in various stages of drug discovery and development. The popularity of virtual screening originates from its cost-effectiveness, its potential to identify novel chemotypes, and mechanistic insights into ligand–protein interactions.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> One of the primary methods applied in structure-based drug discovery is molecular docking. In molecular docking, favorable ligand–protein configurations are identified yielding so-called ligand poses. Their interaction with the protein is quantified in a process termed scoring.<a onclick="showRef(event, 'ref1 ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref1 ref5 ref6 ref7">(1,5−7)</a> To achieve the necessary efficiency of molecular docking, the quantification of ligand–protein interactions is based on a simplistic scoring function. This simplified modeling of protein–ligand interactions is a substantial limitation of the methodology, which is manifested by inaccuracies in pose ranking and typically poor performance in the prediction of absolute or relative binding free energies. Furthermore, correct sampling of ligand poses can be limited by induced-fit effects as well as distinct conformational states of the protein, which is often treated as a rigid body, leading to imprecisions beyond those imposed by scoring functions.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> In addition, intramolecular interactions are sometimes ill-modeled as highlighted by the occurrence of poses with, for example, twisted amide groups, intraligand clashes, or generally unrealistic three-dimensional (3D) structures.<a onclick="showRef(event, 'ref3 ref4 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref9 ref10">(3,4,9,10)</a> Another factor essential for the performance of molecular docking is the quality of the input structures. Common pitfalls and caveats include the protonation state and tautomeric forms of the ligand, as well as peculiar protonation states of acidic and basic residues and flips of glutamine, asparagine, and histidine residues in the protein binding site. In virtual screening applications, the methodology frequently produces a large number of false-positive hits, limiting its applicability as a stand-alone platform.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Even though much effort has been spent to improve results of molecular docking, e.g., by running molecular dynamics (MD) simulations followed by molecular mechanics/generalized Born surface area (MM/GBSA), alchemical binding free energy calculations,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> or modern machine learning methods, the inspection of the docking results by a specialist, especially regarding the predicted binding modes, is of major importance for a successful hit identification or lead optimization.<a onclick="showRef(event, 'ref4 ref6 ref10 ref12 ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref4 ref6 ref10 ref12 ref13 ref14 ref15">(4,6,10,12−15)</a> A recent review reported approximately 50% of 250 virtual screening publications to conduct visual inspection in compound prioritization.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> It was suggested that any person with proper experience can visually evaluate the quality of a docking pose better than any available scoring function.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Still, several computational methods and tools have been developed to assist the decision making. They employ techniques such as interaction fingerprints,<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> scaffold docking,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> or the comparison of binding modes obtained for multiple congeneric compounds<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> to assist in the decision making process. Some of these tools have been only recently developed and are available in the form of a user-friendly web server.</div><div class="NLM_p">As we will describe below, visual inspection of docking results has been used in several successful discovery projects.<a onclick="showRef(event, 'ref3 ref5 ref9 ref21 ref22 ref23 ref24 ref25 ref26 ref27 ref28 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref3 ref5 ref9 ref21 ref22 ref23 ref24 ref25 ref26 ref27 ref28 ref29 ref30 ref31">(3,5,9,21−31)</a> Additionally, visual inspection was also applied during the validation of docking protocols, in particular to evaluate pose prediction by redocking or cross-docking to supplement standard measures such as the root-mean-square deviation (RMSD) and native contacts.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32,33)</a> Unfortunately, most of those studies did only vaguely specify applied criteria or did not mention them at all.<a onclick="showRef(event, 'ref9 ref23 ref26 ref27 ref28 ref29 ref34'); return false;" href="javascript:void(0);" class="ref ref9 ref23 ref26 ref27 ref28 ref29 ref34">(9,23,26−29,34)</a> A recent review focused on decision making in medicinal chemistry in the evaluation of large data sets with respect to potency, selectivity, pharmacokinetics, structural biology, and computational results. With regard to compound prioritization based on such data sets, the authors reported a considerable disagreement among medicinal chemists, presumably arising from their different experience, background, and training.<a onclick="showRef(event, 'ref13 ref35'); return false;" href="javascript:void(0);" class="ref ref13 ref35">(13,35)</a> This discrepancy among different researchers may also exist for visual evaluation of docking poses. Therefore, there is a need to comprehensively review applied criteria to identify consensus guidelines for the visual inspection of ligand binding modes.</div><div class="NLM_p last">Here, we reviewed medicinal and computational chemistry publications, in which visual inspection was applied to complement molecular docking and pose scoring in virtual screening campaigns. We report its successful applications, its limitations, and its direct impact on the accuracy of predictions in compound prioritization. In addition, we conducted a survey by addressing experts in computational medicinal chemistry to identify consensus and disagreement among criteria used for the visual inspection of docking poses. This perspective aims to provide a foundation for a broader discussion in the community for the utility and best strategies for visual inspection of docking results and to offer a guidance to new or young researchers in the field.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Criteria during Visual Inspection</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10306" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10306" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">One of the most frequently assessed criteria in visual inspection is the steric complementarity of a compound and its binding site (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Isothermal titration calorimetry experiments have revealed a correlation between the buried nonpolar surface and the binding free energy of a ligand.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Furthermore, in rational structure-based lead optimization, a partial lack of complementarity can guide the expansion or modification of the ligand. In some studies, shape complementarity was further differentiated into electrostatic and hydrophobic complementarity adding physicochemical properties of protein and ligand in the visual inspection, thus providing a more detailed picture of the interactions in a complex.<a onclick="showRef(event, 'ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38">(36−38)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Criteria Used in Successful Applications of Visual Inspection<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">considered criteria</th><th class="colsep0 rowsep0" align="center">targets</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">structural novelty, hydrogen bonding network, interactions with specific residues,<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a> complementarity, ligand flexibility, water interactions, ligand strain, hydrophobic interactions</td><td class="colsep0 rowsep0" align="left">purine nucleoside phosphorylase<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">binding mode consensus,<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a> ligand protonation, tautomers, complementarity, interactions with specific residues<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">aromatase<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unique interactions,<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> interactions with specific residues,<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a> ligand flexibility, comparison to crystal structures, structural diversity</td><td class="colsep0 rowsep0" align="left">κ-opioid receptor<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">interactions with specific residues,<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a> hydrophobic interactions, ligand strain</td><td class="colsep0 rowsep0" align="left">RANKL receptor<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">interactions with specific residues,<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a> complementarity, hydrophobic interactions, π-stacking, hydrogen bonding network, water interactions, ligand strain</td><td class="colsep0 rowsep0" align="left">ecto-5′-nucleotidase<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">distorted ligand geometry,<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a> ligand strain, nature of solvent-exposed groups, electrostatic repulsion, unsatisfied ligand heteroatoms, hydrophobic interactions, π-stacking, cation−π interactions</td><td class="colsep0 rowsep0" align="left">enoyl-acyl-carrier protein reductase<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">structural novelty, hydrogen bonding network, solubility</td><td class="colsep0 rowsep0" align="left">enoyl-acyl-carrier protein<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unsatisfied ligand heteroatoms, ligand flexibility, distorted ligand geometry, hydrogen bonding network, complementarity</td><td class="colsep0 rowsep0" align="left">EphB4 kinase<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">complementarity, interactions with specific residues,<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a> hydrogen bonding network, hydrophobic interactions, π-stacking, binding mode consensus<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">γ-glutamylcysteine synthase<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">complementarity, binding mode consensus,<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a> improper atom types, hydrophilic–hydrophobic mismatch, nature of solvent-exposed groups</td><td class="colsep0 rowsep0" align="left">Bcl-xL<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">complementarity, π-stacking, hydrogen bonding network interactions with specific residues<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">thymidine monophosphate kinase<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">structural diversity, hydrophobic interactions, hydrogen bonding network, complementarity</td><td class="colsep0 rowsep0" align="left">β2-adrenergic receptor<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">complementarity, structural diversity, ligand flexibility</td><td class="colsep0 rowsep0" align="left">aldose reductase<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">interactions with specific residues,<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a> π-stacking, hydrophobic interactions, hydrogen bonding network</td><td class="colsep0 rowsep0" align="left"><i>N</i>-myristoyltransferase<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Studies in which bioactive compounds were discovered and the criteria that were applied during visual inspection.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Interaction with specific amino acids or cofactors.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Consensus among binding modes of multiple docked ligands.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">Interactions not present in homologous targets.</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">Distorted amides, esters, and other functional groups.</p></div></div></div><div class="NLM_p">Nearly all studies we assessed reported the inspection of hydrogen bonds (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) between the ligand and protein. Due to their relative strength and clearly defined geometry, hydrogen bonds are easy to visually identify and are important factors for binding affinity and selectivity.<a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41,42)</a> Frequently, hydrogen bonding networks with specific binding site residues, often in comparison to co-crystallized ligands or other ligands with a similar chemotype,<a onclick="showRef(event, 'ref28 ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref28 ref43 ref44">(28,43,44)</a> were considered during inspection. For example, in the design of enzyme inhibitors, specific interactions with catalytic residues are important for potent inhibition of the catalytic reaction and for resistance against potential mutations.<a onclick="showRef(event, 'ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref45 ref46">(45,46)</a> Generally, the strength, the stability, and thus the final contribution of hydrogen bonds to binding affinity can vary significantly on the basis of the environment. Hydrogen bonds in a hydrophobic protein environment typically contribute more favorably to the free energy of binding than hydrogen bonds in solvent-accessible regions due to significant differences in desolvation free energy and attenuation of its electrostatic component. In particular, this is observed for charge-assisted hydrogen bonds, also termed salt bridges.<a onclick="showRef(event, 'ref2 ref47'); return false;" href="javascript:void(0);" class="ref ref2 ref47">(2,47)</a> The contributions of salt bridges on the solvent-exposed surface of a binding site to the binding free energy, for example, are often negligible due to the high desolvation cost of the charged protein residue.<a onclick="showRef(event, 'ref2 ref48'); return false;" href="javascript:void(0);" class="ref ref2 ref48">(2,48)</a> Hydrogen bonds to the protein backbone are generally stronger and often less affected by side chain mutations compared to hydrogen bonds formed with side chain atoms.<a onclick="showRef(event, 'ref2 ref7'); return false;" href="javascript:void(0);" class="ref ref2 ref7">(2,7)</a> The loss of entropy of the protein is typically smaller for a backbone hydrogen bond because side chains are inherently more flexible.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> The rigidification of the ligand–protein complex upon formation of a hydrogen bond is coupled to reduced ligand and protein mobility and, therefore, leads to a reduction in entropy. This can be exploited by establishing multiple synergistic hydrogen bonds by modifying a rigid ligand moiety to decrease the effects of entropy–enthalpy compensation.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Thus, the location of hydrogen bonds is essential to achieve potent ligand binding and the effect of protein environment on this type of interaction should be carefully considered during visual analysis. In addition to formed hydrogen bonds, all unsatisfied hydrogen bond donor and acceptor groups without interacting counterparts, in particular those buried in the binding site, need to be evaluated, as they generally result in reduced binding free energies due to an uncompensated desolvation free energy penalty.<a onclick="showRef(event, 'ref2 ref21'); return false;" href="javascript:void(0);" class="ref ref2 ref21">(2,21)</a> In metalloproteins, distinct electrostatic interactions with metal cofactors can be of major importance for ligand potency (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A).<a onclick="showRef(event, 'ref22 ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref22 ref50 ref51">(22,50,51)</a> Interestingly, none of the studies reviewed in this perspective reported the consideration of weak hydrogen bonds, halogen bonds, orthogonal dipole interactions, or amide−π interactions as a criterion during visual inspection (see also survey results for additional details).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/medium/jm0c02227_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/large/jm0c02227_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Examples of binding modes. (A) Ligand–metal interaction of lisinopril with angiotensin-converting enzyme [Protein Data Bank (PDB) entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1O86">1O86</a>]. (B) <i>N</i>-(3,3-Diphenylpropyl)-4-benz-1,2,3-triazine-amine (left) and 3-cyano-<i>N</i>-(3,3-diphenylpropyl)-4-cinnolin-amine (right) bound to scytalone dehydratase in the presence or absence of a water molecule. Displacement of a water molecule by the cyano group resulted in a 2.0 kcal/mol increase in binding free energy. The figure was adapted after a crystal structure (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5STD">5STD</a>) and a study on hydration site prediction.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> (C) 4-Anilino-6,7-dialkoxyquinazoline (left) and 4-anilino-6,7-dialkoxyquinoline-3-carbonitrile (right) bound to EGFR kinase. Displacement of the water molecule resulted in an unfavorable binding free energy. Panel C was adapted after a crystal structure (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M17">1M17</a>) and a study on hydration site prediction.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> (D) Example for a strained ligand conformation in a crystal structure (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QAD">3QAD</a>)<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> with the 2-aminopyrimidine moiety out of plane as indicated by an arrow. The entry was since refined with a more reasonable conformation (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RZF">3RZF</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/large/jm0c02227_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02227&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Solvation effects are a major contributor to the protein–ligand binding thermodynamics. During ligand binding, water molecules are displaced from the solvated cavities.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> While the displacement of water molecules is favorable due to an increase in entropy and sometimes enthalpy, some may form enthalpically strong interactions with the protein and bridge ligand–protein interactions favorably contributing to the binding free energy. Thus, solvation effects such as water-mediated interactions should be considered in visual inspection (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B,C).<a onclick="showRef(event, 'ref22 ref24'); return false;" href="javascript:void(0);" class="ref ref22 ref24">(22,24)</a> However, the proper assessment of the multifaceted water behavior is by far not trivial even for extremely advanced experts. Moreover, the solvation scenario in the binding site in the apo state (no ligand present) is not routinely included in the visual comparison. Additional software for estimating the enthalpic and entropic contributions for the displacement of water molecules, for example, based on MD simulations<a onclick="showRef(event, 'ref39 ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref39 ref52 ref53">(39,52,53)</a> or derived neural networks,<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> is available and can augment docking and scoring. Another related evaluation criterion is the nature of solvent-exposed ligand atoms, as hydrophobic groups protruding into the surrounding solvent are generally considered as unfavorable if they otherwise would have the chance to interact with the protein.<a onclick="showRef(event, 'ref21 ref28'); return false;" href="javascript:void(0);" class="ref ref21 ref28">(21,28)</a></div><div class="NLM_p">In the formation of most complexes between ligand and protein, both entities have to adapt their configuration for optimizing their complementarity. It was estimated that 25% of all proteins undergo a certain degree of induced fit during ligand binding.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> A subsequent study of 188 proteins in the Protein Data Bank, with both holo and apo structures available, reported a much larger fraction of 90% of the proteins undergoing rotamer changes upon binding.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> This energetic cost upon deformation of either the ligand or the protein is termed strain, and its energetic evaluation is typically not optimally represented in the scoring function. Furthermore, due to the often rigid representation of the protein in molecular docking calculations employed for large-scale virtual screening, both binding partners cannot properly adapt to each other. Both shortcomings can result in unrealistic internal geometries of ligands (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D)<a onclick="showRef(event, 'ref7 ref11 ref56'); return false;" href="javascript:void(0);" class="ref ref7 ref11 ref56">(7,11,56)</a> or overly close contacts. For example, twisted amide or ester groups were observed, leading to the rejection of respective binding modes during visual inspection.<a onclick="showRef(event, 'ref3 ref22 ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref3 ref22 ref56 ref57">(3,22,56,57)</a> Thus, ligand and protein strain should be accounted for in a thorough visual inspection. Because protein strain can be considered only if the protein is treated flexibly, flexible docking campaigns and MD-based postprocessing should be used to take protein and ligand conformational adaptations into account.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> However, this may still be computationally too expensive for a large number of ligand–protein complexes as is usually the case in virtual screening tasks. In lead optimization, however, the use of such methods often becomes accessible due to the smaller number of compounds.</div><div class="NLM_p last">Preexisting knowledge of a protein–ligand complex may assist in determining the quality of a binding pose. In the most direct way, available crystal structures of the investigated target protein in complex with known ligands could guide the detection of common or especially relevant ligand–protein interactions. It was shown that the training of docking protocols with the native ligand improved the accuracy of pose prediction. In addition, as homologous targets often share conserved ligands and binding modes,<a onclick="showRef(event, 'ref58 ref59 ref60'); return false;" href="javascript:void(0);" class="ref ref58 ref59 ref60">(58−60)</a> crystal structures of related proteins with a similar co-crystallized ligand can be included in the decision making process.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i5">Case Studies for Applications of Visual Inspection in Computer-Aided Drug Design</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64312" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64312" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As mentioned above, visual inspection has been applied in a vast number of drug discovery projects. In most cases, the inspection was conducted as a final step of compound prioritization before experimental testing.<a onclick="showRef(event, 'ref22 ref25 ref44 ref62'); return false;" href="javascript:void(0);" class="ref ref22 ref25 ref44 ref62">(22,25,44,62)</a> In <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, we summarized several studies that provided a detailed description of the criteria used for visual inspection and that led to the discovery of potent ligands. As mentioned in the previous section, the most commonly assessed feature in visual inspection was complementarity between the ligand and respective binding site, and the presence of ligand–protein interactions such as hydrophobic interactions and hydrogen bonds. Furthermore, interactions with specific binding site residues or cofactors were often accounted for. This was typically motivated by interactions observed between co-crystallized ligands and protein<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> or based on conserved interactions among various ligands docked to the same target protein.<a onclick="showRef(event, 'ref28 ref43'); return false;" href="javascript:void(0);" class="ref ref28 ref43">(28,43)</a> Distorted ligand conformations were also frequently considered as an exclusion criterion for docking poses as they represent unrealistic ligand strain, for example, distortions of amide bonds, esters, or unfavorable <i>cis</i>–<i>trans</i> isomery.<a onclick="showRef(event, 'ref2 ref10 ref21 ref24 ref30'); return false;" href="javascript:void(0);" class="ref ref2 ref10 ref21 ref24 ref30">(2,10,21,24,30)</a></div><div class="NLM_p">On the contrary, features such as interactions with conserved or crystallographic water molecules as well as interactions relevant for ligand selectivity were rarely mentioned in the considered studies.<a onclick="showRef(event, 'ref22 ref24 ref31'); return false;" href="javascript:void(0);" class="ref ref22 ref24 ref31">(22,24,31)</a> Chemical diversity, novelty, and commercial availability of the best scored ligands were not a direct part of visual inspection, but rather criteria for postprocessing of docking results in general.<a onclick="showRef(event, 'ref10 ref24 ref31 ref36 ref44 ref61'); return false;" href="javascript:void(0);" class="ref ref10 ref24 ref31 ref36 ref44 ref61">(10,24,31,36,44,61)</a></div><div class="NLM_p">Most drug design studies that mention visual inspection did not detail the criteria used in this process, despite being an essential component of the selection process. A more detailed description of criteria relevant for the decision process would be desirable as it allows for reproducibility of the results, a common requirement for publishing other elements of computer-aided drug design (CADD) studies.</div><div class="NLM_p">In the following, we highlight some recent studies that demonstrate the usability of visual inspection of binding modes. For example, a large-scale virtual screening study discovered ligands binding to AmpC β-lactamase and D<sub>4</sub> dopamine receptor using a computational library consisting of 170 million compounds.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The authors not only used visual inspection for the discovery of highly potent ligands with novel chemotypes but also compared compounds prioritized with or without prior visualization. Interestingly, the latter analysis revealed that ligands selected on the basis of visual inspection generally presented superior binding affinity in the following experiments, confirming the usefulness of such procedures.</div><div class="NLM_p">The Drug Design Data Resource (D3R) regularly invites experts in the field of computational chemistry and CADD to participate in a challenge to predict ligand poses, rank compounds, and predict relative binding free energies in a blinded fashion.<a onclick="showRef(event, 'ref13 ref14 ref63 ref64'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref63 ref64">(13,14,63,64)</a> While the best-performing results in the second iteration of the challenge did not employ visual inspection, top submissions in the remaining challenges regularly relied on human intervention (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Similarly, the best performing submissions in the preceding SAMPL1 challenge reported visual inspection.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> A study published by participants of the SAMPL4 challenge provided details on their decision process and retrospectively evaluated their predictions with and without human intervention. The best hit rate was achieved when visual inspection was coupled with rescoring.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Before D3R, the Community Structure–Activity Resource (CSAR) held similar challenges, but visual inspection was only briefly mentioned and could not be compared to submissions without human intervention.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Direct Evaluation of Visual Inspection</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">outcome</th><th class="colsep0 rowsep0" align="center">targets</th><th class="colsep0 rowsep0" align="center">challenge</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">two best submissions employed visual inspection</td><td class="colsep0 rowsep0" align="left">JNK3 kinase</td><td class="colsep0 rowsep0" align="left">SAMPL1<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">best submission employed visual inspection</td><td class="colsep0 rowsep0" align="left">HIV integrase</td><td class="colsep0 rowsep0" align="left">SAMPL4<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">four of 11 best and none of the 28 worst submissions employed visual inspection</td><td class="colsep0 rowsep0" align="left">HSP90, MAP4K4</td><td class="colsep0 rowsep0" align="left">D3R 2015<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">none of the top submissions employed visual inspection</td><td class="colsep0 rowsep0" align="left">farnesoid X receptor</td><td class="colsep0 rowsep0" align="left">D3R #2<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">all but two top submissions employed visual inspection, but only one of the worst submissions</td><td class="colsep0 rowsep0" align="left">cathepsin S, VEGFR2, JAK2, p38-α, TIE2, ABL1</td><td class="colsep0 rowsep0" align="left">D3R #3<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">visual inspection augmented docking results in six of 10 top submissions</td><td class="colsep0 rowsep0" align="left">β-secretase 1, cathepsin S</td><td class="colsep0 rowsep0" align="left">D3R #4<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">improved ligand affinity with visual inspection</td><td class="colsep0 rowsep0" align="left">D<sub>4</sub> dopamine receptor, AmpC β-lactamase</td><td class="colsep0 rowsep0" align="left">none<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a></td></tr></tbody></table></div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Limitations of Human Intervention</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71760" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71760" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Even though the studies mentioned above highlight the advantages of human intervention in structure-based drug discovery projects, this practice also suffers from several limitations. Most obviously, the expertise and intuition of the individuals or teams conducting visual inspection strongly influence its outcome,<a onclick="showRef(event, 'ref13 ref16 ref35'); return false;" href="javascript:void(0);" class="ref ref13 ref16 ref35">(13,16,35)</a> as we further substantiate in the evaluation of our survey. Another limitation is the number of binding modes that can be visually inspected in a given amount of time.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> While some studies report the inspection of <100 binding modes or compounds,<a onclick="showRef(event, 'ref69 ref70'); return false;" href="javascript:void(0);" class="ref ref69 ref70">(69,70)</a> in the majority of the studies between 100 and 300 compounds were inspected.<a onclick="showRef(event, 'ref37 ref44 ref61 ref71'); return false;" href="javascript:void(0);" class="ref ref37 ref44 ref61 ref71">(37,44,61,71)</a> Some studies reported the examination of ≤1500 compounds.<a onclick="showRef(event, 'ref3 ref34 ref57 ref72 ref73 ref74'); return false;" href="javascript:void(0);" class="ref ref3 ref34 ref57 ref72 ref73 ref74">(3,34,57,72−74)</a> The latter likely presents the upper limit for the application of visual inspection as it becomes highly time-consuming and more importantly difficult to consistently evaluate such a large number of compounds. As the size of compound libraries that are screened in a single project has grown continuously with recent reports of multiple billion compounds,<a onclick="showRef(event, 'ref75 ref76'); return false;" href="javascript:void(0);" class="ref ref75 ref76">(75,76)</a> the visual inspection might be unfeasible or the hit list has to be narrowed down with other techniques. However, the assessed studies did not state the time frame in which the visual inspection was conducted.</div><div class="NLM_p last">Furthermore, the underlying docking methodology harbors inherent limitations that cannot be compensated by visual inspection alone. For example, it is known that protein flexibility plays an important role in ligand binding. Despite progress in flexible docking engines, most large-scale docking campaigns are still performed without including protein flexibility and structural adaptations. Thus, visual inspection is also based on a static view of the formed complex analogous to a crystal structure.<a onclick="showRef(event, 'ref5 ref7 ref77 ref78 ref79'); return false;" href="javascript:void(0);" class="ref ref5 ref7 ref77 ref78 ref79">(5,7,77−79)</a> Furthermore, loss of configurational entropy or desolvation free energy is difficult to evaluate by visual inspection.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Classical scoring functions, however, primarily evaluate enthalpic contributions using a simplified mathematical representation of interaction terms relevant to ligand–protein binding. Entropic effects, poorly represented in scoring functions, are mostly limited to the simplistic evaluation of ligand desolvation (e.g., hydrophobic contacts) and ligand configurational entropy (e.g., number of rotatable bonds).<a onclick="showRef(event, 'ref5 ref80 ref81 ref82 ref83'); return false;" href="javascript:void(0);" class="ref ref5 ref80 ref81 ref82 ref83">(5,80−83)</a> This improper handling of binding thermodynamics presents an important limitation of docking procedures as, for example, compensatory entropic contributions can significantly reduce binding affinity.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> As visual inspection alone cannot properly investigate those effects, it needs to be coupled with a more dynamic representation of ligand–protein binding, for example, MD simulations. MD simulations can intrinsically account for protein and ligand flexibility, and estimates based on trajectory postprocessing may incorporate entropic effects and thus provide a more precise estimation of free energies of binding compared to simple scoring functions. MD simulation can also be used to provide time-evolved insights into the stability of the ligand–protein complex.<a onclick="showRef(event, 'ref79 ref84'); return false;" href="javascript:void(0);" class="ref ref79 ref84">(79,84)</a> Studies showed that MD simulations combined with MM/GBSA calculations or derivations thereof can improve the binding affinity prediction of poses obtained from docking protocols, although the outcomes are highly case specific.<a onclick="showRef(event, 'ref85 ref86 ref87 ref88'); return false;" href="javascript:void(0);" class="ref ref85 ref86 ref87 ref88">(85−88)</a> Currently, free energy perturbations (FEP) are considered the most accurate methodology for computing binding free energies with an accuracy of approximately 1 kcal/mol.<a onclick="showRef(event, 'ref89 ref90'); return false;" href="javascript:void(0);" class="ref ref89 ref90">(89,90)</a> Despite recent developments in the improved treatment of solvation effects<a onclick="showRef(event, 'ref39 ref91'); return false;" href="javascript:void(0);" class="ref ref39 ref91">(39,91)</a> and its integration in neural network postprocessing methods, standard docking programs can typically reproduce only ∼60% of native binding modes of a ligand within an RMSD threshold of 2 Å using the best scored pose as demonstrated, for example, in a large-scale evaluation of 10 docking programs.<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a> This underlines the necessity of a visual inspection to discard non-native poses.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">Survey of Computational Medicinal Chemists</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81312" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81312" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In the <a class="ref internalNav" href="#sec1" aria-label="Introduction">Introduction</a>, we recalled a significant disagreement among medicinal chemists regarding compound prioritization that is not surprising considering the high dimensionality of the problem.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> To obtain more detailed insight into the current state of the art in visual pose inspection, we conducted a survey of experts in the field from both academia and the pharmaceutical industry. In total, we managed to collect 93 expert opinions. The survey consisted of three parts: (i) information about the participant such as profession and experience, (ii) general questions on visual inspection, used protocols, and decision criteria, and (iii) a challenge to recognize the native binding mode of three ligand–protein complexes. Potential participants for the survey were individually contacted by Email or LinkedIn. All participants have recently published in the area of structure-based drug discovery in peer-reviewed journals. The survey form was composed using a designated platform provided by Google.<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a> In addition, participants were encouraged to share the survey with experienced colleagues.</div><div class="NLM_p">A majority of the responders were from academia (71, 76.3%), followed by 20 (21.5%) experts from the pharmaceutical industry, while only two were from nonprofit organizations or government institutions. The largest group of participants consisted of computational chemists followed by medicinal chemists, and cheminformaticians (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). More than half of the participants reported more than 10 years of experience in the field of computer-aided molecular design (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). As a conservative summation, this amounted to >600 years of experience. The participants rated the importance of visual inspection during a drug design project with 4 (29.0%) or 5 points (64.5%) on a scale from 1 to 5 underlining the popularity of such procedures.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/medium/jm0c02227_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/large/jm0c02227_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Results of the survey. (A) Participants of the survey divided according to their current scientific position. (B) Participants of the survey divided according to years of experience in computer-aided molecular design. (C) Procedures taken by all participants to validate the used docking protocols. (D) Methods of all participants used for postprocessing of docking results. (E) Importance of different criteria valued by all participants, participants from the pharmaceutical industry, and the remaining participants. (F) Importance of different criteria valued by all participants divided according to their years of experience compared to the overall result.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/large/jm0c02227_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02227&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Prior to perspective docking and visual inspection, the validation of the used docking protocol to retrospectively assess pose prediction accuracy as well as the capability to distinguish actives from decoy compounds is generally recommended.<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a> Whereas most participants (80.6%) reported that they performed redocking of co-crystallized ligands into the X-ray protein structure, only around half of them reported cross-docking of multiple ligands to the selected protein structures or using decoys to determine enrichment metrics (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). Surprisingly, 11 participants (evenly spread between academia and industry) do not conduct any of the validation steps mentioned above.</div><div class="NLM_p">Most participants do not rely on docking as a sole computational methodology. Typically, they perform postprocessing of docking results, for example, by using MD simulations, likely to cope with the limited treatment of protein flexibility in standard docking protocols and evaluate the time-evolved stability of the obtained poses (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D).</div><div class="NLM_p">With respect to visual inspection, a majority of participants conduct both individual assessments by themselves combined with an inspection in a team setting to discuss the results with colleagues. Around one-third of the participants perform visual inspection only by themselves, while 10 participants (10.8%) assess binding modes only with colleagues. Interestingly, when we subdivided the participants into the academic and private sector, a larger share of only 12.7% in the industry assessed binding modes with colleagues than in academia (4.6%). The integration of individual and collective opinions among a board of colleagues capitalizes on both the experience of single researchers as well as the excellence of a team, improving the quality of the inspection.</div><div class="NLM_p">Whereas the options to exploit virtual reality (VR) in a CADD setting improved in recent years, which offers new possibilities for humans to interact with 3D data,<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> its use among experts is, to date, still highly limited (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). More than 88% of the participants in our survey conduct visual inspection on a regular desktop computer screen. The use of 3D screens or 3D beamers at the interface of VR and regular screens was reported by 21.5% of participants. With regard to the time point at which visual inspection is conducted within the drug discovery pipeline, its popularity decreases from hit identification to the final drug candidate (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/medium/jm0c02227_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/large/jm0c02227_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Hardware used for visualization by the participants. (B) Tasks for which visual inspection is applied by the participants within a discovery pipeline. (C) Number of correct answers (score) of the participants divided according to affiliation and experience. The percentages indicate the fraction of the total number of participants categorized into a group (e.g., participants from the industry) who achieved a certain score.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/large/jm0c02227_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02227&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In the survey, we evaluated the importance of different criteria used for the assessment of binding poses (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E,F). Consistent with the published studies previously discussed, shape complementarity was valued among the most relevant criteria independent of experience or affiliation, probably as it is simple to deduce at first sight. The analogy of predicted binding poses to co-crystallized ligands in experimentally determined crystal structures was rated to be of similar importance. Interestingly, industrial scientists prioritized these assessments, potentially due to the better availability of such data in this setting. Ligand–protein interactions with specific binding site residues were valued highly in both our survey and the considered publications. Interestingly, ligand strain, the location of ligand–protein interactions with respect to the binding site, and unsatisfied polar ligand or protein heteroatoms were prioritized more highly by scientists in the pharmaceutical industry than in academia. Unsatisfied protein hydrogen bond-donating or -accepting groups were valued more highly by both the most experienced participants and those working in the pharmaceutical industry. However, our questionnaire did not distinguish between unsatisfied hydrogen bonding groups in different locations of the binding site such as deeply buried groups for which the desolvation penalty would have to be compensated by favorable interactions as opposed to solvent-exposed polar groups, which could favorably interact with the solvent. Solvation effects such as the displacement of water molecules during ligand–protein association were regarded more highly by the most experienced participants. Most remarkably, docking scores produced by the scoring functions were valued the least, especially by experts in the pharmaceutical industry. Seemingly, the known unreliability of scoring and ranking by docking algorithms affected its popularity among experts.</div><div class="NLM_p last">All statements regarding differences among the preference of certain criteria between different participant groups display statistical significance based on a two-sided <i>t</i> test at a <i>p</i> = 0.05 level. The scipy python package was used for those tests. Scientists involved in CADD projects, especially those new to the field, will likely profit from the results of our survey as it provides guidance and allows for a comparison of their own procedures and criteria with other scientists in the field.</div></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Binding Mode Prediction Challenge</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56544" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56544" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As mentioned above, our survey included three challenges focused on the identification of native binding modes, where “native” refers to the pose present in the crystal structure. The first example is the complex of (5-pyridin-3-ylfuran-2-yl)methanamine bound to cytochrome P450 2A6 (CYP2A6), in which a terminal amine group is coordinated to the heme iron (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Due to the electrostatic interaction in particular with the metal ion, the p<i>K</i><sub>a</sub> value is significantly decreased compared to that of the unbound state of the ligand, forcing the amine to be uncharged in the complex. In addition, the protonation state of the amine might have been shifted to the neutral state due to a pH value of 8.5 in the crystallization medium as annotated in the Protein Data Bank. To obtain a non-native pose with a feasible interaction profile, we manually reversed the orientation of the ligand where the pyridine nitrogen is involved in metal coordination (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). Surprisingly, the non-native orientation of the ligand was assumed as the correct pose by a slight majority of participants. In the survey, we offered the option to give a justification for the selection for all three examples. As expected, the rather unconventional protonation state of the amine was often referred to when the non-native complex was selected as the answer by the participants. Furthermore, heterocyclic nitrogen-containing ring moieties such as imidazoles and pyridines are classically regarded as coordination partners of the heme iron in CYPs.<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a> The binding mode of the compound with the amine coordinated to the heme iron shows how the acido-basic character of the ligand—here the basicity, which is higher at the aliphatic amine group than at the aromatic ring nitrogen—can influence the interaction with metal ions and, thus, the binding mode of a compound.<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Results of the Challenge and Machine-Based Ranking</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">case 1</th><th class="colsep0 rowsep0" align="center">case 2</th><th class="colsep0 rowsep0" align="center">case 3</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">correct<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">46.2%</td><td class="colsep0 rowsep0" align="left">63.4%</td><td class="colsep0 rowsep0" align="left">59.1%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">wrong<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">53.8%</td><td class="colsep0 rowsep0" align="left">36.6%</td><td class="colsep0 rowsep0" align="left">40.9%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Glide docking<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">correct</td><td class="colsep0 rowsep0" align="left">wrong</td><td class="colsep0 rowsep0" align="left">correct</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">smina docking<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">correct</td><td class="colsep0 rowsep0" align="left">wrong</td><td class="colsep0 rowsep0" align="left">correct</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MM/GBSA<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">correct</td><td class="colsep0 rowsep0" align="left">correct</td><td class="colsep0 rowsep0" align="left">inconclusive<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Outcome of the survey regarding the respective case of the challenge.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Statement if ranking based on the computation of binding free energies using two different docking programs and MM/GBSA calculations did prioritize the correct pose.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">No statistically significant difference between the calculated energies of the native and incorrect poses.</p></div></div></div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/medium/jm0c02227_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/large/jm0c02227_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structures that were included in the survey to distinguish native poses from artifacts. In the survey, the structures were presented in form of an animated GIF image, showing all relevant interactions. (A) Case 1: (5-pyridin-3-ylfuran-2-yl)methanamine bound to CYP2A6 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FDW">2FDW</a>). The native pose is shown at the top, while a manually composed pose is shown at the bottom. (B) Case 2: 1-(5-methylthiophen-2-yl)-3-pyridin-3-ylurea bound to the main protease of SARS-CoV-2 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5RH0">5RH0</a>). The native pose is shown at the top, while a slightly different pose from docking is displayed at the bottom. (C) Case 3: <i>N</i>-[(4-fluorophenyl)methyl]-<i>N</i>-(2-methylpropyl)-6-[(1-methylsulfonylpiperidin-4-yl)amino]pyridine-3-sulfonamide bound to RAR-related orphan receptor γ (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WLB">4WLB</a>). The native pose is shown at the top, while a slightly different pose from docking is displayed at the bottom.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/large/jm0c02227_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02227&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Thus, this case study represents an example in which the intuition and experience of most scientists deceived them from selecting the correct result. This was underlined by the fact that less experienced scientists (fewer than five years of experience) were selecting the correct pose more frequently than the more experienced scientist. Participants selecting the native binding mode mostly considered improved ligand–protein interactions, including the hydrogen bond of the pyridine ring as well as the unfavorable placement of the protonated amine in a hydrophobic region of the pocket. In contrast to the remaining examples, conventional scoring functions as well as MM/GBSA calculations outperformed the human prioritization in this case. The ligand protonation in the input structures for these scoring procedures was submitted as shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> with the native pose in the neutral form, while the terminal amine was protonated in the non-native complex. The largest differences among the individual components of the MM/GBSA term between the native and non-native poses were in the electrostatic and lipophilic contributions.</div><div class="NLM_p">The second example consisted of two similar poses of 1-(5-methylthiophen-2-yl)-3-pyridin-3-ylurea bound to the main protease of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in two slightly different orientations, of which one was obtained by using the Glide SP docking protocol (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B).<a onclick="showRef(event, 'ref97'); return false;" href="javascript:void(0);" class="ref ref97">(97)</a> Both binding poses form two hydrogen bonds: the native pose interacts with the protein backbone as opposed to a flexible side chain in the non-native one. In this case, 63.4% of the participants correctly identified the native pose, the highest percentage among all three case studies. In contrast, both tested docking programs prioritized the non-native binding mode, supporting the previous assumption that expert knowledge can outperform conventional scoring functions.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Participants prioritizing the native pose frequently reported shape complementarity as well as the more stable hydrogen bond interaction with the protein backbone to be decisive for their choice. On the contrary, participants selecting the non-native pose mostly argued on the basis of the internal strain of the ligand and frequently referred to the conformation of the aminopyridine moiety. This example further underlined the benefit of MM/GBSA calculations, as they correctly preferred the native pose as opposed to the used docking methods. This advantage originates from the use of time-evolved MD trajectories in combination with the more accurate treatment of solvation free energies.<a onclick="showRef(event, 'ref89'); return false;" href="javascript:void(0);" class="ref ref89">(89)</a></div><div class="NLM_p last">The third challenge consisted of an inverse agonist bound to RAR-related orphan receptor γ (RORγ) in two similar binding modes regarding the aromatic moieties of the compound. However, the sulfonamide moiety of the ligand is buried within the core of the receptor in the native binding mode, while it is partially solvent-exposed in the non-native docking pose we selected (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). The native pose is preferred due to better hydrophobic complementarity and a more favorable hydrogen bonding network. Interestingly, even though the majority of participants prioritized the native pose, still more than 40% prioritized the non-native pose. As expected, the selection of the correct pose was frequently guided by the improved hydrogen bonding network and the increased extent of burial of the ligand and the two hydrogen bonds. The non-native pose was associated with unfavorable conformational strain and thus disregarded from being the native binding mode when the correct pose was selected. In contrast, the non-native pose was argued to be more favorable due to the solvent-exposed polar sulfonamide. While both docking programs were correctly predicting the native pose, MM/GBSA predicted similar free energies of binding for both poses. The latter might indicate multiple favorable binding modes, but the ligand atoms are well resolved according to the electron density in the accompanying crystal structure. The statistical significance of the MM/GBSA binding free energies, based on the quantification of 125 frames of the last quarter of a 20 ns MD simulation, was evaluated at a <i>p</i> = 0.05 level using the scipy python package. While the results of cases 1 and 3 were statistically significant, the average values in case 2 were only significant at a <i>p</i> = 0.07 level.</div></div><div id="sec7" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67015" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67015" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Due to the inherent drawbacks of molecular docking and pose scoring, visual inspection of predicted protein–ligand complexes was performed in a large number of drug discovery projects. Frequently, only limited insight into the actual decision making process was provided. We reviewed the available medicinal chemistry literature to deduce the applied criteria and summarize the studies that directly evaluated the gain or loss of prediction accuracy upon visual inspection of the docked binding modes. These comparison studies revealed that in most cases human intervention was improving the prediction performance, with only one study showing no improvement. The most frequently mentioned criteria included shape complementarity, hydrogen bonds, and hydrophobic contacts. Furthermore, we discussed the limitations of human interventions, which include the number of poses that can be accurately assessed, the different levels of experience, and the given limitations of the underlying methodology such as the frequent lack of protein flexibility or detailed consideration of entropy. To gain more insight into the decision making of experts in the field, we conducted a survey including a challenge to distinguish native from non-native poses in three case studies. While there was considerable disagreement among the participants, a majority detected the native pose in two of three cases. In the remaining case, in which an uncharged primary amine instead of a heterocyclic nitrogen of an inhibitor undergoes an interaction with the heme iron in CYP2A6, prior experience with CYP inhibitors negatively affected the detection of the native pose, demonstrating that human bias might exist, too. The individual inspection followed by a discussion with colleagues in a team setting, as the majority of participants reported it, can likely cope with such biases and improve the outcome. Similar to the literature review, the surveyed scientists specified shape complementarity, analogy to binding modes observed in co-crystallized complexes, and specific ligand–protein interactions as the most valuable criteria. Among scientists active in an industrial setting, ligand strain and the number of unsatisfied ligand or protein heteroatoms were valued comparatively highly. Docking scores, which are known to be frequently inaccurate, were valued least important among the available criteria. This suggests that the journey to reliable scoring functions is by far not over, as today’s scoring functions are often no match for the complex knowledge and vast experience of computational medicinal chemists. Overall, many factors have to be considered during visual inspection, and this perspective can support the decision making of experts as well as scientists new to the field. On the basis of the survey results, we would like to motivate scientists reviewing docking studies in peer-reviewed journals to routinely insist on docking protocol validation as well as on inclusion of visual analyses and other postprocessing methods.</div></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c02227" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03288" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03288" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Markus A. Lill</span> - <span class="hlFld-Affiliation affiliation">Computational Pharmacy, Department
of Pharmaceutical Sciences, University of
Basel, Klingelbergstrasse 61, 4056 Basel, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3023-5188" title="Orcid link">http://orcid.org/0000-0003-3023-5188</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#4e232f3c253b3d60222722220e3b20272c2f3d602d26"><span class="__cf_email__" data-cfemail="335e52415846401d5f5a5f5f73465d5a5152401d505b">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">André Fischer</span> - <span class="hlFld-Affiliation affiliation">Computational Pharmacy, Department
of Pharmaceutical Sciences, University of
Basel, Klingelbergstrasse 61, 4056 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin Smieško</span> - <span class="hlFld-Affiliation affiliation">Computational Pharmacy, Department
of Pharmaceutical Sciences, University of
Basel, Klingelbergstrasse 61, 4056 Basel, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2758-2680" title="Orcid link">http://orcid.org/0000-0003-2758-2680</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Manuel Sellner</span> - <span class="hlFld-Affiliation affiliation">Computational Pharmacy, Department
of Pharmaceutical Sciences, University of
Basel, Klingelbergstrasse 61, 4056 Basel, Switzerland</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36757" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36757" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">André Fischer</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02227&amp;id=BIO-d7e5335-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>André Fischer</b> obtained his Master’s Degree in Drug Sciences from the University of Basel in 2018, completing it with his work on cytochrome P450 2D6 pharmacogenomics. In the same year, he started his Ph.D. studies in the research group of Computational Pharmacy at the University of Basel where he is currently active. His work includes contributions in the field of computational chemistry with regard to virtual screening, compound selectivity, and ligand-induced conformational changes.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Martin Smieško</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02227&amp;id=BIO-d7e5340-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Martin Smieško</b> graduated in medicinal chemistry from the Faculty of Pharmacy of Comenius University (Bratislava, Slovakia) in 1999. In 2003, he completed his Ph.D. at the same university. In 2006, he joined the Molecular Modeling group of Prof. Angelo Vedani at the University of Basel as a research associate. In 2014, he was promoted to Senior Lecturer. He made various contributions to the field of computer-aided drug design and off-target binding prediction, including the development of a program for flexible docking.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Manuel Sellner</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02227&amp;id=BIO-d7e5345-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Manuel Sellner</b> obtained his Master’s Degree in Drug Sciences following his Bachelor’s Degree in Pharmaceutical Sciences at the University of Basel. He completed his studies with his work on the interactions between cytochrome P450 2D6 and its corresponding reductase. In early 2020, he initiated his Ph.D. studies in the research group of Computational Pharmacy at the University of Basel, where he works on the development of tools for predicting off-target binding in a structure-based approach.</p></figure></div><div class="bio" rid="ath4"><figure id="ath4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Markus A. Lill</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02227&amp;id=BIO-d7e5350-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Markus A. Lill</b> received his Diploma in Physics from the University of Erlangen-Nürnberg and in 2002 his Ph.D. from the Max Planck Institute for Biophysics (Frankfurt, Germany). He completed four years of postdoctoral training at the Biographics Laboratory 3R and his Habilitation in 2006 at the University of Basel. He then became an Assistant Professor and 2012 Associate Professor for Computational Drug Design at Purdue University (West Lafayette, IN). Currently, he is leading the Computational Pharmacy group at the University of Basel, where his research is focused on the development and application of computational methods for drug discovery with a deep focus on physics-based artificial intelligence.</p></figure></div><div class="ack" id="ACK-d7e1111-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87683" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87683" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank everyone who participated in our survey. In addition, the authors gratefully acknowledge the expert knowledge provided by Dr. Christian Kramer (Roche) and Dr. Andreas Bergner (Boehringer Ingelheim) supporting the design of the survey.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">D3R</td><td class="NLM_def"><p class="first last">Drug Design Data Resource</p></td></tr><tr><td class="NLM_term">MM/GBSA</td><td class="NLM_def"><p class="first last">molecular mechanics/generalized Born surface area</p></td></tr><tr><td class="NLM_term">RANKL</td><td class="NLM_def"><p class="first last">receptor activator of nuclear factor κ-B ligand</p></td></tr><tr><td class="NLM_term">RORγ</td><td class="NLM_def"><p class="first last">RAR-related orphan receptor γ</p></td></tr><tr><td class="NLM_term">SAMPL</td><td class="NLM_def"><p class="first last">Statistical Assessment of Modeling of Proteins and Ligands</p></td></tr><tr><td class="NLM_term">SARS-CoV-2</td><td class="NLM_def"><p class="first last">severe acute respiratory syndrome coronavirus 2</p></td></tr><tr><td class="NLM_term">VR</td><td class="NLM_def"><p class="first last">virtual reality</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41176" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41176" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 97 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seeliger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Groot, B. L.</span></span> <span> </span><span class="NLM_article-title">Ligand docking and binding site analysis with PyMOL and Autodock/Vina</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">417</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1007/s10822-010-9352-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1007%2Fs10822-010-9352-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=20401516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntVahtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=417-422&author=D.+Seeligerauthor=B.+L.+de+Groot&title=Ligand+docking+and+binding+site+analysis+with+PyMOL+and+Autodock%2FVina&doi=10.1007%2Fs10822-010-9352-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand docking and binding site analysis with PyMOL and Autodock/Vina</span></div><div class="casAuthors">Seeliger, Daniel; Groot, Bert L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">417-422</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Docking of small mol. compds. into the binding site of a receptor and estg. the binding affinity of the complex is an important part of the structure-based drug design process.  For a thorough understanding of the structural principles that det. the strength of a protein/ligand complex both, an accurate and fast docking protocol and the ability to visualize binding geometries and interactions are mandatory.  Here we present an interface between the popular mol. graphics system PyMOL and the mol. docking suites Autodock and Vina and demonstrate how the combination of docking and visualization can aid structure-based drug design efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTR4WZx4OqG7Vg90H21EOLACvtfcHk0lgwDLJjImdC0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntVahtLo%253D&md5=35f8b79d6c1d6582919878a53b4a5129</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1007%2Fs10822-010-9352-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-010-9352-6%26sid%3Dliteratum%253Aachs%26aulast%3DSeeliger%26aufirst%3DD.%26aulast%3Dde%2BGroot%26aufirst%3DB.%2BL.%26atitle%3DLigand%2520docking%2520and%2520binding%2520site%2520analysis%2520with%2520PyMOL%2520and%2520Autodock%252FVina%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2010%26volume%3D24%26spage%3D417%26epage%3D422%26doi%3D10.1007%2Fs10822-010-9352-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bissantz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">A medicinal chemist’s guide to molecular interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">6241</span>– <span class="NLM_lpage">6241</span>, <span class="refDoi"> DOI: 10.1021/jm100950p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100950p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3cXps1Whtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=6241-6241&author=C.+Bissantzauthor=B.+Kuhnauthor=M.+Stahl&title=A+medicinal+chemist%E2%80%99s+guide+to+molecular+interactions&doi=10.1021%2Fjm100950p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">A Medicinal Chemist's Guide to Molecular Interactions. [Erratum to document cited in CA153:076147]</span></div><div class="casAuthors">Bissantz, Caterina; Kuhn, Bernd; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6241</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  On page 5063, the first sentence of the second paragraph in the left column is incorrect; the correct sentence is given.  On page 5075, the Figure 16 caption is incorrect; the correct caption is given.  On page 5083, Ref. 159 is incorrect; the correct version of the ref. is given.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXK65xi0HgarVg90H21EOLACvtfcHk0likwzBAK15WnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXps1Whtr0%253D&md5=f9d8f9cc15728ffd2c09447b45fdf422</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fjm100950p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100950p%26sid%3Dliteratum%253Aachs%26aulast%3DBissantz%26aufirst%3DC.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DA%2520medicinal%2520chemist%25E2%2580%2599s%2520guide%2520to%2520molecular%2520interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D6241%26epage%3D6241%26doi%3D10.1021%2Fjm100950p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lyu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balius, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levit, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moroz, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Meara, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Algaa, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolmachova, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolmachev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span> <span> </span><span class="NLM_article-title">Ultra-large library docking for discovering new chemotypes</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>566</i></span>,  <span class="NLM_fpage">224</span>– <span class="NLM_lpage">229</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-0917-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1038%2Fs41586-019-0917-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=30728502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmt1yns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=566&publication_year=2019&pages=224-229&author=J.+Lyuauthor=S.+Wangauthor=T.+E.+Baliusauthor=I.+Singhauthor=A.+Levitauthor=Y.+S.+Morozauthor=M.+J.+O%E2%80%99Mearaauthor=T.+Cheauthor=E.+Algaaauthor=K.+Tolmachovaauthor=A.+A.+Tolmachevauthor=B.+K.+Shoichetauthor=B.+L.+Rothauthor=J.+J.+Irwin&title=Ultra-large+library+docking+for+discovering+new+chemotypes&doi=10.1038%2Fs41586-019-0917-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Ultra-large library docking for discovering new chemotypes</span></div><div class="casAuthors">Lyu, Jiankun; Wang, Sheng; Balius, Trent E.; Singh, Isha; Levit, Anat; Moroz, Yurii S.; O'Meara, Matthew J.; Che, Tao; Algaa, Enkhjargal; Tolmachova, Kateryna; Tolmachev, Andrey A.; Shoichet, Brian K.; Roth, Bryan L.; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">566</span>
        (<span class="NLM_cas:issue">7743</span>),
    <span class="NLM_cas:pages">224-229</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Despite intense interest in expanding chem. space, libraries contg. hundreds-of-millions to billions of diverse mols. have remained inaccessible.  Here we investigate structure-based docking of 170 million make-on-demand compds. from 130 well-characterized reactions.  The resulting library is diverse, representing over 10.7 million scaffolds that are otherwise unavailable.  For each compd. in the library, docking against AmpC β-lactamase (AmpC) and the D4 dopamine receptor were simulated.  From the top-ranking mols., 44 and 549 compds. were synthesized and tested for interactions with AmpC and the D4 dopamine receptor, resp.  We found a phenolate inhibitor of AmpC, which revealed a group of inhibitors without known precedent.  This mol. was optimized to 77 nM, which places it among the most potent non-covalent AmpC inhibitors known.  Crystal structures of this and other AmpC inhibitors confirmed the docking predictions.  Against the D4 dopamine receptor, hit rates fell almost monotonically with docking score, and a hit-rate vs. score curve predicted that the library contained 453,000 ligands for the D4 dopamine receptor.  Of 81 new chemotypes discovered, 30 showed submicromolar activity, including a 180-pM subtype-selective agonist of the D4 dopamine receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc_Nuo9pc-Z7Vg90H21EOLACvtfcHk0likwzBAK15WnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmt1yns70%253D&md5=589fb81ef11f9cbec645b9a3818648d1</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-0917-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-0917-9%26sid%3Dliteratum%253Aachs%26aulast%3DLyu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DBalius%26aufirst%3DT.%2BE.%26aulast%3DSingh%26aufirst%3DI.%26aulast%3DLevit%26aufirst%3DA.%26aulast%3DMoroz%26aufirst%3DY.%2BS.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%2BJ.%26aulast%3DChe%26aufirst%3DT.%26aulast%3DAlgaa%26aufirst%3DE.%26aulast%3DTolmachova%26aufirst%3DK.%26aulast%3DTolmachev%26aufirst%3DA.%2BA.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DUltra-large%2520library%2520docking%2520for%2520discovering%2520new%2520chemotypes%26jtitle%3DNature%26date%3D2019%26volume%3D566%26spage%3D224%26epage%3D229%26doi%3D10.1038%2Fs41586-019-0917-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waszkowycz, B.</span></span> <span> </span><span class="NLM_article-title">Towards improving compound selection in structure-based virtual screening</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">219</span>– <span class="NLM_lpage">226</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2007.12.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1016%2Fj.drudis.2007.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=18342797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsVSjsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=219-226&author=B.+Waszkowycz&title=Towards+improving+compound+selection+in+structure-based+virtual+screening&doi=10.1016%2Fj.drudis.2007.12.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Towards improving compound selection in structure-based virtual screening</span></div><div class="casAuthors">Waszkowycz, Bohdan</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5/6</span>),
    <span class="NLM_cas:pages">219-226</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Structure-based virtual screening is now an established technol. for supporting hit finding and lead optimization in drug discovery.  Recent validation studies have highlighted the poor performance of currently used scoring functions in estg. binding affinity and hence in ranking large datasets of docked ligands.  Progress in the anal. of large datasets can be made through the use of appropriate data mining techniques and the derivation of a broader range of descriptors relevant to receptor-ligand binding.  In addn., simple scoring functions can be supplemented by simulation-based scoring protocols.  Developments in workflow design allow the automation of repetitive tasks, and also encourage the routine use of simulation-based methods and the rapid prototyping of novel modeling and anal. procedures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-3q2xzmPBubVg90H21EOLACvtfcHk0likwzBAK15WnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsVSjsbg%253D&md5=a06324f926aadfe4c53f8941f9465d62</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2007.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2007.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DWaszkowycz%26aufirst%3DB.%26atitle%3DTowards%2520improving%2520compound%2520selection%2520in%2520structure-based%2520virtual%2520screening%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2008%26volume%3D13%26spage%3D219%26epage%3D226%26doi%3D10.1016%2Fj.drudis.2007.12.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cappel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jerome, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matter, H.</span></span> <span> </span><span class="NLM_article-title">Impact of different automated binding pose generation approaches on relative binding free energy simulations</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1432</span>– <span class="NLM_lpage">1444</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.9b01118</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.9b01118" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs12ktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=1432-1444&author=D.+Cappelauthor=S.+Jeromeauthor=G.+Hesslerauthor=H.+Matter&title=Impact+of+different+automated+binding+pose+generation+approaches+on+relative+binding+free+energy+simulations&doi=10.1021%2Facs.jcim.9b01118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Different Automated Binding Pose Generation Approaches on Relative Binding Free Energy Simulations</span></div><div class="casAuthors">Cappel, Daniel; Jerome, Steven; Hessler, Gerhard; Matter, Hans</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1432-1444</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Relative binding free energy (RBFE) prediction methods such as free energy perturbation (FEP) are important today for estg. protein-ligand binding affinities.  Significant hardware and algorithmic improvements now allow for simulating congeneric series within days.  Therefore, RBFE calcns. have an enormous potential for structure-based drug discovery.  As typically only a few representative crystal structures for a series are available, other ligands and design proposals must be reliably superimposed for meaningful results.  An obsd. significant effect of the alignment on FEP led us to develop an alignment approach combining docking with max. common substructure (MCS) derived core constraints from the most similar ref. pose, named MCS-docking workflow.  We then studied the effect of binding pose generation on the accuracy of RBFE predictions using six ligand series from five pharmaceutically relevant protein targets.  Overall, the MCS-docking workflow generated consistent poses for most of the ligands in the investigated series.  While multiple alignment methods often resulted in comparable FEP predictions, for most of the cases herein, the MCS-docking workflow produced the best accuracy in predictions.  Furthermore, the FEP anal. data strongly support the hypothesis that the accuracy of RBFE predictions depends on input poses to construct the perturbation map.  Therefore, an automated workflow without manual intervention minimizes potential errors and obtains the most useful predictions with significant impact for structure-based design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGob6kqQDqVs-7Vg90H21EOLACvtfcHk0lgB19RjvPX48g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs12ktb4%253D&md5=c6c5e60560061e35da8e661f17c10a8e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.9b01118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.9b01118%26sid%3Dliteratum%253Aachs%26aulast%3DCappel%26aufirst%3DD.%26aulast%3DJerome%26aufirst%3DS.%26aulast%3DHessler%26aufirst%3DG.%26aulast%3DMatter%26aufirst%3DH.%26atitle%3DImpact%2520of%2520different%2520automated%2520binding%2520pose%2520generation%2520approaches%2520on%2520relative%2520binding%2520free%2520energy%2520simulations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D1432%26epage%3D1444%26doi%3D10.1021%2Facs.jcim.9b01118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, R. T.</span></span> <span> </span><span class="NLM_article-title">Structure-based drug design: Docking and scoring</span>. <i>Curr. Protein Pept. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">312</span>– <span class="NLM_lpage">328</span>, <span class="refDoi"> DOI: 10.2174/138920307781369382</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.2174%2F138920307781369382" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=17696866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpslKrs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=312-328&author=R.+T.+Kroemer&title=Structure-based+drug+design%3A+Docking+and+scoring&doi=10.2174%2F138920307781369382"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based drug design: docking and scoring</span></div><div class="casAuthors">Kroemer, Romano T.</div><div class="citationInfo"><span class="NLM_cas:title">Current Protein and Peptide Science</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">312-328</span>CODEN:
                <span class="NLM_cas:coden">CPPSCM</span>;
        ISSN:<span class="NLM_cas:issn">1389-2037</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">This review gives an introduction into ligand - receptor docking and illustrates the basic underlying concepts.  An overview of different approaches and algorithms is provided.  Although the application of docking and scoring has led to some remarkable successes, there are still some major challenges ahead, which are outlined here as well.  Approaches to address some of these challenges and the latest developments in the area are presented.  Some aspects of the assessment of docking program performance are discussed.  A no. of successful applications of structure-based virtual screening are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbXWKN0_d9g7Vg90H21EOLACvtfcHk0lgB19RjvPX48g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpslKrs7w%253D&md5=5bfb61ca9c4b405a98f50ddc2bb00ed7</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.2174%2F138920307781369382&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138920307781369382%26sid%3Dliteratum%253Aachs%26aulast%3DKroemer%26aufirst%3DR.%2BT.%26atitle%3DStructure-based%2520drug%2520design%253A%2520Docking%2520and%2520scoring%26jtitle%3DCurr.%2520Protein%2520Pept.%2520Sci.%26date%3D2007%26volume%3D8%26spage%3D312%26epage%3D328%26doi%3D10.2174%2F138920307781369382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mobley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dill, K. A.</span></span> <span> </span><span class="NLM_article-title">Binding of small-molecule ligands to proteins: ”What you see” is not always ”what you get</span>. <i>Structure (Oxford, U. K.)</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">489</span>– <span class="NLM_lpage">498</span>, <span class="refDoi"> DOI: 10.1016/j.str.2009.02.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1016%2Fj.str.2009.02.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=19368882" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVCmsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=489-498&author=D.+L.+Mobleyauthor=K.+A.+Dill&title=Binding+of+small-molecule+ligands+to+proteins%3A+%E2%80%9DWhat+you+see%E2%80%9D+is+not+always+%E2%80%9Dwhat+you+get&doi=10.1016%2Fj.str.2009.02.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Binding of Small-Molecule Ligands to Proteins: "What You See" Is Not Always "What You Get"</span></div><div class="casAuthors">Mobley, David L.; Dill, Ken A.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">489-498</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  We review insights from computational studies of affinities of ligands binding to proteins.  The power of structural biol. is in translating knowledge of protein structures into insights about their forces, binding, and mechanisms.  However, the complementary power of computer modeling is in showing "the rest of the story" (i.e., how motions and ensembles and alternative conformers and the entropies and forces that cannot be seen in single mol. structures also contribute to binding affinities).  Upon binding to a protein, a ligand can bind in multiple orientations; the protein or ligand can be deformed by the binding event; waters, ions, or cofactors can have unexpected involvement; and conformational or solvation entropies can sometimes play large and otherwise unpredictable roles.  Computer modeling is helping to elucidate these factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnlrwNwfLm4rVg90H21EOLACvtfcHk0lgB19RjvPX48g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVCmsrk%253D&md5=993623b1444e33a45b2fe8b904dec9c1</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2009.02.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2009.02.010%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DBinding%2520of%2520small-molecule%2520ligands%2520to%2520proteins%253A%2520%25E2%2580%259DWhat%2520you%2520see%25E2%2580%259D%2520is%2520not%2520always%2520%25E2%2580%259Dwhat%2520you%2520get%26jtitle%3DStructure%2520%2528Oxford%252C%2520U.%2520K.%2529%26date%3D2009%26volume%3D17%26spage%3D489%26epage%3D498%26doi%3D10.1016%2Fj.str.2009.02.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lexa, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, H. A.</span></span> <span> </span><span class="NLM_article-title">Protein flexibility in docking and surface mapping</span>. <i>Q. Rev. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">301</span>– <span class="NLM_lpage">343</span>, <span class="refDoi"> DOI: 10.1017/S0033583512000066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1017%2FS0033583512000066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=22569329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1WlsLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2012&pages=301-343&author=K.+W.+Lexaauthor=H.+A.+Carlson&title=Protein+flexibility+in+docking+and+surface+mapping&doi=10.1017%2FS0033583512000066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Protein flexibility in docking and surface mapping</span></div><div class="casAuthors">Lexa, Katrina W.; Carlson, Heather A.</div><div class="citationInfo"><span class="NLM_cas:title">Quarterly Reviews of Biophysics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">301-343</span>CODEN:
                <span class="NLM_cas:coden">QURBAW</span>;
        ISSN:<span class="NLM_cas:issn">0033-5835</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review.  Structure-based drug design has become an essential tool for rapid lead discovery and optimization.  As available structural information has increased, researchers have become increasingly aware of the importance of protein flexibility for accurate description of the native state.  Typical protein-ligand docking efforts still rely on a single rigid receptor, which is an incomplete representation of potential binding conformations of the protein.  These rigid docking efforts typically show the best performance rates between 50 and 75%, while fully flexible docking methods can enhance pose prediction up to 80-95%.  This review examines the current toolbox for flexible protein-ligand docking and receptor surface mapping.  Present limitations and possibilities for future development are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWRT8xbQSJ77Vg90H21EOLACvtfcHk0ljltcBoSOHgKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1WlsLzE&md5=868db49242f652c02cc61a7d4f6d3d0f</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1017%2FS0033583512000066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS0033583512000066%26sid%3Dliteratum%253Aachs%26aulast%3DLexa%26aufirst%3DK.%2BW.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DProtein%2520flexibility%2520in%2520docking%2520and%2520surface%2520mapping%26jtitle%3DQ.%2520Rev.%2520Biophys.%26date%3D2012%26volume%3D45%26spage%3D301%26epage%3D343%26doi%3D10.1017%2FS0033583512000066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Damm-Ganamet, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bembenek, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venable, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castro, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangelschots, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peeters, D. C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcallister, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disepio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirzadegan, T.</span></span> <span> </span><span class="NLM_article-title">A prospective virtual screening study: Enriching hit rates and designing focus libraries to find inhibitors of PI3Kδ and PI3Kγ</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4302</span>– <span class="NLM_lpage">4313</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01974</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01974" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC28XltlOmtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4302-4313&author=K.+L.+Damm-Ganametauthor=S.+D.+Bembenekauthor=J.+W.+Venableauthor=G.+G.+Castroauthor=L.+Mangelschotsauthor=D.+C.+G.+Peetersauthor=H.+M.+Mcallisterauthor=J.+P.+Edwardsauthor=D.+Disepioauthor=T.+Mirzadegan&title=A+prospective+virtual+screening+study%3A+Enriching+hit+rates+and+designing+focus+libraries+to+find+inhibitors+of+PI3K%CE%B4+and+PI3K%CE%B3&doi=10.1021%2Facs.jmedchem.5b01974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A Prospective Virtual Screening Study: Enriching Hit Rates and Designing Focus Libraries To Find Inhibitors of PI3Kδ and PI3Kγ</span></div><div class="casAuthors">Damm-Ganamet, Kelly L.; Bembenek, Scott D.; Venable, Jennifer W.; Castro, Glenda G.; Mangelschots, Lieve; Peeters, Danielle C. G.; Mcallister, Heather M.; Edwards, James P.; Disepio, Daniel; Mirzadegan, Taraneh</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4302-4313</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here, we report a high-throughput virtual screening (HTVS) study using phosphoinositide 3-kinase (both PI3Kγ and PI3Kδ).  Our initial HTVS results of the Janssen corporate database identified small focused libraries with hit rates at 50% inhibition showing a 50-fold increase over those from a HTS (high-throughput screen).  Further, applying constraints based on "chem. intuitive" hydrogen bonds and/or positional requirements resulted in a substantial improvement in the hit rates (vs. no constraints) and reduced docking time.  While we find that docking scoring functions are not capable of providing a reliable relative ranking of a set of compds., a prioritization of groups of compds. (e.g., low, medium, and high) does emerge, which allows for the chem. efforts to be quickly focused on the most viable candidates.  Thus, this illustrates that it is not always necessary to have a high correlation between a computational score and the exptl. data to impact the drug discovery process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgMG23GSsrOrVg90H21EOLACvtfcHk0ljltcBoSOHgKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltlOmtLc%253D&md5=809d86cc271e5b4f603691304cceafc5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01974%26sid%3Dliteratum%253Aachs%26aulast%3DDamm-Ganamet%26aufirst%3DK.%2BL.%26aulast%3DBembenek%26aufirst%3DS.%2BD.%26aulast%3DVenable%26aufirst%3DJ.%2BW.%26aulast%3DCastro%26aufirst%3DG.%2BG.%26aulast%3DMangelschots%26aufirst%3DL.%26aulast%3DPeeters%26aufirst%3DD.%2BC.%2BG.%26aulast%3DMcallister%26aufirst%3DH.%2BM.%26aulast%3DEdwards%26aufirst%3DJ.%2BP.%26aulast%3DDisepio%26aufirst%3DD.%26aulast%3DMirzadegan%26aufirst%3DT.%26atitle%3DA%2520prospective%2520virtual%2520screening%2520study%253A%2520Enriching%2520hit%2520rates%2520and%2520designing%2520focus%2520libraries%2520to%2520find%2520inhibitors%2520of%2520PI3K%25CE%25B4%2520and%2520PI3K%25CE%25B3%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4302%26epage%3D4313%26doi%3D10.1021%2Facs.jmedchem.5b01974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lionta, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spyrou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassilatis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cournia, Z.</span></span> <span> </span><span class="NLM_article-title">Structure-based virtual screening for drug discovery: Principles, applications and recent advances</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1923</span>– <span class="NLM_lpage">1938</span>, <span class="refDoi"> DOI: 10.2174/1568026614666140929124445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.2174%2F1568026614666140929124445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=25262799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGgt7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=1923-1938&author=E.+Liontaauthor=G.+Spyrouauthor=D.+Vassilatisauthor=Z.+Cournia&title=Structure-based+virtual+screening+for+drug+discovery%3A+Principles%2C+applications+and+recent+advances&doi=10.2174%2F1568026614666140929124445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Virtual Screening for Drug Discovery: Principles, Applications and Recent Advances</span></div><div class="casAuthors">Lionta, Evanthia; Spyrou, George; Vassilatis, Demetrios K.; Cournia, Zoe</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1923-1938</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Structure-based drug discovery (SBDD) is becoming an essential tool in assisting fast and cost-efficient lead discovery and optimization.  The application of rational, structure-based drug design is proven to be more efficient than the traditional way of drug discovery since it aims to understand the mol. basis of a disease and utilizes the knowledge of the three-dimensional structure of the biol. target in the process.  In this review, we focus on the principles and applications of Virtual Screening (VS) within the context of SBDD and examine different procedures ranging from the initial stages of the process that include receptor and library pre-processing, to docking, scoring and post-processing of top scoring hits.  Recent improvements in structure-based virtual screening (SBVS) efficiency through ensemble docking, induced fit and consensus docking are also discussed.  The review highlights advances in the field within the framework of several success studies that have led to nM inhibition directly from VS and provides recent trends in library design as well as discusses limitations of the method.  Applications of SBVS in the design of substrates for engineered proteins that enable the discovery of new metabolic and signal transduction pathways and the design of inhibitors of multifunctional proteins are also reviewed.  Finally, we contribute two promising VS protocols recently developed by us that aim to increase inhibitor selectivity.  In the first protocol, we describe the discovery of micromolar inhibitors through SBVS designed to inhibit the mutant H1047R PI3Kα kinase.  Second, we discuss a strategy for the identification of selective binders for the RXRα nuclear receptor.  In this protocol, a set of target structures is constructed for ensemble docking based on binding site shape characterization and clustering, aiming to enhance the hit rate of selective inhibitors for the desired protein target through the SBVS process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGps80ptfisjzLVg90H21EOLACvtfcHk0ljltcBoSOHgKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGgt7fJ&md5=0550a396dd011adf52699b091548345b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.2174%2F1568026614666140929124445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026614666140929124445%26sid%3Dliteratum%253Aachs%26aulast%3DLionta%26aufirst%3DE.%26aulast%3DSpyrou%26aufirst%3DG.%26aulast%3DVassilatis%26aufirst%3DD.%26aulast%3DCournia%26aufirst%3DZ.%26atitle%3DStructure-based%2520virtual%2520screening%2520for%2520drug%2520discovery%253A%2520Principles%252C%2520applications%2520and%2520recent%2520advances%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2014%26volume%3D14%26spage%3D1923%26epage%3D1938%26doi%3D10.2174%2F1568026614666140929124445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cournia, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beuming, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearlman, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radak, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Rigorous free energy simulations in virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4153</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c00116</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c00116" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFGgu7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=4153&author=Z.+Courniaauthor=B.+K.+Allenauthor=T.+Beumingauthor=D.+A.+Pearlmanauthor=B.+K.+Radakauthor=W.+Sherman&title=Rigorous+free+energy+simulations+in+virtual+screening&doi=10.1021%2Facs.jcim.0c00116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Rigorous Free Energy Simulations in Virtual Screening</span></div><div class="casAuthors">Cournia, Zoe; Allen, Bryce K.; Beuming, Thijs; Pearlman, David A.; Radak, Brian K.; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4153-4169</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Virtual high throughput screening (vHTS) in drug discovery is a powerful approach to identify hits: when applied successfully, it can be much faster and cheaper than exptl. high-throughput screening approaches.  However, mainstream vHTS tools have significant limitations: ligand-based methods depend on knowledge of existing chem. matter, while structure-based tools such as docking involve significant approxns. that limit their accuracy.  Recent advances in scientific methods coupled with dramatic speedups in computational processing with GPUs make this an opportune time to consider the role of more rigorous methods that could improve the predictive power of vHTS workflows.  In this Perspective, we assert that alchem. binding free energy methods using all-atom mol. dynamics simulations have matured to the point where they can be applied in virtual screening campaigns as a final scoring stage to prioritize the top mols. for exptl. testing.  Specifically, we propose that alchem. abs. binding free energy (ABFE) calcns. offer the most direct and computationally efficient approach within a rigorous statistical thermodn. framework for computing binding energies of diverse mols., as is required for virtual screening.  ABFE calcns. are particularly attractive for drug discovery at this point in time, where the confluence of large-scale genomics data and insights from chem. biol. have unveiled a large no. of promising disease targets for which no small mol. binders are known, precluding ligand-based approaches, and where traditional docking approaches have foundered to find progressible chem. matter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGol_yim2RNVYrVg90H21EOLACvtfcHk0lgfxmPG1gELZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFGgu7fP&md5=4f144e9f9fe045f00e00ded0a5e8d2a1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c00116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c00116%26sid%3Dliteratum%253Aachs%26aulast%3DCournia%26aufirst%3DZ.%26aulast%3DAllen%26aufirst%3DB.%2BK.%26aulast%3DBeuming%26aufirst%3DT.%26aulast%3DPearlman%26aufirst%3DD.%2BA.%26aulast%3DRadak%26aufirst%3DB.%2BK.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DRigorous%2520free%2520energy%2520simulations%2520in%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D4153%26doi%3D10.1021%2Facs.jcim.0c00116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schulz-Gasch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">Scoring functions for protein-ligand interactions: A critical perspective</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">231</span>– <span class="NLM_lpage">239</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2004.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1016%2Fj.ddtec.2004.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=24981490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1ClsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2004&pages=231-239&author=T.+Schulz-Gaschauthor=M.+Stahl&title=Scoring+functions+for+protein-ligand+interactions%3A+A+critical+perspective&doi=10.1016%2Fj.ddtec.2004.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Scoring functions for protein-ligand interactions: a critical perspective</span></div><div class="casAuthors">Schulz-Gasch, Tanja; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today: Technologies</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">231-239</span>CODEN:
                <span class="NLM_cas:coden">DDTTB5</span>;
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Scoring functions play an essential role in structure-based virtual screening.  They are required to guide the docking of candidate compds. to structures of receptor binding sites, to select probable binding modes, and to discriminate binders from non-binders.  Although many scoring functions have successfully been used to identify novel ligands for a wide variety of targets, much work remains to be done to avoid incorrect prediction of binding modes and high nos. of false positives.  This review gives an overview of the current state of the field and outlines key issues for the further development of scoring functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroEKzPtDzEqbVg90H21EOLACvtfcHk0lgfxmPG1gELZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1ClsLs%253D&md5=72849854914ed59b41dc30a5a197943e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2004.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2004.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DSchulz-Gasch%26aufirst%3DT.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DScoring%2520functions%2520for%2520protein-ligand%2520interactions%253A%2520A%2520critical%2520perspective%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2004%26volume%3D1%26spage%3D231%26epage%3D239%26doi%3D10.1016%2Fj.ddtec.2004.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parks, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaieb, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGaughey, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bembenek, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ameriks, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirzadegan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burley, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaro, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span> <span> </span><span class="NLM_article-title">D3R grand challenge 4: Blind prediction of protein-ligand poses, affinity rankings, and relative binding free energies</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1007/s10822-020-00289-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1007%2Fs10822-020-00289-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=31974851" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1SmtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2020&pages=99-119&author=C.+D.+Parksauthor=Z.+Gaiebauthor=M.+Chiuauthor=H.+Yangauthor=C.+Shaoauthor=W.+P.+Waltersauthor=J.+M.+Jansenauthor=G.+McGaugheyauthor=R.+A.+Lewisauthor=S.+D.+Bembenekauthor=M.+K.+Ameriksauthor=T.+Mirzadeganauthor=S.+K.+Burleyauthor=R.+E.+Amaroauthor=M.+K.+Gilson&title=D3R+grand+challenge+4%3A+Blind+prediction+of+protein-ligand+poses%2C+affinity+rankings%2C+and+relative+binding+free+energies&doi=10.1007%2Fs10822-020-00289-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">D3R grand challenge 4: blind prediction of protein-ligand poses, affinity rankings, and relative binding free energies</span></div><div class="casAuthors">Parks, Conor D.; Gaieb, Zied; Chiu, Michael; Yang, Huanwang; Shao, Chenghua; Walters, W. Patrick; Jansen, Johanna M.; McGaughey, Georgia; Lewis, Richard A.; Bembenek, Scott D.; Ameriks, Michael K.; Mirzadegan, Tara; Burley, Stephen K.; Amaro, Rommie E.; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">99-119</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Abstr.: The Drug Design Data Resource (D3R) aims to identify best practice methods for computer aided drug design through blinded ligand pose prediction and affinity challenges.  Herein, we report on the results of Grand Challenge 4 (GC4).  GC4 focused on proteins beta secretase 1 and Cathepsin S, and was run in an analogous manner to prior challenges.  In Stage 1, participant ability to predict the pose and affinity of BACE1 ligands were assessed.  Following the completion of Stage 1, all BACE1 co-crystal structures were released, and Stage 2 tested affinity rankings with co-crystal structures.  We provide an anal. of the results and discuss insights into detd. best practice methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNni47G6bCWrVg90H21EOLACvtfcHk0lgfxmPG1gELZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1SmtL8%253D&md5=57a12faffcb1def7224696bf23a0e599</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1007%2Fs10822-020-00289-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-020-00289-y%26sid%3Dliteratum%253Aachs%26aulast%3DParks%26aufirst%3DC.%2BD.%26aulast%3DGaieb%26aufirst%3DZ.%26aulast%3DChiu%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DShao%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DW.%2BP.%26aulast%3DJansen%26aufirst%3DJ.%2BM.%26aulast%3DMcGaughey%26aufirst%3DG.%26aulast%3DLewis%26aufirst%3DR.%2BA.%26aulast%3DBembenek%26aufirst%3DS.%2BD.%26aulast%3DAmeriks%26aufirst%3DM.%2BK.%26aulast%3DMirzadegan%26aufirst%3DT.%26aulast%3DBurley%26aufirst%3DS.%2BK.%26aulast%3DAmaro%26aufirst%3DR.%2BE.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DD3R%2520grand%2520challenge%25204%253A%2520Blind%2520prediction%2520of%2520protein-ligand%2520poses%252C%2520affinity%2520rankings%252C%2520and%2520relative%2520binding%2520free%2520energies%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2020%26volume%3D34%26spage%3D99%26epage%3D119%26doi%3D10.1007%2Fs10822-020-00289-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaieb, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parks, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nevins, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bembenek, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ameriks, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirzadegan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burley, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaro, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span> <span> </span><span class="NLM_article-title">D3R grand challenge 3: Blind prediction of protein-ligand poses and affinity rankings</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.1007/s10822-018-0180-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1007%2Fs10822-018-0180-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=30632055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFShtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=1-18&author=Z.+Gaiebauthor=C.+D.+Parksauthor=M.+Chiuauthor=H.+Yangauthor=C.+Shaoauthor=W.+P.+Waltersauthor=M.+H.+Lambertauthor=N.+Nevinsauthor=S.+D.+Bembenekauthor=M.+K.+Ameriksauthor=T.+Mirzadeganauthor=S.+K.+Burleyauthor=R.+E.+Amaroauthor=M.+K.+Gilson&title=D3R+grand+challenge+3%3A+Blind+prediction+of+protein-ligand+poses+and+affinity+rankings&doi=10.1007%2Fs10822-018-0180-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">D3R Grand Challenge 3: blind prediction of protein-ligand poses and affinity rankings</span></div><div class="casAuthors">Gaieb, Zied; Parks, Conor D.; Chiu, Michael; Yang, Huanwang; Shao, Chenghua; Walters, W. Patrick; Lambert, Millard H.; Nevins, Neysa; Bembenek, Scott D.; Ameriks, Michael K.; Mirzadegan, Tara; Burley, Stephen K.; Amaro, Rommie E.; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-18</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The Drug Design Data Resource aims to test and advance the state of the art in protein-ligand modeling by holding community-wide blinded, prediction challenges.  Here, we report on our third major round, Grand Challenge 3 (GC3).  Held 2017-2018, GC3 centered on the protein Cathepsin S and the kinases VEGFR2, JAK2, p38-α, TIE2, and ABL1, and included both pose-prediction and affinity-ranking components.  GC3 was structured much like the prior challenges GC2015 and GC2.  First, Stage 1 tested pose prediction and affinity ranking methods; then all available crystal structures were released, and Stage 2 tested only affinity rankings, now in the context of the available structures.  Unique to GC3 was the addn. of a Stage 1b self-docking subchallenge, in which the protein coordinates from all of the cocrystal structures used in the cross-docking challenge were released, and participants were asked to predict the pose of CatS ligands using these newly released structures.  We provide an overview of the outcomes and discuss insights into trends and best-practices.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJKkuRnCIHS7Vg90H21EOLACvtfcHk0lifzC-n4azV-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFShtbc%253D&md5=110f1b35386467b53febcaf8d2fcdea7</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1007%2Fs10822-018-0180-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-018-0180-4%26sid%3Dliteratum%253Aachs%26aulast%3DGaieb%26aufirst%3DZ.%26aulast%3DParks%26aufirst%3DC.%2BD.%26aulast%3DChiu%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DShao%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DW.%2BP.%26aulast%3DLambert%26aufirst%3DM.%2BH.%26aulast%3DNevins%26aufirst%3DN.%26aulast%3DBembenek%26aufirst%3DS.%2BD.%26aulast%3DAmeriks%26aufirst%3DM.%2BK.%26aulast%3DMirzadegan%26aufirst%3DT.%26aulast%3DBurley%26aufirst%3DS.%2BK.%26aulast%3DAmaro%26aufirst%3DR.%2BE.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DD3R%2520grand%2520challenge%25203%253A%2520Blind%2520prediction%2520of%2520protein-ligand%2520poses%2520and%2520affinity%2520rankings%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2019%26volume%3D33%26spage%3D1%26epage%3D18%26doi%3D10.1007%2Fs10822-018-0180-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, A. M.</span>; <span class="NLM_string-name">Teague, S. J.</span>; <span class="NLM_string-name">Kleywegt, G. J.</span></span> <span> </span><span class="NLM_article-title">Application and limitations of X-ray crystallographic data in structure-guided ligand and drug design</span>. <span class="NLM_year">2007</span>. <a href="http://europepmc.org/abstract/CTX/C7868" class="extLink">http://europepmc.org/abstract/CTX/C7868</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=A.+M.+Davis&author=S.+J.+Teague&author=G.+J.+Kleywegt&title=Application+and+limitations+of+X-ray+crystallographic+data+in+structure-guided+ligand+and+drug+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DA.%2BM.%26atitle%3DApplication%2520and%2520limitations%2520of%2520X-ray%2520crystallographic%2520data%2520in%2520structure-guided%2520ligand%2520and%2520drug%2520design%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ripphausen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Analysis of structure-based virtual screening studies and characterization of identified active compounds</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">613</span>, <span class="refDoi"> DOI: 10.4155/fmc.12.18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.4155%2Ffmc.12.18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=22458680" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC38Xks1Oqtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=603-613&author=P.+Ripphausenauthor=D.+Stumpfeauthor=J.+Bajorath&title=Analysis+of+structure-based+virtual+screening+studies+and+characterization+of+identified+active+compounds&doi=10.4155%2Ffmc.12.18"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of structure-based virtual screening studies and characterization of identified active compounds</span></div><div class="casAuthors">Ripphausen, Peter; Stumpfe, Dagmar; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">603-613</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Structure-based virtual screening makes explicit or implicit use of 3D target structure information to detect novel active compds.  Results of nearly 300 currently available original applications have been analyzed to characterize the state-of-the-art in this field.  Compd. selection from docking calcns. is much influenced by subjective criteria.  Although submicromolar compds. are identified, the majority of docking hits are only weakly potent.  However, only a small percentage of docking hits can be reproduced by ligand-based methods.  When docking calcns. identify potent hits, they often originate from specialized compd. sources (e.g., pharmaceutical compd. decks or target-focused libraries) and also display a notable bias towards kinase targets.  Structure-based virtual screening is the dominant approach to computational hit identification.  Docking calcns. frequently identify active compds.  Limited accuracy of compd. scoring and ranking currently presents a major caveat of the approach that is often compensated for by chem. intuition and knowledge.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHjToM3jECobVg90H21EOLACvtfcHk0lifzC-n4azV-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xks1Oqtbw%253D&md5=d2d0bc27efbbcdd34fcde4da37e5e192</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.18%26sid%3Dliteratum%253Aachs%26aulast%3DRipphausen%26aufirst%3DP.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DAnalysis%2520of%2520structure-based%2520virtual%2520screening%2520studies%2520and%2520characterization%2520of%2520identified%2520active%2520compounds%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D603%26epage%3D613%26doi%3D10.4155%2Ffmc.12.18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaturvedi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatnagar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqi, M. I.</span></span> <span> </span><span class="NLM_article-title">Knowledge based identification of potent antitubercular compounds using structure based virtual screening and structure interaction fingerprints</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">35</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1021/ci8003607</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci8003607" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVOmtrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=35-42&author=A.+Kumarauthor=V.+Chaturvediauthor=S.+Bhatnagarauthor=S.+Sinhaauthor=M.+I.+Siddiqi&title=Knowledge+based+identification+of+potent+antitubercular+compounds+using+structure+based+virtual+screening+and+structure+interaction+fingerprints&doi=10.1021%2Fci8003607"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Knowledge Based Identification of Potent Antitubercular Compounds Using Structure Based Virtual Screening and Structure Interaction Fingerprints</span></div><div class="casAuthors">Kumar, Ashutosh; Chaturvedi, Vinita; Bhatnagar, Shalini; Sinha, Sudhir; Siddiqi, Mohammad Imran</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">35-42</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In view of the worldwide spread of multidrug resistance of Mycobacterium tuberculosis, there is an urgent need to discover antitubercular agents with novel structures.  Thymidine monophosphate kinase from M. tuberculosis (TMPKmt) is an attractive target for antitubercular chemotherapy.  We report here the identification of potent antitubercular compds. targeting TMPKmt using virtual screening methods.  For this purpose we have developed a pharmacophore hypothesis based on the substrate and known TMPKmt inhibitors and employed it to screen the Maybridge small mol. database.  The mol. docking was then performed in order to select the compds. on the basis of their ability to form favorable interactions with the TMPKmt active site.  In addn., we applied straightforward weighting using structure interaction fingerprints to include addnl. knowledge into structure based virtual screening.  Eight compds. were acquired and evaluated for antitubercular activity against M. tuberculosis H37Rv in vitro, and out of these 3 compds. showed MIC of 3.12 μg/mL whereas 2 compds. showed MIC of 12.5 μg/mL.  All the active compds. were found to be nontoxic in Vero cell lines and mice bone marrow macrophages.  All the identified hits highlighted a key hydrogen bonding interaction with Arg74.  The obsd. π-stacking interaction with Phe70 was also produced by the identified hits.  These hits represent promising starting points for structural optimization in hit-to-lead development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4p1Vh5kNAqrVg90H21EOLACvtfcHk0lifzC-n4azV-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVOmtrfE&md5=c0dc91a59fa31b396647c01150dd2641</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fci8003607&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci8003607%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DChaturvedi%26aufirst%3DV.%26aulast%3DBhatnagar%26aufirst%3DS.%26aulast%3DSinha%26aufirst%3DS.%26aulast%3DSiddiqi%26aufirst%3DM.%2BI.%26atitle%3DKnowledge%2520based%2520identification%2520of%2520potent%2520antitubercular%2520compounds%2520using%2520structure%2520based%2520virtual%2520screening%2520and%2520structure%2520interaction%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D35%26epage%3D42%26doi%3D10.1021%2Fci8003607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez-Pérez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miljković, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Assessing the information content of structural and protein-ligand interaction representations for the classification of kinase inhibitor binding modes via machine learning and active learning</span>. <i>J. Cheminf.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">36</span>, <span class="refDoi"> DOI: 10.1186/s13321-020-00434-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1186%2Fs13321-020-00434-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVyks7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&pages=36&author=R.+Rodr%C3%ADguez-P%C3%A9rezauthor=F.+Miljkovi%C4%87author=J.+Bajorath&title=Assessing+the+information+content+of+structural+and+protein-ligand+interaction+representations+for+the+classification+of+kinase+inhibitor+binding+modes+via+machine+learning+and+active+learning&doi=10.1186%2Fs13321-020-00434-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing the information content of structural and protein-ligand interaction representations for the classification of kinase inhibitor binding modes via machine learning and active learning</span></div><div class="casAuthors">Rodriguez-Perez, Raquel; Miljkovic, Filip; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">SpringerOpen</span>)
        </div><div class="casAbstract">We have addressed this prediction task to evaluate and compare the information content of distinct mol. representations including protein-ligand interaction fingerprints and compd. structure-based structural fingerprints (i.e., atom environment/fragment fingerprints).  Accurate predictions of kinase inhibitor binding modes were achieved with random forests using both representations.  The performance of IFPs was consistently superior to atom environment fingerprints, albeit only by less than 10%.  An active learning strategy applying information entropy-based selection of training instances was applied as a diagnostic approach to assess the relative information content of distinct representations.  IFPs were found to capture more binding mode-relevant information than atom environment fingerprints, leading to highly predictive models even when training instances were randomly selected.  By contrast, for atom environment fingerprints, the derivation of accurate models via active learning depended on entropy-based selection of informative training compds.  Notably, higher information content of IFPs confirmed by active learning only resulted in small improvements in global prediction accuracy compared to models derived using atom environment fingerprints.  For practical applications, prediction of binding modes of new kinase inhibitors on the basis of chem. structure is highly attractive.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOV05t-VYny7Vg90H21EOLACvtfcHk0lhw2Tpib5y-dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVyks7vN&md5=c9f9b03c5b347c2642fee97b493b0b00</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1186%2Fs13321-020-00434-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-020-00434-7%26sid%3Dliteratum%253Aachs%26aulast%3DRodr%25C3%25ADguez-P%25C3%25A9rez%26aufirst%3DR.%26aulast%3DMiljkovi%25C4%2587%26aufirst%3DF.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DAssessing%2520the%2520information%2520content%2520of%2520structural%2520and%2520protein-ligand%2520interaction%2520representations%2520for%2520the%2520classification%2520of%2520kinase%2520inhibitor%2520binding%2520modes%2520via%2520machine%2520learning%2520and%2520active%2520learning%26jtitle%3DJ.%2520Cheminf.%26date%3D2020%26volume%3D12%26spage%3D36%26doi%3D10.1186%2Fs13321-020-00434-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Varela-Rial, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majewski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuzzolin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Rosell, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Fabritiis, G.</span></span> <span> </span><span class="NLM_article-title">SkeleDock: A web application for scaffold docking in PlayMolecule</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2673</span>– <span class="NLM_lpage">2677</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c00143</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c00143" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpt1Ghur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=2673-2677&author=A.+Varela-Rialauthor=M.+Majewskiauthor=A.+Cuzzolinauthor=G.+Mart%C3%ADnez-Rosellauthor=G.+de+Fabritiis&title=SkeleDock%3A+A+web+application+for+scaffold+docking+in+PlayMolecule&doi=10.1021%2Facs.jcim.0c00143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">SkeleDock: A Web Application for Scaffold Docking in PlayMolecule</span></div><div class="casAuthors">Varela-Rial, Alejandro; Majewski, Maciej; Cuzzolin, Alberto; Martinez-Rosell, Gerard; De Fabritiis, Gianni</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2673-2677</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SkeleDock is a scaffold docking algorithm which uses the structure of a protein-ligand complex as a template to model the binding mode of a chem. similar system.  This algorithm was evaluated in the D3R Grand Challenge 4 pose prediction challenge, where it achieved competitive performance.  Furthermore, the authors show that if crystd. fragments of the target ligand are available then SkeleDock can outperform rDock docking software at predicting the binding mode.  This Application Note also addresses the capacity of this algorithm to model macrocycles and deal with scaffold hopping.  SkeleDock can be accessed at https://playmol.org/SkeleDock/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgUgig_dif1LVg90H21EOLACvtfcHk0lhw2Tpib5y-dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpt1Ghur4%253D&md5=a15dda548cfa1d9e6891d116b61bcdb8</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c00143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c00143%26sid%3Dliteratum%253Aachs%26aulast%3DVarela-Rial%26aufirst%3DA.%26aulast%3DMajewski%26aufirst%3DM.%26aulast%3DCuzzolin%26aufirst%3DA.%26aulast%3DMart%25C3%25ADnez-Rosell%26aufirst%3DG.%26aulast%3Dde%2BFabritiis%26aufirst%3DG.%26atitle%3DSkeleDock%253A%2520A%2520web%2520application%2520for%2520scaffold%2520docking%2520in%2520PlayMolecule%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D2673%26epage%3D2677%26doi%3D10.1021%2Facs.jcim.0c00143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velázquez-Libera, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durán-Verdugo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valdés-Jiménez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Núñez-Vivanco, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caballero, J.</span></span> <span> </span><span class="NLM_article-title">LigRMSD: A web server for automatic structure matching and RMSD calculations among identical and similar compounds in protein-ligand docking</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">2912</span>– <span class="NLM_lpage">2914</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btaa018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1093%2Fbioinformatics%2Fbtaa018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=31926012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitlOlsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2020&pages=2912-2914&author=J.+L.+Vel%C3%A1zquez-Liberaauthor=F.+Dur%C3%A1n-Verdugoauthor=A.+Vald%C3%A9s-Jim%C3%A9nezauthor=G.+N%C3%BA%C3%B1ez-Vivancoauthor=J.+Caballero&title=LigRMSD%3A+A+web+server+for+automatic+structure+matching+and+RMSD+calculations+among+identical+and+similar+compounds+in+protein-ligand+docking&doi=10.1093%2Fbioinformatics%2Fbtaa018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">LigRMSD: a web server for automatic structure matching and RMSD calculations among identical and similar compounds in protein-ligand docking</span></div><div class="casAuthors">Velazquez-Libera, Jose Luis; Duran-Verdugo, Fabio; Valdes-Jimenez, Alejandro; Nunez-Vivanco, Gabriel; Caballero, Julio</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2912-2914</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4811</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Root mean square deviation (RMSD) is one of the most useful and straightforward features for structural comparison between different conformations of the same mol.  Commonly, protein-ligand docking programs have included some utilities that allow the calcn. of this value; however, they only work efficiently when exists a complete atom label equivalence between the evaluated conformations.  Results: We present LigRMSD, a free web-server for the automatic matching and RMSD calcns. among identical or similar chem. compds.  This server allows the user to submit only a pair of identical or similar mols. or dataset of similar compds. to compare their three-dimensional conformations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd-lb-rDFcrbVg90H21EOLACvtfcHk0liGLWQb1--lug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitlOlsr7I&md5=1c115dd1028c339cbcbf441e7f263e0a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtaa018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtaa018%26sid%3Dliteratum%253Aachs%26aulast%3DVel%25C3%25A1zquez-Libera%26aufirst%3DJ.%2BL.%26aulast%3DDur%25C3%25A1n-Verdugo%26aufirst%3DF.%26aulast%3DVald%25C3%25A9s-Jim%25C3%25A9nez%26aufirst%3DA.%26aulast%3DN%25C3%25BA%25C3%25B1ez-Vivanco%26aufirst%3DG.%26aulast%3DCaballero%26aufirst%3DJ.%26atitle%3DLigRMSD%253A%2520A%2520web%2520server%2520for%2520automatic%2520structure%2520matching%2520and%2520RMSD%2520calculations%2520among%2520identical%2520and%2520similar%2520compounds%2520in%2520protein-ligand%2520docking%26jtitle%3DBioinformatics%26date%3D2020%26volume%3D36%26spage%3D2912%26epage%3D2914%26doi%3D10.1093%2Fbioinformatics%2Fbtaa018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perryman, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekins, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freundlich, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonge, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">A virtual screen discovers novel, fragment-sized inhibitors of Mycobacterium tuberculosis InhA</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">645</span>– <span class="NLM_lpage">659</span>, <span class="refDoi"> DOI: 10.1021/ci500672v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500672v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVehsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=645-659&author=A.+L.+Perrymanauthor=W.+Yuauthor=X.+Wangauthor=S.+Ekinsauthor=S.+Forliauthor=S.-G.+Liauthor=J.+S.+Freundlichauthor=P.+J.+Tongeauthor=A.+J.+Olson&title=A+virtual+screen+discovers+novel%2C+fragment-sized+inhibitors+of+Mycobacterium+tuberculosis+InhA&doi=10.1021%2Fci500672v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A Virtual Screen Discovers Novel, Fragment-Sized Inhibitors of Mycobacterium tuberculosis InhA</span></div><div class="casAuthors">Perryman, Alexander L.; Yu, Weixuan; Wang, Xin; Ekins, Sean; Forli, Stefano; Li, Shao-Gang; Freundlich, Joel S.; Tonge, Peter J.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">645-659</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Isoniazid (INH) is usually administered to treat latent Mycobacterium tuberculosis (Mtb) infections and is used in combination therapy to treat active tuberculosis (TB).  Unfortunately, resistance to this drug is hampering its clin. effectiveness.  INH is a prodrug that must be activated by Mtb catalase-peroxidase (KatG) before it can inhibit InhA (Mtb enoyl-acyl-carrier-protein reductase).  Isoniazid-resistant cases of TB found in clin. settings usually involve mutations in or deletion of katG, which abrogate INH activation.  Compds. that inhibit InhA without requiring prior activation by KatG would not be affected by this resistance mechanism and hence would display continued potency against these drug-resistant isolates of Mtb.  Virtual screening expts. vs. InhA in the GO Fight Against Malaria (GO FAM) project were designed to discover new scaffolds that display base-stacking interactions with the NAD cofactor.  GO FAM expts. included targets from other pathogens, including Mtb, when they had structural similarity to a malaria target.  Eight of the 16 sol. compds. identified by docking against InhA plus visual inspection were modest inhibitors and did not require prior activation by KatG.  The best two inhibitors discovered are both fragment-sized compds. and displayed Ki values of 54 and 59 μM, resp.  Importantly, the novel inhibitors discovered have low structural similarity to known InhA inhibitors and thus help expand the no. of chemotypes on which future medicinal chem. efforts can be focused.  These new fragment hits could eventually help advance the fight against INH-resistant Mtb strains, which pose a significant global health threat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9fifQHkgzX7Vg90H21EOLACvtfcHk0liGLWQb1--lug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVehsrc%253D&md5=f68a7af63dca612e3017260c32593858</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fci500672v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500672v%26sid%3Dliteratum%253Aachs%26aulast%3DPerryman%26aufirst%3DA.%2BL.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DEkins%26aufirst%3DS.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DS.-G.%26aulast%3DFreundlich%26aufirst%3DJ.%2BS.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DA%2520virtual%2520screen%2520discovers%2520novel%252C%2520fragment-sized%2520inhibitors%2520of%2520Mycobacterium%2520tuberculosis%2520InhA%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D645%26epage%3D659%26doi%3D10.1021%2Fci500672v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Viviani, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piccirillo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulrich, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaral, A. T.-d.</span></span> <span> </span><span class="NLM_article-title">Virtual screening approach for the identification of hydroxamic acids as novel human ecto-5-nucleotidase inhibitors</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">621</span>– <span class="NLM_lpage">630</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.9b00884</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.9b00884" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFCns7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=621-630&author=L.+G.+Vivianiauthor=E.+Piccirilloauthor=H.+Ulrichauthor=A.+T.-d.+Amaral&title=Virtual+screening+approach+for+the+identification+of+hydroxamic+acids+as+novel+human+ecto-5-nucleotidase+inhibitors&doi=10.1021%2Facs.jcim.9b00884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Screening Approach for the Identification of Hydroxamic Acids as Novel Human Ecto-5'-Nucleotidase Inhibitors</span></div><div class="casAuthors">Viviani, Lucas G.; Piccirillo, Erika; Ulrich, Henning; Amaral, Antonia T.-do</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">621-630</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ecto-5'-nucleotidase (ecto-5'-NT, CD73) is a zinc-binding metallophosphatase that plays a key role in extracellular purinergic pathways, being implicated in several physiol. and pathophysiol. processes, such as immune homeostasis, inflammation, and tumor progression.  As such, it has been recognized as a promising biol. target for many diseases, including cancer, infections, and autoimmune diseases.  Despite its importance, so far only a few inhibitors of this target enzyme are known, most of which are not suitable as drug candidates.  Here, we aimed to search for hydroxamic acid-contg. compds. as potential human ecto-5'-NT inhibitors, since this group is known to be a strong zinc chelator.  To this end, we performed a hierarchical virtual screening (VS) search consisting of three consecutive steps (filtering for compds. bearing a hydroxamic acid group, shape-based matching, and docking followed by visual inspection), which were applied to screen the ZINC-14 database ("all purchasable subset").  Out of 25 compds. selected by this VS protocol, 12 were acquired and further submitted to enzymic assays for VS exptl. validation.  Four of them (i.e., 33.3%) were found to inhibit human ecto-5'-NT in the low micromolar range.  The most potent one showed an IC50 value of 6.2 ± 1.0μM.  All identified inhibitors satisfy drug-like criteria and provide novel scaffolds to be explored in further hit-to-lead optimization steps.  Furthermore, to the best of our knowledge, they are the first hydroxamic acid-contg. inhibitors of human ecto-5'-NT described so far.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP4Qiu0YGYKLVg90H21EOLACvtfcHk0liGLWQb1--lug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFCns7zJ&md5=e27572f72ce511b572a671b2aa4b6448</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.9b00884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.9b00884%26sid%3Dliteratum%253Aachs%26aulast%3DViviani%26aufirst%3DL.%2BG.%26aulast%3DPiccirillo%26aufirst%3DE.%26aulast%3DUlrich%26aufirst%3DH.%26aulast%3DAmaral%26aufirst%3DA.%2BT.-d.%26atitle%3DVirtual%2520screening%2520approach%2520for%2520the%2520identification%2520of%2520hydroxamic%2520acids%2520as%2520novel%2520human%2520ecto-5-nucleotidase%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D621%26epage%3D630%26doi%3D10.1021%2Facs.jcim.9b00884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toledo-Sherman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deretey, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slon-Usakiewicz, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, J.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redden, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uger, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasternak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, N.</span></span> <span> </span><span class="NLM_article-title">Frontal affinity chromatography with MS detection of EphB2 tyrosine kinase receptor. 2. identification of small-molecule inhibitors via coupling with virtual screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">3221</span>– <span class="NLM_lpage">3230</span>, <span class="refDoi"> DOI: 10.1021/jm0492204</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0492204" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtFSjtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=3221-3230&author=L.+Toledo-Shermanauthor=E.+Dereteyauthor=J.+J.+Slon-Usakiewiczauthor=W.+Ngauthor=J.-R.+Daiauthor=J.+E.+Fosterauthor=P.+R.+Reddenauthor=M.+D.+Ugerauthor=L.+C.+Liaoauthor=A.+Pasternakauthor=N.+Reid&title=Frontal+affinity+chromatography+with+MS+detection+of+EphB2+tyrosine+kinase+receptor.+2.+identification+of+small-molecule+inhibitors+via+coupling+with+virtual+screening&doi=10.1021%2Fjm0492204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Frontal Affinity Chromatography with MS Detection of EphB2 Tyrosine Kinase Receptor. 2. Identification of Small-Molecule Inhibitors via Coupling with Virtual Screening</span></div><div class="casAuthors">Toledo-Sherman, Leticia; Deretey, Eugen; Slon-Usakiewicz, Jacek J.; Ng, William; Dai, Jin-Rui; Foster, J. Estelle; Redden, Peter R.; Uger, Marni D.; Liao, Linda C.; Pasternak, Andrew; Reid, Neil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3221-3230</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have integrated two complementary methods, high-throughput virtual screening with a "high-content" wet screening technique based on frontal affinity chromatog. with mass spectrometry detection (FAC-MS), for identification of hits against the erythropoietin-producing hepatocellular B2 (EphB2) receptor tyrosine kinase domain.  Both an EphB2-directed virtual screen combining docking and scoring and a kinase-directed pharmacophore search strategy were used to identify a compd. set enriched in bioactive compds. against EphB2.  The coupling of virtual screening methodologies with FAC-MS is a unique hybrid approach that can be used to increase the efficacy of both hit discovery and optimization efforts in drug discovery and has successfully identified hits, in particular (I)(36% shift, IC50 = 5.2 μM, Kd = 3.3 μM), as inhibitors for EphB2, a potential cancer target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgtP-ON3dk-7Vg90H21EOLACvtfcHk0li6GgGbPXb6zQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtFSjtbg%253D&md5=b73aba584c10ef6d11d68b1701d3668d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm0492204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0492204%26sid%3Dliteratum%253Aachs%26aulast%3DToledo-Sherman%26aufirst%3DL.%26aulast%3DDeretey%26aufirst%3DE.%26aulast%3DSlon-Usakiewicz%26aufirst%3DJ.%2BJ.%26aulast%3DNg%26aufirst%3DW.%26aulast%3DDai%26aufirst%3DJ.-R.%26aulast%3DFoster%26aufirst%3DJ.%2BE.%26aulast%3DRedden%26aufirst%3DP.%2BR.%26aulast%3DUger%26aufirst%3DM.%2BD.%26aulast%3DLiao%26aufirst%3DL.%2BC.%26aulast%3DPasternak%26aufirst%3DA.%26aulast%3DReid%26aufirst%3DN.%26atitle%3DFrontal%2520affinity%2520chromatography%2520with%2520MS%2520detection%2520of%2520EphB2%2520tyrosine%2520kinase%2520receptor.%25202.%2520identification%2520of%2520small-molecule%2520inhibitors%2520via%2520coupling%2520with%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D3221%26epage%3D3230%26doi%3D10.1021%2Fjm0492204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Postigo, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guido, R. V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliva, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castilho, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">da R.
Pitta, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Albuquerque, J. F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andricopulo, A. D.</span></span> <span> </span><span class="NLM_article-title">Discovery of new inhibitors of Schistosoma mansoni PNP by pharmacophore-based virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1693</span>– <span class="NLM_lpage">1705</span>, <span class="refDoi"> DOI: 10.1021/ci100128k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100128k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvVyltLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1693-1705&author=M.+P.+Postigoauthor=R.+V.+C.+Guidoauthor=G.+Olivaauthor=M.+S.+Castilhoauthor=I.+da+R.%0APittaauthor=J.+F.+C.+de+Albuquerqueauthor=A.+D.+Andricopulo&title=Discovery+of+new+inhibitors+of+Schistosoma+mansoni+PNP+by+pharmacophore-based+virtual+screening&doi=10.1021%2Fci100128k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of New Inhibitors of Schistosoma mansoni PNP by Pharmacophore-Based Virtual Screening</span></div><div class="casAuthors">Postigo, Matheus P.; Guido, Rafael V. C.; Oliva, Glaucius; Castilho, Marcelo S.; Pitta, Ivan da R.; de Albuquerque, Julianna F. C.; Andricopulo, Adriano D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1693-1705</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Schistosomiasis is considered the second most important tropical parasitic disease, with severe socioeconomic consequences for millions of people worldwide.  Schistosoma mansoni, one of the causative agents of human schistosomiasis, is unable to synthesize purine nucleotides de novo, which makes the enzymes of the purine salvage pathway important targets for antischistosomal drug development.  In the present work, we describe the development of a pharmacophore model for ligands of S. mansoni purine nucleoside phosphorylase (SmPNP) as well as a pharmacophore-based virtual screening approach, which resulted in the identification of three thioxothiazolidinones (1-3) with substantial in vitro inhibitory activity against SmPNP.  Synthesis, biochem. evaluation, and structure-activity relationship investigations led to the successful development of a small set of thioxothiazolidinone derivs. harboring a novel chem. scaffold as new competitive inhibitors of SmPNP at the low-micromolar range.  Seven compds. were identified with IC50 values below 100 μM.  The most potent inhibitors 7, 10, and 17 with IC50 of 2, 18, and 38 μM, resp., could represent new potential lead compds. for further development of the therapy of schistosomiasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiGw2pRusgirVg90H21EOLACvtfcHk0li6GgGbPXb6zQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvVyltLk%253D&md5=bb68b1dec1359a46d7f2c388a5a4dc72</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fci100128k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100128k%26sid%3Dliteratum%253Aachs%26aulast%3DPostigo%26aufirst%3DM.%2BP.%26aulast%3DGuido%26aufirst%3DR.%2BV.%2BC.%26aulast%3DOliva%26aufirst%3DG.%26aulast%3DCastilho%26aufirst%3DM.%2BS.%26aulast%3Dda%2BR.%2BPitta%26aufirst%3DI.%26aulast%3Dde%2BAlbuquerque%26aufirst%3DJ.%2BF.%2BC.%26aulast%3DAndricopulo%26aufirst%3DA.%2BD.%26atitle%3DDiscovery%2520of%2520new%2520inhibitors%2520of%2520Schistosoma%2520mansoni%2520PNP%2520by%2520pharmacophore-based%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1693%26epage%3D1705%26doi%3D10.1021%2Fci100128k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caporuscio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbriano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">del Rio, A.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of potent aromatase inhibitors by high-throughput docking</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">4006</span>– <span class="NLM_lpage">4017</span>, <span class="refDoi"> DOI: 10.1021/jm2000689</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2000689" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmsFylsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=4006-4017&author=F.+Caporuscioauthor=G.+Rastelliauthor=C.+Imbrianoauthor=A.+del+Rio&title=Structure-based+design+of+potent+aromatase+inhibitors+by+high-throughput+docking&doi=10.1021%2Fjm2000689"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Potent Aromatase Inhibitors by High-Throughput Docking</span></div><div class="casAuthors">Caporuscio, Fabiana; Rastelli, Giulio; Imbriano, Carol; Del Rio, Alberto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4006-4017</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cytochrome P 450 aromatase catalyzes the conversion of androgen substrates into estrogens.  Aromatase inhibitors (AIs) have been used as first-line drugs in the treatment of estrogen-dependent breast cancer in postmenopausal women.  However, the search for new, more potent, and selective AIs still remains necessary to avoid the risk of possible resistances and reduce toxicity and side effects of current available drugs.  The publication of a high resoln. x-ray structure of human aromatase has opened the way to structure-based virtual screening to identify new small-mol. inhibitors with structural motifs different from all known AIs.  In this context, a high-throughput docking protocol was set up and led to the identification of nanomolar AIs with new core structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBteVWbhM8PLVg90H21EOLACvtfcHk0li6GgGbPXb6zQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmsFylsL8%253D&md5=93b8b8a30807469412b6a88db79b4a4c</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm2000689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2000689%26sid%3Dliteratum%253Aachs%26aulast%3DCaporuscio%26aufirst%3DF.%26aulast%3DRastelli%26aufirst%3DG.%26aulast%3DImbriano%26aufirst%3DC.%26aulast%3Ddel%2BRio%26aufirst%3DA.%26atitle%3DStructure-based%2520design%2520of%2520potent%2520aromatase%2520inhibitors%2520by%2520high-throughput%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D4006%26epage%3D4017%26doi%3D10.1021%2Fjm2000689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumari, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span> <span> </span><span class="NLM_article-title">Drug design targeting protein-protein interactions (PPIs) Using multiple ligand simultaneous docking (MLSD) and drug repositioning: Discovery of raloxifene and bazedoxifene as novel inhibitors of IL-6/GP130 interface</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">632</span>– <span class="NLM_lpage">641</span>, <span class="refDoi"> DOI: 10.1021/jm401144z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401144z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1Ohtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=632-641&author=H.+Liauthor=H.+Xiaoauthor=L.+Linauthor=D.+Jouauthor=V.+Kumariauthor=J.+Linauthor=C.+Li&title=Drug+design+targeting+protein-protein+interactions+%28PPIs%29+Using+multiple+ligand+simultaneous+docking+%28MLSD%29+and+drug+repositioning%3A+Discovery+of+raloxifene+and+bazedoxifene+as+novel+inhibitors+of+IL-6%2FGP130+interface&doi=10.1021%2Fjm401144z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Design Targeting Protein-Protein Interactions (PPIs) Using Multiple Ligand Simultaneous Docking (MLSD) and Drug Repositioning: Discovery of Raloxifene and Bazedoxifene as Novel Inhibitors of IL-6/GP130 Interface</span></div><div class="casAuthors">Li, Huameng; Xiao, Hui; Lin, Li; Jou, David; Kumari, Vandana; Lin, Jiayuh; Li, Chenglong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">632-641</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The IL-6/GP130/STAT3 pathway is crit. for the progression of multiple types of cancers.  The authors report the discovery of raloxifene and bazedoxifene as novel inhibitors of IL-6/GP130 protein-protein interactions (PPIs) using multiple ligand simultaneous docking (MLSD) and drug repositioning approaches.  Multiple drug scaffolds were simultaneously docked into hot spots of GP130 D1 domain by MLSD to compete with the key interacting residues of IL-6, followed by tethering to generate virtual hit compds.  Similarity searches of virtual hits on drug databases identified raloxifene and bazedoxifene as potential inhibitors of IL-6/GP130 interaction.  In cancer cell assays both compds. bind to GP130 and demonstrated selective inhibition of IL-6 induced STAT3 phosphorylation and were significantly more potent than the previously reported natural product inhibitor MDL-A.  The identified drugs represent a new class of lead compds. with piperidine, benzothiophene, and indole scaffolds to inhibit IL-6 induced homodimerization of GP130.  Besides potential direct usage for clinic trials, the two compds. can also serve as lead compds. for optimization to speed the development of drugs selectively targeting the IL-6/GP130/STAT3 cancer signaling pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZJ8PO1lH9g7Vg90H21EOLACvtfcHk0lgYC0G5lwmJ9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1Ohtr8%253D&md5=a4f2887c2ad3a5dc97827e7a7499efe0</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm401144z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401144z%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DXiao%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DL.%26aulast%3DJou%26aufirst%3DD.%26aulast%3DKumari%26aufirst%3DV.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DC.%26atitle%3DDrug%2520design%2520targeting%2520protein-protein%2520interactions%2520%2528PPIs%2529%2520Using%2520multiple%2520ligand%2520simultaneous%2520docking%2520%2528MLSD%2529%2520and%2520drug%2520repositioning%253A%2520Discovery%2520of%2520raloxifene%2520and%2520bazedoxifene%2520as%2520novel%2520inhibitors%2520of%2520IL-6%252FGP130%2520interface%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D632%26epage%3D641%26doi%3D10.1021%2Fjm401144z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.-p.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, H.</span></span> <span> </span><span class="NLM_article-title">Protein flexibility in docking-based virtual screening: Discovery of novel lymphoid-specific tyrosine phosphatase inhibitors using multiple crystal structures</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1973</span>– <span class="NLM_lpage">1983</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00344</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00344" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyntbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=1973-1983&author=X.+Houauthor=K.+Liauthor=X.+Yuauthor=J.-p.+Sunauthor=H.+Fang&title=Protein+flexibility+in+docking-based+virtual+screening%3A+Discovery+of+novel+lymphoid-specific+tyrosine+phosphatase+inhibitors+using+multiple+crystal+structures&doi=10.1021%2Facs.jcim.5b00344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Flexibility in Docking-Based Virtual Screening: Discovery of Novel Lymphoid-Specific Tyrosine Phosphatase Inhibitors Using Multiple Crystal Structures</span></div><div class="casAuthors">Hou, Xuben; Li, Kangshuai; Yu, Xiao; Sun, Jin-peng; Fang, Hao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1973-1983</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Incorporating protein flexibility is a major challenge for docking-based virtual screening.  With an increasing no. of available crystal structures, ensemble docking with multiple protein structures is an efficient approach to deal with protein flexibility.  Herein, the authors report the successful application of a docking-based virtual screen using multiple crystal structures to discover novel inhibitors of lymphoid-specific tyrosine phosphatase (LYP), a potential drug target for autoimmune diseases.  The appropriate use of multiple protein structures allowed a better enrichment than a single structure in the recovery of known inhibitors.  Subsequently, an optimal ensemble of LYP structures was selected and used in docking-based virtual screening.  Eight novel LYP inhibitors (IC50 ranging from 7.95 to 56.6 μM) were identified among 23 hit compds.  Further studies demonstrated that the most active compd. B15 possessed some selectivity over other protein phosphatases and could effectively up-regulate TCR (T cell receptor)-mediated signaling in Jurkat T cells.  These novel hits not only provided good starting points for the development of therapeutic agents useful in autoimmune diseases but also demonstrated the advantages of choosing an appropriate ensemble of protein structures in docking-based virtual screening over using a single protein conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiDcC42tLbKLVg90H21EOLACvtfcHk0lgYC0G5lwmJ9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyntbvM&md5=9296da92e46d98f8363906ff1c14fd70</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00344%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DJ.-p.%26aulast%3DFang%26aufirst%3DH.%26atitle%3DProtein%2520flexibility%2520in%2520docking-based%2520virtual%2520screening%253A%2520Discovery%2520of%2520novel%2520lymphoid-specific%2520tyrosine%2520phosphatase%2520inhibitors%2520using%2520multiple%2520crystal%2520structures%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D1973%26epage%3D1983%26doi%3D10.1021%2Facs.jcim.5b00344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desai, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avery, M.</span></span> <span> </span><span class="NLM_article-title">Targeting the BH3 domain mediated protein-protein interaction of Bcl-xL through virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">906</span>– <span class="NLM_lpage">923</span>, <span class="refDoi"> DOI: 10.1021/ci1000373</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci1000373" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3cXks1CksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=906-923&author=P.+Mukherjeeauthor=P.+Desaiauthor=Y.-D.+Zhouauthor=M.+Avery&title=Targeting+the+BH3+domain+mediated+protein-protein+interaction+of+Bcl-xL+through+virtual+screening&doi=10.1021%2Fci1000373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the BH3 Domain Mediated Protein-Protein Interaction of Bcl-xL through Virtual Screening</span></div><div class="casAuthors">Mukherjee, Prasenjit; Desai, Prashant; Zhou, Yu-Dong; Avery, Mitchell</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">906-923</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Apoptosis, or programmed cell death, forms an important part of the cellular regulation machinery.  The Bcl-2 protein family, comprising of proapoptotic and antiapoptotic members, forms an important part of the cells internal apoptotic pathway.  Overexpression of the antiapoptotic members of the family in a no. of cancer cell lines renders them immune to apoptosis and the ability to survive under conditions of cellular stress.  Inhibition of the antiapoptotic members of the Bcl-2 family are, therefore, an interesting target for the development of anticancer therapy.  An innovative structure-based virtual screening strategy was developed to identify inhibitors of Bcl-xL, an antiapoptotic member of the Bcl-2 family.  Various innovative filters, such as receptor-based pharmacophore, cascade docking approach, cross-docking, and composite scoring with docking pose based descriptors were designed through exhaustive validation studies and implemented in the screening funnel.  The 1.8 million big-n-greasy' subset from ZINC was screened using the protocol, and 45 compds. were finally selected for biol. evaluation against Bcl-xL.  The evaluation led to the identification of one low-micromolar and two weaker inhibitors belonging to novel scaffolds.  Further evaluation of structure-activity relationships around these scaffolds could help in the development of anticancer leads against Bcl-xL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd89CIbgyDLrVg90H21EOLACvtfcHk0lgYC0G5lwmJ9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXks1CksLk%253D&md5=73a5a6b976c3e3c37939e1c96fa86d50</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fci1000373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci1000373%26sid%3Dliteratum%253Aachs%26aulast%3DMukherjee%26aufirst%3DP.%26aulast%3DDesai%26aufirst%3DP.%26aulast%3DZhou%26aufirst%3DY.-D.%26aulast%3DAvery%26aufirst%3DM.%26atitle%3DTargeting%2520the%2520BH3%2520domain%2520mediated%2520protein-protein%2520interaction%2520of%2520Bcl-xL%2520through%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D906%26epage%3D923%26doi%3D10.1021%2Fci1000373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ye, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Identification of selective, cell active inhibitors of protein arginine methyltransferase 5 through structure-based virtual screening and biological assays</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1066</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.8b00050</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00050" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnvV2ltbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=1066-1073&author=F.+Yeauthor=W.+Zhangauthor=X.+Yeauthor=J.+Jinauthor=Z.+Lvauthor=C.+Luo&title=Identification+of+selective%2C+cell+active+inhibitors+of+protein+arginine+methyltransferase+5+through+structure-based+virtual+screening+and+biological+assays&doi=10.1021%2Facs.jcim.8b00050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Selective, Cell Active Inhibitors of Protein Arginine Methyltransferase 5 through Structure-Based Virtual Screening and Biological Assays</span></div><div class="casAuthors">Ye, Fei; Zhang, Weiyao; Ye, Xiaoqing; Jin, Jia; Lv, Zhengbing; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1066-1073</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5), a type II PRMT enzyme, is reported as an important therapeutic target in leukemia and lymphoma.  In the present study, based on the combination of virtual screening and biochem. validations, we discovered a series of small-mol. inhibitors targeting PRMT5.  Among those, DC_Y134 exhibited the most potent activity with IC50 value of 1.7 μM and displayed good selectivity against other methyltransferases.  Further treatment with DC_Y134 inhibited the proliferation of several hematol. malignancy cell lines by causing cell cycle arrest and apoptosis.  Western blot assays indicated that DC_Y134 reduced the cellular sym. dimethylated levels.  In addn., we analyzed the binding mode of DC_Y134 through mol. docking, which revealed that DC_Y134 occupies the binding site of substrate arginine and explained the selectivity of this inhibitor.  Taken together, compd. DC_Y134 could be used to elucidate the biol. roles of PRMT5 and serve as a lead compd. for treatment of hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp59U0G_Qsx7LVg90H21EOLACvtfcHk0lixWkzq3wi-5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnvV2ltbY%253D&md5=d805b7fb6d533d9506c7eb69f1b6e68d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00050%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DYe%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DLv%26aufirst%3DZ.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520selective%252C%2520cell%2520active%2520inhibitors%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520through%2520structure-based%2520virtual%2520screening%2520and%2520biological%2520assays%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2018%26volume%3D58%26spage%3D1066%26epage%3D1073%26doi%3D10.1021%2Facs.jcim.8b00050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span> <span> </span><span class="NLM_article-title">Development of small-molecules targeting receptor activator of nuclear factor-κB ligand (RANKL)-receptor activator of nuclear factor-κB (RANK) protein-protein interaction by structure-based virtual screening and hit optimization</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">5370</span>– <span class="NLM_lpage">5381</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b02027</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b02027" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXptlOjsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=5370-5381&author=M.+Jiangauthor=L.+Pengauthor=K.+Yangauthor=T.+Wangauthor=X.+Yanauthor=T.+Jiangauthor=J.+Xuauthor=J.+Qiauthor=H.+Zhouauthor=N.+Qianauthor=Q.+Zhouauthor=B.+Chenauthor=X.+Xuauthor=L.+Dengauthor=C.+Yang&title=Development+of+small-molecules+targeting+receptor+activator+of+nuclear+factor-%CE%BAB+ligand+%28RANKL%29-receptor+activator+of+nuclear+factor-%CE%BAB+%28RANK%29+protein-protein+interaction+by+structure-based+virtual+screening+and+hit+optimization&doi=10.1021%2Facs.jmedchem.8b02027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Small-Molecules Targeting Receptor Activator of Nuclear Factor-κB Ligand (RANKL)-Receptor Activator of Nuclear Factor-κB (RANK) Protein-Protein Interaction by Structure-Based Virtual Screening and Hit Optimization</span></div><div class="casAuthors">Jiang, Min; Peng, Lei; Yang, Kai; Wang, Tianqi; Yan, Xueming; Jiang, Tao; Xu, Jianrong; Qi, Jin; Zhou, Hanbing; Qian, Niandong; Zhou, Qi; Chen, Bo; Xu, Xing; Deng, Lianfu; Yang, Chunhao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5370-5381</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting RANKL/RANK offers the possibility of developing novel therapeutic approaches to treat bone metabolic diseases.  Multiple efforts have been made to inhibit RANKL.  For example, marketed monoclonal antibody drug Denosumab could inhibit the maturation of osteoclasts by binding to RANKL.  This study is an original approach aimed at discovering small-mol. inhibitors impeding RANKL/RANK protein interaction.  We identified compd. 34 as a potent and selective RANKL/RANK inhibitor by performing structure-based virtual screening and hit optimization.  Disruption of the RANKL/RANK interaction by 34 effectively inhibits RANKL-induced osteoclastogenesis and bone resorption.  The expression of osteoclast marker genes was also suppressed by treatment of 34.  Furthermore, 34 markedly blocked the NFATc1/c-fos pathway.  Thus, our current work demonstrates that the chem. tractability of the difficult PPI (RANKL/RANK) target by a small-mol. compd. 34 offers a potential lead compd. to facilitate the development of new medications for bone-related diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrR7OIZWpsVg7Vg90H21EOLACvtfcHk0lixWkzq3wi-5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXptlOjsrw%253D&md5=dd9823a04adff49e69c3ac5780f259b0</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b02027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b02027%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DM.%26aulast%3DPeng%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DT.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DN.%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.%26atitle%3DDevelopment%2520of%2520small-molecules%2520targeting%2520receptor%2520activator%2520of%2520nuclear%2520factor-%25CE%25BAB%2520ligand%2520%2528RANKL%2529-receptor%2520activator%2520of%2520nuclear%2520factor-%25CE%25BAB%2520%2528RANK%2529%2520protein-protein%2520interaction%2520by%2520structure-based%2520virtual%2520screening%2520and%2520hit%2520optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D5370%26epage%3D5381%26doi%3D10.1021%2Facs.jmedchem.8b02027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Negri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rives, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caspers, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prisinzano, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Javitch, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filizola, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novelselective kappa-opioid receptor agonist using crystal structure-based virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">521</span>– <span class="NLM_lpage">526</span>, <span class="refDoi"> DOI: 10.1021/ci400019t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400019t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjsVymurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=521-526&author=A.+Negriauthor=M.-L.+Rivesauthor=M.+J.+Caspersauthor=T.+E.+Prisinzanoauthor=J.+A.+Javitchauthor=M.+Filizola&title=Discovery+of+a+novelselective+kappa-opioid+receptor+agonist+using+crystal+structure-based+virtual+screening&doi=10.1021%2Fci400019t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Novel Selective Kappa-Opioid Receptor Agonist Using Crystal Structure-Based Virtual Screening</span></div><div class="casAuthors">Negri, Ana; Rives, Marie-Laure; Caspers, Michael J.; Prisinzano, Thomas E.; Javitch, Jonathan A.; Filizola, Marta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">521-526</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Kappa-opioid (KOP) receptor agonists exhibit analgesic effects without activating reward pathways.  In the search for nonaddictive opioid therapeutics and novel chem. tools to study physiol. functions regulated by the KOP receptor, we screened in silico its recently released inactive crystal structure.  A selective novel KOP receptor agonist emerged as a notable result and is proposed as a new chemotype for the study of the KOP receptor in the etiol. of drug addiction, depression, and/or pain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk0lEXXU0bnLVg90H21EOLACvtfcHk0lixWkzq3wi-5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjsVymurk%253D&md5=4ace2403c664ba58106b40a61773bf4b</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fci400019t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400019t%26sid%3Dliteratum%253Aachs%26aulast%3DNegri%26aufirst%3DA.%26aulast%3DRives%26aufirst%3DM.-L.%26aulast%3DCaspers%26aufirst%3DM.%2BJ.%26aulast%3DPrisinzano%26aufirst%3DT.%2BE.%26aulast%3DJavitch%26aufirst%3DJ.%2BA.%26aulast%3DFilizola%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520novelselective%2520kappa-opioid%2520receptor%2520agonist%2520using%2520crystal%2520structure-based%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D521%26epage%3D526%26doi%3D10.1021%2Fci400019t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">Comparing protein-ligand docking programs is difficult</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">325</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1002/prot.20497</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1002%2Fprot.20497" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=15937897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvVyrsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2005&pages=325-332&author=J.+C.+Coleauthor=C.+W.+Murrayauthor=J.+W.+M.+Nissinkauthor=R.+D.+Taylorauthor=R.+Taylor&title=Comparing+protein-ligand+docking+programs+is+difficult&doi=10.1002%2Fprot.20497"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Comparing protein-ligand docking programs is difficult</span></div><div class="casAuthors">Cole, Jason C.; Murray, Christopher W.; Nissink, J. Willem M.; Taylor, Richard D.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">325-332</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A review.  There is currently great interest in comparing protein-ligand docking programs.  A review of recent comparisons shows that it is difficult to draw conclusions of general applicability.  Statistical hypothesis testing is required to ensure that differences in pose-prediction success rates and enrichment rates are significant.  Numerical measures such as root-mean-square deviation need careful interpretation and may profitably be supplemented by interaction-based measures and visual inspection of dockings.  Test sets must be of appropriate diversity and of good exptl. reliability.  The effects of crystal-packing interactions may be important.  The method used for generating starting ligand geometries and positions may have an appreciable effect on docking results.  For fair comparison, programs must be given search problems of equal complexity (e.g. binding-site regions of the same size) and approx. equal time in which to solve them.  Comparisons based on rescoring require local optimization of the ligand in the space of the new objective function.  Re-implementations of published scoring functions may give significantly different results from the originals.  Ostensibly minor details in methodol. may have a profound influence on headline success rates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT2WvCG-13PrVg90H21EOLACvtfcHk0lj7Md-A4wjfjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmvVyrsbc%253D&md5=08ea962f2c33cfaed1438356c244e688</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fprot.20497&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20497%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DComparing%2520protein-ligand%2520docking%2520programs%2520is%2520difficult%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2005%26volume%3D60%26spage%3D325%26epage%3D332%26doi%3D10.1002%2Fprot.20497" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hartshorn, M. J.</span></span> <span> </span><span class="NLM_article-title">AstexViewerTM †: A visualisation aid for structure-based drug design</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">871</span>– <span class="NLM_lpage">881</span>, <span class="refDoi"> DOI: 10.1023/A:1023813504011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1023%2FA%3A1023813504011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=12825620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A280%3ADC%252BD3s3psl2nsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=871-881&author=M.+J.+Hartshorn&title=AstexViewerTM+%E2%80%A0%3A+A+visualisation+aid+for+structure-based+drug+design&doi=10.1023%2FA%3A1023813504011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">AstexViewer: a visualisation aid for structure-based drug design</span></div><div class="casAuthors">Hartshorn Michael J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of computer-aided molecular design</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">871-81</span>
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    </div><div class="casAbstract">AstexViewer is a Java molecular graphics program that can be used for visualisation in many aspects of structure-based drug design.  This paper describes its functionality, implementation and examples of its use.  The program can run as an Applet in a web browser allowing structures to be displayed without installing additional software.  Applications of its use are described for visualisation and as part of a structure based design platform.  The software is being made freely available to the community and may be downloaded from http://www.astex-technology.com/AstexViewer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSPr0A35nMvHjMsgLhIoICsfW6udTcc2eZarT6j5DAfHbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3s3psl2nsA%253D%253D&md5=f9396aae9ecf64f256a9f8588f8097bd</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1023%2FA%3A1023813504011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1023813504011%26sid%3Dliteratum%253Aachs%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26atitle%3DAstexViewerTM%2520%25E2%2580%25A0%253A%2520A%2520visualisation%2520aid%2520for%2520structure-based%2520drug%2520design%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2002%26volume%3D16%26spage%3D871%26epage%3D881%26doi%3D10.1023%2FA%3A1023813504011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caflisch, A.</span></span> <span> </span><span class="NLM_article-title">High-throughput virtual screening using quantum mechanical probes: Discovery of selective kinase inhibitors</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1014</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201000085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1002%2Fcmdc.201000085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=20540063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVOmt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=1007-1014&author=T.+Zhouauthor=A.+Caflisch&title=High-throughput+virtual+screening+using+quantum+mechanical+probes%3A+Discovery+of+selective+kinase+inhibitors&doi=10.1002%2Fcmdc.201000085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Virtual Screening Using Quantum Mechanical Probes: Discovery of Selective Kinase Inhibitors</span></div><div class="casAuthors">Zhou, Ting; Caflisch, Amedeo</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1007-1014</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A procedure based on semi-empirical quantum mech. (QM) calcns. of interaction energy is proposed for the rapid screening of compd. poses generated by high-throughput docking.  Small mols. (consisting of 2-10 atoms and termed "probes") are overlapped with polar groups in the binding site of the protein target.  The interaction energy values between each compd. pose and the probes, calcd. by a semi-empirical Hamiltonian, are used as filters.  The QM probe method does not require fixed partial charges and takes into account polarization and charge-transfer effects which are not captured by conventional force fields.  The procedure is applied to screen ∼100 million poses (of 2.7 million com. available compds.) obtained by high-throughput docking in the ATP binding site of the tyrosine kinase erythropoietin-producing human hepatocellular carcinoma receptor B4 (EphB4).  Three QM probes on the hinge region and one at the entrance pocket are employed to select for binding affinity, while a QM probe on the side chain of the so-called gatekeeper residue (a hypervariable residue in the kinome) is used to enforce selectivity.  The poses with favorable interactions with the five QM probes are filtered further for hydrophobic matching and low ligand strain.  In this way, a single-digit micromolar inhibitor of EphB4 with a relatively good selectivity profile is identified in a multimillion-compd. library upon exptl. tests of only 23 mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopen4977ct_LVg90H21EOLACvtfcHk0lj7Md-A4wjfjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVOmt7o%253D&md5=090c23ffcdf3d19a73eb7637973721d4</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201000085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201000085%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DHigh-throughput%2520virtual%2520screening%2520using%2520quantum%2520mechanical%2520probes%253A%2520Discovery%2520of%2520selective%2520kinase%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2010%26volume%3D5%26spage%3D1007%26epage%3D1014%26doi%3D10.1002%2Fcmdc.201000085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, L.</span></span> <span> </span><span class="NLM_article-title">Decision making in medicinal chemistry: The power of our intuition</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">956</span>– <span class="NLM_lpage">958</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00359</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00359" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1Orur7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=956-958&author=L.+Gomez&title=Decision+making+in+medicinal+chemistry%3A+The+power+of+our+intuition&doi=10.1021%2Facsmedchemlett.8b00359"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Decision Making in Medicinal Chemistry: The Power of Our Intuition</span></div><div class="casAuthors">Gomez, Laurent</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">956-958</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Medicinal chemists rely on their intuition to make decisions regarding the course of a medicinal chem. program.  Our ability to accurately and efficiently process large data sets routinely requires that we reduce the vol. of information to manageable proportions.  This prioritization process, however, can be affected by intuitive biases.  One such situation is structure-activity relationship (SAR) anal. in nonadditive data sets in which attempts to intuitively predict the activity of compds. based on preliminary data can lead to erroneous conclusions.  Matrix anal. can be a useful tool to accurately det. the nature of the SAR and to improve our decision-making process during an analoging campaign.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNYEcXQuaZhrVg90H21EOLACvtfcHk0lgVjEGLwb2DAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1Orur7N&md5=e37470769559afca2691e0a0ee15bae2</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00359%26sid%3Dliteratum%253Aachs%26aulast%3DGomez%26aufirst%3DL.%26atitle%3DDecision%2520making%2520in%2520medicinal%2520chemistry%253A%2520The%2520power%2520of%2520our%2520intuition%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D956%26epage%3D958%26doi%3D10.1021%2Facsmedchemlett.8b00359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenbaum, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fung, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of beta2-adrenergic receptor ligands</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">6843</span>– <span class="NLM_lpage">6848</span>, <span class="refDoi"> DOI: 10.1073/pnas.0812657106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1073%2Fpnas.0812657106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=19342484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsV2qsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=6843-6848&author=P.+Kolbauthor=D.+M.+Rosenbaumauthor=J.+J.+Irwinauthor=J.+J.+Fungauthor=B.+K.+Kobilkaauthor=B.+K.+Shoichet&title=Structure-based+discovery+of+beta2-adrenergic+receptor+ligands&doi=10.1073%2Fpnas.0812657106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of β2-adrenergic receptor ligands</span></div><div class="casAuthors">Kolb, Peter; Rosenbaum, Daniel M.; Irwin, John J.; Fung, Juan Jose; Kobilka, Brian K.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6843-6848</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Aminergic G protein-coupled receptors (GPCRs) have been a major focus of pharmaceutical research for many years.  Due partly to the lack of reliable receptor structures, drug discovery efforts have been largely ligand-based.  The recently detd. X-ray structure of the β2-adrenergic receptor offers an opportunity to investigate the advantages and limitations inherent in a structure-based approach to ligand discovery against this and related GPCR targets.  Approx. 1 million com. available, "lead-like" mols. were docked against the β2-adrenergic receptor structure.  On testing of 25 high-ranking mols., 6 were active with binding affinities <4 μM, with the best mol. binding with a Ki of 9 nM (95% confidence interval 7-10 nM).  Five of these mols. were inverse agonists.  The high hit rate, the high affinity of the most potent mol., the discovery of unprecedented chemotypes among the new inhibitors, and the apparent bias toward inverse agonists among the docking hits, have implications for structure-based approaches against GPCRs that recognize small org. mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo89kM84Li_H7Vg90H21EOLACvtfcHk0lgVjEGLwb2DAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsV2qsro%253D&md5=4d1a4cb2aa3925aa4c99c6b0496417a7</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0812657106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0812657106%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFung%26aufirst%3DJ.%2BJ.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520discovery%2520of%2520beta2-adrenergic%2520receptor%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D6843%26epage%3D6848%26doi%3D10.1073%2Fpnas.0812657106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirohama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K. Y. J.</span></span> <span> </span><span class="NLM_article-title">Identification of sumoylation activating enzyme 1 inhibitors by structure-based virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">809</span>– <span class="NLM_lpage">820</span>, <span class="refDoi"> DOI: 10.1021/ci300618e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300618e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3sXkvF2jsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=809-820&author=A.+Kumarauthor=A.+Itoauthor=M.+Hirohamaauthor=M.+Yoshidaauthor=K.+Y.+J.+Zhang&title=Identification+of+sumoylation+activating+enzyme+1+inhibitors+by+structure-based+virtual+screening&doi=10.1021%2Fci300618e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Sumoylation Activating Enzyme 1 Inhibitors by Structure-Based Virtual Screening</span></div><div class="casAuthors">Kumar, Ashutosh; Ito, Akihiro; Hirohama, Mikako; Yoshida, Minoru; Zhang, Kam Y. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">809-820</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SUMO activating enzyme 1 (SUMO E1) is responsible for the activation of SUMO in the first step of the sumoylation cascade.  SUMO E1 is linked to many human diseases including cancer, thus making it a potential therapeutic target.  There are few reported SUMO E1 inhibitors including several natural products.  To identify small mol. inhibitors of SUMO E1 with better drug-like properties for potential therapeutic studies, we have used structure-based virtual screening to identify hits from the Maybridge small mol. library for biol. assay.  Our virtual screening protocol involves fast docking of the entire small mol. library with rigid protein and ligands followed by redocking of top hits using a method that incorporates both ligand and protein flexibility.  Subsequently, the top-ranking compds. were prioritized using the mol. dynamics simulation-based binding free energy calcn.  Out of 24 compds. that were acquired and tested using in vitro sumoylation assay, four of them showed more than 85% inhibition of sumoylation with the most active compd. showing an IC50 of 14.4 μM.  A similarity search with the most active compd. in the ZINC database has identified three more compds. with improved potency.  These compds. share a common Ph urea scaffold and have been confirmed to inhibit SUMO E1 by in vitro SUMO-1 thioester bond formation assay.  Our study suggests that these Ph urea compds. could be used as a starting point for the development of novel therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquQUhv65EkgbVg90H21EOLACvtfcHk0lgVjEGLwb2DAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXkvF2jsr0%253D&md5=98ccbbc8ea92d0dd60ab176966e68d97</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fci300618e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300618e%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DIto%26aufirst%3DA.%26aulast%3DHirohama%26aufirst%3DM.%26aulast%3DYoshida%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DK.%2BY.%2BJ.%26atitle%3DIdentification%2520of%2520sumoylation%2520activating%2520enzyme%25201%2520inhibitors%2520by%2520structure-based%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D809%26epage%3D820%26doi%3D10.1021%2Fci300618e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackey, M. D.</span></span> <span> </span><span class="NLM_article-title">Electrostatic complementarity as a fast and effective tool to optimize binding and selectivity of protein-ligand complexes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">3036</span>– <span class="NLM_lpage">3050</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01925</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01925" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXks1ChsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3036-3050&author=M.+R.+Bauerauthor=M.+D.+Mackey&title=Electrostatic+complementarity+as+a+fast+and+effective+tool+to+optimize+binding+and+selectivity+of+protein-ligand+complexes&doi=10.1021%2Facs.jmedchem.8b01925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Electrostatic Complementarity as a Fast and Effective Tool to Optimize Binding and Selectivity of Protein-Ligand Complexes</span></div><div class="casAuthors">Bauer, Matthias R.; Mackey, Mark D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3036-3050</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Electrostatic interactions between small mols. and their resp. receptors are essential for mol. recognition and are also key contributors to the binding free energy.  Assessing the electrostatic match of protein-ligand complexes therefore provides important insights into why ligands bind and what can be changed to improve binding.  Ideally, the ligand and protein electrostatic potentials at the protein-ligand interaction interface should maximize their complementarity while minimizing desolvation penalties.  In this work, we present a fast and efficient tool to calc. and visualize the electrostatic complementarity (EC) of protein-ligand complexes.  We compiled benchmark sets demonstrating electrostatically driven structure-activity relationships (SAR) from literature data, including kinase, protein-protein interaction, and GPCR targets, and used these to demonstrate that the EC method can visualize, rationalize, and predict electrostatically driven ligand affinity changes and help to predict compd. selectivity.  The methodol. presented here for the anal. of EC is a powerful and versatile tool for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3pdUzJ9T0trVg90H21EOLACvtfcHk0li_PtaW-yHb7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXks1ChsL8%253D&md5=a1abb5abe7fa8d73bee264bc94fd2da1</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01925%26sid%3Dliteratum%253Aachs%26aulast%3DBauer%26aufirst%3DM.%2BR.%26aulast%3DMackey%26aufirst%3DM.%2BD.%26atitle%3DElectrostatic%2520complementarity%2520as%2520a%2520fast%2520and%2520effective%2520tool%2520to%2520optimize%2520binding%2520and%2520selectivity%2520of%2520protein-ligand%2520complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D3036%26epage%3D3050%26doi%3D10.1021%2Facs.jmedchem.8b01925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span>; <span class="NLM_string-name">Hu, B.</span>; <span class="NLM_string-name">Lill, M. A.</span></span> In  <i>WATsite2.0 with PyMOL plugin: Hydration site prediction and visualization BT - protein function prediction: Methods and protocols</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihara, D.</span></span>, Ed.; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2017</span>; pp  <span class="NLM_fpage">123</span>– <span class="NLM_lpage">134</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1007%2F978-1-4939-7015-5_10" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=123-134&author=Y.+Yang&author=B.+Hu&author=M.+A.+Lillauthor=D.+Kihara&title=WATsite2.0+with+PyMOL+plugin%3A+Hydration+site+prediction+and+visualization+BT+-+protein+function+prediction%3A+Methods+and+protocols"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-7015-5_10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-4939-7015-5_10%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%26btitle%3DWATsite2.0%2520with%2520PyMOL%2520plugin%253A%2520Hydration%2520site%2520prediction%2520and%2520visualization%2520BT%2520-%2520protein%2520function%2520prediction%253A%2520Methods%2520and%2520protocols%26aulast%3DKihara%26aufirst%3DD.%26pub%3DSpringer%26date%3D2017%26spage%3D123%26epage%3D134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liebeschuetz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennemann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span> <span> </span><span class="NLM_article-title">The good, the bad and the twisted: A survey of ligand geometry in protein crystal structures</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">183</span>, <span class="refDoi"> DOI: 10.1007/s10822-011-9538-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1007%2Fs10822-011-9538-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=22246295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjt1eitb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=169-183&author=J.+Liebeschuetzauthor=J.+Hennemannauthor=T.+Olssonauthor=C.+R.+Groom&title=The+good%2C+the+bad+and+the+twisted%3A+A+survey+of+ligand+geometry+in+protein+crystal+structures&doi=10.1007%2Fs10822-011-9538-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The good, the bad and the twisted: a survey of ligand geometry in protein crystal structures</span></div><div class="casAuthors">Liebeschuetz, John; Hennemann, Jana; Olsson, Tjelvar; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">169-183</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The protein databank now contains the structures of over 11,000 ligands bound to proteins.  These structures are invaluable in applied areas such as structure-based drug design, but are also the substrate for understanding the energetics of intermol. interactions with proteins.  Despite their obvious importance, the careful anal. of ligands bound to protein structures lags behind the anal. of the protein structures themselves.  We present an anal. of the geometry of ligands bound to proteins and highlight the role of small mol. crystal structures in enabling mol. modellers to critically evaluate a ligand model's quality and investigate protein-induced strain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpceotnqHSI-rVg90H21EOLACvtfcHk0li_PtaW-yHb7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjt1eitb0%253D&md5=dd85bf6e63910bc85db95509b97c018a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9538-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9538-6%26sid%3Dliteratum%253Aachs%26aulast%3DLiebeschuetz%26aufirst%3DJ.%26aulast%3DHennemann%26aufirst%3DJ.%26aulast%3DOlsson%26aufirst%3DT.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DThe%2520good%252C%2520the%2520bad%2520and%2520the%2520twisted%253A%2520A%2520survey%2520of%2520ligand%2520geometry%2520in%2520protein%2520crystal%2520structures%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D169%26epage%3D183%26doi%3D10.1007%2Fs10822-011-9538-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, C. D.</span>; <span class="NLM_string-name">Nowak, T.</span>; <span class="NLM_string-name">Watson, M. J.</span></span> In  <i>Chapter 2: Measurement interpretation and use of free ligand solution conformations in drug discovery</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawton, G.</span></span>, et al., Eds.; <span class="NLM_publisher-name">Elsevier</span>, <span class="NLM_year">2016</span>; Vol.  <span class="NLM_volume">55</span>, pp  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">147</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=45-147&author=C.+D.+Blundell&author=T.+Nowak&author=M.+J.+Watsonauthor=G.+Lawton&title=Chapter+2%3A+Measurement+interpretation+and+use+of+free+ligand+solution+conformations+in+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DBlundell%26aufirst%3DC.%2BD.%26btitle%3DChapter%25202%253A%2520Measurement%2520interpretation%2520and%2520use%2520of%2520free%2520ligand%2520solution%2520conformations%2520in%2520drug%2520discovery%26aulast%3DLawton%26aufirst%3DG.%26pub%3DElsevier%26date%3D2016%26volume%3D55%26spage%3D45%26epage%3D147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kubinyi, H.</span></span> <span> </span><span class="NLM_article-title">Hydrogen bonding: The last mystery in drug design?</span>. <i>Pharmacokinetic Optimization in Drug Research</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_fpage">513</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.1002/9783906390437.ch28</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1002%2F9783906390437.ch28" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&pages=513-524&author=H.+Kubinyi&title=Hydrogen+bonding%3A+The+last+mystery+in+drug+design%3F&doi=10.1002%2F9783906390437.ch28"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2F9783906390437.ch28&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F9783906390437.ch28%26sid%3Dliteratum%253Aachs%26aulast%3DKubinyi%26aufirst%3DH.%26atitle%3DHydrogen%2520bonding%253A%2520The%2520last%2520mystery%2520in%2520drug%2520design%253F%26jtitle%3DPharmacokinetic%2520Optimization%2520in%2520Drug%2520Research%26date%3D2007%26spage%3D513%26epage%3D524%26doi%3D10.1002%2F9783906390437.ch28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agnihotri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirohi, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahasrabuddhe, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratap, J. V.</span></span> <span> </span><span class="NLM_article-title">Identification of novel inhibitors of Leishmania donovani γ-glutamylcysteine synthetase using structure-based virtual screening, docking, molecular dynamics simulation, and in vitro studies</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">815</span>– <span class="NLM_lpage">825</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00642</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00642" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksFals7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=815-825&author=P.+Agnihotriauthor=A.+K.+Mishraauthor=S.+Mishraauthor=V.+K.+Sirohiauthor=A.+A.+Sahasrabuddheauthor=J.+V.+Pratap&title=Identification+of+novel+inhibitors+of+Leishmania+donovani+%CE%B3-glutamylcysteine+synthetase+using+structure-based+virtual+screening%2C+docking%2C+molecular+dynamics+simulation%2C+and+in+vitro+studies&doi=10.1021%2Facs.jcim.6b00642"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Novel Inhibitors of Leishmania donovani γ-Glutamylcysteine Synthetase Using Structure-Based Virtual Screening, Docking, Molecular Dynamics Simulation, and in Vitro Studies</span></div><div class="casAuthors">Agnihotri, Pragati; Mishra, Arjun K.; Mishra, Shikha; Sirohi, Vijay Kumar; Sahasrabuddhe, Amogh A.; Pratap, J. Venkatesh</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">815-825</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Trypansomatids maintain their redox balance by the trypanothione based redox system, enzymes of which exhibit differences from mammalian homologues.  γ-Glutamylcysteine synthetase (Gcs) is an essential enzyme of this pathway performing the first and rate limiting step.  L-buthionine-S,R-sulfoximine (BSO), a specific inhibitor of Gcs induces toxicity in hosts infected with T. brucei underlining for novel Gcs inhibitors.  The present study reports identification of Ld Gcs inhibitors using computational approaches and their exptl. validation.  Mol. dynamic simulation studies were used to provide better insight to the predicted docked conformation.  Five compds. selected for in vitro anal. have shown significantly higher inhibition of LdGcs ATPase activity as compared to BSO.  These compds. have shown 40-60 % inhibition of L. donovani promastigote proliferation in 100 μM concn. with three compds. showing no toxicity in human HEK293 cell lines.  Anal. of inhibitor-LdGcs complexes shows modifications that could result in increased efficacy of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGyiSfUcoNeLVg90H21EOLACvtfcHk0lhiGYZPL9NsNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksFals7w%253D&md5=37bee20a8d71ec992b198f89a3189587</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00642&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00642%26sid%3Dliteratum%253Aachs%26aulast%3DAgnihotri%26aufirst%3DP.%26aulast%3DMishra%26aufirst%3DA.%2BK.%26aulast%3DMishra%26aufirst%3DS.%26aulast%3DSirohi%26aufirst%3DV.%2BK.%26aulast%3DSahasrabuddhe%26aufirst%3DA.%2BA.%26aulast%3DPratap%26aufirst%3DJ.%2BV.%26atitle%3DIdentification%2520of%2520novel%2520inhibitors%2520of%2520Leishmania%2520donovani%2520%25CE%25B3-glutamylcysteine%2520synthetase%2520using%2520structure-based%2520virtual%2520screening%252C%2520docking%252C%2520molecular%2520dynamics%2520simulation%252C%2520and%2520in%2520vitro%2520studies%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D815%26epage%3D825%26doi%3D10.1021%2Facs.jcim.6b00642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arisawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamada, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizutani, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomioka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel aldose reductase inhibitors using a protein structure-based approach: 3D-database search followed by design and synthesis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1718</span>– <span class="NLM_lpage">1728</span>, <span class="refDoi"> DOI: 10.1021/jm000483h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm000483h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD3MXivFynu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=1718-1728&author=Y.+Iwataauthor=M.+Arisawaauthor=R.+Hamadaauthor=Y.+Kitaauthor=M.+Y.+Mizutaniauthor=N.+Tomiokaauthor=A.+Itaiauthor=S.+Miyamoto&title=Discovery+of+novel+aldose+reductase+inhibitors+using+a+protein+structure-based+approach%3A+3D-database+search+followed+by+design+and+synthesis&doi=10.1021%2Fjm000483h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Aldose Reductase Inhibitors Using a Protein Structure-Based Approach: 3D-Database Search Followed by Design and Synthesis</span></div><div class="casAuthors">Iwata, Yoriko; Arisawa, Mitsuhiro; Hamada, Ryuji; Kita, Yasuyuki; Mizutani, Miho Y.; Tomioka, Nobuo; Itai, Akiko; Miyamoto, Shuichi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1718-1728</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aldose reductase (AR) has been implicated in the etiol. of diabetic complications.  Due to the limited no. of currently available drugs for the treatment of diabetic complications, we have carried out structure-based drug design and synthesis to find new types of AR inhibitors.  With the ADAM&EVE program, a three-dimensional database (ACD3D) was searched using the ligand binding site of the AR crystal structure.  Out of 179 compds. selected through this search followed by visual inspection, 36 compds. were purchased and subjected to a biol. assay.  Ten compds. showed more than 40% inhibition of AR at a 15 μg/mL concn.  In a subsequent lead optimization, a series of analogs of the most active compd. were synthesized based on the docking mode derived by ADAM&EVE.  Many of these congeners exhibited higher activities compared to the mother compd. Indeed, the most potent, synthesized compd. showed a ∼20-fold increase in inhibitory activity (IC50 = 0.21 vs. 4.3 μM).  Furthermore, a hydrophobic subsite was newly inferred, which would be useful for the design of inhibitors with improved affinity for AR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6c8cdbn1oQLVg90H21EOLACvtfcHk0lhiGYZPL9NsNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXivFynu7g%253D&md5=6eff99eefff1e4c4531df078969c73d7</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm000483h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000483h%26sid%3Dliteratum%253Aachs%26aulast%3DIwata%26aufirst%3DY.%26aulast%3DArisawa%26aufirst%3DM.%26aulast%3DHamada%26aufirst%3DR.%26aulast%3DKita%26aufirst%3DY.%26aulast%3DMizutani%26aufirst%3DM.%2BY.%26aulast%3DTomioka%26aufirst%3DN.%26aulast%3DItai%26aufirst%3DA.%26aulast%3DMiyamoto%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520novel%2520aldose%2520reductase%2520inhibitors%2520using%2520a%2520protein%2520structure-based%2520approach%253A%25203D-database%2520search%2520followed%2520by%2520design%2520and%2520synthesis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D1718%26epage%3D1728%26doi%3D10.1021%2Fjm000483h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Da’adoosh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyashita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakaguchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldblum, A.</span></span> <span> </span><span class="NLM_article-title">Computational design of substrate selective inhibition</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">e1007713</span>, <span class="refDoi"> DOI: 10.1371/journal.pcbi.1007713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1371%2Fjournal.pcbi.1007713" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=32196495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXps1Wrtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=e1007713&author=B.+Da%E2%80%99adooshauthor=K.+Kaitoauthor=K.+Miyashitaauthor=M.+Sakaguchiauthor=A.+Goldblum&title=Computational+design+of+substrate+selective+inhibition&doi=10.1371%2Fjournal.pcbi.1007713"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Computational design of substrate selective inhibition</span></div><div class="casAuthors">Da'adoosh, Benny; Kaito, Kon; Miyashita, Keishi; Sakaguchi, Minoru; Goldblum, Amiram</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e1007713/1-e1007713/24</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Most enzymes act on more than a single substrate.  There is frequently a need to block the prodn. of a single pathogenic outcome of enzymic activity on a substrate but to avoid blocking others of its catalytic actions.  Full blocking might cause severe side effects because some products of that catalysis may be vital.  Substrate selectivity is required but not possible to achieve by blocking the catalytic residues of an enzyme.  That is the basis of the need for "Substrate Selective Inhibitors" (SSI), and there are several mols. characterized as SSI.  However, none have yet been designed or discovered by computational methods.  We demonstrate a computational approach to the discovery of Substrate Selective Inhibitors for one enzyme, Prolyl Oligopeptidase (POP) (E.C 3.4.21.26), a serine protease which cleaves small peptides between Pro and other amino acids.  Among those are TSH Releasing Hormone (TRH) and Angiotensin-III (Ang-III), differing in both their binding (Km) and in turnover (kcat).  We used our inhouse "Iterative Stochastic Elimination" (ISE) algorithm and the structure-based "Pharmacophore" approach to construct two models for identifying SSI of POP.  A dataset of ~ 1.8 million com. available mols. was initially reduced to less than 12,000 which were screened by these models to a final set of 20 mols. which were sent for exptl. validation (five random mols. were tested for comparison).  Two mols. out of these 20, one with a high score in the ISE model, the other successful in the pharmacophore model, were confirmed by in vitro measurements.  One is a competitive inhibitor of Ang-III (increases its Km), but non-competitive towards TRH (decreases its Vmax).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLqvWDmk3LwLVg90H21EOLACvtfcHk0lhiGYZPL9NsNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXps1Wrtro%253D&md5=656c78735db5a05489f2649eba99695a</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1007713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1007713%26sid%3Dliteratum%253Aachs%26aulast%3DDa%25E2%2580%2599adoosh%26aufirst%3DB.%26aulast%3DKaito%26aufirst%3DK.%26aulast%3DMiyashita%26aufirst%3DK.%26aulast%3DSakaguchi%26aufirst%3DM.%26aulast%3DGoldblum%26aufirst%3DA.%26atitle%3DComputational%2520design%2520of%2520substrate%2520selective%2520inhibition%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2020%26volume%3D16%26spage%3De1007713%26doi%3D10.1371%2Fjournal.pcbi.1007713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ribeiro, A. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyzack, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borkakoti, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holliday, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. M.</span></span> <span> </span><span class="NLM_article-title">A global analysis of function and conservation of catalytic residues in enzymes</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>295</i></span>,  <span class="NLM_fpage">314</span>– <span class="NLM_lpage">324</span>, <span class="refDoi"> DOI: 10.1074/jbc.REV119.006289</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1074%2Fjbc.REV119.006289" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=31796628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A280%3ADC%252BB3MfmtV2gtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=295&publication_year=2020&pages=314-324&author=A.+J.+M.+Ribeiroauthor=J.+D.+Tyzackauthor=N.+Borkakotiauthor=G.+L.+Hollidayauthor=J.+M.+Thornton&title=A+global+analysis+of+function+and+conservation+of+catalytic+residues+in+enzymes&doi=10.1074%2Fjbc.REV119.006289"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A global analysis of function and conservation of catalytic residues in enzymes</span></div><div class="casAuthors">Ribeiro Antonio J M; Tyzack Jonathan D; Borkakoti Neera; Holliday Gemma L; Thornton Janet M</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of biological chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">295</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">314-324</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The catalytic residues of an enzyme comprise the amino acids located in the active center responsible for accelerating the enzyme-catalyzed reaction.  These residues lower the activation energy of reactions by performing several catalytic functions.  Decades of enzymology research has established general themes regarding the roles of specific residues in these catalytic reactions, but it has been more difficult to explore these roles in a more systematic way.  Here, we review the data on the catalytic residues of 648 enzymes, as annotated in the Mechanism and Catalytic Site Atlas (M-CSA), and compare our results with those in previous studies.  We structured this analysis around three key properties of the catalytic residues: amino acid type, catalytic function, and sequence conservation in homologous proteins.  As expected, we observed that catalysis is mostly accomplished by a small set of residues performing a limited number of catalytic functions.  Catalytic residues are typically highly conserved, but to a smaller degree in homologues that perform different reactions or are nonenzymes (pseudoenzymes).  Cross-analysis yielded further insights revealing which residues perform particular functions and how often.  We obtained more detailed specificity rules for certain functions by identifying the chemical group upon which the residue acts.  Finally, we show the mutation tolerance of the catalytic residues based on their roles.  The characterization of the catalytic residues, their functions, and conservation, as presented here, is key to understanding the impact of mutations in evolution, disease, and enzyme design.  The tools developed for this analysis are available at the M-CSA website and allow for user specific analysis of the same data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQUFwLhBKwYdu2mnZh2OgS0fW6udTcc2eZavVXXb4LF4rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MfmtV2gtg%253D%253D&md5=cee0818e02708f8561f71f5bd8953224</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1074%2Fjbc.REV119.006289&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.REV119.006289%26sid%3Dliteratum%253Aachs%26aulast%3DRibeiro%26aufirst%3DA.%2BJ.%2BM.%26aulast%3DTyzack%26aufirst%3DJ.%2BD.%26aulast%3DBorkakoti%26aufirst%3DN.%26aulast%3DHolliday%26aufirst%3DG.%2BL.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DA%2520global%2520analysis%2520of%2520function%2520and%2520conservation%2520of%2520catalytic%2520residues%2520in%2520enzymes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2020%26volume%3D295%26spage%3D314%26epage%3D324%26doi%3D10.1074%2Fjbc.REV119.006289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devaney, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span> <span> </span><span class="NLM_article-title">Strength of solvent-exposed salt-bridges</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1021/jp982715i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp982715i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADyaK1MXis1Kisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1999&pages=727-736&author=R.+Luoauthor=L.+Davidauthor=H.+Hungauthor=J.+Devaneyauthor=M.+K.+Gilson&title=Strength+of+solvent-exposed+salt-bridges&doi=10.1021%2Fjp982715i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Strength of solvent-exposed salt-bridges</span></div><div class="casAuthors">Luo, Ray; David, Laurent; Hung, Howard; Devaney, Judith; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">727-736</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1089-5647</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A recently developed computer model to study the energetics of solvent-exposed salt-bridges is presented here.  The model uses the "mining min." (MM) method to compute conformational free energies with the CHARMm empirical force and the generalized Born (GB) solvation model.  Satisfactory agreement is obtained in comparison with the measured binding affinities of ion pairs in soln. and with the salt-bridge energetics deduced from studies of salt-bridges in helical peptides.  The calcns. suggest that stabilizing charge-charge interactions in helical peptides do not require well-defined salt-bridge conformations.  This is in agreement with crystallog. studies of charge pairs added to T4 lysozyme by site-directed mutagenesis.  The computer model is also used to make a testable prediction that arginine and phosphotyrosine residues in an (i, i + 4) relationship will form a particularly strong salt-bridge in helical peptides.  The biol. implications of these results are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqC1VmzMk_0sbVg90H21EOLACvtfcHk0lhM9eZjqHj9Ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXis1Kisw%253D%253D&md5=0dad345d3147f4051cb4d59a4195dec1</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjp982715i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp982715i%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DR.%26aulast%3DDavid%26aufirst%3DL.%26aulast%3DHung%26aufirst%3DH.%26aulast%3DDevaney%26aufirst%3DJ.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DStrength%2520of%2520solvent-exposed%2520salt-bridges%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1999%26volume%3D103%26spage%3D727%26epage%3D736%26doi%3D10.1021%2Fjp982715i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira
de Freitas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">A systematic analysis of atomic protein-ligand interactions in the PDB</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1970</span>– <span class="NLM_lpage">1981</span>, <span class="refDoi"> DOI: 10.1039/C7MD00381A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1039%2FC7MD00381A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=29308120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOiu7nE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1970-1981&author=R.+Ferreira%0Ade+Freitasauthor=M.+Schapira&title=A+systematic+analysis+of+atomic+protein-ligand+interactions+in+the+PDB&doi=10.1039%2FC7MD00381A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">A systematic analysis of atomic protein-ligand interactions in the PDB</span></div><div class="casAuthors">Ferreira de Freitas, Renato; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1970-1981</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">As the protein databank (PDB) recently passed the cap of 123 456 structures, it stands more than ever as an important resource not only to analyze structural features of specific biol. systems, but also to study the prevalence of structural patterns obsd. in a large body of unrelated structures, that may reflect rules governing protein folding or mol. recognition.  Here, we compiled a list of 11 016 unique structures of small-mol. ligands bound to proteins - 6444 of which have exptl. binding affinity - representing 750 873 protein-ligand at. interactions, and analyzed the frequency, geometry and impact of each interaction type.  We find that hydrophobic interactions are generally enriched in high-efficiency ligands, but polar interactions are over-represented in fragment inhibitors.  While most observations extd. from the PDB will be familiar to seasoned medicinal chemists, less expected findings, such as the high no. of C-H···O hydrogen bonds or the relatively frequent amide-p stacking between the backbone amide of proteins and arom. rings of ligands, uncover underused ligand design strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8iIbfKGlrqbVg90H21EOLACvtfcHk0lhM9eZjqHj9Ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOiu7nE&md5=04688488e73253f76e4ea384456f89a8</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1039%2FC7MD00381A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7MD00381A%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%2Bde%2BFreitas%26aufirst%3DR.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DA%2520systematic%2520analysis%2520of%2520atomic%2520protein-ligand%2520interactions%2520in%2520the%2520PDB%26jtitle%3DMedChemComm%26date%3D2017%26volume%3D8%26spage%3D1970%26epage%3D1981%26doi%3D10.1039%2FC7MD00381A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldner, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Grafenstein, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liedl, K. R.</span></span> <span> </span><span class="NLM_article-title">Independent metrics for protein backbone and side-chain flexibility: Time scales and effects of ligand binding</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">851</span>– <span class="NLM_lpage">860</span>, <span class="refDoi"> DOI: 10.1021/ct500633u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct500633u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2MXislSmsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=851-860&author=J.+E.+Fuchsauthor=B.+J.+Waldnerauthor=R.+G.+Huberauthor=S.+von+Grafensteinauthor=C.+Kramerauthor=K.+R.+Liedl&title=Independent+metrics+for+protein+backbone+and+side-chain+flexibility%3A+Time+scales+and+effects+of+ligand+binding&doi=10.1021%2Fct500633u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Independent Metrics for Protein Backbone and Side-Chain Flexibility: Time Scales and Effects of Ligand Binding</span></div><div class="casAuthors">Fuchs, Julian E.; Waldner, Birgit J.; Huber, Roland G.; von Grafenstein, Susanne; Kramer, Christian; Liedl, Klaus R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">851-860</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conformational dynamics are central for understanding biomol. structure and function, since biol. macromols. are inherently flexible at room temp. and in soln.  Computational methods are nowadays capable of providing valuable information on the conformational ensembles of biomols.  However, anal. tools and intuitive metrics that capture dynamic information from in silico generated structural ensembles are limited.  In std. work-flows, flexibility in a conformational ensemble is represented through residue-wise root-mean-square fluctuations or B-factors following a global alignment.  Consequently, these approaches relying on global alignments discard valuable information on local dynamics.  Results inherently depend on global flexibility, residue size, and connectivity.  In this study we present a novel approach for capturing positional fluctuations based on multiple local alignments instead of one single global alignment.  The method captures local dynamics within a structural ensemble independent of residue type by splitting individual local and global degrees of freedom of protein backbone and side-chains.  Dependence on residue type and size in the side-chains is removed via normalization with the B-factors of the isolated residue.  As a test case, we demonstrate its application to a mol. dynamics simulation of bovine pancreatic trypsin inhibitor (BPTI) on the millisecond time scale.  This allows for illustrating different time scales of backbone and side-chain flexibility.  Addnl., we demonstrate the effects of ligand binding on side-chain flexibility of three serine proteases.  We expect our new methodol. for quantifying local flexibility to be helpful in unraveling local changes in biomol. dynamics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro4WAfZQnkVbVg90H21EOLACvtfcHk0lhM9eZjqHj9Ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXislSmsr8%253D&md5=b16802904f58e97867c4f0c6360dbc6c</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fct500633u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct500633u%26sid%3Dliteratum%253Aachs%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3DWaldner%26aufirst%3DB.%2BJ.%26aulast%3DHuber%26aufirst%3DR.%2BG.%26aulast%3Dvon%2BGrafenstein%26aufirst%3DS.%26aulast%3DKramer%26aufirst%3DC.%26aulast%3DLiedl%26aufirst%3DK.%2BR.%26atitle%3DIndependent%2520metrics%2520for%2520protein%2520backbone%2520and%2520side-chain%2520flexibility%253A%2520Time%2520scales%2520and%2520effects%2520of%2520ligand%2520binding%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D851%26epage%3D860%26doi%3D10.1021%2Fct500633u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caballero, J.</span></span> <span> </span><span class="NLM_article-title">Considerations for Docking of Selective Angiotensin-Converting Enzyme Inhibitors</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">295</span>, <span class="refDoi"> DOI: 10.3390/molecules25020295</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.3390%2Fmolecules25020295" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjslCisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=295&author=J.+Caballero&title=Considerations+for+Docking+of+Selective+Angiotensin-Converting+Enzyme+Inhibitors&doi=10.3390%2Fmolecules25020295"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Considerations for docking of selective angiotensin-converting enzyme inhibitors</span></div><div class="casAuthors">Caballero, Julio</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">295</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  The angiotensin-converting enzyme (ACE) is a two-domain dipeptidylcarboxypeptidase, which has a direct involvement in the control of blood pressure by performing the hydrolysis of angiotensin I to produce angiotensin II.  At the same time, ACE hydrolyzes other substrates such as the vasodilator peptide bradykinin and the anti-inflammatory peptide N-acetyl-SDKP.  In this sense, ACE inhibitors are bioactive substances with potential use as medicinal products for treatment or prevention of hypertension, heart failures, myocardial infarction, and other important diseases.  This review examd. the most recent literature reporting ACE inhibitors with the help of mol. modeling.  The examples exposed here demonstrate that mol. modeling methods, including docking, mol. dynamics (MD) simulations, quant. structure-activity relationship (QSAR), etc, are essential for a complete structural picture of the mode of action of ACE inhibitors, where mol. docking has a key role.  Examples show that too many works identified ACE inhibitory activities of natural peptides and peptides obtained from hydrolyzates.  In addn., other works report non-peptide compds. extd. from natural sources and synthetic compds.  In all these cases, mol. docking was used to provide explanation of the chem. interactions between inhibitors and the ACE binding sites.  For docking applications, most of the examples exposed here do not consider that: (i) ACE has two domains (nACE and cACE) with available X-ray structures, which are relevant for the design of selective inhibitors, and (ii) nACE and cACE binding sites have large dimensions, which leads to non-reliable solns. during docking calcns.  In support of the soln. of these problems, the structural information found in Protein Data Bank (PDB) was used to perform an interaction fingerprints (IFPs) anal. applied on both nACE and cACE domains.  This anal. provides plots that identify the chem. interactions between ligands and both ACE binding sites, which can be used to guide docking expts. in the search of selective natural components or novel drugs.  In addn., the use of hydrogen bond constraints in the S2 and S2' subsites of nACE and cACE are suggested to guarantee that docking solns. are reliable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC-X-Fv63o0rVg90H21EOLACvtfcHk0lj6glZEiLIXeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjslCisrs%253D&md5=e9756d0d0631a3efd7c0b8c2007cd2de</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.3390%2Fmolecules25020295&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules25020295%26sid%3Dliteratum%253Aachs%26aulast%3DCaballero%26aufirst%3DJ.%26atitle%3DConsiderations%2520for%2520Docking%2520of%2520Selective%2520Angiotensin-Converting%2520Enzyme%2520Inhibitors%26jtitle%3DMolecules%26date%3D2020%26volume%3D25%26spage%3D295%26doi%3D10.3390%2Fmolecules25020295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kannan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melesina, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, A.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrabarti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heimburg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidtkunz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, W.</span></span> <span> </span><span class="NLM_article-title">Discovery of inhibitors of Schistosoma mansoni HDAC8 by combining homology modeling, virtual screening, and in vitro validation</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">3005</span>– <span class="NLM_lpage">3019</span>, <span class="refDoi"> DOI: 10.1021/ci5004653</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci5004653" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFygtLvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=3005-3019&author=S.+Kannanauthor=J.+Melesinaauthor=A.-T.+Hauserauthor=A.+Chakrabartiauthor=T.+Heimburgauthor=K.+Schmidtkunzauthor=A.+Walterauthor=M.+Marekauthor=R.+J.+Pierceauthor=C.+Romierauthor=M.+Jungauthor=W.+Sippl&title=Discovery+of+inhibitors+of+Schistosoma+mansoni+HDAC8+by+combining+homology+modeling%2C+virtual+screening%2C+and+in+vitro+validation&doi=10.1021%2Fci5004653"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Inhibitors of Schistosoma mansoni HDAC8 by Combining Homology Modeling, Virtual Screening, and in Vitro Validation</span></div><div class="casAuthors">Kannan, Srinivasaraghavan; Melesina, Jelena; Hauser, Alexander-Thomas; Chakrabarti, Alokta; Heimburg, Tino; Schmidtkunz, Karin; Walter, Alexandra; Marek, Martin; Pierce, Raymond J.; Romier, Christophe; Jung, Manfred; Sippl, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3005-3019</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Schistosomiasis, caused by S. mansoni, is a tropical disease that affects over 200 million people worldwide.  A novel approach for targeting eukaryotic parasites is to tackle their dynamic epigenetic machinery that is necessary for the extensive phenotypic changes during their life cycle.  We recently identified S. mansoni histone deacetylase 8 (smHDAC8) as a potential target for antiparasitic therapy.  Here we present results from a virtual screening campaign on smHDAC8.  Besides hydroxamates, several sulfonamide-thiazole derivs. were identified by a target-based virtual screening using a homol. model of smHDAC8.  In vitro testing of 75 compds. identified 8 hydroxamates as potent and lead-like inhibitors of the parasitic HDAC8.  Solving of the crystal structure of smHDAC8 with two of the virtual screening hits confirmed the predicted binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoH_psd1ffmerVg90H21EOLACvtfcHk0lj6glZEiLIXeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFygtLvO&md5=512119ba4e5cbcdd0d28570a3acb74b5</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fci5004653&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci5004653%26sid%3Dliteratum%253Aachs%26aulast%3DKannan%26aufirst%3DS.%26aulast%3DMelesina%26aufirst%3DJ.%26aulast%3DHauser%26aufirst%3DA.-T.%26aulast%3DChakrabarti%26aufirst%3DA.%26aulast%3DHeimburg%26aufirst%3DT.%26aulast%3DSchmidtkunz%26aufirst%3DK.%26aulast%3DWalter%26aufirst%3DA.%26aulast%3DMarek%26aufirst%3DM.%26aulast%3DPierce%26aufirst%3DR.%2BJ.%26aulast%3DRomier%26aufirst%3DC.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DSippl%26aufirst%3DW.%26atitle%3DDiscovery%2520of%2520inhibitors%2520of%2520Schistosoma%2520mansoni%2520HDAC8%2520by%2520combining%2520homology%2520modeling%252C%2520virtual%2520screening%252C%2520and%2520in%2520vitro%2520validation%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D3005%26epage%3D3019%26doi%3D10.1021%2Fci5004653" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Masters, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahmoud, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lill, M. A.</span></span> <span> </span><span class="NLM_article-title">Efficient and accurate hydration site profiling for enclosed binding sites</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2183</span>– <span class="NLM_lpage">2188</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.8b00544</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00544" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVGjtrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=2183-2188&author=M.+R.+Mastersauthor=A.+H.+Mahmoudauthor=Y.+Yangauthor=M.+A.+Lill&title=Efficient+and+accurate+hydration+site+profiling+for+enclosed+binding+sites&doi=10.1021%2Facs.jcim.8b00544"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient and Accurate Hydration Site Profiling for Enclosed Binding Sites</span></div><div class="casAuthors">Masters, Matthew R.; Mahmoud, Amr H.; Yang, Ying; Lill, Markus A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2183-2188</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. dynamics (MD) simulations allow for accurate prediction of the thermodn. profile of binding-site water mols. crit. for protein-ligand assocn.  Whereas this hydration-site profiling converges rapidly for solvent-exposed sites independent of the initial water placement, an accurate and reliable placement is required for water mols. in occluded binding sites.  Here, we present an accurate and efficient hydration-site prediction method for occluded binding sites combining water placement based on 3D-RISM and MD simulations using WATsite.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvVWTQeN-bSbVg90H21EOLACvtfcHk0lj6glZEiLIXeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVGjtrrL&md5=c7bcfbc9538595ee2ad2b8c2778ab85f</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00544&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00544%26sid%3Dliteratum%253Aachs%26aulast%3DMasters%26aufirst%3DM.%2BR.%26aulast%3DMahmoud%26aufirst%3DA.%2BH.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DLill%26aufirst%3DM.%2BA.%26atitle%3DEfficient%2520and%2520accurate%2520hydration%2520site%2520profiling%2520for%2520enclosed%2520binding%2520sites%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2018%26volume%3D58%26spage%3D2183%26epage%3D2188%26doi%3D10.1021%2Facs.jcim.8b00544" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Role of the active-site solvent in the thermodynamics of factor Xa ligand binding</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">2817</span>– <span class="NLM_lpage">2831</span>, <span class="refDoi"> DOI: 10.1021/ja0771033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0771033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=2817-2831&author=R.+Abelauthor=T.+Youngauthor=R.+Faridauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Role+of+the+active-site+solvent+in+the+thermodynamics+of+factor+Xa+ligand+binding&doi=10.1021%2Fja0771033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the Active-Site Solvent in the Thermodynamics of Factor Xa Ligand Binding</span></div><div class="casAuthors">Abel, Robert; Young, Tom; Farid, Ramy; Berne, Bruce J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2817-2831</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding the underlying physics of the binding of small-mol. ligands to protein active sites is a key objective of computational chem. and biol.  It is widely believed that displacement of water mols. from the active site by the ligand is a principal (if not the dominant) source of binding free energy.  Although continuum theories of hydration are routinely used to describe the contributions of the solvent to the binding affinity of the complex, it is still an unsettled question as to whether or not these continuum solvation theories describe the underlying mol. physics with sufficient accuracy to reliably rank the binding affinities of a set of ligands for a given protein.  Here we develop a novel, computationally efficient descriptor of the contribution of the solvent to the binding free energy of a small mol. and its assocd. receptor that captures the effects of the ligand displacing the solvent from the protein active site with at. detail.  This descriptor quant. predicts (R2 = 0.81) the binding free energy differences between congeneric ligand pairs for the test system factor Xa, elucidates phys. properties of the active site solvent that appear to be missing in most continuum theories of hydration, and identifies several features of the hydration of the factor Xa active site relevant to the structure-activity relationship of its inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS42r59nNXH7Vg90H21EOLACvtfcHk0lgI1SiAv_c4wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D&md5=8eefe76547a82bccaea38f2c84e6cd5f</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fja0771033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0771033%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DRole%2520of%2520the%2520active-site%2520solvent%2520in%2520the%2520thermodynamics%2520of%2520factor%2520Xa%2520ligand%2520binding%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D2817%26epage%3D2831%26doi%3D10.1021%2Fja0771033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mahmoud, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masters, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lill, M. A.</span></span> <span> </span><span class="NLM_article-title">Elucidating the multiple roles of hydration for accurate protein-ligand binding prediction via deep learning</span>. <i>Commun. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">19</span>, <span class="refDoi"> DOI: 10.1038/s42004-020-0261-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1038%2Fs42004-020-0261-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktl2hsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2020&pages=19&author=A.+H.+Mahmoudauthor=M.+R.+Mastersauthor=Y.+Yangauthor=M.+A.+Lill&title=Elucidating+the+multiple+roles+of+hydration+for+accurate+protein-ligand+binding+prediction+via+deep+learning&doi=10.1038%2Fs42004-020-0261-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Elucidating the multiple roles of hydration for accurate protein-ligand binding prediction via deep learning</span></div><div class="casAuthors">Mahmoud, Amr H.; Masters, Matthew R.; Yang, Ying; Lill, Markus A.</div><div class="citationInfo"><span class="NLM_cas:title">Communications Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19</span>CODEN:
                <span class="NLM_cas:coden">CCOHCT</span>;
        ISSN:<span class="NLM_cas:issn">2399-3669</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Accurate and efficient prediction of protein-ligand interactions has been a long-lasting dream of practitioners in drug discovery.  The insufficient treatment of hydration is widely recognized to be a major limitation for accurate protein-ligand scoring.  Using an integration of mol. dynamics simulations on thousands of protein structures with novel big-data analytics based on convolutional neural networks and deep Taylor decompn., we consistently identify here three different patterns of hydration to be essential for protein-ligand interactions.  In addn. to desolvation and water-mediated interactions, the formation of enthalpically favorable networks of first-shell water mols. around solvent-exposed ligand moieties is identified to be essential for protein-ligand binding.  Despite being currently neglected in drug discovery, this hydration phenomenon could lead to new avenues in optimizing the free energy of ligand binding.  Application of deep neural networks incorporating hydration to docking provides 89% accuracy in binding pose ranking, an essential step for rational structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlxlrtqYVzOrVg90H21EOLACvtfcHk0lgI1SiAv_c4wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktl2hsbs%253D&md5=a99707c1848b372d4031d221e460a3c0</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fs42004-020-0261-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42004-020-0261-x%26sid%3Dliteratum%253Aachs%26aulast%3DMahmoud%26aufirst%3DA.%2BH.%26aulast%3DMasters%26aufirst%3DM.%2BR.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DLill%26aufirst%3DM.%2BA.%26atitle%3DElucidating%2520the%2520multiple%2520roles%2520of%2520hydration%2520for%2520accurate%2520protein-ligand%2520binding%2520prediction%2520via%2520deep%2520learning%26jtitle%3DCommun.%2520Chem.%26date%3D2020%26volume%3D3%26spage%3D19%26doi%3D10.1038%2Fs42004-020-0261-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaudreault, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chartier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najmanovich, R.</span></span> <span> </span><span class="NLM_article-title">Side-chain rotamer changes upon ligand binding: Common, crucial, correlate with entropy and rearrange hydrogen bonding</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">i423</span>– <span class="NLM_lpage">i430</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/bts395</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1093%2Fbioinformatics%2Fbts395" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=22962462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlantLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=i423-i430&author=F.+Gaudreaultauthor=M.+Chartierauthor=R.+Najmanovich&title=Side-chain+rotamer+changes+upon+ligand+binding%3A+Common%2C+crucial%2C+correlate+with+entropy+and+rearrange+hydrogen+bonding&doi=10.1093%2Fbioinformatics%2Fbts395"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Side-chain rotamer changes upon ligand binding: Common, crucial, correlate with entropy and rearrange hydrogen bonding</span></div><div class="casAuthors">Gaudreault, Francis; Chartier, Matthieu; Najmanovich, Rafael</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">i423-i430</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Protein movements form a continuum from large domain rearrangements (including folding and restructuring) to side-chain rotamer changes and small rearrangements.  Understanding side-chain flexibility upon binding is important for understanding mol. recognition events and predicting ligand binding.  Methods: In the present work, we developed a well-curated non-redundant dataset of 188 proteins in pairs of structures in the Apo (unbound) and Holo (bound) forms to study the extent and the factors that guide side-chain rotamer changes upon binding.  Results: Our anal. shows that side-chain rotamer changes are widespread with only 10% of binding sites displaying no conformational changes.  Overall, at most five rotamer changes account for the obsd. movements in 90% of the cases.  Furthermore, rotamer changes are essential in 32% of flexible binding sites.  The different amino acids have a 11-fold difference in their probability to undergo changes.  Side-chain flexibility represents an intrinsic property of amino acids as it correlates well with configurational entropy differences.  Furthermore, on av. b-factors and solvent accessible surface areas can discriminate flexible side-chains in the Apo form.  Finally, there is a rearrangement of the hydrogen-bonding network upon binding primarily with a loss of H-bonds with water mols. and a gain of H-bonds with protein residues for flexible residues.  Interestingly, only 25% of side chains capable of forming H-bonds do so with the ligand upon binding.  In terms of drug design, this last result shows that there is a large no. of potential interactions that may be exploited to modulate the specificity and sensitivity of inhibitors.  Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="7e0c1f181f1b1250101f14131f101108171d163e0d161b0c1c0c1111151b501d1f50">[email protected]</a></div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoerwRd3_qiYrVg90H21EOLACvtfcHk0lgI1SiAv_c4wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlantLfO&md5=1cb775e36034683efd17908930e4639d</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbts395&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbts395%26sid%3Dliteratum%253Aachs%26aulast%3DGaudreault%26aufirst%3DF.%26aulast%3DChartier%26aufirst%3DM.%26aulast%3DNajmanovich%26aufirst%3DR.%26atitle%3DSide-chain%2520rotamer%2520changes%2520upon%2520ligand%2520binding%253A%2520Common%252C%2520crucial%252C%2520correlate%2520with%2520entropy%2520and%2520rearrange%2520hydrogen%2520bonding%26jtitle%3DBioinformatics%26date%3D2012%26volume%3D28%26spage%3Di423%26epage%3Di430%26doi%3D10.1093%2Fbioinformatics%2Fbts395" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ravindranathan, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandiyan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekkati, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bae, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel fibroblast growth factor receptor 1 kinase inhibitors by structure-based virtual screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1662</span>– <span class="NLM_lpage">1672</span>, <span class="refDoi"> DOI: 10.1021/jm901386e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901386e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlKkt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=1662-1672&author=K.+P.+Ravindranathanauthor=V.+Mandiyanauthor=A.+R.+Ekkatiauthor=J.+H.+Baeauthor=J.+Schlessingerauthor=W.+L.+Jorgensen&title=Discovery+of+novel+fibroblast+growth+factor+receptor+1+kinase+inhibitors+by+structure-based+virtual+screening&doi=10.1021%2Fjm901386e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Fibroblast Growth Factor Receptor 1 Kinase Inhibitors by Structure-Based Virtual Screening</span></div><div class="casAuthors">Ravindranathan, Krishna P.; Mandiyan, Valsan; Ekkati, Anil R.; Bae, Jae H.; Schlessinger, Joseph; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1662-1672</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fibroblast growth factors (FGFs) play important roles in embryonic development, angiogenesis, wound healing, and cell proliferation and differentiation.  In search of inhibitors of FGFR1 kinase, 2.2 million compds. were docked into the ATP binding site of the protein.  A co-crystal structure, which shows two alternative conformations for the nucleotide binding loop, is reported.  Docking was performed on both conformations and, ultimately, 23 diverse compds. were purchased and assayed.  Following hit validation, two compds. 10 and 16, a benzylidene deriv. of pseudothiohydantoin and a thienopyrimidinone deriv., resp., were discovered that inhibit FGFR1 kinase with IC50 values of 23 and 50 μM.  Initial optimization of 16 led to the more unsatd. 40 (I), which has significantly enhanced potency, 1.9 μM.  The core structures represent new structural motifs for FGFR1 kinase inhibitors.  The study also illustrates complexities assocd. with the choice of protein structures for docking, possible use of multiple kinase structures to seek selectivity, and hit identification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaes0izfiD1rVg90H21EOLACvtfcHk0ljBno4oY34_Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlKkt78%253D&md5=1edb7b1e76644f21899024906aabbb43</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fjm901386e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901386e%26sid%3Dliteratum%253Aachs%26aulast%3DRavindranathan%26aufirst%3DK.%2BP.%26aulast%3DMandiyan%26aufirst%3DV.%26aulast%3DEkkati%26aufirst%3DA.%2BR.%26aulast%3DBae%26aufirst%3DJ.%2BH.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DDiscovery%2520of%2520novel%2520fibroblast%2520growth%2520factor%2520receptor%25201%2520kinase%2520inhibitors%2520by%2520structure-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D1662%26epage%3D1672%26doi%3D10.1021%2Fjm901386e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cournia, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandavadi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bucala, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span> <span> </span><span class="NLM_article-title">Discovery of human macrophage migration inhibitory factor (MIF)-CD74 antagonists via virtual screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">416</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.1021/jm801100v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801100v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFaisrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=416-424&author=Z.+Courniaauthor=L.+Lengauthor=S.+Gandavadiauthor=X.+Duauthor=R.+Bucalaauthor=W.+L.+Jorgensen&title=Discovery+of+human+macrophage+migration+inhibitory+factor+%28MIF%29-CD74+antagonists+via+virtual+screening&doi=10.1021%2Fjm801100v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Human Macrophage Migration Inhibitory Factor (MIF)-CD74 Antagonists via Virtual Screening</span></div><div class="casAuthors">Cournia, Zoe; Leng, Lin; Gandavadi, Sunilkumar; Du, Xin; Bucala, Richard; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">416-424</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Macrophage migration inhibitory factor (MIF) is a cytokine that is involved in the regulation of inflammation as well as cell proliferation and differentiation.  Deactivation of MIF by antibodies or inhibition of MIF binding to its receptor, CD74, attenuates tumor growth and angiogenesis.  To discover small-mol. inhibitors of MIF's biol. activity, virtual screening was performed by docking 2.1 million compds. into the MIF tautomerase active site.  After visual inspection of 1200 top-ranked MIF-ligand complexes, 26 possible inhibitors were selected and purchased and 23 of them were assayed.  The in vitro binding assay for MIF with CD74 revealed that 11 of the compds. have inhibitory activity in the micromolar regime, including four compds. with IC50 values below 5 μM.  Inhibition of MIF tautomerase activity was also established for many of the compds. with IC50 values as low as 0.5 μM; Michaelis-Menten anal. was performed for two cases and confirmed the competitive inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWMPIUdUO7TLVg90H21EOLACvtfcHk0ljBno4oY34_Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFaisrzF&md5=56f32f68318413b1f1f8a6d141aefb68</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjm801100v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801100v%26sid%3Dliteratum%253Aachs%26aulast%3DCournia%26aufirst%3DZ.%26aulast%3DLeng%26aufirst%3DL.%26aulast%3DGandavadi%26aufirst%3DS.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DBucala%26aufirst%3DR.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DDiscovery%2520of%2520human%2520macrophage%2520migration%2520inhibitory%2520factor%2520%2528MIF%2529-CD74%2520antagonists%2520via%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D416%26epage%3D424%26doi%3D10.1021%2Fjm801100v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dayhoff, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchenko, A. R.</span></span> <span> </span><span class="NLM_article-title">Evolution of protein binding modes in homooligomers</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>395</i></span>,  <span class="NLM_fpage">860</span>– <span class="NLM_lpage">870</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2009.10.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1016%2Fj.jmb.2009.10.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=19879880" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntlyisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=395&publication_year=2010&pages=860-870&author=J.+E.+Dayhoffauthor=B.+A.+Shoemakerauthor=S.+H.+Bryantauthor=A.+R.+Panchenko&title=Evolution+of+protein+binding+modes+in+homooligomers&doi=10.1016%2Fj.jmb.2009.10.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Evolution of Protein Binding Modes in Homooligomers</span></div><div class="casAuthors">Dayhoff, Judith E.; Shoemaker, Benjamin A.; Bryant, Stephen H.; Panchenko, Anna R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">395</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">860-870</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The evolution of protein interactions cannot be deciphered without a detailed anal. of interaction interfaces and binding modes.  We performed a large-scale study of protein homooligomers in terms of their symmetry, interface sizes, and conservation of binding modes.  We also focused specifically on the evolution of protein binding modes from nine families of homooligomers and mapped 60 different binding modes and oligomerization states onto the phylogenetic trees of these families.  We obsd. a significant tendency for the same binding modes to be clustered together and conserved within clades on phylogenetic trees; this trend is esp. pronounced for close homologs with 70% sequence identity or higher.  Some binding modes are conserved among very distant homologs, pointing to their ancient evolutionary origin, while others are very specific for a certain phylogenetic group.  Moreover, we found that the most ancient binding modes have a tendency to involve sym. (isologous) homodimer binding arrangements with larger interfaces, while recently evolved binding modes more often exhibit asym. arrangements and smaller interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKqIRe6N3n5bVg90H21EOLACvtfcHk0ljBno4oY34_Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntlyisQ%253D%253D&md5=192444c54d01efa6903bf0af8fe92df2</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2009.10.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2009.10.052%26sid%3Dliteratum%253Aachs%26aulast%3DDayhoff%26aufirst%3DJ.%2BE.%26aulast%3DShoemaker%26aufirst%3DB.%2BA.%26aulast%3DBryant%26aufirst%3DS.%2BH.%26aulast%3DPanchenko%26aufirst%3DA.%2BR.%26atitle%3DEvolution%2520of%2520protein%2520binding%2520modes%2520in%2520homooligomers%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D395%26spage%3D860%26epage%3D870%26doi%3D10.1016%2Fj.jmb.2009.10.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kasahara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinoshita, K.</span></span> <span> </span><span class="NLM_article-title">Landscape of protein-small ligand binding modes</span>. <i>Protein science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1659</span>– <span class="NLM_lpage">1671</span>, <span class="refDoi"> DOI: 10.1002/pro.2971</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1002%2Fpro.2971" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=27327045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSiurzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2016&pages=1659-1671&author=K.+Kasaharaauthor=K.+Kinoshita&title=Landscape+of+protein-small+ligand+binding+modes&doi=10.1002%2Fpro.2971"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Landscape of protein-small ligand binding modes</span></div><div class="casAuthors">Kasahara, Kota; Kinoshita, Kengo</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1659-1671</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Elucidating the mechanisms of specific small-mol. (ligand) recognition by proteins is a long-standing conundrum.  While the structures of these mols., proteins and ligands, have been extensively studied, protein-ligand interactions, or binding modes, have not been comprehensively analyzed.  Although methods for assessing similarities of binding site structures have been extensively developed, the methods for the computational treatment of binding modes have not been well established.  Here, we developed a computational method for encoding the information about binding modes as graphs, and assessing their similarities.  An all-against-all comparison of 20,040 protein-ligand complexes provided the landscape of the protein-ligand binding modes and its relationships with protein- and chem. spaces.  While similar proteins in the same SCOP Family tend to bind relatively similar ligands with similar binding modes, the correlation between ligand and binding similarities was not very high (R2 = 0.443).  We found many pairs with novel relationships, in which two evolutionally distant proteins recognize dissimilar ligands by similar binding modes (757,474 pairs out of 200,790,780 pairs were categorized into this relationship, in our dataset).  In addn., there were an abundance of pairs of homologous proteins binding to similar ligands with different binding modes (68,217 pairs).  Our results showed that many interesting relationships between protein-ligand complexes are still hidden in the structure database, and our new method for assessing binding mode similarities is effective to find them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD4kKAT8zhJrVg90H21EOLACvtfcHk0ljBno4oY34_Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSiurzP&md5=eb3b5974ebc6c56573c75af3d8ad2037</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2Fpro.2971&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.2971%26sid%3Dliteratum%253Aachs%26aulast%3DKasahara%26aufirst%3DK.%26aulast%3DKinoshita%26aufirst%3DK.%26atitle%3DLandscape%2520of%2520protein-small%2520ligand%2520binding%2520modes%26jtitle%3DProtein%2520science%26date%3D2016%26volume%3D25%26spage%3D1659%26epage%3D1671%26doi%3D10.1002%2Fpro.2971" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Damm-Ganamet, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunbar, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, H. A.</span></span> <span> </span><span class="NLM_article-title">CSAR benchmark exercise 2011–2012: Evaluation of results from docking and relative ranking of blinded congeneric series</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1853</span>– <span class="NLM_lpage">1870</span>, <span class="refDoi"> DOI: 10.1021/ci400025f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400025f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltV2msbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=1853-1870&author=K.+L.+Damm-Ganametauthor=R.+D.+Smithauthor=J.+B.+Dunbarauthor=J.+A.+Stuckeyauthor=H.+A.+Carlson&title=CSAR+benchmark+exercise+2011%E2%80%932012%3A+Evaluation+of+results+from+docking+and+relative+ranking+of+blinded+congeneric+series&doi=10.1021%2Fci400025f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">CSAR Benchmark Exercise 2011-2012: Evaluation of Results from Docking and Relative Ranking of Blinded Congeneric Series</span></div><div class="casAuthors">Damm-Ganamet, Kelly L.; Smith, Richard D.; Dunbar, James B.; Stuckey, Jeanne A.; Carlson, Heather A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1853-1870</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Community Structure-Activity Resource (CSAR) recently held its first blinded exercise based on data provided by Abbott, Vertex, and colleagues at the University of Michigan, Ann Arbor.  A total of 20 research groups submitted results for the benchmark exercise where the goal was to compare different improvements for pose prediction, enrichment, and relative ranking of congeneric series of compds.  The exercise was built around blinded high-quality exptl. data from four protein targets: LpxC, Urokinase, Chk1, and Erk2.  Pose prediction proved to be the most straightforward task, and most methods were able to successfully reproduce binding poses when the crystal structure employed was co-crystd. with a ligand from the same chem. series.  Multiple evaluation metrics were examd., and we found that RMSD and native contact metrics together provide a robust evaluation of the predicted poses.  It was notable that most scoring functions underpredicted contacts between the hetero atoms (i.e., N, O, S, etc.) of the protein and ligand.  Relative ranking was found to be the most difficult area for the methods, but many of the scoring functions were able to properly identify Urokinase actives from the inactives in the series.  Lastly, we found that minimizing the protein and correcting histidine tautomeric states pos. trended with low RMSD for pose prediction but minimizing the ligand neg. trended.  Pregenerated ligand conformations performed better than those that were generated on the fly.  Optimizing docking parameters and pretraining with the native ligand had a pos. effect on the docking performance as did using restraints, substructure fitting, and shape fitting.  Lastly, for both sampling and ranking scoring functions, the use of the empirical scoring function appeared to trend pos. with the RMSD.  Here, by combining the results of many methods, we hope to provide a statistically relevant evaluation and elucidate specific shortcomings of docking methodol. for the community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVoKpI9PikibVg90H21EOLACvtfcHk0lheLFTM7joYdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltV2msbg%253D&md5=80f080270d091cee06ca9eb31345144e</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fci400025f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400025f%26sid%3Dliteratum%253Aachs%26aulast%3DDamm-Ganamet%26aufirst%3DK.%2BL.%26aulast%3DSmith%26aufirst%3DR.%2BD.%26aulast%3DDunbar%26aufirst%3DJ.%2BB.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DCSAR%2520benchmark%2520exercise%25202011%25E2%2580%25932012%253A%2520Evaluation%2520of%2520results%2520from%2520docking%2520and%2520relative%2520ranking%2520of%2520blinded%2520congeneric%2520series%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D1853%26epage%3D1870%26doi%3D10.1021%2Fci400025f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hevener, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehboob, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, P.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truong, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boci, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghassemi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, M. E.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel and potent class of F. tularensis enoyl-reductase (FabI) inhibitors by molecular shape and electrostatic matching</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">268</span>– <span class="NLM_lpage">279</span>, <span class="refDoi"> DOI: 10.1021/jm201168g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201168g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsV2jurbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=268-279&author=K.+E.+Hevenerauthor=S.+Mehboobauthor=P.-C.+Suauthor=K.+Truongauthor=T.+Bociauthor=J.+Dengauthor=M.+Ghassemiauthor=J.+L.+Cookauthor=M.+E.+Johnson&title=Discovery+of+a+novel+and+potent+class+of+F.+tularensis+enoyl-reductase+%28FabI%29+inhibitors+by+molecular+shape+and+electrostatic+matching&doi=10.1021%2Fjm201168g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Novel and Potent Class of F. tularensis Enoyl-Reductase (FabI) Inhibitors by Molecular Shape and Electrostatic Matching</span></div><div class="casAuthors">Hevener, Kirk E.; Mehboob, Shahila; Su, Pin-Chih; Truong, Kent; Boci, Teuta; Deng, Jiangping; Ghassemi, Mahmood; Cook, James L.; Johnson, Michael E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">268-279</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enoyl-acyl carrier protein (ACP) reductase, FabI, is a key enzyme in the bacterial fatty acid biosynthesis pathway (FAS II).  FabI is an NADH-dependent oxidoreductase that acts to reduce enoyl-ACP substrates in a final step of the pathway.  The absence of this enzyme in humans makes it an attractive target for the development of new antibacterial agents.  FabI is known to be unresponsive to structure-based design efforts due to a high degree of induced fit and a mobile flexible loop encompassing the active site.  Here we discuss the development, validation, and careful application of a ligand-based virtual screen used for the identification of novel inhibitors of the Francisella tularensis FabI target.  In this study, four known classes of FabI inhibitors were used as templates for virtual screens that involved mol. shape and electrostatic matching.  The program ROCS was used to search a high-throughput screening library for compds. that matched any of the four mol. shape queries.  Matching compds. were further refined using the program EON, which compares and scores compds. by matching electrostatic properties.  Using these techniques, 50 compds. were selected, ordered, and tested.  The tested compds. possessed novel chem. scaffolds when compared to the input query compds.  Several hits with low micromolar activity were identified and follow-up scaffold-based searches resulted in the identification of a lead series with submicromolar enzyme inhibition, high ligand efficiency, and a novel scaffold.  Addnl., one of the most active compds. showed promising whole-cell antibacterial activity against several Gram-pos. and Gram-neg. species, including the target pathogen.  The results of a preliminary structure-activity relationship anal. are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEn7n0O_iCD7Vg90H21EOLACvtfcHk0lheLFTM7joYdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsV2jurbO&md5=c348cb0264e4f9d740991a15c7973252</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fjm201168g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201168g%26sid%3Dliteratum%253Aachs%26aulast%3DHevener%26aufirst%3DK.%2BE.%26aulast%3DMehboob%26aufirst%3DS.%26aulast%3DSu%26aufirst%3DP.-C.%26aulast%3DTruong%26aufirst%3DK.%26aulast%3DBoci%26aufirst%3DT.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DGhassemi%26aufirst%3DM.%26aulast%3DCook%26aufirst%3DJ.%2BL.%26aulast%3DJohnson%26aufirst%3DM.%2BE.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520and%2520potent%2520class%2520of%2520F.%2520tularensis%2520enoyl-reductase%2520%2528FabI%2529%2520inhibitors%2520by%2520molecular%2520shape%2520and%2520electrostatic%2520matching%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D268%26epage%3D279%26doi%3D10.1021%2Fjm201168g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwivedi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahasrabuddhe, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqi, M. I.</span></span> <span> </span><span class="NLM_article-title">Integrated machine learning, molecular docking and 3D-QSAR based approach for identification of potential inhibitors of trypanosomal N-myristoyltransferase</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">3711</span>– <span class="NLM_lpage">3723</span>, <span class="refDoi"> DOI: 10.1039/C6MB00574H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1039%2FC6MB00574H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=27766319" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1GqtbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=3711-3723&author=N.+Singhauthor=P.+Shahauthor=H.+Dwivediauthor=S.+Mishraauthor=R.+Tripathiauthor=A.+A.+Sahasrabuddheauthor=M.+I.+Siddiqi&title=Integrated+machine+learning%2C+molecular+docking+and+3D-QSAR+based+approach+for+identification+of+potential+inhibitors+of+trypanosomal+N-myristoyltransferase&doi=10.1039%2FC6MB00574H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Integrated machine learning, molecular docking and 3D-QSAR based approach for identification of potential inhibitors of trypanosomal N-myristoyltransferase</span></div><div class="casAuthors">Singh, Nidhi; Shah, Priyanka; Dwivedi, Hemlata; Mishra, Shikha; Tripathi, Renu; Sahasrabuddhe, Amogh A.; Siddiqi, Mohammad Imran</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3711-3723</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In the present study, a machine learning approach, docking studies and CoMFA anal. have been integrated with the objective of translation of knowledge into a pipelined workflow towards the identification of putative hits through the screening of large compd. libraries.  In the proposed pipeline, the reported parasitic NMT inhibitors have been used to develop predictive machine learning classification models.  Simultaneously, a TbNMT complex model was generated to establish the relationship between the binding mode of the inhibitors for LmNMT and TbNMT through mol. dynamics simulation studies.  A 3D-QSAR model was developed and used to predict the activity of the proposed hits in the subsequent step.  The hits classified as active based on the machine learning model were assessed as the potential anti-trypanosomal NMT inhibitors through mol. docking studies, predicted activity using a QSAR model and visual inspection.  In the final step, the proposed pipeline was validated through in vitro expts.  A total of seven hits have been proposed and tested in vitro for evaluation of dual inhibitory activity against Leishmania donovani and Trypanosoma brucei.  Out of these five compds. showed significant inhibition against both of the organisms.  The common topmost active compd. SEW04173 belongs to a pyrazole carboxylate scaffold and is anticipated to enrich the chem. space with enhanced potency through optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC1rsgog_YdrVg90H21EOLACvtfcHk0lheLFTM7joYdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1GqtbbN&md5=69693da42315b852f285b58394313d7f</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1039%2FC6MB00574H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MB00574H%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DN.%26aulast%3DShah%26aufirst%3DP.%26aulast%3DDwivedi%26aufirst%3DH.%26aulast%3DMishra%26aufirst%3DS.%26aulast%3DTripathi%26aufirst%3DR.%26aulast%3DSahasrabuddhe%26aufirst%3DA.%2BA.%26aulast%3DSiddiqi%26aufirst%3DM.%2BI.%26atitle%3DIntegrated%2520machine%2520learning%252C%2520molecular%2520docking%2520and%25203D-QSAR%2520based%2520approach%2520for%2520identification%2520of%2520potential%2520inhibitors%2520of%2520trypanosomal%2520N-myristoyltransferase%26jtitle%3DMol.%2520BioSyst.%26date%3D2016%26volume%3D12%26spage%3D3711%26epage%3D3723%26doi%3D10.1039%2FC6MB00574H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaieb, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gathiaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feher, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burley, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaro, R. E.</span></span> <span> </span><span class="NLM_article-title">D3R grand challenge 2: Blind prediction of protein-ligand poses, affinity rankings, and relative binding free energies</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1007/s10822-017-0088-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1007%2Fs10822-017-0088-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=29204945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFWgtLfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=1-20&author=Z.+Gaiebauthor=S.+Liuauthor=S.+Gathiakaauthor=M.+Chiuauthor=H.+Yangauthor=C.+Shaoauthor=V.+A.+Feherauthor=W.+P.+Waltersauthor=B.+Kuhnauthor=M.+G.+Rudolphauthor=S.+K.+Burleyauthor=M.+K.+Gilsonauthor=R.+E.+Amaro&title=D3R+grand+challenge+2%3A+Blind+prediction+of+protein-ligand+poses%2C+affinity+rankings%2C+and+relative+binding+free+energies&doi=10.1007%2Fs10822-017-0088-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">D3R Grand Challenge 2: blind prediction of protein-ligand poses, affinity rankings, and relative binding free energies</span></div><div class="casAuthors">Gaieb, Zied; Liu, Shuai; Gathiaka, Symon; Chiu, Michael; Yang, Huanwang; Shao, Chenghua; Feher, Victoria A.; Walters, W. Patrick; Kuhn, Bernd; Rudolph, Markus G.; Burley, Stephen K.; Gilson, Michael K.; Amaro, Rommie E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-20</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The Drug Design Data Resource (D3R) ran Grand Challenge 2 (GC2) from Sept. 2016 through Feb. 2017.  This challenge was based on a dataset of structures and affinities for the nuclear receptor farnesoid X receptor (FXR), contributed by F. Hoffman-La Roche.  The dataset contained 102 IC50 values, spanning six orders of magnitude, and 36 high-resoln. co-crystal structures with representatives of four major ligand classes.  Strong global participation was evident, with 49 participants submitting 262 prediction submission packages in total.  Procedurally, GC2 mimicked Grand Challenge 2015 (GC2015), with a Stage 1 subchallenge testing ligand pose prediction methods and ranking and scoring methods, and a Stage 2 subchallenge testing only ligand ranking and scoring methods after the release of all blinded co-crystal structures.  Two smaller curated sets of 18 and 15 ligands were developed to test alchem. free energy methods.  This overview summarizes all aspects of GC2, including the dataset details, challenge procedures, and participant results.  We also consider implications for progress in the field, while highlighting methodol. areas that merit continued development.  Similar to GC2015, the outcome of GC2 underscores the pressing need for methods development in pose prediction, particularly for ligand scaffolds not currently represented in the Protein Data Bank (http://www.pdb.org), and in affinity ranking and scoring of bound ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFXlYV_AkMJLVg90H21EOLACvtfcHk0lhSo9SUUg5TYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFWgtLfE&md5=1f8afb98386bf9d6584d63deafb62114</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1007%2Fs10822-017-0088-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-017-0088-4%26sid%3Dliteratum%253Aachs%26aulast%3DGaieb%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DGathiaka%26aufirst%3DS.%26aulast%3DChiu%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DShao%26aufirst%3DC.%26aulast%3DFeher%26aufirst%3DV.%2BA.%26aulast%3DWalters%26aufirst%3DW.%2BP.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DRudolph%26aufirst%3DM.%2BG.%26aulast%3DBurley%26aufirst%3DS.%2BK.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26aulast%3DAmaro%26aufirst%3DR.%2BE.%26atitle%3DD3R%2520grand%2520challenge%25202%253A%2520Blind%2520prediction%2520of%2520protein-ligand%2520poses%252C%2520affinity%2520rankings%252C%2520and%2520relative%2520binding%2520free%2520energies%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2018%26volume%3D32%26spage%3D1%26epage%3D20%26doi%3D10.1007%2Fs10822-017-0088-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gathiaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubish, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunbar, J. B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burley, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaro, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feher, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span> <span> </span><span class="NLM_article-title">D3R grand challenge 2015: Evaluation of protein-ligand pose and affinity predictions</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">651</span>– <span class="NLM_lpage">668</span>, <span class="refDoi"> DOI: 10.1007/s10822-016-9946-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1007%2Fs10822-016-9946-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=27696240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1SlsbzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=651-668&author=S.+Gathiakaauthor=S.+Liuauthor=M.+Chiuauthor=H.+Yangauthor=J.+A.+Stuckeyauthor=Y.+N.+Kangauthor=J.+Delpropostoauthor=G.+Kubishauthor=J.+B.+J.+Dunbarauthor=H.+A.+Carlsonauthor=S.+K.+Burleyauthor=W.+P.+Waltersauthor=R.+E.+Amaroauthor=V.+A.+Feherauthor=M.+K.+Gilson&title=D3R+grand+challenge+2015%3A+Evaluation+of+protein-ligand+pose+and+affinity+predictions&doi=10.1007%2Fs10822-016-9946-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">D3R grand challenge 2015: Evaluation of protein-ligand pose and affinity predictions</span></div><div class="casAuthors">Gathiaka, Symon; Liu, Shuai; Chiu, Michael; Yang, Huanwang; Stuckey, Jeanne A.; Kang, You Na; Delproposto, Jim; Kubish, Ginger; Dunbar, Jr., James B.; Carlson, Heather A.; Burley, Stephen K.; Walters, W. Patrick; Amaro, Rommie E.; Feher, Victoria A.; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">651-668</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The Drug Design Data Resource (D3R) ran Grand Challenge 2015 between Sept. 2015 and Feb. 2016.  Two targets served as the framework to test community docking and scoring methods: (1) HSP90, donated by AbbVie and the Community Structure Activity Resource (CSAR), and (2) MAP4K4, donated by Genentech.  The challenges for both target datasets were conducted in two stages, with the first stage testing pose predictions and the capacity to rank compds. by affinity with minimal structural data; and the second stage testing methods for ranking compds. with knowledge of at least a subset of the ligand-protein poses.  An addnl. sub-challenge provided small groups of chem. similar HSP90 compds. amenable to alchem. calcns. of relative binding free energy.  Unlike previous blinded Challenges, we did not provide cognate receptors or receptors prepd. with hydrogens and likewise did not require a specified crystal structure to be used for pose or affinity prediction in Stage 1.  Given the freedom to select from over 200 crystal structures of HSP90 in the PDB, participants employed workflows that tested not only core docking and scoring technologies, but also methods for addressing water-mediated ligand-protein interactions, binding pocket flexibility, and the optimal selection of protein structures for use in docking calcns.  Nearly 40 participating groups submitted over 350 prediction sets for Grand Challenge 2015.  This overview describes the datasets and the organization of the challenge components, summarizes the results across all submitted predictions, and considers broad conclusions that may be drawn from this collaborative community endeavor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvQmGRFiZZW7Vg90H21EOLACvtfcHk0lhSo9SUUg5TYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1SlsbzN&md5=f9779f61ad8224f7d6bbaaf0cd97bc24</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1007%2Fs10822-016-9946-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-016-9946-8%26sid%3Dliteratum%253Aachs%26aulast%3DGathiaka%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DChiu%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DKang%26aufirst%3DY.%2BN.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DKubish%26aufirst%3DG.%26aulast%3DDunbar%26aufirst%3DJ.%2BB.%2BJ.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26aulast%3DBurley%26aufirst%3DS.%2BK.%26aulast%3DWalters%26aufirst%3DW.%2BP.%26aulast%3DAmaro%26aufirst%3DR.%2BE.%26aulast%3DFeher%26aufirst%3DV.%2BA.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DD3R%2520grand%2520challenge%25202015%253A%2520Evaluation%2520of%2520protein-ligand%2520pose%2520and%2520affinity%2520predictions%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2016%26volume%3D30%26spage%3D651%26epage%3D668%26doi%3D10.1007%2Fs10822-016-9946-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mobley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wymer, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perryman, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">Blind prediction of HIV integrase binding from the SAMPL4 challenge</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">345</span>, <span class="refDoi"> DOI: 10.1007/s10822-014-9723-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1007%2Fs10822-014-9723-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=24595873" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjsVKqu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=327-345&author=D.+L.+Mobleyauthor=S.+Liuauthor=N.+M.+Limauthor=K.+L.+Wymerauthor=A.+L.+Perrymanauthor=S.+Forliauthor=N.+Dengauthor=J.+Suauthor=K.+Bransonauthor=A.+J.+Olson&title=Blind+prediction+of+HIV+integrase+binding+from+the+SAMPL4+challenge&doi=10.1007%2Fs10822-014-9723-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Blind prediction of HIV integrase binding from the SAMPL4 challenge</span></div><div class="casAuthors">Mobley, David L.; Liu, Shuai; Lim, Nathan M.; Wymer, Karisa L.; Perryman, Alexander L.; Forli, Stefano; Deng, Nanjie; Su, Justin; Branson, Kim; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">327-345</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Here, we give an overview of the protein-ligand binding portion of the Statistical Assessment of Modeling of Proteins and Ligands 4 (SAMPL4) challenge, which focused on predicting binding of HIV integrase inhibitors in the catalytic core domain.  The challenge encompassed three components-a small "virtual screening" challenge, a binding mode prediction component, and a small affinity prediction component.  Here, we give summary results and statistics concerning the performance of all submissions at each of these challenges.  Virtual screening was particularly challenging here in part because, in contrast to more typical virtual screening test sets, the inactive compds. were tested because they were thought to be likely binders, so only the very top predictions performed significantly better than random.  Pose prediction was also quite challenging, in part because inhibitors in the set bind to three different sites, so even identifying the correct binding site was challenging.  Still, the best methods managed low root mean squared deviation predictions in many cases.  Here, we give an overview of results, highlight some features of methods which worked particularly well, and refer the interested reader to papers in this issue which describe specific submissions for addnl. details.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPK-7gJzuLe7Vg90H21EOLACvtfcHk0lhSo9SUUg5TYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjsVKqu7Y%253D&md5=9ec3665496c57b52116768e9de03aefe</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9723-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9723-5%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLim%26aufirst%3DN.%2BM.%26aulast%3DWymer%26aufirst%3DK.%2BL.%26aulast%3DPerryman%26aufirst%3DA.%2BL.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DDeng%26aufirst%3DN.%26aulast%3DSu%26aufirst%3DJ.%26aulast%3DBranson%26aufirst%3DK.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DBlind%2520prediction%2520of%2520HIV%2520integrase%2520binding%2520from%2520the%2520SAMPL4%2520challenge%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2014%26volume%3D28%26spage%3D327%26epage%3D345%26doi%3D10.1007%2Fs10822-014-9723-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perryman, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santiago, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos-Martins, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">Virtual screening with AutoDock Vina and the common pharmacophore engine of a low diversity library of fragments and hits against the three allosteric sites of HIV integrase: Participation in the SAMPL4 protein-ligand binding challenge</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">429</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1007/s10822-014-9709-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1007%2Fs10822-014-9709-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=24493410" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslSmu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=429-441&author=A.+L.+Perrymanauthor=D.+N.+Santiagoauthor=S.+Forliauthor=D.+Santos-Martinsauthor=A.+J.+Olson&title=Virtual+screening+with+AutoDock+Vina+and+the+common+pharmacophore+engine+of+a+low+diversity+library+of+fragments+and+hits+against+the+three+allosteric+sites+of+HIV+integrase%3A+Participation+in+the+SAMPL4+protein-ligand+binding+challenge&doi=10.1007%2Fs10822-014-9709-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening with AutoDock Vina and the common pharmacophore engine of a low diversity library of fragments and hits against the three allosteric sites of HIV integrase: participation in the SAMPL4 protein-ligand binding challenge</span></div><div class="casAuthors">Perryman, Alexander L.; Santiago, Daniel N.; Forli, Stefano; Santos-Martins, Diogo; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">429-441</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">To rigorously assess the tools and protocols that can be used to understand and predict macromol. recognition, and to gain more structural insight into three newly discovered allosteric binding sites on a crit. drug target involved in the treatment of HIV infections, the Olson and Levy labs collaborated on the SAMPL4 challenge.  This computational blind challenge involved predicting protein-ligand binding against the three allosteric sites of HIV integrase (IN), a viral enzyme for which two drugs (that target the active site) have been approved by the FDA.  Pos. control cross-docking expts. were utilized to select 13 receptor models out of an initial ensemble of 41 different crystal structures of HIV IN.  These 13 models of the targets were selected using our new "Rank Difference Ratio" metric.  The first stage of SAMPL4 involved using virtual screens to identify 62 active, allosteric IN inhibitors out of a set of 321 compds.  The second stage involved predicting the binding site(s) and crystallog. binding mode(s) for 57 of these inhibitors.  Our team submitted four entries for the first stage that utilized: (1) AutoDock Vina (AD Vina) plus visual inspection; (2) a new common pharmacophore engine; (3) BEDAM replica exchange free energy simulations, and a Consensus approach that combined the predictions of all three strategies.  Even with the SAMPL4's very challenging compd. library that displayed a significantly lower amt. of structural diversity than most libraries that are conventionally employed in prospective virtual screens, these approaches produced hit rates of 24, 25, 34, and 27 %, resp., on a set with 19 % declared binders.  Our only entry for the second stage challenge was based on the results of AD Vina plus visual inspection, and it ranked third place overall according to several different metrics provided by the SAMPL4 organizers.  The successful results displayed by these approaches highlight the utility of the computational structure-based drug discovery tools and strategies that are being developed to advance the goals of the newly created, multi-institution, NIH-funded center called the "HIV Interaction and Viral Evolution Center".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq58ngCMTIjsrVg90H21EOLACvtfcHk0li9b7qbR_wgww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslSmu7s%253D&md5=fb665768bc09c899e1cb33707777692a</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9709-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9709-3%26sid%3Dliteratum%253Aachs%26aulast%3DPerryman%26aufirst%3DA.%2BL.%26aulast%3DSantiago%26aufirst%3DD.%2BN.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DSantos-Martins%26aufirst%3DD.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DVirtual%2520screening%2520with%2520AutoDock%2520Vina%2520and%2520the%2520common%2520pharmacophore%2520engine%2520of%2520a%2520low%2520diversity%2520library%2520of%2520fragments%2520and%2520hits%2520against%2520the%2520three%2520allosteric%2520sites%2520of%2520HIV%2520integrase%253A%2520Participation%2520in%2520the%2520SAMPL4%2520protein-ligand%2520binding%2520challenge%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2014%26volume%3D28%26spage%3D429%26epage%3D441%26doi%3D10.1007%2Fs10822-014-9709-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm-Ganamet, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunbar, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubish, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tinberg, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khare, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doyle, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, H. A.</span></span> <span> </span><span class="NLM_article-title">CSAR benchmark exercise 2013: Evaluation of results from a combined computational protein design, docking, and scoring/ranking challenge</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1022</span>– <span class="NLM_lpage">1031</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00387</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00387" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFKqs7%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1022-1031&author=R.+D.+Smithauthor=K.+L.+Damm-Ganametauthor=J.+B.+Dunbarauthor=A.+Ahmedauthor=K.+Chinnaswamyauthor=J.+E.+Delpropostoauthor=G.+M.+Kubishauthor=C.+E.+Tinbergauthor=S.+D.+Khareauthor=J.+Douauthor=L.+Doyleauthor=J.+A.+Stuckeyauthor=D.+Bakerauthor=H.+A.+Carlson&title=CSAR+benchmark+exercise+2013%3A+Evaluation+of+results+from+a+combined+computational+protein+design%2C+docking%2C+and+scoring%2Franking+challenge&doi=10.1021%2Facs.jcim.5b00387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">CSAR Benchmark Exercise 2013: Evaluation of Results from a Combined Computational Protein Design, Docking, and Scoring/Ranking Challenge</span></div><div class="casAuthors">Smith, Richard D.; Damm-Ganamet, Kelly L.; Dunbar, James B.; Ahmed, Aqeel; Chinnaswamy, Krishnapriya; Delproposto, James E.; Kubish, Ginger M.; Tinberg, Christine E.; Khare, Sagar D.; Dou, Jiayi; Doyle, Lindsey; Stuckey, Jeanne A.; Baker, David; Carlson, Heather A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1031</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Community Structure-Activity Resource (CSAR) conducted a benchmark exercise to evaluate the current computational methods for protein design, ligand docking, and scoring/ranking.  The exercise consisted of three phases.  The first phase required the participants to identify and rank order which designed sequences were able to bind the small mol. digoxigenin.  The second phase challenged the community to select a near-native pose of digoxigenin from a set of decoy poses for two of the designed proteins.  The third phase investigated the ability of current methods to rank/score the binding affinity of 10 related steroids to one of the designed proteins (pKd = 4.1 to 6.7).  We found that 11 of 13 groups were able to correctly select the sequence that bound digoxigenin, with most groups providing the correct three-dimensional structure for the backbone of the protein as well as all atoms of the active-site residues.  Eleven of the 14 groups were able to select the appropriate pose from a set of plausible decoy poses.  The ability to predict abs. binding affinities is still a difficult task, as 8 of 14 groups were able to correlate scores to affinity (Pearson-r > 0.7) of the designed protein for congeneric steroids and only 5 of 14 groups were able to correlate the ranks of the 10 related ligands (Spearman-ρ > 0.7).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKmOuy7Lp9TbVg90H21EOLACvtfcHk0li9b7qbR_wgww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFKqs7%252FJ&md5=9dfde47772a4220f9b49c1149c0e68bd</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00387%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DR.%2BD.%26aulast%3DDamm-Ganamet%26aufirst%3DK.%2BL.%26aulast%3DDunbar%26aufirst%3DJ.%2BB.%26aulast%3DAhmed%26aufirst%3DA.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%2BE.%26aulast%3DKubish%26aufirst%3DG.%2BM.%26aulast%3DTinberg%26aufirst%3DC.%2BE.%26aulast%3DKhare%26aufirst%3DS.%2BD.%26aulast%3DDou%26aufirst%3DJ.%26aulast%3DDoyle%26aufirst%3DL.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DBaker%26aufirst%3DD.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DCSAR%2520benchmark%2520exercise%25202013%253A%2520Evaluation%2520of%2520results%2520from%2520a%2520combined%2520computational%2520protein%2520design%252C%2520docking%252C%2520and%2520scoring%252Franking%2520challenge%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D1022%26epage%3D1031%26doi%3D10.1021%2Facs.jcim.5b00387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xing, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolodziej, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurumbail, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neal, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skepner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberds, S. L.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent inhibitors of soluble epoxide hydrolase by combinatorial library design and structure-based virtual screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1211</span>– <span class="NLM_lpage">1222</span>, <span class="refDoi"> DOI: 10.1021/jm101382t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101382t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Cqu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1211-1222&author=L.+Xingauthor=J.+J.+McDonaldauthor=S.+A.+Kolodziejauthor=R.+G.+Kurumbailauthor=J.+M.+Williamsauthor=C.+J.+Warrenauthor=J.+M.+O%E2%80%99Nealauthor=J.+E.+Skepnerauthor=S.+L.+Roberds&title=Discovery+of+potent+inhibitors+of+soluble+epoxide+hydrolase+by+combinatorial+library+design+and+structure-based+virtual+screening&doi=10.1021%2Fjm101382t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent Inhibitors of Soluble Epoxide Hydrolase by Combinatorial Library Design and Structure-Based Virtual Screening</span></div><div class="casAuthors">Xing, Li; McDonald, Joseph J.; Kolodziej, Steve A.; Kurumbail, Ravi G.; Williams, Jennifer L.; Warren, Chad J.; O'Neal, Janet M.; Skepner, Jill E.; Roberds, Steven L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1211-1222</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-based virtual screening was applied to design combinatorial libraries to discover novel and potent sol. epoxide hydrolase (sEH) inhibitors.  X-ray crystal structures revealed unique interactions for a benzoxazole template in addn. to the conserved hydrogen bonds with the catalytic machinery of sEH.  By exploitation of the favorable binding elements, two iterations of library design based on amide coupling were employed, guided principally by the docking results of the enumerated virtual products.  Biol. screening of the libraries demonstrated as high as 90% hit rate, of which over two dozen compds. were single digit nanomolar sEH inhibitors by IC50 detn.  In total the library design and synthesis produced more than 300 submicromolar sEH inhibitors.  In cellular systems consistent activities were demonstrated with biochem. measurements.  The SAR understanding of the benzoxazole template provides valuable insights into discovery of novel sEH inhibitors as therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3tpDmXCJWBLVg90H21EOLACvtfcHk0li9b7qbR_wgww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Cqu7s%253D&md5=01775ad103a3a5e153b77157ef184608</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fjm101382t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101382t%26sid%3Dliteratum%253Aachs%26aulast%3DXing%26aufirst%3DL.%26aulast%3DMcDonald%26aufirst%3DJ.%2BJ.%26aulast%3DKolodziej%26aufirst%3DS.%2BA.%26aulast%3DKurumbail%26aufirst%3DR.%2BG.%26aulast%3DWilliams%26aufirst%3DJ.%2BM.%26aulast%3DWarren%26aufirst%3DC.%2BJ.%26aulast%3DO%25E2%2580%2599Neal%26aufirst%3DJ.%2BM.%26aulast%3DSkepner%26aufirst%3DJ.%2BE.%26aulast%3DRoberds%26aufirst%3DS.%2BL.%26atitle%3DDiscovery%2520of%2520potent%2520inhibitors%2520of%2520soluble%2520epoxide%2520hydrolase%2520by%2520combinatorial%2520library%2520design%2520and%2520structure-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1211%26epage%3D1222%26doi%3D10.1021%2Fjm101382t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neranjan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smieško, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lill, M. A.</span></span> <span> </span><span class="NLM_article-title">Potential inhibitors for novel coronavirus protease identified by virtual screening of 606 million compounds</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3626</span>, <span class="refDoi"> DOI: 10.3390/ijms21103626</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.3390%2Fijms21103626" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvF2gs73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2020&pages=3626&author=A.+Fischerauthor=M.+Sellnerauthor=S.+Neranjanauthor=M.+Smie%C5%A1koauthor=M.+A.+Lill&title=Potential+inhibitors+for+novel+coronavirus+protease+identified+by+virtual+screening+of+606+million+compounds&doi=10.3390%2Fijms21103626"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Potential inhibitors for novel coronavirus protease identified by virtual screening of 606 million compounds</span></div><div class="casAuthors">Fischer, Andre; Sellner, Manuel; Neranjan, Santhosh; Smiesko, Martin; Lill, Markus A.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3626</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The rapid outbreak of the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in China followed by its spread around the world poses a serious global concern for public health.  To this date, no specific drugs or vaccines are available to treat SARS-CoV-2 despite its close relation to the SARS-CoV virus that caused a similar epidemic in 2003.  Thus, there remains an urgent need for the identification and development of specific antiviral therapeutics against SARS-CoV-2.  To conquer viral infections, the inhibition of proteases essential for proteolytic processing of viral polyproteins is a conventional therapeutic strategy.  In order to find novel inhibitors, we computationally screened a compd. library of over 606 million compds. for binding at the recently solved crystal structure of the main protease (Mpro) of SARS-CoV-2.  A screening of such a vast chem. space for SARS-CoV-2 Mpro inhibitors has not been reported before.  After shape screening, two docking protocols were applied followed by the detn. of mol. descriptors relevant for pharmacokinetics to narrow down the no. of initial hits.  Next, mol. dynamics simulations were conducted to validate the stability of docked binding modes and comprehensively quantify ligand binding energies.  After evaluation of potential off-target binding, we report a list of 12 purchasable compds., with binding affinity to the target protease that is predicted to be more favorable than that of the cocrystd. peptidomimetic compd.  In order to quickly advise ongoing therapeutic intervention for patients, we evaluated approved antiviral drugs and other protease inhibitors to provide a list of nine compds. for drug repurposing.  Furthermore, we identified the natural compds. (-)-taxifolin and rhamnetin as potential inhibitors of Mpro.  Rhamnetin is already com. available in pharmacies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpxRaxT4yRfrVg90H21EOLACvtfcHk0li9o-R94A10og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvF2gs73E&md5=dae8bf7445d10f43fe32b4fd409c4c9a</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.3390%2Fijms21103626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms21103626%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DA.%26aulast%3DSellner%26aufirst%3DM.%26aulast%3DNeranjan%26aufirst%3DS.%26aulast%3DSmie%25C5%25A1ko%26aufirst%3DM.%26aulast%3DLill%26aufirst%3DM.%2BA.%26atitle%3DPotential%2520inhibitors%2520for%2520novel%2520coronavirus%2520protease%2520identified%2520by%2520virtual%2520screening%2520of%2520606%2520million%2520compounds%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2020%26volume%3D21%26spage%3D3626%26doi%3D10.3390%2Fijms21103626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Broccatelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span> <span> </span><span class="NLM_article-title">Best of both worlds: On the complementarity of ligand-based and structure-based virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1634</span>– <span class="NLM_lpage">1641</span>, <span class="refDoi"> DOI: 10.1021/ci5001604</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci5001604" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXovVWhtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1634-1641&author=F.+Broccatelliauthor=N.+Brown&title=Best+of+both+worlds%3A+On+the+complementarity+of+ligand-based+and+structure-based+virtual+screening&doi=10.1021%2Fci5001604"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Best of Both Worlds: On the Complementarity of Ligand-Based and Structure-Based Virtual Screening</span></div><div class="casAuthors">Broccatelli, Fabio; Brown, Nathan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1634-1641</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual screening with docking is an integral component of drug design, particularly during hit finding phases.  While successful prospective studies of virtual screening exist, it remains a significant challenge to identify best practices a priori due to the many factors that influence the final outcome, including targets, data sets, software, metrics, and expert knowledge of the users.  This study investigates the extent to which ligand-based methods can be applied to improve structure-based methods.  The use of ligand-based methods to modulate the no. of hits identified using the protein-ligand complex and also the diversity of these hits from the crystallog. ligand is discussed.  In this study, 40 CDK2 ligand complexes were used together with two external data sets contg. both actives and inactives from GlaxoSmithKline (GSK) and actives and decoys from the Directory of Useful Decoys (DUD).  Results show how ligand-based modeling can be used to select a more appropriate protein conformation for docking, as well as to assess the reliability of the docking expt.  The time gained by reducing the pool of virtual screening candidates via ligand-based similarity can be invested in more accurate docking procedures, as well as in downstream labor-intensive approaches (e.g., visual inspection) maximizing the use of the chem. and biol. information available.  This provides a framework for mol. modeling scientists that are involved in initiating virtual screening campaigns with practical advice to make best use of the information available to them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmJshEz5T-P7Vg90H21EOLACvtfcHk0li9o-R94A10og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXovVWhtLY%253D&md5=d26bffbe2f144f37fd698c93c631cda8</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Fci5001604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci5001604%26sid%3Dliteratum%253Aachs%26aulast%3DBroccatelli%26aufirst%3DF.%26aulast%3DBrown%26aufirst%3DN.%26atitle%3DBest%2520of%2520both%2520worlds%253A%2520On%2520the%2520complementarity%2520of%2520ligand-based%2520and%2520structure-based%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1634%26epage%3D1641%26doi%3D10.1021%2Fci5001604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goh, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suen, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yau, M.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, K.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Repurposing registered drugs as antagonists for protease-activated receptor 2</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">2079</span>– <span class="NLM_lpage">2084</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00500</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00500" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1altbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2079-2084&author=W.+Xuauthor=J.+Limauthor=C.-Y.+Gohauthor=J.+Y.+Suenauthor=Y.+Jiangauthor=M.-K.+Yauauthor=K.-C.+Wuauthor=L.+Liuauthor=D.+P.+Fairlie&title=Repurposing+registered+drugs+as+antagonists+for+protease-activated+receptor+2&doi=10.1021%2Facs.jcim.5b00500"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Repurposing Registered Drugs as Antagonists for Protease-Activated Receptor 2</span></div><div class="casAuthors">Xu, Weijun; Lim, Junxian; Goh, Chai-Yeen; Suen, Jacky Y.; Jiang, Yuhong; Yau, Mei-Kwan; Wu, Kai-Chen; Liu, Ligong; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2079-2084</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual screening of a drug database identified Carvedilol, Loratadine, Nefazodone and Astemizole as PAR2 antagonists, after ligand docking and mol. dynamics simulations using a PAR2 homol. model and a putative binding mode of a known PAR2 ligand.  The drugs demonstrated competitive binding and antagonism of calcium mobilization and ERK1/2 phosphorylation in CHO-hPAR2 transfected cells, while inhibiting IL-6 secretion in PAR2 expressing MDA-MB-231 breast cancer cells.  This research highlights opportunities for GPCR hit-finding from FDA-approved drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCObXGReIXv7Vg90H21EOLACvtfcHk0li9o-R94A10og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1altbvN&md5=6091a430ca2bc69d0f478fb76c120b5e</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00500%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DW.%26aulast%3DLim%26aufirst%3DJ.%26aulast%3DGoh%26aufirst%3DC.-Y.%26aulast%3DSuen%26aufirst%3DJ.%2BY.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DYau%26aufirst%3DM.-K.%26aulast%3DWu%26aufirst%3DK.-C.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DRepurposing%2520registered%2520drugs%2520as%2520antagonists%2520for%2520protease-activated%2520receptor%25202%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2079%26epage%3D2084%26doi%3D10.1021%2Facs.jcim.5b00500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, W.</span></span> <span> </span><span class="NLM_article-title">Molecular modeling of the 3D structure of 5-HT1AR: Discovery of novel 5-HT1AR agonists via dynamic pharmacophore-based virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">3202</span>– <span class="NLM_lpage">3211</span>, <span class="refDoi"> DOI: 10.1021/ci400481p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400481p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslyks7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=3202-3211&author=L.+Xuauthor=S.+Zhouauthor=K.+Yuauthor=B.+Gaoauthor=H.+Jiangauthor=X.+Zhenauthor=W.+Fu&title=Molecular+modeling+of+the+3D+structure+of+5-HT1AR%3A+Discovery+of+novel+5-HT1AR+agonists+via+dynamic+pharmacophore-based+virtual+screening&doi=10.1021%2Fci400481p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Modeling of the 3D Structure of 5-HT1AR: Discovery of Novel 5-HT1AR Agonists via Dynamic Pharmacophore-Based Virtual Screening</span></div><div class="casAuthors">Xu, Lili; Zhou, Shanglin; Yu, Kunqian; Gao, Bo; Jiang, Hualiang; Zhen, Xuechu; Fu, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3202-3211</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The serotonin receptor subtype 1A (5-HT1AR) has been implicated in several neurol. conditions, and potent 5-HT1AR agonists have therapeutic potential for the treatment of depression, anxiety, schizophrenia, and Parkinson's disease.  In the present study, a homol. model of 5-HT1AR was built based on the latest released high-resoln. crystal structure of the β2AR in its active state (PDB: 3SN6).  A dynamic pharmacophore model, which takes the receptor flexibility into account, was constructed, validated, and applied to our dynamic pharmacophore-based virtual screening approach with the aim to identify potential 5-HT1AR agonists.  The obtained hits were subjected to 5-HT1AR binding and functional assays, and 10 compds. with medium or high Ki and EC50 values were identified.  Among them, FW01 (Ki = 51.9 nM, EC50 = 7 nM) was evaluated as the strongest agonist for 5-HT1AR.  The active 5-HT1AR model and dynamic pharmacophore model obtained from this study can be used for future discovery and design of novel 5-HT1AR agonists.  Also, by integrating all computational and available exptl. data, a stepwise 5-HT1AR signal transduction model induced by agonist FW01 was proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO3o2LACvakrVg90H21EOLACvtfcHk0lgj-C02M-W02w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslyks7bE&md5=6b2ec04d56b6c2e34e013ec295e66775</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fci400481p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400481p%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DGao%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DZhen%26aufirst%3DX.%26aulast%3DFu%26aufirst%3DW.%26atitle%3DMolecular%2520modeling%2520of%2520the%25203D%2520structure%2520of%25205-HT1AR%253A%2520Discovery%2520of%2520novel%25205-HT1AR%2520agonists%2520via%2520dynamic%2520pharmacophore-based%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D3202%26epage%3D3211%26doi%3D10.1021%2Fci400481p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rudling, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almlöf, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scobie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warpman
Berglund, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stenmark, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, J.</span></span> <span> </span><span class="NLM_article-title">Fragment-based discovery and optimization of enzyme inhibitors by docking of commercial chemical space</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8160</span>– <span class="NLM_lpage">8169</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01006</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01006" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeiurnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8160-8169&author=A.+Rudlingauthor=R.+Gustafssonauthor=I.+Alml%C3%B6fauthor=E.+Homanauthor=M.+Scobieauthor=U.+Warpman%0ABerglundauthor=T.+Helledayauthor=P.+Stenmarkauthor=J.+Carlsson&title=Fragment-based+discovery+and+optimization+of+enzyme+inhibitors+by+docking+of+commercial+chemical+space&doi=10.1021%2Facs.jmedchem.7b01006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery and Optimization of Enzyme Inhibitors by Docking of Commercial Chemical Space</span></div><div class="casAuthors">Rudling, Axel; Gustafsson, Robert; Almloef, Ingrid; Homan, Evert; Scobie, Martin; Warpman Berglund, Ulrika; Helleday, Thomas; Stenmark, Paal; Carlsson, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8160-8169</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based lead discovery has emerged as a leading drug development strategy for novel therapeutic targets.  Although fragment-based drug discovery benefits immensely from access to at.-resoln. information, structure-based virtual screening has rarely been used to drive fragment discovery and optimization.  Here, mol. docking of 0.3 million fragments to a crystal structure of cancer target MTH1 was performed.  Twenty-two predicted fragment ligands, for which analogs could be acquired com., were exptl. evaluated.  Five fragments inhibited MTH1 with IC50 values ranging from 6 to 79 μM.  Structure-based optimization guided by predicted binding modes and analogs from com. chem. libraries yielded nanomolar inhibitors.  Subsequently solved crystal structures confirmed binding modes predicted by docking for three scaffolds.  Structure-guided exploration of com. chem. space using mol. docking gives access to fragment libraries that are several orders of magnitude larger than those screened exptl. and can enable efficient optimization of hits to potent leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7SXpOWovFo7Vg90H21EOLACvtfcHk0lgj-C02M-W02w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeiurnF&md5=8858ee50eeb18fa0a397dec221a2effb</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01006%26sid%3Dliteratum%253Aachs%26aulast%3DRudling%26aufirst%3DA.%26aulast%3DGustafsson%26aufirst%3DR.%26aulast%3DAlml%25C3%25B6f%26aufirst%3DI.%26aulast%3DHoman%26aufirst%3DE.%26aulast%3DScobie%26aufirst%3DM.%26aulast%3DWarpman%2BBerglund%26aufirst%3DU.%26aulast%3DHelleday%26aufirst%3DT.%26aulast%3DStenmark%26aufirst%3DP.%26aulast%3DCarlsson%26aufirst%3DJ.%26atitle%3DFragment-based%2520discovery%2520and%2520optimization%2520of%2520enzyme%2520inhibitors%2520by%2520docking%2520of%2520commercial%2520chemical%2520space%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8160%26epage%3D8169%26doi%3D10.1021%2Facs.jmedchem.7b01006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">di
Pizio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laghezza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tortorella, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agamennone, M.</span></span> <span> </span><span class="NLM_article-title">Probing the S1’ site for the identification of non-zinc-binding MMP-2 inhibitors</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1475</span>– <span class="NLM_lpage">1482</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201300186</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1002%2Fcmdc.201300186" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=23873724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A280%3ADC%252BC3sfisVOitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1475-1482&author=A.+di%0APizioauthor=A.+Laghezzaauthor=P.+Tortorellaauthor=M.+Agamennone&title=Probing+the+S1%E2%80%99+site+for+the+identification+of+non-zinc-binding+MMP-2+inhibitors&doi=10.1002%2Fcmdc.201300186"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the S1' site for the identification of non-zinc-binding MMP-2 inhibitors</span></div><div class="casAuthors">Di Pizio Antonella; Laghezza Antonio; Tortorella Paolo; Agamennone Mariangela</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1475-82, 1421</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Matrix metalloproteinases (MMPs) are zinc-dependent enzymes involved in several pathological states.  Among them, MMP-2 is a relevant therapeutic target because of its role in cancer development and progression.  Many MMP inhibitors (MMPIs) have been discovered over the last 30 years, and the majority of them contain a functional group that binds the zinc ion (zinc-binding group; ZBG).  Unfortunately, no MMPIs have reached the market yet, owing to toxic effects due to unselective interactions of the ZBG.  The new generation of MMPIs that do not bind the zinc ion could overcome problems of selectivity and toxicity, but have so far been developed only for MMP-8, -12, and -13.  In this work, a virtual screening protocol was established by combining ligand- and structure-based methods to identify non-zinc-binding MMP-2 inhibitors using a new-generation MMP-8 inhibitor as a probe to find unexplored interactions in the MMP-2 S1' site.  The screening allowed the identification of micromolar MMP-2 inhibitors that putatively avoid binding the zinc ion, as demonstrated by docking calculations.  The LIA model, built to correlate predicted and experimental binding energies of the identified non-zinc-binding MMP-2 hits, underpins the reliability of the predicted docking poses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRctRUUsTkSrxJ5E29YNGvQfW6udTcc2eb4Kqx3aSQVI7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sfisVOitQ%253D%253D&md5=0adbedeb84637087ab42b20932278d49</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300186%26sid%3Dliteratum%253Aachs%26aulast%3Ddi%2BPizio%26aufirst%3DA.%26aulast%3DLaghezza%26aufirst%3DA.%26aulast%3DTortorella%26aufirst%3DP.%26aulast%3DAgamennone%26aufirst%3DM.%26atitle%3DProbing%2520the%2520S1%25E2%2580%2599%2520site%2520for%2520the%2520identification%2520of%2520non-zinc-binding%2520MMP-2%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26spage%3D1475%26epage%3D1482%26doi%3D10.1002%2Fcmdc.201300186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grebner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malmerberg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shewmaker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batista, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowski, J.</span></span> <span> </span><span class="NLM_article-title">Virtual screening in the cloud: How big is big enough?</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4274</span>– <span class="NLM_lpage">4282</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.9b00779</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.9b00779" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVyms77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=4274-4282&author=C.+Grebnerauthor=E.+Malmerbergauthor=A.+Shewmakerauthor=J.+Batistaauthor=A.+Nichollsauthor=J.+Sadowski&title=Virtual+screening+in+the+cloud%3A+How+big+is+big+enough%3F&doi=10.1021%2Facs.jcim.9b00779"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Screening in the Cloud: How Big Is Big Enough?</span></div><div class="casAuthors">Grebner, Christoph; Malmerberg, Erik; Shewmaker, Andrew; Batista, Jose; Nicholls, Anthony; Sadowski, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4274-4282</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual screening is a std. tool in Computer-Assisted Drug Design (CADD).  Early in a project, it is typical to use ligand-based similarity search methods to find suitable hit mols.  However, the no. of compds. which can be screened and the time required are usually limited by computational resources.  We describe here a high-throughput virtual screening project using 3D similarity (FastROCS) and automated evaluation workflows on Orion, a cloud computing platform.  Cloud resources make this approach fully scalable and flexible, allowing the generation and search of billions of virtual mols., and give access to an explicit 3D virtual chem. space not available before.  We discuss the impact of the size of the search space with respect to finding novel chem. hits and the size of the required hit list, as well as computational and economical aspects of resource scaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpycUgVx6GwYLVg90H21EOLACvtfcHk0lgenyzcMkm4YQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVyms77I&md5=5be8eab30a5532334c56a906e7ccf9cd</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.9b00779&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.9b00779%26sid%3Dliteratum%253Aachs%26aulast%3DGrebner%26aufirst%3DC.%26aulast%3DMalmerberg%26aufirst%3DE.%26aulast%3DShewmaker%26aufirst%3DA.%26aulast%3DBatista%26aufirst%3DJ.%26aulast%3DNicholls%26aufirst%3DA.%26aulast%3DSadowski%26aufirst%3DJ.%26atitle%3DVirtual%2520screening%2520in%2520the%2520cloud%253A%2520How%2520big%2520is%2520big%2520enough%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D4274%26epage%3D4282%26doi%3D10.1021%2Facs.jcim.9b00779" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walters, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span> <span> </span><span class="NLM_article-title">New trends in virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4109</span>– <span class="NLM_lpage">4111</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c01009</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c01009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFartL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=4109-4111&author=W.+P.+Waltersauthor=R.+Wang&title=New+trends+in+virtual+screening&doi=10.1021%2Facs.jcim.0c01009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">New Trends in Virtual Screening</span></div><div class="casAuthors">Walters, W. Patrick; Wang, Renxiao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4109-4111</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsyuDmwuVE97Vg90H21EOLACvtfcHk0lgenyzcMkm4YQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFartL3I&md5=09eadcc885e42415fb4594977450be17</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c01009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c01009%26sid%3Dliteratum%253Aachs%26aulast%3DWalters%26aufirst%3DW.%2BP.%26aulast%3DWang%26aufirst%3DR.%26atitle%3DNew%2520trends%2520in%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D4109%26epage%3D4111%26doi%3D10.1021%2Facs.jcim.0c01009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smieško, M.</span></span> <span> </span><span class="NLM_article-title">DOLINA - docking based on a local induced-fit algorithm: Application toward small-molecule binding to nuclear receptors</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1415</span>– <span class="NLM_lpage">1423</span>, <span class="refDoi"> DOI: 10.1021/ci400098y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400098y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosFygu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=1415-1423&author=M.+Smie%C5%A1ko&title=DOLINA+-+docking+based+on+a+local+induced-fit+algorithm%3A+Application+toward+small-molecule+binding+to+nuclear+receptors&doi=10.1021%2Fci400098y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">DOLINA - Docking Based on a Local Induced-Fit Algorithm: Application toward Small-Molecule Binding to Nuclear Receptors</span></div><div class="casAuthors">Smiesko, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1415-1423</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Docking algorithms allowing for ligand and - to various extent - also protein flexibility are nowadays replacing techniques based on rigid protocols.  The algorithm implemented in the Dolina software relies on pharmacophore matching for generating potential ligand poses and treats assocd. local induced-fit changes by combinatorial rearrangement of side-chains lining the binding site.  In Dolina, ligand flexibility is not treated internally, instead a pool of low-energy conformers identified in a conformational search is screened for extended binding-pose candidates.  Grouping rearranged residues in sterically independent families and side-chain conformer clustering are employed to achieve efficient use of the computational resources along with a good accuracy of the generated poses.  Dolina was applied toward docking of small-mol. ligands to three different nuclear receptor ligand binding domains for which in total 18 high-resoln. crystal structures were used as ref.  The selected nuclear receptors feature a deeply buried ligand-binding site where local induced-fit is to be expected, particularly for receptor antagonists.  For each receptor, a crystal structure with a cocrystd. small steroid ligand (template) was chosen as a target system, to which several synthetic ligands of different sizes were docked.  Poses within an RMSD of 2.0 Å from the crystal ref. pose were generated in 91% of the cases.  In 28%, the pose with the lowest RMSD to the ref. pose was ranked as the top one, and in 76% it was ranked among the top five poses.  Detailed descriptions of the docking algorithm and obsd. results are included.  Dolina is available free of charge for academic institutions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPgqbDVuxYKbVg90H21EOLACvtfcHk0lgenyzcMkm4YQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosFygu7c%253D&md5=f2d93edcbf7af9e7eb8001dacc32e970</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1021%2Fci400098y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400098y%26sid%3Dliteratum%253Aachs%26aulast%3DSmie%25C5%25A1ko%26aufirst%3DM.%26atitle%3DDOLINA%2520-%2520docking%2520based%2520on%2520a%2520local%2520induced-fit%2520algorithm%253A%2520Application%2520toward%2520small-molecule%2520binding%2520to%2520nuclear%2520receptors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D1415%26epage%3D1423%26doi%3D10.1021%2Fci400098y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lill, M. A.</span></span> <span> </span><span class="NLM_article-title">Efficient incorporation of protein flexibility and dynamics into molecular docking simulations</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">6157</span>– <span class="NLM_lpage">6169</span>, <span class="refDoi"> DOI: 10.1021/bi2004558</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi2004558" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvFSjurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=6157-6169&author=M.+A.+Lill&title=Efficient+incorporation+of+protein+flexibility+and+dynamics+into+molecular+docking+simulations&doi=10.1021%2Fbi2004558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Incorporation of Protein Flexibility and Dynamics into Molecular Docking Simulations</span></div><div class="casAuthors">Lill, Markus A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">6157-6169</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Flexibility and dynamics are protein characteristics that are essential for the process of mol. recognition.  Conformational changes in the protein that are coupled to ligand binding are described by the biophys. models of induced fit and conformational selection.  Different concepts that incorporate protein flexibility into protein-ligand docking within the context of these two models are reviewed.  Several computational studies that discuss the validity and possible limitations of such approaches will be presented.  Finally, different approaches that incorporate protein dynamics, e.g., configurational entropy, and solvation effects into docking will be highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmYHocKnP0SLVg90H21EOLACvtfcHk0ljIi_2S2cx39w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvFSjurc%253D&md5=a1cd2a2348f91c52e0ea508613288fcc</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1021%2Fbi2004558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi2004558%26sid%3Dliteratum%253Aachs%26aulast%3DLill%26aufirst%3DM.%2BA.%26atitle%3DEfficient%2520incorporation%2520of%2520protein%2520flexibility%2520and%2520dynamics%2520into%2520molecular%2520docking%2520simulations%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D6157%26epage%3D6169%26doi%3D10.1021%2Fbi2004558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Vivo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottegoni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span> <span> </span><span class="NLM_article-title">Role of molecular dynamics and related methods in drug discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4035</span>– <span class="NLM_lpage">4061</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01684</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01684" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlWksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4035-4061&author=M.+de%0AVivoauthor=M.+Masettiauthor=G.+Bottegoniauthor=A.+Cavalli&title=Role+of+molecular+dynamics+and+related+methods+in+drug+discovery&doi=10.1021%2Facs.jmedchem.5b01684"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Role of Molecular Dynamics and Related Methods in Drug Discovery</span></div><div class="casAuthors">De Vivo, Marco; Masetti, Matteo; Bottegoni, Giovanni; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4035-4061</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. dynamics (MD) and related methods are close to becoming routine computational tools for drug discovery.  Their main advantage is in explicitly treating structural flexibility and entropic effects.  This allows a more accurate est. of the thermodn. and kinetics assocd. with drug-target recognition and binding, as better algorithms and hardware architectures increase their use.  Here, we review the theor. background of MD and enhanced sampling methods, focusing on free-energy perturbation, metadynamics, steered MD, and other methods most consistently used to study drug-target binding.  We discuss unbiased MD simulations that nowadays allow the observation of unsupervised ligand-target binding, assessing how these approaches help optimizing target affinity and drug residence time toward improved drug efficacy.  Further issues discussed include allosteric modulation and the role of water mols. in ligand binding and optimization.  We conclude by calling for more prospective studies to attest to these methods' utility in discovering novel drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQb_02a7Nmh7Vg90H21EOLACvtfcHk0ljIi_2S2cx39w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlWksLs%253D&md5=b3d8a4fb5705a3555c2bb2a962420396</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01684%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BVivo%26aufirst%3DM.%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DRole%2520of%2520molecular%2520dynamics%2520and%2520related%2520methods%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4035%26epage%3D4061%26doi%3D10.1021%2Facs.jmedchem.5b01684" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guedes, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, F. S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dardenne, L. E.</span></span> <span> </span><span class="NLM_article-title">Empirical scoring functions for structure-based virtual screening: Applications, critical aspects, and challenges</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1089</span>, <span class="refDoi"> DOI: 10.3389/fphar.2018.01089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.3389%2Ffphar.2018.01089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=30319422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpt1WqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1089&author=I.+A.+Guedesauthor=F.+S.+S.+Pereiraauthor=L.+E.+Dardenne&title=Empirical+scoring+functions+for+structure-based+virtual+screening%3A+Applications%2C+critical+aspects%2C+and+challenges&doi=10.3389%2Ffphar.2018.01089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Empirical scoring functions for structure-based virtual screening: applications, critical aspects, and challenges</span></div><div class="casAuthors">Guedes, Isabella A.; Pereira, Felipe S. S.; Dardenne, Laurent E.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Pharmacology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1089</span>CODEN:
                <span class="NLM_cas:coden">FPRHAU</span>;
        ISSN:<span class="NLM_cas:issn">1663-9812</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">A review.  Structure-based virtual screening (VS) is a widely used approach that employs the knowledge of the three-dimensional structure of the target of interest in the design of new lead compds. from large-scale mol. docking expts.  Through the prediction of the binding mode and affinity of a small mol. within the binding site of the target of interest, it is possible to understand important properties related to the binding process.  Empirical scoring functions are widely used for pose and affinity prediction.  Although pose prediction is performed with satisfactory accuracy, the correct prediction of binding affinity is still a challenging task and crucial for the success of structure-based VS expts.  There are several efforts in distinct fronts to develop even more sophisticated and accurate models for filtering and ranking large libraries of compds.  This paper will cover some recent successful applications and methodol. advances, including strategies to explore the ligand entropy and solvent effects, training with sophisticated machine-learning techniques, and the use of quantum mechanics.  Particular emphasis will be given to the discussion of crit. aspects and further directions for the development of more accurate empirical scoring functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuNmYBXPKivLVg90H21EOLACvtfcHk0ljIi_2S2cx39w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpt1WqsLk%253D&md5=6fd60ccc4d6c2c85a206030023f7c0d1</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2018.01089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2018.01089%26sid%3Dliteratum%253Aachs%26aulast%3DGuedes%26aufirst%3DI.%2BA.%26aulast%3DPereira%26aufirst%3DF.%2BS.%2BS.%26aulast%3DDardenne%26aufirst%3DL.%2BE.%26atitle%3DEmpirical%2520scoring%2520functions%2520for%2520structure-based%2520virtual%2520screening%253A%2520Applications%252C%2520critical%2520aspects%252C%2520and%2520challenges%26jtitle%3DFront.%2520Pharmacol.%26date%3D2018%26volume%3D9%26spage%3D1089%26doi%3D10.3389%2Ffphar.2018.01089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grinter, S. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, X.</span></span> <span> </span><span class="NLM_article-title">Scoring functions and their evaluation methods for protein-ligand docking: Recent advances and future directions</span>. <i>Phys. Chem. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">12899</span>– <span class="NLM_lpage">12908</span>, <span class="refDoi"> DOI: 10.1039/c0cp00151a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1039%2Fc0cp00151a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=20730182" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1GjtLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=12899-12908&author=S.-Y.+Huangauthor=S.+Z.+Grinterauthor=X.+Zou&title=Scoring+functions+and+their+evaluation+methods+for+protein-ligand+docking%3A+Recent+advances+and+future+directions&doi=10.1039%2Fc0cp00151a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Scoring functions and their evaluation methods for protein-ligand docking: recent advances and future directions</span></div><div class="casAuthors">Huang, Sheng-You; Grinter, Sam Z.; Zou, Xiaoqin</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">12899-12908</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The scoring function is one of the most important components in structure-based drug design.  Despite considerable success, accurate and rapid prediction of protein-ligand interactions is still a challenge in mol. docking.  In this perspective, the authors have reviewed 3 basic types of scoring functions (force-field, empirical, and knowledge-based) and the consensus scoring technique that are used for protein-ligand docking.  The commonly-used assessment criteria and publicly available protein-ligand databases for performance evaluation of the scoring functions have also been presented and discussed.  The authors end with a discussion of the challenges faced by existing scoring functions and possible future directions for developing improved scoring functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVhIxJscZuCbVg90H21EOLACvtfcHk0ljIi_2S2cx39w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1GjtLbL&md5=ebb7013a3a2c8f5d190df03329d8d8cd</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1039%2Fc0cp00151a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0cp00151a%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DS.-Y.%26aulast%3DGrinter%26aufirst%3DS.%2BZ.%26aulast%3DZou%26aufirst%3DX.%26atitle%3DScoring%2520functions%2520and%2520their%2520evaluation%2520methods%2520for%2520protein-ligand%2520docking%253A%2520Recent%2520advances%2520and%2520future%2520directions%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2010%26volume%3D12%26spage%3D12899%26epage%3D12908%26doi%3D10.1039%2Fc0cp00151a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span> <span> </span><span class="NLM_article-title">An overview of scoring functions used for protein-ligand interactions in molecular docking</span>. <i>Interdiscip. Sci.: Comput. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">320</span>– <span class="NLM_lpage">328</span>, <span class="refDoi"> DOI: 10.1007/s12539-019-00327-w</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1007%2Fs12539-019-00327-w" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVKgur7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=320-328&author=J.+Liauthor=A.+Fuauthor=L.+Zhang&title=An+overview+of+scoring+functions+used+for+protein-ligand+interactions+in+molecular+docking&doi=10.1007%2Fs12539-019-00327-w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">An Overview of Scoring Functions Used for Protein-Ligand Interactions in Molecular Docking</span></div><div class="casAuthors">Li, Jin; Fu, Ailing; Zhang, Le</div><div class="citationInfo"><span class="NLM_cas:title">Interdisciplinary Sciences: Computational Life Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">320-328</span>CODEN:
                <span class="NLM_cas:coden">ISCLAL</span>;
        ISSN:<span class="NLM_cas:issn">1867-1462</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  Currently, mol. docking is becoming a key tool in drug discovery and mol. modeling applications.  The reliability of mol. docking depends on the accuracy of the adopted scoring function, which can guide and det. the ligand poses when thousands of possible poses of ligand are generated.  The scoring function can be used to det. the binding mode and site of a ligand, predict binding affinity and identify the potential drug leads for a given protein target.  Despite intensive research over the years, accurate and rapid prediction of protein-ligand interactions is still a challenge in mol. docking.  For this reason, this study reviews four basic types of scoring functions, physics-based, empirical, knowledge-based, and machine learning-based scoring functions, based on an up-to-date classification scheme.  We not only discuss the foundations of the four types scoring functions, suitable application areas and shortcomings, but also discuss challenges and potential future study directions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPLeIP3v1p0LVg90H21EOLACvtfcHk0lgLIfvPqt46sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVKgur7O&md5=32d69a1e5260384d3d44d8d1e846c715</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1007%2Fs12539-019-00327-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12539-019-00327-w%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DL.%26atitle%3DAn%2520overview%2520of%2520scoring%2520functions%2520used%2520for%2520protein-ligand%2520interactions%2520in%2520molecular%2520docking%26jtitle%3DInterdiscip.%2520Sci.%253A%2520Comput.%2520Life%2520Sci.%26date%3D2019%26volume%3D11%26spage%3D320%26epage%3D328%26doi%3D10.1007%2Fs12539-019-00327-w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, T.</span></span> <span> </span><span class="NLM_article-title">Comprehensive evaluation of ten docking programs on a diverse set of protein-ligand complexes: The prediction accuracy of sampling power and scoring power</span>. <i>Phys. Chem. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">12964</span>– <span class="NLM_lpage">12975</span>, <span class="refDoi"> DOI: 10.1039/C6CP01555G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1039%2FC6CP01555G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=27108770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvFOns7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2016&pages=12964-12975&author=Z.+Wangauthor=H.+Sunauthor=X.+Yaoauthor=D.+Liauthor=L.+Xuauthor=Y.+Liauthor=S.+Tianauthor=T.+Hou&title=Comprehensive+evaluation+of+ten+docking+programs+on+a+diverse+set+of+protein-ligand+complexes%3A+The+prediction+accuracy+of+sampling+power+and+scoring+power&doi=10.1039%2FC6CP01555G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive evaluation of ten docking programs on a diverse set of protein-ligand complexes: the prediction accuracy of sampling power and scoring power</span></div><div class="casAuthors">Wang, Zhe; Sun, Huiyong; Yao, Xiaojun; Li, Dan; Xu, Lei; Li, Youyong; Tian, Sheng; Hou, Tingjun</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">12964-12975</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">As one of the most popular computational approaches in modern structure-based drug design, mol. docking can be used not only to identify the correct conformation of a ligand within the target binding pocket but also to est. the strength of the interaction between a target and a ligand.  Nowadays, as a variety of docking programs are available for the scientific community, a comprehensive understanding of the advantages and limitations of each docking program is fundamentally important to conduct more reasonable docking studies and docking-based virtual screening.  In the present study, based on an extensive dataset of 2002 protein-ligand complexes from the PDBbind database (version 2014), the performance of ten docking programs, including five com. programs (LigandFit, Glide, GOLD, MOE Dock, and Surflex-Dock) and five academic programs (AutoDock, AutoDock Vina, LeDock, rDock, and UCSF DOCK), was systematically evaluated by examg. the accuracies of binding pose prediction (sampling power) and binding affinity estn. (scoring power).  Our results showed that GOLD and LeDock had the best sampling power (GOLD: 59.8% accuracy for the top scored poses; LeDock: 80.8% accuracy for the best poses) and AutoDock Vina had the best scoring power (rp/rs of 0.564/0.580 and 0.569/0.584 for the top scored poses and best poses), suggesting that the com. programs did not show the expected better performance than the academic ones.  Overall, the ligand binding poses could be identified in most cases by the evaluated docking programs but the ranks of the binding affinities for the entire dataset could not be well predicted by most docking programs.  However, for some types of protein families, relatively high linear correlations between docking scores and exptl. binding affinities could be achieved.  To our knowledge, this study has been the most extensive evaluation of popular mol. docking programs in the last five years.  It is expected that our work can offer useful information for the successful application of these docking tools to different requirements and targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7PQalJNVCubVg90H21EOLACvtfcHk0lgLIfvPqt46sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvFOns7g%253D&md5=5bedadbb6b4eb4dafa95a48701ac720d</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1039%2FC6CP01555G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6CP01555G%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DYao%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DTian%26aufirst%3DS.%26aulast%3DHou%26aufirst%3DT.%26atitle%3DComprehensive%2520evaluation%2520of%2520ten%2520docking%2520programs%2520on%2520a%2520diverse%2520set%2520of%2520protein-ligand%2520complexes%253A%2520The%2520prediction%2520accuracy%2520of%2520sampling%2520power%2520and%2520scoring%2520power%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2016%26volume%3D18%26spage%3D12964%26epage%3D12975%26doi%3D10.1039%2FC6CP01555G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salmaso, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moro, S.</span></span> <span> </span><span class="NLM_article-title">Bridging molecular docking to molecular dynamics in exploring ligand-protein recognition process: An overview</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">923</span>, <span class="refDoi"> DOI: 10.3389/fphar.2018.00923</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.3389%2Ffphar.2018.00923" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=30186166" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtFyltbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=923&author=V.+Salmasoauthor=S.+Moro&title=Bridging+molecular+docking+to+molecular+dynamics+in+exploring+ligand-protein+recognition+process%3A+An+overview&doi=10.3389%2Ffphar.2018.00923"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Bridging molecular docking to molecular dynamics in exploring ligand-protein recognition process: an overview</span></div><div class="casAuthors">Salmaso, Veronica; Moro, Stefano</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Pharmacology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">923/1-923/16</span>CODEN:
                <span class="NLM_cas:coden">FPRHAU</span>;
        ISSN:<span class="NLM_cas:issn">1663-9812</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Computational techniques have been applied in the drug discovery pipeline since the 1980s.  Given the low computational resources of the time, the first mol. modeling strategies relied on a rigid view of the ligand-target binding process.  During the years, the evolution of hardware technologies has gradually allowed simulating the dynamic nature of the binding event.  In this work, we present an overview of the evolution of structure- based drug discovery techniques in the study of ligand-target recognition phenomenon, going fromthe staticmol. docking toward enhancedmol. dynamics strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-35hlGTXB6rVg90H21EOLACvtfcHk0liiy0bVbsFDcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtFyltbw%253D&md5=cd279290cc97675187703a45611dc24f</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2018.00923&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2018.00923%26sid%3Dliteratum%253Aachs%26aulast%3DSalmaso%26aufirst%3DV.%26aulast%3DMoro%26aufirst%3DS.%26atitle%3DBridging%2520molecular%2520docking%2520to%2520molecular%2520dynamics%2520in%2520exploring%2520ligand-protein%2520recognition%2520process%253A%2520An%2520overview%26jtitle%3DFront.%2520Pharmacol.%26date%3D2018%26volume%3D9%26spage%3D923%26doi%3D10.3389%2Ffphar.2018.00923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maffucci, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fumagalli, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Contini, A.</span></span> <span> </span><span class="NLM_article-title">An efficient implementation of the Nwat-MMGBSA method to rescore docking results in medium-throughput virtual screenings</span>. <i>Front. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">43</span>, <span class="refDoi"> DOI: 10.3389/fchem.2018.00043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.3389%2Ffchem.2018.00043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=29556494" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1egsbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=43&author=I.+Maffucciauthor=X.+Huauthor=V.+Fumagalliauthor=A.+Contini&title=An+efficient+implementation+of+the+Nwat-MMGBSA+method+to+rescore+docking+results+in+medium-throughput+virtual+screenings&doi=10.3389%2Ffchem.2018.00043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">An efficient implementation of the nwat-mmgbsa method to rescore docking results in medium-throughput virtual screenings</span></div><div class="casAuthors">Maffucci, Irene; Hu, Xiao; Fumagalli, Valentina; Contini, Alessandro</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Chemistry (Lausanne, Switzerland)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43/1-43/14</span>CODEN:
                <span class="NLM_cas:coden">FCLSAA</span>;
        ISSN:<span class="NLM_cas:issn">2296-2646</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Nwat-MMGBSA is a variant of MM-PB/GBSA based on the inclusion of a no. of explicit water mols. that are the closest to the ligand in each frame of a mol. dynamics trajectory.  This method demonstrated improved correlations between calcd. and exptl. binding energies in both protein-protein interactions and ligand-receptor complexes, in comparison to the std. MM-GBSA.  A protocol optimization, aimed to maximize efficacy and efficiency, is discussed here considering penicillopepsin, HIV1-protease, and BCL-XL as test cases.  Calcns. were performed in triplicates on both classic HPC environments and on std. workstations equipped by a GPU card, evidencing no statistical differences in the results.  No relevant differences in correlation to expts. were also obsd. when performing Nwat-MMGBSA calcns. on 4 or 1 ns long trajectories.  A fully automatic workflow for structure-based virtual screening, performing from library set-up to docking and Nwat-MMGBSA rescoring, has then been developed.  The protocol has been tested against no rescoring or std. MM-GBSA rescoring within a retrospective virtual screening of inhibitors of AmpC β-lactamase and of the Rac1-Tiam1 protein-protein interaction.  In both cases, Nwat-MMGBSA rescoring provided a statistically significant increase in the ROC AUCs of between 20 and 30%, compared to docking scoring or to std. MM-GBSA rescoring.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOtEwZRkQ6BLVg90H21EOLACvtfcHk0liiy0bVbsFDcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1egsbrN&md5=b2925a1d0f9323f99cb4c93af53ced04</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.3389%2Ffchem.2018.00043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffchem.2018.00043%26sid%3Dliteratum%253Aachs%26aulast%3DMaffucci%26aufirst%3DI.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DFumagalli%26aufirst%3DV.%26aulast%3DContini%26aufirst%3DA.%26atitle%3DAn%2520efficient%2520implementation%2520of%2520the%2520Nwat-MMGBSA%2520method%2520to%2520rescore%2520docking%2520results%2520in%2520medium-throughput%2520virtual%2520screenings%26jtitle%3DFront.%2520Chem.%26date%3D2018%26volume%3D6%26spage%3D43%26doi%3D10.3389%2Ffchem.2018.00043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinzi, L.</span></span> <span> </span><span class="NLM_article-title">Refinement and rescoring of virtual screening results</span>. <i>Front. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">498</span>, <span class="refDoi"> DOI: 10.3389/fchem.2019.00498</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.3389%2Ffchem.2019.00498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=31355188" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFGgt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2019&pages=498&author=G.+Rastelliauthor=L.+Pinzi&title=Refinement+and+rescoring+of+virtual+screening+results&doi=10.3389%2Ffchem.2019.00498"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement and rescoring of virtual screening results</span></div><div class="casAuthors">Rastelli, Giulio; Pinzi, Luca</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Chemistry (Lausanne, Switzerland)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">498</span>CODEN:
                <span class="NLM_cas:coden">FCLSAA</span>;
        ISSN:<span class="NLM_cas:issn">2296-2646</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">High-throughput docking is an established computational screening approach in drug design.  This methodol. enables a rapid identification of biol. active hit compds., providing an efficient and cost-effective complement or alternative to exptl. high-throughput screenings.  However, limitations inherent to the methodol. make docking results inevitably approx.  Two major Achille's heels include the use of approximated scoring functions and the limited sampling of the ligand-target complexes.  Therefore, docking results require careful evaluation and further post-docking analyses.  In this article, we will overview our approach to post-docking anal. in virtual screenings.  BEAR (Binding Estn. After Refinement) was developed as a post-docking processing tool that refines docking poses by means of mol. dynamics (MD) and then rescores the ligands based on more accurate scoring functions (MM-PB(GB)SA).  The tool has been validated and used prospectively in drug discovery applications.  Future directions regarding refinement and rescoring in virtual screening are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA7hkak3owGLVg90H21EOLACvtfcHk0liiy0bVbsFDcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFGgt7w%253D&md5=64ccae09466a5aaf9df144137be1fa92</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.3389%2Ffchem.2019.00498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffchem.2019.00498%26sid%3Dliteratum%253Aachs%26aulast%3DRastelli%26aufirst%3DG.%26aulast%3DPinzi%26aufirst%3DL.%26atitle%3DRefinement%2520and%2520rescoring%2520of%2520virtual%2520screening%2520results%26jtitle%3DFront.%2520Chem.%26date%3D2019%26volume%3D7%26spage%3D498%26doi%3D10.3389%2Ffchem.2019.00498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lightstone, F. C.</span></span> <span> </span><span class="NLM_article-title">Toward fully automated high performance computing drug discovery: A massively parallel virtual screening pipeline for docking and molecular mechanics/generalized born surface area rescoring to improve enrichment</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">324</span>– <span class="NLM_lpage">337</span>, <span class="refDoi"> DOI: 10.1021/ci4005145</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci4005145" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOiurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=324-337&author=X.+Zhangauthor=S.+E.+Wongauthor=F.+C.+Lightstone&title=Toward+fully+automated+high+performance+computing+drug+discovery%3A+A+massively+parallel+virtual+screening+pipeline+for+docking+and+molecular+mechanics%2Fgeneralized+born+surface+area+rescoring+to+improve+enrichment&doi=10.1021%2Fci4005145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Toward Fully Automated High Performance Computing Drug Discovery: A Massively Parallel Virtual Screening Pipeline for Docking and Molecular Mechanics/Generalized Born Surface Area Rescoring to Improve Enrichment</span></div><div class="casAuthors">Zhang, Xiaohua; Wong, Sergio E.; Lightstone, Felice C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">324-337</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors announce and evaluate a high throughput virtual screening pipeline for in-silico screening of virtual compd. databases using high performance computing (HPC).  Notable features of this pipeline are an automated receptor prepn. scheme with unsupervised binding site identification.  The pipeline includes receptor/target prepn., ligand prepn., VinaLC docking calcn., and mol. mechanics/generalized Born surface area (MM/GBSA) rescoring using the GB model by Onufriev and co-workers J. Chem. Theory Comput.2007, 3, 156-169.  Furthermore, the authors leverage HPC resources to perform an unprecedented, comprehensive evaluation of MM/GBSA rescoring when applied to the DUD-E data set (Directory of Useful Decoys: Enhanced), in which the authors selected 38 protein targets and a total of ∼0.7 million actives and decoys.  The computer wall time for virtual screening has been reduced drastically on HPC machines, which increases the feasibility of extremely large ligand database screening with more accurate methods.  HPC resources allowed the authors to rescore 20 poses per compd. and evaluate the optimal no. of poses to rescore.  The authors find that keeping 5-10 poses is a good compromise between accuracy and computational expense.  Overall the results demonstrate that MM/GBSA rescoring has higher av. receiver operating characteristic (ROC) area under curve (AUC) values and consistently better early recovery of actives than Vina docking alone.  Specifically, the enrichment performance is target-dependent.  MM/GBSA rescoring significantly out performs Vina docking for the folate enzymes, kinases, and several other enzymes.  The more accurate energy function and solvation terms of the MM/GBSA method allow MM/GBSA to achieve better enrichment, but the rescoring is still limited by the docking method to generate the poses with the correct binding modes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXbi5hOh_RRLVg90H21EOLACvtfcHk0lj9tGuS8BjS1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOiurfN&md5=37b4dbc7c437fc9da70cdf41689d00b2</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1021%2Fci4005145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci4005145%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWong%26aufirst%3DS.%2BE.%26aulast%3DLightstone%26aufirst%3DF.%2BC.%26atitle%3DToward%2520fully%2520automated%2520high%2520performance%2520computing%2520drug%2520discovery%253A%2520A%2520massively%2520parallel%2520virtual%2520screening%2520pipeline%2520for%2520docking%2520and%2520molecular%2520mechanics%252Fgeneralized%2520born%2520surface%2520area%2520rescoring%2520to%2520improve%2520enrichment%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D324%26epage%3D337%26doi%3D10.1021%2Fci4005145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Virtanen, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niinivehmas, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pentikäinen, O. T.</span></span> <span> </span><span class="NLM_article-title">Case-specific performance of MM-PBSA, MM-GBSA, and SIE in virtual screening</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">303</span>– <span class="NLM_lpage">318</span>, <span class="refDoi"> DOI: 10.1016/j.jmgm.2015.10.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1016%2Fj.jmgm.2015.10.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=26550792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsl2gsLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2015&pages=303-318&author=S.+I.+Virtanenauthor=S.+P.+Niinivehmasauthor=O.+T.+Pentik%C3%A4inen&title=Case-specific+performance+of+MM-PBSA%2C+MM-GBSA%2C+and+SIE+in+virtual+screening&doi=10.1016%2Fj.jmgm.2015.10.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Case-specific performance of MM-PBSA, MM-GBSA, and SIE in virtual screening</span></div><div class="casAuthors">Virtanen, Salla I.; Niinivehmas, Sanna P.; Pentikainen, Olli T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">303-318</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">In drug discovery the reliable prediction of binding free energies is of crucial importance.  Methods that combine mol. mechanics force fields with continuum solvent models have become popular because of their high accuracy and relatively good computational efficiency.  In this research we studied the performance of mol. mechanics generalized Born surface area (MM-GBSA), mol. mechanics Poisson-Boltzmann surface area (MM-PBSA), and solvated interaction energy (SIE) both in their virtual screening efficiency and their ability to predict exptl. detd. binding affinities for five different protein targets.  The protein-ligand complexes were derived with two different approaches important in virtual screening: mol. docking and ligand-based similarity search methods.  The results show significant differences between the different binding energy calcn. methods.  However, the length of the mol. dynamics simulation was not of crucial importance for accuracy of results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzdqc_eszDjbVg90H21EOLACvtfcHk0lj9tGuS8BjS1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsl2gsLnF&md5=a1e470a099904f52e26a4f8e2cf91d55</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2015.10.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2015.10.012%26sid%3Dliteratum%253Aachs%26aulast%3DVirtanen%26aufirst%3DS.%2BI.%26aulast%3DNiinivehmas%26aufirst%3DS.%2BP.%26aulast%3DPentik%25C3%25A4inen%26aufirst%3DO.%2BT.%26atitle%3DCase-specific%2520performance%2520of%2520MM-PBSA%252C%2520MM-GBSA%252C%2520and%2520SIE%2520in%2520virtual%2520screening%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2015%26volume%3D62%26spage%3D303%26epage%3D318%26doi%3D10.1016%2Fj.jmgm.2015.10.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, T.</span></span> <span> </span><span class="NLM_article-title">End-Point binding free energy calculation with MM/PBSA and MM/GBSA: Strategies and applications in drug design</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">9478</span>– <span class="NLM_lpage">9508</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.9b00055</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.9b00055" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1ajs7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2019&pages=9478-9508&author=E.+Wangauthor=H.+Sunauthor=J.+Wangauthor=Z.+Wangauthor=H.+Liuauthor=J.+Z.+Zhangauthor=T.+Hou&title=End-Point+binding+free+energy+calculation+with+MM%2FPBSA+and+MM%2FGBSA%3A+Strategies+and+applications+in+drug+design&doi=10.1021%2Facs.chemrev.9b00055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">End-point binding free energy calculation with MM/PBSA and MM/GBSA: strategies and applications in drug design</span></div><div class="casAuthors">Wang, Ercheng; Sun, Huiyong; Wang, Junmei; Wang, Zhe; Liu, Hui; Zhang, John Z. H.; Hou, Tingjun</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">9478-9508</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Mol. mechanics Poisson-Boltzmann surface area (MM/PBSA) and mol. mechanics generalized Born surface area (MM/GBSA) are arguably very popular methods for binding free energy prediction since they are more accurate than most scoring functions of mol. docking and less computationally demanding than alchem. free energy methods.  MM/PBSA and MM/GBSA have been widely used in biomol. studies such as protein folding, protein-ligand binding, protein-protein interaction, etc.  In this review, methods to adjust the polar solvation energy and to improve the performance of MM/PBSA and MM/GBSA calcns. are reviewed and discussed.  The latest applications of MM/GBSA and MM/PBSA in drug design are also presented.  This review intends to provide readers with guidance for practically applying MM/PBSA and MM/GBSA in drug design and related research fields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu30aDvikEv7Vg90H21EOLACvtfcHk0lj9tGuS8BjS1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1ajs7%252FF&md5=eacd5dbfe659d27df5d103caec9e1bb5</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.9b00055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.9b00055%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DE.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DJ.%2BZ.%26aulast%3DHou%26aufirst%3DT.%26atitle%3DEnd-Point%2520binding%2520free%2520energy%2520calculation%2520with%2520MM%252FPBSA%2520and%2520MM%252FGBSA%253A%2520Strategies%2520and%2520applications%2520in%2520drug%2520design%26jtitle%3DChem.%2520Rev.%26date%3D2019%26volume%3D119%26spage%3D9478%26epage%3D9508%26doi%3D10.1021%2Facs.chemrev.9b00055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albanese, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chodera, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkamer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span> <span> </span><span class="NLM_article-title">Is structure-based drug design ready for selectivity optimization?</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">6211</span>– <span class="NLM_lpage">6227</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c00815</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c00815" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFygs7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=6211-6227&author=S.+K.+Albaneseauthor=J.+D.+Choderaauthor=A.+Volkamerauthor=S.+Kengauthor=R.+Abelauthor=L.+Wang&title=Is+structure-based+drug+design+ready+for+selectivity+optimization%3F&doi=10.1021%2Facs.jcim.0c00815"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Is Structure-Based Drug Design Ready for Selectivity Optimization?</span></div><div class="casAuthors">Albanese, Steven K.; Chodera, John D.; Volkamer, Andrea; Keng, Simon; Abel, Robert; Wang, Lingle</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6211-6227</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Alchem. free-energy calcns. are now widely used to drive or maintain potency in small-mol. lead optimization with a roughly 1 kcal/mol accuracy.  Despite this, the potential to use free-energy calcns. to drive optimization of compd. selectivity among two similar targets has been relatively unexplored in published studies.  In the most optimistic scenario, the similarity of binding sites might lead to a fortuitous cancellation of errors and allow selectivity to be predicted more accurately than affinity.  Here, we assess the accuracy with which selectivity can be predicted in the context of small-mol. kinase inhibitors, considering the very similar binding sites of human kinases CDK2 and CDK9 as well as another series of ligands attempting to achieve selectivity between the more distantly related kinases CDK2 and ERK2.  Using a Bayesian anal. approach, we sep. systematic from statistical errors and quantify the correlation in systematic errors between selectivity targets.  We find that, in the CDK2/CDK9 case, a high correlation in systematic errors suggests that free-energy calcns. can have significant impact in aiding chemists in achieving selectivity, while in more distantly related kinases (CDK2/ERK2), the correlation in systematic error suggests that fortuitous cancellation may even occur between systems that are not as closely related.  In both cases, the correlation in systematic error suggests that longer simulations are beneficial to properly balance statistical error with systematic error to take full advantage of the increase in apparent free-energy calcn. accuracy in selectivity prediction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrITuv90vd8L7Vg90H21EOLACvtfcHk0lg0q6-ouANZMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFygs7bJ&md5=08cfdf031b626801b7ba762a559d750f</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c00815&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c00815%26sid%3Dliteratum%253Aachs%26aulast%3DAlbanese%26aufirst%3DS.%2BK.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DVolkamer%26aufirst%3DA.%26aulast%3DKeng%26aufirst%3DS.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DL.%26atitle%3DIs%2520structure-based%2520drug%2520design%2520ready%2520for%2520selectivity%2520optimization%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D6211%26epage%3D6227%26doi%3D10.1021%2Facs.jcim.0c00815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghanbarpour, A.</span>; <span class="NLM_string-name">Mahmoud, A. H.</span>; <span class="NLM_string-name">Lill, M. A.</span></span> <span> </span><span class="NLM_article-title">On-the-fly prediction of protein hydration densities and free energies using deep learning</span>.  <i>arXiv</i> <span class="NLM_year" style="font-weight: bold;">2020</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=A.+Ghanbarpour&author=A.+H.+Mahmoud&author=M.+A.+Lill&title=On-the-fly+prediction+of+protein+hydration+densities+and+free+energies+using+deep+learning"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGhanbarpour%26aufirst%3DA.%26atitle%3DOn-the-fly%2520prediction%2520of%2520protein%2520hydration%2520densities%2520and%2520free%2520energies%2520using%2520deep%2520learning%26jtitle%3DarXiv%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Djenno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insua, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pho, A.</span></span> <span> </span><span class="NLM_article-title">From paper to pixels: Using Google Forms for collaboration and assessment</span>. <i>Library hi tech news</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1108/LHTN-12-2014-0105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1108%2FLHTN-12-2014-0105" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2015&pages=9-13&author=M.+Djennoauthor=G.+Insuaauthor=A.+Pho&title=From+paper+to+pixels%3A+Using+Google+Forms+for+collaboration+and+assessment&doi=10.1108%2FLHTN-12-2014-0105"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1108%2FLHTN-12-2014-0105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1108%252FLHTN-12-2014-0105%26sid%3Dliteratum%253Aachs%26aulast%3DDjenno%26aufirst%3DM.%26aulast%3DInsua%26aufirst%3DG.%26aulast%3DPho%26aufirst%3DA.%26atitle%3DFrom%2520paper%2520to%2520pixels%253A%2520Using%2520Google%2520Forms%2520for%2520collaboration%2520and%2520assessment%26jtitle%3DLibrary%2520hi%2520tech%2520news%26date%3D2015%26volume%3D32%26spage%3D9%26epage%3D13%26doi%3D10.1108%2FLHTN-12-2014-0105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottegoni, G.</span></span> <span> </span><span class="NLM_article-title">Recipes for the selection of experimental protein conformations for virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">186</span>– <span class="NLM_lpage">193</span>, <span class="refDoi"> DOI: 10.1021/ci9003943</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci9003943" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFCrsb3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=186-193&author=R.+Abagyanauthor=M.+Ruedaauthor=G.+Bottegoni&title=Recipes+for+the+selection+of+experimental+protein+conformations+for+virtual+screening&doi=10.1021%2Fci9003943"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Recipes for the Selection of Experimental Protein Conformations for Virtual Screening</span></div><div class="casAuthors">Rueda, Manuel; Bottegoni, Giovanni; Abagyan, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">186-193</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The use of multiple x-ray protein structures has been reported to be an efficient alternative for the representation of the binding pocket flexibility needed for accurate small mols. docking.  However, the docking performance of the individual single conformations varies widely, and adding certain conformations to an ensemble is even counterproductive.  Here the authors used a very large and diverse benchmark of 1068 x-ray protein conformations of 99 therapeutically relevant proteins, first, to compare the performance of the ensemble and single-conformation docking and, second, to find the properties of the best-performing conformers that can be used to select a smaller set of conformers for ensemble docking.  The conformer selection has been validated through retrospective virtual screening expts. aimed at sepg. known ligand binders from decoys.  The authors found that the conformers cocrystd. with the largest ligands displayed high selectivity for binders, and when combined in ensembles they consistently provided better results than randomly chosen protein conformations.  The use of ensembles encompassing between 3 and 5 exptl. conformations consistently improved the docking accuracy and binders vs. decoys sepn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBgQPiTcnZy7Vg90H21EOLACvtfcHk0lg0q6-ouANZMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFCrsb3I&md5=159915d0fb8aa304a49592a0bbb6401f</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1021%2Fci9003943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9003943%26sid%3Dliteratum%253Aachs%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DRueda%26aufirst%3DM.%26aulast%3DBottegoni%26aufirst%3DG.%26atitle%3DRecipes%2520for%2520the%2520selection%2520of%2520experimental%2520protein%2520conformations%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D186%26epage%3D193%26doi%3D10.1021%2Fci9003943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalkas, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vlachakis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsagkrasoulis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kastania, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kossida, S.</span></span> <span> </span><span class="NLM_article-title">State-of-the-art technology in modern computer-aided drug design</span>. <i>Briefings Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">745</span>– <span class="NLM_lpage">752</span>, <span class="refDoi"> DOI: 10.1093/bib/bbs063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1093%2Fbib%2Fbbs063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=23148324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A280%3ADC%252BC3s7jtVamtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=745-752&author=G.+A.+Dalkasauthor=D.+Vlachakisauthor=D.+Tsagkrasoulisauthor=A.+Kastaniaauthor=S.+Kossida&title=State-of-the-art+technology+in+modern+computer-aided+drug+design&doi=10.1093%2Fbib%2Fbbs063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">State-of-the-art technology in modern computer-aided drug design</span></div><div class="casAuthors">Dalkas Georgios A; Vlachakis Dimitrios; Tsagkrasoulis Dimosthenis; Kastania Anastasia; Kossida Sophia</div><div class="citationInfo"><span class="NLM_cas:title">Briefings in bioinformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">745-52</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The quest for small drug-like compounds that selectively inhibit the function of biological targets has always been a major focus in the pharmaceutical industry and in academia as well.  High-throughput screening of compound libraries requires time, cost and resources.  Therefore, the use of alternative methods is necessary for facilitating lead discovery.  Computational techniques that dock small molecules into macromolecular targets and predict the affinity and activity of the small molecule are widely used in drug design and discovery, and have become an integral part of the industrial and academic research.  In this review, we present an overview of some state-of-the-art technologies in modern drug design that have been developed for expediting the search for novel drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRqxJLNY1TepZ6KlWarZTAzfW6udTcc2eZnhe7_XBuq9bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s7jtVamtQ%253D%253D&md5=ae0993a629d24a07cc16be8446902913</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1093%2Fbib%2Fbbs063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbib%252Fbbs063%26sid%3Dliteratum%253Aachs%26aulast%3DDalkas%26aufirst%3DG.%2BA.%26aulast%3DVlachakis%26aufirst%3DD.%26aulast%3DTsagkrasoulis%26aufirst%3DD.%26aulast%3DKastania%26aufirst%3DA.%26aulast%3DKossida%26aufirst%3DS.%26atitle%3DState-of-the-art%2520technology%2520in%2520modern%2520computer-aided%2520drug%2520design%26jtitle%3DBriefings%2520Bioinf.%26date%3D2013%26volume%3D14%26spage%3D745%26epage%3D752%26doi%3D10.1093%2Fbib%2Fbbs063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sevrioukova, I. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulos, T. L.</span></span> <span> </span><span class="NLM_article-title">Pyridine-substituted desoxyritonavir is a more potent inhibitor of cytochrome P450 3A4 than ritonavir</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">3733</span>– <span class="NLM_lpage">3741</span>, <span class="refDoi"> DOI: 10.1021/jm400288z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400288z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmtVylsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=3733-3741&author=I.+F.+Sevrioukovaauthor=T.+L.+Poulos&title=Pyridine-substituted+desoxyritonavir+is+a+more+potent+inhibitor+of+cytochrome+P450+3A4+than+ritonavir&doi=10.1021%2Fjm400288z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Pyridine-Substituted Desoxyritonavir Is a More Potent Inhibitor of Cytochrome P450 3A4 than Ritonavir</span></div><div class="casAuthors">Sevrioukova, Irina F.; Poulos, Thomas L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3733-3741</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Utilization of the cytochrome P 450 3A4 (CYP3A4) inhibitor ritonavir as a pharmacoenhancer for anti-HIV drugs revolutionized the treatment of HIV infection.  However, owing to ritonavir-related complications, there is a need for development of new CYP3A4 inhibitors with improved pharmacochem. properties, which requires a full understanding of the CYP3A4 inactivation mechanisms and the unraveling of possible inhibitor binding modes.  We investigated the mechanism of CYP3A4 interaction with three desoxyritonavir analogs, contg. the heme-ligating imidazole, oxazole, or pyridine group instead of the thiazole moiety (compds. 1, 2, and 3, resp.).  Our data show that compd. 3 is superior to ritonavir in terms of binding affinity and inhibitory potency owing to greater flexibility and the ability to adopt a conformation that minimizes steric clashing and optimizes protein-ligand interactions.  Addnl., Ser119 was identified as a key residue assisting binding of ritonavir-like inhibitors, which emphasizes the importance of polar interactions in the CYP3A4-ligand assocn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx1sbUve3zPLVg90H21EOLACvtfcHk0lhR0PYOHVBN6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmtVylsrc%253D&md5=77a94516a246e5823502f4c89cadc5c0</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1021%2Fjm400288z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400288z%26sid%3Dliteratum%253Aachs%26aulast%3DSevrioukova%26aufirst%3DI.%2BF.%26aulast%3DPoulos%26aufirst%3DT.%2BL.%26atitle%3DPyridine-substituted%2520desoxyritonavir%2520is%2520a%2520more%2520potent%2520inhibitor%2520of%2520cytochrome%2520P450%25203A4%2520than%2520ritonavir%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D3733%26epage%3D3741%26doi%3D10.1021%2Fjm400288z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yano, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denton, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerny, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cashman, J. R.</span></span> <span> </span><span class="NLM_article-title">Synthetic inhibitors of cytochrome P-450 2A6: Inhibitory activity, difference spectra, mechanism of inhibition, and protein cocrystallization</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">6987</span>– <span class="NLM_lpage">7001</span>, <span class="refDoi"> DOI: 10.1021/jm060519r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060519r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFKrs7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6987-7001&author=J.+K.+Yanoauthor=T.+T.+Dentonauthor=M.+A.+Cernyauthor=X.+Zhangauthor=E.+F.+Johnsonauthor=J.+R.+Cashman&title=Synthetic+inhibitors+of+cytochrome+P-450+2A6%3A+Inhibitory+activity%2C+difference+spectra%2C+mechanism+of+inhibition%2C+and+protein+cocrystallization&doi=10.1021%2Fjm060519r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic Inhibitors of Cytochrome P-450 2A6: Inhibitory Activity, Difference Spectra, Mechanism of Inhibition, and Protein Cocrystallization</span></div><div class="casAuthors">Yano, Jason K.; Denton, Travis T.; Cerny, Matthew A.; Zhang, Xiaodong; Johnson, Eric F.; Cashman, John R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6987-7001</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of 3-heteroarom. analogs of nicotine were synthesized to delineate structural and mechanistic requirements for selectively inhibiting human cytochrome P 450 (CYP) 2A6.  Thiophene, substituted thiophene, furan, substituted furan, acetylene, imidazole, substituted imidazole, thiazole, pyrazole, substituted pyrazole, and aliph. and isoxazol moieties were used to replace the N-methylpyrrolidine ring of nicotine.  A no. of potent inhibitors were identified, and several exhibited high selectivity for CYP2A6 relative to CYP2E1, -3A4, -2B6, -2C9, -2C19, and -2D6.  The majority of these inhibitors elicited type II difference spectra indicating the formation of a coordinate covalent bond to the heme iron.  The majority of inhibitors were reversible inhibitors although several mechanism-based inactivators were identified.  Most of the inhibitors were also relatively metabolically stable.  X-ray crystal structures of CYP2A6 cocrystd. with three furan analogs bearing methanamino side chains indicated that the amine side chain coordinated to the heme iron.  The pyridyl moiety was positioned to accept a hydrogen bond from Asn297, and all three inhibitors exhibited orthogonal arom.-arom. interactions with protein side chains.  For comparison, the cocrystal structure of 4,4'-dipyridyl disulfide was also obtained and showed that the pyridine moiety could assume a different orientation than that obsd. for the 3-heteroarom. pyridines examd.  For the 3-heteroromatic pyridines, N-Me and N,N-di-Me amino groups increased the apparent Ki and distorted helix I of the protein.  Substitution of a Ph ring for the pyridyl ring also increased the apparent Ki, which is likely to reflect the loss of the hydrogen bonding interaction with Asn297.  In contrast, inhibitory potency for other P450s was increased, and the selectivity of the Ph analogs for CYP2A6 was decreased relative to the pyridyl compds.  The results suggest that inhibitors that compliment the active site features of CYP2A6 can exhibit significant selectivity for CYP2A6 relative to other human liver drug-metabolizing P450s.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_GlAD-oZ-e7Vg90H21EOLACvtfcHk0lhR0PYOHVBN6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFKrs7bF&md5=0b6df3afd65811b26bf520817a5e5f0e</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1021%2Fjm060519r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060519r%26sid%3Dliteratum%253Aachs%26aulast%3DYano%26aufirst%3DJ.%2BK.%26aulast%3DDenton%26aufirst%3DT.%2BT.%26aulast%3DCerny%26aufirst%3DM.%2BA.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26aulast%3DCashman%26aufirst%3DJ.%2BR.%26atitle%3DSynthetic%2520inhibitors%2520of%2520cytochrome%2520P-450%25202A6%253A%2520Inhibitory%2520activity%252C%2520difference%2520spectra%252C%2520mechanism%2520of%2520inhibition%252C%2520and%2520protein%2520cocrystallization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6987%26epage%3D7001%26doi%3D10.1021%2Fjm060519r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: A new approach for rapid, accurate docking and scoring. 2. enrichment factors in database screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+A+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lhR0PYOHVBN6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520A%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 3 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lizaveta  Gotina</span>, <span class="hlFld-ContribAuthor ">Seon Hee  Seo</span>, <span class="hlFld-ContribAuthor ">Chae Won  Kim</span>, <span class="hlFld-ContribAuthor ">Sang Min  Lim</span>, <span class="hlFld-ContribAuthor ">Ae Nim  Pae</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacophore-Based Virtual Screening of Novel Competitive Inhibitors of the Neurodegenerative Disease Target Kynurenine-3-Monooxygenase. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (11)
                                     , 3314. <a href="https://doi.org/10.3390/molecules26113314" title="DOI URL">https://doi.org/10.3390/molecules26113314</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26113314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26113314%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DPharmacophore-Based%252BVirtual%252BScreening%252Bof%252BNovel%252BCompetitive%252BInhibitors%252Bof%252Bthe%252BNeurodegenerative%252BDisease%252BTarget%252BKynurenine-3-Monooxygenase%26aulast%3DGotina%26aufirst%3DLizaveta%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D11%26spage%3D3314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alejandro  Varela‐Rial</span>, <span class="hlFld-ContribAuthor ">Maciej  Majewski</span>, <span class="hlFld-ContribAuthor ">Gianni  De Fabritiis</span>. </span><span class="cited-content_cbyCitation_article-title">Structure based virtual screening: Fast and slow. </span><span class="cited-content_cbyCitation_journal-name">WIREs Computational Molecular Science</span><span> <strong>2021,</strong> <em>40 </em><a href="https://doi.org/10.1002/wcms.1544" title="DOI URL">https://doi.org/10.1002/wcms.1544</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/wcms.1544&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fwcms.1544%26sid%3Dliteratum%253Aachs%26jtitle%3DWIREs%2520Computational%2520Molecular%2520Science%26atitle%3DStructure%252Bbased%252Bvirtual%252Bscreening%25253A%252BFast%252Band%252Bslow%26aulast%3DVarela%25E2%2580%2590Rial%26aufirst%3DAlejandro%26date%3D2021%26date%3D2021%26volume%3D40" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Talia B.  Kimber</span>, <span class="hlFld-ContribAuthor ">Yonghui  Chen</span>, <span class="hlFld-ContribAuthor ">Andrea  Volkamer</span>. </span><span class="cited-content_cbyCitation_article-title">Deep Learning in Virtual Screening: Recent Applications and Developments. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (9)
                                     , 4435. <a href="https://doi.org/10.3390/ijms22094435" title="DOI URL">https://doi.org/10.3390/ijms22094435</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22094435&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22094435%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DDeep%252BLearning%252Bin%252BVirtual%252BScreening%25253A%252BRecent%252BApplications%252Band%252BDevelopments%26aulast%3DKimber%26aufirst%3DTalia%2BB.%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D9%26spage%3D4435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/medium/jm0c02227_0005.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/large/jm0c02227_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02227&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/medium/jm0c02227_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/large/jm0c02227_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Examples of binding modes. (A) Ligand–metal interaction of lisinopril with angiotensin-converting enzyme [Protein Data Bank (PDB) entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1O86">1O86</a>]. (B) <i>N</i>-(3,3-Diphenylpropyl)-4-benz-1,2,3-triazine-amine (left) and 3-cyano-<i>N</i>-(3,3-diphenylpropyl)-4-cinnolin-amine (right) bound to scytalone dehydratase in the presence or absence of a water molecule. Displacement of a water molecule by the cyano group resulted in a 2.0 kcal/mol increase in binding free energy. The figure was adapted after a crystal structure (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5STD">5STD</a>) and a study on hydration site prediction.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> (C) 4-Anilino-6,7-dialkoxyquinazoline (left) and 4-anilino-6,7-dialkoxyquinoline-3-carbonitrile (right) bound to EGFR kinase. Displacement of the water molecule resulted in an unfavorable binding free energy. Panel C was adapted after a crystal structure (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M17">1M17</a>) and a study on hydration site prediction.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> (D) Example for a strained ligand conformation in a crystal structure (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QAD">3QAD</a>)<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> with the 2-aminopyrimidine moiety out of plane as indicated by an arrow. The entry was since refined with a more reasonable conformation (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RZF">3RZF</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/large/jm0c02227_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02227&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/medium/jm0c02227_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/large/jm0c02227_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Results of the survey. (A) Participants of the survey divided according to their current scientific position. (B) Participants of the survey divided according to years of experience in computer-aided molecular design. (C) Procedures taken by all participants to validate the used docking protocols. (D) Methods of all participants used for postprocessing of docking results. (E) Importance of different criteria valued by all participants, participants from the pharmaceutical industry, and the remaining participants. (F) Importance of different criteria valued by all participants divided according to their years of experience compared to the overall result.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/large/jm0c02227_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02227&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/medium/jm0c02227_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/large/jm0c02227_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Hardware used for visualization by the participants. (B) Tasks for which visual inspection is applied by the participants within a discovery pipeline. (C) Number of correct answers (score) of the participants divided according to affiliation and experience. The percentages indicate the fraction of the total number of participants categorized into a group (e.g., participants from the industry) who achieved a certain score.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/large/jm0c02227_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02227&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/medium/jm0c02227_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/large/jm0c02227_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structures that were included in the survey to distinguish native poses from artifacts. In the survey, the structures were presented in form of an animated GIF image, showing all relevant interactions. (A) Case 1: (5-pyridin-3-ylfuran-2-yl)methanamine bound to CYP2A6 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FDW">2FDW</a>). The native pose is shown at the top, while a manually composed pose is shown at the bottom. (B) Case 2: 1-(5-methylthiophen-2-yl)-3-pyridin-3-ylurea bound to the main protease of SARS-CoV-2 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5RH0">5RH0</a>). The native pose is shown at the top, while a slightly different pose from docking is displayed at the bottom. (C) Case 3: <i>N</i>-[(4-fluorophenyl)methyl]-<i>N</i>-(2-methylpropyl)-6-[(1-methylsulfonylpiperidin-4-yl)amino]pyridine-3-sulfonamide bound to RAR-related orphan receptor γ (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WLB">4WLB</a>). The native pose is shown at the top, while a slightly different pose from docking is displayed at the bottom.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02227/20210305/images/large/jm0c02227_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02227&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i18">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82503" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82503" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 97 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seeliger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Groot, B. L.</span></span> <span> </span><span class="NLM_article-title">Ligand docking and binding site analysis with PyMOL and Autodock/Vina</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">417</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1007/s10822-010-9352-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1007%2Fs10822-010-9352-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=20401516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntVahtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=417-422&author=D.+Seeligerauthor=B.+L.+de+Groot&title=Ligand+docking+and+binding+site+analysis+with+PyMOL+and+Autodock%2FVina&doi=10.1007%2Fs10822-010-9352-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand docking and binding site analysis with PyMOL and Autodock/Vina</span></div><div class="casAuthors">Seeliger, Daniel; Groot, Bert L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">417-422</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Docking of small mol. compds. into the binding site of a receptor and estg. the binding affinity of the complex is an important part of the structure-based drug design process.  For a thorough understanding of the structural principles that det. the strength of a protein/ligand complex both, an accurate and fast docking protocol and the ability to visualize binding geometries and interactions are mandatory.  Here we present an interface between the popular mol. graphics system PyMOL and the mol. docking suites Autodock and Vina and demonstrate how the combination of docking and visualization can aid structure-based drug design efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTR4WZx4OqG7Vg90H21EOLACvtfcHk0libRMS8VfgMNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntVahtLo%253D&md5=35f8b79d6c1d6582919878a53b4a5129</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1007%2Fs10822-010-9352-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-010-9352-6%26sid%3Dliteratum%253Aachs%26aulast%3DSeeliger%26aufirst%3DD.%26aulast%3Dde%2BGroot%26aufirst%3DB.%2BL.%26atitle%3DLigand%2520docking%2520and%2520binding%2520site%2520analysis%2520with%2520PyMOL%2520and%2520Autodock%252FVina%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2010%26volume%3D24%26spage%3D417%26epage%3D422%26doi%3D10.1007%2Fs10822-010-9352-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bissantz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">A medicinal chemist’s guide to molecular interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">6241</span>– <span class="NLM_lpage">6241</span>, <span class="refDoi"> DOI: 10.1021/jm100950p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100950p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3cXps1Whtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=6241-6241&author=C.+Bissantzauthor=B.+Kuhnauthor=M.+Stahl&title=A+medicinal+chemist%E2%80%99s+guide+to+molecular+interactions&doi=10.1021%2Fjm100950p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">A Medicinal Chemist's Guide to Molecular Interactions. [Erratum to document cited in CA153:076147]</span></div><div class="casAuthors">Bissantz, Caterina; Kuhn, Bernd; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6241</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  On page 5063, the first sentence of the second paragraph in the left column is incorrect; the correct sentence is given.  On page 5075, the Figure 16 caption is incorrect; the correct caption is given.  On page 5083, Ref. 159 is incorrect; the correct version of the ref. is given.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXK65xi0HgarVg90H21EOLACvtfcHk0libRMS8VfgMNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXps1Whtr0%253D&md5=f9d8f9cc15728ffd2c09447b45fdf422</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fjm100950p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100950p%26sid%3Dliteratum%253Aachs%26aulast%3DBissantz%26aufirst%3DC.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DA%2520medicinal%2520chemist%25E2%2580%2599s%2520guide%2520to%2520molecular%2520interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D6241%26epage%3D6241%26doi%3D10.1021%2Fjm100950p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lyu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balius, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levit, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moroz, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Meara, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Algaa, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolmachova, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolmachev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span> <span> </span><span class="NLM_article-title">Ultra-large library docking for discovering new chemotypes</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>566</i></span>,  <span class="NLM_fpage">224</span>– <span class="NLM_lpage">229</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-0917-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1038%2Fs41586-019-0917-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=30728502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmt1yns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=566&publication_year=2019&pages=224-229&author=J.+Lyuauthor=S.+Wangauthor=T.+E.+Baliusauthor=I.+Singhauthor=A.+Levitauthor=Y.+S.+Morozauthor=M.+J.+O%E2%80%99Mearaauthor=T.+Cheauthor=E.+Algaaauthor=K.+Tolmachovaauthor=A.+A.+Tolmachevauthor=B.+K.+Shoichetauthor=B.+L.+Rothauthor=J.+J.+Irwin&title=Ultra-large+library+docking+for+discovering+new+chemotypes&doi=10.1038%2Fs41586-019-0917-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Ultra-large library docking for discovering new chemotypes</span></div><div class="casAuthors">Lyu, Jiankun; Wang, Sheng; Balius, Trent E.; Singh, Isha; Levit, Anat; Moroz, Yurii S.; O'Meara, Matthew J.; Che, Tao; Algaa, Enkhjargal; Tolmachova, Kateryna; Tolmachev, Andrey A.; Shoichet, Brian K.; Roth, Bryan L.; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">566</span>
        (<span class="NLM_cas:issue">7743</span>),
    <span class="NLM_cas:pages">224-229</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Despite intense interest in expanding chem. space, libraries contg. hundreds-of-millions to billions of diverse mols. have remained inaccessible.  Here we investigate structure-based docking of 170 million make-on-demand compds. from 130 well-characterized reactions.  The resulting library is diverse, representing over 10.7 million scaffolds that are otherwise unavailable.  For each compd. in the library, docking against AmpC β-lactamase (AmpC) and the D4 dopamine receptor were simulated.  From the top-ranking mols., 44 and 549 compds. were synthesized and tested for interactions with AmpC and the D4 dopamine receptor, resp.  We found a phenolate inhibitor of AmpC, which revealed a group of inhibitors without known precedent.  This mol. was optimized to 77 nM, which places it among the most potent non-covalent AmpC inhibitors known.  Crystal structures of this and other AmpC inhibitors confirmed the docking predictions.  Against the D4 dopamine receptor, hit rates fell almost monotonically with docking score, and a hit-rate vs. score curve predicted that the library contained 453,000 ligands for the D4 dopamine receptor.  Of 81 new chemotypes discovered, 30 showed submicromolar activity, including a 180-pM subtype-selective agonist of the D4 dopamine receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc_Nuo9pc-Z7Vg90H21EOLACvtfcHk0libRMS8VfgMNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmt1yns70%253D&md5=589fb81ef11f9cbec645b9a3818648d1</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-0917-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-0917-9%26sid%3Dliteratum%253Aachs%26aulast%3DLyu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DBalius%26aufirst%3DT.%2BE.%26aulast%3DSingh%26aufirst%3DI.%26aulast%3DLevit%26aufirst%3DA.%26aulast%3DMoroz%26aufirst%3DY.%2BS.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%2BJ.%26aulast%3DChe%26aufirst%3DT.%26aulast%3DAlgaa%26aufirst%3DE.%26aulast%3DTolmachova%26aufirst%3DK.%26aulast%3DTolmachev%26aufirst%3DA.%2BA.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DUltra-large%2520library%2520docking%2520for%2520discovering%2520new%2520chemotypes%26jtitle%3DNature%26date%3D2019%26volume%3D566%26spage%3D224%26epage%3D229%26doi%3D10.1038%2Fs41586-019-0917-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waszkowycz, B.</span></span> <span> </span><span class="NLM_article-title">Towards improving compound selection in structure-based virtual screening</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">219</span>– <span class="NLM_lpage">226</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2007.12.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1016%2Fj.drudis.2007.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=18342797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsVSjsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=219-226&author=B.+Waszkowycz&title=Towards+improving+compound+selection+in+structure-based+virtual+screening&doi=10.1016%2Fj.drudis.2007.12.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Towards improving compound selection in structure-based virtual screening</span></div><div class="casAuthors">Waszkowycz, Bohdan</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5/6</span>),
    <span class="NLM_cas:pages">219-226</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Structure-based virtual screening is now an established technol. for supporting hit finding and lead optimization in drug discovery.  Recent validation studies have highlighted the poor performance of currently used scoring functions in estg. binding affinity and hence in ranking large datasets of docked ligands.  Progress in the anal. of large datasets can be made through the use of appropriate data mining techniques and the derivation of a broader range of descriptors relevant to receptor-ligand binding.  In addn., simple scoring functions can be supplemented by simulation-based scoring protocols.  Developments in workflow design allow the automation of repetitive tasks, and also encourage the routine use of simulation-based methods and the rapid prototyping of novel modeling and anal. procedures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-3q2xzmPBubVg90H21EOLACvtfcHk0lgdIZu7UM0e9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsVSjsbg%253D&md5=a06324f926aadfe4c53f8941f9465d62</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2007.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2007.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DWaszkowycz%26aufirst%3DB.%26atitle%3DTowards%2520improving%2520compound%2520selection%2520in%2520structure-based%2520virtual%2520screening%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2008%26volume%3D13%26spage%3D219%26epage%3D226%26doi%3D10.1016%2Fj.drudis.2007.12.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cappel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jerome, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matter, H.</span></span> <span> </span><span class="NLM_article-title">Impact of different automated binding pose generation approaches on relative binding free energy simulations</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1432</span>– <span class="NLM_lpage">1444</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.9b01118</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.9b01118" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs12ktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=1432-1444&author=D.+Cappelauthor=S.+Jeromeauthor=G.+Hesslerauthor=H.+Matter&title=Impact+of+different+automated+binding+pose+generation+approaches+on+relative+binding+free+energy+simulations&doi=10.1021%2Facs.jcim.9b01118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Different Automated Binding Pose Generation Approaches on Relative Binding Free Energy Simulations</span></div><div class="casAuthors">Cappel, Daniel; Jerome, Steven; Hessler, Gerhard; Matter, Hans</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1432-1444</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Relative binding free energy (RBFE) prediction methods such as free energy perturbation (FEP) are important today for estg. protein-ligand binding affinities.  Significant hardware and algorithmic improvements now allow for simulating congeneric series within days.  Therefore, RBFE calcns. have an enormous potential for structure-based drug discovery.  As typically only a few representative crystal structures for a series are available, other ligands and design proposals must be reliably superimposed for meaningful results.  An obsd. significant effect of the alignment on FEP led us to develop an alignment approach combining docking with max. common substructure (MCS) derived core constraints from the most similar ref. pose, named MCS-docking workflow.  We then studied the effect of binding pose generation on the accuracy of RBFE predictions using six ligand series from five pharmaceutically relevant protein targets.  Overall, the MCS-docking workflow generated consistent poses for most of the ligands in the investigated series.  While multiple alignment methods often resulted in comparable FEP predictions, for most of the cases herein, the MCS-docking workflow produced the best accuracy in predictions.  Furthermore, the FEP anal. data strongly support the hypothesis that the accuracy of RBFE predictions depends on input poses to construct the perturbation map.  Therefore, an automated workflow without manual intervention minimizes potential errors and obtains the most useful predictions with significant impact for structure-based design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGob6kqQDqVs-7Vg90H21EOLACvtfcHk0lgdIZu7UM0e9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs12ktb4%253D&md5=c6c5e60560061e35da8e661f17c10a8e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.9b01118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.9b01118%26sid%3Dliteratum%253Aachs%26aulast%3DCappel%26aufirst%3DD.%26aulast%3DJerome%26aufirst%3DS.%26aulast%3DHessler%26aufirst%3DG.%26aulast%3DMatter%26aufirst%3DH.%26atitle%3DImpact%2520of%2520different%2520automated%2520binding%2520pose%2520generation%2520approaches%2520on%2520relative%2520binding%2520free%2520energy%2520simulations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D1432%26epage%3D1444%26doi%3D10.1021%2Facs.jcim.9b01118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, R. T.</span></span> <span> </span><span class="NLM_article-title">Structure-based drug design: Docking and scoring</span>. <i>Curr. Protein Pept. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">312</span>– <span class="NLM_lpage">328</span>, <span class="refDoi"> DOI: 10.2174/138920307781369382</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.2174%2F138920307781369382" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=17696866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpslKrs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=312-328&author=R.+T.+Kroemer&title=Structure-based+drug+design%3A+Docking+and+scoring&doi=10.2174%2F138920307781369382"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based drug design: docking and scoring</span></div><div class="casAuthors">Kroemer, Romano T.</div><div class="citationInfo"><span class="NLM_cas:title">Current Protein and Peptide Science</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">312-328</span>CODEN:
                <span class="NLM_cas:coden">CPPSCM</span>;
        ISSN:<span class="NLM_cas:issn">1389-2037</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">This review gives an introduction into ligand - receptor docking and illustrates the basic underlying concepts.  An overview of different approaches and algorithms is provided.  Although the application of docking and scoring has led to some remarkable successes, there are still some major challenges ahead, which are outlined here as well.  Approaches to address some of these challenges and the latest developments in the area are presented.  Some aspects of the assessment of docking program performance are discussed.  A no. of successful applications of structure-based virtual screening are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbXWKN0_d9g7Vg90H21EOLACvtfcHk0lgdIZu7UM0e9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpslKrs7w%253D&md5=5bfb61ca9c4b405a98f50ddc2bb00ed7</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.2174%2F138920307781369382&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138920307781369382%26sid%3Dliteratum%253Aachs%26aulast%3DKroemer%26aufirst%3DR.%2BT.%26atitle%3DStructure-based%2520drug%2520design%253A%2520Docking%2520and%2520scoring%26jtitle%3DCurr.%2520Protein%2520Pept.%2520Sci.%26date%3D2007%26volume%3D8%26spage%3D312%26epage%3D328%26doi%3D10.2174%2F138920307781369382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mobley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dill, K. A.</span></span> <span> </span><span class="NLM_article-title">Binding of small-molecule ligands to proteins: ”What you see” is not always ”what you get</span>. <i>Structure (Oxford, U. K.)</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">489</span>– <span class="NLM_lpage">498</span>, <span class="refDoi"> DOI: 10.1016/j.str.2009.02.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1016%2Fj.str.2009.02.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=19368882" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVCmsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=489-498&author=D.+L.+Mobleyauthor=K.+A.+Dill&title=Binding+of+small-molecule+ligands+to+proteins%3A+%E2%80%9DWhat+you+see%E2%80%9D+is+not+always+%E2%80%9Dwhat+you+get&doi=10.1016%2Fj.str.2009.02.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Binding of Small-Molecule Ligands to Proteins: "What You See" Is Not Always "What You Get"</span></div><div class="casAuthors">Mobley, David L.; Dill, Ken A.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">489-498</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  We review insights from computational studies of affinities of ligands binding to proteins.  The power of structural biol. is in translating knowledge of protein structures into insights about their forces, binding, and mechanisms.  However, the complementary power of computer modeling is in showing "the rest of the story" (i.e., how motions and ensembles and alternative conformers and the entropies and forces that cannot be seen in single mol. structures also contribute to binding affinities).  Upon binding to a protein, a ligand can bind in multiple orientations; the protein or ligand can be deformed by the binding event; waters, ions, or cofactors can have unexpected involvement; and conformational or solvation entropies can sometimes play large and otherwise unpredictable roles.  Computer modeling is helping to elucidate these factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnlrwNwfLm4rVg90H21EOLACvtfcHk0ljdnwAOscr2xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVCmsrk%253D&md5=993623b1444e33a45b2fe8b904dec9c1</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2009.02.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2009.02.010%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DBinding%2520of%2520small-molecule%2520ligands%2520to%2520proteins%253A%2520%25E2%2580%259DWhat%2520you%2520see%25E2%2580%259D%2520is%2520not%2520always%2520%25E2%2580%259Dwhat%2520you%2520get%26jtitle%3DStructure%2520%2528Oxford%252C%2520U.%2520K.%2529%26date%3D2009%26volume%3D17%26spage%3D489%26epage%3D498%26doi%3D10.1016%2Fj.str.2009.02.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lexa, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, H. A.</span></span> <span> </span><span class="NLM_article-title">Protein flexibility in docking and surface mapping</span>. <i>Q. Rev. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">301</span>– <span class="NLM_lpage">343</span>, <span class="refDoi"> DOI: 10.1017/S0033583512000066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1017%2FS0033583512000066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=22569329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1WlsLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2012&pages=301-343&author=K.+W.+Lexaauthor=H.+A.+Carlson&title=Protein+flexibility+in+docking+and+surface+mapping&doi=10.1017%2FS0033583512000066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Protein flexibility in docking and surface mapping</span></div><div class="casAuthors">Lexa, Katrina W.; Carlson, Heather A.</div><div class="citationInfo"><span class="NLM_cas:title">Quarterly Reviews of Biophysics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">301-343</span>CODEN:
                <span class="NLM_cas:coden">QURBAW</span>;
        ISSN:<span class="NLM_cas:issn">0033-5835</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review.  Structure-based drug design has become an essential tool for rapid lead discovery and optimization.  As available structural information has increased, researchers have become increasingly aware of the importance of protein flexibility for accurate description of the native state.  Typical protein-ligand docking efforts still rely on a single rigid receptor, which is an incomplete representation of potential binding conformations of the protein.  These rigid docking efforts typically show the best performance rates between 50 and 75%, while fully flexible docking methods can enhance pose prediction up to 80-95%.  This review examines the current toolbox for flexible protein-ligand docking and receptor surface mapping.  Present limitations and possibilities for future development are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWRT8xbQSJ77Vg90H21EOLACvtfcHk0ljdnwAOscr2xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1WlsLzE&md5=868db49242f652c02cc61a7d4f6d3d0f</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1017%2FS0033583512000066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS0033583512000066%26sid%3Dliteratum%253Aachs%26aulast%3DLexa%26aufirst%3DK.%2BW.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DProtein%2520flexibility%2520in%2520docking%2520and%2520surface%2520mapping%26jtitle%3DQ.%2520Rev.%2520Biophys.%26date%3D2012%26volume%3D45%26spage%3D301%26epage%3D343%26doi%3D10.1017%2FS0033583512000066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Damm-Ganamet, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bembenek, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venable, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castro, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangelschots, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peeters, D. C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcallister, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disepio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirzadegan, T.</span></span> <span> </span><span class="NLM_article-title">A prospective virtual screening study: Enriching hit rates and designing focus libraries to find inhibitors of PI3Kδ and PI3Kγ</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4302</span>– <span class="NLM_lpage">4313</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01974</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01974" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC28XltlOmtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4302-4313&author=K.+L.+Damm-Ganametauthor=S.+D.+Bembenekauthor=J.+W.+Venableauthor=G.+G.+Castroauthor=L.+Mangelschotsauthor=D.+C.+G.+Peetersauthor=H.+M.+Mcallisterauthor=J.+P.+Edwardsauthor=D.+Disepioauthor=T.+Mirzadegan&title=A+prospective+virtual+screening+study%3A+Enriching+hit+rates+and+designing+focus+libraries+to+find+inhibitors+of+PI3K%CE%B4+and+PI3K%CE%B3&doi=10.1021%2Facs.jmedchem.5b01974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A Prospective Virtual Screening Study: Enriching Hit Rates and Designing Focus Libraries To Find Inhibitors of PI3Kδ and PI3Kγ</span></div><div class="casAuthors">Damm-Ganamet, Kelly L.; Bembenek, Scott D.; Venable, Jennifer W.; Castro, Glenda G.; Mangelschots, Lieve; Peeters, Danielle C. G.; Mcallister, Heather M.; Edwards, James P.; Disepio, Daniel; Mirzadegan, Taraneh</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4302-4313</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here, we report a high-throughput virtual screening (HTVS) study using phosphoinositide 3-kinase (both PI3Kγ and PI3Kδ).  Our initial HTVS results of the Janssen corporate database identified small focused libraries with hit rates at 50% inhibition showing a 50-fold increase over those from a HTS (high-throughput screen).  Further, applying constraints based on "chem. intuitive" hydrogen bonds and/or positional requirements resulted in a substantial improvement in the hit rates (vs. no constraints) and reduced docking time.  While we find that docking scoring functions are not capable of providing a reliable relative ranking of a set of compds., a prioritization of groups of compds. (e.g., low, medium, and high) does emerge, which allows for the chem. efforts to be quickly focused on the most viable candidates.  Thus, this illustrates that it is not always necessary to have a high correlation between a computational score and the exptl. data to impact the drug discovery process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgMG23GSsrOrVg90H21EOLACvtfcHk0ljdnwAOscr2xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltlOmtLc%253D&md5=809d86cc271e5b4f603691304cceafc5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01974%26sid%3Dliteratum%253Aachs%26aulast%3DDamm-Ganamet%26aufirst%3DK.%2BL.%26aulast%3DBembenek%26aufirst%3DS.%2BD.%26aulast%3DVenable%26aufirst%3DJ.%2BW.%26aulast%3DCastro%26aufirst%3DG.%2BG.%26aulast%3DMangelschots%26aufirst%3DL.%26aulast%3DPeeters%26aufirst%3DD.%2BC.%2BG.%26aulast%3DMcallister%26aufirst%3DH.%2BM.%26aulast%3DEdwards%26aufirst%3DJ.%2BP.%26aulast%3DDisepio%26aufirst%3DD.%26aulast%3DMirzadegan%26aufirst%3DT.%26atitle%3DA%2520prospective%2520virtual%2520screening%2520study%253A%2520Enriching%2520hit%2520rates%2520and%2520designing%2520focus%2520libraries%2520to%2520find%2520inhibitors%2520of%2520PI3K%25CE%25B4%2520and%2520PI3K%25CE%25B3%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4302%26epage%3D4313%26doi%3D10.1021%2Facs.jmedchem.5b01974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lionta, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spyrou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassilatis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cournia, Z.</span></span> <span> </span><span class="NLM_article-title">Structure-based virtual screening for drug discovery: Principles, applications and recent advances</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1923</span>– <span class="NLM_lpage">1938</span>, <span class="refDoi"> DOI: 10.2174/1568026614666140929124445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.2174%2F1568026614666140929124445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=25262799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGgt7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=1923-1938&author=E.+Liontaauthor=G.+Spyrouauthor=D.+Vassilatisauthor=Z.+Cournia&title=Structure-based+virtual+screening+for+drug+discovery%3A+Principles%2C+applications+and+recent+advances&doi=10.2174%2F1568026614666140929124445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Virtual Screening for Drug Discovery: Principles, Applications and Recent Advances</span></div><div class="casAuthors">Lionta, Evanthia; Spyrou, George; Vassilatis, Demetrios K.; Cournia, Zoe</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1923-1938</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Structure-based drug discovery (SBDD) is becoming an essential tool in assisting fast and cost-efficient lead discovery and optimization.  The application of rational, structure-based drug design is proven to be more efficient than the traditional way of drug discovery since it aims to understand the mol. basis of a disease and utilizes the knowledge of the three-dimensional structure of the biol. target in the process.  In this review, we focus on the principles and applications of Virtual Screening (VS) within the context of SBDD and examine different procedures ranging from the initial stages of the process that include receptor and library pre-processing, to docking, scoring and post-processing of top scoring hits.  Recent improvements in structure-based virtual screening (SBVS) efficiency through ensemble docking, induced fit and consensus docking are also discussed.  The review highlights advances in the field within the framework of several success studies that have led to nM inhibition directly from VS and provides recent trends in library design as well as discusses limitations of the method.  Applications of SBVS in the design of substrates for engineered proteins that enable the discovery of new metabolic and signal transduction pathways and the design of inhibitors of multifunctional proteins are also reviewed.  Finally, we contribute two promising VS protocols recently developed by us that aim to increase inhibitor selectivity.  In the first protocol, we describe the discovery of micromolar inhibitors through SBVS designed to inhibit the mutant H1047R PI3Kα kinase.  Second, we discuss a strategy for the identification of selective binders for the RXRα nuclear receptor.  In this protocol, a set of target structures is constructed for ensemble docking based on binding site shape characterization and clustering, aiming to enhance the hit rate of selective inhibitors for the desired protein target through the SBVS process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGps80ptfisjzLVg90H21EOLACvtfcHk0ljdnwAOscr2xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGgt7fJ&md5=0550a396dd011adf52699b091548345b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.2174%2F1568026614666140929124445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026614666140929124445%26sid%3Dliteratum%253Aachs%26aulast%3DLionta%26aufirst%3DE.%26aulast%3DSpyrou%26aufirst%3DG.%26aulast%3DVassilatis%26aufirst%3DD.%26aulast%3DCournia%26aufirst%3DZ.%26atitle%3DStructure-based%2520virtual%2520screening%2520for%2520drug%2520discovery%253A%2520Principles%252C%2520applications%2520and%2520recent%2520advances%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2014%26volume%3D14%26spage%3D1923%26epage%3D1938%26doi%3D10.2174%2F1568026614666140929124445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cournia, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beuming, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearlman, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radak, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Rigorous free energy simulations in virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4153</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c00116</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c00116" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFGgu7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=4153&author=Z.+Courniaauthor=B.+K.+Allenauthor=T.+Beumingauthor=D.+A.+Pearlmanauthor=B.+K.+Radakauthor=W.+Sherman&title=Rigorous+free+energy+simulations+in+virtual+screening&doi=10.1021%2Facs.jcim.0c00116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Rigorous Free Energy Simulations in Virtual Screening</span></div><div class="casAuthors">Cournia, Zoe; Allen, Bryce K.; Beuming, Thijs; Pearlman, David A.; Radak, Brian K.; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4153-4169</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Virtual high throughput screening (vHTS) in drug discovery is a powerful approach to identify hits: when applied successfully, it can be much faster and cheaper than exptl. high-throughput screening approaches.  However, mainstream vHTS tools have significant limitations: ligand-based methods depend on knowledge of existing chem. matter, while structure-based tools such as docking involve significant approxns. that limit their accuracy.  Recent advances in scientific methods coupled with dramatic speedups in computational processing with GPUs make this an opportune time to consider the role of more rigorous methods that could improve the predictive power of vHTS workflows.  In this Perspective, we assert that alchem. binding free energy methods using all-atom mol. dynamics simulations have matured to the point where they can be applied in virtual screening campaigns as a final scoring stage to prioritize the top mols. for exptl. testing.  Specifically, we propose that alchem. abs. binding free energy (ABFE) calcns. offer the most direct and computationally efficient approach within a rigorous statistical thermodn. framework for computing binding energies of diverse mols., as is required for virtual screening.  ABFE calcns. are particularly attractive for drug discovery at this point in time, where the confluence of large-scale genomics data and insights from chem. biol. have unveiled a large no. of promising disease targets for which no small mol. binders are known, precluding ligand-based approaches, and where traditional docking approaches have foundered to find progressible chem. matter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGol_yim2RNVYrVg90H21EOLACvtfcHk0lgrWm36P9EA2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFGgu7fP&md5=4f144e9f9fe045f00e00ded0a5e8d2a1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c00116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c00116%26sid%3Dliteratum%253Aachs%26aulast%3DCournia%26aufirst%3DZ.%26aulast%3DAllen%26aufirst%3DB.%2BK.%26aulast%3DBeuming%26aufirst%3DT.%26aulast%3DPearlman%26aufirst%3DD.%2BA.%26aulast%3DRadak%26aufirst%3DB.%2BK.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DRigorous%2520free%2520energy%2520simulations%2520in%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D4153%26doi%3D10.1021%2Facs.jcim.0c00116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schulz-Gasch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">Scoring functions for protein-ligand interactions: A critical perspective</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">231</span>– <span class="NLM_lpage">239</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2004.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1016%2Fj.ddtec.2004.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=24981490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1ClsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2004&pages=231-239&author=T.+Schulz-Gaschauthor=M.+Stahl&title=Scoring+functions+for+protein-ligand+interactions%3A+A+critical+perspective&doi=10.1016%2Fj.ddtec.2004.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Scoring functions for protein-ligand interactions: a critical perspective</span></div><div class="casAuthors">Schulz-Gasch, Tanja; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today: Technologies</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">231-239</span>CODEN:
                <span class="NLM_cas:coden">DDTTB5</span>;
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Scoring functions play an essential role in structure-based virtual screening.  They are required to guide the docking of candidate compds. to structures of receptor binding sites, to select probable binding modes, and to discriminate binders from non-binders.  Although many scoring functions have successfully been used to identify novel ligands for a wide variety of targets, much work remains to be done to avoid incorrect prediction of binding modes and high nos. of false positives.  This review gives an overview of the current state of the field and outlines key issues for the further development of scoring functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroEKzPtDzEqbVg90H21EOLACvtfcHk0lgrWm36P9EA2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1ClsLs%253D&md5=72849854914ed59b41dc30a5a197943e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2004.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2004.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DSchulz-Gasch%26aufirst%3DT.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DScoring%2520functions%2520for%2520protein-ligand%2520interactions%253A%2520A%2520critical%2520perspective%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2004%26volume%3D1%26spage%3D231%26epage%3D239%26doi%3D10.1016%2Fj.ddtec.2004.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parks, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaieb, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGaughey, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bembenek, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ameriks, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirzadegan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burley, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaro, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span> <span> </span><span class="NLM_article-title">D3R grand challenge 4: Blind prediction of protein-ligand poses, affinity rankings, and relative binding free energies</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1007/s10822-020-00289-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1007%2Fs10822-020-00289-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=31974851" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1SmtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2020&pages=99-119&author=C.+D.+Parksauthor=Z.+Gaiebauthor=M.+Chiuauthor=H.+Yangauthor=C.+Shaoauthor=W.+P.+Waltersauthor=J.+M.+Jansenauthor=G.+McGaugheyauthor=R.+A.+Lewisauthor=S.+D.+Bembenekauthor=M.+K.+Ameriksauthor=T.+Mirzadeganauthor=S.+K.+Burleyauthor=R.+E.+Amaroauthor=M.+K.+Gilson&title=D3R+grand+challenge+4%3A+Blind+prediction+of+protein-ligand+poses%2C+affinity+rankings%2C+and+relative+binding+free+energies&doi=10.1007%2Fs10822-020-00289-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">D3R grand challenge 4: blind prediction of protein-ligand poses, affinity rankings, and relative binding free energies</span></div><div class="casAuthors">Parks, Conor D.; Gaieb, Zied; Chiu, Michael; Yang, Huanwang; Shao, Chenghua; Walters, W. Patrick; Jansen, Johanna M.; McGaughey, Georgia; Lewis, Richard A.; Bembenek, Scott D.; Ameriks, Michael K.; Mirzadegan, Tara; Burley, Stephen K.; Amaro, Rommie E.; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">99-119</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Abstr.: The Drug Design Data Resource (D3R) aims to identify best practice methods for computer aided drug design through blinded ligand pose prediction and affinity challenges.  Herein, we report on the results of Grand Challenge 4 (GC4).  GC4 focused on proteins beta secretase 1 and Cathepsin S, and was run in an analogous manner to prior challenges.  In Stage 1, participant ability to predict the pose and affinity of BACE1 ligands were assessed.  Following the completion of Stage 1, all BACE1 co-crystal structures were released, and Stage 2 tested affinity rankings with co-crystal structures.  We provide an anal. of the results and discuss insights into detd. best practice methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNni47G6bCWrVg90H21EOLACvtfcHk0lgrWm36P9EA2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1SmtL8%253D&md5=57a12faffcb1def7224696bf23a0e599</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1007%2Fs10822-020-00289-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-020-00289-y%26sid%3Dliteratum%253Aachs%26aulast%3DParks%26aufirst%3DC.%2BD.%26aulast%3DGaieb%26aufirst%3DZ.%26aulast%3DChiu%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DShao%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DW.%2BP.%26aulast%3DJansen%26aufirst%3DJ.%2BM.%26aulast%3DMcGaughey%26aufirst%3DG.%26aulast%3DLewis%26aufirst%3DR.%2BA.%26aulast%3DBembenek%26aufirst%3DS.%2BD.%26aulast%3DAmeriks%26aufirst%3DM.%2BK.%26aulast%3DMirzadegan%26aufirst%3DT.%26aulast%3DBurley%26aufirst%3DS.%2BK.%26aulast%3DAmaro%26aufirst%3DR.%2BE.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DD3R%2520grand%2520challenge%25204%253A%2520Blind%2520prediction%2520of%2520protein-ligand%2520poses%252C%2520affinity%2520rankings%252C%2520and%2520relative%2520binding%2520free%2520energies%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2020%26volume%3D34%26spage%3D99%26epage%3D119%26doi%3D10.1007%2Fs10822-020-00289-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaieb, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parks, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nevins, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bembenek, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ameriks, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirzadegan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burley, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaro, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span> <span> </span><span class="NLM_article-title">D3R grand challenge 3: Blind prediction of protein-ligand poses and affinity rankings</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.1007/s10822-018-0180-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1007%2Fs10822-018-0180-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=30632055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFShtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=1-18&author=Z.+Gaiebauthor=C.+D.+Parksauthor=M.+Chiuauthor=H.+Yangauthor=C.+Shaoauthor=W.+P.+Waltersauthor=M.+H.+Lambertauthor=N.+Nevinsauthor=S.+D.+Bembenekauthor=M.+K.+Ameriksauthor=T.+Mirzadeganauthor=S.+K.+Burleyauthor=R.+E.+Amaroauthor=M.+K.+Gilson&title=D3R+grand+challenge+3%3A+Blind+prediction+of+protein-ligand+poses+and+affinity+rankings&doi=10.1007%2Fs10822-018-0180-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">D3R Grand Challenge 3: blind prediction of protein-ligand poses and affinity rankings</span></div><div class="casAuthors">Gaieb, Zied; Parks, Conor D.; Chiu, Michael; Yang, Huanwang; Shao, Chenghua; Walters, W. Patrick; Lambert, Millard H.; Nevins, Neysa; Bembenek, Scott D.; Ameriks, Michael K.; Mirzadegan, Tara; Burley, Stephen K.; Amaro, Rommie E.; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-18</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The Drug Design Data Resource aims to test and advance the state of the art in protein-ligand modeling by holding community-wide blinded, prediction challenges.  Here, we report on our third major round, Grand Challenge 3 (GC3).  Held 2017-2018, GC3 centered on the protein Cathepsin S and the kinases VEGFR2, JAK2, p38-α, TIE2, and ABL1, and included both pose-prediction and affinity-ranking components.  GC3 was structured much like the prior challenges GC2015 and GC2.  First, Stage 1 tested pose prediction and affinity ranking methods; then all available crystal structures were released, and Stage 2 tested only affinity rankings, now in the context of the available structures.  Unique to GC3 was the addn. of a Stage 1b self-docking subchallenge, in which the protein coordinates from all of the cocrystal structures used in the cross-docking challenge were released, and participants were asked to predict the pose of CatS ligands using these newly released structures.  We provide an overview of the outcomes and discuss insights into trends and best-practices.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJKkuRnCIHS7Vg90H21EOLACvtfcHk0ljsJjYo6UajGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFShtbc%253D&md5=110f1b35386467b53febcaf8d2fcdea7</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1007%2Fs10822-018-0180-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-018-0180-4%26sid%3Dliteratum%253Aachs%26aulast%3DGaieb%26aufirst%3DZ.%26aulast%3DParks%26aufirst%3DC.%2BD.%26aulast%3DChiu%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DShao%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DW.%2BP.%26aulast%3DLambert%26aufirst%3DM.%2BH.%26aulast%3DNevins%26aufirst%3DN.%26aulast%3DBembenek%26aufirst%3DS.%2BD.%26aulast%3DAmeriks%26aufirst%3DM.%2BK.%26aulast%3DMirzadegan%26aufirst%3DT.%26aulast%3DBurley%26aufirst%3DS.%2BK.%26aulast%3DAmaro%26aufirst%3DR.%2BE.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DD3R%2520grand%2520challenge%25203%253A%2520Blind%2520prediction%2520of%2520protein-ligand%2520poses%2520and%2520affinity%2520rankings%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2019%26volume%3D33%26spage%3D1%26epage%3D18%26doi%3D10.1007%2Fs10822-018-0180-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, A. M.</span>; <span class="NLM_string-name">Teague, S. J.</span>; <span class="NLM_string-name">Kleywegt, G. J.</span></span> <span> </span><span class="NLM_article-title">Application and limitations of X-ray crystallographic data in structure-guided ligand and drug design</span>. <span class="NLM_year">2007</span>. <a href="http://europepmc.org/abstract/CTX/C7868" class="extLink">http://europepmc.org/abstract/CTX/C7868</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=A.+M.+Davis&author=S.+J.+Teague&author=G.+J.+Kleywegt&title=Application+and+limitations+of+X-ray+crystallographic+data+in+structure-guided+ligand+and+drug+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DA.%2BM.%26atitle%3DApplication%2520and%2520limitations%2520of%2520X-ray%2520crystallographic%2520data%2520in%2520structure-guided%2520ligand%2520and%2520drug%2520design%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ripphausen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Analysis of structure-based virtual screening studies and characterization of identified active compounds</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">613</span>, <span class="refDoi"> DOI: 10.4155/fmc.12.18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.4155%2Ffmc.12.18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=22458680" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC38Xks1Oqtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=603-613&author=P.+Ripphausenauthor=D.+Stumpfeauthor=J.+Bajorath&title=Analysis+of+structure-based+virtual+screening+studies+and+characterization+of+identified+active+compounds&doi=10.4155%2Ffmc.12.18"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of structure-based virtual screening studies and characterization of identified active compounds</span></div><div class="casAuthors">Ripphausen, Peter; Stumpfe, Dagmar; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">603-613</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Structure-based virtual screening makes explicit or implicit use of 3D target structure information to detect novel active compds.  Results of nearly 300 currently available original applications have been analyzed to characterize the state-of-the-art in this field.  Compd. selection from docking calcns. is much influenced by subjective criteria.  Although submicromolar compds. are identified, the majority of docking hits are only weakly potent.  However, only a small percentage of docking hits can be reproduced by ligand-based methods.  When docking calcns. identify potent hits, they often originate from specialized compd. sources (e.g., pharmaceutical compd. decks or target-focused libraries) and also display a notable bias towards kinase targets.  Structure-based virtual screening is the dominant approach to computational hit identification.  Docking calcns. frequently identify active compds.  Limited accuracy of compd. scoring and ranking currently presents a major caveat of the approach that is often compensated for by chem. intuition and knowledge.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHjToM3jECobVg90H21EOLACvtfcHk0ljsJjYo6UajGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xks1Oqtbw%253D&md5=d2d0bc27efbbcdd34fcde4da37e5e192</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.18%26sid%3Dliteratum%253Aachs%26aulast%3DRipphausen%26aufirst%3DP.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DAnalysis%2520of%2520structure-based%2520virtual%2520screening%2520studies%2520and%2520characterization%2520of%2520identified%2520active%2520compounds%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D603%26epage%3D613%26doi%3D10.4155%2Ffmc.12.18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaturvedi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatnagar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqi, M. I.</span></span> <span> </span><span class="NLM_article-title">Knowledge based identification of potent antitubercular compounds using structure based virtual screening and structure interaction fingerprints</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">35</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1021/ci8003607</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci8003607" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVOmtrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=35-42&author=A.+Kumarauthor=V.+Chaturvediauthor=S.+Bhatnagarauthor=S.+Sinhaauthor=M.+I.+Siddiqi&title=Knowledge+based+identification+of+potent+antitubercular+compounds+using+structure+based+virtual+screening+and+structure+interaction+fingerprints&doi=10.1021%2Fci8003607"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Knowledge Based Identification of Potent Antitubercular Compounds Using Structure Based Virtual Screening and Structure Interaction Fingerprints</span></div><div class="casAuthors">Kumar, Ashutosh; Chaturvedi, Vinita; Bhatnagar, Shalini; Sinha, Sudhir; Siddiqi, Mohammad Imran</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">35-42</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In view of the worldwide spread of multidrug resistance of Mycobacterium tuberculosis, there is an urgent need to discover antitubercular agents with novel structures.  Thymidine monophosphate kinase from M. tuberculosis (TMPKmt) is an attractive target for antitubercular chemotherapy.  We report here the identification of potent antitubercular compds. targeting TMPKmt using virtual screening methods.  For this purpose we have developed a pharmacophore hypothesis based on the substrate and known TMPKmt inhibitors and employed it to screen the Maybridge small mol. database.  The mol. docking was then performed in order to select the compds. on the basis of their ability to form favorable interactions with the TMPKmt active site.  In addn., we applied straightforward weighting using structure interaction fingerprints to include addnl. knowledge into structure based virtual screening.  Eight compds. were acquired and evaluated for antitubercular activity against M. tuberculosis H37Rv in vitro, and out of these 3 compds. showed MIC of 3.12 μg/mL whereas 2 compds. showed MIC of 12.5 μg/mL.  All the active compds. were found to be nontoxic in Vero cell lines and mice bone marrow macrophages.  All the identified hits highlighted a key hydrogen bonding interaction with Arg74.  The obsd. π-stacking interaction with Phe70 was also produced by the identified hits.  These hits represent promising starting points for structural optimization in hit-to-lead development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4p1Vh5kNAqrVg90H21EOLACvtfcHk0ljsJjYo6UajGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVOmtrfE&md5=c0dc91a59fa31b396647c01150dd2641</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fci8003607&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci8003607%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DChaturvedi%26aufirst%3DV.%26aulast%3DBhatnagar%26aufirst%3DS.%26aulast%3DSinha%26aufirst%3DS.%26aulast%3DSiddiqi%26aufirst%3DM.%2BI.%26atitle%3DKnowledge%2520based%2520identification%2520of%2520potent%2520antitubercular%2520compounds%2520using%2520structure%2520based%2520virtual%2520screening%2520and%2520structure%2520interaction%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D35%26epage%3D42%26doi%3D10.1021%2Fci8003607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez-Pérez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miljković, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Assessing the information content of structural and protein-ligand interaction representations for the classification of kinase inhibitor binding modes via machine learning and active learning</span>. <i>J. Cheminf.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">36</span>, <span class="refDoi"> DOI: 10.1186/s13321-020-00434-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1186%2Fs13321-020-00434-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVyks7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&pages=36&author=R.+Rodr%C3%ADguez-P%C3%A9rezauthor=F.+Miljkovi%C4%87author=J.+Bajorath&title=Assessing+the+information+content+of+structural+and+protein-ligand+interaction+representations+for+the+classification+of+kinase+inhibitor+binding+modes+via+machine+learning+and+active+learning&doi=10.1186%2Fs13321-020-00434-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing the information content of structural and protein-ligand interaction representations for the classification of kinase inhibitor binding modes via machine learning and active learning</span></div><div class="casAuthors">Rodriguez-Perez, Raquel; Miljkovic, Filip; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">SpringerOpen</span>)
        </div><div class="casAbstract">We have addressed this prediction task to evaluate and compare the information content of distinct mol. representations including protein-ligand interaction fingerprints and compd. structure-based structural fingerprints (i.e., atom environment/fragment fingerprints).  Accurate predictions of kinase inhibitor binding modes were achieved with random forests using both representations.  The performance of IFPs was consistently superior to atom environment fingerprints, albeit only by less than 10%.  An active learning strategy applying information entropy-based selection of training instances was applied as a diagnostic approach to assess the relative information content of distinct representations.  IFPs were found to capture more binding mode-relevant information than atom environment fingerprints, leading to highly predictive models even when training instances were randomly selected.  By contrast, for atom environment fingerprints, the derivation of accurate models via active learning depended on entropy-based selection of informative training compds.  Notably, higher information content of IFPs confirmed by active learning only resulted in small improvements in global prediction accuracy compared to models derived using atom environment fingerprints.  For practical applications, prediction of binding modes of new kinase inhibitors on the basis of chem. structure is highly attractive.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOV05t-VYny7Vg90H21EOLACvtfcHk0liXIjsRpbt0rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVyks7vN&md5=c9f9b03c5b347c2642fee97b493b0b00</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1186%2Fs13321-020-00434-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-020-00434-7%26sid%3Dliteratum%253Aachs%26aulast%3DRodr%25C3%25ADguez-P%25C3%25A9rez%26aufirst%3DR.%26aulast%3DMiljkovi%25C4%2587%26aufirst%3DF.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DAssessing%2520the%2520information%2520content%2520of%2520structural%2520and%2520protein-ligand%2520interaction%2520representations%2520for%2520the%2520classification%2520of%2520kinase%2520inhibitor%2520binding%2520modes%2520via%2520machine%2520learning%2520and%2520active%2520learning%26jtitle%3DJ.%2520Cheminf.%26date%3D2020%26volume%3D12%26spage%3D36%26doi%3D10.1186%2Fs13321-020-00434-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Varela-Rial, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majewski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuzzolin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Rosell, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Fabritiis, G.</span></span> <span> </span><span class="NLM_article-title">SkeleDock: A web application for scaffold docking in PlayMolecule</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2673</span>– <span class="NLM_lpage">2677</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c00143</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c00143" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpt1Ghur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=2673-2677&author=A.+Varela-Rialauthor=M.+Majewskiauthor=A.+Cuzzolinauthor=G.+Mart%C3%ADnez-Rosellauthor=G.+de+Fabritiis&title=SkeleDock%3A+A+web+application+for+scaffold+docking+in+PlayMolecule&doi=10.1021%2Facs.jcim.0c00143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">SkeleDock: A Web Application for Scaffold Docking in PlayMolecule</span></div><div class="casAuthors">Varela-Rial, Alejandro; Majewski, Maciej; Cuzzolin, Alberto; Martinez-Rosell, Gerard; De Fabritiis, Gianni</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2673-2677</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SkeleDock is a scaffold docking algorithm which uses the structure of a protein-ligand complex as a template to model the binding mode of a chem. similar system.  This algorithm was evaluated in the D3R Grand Challenge 4 pose prediction challenge, where it achieved competitive performance.  Furthermore, the authors show that if crystd. fragments of the target ligand are available then SkeleDock can outperform rDock docking software at predicting the binding mode.  This Application Note also addresses the capacity of this algorithm to model macrocycles and deal with scaffold hopping.  SkeleDock can be accessed at https://playmol.org/SkeleDock/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgUgig_dif1LVg90H21EOLACvtfcHk0liXIjsRpbt0rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpt1Ghur4%253D&md5=a15dda548cfa1d9e6891d116b61bcdb8</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c00143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c00143%26sid%3Dliteratum%253Aachs%26aulast%3DVarela-Rial%26aufirst%3DA.%26aulast%3DMajewski%26aufirst%3DM.%26aulast%3DCuzzolin%26aufirst%3DA.%26aulast%3DMart%25C3%25ADnez-Rosell%26aufirst%3DG.%26aulast%3Dde%2BFabritiis%26aufirst%3DG.%26atitle%3DSkeleDock%253A%2520A%2520web%2520application%2520for%2520scaffold%2520docking%2520in%2520PlayMolecule%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D2673%26epage%3D2677%26doi%3D10.1021%2Facs.jcim.0c00143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velázquez-Libera, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durán-Verdugo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valdés-Jiménez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Núñez-Vivanco, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caballero, J.</span></span> <span> </span><span class="NLM_article-title">LigRMSD: A web server for automatic structure matching and RMSD calculations among identical and similar compounds in protein-ligand docking</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">2912</span>– <span class="NLM_lpage">2914</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btaa018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1093%2Fbioinformatics%2Fbtaa018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=31926012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitlOlsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2020&pages=2912-2914&author=J.+L.+Vel%C3%A1zquez-Liberaauthor=F.+Dur%C3%A1n-Verdugoauthor=A.+Vald%C3%A9s-Jim%C3%A9nezauthor=G.+N%C3%BA%C3%B1ez-Vivancoauthor=J.+Caballero&title=LigRMSD%3A+A+web+server+for+automatic+structure+matching+and+RMSD+calculations+among+identical+and+similar+compounds+in+protein-ligand+docking&doi=10.1093%2Fbioinformatics%2Fbtaa018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">LigRMSD: a web server for automatic structure matching and RMSD calculations among identical and similar compounds in protein-ligand docking</span></div><div class="casAuthors">Velazquez-Libera, Jose Luis; Duran-Verdugo, Fabio; Valdes-Jimenez, Alejandro; Nunez-Vivanco, Gabriel; Caballero, Julio</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2912-2914</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4811</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Root mean square deviation (RMSD) is one of the most useful and straightforward features for structural comparison between different conformations of the same mol.  Commonly, protein-ligand docking programs have included some utilities that allow the calcn. of this value; however, they only work efficiently when exists a complete atom label equivalence between the evaluated conformations.  Results: We present LigRMSD, a free web-server for the automatic matching and RMSD calcns. among identical or similar chem. compds.  This server allows the user to submit only a pair of identical or similar mols. or dataset of similar compds. to compare their three-dimensional conformations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd-lb-rDFcrbVg90H21EOLACvtfcHk0liXIjsRpbt0rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitlOlsr7I&md5=1c115dd1028c339cbcbf441e7f263e0a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtaa018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtaa018%26sid%3Dliteratum%253Aachs%26aulast%3DVel%25C3%25A1zquez-Libera%26aufirst%3DJ.%2BL.%26aulast%3DDur%25C3%25A1n-Verdugo%26aufirst%3DF.%26aulast%3DVald%25C3%25A9s-Jim%25C3%25A9nez%26aufirst%3DA.%26aulast%3DN%25C3%25BA%25C3%25B1ez-Vivanco%26aufirst%3DG.%26aulast%3DCaballero%26aufirst%3DJ.%26atitle%3DLigRMSD%253A%2520A%2520web%2520server%2520for%2520automatic%2520structure%2520matching%2520and%2520RMSD%2520calculations%2520among%2520identical%2520and%2520similar%2520compounds%2520in%2520protein-ligand%2520docking%26jtitle%3DBioinformatics%26date%3D2020%26volume%3D36%26spage%3D2912%26epage%3D2914%26doi%3D10.1093%2Fbioinformatics%2Fbtaa018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perryman, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekins, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freundlich, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonge, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">A virtual screen discovers novel, fragment-sized inhibitors of Mycobacterium tuberculosis InhA</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">645</span>– <span class="NLM_lpage">659</span>, <span class="refDoi"> DOI: 10.1021/ci500672v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500672v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVehsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=645-659&author=A.+L.+Perrymanauthor=W.+Yuauthor=X.+Wangauthor=S.+Ekinsauthor=S.+Forliauthor=S.-G.+Liauthor=J.+S.+Freundlichauthor=P.+J.+Tongeauthor=A.+J.+Olson&title=A+virtual+screen+discovers+novel%2C+fragment-sized+inhibitors+of+Mycobacterium+tuberculosis+InhA&doi=10.1021%2Fci500672v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A Virtual Screen Discovers Novel, Fragment-Sized Inhibitors of Mycobacterium tuberculosis InhA</span></div><div class="casAuthors">Perryman, Alexander L.; Yu, Weixuan; Wang, Xin; Ekins, Sean; Forli, Stefano; Li, Shao-Gang; Freundlich, Joel S.; Tonge, Peter J.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">645-659</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Isoniazid (INH) is usually administered to treat latent Mycobacterium tuberculosis (Mtb) infections and is used in combination therapy to treat active tuberculosis (TB).  Unfortunately, resistance to this drug is hampering its clin. effectiveness.  INH is a prodrug that must be activated by Mtb catalase-peroxidase (KatG) before it can inhibit InhA (Mtb enoyl-acyl-carrier-protein reductase).  Isoniazid-resistant cases of TB found in clin. settings usually involve mutations in or deletion of katG, which abrogate INH activation.  Compds. that inhibit InhA without requiring prior activation by KatG would not be affected by this resistance mechanism and hence would display continued potency against these drug-resistant isolates of Mtb.  Virtual screening expts. vs. InhA in the GO Fight Against Malaria (GO FAM) project were designed to discover new scaffolds that display base-stacking interactions with the NAD cofactor.  GO FAM expts. included targets from other pathogens, including Mtb, when they had structural similarity to a malaria target.  Eight of the 16 sol. compds. identified by docking against InhA plus visual inspection were modest inhibitors and did not require prior activation by KatG.  The best two inhibitors discovered are both fragment-sized compds. and displayed Ki values of 54 and 59 μM, resp.  Importantly, the novel inhibitors discovered have low structural similarity to known InhA inhibitors and thus help expand the no. of chemotypes on which future medicinal chem. efforts can be focused.  These new fragment hits could eventually help advance the fight against INH-resistant Mtb strains, which pose a significant global health threat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9fifQHkgzX7Vg90H21EOLACvtfcHk0ljVfFDGKeVOQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVehsrc%253D&md5=f68a7af63dca612e3017260c32593858</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fci500672v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500672v%26sid%3Dliteratum%253Aachs%26aulast%3DPerryman%26aufirst%3DA.%2BL.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DEkins%26aufirst%3DS.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DS.-G.%26aulast%3DFreundlich%26aufirst%3DJ.%2BS.%26aulast%3DTonge%26aufirst%3DP.%2BJ.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DA%2520virtual%2520screen%2520discovers%2520novel%252C%2520fragment-sized%2520inhibitors%2520of%2520Mycobacterium%2520tuberculosis%2520InhA%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D645%26epage%3D659%26doi%3D10.1021%2Fci500672v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Viviani, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piccirillo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulrich, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaral, A. T.-d.</span></span> <span> </span><span class="NLM_article-title">Virtual screening approach for the identification of hydroxamic acids as novel human ecto-5-nucleotidase inhibitors</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">621</span>– <span class="NLM_lpage">630</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.9b00884</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.9b00884" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFCns7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=621-630&author=L.+G.+Vivianiauthor=E.+Piccirilloauthor=H.+Ulrichauthor=A.+T.-d.+Amaral&title=Virtual+screening+approach+for+the+identification+of+hydroxamic+acids+as+novel+human+ecto-5-nucleotidase+inhibitors&doi=10.1021%2Facs.jcim.9b00884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Screening Approach for the Identification of Hydroxamic Acids as Novel Human Ecto-5'-Nucleotidase Inhibitors</span></div><div class="casAuthors">Viviani, Lucas G.; Piccirillo, Erika; Ulrich, Henning; Amaral, Antonia T.-do</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">621-630</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ecto-5'-nucleotidase (ecto-5'-NT, CD73) is a zinc-binding metallophosphatase that plays a key role in extracellular purinergic pathways, being implicated in several physiol. and pathophysiol. processes, such as immune homeostasis, inflammation, and tumor progression.  As such, it has been recognized as a promising biol. target for many diseases, including cancer, infections, and autoimmune diseases.  Despite its importance, so far only a few inhibitors of this target enzyme are known, most of which are not suitable as drug candidates.  Here, we aimed to search for hydroxamic acid-contg. compds. as potential human ecto-5'-NT inhibitors, since this group is known to be a strong zinc chelator.  To this end, we performed a hierarchical virtual screening (VS) search consisting of three consecutive steps (filtering for compds. bearing a hydroxamic acid group, shape-based matching, and docking followed by visual inspection), which were applied to screen the ZINC-14 database ("all purchasable subset").  Out of 25 compds. selected by this VS protocol, 12 were acquired and further submitted to enzymic assays for VS exptl. validation.  Four of them (i.e., 33.3%) were found to inhibit human ecto-5'-NT in the low micromolar range.  The most potent one showed an IC50 value of 6.2 ± 1.0μM.  All identified inhibitors satisfy drug-like criteria and provide novel scaffolds to be explored in further hit-to-lead optimization steps.  Furthermore, to the best of our knowledge, they are the first hydroxamic acid-contg. inhibitors of human ecto-5'-NT described so far.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP4Qiu0YGYKLVg90H21EOLACvtfcHk0ljVfFDGKeVOQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFCns7zJ&md5=e27572f72ce511b572a671b2aa4b6448</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.9b00884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.9b00884%26sid%3Dliteratum%253Aachs%26aulast%3DViviani%26aufirst%3DL.%2BG.%26aulast%3DPiccirillo%26aufirst%3DE.%26aulast%3DUlrich%26aufirst%3DH.%26aulast%3DAmaral%26aufirst%3DA.%2BT.-d.%26atitle%3DVirtual%2520screening%2520approach%2520for%2520the%2520identification%2520of%2520hydroxamic%2520acids%2520as%2520novel%2520human%2520ecto-5-nucleotidase%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D621%26epage%3D630%26doi%3D10.1021%2Facs.jcim.9b00884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toledo-Sherman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deretey, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slon-Usakiewicz, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, J.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redden, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uger, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasternak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, N.</span></span> <span> </span><span class="NLM_article-title">Frontal affinity chromatography with MS detection of EphB2 tyrosine kinase receptor. 2. identification of small-molecule inhibitors via coupling with virtual screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">3221</span>– <span class="NLM_lpage">3230</span>, <span class="refDoi"> DOI: 10.1021/jm0492204</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0492204" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtFSjtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=3221-3230&author=L.+Toledo-Shermanauthor=E.+Dereteyauthor=J.+J.+Slon-Usakiewiczauthor=W.+Ngauthor=J.-R.+Daiauthor=J.+E.+Fosterauthor=P.+R.+Reddenauthor=M.+D.+Ugerauthor=L.+C.+Liaoauthor=A.+Pasternakauthor=N.+Reid&title=Frontal+affinity+chromatography+with+MS+detection+of+EphB2+tyrosine+kinase+receptor.+2.+identification+of+small-molecule+inhibitors+via+coupling+with+virtual+screening&doi=10.1021%2Fjm0492204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Frontal Affinity Chromatography with MS Detection of EphB2 Tyrosine Kinase Receptor. 2. Identification of Small-Molecule Inhibitors via Coupling with Virtual Screening</span></div><div class="casAuthors">Toledo-Sherman, Leticia; Deretey, Eugen; Slon-Usakiewicz, Jacek J.; Ng, William; Dai, Jin-Rui; Foster, J. Estelle; Redden, Peter R.; Uger, Marni D.; Liao, Linda C.; Pasternak, Andrew; Reid, Neil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3221-3230</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have integrated two complementary methods, high-throughput virtual screening with a "high-content" wet screening technique based on frontal affinity chromatog. with mass spectrometry detection (FAC-MS), for identification of hits against the erythropoietin-producing hepatocellular B2 (EphB2) receptor tyrosine kinase domain.  Both an EphB2-directed virtual screen combining docking and scoring and a kinase-directed pharmacophore search strategy were used to identify a compd. set enriched in bioactive compds. against EphB2.  The coupling of virtual screening methodologies with FAC-MS is a unique hybrid approach that can be used to increase the efficacy of both hit discovery and optimization efforts in drug discovery and has successfully identified hits, in particular (I)(36% shift, IC50 = 5.2 μM, Kd = 3.3 μM), as inhibitors for EphB2, a potential cancer target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgtP-ON3dk-7Vg90H21EOLACvtfcHk0ljVfFDGKeVOQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtFSjtbg%253D&md5=b73aba584c10ef6d11d68b1701d3668d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm0492204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0492204%26sid%3Dliteratum%253Aachs%26aulast%3DToledo-Sherman%26aufirst%3DL.%26aulast%3DDeretey%26aufirst%3DE.%26aulast%3DSlon-Usakiewicz%26aufirst%3DJ.%2BJ.%26aulast%3DNg%26aufirst%3DW.%26aulast%3DDai%26aufirst%3DJ.-R.%26aulast%3DFoster%26aufirst%3DJ.%2BE.%26aulast%3DRedden%26aufirst%3DP.%2BR.%26aulast%3DUger%26aufirst%3DM.%2BD.%26aulast%3DLiao%26aufirst%3DL.%2BC.%26aulast%3DPasternak%26aufirst%3DA.%26aulast%3DReid%26aufirst%3DN.%26atitle%3DFrontal%2520affinity%2520chromatography%2520with%2520MS%2520detection%2520of%2520EphB2%2520tyrosine%2520kinase%2520receptor.%25202.%2520identification%2520of%2520small-molecule%2520inhibitors%2520via%2520coupling%2520with%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D3221%26epage%3D3230%26doi%3D10.1021%2Fjm0492204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Postigo, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guido, R. V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliva, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castilho, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">da R.
Pitta, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Albuquerque, J. F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andricopulo, A. D.</span></span> <span> </span><span class="NLM_article-title">Discovery of new inhibitors of Schistosoma mansoni PNP by pharmacophore-based virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1693</span>– <span class="NLM_lpage">1705</span>, <span class="refDoi"> DOI: 10.1021/ci100128k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100128k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvVyltLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1693-1705&author=M.+P.+Postigoauthor=R.+V.+C.+Guidoauthor=G.+Olivaauthor=M.+S.+Castilhoauthor=I.+da+R.%0APittaauthor=J.+F.+C.+de+Albuquerqueauthor=A.+D.+Andricopulo&title=Discovery+of+new+inhibitors+of+Schistosoma+mansoni+PNP+by+pharmacophore-based+virtual+screening&doi=10.1021%2Fci100128k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of New Inhibitors of Schistosoma mansoni PNP by Pharmacophore-Based Virtual Screening</span></div><div class="casAuthors">Postigo, Matheus P.; Guido, Rafael V. C.; Oliva, Glaucius; Castilho, Marcelo S.; Pitta, Ivan da R.; de Albuquerque, Julianna F. C.; Andricopulo, Adriano D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1693-1705</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Schistosomiasis is considered the second most important tropical parasitic disease, with severe socioeconomic consequences for millions of people worldwide.  Schistosoma mansoni, one of the causative agents of human schistosomiasis, is unable to synthesize purine nucleotides de novo, which makes the enzymes of the purine salvage pathway important targets for antischistosomal drug development.  In the present work, we describe the development of a pharmacophore model for ligands of S. mansoni purine nucleoside phosphorylase (SmPNP) as well as a pharmacophore-based virtual screening approach, which resulted in the identification of three thioxothiazolidinones (1-3) with substantial in vitro inhibitory activity against SmPNP.  Synthesis, biochem. evaluation, and structure-activity relationship investigations led to the successful development of a small set of thioxothiazolidinone derivs. harboring a novel chem. scaffold as new competitive inhibitors of SmPNP at the low-micromolar range.  Seven compds. were identified with IC50 values below 100 μM.  The most potent inhibitors 7, 10, and 17 with IC50 of 2, 18, and 38 μM, resp., could represent new potential lead compds. for further development of the therapy of schistosomiasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiGw2pRusgirVg90H21EOLACvtfcHk0lhwIkUq3uyqOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvVyltLk%253D&md5=bb68b1dec1359a46d7f2c388a5a4dc72</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fci100128k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100128k%26sid%3Dliteratum%253Aachs%26aulast%3DPostigo%26aufirst%3DM.%2BP.%26aulast%3DGuido%26aufirst%3DR.%2BV.%2BC.%26aulast%3DOliva%26aufirst%3DG.%26aulast%3DCastilho%26aufirst%3DM.%2BS.%26aulast%3Dda%2BR.%2BPitta%26aufirst%3DI.%26aulast%3Dde%2BAlbuquerque%26aufirst%3DJ.%2BF.%2BC.%26aulast%3DAndricopulo%26aufirst%3DA.%2BD.%26atitle%3DDiscovery%2520of%2520new%2520inhibitors%2520of%2520Schistosoma%2520mansoni%2520PNP%2520by%2520pharmacophore-based%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1693%26epage%3D1705%26doi%3D10.1021%2Fci100128k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caporuscio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbriano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">del Rio, A.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of potent aromatase inhibitors by high-throughput docking</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">4006</span>– <span class="NLM_lpage">4017</span>, <span class="refDoi"> DOI: 10.1021/jm2000689</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2000689" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmsFylsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=4006-4017&author=F.+Caporuscioauthor=G.+Rastelliauthor=C.+Imbrianoauthor=A.+del+Rio&title=Structure-based+design+of+potent+aromatase+inhibitors+by+high-throughput+docking&doi=10.1021%2Fjm2000689"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Potent Aromatase Inhibitors by High-Throughput Docking</span></div><div class="casAuthors">Caporuscio, Fabiana; Rastelli, Giulio; Imbriano, Carol; Del Rio, Alberto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4006-4017</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cytochrome P 450 aromatase catalyzes the conversion of androgen substrates into estrogens.  Aromatase inhibitors (AIs) have been used as first-line drugs in the treatment of estrogen-dependent breast cancer in postmenopausal women.  However, the search for new, more potent, and selective AIs still remains necessary to avoid the risk of possible resistances and reduce toxicity and side effects of current available drugs.  The publication of a high resoln. x-ray structure of human aromatase has opened the way to structure-based virtual screening to identify new small-mol. inhibitors with structural motifs different from all known AIs.  In this context, a high-throughput docking protocol was set up and led to the identification of nanomolar AIs with new core structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBteVWbhM8PLVg90H21EOLACvtfcHk0lhwIkUq3uyqOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmsFylsL8%253D&md5=93b8b8a30807469412b6a88db79b4a4c</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm2000689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2000689%26sid%3Dliteratum%253Aachs%26aulast%3DCaporuscio%26aufirst%3DF.%26aulast%3DRastelli%26aufirst%3DG.%26aulast%3DImbriano%26aufirst%3DC.%26aulast%3Ddel%2BRio%26aufirst%3DA.%26atitle%3DStructure-based%2520design%2520of%2520potent%2520aromatase%2520inhibitors%2520by%2520high-throughput%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D4006%26epage%3D4017%26doi%3D10.1021%2Fjm2000689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumari, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span> <span> </span><span class="NLM_article-title">Drug design targeting protein-protein interactions (PPIs) Using multiple ligand simultaneous docking (MLSD) and drug repositioning: Discovery of raloxifene and bazedoxifene as novel inhibitors of IL-6/GP130 interface</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">632</span>– <span class="NLM_lpage">641</span>, <span class="refDoi"> DOI: 10.1021/jm401144z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401144z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1Ohtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=632-641&author=H.+Liauthor=H.+Xiaoauthor=L.+Linauthor=D.+Jouauthor=V.+Kumariauthor=J.+Linauthor=C.+Li&title=Drug+design+targeting+protein-protein+interactions+%28PPIs%29+Using+multiple+ligand+simultaneous+docking+%28MLSD%29+and+drug+repositioning%3A+Discovery+of+raloxifene+and+bazedoxifene+as+novel+inhibitors+of+IL-6%2FGP130+interface&doi=10.1021%2Fjm401144z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Design Targeting Protein-Protein Interactions (PPIs) Using Multiple Ligand Simultaneous Docking (MLSD) and Drug Repositioning: Discovery of Raloxifene and Bazedoxifene as Novel Inhibitors of IL-6/GP130 Interface</span></div><div class="casAuthors">Li, Huameng; Xiao, Hui; Lin, Li; Jou, David; Kumari, Vandana; Lin, Jiayuh; Li, Chenglong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">632-641</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The IL-6/GP130/STAT3 pathway is crit. for the progression of multiple types of cancers.  The authors report the discovery of raloxifene and bazedoxifene as novel inhibitors of IL-6/GP130 protein-protein interactions (PPIs) using multiple ligand simultaneous docking (MLSD) and drug repositioning approaches.  Multiple drug scaffolds were simultaneously docked into hot spots of GP130 D1 domain by MLSD to compete with the key interacting residues of IL-6, followed by tethering to generate virtual hit compds.  Similarity searches of virtual hits on drug databases identified raloxifene and bazedoxifene as potential inhibitors of IL-6/GP130 interaction.  In cancer cell assays both compds. bind to GP130 and demonstrated selective inhibition of IL-6 induced STAT3 phosphorylation and were significantly more potent than the previously reported natural product inhibitor MDL-A.  The identified drugs represent a new class of lead compds. with piperidine, benzothiophene, and indole scaffolds to inhibit IL-6 induced homodimerization of GP130.  Besides potential direct usage for clinic trials, the two compds. can also serve as lead compds. for optimization to speed the development of drugs selectively targeting the IL-6/GP130/STAT3 cancer signaling pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZJ8PO1lH9g7Vg90H21EOLACvtfcHk0lhwIkUq3uyqOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1Ohtr8%253D&md5=a4f2887c2ad3a5dc97827e7a7499efe0</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm401144z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401144z%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DXiao%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DL.%26aulast%3DJou%26aufirst%3DD.%26aulast%3DKumari%26aufirst%3DV.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DC.%26atitle%3DDrug%2520design%2520targeting%2520protein-protein%2520interactions%2520%2528PPIs%2529%2520Using%2520multiple%2520ligand%2520simultaneous%2520docking%2520%2528MLSD%2529%2520and%2520drug%2520repositioning%253A%2520Discovery%2520of%2520raloxifene%2520and%2520bazedoxifene%2520as%2520novel%2520inhibitors%2520of%2520IL-6%252FGP130%2520interface%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D632%26epage%3D641%26doi%3D10.1021%2Fjm401144z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.-p.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, H.</span></span> <span> </span><span class="NLM_article-title">Protein flexibility in docking-based virtual screening: Discovery of novel lymphoid-specific tyrosine phosphatase inhibitors using multiple crystal structures</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1973</span>– <span class="NLM_lpage">1983</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00344</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00344" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyntbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=1973-1983&author=X.+Houauthor=K.+Liauthor=X.+Yuauthor=J.-p.+Sunauthor=H.+Fang&title=Protein+flexibility+in+docking-based+virtual+screening%3A+Discovery+of+novel+lymphoid-specific+tyrosine+phosphatase+inhibitors+using+multiple+crystal+structures&doi=10.1021%2Facs.jcim.5b00344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Flexibility in Docking-Based Virtual Screening: Discovery of Novel Lymphoid-Specific Tyrosine Phosphatase Inhibitors Using Multiple Crystal Structures</span></div><div class="casAuthors">Hou, Xuben; Li, Kangshuai; Yu, Xiao; Sun, Jin-peng; Fang, Hao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1973-1983</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Incorporating protein flexibility is a major challenge for docking-based virtual screening.  With an increasing no. of available crystal structures, ensemble docking with multiple protein structures is an efficient approach to deal with protein flexibility.  Herein, the authors report the successful application of a docking-based virtual screen using multiple crystal structures to discover novel inhibitors of lymphoid-specific tyrosine phosphatase (LYP), a potential drug target for autoimmune diseases.  The appropriate use of multiple protein structures allowed a better enrichment than a single structure in the recovery of known inhibitors.  Subsequently, an optimal ensemble of LYP structures was selected and used in docking-based virtual screening.  Eight novel LYP inhibitors (IC50 ranging from 7.95 to 56.6 μM) were identified among 23 hit compds.  Further studies demonstrated that the most active compd. B15 possessed some selectivity over other protein phosphatases and could effectively up-regulate TCR (T cell receptor)-mediated signaling in Jurkat T cells.  These novel hits not only provided good starting points for the development of therapeutic agents useful in autoimmune diseases but also demonstrated the advantages of choosing an appropriate ensemble of protein structures in docking-based virtual screening over using a single protein conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiDcC42tLbKLVg90H21EOLACvtfcHk0lgLM7w62iCafw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyntbvM&md5=9296da92e46d98f8363906ff1c14fd70</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00344%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DJ.-p.%26aulast%3DFang%26aufirst%3DH.%26atitle%3DProtein%2520flexibility%2520in%2520docking-based%2520virtual%2520screening%253A%2520Discovery%2520of%2520novel%2520lymphoid-specific%2520tyrosine%2520phosphatase%2520inhibitors%2520using%2520multiple%2520crystal%2520structures%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D1973%26epage%3D1983%26doi%3D10.1021%2Facs.jcim.5b00344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desai, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avery, M.</span></span> <span> </span><span class="NLM_article-title">Targeting the BH3 domain mediated protein-protein interaction of Bcl-xL through virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">906</span>– <span class="NLM_lpage">923</span>, <span class="refDoi"> DOI: 10.1021/ci1000373</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci1000373" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3cXks1CksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=906-923&author=P.+Mukherjeeauthor=P.+Desaiauthor=Y.-D.+Zhouauthor=M.+Avery&title=Targeting+the+BH3+domain+mediated+protein-protein+interaction+of+Bcl-xL+through+virtual+screening&doi=10.1021%2Fci1000373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the BH3 Domain Mediated Protein-Protein Interaction of Bcl-xL through Virtual Screening</span></div><div class="casAuthors">Mukherjee, Prasenjit; Desai, Prashant; Zhou, Yu-Dong; Avery, Mitchell</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">906-923</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Apoptosis, or programmed cell death, forms an important part of the cellular regulation machinery.  The Bcl-2 protein family, comprising of proapoptotic and antiapoptotic members, forms an important part of the cells internal apoptotic pathway.  Overexpression of the antiapoptotic members of the family in a no. of cancer cell lines renders them immune to apoptosis and the ability to survive under conditions of cellular stress.  Inhibition of the antiapoptotic members of the Bcl-2 family are, therefore, an interesting target for the development of anticancer therapy.  An innovative structure-based virtual screening strategy was developed to identify inhibitors of Bcl-xL, an antiapoptotic member of the Bcl-2 family.  Various innovative filters, such as receptor-based pharmacophore, cascade docking approach, cross-docking, and composite scoring with docking pose based descriptors were designed through exhaustive validation studies and implemented in the screening funnel.  The 1.8 million big-n-greasy' subset from ZINC was screened using the protocol, and 45 compds. were finally selected for biol. evaluation against Bcl-xL.  The evaluation led to the identification of one low-micromolar and two weaker inhibitors belonging to novel scaffolds.  Further evaluation of structure-activity relationships around these scaffolds could help in the development of anticancer leads against Bcl-xL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd89CIbgyDLrVg90H21EOLACvtfcHk0lgLM7w62iCafw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXks1CksLk%253D&md5=73a5a6b976c3e3c37939e1c96fa86d50</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fci1000373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci1000373%26sid%3Dliteratum%253Aachs%26aulast%3DMukherjee%26aufirst%3DP.%26aulast%3DDesai%26aufirst%3DP.%26aulast%3DZhou%26aufirst%3DY.-D.%26aulast%3DAvery%26aufirst%3DM.%26atitle%3DTargeting%2520the%2520BH3%2520domain%2520mediated%2520protein-protein%2520interaction%2520of%2520Bcl-xL%2520through%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D906%26epage%3D923%26doi%3D10.1021%2Fci1000373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ye, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Identification of selective, cell active inhibitors of protein arginine methyltransferase 5 through structure-based virtual screening and biological assays</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1066</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.8b00050</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00050" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnvV2ltbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=1066-1073&author=F.+Yeauthor=W.+Zhangauthor=X.+Yeauthor=J.+Jinauthor=Z.+Lvauthor=C.+Luo&title=Identification+of+selective%2C+cell+active+inhibitors+of+protein+arginine+methyltransferase+5+through+structure-based+virtual+screening+and+biological+assays&doi=10.1021%2Facs.jcim.8b00050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Selective, Cell Active Inhibitors of Protein Arginine Methyltransferase 5 through Structure-Based Virtual Screening and Biological Assays</span></div><div class="casAuthors">Ye, Fei; Zhang, Weiyao; Ye, Xiaoqing; Jin, Jia; Lv, Zhengbing; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1066-1073</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5), a type II PRMT enzyme, is reported as an important therapeutic target in leukemia and lymphoma.  In the present study, based on the combination of virtual screening and biochem. validations, we discovered a series of small-mol. inhibitors targeting PRMT5.  Among those, DC_Y134 exhibited the most potent activity with IC50 value of 1.7 μM and displayed good selectivity against other methyltransferases.  Further treatment with DC_Y134 inhibited the proliferation of several hematol. malignancy cell lines by causing cell cycle arrest and apoptosis.  Western blot assays indicated that DC_Y134 reduced the cellular sym. dimethylated levels.  In addn., we analyzed the binding mode of DC_Y134 through mol. docking, which revealed that DC_Y134 occupies the binding site of substrate arginine and explained the selectivity of this inhibitor.  Taken together, compd. DC_Y134 could be used to elucidate the biol. roles of PRMT5 and serve as a lead compd. for treatment of hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp59U0G_Qsx7LVg90H21EOLACvtfcHk0lgLM7w62iCafw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnvV2ltbY%253D&md5=d805b7fb6d533d9506c7eb69f1b6e68d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00050%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DYe%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DLv%26aufirst%3DZ.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520selective%252C%2520cell%2520active%2520inhibitors%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520through%2520structure-based%2520virtual%2520screening%2520and%2520biological%2520assays%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2018%26volume%3D58%26spage%3D1066%26epage%3D1073%26doi%3D10.1021%2Facs.jcim.8b00050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span> <span> </span><span class="NLM_article-title">Development of small-molecules targeting receptor activator of nuclear factor-κB ligand (RANKL)-receptor activator of nuclear factor-κB (RANK) protein-protein interaction by structure-based virtual screening and hit optimization</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">5370</span>– <span class="NLM_lpage">5381</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b02027</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b02027" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXptlOjsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=5370-5381&author=M.+Jiangauthor=L.+Pengauthor=K.+Yangauthor=T.+Wangauthor=X.+Yanauthor=T.+Jiangauthor=J.+Xuauthor=J.+Qiauthor=H.+Zhouauthor=N.+Qianauthor=Q.+Zhouauthor=B.+Chenauthor=X.+Xuauthor=L.+Dengauthor=C.+Yang&title=Development+of+small-molecules+targeting+receptor+activator+of+nuclear+factor-%CE%BAB+ligand+%28RANKL%29-receptor+activator+of+nuclear+factor-%CE%BAB+%28RANK%29+protein-protein+interaction+by+structure-based+virtual+screening+and+hit+optimization&doi=10.1021%2Facs.jmedchem.8b02027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Small-Molecules Targeting Receptor Activator of Nuclear Factor-κB Ligand (RANKL)-Receptor Activator of Nuclear Factor-κB (RANK) Protein-Protein Interaction by Structure-Based Virtual Screening and Hit Optimization</span></div><div class="casAuthors">Jiang, Min; Peng, Lei; Yang, Kai; Wang, Tianqi; Yan, Xueming; Jiang, Tao; Xu, Jianrong; Qi, Jin; Zhou, Hanbing; Qian, Niandong; Zhou, Qi; Chen, Bo; Xu, Xing; Deng, Lianfu; Yang, Chunhao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5370-5381</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting RANKL/RANK offers the possibility of developing novel therapeutic approaches to treat bone metabolic diseases.  Multiple efforts have been made to inhibit RANKL.  For example, marketed monoclonal antibody drug Denosumab could inhibit the maturation of osteoclasts by binding to RANKL.  This study is an original approach aimed at discovering small-mol. inhibitors impeding RANKL/RANK protein interaction.  We identified compd. 34 as a potent and selective RANKL/RANK inhibitor by performing structure-based virtual screening and hit optimization.  Disruption of the RANKL/RANK interaction by 34 effectively inhibits RANKL-induced osteoclastogenesis and bone resorption.  The expression of osteoclast marker genes was also suppressed by treatment of 34.  Furthermore, 34 markedly blocked the NFATc1/c-fos pathway.  Thus, our current work demonstrates that the chem. tractability of the difficult PPI (RANKL/RANK) target by a small-mol. compd. 34 offers a potential lead compd. to facilitate the development of new medications for bone-related diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrR7OIZWpsVg7Vg90H21EOLACvtfcHk0liD4w0M_TUURg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXptlOjsrw%253D&md5=dd9823a04adff49e69c3ac5780f259b0</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b02027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b02027%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DM.%26aulast%3DPeng%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DT.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DN.%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.%26atitle%3DDevelopment%2520of%2520small-molecules%2520targeting%2520receptor%2520activator%2520of%2520nuclear%2520factor-%25CE%25BAB%2520ligand%2520%2528RANKL%2529-receptor%2520activator%2520of%2520nuclear%2520factor-%25CE%25BAB%2520%2528RANK%2529%2520protein-protein%2520interaction%2520by%2520structure-based%2520virtual%2520screening%2520and%2520hit%2520optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D5370%26epage%3D5381%26doi%3D10.1021%2Facs.jmedchem.8b02027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Negri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rives, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caspers, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prisinzano, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Javitch, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filizola, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novelselective kappa-opioid receptor agonist using crystal structure-based virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">521</span>– <span class="NLM_lpage">526</span>, <span class="refDoi"> DOI: 10.1021/ci400019t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400019t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjsVymurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=521-526&author=A.+Negriauthor=M.-L.+Rivesauthor=M.+J.+Caspersauthor=T.+E.+Prisinzanoauthor=J.+A.+Javitchauthor=M.+Filizola&title=Discovery+of+a+novelselective+kappa-opioid+receptor+agonist+using+crystal+structure-based+virtual+screening&doi=10.1021%2Fci400019t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Novel Selective Kappa-Opioid Receptor Agonist Using Crystal Structure-Based Virtual Screening</span></div><div class="casAuthors">Negri, Ana; Rives, Marie-Laure; Caspers, Michael J.; Prisinzano, Thomas E.; Javitch, Jonathan A.; Filizola, Marta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">521-526</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Kappa-opioid (KOP) receptor agonists exhibit analgesic effects without activating reward pathways.  In the search for nonaddictive opioid therapeutics and novel chem. tools to study physiol. functions regulated by the KOP receptor, we screened in silico its recently released inactive crystal structure.  A selective novel KOP receptor agonist emerged as a notable result and is proposed as a new chemotype for the study of the KOP receptor in the etiol. of drug addiction, depression, and/or pain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk0lEXXU0bnLVg90H21EOLACvtfcHk0liD4w0M_TUURg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjsVymurk%253D&md5=4ace2403c664ba58106b40a61773bf4b</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fci400019t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400019t%26sid%3Dliteratum%253Aachs%26aulast%3DNegri%26aufirst%3DA.%26aulast%3DRives%26aufirst%3DM.-L.%26aulast%3DCaspers%26aufirst%3DM.%2BJ.%26aulast%3DPrisinzano%26aufirst%3DT.%2BE.%26aulast%3DJavitch%26aufirst%3DJ.%2BA.%26aulast%3DFilizola%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520novelselective%2520kappa-opioid%2520receptor%2520agonist%2520using%2520crystal%2520structure-based%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D521%26epage%3D526%26doi%3D10.1021%2Fci400019t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">Comparing protein-ligand docking programs is difficult</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">325</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1002/prot.20497</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1002%2Fprot.20497" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=15937897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvVyrsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2005&pages=325-332&author=J.+C.+Coleauthor=C.+W.+Murrayauthor=J.+W.+M.+Nissinkauthor=R.+D.+Taylorauthor=R.+Taylor&title=Comparing+protein-ligand+docking+programs+is+difficult&doi=10.1002%2Fprot.20497"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Comparing protein-ligand docking programs is difficult</span></div><div class="casAuthors">Cole, Jason C.; Murray, Christopher W.; Nissink, J. Willem M.; Taylor, Richard D.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">325-332</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A review.  There is currently great interest in comparing protein-ligand docking programs.  A review of recent comparisons shows that it is difficult to draw conclusions of general applicability.  Statistical hypothesis testing is required to ensure that differences in pose-prediction success rates and enrichment rates are significant.  Numerical measures such as root-mean-square deviation need careful interpretation and may profitably be supplemented by interaction-based measures and visual inspection of dockings.  Test sets must be of appropriate diversity and of good exptl. reliability.  The effects of crystal-packing interactions may be important.  The method used for generating starting ligand geometries and positions may have an appreciable effect on docking results.  For fair comparison, programs must be given search problems of equal complexity (e.g. binding-site regions of the same size) and approx. equal time in which to solve them.  Comparisons based on rescoring require local optimization of the ligand in the space of the new objective function.  Re-implementations of published scoring functions may give significantly different results from the originals.  Ostensibly minor details in methodol. may have a profound influence on headline success rates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT2WvCG-13PrVg90H21EOLACvtfcHk0liD4w0M_TUURg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmvVyrsbc%253D&md5=08ea962f2c33cfaed1438356c244e688</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fprot.20497&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20497%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DComparing%2520protein-ligand%2520docking%2520programs%2520is%2520difficult%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2005%26volume%3D60%26spage%3D325%26epage%3D332%26doi%3D10.1002%2Fprot.20497" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hartshorn, M. J.</span></span> <span> </span><span class="NLM_article-title">AstexViewerTM †: A visualisation aid for structure-based drug design</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">871</span>– <span class="NLM_lpage">881</span>, <span class="refDoi"> DOI: 10.1023/A:1023813504011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1023%2FA%3A1023813504011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=12825620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A280%3ADC%252BD3s3psl2nsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=871-881&author=M.+J.+Hartshorn&title=AstexViewerTM+%E2%80%A0%3A+A+visualisation+aid+for+structure-based+drug+design&doi=10.1023%2FA%3A1023813504011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">AstexViewer: a visualisation aid for structure-based drug design</span></div><div class="casAuthors">Hartshorn Michael J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of computer-aided molecular design</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">871-81</span>
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    </div><div class="casAbstract">AstexViewer is a Java molecular graphics program that can be used for visualisation in many aspects of structure-based drug design.  This paper describes its functionality, implementation and examples of its use.  The program can run as an Applet in a web browser allowing structures to be displayed without installing additional software.  Applications of its use are described for visualisation and as part of a structure based design platform.  The software is being made freely available to the community and may be downloaded from http://www.astex-technology.com/AstexViewer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSPr0A35nMvHjMsgLhIoICsfW6udTcc2eZlw1TEDTvKErntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3s3psl2nsA%253D%253D&md5=f9396aae9ecf64f256a9f8588f8097bd</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1023%2FA%3A1023813504011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1023813504011%26sid%3Dliteratum%253Aachs%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26atitle%3DAstexViewerTM%2520%25E2%2580%25A0%253A%2520A%2520visualisation%2520aid%2520for%2520structure-based%2520drug%2520design%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2002%26volume%3D16%26spage%3D871%26epage%3D881%26doi%3D10.1023%2FA%3A1023813504011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caflisch, A.</span></span> <span> </span><span class="NLM_article-title">High-throughput virtual screening using quantum mechanical probes: Discovery of selective kinase inhibitors</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1014</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201000085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1002%2Fcmdc.201000085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=20540063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVOmt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=1007-1014&author=T.+Zhouauthor=A.+Caflisch&title=High-throughput+virtual+screening+using+quantum+mechanical+probes%3A+Discovery+of+selective+kinase+inhibitors&doi=10.1002%2Fcmdc.201000085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Virtual Screening Using Quantum Mechanical Probes: Discovery of Selective Kinase Inhibitors</span></div><div class="casAuthors">Zhou, Ting; Caflisch, Amedeo</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1007-1014</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A procedure based on semi-empirical quantum mech. (QM) calcns. of interaction energy is proposed for the rapid screening of compd. poses generated by high-throughput docking.  Small mols. (consisting of 2-10 atoms and termed "probes") are overlapped with polar groups in the binding site of the protein target.  The interaction energy values between each compd. pose and the probes, calcd. by a semi-empirical Hamiltonian, are used as filters.  The QM probe method does not require fixed partial charges and takes into account polarization and charge-transfer effects which are not captured by conventional force fields.  The procedure is applied to screen ∼100 million poses (of 2.7 million com. available compds.) obtained by high-throughput docking in the ATP binding site of the tyrosine kinase erythropoietin-producing human hepatocellular carcinoma receptor B4 (EphB4).  Three QM probes on the hinge region and one at the entrance pocket are employed to select for binding affinity, while a QM probe on the side chain of the so-called gatekeeper residue (a hypervariable residue in the kinome) is used to enforce selectivity.  The poses with favorable interactions with the five QM probes are filtered further for hydrophobic matching and low ligand strain.  In this way, a single-digit micromolar inhibitor of EphB4 with a relatively good selectivity profile is identified in a multimillion-compd. library upon exptl. tests of only 23 mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopen4977ct_LVg90H21EOLACvtfcHk0ljyrxE-JQxqAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVOmt7o%253D&md5=090c23ffcdf3d19a73eb7637973721d4</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201000085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201000085%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DCaflisch%26aufirst%3DA.%26atitle%3DHigh-throughput%2520virtual%2520screening%2520using%2520quantum%2520mechanical%2520probes%253A%2520Discovery%2520of%2520selective%2520kinase%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2010%26volume%3D5%26spage%3D1007%26epage%3D1014%26doi%3D10.1002%2Fcmdc.201000085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, L.</span></span> <span> </span><span class="NLM_article-title">Decision making in medicinal chemistry: The power of our intuition</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">956</span>– <span class="NLM_lpage">958</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00359</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00359" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1Orur7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=956-958&author=L.+Gomez&title=Decision+making+in+medicinal+chemistry%3A+The+power+of+our+intuition&doi=10.1021%2Facsmedchemlett.8b00359"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Decision Making in Medicinal Chemistry: The Power of Our Intuition</span></div><div class="casAuthors">Gomez, Laurent</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">956-958</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Medicinal chemists rely on their intuition to make decisions regarding the course of a medicinal chem. program.  Our ability to accurately and efficiently process large data sets routinely requires that we reduce the vol. of information to manageable proportions.  This prioritization process, however, can be affected by intuitive biases.  One such situation is structure-activity relationship (SAR) anal. in nonadditive data sets in which attempts to intuitively predict the activity of compds. based on preliminary data can lead to erroneous conclusions.  Matrix anal. can be a useful tool to accurately det. the nature of the SAR and to improve our decision-making process during an analoging campaign.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNYEcXQuaZhrVg90H21EOLACvtfcHk0ljyrxE-JQxqAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1Orur7N&md5=e37470769559afca2691e0a0ee15bae2</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00359%26sid%3Dliteratum%253Aachs%26aulast%3DGomez%26aufirst%3DL.%26atitle%3DDecision%2520making%2520in%2520medicinal%2520chemistry%253A%2520The%2520power%2520of%2520our%2520intuition%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D956%26epage%3D958%26doi%3D10.1021%2Facsmedchemlett.8b00359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenbaum, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fung, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of beta2-adrenergic receptor ligands</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">6843</span>– <span class="NLM_lpage">6848</span>, <span class="refDoi"> DOI: 10.1073/pnas.0812657106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1073%2Fpnas.0812657106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=19342484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsV2qsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=6843-6848&author=P.+Kolbauthor=D.+M.+Rosenbaumauthor=J.+J.+Irwinauthor=J.+J.+Fungauthor=B.+K.+Kobilkaauthor=B.+K.+Shoichet&title=Structure-based+discovery+of+beta2-adrenergic+receptor+ligands&doi=10.1073%2Fpnas.0812657106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of β2-adrenergic receptor ligands</span></div><div class="casAuthors">Kolb, Peter; Rosenbaum, Daniel M.; Irwin, John J.; Fung, Juan Jose; Kobilka, Brian K.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6843-6848</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Aminergic G protein-coupled receptors (GPCRs) have been a major focus of pharmaceutical research for many years.  Due partly to the lack of reliable receptor structures, drug discovery efforts have been largely ligand-based.  The recently detd. X-ray structure of the β2-adrenergic receptor offers an opportunity to investigate the advantages and limitations inherent in a structure-based approach to ligand discovery against this and related GPCR targets.  Approx. 1 million com. available, "lead-like" mols. were docked against the β2-adrenergic receptor structure.  On testing of 25 high-ranking mols., 6 were active with binding affinities <4 μM, with the best mol. binding with a Ki of 9 nM (95% confidence interval 7-10 nM).  Five of these mols. were inverse agonists.  The high hit rate, the high affinity of the most potent mol., the discovery of unprecedented chemotypes among the new inhibitors, and the apparent bias toward inverse agonists among the docking hits, have implications for structure-based approaches against GPCRs that recognize small org. mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo89kM84Li_H7Vg90H21EOLACvtfcHk0ljyrxE-JQxqAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsV2qsro%253D&md5=4d1a4cb2aa3925aa4c99c6b0496417a7</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0812657106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0812657106%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFung%26aufirst%3DJ.%2BJ.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520discovery%2520of%2520beta2-adrenergic%2520receptor%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D6843%26epage%3D6848%26doi%3D10.1073%2Fpnas.0812657106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirohama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K. Y. J.</span></span> <span> </span><span class="NLM_article-title">Identification of sumoylation activating enzyme 1 inhibitors by structure-based virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">809</span>– <span class="NLM_lpage">820</span>, <span class="refDoi"> DOI: 10.1021/ci300618e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300618e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3sXkvF2jsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=809-820&author=A.+Kumarauthor=A.+Itoauthor=M.+Hirohamaauthor=M.+Yoshidaauthor=K.+Y.+J.+Zhang&title=Identification+of+sumoylation+activating+enzyme+1+inhibitors+by+structure-based+virtual+screening&doi=10.1021%2Fci300618e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Sumoylation Activating Enzyme 1 Inhibitors by Structure-Based Virtual Screening</span></div><div class="casAuthors">Kumar, Ashutosh; Ito, Akihiro; Hirohama, Mikako; Yoshida, Minoru; Zhang, Kam Y. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">809-820</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SUMO activating enzyme 1 (SUMO E1) is responsible for the activation of SUMO in the first step of the sumoylation cascade.  SUMO E1 is linked to many human diseases including cancer, thus making it a potential therapeutic target.  There are few reported SUMO E1 inhibitors including several natural products.  To identify small mol. inhibitors of SUMO E1 with better drug-like properties for potential therapeutic studies, we have used structure-based virtual screening to identify hits from the Maybridge small mol. library for biol. assay.  Our virtual screening protocol involves fast docking of the entire small mol. library with rigid protein and ligands followed by redocking of top hits using a method that incorporates both ligand and protein flexibility.  Subsequently, the top-ranking compds. were prioritized using the mol. dynamics simulation-based binding free energy calcn.  Out of 24 compds. that were acquired and tested using in vitro sumoylation assay, four of them showed more than 85% inhibition of sumoylation with the most active compd. showing an IC50 of 14.4 μM.  A similarity search with the most active compd. in the ZINC database has identified three more compds. with improved potency.  These compds. share a common Ph urea scaffold and have been confirmed to inhibit SUMO E1 by in vitro SUMO-1 thioester bond formation assay.  Our study suggests that these Ph urea compds. could be used as a starting point for the development of novel therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquQUhv65EkgbVg90H21EOLACvtfcHk0lgWKgHOA_M5aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXkvF2jsr0%253D&md5=98ccbbc8ea92d0dd60ab176966e68d97</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fci300618e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300618e%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DIto%26aufirst%3DA.%26aulast%3DHirohama%26aufirst%3DM.%26aulast%3DYoshida%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DK.%2BY.%2BJ.%26atitle%3DIdentification%2520of%2520sumoylation%2520activating%2520enzyme%25201%2520inhibitors%2520by%2520structure-based%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D809%26epage%3D820%26doi%3D10.1021%2Fci300618e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackey, M. D.</span></span> <span> </span><span class="NLM_article-title">Electrostatic complementarity as a fast and effective tool to optimize binding and selectivity of protein-ligand complexes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">3036</span>– <span class="NLM_lpage">3050</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01925</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01925" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXks1ChsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3036-3050&author=M.+R.+Bauerauthor=M.+D.+Mackey&title=Electrostatic+complementarity+as+a+fast+and+effective+tool+to+optimize+binding+and+selectivity+of+protein-ligand+complexes&doi=10.1021%2Facs.jmedchem.8b01925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Electrostatic Complementarity as a Fast and Effective Tool to Optimize Binding and Selectivity of Protein-Ligand Complexes</span></div><div class="casAuthors">Bauer, Matthias R.; Mackey, Mark D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3036-3050</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Electrostatic interactions between small mols. and their resp. receptors are essential for mol. recognition and are also key contributors to the binding free energy.  Assessing the electrostatic match of protein-ligand complexes therefore provides important insights into why ligands bind and what can be changed to improve binding.  Ideally, the ligand and protein electrostatic potentials at the protein-ligand interaction interface should maximize their complementarity while minimizing desolvation penalties.  In this work, we present a fast and efficient tool to calc. and visualize the electrostatic complementarity (EC) of protein-ligand complexes.  We compiled benchmark sets demonstrating electrostatically driven structure-activity relationships (SAR) from literature data, including kinase, protein-protein interaction, and GPCR targets, and used these to demonstrate that the EC method can visualize, rationalize, and predict electrostatically driven ligand affinity changes and help to predict compd. selectivity.  The methodol. presented here for the anal. of EC is a powerful and versatile tool for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3pdUzJ9T0trVg90H21EOLACvtfcHk0lgWKgHOA_M5aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXks1ChsL8%253D&md5=a1abb5abe7fa8d73bee264bc94fd2da1</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01925%26sid%3Dliteratum%253Aachs%26aulast%3DBauer%26aufirst%3DM.%2BR.%26aulast%3DMackey%26aufirst%3DM.%2BD.%26atitle%3DElectrostatic%2520complementarity%2520as%2520a%2520fast%2520and%2520effective%2520tool%2520to%2520optimize%2520binding%2520and%2520selectivity%2520of%2520protein-ligand%2520complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D3036%26epage%3D3050%26doi%3D10.1021%2Facs.jmedchem.8b01925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span>; <span class="NLM_string-name">Hu, B.</span>; <span class="NLM_string-name">Lill, M. A.</span></span> In  <i>WATsite2.0 with PyMOL plugin: Hydration site prediction and visualization BT - protein function prediction: Methods and protocols</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihara, D.</span></span>, Ed.; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2017</span>; pp  <span class="NLM_fpage">123</span>– <span class="NLM_lpage">134</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1007%2F978-1-4939-7015-5_10" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=123-134&author=Y.+Yang&author=B.+Hu&author=M.+A.+Lillauthor=D.+Kihara&title=WATsite2.0+with+PyMOL+plugin%3A+Hydration+site+prediction+and+visualization+BT+-+protein+function+prediction%3A+Methods+and+protocols"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-7015-5_10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-4939-7015-5_10%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%26btitle%3DWATsite2.0%2520with%2520PyMOL%2520plugin%253A%2520Hydration%2520site%2520prediction%2520and%2520visualization%2520BT%2520-%2520protein%2520function%2520prediction%253A%2520Methods%2520and%2520protocols%26aulast%3DKihara%26aufirst%3DD.%26pub%3DSpringer%26date%3D2017%26spage%3D123%26epage%3D134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liebeschuetz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennemann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span> <span> </span><span class="NLM_article-title">The good, the bad and the twisted: A survey of ligand geometry in protein crystal structures</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">183</span>, <span class="refDoi"> DOI: 10.1007/s10822-011-9538-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1007%2Fs10822-011-9538-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=22246295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjt1eitb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=169-183&author=J.+Liebeschuetzauthor=J.+Hennemannauthor=T.+Olssonauthor=C.+R.+Groom&title=The+good%2C+the+bad+and+the+twisted%3A+A+survey+of+ligand+geometry+in+protein+crystal+structures&doi=10.1007%2Fs10822-011-9538-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The good, the bad and the twisted: a survey of ligand geometry in protein crystal structures</span></div><div class="casAuthors">Liebeschuetz, John; Hennemann, Jana; Olsson, Tjelvar; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">169-183</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The protein databank now contains the structures of over 11,000 ligands bound to proteins.  These structures are invaluable in applied areas such as structure-based drug design, but are also the substrate for understanding the energetics of intermol. interactions with proteins.  Despite their obvious importance, the careful anal. of ligands bound to protein structures lags behind the anal. of the protein structures themselves.  We present an anal. of the geometry of ligands bound to proteins and highlight the role of small mol. crystal structures in enabling mol. modellers to critically evaluate a ligand model's quality and investigate protein-induced strain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpceotnqHSI-rVg90H21EOLACvtfcHk0lgWKgHOA_M5aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjt1eitb0%253D&md5=dd85bf6e63910bc85db95509b97c018a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9538-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9538-6%26sid%3Dliteratum%253Aachs%26aulast%3DLiebeschuetz%26aufirst%3DJ.%26aulast%3DHennemann%26aufirst%3DJ.%26aulast%3DOlsson%26aufirst%3DT.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DThe%2520good%252C%2520the%2520bad%2520and%2520the%2520twisted%253A%2520A%2520survey%2520of%2520ligand%2520geometry%2520in%2520protein%2520crystal%2520structures%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D169%26epage%3D183%26doi%3D10.1007%2Fs10822-011-9538-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, C. D.</span>; <span class="NLM_string-name">Nowak, T.</span>; <span class="NLM_string-name">Watson, M. J.</span></span> In  <i>Chapter 2: Measurement interpretation and use of free ligand solution conformations in drug discovery</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawton, G.</span></span>, et al., Eds.; <span class="NLM_publisher-name">Elsevier</span>, <span class="NLM_year">2016</span>; Vol.  <span class="NLM_volume">55</span>, pp  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">147</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=45-147&author=C.+D.+Blundell&author=T.+Nowak&author=M.+J.+Watsonauthor=G.+Lawton&title=Chapter+2%3A+Measurement+interpretation+and+use+of+free+ligand+solution+conformations+in+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DBlundell%26aufirst%3DC.%2BD.%26btitle%3DChapter%25202%253A%2520Measurement%2520interpretation%2520and%2520use%2520of%2520free%2520ligand%2520solution%2520conformations%2520in%2520drug%2520discovery%26aulast%3DLawton%26aufirst%3DG.%26pub%3DElsevier%26date%3D2016%26volume%3D55%26spage%3D45%26epage%3D147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kubinyi, H.</span></span> <span> </span><span class="NLM_article-title">Hydrogen bonding: The last mystery in drug design?</span>. <i>Pharmacokinetic Optimization in Drug Research</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_fpage">513</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.1002/9783906390437.ch28</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1002%2F9783906390437.ch28" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&pages=513-524&author=H.+Kubinyi&title=Hydrogen+bonding%3A+The+last+mystery+in+drug+design%3F&doi=10.1002%2F9783906390437.ch28"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2F9783906390437.ch28&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F9783906390437.ch28%26sid%3Dliteratum%253Aachs%26aulast%3DKubinyi%26aufirst%3DH.%26atitle%3DHydrogen%2520bonding%253A%2520The%2520last%2520mystery%2520in%2520drug%2520design%253F%26jtitle%3DPharmacokinetic%2520Optimization%2520in%2520Drug%2520Research%26date%3D2007%26spage%3D513%26epage%3D524%26doi%3D10.1002%2F9783906390437.ch28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agnihotri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirohi, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahasrabuddhe, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratap, J. V.</span></span> <span> </span><span class="NLM_article-title">Identification of novel inhibitors of Leishmania donovani γ-glutamylcysteine synthetase using structure-based virtual screening, docking, molecular dynamics simulation, and in vitro studies</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">815</span>– <span class="NLM_lpage">825</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00642</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00642" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksFals7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=815-825&author=P.+Agnihotriauthor=A.+K.+Mishraauthor=S.+Mishraauthor=V.+K.+Sirohiauthor=A.+A.+Sahasrabuddheauthor=J.+V.+Pratap&title=Identification+of+novel+inhibitors+of+Leishmania+donovani+%CE%B3-glutamylcysteine+synthetase+using+structure-based+virtual+screening%2C+docking%2C+molecular+dynamics+simulation%2C+and+in+vitro+studies&doi=10.1021%2Facs.jcim.6b00642"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Novel Inhibitors of Leishmania donovani γ-Glutamylcysteine Synthetase Using Structure-Based Virtual Screening, Docking, Molecular Dynamics Simulation, and in Vitro Studies</span></div><div class="casAuthors">Agnihotri, Pragati; Mishra, Arjun K.; Mishra, Shikha; Sirohi, Vijay Kumar; Sahasrabuddhe, Amogh A.; Pratap, J. Venkatesh</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">815-825</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Trypansomatids maintain their redox balance by the trypanothione based redox system, enzymes of which exhibit differences from mammalian homologues.  γ-Glutamylcysteine synthetase (Gcs) is an essential enzyme of this pathway performing the first and rate limiting step.  L-buthionine-S,R-sulfoximine (BSO), a specific inhibitor of Gcs induces toxicity in hosts infected with T. brucei underlining for novel Gcs inhibitors.  The present study reports identification of Ld Gcs inhibitors using computational approaches and their exptl. validation.  Mol. dynamic simulation studies were used to provide better insight to the predicted docked conformation.  Five compds. selected for in vitro anal. have shown significantly higher inhibition of LdGcs ATPase activity as compared to BSO.  These compds. have shown 40-60 % inhibition of L. donovani promastigote proliferation in 100 μM concn. with three compds. showing no toxicity in human HEK293 cell lines.  Anal. of inhibitor-LdGcs complexes shows modifications that could result in increased efficacy of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGyiSfUcoNeLVg90H21EOLACvtfcHk0lhQ3R4L7P4hZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksFals7w%253D&md5=37bee20a8d71ec992b198f89a3189587</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00642&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00642%26sid%3Dliteratum%253Aachs%26aulast%3DAgnihotri%26aufirst%3DP.%26aulast%3DMishra%26aufirst%3DA.%2BK.%26aulast%3DMishra%26aufirst%3DS.%26aulast%3DSirohi%26aufirst%3DV.%2BK.%26aulast%3DSahasrabuddhe%26aufirst%3DA.%2BA.%26aulast%3DPratap%26aufirst%3DJ.%2BV.%26atitle%3DIdentification%2520of%2520novel%2520inhibitors%2520of%2520Leishmania%2520donovani%2520%25CE%25B3-glutamylcysteine%2520synthetase%2520using%2520structure-based%2520virtual%2520screening%252C%2520docking%252C%2520molecular%2520dynamics%2520simulation%252C%2520and%2520in%2520vitro%2520studies%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D815%26epage%3D825%26doi%3D10.1021%2Facs.jcim.6b00642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arisawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamada, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizutani, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomioka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel aldose reductase inhibitors using a protein structure-based approach: 3D-database search followed by design and synthesis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1718</span>– <span class="NLM_lpage">1728</span>, <span class="refDoi"> DOI: 10.1021/jm000483h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm000483h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD3MXivFynu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=1718-1728&author=Y.+Iwataauthor=M.+Arisawaauthor=R.+Hamadaauthor=Y.+Kitaauthor=M.+Y.+Mizutaniauthor=N.+Tomiokaauthor=A.+Itaiauthor=S.+Miyamoto&title=Discovery+of+novel+aldose+reductase+inhibitors+using+a+protein+structure-based+approach%3A+3D-database+search+followed+by+design+and+synthesis&doi=10.1021%2Fjm000483h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Aldose Reductase Inhibitors Using a Protein Structure-Based Approach: 3D-Database Search Followed by Design and Synthesis</span></div><div class="casAuthors">Iwata, Yoriko; Arisawa, Mitsuhiro; Hamada, Ryuji; Kita, Yasuyuki; Mizutani, Miho Y.; Tomioka, Nobuo; Itai, Akiko; Miyamoto, Shuichi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1718-1728</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aldose reductase (AR) has been implicated in the etiol. of diabetic complications.  Due to the limited no. of currently available drugs for the treatment of diabetic complications, we have carried out structure-based drug design and synthesis to find new types of AR inhibitors.  With the ADAM&EVE program, a three-dimensional database (ACD3D) was searched using the ligand binding site of the AR crystal structure.  Out of 179 compds. selected through this search followed by visual inspection, 36 compds. were purchased and subjected to a biol. assay.  Ten compds. showed more than 40% inhibition of AR at a 15 μg/mL concn.  In a subsequent lead optimization, a series of analogs of the most active compd. were synthesized based on the docking mode derived by ADAM&EVE.  Many of these congeners exhibited higher activities compared to the mother compd. Indeed, the most potent, synthesized compd. showed a ∼20-fold increase in inhibitory activity (IC50 = 0.21 vs. 4.3 μM).  Furthermore, a hydrophobic subsite was newly inferred, which would be useful for the design of inhibitors with improved affinity for AR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6c8cdbn1oQLVg90H21EOLACvtfcHk0lhQ3R4L7P4hZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXivFynu7g%253D&md5=6eff99eefff1e4c4531df078969c73d7</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm000483h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000483h%26sid%3Dliteratum%253Aachs%26aulast%3DIwata%26aufirst%3DY.%26aulast%3DArisawa%26aufirst%3DM.%26aulast%3DHamada%26aufirst%3DR.%26aulast%3DKita%26aufirst%3DY.%26aulast%3DMizutani%26aufirst%3DM.%2BY.%26aulast%3DTomioka%26aufirst%3DN.%26aulast%3DItai%26aufirst%3DA.%26aulast%3DMiyamoto%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520novel%2520aldose%2520reductase%2520inhibitors%2520using%2520a%2520protein%2520structure-based%2520approach%253A%25203D-database%2520search%2520followed%2520by%2520design%2520and%2520synthesis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D1718%26epage%3D1728%26doi%3D10.1021%2Fjm000483h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Da’adoosh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyashita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakaguchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldblum, A.</span></span> <span> </span><span class="NLM_article-title">Computational design of substrate selective inhibition</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">e1007713</span>, <span class="refDoi"> DOI: 10.1371/journal.pcbi.1007713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1371%2Fjournal.pcbi.1007713" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=32196495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXps1Wrtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=e1007713&author=B.+Da%E2%80%99adooshauthor=K.+Kaitoauthor=K.+Miyashitaauthor=M.+Sakaguchiauthor=A.+Goldblum&title=Computational+design+of+substrate+selective+inhibition&doi=10.1371%2Fjournal.pcbi.1007713"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Computational design of substrate selective inhibition</span></div><div class="casAuthors">Da'adoosh, Benny; Kaito, Kon; Miyashita, Keishi; Sakaguchi, Minoru; Goldblum, Amiram</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e1007713/1-e1007713/24</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Most enzymes act on more than a single substrate.  There is frequently a need to block the prodn. of a single pathogenic outcome of enzymic activity on a substrate but to avoid blocking others of its catalytic actions.  Full blocking might cause severe side effects because some products of that catalysis may be vital.  Substrate selectivity is required but not possible to achieve by blocking the catalytic residues of an enzyme.  That is the basis of the need for "Substrate Selective Inhibitors" (SSI), and there are several mols. characterized as SSI.  However, none have yet been designed or discovered by computational methods.  We demonstrate a computational approach to the discovery of Substrate Selective Inhibitors for one enzyme, Prolyl Oligopeptidase (POP) (E.C 3.4.21.26), a serine protease which cleaves small peptides between Pro and other amino acids.  Among those are TSH Releasing Hormone (TRH) and Angiotensin-III (Ang-III), differing in both their binding (Km) and in turnover (kcat).  We used our inhouse "Iterative Stochastic Elimination" (ISE) algorithm and the structure-based "Pharmacophore" approach to construct two models for identifying SSI of POP.  A dataset of ~ 1.8 million com. available mols. was initially reduced to less than 12,000 which were screened by these models to a final set of 20 mols. which were sent for exptl. validation (five random mols. were tested for comparison).  Two mols. out of these 20, one with a high score in the ISE model, the other successful in the pharmacophore model, were confirmed by in vitro measurements.  One is a competitive inhibitor of Ang-III (increases its Km), but non-competitive towards TRH (decreases its Vmax).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLqvWDmk3LwLVg90H21EOLACvtfcHk0lhQ3R4L7P4hZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXps1Wrtro%253D&md5=656c78735db5a05489f2649eba99695a</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1007713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1007713%26sid%3Dliteratum%253Aachs%26aulast%3DDa%25E2%2580%2599adoosh%26aufirst%3DB.%26aulast%3DKaito%26aufirst%3DK.%26aulast%3DMiyashita%26aufirst%3DK.%26aulast%3DSakaguchi%26aufirst%3DM.%26aulast%3DGoldblum%26aufirst%3DA.%26atitle%3DComputational%2520design%2520of%2520substrate%2520selective%2520inhibition%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2020%26volume%3D16%26spage%3De1007713%26doi%3D10.1371%2Fjournal.pcbi.1007713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ribeiro, A. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyzack, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borkakoti, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holliday, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. M.</span></span> <span> </span><span class="NLM_article-title">A global analysis of function and conservation of catalytic residues in enzymes</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>295</i></span>,  <span class="NLM_fpage">314</span>– <span class="NLM_lpage">324</span>, <span class="refDoi"> DOI: 10.1074/jbc.REV119.006289</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1074%2Fjbc.REV119.006289" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=31796628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A280%3ADC%252BB3MfmtV2gtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=295&publication_year=2020&pages=314-324&author=A.+J.+M.+Ribeiroauthor=J.+D.+Tyzackauthor=N.+Borkakotiauthor=G.+L.+Hollidayauthor=J.+M.+Thornton&title=A+global+analysis+of+function+and+conservation+of+catalytic+residues+in+enzymes&doi=10.1074%2Fjbc.REV119.006289"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A global analysis of function and conservation of catalytic residues in enzymes</span></div><div class="casAuthors">Ribeiro Antonio J M; Tyzack Jonathan D; Borkakoti Neera; Holliday Gemma L; Thornton Janet M</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of biological chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">295</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">314-324</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The catalytic residues of an enzyme comprise the amino acids located in the active center responsible for accelerating the enzyme-catalyzed reaction.  These residues lower the activation energy of reactions by performing several catalytic functions.  Decades of enzymology research has established general themes regarding the roles of specific residues in these catalytic reactions, but it has been more difficult to explore these roles in a more systematic way.  Here, we review the data on the catalytic residues of 648 enzymes, as annotated in the Mechanism and Catalytic Site Atlas (M-CSA), and compare our results with those in previous studies.  We structured this analysis around three key properties of the catalytic residues: amino acid type, catalytic function, and sequence conservation in homologous proteins.  As expected, we observed that catalysis is mostly accomplished by a small set of residues performing a limited number of catalytic functions.  Catalytic residues are typically highly conserved, but to a smaller degree in homologues that perform different reactions or are nonenzymes (pseudoenzymes).  Cross-analysis yielded further insights revealing which residues perform particular functions and how often.  We obtained more detailed specificity rules for certain functions by identifying the chemical group upon which the residue acts.  Finally, we show the mutation tolerance of the catalytic residues based on their roles.  The characterization of the catalytic residues, their functions, and conservation, as presented here, is key to understanding the impact of mutations in evolution, disease, and enzyme design.  The tools developed for this analysis are available at the M-CSA website and allow for user specific analysis of the same data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQUFwLhBKwYdu2mnZh2OgS0fW6udTcc2eZkeCLpgq731Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MfmtV2gtg%253D%253D&md5=cee0818e02708f8561f71f5bd8953224</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1074%2Fjbc.REV119.006289&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.REV119.006289%26sid%3Dliteratum%253Aachs%26aulast%3DRibeiro%26aufirst%3DA.%2BJ.%2BM.%26aulast%3DTyzack%26aufirst%3DJ.%2BD.%26aulast%3DBorkakoti%26aufirst%3DN.%26aulast%3DHolliday%26aufirst%3DG.%2BL.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DA%2520global%2520analysis%2520of%2520function%2520and%2520conservation%2520of%2520catalytic%2520residues%2520in%2520enzymes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2020%26volume%3D295%26spage%3D314%26epage%3D324%26doi%3D10.1074%2Fjbc.REV119.006289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devaney, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span> <span> </span><span class="NLM_article-title">Strength of solvent-exposed salt-bridges</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1021/jp982715i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp982715i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADyaK1MXis1Kisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1999&pages=727-736&author=R.+Luoauthor=L.+Davidauthor=H.+Hungauthor=J.+Devaneyauthor=M.+K.+Gilson&title=Strength+of+solvent-exposed+salt-bridges&doi=10.1021%2Fjp982715i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Strength of solvent-exposed salt-bridges</span></div><div class="casAuthors">Luo, Ray; David, Laurent; Hung, Howard; Devaney, Judith; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">727-736</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1089-5647</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A recently developed computer model to study the energetics of solvent-exposed salt-bridges is presented here.  The model uses the "mining min." (MM) method to compute conformational free energies with the CHARMm empirical force and the generalized Born (GB) solvation model.  Satisfactory agreement is obtained in comparison with the measured binding affinities of ion pairs in soln. and with the salt-bridge energetics deduced from studies of salt-bridges in helical peptides.  The calcns. suggest that stabilizing charge-charge interactions in helical peptides do not require well-defined salt-bridge conformations.  This is in agreement with crystallog. studies of charge pairs added to T4 lysozyme by site-directed mutagenesis.  The computer model is also used to make a testable prediction that arginine and phosphotyrosine residues in an (i, i + 4) relationship will form a particularly strong salt-bridge in helical peptides.  The biol. implications of these results are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqC1VmzMk_0sbVg90H21EOLACvtfcHk0liQm4nCL_xfXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXis1Kisw%253D%253D&md5=0dad345d3147f4051cb4d59a4195dec1</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjp982715i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp982715i%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DR.%26aulast%3DDavid%26aufirst%3DL.%26aulast%3DHung%26aufirst%3DH.%26aulast%3DDevaney%26aufirst%3DJ.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DStrength%2520of%2520solvent-exposed%2520salt-bridges%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1999%26volume%3D103%26spage%3D727%26epage%3D736%26doi%3D10.1021%2Fjp982715i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira
de Freitas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">A systematic analysis of atomic protein-ligand interactions in the PDB</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1970</span>– <span class="NLM_lpage">1981</span>, <span class="refDoi"> DOI: 10.1039/C7MD00381A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1039%2FC7MD00381A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=29308120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOiu7nE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1970-1981&author=R.+Ferreira%0Ade+Freitasauthor=M.+Schapira&title=A+systematic+analysis+of+atomic+protein-ligand+interactions+in+the+PDB&doi=10.1039%2FC7MD00381A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">A systematic analysis of atomic protein-ligand interactions in the PDB</span></div><div class="casAuthors">Ferreira de Freitas, Renato; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1970-1981</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">As the protein databank (PDB) recently passed the cap of 123 456 structures, it stands more than ever as an important resource not only to analyze structural features of specific biol. systems, but also to study the prevalence of structural patterns obsd. in a large body of unrelated structures, that may reflect rules governing protein folding or mol. recognition.  Here, we compiled a list of 11 016 unique structures of small-mol. ligands bound to proteins - 6444 of which have exptl. binding affinity - representing 750 873 protein-ligand at. interactions, and analyzed the frequency, geometry and impact of each interaction type.  We find that hydrophobic interactions are generally enriched in high-efficiency ligands, but polar interactions are over-represented in fragment inhibitors.  While most observations extd. from the PDB will be familiar to seasoned medicinal chemists, less expected findings, such as the high no. of C-H···O hydrogen bonds or the relatively frequent amide-p stacking between the backbone amide of proteins and arom. rings of ligands, uncover underused ligand design strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8iIbfKGlrqbVg90H21EOLACvtfcHk0liQm4nCL_xfXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOiu7nE&md5=04688488e73253f76e4ea384456f89a8</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1039%2FC7MD00381A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7MD00381A%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%2Bde%2BFreitas%26aufirst%3DR.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DA%2520systematic%2520analysis%2520of%2520atomic%2520protein-ligand%2520interactions%2520in%2520the%2520PDB%26jtitle%3DMedChemComm%26date%3D2017%26volume%3D8%26spage%3D1970%26epage%3D1981%26doi%3D10.1039%2FC7MD00381A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldner, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Grafenstein, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liedl, K. R.</span></span> <span> </span><span class="NLM_article-title">Independent metrics for protein backbone and side-chain flexibility: Time scales and effects of ligand binding</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">851</span>– <span class="NLM_lpage">860</span>, <span class="refDoi"> DOI: 10.1021/ct500633u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct500633u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2MXislSmsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=851-860&author=J.+E.+Fuchsauthor=B.+J.+Waldnerauthor=R.+G.+Huberauthor=S.+von+Grafensteinauthor=C.+Kramerauthor=K.+R.+Liedl&title=Independent+metrics+for+protein+backbone+and+side-chain+flexibility%3A+Time+scales+and+effects+of+ligand+binding&doi=10.1021%2Fct500633u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Independent Metrics for Protein Backbone and Side-Chain Flexibility: Time Scales and Effects of Ligand Binding</span></div><div class="casAuthors">Fuchs, Julian E.; Waldner, Birgit J.; Huber, Roland G.; von Grafenstein, Susanne; Kramer, Christian; Liedl, Klaus R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">851-860</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conformational dynamics are central for understanding biomol. structure and function, since biol. macromols. are inherently flexible at room temp. and in soln.  Computational methods are nowadays capable of providing valuable information on the conformational ensembles of biomols.  However, anal. tools and intuitive metrics that capture dynamic information from in silico generated structural ensembles are limited.  In std. work-flows, flexibility in a conformational ensemble is represented through residue-wise root-mean-square fluctuations or B-factors following a global alignment.  Consequently, these approaches relying on global alignments discard valuable information on local dynamics.  Results inherently depend on global flexibility, residue size, and connectivity.  In this study we present a novel approach for capturing positional fluctuations based on multiple local alignments instead of one single global alignment.  The method captures local dynamics within a structural ensemble independent of residue type by splitting individual local and global degrees of freedom of protein backbone and side-chains.  Dependence on residue type and size in the side-chains is removed via normalization with the B-factors of the isolated residue.  As a test case, we demonstrate its application to a mol. dynamics simulation of bovine pancreatic trypsin inhibitor (BPTI) on the millisecond time scale.  This allows for illustrating different time scales of backbone and side-chain flexibility.  Addnl., we demonstrate the effects of ligand binding on side-chain flexibility of three serine proteases.  We expect our new methodol. for quantifying local flexibility to be helpful in unraveling local changes in biomol. dynamics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro4WAfZQnkVbVg90H21EOLACvtfcHk0liQm4nCL_xfXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXislSmsr8%253D&md5=b16802904f58e97867c4f0c6360dbc6c</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fct500633u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct500633u%26sid%3Dliteratum%253Aachs%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3DWaldner%26aufirst%3DB.%2BJ.%26aulast%3DHuber%26aufirst%3DR.%2BG.%26aulast%3Dvon%2BGrafenstein%26aufirst%3DS.%26aulast%3DKramer%26aufirst%3DC.%26aulast%3DLiedl%26aufirst%3DK.%2BR.%26atitle%3DIndependent%2520metrics%2520for%2520protein%2520backbone%2520and%2520side-chain%2520flexibility%253A%2520Time%2520scales%2520and%2520effects%2520of%2520ligand%2520binding%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D851%26epage%3D860%26doi%3D10.1021%2Fct500633u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caballero, J.</span></span> <span> </span><span class="NLM_article-title">Considerations for Docking of Selective Angiotensin-Converting Enzyme Inhibitors</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">295</span>, <span class="refDoi"> DOI: 10.3390/molecules25020295</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.3390%2Fmolecules25020295" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjslCisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=295&author=J.+Caballero&title=Considerations+for+Docking+of+Selective+Angiotensin-Converting+Enzyme+Inhibitors&doi=10.3390%2Fmolecules25020295"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Considerations for docking of selective angiotensin-converting enzyme inhibitors</span></div><div class="casAuthors">Caballero, Julio</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">295</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  The angiotensin-converting enzyme (ACE) is a two-domain dipeptidylcarboxypeptidase, which has a direct involvement in the control of blood pressure by performing the hydrolysis of angiotensin I to produce angiotensin II.  At the same time, ACE hydrolyzes other substrates such as the vasodilator peptide bradykinin and the anti-inflammatory peptide N-acetyl-SDKP.  In this sense, ACE inhibitors are bioactive substances with potential use as medicinal products for treatment or prevention of hypertension, heart failures, myocardial infarction, and other important diseases.  This review examd. the most recent literature reporting ACE inhibitors with the help of mol. modeling.  The examples exposed here demonstrate that mol. modeling methods, including docking, mol. dynamics (MD) simulations, quant. structure-activity relationship (QSAR), etc, are essential for a complete structural picture of the mode of action of ACE inhibitors, where mol. docking has a key role.  Examples show that too many works identified ACE inhibitory activities of natural peptides and peptides obtained from hydrolyzates.  In addn., other works report non-peptide compds. extd. from natural sources and synthetic compds.  In all these cases, mol. docking was used to provide explanation of the chem. interactions between inhibitors and the ACE binding sites.  For docking applications, most of the examples exposed here do not consider that: (i) ACE has two domains (nACE and cACE) with available X-ray structures, which are relevant for the design of selective inhibitors, and (ii) nACE and cACE binding sites have large dimensions, which leads to non-reliable solns. during docking calcns.  In support of the soln. of these problems, the structural information found in Protein Data Bank (PDB) was used to perform an interaction fingerprints (IFPs) anal. applied on both nACE and cACE domains.  This anal. provides plots that identify the chem. interactions between ligands and both ACE binding sites, which can be used to guide docking expts. in the search of selective natural components or novel drugs.  In addn., the use of hydrogen bond constraints in the S2 and S2' subsites of nACE and cACE are suggested to guarantee that docking solns. are reliable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC-X-Fv63o0rVg90H21EOLACvtfcHk0liDy4oUkyEPeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjslCisrs%253D&md5=e9756d0d0631a3efd7c0b8c2007cd2de</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.3390%2Fmolecules25020295&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules25020295%26sid%3Dliteratum%253Aachs%26aulast%3DCaballero%26aufirst%3DJ.%26atitle%3DConsiderations%2520for%2520Docking%2520of%2520Selective%2520Angiotensin-Converting%2520Enzyme%2520Inhibitors%26jtitle%3DMolecules%26date%3D2020%26volume%3D25%26spage%3D295%26doi%3D10.3390%2Fmolecules25020295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kannan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melesina, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, A.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrabarti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heimburg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidtkunz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, W.</span></span> <span> </span><span class="NLM_article-title">Discovery of inhibitors of Schistosoma mansoni HDAC8 by combining homology modeling, virtual screening, and in vitro validation</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">3005</span>– <span class="NLM_lpage">3019</span>, <span class="refDoi"> DOI: 10.1021/ci5004653</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci5004653" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFygtLvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=3005-3019&author=S.+Kannanauthor=J.+Melesinaauthor=A.-T.+Hauserauthor=A.+Chakrabartiauthor=T.+Heimburgauthor=K.+Schmidtkunzauthor=A.+Walterauthor=M.+Marekauthor=R.+J.+Pierceauthor=C.+Romierauthor=M.+Jungauthor=W.+Sippl&title=Discovery+of+inhibitors+of+Schistosoma+mansoni+HDAC8+by+combining+homology+modeling%2C+virtual+screening%2C+and+in+vitro+validation&doi=10.1021%2Fci5004653"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Inhibitors of Schistosoma mansoni HDAC8 by Combining Homology Modeling, Virtual Screening, and in Vitro Validation</span></div><div class="casAuthors">Kannan, Srinivasaraghavan; Melesina, Jelena; Hauser, Alexander-Thomas; Chakrabarti, Alokta; Heimburg, Tino; Schmidtkunz, Karin; Walter, Alexandra; Marek, Martin; Pierce, Raymond J.; Romier, Christophe; Jung, Manfred; Sippl, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3005-3019</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Schistosomiasis, caused by S. mansoni, is a tropical disease that affects over 200 million people worldwide.  A novel approach for targeting eukaryotic parasites is to tackle their dynamic epigenetic machinery that is necessary for the extensive phenotypic changes during their life cycle.  We recently identified S. mansoni histone deacetylase 8 (smHDAC8) as a potential target for antiparasitic therapy.  Here we present results from a virtual screening campaign on smHDAC8.  Besides hydroxamates, several sulfonamide-thiazole derivs. were identified by a target-based virtual screening using a homol. model of smHDAC8.  In vitro testing of 75 compds. identified 8 hydroxamates as potent and lead-like inhibitors of the parasitic HDAC8.  Solving of the crystal structure of smHDAC8 with two of the virtual screening hits confirmed the predicted binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoH_psd1ffmerVg90H21EOLACvtfcHk0liDy4oUkyEPeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFygtLvO&md5=512119ba4e5cbcdd0d28570a3acb74b5</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fci5004653&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci5004653%26sid%3Dliteratum%253Aachs%26aulast%3DKannan%26aufirst%3DS.%26aulast%3DMelesina%26aufirst%3DJ.%26aulast%3DHauser%26aufirst%3DA.-T.%26aulast%3DChakrabarti%26aufirst%3DA.%26aulast%3DHeimburg%26aufirst%3DT.%26aulast%3DSchmidtkunz%26aufirst%3DK.%26aulast%3DWalter%26aufirst%3DA.%26aulast%3DMarek%26aufirst%3DM.%26aulast%3DPierce%26aufirst%3DR.%2BJ.%26aulast%3DRomier%26aufirst%3DC.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DSippl%26aufirst%3DW.%26atitle%3DDiscovery%2520of%2520inhibitors%2520of%2520Schistosoma%2520mansoni%2520HDAC8%2520by%2520combining%2520homology%2520modeling%252C%2520virtual%2520screening%252C%2520and%2520in%2520vitro%2520validation%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D3005%26epage%3D3019%26doi%3D10.1021%2Fci5004653" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Masters, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahmoud, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lill, M. A.</span></span> <span> </span><span class="NLM_article-title">Efficient and accurate hydration site profiling for enclosed binding sites</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2183</span>– <span class="NLM_lpage">2188</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.8b00544</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00544" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVGjtrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=2183-2188&author=M.+R.+Mastersauthor=A.+H.+Mahmoudauthor=Y.+Yangauthor=M.+A.+Lill&title=Efficient+and+accurate+hydration+site+profiling+for+enclosed+binding+sites&doi=10.1021%2Facs.jcim.8b00544"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient and Accurate Hydration Site Profiling for Enclosed Binding Sites</span></div><div class="casAuthors">Masters, Matthew R.; Mahmoud, Amr H.; Yang, Ying; Lill, Markus A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2183-2188</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. dynamics (MD) simulations allow for accurate prediction of the thermodn. profile of binding-site water mols. crit. for protein-ligand assocn.  Whereas this hydration-site profiling converges rapidly for solvent-exposed sites independent of the initial water placement, an accurate and reliable placement is required for water mols. in occluded binding sites.  Here, we present an accurate and efficient hydration-site prediction method for occluded binding sites combining water placement based on 3D-RISM and MD simulations using WATsite.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvVWTQeN-bSbVg90H21EOLACvtfcHk0liDy4oUkyEPeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVGjtrrL&md5=c7bcfbc9538595ee2ad2b8c2778ab85f</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00544&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00544%26sid%3Dliteratum%253Aachs%26aulast%3DMasters%26aufirst%3DM.%2BR.%26aulast%3DMahmoud%26aufirst%3DA.%2BH.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DLill%26aufirst%3DM.%2BA.%26atitle%3DEfficient%2520and%2520accurate%2520hydration%2520site%2520profiling%2520for%2520enclosed%2520binding%2520sites%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2018%26volume%3D58%26spage%3D2183%26epage%3D2188%26doi%3D10.1021%2Facs.jcim.8b00544" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Role of the active-site solvent in the thermodynamics of factor Xa ligand binding</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">2817</span>– <span class="NLM_lpage">2831</span>, <span class="refDoi"> DOI: 10.1021/ja0771033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0771033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=2817-2831&author=R.+Abelauthor=T.+Youngauthor=R.+Faridauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Role+of+the+active-site+solvent+in+the+thermodynamics+of+factor+Xa+ligand+binding&doi=10.1021%2Fja0771033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the Active-Site Solvent in the Thermodynamics of Factor Xa Ligand Binding</span></div><div class="casAuthors">Abel, Robert; Young, Tom; Farid, Ramy; Berne, Bruce J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2817-2831</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding the underlying physics of the binding of small-mol. ligands to protein active sites is a key objective of computational chem. and biol.  It is widely believed that displacement of water mols. from the active site by the ligand is a principal (if not the dominant) source of binding free energy.  Although continuum theories of hydration are routinely used to describe the contributions of the solvent to the binding affinity of the complex, it is still an unsettled question as to whether or not these continuum solvation theories describe the underlying mol. physics with sufficient accuracy to reliably rank the binding affinities of a set of ligands for a given protein.  Here we develop a novel, computationally efficient descriptor of the contribution of the solvent to the binding free energy of a small mol. and its assocd. receptor that captures the effects of the ligand displacing the solvent from the protein active site with at. detail.  This descriptor quant. predicts (R2 = 0.81) the binding free energy differences between congeneric ligand pairs for the test system factor Xa, elucidates phys. properties of the active site solvent that appear to be missing in most continuum theories of hydration, and identifies several features of the hydration of the factor Xa active site relevant to the structure-activity relationship of its inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS42r59nNXH7Vg90H21EOLACvtfcHk0lj9Etg7ZUwdLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D&md5=8eefe76547a82bccaea38f2c84e6cd5f</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fja0771033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0771033%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DRole%2520of%2520the%2520active-site%2520solvent%2520in%2520the%2520thermodynamics%2520of%2520factor%2520Xa%2520ligand%2520binding%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D2817%26epage%3D2831%26doi%3D10.1021%2Fja0771033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mahmoud, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masters, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lill, M. A.</span></span> <span> </span><span class="NLM_article-title">Elucidating the multiple roles of hydration for accurate protein-ligand binding prediction via deep learning</span>. <i>Commun. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">19</span>, <span class="refDoi"> DOI: 10.1038/s42004-020-0261-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1038%2Fs42004-020-0261-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktl2hsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2020&pages=19&author=A.+H.+Mahmoudauthor=M.+R.+Mastersauthor=Y.+Yangauthor=M.+A.+Lill&title=Elucidating+the+multiple+roles+of+hydration+for+accurate+protein-ligand+binding+prediction+via+deep+learning&doi=10.1038%2Fs42004-020-0261-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Elucidating the multiple roles of hydration for accurate protein-ligand binding prediction via deep learning</span></div><div class="casAuthors">Mahmoud, Amr H.; Masters, Matthew R.; Yang, Ying; Lill, Markus A.</div><div class="citationInfo"><span class="NLM_cas:title">Communications Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19</span>CODEN:
                <span class="NLM_cas:coden">CCOHCT</span>;
        ISSN:<span class="NLM_cas:issn">2399-3669</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Accurate and efficient prediction of protein-ligand interactions has been a long-lasting dream of practitioners in drug discovery.  The insufficient treatment of hydration is widely recognized to be a major limitation for accurate protein-ligand scoring.  Using an integration of mol. dynamics simulations on thousands of protein structures with novel big-data analytics based on convolutional neural networks and deep Taylor decompn., we consistently identify here three different patterns of hydration to be essential for protein-ligand interactions.  In addn. to desolvation and water-mediated interactions, the formation of enthalpically favorable networks of first-shell water mols. around solvent-exposed ligand moieties is identified to be essential for protein-ligand binding.  Despite being currently neglected in drug discovery, this hydration phenomenon could lead to new avenues in optimizing the free energy of ligand binding.  Application of deep neural networks incorporating hydration to docking provides 89% accuracy in binding pose ranking, an essential step for rational structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlxlrtqYVzOrVg90H21EOLACvtfcHk0lj9Etg7ZUwdLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktl2hsbs%253D&md5=a99707c1848b372d4031d221e460a3c0</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fs42004-020-0261-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42004-020-0261-x%26sid%3Dliteratum%253Aachs%26aulast%3DMahmoud%26aufirst%3DA.%2BH.%26aulast%3DMasters%26aufirst%3DM.%2BR.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DLill%26aufirst%3DM.%2BA.%26atitle%3DElucidating%2520the%2520multiple%2520roles%2520of%2520hydration%2520for%2520accurate%2520protein-ligand%2520binding%2520prediction%2520via%2520deep%2520learning%26jtitle%3DCommun.%2520Chem.%26date%3D2020%26volume%3D3%26spage%3D19%26doi%3D10.1038%2Fs42004-020-0261-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaudreault, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chartier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najmanovich, R.</span></span> <span> </span><span class="NLM_article-title">Side-chain rotamer changes upon ligand binding: Common, crucial, correlate with entropy and rearrange hydrogen bonding</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">i423</span>– <span class="NLM_lpage">i430</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/bts395</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1093%2Fbioinformatics%2Fbts395" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=22962462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlantLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=i423-i430&author=F.+Gaudreaultauthor=M.+Chartierauthor=R.+Najmanovich&title=Side-chain+rotamer+changes+upon+ligand+binding%3A+Common%2C+crucial%2C+correlate+with+entropy+and+rearrange+hydrogen+bonding&doi=10.1093%2Fbioinformatics%2Fbts395"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Side-chain rotamer changes upon ligand binding: Common, crucial, correlate with entropy and rearrange hydrogen bonding</span></div><div class="casAuthors">Gaudreault, Francis; Chartier, Matthieu; Najmanovich, Rafael</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">i423-i430</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Protein movements form a continuum from large domain rearrangements (including folding and restructuring) to side-chain rotamer changes and small rearrangements.  Understanding side-chain flexibility upon binding is important for understanding mol. recognition events and predicting ligand binding.  Methods: In the present work, we developed a well-curated non-redundant dataset of 188 proteins in pairs of structures in the Apo (unbound) and Holo (bound) forms to study the extent and the factors that guide side-chain rotamer changes upon binding.  Results: Our anal. shows that side-chain rotamer changes are widespread with only 10% of binding sites displaying no conformational changes.  Overall, at most five rotamer changes account for the obsd. movements in 90% of the cases.  Furthermore, rotamer changes are essential in 32% of flexible binding sites.  The different amino acids have a 11-fold difference in their probability to undergo changes.  Side-chain flexibility represents an intrinsic property of amino acids as it correlates well with configurational entropy differences.  Furthermore, on av. b-factors and solvent accessible surface areas can discriminate flexible side-chains in the Apo form.  Finally, there is a rearrangement of the hydrogen-bonding network upon binding primarily with a loss of H-bonds with water mols. and a gain of H-bonds with protein residues for flexible residues.  Interestingly, only 25% of side chains capable of forming H-bonds do so with the ligand upon binding.  In terms of drug design, this last result shows that there is a large no. of potential interactions that may be exploited to modulate the specificity and sensitivity of inhibitors.  Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="ec9e8d8a8d8980c2828d86818d82839a858f84ac9f84899e8e9e83838789c28f8dc2">[email protected]</a></div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoerwRd3_qiYrVg90H21EOLACvtfcHk0lj9Etg7ZUwdLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlantLfO&md5=1cb775e36034683efd17908930e4639d</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbts395&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbts395%26sid%3Dliteratum%253Aachs%26aulast%3DGaudreault%26aufirst%3DF.%26aulast%3DChartier%26aufirst%3DM.%26aulast%3DNajmanovich%26aufirst%3DR.%26atitle%3DSide-chain%2520rotamer%2520changes%2520upon%2520ligand%2520binding%253A%2520Common%252C%2520crucial%252C%2520correlate%2520with%2520entropy%2520and%2520rearrange%2520hydrogen%2520bonding%26jtitle%3DBioinformatics%26date%3D2012%26volume%3D28%26spage%3Di423%26epage%3Di430%26doi%3D10.1093%2Fbioinformatics%2Fbts395" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ravindranathan, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandiyan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekkati, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bae, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel fibroblast growth factor receptor 1 kinase inhibitors by structure-based virtual screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1662</span>– <span class="NLM_lpage">1672</span>, <span class="refDoi"> DOI: 10.1021/jm901386e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901386e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlKkt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=1662-1672&author=K.+P.+Ravindranathanauthor=V.+Mandiyanauthor=A.+R.+Ekkatiauthor=J.+H.+Baeauthor=J.+Schlessingerauthor=W.+L.+Jorgensen&title=Discovery+of+novel+fibroblast+growth+factor+receptor+1+kinase+inhibitors+by+structure-based+virtual+screening&doi=10.1021%2Fjm901386e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Fibroblast Growth Factor Receptor 1 Kinase Inhibitors by Structure-Based Virtual Screening</span></div><div class="casAuthors">Ravindranathan, Krishna P.; Mandiyan, Valsan; Ekkati, Anil R.; Bae, Jae H.; Schlessinger, Joseph; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1662-1672</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fibroblast growth factors (FGFs) play important roles in embryonic development, angiogenesis, wound healing, and cell proliferation and differentiation.  In search of inhibitors of FGFR1 kinase, 2.2 million compds. were docked into the ATP binding site of the protein.  A co-crystal structure, which shows two alternative conformations for the nucleotide binding loop, is reported.  Docking was performed on both conformations and, ultimately, 23 diverse compds. were purchased and assayed.  Following hit validation, two compds. 10 and 16, a benzylidene deriv. of pseudothiohydantoin and a thienopyrimidinone deriv., resp., were discovered that inhibit FGFR1 kinase with IC50 values of 23 and 50 μM.  Initial optimization of 16 led to the more unsatd. 40 (I), which has significantly enhanced potency, 1.9 μM.  The core structures represent new structural motifs for FGFR1 kinase inhibitors.  The study also illustrates complexities assocd. with the choice of protein structures for docking, possible use of multiple kinase structures to seek selectivity, and hit identification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaes0izfiD1rVg90H21EOLACvtfcHk0lhUPRY7vBRTfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlKkt78%253D&md5=1edb7b1e76644f21899024906aabbb43</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fjm901386e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901386e%26sid%3Dliteratum%253Aachs%26aulast%3DRavindranathan%26aufirst%3DK.%2BP.%26aulast%3DMandiyan%26aufirst%3DV.%26aulast%3DEkkati%26aufirst%3DA.%2BR.%26aulast%3DBae%26aufirst%3DJ.%2BH.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DDiscovery%2520of%2520novel%2520fibroblast%2520growth%2520factor%2520receptor%25201%2520kinase%2520inhibitors%2520by%2520structure-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D1662%26epage%3D1672%26doi%3D10.1021%2Fjm901386e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cournia, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandavadi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bucala, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span> <span> </span><span class="NLM_article-title">Discovery of human macrophage migration inhibitory factor (MIF)-CD74 antagonists via virtual screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">416</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.1021/jm801100v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801100v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFaisrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=416-424&author=Z.+Courniaauthor=L.+Lengauthor=S.+Gandavadiauthor=X.+Duauthor=R.+Bucalaauthor=W.+L.+Jorgensen&title=Discovery+of+human+macrophage+migration+inhibitory+factor+%28MIF%29-CD74+antagonists+via+virtual+screening&doi=10.1021%2Fjm801100v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Human Macrophage Migration Inhibitory Factor (MIF)-CD74 Antagonists via Virtual Screening</span></div><div class="casAuthors">Cournia, Zoe; Leng, Lin; Gandavadi, Sunilkumar; Du, Xin; Bucala, Richard; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">416-424</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Macrophage migration inhibitory factor (MIF) is a cytokine that is involved in the regulation of inflammation as well as cell proliferation and differentiation.  Deactivation of MIF by antibodies or inhibition of MIF binding to its receptor, CD74, attenuates tumor growth and angiogenesis.  To discover small-mol. inhibitors of MIF's biol. activity, virtual screening was performed by docking 2.1 million compds. into the MIF tautomerase active site.  After visual inspection of 1200 top-ranked MIF-ligand complexes, 26 possible inhibitors were selected and purchased and 23 of them were assayed.  The in vitro binding assay for MIF with CD74 revealed that 11 of the compds. have inhibitory activity in the micromolar regime, including four compds. with IC50 values below 5 μM.  Inhibition of MIF tautomerase activity was also established for many of the compds. with IC50 values as low as 0.5 μM; Michaelis-Menten anal. was performed for two cases and confirmed the competitive inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWMPIUdUO7TLVg90H21EOLACvtfcHk0lhUPRY7vBRTfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFaisrzF&md5=56f32f68318413b1f1f8a6d141aefb68</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjm801100v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801100v%26sid%3Dliteratum%253Aachs%26aulast%3DCournia%26aufirst%3DZ.%26aulast%3DLeng%26aufirst%3DL.%26aulast%3DGandavadi%26aufirst%3DS.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DBucala%26aufirst%3DR.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DDiscovery%2520of%2520human%2520macrophage%2520migration%2520inhibitory%2520factor%2520%2528MIF%2529-CD74%2520antagonists%2520via%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D416%26epage%3D424%26doi%3D10.1021%2Fjm801100v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dayhoff, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchenko, A. R.</span></span> <span> </span><span class="NLM_article-title">Evolution of protein binding modes in homooligomers</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>395</i></span>,  <span class="NLM_fpage">860</span>– <span class="NLM_lpage">870</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2009.10.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1016%2Fj.jmb.2009.10.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=19879880" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntlyisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=395&publication_year=2010&pages=860-870&author=J.+E.+Dayhoffauthor=B.+A.+Shoemakerauthor=S.+H.+Bryantauthor=A.+R.+Panchenko&title=Evolution+of+protein+binding+modes+in+homooligomers&doi=10.1016%2Fj.jmb.2009.10.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Evolution of Protein Binding Modes in Homooligomers</span></div><div class="casAuthors">Dayhoff, Judith E.; Shoemaker, Benjamin A.; Bryant, Stephen H.; Panchenko, Anna R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">395</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">860-870</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The evolution of protein interactions cannot be deciphered without a detailed anal. of interaction interfaces and binding modes.  We performed a large-scale study of protein homooligomers in terms of their symmetry, interface sizes, and conservation of binding modes.  We also focused specifically on the evolution of protein binding modes from nine families of homooligomers and mapped 60 different binding modes and oligomerization states onto the phylogenetic trees of these families.  We obsd. a significant tendency for the same binding modes to be clustered together and conserved within clades on phylogenetic trees; this trend is esp. pronounced for close homologs with 70% sequence identity or higher.  Some binding modes are conserved among very distant homologs, pointing to their ancient evolutionary origin, while others are very specific for a certain phylogenetic group.  Moreover, we found that the most ancient binding modes have a tendency to involve sym. (isologous) homodimer binding arrangements with larger interfaces, while recently evolved binding modes more often exhibit asym. arrangements and smaller interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKqIRe6N3n5bVg90H21EOLACvtfcHk0lhUPRY7vBRTfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntlyisQ%253D%253D&md5=192444c54d01efa6903bf0af8fe92df2</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2009.10.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2009.10.052%26sid%3Dliteratum%253Aachs%26aulast%3DDayhoff%26aufirst%3DJ.%2BE.%26aulast%3DShoemaker%26aufirst%3DB.%2BA.%26aulast%3DBryant%26aufirst%3DS.%2BH.%26aulast%3DPanchenko%26aufirst%3DA.%2BR.%26atitle%3DEvolution%2520of%2520protein%2520binding%2520modes%2520in%2520homooligomers%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D395%26spage%3D860%26epage%3D870%26doi%3D10.1016%2Fj.jmb.2009.10.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kasahara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinoshita, K.</span></span> <span> </span><span class="NLM_article-title">Landscape of protein-small ligand binding modes</span>. <i>Protein science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1659</span>– <span class="NLM_lpage">1671</span>, <span class="refDoi"> DOI: 10.1002/pro.2971</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1002%2Fpro.2971" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=27327045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSiurzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2016&pages=1659-1671&author=K.+Kasaharaauthor=K.+Kinoshita&title=Landscape+of+protein-small+ligand+binding+modes&doi=10.1002%2Fpro.2971"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Landscape of protein-small ligand binding modes</span></div><div class="casAuthors">Kasahara, Kota; Kinoshita, Kengo</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1659-1671</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Elucidating the mechanisms of specific small-mol. (ligand) recognition by proteins is a long-standing conundrum.  While the structures of these mols., proteins and ligands, have been extensively studied, protein-ligand interactions, or binding modes, have not been comprehensively analyzed.  Although methods for assessing similarities of binding site structures have been extensively developed, the methods for the computational treatment of binding modes have not been well established.  Here, we developed a computational method for encoding the information about binding modes as graphs, and assessing their similarities.  An all-against-all comparison of 20,040 protein-ligand complexes provided the landscape of the protein-ligand binding modes and its relationships with protein- and chem. spaces.  While similar proteins in the same SCOP Family tend to bind relatively similar ligands with similar binding modes, the correlation between ligand and binding similarities was not very high (R2 = 0.443).  We found many pairs with novel relationships, in which two evolutionally distant proteins recognize dissimilar ligands by similar binding modes (757,474 pairs out of 200,790,780 pairs were categorized into this relationship, in our dataset).  In addn., there were an abundance of pairs of homologous proteins binding to similar ligands with different binding modes (68,217 pairs).  Our results showed that many interesting relationships between protein-ligand complexes are still hidden in the structure database, and our new method for assessing binding mode similarities is effective to find them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD4kKAT8zhJrVg90H21EOLACvtfcHk0lgEC_QSAilDtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSiurzP&md5=eb3b5974ebc6c56573c75af3d8ad2037</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2Fpro.2971&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.2971%26sid%3Dliteratum%253Aachs%26aulast%3DKasahara%26aufirst%3DK.%26aulast%3DKinoshita%26aufirst%3DK.%26atitle%3DLandscape%2520of%2520protein-small%2520ligand%2520binding%2520modes%26jtitle%3DProtein%2520science%26date%3D2016%26volume%3D25%26spage%3D1659%26epage%3D1671%26doi%3D10.1002%2Fpro.2971" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Damm-Ganamet, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunbar, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, H. A.</span></span> <span> </span><span class="NLM_article-title">CSAR benchmark exercise 2011–2012: Evaluation of results from docking and relative ranking of blinded congeneric series</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1853</span>– <span class="NLM_lpage">1870</span>, <span class="refDoi"> DOI: 10.1021/ci400025f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400025f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltV2msbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=1853-1870&author=K.+L.+Damm-Ganametauthor=R.+D.+Smithauthor=J.+B.+Dunbarauthor=J.+A.+Stuckeyauthor=H.+A.+Carlson&title=CSAR+benchmark+exercise+2011%E2%80%932012%3A+Evaluation+of+results+from+docking+and+relative+ranking+of+blinded+congeneric+series&doi=10.1021%2Fci400025f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">CSAR Benchmark Exercise 2011-2012: Evaluation of Results from Docking and Relative Ranking of Blinded Congeneric Series</span></div><div class="casAuthors">Damm-Ganamet, Kelly L.; Smith, Richard D.; Dunbar, James B.; Stuckey, Jeanne A.; Carlson, Heather A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1853-1870</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Community Structure-Activity Resource (CSAR) recently held its first blinded exercise based on data provided by Abbott, Vertex, and colleagues at the University of Michigan, Ann Arbor.  A total of 20 research groups submitted results for the benchmark exercise where the goal was to compare different improvements for pose prediction, enrichment, and relative ranking of congeneric series of compds.  The exercise was built around blinded high-quality exptl. data from four protein targets: LpxC, Urokinase, Chk1, and Erk2.  Pose prediction proved to be the most straightforward task, and most methods were able to successfully reproduce binding poses when the crystal structure employed was co-crystd. with a ligand from the same chem. series.  Multiple evaluation metrics were examd., and we found that RMSD and native contact metrics together provide a robust evaluation of the predicted poses.  It was notable that most scoring functions underpredicted contacts between the hetero atoms (i.e., N, O, S, etc.) of the protein and ligand.  Relative ranking was found to be the most difficult area for the methods, but many of the scoring functions were able to properly identify Urokinase actives from the inactives in the series.  Lastly, we found that minimizing the protein and correcting histidine tautomeric states pos. trended with low RMSD for pose prediction but minimizing the ligand neg. trended.  Pregenerated ligand conformations performed better than those that were generated on the fly.  Optimizing docking parameters and pretraining with the native ligand had a pos. effect on the docking performance as did using restraints, substructure fitting, and shape fitting.  Lastly, for both sampling and ranking scoring functions, the use of the empirical scoring function appeared to trend pos. with the RMSD.  Here, by combining the results of many methods, we hope to provide a statistically relevant evaluation and elucidate specific shortcomings of docking methodol. for the community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVoKpI9PikibVg90H21EOLACvtfcHk0lgEC_QSAilDtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltV2msbg%253D&md5=80f080270d091cee06ca9eb31345144e</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fci400025f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400025f%26sid%3Dliteratum%253Aachs%26aulast%3DDamm-Ganamet%26aufirst%3DK.%2BL.%26aulast%3DSmith%26aufirst%3DR.%2BD.%26aulast%3DDunbar%26aufirst%3DJ.%2BB.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DCSAR%2520benchmark%2520exercise%25202011%25E2%2580%25932012%253A%2520Evaluation%2520of%2520results%2520from%2520docking%2520and%2520relative%2520ranking%2520of%2520blinded%2520congeneric%2520series%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D1853%26epage%3D1870%26doi%3D10.1021%2Fci400025f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hevener, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehboob, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, P.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truong, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boci, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghassemi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, M. E.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel and potent class of F. tularensis enoyl-reductase (FabI) inhibitors by molecular shape and electrostatic matching</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">268</span>– <span class="NLM_lpage">279</span>, <span class="refDoi"> DOI: 10.1021/jm201168g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201168g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsV2jurbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=268-279&author=K.+E.+Hevenerauthor=S.+Mehboobauthor=P.-C.+Suauthor=K.+Truongauthor=T.+Bociauthor=J.+Dengauthor=M.+Ghassemiauthor=J.+L.+Cookauthor=M.+E.+Johnson&title=Discovery+of+a+novel+and+potent+class+of+F.+tularensis+enoyl-reductase+%28FabI%29+inhibitors+by+molecular+shape+and+electrostatic+matching&doi=10.1021%2Fjm201168g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Novel and Potent Class of F. tularensis Enoyl-Reductase (FabI) Inhibitors by Molecular Shape and Electrostatic Matching</span></div><div class="casAuthors">Hevener, Kirk E.; Mehboob, Shahila; Su, Pin-Chih; Truong, Kent; Boci, Teuta; Deng, Jiangping; Ghassemi, Mahmood; Cook, James L.; Johnson, Michael E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">268-279</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enoyl-acyl carrier protein (ACP) reductase, FabI, is a key enzyme in the bacterial fatty acid biosynthesis pathway (FAS II).  FabI is an NADH-dependent oxidoreductase that acts to reduce enoyl-ACP substrates in a final step of the pathway.  The absence of this enzyme in humans makes it an attractive target for the development of new antibacterial agents.  FabI is known to be unresponsive to structure-based design efforts due to a high degree of induced fit and a mobile flexible loop encompassing the active site.  Here we discuss the development, validation, and careful application of a ligand-based virtual screen used for the identification of novel inhibitors of the Francisella tularensis FabI target.  In this study, four known classes of FabI inhibitors were used as templates for virtual screens that involved mol. shape and electrostatic matching.  The program ROCS was used to search a high-throughput screening library for compds. that matched any of the four mol. shape queries.  Matching compds. were further refined using the program EON, which compares and scores compds. by matching electrostatic properties.  Using these techniques, 50 compds. were selected, ordered, and tested.  The tested compds. possessed novel chem. scaffolds when compared to the input query compds.  Several hits with low micromolar activity were identified and follow-up scaffold-based searches resulted in the identification of a lead series with submicromolar enzyme inhibition, high ligand efficiency, and a novel scaffold.  Addnl., one of the most active compds. showed promising whole-cell antibacterial activity against several Gram-pos. and Gram-neg. species, including the target pathogen.  The results of a preliminary structure-activity relationship anal. are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEn7n0O_iCD7Vg90H21EOLACvtfcHk0lgEC_QSAilDtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsV2jurbO&md5=c348cb0264e4f9d740991a15c7973252</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fjm201168g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201168g%26sid%3Dliteratum%253Aachs%26aulast%3DHevener%26aufirst%3DK.%2BE.%26aulast%3DMehboob%26aufirst%3DS.%26aulast%3DSu%26aufirst%3DP.-C.%26aulast%3DTruong%26aufirst%3DK.%26aulast%3DBoci%26aufirst%3DT.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DGhassemi%26aufirst%3DM.%26aulast%3DCook%26aufirst%3DJ.%2BL.%26aulast%3DJohnson%26aufirst%3DM.%2BE.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520and%2520potent%2520class%2520of%2520F.%2520tularensis%2520enoyl-reductase%2520%2528FabI%2529%2520inhibitors%2520by%2520molecular%2520shape%2520and%2520electrostatic%2520matching%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D268%26epage%3D279%26doi%3D10.1021%2Fjm201168g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwivedi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahasrabuddhe, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqi, M. I.</span></span> <span> </span><span class="NLM_article-title">Integrated machine learning, molecular docking and 3D-QSAR based approach for identification of potential inhibitors of trypanosomal N-myristoyltransferase</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">3711</span>– <span class="NLM_lpage">3723</span>, <span class="refDoi"> DOI: 10.1039/C6MB00574H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1039%2FC6MB00574H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=27766319" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1GqtbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=3711-3723&author=N.+Singhauthor=P.+Shahauthor=H.+Dwivediauthor=S.+Mishraauthor=R.+Tripathiauthor=A.+A.+Sahasrabuddheauthor=M.+I.+Siddiqi&title=Integrated+machine+learning%2C+molecular+docking+and+3D-QSAR+based+approach+for+identification+of+potential+inhibitors+of+trypanosomal+N-myristoyltransferase&doi=10.1039%2FC6MB00574H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Integrated machine learning, molecular docking and 3D-QSAR based approach for identification of potential inhibitors of trypanosomal N-myristoyltransferase</span></div><div class="casAuthors">Singh, Nidhi; Shah, Priyanka; Dwivedi, Hemlata; Mishra, Shikha; Tripathi, Renu; Sahasrabuddhe, Amogh A.; Siddiqi, Mohammad Imran</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3711-3723</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In the present study, a machine learning approach, docking studies and CoMFA anal. have been integrated with the objective of translation of knowledge into a pipelined workflow towards the identification of putative hits through the screening of large compd. libraries.  In the proposed pipeline, the reported parasitic NMT inhibitors have been used to develop predictive machine learning classification models.  Simultaneously, a TbNMT complex model was generated to establish the relationship between the binding mode of the inhibitors for LmNMT and TbNMT through mol. dynamics simulation studies.  A 3D-QSAR model was developed and used to predict the activity of the proposed hits in the subsequent step.  The hits classified as active based on the machine learning model were assessed as the potential anti-trypanosomal NMT inhibitors through mol. docking studies, predicted activity using a QSAR model and visual inspection.  In the final step, the proposed pipeline was validated through in vitro expts.  A total of seven hits have been proposed and tested in vitro for evaluation of dual inhibitory activity against Leishmania donovani and Trypanosoma brucei.  Out of these five compds. showed significant inhibition against both of the organisms.  The common topmost active compd. SEW04173 belongs to a pyrazole carboxylate scaffold and is anticipated to enrich the chem. space with enhanced potency through optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC1rsgog_YdrVg90H21EOLACvtfcHk0lh0N7xxyHFS2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1GqtbbN&md5=69693da42315b852f285b58394313d7f</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1039%2FC6MB00574H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MB00574H%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DN.%26aulast%3DShah%26aufirst%3DP.%26aulast%3DDwivedi%26aufirst%3DH.%26aulast%3DMishra%26aufirst%3DS.%26aulast%3DTripathi%26aufirst%3DR.%26aulast%3DSahasrabuddhe%26aufirst%3DA.%2BA.%26aulast%3DSiddiqi%26aufirst%3DM.%2BI.%26atitle%3DIntegrated%2520machine%2520learning%252C%2520molecular%2520docking%2520and%25203D-QSAR%2520based%2520approach%2520for%2520identification%2520of%2520potential%2520inhibitors%2520of%2520trypanosomal%2520N-myristoyltransferase%26jtitle%3DMol.%2520BioSyst.%26date%3D2016%26volume%3D12%26spage%3D3711%26epage%3D3723%26doi%3D10.1039%2FC6MB00574H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaieb, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gathiaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feher, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burley, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaro, R. E.</span></span> <span> </span><span class="NLM_article-title">D3R grand challenge 2: Blind prediction of protein-ligand poses, affinity rankings, and relative binding free energies</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1007/s10822-017-0088-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1007%2Fs10822-017-0088-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=29204945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFWgtLfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=1-20&author=Z.+Gaiebauthor=S.+Liuauthor=S.+Gathiakaauthor=M.+Chiuauthor=H.+Yangauthor=C.+Shaoauthor=V.+A.+Feherauthor=W.+P.+Waltersauthor=B.+Kuhnauthor=M.+G.+Rudolphauthor=S.+K.+Burleyauthor=M.+K.+Gilsonauthor=R.+E.+Amaro&title=D3R+grand+challenge+2%3A+Blind+prediction+of+protein-ligand+poses%2C+affinity+rankings%2C+and+relative+binding+free+energies&doi=10.1007%2Fs10822-017-0088-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">D3R Grand Challenge 2: blind prediction of protein-ligand poses, affinity rankings, and relative binding free energies</span></div><div class="casAuthors">Gaieb, Zied; Liu, Shuai; Gathiaka, Symon; Chiu, Michael; Yang, Huanwang; Shao, Chenghua; Feher, Victoria A.; Walters, W. Patrick; Kuhn, Bernd; Rudolph, Markus G.; Burley, Stephen K.; Gilson, Michael K.; Amaro, Rommie E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-20</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The Drug Design Data Resource (D3R) ran Grand Challenge 2 (GC2) from Sept. 2016 through Feb. 2017.  This challenge was based on a dataset of structures and affinities for the nuclear receptor farnesoid X receptor (FXR), contributed by F. Hoffman-La Roche.  The dataset contained 102 IC50 values, spanning six orders of magnitude, and 36 high-resoln. co-crystal structures with representatives of four major ligand classes.  Strong global participation was evident, with 49 participants submitting 262 prediction submission packages in total.  Procedurally, GC2 mimicked Grand Challenge 2015 (GC2015), with a Stage 1 subchallenge testing ligand pose prediction methods and ranking and scoring methods, and a Stage 2 subchallenge testing only ligand ranking and scoring methods after the release of all blinded co-crystal structures.  Two smaller curated sets of 18 and 15 ligands were developed to test alchem. free energy methods.  This overview summarizes all aspects of GC2, including the dataset details, challenge procedures, and participant results.  We also consider implications for progress in the field, while highlighting methodol. areas that merit continued development.  Similar to GC2015, the outcome of GC2 underscores the pressing need for methods development in pose prediction, particularly for ligand scaffolds not currently represented in the Protein Data Bank (http://www.pdb.org), and in affinity ranking and scoring of bound ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFXlYV_AkMJLVg90H21EOLACvtfcHk0lh0N7xxyHFS2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFWgtLfE&md5=1f8afb98386bf9d6584d63deafb62114</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1007%2Fs10822-017-0088-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-017-0088-4%26sid%3Dliteratum%253Aachs%26aulast%3DGaieb%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DGathiaka%26aufirst%3DS.%26aulast%3DChiu%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DShao%26aufirst%3DC.%26aulast%3DFeher%26aufirst%3DV.%2BA.%26aulast%3DWalters%26aufirst%3DW.%2BP.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DRudolph%26aufirst%3DM.%2BG.%26aulast%3DBurley%26aufirst%3DS.%2BK.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26aulast%3DAmaro%26aufirst%3DR.%2BE.%26atitle%3DD3R%2520grand%2520challenge%25202%253A%2520Blind%2520prediction%2520of%2520protein-ligand%2520poses%252C%2520affinity%2520rankings%252C%2520and%2520relative%2520binding%2520free%2520energies%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2018%26volume%3D32%26spage%3D1%26epage%3D20%26doi%3D10.1007%2Fs10822-017-0088-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gathiaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubish, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunbar, J. B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burley, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaro, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feher, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span> <span> </span><span class="NLM_article-title">D3R grand challenge 2015: Evaluation of protein-ligand pose and affinity predictions</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">651</span>– <span class="NLM_lpage">668</span>, <span class="refDoi"> DOI: 10.1007/s10822-016-9946-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1007%2Fs10822-016-9946-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=27696240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1SlsbzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=651-668&author=S.+Gathiakaauthor=S.+Liuauthor=M.+Chiuauthor=H.+Yangauthor=J.+A.+Stuckeyauthor=Y.+N.+Kangauthor=J.+Delpropostoauthor=G.+Kubishauthor=J.+B.+J.+Dunbarauthor=H.+A.+Carlsonauthor=S.+K.+Burleyauthor=W.+P.+Waltersauthor=R.+E.+Amaroauthor=V.+A.+Feherauthor=M.+K.+Gilson&title=D3R+grand+challenge+2015%3A+Evaluation+of+protein-ligand+pose+and+affinity+predictions&doi=10.1007%2Fs10822-016-9946-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">D3R grand challenge 2015: Evaluation of protein-ligand pose and affinity predictions</span></div><div class="casAuthors">Gathiaka, Symon; Liu, Shuai; Chiu, Michael; Yang, Huanwang; Stuckey, Jeanne A.; Kang, You Na; Delproposto, Jim; Kubish, Ginger; Dunbar, Jr., James B.; Carlson, Heather A.; Burley, Stephen K.; Walters, W. Patrick; Amaro, Rommie E.; Feher, Victoria A.; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">651-668</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The Drug Design Data Resource (D3R) ran Grand Challenge 2015 between Sept. 2015 and Feb. 2016.  Two targets served as the framework to test community docking and scoring methods: (1) HSP90, donated by AbbVie and the Community Structure Activity Resource (CSAR), and (2) MAP4K4, donated by Genentech.  The challenges for both target datasets were conducted in two stages, with the first stage testing pose predictions and the capacity to rank compds. by affinity with minimal structural data; and the second stage testing methods for ranking compds. with knowledge of at least a subset of the ligand-protein poses.  An addnl. sub-challenge provided small groups of chem. similar HSP90 compds. amenable to alchem. calcns. of relative binding free energy.  Unlike previous blinded Challenges, we did not provide cognate receptors or receptors prepd. with hydrogens and likewise did not require a specified crystal structure to be used for pose or affinity prediction in Stage 1.  Given the freedom to select from over 200 crystal structures of HSP90 in the PDB, participants employed workflows that tested not only core docking and scoring technologies, but also methods for addressing water-mediated ligand-protein interactions, binding pocket flexibility, and the optimal selection of protein structures for use in docking calcns.  Nearly 40 participating groups submitted over 350 prediction sets for Grand Challenge 2015.  This overview describes the datasets and the organization of the challenge components, summarizes the results across all submitted predictions, and considers broad conclusions that may be drawn from this collaborative community endeavor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvQmGRFiZZW7Vg90H21EOLACvtfcHk0lh0N7xxyHFS2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1SlsbzN&md5=f9779f61ad8224f7d6bbaaf0cd97bc24</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1007%2Fs10822-016-9946-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-016-9946-8%26sid%3Dliteratum%253Aachs%26aulast%3DGathiaka%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DChiu%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DKang%26aufirst%3DY.%2BN.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DKubish%26aufirst%3DG.%26aulast%3DDunbar%26aufirst%3DJ.%2BB.%2BJ.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26aulast%3DBurley%26aufirst%3DS.%2BK.%26aulast%3DWalters%26aufirst%3DW.%2BP.%26aulast%3DAmaro%26aufirst%3DR.%2BE.%26aulast%3DFeher%26aufirst%3DV.%2BA.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DD3R%2520grand%2520challenge%25202015%253A%2520Evaluation%2520of%2520protein-ligand%2520pose%2520and%2520affinity%2520predictions%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2016%26volume%3D30%26spage%3D651%26epage%3D668%26doi%3D10.1007%2Fs10822-016-9946-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mobley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wymer, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perryman, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">Blind prediction of HIV integrase binding from the SAMPL4 challenge</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">345</span>, <span class="refDoi"> DOI: 10.1007/s10822-014-9723-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1007%2Fs10822-014-9723-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=24595873" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjsVKqu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=327-345&author=D.+L.+Mobleyauthor=S.+Liuauthor=N.+M.+Limauthor=K.+L.+Wymerauthor=A.+L.+Perrymanauthor=S.+Forliauthor=N.+Dengauthor=J.+Suauthor=K.+Bransonauthor=A.+J.+Olson&title=Blind+prediction+of+HIV+integrase+binding+from+the+SAMPL4+challenge&doi=10.1007%2Fs10822-014-9723-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Blind prediction of HIV integrase binding from the SAMPL4 challenge</span></div><div class="casAuthors">Mobley, David L.; Liu, Shuai; Lim, Nathan M.; Wymer, Karisa L.; Perryman, Alexander L.; Forli, Stefano; Deng, Nanjie; Su, Justin; Branson, Kim; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">327-345</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Here, we give an overview of the protein-ligand binding portion of the Statistical Assessment of Modeling of Proteins and Ligands 4 (SAMPL4) challenge, which focused on predicting binding of HIV integrase inhibitors in the catalytic core domain.  The challenge encompassed three components-a small "virtual screening" challenge, a binding mode prediction component, and a small affinity prediction component.  Here, we give summary results and statistics concerning the performance of all submissions at each of these challenges.  Virtual screening was particularly challenging here in part because, in contrast to more typical virtual screening test sets, the inactive compds. were tested because they were thought to be likely binders, so only the very top predictions performed significantly better than random.  Pose prediction was also quite challenging, in part because inhibitors in the set bind to three different sites, so even identifying the correct binding site was challenging.  Still, the best methods managed low root mean squared deviation predictions in many cases.  Here, we give an overview of results, highlight some features of methods which worked particularly well, and refer the interested reader to papers in this issue which describe specific submissions for addnl. details.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPK-7gJzuLe7Vg90H21EOLACvtfcHk0ljMQyackYR0pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjsVKqu7Y%253D&md5=9ec3665496c57b52116768e9de03aefe</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9723-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9723-5%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLim%26aufirst%3DN.%2BM.%26aulast%3DWymer%26aufirst%3DK.%2BL.%26aulast%3DPerryman%26aufirst%3DA.%2BL.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DDeng%26aufirst%3DN.%26aulast%3DSu%26aufirst%3DJ.%26aulast%3DBranson%26aufirst%3DK.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DBlind%2520prediction%2520of%2520HIV%2520integrase%2520binding%2520from%2520the%2520SAMPL4%2520challenge%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2014%26volume%3D28%26spage%3D327%26epage%3D345%26doi%3D10.1007%2Fs10822-014-9723-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perryman, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santiago, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos-Martins, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">Virtual screening with AutoDock Vina and the common pharmacophore engine of a low diversity library of fragments and hits against the three allosteric sites of HIV integrase: Participation in the SAMPL4 protein-ligand binding challenge</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">429</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1007/s10822-014-9709-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1007%2Fs10822-014-9709-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=24493410" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslSmu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=429-441&author=A.+L.+Perrymanauthor=D.+N.+Santiagoauthor=S.+Forliauthor=D.+Santos-Martinsauthor=A.+J.+Olson&title=Virtual+screening+with+AutoDock+Vina+and+the+common+pharmacophore+engine+of+a+low+diversity+library+of+fragments+and+hits+against+the+three+allosteric+sites+of+HIV+integrase%3A+Participation+in+the+SAMPL4+protein-ligand+binding+challenge&doi=10.1007%2Fs10822-014-9709-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening with AutoDock Vina and the common pharmacophore engine of a low diversity library of fragments and hits against the three allosteric sites of HIV integrase: participation in the SAMPL4 protein-ligand binding challenge</span></div><div class="casAuthors">Perryman, Alexander L.; Santiago, Daniel N.; Forli, Stefano; Santos-Martins, Diogo; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">429-441</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">To rigorously assess the tools and protocols that can be used to understand and predict macromol. recognition, and to gain more structural insight into three newly discovered allosteric binding sites on a crit. drug target involved in the treatment of HIV infections, the Olson and Levy labs collaborated on the SAMPL4 challenge.  This computational blind challenge involved predicting protein-ligand binding against the three allosteric sites of HIV integrase (IN), a viral enzyme for which two drugs (that target the active site) have been approved by the FDA.  Pos. control cross-docking expts. were utilized to select 13 receptor models out of an initial ensemble of 41 different crystal structures of HIV IN.  These 13 models of the targets were selected using our new "Rank Difference Ratio" metric.  The first stage of SAMPL4 involved using virtual screens to identify 62 active, allosteric IN inhibitors out of a set of 321 compds.  The second stage involved predicting the binding site(s) and crystallog. binding mode(s) for 57 of these inhibitors.  Our team submitted four entries for the first stage that utilized: (1) AutoDock Vina (AD Vina) plus visual inspection; (2) a new common pharmacophore engine; (3) BEDAM replica exchange free energy simulations, and a Consensus approach that combined the predictions of all three strategies.  Even with the SAMPL4's very challenging compd. library that displayed a significantly lower amt. of structural diversity than most libraries that are conventionally employed in prospective virtual screens, these approaches produced hit rates of 24, 25, 34, and 27 %, resp., on a set with 19 % declared binders.  Our only entry for the second stage challenge was based on the results of AD Vina plus visual inspection, and it ranked third place overall according to several different metrics provided by the SAMPL4 organizers.  The successful results displayed by these approaches highlight the utility of the computational structure-based drug discovery tools and strategies that are being developed to advance the goals of the newly created, multi-institution, NIH-funded center called the "HIV Interaction and Viral Evolution Center".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq58ngCMTIjsrVg90H21EOLACvtfcHk0ljMQyackYR0pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslSmu7s%253D&md5=fb665768bc09c899e1cb33707777692a</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9709-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9709-3%26sid%3Dliteratum%253Aachs%26aulast%3DPerryman%26aufirst%3DA.%2BL.%26aulast%3DSantiago%26aufirst%3DD.%2BN.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DSantos-Martins%26aufirst%3DD.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DVirtual%2520screening%2520with%2520AutoDock%2520Vina%2520and%2520the%2520common%2520pharmacophore%2520engine%2520of%2520a%2520low%2520diversity%2520library%2520of%2520fragments%2520and%2520hits%2520against%2520the%2520three%2520allosteric%2520sites%2520of%2520HIV%2520integrase%253A%2520Participation%2520in%2520the%2520SAMPL4%2520protein-ligand%2520binding%2520challenge%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2014%26volume%3D28%26spage%3D429%26epage%3D441%26doi%3D10.1007%2Fs10822-014-9709-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm-Ganamet, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunbar, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubish, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tinberg, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khare, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doyle, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, H. A.</span></span> <span> </span><span class="NLM_article-title">CSAR benchmark exercise 2013: Evaluation of results from a combined computational protein design, docking, and scoring/ranking challenge</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1022</span>– <span class="NLM_lpage">1031</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00387</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00387" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFKqs7%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1022-1031&author=R.+D.+Smithauthor=K.+L.+Damm-Ganametauthor=J.+B.+Dunbarauthor=A.+Ahmedauthor=K.+Chinnaswamyauthor=J.+E.+Delpropostoauthor=G.+M.+Kubishauthor=C.+E.+Tinbergauthor=S.+D.+Khareauthor=J.+Douauthor=L.+Doyleauthor=J.+A.+Stuckeyauthor=D.+Bakerauthor=H.+A.+Carlson&title=CSAR+benchmark+exercise+2013%3A+Evaluation+of+results+from+a+combined+computational+protein+design%2C+docking%2C+and+scoring%2Franking+challenge&doi=10.1021%2Facs.jcim.5b00387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">CSAR Benchmark Exercise 2013: Evaluation of Results from a Combined Computational Protein Design, Docking, and Scoring/Ranking Challenge</span></div><div class="casAuthors">Smith, Richard D.; Damm-Ganamet, Kelly L.; Dunbar, James B.; Ahmed, Aqeel; Chinnaswamy, Krishnapriya; Delproposto, James E.; Kubish, Ginger M.; Tinberg, Christine E.; Khare, Sagar D.; Dou, Jiayi; Doyle, Lindsey; Stuckey, Jeanne A.; Baker, David; Carlson, Heather A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1031</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Community Structure-Activity Resource (CSAR) conducted a benchmark exercise to evaluate the current computational methods for protein design, ligand docking, and scoring/ranking.  The exercise consisted of three phases.  The first phase required the participants to identify and rank order which designed sequences were able to bind the small mol. digoxigenin.  The second phase challenged the community to select a near-native pose of digoxigenin from a set of decoy poses for two of the designed proteins.  The third phase investigated the ability of current methods to rank/score the binding affinity of 10 related steroids to one of the designed proteins (pKd = 4.1 to 6.7).  We found that 11 of 13 groups were able to correctly select the sequence that bound digoxigenin, with most groups providing the correct three-dimensional structure for the backbone of the protein as well as all atoms of the active-site residues.  Eleven of the 14 groups were able to select the appropriate pose from a set of plausible decoy poses.  The ability to predict abs. binding affinities is still a difficult task, as 8 of 14 groups were able to correlate scores to affinity (Pearson-r > 0.7) of the designed protein for congeneric steroids and only 5 of 14 groups were able to correlate the ranks of the 10 related ligands (Spearman-ρ > 0.7).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKmOuy7Lp9TbVg90H21EOLACvtfcHk0ljSBUV7Mn3JKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFKqs7%252FJ&md5=9dfde47772a4220f9b49c1149c0e68bd</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00387%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DR.%2BD.%26aulast%3DDamm-Ganamet%26aufirst%3DK.%2BL.%26aulast%3DDunbar%26aufirst%3DJ.%2BB.%26aulast%3DAhmed%26aufirst%3DA.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%2BE.%26aulast%3DKubish%26aufirst%3DG.%2BM.%26aulast%3DTinberg%26aufirst%3DC.%2BE.%26aulast%3DKhare%26aufirst%3DS.%2BD.%26aulast%3DDou%26aufirst%3DJ.%26aulast%3DDoyle%26aufirst%3DL.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DBaker%26aufirst%3DD.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DCSAR%2520benchmark%2520exercise%25202013%253A%2520Evaluation%2520of%2520results%2520from%2520a%2520combined%2520computational%2520protein%2520design%252C%2520docking%252C%2520and%2520scoring%252Franking%2520challenge%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D1022%26epage%3D1031%26doi%3D10.1021%2Facs.jcim.5b00387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xing, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolodziej, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurumbail, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neal, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skepner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberds, S. L.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent inhibitors of soluble epoxide hydrolase by combinatorial library design and structure-based virtual screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1211</span>– <span class="NLM_lpage">1222</span>, <span class="refDoi"> DOI: 10.1021/jm101382t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101382t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Cqu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1211-1222&author=L.+Xingauthor=J.+J.+McDonaldauthor=S.+A.+Kolodziejauthor=R.+G.+Kurumbailauthor=J.+M.+Williamsauthor=C.+J.+Warrenauthor=J.+M.+O%E2%80%99Nealauthor=J.+E.+Skepnerauthor=S.+L.+Roberds&title=Discovery+of+potent+inhibitors+of+soluble+epoxide+hydrolase+by+combinatorial+library+design+and+structure-based+virtual+screening&doi=10.1021%2Fjm101382t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent Inhibitors of Soluble Epoxide Hydrolase by Combinatorial Library Design and Structure-Based Virtual Screening</span></div><div class="casAuthors">Xing, Li; McDonald, Joseph J.; Kolodziej, Steve A.; Kurumbail, Ravi G.; Williams, Jennifer L.; Warren, Chad J.; O'Neal, Janet M.; Skepner, Jill E.; Roberds, Steven L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1211-1222</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-based virtual screening was applied to design combinatorial libraries to discover novel and potent sol. epoxide hydrolase (sEH) inhibitors.  X-ray crystal structures revealed unique interactions for a benzoxazole template in addn. to the conserved hydrogen bonds with the catalytic machinery of sEH.  By exploitation of the favorable binding elements, two iterations of library design based on amide coupling were employed, guided principally by the docking results of the enumerated virtual products.  Biol. screening of the libraries demonstrated as high as 90% hit rate, of which over two dozen compds. were single digit nanomolar sEH inhibitors by IC50 detn.  In total the library design and synthesis produced more than 300 submicromolar sEH inhibitors.  In cellular systems consistent activities were demonstrated with biochem. measurements.  The SAR understanding of the benzoxazole template provides valuable insights into discovery of novel sEH inhibitors as therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3tpDmXCJWBLVg90H21EOLACvtfcHk0ljSBUV7Mn3JKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Cqu7s%253D&md5=01775ad103a3a5e153b77157ef184608</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fjm101382t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101382t%26sid%3Dliteratum%253Aachs%26aulast%3DXing%26aufirst%3DL.%26aulast%3DMcDonald%26aufirst%3DJ.%2BJ.%26aulast%3DKolodziej%26aufirst%3DS.%2BA.%26aulast%3DKurumbail%26aufirst%3DR.%2BG.%26aulast%3DWilliams%26aufirst%3DJ.%2BM.%26aulast%3DWarren%26aufirst%3DC.%2BJ.%26aulast%3DO%25E2%2580%2599Neal%26aufirst%3DJ.%2BM.%26aulast%3DSkepner%26aufirst%3DJ.%2BE.%26aulast%3DRoberds%26aufirst%3DS.%2BL.%26atitle%3DDiscovery%2520of%2520potent%2520inhibitors%2520of%2520soluble%2520epoxide%2520hydrolase%2520by%2520combinatorial%2520library%2520design%2520and%2520structure-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1211%26epage%3D1222%26doi%3D10.1021%2Fjm101382t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neranjan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smieško, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lill, M. A.</span></span> <span> </span><span class="NLM_article-title">Potential inhibitors for novel coronavirus protease identified by virtual screening of 606 million compounds</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3626</span>, <span class="refDoi"> DOI: 10.3390/ijms21103626</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.3390%2Fijms21103626" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvF2gs73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2020&pages=3626&author=A.+Fischerauthor=M.+Sellnerauthor=S.+Neranjanauthor=M.+Smie%C5%A1koauthor=M.+A.+Lill&title=Potential+inhibitors+for+novel+coronavirus+protease+identified+by+virtual+screening+of+606+million+compounds&doi=10.3390%2Fijms21103626"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Potential inhibitors for novel coronavirus protease identified by virtual screening of 606 million compounds</span></div><div class="casAuthors">Fischer, Andre; Sellner, Manuel; Neranjan, Santhosh; Smiesko, Martin; Lill, Markus A.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3626</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The rapid outbreak of the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in China followed by its spread around the world poses a serious global concern for public health.  To this date, no specific drugs or vaccines are available to treat SARS-CoV-2 despite its close relation to the SARS-CoV virus that caused a similar epidemic in 2003.  Thus, there remains an urgent need for the identification and development of specific antiviral therapeutics against SARS-CoV-2.  To conquer viral infections, the inhibition of proteases essential for proteolytic processing of viral polyproteins is a conventional therapeutic strategy.  In order to find novel inhibitors, we computationally screened a compd. library of over 606 million compds. for binding at the recently solved crystal structure of the main protease (Mpro) of SARS-CoV-2.  A screening of such a vast chem. space for SARS-CoV-2 Mpro inhibitors has not been reported before.  After shape screening, two docking protocols were applied followed by the detn. of mol. descriptors relevant for pharmacokinetics to narrow down the no. of initial hits.  Next, mol. dynamics simulations were conducted to validate the stability of docked binding modes and comprehensively quantify ligand binding energies.  After evaluation of potential off-target binding, we report a list of 12 purchasable compds., with binding affinity to the target protease that is predicted to be more favorable than that of the cocrystd. peptidomimetic compd.  In order to quickly advise ongoing therapeutic intervention for patients, we evaluated approved antiviral drugs and other protease inhibitors to provide a list of nine compds. for drug repurposing.  Furthermore, we identified the natural compds. (-)-taxifolin and rhamnetin as potential inhibitors of Mpro.  Rhamnetin is already com. available in pharmacies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpxRaxT4yRfrVg90H21EOLACvtfcHk0ljSBUV7Mn3JKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvF2gs73E&md5=dae8bf7445d10f43fe32b4fd409c4c9a</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.3390%2Fijms21103626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms21103626%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DA.%26aulast%3DSellner%26aufirst%3DM.%26aulast%3DNeranjan%26aufirst%3DS.%26aulast%3DSmie%25C5%25A1ko%26aufirst%3DM.%26aulast%3DLill%26aufirst%3DM.%2BA.%26atitle%3DPotential%2520inhibitors%2520for%2520novel%2520coronavirus%2520protease%2520identified%2520by%2520virtual%2520screening%2520of%2520606%2520million%2520compounds%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2020%26volume%3D21%26spage%3D3626%26doi%3D10.3390%2Fijms21103626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Broccatelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span> <span> </span><span class="NLM_article-title">Best of both worlds: On the complementarity of ligand-based and structure-based virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1634</span>– <span class="NLM_lpage">1641</span>, <span class="refDoi"> DOI: 10.1021/ci5001604</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci5001604" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2cXovVWhtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1634-1641&author=F.+Broccatelliauthor=N.+Brown&title=Best+of+both+worlds%3A+On+the+complementarity+of+ligand-based+and+structure-based+virtual+screening&doi=10.1021%2Fci5001604"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Best of Both Worlds: On the Complementarity of Ligand-Based and Structure-Based Virtual Screening</span></div><div class="casAuthors">Broccatelli, Fabio; Brown, Nathan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1634-1641</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual screening with docking is an integral component of drug design, particularly during hit finding phases.  While successful prospective studies of virtual screening exist, it remains a significant challenge to identify best practices a priori due to the many factors that influence the final outcome, including targets, data sets, software, metrics, and expert knowledge of the users.  This study investigates the extent to which ligand-based methods can be applied to improve structure-based methods.  The use of ligand-based methods to modulate the no. of hits identified using the protein-ligand complex and also the diversity of these hits from the crystallog. ligand is discussed.  In this study, 40 CDK2 ligand complexes were used together with two external data sets contg. both actives and inactives from GlaxoSmithKline (GSK) and actives and decoys from the Directory of Useful Decoys (DUD).  Results show how ligand-based modeling can be used to select a more appropriate protein conformation for docking, as well as to assess the reliability of the docking expt.  The time gained by reducing the pool of virtual screening candidates via ligand-based similarity can be invested in more accurate docking procedures, as well as in downstream labor-intensive approaches (e.g., visual inspection) maximizing the use of the chem. and biol. information available.  This provides a framework for mol. modeling scientists that are involved in initiating virtual screening campaigns with practical advice to make best use of the information available to them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmJshEz5T-P7Vg90H21EOLACvtfcHk0lgdGwD8Ci_VNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXovVWhtLY%253D&md5=d26bffbe2f144f37fd698c93c631cda8</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Fci5001604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci5001604%26sid%3Dliteratum%253Aachs%26aulast%3DBroccatelli%26aufirst%3DF.%26aulast%3DBrown%26aufirst%3DN.%26atitle%3DBest%2520of%2520both%2520worlds%253A%2520On%2520the%2520complementarity%2520of%2520ligand-based%2520and%2520structure-based%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1634%26epage%3D1641%26doi%3D10.1021%2Fci5001604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goh, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suen, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yau, M.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, K.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Repurposing registered drugs as antagonists for protease-activated receptor 2</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">2079</span>– <span class="NLM_lpage">2084</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00500</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00500" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1altbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2079-2084&author=W.+Xuauthor=J.+Limauthor=C.-Y.+Gohauthor=J.+Y.+Suenauthor=Y.+Jiangauthor=M.-K.+Yauauthor=K.-C.+Wuauthor=L.+Liuauthor=D.+P.+Fairlie&title=Repurposing+registered+drugs+as+antagonists+for+protease-activated+receptor+2&doi=10.1021%2Facs.jcim.5b00500"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Repurposing Registered Drugs as Antagonists for Protease-Activated Receptor 2</span></div><div class="casAuthors">Xu, Weijun; Lim, Junxian; Goh, Chai-Yeen; Suen, Jacky Y.; Jiang, Yuhong; Yau, Mei-Kwan; Wu, Kai-Chen; Liu, Ligong; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2079-2084</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual screening of a drug database identified Carvedilol, Loratadine, Nefazodone and Astemizole as PAR2 antagonists, after ligand docking and mol. dynamics simulations using a PAR2 homol. model and a putative binding mode of a known PAR2 ligand.  The drugs demonstrated competitive binding and antagonism of calcium mobilization and ERK1/2 phosphorylation in CHO-hPAR2 transfected cells, while inhibiting IL-6 secretion in PAR2 expressing MDA-MB-231 breast cancer cells.  This research highlights opportunities for GPCR hit-finding from FDA-approved drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCObXGReIXv7Vg90H21EOLACvtfcHk0lgdGwD8Ci_VNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1altbvN&md5=6091a430ca2bc69d0f478fb76c120b5e</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00500%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DW.%26aulast%3DLim%26aufirst%3DJ.%26aulast%3DGoh%26aufirst%3DC.-Y.%26aulast%3DSuen%26aufirst%3DJ.%2BY.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DYau%26aufirst%3DM.-K.%26aulast%3DWu%26aufirst%3DK.-C.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DRepurposing%2520registered%2520drugs%2520as%2520antagonists%2520for%2520protease-activated%2520receptor%25202%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2079%26epage%3D2084%26doi%3D10.1021%2Facs.jcim.5b00500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, W.</span></span> <span> </span><span class="NLM_article-title">Molecular modeling of the 3D structure of 5-HT1AR: Discovery of novel 5-HT1AR agonists via dynamic pharmacophore-based virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">3202</span>– <span class="NLM_lpage">3211</span>, <span class="refDoi"> DOI: 10.1021/ci400481p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400481p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslyks7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=3202-3211&author=L.+Xuauthor=S.+Zhouauthor=K.+Yuauthor=B.+Gaoauthor=H.+Jiangauthor=X.+Zhenauthor=W.+Fu&title=Molecular+modeling+of+the+3D+structure+of+5-HT1AR%3A+Discovery+of+novel+5-HT1AR+agonists+via+dynamic+pharmacophore-based+virtual+screening&doi=10.1021%2Fci400481p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Modeling of the 3D Structure of 5-HT1AR: Discovery of Novel 5-HT1AR Agonists via Dynamic Pharmacophore-Based Virtual Screening</span></div><div class="casAuthors">Xu, Lili; Zhou, Shanglin; Yu, Kunqian; Gao, Bo; Jiang, Hualiang; Zhen, Xuechu; Fu, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3202-3211</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The serotonin receptor subtype 1A (5-HT1AR) has been implicated in several neurol. conditions, and potent 5-HT1AR agonists have therapeutic potential for the treatment of depression, anxiety, schizophrenia, and Parkinson's disease.  In the present study, a homol. model of 5-HT1AR was built based on the latest released high-resoln. crystal structure of the β2AR in its active state (PDB: 3SN6).  A dynamic pharmacophore model, which takes the receptor flexibility into account, was constructed, validated, and applied to our dynamic pharmacophore-based virtual screening approach with the aim to identify potential 5-HT1AR agonists.  The obtained hits were subjected to 5-HT1AR binding and functional assays, and 10 compds. with medium or high Ki and EC50 values were identified.  Among them, FW01 (Ki = 51.9 nM, EC50 = 7 nM) was evaluated as the strongest agonist for 5-HT1AR.  The active 5-HT1AR model and dynamic pharmacophore model obtained from this study can be used for future discovery and design of novel 5-HT1AR agonists.  Also, by integrating all computational and available exptl. data, a stepwise 5-HT1AR signal transduction model induced by agonist FW01 was proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO3o2LACvakrVg90H21EOLACvtfcHk0lgdGwD8Ci_VNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslyks7bE&md5=6b2ec04d56b6c2e34e013ec295e66775</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fci400481p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400481p%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DGao%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DZhen%26aufirst%3DX.%26aulast%3DFu%26aufirst%3DW.%26atitle%3DMolecular%2520modeling%2520of%2520the%25203D%2520structure%2520of%25205-HT1AR%253A%2520Discovery%2520of%2520novel%25205-HT1AR%2520agonists%2520via%2520dynamic%2520pharmacophore-based%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D3202%26epage%3D3211%26doi%3D10.1021%2Fci400481p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rudling, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almlöf, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scobie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warpman
Berglund, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stenmark, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, J.</span></span> <span> </span><span class="NLM_article-title">Fragment-based discovery and optimization of enzyme inhibitors by docking of commercial chemical space</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8160</span>– <span class="NLM_lpage">8169</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01006</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01006" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeiurnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8160-8169&author=A.+Rudlingauthor=R.+Gustafssonauthor=I.+Alml%C3%B6fauthor=E.+Homanauthor=M.+Scobieauthor=U.+Warpman%0ABerglundauthor=T.+Helledayauthor=P.+Stenmarkauthor=J.+Carlsson&title=Fragment-based+discovery+and+optimization+of+enzyme+inhibitors+by+docking+of+commercial+chemical+space&doi=10.1021%2Facs.jmedchem.7b01006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery and Optimization of Enzyme Inhibitors by Docking of Commercial Chemical Space</span></div><div class="casAuthors">Rudling, Axel; Gustafsson, Robert; Almloef, Ingrid; Homan, Evert; Scobie, Martin; Warpman Berglund, Ulrika; Helleday, Thomas; Stenmark, Paal; Carlsson, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8160-8169</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based lead discovery has emerged as a leading drug development strategy for novel therapeutic targets.  Although fragment-based drug discovery benefits immensely from access to at.-resoln. information, structure-based virtual screening has rarely been used to drive fragment discovery and optimization.  Here, mol. docking of 0.3 million fragments to a crystal structure of cancer target MTH1 was performed.  Twenty-two predicted fragment ligands, for which analogs could be acquired com., were exptl. evaluated.  Five fragments inhibited MTH1 with IC50 values ranging from 6 to 79 μM.  Structure-based optimization guided by predicted binding modes and analogs from com. chem. libraries yielded nanomolar inhibitors.  Subsequently solved crystal structures confirmed binding modes predicted by docking for three scaffolds.  Structure-guided exploration of com. chem. space using mol. docking gives access to fragment libraries that are several orders of magnitude larger than those screened exptl. and can enable efficient optimization of hits to potent leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7SXpOWovFo7Vg90H21EOLACvtfcHk0lgdGwD8Ci_VNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeiurnF&md5=8858ee50eeb18fa0a397dec221a2effb</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01006%26sid%3Dliteratum%253Aachs%26aulast%3DRudling%26aufirst%3DA.%26aulast%3DGustafsson%26aufirst%3DR.%26aulast%3DAlml%25C3%25B6f%26aufirst%3DI.%26aulast%3DHoman%26aufirst%3DE.%26aulast%3DScobie%26aufirst%3DM.%26aulast%3DWarpman%2BBerglund%26aufirst%3DU.%26aulast%3DHelleday%26aufirst%3DT.%26aulast%3DStenmark%26aufirst%3DP.%26aulast%3DCarlsson%26aufirst%3DJ.%26atitle%3DFragment-based%2520discovery%2520and%2520optimization%2520of%2520enzyme%2520inhibitors%2520by%2520docking%2520of%2520commercial%2520chemical%2520space%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8160%26epage%3D8169%26doi%3D10.1021%2Facs.jmedchem.7b01006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">di
Pizio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laghezza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tortorella, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agamennone, M.</span></span> <span> </span><span class="NLM_article-title">Probing the S1’ site for the identification of non-zinc-binding MMP-2 inhibitors</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1475</span>– <span class="NLM_lpage">1482</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201300186</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1002%2Fcmdc.201300186" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=23873724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A280%3ADC%252BC3sfisVOitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1475-1482&author=A.+di%0APizioauthor=A.+Laghezzaauthor=P.+Tortorellaauthor=M.+Agamennone&title=Probing+the+S1%E2%80%99+site+for+the+identification+of+non-zinc-binding+MMP-2+inhibitors&doi=10.1002%2Fcmdc.201300186"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the S1' site for the identification of non-zinc-binding MMP-2 inhibitors</span></div><div class="casAuthors">Di Pizio Antonella; Laghezza Antonio; Tortorella Paolo; Agamennone Mariangela</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1475-82, 1421</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Matrix metalloproteinases (MMPs) are zinc-dependent enzymes involved in several pathological states.  Among them, MMP-2 is a relevant therapeutic target because of its role in cancer development and progression.  Many MMP inhibitors (MMPIs) have been discovered over the last 30 years, and the majority of them contain a functional group that binds the zinc ion (zinc-binding group; ZBG).  Unfortunately, no MMPIs have reached the market yet, owing to toxic effects due to unselective interactions of the ZBG.  The new generation of MMPIs that do not bind the zinc ion could overcome problems of selectivity and toxicity, but have so far been developed only for MMP-8, -12, and -13.  In this work, a virtual screening protocol was established by combining ligand- and structure-based methods to identify non-zinc-binding MMP-2 inhibitors using a new-generation MMP-8 inhibitor as a probe to find unexplored interactions in the MMP-2 S1' site.  The screening allowed the identification of micromolar MMP-2 inhibitors that putatively avoid binding the zinc ion, as demonstrated by docking calculations.  The LIA model, built to correlate predicted and experimental binding energies of the identified non-zinc-binding MMP-2 hits, underpins the reliability of the predicted docking poses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRctRUUsTkSrxJ5E29YNGvQfW6udTcc2eY6vkJCxWw4prntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sfisVOitQ%253D%253D&md5=0adbedeb84637087ab42b20932278d49</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300186%26sid%3Dliteratum%253Aachs%26aulast%3Ddi%2BPizio%26aufirst%3DA.%26aulast%3DLaghezza%26aufirst%3DA.%26aulast%3DTortorella%26aufirst%3DP.%26aulast%3DAgamennone%26aufirst%3DM.%26atitle%3DProbing%2520the%2520S1%25E2%2580%2599%2520site%2520for%2520the%2520identification%2520of%2520non-zinc-binding%2520MMP-2%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26spage%3D1475%26epage%3D1482%26doi%3D10.1002%2Fcmdc.201300186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grebner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malmerberg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shewmaker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batista, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowski, J.</span></span> <span> </span><span class="NLM_article-title">Virtual screening in the cloud: How big is big enough?</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4274</span>– <span class="NLM_lpage">4282</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.9b00779</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.9b00779" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVyms77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=4274-4282&author=C.+Grebnerauthor=E.+Malmerbergauthor=A.+Shewmakerauthor=J.+Batistaauthor=A.+Nichollsauthor=J.+Sadowski&title=Virtual+screening+in+the+cloud%3A+How+big+is+big+enough%3F&doi=10.1021%2Facs.jcim.9b00779"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Screening in the Cloud: How Big Is Big Enough?</span></div><div class="casAuthors">Grebner, Christoph; Malmerberg, Erik; Shewmaker, Andrew; Batista, Jose; Nicholls, Anthony; Sadowski, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4274-4282</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual screening is a std. tool in Computer-Assisted Drug Design (CADD).  Early in a project, it is typical to use ligand-based similarity search methods to find suitable hit mols.  However, the no. of compds. which can be screened and the time required are usually limited by computational resources.  We describe here a high-throughput virtual screening project using 3D similarity (FastROCS) and automated evaluation workflows on Orion, a cloud computing platform.  Cloud resources make this approach fully scalable and flexible, allowing the generation and search of billions of virtual mols., and give access to an explicit 3D virtual chem. space not available before.  We discuss the impact of the size of the search space with respect to finding novel chem. hits and the size of the required hit list, as well as computational and economical aspects of resource scaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpycUgVx6GwYLVg90H21EOLACvtfcHk0lh-rn8r8JtV2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVyms77I&md5=5be8eab30a5532334c56a906e7ccf9cd</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.9b00779&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.9b00779%26sid%3Dliteratum%253Aachs%26aulast%3DGrebner%26aufirst%3DC.%26aulast%3DMalmerberg%26aufirst%3DE.%26aulast%3DShewmaker%26aufirst%3DA.%26aulast%3DBatista%26aufirst%3DJ.%26aulast%3DNicholls%26aufirst%3DA.%26aulast%3DSadowski%26aufirst%3DJ.%26atitle%3DVirtual%2520screening%2520in%2520the%2520cloud%253A%2520How%2520big%2520is%2520big%2520enough%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D4274%26epage%3D4282%26doi%3D10.1021%2Facs.jcim.9b00779" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walters, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span> <span> </span><span class="NLM_article-title">New trends in virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4109</span>– <span class="NLM_lpage">4111</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c01009</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c01009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFartL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=4109-4111&author=W.+P.+Waltersauthor=R.+Wang&title=New+trends+in+virtual+screening&doi=10.1021%2Facs.jcim.0c01009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">New Trends in Virtual Screening</span></div><div class="casAuthors">Walters, W. Patrick; Wang, Renxiao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4109-4111</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsyuDmwuVE97Vg90H21EOLACvtfcHk0liGEEFCyXXLEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFartL3I&md5=09eadcc885e42415fb4594977450be17</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c01009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c01009%26sid%3Dliteratum%253Aachs%26aulast%3DWalters%26aufirst%3DW.%2BP.%26aulast%3DWang%26aufirst%3DR.%26atitle%3DNew%2520trends%2520in%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D4109%26epage%3D4111%26doi%3D10.1021%2Facs.jcim.0c01009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smieško, M.</span></span> <span> </span><span class="NLM_article-title">DOLINA - docking based on a local induced-fit algorithm: Application toward small-molecule binding to nuclear receptors</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1415</span>– <span class="NLM_lpage">1423</span>, <span class="refDoi"> DOI: 10.1021/ci400098y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400098y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosFygu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=1415-1423&author=M.+Smie%C5%A1ko&title=DOLINA+-+docking+based+on+a+local+induced-fit+algorithm%3A+Application+toward+small-molecule+binding+to+nuclear+receptors&doi=10.1021%2Fci400098y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">DOLINA - Docking Based on a Local Induced-Fit Algorithm: Application toward Small-Molecule Binding to Nuclear Receptors</span></div><div class="casAuthors">Smiesko, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1415-1423</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Docking algorithms allowing for ligand and - to various extent - also protein flexibility are nowadays replacing techniques based on rigid protocols.  The algorithm implemented in the Dolina software relies on pharmacophore matching for generating potential ligand poses and treats assocd. local induced-fit changes by combinatorial rearrangement of side-chains lining the binding site.  In Dolina, ligand flexibility is not treated internally, instead a pool of low-energy conformers identified in a conformational search is screened for extended binding-pose candidates.  Grouping rearranged residues in sterically independent families and side-chain conformer clustering are employed to achieve efficient use of the computational resources along with a good accuracy of the generated poses.  Dolina was applied toward docking of small-mol. ligands to three different nuclear receptor ligand binding domains for which in total 18 high-resoln. crystal structures were used as ref.  The selected nuclear receptors feature a deeply buried ligand-binding site where local induced-fit is to be expected, particularly for receptor antagonists.  For each receptor, a crystal structure with a cocrystd. small steroid ligand (template) was chosen as a target system, to which several synthetic ligands of different sizes were docked.  Poses within an RMSD of 2.0 Å from the crystal ref. pose were generated in 91% of the cases.  In 28%, the pose with the lowest RMSD to the ref. pose was ranked as the top one, and in 76% it was ranked among the top five poses.  Detailed descriptions of the docking algorithm and obsd. results are included.  Dolina is available free of charge for academic institutions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPgqbDVuxYKbVg90H21EOLACvtfcHk0liGEEFCyXXLEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosFygu7c%253D&md5=f2d93edcbf7af9e7eb8001dacc32e970</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1021%2Fci400098y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400098y%26sid%3Dliteratum%253Aachs%26aulast%3DSmie%25C5%25A1ko%26aufirst%3DM.%26atitle%3DDOLINA%2520-%2520docking%2520based%2520on%2520a%2520local%2520induced-fit%2520algorithm%253A%2520Application%2520toward%2520small-molecule%2520binding%2520to%2520nuclear%2520receptors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D1415%26epage%3D1423%26doi%3D10.1021%2Fci400098y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lill, M. A.</span></span> <span> </span><span class="NLM_article-title">Efficient incorporation of protein flexibility and dynamics into molecular docking simulations</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">6157</span>– <span class="NLM_lpage">6169</span>, <span class="refDoi"> DOI: 10.1021/bi2004558</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi2004558" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvFSjurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=6157-6169&author=M.+A.+Lill&title=Efficient+incorporation+of+protein+flexibility+and+dynamics+into+molecular+docking+simulations&doi=10.1021%2Fbi2004558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Incorporation of Protein Flexibility and Dynamics into Molecular Docking Simulations</span></div><div class="casAuthors">Lill, Markus A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">6157-6169</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Flexibility and dynamics are protein characteristics that are essential for the process of mol. recognition.  Conformational changes in the protein that are coupled to ligand binding are described by the biophys. models of induced fit and conformational selection.  Different concepts that incorporate protein flexibility into protein-ligand docking within the context of these two models are reviewed.  Several computational studies that discuss the validity and possible limitations of such approaches will be presented.  Finally, different approaches that incorporate protein dynamics, e.g., configurational entropy, and solvation effects into docking will be highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmYHocKnP0SLVg90H21EOLACvtfcHk0lg71vTrVhfTCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvFSjurc%253D&md5=a1cd2a2348f91c52e0ea508613288fcc</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1021%2Fbi2004558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi2004558%26sid%3Dliteratum%253Aachs%26aulast%3DLill%26aufirst%3DM.%2BA.%26atitle%3DEfficient%2520incorporation%2520of%2520protein%2520flexibility%2520and%2520dynamics%2520into%2520molecular%2520docking%2520simulations%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D6157%26epage%3D6169%26doi%3D10.1021%2Fbi2004558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Vivo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottegoni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span> <span> </span><span class="NLM_article-title">Role of molecular dynamics and related methods in drug discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4035</span>– <span class="NLM_lpage">4061</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01684</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01684" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlWksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4035-4061&author=M.+de%0AVivoauthor=M.+Masettiauthor=G.+Bottegoniauthor=A.+Cavalli&title=Role+of+molecular+dynamics+and+related+methods+in+drug+discovery&doi=10.1021%2Facs.jmedchem.5b01684"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Role of Molecular Dynamics and Related Methods in Drug Discovery</span></div><div class="casAuthors">De Vivo, Marco; Masetti, Matteo; Bottegoni, Giovanni; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4035-4061</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. dynamics (MD) and related methods are close to becoming routine computational tools for drug discovery.  Their main advantage is in explicitly treating structural flexibility and entropic effects.  This allows a more accurate est. of the thermodn. and kinetics assocd. with drug-target recognition and binding, as better algorithms and hardware architectures increase their use.  Here, we review the theor. background of MD and enhanced sampling methods, focusing on free-energy perturbation, metadynamics, steered MD, and other methods most consistently used to study drug-target binding.  We discuss unbiased MD simulations that nowadays allow the observation of unsupervised ligand-target binding, assessing how these approaches help optimizing target affinity and drug residence time toward improved drug efficacy.  Further issues discussed include allosteric modulation and the role of water mols. in ligand binding and optimization.  We conclude by calling for more prospective studies to attest to these methods' utility in discovering novel drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQb_02a7Nmh7Vg90H21EOLACvtfcHk0lg71vTrVhfTCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlWksLs%253D&md5=b3d8a4fb5705a3555c2bb2a962420396</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01684%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BVivo%26aufirst%3DM.%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DRole%2520of%2520molecular%2520dynamics%2520and%2520related%2520methods%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4035%26epage%3D4061%26doi%3D10.1021%2Facs.jmedchem.5b01684" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guedes, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, F. S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dardenne, L. E.</span></span> <span> </span><span class="NLM_article-title">Empirical scoring functions for structure-based virtual screening: Applications, critical aspects, and challenges</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1089</span>, <span class="refDoi"> DOI: 10.3389/fphar.2018.01089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.3389%2Ffphar.2018.01089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=30319422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpt1WqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1089&author=I.+A.+Guedesauthor=F.+S.+S.+Pereiraauthor=L.+E.+Dardenne&title=Empirical+scoring+functions+for+structure-based+virtual+screening%3A+Applications%2C+critical+aspects%2C+and+challenges&doi=10.3389%2Ffphar.2018.01089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Empirical scoring functions for structure-based virtual screening: applications, critical aspects, and challenges</span></div><div class="casAuthors">Guedes, Isabella A.; Pereira, Felipe S. S.; Dardenne, Laurent E.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Pharmacology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1089</span>CODEN:
                <span class="NLM_cas:coden">FPRHAU</span>;
        ISSN:<span class="NLM_cas:issn">1663-9812</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">A review.  Structure-based virtual screening (VS) is a widely used approach that employs the knowledge of the three-dimensional structure of the target of interest in the design of new lead compds. from large-scale mol. docking expts.  Through the prediction of the binding mode and affinity of a small mol. within the binding site of the target of interest, it is possible to understand important properties related to the binding process.  Empirical scoring functions are widely used for pose and affinity prediction.  Although pose prediction is performed with satisfactory accuracy, the correct prediction of binding affinity is still a challenging task and crucial for the success of structure-based VS expts.  There are several efforts in distinct fronts to develop even more sophisticated and accurate models for filtering and ranking large libraries of compds.  This paper will cover some recent successful applications and methodol. advances, including strategies to explore the ligand entropy and solvent effects, training with sophisticated machine-learning techniques, and the use of quantum mechanics.  Particular emphasis will be given to the discussion of crit. aspects and further directions for the development of more accurate empirical scoring functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuNmYBXPKivLVg90H21EOLACvtfcHk0lg71vTrVhfTCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpt1WqsLk%253D&md5=6fd60ccc4d6c2c85a206030023f7c0d1</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2018.01089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2018.01089%26sid%3Dliteratum%253Aachs%26aulast%3DGuedes%26aufirst%3DI.%2BA.%26aulast%3DPereira%26aufirst%3DF.%2BS.%2BS.%26aulast%3DDardenne%26aufirst%3DL.%2BE.%26atitle%3DEmpirical%2520scoring%2520functions%2520for%2520structure-based%2520virtual%2520screening%253A%2520Applications%252C%2520critical%2520aspects%252C%2520and%2520challenges%26jtitle%3DFront.%2520Pharmacol.%26date%3D2018%26volume%3D9%26spage%3D1089%26doi%3D10.3389%2Ffphar.2018.01089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grinter, S. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, X.</span></span> <span> </span><span class="NLM_article-title">Scoring functions and their evaluation methods for protein-ligand docking: Recent advances and future directions</span>. <i>Phys. Chem. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">12899</span>– <span class="NLM_lpage">12908</span>, <span class="refDoi"> DOI: 10.1039/c0cp00151a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1039%2Fc0cp00151a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=20730182" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1GjtLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=12899-12908&author=S.-Y.+Huangauthor=S.+Z.+Grinterauthor=X.+Zou&title=Scoring+functions+and+their+evaluation+methods+for+protein-ligand+docking%3A+Recent+advances+and+future+directions&doi=10.1039%2Fc0cp00151a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Scoring functions and their evaluation methods for protein-ligand docking: recent advances and future directions</span></div><div class="casAuthors">Huang, Sheng-You; Grinter, Sam Z.; Zou, Xiaoqin</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">12899-12908</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The scoring function is one of the most important components in structure-based drug design.  Despite considerable success, accurate and rapid prediction of protein-ligand interactions is still a challenge in mol. docking.  In this perspective, the authors have reviewed 3 basic types of scoring functions (force-field, empirical, and knowledge-based) and the consensus scoring technique that are used for protein-ligand docking.  The commonly-used assessment criteria and publicly available protein-ligand databases for performance evaluation of the scoring functions have also been presented and discussed.  The authors end with a discussion of the challenges faced by existing scoring functions and possible future directions for developing improved scoring functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVhIxJscZuCbVg90H21EOLACvtfcHk0lg71vTrVhfTCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1GjtLbL&md5=ebb7013a3a2c8f5d190df03329d8d8cd</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1039%2Fc0cp00151a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0cp00151a%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DS.-Y.%26aulast%3DGrinter%26aufirst%3DS.%2BZ.%26aulast%3DZou%26aufirst%3DX.%26atitle%3DScoring%2520functions%2520and%2520their%2520evaluation%2520methods%2520for%2520protein-ligand%2520docking%253A%2520Recent%2520advances%2520and%2520future%2520directions%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2010%26volume%3D12%26spage%3D12899%26epage%3D12908%26doi%3D10.1039%2Fc0cp00151a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span> <span> </span><span class="NLM_article-title">An overview of scoring functions used for protein-ligand interactions in molecular docking</span>. <i>Interdiscip. Sci.: Comput. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">320</span>– <span class="NLM_lpage">328</span>, <span class="refDoi"> DOI: 10.1007/s12539-019-00327-w</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1007%2Fs12539-019-00327-w" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVKgur7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=320-328&author=J.+Liauthor=A.+Fuauthor=L.+Zhang&title=An+overview+of+scoring+functions+used+for+protein-ligand+interactions+in+molecular+docking&doi=10.1007%2Fs12539-019-00327-w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">An Overview of Scoring Functions Used for Protein-Ligand Interactions in Molecular Docking</span></div><div class="casAuthors">Li, Jin; Fu, Ailing; Zhang, Le</div><div class="citationInfo"><span class="NLM_cas:title">Interdisciplinary Sciences: Computational Life Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">320-328</span>CODEN:
                <span class="NLM_cas:coden">ISCLAL</span>;
        ISSN:<span class="NLM_cas:issn">1867-1462</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  Currently, mol. docking is becoming a key tool in drug discovery and mol. modeling applications.  The reliability of mol. docking depends on the accuracy of the adopted scoring function, which can guide and det. the ligand poses when thousands of possible poses of ligand are generated.  The scoring function can be used to det. the binding mode and site of a ligand, predict binding affinity and identify the potential drug leads for a given protein target.  Despite intensive research over the years, accurate and rapid prediction of protein-ligand interactions is still a challenge in mol. docking.  For this reason, this study reviews four basic types of scoring functions, physics-based, empirical, knowledge-based, and machine learning-based scoring functions, based on an up-to-date classification scheme.  We not only discuss the foundations of the four types scoring functions, suitable application areas and shortcomings, but also discuss challenges and potential future study directions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPLeIP3v1p0LVg90H21EOLACvtfcHk0lgrde38J3NiiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVKgur7O&md5=32d69a1e5260384d3d44d8d1e846c715</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1007%2Fs12539-019-00327-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12539-019-00327-w%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DL.%26atitle%3DAn%2520overview%2520of%2520scoring%2520functions%2520used%2520for%2520protein-ligand%2520interactions%2520in%2520molecular%2520docking%26jtitle%3DInterdiscip.%2520Sci.%253A%2520Comput.%2520Life%2520Sci.%26date%3D2019%26volume%3D11%26spage%3D320%26epage%3D328%26doi%3D10.1007%2Fs12539-019-00327-w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, T.</span></span> <span> </span><span class="NLM_article-title">Comprehensive evaluation of ten docking programs on a diverse set of protein-ligand complexes: The prediction accuracy of sampling power and scoring power</span>. <i>Phys. Chem. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">12964</span>– <span class="NLM_lpage">12975</span>, <span class="refDoi"> DOI: 10.1039/C6CP01555G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1039%2FC6CP01555G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=27108770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvFOns7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2016&pages=12964-12975&author=Z.+Wangauthor=H.+Sunauthor=X.+Yaoauthor=D.+Liauthor=L.+Xuauthor=Y.+Liauthor=S.+Tianauthor=T.+Hou&title=Comprehensive+evaluation+of+ten+docking+programs+on+a+diverse+set+of+protein-ligand+complexes%3A+The+prediction+accuracy+of+sampling+power+and+scoring+power&doi=10.1039%2FC6CP01555G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive evaluation of ten docking programs on a diverse set of protein-ligand complexes: the prediction accuracy of sampling power and scoring power</span></div><div class="casAuthors">Wang, Zhe; Sun, Huiyong; Yao, Xiaojun; Li, Dan; Xu, Lei; Li, Youyong; Tian, Sheng; Hou, Tingjun</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">12964-12975</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">As one of the most popular computational approaches in modern structure-based drug design, mol. docking can be used not only to identify the correct conformation of a ligand within the target binding pocket but also to est. the strength of the interaction between a target and a ligand.  Nowadays, as a variety of docking programs are available for the scientific community, a comprehensive understanding of the advantages and limitations of each docking program is fundamentally important to conduct more reasonable docking studies and docking-based virtual screening.  In the present study, based on an extensive dataset of 2002 protein-ligand complexes from the PDBbind database (version 2014), the performance of ten docking programs, including five com. programs (LigandFit, Glide, GOLD, MOE Dock, and Surflex-Dock) and five academic programs (AutoDock, AutoDock Vina, LeDock, rDock, and UCSF DOCK), was systematically evaluated by examg. the accuracies of binding pose prediction (sampling power) and binding affinity estn. (scoring power).  Our results showed that GOLD and LeDock had the best sampling power (GOLD: 59.8% accuracy for the top scored poses; LeDock: 80.8% accuracy for the best poses) and AutoDock Vina had the best scoring power (rp/rs of 0.564/0.580 and 0.569/0.584 for the top scored poses and best poses), suggesting that the com. programs did not show the expected better performance than the academic ones.  Overall, the ligand binding poses could be identified in most cases by the evaluated docking programs but the ranks of the binding affinities for the entire dataset could not be well predicted by most docking programs.  However, for some types of protein families, relatively high linear correlations between docking scores and exptl. binding affinities could be achieved.  To our knowledge, this study has been the most extensive evaluation of popular mol. docking programs in the last five years.  It is expected that our work can offer useful information for the successful application of these docking tools to different requirements and targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7PQalJNVCubVg90H21EOLACvtfcHk0lgrde38J3NiiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvFOns7g%253D&md5=5bedadbb6b4eb4dafa95a48701ac720d</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1039%2FC6CP01555G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6CP01555G%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DYao%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DTian%26aufirst%3DS.%26aulast%3DHou%26aufirst%3DT.%26atitle%3DComprehensive%2520evaluation%2520of%2520ten%2520docking%2520programs%2520on%2520a%2520diverse%2520set%2520of%2520protein-ligand%2520complexes%253A%2520The%2520prediction%2520accuracy%2520of%2520sampling%2520power%2520and%2520scoring%2520power%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2016%26volume%3D18%26spage%3D12964%26epage%3D12975%26doi%3D10.1039%2FC6CP01555G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salmaso, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moro, S.</span></span> <span> </span><span class="NLM_article-title">Bridging molecular docking to molecular dynamics in exploring ligand-protein recognition process: An overview</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">923</span>, <span class="refDoi"> DOI: 10.3389/fphar.2018.00923</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.3389%2Ffphar.2018.00923" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=30186166" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtFyltbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=923&author=V.+Salmasoauthor=S.+Moro&title=Bridging+molecular+docking+to+molecular+dynamics+in+exploring+ligand-protein+recognition+process%3A+An+overview&doi=10.3389%2Ffphar.2018.00923"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Bridging molecular docking to molecular dynamics in exploring ligand-protein recognition process: an overview</span></div><div class="casAuthors">Salmaso, Veronica; Moro, Stefano</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Pharmacology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">923/1-923/16</span>CODEN:
                <span class="NLM_cas:coden">FPRHAU</span>;
        ISSN:<span class="NLM_cas:issn">1663-9812</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Computational techniques have been applied in the drug discovery pipeline since the 1980s.  Given the low computational resources of the time, the first mol. modeling strategies relied on a rigid view of the ligand-target binding process.  During the years, the evolution of hardware technologies has gradually allowed simulating the dynamic nature of the binding event.  In this work, we present an overview of the evolution of structure- based drug discovery techniques in the study of ligand-target recognition phenomenon, going fromthe staticmol. docking toward enhancedmol. dynamics strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-35hlGTXB6rVg90H21EOLACvtfcHk0lgrde38J3NiiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtFyltbw%253D&md5=cd279290cc97675187703a45611dc24f</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2018.00923&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2018.00923%26sid%3Dliteratum%253Aachs%26aulast%3DSalmaso%26aufirst%3DV.%26aulast%3DMoro%26aufirst%3DS.%26atitle%3DBridging%2520molecular%2520docking%2520to%2520molecular%2520dynamics%2520in%2520exploring%2520ligand-protein%2520recognition%2520process%253A%2520An%2520overview%26jtitle%3DFront.%2520Pharmacol.%26date%3D2018%26volume%3D9%26spage%3D923%26doi%3D10.3389%2Ffphar.2018.00923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maffucci, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fumagalli, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Contini, A.</span></span> <span> </span><span class="NLM_article-title">An efficient implementation of the Nwat-MMGBSA method to rescore docking results in medium-throughput virtual screenings</span>. <i>Front. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">43</span>, <span class="refDoi"> DOI: 10.3389/fchem.2018.00043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.3389%2Ffchem.2018.00043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=29556494" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1egsbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=43&author=I.+Maffucciauthor=X.+Huauthor=V.+Fumagalliauthor=A.+Contini&title=An+efficient+implementation+of+the+Nwat-MMGBSA+method+to+rescore+docking+results+in+medium-throughput+virtual+screenings&doi=10.3389%2Ffchem.2018.00043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">An efficient implementation of the nwat-mmgbsa method to rescore docking results in medium-throughput virtual screenings</span></div><div class="casAuthors">Maffucci, Irene; Hu, Xiao; Fumagalli, Valentina; Contini, Alessandro</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Chemistry (Lausanne, Switzerland)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43/1-43/14</span>CODEN:
                <span class="NLM_cas:coden">FCLSAA</span>;
        ISSN:<span class="NLM_cas:issn">2296-2646</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Nwat-MMGBSA is a variant of MM-PB/GBSA based on the inclusion of a no. of explicit water mols. that are the closest to the ligand in each frame of a mol. dynamics trajectory.  This method demonstrated improved correlations between calcd. and exptl. binding energies in both protein-protein interactions and ligand-receptor complexes, in comparison to the std. MM-GBSA.  A protocol optimization, aimed to maximize efficacy and efficiency, is discussed here considering penicillopepsin, HIV1-protease, and BCL-XL as test cases.  Calcns. were performed in triplicates on both classic HPC environments and on std. workstations equipped by a GPU card, evidencing no statistical differences in the results.  No relevant differences in correlation to expts. were also obsd. when performing Nwat-MMGBSA calcns. on 4 or 1 ns long trajectories.  A fully automatic workflow for structure-based virtual screening, performing from library set-up to docking and Nwat-MMGBSA rescoring, has then been developed.  The protocol has been tested against no rescoring or std. MM-GBSA rescoring within a retrospective virtual screening of inhibitors of AmpC β-lactamase and of the Rac1-Tiam1 protein-protein interaction.  In both cases, Nwat-MMGBSA rescoring provided a statistically significant increase in the ROC AUCs of between 20 and 30%, compared to docking scoring or to std. MM-GBSA rescoring.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOtEwZRkQ6BLVg90H21EOLACvtfcHk0ljnUeJclW78XA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1egsbrN&md5=b2925a1d0f9323f99cb4c93af53ced04</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.3389%2Ffchem.2018.00043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffchem.2018.00043%26sid%3Dliteratum%253Aachs%26aulast%3DMaffucci%26aufirst%3DI.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DFumagalli%26aufirst%3DV.%26aulast%3DContini%26aufirst%3DA.%26atitle%3DAn%2520efficient%2520implementation%2520of%2520the%2520Nwat-MMGBSA%2520method%2520to%2520rescore%2520docking%2520results%2520in%2520medium-throughput%2520virtual%2520screenings%26jtitle%3DFront.%2520Chem.%26date%3D2018%26volume%3D6%26spage%3D43%26doi%3D10.3389%2Ffchem.2018.00043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinzi, L.</span></span> <span> </span><span class="NLM_article-title">Refinement and rescoring of virtual screening results</span>. <i>Front. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">498</span>, <span class="refDoi"> DOI: 10.3389/fchem.2019.00498</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.3389%2Ffchem.2019.00498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=31355188" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFGgt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2019&pages=498&author=G.+Rastelliauthor=L.+Pinzi&title=Refinement+and+rescoring+of+virtual+screening+results&doi=10.3389%2Ffchem.2019.00498"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement and rescoring of virtual screening results</span></div><div class="casAuthors">Rastelli, Giulio; Pinzi, Luca</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Chemistry (Lausanne, Switzerland)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">498</span>CODEN:
                <span class="NLM_cas:coden">FCLSAA</span>;
        ISSN:<span class="NLM_cas:issn">2296-2646</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">High-throughput docking is an established computational screening approach in drug design.  This methodol. enables a rapid identification of biol. active hit compds., providing an efficient and cost-effective complement or alternative to exptl. high-throughput screenings.  However, limitations inherent to the methodol. make docking results inevitably approx.  Two major Achille's heels include the use of approximated scoring functions and the limited sampling of the ligand-target complexes.  Therefore, docking results require careful evaluation and further post-docking analyses.  In this article, we will overview our approach to post-docking anal. in virtual screenings.  BEAR (Binding Estn. After Refinement) was developed as a post-docking processing tool that refines docking poses by means of mol. dynamics (MD) and then rescores the ligands based on more accurate scoring functions (MM-PB(GB)SA).  The tool has been validated and used prospectively in drug discovery applications.  Future directions regarding refinement and rescoring in virtual screening are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA7hkak3owGLVg90H21EOLACvtfcHk0ljnUeJclW78XA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFGgt7w%253D&md5=64ccae09466a5aaf9df144137be1fa92</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.3389%2Ffchem.2019.00498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffchem.2019.00498%26sid%3Dliteratum%253Aachs%26aulast%3DRastelli%26aufirst%3DG.%26aulast%3DPinzi%26aufirst%3DL.%26atitle%3DRefinement%2520and%2520rescoring%2520of%2520virtual%2520screening%2520results%26jtitle%3DFront.%2520Chem.%26date%3D2019%26volume%3D7%26spage%3D498%26doi%3D10.3389%2Ffchem.2019.00498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lightstone, F. C.</span></span> <span> </span><span class="NLM_article-title">Toward fully automated high performance computing drug discovery: A massively parallel virtual screening pipeline for docking and molecular mechanics/generalized born surface area rescoring to improve enrichment</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">324</span>– <span class="NLM_lpage">337</span>, <span class="refDoi"> DOI: 10.1021/ci4005145</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci4005145" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOiurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=324-337&author=X.+Zhangauthor=S.+E.+Wongauthor=F.+C.+Lightstone&title=Toward+fully+automated+high+performance+computing+drug+discovery%3A+A+massively+parallel+virtual+screening+pipeline+for+docking+and+molecular+mechanics%2Fgeneralized+born+surface+area+rescoring+to+improve+enrichment&doi=10.1021%2Fci4005145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Toward Fully Automated High Performance Computing Drug Discovery: A Massively Parallel Virtual Screening Pipeline for Docking and Molecular Mechanics/Generalized Born Surface Area Rescoring to Improve Enrichment</span></div><div class="casAuthors">Zhang, Xiaohua; Wong, Sergio E.; Lightstone, Felice C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">324-337</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors announce and evaluate a high throughput virtual screening pipeline for in-silico screening of virtual compd. databases using high performance computing (HPC).  Notable features of this pipeline are an automated receptor prepn. scheme with unsupervised binding site identification.  The pipeline includes receptor/target prepn., ligand prepn., VinaLC docking calcn., and mol. mechanics/generalized Born surface area (MM/GBSA) rescoring using the GB model by Onufriev and co-workers J. Chem. Theory Comput.2007, 3, 156-169.  Furthermore, the authors leverage HPC resources to perform an unprecedented, comprehensive evaluation of MM/GBSA rescoring when applied to the DUD-E data set (Directory of Useful Decoys: Enhanced), in which the authors selected 38 protein targets and a total of ∼0.7 million actives and decoys.  The computer wall time for virtual screening has been reduced drastically on HPC machines, which increases the feasibility of extremely large ligand database screening with more accurate methods.  HPC resources allowed the authors to rescore 20 poses per compd. and evaluate the optimal no. of poses to rescore.  The authors find that keeping 5-10 poses is a good compromise between accuracy and computational expense.  Overall the results demonstrate that MM/GBSA rescoring has higher av. receiver operating characteristic (ROC) area under curve (AUC) values and consistently better early recovery of actives than Vina docking alone.  Specifically, the enrichment performance is target-dependent.  MM/GBSA rescoring significantly out performs Vina docking for the folate enzymes, kinases, and several other enzymes.  The more accurate energy function and solvation terms of the MM/GBSA method allow MM/GBSA to achieve better enrichment, but the rescoring is still limited by the docking method to generate the poses with the correct binding modes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXbi5hOh_RRLVg90H21EOLACvtfcHk0li77NjAXss0bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOiurfN&md5=37b4dbc7c437fc9da70cdf41689d00b2</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1021%2Fci4005145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci4005145%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWong%26aufirst%3DS.%2BE.%26aulast%3DLightstone%26aufirst%3DF.%2BC.%26atitle%3DToward%2520fully%2520automated%2520high%2520performance%2520computing%2520drug%2520discovery%253A%2520A%2520massively%2520parallel%2520virtual%2520screening%2520pipeline%2520for%2520docking%2520and%2520molecular%2520mechanics%252Fgeneralized%2520born%2520surface%2520area%2520rescoring%2520to%2520improve%2520enrichment%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D324%26epage%3D337%26doi%3D10.1021%2Fci4005145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Virtanen, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niinivehmas, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pentikäinen, O. T.</span></span> <span> </span><span class="NLM_article-title">Case-specific performance of MM-PBSA, MM-GBSA, and SIE in virtual screening</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">303</span>– <span class="NLM_lpage">318</span>, <span class="refDoi"> DOI: 10.1016/j.jmgm.2015.10.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1016%2Fj.jmgm.2015.10.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=26550792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsl2gsLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2015&pages=303-318&author=S.+I.+Virtanenauthor=S.+P.+Niinivehmasauthor=O.+T.+Pentik%C3%A4inen&title=Case-specific+performance+of+MM-PBSA%2C+MM-GBSA%2C+and+SIE+in+virtual+screening&doi=10.1016%2Fj.jmgm.2015.10.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Case-specific performance of MM-PBSA, MM-GBSA, and SIE in virtual screening</span></div><div class="casAuthors">Virtanen, Salla I.; Niinivehmas, Sanna P.; Pentikainen, Olli T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">303-318</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">In drug discovery the reliable prediction of binding free energies is of crucial importance.  Methods that combine mol. mechanics force fields with continuum solvent models have become popular because of their high accuracy and relatively good computational efficiency.  In this research we studied the performance of mol. mechanics generalized Born surface area (MM-GBSA), mol. mechanics Poisson-Boltzmann surface area (MM-PBSA), and solvated interaction energy (SIE) both in their virtual screening efficiency and their ability to predict exptl. detd. binding affinities for five different protein targets.  The protein-ligand complexes were derived with two different approaches important in virtual screening: mol. docking and ligand-based similarity search methods.  The results show significant differences between the different binding energy calcn. methods.  However, the length of the mol. dynamics simulation was not of crucial importance for accuracy of results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzdqc_eszDjbVg90H21EOLACvtfcHk0li77NjAXss0bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsl2gsLnF&md5=a1e470a099904f52e26a4f8e2cf91d55</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2015.10.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2015.10.012%26sid%3Dliteratum%253Aachs%26aulast%3DVirtanen%26aufirst%3DS.%2BI.%26aulast%3DNiinivehmas%26aufirst%3DS.%2BP.%26aulast%3DPentik%25C3%25A4inen%26aufirst%3DO.%2BT.%26atitle%3DCase-specific%2520performance%2520of%2520MM-PBSA%252C%2520MM-GBSA%252C%2520and%2520SIE%2520in%2520virtual%2520screening%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2015%26volume%3D62%26spage%3D303%26epage%3D318%26doi%3D10.1016%2Fj.jmgm.2015.10.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, T.</span></span> <span> </span><span class="NLM_article-title">End-Point binding free energy calculation with MM/PBSA and MM/GBSA: Strategies and applications in drug design</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">9478</span>– <span class="NLM_lpage">9508</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.9b00055</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.9b00055" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1ajs7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2019&pages=9478-9508&author=E.+Wangauthor=H.+Sunauthor=J.+Wangauthor=Z.+Wangauthor=H.+Liuauthor=J.+Z.+Zhangauthor=T.+Hou&title=End-Point+binding+free+energy+calculation+with+MM%2FPBSA+and+MM%2FGBSA%3A+Strategies+and+applications+in+drug+design&doi=10.1021%2Facs.chemrev.9b00055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">End-point binding free energy calculation with MM/PBSA and MM/GBSA: strategies and applications in drug design</span></div><div class="casAuthors">Wang, Ercheng; Sun, Huiyong; Wang, Junmei; Wang, Zhe; Liu, Hui; Zhang, John Z. H.; Hou, Tingjun</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">9478-9508</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Mol. mechanics Poisson-Boltzmann surface area (MM/PBSA) and mol. mechanics generalized Born surface area (MM/GBSA) are arguably very popular methods for binding free energy prediction since they are more accurate than most scoring functions of mol. docking and less computationally demanding than alchem. free energy methods.  MM/PBSA and MM/GBSA have been widely used in biomol. studies such as protein folding, protein-ligand binding, protein-protein interaction, etc.  In this review, methods to adjust the polar solvation energy and to improve the performance of MM/PBSA and MM/GBSA calcns. are reviewed and discussed.  The latest applications of MM/GBSA and MM/PBSA in drug design are also presented.  This review intends to provide readers with guidance for practically applying MM/PBSA and MM/GBSA in drug design and related research fields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu30aDvikEv7Vg90H21EOLACvtfcHk0li77NjAXss0bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1ajs7%252FF&md5=eacd5dbfe659d27df5d103caec9e1bb5</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.9b00055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.9b00055%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DE.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DJ.%2BZ.%26aulast%3DHou%26aufirst%3DT.%26atitle%3DEnd-Point%2520binding%2520free%2520energy%2520calculation%2520with%2520MM%252FPBSA%2520and%2520MM%252FGBSA%253A%2520Strategies%2520and%2520applications%2520in%2520drug%2520design%26jtitle%3DChem.%2520Rev.%26date%3D2019%26volume%3D119%26spage%3D9478%26epage%3D9508%26doi%3D10.1021%2Facs.chemrev.9b00055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albanese, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chodera, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkamer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span> <span> </span><span class="NLM_article-title">Is structure-based drug design ready for selectivity optimization?</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">6211</span>– <span class="NLM_lpage">6227</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c00815</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c00815" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFygs7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=6211-6227&author=S.+K.+Albaneseauthor=J.+D.+Choderaauthor=A.+Volkamerauthor=S.+Kengauthor=R.+Abelauthor=L.+Wang&title=Is+structure-based+drug+design+ready+for+selectivity+optimization%3F&doi=10.1021%2Facs.jcim.0c00815"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Is Structure-Based Drug Design Ready for Selectivity Optimization?</span></div><div class="casAuthors">Albanese, Steven K.; Chodera, John D.; Volkamer, Andrea; Keng, Simon; Abel, Robert; Wang, Lingle</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6211-6227</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Alchem. free-energy calcns. are now widely used to drive or maintain potency in small-mol. lead optimization with a roughly 1 kcal/mol accuracy.  Despite this, the potential to use free-energy calcns. to drive optimization of compd. selectivity among two similar targets has been relatively unexplored in published studies.  In the most optimistic scenario, the similarity of binding sites might lead to a fortuitous cancellation of errors and allow selectivity to be predicted more accurately than affinity.  Here, we assess the accuracy with which selectivity can be predicted in the context of small-mol. kinase inhibitors, considering the very similar binding sites of human kinases CDK2 and CDK9 as well as another series of ligands attempting to achieve selectivity between the more distantly related kinases CDK2 and ERK2.  Using a Bayesian anal. approach, we sep. systematic from statistical errors and quantify the correlation in systematic errors between selectivity targets.  We find that, in the CDK2/CDK9 case, a high correlation in systematic errors suggests that free-energy calcns. can have significant impact in aiding chemists in achieving selectivity, while in more distantly related kinases (CDK2/ERK2), the correlation in systematic error suggests that fortuitous cancellation may even occur between systems that are not as closely related.  In both cases, the correlation in systematic error suggests that longer simulations are beneficial to properly balance statistical error with systematic error to take full advantage of the increase in apparent free-energy calcn. accuracy in selectivity prediction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrITuv90vd8L7Vg90H21EOLACvtfcHk0lhVC8IUnQCxnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFygs7bJ&md5=08cfdf031b626801b7ba762a559d750f</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c00815&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c00815%26sid%3Dliteratum%253Aachs%26aulast%3DAlbanese%26aufirst%3DS.%2BK.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DVolkamer%26aufirst%3DA.%26aulast%3DKeng%26aufirst%3DS.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DL.%26atitle%3DIs%2520structure-based%2520drug%2520design%2520ready%2520for%2520selectivity%2520optimization%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D6211%26epage%3D6227%26doi%3D10.1021%2Facs.jcim.0c00815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghanbarpour, A.</span>; <span class="NLM_string-name">Mahmoud, A. H.</span>; <span class="NLM_string-name">Lill, M. A.</span></span> <span> </span><span class="NLM_article-title">On-the-fly prediction of protein hydration densities and free energies using deep learning</span>.  <i>arXiv</i> <span class="NLM_year" style="font-weight: bold;">2020</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=A.+Ghanbarpour&author=A.+H.+Mahmoud&author=M.+A.+Lill&title=On-the-fly+prediction+of+protein+hydration+densities+and+free+energies+using+deep+learning"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGhanbarpour%26aufirst%3DA.%26atitle%3DOn-the-fly%2520prediction%2520of%2520protein%2520hydration%2520densities%2520and%2520free%2520energies%2520using%2520deep%2520learning%26jtitle%3DarXiv%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Djenno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insua, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pho, A.</span></span> <span> </span><span class="NLM_article-title">From paper to pixels: Using Google Forms for collaboration and assessment</span>. <i>Library hi tech news</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1108/LHTN-12-2014-0105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1108%2FLHTN-12-2014-0105" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2015&pages=9-13&author=M.+Djennoauthor=G.+Insuaauthor=A.+Pho&title=From+paper+to+pixels%3A+Using+Google+Forms+for+collaboration+and+assessment&doi=10.1108%2FLHTN-12-2014-0105"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1108%2FLHTN-12-2014-0105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1108%252FLHTN-12-2014-0105%26sid%3Dliteratum%253Aachs%26aulast%3DDjenno%26aufirst%3DM.%26aulast%3DInsua%26aufirst%3DG.%26aulast%3DPho%26aufirst%3DA.%26atitle%3DFrom%2520paper%2520to%2520pixels%253A%2520Using%2520Google%2520Forms%2520for%2520collaboration%2520and%2520assessment%26jtitle%3DLibrary%2520hi%2520tech%2520news%26date%3D2015%26volume%3D32%26spage%3D9%26epage%3D13%26doi%3D10.1108%2FLHTN-12-2014-0105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottegoni, G.</span></span> <span> </span><span class="NLM_article-title">Recipes for the selection of experimental protein conformations for virtual screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">186</span>– <span class="NLM_lpage">193</span>, <span class="refDoi"> DOI: 10.1021/ci9003943</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci9003943" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFCrsb3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=186-193&author=R.+Abagyanauthor=M.+Ruedaauthor=G.+Bottegoni&title=Recipes+for+the+selection+of+experimental+protein+conformations+for+virtual+screening&doi=10.1021%2Fci9003943"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Recipes for the Selection of Experimental Protein Conformations for Virtual Screening</span></div><div class="casAuthors">Rueda, Manuel; Bottegoni, Giovanni; Abagyan, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">186-193</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The use of multiple x-ray protein structures has been reported to be an efficient alternative for the representation of the binding pocket flexibility needed for accurate small mols. docking.  However, the docking performance of the individual single conformations varies widely, and adding certain conformations to an ensemble is even counterproductive.  Here the authors used a very large and diverse benchmark of 1068 x-ray protein conformations of 99 therapeutically relevant proteins, first, to compare the performance of the ensemble and single-conformation docking and, second, to find the properties of the best-performing conformers that can be used to select a smaller set of conformers for ensemble docking.  The conformer selection has been validated through retrospective virtual screening expts. aimed at sepg. known ligand binders from decoys.  The authors found that the conformers cocrystd. with the largest ligands displayed high selectivity for binders, and when combined in ensembles they consistently provided better results than randomly chosen protein conformations.  The use of ensembles encompassing between 3 and 5 exptl. conformations consistently improved the docking accuracy and binders vs. decoys sepn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBgQPiTcnZy7Vg90H21EOLACvtfcHk0lhVC8IUnQCxnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFCrsb3I&md5=159915d0fb8aa304a49592a0bbb6401f</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1021%2Fci9003943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9003943%26sid%3Dliteratum%253Aachs%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DRueda%26aufirst%3DM.%26aulast%3DBottegoni%26aufirst%3DG.%26atitle%3DRecipes%2520for%2520the%2520selection%2520of%2520experimental%2520protein%2520conformations%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D186%26epage%3D193%26doi%3D10.1021%2Fci9003943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalkas, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vlachakis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsagkrasoulis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kastania, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kossida, S.</span></span> <span> </span><span class="NLM_article-title">State-of-the-art technology in modern computer-aided drug design</span>. <i>Briefings Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">745</span>– <span class="NLM_lpage">752</span>, <span class="refDoi"> DOI: 10.1093/bib/bbs063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=10.1093%2Fbib%2Fbbs063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=23148324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A280%3ADC%252BC3s7jtVamtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=745-752&author=G.+A.+Dalkasauthor=D.+Vlachakisauthor=D.+Tsagkrasoulisauthor=A.+Kastaniaauthor=S.+Kossida&title=State-of-the-art+technology+in+modern+computer-aided+drug+design&doi=10.1093%2Fbib%2Fbbs063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">State-of-the-art technology in modern computer-aided drug design</span></div><div class="casAuthors">Dalkas Georgios A; Vlachakis Dimitrios; Tsagkrasoulis Dimosthenis; Kastania Anastasia; Kossida Sophia</div><div class="citationInfo"><span class="NLM_cas:title">Briefings in bioinformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">745-52</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The quest for small drug-like compounds that selectively inhibit the function of biological targets has always been a major focus in the pharmaceutical industry and in academia as well.  High-throughput screening of compound libraries requires time, cost and resources.  Therefore, the use of alternative methods is necessary for facilitating lead discovery.  Computational techniques that dock small molecules into macromolecular targets and predict the affinity and activity of the small molecule are widely used in drug design and discovery, and have become an integral part of the industrial and academic research.  In this review, we present an overview of some state-of-the-art technologies in modern drug design that have been developed for expediting the search for novel drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRqxJLNY1TepZ6KlWarZTAzfW6udTcc2eZdgXztbj_hmbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s7jtVamtQ%253D%253D&md5=ae0993a629d24a07cc16be8446902913</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1093%2Fbib%2Fbbs063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbib%252Fbbs063%26sid%3Dliteratum%253Aachs%26aulast%3DDalkas%26aufirst%3DG.%2BA.%26aulast%3DVlachakis%26aufirst%3DD.%26aulast%3DTsagkrasoulis%26aufirst%3DD.%26aulast%3DKastania%26aufirst%3DA.%26aulast%3DKossida%26aufirst%3DS.%26atitle%3DState-of-the-art%2520technology%2520in%2520modern%2520computer-aided%2520drug%2520design%26jtitle%3DBriefings%2520Bioinf.%26date%3D2013%26volume%3D14%26spage%3D745%26epage%3D752%26doi%3D10.1093%2Fbib%2Fbbs063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sevrioukova, I. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulos, T. L.</span></span> <span> </span><span class="NLM_article-title">Pyridine-substituted desoxyritonavir is a more potent inhibitor of cytochrome P450 3A4 than ritonavir</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">3733</span>– <span class="NLM_lpage">3741</span>, <span class="refDoi"> DOI: 10.1021/jm400288z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400288z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmtVylsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=3733-3741&author=I.+F.+Sevrioukovaauthor=T.+L.+Poulos&title=Pyridine-substituted+desoxyritonavir+is+a+more+potent+inhibitor+of+cytochrome+P450+3A4+than+ritonavir&doi=10.1021%2Fjm400288z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Pyridine-Substituted Desoxyritonavir Is a More Potent Inhibitor of Cytochrome P450 3A4 than Ritonavir</span></div><div class="casAuthors">Sevrioukova, Irina F.; Poulos, Thomas L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3733-3741</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Utilization of the cytochrome P 450 3A4 (CYP3A4) inhibitor ritonavir as a pharmacoenhancer for anti-HIV drugs revolutionized the treatment of HIV infection.  However, owing to ritonavir-related complications, there is a need for development of new CYP3A4 inhibitors with improved pharmacochem. properties, which requires a full understanding of the CYP3A4 inactivation mechanisms and the unraveling of possible inhibitor binding modes.  We investigated the mechanism of CYP3A4 interaction with three desoxyritonavir analogs, contg. the heme-ligating imidazole, oxazole, or pyridine group instead of the thiazole moiety (compds. 1, 2, and 3, resp.).  Our data show that compd. 3 is superior to ritonavir in terms of binding affinity and inhibitory potency owing to greater flexibility and the ability to adopt a conformation that minimizes steric clashing and optimizes protein-ligand interactions.  Addnl., Ser119 was identified as a key residue assisting binding of ritonavir-like inhibitors, which emphasizes the importance of polar interactions in the CYP3A4-ligand assocn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx1sbUve3zPLVg90H21EOLACvtfcHk0liVKlccgBLBIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmtVylsrc%253D&md5=77a94516a246e5823502f4c89cadc5c0</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1021%2Fjm400288z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400288z%26sid%3Dliteratum%253Aachs%26aulast%3DSevrioukova%26aufirst%3DI.%2BF.%26aulast%3DPoulos%26aufirst%3DT.%2BL.%26atitle%3DPyridine-substituted%2520desoxyritonavir%2520is%2520a%2520more%2520potent%2520inhibitor%2520of%2520cytochrome%2520P450%25203A4%2520than%2520ritonavir%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D3733%26epage%3D3741%26doi%3D10.1021%2Fjm400288z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yano, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denton, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerny, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cashman, J. R.</span></span> <span> </span><span class="NLM_article-title">Synthetic inhibitors of cytochrome P-450 2A6: Inhibitory activity, difference spectra, mechanism of inhibition, and protein cocrystallization</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">6987</span>– <span class="NLM_lpage">7001</span>, <span class="refDoi"> DOI: 10.1021/jm060519r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060519r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFKrs7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6987-7001&author=J.+K.+Yanoauthor=T.+T.+Dentonauthor=M.+A.+Cernyauthor=X.+Zhangauthor=E.+F.+Johnsonauthor=J.+R.+Cashman&title=Synthetic+inhibitors+of+cytochrome+P-450+2A6%3A+Inhibitory+activity%2C+difference+spectra%2C+mechanism+of+inhibition%2C+and+protein+cocrystallization&doi=10.1021%2Fjm060519r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic Inhibitors of Cytochrome P-450 2A6: Inhibitory Activity, Difference Spectra, Mechanism of Inhibition, and Protein Cocrystallization</span></div><div class="casAuthors">Yano, Jason K.; Denton, Travis T.; Cerny, Matthew A.; Zhang, Xiaodong; Johnson, Eric F.; Cashman, John R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6987-7001</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of 3-heteroarom. analogs of nicotine were synthesized to delineate structural and mechanistic requirements for selectively inhibiting human cytochrome P 450 (CYP) 2A6.  Thiophene, substituted thiophene, furan, substituted furan, acetylene, imidazole, substituted imidazole, thiazole, pyrazole, substituted pyrazole, and aliph. and isoxazol moieties were used to replace the N-methylpyrrolidine ring of nicotine.  A no. of potent inhibitors were identified, and several exhibited high selectivity for CYP2A6 relative to CYP2E1, -3A4, -2B6, -2C9, -2C19, and -2D6.  The majority of these inhibitors elicited type II difference spectra indicating the formation of a coordinate covalent bond to the heme iron.  The majority of inhibitors were reversible inhibitors although several mechanism-based inactivators were identified.  Most of the inhibitors were also relatively metabolically stable.  X-ray crystal structures of CYP2A6 cocrystd. with three furan analogs bearing methanamino side chains indicated that the amine side chain coordinated to the heme iron.  The pyridyl moiety was positioned to accept a hydrogen bond from Asn297, and all three inhibitors exhibited orthogonal arom.-arom. interactions with protein side chains.  For comparison, the cocrystal structure of 4,4'-dipyridyl disulfide was also obtained and showed that the pyridine moiety could assume a different orientation than that obsd. for the 3-heteroarom. pyridines examd.  For the 3-heteroromatic pyridines, N-Me and N,N-di-Me amino groups increased the apparent Ki and distorted helix I of the protein.  Substitution of a Ph ring for the pyridyl ring also increased the apparent Ki, which is likely to reflect the loss of the hydrogen bonding interaction with Asn297.  In contrast, inhibitory potency for other P450s was increased, and the selectivity of the Ph analogs for CYP2A6 was decreased relative to the pyridyl compds.  The results suggest that inhibitors that compliment the active site features of CYP2A6 can exhibit significant selectivity for CYP2A6 relative to other human liver drug-metabolizing P450s.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_GlAD-oZ-e7Vg90H21EOLACvtfcHk0liVKlccgBLBIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFKrs7bF&md5=0b6df3afd65811b26bf520817a5e5f0e</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1021%2Fjm060519r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060519r%26sid%3Dliteratum%253Aachs%26aulast%3DYano%26aufirst%3DJ.%2BK.%26aulast%3DDenton%26aufirst%3DT.%2BT.%26aulast%3DCerny%26aufirst%3DM.%2BA.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26aulast%3DCashman%26aufirst%3DJ.%2BR.%26atitle%3DSynthetic%2520inhibitors%2520of%2520cytochrome%2520P-450%25202A6%253A%2520Inhibitory%2520activity%252C%2520difference%2520spectra%252C%2520mechanism%2520of%2520inhibition%252C%2520and%2520protein%2520cocrystallization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6987%26epage%3D7001%26doi%3D10.1021%2Fjm060519r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: A new approach for rapid, accurate docking and scoring. 2. enrichment factors in database screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+A+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0liVKlccgBLBIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520A%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1O86" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1O86','PDB','1O86'); return false;">PDB: 1O86</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5STD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5STD','PDB','5STD'); return false;">PDB: 5STD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M17" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M17','PDB','1M17'); return false;">PDB: 1M17</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QAD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QAD','PDB','3QAD'); return false;">PDB: 3QAD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RZF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RZF','PDB','3RZF'); return false;">PDB: 3RZF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FDW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FDW','PDB','2FDW'); return false;">PDB: 2FDW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5RH0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5RH0','PDB','5RH0'); return false;">PDB: 5RH0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WLB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WLB','PDB','4WLB'); return false;">PDB: 4WLB</a></li></ul></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02227&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-5%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02227" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02227" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677aab717b183cdc","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
