
<html lang="en"     class="pb-page"  data-request-id="933f5362-1285-49cc-8977-7586beb2ff1f"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2015.58.issue-17;website:website:acspubs;article:article:10.1021/acs.jmedchem.5b01082;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Targeting Drug Resistance in EGFR with Covalent Inhibitors: A Structure-Based Design Approach" /></meta><meta name="dc.Creator" content="Julian  Engel" /></meta><meta name="dc.Creator" content="André  Richters" /></meta><meta name="dc.Creator" content="Matthäus  Getlik" /></meta><meta name="dc.Creator" content="Stefano  Tomassi" /></meta><meta name="dc.Creator" content="Marina  Keul" /></meta><meta name="dc.Creator" content="Martin  Termathe" /></meta><meta name="dc.Creator" content="Jonas  Lategahn" /></meta><meta name="dc.Creator" content="Christian  Becker" /></meta><meta name="dc.Creator" content="Svenja  Mayer-Wrangowski" /></meta><meta name="dc.Creator" content="Christian  Grütter" /></meta><meta name="dc.Creator" content="Niklas  Uhlenbrock" /></meta><meta name="dc.Creator" content="Jasmin  Krüll" /></meta><meta name="dc.Creator" content="Niklas  Schaumann" /></meta><meta name="dc.Creator" content="Simone  Eppmann" /></meta><meta name="dc.Creator" content="Patrick  Kibies" /></meta><meta name="dc.Creator" content="Franziska  Hoffgaard" /></meta><meta name="dc.Creator" content="Jochen  Heil" /></meta><meta name="dc.Creator" content="Sascha  Menninger" /></meta><meta name="dc.Creator" content="Sandra  Ortiz-Cuaran" /></meta><meta name="dc.Creator" content="Johannes M.  Heuckmann" /></meta><meta name="dc.Creator" content="Verena  Tinnefeld" /></meta><meta name="dc.Creator" content="René P.  Zahedi" /></meta><meta name="dc.Creator" content="Martin L.  Sos" /></meta><meta name="dc.Creator" content="Carsten  Schultz-Fademrecht" /></meta><meta name="dc.Creator" content="Roman K.  Thomas" /></meta><meta name="dc.Creator" content="Stefan M.  Kast" /></meta><meta name="dc.Creator" content="Daniel  Rauh" /></meta><meta name="dc.Description" content="Receptor tyrosine kinases represent one of the prime targets in cancer therapy, as the dysregulation of these elementary transducers of extracellular signals, like the epidermal growth factor recep..." /></meta><meta name="Description" content="Receptor tyrosine kinases represent one of the prime targets in cancer therapy, as the dysregulation of these elementary transducers of extracellular signals, like the epidermal growth factor recep..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 31, 2015" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b01082" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2015 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01082" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b01082" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01082" /></link>
        
    
    

<title>Targeting Drug Resistance in EGFR with Covalent Inhibitors: A Structure-Based Design Approach | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b01082" /></meta><meta property="og:title" content="Targeting Drug Resistance in EGFR with Covalent Inhibitors: A Structure-Based Design Approach" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0013.jpeg" /></meta><meta property="og:description" content="Receptor tyrosine kinases represent one of the prime targets in cancer therapy, as the dysregulation of these elementary transducers of extracellular signals, like the epidermal growth factor receptor (EGFR), contributes to the onset of cancer, such as non-small cell lung cancer (NSCLC). Strong efforts were directed to the development of irreversible inhibitors and led to compound CO-1686, which takes advantage of increased residence time at EGFR by alkylating Cys797 and thereby preventing toxic effects. Here, we present a structure-based approach, rationalized by subsequent computational analysis of conformational ligand ensembles in solution, to design novel and irreversible EGFR inhibitors based on a screening hit that was identified in a phenotype screen of 80 NSCLC cell lines against approximately 1500 compounds. Using protein X-ray crystallography, we deciphered the binding mode in engineered cSrc (T338M/S345C), a validated model system for EGFR-T790M, which constituted the basis for further rational design approaches. Chemical synthesis led to further compound collections that revealed increased biochemical potency and, in part, selectivity toward mutated (L858R and L858R/T790M) vs nonmutated EGFR. Further cell-based and kinetic studies were performed to substantiate our initial findings. Utilizing proteolytic digestion and nano-LC-MS/MS analysis, we confirmed the alkylation of Cys797." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b01082"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b01082">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b01082&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b01082&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b01082&amp;href=/doi/10.1021/acs.jmedchem.5b01082" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2015</span><span class="cit-fg-volume">, 58</span><span class="cit-fg-issue">, 17</span><span class="cit-fg-pageRange">, 6844-6863</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/58/17" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b00585" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.5b01095" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Targeting Drug Resistance in EGFR with Covalent Inhibitors: A Structure-Based Design Approach</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Julian++Engel">Julian Engel</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andr%C3%A9++Richters">André Richters</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Matth%C3%A4us++Getlik">Matthäus Getlik</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stefano++Tomassi">Stefano Tomassi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Marina++Keul">Marina Keul</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Martin++Termathe">Martin Termathe</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jonas++Lategahn">Jonas Lategahn</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Christian++Becker">Christian Becker</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Svenja++Mayer-Wrangowski">Svenja Mayer-Wrangowski</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Christian++Gr%C3%BCtter">Christian Grütter</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Niklas++Uhlenbrock">Niklas Uhlenbrock</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jasmin++Kr%C3%BCll">Jasmin Krüll</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Niklas++Schaumann">Niklas Schaumann</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Simone++Eppmann">Simone Eppmann</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Patrick++Kibies">Patrick Kibies</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Franziska++Hoffgaard">Franziska Hoffgaard</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jochen++Heil">Jochen Heil</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sascha++Menninger">Sascha Menninger</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sandra++Ortiz-Cuaran">Sandra Ortiz-Cuaran</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Johannes+M.++Heuckmann">Johannes M. Heuckmann</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Verena++Tinnefeld">Verena Tinnefeld</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ren%C3%A9+P.++Zahedi">René P. Zahedi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Martin+L.++Sos">Martin L. Sos</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Carsten++Schultz-Fademrecht">Carsten Schultz-Fademrecht</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Roman+K.++Thomas">Roman K. Thomas</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>◆</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stefan+M.++Kast">Stefan M. Kast</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Daniel++Rauh">Daniel Rauh</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Chemistry and Chemical Biology, TU Dortmund University, Otto-Hahn-Straße 6, D-44227 Dortmund, Germany</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Chemical Genomics Centre of the Max-Planck Society, Otto-Hahn-Straße 15, D-44227 Dortmund, Germany</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Lead Discovery Center GmbH, Otto-Hahn-Straße 15, D-44227 Dortmund, Germany</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Department of Translational Genomics, Medical Faculty, University of Cologne, Weyertal 115b, D-50931 Cologne, Germany</span></div><div class="aff-info" id="aff7"><span class="aff-symbol">⊥</span> <span class="aff-text">Leibniz-Institut für Analytische Wissenschaften − ISAS − e.V., D-44139 Dortmund, Germany</span></div><div class="aff-info" id="aff6"><span class="aff-symbol">#</span> <span class="aff-text">Molecular Pathology, University Hospital of Cologne, Kerpenerstraße 62, D-50937 Cologne, Germany</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">◆</span> <span class="aff-text">Department of Pathology, University of Cologne, Joseph-Stelzmann Straße 9, D-50931 Cologne, Germany</span></div><div class="corresp-info"><strong>*</strong>(S.M.K.) Phone: +49 (0)231-755 3906: Fax: +49 (0)231-755 3748; E-mail: <a href="/cdn-cgi/l/email-protection#84f7f0e1e2e5eaaaefe5f7f0c4f0f1a9e0ebf6f0e9f1eae0aae0e1"><span class="__cf_email__" data-cfemail="67141302010609490c0614132713124a030815130a120903490302">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>(D.R.) Phone: +49 (0)231-755 7080; Fax: +49 (0)231-755 7082; E-mail: <a href="/cdn-cgi/l/email-protection#563237383f333a782437233e1622237b323924223b233832783233"><span class="__cf_email__" data-cfemail="7e1a1f10171b12500c1f0b163e0a0b531a110c0a130b101a501a1b">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01082&amp;href=/doi/10.1021%2Facs.jmedchem.5b01082" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2015</span></span><span class="cit-volume">, 58</span><span class="cit-issue">, 17</span><span class="cit-pageRange">, 6844–6863</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 14, 2015</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>16 April 2015</li><li><span class="item_label"><b>Published</b> online</span>31 August 2015</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 September 2015</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b01082" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01082</a></div><div class="article_header-article-copyright"><strong>Copyright © 2015 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6844%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJulian%2BEngel%252C%2BAndr%25C3%25A9%2BRichters%252C%2BMatth%25C3%25A4us%2BGetlik%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D58%26issueNum%3D17%26contentID%3Dacs.jmedchem.5b01082%26title%3DTargeting%2BDrug%2BResistance%2Bin%2BEGFR%2Bwith%2BCovalent%2BInhibitors%253A%2BA%2BStructure-Based%2BDesign%2BApproach%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6863%26publicationDate%3DSeptember%2B2015">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b01082"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">6465</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">60</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b01082" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Targeting Drug Resistance in EGFR with Covalent Inhibitors: A Structure-Based Design Approach&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Julian&quot;,&quot;last_name&quot;:&quot;Engel&quot;},{&quot;first_name&quot;:&quot;André&quot;,&quot;last_name&quot;:&quot;Richters&quot;},{&quot;first_name&quot;:&quot;Matthäus&quot;,&quot;last_name&quot;:&quot;Getlik&quot;},{&quot;first_name&quot;:&quot;Stefano&quot;,&quot;last_name&quot;:&quot;Tomassi&quot;},{&quot;first_name&quot;:&quot;Marina&quot;,&quot;last_name&quot;:&quot;Keul&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;Termathe&quot;},{&quot;first_name&quot;:&quot;Jonas&quot;,&quot;last_name&quot;:&quot;Lategahn&quot;},{&quot;first_name&quot;:&quot;Christian&quot;,&quot;last_name&quot;:&quot;Becker&quot;},{&quot;first_name&quot;:&quot;Svenja&quot;,&quot;last_name&quot;:&quot;Mayer-Wrangowski&quot;},{&quot;first_name&quot;:&quot;Christian&quot;,&quot;last_name&quot;:&quot;Grütter&quot;},{&quot;first_name&quot;:&quot;Niklas&quot;,&quot;last_name&quot;:&quot;Uhlenbrock&quot;},{&quot;first_name&quot;:&quot;Jasmin&quot;,&quot;last_name&quot;:&quot;Krüll&quot;},{&quot;first_name&quot;:&quot;Niklas&quot;,&quot;last_name&quot;:&quot;Schaumann&quot;},{&quot;first_name&quot;:&quot;Simone&quot;,&quot;last_name&quot;:&quot;Eppmann&quot;},{&quot;first_name&quot;:&quot;Patrick&quot;,&quot;last_name&quot;:&quot;Kibies&quot;},{&quot;first_name&quot;:&quot;Franziska&quot;,&quot;last_name&quot;:&quot;Hoffgaard&quot;},{&quot;first_name&quot;:&quot;Jochen&quot;,&quot;last_name&quot;:&quot;Heil&quot;},{&quot;first_name&quot;:&quot;Sascha&quot;,&quot;last_name&quot;:&quot;Menninger&quot;},{&quot;first_name&quot;:&quot;Sandra&quot;,&quot;last_name&quot;:&quot;Ortiz-Cuaran&quot;},{&quot;first_name&quot;:&quot;Johannes&quot;,&quot;last_name&quot;:&quot;M. Heuckmann&quot;},{&quot;first_name&quot;:&quot;Verena&quot;,&quot;last_name&quot;:&quot;Tinnefeld&quot;},{&quot;first_name&quot;:&quot;René&quot;,&quot;last_name&quot;:&quot;P. Zahedi&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;L. Sos&quot;},{&quot;first_name&quot;:&quot;Carsten&quot;,&quot;last_name&quot;:&quot;Schultz-Fademrecht&quot;},{&quot;first_name&quot;:&quot;Roman&quot;,&quot;last_name&quot;:&quot;K. Thomas&quot;},{&quot;first_name&quot;:&quot;Stefan&quot;,&quot;last_name&quot;:&quot;M. Kast&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;Rauh&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2015&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;31&quot;,&quot;issue&quot;:&quot;17&quot;,&quot;volume&quot;:&quot;58&quot;,&quot;pages&quot;:&quot;6844-6863&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b01082&quot;},&quot;abstract&quot;:&quot;Receptor tyrosine kinases represent one of the prime targets in cancer therapy, as the dysregulation of these elementary transducers of extracellular signals, like the epidermal growth factor receptor (EGFR), contributes to the onset of cancer, such as non-small cell lung cancer (NSCLC). Strong efforts were directed to the development of irreversible inhibitors and led to compound CO-1686, which takes advantage of increased residence time at EGFR by alkylating Cys797 and thereby preventing toxic effects. Here, we present a structure-based approach, rationalized by subsequent computational analysis of conformational ligand ensembles in solution, to design novel and irreversible EGFR inhibitors based on a screening hit that was identified in a phenotype screen of 80 NSCLC cell lines against approximately 1500 compounds. Using protein X-ray crystallography, we deciphered the binding mode in engineered cSrc (T338M/S345C), a validated model system for EGFR-T790M, which constituted the basis for further rational de&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01082&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01082" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01082&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01082" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01082&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01082" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01082&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01082&amp;href=/doi/10.1021/acs.jmedchem.5b01082" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b01082" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b01082" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b01082%26sid%3Dliteratum%253Aachs%26pmid%3D26275028%26genre%3Darticle%26aulast%3DEngel%26date%3D2015%26atitle%3DTargeting%2BDrug%2BResistance%2Bin%2BEGFR%2Bwith%2BCovalent%2BInhibitors%253A%2BA%2BStructure-Based%2BDesign%2BApproach%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D58%26issue%3D17%26spage%3D6844%26epage%3D6863%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291917" title="Pyrimidine">Pyrimidine</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291026" title="Conformation">Conformation</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/58/17" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/jmcmar.2015.58.issue-17/20150910/jmcmar.2015.58.issue-17.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01082&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Receptor tyrosine kinases represent one of the prime targets in cancer therapy, as the dysregulation of these elementary transducers of extracellular signals, like the epidermal growth factor receptor (EGFR), contributes to the onset of cancer, such as non-small cell lung cancer (NSCLC). Strong efforts were directed to the development of irreversible inhibitors and led to compound CO-1686, which takes advantage of increased residence time at EGFR by alkylating Cys797 and thereby preventing toxic effects. Here, we present a structure-based approach, rationalized by subsequent computational analysis of conformational ligand ensembles in solution, to design novel and irreversible EGFR inhibitors based on a screening hit that was identified in a phenotype screen of 80 NSCLC cell lines against approximately 1500 compounds. Using protein X-ray crystallography, we deciphered the binding mode in engineered cSrc (T338M/S345C), a validated model system for EGFR-T790M, which constituted the basis for further rational design approaches. Chemical synthesis led to further compound collections that revealed increased biochemical potency and, in part, selectivity toward mutated (L858R and L858R/T790M) vs nonmutated EGFR. Further cell-based and kinetic studies were performed to substantiate our initial findings. Utilizing proteolytic digestion and nano-LC-MS/MS analysis, we confirmed the alkylation of Cys797.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85237" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85237" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The epidermal growth factor receptor (EGFR), a receptor tyrosine kinase (RTK), is a key mediator in cellular signaling related to cell growth, proliferation, survival, and migration.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> Its aberrant activity plays a pivotal role in the development and growth of tumor cells and is associated with the onset and progression of non-small cell lung cancer (NSCLC).<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4, 5)</a> Mutations in the catalytic domain of EGFR, accounting for increased kinase activity and ligand independency, have been discovered as oncogenic drivers in NSCLC. The most prevalent activating mutations are represented by the L858R, a single point substitution in exon 21, and the exon 19 deletion (delE746-A750). Patients harboring these specific activating mutations show a significant clinical response (50–80%) to first-generation reversible EGFR Type-I inhibitors gefitinib<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> and erlotinib.<a onclick="showRef(event, 'ref7 ref8 ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9 ref10 ref11 ref12">(7-12)</a> Therefore, EGFR activating mutations serve as predictive markers for the treatment of NSCLC with EGFR tyrosine kinase inhibitors (TKIs), demonstrating the potential of targeted cancer therapy. However, patients responding to these drugs develop secondary drug resistance mutations and suffer from a dramatic relapse within months. In 50% of these cases, the acquired resistance results from a mutation of the gatekeeper residue (T790M). This leads to increased ATP affinity when compared to other oncogenic primary mutant variants.<a onclick="showRef(event, 'ref13 ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15 ref16">(13-16)</a> Moreover, the sterically more demanding methionine provokes a clash with 4-aminoquinazoline-based drugs erlotinib and gefitinib and thereby induces a slightly different binding geometry that results in a complete loss of inhibitory activity.<a onclick="showRef(event, 'ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19">(17-19)</a> Second-generation EGFR inhibitors, including neratinib,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> dacomitinib,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and afatinib<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A), contain electrophilic Michael-acceptor systems to target a rare cysteine (Cys797) at the lip of the ATP binding cleft of EGFR. These new inhibitors were thought to overcome T790M drug resistance due to the covalent modification of Cys797 and the resulting increased target residence time.<a onclick="showRef(event, 'ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25">(23-25)</a> Despite promising <i>in vitro</i> results, these inhibitors exhibit insufficient efficacy in patients at clinically achievable concentrations. Thus, on-target toxicity represents the dose-limiting factor. In addition to the oncogenic mutant variants, the wild-type form of EGFR is likewise inhibited, and severe side effects during therapy are observed (e.g., skin rash and diarrhea).<a onclick="showRef(event, 'ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28">(26-28)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Covalent and reversible EGFR inhibitors. (A) First-, second-, and third-generation EGFR inhibitors. (B) Screening hit <b>1a</b> from a phenotype screen of 80 NSCLC cell lines. (C) X-ray crystal structure of <b>1a</b> in complex with cSrc-DM (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D12">5D12</a>). Michael acceptors are highlighted (green).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01082&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">For this reason, third-generation EGFR inhibitors, including WZ4002,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> CO-1686 (Rociletinib),<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30, 31)</a> and AZD9291<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32, 33)</a> (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A), have been developed, and are based on an amino pyrimidine scaffold. While WZ4002 has not progressed into human clinical trials, CO-1686 (Rociletinib) and AZD9291 are currently at the leading edge of NSCLC treatment, exhibiting beneficial effects in terms of residence time, toxicity profile, and further ADME parameters.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Notably, these compounds do not interfere with larger gatekeeper residues such as of the methionine in the T790M variant, and therefore, retain their inhibitory efficacy toward these clinically relevant mutants. The work of Walter and colleagues showed that covalent modification of EGFR by small molecule inhibitors is feasible for overcoming gatekeeper mutations in cancer treatment.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> However, only a few compounds possessing the necessary structural features are under investigation in clinical trials, and further scaffolds are needed for directed treatments.</div><div class="NLM_p last">Recently, we performed a phenotype screen of 80 NSCLC cell lines against 1500 small molecule inhibitors and identified quinazoline <b>1a</b> (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B) as having inhibitory impact on the drug-resistant H1975 cell line (L858R/T790M).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> However, we also observed significant nonspecific inhibitory effects, most likely caused by off-target activity. Nevertheless, the identified candidate is composed of a classical 4-aminoquinazoline but with a unique substitution pattern. This kinase inhibitor is decorated with a sterically demanding phenyl moiety at the 2-position that prevents the adoption of the typical quinazoline binding mode and that utilizes the 4-aminopyrazole moiety as an alternative hinge binding element (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B). Moreover, a Michael-acceptor system is incorporated in the 7-position of <b>1a</b>, which may provide the ability to target Cys797 covalently in EGFR. However, X-ray crystallography studies, utilizing engineered cSrc (T338M/S345C) as a validated model system for EGFR (T790M), did not show evidence of the desired covalent modulation of Cys345, which is homologous to Cys797 in EGFR (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>C).<a onclick="showRef(event, 'ref18 ref36'); return false;" href="javascript:void(0);" class="ref ref18 ref36">(18, 36)</a> We used engineered cSrc (T338M/S345C) as an initial model system, as it provides structural insights. However, crystal structures of EGFR-T790M in complex with irreversible ligands have been already published and likewise guided our design studies. Here, we present the follow-up chemical optimization of potent and irreversible EGFR inhibitors based on the screening hit from the phenotype screen of 80 NSCLC cell lines.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Structure-based design studies guided us to scaffolds including quinazolines and pyrimidines both targeting the drug-resistant T790M as well as the mutant variants of EGFR that harbor activating mutations. Organic synthesis led to a focused inhibitor collection, and subsequent characterization using activity-based assay systems elicited potent inhibition profiles. Using X-ray crystallography, we gained valuable insights into the mode of action of the new series of inhibitors and its structure–activity relationships (SARs) that led to the design of further derivatives with improved inhibitory activity and, accordingly, better mutant selectivity. Since restricted conformational freedom of ligands in free solution is an important beneficial factor governing binding affinity, we also characterized the population of relevant conformational basins by computational means in order to rationalize the experimental observations. This work demonstrates the strength of the interplay of medicinal chemistry methods, preparative organic chemistry, biochemistry, structure biology, and computational methods in rational drug design approaches.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15325" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15325" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The relevance of oncogenic EGFR for the treatment of NSCLC and the ongoing interest in the role of acquired drug resistant mutations in targeted cancer therapy emphasize the need for strategies to identify and improve new kinase inhibitors targeting gatekeeper mutant forms of oncogenes, such as EGFR-T790M. Recently, these efforts were focused on the development of irreversible inhibitors, which covalently modify Cys797 at the lip of the ATP binding cleft.<a onclick="showRef(event, 'ref20 ref21 ref22 ref30 ref32'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22 ref30 ref32">(20-22, 30, 32)</a> Modulators that incorporate thiol-reactive groups are beneficial, as they increase the target residence time at the protein of interest upon covalent binding. Therefore, the off-rate of these inhibitors is directly associated with the cellular turnover rate of the respective protein.<a onclick="showRef(event, 'ref23 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref23 ref37 ref38">(23, 37, 38)</a></div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Rational Design of EGFR Inhibitors Based on Screening Hit Analysis</h3><div class="NLM_p">Intrigued by the substitution pattern of <b>1a</b>, we recently solved the crystal structure of <b>1a</b> in complex with mutated cSrc to gain deeper structural insights into the binding mode and to constitute a structural basis for further design and optimization cycles (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>C). Here, we used the previously engineered and validated cSrc kinase (T338M/S345C) as a model system for drug-resistant EGFR-T790M that has been proven to be appropriate to provide primarily structural information.<a onclick="showRef(event, 'ref18 ref36'); return false;" href="javascript:void(0);" class="ref ref18 ref36">(18, 36)</a></div><div class="NLM_p">Notably, the cocrystal structure of <b>1a</b> and cSrc revealed a novel but reversible binding mode in which the 4-aminopyrazole moiety addresses the hinge region by three direct hydrogen bonds to the peptide backbone of residues Glu339 and Met341. Furthermore, the hinge binding element does not affect the gatekeeper residue (Met338). Moreover, the Michael-acceptor system in the 7-position of the quinazoline is turned outward from the ATP binding cleft, whereas the 2-phenyl moiety accesses the binding pocket, preventing the formation of a covalent bond to Cys345. Therefore, the observed inhibitory effect is not caused by covalent modification of EGFR, which is consistent with the inhibitory potency of the reversible counterpart <b>1b</b> as compared to <b>1a</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Both <b>1a</b> and <b>1b</b> did not show any significant kinase inhibition of EGFR-WT and illustrated comparable inhibitory effects on the activating mutant of EGFR (1.9 vs 2.1 μM for <b>1a</b> and <b>1b</b>, respectively) and on the drug-resistant mutant variant (2.2 and 2.2 μM), strengthening the conclusion that the Michael acceptor adopts a nonreactive binding geometry in which it does not impact the inhibitory efficacy of <b>1a</b> as compared to <b>1b</b>. This remarkable observation, in turn, allows structure-guided development of inhibitors to be undertaken with respect to the improved inhibitory effects evoked by covalent modification of Cys797. On the basis of this X-ray structure, we generated superimposed models to further investigate the options for the covalent bond formation and proposed various scaffolds and substitution patterns capable of covalent modification (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). The phenyl moiety in the 2-position of <b>1a</b> represented a convenient attachment point for incorporating a Michael acceptor (e.g., acrylamide) (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A). We also proposed a pyrimidine as an alternative scaffold. This variant retains the aminopyrazole moiety as the hinge binding element and may increase flexibility due to the reduced core structure (pyrimidine vs quinazoline). We suggested two different linkages of the Michael acceptor system to the pyrimidine core to investigate different conformations and to evaluate the preferred geometry and orientation of the electrophile. Therefore, the electrophile was attached directly (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B) and by a linked phenylether (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>C,D) to the pyrimidine core. Moreover, we paid particular attention to the optimization of the hinge binding element with the aim to enhance the interdependency between the methionine gatekeeper and the ligand by expanding the hydrophobic character of the 4-substituent (e.g., 1-methylpyrazole in <b>5b</b> and indazole in <b>6a</b>). Recently, this interaction was shown to have a dramatic effect on inhibitory activity and mutant selectivity in the case of various EGFR inhibitors.<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39, 40)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Modeling studies of various hinge-binding motifs. (A) Quinazoline- (<b>2b</b>) and (B–D) pyrimidine-based inhibitors (<b>3a</b>, <b>5b</b>, and <b>6a</b>) based on the X-ray crystal structure of <b>1a</b> in complex with cSrc-DM (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D12">5D12</a>). Proposed hydrogen bonds are shown as red dotted lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01082&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Overview of a Focused Small Molecule Library of Pyrimidine- and Quinazoline-Based Inhibitors and Corresponding IC<sub>50</sub> Determinations on Different EGFR Mutant Variants</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0011.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0012.gif" alt="" id="fx2" /></img><div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Synthesis of a Focused Small Molecule Library</h3><div class="NLM_p">We set out to synthesize a focused library of pyrimidine-based inhibitors with diverse substitution patterns. Our efforts were directed toward the optimal orientation of the Michael-acceptor system with respect to the alkylation of Cys797 in EGFR. Moreover, we focused on the derivatization of the hinge binding element to enhance the hydrophobic interaction with the gatekeeper residue while simultaneously retaining the crucial direct hinge interactions. Therefore, seven different synthetic routes were established to produce pyrimidine-based compounds <b>3a</b>–<b>b</b>, <b>4</b>, <b>5l</b>–<b>m</b> and <b>5a</b>–<b>k</b>, <b>5n</b>–<b>o</b>, <b>5p</b>–<b>q</b>, <b>6a</b>–<b>e</b>, and <b>7</b>. In the case of <b>3a</b>–<b>b</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>), we started with a nucleophilic aromatic substitution using 2-phenyl-substituted pyrimidine <b>8</b> and 5-aminopyrazole, resulting in intermediate <b>9</b>. Subsequent decoration of the pyrazole with the Boc-protecting group led to <b>10</b>, which was then converted into <b>3a</b> and its reversible counterpart <b>3b</b> by amide formation using acryloyl or propionyl chloride and subsequent deprotection with TFA.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0008.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of a Subset of Pyrimidine Derivatives (<b>3a</b>–<b>b</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01082&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) 3-aminopyrazole, NaI, HCl (cat.), <i>n</i>-BuOH, 120 °C, 72%; (ii) Boc<sub>2</sub>O, Et<sub>3</sub>N, DCM, rt, 21%; (iii) acryloyl or propionyl chloride, DIPEA, THF, 0 °C; (iv) TFA in DCM, rt, 58–64% (two steps).</p></p></figure><div class="NLM_p">To retain the flexibility of the pyrimidine-based inhibitors, but to vary the substituent at the 2- and 4-position, we synthesized several similar derivatives by the synthetic route shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Driven by our modeling studies (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>), we proposed different attachment points for incorporating the Michael-acceptor moiety to scrutinize and optimize the geometry and orientation toward Cys797. Therefore, we incorporated an electrophile utilizing a phenyl linker (<b>4</b>), and we likewise introduced a phenolic linkage to gain increased flexibility that allows for altered and potentially beneficial orientation toward Cys797 (<b>5l</b>). Also, we proposed further molecules incorporating a proton at the pyrimidine 6-position to gain information about the significance of a solubility group on this position. 2,4-Dichloropyrimidine was preferentially decorated with 3-methyl-1<i>H</i>-pyrazol-5-amine at the 4-position to afford key intermediate <b>12</b>. Subsequently, the pyrazole moiety was protected using dihydropyran. Then, (3-nitrophenyl)boronic acid was used to introduce the 3-nitrophenyl moiety into the pyrimidine 2-position under Suzuki conditions, resulting in intermediate <b>14a</b>, which was reduced to the corresponding amine <b>14b</b>. The acrylamide Michael-acceptor system was introduced by amide formation using acryloyl chloride. THP deprotection with TFA led to compound <b>4</b>. Starting from key intermediate <b>12</b>, we likewise generated compounds with a bridging ether at the 2-position to increase the conformational flexibility of the Michael-acceptor-decorated moiety. Nucleophilic aromatic substitution of <b>12</b> utilizing 3-nitrophenol gave intermediate <b>15a</b>, and subsequent reduction with Pd/C and ammonium formate led to the amino equivalent <b>15b</b>. Reaction of amine <b>15b</b> with various acid chlorides yielded compounds <b>5l</b>–<b>m</b>.</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0009.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of a Subset of Pyrimidine Derivatives (<b>4a</b>, <b>5l</b>–<b>m</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01082&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) 5-methyl-1<i>H</i>-pyrazol-3-amine, DIPEA, DMF, 90 °C, 54%; (ii) DHP, <i>p</i>TsOH, THF, 90 °C, 64%; (iii) (3-nitrophenyl)boronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, DMF/H<sub>2</sub>O (9:1), 90 °C, 70%; (iv) ammonium formate, Pd/C, EtOH, 90 °C; (v) acryloyl chloride, DIPEA, THF, 0 °C, 26% (<b>5l</b>); (vi) TFA in DCM, rt, 73% (two steps); (vii) 3-nitrophenol, K<sub>2</sub>CO<sub>3</sub>, DMF, 140 °C, 20%; (viii) propionyl chloride, DIPEA, THF, 0 °C, 27%.</p></p></figure><div class="NLM_p">In a third approach (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>), we applied a generic route to build up 2,4,6-substituted pyrimidine compounds <b>5a</b>–<b>k</b>, <b>5n</b>–<b>q</b>, and <b>6a</b>–<b>d</b>. Initially, 4,6-dichloro-2-(methylthio)pyrimidine was oxidized to the corresponding sulfone <b>17</b> using <i>m</i>CPBA to generate an appropriate leaving group at the pyrimidine 2-position. Consecutive nucleophilic aromatic substitution with the ortho-, meta-, or para-substituted nitrophenol derivatives led to intermediates <b>18a</b>–<b>c</b>. Substitution of one aromatic chlorine with either amino-pyrazoles or amino-indazoles resulted in intermediates <b>19a</b>–<b>c</b> and <b>20</b>–<b>22</b>, which were then derivatized with 1-methylpiperazine at the pyrimidine 6-position, providing <b>23a</b>–<b>c</b> and <b>24</b>–<b>26</b>. The decoration of the pyrazoles or indazoles with the Boc-protecting group led to <b>27b</b>–<b>c</b> and <b>28</b>–<b>30</b>. Subsequent reduction of the aromatic nitro-group was performed by using Pd/C and ammonium formate and resulted in the corresponding amino derivatives <b>31b</b>–<b>c</b> and <b>32</b>–<b>34</b>. Ultimately, these amines were decorated with Michael acceptors using acid chlorides or carboxylic acids, under standard amide coupling conditions (EDC/HOBt), respectively. The reversible counterparts were produced accordingly utilizing propionyl chloride. Final Boc-deprotection was conducted using TFA and led to compounds <b>5a</b>–<b>k</b>, <b>5n</b>–<b>q</b>, and <b>6a</b>–<b>d</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>).</div><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0010.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of a Subset of Pyrimidine Derivatives (<b>5a</b>–<b>k</b>, <b>5n</b>–<b>q</b>, <b>6a</b>–<b>d</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01082&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) <i>m</i>CPBA, DCM/THF, 0 °C – rt, 86%; (ii) nitrophenol, NaH, THF, 0 °C – rt, 29–84%; (iii) amine, DIPEA, NaI, DMF, 85–110 °C, 60–82%; (iv) methylpiperazine, 110 °C, 62–82%; (v) Boc<sub>2</sub>O, Et<sub>3</sub>N, MeOH, rt or Boc<sub>2</sub>O, Cs<sub>2</sub>CO<sub>3</sub>, THF, rt, 56–95%; (vi) ammonium formate, Pd/C, EtOH or EtOAc, 90 °C, 48–87%; (vii) acryloyl or propionyl chloride, DIPEA, THF, 0 °C; (viii) (<i>E</i>)-4-(dimethylamino)but-2-enoic acid or (<i>E</i>)-4,4,4-trifluorobut-2-enoic acid, EDC, HOBt, DIPEA, DCM, rt or (<i>E</i>)-4-(dimethylamino)but-2-enoic acid, oxalyl chloride, DMF(cat.), THF, rt; (ix) TFA in DCM, rt, 33–95% (two steps).</p></p></figure><div class="NLM_p last">Synthetic schemes and preparations of quinazoline-based compounds <b>1a</b>–<b>b</b> and <b>2a</b>–<b>f</b> have been recently described elsewhere.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Synthetic schemes of compounds <b>6e</b> and <b>7</b> as well as detailed synthetic procedures are shown in the Supporting Information (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01082/suppl_file/jm5b01082_si_002.pdf" class="ext-link">Schemes S1 and S2</a>, respectively).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Activity-Based Characterization of Quinazoline and Pyrimidine-Based Scaffolds on EGFR Inhibition</h3><div class="NLM_p">We characterized the focused compound collection against various types of EGFR including wild-type and mutant forms L858R and L858R/T790M (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) using an activity-based assay that quantifies phosphorylation of an artificial substrate by a particular kinase through homogeneous time-resolved FRET measurements (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). Since <b>1a</b> was decorated with an acrylamide warhead, we tested the reversible counterpart and observed equal potency on EGFR-L858R and L858R/T790M (IC<sub>50</sub> ∼ 2.0 μM), whereas <b>1a</b> displayed a reduced effect on the wild-type form of EGFR (IC<sub>50</sub> > 10 μM). However, due to these results and in concordance with the observed reversible binding mode (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>C), we generated further quinazoline-based compounds (<b>2a</b>–<b>f</b>) with the Michael acceptors attached to different positions (ortho, meta, para) to increase the probability of alkylating Cys797. Compounds <b>2b</b> and <b>2e</b> demonstrated significant inhibitory effects in the investigated EGFR variants. We observed IC<sub>50</sub> values of 1.1 μM (wt), 0.5 μM (L858R), and 0.1 μM (L858R/T790M) for compound <b>2b</b>, whereas the reversible counterpart <b>2e</b> was 4 times less potent (3.7, 1.9, and 0.4 μM, respectively), indicating that, in contrast to <b>2b</b>, no covalent bond was formed with <b>2e</b>. Altering the attachment point of the acrylamide/propionamide motif to the ortho and para sites (<b>2a</b>, <b>2d</b> and <b>2c</b>,<b> 2f</b>) did not result in notable inhibitory effects, except for <b>2a</b>, with moderate inhibitory effects on L858R (2.8 μM) and L858R/T790M (1.1 μM).</div><div class="NLM_p">When we decreased the spatial arrangement by utilizing a pyrimidine core instead of a quinazoline (<b>3a</b>), we achieved similar IC<sub>50</sub> values (0.9, 0.8, 0.3 μM), although the Michael acceptor was not attached to the 2-phenyl moiety, as in the case of <b>2a</b>–<b>f</b>, but was at the pyrimidine 6-position instead. Both attachment points were likewise eligible to elicit potent inhibition on EGFR. Furthermore, the reversible counterpart <b>3b</b> did not significantly inhibit EGFR, supporting our hypothesis of covalent bond formation between Cys797 and <b>3a</b>.</div><div class="NLM_p">On the basis of modeling studies (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>), we attached the Michael acceptor to the 2-phenyl moiety (<b>4</b>) to increase inhibitory activity. We incorporated an additional methyl group at the pyrazole 5-position, which we proposed would form hydrophobic interactions with the methionine gatekeeper residue in EGFR-L858R/T790M and increase the compound’s selectivity toward this variant. Several EGFR inhibitors have been equipped with either halogen or methyl substituents directed toward the methionine gatekeeper residue.<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29, 30)</a> However, we did not observe any beneficial effect for compound <b>4</b> with respect to the inhibitory potency toward the investigated EGFR variants (IC<sub>50</sub> > 10 μM (wt), 2.5 μM (L858R), and 2.3 μM (L858R/T790M)). Further approaches were undertaken to position the Michael acceptor in proximity to Cys797 by connecting the phenylacrylamide through a flexible ether linkage. We speculated that this would translate into an improved orientation toward the reactive cysteine and favor subsequent covalent bond formation. While the wild-type form was not inhibited (IC<sub>50</sub> > 10 μM), we observed moderate inhibition of the L858R (1.8 μM) as well as the L858R/T790M mutant (0.9 μM) by <b>5l</b>. The reversible counterpart <b>5m</b> certainly did not elicit substantial inhibitory activity toward any of the EGFR variants.</div><div class="NLM_p">These results prompted us to introduce a solubilizing group (1-methylpiperazine) to the 6-position of the pyrimidine to restrict the structural flexibility with respect to hinge binding and optimally orient the 2-phenyl moiety to preserve its conformational flexibility to reach Cys797 for covalent modification. Compounds <b>5a</b>–<b>c</b> were generated to investigate the influence of ortho-, meta-, and para-substituted acrylamides on the inhibition and also on the formation of a covalent bond by the thiol-reactive warhead. For <b>5a</b> (ortho), only moderate inhibition was observed for wild-type (IC<sub>50</sub> = 0.9 μM), L858R (IC<sub>50</sub> = 0.6 μM), and L858R/T790M (IC<sub>50</sub> = 2.1 μM). Equal behavior was demonstrated for the corresponding para-substituted compound <b>5c</b> (3.1 μM, wt; 1.8 μM, L858R; 2.4 μM, L858R/T790M). Notably, meta-substitution as in <b>5b</b> led to remarkable inhibition of EGFR-L858R (IC<sub>50</sub> = 0.03 μM), whereas the double mutant form was less addressed (IC<sub>50</sub> = 0.5 μM). Interestingly, <b>5b</b> did not show any significant inhibition of the wild-type. With respect to the reversible counterpart <b>5e</b>, which is equipped with a propionyl moiety that cannot undergo a Michael addition to form a covalent bond to Cys797, we observed a dramatic loss in inhibitory activity against both activating and drug-resistant mutant variants of EGFR (60-fold and about 20-fold, respectively) as compared to its covalent counterpart <b>5b</b>, thus indicating that <b>5b</b> may interact with EGFR in a covalent manner. This observation further strengthened the conclusion that covalently targeting EGFR-T790M is a crucial feature to achieve sufficient efficacy. The corresponding <b>5n</b> solely possesses a proton instead of the methyl substitution at the pyrazole 5-position and therefore resulted in moderate inhibitory activity on the mutant variants (2.0 and 1.9 μM for L858R and L858R/T790M, respectively), whereas no inhibitory activity was observed against the wild-type, supporting the concept that favorable hydrophobic gatekeeper interactions may lead to improved protein–ligand interactions. To further strengthen this concept, we introduced an indazole into the most promising compound, <b>5b</b>, to increase the spatial and hydrophobic character while retaining the crucial hinge binding motif as well as the meta-substituted phenylether as a linker system for the electrophile, resulting in <b>6a</b>. We observed a strikingly increased effect on the drug-resistant mutant of EGFR (IC<sub>50</sub> = 0.07 μM) for <b>6a.</b> The mutant variant harboring the activating mutation is 7-fold less well inhibited (0.5 μM, L858R), whereas only moderate activity could be observed against the wild-type (IC<sub>50</sub> = 1.7 μM), indicating a favorable interaction of the methionine gatekeeper and the indazole moiety. With respect to the reversible counterpart (<b>6b</b>), we did not observe an inhibitory effect on any of the EGFR variants, thereby demonstrating the importance of a covalent bond. We intended to further increase the gatekeeper interacting element by introducing a bromine in the 5-position of the indazole and likewise to sterically constrain the inhibitor’s conformation (<b>6d</b>). We observed similar behavior for <b>6d</b> against the wild-type (IC<sub>50</sub> = 2.9 μM) as well as against the mutant form EGFR-L858R (0.4 μM), whereas <b>6d</b> displayed a slightly reduced effect against the drug-resistant mutant variant (0.3 μM, L858R/T790M). In order to investigate the electrophilic characteristics of the Michael-acceptor system, we equipped the acrylamide with an additional dimethylamino moiety (compound <b>6c</b>), which is supposed to increase the reactivity of the thiol nucleophile for subsequent Michael addition (Cys797) by intramolecular base catalysis.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> This modification, however, led to a significant loss of inhibitory activity against all investigated EGFR mutants (>10 μM, wt; 3.6 μM, L858R; 0.7 μM, L858R/T790M). These observations indicate that the acrylamide moiety represents a more favored electrophile for this scaffold in terms of size, reactivity, and orientation.</div><div class="NLM_p">Notably, initial ESI-MS analyses of EGFR-T790M treated with <b>3a</b>, <b>5b</b>, or <b>6a</b> resulted in mass increases equivalent to the corresponding single-labeled EGFR-T790M (303 Da (<b>3a</b>), 436 Da (<b>5b</b>), and 468 Da (<b>6a</b>), respectively), as compared to control EGFR-T790M treated with DMSO, indicating a covalent inhibitor adduct with EGFR-T790M (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01082/suppl_file/jm5b01082_si_002.pdf" class="ext-link">Figure S1</a>). Furthermore, utilizing proteolytic digestion of <b>5b</b> and <b>6a</b> and nano-LC-MS/MS analysis, we precisely confirmed selective Cys797 modification for both compounds (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.5b01082/suppl_file/jm5b01082_si_002.pdf" class="ext-link">Supporting Information</a>).</div><figure id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Extracted ion chromatograms (XICs) of the three different isoforms of the peptide GCLLDYVR identified in (A) the control sample and after treatment with (B) <b>5b</b> and (C) <b>6a</b>. As expected, the carbamidomethylated (i.e., nonmodified) peptide is almost absent in samples the inhibitor-treated samples, whereas the corresponding modified versions can be detected only in the respective samples. All XICs are shown with a tolerance of 4 ppm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01082&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Activity-Based Characterization of Quinazoline and Pyrimidine-Based Scaffolds on the EGFR Surrogate System cSrc</h3><div class="NLM_p last">To account for the validity of our EGFR surrogate system, we likewise tested the focused set of compounds on cSrc and selected mutant variants. In particular, we determined the IC<sub>50</sub> values with respect to cSrc wild-type that lacks both a large gatekeeper and a reactive Cys at the lip of the ATP pocket. Furthermore, we investigated the inhibitory activities on engineered cSrc gatekeeper mutant variant T338M as well as on a mutant incorporating the gatekeeper mutation T338M and the reactive cysteine (S345C) to serve as model system for EGFR-T790M (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01082/suppl_file/jm5b01082_si_002.pdf" class="ext-link">Table S1</a>). For the parent compound <b>1a</b>, we observed moderate inhibitory activity toward cSrc wild-type (IC<sub>50</sub> = 0.211 μM), whereas inhibition of cSrc T338M, as well as T338M/S345C, was 6- to 9-fold stronger, but it did not improve with introducing S345C (IC<sub>50</sub> = 0.065 μM on T338M vs 0.043 μM on T338M/S345C), indicating that the mutant form is addressed more efficiently but not covalently in the case of T338M/S345C. The reversible counterpart <b>1b</b> likewise reflects improved inhibitory activity on the mutated variants as compared to that on wild-type cSrc, substantiating our conclusion (IC<sub>50</sub> = 0.336 vs 0.060 vs 0.048 μM). Improved inhibitory effects toward the gatekeeper mutant forms of cSrc were observed and reflect a general trend throughout the focused compound collection (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01082/suppl_file/jm5b01082_si_002.pdf" class="ext-link">Table S1</a>). However, for <b>5b</b>, we observed distinct differences among the mutated variants. In detail, an IC<sub>50</sub> value of 0.170 μM was determined for cSrc T338M, whereas cSrc T338M/S345C was inhibited 5-fold more efficiently (IC<sub>50</sub> = 0.028 μM). On the contrary, we did not observe increasing potency for the analogous reversible compound <b>5e</b> (IC<sub>50</sub> = 0.254 and 0.349 μM, respectively). Analogous characteristics were found for <b>6a</b> and <b>6b</b>, accounting for more efficient inhibition of cSrc T338M/S345C in the case of Michael-acceptor-decorated <b>6a</b> (IC<sub>50</sub> = 0.017 μM) as compared to that with the reversible analogue <b>6b</b> (IC<sub>50</sub> = 0.260 μM). These results validated further the use of engineered cSrc as a surrogate system for EGFR and its mutants.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Kinetic Characterization of Covalent Bond Formation of <b>5b</b> and <b>6a</b> to EGFR Mutant Variants</h3><div class="NLM_p">Covalent binding of inhibitors to their respective target proteins requires distinct spatial orientations that allow for efficient and irreversible modification of a unique cysteine. In particular, Cys797 in EGFR serves as the anchor point for irreversible inhibitors equipped with warheads. In order to investigate the rate and efficiency of covalent bond formation of our most effective inhibitors, <b>5b</b> and <b>6a</b>, we set out to determine the corresponding <i>K</i><sub>i</sub> and <i>k</i><sub>inact</sub> parameters with respect to the mutant variants L858R and T790M/L858R of EGFR by utilizing an activity-based assay system (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). We monitored the respective IC<sub>50</sub> values of <b>5b</b> and <b>6a</b> after treatment of the respective proteins in a time-dependent manner and correlated the according values to the respective incubation times, which then translated into the determination of <i>K</i><sub>i</sub> and <i>k</i><sub>inact</sub> as described in the literature (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>).<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> On the basis of the kinetic characterization, we demonstrated <b>5b</b> as well as <b>6a</b> to exhibit particularly high affinity toward EGFR L858R/T790M (<i>K</i><sub>i</sub> = 0.64 and 0.32 nM, respectively), whereas they solely account for specific, moderate reactivity (<i>k</i><sub>inact</sub> = 0.116 and 0.137 min<sup>–1</sup>, respectively) as compared to that of known reference compounds.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> On the contrary, we likewise illustrated that both the affinity and specific reactivity of <b>5b</b> and <b>6a</b> toward EGFR L858R are significantly impaired (<i>K</i><sub>i</sub> = 70.2 and 833 nM, respectively; <i>k</i><sub>inact</sub> = 0.017 and 0.055 min<sup>–1</sup>, respectively) as compared to that toward EGFR L858R/T790M.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Overview of Kinetic Parameters <i>K</i><sub>i</sub>, <i>k</i><sub>inact</sub>, and <i>k</i><sub>inact</sub>/<i>K</i><sub>i</sub> Determined for <b>5b</b> and <b>6a</b> on EGFR L858R and L858R/T790M</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">EGFR</th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>i</sub> [nM]</th><th class="colsep0 rowsep0" align="center" char="±"><i>k</i><sub>inact</sub> [min<sup>–1</sup>]</th><th class="colsep0 rowsep0" align="center" char="±"><i>k</i><sub>inact</sub>/<i>K</i><sub>i</sub> [μM<sup>–1</sup> s<sup>–1</sup>]</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left"><b>5b</b></td><td class="colsep0 rowsep0" align="left">L858R</td><td class="colsep0 rowsep0" align="char" char="±">70.2 ± 8.23</td><td class="colsep0 rowsep0" align="char" char="±">0.017 ± 0.007</td><td class="colsep0 rowsep0" align="char" char="±">0.004 ± 0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">L858R/T790M</td><td class="colsep0 rowsep0" align="char" char="±">0.64 ± 0.30</td><td class="colsep0 rowsep0" align="char" char="±">0.116 ± 0.082</td><td class="colsep0 rowsep0" align="char" char="±">2.83 ± 0.96</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left"><b>6a</b></td><td class="colsep0 rowsep0" align="left">L858R</td><td class="colsep0 rowsep0" align="char" char="±">833 ± 429</td><td class="colsep0 rowsep0" align="char" char="±">0.055 ± 0.022</td><td class="colsep0 rowsep0" align="char" char="±">0.001 ± 0.0002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">L858R/T790M</td><td class="colsep0 rowsep0" align="char" char="±">0.32 ± 0.10</td><td class="colsep0 rowsep0" align="char" char="±">0.137 ± 0.091</td><td class="colsep0 rowsep0" align="char" char="±">7.01 ± 4.83</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">CO-1686</td><td class="colsep0 rowsep0" align="left">L858R</td><td class="colsep0 rowsep0" align="char" char="±">1.75 ± 0.19</td><td class="colsep0 rowsep0" align="char" char="±">0.179 ± 0.053</td><td class="colsep0 rowsep0" align="char" char="±">1.67 ± 0.32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">L858R/T790M</td><td class="colsep0 rowsep0" align="char" char="±">1.65 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">0.289 ± 0.052</td><td class="colsep0 rowsep0" align="char" char="±">2.95 ± 0.66</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">AZD9291</td><td class="colsep0 rowsep0" align="left">L858R</td><td class="colsep0 rowsep0" align="char" char="±">1.58 ± 0.29</td><td class="colsep0 rowsep0" align="char" char="±">0.299 ± 0.014</td><td class="colsep0 rowsep0" align="char" char="±">3.24 ± 0.46</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">L858R/T790M</td><td class="colsep0 rowsep0" align="char" char="±">1.46 ± 0.13</td><td class="colsep0 rowsep0" align="char" char="±">0.331 ± 0.064</td><td class="colsep0 rowsep0" align="char" char="±">3.75 ± 0.39</td></tr></tbody></table></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Complex Crystal Structures of <b>5b</b> and <b>5c</b></h3><div class="NLM_p">In order to investigate the binding modes of our newly designed inhibitors and to substantiate the observed SARs, we generated complex crystal structures of representative compounds of our focused library. We cocrystallized <b>5b</b> and <b>5c</b> (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>) in complex with engineered cSrc (T338M/S345C) as the model system validated for EGFR-T790M.<a onclick="showRef(event, 'ref18 ref36'); return false;" href="javascript:void(0);" class="ref ref18 ref36">(18, 36)</a> We focused on <b>5b</b> and <b>5c</b>, as they exclusively differ in the substitution pattern of the attachment point of the Michael acceptor to the phenylether linker system, but they exhibited a dramatic discrepancy in inhibitory activity toward the investigated EGFR mutant variants.</div><figure id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Pyrimidine-based compounds in complex with cSrc-DM. Diagrams of the experimental electron densities of cSrc-DM–<b>5b</b> at 2.3 Å (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D11">5D11</a>) (A) and cSrc-DM–<b>5c</b> at 2.7 Å (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D10">5D10</a>) (B) resolution are shown (2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map contoured at 1σ). Hydrogen-bond interactions of the inhibitors with the hinge region (wheat) are illustrated by red dotted lines. The structural elements comprising the DFG motif as well as helix C are displayed in violet and cyan, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01082&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In accordance with the X-ray structure of <b>1a</b>, the complex crystal structures of both <b>5b</b> and <b>5c</b> revealed a hinge contact addressed by three hydrogen bonds to the peptide backbone of residues Glu339 and Met341. The pyrazole amine forms a hydrogen bond to the carbonyl group of Glu339, whereas Met341 is addressed by <i>N</i>2 of the pyrazole that serves as the hydrogen-bond donor. The 3-amino moiety of the pyrazole forms the third hydrogen bond to the carbonyl group of Met341. The cocrystal structure of cSrc-DM–<b>5b</b> confirms the covalent bond between Cys345 and the inhibitor. The phenylether carrying an acrylamide in the meta-position adopted a perpendicular orientation toward the pyrimidine core, thereby facilitating a close proximity and an optimal geometry of the electrophile toward Cys345 that was crucial for alkylation. The 6-substituted methylpiperazine extended to the solvent-exposed region of the ATP binding cleft, functioning as a solubilizing group. As anticipated by modeling studies, the newly introduced methyl group in the 5-position of the pyrazolo group accurately points toward the methionine gatekeeper, most likely inducing hydrophobic protein–ligand interactions. The cocrystal structure cSrc-DM–<b>5c</b>, in contrast, revealed a reversible binding mode for <b>5c</b>, in which the phenylether (acrylamide in the para-position) adopted a stretched or slightly bent conformation, accessing a subpocket adjacent to the gatekeeper residue and thereby preventing the formation of a covalent bond to Cys345.</div><div class="NLM_p last">To date, only covalent inhibitors have been sufficiently effective at overcoming T790M drug resistance. Therefore, these X-ray structures demonstrate the exceptional importance of optimized Michael-acceptor systems for each inhibitor scaffold with respect to orientation, basicity, and reactivity. These remarkable structural observations reveal a novel scaffold possessing the ability to form a covalent bond and they likewise elucidate the opposing inhibitory characteristics of <b>5b</b> and <b>5c</b>. Furthermore, this structure-guided approach provides valuable insights into the SAR and may stimulate further approaches to develop covalent inhibitors targeting drug resistance in EGFR.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Evaluation of Selected Derivatives on Drug-Resistant NSCLC Cell Line</h3><div class="NLM_p">We investigated inhibitory effects of a selected subset of representative quinazoline- and pyrimidine-based compounds in preliminary studies utilizing the drug-resistant NSCLC cell line H1975, and we additionally included the HCC827 (exon 19 deletion) and metastatic lung cancer cells harboring wild-type EGFR (A431 and H661; <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). Each cell type was treated with quinazoline <b>1a</b> or 2,4,6-substituted pyrimidines <b>5b</b>, <b>5e</b>, <b>5j</b>, <b>6a</b>, <b>6b</b>, <b>6c</b>, <b>6d</b>, <b>6e</b>, or <b>7</b> (0–30 μM) for 96 h. Although several covalent inhibitors of our focused library displayed strong inhibitory effects in biochemical evaluations using isolated kinases, we observed poor translation to cellular potency. Although parent compound <b>1a</b> equally addressed NSCLC lines harboring wild-type and gatekeeper mutated EGFR (GI<sub>50</sub> = 0.7, 1.9, and 1.6 μM for H661, HCC827, and H1975, respectively), none of the structurally altered pyrimidines displayed improved effects in terms of inhibitory efficacy and mutant selectivity. For instance, a distinct biochemical selectivity profile was observed for <b>6a</b> and its reversible counterpart <b>6b</b> targeting the mutant form of EGFR while sparing the wild-type (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). On the contrary, this trend was not reflected in cellular evaluations since <b>6a</b> solely exhibited moderate effects on the activating and drug-resistant cell lines HCC827 and H1975 (GI<sub>50</sub> = 2.5 and 9.8 μM, respectively) and likewise displayed comparable inhibitory activity on the wild-type cell line A431 (GI<sub>50</sub> = 8.6 μM). However, the reversible counterpart <b>6b</b> did not display any significant effect on the NSCLC cell lines, thereby indicating a certain beneficial cellular effect caused by the inhibitor’s properties to form covalent adducts. We observed a slightly different tendency for <b>5b</b> and <b>5e</b> since cellular evaluations led to equal GI<sub>50</sub> values of 5.7 and 8.4 μM, respectively, on EGFR wild-type cell lines, but, on the contrary, <b>5e</b> did not account for any significant inhibitory effect in HCC827 and H1975 at concentrations up to 30 μM, whereas Michael-acceptor-equipped derivative <b>5b</b> illustrated moderate inhibition on both activating and drug-resistant mutant cell lines (GI<sub>50</sub> = 8.2 and 20 μM). Of note, nitro-decorated compound <b>5h</b> represents one of the most active inhibitors in our cellular setting, eliciting GI<sub>50</sub> values of 3.5, 8.9, and 6.5 μM toward H661, HCC827, and H1975, respectively.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. GI<sub>50</sub> Values of a Selected Subset of Representative Quinazolines- and Pyrimidine-Based Compounds on NSCLC Cell Lines A431, H661, HCC827, and H1975<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center">NSCLC cell lines GI<sub>50</sub> (μM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">A431/H661 wt</th><th class="colsep0 rowsep0" align="center">HCC827 delE746-A750</th><th class="colsep0 rowsep0" align="center">H1975 L858R/T790M</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1a</b></td><td class="colsep0 rowsep0" align="left">0.7<a class="ref internalNav" href="#t3fn4" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">1.9<a class="ref internalNav" href="#t3fn5" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5b</b></td><td class="colsep0 rowsep0" align="left">5.7<a class="ref internalNav" href="#t3fn4" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">8.2 ± 0.7</td><td class="colsep0 rowsep0" align="left">20 ± 3.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5e</b></td><td class="colsep0 rowsep0" align="left">8.4<a class="ref internalNav" href="#t3fn4" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">>30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5h</b></td><td class="colsep0 rowsep0" align="left">3.5<a class="ref internalNav" href="#t3fn4" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">8.9 ± 0.7</td><td class="colsep0 rowsep0" align="left">6.5 ± 2.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5j</b></td><td class="colsep0 rowsep0" align="left">20<a class="ref internalNav" href="#t3fn4" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">>30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6a</b></td><td class="colsep0 rowsep0" align="left">8.6 ± 2.3<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">2.5 ± 0.9</td><td class="colsep0 rowsep0" align="left">9.8 ± 2.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6b</b></td><td class="colsep0 rowsep0" align="left">22 ± 2.2<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">30 ± 0.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">>30<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">n.i.<a class="ref internalNav" href="#t3fn2" aria-label="e">e</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6d</b></td><td class="colsep0 rowsep0" align="left">11 ± 2.3<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">8.6 ± 1.6</td><td class="colsep0 rowsep0" align="left">18 ± 3.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6e</b></td><td class="colsep0 rowsep0" align="left">>30<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">25 ± 4.0</td><td class="colsep0 rowsep0" align="left">n.i.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">12 ± 1.1<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">16 ± 1.7</td><td class="colsep0 rowsep0" align="left">11 ± 0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CO-1686</td><td class="colsep0 rowsep0" align="left">1.7 ± 0.1<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">0.03 ± 0.007</td><td class="colsep0 rowsep0" align="left">0.03 ± 0.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AZD9291</td><td class="colsep0 rowsep0" align="left">0.6 ± 0.08<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">>0.01</td><td class="colsep0 rowsep0" align="left">>0.01</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">Values without a standard deviation were obtained from a single measurement in triplicates.</p></div><div class="footnote" id="t3fn4"><sup>b</sup><p class="last">H661.</p></div><div class="footnote" id="t3fn3"><sup>c</sup><p class="last">A431.</p></div><div class="footnote" id="t3fn5"><sup>d</sup><p class="last">PC9del1.</p></div><div class="footnote" id="t3fn2"><sup>e</sup><p class="last">n.i., no inhibitory effect at 30 μM compound concentration.</p></div></div></div><div class="NLM_p">Interestingly, we observed similar cellular effects for the majority of our compounds on the cell line harboring the activating mutation (HCC827) as compared to the drug-resistant H1975 cell line without a dramatic loss in activity. However, the investigated compounds did not account for a significant reduction of viability in cellular settings, indicating the need for a stronger inhibitory effect on the kinase activity and likewise strengthening the need for further investigation with respect to pharmacokinetic properties such as solubility and cell permeability.</div><div class="NLM_p last">In order to assess these pharmacokinetic parameters and to pursue the poor translation of biochemical activity into cellular-based potency, we set out to investigate the solubility and cell permeability for the most promising compounds, <b>5b</b> and <b>6a</b>, using <i>in vitro</i> assays (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). The kinetic solubility for <b>5b</b> and <b>6a</b> was measured at pH 7.4 and revealed a high aqueous solubility of 432 and 277 μM for <b>5b</b> and <b>6a</b>, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01082/suppl_file/jm5b01082_si_002.pdf" class="ext-link">Table S3</a>). Furthermore, we gained valuable insights into the cellular absorption using an artificial membrane permeation assay (PAMPA). <b>5b</b> exhibited a rather low cellular penetration at pH 7.4 (PAMPA, flux 7%), whereas <b>6a</b>, equipped with an indazole instead of a pyrazolo moiety, showed a 7-fold higher permeability (PAMPA, flux 47%), representing sufficient cellular penetration in an artificial membrane system according to the assay’s classification.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> However, we further investigated the cell permeability of <b>5b</b> and <b>6a</b> using a Caco-2 cell assay, in which the ratio of influx/efflux across Caco-2 cells is determined, and, therefore, is more appropriate to reflect the setting in a cellular environment. Strikingly, we observed a high efflux rate for both compounds (efflux over cell penetration: 32:1 and 35:1 for <b>5b</b> and <b>6a</b>), indicating a poor overall absorption that leads to a low receptor occupancy in the cells, thus providing a reasonable explanation for the compound’s moderate cellular efficacy as compared to their inhibitory activity with respect to isolated kinases.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Computational Studies of Ligands’ Conformation in Solution</h3><div class="NLM_p">The drug CO-1686, as one of the current front runners in EGFR-related cancer therapy, is supposed to be conformationally constrained and therefore most likely to adopt a preconfigured conformation that is very similar to the observed binding mode in the WZ4002–EGFR complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ika">3ika</a>). Preconfigured compounds take advantage of the fact that the free energy penalty incurred upon binding a ligand in an optimum conformation to the protein of interest is markedly lower than for those compounds that present considerably different conformational states in aqueous solution and therefore must undergo unfavorable conformational changes upon ligand binding. For this reason, our current efforts, especially by introducing an indazole moiety, were directed to the design of compounds that take advantage of a certain preconfiguration to minimize disfavored conformational changes upon intrinsic compound rearrangements. In order to test these propositions, we performed theoretical calculations of the conformational ensembles for selected compounds in order to identify most favored conformations that could be adopted in aqueous solution and their similarity to the crystallographic binding mode.</div><div class="NLM_p">Conceptually, (near) exhaustive search of the conformational space of typical EGFR ligands in solution is problematic due to the large number of accessible states, which requires sampling schemes followed by geometrical clustering approaches in order to identify representative structures.<a onclick="showRef(event, 'ref45 ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref45 ref46 ref47">(45-47)</a> Moreover, empirical force field-based methods, for instance, based on the general Amber force field (GAFF<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48, 49)</a>), are not accurate enough to discriminate possibly subtle population shifts arising from free energy differences below 1 kcal mol<sup>–1</sup>. This represents the typical so-called chemical accuracy, which poses a systematic limit to force field calculations and simulations, for instance, of small ligands in solution<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> or of protein–ligand interactions.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> To overcome this uncertainty, we developed a strategy to estimate the configurational free energy of conformational basins of free ligands in solution based on quantum-chemical calculations that takes into account an aqueous solution environment. To address the complexity of the conformational space, we started by generating a, for all practical purposes, uniformly distributed ensemble of structures by sampling from a high-temperature simulation trajectory of the ligand with fixed charges immersed in a Poisson–Boltzmann-type continuum model of the solvent. After applying a sufficiently large energetic cutoff of 3 kcal mol<sup>–1</sup> (corresponding to a Boltzmann weight of skipped conformations of less than 0.007 at ambient temperature) in order to remove high-energy structures, the resulting snapshots were geometrically clustered. The minimum energy geometries of each cluster (still from a fixed-charge force field) were subjected to further quantum-chemical treatments. These consisted, for each cluster optimum, of a preoptimization step under vacuum conditions followed by final optimization, taking into account a solvent by the polarizable continuum model (PCM<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a>), yielding the configuration’s free energy in solution <i>G</i><sup>PCM</sup>. Finally, the free energies of the resulting structures in solution were rescored by the embedded-cluster reference interaction site model (EC-RISM,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> giving values <i>G</i><sup>EC-RISM</sup>) integral equation theory, using settings adopted earlier for tautomer<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> and chemical shift<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> predictions in water. The number of cluster members was taken as degeneracy factor <i>g</i><sub><i>i</i>,<i>j</i></sub> for cluster <i>i</i> in the conformational basin <i>j</i>, where the latter has been defined by grouping clusters that have visually sufficient distance between each other (quantified in the caption to <a class="ref internalNav" href="#fig5" aria-label="Figures 5">Figures 5</a> and <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>), as measured by their root-mean-square deviation (RMSD) with respect to the crystallographic binding mode. Such a basin corresponds to a certain distribution of dihedral angles (data and atom indices are discussed below) that define a specific conformation. Different basins are, therefore, characteristic dihedral angle groups, allowing for limited fluctuations represented by the other members within a basin. Neglecting contributions from vibrational entropy associated with each cluster well, the cluster population is given by the Boltzmann weight at ambient target temperature <i>T</i><span class="NLM_disp-formula" id="eq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/jm-2015-01082e_m001.gif" alt="" /></img><span class="labelSpan">(1)</span></span>where <i>R</i> is the gas constant; the population of each conformational basin is defined by<span class="NLM_disp-formula" id="eq2"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/jm-2015-01082e_m002.gif" alt="" /></img><span class="labelSpan">(2)</span></span></div><figure id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0006.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Minimum free energy conformations and conformational basins of inhibitors. Left: Structures in their predominant protonation state. Right: Population diagrams resulting from the computational analysis of conformational basins (identified by different colors) of selected compounds for different methods to treat the solvation contribution to the total free energy (PCM, triangles; EC-RISM, squares). Top: WZ4002 (note that a single PCM data point at RMSD = 0.222 nm with population 0.43 is not shown in order to limit the scale). Bottom: WZ4002-H (Cl substituted by H). Middle: Superimposition of corresponding minimum free energy conformations per basin (also shown as insets); red models (with blue balls representing nitrogen) show the respective reference conformation (WZ4002, PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3IKA">3IKA</a>) of the ligand in the binding site. Boundaries between basins are (in nm, from left to right) for WZ4002, group 1: RMSD < 0.16, 2: 0.16 ≤ RMSD < 0.3, 3: 0.3 ≤ RMSD < 0.38, 4: 0.38 ≤ RMSD; for WZ4002-H, group 1: RMSD < 0.2, 2: 0.2 ≤ RMSD < 0.36, 0.36 ≤ RMSD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01082&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0007.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Population analysis results for (from top to bottom) <b>5b</b>, <b>5n</b>, <b>6a</b> in analogy to <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>. Reference conformation (red model) in all cases is the crystallographic binding mode of <b>5b</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D11">5D11</a>). Boundaries between basins are (in nm, from left to right) for <b>5b</b>, group 1: RMSD < 0.16, 2: 0.16 ≤ RMSD < 0.25, 3: 0.25 ≤ RMSD; <b>5n</b>, group 1: RMSD < 0.14, 2: 0.14 ≤ RMSD < 0.25, 3: 0.25 ≤ RMSD; <b>6a</b>, group 1: RMSD < 0.15, 2: 0.15 ≤ RMSD < 0.25, 3: 0.25 ≤ RMSD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01082&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">This methodology has been applied initially to WZ4002 in order to test the performance with respect to the proposition that a substitution of chlorine by hydrogen could lead to more conformational freedom and, correspondingly, less inhibitory activity.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Results are summarized in <a class="ref internalNav" href="#tbl4" aria-label="Tables 4">Tables 4</a> and <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a> as well as <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>. Most importantly, only four dihedral angels are relevant for defining the accessible conformational space, as shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>. Here, WZ4002 is the only compound that exhibits a close match between reference crystal structure and basin 1 (smallest RMSD) in terms of three out of four dihedrals, while the population of this basin is nonzero. In this sense, WZ4002 is indeed preconfigured since only a single torsional rearrangement of members of basin 1 is required in order to bind. In contrast, the structurally completely different basin 4 (largest RMSD) has to rotate all four dihedrals to facilitate binding. Although we cannot estimate the binding pathway (in the sense of making assumptions about the possibility of internal rotations at certain points on the binding reaction coordinate), it is clear that the visually elongated form is not likely to bind directly. However, its population is basically zero in free solution, so all essentially contributing conformations of the compound are, to a certain degree, preconfigured, as measured by the preconfiguration ratio (PR) defined in <a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a> being 100:0.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Torsion Angle Characteristics (Degrees) of Reference Structures and Predicted Main Conformations Per Basin<a class="ref internalNav" href="#tbl4-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">dihedral atoms</th><th class="colsep0 rowsep0" align="center" char=".">(1,2,3,4)</th><th class="colsep0 rowsep0" align="center" char=".">(2,3,4,5)</th><th class="colsep0 rowsep0" align="center" char=".">(5,6,7,8),<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a> (6,7,8,9)<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">(6,7,8,9),<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a> (7,8,9,10)<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center"><i>N</i><sub>match</sub></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">WZ4002</td><td class="colsep0 rowsep0" align="left">crystal</td><td class="colsep0 rowsep0" align="char" char=".">–109.1</td><td class="colsep0 rowsep0" align="char" char=".">10.9</td><td class="colsep0 rowsep0" align="char" char=".">–1.4</td><td class="colsep0 rowsep0" align="char" char=".">153.2</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">basin</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char=".">–104.8</td><td class="colsep0 rowsep0" align="char" char=".">6.1</td><td class="colsep0 rowsep0" align="char" char=".">–1.5</td><td class="colsep0 rowsep0" align="char" char=".">–150.0</td><td class="colsep0 rowsep0" align="left">3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="char" char=".">89.0</td><td class="colsep0 rowsep0" align="char" char=".">2.0</td><td class="colsep0 rowsep0" align="char" char=".">9.6</td><td class="colsep0 rowsep0" align="char" char=".">117.3</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="char" char=".">82.7</td><td class="colsep0 rowsep0" align="char" char=".">3.7</td><td class="colsep0 rowsep0" align="char" char=".">179.8</td><td class="colsep0 rowsep0" align="char" char=".">–177.6</td><td class="colsep0 rowsep0" align="left">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="char" char=".">24.0</td><td class="colsep0 rowsep0" align="char" char=".">–113.5</td><td class="colsep0 rowsep0" align="char" char=".">–176.8</td><td class="colsep0 rowsep0" align="char" char=".">177.6</td><td class="colsep0 rowsep0" align="left">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">WZ4002-H</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">basin</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char=".">–86.9</td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="char" char=".">–0.5</td><td class="colsep0 rowsep0" align="char" char=".">–178.5</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="char" char=".">80.2</td><td class="colsep0 rowsep0" align="char" char=".">7.7</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td><td class="colsep0 rowsep0" align="char" char=".">178.0</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="char" char=".">89.0</td><td class="colsep0 rowsep0" align="char" char=".">7.5</td><td class="colsep0 rowsep0" align="char" char=".">179.9</td><td class="colsep0 rowsep0" align="char" char=".">–178.5</td><td class="colsep0 rowsep0" align="left">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5b</b></td><td class="colsep0 rowsep0" align="left">crystal</td><td class="colsep0 rowsep0" align="char" char=".">89.3</td><td class="colsep0 rowsep0" align="char" char=".">13.6</td><td class="colsep0 rowsep0" align="char" char=".">–1.7</td><td class="colsep0 rowsep0" align="char" char=".">176.2</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">basin</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char=".">84.1</td><td class="colsep0 rowsep0" align="char" char=".">4.0</td><td class="colsep0 rowsep0" align="char" char=".">–179.9</td><td class="colsep0 rowsep0" align="char" char=".">–0.2</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="char" char=".">51.0</td><td class="colsep0 rowsep0" align="char" char=".">22.9</td><td class="colsep0 rowsep0" align="char" char=".">178.6</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td><td class="colsep0 rowsep0" align="left">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="char" char=".">–51.2</td><td class="colsep0 rowsep0" align="char" char=".">–22.7</td><td class="colsep0 rowsep0" align="char" char=".">–179.4</td><td class="colsep0 rowsep0" align="char" char=".">0.0</td><td class="colsep0 rowsep0" align="left">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5n</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">basin</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char=".">95.2</td><td class="colsep0 rowsep0" align="char" char=".">–1.0</td><td class="colsep0 rowsep0" align="char" char=".">0.7</td><td class="colsep0 rowsep0" align="char" char=".">–0.7</td><td class="colsep0 rowsep0" align="left">3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="char" char=".">50.4</td><td class="colsep0 rowsep0" align="char" char=".">22.5</td><td class="colsep0 rowsep0" align="char" char=".">–0.2</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="char" char=".">–51.2</td><td class="colsep0 rowsep0" align="char" char=".">–21.4</td><td class="colsep0 rowsep0" align="char" char=".">0.7</td><td class="colsep0 rowsep0" align="char" char=".">0.2</td><td class="colsep0 rowsep0" align="left">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6a</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">basin</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char=".">96.1</td><td class="colsep0 rowsep0" align="char" char=".">–2.0</td><td class="colsep0 rowsep0" align="char" char=".">–179.8</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="char" char=".">51.2</td><td class="colsep0 rowsep0" align="char" char=".">22.4</td><td class="colsep0 rowsep0" align="char" char=".">178.8</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td><td class="colsep0 rowsep0" align="left">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="char" char=".">–52.2</td><td class="colsep0 rowsep0" align="char" char=".">–20.6</td><td class="colsep0 rowsep0" align="char" char=".">–179.1</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="left">0</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl4-fn1"><div class="footnote" id="tbl4-fn1"><sup>a</sup><p class="last">Constituting atom numberings are specified in <a class="ref internalNav" href="#fig4" aria-label="Figures 4">Figures 4</a> and <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>. Crystallographic reference binding mode geometries for comparison between experiment and computational prediction are WZ4002 for both WZ4002 Cl and H derivatives and <b>5b</b> for <b>5b</b>, <b>5n</b>, and <b>6a</b>. The number of matching dihedrals (<i>N</i><sub>match</sub>) is defined as the sum of similar dihedral angle values per main conformer in comparison with the respective reference structure to within 20° (taking into account periodicity). For instance, the resulting count 3 for basin 1 of WZ4002 is the number of corresponding dihedral angle row entries (1,2,3,4), (2,3,4,5), and (5,6,7,8); the last angle deviates by ca. 57°.</p></div><div class="footnote" id="t4fn2"><sup>b</sup><p class="last">For WZ4002 and WZ4002-H.</p></div><div class="footnote" id="t4fn3"><sup>c</sup><p class="last">For <b>5b</b>, <b>5n</b>, and <b>6a</b>.</p></div></div></div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Population Analysis of Conformational Basins of Selected Compounds for Different Methods To Treat the Solvation Contribution to the Total Free Energy<a class="ref internalNav" href="#tbl5-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char=":" /></col><col align="char" char=":" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">method</th><th class="colsep0 rowsep0" align="center" char="."><i>p</i><sub>1</sub></th><th class="colsep0 rowsep0" align="center" char="."><i>p</i><sub>2</sub></th><th class="colsep0 rowsep0" align="center" char="."><i>p</i><sub>3</sub></th><th class="colsep0 rowsep0" align="center" char="."><i>p</i><sub>4</sub></th><th class="colsep0 rowsep0" align="center" char=":">PR</th><th class="colsep0 rowsep0" align="center" char=":"> ⟨PR⟩</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">WZ4002</td><td class="colsep0 rowsep0" align="left">PCM</td><td class="colsep0 rowsep0" align="char" char=".">0.210</td><td class="colsep0 rowsep0" align="char" char=".">0.668</td><td class="colsep0 rowsep0" align="char" char=".">0.122</td><td class="colsep0 rowsep0" align="char" char=".">0.000</td><td class="colsep0 rowsep0" align="char" char=":">100:0</td><td class="colsep0 rowsep0" rowspan="2" align="char" valign="middle" char=":">100:0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">EC-RISM</td><td class="colsep0 rowsep0" align="char" char=".">0.043</td><td class="colsep0 rowsep0" align="char" char=".">0.103</td><td class="colsep0 rowsep0" align="char" char=".">0.854</td><td class="colsep0 rowsep0" align="char" char=".">0.000</td><td class="colsep0 rowsep0" align="char" char=":">100:0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">WZ4002-H</td><td class="colsep0 rowsep0" align="left">PCM</td><td class="colsep0 rowsep0" align="char" char=".">0.407</td><td class="colsep0 rowsep0" align="char" char=".">0.445</td><td class="colsep0 rowsep0" align="char" char=".">0.148</td><td class="colsep0 rowsep0" align="char" char=".">−</td><td class="colsep0 rowsep0" align="char" char=":">−85:15</td><td class="colsep0 rowsep0" rowspan="2" align="char" valign="middle" char=":">52:48</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">EC-RISM</td><td class="colsep0 rowsep0" align="char" char=".">0.015</td><td class="colsep0 rowsep0" align="char" char=".">0.177</td><td class="colsep0 rowsep0" align="char" char=".">0.808</td><td class="colsep0 rowsep0" align="char" char=".">−</td><td class="colsep0 rowsep0" align="char" char=":">19:81</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5b</b></td><td class="colsep0 rowsep0" align="left">PCM</td><td class="colsep0 rowsep0" align="char" char=".">0.000</td><td class="colsep0 rowsep0" align="char" char=".">0.519</td><td class="colsep0 rowsep0" align="char" char=".">0.481</td><td class="colsep0 rowsep0" align="char" char=".">−</td><td class="colsep0 rowsep0" align="char" char=":">52:48</td><td class="colsep0 rowsep0" rowspan="2" align="char" valign="middle" char=":">48.5:51.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">EC-RISM</td><td class="colsep0 rowsep0" align="char" char=".">0.000</td><td class="colsep0 rowsep0" align="char" char=".">0.447</td><td class="colsep0 rowsep0" align="char" char=".">0.553</td><td class="colsep0 rowsep0" align="char" char=".">−</td><td class="colsep0 rowsep0" align="char" char=":">45:55</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5n</b></td><td class="colsep0 rowsep0" align="left">PCM</td><td class="colsep0 rowsep0" align="char" char=".">0.000</td><td class="colsep0 rowsep0" align="char" char=".">0.515</td><td class="colsep0 rowsep0" align="char" char=".">0.484</td><td class="colsep0 rowsep0" align="char" char=".">−</td><td class="colsep0 rowsep0" align="char" char=":">52:48</td><td class="colsep0 rowsep0" rowspan="2" align="char" valign="middle" char=":">52.5:47.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">EC-RISM</td><td class="colsep0 rowsep0" align="char" char=".">0.000</td><td class="colsep0 rowsep0" align="char" char=".">0.534</td><td class="colsep0 rowsep0" align="char" char=".">0.466</td><td class="colsep0 rowsep0" align="char" char=".">−</td><td class="colsep0 rowsep0" align="char" char=":">53:47</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6a</b></td><td class="colsep0 rowsep0" align="left">PCM</td><td class="colsep0 rowsep0" align="char" char=".">0.000</td><td class="colsep0 rowsep0" align="char" char=".">0.621</td><td class="colsep0 rowsep0" align="char" char=".">0.379</td><td class="colsep0 rowsep0" align="char" char=".">−</td><td class="colsep0 rowsep0" align="char" char=":">62:38</td><td class="colsep0 rowsep0" rowspan="2" align="char" valign="middle" char=":">53.5:46.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">EC-RISM</td><td class="colsep0 rowsep0" align="char" char=".">0.002</td><td class="colsep0 rowsep0" align="char" char=".">0.451</td><td class="colsep0 rowsep0" align="char" char=".">0.549</td><td class="colsep0 rowsep0" align="char" char=".">−</td><td class="colsep0 rowsep0" align="char" char=":">45:55</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl5-fn1"><div class="footnote" id="tbl5-fn1"><sup>a</sup><p class="last">PR (averaged over PCM and EC-RISM data in the last column) denotes the preconfiguration ratio, defined as the inverse of the ratio of population of the farthest basin (measured by the RMSD) and the sum of all others. Specifically, PR = (<i>p</i><sub>1</sub> + <i>p</i><sub>2</sub> + <i>p</i><sub>3</sub>)/<i>p</i><sub>4</sub> for WZ4002 and PR = (<i>p</i><sub>1</sub> + <i>p</i><sub>2</sub>)/<i>p</i><sub>3</sub> for the remaining compounds.</p></div></div></div><div class="NLM_p">In contrast, dihedral and population data for WZ4002-H show that the latter derivative exhibits a form closest to the binding mode where at least two dihedrals have to rotate. This is already an indication of less likely binding. Moreover, the PR in this case shows a shift toward elongated conformations (basin 3), whereas the other two conformations are U-shaped, as defined by dihedral angles spanned by atoms (2,3,4,5) and (5,6,7,8) being close to 0 (see <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). Note that the diffuse RMSD boundary between basins 2 and 3 is nevertheless clearly characterized by a change of angle (5,6,7,8) by almost 180°. In summary, the reduced WZ4002-H activity is indeed correlated with a loss of conformational preconfiguration propensity.</div><div class="NLM_p last">In order to understand the conformational space of the present set of compounds, we examined analogously <b>5b</b>, <b>5n</b>, and <b>6a</b>. Numbers for dihedral angles, populations, and graphical representations of main conformers are also shown in <a class="ref internalNav" href="#tbl4" aria-label="Tables 4">Tables 4</a> and <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a> as well as in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>. Again, we find that it is possible to reduce the conformational space to four essential dihedral angles. Remarkably, at least <b>5n</b> possesses an accessible state where three out of four dihedrals match the crystal binding mode of reference compound <b>5b</b> fully, but this state is practically unpopulated for all <b>5b</b>, <b>5n</b>, and <b>6a</b>, in contrast to the best matching WZ4002 conformer. All three compounds have to undergo a rotation of at least one dihedral angle by 180°, whereas WZ4002’s best matching conformation has to rotate only a single dihedral (6,7,8,9) by 57°, as mentioned in <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>. Also, all three compounds show some tendency to form mirror-like image relations, defined by the inverted angles (1,2,3,4) and (2,3,4,5) between basins 2 and 3, which are similarly populated. Some degree of preconfiguration exists for this class of compounds measured by summing populations of basins 1 (which contributes only marginally) and 2 that come closest to the binding. It is important to note that <b>6a</b>, the most active compound against the L858R/T790M double mutant of EGFR, indeed shows some increased propensity toward the preconfigured conformation as compared to <b>5b</b> and <b>5n</b>, corroborating the proposition that conformational restriction is a valid design goal that can be predicted by computational means.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28706" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28706" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Drug-resistance mutations within the catalytic pocket, which occur as a consequence of secondary mutations after first-line treatment, represent a major challenge in modern cancer therapy, as the target proteins have to be considered as entirely different enzymes and, therefore, require newly developed inhibitors.<a onclick="showRef(event, 'ref31 ref33 ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref31 ref33 ref56 ref57">(31, 33, 56, 57)</a> However, in the case of the T790M mutation in EGFR, recently designed third-generation inhibitors bearing a warhead for covalent modification of a unique and reactive cysteine at the lip of the ATP binding cleft in EGFR provide evidence for a successful mode of action without evolving severe side effects.<a onclick="showRef(event, 'ref31 ref33 ref58'); return false;" href="javascript:void(0);" class="ref ref31 ref33 ref58">(31, 33, 58)</a> Covalent drugs equipped with Michael acceptors have not been appreciated in medicinal chemistry for a long time. Such drugs were mainly seen as acting nonspecifically because of their reactive groups and thereby driving toxicity, such as idiosyncratic drug-related toxicity, by eliciting adverse immune response.<a onclick="showRef(event, 'ref59 ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref59 ref60 ref61">(59-61)</a> Recently, it has been found that the specific fine tuning of these covalent drugs leads to distinct therapeutic and selective effects, which are not associated with toxicity <i>in vivo.</i> It has turned out that an acrylamide warhead, which is perfectly attached to a drug in terms of geometry and orientation toward the reactive cysteine, appears to represent an excellent compromise regarding reactivity and toxicity, as it is feasible to specifically alkylate the target protein <i>in vivo</i> while omitting unwanted irreversible interactions with arbitrary proteins.<a onclick="showRef(event, 'ref31 ref33 ref62 ref63'); return false;" href="javascript:void(0);" class="ref ref31 ref33 ref62 ref63">(31, 33, 62, 63)</a> Current design approaches take advantage of cysteine mapping along the kinome and, therefore, the concept of covalently modifying target protein to increase the target residence time and elicit temporal target selectivity has been revived in the past decade. Current efforts in medicinal chemistry, for instance, are directed toward the design and development of selective and irreversibly bound compounds to overcome secondary drug resistance mutations, as in the case of EGFR-T790M.<a onclick="showRef(event, 'ref20 ref21 ref22 ref64'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22 ref64">(20-22, 64)</a> Irreversible inhibitors such as CO-1686 and AZD9291 are currently at the leading edge of clinically relevant therapeutics for the treatment of advanced NSCLC and may stimulate further rational design approaches adapted from covalent modification of target proteins.<a onclick="showRef(event, 'ref30 ref32'); return false;" href="javascript:void(0);" class="ref ref30 ref32">(30, 32)</a> Here, we have presented a structure-guided development of irreversible inhibitors to selectively target the L858R/T790M mutant form of EGFR. Initial screening of 1500 compounds against 80 NSCLC cell lines led us to the identification of <b>1a</b>, revealing an attractive chemotype and showing effective inhibitory impact on H1975, an EGFR-L858R/T790M mutant lung cancer cell line. However, the pharmacological effect was not limited to H1975, but extended to the majority of investigated cell lines, which indicates rather broad specificity. We set out to utilize the general quinazoline scaffold of <b>1a</b> as the structural basis for further rational design approaches that would result in compounds with increased specificity while retaining cellular potency. Therefore, we utilized X-ray crystallography and discovered the crystal structure of <b>1a</b> in complex with genetically engineered cSrc as the initial model system for the therapeutically relevant kinase EGFR. By methods of synthetic chemistry, we produced focused compound libraries and particularly paid attention to the variation of the quinazoline core structure, resulting in further quinazolines (<b>2a</b>–<b>f</b>) as well as various substituted pyrimidines (<b>3a</b>–<b>b</b>, <b>4</b>, <b>5a</b>–<b>q</b>, <b>6a</b>–<b>e</b>, <b>7</b>). The latter were extensively elaborated with respect to different substitution patterns (e.g., Michael-acceptor position, 2,4- vs 2,4,6,-substitution) and revealed distinct inhibitory effects in biochemical characterizations using wild-type as well as L858R and accordingly L858R/T790M mutated EGFR. While quinazoline-based inhibitor <b>2b</b> illustrated increasing inhibitory impact in biochemical characterizations (IC<sub>50</sub> = 1.1 μM (wt) > 0.5 μM L858R > 0.1 μM L858R/T790M) and thereby displayed mutant selectivity, the related derivatives <b>2a</b> and <b>2c</b>, differing only by the position of the attachment of the Michael acceptor, basically did not evoke any effect. Shortened pyrimidine scaffolds <b>3a</b> and <b>5l</b>, however, illustrated the same mutant selectivity. Notably, for the 2,4,6-substituted scaffold <b>5b</b>, the mutant selective inhibition profile was somehow inverted since the L858R mutant form of EGFR was predominantly addressed (IC<sub>50</sub> = 0.4 μM (wt) > 0.03 μM (L858R) < 0.5 μM (L858R/T790M)), whereas <b>6a</b> exhibited an excellent as well as a mutant-selective effect on EGFR mutant variants (IC<sub>50</sub> = 2.2 μM (wt) > 0.5 μM (L858)R > 0.07 μM (L858R/T790M)). Constitutive protein X-ray experiments using a validated model system of cSrc (T338M/S345C) representing EGFR-T790M and promising inhibitors from our focused library revealed binding modes that are in accordance with covalent modification of the target protein, as in the case of <b>5b</b>. Moreover, MS-based analyses further supported our result of covalent modification of EGFR by demonstrating an increased molecular weight of the protein–ligand complex that matched the respective monolabeled species, as in the case of EGFR–<b>3a</b>, EGFR–<b>5b</b>, and EGFR–<b>6a</b>. In addition, we confirmed the alkylation of Cys797 for both inhibitors by conducting proteolytic digestion of <b>5b</b> and <b>6a</b> and subsequent nano-LC-MS/MS analysis. As a proof-of-concept study, we likewise tested our focused compound collection with respect to their inhibitory activities on cSrc as our initial EGFR surrogate system and found the majority our covalent inhibitors to more efficiently inhibit cSrc T338M/S345C as compared to cSrc T338M and cSrc wild-type.</div><div class="NLM_p">In a further experimental setup, we set out to investigate the kinetic properties of <b>5b</b> and <b>6a</b> with respect to time-dependent affinity and specific reactivity toward EGFR mutant variants (L858R and L858R/T790M) by determination of the respective <i>K</i><sub>i</sub> and <i>k</i><sub>inact</sub> parameters. On the basis of this analysis, we found that the inhibitors address the gatekeeper mutant form of EGFR (L858R/T790M) with high affinity, reflected by <i>K</i><sub>i</sub> values in the subnanomolar range, whereas the rate of inactivation remained fairly moderate (<i>k</i><sub>inact</sub>). On the contrary, the activating mutant (L858R) was addressed with significantly lower affinity, revealing <i>K</i><sub>i</sub> values of 70 nM (<b>5b</b>) and 833 nM (<b>6b</b>), and beyond that, the rate of inactivation was marginal (<i>k</i><sub>inact</sub> ≤ 0.055 min<sup>–1</sup>). Covalent inhibitors achieve their potency through a two-step-mechanism. In the first step, a covalent inhibitor reversibly interacts with the target kinase to build a crucial noncovalent drug–target complex. In a second step, a covalent bond is exclusively formed <i>in vivo</i> in which the inhibitor incorporates an optimized electrophile located in close proximity toward the reactive cysteine that allows for rapid alkylation. Therefore, a potent covalent inhibitor should comprise two essential features: (i) a strong reversible binding interaction and (ii) an optimized electrophile that facilitates rapid covalent bond formation.</div><div class="NLM_p">We observed discriminative inhibitory activities of <b>5b</b> and its reversible counterpart <b>5e</b> that clearly demonstrates the importance of covalently targeting the drug-resistant mutant of EGFR, as <b>5e</b> exhibited a 60-fold and >20-fold loss in inhibition against EGFR-L858R and EGFR-L858R/T790M. As a Michael acceptor represents the only structural difference between these two inhibitors, the shift in activity might directly correlate with beneficial effects due to the covalency of the compound and its associated prolonged drug-target residence time. In conjunction with the MS-based analyzes and the X-ray crystal structures, our most promising pyrimidine-based inhibitors, <b>5b</b> and <b>6a</b>, seem to form a covalent adduct with EGFR <i>in vitro</i>. These results explicitly revealed the potential of covalent inhibitors and highlighted the benefits of covalent design strategies for further drug development.</div><div class="NLM_p">We also conducted theoretical calculations, investigating the potentially preconfigured conformation of our 2,4,6,-substituted pyrimidine scaffolds (<b>5b</b>, <b>5n</b>, and <b>6a</b>) and the WZ4002/CO-1686 shape. Comparative analysis of both structure types revealed that the U-shape of WZ4002, the significantly populated geometry (out of four basins) that resembles the binding mode most closely, to a large extent, already exists in solution prior to interacting with the target protein. This favorable preconfiguration triggers the superior inhibitory effect of this compound because the free energy penalty that must be paid in order to bind to the target protein is much lower than that incurred by nonpreconfigured species. In the case of our compounds, we similarly found three discrete conformational basins, of which only two are significantly populated with mirror-like conformations that are adopted in solution. For the nonzero populated basin closest to the binding mode, at least two dihedrals have to rotate, which may result in an increased penalty and, therefore, reduce the inhibitory effect. Preconfiguration exists for this class but not to the same extent and similar relevance as for WZ4002. The compound that is most active against the double mutant exhibits the largest average preconfiguration ratio, although the effect is too small to be the only reason for the observed activity trend. The results presented in this article indicate that conformation control during ligand design is important and furthermore demonstrate the feasibility of computational tools to address this challenge.</div><div class="NLM_p">A further investigation of pharmacokinetic parameters such as solubility, cell permeability, and drug absorption for <b>5b</b> and <b>6a</b> revealed an excellent aqueous solubility for both compounds and a sufficient cellular permeability for <b>6a</b> in an artificial membrane system. However, we observed a dramatic efflux rate for <b>5b</b> and <b>6a</b>, illustrating a moderate drug absorption that leads to low cellular receptor occupancy and thereby providing a reasonable explanation for their moderate cellular efficacy in the NSCLC cell lines.</div><div class="NLM_p last">In summary, we have identified a novel class of irreversible and mutant-selective EGFR inhibitors, which may stimulate further approaches to develop covalently modulating inhibitors with increased target residence time.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46311" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46311" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Reagents and Materials</h3><div class="NLM_p last">All supplies for the EGFR HTRF assay kit were purchased from CisBio (Bagnols-sur-Cèze, France). Small volume (20 μL fill volume) white round-bottomed 384-well plates were obtained from Greiner Bio-One GmbH (Solingen, Germany).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Activity-Based Assay for IC<sub>50</sub> Determination and Kinetic Characterization</h3><div class="NLM_p">IC<sub>50</sub> determinations for EGFR and its mutants (Carna Biosciences, lot 13CBS-0005K, for EGFR-wt; Invitrogen, lot 279551C, for EGFR-L858R, and Invitrogen, lot 350247C, for EGFR-L858R/T790M) were performed with the HTRF KinEASE-TK assay from Cisbio according to the manufacturer’s instructions. Briefly, the amount of EGFR in each reaction well was set to 0.6 ng of EGFR wild-type (0.67 nM), 0.1 ng of EGFR L858R (0.11 nM), or 0.07 ng of EGFR T790M/L858R (0.08 nM). An artificial substrate peptide (TK-substrate from Cisbio) was phosphorylated by EGFR. After completion of the reaction (reaction times: 25 min for wt, 15 min for L858R, 20 min for T790M/L858R), the reaction was stopped by addition of buffer containing EDTA, and an anti-phosphotyrosine antibody labeled with europium cryptate and streptavidin labeled with the fluorophore XL665 were added. FRET between europium cryptate and XL665 was measured after an additional hour of incubation to quantify the phosphorylation of the substrate peptide. ATP concentrations were set at their respective <i>K</i><sub>m</sub> values (9.5 μM for EGFR-wt, 9 μM for EGFR-L858R, and 4 μM for EGFR-L858R/T790M), and substrate concentrations of 1 μM, 225 nM, and 200 nM, respectively, were used. Kinase and inhibitor were preincubated for 30 min (EGFR-wt) and 1 h (EGFR-L858R and EGFR-L858R/T790M) before the reaction was started by addition of ATP and substrate peptide. A Tecan infinite M1000 plate reader was used to measure the fluorescence of the samples at 620 nm (Eu-labeled antibody) and 665 nm (XL665-labeled streptavidin) 60 μs after excitation at 317 nm. The quotient of both intensities for reactions made with eight different inhibitor concentrations was fit to a Hill four-parameter equation using XLfit (IDBS, Surrey, UK) to determine IC<sub>50</sub> values. Each reaction was performed in duplicate, and at least three independent determinations of each IC<sub>50</sub> were made.</div><div class="NLM_p last">For kinetic characterization (<i>k</i><sub>inact</sub>/<i>K</i><sub>i</sub>) of <b>5a</b> and <b>6b</b>, the respective inhibitors were incubated with EGFR wild-type, L858R, or T790M/L858R over different periods of time (2–90 min), whereas durations of enzymatic and stop reactions were kept constant, as stated above. A 6-fold dilution series (eight data points per IC<sub>50</sub> curve) starting at 20 μM final compound concentration was applied. Compound dilutions were generated using the ECHO 520 Liquid Handler acoustic dispensing system from Labcyte (Sunnyvale, CA, USA) and Echo Dose-Response v1.5.4 software. Calculated IC<sub>50</sub> values were plotted versus incubation time, and data was fit as described in the literature to determine <i>k</i><sub>inact</sub> and <i>K</i><sub><i>i</i></sub> (see <a href="/doi/suppl/10.1021/acs.jmedchem.5b01082/suppl_file/jm5b01082_si_002.pdf" class="ext-link">Supporting Information</a>).<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Crystallization and Structure Determination of cSrc-DM–<b>1a</b>, cSrc-DM–<b>5b</b>, and cSrc-DM–<b>5c</b></h3><div class="NLM_p">Inhibitor <b>1a</b> was cocrystallized with cSrc-DM (cSrc-DM was expressed and purified as described elsewhere<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a>) using conditions similar to those previously reported by Michalczyk et al.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Briefly, final concentrations of 900 μM inhibitor (100 mM stock in DMSO) and 180 μM cSrc-DM (stored in 50 mM Tris pH 8.0, 100 mM NaCl, 1 mM DTT, and 5% glycerol (v/v)) were preincubated for 1 h on ice to form the enzyme–inhibitor complex prior to crystallization. Crystals were grown using the hanging drop method at 20 °C after mixing 1 μL of protein–inhibitor solution with 1 μL of reservoir solution (0.085–0.125 mM MES, pH 6.0–7.0, 9–11.5% PEG20000).</div><div class="NLM_p last">For generating the cocrystal structures of cSrc-DM–<b>5b</b> and cSrc-DM–<b>5c</b>, 565 μM inhibitor (prepared in DMSO) was preincubated along with 282 μM cSrc-DM for 2 h at room temperature to form the covalent enzyme–inhibitor complex prior to crystallization. The crystals were grown using the hanging drop method at 25 °C after mixing 1–1.5 μL of protein–inhibitor solution with 1 μL of reservoir solution (0–0.03 M NaCl, 9–20% ethylene glycol for cSrc-DM–<b>5b</b> and 0.01 M NaCl, 11% ethylene glycol for cSrc-DM–<b>5c</b>). All crystals were frozen with further addition of 25–30% (v/v) glycerol. The data set for the complex crystal structure of cSrc-DM–<b>5c</b> was collected in-house to a resolution of 2.7 Å using wavelengths close to 1.5 Å. Diffraction data of cSrc-DM–<b>1a</b> and −<b>5b</b> were collected at the PX10SA beamline of the Swiss Light Source (PSI, Villingen, Switzerland) to resolutions of 3.0 and 2.3 Å, respectively, using wavelengths close to 1 Å. All data sets were processed with XDS<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> and scaled using XSCALE.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Structure Determination and Refinement of cSrc-DM–<b>1a</b>, cSrc-DM–<b>5b</b>, and cSrc-DM–<b>5c</b></h3><div class="NLM_p last">The complex crystal structures of cSrc-DM–<b>1a</b>, cSrc-DM–<b>5b</b>, and cSrc-DM–<b>5c</b> were solved by molecular replacement with PHASER<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> using structures <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HWP">2HWP</a><a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> or <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OIQ">2OIQ</a><a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> as template. The two cSrc molecules in the asymmetric unit were manually modified using the program COOT.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> The model was first refined with CNS<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> using simulated annealing to remove model bias. The final refinement was performed with REFMAC5.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> Inhibitor topology files were generated using the Dundee PRODRG2 server.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> Refined structures were validated with PROCHECK.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> Data collection, structure refinement statistics, PDB ID codes, and further details regarding data collection are provided in <a href="/doi/suppl/10.1021/acs.jmedchem.5b01082/suppl_file/jm5b01082_si_002.pdf" class="ext-link">Table S2</a>. PyMOL<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> was used to generate the figures.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Viability Assay</h3><div class="NLM_p last">H1975, A431, H661, and HCC827 cells were obtained from the American Type Culture Collection (ATCC). Cells were seeded on day 1 at cell numbers that ensure linearity and optimal signal intensity. After culturing for 24 h in serum- and antibiotics-containing media in humidified chambers at 37 °C/5% CO<sub>2</sub>, the cells were incubated for 96 h with EGFR inhibitors in serial dilutions (14 nM to 30 μM) and DMSO as a control. Viability studies were carried out on day 5 using the CellTiter Glo assay (Promega, USA), which is a homogeneous method of determining the number of viable cells in culture. It is based on quantification of ATP, indicating the presence of metabolically active cells. For these studies, CellTiter Glo reagent was prepared according to the instructions of the kit. Thereon, reagent and assay plates were equilibrated at room temperature for 20 min. Equal volumes of the reagent were added to the volume of culture medium present in each well. The plates were mixed for 2 min on an orbital shaker. The microplates were then incubated at room temperature for 10 min for stabilization of the luminescent signal. Following incubation, the luminescence was recorded on a Victor microplate reader (PerkinElmer) using 200 ms integration time. The data was then analyzed with Excel using the XLfit-Plugin (dose–response fit 205) for GI<sub>50</sub> determination. For quality control, the <i>Z</i>′-factor was calculated from 16 positive and negative control values. Only assay results showing a <i>Z</i>′-factor ≥ 0.5 were used for further analysis. All experimental points were measured in duplicates for each plate and were replicated in at least two plates.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Solubility and Cell Permeability Measurements</h3><div class="NLM_p last">For determining kinetic solubility, the compound was diluted from a 10 mM stock in DMSO to a final concentration of 500 μM in 50 mM Hepes buffer, pH 7.4. Following an incubation of 90 min at room temperature on a shaker, the aqueous dilution was filtered through a 0.2 μm PVDF filter, and the optical density between 250 and 500 nm was measured in intervals of 10 nm. The kinetic solubility was calculated from the area under the curve (AUC) between 250 and 500 nm and normalized to absorption of a dilution of the compound in acetonitrile.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> Absorption was assessed using PAMPA (parallel artificial membrane permeability assay) and Caco-2 cell culture. For PAMPA, the compound was diluted from a 10 mM stock in DMSO to a final concentration of 500 μM in 50 mM Hepes buffer, pH 7.4, and transferred onto a transwell membrane covered with a membrane-forming solution of 10% 1,2-dioleyl-<i>sn</i>-glycero-3-phosphocholine (Sigma-Aldrich) and 0.5% (w/v) cholesterol (Sigma-Aldrich) in dodecane. Following an incubation of 16 h at room temperature in a wet chamber, the optical density of the solution in the receiver well was measured between 250 and 500 nm in intervals of 10 nm. The percent flux was calculated from the AUC between 250 and 500 nm and normalized to the absorption of the compound following a 16 h incubation in a parallel transwell containing a membrane covered with 50% methanol in 50 mM Hepes buffer, pH 7.4.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> For Caco-2 cell culture, a 10 mM DMSO stock of the compound was diluted to a final concentration of 5 μM in HBSS buffer, pH 7.4, and incubated for 2 h at 37 °C and 5% CO<sub>2</sub> on a monolayer of Caco-2 cells (ATCC) that had been grown on a transwell membrane (Millipore, Schwalbach, Germany) for 21 days. The compound concentration was measured in the receiver as well as the donor well. Apparent permeability (<i>P</i><sub>app</sub>) from either the apical to basolateral direction or vice versa was calculated by the equation <i>P</i><sub>app</sub> = 1/<i>AC</i><sub>0</sub>(d<i>Q</i>/d<i>t</i>), where <i>A</i> is the membrane surface area, <i>C</i><sub>0</sub> is the donor drug concentration at <i>t</i> = 0, and d<i>Q</i>/d<i>t</i> is the amount of drug transported within the given time period of 2 h.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Mass Spectrometry Experiments</h3><div class="NLM_p">We used the drug-resistant mutant variant of EGFR (EGFR-T790M) for MS experiments. We incubated a mixture of 52 μM of protein with 100 μM of inhibitor in buffer (25 mM Tris, 250 mM NaCl, 10% glycerol, 1 mM TCEP, pH 8) on ice for 1 h. We analyzed the aliquots by ESI-MS using an Agilent 1100 series HPLC system connected to a ThermoFinnigan LTQ linear ion trap mass spectrometer. A 6 μL sample was injected and separated using a Vydac 214TP C4 5 μm column (150 mm × 2.1 mm) starting at 20% solvent B for 5 min followed by a gradient up to 90% solvent B over 14 min with a flow rate of 210 μL/min, with 0.1% TFA in water as solvent A and 0.1% TFA in acetonitrile. A mass range of 700 to 2000 <i>m</i>/<i>z</i> was scanned, and raw data was deconvoluted and analyzed with MagTran software.<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a></div><div class="NLM_p">For nano-LC-MS/MS measurements, the samples were incubated with 10 mM iodoacetamide for 30 min at RT in the dark in order to carbamidomethylate all free cysteine residues. Next, 1 μg of each sample was digested using the broad specificity protease subtilisin (Sigma-Aldrich) to obtain high sequence coverage. Subtilisin was added in a 1:10 ratio (protease/protein), and the samples were digested for 20 min at 5 °C. The digestion was quenched by addition of 1% TFA, and 2 pmol per sample was analyzed by nano-LC-MS/MS using an Orbitrap Fusion mass spectrometer (Thermo Scientific), online-coupled to an Ultimate 3000 nano RSLC system (Thermo Scientific).</div><div class="NLM_p last">Peptides were preconcentrated on a 75 μm × 2 cm C18 trapping column for 10 min using 0.1% TFA (v/v) at a flow rate of 20 μL/min, followed by separation on a 75 μm × 50 cm C18 main column (both Pepmap, Thermo Scientific) with a 60 min LC gradient ranging from 3 to 42% B (84% acetonitrile in 0.1% formic acid) at a flow rate of 250 nL/min. MS survey scans were acquired in the Orbitrap from 300 to 1500 <i>m</i>/<i>z</i> at a resolution of 120 000 using the polysiloxane ion at 445.12003 <i>m</i>/<i>z</i> as lock mass. The most intense ions were isolated with a 1.6 Da window for top speed (3 s) fragmentation using a charge state-based decision tree including collision-induced dissociation (CID; +1), higher-energy collisional dissociation (HCD; +2, +3), and electron transfer dissociation (ETD > +3). MS/MS spectra were acquired in the Orbitrap at a resolution of 15 000, taking into account a dynamic exclusion of 6 s. Normalized collision energies of 35 and 30 were used for CID and HCD, respectively. For ETD reagent target and ETD, reaction times were set to 200 000 and 100 ms, respectively. AGC target values were set to 400 000 for MS and 50 000 for MS/MS (100 000 for ETD). Raw data were searched against an in-house-generated database containing the sequence of EGFR-T790M in a yeast background (SGD database, 6879 target sequences) using Mascot 2.4 with the following parameters: (i) “none” as protease, (ii) MS and MS/MS tolerances of 5 ppm and 0.02 Da, and (iii) carbamidomethylation at Cys (+57.0214 Da), <b>5b</b> at Cys (+434.2178 Da), and <b>6a</b> at Cys (470.2178 Da) as variable modifications.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Construct Design of EGFR-T790M</h3><div class="NLM_p last">DNA encoding residues comprising the kinase domain of human EGFR (UniProt entry <a href="https://P00533" class="ext-link">P00533</a>, residues 695–1022) were synthesized (GeneArt, Life Technologies) including an N-terminal polyhistidine tag and recognition site for thrombin protease (MGHHHHHHVDLVPRG); the point mutation T790M was introduced by side-directed mutagenesis (QuikChange, Stratagene/Agilent Technologies). The construct was cloned into pIEX/Bac3 expression vector (Merck Millipore), using <i>Nco</i>I and <i>Bsu36</i>I restriction sites, for deployment in the BacMagic expression system (Merck Millipore). Transfection, virus generation, and amplification were carried out in <i>Spodoptera frugiperda</i> cell line Sf9 following the BacMagic protocol.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Protein Expression and Purification</h3><div class="NLM_p last">The protein was expressed in Sf9 cells using the BacMagic system. Following protein expression, the cells were harvested (3000<i>g</i>, 10 min), resuspended in buffer A (50 mM Tris, 500 mM NaCl, 10% glycerol, 1 mM DTT, pH 8), and homogenized by french press. The lysate was cleared by centrifugation at 40 000<i>g</i> for 1 h and loaded on a column packed with Ni-NTA Superflow resin (Qiagen). The elution was done with a gradient of buffer B (buffer A + 500 mM imidazole) from 0 to 250 mM imidazole. Thrombin cleavage was carried out by dialysis against buffer C (25 mM TRIS, 50 mM NaCl, 10% glycerol, 1 mM EDTA, 1 mM DTT, pH 8) overnight at 8 °C. Following the cleavage reaction, the protein solution was loaded on a HiTrap-Q-HP column (GE Healthcare). The protein was eluted with buffer D (buffer C + 1 M NaCl) with a gradient of 0–250 mM NaCl. For the final purification step, the fractions containing EGFR were combined, concentrated, and applied to a HiLoad 16/600 superdex 75 pg column (GE Healthcare) in buffer E (25 mM Tris, 250 mM NaCl, 1 mM DTT, 10% glycerol, pH 8). The purified protein was concentrated to 5 mg/mL and stored at −80 °C for further use.</div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30">Theoretical Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05104" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05104" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Generation of Conformations</h3><div class="NLM_p last">For all compounds studied (WZ4002, WZ4002-H, <b>5b</b>, <b>5n</b>, <b>6a</b>), we applied an identical workflow that emerged after careful optimization of procedural parameters initially identified for WZ4002. Briefly, the protonation state at pH 7.4 in water was estimated by MoKa 2.5.2<a onclick="showRef(event, 'ref75 ref76'); return false;" href="javascript:void(0);" class="ref ref75 ref76">(75, 76)</a> (S. Güssregen, Sanofi, private communication), resulting in a singly protonated (i.e., net-charged) species with an estimated specific tautomer preference (protonated secondary nitrogen of the piperazyl ring) of 82.3%. The analogous tautomer was chosen for all other species. For fixed-charge force field simulations, GAFF<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48, 49)</a> was applied with AM1-BCC charges computed by Antechamber.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Molecular dynamics simulations were performed by Amber 12<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> at a temperature of 500 K for 10<sup>8</sup> time steps of 1 fs duration (100 ns) of a single compound exposed to a linearized Poisson–Boltzmann model of water (ALPB<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a>) using an Andersen thermostat.<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> We checked that an increase of the simulation time beyond that did not generate further conformations. After discarding the first 10 000 steps, every 10 000th snapshot was stored for subsequent geometry optimization to the next-nearest local minimum down to a maximum gradient norm of 10<sup>–4</sup> kcal mol<sup>–1</sup> Å<sup>–1</sup>. Structures with force field energies of 3 kcal mol<sup>–1</sup> above the global minimum were removed from the final set before similar conformations were clustered using the utility g_cluster from Gromacs 4.6.3<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> that applies the Jarvis–Patrick algorithm<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> for superimposed molecules with an RMSD criterion of 0.7 Å. This procedure yields the following numbers of conformations after applying the energy cutoff and numbers of clusters for WZ4002, WZ4002-H, <b>5b</b>, <b>5n</b>, and <b>6a</b>: 7750/154, 8109/143, 7685/131, 7788/111, and 2147/451, respectively.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Quantum Chemical Calculations</h3><div class="NLM_p last">The global minima found for each cluster were optimized first <i>in vacuo</i> using density functional theory with the B3LYP<a onclick="showRef(event, 'ref82 ref83'); return false;" href="javascript:void(0);" class="ref ref82 ref83">(82, 83)</a> exchange-correlation model as implemented in Gaussian 09<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> with the 6-31G(d) basis set,<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> followed by PCM optimization using default settings for water and the same basis set. The resulting sum of electronic and solvation energies defines <i>G</i><sup>PCM</sup>. The same set of structures was subjected to single-point EC-RISM calculations at 298.15 K with the water model used earlier.<a onclick="showRef(event, 'ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref54 ref55">(54, 55)</a> The quantum chemical level of theory here was B3LYP/6-311+G(d), and the closure used was second-order partial series expansion (PSE-2)<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a> utilizing a grid size of 0.3 Å and a maximum box length of 30 Å. Convergence was achieved when the free energy in solution (defining <i>G</i><sup>EC-RISM</sup> as the sum of electronic energy and excess chemical potential) fell below 0.01 kcal mol<sup>–1</sup>. RMSDs to the binding mode were calculated with respect to the WZ4002 complex structure<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> for all WZ4002 derivatives and with respect to <b>5b</b> for <b>5b</b>, <b>5n</b>, and <b>6a</b>. All basin structures were visualized by VMD 1.9.1,<a onclick="showRef(event, 'ref87'); return false;" href="javascript:void(0);" class="ref ref87">(87)</a> and their geometries are provided in <a href="/doi/suppl/10.1021/acs.jmedchem.5b01082/suppl_file/jm5b01082_si_001.zip" class="ext-link">Supporting Information</a>.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01082">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80605" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80605" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b01082" class="ext-link">10.1021/acs.jmedchem.5b01082</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Structures of <b>5b</b>, <b>5n</b>, <b>6a</b>, WZ4002, and WZ4002-H in Jmol format (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01082/suppl_file/jm5b01082_si_001.zip" class="ext-link">ZIP</a>).</p></li><li><p class="inline">Synthesis of pyrimidine-based <b>6e</b> (Scheme S1); synthesis of pyrimidine-based <b>7</b> (Scheme S2); mass spectrometry experiments (Figure S1); IC<sub>50</sub> determinations of cSrc and mutant variants (Table S1); X-ray data collection and refinement statistics (cSrc-DM–<b>1a</b>, −<b>5b</b>, and −<b>5c</b>) (Table S2); solubility and cellular permeability parameters of <b>5b</b> and <b>6a</b> (Table S3); nano-LC-MS/MS base peak chromatograms of <b>5b</b> and <b>6a</b> after proteolytic digestion (Figure S2); nano-LC-MS/MS fragment ion spectra of GCLLDYVR <b>5b</b> and <b>6a</b> (Figure S3); kinetic characterization of <b>5a</b> and <b>6b</b> by determination of <i>k</i><sub>inact</sub>/<i>K</i><sub>i</sub> (Figure S4); expression and purification of cSrc and its mutant variants; activity-based assay for IC<sub>50</sub> determination of cSrc and mutant variants thereof; and experimental section (chemistry) (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01082/suppl_file/jm5b01082_si_002.pdf" class="ext-link">PDF</a>).</p></li><li><p class="inline">IC<sub>50</sub> of compounds against wt and mutant EGFR (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01082/suppl_file/jm5b01082_si_003.xlsx" class="ext-link">XLSX</a>).</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01082/suppl_file/jm5b01082_si_001.zip">jm5b01082_si_001.zip (21.14 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01082/suppl_file/jm5b01082_si_002.pdf">jm5b01082_si_002.pdf (1.51 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01082/suppl_file/jm5b01082_si_003.xlsx">jm5b01082_si_003.xlsx (18.09 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.5b01082" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62507" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62507" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefan M. Kast</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Straße
6, D-44227 Dortmund, Germany</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#bfcccbdad9ded191d4decccbffcbca92dbd0cdcbd2cad1db91dbda"><span class="__cf_email__" data-cfemail="0d7e79686b6c6323666c7e794d79782069627f7960786369236968">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel Rauh</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Straße
6, D-44227 Dortmund, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Chemical
Genomics Centre of the Max-Planck Society, Otto-Hahn-Straße 15, D-44227 Dortmund, Germany</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#4f2b2e21262a23613d2e3a270f3b3a622b203d3b223a212b612b2a"><span class="__cf_email__" data-cfemail="0561646b6c60692b7764706d45717028616a777168706b612b6160">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Julian Engel</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Straße
6, D-44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">André Richters</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Straße
6, D-44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthäus Getlik</span> - <span class="hlFld-Affiliation affiliation">Chemical
Genomics Centre of the Max-Planck Society, Otto-Hahn-Straße 15, D-44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefano Tomassi</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Straße
6, D-44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marina Keul</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Straße
6, D-44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin Termathe</span> - <span class="hlFld-Affiliation affiliation">Chemical
Genomics Centre of the Max-Planck Society, Otto-Hahn-Straße 15, D-44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jonas Lategahn</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Straße
6, D-44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christian Becker</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Straße
6, D-44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Svenja Mayer-Wrangowski</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Straße
6, D-44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christian Grütter</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Straße
6, D-44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Niklas Uhlenbrock</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Straße
6, D-44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jasmin Krüll</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Straße
6, D-44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Niklas Schaumann</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Straße
6, D-44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Simone Eppmann</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Straße
6, D-44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Patrick Kibies</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Straße
6, D-44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Franziska Hoffgaard</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Straße
6, D-44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jochen Heil</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Straße
6, D-44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sascha Menninger</span> - <span class="hlFld-Affiliation affiliation">Lead
Discovery Center GmbH, Otto-Hahn-Straße 15, D-44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sandra Ortiz-Cuaran</span> - <span class="hlFld-Affiliation affiliation">Department
of Translational Genomics, Medical Faculty, University of Cologne, Weyertal 115b, D-50931 Cologne, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Johannes M. Heuckmann</span> - <span class="hlFld-Affiliation affiliation">Department
of Translational Genomics, Medical Faculty, University of Cologne, Weyertal 115b, D-50931 Cologne, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Verena Tinnefeld</span> - <span class="hlFld-Affiliation affiliation">Leibniz-Institut
für Analytische Wissenschaften − ISAS − e.V., D-44139 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">René P. Zahedi</span> - <span class="hlFld-Affiliation affiliation">Leibniz-Institut
für Analytische Wissenschaften − ISAS − e.V., D-44139 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin L. Sos</span> - <span class="hlFld-Affiliation affiliation">Department
of Translational Genomics, Medical Faculty, University of Cologne, Weyertal 115b, D-50931 Cologne, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Molecular
Pathology, University Hospital of Cologne, Kerpenerstraße 62, D-50937 Cologne, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carsten Schultz-Fademrecht</span> - <span class="hlFld-Affiliation affiliation">Lead
Discovery Center GmbH, Otto-Hahn-Straße 15, D-44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Roman K. Thomas</span> - <span class="hlFld-Affiliation affiliation">Department
of Translational Genomics, Medical Faculty, University of Cologne, Weyertal 115b, D-50931 Cologne, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pathology, University of Cologne, Joseph-Stelzmann Straße 9, D-50931 Cologne, Germany</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): R.K.T. is a founder and shareholder of Blackfield AG/New Oncology; reports receiving commercial research grants from AstraZeneca, EOS, and Merck KGaA; and honoraria from AstraZeneca, Bayer, Blackfield AG/New Oncology, Boehringer Ingelheim, Clovis Oncology, Daiichi-Sankyo, Eli Lilly, Johnson & Johnson, Merck KgaA, MSD, Puma, Roche, and Sanofi. D.R. received consultant and lecture fees from Astra-Zeneca, Merck-Serono, Takeda, Pfizer, Novartis, Boehringer Ingelheim, and Sanofi-Aventis.<br /></br></div></li></ul></div><div class="ack" id="ACK-d774e4456-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i34">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67376" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67376" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was cofunded by the German federal state North Rhine-Westphalia (NRW) and the European Union (European Regional Development Fund: Investing In Your Future) as part of the PerMed NRW initiative (NEGECA), the German Federal Ministry for Science and Education (BMBF) as part of the NGFNplus program (grant nos. 01GS08104 and 01GS08101) and the hpCADD project (grant no. 01IH11002B) and as part of the e:Med program (grant nos. 01ZX1303C, 01ZX1303A, and 01ZX1406), and by the Deutsche Forschungsgemeinschaft (DFG; through TH1386/3-1, RA 1055/3-1, and KA1381/5-1). R.P.Z. and V.T. thank the Ministry for Innovation, Science and Research of the Federal State of North Rhine-Westphalia for financial support. Computer time on the LiDOng cluster at the ITMC Dortmund is also gratefully acknowledged.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i35" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i35"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i36" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i36"> Abbreviations Used</h2><tr><td class="NLM_term">ADME</td><td class="NLM_def"><p class="first last">administration, distribution, metabolization and excretion</p></td></tr><tr><td class="NLM_term">ALPB</td><td class="NLM_def"><p class="first last">analytical linearized Poisson–Boltzmann</p></td></tr><tr><td class="NLM_term">AM1-BCC</td><td class="NLM_def"><p class="first last">semiempirical (AM1) with bond charge correction (BCC)</p></td></tr><tr><td class="NLM_term">B3LYP</td><td class="NLM_def"><p class="first last">Becke, three-parameter, Lee–Yang–Parr</p></td></tr><tr><td class="NLM_term">cSrc-DM</td><td class="NLM_def"><p class="first last">cSrc-(T338M/S345C)</p></td></tr><tr><td class="NLM_term">EC-RISM</td><td class="NLM_def"><p class="first last">embedded cluster reference interaction site model</p></td></tr><tr><td class="NLM_term">EGFR</td><td class="NLM_def"><p class="first last">epidermal growth factor receptor</p></td></tr><tr><td class="NLM_term">GAFF</td><td class="NLM_def"><p class="first last">general Amber force field</p></td></tr><tr><td class="NLM_term"><i>m</i>CPBA</td><td class="NLM_def"><p class="first last">3-chloroperoxybenzoic acid</p></td></tr><tr><td class="NLM_term">ms</td><td class="NLM_def"><p class="first last">mass-spectrometry</p></td></tr><tr><td class="NLM_term">NSCLC</td><td class="NLM_def"><p class="first last">non-small cell lung cancer</p></td></tr><tr><td class="NLM_term">PCM</td><td class="NLM_def"><p class="first last">polarizable continuum model</p></td></tr><tr><td class="NLM_term">PR</td><td class="NLM_def"><p class="first last">preconfiguration ratio</p></td></tr><tr><td class="NLM_term">PSE</td><td class="NLM_def"><p class="first last">partial series expansion</p></td></tr><tr><td class="NLM_term">RMSD</td><td class="NLM_def"><p class="first last">root-mean-square deviation</p></td></tr><tr><td class="NLM_term">RTK</td><td class="NLM_def"><p class="first last">receptor tyrosine kinase</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">TKI</td><td class="NLM_def"><p class="first last">tyrosine kinase inhibitor</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i37">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81412" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81412" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 87 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekar, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, A.</span><span> </span><span class="NLM_article-title">Epidermal growth factor receptor-mediated cell motility: phospholipase C activity is required, but mitogen-activated protein kinase activity is not sufficient for induced cell movement</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">127</span><span class="NLM_x">, </span> <span class="NLM_fpage">847</span><span class="NLM_x">–</span> <span class="NLM_lpage">857</span><span class="refDoi"> DOI: 10.1083/jcb.127.3.847</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1083%2Fjcb.127.3.847" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=7962064" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaK2cXmsVOitLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=1994&pages=847-857&author=P.+Chenauthor=H.+Xieauthor=M.+C.+Sekarauthor=K.+Guptaauthor=A.+Wells&title=Epidermal+growth+factor+receptor-mediated+cell+motility%3A+phospholipase+C+activity+is+required%2C+but+mitogen-activated+protein+kinase+activity+is+not+sufficient+for+induced+cell+movement&doi=10.1083%2Fjcb.127.3.847"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal growth factor receptor-mediated cell motility: phospholipase C activity is required, but mitogen-activated protein kinase activity is not sufficient for induced cell movement</span></div><div class="casAuthors">Chen, Philip; Xie, Heng; Sekar, M. Chandra; Gupta, Kiran; Wells, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Biology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">847-57</span>CODEN:
                <span class="NLM_cas:coden">JCLBA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9525</span>.
    </div><div class="casAbstract">The authors utilized signaling-restricted EGFR mutants expressed in receptor devoid NR6 cells to det. if PLC activation is necessary for EGFR-mediated cell movement.  Exposure to EGF (25 nm) augmented PLC activity in all five EGFR mutant cell lines which also responded by increased cell movement.  Basal phosphoinositide turnover was not affected by EGF in the lines which do not present the enhanced motility response.  The correlation between EGFR-mediated cell motility and PLC activity suggested, but did not prove, a causal link.  A specific inhibitor of PLC, U73122 (1 μM) diminished both the EGF-induced motility and PLC responses, while its inactive analog U73343 had no effect on these responses.  Both the PLC and motility responses were decreased by expression of a dominant-neg. PLCγ-1 fragment in EGF-responsive infectant lines.  Lastly, anti-sense oligonucleotides (10 μM) to PLCγ-1 reduced both responses in NR6 cells expressing wild-type EGFR.  These findings strongly support PLCγ as the immediate post receptor effector in this motogenic pathway.  The authors have demonstrated previously that EGFR-mediated cell motility and mitogenic pathway.  The authors have demonstrated previously that EGFR-mediated cell motility and mitogenic signaling pathways are separable.  The point of divergence is undefined.  All kinase-active EGFR mutants induced the mitogenic response while only those which are autophosphorylated induced PLC activity.  U73122 did not affect EGF-induced thymidine incorporation in these motility-responsive infectant cell lines.  In addn., the dominant-neg. PLCγ-1 fragment did not diminish EGF-induced thymidine incorporation.  All kinase active EGFR stimulated mitogen-activated protein (MAP) kinase activity, regardless of whether the receptors induced cell movement; this EGF-induced MAP kinase activity was not affected by U73122 at concns. that depressed the motility response.  Thus, the signaling pathways which lead to motility and cell proliferation diverge at the immediate post-receptor stage, and the authors suggest that this is accomplished by differential activation of effector mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBfocjcTbqMLVg90H21EOLACvtfcHk0lhIp7QS90VkyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmsVOitLk%253D&md5=9557cc67efc30b158a929bafb216a96a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1083%2Fjcb.127.3.847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.127.3.847%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DP.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DSekar%26aufirst%3DM.%2BC.%26aulast%3DGupta%26aufirst%3DK.%26aulast%3DWells%26aufirst%3DA.%26atitle%3DEpidermal%2520growth%2520factor%2520receptor-mediated%2520cell%2520motility%253A%2520phospholipase%2520C%2520activity%2520is%2520required%252C%2520but%2520mitogen-activated%2520protein%2520kinase%2520activity%2520is%2520not%2520sufficient%2520for%2520induced%2520cell%2520movement%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D1994%26volume%3D127%26spage%3D847%26epage%3D857%26doi%3D10.1083%2Fjcb.127.3.847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Datta, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dudek, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masters, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gotoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenberg, M. E.</span><span> </span><span class="NLM_article-title">Akt phosphorylation of BAD couples survival signals to the cell-intrinsic death machinery</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">231</span><span class="NLM_x">–</span> <span class="NLM_lpage">241</span><span class="refDoi"> DOI: 10.1016/S0092-8674(00)80405-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1016%2FS0092-8674%2800%2980405-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=9346240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaK2sXntVyiu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=1997&pages=231-241&author=S.+R.+Dattaauthor=H.+Dudekauthor=X.+Taoauthor=S.+Mastersauthor=H.+Fuauthor=Y.+Gotohauthor=M.+E.+Greenberg&title=Akt+phosphorylation+of+BAD+couples+survival+signals+to+the+cell-intrinsic+death+machinery&doi=10.1016%2FS0092-8674%2800%2980405-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Akt phosphorylation of BAD couples survival signals to the cell-intrinsic death machinery</span></div><div class="casAuthors">Datta, Sandeep Robert; Dudek, Henryk; Tao, Xu; Masters, Shane; Fu, Haian; Gotoh, Yukiko; Greenberg, Michael E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">231-241</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Growth factors can promote cell survival by activating the phosphatidylinositide-3'-OH kinase and its downstream target, the serine-threonine kinase Akt.  However, the mechanism by which Akt functions to promote survival is not understood.  The authors show that growth factor activation of the PI3'-K/Akt signaling pathway culminates in the phosphorylation of the BCL-2 family member BAD, thereby suppressing apoptosis and promoting cell survival.  Akt phosphorylates BAD in vitro and in vivo, and blocks the BAD-induced death of primary neurons in a site-specific manner.  These findings define a mechanism by which growth factors directly inactivate a crit. component of the cell-intrinsic death machinery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiB9FN9LI-RLVg90H21EOLACvtfcHk0ljbR9BbhAcD5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXntVyiu7k%253D&md5=7cec99113fddfdf23c290bec26a9f0dc</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2980405-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252980405-5%26sid%3Dliteratum%253Aachs%26aulast%3DDatta%26aufirst%3DS.%2BR.%26aulast%3DDudek%26aufirst%3DH.%26aulast%3DTao%26aufirst%3DX.%26aulast%3DMasters%26aufirst%3DS.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DGotoh%26aufirst%3DY.%26aulast%3DGreenberg%26aufirst%3DM.%2BE.%26atitle%3DAkt%2520phosphorylation%2520of%2520BAD%2520couples%2520survival%2520signals%2520to%2520the%2520cell-intrinsic%2520death%2520machinery%26jtitle%3DCell%26date%3D1997%26volume%3D91%26spage%3D231%26epage%3D241%26doi%3D10.1016%2FS0092-8674%2800%2980405-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Pages, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lenormand, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">L’Allemain, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambard, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meloche, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pouyssegur, J.</span><span> </span><span class="NLM_article-title">Mitogen-activated protein kinases p42mapk and p44mapk are required for fibroblast proliferation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">90</span><span class="NLM_x">, </span> <span class="NLM_fpage">8319</span><span class="NLM_x">–</span> <span class="NLM_lpage">8323</span><span class="refDoi"> DOI: 10.1073/pnas.90.18.8319</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1073%2Fpnas.90.18.8319" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1993&pages=8319-8323&author=G.+Pagesauthor=P.+Lenormandauthor=G.+L%E2%80%99Allemainauthor=J.+C.+Chambardauthor=S.+Melocheauthor=J.+Pouyssegur&title=Mitogen-activated+protein+kinases+p42mapk+and+p44mapk+are+required+for+fibroblast+proliferation&doi=10.1073%2Fpnas.90.18.8319"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1073%2Fpnas.90.18.8319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.90.18.8319%26sid%3Dliteratum%253Aachs%26aulast%3DPages%26aufirst%3DG.%26aulast%3DLenormand%26aufirst%3DP.%26aulast%3DL%25E2%2580%2599Allemain%26aufirst%3DG.%26aulast%3DChambard%26aufirst%3DJ.%2BC.%26aulast%3DMeloche%26aufirst%3DS.%26aulast%3DPouyssegur%26aufirst%3DJ.%26atitle%3DMitogen-activated%2520protein%2520kinases%2520p42mapk%2520and%2520p44mapk%2520are%2520required%2520for%2520fibroblast%2520proliferation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1993%26volume%3D90%26spage%3D8319%26epage%3D8323%26doi%3D10.1073%2Fpnas.90.18.8319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Rosell, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moran, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Queralt, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porta, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cardenal, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camps, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majem, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez-Vivanco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isla, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Provencio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Massuti, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez-Larriba, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paz-Ares, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bover, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia-Campelo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moreno, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Catot, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rolfo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reguart, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmero, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanchez, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bertran-Alamillo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Molina, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanchez, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taron, M.</span><span> </span><span class="NLM_article-title">Spanish Lung Cancer, G. Screening for epidermal growth factor receptor mutations in lung cancer</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">361</span><span class="NLM_x">, </span> <span class="NLM_fpage">958</span><span class="NLM_x">–</span> <span class="NLM_lpage">967</span><span class="refDoi"> DOI: 10.1056/NEJMoa0904554</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1056%2FNEJMoa0904554" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=19692684" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVOqsL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=361&publication_year=2009&pages=958-967&author=R.+Rosellauthor=T.+Moranauthor=C.+Queraltauthor=R.+Portaauthor=F.+Cardenalauthor=C.+Campsauthor=M.+Majemauthor=G.+Lopez-Vivancoauthor=D.+Islaauthor=M.+Provencioauthor=A.+Insaauthor=B.+Massutiauthor=J.+L.+Gonzalez-Larribaauthor=L.+Paz-Aresauthor=I.+Boverauthor=R.+Garcia-Campeloauthor=M.+A.+Morenoauthor=S.+Catotauthor=C.+Rolfoauthor=N.+Reguartauthor=R.+Palmeroauthor=J.+M.+Sanchezauthor=R.+Bastusauthor=C.+Mayoauthor=J.+Bertran-Alamilloauthor=M.+A.+Molinaauthor=J.+J.+Sanchezauthor=M.+Taron&title=Spanish+Lung+Cancer%2C+G.+Screening+for+epidermal+growth+factor+receptor+mutations+in+lung+cancer&doi=10.1056%2FNEJMoa0904554"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Screening for epidermal growth factor receptor mutations in lung cancer</span></div><div class="casAuthors">Rosell, Rafael; Moran, Teresa; Queralt, Cristina; Porta, Rut; Cardenal, Felipe; Camps, Carlos; Majem, Margarita; Lopez-Vivanco, Guillermo; Isla, Dolores; Provencio, Mariano; Insa, Amelia; Massuti, Bartomeu; Gonzalez-Larriba, Jose Luis; Paz-Ares, Luis; Bover, Isabel; Garcia-Campelo, Rosario; Moreno, Miguel Angel; Catot, Silvia; Rolfo, Christian; Reguart, Noemi; Palmero, Ramon; Sanchez, Jose Miguel; Bastus, Roman; Mayo, Clara; Bertran-Alamillo, Jordi; Molina, Miguel Angel; Sanchez, Jose Javier; Taron, Miquel; Cuello, M.; Pallares, C.; Lianes, P.; Remon, J.; Ibeas, R.; Martinez del Prado, P.; Angeles Sala, M.; Santander-Lobera, C.; Velez de Mendizabal, E.; Vinolas, N.; Terrasa, J.; Valdivia, J.; Diz, P.; Jimenez-Berlana, U.; Velasco-Ortiz de Taranco, A.; Ales Martinez, E.; Sanchez-Escribano, R.; Carrato, A.; Guillen-Ponce, C.; Mesia, C.; Antonio Macias, J.; Lopez-Brea, M.; Oramas, J.; Barneto, I.; Garrido, P.; Mayol, M. J.; Lopez, A.; Artal, A.; Saenz, A.; Hernando, S.; Cobo, M.; Blanco, R.; Bernabe, R.; Guillem, V.; Angel Munoz, M.; Maestu, I.; Salvatierra, A.; De Las Penas, R.; Alfaro, J.; Alberola, V.; Juan, O.; Martin, C.; Puertas, J.; Felip, E.; Gonzalez-Gonzalez, J. F.; Iglesias-Docampo, L.; Dorta, F. J.; Martinez-Aguillo, M.; Salgado, E.; Mesia, R.; Lastra-Aras, E.; Garcia-Munoz, J. P.; Lastra, R.; Alvarez, I.; Roig, J.; Oruezabal, J.; Poveda, A.; Lavernia, J.; Gutierrez, D.; Filipovich, E.; Aguiar, D.; Rodriguez, D.; Buxo, J.; Cardona, A. F.; Bes, P.; Paredes, A.; Tortorella, A. M.; Moreno, J. A.; Martinez-Garcia, J.; Alonso, J. L.; Lopez-Martin, A.; Echarri-Gonzalez, M. J.; Van Kooten, M.; Guerrero, A.; Domine, M.; Diaz, I.; Heras, L.; Garcia, R.; Anton, I.; Jarchum, G.; Bartolucci, R. E.; Lomas, M.; Rubiales, A.; Duque, J. L.; Escriva de Romani, S.; Barbeta, E.; Reina, J. J.; Castro, J.; Belda, C.; Vidal, J. M.; Trigo, J. M.; Vadell, C.; Zarba, J. J.; Esunza, P.; Garau, I.; Lopez-Pousa, A.; De la Gandara, I.; Widakowich, J.; Morales, S.; Martinez, M.; Luis, R.; de la Colina, M.; Calzas, J.; Garcia-Castro, I.; Ruiz, C.; Lopez-Criado, P.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">361</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">958-967</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Activating mutations in the epidermal growth factor receptor gene (EGFR) confer hypersensitivity to the tyrosine kinase inhibitors gefitinib and erlotinib in patients with advanced non-small-cell lung cancer.  We evaluated the feasibility of large-scale screening for EGFR mutations in such patients and analyzed the assocn. between the mutations and the outcome of erlotinib treatment.  From Apr. 2005 through Nov. 2008, lung cancers from 2105 patients in 129 institutions in Spain were screened for EGFR mutations.  The anal. was performed in a central lab.  Patients with tumors carrying EGFR mutations were eligible for erlotinib treatment.  EGFR mutations were found in 350 of 2105 patients (16.6%).  Mutations were more frequent in women (69.7%), in patients who had never smoked (66.6%), and in those with adenocarcinomas (80.9%) (P < 0.001 for all comparisons).  The mutations were deletions in exon 19 (62.2%) and L858R (37.8%).  Median progression-free survival and overall survival for 217 patients who received erlotinib were 14 mo and 27 mo, resp.  The adjusted hazard ratios for the duration of progression-free survival were 2.94 for men (P < 0.001); 1.92 for the presence of the L858R mutation, as compared with a deletion in exon 19 (P = 0.02); and 1.68 for the presence of the L858R mutation in paired serum DNA, as compared with the absence of the mutation (P = 0.02).  The most common adverse events were mild rashes and diarrhea; grade 3 cutaneous toxic effects were recorded in 16 patients (7.4%) and grade 3 diarrhea in 8 patients (3.7%).  Large-scale screening of patients with lung cancer for EGFR mutations is feasible and can have a role in decisions about treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvZWnSnU_ScbVg90H21EOLACvtfcHk0ljbR9BbhAcD5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVOqsL7M&md5=288ea35bd5201579d8c89ad6d7c4306a</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa0904554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa0904554%26sid%3Dliteratum%253Aachs%26aulast%3DRosell%26aufirst%3DR.%26aulast%3DMoran%26aufirst%3DT.%26aulast%3DQueralt%26aufirst%3DC.%26aulast%3DPorta%26aufirst%3DR.%26aulast%3DCardenal%26aufirst%3DF.%26aulast%3DCamps%26aufirst%3DC.%26aulast%3DMajem%26aufirst%3DM.%26aulast%3DLopez-Vivanco%26aufirst%3DG.%26aulast%3DIsla%26aufirst%3DD.%26aulast%3DProvencio%26aufirst%3DM.%26aulast%3DInsa%26aufirst%3DA.%26aulast%3DMassuti%26aufirst%3DB.%26aulast%3DGonzalez-Larriba%26aufirst%3DJ.%2BL.%26aulast%3DPaz-Ares%26aufirst%3DL.%26aulast%3DBover%26aufirst%3DI.%26aulast%3DGarcia-Campelo%26aufirst%3DR.%26aulast%3DMoreno%26aufirst%3DM.%2BA.%26aulast%3DCatot%26aufirst%3DS.%26aulast%3DRolfo%26aufirst%3DC.%26aulast%3DReguart%26aufirst%3DN.%26aulast%3DPalmero%26aufirst%3DR.%26aulast%3DSanchez%26aufirst%3DJ.%2BM.%26aulast%3DBastus%26aufirst%3DR.%26aulast%3DMayo%26aufirst%3DC.%26aulast%3DBertran-Alamillo%26aufirst%3DJ.%26aulast%3DMolina%26aufirst%3DM.%2BA.%26aulast%3DSanchez%26aufirst%3DJ.%2BJ.%26aulast%3DTaron%26aufirst%3DM.%26atitle%3DSpanish%2520Lung%2520Cancer%252C%2520G.%2520Screening%2520for%2520epidermal%2520growth%2520factor%2520receptor%2520mutations%2520in%2520lung%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2009%26volume%3D361%26spage%3D958%26epage%3D967%26doi%3D10.1056%2FNEJMoa0904554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Suzuki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobashi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakurai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishikawa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ooi, A.</span><span> </span><span class="NLM_article-title">Protein overexpression and gene amplification of epidermal growth factor receptor in non-small cell lung carcinomas. An immunohistochemical and fluorescence in situ hybridization study</span> <span class="citation_source-journal">Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">1265</span><span class="NLM_x">–</span> <span class="NLM_lpage">1273</span><span class="refDoi"> DOI: 10.1002/cncr.20909</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1002%2Fcncr.20909" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2005&pages=1265-1273&author=S.+Suzukiauthor=Y.+Dobashiauthor=H.+Sakuraiauthor=K.+Nishikawaauthor=M.+Hanawaauthor=A.+Ooi&title=Protein+overexpression+and+gene+amplification+of+epidermal+growth+factor+receptor+in+non-small+cell+lung+carcinomas.+An+immunohistochemical+and+fluorescence+in+situ+hybridization+study&doi=10.1002%2Fcncr.20909"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fcncr.20909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcncr.20909%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DS.%26aulast%3DDobashi%26aufirst%3DY.%26aulast%3DSakurai%26aufirst%3DH.%26aulast%3DNishikawa%26aufirst%3DK.%26aulast%3DHanawa%26aufirst%3DM.%26aulast%3DOoi%26aufirst%3DA.%26atitle%3DProtein%2520overexpression%2520and%2520gene%2520amplification%2520of%2520epidermal%2520growth%2520factor%2520receptor%2520in%2520non-small%2520cell%2520lung%2520carcinomas.%2520An%2520immunohistochemical%2520and%2520fluorescence%2520in%2520situ%2520hybridization%2520study%26jtitle%3DCancer%26date%3D2005%26volume%3D103%26spage%3D1265%26epage%3D1273%26doi%3D10.1002%2Fcncr.20909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Barker, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grundy, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godfrey, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barlow, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Healy, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodburn, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashton, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curry, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scarlett, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henthorn, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richards, L.</span><span> </span><span class="NLM_article-title">Studies leading to the identification of ZD1839 (IRESSA): an orally active, selective epidermal growth factor receptor tyrosine kinase inhibitor targeted to the treatment of cancer</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1911</span><span class="NLM_x">–</span> <span class="NLM_lpage">1914</span><span class="refDoi"> DOI: 10.1016/S0960-894X(01)00344-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1016%2FS0960-894X%2801%2900344-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=11459659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltFylsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=1911-1914&author=A.+J.+Barkerauthor=K.+H.+Gibsonauthor=W.+Grundyauthor=A.+A.+Godfreyauthor=J.+J.+Barlowauthor=M.+P.+Healyauthor=J.+R.+Woodburnauthor=S.+E.+Ashtonauthor=B.+J.+Curryauthor=L.+Scarlettauthor=L.+Henthornauthor=L.+Richards&title=Studies+leading+to+the+identification+of+ZD1839+%28IRESSA%29%3A+an+orally+active%2C+selective+epidermal+growth+factor+receptor+tyrosine+kinase+inhibitor+targeted+to+the+treatment+of+cancer&doi=10.1016%2FS0960-894X%2801%2900344-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Studies leading to the identification of ZD1839 (iressa): an orally active, selective epidermal growth factor receptor tyrosine kinase inhibitor targeted to the treatment of cancer</span></div><div class="casAuthors">Barker, A. J.; Gibson, K. H.; Grundy, W.; Godfrey, A. A.; Barlow, J. J.; Healy, M. P.; Woodburn, J. R.; Ashton, S. E.; Curry, B. J.; Scarlett, L.; Henthorn, L.; Richards, L.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1911-1914</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">This paper describes the development of the epidermal growth factor receptor tyrosine kinase inhibitor ZD1839 from a lead series of 4-anilinoquinazoline compds.  ZD1839 has suitable properties for use as a clin. effective drug and shows activity against human tumors.  In particular, the use of pharmacokinetic data in the development of ZD1839 is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvsJ1FTqEwvbVg90H21EOLACvtfcHk0lj4j-3FnarpCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltFylsLo%253D&md5=ebb210453ade246e43de9fd07ae1d581</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2801%2900344-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252801%252900344-4%26sid%3Dliteratum%253Aachs%26aulast%3DBarker%26aufirst%3DA.%2BJ.%26aulast%3DGibson%26aufirst%3DK.%2BH.%26aulast%3DGrundy%26aufirst%3DW.%26aulast%3DGodfrey%26aufirst%3DA.%2BA.%26aulast%3DBarlow%26aufirst%3DJ.%2BJ.%26aulast%3DHealy%26aufirst%3DM.%2BP.%26aulast%3DWoodburn%26aufirst%3DJ.%2BR.%26aulast%3DAshton%26aufirst%3DS.%2BE.%26aulast%3DCurry%26aufirst%3DB.%2BJ.%26aulast%3DScarlett%26aufirst%3DL.%26aulast%3DHenthorn%26aufirst%3DL.%26aulast%3DRichards%26aufirst%3DL.%26atitle%3DStudies%2520leading%2520to%2520the%2520identification%2520of%2520ZD1839%2520%2528IRESSA%2529%253A%2520an%2520orally%2520active%252C%2520selective%2520epidermal%2520growth%2520factor%2520receptor%2520tyrosine%2520kinase%2520inhibitor%2520targeted%2520to%2520the%2520treatment%2520of%2520cancer%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2001%26volume%3D11%26spage%3D1911%26epage%3D1914%26doi%3D10.1016%2FS0960-894X%2801%2900344-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Greulich, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jänne, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alvarez, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zappaterra, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bulmer, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frank, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sellers, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M.</span><span> </span><span class="NLM_article-title">Oncogenic transformation by inhibitor-sensitive and -resistant EGFR mutants</span> <span class="citation_source-journal">PLoS Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">e313</span><span class="refDoi"> DOI: 10.1371/journal.pmed.0020313</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1371%2Fjournal.pmed.0020313" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2005&pages=e313&author=H.+Greulichauthor=T.+H.+Chenauthor=W.+Fengauthor=P.+A.+J%C3%A4nneauthor=J.+V.+Alvarezauthor=M.+Zappaterraauthor=S.+E.+Bulmerauthor=D.+A.+Frankauthor=W.+C.+Hahnauthor=W.+R.+Sellersauthor=M.+Meyerson&title=Oncogenic+transformation+by+inhibitor-sensitive+and+-resistant+EGFR+mutants&doi=10.1371%2Fjournal.pmed.0020313"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pmed.0020313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pmed.0020313%26sid%3Dliteratum%253Aachs%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DT.%2BH.%26aulast%3DFeng%26aufirst%3DW.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DAlvarez%26aufirst%3DJ.%2BV.%26aulast%3DZappaterra%26aufirst%3DM.%26aulast%3DBulmer%26aufirst%3DS.%2BE.%26aulast%3DFrank%26aufirst%3DD.%2BA.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DMeyerson%26aufirst%3DM.%26atitle%3DOncogenic%2520transformation%2520by%2520inhibitor-sensitive%2520and%2520-resistant%2520EGFR%2520mutants%26jtitle%3DPLoS%2520Med.%26date%3D2005%26volume%3D2%26spage%3De313%26doi%3D10.1371%2Fjournal.pmed.0020313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Lynch, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sordella, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurubhagavatula, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okimoto, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brannigan, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haserlat, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Supko, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haluska, F. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Louis, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christiani, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Settleman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haber, D. A.</span><span> </span><span class="NLM_article-title">Activating mutations in the epidermal growth factor receptor underlying responsiveness of non-small-cell lung cancer to gefitinib</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">350</span><span class="NLM_x">, </span> <span class="NLM_fpage">2129</span><span class="NLM_x">–</span> <span class="NLM_lpage">2139</span><span class="refDoi"> DOI: 10.1056/NEJMoa040938</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1056%2FNEJMoa040938" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=15118073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2cXktF2js7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=350&publication_year=2004&pages=2129-2139&author=T.+J.+Lynchauthor=D.+W.+Bellauthor=R.+Sordellaauthor=S.+Gurubhagavatulaauthor=R.+A.+Okimotoauthor=B.+W.+Branniganauthor=P.+L.+Harrisauthor=S.+M.+Haserlatauthor=J.+G.+Supkoauthor=F.+G.+Haluskaauthor=D.+N.+Louisauthor=D.+C.+Christianiauthor=J.+Settlemanauthor=D.+A.+Haber&title=Activating+mutations+in+the+epidermal+growth+factor+receptor+underlying+responsiveness+of+non-small-cell+lung+cancer+to+gefitinib&doi=10.1056%2FNEJMoa040938"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Activating mutations in the epidermal growth factor receptor underlying responsiveness of non-small-cell lung cancer to gefitinib</span></div><div class="casAuthors">Lynch, Thomas J.; Bell, Daphne W.; Sordella, Raffaella; Gurubhagavatula, Sarada; Okimoto, Ross A.; Brannigan, Brian W.; Harris, Patricia L.; Haserlat, Sara M.; Supko, Jeffrey G.; Haluska, Frank G.; Louis, David N.; Christiani, David C.; Settleman, Jeff; Haber, Daniel A.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">350</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2129-2139</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Most patients with non-small-cell lung cancer have no response to the tyrosine kinase inhibitor gefitinib, which targets the epidermal growth factor receptor (EGFR).  However, about 10% of patients have a rapid and often dramatic clin. response.  The mol. mechanisms underlying sensitivity to gefitinib are unknown.  We searched for mutations in the EGFR gene in primary tumors from patients with non-small-cell lung cancer who had a response to gefitinib, those who did not have a response, and those who had not been exposed to gefitinib.  The functional consequences of identified mutations were evaluated after the mutant proteins were expressed in cultured cells.  Somatic mutations were identified in the tyrosine kinase domain of the EGFR gene in eight of nine patients with gefitinib-responsive lung cancer, as compared with none of the seven patients with no response (P < 0.001).  Mutations were either small, in-frame deletions or amino acid substitutions clustered around the ATP-binding pocket of the tyrosine kinase domain.  Similar mutations were detected in tumors from 2 of 25 patients with primary non-small-cell lung cancer who had not been exposed to gefitinib (8%).  All mutations were heterozygous, and identical mutations were obsd. in multiple patients, suggesting an additive specific gain of function.  In vitro, EGFR mutants demonstrated enhanced tyrosine kinase activity in response to epidermal growth factor and increased sensitivity to inhibition by gefitinib.  A subgroup of patients with non-small-cell lung cancer have specific mutations in the EGFR gene, which correlate with clin. responsiveness to the tyrosine kinase inhibitor gefitinib.  These mutations lead to increased growth factor signaling and confer susceptibility to the inhibitor.  Screening for such mutations in lung cancers may identify patients who will have a response to gefitinib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXwWGeAQrNTLVg90H21EOLACvtfcHk0lhhhAt1ndl4rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXktF2js7c%253D&md5=fad70aa0f32ad4105ad6f3db98135bf9</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa040938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa040938%26sid%3Dliteratum%253Aachs%26aulast%3DLynch%26aufirst%3DT.%2BJ.%26aulast%3DBell%26aufirst%3DD.%2BW.%26aulast%3DSordella%26aufirst%3DR.%26aulast%3DGurubhagavatula%26aufirst%3DS.%26aulast%3DOkimoto%26aufirst%3DR.%2BA.%26aulast%3DBrannigan%26aufirst%3DB.%2BW.%26aulast%3DHarris%26aufirst%3DP.%2BL.%26aulast%3DHaserlat%26aufirst%3DS.%2BM.%26aulast%3DSupko%26aufirst%3DJ.%2BG.%26aulast%3DHaluska%26aufirst%3DF.%2BG.%26aulast%3DLouis%26aufirst%3DD.%2BN.%26aulast%3DChristiani%26aufirst%3DD.%2BC.%26aulast%3DSettleman%26aufirst%3DJ.%26aulast%3DHaber%26aufirst%3DD.%2BA.%26atitle%3DActivating%2520mutations%2520in%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520underlying%2520responsiveness%2520of%2520non-small-cell%2520lung%2520cancer%2520to%2520gefitinib%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2004%26volume%3D350%26spage%3D2129%26epage%3D2139%26doi%3D10.1056%2FNEJMoa040938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Moyer, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbacci, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwata, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnold, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boman, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cunningham, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiOrio, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doty, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morin, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neveu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollack, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pustilnik, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reynolds, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sloan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theleman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, P.</span><span> </span><span class="NLM_article-title">Induction of apoptosis and cell cycle arrest by CP-358,774, an inhibitor of epidermal growth factor receptor tyrosine kinase</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">4838</span><span class="NLM_x">–</span> <span class="NLM_lpage">4848</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=9354447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaK2sXnt12nsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1997&pages=4838-4848&author=J.+D.+Moyerauthor=E.+G.+Barbacciauthor=K.+K.+Iwataauthor=L.+Arnoldauthor=B.+Bomanauthor=A.+Cunninghamauthor=C.+DiOrioauthor=J.+Dotyauthor=M.+J.+Morinauthor=M.+P.+Moyerauthor=M.+Neveuauthor=V.+A.+Pollackauthor=L.+R.+Pustilnikauthor=M.+M.+Reynoldsauthor=D.+Sloanauthor=A.+Thelemanauthor=P.+Miller&title=Induction+of+apoptosis+and+cell+cycle+arrest+by+CP-358%2C774%2C+an+inhibitor+of+epidermal+growth+factor+receptor+tyrosine+kinase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Induction of apoptosis and cell cycle arrest by CP-358774, an inhibitor of epidermal growth factor receptor tyrosine kinase</span></div><div class="casAuthors">Moyer, James D.; Barbacci, Elsa G.; Iwata, Kenneth K.; Arnold, Lee; Boman, Bruce; Cunningham, Ann; Diorio, Catherine; Doty, Jonathan; Morin, Michael J.; Moyer, Mikel P.; Neveu, Mark; Pollack, Vincent A.; Pustilnik, Leslie R.; Reynolds, Margaret M.; Sloan, Don; Theleman, April; Miller, Penny</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">4838-4848</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR) is overexpressed in a significant percentage of carcinomas and contributes to the malignant phenotype.  CP-358774 is a directly acting inhibitor of human EGFR tyrosine kinase with an IC50 of 2 nM and reduces EGFR autophosphorylation in intact tumor cells with an IC50 of 20 nM.  This inhibition is selective for EGFR tyrosine kinase relative to other tyrosine kinases we have examd., both in assays of isolated kinases and whole cells.  At doses of 100 mg/kg, CP-358774 completely prevents EGF-induced autophosphorylation of EGFR in human HN5 tumors growing as xenografts in athymic mice and of the hepatic EGFR of the treated mice.  CP-358774 inhibits the proliferation of DiFi human colon tumor cells at submicromolar concns. in cell culture and blocks cell cycle progression at the G1 phase.  This inhibitor produces a marked accumulation of retinoblastoma protein in its underphosphorylated form and accumulation of p27KIP1 in DiFi cells, which may contribute to the cell cycle block.  Inhibition of the EGFR also triggers apoptosis in these cells as detd. by formation of DNA fragments and other criteria.  These results indicate that CP-358774 has potential for the treatment of tumors that are dependent on the EGFR pathway for proliferation or survival.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpghArQOnGlkbVg90H21EOLACvtfcHk0lhhhAt1ndl4rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnt12nsr4%253D&md5=0a6e9697bf290d8e7c44cbe2a125e0b1</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMoyer%26aufirst%3DJ.%2BD.%26aulast%3DBarbacci%26aufirst%3DE.%2BG.%26aulast%3DIwata%26aufirst%3DK.%2BK.%26aulast%3DArnold%26aufirst%3DL.%26aulast%3DBoman%26aufirst%3DB.%26aulast%3DCunningham%26aufirst%3DA.%26aulast%3DDiOrio%26aufirst%3DC.%26aulast%3DDoty%26aufirst%3DJ.%26aulast%3DMorin%26aufirst%3DM.%2BJ.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DNeveu%26aufirst%3DM.%26aulast%3DPollack%26aufirst%3DV.%2BA.%26aulast%3DPustilnik%26aufirst%3DL.%2BR.%26aulast%3DReynolds%26aufirst%3DM.%2BM.%26aulast%3DSloan%26aufirst%3DD.%26aulast%3DTheleman%26aufirst%3DA.%26aulast%3DMiller%26aufirst%3DP.%26atitle%3DInduction%2520of%2520apoptosis%2520and%2520cell%2520cycle%2520arrest%2520by%2520CP-358%252C774%252C%2520an%2520inhibitor%2520of%2520epidermal%2520growth%2520factor%2520receptor%2520tyrosine%2520kinase%26jtitle%3DCancer%2520Res.%26date%3D1997%26volume%3D57%26spage%3D4838%26epage%3D4848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Paez, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jänne, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tracy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greulich, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gabriel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaye, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindeman, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boggon, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naoki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujii, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eck, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sellers, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M.</span><span> </span><span class="NLM_article-title">EGFR mutations in lung cancer: correlation with clinical response to gefitinib therapy</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">304</span><span class="NLM_x">, </span> <span class="NLM_fpage">1497</span><span class="NLM_x">–</span> <span class="NLM_lpage">1500</span><span class="refDoi"> DOI: 10.1126/science.1099314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1126%2Fscience.1099314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=15118125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVGmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=304&publication_year=2004&pages=1497-1500&author=J.+G.+Paezauthor=P.+A.+J%C3%A4nneauthor=J.+C.+Leeauthor=S.+Tracyauthor=H.+Greulichauthor=S.+Gabrielauthor=P.+Hermanauthor=F.+J.+Kayeauthor=N.+Lindemanauthor=T.+J.+Boggonauthor=K.+Naokiauthor=H.+Sasakiauthor=Y.+Fujiiauthor=M.+J.+Eckauthor=W.+R.+Sellersauthor=B.+E.+Johnsonauthor=M.+Meyerson&title=EGFR+mutations+in+lung+cancer%3A+correlation+with+clinical+response+to+gefitinib+therapy&doi=10.1126%2Fscience.1099314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR Mutations in Lung Cancer: Correlation with Clinical Response to Gefitinib Therapy</span></div><div class="casAuthors">Paez, J. Guillermo; Jaenne, Pasi A.; Lee, Jeffrey C.; Tracy, Sean; Greulich, Heidi; Gabriel, Stacey; Herman, Paula; Kaye, Frederic J.; Lindeman, Neal; Boggon, Titus J.; Naoki, Katsuhiko; Sasaki, Hidefumi; Fujii, Yoshitaka; Eck, Michael J.; Sellers, William R.; Johnson, Bruce E.; Meyerson, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">304</span>
        (<span class="NLM_cas:issue">5676</span>),
    <span class="NLM_cas:pages">1497-1500</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Receptor tyrosine kinase genes were sequenced in non-small cell lung cancer (NSCLC) and matched normal tissue.  Somatic mutations of the epidermal growth factor receptor gene EGFR were found in 15 of 58 unselected tumors from Japan and 1 of 61 from the United States.  Treatment with the EGFR kinase inhibitor gefitinib (Iressa) causes tumor regression in some patients with NSCLC, more frequently in Japan.  EGFR mutations were found in addnl. lung cancer samples from U.S. patients who responded to gefitinib therapy and in a lung adenocarcinoma cell line that was hypersensitive to growth inhibition by gefitinib, but not in gefitinib-insensitive tumors or cell lines.  These results suggest that EGFR mutations may predict sensitivity to gefitinib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGxLJH5f3Kb7Vg90H21EOLACvtfcHk0lhhhAt1ndl4rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVGmsbs%253D&md5=6df1b11bf9b3c284fbe000e34c61ebac</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.1099314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1099314%26sid%3Dliteratum%253Aachs%26aulast%3DPaez%26aufirst%3DJ.%2BG.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26aulast%3DTracy%26aufirst%3DS.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DGabriel%26aufirst%3DS.%26aulast%3DHerman%26aufirst%3DP.%26aulast%3DKaye%26aufirst%3DF.%2BJ.%26aulast%3DLindeman%26aufirst%3DN.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26aulast%3DNaoki%26aufirst%3DK.%26aulast%3DSasaki%26aufirst%3DH.%26aulast%3DFujii%26aufirst%3DY.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DJohnson%26aufirst%3DB.%2BE.%26aulast%3DMeyerson%26aufirst%3DM.%26atitle%3DEGFR%2520mutations%2520in%2520lung%2520cancer%253A%2520correlation%2520with%2520clinical%2520response%2520to%2520gefitinib%2520therapy%26jtitle%3DScience%26date%3D2004%26volume%3D304%26spage%3D1497%26epage%3D1500%26doi%3D10.1126%2Fscience.1099314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Pao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zakowski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doherty, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Politi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarkaria, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heelan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fulton, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mardis, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kupfer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kris, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varmus, H.</span><span> </span><span class="NLM_article-title">EGF receptor gene mutations are common in lung cancers from ″never smokers″ and are associated with sensitivity of tumors to gefitinib and erlotinib</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">13306</span><span class="NLM_x">–</span> <span class="NLM_lpage">13311</span><span class="refDoi"> DOI: 10.1073/pnas.0405220101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1073%2Fpnas.0405220101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=15329413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnvFems70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=13306-13311&author=W.+Paoauthor=V.+Millerauthor=M.+Zakowskiauthor=J.+Dohertyauthor=K.+Politiauthor=I.+Sarkariaauthor=B.+Singhauthor=R.+Heelanauthor=V.+Ruschauthor=L.+Fultonauthor=E.+Mardisauthor=D.+Kupferauthor=R.+Wilsonauthor=M.+Krisauthor=H.+Varmus&title=EGF+receptor+gene+mutations+are+common+in+lung+cancers+from+%E2%80%B3never+smokers%E2%80%B3+and+are+associated+with+sensitivity+of+tumors+to+gefitinib+and+erlotinib&doi=10.1073%2Fpnas.0405220101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">EGF receptor gene mutations are common in lung cancers from "never smokers" and are associated with sensitivity of tumors to gefitinib and erlotinib</span></div><div class="casAuthors">Pao, William; Miller, Vincent; Zakowski, Maureen; Doherty, Jennifer; Politi, Katerina; Sarkaria, Inderpal; Singh, Bhuvanesh; Heelan, Robert; Rusch, Valerie; Fulton, Lucinda; Mardis, Elaine; Kupfer, Doris; Wilson, Richard; Kris, Mark; Varmus, Harold</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">13306-13311</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Somatic mutations in the tyrosine kinase (TK) domain of the epidermal growth factor receptor (EGFR) gene are reportedly assocd. with sensitivity of lung cancers to gefitinib (Iressa), kinase inhibitor.  In-frame deletions occur in exon 19, whereas point mutations occur frequently in codon 858 (exon 21).  We found from sequencing the EGFR TK domain that 7 of 10 gefitinib-sensitive tumors had similar types of alterations; no mutations were found in eight gefitinib-refractory tumors (P = 0.004).  Five of seven tumors sensitive to erlotinib (Tarceva), a related kinase inhibitor for which the clin. relevant target is undocumented, had analogous somatic mutations, as opposed to none of 10 erlotinib-refractory tumors (P = 0.003).  Because most mutation-pos. tumors were adenocarcinomas from patients who smoked <100 cigarettes in a lifetime ("never smokers"), we screened EGFR exons 2-28 in 15 adenocarcinomas resected from untreated never smokers.  Seven tumors had TK domain mutations, in contrast to 4 of 81 non-small cell lung cancers resected from untreated former or current smokers (P = 0.0001).  Immunoblotting of lysates from cells transiently transfected with various EGFR constructs demonstrated that, compared to wild-type protein, an exon 19 deletion mutant induced diminished levels of phosphotyrosine, whereas the phosphorylation at tyrosine 1092 of an exon 21 point mutant was inhibited at 10-fold lower concns. of drug.  Collectively, these data show that adenocarcinomas from never smokers comprise a distinct subset of lung cancers, frequently contg. mutations within the TK domain of EGFR that are assocd. with gefitinib and erlotinib sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovevWonTGqELVg90H21EOLACvtfcHk0lgGcq06p0xnkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnvFems70%253D&md5=38f491968552501b7463a688fa0e62ff</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0405220101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0405220101%26sid%3Dliteratum%253Aachs%26aulast%3DPao%26aufirst%3DW.%26aulast%3DMiller%26aufirst%3DV.%26aulast%3DZakowski%26aufirst%3DM.%26aulast%3DDoherty%26aufirst%3DJ.%26aulast%3DPoliti%26aufirst%3DK.%26aulast%3DSarkaria%26aufirst%3DI.%26aulast%3DSingh%26aufirst%3DB.%26aulast%3DHeelan%26aufirst%3DR.%26aulast%3DRusch%26aufirst%3DV.%26aulast%3DFulton%26aufirst%3DL.%26aulast%3DMardis%26aufirst%3DE.%26aulast%3DKupfer%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DR.%26aulast%3DKris%26aufirst%3DM.%26aulast%3DVarmus%26aufirst%3DH.%26atitle%3DEGF%2520receptor%2520gene%2520mutations%2520are%2520common%2520in%2520lung%2520cancers%2520from%2520%25E2%2580%25B3never%2520smokers%25E2%2580%25B3%2520and%2520are%2520associated%2520with%2520sensitivity%2520of%2520tumors%2520to%2520gefitinib%2520and%2520erlotinib%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26spage%3D13306%26epage%3D13311%26doi%3D10.1073%2Fpnas.0405220101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Sordella, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haber, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Settleman, J.</span><span> </span><span class="NLM_article-title">Gefitinib-sensitizing EGFR mutations in lung cancer activate anti-apoptotic pathways</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">305</span><span class="NLM_x">, </span> <span class="NLM_fpage">1163</span><span class="NLM_x">–</span> <span class="NLM_lpage">1167</span><span class="refDoi"> DOI: 10.1126/science.1101637</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1126%2Fscience.1101637" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=15284455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmslCmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=305&publication_year=2004&pages=1163-1167&author=R.+Sordellaauthor=D.+W.+Bellauthor=D.+A.+Haberauthor=J.+Settleman&title=Gefitinib-sensitizing+EGFR+mutations+in+lung+cancer+activate+anti-apoptotic+pathways&doi=10.1126%2Fscience.1101637"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Gefitinib-Sensitizing EGFR Mutations in Lung Cancer Activate Anti-Apoptotic Pathways</span></div><div class="casAuthors">Sordella, Raffaella; Bell, Daphne W.; Haber, Daniel A.; Settleman, Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">305</span>
        (<span class="NLM_cas:issue">5687</span>),
    <span class="NLM_cas:pages">1163-1167</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Gefitinib (Iressa, AstraZeneca Pharmaceuticals) is a tyrosine kinase inhibitor that targets the epidermal growth factor receptor (EGFR) and induces dramatic clin. responses in nonsmall cell lung cancers (NSCLCs) with activating mutations within the EGFR kinase domain.  The authors report that these mutant EGFRs selectively activate Akt and signal transduction and activator of transcription (STAT) signaling pathways, which promote cell survival, but have no effect on extracellular signal-regulated kinase signaling, which induces proliferation.  NSCLC cells expressing mutant EGFRs underwent extensive apoptosis after small interfering RNA-mediated knockdown of the mutant EGFR or treatment with pharmacol. inhibitors of Akt and STAT signaling and were relatively resistant to apoptosis induced by conventional chemotherapeutic drugs.  Thus, mutant EGFRs selectively transduce survival signals on which NSCLCs become dependent; inhibition of those signals by gefitinib may contribute to the drug's efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrKHDqaSobm7Vg90H21EOLACvtfcHk0lgGcq06p0xnkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmslCmur8%253D&md5=e9742920d5e320357b774495746e6581</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1126%2Fscience.1101637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1101637%26sid%3Dliteratum%253Aachs%26aulast%3DSordella%26aufirst%3DR.%26aulast%3DBell%26aufirst%3DD.%2BW.%26aulast%3DHaber%26aufirst%3DD.%2BA.%26aulast%3DSettleman%26aufirst%3DJ.%26atitle%3DGefitinib-sensitizing%2520EGFR%2520mutations%2520in%2520lung%2520cancer%2520activate%2520anti-apoptotic%2520pathways%26jtitle%3DScience%26date%3D2004%26volume%3D305%26spage%3D1163%26epage%3D1167%26doi%3D10.1126%2Fscience.1101637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Kobayashi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boggon, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dayaram, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jänne, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kocher, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eck, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tenen, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halmos, B.</span><span> </span><span class="NLM_article-title">EGFR mutation and resistance of non-small-cell lung cancer to gefitinib</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">352</span><span class="NLM_x">, </span> <span class="NLM_fpage">786</span><span class="NLM_x">–</span> <span class="NLM_lpage">792</span><span class="refDoi"> DOI: 10.1056/NEJMoa044238</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1056%2FNEJMoa044238" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=15728811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhsFCiurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2005&pages=786-792&author=S.+Kobayashiauthor=T.+J.+Boggonauthor=T.+Dayaramauthor=P.+A.+J%C3%A4nneauthor=O.+Kocherauthor=M.+Meyersonauthor=B.+E.+Johnsonauthor=M.+J.+Eckauthor=D.+G.+Tenenauthor=B.+Halmos&title=EGFR+mutation+and+resistance+of+non-small-cell+lung+cancer+to+gefitinib&doi=10.1056%2FNEJMoa044238"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR mutation and resistance of non-small-cell lung cancer to gefitinib</span></div><div class="casAuthors">Kobayashi, Susumu; Boggon, Titus J.; Dayaram, Tajhal; Jaenne, Pasi A.; Kocher, Olivier; Meyerson, Matthew; Johnson, Bruce E.; Eck, Michael J.; Tenen, Daniel G.; Halmos, Balizs</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">352</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">786-792</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Mutations of the epidermal growth factor receptor (EGFR) gene were identified in specimens from patients with non-small-cell lung cancer who have a response to anilinoquinazoline EGFR inhibitors.  Despite the dramatic responses to such inhibitors, most patients ultimately have a relapse.  The mechanism of the drug resistance is unknown.  Here the authors report the case of a patient with EGFR-mutant, gefitinib-responsive, advanced non-small-cell lung cancer who had a relapse after 2 years of complete remission during treatment with gefitinib.  The DNA sequence of the EGFR gene in his tumor biopsy specimen at relapse revealed the presence of a second point mutation, resulting in threonine-to-methionine amino acid change at position 790 of EGFR.  Structural modeling and biochem. studies showed that this second mutation led to gefitinib resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8XXhT1hrbe7Vg90H21EOLACvtfcHk0lgGcq06p0xnkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhsFCiurk%253D&md5=808223ad566e2a3748977727cc5a9846</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa044238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa044238%26sid%3Dliteratum%253Aachs%26aulast%3DKobayashi%26aufirst%3DS.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26aulast%3DDayaram%26aufirst%3DT.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DKocher%26aufirst%3DO.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DB.%2BE.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DTenen%26aufirst%3DD.%2BG.%26aulast%3DHalmos%26aufirst%3DB.%26atitle%3DEGFR%2520mutation%2520and%2520resistance%2520of%2520non-small-cell%2520lung%2520cancer%2520to%2520gefitinib%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2005%26volume%3D352%26spage%3D786%26epage%3D792%26doi%3D10.1056%2FNEJMoa044238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Pao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Politi, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riely, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somwar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zakowski, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kris, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varmus, H.</span><span> </span><span class="NLM_article-title">Acquired resistance of lung adenocarcinomas to gefitinib or erlotinib is associated with a second mutation in the EGFR kinase domain</span> <span class="citation_source-journal">PLoS Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">e73</span><span class="refDoi"> DOI: 10.1371/journal.pmed.0020073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1371%2Fjournal.pmed.0020073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=15737014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A280%3ADC%252BD2MzjtVehsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2005&pages=e73&author=W.+Paoauthor=V.+A.+Millerauthor=K.+A.+Politiauthor=G.+J.+Rielyauthor=R.+Somwarauthor=M.+F.+Zakowskiauthor=M.+G.+Krisauthor=H.+Varmus&title=Acquired+resistance+of+lung+adenocarcinomas+to+gefitinib+or+erlotinib+is+associated+with+a+second+mutation+in+the+EGFR+kinase+domain&doi=10.1371%2Fjournal.pmed.0020073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Acquired resistance of lung adenocarcinomas to gefitinib or erlotinib is associated with a second mutation in the EGFR kinase domain</span></div><div class="casAuthors">Pao William; Miller Vincent A; Politi Katerina A; Riely Gregory J; Somwar Romel; Zakowski Maureen F; Kris Mark G; Varmus Harold</div><div class="citationInfo"><span class="NLM_cas:title">PLoS medicine</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e73</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Lung adenocarcinomas from patients who respond to the tyrosine kinase inhibitors gefitinib (Iressa) or erlotinib (Tarceva) usually harbor somatic gain-of-function mutations in exons encoding the kinase domain of the epidermal growth factor receptor (EGFR).  Despite initial responses, patients eventually progress by unknown mechanisms of "acquired" resistance.  METHODS AND FINDINGS:  We show that in two of five patients with acquired resistance to gefitinib or erlotinib, progressing tumors contain, in addition to a primary drug-sensitive mutation in EGFR, a secondary mutation in exon 20, which leads to substitution of methionine for threonine at position 790 (T790M) in the kinase domain.  Tumor cells from a sixth patient with a drug-sensitive EGFR mutation whose tumor progressed on adjuvant gefitinib after complete resection also contained the T790M mutation.  This mutation was not detected in untreated tumor samples.  Moreover, no tumors with acquired resistance had KRAS mutations, which have been associated with primary resistance to these drugs.  Biochemical analyses of transfected cells and growth inhibition studies with lung cancer cell lines demonstrate that the T790M mutation confers resistance to EGFR mutants usually sensitive to either gefitinib or erlotinib.  Interestingly, a mutation analogous to T790M has been observed in other kinases with acquired resistance to another kinase inhibitor, imatinib (Gleevec).  CONCLUSION:  In patients with tumors bearing gefitinib- or erlotinib-sensitive EGFR mutations, resistant subclones containing an additional EGFR mutation emerge in the presence of drug.  This observation should help guide the search for more effective therapy against a specific subset of lung cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcREkXMV3u1nwwMthz5_Rx5rfW6udTcc2eZhpYOL3ljiRLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2MzjtVehsw%253D%253D&md5=3e626105e0dea2a38997156ae513f56b</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pmed.0020073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pmed.0020073%26sid%3Dliteratum%253Aachs%26aulast%3DPao%26aufirst%3DW.%26aulast%3DMiller%26aufirst%3DV.%2BA.%26aulast%3DPoliti%26aufirst%3DK.%2BA.%26aulast%3DRiely%26aufirst%3DG.%2BJ.%26aulast%3DSomwar%26aufirst%3DR.%26aulast%3DZakowski%26aufirst%3DM.%2BF.%26aulast%3DKris%26aufirst%3DM.%2BG.%26aulast%3DVarmus%26aufirst%3DH.%26atitle%3DAcquired%2520resistance%2520of%2520lung%2520adenocarcinomas%2520to%2520gefitinib%2520or%2520erlotinib%2520is%2520associated%2520with%2520a%2520second%2520mutation%2520in%2520the%2520EGFR%2520kinase%2520domain%26jtitle%3DPLoS%2520Med.%26date%3D2005%26volume%3D2%26spage%3De73%26doi%3D10.1371%2Fjournal.pmed.0020073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Sequist, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waltman, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dias-Santagata, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Digumarthy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turke, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fidias, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergethon, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gettinger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosper, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akhavanfard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heist, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Temel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wain, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vernovsky, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mark, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lanuti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iafrate, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mino-Kenudson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelman, J. A.</span><span> </span><span class="NLM_article-title">Genotypic and histological evolution of lung cancers acquiring resistance to EGFR inhibitors</span> <span class="citation_source-journal">Sci. Transl. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">75ra26</span><span class="refDoi"> DOI: 10.1126/scitranslmed.3002003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1126%2Fscitranslmed.3002003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=21430269" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=75ra26&author=L.+V.+Sequistauthor=B.+A.+Waltmanauthor=D.+Dias-Santagataauthor=S.+Digumarthyauthor=A.+B.+Turkeauthor=P.+Fidiasauthor=K.+Bergethonauthor=A.+T.+Shawauthor=S.+Gettingerauthor=A.+K.+Cosperauthor=S.+Akhavanfardauthor=R.+S.+Heistauthor=J.+Temelauthor=J.+G.+Christensenauthor=J.+C.+Wainauthor=T.+J.+Lynchauthor=K.+Vernovskyauthor=E.+J.+Markauthor=M.+Lanutiauthor=A.+J.+Iafrateauthor=M.+Mino-Kenudsonauthor=J.+A.+Engelman&title=Genotypic+and+histological+evolution+of+lung+cancers+acquiring+resistance+to+EGFR+inhibitors&doi=10.1126%2Fscitranslmed.3002003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.3002003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.3002003%26sid%3Dliteratum%253Aachs%26aulast%3DSequist%26aufirst%3DL.%2BV.%26aulast%3DWaltman%26aufirst%3DB.%2BA.%26aulast%3DDias-Santagata%26aufirst%3DD.%26aulast%3DDigumarthy%26aufirst%3DS.%26aulast%3DTurke%26aufirst%3DA.%2BB.%26aulast%3DFidias%26aufirst%3DP.%26aulast%3DBergethon%26aufirst%3DK.%26aulast%3DShaw%26aufirst%3DA.%2BT.%26aulast%3DGettinger%26aufirst%3DS.%26aulast%3DCosper%26aufirst%3DA.%2BK.%26aulast%3DAkhavanfard%26aufirst%3DS.%26aulast%3DHeist%26aufirst%3DR.%2BS.%26aulast%3DTemel%26aufirst%3DJ.%26aulast%3DChristensen%26aufirst%3DJ.%2BG.%26aulast%3DWain%26aufirst%3DJ.%2BC.%26aulast%3DLynch%26aufirst%3DT.%2BJ.%26aulast%3DVernovsky%26aufirst%3DK.%26aulast%3DMark%26aufirst%3DE.%2BJ.%26aulast%3DLanuti%26aufirst%3DM.%26aulast%3DIafrate%26aufirst%3DA.%2BJ.%26aulast%3DMino-Kenudson%26aufirst%3DM.%26aulast%3DEngelman%26aufirst%3DJ.%2BA.%26atitle%3DGenotypic%2520and%2520histological%2520evolution%2520of%2520lung%2520cancers%2520acquiring%2520resistance%2520to%2520EGFR%2520inhibitors%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2011%26volume%3D3%26spage%3D75ra26%26doi%3D10.1126%2Fscitranslmed.3002003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Yun, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mengwasser, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toms, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woo, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greulich, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eck, M. J.</span><span> </span><span class="NLM_article-title">The T790M mutation in EGFR kinase causes drug resistance by increasing the affinity for ATP</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">2070</span><span class="NLM_x">–</span> <span class="NLM_lpage">2075</span><span class="refDoi"> DOI: 10.1073/pnas.0709662105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1073%2Fpnas.0709662105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=18227510" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitl2mu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=2070-2075&author=C.+H.+Yunauthor=K.+E.+Mengwasserauthor=A.+V.+Tomsauthor=M.+S.+Wooauthor=H.+Greulichauthor=K.+K.+Wongauthor=M.+Meyersonauthor=M.+J.+Eck&title=The+T790M+mutation+in+EGFR+kinase+causes+drug+resistance+by+increasing+the+affinity+for+ATP&doi=10.1073%2Fpnas.0709662105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The T790M mutation in EGFR kinase causes drug resistance by increasing the affinity for ATP</span></div><div class="casAuthors">Yun, Cai-Hong; Mengwasser, Kristen E.; Toms, Angela V.; Woo, Michele S.; Greulich, Heidi; Wong, Kwok-Kin; Meyerson, Matthew; Eck, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2070-2075</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Lung cancers caused by activating mutations in the epidermal growth factor receptor (EGFR) are initially responsive to small mol. tyrosine kinase inhibitors (TKIs), but the efficacy of these agents is often limited because of the emergence of drug resistance conferred by a second mutation, T790M.  Threonine 790 is the "gatekeeper" residue, an important determinant of inhibitor specificity in the ATP binding pocket.  The T790M mutation has been thought to cause resistance by sterically blocking binding of TKIs such as gefitinib and erlotinib, but this explanation is difficult to reconcile with the fact that it remains sensitive to structurally similar irreversible inhibitors.  Here, we show by using a direct binding assay that T790M mutants retain low-nanomolar affinity for gefitinib.  Furthermore, we show that the T790M mutation activates WT EGFR and that introduction of the T790M mutation increases the ATP affinity of the oncogenic L858R mutant by more than an order of magnitude.  The increased ATP affinity is the primary mechanism by which the T790M mutation confers drug resistance.  Crystallog. anal. of the T790M mutant shows how it can adapt to accommodate tight binding of diverse inhibitors, including the irreversible inhibitor HKI-272, and also suggests a structural mechanism for catalytic activation.  We conclude that the T790M mutation is a "generic" resistance mutation that will reduce the potency of any ATP-competitive kinase inhibitor and that irreversible inhibitors overcome this resistance simply through covalent binding, not as a result of an alternative binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXXP7TBlyrdrVg90H21EOLACvtfcHk0lhyC-l-tG2FOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitl2mu7k%253D&md5=f5f5d791a65f73ca628d95e9922b8ea8</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0709662105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0709662105%26sid%3Dliteratum%253Aachs%26aulast%3DYun%26aufirst%3DC.%2BH.%26aulast%3DMengwasser%26aufirst%3DK.%2BE.%26aulast%3DToms%26aufirst%3DA.%2BV.%26aulast%3DWoo%26aufirst%3DM.%2BS.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26atitle%3DThe%2520T790M%2520mutation%2520in%2520EGFR%2520kinase%2520causes%2520drug%2520resistance%2520by%2520increasing%2520the%2520affinity%2520for%2520ATP%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D2070%26epage%3D2075%26doi%3D10.1073%2Fpnas.0709662105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Klüter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simard, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rode, H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grütter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pawar, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raaijmakers, H. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barf, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabiller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Otterlo, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rauh, D.</span><span> </span><span class="NLM_article-title">Characterization of irreversible kinase inhibitors by directly detecting covalent bond formation: a tool for dissecting kinase drug resistance</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">2557</span><span class="NLM_x">–</span> <span class="NLM_lpage">2566</span><span class="refDoi"> DOI: 10.1002/cbic.201000352</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1002%2Fcbic.201000352" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=21080395" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFahsLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=2557-2566&author=S.+Kl%C3%BCterauthor=J.+R.+Simardauthor=H.+B.+Rodeauthor=C.+Gr%C3%BCtterauthor=V.+Pawarauthor=H.+C.+Raaijmakersauthor=T.+A.+Barfauthor=M.+Rabillerauthor=W.+A.+van+Otterloauthor=D.+Rauh&title=Characterization+of+irreversible+kinase+inhibitors+by+directly+detecting+covalent+bond+formation%3A+a+tool+for+dissecting+kinase+drug+resistance&doi=10.1002%2Fcbic.201000352"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of Irreversible Kinase Inhibitors by Directly Detecting Covalent Bond Formation: A Tool for Dissecting Kinase Drug Resistance</span></div><div class="casAuthors">Klueter, Sabine; Simard, Jeffrey R.; Rode, Haridas B.; Gruetter, Christian; Pawar, Vijaykumar; Raaijmakers, Hans C. A.; Barf, Tjeerd A.; Rabiller, Matthias; van Otterlo, Willem A. L.; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2557-2566</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Targeting protein kinases in cancer therapy with irreversible small-mol. inhibitors is moving to the forefront of kinase-inhibitor research and is thought to be an effective means of overcoming mutation-assocd. drug resistance in epidermal growth factor receptor kinase (EGFR).  We generated a detection technique that allows direct measurements of covalent bond formation without relying on kinase activity, thereby allowing the straightforward investigation of the influence of steric clashes on covalent inhibitors in different resistant kinase mutants.  The obtained results are discussed together with structural biol. and biochem. studies of catalytic activity in both wild-type and gatekeeper mutated kinase variants to draw conclusions about the impact of steric hindrance and increased catalytic activity in drug-resistant kinase variants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXBFAgMOxkBLVg90H21EOLACvtfcHk0lhyC-l-tG2FOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFahsLrE&md5=5f48cde54e6bad28b4d4f9d47a4e3e3e</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201000352&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201000352%26sid%3Dliteratum%253Aachs%26aulast%3DKl%25C3%25BCter%26aufirst%3DS.%26aulast%3DSimard%26aufirst%3DJ.%2BR.%26aulast%3DRode%26aufirst%3DH.%2BB.%26aulast%3DGr%25C3%25BCtter%26aufirst%3DC.%26aulast%3DPawar%26aufirst%3DV.%26aulast%3DRaaijmakers%26aufirst%3DH.%2BC.%26aulast%3DBarf%26aufirst%3DT.%2BA.%26aulast%3DRabiller%26aufirst%3DM.%26aulast%3Dvan%2BOtterlo%26aufirst%3DW.%2BA.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DCharacterization%2520of%2520irreversible%2520kinase%2520inhibitors%2520by%2520directly%2520detecting%2520covalent%2520bond%2520formation%253A%2520a%2520tool%2520for%2520dissecting%2520kinase%2520drug%2520resistance%26jtitle%3DChemBioChem%26date%3D2010%26volume%3D11%26spage%3D2557%26epage%3D2566%26doi%3D10.1002%2Fcbic.201000352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Michalczyk, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kluter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rode, H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simard, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grutter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabiller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rauh, D.</span><span> </span><span class="NLM_article-title">Structural insights into how irreversible inhibitors can overcome drug resistance in EGFR</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">3482</span><span class="NLM_x">–</span> <span class="NLM_lpage">3488</span><span class="refDoi"> DOI: 10.1016/j.bmc.2008.02.053</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1016%2Fj.bmc.2008.02.053" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=18316192" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1cXks1Gruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=3482-3488&author=A.+Michalczykauthor=S.+Kluterauthor=H.+B.+Rodeauthor=J.+R.+Simardauthor=C.+Grutterauthor=M.+Rabillerauthor=D.+Rauh&title=Structural+insights+into+how+irreversible+inhibitors+can+overcome+drug+resistance+in+EGFR&doi=10.1016%2Fj.bmc.2008.02.053"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into how irreversible inhibitors can overcome drug resistance in EGFR</span></div><div class="casAuthors">Michalczyk, Anja; Klueter, Sabine; Rode, Haridas B.; Simard, Jeffrey R.; Gruetter, Christian; Rabiller, Matthias; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3482-3488</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Resistance to kinase-targeted cancer drugs has recently been linked to a single point mutation in the ATP binding site of the kinase.  In EGFR, the crucial Thr790 gatekeeper residue is mutated to a Met and prevents reversible ATP competitive inhibitors from binding.  Irreversible 4-(phenylamino)quinazolines have been shown to overcome this drug resistance and are currently in clin. trials.  In order to obtain a detailed structural understanding of how irreversible inhibitors overcome drug resistance, the authors used Src kinase as a model system for drug resistant EGFR-T790M.  The authors report the first crystal structure of a drug resistant kinase in complex with an irreversible inhibitor.  This 4-(phenylamino)quinazoline inhibits wild type and drug resistant EGFR in vitro at low nM concns.  The cocrystal structure of drug resistant cSrc-T338M kinase domain provides the structural basis of this activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrp52JtORqsX7Vg90H21EOLACvtfcHk0lgB2twaIc5mLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXks1Gruro%253D&md5=b9fc704067e495279bad4913e9c703ab</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2008.02.053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2008.02.053%26sid%3Dliteratum%253Aachs%26aulast%3DMichalczyk%26aufirst%3DA.%26aulast%3DKluter%26aufirst%3DS.%26aulast%3DRode%26aufirst%3DH.%2BB.%26aulast%3DSimard%26aufirst%3DJ.%2BR.%26aulast%3DGrutter%26aufirst%3DC.%26aulast%3DRabiller%26aufirst%3DM.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DStructural%2520insights%2520into%2520how%2520irreversible%2520inhibitors%2520can%2520overcome%2520drug%2520resistance%2520in%2520EGFR%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2008%26volume%3D16%26spage%3D3482%26epage%3D3488%26doi%3D10.1016%2Fj.bmc.2008.02.053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Sos, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rode, H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heynck, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peifer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klüter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pawar, V. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reuter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heuckmann, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruddigkeit, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabiller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simard, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getlik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuza, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greulich, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rauh, D.</span><span> </span><span class="NLM_article-title">Chemogenomic profiling provides insights into the limited activity of irreversible EGFR Inhibitors in tumor cells expressing the T790M EGFR resistance mutation</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x">, </span> <span class="NLM_fpage">868</span><span class="NLM_x">–</span> <span class="NLM_lpage">874</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-09-3106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1158%2F0008-5472.CAN-09-3106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=20103621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12ru7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2010&pages=868-874&author=M.+L.+Sosauthor=H.+B.+Rodeauthor=S.+Heynckauthor=M.+Peiferauthor=F.+Fischerauthor=S.+Kl%C3%BCterauthor=V.+G.+Pawarauthor=C.+Reuterauthor=J.+M.+Heuckmannauthor=J.+Weissauthor=L.+Ruddigkeitauthor=M.+Rabillerauthor=M.+Kokerauthor=J.+R.+Simardauthor=M.+Getlikauthor=Y.+Yuzaauthor=T.+H.+Chenauthor=H.+Greulichauthor=R.+K.+Thomasauthor=D.+Rauh&title=Chemogenomic+profiling+provides+insights+into+the+limited+activity+of+irreversible+EGFR+Inhibitors+in+tumor+cells+expressing+the+T790M+EGFR+resistance+mutation&doi=10.1158%2F0008-5472.CAN-09-3106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomic Profiling Provides Insights into the Limited Activity of Irreversible EGFR Inhibitors in Tumor Cells Expressing the T790M EGFR Resistance Mutation</span></div><div class="casAuthors">Sos, Martin L.; Rode, Haridas B.; Heynck, Stefanie; Peifer, Martin; Fischer, Florian; Klueter, Sabine; Pawar, Vijaykumar G.; Reuter, Cecile; Heuckmann, Johannes M.; Weiss, Jonathan; Ruddigkeit, Lars; Rabiller, Matthias; Koker, Mirjam; Simard, Jeffrey R.; Getlik, Matthaeus; Yuza, Yuki; Chen, Tzu-Hsiu; Greulich, Heidi; Thomas, Roman K.; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">868-874</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Reversible epidermal growth factor receptor (EGFR) inhibitors are the first class of small mols. to improve progression-free survival of patients with EGFR-mutated lung cancers.  Second-generation EGFR inhibitors introduced to overcome acquired resistance by the T790M resistance mutation of EGFR have thus far shown limited clin. activity in patients with T790M-mutant tumors.  In this study, we systematically analyzed the determinants of the activity and selectivity of the second-generation EGFR inhibitors.  A focused library of irreversible as well as structurally corresponding reversible EGFR-inhibitors was synthesized for chemogenomic profiling involving over 79 genetically defined NSCLC and 19 EGFR-dependent cell lines.  Overall, our results show that the growth-inhibitory potency of all irreversible inhibitors against the EGFRT790M resistance mutation was limited by reduced target inhibition, linked to decreased binding velocity to the mutant kinase.  Combined treatment of T790M-mutant tumor cells with BIBW-2992 and the phosphoinositide-3-kinase/mammalian target of rapamycin inhibitor PI-103 led to synergistic induction of apoptosis.  Our findings offer a mechanistic explanation for the limited efficacy of irreversible EGFR inhibitors in EGFRT790M gatekeeper-mutant tumors, and they prompt combination treatment strategies involving inhibitors that target signaling downstream of the EGFR.  Cancer Res; 70(3); 868-74.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNjzYq32JnJ7Vg90H21EOLACvtfcHk0lgB2twaIc5mLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12ru7Y%253D&md5=db40fb75245450e8f79fca7e37bad588</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-09-3106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-09-3106%26sid%3Dliteratum%253Aachs%26aulast%3DSos%26aufirst%3DM.%2BL.%26aulast%3DRode%26aufirst%3DH.%2BB.%26aulast%3DHeynck%26aufirst%3DS.%26aulast%3DPeifer%26aufirst%3DM.%26aulast%3DFischer%26aufirst%3DF.%26aulast%3DKl%25C3%25BCter%26aufirst%3DS.%26aulast%3DPawar%26aufirst%3DV.%2BG.%26aulast%3DReuter%26aufirst%3DC.%26aulast%3DHeuckmann%26aufirst%3DJ.%2BM.%26aulast%3DWeiss%26aufirst%3DJ.%26aulast%3DRuddigkeit%26aufirst%3DL.%26aulast%3DRabiller%26aufirst%3DM.%26aulast%3DKoker%26aufirst%3DM.%26aulast%3DSimard%26aufirst%3DJ.%2BR.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DYuza%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DT.%2BH.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DThomas%26aufirst%3DR.%2BK.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DChemogenomic%2520profiling%2520provides%2520insights%2520into%2520the%2520limited%2520activity%2520of%2520irreversible%2520EGFR%2520Inhibitors%2520in%2520tumor%2520cells%2520expressing%2520the%2520T790M%2520EGFR%2520resistance%2520mutation%26jtitle%3DCancer%2520Res.%26date%3D2010%26volume%3D70%26spage%3D868%26epage%3D874%26doi%3D10.1158%2F0008-5472.CAN-09-3106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Kwak, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sordella, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godin-Heymann, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okimoto, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brannigan, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Driscoll, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fidias, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabindran, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGinnis, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wissner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isselbacher, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Settleman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haber, D. A.</span><span> </span><span class="NLM_article-title">Irreversible inhibitors of the EGF receptor may circumvent acquired resistance to gefitinib</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">7665</span><span class="NLM_x">–</span> <span class="NLM_lpage">7670</span><span class="refDoi"> DOI: 10.1073/pnas.0502860102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1073%2Fpnas.0502860102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=15897464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkslOmsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=7665-7670&author=E.+L.+Kwakauthor=R.+Sordellaauthor=D.+W.+Bellauthor=N.+Godin-Heymannauthor=R.+A.+Okimotoauthor=B.+W.+Branniganauthor=P.+L.+Harrisauthor=D.+R.+Driscollauthor=P.+Fidiasauthor=T.+J.+Lynchauthor=S.+K.+Rabindranauthor=J.+P.+McGinnisauthor=A.+Wissnerauthor=S.+V.+Sharmaauthor=K.+J.+Isselbacherauthor=J.+Settlemanauthor=D.+A.+Haber&title=Irreversible+inhibitors+of+the+EGF+receptor+may+circumvent+acquired+resistance+to+gefitinib&doi=10.1073%2Fpnas.0502860102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Irreversible inhibitors of the EGF receptor may circumvent acquired resistance to gefitinib</span></div><div class="casAuthors">Kwak, Eunice L.; Sordella, Raffaella; Bell, Daphne W.; Godin-Heymann, Nadia; Okimoto, Ross A.; Brannigan, Brian W.; Harris, Patricia L.; Driscoll, David R.; Fidias, Panos; Lynch, Thomas J.; Rabindran, Sridhar K.; McGinnis, John P.; Wissner, Allan; Sharma, Sreenath V.; Isselbacher, Kurt J.; Settleman, Jeffrey; Haber, Daniel A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7665-7670</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Nonsmall cell lung cancers (NSCLCs) with activating mutations in the kinase domain of the epidermal growth factor receptor (EGFR) demonstrate dramatic, but transient, responses to the reversible tyrosine kinase inhibitors gefitinib (Iressa) and erlotinib (Tarceva).  Some recurrent tumors have a common secondary mutation in the EGFR kinase domain, T790M, conferring drug resistance, but in other cases the mechanism underlying acquired resistance is unknown.  In studying multiple sites of recurrent NSCLCs, the authors detected T790M in only a small percentage of tumor cells.  To identify addnl. mechanisms of acquired resistance to gefitinib, the authors used NSCLC cells harboring an activating EGFR mutation to generate multiple resistant clones in vitro.  These drug-resistant cells demonstrate continued dependence on EGFR and ERBB2 signaling for their viability and have not acquired secondary EGFR mutations.  However, they display increased internalization of ligand-activated EGFR, consistent with altered receptor trafficking.  Although gefitinib-resistant clones are cross-resistant to related anilinoquinazolines, they demonstrate sensitivity to a class of irreversible inhibitors of EGFR.  These inhibitors also show effective inhibition of signaling by T790M-mutant EGFR and killing of NSCLC cells with the T790M mutation.  Both mechanisms of gefitinib resistance are therefore circumvented by irreversible tyrosine kinase inhibitors.  Our findings suggest that one of these, HKI-272, may prove highly effective in the treatment of EGFR-mutant NSCLCs, including tumors that have become resistant to gefitinib or erlotinib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqW5BSM8yaZe7Vg90H21EOLACvtfcHk0lgB2twaIc5mLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkslOmsr0%253D&md5=f3fb866c3673fd4dff8a3f0a8c9be65b</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0502860102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0502860102%26sid%3Dliteratum%253Aachs%26aulast%3DKwak%26aufirst%3DE.%2BL.%26aulast%3DSordella%26aufirst%3DR.%26aulast%3DBell%26aufirst%3DD.%2BW.%26aulast%3DGodin-Heymann%26aufirst%3DN.%26aulast%3DOkimoto%26aufirst%3DR.%2BA.%26aulast%3DBrannigan%26aufirst%3DB.%2BW.%26aulast%3DHarris%26aufirst%3DP.%2BL.%26aulast%3DDriscoll%26aufirst%3DD.%2BR.%26aulast%3DFidias%26aufirst%3DP.%26aulast%3DLynch%26aufirst%3DT.%2BJ.%26aulast%3DRabindran%26aufirst%3DS.%2BK.%26aulast%3DMcGinnis%26aufirst%3DJ.%2BP.%26aulast%3DWissner%26aufirst%3DA.%26aulast%3DSharma%26aufirst%3DS.%2BV.%26aulast%3DIsselbacher%26aufirst%3DK.%2BJ.%26aulast%3DSettleman%26aufirst%3DJ.%26aulast%3DHaber%26aufirst%3DD.%2BA.%26atitle%3DIrreversible%2520inhibitors%2520of%2520the%2520EGF%2520receptor%2520may%2520circumvent%2520acquired%2520resistance%2520to%2520gefitinib%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D7665%26epage%3D7670%26doi%3D10.1073%2Fpnas.0502860102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Engelman, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zejnullahu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gale, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lifshits, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzales, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimamura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vincent, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naumov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Althaus, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gandhi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heymach, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jänne, P. A.</span><span> </span><span class="NLM_article-title">PF00299804, an irreversible pan-ERBB inhibitor, is effective in lung cancer models with EGFR and ERBB2 mutations that are resistant to gefitinib</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">11924</span><span class="NLM_x">–</span> <span class="NLM_lpage">11932</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-07-1885</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1158%2F0008-5472.CAN-07-1885" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=18089823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVCjs7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=11924-11932&author=J.+A.+Engelmanauthor=K.+Zejnullahuauthor=C.+M.+Galeauthor=E.+Lifshitsauthor=A.+J.+Gonzalesauthor=T.+Shimamuraauthor=F.+Zhaoauthor=P.+W.+Vincentauthor=G.+N.+Naumovauthor=J.+E.+Bradnerauthor=I.+W.+Althausauthor=L.+Gandhiauthor=G.+I.+Shapiroauthor=J.+M.+Nelsonauthor=J.+V.+Heymachauthor=M.+Meyersonauthor=K.+K.+Wongauthor=P.+A.+J%C3%A4nne&title=PF00299804%2C+an+irreversible+pan-ERBB+inhibitor%2C+is+effective+in+lung+cancer+models+with+EGFR+and+ERBB2+mutations+that+are+resistant+to+gefitinib&doi=10.1158%2F0008-5472.CAN-07-1885"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">PF00299804, an Irreversible Pan-ERBB Inhibitor, Is Effective in Lung Cancer Models with EGFR and ERBB2 Mutations that Are Resistant to Gefitinib</span></div><div class="casAuthors">Engelman, Jeffrey A.; Zejnullahu, Kreshnik; Gale, Christopher-Michael; Lifshits, Eugene; Gonzales, Andrea J.; Shimamura, Takeshi; Zhao, Feng; Vincent, Patrick W.; Naumov, George N.; Bradner, James E.; Althaus, Irene W.; Gandhi, Leena; Shapiro, Geoffrey I.; Nelson, James M.; Heymach, John V.; Meyerson, Matthew; Wong, Kwok-Kin; Jaenne, Pasi A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11924-11932</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors gefitinib and erlotinib are effective treatments for a subset of non-small cell lung cancers.  In particular, cancers with specific EGFR-activating mutations seem to be the most sensitive to these agents.  However, despite their initial response, such cancers almost invariably develop resistance.  In 50% of such cancers, a secondary EGFR mutation, T790M, has been identified that renders gefitinib and erlotinib ineffective inhibitors of EGFR kinase activity.  Thus, there is a clin. need to develop novel EGFR inhibitors that can effectively inactivate T790M-contg. EGFR proteins.  In this study, we evaluate the effectiveness of a novel compd., PF00299804, an irreversible pan-ERBB inhibitor.  The results from these studies show that PF00299804 is a potent inhibitor of EGFR-activating mutations as well as the EGFR T790M resistance mutation both in vitro and in vivo.  Addnl., PF00299804 is a highly effective inhibitor of both the wild-type ERBB2 and the gefitinib-resistant oncogenic ERBB2 mutation identified in lung cancers.  These preclin. evaluations support further clin. development of PF00299804 for cancers with mutations and/or amplifications of ERBB family members. [Cancer Res 2007;67(24):11924-32].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq02Cx_x26RirVg90H21EOLACvtfcHk0lhBoCIuJVJ56w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVCjs7nL&md5=c210cf2fb3d1479df5915d317259696f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-07-1885&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-07-1885%26sid%3Dliteratum%253Aachs%26aulast%3DEngelman%26aufirst%3DJ.%2BA.%26aulast%3DZejnullahu%26aufirst%3DK.%26aulast%3DGale%26aufirst%3DC.%2BM.%26aulast%3DLifshits%26aufirst%3DE.%26aulast%3DGonzales%26aufirst%3DA.%2BJ.%26aulast%3DShimamura%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DF.%26aulast%3DVincent%26aufirst%3DP.%2BW.%26aulast%3DNaumov%26aufirst%3DG.%2BN.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DAlthaus%26aufirst%3DI.%2BW.%26aulast%3DGandhi%26aufirst%3DL.%26aulast%3DShapiro%26aufirst%3DG.%2BI.%26aulast%3DNelson%26aufirst%3DJ.%2BM.%26aulast%3DHeymach%26aufirst%3DJ.%2BV.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26atitle%3DPF00299804%252C%2520an%2520irreversible%2520pan-ERBB%2520inhibitor%252C%2520is%2520effective%2520in%2520lung%2520cancer%2520models%2520with%2520EGFR%2520and%2520ERBB2%2520mutations%2520that%2520are%2520resistant%2520to%2520gefitinib%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26spage%3D11924%26epage%3D11932%26doi%3D10.1158%2F0008-5472.CAN-07-1885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ambrogio, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimamura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chirieac, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padera, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baum, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Himmelsbach, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rettig, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solca, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greulich, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span> </span><span class="NLM_article-title">BIBW2992, an irreversible EGFR/HER2 inhibitor highly effective in preclinical lung cancer models</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">4702</span><span class="NLM_x">–</span> <span class="NLM_lpage">4711</span><span class="refDoi"> DOI: 10.1038/onc.2008.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1038%2Fonc.2008.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=18408761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptlKnsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=4702-4711&author=D.+Liauthor=L.+Ambrogioauthor=T.+Shimamuraauthor=S.+Kuboauthor=M.+Takahashiauthor=L.+R.+Chirieacauthor=R.+F.+Paderaauthor=G.+I.+Shapiroauthor=A.+Baumauthor=F.+Himmelsbachauthor=W.+J.+Rettigauthor=M.+Meyersonauthor=F.+Solcaauthor=H.+Greulichauthor=K.+K.+Wong&title=BIBW2992%2C+an+irreversible+EGFR%2FHER2+inhibitor+highly+effective+in+preclinical+lung+cancer+models&doi=10.1038%2Fonc.2008.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">BIBW2992, an irreversible EGFR/HER2 inhibitor highly effective in preclinical lung cancer models</span></div><div class="casAuthors">Li, D.; Ambrogio, L.; Shimamura, T.; Kubo, S.; Takahashi, M.; Chirieac, L. R.; Padera, R. F.; Shapiro, G. I.; Baum, A.; Himmelsbach, F.; Rettig, W. J.; Meyerson, M.; Solca, F.; Greulich, H.; Wong, K-K.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">4702-4711</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Genetic alterations in the kinase domain of the epidermal growth factor receptor (EGFR) in non-small cell lung cancer (NSCLC) patients are assocd. with sensitivity to treatment with small mol. tyrosine kinase inhibitors.  Although first-generation reversible, ATP-competitive inhibitors showed encouraging clin. responses in lung adenocarcinoma tumors harboring such EGFR mutations, almost all patients developed resistance to these inhibitors over time.  Such resistance to first-generation EGFR inhibitors was frequently linked to an acquired T790M point mutation in the kinase domain of EGFR, or upregulation of signaling pathways downstream of HER3.  Overcoming these mechanisms of resistance, as well as primary resistance to reversible EGFR inhibitors driven by a subset of EGFR mutations, will be necessary for development of an effective targeted therapy regimen.  Here, we show that BIBW2992, an anilino-quinazoline designed to irreversibly bind EGFR and HER2, potently suppresses the kinase activity of wild-type and activated EGFR and HER2 mutants, including erlotinib-resistant isoforms.  Consistent with this activity, BIBW2992 suppresses transformation in isogenic cell-based assays, inhibits survival of cancer cell lines and induces tumor regression in xenograft and transgenic lung cancer models, with superior activity over erlotinib.  These findings encourage further testing of BIBW2992 in lung cancer patients harboring EGFR or HER2 oncogenes.  Oncogene (2008) 27, 4702-4711; doi:10.1038/onc.2008.109; published online 14 Apr. 2008.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5PQdKep-Sp7Vg90H21EOLACvtfcHk0lhBoCIuJVJ56w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptlKnsbk%253D&md5=24945893e83b4e3d01c267100fdaf57c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fonc.2008.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2008.109%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.%26aulast%3DAmbrogio%26aufirst%3DL.%26aulast%3DShimamura%26aufirst%3DT.%26aulast%3DKubo%26aufirst%3DS.%26aulast%3DTakahashi%26aufirst%3DM.%26aulast%3DChirieac%26aufirst%3DL.%2BR.%26aulast%3DPadera%26aufirst%3DR.%2BF.%26aulast%3DShapiro%26aufirst%3DG.%2BI.%26aulast%3DBaum%26aufirst%3DA.%26aulast%3DHimmelsbach%26aufirst%3DF.%26aulast%3DRettig%26aufirst%3DW.%2BJ.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DSolca%26aufirst%3DF.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DWong%26aufirst%3DK.%2BK.%26atitle%3DBIBW2992%252C%2520an%2520irreversible%2520EGFR%252FHER2%2520inhibitor%2520highly%2520effective%2520in%2520preclinical%2520lung%2520cancer%2520models%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D4702%26epage%3D4711%26doi%3D10.1038%2Fonc.2008.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pompliano, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meek, T. D.</span><span> </span><span class="NLM_article-title">Drug-target residence time and its implications for lead optimization</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">730</span><span class="NLM_x">–</span> <span class="NLM_lpage">739</span><span class="refDoi"> DOI: 10.1038/nrd2082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1038%2Fnrd2082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=16888652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=730-739&author=R.+A.+Copelandauthor=D.+L.+Pomplianoauthor=T.+D.+Meek&title=Drug-target+residence+time+and+its+implications+for+lead+optimization&doi=10.1038%2Fnrd2082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target residence time and its implications for lead optimization</span></div><div class="casAuthors">Copeland, Robert A.; Pompliano, David L.; Meek, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">730-739</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Much of drug discovery today is predicated on the concept of selective targeting of particular bioactive macromols. by low-mol.-mass drugs.  The binding of drugs to their macromol. targets is therefore seen as paramount for pharmacol. activity.  In vitro assessment of drug-target interactions is classically quantified in terms of binding parameters such as IC50 or Kd.  This article presents an alternative perspective on drug optimization in terms of drug-target binary complex residence time, as quantified by the dissociative half-life of the drug-target binary complex.  We describe the potential advantages of long residence time in terms of duration of pharmacol. effect and target selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry6SJrMlrrTbVg90H21EOLACvtfcHk0lhBoCIuJVJ56w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D&md5=60ede2301584b10ac4e8fa18e1e6d107</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrd2082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2082%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DPompliano%26aufirst%3DD.%2BL.%26aulast%3DMeek%26aufirst%3DT.%2BD.%26atitle%3DDrug-target%2520residence%2520time%2520and%2520its%2520implications%2520for%2520lead%2520optimization%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D730%26epage%3D739%26doi%3D10.1038%2Fnrd2082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Heuckmann, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rauh, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, R. K.</span><span> </span><span class="NLM_article-title">Epidermal growth factor receptor (EGFR) signaling and covalent EGFR inhibition in lung cancer</span> <span class="citation_source-journal">J. Clin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">3417</span><span class="NLM_x">–</span> <span class="NLM_lpage">3420</span><span class="refDoi"> DOI: 10.1200/JCO.2012.43.1825</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1200%2FJCO.2012.43.1825" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=22915655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Glt73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2012&pages=3417-3420&author=J.+M.+Heuckmannauthor=D.+Rauhauthor=R.+K.+Thomas&title=Epidermal+growth+factor+receptor+%28EGFR%29+signaling+and+covalent+EGFR+inhibition+in+lung+cancer&doi=10.1200%2FJCO.2012.43.1825"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal growth factor receptor (EGFR) signaling and covalent EGF inhibition in lung cancer</span></div><div class="casAuthors">Heuckmann, Johannes M.; Rauh, Daniel; Thomas, Roman K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">3417-3420</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">0732-183X</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">A review.  The research of Ramalingam et al. (2012) entitled, 'Randomized phase II study of dacomitinib (PF-00299804), an irreversible pan-human epidermal growth factor receptor inhibitor, vs. erlotinib in patients with advanced non-small-cell lung cancer' is reviewed with commentary and refs.  In their paper, a novel irreversible ERBB family inhibitor, dacomitinib, improved progression-free survival compared with erlotinib in unselected patients with non-small-cell lung cancer without prior epidermal growth factor receptor-directed therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8PW4hGdL2k7Vg90H21EOLACvtfcHk0li6kyPIaZdRZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Glt73O&md5=45192d5ee846d2c25302286226cff733</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1200%2FJCO.2012.43.1825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2012.43.1825%26sid%3Dliteratum%253Aachs%26aulast%3DHeuckmann%26aufirst%3DJ.%2BM.%26aulast%3DRauh%26aufirst%3DD.%26aulast%3DThomas%26aufirst%3DR.%2BK.%26atitle%3DEpidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520signaling%2520and%2520covalent%2520EGFR%2520inhibition%2520in%2520lung%2520cancer%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2012%26volume%3D30%26spage%3D3417%26epage%3D3420%26doi%3D10.1200%2FJCO.2012.43.1825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Singh, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petter, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baillie, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitty, A.</span><span> </span><span class="NLM_article-title">The resurgence of covalent drugs</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">317</span><span class="refDoi"> DOI: 10.1038/nrd3410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1038%2Fnrd3410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=21455239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=307-317&author=J.+Singhauthor=R.+C.+Petterauthor=T.+A.+Baillieauthor=A.+Whitty&title=The+resurgence+of+covalent+drugs&doi=10.1038%2Fnrd3410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The resurgence of covalent drugs</span></div><div class="casAuthors">Singh, Juswinder; Petter, Russell C.; Baillie, Thomas A.; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent drugs haveproved to be successful therapies for various indications, but largely owing to safety concerns, they are rarely considered when initiating a target-directed drug discovery project.  There is a need to reassess this important class of drugs, and to reconcile the discordance between the historic success of covalent drugs and the reluctance of most drug discovery teams to include them in their armamentarium.  This Review surveys the prevalence and pharmacol. advantages of covalent drugs, discusses how potential risks and challenges may be addressed through innovative design, and presents the broad opportunities provided by targeted covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreDCEmORV_NbVg90H21EOLACvtfcHk0li6kyPIaZdRZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D&md5=2190289081e151416c097be4a5b04460</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrd3410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3410%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DPetter%26aufirst%3DR.%2BC.%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DThe%2520resurgence%2520of%2520covalent%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D307%26epage%3D317%26doi%3D10.1038%2Fnrd3410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Katakami, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atagi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hida, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horai, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inoue, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichinose, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koboyashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiura, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishio, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ebisawa, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shahidi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamamoto, N.</span><span> </span><span class="NLM_article-title">LUX-Lung 4: a phase II trial of afatinib in patients with advanced non-small-cell lung cancer who progressed during prior treatment with erlotinib, gefitinib, or both</span> <span class="citation_source-journal">J. Clin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">3335</span><span class="NLM_x">–</span> <span class="NLM_lpage">3341</span><span class="refDoi"> DOI: 10.1200/JCO.2012.45.0981</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1200%2FJCO.2012.45.0981" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=23816963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1KrsrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=3335-3341&author=N.+Katakamiauthor=S.+Atagiauthor=K.+Gotoauthor=T.+Hidaauthor=T.+Horaiauthor=A.+Inoueauthor=Y.+Ichinoseauthor=K.+Koboyashiauthor=K.+Takedaauthor=K.+Kiuraauthor=K.+Nishioauthor=Y.+Sekiauthor=R.+Ebisawaauthor=M.+Shahidiauthor=N.+Yamamoto&title=LUX-Lung+4%3A+a+phase+II+trial+of+afatinib+in+patients+with+advanced+non-small-cell+lung+cancer+who+progressed+during+prior+treatment+with+erlotinib%2C+gefitinib%2C+or+both&doi=10.1200%2FJCO.2012.45.0981"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">LUX-lung 4: a phase II trial of afatinib in patients with advanced non-small-cell lung cancer who progressed during prior treatment with erlotinib, gefitinib, or both</span></div><div class="casAuthors">Katakami, Noboyuki; Atagi, Shinji; Goto, Koichi; Hida, Toyoaki; Horai, Takeshi; Inoue, Akira; Ichinose, Yukito; Koboyashi, Kunihiko; Takeda, Koji; Kiura, Katsuyuki; Nishio, Kazuto; Seki, Yoko; Ebisawa, Ryuichi; Shahidi, Mehdi; Yamamoto, Nobuyuki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">3335-3341</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">0732-183X</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">Purpose New mol. targeted agents are needed for patients with non-small-cell lung cancer (NSCLC) who progress while receiving erlotinib, gefitinib, or both.  Afatinib, an oral irreversible ErbB family blocker, has preclin. activity in epidermal growth factor receptor (EGFR [ErbB1]) mutant models with EGFR-activating mutations, including T790M.  Patients and Methods This was a Japanese single-arm phase II trial conducted in patients with stage IIIB to IV pulmonary adenocarcinoma who progressed after ≥ 12 wk of prior erlotinib and/or gefitinib.  Patients received afatinib 50 mg per day.  The primary end point was objective response rate (complete response or partial response) by independent review.  Secondary end points included progression-free survival (PFS), overall survival (OS), and safety.  Results Of 62 treated patients, 45 (72.6%) were EGFR mutation pos. in their primary tumor according to local and/or central lab. analyses.  Fifty-one patients (82.3%) fulfilled the criteria of acquired resistance to erlotinib and/or gefitinib.  Of 61 evaluable patients, five (8.2%; 95% CI, 2.7% to 18.1 %) had a confirmed objective response rate (partial response).  Median PFS was 4.4 mo (95% CI, 2.8 to 4.6 mo), and median OS was 19.0 mo (95% CI, 14.9 mo to not achieved).  Two patients had acquired T790M mutations: L858R + T790M, and deletion in exon 19 + T790M; they had stable disease for 9 mo and 1 mo, resp.  The most common afatinib-related adverse events (AEs) were diarrhea (100%) and rash/acne (91.9%).  Treatment-related AEs leading to afatinib discontinuation were experienced by 18 patients (29%), of whom four also had progressive disease.  Conclusion Afatinib demonstrated modest but noteworthy efficacy in patients with NSCLC who had received third- or fourth-line treatment and who progressed while receiving erlotinib and/or gefitinib, including those with acquired resistance to erlotinib, gefitinib, or both.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpM0F6fZnWKTbVg90H21EOLACvtfcHk0li6kyPIaZdRZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1KrsrzK&md5=1666aeb642dbc953dd9bcc818bab9098</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1200%2FJCO.2012.45.0981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2012.45.0981%26sid%3Dliteratum%253Aachs%26aulast%3DKatakami%26aufirst%3DN.%26aulast%3DAtagi%26aufirst%3DS.%26aulast%3DGoto%26aufirst%3DK.%26aulast%3DHida%26aufirst%3DT.%26aulast%3DHorai%26aufirst%3DT.%26aulast%3DInoue%26aufirst%3DA.%26aulast%3DIchinose%26aufirst%3DY.%26aulast%3DKoboyashi%26aufirst%3DK.%26aulast%3DTakeda%26aufirst%3DK.%26aulast%3DKiura%26aufirst%3DK.%26aulast%3DNishio%26aufirst%3DK.%26aulast%3DSeki%26aufirst%3DY.%26aulast%3DEbisawa%26aufirst%3DR.%26aulast%3DShahidi%26aufirst%3DM.%26aulast%3DYamamoto%26aufirst%3DN.%26atitle%3DLUX-Lung%25204%253A%2520a%2520phase%2520II%2520trial%2520of%2520afatinib%2520in%2520patients%2520with%2520advanced%2520non-small-cell%2520lung%2520cancer%2520who%2520progressed%2520during%2520prior%2520treatment%2520with%2520erlotinib%252C%2520gefitinib%252C%2520or%2520both%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2013%26volume%3D31%26spage%3D3335%26epage%3D3341%26doi%3D10.1200%2FJCO.2012.45.0981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Miller, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirsh, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cadranel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heo, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crino, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chao, T. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shahidi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cong, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorence, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J. C.</span><span> </span><span class="NLM_article-title">Afatinib versus placebo for patients with advanced, metastatic non-small-cell lung cancer after failure of erlotinib, gefitinib, or both, and one or two lines of chemotherapy (LUX-Lung 1): a phase 2b/3 randomised trial</span> <span class="citation_source-journal">Lancet Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">528</span><span class="NLM_x">–</span> <span class="NLM_lpage">538</span><span class="refDoi"> DOI: 10.1016/S1470-2045(12)70087-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1016%2FS1470-2045%2812%2970087-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=22452896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmslyhsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=528-538&author=V.+A.+Millerauthor=V.+Hirshauthor=J.+Cadranelauthor=Y.+M.+Chenauthor=K.+Parkauthor=S.+W.+Kimauthor=C.+Zhouauthor=W.+C.+Suauthor=M.+Wangauthor=Y.+Sunauthor=D.+S.+Heoauthor=L.+Crinoauthor=E.+H.+Tanauthor=T.+Y.+Chaoauthor=M.+Shahidiauthor=X.+J.+Congauthor=R.+M.+Lorenceauthor=J.+C.+Yang&title=Afatinib+versus+placebo+for+patients+with+advanced%2C+metastatic+non-small-cell+lung+cancer+after+failure+of+erlotinib%2C+gefitinib%2C+or+both%2C+and+one+or+two+lines+of+chemotherapy+%28LUX-Lung+1%29%3A+a+phase+2b%2F3+randomised+trial&doi=10.1016%2FS1470-2045%2812%2970087-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Afatinib versus placebo for patients with advanced, metastatic non-small-cell lung cancer after failure of erlotinib, gefitinib, or both, and one or two lines of chemotherapy (LUX-Lung 1): a phase 2b/3 randomised trial</span></div><div class="casAuthors">Miller, Vincent A.; Hirsh, Vera; Cadranel, Jacques; Chen, Yuh-Min; Park, Keunchil; Kim, Sang-We; Zhou, Caicun; Su, Wu-Chou; Wang, Mengzhao; Sun, Yan; Heo, Dae Seog; Crino, Lucio; Tan, Eng-Huat; Chao, Tsu-Yi; Shahidi, Mehdi; Cong, Xiuyu Julie; Lorence, Robert M.; Yang, James Chih-Hsin</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">528-538</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Summary: Background: Afatinib, an irreversible ErbB-family blocker, has shown preclin. activity when tested in EGFR mutant models with mutations that confer resistance to EGFR tyrosine-kinase inhibitors.  We aimed to assess its efficacy in patients with advanced lung adenocarcinoma with previous treatment failure on EGFR tyrosine-kinase inhibitors.  Methods: In this phase 2b/3 trial, we enrolled patients with stage IIIB or IV adenocarcinoma and an Eastern Cooperative Oncol. Group performance (ECOG) performance score of 0-2 who had received one or two previous chemotherapy regimens and had disease progression after at least 12 wk of treatment with erlotinib or gefitinib.  We used a computer-generated sequence to randomly allocate patients (2:1) to either afatinib (50 mg per day) or placebo; all patients received best supportive care.  Randomisation was done in blocks of three and was stratified by sex and baseline ECOG performance status (0-1 vs 2).  Investigators, patients, and the trial sponsor were masked to treatment assignment.  The primary endpoint was overall survival (from date of randomisation to death), analyzed on an intention-to-treat basis.  This study is registered with ClinicalTrials.gov, no. NCT00656136.  Findings: Between May 26, 2008, and Sept 21, 2009, we identified 697 patients, 585 of whom were randomly allocated to treatment (390 to afatinib, 195 to placebo).  Median overall survival was 10·8 mo (95% CI 10·0-12·0) in the afatinib group and 12·0 mo (10·2-14·3) in the placebo group (hazard ratio 1·08, 95% CI 0·86-1·35; p=0·74).  Median progression-free survival was longer in the afatinib group (3·3 mo, 95% CI 2·79-4·40) than it was in the placebo group (1·1 mo, 0·95-1·68; hazard ratio 0·38, 95% CI 0·31-0·48; p<0·0001).  No complete responses to treatment were noted; 29 (7%) patients had a partial response in the afatinib group, as did one patient in the placebo group.  Subsequent cancer treatment was given to 257 (68%) patients in the afatinib group and 153 (79%) patients in the placebo group.  The most common adverse events in the afatinib group were diarrhoea (339 [87%] of 390 patients; 66 [17%] were grade 3) and rash or acne (305 [78%] patients; 56 [14%] were grade 3).  These events occurred less often in the placebo group (18 [9%] of 195 patients had diarrhoea; 31 [16%] had rash or acne), all being grade 1 or 2.  Drug-related serious adverse events occurred in 39 (10%) patients in the afatinib group and one (<1%) patient in the placebo group.  We recorded two possibly treatment-related deaths in the afatinib group.  Interpretation: Although we recorded no benefit in terms of overall survival with afatinib (which might have been affected by cancer treatments given after progression in both groups), our findings for progression-free survival and response to treatment suggest that afatinib could be of some benefit to patients with advanced lung adenocarcinoma who have failed at least 12 wk of previous EGFR tyrosine-kinase inhibitor treatment.  Funding: Boehringer Ingelheim Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDrR4FNSnKJbVg90H21EOLACvtfcHk0lhxJAjIAJGnQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmslyhsbg%253D&md5=6ddd7f552e8ab1ebb43fb699e27894f5</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2812%2970087-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252812%252970087-6%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DV.%2BA.%26aulast%3DHirsh%26aufirst%3DV.%26aulast%3DCadranel%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%2BM.%26aulast%3DPark%26aufirst%3DK.%26aulast%3DKim%26aufirst%3DS.%2BW.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DSu%26aufirst%3DW.%2BC.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DHeo%26aufirst%3DD.%2BS.%26aulast%3DCrino%26aufirst%3DL.%26aulast%3DTan%26aufirst%3DE.%2BH.%26aulast%3DChao%26aufirst%3DT.%2BY.%26aulast%3DShahidi%26aufirst%3DM.%26aulast%3DCong%26aufirst%3DX.%2BJ.%26aulast%3DLorence%26aufirst%3DR.%2BM.%26aulast%3DYang%26aufirst%3DJ.%2BC.%26atitle%3DAfatinib%2520versus%2520placebo%2520for%2520patients%2520with%2520advanced%252C%2520metastatic%2520non-small-cell%2520lung%2520cancer%2520after%2520failure%2520of%2520erlotinib%252C%2520gefitinib%252C%2520or%2520both%252C%2520and%2520one%2520or%2520two%2520lines%2520of%2520chemotherapy%2520%2528LUX-Lung%25201%2529%253A%2520a%2520phase%25202b%252F3%2520randomised%2520trial%26jtitle%3DLancet%2520Oncol.%26date%3D2012%26volume%3D13%26spage%3D528%26epage%3D538%26doi%3D10.1016%2FS1470-2045%2812%2970087-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Sequist, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Besse, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gitlitz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eaton, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zacharchuk, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freyman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ananthakrishnan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soria, J. C.</span><span> </span><span class="NLM_article-title">Neratinib, an irreversible pan-ErbB receptor tyrosine kinase inhibitor: results of a phase II trial in patients with advanced non-small-cell lung cancer</span> <span class="citation_source-journal">J. Clin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">3076</span><span class="NLM_x">–</span> <span class="NLM_lpage">3083</span><span class="refDoi"> DOI: 10.1200/JCO.2009.27.9414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1200%2FJCO.2009.27.9414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=20479403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptlaju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2010&pages=3076-3083&author=L.+V.+Sequistauthor=B.+Besseauthor=T.+J.+Lynchauthor=V.+A.+Millerauthor=K.+K.+Wongauthor=B.+Gitlitzauthor=K.+Eatonauthor=C.+Zacharchukauthor=A.+Freymanauthor=C.+Powellauthor=R.+Ananthakrishnanauthor=S.+Quinnauthor=J.+C.+Soria&title=Neratinib%2C+an+irreversible+pan-ErbB+receptor+tyrosine+kinase+inhibitor%3A+results+of+a+phase+II+trial+in+patients+with+advanced+non-small-cell+lung+cancer&doi=10.1200%2FJCO.2009.27.9414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Neratinib, an irreversible Pan-ErbB, receptor tyrosine kinase inhibitor: results of a phase II trial in patients with advanced non-small-cell lung cancer</span></div><div class="casAuthors">Sequist, Lecia V.; Besse, Benjamin; Lynch, Thomas J.; Miller, Vincent A.; Wong, Kwok K.; Gitlitz, Barbara; Eaton, Keith; Zacharchuk, Charles; Freyman, Amy; Powell, Christine; Ananthakrishnan, Revathi; Quinn, Susan; Soria, Jean-Charles</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3076-3083</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">0732-183X</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors (TKIs) have had a significant impact on non-small-cell lung cancer (NSCLC) outcomes, particularly for patients with EGFR mutations.  Resistance emerges after 9 to 12 mo, primarily mediated by the T790M resistance mutation.  We studied neratinib, an irreversible pan-ErbB TKI that may overcome T790M.  Patients with advanced NSCLC underwent EGFR sequencing of available tumor tissue at enrollment.  Those with ≥ 12 wk of prior TKI therapy were placed in arm A if they were EGFR mutation pos. or arm B if they were wild-type.  Arm C included TKI-naive patients with adenocarcinoma and light smoking histories (≤ 20 pack-years).  All patients received daily oral neratinib, initially at 320 mg but subsequently reduced to 240 mg because of excessive diarrhea.  The primary end point was objective response rate (RR).  167 Patients were treated: 91 in arm A, 48 in arm B, and 28 in arm C.  Diarrhea was the most common toxicity; grade 3 incidence was 50% at 320 mg but improved to 25% after dose redn.  The RR was 3% in arm A and zero in arms B and C.  No patients with known T790M responded.  Notably, three of four patients with an exon 18 G719X EGFR mutation had a partial response and the fourth had stable disease lasting 40 wk.  Neratinib had low activity in patients with prior benefit from TKIs and in TKI-naive patients, potentially because of insufficient bioavailability from diarrhea-imposed dose limitation.  Responses were seen in patients with the rare G719X EGFR mutation, highlighting the importance of obtaining comprehensive genetic information on trials of targeted agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-Z3gsvbWbOrVg90H21EOLACvtfcHk0lhxJAjIAJGnQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptlaju7c%253D&md5=460e088e58919bbfcf8c153772f157b0</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1200%2FJCO.2009.27.9414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2009.27.9414%26sid%3Dliteratum%253Aachs%26aulast%3DSequist%26aufirst%3DL.%2BV.%26aulast%3DBesse%26aufirst%3DB.%26aulast%3DLynch%26aufirst%3DT.%2BJ.%26aulast%3DMiller%26aufirst%3DV.%2BA.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DGitlitz%26aufirst%3DB.%26aulast%3DEaton%26aufirst%3DK.%26aulast%3DZacharchuk%26aufirst%3DC.%26aulast%3DFreyman%26aufirst%3DA.%26aulast%3DPowell%26aufirst%3DC.%26aulast%3DAnanthakrishnan%26aufirst%3DR.%26aulast%3DQuinn%26aufirst%3DS.%26aulast%3DSoria%26aufirst%3DJ.%2BC.%26atitle%3DNeratinib%252C%2520an%2520irreversible%2520pan-ErbB%2520receptor%2520tyrosine%2520kinase%2520inhibitor%253A%2520results%2520of%2520a%2520phase%2520II%2520trial%2520in%2520patients%2520with%2520advanced%2520non-small-cell%2520lung%2520cancer%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2010%26volume%3D28%26spage%3D3076%26epage%3D3083%26doi%3D10.1200%2FJCO.2009.27.9414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Zhou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ercan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yun, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capelletti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortot, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chirieac, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iacob, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padera, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engen, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eck, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jänne, P. A.</span><span> </span><span class="NLM_article-title">Novel mutant-selective EGFR kinase inhibitors against EGFR T790M</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">462</span><span class="NLM_x">, </span> <span class="NLM_fpage">1070</span><span class="NLM_x">–</span> <span class="NLM_lpage">1074</span><span class="refDoi"> DOI: 10.1038/nature08622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1038%2Fnature08622" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=20033049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhs1SktbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=1070-1074&author=W.+Zhouauthor=D.+Ercanauthor=L.+Chenauthor=C.+H.+Yunauthor=D.+Liauthor=M.+Capellettiauthor=A.+B.+Cortotauthor=L.+Chirieacauthor=R.+E.+Iacobauthor=R.+Paderaauthor=J.+R.+Engenauthor=K.+K.+Wongauthor=M.+J.+Eckauthor=N.+S.+Grayauthor=P.+A.+J%C3%A4nne&title=Novel+mutant-selective+EGFR+kinase+inhibitors+against+EGFR+T790M&doi=10.1038%2Fnature08622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Novel mutant-selective EGFR kinase inhibitors against EGFR T790M</span></div><div class="casAuthors">Zhou, Wenjun; Ercan, Dalia; Chen, Liang; Yun, Cai-Hong; Li, Danan; Capelletti, Marzia; Cortot, Alexis B.; Chirieac, Lucian; Iacob, Roxana E.; Padera, Robert; Engen, John R.; Wong, Kwok-Kin; Eck, Michael J.; Gray, Nathanael S.; Janne, Pasi A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">462</span>
        (<span class="NLM_cas:issue">7276</span>),
    <span class="NLM_cas:pages">1070-1074</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. efficacy of epidermal growth factor receptor (EGFR) kinase inhibitors in EGFR-mutant non-small-cell lung cancer (NSCLC) is limited by the development of drug-resistance mutations, including the gatekeeper T790M mutation.  Strategies targeting EGFR T790M with irreversible inhibitors have had limited success and are assocd. with toxicity due to concurrent inhibition of wild-type EGFR.  All current EGFR inhibitors possess a structurally related quinazoline-based core scaffold and were identified as ATP-competitive inhibitors of wild-type EGFR.  Here we identify a covalent pyrimidine EGFR inhibitor by screening an irreversible kinase inhibitor library specifically against EGFR T790M.  These agents are 30- to 100-fold more potent against EGFR T790M, and up to 100-fold less potent against wild-type EGFR, than quinazoline-based EGFR inhibitors in vitro.  They are also effective in murine models of lung cancer driven by EGFR T790M.  Co-crystn. studies reveal a structural basis for the increased potency and mutant selectivity of these agents.  These mutant-selective irreversible EGFR kinase inhibitors may be clin. more effective and better tolerated than quinazoline-based inhibitors.  Our findings demonstrate that functional pharmacol. screens against clin. important mutant kinases represent a powerful strategy to identify new classes of mutant-selective kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcksZlIe-1bbVg90H21EOLACvtfcHk0ljWjkzWqnJWDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhs1SktbvI&md5=9b5d41a2be401c67a4b9f04a41cf1725</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnature08622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08622%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DYun%26aufirst%3DC.%2BH.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DCapelletti%26aufirst%3DM.%26aulast%3DCortot%26aufirst%3DA.%2BB.%26aulast%3DChirieac%26aufirst%3DL.%26aulast%3DIacob%26aufirst%3DR.%2BE.%26aulast%3DPadera%26aufirst%3DR.%26aulast%3DEngen%26aufirst%3DJ.%2BR.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26atitle%3DNovel%2520mutant-selective%2520EGFR%2520kinase%2520inhibitors%2520against%2520EGFR%2520T790M%26jtitle%3DNature%26date%3D2009%26volume%3D462%26spage%3D1070%26epage%3D1074%26doi%3D10.1038%2Fnature08622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Walter, A. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sjin, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haringsma, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubrovskiy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labenski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheets, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St. Martin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karp, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Kalken, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaturvedi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nacht, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petter, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westlin, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaw-Tsai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raponi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Dyke, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Etter, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weaver, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmons, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harding, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, A.</span><span> </span><span class="NLM_article-title">Discovery of a mutant-selective covalent inhibitor of EGFR that overcomes T790M-mediated resistance in NSCLC</span> <span class="citation_source-journal">Cancer Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1404</span><span class="NLM_x">–</span> <span class="NLM_lpage">1415</span><span class="refDoi"> DOI: 10.1158/2159-8290.CD-13-0314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1158%2F2159-8290.CD-13-0314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=24065731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvV2rtLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2013&pages=1404-1415&author=A.+O.+Walterauthor=R.+T.+Sjinauthor=H.+J.+Haringsmaauthor=K.+Ohashiauthor=J.+Sunauthor=K.+Leeauthor=A.+Dubrovskiyauthor=M.+Labenskiauthor=Z.+Zhuauthor=Z.+Wangauthor=M.+Sheetsauthor=T.+St.+Martinauthor=R.+Karpauthor=D.+van+Kalkenauthor=P.+Chaturvediauthor=D.+Niuauthor=M.+Nachtauthor=R.+C.+Petterauthor=W.+Westlinauthor=K.+Linauthor=S.+Jaw-Tsaiauthor=M.+Raponiauthor=T.+Van+Dykeauthor=J.+Etterauthor=Z.+Weaverauthor=W.+Paoauthor=J.+Singhauthor=A.+D.+Simmonsauthor=T.+C.+Hardingauthor=A.+Allen&title=Discovery+of+a+mutant-selective+covalent+inhibitor+of+EGFR+that+overcomes+T790M-mediated+resistance+in+NSCLC&doi=10.1158%2F2159-8290.CD-13-0314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Mutant-Selective Covalent Inhibitor of EGFR that Overcomes T790M-Mediated Resistance in NSCLC</span></div><div class="casAuthors">Walter, Annette O.; Sjin, Robert Tjin Tham; Haringsma, Henry J.; Ohashi, Kadoaki; Sun, Jing; Lee, Kwangho; Dubrovskiy, Aleksandr; Labenski, Matthew; Zhu, Zhendong; Wang, Zhigang; Sheets, Michael; St. Martin, Thia; Karp, Russell; van Kalken, Dan; Chaturvedi, Prasoon; Niu, Deqiang; Nacht, Mariana; Petter, Russell C.; Westlin, William; Lin, Kevin; Jaw-Tsai, Sarah; Raponi, Mitch; Van Dyke, Terry; Etter, Jeff; Weaver, Zoe; Pao, William; Singh, Juswinder; Simmons, Andrew D.; Harding, Thomas C.; Allen, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1404-1415</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Patients with non-small cell lung cancer (NSCLC) with activating EGF receptor (EGFR) mutations initially respond to first-generation reversible EGFR tyrosine kinase inhibitors.  However, clin. efficacy is limited by acquired resistance, frequently driven by the EGFRT790M mutation.  CO-1686 is a novel, irreversible, and orally delivered kinase inhibitor that specifically targets the mutant forms of EGFR, including T790M, while exhibiting minimal activity toward the wild-type (WT) receptor.  Oral administration of CO-1686 as single agent induces tumor regression in EGFR-mutated NSCLC tumor xenograft and transgenic models.  Minimal activity of CO-1686 against the WT EGFR receptor was obsd.  In NSCLC cells with acquired resistance to CO-1686 in vitro, there was no evidence of addnl. mutations or amplification of the EGFR gene, but resistant cells exhibited signs of epithelial-mesenchymal transition and demonstrated increased sensitivity to AKT inhibitors.  These results suggest that CO-1686 may offer a novel therapeutic option for patients with mutant EGFR NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR0CDWQZ5wn7Vg90H21EOLACvtfcHk0ljWjkzWqnJWDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvV2rtLvN&md5=3d111f4bf958f2fda6890fbc0e1d99f5</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-13-0314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-13-0314%26sid%3Dliteratum%253Aachs%26aulast%3DWalter%26aufirst%3DA.%2BO.%26aulast%3DSjin%26aufirst%3DR.%2BT.%26aulast%3DHaringsma%26aufirst%3DH.%2BJ.%26aulast%3DOhashi%26aufirst%3DK.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DDubrovskiy%26aufirst%3DA.%26aulast%3DLabenski%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DSheets%26aufirst%3DM.%26aulast%3DSt.%2BMartin%26aufirst%3DT.%26aulast%3DKarp%26aufirst%3DR.%26aulast%3Dvan%2BKalken%26aufirst%3DD.%26aulast%3DChaturvedi%26aufirst%3DP.%26aulast%3DNiu%26aufirst%3DD.%26aulast%3DNacht%26aufirst%3DM.%26aulast%3DPetter%26aufirst%3DR.%2BC.%26aulast%3DWestlin%26aufirst%3DW.%26aulast%3DLin%26aufirst%3DK.%26aulast%3DJaw-Tsai%26aufirst%3DS.%26aulast%3DRaponi%26aufirst%3DM.%26aulast%3DVan%2BDyke%26aufirst%3DT.%26aulast%3DEtter%26aufirst%3DJ.%26aulast%3DWeaver%26aufirst%3DZ.%26aulast%3DPao%26aufirst%3DW.%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DSimmons%26aufirst%3DA.%2BD.%26aulast%3DHarding%26aufirst%3DT.%2BC.%26aulast%3DAllen%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520a%2520mutant-selective%2520covalent%2520inhibitor%2520of%2520EGFR%2520that%2520overcomes%2520T790M-mediated%2520resistance%2520in%2520NSCLC%26jtitle%3DCancer%2520Discovery%26date%3D2013%26volume%3D3%26spage%3D1404%26epage%3D1415%26doi%3D10.1158%2F2159-8290.CD-13-0314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Sequist, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soria, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldman, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wakelee, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gadgeel, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varga, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Papadimitrakopoulou, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oxnard, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dziadziuszko, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aisner, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doebele, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galasso, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garon, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heist, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Logan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neal, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendenhall, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nichols, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piotrowska, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wozniak, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raponi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlovich, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaw-Tsai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isaacson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Despain, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matheny, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rolfe, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camidge, D. R.</span><span> </span><span class="NLM_article-title">Rociletinib in EGFR-mutated non-small-cell lung cancer</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">372</span><span class="NLM_x">, </span> <span class="NLM_fpage">1700</span><span class="NLM_x">–</span> <span class="NLM_lpage">1709</span><span class="refDoi"> DOI: 10.1056/NEJMoa1413654</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1056%2FNEJMoa1413654" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=25923550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A280%3ADC%252BC2Mjpt1yltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=372&publication_year=2015&pages=1700-1709&author=L.+V.+Sequistauthor=J.+C.+Soriaauthor=J.+W.+Goldmanauthor=H.+A.+Wakeleeauthor=S.+M.+Gadgeelauthor=A.+Vargaauthor=V.+Papadimitrakopoulouauthor=B.+J.+Solomonauthor=G.+R.+Oxnardauthor=R.+Dziadziuszkoauthor=D.+L.+Aisnerauthor=R.+C.+Doebeleauthor=C.+Galassoauthor=E.+B.+Garonauthor=R.+S.+Heistauthor=J.+Loganauthor=J.+W.+Nealauthor=M.+A.+Mendenhallauthor=S.+Nicholsauthor=Z.+Piotrowskaauthor=A.+J.+Wozniakauthor=M.+Raponiauthor=C.+A.+Karlovichauthor=S.+Jaw-Tsaiauthor=J.+Isaacsonauthor=D.+Despainauthor=S.+L.+Mathenyauthor=L.+Rolfeauthor=A.+R.+Allenauthor=D.+R.+Camidge&title=Rociletinib+in+EGFR-mutated+non-small-cell+lung+cancer&doi=10.1056%2FNEJMoa1413654"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Rociletinib in EGFR-mutated non-small-cell lung cancer</span></div><div class="casAuthors">Sequist Lecia V; Soria Jean-Charles; Goldman Jonathan W; Wakelee Heather A; Gadgeel Shirish M; Varga Andrea; Papadimitrakopoulou Vassiliki; Solomon Benjamin J; Oxnard Geoffrey R; Dziadziuszko Rafal; Aisner Dara L; Doebele Robert C; Galasso Cathy; Garon Edward B; Heist Rebecca S; Logan Jennifer; Neal Joel W; Mendenhall Melody A; Nichols Suzanne; Piotrowska Zofia; Wozniak Antoinette J; Raponi Mitch; Karlovich Chris A; Jaw-Tsai Sarah; Isaacson Jeffrey; Despain Darrin; Matheny Shannon L; Rolfe Lindsey; Allen Andrew R; Camidge D Ross</div><div class="citationInfo"><span class="NLM_cas:title">The New England journal of medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">372</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">1700-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Non-small-cell lung cancer (NSCLC) with a mutation in the gene encoding epidermal growth factor receptor (EGFR) is sensitive to approved EGFR inhibitors, but resistance develops, mediated by the T790M EGFR mutation in most cases.  Rociletinib (CO-1686) is an EGFR inhibitor active in preclinical models of EGFR-mutated NSCLC with or without T790M.  METHODS:  In this phase 1-2 study, we administered rociletinib to patients with EGFR-mutated NSCLC who had disease progression during previous treatment with an existing EGFR inhibitor.  In the expansion (phase 2) part of the study, patients with T790M-positive disease received rociletinib at a dose of 500 mg twice daily, 625 mg twice daily, or 750 mg twice daily.  Key objectives were assessment of safety, side-effect profile, pharmacokinetics, and preliminary antitumor activity of rociletinib.  Tumor biopsies to identify T790M were performed during screening.  Treatment was administered in continuous 21-day cycles.  RESULTS:  A total of 130 patients were enrolled.  The first 57 patients to be enrolled received the free-base form of rociletinib (150 mg once daily to 900 mg twice daily).  The remaining patients received the hydrogen bromide salt (HBr) form (500 mg twice daily to 1000 mg twice daily).  A maximum tolerated dose (the highest dose associated with a rate of dose-limiting toxic effects of less than 33%) was not identified.  The only common dose-limiting adverse event was hyperglycemia.  In an efficacy analysis that included patients who received free-base rociletinib at a dose of 900 mg twice daily or the HBr form at any dose, the objective response rate among the 46 patients with T790M-positive disease who could be evaluated was 59% (95% confidence interval [CI], 45 to 73), and the rate among the 17 patients with T790M-negative disease who could be evaluated was 29% (95% CI, 8 to 51).  CONCLUSIONS:  Rociletinib was active in patients with EGFR-mutated NSCLC associated with the T790M resistance mutation. (Funded by Clovis Oncology; ClinicalTrials.gov number, NCT01526928.).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQME11ovAjH3ZPq5LRczm8ffW6udTcc2eazQFMFwxEFcrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mjpt1yltQ%253D%253D&md5=5a5d21d36c081a5ef0f9e650d5cc84b9</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1413654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1413654%26sid%3Dliteratum%253Aachs%26aulast%3DSequist%26aufirst%3DL.%2BV.%26aulast%3DSoria%26aufirst%3DJ.%2BC.%26aulast%3DGoldman%26aufirst%3DJ.%2BW.%26aulast%3DWakelee%26aufirst%3DH.%2BA.%26aulast%3DGadgeel%26aufirst%3DS.%2BM.%26aulast%3DVarga%26aufirst%3DA.%26aulast%3DPapadimitrakopoulou%26aufirst%3DV.%26aulast%3DSolomon%26aufirst%3DB.%2BJ.%26aulast%3DOxnard%26aufirst%3DG.%2BR.%26aulast%3DDziadziuszko%26aufirst%3DR.%26aulast%3DAisner%26aufirst%3DD.%2BL.%26aulast%3DDoebele%26aufirst%3DR.%2BC.%26aulast%3DGalasso%26aufirst%3DC.%26aulast%3DGaron%26aufirst%3DE.%2BB.%26aulast%3DHeist%26aufirst%3DR.%2BS.%26aulast%3DLogan%26aufirst%3DJ.%26aulast%3DNeal%26aufirst%3DJ.%2BW.%26aulast%3DMendenhall%26aufirst%3DM.%2BA.%26aulast%3DNichols%26aufirst%3DS.%26aulast%3DPiotrowska%26aufirst%3DZ.%26aulast%3DWozniak%26aufirst%3DA.%2BJ.%26aulast%3DRaponi%26aufirst%3DM.%26aulast%3DKarlovich%26aufirst%3DC.%2BA.%26aulast%3DJaw-Tsai%26aufirst%3DS.%26aulast%3DIsaacson%26aufirst%3DJ.%26aulast%3DDespain%26aufirst%3DD.%26aulast%3DMatheny%26aufirst%3DS.%2BL.%26aulast%3DRolfe%26aufirst%3DL.%26aulast%3DAllen%26aufirst%3DA.%2BR.%26aulast%3DCamidge%26aufirst%3DD.%2BR.%26atitle%3DRociletinib%2520in%2520EGFR-mutated%2520non-small-cell%2520lung%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2015%26volume%3D372%26spage%3D1700%26epage%3D1709%26doi%3D10.1056%2FNEJMoa1413654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Cross, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashton, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghiorghiu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberlein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nebhan, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spitzler, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orme, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finlay, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mellor, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brewer, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichihara, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Kadhimi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowlinson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klinowska, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richmond, G. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cantarini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ranson, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pao, W.</span><span> </span><span class="NLM_article-title">AZD9291, an irreversible EGFR TKI, overcomes T790M-mediated resistance to EGFR inhibitors in lung cancer</span> <span class="citation_source-journal">Cancer Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1046</span><span class="NLM_x">–</span> <span class="NLM_lpage">1061</span><span class="refDoi"> DOI: 10.1158/2159-8290.CD-14-0337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1158%2F2159-8290.CD-14-0337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=24893891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVykur%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=1046-1061&author=D.+A.+Crossauthor=S.+E.+Ashtonauthor=S.+Ghiorghiuauthor=C.+Eberleinauthor=C.+A.+Nebhanauthor=P.+J.+Spitzlerauthor=J.+P.+Ormeauthor=M.+R.+Finlayauthor=R.+A.+Wardauthor=M.+J.+Mellorauthor=G.+Hughesauthor=A.+Rahiauthor=V.+N.+Jacobsauthor=M.+R.+Brewerauthor=E.+Ichiharaauthor=J.+Sunauthor=H.+Jinauthor=P.+Ballardauthor=K.+Al-Kadhimiauthor=R.+Rowlinsonauthor=T.+Klinowskaauthor=G.+H.+Richmondauthor=M.+Cantariniauthor=D.+W.+Kimauthor=M.+R.+Ransonauthor=W.+Pao&title=AZD9291%2C+an+irreversible+EGFR+TKI%2C+overcomes+T790M-mediated+resistance+to+EGFR+inhibitors+in+lung+cancer&doi=10.1158%2F2159-8290.CD-14-0337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">AZD9291, an Irreversible EGFR TKI, Overcomes T790M-Mediated Resistance to EGFR Inhibitors in Lung Cancer</span></div><div class="casAuthors">Cross, Darren A. E.; Ashton, Susan E.; Ghiorghiu, Serban; Eberlein, Cath; Nebhan, Caroline A.; Spitzler, Paula J.; Orme, Jonathon P.; Finlay, M. Raymond V.; Ward, Richard A.; Mellor, Martine J.; Hughes, Gareth; Rahi, Amar; Jacobs, Vivien N.; Brewer, Monica Red; Ichihara, Eiki; Sun, Jing; Jin, Hailing; Ballard, Peter; Al-Kadhimi, Katherine; Rowlinson, Rachel; Klinowska, Teresa; Richmond, Graham H. P.; Cantarini, Mireille; Kim, Dong-Wan; Ranson, Malcolm R.; Pao, William</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1046-1061</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A third-generation EGFR inhibitor selectively targets EGFR mutants, including T790M, but not wild-type EGFR, and induces durable antitumor responses in preclin. models and patients with NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDo3e2pAh8YbVg90H21EOLACvtfcHk0li_FvUevcl8_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVykur%252FP&md5=f72e56f9463236ddc89eef32a4ab61b3</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-14-0337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-14-0337%26sid%3Dliteratum%253Aachs%26aulast%3DCross%26aufirst%3DD.%2BA.%26aulast%3DAshton%26aufirst%3DS.%2BE.%26aulast%3DGhiorghiu%26aufirst%3DS.%26aulast%3DEberlein%26aufirst%3DC.%26aulast%3DNebhan%26aufirst%3DC.%2BA.%26aulast%3DSpitzler%26aufirst%3DP.%2BJ.%26aulast%3DOrme%26aufirst%3DJ.%2BP.%26aulast%3DFinlay%26aufirst%3DM.%2BR.%26aulast%3DWard%26aufirst%3DR.%2BA.%26aulast%3DMellor%26aufirst%3DM.%2BJ.%26aulast%3DHughes%26aufirst%3DG.%26aulast%3DRahi%26aufirst%3DA.%26aulast%3DJacobs%26aufirst%3DV.%2BN.%26aulast%3DBrewer%26aufirst%3DM.%2BR.%26aulast%3DIchihara%26aufirst%3DE.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DH.%26aulast%3DBallard%26aufirst%3DP.%26aulast%3DAl-Kadhimi%26aufirst%3DK.%26aulast%3DRowlinson%26aufirst%3DR.%26aulast%3DKlinowska%26aufirst%3DT.%26aulast%3DRichmond%26aufirst%3DG.%2BH.%26aulast%3DCantarini%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DD.%2BW.%26aulast%3DRanson%26aufirst%3DM.%2BR.%26aulast%3DPao%26aufirst%3DW.%26atitle%3DAZD9291%252C%2520an%2520irreversible%2520EGFR%2520TKI%252C%2520overcomes%2520T790M-mediated%2520resistance%2520to%2520EGFR%2520inhibitors%2520in%2520lung%2520cancer%26jtitle%3DCancer%2520Discovery%26date%3D2014%26volume%3D4%26spage%3D1046%26epage%3D1061%26doi%3D10.1158%2F2159-8290.CD-14-0337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Jänne, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Planchard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohe, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramalingam, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahn, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horn, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haggstrom, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felip, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frewer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cantarini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dickinson, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghiorghiu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ranson, M.</span><span> </span><span class="NLM_article-title">AZD9291 in EGFR inhibitor-resistant non-small-cell lung cancer</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">372</span><span class="NLM_x">, </span> <span class="NLM_fpage">1689</span><span class="NLM_x">–</span> <span class="NLM_lpage">1699</span><span class="refDoi"> DOI: 10.1056/NEJMoa1411817</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1056%2FNEJMoa1411817" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=25923549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A280%3ADC%252BC2Mjpt1yltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=372&publication_year=2015&pages=1689-1699&author=P.+A.+J%C3%A4nneauthor=J.+C.+Yangauthor=D.+W.+Kimauthor=D.+Planchardauthor=Y.+Oheauthor=S.+S.+Ramalingamauthor=M.+J.+Ahnauthor=S.+W.+Kimauthor=W.+C.+Suauthor=L.+Hornauthor=D.+Haggstromauthor=E.+Felipauthor=J.+H.+Kimauthor=P.+Frewerauthor=M.+Cantariniauthor=K.+H.+Brownauthor=P.+A.+Dickinsonauthor=S.+Ghiorghiuauthor=M.+Ranson&title=AZD9291+in+EGFR+inhibitor-resistant+non-small-cell+lung+cancer&doi=10.1056%2FNEJMoa1411817"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">AZD9291 in EGFR inhibitor-resistant non-small-cell lung cancer</span></div><div class="casAuthors">Janne Pasi A; Yang James Chih-Hsin; Kim Dong-Wan; Planchard David; Ohe Yuichiro; Ramalingam Suresh S; Ahn Myung-Ju; Kim Sang-We; Su Wu-Chou; Horn Leora; Haggstrom Daniel; Felip Enriqueta; Kim Joo-Hang; Frewer Paul; Cantarini Mireille; Brown Kathryn H; Dickinson Paul A; Ghiorghiu Serban; Ranson Malcolm</div><div class="citationInfo"><span class="NLM_cas:title">The New England journal of medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">372</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">1689-99</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The EGFR T790M mutation is the most common mechanism of drug resistance to epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors in patients who have lung cancer with an EGFR mutation (EGFR-mutated lung cancer).  In preclinical models, the EGFR inhibitor AZD9291 has been shown to be effective against both EGFR tyrosine kinase inhibitor-sensitizing and T790M resistance mutations.  METHODS:  We administered AZD9291 at doses of 20 to 240 mg once daily in patients with advanced lung cancer who had radiologically documented disease progression after previous treatment with EGFR tyrosine kinase inhibitors.  The study included dose-escalation cohorts and dose-expansion cohorts.  In the expansion cohorts, prestudy tumor biopsies were required for central determination of EGFR T790M status.  Patients were assessed for safety, pharmacokinetics, and efficacy.  RESULTS:  A total of 253 patients were treated.  Among 31 patients enrolled in the dose-escalation cohorts, no dose-limiting toxic effects occurred at the doses evaluated.  An additional 222 patients were treated in five expansion cohorts.  The most common all-cause adverse events were diarrhea, rash, nausea, and decreased appetite.  The overall objective tumor response rate was 51% (95% confidence interval [CI], 45 to 58).  Among 127 patients with centrally confirmed EGFR T790M who could be evaluated for response, the response rate was 61% (95% CI, 52 to 70).  In contrast, among 61 patients without centrally detectable EGFR T790M who could be evaluated for response, the response rate was 21% (95% CI, 12 to 34).  The median progression-free survival was 9.6 months (95% CI, 8.3 to not reached) in EGFR T790M-positive patients and 2.8 months (95% CI, 2.1 to 4.3) in EGFR T790M-negative patients.  CONCLUSIONS:  AZD9291 was highly active in patients with lung cancer with the EGFR T790M mutation who had had disease progression during prior therapy with EGFR tyrosine kinase inhibitors. (Funded by AstraZeneca; ClinicalTrials.gov number, NCT01802632.).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQME11ovAjH3SnGiHuhFGv8fW6udTcc2eZEmjMK0Hlmt7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mjpt1yltA%253D%253D&md5=7304f406bb2e568fb08b6d524d027389</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1411817&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1411817%26sid%3Dliteratum%253Aachs%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DYang%26aufirst%3DJ.%2BC.%26aulast%3DKim%26aufirst%3DD.%2BW.%26aulast%3DPlanchard%26aufirst%3DD.%26aulast%3DOhe%26aufirst%3DY.%26aulast%3DRamalingam%26aufirst%3DS.%2BS.%26aulast%3DAhn%26aufirst%3DM.%2BJ.%26aulast%3DKim%26aufirst%3DS.%2BW.%26aulast%3DSu%26aufirst%3DW.%2BC.%26aulast%3DHorn%26aufirst%3DL.%26aulast%3DHaggstrom%26aufirst%3DD.%26aulast%3DFelip%26aufirst%3DE.%26aulast%3DKim%26aufirst%3DJ.%2BH.%26aulast%3DFrewer%26aufirst%3DP.%26aulast%3DCantarini%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DK.%2BH.%26aulast%3DDickinson%26aufirst%3DP.%2BA.%26aulast%3DGhiorghiu%26aufirst%3DS.%26aulast%3DRanson%26aufirst%3DM.%26atitle%3DAZD9291%2520in%2520EGFR%2520inhibitor-resistant%2520non-small-cell%2520lung%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2015%26volume%3D372%26spage%3D1689%26epage%3D1699%26doi%3D10.1056%2FNEJMoa1411817" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Finlay, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderton, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashton, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bethel, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Box, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradbury, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butterworth, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chorley, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colclough, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cross, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Currie, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grist, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hassall, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kemmitt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klinowska, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamont, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamont, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McFarland, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mellor, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orme, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perkins, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perkins, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richmond, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waring, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whittaker, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wrigley, G. L.</span><span> </span><span class="NLM_article-title">Discovery of a potent and selective EGFR inhibitor (AZD9291) of both sensitizing and T790M resistance mutations that spares the wild-type form of the receptor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">8249</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.1021/jm500973a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500973a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8249-67&author=M.+R.+Finlayauthor=M.+Andertonauthor=S.+Ashtonauthor=P.+Ballardauthor=P.+A.+Bethelauthor=M.+R.+Boxauthor=R.+H.+Bradburyauthor=S.+J.+Brownauthor=S.+Butterworthauthor=A.+Campbellauthor=C.+Chorleyauthor=N.+Colcloughauthor=D.+A.+Crossauthor=G.+S.+Currieauthor=M.+Gristauthor=L.+Hassallauthor=G.+B.+Hillauthor=D.+Jamesauthor=M.+Jamesauthor=P.+Kemmittauthor=T.+Klinowskaauthor=G.+Lamontauthor=S.+G.+Lamontauthor=N.+Martinauthor=H.+L.+McFarlandauthor=M.+J.+Mellorauthor=J.+P.+Ormeauthor=D.+Perkinsauthor=P.+Perkinsauthor=G.+Richmondauthor=P.+Smithauthor=R.+A.+Wardauthor=M.+J.+Waringauthor=D.+Whittakerauthor=S.+Wellsauthor=G.+L.+Wrigley&title=Discovery+of+a+potent+and+selective+EGFR+inhibitor+%28AZD9291%29+of+both+sensitizing+and+T790M+resistance+mutations+that+spares+the+wild-type+form+of+the+receptor&doi=10.1021%2Fjm500973a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm500973a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500973a%26sid%3Dliteratum%253Aachs%26aulast%3DFinlay%26aufirst%3DM.%2BR.%26aulast%3DAnderton%26aufirst%3DM.%26aulast%3DAshton%26aufirst%3DS.%26aulast%3DBallard%26aufirst%3DP.%26aulast%3DBethel%26aufirst%3DP.%2BA.%26aulast%3DBox%26aufirst%3DM.%2BR.%26aulast%3DBradbury%26aufirst%3DR.%2BH.%26aulast%3DBrown%26aufirst%3DS.%2BJ.%26aulast%3DButterworth%26aufirst%3DS.%26aulast%3DCampbell%26aufirst%3DA.%26aulast%3DChorley%26aufirst%3DC.%26aulast%3DColclough%26aufirst%3DN.%26aulast%3DCross%26aufirst%3DD.%2BA.%26aulast%3DCurrie%26aufirst%3DG.%2BS.%26aulast%3DGrist%26aufirst%3DM.%26aulast%3DHassall%26aufirst%3DL.%26aulast%3DHill%26aufirst%3DG.%2BB.%26aulast%3DJames%26aufirst%3DD.%26aulast%3DJames%26aufirst%3DM.%26aulast%3DKemmitt%26aufirst%3DP.%26aulast%3DKlinowska%26aufirst%3DT.%26aulast%3DLamont%26aufirst%3DG.%26aulast%3DLamont%26aufirst%3DS.%2BG.%26aulast%3DMartin%26aufirst%3DN.%26aulast%3DMcFarland%26aufirst%3DH.%2BL.%26aulast%3DMellor%26aufirst%3DM.%2BJ.%26aulast%3DOrme%26aufirst%3DJ.%2BP.%26aulast%3DPerkins%26aufirst%3DD.%26aulast%3DPerkins%26aufirst%3DP.%26aulast%3DRichmond%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DP.%26aulast%3DWard%26aufirst%3DR.%2BA.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DWhittaker%26aufirst%3DD.%26aulast%3DWells%26aufirst%3DS.%26aulast%3DWrigley%26aufirst%3DG.%2BL.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520and%2520selective%2520EGFR%2520inhibitor%2520%2528AZD9291%2529%2520of%2520both%2520sensitizing%2520and%2520T790M%2520resistance%2520mutations%2520that%2520spares%2520the%2520wild-type%2520form%2520of%2520the%2520receptor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8249%26epage%3D67%26doi%3D10.1021%2Fjm500973a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Dietlein, F.; Heuckmann, J.; Choidas, A.; Basu, D.; Habenberger, P.; Fang, Z.; Ortiz-Cuaran, S.; Leenders, F.; Eickhoff, J.; Koch, U.; Getlik, M.; Termathe, M.; Sallouh, M.; Greff, Z.; Varga, Z.; Balke-Want, H.; Sos, M. L.; Peifer, M.; Reinhardt, H. C.; Örfi, L.; Kéri, G.; Ansen, S.; Heukamp, L. C.; Büttner, R.; Rauh, D.; Klebl, B.; Thomas, R. K.</span><span> </span><span class="NLM_article-title">Reconstructing structure-activity relationships of kinase inhibitors by automated chemopanning</span>. Submitted for publication.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Dietlein%2C+F.%3B+Heuckmann%2C+J.%3B+Choidas%2C+A.%3B+Basu%2C+D.%3B+Habenberger%2C+P.%3B+Fang%2C+Z.%3B+Ortiz-Cuaran%2C+S.%3B+Leenders%2C+F.%3B+Eickhoff%2C+J.%3B+Koch%2C+U.%3B+Getlik%2C+M.%3B+Termathe%2C+M.%3B+Sallouh%2C+M.%3B+Greff%2C+Z.%3B+Varga%2C+Z.%3B+Balke-Want%2C+H.%3B+Sos%2C+M.+L.%3B+Peifer%2C+M.%3B+Reinhardt%2C+H.+C.%3B+%C3%96rfi%2C+L.%3B+K%C3%A9ri%2C+G.%3B+Ansen%2C+S.%3B+Heukamp%2C+L.+C.%3B+B%C3%BCttner%2C+R.%3B+Rauh%2C+D.%3B+Klebl%2C+B.%3B+Thomas%2C+R.+K.+Reconstructing+structure-activity+relationships+of+kinase+inhibitors+by+automated+chemopanning.+Submitted+for+publication."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dunpublished%26sid%3Dliteratum%253Aachs%26aulast%3DDietlein%26aufirst%3DF.%26atitle%3DReconstructing%2520structure-activity%2520relationships%2520of%2520kinase%2520inhibitors%2520by%2520automated%2520chemopanning" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Blair, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rauh, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yun, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, Q. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rode, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eck, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span> </span><span class="NLM_article-title">Structure-guided development of affinity probes for tyrosine kinases using chemical genetics</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">229</span><span class="NLM_x">–</span> <span class="NLM_lpage">238</span><span class="refDoi"> DOI: 10.1038/nchembio866</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1038%2Fnchembio866" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=17334377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtFKnsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=229-238&author=J.+A.+Blairauthor=D.+Rauhauthor=C.+Kungauthor=C.+H.+Yunauthor=Q.+W.+Fanauthor=H.+Rodeauthor=C.+Zhangauthor=M.+J.+Eckauthor=W.+A.+Weissauthor=K.+M.+Shokat&title=Structure-guided+development+of+affinity+probes+for+tyrosine+kinases+using+chemical+genetics&doi=10.1038%2Fnchembio866"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-guided development of affinity probes for tyrosine kinases using chemical genetics</span></div><div class="casAuthors">Blair, Jimmy A.; Rauh, Daniel; Kung, Charles; Yun, Cai-Hong; Fan, Qi-Wen; Rode, Haridas; Zhang, Chao; Eck, Michael J.; Weiss, William A.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">229-238</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">As key components in nearly every signal transduction pathway, protein kinases are attractive targets for the regulation of cellular signaling by small-mol. inhibitors.  The authors report the structure-guided development of 6-acrylamido-4-anilinoquinazoline irreversible kinase inhibitors that potently and selectively target rationally designed kinases bearing two selectivity elements that are not found together in any wild-type kinase: an electrophile-targeted cysteine residue and a glycine gatekeeper residue.  Cocrystal structures of two irreversible quinazoline inhibitors bound to either epidermal growth factor receptor (EGFR) or engineered c-Src show covalent inhibitor binding to the targeted cysteine (Cys797 in EGFR and Cys345 in engineered c-Src).  Cocrystal structures of two irreversible quinazoline inhibitors (I, II) bound to either epidermal growth factor receptor (EGFR) or engineered c-Src show covalent inhibitor binding to the targeted cysteine (Cys797 in EGFR and Cys345 in engineered c-Src).  Based on these structures, the authors developed a fluorescent 6-acrylamido-4-anilinoquinazoline affinity probe (III) to report the fraction of kinase necessary for cellular signaling, and the authors used these reagents to quantitate the relationship between EGFR stimulation by EGF and its downstream outputs-Akt, Erk1 and Erk2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrG0KmYAm-skrVg90H21EOLACvtfcHk0liyFC6OoQkkZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtFKnsr0%253D&md5=ce4d24a8d65d7080645bf382cd65dbca</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnchembio866&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio866%26sid%3Dliteratum%253Aachs%26aulast%3DBlair%26aufirst%3DJ.%2BA.%26aulast%3DRauh%26aufirst%3DD.%26aulast%3DKung%26aufirst%3DC.%26aulast%3DYun%26aufirst%3DC.%2BH.%26aulast%3DFan%26aufirst%3DQ.%2BW.%26aulast%3DRode%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DWeiss%26aufirst%3DW.%2BA.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DStructure-guided%2520development%2520of%2520affinity%2520probes%2520for%2520tyrosine%2520kinases%2520using%2520chemical%2520genetics%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D3%26spage%3D229%26epage%3D238%26doi%3D10.1038%2Fnchembio866" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span> </span><span class="NLM_article-title">Residence time of receptor-ligand complexes and its effect on biological function</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">5481</span><span class="NLM_x">–</span> <span class="NLM_lpage">5492</span><span class="refDoi"> DOI: 10.1021/bi8002023</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi8002023" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1cXks1Ont7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=5481-5492&author=P.+J.+Tumminoauthor=R.+A.+Copeland&title=Residence+time+of+receptor-ligand+complexes+and+its+effect+on+biological+function&doi=10.1021%2Fbi8002023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Residence Time of Receptor-Ligand Complexes and Its Effect on Biological Function</span></div><div class="casAuthors">Tummino, Peter J.; Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5481-5492</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The formation and duration of binary receptor-ligand complexes are fundamental to many physiol. processes.  Most often, the effectiveness of interaction between a receptor and its ligand is quantified in terms of closed system, equil. affinity measurements, such as IC50 and Kd.  In the context of in vivo biol., however, the extent and duration of responses to receptor-ligand interactions depend greatly on the time period over which the ligand is in residence on its receptor.  Here we define receptor-ligand complex residence time in quant. terms and describe its significance to biol. function.  Examples of the importance of residence time are presented for natural ligands of different receptor types.  The impact of residence time on the optimization of potential ligands as drugs for human medicine is also described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHshN2Bt_r9LVg90H21EOLACvtfcHk0liyFC6OoQkkZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXks1Ont7s%253D&md5=57f79f69f12c6ed14e3347638381d943</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fbi8002023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi8002023%26sid%3Dliteratum%253Aachs%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DResidence%2520time%2520of%2520receptor-ligand%2520complexes%2520and%2520its%2520effect%2520on%2520biological%2520function%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D5481%26epage%3D5492%26doi%3D10.1021%2Fbi8002023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Barf, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaptein, A.</span><span> </span><span class="NLM_article-title">Irreversible protein kinase inhibitors: balancing the benefits and risks</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">6243</span><span class="NLM_x">–</span> <span class="NLM_lpage">6262</span><span class="refDoi"> DOI: 10.1021/jm3003203</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3003203" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsVerur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6243-6262&author=T.+Barfauthor=A.+Kaptein&title=Irreversible+protein+kinase+inhibitors%3A+balancing+the+benefits+and+risks&doi=10.1021%2Fjm3003203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Irreversible Protein Kinase Inhibitors: Balancing the Benefits and Risks</span></div><div class="casAuthors">Barf, Tjeerd; Kaptein, Allard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6243-6262</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Not applicable for a Perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4PLlisO0OVbVg90H21EOLACvtfcHk0liyFC6OoQkkZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsVerur8%253D&md5=5d87850857d37faab02e6037ef8140a4</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm3003203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3003203%26sid%3Dliteratum%253Aachs%26aulast%3DBarf%26aufirst%3DT.%26aulast%3DKaptein%26aufirst%3DA.%26atitle%3DIrreversible%2520protein%2520kinase%2520inhibitors%253A%2520balancing%2520the%2520benefits%2520and%2520risks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6243%26epage%3D6262%26doi%3D10.1021%2Fjm3003203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Ward, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderton, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashton, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bethel, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Box, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butterworth, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colclough, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chorley, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuaqui, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cross, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dakin, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Debreczeni, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberlein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finlay, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grist, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klinowska, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orme, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waring, M. J.</span><span> </span><span class="NLM_article-title">Structure- and reactivity-based development of covalent inhibitors of the activating and gatekeeper mutant forms of the epidermal growth factor receptor (EGFR)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">7025</span><span class="NLM_x">–</span> <span class="NLM_lpage">7048</span><span class="refDoi"> DOI: 10.1021/jm400822z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400822z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1CltL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7025-7048&author=R.+A.+Wardauthor=M.+J.+Andertonauthor=S.+Ashtonauthor=P.+A.+Bethelauthor=M.+Boxauthor=S.+Butterworthauthor=N.+Colcloughauthor=C.+G.+Chorleyauthor=C.+Chuaquiauthor=D.+A.+Crossauthor=L.+A.+Dakinauthor=J.+E.+Debreczeniauthor=C.+Eberleinauthor=M.+R.+Finlayauthor=G.+B.+Hillauthor=M.+Gristauthor=T.+C.+Klinowskaauthor=C.+Laneauthor=S.+Martinauthor=J.+P.+Ormeauthor=P.+Smithauthor=F.+Wangauthor=M.+J.+Waring&title=Structure-+and+reactivity-based+development+of+covalent+inhibitors+of+the+activating+and+gatekeeper+mutant+forms+of+the+epidermal+growth+factor+receptor+%28EGFR%29&doi=10.1021%2Fjm400822z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Structure- and Reactivity-Based Development of Covalent Inhibitors of the Activating and Gatekeeper Mutant Forms of the Epidermal Growth Factor Receptor (EGFR)</span></div><div class="casAuthors">Ward, Richard A.; Anderton, Mark J.; Ashton, Susan; Bethel, Paul A.; Box, Matthew; Butterworth, Sam; Colclough, Nicola; Chorley, Christopher G.; Chuaqui, Claudio; Cross, Darren A. E.; Dakin, Les A.; Debreczeni, Judit E.; Eberlein, Cath; Finlay, M. Raymond V.; Hill, George B.; Grist, Matthew; Klinowska, Teresa C. M.; Lane, Clare; Martin, Scott; Orme, Jonathon P.; Smith, Peter; Wang, Fengjiang; Waring, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7025-7048</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel series of small-mol. inhibitors has been developed to target the double mutant form of the epidermal growth factor receptor (EGFR) tyrosine kinase, which is resistant to treatment with gefitinib and erlotinib.  Our reported compds. also show selectivity over wild-type EGFR.  Guided by mol. modeling, this series was evolved to target a cysteine residue in the ATP binding site via covalent bond formation and demonstrates high levels of activity in cellular models of the double mutant form of EGFR.  In addn., these compds. show significant activity against the activating mutations, which gefitinib and erlotinib target and inhibition of which gives rise to their obsd. clin. efficacy.  A glutathione (GSH)-based assay was used to measure thiol reactivity toward the electrophilic functionality of the inhibitor series, enabling both the identification of a suitable reactivity window for their potency and the development of a reactivity quant. structure-property relationship (QSPR) to support design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKi6oNvtQPEbVg90H21EOLACvtfcHk0lh8xQ2KeibcvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1CltL%252FN&md5=59c86415d71ac2abe5c6d4486e5d2517</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm400822z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400822z%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DR.%2BA.%26aulast%3DAnderton%26aufirst%3DM.%2BJ.%26aulast%3DAshton%26aufirst%3DS.%26aulast%3DBethel%26aufirst%3DP.%2BA.%26aulast%3DBox%26aufirst%3DM.%26aulast%3DButterworth%26aufirst%3DS.%26aulast%3DColclough%26aufirst%3DN.%26aulast%3DChorley%26aufirst%3DC.%2BG.%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DCross%26aufirst%3DD.%2BA.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DDebreczeni%26aufirst%3DJ.%2BE.%26aulast%3DEberlein%26aufirst%3DC.%26aulast%3DFinlay%26aufirst%3DM.%2BR.%26aulast%3DHill%26aufirst%3DG.%2BB.%26aulast%3DGrist%26aufirst%3DM.%26aulast%3DKlinowska%26aufirst%3DT.%2BC.%26aulast%3DLane%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DOrme%26aufirst%3DJ.%2BP.%26aulast%3DSmith%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26atitle%3DStructure-%2520and%2520reactivity-based%2520development%2520of%2520covalent%2520inhibitors%2520of%2520the%2520activating%2520and%2520gatekeeper%2520mutant%2520forms%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7025%26epage%3D7048%26doi%3D10.1021%2Fjm400822z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Zhou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ercan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jänne, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Discovery of selective irreversible inhibitors for EGFR-T790M</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">638</span><span class="NLM_x">–</span> <span class="NLM_lpage">643</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2010.12.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1016%2Fj.bmcl.2010.12.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=21208802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVKrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=638-643&author=W.+Zhouauthor=D.+Ercanauthor=P.+A.+J%C3%A4nneauthor=N.+S.+Gray&title=Discovery+of+selective+irreversible+inhibitors+for+EGFR-T790M&doi=10.1016%2Fj.bmcl.2010.12.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of selective irreversible inhibitors for EGFR-T790M</span></div><div class="casAuthors">Zhou, Wenjun; Ercan, Dalia; Jaenne, Pasi A.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">638-643</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Targeting the epidermal growth factor receptor kinase (EGFR) with ATP-competitive kinase inhibitors results in dramatic but short-lived responses in patients with EGFR mutant non small cell lung cancer.  A series of novel covalent EGFR kinase inhibitors with selectivity for the clin. relevant T790M gatekeeper' resistance mutation relative to wild-type EGFR were discovered by library screening.  A representative compd. 3i (I) was obtained through a systematic SAR study guided by mutant EGFR-dependent cellular proliferation assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB2Qq3f78-wLVg90H21EOLACvtfcHk0lh8xQ2KeibcvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVKrtA%253D%253D&md5=c19f3f288ab99aaaab0a23bcf5273daf</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.12.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.12.036%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDiscovery%2520of%2520selective%2520irreversible%2520inhibitors%2520for%2520EGFR-T790M%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D638%26epage%3D643%26doi%3D10.1016%2Fj.bmcl.2010.12.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Wissner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mansour, T. S.</span><span> </span><span class="NLM_article-title">The development of HKI-272 and related compounds for the treatment of cancer</span> <span class="citation_source-journal">Arch. Pharm. (Weinheim, Ger.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">341</span><span class="NLM_x">, </span> <span class="NLM_fpage">465</span><span class="NLM_x">–</span> <span class="NLM_lpage">477</span><span class="refDoi"> DOI: 10.1002/ardp.200800009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1002%2Fardp.200800009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=18493974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVKnsb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2008&pages=465-477&author=A.+Wissnerauthor=T.+S.+Mansour&title=The+development+of+HKI-272+and+related+compounds+for+the+treatment+of+cancer&doi=10.1002%2Fardp.200800009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The development of HKI-272 and related compounds for the treatment of cancer</span></div><div class="casAuthors">Wissner, Allan; Mansour, Tarek S.</div><div class="citationInfo"><span class="NLM_cas:title">Archiv der Pharmazie (Weinheim, Germany)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">465-477</span>CODEN:
                <span class="NLM_cas:coden">ARPMAS</span>;
        ISSN:<span class="NLM_cas:issn">0365-6233</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The development of HKI-272 and EKB-569 for the treatment of cancer is described.  These compds. function as irreversible inhibitors of some members of the ErbB family of receptor tyrosine kinases.  In particular, they target epidermal growth factor receptor (EGFR, also known as ErbB-1) and human epidermal growth factor receptor-2 (HER2, also known as ErbB-2).  Both, HKI-272 and EKB-569 are 4-anilino-3-cyano quinoline derivs. that contain a 4-(dimethylamino)crotonamide Michael-acceptor group at the 6-position.  These compds. inhibit the function of the target enzymes by forming a covalent interaction with a conserved cysteine residue located in the kinase domains of these proteins.  The potential advantages of using irreversible inhibitors for this purpose are discussed.  We summarize the recent findings concerning some somatic mutations in EGFR and their relevance with respect to the irreversible inhibitors.  In particular, we highlight the findings that these irreversible inhibitors retain activity against tumors that have acquired a resistance to the reversible binding inhibitors gefitinib and erlotinib.  The promising interim clin. trial results for HKI-272 and EKB-569 in treating colon, lung, and breast cancers are summarized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCSokM42ZhprVg90H21EOLACvtfcHk0lh8xQ2KeibcvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVKnsb7E&md5=2e46d99ca74e297e6a7e860634bf096d</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fardp.200800009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fardp.200800009%26sid%3Dliteratum%253Aachs%26aulast%3DWissner%26aufirst%3DA.%26aulast%3DMansour%26aufirst%3DT.%2BS.%26atitle%3DThe%2520development%2520of%2520HKI-272%2520and%2520related%2520compounds%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DArch.%2520Pharm.%2520%2528Weinheim%252C%2520Ger.%2529%26date%3D2008%26volume%3D341%26spage%3D465%26epage%3D477%26doi%3D10.1002%2Fardp.200800009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Krippendorff, B. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neuhaus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lienau, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reichel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huisinga, W.</span><span> </span><span class="NLM_article-title">Mechanism-based inhibition: deriving K(I) and k(inact) directly from time-dependent IC(50) values</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">913</span><span class="NLM_x">–</span> <span class="NLM_lpage">923</span><span class="refDoi"> DOI: 10.1177/1087057109336751</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1177%2F1087057109336751" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=19675314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlSjtLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=913-923&author=B.+F.+Krippendorffauthor=R.+Neuhausauthor=P.+Lienauauthor=A.+Reichelauthor=W.+Huisinga&title=Mechanism-based+inhibition%3A+deriving+K%28I%29+and+k%28inact%29+directly+from+time-dependent+IC%2850%29+values&doi=10.1177%2F1087057109336751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism-based inhibition: deriving KI and kinact directly from time-dependent IC50 values</span></div><div class="casAuthors">Krippendorff, Ben-Fillippo; Neuhaus, Roland; Lienau, Philip; Reichel, Andreas; Huisinga, Wilhelm</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">913-923</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The potential of enzyme inhibition of a drug is frequently quantified in terms of IC50 values.  Although this is a suitable quantity for reversible inhibitors, concerns arise when dealing with irreversible or mechanism-based inhibitors (MBIs).  IC50 values of MBIs are time dependent, causing serious problems when aiming at ranking different compds. with respect to their inhibitory potential.  As a consequence, most studies and ranking schemes related to MBIs rely on the inhibition const. (KI) and the rate of enzyme inactivation (kinact) rather than on IC50 values.  In this article, the authors derive a novel relation between potentially time-dependent IC50 values and KI, kinact parameters for different types of inhibition.  This allows for direct estn. of KI and kinact values from time-dependent IC50 values, even without the need of addnl. preincubation expts.  The application of this approach is illustrated using a fluorimetric assay to access the drug-drug interaction potential assocd. with new chem. entities.  The approach can easily be implemented using std. software tools (e.g., XLfit) and may also be suitable for applications where mechanism-based inhibition is a desired mode of action (e.g., at particular pharmacol. drug targets).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSOIpe0vN8JLVg90H21EOLACvtfcHk0lgY1MvCMGDdjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlSjtLjF&md5=93cb5f8644c7e1bdf872cf46756039cf</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1177%2F1087057109336751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057109336751%26sid%3Dliteratum%253Aachs%26aulast%3DKrippendorff%26aufirst%3DB.%2BF.%26aulast%3DNeuhaus%26aufirst%3DR.%26aulast%3DLienau%26aufirst%3DP.%26aulast%3DReichel%26aufirst%3DA.%26aulast%3DHuisinga%26aufirst%3DW.%26atitle%3DMechanism-based%2520inhibition%253A%2520deriving%2520K%2528I%2529%2520and%2520k%2528inact%2529%2520directly%2520from%2520time-dependent%2520IC%252850%2529%2520values%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26spage%3D913%26epage%3D923%26doi%3D10.1177%2F1087057109336751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Schwartz, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuzmic, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solowiej, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergqvist, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolanos, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almaden, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagata, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalvie, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kath, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, B. W.</span><span> </span><span class="NLM_article-title">Covalent EGFR inhibitor analysis reveals importance of reversible interactions to potency and mechanisms of drug resistance</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">173</span><span class="NLM_x">–</span> <span class="NLM_lpage">178</span><span class="refDoi"> DOI: 10.1073/pnas.1313733111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1073%2Fpnas.1313733111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=24347635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC2cXps1eluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=173-178&author=P.+A.+Schwartzauthor=P.+Kuzmicauthor=J.+Solowiejauthor=S.+Bergqvistauthor=B.+Bolanosauthor=C.+Almadenauthor=A.+Nagataauthor=K.+Ryanauthor=J.+Fengauthor=D.+Dalvieauthor=J.+C.+Kathauthor=M.+Xuauthor=R.+Waniauthor=B.+W.+Murray&title=Covalent+EGFR+inhibitor+analysis+reveals+importance+of+reversible+interactions+to+potency+and+mechanisms+of+drug+resistance&doi=10.1073%2Fpnas.1313733111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent EGFR inhibitor analysis reveals importance of reversible interactions to potency and mechanisms of drug resistance</span></div><div class="casAuthors">Schwartz, Phillip A.; Kuzmic, Petr; Solowiej, James; Bergqvist, Simon; Bolanos, Ben; Almaden, Chau; Nagata, Asako; Ryan, Kevin; Feng, Junli; Dalvie, Deepak; Kath, John C.; Xu, Meirong; Wani, Revati; Murray, Brion William</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">173-178</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Covalent inhibition is a reemerging paradigm in kinase drug design, but the roles of inhibitor binding affinity and chem. reactivity in overall potency are not well-understood.  To characterize the underlying mol. processes at a microscopic level and det. the appropriate kinetic consts., specialized exptl. design and advanced numerical integration of differential equations are developed.  Previously uncharacterized investigational covalent drugs reported here are shown to be extremely effective epidermal growth factor receptor (EGFR) inhibitors (kinact/Ki in the range 105-107 M-1s-1), despite their low specific reactivity (kinact ≤ 2.1 × 10-3 s-1), which is compensated for by high binding affinities (Ki < 1 nM).  For inhibitors relying on reactivity to achieve potency, noncovalent enzyme-inhibitor complex partitioning between inhibitor dissocn. and bond formation is central.  Interestingly, reversible binding affinity of EGFR covalent inhibitors is highly correlated with antitumor cell potency.  Furthermore, cellular potency for a subset of covalent inhibitors can be accounted for solely through reversible interactions.  One reversible interaction is between EGFR-Cys797 nucleophile and the inhibitor's reactive group, which may also contribute to drug resistance.  Because covalent inhibitors target a cysteine residue, the effects of its oxidn. on enzyme catalysis and inhibitor pharmacol. are characterized.  Oxidn. of the EGFR cysteine nucleophile does not alter catalysis but has widely varied effects on inhibitor potency depending on the EGFR context (e.g., oncogenic mutations), type of oxidn. (sulfinylation or glutathiolation), and inhibitor architecture.  These methods, parameters, and insights provide a rational framework for assessing and designing effective covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTpPuRq8MwCLVg90H21EOLACvtfcHk0lgY1MvCMGDdjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXps1eluw%253D%253D&md5=47197c5e7876a238eefb2e8e7de305ea</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1313733111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1313733111%26sid%3Dliteratum%253Aachs%26aulast%3DSchwartz%26aufirst%3DP.%2BA.%26aulast%3DKuzmic%26aufirst%3DP.%26aulast%3DSolowiej%26aufirst%3DJ.%26aulast%3DBergqvist%26aufirst%3DS.%26aulast%3DBolanos%26aufirst%3DB.%26aulast%3DAlmaden%26aufirst%3DC.%26aulast%3DNagata%26aufirst%3DA.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DDalvie%26aufirst%3DD.%26aulast%3DKath%26aufirst%3DJ.%2BC.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DWani%26aufirst%3DR.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26atitle%3DCovalent%2520EGFR%2520inhibitor%2520analysis%2520reveals%2520importance%2520of%2520reversible%2520interactions%2520to%2520potency%2520and%2520mechanisms%2520of%2520drug%2520resistance%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D173%26epage%3D178%26doi%3D10.1073%2Fpnas.1313733111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Kansy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senner, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gubernator, K.</span><span> </span><span class="NLM_article-title">Physicochemical high throughput screening: parallel artificial membrane permeation assay in the description of passive absorption processes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">1007</span><span class="NLM_x">–</span> <span class="NLM_lpage">1010</span><span class="refDoi"> DOI: 10.1021/jm970530e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm970530e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaK1cXhs12kt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=1007-1010&author=M.+Kansyauthor=F.+Sennerauthor=K.+Gubernator&title=Physicochemical+high+throughput+screening%3A+parallel+artificial+membrane+permeation+assay+in+the+description+of+passive+absorption+processes&doi=10.1021%2Fjm970530e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical High Throughput Screening: Parallel Artificial Membrane Permeation Assay in the Description of Passive Absorption Processes</span></div><div class="casAuthors">Kansy, Manfred; Senner, Frank; Gubernator, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1007-1010</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The majority of pharmaceutical discovery projects aim for an orally available form of a new therapeutic principle.  One obstacle for per oral treatment is intestinal absorption.  A simple predictive exptl. method to assess the potential of intestinal absorption is therefore highly welcome.  The authors describe here the PAMPA (Parallel Artificial Membrane Permeation Assay) method, a simple, robust, high throughput screen (HTS) which has been shown to be predictive for passive diffusion through membranes and thus for oral absorption.  In addn. PAMPA can deliver information in parallel on the lipophilicity, the ionization state and the soly. of a compd.  Therefore PAMPA complements HTS, combinatorial chem. and parallel synthesis efforts in preclin. pharmaceutical research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqcwCGodCr1rVg90H21EOLACvtfcHk0lgY1MvCMGDdjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhs12kt70%253D&md5=85d8ab409313b583de52348eb4d895da</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm970530e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm970530e%26sid%3Dliteratum%253Aachs%26aulast%3DKansy%26aufirst%3DM.%26aulast%3DSenner%26aufirst%3DF.%26aulast%3DGubernator%26aufirst%3DK.%26atitle%3DPhysicochemical%2520high%2520throughput%2520screening%253A%2520parallel%2520artificial%2520membrane%2520permeation%2520assay%2520in%2520the%2520description%2520of%2520passive%2520absorption%2520processes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D1007%26epage%3D1010%26doi%3D10.1021%2Fjm970530e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Shim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackerell, A. D.,  Jr.</span><span> </span><span class="NLM_article-title">Computational ligand-based rational design: Role of conformational sampling and force fields in model development</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">356</span><span class="NLM_x">–</span> <span class="NLM_lpage">370</span><span class="refDoi"> DOI: 10.1039/c1md00044f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1039%2Fc1md00044f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=21716805" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3MXls1Kht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=356-370&author=J.+Shimauthor=A.+D.+Mackerell&title=Computational+ligand-based+rational+design%3A+Role+of+conformational+sampling+and+force+fields+in+model+development&doi=10.1039%2Fc1md00044f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Computational ligand-based rational design: role of conformational sampling and force fields in model development</span></div><div class="casAuthors">Shim, Jihyun; MacKerell, Alexander D., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">356-370</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  A significant no. of drug discovery efforts are based on natural products or high throughput screens from which compds. showing potential therapeutic effects are identified without knowledge of the target mol. or its 3D structure.  In such cases computational ligand-based drug design (LBDD) can accelerate the drug discovery processes.  LBDD is a general approach to elucidate the relationship of a compd.'s structure and physicochem. attributes to its biol. activity.  The resulting structure-activity relationship (SAR) may then act as the basis for the prediction of compds. with improved biol. attributes.  LBDD methods range from pharmacophore models identifying essential features of ligands responsible for their activity, quant. structure-activity relationships (QSAR) yielding quant. ests. of activities based on physiochem. properties, and to similarity searching, which explores compds. with similar properties as well as various combinations of the above.  A no. of recent LBDD approaches involve the use of multiple conformations of the ligands being studied.  One of the basic components to generate multiple conformations in LBDD is mol. mechanics (MM), which apply an empirical energy function to relate conformation to energies and forces.  The collection of conformations for ligands is then combined with functional data using methods ranging from regression anal. to neural networks, from which the SAR is detd.  Accordingly, for effective application of LBDD for SAR detns. it is important that the compds. be accurately modelled such that the appropriate range of conformations accessible to the ligands is identified.  Such accurate modeling is largely based on use of the appropriate empirical force field for the mols. being investigated and the approaches used to generate the conformations.  The present chapter includes a brief overview of currently used SAR methods in LBDD followed by a more detailed presentation of issues and limitations assocd. with empirical energy functions and conformational sampling methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkNp46K1NKk7Vg90H21EOLACvtfcHk0lh5J5aMimZCzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXls1Kht7c%253D&md5=70d822d1ca2faa484f723fb109597b77</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1039%2Fc1md00044f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1md00044f%26sid%3Dliteratum%253Aachs%26aulast%3DShim%26aufirst%3DJ.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26atitle%3DComputational%2520ligand-based%2520rational%2520design%253A%2520Role%2520of%2520conformational%2520sampling%2520and%2520force%2520fields%2520in%2520model%2520development%26jtitle%3DMedChemComm%26date%3D2011%26volume%3D2%26spage%3D356%26epage%3D370%26doi%3D10.1039%2Fc1md00044f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Richards, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brant, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boulanger, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cairo, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wulff, J. E.</span><span> </span><span class="NLM_article-title">Conformational analysis of peramivir reveals critical differences between free and enzyme-bound states</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1483</span><span class="NLM_x">–</span> <span class="NLM_lpage">1488</span><span class="refDoi"> DOI: 10.1039/C4MD00168K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1039%2FC4MD00168K" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1483-1488&author=M.+R.+Richardsauthor=M.+G.+Brantauthor=M.+J.+Boulangerauthor=C.+W.+Cairoauthor=J.+E.+Wulff&title=Conformational+analysis+of+peramivir+reveals+critical+differences+between+free+and+enzyme-bound+states&doi=10.1039%2FC4MD00168K"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1039%2FC4MD00168K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00168K%26sid%3Dliteratum%253Aachs%26aulast%3DRichards%26aufirst%3DM.%2BR.%26aulast%3DBrant%26aufirst%3DM.%2BG.%26aulast%3DBoulanger%26aufirst%3DM.%2BJ.%26aulast%3DCairo%26aufirst%3DC.%2BW.%26aulast%3DWulff%26aufirst%3DJ.%2BE.%26atitle%3DConformational%2520analysis%2520of%2520peramivir%2520reveals%2520critical%2520differences%2520between%2520free%2520and%2520enzyme-bound%2520states%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D1483%26epage%3D1488%26doi%3D10.1039%2FC4MD00168K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Forti, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavasotto, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orozco, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barril, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luque, F. J.</span><span> </span><span class="NLM_article-title">A Multilevel Strategy for the Exploration of the Conformational Flexibility of Small Molecules</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1808</span><span class="NLM_x">–</span> <span class="NLM_lpage">1819</span><span class="refDoi"> DOI: 10.1021/ct300097s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300097s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVKmtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=1808-1819&author=F.+Fortiauthor=C.+N.+Cavasottoauthor=M.+Orozcoauthor=X.+Barrilauthor=F.+J.+Luque&title=A+Multilevel+Strategy+for+the+Exploration+of+the+Conformational+Flexibility+of+Small+Molecules&doi=10.1021%2Fct300097s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">A Multilevel Strategy for the Exploration of the Conformational Flexibility of Small Molecules</span></div><div class="casAuthors">Forti, Flavio; Cavasotto, Claudio N.; Orozco, Modesto; Barril, Xavier; Luque, F. Javier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1808-1819</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Predicting the conformational preferences of flexible compds. is still a challenging problem with important implications in areas such as mol. recognition and drug design.  In this work, we describe a multilevel strategy to explore the conformational preferences of mols.  The method relies on the predominant-state approxn., which partitions the conformational space into distinct conformational wells.  Moreover, it combines low-level (LL) methods for sampling the conformational min. and high-level (HL) techniques for calibrating their relative stability.  In the implementation used in this study, the LL sampling is performed with the semiempirical RM1 Hamiltonian, and solvent effects are included using the RM1 version of the Miertus, Scrocco, and Tomasi (MST) continuum solvation model.  The HL refinement of the conformational wells is performed by combining geometry optimizations of the min. at the B3LYP (gas phase) or MST-B3LYP (soln.) level, followed by single point MP2 computations using Dunning's augmented basis sets.  Then, the effective free energy of a conformational well is estd. by combining the MP2 energy, supplemented with the MST-B3LYP solvation free energy for a conformational search in soln., with the local curvature of the well sampled at the semiempirical level.  Applications of this strategy involve the exploration of the conformational preferences of 1,2-dichloroethane and neutral histamine in both the gas phase and water soln.  Finally, the multilevel strategy is used to est. the reorganization cost required for selecting the bioactive conformation of HIV reverse transcriptase inhibitors, which is estd. to be at most 1.3 kcal/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFC-5jHPspaLVg90H21EOLACvtfcHk0lh5J5aMimZCzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVKmtLY%253D&md5=35eb55f0805c4c929d9706230a9b41b1</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fct300097s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300097s%26sid%3Dliteratum%253Aachs%26aulast%3DForti%26aufirst%3DF.%26aulast%3DCavasotto%26aufirst%3DC.%2BN.%26aulast%3DOrozco%26aufirst%3DM.%26aulast%3DBarril%26aufirst%3DX.%26aulast%3DLuque%26aufirst%3DF.%2BJ.%26atitle%3DA%2520Multilevel%2520Strategy%2520for%2520the%2520Exploration%2520of%2520the%2520Conformational%2520Flexibility%2520of%2520Small%2520Molecules%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D1808%26epage%3D1819%26doi%3D10.1021%2Fct300097s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Development and testing of a general amber force field</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1157</span><span class="NLM_x">–</span> <span class="NLM_lpage">1174</span><span class="refDoi"> DOI: 10.1002/jcc.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+a+general+amber+force+field&doi=10.1002%2Fjcc.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0lh5J5aMimZCzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520a%2520general%2520amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174%26doi%3D10.1002%2Fjcc.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Automatic atom type and bond type perception in molecular mechanical calculations</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">247</span><span class="NLM_x">–</span> <span class="NLM_lpage">260</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2005.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1016%2Fj.jmgm.2005.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=16458552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=247-260&author=J.+Wangauthor=W.+Wangauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Automatic+atom+type+and+bond+type+perception+in+molecular+mechanical+calculations&doi=10.1016%2Fj.jmgm.2005.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic atom type and bond type perception in molecular mechanical calculations</span></div><div class="casAuthors">Wang, Junmei; Wang, Wei; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-260</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In mol. mechanics (MM) studies, atom types and/or bond types of mols. are needed to det. prior to energy calcns.  The authors present here an automatic algorithm of perceiving atom types that are defined in a description table, and an automatic algorithm of assigning bond types just based on at. connectivity.  The algorithms have been implemented in a new module of the AMBER packages.  This auxiliary module, antechamber (roughly meaning "before AMBER"), can be applied to generate necessary inputs of leap-the AMBER program to generate topologies for minimization, mol. dynamics, etc., for most org. mols.  The algorithms behind the manipulations may be useful for other mol. mech. packages as well as applications that need to designate atom types and bond types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqizj6F18nBYbVg90H21EOLACvtfcHk0liQNZcOitBGaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D&md5=8031a21d2784d5dea12e70868522aa61</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2005.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2005.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DAutomatic%2520atom%2520type%2520and%2520bond%2520type%2520perception%2520in%2520molecular%2520mechanical%2520calculations%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2006%26volume%3D25%26spage%3D247%26epage%3D260%26doi%3D10.1016%2Fj.jmgm.2005.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dumont, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span> </span><span class="NLM_article-title">Comparison of charge models for fixed-charge force fields: small-molecule hydration free energies in explicit solvent</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">2242</span><span class="NLM_x">–</span> <span class="NLM_lpage">2254</span><span class="refDoi"> DOI: 10.1021/jp0667442</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp0667442" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsFWmsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2007&pages=2242-2254&author=D.+L.+Mobleyauthor=E.+Dumontauthor=J.+D.+Choderaauthor=K.+A.+Dill&title=Comparison+of+charge+models+for+fixed-charge+force+fields%3A+small-molecule+hydration+free+energies+in+explicit+solvent&doi=10.1021%2Fjp0667442"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Charge Models for Fixed-Charge Force Fields: Small-Molecule Hydration Free Energies in Explicit Solvent</span></div><div class="casAuthors">Mobley, David L.; Dumont, Elise; Chodera, John D.; Dill, Ken A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2242-2254</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In mol. simulations with fixed-charge force fields, the choice of partial at. charges influences numerous computed phys. properties, including binding free energies.  Many mol. mechanics force fields specify how nonbonded parameters should be detd., but various choices are often available for how these charges are to be detd. for arbitrary small mols.  Here, we compute hydration free energies for a set of 44 small, neutral mols. in two different explicit water models (TIP3P and TIP4P-Ew) to examine the influence of charge model on agreement with expt.  Using the AMBER GAFF force field for nonbonded parameters, we test several different methods for obtaining partial at. charges, including two fast methods exploiting semiempirical quantum calcns. and methods deriving charges from the electrostatic potentials computed with several different levels of ab initio quantum calcns. with and without a continuum reaction field treatment of solvent.  We find that the best charge sets give a root-mean-square error from expt. of roughly 1 kcal/mol.  Surprisingly, agreement with exptl. hydration free energies does not increase substantially with increasing level of quantum theory, even when the quantum calcns. are performed with a reaction field treatment to better model the aq. phase.  We also find that the semiempirical AM1-BCC method for computing charges works almost as well as any of the more computationally expensive ab initio methods and that the root-mean-square error reported here is similar to that for implicit solvent models reported in the literature.  Further, we find that the discrepancy with exptl. hydration free energies grows substantially with the polarity of the compd., as does its variation across theory levels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp00p5CsvXGx7Vg90H21EOLACvtfcHk0liQNZcOitBGaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsFWmsrc%253D&md5=5710fc0d95bead57202b33afa2093431</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjp0667442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp0667442%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DDumont%26aufirst%3DE.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DComparison%2520of%2520charge%2520models%2520for%2520fixed-charge%2520force%2520fields%253A%2520small-molecule%2520hydration%2520free%2520energies%2520in%2520explicit%2520solvent%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2007%26volume%3D111%26spage%3D2242%26epage%3D2254%26doi%3D10.1021%2Fjp0667442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Yilmazer, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korth, M.</span><span> </span><span class="NLM_article-title">Comparison of molecular mechanics, semi-empirical quantum mechanical, and density functional theory methods for scoring protein-ligand interactions</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">117</span><span class="NLM_x">, </span> <span class="NLM_fpage">8075</span><span class="NLM_x">–</span> <span class="NLM_lpage">8084</span><span class="refDoi"> DOI: 10.1021/jp402719k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp402719k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptlChur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2013&pages=8075-8084&author=N.+D.+Yilmazerauthor=M.+Korth&title=Comparison+of+molecular+mechanics%2C+semi-empirical+quantum+mechanical%2C+and+density+functional+theory+methods+for+scoring+protein-ligand+interactions&doi=10.1021%2Fjp402719k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Molecular Mechanics, Semi-Empirical Quantum Mechanical, and Density Functional Theory Methods for Scoring Protein-Ligand Interactions</span></div><div class="casAuthors">Yilmazer, Nusret Duygu; Korth, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">8075-8084</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Correctly ranking protein-ligand interactions with respect to overall free energy of binding is a grand challenge for virtual drug design.  Here the authors compare the performance of various quantum chem. approaches for tackling this so-called "scoring" problem.  Relying on systematically generated benchmark sets of large protein/ligand model complexes based on the PDBbind database, the performance depends first of all on the general level of theory.  Comparing classical mol. mechanics (MM), semiempirical quantum mech. (SQM), and d. functional theory (DFT) based methods, enhanced SQM approaches perform very similar to DFT methods and substantially different from MM potentials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5xB3KEWdidrVg90H21EOLACvtfcHk0liQNZcOitBGaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptlChur8%253D&md5=c9cc8ad508d7f791f4f153a3e73d32b1</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjp402719k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp402719k%26sid%3Dliteratum%253Aachs%26aulast%3DYilmazer%26aufirst%3DN.%2BD.%26aulast%3DKorth%26aufirst%3DM.%26atitle%3DComparison%2520of%2520molecular%2520mechanics%252C%2520semi-empirical%2520quantum%2520mechanical%252C%2520and%2520density%2520functional%2520theory%2520methods%2520for%2520scoring%2520protein-ligand%2520interactions%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2013%26volume%3D117%26spage%3D8075%26epage%3D8084%26doi%3D10.1021%2Fjp402719k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Tomasi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mennucci, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cammi, R.</span><span> </span><span class="NLM_article-title">Quantum mechanical continuum solvation models</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">2999</span><span class="NLM_x">–</span> <span class="NLM_lpage">3093</span><span class="refDoi"> DOI: 10.1021/cr9904009</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr9904009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmsVynurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2005&pages=2999-3093&author=J.+Tomasiauthor=B.+Mennucciauthor=R.+Cammi&title=Quantum+mechanical+continuum+solvation+models&doi=10.1021%2Fcr9904009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Quantum Mechanical Continuum Solvation Models</span></div><div class="casAuthors">Tomasi, Jacopo; Mennucci, Benedetta; Cammi, Roberto</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2999-3093</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkWabJEwaQ7LVg90H21EOLACvtfcHk0liATj_IYaTFKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmsVynurc%253D&md5=462420dd18b3006ee63d1298b66db247</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fcr9904009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr9904009%26sid%3Dliteratum%253Aachs%26aulast%3DTomasi%26aufirst%3DJ.%26aulast%3DMennucci%26aufirst%3DB.%26aulast%3DCammi%26aufirst%3DR.%26atitle%3DQuantum%2520mechanical%2520continuum%2520solvation%2520models%26jtitle%3DChem.%2520Rev.%26date%3D2005%26volume%3D105%26spage%3D2999%26epage%3D3093%26doi%3D10.1021%2Fcr9904009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Kloss, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heil, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kast, S. M.</span><span> </span><span class="NLM_article-title">Quantum chemistry in solution by combining 3D integral equation theory with a cluster embedding approach</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">4337</span><span class="NLM_x">–</span> <span class="NLM_lpage">4343</span><span class="refDoi"> DOI: 10.1021/jp710680m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp710680m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtlOjtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2008&pages=4337-4343&author=T.+Klossauthor=J.+Heilauthor=S.+M.+Kast&title=Quantum+chemistry+in+solution+by+combining+3D+integral+equation+theory+with+a+cluster+embedding+approach&doi=10.1021%2Fjp710680m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Quantum Chemistry in Solution by Combining 3D Integral Equation Theory with a Cluster Embedding Approach</span></div><div class="casAuthors">Kloss, Thomas; Heil, Jochen; Kast, Stefan M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4337-4343</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Free energy changes assocd. with chem. reactions in soln. are treated by integral equation theory in the form of the 3D ref. interaction site model (RISM) in combination with quantum-chem. calcns. via an embedded cluster approach (EC-RISM).  The electronic structure of the solute is computed self-consistently with the solvent structure by mapping the charge distribution of the solvent onto a set of discrete background point charges that are added to the mol. Hamiltonian.  The EC-RISM procedure yields chem. accuracy in free energy predictions for several benchmark systems without adjusting empirical parameters.  We apply the method to the std. reaction free energy for the gauche-trans equil. of 1,2-dichloroethane in water and to pKa shift calcns. for trifluoroacetic acid/acetic acid and 4-nitroaniline/aniline in water.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYZZ5wmyxtZbVg90H21EOLACvtfcHk0liATj_IYaTFKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtlOjtLg%253D&md5=3370f057079bd764b0aa70adb889a0d0</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjp710680m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp710680m%26sid%3Dliteratum%253Aachs%26aulast%3DKloss%26aufirst%3DT.%26aulast%3DHeil%26aufirst%3DJ.%26aulast%3DKast%26aufirst%3DS.%2BM.%26atitle%3DQuantum%2520chemistry%2520in%2520solution%2520by%2520combining%25203D%2520integral%2520equation%2520theory%2520with%2520a%2520cluster%2520embedding%2520approach%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2008%26volume%3D112%26spage%3D4337%26epage%3D4343%26doi%3D10.1021%2Fjp710680m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Kast, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heil, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gussregen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, K. F.</span><span> </span><span class="NLM_article-title">Prediction of tautomer ratios by embedded-cluster integral equation theory</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">353</span><span class="refDoi"> DOI: 10.1007/s10822-010-9340-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1007%2Fs10822-010-9340-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=20352296" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsFSmsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=343-353&author=S.+M.+Kastauthor=J.+Heilauthor=S.+Gussregenauthor=K.+F.+Schmidt&title=Prediction+of+tautomer+ratios+by+embedded-cluster+integral+equation+theory&doi=10.1007%2Fs10822-010-9340-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of tautomer ratios by embedded-cluster integral equation theory</span></div><div class="casAuthors">Kast, Stefan M.; Heil, Jochen; Guessregen, Stefan; Schmidt, K. Friedemann</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">343-353</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The "embedded cluster ref. interaction site model" (EC-RISM) approach combines statistical-mech. integral equation theory and quantum-chem. calcns. for predicting thermodn. data for chem. reactions in soln.  The electronic structure of the solute is detd. self-consistently with the structure of the solvent that is described by 3D RISM integral equation theory.  The continuous solvent-site distribution is mapped onto a set of discrete background charges ("embedded cluster") that represent an addnl. contribution to the mol. Hamiltonian.  The EC-RISM anal. of the SAMPL2 challenge set of tautomers proceeds in three stages.  Firstly, the group of compds. for which quant. exptl. free energy data was provided was taken to det. appropriate levels of quantum-chem. theory for geometry optimization and free energy prediction.  Secondly, the resulting workflow was applied to the full set, allowing for chem. interpretations of the results.  Thirdly, disclosure of exptl. data for parts of the compds. facilitated a detailed anal. of methodical issues and suggestions for future improvements of the model.  Without specifically adjusting parameters, the EC-RISM model yields the smallest value of the root mean square error for the first set (0.6 kcal mol-1) as well as for the full set of quant. reaction data (2.0 kcal mol-1) among the SAMPL2 participants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZaDnWt2LKubVg90H21EOLACvtfcHk0liATj_IYaTFKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsFSmsLk%253D&md5=5ef934a4e8128d7abed3d4fc7db1bba3</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1007%2Fs10822-010-9340-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-010-9340-x%26sid%3Dliteratum%253Aachs%26aulast%3DKast%26aufirst%3DS.%2BM.%26aulast%3DHeil%26aufirst%3DJ.%26aulast%3DGussregen%26aufirst%3DS.%26aulast%3DSchmidt%26aufirst%3DK.%2BF.%26atitle%3DPrediction%2520of%2520tautomer%2520ratios%2520by%2520embedded-cluster%2520integral%2520equation%2520theory%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2010%26volume%3D24%26spage%3D343%26epage%3D353%26doi%3D10.1007%2Fs10822-010-9340-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Frach, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kast, S. M.</span><span> </span><span class="NLM_article-title">Solvation effects on chemical shifts by embedded cluster integral equation theory</span> <span class="citation_source-journal">J. Phys. Chem. A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">118</span><span class="NLM_x">, </span> <span class="NLM_fpage">11620</span><span class="NLM_x">–</span> <span class="NLM_lpage">11628</span><span class="refDoi"> DOI: 10.1021/jp5084407</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp5084407" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVyhurjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2014&pages=11620-11628&author=R.+Frachauthor=S.+M.+Kast&title=Solvation+effects+on+chemical+shifts+by+embedded+cluster+integral+equation+theory&doi=10.1021%2Fjp5084407"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Solvation Effects on Chemical Shifts by Embedded Cluster Integral Equation Theory</span></div><div class="casAuthors">Frach, Roland; Kast, Stefan M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry A</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">11620-11628</span>CODEN:
                <span class="NLM_cas:coden">JPCAFH</span>;
        ISSN:<span class="NLM_cas:issn">1089-5639</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The accurate computational prediction of NMR parameters like chem. shifts represents a challenge if the species studied is immersed in strongly polarizing environments such as water.  Common approaches to treating a solvent in the form of, e.g., the polarizable continuum model (PCM) ignore strong directional interactions such as H-bonds to the solvent which can have substantial impact on magnetic shieldings.  We here present a computational methodol. that accounts for at.-level solvent effects on NMR parameters by extending the embedded cluster ref. interaction site model (EC-RISM) integral equation theory to the prediction of chem. shifts of N-methylacetamide (NMA) in aq. soln.  We examine the influence of various so-called closure approxns. of the underlying three-dimensional RISM theory as well as the impact of basis set size and different treatment of electrostatic solute-solvent interactions.  We find considerable and systematic improvement over ref. PCM and gas phase calcns.  A smaller basis set in combination with a simple point charge model already yields good performance which can be further improved by employing exact electrostatic quantum-mech. solute-solvent interaction energies.  A larger basis set benefits more significantly from exact over point charge electrostatics, which can be related to differences of the solvent's charge distribution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoh_Whnopy6mLVg90H21EOLACvtfcHk0liATj_IYaTFKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVyhurjP&md5=ffddfe146c4663c6f1b75f6f3702899b</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjp5084407&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp5084407%26sid%3Dliteratum%253Aachs%26aulast%3DFrach%26aufirst%3DR.%26aulast%3DKast%26aufirst%3DS.%2BM.%26atitle%3DSolvation%2520effects%2520on%2520chemical%2520shifts%2520by%2520embedded%2520cluster%2520integral%2520equation%2520theory%26jtitle%3DJ.%2520Phys.%2520Chem.%2520A%26date%3D2014%26volume%3D118%26spage%3D11620%26epage%3D11628%26doi%3D10.1021%2Fjp5084407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Cortes, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talpaz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kantarjian, H.</span><span> </span><span class="NLM_article-title">Ponatinib in Philadelphia chromosome-positive leukemias</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">370</span><span class="NLM_x">, </span> <span class="NLM_fpage">577</span><span class="refDoi"> DOI: 10.1056/NEJMc1315234</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1056%2FNEJMc1315234" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=24499221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFaktbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=370&publication_year=2014&pages=577&author=J.+E.+Cortesauthor=M.+Talpazauthor=H.+Kantarjian&title=Ponatinib+in+Philadelphia+chromosome-positive+leukemias&doi=10.1056%2FNEJMc1315234"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">The authors reply to comment on: Ponatinib in Philadelphia chromosome-positive leukemias</span></div><div class="casAuthors">Cortes, Jorge E.; Talpaz, Moshe; Kantarjian, Hagop</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">370</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">577</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlfmtGglm6RrVg90H21EOLACvtfcHk0lhoQBup_l0Ujw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFaktbk%253D&md5=31aed97c18eac02899b013a25b9fef3b</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1056%2FNEJMc1315234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMc1315234%26sid%3Dliteratum%253Aachs%26aulast%3DCortes%26aufirst%3DJ.%2BE.%26aulast%3DTalpaz%26aufirst%3DM.%26aulast%3DKantarjian%26aufirst%3DH.%26atitle%3DPonatinib%2520in%2520Philadelphia%2520chromosome-positive%2520leukemias%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2014%26volume%3D370%26spage%3D577%26doi%3D10.1056%2FNEJMc1315234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Pemovska, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kontro, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cronin, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McTigue, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kallioniemi, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porkka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wennerberg, K.</span><span> </span><span class="NLM_article-title">Axitinib effectively inhibits BCR-ABL1(T315I) with a distinct binding conformation</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">519</span><span class="NLM_x">, </span> <span class="NLM_fpage">102</span><span class="NLM_x">–</span> <span class="NLM_lpage">105</span><span class="refDoi"> DOI: 10.1038/nature14119</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1038%2Fnature14119" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=25686603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjt1Sntbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=519&publication_year=2015&pages=102-105&author=T.+Pemovskaauthor=E.+Johnsonauthor=M.+Kontroauthor=G.+A.+Repaskyauthor=J.+Chenauthor=P.+Wellsauthor=C.+N.+Croninauthor=M.+McTigueauthor=O.+Kallioniemiauthor=K.+Porkkaauthor=B.+W.+Murrayauthor=K.+Wennerberg&title=Axitinib+effectively+inhibits+BCR-ABL1%28T315I%29+with+a+distinct+binding+conformation&doi=10.1038%2Fnature14119"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Axitinib effectively inhibits BCR-ABL1(T315I) with a distinct binding conformation</span></div><div class="casAuthors">Pemovska, Tea; Johnson, Eric; Kontro, Mika; Repasky, Gretchen A.; Chen, Jeffrey; Wells, Peter; Cronin, Ciaran N.; McTigue, Michele; Kallioniemi, Olli; Porkka, Kimmo; Murray, Brion W.; Wennerberg, Krister</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">519</span>
        (<span class="NLM_cas:issue">7541</span>),
    <span class="NLM_cas:pages">102-105</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The BCR-ABL1 fusion gene is a driver oncogene in chronic myeloid leukemia and 30-50% of cases of adult acute lymphoblastic leukemia.  Introduction of ABL1 kinase inhibitors (for example, imatinib) has markedly improved patient survival, but acquired drug resistance remains a challenge.  Point mutations in the ABL1 kinase domain weaken inhibitor binding and represent the most common clin. resistance mechanism.  The BCR-ABL1 kinase domain gatekeeper mutation Thr315Ile (T315I) confers resistance to all approved ABL1 inhibitors except ponatinib, which has toxicity limitations.  Here we combine comprehensive drug sensitivity and resistance profiling of patient cells ex vivo with structural anal. to establish the VEGFR tyrosine kinase inhibitor axitinib as a selective and effective inhibitor for T315I-mutant BCR-ABL1-driven leukemia.  Axitinib potently inhibited BCR-ABL1(T315I), at both biochem. and cellular levels, by binding to the active form of ABL1(T315I) in a mutation-selective binding mode.  These findings suggest that the T315I mutation shifts the conformational equil. of the kinase in favor of an active (DFG-in) A-loop conformation, which has more optimal binding interactions with axitinib.  Treatment of a T315I chronic myeloid leukemia patient with axitinib resulted in a rapid redn. of T315I-pos. cells from bone marrow.  Taken together, our findings demonstrate an unexpected opportunity to repurpose axitinib, an anti-angiogenic drug approved for renal cancer, as an inhibitor for ABL1 gatekeeper mutant drug-resistant leukemia patients.  This study shows that wild-type proteins do not always sample the conformations available to disease-relevant mutant proteins and that comprehensive drug testing of patient-derived cells can identify unpredictable, clin. significant drug-repositioning opportunities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreXgtLS5A1lLVg90H21EOLACvtfcHk0lhoQBup_l0Ujw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjt1Sntbs%253D&md5=af8549ae0527bfcbda6d6e05b548f928</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2Fnature14119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14119%26sid%3Dliteratum%253Aachs%26aulast%3DPemovska%26aufirst%3DT.%26aulast%3DJohnson%26aufirst%3DE.%26aulast%3DKontro%26aufirst%3DM.%26aulast%3DRepasky%26aufirst%3DG.%2BA.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWells%26aufirst%3DP.%26aulast%3DCronin%26aufirst%3DC.%2BN.%26aulast%3DMcTigue%26aufirst%3DM.%26aulast%3DKallioniemi%26aufirst%3DO.%26aulast%3DPorkka%26aufirst%3DK.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26aulast%3DWennerberg%26aufirst%3DK.%26atitle%3DAxitinib%2520effectively%2520inhibits%2520BCR-ABL1%2528T315I%2529%2520with%2520a%2520distinct%2520binding%2520conformation%26jtitle%3DNature%26date%3D2015%26volume%3D519%26spage%3D102%26epage%3D105%26doi%3D10.1038%2Fnature14119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Leproult, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barluenga, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moras, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wurtz, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winssinger, N.</span><span> </span><span class="NLM_article-title">Cysteine mapping in conformationally distinct kinase nucleotide binding sites: application to the design of selective covalent inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1347</span><span class="NLM_x">–</span> <span class="NLM_lpage">1355</span><span class="refDoi"> DOI: 10.1021/jm101396q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101396q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFeis70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1347-1355&author=E.+Leproultauthor=S.+Barluengaauthor=D.+Morasauthor=J.+M.+Wurtzauthor=N.+Winssinger&title=Cysteine+mapping+in+conformationally+distinct+kinase+nucleotide+binding+sites%3A+application+to+the+design+of+selective+covalent+inhibitors&doi=10.1021%2Fjm101396q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Cysteine mapping in conformationally distinct kinase nucleotide binding sites: application to the design of selective covalent inhibitors</span></div><div class="casAuthors">Leproult, Emeline; Barluenga, Sofia; Moras, Dino; Wurtz, Jean-Marie; Winssinger, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1347-1355</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Kinases have emerged as one of the most prolific therapeutic targets.  An important criterion in the therapeutic success of inhibitors targeting the nucleotide binding pocket of kinases is the inhibitor residence time.  Recently, covalent kinase inhibitors have attracted attention since they confer terminal inhibition and should thus be more effective than reversible inhibitors with transient inhibition.  The most robust approach to design irreversible inhibitors is to capitalize on the nucleophilicity of a cysteine thiol group present in the target protein.  Herein, the authors report a systematic anal. of cysteine residues present in the nucleotide binding site of kinases, which could be harnessed for irreversible inhibition, taking into consideration the different kinase conformations.  The authors demonstrate the predictive power of this anal. with the design and validation of an irreversible inhibitor of KIT/PDGFR kinases.  This is the first example of a covalent kinase inhibitor that combines a pharmacophore addressing the DFG-out conformation with a covalent trap.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkzqt0gMrz3rVg90H21EOLACvtfcHk0lhoQBup_l0Ujw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFeis70%253D&md5=329ae5eaa6102b6dd34a0d1274631247</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fjm101396q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101396q%26sid%3Dliteratum%253Aachs%26aulast%3DLeproult%26aufirst%3DE.%26aulast%3DBarluenga%26aufirst%3DS.%26aulast%3DMoras%26aufirst%3DD.%26aulast%3DWurtz%26aufirst%3DJ.%2BM.%26aulast%3DWinssinger%26aufirst%3DN.%26atitle%3DCysteine%2520mapping%2520in%2520conformationally%2520distinct%2520kinase%2520nucleotide%2520binding%2520sites%253A%2520application%2520to%2520the%2520design%2520of%2520selective%2520covalent%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1347%26epage%3D1355%26doi%3D10.1021%2Fjm101396q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Erve, J. C.</span><span> </span><span class="NLM_article-title">Chemical toxicology: reactive intermediates and their role in pharmacology and toxicology</span> <span class="citation_source-journal">Expert Opin. Drug Metab. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">923</span><span class="NLM_x">–</span> <span class="NLM_lpage">946</span><span class="refDoi"> DOI: 10.1517/17425255.2.6.923</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1517%2F17425255.2.6.923" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=17125409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1eks7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=923-946&author=J.+C.+Erve&title=Chemical+toxicology%3A+reactive+intermediates+and+their+role+in+pharmacology+and+toxicology&doi=10.1517%2F17425255.2.6.923"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical toxicology: reactive intermediates and their role in pharmacology and toxicology</span></div><div class="casAuthors">Erve, John C. L.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Metabolism & Toxicology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">923-946</span>CODEN:
                <span class="NLM_cas:coden">EODMAP</span>;
        ISSN:<span class="NLM_cas:issn">1742-5255</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Reactive intermediates formed during the metab. of drugs have been investigated extensively over the past decades.  Today, interest in reactive intermediates in drug discovery is focused on minimizing bioactivation in hopes of reducing the risk of causing so-called idiosyncratic toxicity.  These efforts are justified based on the hapten hypothesis', namely, that on binding to protein, reactive intermediates may elicit an immune response to the modified protein, leading to a cascade of events that ultimately manifests as a toxic outcome.  However, the pharmacol. action of certain drugs depends on reactive intermediates that modify crit. amino acid residues of proteins, typically enzymes, thereby altering their activity.  Thus, the notion that reactive intermediates are inherently dangerous is unjustified.  When a reactive intermediate is necessary for the desired pharmacol. effect of a drug, the selectivity it displays towards the target protein is crucial, as off-target binding may produce unwanted toxicities.  On the other hand, reactive intermediates may play no role in toxicity.  This review provides a balanced perspective, primarily focusing on the proposed role of reactive intermediates in drug toxicity, while also highlighting examples in which they are involved in causing the desired pharmacol.  It is hoped that this knowledge can help scientists involved in drug discovery and development in their challenging task of producing safe and effective drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBIFLSUGUWN7Vg90H21EOLACvtfcHk0lhn_6kQPAp3qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1eks7nK&md5=320b9ade5e38e5d21b92a4e2781e9e62</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1517%2F17425255.2.6.923&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425255.2.6.923%26sid%3Dliteratum%253Aachs%26aulast%3DErve%26aufirst%3DJ.%2BC.%26atitle%3DChemical%2520toxicology%253A%2520reactive%2520intermediates%2520and%2520their%2520role%2520in%2520pharmacology%2520and%2520toxicology%26jtitle%3DExpert%2520Opin.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2006%26volume%3D2%26spage%3D923%26epage%3D946%26doi%3D10.1517%2F17425255.2.6.923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Guengerich, F. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacDonald, J. S.</span><span> </span><span class="NLM_article-title">Applying mechanisms of chemical toxicity to predict drug safety</span> <span class="citation_source-journal">Chem. Res. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">344</span><span class="NLM_x">–</span> <span class="NLM_lpage">369</span><span class="refDoi"> DOI: 10.1021/tx600260a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx600260a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhslahsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2007&pages=344-369&author=F.+P.+Guengerichauthor=J.+S.+MacDonald&title=Applying+mechanisms+of+chemical+toxicity+to+predict+drug+safety&doi=10.1021%2Ftx600260a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Applying Mechanisms of Chemical Toxicity to Predict Drug Safety</span></div><div class="casAuthors">Guengerich, F. Peter; MacDonald, James S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">344-369</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Toxicol. can no longer be used only as a science that reacts to problems but must be more proactive in predicting potential human safety issues with new drug candidates.  Success in this area must be based on an understanding of the mechanisms of toxicity.  This review summarizes and extends some of the concepts of an American Chem. Society ProSpectives meeting on the title subject held in June 2006.  One important area is the discernment of the exact nature of the most common problems in drug toxicity.  Knowledge of chem. structure alerts and relevant biol. pathways are important.  Biol. activation to reactive products and off-target pharmacol. are considered to be major contexts of drug toxicity, although defining exactly what the contributions are is not trivial.  Some newer approaches to screening for both have been developed.  A goal in predictive toxicol. is the use of in vitro methods and database development to make predictions concerning potential modes of toxicity and to stratify drug candidates for further development.  Such predictions are desirable for several economic and other reasons but are certainly not routine yet.  However, progress has been made using several approaches.  Some examples of the application of studies of wide-scale biol. responses are now available, with incorporation into development paradigms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxyWz1mKS0h7Vg90H21EOLACvtfcHk0lhn_6kQPAp3qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhslahsrk%253D&md5=d1ef7299d11c39d0ff3b031779915e54</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Ftx600260a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx600260a%26sid%3Dliteratum%253Aachs%26aulast%3DGuengerich%26aufirst%3DF.%2BP.%26aulast%3DMacDonald%26aufirst%3DJ.%2BS.%26atitle%3DApplying%2520mechanisms%2520of%2520chemical%2520toxicity%2520to%2520predict%2520drug%2520safety%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2007%26volume%3D20%26spage%3D344%26epage%3D369%26doi%3D10.1021%2Ftx600260a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Park, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boobis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldring, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenna, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laverty, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naisbitt, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicoll-Griffith, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obach, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Routledge, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tweedie, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vermeulen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baillie, T. A.</span><span> </span><span class="NLM_article-title">Managing the challenge of chemically reactive metabolites in drug development</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">292</span><span class="NLM_x">–</span> <span class="NLM_lpage">306</span><span class="refDoi"> DOI: 10.1038/nrd3408</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1038%2Fnrd3408" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=21455238" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGnu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=292-306&author=B.+K.+Parkauthor=A.+Boobisauthor=S.+Clarkeauthor=C.+E.+Goldringauthor=D.+Jonesauthor=J.+G.+Kennaauthor=C.+Lambertauthor=H.+G.+Lavertyauthor=D.+J.+Naisbittauthor=S.+Nelsonauthor=D.+A.+Nicoll-Griffithauthor=R.+S.+Obachauthor=P.+Routledgeauthor=D.+A.+Smithauthor=D.+J.+Tweedieauthor=N.+Vermeulenauthor=D.+P.+Williamsauthor=I.+D.+Wilsonauthor=T.+A.+Baillie&title=Managing+the+challenge+of+chemically+reactive+metabolites+in+drug+development&doi=10.1038%2Fnrd3408"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Managing the challenge of chemically reactive metabolites in drug development</span></div><div class="casAuthors">Park, B. Kevin; Boobis, Alan; Clarke, Stephen; Goldring, Chris E. P.; Jones, David; Kenna, J. Gerry; Lambert, Craig; Laverty, Hugh G.; Naisbitt, Dean J.; Nelson, Sidney; Nicoll-Griffith, Deborah A.; Obach, R. Scott; Routledge, Philip; Smith, Dennis A.; Tweedie, Donald J.; Vermeulen, Nico; Williams, Dominic P.; Wilson, Ian D.; Baillie, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">292-306</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The normal metab. of drugs can generate metabolites that have intrinsic chem. reactivity towards cellular mols., and therefore have the potential to alter biol. function and initiate serious adverse drug reactions.  Here, we present an assessment of the current approaches used for the evaluation of chem. reactive metabolites.  We also describe how these approaches are being used within the pharmaceutical industry to assess and minimize the potential of drug candidates to cause toxicity.  At early stages of drug discovery, iteration between medicinal chem. and drug metab. can eliminate perceived reactive metabolite-mediated chem. liabilities without compromising pharmacol. activity or the need for extensive safety evaluation beyond std. practices.  In the future, reactive metabolite evaluation may also be useful during clin. development for improving clin. risk assessment and risk management.  Currently, there remains a huge gap in our understanding of the basic mechanisms that underlie chem. stress-mediated adverse reactions in humans.  This Review summarizes our views on this complex topic, and includes insights into practices considered by the pharmaceutical industry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBv41vr7ZlNLVg90H21EOLACvtfcHk0lhn_6kQPAp3qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGnu78%253D&md5=8f6865f17bbe50b51aa4282379e82a08</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fnrd3408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3408%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DB.%2BK.%26aulast%3DBoobis%26aufirst%3DA.%26aulast%3DClarke%26aufirst%3DS.%26aulast%3DGoldring%26aufirst%3DC.%2BE.%26aulast%3DJones%26aufirst%3DD.%26aulast%3DKenna%26aufirst%3DJ.%2BG.%26aulast%3DLambert%26aufirst%3DC.%26aulast%3DLaverty%26aufirst%3DH.%2BG.%26aulast%3DNaisbitt%26aufirst%3DD.%2BJ.%26aulast%3DNelson%26aufirst%3DS.%26aulast%3DNicoll-Griffith%26aufirst%3DD.%2BA.%26aulast%3DObach%26aufirst%3DR.%2BS.%26aulast%3DRoutledge%26aufirst%3DP.%26aulast%3DSmith%26aufirst%3DD.%2BA.%26aulast%3DTweedie%26aufirst%3DD.%2BJ.%26aulast%3DVermeulen%26aufirst%3DN.%26aulast%3DWilliams%26aufirst%3DD.%2BP.%26aulast%3DWilson%26aufirst%3DI.%2BD.%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26atitle%3DManaging%2520the%2520challenge%2520of%2520chemically%2520reactive%2520metabolites%2520in%2520drug%2520development%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D292%26epage%3D306%26doi%3D10.1038%2Fnrd3408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Dungo, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keating, G. M.</span><span> </span><span class="NLM_article-title">Afatinib: first global approval</span> <span class="citation_source-journal">Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">1503</span><span class="NLM_x">–</span> <span class="NLM_lpage">1515</span><span class="refDoi"> DOI: 10.1007/s40265-013-0111-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1007%2Fs40265-013-0111-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=23982599" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVWmurnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=1503-1515&author=R.+T.+Dungoauthor=G.+M.+Keating&title=Afatinib%3A+first+global+approval&doi=10.1007%2Fs40265-013-0111-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Afatinib: First Global Approval</span></div><div class="casAuthors">Dungo, Rosselle T.; Keating, Gillian M.</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1503-1515</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">A review.  Afatinib, an irreversible inhibitor of the ErbB family of tyrosine kinases, is under development with Boehringer Ingelheim for the once-daily, oral treatment of cancer.  Afatinib downregulates ErbB signalling by covalently binding to epidermal growth factor receptor (EGFR), human epidermal growth factor receptor (HER) 2 and HER4, irreversibly inhibiting tyrosine kinase autophosphorylation.  It also inhibits transphosphorylation of HER3.  Oral afatinib (Gilotrif®) has been approved in the US for the first-line treatment of patients with metastatic non-small-cell lung cancer (NSCLC) who have tumors with EGFR exon 19 deletions or exon 21 (L858R) substitution mutations as detected by a US FDA-approved test.  Afatinib has also been approved in Taiwan for the first-line treatment of patients with EGFR mutation-pos. NSCLC.  In addn., the European Medicines Agency's Committee for Medicinal Products for Human Use has recommended the approval of afatinib (Giotrif) for the treatment of patients with locally advanced or metastatic NSCLC with activating EGFR mutations who are EGFR tyrosine kinase inhibitor naive.  Afatinib is also under regulatory review in Canada, Japan and other Asian countries.  This article summarizes the milestones in the development of afatinib, leading to this first approval in patients with metastatic NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdPcU5aIqrY7Vg90H21EOLACvtfcHk0ljlEIYai6P9SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVWmurnM&md5=21851cbfac56e2fffcba6c731f123cff</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1007%2Fs40265-013-0111-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-013-0111-6%26sid%3Dliteratum%253Aachs%26aulast%3DDungo%26aufirst%3DR.%2BT.%26aulast%3DKeating%26aufirst%3DG.%2BM.%26atitle%3DAfatinib%253A%2520first%2520global%2520approval%26jtitle%3DDrugs%26date%3D2013%26volume%3D73%26spage%3D1503%26epage%3D1515%26doi%3D10.1007%2Fs40265-013-0111-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Cameron, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanford, M.</span><span> </span><span class="NLM_article-title">Ibrutinib: first global approval</span> <span class="citation_source-journal">Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">74</span><span class="NLM_x">, </span> <span class="NLM_fpage">263</span><span class="NLM_x">–</span> <span class="NLM_lpage">271</span><span class="refDoi"> DOI: 10.1007/s40265-014-0178-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1007%2Fs40265-014-0178-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=24464309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVyjsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=263-271&author=F.+Cameronauthor=M.+Sanford&title=Ibrutinib%3A+first+global+approval&doi=10.1007%2Fs40265-014-0178-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Ibrutinib: First Global Approval</span></div><div class="casAuthors">Cameron, Fiona; Sanford, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-271</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">A review.  Ibrutinib (Imbruvica®) is a small mol., first-in-class, once-daily, orally available, Bruton's tyrosine kinase inhibitor that is under development for the treatment of B cell malignancies, including chronic lymphocytic leukemia (CLL), mantle cell lymphoma (MCL) and diffuse large B cell lymphoma (DLBCL), as well as multiple myeloma (MM), follicular lymphoma (FL) and Waldenstrom's macroglobulinemia (WM).  It has been developed by Pharmacyclics, Inc. and Janssen Biotech, Inc.  Ibrutinib acts by blocking B-cell antigen receptor signalling, thereby reducing malignant proliferation of B cells and inducing cell death.  Based chiefly on findings from a phase Ib/II study, ibrutinib has been approved in the USA for the treatment of MCL in previously treated patients and is one of the first approvals through the US FDA's Breakthrough Therapy Designation Pathway.  An application has been filed in the EU seeking regulatory approval in this indication.  In both the USA and EU, further applications have been filed with regulatory bodies seeking approval for the use of ibrutinib in patients with previously treated CLL/small lymphocytic lymphoma (SLL).  Phase III trials are underway worldwide to evaluate ibrutinib in the treatment of patients with CLL/SLL, DLBCL and MCL, and the agent is in phase II development for use in WM, FL and MM.  This article summarizes the milestones in the development of ibrutinib leading to its first approval in MCL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAwhrk_pQFtLVg90H21EOLACvtfcHk0ljlEIYai6P9SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVyjsL4%253D&md5=de4c4ee9a8b2e72c866c578ccf5db22b</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1007%2Fs40265-014-0178-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-014-0178-8%26sid%3Dliteratum%253Aachs%26aulast%3DCameron%26aufirst%3DF.%26aulast%3DSanford%26aufirst%3DM.%26atitle%3DIbrutinib%253A%2520first%2520global%2520approval%26jtitle%3DDrugs%26date%3D2014%26volume%3D74%26spage%3D263%26epage%3D271%26doi%3D10.1007%2Fs40265-014-0178-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Zambaldo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sadhu, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karthikeyan, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barluenga, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daguer, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winssinger, N.</span><span> </span><span class="NLM_article-title">Selective affinity-based probe for oncogenic kinases suitable for live cell imaging</span> <span class="citation_source-journal">Chemical Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">2088</span><span class="NLM_x">–</span> <span class="NLM_lpage">2092</span><span class="refDoi"> DOI: 10.1039/c3sc21856b</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1039%2Fc3sc21856b" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2088-2092&author=C.+Zambaldoauthor=K.+K.+Sadhuauthor=G.+Karthikeyanauthor=S.+Barluengaauthor=J.+P.+Daguerauthor=N.+Winssinger&title=Selective+affinity-based+probe+for+oncogenic+kinases+suitable+for+live+cell+imaging&doi=10.1039%2Fc3sc21856b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1039%2Fc3sc21856b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3sc21856b%26sid%3Dliteratum%253Aachs%26aulast%3DZambaldo%26aufirst%3DC.%26aulast%3DSadhu%26aufirst%3DK.%2BK.%26aulast%3DKarthikeyan%26aufirst%3DG.%26aulast%3DBarluenga%26aufirst%3DS.%26aulast%3DDaguer%26aufirst%3DJ.%2BP.%26aulast%3DWinssinger%26aufirst%3DN.%26atitle%3DSelective%2520affinity-based%2520probe%2520for%2520oncogenic%2520kinases%2520suitable%2520for%2520live%2520cell%2520imaging%26jtitle%3DChemical%2520Science%26date%3D2013%26volume%3D4%26spage%3D2088%26epage%3D2092%26doi%3D10.1039%2Fc3sc21856b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">Automatic Processing of Rotation Diffraction Data from Crystals of Initially Unknown Symmetry and Cell Constants</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">795</span><span class="NLM_x">–</span> <span class="NLM_lpage">800</span><span class="refDoi"> DOI: 10.1107/S0021889893005588</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1107%2FS0021889893005588" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaK2cXptFeltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1993&pages=795-800&author=W.+Kabsch&title=Automatic+Processing+of+Rotation+Diffraction+Data+from+Crystals+of+Initially+Unknown+Symmetry+and+Cell+Constants&doi=10.1107%2FS0021889893005588"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic processing of rotation diffraction data from crystals of initially unknown symmetry and cell constants</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">795-800</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">An algorithm was developed for the automatic interpretation of a given set of obsd. reciprocal-lattice points.  It exts. a reduced cell and assigns indexes to each reflection by a graph-theor. implementation of the local indexing method.  All possible symmetries of the obsd. lattice compatible with the metric of the reduced cell are recognized and reported, together with the unit-cell consts. and the linear index transformation relating the conventional to the reduced cell.  This algorithm was incorporated into the program XDS (K., 1988), which is now able to process single-crystal area-detector data without prior knowledge of the symmetry and the unit-cell consts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD1lV8_7cDv7Vg90H21EOLACvtfcHk0liAE8-qTqbjRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXptFeltw%253D%253D&md5=2b703225206ce99af0d7e38acf6d75b3</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1107%2FS0021889893005588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889893005588%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DAutomatic%2520Processing%2520of%2520Rotation%2520Diffraction%2520Data%2520from%2520Crystals%2520of%2520Initially%2520Unknown%2520Symmetry%2520and%2520Cell%2520Constants%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1993%26volume%3D26%26spage%3D795%26epage%3D800%26doi%3D10.1107%2FS0021889893005588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Pushing the boundaries of molecular replacement with maximum likelihood</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">1373</span><span class="NLM_x">–</span> <span class="NLM_lpage">1382</span><span class="refDoi"> DOI: 10.1107/S0907444901012471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1107%2FS0907444901012471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=11567148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFGltrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2001&pages=1373-1382&author=R.+J.+Read&title=Pushing+the+boundaries+of+molecular+replacement+with+maximum+likelihood&doi=10.1107%2FS0907444901012471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Pushing the boundaries of molecular replacement with maximum likelihood</span></div><div class="casAuthors">Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">D57</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1373-1382</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">The mol.-replacement method works well with good models and simple unit cells, but often fails with more difficult problems.  Experience with likelihood in other areas of crystallog. suggests that it would improve performance significantly.  For mol. replacement, the form of the required likelihood function depends on whether there is ambiguity in the relative phases of the contributions from symmetry-related mols. (e.g. rotation vs. translation searches).  Likelihood functions used in structure refinement are appropriate only for translation (or six-dimensional) searches, where the correct translation will place all of the atoms in the model approx. correctly.  A new likelihood function that allows for unknown relative phases is suitable for rotation searches.  It is shown that correlations between sequence identity and coordinate error can be used to calibrate parameters for model quality in the likelihood functions.  Multiple models of a mol. can be combined in a statistically valid way by setting up the joint probability distribution of the true and model structure factors as a multivariate complex normal distribution, from which the conditional distribution of the true structure factor given the models can be derived.  Tests in a new mol.-replacement program, Beast, show that the likelihood-based targets are more sensitive and more accurate than previous targets.  The new multiple-model likelihood function has a dramatic impact on success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo23lasNP9RyLVg90H21EOLACvtfcHk0liAE8-qTqbjRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFGltrc%253D&md5=b342c58edd40265324e0241ff75cec67</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1107%2FS0907444901012471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444901012471%26sid%3Dliteratum%253Aachs%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPushing%2520the%2520boundaries%2520of%2520molecular%2520replacement%2520with%2520maximum%2520likelihood%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2001%26volume%3D57%26spage%3D1373%26epage%3D1382%26doi%3D10.1107%2FS0907444901012471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Seeliger, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagar, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frank, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henderson, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span> </span><span class="NLM_article-title">c-Src binds to the cancer drug imatinib with an inactive Abl/c-Kit conformation and a distributed thermodynamic penalty</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">299</span><span class="NLM_x">–</span> <span class="NLM_lpage">311</span><span class="refDoi"> DOI: 10.1016/j.str.2007.01.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1016%2Fj.str.2007.01.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=17355866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2sXisleisbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2007&pages=299-311&author=M.+A.+Seeligerauthor=B.+Nagarauthor=F.+Frankauthor=X.+Caoauthor=M.+N.+Hendersonauthor=J.+Kuriyan&title=c-Src+binds+to+the+cancer+drug+imatinib+with+an+inactive+Abl%2Fc-Kit+conformation+and+a+distributed+thermodynamic+penalty&doi=10.1016%2Fj.str.2007.01.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">c-Src Binds to the Cancer Drug Imatinib with an Inactive Abl/c-Kit Conformation and a Distributed Thermodynamic Penalty</span></div><div class="casAuthors">Seeliger, Markus A.; Nagar, Bhushan; Frank, Filipp; Cao, Xiaoxian; Henderson, M. Nidanie; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">299-311</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The cancer drug imatinib inhibits the tyrosine kinases c-Abl, c-Kit, and the PDGF receptor.  Imatinib is less effective against c-Src, which is difficult to understand because residues interacting with imatinib in crystal structures of Abl and c-Kit are conserved in c-Src.  The crystal structure of the c-Src kinase domain in complex with imatinib closely resembles that of Abl•imatinib and c-Kit•imatinib, and differs significantly from the inactive "Src/CDK" conformation of the Src family kinases.  Attempts to increase the affinity of c-Src for imatinib by swapping residues with the corresponding residues in Abl have not been successful, suggesting that the thermodn. penalty for adoption of the imatinib-binding conformation by c-Src is distributed over a broad region of the structure.  Two mutations that are expected to destabilize the inactive Src/CDK conformation increase drug sensitivity 15-fold, suggesting that the free-energy balance between different inactive states is a key to imatinib binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2SQGjjXdg1LVg90H21EOLACvtfcHk0liAE8-qTqbjRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXisleisbY%253D&md5=ddfbcf06d91b408ca5fb25af9774c334</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2007.01.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2007.01.015%26sid%3Dliteratum%253Aachs%26aulast%3DSeeliger%26aufirst%3DM.%2BA.%26aulast%3DNagar%26aufirst%3DB.%26aulast%3DFrank%26aufirst%3DF.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DHenderson%26aufirst%3DM.%2BN.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3Dc-Src%2520binds%2520to%2520the%2520cancer%2520drug%2520imatinib%2520with%2520an%2520inactive%2520Abl%252Fc-Kit%2520conformation%2520and%2520a%2520distributed%2520thermodynamic%2520penalty%26jtitle%3DStructure%26date%3D2007%26volume%3D15%26spage%3D299%26epage%3D311%26doi%3D10.1016%2Fj.str.2007.01.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span><span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lifFnRA6A2_FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Brünger, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clore, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeLano, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gros, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuszewski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilges, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannu, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simonson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warren, G. L.</span><span> </span><span class="NLM_article-title">Crystallography & NMR system: A new software suite for macromolecular structure determination</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">905</span><span class="NLM_x">–</span> <span class="NLM_lpage">921</span><span class="refDoi"> DOI: 10.1107/S0907444998003254</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1107%2FS0907444998003254" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=9757107" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A280%3ADyaK1cvjslKhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1998&pages=905-921&author=A.+T.+Br%C3%BCngerauthor=P.+D.+Adamsauthor=G.+M.+Cloreauthor=W.+L.+DeLanoauthor=P.+Grosauthor=R.+W.+Grosse-Kunstleveauthor=J.+S.+Jiangauthor=J.+Kuszewskiauthor=M.+Nilgesauthor=N.+S.+Pannuauthor=R.+J.+Readauthor=L.+M.+Riceauthor=T.+Simonsonauthor=G.+L.+Warren&title=Crystallography+%26+NMR+system%3A+A+new+software+suite+for+macromolecular+structure+determination&doi=10.1107%2FS0907444998003254"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallography & NMR system: A new software suite for macromolecular structure determination</span></div><div class="casAuthors">Brunger A T; Adams P D; Clore G M; DeLano W L; Gros P; Grosse-Kunstleve R W; Jiang J S; Kuszewski J; Nilges M; Pannu N S; Read R J; Rice L M; Simonson T; Warren G L</div><div class="citationInfo"><span class="NLM_cas:title">Acta crystallographica. Section D, Biological crystallography</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">Pt 5</span>),
    <span class="NLM_cas:pages">905-21</span>
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    </div><div class="casAbstract">A new software suite, called Crystallography & NMR System (CNS), has been developed for macromolecular structure determination by X-ray crystallography or solution nuclear magnetic resonance (NMR) spectroscopy.  In contrast to existing structure-determination programs, the architecture of CNS is highly flexible, allowing for extension to other structure-determination methods, such as electron microscopy and solid-state NMR spectroscopy.  CNS has a hierarchical structure: a high-level hypertext markup language (HTML) user interface, task-oriented user input files, module files, a symbolic structure-determination language (CNS language), and low-level source code.  Each layer is accessible to the user.  The novice user may just use the HTML interface, while the more advanced user may use any of the other layers.  The source code will be distributed, thus source-code modification is possible.  The CNS language is sufficiently powerful and flexible that many new algorithms can be easily implemented in the CNS language without changes to the source code.  The CNS language allows the user to perform operations on data structures, such as structure factors, electron-density maps, and atomic properties.  The power of the CNS language has been demonstrated by the implementation of a comprehensive set of crystallographic procedures for phasing, density modification and refinement.  User-friendly task-oriented input files are available for nearly all aspects of macromolecular structure determination by X-ray crystallography and solution NMR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRAeLCFcGBdlx0LsK7uFuY7fW6udTcc2ebRBFX0mUkii7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1cvjslKhsw%253D%253D&md5=37afee1b07ac44d3f3730a39061b4e79</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1107%2FS0907444998003254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444998003254%26sid%3Dliteratum%253Aachs%26aulast%3DBr%25C3%25BCnger%26aufirst%3DA.%2BT.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DClore%26aufirst%3DG.%2BM.%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26aulast%3DGros%26aufirst%3DP.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DJiang%26aufirst%3DJ.%2BS.%26aulast%3DKuszewski%26aufirst%3DJ.%26aulast%3DNilges%26aufirst%3DM.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRice%26aufirst%3DL.%2BM.%26aulast%3DSimonson%26aufirst%3DT.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26atitle%3DCrystallography%2520%2526%2520NMR%2520system%253A%2520A%2520new%2520software%2520suite%2520for%2520macromolecular%2520structure%2520determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1998%26volume%3D54%26spage%3D905%26epage%3D921%26doi%3D10.1107%2FS0907444998003254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span><span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lifFnRA6A2_FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Laskowski, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macarthur, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moss, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J. M.</span><span> </span><span class="NLM_article-title">Procheck - a Program to Check the Stereochemical Quality of Protein Structures</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">283</span><span class="NLM_x">–</span> <span class="NLM_lpage">291</span><span class="refDoi"> DOI: 10.1107/S0021889892009944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1107%2FS0021889892009944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1993&pages=283-291&author=R.+A.+Laskowskiauthor=M.+W.+Macarthurauthor=D.+S.+Mossauthor=J.+M.+Thornton&title=Procheck+-+a+Program+to+Check+the+Stereochemical+Quality+of+Protein+Structures&doi=10.1107%2FS0021889892009944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">PROCHECK: a program to check the stereochemical quality of protein structures</span></div><div class="casAuthors">Laskowski, Roman A.; MacArthur, Malcolm W.; Moss, David S.; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">283-91</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">The PROCHECK suite of programs provides a detailed check on the stereochem. of a protein structure.  Its outputs comprise a no. of plots in PostScript format and a comprehensive residue-by-residue listing.  These give an assessment of the overall quality of the structure as compared with well-refined structures of the same resoln. and also highlight regions that may need further investigation.  The PROCHECK programs are useful for assessing the quality not only of protein structures in the process of being solved but also of existing structures and of those being modeled on known structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCe5FgJf4KH7Vg90H21EOLACvtfcHk0lhXC8lCYC5how"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D&md5=fb69fc4410cd716aaaa7cc0db06b3ed2</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1107%2FS0021889892009944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889892009944%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DMacarthur%26aufirst%3DM.%2BW.%26aulast%3DMoss%26aufirst%3DD.%2BS.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DProcheck%2520-%2520a%2520Program%2520to%2520Check%2520the%2520Stereochemical%2520Quality%2520of%2520Protein%2520Structures%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1993%26volume%3D26%26spage%3D283%26epage%3D291%26doi%3D10.1107%2FS0021889892009944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">DeLano, W. L.</span>The PyMOL Molecular Graphics System; <span class="NLM_publisher-name">DeLano Scientific</span>: <span class="NLM_publisher-loc">San Carlos, CA</span>, 2002. <a href="http://www.pymol.org/" class="extLink">http://www.pymol.org/</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=DeLano%2C+W.+L.The+PyMOL+Molecular+Graphics+System%3B+DeLano+Scientific%3A+San+Carlos%2C+CA%2C+2002.+http%3A%2F%2Fwww.pymol.org%2F."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DDeLano%2520Scientific" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group">Avdeev, A.</span><span> </span><span class="NLM_article-title">High-throughput measurements of solubility profiles</span> <span class="citation_source-journal">Pharmacokinetic Optimization in Drug Research</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_fpage">305</span><span class="NLM_x">–</span> <span class="NLM_lpage">325</span><span class="refDoi"> DOI: 10.1002/9783906390437.ch18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1002%2F9783906390437.ch18" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2001&pages=305-325&author=A.+Avdeev&title=High-throughput+measurements+of+solubility+profiles&doi=10.1002%2F9783906390437.ch18"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1002%2F9783906390437.ch18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F9783906390437.ch18%26sid%3Dliteratum%253Aachs%26aulast%3DAvdeev%26aufirst%3DA.%26atitle%3DHigh-throughput%2520measurements%2520of%2520solubility%2520profiles%26jtitle%3DPharmacokinetic%2520Optimization%2520in%2520Drug%2520Research%26date%3D2001%26spage%3D305%26epage%3D325%26doi%3D10.1002%2F9783906390437.ch18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group">Zhang, Z. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, A. G.</span><span> </span><span class="NLM_article-title">A universal algorithm for fast and automated charge state deconvolution of electrospray mass-to-charge ratio spectra</span> <span class="citation_source-journal">J. Am. Soc. Mass Spectrom.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">225</span><span class="NLM_x">–</span> <span class="NLM_lpage">233</span><span class="refDoi"> DOI: 10.1016/S1044-0305(97)00284-5</span> </span><div class="citationLinks">[<a href="/doi/10.1016/S1044-0305%2897%2900284-5" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaK1cXhtlynsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1998&pages=225-233&author=Z.+Q.+Zhangauthor=A.+G.+Marshall&title=A+universal+algorithm+for+fast+and+automated+charge+state+deconvolution+of+electrospray+mass-to-charge+ratio+spectra&doi=10.1016%2FS1044-0305%2897%2900284-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">A universal algorithm for fast and automated charge state deconvolution of electrospray mass-to-charge ratio spectra</span></div><div class="casAuthors">Zhang, Zhongqi; Marshall, Alan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Society for Mass Spectrometry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">225-233</span>CODEN:
                <span class="NLM_cas:coden">JAMSEF</span>;
        ISSN:<span class="NLM_cas:issn">1044-0305</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">This article describes a new algorithm for charge state detn. and deconvolution of electrospray ionization (ESI) mass-to-charge ratio spectra.  The algorithm (ZSCORE) is based on a charge scoring scheme that incorporates all above-threshold members of a family of charge states or isotopic components, and deconvolves both low- and high-resoln. mass-to-charge ratio spectra, with or without a peak list (stick plot).  A scoring wt. factor, log (I/I0), in which I is the signal magnitude at a calcd. mass-to-charge ratio, and I0 is the signal threshold near that mass-to-charge ratio, was used in most cases.  For high-resoln. mass-to-charge ratio spectra in which all isotopic peaks are resolved, the algorithm can deconvolve overlapped isotopic multiplets of the same of different charge state.  Compared to other deconvolution techniques, the algorithm is robust, rapid, and fully automated (i.e., no user input during the deconvolution process).  It eliminates artifact peaks without introducing peak distortions.  Its performance is demonstrated for exptl. ESI Fourier transform ICR mass-to-charge ratio spectra (both low and high resoln.).  Charge state deconvolution to yield a zero-charge mass spectrum should prove particularly useful for interpreting spectra of complex mixts., identifying contaminants, noncovalent adducts, fragments (N-terminal, C-terminal, internal), and chem. modifications of electrosprayed biomacromols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-7hFdAcR_BbVg90H21EOLACvtfcHk0lhXC8lCYC5how"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhtlynsbo%253D&md5=5bee6ce6556beed4ee9004cfe6c7ecad</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2FS1044-0305%2897%2900284-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1044-0305%252897%252900284-5%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%2BQ.%26aulast%3DMarshall%26aufirst%3DA.%2BG.%26atitle%3DA%2520universal%2520algorithm%2520for%2520fast%2520and%2520automated%2520charge%2520state%2520deconvolution%2520of%2520electrospray%2520mass-to-charge%2520ratio%2520spectra%26jtitle%3DJ.%2520Am.%2520Soc.%2520Mass%2520Spectrom.%26date%3D1998%26volume%3D9%26spage%3D225%26epage%3D233%26doi%3D10.1016%2FS1044-0305%2897%2900284-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group">Milletti, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storchi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sforna, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span> </span><span class="NLM_article-title">New and original pKa prediction method using grid molecular interaction fields</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2172</span><span class="NLM_x">–</span> <span class="NLM_lpage">2181</span><span class="refDoi"> DOI: 10.1021/ci700018y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci700018y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFSnsLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=2172-2181&author=F.+Millettiauthor=L.+Storchiauthor=G.+Sfornaauthor=G.+Cruciani&title=New+and+original+pKa+prediction+method+using+grid+molecular+interaction+fields&doi=10.1021%2Fci700018y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">New and Original pKa Prediction Method Using Grid Molecular Interaction Fields</span></div><div class="casAuthors">Milletti, Francesca; Storchi, Loriano; Sforna, Gianluca; Cruciani, Gabriele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2172-2181</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One of the most important physicochem. properties of a mol. is pKa.  It is known that two parameters imperative in ADME profiling, soly., and lipophilicity are governed by pKa, and receptor binding can be influenced by pKa.  Because most drugs are ionized in physiol. conditions, pKa is particularly relevant to medicinal chem.  Despite the numerous advances in high-throughput measurements, in silico detn. is still the fastest and cheapest way of obtaining pKa.  This paper presents a new original computational method for pKa prediction of org. compds.  Descriptors were generated using the program GRID, and these descriptors are based on mol. interaction fields precomputed on a set of mol. fragments.  The new method was developed, trained, and cross-validated by using a large and diverse data set of 24 617 pKa values.  This paper presents the results for a class of 421 acidic nitrogen compds. (RMSE = 0.41, r2 = 0.97, q2 = 0.87) and for a class of 947 six-membered N-heterocyclic bases (RMSE = 0.60, r2 = 0.93, q2 = 0.85).  For external validation 28 novel compds. were selected that covered nine different ionizable groups, and 39 pKa values could be exptl. detd. by spectral gradient anal. (SGA).  Comparison of exptl. pKa with calcd. pKa demonstrated that the predictive ability of the method is good (external set, r2 = 0.85, RMSE = 0.90).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzEk0uiS1E6rVg90H21EOLACvtfcHk0lhXC8lCYC5how"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFSnsLrF&md5=f2b1dec853ccb0ef1a40540d234232fb</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fci700018y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700018y%26sid%3Dliteratum%253Aachs%26aulast%3DMilletti%26aufirst%3DF.%26aulast%3DStorchi%26aufirst%3DL.%26aulast%3DSforna%26aufirst%3DG.%26aulast%3DCruciani%26aufirst%3DG.%26atitle%3DNew%2520and%2520original%2520pKa%2520prediction%2520method%2520using%2520grid%2520molecular%2520interaction%2520fields%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D2172%26epage%3D2181%26doi%3D10.1021%2Fci700018y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group">Milletti, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storchi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sforna, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cross, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span> </span><span class="NLM_article-title">Tautomer enumeration and stability prediction for virtual screening on large chemical databases</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">68</span><span class="NLM_x">–</span> <span class="NLM_lpage">75</span><span class="refDoi"> DOI: 10.1021/ci800340j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800340j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisVahug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=68-75&author=F.+Millettiauthor=L.+Storchiauthor=G.+Sfornaauthor=S.+Crossauthor=G.+Cruciani&title=Tautomer+enumeration+and+stability+prediction+for+virtual+screening+on+large+chemical+databases&doi=10.1021%2Fci800340j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Tautomer enumeration and stability prediction for virtual screening on large chemical databases</span></div><div class="casAuthors">Milletti, Francesca; Storchi, Loriano; Sforna, Gianluca; Cross, Simon; Cruciani, Gabriele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">68-75</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tautomeric rearrangements affect the results of cheminformatics applications that depend on the knowledge of the 2D or 3D structure of a compd., such as tools for database searches, fingerprint generation, virtual screening, and phys.-chem. properties prediction.  In this paper we present TauThor, a tool to enumerate tautomers and predict tautomer stability in the aq. medium.  The enumeration is based on a recursive process that generates tautomers according to the general scheme HX-Y=Z .rdblhar. X=Y-ZH.  The stability of a tautomer is calcd. by using a library of 145 fragments assocd. with exptl. tautomeric percentages in water and a pKa based-method that utilizes pKa values predicted by MoKa.  Predicted tautomeric ratios based on pKa calcns. were benchmarked against literature data for a set of eleven compds.  The FDA approved drugs database, the NCI database and two vendor databases - Specs Screening Library and Asinex Gold Collection - were used to illustrate the impact of tautomerism on chem. libraries and to evaluate the relative occurrences of alternative tautomeric forms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYlrU_55G2ArVg90H21EOLACvtfcHk0liBrC4GuoNufQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisVahug%253D%253D&md5=eac540b04bcf7642cd3904c084504eb4</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Fci800340j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800340j%26sid%3Dliteratum%253Aachs%26aulast%3DMilletti%26aufirst%3DF.%26aulast%3DStorchi%26aufirst%3DL.%26aulast%3DSforna%26aufirst%3DG.%26aulast%3DCross%26aufirst%3DS.%26aulast%3DCruciani%26aufirst%3DG.%26atitle%3DTautomer%2520enumeration%2520and%2520stability%2520prediction%2520for%2520virtual%2520screening%2520on%2520large%2520chemical%2520databases%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D68%26epage%3D75%26doi%3D10.1021%2Fci800340j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group">Case, D. A.; Darden, T. A.; Cheatham, T. E.; Simmerling, C. L.; Wang, J.; Duke, R. E.; Luo, R.; Walker, R. C.; Zhang, W.; Merz, K. M.; Roberts, B.; Hayik, S.; Roitberg, A.; Seabra, G.; Swails, J.; Goetz, A. W.; Kolossváry, I.; Wong, K. F.; Paesani, F.; Vanicek, J.; Wolf, R. M.; Liu, J.; Wu, X.; Brozell, S. R.; Steinbrecher, T.; Gohlke, H.; Cai, Q.; Ye, X.; Wang, J.; Hsieh, M. J.; Cui, G.; Roe, D. R.; Mathews, D. H.; Seetin, M. G.; Solomon-Ferrer, R.; Sagui, C.; Babin, V.; Luchko, T.; Gusarov, S.; Kovalenko, A.; Kollman, P. A.</span>Amber12; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=D.+A.+Case&author=T.+A.+Darden&author=T.+E.+Cheatham&author=C.+L.+Simmerling&author=J.+Wang&author=R.+E.+Duke&author=R.+Luo&author=R.+C.+Walker&author=W.+Zhang&author=K.+M.+Merz&author=B.+Roberts&author=S.+Hayik&author=A.+Roitberg&author=G.+Seabra&author=J.+Swails&author=A.+W.+Goetz&author=I.+Kolossv%C3%A1ry&author=K.+F.+Wong&author=F.+Paesani&author=J.+Vanicek&author=R.+M.+Wolf&author=J.+Liu&author=X.+Wu&author=S.+R.+Brozell&author=T.+Steinbrecher&author=H.+Gohlke&author=Q.+Cai&author=X.+Ye&author=J.+Wang&author=M.+J.+Hsieh&author=G.+Cui&author=D.+R.+Roe&author=D.+H.+Mathews&author=M.+G.+Seetin&author=R.+Solomon-Ferrer&author=C.+Sagui&author=V.+Babin&author=T.+Luchko&author=S.+Gusarov&author=A.+Kovalenko&author=P.+A.+Kollman&title=Amber12"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26jtitle%3DAmber12%26pub%3DUniversity%2520of%2520California%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group">Sigalov, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fenley, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Onufriev, A.</span><span> </span><span class="NLM_article-title">Analytical electrostatics for biomolecules: beyond the generalized Born approximation</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">124902</span><span class="refDoi"> DOI: 10.1063/1.2177251</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1063%2F1.2177251" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=16599720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsVGisL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2006&pages=124902&author=G.+Sigalovauthor=A.+Fenleyauthor=A.+Onufriev&title=Analytical+electrostatics+for+biomolecules%3A+beyond+the+generalized+Born+approximation&doi=10.1063%2F1.2177251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Analytical electrostatics for biomolecules: Beyond the generalized Born approximation</span></div><div class="casAuthors">Sigalov, Grigori; Fenley, Andrew; Onufriev, Alexey</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">124902/1-124902/14</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The modeling and simulation of macromols. in soln. often benefits from fast anal. approxns. for the electrostatic interactions.  In our previous work, we proposed a method based on an approx. anal. soln. of the linearized Poisson-Boltzmann equation for a sphere.  In the current work, we extend the method to biomols. of arbitrary shape and provide computationally efficient algorithms for estn. of the parameters of the model.  This approach, which we tentatively call ALPB here, is tested against the std. numerical Poisson-Boltzmann (NPB) treatment on a set of 579 representative proteins, nucleic acids, and small peptides.  The tests are performed across a wide range of solvent/solute dielecs. and at biol. relevant salt concns.  Over the range of the solvent and solute parameters tested, the systematic deviation (from the NPB ref.) of solvation energies computed by ALPB is 0.5-3.5 kcal/mol, which is 5-50 times smaller than that of the conventional generalized Born approxn. widely used in this context.  At the same time, ALPB is equally computationally efficient.  The new model is incorporated into the AMBER mol. modeling package and tested on small proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSvnn4dOEBabVg90H21EOLACvtfcHk0liBrC4GuoNufQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsVGisL8%253D&md5=abdf348673236503a15ec13e7493bd17</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1063%2F1.2177251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.2177251%26sid%3Dliteratum%253Aachs%26aulast%3DSigalov%26aufirst%3DG.%26aulast%3DFenley%26aufirst%3DA.%26aulast%3DOnufriev%26aufirst%3DA.%26atitle%3DAnalytical%2520electrostatics%2520for%2520biomolecules%253A%2520beyond%2520the%2520generalized%2520Born%2520approximation%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2006%26volume%3D124%26spage%3D124902%26doi%3D10.1063%2F1.2177251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group">Andersen, H. C.</span><span> </span><span class="NLM_article-title">Molecular-Dynamics Simulations at Constant Pressure and-or Temperature</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1980</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">2384</span><span class="NLM_x">–</span> <span class="NLM_lpage">2393</span><span class="refDoi"> DOI: 10.1063/1.439486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1063%2F1.439486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaL3cXhs12msbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=1980&pages=2384-2393&author=H.+C.+Andersen&title=Molecular-Dynamics+Simulations+at+Constant+Pressure+and-or+Temperature&doi=10.1063%2F1.439486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics simulations at constant pressure and/or temperature</span></div><div class="casAuthors">Andersen, Hans C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2384-93</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">In the mol. dynamics simulation method for fluids, the equations of motion for a collection of particles in a fixed vol. are solved numerically.  The energy, vol., and no. of particles are const. for a particular simulation, and it is assumed that time avs. of properties of the simulated fluid are equal to microcanonical ensemble avs. of the same properties.  In some situations, it is desirable to perform simulations of a fluid for particular values of temp. and/or pressure or under conditions in which the energy and vol. of the fluid can fluctuate.  Three methods for performing mol. dynamics simulations under conditions of const. temp. and/or pressure are discussed.  Time avs. of properties of the simulated fluid are equal to avs. over the isenthalpic-isobaric, canonical, and isothermal-isobaric ensembles. Each method is a way of describing the dynamics of a certain no. of particles in a vol. element of a fluid while taking into account the influence of surrounding particles in changing the energy and/or d. of the simulated vol. element.  The influence of the surroundings is taken into account without unwanted surface effects.  Examples of situations where these methods may be useful are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGof3p6WlEFZTbVg90H21EOLACvtfcHk0liBrC4GuoNufQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXhs12msbo%253D&md5=54e4faa1c0ea4385dc565d8db5a31edc</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1063%2F1.439486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.439486%26sid%3Dliteratum%253Aachs%26aulast%3DAndersen%26aufirst%3DH.%2BC.%26atitle%3DMolecular-Dynamics%2520Simulations%2520at%2520Constant%2520Pressure%2520and-or%2520Temperature%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1980%26volume%3D72%26spage%3D2384%26epage%3D2393%26doi%3D10.1063%2F1.439486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group">Van der Spoel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindahl, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groenhof, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mark, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J. C.</span><span> </span><span class="NLM_article-title">GROMACS: Fast, flexible, and free</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1701</span><span class="NLM_x">–</span> <span class="NLM_lpage">1718</span><span class="refDoi"> DOI: 10.1002/jcc.20291</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1002%2Fjcc.20291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=16211538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1701-1718&author=D.+Van+der+Spoelauthor=E.+Lindahlauthor=B.+Hessauthor=G.+Groenhofauthor=A.+E.+Markauthor=H.+J.+C.+Berendsen&title=GROMACS%3A+Fast%2C+flexible%2C+and+free&doi=10.1002%2Fjcc.20291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS: Fast, flexible, and free</span></div><div class="casAuthors">Van Der Spoel, David; Lindahl, Erik; Hess, Berk; Groenhof, Gerrit; Mark, Alan E.; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1701-1718</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This article describes the software suite GROMACS (Groningen MAchine for Chem. Simulation) that was developed at the University of Groningen, The Netherlands, in the early 1990s.  The software, written in ANSI C, originates from a parallel hardware project, and is well suited for parallelization on processor clusters.  By careful optimization of neighbor searching and of inner loop performance, GROMACS is a very fast program for mol. dynamics simulation.  It does not have a force field of its own, but is compatible with GROMOS, OPLS, AMBER, and ENCAD force fields.  In addn., it can handle polarizable shell models and flexible constraints.  The program is versatile, as force routines can be added by the user, tabulated functions can be specified, and analyses can be easily customized.  Nonequil. dynamics and free energy detns. are incorporated.  Interfaces with popular quantum-chem. packages (MOPAC, GAMES-UK, GAUSSIAN) are provided to perform mixed MM/QM simulations.  The package includes about 100 utility and anal. programs.  GROMACS is in the public domain and distributed (with source code and documentation) under the GNU General Public License.  It is maintained by a group of developers from the Universities of Groningen, Uppsala, and Stockholm, and the Max Planck Institute for Polymer Research in Mainz.  Its Web site is http://www.gromacs.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxInIDyt9Y8bVg90H21EOLACvtfcHk0lisTG19i0S_JA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO&md5=4aa71b1dc0b5b4978a88b0568245a265</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20291%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2Bder%2BSpoel%26aufirst%3DD.%26aulast%3DLindahl%26aufirst%3DE.%26aulast%3DHess%26aufirst%3DB.%26aulast%3DGroenhof%26aufirst%3DG.%26aulast%3DMark%26aufirst%3DA.%2BE.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DGROMACS%253A%2520Fast%252C%2520flexible%252C%2520and%2520free%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1701%26epage%3D1718%26doi%3D10.1002%2Fjcc.20291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group">Jarvis, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patrick, E. A.</span><span> </span><span class="NLM_article-title">Clustering Using a Similarity Measure Based on Shared near Neighbors</span> <span class="citation_source-journal">IEEE Trans. Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">C-22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1025</span><span class="NLM_x">–</span> <span class="NLM_lpage">1034</span><span class="refDoi"> DOI: 10.1109/T-C.1973.223640</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1109%2FT-C.1973.223640" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=C-22&publication_year=1973&pages=1025-1034&author=R.+A.+Jarvisauthor=E.+A.+Patrick&title=Clustering+Using+a+Similarity+Measure+Based+on+Shared+near+Neighbors&doi=10.1109%2FT-C.1973.223640"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1109%2FT-C.1973.223640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1109%252FT-C.1973.223640%26sid%3Dliteratum%253Aachs%26aulast%3DJarvis%26aufirst%3DR.%2BA.%26aulast%3DPatrick%26aufirst%3DE.%2BA.%26atitle%3DClustering%2520Using%2520a%2520Similarity%2520Measure%2520Based%2520on%2520Shared%2520near%2520Neighbors%26jtitle%3DIEEE%2520Trans.%2520Comput.%26date%3D1973%26volume%3DC-22%26spage%3D1025%26epage%3D1034%26doi%3D10.1109%2FT-C.1973.223640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group">Becke, A. D.</span><span> </span><span class="NLM_article-title">Density-functional exchange-energy approximation with correct asymptotic behavior</span> <span class="citation_source-journal">Phys. Rev. A: At., Mol., Opt. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">3098</span><span class="NLM_x">–</span> <span class="NLM_lpage">3100</span><span class="refDoi"> DOI: 10.1103/PhysRevA.38.3098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1103%2FPhysRevA.38.3098" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=9900728" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaL1cXmtlOhsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1988&pages=3098-3100&author=A.+D.+Becke&title=Density-functional+exchange-energy+approximation+with+correct+asymptotic+behavior&doi=10.1103%2FPhysRevA.38.3098"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Density-functional exchange-energy approximation with correct asymptotic behavior</span></div><div class="casAuthors">Becke, A. D.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review A:  Atomic, Molecular, and Optical Physics</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3098-100</span>CODEN:
                <span class="NLM_cas:coden">PLRAAN</span>;
        ISSN:<span class="NLM_cas:issn">0556-2791</span>.
    </div><div class="casAbstract">Current gradient-cor. d.-functional approxns. for the exchange energies of at. and mol. systems fail to reproduce the correct 1/r asymptotic behavior of the exchange-energy d.  A gradient-cor. exchange-energy functional is given with the proper asymptotic limit.  This functional, contg. only one parameter, fits the exact Hartree-Fock exchange energies of a wide variety of at. systems with remarkable accuracy, surpassing the performance of previous functionals contg. two parameters or more.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKr-bPUej6TLVg90H21EOLACvtfcHk0lisTG19i0S_JA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXmtlOhsLo%253D&md5=d4d219c134a5a90f689a8abed04d82cc</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1103%2FPhysRevA.38.3098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevA.38.3098%26sid%3Dliteratum%253Aachs%26aulast%3DBecke%26aufirst%3DA.%2BD.%26atitle%3DDensity-functional%2520exchange-energy%2520approximation%2520with%2520correct%2520asymptotic%2520behavior%26jtitle%3DPhys.%2520Rev.%2520A%253A%2520At.%252C%2520Mol.%252C%2520Opt.%2520Phys.%26date%3D1988%26volume%3D38%26spage%3D3098%26epage%3D3100%26doi%3D10.1103%2FPhysRevA.38.3098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group">Lee, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parr, R. G.</span><span> </span><span class="NLM_article-title">Development of the Colle-Salvetti correlation-energy formula into a functional of the electron density</span> <span class="citation_source-journal">Phys. Rev. B: Condens. Matter Mater. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">785</span><span class="NLM_x">–</span> <span class="NLM_lpage">789</span><span class="refDoi"> DOI: 10.1103/PhysRevB.37.785</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1103%2FPhysRevB.37.785" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=9944570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaL1cXktFWrtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1988&pages=785-789&author=C.+Leeauthor=W.+Yangauthor=R.+G.+Parr&title=Development+of+the+Colle-Salvetti+correlation-energy+formula+into+a+functional+of+the+electron+density&doi=10.1103%2FPhysRevB.37.785"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Development of the Colle-Salvetti correlation-energy formula into a functional of the electron density</span></div><div class="casAuthors">Lee, Chengteh; Yang, Weitao; Parr, Robert G.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review B:  Condensed Matter and Materials Physics</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">785-9</span>CODEN:
                <span class="NLM_cas:coden">PRBMDO</span>;
        ISSN:<span class="NLM_cas:issn">0163-1829</span>.
    </div><div class="casAbstract">A correlation-energy formula due to R. Colle and D. Salvetti (1975), in which the correlation energy d. is expressed in terms of the electron d. and a Laplacian of the 2nd-order Hartree-Fock d. matrix, is restated as a formula involving the d. and local kinetic-energy d.  On insertion of gradient expansions for the local kinetic-energy d., d.-functional formulas for the correlation energy and correlation potential are then obtained.  Through numerical calcns. on a no. of atoms, pos. ions, and mols., of both open- and closed-shell type, it is demonstrated that these formulas, like the original Colle-Salvetti formulas, give correlation energies within a few percent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8yJ1cBkpi67Vg90H21EOLACvtfcHk0lisTG19i0S_JA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXktFWrtbw%253D&md5=ee7b59267a2ff72e15171a481819ccf8</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1103%2FPhysRevB.37.785&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevB.37.785%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DParr%26aufirst%3DR.%2BG.%26atitle%3DDevelopment%2520of%2520the%2520Colle-Salvetti%2520correlation-energy%2520formula%2520into%2520a%2520functional%2520of%2520the%2520electron%2520density%26jtitle%3DPhys.%2520Rev.%2520B%253A%2520Condens.%2520Matter%2520Mater.%2520Phys.%26date%3D1988%26volume%3D37%26spage%3D785%26epage%3D789%26doi%3D10.1103%2FPhysRevB.37.785" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group">Frisch, M. J.; Trucks, G. W.; Schlegel, H. B.; Scuseria, G. E.; Robb, M. A.; Cheeseman, J. R.; Scalmani, G.; Barone, V.; Mennucci, B.; Petersson, G. A.; Nakatsuji, H.; Caricato, M.; Li, X.; Hratchian, H. P.; Izmaylov, A. F.; Bloino, J.; Zheng, G.; Sonnenberg, J. L.; Hada, M.; Ehara, M.; Toyota, K.; Fukuda, R.; Hasegawa, J.; Ishida, M.; Nakajima, T.; Honda, Y.; Kitao, O.; Nakai, H.; Vreven, T.; Montgomery, J. A.,  Jr.; Peralta, J. E.; Ogliaro, F.; Bearpark, M.; Heyd, J. J.; Brothers, E.; Kudin, K. N.; Staroverov, V. N.; Kobayashi, R.; Normand, J.; Raghavachari, K.; Rendell, A.; Burant, J. C.; Iyengar, S. S.; Tomasi, J.; Cossi, M.; Rega, N.; Millam, J. M.; Klene, M.; Knox, J. E.; Cross, J. B.; Bakken, V.; Adamo, C.; Jaramillo, J.; Gomperts, R.; Stratmann, R. E.; Yazyev, O.; Austin, A. J.; Cammi, R.; Pomelli, C.; Ochterski, J. W.; Martin, R. L.; Morokuma, K.; Zakrzewski, V. G.; Voth, G. A.; Salvador, P.; Dannenberg, J. J.; Dapprich, S.; Daniels, A. D.; Farkas, O.; Foresman, J. B.; Ortiz, J. V.; Cioslowski, J.; Fox, D. J.</span>Gaussian 09; <span class="NLM_publisher-name">Gaussian, Inc.</span>: <span class="NLM_publisher-loc">Wallingford, CT</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=M.+J.+Frisch&author=G.+W.+Trucks&author=H.+B.+Schlegel&author=G.+E.+Scuseria&author=M.+A.+Robb&author=J.+R.+Cheeseman&author=G.+Scalmani&author=V.+Barone&author=B.+Mennucci&author=G.+A.+Petersson&author=H.+Nakatsuji&author=M.+Caricato&author=X.+Li&author=H.+P.+Hratchian&author=A.+F.+Izmaylov&author=J.+Bloino&author=G.+Zheng&author=J.+L.+Sonnenberg&author=M.+Hada&author=M.+Ehara&author=K.+Toyota&author=R.+Fukuda&author=J.+Hasegawa&author=M.+Ishida&author=T.+Nakajima&author=Y.+Honda&author=O.+Kitao&author=H.+Nakai&author=T.+Vreven&author=J.+A.+Montgomery&author=J.+E.+Peralta&author=F.+Ogliaro&author=M.+Bearpark&author=J.+J.+Heyd&author=E.+Brothers&author=K.+N.+Kudin&author=V.+N.+Staroverov&author=R.+Kobayashi&author=J.+Normand&author=K.+Raghavachari&author=A.+Rendell&author=J.+C.+Burant&author=S.+S.+Iyengar&author=J.+Tomasi&author=M.+Cossi&author=N.+Rega&author=J.+M.+Millam&author=M.+Klene&author=J.+E.+Knox&author=J.+B.+Cross&author=V.+Bakken&author=C.+Adamo&author=J.+Jaramillo&author=R.+Gomperts&author=R.+E.+Stratmann&author=O.+Yazyev&author=A.+J.+Austin&author=R.+Cammi&author=C.+Pomelli&author=J.+W.+Ochterski&author=R.+L.+Martin&author=K.+Morokuma&author=V.+G.+Zakrzewski&author=G.+A.+Voth&author=P.+Salvador&author=J.+J.+Dannenberg&author=S.+Dapprich&author=A.+D.+Daniels&author=O.+Farkas&author=J.+B.+Foresman&author=J.+V.+Ortiz&author=J.+Cioslowski&author=D.+J.+Fox&title=Gaussian+09"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26jtitle%3DGaussian%252009%26pub%3DGaussian%252C%2520Inc%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group">Ditchfield, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hehre, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pople, J. A.</span><span> </span><span class="NLM_article-title">Self-Consistent Molecular-Orbital Methods 0.9. Extended Gaussian-Type Basis for Molecular-Orbital Studies of Organic Molecules</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1971</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">724</span><span class="NLM_x">–</span> <span class="NLM_lpage">728</span><span class="refDoi"> DOI: 10.1063/1.1674902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1063%2F1.1674902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaE3MXksFOiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1971&pages=724-728&author=R.+Ditchfieldauthor=W.+J.+Hehreauthor=J.+A.+Pople&title=Self-Consistent+Molecular-Orbital+Methods+0.9.+Extended+Gaussian-Type+Basis+for+Molecular-Orbital+Studies+of+Organic+Molecules&doi=10.1063%2F1.1674902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Self-consistent molecular-orbital methods.  IX.  Extended Gaussian-type basis for molecular-orbital studies of organic molecules</span></div><div class="casAuthors">Ditchfield, R.; Hehre, Warren J.; Pople, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1971</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">724-8</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An extended basis set of at. functions expressed as fixed linear combinations of Gaussian functions is presented for H and the first-row atoms C to F.  In this set. described as 4-31 G, each inner shell is represented by a single basis function taken as a sum of 4 Gaussians, and each valence orbital is split into inner and outer parts described by 3 and 1 Gaussian function, resp.  The expansion coeffs. and Gaussian exponents are detd. by minimizing the total calcd. energy of the at. ground state.  This basis set is then used in single-determinant MO studies of a group of small polyat. mols.  Optimization of valence-shell scaling factors shows that considerable rescaling of at. functions occurs in mols., the largest effects being obsd. for H and C.  However, the range of optimum scale factors for each atom is small enough to allow the selection of a std. mol. set.  The use of this std. basis gives theoretical equil. geometries in reasonable agreement with expt.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqICTNxdJ_us7Vg90H21EOLACvtfcHk0liMc5iSAevdNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3MXksFOiuw%253D%253D&md5=abce439752b61fad3aa748515ec78c71</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1063%2F1.1674902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1674902%26sid%3Dliteratum%253Aachs%26aulast%3DDitchfield%26aufirst%3DR.%26aulast%3DHehre%26aufirst%3DW.%2BJ.%26aulast%3DPople%26aufirst%3DJ.%2BA.%26atitle%3DSelf-Consistent%2520Molecular-Orbital%2520Methods%25200.9.%2520Extended%2520Gaussian-Type%2520Basis%2520for%2520Molecular-Orbital%2520Studies%2520of%2520Organic%2520Molecules%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1971%26volume%3D54%26spage%3D724%26epage%3D728%26doi%3D10.1063%2F1.1674902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group">Kast, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kloss, T.</span><span> </span><span class="NLM_article-title">Closed-form expressions of the chemical potential for integral equation closures with certain bridge functions</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">236101</span><span class="refDoi"> DOI: 10.1063/1.3041709</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1063%2F1.3041709" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=19102559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFansb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2008&pages=236101&author=S.+M.+Kastauthor=T.+Kloss&title=Closed-form+expressions+of+the+chemical+potential+for+integral+equation+closures+with+certain+bridge+functions&doi=10.1063%2F1.3041709"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Closed-form expressions of the chemical potential for integral equation closures with certain bridge functions</span></div><div class="casAuthors">Kast, Stefan M.; Kloss, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">236101/1-236101/3</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">A general, path-independent expression is derived for the excess chem. potential of integral equation closure approxns. that contain a bridge function which depends on a renormalized indirect correlation function.  Closed-form results are obtained for various cases, among them a partial series expansion of the hypernetted chain closure.  (c) 2008 American Institute of Physics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8xHiEs2dHmbVg90H21EOLACvtfcHk0liMc5iSAevdNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFansb%252FI&md5=9ce23be4d261f269e89ec6248f28615f</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1063%2F1.3041709&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.3041709%26sid%3Dliteratum%253Aachs%26aulast%3DKast%26aufirst%3DS.%2BM.%26aulast%3DKloss%26aufirst%3DT.%26atitle%3DClosed-form%2520expressions%2520of%2520the%2520chemical%2520potential%2520for%2520integral%2520equation%2520closures%2520with%2520certain%2520bridge%2520functions%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2008%26volume%3D129%26spage%3D236101%26doi%3D10.1063%2F1.3041709" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group">Humphrey, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalke, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulten, K.</span><span> </span><span class="NLM_article-title">VMD: visual molecular dynamics</span> <span class="citation_source-journal">J. Mol. Graphics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span><span class="NLM_x">–</span> <span class="NLM_lpage">38</span><span class="refDoi"> DOI: 10.1016/0263-7855(96)00018-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1016%2F0263-7855%2896%2900018-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=8744570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaK28Xis12nsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1996&pages=33-38&author=W.+Humphreyauthor=A.+Dalkeauthor=K.+Schulten&title=VMD%3A+visual+molecular+dynamics&doi=10.1016%2F0263-7855%2896%2900018-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">VDM: visual molecular dynamics</span></div><div class="casAuthors">Humphrey, William; Dalke, Andrew; Schulten, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-8, plates, 27-28</span>CODEN:
                <span class="NLM_cas:coden">JMGRDV</span>;
        ISSN:<span class="NLM_cas:issn">0263-7855</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">VMD is a mol. graphics program designed for the display and anal. of mol. assemblies, in particular, biopolymers such as proteins and nucleic acids.  VMD can simultaneously display any no. of structures using a wide variety of rendering styles and coloring methods.  Mols. are displayed as one or more "representations," in which each representation embodies a particular rendering method and coloring scheme for a selected subset of atoms.  The atoms displayed in each representation are chosen using an extensive atom selection syntax, which includes Boolean operators and regular expressions.  VMD provides a complete graphical user interface for program control, as well as a text interface using the Tcl embeddable parser to allow for complex scripts with variable substitution, control loops, and function calls.  Full session logging is supported, which produces a VMD command script for later playback.  High-resoln. raster images of displayed mols. may be produced by generating input scripts for use by a no. of photorealistic image-rendering applications.  VMD has also been expressly designed with the ability to animate mol. dynamics (MD) simulation trajectories, imported either from files or from a direct connection to a running MD simulation.  VMD is the visualization component of MDScope, a set of tools for interactive problem solving in structural biol., which also includes the parallel MD program NAMD, and the MDCOMM software used to connect the visualization and simulation programs, VMD is written in C++, using an object-oriented design; the program, including source code and extensive documentation, is freely available via anonymous ftp and through the World Wide Web.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0dd54QvBO97Vg90H21EOLACvtfcHk0lilvGx0dy_GUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xis12nsrg%253D&md5=1e3094ec3151fb85c5ff05f8505c78d5</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1016%2F0263-7855%2896%2900018-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0263-7855%252896%252900018-5%26sid%3Dliteratum%253Aachs%26aulast%3DHumphrey%26aufirst%3DW.%26aulast%3DDalke%26aufirst%3DA.%26aulast%3DSchulten%26aufirst%3DK.%26atitle%3DVMD%253A%2520visual%2520molecular%2520dynamics%26jtitle%3DJ.%2520Mol.%2520Graphics%26date%3D1996%26volume%3D14%26spage%3D33%26epage%3D38%26doi%3D10.1016%2F0263-7855%2896%2900018-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 60 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Levente M. Mihalovits, György G. Ferenczy, <span class="NLM_string-name hlFld-ContribAuthor">György M. Keserű</span>. </span><span class="cited-content_cbyCitation_article-title">Affinity and Selectivity Assessment of Covalent Inhibitors by Free Energy Calculations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (12)
                                     , 6579-6594. <a href="https://doi.org/10.1021/acs.jcim.0c00834" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00834</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00834&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00834%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAffinity%252Band%252BSelectivity%252BAssessment%252Bof%252BCovalent%252BInhibitors%252Bby%252BFree%252BEnergy%252BCalculations%26aulast%3DMihalovits%26aufirst%3DLevente%2BM.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D22072020%26date%3D09122020%26volume%3D60%26issue%3D12%26spage%3D6579%26epage%3D6594" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mami Sato, Hirokazu Fuchida, Naoya Shindo, Keiko Kuwata, Keisuke Tokunaga, Guo Xiao-Lin, Ryo Inamori, Keitaro Hosokawa, Kosuke Watari, Tomohiro Shibata, Naoya Matsunaga, Satoru Koyanagi, Shigehiro Ohdo, Mayumi Ono, <span class="NLM_string-name hlFld-ContribAuthor">Akio Ojida</span>. </span><span class="cited-content_cbyCitation_article-title">Selective Covalent Targeting of Mutated EGFR(T790M) with Chlorofluoroacetamide-Pyrimidines. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (6)
                                     , 1137-1144. <a href="https://doi.org/10.1021/acsmedchemlett.9b00574" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00574</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00574%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSelective%252BCovalent%252BTargeting%252Bof%252BMutated%252BEGFR%252528T790M%252529%252Bwith%252BChlorofluoroacetamide-Pyrimidines%26aulast%3DSato%26aufirst%3DMami%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D04122019%26date%3D08042020%26date%3D10042020%26date%3D08042020%26volume%3D11%26issue%3D6%26spage%3D1137%26epage%3D1144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jonathan Pettinger, Michael Carter, Keith Jones, <span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Cheeseman</span>. </span><span class="cited-content_cbyCitation_article-title">Kinetic Optimization of Lysine-Targeting Covalent Inhibitors of HSP72. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (24)
                                     , 11383-11398. <a href="https://doi.org/10.1021/acs.jmedchem.9b01709" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01709</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01709&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01709%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DKinetic%252BOptimization%252Bof%252BLysine-Targeting%252BCovalent%252BInhibitors%252Bof%252BHSP72%26aulast%3DPettinger%26aufirst%3DJonathan%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D16102019%26date%3D06122019%26date%3D14112019%26volume%3D62%26issue%3D24%26spage%3D11383%26epage%3D11398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Carlo Baggio, Parima Udompholkul, Luca Gambini, Ahmed F. Salem, Jennifer Jossart, J. Jefferson P. Perry, <span class="NLM_string-name hlFld-ContribAuthor">Maurizio Pellecchia</span>. </span><span class="cited-content_cbyCitation_article-title">Aryl-fluorosulfate-based Lysine Covalent Pan-Inhibitors of Apoptosis Protein (IAP) Antagonists with Cellular Efficacy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (20)
                                     , 9188-9200. <a href="https://doi.org/10.1021/acs.jmedchem.9b01108" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01108</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01108%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAryl-fluorosulfate-based%252BLysine%252BCovalent%252BPan-Inhibitors%252Bof%252BApoptosis%252BProtein%252B%252528IAP%252529%252BAntagonists%252Bwith%252BCellular%252BEfficacy%26aulast%3DBaggio%26aufirst%3DCarlo%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D09072019%26date%3D08102019%26date%3D24092019%26volume%3D62%26issue%3D20%26spage%3D9188%26epage%3D9200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Farideh Badichi Akher, Abdolkarim Farrokhzadeh, Neil Ravenscroft, <span class="NLM_string-name hlFld-ContribAuthor">Michelle M. Kuttel</span>. </span><span class="cited-content_cbyCitation_article-title">A Mechanistic Study of a Potent and Selective Epidermal Growth Factor Receptor Inhibitor against the L858R/T790M Resistance Mutation. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2019,</strong> <em>58 </em>
                                    (41)
                                     , 4246-4259. <a href="https://doi.org/10.1021/acs.biochem.9b00710" title="DOI URL">https://doi.org/10.1021/acs.biochem.9b00710</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.9b00710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.9b00710%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DA%252BMechanistic%252BStudy%252Bof%252Ba%252BPotent%252Band%252BSelective%252BEpidermal%252BGrowth%252BFactor%252BReceptor%252BInhibitor%252Bagainst%252Bthe%252BL858R%25252FT790M%252BResistance%252BMutation%26aulast%3DAkher%26aufirst%3DFarideh%2BBadichi%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D13082019%26date%3D23092019%26date%3D07102019%26volume%3D58%26issue%3D41%26spage%3D4246%26epage%3D4259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Patrik Wolle, Julian Engel, Steven Smith, Lisa Goebel, Elisabeth Hennes, Jonas Lategahn, <span class="NLM_string-name hlFld-ContribAuthor">Daniel Rauh</span>. </span><span class="cited-content_cbyCitation_article-title">Characterization of Covalent Pyrazolopyrimidine–MKK7 Complexes and a Report on a Unique DFG-in/Leu-in Conformation of Mitogen-Activated Protein Kinase Kinase 7 (MKK7). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (11)
                                     , 5541-5546. <a href="https://doi.org/10.1021/acs.jmedchem.9b00472" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00472</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00472%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCharacterization%252Bof%252BCovalent%252BPyrazolopyrimidine%2525E2%252580%252593MKK7%252BComplexes%252Band%252Ba%252BReport%252Bon%252Ba%252BUnique%252BDFG-in%25252FLeu-in%252BConformation%252Bof%252BMitogen-Activated%252BProtein%252BKinase%252BKinase%252B7%252B%252528MKK7%252529%26aulast%3DWolle%26aufirst%3DPatrik%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D18032019%26date%3D31052019%26date%3D14052019%26volume%3D62%26issue%3D11%26spage%3D5541%26epage%3D5546" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yuming Wang, Lijun Li, Jun Fan, Yang Dai, Alan Jiang, Meiyu Geng, Jing Ai, <span class="NLM_string-name hlFld-ContribAuthor">Wenhu Duan</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Potent Irreversible Pan-Fibroblast Growth Factor Receptor (FGFR) Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (20)
                                     , 9085-9104. <a href="https://doi.org/10.1021/acs.jmedchem.7b01843" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01843</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01843&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01843%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BPotent%252BIrreversible%252BPan-Fibroblast%252BGrowth%252BFactor%252BReceptor%252B%252528FGFR%252529%252BInhibitors%26aulast%3DWang%26aufirst%3DYuming%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D14122017%26date%3D16032018%26date%3D09032018%26volume%3D61%26issue%3D20%26spage%3D9085%26epage%3D9104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ye Ding, Dengfeng Li, Chunyong Ding, Pingyuan Wang, Zhiqing Liu, Eric A. Wold, Na Ye, Haiying Chen, Mark A. White, Qiang Shen, <span class="NLM_string-name hlFld-ContribAuthor">Jia Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Regio- and Stereospecific Synthesis of Oridonin D-Ring Aziridinated Analogues for the Treatment of Triple-Negative Breast Cancer via Mediated Irreversible Covalent Warheads. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (7)
                                     , 2737-2752. <a href="https://doi.org/10.1021/acs.jmedchem.7b01514" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01514</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01514%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRegio-%252Band%252BStereospecific%252BSynthesis%252Bof%252BOridonin%252BD-Ring%252BAziridinated%252BAnalogues%252Bfor%252Bthe%252BTreatment%252Bof%252BTriple-Negative%252BBreast%252BCancer%252Bvia%252BMediated%252BIrreversible%252BCovalent%252BWarheads%26aulast%3DDing%26aufirst%3DYe%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D11102017%26date%3D20032018%26date%3D12032018%26volume%3D61%26issue%3D7%26spage%3D2737%26epage%3D2752" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Humberto  Gonczarowska-Jorge</span>, <span class="hlFld-ContribAuthor ">Stefan  Loroch</span>, <span class="hlFld-ContribAuthor ">Margherita  Dell’Aica</span>, <span class="hlFld-ContribAuthor ">Albert  Sickmann</span>, <span class="hlFld-ContribAuthor ">Andreas  Roos</span>, and <span class="hlFld-ContribAuthor ">René P.  Zahedi</span>  . </span><span class="cited-content_cbyCitation_article-title">Quantifying Missing (Phospho)Proteome Regions with the Broad-Specificity Protease Subtilisin. </span><span class="cited-content_cbyCitation_journal-name">Analytical Chemistry</span><span> <strong>2017,</strong> <em>89 </em>
                                    (24)
                                     , 13137-13145. <a href="https://doi.org/10.1021/acs.analchem.7b02395" title="DOI URL">https://doi.org/10.1021/acs.analchem.7b02395</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.analchem.7b02395&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.analchem.7b02395%26sid%3Dliteratum%253Aachs%26jtitle%3DAnalytical%2520Chemistry%26atitle%3DQuantifying%252BMissing%252B%252528Phospho%252529Proteome%252BRegions%252Bwith%252Bthe%252BBroad-Specificity%252BProtease%252BSubtilisin%26aulast%3DGonczarowska-Jorge%26aufirst%3DHumberto%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D21062017%26date%3D13112017%26date%3D28112017%26date%3D19122017%26date%3D14112017%26volume%3D89%26issue%3D24%26spage%3D13137%26epage%3D13145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Julian  Engel</span>, <span class="hlFld-ContribAuthor ">Steven  Smith</span>, <span class="hlFld-ContribAuthor ">Jonas  Lategahn</span>, <span class="hlFld-ContribAuthor ">Hannah L.  Tumbrink</span>, <span class="hlFld-ContribAuthor ">Lisa  Goebel</span>, <span class="hlFld-ContribAuthor ">Christian  Becker</span>, <span class="hlFld-ContribAuthor ">Elisabeth  Hennes</span>, <span class="hlFld-ContribAuthor ">Marina  Keul</span>, <span class="hlFld-ContribAuthor ">Anke  Unger</span>, <span class="hlFld-ContribAuthor ">Heiko  Müller</span>, <span class="hlFld-ContribAuthor ">Matthias  Baumann</span>, <span class="hlFld-ContribAuthor ">Carsten  Schultz-Fademrecht</span>, <span class="hlFld-ContribAuthor ">Georgia  Günther</span>, <span class="hlFld-ContribAuthor ">Jan G.  Hengstler</span>, and <span class="hlFld-ContribAuthor ">Daniel  Rauh</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure-Guided Development of Covalent and Mutant-Selective Pyrazolopyrimidines to Target T790M Drug Resistance in Epidermal Growth Factor Receptor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (18)
                                     , 7725-7744. <a href="https://doi.org/10.1021/acs.jmedchem.7b00515" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00515</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00515%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Guided%252BDevelopment%252Bof%252BCovalent%252Band%252BMutant-Selective%252BPyrazolopyrimidines%252Bto%252BTarget%252BT790M%252BDrug%252BResistance%252Bin%252BEpidermal%252BGrowth%252BFactor%252BReceptor%26aulast%3DEngel%26aufirst%3DJulian%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D03042017%26date%3D18092017%26date%3D28092017%26date%3D30082017%26volume%3D60%26issue%3D18%26spage%3D7725%26epage%3D7744" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marcel  Günther</span>, <span class="hlFld-ContribAuthor ">Jonas  Lategahn</span>, <span class="hlFld-ContribAuthor ">Michael  Juchum</span>, <span class="hlFld-ContribAuthor ">Eva  Döring</span>, <span class="hlFld-ContribAuthor ">Marina  Keul</span>, <span class="hlFld-ContribAuthor ">Julian  Engel</span>, <span class="hlFld-ContribAuthor ">Hannah L.  Tumbrink</span>, <span class="hlFld-ContribAuthor ">Daniel  Rauh</span>, and <span class="hlFld-ContribAuthor ">Stefan  Laufer</span>  . </span><span class="cited-content_cbyCitation_article-title">Trisubstituted Pyridinylimidazoles as Potent Inhibitors of the Clinically Resistant L858R/T790M/C797S EGFR Mutant: Targeting of Both Hydrophobic Regions and the Phosphate Binding Site. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (13)
                                     , 5613-5637. <a href="https://doi.org/10.1021/acs.jmedchem.7b00316" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00316</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00316%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTrisubstituted%252BPyridinylimidazoles%252Bas%252BPotent%252BInhibitors%252Bof%252Bthe%252BClinically%252BResistant%252BL858R%25252FT790M%25252FC797S%252BEGFR%252BMutant%25253A%252BTargeting%252Bof%252BBoth%252BHydrophobic%252BRegions%252Band%252Bthe%252BPhosphate%252BBinding%252BSite%26aulast%3DG%25C3%25BCnther%26aufirst%3DMarcel%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D27022017%26date%3D21062017%26date%3D13072017%26date%3D12062017%26volume%3D60%26issue%3D13%26spage%3D5613%26epage%3D5637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Steven  Smith</span>, <span class="hlFld-ContribAuthor ">Marina  Keul</span>, <span class="hlFld-ContribAuthor ">Julian  Engel</span>, <span class="hlFld-ContribAuthor ">Debjit  Basu</span>, <span class="hlFld-ContribAuthor ">Simone  Eppmann</span>, and <span class="hlFld-ContribAuthor ">Daniel  Rauh</span>  . </span><span class="cited-content_cbyCitation_article-title">Characterization of Covalent-Reversible EGFR Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2017,</strong> <em>2 </em>
                                    (4)
                                     , 1563-1575. <a href="https://doi.org/10.1021/acsomega.7b00157" title="DOI URL">https://doi.org/10.1021/acsomega.7b00157</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.7b00157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.7b00157%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DCharacterization%252Bof%252BCovalent-Reversible%252BEGFR%252BInhibitors%26aulast%3DSmith%26aufirst%3DSteven%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D10022017%26date%3D03042017%26date%3D30042017%26volume%3D2%26issue%3D4%26spage%3D1563%26epage%3D1575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stefano  Tomassi</span>, <span class="hlFld-ContribAuthor ">Jonas  Lategahn</span>, <span class="hlFld-ContribAuthor ">Julian  Engel</span>, <span class="hlFld-ContribAuthor ">Marina  Keul</span>, <span class="hlFld-ContribAuthor ">Hannah L.  Tumbrink</span>, <span class="hlFld-ContribAuthor ">Julia  Ketzer</span>, <span class="hlFld-ContribAuthor ">Thomas  Mühlenberg</span>, <span class="hlFld-ContribAuthor ">Matthias  Baumann</span>, <span class="hlFld-ContribAuthor ">Carsten  Schultz-Fademrecht</span>, <span class="hlFld-ContribAuthor ">Sebastian  Bauer</span>, and <span class="hlFld-ContribAuthor ">Daniel  Rauh</span>  . </span><span class="cited-content_cbyCitation_article-title">Indazole-Based Covalent Inhibitors To Target Drug-Resistant Epidermal Growth Factor Receptor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (6)
                                     , 2361-2372. <a href="https://doi.org/10.1021/acs.jmedchem.6b01626" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01626</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01626%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIndazole-Based%252BCovalent%252BInhibitors%252BTo%252BTarget%252BDrug-Resistant%252BEpidermal%252BGrowth%252BFactor%252BReceptor%26aulast%3DTomassi%26aufirst%3DStefano%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D02112016%26date%3D10032017%26date%3D23032017%26date%3D22022017%26volume%3D60%26issue%3D6%26spage%3D2361%26epage%3D2372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yongjia  Hao</span>, <span class="hlFld-ContribAuthor ">Xia  Wang</span>, <span class="hlFld-ContribAuthor ">Tao  Zhang</span>, <span class="hlFld-ContribAuthor ">Deheng  Sun</span>, <span class="hlFld-ContribAuthor ">Yi  Tong</span>, <span class="hlFld-ContribAuthor ">Yuqiong  Xu</span>, <span class="hlFld-ContribAuthor ">Haiyang  Chen</span>, <span class="hlFld-ContribAuthor ">Linjiang  Tong</span>, <span class="hlFld-ContribAuthor ">Lili  Zhu</span>, <span class="hlFld-ContribAuthor ">Zhenjiang  Zhao</span>, <span class="hlFld-ContribAuthor ">Zhuo  Chen</span>, <span class="hlFld-ContribAuthor ">Jian  Ding</span>, <span class="hlFld-ContribAuthor ">Hua  Xie</span>, <span class="hlFld-ContribAuthor ">Yufang  Xu</span>, and <span class="hlFld-ContribAuthor ">Honglin  Li</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery and Structural Optimization of N5-Substituted 6,7-Dioxo-6,7-dihydropteridines as Potent and Selective Epidermal Growth Factor Receptor (EGFR) Inhibitors against L858R/T790M Resistance Mutation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (15)
                                     , 7111-7124. <a href="https://doi.org/10.1021/acs.jmedchem.6b00403" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b00403</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b00403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b00403%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252BStructural%252BOptimization%252Bof%252BN5-Substituted%252B6%25252C7-Dioxo-6%25252C7-dihydropteridines%252Bas%252BPotent%252Band%252BSelective%252BEpidermal%252BGrowth%252BFactor%252BReceptor%252B%252528EGFR%252529%252BInhibitors%252Bagainst%252BL858R%25252FT790M%252BResistance%252BMutation%26aulast%3DHao%26aufirst%3DYongjia%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D26032016%26date%3D21072016%26date%3D11082016%26date%3D11072016%26volume%3D59%26issue%3D15%26spage%3D7111%26epage%3D7124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zheng  Zhao</span>, <span class="hlFld-ContribAuthor ">Li  Xie</span>, <span class="hlFld-ContribAuthor ">Lei  Xie</span>, and <span class="hlFld-ContribAuthor ">Philip E.  Bourne</span>  . </span><span class="cited-content_cbyCitation_article-title">Delineation of Polypharmacology across the Human Structural Kinome Using a Functional Site Interaction Fingerprint Approach. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (9)
                                     , 4326-4341. <a href="https://doi.org/10.1021/acs.jmedchem.5b02041" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b02041</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b02041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b02041%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDelineation%252Bof%252BPolypharmacology%252Bacross%252Bthe%252BHuman%252BStructural%252BKinome%252BUsing%252Ba%252BFunctional%252BSite%252BInteraction%252BFingerprint%252BApproach%26aulast%3DZhao%26aufirst%3DZheng%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D31122015%26date%3D17032016%26date%3D12052016%26date%3D01032016%26volume%3D59%26issue%3D9%26spage%3D4326%26epage%3D4341" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Edward J.  Hennessy</span>, <span class="hlFld-ContribAuthor ">Claudio  Chuaqui</span>, <span class="hlFld-ContribAuthor ">Susan  Ashton</span>, <span class="hlFld-ContribAuthor ">Nicola  Colclough</span>, <span class="hlFld-ContribAuthor ">Darren A. E.  Cross</span>, <span class="hlFld-ContribAuthor ">Judit É.  Debreczeni</span>, <span class="hlFld-ContribAuthor ">Cath  Eberlein</span>, <span class="hlFld-ContribAuthor ">Lakshmaiah  Gingipalli</span>, <span class="hlFld-ContribAuthor ">Teresa C. M.  Klinowska</span>, <span class="hlFld-ContribAuthor ">Jonathan P.  Orme</span>, <span class="hlFld-ContribAuthor ">Li  Sha</span>, and <span class="hlFld-ContribAuthor ">Xiaoyun  Wu</span>  . </span><span class="cited-content_cbyCitation_article-title">Utilization of Structure-Based Design to Identify Novel, Irreversible Inhibitors of EGFR Harboring the T790M Mutation. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2016,</strong> <em>7 </em>
                                    (5)
                                     , 514-519. <a href="https://doi.org/10.1021/acsmedchemlett.6b00058" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.6b00058</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.6b00058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.6b00058%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DUtilization%252Bof%252BStructure-Based%252BDesign%252Bto%252BIdentify%252BNovel%25252C%252BIrreversible%252BInhibitors%252Bof%252BEGFR%252BHarboring%252Bthe%252BT790M%252BMutation%26aulast%3DHennessy%26aufirst%3DEdward%2BJ.%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D09022016%26date%3D21032016%26date%3D24032016%26date%3D12052016%26date%3D21032016%26volume%3D7%26issue%3D5%26spage%3D514%26epage%3D519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hengmiao  Cheng</span>, <span class="hlFld-ContribAuthor ">Sajiv K.  Nair</span>, <span class="hlFld-ContribAuthor ">Brion W.  Murray</span>, <span class="hlFld-ContribAuthor ">Chau  Almaden</span>, <span class="hlFld-ContribAuthor ">Simon  Bailey</span>, <span class="hlFld-ContribAuthor ">Sangita  Baxi</span>, <span class="hlFld-ContribAuthor ">Doug  Behenna</span>, <span class="hlFld-ContribAuthor ">Sujin  Cho-Schultz</span>, <span class="hlFld-ContribAuthor ">Deepak  Dalvie</span>, <span class="hlFld-ContribAuthor ">Dac M.  Dinh</span>, <span class="hlFld-ContribAuthor ">Martin P.  Edwards</span>, <span class="hlFld-ContribAuthor ">Jun Li  Feng</span>, <span class="hlFld-ContribAuthor ">Rose Ann  Ferre</span>, <span class="hlFld-ContribAuthor ">Ketan S.  Gajiwala</span>, <span class="hlFld-ContribAuthor ">Michelle D.  Hemkens</span>, <span class="hlFld-ContribAuthor ">Amy  Jackson-Fisher</span>, <span class="hlFld-ContribAuthor ">Mehran  Jalaie</span>, <span class="hlFld-ContribAuthor ">Ted O.  Johnson</span>, <span class="hlFld-ContribAuthor ">Robert S.  Kania</span>, <span class="hlFld-ContribAuthor ">Susan  Kephart</span>, <span class="hlFld-ContribAuthor ">Jennifer  Lafontaine</span>, <span class="hlFld-ContribAuthor ">Beth  Lunney</span>, <span class="hlFld-ContribAuthor ">Kevin K.-C.  Liu</span>, <span class="hlFld-ContribAuthor ">Zhengyu  Liu</span>, <span class="hlFld-ContribAuthor ">Jean  Matthews</span>, <span class="hlFld-ContribAuthor ">Asako  Nagata</span>, <span class="hlFld-ContribAuthor ">Sherry  Niessen</span>, <span class="hlFld-ContribAuthor ">Martha A.  Ornelas</span>, <span class="hlFld-ContribAuthor ">Suvi T. M.  Orr</span>, <span class="hlFld-ContribAuthor ">Mason  Pairish</span>, <span class="hlFld-ContribAuthor ">Simon  Planken</span>, <span class="hlFld-ContribAuthor ">Shijian  Ren</span>, <span class="hlFld-ContribAuthor ">Daniel  Richter</span>, <span class="hlFld-ContribAuthor ">Kevin  Ryan</span>, <span class="hlFld-ContribAuthor ">Neal  Sach</span>, <span class="hlFld-ContribAuthor ">Hong  Shen</span>, <span class="hlFld-ContribAuthor ">Tod  Smeal</span>, <span class="hlFld-ContribAuthor ">Jim  Solowiej</span>, <span class="hlFld-ContribAuthor ">Scott  Sutton</span>, <span class="hlFld-ContribAuthor ">Khanh  Tran</span>, <span class="hlFld-ContribAuthor ">Elaine  Tseng</span>, <span class="hlFld-ContribAuthor ">William  Vernier</span>, <span class="hlFld-ContribAuthor ">Marlena  Walls</span>, <span class="hlFld-ContribAuthor ">Shuiwang  Wang</span>, <span class="hlFld-ContribAuthor ">Scott L.  Weinrich</span>, <span class="hlFld-ContribAuthor ">Shuibo  Xin</span>, <span class="hlFld-ContribAuthor ">Haiwei  Xu</span>, <span class="hlFld-ContribAuthor ">Min-Jean  Yin</span>, <span class="hlFld-ContribAuthor ">Michael  Zientek</span>, <span class="hlFld-ContribAuthor ">Ru  Zhou</span>, and <span class="hlFld-ContribAuthor ">John C.  Kath</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of 1-{(3R,4R)-3-[({5-Chloro-2-[(1-methyl-1H-pyrazol-4-yl)amino]-7H-pyrrolo[2,3-d]pyrimidin-4-yl}oxy)methyl]-4-methoxypyrrolidin-1-yl}prop-2-en-1-one (PF-06459988), a Potent, WT Sparing, Irreversible Inhibitor of T790M-Containing EGFR Mutants. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (5)
                                     , 2005-2024. <a href="https://doi.org/10.1021/acs.jmedchem.5b01633" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01633</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01633%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252B1-%25257B%2525283R%25252C4R%252529-3-%25255B%252528%25257B5-Chloro-2-%25255B%2525281-methyl-1H-pyrazol-4-yl%252529amino%25255D-7H-pyrrolo%25255B2%25252C3-d%25255Dpyrimidin-4-yl%25257Doxy%252529methyl%25255D-4-methoxypyrrolidin-1-yl%25257Dprop-2-en-1-one%252B%252528PF-06459988%252529%25252C%252Ba%252BPotent%25252C%252BWT%252BSparing%25252C%252BIrreversible%252BInhibitor%252Bof%252BT790M-Containing%252BEGFR%252BMutants%26aulast%3DCheng%26aufirst%3DHengmiao%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D17102015%26date%3D28012016%26date%3D10032016%26date%3D12012016%26volume%3D59%26issue%3D5%26spage%3D2005%26epage%3D2024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aimen  Aljoundi</span>, <span class="hlFld-ContribAuthor ">Ahmed  El Rashedy</span>, <span class="hlFld-ContribAuthor ">Mahmoud E. S.  Soliman</span>. </span><span class="cited-content_cbyCitation_article-title">Comparison of irreversible inhibition targeting HSP72 protein: the resurgence of covalent drug developments. </span><span class="cited-content_cbyCitation_journal-name">Molecular Simulation</span><span> <strong>2021,</strong> <em>61 </em>, 1-11. <a href="https://doi.org/10.1080/08927022.2021.1949457" title="DOI URL">https://doi.org/10.1080/08927022.2021.1949457</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/08927022.2021.1949457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F08927022.2021.1949457%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Simulation%26atitle%3DComparison%252Bof%252Birreversible%252Binhibition%252Btargeting%252BHSP72%252Bprotein%25253A%252Bthe%252Bresurgence%252Bof%252Bcovalent%252Bdrug%252Bdevelopments%26aulast%3DAljoundi%26aufirst%3DAimen%26date%3D2021%26date%3D2021%26volume%3D61%26spage%3D1%26epage%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Surya Kant  Tripathi</span>, <span class="hlFld-ContribAuthor ">Bijesh K.  Biswal</span>. </span><span class="cited-content_cbyCitation_article-title">Allosteric mutant-selective fourth-generation EGFR inhibitors as an efficient combination therapeutic in the treatment of non-small cell lung carcinoma. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2021,</strong> <em>26 </em>
                                    (6)
                                     , 1466-1472. <a href="https://doi.org/10.1016/j.drudis.2021.02.005" title="DOI URL">https://doi.org/10.1016/j.drudis.2021.02.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2021.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2021.02.005%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DAllosteric%252Bmutant-selective%252Bfourth-generation%252BEGFR%252Binhibitors%252Bas%252Ban%252Befficient%252Bcombination%252Btherapeutic%252Bin%252Bthe%252Btreatment%252Bof%252Bnon-small%252Bcell%252Blung%252Bcarcinoma%26aulast%3DTripathi%26aufirst%3DSurya%2BKant%26date%3D2021%26volume%3D26%26issue%3D6%26spage%3D1466%26epage%3D1472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Prateek  Pathak</span>, <span class="hlFld-ContribAuthor ">Hrvoje  Rimac</span>, <span class="hlFld-ContribAuthor ">Maria  Grishina</span>, <span class="hlFld-ContribAuthor ">Amita  Verma</span>, <span class="hlFld-ContribAuthor ">Vladimir  Potemkin</span>. </span><span class="cited-content_cbyCitation_article-title">Hybrid Quinazoline 1,3,5‐Triazines as Epidermal Growth Factor Receptor (EGFR) Inhibitors with Anticancer Activity: Design, Synthesis, and Computational Study. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>16 </em>
                                    (5)
                                     , 822-838. <a href="https://doi.org/10.1002/cmdc.202000646" title="DOI URL">https://doi.org/10.1002/cmdc.202000646</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000646%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DHybrid%252BQuinazoline%252B1%25252C3%25252C5%2525E2%252580%252590Triazines%252Bas%252BEpidermal%252BGrowth%252BFactor%252BReceptor%252B%252528EGFR%252529%252BInhibitors%252Bwith%252BAnticancer%252BActivity%25253A%252BDesign%25252C%252BSynthesis%25252C%252Band%252BComputational%252BStudy%26aulast%3DPathak%26aufirst%3DPrateek%26date%3D2021%26date%3D2020%26volume%3D16%26issue%3D5%26spage%3D822%26epage%3D838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jun  Liu</span>, <span class="hlFld-ContribAuthor ">Lin  Li</span>, <span class="hlFld-ContribAuthor ">Xiaobin  Li</span>, <span class="hlFld-ContribAuthor ">Xin  Wang</span>, <span class="hlFld-ContribAuthor ">Xiaoyu  Zhao</span>, <span class="hlFld-ContribAuthor ">Yanan  Qiao</span>, <span class="hlFld-ContribAuthor ">Yuliang  Xu</span>, <span class="hlFld-ContribAuthor ">Yong  Sun</span>, <span class="hlFld-ContribAuthor ">Lilin  Qian</span>, <span class="hlFld-ContribAuthor ">Zhaopeng  Liu</span>, <span class="hlFld-ContribAuthor ">Aiguo  Ji</span>, <span class="hlFld-ContribAuthor ">Hongxiang  Lou</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of lysosome-targeted covalent anticancer agents based on isosteviol skeleton. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>209 </em>, 112896. <a href="https://doi.org/10.1016/j.ejmech.2020.112896" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112896</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112896%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Blysosome-targeted%252Bcovalent%252Banticancer%252Bagents%252Bbased%252Bon%252Bisosteviol%252Bskeleton%26aulast%3DLiu%26aufirst%3DJun%26date%3D2021%26volume%3D209%26spage%3D112896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Volodymyr A.  Tkachuk</span>, <span class="hlFld-ContribAuthor ">Vyacheslav O.  Shishkanu</span>, <span class="hlFld-ContribAuthor ">Tetiana M.  Tkachuk</span>, <span class="hlFld-ContribAuthor ">Svitlana V.  Shishkina</span>, <span class="hlFld-ContribAuthor ">Olga V.  Hordiyenko</span>. </span><span class="cited-content_cbyCitation_article-title">2-Carbamimidoylbenzoic Acid as a New Effective and Available Precursor for the Synthesis of Substituted 2-(Pyrimidin-2-yl)benzoic Acids. </span><span class="cited-content_cbyCitation_journal-name">Synthesis</span><span> <strong>2021,</strong> <em>53 </em>
                                    (02)
                                     , 371-382. <a href="https://doi.org/10.1055/s-0040-1705941" title="DOI URL">https://doi.org/10.1055/s-0040-1705941</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1055/s-0040-1705941&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1055%2Fs-0040-1705941%26sid%3Dliteratum%253Aachs%26jtitle%3DSynthesis%26atitle%3D2-Carbamimidoylbenzoic%252BAcid%252Bas%252Ba%252BNew%252BEffective%252Band%252BAvailable%252BPrecursor%252Bfor%252Bthe%252BSynthesis%252Bof%252BSubstituted%252B2-%252528Pyrimidin-2-yl%252529benzoic%252BAcids%26aulast%3DTkachuk%26aufirst%3DVolodymyr%2BA.%26date%3D2021%26date%3D2020%26volume%3D53%26issue%3D02%26spage%3D371%26epage%3D382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Luca  Gambini</span>, <span class="hlFld-ContribAuthor ">Parima  Udompholkul</span>, <span class="hlFld-ContribAuthor ">Ahmed F.  Salem</span>, <span class="hlFld-ContribAuthor ">Carlo  Baggio</span>, <span class="hlFld-ContribAuthor ">Maurizio  Pellecchia</span>. </span><span class="cited-content_cbyCitation_article-title">Stability and Cell Permeability of Sulfonyl Fluorides in the Design of Lys‐Covalent Antagonists of Protein‐Protein Interactions. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2020,</strong> <em>15 </em>
                                    (22)
                                     , 2176-2184. <a href="https://doi.org/10.1002/cmdc.202000355" title="DOI URL">https://doi.org/10.1002/cmdc.202000355</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000355%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DStability%252Band%252BCell%252BPermeability%252Bof%252BSulfonyl%252BFluorides%252Bin%252Bthe%252BDesign%252Bof%252BLys%2525E2%252580%252590Covalent%252BAntagonists%252Bof%252BProtein%2525E2%252580%252590Protein%252BInteractions%26aulast%3DGambini%26aufirst%3DLuca%26date%3D2020%26date%3D2020%26volume%3D15%26issue%3D22%26spage%3D2176%26epage%3D2184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vibhu  Jha</span>, <span class="hlFld-ContribAuthor ">Marco  Macchia</span>, <span class="hlFld-ContribAuthor ">Tiziano  Tuccinardi</span>, <span class="hlFld-ContribAuthor ">Giulio  Poli</span>. </span><span class="cited-content_cbyCitation_article-title">Three-Dimensional Interactions Analysis of the Anticancer Target c-Src Kinase with Its Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2020,</strong> <em>12 </em>
                                    (8)
                                     , 2327. <a href="https://doi.org/10.3390/cancers12082327" title="DOI URL">https://doi.org/10.3390/cancers12082327</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers12082327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers12082327%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DThree-Dimensional%252BInteractions%252BAnalysis%252Bof%252Bthe%252BAnticancer%252BTarget%252Bc-Src%252BKinase%252Bwith%252BIts%252BInhibitors%26aulast%3DJha%26aufirst%3DVibhu%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D8%26spage%3D2327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiuli  Wu</span>, <span class="hlFld-ContribAuthor ">Yange  Zhang</span>, <span class="hlFld-ContribAuthor ">Wenzhen  Li</span>, <span class="hlFld-ContribAuthor ">Tingmei  Liu</span>, <span class="hlFld-ContribAuthor ">Yu  Yang</span>, <span class="hlFld-ContribAuthor ">Yijie  Wang</span>, <span class="hlFld-ContribAuthor ">Yiwen  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">A Validated LC Method for the Enantiomeric Separation of EAI045 on Chiral Stationary Phase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chromatographic Science</span><span> <strong>2020,</strong> <em>58 </em>
                                    (6)
                                     , 562-568. <a href="https://doi.org/10.1093/chromsci/bmz125" title="DOI URL">https://doi.org/10.1093/chromsci/bmz125</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/chromsci/bmz125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fchromsci%2Fbmz125%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chromatographic%2520Science%26atitle%3DA%252BValidated%252BLC%252BMethod%252Bfor%252Bthe%252BEnantiomeric%252BSeparation%252Bof%252BEAI045%252Bon%252BChiral%252BStationary%252BPhase%26aulast%3DWu%26aufirst%3DXiuli%26date%3D2020%26date%3D2020%26volume%3D58%26issue%3D6%26spage%3D562%26epage%3D568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adileh  Ayati</span>, <span class="hlFld-ContribAuthor ">Setareh  Moghimi</span>, <span class="hlFld-ContribAuthor ">Somayeh  Salarinejad</span>, <span class="hlFld-ContribAuthor ">Maliheh  Safavi</span>, <span class="hlFld-ContribAuthor ">Behjat  Pouramiri</span>, <span class="hlFld-ContribAuthor ">Alireza  Foroumadi</span>. </span><span class="cited-content_cbyCitation_article-title">A review on progression of epidermal growth factor receptor (EGFR) inhibitors as an efficient approach in cancer targeted therapy. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2020,</strong> <em>99 </em>, 103811. <a href="https://doi.org/10.1016/j.bioorg.2020.103811" title="DOI URL">https://doi.org/10.1016/j.bioorg.2020.103811</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2020.103811&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2020.103811%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DA%252Breview%252Bon%252Bprogression%252Bof%252Bepidermal%252Bgrowth%252Bfactor%252Breceptor%252B%252528EGFR%252529%252Binhibitors%252Bas%252Ban%252Befficient%252Bapproach%252Bin%252Bcancer%252Btargeted%252Btherapy%26aulast%3DAyati%26aufirst%3DAdileh%26date%3D2020%26volume%3D99%26spage%3D103811" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zixuan  Chu</span>, <span class="hlFld-ContribAuthor ">Mingxing  Xue</span>, <span class="hlFld-ContribAuthor ">Shan  Xiong</span>. </span><span class="cited-content_cbyCitation_article-title">Development and Validation of an LC-MS/MS Method for Quantitative Determination of EAI045, A Novel EGFR Inhibitor, in Rat Plasma. </span><span class="cited-content_cbyCitation_journal-name">Current Pharmaceutical Analysis</span><span> <strong>2020,</strong> <em>16 </em>
                                    (3)
                                     , 273-279. <a href="https://doi.org/10.2174/1573412915666190206125557" title="DOI URL">https://doi.org/10.2174/1573412915666190206125557</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1573412915666190206125557&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1573412915666190206125557%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Pharmaceutical%2520Analysis%26atitle%3DDevelopment%252Band%252BValidation%252Bof%252Ban%252BLC-MS%25252FMS%252BMethod%252Bfor%252BQuantitative%252BDetermination%252Bof%252BEAI045%25252C%252BA%252BNovel%252BEGFR%252BInhibitor%25252C%252Bin%252BRat%252BPlasma%26aulast%3DChu%26aufirst%3DZixuan%26date%3D2020%26volume%3D16%26issue%3D3%26spage%3D273%26epage%3D279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hung Huy  Nguyen</span>, <span class="hlFld-ContribAuthor ">Péter  Ábrányi-Balogh</span>, <span class="hlFld-ContribAuthor ">László  Petri</span>, <span class="hlFld-ContribAuthor ">Attila  Mészáros</span>, <span class="hlFld-ContribAuthor ">Kris  Pauwels</span>, <span class="hlFld-ContribAuthor ">Guy  Vandenbussche</span>, <span class="hlFld-ContribAuthor ">György Miklós  Keserű</span>, <span class="hlFld-ContribAuthor ">Peter  Tompa</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting an Intrinsically Disordered Protein by Covalent Modification. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 835-854. <a href="https://doi.org/10.1007/978-1-0716-0524-0_43" title="DOI URL">https://doi.org/10.1007/978-1-0716-0524-0_43</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-0716-0524-0_43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-0716-0524-0_43%26sid%3Dliteratum%253Aachs%26atitle%3DTargeting%252Ban%252BIntrinsically%252BDisordered%252BProtein%252Bby%252BCovalent%252BModification%26aulast%3DNguyen%26aufirst%3DHung%2BHuy%26date%3D2020%26date%3D2020%26spage%3D835%26epage%3D854%26pub%3DSpringer%2520US%26atitle%3DIntrinsically%252BDisordered%252BProteins%26aulast%3DKragelund%26aufirst%3DBirthe%2BB.%26date%3D2020%26volume%3D2141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alessandro  Leonetti</span>, <span class="hlFld-ContribAuthor ">Sugandhi  Sharma</span>, <span class="hlFld-ContribAuthor ">Roberta  Minari</span>, <span class="hlFld-ContribAuthor ">Paola  Perego</span>, <span class="hlFld-ContribAuthor ">Elisa  Giovannetti</span>, <span class="hlFld-ContribAuthor ">Marcello  Tiseo</span>. </span><span class="cited-content_cbyCitation_article-title">Resistance mechanisms to osimertinib in EGFR-mutated non-small cell lung cancer. </span><span class="cited-content_cbyCitation_journal-name">British Journal of Cancer</span><span> <strong>2019,</strong> <em>121 </em>
                                    (9)
                                     , 725-737. <a href="https://doi.org/10.1038/s41416-019-0573-8" title="DOI URL">https://doi.org/10.1038/s41416-019-0573-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41416-019-0573-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41416-019-0573-8%26sid%3Dliteratum%253Aachs%26jtitle%3DBritish%2520Journal%2520of%2520Cancer%26atitle%3DResistance%252Bmechanisms%252Bto%252Bosimertinib%252Bin%252BEGFR-mutated%252Bnon-small%252Bcell%252Blung%252Bcancer%26aulast%3DLeonetti%26aufirst%3DAlessandro%26date%3D2019%26date%3D2019%26volume%3D121%26issue%3D9%26spage%3D725%26epage%3D737" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guodong  Wang</span>, <span class="hlFld-ContribAuthor ">Jian  Sun</span>, <span class="hlFld-ContribAuthor ">Kai  Wang</span>, <span class="hlFld-ContribAuthor ">Junfen  Han</span>, <span class="hlFld-ContribAuthor ">Hongshuang  Li</span>, <span class="hlFld-ContribAuthor ">Guiyun  Duan</span>, <span class="hlFld-ContribAuthor ">Guirong  You</span>, <span class="hlFld-ContribAuthor ">Furong  Li</span>, <span class="hlFld-ContribAuthor ">Chengcai  Xia</span>. </span><span class="cited-content_cbyCitation_article-title">Palladium-catalyzed direct C–H nitration and intramolecular C–H functionalization for the synthesis of 3-nitro-1-(phenylsulfonyl)-1
              H
              -indazole derivatives. </span><span class="cited-content_cbyCitation_journal-name">Organic Chemistry Frontiers</span><span> <strong>2019,</strong> <em>6 </em>
                                    (10)
                                     , 1608-1612. <a href="https://doi.org/10.1039/C9QO00367C" title="DOI URL">https://doi.org/10.1039/C9QO00367C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9QO00367C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9QO00367C%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Chemistry%2520Frontiers%26atitle%3DPalladium-catalyzed%252Bdirect%252BC%2525E2%252580%252593H%252Bnitration%252Band%252Bintramolecular%252BC%2525E2%252580%252593H%252Bfunctionalization%252Bfor%252Bthe%252Bsynthesis%252Bof%252B3-nitro-1-%252528phenylsulfonyl%252529-1%252BH%252B-indazole%252Bderivatives%26aulast%3DWang%26aufirst%3DGuodong%26date%3D2019%26date%3D2019%26volume%3D6%26issue%3D10%26spage%3D1608%26epage%3D1612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Farideh Badichi  Akher</span>, <span class="hlFld-ContribAuthor ">Abdolkarim  Farrokhzadeh</span>, <span class="hlFld-ContribAuthor ">Mahmoud E. S.  Soliman</span>. </span><span class="cited-content_cbyCitation_article-title">Noteworthy effect of slight variation in aliphatic chain length of trisubstituted imidazole inhibitors against epidermal growth factor receptor L858R/T790M/C797S mutant in cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Chemical Biology & Drug Design</span><span> <strong>2019,</strong> <em>93 </em>
                                    (5)
                                     , 798-810. <a href="https://doi.org/10.1111/cbdd.13467" title="DOI URL">https://doi.org/10.1111/cbdd.13467</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cbdd.13467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcbdd.13467%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Biology%2520%2526%2520Drug%2520Design%26atitle%3DNoteworthy%252Beffect%252Bof%252Bslight%252Bvariation%252Bin%252Baliphatic%252Bchain%252Blength%252Bof%252Btrisubstituted%252Bimidazole%252Binhibitors%252Bagainst%252Bepidermal%252Bgrowth%252Bfactor%252Breceptor%252BL858R%25252FT790M%25252FC797S%252Bmutant%252Bin%252Bcancer%252Btherapy%26aulast%3DAkher%26aufirst%3DFarideh%2BBadichi%26date%3D2019%26date%3D2019%26volume%3D93%26issue%3D5%26spage%3D798%26epage%3D810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhihui  Zhou</span>, <span class="hlFld-ContribAuthor ">Caolin  Wang</span>, <span class="hlFld-ContribAuthor ">Zhen  Xiao</span>, <span class="hlFld-ContribAuthor ">Qi  Yang</span>, <span class="hlFld-ContribAuthor ">Shan  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of 2-chloro-4-(3-nitrophenoxy) thieno [3, 2-d] pyrimidine. </span><span class="cited-content_cbyCitation_journal-name">IOP Conference Series: Earth and Environmental Science</span><span> <strong>2019,</strong> <em>252 </em>, 022091. <a href="https://doi.org/10.1088/1755-1315/252/2/022091" title="DOI URL">https://doi.org/10.1088/1755-1315/252/2/022091</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1088/1755-1315/252/2/022091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1088%2F1755-1315%2F252%2F2%2F022091%26sid%3Dliteratum%253Aachs%26jtitle%3DIOP%2520Conference%2520Series%253A%2520Earth%2520and%2520Environmental%2520Science%26atitle%3DSynthesis%252Bof%252B2-chloro-4-%2525283-nitrophenoxy%252529%252Bthieno%252B%25255B3%25252C%252B2-d%25255D%252Bpyrimidine%26aulast%3DZhou%26aufirst%3DZhihui%26date%3D2019%26date%3D2019%26volume%3D252%26spage%3D022091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Farideh Badichi  Akher</span>, <span class="hlFld-ContribAuthor ">Abdolkarim  Farrokhzadeh</span>, <span class="hlFld-ContribAuthor ">Mahmoud E. S.  Soliman</span>. </span><span class="cited-content_cbyCitation_article-title">Covalent vs. Non‐Covalent Inhibition: Tackling Drug Resistance in EGFR – A Thorough Dynamic Perspective. </span><span class="cited-content_cbyCitation_journal-name">Chemistry & Biodiversity</span><span> <strong>2019,</strong> <em>16 </em>
                                    (3)
                                     , e1800518. <a href="https://doi.org/10.1002/cbdv.201800518" title="DOI URL">https://doi.org/10.1002/cbdv.201800518</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbdv.201800518&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbdv.201800518%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%2526%2520Biodiversity%26atitle%3DCovalent%252Bvs.%252BNon%2525E2%252580%252590Covalent%252BInhibition%25253A%252BTackling%252BDrug%252BResistance%252Bin%252BEGFR%252B%2525E2%252580%252593%252BA%252BThorough%252BDynamic%252BPerspective%26aulast%3DAkher%26aufirst%3DFarideh%2BBadichi%26date%3D2019%26date%3D2019%26volume%3D16%26issue%3D3%26spage%3De1800518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Harun M.  Patel</span>, <span class="hlFld-ContribAuthor ">Rahul  Pawara</span>, <span class="hlFld-ContribAuthor ">Sanjay J.  Surana</span>. </span><span class="cited-content_cbyCitation_article-title">Current Approaches in the Development of Covalent Irreversible EGFR Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 45-128. <a href="https://doi.org/10.1016/B978-0-08-102661-8.00003-2" title="DOI URL">https://doi.org/10.1016/B978-0-08-102661-8.00003-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-08-102661-8.00003-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-08-102661-8.00003-2%26sid%3Dliteratum%253Aachs%26atitle%3DCurrent%252BApproaches%252Bin%252Bthe%252BDevelopment%252Bof%252BCovalent%252BIrreversible%252BEGFR%252BInhibitors%26aulast%3DPatel%26aufirst%3DHarun%2BM.%26date%3D2019%26spage%3D45%26epage%3D128%26pub%3DElsevier%26atitle%3DThird%252BGeneration%252BEGFR%252BInhibitors%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Péter  Ábrányi-Balogh</span>, <span class="hlFld-ContribAuthor ">László  Petri</span>, <span class="hlFld-ContribAuthor ">Tímea  Imre</span>, <span class="hlFld-ContribAuthor ">Péter  Szijj</span>, <span class="hlFld-ContribAuthor ">Andrea  Scarpino</span>, <span class="hlFld-ContribAuthor ">Martina  Hrast</span>, <span class="hlFld-ContribAuthor ">Ana  Mitrović</span>, <span class="hlFld-ContribAuthor ">Urša Pečar  Fonovič</span>, <span class="hlFld-ContribAuthor ">Krisztina  Németh</span>, <span class="hlFld-ContribAuthor ">Hélène  Barreteau</span>, <span class="hlFld-ContribAuthor ">David I.  Roper</span>, <span class="hlFld-ContribAuthor ">Kata  Horváti</span>, <span class="hlFld-ContribAuthor ">György G.  Ferenczy</span>, <span class="hlFld-ContribAuthor ">Janko  Kos</span>, <span class="hlFld-ContribAuthor ">Janez  Ilaš</span>, <span class="hlFld-ContribAuthor ">Stanislav  Gobec</span>, <span class="hlFld-ContribAuthor ">György M.  Keserű</span>. </span><span class="cited-content_cbyCitation_article-title">A road map for prioritizing warheads for cysteine targeting covalent inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>160 </em>, 94-107. <a href="https://doi.org/10.1016/j.ejmech.2018.10.010" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.10.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.10.010%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252Broad%252Bmap%252Bfor%252Bprioritizing%252Bwarheads%252Bfor%252Bcysteine%252Btargeting%252Bcovalent%252Binhibitors%26aulast%3D%25C3%2581br%25C3%25A1nyi-Balogh%26aufirst%3DP%25C3%25A9ter%26date%3D2018%26volume%3D160%26spage%3D94%26epage%3D107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ruth A.  Mitchell</span>, <span class="hlFld-ContribAuthor ">Rodney B.  Luwor</span>, <span class="hlFld-ContribAuthor ">Antony W.  Burgess</span>. </span><span class="cited-content_cbyCitation_article-title">Epidermal growth factor receptor: Structure-function informing the design of anticancer therapeutics. </span><span class="cited-content_cbyCitation_journal-name">Experimental Cell Research</span><span> <strong>2018,</strong> <em>371 </em>
                                    (1)
                                     , 1-19. <a href="https://doi.org/10.1016/j.yexcr.2018.08.009" title="DOI URL">https://doi.org/10.1016/j.yexcr.2018.08.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.yexcr.2018.08.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.yexcr.2018.08.009%26sid%3Dliteratum%253Aachs%26jtitle%3DExperimental%2520Cell%2520Research%26atitle%3DEpidermal%252Bgrowth%252Bfactor%252Breceptor%25253A%252BStructure-function%252Binforming%252Bthe%252Bdesign%252Bof%252Banticancer%252Btherapeutics%26aulast%3DMitchell%26aufirst%3DRuth%2BA.%26date%3D2018%26volume%3D371%26issue%3D1%26spage%3D1%26epage%3D19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jinyun  Dong</span>, <span class="hlFld-ContribAuthor ">Qijing  Zhang</span>, <span class="hlFld-ContribAuthor ">Zengtao  Wang</span>, <span class="hlFld-ContribAuthor ">Guang  Huang</span>, <span class="hlFld-ContribAuthor ">Shaoshun  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Advances in the Development of Indazole-based Anticancer Agents. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2018,</strong> <em>13 </em>
                                    (15)
                                     , 1490-1507. <a href="https://doi.org/10.1002/cmdc.201800253" title="DOI URL">https://doi.org/10.1002/cmdc.201800253</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201800253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201800253%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DRecent%252BAdvances%252Bin%252Bthe%252BDevelopment%252Bof%252BIndazole-based%252BAnticancer%252BAgents%26aulast%3DDong%26aufirst%3DJinyun%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D15%26spage%3D1490%26epage%3D1507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xing  Lu</span>, <span class="hlFld-ContribAuthor ">Yi-Ming  Wu</span>, <span class="hlFld-ContribAuthor ">Jing-Mei  Yang</span>, <span class="hlFld-ContribAuthor ">Feng-E.  Ma</span>, <span class="hlFld-ContribAuthor ">Liang-Ping  Li</span>, <span class="hlFld-ContribAuthor ">Sheng  Chen</span>, <span class="hlFld-ContribAuthor ">Ye  Zhang</span>, <span class="hlFld-ContribAuthor ">Qing-Ling  Ni</span>, <span class="hlFld-ContribAuthor ">Ying-Ming  Pan</span>, <span class="hlFld-ContribAuthor ">Xue  Hong</span>, <span class="hlFld-ContribAuthor ">Yan  Peng</span>. </span><span class="cited-content_cbyCitation_article-title">Preparation of Rhodium(III) complexes with 2(1H)-quinolinone derivatives and evaluation of their in vitro and in vivo antitumor activity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>151 </em>, 226-236. <a href="https://doi.org/10.1016/j.ejmech.2018.03.074" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.03.074</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.03.074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.03.074%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPreparation%252Bof%252BRhodium%252528III%252529%252Bcomplexes%252Bwith%252B2%2525281H%252529-quinolinone%252Bderivatives%252Band%252Bevaluation%252Bof%252Btheir%252Bin%2525C2%2525A0vitro%252Band%252Bin%2525C2%2525A0vivo%252Bantitumor%252Bactivity%26aulast%3DLu%26aufirst%3DXing%26date%3D2018%26volume%3D151%26spage%3D226%26epage%3D236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kenneth K.  Hallenbeck</span>, <span class="hlFld-ContribAuthor ">Julia L.  Davies</span>, <span class="hlFld-ContribAuthor ">Connie  Merron</span>, <span class="hlFld-ContribAuthor ">Pierce  Ogden</span>, <span class="hlFld-ContribAuthor ">Eline  Sijbesma</span>, <span class="hlFld-ContribAuthor ">Christian  Ottmann</span>, <span class="hlFld-ContribAuthor ">Adam R.  Renslo</span>, <span class="hlFld-ContribAuthor ">Christopher  Wilson</span>, <span class="hlFld-ContribAuthor ">Michelle R.  Arkin</span>. </span><span class="cited-content_cbyCitation_article-title">A Liquid Chromatography/Mass Spectrometry Method for Screening Disulfide Tethering Fragments. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2018,</strong> <em>23 </em>
                                    (2)
                                     , 183-192. <a href="https://doi.org/10.1177/2472555217732072" title="DOI URL">https://doi.org/10.1177/2472555217732072</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/2472555217732072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F2472555217732072%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DA%252BLiquid%252BChromatography%25252FMass%252BSpectrometry%252BMethod%252Bfor%252BScreening%252BDisulfide%252BTethering%252BFragments%26aulast%3DHallenbeck%26aufirst%3DKenneth%2BK.%26date%3D2018%26date%3D2017%26volume%3D23%26issue%3D2%26spage%3D183%26epage%3D192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mike  Bührmann</span>, <span class="hlFld-ContribAuthor ">Bianca M.  Wiedemann</span>, <span class="hlFld-ContribAuthor ">Matthias P.  Müller</span>, <span class="hlFld-ContribAuthor ">Julia  Hardick</span>, <span class="hlFld-ContribAuthor ">Maria  Ecke</span>, <span class="hlFld-ContribAuthor ">Daniel  Rauh</span>, . </span><span class="cited-content_cbyCitation_article-title">Structure-based design, synthesis and crystallization of 2-arylquinazolines as lipid pocket ligands of p38α MAPK. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2017,</strong> <em>12 </em>
                                    (9)
                                     , e0184627. <a href="https://doi.org/10.1371/journal.pone.0184627" title="DOI URL">https://doi.org/10.1371/journal.pone.0184627</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0184627&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0184627%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DStructure-based%252Bdesign%25252C%252Bsynthesis%252Band%252Bcrystallization%252Bof%252B2-arylquinazolines%252Bas%252Blipid%252Bpocket%252Bligands%252Bof%252Bp38%2525CE%2525B1%252BMAPK%26aulast%3DB%25C3%25BChrmann%26aufirst%3DMike%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D9%26spage%3De0184627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bin-Chi  Liao</span>, <span class="hlFld-ContribAuthor ">Chia-Chi  Lin</span>, <span class="hlFld-ContribAuthor ">Jih-Hsiang  Lee</span>, <span class="hlFld-ContribAuthor ">James Chih-Hsin  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Optimal management of EGFR -mutant non-small cell lung cancer with disease progression on first-line tyrosine kinase inhibitor therapy. </span><span class="cited-content_cbyCitation_journal-name">Lung Cancer</span><span> <strong>2017,</strong> <em>110 </em>, 7-13. <a href="https://doi.org/10.1016/j.lungcan.2017.05.009" title="DOI URL">https://doi.org/10.1016/j.lungcan.2017.05.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.lungcan.2017.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.lungcan.2017.05.009%26sid%3Dliteratum%253Aachs%26jtitle%3DLung%2520Cancer%26atitle%3DOptimal%252Bmanagement%252Bof%252BEGFR%252B-mutant%252Bnon-small%252Bcell%252Blung%252Bcancer%252Bwith%252Bdisease%252Bprogression%252Bon%252Bfirst-line%252Btyrosine%252Bkinase%252Binhibitor%252Btherapy%26aulast%3DLiao%26aufirst%3DBin-Chi%26date%3D2017%26volume%3D110%26spage%3D7%26epage%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lucas  Robke</span>, <span class="hlFld-ContribAuthor ">Luca  Laraia</span>, <span class="hlFld-ContribAuthor ">Marjorie A.  Carnero Corrales</span>, <span class="hlFld-ContribAuthor ">Georgios  Konstantinidis</span>, <span class="hlFld-ContribAuthor ">Makoto  Muroi</span>, <span class="hlFld-ContribAuthor ">André  Richters</span>, <span class="hlFld-ContribAuthor ">Michael  Winzker</span>, <span class="hlFld-ContribAuthor ">Tobias  Engbring</span>, <span class="hlFld-ContribAuthor ">Stefano  Tomassi</span>, <span class="hlFld-ContribAuthor ">Nobumoto  Watanabe</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Osada</span>, <span class="hlFld-ContribAuthor ">Daniel  Rauh</span>, <span class="hlFld-ContribAuthor ">Herbert  Waldmann</span>, <span class="hlFld-ContribAuthor ">Yao-Wen  Wu</span>, <span class="hlFld-ContribAuthor ">Julian  Engel</span>. </span><span class="cited-content_cbyCitation_article-title">Phenotypic Identification of a Novel Autophagy Inhibitor Chemotype Targeting Lipid Kinase VPS34. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2017,</strong> <em>129 </em>
                                    (28)
                                     , 8265-8269. <a href="https://doi.org/10.1002/ange.201703738" title="DOI URL">https://doi.org/10.1002/ange.201703738</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201703738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201703738%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DPhenotypic%252BIdentification%252Bof%252Ba%252BNovel%252BAutophagy%252BInhibitor%252BChemotype%252BTargeting%252BLipid%252BKinase%252BVPS34%26aulast%3DRobke%26aufirst%3DLucas%26date%3D2017%26date%3D2017%26volume%3D129%26issue%3D28%26spage%3D8265%26epage%3D8269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lucas  Robke</span>, <span class="hlFld-ContribAuthor ">Luca  Laraia</span>, <span class="hlFld-ContribAuthor ">Marjorie A.  Carnero Corrales</span>, <span class="hlFld-ContribAuthor ">Georgios  Konstantinidis</span>, <span class="hlFld-ContribAuthor ">Makoto  Muroi</span>, <span class="hlFld-ContribAuthor ">André  Richters</span>, <span class="hlFld-ContribAuthor ">Michael  Winzker</span>, <span class="hlFld-ContribAuthor ">Tobias  Engbring</span>, <span class="hlFld-ContribAuthor ">Stefano  Tomassi</span>, <span class="hlFld-ContribAuthor ">Nobumoto  Watanabe</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Osada</span>, <span class="hlFld-ContribAuthor ">Daniel  Rauh</span>, <span class="hlFld-ContribAuthor ">Herbert  Waldmann</span>, <span class="hlFld-ContribAuthor ">Yao-Wen  Wu</span>, <span class="hlFld-ContribAuthor ">Julian  Engel</span>. </span><span class="cited-content_cbyCitation_article-title">Phenotypic Identification of a Novel Autophagy Inhibitor Chemotype Targeting Lipid Kinase VPS34. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2017,</strong> <em>56 </em>
                                    (28)
                                     , 8153-8157. <a href="https://doi.org/10.1002/anie.201703738" title="DOI URL">https://doi.org/10.1002/anie.201703738</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201703738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201703738%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DPhenotypic%252BIdentification%252Bof%252Ba%252BNovel%252BAutophagy%252BInhibitor%252BChemotype%252BTargeting%252BLipid%252BKinase%252BVPS34%26aulast%3DRobke%26aufirst%3DLucas%26date%3D2017%26date%3D2017%26volume%3D56%26issue%3D28%26spage%3D8153%26epage%3D8157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Weiwei  Han</span>, <span class="hlFld-ContribAuthor ">Yongli  Du</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Development of the Second and Third Generation Irreversible Epidermal Growth Factor Receptor Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Chemistry & Biodiversity</span><span> <strong>2017,</strong> <em>14 </em>
                                    (7)
                                     , e1600372. <a href="https://doi.org/10.1002/cbdv.201600372" title="DOI URL">https://doi.org/10.1002/cbdv.201600372</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbdv.201600372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbdv.201600372%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%2526%2520Biodiversity%26atitle%3DRecent%252BDevelopment%252Bof%252Bthe%252BSecond%252Band%252BThird%252BGeneration%252BIrreversible%252BEpidermal%252BGrowth%252BFactor%252BReceptor%252BInhibitors%26aulast%3DHan%26aufirst%3DWeiwei%26date%3D2017%26date%3D2017%26volume%3D14%26issue%3D7%26spage%3De1600372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dennis  Plenker</span>, <span class="hlFld-ContribAuthor ">Maximilian  Riedel</span>, <span class="hlFld-ContribAuthor ">Johannes  Brägelmann</span>, <span class="hlFld-ContribAuthor ">Marcel A.  Dammert</span>, <span class="hlFld-ContribAuthor ">Rakhee  Chauhan</span>, <span class="hlFld-ContribAuthor ">Phillip P.  Knowles</span>, <span class="hlFld-ContribAuthor ">Carina  Lorenz</span>, <span class="hlFld-ContribAuthor ">Marina  Keul</span>, <span class="hlFld-ContribAuthor ">Mike  Bührmann</span>, <span class="hlFld-ContribAuthor ">Oliver  Pagel</span>, <span class="hlFld-ContribAuthor ">Verena  Tischler</span>, <span class="hlFld-ContribAuthor ">Andreas H.  Scheel</span>, <span class="hlFld-ContribAuthor ">Daniel  Schütte</span>, <span class="hlFld-ContribAuthor ">Yanrui  Song</span>, <span class="hlFld-ContribAuthor ">Justina  Stark</span>, <span class="hlFld-ContribAuthor ">Florian  Mrugalla</span>, <span class="hlFld-ContribAuthor ">Yannic  Alber</span>, <span class="hlFld-ContribAuthor ">André  Richters</span>, <span class="hlFld-ContribAuthor ">Julian  Engel</span>, <span class="hlFld-ContribAuthor ">Frauke  Leenders</span>, <span class="hlFld-ContribAuthor ">Johannes M.  Heuckmann</span>, <span class="hlFld-ContribAuthor ">Jürgen  Wolf</span>, <span class="hlFld-ContribAuthor ">Joachim  Diebold</span>, <span class="hlFld-ContribAuthor ">Georg  Pall</span>, <span class="hlFld-ContribAuthor ">Martin  Peifer</span>, <span class="hlFld-ContribAuthor ">Maarten  Aerts</span>, <span class="hlFld-ContribAuthor ">Kris  Gevaert</span>, <span class="hlFld-ContribAuthor ">René P.  Zahedi</span>, <span class="hlFld-ContribAuthor ">Reinhard  Buettner</span>, <span class="hlFld-ContribAuthor ">Kevan M.  Shokat</span>, <span class="hlFld-ContribAuthor ">Neil Q.  McDonald</span>, <span class="hlFld-ContribAuthor ">Stefan M.  Kast</span>, <span class="hlFld-ContribAuthor ">Oliver  Gautschi</span>, <span class="hlFld-ContribAuthor ">Roman K.  Thomas</span>, <span class="hlFld-ContribAuthor ">Martin L.  Sos</span>. </span><span class="cited-content_cbyCitation_article-title">Drugging the catalytically inactive state of RET kinase in RET-rearranged tumors. </span><span class="cited-content_cbyCitation_journal-name">Science Translational Medicine</span><span> <strong>2017,</strong> <em>9 </em>
                                    (394)
                                     , eaah6144. <a href="https://doi.org/10.1126/scitranslmed.aah6144" title="DOI URL">https://doi.org/10.1126/scitranslmed.aah6144</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1126/scitranslmed.aah6144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1126%2Fscitranslmed.aah6144%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%2520Translational%2520Medicine%26atitle%3DDrugging%252Bthe%252Bcatalytically%252Binactive%252Bstate%252Bof%252BRET%252Bkinase%252Bin%252BRET-rearranged%252Btumors%26aulast%3DPlenker%26aufirst%3DDennis%26date%3D2017%26date%3D2017%26volume%3D9%26issue%3D394%26spage%3Deaah6144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Haixing  Guan</span>, <span class="hlFld-ContribAuthor ">Yongli  Du</span>, <span class="hlFld-ContribAuthor ">Yang  Ning</span>, <span class="hlFld-ContribAuthor ">Xin  Cao</span>. </span><span class="cited-content_cbyCitation_article-title">A brief perspective of drug resistance toward EGFR inhibitors: the crystal structures of EGFRs and their variants. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2017,</strong> <em>9 </em>
                                    (7)
                                     , 693-704. <a href="https://doi.org/10.4155/fmc-2016-0222" title="DOI URL">https://doi.org/10.4155/fmc-2016-0222</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2016-0222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2016-0222%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DA%252Bbrief%252Bperspective%252Bof%252Bdrug%252Bresistance%252Btoward%252BEGFR%252Binhibitors%25253A%252Bthe%252Bcrystal%252Bstructures%252Bof%252BEGFRs%252Band%252Btheir%252Bvariants%26aulast%3DGuan%26aufirst%3DHaixing%26date%3D2017%26volume%3D9%26issue%3D7%26spage%3D693%26epage%3D704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ramalingam  Boobalan</span>, <span class="hlFld-ContribAuthor ">Kuang-Kai  Liu</span>, <span class="hlFld-ContribAuthor ">Jui-I.  Chao</span>, <span class="hlFld-ContribAuthor ">Chinpiao  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and biological assay of erlotinib analogues and BSA-conjugated erlotinib analogue. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>27 </em>
                                    (8)
                                     , 1784-1788. <a href="https://doi.org/10.1016/j.bmcl.2017.02.059" title="DOI URL">https://doi.org/10.1016/j.bmcl.2017.02.059</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2017.02.059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2017.02.059%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSynthesis%252Band%252Bbiological%252Bassay%252Bof%252Berlotinib%252Banalogues%252Band%252BBSA-conjugated%252Berlotinib%252Banalogue%26aulast%3DBoobalan%26aufirst%3DRamalingam%26date%3D2017%26volume%3D27%26issue%3D8%26spage%3D1784%26epage%3D1788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuhang  Wang</span>, <span class="hlFld-ContribAuthor ">Yongping  Song</span>, <span class="hlFld-ContribAuthor ">Delong  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">EAI045: The fourth-generation EGFR inhibitor overcoming T790M and C797S resistance. </span><span class="cited-content_cbyCitation_journal-name">Cancer Letters</span><span> <strong>2017,</strong> <em>385 </em>, 51-54. <a href="https://doi.org/10.1016/j.canlet.2016.11.008" title="DOI URL">https://doi.org/10.1016/j.canlet.2016.11.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.canlet.2016.11.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.canlet.2016.11.008%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Letters%26atitle%3DEAI045%25253A%252BThe%252Bfourth-generation%252BEGFR%252Binhibitor%252Bovercoming%252BT790M%252Band%252BC797S%252Bresistance%26aulast%3DWang%26aufirst%3DShuhang%26date%3D2017%26volume%3D385%26spage%3D51%26epage%3D54" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qiang  Fu</span>, <span class="hlFld-ContribAuthor ">Jing  Cheng</span>, <span class="hlFld-ContribAuthor ">Jindai  Zhang</span>, <span class="hlFld-ContribAuthor ">Yonglei  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiaobing  Chen</span>, <span class="hlFld-ContribAuthor ">Suxia  Luo</span>, <span class="hlFld-ContribAuthor ">Jianguo  Xie</span>. </span><span class="cited-content_cbyCitation_article-title">miR-20b reduces 5-FU resistance by suppressing the ADAM9/EGFR signaling pathway in colon cancer. </span><span class="cited-content_cbyCitation_journal-name">Oncology Reports</span><span> <strong>2017,</strong> <em>37 </em>
                                    (1)
                                     , 123-130. <a href="https://doi.org/10.3892/or.2016.5259" title="DOI URL">https://doi.org/10.3892/or.2016.5259</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3892/or.2016.5259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3892%2For.2016.5259%26sid%3Dliteratum%253Aachs%26jtitle%3DOncology%2520Reports%26atitle%3DmiR-20b%252Breduces%252B5-FU%252Bresistance%252Bby%252Bsuppressing%252Bthe%252BADAM9%25252FEGFR%252Bsignaling%252Bpathway%252Bin%252Bcolon%252Bcancer%26aulast%3DFu%26aufirst%3DQiang%26date%3D2017%26date%3D2016%26volume%3D37%26issue%3D1%26spage%3D123%26epage%3D130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuhang  Wang</span>, <span class="hlFld-ContribAuthor ">Yongping  Song</span>, <span class="hlFld-ContribAuthor ">Feifei  Yan</span>, <span class="hlFld-ContribAuthor ">Delong  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Mechanisms of resistance to third-generation EGFR tyrosine kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Frontiers of Medicine</span><span> <strong>2016,</strong> <em>10 </em>
                                    (4)
                                     , 383-388. <a href="https://doi.org/10.1007/s11684-016-0488-1" title="DOI URL">https://doi.org/10.1007/s11684-016-0488-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11684-016-0488-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11684-016-0488-1%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520of%2520Medicine%26atitle%3DMechanisms%252Bof%252Bresistance%252Bto%252Bthird-generation%252BEGFR%252Btyrosine%252Bkinase%252Binhibitors%26aulast%3DWang%26aufirst%3DShuhang%26date%3D2016%26date%3D2016%26volume%3D10%26issue%3D4%26spage%3D383%26epage%3D388" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bin-Chi  Liao</span>, <span class="hlFld-ContribAuthor ">Chia-Chi  Lin</span>, <span class="hlFld-ContribAuthor ">Jih-Hsiang  Lee</span>, <span class="hlFld-ContribAuthor ">James Chih-Hsin  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Update on recent preclinical and clinical studies of T790M mutant-specific irreversible epidermal growth factor receptor tyrosine kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomedical Science</span><span> <strong>2016,</strong> <em>23 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12929-016-0305-9" title="DOI URL">https://doi.org/10.1186/s12929-016-0305-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12929-016-0305-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12929-016-0305-9%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomedical%2520Science%26atitle%3DUpdate%252Bon%252Brecent%252Bpreclinical%252Band%252Bclinical%252Bstudies%252Bof%252BT790M%252Bmutant-specific%252Birreversible%252Bepidermal%252Bgrowth%252Bfactor%252Breceptor%252Btyrosine%252Bkinase%252Binhibitors%26aulast%3DLiao%26aufirst%3DBin-Chi%26date%3D2016%26date%3D2016%26volume%3D23%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Karamatullah  Danyal</span>, <span class="hlFld-ContribAuthor ">Willem  de Jong</span>, <span class="hlFld-ContribAuthor ">Edmund  O'Brien</span>, <span class="hlFld-ContribAuthor ">Robert A.  Bauer</span>, <span class="hlFld-ContribAuthor ">David E.  Heppner</span>, <span class="hlFld-ContribAuthor ">Andrew C.  Little</span>, <span class="hlFld-ContribAuthor ">Milena  Hristova</span>, <span class="hlFld-ContribAuthor ">Aida  Habibovic</span>, <span class="hlFld-ContribAuthor ">Albert  van der Vliet</span>. </span><span class="cited-content_cbyCitation_article-title">Acrolein and thiol-reactive electrophiles suppress allergen-induced innate airway epithelial responses by inhibition of DUOX1 and EGFR. </span><span class="cited-content_cbyCitation_journal-name">American Journal of Physiology-Lung Cellular and Molecular Physiology</span><span> <strong>2016,</strong> <em>311 </em>
                                    (5)
                                     , L913-L923. <a href="https://doi.org/10.1152/ajplung.00276.2016" title="DOI URL">https://doi.org/10.1152/ajplung.00276.2016</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1152/ajplung.00276.2016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1152%2Fajplung.00276.2016%26sid%3Dliteratum%253Aachs%26jtitle%3DAmerican%2520Journal%2520of%2520Physiology-Lung%2520Cellular%2520and%2520Molecular%2520Physiology%26atitle%3DAcrolein%252Band%252Bthiol-reactive%252Belectrophiles%252Bsuppress%252Ballergen-induced%252Binnate%252Bairway%252Bepithelial%252Bresponses%252Bby%252Binhibition%252Bof%252BDUOX1%252Band%252BEGFR%26aulast%3DDanyal%26aufirst%3DKaramatullah%26date%3D2016%26volume%3D311%26issue%3D5%26spage%3DL913%26epage%3DL923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Julian  Engel</span>, <span class="hlFld-ContribAuthor ">Christian  Becker</span>, <span class="hlFld-ContribAuthor ">Jonas  Lategahn</span>, <span class="hlFld-ContribAuthor ">Marina  Keul</span>, <span class="hlFld-ContribAuthor ">Julia  Ketzer</span>, <span class="hlFld-ContribAuthor ">Thomas  Mühlenberg</span>, <span class="hlFld-ContribAuthor ">Laxmikanth  Kollipara</span>, <span class="hlFld-ContribAuthor ">Carsten  Schultz-Fademrecht</span>, <span class="hlFld-ContribAuthor ">René P.  Zahedi</span>, <span class="hlFld-ContribAuthor ">Sebastian  Bauer</span>, <span class="hlFld-ContribAuthor ">Daniel  Rauh</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibition wirkstoffresistenter Mutationsvarianten der Rezeptortyrosinkinase EGFR. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2016,</strong> <em>128 </em>
                                    (36)
                                     , 11069-11073. <a href="https://doi.org/10.1002/ange.201605011" title="DOI URL">https://doi.org/10.1002/ange.201605011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201605011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201605011%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DInhibition%252Bwirkstoffresistenter%252BMutationsvarianten%252Bder%252BRezeptortyrosinkinase%252BEGFR%26aulast%3DEngel%26aufirst%3DJulian%26date%3D2016%26date%3D2016%26volume%3D128%26issue%3D36%26spage%3D11069%26epage%3D11073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Julian  Engel</span>, <span class="hlFld-ContribAuthor ">Christian  Becker</span>, <span class="hlFld-ContribAuthor ">Jonas  Lategahn</span>, <span class="hlFld-ContribAuthor ">Marina  Keul</span>, <span class="hlFld-ContribAuthor ">Julia  Ketzer</span>, <span class="hlFld-ContribAuthor ">Thomas  Mühlenberg</span>, <span class="hlFld-ContribAuthor ">Laxmikanth  Kollipara</span>, <span class="hlFld-ContribAuthor ">Carsten  Schultz-Fademrecht</span>, <span class="hlFld-ContribAuthor ">René P.  Zahedi</span>, <span class="hlFld-ContribAuthor ">Sebastian  Bauer</span>, <span class="hlFld-ContribAuthor ">Daniel  Rauh</span>. </span><span class="cited-content_cbyCitation_article-title">Insight into the Inhibition of Drug-Resistant Mutants of the Receptor Tyrosine Kinase EGFR. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2016,</strong> <em>55 </em>
                                    (36)
                                     , 10909-10912. <a href="https://doi.org/10.1002/anie.201605011" title="DOI URL">https://doi.org/10.1002/anie.201605011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201605011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201605011%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DInsight%252Binto%252Bthe%252BInhibition%252Bof%252BDrug-Resistant%252BMutants%252Bof%252Bthe%252BReceptor%252BTyrosine%252BKinase%252BEGFR%26aulast%3DEngel%26aufirst%3DJulian%26date%3D2016%26date%3D2016%26volume%3D55%26issue%3D36%26spage%3D10909%26epage%3D10912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christian  Becker</span>, <span class="hlFld-ContribAuthor ">Sinan  Öcal</span>, <span class="hlFld-ContribAuthor ">Hoang D.  Nguyen</span>, <span class="hlFld-ContribAuthor ">Trang  Phan</span>, <span class="hlFld-ContribAuthor ">Marina  Keul</span>, <span class="hlFld-ContribAuthor ">Jeffrey R.  Simard</span>, <span class="hlFld-ContribAuthor ">Daniel  Rauh</span>. </span><span class="cited-content_cbyCitation_article-title">Monitoring Conformational Changes in the Receptor Tyrosine Kinase EGFR. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2016,</strong> <em>17 </em>
                                    (11)
                                     , 990-994. <a href="https://doi.org/10.1002/cbic.201600115" title="DOI URL">https://doi.org/10.1002/cbic.201600115</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.201600115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.201600115%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DMonitoring%252BConformational%252BChanges%252Bin%252Bthe%252BReceptor%252BTyrosine%252BKinase%252BEGFR%26aulast%3DBecker%26aufirst%3DChristian%26date%3D2016%26date%3D2016%26volume%3D17%26issue%3D11%26spage%3D990%26epage%3D994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sang Min  Lim</span>, <span class="hlFld-ContribAuthor ">Yujeong  Jeong</span>, <span class="hlFld-ContribAuthor ">Sungwoo  Hong</span>. </span><span class="cited-content_cbyCitation_article-title">Strategies to overcome acquired resistances conferred by mutations in the kinase domain of EGFR. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2016,</strong> <em>8 </em>
                                    (8)
                                     , 853-878. <a href="https://doi.org/10.4155/fmc-2016-0019" title="DOI URL">https://doi.org/10.4155/fmc-2016-0019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2016-0019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2016-0019%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DStrategies%252Bto%252Bovercome%252Bacquired%252Bresistances%252Bconferred%252Bby%252Bmutations%252Bin%252Bthe%252Bkinase%252Bdomain%252Bof%252BEGFR%26aulast%3DLim%26aufirst%3DSang%2BMin%26date%3D2016%26volume%3D8%26issue%3D8%26spage%3D853%26epage%3D878" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christoph  Hölzl</span>, <span class="hlFld-ContribAuthor ">Patrick  Kibies</span>, <span class="hlFld-ContribAuthor ">Sho  Imoto</span>, <span class="hlFld-ContribAuthor ">Roland  Frach</span>, <span class="hlFld-ContribAuthor ">Saba  Suladze</span>, <span class="hlFld-ContribAuthor ">Roland  Winter</span>, <span class="hlFld-ContribAuthor ">Dominik  Marx</span>, <span class="hlFld-ContribAuthor ">Dominik  Horinek</span>, <span class="hlFld-ContribAuthor ">Stefan M.  Kast</span>. </span><span class="cited-content_cbyCitation_article-title">Design principles for high–pressure force fields: Aqueous TMAO solutions from ambient to kilobar pressures. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Chemical Physics</span><span> <strong>2016,</strong> <em>144 </em>
                                    (14)
                                     , 144104. <a href="https://doi.org/10.1063/1.4944991" title="DOI URL">https://doi.org/10.1063/1.4944991</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1063/1.4944991&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1063%2F1.4944991%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Chemical%2520Physics%26atitle%3DDesign%252Bprinciples%252Bfor%252Bhigh%2525E2%252580%252593pressure%252Bforce%252Bfields%25253A%252BAqueous%252BTMAO%252Bsolutions%252Bfrom%252Bambient%252Bto%252Bkilobar%252Bpressures%26aulast%3DH%25C3%25B6lzl%26aufirst%3DChristoph%26date%3D2016%26volume%3D144%26issue%3D14%26spage%3D144104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hengmiao  Cheng</span>, <span class="hlFld-ContribAuthor ">Sajiv K.  Nair</span>, <span class="hlFld-ContribAuthor ">Brion W.  Murray</span>. </span><span class="cited-content_cbyCitation_article-title">Recent progress on third generation covalent EGFR inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2016,</strong> <em>26 </em>
                                    (8)
                                     , 1861-1868. <a href="https://doi.org/10.1016/j.bmcl.2016.02.067" title="DOI URL">https://doi.org/10.1016/j.bmcl.2016.02.067</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2016.02.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2016.02.067%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DRecent%252Bprogress%252Bon%252Bthird%252Bgeneration%252Bcovalent%252BEGFR%252Binhibitors%26aulast%3DCheng%26aufirst%3DHengmiao%26date%3D2016%26volume%3D26%26issue%3D8%26spage%3D1861%26epage%3D1868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Concepción  González-Bello</span>. </span><span class="cited-content_cbyCitation_article-title">Designing Irreversible Inhibitors-Worth the Effort?. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2016,</strong> <em>11 </em>
                                    (1)
                                     , 22-30. <a href="https://doi.org/10.1002/cmdc.201500469" title="DOI URL">https://doi.org/10.1002/cmdc.201500469</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201500469&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201500469%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DDesigning%252BIrreversible%252BInhibitors-Worth%252Bthe%252BEffort%25253F%26aulast%3DGonz%25C3%25A1lez-Bello%26aufirst%3DConcepci%25C3%25B3n%26date%3D2016%26date%3D2015%26volume%3D11%26issue%3D1%26spage%3D22%26epage%3D30" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuai  Wang</span>, <span class="hlFld-ContribAuthor ">Feng  Liu</span>, <span class="hlFld-ContribAuthor ">Jingyan  Zhu</span>, <span class="hlFld-ContribAuthor ">Peng  Chen</span>, <span class="hlFld-ContribAuthor ">Hongxing  Liu</span>, <span class="hlFld-ContribAuthor ">Qi  Liu</span>, <span class="hlFld-ContribAuthor ">Junqing  Han</span>. </span><span class="cited-content_cbyCitation_article-title">DNA Repair Genes ERCC1 and BRCA1 Expression in Non-Small Cell Lung Cancer Chemotherapy Drug Resistance. </span><span class="cited-content_cbyCitation_journal-name">Medical Science Monitor</span><span> <strong>2016,</strong> <em>22 </em>, 1999-2005. <a href="https://doi.org/10.12659/MSM.896606" title="DOI URL">https://doi.org/10.12659/MSM.896606</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.12659/MSM.896606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.12659%2FMSM.896606%26sid%3Dliteratum%253Aachs%26jtitle%3DMedical%2520Science%2520Monitor%26atitle%3DDNA%252BRepair%252BGenes%252BERCC1%252Band%252BBRCA1%252BExpression%252Bin%252BNon-Small%252BCell%252BLung%252BCancer%252BChemotherapy%252BDrug%252BResistance%26aulast%3DWang%26aufirst%3DShuai%26date%3D2016%26date%3D2016%26volume%3D22%26spage%3D1999%26epage%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01082&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Covalent and reversible EGFR inhibitors. (A) First-, second-, and third-generation EGFR inhibitors. (B) Screening hit <b>1a</b> from a phenotype screen of 80 NSCLC cell lines. (C) X-ray crystal structure of <b>1a</b> in complex with cSrc-DM (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D12">5D12</a>). Michael acceptors are highlighted (green).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01082&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Modeling studies of various hinge-binding motifs. (A) Quinazoline- (<b>2b</b>) and (B–D) pyrimidine-based inhibitors (<b>3a</b>, <b>5b</b>, and <b>6a</b>) based on the X-ray crystal structure of <b>1a</b> in complex with cSrc-DM (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D12">5D12</a>). Proposed hydrogen bonds are shown as red dotted lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01082&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0008.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of a Subset of Pyrimidine Derivatives (<b>3a</b>–<b>b</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01082&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) 3-aminopyrazole, NaI, HCl (cat.), <i>n</i>-BuOH, 120 °C, 72%; (ii) Boc<sub>2</sub>O, Et<sub>3</sub>N, DCM, rt, 21%; (iii) acryloyl or propionyl chloride, DIPEA, THF, 0 °C; (iv) TFA in DCM, rt, 58–64% (two steps).</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0009.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of a Subset of Pyrimidine Derivatives (<b>4a</b>, <b>5l</b>–<b>m</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01082&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) 5-methyl-1<i>H</i>-pyrazol-3-amine, DIPEA, DMF, 90 °C, 54%; (ii) DHP, <i>p</i>TsOH, THF, 90 °C, 64%; (iii) (3-nitrophenyl)boronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, DMF/H<sub>2</sub>O (9:1), 90 °C, 70%; (iv) ammonium formate, Pd/C, EtOH, 90 °C; (v) acryloyl chloride, DIPEA, THF, 0 °C, 26% (<b>5l</b>); (vi) TFA in DCM, rt, 73% (two steps); (vii) 3-nitrophenol, K<sub>2</sub>CO<sub>3</sub>, DMF, 140 °C, 20%; (viii) propionyl chloride, DIPEA, THF, 0 °C, 27%.</p></p></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0010.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of a Subset of Pyrimidine Derivatives (<b>5a</b>–<b>k</b>, <b>5n</b>–<b>q</b>, <b>6a</b>–<b>d</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01082&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) <i>m</i>CPBA, DCM/THF, 0 °C – rt, 86%; (ii) nitrophenol, NaH, THF, 0 °C – rt, 29–84%; (iii) amine, DIPEA, NaI, DMF, 85–110 °C, 60–82%; (iv) methylpiperazine, 110 °C, 62–82%; (v) Boc<sub>2</sub>O, Et<sub>3</sub>N, MeOH, rt or Boc<sub>2</sub>O, Cs<sub>2</sub>CO<sub>3</sub>, THF, rt, 56–95%; (vi) ammonium formate, Pd/C, EtOH or EtOAc, 90 °C, 48–87%; (vii) acryloyl or propionyl chloride, DIPEA, THF, 0 °C; (viii) (<i>E</i>)-4-(dimethylamino)but-2-enoic acid or (<i>E</i>)-4,4,4-trifluorobut-2-enoic acid, EDC, HOBt, DIPEA, DCM, rt or (<i>E</i>)-4-(dimethylamino)but-2-enoic acid, oxalyl chloride, DMF(cat.), THF, rt; (ix) TFA in DCM, rt, 33–95% (two steps).</p></p></figure><figure data-id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Extracted ion chromatograms (XICs) of the three different isoforms of the peptide GCLLDYVR identified in (A) the control sample and after treatment with (B) <b>5b</b> and (C) <b>6a</b>. As expected, the carbamidomethylated (i.e., nonmodified) peptide is almost absent in samples the inhibitor-treated samples, whereas the corresponding modified versions can be detected only in the respective samples. All XICs are shown with a tolerance of 4 ppm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01082&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Pyrimidine-based compounds in complex with cSrc-DM. Diagrams of the experimental electron densities of cSrc-DM–<b>5b</b> at 2.3 Å (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D11">5D11</a>) (A) and cSrc-DM–<b>5c</b> at 2.7 Å (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D10">5D10</a>) (B) resolution are shown (2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map contoured at 1σ). Hydrogen-bond interactions of the inhibitors with the hinge region (wheat) are illustrated by red dotted lines. The structural elements comprising the DFG motif as well as helix C are displayed in violet and cyan, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01082&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0006.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Minimum free energy conformations and conformational basins of inhibitors. Left: Structures in their predominant protonation state. Right: Population diagrams resulting from the computational analysis of conformational basins (identified by different colors) of selected compounds for different methods to treat the solvation contribution to the total free energy (PCM, triangles; EC-RISM, squares). Top: WZ4002 (note that a single PCM data point at RMSD = 0.222 nm with population 0.43 is not shown in order to limit the scale). Bottom: WZ4002-H (Cl substituted by H). Middle: Superimposition of corresponding minimum free energy conformations per basin (also shown as insets); red models (with blue balls representing nitrogen) show the respective reference conformation (WZ4002, PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3IKA">3IKA</a>) of the ligand in the binding site. Boundaries between basins are (in nm, from left to right) for WZ4002, group 1: RMSD < 0.16, 2: 0.16 ≤ RMSD < 0.3, 3: 0.3 ≤ RMSD < 0.38, 4: 0.38 ≤ RMSD; for WZ4002-H, group 1: RMSD < 0.2, 2: 0.2 ≤ RMSD < 0.36, 0.36 ≤ RMSD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01082&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/medium/jm-2015-01082e_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0007.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Population analysis results for (from top to bottom) <b>5b</b>, <b>5n</b>, <b>6a</b> in analogy to <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>. Reference conformation (red model) in all cases is the crystallographic binding mode of <b>5b</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D11">5D11</a>). Boundaries between basins are (in nm, from left to right) for <b>5b</b>, group 1: RMSD < 0.16, 2: 0.16 ≤ RMSD < 0.25, 3: 0.25 ≤ RMSD; <b>5n</b>, group 1: RMSD < 0.14, 2: 0.14 ≤ RMSD < 0.25, 3: 0.25 ≤ RMSD; <b>6a</b>, group 1: RMSD < 0.15, 2: 0.15 ≤ RMSD < 0.25, 3: 0.25 ≤ RMSD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-17/acs.jmedchem.5b01082/20150904/images/large/jm-2015-01082e_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01082&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i37">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84741" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84741" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 87 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekar, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, A.</span><span> </span><span class="NLM_article-title">Epidermal growth factor receptor-mediated cell motility: phospholipase C activity is required, but mitogen-activated protein kinase activity is not sufficient for induced cell movement</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">127</span><span class="NLM_x">, </span> <span class="NLM_fpage">847</span><span class="NLM_x">–</span> <span class="NLM_lpage">857</span><span class="refDoi"> DOI: 10.1083/jcb.127.3.847</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1083%2Fjcb.127.3.847" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=7962064" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaK2cXmsVOitLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=1994&pages=847-857&author=P.+Chenauthor=H.+Xieauthor=M.+C.+Sekarauthor=K.+Guptaauthor=A.+Wells&title=Epidermal+growth+factor+receptor-mediated+cell+motility%3A+phospholipase+C+activity+is+required%2C+but+mitogen-activated+protein+kinase+activity+is+not+sufficient+for+induced+cell+movement&doi=10.1083%2Fjcb.127.3.847"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal growth factor receptor-mediated cell motility: phospholipase C activity is required, but mitogen-activated protein kinase activity is not sufficient for induced cell movement</span></div><div class="casAuthors">Chen, Philip; Xie, Heng; Sekar, M. Chandra; Gupta, Kiran; Wells, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Biology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">847-57</span>CODEN:
                <span class="NLM_cas:coden">JCLBA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9525</span>.
    </div><div class="casAbstract">The authors utilized signaling-restricted EGFR mutants expressed in receptor devoid NR6 cells to det. if PLC activation is necessary for EGFR-mediated cell movement.  Exposure to EGF (25 nm) augmented PLC activity in all five EGFR mutant cell lines which also responded by increased cell movement.  Basal phosphoinositide turnover was not affected by EGF in the lines which do not present the enhanced motility response.  The correlation between EGFR-mediated cell motility and PLC activity suggested, but did not prove, a causal link.  A specific inhibitor of PLC, U73122 (1 μM) diminished both the EGF-induced motility and PLC responses, while its inactive analog U73343 had no effect on these responses.  Both the PLC and motility responses were decreased by expression of a dominant-neg. PLCγ-1 fragment in EGF-responsive infectant lines.  Lastly, anti-sense oligonucleotides (10 μM) to PLCγ-1 reduced both responses in NR6 cells expressing wild-type EGFR.  These findings strongly support PLCγ as the immediate post receptor effector in this motogenic pathway.  The authors have demonstrated previously that EGFR-mediated cell motility and mitogenic pathway.  The authors have demonstrated previously that EGFR-mediated cell motility and mitogenic signaling pathways are separable.  The point of divergence is undefined.  All kinase-active EGFR mutants induced the mitogenic response while only those which are autophosphorylated induced PLC activity.  U73122 did not affect EGF-induced thymidine incorporation in these motility-responsive infectant cell lines.  In addn., the dominant-neg. PLCγ-1 fragment did not diminish EGF-induced thymidine incorporation.  All kinase active EGFR stimulated mitogen-activated protein (MAP) kinase activity, regardless of whether the receptors induced cell movement; this EGF-induced MAP kinase activity was not affected by U73122 at concns. that depressed the motility response.  Thus, the signaling pathways which lead to motility and cell proliferation diverge at the immediate post-receptor stage, and the authors suggest that this is accomplished by differential activation of effector mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBfocjcTbqMLVg90H21EOLACvtfcHk0lh06fW4GwT5AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmsVOitLk%253D&md5=9557cc67efc30b158a929bafb216a96a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1083%2Fjcb.127.3.847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.127.3.847%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DP.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DSekar%26aufirst%3DM.%2BC.%26aulast%3DGupta%26aufirst%3DK.%26aulast%3DWells%26aufirst%3DA.%26atitle%3DEpidermal%2520growth%2520factor%2520receptor-mediated%2520cell%2520motility%253A%2520phospholipase%2520C%2520activity%2520is%2520required%252C%2520but%2520mitogen-activated%2520protein%2520kinase%2520activity%2520is%2520not%2520sufficient%2520for%2520induced%2520cell%2520movement%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D1994%26volume%3D127%26spage%3D847%26epage%3D857%26doi%3D10.1083%2Fjcb.127.3.847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Datta, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dudek, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masters, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gotoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenberg, M. E.</span><span> </span><span class="NLM_article-title">Akt phosphorylation of BAD couples survival signals to the cell-intrinsic death machinery</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">231</span><span class="NLM_x">–</span> <span class="NLM_lpage">241</span><span class="refDoi"> DOI: 10.1016/S0092-8674(00)80405-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1016%2FS0092-8674%2800%2980405-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=9346240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaK2sXntVyiu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=1997&pages=231-241&author=S.+R.+Dattaauthor=H.+Dudekauthor=X.+Taoauthor=S.+Mastersauthor=H.+Fuauthor=Y.+Gotohauthor=M.+E.+Greenberg&title=Akt+phosphorylation+of+BAD+couples+survival+signals+to+the+cell-intrinsic+death+machinery&doi=10.1016%2FS0092-8674%2800%2980405-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Akt phosphorylation of BAD couples survival signals to the cell-intrinsic death machinery</span></div><div class="casAuthors">Datta, Sandeep Robert; Dudek, Henryk; Tao, Xu; Masters, Shane; Fu, Haian; Gotoh, Yukiko; Greenberg, Michael E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">231-241</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Growth factors can promote cell survival by activating the phosphatidylinositide-3'-OH kinase and its downstream target, the serine-threonine kinase Akt.  However, the mechanism by which Akt functions to promote survival is not understood.  The authors show that growth factor activation of the PI3'-K/Akt signaling pathway culminates in the phosphorylation of the BCL-2 family member BAD, thereby suppressing apoptosis and promoting cell survival.  Akt phosphorylates BAD in vitro and in vivo, and blocks the BAD-induced death of primary neurons in a site-specific manner.  These findings define a mechanism by which growth factors directly inactivate a crit. component of the cell-intrinsic death machinery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiB9FN9LI-RLVg90H21EOLACvtfcHk0lh06fW4GwT5AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXntVyiu7k%253D&md5=7cec99113fddfdf23c290bec26a9f0dc</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2980405-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252980405-5%26sid%3Dliteratum%253Aachs%26aulast%3DDatta%26aufirst%3DS.%2BR.%26aulast%3DDudek%26aufirst%3DH.%26aulast%3DTao%26aufirst%3DX.%26aulast%3DMasters%26aufirst%3DS.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DGotoh%26aufirst%3DY.%26aulast%3DGreenberg%26aufirst%3DM.%2BE.%26atitle%3DAkt%2520phosphorylation%2520of%2520BAD%2520couples%2520survival%2520signals%2520to%2520the%2520cell-intrinsic%2520death%2520machinery%26jtitle%3DCell%26date%3D1997%26volume%3D91%26spage%3D231%26epage%3D241%26doi%3D10.1016%2FS0092-8674%2800%2980405-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Pages, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lenormand, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">L’Allemain, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambard, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meloche, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pouyssegur, J.</span><span> </span><span class="NLM_article-title">Mitogen-activated protein kinases p42mapk and p44mapk are required for fibroblast proliferation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">90</span><span class="NLM_x">, </span> <span class="NLM_fpage">8319</span><span class="NLM_x">–</span> <span class="NLM_lpage">8323</span><span class="refDoi"> DOI: 10.1073/pnas.90.18.8319</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1073%2Fpnas.90.18.8319" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1993&pages=8319-8323&author=G.+Pagesauthor=P.+Lenormandauthor=G.+L%E2%80%99Allemainauthor=J.+C.+Chambardauthor=S.+Melocheauthor=J.+Pouyssegur&title=Mitogen-activated+protein+kinases+p42mapk+and+p44mapk+are+required+for+fibroblast+proliferation&doi=10.1073%2Fpnas.90.18.8319"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1073%2Fpnas.90.18.8319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.90.18.8319%26sid%3Dliteratum%253Aachs%26aulast%3DPages%26aufirst%3DG.%26aulast%3DLenormand%26aufirst%3DP.%26aulast%3DL%25E2%2580%2599Allemain%26aufirst%3DG.%26aulast%3DChambard%26aufirst%3DJ.%2BC.%26aulast%3DMeloche%26aufirst%3DS.%26aulast%3DPouyssegur%26aufirst%3DJ.%26atitle%3DMitogen-activated%2520protein%2520kinases%2520p42mapk%2520and%2520p44mapk%2520are%2520required%2520for%2520fibroblast%2520proliferation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1993%26volume%3D90%26spage%3D8319%26epage%3D8323%26doi%3D10.1073%2Fpnas.90.18.8319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Rosell, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moran, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Queralt, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porta, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cardenal, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camps, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majem, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez-Vivanco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isla, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Provencio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Massuti, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez-Larriba, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paz-Ares, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bover, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia-Campelo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moreno, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Catot, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rolfo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reguart, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmero, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanchez, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bertran-Alamillo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Molina, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanchez, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taron, M.</span><span> </span><span class="NLM_article-title">Spanish Lung Cancer, G. Screening for epidermal growth factor receptor mutations in lung cancer</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">361</span><span class="NLM_x">, </span> <span class="NLM_fpage">958</span><span class="NLM_x">–</span> <span class="NLM_lpage">967</span><span class="refDoi"> DOI: 10.1056/NEJMoa0904554</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1056%2FNEJMoa0904554" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=19692684" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVOqsL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=361&publication_year=2009&pages=958-967&author=R.+Rosellauthor=T.+Moranauthor=C.+Queraltauthor=R.+Portaauthor=F.+Cardenalauthor=C.+Campsauthor=M.+Majemauthor=G.+Lopez-Vivancoauthor=D.+Islaauthor=M.+Provencioauthor=A.+Insaauthor=B.+Massutiauthor=J.+L.+Gonzalez-Larribaauthor=L.+Paz-Aresauthor=I.+Boverauthor=R.+Garcia-Campeloauthor=M.+A.+Morenoauthor=S.+Catotauthor=C.+Rolfoauthor=N.+Reguartauthor=R.+Palmeroauthor=J.+M.+Sanchezauthor=R.+Bastusauthor=C.+Mayoauthor=J.+Bertran-Alamilloauthor=M.+A.+Molinaauthor=J.+J.+Sanchezauthor=M.+Taron&title=Spanish+Lung+Cancer%2C+G.+Screening+for+epidermal+growth+factor+receptor+mutations+in+lung+cancer&doi=10.1056%2FNEJMoa0904554"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Screening for epidermal growth factor receptor mutations in lung cancer</span></div><div class="casAuthors">Rosell, Rafael; Moran, Teresa; Queralt, Cristina; Porta, Rut; Cardenal, Felipe; Camps, Carlos; Majem, Margarita; Lopez-Vivanco, Guillermo; Isla, Dolores; Provencio, Mariano; Insa, Amelia; Massuti, Bartomeu; Gonzalez-Larriba, Jose Luis; Paz-Ares, Luis; Bover, Isabel; Garcia-Campelo, Rosario; Moreno, Miguel Angel; Catot, Silvia; Rolfo, Christian; Reguart, Noemi; Palmero, Ramon; Sanchez, Jose Miguel; Bastus, Roman; Mayo, Clara; Bertran-Alamillo, Jordi; Molina, Miguel Angel; Sanchez, Jose Javier; Taron, Miquel; Cuello, M.; Pallares, C.; Lianes, P.; Remon, J.; Ibeas, R.; Martinez del Prado, P.; Angeles Sala, M.; Santander-Lobera, C.; Velez de Mendizabal, E.; Vinolas, N.; Terrasa, J.; Valdivia, J.; Diz, P.; Jimenez-Berlana, U.; Velasco-Ortiz de Taranco, A.; Ales Martinez, E.; Sanchez-Escribano, R.; Carrato, A.; Guillen-Ponce, C.; Mesia, C.; Antonio Macias, J.; Lopez-Brea, M.; Oramas, J.; Barneto, I.; Garrido, P.; Mayol, M. J.; Lopez, A.; Artal, A.; Saenz, A.; Hernando, S.; Cobo, M.; Blanco, R.; Bernabe, R.; Guillem, V.; Angel Munoz, M.; Maestu, I.; Salvatierra, A.; De Las Penas, R.; Alfaro, J.; Alberola, V.; Juan, O.; Martin, C.; Puertas, J.; Felip, E.; Gonzalez-Gonzalez, J. F.; Iglesias-Docampo, L.; Dorta, F. J.; Martinez-Aguillo, M.; Salgado, E.; Mesia, R.; Lastra-Aras, E.; Garcia-Munoz, J. P.; Lastra, R.; Alvarez, I.; Roig, J.; Oruezabal, J.; Poveda, A.; Lavernia, J.; Gutierrez, D.; Filipovich, E.; Aguiar, D.; Rodriguez, D.; Buxo, J.; Cardona, A. F.; Bes, P.; Paredes, A.; Tortorella, A. M.; Moreno, J. A.; Martinez-Garcia, J.; Alonso, J. L.; Lopez-Martin, A.; Echarri-Gonzalez, M. J.; Van Kooten, M.; Guerrero, A.; Domine, M.; Diaz, I.; Heras, L.; Garcia, R.; Anton, I.; Jarchum, G.; Bartolucci, R. E.; Lomas, M.; Rubiales, A.; Duque, J. L.; Escriva de Romani, S.; Barbeta, E.; Reina, J. J.; Castro, J.; Belda, C.; Vidal, J. M.; Trigo, J. M.; Vadell, C.; Zarba, J. J.; Esunza, P.; Garau, I.; Lopez-Pousa, A.; De la Gandara, I.; Widakowich, J.; Morales, S.; Martinez, M.; Luis, R.; de la Colina, M.; Calzas, J.; Garcia-Castro, I.; Ruiz, C.; Lopez-Criado, P.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">361</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">958-967</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Activating mutations in the epidermal growth factor receptor gene (EGFR) confer hypersensitivity to the tyrosine kinase inhibitors gefitinib and erlotinib in patients with advanced non-small-cell lung cancer.  We evaluated the feasibility of large-scale screening for EGFR mutations in such patients and analyzed the assocn. between the mutations and the outcome of erlotinib treatment.  From Apr. 2005 through Nov. 2008, lung cancers from 2105 patients in 129 institutions in Spain were screened for EGFR mutations.  The anal. was performed in a central lab.  Patients with tumors carrying EGFR mutations were eligible for erlotinib treatment.  EGFR mutations were found in 350 of 2105 patients (16.6%).  Mutations were more frequent in women (69.7%), in patients who had never smoked (66.6%), and in those with adenocarcinomas (80.9%) (P < 0.001 for all comparisons).  The mutations were deletions in exon 19 (62.2%) and L858R (37.8%).  Median progression-free survival and overall survival for 217 patients who received erlotinib were 14 mo and 27 mo, resp.  The adjusted hazard ratios for the duration of progression-free survival were 2.94 for men (P < 0.001); 1.92 for the presence of the L858R mutation, as compared with a deletion in exon 19 (P = 0.02); and 1.68 for the presence of the L858R mutation in paired serum DNA, as compared with the absence of the mutation (P = 0.02).  The most common adverse events were mild rashes and diarrhea; grade 3 cutaneous toxic effects were recorded in 16 patients (7.4%) and grade 3 diarrhea in 8 patients (3.7%).  Large-scale screening of patients with lung cancer for EGFR mutations is feasible and can have a role in decisions about treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvZWnSnU_ScbVg90H21EOLACvtfcHk0lgUv1QclKko_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVOqsL7M&md5=288ea35bd5201579d8c89ad6d7c4306a</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa0904554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa0904554%26sid%3Dliteratum%253Aachs%26aulast%3DRosell%26aufirst%3DR.%26aulast%3DMoran%26aufirst%3DT.%26aulast%3DQueralt%26aufirst%3DC.%26aulast%3DPorta%26aufirst%3DR.%26aulast%3DCardenal%26aufirst%3DF.%26aulast%3DCamps%26aufirst%3DC.%26aulast%3DMajem%26aufirst%3DM.%26aulast%3DLopez-Vivanco%26aufirst%3DG.%26aulast%3DIsla%26aufirst%3DD.%26aulast%3DProvencio%26aufirst%3DM.%26aulast%3DInsa%26aufirst%3DA.%26aulast%3DMassuti%26aufirst%3DB.%26aulast%3DGonzalez-Larriba%26aufirst%3DJ.%2BL.%26aulast%3DPaz-Ares%26aufirst%3DL.%26aulast%3DBover%26aufirst%3DI.%26aulast%3DGarcia-Campelo%26aufirst%3DR.%26aulast%3DMoreno%26aufirst%3DM.%2BA.%26aulast%3DCatot%26aufirst%3DS.%26aulast%3DRolfo%26aufirst%3DC.%26aulast%3DReguart%26aufirst%3DN.%26aulast%3DPalmero%26aufirst%3DR.%26aulast%3DSanchez%26aufirst%3DJ.%2BM.%26aulast%3DBastus%26aufirst%3DR.%26aulast%3DMayo%26aufirst%3DC.%26aulast%3DBertran-Alamillo%26aufirst%3DJ.%26aulast%3DMolina%26aufirst%3DM.%2BA.%26aulast%3DSanchez%26aufirst%3DJ.%2BJ.%26aulast%3DTaron%26aufirst%3DM.%26atitle%3DSpanish%2520Lung%2520Cancer%252C%2520G.%2520Screening%2520for%2520epidermal%2520growth%2520factor%2520receptor%2520mutations%2520in%2520lung%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2009%26volume%3D361%26spage%3D958%26epage%3D967%26doi%3D10.1056%2FNEJMoa0904554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Suzuki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobashi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakurai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishikawa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ooi, A.</span><span> </span><span class="NLM_article-title">Protein overexpression and gene amplification of epidermal growth factor receptor in non-small cell lung carcinomas. An immunohistochemical and fluorescence in situ hybridization study</span> <span class="citation_source-journal">Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">1265</span><span class="NLM_x">–</span> <span class="NLM_lpage">1273</span><span class="refDoi"> DOI: 10.1002/cncr.20909</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1002%2Fcncr.20909" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2005&pages=1265-1273&author=S.+Suzukiauthor=Y.+Dobashiauthor=H.+Sakuraiauthor=K.+Nishikawaauthor=M.+Hanawaauthor=A.+Ooi&title=Protein+overexpression+and+gene+amplification+of+epidermal+growth+factor+receptor+in+non-small+cell+lung+carcinomas.+An+immunohistochemical+and+fluorescence+in+situ+hybridization+study&doi=10.1002%2Fcncr.20909"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fcncr.20909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcncr.20909%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DS.%26aulast%3DDobashi%26aufirst%3DY.%26aulast%3DSakurai%26aufirst%3DH.%26aulast%3DNishikawa%26aufirst%3DK.%26aulast%3DHanawa%26aufirst%3DM.%26aulast%3DOoi%26aufirst%3DA.%26atitle%3DProtein%2520overexpression%2520and%2520gene%2520amplification%2520of%2520epidermal%2520growth%2520factor%2520receptor%2520in%2520non-small%2520cell%2520lung%2520carcinomas.%2520An%2520immunohistochemical%2520and%2520fluorescence%2520in%2520situ%2520hybridization%2520study%26jtitle%3DCancer%26date%3D2005%26volume%3D103%26spage%3D1265%26epage%3D1273%26doi%3D10.1002%2Fcncr.20909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Barker, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grundy, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godfrey, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barlow, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Healy, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodburn, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashton, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curry, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scarlett, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henthorn, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richards, L.</span><span> </span><span class="NLM_article-title">Studies leading to the identification of ZD1839 (IRESSA): an orally active, selective epidermal growth factor receptor tyrosine kinase inhibitor targeted to the treatment of cancer</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1911</span><span class="NLM_x">–</span> <span class="NLM_lpage">1914</span><span class="refDoi"> DOI: 10.1016/S0960-894X(01)00344-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1016%2FS0960-894X%2801%2900344-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=11459659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltFylsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=1911-1914&author=A.+J.+Barkerauthor=K.+H.+Gibsonauthor=W.+Grundyauthor=A.+A.+Godfreyauthor=J.+J.+Barlowauthor=M.+P.+Healyauthor=J.+R.+Woodburnauthor=S.+E.+Ashtonauthor=B.+J.+Curryauthor=L.+Scarlettauthor=L.+Henthornauthor=L.+Richards&title=Studies+leading+to+the+identification+of+ZD1839+%28IRESSA%29%3A+an+orally+active%2C+selective+epidermal+growth+factor+receptor+tyrosine+kinase+inhibitor+targeted+to+the+treatment+of+cancer&doi=10.1016%2FS0960-894X%2801%2900344-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Studies leading to the identification of ZD1839 (iressa): an orally active, selective epidermal growth factor receptor tyrosine kinase inhibitor targeted to the treatment of cancer</span></div><div class="casAuthors">Barker, A. J.; Gibson, K. H.; Grundy, W.; Godfrey, A. A.; Barlow, J. J.; Healy, M. P.; Woodburn, J. R.; Ashton, S. E.; Curry, B. J.; Scarlett, L.; Henthorn, L.; Richards, L.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1911-1914</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">This paper describes the development of the epidermal growth factor receptor tyrosine kinase inhibitor ZD1839 from a lead series of 4-anilinoquinazoline compds.  ZD1839 has suitable properties for use as a clin. effective drug and shows activity against human tumors.  In particular, the use of pharmacokinetic data in the development of ZD1839 is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvsJ1FTqEwvbVg90H21EOLACvtfcHk0ljf7kzq0ml3kw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltFylsLo%253D&md5=ebb210453ade246e43de9fd07ae1d581</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2801%2900344-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252801%252900344-4%26sid%3Dliteratum%253Aachs%26aulast%3DBarker%26aufirst%3DA.%2BJ.%26aulast%3DGibson%26aufirst%3DK.%2BH.%26aulast%3DGrundy%26aufirst%3DW.%26aulast%3DGodfrey%26aufirst%3DA.%2BA.%26aulast%3DBarlow%26aufirst%3DJ.%2BJ.%26aulast%3DHealy%26aufirst%3DM.%2BP.%26aulast%3DWoodburn%26aufirst%3DJ.%2BR.%26aulast%3DAshton%26aufirst%3DS.%2BE.%26aulast%3DCurry%26aufirst%3DB.%2BJ.%26aulast%3DScarlett%26aufirst%3DL.%26aulast%3DHenthorn%26aufirst%3DL.%26aulast%3DRichards%26aufirst%3DL.%26atitle%3DStudies%2520leading%2520to%2520the%2520identification%2520of%2520ZD1839%2520%2528IRESSA%2529%253A%2520an%2520orally%2520active%252C%2520selective%2520epidermal%2520growth%2520factor%2520receptor%2520tyrosine%2520kinase%2520inhibitor%2520targeted%2520to%2520the%2520treatment%2520of%2520cancer%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2001%26volume%3D11%26spage%3D1911%26epage%3D1914%26doi%3D10.1016%2FS0960-894X%2801%2900344-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Greulich, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jänne, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alvarez, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zappaterra, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bulmer, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frank, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sellers, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M.</span><span> </span><span class="NLM_article-title">Oncogenic transformation by inhibitor-sensitive and -resistant EGFR mutants</span> <span class="citation_source-journal">PLoS Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">e313</span><span class="refDoi"> DOI: 10.1371/journal.pmed.0020313</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1371%2Fjournal.pmed.0020313" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2005&pages=e313&author=H.+Greulichauthor=T.+H.+Chenauthor=W.+Fengauthor=P.+A.+J%C3%A4nneauthor=J.+V.+Alvarezauthor=M.+Zappaterraauthor=S.+E.+Bulmerauthor=D.+A.+Frankauthor=W.+C.+Hahnauthor=W.+R.+Sellersauthor=M.+Meyerson&title=Oncogenic+transformation+by+inhibitor-sensitive+and+-resistant+EGFR+mutants&doi=10.1371%2Fjournal.pmed.0020313"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pmed.0020313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pmed.0020313%26sid%3Dliteratum%253Aachs%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DT.%2BH.%26aulast%3DFeng%26aufirst%3DW.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DAlvarez%26aufirst%3DJ.%2BV.%26aulast%3DZappaterra%26aufirst%3DM.%26aulast%3DBulmer%26aufirst%3DS.%2BE.%26aulast%3DFrank%26aufirst%3DD.%2BA.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DMeyerson%26aufirst%3DM.%26atitle%3DOncogenic%2520transformation%2520by%2520inhibitor-sensitive%2520and%2520-resistant%2520EGFR%2520mutants%26jtitle%3DPLoS%2520Med.%26date%3D2005%26volume%3D2%26spage%3De313%26doi%3D10.1371%2Fjournal.pmed.0020313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Lynch, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sordella, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurubhagavatula, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okimoto, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brannigan, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haserlat, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Supko, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haluska, F. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Louis, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christiani, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Settleman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haber, D. A.</span><span> </span><span class="NLM_article-title">Activating mutations in the epidermal growth factor receptor underlying responsiveness of non-small-cell lung cancer to gefitinib</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">350</span><span class="NLM_x">, </span> <span class="NLM_fpage">2129</span><span class="NLM_x">–</span> <span class="NLM_lpage">2139</span><span class="refDoi"> DOI: 10.1056/NEJMoa040938</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1056%2FNEJMoa040938" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=15118073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2cXktF2js7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=350&publication_year=2004&pages=2129-2139&author=T.+J.+Lynchauthor=D.+W.+Bellauthor=R.+Sordellaauthor=S.+Gurubhagavatulaauthor=R.+A.+Okimotoauthor=B.+W.+Branniganauthor=P.+L.+Harrisauthor=S.+M.+Haserlatauthor=J.+G.+Supkoauthor=F.+G.+Haluskaauthor=D.+N.+Louisauthor=D.+C.+Christianiauthor=J.+Settlemanauthor=D.+A.+Haber&title=Activating+mutations+in+the+epidermal+growth+factor+receptor+underlying+responsiveness+of+non-small-cell+lung+cancer+to+gefitinib&doi=10.1056%2FNEJMoa040938"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Activating mutations in the epidermal growth factor receptor underlying responsiveness of non-small-cell lung cancer to gefitinib</span></div><div class="casAuthors">Lynch, Thomas J.; Bell, Daphne W.; Sordella, Raffaella; Gurubhagavatula, Sarada; Okimoto, Ross A.; Brannigan, Brian W.; Harris, Patricia L.; Haserlat, Sara M.; Supko, Jeffrey G.; Haluska, Frank G.; Louis, David N.; Christiani, David C.; Settleman, Jeff; Haber, Daniel A.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">350</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2129-2139</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Most patients with non-small-cell lung cancer have no response to the tyrosine kinase inhibitor gefitinib, which targets the epidermal growth factor receptor (EGFR).  However, about 10% of patients have a rapid and often dramatic clin. response.  The mol. mechanisms underlying sensitivity to gefitinib are unknown.  We searched for mutations in the EGFR gene in primary tumors from patients with non-small-cell lung cancer who had a response to gefitinib, those who did not have a response, and those who had not been exposed to gefitinib.  The functional consequences of identified mutations were evaluated after the mutant proteins were expressed in cultured cells.  Somatic mutations were identified in the tyrosine kinase domain of the EGFR gene in eight of nine patients with gefitinib-responsive lung cancer, as compared with none of the seven patients with no response (P < 0.001).  Mutations were either small, in-frame deletions or amino acid substitutions clustered around the ATP-binding pocket of the tyrosine kinase domain.  Similar mutations were detected in tumors from 2 of 25 patients with primary non-small-cell lung cancer who had not been exposed to gefitinib (8%).  All mutations were heterozygous, and identical mutations were obsd. in multiple patients, suggesting an additive specific gain of function.  In vitro, EGFR mutants demonstrated enhanced tyrosine kinase activity in response to epidermal growth factor and increased sensitivity to inhibition by gefitinib.  A subgroup of patients with non-small-cell lung cancer have specific mutations in the EGFR gene, which correlate with clin. responsiveness to the tyrosine kinase inhibitor gefitinib.  These mutations lead to increased growth factor signaling and confer susceptibility to the inhibitor.  Screening for such mutations in lung cancers may identify patients who will have a response to gefitinib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXwWGeAQrNTLVg90H21EOLACvtfcHk0lhg2SQCj2b6SA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXktF2js7c%253D&md5=fad70aa0f32ad4105ad6f3db98135bf9</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa040938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa040938%26sid%3Dliteratum%253Aachs%26aulast%3DLynch%26aufirst%3DT.%2BJ.%26aulast%3DBell%26aufirst%3DD.%2BW.%26aulast%3DSordella%26aufirst%3DR.%26aulast%3DGurubhagavatula%26aufirst%3DS.%26aulast%3DOkimoto%26aufirst%3DR.%2BA.%26aulast%3DBrannigan%26aufirst%3DB.%2BW.%26aulast%3DHarris%26aufirst%3DP.%2BL.%26aulast%3DHaserlat%26aufirst%3DS.%2BM.%26aulast%3DSupko%26aufirst%3DJ.%2BG.%26aulast%3DHaluska%26aufirst%3DF.%2BG.%26aulast%3DLouis%26aufirst%3DD.%2BN.%26aulast%3DChristiani%26aufirst%3DD.%2BC.%26aulast%3DSettleman%26aufirst%3DJ.%26aulast%3DHaber%26aufirst%3DD.%2BA.%26atitle%3DActivating%2520mutations%2520in%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520underlying%2520responsiveness%2520of%2520non-small-cell%2520lung%2520cancer%2520to%2520gefitinib%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2004%26volume%3D350%26spage%3D2129%26epage%3D2139%26doi%3D10.1056%2FNEJMoa040938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Moyer, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbacci, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwata, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnold, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boman, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cunningham, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiOrio, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doty, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morin, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neveu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollack, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pustilnik, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reynolds, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sloan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theleman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, P.</span><span> </span><span class="NLM_article-title">Induction of apoptosis and cell cycle arrest by CP-358,774, an inhibitor of epidermal growth factor receptor tyrosine kinase</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">4838</span><span class="NLM_x">–</span> <span class="NLM_lpage">4848</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=9354447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaK2sXnt12nsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1997&pages=4838-4848&author=J.+D.+Moyerauthor=E.+G.+Barbacciauthor=K.+K.+Iwataauthor=L.+Arnoldauthor=B.+Bomanauthor=A.+Cunninghamauthor=C.+DiOrioauthor=J.+Dotyauthor=M.+J.+Morinauthor=M.+P.+Moyerauthor=M.+Neveuauthor=V.+A.+Pollackauthor=L.+R.+Pustilnikauthor=M.+M.+Reynoldsauthor=D.+Sloanauthor=A.+Thelemanauthor=P.+Miller&title=Induction+of+apoptosis+and+cell+cycle+arrest+by+CP-358%2C774%2C+an+inhibitor+of+epidermal+growth+factor+receptor+tyrosine+kinase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Induction of apoptosis and cell cycle arrest by CP-358774, an inhibitor of epidermal growth factor receptor tyrosine kinase</span></div><div class="casAuthors">Moyer, James D.; Barbacci, Elsa G.; Iwata, Kenneth K.; Arnold, Lee; Boman, Bruce; Cunningham, Ann; Diorio, Catherine; Doty, Jonathan; Morin, Michael J.; Moyer, Mikel P.; Neveu, Mark; Pollack, Vincent A.; Pustilnik, Leslie R.; Reynolds, Margaret M.; Sloan, Don; Theleman, April; Miller, Penny</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">4838-4848</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR) is overexpressed in a significant percentage of carcinomas and contributes to the malignant phenotype.  CP-358774 is a directly acting inhibitor of human EGFR tyrosine kinase with an IC50 of 2 nM and reduces EGFR autophosphorylation in intact tumor cells with an IC50 of 20 nM.  This inhibition is selective for EGFR tyrosine kinase relative to other tyrosine kinases we have examd., both in assays of isolated kinases and whole cells.  At doses of 100 mg/kg, CP-358774 completely prevents EGF-induced autophosphorylation of EGFR in human HN5 tumors growing as xenografts in athymic mice and of the hepatic EGFR of the treated mice.  CP-358774 inhibits the proliferation of DiFi human colon tumor cells at submicromolar concns. in cell culture and blocks cell cycle progression at the G1 phase.  This inhibitor produces a marked accumulation of retinoblastoma protein in its underphosphorylated form and accumulation of p27KIP1 in DiFi cells, which may contribute to the cell cycle block.  Inhibition of the EGFR also triggers apoptosis in these cells as detd. by formation of DNA fragments and other criteria.  These results indicate that CP-358774 has potential for the treatment of tumors that are dependent on the EGFR pathway for proliferation or survival.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpghArQOnGlkbVg90H21EOLACvtfcHk0ljMchZ3ciJBiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnt12nsr4%253D&md5=0a6e9697bf290d8e7c44cbe2a125e0b1</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMoyer%26aufirst%3DJ.%2BD.%26aulast%3DBarbacci%26aufirst%3DE.%2BG.%26aulast%3DIwata%26aufirst%3DK.%2BK.%26aulast%3DArnold%26aufirst%3DL.%26aulast%3DBoman%26aufirst%3DB.%26aulast%3DCunningham%26aufirst%3DA.%26aulast%3DDiOrio%26aufirst%3DC.%26aulast%3DDoty%26aufirst%3DJ.%26aulast%3DMorin%26aufirst%3DM.%2BJ.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DNeveu%26aufirst%3DM.%26aulast%3DPollack%26aufirst%3DV.%2BA.%26aulast%3DPustilnik%26aufirst%3DL.%2BR.%26aulast%3DReynolds%26aufirst%3DM.%2BM.%26aulast%3DSloan%26aufirst%3DD.%26aulast%3DTheleman%26aufirst%3DA.%26aulast%3DMiller%26aufirst%3DP.%26atitle%3DInduction%2520of%2520apoptosis%2520and%2520cell%2520cycle%2520arrest%2520by%2520CP-358%252C774%252C%2520an%2520inhibitor%2520of%2520epidermal%2520growth%2520factor%2520receptor%2520tyrosine%2520kinase%26jtitle%3DCancer%2520Res.%26date%3D1997%26volume%3D57%26spage%3D4838%26epage%3D4848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Paez, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jänne, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tracy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greulich, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gabriel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaye, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindeman, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boggon, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naoki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujii, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eck, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sellers, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M.</span><span> </span><span class="NLM_article-title">EGFR mutations in lung cancer: correlation with clinical response to gefitinib therapy</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">304</span><span class="NLM_x">, </span> <span class="NLM_fpage">1497</span><span class="NLM_x">–</span> <span class="NLM_lpage">1500</span><span class="refDoi"> DOI: 10.1126/science.1099314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1126%2Fscience.1099314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=15118125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVGmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=304&publication_year=2004&pages=1497-1500&author=J.+G.+Paezauthor=P.+A.+J%C3%A4nneauthor=J.+C.+Leeauthor=S.+Tracyauthor=H.+Greulichauthor=S.+Gabrielauthor=P.+Hermanauthor=F.+J.+Kayeauthor=N.+Lindemanauthor=T.+J.+Boggonauthor=K.+Naokiauthor=H.+Sasakiauthor=Y.+Fujiiauthor=M.+J.+Eckauthor=W.+R.+Sellersauthor=B.+E.+Johnsonauthor=M.+Meyerson&title=EGFR+mutations+in+lung+cancer%3A+correlation+with+clinical+response+to+gefitinib+therapy&doi=10.1126%2Fscience.1099314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR Mutations in Lung Cancer: Correlation with Clinical Response to Gefitinib Therapy</span></div><div class="casAuthors">Paez, J. Guillermo; Jaenne, Pasi A.; Lee, Jeffrey C.; Tracy, Sean; Greulich, Heidi; Gabriel, Stacey; Herman, Paula; Kaye, Frederic J.; Lindeman, Neal; Boggon, Titus J.; Naoki, Katsuhiko; Sasaki, Hidefumi; Fujii, Yoshitaka; Eck, Michael J.; Sellers, William R.; Johnson, Bruce E.; Meyerson, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">304</span>
        (<span class="NLM_cas:issue">5676</span>),
    <span class="NLM_cas:pages">1497-1500</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Receptor tyrosine kinase genes were sequenced in non-small cell lung cancer (NSCLC) and matched normal tissue.  Somatic mutations of the epidermal growth factor receptor gene EGFR were found in 15 of 58 unselected tumors from Japan and 1 of 61 from the United States.  Treatment with the EGFR kinase inhibitor gefitinib (Iressa) causes tumor regression in some patients with NSCLC, more frequently in Japan.  EGFR mutations were found in addnl. lung cancer samples from U.S. patients who responded to gefitinib therapy and in a lung adenocarcinoma cell line that was hypersensitive to growth inhibition by gefitinib, but not in gefitinib-insensitive tumors or cell lines.  These results suggest that EGFR mutations may predict sensitivity to gefitinib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGxLJH5f3Kb7Vg90H21EOLACvtfcHk0ljMchZ3ciJBiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVGmsbs%253D&md5=6df1b11bf9b3c284fbe000e34c61ebac</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.1099314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1099314%26sid%3Dliteratum%253Aachs%26aulast%3DPaez%26aufirst%3DJ.%2BG.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26aulast%3DTracy%26aufirst%3DS.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DGabriel%26aufirst%3DS.%26aulast%3DHerman%26aufirst%3DP.%26aulast%3DKaye%26aufirst%3DF.%2BJ.%26aulast%3DLindeman%26aufirst%3DN.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26aulast%3DNaoki%26aufirst%3DK.%26aulast%3DSasaki%26aufirst%3DH.%26aulast%3DFujii%26aufirst%3DY.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DJohnson%26aufirst%3DB.%2BE.%26aulast%3DMeyerson%26aufirst%3DM.%26atitle%3DEGFR%2520mutations%2520in%2520lung%2520cancer%253A%2520correlation%2520with%2520clinical%2520response%2520to%2520gefitinib%2520therapy%26jtitle%3DScience%26date%3D2004%26volume%3D304%26spage%3D1497%26epage%3D1500%26doi%3D10.1126%2Fscience.1099314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Pao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zakowski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doherty, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Politi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarkaria, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heelan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fulton, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mardis, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kupfer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kris, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varmus, H.</span><span> </span><span class="NLM_article-title">EGF receptor gene mutations are common in lung cancers from ″never smokers″ and are associated with sensitivity of tumors to gefitinib and erlotinib</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">13306</span><span class="NLM_x">–</span> <span class="NLM_lpage">13311</span><span class="refDoi"> DOI: 10.1073/pnas.0405220101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1073%2Fpnas.0405220101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=15329413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnvFems70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=13306-13311&author=W.+Paoauthor=V.+Millerauthor=M.+Zakowskiauthor=J.+Dohertyauthor=K.+Politiauthor=I.+Sarkariaauthor=B.+Singhauthor=R.+Heelanauthor=V.+Ruschauthor=L.+Fultonauthor=E.+Mardisauthor=D.+Kupferauthor=R.+Wilsonauthor=M.+Krisauthor=H.+Varmus&title=EGF+receptor+gene+mutations+are+common+in+lung+cancers+from+%E2%80%B3never+smokers%E2%80%B3+and+are+associated+with+sensitivity+of+tumors+to+gefitinib+and+erlotinib&doi=10.1073%2Fpnas.0405220101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">EGF receptor gene mutations are common in lung cancers from "never smokers" and are associated with sensitivity of tumors to gefitinib and erlotinib</span></div><div class="casAuthors">Pao, William; Miller, Vincent; Zakowski, Maureen; Doherty, Jennifer; Politi, Katerina; Sarkaria, Inderpal; Singh, Bhuvanesh; Heelan, Robert; Rusch, Valerie; Fulton, Lucinda; Mardis, Elaine; Kupfer, Doris; Wilson, Richard; Kris, Mark; Varmus, Harold</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">13306-13311</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Somatic mutations in the tyrosine kinase (TK) domain of the epidermal growth factor receptor (EGFR) gene are reportedly assocd. with sensitivity of lung cancers to gefitinib (Iressa), kinase inhibitor.  In-frame deletions occur in exon 19, whereas point mutations occur frequently in codon 858 (exon 21).  We found from sequencing the EGFR TK domain that 7 of 10 gefitinib-sensitive tumors had similar types of alterations; no mutations were found in eight gefitinib-refractory tumors (P = 0.004).  Five of seven tumors sensitive to erlotinib (Tarceva), a related kinase inhibitor for which the clin. relevant target is undocumented, had analogous somatic mutations, as opposed to none of 10 erlotinib-refractory tumors (P = 0.003).  Because most mutation-pos. tumors were adenocarcinomas from patients who smoked <100 cigarettes in a lifetime ("never smokers"), we screened EGFR exons 2-28 in 15 adenocarcinomas resected from untreated never smokers.  Seven tumors had TK domain mutations, in contrast to 4 of 81 non-small cell lung cancers resected from untreated former or current smokers (P = 0.0001).  Immunoblotting of lysates from cells transiently transfected with various EGFR constructs demonstrated that, compared to wild-type protein, an exon 19 deletion mutant induced diminished levels of phosphotyrosine, whereas the phosphorylation at tyrosine 1092 of an exon 21 point mutant was inhibited at 10-fold lower concns. of drug.  Collectively, these data show that adenocarcinomas from never smokers comprise a distinct subset of lung cancers, frequently contg. mutations within the TK domain of EGFR that are assocd. with gefitinib and erlotinib sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovevWonTGqELVg90H21EOLACvtfcHk0lh656cxuz5r_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnvFems70%253D&md5=38f491968552501b7463a688fa0e62ff</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0405220101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0405220101%26sid%3Dliteratum%253Aachs%26aulast%3DPao%26aufirst%3DW.%26aulast%3DMiller%26aufirst%3DV.%26aulast%3DZakowski%26aufirst%3DM.%26aulast%3DDoherty%26aufirst%3DJ.%26aulast%3DPoliti%26aufirst%3DK.%26aulast%3DSarkaria%26aufirst%3DI.%26aulast%3DSingh%26aufirst%3DB.%26aulast%3DHeelan%26aufirst%3DR.%26aulast%3DRusch%26aufirst%3DV.%26aulast%3DFulton%26aufirst%3DL.%26aulast%3DMardis%26aufirst%3DE.%26aulast%3DKupfer%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DR.%26aulast%3DKris%26aufirst%3DM.%26aulast%3DVarmus%26aufirst%3DH.%26atitle%3DEGF%2520receptor%2520gene%2520mutations%2520are%2520common%2520in%2520lung%2520cancers%2520from%2520%25E2%2580%25B3never%2520smokers%25E2%2580%25B3%2520and%2520are%2520associated%2520with%2520sensitivity%2520of%2520tumors%2520to%2520gefitinib%2520and%2520erlotinib%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26spage%3D13306%26epage%3D13311%26doi%3D10.1073%2Fpnas.0405220101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Sordella, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haber, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Settleman, J.</span><span> </span><span class="NLM_article-title">Gefitinib-sensitizing EGFR mutations in lung cancer activate anti-apoptotic pathways</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">305</span><span class="NLM_x">, </span> <span class="NLM_fpage">1163</span><span class="NLM_x">–</span> <span class="NLM_lpage">1167</span><span class="refDoi"> DOI: 10.1126/science.1101637</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1126%2Fscience.1101637" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=15284455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmslCmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=305&publication_year=2004&pages=1163-1167&author=R.+Sordellaauthor=D.+W.+Bellauthor=D.+A.+Haberauthor=J.+Settleman&title=Gefitinib-sensitizing+EGFR+mutations+in+lung+cancer+activate+anti-apoptotic+pathways&doi=10.1126%2Fscience.1101637"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Gefitinib-Sensitizing EGFR Mutations in Lung Cancer Activate Anti-Apoptotic Pathways</span></div><div class="casAuthors">Sordella, Raffaella; Bell, Daphne W.; Haber, Daniel A.; Settleman, Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">305</span>
        (<span class="NLM_cas:issue">5687</span>),
    <span class="NLM_cas:pages">1163-1167</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Gefitinib (Iressa, AstraZeneca Pharmaceuticals) is a tyrosine kinase inhibitor that targets the epidermal growth factor receptor (EGFR) and induces dramatic clin. responses in nonsmall cell lung cancers (NSCLCs) with activating mutations within the EGFR kinase domain.  The authors report that these mutant EGFRs selectively activate Akt and signal transduction and activator of transcription (STAT) signaling pathways, which promote cell survival, but have no effect on extracellular signal-regulated kinase signaling, which induces proliferation.  NSCLC cells expressing mutant EGFRs underwent extensive apoptosis after small interfering RNA-mediated knockdown of the mutant EGFR or treatment with pharmacol. inhibitors of Akt and STAT signaling and were relatively resistant to apoptosis induced by conventional chemotherapeutic drugs.  Thus, mutant EGFRs selectively transduce survival signals on which NSCLCs become dependent; inhibition of those signals by gefitinib may contribute to the drug's efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrKHDqaSobm7Vg90H21EOLACvtfcHk0lh656cxuz5r_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmslCmur8%253D&md5=e9742920d5e320357b774495746e6581</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1126%2Fscience.1101637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1101637%26sid%3Dliteratum%253Aachs%26aulast%3DSordella%26aufirst%3DR.%26aulast%3DBell%26aufirst%3DD.%2BW.%26aulast%3DHaber%26aufirst%3DD.%2BA.%26aulast%3DSettleman%26aufirst%3DJ.%26atitle%3DGefitinib-sensitizing%2520EGFR%2520mutations%2520in%2520lung%2520cancer%2520activate%2520anti-apoptotic%2520pathways%26jtitle%3DScience%26date%3D2004%26volume%3D305%26spage%3D1163%26epage%3D1167%26doi%3D10.1126%2Fscience.1101637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Kobayashi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boggon, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dayaram, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jänne, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kocher, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eck, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tenen, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halmos, B.</span><span> </span><span class="NLM_article-title">EGFR mutation and resistance of non-small-cell lung cancer to gefitinib</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">352</span><span class="NLM_x">, </span> <span class="NLM_fpage">786</span><span class="NLM_x">–</span> <span class="NLM_lpage">792</span><span class="refDoi"> DOI: 10.1056/NEJMoa044238</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1056%2FNEJMoa044238" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=15728811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhsFCiurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2005&pages=786-792&author=S.+Kobayashiauthor=T.+J.+Boggonauthor=T.+Dayaramauthor=P.+A.+J%C3%A4nneauthor=O.+Kocherauthor=M.+Meyersonauthor=B.+E.+Johnsonauthor=M.+J.+Eckauthor=D.+G.+Tenenauthor=B.+Halmos&title=EGFR+mutation+and+resistance+of+non-small-cell+lung+cancer+to+gefitinib&doi=10.1056%2FNEJMoa044238"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR mutation and resistance of non-small-cell lung cancer to gefitinib</span></div><div class="casAuthors">Kobayashi, Susumu; Boggon, Titus J.; Dayaram, Tajhal; Jaenne, Pasi A.; Kocher, Olivier; Meyerson, Matthew; Johnson, Bruce E.; Eck, Michael J.; Tenen, Daniel G.; Halmos, Balizs</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">352</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">786-792</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Mutations of the epidermal growth factor receptor (EGFR) gene were identified in specimens from patients with non-small-cell lung cancer who have a response to anilinoquinazoline EGFR inhibitors.  Despite the dramatic responses to such inhibitors, most patients ultimately have a relapse.  The mechanism of the drug resistance is unknown.  Here the authors report the case of a patient with EGFR-mutant, gefitinib-responsive, advanced non-small-cell lung cancer who had a relapse after 2 years of complete remission during treatment with gefitinib.  The DNA sequence of the EGFR gene in his tumor biopsy specimen at relapse revealed the presence of a second point mutation, resulting in threonine-to-methionine amino acid change at position 790 of EGFR.  Structural modeling and biochem. studies showed that this second mutation led to gefitinib resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8XXhT1hrbe7Vg90H21EOLACvtfcHk0li0Ync-yINRgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhsFCiurk%253D&md5=808223ad566e2a3748977727cc5a9846</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa044238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa044238%26sid%3Dliteratum%253Aachs%26aulast%3DKobayashi%26aufirst%3DS.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26aulast%3DDayaram%26aufirst%3DT.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DKocher%26aufirst%3DO.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DB.%2BE.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DTenen%26aufirst%3DD.%2BG.%26aulast%3DHalmos%26aufirst%3DB.%26atitle%3DEGFR%2520mutation%2520and%2520resistance%2520of%2520non-small-cell%2520lung%2520cancer%2520to%2520gefitinib%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2005%26volume%3D352%26spage%3D786%26epage%3D792%26doi%3D10.1056%2FNEJMoa044238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Pao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Politi, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riely, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somwar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zakowski, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kris, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varmus, H.</span><span> </span><span class="NLM_article-title">Acquired resistance of lung adenocarcinomas to gefitinib or erlotinib is associated with a second mutation in the EGFR kinase domain</span> <span class="citation_source-journal">PLoS Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">e73</span><span class="refDoi"> DOI: 10.1371/journal.pmed.0020073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1371%2Fjournal.pmed.0020073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=15737014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A280%3ADC%252BD2MzjtVehsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2005&pages=e73&author=W.+Paoauthor=V.+A.+Millerauthor=K.+A.+Politiauthor=G.+J.+Rielyauthor=R.+Somwarauthor=M.+F.+Zakowskiauthor=M.+G.+Krisauthor=H.+Varmus&title=Acquired+resistance+of+lung+adenocarcinomas+to+gefitinib+or+erlotinib+is+associated+with+a+second+mutation+in+the+EGFR+kinase+domain&doi=10.1371%2Fjournal.pmed.0020073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Acquired resistance of lung adenocarcinomas to gefitinib or erlotinib is associated with a second mutation in the EGFR kinase domain</span></div><div class="casAuthors">Pao William; Miller Vincent A; Politi Katerina A; Riely Gregory J; Somwar Romel; Zakowski Maureen F; Kris Mark G; Varmus Harold</div><div class="citationInfo"><span class="NLM_cas:title">PLoS medicine</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e73</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Lung adenocarcinomas from patients who respond to the tyrosine kinase inhibitors gefitinib (Iressa) or erlotinib (Tarceva) usually harbor somatic gain-of-function mutations in exons encoding the kinase domain of the epidermal growth factor receptor (EGFR).  Despite initial responses, patients eventually progress by unknown mechanisms of "acquired" resistance.  METHODS AND FINDINGS:  We show that in two of five patients with acquired resistance to gefitinib or erlotinib, progressing tumors contain, in addition to a primary drug-sensitive mutation in EGFR, a secondary mutation in exon 20, which leads to substitution of methionine for threonine at position 790 (T790M) in the kinase domain.  Tumor cells from a sixth patient with a drug-sensitive EGFR mutation whose tumor progressed on adjuvant gefitinib after complete resection also contained the T790M mutation.  This mutation was not detected in untreated tumor samples.  Moreover, no tumors with acquired resistance had KRAS mutations, which have been associated with primary resistance to these drugs.  Biochemical analyses of transfected cells and growth inhibition studies with lung cancer cell lines demonstrate that the T790M mutation confers resistance to EGFR mutants usually sensitive to either gefitinib or erlotinib.  Interestingly, a mutation analogous to T790M has been observed in other kinases with acquired resistance to another kinase inhibitor, imatinib (Gleevec).  CONCLUSION:  In patients with tumors bearing gefitinib- or erlotinib-sensitive EGFR mutations, resistant subclones containing an additional EGFR mutation emerge in the presence of drug.  This observation should help guide the search for more effective therapy against a specific subset of lung cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcREkXMV3u1nwwMthz5_Rx5rfW6udTcc2eZtVbT21_Jy27ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2MzjtVehsw%253D%253D&md5=3e626105e0dea2a38997156ae513f56b</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pmed.0020073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pmed.0020073%26sid%3Dliteratum%253Aachs%26aulast%3DPao%26aufirst%3DW.%26aulast%3DMiller%26aufirst%3DV.%2BA.%26aulast%3DPoliti%26aufirst%3DK.%2BA.%26aulast%3DRiely%26aufirst%3DG.%2BJ.%26aulast%3DSomwar%26aufirst%3DR.%26aulast%3DZakowski%26aufirst%3DM.%2BF.%26aulast%3DKris%26aufirst%3DM.%2BG.%26aulast%3DVarmus%26aufirst%3DH.%26atitle%3DAcquired%2520resistance%2520of%2520lung%2520adenocarcinomas%2520to%2520gefitinib%2520or%2520erlotinib%2520is%2520associated%2520with%2520a%2520second%2520mutation%2520in%2520the%2520EGFR%2520kinase%2520domain%26jtitle%3DPLoS%2520Med.%26date%3D2005%26volume%3D2%26spage%3De73%26doi%3D10.1371%2Fjournal.pmed.0020073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Sequist, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waltman, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dias-Santagata, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Digumarthy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turke, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fidias, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergethon, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gettinger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosper, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akhavanfard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heist, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Temel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wain, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vernovsky, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mark, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lanuti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iafrate, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mino-Kenudson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelman, J. A.</span><span> </span><span class="NLM_article-title">Genotypic and histological evolution of lung cancers acquiring resistance to EGFR inhibitors</span> <span class="citation_source-journal">Sci. Transl. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">75ra26</span><span class="refDoi"> DOI: 10.1126/scitranslmed.3002003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1126%2Fscitranslmed.3002003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=21430269" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=75ra26&author=L.+V.+Sequistauthor=B.+A.+Waltmanauthor=D.+Dias-Santagataauthor=S.+Digumarthyauthor=A.+B.+Turkeauthor=P.+Fidiasauthor=K.+Bergethonauthor=A.+T.+Shawauthor=S.+Gettingerauthor=A.+K.+Cosperauthor=S.+Akhavanfardauthor=R.+S.+Heistauthor=J.+Temelauthor=J.+G.+Christensenauthor=J.+C.+Wainauthor=T.+J.+Lynchauthor=K.+Vernovskyauthor=E.+J.+Markauthor=M.+Lanutiauthor=A.+J.+Iafrateauthor=M.+Mino-Kenudsonauthor=J.+A.+Engelman&title=Genotypic+and+histological+evolution+of+lung+cancers+acquiring+resistance+to+EGFR+inhibitors&doi=10.1126%2Fscitranslmed.3002003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.3002003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.3002003%26sid%3Dliteratum%253Aachs%26aulast%3DSequist%26aufirst%3DL.%2BV.%26aulast%3DWaltman%26aufirst%3DB.%2BA.%26aulast%3DDias-Santagata%26aufirst%3DD.%26aulast%3DDigumarthy%26aufirst%3DS.%26aulast%3DTurke%26aufirst%3DA.%2BB.%26aulast%3DFidias%26aufirst%3DP.%26aulast%3DBergethon%26aufirst%3DK.%26aulast%3DShaw%26aufirst%3DA.%2BT.%26aulast%3DGettinger%26aufirst%3DS.%26aulast%3DCosper%26aufirst%3DA.%2BK.%26aulast%3DAkhavanfard%26aufirst%3DS.%26aulast%3DHeist%26aufirst%3DR.%2BS.%26aulast%3DTemel%26aufirst%3DJ.%26aulast%3DChristensen%26aufirst%3DJ.%2BG.%26aulast%3DWain%26aufirst%3DJ.%2BC.%26aulast%3DLynch%26aufirst%3DT.%2BJ.%26aulast%3DVernovsky%26aufirst%3DK.%26aulast%3DMark%26aufirst%3DE.%2BJ.%26aulast%3DLanuti%26aufirst%3DM.%26aulast%3DIafrate%26aufirst%3DA.%2BJ.%26aulast%3DMino-Kenudson%26aufirst%3DM.%26aulast%3DEngelman%26aufirst%3DJ.%2BA.%26atitle%3DGenotypic%2520and%2520histological%2520evolution%2520of%2520lung%2520cancers%2520acquiring%2520resistance%2520to%2520EGFR%2520inhibitors%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2011%26volume%3D3%26spage%3D75ra26%26doi%3D10.1126%2Fscitranslmed.3002003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Yun, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mengwasser, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toms, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woo, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greulich, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eck, M. J.</span><span> </span><span class="NLM_article-title">The T790M mutation in EGFR kinase causes drug resistance by increasing the affinity for ATP</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">2070</span><span class="NLM_x">–</span> <span class="NLM_lpage">2075</span><span class="refDoi"> DOI: 10.1073/pnas.0709662105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1073%2Fpnas.0709662105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=18227510" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitl2mu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=2070-2075&author=C.+H.+Yunauthor=K.+E.+Mengwasserauthor=A.+V.+Tomsauthor=M.+S.+Wooauthor=H.+Greulichauthor=K.+K.+Wongauthor=M.+Meyersonauthor=M.+J.+Eck&title=The+T790M+mutation+in+EGFR+kinase+causes+drug+resistance+by+increasing+the+affinity+for+ATP&doi=10.1073%2Fpnas.0709662105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The T790M mutation in EGFR kinase causes drug resistance by increasing the affinity for ATP</span></div><div class="casAuthors">Yun, Cai-Hong; Mengwasser, Kristen E.; Toms, Angela V.; Woo, Michele S.; Greulich, Heidi; Wong, Kwok-Kin; Meyerson, Matthew; Eck, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2070-2075</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Lung cancers caused by activating mutations in the epidermal growth factor receptor (EGFR) are initially responsive to small mol. tyrosine kinase inhibitors (TKIs), but the efficacy of these agents is often limited because of the emergence of drug resistance conferred by a second mutation, T790M.  Threonine 790 is the "gatekeeper" residue, an important determinant of inhibitor specificity in the ATP binding pocket.  The T790M mutation has been thought to cause resistance by sterically blocking binding of TKIs such as gefitinib and erlotinib, but this explanation is difficult to reconcile with the fact that it remains sensitive to structurally similar irreversible inhibitors.  Here, we show by using a direct binding assay that T790M mutants retain low-nanomolar affinity for gefitinib.  Furthermore, we show that the T790M mutation activates WT EGFR and that introduction of the T790M mutation increases the ATP affinity of the oncogenic L858R mutant by more than an order of magnitude.  The increased ATP affinity is the primary mechanism by which the T790M mutation confers drug resistance.  Crystallog. anal. of the T790M mutant shows how it can adapt to accommodate tight binding of diverse inhibitors, including the irreversible inhibitor HKI-272, and also suggests a structural mechanism for catalytic activation.  We conclude that the T790M mutation is a "generic" resistance mutation that will reduce the potency of any ATP-competitive kinase inhibitor and that irreversible inhibitors overcome this resistance simply through covalent binding, not as a result of an alternative binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXXP7TBlyrdrVg90H21EOLACvtfcHk0lgSVA9gmSxf5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitl2mu7k%253D&md5=f5f5d791a65f73ca628d95e9922b8ea8</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0709662105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0709662105%26sid%3Dliteratum%253Aachs%26aulast%3DYun%26aufirst%3DC.%2BH.%26aulast%3DMengwasser%26aufirst%3DK.%2BE.%26aulast%3DToms%26aufirst%3DA.%2BV.%26aulast%3DWoo%26aufirst%3DM.%2BS.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26atitle%3DThe%2520T790M%2520mutation%2520in%2520EGFR%2520kinase%2520causes%2520drug%2520resistance%2520by%2520increasing%2520the%2520affinity%2520for%2520ATP%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D2070%26epage%3D2075%26doi%3D10.1073%2Fpnas.0709662105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Klüter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simard, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rode, H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grütter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pawar, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raaijmakers, H. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barf, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabiller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Otterlo, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rauh, D.</span><span> </span><span class="NLM_article-title">Characterization of irreversible kinase inhibitors by directly detecting covalent bond formation: a tool for dissecting kinase drug resistance</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">2557</span><span class="NLM_x">–</span> <span class="NLM_lpage">2566</span><span class="refDoi"> DOI: 10.1002/cbic.201000352</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1002%2Fcbic.201000352" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=21080395" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFahsLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=2557-2566&author=S.+Kl%C3%BCterauthor=J.+R.+Simardauthor=H.+B.+Rodeauthor=C.+Gr%C3%BCtterauthor=V.+Pawarauthor=H.+C.+Raaijmakersauthor=T.+A.+Barfauthor=M.+Rabillerauthor=W.+A.+van+Otterloauthor=D.+Rauh&title=Characterization+of+irreversible+kinase+inhibitors+by+directly+detecting+covalent+bond+formation%3A+a+tool+for+dissecting+kinase+drug+resistance&doi=10.1002%2Fcbic.201000352"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of Irreversible Kinase Inhibitors by Directly Detecting Covalent Bond Formation: A Tool for Dissecting Kinase Drug Resistance</span></div><div class="casAuthors">Klueter, Sabine; Simard, Jeffrey R.; Rode, Haridas B.; Gruetter, Christian; Pawar, Vijaykumar; Raaijmakers, Hans C. A.; Barf, Tjeerd A.; Rabiller, Matthias; van Otterlo, Willem A. L.; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2557-2566</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Targeting protein kinases in cancer therapy with irreversible small-mol. inhibitors is moving to the forefront of kinase-inhibitor research and is thought to be an effective means of overcoming mutation-assocd. drug resistance in epidermal growth factor receptor kinase (EGFR).  We generated a detection technique that allows direct measurements of covalent bond formation without relying on kinase activity, thereby allowing the straightforward investigation of the influence of steric clashes on covalent inhibitors in different resistant kinase mutants.  The obtained results are discussed together with structural biol. and biochem. studies of catalytic activity in both wild-type and gatekeeper mutated kinase variants to draw conclusions about the impact of steric hindrance and increased catalytic activity in drug-resistant kinase variants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXBFAgMOxkBLVg90H21EOLACvtfcHk0lgSVA9gmSxf5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFahsLrE&md5=5f48cde54e6bad28b4d4f9d47a4e3e3e</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201000352&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201000352%26sid%3Dliteratum%253Aachs%26aulast%3DKl%25C3%25BCter%26aufirst%3DS.%26aulast%3DSimard%26aufirst%3DJ.%2BR.%26aulast%3DRode%26aufirst%3DH.%2BB.%26aulast%3DGr%25C3%25BCtter%26aufirst%3DC.%26aulast%3DPawar%26aufirst%3DV.%26aulast%3DRaaijmakers%26aufirst%3DH.%2BC.%26aulast%3DBarf%26aufirst%3DT.%2BA.%26aulast%3DRabiller%26aufirst%3DM.%26aulast%3Dvan%2BOtterlo%26aufirst%3DW.%2BA.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DCharacterization%2520of%2520irreversible%2520kinase%2520inhibitors%2520by%2520directly%2520detecting%2520covalent%2520bond%2520formation%253A%2520a%2520tool%2520for%2520dissecting%2520kinase%2520drug%2520resistance%26jtitle%3DChemBioChem%26date%3D2010%26volume%3D11%26spage%3D2557%26epage%3D2566%26doi%3D10.1002%2Fcbic.201000352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Michalczyk, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kluter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rode, H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simard, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grutter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabiller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rauh, D.</span><span> </span><span class="NLM_article-title">Structural insights into how irreversible inhibitors can overcome drug resistance in EGFR</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">3482</span><span class="NLM_x">–</span> <span class="NLM_lpage">3488</span><span class="refDoi"> DOI: 10.1016/j.bmc.2008.02.053</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1016%2Fj.bmc.2008.02.053" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=18316192" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1cXks1Gruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=3482-3488&author=A.+Michalczykauthor=S.+Kluterauthor=H.+B.+Rodeauthor=J.+R.+Simardauthor=C.+Grutterauthor=M.+Rabillerauthor=D.+Rauh&title=Structural+insights+into+how+irreversible+inhibitors+can+overcome+drug+resistance+in+EGFR&doi=10.1016%2Fj.bmc.2008.02.053"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into how irreversible inhibitors can overcome drug resistance in EGFR</span></div><div class="casAuthors">Michalczyk, Anja; Klueter, Sabine; Rode, Haridas B.; Simard, Jeffrey R.; Gruetter, Christian; Rabiller, Matthias; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3482-3488</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Resistance to kinase-targeted cancer drugs has recently been linked to a single point mutation in the ATP binding site of the kinase.  In EGFR, the crucial Thr790 gatekeeper residue is mutated to a Met and prevents reversible ATP competitive inhibitors from binding.  Irreversible 4-(phenylamino)quinazolines have been shown to overcome this drug resistance and are currently in clin. trials.  In order to obtain a detailed structural understanding of how irreversible inhibitors overcome drug resistance, the authors used Src kinase as a model system for drug resistant EGFR-T790M.  The authors report the first crystal structure of a drug resistant kinase in complex with an irreversible inhibitor.  This 4-(phenylamino)quinazoline inhibits wild type and drug resistant EGFR in vitro at low nM concns.  The cocrystal structure of drug resistant cSrc-T338M kinase domain provides the structural basis of this activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrp52JtORqsX7Vg90H21EOLACvtfcHk0lgSVA9gmSxf5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXks1Gruro%253D&md5=b9fc704067e495279bad4913e9c703ab</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2008.02.053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2008.02.053%26sid%3Dliteratum%253Aachs%26aulast%3DMichalczyk%26aufirst%3DA.%26aulast%3DKluter%26aufirst%3DS.%26aulast%3DRode%26aufirst%3DH.%2BB.%26aulast%3DSimard%26aufirst%3DJ.%2BR.%26aulast%3DGrutter%26aufirst%3DC.%26aulast%3DRabiller%26aufirst%3DM.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DStructural%2520insights%2520into%2520how%2520irreversible%2520inhibitors%2520can%2520overcome%2520drug%2520resistance%2520in%2520EGFR%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2008%26volume%3D16%26spage%3D3482%26epage%3D3488%26doi%3D10.1016%2Fj.bmc.2008.02.053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Sos, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rode, H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heynck, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peifer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klüter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pawar, V. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reuter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heuckmann, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruddigkeit, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabiller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simard, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getlik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuza, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greulich, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rauh, D.</span><span> </span><span class="NLM_article-title">Chemogenomic profiling provides insights into the limited activity of irreversible EGFR Inhibitors in tumor cells expressing the T790M EGFR resistance mutation</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x">, </span> <span class="NLM_fpage">868</span><span class="NLM_x">–</span> <span class="NLM_lpage">874</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-09-3106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1158%2F0008-5472.CAN-09-3106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=20103621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12ru7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2010&pages=868-874&author=M.+L.+Sosauthor=H.+B.+Rodeauthor=S.+Heynckauthor=M.+Peiferauthor=F.+Fischerauthor=S.+Kl%C3%BCterauthor=V.+G.+Pawarauthor=C.+Reuterauthor=J.+M.+Heuckmannauthor=J.+Weissauthor=L.+Ruddigkeitauthor=M.+Rabillerauthor=M.+Kokerauthor=J.+R.+Simardauthor=M.+Getlikauthor=Y.+Yuzaauthor=T.+H.+Chenauthor=H.+Greulichauthor=R.+K.+Thomasauthor=D.+Rauh&title=Chemogenomic+profiling+provides+insights+into+the+limited+activity+of+irreversible+EGFR+Inhibitors+in+tumor+cells+expressing+the+T790M+EGFR+resistance+mutation&doi=10.1158%2F0008-5472.CAN-09-3106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomic Profiling Provides Insights into the Limited Activity of Irreversible EGFR Inhibitors in Tumor Cells Expressing the T790M EGFR Resistance Mutation</span></div><div class="casAuthors">Sos, Martin L.; Rode, Haridas B.; Heynck, Stefanie; Peifer, Martin; Fischer, Florian; Klueter, Sabine; Pawar, Vijaykumar G.; Reuter, Cecile; Heuckmann, Johannes M.; Weiss, Jonathan; Ruddigkeit, Lars; Rabiller, Matthias; Koker, Mirjam; Simard, Jeffrey R.; Getlik, Matthaeus; Yuza, Yuki; Chen, Tzu-Hsiu; Greulich, Heidi; Thomas, Roman K.; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">868-874</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Reversible epidermal growth factor receptor (EGFR) inhibitors are the first class of small mols. to improve progression-free survival of patients with EGFR-mutated lung cancers.  Second-generation EGFR inhibitors introduced to overcome acquired resistance by the T790M resistance mutation of EGFR have thus far shown limited clin. activity in patients with T790M-mutant tumors.  In this study, we systematically analyzed the determinants of the activity and selectivity of the second-generation EGFR inhibitors.  A focused library of irreversible as well as structurally corresponding reversible EGFR-inhibitors was synthesized for chemogenomic profiling involving over 79 genetically defined NSCLC and 19 EGFR-dependent cell lines.  Overall, our results show that the growth-inhibitory potency of all irreversible inhibitors against the EGFRT790M resistance mutation was limited by reduced target inhibition, linked to decreased binding velocity to the mutant kinase.  Combined treatment of T790M-mutant tumor cells with BIBW-2992 and the phosphoinositide-3-kinase/mammalian target of rapamycin inhibitor PI-103 led to synergistic induction of apoptosis.  Our findings offer a mechanistic explanation for the limited efficacy of irreversible EGFR inhibitors in EGFRT790M gatekeeper-mutant tumors, and they prompt combination treatment strategies involving inhibitors that target signaling downstream of the EGFR.  Cancer Res; 70(3); 868-74.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNjzYq32JnJ7Vg90H21EOLACvtfcHk0ljb-Jm_pEdsNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12ru7Y%253D&md5=db40fb75245450e8f79fca7e37bad588</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-09-3106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-09-3106%26sid%3Dliteratum%253Aachs%26aulast%3DSos%26aufirst%3DM.%2BL.%26aulast%3DRode%26aufirst%3DH.%2BB.%26aulast%3DHeynck%26aufirst%3DS.%26aulast%3DPeifer%26aufirst%3DM.%26aulast%3DFischer%26aufirst%3DF.%26aulast%3DKl%25C3%25BCter%26aufirst%3DS.%26aulast%3DPawar%26aufirst%3DV.%2BG.%26aulast%3DReuter%26aufirst%3DC.%26aulast%3DHeuckmann%26aufirst%3DJ.%2BM.%26aulast%3DWeiss%26aufirst%3DJ.%26aulast%3DRuddigkeit%26aufirst%3DL.%26aulast%3DRabiller%26aufirst%3DM.%26aulast%3DKoker%26aufirst%3DM.%26aulast%3DSimard%26aufirst%3DJ.%2BR.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DYuza%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DT.%2BH.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DThomas%26aufirst%3DR.%2BK.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DChemogenomic%2520profiling%2520provides%2520insights%2520into%2520the%2520limited%2520activity%2520of%2520irreversible%2520EGFR%2520Inhibitors%2520in%2520tumor%2520cells%2520expressing%2520the%2520T790M%2520EGFR%2520resistance%2520mutation%26jtitle%3DCancer%2520Res.%26date%3D2010%26volume%3D70%26spage%3D868%26epage%3D874%26doi%3D10.1158%2F0008-5472.CAN-09-3106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Kwak, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sordella, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godin-Heymann, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okimoto, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brannigan, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Driscoll, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fidias, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabindran, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGinnis, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wissner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isselbacher, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Settleman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haber, D. A.</span><span> </span><span class="NLM_article-title">Irreversible inhibitors of the EGF receptor may circumvent acquired resistance to gefitinib</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">7665</span><span class="NLM_x">–</span> <span class="NLM_lpage">7670</span><span class="refDoi"> DOI: 10.1073/pnas.0502860102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1073%2Fpnas.0502860102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=15897464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkslOmsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=7665-7670&author=E.+L.+Kwakauthor=R.+Sordellaauthor=D.+W.+Bellauthor=N.+Godin-Heymannauthor=R.+A.+Okimotoauthor=B.+W.+Branniganauthor=P.+L.+Harrisauthor=D.+R.+Driscollauthor=P.+Fidiasauthor=T.+J.+Lynchauthor=S.+K.+Rabindranauthor=J.+P.+McGinnisauthor=A.+Wissnerauthor=S.+V.+Sharmaauthor=K.+J.+Isselbacherauthor=J.+Settlemanauthor=D.+A.+Haber&title=Irreversible+inhibitors+of+the+EGF+receptor+may+circumvent+acquired+resistance+to+gefitinib&doi=10.1073%2Fpnas.0502860102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Irreversible inhibitors of the EGF receptor may circumvent acquired resistance to gefitinib</span></div><div class="casAuthors">Kwak, Eunice L.; Sordella, Raffaella; Bell, Daphne W.; Godin-Heymann, Nadia; Okimoto, Ross A.; Brannigan, Brian W.; Harris, Patricia L.; Driscoll, David R.; Fidias, Panos; Lynch, Thomas J.; Rabindran, Sridhar K.; McGinnis, John P.; Wissner, Allan; Sharma, Sreenath V.; Isselbacher, Kurt J.; Settleman, Jeffrey; Haber, Daniel A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7665-7670</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Nonsmall cell lung cancers (NSCLCs) with activating mutations in the kinase domain of the epidermal growth factor receptor (EGFR) demonstrate dramatic, but transient, responses to the reversible tyrosine kinase inhibitors gefitinib (Iressa) and erlotinib (Tarceva).  Some recurrent tumors have a common secondary mutation in the EGFR kinase domain, T790M, conferring drug resistance, but in other cases the mechanism underlying acquired resistance is unknown.  In studying multiple sites of recurrent NSCLCs, the authors detected T790M in only a small percentage of tumor cells.  To identify addnl. mechanisms of acquired resistance to gefitinib, the authors used NSCLC cells harboring an activating EGFR mutation to generate multiple resistant clones in vitro.  These drug-resistant cells demonstrate continued dependence on EGFR and ERBB2 signaling for their viability and have not acquired secondary EGFR mutations.  However, they display increased internalization of ligand-activated EGFR, consistent with altered receptor trafficking.  Although gefitinib-resistant clones are cross-resistant to related anilinoquinazolines, they demonstrate sensitivity to a class of irreversible inhibitors of EGFR.  These inhibitors also show effective inhibition of signaling by T790M-mutant EGFR and killing of NSCLC cells with the T790M mutation.  Both mechanisms of gefitinib resistance are therefore circumvented by irreversible tyrosine kinase inhibitors.  Our findings suggest that one of these, HKI-272, may prove highly effective in the treatment of EGFR-mutant NSCLCs, including tumors that have become resistant to gefitinib or erlotinib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqW5BSM8yaZe7Vg90H21EOLACvtfcHk0ljb-Jm_pEdsNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkslOmsr0%253D&md5=f3fb866c3673fd4dff8a3f0a8c9be65b</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0502860102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0502860102%26sid%3Dliteratum%253Aachs%26aulast%3DKwak%26aufirst%3DE.%2BL.%26aulast%3DSordella%26aufirst%3DR.%26aulast%3DBell%26aufirst%3DD.%2BW.%26aulast%3DGodin-Heymann%26aufirst%3DN.%26aulast%3DOkimoto%26aufirst%3DR.%2BA.%26aulast%3DBrannigan%26aufirst%3DB.%2BW.%26aulast%3DHarris%26aufirst%3DP.%2BL.%26aulast%3DDriscoll%26aufirst%3DD.%2BR.%26aulast%3DFidias%26aufirst%3DP.%26aulast%3DLynch%26aufirst%3DT.%2BJ.%26aulast%3DRabindran%26aufirst%3DS.%2BK.%26aulast%3DMcGinnis%26aufirst%3DJ.%2BP.%26aulast%3DWissner%26aufirst%3DA.%26aulast%3DSharma%26aufirst%3DS.%2BV.%26aulast%3DIsselbacher%26aufirst%3DK.%2BJ.%26aulast%3DSettleman%26aufirst%3DJ.%26aulast%3DHaber%26aufirst%3DD.%2BA.%26atitle%3DIrreversible%2520inhibitors%2520of%2520the%2520EGF%2520receptor%2520may%2520circumvent%2520acquired%2520resistance%2520to%2520gefitinib%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D7665%26epage%3D7670%26doi%3D10.1073%2Fpnas.0502860102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Engelman, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zejnullahu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gale, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lifshits, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzales, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimamura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vincent, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naumov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Althaus, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gandhi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heymach, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jänne, P. A.</span><span> </span><span class="NLM_article-title">PF00299804, an irreversible pan-ERBB inhibitor, is effective in lung cancer models with EGFR and ERBB2 mutations that are resistant to gefitinib</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">11924</span><span class="NLM_x">–</span> <span class="NLM_lpage">11932</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-07-1885</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1158%2F0008-5472.CAN-07-1885" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=18089823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVCjs7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=11924-11932&author=J.+A.+Engelmanauthor=K.+Zejnullahuauthor=C.+M.+Galeauthor=E.+Lifshitsauthor=A.+J.+Gonzalesauthor=T.+Shimamuraauthor=F.+Zhaoauthor=P.+W.+Vincentauthor=G.+N.+Naumovauthor=J.+E.+Bradnerauthor=I.+W.+Althausauthor=L.+Gandhiauthor=G.+I.+Shapiroauthor=J.+M.+Nelsonauthor=J.+V.+Heymachauthor=M.+Meyersonauthor=K.+K.+Wongauthor=P.+A.+J%C3%A4nne&title=PF00299804%2C+an+irreversible+pan-ERBB+inhibitor%2C+is+effective+in+lung+cancer+models+with+EGFR+and+ERBB2+mutations+that+are+resistant+to+gefitinib&doi=10.1158%2F0008-5472.CAN-07-1885"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">PF00299804, an Irreversible Pan-ERBB Inhibitor, Is Effective in Lung Cancer Models with EGFR and ERBB2 Mutations that Are Resistant to Gefitinib</span></div><div class="casAuthors">Engelman, Jeffrey A.; Zejnullahu, Kreshnik; Gale, Christopher-Michael; Lifshits, Eugene; Gonzales, Andrea J.; Shimamura, Takeshi; Zhao, Feng; Vincent, Patrick W.; Naumov, George N.; Bradner, James E.; Althaus, Irene W.; Gandhi, Leena; Shapiro, Geoffrey I.; Nelson, James M.; Heymach, John V.; Meyerson, Matthew; Wong, Kwok-Kin; Jaenne, Pasi A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11924-11932</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors gefitinib and erlotinib are effective treatments for a subset of non-small cell lung cancers.  In particular, cancers with specific EGFR-activating mutations seem to be the most sensitive to these agents.  However, despite their initial response, such cancers almost invariably develop resistance.  In 50% of such cancers, a secondary EGFR mutation, T790M, has been identified that renders gefitinib and erlotinib ineffective inhibitors of EGFR kinase activity.  Thus, there is a clin. need to develop novel EGFR inhibitors that can effectively inactivate T790M-contg. EGFR proteins.  In this study, we evaluate the effectiveness of a novel compd., PF00299804, an irreversible pan-ERBB inhibitor.  The results from these studies show that PF00299804 is a potent inhibitor of EGFR-activating mutations as well as the EGFR T790M resistance mutation both in vitro and in vivo.  Addnl., PF00299804 is a highly effective inhibitor of both the wild-type ERBB2 and the gefitinib-resistant oncogenic ERBB2 mutation identified in lung cancers.  These preclin. evaluations support further clin. development of PF00299804 for cancers with mutations and/or amplifications of ERBB family members. [Cancer Res 2007;67(24):11924-32].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq02Cx_x26RirVg90H21EOLACvtfcHk0liE_VcC05yqEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVCjs7nL&md5=c210cf2fb3d1479df5915d317259696f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-07-1885&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-07-1885%26sid%3Dliteratum%253Aachs%26aulast%3DEngelman%26aufirst%3DJ.%2BA.%26aulast%3DZejnullahu%26aufirst%3DK.%26aulast%3DGale%26aufirst%3DC.%2BM.%26aulast%3DLifshits%26aufirst%3DE.%26aulast%3DGonzales%26aufirst%3DA.%2BJ.%26aulast%3DShimamura%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DF.%26aulast%3DVincent%26aufirst%3DP.%2BW.%26aulast%3DNaumov%26aufirst%3DG.%2BN.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DAlthaus%26aufirst%3DI.%2BW.%26aulast%3DGandhi%26aufirst%3DL.%26aulast%3DShapiro%26aufirst%3DG.%2BI.%26aulast%3DNelson%26aufirst%3DJ.%2BM.%26aulast%3DHeymach%26aufirst%3DJ.%2BV.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26atitle%3DPF00299804%252C%2520an%2520irreversible%2520pan-ERBB%2520inhibitor%252C%2520is%2520effective%2520in%2520lung%2520cancer%2520models%2520with%2520EGFR%2520and%2520ERBB2%2520mutations%2520that%2520are%2520resistant%2520to%2520gefitinib%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26spage%3D11924%26epage%3D11932%26doi%3D10.1158%2F0008-5472.CAN-07-1885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ambrogio, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimamura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chirieac, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padera, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baum, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Himmelsbach, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rettig, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solca, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greulich, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span> </span><span class="NLM_article-title">BIBW2992, an irreversible EGFR/HER2 inhibitor highly effective in preclinical lung cancer models</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">4702</span><span class="NLM_x">–</span> <span class="NLM_lpage">4711</span><span class="refDoi"> DOI: 10.1038/onc.2008.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1038%2Fonc.2008.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=18408761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptlKnsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=4702-4711&author=D.+Liauthor=L.+Ambrogioauthor=T.+Shimamuraauthor=S.+Kuboauthor=M.+Takahashiauthor=L.+R.+Chirieacauthor=R.+F.+Paderaauthor=G.+I.+Shapiroauthor=A.+Baumauthor=F.+Himmelsbachauthor=W.+J.+Rettigauthor=M.+Meyersonauthor=F.+Solcaauthor=H.+Greulichauthor=K.+K.+Wong&title=BIBW2992%2C+an+irreversible+EGFR%2FHER2+inhibitor+highly+effective+in+preclinical+lung+cancer+models&doi=10.1038%2Fonc.2008.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">BIBW2992, an irreversible EGFR/HER2 inhibitor highly effective in preclinical lung cancer models</span></div><div class="casAuthors">Li, D.; Ambrogio, L.; Shimamura, T.; Kubo, S.; Takahashi, M.; Chirieac, L. R.; Padera, R. F.; Shapiro, G. I.; Baum, A.; Himmelsbach, F.; Rettig, W. J.; Meyerson, M.; Solca, F.; Greulich, H.; Wong, K-K.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">4702-4711</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Genetic alterations in the kinase domain of the epidermal growth factor receptor (EGFR) in non-small cell lung cancer (NSCLC) patients are assocd. with sensitivity to treatment with small mol. tyrosine kinase inhibitors.  Although first-generation reversible, ATP-competitive inhibitors showed encouraging clin. responses in lung adenocarcinoma tumors harboring such EGFR mutations, almost all patients developed resistance to these inhibitors over time.  Such resistance to first-generation EGFR inhibitors was frequently linked to an acquired T790M point mutation in the kinase domain of EGFR, or upregulation of signaling pathways downstream of HER3.  Overcoming these mechanisms of resistance, as well as primary resistance to reversible EGFR inhibitors driven by a subset of EGFR mutations, will be necessary for development of an effective targeted therapy regimen.  Here, we show that BIBW2992, an anilino-quinazoline designed to irreversibly bind EGFR and HER2, potently suppresses the kinase activity of wild-type and activated EGFR and HER2 mutants, including erlotinib-resistant isoforms.  Consistent with this activity, BIBW2992 suppresses transformation in isogenic cell-based assays, inhibits survival of cancer cell lines and induces tumor regression in xenograft and transgenic lung cancer models, with superior activity over erlotinib.  These findings encourage further testing of BIBW2992 in lung cancer patients harboring EGFR or HER2 oncogenes.  Oncogene (2008) 27, 4702-4711; doi:10.1038/onc.2008.109; published online 14 Apr. 2008.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5PQdKep-Sp7Vg90H21EOLACvtfcHk0liE_VcC05yqEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptlKnsbk%253D&md5=24945893e83b4e3d01c267100fdaf57c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fonc.2008.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2008.109%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.%26aulast%3DAmbrogio%26aufirst%3DL.%26aulast%3DShimamura%26aufirst%3DT.%26aulast%3DKubo%26aufirst%3DS.%26aulast%3DTakahashi%26aufirst%3DM.%26aulast%3DChirieac%26aufirst%3DL.%2BR.%26aulast%3DPadera%26aufirst%3DR.%2BF.%26aulast%3DShapiro%26aufirst%3DG.%2BI.%26aulast%3DBaum%26aufirst%3DA.%26aulast%3DHimmelsbach%26aufirst%3DF.%26aulast%3DRettig%26aufirst%3DW.%2BJ.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DSolca%26aufirst%3DF.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DWong%26aufirst%3DK.%2BK.%26atitle%3DBIBW2992%252C%2520an%2520irreversible%2520EGFR%252FHER2%2520inhibitor%2520highly%2520effective%2520in%2520preclinical%2520lung%2520cancer%2520models%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D4702%26epage%3D4711%26doi%3D10.1038%2Fonc.2008.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pompliano, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meek, T. D.</span><span> </span><span class="NLM_article-title">Drug-target residence time and its implications for lead optimization</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">730</span><span class="NLM_x">–</span> <span class="NLM_lpage">739</span><span class="refDoi"> DOI: 10.1038/nrd2082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1038%2Fnrd2082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=16888652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=730-739&author=R.+A.+Copelandauthor=D.+L.+Pomplianoauthor=T.+D.+Meek&title=Drug-target+residence+time+and+its+implications+for+lead+optimization&doi=10.1038%2Fnrd2082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target residence time and its implications for lead optimization</span></div><div class="casAuthors">Copeland, Robert A.; Pompliano, David L.; Meek, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">730-739</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Much of drug discovery today is predicated on the concept of selective targeting of particular bioactive macromols. by low-mol.-mass drugs.  The binding of drugs to their macromol. targets is therefore seen as paramount for pharmacol. activity.  In vitro assessment of drug-target interactions is classically quantified in terms of binding parameters such as IC50 or Kd.  This article presents an alternative perspective on drug optimization in terms of drug-target binary complex residence time, as quantified by the dissociative half-life of the drug-target binary complex.  We describe the potential advantages of long residence time in terms of duration of pharmacol. effect and target selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry6SJrMlrrTbVg90H21EOLACvtfcHk0liE_VcC05yqEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D&md5=60ede2301584b10ac4e8fa18e1e6d107</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrd2082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2082%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DPompliano%26aufirst%3DD.%2BL.%26aulast%3DMeek%26aufirst%3DT.%2BD.%26atitle%3DDrug-target%2520residence%2520time%2520and%2520its%2520implications%2520for%2520lead%2520optimization%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D730%26epage%3D739%26doi%3D10.1038%2Fnrd2082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Heuckmann, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rauh, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, R. K.</span><span> </span><span class="NLM_article-title">Epidermal growth factor receptor (EGFR) signaling and covalent EGFR inhibition in lung cancer</span> <span class="citation_source-journal">J. Clin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">3417</span><span class="NLM_x">–</span> <span class="NLM_lpage">3420</span><span class="refDoi"> DOI: 10.1200/JCO.2012.43.1825</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1200%2FJCO.2012.43.1825" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=22915655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Glt73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2012&pages=3417-3420&author=J.+M.+Heuckmannauthor=D.+Rauhauthor=R.+K.+Thomas&title=Epidermal+growth+factor+receptor+%28EGFR%29+signaling+and+covalent+EGFR+inhibition+in+lung+cancer&doi=10.1200%2FJCO.2012.43.1825"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal growth factor receptor (EGFR) signaling and covalent EGF inhibition in lung cancer</span></div><div class="casAuthors">Heuckmann, Johannes M.; Rauh, Daniel; Thomas, Roman K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">3417-3420</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">0732-183X</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">A review.  The research of Ramalingam et al. (2012) entitled, 'Randomized phase II study of dacomitinib (PF-00299804), an irreversible pan-human epidermal growth factor receptor inhibitor, vs. erlotinib in patients with advanced non-small-cell lung cancer' is reviewed with commentary and refs.  In their paper, a novel irreversible ERBB family inhibitor, dacomitinib, improved progression-free survival compared with erlotinib in unselected patients with non-small-cell lung cancer without prior epidermal growth factor receptor-directed therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8PW4hGdL2k7Vg90H21EOLACvtfcHk0liyLSs25sDLHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Glt73O&md5=45192d5ee846d2c25302286226cff733</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1200%2FJCO.2012.43.1825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2012.43.1825%26sid%3Dliteratum%253Aachs%26aulast%3DHeuckmann%26aufirst%3DJ.%2BM.%26aulast%3DRauh%26aufirst%3DD.%26aulast%3DThomas%26aufirst%3DR.%2BK.%26atitle%3DEpidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520signaling%2520and%2520covalent%2520EGFR%2520inhibition%2520in%2520lung%2520cancer%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2012%26volume%3D30%26spage%3D3417%26epage%3D3420%26doi%3D10.1200%2FJCO.2012.43.1825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Singh, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petter, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baillie, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitty, A.</span><span> </span><span class="NLM_article-title">The resurgence of covalent drugs</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">317</span><span class="refDoi"> DOI: 10.1038/nrd3410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1038%2Fnrd3410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=21455239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=307-317&author=J.+Singhauthor=R.+C.+Petterauthor=T.+A.+Baillieauthor=A.+Whitty&title=The+resurgence+of+covalent+drugs&doi=10.1038%2Fnrd3410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The resurgence of covalent drugs</span></div><div class="casAuthors">Singh, Juswinder; Petter, Russell C.; Baillie, Thomas A.; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent drugs haveproved to be successful therapies for various indications, but largely owing to safety concerns, they are rarely considered when initiating a target-directed drug discovery project.  There is a need to reassess this important class of drugs, and to reconcile the discordance between the historic success of covalent drugs and the reluctance of most drug discovery teams to include them in their armamentarium.  This Review surveys the prevalence and pharmacol. advantages of covalent drugs, discusses how potential risks and challenges may be addressed through innovative design, and presents the broad opportunities provided by targeted covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreDCEmORV_NbVg90H21EOLACvtfcHk0liyLSs25sDLHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D&md5=2190289081e151416c097be4a5b04460</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrd3410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3410%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DPetter%26aufirst%3DR.%2BC.%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DThe%2520resurgence%2520of%2520covalent%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D307%26epage%3D317%26doi%3D10.1038%2Fnrd3410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Katakami, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atagi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hida, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horai, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inoue, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichinose, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koboyashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiura, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishio, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ebisawa, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shahidi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamamoto, N.</span><span> </span><span class="NLM_article-title">LUX-Lung 4: a phase II trial of afatinib in patients with advanced non-small-cell lung cancer who progressed during prior treatment with erlotinib, gefitinib, or both</span> <span class="citation_source-journal">J. Clin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">3335</span><span class="NLM_x">–</span> <span class="NLM_lpage">3341</span><span class="refDoi"> DOI: 10.1200/JCO.2012.45.0981</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1200%2FJCO.2012.45.0981" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=23816963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1KrsrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=3335-3341&author=N.+Katakamiauthor=S.+Atagiauthor=K.+Gotoauthor=T.+Hidaauthor=T.+Horaiauthor=A.+Inoueauthor=Y.+Ichinoseauthor=K.+Koboyashiauthor=K.+Takedaauthor=K.+Kiuraauthor=K.+Nishioauthor=Y.+Sekiauthor=R.+Ebisawaauthor=M.+Shahidiauthor=N.+Yamamoto&title=LUX-Lung+4%3A+a+phase+II+trial+of+afatinib+in+patients+with+advanced+non-small-cell+lung+cancer+who+progressed+during+prior+treatment+with+erlotinib%2C+gefitinib%2C+or+both&doi=10.1200%2FJCO.2012.45.0981"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">LUX-lung 4: a phase II trial of afatinib in patients with advanced non-small-cell lung cancer who progressed during prior treatment with erlotinib, gefitinib, or both</span></div><div class="casAuthors">Katakami, Noboyuki; Atagi, Shinji; Goto, Koichi; Hida, Toyoaki; Horai, Takeshi; Inoue, Akira; Ichinose, Yukito; Koboyashi, Kunihiko; Takeda, Koji; Kiura, Katsuyuki; Nishio, Kazuto; Seki, Yoko; Ebisawa, Ryuichi; Shahidi, Mehdi; Yamamoto, Nobuyuki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">3335-3341</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">0732-183X</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">Purpose New mol. targeted agents are needed for patients with non-small-cell lung cancer (NSCLC) who progress while receiving erlotinib, gefitinib, or both.  Afatinib, an oral irreversible ErbB family blocker, has preclin. activity in epidermal growth factor receptor (EGFR [ErbB1]) mutant models with EGFR-activating mutations, including T790M.  Patients and Methods This was a Japanese single-arm phase II trial conducted in patients with stage IIIB to IV pulmonary adenocarcinoma who progressed after ≥ 12 wk of prior erlotinib and/or gefitinib.  Patients received afatinib 50 mg per day.  The primary end point was objective response rate (complete response or partial response) by independent review.  Secondary end points included progression-free survival (PFS), overall survival (OS), and safety.  Results Of 62 treated patients, 45 (72.6%) were EGFR mutation pos. in their primary tumor according to local and/or central lab. analyses.  Fifty-one patients (82.3%) fulfilled the criteria of acquired resistance to erlotinib and/or gefitinib.  Of 61 evaluable patients, five (8.2%; 95% CI, 2.7% to 18.1 %) had a confirmed objective response rate (partial response).  Median PFS was 4.4 mo (95% CI, 2.8 to 4.6 mo), and median OS was 19.0 mo (95% CI, 14.9 mo to not achieved).  Two patients had acquired T790M mutations: L858R + T790M, and deletion in exon 19 + T790M; they had stable disease for 9 mo and 1 mo, resp.  The most common afatinib-related adverse events (AEs) were diarrhea (100%) and rash/acne (91.9%).  Treatment-related AEs leading to afatinib discontinuation were experienced by 18 patients (29%), of whom four also had progressive disease.  Conclusion Afatinib demonstrated modest but noteworthy efficacy in patients with NSCLC who had received third- or fourth-line treatment and who progressed while receiving erlotinib and/or gefitinib, including those with acquired resistance to erlotinib, gefitinib, or both.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpM0F6fZnWKTbVg90H21EOLACvtfcHk0liyLSs25sDLHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1KrsrzK&md5=1666aeb642dbc953dd9bcc818bab9098</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1200%2FJCO.2012.45.0981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2012.45.0981%26sid%3Dliteratum%253Aachs%26aulast%3DKatakami%26aufirst%3DN.%26aulast%3DAtagi%26aufirst%3DS.%26aulast%3DGoto%26aufirst%3DK.%26aulast%3DHida%26aufirst%3DT.%26aulast%3DHorai%26aufirst%3DT.%26aulast%3DInoue%26aufirst%3DA.%26aulast%3DIchinose%26aufirst%3DY.%26aulast%3DKoboyashi%26aufirst%3DK.%26aulast%3DTakeda%26aufirst%3DK.%26aulast%3DKiura%26aufirst%3DK.%26aulast%3DNishio%26aufirst%3DK.%26aulast%3DSeki%26aufirst%3DY.%26aulast%3DEbisawa%26aufirst%3DR.%26aulast%3DShahidi%26aufirst%3DM.%26aulast%3DYamamoto%26aufirst%3DN.%26atitle%3DLUX-Lung%25204%253A%2520a%2520phase%2520II%2520trial%2520of%2520afatinib%2520in%2520patients%2520with%2520advanced%2520non-small-cell%2520lung%2520cancer%2520who%2520progressed%2520during%2520prior%2520treatment%2520with%2520erlotinib%252C%2520gefitinib%252C%2520or%2520both%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2013%26volume%3D31%26spage%3D3335%26epage%3D3341%26doi%3D10.1200%2FJCO.2012.45.0981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Miller, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirsh, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cadranel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heo, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crino, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chao, T. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shahidi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cong, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorence, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J. C.</span><span> </span><span class="NLM_article-title">Afatinib versus placebo for patients with advanced, metastatic non-small-cell lung cancer after failure of erlotinib, gefitinib, or both, and one or two lines of chemotherapy (LUX-Lung 1): a phase 2b/3 randomised trial</span> <span class="citation_source-journal">Lancet Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">528</span><span class="NLM_x">–</span> <span class="NLM_lpage">538</span><span class="refDoi"> DOI: 10.1016/S1470-2045(12)70087-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1016%2FS1470-2045%2812%2970087-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=22452896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmslyhsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=528-538&author=V.+A.+Millerauthor=V.+Hirshauthor=J.+Cadranelauthor=Y.+M.+Chenauthor=K.+Parkauthor=S.+W.+Kimauthor=C.+Zhouauthor=W.+C.+Suauthor=M.+Wangauthor=Y.+Sunauthor=D.+S.+Heoauthor=L.+Crinoauthor=E.+H.+Tanauthor=T.+Y.+Chaoauthor=M.+Shahidiauthor=X.+J.+Congauthor=R.+M.+Lorenceauthor=J.+C.+Yang&title=Afatinib+versus+placebo+for+patients+with+advanced%2C+metastatic+non-small-cell+lung+cancer+after+failure+of+erlotinib%2C+gefitinib%2C+or+both%2C+and+one+or+two+lines+of+chemotherapy+%28LUX-Lung+1%29%3A+a+phase+2b%2F3+randomised+trial&doi=10.1016%2FS1470-2045%2812%2970087-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Afatinib versus placebo for patients with advanced, metastatic non-small-cell lung cancer after failure of erlotinib, gefitinib, or both, and one or two lines of chemotherapy (LUX-Lung 1): a phase 2b/3 randomised trial</span></div><div class="casAuthors">Miller, Vincent A.; Hirsh, Vera; Cadranel, Jacques; Chen, Yuh-Min; Park, Keunchil; Kim, Sang-We; Zhou, Caicun; Su, Wu-Chou; Wang, Mengzhao; Sun, Yan; Heo, Dae Seog; Crino, Lucio; Tan, Eng-Huat; Chao, Tsu-Yi; Shahidi, Mehdi; Cong, Xiuyu Julie; Lorence, Robert M.; Yang, James Chih-Hsin</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">528-538</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Summary: Background: Afatinib, an irreversible ErbB-family blocker, has shown preclin. activity when tested in EGFR mutant models with mutations that confer resistance to EGFR tyrosine-kinase inhibitors.  We aimed to assess its efficacy in patients with advanced lung adenocarcinoma with previous treatment failure on EGFR tyrosine-kinase inhibitors.  Methods: In this phase 2b/3 trial, we enrolled patients with stage IIIB or IV adenocarcinoma and an Eastern Cooperative Oncol. Group performance (ECOG) performance score of 0-2 who had received one or two previous chemotherapy regimens and had disease progression after at least 12 wk of treatment with erlotinib or gefitinib.  We used a computer-generated sequence to randomly allocate patients (2:1) to either afatinib (50 mg per day) or placebo; all patients received best supportive care.  Randomisation was done in blocks of three and was stratified by sex and baseline ECOG performance status (0-1 vs 2).  Investigators, patients, and the trial sponsor were masked to treatment assignment.  The primary endpoint was overall survival (from date of randomisation to death), analyzed on an intention-to-treat basis.  This study is registered with ClinicalTrials.gov, no. NCT00656136.  Findings: Between May 26, 2008, and Sept 21, 2009, we identified 697 patients, 585 of whom were randomly allocated to treatment (390 to afatinib, 195 to placebo).  Median overall survival was 10·8 mo (95% CI 10·0-12·0) in the afatinib group and 12·0 mo (10·2-14·3) in the placebo group (hazard ratio 1·08, 95% CI 0·86-1·35; p=0·74).  Median progression-free survival was longer in the afatinib group (3·3 mo, 95% CI 2·79-4·40) than it was in the placebo group (1·1 mo, 0·95-1·68; hazard ratio 0·38, 95% CI 0·31-0·48; p<0·0001).  No complete responses to treatment were noted; 29 (7%) patients had a partial response in the afatinib group, as did one patient in the placebo group.  Subsequent cancer treatment was given to 257 (68%) patients in the afatinib group and 153 (79%) patients in the placebo group.  The most common adverse events in the afatinib group were diarrhoea (339 [87%] of 390 patients; 66 [17%] were grade 3) and rash or acne (305 [78%] patients; 56 [14%] were grade 3).  These events occurred less often in the placebo group (18 [9%] of 195 patients had diarrhoea; 31 [16%] had rash or acne), all being grade 1 or 2.  Drug-related serious adverse events occurred in 39 (10%) patients in the afatinib group and one (<1%) patient in the placebo group.  We recorded two possibly treatment-related deaths in the afatinib group.  Interpretation: Although we recorded no benefit in terms of overall survival with afatinib (which might have been affected by cancer treatments given after progression in both groups), our findings for progression-free survival and response to treatment suggest that afatinib could be of some benefit to patients with advanced lung adenocarcinoma who have failed at least 12 wk of previous EGFR tyrosine-kinase inhibitor treatment.  Funding: Boehringer Ingelheim Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDrR4FNSnKJbVg90H21EOLACvtfcHk0lhmudmwbOMYiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmslyhsbg%253D&md5=6ddd7f552e8ab1ebb43fb699e27894f5</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2812%2970087-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252812%252970087-6%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DV.%2BA.%26aulast%3DHirsh%26aufirst%3DV.%26aulast%3DCadranel%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%2BM.%26aulast%3DPark%26aufirst%3DK.%26aulast%3DKim%26aufirst%3DS.%2BW.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DSu%26aufirst%3DW.%2BC.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DHeo%26aufirst%3DD.%2BS.%26aulast%3DCrino%26aufirst%3DL.%26aulast%3DTan%26aufirst%3DE.%2BH.%26aulast%3DChao%26aufirst%3DT.%2BY.%26aulast%3DShahidi%26aufirst%3DM.%26aulast%3DCong%26aufirst%3DX.%2BJ.%26aulast%3DLorence%26aufirst%3DR.%2BM.%26aulast%3DYang%26aufirst%3DJ.%2BC.%26atitle%3DAfatinib%2520versus%2520placebo%2520for%2520patients%2520with%2520advanced%252C%2520metastatic%2520non-small-cell%2520lung%2520cancer%2520after%2520failure%2520of%2520erlotinib%252C%2520gefitinib%252C%2520or%2520both%252C%2520and%2520one%2520or%2520two%2520lines%2520of%2520chemotherapy%2520%2528LUX-Lung%25201%2529%253A%2520a%2520phase%25202b%252F3%2520randomised%2520trial%26jtitle%3DLancet%2520Oncol.%26date%3D2012%26volume%3D13%26spage%3D528%26epage%3D538%26doi%3D10.1016%2FS1470-2045%2812%2970087-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Sequist, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Besse, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gitlitz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eaton, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zacharchuk, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freyman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ananthakrishnan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soria, J. C.</span><span> </span><span class="NLM_article-title">Neratinib, an irreversible pan-ErbB receptor tyrosine kinase inhibitor: results of a phase II trial in patients with advanced non-small-cell lung cancer</span> <span class="citation_source-journal">J. Clin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">3076</span><span class="NLM_x">–</span> <span class="NLM_lpage">3083</span><span class="refDoi"> DOI: 10.1200/JCO.2009.27.9414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1200%2FJCO.2009.27.9414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=20479403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptlaju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2010&pages=3076-3083&author=L.+V.+Sequistauthor=B.+Besseauthor=T.+J.+Lynchauthor=V.+A.+Millerauthor=K.+K.+Wongauthor=B.+Gitlitzauthor=K.+Eatonauthor=C.+Zacharchukauthor=A.+Freymanauthor=C.+Powellauthor=R.+Ananthakrishnanauthor=S.+Quinnauthor=J.+C.+Soria&title=Neratinib%2C+an+irreversible+pan-ErbB+receptor+tyrosine+kinase+inhibitor%3A+results+of+a+phase+II+trial+in+patients+with+advanced+non-small-cell+lung+cancer&doi=10.1200%2FJCO.2009.27.9414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Neratinib, an irreversible Pan-ErbB, receptor tyrosine kinase inhibitor: results of a phase II trial in patients with advanced non-small-cell lung cancer</span></div><div class="casAuthors">Sequist, Lecia V.; Besse, Benjamin; Lynch, Thomas J.; Miller, Vincent A.; Wong, Kwok K.; Gitlitz, Barbara; Eaton, Keith; Zacharchuk, Charles; Freyman, Amy; Powell, Christine; Ananthakrishnan, Revathi; Quinn, Susan; Soria, Jean-Charles</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3076-3083</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">0732-183X</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors (TKIs) have had a significant impact on non-small-cell lung cancer (NSCLC) outcomes, particularly for patients with EGFR mutations.  Resistance emerges after 9 to 12 mo, primarily mediated by the T790M resistance mutation.  We studied neratinib, an irreversible pan-ErbB TKI that may overcome T790M.  Patients with advanced NSCLC underwent EGFR sequencing of available tumor tissue at enrollment.  Those with ≥ 12 wk of prior TKI therapy were placed in arm A if they were EGFR mutation pos. or arm B if they were wild-type.  Arm C included TKI-naive patients with adenocarcinoma and light smoking histories (≤ 20 pack-years).  All patients received daily oral neratinib, initially at 320 mg but subsequently reduced to 240 mg because of excessive diarrhea.  The primary end point was objective response rate (RR).  167 Patients were treated: 91 in arm A, 48 in arm B, and 28 in arm C.  Diarrhea was the most common toxicity; grade 3 incidence was 50% at 320 mg but improved to 25% after dose redn.  The RR was 3% in arm A and zero in arms B and C.  No patients with known T790M responded.  Notably, three of four patients with an exon 18 G719X EGFR mutation had a partial response and the fourth had stable disease lasting 40 wk.  Neratinib had low activity in patients with prior benefit from TKIs and in TKI-naive patients, potentially because of insufficient bioavailability from diarrhea-imposed dose limitation.  Responses were seen in patients with the rare G719X EGFR mutation, highlighting the importance of obtaining comprehensive genetic information on trials of targeted agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-Z3gsvbWbOrVg90H21EOLACvtfcHk0lhmudmwbOMYiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptlaju7c%253D&md5=460e088e58919bbfcf8c153772f157b0</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1200%2FJCO.2009.27.9414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2009.27.9414%26sid%3Dliteratum%253Aachs%26aulast%3DSequist%26aufirst%3DL.%2BV.%26aulast%3DBesse%26aufirst%3DB.%26aulast%3DLynch%26aufirst%3DT.%2BJ.%26aulast%3DMiller%26aufirst%3DV.%2BA.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DGitlitz%26aufirst%3DB.%26aulast%3DEaton%26aufirst%3DK.%26aulast%3DZacharchuk%26aufirst%3DC.%26aulast%3DFreyman%26aufirst%3DA.%26aulast%3DPowell%26aufirst%3DC.%26aulast%3DAnanthakrishnan%26aufirst%3DR.%26aulast%3DQuinn%26aufirst%3DS.%26aulast%3DSoria%26aufirst%3DJ.%2BC.%26atitle%3DNeratinib%252C%2520an%2520irreversible%2520pan-ErbB%2520receptor%2520tyrosine%2520kinase%2520inhibitor%253A%2520results%2520of%2520a%2520phase%2520II%2520trial%2520in%2520patients%2520with%2520advanced%2520non-small-cell%2520lung%2520cancer%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2010%26volume%3D28%26spage%3D3076%26epage%3D3083%26doi%3D10.1200%2FJCO.2009.27.9414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Zhou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ercan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yun, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capelletti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortot, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chirieac, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iacob, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padera, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engen, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eck, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jänne, P. A.</span><span> </span><span class="NLM_article-title">Novel mutant-selective EGFR kinase inhibitors against EGFR T790M</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">462</span><span class="NLM_x">, </span> <span class="NLM_fpage">1070</span><span class="NLM_x">–</span> <span class="NLM_lpage">1074</span><span class="refDoi"> DOI: 10.1038/nature08622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1038%2Fnature08622" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=20033049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhs1SktbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=1070-1074&author=W.+Zhouauthor=D.+Ercanauthor=L.+Chenauthor=C.+H.+Yunauthor=D.+Liauthor=M.+Capellettiauthor=A.+B.+Cortotauthor=L.+Chirieacauthor=R.+E.+Iacobauthor=R.+Paderaauthor=J.+R.+Engenauthor=K.+K.+Wongauthor=M.+J.+Eckauthor=N.+S.+Grayauthor=P.+A.+J%C3%A4nne&title=Novel+mutant-selective+EGFR+kinase+inhibitors+against+EGFR+T790M&doi=10.1038%2Fnature08622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Novel mutant-selective EGFR kinase inhibitors against EGFR T790M</span></div><div class="casAuthors">Zhou, Wenjun; Ercan, Dalia; Chen, Liang; Yun, Cai-Hong; Li, Danan; Capelletti, Marzia; Cortot, Alexis B.; Chirieac, Lucian; Iacob, Roxana E.; Padera, Robert; Engen, John R.; Wong, Kwok-Kin; Eck, Michael J.; Gray, Nathanael S.; Janne, Pasi A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">462</span>
        (<span class="NLM_cas:issue">7276</span>),
    <span class="NLM_cas:pages">1070-1074</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. efficacy of epidermal growth factor receptor (EGFR) kinase inhibitors in EGFR-mutant non-small-cell lung cancer (NSCLC) is limited by the development of drug-resistance mutations, including the gatekeeper T790M mutation.  Strategies targeting EGFR T790M with irreversible inhibitors have had limited success and are assocd. with toxicity due to concurrent inhibition of wild-type EGFR.  All current EGFR inhibitors possess a structurally related quinazoline-based core scaffold and were identified as ATP-competitive inhibitors of wild-type EGFR.  Here we identify a covalent pyrimidine EGFR inhibitor by screening an irreversible kinase inhibitor library specifically against EGFR T790M.  These agents are 30- to 100-fold more potent against EGFR T790M, and up to 100-fold less potent against wild-type EGFR, than quinazoline-based EGFR inhibitors in vitro.  They are also effective in murine models of lung cancer driven by EGFR T790M.  Co-crystn. studies reveal a structural basis for the increased potency and mutant selectivity of these agents.  These mutant-selective irreversible EGFR kinase inhibitors may be clin. more effective and better tolerated than quinazoline-based inhibitors.  Our findings demonstrate that functional pharmacol. screens against clin. important mutant kinases represent a powerful strategy to identify new classes of mutant-selective kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcksZlIe-1bbVg90H21EOLACvtfcHk0lhOcicR-Tc4ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhs1SktbvI&md5=9b5d41a2be401c67a4b9f04a41cf1725</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnature08622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08622%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DYun%26aufirst%3DC.%2BH.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DCapelletti%26aufirst%3DM.%26aulast%3DCortot%26aufirst%3DA.%2BB.%26aulast%3DChirieac%26aufirst%3DL.%26aulast%3DIacob%26aufirst%3DR.%2BE.%26aulast%3DPadera%26aufirst%3DR.%26aulast%3DEngen%26aufirst%3DJ.%2BR.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26atitle%3DNovel%2520mutant-selective%2520EGFR%2520kinase%2520inhibitors%2520against%2520EGFR%2520T790M%26jtitle%3DNature%26date%3D2009%26volume%3D462%26spage%3D1070%26epage%3D1074%26doi%3D10.1038%2Fnature08622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Walter, A. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sjin, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haringsma, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubrovskiy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labenski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheets, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St. Martin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karp, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Kalken, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaturvedi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nacht, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petter, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westlin, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaw-Tsai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raponi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Dyke, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Etter, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weaver, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmons, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harding, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, A.</span><span> </span><span class="NLM_article-title">Discovery of a mutant-selective covalent inhibitor of EGFR that overcomes T790M-mediated resistance in NSCLC</span> <span class="citation_source-journal">Cancer Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1404</span><span class="NLM_x">–</span> <span class="NLM_lpage">1415</span><span class="refDoi"> DOI: 10.1158/2159-8290.CD-13-0314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1158%2F2159-8290.CD-13-0314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=24065731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvV2rtLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2013&pages=1404-1415&author=A.+O.+Walterauthor=R.+T.+Sjinauthor=H.+J.+Haringsmaauthor=K.+Ohashiauthor=J.+Sunauthor=K.+Leeauthor=A.+Dubrovskiyauthor=M.+Labenskiauthor=Z.+Zhuauthor=Z.+Wangauthor=M.+Sheetsauthor=T.+St.+Martinauthor=R.+Karpauthor=D.+van+Kalkenauthor=P.+Chaturvediauthor=D.+Niuauthor=M.+Nachtauthor=R.+C.+Petterauthor=W.+Westlinauthor=K.+Linauthor=S.+Jaw-Tsaiauthor=M.+Raponiauthor=T.+Van+Dykeauthor=J.+Etterauthor=Z.+Weaverauthor=W.+Paoauthor=J.+Singhauthor=A.+D.+Simmonsauthor=T.+C.+Hardingauthor=A.+Allen&title=Discovery+of+a+mutant-selective+covalent+inhibitor+of+EGFR+that+overcomes+T790M-mediated+resistance+in+NSCLC&doi=10.1158%2F2159-8290.CD-13-0314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Mutant-Selective Covalent Inhibitor of EGFR that Overcomes T790M-Mediated Resistance in NSCLC</span></div><div class="casAuthors">Walter, Annette O.; Sjin, Robert Tjin Tham; Haringsma, Henry J.; Ohashi, Kadoaki; Sun, Jing; Lee, Kwangho; Dubrovskiy, Aleksandr; Labenski, Matthew; Zhu, Zhendong; Wang, Zhigang; Sheets, Michael; St. Martin, Thia; Karp, Russell; van Kalken, Dan; Chaturvedi, Prasoon; Niu, Deqiang; Nacht, Mariana; Petter, Russell C.; Westlin, William; Lin, Kevin; Jaw-Tsai, Sarah; Raponi, Mitch; Van Dyke, Terry; Etter, Jeff; Weaver, Zoe; Pao, William; Singh, Juswinder; Simmons, Andrew D.; Harding, Thomas C.; Allen, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1404-1415</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Patients with non-small cell lung cancer (NSCLC) with activating EGF receptor (EGFR) mutations initially respond to first-generation reversible EGFR tyrosine kinase inhibitors.  However, clin. efficacy is limited by acquired resistance, frequently driven by the EGFRT790M mutation.  CO-1686 is a novel, irreversible, and orally delivered kinase inhibitor that specifically targets the mutant forms of EGFR, including T790M, while exhibiting minimal activity toward the wild-type (WT) receptor.  Oral administration of CO-1686 as single agent induces tumor regression in EGFR-mutated NSCLC tumor xenograft and transgenic models.  Minimal activity of CO-1686 against the WT EGFR receptor was obsd.  In NSCLC cells with acquired resistance to CO-1686 in vitro, there was no evidence of addnl. mutations or amplification of the EGFR gene, but resistant cells exhibited signs of epithelial-mesenchymal transition and demonstrated increased sensitivity to AKT inhibitors.  These results suggest that CO-1686 may offer a novel therapeutic option for patients with mutant EGFR NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR0CDWQZ5wn7Vg90H21EOLACvtfcHk0lhOcicR-Tc4ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvV2rtLvN&md5=3d111f4bf958f2fda6890fbc0e1d99f5</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-13-0314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-13-0314%26sid%3Dliteratum%253Aachs%26aulast%3DWalter%26aufirst%3DA.%2BO.%26aulast%3DSjin%26aufirst%3DR.%2BT.%26aulast%3DHaringsma%26aufirst%3DH.%2BJ.%26aulast%3DOhashi%26aufirst%3DK.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DDubrovskiy%26aufirst%3DA.%26aulast%3DLabenski%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DSheets%26aufirst%3DM.%26aulast%3DSt.%2BMartin%26aufirst%3DT.%26aulast%3DKarp%26aufirst%3DR.%26aulast%3Dvan%2BKalken%26aufirst%3DD.%26aulast%3DChaturvedi%26aufirst%3DP.%26aulast%3DNiu%26aufirst%3DD.%26aulast%3DNacht%26aufirst%3DM.%26aulast%3DPetter%26aufirst%3DR.%2BC.%26aulast%3DWestlin%26aufirst%3DW.%26aulast%3DLin%26aufirst%3DK.%26aulast%3DJaw-Tsai%26aufirst%3DS.%26aulast%3DRaponi%26aufirst%3DM.%26aulast%3DVan%2BDyke%26aufirst%3DT.%26aulast%3DEtter%26aufirst%3DJ.%26aulast%3DWeaver%26aufirst%3DZ.%26aulast%3DPao%26aufirst%3DW.%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DSimmons%26aufirst%3DA.%2BD.%26aulast%3DHarding%26aufirst%3DT.%2BC.%26aulast%3DAllen%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520a%2520mutant-selective%2520covalent%2520inhibitor%2520of%2520EGFR%2520that%2520overcomes%2520T790M-mediated%2520resistance%2520in%2520NSCLC%26jtitle%3DCancer%2520Discovery%26date%3D2013%26volume%3D3%26spage%3D1404%26epage%3D1415%26doi%3D10.1158%2F2159-8290.CD-13-0314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Sequist, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soria, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldman, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wakelee, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gadgeel, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varga, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Papadimitrakopoulou, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oxnard, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dziadziuszko, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aisner, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doebele, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galasso, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garon, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heist, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Logan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neal, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendenhall, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nichols, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piotrowska, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wozniak, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raponi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlovich, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaw-Tsai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isaacson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Despain, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matheny, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rolfe, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camidge, D. R.</span><span> </span><span class="NLM_article-title">Rociletinib in EGFR-mutated non-small-cell lung cancer</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">372</span><span class="NLM_x">, </span> <span class="NLM_fpage">1700</span><span class="NLM_x">–</span> <span class="NLM_lpage">1709</span><span class="refDoi"> DOI: 10.1056/NEJMoa1413654</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1056%2FNEJMoa1413654" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=25923550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A280%3ADC%252BC2Mjpt1yltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=372&publication_year=2015&pages=1700-1709&author=L.+V.+Sequistauthor=J.+C.+Soriaauthor=J.+W.+Goldmanauthor=H.+A.+Wakeleeauthor=S.+M.+Gadgeelauthor=A.+Vargaauthor=V.+Papadimitrakopoulouauthor=B.+J.+Solomonauthor=G.+R.+Oxnardauthor=R.+Dziadziuszkoauthor=D.+L.+Aisnerauthor=R.+C.+Doebeleauthor=C.+Galassoauthor=E.+B.+Garonauthor=R.+S.+Heistauthor=J.+Loganauthor=J.+W.+Nealauthor=M.+A.+Mendenhallauthor=S.+Nicholsauthor=Z.+Piotrowskaauthor=A.+J.+Wozniakauthor=M.+Raponiauthor=C.+A.+Karlovichauthor=S.+Jaw-Tsaiauthor=J.+Isaacsonauthor=D.+Despainauthor=S.+L.+Mathenyauthor=L.+Rolfeauthor=A.+R.+Allenauthor=D.+R.+Camidge&title=Rociletinib+in+EGFR-mutated+non-small-cell+lung+cancer&doi=10.1056%2FNEJMoa1413654"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Rociletinib in EGFR-mutated non-small-cell lung cancer</span></div><div class="casAuthors">Sequist Lecia V; Soria Jean-Charles; Goldman Jonathan W; Wakelee Heather A; Gadgeel Shirish M; Varga Andrea; Papadimitrakopoulou Vassiliki; Solomon Benjamin J; Oxnard Geoffrey R; Dziadziuszko Rafal; Aisner Dara L; Doebele Robert C; Galasso Cathy; Garon Edward B; Heist Rebecca S; Logan Jennifer; Neal Joel W; Mendenhall Melody A; Nichols Suzanne; Piotrowska Zofia; Wozniak Antoinette J; Raponi Mitch; Karlovich Chris A; Jaw-Tsai Sarah; Isaacson Jeffrey; Despain Darrin; Matheny Shannon L; Rolfe Lindsey; Allen Andrew R; Camidge D Ross</div><div class="citationInfo"><span class="NLM_cas:title">The New England journal of medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">372</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">1700-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Non-small-cell lung cancer (NSCLC) with a mutation in the gene encoding epidermal growth factor receptor (EGFR) is sensitive to approved EGFR inhibitors, but resistance develops, mediated by the T790M EGFR mutation in most cases.  Rociletinib (CO-1686) is an EGFR inhibitor active in preclinical models of EGFR-mutated NSCLC with or without T790M.  METHODS:  In this phase 1-2 study, we administered rociletinib to patients with EGFR-mutated NSCLC who had disease progression during previous treatment with an existing EGFR inhibitor.  In the expansion (phase 2) part of the study, patients with T790M-positive disease received rociletinib at a dose of 500 mg twice daily, 625 mg twice daily, or 750 mg twice daily.  Key objectives were assessment of safety, side-effect profile, pharmacokinetics, and preliminary antitumor activity of rociletinib.  Tumor biopsies to identify T790M were performed during screening.  Treatment was administered in continuous 21-day cycles.  RESULTS:  A total of 130 patients were enrolled.  The first 57 patients to be enrolled received the free-base form of rociletinib (150 mg once daily to 900 mg twice daily).  The remaining patients received the hydrogen bromide salt (HBr) form (500 mg twice daily to 1000 mg twice daily).  A maximum tolerated dose (the highest dose associated with a rate of dose-limiting toxic effects of less than 33%) was not identified.  The only common dose-limiting adverse event was hyperglycemia.  In an efficacy analysis that included patients who received free-base rociletinib at a dose of 900 mg twice daily or the HBr form at any dose, the objective response rate among the 46 patients with T790M-positive disease who could be evaluated was 59% (95% confidence interval [CI], 45 to 73), and the rate among the 17 patients with T790M-negative disease who could be evaluated was 29% (95% CI, 8 to 51).  CONCLUSIONS:  Rociletinib was active in patients with EGFR-mutated NSCLC associated with the T790M resistance mutation. (Funded by Clovis Oncology; ClinicalTrials.gov number, NCT01526928.).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQME11ovAjH3ZPq5LRczm8ffW6udTcc2eYYCLjQtXwMt7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mjpt1yltQ%253D%253D&md5=5a5d21d36c081a5ef0f9e650d5cc84b9</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1413654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1413654%26sid%3Dliteratum%253Aachs%26aulast%3DSequist%26aufirst%3DL.%2BV.%26aulast%3DSoria%26aufirst%3DJ.%2BC.%26aulast%3DGoldman%26aufirst%3DJ.%2BW.%26aulast%3DWakelee%26aufirst%3DH.%2BA.%26aulast%3DGadgeel%26aufirst%3DS.%2BM.%26aulast%3DVarga%26aufirst%3DA.%26aulast%3DPapadimitrakopoulou%26aufirst%3DV.%26aulast%3DSolomon%26aufirst%3DB.%2BJ.%26aulast%3DOxnard%26aufirst%3DG.%2BR.%26aulast%3DDziadziuszko%26aufirst%3DR.%26aulast%3DAisner%26aufirst%3DD.%2BL.%26aulast%3DDoebele%26aufirst%3DR.%2BC.%26aulast%3DGalasso%26aufirst%3DC.%26aulast%3DGaron%26aufirst%3DE.%2BB.%26aulast%3DHeist%26aufirst%3DR.%2BS.%26aulast%3DLogan%26aufirst%3DJ.%26aulast%3DNeal%26aufirst%3DJ.%2BW.%26aulast%3DMendenhall%26aufirst%3DM.%2BA.%26aulast%3DNichols%26aufirst%3DS.%26aulast%3DPiotrowska%26aufirst%3DZ.%26aulast%3DWozniak%26aufirst%3DA.%2BJ.%26aulast%3DRaponi%26aufirst%3DM.%26aulast%3DKarlovich%26aufirst%3DC.%2BA.%26aulast%3DJaw-Tsai%26aufirst%3DS.%26aulast%3DIsaacson%26aufirst%3DJ.%26aulast%3DDespain%26aufirst%3DD.%26aulast%3DMatheny%26aufirst%3DS.%2BL.%26aulast%3DRolfe%26aufirst%3DL.%26aulast%3DAllen%26aufirst%3DA.%2BR.%26aulast%3DCamidge%26aufirst%3DD.%2BR.%26atitle%3DRociletinib%2520in%2520EGFR-mutated%2520non-small-cell%2520lung%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2015%26volume%3D372%26spage%3D1700%26epage%3D1709%26doi%3D10.1056%2FNEJMoa1413654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Cross, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashton, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghiorghiu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberlein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nebhan, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spitzler, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orme, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finlay, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mellor, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brewer, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichihara, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Kadhimi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowlinson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klinowska, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richmond, G. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cantarini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ranson, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pao, W.</span><span> </span><span class="NLM_article-title">AZD9291, an irreversible EGFR TKI, overcomes T790M-mediated resistance to EGFR inhibitors in lung cancer</span> <span class="citation_source-journal">Cancer Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1046</span><span class="NLM_x">–</span> <span class="NLM_lpage">1061</span><span class="refDoi"> DOI: 10.1158/2159-8290.CD-14-0337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1158%2F2159-8290.CD-14-0337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=24893891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVykur%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=1046-1061&author=D.+A.+Crossauthor=S.+E.+Ashtonauthor=S.+Ghiorghiuauthor=C.+Eberleinauthor=C.+A.+Nebhanauthor=P.+J.+Spitzlerauthor=J.+P.+Ormeauthor=M.+R.+Finlayauthor=R.+A.+Wardauthor=M.+J.+Mellorauthor=G.+Hughesauthor=A.+Rahiauthor=V.+N.+Jacobsauthor=M.+R.+Brewerauthor=E.+Ichiharaauthor=J.+Sunauthor=H.+Jinauthor=P.+Ballardauthor=K.+Al-Kadhimiauthor=R.+Rowlinsonauthor=T.+Klinowskaauthor=G.+H.+Richmondauthor=M.+Cantariniauthor=D.+W.+Kimauthor=M.+R.+Ransonauthor=W.+Pao&title=AZD9291%2C+an+irreversible+EGFR+TKI%2C+overcomes+T790M-mediated+resistance+to+EGFR+inhibitors+in+lung+cancer&doi=10.1158%2F2159-8290.CD-14-0337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">AZD9291, an Irreversible EGFR TKI, Overcomes T790M-Mediated Resistance to EGFR Inhibitors in Lung Cancer</span></div><div class="casAuthors">Cross, Darren A. E.; Ashton, Susan E.; Ghiorghiu, Serban; Eberlein, Cath; Nebhan, Caroline A.; Spitzler, Paula J.; Orme, Jonathon P.; Finlay, M. Raymond V.; Ward, Richard A.; Mellor, Martine J.; Hughes, Gareth; Rahi, Amar; Jacobs, Vivien N.; Brewer, Monica Red; Ichihara, Eiki; Sun, Jing; Jin, Hailing; Ballard, Peter; Al-Kadhimi, Katherine; Rowlinson, Rachel; Klinowska, Teresa; Richmond, Graham H. P.; Cantarini, Mireille; Kim, Dong-Wan; Ranson, Malcolm R.; Pao, William</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1046-1061</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A third-generation EGFR inhibitor selectively targets EGFR mutants, including T790M, but not wild-type EGFR, and induces durable antitumor responses in preclin. models and patients with NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDo3e2pAh8YbVg90H21EOLACvtfcHk0lieMoMJSL4Gig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVykur%252FP&md5=f72e56f9463236ddc89eef32a4ab61b3</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-14-0337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-14-0337%26sid%3Dliteratum%253Aachs%26aulast%3DCross%26aufirst%3DD.%2BA.%26aulast%3DAshton%26aufirst%3DS.%2BE.%26aulast%3DGhiorghiu%26aufirst%3DS.%26aulast%3DEberlein%26aufirst%3DC.%26aulast%3DNebhan%26aufirst%3DC.%2BA.%26aulast%3DSpitzler%26aufirst%3DP.%2BJ.%26aulast%3DOrme%26aufirst%3DJ.%2BP.%26aulast%3DFinlay%26aufirst%3DM.%2BR.%26aulast%3DWard%26aufirst%3DR.%2BA.%26aulast%3DMellor%26aufirst%3DM.%2BJ.%26aulast%3DHughes%26aufirst%3DG.%26aulast%3DRahi%26aufirst%3DA.%26aulast%3DJacobs%26aufirst%3DV.%2BN.%26aulast%3DBrewer%26aufirst%3DM.%2BR.%26aulast%3DIchihara%26aufirst%3DE.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DH.%26aulast%3DBallard%26aufirst%3DP.%26aulast%3DAl-Kadhimi%26aufirst%3DK.%26aulast%3DRowlinson%26aufirst%3DR.%26aulast%3DKlinowska%26aufirst%3DT.%26aulast%3DRichmond%26aufirst%3DG.%2BH.%26aulast%3DCantarini%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DD.%2BW.%26aulast%3DRanson%26aufirst%3DM.%2BR.%26aulast%3DPao%26aufirst%3DW.%26atitle%3DAZD9291%252C%2520an%2520irreversible%2520EGFR%2520TKI%252C%2520overcomes%2520T790M-mediated%2520resistance%2520to%2520EGFR%2520inhibitors%2520in%2520lung%2520cancer%26jtitle%3DCancer%2520Discovery%26date%3D2014%26volume%3D4%26spage%3D1046%26epage%3D1061%26doi%3D10.1158%2F2159-8290.CD-14-0337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Jänne, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Planchard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohe, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramalingam, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahn, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horn, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haggstrom, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felip, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frewer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cantarini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dickinson, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghiorghiu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ranson, M.</span><span> </span><span class="NLM_article-title">AZD9291 in EGFR inhibitor-resistant non-small-cell lung cancer</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">372</span><span class="NLM_x">, </span> <span class="NLM_fpage">1689</span><span class="NLM_x">–</span> <span class="NLM_lpage">1699</span><span class="refDoi"> DOI: 10.1056/NEJMoa1411817</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1056%2FNEJMoa1411817" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=25923549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A280%3ADC%252BC2Mjpt1yltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=372&publication_year=2015&pages=1689-1699&author=P.+A.+J%C3%A4nneauthor=J.+C.+Yangauthor=D.+W.+Kimauthor=D.+Planchardauthor=Y.+Oheauthor=S.+S.+Ramalingamauthor=M.+J.+Ahnauthor=S.+W.+Kimauthor=W.+C.+Suauthor=L.+Hornauthor=D.+Haggstromauthor=E.+Felipauthor=J.+H.+Kimauthor=P.+Frewerauthor=M.+Cantariniauthor=K.+H.+Brownauthor=P.+A.+Dickinsonauthor=S.+Ghiorghiuauthor=M.+Ranson&title=AZD9291+in+EGFR+inhibitor-resistant+non-small-cell+lung+cancer&doi=10.1056%2FNEJMoa1411817"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">AZD9291 in EGFR inhibitor-resistant non-small-cell lung cancer</span></div><div class="casAuthors">Janne Pasi A; Yang James Chih-Hsin; Kim Dong-Wan; Planchard David; Ohe Yuichiro; Ramalingam Suresh S; Ahn Myung-Ju; Kim Sang-We; Su Wu-Chou; Horn Leora; Haggstrom Daniel; Felip Enriqueta; Kim Joo-Hang; Frewer Paul; Cantarini Mireille; Brown Kathryn H; Dickinson Paul A; Ghiorghiu Serban; Ranson Malcolm</div><div class="citationInfo"><span class="NLM_cas:title">The New England journal of medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">372</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">1689-99</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The EGFR T790M mutation is the most common mechanism of drug resistance to epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors in patients who have lung cancer with an EGFR mutation (EGFR-mutated lung cancer).  In preclinical models, the EGFR inhibitor AZD9291 has been shown to be effective against both EGFR tyrosine kinase inhibitor-sensitizing and T790M resistance mutations.  METHODS:  We administered AZD9291 at doses of 20 to 240 mg once daily in patients with advanced lung cancer who had radiologically documented disease progression after previous treatment with EGFR tyrosine kinase inhibitors.  The study included dose-escalation cohorts and dose-expansion cohorts.  In the expansion cohorts, prestudy tumor biopsies were required for central determination of EGFR T790M status.  Patients were assessed for safety, pharmacokinetics, and efficacy.  RESULTS:  A total of 253 patients were treated.  Among 31 patients enrolled in the dose-escalation cohorts, no dose-limiting toxic effects occurred at the doses evaluated.  An additional 222 patients were treated in five expansion cohorts.  The most common all-cause adverse events were diarrhea, rash, nausea, and decreased appetite.  The overall objective tumor response rate was 51% (95% confidence interval [CI], 45 to 58).  Among 127 patients with centrally confirmed EGFR T790M who could be evaluated for response, the response rate was 61% (95% CI, 52 to 70).  In contrast, among 61 patients without centrally detectable EGFR T790M who could be evaluated for response, the response rate was 21% (95% CI, 12 to 34).  The median progression-free survival was 9.6 months (95% CI, 8.3 to not reached) in EGFR T790M-positive patients and 2.8 months (95% CI, 2.1 to 4.3) in EGFR T790M-negative patients.  CONCLUSIONS:  AZD9291 was highly active in patients with lung cancer with the EGFR T790M mutation who had had disease progression during prior therapy with EGFR tyrosine kinase inhibitors. (Funded by AstraZeneca; ClinicalTrials.gov number, NCT01802632.).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQME11ovAjH3SnGiHuhFGv8fW6udTcc2eZYyW7TkdCDNrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mjpt1yltA%253D%253D&md5=7304f406bb2e568fb08b6d524d027389</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1411817&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1411817%26sid%3Dliteratum%253Aachs%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DYang%26aufirst%3DJ.%2BC.%26aulast%3DKim%26aufirst%3DD.%2BW.%26aulast%3DPlanchard%26aufirst%3DD.%26aulast%3DOhe%26aufirst%3DY.%26aulast%3DRamalingam%26aufirst%3DS.%2BS.%26aulast%3DAhn%26aufirst%3DM.%2BJ.%26aulast%3DKim%26aufirst%3DS.%2BW.%26aulast%3DSu%26aufirst%3DW.%2BC.%26aulast%3DHorn%26aufirst%3DL.%26aulast%3DHaggstrom%26aufirst%3DD.%26aulast%3DFelip%26aufirst%3DE.%26aulast%3DKim%26aufirst%3DJ.%2BH.%26aulast%3DFrewer%26aufirst%3DP.%26aulast%3DCantarini%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DK.%2BH.%26aulast%3DDickinson%26aufirst%3DP.%2BA.%26aulast%3DGhiorghiu%26aufirst%3DS.%26aulast%3DRanson%26aufirst%3DM.%26atitle%3DAZD9291%2520in%2520EGFR%2520inhibitor-resistant%2520non-small-cell%2520lung%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2015%26volume%3D372%26spage%3D1689%26epage%3D1699%26doi%3D10.1056%2FNEJMoa1411817" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Finlay, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderton, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashton, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bethel, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Box, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradbury, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butterworth, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chorley, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colclough, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cross, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Currie, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grist, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hassall, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kemmitt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klinowska, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamont, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamont, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McFarland, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mellor, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orme, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perkins, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perkins, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richmond, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waring, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whittaker, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wrigley, G. L.</span><span> </span><span class="NLM_article-title">Discovery of a potent and selective EGFR inhibitor (AZD9291) of both sensitizing and T790M resistance mutations that spares the wild-type form of the receptor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">8249</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.1021/jm500973a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500973a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8249-67&author=M.+R.+Finlayauthor=M.+Andertonauthor=S.+Ashtonauthor=P.+Ballardauthor=P.+A.+Bethelauthor=M.+R.+Boxauthor=R.+H.+Bradburyauthor=S.+J.+Brownauthor=S.+Butterworthauthor=A.+Campbellauthor=C.+Chorleyauthor=N.+Colcloughauthor=D.+A.+Crossauthor=G.+S.+Currieauthor=M.+Gristauthor=L.+Hassallauthor=G.+B.+Hillauthor=D.+Jamesauthor=M.+Jamesauthor=P.+Kemmittauthor=T.+Klinowskaauthor=G.+Lamontauthor=S.+G.+Lamontauthor=N.+Martinauthor=H.+L.+McFarlandauthor=M.+J.+Mellorauthor=J.+P.+Ormeauthor=D.+Perkinsauthor=P.+Perkinsauthor=G.+Richmondauthor=P.+Smithauthor=R.+A.+Wardauthor=M.+J.+Waringauthor=D.+Whittakerauthor=S.+Wellsauthor=G.+L.+Wrigley&title=Discovery+of+a+potent+and+selective+EGFR+inhibitor+%28AZD9291%29+of+both+sensitizing+and+T790M+resistance+mutations+that+spares+the+wild-type+form+of+the+receptor&doi=10.1021%2Fjm500973a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm500973a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500973a%26sid%3Dliteratum%253Aachs%26aulast%3DFinlay%26aufirst%3DM.%2BR.%26aulast%3DAnderton%26aufirst%3DM.%26aulast%3DAshton%26aufirst%3DS.%26aulast%3DBallard%26aufirst%3DP.%26aulast%3DBethel%26aufirst%3DP.%2BA.%26aulast%3DBox%26aufirst%3DM.%2BR.%26aulast%3DBradbury%26aufirst%3DR.%2BH.%26aulast%3DBrown%26aufirst%3DS.%2BJ.%26aulast%3DButterworth%26aufirst%3DS.%26aulast%3DCampbell%26aufirst%3DA.%26aulast%3DChorley%26aufirst%3DC.%26aulast%3DColclough%26aufirst%3DN.%26aulast%3DCross%26aufirst%3DD.%2BA.%26aulast%3DCurrie%26aufirst%3DG.%2BS.%26aulast%3DGrist%26aufirst%3DM.%26aulast%3DHassall%26aufirst%3DL.%26aulast%3DHill%26aufirst%3DG.%2BB.%26aulast%3DJames%26aufirst%3DD.%26aulast%3DJames%26aufirst%3DM.%26aulast%3DKemmitt%26aufirst%3DP.%26aulast%3DKlinowska%26aufirst%3DT.%26aulast%3DLamont%26aufirst%3DG.%26aulast%3DLamont%26aufirst%3DS.%2BG.%26aulast%3DMartin%26aufirst%3DN.%26aulast%3DMcFarland%26aufirst%3DH.%2BL.%26aulast%3DMellor%26aufirst%3DM.%2BJ.%26aulast%3DOrme%26aufirst%3DJ.%2BP.%26aulast%3DPerkins%26aufirst%3DD.%26aulast%3DPerkins%26aufirst%3DP.%26aulast%3DRichmond%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DP.%26aulast%3DWard%26aufirst%3DR.%2BA.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DWhittaker%26aufirst%3DD.%26aulast%3DWells%26aufirst%3DS.%26aulast%3DWrigley%26aufirst%3DG.%2BL.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520and%2520selective%2520EGFR%2520inhibitor%2520%2528AZD9291%2529%2520of%2520both%2520sensitizing%2520and%2520T790M%2520resistance%2520mutations%2520that%2520spares%2520the%2520wild-type%2520form%2520of%2520the%2520receptor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8249%26epage%3D67%26doi%3D10.1021%2Fjm500973a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Dietlein, F.; Heuckmann, J.; Choidas, A.; Basu, D.; Habenberger, P.; Fang, Z.; Ortiz-Cuaran, S.; Leenders, F.; Eickhoff, J.; Koch, U.; Getlik, M.; Termathe, M.; Sallouh, M.; Greff, Z.; Varga, Z.; Balke-Want, H.; Sos, M. L.; Peifer, M.; Reinhardt, H. C.; Örfi, L.; Kéri, G.; Ansen, S.; Heukamp, L. C.; Büttner, R.; Rauh, D.; Klebl, B.; Thomas, R. K.</span><span> </span><span class="NLM_article-title">Reconstructing structure-activity relationships of kinase inhibitors by automated chemopanning</span>. Submitted for publication.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Dietlein%2C+F.%3B+Heuckmann%2C+J.%3B+Choidas%2C+A.%3B+Basu%2C+D.%3B+Habenberger%2C+P.%3B+Fang%2C+Z.%3B+Ortiz-Cuaran%2C+S.%3B+Leenders%2C+F.%3B+Eickhoff%2C+J.%3B+Koch%2C+U.%3B+Getlik%2C+M.%3B+Termathe%2C+M.%3B+Sallouh%2C+M.%3B+Greff%2C+Z.%3B+Varga%2C+Z.%3B+Balke-Want%2C+H.%3B+Sos%2C+M.+L.%3B+Peifer%2C+M.%3B+Reinhardt%2C+H.+C.%3B+%C3%96rfi%2C+L.%3B+K%C3%A9ri%2C+G.%3B+Ansen%2C+S.%3B+Heukamp%2C+L.+C.%3B+B%C3%BCttner%2C+R.%3B+Rauh%2C+D.%3B+Klebl%2C+B.%3B+Thomas%2C+R.+K.+Reconstructing+structure-activity+relationships+of+kinase+inhibitors+by+automated+chemopanning.+Submitted+for+publication."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dunpublished%26sid%3Dliteratum%253Aachs%26aulast%3DDietlein%26aufirst%3DF.%26atitle%3DReconstructing%2520structure-activity%2520relationships%2520of%2520kinase%2520inhibitors%2520by%2520automated%2520chemopanning" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Blair, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rauh, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yun, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, Q. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rode, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eck, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span> </span><span class="NLM_article-title">Structure-guided development of affinity probes for tyrosine kinases using chemical genetics</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">229</span><span class="NLM_x">–</span> <span class="NLM_lpage">238</span><span class="refDoi"> DOI: 10.1038/nchembio866</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1038%2Fnchembio866" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=17334377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtFKnsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=229-238&author=J.+A.+Blairauthor=D.+Rauhauthor=C.+Kungauthor=C.+H.+Yunauthor=Q.+W.+Fanauthor=H.+Rodeauthor=C.+Zhangauthor=M.+J.+Eckauthor=W.+A.+Weissauthor=K.+M.+Shokat&title=Structure-guided+development+of+affinity+probes+for+tyrosine+kinases+using+chemical+genetics&doi=10.1038%2Fnchembio866"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-guided development of affinity probes for tyrosine kinases using chemical genetics</span></div><div class="casAuthors">Blair, Jimmy A.; Rauh, Daniel; Kung, Charles; Yun, Cai-Hong; Fan, Qi-Wen; Rode, Haridas; Zhang, Chao; Eck, Michael J.; Weiss, William A.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">229-238</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">As key components in nearly every signal transduction pathway, protein kinases are attractive targets for the regulation of cellular signaling by small-mol. inhibitors.  The authors report the structure-guided development of 6-acrylamido-4-anilinoquinazoline irreversible kinase inhibitors that potently and selectively target rationally designed kinases bearing two selectivity elements that are not found together in any wild-type kinase: an electrophile-targeted cysteine residue and a glycine gatekeeper residue.  Cocrystal structures of two irreversible quinazoline inhibitors bound to either epidermal growth factor receptor (EGFR) or engineered c-Src show covalent inhibitor binding to the targeted cysteine (Cys797 in EGFR and Cys345 in engineered c-Src).  Cocrystal structures of two irreversible quinazoline inhibitors (I, II) bound to either epidermal growth factor receptor (EGFR) or engineered c-Src show covalent inhibitor binding to the targeted cysteine (Cys797 in EGFR and Cys345 in engineered c-Src).  Based on these structures, the authors developed a fluorescent 6-acrylamido-4-anilinoquinazoline affinity probe (III) to report the fraction of kinase necessary for cellular signaling, and the authors used these reagents to quantitate the relationship between EGFR stimulation by EGF and its downstream outputs-Akt, Erk1 and Erk2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrG0KmYAm-skrVg90H21EOLACvtfcHk0ljBgzFxgZke5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtFKnsr0%253D&md5=ce4d24a8d65d7080645bf382cd65dbca</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnchembio866&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio866%26sid%3Dliteratum%253Aachs%26aulast%3DBlair%26aufirst%3DJ.%2BA.%26aulast%3DRauh%26aufirst%3DD.%26aulast%3DKung%26aufirst%3DC.%26aulast%3DYun%26aufirst%3DC.%2BH.%26aulast%3DFan%26aufirst%3DQ.%2BW.%26aulast%3DRode%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DWeiss%26aufirst%3DW.%2BA.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DStructure-guided%2520development%2520of%2520affinity%2520probes%2520for%2520tyrosine%2520kinases%2520using%2520chemical%2520genetics%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D3%26spage%3D229%26epage%3D238%26doi%3D10.1038%2Fnchembio866" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span> </span><span class="NLM_article-title">Residence time of receptor-ligand complexes and its effect on biological function</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">5481</span><span class="NLM_x">–</span> <span class="NLM_lpage">5492</span><span class="refDoi"> DOI: 10.1021/bi8002023</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi8002023" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1cXks1Ont7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=5481-5492&author=P.+J.+Tumminoauthor=R.+A.+Copeland&title=Residence+time+of+receptor-ligand+complexes+and+its+effect+on+biological+function&doi=10.1021%2Fbi8002023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Residence Time of Receptor-Ligand Complexes and Its Effect on Biological Function</span></div><div class="casAuthors">Tummino, Peter J.; Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5481-5492</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The formation and duration of binary receptor-ligand complexes are fundamental to many physiol. processes.  Most often, the effectiveness of interaction between a receptor and its ligand is quantified in terms of closed system, equil. affinity measurements, such as IC50 and Kd.  In the context of in vivo biol., however, the extent and duration of responses to receptor-ligand interactions depend greatly on the time period over which the ligand is in residence on its receptor.  Here we define receptor-ligand complex residence time in quant. terms and describe its significance to biol. function.  Examples of the importance of residence time are presented for natural ligands of different receptor types.  The impact of residence time on the optimization of potential ligands as drugs for human medicine is also described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHshN2Bt_r9LVg90H21EOLACvtfcHk0ljBgzFxgZke5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXks1Ont7s%253D&md5=57f79f69f12c6ed14e3347638381d943</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fbi8002023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi8002023%26sid%3Dliteratum%253Aachs%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DResidence%2520time%2520of%2520receptor-ligand%2520complexes%2520and%2520its%2520effect%2520on%2520biological%2520function%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D5481%26epage%3D5492%26doi%3D10.1021%2Fbi8002023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Barf, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaptein, A.</span><span> </span><span class="NLM_article-title">Irreversible protein kinase inhibitors: balancing the benefits and risks</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">6243</span><span class="NLM_x">–</span> <span class="NLM_lpage">6262</span><span class="refDoi"> DOI: 10.1021/jm3003203</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3003203" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsVerur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6243-6262&author=T.+Barfauthor=A.+Kaptein&title=Irreversible+protein+kinase+inhibitors%3A+balancing+the+benefits+and+risks&doi=10.1021%2Fjm3003203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Irreversible Protein Kinase Inhibitors: Balancing the Benefits and Risks</span></div><div class="casAuthors">Barf, Tjeerd; Kaptein, Allard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6243-6262</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Not applicable for a Perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4PLlisO0OVbVg90H21EOLACvtfcHk0ljBgzFxgZke5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsVerur8%253D&md5=5d87850857d37faab02e6037ef8140a4</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm3003203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3003203%26sid%3Dliteratum%253Aachs%26aulast%3DBarf%26aufirst%3DT.%26aulast%3DKaptein%26aufirst%3DA.%26atitle%3DIrreversible%2520protein%2520kinase%2520inhibitors%253A%2520balancing%2520the%2520benefits%2520and%2520risks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6243%26epage%3D6262%26doi%3D10.1021%2Fjm3003203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Ward, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderton, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashton, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bethel, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Box, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butterworth, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colclough, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chorley, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuaqui, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cross, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dakin, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Debreczeni, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberlein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finlay, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grist, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klinowska, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orme, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waring, M. J.</span><span> </span><span class="NLM_article-title">Structure- and reactivity-based development of covalent inhibitors of the activating and gatekeeper mutant forms of the epidermal growth factor receptor (EGFR)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">7025</span><span class="NLM_x">–</span> <span class="NLM_lpage">7048</span><span class="refDoi"> DOI: 10.1021/jm400822z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400822z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1CltL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7025-7048&author=R.+A.+Wardauthor=M.+J.+Andertonauthor=S.+Ashtonauthor=P.+A.+Bethelauthor=M.+Boxauthor=S.+Butterworthauthor=N.+Colcloughauthor=C.+G.+Chorleyauthor=C.+Chuaquiauthor=D.+A.+Crossauthor=L.+A.+Dakinauthor=J.+E.+Debreczeniauthor=C.+Eberleinauthor=M.+R.+Finlayauthor=G.+B.+Hillauthor=M.+Gristauthor=T.+C.+Klinowskaauthor=C.+Laneauthor=S.+Martinauthor=J.+P.+Ormeauthor=P.+Smithauthor=F.+Wangauthor=M.+J.+Waring&title=Structure-+and+reactivity-based+development+of+covalent+inhibitors+of+the+activating+and+gatekeeper+mutant+forms+of+the+epidermal+growth+factor+receptor+%28EGFR%29&doi=10.1021%2Fjm400822z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Structure- and Reactivity-Based Development of Covalent Inhibitors of the Activating and Gatekeeper Mutant Forms of the Epidermal Growth Factor Receptor (EGFR)</span></div><div class="casAuthors">Ward, Richard A.; Anderton, Mark J.; Ashton, Susan; Bethel, Paul A.; Box, Matthew; Butterworth, Sam; Colclough, Nicola; Chorley, Christopher G.; Chuaqui, Claudio; Cross, Darren A. E.; Dakin, Les A.; Debreczeni, Judit E.; Eberlein, Cath; Finlay, M. Raymond V.; Hill, George B.; Grist, Matthew; Klinowska, Teresa C. M.; Lane, Clare; Martin, Scott; Orme, Jonathon P.; Smith, Peter; Wang, Fengjiang; Waring, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7025-7048</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel series of small-mol. inhibitors has been developed to target the double mutant form of the epidermal growth factor receptor (EGFR) tyrosine kinase, which is resistant to treatment with gefitinib and erlotinib.  Our reported compds. also show selectivity over wild-type EGFR.  Guided by mol. modeling, this series was evolved to target a cysteine residue in the ATP binding site via covalent bond formation and demonstrates high levels of activity in cellular models of the double mutant form of EGFR.  In addn., these compds. show significant activity against the activating mutations, which gefitinib and erlotinib target and inhibition of which gives rise to their obsd. clin. efficacy.  A glutathione (GSH)-based assay was used to measure thiol reactivity toward the electrophilic functionality of the inhibitor series, enabling both the identification of a suitable reactivity window for their potency and the development of a reactivity quant. structure-property relationship (QSPR) to support design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKi6oNvtQPEbVg90H21EOLACvtfcHk0lie349QrCfHng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1CltL%252FN&md5=59c86415d71ac2abe5c6d4486e5d2517</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm400822z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400822z%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DR.%2BA.%26aulast%3DAnderton%26aufirst%3DM.%2BJ.%26aulast%3DAshton%26aufirst%3DS.%26aulast%3DBethel%26aufirst%3DP.%2BA.%26aulast%3DBox%26aufirst%3DM.%26aulast%3DButterworth%26aufirst%3DS.%26aulast%3DColclough%26aufirst%3DN.%26aulast%3DChorley%26aufirst%3DC.%2BG.%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DCross%26aufirst%3DD.%2BA.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DDebreczeni%26aufirst%3DJ.%2BE.%26aulast%3DEberlein%26aufirst%3DC.%26aulast%3DFinlay%26aufirst%3DM.%2BR.%26aulast%3DHill%26aufirst%3DG.%2BB.%26aulast%3DGrist%26aufirst%3DM.%26aulast%3DKlinowska%26aufirst%3DT.%2BC.%26aulast%3DLane%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DOrme%26aufirst%3DJ.%2BP.%26aulast%3DSmith%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26atitle%3DStructure-%2520and%2520reactivity-based%2520development%2520of%2520covalent%2520inhibitors%2520of%2520the%2520activating%2520and%2520gatekeeper%2520mutant%2520forms%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7025%26epage%3D7048%26doi%3D10.1021%2Fjm400822z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Zhou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ercan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jänne, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Discovery of selective irreversible inhibitors for EGFR-T790M</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">638</span><span class="NLM_x">–</span> <span class="NLM_lpage">643</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2010.12.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1016%2Fj.bmcl.2010.12.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=21208802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVKrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=638-643&author=W.+Zhouauthor=D.+Ercanauthor=P.+A.+J%C3%A4nneauthor=N.+S.+Gray&title=Discovery+of+selective+irreversible+inhibitors+for+EGFR-T790M&doi=10.1016%2Fj.bmcl.2010.12.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of selective irreversible inhibitors for EGFR-T790M</span></div><div class="casAuthors">Zhou, Wenjun; Ercan, Dalia; Jaenne, Pasi A.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">638-643</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Targeting the epidermal growth factor receptor kinase (EGFR) with ATP-competitive kinase inhibitors results in dramatic but short-lived responses in patients with EGFR mutant non small cell lung cancer.  A series of novel covalent EGFR kinase inhibitors with selectivity for the clin. relevant T790M gatekeeper' resistance mutation relative to wild-type EGFR were discovered by library screening.  A representative compd. 3i (I) was obtained through a systematic SAR study guided by mutant EGFR-dependent cellular proliferation assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB2Qq3f78-wLVg90H21EOLACvtfcHk0lie349QrCfHng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVKrtA%253D%253D&md5=c19f3f288ab99aaaab0a23bcf5273daf</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.12.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.12.036%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDiscovery%2520of%2520selective%2520irreversible%2520inhibitors%2520for%2520EGFR-T790M%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D638%26epage%3D643%26doi%3D10.1016%2Fj.bmcl.2010.12.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Wissner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mansour, T. S.</span><span> </span><span class="NLM_article-title">The development of HKI-272 and related compounds for the treatment of cancer</span> <span class="citation_source-journal">Arch. Pharm. (Weinheim, Ger.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">341</span><span class="NLM_x">, </span> <span class="NLM_fpage">465</span><span class="NLM_x">–</span> <span class="NLM_lpage">477</span><span class="refDoi"> DOI: 10.1002/ardp.200800009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1002%2Fardp.200800009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=18493974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVKnsb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2008&pages=465-477&author=A.+Wissnerauthor=T.+S.+Mansour&title=The+development+of+HKI-272+and+related+compounds+for+the+treatment+of+cancer&doi=10.1002%2Fardp.200800009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The development of HKI-272 and related compounds for the treatment of cancer</span></div><div class="casAuthors">Wissner, Allan; Mansour, Tarek S.</div><div class="citationInfo"><span class="NLM_cas:title">Archiv der Pharmazie (Weinheim, Germany)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">465-477</span>CODEN:
                <span class="NLM_cas:coden">ARPMAS</span>;
        ISSN:<span class="NLM_cas:issn">0365-6233</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The development of HKI-272 and EKB-569 for the treatment of cancer is described.  These compds. function as irreversible inhibitors of some members of the ErbB family of receptor tyrosine kinases.  In particular, they target epidermal growth factor receptor (EGFR, also known as ErbB-1) and human epidermal growth factor receptor-2 (HER2, also known as ErbB-2).  Both, HKI-272 and EKB-569 are 4-anilino-3-cyano quinoline derivs. that contain a 4-(dimethylamino)crotonamide Michael-acceptor group at the 6-position.  These compds. inhibit the function of the target enzymes by forming a covalent interaction with a conserved cysteine residue located in the kinase domains of these proteins.  The potential advantages of using irreversible inhibitors for this purpose are discussed.  We summarize the recent findings concerning some somatic mutations in EGFR and their relevance with respect to the irreversible inhibitors.  In particular, we highlight the findings that these irreversible inhibitors retain activity against tumors that have acquired a resistance to the reversible binding inhibitors gefitinib and erlotinib.  The promising interim clin. trial results for HKI-272 and EKB-569 in treating colon, lung, and breast cancers are summarized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCSokM42ZhprVg90H21EOLACvtfcHk0lie349QrCfHng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVKnsb7E&md5=2e46d99ca74e297e6a7e860634bf096d</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fardp.200800009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fardp.200800009%26sid%3Dliteratum%253Aachs%26aulast%3DWissner%26aufirst%3DA.%26aulast%3DMansour%26aufirst%3DT.%2BS.%26atitle%3DThe%2520development%2520of%2520HKI-272%2520and%2520related%2520compounds%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DArch.%2520Pharm.%2520%2528Weinheim%252C%2520Ger.%2529%26date%3D2008%26volume%3D341%26spage%3D465%26epage%3D477%26doi%3D10.1002%2Fardp.200800009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Krippendorff, B. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neuhaus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lienau, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reichel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huisinga, W.</span><span> </span><span class="NLM_article-title">Mechanism-based inhibition: deriving K(I) and k(inact) directly from time-dependent IC(50) values</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">913</span><span class="NLM_x">–</span> <span class="NLM_lpage">923</span><span class="refDoi"> DOI: 10.1177/1087057109336751</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1177%2F1087057109336751" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=19675314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlSjtLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=913-923&author=B.+F.+Krippendorffauthor=R.+Neuhausauthor=P.+Lienauauthor=A.+Reichelauthor=W.+Huisinga&title=Mechanism-based+inhibition%3A+deriving+K%28I%29+and+k%28inact%29+directly+from+time-dependent+IC%2850%29+values&doi=10.1177%2F1087057109336751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism-based inhibition: deriving KI and kinact directly from time-dependent IC50 values</span></div><div class="casAuthors">Krippendorff, Ben-Fillippo; Neuhaus, Roland; Lienau, Philip; Reichel, Andreas; Huisinga, Wilhelm</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">913-923</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The potential of enzyme inhibition of a drug is frequently quantified in terms of IC50 values.  Although this is a suitable quantity for reversible inhibitors, concerns arise when dealing with irreversible or mechanism-based inhibitors (MBIs).  IC50 values of MBIs are time dependent, causing serious problems when aiming at ranking different compds. with respect to their inhibitory potential.  As a consequence, most studies and ranking schemes related to MBIs rely on the inhibition const. (KI) and the rate of enzyme inactivation (kinact) rather than on IC50 values.  In this article, the authors derive a novel relation between potentially time-dependent IC50 values and KI, kinact parameters for different types of inhibition.  This allows for direct estn. of KI and kinact values from time-dependent IC50 values, even without the need of addnl. preincubation expts.  The application of this approach is illustrated using a fluorimetric assay to access the drug-drug interaction potential assocd. with new chem. entities.  The approach can easily be implemented using std. software tools (e.g., XLfit) and may also be suitable for applications where mechanism-based inhibition is a desired mode of action (e.g., at particular pharmacol. drug targets).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSOIpe0vN8JLVg90H21EOLACvtfcHk0limAQsAIEw_sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlSjtLjF&md5=93cb5f8644c7e1bdf872cf46756039cf</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1177%2F1087057109336751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057109336751%26sid%3Dliteratum%253Aachs%26aulast%3DKrippendorff%26aufirst%3DB.%2BF.%26aulast%3DNeuhaus%26aufirst%3DR.%26aulast%3DLienau%26aufirst%3DP.%26aulast%3DReichel%26aufirst%3DA.%26aulast%3DHuisinga%26aufirst%3DW.%26atitle%3DMechanism-based%2520inhibition%253A%2520deriving%2520K%2528I%2529%2520and%2520k%2528inact%2529%2520directly%2520from%2520time-dependent%2520IC%252850%2529%2520values%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26spage%3D913%26epage%3D923%26doi%3D10.1177%2F1087057109336751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Schwartz, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuzmic, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solowiej, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergqvist, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolanos, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almaden, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagata, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalvie, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kath, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, B. W.</span><span> </span><span class="NLM_article-title">Covalent EGFR inhibitor analysis reveals importance of reversible interactions to potency and mechanisms of drug resistance</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">173</span><span class="NLM_x">–</span> <span class="NLM_lpage">178</span><span class="refDoi"> DOI: 10.1073/pnas.1313733111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1073%2Fpnas.1313733111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=24347635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC2cXps1eluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=173-178&author=P.+A.+Schwartzauthor=P.+Kuzmicauthor=J.+Solowiejauthor=S.+Bergqvistauthor=B.+Bolanosauthor=C.+Almadenauthor=A.+Nagataauthor=K.+Ryanauthor=J.+Fengauthor=D.+Dalvieauthor=J.+C.+Kathauthor=M.+Xuauthor=R.+Waniauthor=B.+W.+Murray&title=Covalent+EGFR+inhibitor+analysis+reveals+importance+of+reversible+interactions+to+potency+and+mechanisms+of+drug+resistance&doi=10.1073%2Fpnas.1313733111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent EGFR inhibitor analysis reveals importance of reversible interactions to potency and mechanisms of drug resistance</span></div><div class="casAuthors">Schwartz, Phillip A.; Kuzmic, Petr; Solowiej, James; Bergqvist, Simon; Bolanos, Ben; Almaden, Chau; Nagata, Asako; Ryan, Kevin; Feng, Junli; Dalvie, Deepak; Kath, John C.; Xu, Meirong; Wani, Revati; Murray, Brion William</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">173-178</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Covalent inhibition is a reemerging paradigm in kinase drug design, but the roles of inhibitor binding affinity and chem. reactivity in overall potency are not well-understood.  To characterize the underlying mol. processes at a microscopic level and det. the appropriate kinetic consts., specialized exptl. design and advanced numerical integration of differential equations are developed.  Previously uncharacterized investigational covalent drugs reported here are shown to be extremely effective epidermal growth factor receptor (EGFR) inhibitors (kinact/Ki in the range 105-107 M-1s-1), despite their low specific reactivity (kinact ≤ 2.1 × 10-3 s-1), which is compensated for by high binding affinities (Ki < 1 nM).  For inhibitors relying on reactivity to achieve potency, noncovalent enzyme-inhibitor complex partitioning between inhibitor dissocn. and bond formation is central.  Interestingly, reversible binding affinity of EGFR covalent inhibitors is highly correlated with antitumor cell potency.  Furthermore, cellular potency for a subset of covalent inhibitors can be accounted for solely through reversible interactions.  One reversible interaction is between EGFR-Cys797 nucleophile and the inhibitor's reactive group, which may also contribute to drug resistance.  Because covalent inhibitors target a cysteine residue, the effects of its oxidn. on enzyme catalysis and inhibitor pharmacol. are characterized.  Oxidn. of the EGFR cysteine nucleophile does not alter catalysis but has widely varied effects on inhibitor potency depending on the EGFR context (e.g., oncogenic mutations), type of oxidn. (sulfinylation or glutathiolation), and inhibitor architecture.  These methods, parameters, and insights provide a rational framework for assessing and designing effective covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTpPuRq8MwCLVg90H21EOLACvtfcHk0limAQsAIEw_sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXps1eluw%253D%253D&md5=47197c5e7876a238eefb2e8e7de305ea</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1313733111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1313733111%26sid%3Dliteratum%253Aachs%26aulast%3DSchwartz%26aufirst%3DP.%2BA.%26aulast%3DKuzmic%26aufirst%3DP.%26aulast%3DSolowiej%26aufirst%3DJ.%26aulast%3DBergqvist%26aufirst%3DS.%26aulast%3DBolanos%26aufirst%3DB.%26aulast%3DAlmaden%26aufirst%3DC.%26aulast%3DNagata%26aufirst%3DA.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DDalvie%26aufirst%3DD.%26aulast%3DKath%26aufirst%3DJ.%2BC.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DWani%26aufirst%3DR.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26atitle%3DCovalent%2520EGFR%2520inhibitor%2520analysis%2520reveals%2520importance%2520of%2520reversible%2520interactions%2520to%2520potency%2520and%2520mechanisms%2520of%2520drug%2520resistance%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D173%26epage%3D178%26doi%3D10.1073%2Fpnas.1313733111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Kansy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senner, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gubernator, K.</span><span> </span><span class="NLM_article-title">Physicochemical high throughput screening: parallel artificial membrane permeation assay in the description of passive absorption processes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">1007</span><span class="NLM_x">–</span> <span class="NLM_lpage">1010</span><span class="refDoi"> DOI: 10.1021/jm970530e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm970530e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaK1cXhs12kt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=1007-1010&author=M.+Kansyauthor=F.+Sennerauthor=K.+Gubernator&title=Physicochemical+high+throughput+screening%3A+parallel+artificial+membrane+permeation+assay+in+the+description+of+passive+absorption+processes&doi=10.1021%2Fjm970530e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical High Throughput Screening: Parallel Artificial Membrane Permeation Assay in the Description of Passive Absorption Processes</span></div><div class="casAuthors">Kansy, Manfred; Senner, Frank; Gubernator, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1007-1010</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The majority of pharmaceutical discovery projects aim for an orally available form of a new therapeutic principle.  One obstacle for per oral treatment is intestinal absorption.  A simple predictive exptl. method to assess the potential of intestinal absorption is therefore highly welcome.  The authors describe here the PAMPA (Parallel Artificial Membrane Permeation Assay) method, a simple, robust, high throughput screen (HTS) which has been shown to be predictive for passive diffusion through membranes and thus for oral absorption.  In addn. PAMPA can deliver information in parallel on the lipophilicity, the ionization state and the soly. of a compd.  Therefore PAMPA complements HTS, combinatorial chem. and parallel synthesis efforts in preclin. pharmaceutical research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqcwCGodCr1rVg90H21EOLACvtfcHk0limAQsAIEw_sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhs12kt70%253D&md5=85d8ab409313b583de52348eb4d895da</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm970530e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm970530e%26sid%3Dliteratum%253Aachs%26aulast%3DKansy%26aufirst%3DM.%26aulast%3DSenner%26aufirst%3DF.%26aulast%3DGubernator%26aufirst%3DK.%26atitle%3DPhysicochemical%2520high%2520throughput%2520screening%253A%2520parallel%2520artificial%2520membrane%2520permeation%2520assay%2520in%2520the%2520description%2520of%2520passive%2520absorption%2520processes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D1007%26epage%3D1010%26doi%3D10.1021%2Fjm970530e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Shim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackerell, A. D.,  Jr.</span><span> </span><span class="NLM_article-title">Computational ligand-based rational design: Role of conformational sampling and force fields in model development</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">356</span><span class="NLM_x">–</span> <span class="NLM_lpage">370</span><span class="refDoi"> DOI: 10.1039/c1md00044f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1039%2Fc1md00044f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=21716805" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3MXls1Kht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=356-370&author=J.+Shimauthor=A.+D.+Mackerell&title=Computational+ligand-based+rational+design%3A+Role+of+conformational+sampling+and+force+fields+in+model+development&doi=10.1039%2Fc1md00044f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Computational ligand-based rational design: role of conformational sampling and force fields in model development</span></div><div class="casAuthors">Shim, Jihyun; MacKerell, Alexander D., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">356-370</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  A significant no. of drug discovery efforts are based on natural products or high throughput screens from which compds. showing potential therapeutic effects are identified without knowledge of the target mol. or its 3D structure.  In such cases computational ligand-based drug design (LBDD) can accelerate the drug discovery processes.  LBDD is a general approach to elucidate the relationship of a compd.'s structure and physicochem. attributes to its biol. activity.  The resulting structure-activity relationship (SAR) may then act as the basis for the prediction of compds. with improved biol. attributes.  LBDD methods range from pharmacophore models identifying essential features of ligands responsible for their activity, quant. structure-activity relationships (QSAR) yielding quant. ests. of activities based on physiochem. properties, and to similarity searching, which explores compds. with similar properties as well as various combinations of the above.  A no. of recent LBDD approaches involve the use of multiple conformations of the ligands being studied.  One of the basic components to generate multiple conformations in LBDD is mol. mechanics (MM), which apply an empirical energy function to relate conformation to energies and forces.  The collection of conformations for ligands is then combined with functional data using methods ranging from regression anal. to neural networks, from which the SAR is detd.  Accordingly, for effective application of LBDD for SAR detns. it is important that the compds. be accurately modelled such that the appropriate range of conformations accessible to the ligands is identified.  Such accurate modeling is largely based on use of the appropriate empirical force field for the mols. being investigated and the approaches used to generate the conformations.  The present chapter includes a brief overview of currently used SAR methods in LBDD followed by a more detailed presentation of issues and limitations assocd. with empirical energy functions and conformational sampling methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkNp46K1NKk7Vg90H21EOLACvtfcHk0lhM9dPck9vnTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXls1Kht7c%253D&md5=70d822d1ca2faa484f723fb109597b77</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1039%2Fc1md00044f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1md00044f%26sid%3Dliteratum%253Aachs%26aulast%3DShim%26aufirst%3DJ.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26atitle%3DComputational%2520ligand-based%2520rational%2520design%253A%2520Role%2520of%2520conformational%2520sampling%2520and%2520force%2520fields%2520in%2520model%2520development%26jtitle%3DMedChemComm%26date%3D2011%26volume%3D2%26spage%3D356%26epage%3D370%26doi%3D10.1039%2Fc1md00044f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Richards, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brant, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boulanger, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cairo, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wulff, J. E.</span><span> </span><span class="NLM_article-title">Conformational analysis of peramivir reveals critical differences between free and enzyme-bound states</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1483</span><span class="NLM_x">–</span> <span class="NLM_lpage">1488</span><span class="refDoi"> DOI: 10.1039/C4MD00168K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1039%2FC4MD00168K" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1483-1488&author=M.+R.+Richardsauthor=M.+G.+Brantauthor=M.+J.+Boulangerauthor=C.+W.+Cairoauthor=J.+E.+Wulff&title=Conformational+analysis+of+peramivir+reveals+critical+differences+between+free+and+enzyme-bound+states&doi=10.1039%2FC4MD00168K"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1039%2FC4MD00168K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00168K%26sid%3Dliteratum%253Aachs%26aulast%3DRichards%26aufirst%3DM.%2BR.%26aulast%3DBrant%26aufirst%3DM.%2BG.%26aulast%3DBoulanger%26aufirst%3DM.%2BJ.%26aulast%3DCairo%26aufirst%3DC.%2BW.%26aulast%3DWulff%26aufirst%3DJ.%2BE.%26atitle%3DConformational%2520analysis%2520of%2520peramivir%2520reveals%2520critical%2520differences%2520between%2520free%2520and%2520enzyme-bound%2520states%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D1483%26epage%3D1488%26doi%3D10.1039%2FC4MD00168K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Forti, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavasotto, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orozco, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barril, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luque, F. J.</span><span> </span><span class="NLM_article-title">A Multilevel Strategy for the Exploration of the Conformational Flexibility of Small Molecules</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1808</span><span class="NLM_x">–</span> <span class="NLM_lpage">1819</span><span class="refDoi"> DOI: 10.1021/ct300097s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300097s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVKmtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=1808-1819&author=F.+Fortiauthor=C.+N.+Cavasottoauthor=M.+Orozcoauthor=X.+Barrilauthor=F.+J.+Luque&title=A+Multilevel+Strategy+for+the+Exploration+of+the+Conformational+Flexibility+of+Small+Molecules&doi=10.1021%2Fct300097s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">A Multilevel Strategy for the Exploration of the Conformational Flexibility of Small Molecules</span></div><div class="casAuthors">Forti, Flavio; Cavasotto, Claudio N.; Orozco, Modesto; Barril, Xavier; Luque, F. Javier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1808-1819</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Predicting the conformational preferences of flexible compds. is still a challenging problem with important implications in areas such as mol. recognition and drug design.  In this work, we describe a multilevel strategy to explore the conformational preferences of mols.  The method relies on the predominant-state approxn., which partitions the conformational space into distinct conformational wells.  Moreover, it combines low-level (LL) methods for sampling the conformational min. and high-level (HL) techniques for calibrating their relative stability.  In the implementation used in this study, the LL sampling is performed with the semiempirical RM1 Hamiltonian, and solvent effects are included using the RM1 version of the Miertus, Scrocco, and Tomasi (MST) continuum solvation model.  The HL refinement of the conformational wells is performed by combining geometry optimizations of the min. at the B3LYP (gas phase) or MST-B3LYP (soln.) level, followed by single point MP2 computations using Dunning's augmented basis sets.  Then, the effective free energy of a conformational well is estd. by combining the MP2 energy, supplemented with the MST-B3LYP solvation free energy for a conformational search in soln., with the local curvature of the well sampled at the semiempirical level.  Applications of this strategy involve the exploration of the conformational preferences of 1,2-dichloroethane and neutral histamine in both the gas phase and water soln.  Finally, the multilevel strategy is used to est. the reorganization cost required for selecting the bioactive conformation of HIV reverse transcriptase inhibitors, which is estd. to be at most 1.3 kcal/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFC-5jHPspaLVg90H21EOLACvtfcHk0lhM9dPck9vnTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVKmtLY%253D&md5=35eb55f0805c4c929d9706230a9b41b1</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fct300097s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300097s%26sid%3Dliteratum%253Aachs%26aulast%3DForti%26aufirst%3DF.%26aulast%3DCavasotto%26aufirst%3DC.%2BN.%26aulast%3DOrozco%26aufirst%3DM.%26aulast%3DBarril%26aufirst%3DX.%26aulast%3DLuque%26aufirst%3DF.%2BJ.%26atitle%3DA%2520Multilevel%2520Strategy%2520for%2520the%2520Exploration%2520of%2520the%2520Conformational%2520Flexibility%2520of%2520Small%2520Molecules%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D1808%26epage%3D1819%26doi%3D10.1021%2Fct300097s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Development and testing of a general amber force field</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1157</span><span class="NLM_x">–</span> <span class="NLM_lpage">1174</span><span class="refDoi"> DOI: 10.1002/jcc.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+a+general+amber+force+field&doi=10.1002%2Fjcc.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0lijuxIgWPZ-xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520a%2520general%2520amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174%26doi%3D10.1002%2Fjcc.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Automatic atom type and bond type perception in molecular mechanical calculations</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">247</span><span class="NLM_x">–</span> <span class="NLM_lpage">260</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2005.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1016%2Fj.jmgm.2005.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=16458552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=247-260&author=J.+Wangauthor=W.+Wangauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Automatic+atom+type+and+bond+type+perception+in+molecular+mechanical+calculations&doi=10.1016%2Fj.jmgm.2005.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic atom type and bond type perception in molecular mechanical calculations</span></div><div class="casAuthors">Wang, Junmei; Wang, Wei; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-260</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In mol. mechanics (MM) studies, atom types and/or bond types of mols. are needed to det. prior to energy calcns.  The authors present here an automatic algorithm of perceiving atom types that are defined in a description table, and an automatic algorithm of assigning bond types just based on at. connectivity.  The algorithms have been implemented in a new module of the AMBER packages.  This auxiliary module, antechamber (roughly meaning "before AMBER"), can be applied to generate necessary inputs of leap-the AMBER program to generate topologies for minimization, mol. dynamics, etc., for most org. mols.  The algorithms behind the manipulations may be useful for other mol. mech. packages as well as applications that need to designate atom types and bond types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqizj6F18nBYbVg90H21EOLACvtfcHk0lijuxIgWPZ-xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D&md5=8031a21d2784d5dea12e70868522aa61</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2005.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2005.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DAutomatic%2520atom%2520type%2520and%2520bond%2520type%2520perception%2520in%2520molecular%2520mechanical%2520calculations%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2006%26volume%3D25%26spage%3D247%26epage%3D260%26doi%3D10.1016%2Fj.jmgm.2005.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dumont, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span> </span><span class="NLM_article-title">Comparison of charge models for fixed-charge force fields: small-molecule hydration free energies in explicit solvent</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">2242</span><span class="NLM_x">–</span> <span class="NLM_lpage">2254</span><span class="refDoi"> DOI: 10.1021/jp0667442</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp0667442" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsFWmsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2007&pages=2242-2254&author=D.+L.+Mobleyauthor=E.+Dumontauthor=J.+D.+Choderaauthor=K.+A.+Dill&title=Comparison+of+charge+models+for+fixed-charge+force+fields%3A+small-molecule+hydration+free+energies+in+explicit+solvent&doi=10.1021%2Fjp0667442"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Charge Models for Fixed-Charge Force Fields: Small-Molecule Hydration Free Energies in Explicit Solvent</span></div><div class="casAuthors">Mobley, David L.; Dumont, Elise; Chodera, John D.; Dill, Ken A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2242-2254</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In mol. simulations with fixed-charge force fields, the choice of partial at. charges influences numerous computed phys. properties, including binding free energies.  Many mol. mechanics force fields specify how nonbonded parameters should be detd., but various choices are often available for how these charges are to be detd. for arbitrary small mols.  Here, we compute hydration free energies for a set of 44 small, neutral mols. in two different explicit water models (TIP3P and TIP4P-Ew) to examine the influence of charge model on agreement with expt.  Using the AMBER GAFF force field for nonbonded parameters, we test several different methods for obtaining partial at. charges, including two fast methods exploiting semiempirical quantum calcns. and methods deriving charges from the electrostatic potentials computed with several different levels of ab initio quantum calcns. with and without a continuum reaction field treatment of solvent.  We find that the best charge sets give a root-mean-square error from expt. of roughly 1 kcal/mol.  Surprisingly, agreement with exptl. hydration free energies does not increase substantially with increasing level of quantum theory, even when the quantum calcns. are performed with a reaction field treatment to better model the aq. phase.  We also find that the semiempirical AM1-BCC method for computing charges works almost as well as any of the more computationally expensive ab initio methods and that the root-mean-square error reported here is similar to that for implicit solvent models reported in the literature.  Further, we find that the discrepancy with exptl. hydration free energies grows substantially with the polarity of the compd., as does its variation across theory levels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp00p5CsvXGx7Vg90H21EOLACvtfcHk0lijuxIgWPZ-xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsFWmsrc%253D&md5=5710fc0d95bead57202b33afa2093431</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjp0667442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp0667442%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DDumont%26aufirst%3DE.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DComparison%2520of%2520charge%2520models%2520for%2520fixed-charge%2520force%2520fields%253A%2520small-molecule%2520hydration%2520free%2520energies%2520in%2520explicit%2520solvent%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2007%26volume%3D111%26spage%3D2242%26epage%3D2254%26doi%3D10.1021%2Fjp0667442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Yilmazer, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korth, M.</span><span> </span><span class="NLM_article-title">Comparison of molecular mechanics, semi-empirical quantum mechanical, and density functional theory methods for scoring protein-ligand interactions</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">117</span><span class="NLM_x">, </span> <span class="NLM_fpage">8075</span><span class="NLM_x">–</span> <span class="NLM_lpage">8084</span><span class="refDoi"> DOI: 10.1021/jp402719k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp402719k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptlChur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2013&pages=8075-8084&author=N.+D.+Yilmazerauthor=M.+Korth&title=Comparison+of+molecular+mechanics%2C+semi-empirical+quantum+mechanical%2C+and+density+functional+theory+methods+for+scoring+protein-ligand+interactions&doi=10.1021%2Fjp402719k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Molecular Mechanics, Semi-Empirical Quantum Mechanical, and Density Functional Theory Methods for Scoring Protein-Ligand Interactions</span></div><div class="casAuthors">Yilmazer, Nusret Duygu; Korth, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">8075-8084</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Correctly ranking protein-ligand interactions with respect to overall free energy of binding is a grand challenge for virtual drug design.  Here the authors compare the performance of various quantum chem. approaches for tackling this so-called "scoring" problem.  Relying on systematically generated benchmark sets of large protein/ligand model complexes based on the PDBbind database, the performance depends first of all on the general level of theory.  Comparing classical mol. mechanics (MM), semiempirical quantum mech. (SQM), and d. functional theory (DFT) based methods, enhanced SQM approaches perform very similar to DFT methods and substantially different from MM potentials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5xB3KEWdidrVg90H21EOLACvtfcHk0lgtqOXW9q575Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptlChur8%253D&md5=c9cc8ad508d7f791f4f153a3e73d32b1</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjp402719k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp402719k%26sid%3Dliteratum%253Aachs%26aulast%3DYilmazer%26aufirst%3DN.%2BD.%26aulast%3DKorth%26aufirst%3DM.%26atitle%3DComparison%2520of%2520molecular%2520mechanics%252C%2520semi-empirical%2520quantum%2520mechanical%252C%2520and%2520density%2520functional%2520theory%2520methods%2520for%2520scoring%2520protein-ligand%2520interactions%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2013%26volume%3D117%26spage%3D8075%26epage%3D8084%26doi%3D10.1021%2Fjp402719k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Tomasi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mennucci, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cammi, R.</span><span> </span><span class="NLM_article-title">Quantum mechanical continuum solvation models</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">2999</span><span class="NLM_x">–</span> <span class="NLM_lpage">3093</span><span class="refDoi"> DOI: 10.1021/cr9904009</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr9904009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmsVynurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2005&pages=2999-3093&author=J.+Tomasiauthor=B.+Mennucciauthor=R.+Cammi&title=Quantum+mechanical+continuum+solvation+models&doi=10.1021%2Fcr9904009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Quantum Mechanical Continuum Solvation Models</span></div><div class="casAuthors">Tomasi, Jacopo; Mennucci, Benedetta; Cammi, Roberto</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2999-3093</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkWabJEwaQ7LVg90H21EOLACvtfcHk0lgtqOXW9q575Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmsVynurc%253D&md5=462420dd18b3006ee63d1298b66db247</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fcr9904009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr9904009%26sid%3Dliteratum%253Aachs%26aulast%3DTomasi%26aufirst%3DJ.%26aulast%3DMennucci%26aufirst%3DB.%26aulast%3DCammi%26aufirst%3DR.%26atitle%3DQuantum%2520mechanical%2520continuum%2520solvation%2520models%26jtitle%3DChem.%2520Rev.%26date%3D2005%26volume%3D105%26spage%3D2999%26epage%3D3093%26doi%3D10.1021%2Fcr9904009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Kloss, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heil, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kast, S. M.</span><span> </span><span class="NLM_article-title">Quantum chemistry in solution by combining 3D integral equation theory with a cluster embedding approach</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">4337</span><span class="NLM_x">–</span> <span class="NLM_lpage">4343</span><span class="refDoi"> DOI: 10.1021/jp710680m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp710680m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtlOjtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2008&pages=4337-4343&author=T.+Klossauthor=J.+Heilauthor=S.+M.+Kast&title=Quantum+chemistry+in+solution+by+combining+3D+integral+equation+theory+with+a+cluster+embedding+approach&doi=10.1021%2Fjp710680m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Quantum Chemistry in Solution by Combining 3D Integral Equation Theory with a Cluster Embedding Approach</span></div><div class="casAuthors">Kloss, Thomas; Heil, Jochen; Kast, Stefan M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4337-4343</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Free energy changes assocd. with chem. reactions in soln. are treated by integral equation theory in the form of the 3D ref. interaction site model (RISM) in combination with quantum-chem. calcns. via an embedded cluster approach (EC-RISM).  The electronic structure of the solute is computed self-consistently with the solvent structure by mapping the charge distribution of the solvent onto a set of discrete background point charges that are added to the mol. Hamiltonian.  The EC-RISM procedure yields chem. accuracy in free energy predictions for several benchmark systems without adjusting empirical parameters.  We apply the method to the std. reaction free energy for the gauche-trans equil. of 1,2-dichloroethane in water and to pKa shift calcns. for trifluoroacetic acid/acetic acid and 4-nitroaniline/aniline in water.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYZZ5wmyxtZbVg90H21EOLACvtfcHk0lgtqOXW9q575Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtlOjtLg%253D&md5=3370f057079bd764b0aa70adb889a0d0</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjp710680m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp710680m%26sid%3Dliteratum%253Aachs%26aulast%3DKloss%26aufirst%3DT.%26aulast%3DHeil%26aufirst%3DJ.%26aulast%3DKast%26aufirst%3DS.%2BM.%26atitle%3DQuantum%2520chemistry%2520in%2520solution%2520by%2520combining%25203D%2520integral%2520equation%2520theory%2520with%2520a%2520cluster%2520embedding%2520approach%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2008%26volume%3D112%26spage%3D4337%26epage%3D4343%26doi%3D10.1021%2Fjp710680m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Kast, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heil, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gussregen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, K. F.</span><span> </span><span class="NLM_article-title">Prediction of tautomer ratios by embedded-cluster integral equation theory</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">353</span><span class="refDoi"> DOI: 10.1007/s10822-010-9340-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1007%2Fs10822-010-9340-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=20352296" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsFSmsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=343-353&author=S.+M.+Kastauthor=J.+Heilauthor=S.+Gussregenauthor=K.+F.+Schmidt&title=Prediction+of+tautomer+ratios+by+embedded-cluster+integral+equation+theory&doi=10.1007%2Fs10822-010-9340-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of tautomer ratios by embedded-cluster integral equation theory</span></div><div class="casAuthors">Kast, Stefan M.; Heil, Jochen; Guessregen, Stefan; Schmidt, K. Friedemann</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">343-353</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The "embedded cluster ref. interaction site model" (EC-RISM) approach combines statistical-mech. integral equation theory and quantum-chem. calcns. for predicting thermodn. data for chem. reactions in soln.  The electronic structure of the solute is detd. self-consistently with the structure of the solvent that is described by 3D RISM integral equation theory.  The continuous solvent-site distribution is mapped onto a set of discrete background charges ("embedded cluster") that represent an addnl. contribution to the mol. Hamiltonian.  The EC-RISM anal. of the SAMPL2 challenge set of tautomers proceeds in three stages.  Firstly, the group of compds. for which quant. exptl. free energy data was provided was taken to det. appropriate levels of quantum-chem. theory for geometry optimization and free energy prediction.  Secondly, the resulting workflow was applied to the full set, allowing for chem. interpretations of the results.  Thirdly, disclosure of exptl. data for parts of the compds. facilitated a detailed anal. of methodical issues and suggestions for future improvements of the model.  Without specifically adjusting parameters, the EC-RISM model yields the smallest value of the root mean square error for the first set (0.6 kcal mol-1) as well as for the full set of quant. reaction data (2.0 kcal mol-1) among the SAMPL2 participants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZaDnWt2LKubVg90H21EOLACvtfcHk0lg--CYeFkOgrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsFSmsLk%253D&md5=5ef934a4e8128d7abed3d4fc7db1bba3</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1007%2Fs10822-010-9340-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-010-9340-x%26sid%3Dliteratum%253Aachs%26aulast%3DKast%26aufirst%3DS.%2BM.%26aulast%3DHeil%26aufirst%3DJ.%26aulast%3DGussregen%26aufirst%3DS.%26aulast%3DSchmidt%26aufirst%3DK.%2BF.%26atitle%3DPrediction%2520of%2520tautomer%2520ratios%2520by%2520embedded-cluster%2520integral%2520equation%2520theory%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2010%26volume%3D24%26spage%3D343%26epage%3D353%26doi%3D10.1007%2Fs10822-010-9340-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Frach, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kast, S. M.</span><span> </span><span class="NLM_article-title">Solvation effects on chemical shifts by embedded cluster integral equation theory</span> <span class="citation_source-journal">J. Phys. Chem. A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">118</span><span class="NLM_x">, </span> <span class="NLM_fpage">11620</span><span class="NLM_x">–</span> <span class="NLM_lpage">11628</span><span class="refDoi"> DOI: 10.1021/jp5084407</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp5084407" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVyhurjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2014&pages=11620-11628&author=R.+Frachauthor=S.+M.+Kast&title=Solvation+effects+on+chemical+shifts+by+embedded+cluster+integral+equation+theory&doi=10.1021%2Fjp5084407"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Solvation Effects on Chemical Shifts by Embedded Cluster Integral Equation Theory</span></div><div class="casAuthors">Frach, Roland; Kast, Stefan M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry A</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">11620-11628</span>CODEN:
                <span class="NLM_cas:coden">JPCAFH</span>;
        ISSN:<span class="NLM_cas:issn">1089-5639</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The accurate computational prediction of NMR parameters like chem. shifts represents a challenge if the species studied is immersed in strongly polarizing environments such as water.  Common approaches to treating a solvent in the form of, e.g., the polarizable continuum model (PCM) ignore strong directional interactions such as H-bonds to the solvent which can have substantial impact on magnetic shieldings.  We here present a computational methodol. that accounts for at.-level solvent effects on NMR parameters by extending the embedded cluster ref. interaction site model (EC-RISM) integral equation theory to the prediction of chem. shifts of N-methylacetamide (NMA) in aq. soln.  We examine the influence of various so-called closure approxns. of the underlying three-dimensional RISM theory as well as the impact of basis set size and different treatment of electrostatic solute-solvent interactions.  We find considerable and systematic improvement over ref. PCM and gas phase calcns.  A smaller basis set in combination with a simple point charge model already yields good performance which can be further improved by employing exact electrostatic quantum-mech. solute-solvent interaction energies.  A larger basis set benefits more significantly from exact over point charge electrostatics, which can be related to differences of the solvent's charge distribution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoh_Whnopy6mLVg90H21EOLACvtfcHk0lg--CYeFkOgrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVyhurjP&md5=ffddfe146c4663c6f1b75f6f3702899b</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjp5084407&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp5084407%26sid%3Dliteratum%253Aachs%26aulast%3DFrach%26aufirst%3DR.%26aulast%3DKast%26aufirst%3DS.%2BM.%26atitle%3DSolvation%2520effects%2520on%2520chemical%2520shifts%2520by%2520embedded%2520cluster%2520integral%2520equation%2520theory%26jtitle%3DJ.%2520Phys.%2520Chem.%2520A%26date%3D2014%26volume%3D118%26spage%3D11620%26epage%3D11628%26doi%3D10.1021%2Fjp5084407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Cortes, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talpaz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kantarjian, H.</span><span> </span><span class="NLM_article-title">Ponatinib in Philadelphia chromosome-positive leukemias</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">370</span><span class="NLM_x">, </span> <span class="NLM_fpage">577</span><span class="refDoi"> DOI: 10.1056/NEJMc1315234</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1056%2FNEJMc1315234" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=24499221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFaktbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=370&publication_year=2014&pages=577&author=J.+E.+Cortesauthor=M.+Talpazauthor=H.+Kantarjian&title=Ponatinib+in+Philadelphia+chromosome-positive+leukemias&doi=10.1056%2FNEJMc1315234"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">The authors reply to comment on: Ponatinib in Philadelphia chromosome-positive leukemias</span></div><div class="casAuthors">Cortes, Jorge E.; Talpaz, Moshe; Kantarjian, Hagop</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">370</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">577</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlfmtGglm6RrVg90H21EOLACvtfcHk0lg--CYeFkOgrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFaktbk%253D&md5=31aed97c18eac02899b013a25b9fef3b</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1056%2FNEJMc1315234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMc1315234%26sid%3Dliteratum%253Aachs%26aulast%3DCortes%26aufirst%3DJ.%2BE.%26aulast%3DTalpaz%26aufirst%3DM.%26aulast%3DKantarjian%26aufirst%3DH.%26atitle%3DPonatinib%2520in%2520Philadelphia%2520chromosome-positive%2520leukemias%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2014%26volume%3D370%26spage%3D577%26doi%3D10.1056%2FNEJMc1315234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Pemovska, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kontro, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cronin, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McTigue, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kallioniemi, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porkka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wennerberg, K.</span><span> </span><span class="NLM_article-title">Axitinib effectively inhibits BCR-ABL1(T315I) with a distinct binding conformation</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">519</span><span class="NLM_x">, </span> <span class="NLM_fpage">102</span><span class="NLM_x">–</span> <span class="NLM_lpage">105</span><span class="refDoi"> DOI: 10.1038/nature14119</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1038%2Fnature14119" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=25686603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjt1Sntbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=519&publication_year=2015&pages=102-105&author=T.+Pemovskaauthor=E.+Johnsonauthor=M.+Kontroauthor=G.+A.+Repaskyauthor=J.+Chenauthor=P.+Wellsauthor=C.+N.+Croninauthor=M.+McTigueauthor=O.+Kallioniemiauthor=K.+Porkkaauthor=B.+W.+Murrayauthor=K.+Wennerberg&title=Axitinib+effectively+inhibits+BCR-ABL1%28T315I%29+with+a+distinct+binding+conformation&doi=10.1038%2Fnature14119"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Axitinib effectively inhibits BCR-ABL1(T315I) with a distinct binding conformation</span></div><div class="casAuthors">Pemovska, Tea; Johnson, Eric; Kontro, Mika; Repasky, Gretchen A.; Chen, Jeffrey; Wells, Peter; Cronin, Ciaran N.; McTigue, Michele; Kallioniemi, Olli; Porkka, Kimmo; Murray, Brion W.; Wennerberg, Krister</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">519</span>
        (<span class="NLM_cas:issue">7541</span>),
    <span class="NLM_cas:pages">102-105</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The BCR-ABL1 fusion gene is a driver oncogene in chronic myeloid leukemia and 30-50% of cases of adult acute lymphoblastic leukemia.  Introduction of ABL1 kinase inhibitors (for example, imatinib) has markedly improved patient survival, but acquired drug resistance remains a challenge.  Point mutations in the ABL1 kinase domain weaken inhibitor binding and represent the most common clin. resistance mechanism.  The BCR-ABL1 kinase domain gatekeeper mutation Thr315Ile (T315I) confers resistance to all approved ABL1 inhibitors except ponatinib, which has toxicity limitations.  Here we combine comprehensive drug sensitivity and resistance profiling of patient cells ex vivo with structural anal. to establish the VEGFR tyrosine kinase inhibitor axitinib as a selective and effective inhibitor for T315I-mutant BCR-ABL1-driven leukemia.  Axitinib potently inhibited BCR-ABL1(T315I), at both biochem. and cellular levels, by binding to the active form of ABL1(T315I) in a mutation-selective binding mode.  These findings suggest that the T315I mutation shifts the conformational equil. of the kinase in favor of an active (DFG-in) A-loop conformation, which has more optimal binding interactions with axitinib.  Treatment of a T315I chronic myeloid leukemia patient with axitinib resulted in a rapid redn. of T315I-pos. cells from bone marrow.  Taken together, our findings demonstrate an unexpected opportunity to repurpose axitinib, an anti-angiogenic drug approved for renal cancer, as an inhibitor for ABL1 gatekeeper mutant drug-resistant leukemia patients.  This study shows that wild-type proteins do not always sample the conformations available to disease-relevant mutant proteins and that comprehensive drug testing of patient-derived cells can identify unpredictable, clin. significant drug-repositioning opportunities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreXgtLS5A1lLVg90H21EOLACvtfcHk0lhMwqfc9C8fVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjt1Sntbs%253D&md5=af8549ae0527bfcbda6d6e05b548f928</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2Fnature14119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14119%26sid%3Dliteratum%253Aachs%26aulast%3DPemovska%26aufirst%3DT.%26aulast%3DJohnson%26aufirst%3DE.%26aulast%3DKontro%26aufirst%3DM.%26aulast%3DRepasky%26aufirst%3DG.%2BA.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWells%26aufirst%3DP.%26aulast%3DCronin%26aufirst%3DC.%2BN.%26aulast%3DMcTigue%26aufirst%3DM.%26aulast%3DKallioniemi%26aufirst%3DO.%26aulast%3DPorkka%26aufirst%3DK.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26aulast%3DWennerberg%26aufirst%3DK.%26atitle%3DAxitinib%2520effectively%2520inhibits%2520BCR-ABL1%2528T315I%2529%2520with%2520a%2520distinct%2520binding%2520conformation%26jtitle%3DNature%26date%3D2015%26volume%3D519%26spage%3D102%26epage%3D105%26doi%3D10.1038%2Fnature14119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Leproult, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barluenga, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moras, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wurtz, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winssinger, N.</span><span> </span><span class="NLM_article-title">Cysteine mapping in conformationally distinct kinase nucleotide binding sites: application to the design of selective covalent inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1347</span><span class="NLM_x">–</span> <span class="NLM_lpage">1355</span><span class="refDoi"> DOI: 10.1021/jm101396q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101396q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFeis70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1347-1355&author=E.+Leproultauthor=S.+Barluengaauthor=D.+Morasauthor=J.+M.+Wurtzauthor=N.+Winssinger&title=Cysteine+mapping+in+conformationally+distinct+kinase+nucleotide+binding+sites%3A+application+to+the+design+of+selective+covalent+inhibitors&doi=10.1021%2Fjm101396q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Cysteine mapping in conformationally distinct kinase nucleotide binding sites: application to the design of selective covalent inhibitors</span></div><div class="casAuthors">Leproult, Emeline; Barluenga, Sofia; Moras, Dino; Wurtz, Jean-Marie; Winssinger, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1347-1355</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Kinases have emerged as one of the most prolific therapeutic targets.  An important criterion in the therapeutic success of inhibitors targeting the nucleotide binding pocket of kinases is the inhibitor residence time.  Recently, covalent kinase inhibitors have attracted attention since they confer terminal inhibition and should thus be more effective than reversible inhibitors with transient inhibition.  The most robust approach to design irreversible inhibitors is to capitalize on the nucleophilicity of a cysteine thiol group present in the target protein.  Herein, the authors report a systematic anal. of cysteine residues present in the nucleotide binding site of kinases, which could be harnessed for irreversible inhibition, taking into consideration the different kinase conformations.  The authors demonstrate the predictive power of this anal. with the design and validation of an irreversible inhibitor of KIT/PDGFR kinases.  This is the first example of a covalent kinase inhibitor that combines a pharmacophore addressing the DFG-out conformation with a covalent trap.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkzqt0gMrz3rVg90H21EOLACvtfcHk0lhMwqfc9C8fVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFeis70%253D&md5=329ae5eaa6102b6dd34a0d1274631247</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fjm101396q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101396q%26sid%3Dliteratum%253Aachs%26aulast%3DLeproult%26aufirst%3DE.%26aulast%3DBarluenga%26aufirst%3DS.%26aulast%3DMoras%26aufirst%3DD.%26aulast%3DWurtz%26aufirst%3DJ.%2BM.%26aulast%3DWinssinger%26aufirst%3DN.%26atitle%3DCysteine%2520mapping%2520in%2520conformationally%2520distinct%2520kinase%2520nucleotide%2520binding%2520sites%253A%2520application%2520to%2520the%2520design%2520of%2520selective%2520covalent%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1347%26epage%3D1355%26doi%3D10.1021%2Fjm101396q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Erve, J. C.</span><span> </span><span class="NLM_article-title">Chemical toxicology: reactive intermediates and their role in pharmacology and toxicology</span> <span class="citation_source-journal">Expert Opin. Drug Metab. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">923</span><span class="NLM_x">–</span> <span class="NLM_lpage">946</span><span class="refDoi"> DOI: 10.1517/17425255.2.6.923</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1517%2F17425255.2.6.923" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=17125409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1eks7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=923-946&author=J.+C.+Erve&title=Chemical+toxicology%3A+reactive+intermediates+and+their+role+in+pharmacology+and+toxicology&doi=10.1517%2F17425255.2.6.923"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical toxicology: reactive intermediates and their role in pharmacology and toxicology</span></div><div class="casAuthors">Erve, John C. L.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Metabolism & Toxicology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">923-946</span>CODEN:
                <span class="NLM_cas:coden">EODMAP</span>;
        ISSN:<span class="NLM_cas:issn">1742-5255</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Reactive intermediates formed during the metab. of drugs have been investigated extensively over the past decades.  Today, interest in reactive intermediates in drug discovery is focused on minimizing bioactivation in hopes of reducing the risk of causing so-called idiosyncratic toxicity.  These efforts are justified based on the hapten hypothesis', namely, that on binding to protein, reactive intermediates may elicit an immune response to the modified protein, leading to a cascade of events that ultimately manifests as a toxic outcome.  However, the pharmacol. action of certain drugs depends on reactive intermediates that modify crit. amino acid residues of proteins, typically enzymes, thereby altering their activity.  Thus, the notion that reactive intermediates are inherently dangerous is unjustified.  When a reactive intermediate is necessary for the desired pharmacol. effect of a drug, the selectivity it displays towards the target protein is crucial, as off-target binding may produce unwanted toxicities.  On the other hand, reactive intermediates may play no role in toxicity.  This review provides a balanced perspective, primarily focusing on the proposed role of reactive intermediates in drug toxicity, while also highlighting examples in which they are involved in causing the desired pharmacol.  It is hoped that this knowledge can help scientists involved in drug discovery and development in their challenging task of producing safe and effective drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBIFLSUGUWN7Vg90H21EOLACvtfcHk0lhMwqfc9C8fVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1eks7nK&md5=320b9ade5e38e5d21b92a4e2781e9e62</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1517%2F17425255.2.6.923&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425255.2.6.923%26sid%3Dliteratum%253Aachs%26aulast%3DErve%26aufirst%3DJ.%2BC.%26atitle%3DChemical%2520toxicology%253A%2520reactive%2520intermediates%2520and%2520their%2520role%2520in%2520pharmacology%2520and%2520toxicology%26jtitle%3DExpert%2520Opin.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2006%26volume%3D2%26spage%3D923%26epage%3D946%26doi%3D10.1517%2F17425255.2.6.923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Guengerich, F. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacDonald, J. S.</span><span> </span><span class="NLM_article-title">Applying mechanisms of chemical toxicity to predict drug safety</span> <span class="citation_source-journal">Chem. Res. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">344</span><span class="NLM_x">–</span> <span class="NLM_lpage">369</span><span class="refDoi"> DOI: 10.1021/tx600260a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx600260a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhslahsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2007&pages=344-369&author=F.+P.+Guengerichauthor=J.+S.+MacDonald&title=Applying+mechanisms+of+chemical+toxicity+to+predict+drug+safety&doi=10.1021%2Ftx600260a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Applying Mechanisms of Chemical Toxicity to Predict Drug Safety</span></div><div class="casAuthors">Guengerich, F. Peter; MacDonald, James S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">344-369</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Toxicol. can no longer be used only as a science that reacts to problems but must be more proactive in predicting potential human safety issues with new drug candidates.  Success in this area must be based on an understanding of the mechanisms of toxicity.  This review summarizes and extends some of the concepts of an American Chem. Society ProSpectives meeting on the title subject held in June 2006.  One important area is the discernment of the exact nature of the most common problems in drug toxicity.  Knowledge of chem. structure alerts and relevant biol. pathways are important.  Biol. activation to reactive products and off-target pharmacol. are considered to be major contexts of drug toxicity, although defining exactly what the contributions are is not trivial.  Some newer approaches to screening for both have been developed.  A goal in predictive toxicol. is the use of in vitro methods and database development to make predictions concerning potential modes of toxicity and to stratify drug candidates for further development.  Such predictions are desirable for several economic and other reasons but are certainly not routine yet.  However, progress has been made using several approaches.  Some examples of the application of studies of wide-scale biol. responses are now available, with incorporation into development paradigms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxyWz1mKS0h7Vg90H21EOLACvtfcHk0lguwl6WChR2ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhslahsrk%253D&md5=d1ef7299d11c39d0ff3b031779915e54</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Ftx600260a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx600260a%26sid%3Dliteratum%253Aachs%26aulast%3DGuengerich%26aufirst%3DF.%2BP.%26aulast%3DMacDonald%26aufirst%3DJ.%2BS.%26atitle%3DApplying%2520mechanisms%2520of%2520chemical%2520toxicity%2520to%2520predict%2520drug%2520safety%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2007%26volume%3D20%26spage%3D344%26epage%3D369%26doi%3D10.1021%2Ftx600260a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Park, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boobis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldring, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenna, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laverty, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naisbitt, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicoll-Griffith, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obach, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Routledge, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tweedie, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vermeulen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baillie, T. A.</span><span> </span><span class="NLM_article-title">Managing the challenge of chemically reactive metabolites in drug development</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">292</span><span class="NLM_x">–</span> <span class="NLM_lpage">306</span><span class="refDoi"> DOI: 10.1038/nrd3408</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1038%2Fnrd3408" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=21455238" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGnu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=292-306&author=B.+K.+Parkauthor=A.+Boobisauthor=S.+Clarkeauthor=C.+E.+Goldringauthor=D.+Jonesauthor=J.+G.+Kennaauthor=C.+Lambertauthor=H.+G.+Lavertyauthor=D.+J.+Naisbittauthor=S.+Nelsonauthor=D.+A.+Nicoll-Griffithauthor=R.+S.+Obachauthor=P.+Routledgeauthor=D.+A.+Smithauthor=D.+J.+Tweedieauthor=N.+Vermeulenauthor=D.+P.+Williamsauthor=I.+D.+Wilsonauthor=T.+A.+Baillie&title=Managing+the+challenge+of+chemically+reactive+metabolites+in+drug+development&doi=10.1038%2Fnrd3408"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Managing the challenge of chemically reactive metabolites in drug development</span></div><div class="casAuthors">Park, B. Kevin; Boobis, Alan; Clarke, Stephen; Goldring, Chris E. P.; Jones, David; Kenna, J. Gerry; Lambert, Craig; Laverty, Hugh G.; Naisbitt, Dean J.; Nelson, Sidney; Nicoll-Griffith, Deborah A.; Obach, R. Scott; Routledge, Philip; Smith, Dennis A.; Tweedie, Donald J.; Vermeulen, Nico; Williams, Dominic P.; Wilson, Ian D.; Baillie, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">292-306</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The normal metab. of drugs can generate metabolites that have intrinsic chem. reactivity towards cellular mols., and therefore have the potential to alter biol. function and initiate serious adverse drug reactions.  Here, we present an assessment of the current approaches used for the evaluation of chem. reactive metabolites.  We also describe how these approaches are being used within the pharmaceutical industry to assess and minimize the potential of drug candidates to cause toxicity.  At early stages of drug discovery, iteration between medicinal chem. and drug metab. can eliminate perceived reactive metabolite-mediated chem. liabilities without compromising pharmacol. activity or the need for extensive safety evaluation beyond std. practices.  In the future, reactive metabolite evaluation may also be useful during clin. development for improving clin. risk assessment and risk management.  Currently, there remains a huge gap in our understanding of the basic mechanisms that underlie chem. stress-mediated adverse reactions in humans.  This Review summarizes our views on this complex topic, and includes insights into practices considered by the pharmaceutical industry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBv41vr7ZlNLVg90H21EOLACvtfcHk0lguwl6WChR2ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGnu78%253D&md5=8f6865f17bbe50b51aa4282379e82a08</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fnrd3408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3408%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DB.%2BK.%26aulast%3DBoobis%26aufirst%3DA.%26aulast%3DClarke%26aufirst%3DS.%26aulast%3DGoldring%26aufirst%3DC.%2BE.%26aulast%3DJones%26aufirst%3DD.%26aulast%3DKenna%26aufirst%3DJ.%2BG.%26aulast%3DLambert%26aufirst%3DC.%26aulast%3DLaverty%26aufirst%3DH.%2BG.%26aulast%3DNaisbitt%26aufirst%3DD.%2BJ.%26aulast%3DNelson%26aufirst%3DS.%26aulast%3DNicoll-Griffith%26aufirst%3DD.%2BA.%26aulast%3DObach%26aufirst%3DR.%2BS.%26aulast%3DRoutledge%26aufirst%3DP.%26aulast%3DSmith%26aufirst%3DD.%2BA.%26aulast%3DTweedie%26aufirst%3DD.%2BJ.%26aulast%3DVermeulen%26aufirst%3DN.%26aulast%3DWilliams%26aufirst%3DD.%2BP.%26aulast%3DWilson%26aufirst%3DI.%2BD.%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26atitle%3DManaging%2520the%2520challenge%2520of%2520chemically%2520reactive%2520metabolites%2520in%2520drug%2520development%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D292%26epage%3D306%26doi%3D10.1038%2Fnrd3408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Dungo, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keating, G. M.</span><span> </span><span class="NLM_article-title">Afatinib: first global approval</span> <span class="citation_source-journal">Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">1503</span><span class="NLM_x">–</span> <span class="NLM_lpage">1515</span><span class="refDoi"> DOI: 10.1007/s40265-013-0111-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1007%2Fs40265-013-0111-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=23982599" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVWmurnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=1503-1515&author=R.+T.+Dungoauthor=G.+M.+Keating&title=Afatinib%3A+first+global+approval&doi=10.1007%2Fs40265-013-0111-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Afatinib: First Global Approval</span></div><div class="casAuthors">Dungo, Rosselle T.; Keating, Gillian M.</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1503-1515</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">A review.  Afatinib, an irreversible inhibitor of the ErbB family of tyrosine kinases, is under development with Boehringer Ingelheim for the once-daily, oral treatment of cancer.  Afatinib downregulates ErbB signalling by covalently binding to epidermal growth factor receptor (EGFR), human epidermal growth factor receptor (HER) 2 and HER4, irreversibly inhibiting tyrosine kinase autophosphorylation.  It also inhibits transphosphorylation of HER3.  Oral afatinib (Gilotrif®) has been approved in the US for the first-line treatment of patients with metastatic non-small-cell lung cancer (NSCLC) who have tumors with EGFR exon 19 deletions or exon 21 (L858R) substitution mutations as detected by a US FDA-approved test.  Afatinib has also been approved in Taiwan for the first-line treatment of patients with EGFR mutation-pos. NSCLC.  In addn., the European Medicines Agency's Committee for Medicinal Products for Human Use has recommended the approval of afatinib (Giotrif) for the treatment of patients with locally advanced or metastatic NSCLC with activating EGFR mutations who are EGFR tyrosine kinase inhibitor naive.  Afatinib is also under regulatory review in Canada, Japan and other Asian countries.  This article summarizes the milestones in the development of afatinib, leading to this first approval in patients with metastatic NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdPcU5aIqrY7Vg90H21EOLACvtfcHk0lguwl6WChR2ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVWmurnM&md5=21851cbfac56e2fffcba6c731f123cff</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1007%2Fs40265-013-0111-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-013-0111-6%26sid%3Dliteratum%253Aachs%26aulast%3DDungo%26aufirst%3DR.%2BT.%26aulast%3DKeating%26aufirst%3DG.%2BM.%26atitle%3DAfatinib%253A%2520first%2520global%2520approval%26jtitle%3DDrugs%26date%3D2013%26volume%3D73%26spage%3D1503%26epage%3D1515%26doi%3D10.1007%2Fs40265-013-0111-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Cameron, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanford, M.</span><span> </span><span class="NLM_article-title">Ibrutinib: first global approval</span> <span class="citation_source-journal">Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">74</span><span class="NLM_x">, </span> <span class="NLM_fpage">263</span><span class="NLM_x">–</span> <span class="NLM_lpage">271</span><span class="refDoi"> DOI: 10.1007/s40265-014-0178-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1007%2Fs40265-014-0178-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=24464309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVyjsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=263-271&author=F.+Cameronauthor=M.+Sanford&title=Ibrutinib%3A+first+global+approval&doi=10.1007%2Fs40265-014-0178-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Ibrutinib: First Global Approval</span></div><div class="casAuthors">Cameron, Fiona; Sanford, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-271</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">A review.  Ibrutinib (Imbruvica®) is a small mol., first-in-class, once-daily, orally available, Bruton's tyrosine kinase inhibitor that is under development for the treatment of B cell malignancies, including chronic lymphocytic leukemia (CLL), mantle cell lymphoma (MCL) and diffuse large B cell lymphoma (DLBCL), as well as multiple myeloma (MM), follicular lymphoma (FL) and Waldenstrom's macroglobulinemia (WM).  It has been developed by Pharmacyclics, Inc. and Janssen Biotech, Inc.  Ibrutinib acts by blocking B-cell antigen receptor signalling, thereby reducing malignant proliferation of B cells and inducing cell death.  Based chiefly on findings from a phase Ib/II study, ibrutinib has been approved in the USA for the treatment of MCL in previously treated patients and is one of the first approvals through the US FDA's Breakthrough Therapy Designation Pathway.  An application has been filed in the EU seeking regulatory approval in this indication.  In both the USA and EU, further applications have been filed with regulatory bodies seeking approval for the use of ibrutinib in patients with previously treated CLL/small lymphocytic lymphoma (SLL).  Phase III trials are underway worldwide to evaluate ibrutinib in the treatment of patients with CLL/SLL, DLBCL and MCL, and the agent is in phase II development for use in WM, FL and MM.  This article summarizes the milestones in the development of ibrutinib leading to its first approval in MCL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAwhrk_pQFtLVg90H21EOLACvtfcHk0lhI9KukrkFOmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVyjsL4%253D&md5=de4c4ee9a8b2e72c866c578ccf5db22b</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1007%2Fs40265-014-0178-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-014-0178-8%26sid%3Dliteratum%253Aachs%26aulast%3DCameron%26aufirst%3DF.%26aulast%3DSanford%26aufirst%3DM.%26atitle%3DIbrutinib%253A%2520first%2520global%2520approval%26jtitle%3DDrugs%26date%3D2014%26volume%3D74%26spage%3D263%26epage%3D271%26doi%3D10.1007%2Fs40265-014-0178-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Zambaldo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sadhu, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karthikeyan, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barluenga, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daguer, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winssinger, N.</span><span> </span><span class="NLM_article-title">Selective affinity-based probe for oncogenic kinases suitable for live cell imaging</span> <span class="citation_source-journal">Chemical Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">2088</span><span class="NLM_x">–</span> <span class="NLM_lpage">2092</span><span class="refDoi"> DOI: 10.1039/c3sc21856b</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1039%2Fc3sc21856b" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2088-2092&author=C.+Zambaldoauthor=K.+K.+Sadhuauthor=G.+Karthikeyanauthor=S.+Barluengaauthor=J.+P.+Daguerauthor=N.+Winssinger&title=Selective+affinity-based+probe+for+oncogenic+kinases+suitable+for+live+cell+imaging&doi=10.1039%2Fc3sc21856b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1039%2Fc3sc21856b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3sc21856b%26sid%3Dliteratum%253Aachs%26aulast%3DZambaldo%26aufirst%3DC.%26aulast%3DSadhu%26aufirst%3DK.%2BK.%26aulast%3DKarthikeyan%26aufirst%3DG.%26aulast%3DBarluenga%26aufirst%3DS.%26aulast%3DDaguer%26aufirst%3DJ.%2BP.%26aulast%3DWinssinger%26aufirst%3DN.%26atitle%3DSelective%2520affinity-based%2520probe%2520for%2520oncogenic%2520kinases%2520suitable%2520for%2520live%2520cell%2520imaging%26jtitle%3DChemical%2520Science%26date%3D2013%26volume%3D4%26spage%3D2088%26epage%3D2092%26doi%3D10.1039%2Fc3sc21856b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">Automatic Processing of Rotation Diffraction Data from Crystals of Initially Unknown Symmetry and Cell Constants</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">795</span><span class="NLM_x">–</span> <span class="NLM_lpage">800</span><span class="refDoi"> DOI: 10.1107/S0021889893005588</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1107%2FS0021889893005588" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaK2cXptFeltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1993&pages=795-800&author=W.+Kabsch&title=Automatic+Processing+of+Rotation+Diffraction+Data+from+Crystals+of+Initially+Unknown+Symmetry+and+Cell+Constants&doi=10.1107%2FS0021889893005588"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic processing of rotation diffraction data from crystals of initially unknown symmetry and cell constants</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">795-800</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">An algorithm was developed for the automatic interpretation of a given set of obsd. reciprocal-lattice points.  It exts. a reduced cell and assigns indexes to each reflection by a graph-theor. implementation of the local indexing method.  All possible symmetries of the obsd. lattice compatible with the metric of the reduced cell are recognized and reported, together with the unit-cell consts. and the linear index transformation relating the conventional to the reduced cell.  This algorithm was incorporated into the program XDS (K., 1988), which is now able to process single-crystal area-detector data without prior knowledge of the symmetry and the unit-cell consts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD1lV8_7cDv7Vg90H21EOLACvtfcHk0lhI9KukrkFOmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXptFeltw%253D%253D&md5=2b703225206ce99af0d7e38acf6d75b3</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1107%2FS0021889893005588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889893005588%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DAutomatic%2520Processing%2520of%2520Rotation%2520Diffraction%2520Data%2520from%2520Crystals%2520of%2520Initially%2520Unknown%2520Symmetry%2520and%2520Cell%2520Constants%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1993%26volume%3D26%26spage%3D795%26epage%3D800%26doi%3D10.1107%2FS0021889893005588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Pushing the boundaries of molecular replacement with maximum likelihood</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">1373</span><span class="NLM_x">–</span> <span class="NLM_lpage">1382</span><span class="refDoi"> DOI: 10.1107/S0907444901012471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1107%2FS0907444901012471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=11567148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFGltrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2001&pages=1373-1382&author=R.+J.+Read&title=Pushing+the+boundaries+of+molecular+replacement+with+maximum+likelihood&doi=10.1107%2FS0907444901012471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Pushing the boundaries of molecular replacement with maximum likelihood</span></div><div class="casAuthors">Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">D57</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1373-1382</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">The mol.-replacement method works well with good models and simple unit cells, but often fails with more difficult problems.  Experience with likelihood in other areas of crystallog. suggests that it would improve performance significantly.  For mol. replacement, the form of the required likelihood function depends on whether there is ambiguity in the relative phases of the contributions from symmetry-related mols. (e.g. rotation vs. translation searches).  Likelihood functions used in structure refinement are appropriate only for translation (or six-dimensional) searches, where the correct translation will place all of the atoms in the model approx. correctly.  A new likelihood function that allows for unknown relative phases is suitable for rotation searches.  It is shown that correlations between sequence identity and coordinate error can be used to calibrate parameters for model quality in the likelihood functions.  Multiple models of a mol. can be combined in a statistically valid way by setting up the joint probability distribution of the true and model structure factors as a multivariate complex normal distribution, from which the conditional distribution of the true structure factor given the models can be derived.  Tests in a new mol.-replacement program, Beast, show that the likelihood-based targets are more sensitive and more accurate than previous targets.  The new multiple-model likelihood function has a dramatic impact on success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo23lasNP9RyLVg90H21EOLACvtfcHk0lhI9KukrkFOmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFGltrc%253D&md5=b342c58edd40265324e0241ff75cec67</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1107%2FS0907444901012471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444901012471%26sid%3Dliteratum%253Aachs%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPushing%2520the%2520boundaries%2520of%2520molecular%2520replacement%2520with%2520maximum%2520likelihood%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2001%26volume%3D57%26spage%3D1373%26epage%3D1382%26doi%3D10.1107%2FS0907444901012471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Seeliger, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagar, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frank, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henderson, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuriyan, J.</span><span> </span><span class="NLM_article-title">c-Src binds to the cancer drug imatinib with an inactive Abl/c-Kit conformation and a distributed thermodynamic penalty</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">299</span><span class="NLM_x">–</span> <span class="NLM_lpage">311</span><span class="refDoi"> DOI: 10.1016/j.str.2007.01.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1016%2Fj.str.2007.01.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=17355866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2sXisleisbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2007&pages=299-311&author=M.+A.+Seeligerauthor=B.+Nagarauthor=F.+Frankauthor=X.+Caoauthor=M.+N.+Hendersonauthor=J.+Kuriyan&title=c-Src+binds+to+the+cancer+drug+imatinib+with+an+inactive+Abl%2Fc-Kit+conformation+and+a+distributed+thermodynamic+penalty&doi=10.1016%2Fj.str.2007.01.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">c-Src Binds to the Cancer Drug Imatinib with an Inactive Abl/c-Kit Conformation and a Distributed Thermodynamic Penalty</span></div><div class="casAuthors">Seeliger, Markus A.; Nagar, Bhushan; Frank, Filipp; Cao, Xiaoxian; Henderson, M. Nidanie; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">299-311</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The cancer drug imatinib inhibits the tyrosine kinases c-Abl, c-Kit, and the PDGF receptor.  Imatinib is less effective against c-Src, which is difficult to understand because residues interacting with imatinib in crystal structures of Abl and c-Kit are conserved in c-Src.  The crystal structure of the c-Src kinase domain in complex with imatinib closely resembles that of Abl•imatinib and c-Kit•imatinib, and differs significantly from the inactive "Src/CDK" conformation of the Src family kinases.  Attempts to increase the affinity of c-Src for imatinib by swapping residues with the corresponding residues in Abl have not been successful, suggesting that the thermodn. penalty for adoption of the imatinib-binding conformation by c-Src is distributed over a broad region of the structure.  Two mutations that are expected to destabilize the inactive Src/CDK conformation increase drug sensitivity 15-fold, suggesting that the free-energy balance between different inactive states is a key to imatinib binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2SQGjjXdg1LVg90H21EOLACvtfcHk0lhL8YHOlS1hmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXisleisbY%253D&md5=ddfbcf06d91b408ca5fb25af9774c334</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2007.01.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2007.01.015%26sid%3Dliteratum%253Aachs%26aulast%3DSeeliger%26aufirst%3DM.%2BA.%26aulast%3DNagar%26aufirst%3DB.%26aulast%3DFrank%26aufirst%3DF.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DHenderson%26aufirst%3DM.%2BN.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3Dc-Src%2520binds%2520to%2520the%2520cancer%2520drug%2520imatinib%2520with%2520an%2520inactive%2520Abl%252Fc-Kit%2520conformation%2520and%2520a%2520distributed%2520thermodynamic%2520penalty%26jtitle%3DStructure%26date%3D2007%26volume%3D15%26spage%3D299%26epage%3D311%26doi%3D10.1016%2Fj.str.2007.01.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span><span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lhL8YHOlS1hmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Brünger, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clore, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeLano, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gros, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuszewski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilges, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannu, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simonson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warren, G. L.</span><span> </span><span class="NLM_article-title">Crystallography & NMR system: A new software suite for macromolecular structure determination</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">905</span><span class="NLM_x">–</span> <span class="NLM_lpage">921</span><span class="refDoi"> DOI: 10.1107/S0907444998003254</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1107%2FS0907444998003254" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=9757107" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A280%3ADyaK1cvjslKhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1998&pages=905-921&author=A.+T.+Br%C3%BCngerauthor=P.+D.+Adamsauthor=G.+M.+Cloreauthor=W.+L.+DeLanoauthor=P.+Grosauthor=R.+W.+Grosse-Kunstleveauthor=J.+S.+Jiangauthor=J.+Kuszewskiauthor=M.+Nilgesauthor=N.+S.+Pannuauthor=R.+J.+Readauthor=L.+M.+Riceauthor=T.+Simonsonauthor=G.+L.+Warren&title=Crystallography+%26+NMR+system%3A+A+new+software+suite+for+macromolecular+structure+determination&doi=10.1107%2FS0907444998003254"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallography & NMR system: A new software suite for macromolecular structure determination</span></div><div class="casAuthors">Brunger A T; Adams P D; Clore G M; DeLano W L; Gros P; Grosse-Kunstleve R W; Jiang J S; Kuszewski J; Nilges M; Pannu N S; Read R J; Rice L M; Simonson T; Warren G L</div><div class="citationInfo"><span class="NLM_cas:title">Acta crystallographica. Section D, Biological crystallography</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">Pt 5</span>),
    <span class="NLM_cas:pages">905-21</span>
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    </div><div class="casAbstract">A new software suite, called Crystallography & NMR System (CNS), has been developed for macromolecular structure determination by X-ray crystallography or solution nuclear magnetic resonance (NMR) spectroscopy.  In contrast to existing structure-determination programs, the architecture of CNS is highly flexible, allowing for extension to other structure-determination methods, such as electron microscopy and solid-state NMR spectroscopy.  CNS has a hierarchical structure: a high-level hypertext markup language (HTML) user interface, task-oriented user input files, module files, a symbolic structure-determination language (CNS language), and low-level source code.  Each layer is accessible to the user.  The novice user may just use the HTML interface, while the more advanced user may use any of the other layers.  The source code will be distributed, thus source-code modification is possible.  The CNS language is sufficiently powerful and flexible that many new algorithms can be easily implemented in the CNS language without changes to the source code.  The CNS language allows the user to perform operations on data structures, such as structure factors, electron-density maps, and atomic properties.  The power of the CNS language has been demonstrated by the implementation of a comprehensive set of crystallographic procedures for phasing, density modification and refinement.  User-friendly task-oriented input files are available for nearly all aspects of macromolecular structure determination by X-ray crystallography and solution NMR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRAeLCFcGBdlx0LsK7uFuY7fW6udTcc2eYbobSxIxPXfrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1cvjslKhsw%253D%253D&md5=37afee1b07ac44d3f3730a39061b4e79</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1107%2FS0907444998003254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444998003254%26sid%3Dliteratum%253Aachs%26aulast%3DBr%25C3%25BCnger%26aufirst%3DA.%2BT.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DClore%26aufirst%3DG.%2BM.%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26aulast%3DGros%26aufirst%3DP.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DJiang%26aufirst%3DJ.%2BS.%26aulast%3DKuszewski%26aufirst%3DJ.%26aulast%3DNilges%26aufirst%3DM.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRice%26aufirst%3DL.%2BM.%26aulast%3DSimonson%26aufirst%3DT.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26atitle%3DCrystallography%2520%2526%2520NMR%2520system%253A%2520A%2520new%2520software%2520suite%2520for%2520macromolecular%2520structure%2520determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1998%26volume%3D54%26spage%3D905%26epage%3D921%26doi%3D10.1107%2FS0907444998003254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span><span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0ljfJUbPWU_etA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Laskowski, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macarthur, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moss, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J. M.</span><span> </span><span class="NLM_article-title">Procheck - a Program to Check the Stereochemical Quality of Protein Structures</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">283</span><span class="NLM_x">–</span> <span class="NLM_lpage">291</span><span class="refDoi"> DOI: 10.1107/S0021889892009944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1107%2FS0021889892009944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1993&pages=283-291&author=R.+A.+Laskowskiauthor=M.+W.+Macarthurauthor=D.+S.+Mossauthor=J.+M.+Thornton&title=Procheck+-+a+Program+to+Check+the+Stereochemical+Quality+of+Protein+Structures&doi=10.1107%2FS0021889892009944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">PROCHECK: a program to check the stereochemical quality of protein structures</span></div><div class="casAuthors">Laskowski, Roman A.; MacArthur, Malcolm W.; Moss, David S.; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">283-91</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">The PROCHECK suite of programs provides a detailed check on the stereochem. of a protein structure.  Its outputs comprise a no. of plots in PostScript format and a comprehensive residue-by-residue listing.  These give an assessment of the overall quality of the structure as compared with well-refined structures of the same resoln. and also highlight regions that may need further investigation.  The PROCHECK programs are useful for assessing the quality not only of protein structures in the process of being solved but also of existing structures and of those being modeled on known structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCe5FgJf4KH7Vg90H21EOLACvtfcHk0ljfJUbPWU_etA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D&md5=fb69fc4410cd716aaaa7cc0db06b3ed2</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1107%2FS0021889892009944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889892009944%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DMacarthur%26aufirst%3DM.%2BW.%26aulast%3DMoss%26aufirst%3DD.%2BS.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DProcheck%2520-%2520a%2520Program%2520to%2520Check%2520the%2520Stereochemical%2520Quality%2520of%2520Protein%2520Structures%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1993%26volume%3D26%26spage%3D283%26epage%3D291%26doi%3D10.1107%2FS0021889892009944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">DeLano, W. L.</span>The PyMOL Molecular Graphics System; <span class="NLM_publisher-name">DeLano Scientific</span>: <span class="NLM_publisher-loc">San Carlos, CA</span>, 2002. <a href="http://www.pymol.org/" class="extLink">http://www.pymol.org/</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=DeLano%2C+W.+L.The+PyMOL+Molecular+Graphics+System%3B+DeLano+Scientific%3A+San+Carlos%2C+CA%2C+2002.+http%3A%2F%2Fwww.pymol.org%2F."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DDeLano%2520Scientific" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group">Avdeev, A.</span><span> </span><span class="NLM_article-title">High-throughput measurements of solubility profiles</span> <span class="citation_source-journal">Pharmacokinetic Optimization in Drug Research</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_fpage">305</span><span class="NLM_x">–</span> <span class="NLM_lpage">325</span><span class="refDoi"> DOI: 10.1002/9783906390437.ch18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1002%2F9783906390437.ch18" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2001&pages=305-325&author=A.+Avdeev&title=High-throughput+measurements+of+solubility+profiles&doi=10.1002%2F9783906390437.ch18"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1002%2F9783906390437.ch18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F9783906390437.ch18%26sid%3Dliteratum%253Aachs%26aulast%3DAvdeev%26aufirst%3DA.%26atitle%3DHigh-throughput%2520measurements%2520of%2520solubility%2520profiles%26jtitle%3DPharmacokinetic%2520Optimization%2520in%2520Drug%2520Research%26date%3D2001%26spage%3D305%26epage%3D325%26doi%3D10.1002%2F9783906390437.ch18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group">Zhang, Z. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, A. G.</span><span> </span><span class="NLM_article-title">A universal algorithm for fast and automated charge state deconvolution of electrospray mass-to-charge ratio spectra</span> <span class="citation_source-journal">J. Am. Soc. Mass Spectrom.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">225</span><span class="NLM_x">–</span> <span class="NLM_lpage">233</span><span class="refDoi"> DOI: 10.1016/S1044-0305(97)00284-5</span> </span><div class="citationLinks">[<a href="/doi/10.1016/S1044-0305%2897%2900284-5" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaK1cXhtlynsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1998&pages=225-233&author=Z.+Q.+Zhangauthor=A.+G.+Marshall&title=A+universal+algorithm+for+fast+and+automated+charge+state+deconvolution+of+electrospray+mass-to-charge+ratio+spectra&doi=10.1016%2FS1044-0305%2897%2900284-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">A universal algorithm for fast and automated charge state deconvolution of electrospray mass-to-charge ratio spectra</span></div><div class="casAuthors">Zhang, Zhongqi; Marshall, Alan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Society for Mass Spectrometry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">225-233</span>CODEN:
                <span class="NLM_cas:coden">JAMSEF</span>;
        ISSN:<span class="NLM_cas:issn">1044-0305</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">This article describes a new algorithm for charge state detn. and deconvolution of electrospray ionization (ESI) mass-to-charge ratio spectra.  The algorithm (ZSCORE) is based on a charge scoring scheme that incorporates all above-threshold members of a family of charge states or isotopic components, and deconvolves both low- and high-resoln. mass-to-charge ratio spectra, with or without a peak list (stick plot).  A scoring wt. factor, log (I/I0), in which I is the signal magnitude at a calcd. mass-to-charge ratio, and I0 is the signal threshold near that mass-to-charge ratio, was used in most cases.  For high-resoln. mass-to-charge ratio spectra in which all isotopic peaks are resolved, the algorithm can deconvolve overlapped isotopic multiplets of the same of different charge state.  Compared to other deconvolution techniques, the algorithm is robust, rapid, and fully automated (i.e., no user input during the deconvolution process).  It eliminates artifact peaks without introducing peak distortions.  Its performance is demonstrated for exptl. ESI Fourier transform ICR mass-to-charge ratio spectra (both low and high resoln.).  Charge state deconvolution to yield a zero-charge mass spectrum should prove particularly useful for interpreting spectra of complex mixts., identifying contaminants, noncovalent adducts, fragments (N-terminal, C-terminal, internal), and chem. modifications of electrosprayed biomacromols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-7hFdAcR_BbVg90H21EOLACvtfcHk0ljfJUbPWU_etA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhtlynsbo%253D&md5=5bee6ce6556beed4ee9004cfe6c7ecad</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2FS1044-0305%2897%2900284-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1044-0305%252897%252900284-5%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%2BQ.%26aulast%3DMarshall%26aufirst%3DA.%2BG.%26atitle%3DA%2520universal%2520algorithm%2520for%2520fast%2520and%2520automated%2520charge%2520state%2520deconvolution%2520of%2520electrospray%2520mass-to-charge%2520ratio%2520spectra%26jtitle%3DJ.%2520Am.%2520Soc.%2520Mass%2520Spectrom.%26date%3D1998%26volume%3D9%26spage%3D225%26epage%3D233%26doi%3D10.1016%2FS1044-0305%2897%2900284-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group">Milletti, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storchi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sforna, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span> </span><span class="NLM_article-title">New and original pKa prediction method using grid molecular interaction fields</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2172</span><span class="NLM_x">–</span> <span class="NLM_lpage">2181</span><span class="refDoi"> DOI: 10.1021/ci700018y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci700018y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFSnsLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=2172-2181&author=F.+Millettiauthor=L.+Storchiauthor=G.+Sfornaauthor=G.+Cruciani&title=New+and+original+pKa+prediction+method+using+grid+molecular+interaction+fields&doi=10.1021%2Fci700018y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">New and Original pKa Prediction Method Using Grid Molecular Interaction Fields</span></div><div class="casAuthors">Milletti, Francesca; Storchi, Loriano; Sforna, Gianluca; Cruciani, Gabriele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2172-2181</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One of the most important physicochem. properties of a mol. is pKa.  It is known that two parameters imperative in ADME profiling, soly., and lipophilicity are governed by pKa, and receptor binding can be influenced by pKa.  Because most drugs are ionized in physiol. conditions, pKa is particularly relevant to medicinal chem.  Despite the numerous advances in high-throughput measurements, in silico detn. is still the fastest and cheapest way of obtaining pKa.  This paper presents a new original computational method for pKa prediction of org. compds.  Descriptors were generated using the program GRID, and these descriptors are based on mol. interaction fields precomputed on a set of mol. fragments.  The new method was developed, trained, and cross-validated by using a large and diverse data set of 24 617 pKa values.  This paper presents the results for a class of 421 acidic nitrogen compds. (RMSE = 0.41, r2 = 0.97, q2 = 0.87) and for a class of 947 six-membered N-heterocyclic bases (RMSE = 0.60, r2 = 0.93, q2 = 0.85).  For external validation 28 novel compds. were selected that covered nine different ionizable groups, and 39 pKa values could be exptl. detd. by spectral gradient anal. (SGA).  Comparison of exptl. pKa with calcd. pKa demonstrated that the predictive ability of the method is good (external set, r2 = 0.85, RMSE = 0.90).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzEk0uiS1E6rVg90H21EOLACvtfcHk0ljh16M5vt1z0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFSnsLrF&md5=f2b1dec853ccb0ef1a40540d234232fb</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fci700018y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700018y%26sid%3Dliteratum%253Aachs%26aulast%3DMilletti%26aufirst%3DF.%26aulast%3DStorchi%26aufirst%3DL.%26aulast%3DSforna%26aufirst%3DG.%26aulast%3DCruciani%26aufirst%3DG.%26atitle%3DNew%2520and%2520original%2520pKa%2520prediction%2520method%2520using%2520grid%2520molecular%2520interaction%2520fields%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D2172%26epage%3D2181%26doi%3D10.1021%2Fci700018y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group">Milletti, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storchi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sforna, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cross, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span> </span><span class="NLM_article-title">Tautomer enumeration and stability prediction for virtual screening on large chemical databases</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">68</span><span class="NLM_x">–</span> <span class="NLM_lpage">75</span><span class="refDoi"> DOI: 10.1021/ci800340j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800340j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisVahug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=68-75&author=F.+Millettiauthor=L.+Storchiauthor=G.+Sfornaauthor=S.+Crossauthor=G.+Cruciani&title=Tautomer+enumeration+and+stability+prediction+for+virtual+screening+on+large+chemical+databases&doi=10.1021%2Fci800340j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Tautomer enumeration and stability prediction for virtual screening on large chemical databases</span></div><div class="casAuthors">Milletti, Francesca; Storchi, Loriano; Sforna, Gianluca; Cross, Simon; Cruciani, Gabriele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">68-75</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tautomeric rearrangements affect the results of cheminformatics applications that depend on the knowledge of the 2D or 3D structure of a compd., such as tools for database searches, fingerprint generation, virtual screening, and phys.-chem. properties prediction.  In this paper we present TauThor, a tool to enumerate tautomers and predict tautomer stability in the aq. medium.  The enumeration is based on a recursive process that generates tautomers according to the general scheme HX-Y=Z .rdblhar. X=Y-ZH.  The stability of a tautomer is calcd. by using a library of 145 fragments assocd. with exptl. tautomeric percentages in water and a pKa based-method that utilizes pKa values predicted by MoKa.  Predicted tautomeric ratios based on pKa calcns. were benchmarked against literature data for a set of eleven compds.  The FDA approved drugs database, the NCI database and two vendor databases - Specs Screening Library and Asinex Gold Collection - were used to illustrate the impact of tautomerism on chem. libraries and to evaluate the relative occurrences of alternative tautomeric forms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYlrU_55G2ArVg90H21EOLACvtfcHk0ljh16M5vt1z0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisVahug%253D%253D&md5=eac540b04bcf7642cd3904c084504eb4</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Fci800340j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800340j%26sid%3Dliteratum%253Aachs%26aulast%3DMilletti%26aufirst%3DF.%26aulast%3DStorchi%26aufirst%3DL.%26aulast%3DSforna%26aufirst%3DG.%26aulast%3DCross%26aufirst%3DS.%26aulast%3DCruciani%26aufirst%3DG.%26atitle%3DTautomer%2520enumeration%2520and%2520stability%2520prediction%2520for%2520virtual%2520screening%2520on%2520large%2520chemical%2520databases%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D68%26epage%3D75%26doi%3D10.1021%2Fci800340j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group">Case, D. A.; Darden, T. A.; Cheatham, T. E.; Simmerling, C. L.; Wang, J.; Duke, R. E.; Luo, R.; Walker, R. C.; Zhang, W.; Merz, K. M.; Roberts, B.; Hayik, S.; Roitberg, A.; Seabra, G.; Swails, J.; Goetz, A. W.; Kolossváry, I.; Wong, K. F.; Paesani, F.; Vanicek, J.; Wolf, R. M.; Liu, J.; Wu, X.; Brozell, S. R.; Steinbrecher, T.; Gohlke, H.; Cai, Q.; Ye, X.; Wang, J.; Hsieh, M. J.; Cui, G.; Roe, D. R.; Mathews, D. H.; Seetin, M. G.; Solomon-Ferrer, R.; Sagui, C.; Babin, V.; Luchko, T.; Gusarov, S.; Kovalenko, A.; Kollman, P. A.</span>Amber12; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=D.+A.+Case&author=T.+A.+Darden&author=T.+E.+Cheatham&author=C.+L.+Simmerling&author=J.+Wang&author=R.+E.+Duke&author=R.+Luo&author=R.+C.+Walker&author=W.+Zhang&author=K.+M.+Merz&author=B.+Roberts&author=S.+Hayik&author=A.+Roitberg&author=G.+Seabra&author=J.+Swails&author=A.+W.+Goetz&author=I.+Kolossv%C3%A1ry&author=K.+F.+Wong&author=F.+Paesani&author=J.+Vanicek&author=R.+M.+Wolf&author=J.+Liu&author=X.+Wu&author=S.+R.+Brozell&author=T.+Steinbrecher&author=H.+Gohlke&author=Q.+Cai&author=X.+Ye&author=J.+Wang&author=M.+J.+Hsieh&author=G.+Cui&author=D.+R.+Roe&author=D.+H.+Mathews&author=M.+G.+Seetin&author=R.+Solomon-Ferrer&author=C.+Sagui&author=V.+Babin&author=T.+Luchko&author=S.+Gusarov&author=A.+Kovalenko&author=P.+A.+Kollman&title=Amber12"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26jtitle%3DAmber12%26pub%3DUniversity%2520of%2520California%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group">Sigalov, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fenley, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Onufriev, A.</span><span> </span><span class="NLM_article-title">Analytical electrostatics for biomolecules: beyond the generalized Born approximation</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">124902</span><span class="refDoi"> DOI: 10.1063/1.2177251</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1063%2F1.2177251" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=16599720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsVGisL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2006&pages=124902&author=G.+Sigalovauthor=A.+Fenleyauthor=A.+Onufriev&title=Analytical+electrostatics+for+biomolecules%3A+beyond+the+generalized+Born+approximation&doi=10.1063%2F1.2177251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Analytical electrostatics for biomolecules: Beyond the generalized Born approximation</span></div><div class="casAuthors">Sigalov, Grigori; Fenley, Andrew; Onufriev, Alexey</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">124902/1-124902/14</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The modeling and simulation of macromols. in soln. often benefits from fast anal. approxns. for the electrostatic interactions.  In our previous work, we proposed a method based on an approx. anal. soln. of the linearized Poisson-Boltzmann equation for a sphere.  In the current work, we extend the method to biomols. of arbitrary shape and provide computationally efficient algorithms for estn. of the parameters of the model.  This approach, which we tentatively call ALPB here, is tested against the std. numerical Poisson-Boltzmann (NPB) treatment on a set of 579 representative proteins, nucleic acids, and small peptides.  The tests are performed across a wide range of solvent/solute dielecs. and at biol. relevant salt concns.  Over the range of the solvent and solute parameters tested, the systematic deviation (from the NPB ref.) of solvation energies computed by ALPB is 0.5-3.5 kcal/mol, which is 5-50 times smaller than that of the conventional generalized Born approxn. widely used in this context.  At the same time, ALPB is equally computationally efficient.  The new model is incorporated into the AMBER mol. modeling package and tested on small proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSvnn4dOEBabVg90H21EOLACvtfcHk0lilCOCfXg7ZWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsVGisL8%253D&md5=abdf348673236503a15ec13e7493bd17</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1063%2F1.2177251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.2177251%26sid%3Dliteratum%253Aachs%26aulast%3DSigalov%26aufirst%3DG.%26aulast%3DFenley%26aufirst%3DA.%26aulast%3DOnufriev%26aufirst%3DA.%26atitle%3DAnalytical%2520electrostatics%2520for%2520biomolecules%253A%2520beyond%2520the%2520generalized%2520Born%2520approximation%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2006%26volume%3D124%26spage%3D124902%26doi%3D10.1063%2F1.2177251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group">Andersen, H. C.</span><span> </span><span class="NLM_article-title">Molecular-Dynamics Simulations at Constant Pressure and-or Temperature</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1980</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">2384</span><span class="NLM_x">–</span> <span class="NLM_lpage">2393</span><span class="refDoi"> DOI: 10.1063/1.439486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1063%2F1.439486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaL3cXhs12msbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=1980&pages=2384-2393&author=H.+C.+Andersen&title=Molecular-Dynamics+Simulations+at+Constant+Pressure+and-or+Temperature&doi=10.1063%2F1.439486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics simulations at constant pressure and/or temperature</span></div><div class="casAuthors">Andersen, Hans C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2384-93</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">In the mol. dynamics simulation method for fluids, the equations of motion for a collection of particles in a fixed vol. are solved numerically.  The energy, vol., and no. of particles are const. for a particular simulation, and it is assumed that time avs. of properties of the simulated fluid are equal to microcanonical ensemble avs. of the same properties.  In some situations, it is desirable to perform simulations of a fluid for particular values of temp. and/or pressure or under conditions in which the energy and vol. of the fluid can fluctuate.  Three methods for performing mol. dynamics simulations under conditions of const. temp. and/or pressure are discussed.  Time avs. of properties of the simulated fluid are equal to avs. over the isenthalpic-isobaric, canonical, and isothermal-isobaric ensembles. Each method is a way of describing the dynamics of a certain no. of particles in a vol. element of a fluid while taking into account the influence of surrounding particles in changing the energy and/or d. of the simulated vol. element.  The influence of the surroundings is taken into account without unwanted surface effects.  Examples of situations where these methods may be useful are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGof3p6WlEFZTbVg90H21EOLACvtfcHk0lilCOCfXg7ZWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXhs12msbo%253D&md5=54e4faa1c0ea4385dc565d8db5a31edc</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1063%2F1.439486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.439486%26sid%3Dliteratum%253Aachs%26aulast%3DAndersen%26aufirst%3DH.%2BC.%26atitle%3DMolecular-Dynamics%2520Simulations%2520at%2520Constant%2520Pressure%2520and-or%2520Temperature%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1980%26volume%3D72%26spage%3D2384%26epage%3D2393%26doi%3D10.1063%2F1.439486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group">Van der Spoel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindahl, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groenhof, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mark, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J. C.</span><span> </span><span class="NLM_article-title">GROMACS: Fast, flexible, and free</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1701</span><span class="NLM_x">–</span> <span class="NLM_lpage">1718</span><span class="refDoi"> DOI: 10.1002/jcc.20291</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1002%2Fjcc.20291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=16211538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1701-1718&author=D.+Van+der+Spoelauthor=E.+Lindahlauthor=B.+Hessauthor=G.+Groenhofauthor=A.+E.+Markauthor=H.+J.+C.+Berendsen&title=GROMACS%3A+Fast%2C+flexible%2C+and+free&doi=10.1002%2Fjcc.20291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS: Fast, flexible, and free</span></div><div class="casAuthors">Van Der Spoel, David; Lindahl, Erik; Hess, Berk; Groenhof, Gerrit; Mark, Alan E.; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1701-1718</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This article describes the software suite GROMACS (Groningen MAchine for Chem. Simulation) that was developed at the University of Groningen, The Netherlands, in the early 1990s.  The software, written in ANSI C, originates from a parallel hardware project, and is well suited for parallelization on processor clusters.  By careful optimization of neighbor searching and of inner loop performance, GROMACS is a very fast program for mol. dynamics simulation.  It does not have a force field of its own, but is compatible with GROMOS, OPLS, AMBER, and ENCAD force fields.  In addn., it can handle polarizable shell models and flexible constraints.  The program is versatile, as force routines can be added by the user, tabulated functions can be specified, and analyses can be easily customized.  Nonequil. dynamics and free energy detns. are incorporated.  Interfaces with popular quantum-chem. packages (MOPAC, GAMES-UK, GAUSSIAN) are provided to perform mixed MM/QM simulations.  The package includes about 100 utility and anal. programs.  GROMACS is in the public domain and distributed (with source code and documentation) under the GNU General Public License.  It is maintained by a group of developers from the Universities of Groningen, Uppsala, and Stockholm, and the Max Planck Institute for Polymer Research in Mainz.  Its Web site is http://www.gromacs.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxInIDyt9Y8bVg90H21EOLACvtfcHk0lilCOCfXg7ZWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO&md5=4aa71b1dc0b5b4978a88b0568245a265</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20291%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2Bder%2BSpoel%26aufirst%3DD.%26aulast%3DLindahl%26aufirst%3DE.%26aulast%3DHess%26aufirst%3DB.%26aulast%3DGroenhof%26aufirst%3DG.%26aulast%3DMark%26aufirst%3DA.%2BE.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DGROMACS%253A%2520Fast%252C%2520flexible%252C%2520and%2520free%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1701%26epage%3D1718%26doi%3D10.1002%2Fjcc.20291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group">Jarvis, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patrick, E. A.</span><span> </span><span class="NLM_article-title">Clustering Using a Similarity Measure Based on Shared near Neighbors</span> <span class="citation_source-journal">IEEE Trans. Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">C-22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1025</span><span class="NLM_x">–</span> <span class="NLM_lpage">1034</span><span class="refDoi"> DOI: 10.1109/T-C.1973.223640</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1109%2FT-C.1973.223640" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=C-22&publication_year=1973&pages=1025-1034&author=R.+A.+Jarvisauthor=E.+A.+Patrick&title=Clustering+Using+a+Similarity+Measure+Based+on+Shared+near+Neighbors&doi=10.1109%2FT-C.1973.223640"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1109%2FT-C.1973.223640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1109%252FT-C.1973.223640%26sid%3Dliteratum%253Aachs%26aulast%3DJarvis%26aufirst%3DR.%2BA.%26aulast%3DPatrick%26aufirst%3DE.%2BA.%26atitle%3DClustering%2520Using%2520a%2520Similarity%2520Measure%2520Based%2520on%2520Shared%2520near%2520Neighbors%26jtitle%3DIEEE%2520Trans.%2520Comput.%26date%3D1973%26volume%3DC-22%26spage%3D1025%26epage%3D1034%26doi%3D10.1109%2FT-C.1973.223640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group">Becke, A. D.</span><span> </span><span class="NLM_article-title">Density-functional exchange-energy approximation with correct asymptotic behavior</span> <span class="citation_source-journal">Phys. Rev. A: At., Mol., Opt. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">3098</span><span class="NLM_x">–</span> <span class="NLM_lpage">3100</span><span class="refDoi"> DOI: 10.1103/PhysRevA.38.3098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1103%2FPhysRevA.38.3098" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=9900728" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaL1cXmtlOhsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1988&pages=3098-3100&author=A.+D.+Becke&title=Density-functional+exchange-energy+approximation+with+correct+asymptotic+behavior&doi=10.1103%2FPhysRevA.38.3098"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Density-functional exchange-energy approximation with correct asymptotic behavior</span></div><div class="casAuthors">Becke, A. D.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review A:  Atomic, Molecular, and Optical Physics</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3098-100</span>CODEN:
                <span class="NLM_cas:coden">PLRAAN</span>;
        ISSN:<span class="NLM_cas:issn">0556-2791</span>.
    </div><div class="casAbstract">Current gradient-cor. d.-functional approxns. for the exchange energies of at. and mol. systems fail to reproduce the correct 1/r asymptotic behavior of the exchange-energy d.  A gradient-cor. exchange-energy functional is given with the proper asymptotic limit.  This functional, contg. only one parameter, fits the exact Hartree-Fock exchange energies of a wide variety of at. systems with remarkable accuracy, surpassing the performance of previous functionals contg. two parameters or more.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKr-bPUej6TLVg90H21EOLACvtfcHk0lilCOCfXg7ZWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXmtlOhsLo%253D&md5=d4d219c134a5a90f689a8abed04d82cc</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1103%2FPhysRevA.38.3098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevA.38.3098%26sid%3Dliteratum%253Aachs%26aulast%3DBecke%26aufirst%3DA.%2BD.%26atitle%3DDensity-functional%2520exchange-energy%2520approximation%2520with%2520correct%2520asymptotic%2520behavior%26jtitle%3DPhys.%2520Rev.%2520A%253A%2520At.%252C%2520Mol.%252C%2520Opt.%2520Phys.%26date%3D1988%26volume%3D38%26spage%3D3098%26epage%3D3100%26doi%3D10.1103%2FPhysRevA.38.3098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group">Lee, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parr, R. G.</span><span> </span><span class="NLM_article-title">Development of the Colle-Salvetti correlation-energy formula into a functional of the electron density</span> <span class="citation_source-journal">Phys. Rev. B: Condens. Matter Mater. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">785</span><span class="NLM_x">–</span> <span class="NLM_lpage">789</span><span class="refDoi"> DOI: 10.1103/PhysRevB.37.785</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1103%2FPhysRevB.37.785" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=9944570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaL1cXktFWrtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1988&pages=785-789&author=C.+Leeauthor=W.+Yangauthor=R.+G.+Parr&title=Development+of+the+Colle-Salvetti+correlation-energy+formula+into+a+functional+of+the+electron+density&doi=10.1103%2FPhysRevB.37.785"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Development of the Colle-Salvetti correlation-energy formula into a functional of the electron density</span></div><div class="casAuthors">Lee, Chengteh; Yang, Weitao; Parr, Robert G.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review B:  Condensed Matter and Materials Physics</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">785-9</span>CODEN:
                <span class="NLM_cas:coden">PRBMDO</span>;
        ISSN:<span class="NLM_cas:issn">0163-1829</span>.
    </div><div class="casAbstract">A correlation-energy formula due to R. Colle and D. Salvetti (1975), in which the correlation energy d. is expressed in terms of the electron d. and a Laplacian of the 2nd-order Hartree-Fock d. matrix, is restated as a formula involving the d. and local kinetic-energy d.  On insertion of gradient expansions for the local kinetic-energy d., d.-functional formulas for the correlation energy and correlation potential are then obtained.  Through numerical calcns. on a no. of atoms, pos. ions, and mols., of both open- and closed-shell type, it is demonstrated that these formulas, like the original Colle-Salvetti formulas, give correlation energies within a few percent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8yJ1cBkpi67Vg90H21EOLACvtfcHk0ljMibXZ83l5Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXktFWrtbw%253D&md5=ee7b59267a2ff72e15171a481819ccf8</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1103%2FPhysRevB.37.785&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevB.37.785%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DParr%26aufirst%3DR.%2BG.%26atitle%3DDevelopment%2520of%2520the%2520Colle-Salvetti%2520correlation-energy%2520formula%2520into%2520a%2520functional%2520of%2520the%2520electron%2520density%26jtitle%3DPhys.%2520Rev.%2520B%253A%2520Condens.%2520Matter%2520Mater.%2520Phys.%26date%3D1988%26volume%3D37%26spage%3D785%26epage%3D789%26doi%3D10.1103%2FPhysRevB.37.785" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group">Frisch, M. J.; Trucks, G. W.; Schlegel, H. B.; Scuseria, G. E.; Robb, M. A.; Cheeseman, J. R.; Scalmani, G.; Barone, V.; Mennucci, B.; Petersson, G. A.; Nakatsuji, H.; Caricato, M.; Li, X.; Hratchian, H. P.; Izmaylov, A. F.; Bloino, J.; Zheng, G.; Sonnenberg, J. L.; Hada, M.; Ehara, M.; Toyota, K.; Fukuda, R.; Hasegawa, J.; Ishida, M.; Nakajima, T.; Honda, Y.; Kitao, O.; Nakai, H.; Vreven, T.; Montgomery, J. A.,  Jr.; Peralta, J. E.; Ogliaro, F.; Bearpark, M.; Heyd, J. J.; Brothers, E.; Kudin, K. N.; Staroverov, V. N.; Kobayashi, R.; Normand, J.; Raghavachari, K.; Rendell, A.; Burant, J. C.; Iyengar, S. S.; Tomasi, J.; Cossi, M.; Rega, N.; Millam, J. M.; Klene, M.; Knox, J. E.; Cross, J. B.; Bakken, V.; Adamo, C.; Jaramillo, J.; Gomperts, R.; Stratmann, R. E.; Yazyev, O.; Austin, A. J.; Cammi, R.; Pomelli, C.; Ochterski, J. W.; Martin, R. L.; Morokuma, K.; Zakrzewski, V. G.; Voth, G. A.; Salvador, P.; Dannenberg, J. J.; Dapprich, S.; Daniels, A. D.; Farkas, O.; Foresman, J. B.; Ortiz, J. V.; Cioslowski, J.; Fox, D. J.</span>Gaussian 09; <span class="NLM_publisher-name">Gaussian, Inc.</span>: <span class="NLM_publisher-loc">Wallingford, CT</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=M.+J.+Frisch&author=G.+W.+Trucks&author=H.+B.+Schlegel&author=G.+E.+Scuseria&author=M.+A.+Robb&author=J.+R.+Cheeseman&author=G.+Scalmani&author=V.+Barone&author=B.+Mennucci&author=G.+A.+Petersson&author=H.+Nakatsuji&author=M.+Caricato&author=X.+Li&author=H.+P.+Hratchian&author=A.+F.+Izmaylov&author=J.+Bloino&author=G.+Zheng&author=J.+L.+Sonnenberg&author=M.+Hada&author=M.+Ehara&author=K.+Toyota&author=R.+Fukuda&author=J.+Hasegawa&author=M.+Ishida&author=T.+Nakajima&author=Y.+Honda&author=O.+Kitao&author=H.+Nakai&author=T.+Vreven&author=J.+A.+Montgomery&author=J.+E.+Peralta&author=F.+Ogliaro&author=M.+Bearpark&author=J.+J.+Heyd&author=E.+Brothers&author=K.+N.+Kudin&author=V.+N.+Staroverov&author=R.+Kobayashi&author=J.+Normand&author=K.+Raghavachari&author=A.+Rendell&author=J.+C.+Burant&author=S.+S.+Iyengar&author=J.+Tomasi&author=M.+Cossi&author=N.+Rega&author=J.+M.+Millam&author=M.+Klene&author=J.+E.+Knox&author=J.+B.+Cross&author=V.+Bakken&author=C.+Adamo&author=J.+Jaramillo&author=R.+Gomperts&author=R.+E.+Stratmann&author=O.+Yazyev&author=A.+J.+Austin&author=R.+Cammi&author=C.+Pomelli&author=J.+W.+Ochterski&author=R.+L.+Martin&author=K.+Morokuma&author=V.+G.+Zakrzewski&author=G.+A.+Voth&author=P.+Salvador&author=J.+J.+Dannenberg&author=S.+Dapprich&author=A.+D.+Daniels&author=O.+Farkas&author=J.+B.+Foresman&author=J.+V.+Ortiz&author=J.+Cioslowski&author=D.+J.+Fox&title=Gaussian+09"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26jtitle%3DGaussian%252009%26pub%3DGaussian%252C%2520Inc%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group">Ditchfield, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hehre, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pople, J. A.</span><span> </span><span class="NLM_article-title">Self-Consistent Molecular-Orbital Methods 0.9. Extended Gaussian-Type Basis for Molecular-Orbital Studies of Organic Molecules</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1971</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">724</span><span class="NLM_x">–</span> <span class="NLM_lpage">728</span><span class="refDoi"> DOI: 10.1063/1.1674902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1063%2F1.1674902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaE3MXksFOiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1971&pages=724-728&author=R.+Ditchfieldauthor=W.+J.+Hehreauthor=J.+A.+Pople&title=Self-Consistent+Molecular-Orbital+Methods+0.9.+Extended+Gaussian-Type+Basis+for+Molecular-Orbital+Studies+of+Organic+Molecules&doi=10.1063%2F1.1674902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Self-consistent molecular-orbital methods.  IX.  Extended Gaussian-type basis for molecular-orbital studies of organic molecules</span></div><div class="casAuthors">Ditchfield, R.; Hehre, Warren J.; Pople, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1971</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">724-8</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An extended basis set of at. functions expressed as fixed linear combinations of Gaussian functions is presented for H and the first-row atoms C to F.  In this set. described as 4-31 G, each inner shell is represented by a single basis function taken as a sum of 4 Gaussians, and each valence orbital is split into inner and outer parts described by 3 and 1 Gaussian function, resp.  The expansion coeffs. and Gaussian exponents are detd. by minimizing the total calcd. energy of the at. ground state.  This basis set is then used in single-determinant MO studies of a group of small polyat. mols.  Optimization of valence-shell scaling factors shows that considerable rescaling of at. functions occurs in mols., the largest effects being obsd. for H and C.  However, the range of optimum scale factors for each atom is small enough to allow the selection of a std. mol. set.  The use of this std. basis gives theoretical equil. geometries in reasonable agreement with expt.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqICTNxdJ_us7Vg90H21EOLACvtfcHk0ljMibXZ83l5Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3MXksFOiuw%253D%253D&md5=abce439752b61fad3aa748515ec78c71</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1063%2F1.1674902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1674902%26sid%3Dliteratum%253Aachs%26aulast%3DDitchfield%26aufirst%3DR.%26aulast%3DHehre%26aufirst%3DW.%2BJ.%26aulast%3DPople%26aufirst%3DJ.%2BA.%26atitle%3DSelf-Consistent%2520Molecular-Orbital%2520Methods%25200.9.%2520Extended%2520Gaussian-Type%2520Basis%2520for%2520Molecular-Orbital%2520Studies%2520of%2520Organic%2520Molecules%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1971%26volume%3D54%26spage%3D724%26epage%3D728%26doi%3D10.1063%2F1.1674902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group">Kast, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kloss, T.</span><span> </span><span class="NLM_article-title">Closed-form expressions of the chemical potential for integral equation closures with certain bridge functions</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">236101</span><span class="refDoi"> DOI: 10.1063/1.3041709</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1063%2F1.3041709" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=19102559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFansb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2008&pages=236101&author=S.+M.+Kastauthor=T.+Kloss&title=Closed-form+expressions+of+the+chemical+potential+for+integral+equation+closures+with+certain+bridge+functions&doi=10.1063%2F1.3041709"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Closed-form expressions of the chemical potential for integral equation closures with certain bridge functions</span></div><div class="casAuthors">Kast, Stefan M.; Kloss, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">236101/1-236101/3</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">A general, path-independent expression is derived for the excess chem. potential of integral equation closure approxns. that contain a bridge function which depends on a renormalized indirect correlation function.  Closed-form results are obtained for various cases, among them a partial series expansion of the hypernetted chain closure.  (c) 2008 American Institute of Physics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8xHiEs2dHmbVg90H21EOLACvtfcHk0lhJPuLyP7k6Mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFansb%252FI&md5=9ce23be4d261f269e89ec6248f28615f</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1063%2F1.3041709&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.3041709%26sid%3Dliteratum%253Aachs%26aulast%3DKast%26aufirst%3DS.%2BM.%26aulast%3DKloss%26aufirst%3DT.%26atitle%3DClosed-form%2520expressions%2520of%2520the%2520chemical%2520potential%2520for%2520integral%2520equation%2520closures%2520with%2520certain%2520bridge%2520functions%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2008%26volume%3D129%26spage%3D236101%26doi%3D10.1063%2F1.3041709" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group">Humphrey, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalke, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulten, K.</span><span> </span><span class="NLM_article-title">VMD: visual molecular dynamics</span> <span class="citation_source-journal">J. Mol. Graphics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span><span class="NLM_x">–</span> <span class="NLM_lpage">38</span><span class="refDoi"> DOI: 10.1016/0263-7855(96)00018-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=10.1016%2F0263-7855%2896%2900018-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=8744570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;key=1%3ACAS%3A528%3ADyaK28Xis12nsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1996&pages=33-38&author=W.+Humphreyauthor=A.+Dalkeauthor=K.+Schulten&title=VMD%3A+visual+molecular+dynamics&doi=10.1016%2F0263-7855%2896%2900018-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">VDM: visual molecular dynamics</span></div><div class="casAuthors">Humphrey, William; Dalke, Andrew; Schulten, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-8, plates, 27-28</span>CODEN:
                <span class="NLM_cas:coden">JMGRDV</span>;
        ISSN:<span class="NLM_cas:issn">0263-7855</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">VMD is a mol. graphics program designed for the display and anal. of mol. assemblies, in particular, biopolymers such as proteins and nucleic acids.  VMD can simultaneously display any no. of structures using a wide variety of rendering styles and coloring methods.  Mols. are displayed as one or more "representations," in which each representation embodies a particular rendering method and coloring scheme for a selected subset of atoms.  The atoms displayed in each representation are chosen using an extensive atom selection syntax, which includes Boolean operators and regular expressions.  VMD provides a complete graphical user interface for program control, as well as a text interface using the Tcl embeddable parser to allow for complex scripts with variable substitution, control loops, and function calls.  Full session logging is supported, which produces a VMD command script for later playback.  High-resoln. raster images of displayed mols. may be produced by generating input scripts for use by a no. of photorealistic image-rendering applications.  VMD has also been expressly designed with the ability to animate mol. dynamics (MD) simulation trajectories, imported either from files or from a direct connection to a running MD simulation.  VMD is the visualization component of MDScope, a set of tools for interactive problem solving in structural biol., which also includes the parallel MD program NAMD, and the MDCOMM software used to connect the visualization and simulation programs, VMD is written in C++, using an object-oriented design; the program, including source code and extensive documentation, is freely available via anonymous ftp and through the World Wide Web.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0dd54QvBO97Vg90H21EOLACvtfcHk0lhJPuLyP7k6Mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xis12nsrg%253D&md5=1e3094ec3151fb85c5ff05f8505c78d5</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1016%2F0263-7855%2896%2900018-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0263-7855%252896%252900018-5%26sid%3Dliteratum%253Aachs%26aulast%3DHumphrey%26aufirst%3DW.%26aulast%3DDalke%26aufirst%3DA.%26aulast%3DSchulten%26aufirst%3DK.%26atitle%3DVMD%253A%2520visual%2520molecular%2520dynamics%26jtitle%3DJ.%2520Mol.%2520Graphics%26date%3D1996%26volume%3D14%26spage%3D33%26epage%3D38%26doi%3D10.1016%2F0263-7855%2896%2900018-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D12" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D12','PDB','5D12'); return false;">PDB: 5D12</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D11" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D11','PDB','5D11'); return false;">PDB: 5D11</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D10" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D10','PDB','5D10'); return false;">PDB: 5D10</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ika" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ika','PDB','3ika'); return false;">PDB: 3ika</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3IKA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3IKA','PDB','3IKA'); return false;">PDB: 3IKA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HWP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HWP','PDB','2HWP'); return false;">PDB: 2HWP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OIQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OIQ','PDB','2OIQ'); return false;">PDB: 2OIQ</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i33"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01082">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_51503"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b01082">10.1021/acs.jmedchem.5b01082</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Structures of <b>5b</b>, <b>5n</b>, <b>6a</b>, WZ4002, and WZ4002-H in Jmol format (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01082/suppl_file/jm5b01082_si_001.zip">ZIP</a>).</p></li><li><p class="inline">Synthesis of pyrimidine-based <b>6e</b> (Scheme S1); synthesis of pyrimidine-based <b>7</b> (Scheme S2); mass spectrometry experiments (Figure S1); IC<sub>50</sub> determinations of cSrc and mutant variants (Table S1); X-ray data collection and refinement statistics (cSrc-DM–<b>1a</b>, −<b>5b</b>, and −<b>5c</b>) (Table S2); solubility and cellular permeability parameters of <b>5b</b> and <b>6a</b> (Table S3); nano-LC-MS/MS base peak chromatograms of <b>5b</b> and <b>6a</b> after proteolytic digestion (Figure S2); nano-LC-MS/MS fragment ion spectra of GCLLDYVR <b>5b</b> and <b>6a</b> (Figure S3); kinetic characterization of <b>5a</b> and <b>6b</b> by determination of <i>k</i><sub>inact</sub>/<i>K</i><sub>i</sub> (Figure S4); expression and purification of cSrc and its mutant variants; activity-based assay for IC<sub>50</sub> determination of cSrc and mutant variants thereof; and experimental section (chemistry) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01082/suppl_file/jm5b01082_si_002.pdf">PDF</a>).</p></li><li><p class="inline">IC<sub>50</sub> of compounds against wt and mutant EGFR (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01082/suppl_file/jm5b01082_si_003.xlsx">XLSX</a>).</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01082/suppl_file/jm5b01082_si_001.zip">jm5b01082_si_001.zip (21.14 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01082/suppl_file/jm5b01082_si_002.pdf">jm5b01082_si_002.pdf (1.51 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01082/suppl_file/jm5b01082_si_003.xlsx">jm5b01082_si_003.xlsx (18.09 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b01082&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2015.58.issue-17%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01082%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b01082" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677a5e66397e1955","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
