
<html lang="en"     class="pb-page"  data-request-id="5c68e21a-ba7c-46b4-af95-932c736e6462"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;article:article:10.1021/acs.jmedchem.1c00024;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2021.64.issue-13;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Fragment-Derived Selective Inhibitors of Dual-Specificity Kinases DYRK1A and DYRK1B" /></meta><meta name="dc.Creator" content="David  Lee Walmsley" /></meta><meta name="dc.Creator" content="James B.  Murray" /></meta><meta name="dc.Creator" content="Pawel  Dokurno" /></meta><meta name="dc.Creator" content="Andrew J.  Massey" /></meta><meta name="dc.Creator" content="Karen  Benwell" /></meta><meta name="dc.Creator" content="Andrea  Fiumana" /></meta><meta name="dc.Creator" content="Nicolas  Foloppe" /></meta><meta name="dc.Creator" content="Stuart  Ray" /></meta><meta name="dc.Creator" content="Julia  Smith" /></meta><meta name="dc.Creator" content="Allan E.  Surgenor" /></meta><meta name="dc.Creator" content="Thomas  Edmonds" /></meta><meta name="dc.Creator" content="Didier  Demarles" /></meta><meta name="dc.Creator" content="Mike  Burbridge" /></meta><meta name="dc.Creator" content="Francisco  Cruzalegui" /></meta><meta name="dc.Creator" content="Andras  Kotschy" /></meta><meta name="dc.Creator" content="Roderick E.  Hubbard" /></meta><meta name="dc.Description" content="The serine/threonine kinase DYRK1A has been implicated in regulation of a variety of cellular processes associated with cancer progression, including cell cycle control, DNA damage repair, protecti..." /></meta><meta name="Description" content="The serine/threonine kinase DYRK1A has been implicated in regulation of a variety of cellular processes associated with cancer progression, including cell cycle control, DNA damage repair, protecti..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 18, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00024" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 The Authors. Published by American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00024" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00024" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00024" /></link>
        
    
    

<title>Fragment-Derived Selective Inhibitors of Dual-Specificity Kinases DYRK1A and DYRK1B | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00024" /></meta><meta property="og:title" content="Fragment-Derived Selective Inhibitors of Dual-Specificity Kinases DYRK1A and DYRK1B" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0025.jpeg" /></meta><meta property="og:description" content="The serine/threonine kinase DYRK1A has been implicated in regulation of a variety of cellular processes associated with cancer progression, including cell cycle control, DNA damage repair, protection from apoptosis, cell differentiation, and metastasis. In addition, elevated-level DYRK1A activity has been associated with increased severity of symptoms in Down’s syndrome. A selective inhibitor of DYRK1A could therefore be of therapeutic benefit. We have used fragment and structure-based discovery methods to identify a highly selective, well-tolerated, brain-penetrant DYRK1A inhibitor which showed in vivo activity in a tumor model. The inhibitor provides a useful tool compound for further exploration of the effect of DYRK1A inhibition in models of disease." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00024"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00024">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00024&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00024&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00024&amp;href=/doi/10.1021/acs.jmedchem.1c00024" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 13</span><span class="cit-fg-pageRange">, 8971-8991</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/13" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00004" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00093" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Fragment-Derived Selective Inhibitors of Dual-Specificity Kinases DYRK1A and DYRK1B</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">David Lee Walmsley</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David Lee Walmsley</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vernalis (R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David++Lee+Walmsley">David Lee Walmsley</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">James B. Murray</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James B. Murray</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vernalis (R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+B.++Murray">James B. Murray</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-1007-8218" title="Orcid link">https://orcid.org/0000-0003-1007-8218</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Pawel Dokurno</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pawel Dokurno</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vernalis (R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pawel++Dokurno">Pawel Dokurno</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-7332-8889" title="Orcid link">https://orcid.org/0000-0002-7332-8889</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrew J. Massey</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrew J. Massey</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vernalis (R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrew+J.++Massey">Andrew J. Massey</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-0276-2573" title="Orcid link">https://orcid.org/0000-0002-0276-2573</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Karen Benwell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Karen Benwell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vernalis (R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Karen++Benwell">Karen Benwell</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrea Fiumana</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrea Fiumana</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vernalis (R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrea++Fiumana">Andrea Fiumana</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicolas Foloppe</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicolas Foloppe</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vernalis (R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicolas++Foloppe">Nicolas Foloppe</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stuart Ray</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stuart Ray</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vernalis (R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stuart++Ray">Stuart Ray</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Julia Smith</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Julia Smith</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vernalis (R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Julia++Smith">Julia Smith</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Allan E. Surgenor</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Allan E. Surgenor</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vernalis (R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Allan+E.++Surgenor">Allan E. Surgenor</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-9869-1430" title="Orcid link">https://orcid.org/0000-0002-9869-1430</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Thomas Edmonds</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas Edmonds</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institut de Recherches Servier, 125 Chemin de Ronde, Croissy-sur-Seine 78290, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas++Edmonds">Thomas Edmonds</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Didier Demarles</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Didier Demarles</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Technologie Servier, 27 Rue Eugène Vignat, Orleans 45000, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Didier++Demarles">Didier Demarles</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mike Burbridge</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mike Burbridge</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institut de Recherches Servier, 125 Chemin de Ronde, Croissy-sur-Seine 78290, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mike++Burbridge">Mike Burbridge</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Francisco Cruzalegui</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Francisco Cruzalegui</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institut de Recherches Servier, 125 Chemin de Ronde, Croissy-sur-Seine 78290, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Francisco++Cruzalegui">Francisco Cruzalegui</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andras Kotschy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andras Kotschy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Servier Research Institute of Medicinal Chemistry, Záhony u. 7., Budapest H-1031, Hungary</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andras++Kotschy">Andras Kotschy</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-7675-3864" title="Orcid link">https://orcid.org/0000-0002-7675-3864</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Roderick E. Hubbard</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Roderick E. Hubbard</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vernalis (R&D) Ltd., Granta Park, Cambridge CB21 6GB, U.K.</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#31431f59445353504355714754435f505d58421f525e5c"><span class="__cf_email__" data-cfemail="f98bd7918c9b9b988b9db98f9c8b979895908ad79a9694">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Roderick+E.++Hubbard">Roderick E. Hubbard</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-8233-7461" title="Orcid link">https://orcid.org/0000-0002-8233-7461</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00024&amp;href=/doi/10.1021%2Facs.jmedchem.1c00024" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 13</span><span class="cit-pageRange">, 8971–8991</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 18, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>7 January 2021</li><li><span class="item_label"><b>Published</b> online</span>18 June 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 July 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00024" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00024</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 The Authors. Published by American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="https://acsopenscience.org/open-access/licensing-options/"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by-nc"><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_cc-1612971093887.svg" alt="CC: Creative Commons" /></img><span><strong>CC: </strong>Creative Commons</span></a><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_by-1612971093257.svg" alt="BY: Credit must be given to the creator" /></img><span><strong>BY: </strong>Credit must be given to the creator</span></a><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_nc-1612971093307.svg" alt="NC: Only noncommercial uses of the work are permitted" /></img><span><strong>NC: </strong>Only noncommercial uses of the work are permitted</span></a><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_nd-1612971093637.svg" alt="ND: No derivatives or adaptations of the work are permitted" /></img><span><strong>ND: </strong>No derivatives or adaptations of the work are permitted</span></a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00024"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2747</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00024" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Fragment-Derived Selective Inhibitors of Dual-Specificity Kinases DYRK1A and DYRK1B&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Lee Walmsley&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;B. Murray&quot;},{&quot;first_name&quot;:&quot;Pawel&quot;,&quot;last_name&quot;:&quot;Dokurno&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;J. Massey&quot;},{&quot;first_name&quot;:&quot;Karen&quot;,&quot;last_name&quot;:&quot;Benwell&quot;},{&quot;first_name&quot;:&quot;Andrea&quot;,&quot;last_name&quot;:&quot;Fiumana&quot;},{&quot;first_name&quot;:&quot;Nicolas&quot;,&quot;last_name&quot;:&quot;Foloppe&quot;},{&quot;first_name&quot;:&quot;Stuart&quot;,&quot;last_name&quot;:&quot;Ray&quot;},{&quot;first_name&quot;:&quot;Julia&quot;,&quot;last_name&quot;:&quot;Smith&quot;},{&quot;first_name&quot;:&quot;Allan&quot;,&quot;last_name&quot;:&quot;E. Surgenor&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;Edmonds&quot;},{&quot;first_name&quot;:&quot;Didier&quot;,&quot;last_name&quot;:&quot;Demarles&quot;},{&quot;first_name&quot;:&quot;Mike&quot;,&quot;last_name&quot;:&quot;Burbridge&quot;},{&quot;first_name&quot;:&quot;Francisco&quot;,&quot;last_name&quot;:&quot;Cruzalegui&quot;},{&quot;first_name&quot;:&quot;Andras&quot;,&quot;last_name&quot;:&quot;Kotschy&quot;},{&quot;first_name&quot;:&quot;Roderick&quot;,&quot;last_name&quot;:&quot;E. Hubbard&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;18&quot;,&quot;issue&quot;:&quot;13&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;8971-8991&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00024&quot;},&quot;abstract&quot;:&quot;The serine/threonine kinase DYRK1A has been implicated in regulation of a variety of cellular processes associated with cancer progression, including cell cycle control, DNA damage repair, protection from apoptosis, cell differentiation, and metastasis. In addition, elevated-level DYRK1A activity has been associated with increased severity of symptoms in Down’s syndrome. A selective inhibitor of DYRK1A could therefore be of therapeutic benefit. We have used fragment and structure-based discovery methods to identify a highly selective, well-tolerated, brain-penetrant DYRK1A inhibitor which showed in vivo activity in a tumor model. The inhibitor provides a useful tool compound for further exploration of the effect of DYRK1A inhibition in models of disease.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00024&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00024" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00024&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00024" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00024&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00024" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00024&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00024&amp;href=/doi/10.1021/acs.jmedchem.1c00024" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00024" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00024" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (10 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00024%26sid%3Dliteratum%253Aachs%26pmid%3D34143631%26genre%3Darticle%26aulast%3DLee%2BWalmsley%26date%3D2021%26atitle%3DFragment-Derived%2BSelective%2BInhibitors%2Bof%2BDual-Specificity%2BKinases%2BDYRK1A%2Band%2BDYRK1B%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D13%26spage%3D8971%26epage%3D8991%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/13" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/jmcmar.2021.64.issue-13/20210708/jmcmar.2021.64.issue-13.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0025.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The serine/threonine kinase DYRK1A has been implicated in regulation of a variety of cellular processes associated with cancer progression, including cell cycle control, DNA damage repair, protection from apoptosis, cell differentiation, and metastasis. In addition, elevated-level DYRK1A activity has been associated with increased severity of symptoms in Down’s syndrome. A selective inhibitor of DYRK1A could therefore be of therapeutic benefit. We have used fragment and structure-based discovery methods to identify a highly selective, well-tolerated, brain-penetrant DYRK1A inhibitor which showed in vivo activity in a tumor model. The inhibitor provides a useful tool compound for further exploration of the effect of DYRK1A inhibition in models of disease.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67616" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67616" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">DYRK1A is the best-studied member of the evolutionarily conserved DYRK (dual-specificity tyrosine-regulated kinase) branch of the cyclin-dependent, mitogen-activated, glycogen synthase, and CDC-like (CMGC) family of serine/threonine kinases that consists of two classes: DYRK1A and DYRK1B (Class 1) and DYRK2, DYRK3, and DYRK4 (Class 2). DYRK1A is activated by self-phosphorylation of S520, leading to 14-3-3β binding which stimulates the catalytic activity.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a></div><div class="NLM_p">DYRK1A and 1B play an important role in cell cycle control and modulation of DYRK1A/B in sarcoma,<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> and lung,<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> pancreatic,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> and ovarian cancer<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> can result in dysregulation of cell cycle control. These kinases regulate the induction of quiescence via control of the DREAM complex,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> thereby controlling the G0-G1 transition and exit from quiescence.<a onclick="showRef(event, 'ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10">(8−10)</a> DYRK1A and 1B can also phosphorylate Cyclin D1 (resulting in its destabilization)<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> and p27Kip1,<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> thereby altering the cell’s decision to enter the cell cycle. Additionally, DYRK1A kinase activity stabilizes epidermal growth factor (EGFR) in glioblastoma through regulation of Sprouty2, and inhibition of DYRK1A reduces EGFR-dependent glioblastoma growth.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> DYRK1A and 1B control the activity of the Hedgehog signaling effector GLI through direct phosphorylation, resulting in increased nuclear accumulation and transcriptional activity.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Pharmacological inhibition of DYRK1B repressed Hedgehog signaling and inhibited the proliferation of GLI1-dependent pancreatic cancer cells.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Recent work suggests that suppression of DYRK1A restrains myeloid cell leukemia 1 (MCL-1) expression and sensitizes nonsmall cell lung cancer cells to B-cell lymphoma 2 (Bcl-2) inhibitors.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Of the Class 2 DYRKs, DYRK2 has been implicated to have multiple roles in cancer development including a role in the G1/S and G2/M cell cycle transition, epithelial–mesenchymal transition, cancer stemness, and DNA damage response through p53 phosphorylation. In lower eukaryotes, DYRK2 has been implicated in tissue development, thereby implying a role for this kinase in cell differentiation and homeostasis in mammals.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> These observations suggest that targeting DYRK kinase activity could be of therapeutic benefit for patients presenting with a diverse range of cancers.</div><div class="NLM_p">There has also been interest in DYRK1A in patients with Down’s syndrome. The gene is located on chromosome 21, and the subsequent elevated levels of DYRK1A in Down’s syndrome patients has been correlated with an increase in severity of symptoms and a poorer prognosis.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Recently, it has been suggested that a natural product inhibitor of DYRK1A, the polyphenol from green tea, epigallocatechin gallate (EGCG), has efficacy in a murine model of Down’s syndrome through chromosome 16 trisomy.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Although the wider activity, instability, and poor compound properties of EGCG complicate interpretation of these results, inhibiting DYRK kinase activity could also be of therapeutic benefit to Down’s syndrome patients.</div><div class="NLM_p">In addition to Down’s syndrome, recent studies have suggested a role for DYRK1A in other neurodegenerative disorders such as Alzheimer’s disease (including early onset AD in Down’s syndrome patients<a onclick="showRef(event, 'ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25">(23−25)</a>) and Parkinson’s disease.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> DYRK1A has also been demonstrated to regulate the proliferation and cell size of beta cells (and therefore their regenerative capacity) in the pancreas<a onclick="showRef(event, 'ref28 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30">(28−30)</a>.</div><div class="NLM_p">Our objective was to use fragment and structure-based lead discovery methods to identify potent and selective adenosine triphosphate (ATP) competitive inhibitors of DYRK1A and DYRK1B to be used to probe the therapeutic benefit of inhibiting this class of kinase. Our work was initiated concurrently to the release of the first structure of the kinase domain of DYRK1A (protein databank (PDB) code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VX3">2VX3</a><a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a>) by the Structural Genomics Consortium in 2009. Here, we describe the initial fragment-based hit discovery efforts that identified multiple series of hit compounds and the lead optimization aided by molecular modeling that generated pyrrolopyrimidine compounds that were profiled in several cellular and <i>in vivo</i> models. An accompanying paper<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> describes a companion optimization project to generate imidazopyridine compounds that were similarly profiled. DYRK1A and DYRK1B vary by only one amino acid in the ATP-binding pocket; the Class 2 DYRKs have more variation.</div><div class="NLM_p last">The lead molecule, <b>34</b>, from this pyrrolopyrimidine hit series inhibited DYRK1A and 1B with low nM potency and exhibited excellent selectivity versus the Class 2 DYRKs. It potently inhibited DYRK1A autophosphorylation in a cellular model and had good oral bioavailability with a robust pharmacokinetic (PK)/pharmacodynamic (PD) relationship with <i>in vivo</i> activity in a glioblastoma xenograft model. This molecule is a potentially useful tool compound for further study of the effect of DYRK1A inhibition in models of human disease.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52800" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52800" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Fragment screening and structural studies used a construct of the DYRK1A kinase domain (hereafter termed Dyrk1a) similar to that used to generate the crystal structure deposited with PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2vx3">2vx3</a><a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> (see the <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for details). The kinase p21-associated kinase 4 (4PAK4) was used as a counterscreen throughout the project, and selectivity against other kinases was measured as the project progressed. The inhibition of a selection of protein kinases was measured using either an adenosine diphosphate (ADP) Hunter or time-resolved fluorescence energy transfer (TR-FRET) assay, and the inhibition of GSK3-β was measured by a radiometric assay (details in the <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a>).</div><div class="NLM_p">We solved the crystal structure of Dyrk1a with 5′-adenylyl-β,γ-imidodiphosphate (ADPNP) bound. <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> shows a comparison between this structure of Dyrk1a and the published structure of PAK4 with ADPNP.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (a) Ribbon structure of Dyrk1a, highlighting hinge (red) and ADPNP bound in the stick (monomer B in the crystal structure, PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A4O">7A4O</a>) and the portion of the structure (light blue) removed in all subsequent figures (residues E160 to D178 which include the glycine loop). (b) Closeup of the ATP binding site showing side by side ADPNP bound to Dyrk1a (main chain white) and PAK4 (main chain pink) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FIG">4FIG</a>). (c) Standard orientation and convention (unless stated) used for figures of compounds (element coloring with purple carbons) binding to Dyrk1a—remove the glycine loop from E160 to D178 to view compound binding, peptide backbone as a ribbon structure for the remainder of the protein except for Dyrk1a F238 to N244 (hinge region) and V306 to F308 with additional selected side chains and water molecules (none in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) colored on the element in stick (gray carbons) and protein backbone and side chain hydrogen bonds as blue and green dashed lines with nonstandard interactions in orange dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Dyrk1a has the conventional fold of a protein kinase with a larger, predominantly α-helical subdomain connected to a predominantly β-sheet subdomain, forming a cleft that binds ATP. The subdomains are connected by a so-called “hinge” region where the main chain atoms hydrogen-bond to the donor–acceptor–donor motif of the adenosine moiety of ATP, positioning the sugar and phosphate groups adjacent to the catalytic machinery. The structure reveals an unusual conformation for the main chain carbonyl of L241 in Dyrk1a compared to that seen for the equivalent amino acid (L398) in PAK4 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b) and other kinases (data not shown).</div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Identifying and Characterizing the Initial Fragment Hits</h3><div class="NLM_p">The Vernalis fragment library consisting of 1063 compounds at the time<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> was screened using three ligand-observed nuclear magnetic resonance (NMR) experiments<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> for binding to the ATP-binding site of Dyrk1a in competition with the nonspecific kinase inhibitor, staurosporine. Compounds that were confirmed as binding competitively in all three NMR experiments were assayed in an ADP Hunter biochemical assay (hereafter the functional assay) to generate 166 fragment hits, a hit rate of 15%.</div><div class="NLM_p"><a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> lists the 8 hits with the highest ligand efficiency [LE,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> defined as free energy of binding (Δ<i>G</i>)/heavy atom count (HAC)]. A high hit rate in an NMR fragment screen indicates<a onclick="showRef(event, 'ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref38 ref39">(38,39)</a> a highly efficient site for binding small molecules (often termed druggable<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a>), and the high LE of the hits reinforces this for the ATP binding site in Dyrk1a. In addition, these fragment hits show remarkable selectivity, given some contain relatively standard kinase ATP binding motifs. This can be explained by inspection of the Dyrk1a structure (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>c) which shows a distinctive conformation for the carbonyl of the hinge amino acid, L241.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Fragment Hits Ranked by Ligand Efficiency Measured in ADP Hunter Assay</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0014.gif" alt="" id="gr13" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">% inhibition in PAK4 assay with the 200 μM (400 μM) compound; binding pose in the crystal structure in the same orientation as <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>c. See <a href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_001.pdf" class="ext-link">Tables S1 and S2</a> for structure determination and refinement statistics; the accession codes for coordinates deposited to the PDB are listed at the end of the manuscript. All 2D chemical structures are oriented to approximate the binding pose seen in crystal structures.</p></div></div><div></div></div><div class="NLM_p">We determined the crystal structures for the eight fragment hits bound to Dyrk1a (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_001.pdf" class="ext-link">S1 and S2</a>). The fragments exhibited both “classical” hinge binding with a donor–acceptor–donor hydrogen bonding motif and “nonclassical” where as few as one hydrogen bond anchors the fragment to the hinge in the binding site. This diversity of binding modes and motifs demonstrates one of the benefits of fragment-based discovery: even for the widely studied ATP binding site of a kinase, a fragment screen can identify compounds that explore features that can be exploited for selectivity, identify novel binding modes that are not often discovered when screening larger compounds, and provide a wide diversity of hit compounds from which the medicinal chemist can choose the most appropriate one for optimization.</div><div class="NLM_p">As the fragment screen was completed, the crystal structure (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ANR">3ANR</a>) and properties of harmine (<b>9</b>) on Dyrk1a were published<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> and the activity of meriolin derivatives was reported.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a></div><div class="NLM_p"><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> compares the binding of harmine (<b>9</b>) and meriolin-3 (<b>10</b>) to Dyrk1a with some of the fragment hits. As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a, fragment <b>3</b> and harmine both have a “nonclassical” binding mode, making just one hydrogen bond to the hinge and with the methyl group of the methoxy in the pocket formed by the unusual conformation of the main chain of L241 and one to E203. This fragment is very similar to fragment <b>4</b>. The optimization of fragment <b>3</b> with a similar binding mode to harmine is described in a separate publication.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The fragment <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b) makes a “classical” set of hydrogen bonds with the hinge, overlapping with the core of <b>10</b> which also interacts with E203 through the NH<sub>2</sub> of the pendant amino pyrimidine (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b). Fragment <b>5</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c) was selected for further study as it makes a comprehensive set of interactions not only with the hinge but also with the catalytic residues at the base of the binding site through a well-defined network of hydrogen bonds to water.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Detailed comparison of the binding mode of literature compounds and selected fragments to Dyrk1a showing the protein structure (white ribbon), selected waters, and key interactions with the fragment (a) harmine (<b>9</b>—green carbons, PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ANR">3ANR</a>) and fragment <b>3</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A4W">7A4W</a>); (b) meriolin-3 (<b>10</b>—green carbons, PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A5B">7A5B</a>) and fragment <b>1</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A4R">7A4R</a>); and (c) fragment <b>5</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A51">7A51</a>). See the legend to <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> for conventions used in the representation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Optimization from the Fragment to Lead Pyrrolopyrimidine Series</h3><div class="NLM_p">The first step of optimization was to exploit the pocket proximal to the carbonyl of L241. Inspection of the crystal structures (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) suggested that a methyl group could be incorporated into the 2-position of the pyrrolopyrimidine scaffold of <b>5</b>. As can be seen in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, this improved the affinity of compounds for DYRK1A and retained selectivity over PAK4. We also investigated substitution of the 6-position bromine with various aryls and heteroaryls to improve potency further by building groups that replaced the solvent to interact with K188 and E203.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Improving Interactions with the Carbonyl of L241 and the Catalytic Site; ND—Not Determined</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0015.gif" alt="" id="GRAPHIC-d7e636-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0016.gif" alt="" id="gr14" /></img><div></div></div><div class="NLM_p">The crystal structure (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) of the most ligand-efficient compound of the series (<b>16</b>) bound to Dyrk1a confirmed the success of the structure-based design and the structural basis for retention of good selectivity against PAK4. The 2-methyl occupies the pocket adjacent to carbonyl of L241, and the pyridine nitrogen hydrogen-bonds to K188 with the other polar atoms of the ligand making interactions with defined solvent positions.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structure of <b>16</b> bound to Dyrk1a (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A5D">7A5D</a>). See the legend to <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> for conventions used in the representation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">It was noted in the structures of <b>10</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b) and the most ligand efficient fragment (<b>1</b>) that the 4-amino group of the pyrimidine ring of <b>16</b> is within a pyrrole ring, making a “classical” hinge binding interaction with the carbonyl of E239. A series of 2-methyl-5-aryl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidines were therefore synthesized as listed in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, keeping the methyl group to target the L241 pocket and designed to retain the main interactions with the remainder of the binding site. A 4-position methoxy was also included for further elaboration.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Changing to a Pyrrolo[2,3-<i>d</i>]pyrimidine Scaffold</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0017.gif" alt="" id="GRAPHIC-d7e699-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0018.gif" alt="" id="gr15" /></img><div></div></div><div class="NLM_p">The three most potent compounds (<b>18</b>, <b>19</b>, and <b>20</b>) were profiled for their inhibition of a mini-panel of kinases (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Selectivity in a Small Assay Panel of Protein Kinases (IC<sub>50</sub> μM)</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd no</th><th class="colsep0 rowsep0" align="center" char=".">ADP DYRK1A</th><th class="colsep0 rowsep0" align="center">TR FRET DYRK1A</th><th class="colsep0 rowsep0" align="center">TR FRET DYRK1B</th><th class="colsep0 rowsep0" align="center">ADP DYRK2</th><th class="colsep0 rowsep0" align="center">TR FRET CDK9</th><th class="colsep0 rowsep0" align="center" char=".">ADP PAK4</th><th class="colsep0 rowsep0" align="center" char=".">ADP Pim1</th><th class="colsep0 rowsep0" align="center" char=".">ADP Aurora A</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="char" char=".">0.05</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">>50</td><td class="colsep0 rowsep0" align="char" char=".">>50</td><td class="colsep0 rowsep0" align="char" char=".">>50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="char" char=".">0.02</td><td class="colsep0 rowsep0" align="left">0.02</td><td class="colsep0 rowsep0" align="left">0.02</td><td class="colsep0 rowsep0" align="left">0.06</td><td class="colsep0 rowsep0" align="left">4.4</td><td class="colsep0 rowsep0" align="char" char=".">>50</td><td class="colsep0 rowsep0" align="char" char=".">2.8</td><td class="colsep0 rowsep0" align="char" char=".">>50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="char" char=".">0.04</td><td class="colsep0 rowsep0" align="left">0.03</td><td class="colsep0 rowsep0" align="left">0.04</td><td class="colsep0 rowsep0" align="left">0.30</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="char" char=".">>50</td><td class="colsep0 rowsep0" align="char" char=".">9.6</td><td class="colsep0 rowsep0" align="char" char=".">>50</td></tr></tbody></table></div></div><div class="NLM_p">The most potent compound (<b>19</b>) combines high target affinity (and LE of 0.57), almost equipotent inhibition of the other profiled DYRK enzymes and with good selectivity against other kinases. The 4-position methoxy on the pyrimidine presents a vector toward the amino acids in the sugar pocket of the ATP site and the glycine loop regions. A selection of single-ring aromatic moieties were synthesized at this 4-position (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>).</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. SAR and Selectivity of O-Linked Aryls at 4-Position of the Pyrimidine Ring (IC<sub>50</sub> μM)</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0019.gif" alt="" id="GRAPHIC-d7e876-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0020.gif" alt="" id="gr16" /></img><div></div></div><div class="NLM_p">Simple aromatics such as thiophene (<b>24</b>) achieve single-digit nanomolar potency in the DYRK1A functional assay, more than 2 orders of magnitude greater than inhibition of Gsk3 β and cyclin-dependent kinase 9 (CDK9). The crystal structure of <b>24</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>) bound to Dyrk1a confirmed the predicted interactions that were made with additional Van Der Waals interactions made in a lipophilic region under the glycine loop (not shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>) contributed to by G168 and the side chains of I165, F170, V173, and K188. However, the 4-alkoxy compounds were unstable in aqueous media at pH < 5 over a relatively short period of a few hours due to hydrolysis of the pendant methoxy (as analyzed by liquid chromatography–mass spectrometry, LCMS), presumably via protonation of the pyrimidine ring.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structure of <b>24</b> bound to Dyrk1a (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A5L">7A5L</a>). See the legend to <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> for conventions used in the representation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Similar benzylic motifs were introduced to make glycine loop interactions by installing benzylamines at the 4-position. A number of 5-membered heterocycles had similar potency and selectivity (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>) and were found to be stable under a range of aqueous conditions.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. SAR and Selectivity of Aryls at 4-Amino Position (IC<sub>50</sub> μM)</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0021.gif" alt="" id="GRAPHIC-d7e927-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0022.gif" alt="" id="gr17" /></img><div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Lead Optimization to the Compound Suitable for <i>in Vivo</i> Studies</h3><div class="NLM_p">The next stage in optimization was to investigate further substituents at the 4-position of the pyrimidine aromatic ring. A number of analogues were synthesized, and a biaryl compound (<b>37</b>) was introduced by direct coupling. These were all successful in retaining potency but with an altered selectivity profile, particularly against DYRK2. The cellular activity and metabolic stability of this set of compounds were also profiled (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>).</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Investigation of the SAR and Cellular Activity of Aryls at the 4-Position</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0023.gif" alt="" id="GRAPHIC-d7e950-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0024.gif" alt="" id="gr18" /></img><div></div></div><div class="NLM_p">As can be seen in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, introduction of ortho substituents on the pendant benzyl ring induces a twist of the aromatic to be perpendicular to the plane of the pyrrolopyrimidine core. This twist displaces a water molecule in the structure. <b>34</b> was taken as the most promising lead compound. Although the metabolic stability is moderate, it shows good cellular activity with the IC<sub>50</sub> for inhibition of the autophosphorylation of S520 on DYRK1A in U2OS cells only threefold less than the IC<sub>50</sub> in the functional assay and with an EC<sub>50</sub> of less than 1 nM in a nano-BRET assay (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_001.pdf" class="ext-link">Figure S1</a>). These results are consistent with the cell permeability properties of the compound, measured in a Caco2 assay (see <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). In addition, the very high plasma protein binding (unbound fraction <i>f</i><sub>u</sub> <1% in mouse and human plasma) should mitigate the measured effect on human ether-a-go-go-related gene (hERG), suggesting that the compound was suitable for further <i>in vitro</i> and <i>in vivo</i> profiling.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structure of <b>34</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A5N">7A5N</a>) bound to Dyrk1a. (a) See the legend in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> for conventions used in the representation. (b). Alternate view to show hydrophobic interactions with underside of the glycine loop.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> Selectivity</h4><div class="NLM_p last">The selectivity profile for inhibition of other protein kinases by <b>34</b> was characterized in an Ambit kinase assay which measured the percentage activity remaining for each of 442 kinases (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_001.pdf" class="ext-link">Table S4</a>). Only two kinases were inhibited to less than 10% activity at 10 nM compound, DYRK1A (0.4%) and DYRK1B (5%), and only eight other kinases to less than 90% activity, most notably CLK1 (24%), CLK2 (10%), CLK4 (23%), and HIPK2 (53%), showing excellent selectivity against DYRK2 (88%). At 1 μM compound, only 15 kinases were inhibited to less than 50% activity. Although IC<sub>50</sub> values for compounds inhibiting a particular kinase can be compared, it is more difficult to compare such values across enzymes where there will be different <i>K</i><sub>M</sub>s for ATP, substrate, and assay conditions. We therefore determined the <i>K</i><sub>d</sub> for <b>34</b> binding to DYRK1A, DYRK1B, CLK1, CLK2, and CLK4 in a DiscoveRx Kinomescan binding assay which gave values of 0.24, 0.05, 2.6, 0.9, and 0.8 nM, respectively, confirming the selectivity for DYRK1A and the high affinity for CLK kinases. An overlay (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_001.pdf" class="ext-link">Figure S2</a>) of a published structure of CLK1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q8K">6Q8K</a><a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a>) with that of <b>34</b> bound to Dyrk1a shows that L244 in CLK1 (equivalent to L241 in Dyrk1a) adopts a similar conformation, providing some explanation for the observed selectivity, which is a feature seen for other DYRK1A inhibitors.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div></div><div id="sec2_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> Brain Permeability</h4><div class="NLM_p last">Preliminary assessment of oral versus iv PK (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_001.pdf" class="ext-link">Figure S3</a>) showed that the compound has moderate oral bioavailability in the mouse (% <i>F</i> = 33). Three animals (female SCID mice) were dosed with compound <b>34</b> at po 25 mg/kg, and the mean plasma and brain concentrations (2 h after dosing) of the compound were measured as 1849 and 133 ng/mL, respectively (see <a href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_001.pdf" class="ext-link">Table S5</a>). The calculated brain to plasma partition was 0.1.</div></div><div id="sec2_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> ADME-PK</h4><div class="NLM_p">Female nude BALB/c mice were dosed po at 25 and 100 mg/kg, and the plasma concentration of <b>34</b> was measured as shown in <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a> (details in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_001.pdf" class="ext-link">Table S6</a>). The compound was absorbed slowly, giving a Cmax a few hours after dosing and showing moderate exposure.</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Mean Plasma Concentration ± SD in ng/mL of <b>34</b> Measured after po Administration (Three Animals/Time Point)</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="±">0.5 h</th><th class="colsep0 rowsep0" align="center" char="±">2h</th><th class="colsep0 rowsep0" align="center" char="±">6h</th><th class="colsep0 rowsep0" align="center">16h</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">25 mpk</td><td class="colsep0 rowsep0" align="char" char="±">850 ± 246</td><td class="colsep0 rowsep0" align="char" char="±">1828 ± 805</td><td class="colsep0 rowsep0" align="char" char="±">451 ± 503</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">100 mpk</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">12176 ± 7809</td><td class="colsep0 rowsep0" align="char" char="±">15177 ± 2430</td><td class="colsep0 rowsep0" align="left">322 ± 503</td></tr></tbody></table></div></div></div><div id="sec2_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> <i>In vivo</i> Efficacy</h4><div class="NLM_p">These properties of <b>34</b>, namely, kinase selectivity (particularly against DYRK2), moderate exposure, and brain penetration, and the fact that the compound is tolerated <i>in vivo</i> make <b>34</b> a suitable tool for probing the potential of DYRK1A/1B inhibition in various disease models, keeping in mind potential interference from inhibition of the CLK family at high concentrations. In the study reported here, the <i>in vivo</i> efficacy was assessed in a human xenograft tumor model.</div><div class="NLM_p">Previous work has demonstrated that glioblastoma cells are dependent on DYRK1A for growth, both <i>in vitro</i> and <i>in vivo</i>.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Compound <b>34</b> inhibited the proliferation of U87MG glioblastoma cells grown either as a 2D monolayer (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_001.pdf" class="ext-link">Figure S4A</a>) or as multicellular tumor spheroids (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_001.pdf" class="ext-link">Figure S4B</a>). Activity was dependent on serum concentration with increased activity observed under lower serum conditions. <b>34</b> was administered po at 25, 100, and 200 mg/kg to female nude mice bearing established U87MG tumors and tumor concentration measured at 6 and 16 h (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C) post dose. Good plasma and tumor exposure were observed following either a 100 or 200 mg/kg oral dose with compound levels in the tumor above that needed to inhibit U87MG cell growth <i>in vitro</i> for at least 16 h following the administration of the highest dose. It was not possible to measure the DYRK1A biomarker (pS520 phosphorylation) due to the low level of expression in these cells. Further work is ongoing to identify novel biomarkers of DYRK1A inhibition. As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A, <b>34</b> significantly inhibited the proliferation of established U87MG xenografts with tumor growth delays (TGDs) of 93% (<i>P</i> < 0.01) and 224% (<i>P</i> < 0.001) following daily oral administration of 100 and 200 mg/kg, respectively. Little to no toxicity was observed at these doses (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compound <b>34</b> inhibits the growth of U87MG glioblastoma xenografts. Nude mice with established U87MG subcutaneous tumor xenografts were dosed with either 100 or 200 mg/kg PO <b>34</b> daily for 21 days (qdx21). Tumor volume (<b>A</b>) was determined by twice weekly caliper measurement, and body weight (<b>B</b>) was recorded.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Comparison of the Binding of 34 with Other Recently Published DYRK1A Inhibitors</h3><div class="NLM_p">A number of recent studies have generated crystal structures of other series of inhibitors of DYRK1A, derived from natural products such as the leucettines<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> and harmines<a onclick="showRef(event, 'ref41 ref45'); return false;" href="javascript:void(0);" class="ref ref41 ref45">(41,45)</a> or from profiling<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> or elaboration<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> of known kinase binding motifs and fragment molecules.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a><a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a> shows the chemical structures of representatives of these chemical series and <b>34</b>, oriented approximately as they bind in Dyrk1a and highlighting various features, in particular, the interactions with amino acids in the kinase hinge and with K188. The binding pose as seen in the crystal structures for these compounds is shown in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_001.pdf" class="ext-link">Figure S5</a>.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. <b>34</b> and representatives of different classes of DYRK1A inhibitors drawn in approximately the relative orientation seen in the available crystal structures for binding to Dyrk1a. Shading highlights equivalent features: green—equivalent to pyrollopyrimidine hinge binding; gray—equivalent to the methyl of <b>34</b>, binding in the small hydrophobic pocket adjacent to carbonyl of L241; yellow—equivalent pyrimidine; blue–equivalent interactions to the hydrophobic pocket; and pink—interaction with K188. See the text for references.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The most similar hinge-binding heterocycle to the pyrrollopyrimidine of <b>34</b> is the pyrollopyridine of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EIF">6EIF</a><a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> (<b>38</b>) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S1H">6S1H</a> (<b>39</b>) with two of the heterocyclic nitrogens of the amino-pyrimidine in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EIL">6EIL</a> (<b>45</b>) making the same hydrogen bonds. Similarly, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QU2">6QU2</a> (<b>40</b>) has a pyrimidine as part of a tricycle in an equivalent position as that in <b>34</b> with a cyclopropyl group occupying the position of the methyl. A number of methyl amino pyrimidines such as <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S1H">6S1H</a> (<b>39</b>) position the methyl in the same position but with different positions and orientations of the pyrimidine. There are a large number of other structures where atoms or small groups fill the “methyl” hole—some of which are derived from natural compounds such as harmine as in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ANR">3ANR</a> (<b>9</b>) and leucettine as in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AZE">4AZE</a> (<b>42</b>) and others where a methoxy benzene on benzothiazole, as in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A4L">5A4L</a> (<b>43</b>), or halogen in the methyl position, such as Br in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YLL">4YLL</a> (<b>44</b>).</div><div class="NLM_p last">The interaction made with K188 through compound optimization (as seen in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>) is a common feature of many DYRK1A inhibitors such as the equivalent pyridine ring in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EIF">6EIF</a> (<b>38</b>) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QU2">6QU2</a> (<b>40</b>), although just past the conventional limit of hydrogen bonding distance (3.5 Å) for <b>34</b> and for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EIL">6EIL</a> (<b>45</b>). There are, however, no published structures where the substituents mimic the full extent of the interactions made by the difluorobenzene substituent in <b>34</b> that sits within a hydrophobic region. The only compound for which a substructure reaches the same pocket is the iodine substituent of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YLL">4YLL</a> (<b>44</b>); however, this does not fully occupy the available space. One interesting comparison is between the pose for <b>41</b> and that for <b>39</b>—the change from a pyrolo-pyridine (<b>41</b>) to a pyrazolo-pyradazine (<b>39</b>) leads to a flip of the binding mode, presumably because of changed hydrogen bonding potential.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27412" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27412" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Compounds <b>11</b>–<b>17</b> shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> introduced aryls and heteroaryls into the 6-position, and these were prepared in an analogous manner to <b>16</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div><figure id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of Compound <b>16</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 33% HBr in acetic acid, acetone, −10 to −5 °C, 3 h; (b) triethyl orthoacetate, MeCN, reflux, yield 63%; (c) 7 N NH<sub>3</sub> in MeOH, sealed vessel, 105 °C, 6 h, yield 59%; (d) NaH, SEM-Cl, DMF, yield 57%, yield 48%; (e) pyridine-3-boronic acid, K<sub>2</sub>CO<sub>3</sub>, THF, water [1,1_-Bis(diphenylphosphino)ferrocene]dichloropalladium(II) (10 mol %), 130 °C; and (e) ethylenediamine, TBAF (1 M in THF), 120 °C microwave, 30 min, yield 38%.</p></p></figure><div class="NLM_p"><b>16</b> was prepared starting from commercially available 1,1,2,2-ethenetetracarbonitrile and cyclization to form the amino pyrrole <b>38</b> using HBr in acetic acid (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Ring closure to form the pyrimidine ring was performed using triethyl orthoacetate under refluxing conditions followed by methanolic ammonia in a sealed vessel to isolate key intermediate <b>39</b>. The 6-position bromo facilitated in installation of aromatics and heteroaromatics such as the 3-pyridinyl moiety via a Suzuki coupling to form compound <b>16</b>.</div><div class="NLM_p">To investigate different substituents at the 4-position, we prepared a series of 4-alkoxy compounds (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). These were prepared via a similar route to <b>19</b>, as outlined in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Starting from commercially available 4-chloro-2-methyl-7<i>h</i>-pyrrolo[2,3-<i>d</i>]pyrimidine, the pyrrole-7<i>H</i> was first protected with a SEM group and then brominated at the 5-position using <i>N</i>-bromosuccinamide. The formation of the appropriate alkoxide with sodium hydride and SNAr to substitute the 4-chloro occurs. The 5-position pyridines were installed via Suzuki coupling and finally deprotection of the SEM using tetrabutyl ammonium fluoride solution.</div><figure id="sch2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>19</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) SEM-Cl, NaH, DMF, 0 °C, 2 h, yield quant; (b) NBS, DMF, rt, yield 72%; (c) NaH, MeOH, 0 °C, yield quant; (d) (2-aminopyridin-4-yl)boronic acid, THF, water, K<sub>2</sub>CO<sub>3</sub>, cat [1,1′-Bis(diphenylphosphino)ferrocene]dichloropalladium(II), 1 h microwave, 140 °C, yield 32%; and (e) TBAF (1 M in THF), ethylenediamine, THF, yield 43%.</p></p></figure><div class="NLM_p">In a similar manner, we used the appropriate primary amines to install 4-aminoaryls such as <b>34</b> of which the preparation is outlined in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, using triethylamine as a tertiary base. Compounds shown in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a> were prepared via a similar route, starting with the SEM-protected intermediate <b>19a</b>.</div><figure id="sch3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <b>34</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 2,6-difluorobenzylamine, TEA, EtOH, 80 °C, 16 h, yield 87%; (b) 2-aminopyridine-4-boronic acid pinacol ester, K<sub>2</sub>CO<sub>3</sub>, Pd-118 (cat), THF, water, 60 °C, 16 h, yield: 76%; and (c) TBAF (1 M in THF) ethylenediamine, THF, 80 °C, 14 h, yield 59%.</p></p></figure><div class="NLM_p">In order to investigate pyrimidines at the 5-position, we needed to change the protection and deprotection strategy for the pyrrole ring 7 N. To prepare <b>35</b>, we first installed the benzene sulfonyl-protecting group and reversed the coupling partners to install the 5-postion aminopyridine. This route involved the preparation of the intermediate 5-boronate ester (<b>50</b>) prior to the Suzuki coupling with amino-4-chloropyrimidine, as shown in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>.</div><figure id="sch4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of <b>35</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) benzene sulfonyl chloride, NaH, DMF, 0 °C, 16 h, yield 91%; (b) 2,6-difluorobenzylamine, TEA, ethanol, 80 °C, 16 h, yield 93%; (c) bis(pinacolato)diboron, KOAc, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, THF, 140 °C, 1 h, yield 61%; (d) 2-amino-4-chloropyrimidine, K<sub>2</sub>CO<sub>3</sub>, Pd118, THF, 90 °C, 2 h, yield 64%; and (e) K<sub>2</sub>CO<sub>3</sub>, methanol, RT, 16 h, yield 40%.</p></p></figure><div class="NLM_p">To investigate 4-aryl substituents, we again modified the 7 N protection strategy employing boc-protection after installation of the 4 aryls via Suzuki coupling. We found that bromination in the 5-position and protection of the 7 N with boc-anhydride could be performed in a single step isolating the intermediate <b>53</b> by purification, as shown in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. A second Suzuki coupling under conditions similar to previous examples installed the 5-position aminopyridine moiety.</div><figure id="sch5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0013.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of <b>37</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 3,4-(methylenedioxy)-phenyl boronic acid, Cs<sub>2</sub>CO<sub>3</sub>, palladium(dppf)Cl<sub>2</sub>, THF, water, 130 °C, 45 min, yield quant; (b) NBS, DMF 0 °C, 2 h, yield 54%; (c) boc-anhydride, DMAP, Et<sub>3</sub>N; (d), boc-2-aminopyridine-4-boronic acid pinacol ester, K<sub>2</sub>CO<sub>3</sub>, Pd118, THF, water, 65 °C, 16 h; and (e) HCl in MeOH, microwave 90 °C, 1 h, yield 27% (over two steps).</p></p></figure><div class="NLM_p last">All tested compounds were of >95% purity (NMR spectra for <b>34</b> and MS traces for <b>34</b>–<b>37</b> in the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_001.pdf" class="ext-link">Supporting Information</a>).</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00273" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00273" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We have used fragment and structure-based discovery methods to identify <b>34</b>, a potent, <i>in vivo</i>-tolerated, selective inhibitor of DYRK1A kinase.</div><div class="NLM_p">The methyl group on the pyryllopyrimidine of <b>34</b> is an important determinant of selectivity for DYRK1A against other kinases, as it occupies a small hydrophobic pocket adjacent to Leu 244 for which the carbonyl of the backbone peptide bond has a different orientation to that seen in most protein kinases. As shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>, the introduction of other hydrophobic groups in this region is a feature of other DYRK1A inhibitors identified elsewhere. However, other features of <b>34</b> are distinctive. The difluoro-benzene moiety is in a portion of the active site of Dyrk1a not occupied by compounds in other published structures; similarly, the amino group on the pyridine moiety makes a hydrogen bond not mirrored by other compounds for which structures are available. Although <b>34</b> has similar potency to an inhibitor of DYRK1A and DYRK1B enzymes, it is highly selective against most other kinases; one of the exceptions is CLK1 for which a crystal structure demonstrates a similar hydrophobic pocket in the kinase hinge due to a similar peptide conformation in the kinase hinge to that seen in Dyrk1a.</div><div class="NLM_p last"><b>34</b> has many of the properties desired for a tool compound to probe the biological effects of DYRK1A inhibition. <i>In vitro</i>, it demonstrates potent inhibition of DYRK1A with concomitant target engagement in cells. It is orally bioavailable and tolerated at doses over 100 mg/kg and achieves sufficient plasma and tumor exposure to show efficacy in an <i>in vivo</i> model of glioblastoma with a PD marker (phosphorylation of S520), confirming cellular target engagement. This compound will be a useful tool to explore further the functional consequence of DYRK1A inhibition which may lead to new therapeutic strategies in oncology and Down’s syndrome.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17853" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17853" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">All chemicals were used directly as received from commercial suppliers. <sup>1</sup>H spectra were recorded on AVANCE III HD 400 (400 MHz for <sup>1</sup>H, BRUKER). Chemical shifts for <sup>1</sup>H are reported in parts per million (ppm) referenced to the residual solvent peak (DMSO-<i>d</i><sub>6</sub>, δ = 2.50 ppm). Abbreviations used to describe the peak signals in <sup>1</sup>H NMR data are s = singlet, br = broad, d = doublet, dd = double doublet, dt = double triplet, t = triplet, td = triple doublet, tt = triple triplet, q = quartet, quin = quintet, and m = multiplet. All compounds were determined as having purity >95% unless stated otherwise, and all final tested compounds were purified to >95% by reverse-phase high-performance liquid chromatography (HPLC), or normal-phase silica gel column chromatography. The purity was assessed by reverse-phase ultraperformance liquid chromatography with a gradient of 5–98% acetonitrile in water (with either the acid or base modifier) and monitored by UV absorption at 210–450 nm.</div><div class="NLM_p">Unless otherwise described, the following conditions were used for compound synthesis, purification, and analysis. A CEM microwave was used for the microwave reaction conditions. OH-type silica gel column chromatography was performed using ISCO Combi-flash RF or RF Gold. The prep-HPLC purifications were performed using Waters prep LCMS [ionization method: electron spray ionization (ESI)].</div><div class="NLM_p">The following compounds <b>11</b>–<b>15</b> were prepared in an analogous manner to compound <b>16</b>.</div><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> 4-Amino-7-ethyl-2-methyl-6-(pyridine-3-yl)-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine-5-carbonitrile (<b>11</b>)</h3><div class="NLM_p last">It was isolated as a colorless solid, 32 mg (0.11 mmol) (yield 7%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.89–8.78 (m, 2H), 8.14 (m, 1H), 7.73–7.58 (m, 2H), 6.84 (s, 2H), 4.17 (q, <i>J</i> = 7.16 Hz, 2H), 2.47 (s, 3H), 1.15 (t, <i>J</i> = 7.16 Hz, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 279[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>15</sub>H<sub>14</sub>N<sub>6</sub>+H]<sup>+</sup>, 279.1284; found, 279.1356.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 4-Amino-6-(2,3-difluorophenyl)-7-ethyl-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine-5-carbonitrile (<b>12</b>)</h3><div class="NLM_p last">It was isolated as a colorless solid, 15 mg (0.05 mmol) (yield 3%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.83–7.71 (m, 1H), 7.58–7.45 (m, 2H), 6.88 (s, 2H), 4.10 (q, <i>J</i> = 7.14 Hz, 2H), 2.47 (s, 3H), 1.15 (t, <i>J</i> = 7.15 Hz, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 314[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>16</sub>H<sub>13</sub>N<sub>5</sub>F<sub>2</sub>+H]<sup>+</sup>, 314.1139; found, 314.1217.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 4-Amino-6-(2,3-difluorophenyl)-7-methyl-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine-5-carbonitrile (<b>13</b>)</h3><div class="NLM_p last">It was isolated as a colorless solid, 18 mg (0.06 mmol) (yield 32%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.82–7.70 (m, 1H), 7.59–7.44 (m, 2H), 6.89 (s, 2H), 3.61 (d, <i>J</i> = 0.97 Hz, 3H), 2.46 (s, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 300[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>15</sub>H<sub>11</sub>N<sub>5</sub>F2+H]<sup>+</sup>, 300.0983; found, 300.1059.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 4-Amino-6-(5-methoxypyridin-3-yl)-2,7-dimethyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine-5-carbonitrile (<b>14</b>)</h3><div class="NLM_p last">It was isolated as a colorless solid, 28 mg (0.10 mmol) (yield 13%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.52 (d, <i>J</i> = 2.9 Hz, 1H), 8.47 (d, <i>J</i> = 1.8 Hz, 1H), 7.76 (dd, <i>J</i> = 2.9, 1.8 Hz, 1H), 3.93 (s, 3H), 3.69 (s, 3H), 3.35 (s, 2H), 2.47 (s, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 295[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>15</sub>H<sub>14</sub>N<sub>6</sub>O + H]<sup>+</sup>, 295.1229; found, 295.1307.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 4-Amino-2,7-dimethyl-6-(pyridine-3-yl)-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine-5-carbonitrile (<b>15</b>)</h3><div class="NLM_p last">It was isolated as a colorless solid, 28 mg (0.11 mmol) (yield 28%). <sup>1</sup>H NMR (400 MHz, DMSO): δ 8.90 (dd, <i>J</i> = 2.3, 0.9 Hz, 1H), 8.80 (dd, <i>J</i> = 4.9, 1.6 Hz, 1H), 8.17 (dd, <i>J</i> = 2.4, 1.7 Hz, 1H), 7.68 (dd, <i>J</i> = 4.9, 0.9 Hz, 1H), 6.84 (s, 2H), 3.68 (s, 3H), 2.47 (s, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 265[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>14</sub>H<sub>12</sub>N<sub>6</sub>+H]<sup>+</sup>, 265.1123; found, 265.1199.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 4-Amino-2-methyl-6-(pyridine-3-yl)-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine-5-carbonitrile (<b>16</b>). Step 1</h3><div id="sec5_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 2-Amino-5-bromo-1<i>H</i>-pyrrole-3,4-dicarbonitrile (<b>38</b>)</h4><div class="NLM_p last">A 3 L flask was fitted with a condenser, a drying tube, and an internal thermometer. The flask was cooled in a methanol/ice bath. Ethyl acetate (1 L), acetone (185 mL), and 1,1,2,2-ethenetetracarbonitrile, Sigma-Aldrich T8809 (50 g, 0.39 mol), were added. The solution was cooled to −10 °C (methanol/ice bath), and HBr (287 mL, 33% in acetic acid solution, 3 equiv) was added dropwise at such a rate as to maintain −5 °C (over ∼1 h). A yellow solid began to precipitate during the addition. After the addition was complete, the suspension was stirred for an additional 3 h. The yellow precipitate was collected by filtration and washed with ethyl acetate (2 × 100 mL) and diethyl ether (2 × 100 mL). The solid cake was then suspended in water (600 mL), and the pH was adjusted to 11 with 50% NaOH, giving a dark solution. Decolorizing charcoal (<i>ca.</i> 15 g) was then added to the solution and filtered through celite, and the filtrate was acidified to pH 5 with glacial acetic acid. The mixture was allowed to stand at rt overnight. The product was then collected by filtration and dried at 40 °C in a vacuum oven for 5 days to give the title compound as a brown solid, 52 g (yield 63%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.33 (s, 1H), 6.49 (s, 2H).</div></div><div id="sec5_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 4-Amino-6-bromo-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine-5-carbonitrile (<b>39</b>)</h4><div class="NLM_p last"><b>38</b> (8.44 g, 40 mmol) was suspended in dry acetonitrile (150 mL). Triethyl orthoacetate (11 mL, 60 mmol, 1.5 equiv) was added, and the mixture was heated at rt for ∼1 h. The dark brown solution was cooled to rt and filtered through a filter paper. Acetonitrile was removed under vacuum to afford a brown solid which was transferred to a steel reaction vessel. 7 N NH<sub>3</sub> in MeOH (200 mL) was added, and the vessel was then sealed and heated to 105 °C for 6 h. The vessel was cooled to rt overnight, and the resulting solid was collected by filtration. The crude solid was washed several times with methanol followed by diethyl ether to furnish the title compound as a gray solid, 6 g (yield 59%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.81 (s, 1H), 7.34 (s, 2H), 2.43 (s, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 254[M + H]<sup>+</sup>.</div></div><div id="sec5_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 4-Amino-6-bromo-2-methyl-7-<sup>23</sup>-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine-5-carbonitrile (<b>40</b>)</h4><div class="NLM_p last">To a suspension of <b>39</b> (200 mg, 0.8 mmol) in dry dimethylformamide (DMF) (10 mL) was added sodium hydride (60% in mineral oil (32 mg, 0.8 mmol, 1 equiv) at rt). The suspension became a dark clear solution after <i>ca.</i> 5 min. The mixture was stirred for 30 min before adding SEM-Cl (140 μL, 0.8 mmol, 1 equiv) at rt and stirred under nitrogen for 2 h. The reaction mixture was diluted with ethyl acetate (50 mL), washed with brine (2 × 50 mL), dried over anhydrous magnesium sulfate, and concentrated under vacuum. Purification was performed by flash column chromatography to furnish the title compound as a mixture of regio-isomers (ratio ∼1:1), 172 mg (yield 57%). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 384[M + H]<sup>+</sup> (2 overlapping peaks, isomers seen by LCMS).</div></div><div id="sec5_6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 4-Amino-2-methyl-6-(pyridine-3-yl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine-5-carbonitrile (<b>41</b>)</h4><div class="NLM_p last"><b>40</b> (100 mg, 0.26 mmol) and pyridine-3-boronic acid (48 mg, 0.39 mmol, 1.5 equiv) were dissolved in a mixture of THF/water (5:1, 6 mL) under nitrogen. The mixture was treated with potassium carbonate (311 mg, 0.52 mmol, 2 equiv), and [1,1-bis(diphenylphosphino)ferrocene]dichloropalladium(II) (19 mg, 10 mol %) was added, and the resulting mixture was degassed with a stream of nitrogen for 5 min. The reaction mixture was heated at 130 °C in a CEM microwave for 60 min. The whole mixture was then diluted with dichloromethane (20 mL), and water was added (20 mL). The organic layer was separated, washed with brine, dried over magnesium sulfate, and concentrated under vacuum. The crude residue was purified using flash column chromatography to furnish the title compound as a yellow solid, 48 mg (yield 48%). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 381[M + H]<sup>+</sup>.</div></div><div id="sec5_6_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 4-Amino-2-methyl-6-(pyridine-3-yl)-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine-5-carbonitrile (<b>16</b>)</h4><div class="NLM_p last">To a solution of 3001-AF-085 4-amino-2-methyl-6-(pyridine-3-yl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine-5-carbonitrile (48 mg, 0.13 mmol) in tetrahydrofuran (THF) (3 mL) was added ethylenediamine (42 μL, 0.63 mmol, 5 equiv) followed by TBAF (1 M solution in THF) (0.63 mL, 0.63 mmol, 5 equiv). The reaction was heated to 120 °C for 30 min in a microwave. The reaction mixture was poured into a mixture of water and ethyl acetate (10 mL, 1:1) and stirred for ∼15 min. The aqueous layer was separated and concentrated under vacuum. The resulting precipitate was triturated in acetonitrile, and the solid was collected by filtration. The solid was washed lightly with diethyl ether and dried under vacuum to give the title compound as a colorless solid, 12 mg (yield 38%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.02 (s, 1H), 9.11 (d, <i>J</i> = 2.3 Hz, 1H), 8.70 (dd, <i>J</i> = 4.8, 1.6 Hz, 1H), 8.29 (dt, <i>J</i> = 8.1, 2.0 Hz, 1H), 7.64 (dd, <i>J</i> = 8.0, 4.7 Hz, 1H), 6.79 (s, 2H), 2.43 (s, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 251[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>13</sub>H<sub>10</sub>N<sub>6</sub>+H]<sup>+</sup>, 251.0967; found, 251.1042.</div></div><div id="sec5_6_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 4-Amino-6-(2,3-difluorophenyl)-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine-5-carbonitrile (<b>17</b>)</h4><div class="NLM_p last">Compound <b>17</b> was prepared in an analogous manner to compound <b>16</b>. The title compound was isolated as a colorless solid, 10 mg (0.03 mmol) (yield 31%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.04 (s, 1H), 7.66 (q, <i>J</i> = 8.7 Hz, 1H), 7.61–7.52 (m, 1H), 7.45 (q, <i>J</i> = 5.4, 3.4 Hz, 1H), 6.81 (s, 2H), 2.43 (s, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 286[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>14</sub>H<sub>9</sub>N<sub>5</sub>F<sub>2</sub>+H]<sup>+</sup>, 286.0826; found, 286.0902.</div></div><div id="sec5_6_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 4-{4-Methoxy-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}pyridine (<b>18</b>)</h4><div class="NLM_p last">4-{4-methoxy-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}pyridine (<b>18</b>) was as prepared in an analogous manner to compound <b>19</b>. The title compound was isolated as a colorless solid, 15 mg (yield 21%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.33 (s, 1H), 8.58–8.50 (m, 2H), 7.85 (s, 1H), 7.78–7.72 (m, 2H), 4.05 (s, 3H), 2.57 (s, 3H). MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> 241[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>13</sub>H<sub>12</sub>N<sub>4</sub>O + H]<sup>+</sup>, 241.1011; found, 241.1082.</div></div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 4-{4-Methoxy-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}pyridine-2-amine (<b>19</b>). Step 1</h3><div id="sec5_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 4-Chloro-2-methyl-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>42</b>)</h4><div class="NLM_p last">To a solution of 4-chloro-2-methyl-7h-pyrrolo[2,3-<i>d</i>]pyrimidine (Combi-Blocks ST-3008) (1.85 g, 11 mmol) in dry DMF (30 mL) was added sodium hydride (60% in mineral oil) (290 mg, 12 mmol) at 0 °C under nitrogen. The mixture was stirred for 30 min before adding SEM-Cl (2.34 mL, 13 mmol 1.2 equiv) dropwise at 0 °C, and after addition, the whole mixture was stirred under nitrogen for 2 h. The reaction mixture was diluted with diethyl ether (100 mL), washed with brine (2 × 50 mL), dried over anhydrous magnesium sulfate, and concentrated under vacuum to give the title compound as brown oil, 3.57 g (yield quant) (purity determined by LCMS 90%) The crude product title compound was used in the subsequent step without further purification. MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 298 [M + H]<sup>+</sup>.</div></div><div id="sec5_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 5-Bromo-4-chloro-2-methyl-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>43</b>)</h4><div class="NLM_p last">To a solution of crude <b>42</b> (3.3 g, 11 mmol) in DMF (30 mL) was added <i>N</i>-bromosuccinamide (2 g, 11 mmol, 1 equiv) in DMF (5 mL) at 0 °C under N<sub>2</sub>. It was stirred for 30 min at 0 °C and then allowed to warm to rt over 30 min. The reaction mixture was diluted with water (20 mL) and extracted with diethyl ether (30 mL). The organic layer was washed several times with water, dried over anhydrous magnesium sulfate, and concentrated in vacuum. The residue was purified by flash column chromatography to give the title compound as a colorless solid, 3 g (yield 72% over two steps). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.13 (s, 1H), 5.65 (s, 2H), 3.66–3.57 (m, 2H), 2.74 (s, 3H), 0.98–0.88 (m, 2H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 378[M + H]<sup>+</sup>.</div></div><div id="sec5_7_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 5-Bromo-4-methoxy-2-methyl-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>44</b>)</h4><div class="NLM_p last">To a suspension of sodium hydride (60% suspension in mineral oil) (530 mg, 13 mmol, 2 equiv) in THF (10 mL) was added methanol (346 μL, 8.6 mmol, 1.3 equiv) dropwise at 0 °C (ice bath cooling). The mixture was stirred for approximately 10 min before adding <b>43</b> (2.5 g, 6.6 mmol) in THF (10 mL) under N<sub>2</sub> at 0 °C. The reaction mixture was stirred at 0 °C for 30 min and allowed to warm to rt over 1 h. The reaction was quenched with saturated NH<sub>4</sub>Cl<sub>(aq)</sub> (50 mL) and extracted with ethyl acetate (50 mL). The organic layer was washed with brine, dried over anhydrous magnesium sulfate, and concentrated under vacuum to give the title compound as light yellow oil (2.4 g, yield quant). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 372[M + H]<sup>+</sup>.</div></div><div id="sec5_7_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 4-(4-Methoxy-2-methyl-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl)pyridine-2-amine (<b>45</b>)</h4><div class="NLM_p last"><b>44</b> (150 mg, 0.4 mmol) and (2-aminopyridin-4-yl)boronic acid (133 mg, 0.6 mmol, 1.5 equiv) were dissolved in a mixture of THF and water (6:1, 5 mL) under nitrogen. Potassium carbonate (166 mg, 1.2 mmol, 3 equiv) and [1,1′-bis(diphenylphosphino)ferrocene]dichloropalladium(II) (15 mg, 5 mol %) were added, and the resulting mixture was degassed with a stream of N<sub>2</sub> for 5 min. The reaction mixture was heated at 140 °C in a CEM microwave for 1 h. The product mixture was diluted with water (10 mL) and extracted with ethyl acetate. The organic layer was washed with brine, dried over anhydrous magnesium sulfate, and concentrated under vacuum. The resulting crude product oil was taken up in dichloromethane and dry-loaded onto ISOLUTE before purification by flash column chromatography to afford the title compound as a colorless solid, 50 mg (yield 32%). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 386[M + H]<sup>+</sup>.</div></div><div id="sec5_7_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 4-{4-Methoxy-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}pyridine-2-amine (<b>19</b>)</h4><div class="NLM_p">To a solution of <b>45</b> (50 mg, 0.13 mmol) in THF (3 mL) was added ethylenediamine (43 μL 0.65 mmol, 5 equiv) followed by TBAF (1 M in THF, 5 equiv). The reaction was stirred at 80 °C for 2 h in the microwave. The reaction mixture was poured into water and ethyl acetate (20 mL, 1:1) and stirred for 15 min. The organic layer was separated, washed with brine, dried over anhydrous magnesium sulfate, and concentrated under vacuum. The residue was then triturated with acetonitrile and collected by filtration to furnish the title compound as a colorless solid, 15 mg (yield 43%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.16 (s, 1H), 7.86 (dd, <i>J</i> = 5.4, 0.7 Hz, 1H), 7.63 (d, <i>J</i> = 2.5 Hz, 1H), 6.89–6.79 (m, 2H), 5.83 (s, 2H), 4.04 (s, 3H), 2.56 (s, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 256[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>13</sub>H<sub>13</sub>N<sub>5</sub>O + H]<sup>+</sup>, 256.1123; found, 256.1196.</div><div class="NLM_p last">The following compounds <b>20–28</b> were prepared in an analogous manner to compound <b>19</b>.</div></div><div id="sec5_7_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 4-{4-Methoxy-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-methylpyridine (<b>20</b>)</h4><div class="NLM_p last">It was isolated as a colorless solid, 15 mg (yield 45%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.29 (s, 1H), 8.39 (dd, <i>J</i> = 5.3, 0.7 Hz, 1H), 7.81 (s, 1H), 7.64–7.59 (m, 1H), 7.54 (dd, <i>J</i> = 5.4, 1.8 Hz, 1H), 4.04 (s, 3H), 2.57 (s, 3H), 2.50 (s, 2H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 255[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>14</sub>H<sub>14</sub>N<sub>4</sub>O + H]<sup>+</sup>, 255.1168; found, 255.1238.</div></div><div id="sec5_7_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 3-{4-Methoxy-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}benzamide (<b>21</b>)</h4><div class="NLM_p last">It was isolated as a colorless solid, 16 mg (yield 26%). <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.10 (s, 1H), 8.19 (t, <i>J</i> = 1.8 Hz, 1H), 7.99 (s, 1H), 7.83 (dd, <i>J</i> = 7.8, 1.4 Hz, 1H), 7.74 (dd, <i>J</i> = 7.8, 1.4 Hz, 1H), 7.58 (s, 1H), 7.46 (t, <i>J</i> = 7.7 Hz, 1H), 7.38 (s, 1H), 3.99 (s, 3H), 2.57 (s, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 283[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>15</sub>H<sub>14</sub>N<sub>4</sub>O<sub>2</sub>+H]<sup>+</sup>, 283.1117; found, 283.1193.</div></div><div id="sec5_7_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 2-Fluoro-4-{4-methoxy-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}pyridine (<b>22</b>)</h4><div class="NLM_p last">It was isolated as a colorless solid, 20 mg (yield 40%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.52 (s, 1H), 8.26 (d, <i>J</i> = 5.3 Hz, 1H), 8.08 (s, 1H), 7.83 (dd, <i>J</i> = 5.4, 2.2, 1.4 Hz, 1H), 7.64–7.58 (m, 1H), 4.13 (s, 3H), 2.64 (s, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 259[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>13</sub>H<sub>11</sub>N<sub>4</sub>OF + H]<sup>+</sup>, 259.0917; found, 259.0996.</div></div><div id="sec5_7_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 5-(4-fluorophenyl)-4-methoxy-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>23</b>)</h4><div class="NLM_p last">It was isolated as a colorless solid, 40 mg (yield 55%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.05 (s, 1H), 7.74–7.64 (m, 2H), 7.49 (s, 1H), 7.27–7.16 (m, 2H), 3.99 (s, 3H), 2.56 (s, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 258[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>14</sub>H<sub>12</sub>N<sub>3</sub>OF + H]<sup>+</sup>, 258.0964; found, 258.1043.</div></div><div id="sec5_7_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 4-{2-Methyl-4-[(thiophen-3-yl)methoxy]-7H-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}pyridine-2-amine (<b>24</b>)</h4><div class="NLM_p last">It was isolated as a colorless solid, 15 mg (yield 37%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.26 (s, 1H), 7.88 (dd, <i>J</i> = 5.3, 0.7 Hz, 1H), 7.69 (s, 2H), 7.74–7.60 (m, 2H), 7.37 (dd, <i>J</i> = 4.9, 1.3 Hz, 1H), 6.95–6.85 (m, 2H), 5.76 (s, 2H), 5.63 (s, 2H), 2.67 (s, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 338[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>17</sub>H<sub>15</sub>N<sub>5</sub>OS + H]<sup>+</sup>, 338.0995; found, 338.1068.</div></div><div id="sec5_7_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 4-{2-Methyl-4-[(5-methyl-1,2-oxazol-3-yl)methoxy]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}pyridine (<b>25</b>)</h4><div class="NLM_p last">It was isolated as a colorless solid, 8 mg (yield 38%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.41 (s, 1H), 8.50–8.44 (m, 2H), 7.89 (s, 1H), 7.79–7.72 (m, 2H), 6.38 (d, <i>J</i> = 1.1 Hz, 1H), 5.59 (s, 2H), 2.59 (s, 3H), 2.45 (s, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 322[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>17</sub>H<sub>15</sub>N<sub>5</sub>O<sub>2</sub>+H]<sup>+</sup>, 322.1226; found, 322.1299.</div></div><div id="sec5_7_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> {2-Methyl-4-[(1,3-thiazol-4-yl)methoxy]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}pyridine-2-amine (<b>26</b>)</h4><div class="NLM_p last">It was isolated as a colorless solid, 3 mg (yield 6%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.28 (s, 1H), 9.24 (d, <i>J</i> = 2.0 Hz, 1H), 7.88–7.80 (m, 2H), 7.72 (s, 1H), 6.97–6.90 (m, 1H), 6.92 (s, 2H), 5.79–5.71 (m, 4H), 2.66 (s, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 339[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>16</sub>H<sub>14</sub>N<sub>6</sub>OS + H]<sup>+</sup>, 339.0948; found, 339.1019.</div></div><div id="sec5_7_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 4-{2-Methyl-4-[(1,3-thiazol-5-yl)methoxy]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}pyridine-2-amine (<b>27</b>)</h4><div class="NLM_p last">It was isolated as a colorless solid, 10 mg (yield 40%). <sup>1</sup>H NMR (399 MHz, DMSO): δ 12.24 (s, 1H), 9.10 (d, <i>J</i> = 0.8 Hz, 1H), 8.09 (d, <i>J</i> = 0.9 Hz, 1H), 7.85–7.79 (d, 1H), 7.64 (s, 1H), 6.81 (dd, 2H), 5.82 (s, 2H), 5.75 (s, 2H), 2.63 (s, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 339[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>16</sub>H<sub>14</sub>N<sub>6</sub>OS + H]<sup>+</sup>, 339.0948; found, 339.1025.</div></div><div id="sec5_7_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 4-[4-(Benzyloxy)-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl]pyridine (<b>28</b>)</h4><div class="NLM_p">It was isolated as a colorless solid, 22 mg (yield 75%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.35 (s, 1H), 8.41–8.35 (m, 2H), 7.86 (s, 1H), 7.77–7.70 (m, 2H), 7.57–7.49 (m, 2H), 7.47–7.32 (m, 3H), 5.57 (s, 2H), 2.60 (s, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 317[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>19</sub>H<sub>16</sub>N<sub>4</sub>O + H]<sup>+</sup>, 317.1324; found, 317.1391.</div><div class="NLM_p last">The following compounds were prepared in parallel in an analogous manner to <b>34</b>. To a solution of the SEM-protected precursor in THF (5 mL) was added ethylenediamine (5 equiv) followed by TBAF (1 M in THF, 5 equiv), and the mixture was stirred at 80 °C for 3 h. The individual reaction mixtures were concentrated under vacuum and evaporated onto ISOLUTE. Purification was performed by flash column chromatography followed by trituration with acetonitrile (3 × 2 mL), and the resultant solids were collected via filtration. The solids were washed lightly with diethyl ether (2 mL) and then dried under vacuum. The purified compounds were characterized by LCMS and <sup>1</sup>H NMR.</div></div><div id="sec5_7_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 4-[4-(benzylamino)-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl]pyridine-2-amine (<b>29</b>)</h4><div class="NLM_p last">It was isolated as a colorless solid, 51 mg (0.15 mmol) (yield 21%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.78–11.72 (m, 1H), 7.88 (dd, <i>J</i> = 0.71, 5.18 Hz, 1H), 7.42–7.28 (m, 4H), 7.33–7.19 (m, 2H), 6.60–6.49 (m, 2H), 5.99–5.88 (m, 3H), 4.70 (d, <i>J</i> = 5.61 Hz, 2H), 2.42 (s, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 331[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>19</sub>H<sub>18</sub>N<sub>6</sub>+H]<sup>+</sup>, 331.1593; found, 331.1589.</div></div><div id="sec5_7_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 4-(2-Methyl-4-{[(thiophen-2-yl)methyl]amino}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl)pyridine-2-amine (<b>30</b>)</h4><div class="NLM_p last">It was isolated as a colorless solid, 31 mg (0.09 mmol) (yield 24%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.77 (s, 1H), 7.91 (dd, <i>J</i> = 0.73, 5.24 Hz, 1H), 7.36 (dd, <i>J</i> = 1.26, 5.14 Hz, 1H), 7.25 (d, <i>J</i> = 1.65 Hz, 1H), 7.08–7.01 (m, 1H), 6.94 (dd, <i>J</i> = 3.43, 5.11 Hz, 1H), 6.58–6.46 (m, 2H), 6.07 (t, <i>J</i> = 5.86 Hz, 1H), 5.94 (s, 2H), 4.86 (d, <i>J</i> = 5.60 Hz, 2H), 2.47 (s, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 337[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>17</sub>H<sub>16</sub>N<sub>6</sub>S + H]<sup>+</sup>, 337.1149; found, 337.1222.</div></div><div id="sec5_7_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 4-(2-Methyl-4-{[(thiophen-3-yl)methyl]amino}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl)pyridine-2-amine (<b>31</b>)</h4><div class="NLM_p last">It was isolated as a colorless solid, 54 mg (0.16 mmol) (yield 45%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.75 (s, 1H), 7.90 (d, <i>J</i> = 5.20 Hz, 1H), 7.49 (dd, <i>J</i> = 2.93, 4.95 Hz, 1H), 7.38 (d, <i>J</i> = 3.33 Hz, 1H), 7.24 (s, 1H), 7.15 (dd, <i>J</i> = 1.35, 4.91 Hz, 1H), 6.57 (dd, <i>J</i> = 1.55, 5.17 Hz, 1H), 6.51 (s, 1H), 5.98 (s, 2H), 5.83 (t, <i>J</i> = 5.57 Hz, 1H), 4.68 (d, <i>J</i> = 5.42 Hz, 2H), 2.45 (s, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 337[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>17</sub>H<sub>16</sub>N<sub>6</sub>S + H]<sup>+</sup>, 337.1157; found, 337.1231.</div></div><div id="sec5_7_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 4-(4-{[(Furan-3-yl)methyl]amino}-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl)pyridine-2-amine (<b>32</b>)</h4><div class="NLM_p last">It was isolated as a colorless solid, 30 mg (0.09 mmol) (yield 24%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.78–11.72 (m, 1H), 7.91 (dd, <i>J</i> = 0.70, 5.18 Hz, 1H), 7.66–7.57 (m, 2H), 7.24 (d, <i>J</i> = 2.03 Hz, 1H), 6.59–6.52 (m, 2H), 6.50 (dd, <i>J</i> = 0.77, 1.60 Hz, 1H), 5.99 (s, 2H), 5.68 (t, <i>J</i> = 5.45 Hz, 1H), 4.51 (d, <i>J</i> = 5.23 Hz, 2H), 2.46 (s, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 321[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>17</sub>H<sub>16</sub>N<sub>6</sub>O + H]<sup>+</sup>, 320.1394; found, 321.1466.</div></div><div id="sec5_7_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 4-(2-Methyl-4-{[(1,3-thiazol-4-yl)methyl]amino}-7H-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl)pyridine-2-amine (<b>33</b>)</h4><div class="NLM_p last">It was isolated as a colorless solid, 6 mg (0.02 mmol) (yield 3%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.78 (s, 1H), 9.07 (d, <i>J</i> = 1.96 Hz, 1H), 7.92 (d, <i>J</i> = 5.26 Hz, 1H), 7.53 (dd, <i>J</i> = 1.02, 2.03 Hz, 1H), 7.26 (d, <i>J</i> = 2.44 Hz, 1H), 6.60 (dd, <i>J</i> = 1.53, 5.19 Hz, 1H), 6.53 (d, <i>J</i> = 1.72 Hz, 1H), 6.09 (q, <i>J</i> = 4.64, 5.62 Hz, 1H), 5.96 (s, 2H), 4.88–4.79 (m, 2H), 3.22–3.12 (m, 3H), 2.43 (s, 3H), 1.57 (dt, <i>J</i> = 7.32, 15.32 Hz, 3H), 1.32 (h, <i>J</i> = 7.35 Hz, 3H), 0.94 (t, <i>J</i> = 7.35 Hz, 4H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 338[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>16</sub>H<sub>15</sub>N<sub>7</sub>S + H]<sup>+</sup>, 337.1114; found, 338.1189.</div></div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> 4-(4-{[(3,5-Difluorophenyl)methyl]amino}-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl)pyridine-2-amine (34). Step 1</h3><div id="sec5_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 5-Bromo-<i>N</i>-[(3,5-difluorophenyl)methyl]-2-methyl-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine (<b>46</b>)</h4><div class="NLM_p last">To a solution of <b>43</b> (5 g, 13.3 mmol) in ethanol (50 mL) was added 2,6-difluorobenzylamine (Combi-Blocks SS-4198) (3.17 mL, 26.5 mmol, 2 equiv) and TEA (5.5 mL, 39.8 mmol, 5 equiv) and heated at 80 °C for 16 h. The reaction mixture was concentrated under vacuum, and the residue was retaken up in EtOAc (100 mL), washed with water and brine, and dried over anhydrous magnesium sulfate. The filtered organics were evaporated and dry-loaded onto ISOLUTE, and purification was performed by flash column chromatography (120 g silica column) to furnish the title compound as clear oil, 5.56 g (yield 87%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.55 (s, 2H), 7.46 (tt, <i>J</i> = 8.6, 6.6 Hz, 2H), 7.23–7.11 (m, 4H), 6.80 (t, <i>J</i> = 5.9 Hz, 2H), 5.51 (s, 4H), 4.92 (d, <i>J</i> = 5.8 Hz, 4H), 3.62–3.53 (m, 4H), 2.49 (s, 6H), 1.27 (t, <i>J</i> = 7.1 Hz, 1H), 1.00–0.85 (m, 4H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 485[M + H]<sup>+</sup>.</div></div><div id="sec5_8_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 5-Bromo-<i>N</i>-[(3,5-difluorophenyl)methyl]-2-methyl-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine (<b>47</b>)</h4><div class="NLM_p"><b>46</b> (5.6 g, 11.5 mmol) and 2-aminopyridine-4-boronic acid pinacol ester (Combi-Blocks PN-0151) (4.3 g, 13.8 mmol, 1.2 equiv) were dissolved in a mixture of THF and water (4:1, 200 mL) under N<sub>2</sub>. The mixture was treated with potassium carbonate (4.9 g, 36 mmol, 3 equiv) and Pd-118 (375 mg, 5 mol %), and the mixture was degassed with a stream of N<sub>2</sub> for 5 min. The whole mixture was heated up to 60 °C for 16 h, and the reaction was monitored by LCMS. The reaction mixture was cooled to rt, filtered through a plug of celite, and diluted with ethyl acetate. The organic layer was washed with brine, dried over anhydrous magnesium sulfate, concentrated under vacuum, and dry-loaded by evaporation onto ISOLUTE. Purification was performed by flash column chromatography (120 g silica). Trituration with iso-hexane gave a suspended solid which was collected by filtration and dried under vacuum to furnish the title compound as a colorless solid, 4.33 g (yield: 76%). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 497[M + H]<sup>+</sup>.</div><div class="NLM_p">4-(4-{[(3,5-Difluorophenyl)methyl]amino}-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl)pyridine-2-amine (<b>34</b>).</div><div class="NLM_p last">To a solution of <b>47</b> (1 equiv) in THF (100 mL) was added ethylenediamine (6 equiv) followed by TBAF (1 M solution in THF, 6 equiv), and the reaction mixture was heated to 80 °C for a total of 14 h. The reaction mixture was cooled, diluted with ethyl acetate (100 mL), washed with water (2 × 100 mL) and brine, dried over anhydrous magnesium sulfate, and concentrated under vacuum. The residue was triturated to a solid using ethyl acetate and acetonitrile, followed by diethyl ether, and the solid was collected by filtration. The solid product was dried under vacuum to give the title compound as a colorless solid, 1.89 g (yield 59%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 11.75 (s, 1H), 7.90 (d, <i>J</i> = 5.18 Hz, 1H), 7.37 (m, <i>J</i> = 6.61, 8.41 Hz, 1H), 7.24 (d, <i>J</i> = 2.29 Hz, 1H), 7.09 (t, <i>J</i> = 7.94 Hz, 2H), 6.54–6.45 (m, 2H), 5.93 (s, 2H), 5.85 (t, <i>J</i> = 5.79 Hz, 1H), 4.77 (d, <i>J</i> = 5.65 Hz, 2H), 2.43 (s, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 367[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>19</sub>H<sub>16</sub>N<sub>6</sub>F<sub>2</sub>+H]<sup>+</sup>, 367.1405; found, 367.1471.</div></div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> 4-(4-{[(3,5-Difluorophenyl)methyl]amino}-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl)pyrimidin-2-amine (<b>35</b>). Step 1</h3><div id="sec5_9_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 7-(Benzenesulfonyl)-5-bromo-4-chloro-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>48</b>)</h4><div class="NLM_p last">To a solution of 5-bromo-4-chloro-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (Combi-Blocks QV-2886) (1.86 g, 7.55 mmol) in DMF (40 mL) was added sodium hydride (60% oil dispersion) (332 mg, 8.3 mmol, 1.1 equiv) at 0 °C under nitrogen. The reaction mixture was stirred at 0 °C for 30 min before adding benzene sulfonyl chloride (1.16 mL, 9 mmol, 1.2 equiv) at 0 °C. The reaction mixture was allowed to warm to room temperature gradually, and stirring was continued under nitrogen for 16 h. The reaction mixture was poured into water (50 mL) and stirred at room temperature for 20 min to break up the precipitated solid. The resulting fine precipitate was collected by filtration and washed several times with water and air-dried on a sintered funnel to furnish the title compound as a white solid, 2.64 g (yield 91%). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 387[M + H]<sup>+</sup>.</div></div><div id="sec5_9_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 7-(Benzenesulfonyl)-5-bromo-<i>N</i>-[(3,5-difluorophenyl)methyl]-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine (<b>49</b>)</h4><div class="NLM_p last">To a suspension of <b>48</b> (1 g, 2.59 mmol) in ethanol (20 mL) was added 2,6-difluorobenzylamine (0.62 mL, 5.17 mmol, 2 equiv) and triethylamine (1 mL, 7.8 mmol, 3 equiv), and the mixture was heated at 80 °C for 16 h. The clear reaction mixture was cooled to rt upon which a yellow precipitate was formed. Water (30 mL) was added, and the suspension was stirred for ∼20 min before the precipitate was collected by filtration. The solid filter cake was washed further with water and air-dried on the filter thoroughly before washing with iso-hexane. The solid was then dried under vacuum to furnish the title compound as a colorless solid, 1.19 g (yield 93%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.17–8.09 (m, 2H), 7.81–7.71 (m, 2H), 7.65 (dd, <i>J</i> = 8.5, 7.2 Hz, 2H), 7.35 (m, <i>J</i> = 8.4, 6.6 Hz, 1H), 7.13–6.97 (m, 3H), 4.77 (d, <i>J</i> = 5.7 Hz, 2H), 3.31 (s, 1H), 2.41 (s, 3H).</div></div><div id="sec5_9_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 7-(Benzenesulfonyl)-<i>N</i>-[(3,5-difluorophenyl)methyl]-2-methyl-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine (<b>50</b>)</h4><div class="NLM_p last">To a microwave vial was added <b>49</b> (1 g 2 mmol), bis(pinacolato)diboron (617 mg, 2.43 mmol, 1.2 equiv), and KOAc (600 mg 6 mmol 3 equiv), and the mixture was dissolved in THF (15 mL) and sparged with a stream of N<sub>2</sub> for 10 min. PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub> (70 mg, 5 mol %) was then added, and the mixture was heated to 140 °C for 1 h. LCMS showed the target product and also some des-bromo byproduct. The cooled reaction mixture was diluted with ethyl acetate (50 mL) and washed with saturated brine solution, and the organics were dried over anhydrous magnesium sulfate. The filtered organics were reduced in volume under vacuum and dry-loaded onto ISOLUTE by evaporation. The product was purified via automated chromatography, eluting gradient 0–20% ethyl acetate/iso-hexane, and fractions were evaporated to furnish the title compound as a colorless solid, 0.68 g (yield 61%). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 541[M + H]<sup>+</sup>.</div></div><div id="sec5_9_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> 4-[7-(Benzenesulfonyl)-4-{[(3,5-difluorophenyl)methyl]amino}-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl]pyrimidin-2-amine (<b>51</b>)</h4><div class="NLM_p last"><b>50</b> (915 mg, 1.69 mmol), 2-amino-4-chloropyrimidine (329 mg, 2.54 mmol, 1.5 equiv), and K<sub>2</sub>CO<sub>3</sub> (700 mg 5 mmol, 3 equiv) were combined in a microwave vial and dissolved in a mixture of THF (16 mL)/water (4 mL). The mixture was sparged with a stream of N<sub>2</sub> for 10 min before treatment with Pd118 (55 mg 5 mol %), and the reaction mixture was heated to 90 °C for 2 h in a CEM microwave. The reaction mixture was diluted with dichloromethane (200 mL) and washed with water (200 mL). The organics were separated, dried over MgSO4, and concentrated under vacuum. The residue was triturated with acetonitrile to furnish a light beige solid, 551 mg (yield 64%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.44 (s, 1H), 8.29 (d, <i>J</i> = 5.3 Hz, 1H), 8.20–8.13 (m, 2H), 7.80–7.71 (m, 1H), 7.65 (dd, <i>J</i> = 8.5, 7.1 Hz, 2H), 7.37 (d, <i>J</i> = 5.4 Hz, 1H), 7.30 (m, <i>J</i> = 14.8, 8.4, 6.5 Hz, 1H), 7.01 (t, <i>J</i> = 7.9 Hz, 2H), 6.70 (s, 2H), 4.90 (d, <i>J</i> = 5.9 Hz, 2H), 3.31 (s, 1H), 2.38 (s, 3H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 508[M + H]<sup>+</sup>.</div></div><div id="sec5_9_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> 4-(4-{[(3,5-Difluorophenyl)methyl]amino}-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl)pyrimidin-2-amine (<b>35</b>)</h4><div class="NLM_p last">To a suspension of <b>51</b> (550 mg, 1.08 mmol) in methanol (10 mL) was added K<sub>2</sub>CO<sub>3</sub> (∼1.4 g, 10 mmol, excess) and stirred at room temperature for 16 h. The suspension was filtered, and the solids were washed with further methanol. The solid residue was then washed several times with water to remove the remaining inorganics, and the remaining solid was air-dried on the filter thoroughly before lightly washing with hexane. The resultant solid was dried under a vacuum oven at 60 °C to give the title compound as a colorless solid, 159 mg, (yield 40%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 11.96 (s, 1H), 8.14 (d, <i>J</i> = 5.41 Hz, 1H), 8.04 (s, 1H), 7.33 (m, <i>J</i> = 7.25 Hz, 1H), 7.12 (d, <i>J</i> = 5.44 Hz, 1H), 7.05 (t, <i>J</i> = 7.80 Hz, 2H), 6.34 (s, 2H), 4.91 (d, <i>J</i> = 5.84 Hz, 2H), 3.39 (s, 1H), 2.36 (s, 3H), 1.24 (s, 1H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 368[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>18</sub>H<sub>15</sub>N<sub>7</sub>F<sub>2</sub>+H]<sup>+</sup>, 368.1357; found, 368.1413.</div></div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> 4-[4-(2<i>H</i>-1,3-benzodioxol-5-yl)-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl]pyridine-2-amine (<b>37</b>). Step 1</h3><div id="sec5_10_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 4-(2<i>H</i>-1,3-benzodioxol-5-yl)-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>52</b>)</h4><div class="NLM_p last">A mixture of 4-chloro-2-methyl-7h-pyrrolo[2,3-<i>d</i>]pyrimidine (Combi-Blocks ST-3008) (502 mg, 3 mmol), 3,4-(methylenedioxy)-phenyl boronic acid (522 mg, 3.15 mmol), and (2 g, 6 mmol, 2 equiv) of cesium carbonate was treated with a spatula tip of the palladium(dppf)Cl<sub>2</sub> catalyst, and the whole mixture was suspended in 10 mL of tetrahydrofuran and 1 mL of water in a microwave vessel. The whole mixture was degassed by a stream of nitrogen for 10 min. The sealed vessel and contents were heated to 130 °C for 45 min, and the reaction mixture was allowed to cool. LCMS showed a single product peak. The reaction mixture was diluted with 150 mL of water, the solid was collected by filtration, and the filter cake was dried under vacuum in air on a sinter funnel to furnish the title compound, 750 mg, as a colorless solid (yield quant). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 254[M + H]<sup>+</sup>.</div></div><div id="sec5_10_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> <i>tert</i>-Butyl 4-(2<i>H</i>-1,3-benzodioxol-5-yl)-5-bromo-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine-7-carboxylate (<b>53</b>)</h4><div class="NLM_p last"><b>52</b> (750 mg 2.96 mmol) was dissolved in anhydrous DMF under N<sub>2</sub> and cooled to 0 °C. <i>N</i>-bromosuccinamide (600 mg 3.35 mmol, 1.1 equiv) was then added portionwise over ∼15 min. The reaction mixture was allowed to stir, slowly warming to room temperature over 2 h. The mixture was then treated with boc-anhydride (800 mg 3.67 mmol 1.2 equiv), DMAP (5 mg 0.01 equiv), and triethylamine, and stirring at room temperature was continued for a further 16 h. Water was added, and the mixture was extracted with ethyl acetate twice. The combined organics were washed with saturated brine solution (aq) and dried over anhydrous magnesium sulfate. The filtered organics were concentrated under vacuum and purified via automated column chromatography on 24 g SiO<sub>2</sub> cartridge eluting hexane—55% ethyl acetate/hexane mixture. Selected fractions were evaporated, and the resultant solid was dried under vacuum to furnish the title compound as off-white powder, 697 mg (yield 54%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.00 (s, 1H), 7.23–7.13 (m, 2H), 7.07 (d, <i>J</i> = 8.0 Hz, 1H), 6.14 (s, 2H), 2.72 (s, 3H), 1.63 (s, 9H). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: 431.8[M + H]<sup>+</sup>.</div></div><div id="sec5_10_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 4-[4-(2<i>H</i>-1,3-benzodioxol-5-yl)-2-methyl-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl]pyridine-2-amine (<b>37</b>)</h4><div class="NLM_p last"><b>53</b> (432 mg 1 mmol), K<sub>2</sub>CO<sub>3</sub> (415 mg 3 mmol 3 equiv), and boc-2-aminopyridine-4-boronic acid pinacol ester (Combi-Blocks PN-6264) (352 mg 1.1 mmol 1.1 equiv) were combined and dissolved in a mixture of THF (15 mL) and water (4 mL). The mixture was sparged with N<sub>2</sub> for 10 min, and the catalyst Pd118 (33 mg, 5 mol %) was then added. The whole mixture was then heated to 65 °C for 16 h. The reaction mixture was diluted with dichloromethane and washed with water, followed by saturated brine solution. The organics were then dried over anhydrous magnesium sulfate and concentrated under vacuum. The residue was purified via automated column chromatography on the 24 g SiO<sub>2</sub> cartridge eluting neat hexane to 10% ethyl acetate/hexane. The purified intermediate boc-protected compound was dissolved in methanol (2 mL) and treated with 1.25 M HCl in MeOH (4 mL, 5 mmol, 5 equiv), and the solution was heated in a CEM microwave at 90 °C for 1 h. Solvents were then removed <i>in vacuo</i>, residues were taken up in 2:1 dichloromethane/isopropanol and washed with saturated sodium bicarbonate solution (aq), and the organics were dried over anhydrous magnesium sulfate and evaporated to dryness. The residue solid was triturated in diethyl ether and collected by filtration. The solid filter cake was dried under vacuum to furnish the title compound as a colorless solid, 95 mg (yield 27%).<sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 12.28 (s, 1H), 7.68 (s, 1H), 7.60 (dd, <i>J</i> = 0.72, 5.30 Hz, 1H), 6.98 (d, <i>J</i> = 1.66 Hz, 1H), 6.85 (dd, <i>J</i> = 1.68, 8.04 Hz, 1H), 6.74 (d, <i>J</i> = 8.02 Hz, 1H), 6.16 (dd, <i>J</i> = 0.77, 1.57 Hz, 1H), 6.02 (s, 2H), 6.05–5.98 (m, 1H), 5.65 (s, 2H), 2.69 (s, 3H). MS (ESI<sup>+</sup>) <i>m/z</i>: 346[M + H]<sup>+</sup>. HRMS (ESI<sup>+</sup>/APCI): calcd for [C<sub>19</sub>H<sub>15</sub>N<sub>5</sub>O<sub>2</sub>+H]<sup>+</sup>, 346.1226; found, 346.1127.</div></div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> Protein Production</h3><div class="NLM_p">Three protein constructs of DYRK1A were used for crystal structure determination.</div><div class="NLM_p">His-TEV-DYRK1A(127-485) was used to generate crystals of Dyrk1a with ADPNP, <b>3</b>, <b>8</b>, and <b>10</b> bound. This is the same construct as that used for the published structure (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VX3">2VX3</a><a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a>), removing the DYRK homology (DH) domain and the C terminus. Electron density was not observed for the initial <i>N</i>-terminal amino acids; a construct was therefore designed with slightly truncated N and C terminal sequences with a thrombin cleavage site (His-thrombin-DYRK1A(148-479-ala-ala)) which generated crystals of Dyrk1a with <b>1</b> and <b>6</b> bound. A further construct was designed (His-thrombin-DYRK1A(148–485)) for all other crystal structures reported here.</div><div class="NLM_p">A clone codon optimized for expression in <i>Escherichia coli</i> for His–TEV–DYRK1A(127–485) in pET21a was generated by gene synthesis (Celtek BioScience, USA). This clone was used as a template for PCR to generate an <i>N</i>-terminal truncation ligated to pET15b, His-thrombin-DYRK1A(148–485), and subsequent site-directed mutagenesis (Agilent) yielded a C-terminal truncate, His-Thrombin-DYRK1A(148–479-ala-ala). <i>E. coli</i> BL21(DE3) pLysS were transformed with the DYRK1A encoding plasmid and cultured at 37 °C in Luria–Bertani (LB) supplemented with 100 μg/mL ampicillin. Once the culture had reached an O.D.600 nm of 0.4, the incubator temperature was reduced to 22 °C. At O.D.600 nm of 0.6, expression was induced by the addition of 1 mM IPTG (final concentration). The cells were pelleted after 16hrs and stored at −80 °C.</div><div class="NLM_p">A similar protein purification protocol was used for all constructs. Cell pellets were resuspended in lysis buffer [50 mM <i>N</i>-(2-hydroxyethyl)piperazine-<i>N</i>′-ethanesulfonic acid (HEPES) pH7.5, 500 mM NaCl, 5% glycerol containing ethylenediaminetetraacetic acid (EDTA)-free protease inhibitor cocktail (Roche) and DNase I (Sigma)] and lysed by mechanical homogenization. The lysate was centrifuged at 20,000 rpm for 60 mins, 4 °C. The clarified lysate was applied to a 1 mL HiTrap chelating column (Cytiva) charged with Ni<sup>2+</sup> pre-equilibrated with lysis buffer. After washing the column, the protein of interest was eluted using a series of step gradients of lysis buffer + 500 mM imidazole. Sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE) analysis confirmed the fractions of interest which were pooled and concentrated using an Amicon Ultra 30 kDa MWCO concentrator (Merck Millipore). The sample was applied to a HiLoad 16/60 Superdex 75 pg size exclusion chromatography column (Cytiva) pre-equilibrated in 50 mM HEPES pH7.5, 500 mM NaCl, and 5 mM DTT. Samples were analyzed by SDS-PAGE and concentrated using an Amicon 30 kDa MWCO concentrator (Merck Millipore) to 18 mg/mL. The purified protein was aliquoted and snap-frozen prior to being stored at −80 °C.</div><div class="NLM_p last">If required, the <i>N</i>-terminal His-tag was removed by incubation overnight at 4 °C, with AcTEV protease (ThermoFisher) at 1 U/3 μg purified protein after affinity chromatography. A repeated affinity purification resulted in the protein of interest in the nonabsorbed fraction before size exclusion chromatography was performed.</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> Fragment Screen</h3><div class="NLM_p last">All protein NMR spectra were measured on a 600 MHz Bruker AVANCE III spectrometer fitted with a cryoprobe. Ligand-observed fragment screening was performed with 500 μM of each compound, 10 μM Dyrk1a (produced from His-TEV-DYRK1A(127–485) with the tag removed) in 20 mM HEPES pH 7.4, and 15 μM NaCl, in mixtures of six compounds per sample as described previously.<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a></div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> Crystallization of Dyrk1a</h3><div class="NLM_p">Crystal structures of different ligands complexed to Dyrk1a were obtained using the following protein constructs—AMPNP; compounds 3, 8, 10, and 34 with His-TEV-DYRK1A (aa127-485); compounds 1 and 6 with His-Thrombin-DYRK1A (148-479-Ala-Ala); and all others with His-Thrombin -DYRK1A (aa127-485).</div><div class="NLM_p">For all compounds other than <b>34</b>, crystals were prepared by replacement of AMPNP from Dyrk1a/AMPNP crystals. Purified Dyrk1a protein (all constructs) at ∼15 mg/mL concentration was mixed with fourfold excess (molar ratio) of AMPNP in vapor diffusion experiments. The best crystals were obtained from a solution containing 0.1 M MES buffer at pH 6.5, 12% Peg3350, and 0.2 M MgCl<sub>2</sub> at 20 °C. To replace the AMPNP, the crystals were soaked for ∼24 h in a crystallization solution containing 4–10 mM ligand (final DMSO concentration of 10–20%).</div><div class="NLM_p last">Crystals of the complex of Dyrk1a with <b>34</b> were obtained from a protein solution at ∼15 mg/ml mixed with approximately fourfold excess of <b>34</b> (final DMSO concentration of 8%) that had been incubated overnight. Crystals appeared from vapor diffusion experiments, with the best crystals obtained from conditions containing 0.1 M Hepes buffer at pH 7.5, 20% Peg4 k, and 0.2 M MgCl<sub>2</sub>, also at 20 °C.</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> Crystallographic Data Collection</h3><div class="NLM_p last">Crystals were cryoprotected in a solution of crystallization buffer enriched in glycerol up to 20–25% and flash frozen in liquid nitrogen. <a href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_001.pdf" class="ext-link">Table S1</a> summarizes the data collection statistics, with data obtained from ESRF (beamline id23-1) or DLS (i02, i03, and i04 beamlines). Data were processed and scaled with DENZO/Scalepack package<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> or XDS<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> software.</div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> Structure Determination</h3><div class="NLM_p">Structures of Dyrk1a complexes were solved by molecular replacement (MolRep<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a>) using the coordinates from PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VX3">2VX3</a><a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> as a model.</div><div class="NLM_p last">The refined structure of the relevant protein construct (stripped of the compound) was used as a starting model for solving (by molecular replacement) subsequent structures of the complexes with ligands. After structure solution, several cycles of refinement were conducted with Refmac5<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> software from the CCP4 package,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> alternating with manual rebuilding using the program COOT.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Compounds were modeled in the surplus peaks in the difference Fourier electron density maps. Topology files for the ligands were created by ProDrug,<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> also from within the CCP4 package. Refinement statistics for each new structure reported in this paper are in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_001.pdf" class="ext-link">Table S2</a>.</div></div><div id="sec5_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> Protein Kinase Assays</h3><div class="NLM_p">A number of different assay formats were used to measure inhibition of different kinases by compounds described in this paper.</div><div class="NLM_p">ADP Hunter Plus (DiscoverX) was used to measure the activity of the protein kinases DYRK1A, PAK4, Pim1, Aurora A, and DYRK2.</div><div class="NLM_p">A TR-FRET assay was used to measure the activity of the protein kinases DYRK1A, DYRK1B, DYRK2, DYRK3, DYRK4, CDK9, and CLK1.</div><div class="NLM_p">The <i>K</i><sub>d</sub> for binding of compounds to kinases was determined in an 11-point threefold serial dilution in the KINOMEscan assay available at DiscoveRx (now Eurofins).</div><div class="NLM_p">A radiometric assay was used to measure the activity of the protein kinase GSK3-β.</div><div class="NLM_p">The inhibition of TTK was measured at CEREP (now Eurofins).</div><div id="sec5_16_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> ADP Hunter Plus Assay</h4><div class="NLM_p last">The ADP Hunter Plus assay measures the accumulation of ADP. Protein kinase assays were performed in 96-well black plates (corning #3686) at 30 °C in a final volume of 20 μL in 50 mM Hepes, pH 7.4, 20 mM NaCl, 10 mM MgCl<sub>2</sub>, 1 mM EGTA, 0.01 mg/mL BGG, 0.02% Tween 20, and 2% DMSO. The order of the addition of reagents to the assay plate was 5 μL of compound/8% DMSO, 10 μL of kinase/substrate mix, and 5 μL of ATP. Final concentrations and assay conditions used are described in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_001.pdf" class="ext-link">Table S3</a>. Following incubation, 5 μL of Reagent A was added, followed by 10 μL of Reagent B and plate incubated for a further 20 min at 30 °C. Finally, 5 μL of the stop reagent was added, and the plate was read on a Molecular Devices flexstation plate reader using Ex/Em 530/590 nm in conjunction with a 585 nm cutoff dichroic mirror.</div></div><div id="sec5_16_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> TR-FRET Assay</h4><div class="NLM_p last">The TR-FRET assay was performed on a TECAN Evo robotic platform. The compound at 5% DMSO was added to reaction buffer (50 mM Hepes pH7.4, 1 mM EGTA, 10 mM MgCl<sub>2</sub>; 2 mM DTT & 0.01% Tween20) containing 20 ng/μL DYRK1A Kinase (Carna Biosciences #04–130), 100 μM Ulight-MBP substrate (PerkinElmer #TRF109L), and 10 μM ATP cosubstrate (Sigma-Aldrich #A2383) and incubated for 40 min at room temperature. The kinase reaction was stopped by adding 10 mM EDTA (Sigma-Aldrich #E7889) and 1 nM anti-phospho-MBP (PerkinElmer #TRF0201L) and incubated for 1 h minimum at room temperature. Plates were read on a multimode reader (PerkinElmer Envision 2400) with an excitation wavelength of 337 nm and emission wavelengths of 620 nm for the donor and 665 nm for the acceptor. A similar format was used for other TR-FRET kinase inhibition assays.</div></div><div id="sec5_16_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> Radiometric Assay</h4><div class="NLM_p last">The radiometric assay for glycogen synthase 3β (GSK3β) inhibition was performed in 96-well plates (Greiner; VWR #651201) at 30 °C for 40 min in a final volume of 50 μL where reagents were added in the following order: 1 μL of compound/100% DMSO, 39 μL of kinase/substrate mix, and 10 μL of ATP/<sup>33</sup>P ATP mix. Final concentrations used were 39 nM GSK3β (Merck #14–306), 16.5 μM GSK3βtide substrate (YRRAAVPPSPSLSRHSSPHQ(pS)EDEEE), 10 μM ATP, 0.5μCi <sup>33</sup>P ATP, 50 mM Hepes, pH 7.5, 15 mM Mg(C<sub>2</sub>H<sub>3</sub>O<sub>2</sub>)<sub>2</sub>, 0.01 mg/mL bovine serum albumin (BSA), and 2% DMSO. Following incubation, the assay was stopped by the addition of 50 μL of 50 mM phosphoric acid solution and 90 μL of assay reaction transferred to a prewetted filter plate, washed three times with 100 μL of 50 mM phosphoric acid solution, dried, and counted in the presence of 50 μL of scintillation fluid on a Trilux plate reader.</div></div></div><div id="sec5_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> DYRK1A (pS520) Cell Assay</h3><div class="NLM_p last">U2OS cells were transiently transfected with a plasmid expressing full length wild-type DYRK1A (pcDNA3.1). After 5 h, compounds were added, and cells were incubated for a further 5 h. Cells were lysed, and equal amounts of protein were separated by SDS-PAGE and transferred to the poly(vinylidene difluoride) membrane. Membranes were probed for pS520 DYRK1A (custom rabbit-polyclonal antibody immunized with the synthetic peptide SNSGRARpSDPTHQHR, Eurogentec; 1:500 in 5% BSA-TBST) and detected with anti-rabbit IgG-HRP (7074, CST; 1:3000) and then quantified on an electrochemical luminescence camera (BioRad). Membranes were subsequently stripped and reprobed with a total anti-DYRK1A antibody (H0000-1859, Abnova; 1:1000 in 5% BSA-TBST) and quantified as mentioned above.</div></div><div id="sec5_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> Caco2—Papp Assay</h3><div class="NLM_p last">The Caco-2 cell line was cultured in adapted media and used between 21 and 30 days post seeding. The compound was incubated at 20 μM in the apical chamber in HBSS buffer containing 0.1% BSA. Samples (duplicate) were collected at 30 and 90 min in the basolateral chamber and analyzed by HPLC-LC/MS/MS. The apparent permeability was calculated as follows: Papp (cm/s) = K·Vr/A·60, where <i>V</i><sub>r</sub> is the volume of the receiver chamber (mL), <i>A</i> is the surface area of the membrane (cm<sup>2</sup>), and <i>K</i> is the slope of the cumulated absorbed fraction as a function of time (min<sup>–1</sup>).</div></div><div id="sec5_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> Metabolic Stability (Mouse/Rat/Human)</h3><div class="NLM_p">Pooled liver microsomes from mouse (CD-1), rat (Sprague–Dawley), and human liver microsomes (mixed gender) are used with a final microsomal protein concentration of 0.3 mg/mL. The compound (0.1 μM) was tested in duplicate. After preincubation with pooled liver microsomes (10 min, 37 °C), the reaction is initiated by adding a reduced nicotinamide adenine dinucleotide phosphate-generating system and incubated in a 37 °C shaking water bath. Samples were collected at different time points (0, 7, 17, 30, and 45 min), and the reaction was stopped by a mixture of 20 nM antipyrine and 0.1% formic acid in acetonitrile/methanol/water (35/35/30, v/v/v). Samples were analyzed by HPLC-MS/MS.</div><div class="NLM_p last">The hepatic blood clearance was extrapolated using the following equation<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_m001.gif" alt="" id="_i86" /></img></span>where LW is the liver weight (g), sc is the scaling factor (g protein/g liver), <i>Q</i><sub>h</sub> is the hepatic blood flow (mL/min), and <i>f</i><sub>u</sub> is the blood unbound fraction (1 by default), and the intrinsic clearance (Cl<sub>int</sub>) is as follows<span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_m002.gif" alt="" id="_i87" /></img></span></div></div><div id="sec5_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> Plasma protein Binding (Mouse/Human) Assay</h3><div class="NLM_p">Each pool of mouse and human plasma was spiked with the compound to obtain a final concentration of 5 μM. Dialysis was performed (in triplicate) using 300 μL of each plasma spiked (donor chamber) and 500 μL of phosphate buffer pH 7.4 (receiver chamber). All replicates were incubated for 3 h at 37 °C using rapid equilibrium dialysis (device-Pierce) under orbital stirring. All plasma and buffer samples (40 μL) were analyzed with the HPLC-LC-MS/MS method.</div><div class="NLM_p">The extent of plasma protein binding in each sample was calculated as follows<span class="NLM_disp-formula" id="ueq3"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_m003.gif" alt="" id="_i89" /></img></span></div><div class="NLM_p last">Where <i>f</i><sub>u</sub> is the unbound fraction expressed as a percentage, <i>C</i><sub>pl</sub> is the concentration of the compound in the plasma at the end of dialysis, and <i>C</i><sub>u</sub> is the concentration of the unbound compound in the buffer. The unbound fraction (<i>f</i><sub>u</sub> %) was determined as the mean of the unbound fraction calculated from the triplicate.</div></div><div id="sec5_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> hERG Assay</h3><div class="NLM_p last">The effect of compounds on hERG activity was studied at room temperature (22 ± 2 °C) on hERG currents in HEK-293 cells stably expressing the hERG potassium channel. Cells were maintained at a potential of −80 mV, and hERG currents were activated by cell depolarization (+30 mV, 1 s) followed by cell repolarization (40 mV, 1 s). Cells were incubated with DMSO 0.1% in extracellular solution for 10 min (baseline) followed by three increasing consecutive concentrations of the compound (1, 3, and 10 μmol/L, <i>n</i> = 4) until the steady state was reached. The hERG tail current amplitude was determined by the difference between the tail current amplitude upon repolarization to −40 mV and the base current amplitude at −80 mV. The percentage of inhibition was determined as compared to the baseline.</div></div><div id="sec5_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> <i>In Vivo</i> Efficacy Studies</h3><div class="NLM_p">Female nude mice (Crl/NU(NCr)-<i>Foxn1nu</i>) were purchased from Charles River Laboratories. Xenograft studies were undertaken by Charles River Laboratories Discovery Services, North Carolina, and accredited by the Association for Assessment and Accreditation of Laboratory Animal Care International. The U87MG tumor line was maintained by serial SC transplantation in female athymic nude mice. Tumor fragments, approximately 1 mm<sup>3</sup> each, were implanted SC into the right flank of each animal and allowed to grow toward a target size of 100–150 mm<sup>3</sup>. On day 1 of the study, tumors were randomized into treatment groups before compound administration. For tumor PK studies, the tumors were allowed to reach a target volume of 300 mm<sup>3</sup> prior to randomization.</div><div class="NLM_p">Compound <b>34</b> was formulated in 60 mM HCl and 20% hydroxypropyl-β-cyclodextrin and administered by oral gavage once daily for 21 days. The tumor size was measured with electronic calipers, and the tumor volume was calculated according to the formula ((width × width) × length)/2. Body weight was measured daily for the first 5 days and then twice weekly thereafter. The study endpoint was defined as a tumor volume of 2000 mm<sup>3</sup> or D28, whichever came first. Each animal was euthanized when its tumor reached the endpoint, and the last day of the study was D28.</div><div class="NLM_p">The time to endpoint (TTE) for each mouse was calculated from the following equation: TTE = log 10 (endpoint volume) – <i>b</i>/<i>m</i> where <i>b</i> is the intercept and <i>m</i> is the slope of the line obtained by linear regression of a log-transformed tumor growth data set. The data set is composed of the first observation that exceeded the study endpoint volume and the three consecutive observations that immediately preceded the attainment of the endpoint volume. Animals that did not reach endpoint were euthanized at the end of the study and assigned a TTE value equal to the last day of the study (D28). Animals determined to have died from treatment-related (TR) causes were assigned a TTE value equal to the day of death. If the animal died from nontreatment-related causes, it was excluded from the TTE calculations.</div><div class="NLM_p last">The treatment outcome is evaluated by TGD, which is defined as the increase in the median TTE in a treatment group compared to the control group: TGD = <i>T</i> – <i>C</i> expressed in days or as a percentage of the median TTE of the control group: % TGD = (<i>T</i> – <i>C</i>)/<i>C</i> × 100 where <i>T</i> = median TTE for a treatment group and <i>C</i> = median TTE for the control group.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i92"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00024" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81144" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81144" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00024?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00024</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Supporting figures, tables and data: Compound <b>34</b> target engagement with DYRK1A demonstrated in nano-BRET assay; comparison of the conformation of equivalent leucine residues in the kinase hinge of CLK1 and Dyrk1a structures; in vivo PK characterisation of compound <b>34</b> in mice; compound <b>34</b> exhibits activity against U87MG glioblastoma cells in vitro; comparison of binding pose of <b>34</b> with recently published inhibitors of DYRK1A; crystallographic data collection; refinement statistics; final concentrations of reagents in the kinase inhibition assays; kinase profiling of <b>34</b>; brain penetrance of <b>34</b>; in vivo PK; complete list of kinome screen data; LC chromatograms, and 1H NMR spectra of key compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_001.pdf">jm1c00024_si_001.pdf (2.59 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_002.csv">jm1c00024_si_002.csv (2.19 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00024" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37548" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37548" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Roderick E. Hubbard</span> - <span class="hlFld-Affiliation affiliation">Vernalis
(R&D) Ltd., Granta
Park, Cambridge CB21 6GB, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-8233-7461" title="Orcid link">https://orcid.org/0000-0002-8233-7461</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#34461a5c415656554650744251465a55585d471a575b59"><span class="__cf_email__" data-cfemail="11633f79647373706375516774637f707d78623f727e7c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Lee Walmsley</span> - <span class="hlFld-Affiliation affiliation">Vernalis
(R&D) Ltd., Granta
Park, Cambridge CB21 6GB, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James B. Murray</span> - <span class="hlFld-Affiliation affiliation">Vernalis
(R&D) Ltd., Granta
Park, Cambridge CB21 6GB, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-1007-8218" title="Orcid link">https://orcid.org/0000-0003-1007-8218</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pawel Dokurno</span> - <span class="hlFld-Affiliation affiliation">Vernalis
(R&D) Ltd., Granta
Park, Cambridge CB21 6GB, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-7332-8889" title="Orcid link">https://orcid.org/0000-0002-7332-8889</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew J. Massey</span> - <span class="hlFld-Affiliation affiliation">Vernalis
(R&D) Ltd., Granta
Park, Cambridge CB21 6GB, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-0276-2573" title="Orcid link">https://orcid.org/0000-0002-0276-2573</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Karen Benwell</span> - <span class="hlFld-Affiliation affiliation">Vernalis
(R&D) Ltd., Granta
Park, Cambridge CB21 6GB, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrea Fiumana</span> - <span class="hlFld-Affiliation affiliation">Vernalis
(R&D) Ltd., Granta
Park, Cambridge CB21 6GB, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicolas Foloppe</span> - <span class="hlFld-Affiliation affiliation">Vernalis
(R&D) Ltd., Granta
Park, Cambridge CB21 6GB, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stuart Ray</span> - <span class="hlFld-Affiliation affiliation">Vernalis
(R&D) Ltd., Granta
Park, Cambridge CB21 6GB, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Julia Smith</span> - <span class="hlFld-Affiliation affiliation">Vernalis
(R&D) Ltd., Granta
Park, Cambridge CB21 6GB, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Allan E. Surgenor</span> - <span class="hlFld-Affiliation affiliation">Vernalis
(R&D) Ltd., Granta
Park, Cambridge CB21 6GB, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-9869-1430" title="Orcid link">https://orcid.org/0000-0002-9869-1430</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas Edmonds</span> - <span class="hlFld-Affiliation affiliation">Institut
de Recherches Servier, 125 Chemin de Ronde, Croissy-sur-Seine 78290, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Didier Demarles</span> - <span class="hlFld-Affiliation affiliation">Technologie
Servier, 27 Rue Eugène
Vignat, Orleans 45000, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mike Burbridge</span> - <span class="hlFld-Affiliation affiliation">Institut
de Recherches Servier, 125 Chemin de Ronde, Croissy-sur-Seine 78290, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Francisco Cruzalegui</span> - <span class="hlFld-Affiliation affiliation">Institut
de Recherches Servier, 125 Chemin de Ronde, Croissy-sur-Seine 78290, France</span>; 
    <span>Present Address:
                        Francisco Cruzalegui:Evotec (France) SAS 195 Route d’Espagne 31036 Toulouse, France E-mail: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="ef8c9d9a958e838a889a86c1899d8e818c869c8c80af809d8e81888ac1899d">[email protected]</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andras Kotschy</span> - <span class="hlFld-Affiliation affiliation">Servier
Research Institute of Medicinal Chemistry, Záhony u. 7., Budapest H-1031, Hungary</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-7675-3864" title="Orcid link">https://orcid.org/0000-0002-7675-3864</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">The authors thank co-workers at the Analytical Division of the Servier Research Institute of Medicinal Chemistry for providing the detailed chemical analysis of the compounds and fromVernalis, and we thank Hayley Angove for assistance with the assay protocols.</p><br /></br><p class="inlineNote">The X-ray structures of compounds complexed with Dyrk1a reported in this paper have been deposited in the PDB with the following codes. ADPNP: 7A4O, <b>1</b>: 7A4R, <b>2</b>: 7A4S, <b>3</b>: 7A4W; <b>4</b>: 7A4Z, <b>5</b>: 7A51, <b>6</b>: 7A52, <b>7</b>: 7A53, <b>8</b>: 7A55, <b>10</b>: 7A5B, <b>16</b>: 7A5D, <b>24</b>: 7A5L, and <b>34</b>: 7A5N. The authors will release the atomic coordinates and experimental data upon article publication.</p></div></li></ul></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">DYRK</td><td class="NLM_def"><p class="first last">dual-specificity tyrosine-regulated kinase</p></td></tr><tr><td class="NLM_term">CMGC</td><td class="NLM_def"><p class="first last">cyclin-dependent, mitogen-activated, glycogen synthase, and CDC-like</p></td></tr><tr><td class="NLM_term">EGFR</td><td class="NLM_def"><p class="first last">epidermal growth factor</p></td></tr><tr><td class="NLM_term">GLI</td><td class="NLM_def"><p class="first last">glioma-associated oncogene</p></td></tr><tr><td class="NLM_term">Mcl-1</td><td class="NLM_def"><p class="first last">myeloid cell leukemia 1</p></td></tr><tr><td class="NLM_term">Bcl-2</td><td class="NLM_def"><p class="first last">B-cell lymphoma 2</p></td></tr><tr><td class="NLM_term">EGCG</td><td class="NLM_def"><p class="first last">epigallocatechin gallate</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">protein databank</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetic</p></td></tr><tr><td class="NLM_term">PD</td><td class="NLM_def"><p class="first last">pharmacodynamic</p></td></tr><tr><td class="NLM_term">PAK4</td><td class="NLM_def"><p class="first last">p21-associated kinase 4</p></td></tr><tr><td class="NLM_term">ADP</td><td class="NLM_def"><p class="first last">adenosine diphosphate</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved fluorescence energy transfer</p></td></tr><tr><td class="NLM_term">GSK3β</td><td class="NLM_def"><p class="first last">glycogen synthase 3β</p></td></tr><tr><td class="NLM_term">ADPNP</td><td class="NLM_def"><p class="first last">5′-adenylyl-β,γ-imidodiphosphate</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">HAC</td><td class="NLM_def"><p class="first last">heavy atom count</p></td></tr><tr><td class="NLM_term">ATP</td><td class="NLM_def"><p class="first last">adenosine triphosphate</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">ligand efficiency</p></td></tr><tr><td class="NLM_term">CDK9</td><td class="NLM_def"><p class="first last">cyclin-dependent kinase 9</p></td></tr><tr><td class="NLM_term">CLK</td><td class="NLM_def"><p class="first last">CDC-like kinase</p></td></tr><tr><td class="NLM_term">hERG</td><td class="NLM_def"><p class="first last">human ether-a-go-go-related gene</p></td></tr><tr><td class="NLM_term">HIPK2</td><td class="NLM_def"><p class="first last">homeodomain interacting protein kinase 2</p></td></tr><tr><td class="NLM_term">SCID</td><td class="NLM_def"><p class="first last">severe combined immunodeficiency; po, per os</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i98">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16287" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16287" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 55 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altafaj, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aranda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de la Luna, S.</span></span> <span> </span><span class="NLM_article-title">Dyrk1a autophosphorylation on serine residue 520 modulates its kinase activity via 14-3-3 binding</span>. <i>Mol. Biol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1167</span>– <span class="NLM_lpage">1178</span>, <span class="refDoi"> DOI: 10.1091/mbc.e06-08-0668</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1091%2Fmbc.e06-08-0668" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=17229891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2sXktVSmtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2007&pages=1167-1178&author=M.+Alvarezauthor=X.+Altafajauthor=S.+Arandaauthor=S.+de+la+Luna&title=Dyrk1a+autophosphorylation+on+serine+residue+520+modulates+its+kinase+activity+via+14-3-3+binding&doi=10.1091%2Fmbc.e06-08-0668"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">DYRK1A autophosphorylation on serine residue 520 modulates its kinase activity via 14-3-3 binding</span></div><div class="casAuthors">Alvarez, Monica; Altafaj, Xavier; Aranda, Sergi; de la Luna, Susana</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology of the Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1167-1178</span>CODEN:
                <span class="NLM_cas:coden">MBCEEV</span>;
        ISSN:<span class="NLM_cas:issn">1059-1524</span>.
    
            (<span class="NLM_cas:orgname">American Society for Cell Biology</span>)
        </div><div class="casAbstract">Dual-specificity tyrosine-phosphorylated and regulated kinase (DYRK) proteins are an evolutionarily conserved family of protein kinases, with members identified from yeast to humans, that participate in a variety of cellular processes.  DYRKs are serine/threonine protein kinases that are activated by autophosphorylation on a tyrosine residue in the activation loop.  The family member DYRK1A has been shown to phosphorylate several cytosolic proteins and a no. of splicing and transcription factors, including members of the nuclear factor of activated T cells family.  In the present study, we show that DYRK1A autophosphorylates, via an intramol. mechanism, on Ser-520, in the PEST domain of the protein.  We also show that phosphorylation of this residue, which we show is subjected to dynamic changes in vivo, mediates the interaction of DYRK1A with 14-3-3β.  A second 14-3-3 binding site is present within the N-terminal of the protein.  In the context of the DYRK1A mol., neither site can act independently of the other.  Bacterially produced DYRK1A and the mutant DYRK1A/S520A have similar kinase activities, suggesting that Ser-520 phosphorylation does not affect the intrinsic kinase activity on its own.  Instead, we demonstrate that this phosphorylation allows the binding of 14-3-3β, which in turn stimulates the catalytic activity of DYRK1A.  These findings provide evidence for a novel mechanism for the regulation of DYRK1A kinase activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3iiUoSe8NbbVg90H21EOLACvtfcHk0ljBnVeFQ4DaMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXktVSmtrc%253D&md5=7ed3a1f45bc539f97aaeea6972f9e78c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1091%2Fmbc.e06-08-0668&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1091%252Fmbc.e06-08-0668%26sid%3Dliteratum%253Aachs%26aulast%3DAlvarez%26aufirst%3DM.%26aulast%3DAltafaj%26aufirst%3DX.%26aulast%3DAranda%26aufirst%3DS.%26aulast%3Dde%2Bla%2BLuna%26aufirst%3DS.%26atitle%3DDyrk1a%2520autophosphorylation%2520on%2520serine%2520residue%2520520%2520modulates%2520its%2520kinase%2520activity%2520via%252014-3-3%2520binding%26jtitle%3DMol.%2520Biol.%2520Cell%26date%3D2007%26volume%3D18%26spage%3D1167%26epage%3D1178%26doi%3D10.1091%2Fmbc.e06-08-0668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mercer, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewton, D. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naqvi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, E.</span></span> <span> </span><span class="NLM_article-title">Mirk/dyrk1b mediates cell survival in rhabdomyosarcomas</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">5143</span>– <span class="NLM_lpage">5150</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-05-1539</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1158%2F0008-5472.CAN-05-1539" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=16707437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD28XksF2isrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=5143-5150&author=S.+E.+Mercerauthor=D.+Z.+Ewtonauthor=S.+Shahauthor=A.+Naqviauthor=E.+Friedman&title=Mirk%2Fdyrk1b+mediates+cell+survival+in+rhabdomyosarcomas&doi=10.1158%2F0008-5472.can-05-1539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Mirk/Dyrk1b Mediates Cell Survival in rhabdomyosarcomas</span></div><div class="casAuthors">Mercer, Stephen E.; Ewton, Daina Z.; Shah, Sejal; Naqvi, Asghar; Friedman, Eileen</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5143-5150</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Rhabdomyosarcoma is the most common sarcoma in children and is difficult to treat if the primary tumor is non-resectable or if the disease presents with metastases.  The function of the serine/threonine kinase Mirk was investigated in this cancer.  Mirk has both growth arrest and survival functions in terminally differentiating skeletal myoblasts.  Maintenance of Mirk growth arrest properties would cause down-regulation of Mirk in transformed myoblasts.  Alternatively, Mirk expression would be retained if rhabdomyosarcoma cells used Mirk survival capability.  Mirk expression was significant in 12 of 16 clin. cases of rhabdomyosarcoma.  Mirk was detected in each rhabdomyosarcoma cell line examd.  Mirk was a functional kinase in each of three rhabdomyosarcoma cell lines, where it proved to be more active than in C2C12 skeletal myoblasts.  Mirk mediated survival of the majority of clonogenic rhabdomyosarcoma cells.  Knockdown of Mirk by RNA interference reduced the fraction of RD and of Rh30 rhabdomyosarcoma cells capable of colony formation 3- to 4-fold in multiple expts.  Depletion of Mirk induced cell death by apoptosis, as shown by increased nos. of terminal deoxynucleotidyl transferase-mediated nick-end labeling-pos. cells and by increased binding of Annexin V.  Mirk is a stress-activated kinase that mediates expression of contractile proteins in differentiating myoblasts, but Mirk is not essential for muscle formation in the embryo.  It is likely that Mirk also facilitates survival of satellite cell-derived rhabdomyoblasts in regenerating skeletal muscle and aids their differentiation.  This survival function is maintained in rhabdomyosarcoma, where Mirk may be a novel therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXnJZi1ESdPLVg90H21EOLACvtfcHk0lgjKn06484QeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XksF2isrY%253D&md5=adf0683dd6a03347c2574d66f4b53451</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-05-1539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-05-1539%26sid%3Dliteratum%253Aachs%26aulast%3DMercer%26aufirst%3DS.%2BE.%26aulast%3DEwton%26aufirst%3DD.%2BZ.%26aulast%3DShah%26aufirst%3DS.%26aulast%3DNaqvi%26aufirst%3DA.%26aulast%3DFriedman%26aufirst%3DE.%26atitle%3DMirk%252Fdyrk1b%2520mediates%2520cell%2520survival%2520in%2520rhabdomyosarcomas%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D5143%26epage%3D5150%26doi%3D10.1158%2F0008-5472.can-05-1539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstein, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hornicek, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mankin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z.</span></span> <span> </span><span class="NLM_article-title">The kinase mirk is a potential therapeutic target in osteosarcoma</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">552</span>– <span class="NLM_lpage">558</span>, <span class="refDoi"> DOI: 10.1093/carcin/bgp330</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1093%2Fcarcin%2Fbgp330" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=20042639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkt1OlsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=552-558&author=C.+Yangauthor=D.+Jiauthor=E.+J.+Weinsteinauthor=E.+Choyauthor=F.+J.+Hornicekauthor=K.+B.+Woodauthor=X.+Liuauthor=H.+Mankinauthor=Z.+Duan&title=The+kinase+mirk+is+a+potential+therapeutic+target+in+osteosarcoma&doi=10.1093%2Fcarcin%2Fbgp330"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The kinase Mirk is a potential therapeutic target in osteosarcoma</span></div><div class="casAuthors">Yang, Cao; Ji, Diana; Weinstein, Edward J.; Choy, Edwin; Hornicek, Francis J.; Wood, Kirkham B.; Liu, Xianzhe; Mankin, Henry; Duan, Zhenfeng</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">552-558</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Osteosarcoma is the most common primary malignant bone tumor affecting children and adolescents.  The majority of patients are treated by surgery and chemotherapy but have limited alternative therapeutic options.  Kinases play an important role in the growth and survival of tumor cells.  We aim to identify specific kinases to be vital in the survival of osteosarcoma cells and thus may be a key target in creating novel anticancer therapies.  A lentiviral short hairpin RNA kinase library, screened osteosarcoma cells, identified kinase minibrain-related kinase (Mirk) (Dyrk1B) as a potential target.  Knockdown Mirk expression could inhibit cell growth and induce apoptosis.  Chem. synthetic small interfering RNA knockdown and complementary DNA rescue assay further confirmed the results from the decrease of Mirk gene expression.  The relationship between Mirk gene expression and the clin. characteristics of patients with osteosarcoma was investigated using tissue microarray and immunohistochem. anal.  The data indicate that the overall survival rate of patients with Mirk high staining (high levels of Mirk protein expression) is significantly shorter than those with Mirk low staining and moderate staining.  This highlights Mirk's potential to serve as a promising target for mol. therapy in the treatment of osteosarcoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtj9Sf-jca27Vg90H21EOLACvtfcHk0lgjKn06484QeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkt1OlsL0%253D&md5=7c933da4b2b727f81e141036f2b26acf</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp330%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.%26aulast%3DJi%26aufirst%3DD.%26aulast%3DWeinstein%26aufirst%3DE.%2BJ.%26aulast%3DChoy%26aufirst%3DE.%26aulast%3DHornicek%26aufirst%3DF.%2BJ.%26aulast%3DWood%26aufirst%3DK.%2BB.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DMankin%26aufirst%3DH.%26aulast%3DDuan%26aufirst%3DZ.%26atitle%3DThe%2520kinase%2520mirk%2520is%2520a%2520potential%2520therapeutic%2520target%2520in%2520osteosarcoma%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26spage%3D552%26epage%3D558%26doi%3D10.1093%2Fcarcin%2Fbgp330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schell, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bepler, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haura, E. B.</span></span> <span> </span><span class="NLM_article-title">Mirk/dyrk1b, a novel therapeutic target, mediates cell survival in non-small cell lung cancer cells</span>. <i>Canc. Biol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1671</span>– <span class="NLM_lpage">1679</span>, <span class="refDoi"> DOI: 10.4161/cbt.8.17.9322</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.4161%2Fcbt.8.17.9322" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=19633423" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlslCiu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=1671-1679&author=J.+Gaoauthor=Z.+Zhengauthor=B.+Rawalauthor=M.+J.+Schellauthor=G.+Beplerauthor=E.+B.+Haura&title=Mirk%2Fdyrk1b%2C+a+novel+therapeutic+target%2C+mediates+cell+survival+in+non-small+cell+lung+cancer+cells&doi=10.4161%2Fcbt.8.17.9322"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Mirk/Dyrk1B, a novel therapeutic target, mediates cell survival in non-small cell lung cancer cells</span></div><div class="casAuthors">Gao, Jingchun; Zheng, Zhong; Rawal, Bhupendra; Schell, Michael J.; Bepler, Gerold; Haura, Eric B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Biology & Therapy</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">1671-1679</span>CODEN:
                <span class="NLM_cas:coden">CBTAAO</span>;
        ISSN:<span class="NLM_cas:issn">1538-4047</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Minibrain-related kinase (Mirk) is a member of the dual specificity tyrosine-phosphorylation-regulated kinase (Dyrk)/minibrain family of dual-specificity protein kinases and is identical to Dyrk1B.  Mirk/Dyrk1B is a serine/threonine kinase that has been found to be upregulated in solid tumors and mediates cell survival in colon cancer, pancreatic ductal adenocarcinoma and rhabdomyosarcomas.  There is little known about Mirk in lung cancer.  In the present study, we showed that Mirk protein was widely overexpressed in 13 of 19 NSCLC cell lines.  Mirk immunopptn. coupled with anti-phosphotyrosine western blotting confirmed tyrosine phosphorylation of Mirk in NSCLC cells.  Mirk knockdown by small interfering RNA induced cell apoptosis concomitant with upregulation of Bak, a Bcl-2 family member, and downregulation of signal transducers and activators of transcription 3 (STAT3) tyrosine phosphorylation.  Mirk knockdown led to decreased cell colony formation in vitro as well as delayed tumor growth in an orthotopic mouse model and sensitized cells to cisplatin-induced apoptosis.  Using automated quant. detn. of the Mirk protein in tumor specimens of patients with early-stage lung cancer, overexpression of Mirk was found in nearly 90% of tumor specimens in both the cytoplasm and nucleus.  These results suggest that Mirk is overexpressed in lung cancer, acts as a survival factor in lung cancer cells and may be a novel therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1x545JhuAN7Vg90H21EOLACvtfcHk0lgjKn06484QeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlslCiu7s%253D&md5=ff77dbf7fd354ccdd3723a86c3cd79f2</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.4161%2Fcbt.8.17.9322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcbt.8.17.9322%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DZ.%26aulast%3DRawal%26aufirst%3DB.%26aulast%3DSchell%26aufirst%3DM.%2BJ.%26aulast%3DBepler%26aufirst%3DG.%26aulast%3DHaura%26aufirst%3DE.%2BB.%26atitle%3DMirk%252Fdyrk1b%252C%2520a%2520novel%2520therapeutic%2520target%252C%2520mediates%2520cell%2520survival%2520in%2520non-small%2520cell%2520lung%2520cancer%2520cells%26jtitle%3DCanc.%2520Biol.%2520Ther.%26date%3D2009%26volume%3D8%26spage%3D1671%26epage%3D1679%26doi%3D10.4161%2Fcbt.8.17.9322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewton, D. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naqvi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercer, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, E.</span></span> <span> </span><span class="NLM_article-title">The kinase mirk/dyrk1b mediates cell survival in pancreatic ductal adenocarcinoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">4149</span>– <span class="NLM_lpage">4158</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-05-3089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1158%2F0008-5472.CAN-05-3089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=16618736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsFSlt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=4149-4158&author=X.+Dengauthor=D.+Z.+Ewtonauthor=S.+Liauthor=A.+Naqviauthor=S.+E.+Mercerauthor=S.+Landasauthor=E.+Friedman&title=The+kinase+mirk%2Fdyrk1b+mediates+cell+survival+in+pancreatic+ductal+adenocarcinoma&doi=10.1158%2F0008-5472.can-05-3089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The kinase Mirk/Dyrk1B mediates cell survival in pancreatic ductal adenocarcinoma</span></div><div class="casAuthors">Deng, Xiaobing; Ewton, Daina Z.; Li, Sheena; Naqvi, Asghar; Mercer, Stephen E.; Landas, Steve; Friedman, Eileen</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4149-4158</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Ductal adenocarcinoma of the pancreas is almost uniformly lethal as this cancer is invariably detected at an advanced stage and is resistant to treatment.  The serine/threonine kinase Mirk/Dyrk1B has been shown to be antiapoptotic in rhabdomyosarcomas.  We have now investigated whether Mirk might mediate survival in another cancer in which Mirk is widely expressed, pancreatic ductal adenocarcinoma.  Mirk was an active kinase in each pancreatic cancer cell line where it was detected.  Mirk knockdown by RNA interference (RNAi) reduced the clonogenicity of Panc1 pancreatic cancer cells 4-fold and decreased tumor cell no., showing that Mirk mediates survival in these cells.  Mirk knockdown by synthetic duplex RNAis in Panc1, AsPc1, and SU86.86 pancreatic cancer cells induced apoptosis and enhanced the apoptosis induced by gemcitibine.  Mirk knockdown did not increase the abundance or activation of Akt.  However, four of five pancreatic carcinoma cell lines exhibited either elevated Mirk activity or elevated Akt activity, suggesting that pancreatic cancer cells primarily rely on Mirk or Akt for survival signaling.  Mirk protein was detected by immunohistochem. in 25 of 28 cases (89%) of pancreatic ductal adenocarcinoma, with elevated expression in 11 cases (39%).  Increased expression of Mirk was seen in pancreatic carcinomas compared with primary cultures of normal ductal epithelium by serial anal. of gene expression and by immunohistochem.  Thus, Mirk is a survival factor for pancreatic ductal adenocarcinoma.  Because knockout of Mirk does not cause embryonic lethality, Mirk is not essential for normal cell growth and may represent a novel therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTvhrZzfM4ebVg90H21EOLACvtfcHk0lgGIIpOOq0G4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsFSlt7w%253D&md5=317c3012cae9140c4c56d7eef07ab6dc</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-05-3089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-05-3089%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DEwton%26aufirst%3DD.%2BZ.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DNaqvi%26aufirst%3DA.%26aulast%3DMercer%26aufirst%3DS.%2BE.%26aulast%3DLandas%26aufirst%3DS.%26aulast%3DFriedman%26aufirst%3DE.%26atitle%3DThe%2520kinase%2520mirk%252Fdyrk1b%2520mediates%2520cell%2520survival%2520in%2520pancreatic%2520ductal%2520adenocarcinoma%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D4149%26epage%3D4158%26doi%3D10.1158%2F0008-5472.can-05-3089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, E. A.</span></span> <span> </span><span class="NLM_article-title">Ovarian cancer cells, not normal cells, are damaged by mirk/dyrk1b kinase inhibition</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">2258</span>– <span class="NLM_lpage">2269</span>, <span class="refDoi"> DOI: 10.1002/ijc.27917</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1002%2Fijc.27917" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=23114871" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs12rsbfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2013&pages=2258-2269&author=J.+Huauthor=H.+Dengauthor=E.+A.+Friedman&title=Ovarian+cancer+cells%2C+not+normal+cells%2C+are+damaged+by+mirk%2Fdyrk1b+kinase+inhibition&doi=10.1002%2Fijc.27917"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Ovarian cancer cells, not normal cells, are damaged by Mirk/Dyrk1B kinase inhibition</span></div><div class="casAuthors">Hu, Jing; Deng, Holly; Friedman, Eileen A.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2258-2269</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Prior studies had shown that the Mirk/dyrk1B gene is amplified/upregulated in about 75% of ovarian cancers, that protein levels of this kinase are elevated in quiescent G0 cells and that Mirk maintains tumor cells in quiescence by initiating rapid degrdn. of cyclin D isoforms and by phosphorylation of a member of the DREAM complex.  Depletion of Mirk/dyrk1B led to increased cyclin D levels, an elevated reactive oxygen species (ROS) content and loss of viability.  However, many normal cells in vivo are quiescent, and therefore, targeting a kinase found in quiescent cells might be problematic.  In our study, Mirk kinase activity was found to be higher in ovarian cancer cells than in normal cells.  Pharmacol. inhibition of Mirk/dyrk1B kinase increased cyclin D levels both in quiescent normal diploid cells and in quiescent CDKN2A-neg. ovarian cancer cells, but led to more active CDK4/cyclin D complexes in quiescent ovarian cancer cells, allowing them to escape G0/G1 quiescence, enter cycle with high ROS levels and undergo apoptosis.  The ROS scavenger N-acetyl cysteine reduced both the amt. of cleaved poly(ADP-ribose) polymerase (PARP) and the extent of cancer cell loss.  In contrast, normal cells were spared because of their expression of cyclin directed kinase (CDK) inhibitors that blocked unregulated cycling.  Quiescent early passage normal ovarian epithelial cells and two strains of quiescent normal diploid fibroblasts remained viable after the inhibition of Mirk/dyrk1B kinase, and the few cells that left G0/G1 quiescence were accumulated in G2+M.  Thus, inhibition of Mirk kinase targeted quiescent ovarian cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWFUPgfQBJiLVg90H21EOLACvtfcHk0lgGIIpOOq0G4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs12rsbfF&md5=e0cf1c67ac4a66f2e12055bb0df0df0b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fijc.27917&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.27917%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DFriedman%26aufirst%3DE.%2BA.%26atitle%3DOvarian%2520cancer%2520cells%252C%2520not%2520normal%2520cells%252C%2520are%2520damaged%2520by%2520mirk%252Fdyrk1b%2520kinase%2520inhibition%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2013%26volume%3D132%26spage%3D2258%26epage%3D2269%26doi%3D10.1002%2Fijc.27917" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Litovchick, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florens, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Washburn, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeCaprio, J. A.</span></span> <span> </span><span class="NLM_article-title">Dyrk1a protein kinase promotes quiescence and senescence through dream complex assembly</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">801</span>– <span class="NLM_lpage">813</span>, <span class="refDoi"> DOI: 10.1101/gad.2034211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1101%2Fgad.2034211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=21498570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltlKntrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=801-813&author=L.+Litovchickauthor=L.+A.+Florensauthor=S.+K.+Swansonauthor=M.+P.+Washburnauthor=J.+A.+DeCaprio&title=Dyrk1a+protein+kinase+promotes+quiescence+and+senescence+through+dream+complex+assembly&doi=10.1101%2Fgad.2034211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">DYRK1A protein kinase promotes quiescence and senescence through DREAM complex assembly</span></div><div class="casAuthors">Litovchick, Larisa; Florens, Laurence A.; Swanson, Selene K.; Washburn, Michael P.; DeCaprio, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">801-813</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">In the absence of growth signals, cells exit the cell cycle and enter into G0 or quiescence.  Alternatively, cells enter senescence in response to inappropriate growth signals such as oncogene expression.  The mol. mechanisms required for cell cycle exit into quiescence or senescence are poorly understood.  The DREAM (DP, RB [retinoblastoma], E2F, and MuvB) complex represses cell cycle-dependent genes during quiescence.  DREAM contains p130, E2F4, DP1, and a stable core complex of five MuvB-like proteins: LIN9, LIN37, LIN52, LIN54, and RBBP4.  In mammalian cells, the MuvB core dissocs. from p130 upon entry into the cell cycle and binds to BMYB during S phase to activate the transcription of genes expressed late in the cell cycle.  We used mass spectroscopic anal. to identify phosphorylation sites that regulate the switch of the MuvB core from BMYB to DREAM.  Here we report that DYRK1A can specifically phosphorylate LIN52 on serine residue 28, and that this phosphorylation is required for DREAM assembly.  Inhibiting DYRK1A activity or point mutation of LIN52 disrupts DREAM assembly and reduces the ability of cells to enter quiescence or undergo Ras-induced senescence.  These data reveal an important role for DYRK1A in the regulation of DREAM activity and entry into quiescence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptXCB-wHS9bLVg90H21EOLACvtfcHk0lgGIIpOOq0G4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltlKntrc%253D&md5=0d3048d4ea6a370a7a230b8f250e3867</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1101%2Fgad.2034211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.2034211%26sid%3Dliteratum%253Aachs%26aulast%3DLitovchick%26aufirst%3DL.%26aulast%3DFlorens%26aufirst%3DL.%2BA.%26aulast%3DSwanson%26aufirst%3DS.%2BK.%26aulast%3DWashburn%26aufirst%3DM.%2BP.%26aulast%3DDeCaprio%26aufirst%3DJ.%2BA.%26atitle%3DDyrk1a%2520protein%2520kinase%2520promotes%2520quiescence%2520and%2520senescence%2520through%2520dream%2520complex%2520assembly%26jtitle%3DGenes%2520Dev.%26date%3D2011%26volume%3D25%26spage%3D801%26epage%3D813%26doi%3D10.1101%2Fgad.2034211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewton, D. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, E.</span></span> <span> </span><span class="NLM_article-title">Mirk regulates the exit of colon cancer cells from quiescence</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">22916</span>– <span class="NLM_lpage">22925</span>, <span class="refDoi"> DOI: 10.1074/jbc.m109.035519</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1074%2Fjbc.M109.035519" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=19542220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpvFKitbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=22916-22925&author=K.+Jinauthor=D.+Z.+Ewtonauthor=S.+Parkauthor=J.+Huauthor=E.+Friedman&title=Mirk+regulates+the+exit+of+colon+cancer+cells+from+quiescence&doi=10.1074%2Fjbc.m109.035519"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Mirk Regulates the Exit of Colon Cancer Cells from Quiescence</span></div><div class="casAuthors">Jin, Kideok; Ewton, Daina Z.; Park, Sunju; Hu, Jing; Friedman, Eileen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">22916-22925</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Mirk/Dyrk1B is a serine/threonine kinase widely expressed in colon cancers.  Serum starvation induced HD6 colon carcinoma cells to enter a quiescent G0 state, characterized by a 2N DNA content and a lower RNA content than G1 cells.  Compared with cycling cells, quiescent cells exhibited 16-fold higher levels of the retinoblastoma protein p130/Rb2, which sequesters E2F4 to block entry into G1, 10-fold elevated levels of the CDK inhibitor p27kip1, and 10-fold higher levels of Mirk.  However, depletion of Mirk did not prevent entry into G0, but enabled quiescent HD6, SW480, and colo320 colon carcinoma cells to acquire some biochem. characteristics of G1 cells, including increased levels of cyclin D1 and cyclin D3 because of slower turnover, increased activity of their CDK4/cyclin D complexes, and increased phosphorylation and decreased E2F4 sequestering ability of the CDK4 target, p130/Rb2.  As a result, depletion of Mirk allowed some cells to escape quiescence and enabled cells released from quiescence to traverse G1 more quickly.  The kinase activity of Mirk was increased by the chemotherapeutic drug 5-fluorouracil (5-FU).  Treatment of p53 mutant colon cancer cells with 5-FU led to an elongated G1 in a Mirk-dependent manner, as G1 was shortened by ectopic overexpression of cyclin D1 mutated at the Mirk phosphorylation site (T288A), but not by wild-type cyclin D1.  Mirk, through regulating cyclin D turnover, and the CDK inhibitor p27, as shown by depletion studies, functioned independently and additively to regulate the exit of tumor cells from quiescence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaIbbM7RpsrbVg90H21EOLACvtfcHk0lh4dY6gk8hnCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpvFKitbg%253D&md5=98ffa71de12dddd61f2d4109e706ce07</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.035519&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.035519%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DK.%26aulast%3DEwton%26aufirst%3DD.%2BZ.%26aulast%3DPark%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DFriedman%26aufirst%3DE.%26atitle%3DMirk%2520regulates%2520the%2520exit%2520of%2520colon%2520cancer%2520cells%2520from%2520quiescence%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D22916%26epage%3D22925%26doi%3D10.1074%2Fjbc.m109.035519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boichuk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makielski, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litovchick, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baron, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zewe, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wozniak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehalek, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korzeniewski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seneviratne, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schöffski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debiec-Rychter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeCaprio, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duensing, A.</span></span> <span> </span><span class="NLM_article-title">The dream complex mediates gist cell quiescence and is a novel therapeutic target to enhance imatinib-induced apoptosis</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">5120</span>– <span class="NLM_lpage">5129</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-13-0579</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1158%2F0008-5472.CAN-13-0579" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=23786773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1OisrfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=5120-5129&author=S.+Boichukauthor=J.+A.+Parryauthor=K.+R.+Makielskiauthor=L.+Litovchickauthor=J.+L.+Baronauthor=J.+P.+Zeweauthor=A.+Wozniakauthor=K.+R.+Mehalekauthor=N.+Korzeniewskiauthor=D.+S.+Seneviratneauthor=P.+Sch%C3%B6ffskiauthor=M.+Debiec-Rychterauthor=J.+A.+DeCaprioauthor=A.+Duensing&title=The+dream+complex+mediates+gist+cell+quiescence+and+is+a+novel+therapeutic+target+to+enhance+imatinib-induced+apoptosis&doi=10.1158%2F0008-5472.can-13-0579"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The DREAM Complex Mediates GIST Cell Quiescence and Is a Novel Therapeutic Target to Enhance Imatinib-Induced Apoptosis</span></div><div class="casAuthors">Boichuk, Sergei; Parry, Joshua A.; Makielski, Kathleen R.; Litovchick, Larisa; Baron, Julianne L.; Zewe, James P.; Wozniak, Agnieszka; Mehalek, Keith R.; Korzeniewski, Nina; Seneviratne, Danushka S.; Schoeffski, Patrick; Debiec-Rychter, Maria; DeCaprio, James A.; Duensing, Anette</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5120-5129</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Gastrointestinal stromal tumors (GIST) can be successfully treated with imatinib mesylate (Gleevec); however, complete remissions are rare and patients frequently achieve disease stabilization in the presence of residual tumor masses.  The clin. observation that discontinuation of treatment can lead to tumor progression suggests that residual tumor cells are, in fact, quiescent and, therefore, able to re-enter the cell-division cycle.  In line with this notion, we have previously shown that imatinib induces GIST cell quiescence in vitro through the APCCDH1-SKP2-p27Klp1 signaling axis.  Here, we provide evidence that imatinib induces GIST cell quiescence in vivo and that this process also involves the DREAM complex, a multisubunit complex that has recently been identified as an addnl. key regulator of quiescence.  Importantly, inhibition of DREAM complex formation by depletion of the DREAM regulatory kinase DYRK1A or its target L1N52 was found to enhance imatinib-induced cell death.  Our results show that imatinib induces apoptosis in a fraction of GIST cells while, at the same time, a subset of cells undergoes quiescence involving the DREAM complex.  Inhibition of this process enhances imatinib-induced apoptosis, which opens the opportunity for future therapeutic interventions to target the DREAM complex for more efficient imatinib responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDU6EXoSaNMrVg90H21EOLACvtfcHk0lh4dY6gk8hnCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1OisrfL&md5=49f454ebe6457a4613dad2f5b289bcb9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-0579&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-0579%26sid%3Dliteratum%253Aachs%26aulast%3DBoichuk%26aufirst%3DS.%26aulast%3DParry%26aufirst%3DJ.%2BA.%26aulast%3DMakielski%26aufirst%3DK.%2BR.%26aulast%3DLitovchick%26aufirst%3DL.%26aulast%3DBaron%26aufirst%3DJ.%2BL.%26aulast%3DZewe%26aufirst%3DJ.%2BP.%26aulast%3DWozniak%26aufirst%3DA.%26aulast%3DMehalek%26aufirst%3DK.%2BR.%26aulast%3DKorzeniewski%26aufirst%3DN.%26aulast%3DSeneviratne%26aufirst%3DD.%2BS.%26aulast%3DSch%25C3%25B6ffski%26aufirst%3DP.%26aulast%3DDebiec-Rychter%26aufirst%3DM.%26aulast%3DDeCaprio%26aufirst%3DJ.%2BA.%26aulast%3DDuensing%26aufirst%3DA.%26atitle%3DThe%2520dream%2520complex%2520mediates%2520gist%2520cell%2520quiescence%2520and%2520is%2520a%2520novel%2520therapeutic%2520target%2520to%2520enhance%2520imatinib-induced%2520apoptosis%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D5120%26epage%3D5129%26doi%3D10.1158%2F0008-5472.can-13-0579" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Becker, W.</span></span> <span> </span><span class="NLM_article-title">A wake-up call to quiescent cancer cells - potential use ofDYRK1B inhibitors in cancer therapy</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">1203</span>– <span class="NLM_lpage">1211</span>, <span class="refDoi"> DOI: 10.1111/febs.14347</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1111%2Ffebs.14347" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=29193696" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFKhurnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2018&pages=1203-1211&author=W.+Becker&title=A+wake-up+call+to+quiescent+cancer+cells+-+potential+use+ofDYRK1B+inhibitors+in+cancer+therapy&doi=10.1111%2Ffebs.14347"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A wake-up call to quiescent cancer cells - potential use of DYRK1B inhibitors in cancer therapy</span></div><div class="casAuthors">Becker, Walter</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1203-1211</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Nondividing cancer cells are relatively resistant to chemotherapeutic drugs and environmental stress factors.  Promoting cell cycle re-entry of quiescent cancer cells is a potential strategy to enhance the cytotoxicity of agents that target cycling cells.  It is therefore important to elucidate the mechanisms by which these cells are maintained in the quiescent state.  The protein kinase dual specificity tyrosine phosphorylation-regulated kinase 1B (DYRK1B) is overexpressed in a subset of cancers and maintains cellular quiescence by counteracting G0/G1-S phase transition.  Specifically, DYRK1B controls the S phase checkpoint by stabilizing the cyclin-dependent kinase (CDK) inhibitor p27Kip1 and inducing the degrdn. of cyclin D.  DYRK1B also stabilizes the DREAM complex that represses cell cycle gene expression in G0 arrested cells.  In addn., DYRK1B enhances cell survival by upregulating antioxidant gene expression and reducing intracellular levels of reactive oxygen species (ROS).  Substantial evidence indicates that depletion or inhibition of DYRK1B drives cell cycle re-entry and enhances apoptosis of those quiescent cancer cells with high expression of DYRK1B.  Furthermore, small mol. DYRK1B inhibitors sensitize cells to the cytotoxic effects of anticancer drugs that target proliferating cells.  These encouraging findings justify continued efforts to investigate the use of DYRK1B inhibitors to disrupt the quiescent state and overturn chemoresistance of noncycling cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPcdYql-muXLVg90H21EOLACvtfcHk0lh4dY6gk8hnCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFKhurnM&md5=67216b30cc75747e7122118bcc7f1f53</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1111%2Ffebs.14347&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.14347%26sid%3Dliteratum%253Aachs%26aulast%3DBecker%26aufirst%3DW.%26atitle%3DA%2520wake-up%2520call%2520to%2520quiescent%2520cancer%2520cells%2520-%2520potential%2520use%2520ofDYRK1B%2520inhibitors%2520in%2520cancer%2520therapy%26jtitle%3DFEBS%2520J.%26date%3D2018%26volume%3D285%26spage%3D1203%26epage%3D1211%26doi%3D10.1111%2Ffebs.14347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewton, D. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercer, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, E.</span></span> <span> </span><span class="NLM_article-title">Mirk/dyrk1b kinase destabilizes cyclin d1 by phosphorylation at threonine 288</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">27790</span>– <span class="NLM_lpage">27798</span>, <span class="refDoi"> DOI: 10.1074/jbc.m403042200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1074%2Fjbc.M403042200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=15075324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2cXkvFGnu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=27790-27798&author=Y.+Zouauthor=D.+Z.+Ewtonauthor=X.+Dengauthor=S.+E.+Mercerauthor=E.+Friedman&title=Mirk%2Fdyrk1b+kinase+destabilizes+cyclin+d1+by+phosphorylation+at+threonine+288&doi=10.1074%2Fjbc.m403042200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Mirk/dyrk1B Kinase Destabilizes Cyclin D1 by Phosphorylation at Threonine 288</span></div><div class="casAuthors">Zou, Yonglong; Ewton, Daina Z.; Deng, Xiaobing; Mercer, Stephen E.; Friedman, Eileen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">27790-27798</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The phosphorylation of cyclin D1 at threonine 286 by glycogen synthase kinase 3β (GSK3β) has been shown to be required for the ubiquitination and nuclear export of cyclin D1 and its subsequent degrdn. in the proteasome.  The mutation of the nearby threonine 288 residue to nonphosphorylatable alanine has also been shown to reduce the ubiquitination of cyclin D1, suggesting that phosphorylation at threonine 288 may also lead to degrdn. of cyclin D1.  We now demonstrate that the G0/G1-active arginine-directed protein kinase Mirk/dyrk1B binds to cyclin D1 and phosphorylates cyclin D1 at threonine 288 in vivo and that the cyclin D1-T288A construct is more stable than wild-type cyclin D1.  Transient overexpression of Mirk in nontransformed Mv1Lu lung epithelial cells blocked cells in G0/G1.  Depletion of endogenous Mirk by RNA interference increased cyclin D1 protein levels but not mRNA levels, indicating that Mirk destabilizes cyclin D1 protein.  Destabilization was confirmed by induction of a stable Mirk transfectant of Mv1Lu cells, which blocked cell migration (Zou, Y., Lim, S., Lee, K., Deng, X., and Friedman, E. (2003) J. Biol. Chem. 278, 49573-49581), and caused a decrease in the half-life of endogenous cyclin D1, concomitant with an increase in Mirk expression.  In vitro cyclin D1 was phosphorylated in an additive fashion by Mirk and GSK3β.  Mirk-phosphorylated cyclin D1 mutated at the GSK3β phosphorylation site and was capable of phosphorylating cyclin D1 in the presence of the GSK3β inhibitor LiCl.  Mirk may function together with GSK3β to assist cell arrest in G0/G1 by destabilizing cyclin D1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppbNHWpluCW7Vg90H21EOLACvtfcHk0ljNwJMMKGzQgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXkvFGnu7Y%253D&md5=36ad27a9054ad5d2fabc4c5d7c52926f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M403042200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M403042200%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DY.%26aulast%3DEwton%26aufirst%3DD.%2BZ.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DMercer%26aufirst%3DS.%2BE.%26aulast%3DFriedman%26aufirst%3DE.%26atitle%3DMirk%252Fdyrk1b%2520kinase%2520destabilizes%2520cyclin%2520d1%2520by%2520phosphorylation%2520at%2520threonine%2520288%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D27790%26epage%3D27798%26doi%3D10.1074%2Fjbc.m403042200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ashford, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oxley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kettle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guichard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lochhead, P. A.</span></span> <span> </span><span class="NLM_article-title">A novel DYRK1B inhibitor AZ191 demonstrates that DYRK1B acts independently of GSK3β to phosphorylate cyclin D1 at Thr286, not Thr288</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>457</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1042/bj20130461</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1042%2FBJ20130461" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=24134204" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFSltbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=457&publication_year=2014&pages=43-56&author=A.+L.+Ashfordauthor=D.+Oxleyauthor=J.+Kettleauthor=K.+Hudsonauthor=S.+Guichardauthor=S.+J.+Cookauthor=P.+A.+Lochhead&title=A+novel+DYRK1B+inhibitor+AZ191+demonstrates+that+DYRK1B+acts+independently+of+GSK3%CE%B2+to+phosphorylate+cyclin+D1+at+Thr286%2C+not+Thr288&doi=10.1042%2Fbj20130461"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A novel DYRK1B inhibitor AZ191 demonstrates that DYRK1B acts independently of GSK3β to phosphorylate cyclin D1 at Thr286, not Thr288</span></div><div class="casAuthors">Ashford, Anne L.; Oxley, David; Kettle, Jason; Hudson, Kevin; Guichard, Sylvie; Cook, Simon J.; Lochhead, Pamela A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">457</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-56</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">DYRK1B (dual-specificity tyrosine phosphorylation-regulated kinase 1B) is amplified in certain cancers and may be an oncogene; however, our knowledge of DYRK1B has been limited by the lack of selective inhibitors.  In the present study we describe AZ191, a potent small mol. inhibitor that selectively inhibits DYRK1B in vitro and in cells.  CCND1 (cyclin D1), a key regulator of the mammalian G1-S-phase transition, is phosphorylated on Thr286 by GSK3β (glycogen synthase kinase 3β) to promote its degrdn.  DYRK1B has also been proposed to promote CCND1 turnover, but was reported to phosphorylate Thr288 rather than Thr286.  Using in vitro kinase assays, phospho-specific immunoblot anal. and MS in conjunction with AZ191 we now show that DYRK1B phosphorylates CCND1 at Thr286, not Thr288, in vitro and in cells.  In HEK (human embryonic kidney)-293 and PANC-1 cells (which exhibit DYRK1B amplification) DYRK1B drives Thr286 phosphorylation and proteasome-dependent turnover of CCND1 and this is abolished by AZ191 or DYRK1B RNAi, but not by GSK3β inhibitors or GSK3β RNAi.  DYRK1B expression causes a G1-phase cell-cycle arrest, but overexpression of CCND1 (wild-type or T286A) fails to overcome this; indeed, DYRK1B also promotes the expression of p21CIP1 (21 kDa CDK-interacting protein 1) and p27KIP1 (CDK-inhibitory protein 1).  The results of the present study demonstrate for the first time that DYRK1B is a novel Thr286-CCND1 kinase that acts independently of GSK3β to promote CCND1 degrdn.  Furthermore, we anticipate that AZ191 may prove useful in defining further substrates and biol. functions of DYRK1B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryxVa5ahY3ArVg90H21EOLACvtfcHk0ljNwJMMKGzQgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFSltbbF&md5=1951b0ef61c2fdaed0a15de224dec3dc</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1042%2FBJ20130461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20130461%26sid%3Dliteratum%253Aachs%26aulast%3DAshford%26aufirst%3DA.%2BL.%26aulast%3DOxley%26aufirst%3DD.%26aulast%3DKettle%26aufirst%3DJ.%26aulast%3DHudson%26aufirst%3DK.%26aulast%3DGuichard%26aufirst%3DS.%26aulast%3DCook%26aufirst%3DS.%2BJ.%26aulast%3DLochhead%26aufirst%3DP.%2BA.%26atitle%3DA%2520novel%2520DYRK1B%2520inhibitor%2520AZ191%2520demonstrates%2520that%2520DYRK1B%2520acts%2520independently%2520of%2520GSK3%25CE%25B2%2520to%2520phosphorylate%2520cyclin%2520D1%2520at%2520Thr286%252C%2520not%2520Thr288%26jtitle%3DBiochem.%2520J.%26date%3D2014%26volume%3D457%26spage%3D43%26epage%3D56%26doi%3D10.1042%2Fbj20130461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercer, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewton, D. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, E.</span></span> <span> </span><span class="NLM_article-title">The Cyclin-dependent Kinase Inhibitor p27Kip1 Is Stabilized in G0 by Mirk/dyrk1B Kinase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">22498</span>– <span class="NLM_lpage">22504</span>, <span class="refDoi"> DOI: 10.1074/jbc.m400479200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1074%2Fjbc.M400479200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=15010468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjvF2rtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=22498-22504&author=X.+Dengauthor=S.+E.+Mercerauthor=S.+Shahauthor=D.+Z.+Ewtonauthor=E.+Friedman&title=The+Cyclin-dependent+Kinase+Inhibitor+p27Kip1+Is+Stabilized+in+G0+by+Mirk%2Fdyrk1B+Kinase&doi=10.1074%2Fjbc.m400479200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The Cyclin-dependent Kinase Inhibitor p27Kip1 Is Stabilized in G0 by Mirk/dyrk1B Kinase</span></div><div class="casAuthors">Deng, Xiaobing; Mercer, Stephen E.; Shah, Sejal; Ewton, Daina Z.; Friedman, Eileen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">22498-22504</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Elevated levels of the cyclin-dependent kinase (CDK) inhibitor p27 block the cell in G0/G1 until mitogenic signals activate G1 cyclins and initiate proliferation.  Post-translational regulation of p27 by different phosphorylation events is crit. in allowing cells to proceed through the cell cycle.  We now demonstrate that the arginine-directed kinase, Mirk/dyrk1B, is maximally active in G0 in NIH3T3 cells, when it stabilizes p27 by phosphorylating it at Ser-10.  The phospho-mimetic mutant p27-S10D was more stable, and the non-phosphorylatable mutant p27-S10A was less stable than wild-type when expressed in G0-arrested cells.  Following phosphorylation by Mirk, p27 remains a functional CDK inhibitor, capable of binding to CDK2.  Mirk did not induce the translocation of p27 from the nucleus in G0, but instead co-localized with nuclear p27.  Depletion of Mirk by RNA interference decreased the phosphorylation of p27 at Ser-10 and the stability of endogenous p27.  RNAi to Mirk increased cell entry from G0 into G1 as shown by increased expression of proliferating cell nuclear antigen and decreased expression of p27.  These data suggest a model in which Mirk increases the amt. of nuclear p27 by stabilizing it during G0 when Mirk is most abundant.  Mitogen stimulation then causes cells to enter G1, reduces Mirk levels (Deng, X., Ewton, D., Pawlikowski, B., Maimone, M., and Friedman, E. (2003) J. Biol. Chem. 278, 41347-41354), and initiates the translocation of p27 to the cytoplasm.  In addn., depletion of Mirk by RNAi in postmitotic C2C12 myoblasts decreased protein but not mRNA levels of p27, suggesting that stabilization of p27 by Mirk also occurs during differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpx5aOLqkCRELVg90H21EOLACvtfcHk0ljNwJMMKGzQgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjvF2rtL0%253D&md5=4847b1a2365a17e44de4b99e10ec02ec</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M400479200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M400479200%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DMercer%26aufirst%3DS.%2BE.%26aulast%3DShah%26aufirst%3DS.%26aulast%3DEwton%26aufirst%3DD.%2BZ.%26aulast%3DFriedman%26aufirst%3DE.%26atitle%3DThe%2520Cyclin-dependent%2520Kinase%2520Inhibitor%2520p27Kip1%2520Is%2520Stabilized%2520in%2520G0%2520by%2520Mirk%252Fdyrk1B%2520Kinase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D22498%26epage%3D22504%26doi%3D10.1074%2Fjbc.m400479200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janumyan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sansam, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, E.</span></span> <span> </span><span class="NLM_article-title">G0 function of bcl2 and bcl-xl requires bax, bak, and p27 phosphorylation by mirk, revealing a novel role of bax and bak in quiescence regulation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">34108</span>– <span class="NLM_lpage">34120</span>, <span class="refDoi"> DOI: 10.1074/jbc.m806294200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1074%2Fjbc.M806294200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=18818203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVelu73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=34108-34120&author=Y.+Janumyanauthor=Q.+Cuiauthor=L.+Yanauthor=C.+G.+Sansamauthor=M.+Valentinauthor=E.+Yang&title=G0+function+of+bcl2+and+bcl-xl+requires+bax%2C+bak%2C+and+p27+phosphorylation+by+mirk%2C+revealing+a+novel+role+of+bax+and+bak+in+quiescence+regulation&doi=10.1074%2Fjbc.m806294200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">G0 Function of BCL2 and BCL-xL Requires BAX, BAK, and p27 Phosphorylation by Mirk, Revealing a Novel Role of BAX and BAK in Quiescence Regulation</span></div><div class="casAuthors">Janumyan, Yelena; Cui, Qinghua; Yan, Ling; Sansam, Courtney G.; Valentin, Mayda; Yang, Elizabeth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">34108-34120</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">BCL2 and BCL-xL facilitate G0 quiescence by decreasing RNA content and cell size and up-regulating p27 protein, but the precise mechanism is not understood.  We investigated the relationship between cell cycle regulation and the anti-apoptosis function of BCL2 and BCL-xL.  Neither caspase inhibition nor abrogation of mitochondria-dependent apoptosis by BAX and BAK deletion fully recapitulated the G0 effects of BCL2 or BCL-xL, suggesting that mechanisms in addn. to anti-apoptosis are involved in the cell cycle arrest function of BCL2 or BCL-xL.  We found that BCL2 and BCL-xL expression in bax-/- bak-/- cells did not confer cell cycle effects, consistent with the G0 function of BCL2 and BCL-xL being mediated through BAX or BAK.  Stabilization of p27 in G0 in BCL2 or BCL-xL cells was due to phosphorylation of p27 at Ser10 by the kinase Mirk.  In bax-/- bak-/- cells, total p27 and p27 phosphorylated at Ser10 were elevated.  Re-expression of BAX in bax-/- bak-/- cells and silencing of BAX and BAK in wild type cells confirmed that endogenous BAX and BAK modulated p27.  These data revealed a novel role for BAX and BAK in the regulation of G0 quiescence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAPrDVuZaTJ7Vg90H21EOLACvtfcHk0ljNwJMMKGzQgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVelu73O&md5=544796b9d792884a8df39132c01a39cc</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M806294200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M806294200%26sid%3Dliteratum%253Aachs%26aulast%3DJanumyan%26aufirst%3DY.%26aulast%3DCui%26aufirst%3DQ.%26aulast%3DYan%26aufirst%3DL.%26aulast%3DSansam%26aufirst%3DC.%2BG.%26aulast%3DValentin%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DE.%26atitle%3DG0%2520function%2520of%2520bcl2%2520and%2520bcl-xl%2520requires%2520bax%252C%2520bak%252C%2520and%2520p27%2520phosphorylation%2520by%2520mirk%252C%2520revealing%2520a%2520novel%2520role%2520of%2520bax%2520and%2520bak%2520in%2520quiescence%2520regulation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D34108%26epage%3D34120%26doi%3D10.1074%2Fjbc.m806294200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pozo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahonero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liñares, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayuso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagiwara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricoy, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernández-Laín, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sepúlveda, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Gómez, P.</span></span> <span> </span><span class="NLM_article-title">Inhibition of dyrk1a destabilizes egfr and reduces egfr-dependent glioblastoma growth</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">2475</span>– <span class="NLM_lpage">2487</span>, <span class="refDoi"> DOI: 10.1172/jci63623</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1172%2FJCI63623" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=23635774" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpsFWktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2013&pages=2475-2487&author=N.+Pozoauthor=C.+Zahoneroauthor=P.+Fernandezauthor=J.+M.+Li%C3%B1aresauthor=A.+Ayusoauthor=M.+Hagiwaraauthor=A.+P%C3%A9rezauthor=J.+R.+Ricoyauthor=A.+Hern%C3%A1ndez-La%C3%ADnauthor=J.+M.+Sep%C3%BAlvedaauthor=P.+S%C3%A1nchez-G%C3%B3mez&title=Inhibition+of+dyrk1a+destabilizes+egfr+and+reduces+egfr-dependent+glioblastoma+growth&doi=10.1172%2Fjci63623"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of DYRK1A destabilizes EGFR and reduces EGFR-dependent glioblastoma growth</span></div><div class="casAuthors">Pozo, Natividad; Zahonero, Cristina; Fernandez, Paloma; Linares, Jose M.; Ayuso, Angel; Hagiwara, Masatoshi; Perez, Angel; Ricoy, Jose R.; Hernandez-Lain, Aurelio; Sepulveda, Juan M.; Sanchez-Gomez, Pilar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2475-2487</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">Glioblastomas (GBMs) are very aggressive tumors that are resistant to conventional chemo- and radiotherapy.  New mol. therapeutic strategies are required to effectively eliminate the subpopulation of GBM tumor-initiating cells that are responsible for relapse.  Since EGFR is altered in 50% of GBMs, it represents one of the most promising targets; however, EGFR kinase inhibitors have produced poor results in clin. assays, with no clear explanation for the obsd. resistance.  We uncovered a fundamental role for the dual-specificity tyrosine phosphorylation-regulated kinase, DYRK1A, in regulating EGFR in GBMs.  We found that DYRK1A was highly expressed in these tumors and that its expression was correlated with that of EGFR.  Moreover, DYRK1A inhibition promoted EGFR degrdn. in primary GBM cell lines and neural progenitor cells, sharply reducing the self-renewal capacity of normal and tumorigenic cells.  Most importantly, our data suggest that a subset of GBMs depends on high surface EGFR levels, as DYRK1A inhibition compromised their survival and produced a profound decrease in tumor burden.  We propose that the recovery of EGFR stability is a key oncogenic event in a large proportion of gliomas and that pharmacol. inhibition of DYRK1A could represent a promising therapeutic intervention for EGFR-dependent GBMs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFv6XKpg04mLVg90H21EOLACvtfcHk0ljvU0FK46g_SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpsFWktr8%253D&md5=41c1ef0531890b30dc6357a85fd6655c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1172%2FJCI63623&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI63623%26sid%3Dliteratum%253Aachs%26aulast%3DPozo%26aufirst%3DN.%26aulast%3DZahonero%26aufirst%3DC.%26aulast%3DFernandez%26aufirst%3DP.%26aulast%3DLi%25C3%25B1ares%26aufirst%3DJ.%2BM.%26aulast%3DAyuso%26aufirst%3DA.%26aulast%3DHagiwara%26aufirst%3DM.%26aulast%3DP%25C3%25A9rez%26aufirst%3DA.%26aulast%3DRicoy%26aufirst%3DJ.%2BR.%26aulast%3DHern%25C3%25A1ndez-La%25C3%25ADn%26aufirst%3DA.%26aulast%3DSep%25C3%25BAlveda%26aufirst%3DJ.%2BM.%26aulast%3DS%25C3%25A1nchez-G%25C3%25B3mez%26aufirst%3DP.%26atitle%3DInhibition%2520of%2520dyrk1a%2520destabilizes%2520egfr%2520and%2520reduces%2520egfr-dependent%2520glioblastoma%2520growth%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2013%26volume%3D123%26spage%3D2475%26epage%3D2487%26doi%3D10.1172%2Fjci63623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maye, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogerman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tejedor, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toftgard, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span> <span> </span><span class="NLM_article-title">Regulation of gli1 transcriptional activity in the nucleus by dyrk1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">35156</span>– <span class="NLM_lpage">35161</span>, <span class="refDoi"> DOI: 10.1074/jbc.m206743200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1074%2Fjbc.M206743200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=12138125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD38Xnt1Wgu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=35156-35161&author=J.+Maoauthor=P.+Mayeauthor=P.+Kogermanauthor=F.+J.+Tejedorauthor=R.+Toftgardauthor=W.+Xieauthor=G.+Wuauthor=D.+Wu&title=Regulation+of+gli1+transcriptional+activity+in+the+nucleus+by+dyrk1&doi=10.1074%2Fjbc.m206743200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of Gli1 Transcriptional Activity in the Nucleus by Dyrk1</span></div><div class="casAuthors">Mao, Junhao; Maye, Peter; Kogerman, Priit; Tejedor, Francisco J.; Toftgard, Rune; Xie, Wei; Wu, Guanqing; Wu, Dianqing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">35156-35161</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">To investigate the cellular role of dual specificity Yak1-related kinase (Dyrk) 1, a nuclear localized dual specificity protein kinase, we examd. its effect on transcriptional regulation using reporter gene assays.  We found that Dyrk1 can substantially enhance Gli1-dependent, but not LEF-1-, c-Jun-, or Elk-dependent, gene transcription.  In part, Dyrk1 does this through retaining Gli1 in the nucleus.  However, we also demonstrate that Dyrk1 can enhance the transcriptional activity of Gli1-AHA, a nuclear export mutant, suggesting that Dyrk1 may be more directly involved in regulating the transcriptional activity of Gli1.  In addn., Dyrk1 acted synergistically with Sonic hedgehog (Shh) to induce gene transcription and differentiation in mouse C3H10T1/2 cells.  The failure of Shh to stimulate Dyrk1 kinase activity suggests that Dyrk1 may not be directly regulated by the Shh signaling pathway but functionally interacts with it.  Thus, Gli1 transcriptional activity may be subjected to further regulation in the cell nucleus by a pathway distinct from Shh signaling, one mediated by Dyrk1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQU-enyGCh_bVg90H21EOLACvtfcHk0ljvU0FK46g_SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xnt1Wgu7c%253D&md5=2a2702c9b9f3acc13204176735ffc62f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M206743200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M206743200%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DJ.%26aulast%3DMaye%26aufirst%3DP.%26aulast%3DKogerman%26aufirst%3DP.%26aulast%3DTejedor%26aufirst%3DF.%2BJ.%26aulast%3DToftgard%26aufirst%3DR.%26aulast%3DXie%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DD.%26atitle%3DRegulation%2520of%2520gli1%2520transcriptional%2520activity%2520in%2520the%2520nucleus%2520by%2520dyrk1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D35156%26epage%3D35161%26doi%3D10.1074%2Fjbc.m206743200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gruber, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutzinger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmer, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parigger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sternberg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cegielkowski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaja, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leban, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamm, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aberger, F.</span></span> <span> </span><span class="NLM_article-title">Dyrk1b as therapeutic target in hedgehog/gli-dependent cancer cells with smoothened inhibitor resistance</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">7134</span>– <span class="NLM_lpage">7148</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.6910</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.18632%2Foncotarget.6910" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=26784250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A280%3ADC%252BC28nhslSksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=7134-7148&author=W.+Gruberauthor=M.+Hutzingerauthor=D.+P.+Elmerauthor=T.+Pariggerauthor=C.+Sternbergauthor=L.+Cegielkowskiauthor=M.+Zajaauthor=J.+Lebanauthor=S.+Michelauthor=S.+Hammauthor=D.+Vittauthor=F.+Aberger&title=Dyrk1b+as+therapeutic+target+in+hedgehog%2Fgli-dependent+cancer+cells+with+smoothened+inhibitor+resistance&doi=10.18632%2Foncotarget.6910"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">DYRK1B as therapeutic target in Hedgehog/GLI-dependent cancer cells with Smoothened inhibitor resistance</span></div><div class="casAuthors">Gruber Wolfgang; Hutzinger Martin; Elmer Dominik Patrick; Parigger Thomas; Sternberg Christina; Cegielkowski Lukasz; Aberger Fritz; Zaja Mirko; Michel Susanne; Hamm Svetlana; Vitt Daniel; Leban Johann; Vitt Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">7134-48</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A wide range of human malignancies displays aberrant activation of Hedgehog (HH)/GLI signaling, including cancers of the skin, brain, gastrointestinal tract and hematopoietic system.  Targeting oncogenic HH/GLI signaling with small molecule inhibitors of the essential pathway effector Smoothened (SMO) has shown remarkable therapeutic effects in patients with advanced and metastatic basal cell carcinoma.  However, acquired and de novo resistance to SMO inhibitors poses severe limitations to the use of SMO antagonists and urgently calls for the identification of novel targets and compounds.Here we report on the identification of the Dual-Specificity-Tyrosine-Phosphorylation-Regulated Kinase 1B (DYRK1B) as critical positive regulator of HH/GLI signaling downstream of SMO.  Genetic and chemical inhibition of DYRK1B in human and mouse cancer cells resulted in marked repression of HH signaling and GLI1 expression, respectively.  Importantly, DYRK1B inhibition profoundly impaired GLI1 expression in both SMO-inhibitor sensitive and resistant settings.  We further introduce a novel small molecule DYRK1B inhibitor, DYRKi, with suitable pharmacologic properties to impair SMO-dependent and SMO-independent oncogenic GLI activity.  The results support the use of DYRK1B antagonists for the treatment of HH/GLI-associated cancers where SMO inhibitors fail to demonstrate therapeutic efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSFmF3n8no4EZw7prLlxLOzfW6udTcc2eaChLBWnjAyy7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28nhslSksw%253D%253D&md5=166f2cf11d5cbc8b78bfd91cd5a28b1b</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.6910&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.6910%26sid%3Dliteratum%253Aachs%26aulast%3DGruber%26aufirst%3DW.%26aulast%3DHutzinger%26aufirst%3DM.%26aulast%3DElmer%26aufirst%3DD.%2BP.%26aulast%3DParigger%26aufirst%3DT.%26aulast%3DSternberg%26aufirst%3DC.%26aulast%3DCegielkowski%26aufirst%3DL.%26aulast%3DZaja%26aufirst%3DM.%26aulast%3DLeban%26aufirst%3DJ.%26aulast%3DMichel%26aufirst%3DS.%26aulast%3DHamm%26aufirst%3DS.%26aulast%3DVitt%26aufirst%3DD.%26aulast%3DAberger%26aufirst%3DF.%26atitle%3DDyrk1b%2520as%2520therapeutic%2520target%2520in%2520hedgehog%252Fgli-dependent%2520cancer%2520cells%2520with%2520smoothened%2520inhibitor%2520resistance%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D7134%26epage%3D7148%26doi%3D10.18632%2Foncotarget.6910" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, N.</span></span> <span> </span><span class="NLM_article-title">Dyrk1a suppression restrains mcl-1 expression and sensitizes nsclc cells to bcl-2 inhibitors</span>. <i>Cancer Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">387</span>– <span class="NLM_lpage">400</span>, <span class="refDoi"> DOI: 10.20892/j.issn.2095-3941.2019.0380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.20892%2Fj.issn.2095-3941.2019.0380" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=32587776" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitlalsb7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2020&pages=387-400&author=Y.+Liauthor=D.+Zhouauthor=S.+Xuauthor=M.+Raoauthor=Z.+Zhangauthor=L.+Wuauthor=C.+Zhangauthor=N.+Lin&title=Dyrk1a+suppression+restrains+mcl-1+expression+and+sensitizes+nsclc+cells+to+bcl-2+inhibitors&doi=10.20892%2Fj.issn.2095-3941.2019.0380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">DYRK1A suppression restrains Mcl-1 expression and sensitizes NSCLC cells to Bcl-2 inhibitors</span></div><div class="casAuthors">Li, Yangling; Zhou, Dongmei; Xu, Shuang; Rao, Mingjun; Zhang, Zuoyan; Wu, Linwen; Zhang, Chong; Lin, Nengming</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Biology & Medicine</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">387-400</span>CODEN:
                <span class="NLM_cas:coden">CBMADQ</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Tianjin Medical University Cancer Institute and Hospital</span>)
        </div><div class="casAbstract">Objective: Mcl-1 overexpression confers acquired resistance to Bcl-2 inhibitors in non-small cell lung cancer (NSCLC), but no direct Mcl-1 inhibitor is currently available for clin. use.  Thus, novel therapeutic strategies are urgently needed to target Mcl-1 and sensitize the anti-NSCLC activity of Bcl-2 inhibitors.  Methods: Cell proliferation was measured using sulforhodamine B and colony formation assays, and apoptosis was detected with Annexin V-FITC staining.  Gene expression was manipulated using siRNAs and plasmids.  Real-time PCR and Western blot were used to measure mRNA and protein levels.  Immunopptn. and immunofluorescence were used to analyze co-localization of dual specificity tyrosine-phosphorylation-regulated kinase 1A (DYRK1A) and Mcl-1.  Results: Suppression of DYRK1A resulted in reduced Mcl-1 expression in NSCLC cells, whereas overexpression of DYRK1A significantly increased Mcl-1 expression.  Suppression of DYRK1A did not alter Mcl-1 mRNA levels, but did result in an accelerated degrdn. of Mcl-1 protein in NSCLC cells.  Furthermore, DYRK1A mediated proteasome-dependent degrdn. of Mcl-1 in NSCLC cells, and DYRK1A co-localized with Mcl-1 in NSCLC cells and was co-expressed with Mcl-1 in tumor samples from lung cancer patients, suggesting that Mcl-1 may be a novel DYRK1A substrate.  We showed that combined therapy with harmine and Bcl-2 antagonists significantly inhibited cell proliferation and induced apoptosis in NSCLC cell lines as well as primary NSCLC cells.  Conclusions: Mcl-1 is a novel DYRK1A substrate, and the role of DYRK1A in promoting Mcl-1 stability makes it an attractive target for decreasing Bcl-2 inhibitor resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-Vu4TyDqLd7Vg90H21EOLACvtfcHk0lhVX1OXmgUGxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitlalsb7O&md5=1958de0db80c59bb12e2bda8281b98e7</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.20892%2Fj.issn.2095-3941.2019.0380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.20892%252Fj.issn.2095-3941.2019.0380%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DRao%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DN.%26atitle%3DDyrk1a%2520suppression%2520restrains%2520mcl-1%2520expression%2520and%2520sensitizes%2520nsclc%2520cells%2520to%2520bcl-2%2520inhibitors%26jtitle%3DCancer%2520Biol.%2520Med.%26date%3D2020%26volume%3D17%26spage%3D387%26epage%3D400%26doi%3D10.20892%2Fj.issn.2095-3941.2019.0380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, K.</span></span> <span> </span><span class="NLM_article-title">Multiple functions of dyrk2 in cancer and tissue development</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>593</i></span>,  <span class="NLM_fpage">2953</span>– <span class="NLM_lpage">2965</span>, <span class="refDoi"> DOI: 10.1002/1873-3468.13601</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1002%2F1873-3468.13601" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=31505048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslynsrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=593&publication_year=2019&pages=2953-2965&author=S.+Yoshidaauthor=K.+Yoshida&title=Multiple+functions+of+dyrk2+in+cancer+and+tissue+development&doi=10.1002%2F1873-3468.13601"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple functions of DYRK2 in cancer and tissue development</span></div><div class="casAuthors">Yoshida, Saishu; Yoshida, Kiyotsugu</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">593</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2953-2965</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Dual-specificity tyrosine-regulated kinases (DYRKs) are evolutionarily conserved from yeast to mammals.  Accumulating studies have revealed that DYRKs have important roles in regulation of the cell cycle and survival.  DYRK2, a member of the class II DYRK family protein, is a key regulator of p53, and phosphorylates it at Ser46 to induce apoptosis in response to DNA damage.  Moreover, recent studies have uncovered that DYRK2 regulates G1/S transition, epithelial-mesenchymal-transition, and stemness in human cancer cells.  DYRK2 also appears to have roles in tissue development in lower eukaryotes.  Thus, the elucidation of mechanisms for DYRK2 during mammalian tissue development will promote the understanding of cell differentiation, tissue homeostasis, and congenital diseases as well as cancer.  In this review, we discuss the roles of DYRK2 in tumor cells.  Moreover, we focus on DYRK2-dependent developmental mechanisms in several species including fly (Drosophila), worm (Caenorhabditis elegans), zebrafish (Danio rerio), and mammals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrypGm-jUS2DLVg90H21EOLACvtfcHk0lhVX1OXmgUGxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslynsrbJ&md5=1d8ccb464ded9d099472ad6da211f216</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2F1873-3468.13601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1873-3468.13601%26sid%3Dliteratum%253Aachs%26aulast%3DYoshida%26aufirst%3DS.%26aulast%3DYoshida%26aufirst%3DK.%26atitle%3DMultiple%2520functions%2520of%2520dyrk2%2520in%2520cancer%2520and%2520tissue%2520development%26jtitle%3DFEBS%2520Lett.%26date%3D2019%26volume%3D593%26spage%3D2953%26epage%3D2965%26doi%3D10.1002%2F1873-3468.13601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Correa-Saez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimenez-Izquierdo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrido-Rodriguez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrugares, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munoz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calzado, M. A.</span></span> <span> </span><span class="NLM_article-title">Updating dual-specificity tyrosine-phosphorylation-regulated kinase 2 (dyrk2): Molecular basis, functions and role in diseases</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">4747</span>– <span class="NLM_lpage">4763</span>, <span class="refDoi"> DOI: 10.1007/s00018-020-03556-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1007%2Fs00018-020-03556-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=32462403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVantb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2020&pages=4747-4763&author=A.+Correa-Saezauthor=R.+Jimenez-Izquierdoauthor=M.+Garrido-Rodriguezauthor=R.+Morrugaresauthor=E.+Munozauthor=M.+A.+Calzado&title=Updating+dual-specificity+tyrosine-phosphorylation-regulated+kinase+2+%28dyrk2%29%3A+Molecular+basis%2C+functions+and+role+in+diseases&doi=10.1007%2Fs00018-020-03556-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Updating dual-specificity tyrosine-phosphorylation-regulated kinase 2 (DYRK2): molecular basis, functions and role in diseases</span></div><div class="casAuthors">Correa-Saez, Alejandro; Jimenez-Izquierdo, Rafael; Garrido-Rodriguez, Martin; Morrugares, Rosario; Munoz, Eduardo; Calzado, Marco A.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">4747-4763</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">A review.  Abstr.: Members of the dual-specificity tyrosine-regulated kinase (DYRKs) subfamily possess a distinctive capacity to phosphorylate tyrosine, serine, and threonine residues.  Among the DYRK class II members, DYRK2 is considered a unique protein due to its role in disease.  According to the post-transcriptional and post-translational modifications, DYRK2 expression greatly differs among human tissues.  Regarding its mechanism of action, this kinase performs direct phosphorylation on its substrates or acts as a priming kinase, enabling subsequent substrate phosphorylation by GSK3β.  Moreover, DYRK2 acts as a scaffold for the EDVP E3 ligase complex during the G2/M phase of cell cycle.  DYRK2 functions such as cell survival, cell development, cell differentiation, proteasome regulation, and microtubules were studied in complete detail in this review.  We have also gathered available information from different bioinformatic resources to show DYRK2 interactome, normal and tumoral tissue expression, and recurrent cancer mutations.  Then, here we present an innovative approach to clarify DYRK2 functionality and importance.  DYRK2 roles in diseases have been studied in detail, highlighting this kinase as a key protein in cancer development.  First, DYRK2 regulation of c-Jun, c-Myc, Rpt3, TERT, and katanin p60 reveals the implication of this kinase in cell-cycle-mediated cancer development.  Addnl., depletion of this kinase correlated with reduced apoptosis, with consequences on cancer patient response to chemotherapy.  Other functions like cancer stem cell formation and epithelial-mesenchymal transition regulation are also controlled by DYRK2.  Furthermore, the pharmacol. modulation of this protein by different inhibitors (harmine, curcumine, LDN192960, and ID-8) has enabled to clarify DYRK2 functionality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-K55FlWvWvbVg90H21EOLACvtfcHk0liP9VI7No6lGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVantb3K&md5=d0f06b890a649b09cb386e74140c2b10</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2Fs00018-020-03556-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-020-03556-1%26sid%3Dliteratum%253Aachs%26aulast%3DCorrea-Saez%26aufirst%3DA.%26aulast%3DJimenez-Izquierdo%26aufirst%3DR.%26aulast%3DGarrido-Rodriguez%26aufirst%3DM.%26aulast%3DMorrugares%26aufirst%3DR.%26aulast%3DMunoz%26aufirst%3DE.%26aulast%3DCalzado%26aufirst%3DM.%2BA.%26atitle%3DUpdating%2520dual-specificity%2520tyrosine-phosphorylation-regulated%2520kinase%25202%2520%2528dyrk2%2529%253A%2520Molecular%2520basis%252C%2520functions%2520and%2520role%2520in%2520diseases%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2020%26volume%3D77%26spage%3D4747%26epage%3D4763%26doi%3D10.1007%2Fs00018-020-03556-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stringer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodlett, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roper, R. J.</span></span> <span> </span><span class="NLM_article-title">Targeting trisomic treatments: Optimizing dyrk1a inhibition to improve down syndrome deficits</span>. <i>Mol. Genet. Genomic Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">451</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.1002/mgg3.334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1002%2Fmgg3.334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=28944229" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOntL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2017&pages=451-465&author=M.+Stringerauthor=C.+R.+Goodlettauthor=R.+J.+Roper&title=Targeting+trisomic+treatments%3A+Optimizing+dyrk1a+inhibition+to+improve+down+syndrome+deficits&doi=10.1002%2Fmgg3.334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting trisomic treatments: optimizing Dyrk1a inhibition to improve Down syndrome deficits</span></div><div class="casAuthors">Stringer, Megan; Goodlett, Charles R.; Roper, Randall J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Genetics & Genomic Medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">451-465</span>CODEN:
                <span class="NLM_cas:coden">MGGMC6</span>;
        ISSN:<span class="NLM_cas:issn">2324-9269</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Overexpression of Dual-specificity tyrosine-phosphorylated regulated kinase 1A (DYRK1A), located on human chromosome 21, may alter mol. processes linked to developmental deficits in Down syndrome (DS).  Trisomic DYRK1A is a rational therapeutic target, and although redns. in Dyrk1a genetic dosage have shown improvements in trisomic mouse models, attempts to reduce Dyrk1a activity by pharmacol. mechanisms and correct these DS-assocd. phenotypes have been largely unsuccessful.  Epigallocatechin-3-gallate (EGCG) inhibits DYRK1A activity in vitro and this action has been postulated to account for improvement of some DS-assocd. phenotypes that have been reported in preclin. studies and clin. trials.  However, the beneficial effects of EGCG are inconsistent and there is no direct evidence that any obsd. improvement actually occurs through Dyrk1a inhibition.  Inconclusive outcomes likely reflect a lack of knowledge about the tissue-specific patterns of spatial and temporal overexpression and elevated activity of Dyrk1a that may contribute to emerging DS traits during development.  Emerging evidence indicates that Dyrk1a expression varies over the life span in DS mouse models, yet preclin. therapeutic treatments targeting Dyrk1a have largely not considered these developmental changes.  Therapies intended to improve DS phenotypes through normalizing trisomic Dyrk1a need to optimize the timing and dose of treatment to match the spatiotemporal patterning of excessive Dyrk1a activity in relevant tissues.  This will require more precise identification of developmental periods of vulnerability to enduring adverse effects of elevated Dyrk1a, representing the concurrence of increased Dyrk1a expression together with hypothesized tissue-specific-sensitive periods when Dyrk1a regulates cellular processes that shape the long-term functional properties of the tissue.  Future efforts targeting inhibition of trisomic Dyrk1a should identify these putative spatiotemporally specific developmental sensitive periods and det. whether normalizing Dyrk1a activity then can lead to improved outcomes in DS phenotypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7hKbmonKy97Vg90H21EOLACvtfcHk0liP9VI7No6lGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOntL%252FN&md5=5a99b154e1a12b6f8af4d73e391bb966</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fmgg3.334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmgg3.334%26sid%3Dliteratum%253Aachs%26aulast%3DStringer%26aufirst%3DM.%26aulast%3DGoodlett%26aufirst%3DC.%2BR.%26aulast%3DRoper%26aufirst%3DR.%2BJ.%26atitle%3DTargeting%2520trisomic%2520treatments%253A%2520Optimizing%2520dyrk1a%2520inhibition%2520to%2520improve%2520down%2520syndrome%2520deficits%26jtitle%3DMol.%2520Genet.%2520Genomic%2520Med.%26date%3D2017%26volume%3D5%26spage%3D451%26epage%3D465%26doi%3D10.1002%2Fmgg3.334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goodlett, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stringer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaCombe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roper, R. J.</span></span> <span> </span><span class="NLM_article-title">Evaluation of the therapeutic potential of epigallocatechin-3-gallate (egcg) via oral gavage in young adult down syndrome mice</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">10426</span>, <span class="refDoi"> DOI: 10.1038/s41598-020-67133-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1038%2Fs41598-020-67133-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=32591597" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlSitrbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=10426&author=C.+R.+Goodlettauthor=M.+Stringerauthor=J.+LaCombeauthor=R.+Patelauthor=J.+M.+Wallaceauthor=R.+J.+Roper&title=Evaluation+of+the+therapeutic+potential+of+epigallocatechin-3-gallate+%28egcg%29+via+oral+gavage+in+young+adult+down+syndrome+mice&doi=10.1038%2Fs41598-020-67133-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of the therapeutic potential of Epigallocatechin-3-gallate (EGCG) via oral gavage in young adult Down syndrome mice</span></div><div class="casAuthors">Goodlett, Charles R.; Stringer, Megan; LaCombe, Jonathan; Patel, Roshni; Wallace, Joseph M.; Roper, Randall J.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">10426</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Epigallocatechin-3-gallate (EGCG) is a candidate therapeutic for Down syndrome (DS) phenotypes based on in vitro inhibition of DYRK1A, a triplicated gene product of Trisomy 21 (Ts21).  Consumption of green tea exts. contg. EGCG improved some cognitive and behavioral outcomes in DS mouse models and in humans with Ts21.  In contrast, treatment with pure EGCG in DS mouse models did not improve neurobehavioral phenotypes.  This study tested the hypothesis that 200 mg/kg/day of pure EGCG, given via oral gavage, would improve neurobehavioral and skeletal phenotypes in the Ts65Dn DS mouse model.  Serum EGCG levels post-gavage were significantly higher in trisomic mice than in euploid mice.  Daily EGCG gavage treatments over three weeks resulted in growth deficits in both euploid and trisomic mice.  Compared to vehicle treatment, EGCG did not significantly improve behavioral performance of Ts65Dn mice in the multivariate concentric square field, balance beam, or Morris water maze tasks, but reduced swimming speed.  Furthermore, EGCG resulted in reduced cortical bone structure and strength in Ts65Dn mice.  These outcomes failed to support the therapeutic potential of EGCG, and the deleterious effects on growth and skeletal phenotypes underscore the need for caution in high-dose EGCG supplements as an intervention in DS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolMr6fUwdoLrVg90H21EOLACvtfcHk0liP9VI7No6lGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlSitrbM&md5=ce6e23ab184a700a54f3c403c401e4f4</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fs41598-020-67133-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-020-67133-z%26sid%3Dliteratum%253Aachs%26aulast%3DGoodlett%26aufirst%3DC.%2BR.%26aulast%3DStringer%26aufirst%3DM.%26aulast%3DLaCombe%26aufirst%3DJ.%26aulast%3DPatel%26aufirst%3DR.%26aulast%3DWallace%26aufirst%3DJ.%2BM.%26aulast%3DRoper%26aufirst%3DR.%2BJ.%26atitle%3DEvaluation%2520of%2520the%2520therapeutic%2520potential%2520of%2520epigallocatechin-3-gallate%2520%2528egcg%2529%2520via%2520oral%2520gavage%2520in%2520young%2520adult%2520down%2520syndrome%2520mice%26jtitle%3DSci.%2520Rep.%26date%3D2020%26volume%3D10%26spage%3D10426%26doi%3D10.1038%2Fs41598-020-67133-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryoo, S.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radnaabazar, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, J.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, I.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheon, Y.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, W.-J.</span></span> <span> </span><span class="NLM_article-title">DYRK1A-mediated Hyperphosphorylation of Tau</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">34850</span>– <span class="NLM_lpage">34857</span>, <span class="refDoi"> DOI: 10.1074/jbc.m707358200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1074%2Fjbc.M707358200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=17906291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlCrt77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=34850-34857&author=S.-R.+Ryooauthor=H.+K.+Jeongauthor=C.+Radnaabazarauthor=J.-J.+Yooauthor=H.-J.+Choauthor=H.-W.+Leeauthor=I.-S.+Kimauthor=Y.-H.+Cheonauthor=Y.+S.+Ahnauthor=S.-H.+Chungauthor=W.-J.+Song&title=DYRK1A-mediated+Hyperphosphorylation+of+Tau&doi=10.1074%2Fjbc.m707358200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">DYRK1A-mediated hyperphosphorylation of Tau: A functional link between Down syndrome and Alzheimer disease</span></div><div class="casAuthors">Ryoo, Soo-Ryoon; Jeong, Hey Kyeong; Radnaabazar, Chinzorig; Yoo, Jin-Ju; Cho, Hyun-Jeong; Lee, Hye-Won; Kim, In-Sook; Cheon, Young-Hee; Ahn, Young Soo; Chung, Sul-Hee; Song, Woo-Joo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">34850-34857</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Most individuals with Down syndrome show early onset of Alzheimer disease (AD), resulting from the extra copy of chromosome 21.  Located on this chromosome is a gene that encodes the dual specificity tyrosine phosphorylation-regulated kinase 1A (DYRK1A).  One of the pathol. hallmarks in AD is the presence of neurofibrillary tangles (NFTs), which are insol. deposits that consist of abnormally hyperphosphorylated Tau.  Previously it was reported that Tau at the Thr-212 residue was phosphorylated by Dyrk1A in vitro.  To det. the physiol. significance of this phosphorylation, an anal. was made of the amt. of phospho-Thr-212-Tau (pT212) in the brain of transgenic mice that overexpress the human DYRK1A protein (DYRK1A TG mice) that we recently generated.  A significant increase in the amt. of pT212 was found in the brains of DYRK1A transgenic mice when compared with age-matched littermate controls.  We further examd. whether Dyrk1A phosphorylates other Tau residues that are implicated in NFTs.  We found that Dyrk1A also phosphorylates Tau at Ser-202 and Ser-404 in vitro.  Phosphorylation by Dyrk1A strongly inhibited the ability of Tau to promote microtubule assembly.  Following this, using mammalian cells and DYRK1A TG mouse brains, it was demonstrated that the amts. of phospho-Ser-202-Tau and phospho-Ser-404-Tau are enhanced when DYRK1A amts. are high.  These results provide the first in vivo evidence for a physiol. role of DYRK1A in the hyperphosphorylation of Tau and suggest that the extra copy of the DYRK1A gene contributes to the early onset of AD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTPRnoJf4Ux7Vg90H21EOLACvtfcHk0liP9VI7No6lGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlCrt77K&md5=8105e5b9e3c9134aafebbe7af1e5cfec</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M707358200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M707358200%26sid%3Dliteratum%253Aachs%26aulast%3DRyoo%26aufirst%3DS.-R.%26aulast%3DJeong%26aufirst%3DH.%2BK.%26aulast%3DRadnaabazar%26aufirst%3DC.%26aulast%3DYoo%26aufirst%3DJ.-J.%26aulast%3DCho%26aufirst%3DH.-J.%26aulast%3DLee%26aufirst%3DH.-W.%26aulast%3DKim%26aufirst%3DI.-S.%26aulast%3DCheon%26aufirst%3DY.-H.%26aulast%3DAhn%26aufirst%3DY.%2BS.%26aulast%3DChung%26aufirst%3DS.-H.%26aulast%3DSong%26aufirst%3DW.-J.%26atitle%3DDYRK1A-mediated%2520Hyperphosphorylation%2520of%2520Tau%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D34850%26epage%3D34857%26doi%3D10.1074%2Fjbc.m707358200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferrer, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrachina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puig, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez de Lagrán, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martí, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avila, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dierssen, M.</span></span> <span> </span><span class="NLM_article-title">Constitutive dyrk1a is abnormally expressed in alzheimer disease, down syndrome, pick disease, and related transgenic models</span>. <i>Neurobiol. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">392</span>– <span class="NLM_lpage">400</span>, <span class="refDoi"> DOI: 10.1016/j.nbd.2005.03.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1016%2Fj.nbd.2005.03.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=16242644" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFCjs7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2005&pages=392-400&author=I.+Ferrerauthor=M.+Barrachinaauthor=B.+Puigauthor=M.+Mart%C3%ADnez+de+Lagr%C3%A1nauthor=E.+Mart%C3%ADauthor=J.+Avilaauthor=M.+Dierssen&title=Constitutive+dyrk1a+is+abnormally+expressed+in+alzheimer+disease%2C+down+syndrome%2C+pick+disease%2C+and+related+transgenic+models&doi=10.1016%2Fj.nbd.2005.03.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Constitutive Dyrk1A is abnormally expressed in Alzheimer disease, Down syndrome, Pick disease, and related transgenic models</span></div><div class="casAuthors">Ferrer, Isidro; Barrachina, Marta; Puig, Berta; Martinez de Lagran, M.; Marti, Eulalia; Avila, Jesus; Dierssen, Mara</div><div class="citationInfo"><span class="NLM_cas:title">Neurobiology of Disease</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">392-400</span>CODEN:
                <span class="NLM_cas:coden">NUDIEM</span>;
        ISSN:<span class="NLM_cas:issn">0969-9961</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">DYRK1A, dual-specificity tyrosine-regulated kinase 1A, maps to human chromosome 21 within the Down syndrome (DS) crit. region.  Dyrk1 phosphorylates the human microtubule-assocd. protein tau at Thr212 in vitro, a residue that is phosphorylated in fetal tau and hyper-phosphorylated in Alzheimer disease (AD) and tauopathies, including Pick disease (PiD).  Furthermore, phosphorylation of Thr212 primes tau for phosphorylation by glycogen synthase kinase 3 (GSK-3).  The present study examines Dyrk1A in the cerebral cortex of sporadic AD, adult DS with assocd. AD, and PiD.  Increased Dyrk1A immunoreactivity has been found in the cytoplasm and nuclei of scattered neurons of the neocortex, entorhinal cortex, and hippocampus in AD, DS, and PiD.  Dyrk1A is found in sarkosyl-insol. fractions which are enriched in phosphorylated tau in AD brains, thus suggesting a possible assocn. of Dyrk1A with neurofibrillary tangle pathol.  Yet, no clear relationship has been obsd. between tau phosphorylation at Thr212, and GSK-3 and Dyrk1A expression in diseased brains.  Transgenic mice bearing a triple tau mutation (G272V, P301L, and R406W) and expressing hyper-phosphoyrylated tau in neurons of the entorhinal cortex, hippocampus, and cerebral neocortex show increased expression of Dyrk1A in individual neurons in the same regions.  However, transgenic mice over-expressing Dyrk1A do not show increased phosphorylation of tau at Thr212, thus suggesting that Dyrk1A over-expression does not trigger per se hyper-phosphorylation of tau at Thr212 in vivo.  The present observations indicate modifications in the expression of constitutive Dyrk1A in the cytoplasm and nuclei of neurons in various neurodegenerative diseases assocd. with tau phosphorylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpW8vHkFXvWMLVg90H21EOLACvtfcHk0lhgV5tD5OExQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFCjs7bN&md5=dd69cd9e3cdb10618294417873670311</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.nbd.2005.03.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.nbd.2005.03.020%26sid%3Dliteratum%253Aachs%26aulast%3DFerrer%26aufirst%3DI.%26aulast%3DBarrachina%26aufirst%3DM.%26aulast%3DPuig%26aufirst%3DB.%26aulast%3DMart%25C3%25ADnez%2Bde%2BLagr%25C3%25A1n%26aufirst%3DM.%26aulast%3DMart%25C3%25AD%26aufirst%3DE.%26aulast%3DAvila%26aufirst%3DJ.%26aulast%3DDierssen%26aufirst%3DM.%26atitle%3DConstitutive%2520dyrk1a%2520is%2520abnormally%2520expressed%2520in%2520alzheimer%2520disease%252C%2520down%2520syndrome%252C%2520pick%2520disease%252C%2520and%2520related%2520transgenic%2520models%26jtitle%3DNeurobiol.%2520Dis.%26date%3D2005%26volume%3D20%26spage%3D392%26epage%3D400%26doi%3D10.1016%2Fj.nbd.2005.03.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamino, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuripa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kudo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamagata, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akatsu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funakoshi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishitomi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakaguchi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uema, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, M.</span></span> <span> </span><span class="NLM_article-title">The DYRK1A gene, encoded in chromosome 21 Down syndrome critical region, bridges between β-amyloid production and tau phosphorylation in Alzheimer disease</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.1093/hmg/ddl437</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1093%2Fhmg%2Fddl437" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=17135279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotVWksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2007&pages=15-23&author=R.+Kimuraauthor=K.+Kaminoauthor=M.+Yamamotoauthor=A.+Nuripaauthor=T.+Kidaauthor=H.+Kazuiauthor=R.+Hashimotoauthor=T.+Tanakaauthor=T.+Kudoauthor=H.+Yamagataauthor=Y.+Tabaraauthor=T.+Mikiauthor=H.+Akatsuauthor=K.+Kosakaauthor=E.+Funakoshiauthor=K.+Nishitomiauthor=G.+Sakaguchiauthor=A.+Katoauthor=H.+Hattoriauthor=T.+Uemaauthor=M.+Takeda&title=The+DYRK1A+gene%2C+encoded+in+chromosome+21+Down+syndrome+critical+region%2C+bridges+between+%CE%B2-amyloid+production+and+tau+phosphorylation+in+Alzheimer+disease&doi=10.1093%2Fhmg%2Fddl437"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The DYRK1A gene, encoded in chromosome 21 Down syndrome critical region, bridges between β-amyloid production and tau phosphorylation in Alzheimer disease</span></div><div class="casAuthors">Kimura, Ryo; Kamino, Kouzin; Yamamoto, Mitsuko; Nuripa, Aidaralieva; Kida, Tomoyuki; Kazui, Hiroaki; Hashimoto, Ryota; Tanaka, Toshihisa; Kudo, Takashi; Yamagata, Hidehisa; Tabara, Yasuharu; Miki, Tetsuro; Akatsu, Hiroyasu; Kosaka, Kenji; Funakoshi, Eishi; Nishitomi, Kouhei; Sakaguchi, Gaku; Kato, Akira; Hattori, Hideyuki; Uema, Takeshi; Takeda, Masatoshi</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-23</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">We scanned throughout chromosome 21 to assess genetic assocns. with late-onset Alzheimer disease (AD) using 374 Japanese patients and 375 population-based controls, because trisomy 21 is known to be assocd. with early deposition of β-amyloid (Aβ) in the brain.  Among 417 markers spanning 33 Mb, 22 markers showed assocns. with either the allele or the genotype frequency (P < 0.05).  Logistic regression anal. with age, sex and apolipoprotein E (APOE)-ε4 dose supported genetic risk of 17 markers, of which eight markers were linked to the SAMSN1, PRSS7, NCAM2, RUNX1, DYRK1A and KCNJ6 genes.  In logistic regression, the DYRK1A (dual-specificity tyrosine-regulated kinase 1A) gene, located in the Down syndrome crit. region, showed the highest significance [OR = 2.99 (95% CI: 1.72-5.19), P = 0.001], whereas the RUNX1 gene showed a high odds ratio [OR = 23.3 (95% CI: 2.76-196.5), P = 0.038].  DYRK1A mRNA level in the hippocampus was significantly elevated in patients with AD when compared with pathol. controls (P < 0.01).  DYRK1A mRNA level was upregulated along with an increase in the Aβ-level in the brain of transgenic mice, overproducing Aβ at 9 mo of age.  In neuroblastoma cells, Aβ induced an increase in the DYRK1A transcript, which also led to tau phosphorylation at Thr212 under the overexpression of tau.  Therefore, the upregulation of DYRK1A transcription results from Aβ loading, further leading to tau phosphorylation.  These result indicates that DYRK1A could be a key mol. bridging between β-amyloid prodn. and tau phosphorylation in AD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpETsvyVeafabVg90H21EOLACvtfcHk0lhgV5tD5OExQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotVWksA%253D%253D&md5=5aa7f15a92ce6328d33cb5cb179b2be2</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2Fddl437&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252Fddl437%26sid%3Dliteratum%253Aachs%26aulast%3DKimura%26aufirst%3DR.%26aulast%3DKamino%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DNuripa%26aufirst%3DA.%26aulast%3DKida%26aufirst%3DT.%26aulast%3DKazui%26aufirst%3DH.%26aulast%3DHashimoto%26aufirst%3DR.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DKudo%26aufirst%3DT.%26aulast%3DYamagata%26aufirst%3DH.%26aulast%3DTabara%26aufirst%3DY.%26aulast%3DMiki%26aufirst%3DT.%26aulast%3DAkatsu%26aufirst%3DH.%26aulast%3DKosaka%26aufirst%3DK.%26aulast%3DFunakoshi%26aufirst%3DE.%26aulast%3DNishitomi%26aufirst%3DK.%26aulast%3DSakaguchi%26aufirst%3DG.%26aulast%3DKato%26aufirst%3DA.%26aulast%3DHattori%26aufirst%3DH.%26aulast%3DUema%26aufirst%3DT.%26aulast%3DTakeda%26aufirst%3DM.%26atitle%3DThe%2520DYRK1A%2520gene%252C%2520encoded%2520in%2520chromosome%252021%2520Down%2520syndrome%2520critical%2520region%252C%2520bridges%2520between%2520%25CE%25B2-amyloid%2520production%2520and%2520tau%2520phosphorylation%2520in%2520Alzheimer%2520disease%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2007%26volume%3D16%26spage%3D15%26epage%3D23%26doi%3D10.1093%2Fhmg%2Fddl437" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barallobre, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bové, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laguna, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delabar, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vila, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arbonés, M. L.</span></span> <span> </span><span class="NLM_article-title">Dyrk1a promotes dopaminergic neuron survival in the developing brain and in a mouse model of parkinson’s disease</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e1289</span> <span class="refDoi"> DOI: 10.1038/cddis.2014.253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1038%2Fcddis.2014.253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=24922073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpsVGqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&author=M.+J.+Barallobreauthor=C.+Perierauthor=J.+Bov%C3%A9author=A.+Lagunaauthor=J.+M.+Delabarauthor=M.+Vilaauthor=M.+L.+Arbon%C3%A9s&title=Dyrk1a+promotes+dopaminergic+neuron+survival+in+the+developing+brain+and+in+a+mouse+model+of+parkinson%E2%80%99s+disease&doi=10.1038%2Fcddis.2014.253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">DYRK1A promotes dopaminergic neuron survival in the developing brain and in a mouse model of Parkinson's disease</span></div><div class="casAuthors">Barallobre, M. J.; Perier, C.; Bove, J.; Laguna, A.; Delabar, J. M.; Vila, M.; Arbones, M. L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Disease</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e1289</span>CODEN:
                <span class="NLM_cas:coden">CDDEA4</span>;
        ISSN:<span class="NLM_cas:issn">2041-4889</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In the brain, programmed cell death (PCD) serves to adjust the nos. of the different types of neurons during development, and its pathol. reactivation in the adult leads to neurodegeneration.  Dual-specificity tyrosine-(Y)-phosphorylation regulated kinase 1A (DYRK1A) is a pleiotropic kinase involved in neural proliferation and cell death, and its role during brain growth is evolutionarily conserved.  Human DYRK1A lies in the Down syndrome crit. region on chromosome 21, and heterozygous mutations in the gene cause microcephaly and neurol. dysfunction.  The mouse model for DYRK1A haploinsufficiency (the Dyrk1a+/- mouse) presents neuronal deficits in specific regions of the adult brain, including the substantia nigra (SN), although the mechanisms underlying these pathogenic effects remain unclear.  Here we study the effect of DYRK1A copy no. variation on dopaminergic cell homeostasis.  We show that mesencephalic DA (mDA) neurons are generated in the embryo at normal rates in the Dyrk1a haploinsufficient model and in a model (the mBACtgDyrk1a mouse) that carries three copies of Dyrk1a.  We also show that the no. of mDA cells diminishes in postnatal Dyrk1a+/- mice and increases in mBACtgDyrk1a mice due to an abnormal activity of the mitochondrial caspase9 (Casp9)-dependent apoptotic pathway during the main wave of PCD that affects these neurons.  In addn., we show that the cell death induced by 1-methyl-4-phenyl-1,2,3,6 tetrahydropyridine (MPTP), a toxin that activates Casp9-dependent apoptosis in mDA neurons, is attenuated in adult mBACtgDyrk1a mice, leading to an increased survival of SN DA neurons 21 days after MPTP intoxication.  Finally, we present data indicating that Dyrk1a phosphorylation of Casp9 at the Thr125 residue is the mechanism by which this kinase hinders both physiol. and pathol. PCD in mDA neurons.  These data provide new insight into the mechanisms that control cell death in brain DA neurons and they show that deregulation of developmental apoptosis may contribute to the phenotype of patients with imbalanced DYRK1A gene dosage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2sgRdeNjB2rVg90H21EOLACvtfcHk0ljMXPprDeZP8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpsVGqt70%253D&md5=4b0f7afaa86266fa556f74c6f12960b2</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fcddis.2014.253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddis.2014.253%26sid%3Dliteratum%253Aachs%26aulast%3DBarallobre%26aufirst%3DM.%2BJ.%26aulast%3DPerier%26aufirst%3DC.%26aulast%3DBov%25C3%25A9%26aufirst%3DJ.%26aulast%3DLaguna%26aufirst%3DA.%26aulast%3DDelabar%26aufirst%3DJ.%2BM.%26aulast%3DVila%26aufirst%3DM.%26aulast%3DArbon%25C3%25A9s%26aufirst%3DM.%2BL.%26atitle%3DDyrk1a%2520promotes%2520dopaminergic%2520neuron%2520survival%2520in%2520the%2520developing%2520brain%2520and%2520in%2520a%2520mouse%2520model%2520of%2520parkinson%25E2%2580%2599s%2520disease%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2014%26volume%3D5%26doi%3D10.1038%2Fcddis.2014.253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwatsubo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paik, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, K. C.</span></span> <span> </span><span class="NLM_article-title">Dyrk1A Phosphorylates α-Synuclein and Enhances Intracellular Inclusion Formation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">33250</span>– <span class="NLM_lpage">33257</span>, <span class="refDoi"> DOI: 10.1074/jbc.m606147200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1074%2Fjbc.M606147200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=16959772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFegsb7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=33250-33257&author=E.+J.+Kimauthor=J.+Y.+Sungauthor=H.+J.+Leeauthor=H.+Rhimauthor=M.+Hasegawaauthor=T.+Iwatsuboauthor=D.+S.+Minauthor=J.+Kimauthor=S.+R.+Paikauthor=K.+C.+Chung&title=Dyrk1A+Phosphorylates+%CE%B1-Synuclein+and+Enhances+Intracellular+Inclusion+Formation&doi=10.1074%2Fjbc.m606147200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Dyrk1A phosphorylates α-Synuclein and enhances intracellular inclusion formation</span></div><div class="casAuthors">Kim, Eun Joo; Sung, Jee Young; Lee, Hyun Jung; Rhim, Hyewhon; Hasegawa, Masato; Iwatsubo, Takeshi; Min, Do Sik; Kim, Jongsun; Paik, Seung R.; Chung, Kwang Chul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">33250-33257</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Lewy bodies (LBs) are pathol. hallmarks of Parkinson disease (PD) but also occur in Alzheimer disease (AD) and dementia of LBs. α-Synuclein, the major component of LBs, is obsd. in the brain of Down syndrome (DS) patients with AD.  Dyrk1A, a dual specificity tyrosine-regulated kinase (Dyrk) family member, is the mammalian ortholog of the Drosophila minibrain (Mnb) gene, essential for normal postembryonic neurogenesis.  The Dyrk1A gene resides in the human chromosome 21q22.2 region, which is assocd. with DS anomalies, including mental retardation.  In this study, we examd. whether Dyrk1A interacts with α-synuclein and subsequently affects intracellular α-synuclein inclusion formation in immortalized hippocampal neuronal (H19-7) cells.  Dyrk1A selectively binds to α-synuclein in transformed and primary neuronal cells.  α-Synuclein overexpression, followed by basic fibroblast growth factor-induced neuronal differentiation, resulted in cell death.  We obsd. that accompanying cell death was increased α-synuclein phosphorylation and intracytoplasmic aggregation.  In addn., the transfection of kinase-inactive Dyrk1A or Dyrk1A small interfering RNA blocked α-synuclein phosphorylation and aggregate formation.  In vitro kinase assay of anti-Dyrk1A immunocomplexes demonstrated that Dyrk1A could phosphorylate α-synuclein at Ser-87.  Furthermore, aggregates formed by phosphorylated α-synuclein have a distinct morphol. and are more neurotoxic compared with aggregates composed of unmodified wild type α-synuclein.  These findings suggest α-synuclein inclusion formation regulated by Dyrk1A, potentially affecting neuronal cell viability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgp1DXYX4SP7Vg90H21EOLACvtfcHk0ljMXPprDeZP8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFegsb7P&md5=c3b2c23a59da54a7f9444a7c6d01c318</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M606147200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M606147200%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DE.%2BJ.%26aulast%3DSung%26aufirst%3DJ.%2BY.%26aulast%3DLee%26aufirst%3DH.%2BJ.%26aulast%3DRhim%26aufirst%3DH.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DIwatsubo%26aufirst%3DT.%26aulast%3DMin%26aufirst%3DD.%2BS.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DPaik%26aufirst%3DS.%2BR.%26aulast%3DChung%26aufirst%3DK.%2BC.%26atitle%3DDyrk1A%2520Phosphorylates%2520%25CE%25B1-Synuclein%2520and%2520Enhances%2520Intracellular%2520Inclusion%2520Formation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D33250%26epage%3D33257%26doi%3D10.1074%2Fjbc.m606147200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dirice, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walpita, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetere, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahraman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dančík, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, B. K.</span></span> <span> </span><span class="NLM_article-title">Inhibition of DYRK1A Stimulates Human β-Cell Proliferation</span>. <i>Diabetes</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1660</span>– <span class="NLM_lpage">1671</span>, <span class="refDoi"> DOI: 10.2337/db15-1127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.2337%2Fdb15-1127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=26953159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWns7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2016&pages=1660-1671&author=E.+Diriceauthor=D.+Walpitaauthor=A.+Vetereauthor=B.+C.+Meierauthor=S.+Kahramanauthor=J.+Huauthor=V.+Dan%C4%8D%C3%ADkauthor=S.+M.+Burnsauthor=T.+J.+Gilbertauthor=D.+E.+Olsonauthor=P.+A.+Clemonsauthor=R.+N.+Kulkarniauthor=B.+K.+Wagner&title=Inhibition+of+DYRK1A+Stimulates+Human+%CE%B2-Cell+Proliferation&doi=10.2337%2Fdb15-1127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of DYRK1A stimulates human beta-cell proliferation</span></div><div class="casAuthors">Dirice, Ercument; Walpita, Deepika; Vetere, Amedeo; Meier, Bennett C.; Kahraman, Sevim; Hu, Jiang; Dancik, Vlado; Burns, Sean M.; Gilbert, Tamara J.; Olson, David E.; Clemons, Paul A.; Kulkarni, Rohit N.; Wagner, Bridget K.</div><div class="citationInfo"><span class="NLM_cas:title">Diabetes</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1660-1671</span>CODEN:
                <span class="NLM_cas:coden">DIAEAZ</span>;
        ISSN:<span class="NLM_cas:issn">0012-1797</span>.
    
            (<span class="NLM_cas:orgname">American Diabetes Association, Inc.</span>)
        </div><div class="casAbstract">Restoring functional β-cell mass is an important therapeutic goal for both type 1 and type 2 diabetes (1).  While proliferation of existing β-cells is the primary means of β-cell replacement in rodents (2), it is unclear whether a similar principle applies to humans, as human β-cells are remarkably resistant to stimulation of division (3,4).  Here, we show that 5-iodotubercidin (5-IT), an annotated adenosine kinase inhibitor previously reported to increase proliferation in rodent and porcine islets (5), strongly and selectively increases human β-cell proliferation in vitro and in vivo.  Remarkably, 5-IT also increased glucose-dependent insulin secretion after prolonged treatment.  Kinome profiling revealed 5-IT to be a potent and selective inhibitor of the dual-specificity tyrosine phosphorylation-regulated kinase (DYRK) and cell division cycle-like kinase families.  Induction of β-cell proliferation by either 5-IT or harmine, another natural product DYRK1A inhibitor, was suppressed by coincubation with the calcineurin inhibitor FK506, suggesting involvement of DYRK1A and nuclear factor of activated T cells signaling.  Gene expression profiling in whole islets treated with 5-IT revealed induction of proliferation- and cell cycle-related genes, suggesting that true proliferation is induced by 5-IT.  Furthermore, 5-IT promotes β-cell proliferation in human islets grafted under the kidney capsule of NOD-scid IL2Rgnull mice.  These results point to inhibition of DYRK1A as a therapeutic strategy to increase human β-cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqLKPVi2wkdrVg90H21EOLACvtfcHk0ljMXPprDeZP8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWns7vP&md5=c383c8d1f2c1739252ad3cfe4361585c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.2337%2Fdb15-1127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2337%252Fdb15-1127%26sid%3Dliteratum%253Aachs%26aulast%3DDirice%26aufirst%3DE.%26aulast%3DWalpita%26aufirst%3DD.%26aulast%3DVetere%26aufirst%3DA.%26aulast%3DMeier%26aufirst%3DB.%2BC.%26aulast%3DKahraman%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DDan%25C4%258D%25C3%25ADk%26aufirst%3DV.%26aulast%3DBurns%26aufirst%3DS.%2BM.%26aulast%3DGilbert%26aufirst%3DT.%2BJ.%26aulast%3DOlson%26aufirst%3DD.%2BE.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DKulkarni%26aufirst%3DR.%2BN.%26aulast%3DWagner%26aufirst%3DB.%2BK.%26atitle%3DInhibition%2520of%2520DYRK1A%2520Stimulates%2520Human%2520%25CE%25B2-Cell%2520Proliferation%26jtitle%3DDiabetes%26date%3D2016%26volume%3D65%26spage%3D1660%26epage%3D1671%26doi%3D10.2337%2Fdb15-1127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen-Tran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meeusen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamireddy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swafford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bursalaya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiDonato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harb, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippi, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deaton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suarez-Pinzon, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herman, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hering, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin Seidel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNamara, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glynne, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laffitte, B.</span></span> <span> </span><span class="NLM_article-title">Inhibition of dyrk1a and gsk3b induces human beta-cell proliferation</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">8372</span>, <span class="refDoi"> DOI: 10.1038/ncomms9372</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1038%2Fncomms9372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=26496802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslegu7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8372&author=W.+Shenauthor=B.+Taylorauthor=Q.+Jinauthor=V.+Nguyen-Tranauthor=S.+Meeusenauthor=Y.+Q.+Zhangauthor=A.+Kamireddyauthor=A.+Swaffordauthor=A.+F.+Powersauthor=J.+Walkerauthor=J.+Lambauthor=B.+Bursalayaauthor=M.+DiDonatoauthor=G.+Harbauthor=M.+Qiuauthor=C.+M.+Filippiauthor=L.+Deatonauthor=C.+N.+Turkauthor=W.+L.+Suarez-Pinzonauthor=Y.+Liuauthor=X.+Haoauthor=T.+Moauthor=S.+Yanauthor=J.+Liauthor=A.+E.+Hermanauthor=B.+J.+Heringauthor=T.+Wuauthor=H.+Martin+Seidelauthor=P.+McNamaraauthor=R.+Glynneauthor=B.+Laffitte&title=Inhibition+of+dyrk1a+and+gsk3b+induces+human+beta-cell+proliferation&doi=10.1038%2Fncomms9372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of DYRK1A and GSK3B induces human β-cell proliferation</span></div><div class="casAuthors">Shen, Weijun; Taylor, Brandon; Jin, Qihui; Nguyen-Tran, Van; Meeusen, Shelly; Zhang, You-Qing; Kamireddy, Anwesh; Swafford, Austin; Powers, Andrew F.; Walker, John; Lamb, John; Bursalaya, Badry; DiDonato, Michael; Harb, George; Qiu, Minhua; Filippi, Christophe M.; Deaton, Lisa; Turk, Carolina N.; Suarez-Pinzon, Wilma L.; Liu, Yahu; Hao, Xueshi; Mo, Tingting; Yan, Shanshan; Li, Jing; Herman, Ann E.; Hering, Bernhard J.; Wu, Tom; Seidel, H. Martin; McNamara, Peter; Glynne, Richard; Laffitte, Bryan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8372</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Insufficient pancreatic β-cell mass or function results in diabetes mellitus.  While significant progress has been made in regulating insulin secretion from β-cells in diabetic patients, no pharmacol. agents have been described that increase β-cell replication in humans.  Here we report aminopyrazine compds. that stimulate robust β-cell proliferation in adult primary islets, most likely as a result of combined inhibition of DYRK1A and GSK3B.  Aminopyrazine-treated human islets retain functionality in vitro and after transplantation into diabetic mice.  Oral dosing of these compds. in diabetic mice induces β-cell proliferation, increases β-cell mass and insulin content, and improves glycemic control.  Biochem., genetic and cell biol. data point to Dyrk1a as the key mol. target.  This study supports the feasibility of treating diabetes with an oral therapy to restore β-cell mass, and highlights a tractable pathway for future drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVHzP-VkH5nrVg90H21EOLACvtfcHk0lh0PXsb_pR5Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslegu7%252FK&md5=5ef960af3bfd2795b9fdce52876e5d59</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fncomms9372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms9372%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DW.%26aulast%3DTaylor%26aufirst%3DB.%26aulast%3DJin%26aufirst%3DQ.%26aulast%3DNguyen-Tran%26aufirst%3DV.%26aulast%3DMeeusen%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.%2BQ.%26aulast%3DKamireddy%26aufirst%3DA.%26aulast%3DSwafford%26aufirst%3DA.%26aulast%3DPowers%26aufirst%3DA.%2BF.%26aulast%3DWalker%26aufirst%3DJ.%26aulast%3DLamb%26aufirst%3DJ.%26aulast%3DBursalaya%26aufirst%3DB.%26aulast%3DDiDonato%26aufirst%3DM.%26aulast%3DHarb%26aufirst%3DG.%26aulast%3DQiu%26aufirst%3DM.%26aulast%3DFilippi%26aufirst%3DC.%2BM.%26aulast%3DDeaton%26aufirst%3DL.%26aulast%3DTurk%26aufirst%3DC.%2BN.%26aulast%3DSuarez-Pinzon%26aufirst%3DW.%2BL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHao%26aufirst%3DX.%26aulast%3DMo%26aufirst%3DT.%26aulast%3DYan%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DHerman%26aufirst%3DA.%2BE.%26aulast%3DHering%26aufirst%3DB.%2BJ.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DMartin%2BSeidel%26aufirst%3DH.%26aulast%3DMcNamara%26aufirst%3DP.%26aulast%3DGlynne%26aufirst%3DR.%26aulast%3DLaffitte%26aufirst%3DB.%26atitle%3DInhibition%2520of%2520dyrk1a%2520and%2520gsk3b%2520induces%2520human%2520beta-cell%2520proliferation%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D8372%26doi%3D10.1038%2Fncomms9372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rachdi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kariyawasam, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aïello, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herault, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delabar, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polak, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scharfmann, R.</span></span> <span> </span><span class="NLM_article-title">Dyrk1A induces pancreatic β cell mass expansion and improves glucose tolerance</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2221</span>– <span class="NLM_lpage">2229</span>, <span class="refDoi"> DOI: 10.4161/cc.29250</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.4161%2Fcc.29250" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=24870561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1WisLzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=2221-2229&author=L.+Rachdiauthor=D.+Kariyawasamauthor=V.+A%C3%AFelloauthor=Y.+Heraultauthor=N.+Janelauthor=J.-M.+Delabarauthor=M.+Polakauthor=R.+Scharfmann&title=Dyrk1A+induces+pancreatic+%CE%B2+cell+mass+expansion+and+improves+glucose+tolerance&doi=10.4161%2Fcc.29250"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Dyrk1A induces pancreatic β cell mass expansion and improves glucose tolerance</span></div><div class="casAuthors">Rachdi, Latif; Kariyawasam, Dulanjalee; Aiello, Virginie; Herault, Yann; Janel, Nathalie; Delabar, Jean-Maurice; Polak, Michel; Scharfmann, Raphael</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2221-2229</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Type 2 diabetes is caused by a limited capacity of insulin-producing pancreatic β cells to increase their mass and function in response to insulin resistance.  The signaling pathways that pos. regulate functional β cell mass have not been fully elucidated.  DYRK1A (also called minibrain/MNB) is a member of the dual-specificity tyrosine phosphorylation-regulated kinase (DYRK) family.  A significant amt. of data implicates DYRK1A in brain growth and Down syndrome, and recent data indicate that Dyrk1A haploinsufficient mice have a low functional β cell mass.  Here we ask whether Dyrk1A upregulation could be a way to increase functional β cell mass.  We used mice overexpressing Dyrk1A under the control of its own regulatory sequences (mBACTgDyrk1A).  These mice exhibit decreased glucose levels and hyperinsulinemia in the fasting state.  Improved glucose tolerance is obsd. in these mice as early as 4 wk of age.  Upregulation of Dyrk1A in β cells induces expansion of β cell mass through increased proliferation and cell size.  Importantly, mBACTgDyrk1A mice are protected against high-fat-diet-induced β cell failure through increase in β cell mass and insulin sensitivity.  These studies show the crucial role of the DYRK1A pathway in the regulation of β cell mass and carbohydrate metab. in vivo.  Activating the DYRK1A pathway could thus represent an innovative way to increase functional β cell mass.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre5nXGBuindrVg90H21EOLACvtfcHk0lh0PXsb_pR5Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1WisLzP&md5=92dc7241ee0f258a91351b6bffd941d8</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.4161%2Fcc.29250&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.29250%26sid%3Dliteratum%253Aachs%26aulast%3DRachdi%26aufirst%3DL.%26aulast%3DKariyawasam%26aufirst%3DD.%26aulast%3DA%25C3%25AFello%26aufirst%3DV.%26aulast%3DHerault%26aufirst%3DY.%26aulast%3DJanel%26aufirst%3DN.%26aulast%3DDelabar%26aufirst%3DJ.-M.%26aulast%3DPolak%26aufirst%3DM.%26aulast%3DScharfmann%26aufirst%3DR.%26atitle%3DDyrk1A%2520induces%2520pancreatic%2520%25CE%25B2%2520cell%2520mass%2520expansion%2520and%2520improves%2520glucose%2520tolerance%26jtitle%3DCell%2520Cycle%26date%3D2014%26volume%3D13%26spage%3D2221%26epage%3D2229%26doi%3D10.4161%2Fcc.29250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soundararajan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roos, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kettenbach, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerber, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eswaran, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span> <span> </span><span class="NLM_article-title">Structures of down syndrome kinases, dyrks, reveal mechanisms of kinase activation and substrate recognition</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">986</span>– <span class="NLM_lpage">996</span>, <span class="refDoi"> DOI: 10.1016/j.str.2013.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1016%2Fj.str.2013.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=23665168" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsVygtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=986-996&author=M.+Soundararajanauthor=A.+K.+Roosauthor=P.+Savitskyauthor=P.+Filippakopoulosauthor=A.+N.+Kettenbachauthor=J.+V.+Olsenauthor=S.+A.+Gerberauthor=J.+Eswaranauthor=S.+Knappauthor=J.+M.+Elkins&title=Structures+of+down+syndrome+kinases%2C+dyrks%2C+reveal+mechanisms+of+kinase+activation+and+substrate+recognition&doi=10.1016%2Fj.str.2013.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of Down Syndrome Kinases, DYRKs, Reveal Mechanisms of Kinase Activation and Substrate Recognition</span></div><div class="casAuthors">Soundararajan, Meera; Roos, Annette K.; Savitsky, Pavel; Filippakopoulos, Panagis; Kettenbach, Arminja N.; Olsen, Jesper V.; Gerber, Scott A.; Eswaran, Jeyanthy; Knapp, Stefan; Elkins, Jonathan M.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">986-996</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Dual-specificity tyrosine-(Y)-phosphorylation-regulated kinases (DYRKs) play key roles in brain development, regulation of splicing, and apoptosis, and are potential drug targets for neurodegenerative diseases and cancer.  We present crystal structures of one representative member of each DYRK subfamily: DYRK1A with an ATP-mimetic inhibitor and consensus peptide, and DYRK2 including NAPA and DH (DYRK homol.) box regions.  The current activation model suggests that DYRKs are Ser/Thr kinases that only autophosphorylate the second tyrosine of the activation loop YxY motif during protein translation.  The structures explain the roles of this tyrosine and of the DH box in DYRK activation and provide a structural model for DYRK substrate recognition.  Phosphorylation of a library of naturally occurring peptides identified substrate motifs that lack proline in the P+1 position, suggesting that DYRK1A is not a strictly proline-directed kinase.  Our data also show that DYRK1A wild-type and Y321F mutant retain tyrosine autophosphorylation activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDShaymgKcfLVg90H21EOLACvtfcHk0lh0PXsb_pR5Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsVygtL0%253D&md5=f101707a987083c7d3d637b39332db35</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2013.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2013.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DSoundararajan%26aufirst%3DM.%26aulast%3DRoos%26aufirst%3DA.%2BK.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKettenbach%26aufirst%3DA.%2BN.%26aulast%3DOlsen%26aufirst%3DJ.%2BV.%26aulast%3DGerber%26aufirst%3DS.%2BA.%26aulast%3DEswaran%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26atitle%3DStructures%2520of%2520down%2520syndrome%2520kinases%252C%2520dyrks%252C%2520reveal%2520mechanisms%2520of%2520kinase%2520activation%2520and%2520substrate%2520recognition%26jtitle%3DStructure%26date%3D2013%26volume%3D21%26spage%3D986%26epage%3D996%26doi%3D10.1016%2Fj.str.2013.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weber, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sipos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paczal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balint, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kun, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foloppe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dokurno, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massey, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walmsley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benwell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edmonds, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demarles, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruno, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burbridge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruzalegui, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotschy, A.</span></span> <span> </span><span class="NLM_article-title">Structure-Guided Discovery of Potent and Selective DYRK1A Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.1c00023</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.1c00023" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2021&author=C.+Weberauthor=M.+Siposauthor=A.+Paczalauthor=B.+Balintauthor=V.+Kunauthor=N.+Foloppeauthor=P.+Dokurnoauthor=A.+J.+Masseyauthor=D.+L.+Walmsleyauthor=R.+E.+Hubbardauthor=J.+Murrayauthor=K.+Benwellauthor=T.+Edmondsauthor=D.+Demarlesauthor=A.+Brunoauthor=M.+Burbridgeauthor=F.+Cruzaleguiauthor=A.+Kotschy&title=Structure-Guided+Discovery+of+Potent+and+Selective+DYRK1A+Inhibitors&doi=10.1021%2Facs.jmedchem.1c00023"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.1c00023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.1c00023%26sid%3Dliteratum%253Aachs%26aulast%3DWeber%26aufirst%3DC.%26aulast%3DSipos%26aufirst%3DM.%26aulast%3DPaczal%26aufirst%3DA.%26aulast%3DBalint%26aufirst%3DB.%26aulast%3DKun%26aufirst%3DV.%26aulast%3DFoloppe%26aufirst%3DN.%26aulast%3DDokurno%26aufirst%3DP.%26aulast%3DMassey%26aufirst%3DA.%2BJ.%26aulast%3DWalmsley%26aufirst%3DD.%2BL.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DBenwell%26aufirst%3DK.%26aulast%3DEdmonds%26aufirst%3DT.%26aulast%3DDemarles%26aufirst%3DD.%26aulast%3DBruno%26aufirst%3DA.%26aulast%3DBurbridge%26aufirst%3DM.%26aulast%3DCruzalegui%26aufirst%3DF.%26aulast%3DKotschy%26aufirst%3DA.%26atitle%3DStructure-Guided%2520Discovery%2520of%2520Potent%2520and%2520Selective%2520DYRK1A%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2021%26doi%3D10.1021%2Facs.jmedchem.1c00023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ha, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krauthammer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halaban, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boggon, T. J.</span></span> <span> </span><span class="NLM_article-title">Type ii p21-activated kinases (paks) are regulated by an autoinhibitory pseudosubstrate</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">16107</span>– <span class="NLM_lpage">16112</span>, <span class="refDoi"> DOI: 10.1073/pnas.1214447109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1073%2Fpnas.1214447109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=22988085" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGhsL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=16107-16112&author=B.+H.+Haauthor=M.+J.+Davisauthor=C.+Chenauthor=H.+J.+Louauthor=J.+Gaoauthor=R.+Zhangauthor=M.+Krauthammerauthor=R.+Halabanauthor=J.+Schlessingerauthor=B.+E.+Turkauthor=T.+J.+Boggon&title=Type+ii+p21-activated+kinases+%28paks%29+are+regulated+by+an+autoinhibitory+pseudosubstrate&doi=10.1073%2Fpnas.1214447109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Type II p21-activated kinases (PAKs) are regulated by an autoinhibitory pseudosubstrate</span></div><div class="casAuthors">Ha, Byung Hak; Davis, Matthew J.; Chen, Catherine; Lou, Hua Jane; Gao, Jia; Zhang, Rong; Krauthammer, Michael; Halaban, Ruth; Schlessinger, Joseph; Turk, Benjamin E.; Boggon, Titus J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">16107-16112, S16107/1-S16107/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The type II p21-activated kinases (PAKs) are key effectors of RHO-family GTPases involved in cell motility, survival, and proliferation.  Using a structure-guided approach, we discovered that type II PAKs are regulated by an N-terminal autoinhibitory pseudosubstrate motif centered on a crit. proline residue, and that this regulation occurs independently of activation loop phosphorylation.  We detd. six X-ray crystal structures of either full-length PAK4 or its catalytic domain, that demonstrate the mol. basis for pseudosubstrate binding to the active state with phosphorylated activation loop.  We show that full-length PAK4 is constitutively autoinhibited, but mutation of the pseudo-substrate releases this inhibition and causes increased phosphorylation of the apoptotic regulation protein Bcl-2/Bcl-XL antagonist causing cell death and cellular morphol. changes.  We also find that PAK6 is regulated by the pseudosubstrate region, indicating a common type II PAK autoregulatory mechanism.  Finally, we find Src SH3, but not β-PIX SH3, can activate PAK4.  We provide a unique understanding for type II PAK regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreLSMEq29IlrVg90H21EOLACvtfcHk0lhHH6Ap_ceJ6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGhsL3J&md5=65e13a76626af2ecb01509f70d36711e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1214447109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1214447109%26sid%3Dliteratum%253Aachs%26aulast%3DHa%26aufirst%3DB.%2BH.%26aulast%3DDavis%26aufirst%3DM.%2BJ.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DLou%26aufirst%3DH.%2BJ.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DKrauthammer%26aufirst%3DM.%26aulast%3DHalaban%26aufirst%3DR.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DTurk%26aufirst%3DB.%2BE.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26atitle%3DType%2520ii%2520p21-activated%2520kinases%2520%2528paks%2529%2520are%2520regulated%2520by%2520an%2520autoinhibitory%2520pseudosubstrate%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D16107%26epage%3D16112%26doi%3D10.1073%2Fpnas.1214447109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baurin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aboul-Ela, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barril, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drysdale, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dymock, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finch, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fromont, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmonite, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">Design and characterization of libraries of molecular fragments for use in nmr screening against protein targets</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">2157</span>– <span class="NLM_lpage">2166</span>, <span class="refDoi"> DOI: 10.1021/ci049806z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049806z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotlaru78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=2157-2166&author=N.+Baurinauthor=F.+Aboul-Elaauthor=X.+Barrilauthor=B.+Davisauthor=M.+Drysdaleauthor=B.+Dymockauthor=H.+Finchauthor=C.+Fromontauthor=C.+Richardsonauthor=H.+Simmoniteauthor=R.+E.+Hubbard&title=Design+and+characterization+of+libraries+of+molecular+fragments+for+use+in+nmr+screening+against+protein+targets&doi=10.1021%2Fci049806z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Characterization of Libraries of Molecular Fragments for Use in NMR Screening against Protein Targets</span></div><div class="casAuthors">Baurin, Nicolas; Aboul-Ela, Fareed; Barril, Xavier; Davis, Ben; Drysdale, Martin; Dymock, Brian; Finch, Harry; Fromont, Christophe; Richardson, Christine; Simmonite, Heather; Hubbard, Roderick E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2157-2166</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have designed four generations of a low mol. wt. fragment library for use in NMR-based screening against protein targets.  The library initially contained 723 fragments which were selected manually from the Available Chems. Directory.  A series of in silico filters and property calcns. were developed to automate the selection process, allowing a larger database of 1.79M available compds. to be searched for a further 357 compds. that were added to the library.  A kinase binding pharmacophore was then derived to select 174 kinase-focused fragments.  Finally, an addnl. 61 fragments were selected to increase the no. of different pharmacophores represented within the library.  All of the fragments added to the library passed quality checks to ensure they were suitable for the screening protocol, with appropriate soly., purity, chem. stability, and unambiguous NMR spectrum.  The successive generations of libraries have been characterized through anal. of structural properties (mol. wt., lipophilicity, polar surface area, no. of rotatable bonds, and hydrogen-bonding potential) and by analyzing their pharmacophoric complexity.  These calcns. have been used to compare the fragment libraries with a drug-like ref. set of compds. and a set of mols. that bind to protein active sites.  In addn., an anal. of the overall results of screening the library against the ATP binding site of two protein targets (HSP90 and CDK2) reveals different patterns of fragment binding, demonstrating that the approach can find selective compds. that discriminate between related binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo652Eyyn5Lc7Vg90H21EOLACvtfcHk0lhHH6Ap_ceJ6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotlaru78%253D&md5=7376be270294dae6997609448513c5d7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fci049806z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049806z%26sid%3Dliteratum%253Aachs%26aulast%3DBaurin%26aufirst%3DN.%26aulast%3DAboul-Ela%26aufirst%3DF.%26aulast%3DBarril%26aufirst%3DX.%26aulast%3DDavis%26aufirst%3DB.%26aulast%3DDrysdale%26aufirst%3DM.%26aulast%3DDymock%26aufirst%3DB.%26aulast%3DFinch%26aufirst%3DH.%26aulast%3DFromont%26aufirst%3DC.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DSimmonite%26aufirst%3DH.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DDesign%2520and%2520characterization%2520of%2520libraries%2520of%2520molecular%2520fragments%2520for%2520use%2520in%2520nmr%2520screening%2520against%2520protein%2520targets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2004%26volume%3D44%26spage%3D2157%26epage%3D2166%26doi%3D10.1021%2Fci049806z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. B.</span></span> <span> </span><span class="NLM_article-title">Experiences in fragment-based lead discovery</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>493</i></span>,  <span class="NLM_fpage">509</span>– <span class="NLM_lpage">531</span>, <span class="refDoi"> DOI: 10.1016/b978-0-12-381274-2.00020-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1016%2FB978-0-12-381274-2.00020-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=21371604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslers70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=493&publication_year=2011&pages=509-531&author=R.+E.+Hubbardauthor=J.+B.+Murray&title=Experiences+in+fragment-based+lead+discovery&doi=10.1016%2Fb978-0-12-381274-2.00020-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Experiences in fragment-based lead discovery</span></div><div class="casAuthors">Hubbard, Roderick E.; Murray, James B.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">493</span>
        (<span class="NLM_cas:issue">Fragment-Based Drug Design</span>),
    <span class="NLM_cas:pages">509-531</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  This chapter summarizes the experience at Vernalis over the past decade in developing and applying fragment-based discovery methods across a range of different targets.  The emphasis will be on the practical aspects of the different biophys. techniques (surface plasmon resonance (SPR), differential scanning fluorimetry (DSF), isothermal titrn. calorimetry, NMR, and X-ray crystallog.) that can be used to identify fragments that bind to targets and a discussion of the criteria and strategies for selecting and evolving fragments to lead compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbWFWP7PylG7Vg90H21EOLACvtfcHk0liRa7cMUZojPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslers70%253D&md5=358abcb0648a3482c6be40f5b3d04ff9</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-381274-2.00020-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-381274-2.00020-0%26sid%3Dliteratum%253Aachs%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DMurray%26aufirst%3DJ.%2BB.%26atitle%3DExperiences%2520in%2520fragment-based%2520lead%2520discovery%26jtitle%3DMethods%2520Enzymol.%26date%3D2011%26volume%3D493%26spage%3D509%26epage%3D531%26doi%3D10.1016%2Fb978-0-12-381274-2.00020-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drysdale, M.</span></span> <span> </span><span class="NLM_article-title">The seeds approach: Integrating fragments into drug discovery</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1568</span>– <span class="NLM_lpage">1581</span>, <span class="refDoi"> DOI: 10.2174/156802607782341109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.2174%2F156802607782341109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=17979768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVOrur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=1568-1581&author=R.+Hubbardauthor=B.+Davisauthor=I.+Chenauthor=M.+Drysdale&title=The+seeds+approach%3A+Integrating+fragments+into+drug+discovery&doi=10.2174%2F156802607782341109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The SeeDs approach: integrating fragments into drug discovery</span></div><div class="casAuthors">Hubbard, Roderick E.; Davis, Ben; Chen, Ijen; Drysdale, Martin J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1568-1581</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Finding novel compds. as starting points for optimization is a major challenge in drug discovery research.  Fragment-based methods have emerged in the past ten years as an effective way to sample chem. diversity with a limited no. of low mol. wt. compds.  The structures of the fragments(s) binding to the protein can then be used to design new compds. with increased affinity, specificity and novelty.  This article describes the Vernalis approach to fragment based drug discovery, called SeeDs (Structural exploitation of exptl. Drug startpoints).  The approach includes the design of a fragment library, identification of fragments that bind competitively to a target by ligand-based NMR techniques and protein crystal structures to characterize binding.  Fragments that bind are then evolved to hits, either by growing the fragment or by combining structural features from a no. of compds.  The process is illustrated with examples from recent medicinal chem. programs to discover compds. against the oncol. targets Hsp90 and PDK1.  In addn., we summarize our experience with using mol. docking calcns. to predict fragment binding and anecdotes on the selectivity and binding modes for fragments seen against a range of targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox6z6oFlEi67Vg90H21EOLACvtfcHk0liRa7cMUZojPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVOrur7J&md5=6bfa0109e33c2c23cc4c5a5bcd1ed12f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.2174%2F156802607782341109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802607782341109%26sid%3Dliteratum%253Aachs%26aulast%3DHubbard%26aufirst%3DR.%26aulast%3DDavis%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DI.%26aulast%3DDrysdale%26aufirst%3DM.%26atitle%3DThe%2520seeds%2520approach%253A%2520Integrating%2520fragments%2520into%2520drug%2520discovery%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2007%26volume%3D7%26spage%3D1568%26epage%3D1581%26doi%3D10.2174%2F156802607782341109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alex, A.</span></span> <span> </span><span class="NLM_article-title">Ligand efficiency: A useful metric for lead selection</span>. <i>Drug Discov. Today</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">430</span>– <span class="NLM_lpage">431</span>, <span class="refDoi"> DOI: 10.1016/s1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+A+useful+metric+for+lead+selection&doi=10.1016%2Fs1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2eaSnLAaVWGzkrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520A%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discov.%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2Fs1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">Lessons for fragment library design: Analysis of output from multiple screening campaigns</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">620</span>, <span class="refDoi"> DOI: 10.1007/s10822-009-9280-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1007%2Fs10822-009-9280-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=19495994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD1MXot1Gmtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=603-620&author=I.-J.+Chenauthor=R.+E.+Hubbard&title=Lessons+for+fragment+library+design%3A+Analysis+of+output+from+multiple+screening+campaigns&doi=10.1007%2Fs10822-009-9280-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons for fragment library design: analysis of output from multiple screening campaigns</span></div><div class="casAuthors">Chen, I.-Jen; Hubbard, Roderick E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">603-620</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Over the past 8 years, we have developed, refined and applied a fragment based discovery approach to a range of protein targets.  Here we report computational analyses of various aspects of our fragment library and the results obtained for fragment screening.  We reinforce the finding of others that the exptl. obsd. hit rate for screening fragments can be related to a computationally defined druggability index for the target.  In general, the physicochem. properties of the fragment hits display the same profile as the library, as is expected for a truly diverse library which probes the relevant chem. space.  An anal. of the fragment hits against various protein classes has shown that the physicochem. properties of the fragments are complementary to the properties of the target binding site.  The effectiveness of some fragments appears to be achieved by an appropriate mix of pharmacophore features and enhanced aromaticity, with hydrophobic interactions playing an important role.  The anal. emphasizes that it is possible to identify small fragments that are specific for different binding sites.  To conclude, we discuss how the results could inform further development and improvement of our fragment library.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBo4yzAG3RSbVg90H21EOLACvtfcHk0liC2ns0e1XGCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXot1Gmtbc%253D&md5=f544180cae108c2d7365037388df0262</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1007%2Fs10822-009-9280-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-009-9280-5%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DI.-J.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DLessons%2520for%2520fragment%2520library%2520design%253A%2520Analysis%2520of%2520output%2520from%2520multiple%2520screening%2520campaigns%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2009%26volume%3D23%26spage%3D603%26epage%3D620%26doi%3D10.1007%2Fs10822-009-9280-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hajduk, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huth, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Druggability indices for protein targets derived from nmr-based screening data</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">2518</span>– <span class="NLM_lpage">2525</span>, <span class="refDoi"> DOI: 10.1021/jm049131r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049131r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhslajtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2518-2525&author=P.+J.+Hajdukauthor=J.+R.+Huthauthor=S.+W.+Fesik&title=Druggability+indices+for+protein+targets+derived+from+nmr-based+screening+data&doi=10.1021%2Fjm049131r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Indices for Protein Targets Derived from NMR-Based Screening Data</span></div><div class="casAuthors">Hajduk, Philip J.; Huth, Jeffrey R.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2518-2525</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An anal. of heteronuclear-NMR-based screening data is used to derive relationships between the ability of small mols. to bind to a protein and various parameters that describe the protein binding site.  It is found that a simple model including terms for polar and apolar surface area, surface complexity, and pocket dimensions accurately predicts the exptl. screening hit rates with an R2 of 0.72, an adjusted R2 of 0.65, and a leave-one-out Q2 of 0.56.  Application of the model to predict the druggability of protein targets not used in the training set correctly classified 94% of the proteins for which high-affinity, noncovalent, druglike leads have been reported.  In addn. to understanding the pocket characteristics that contribute to high-affinity binding, the relationships that have been defined allow for quant. comparative analyses of protein binding sites for use in target assessment and validation, virtual ligand screening, and structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0d3V84DZMK7Vg90H21EOLACvtfcHk0liC2ns0e1XGCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhslajtr0%253D&md5=134793e571e8ef1858a734c1a15ce343</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm049131r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049131r%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DHuth%26aufirst%3DJ.%2BR.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDruggability%2520indices%2520for%2520protein%2520targets%2520derived%2520from%2520nmr-based%2520screening%2520data%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2518%26epage%3D2525%26doi%3D10.1021%2Fjm049131r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span> <span> </span><span class="NLM_article-title">Identifying and characterizing binding sites and assessing druggability</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1021/ci800324m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800324m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=377-389&author=T.+A.+Halgren&title=Identifying+and+characterizing+binding+sites+and+assessing+druggability&doi=10.1021%2Fci800324m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying and Characterizing Binding Sites and Assessing Druggability</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">377-389</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Identification and characterization of binding sites is key in the process of structure-based drug design.  In some cases there may not be any information about the binding site for a target of interest.  In other cases, a putative binding site has been identified by computational or exptl. means, but the druggability of the target is not known.  Even when a site for a given target is known, it may be desirable to find addnl. sites whose targeting could produce a desired biol. response.  A new program, called SiteMap, is presented for identifying and analyzing binding sites and for predicting target druggability.  In a large-scale validation, SiteMap correctly identifies the known binding site as the top-ranked site in 86% of the cases, with best results (>98%) coming for sites that bind ligands with subnanomolar affinity.  In addn., a modified version of the score employed for binding-site identification allows SiteMap to accurately classify the druggability of proteins as measured by their ability to bind passively absorbed small mols. tightly.  In characterizing binding sites, SiteMap provides quant. and graphical information that can help guide efforts to critically assess virtual hits in a lead-discovery application or to modify ligand structure to enhance potency or improve phys. properties in a lead-optimization context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfAt0-gvcXDLVg90H21EOLACvtfcHk0liC2ns0e1XGCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D&md5=f03c7b27e30ae03d10e4127ed1f1ed3c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fci800324m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800324m%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DIdentifying%2520and%2520characterizing%2520binding%2520sites%2520and%2520assessing%2520druggability%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D377%26epage%3D389%26doi%3D10.1021%2Fci800324m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ogawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nonaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohnishi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiramatsu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kii, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onogi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibuya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosoya, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagiwara, M.</span></span> <span> </span><span class="NLM_article-title">Development of a novel selective inhibitor of the down syndrome-related kinase dyrk1a</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">86</span>, <span class="refDoi"> DOI: 10.1038/ncomms1090</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1038%2Fncomms1090" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=20981014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A280%3ADC%252BC3cbislehtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=86&author=Y.+Ogawaauthor=Y.+Nonakaauthor=T.+Gotoauthor=E.+Ohnishiauthor=T.+Hiramatsuauthor=I.+Kiiauthor=M.+Yoshidaauthor=T.+Ikuraauthor=H.+Onogiauthor=H.+Shibuyaauthor=T.+Hosoyaauthor=N.+Itoauthor=M.+Hagiwara&title=Development+of+a+novel+selective+inhibitor+of+the+down+syndrome-related+kinase+dyrk1a&doi=10.1038%2Fncomms1090"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a novel selective inhibitor of the Down syndrome-related kinase Dyrk1A</span></div><div class="casAuthors">Ogawa Yasushi; Nonaka Yosuke; Goto Toshiyasu; Ohnishi Eriko; Hiramatsu Toshiyuki; Kii Isao; Yoshida Miyo; Ikura Teikichi; Onogi Hiroshi; Shibuya Hiroshi; Hosoya Takamitsu; Ito Nobutoshi; Hagiwara Masatoshi</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">86</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Dyrk1A (dual-specificity tyrosine-(Y)-phosphorylation-regulated kinase 1A) is a serine/threonine kinase essential for brain development and function, and its excessive activity is considered a pathogenic factor in Down syndrome.  The development of potent, selective inhibitors of Dyrk1A would help to elucidate the molecular mechanisms of normal and diseased brains, and may provide a new lead compound for molecular-targeted drug discovery.  Here, we report a novel Dyrk1A inhibitor, INDY, a benzothiazole derivative showing a potent ATP-competitive inhibitory effect with IC(50) and K(i) values of 0.24 and 0.18 μM, respectively.  X-ray crystallography of the Dyrk1A/INDY complex revealed the binding of INDY in the ATP pocket of the enzyme.  INDY effectively reversed the aberrant tau-phosphorylation and rescued the repressed NFAT (nuclear factor of activated T cell) signalling induced by Dyrk1A overexpression.  Importantly, proINDY, a prodrug of INDY, effectively recovered Xenopus embryos from head malformation induced by Dyrk1A overexpression, resulting in normally developed embryos and demonstrating the utility of proINDY in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTOCqf6rsXbPtenUz0lpNqnfW6udTcc2eY3xb_bc8wkn7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cbislehtw%253D%253D&md5=ac1d5972bec4085de101bb86194babd1</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fncomms1090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms1090%26sid%3Dliteratum%253Aachs%26aulast%3DOgawa%26aufirst%3DY.%26aulast%3DNonaka%26aufirst%3DY.%26aulast%3DGoto%26aufirst%3DT.%26aulast%3DOhnishi%26aufirst%3DE.%26aulast%3DHiramatsu%26aufirst%3DT.%26aulast%3DKii%26aufirst%3DI.%26aulast%3DYoshida%26aufirst%3DM.%26aulast%3DIkura%26aufirst%3DT.%26aulast%3DOnogi%26aufirst%3DH.%26aulast%3DShibuya%26aufirst%3DH.%26aulast%3DHosoya%26aufirst%3DT.%26aulast%3DIto%26aufirst%3DN.%26aulast%3DHagiwara%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520a%2520novel%2520selective%2520inhibitor%2520of%2520the%2520down%2520syndrome-related%2520kinase%2520dyrk1a%26jtitle%3DNat.%2520Commun.%26date%3D2010%26volume%3D1%26spage%3D86%26doi%3D10.1038%2Fncomms1090" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akue-Gedu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debiton, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferandin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meijer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prudhomme, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anizon, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreau, P.</span></span> <span> </span><span class="NLM_article-title">Synthesis and biological activities of aminopyrimidyl-indoles structurally related to meridianins</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">4420</span>– <span class="NLM_lpage">4424</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2009.05.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1016%2Fj.bmc.2009.05.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=19477650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1Gru7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=4420-4424&author=R.+Akue-Geduauthor=E.+Debitonauthor=Y.+Ferandinauthor=L.+Meijerauthor=M.+Prudhommeauthor=F.+Anizonauthor=P.+Moreau&title=Synthesis+and+biological+activities+of+aminopyrimidyl-indoles+structurally+related+to+meridianins&doi=10.1016%2Fj.bmc.2009.05.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and biological activities of aminopyrimidyl-indoles structurally related to meridianins</span></div><div class="casAuthors">Akue-Gedu, Rufine; Debiton, Eric; Ferandin, Yoan; Meijer, Laurent; Prudhomme, Michelle; Anizon, Fabrice; Moreau, Pascale</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4420-4424</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The synthesis of new meridianin derivs. substituted at the C-5 position of the 2-aminopyrimidine ring by various aryl groups and substituted or not by a Me group on the indole nitrogen is described.  These compds. were tested for their kinase inhibitory potencies toward five kinases (CDK5/p25, CK1δ/ε, GSK-3α/β, Dyrk1A and Erk2) as well as their in vitro antiproliferative activities toward a human fibroblast primary culture and two human solid cancer cell lines (MCF-7 and PA 1).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYufDKqEQZNrVg90H21EOLACvtfcHk0ljuHnSGqvB4fQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1Gru7c%253D&md5=f14ea49d5c54c32f5a6be1d9214d4bc7</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2009.05.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2009.05.017%26sid%3Dliteratum%253Aachs%26aulast%3DAkue-Gedu%26aufirst%3DR.%26aulast%3DDebiton%26aufirst%3DE.%26aulast%3DFerandin%26aufirst%3DY.%26aulast%3DMeijer%26aufirst%3DL.%26aulast%3DPrudhomme%26aufirst%3DM.%26aulast%3DAnizon%26aufirst%3DF.%26aulast%3DMoreau%26aufirst%3DP.%26atitle%3DSynthesis%2520and%2520biological%2520activities%2520of%2520aminopyrimidyl-indoles%2520structurally%2520related%2520to%2520meridianins%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2009%26volume%3D17%26spage%3D4420%26epage%3D4424%26doi%3D10.1016%2Fj.bmc.2009.05.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tazarki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeinyeh, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esvan, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schröder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joerger, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khiari, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Josselin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baratte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bach, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruchaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anizon, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giraud, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreau, P.</span></span> <span> </span><span class="NLM_article-title">New pyrido[3,4-g]quinazoline derivatives as clk1 and dyrk1a inhibitors: Synthesis, biological evaluation and binding mode analysis</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>166</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.01.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1016%2Fj.ejmech.2019.01.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=30731399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC1MXis1aqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2019&pages=304-317&author=H.+Tazarkiauthor=W.+Zeinyehauthor=Y.+J.+Esvanauthor=S.+Knappauthor=D.+Chatterjeeauthor=M.+Schr%C3%B6derauthor=A.+C.+Joergerauthor=J.+Khiariauthor=B.+Josselinauthor=B.+Baratteauthor=S.+Bachauthor=S.+Ruchaudauthor=F.+Anizonauthor=F.+Giraudauthor=P.+Moreau&title=New+pyrido%5B3%2C4-g%5Dquinazoline+derivatives+as+clk1+and+dyrk1a+inhibitors%3A+Synthesis%2C+biological+evaluation+and+binding+mode+analysis&doi=10.1016%2Fj.ejmech.2019.01.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">New pyrido[3,4-g]quinazoline derivatives as CLK1 and DYRK1A inhibitors: synthesis, biological evaluation and binding mode analysis</span></div><div class="casAuthors">Tazarki, Helmi; Zeinyeh, Wael; Esvan, Yannick J.; Knapp, Stefan; Chatterjee, Deep; Schroder, Martin; Joerger, Andreas C.; Khiari, Jameleddine; Josselin, Beatrice; Baratte, Blandine; Bach, Stephane; Ruchaud, Sandrine; Anizon, Fabrice; Giraud, Francis; Moreau, Pascale</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">166</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-317</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Synthesis of new pyrido[3,4-g]quinazoline derivs. I [R = (CH2)2-NMe2, (CH2)2-morpholin-1-yl, (CH2)3-NMe2, etc.; R1 = NH2, NO2] and the evaluation of the inhibitory potencies of these compds. toward CDK5, CK1, GSK3, CLK1 and DYRK1A were described.  Introduction of aminoalkylamino groups at the 2-position resulted in several compds. with low nanomolar affinity and selective inhibition of CLK1 and/or DYRK1A.  Their evaluation on several immortalized or cancerous cell lines showed varying degree of cell viability redn.  Co-crystal structures of CLK1 with two of the most potent compds. I [R = Me, R1 = NO2; R = (CH2)3-morpholin-1-yl, R2 = NH2] revealed two alternative binding modes of the pyrido[3,4-g]quinazoline scaffold that can be exploited for future inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK9VnCqI3-v7Vg90H21EOLACvtfcHk0ljuHnSGqvB4fQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXis1aqsLk%253D&md5=d572d06711c1ecdf401051af38655449</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.01.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.01.052%26sid%3Dliteratum%253Aachs%26aulast%3DTazarki%26aufirst%3DH.%26aulast%3DZeinyeh%26aufirst%3DW.%26aulast%3DEsvan%26aufirst%3DY.%2BJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DChatterjee%26aufirst%3DD.%26aulast%3DSchr%25C3%25B6der%26aufirst%3DM.%26aulast%3DJoerger%26aufirst%3DA.%2BC.%26aulast%3DKhiari%26aufirst%3DJ.%26aulast%3DJosselin%26aufirst%3DB.%26aulast%3DBaratte%26aufirst%3DB.%26aulast%3DBach%26aufirst%3DS.%26aulast%3DRuchaud%26aufirst%3DS.%26aulast%3DAnizon%26aufirst%3DF.%26aulast%3DGiraud%26aufirst%3DF.%26aulast%3DMoreau%26aufirst%3DP.%26atitle%3DNew%2520pyrido%255B3%252C4-g%255Dquinazoline%2520derivatives%2520as%2520clk1%2520and%2520dyrk1a%2520inhibitors%253A%2520Synthesis%252C%2520biological%2520evaluation%2520and%2520binding%2520mode%2520analysis%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D166%26spage%3D304%26epage%3D317%26doi%3D10.1016%2Fj.ejmech.2019.01.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tahtouh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soundararajan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durieu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delhommel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oberholzer, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearl, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carreaux, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bazureau, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meijer, L.</span></span> <span> </span><span class="NLM_article-title">Selectivity, cocrystal structures, and neuroprotective properties of leucettines, a family of protein kinase inhibitors derived from the marine sponge alkaloid leucettamine b</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9312</span>– <span class="NLM_lpage">9330</span>, <span class="refDoi"> DOI: 10.1021/jm301034u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301034u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2mu7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9312-9330&author=T.+Tahtouhauthor=J.+M.+Elkinsauthor=P.+Filippakopoulosauthor=M.+Soundararajanauthor=G.+Burgyauthor=E.+Durieuauthor=C.+Cochetauthor=R.+S.+Schmidauthor=D.+C.+Loauthor=F.+Delhommelauthor=A.+E.+Oberholzerauthor=L.+H.+Pearlauthor=F.+Carreauxauthor=J.-P.+Bazureauauthor=S.+Knappauthor=L.+Meijer&title=Selectivity%2C+cocrystal+structures%2C+and+neuroprotective+properties+of+leucettines%2C+a+family+of+protein+kinase+inhibitors+derived+from+the+marine+sponge+alkaloid+leucettamine+b&doi=10.1021%2Fjm301034u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Selectivity, Cocrystal Structures, and Neuroprotective Properties of Leucettines, a Family of Protein Kinase Inhibitors Derived from the Marine Sponge Alkaloid Leucettamine B</span></div><div class="casAuthors">Tahtouh, Tania; Elkins, Jonathan M.; Filippakopoulos, Panagis; Soundararajan, Meera; Burgy, Guillaume; Durieu, Emilie; Cochet, Claude; Schmid, Ralf S.; Lo, Donald C.; Delhommel, Florent; Oberholzer, Anselm E.; Pearl, Laurence H.; Carreaux, Francois; Bazureau, Jean-Pierre; Knapp, Stefan; Meijer, Laurent</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9312-9330</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DYRKs (dual specificity, tyrosine phosphorylation regulated kinases) and CLKs (cdc2-like kinases) are implicated in the onset and development of Alzheimer's disease and Down syndrome.  The marine sponge alkaloid leucettamine B was recently identified as an inhibitor of DYRKs/CLKs.  Synthesis of analogs (leucettines) led to an optimized product, leucettine L41.  Leucettines were cocrystd. with DYRK1A, DYRK2, CLK3, PIM1, and GSK-3β.  The selectivity of L41 was studied by activity and interaction assays of recombinant kinases and affinity chromatog. and competition affinity assays.  These approaches revealed unexpected potential secondary targets such as CK2, SLK, and the lipid kinase PIKfyve/Vac14.  L41 displayed neuroprotective effects on glutamate-induced HT22 cell death.  L41 also reduced amyloid precursor protein-induced cell death in cultured rat brain slices.  The unusual multitarget selectivity of leucettines may account for their neuroprotective effects.  This family of kinase inhibitors deserves further optimization as potential therapeutics against neurodegenerative diseases such as Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOhwr44JiuCrVg90H21EOLACvtfcHk0ljuHnSGqvB4fQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2mu7vF&md5=a1749c347c2e8e0409d8fdd475dc70f9</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm301034u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301034u%26sid%3Dliteratum%253Aachs%26aulast%3DTahtouh%26aufirst%3DT.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DSoundararajan%26aufirst%3DM.%26aulast%3DBurgy%26aufirst%3DG.%26aulast%3DDurieu%26aufirst%3DE.%26aulast%3DCochet%26aufirst%3DC.%26aulast%3DSchmid%26aufirst%3DR.%2BS.%26aulast%3DLo%26aufirst%3DD.%2BC.%26aulast%3DDelhommel%26aufirst%3DF.%26aulast%3DOberholzer%26aufirst%3DA.%2BE.%26aulast%3DPearl%26aufirst%3DL.%2BH.%26aulast%3DCarreaux%26aufirst%3DF.%26aulast%3DBazureau%26aufirst%3DJ.-P.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMeijer%26aufirst%3DL.%26atitle%3DSelectivity%252C%2520cocrystal%2520structures%252C%2520and%2520neuroprotective%2520properties%2520of%2520leucettines%252C%2520a%2520family%2520of%2520protein%2520kinase%2520inhibitors%2520derived%2520from%2520the%2520marine%2520sponge%2520alkaloid%2520leucettamine%2520b%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9312%26epage%3D9330%26doi%3D10.1021%2Fjm301034u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Falke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loaëc, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lozach, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abu Jhaisha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meijer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunick, C.</span></span> <span> </span><span class="NLM_article-title">10-iodo-11h-indolo[3,2-c]quinoline-6-carboxylic acids are selective inhibitors of dyrk1a</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">3131</span>– <span class="NLM_lpage">3143</span>, <span class="refDoi"> DOI: 10.1021/jm501994d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501994d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjs1GhtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=3131-3143&author=H.+Falkeauthor=A.+Chaikuadauthor=A.+Beckerauthor=N.+Loa%C3%ABcauthor=O.+Lozachauthor=S.+Abu+Jhaishaauthor=W.+Beckerauthor=P.+G.+Jonesauthor=L.+Preuauthor=K.+Baumannauthor=S.+Knappauthor=L.+Meijerauthor=C.+Kunick&title=10-iodo-11h-indolo%5B3%2C2-c%5Dquinoline-6-carboxylic+acids+are+selective+inhibitors+of+dyrk1a&doi=10.1021%2Fjm501994d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">10-Iodo-11H-indolo[3,2-c]quinoline-6-carboxylic Acids Are Selective Inhibitors of DYRK1A</span></div><div class="casAuthors">Falke, Hannes; Chaikuad, Apirat; Becker, Anja; Loaec, Nadege; Lozach, Olivier; Abu Jhaisha, Samira; Becker, Walter; Jones, Peter G.; Preu, Lutz; Baumann, Knut; Knapp, Stefan; Meijer, Laurent; Kunick, Conrad</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3131-3143</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The protein kinase DYRK1A has been suggested to act as one of the intracellular regulators contributing to neurol. alterations found in individuals with Down syndrome.  For an assessment of the role of DYRK1A, selective synthetic inhibitors are valuable pharmacol. tools.  However, the DYRK1A inhibitors described in the literature so far either are not sufficiently selective or have not been tested against closely related kinases from the DYRK and the CLK protein kinase families.  The aim of this study was the identification of DYRK1A inhibitors exhibiting selectivity vs. the structurally and functionally closely related DYRK and CLK isoforms.  Structure modification of the screening hit 11H-indolo[3,2-c]quinoline-6-carboxylic acid revealed structure-activity relationships for kinase inhibition and enabled the design of 10-iodo-substituted derivs. as very potent DYRK1A inhibitors with considerable selectivity against CLKs.  X-ray structure detn. of three 11H-indolo[3,2-c]quinoline-6-carboxylic acids cocrystd. with DYRK1A confirmed the predicted binding mode within the ATP binding site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpe4rHbgYJUsbVg90H21EOLACvtfcHk0lhTX3aAWl560Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjs1GhtrY%253D&md5=70efbb6ad590a1240764ccc78af019ec</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm501994d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501994d%26sid%3Dliteratum%253Aachs%26aulast%3DFalke%26aufirst%3DH.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DLoa%25C3%25ABc%26aufirst%3DN.%26aulast%3DLozach%26aufirst%3DO.%26aulast%3DAbu%2BJhaisha%26aufirst%3DS.%26aulast%3DBecker%26aufirst%3DW.%26aulast%3DJones%26aufirst%3DP.%2BG.%26aulast%3DPreu%26aufirst%3DL.%26aulast%3DBaumann%26aufirst%3DK.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMeijer%26aufirst%3DL.%26aulast%3DKunick%26aufirst%3DC.%26atitle%3D10-iodo-11h-indolo%255B3%252C2-c%255Dquinoline-6-carboxylic%2520acids%2520are%2520selective%2520inhibitors%2520of%2520dyrk1a%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D3131%26epage%3D3143%26doi%3D10.1021%2Fjm501994d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Czarna, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zelencova, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kildalsen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seternes, O. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engh, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothweiler, U.</span></span> <span> </span><span class="NLM_article-title">Novel scaffolds for dual specificity tyrosine-phosphorylation-regulated kinase (dyrk1a) inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">7560</span>– <span class="NLM_lpage">7572</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01847</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01847" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVygtrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=7560-7572&author=A.+Czarnaauthor=J.+Wangauthor=D.+Zelencovaauthor=Y.+Liuauthor=X.+Dengauthor=H.+G.+Choiauthor=T.+Zhangauthor=W.+Zhouauthor=J.+W.+Changauthor=H.+Kildalsenauthor=O.+M.+Seternesauthor=N.+S.+Grayauthor=R.+A.+Enghauthor=U.+Rothweiler&title=Novel+scaffolds+for+dual+specificity+tyrosine-phosphorylation-regulated+kinase+%28dyrk1a%29+inhibitors&doi=10.1021%2Facs.jmedchem.7b01847"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Scaffolds for Dual Specificity Tyrosine-Phosphorylation-Regulated Kinase (DYRK1A) Inhibitors</span></div><div class="casAuthors">Czarna, Anna; Wang, Jinhua; Zelencova, Diana; Liu, Yao; Deng, Xianming; Choi, Hwan Geun; Zhang, Tinghu; Zhou, Wenjun; Chang, Jae Won; Kildalsen, Hanne; Seternes, Ole Morten; Gray, Nathanael S.; Engh, Richard A.; Rothweiler, Ulli</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7560-7572</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DYRK1A is one of five members of the dual-specificity tyrosine (Y) phosphorylation-regulated kinase (DYRK) family.  The DYRK1A gene is located in the Down syndrome crit. region and regulates cellular processes related to proliferation and differentiation of neuronal progenitor cells during early development.  This has focused research on its role in neuronal degenerative diseases, including Alzheimer's and Down syndrome.  Recent studies have also shown a possible role of DYRK1A in diabetes.  Here we report a variety of scaffolds not generally known for DYRK1A inhibition, demonstrating their effects in in vitro assays and also in cell cultures.  These inhibitors effectively block the tau phosphorylation that is a hallmark of Alzheimer's disease.  The crystal structures of these inhibitors support the design of optimized and novel therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqC4Uq_4mv-grVg90H21EOLACvtfcHk0lhTX3aAWl560Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVygtrjF&md5=c4b55fc7510b9f9d80726933b32be508</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01847%26sid%3Dliteratum%253Aachs%26aulast%3DCzarna%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZelencova%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DChang%26aufirst%3DJ.%2BW.%26aulast%3DKildalsen%26aufirst%3DH.%26aulast%3DSeternes%26aufirst%3DO.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DEngh%26aufirst%3DR.%2BA.%26aulast%3DRothweiler%26aufirst%3DU.%26atitle%3DNovel%2520scaffolds%2520for%2520dual%2520specificity%2520tyrosine-phosphorylation-regulated%2520kinase%2520%2528dyrk1a%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D7560%26epage%3D7572%26doi%3D10.1021%2Facs.jmedchem.7b01847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fruit, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couly, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhansali, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rammohan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindberg, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crispino, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meijer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besson, T.</span></span> <span> </span><span class="NLM_article-title">Biological characterization of 8-cyclopropyl-2-(pyridin-3-yl)thiazolo[5,4-f]quinazolin-9(8h)-one, a promising inhibitor of dyrk1a</span>. <i>Pharmaceuticals</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">185</span>, <span class="refDoi"> DOI: 10.3390/ph12040185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.3390%2Fph12040185" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslalt7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2019&pages=185&author=C.+Fruitauthor=F.+Coulyauthor=R.+Bhansaliauthor=M.+Rammohanauthor=M.+F.+Lindbergauthor=J.+D.+Crispinoauthor=L.+Meijerauthor=T.+Besson&title=Biological+characterization+of+8-cyclopropyl-2-%28pyridin-3-yl%29thiazolo%5B5%2C4-f%5Dquinazolin-9%288h%29-one%2C+a+promising+inhibitor+of+dyrk1a&doi=10.3390%2Fph12040185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Biological characterization of 8-cyclopropyl-2-(pyridin-3-yl)thiazolo[5,4-f]quinazolin-9(8H)-one, a promising inhibitor of DYRK1A</span></div><div class="casAuthors">Fruit, Corinne; Couly, Florence; Bhansali, Rahul; Rammohan, Malini; Lindberg, Mattias F.; Crispino, John D.; Meijer, Laurent; Besson, Thierry</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceuticals</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">185</span>CODEN:
                <span class="NLM_cas:coden">PHARH2</span>;
        ISSN:<span class="NLM_cas:issn">1424-8247</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Dual-specificity tyrosine phosphorylation-regulated kinases (DYRKs) hyperactivity has been linked to the development of a no. of human malignancies.  DYRK1A is the most studied family member, and the discovery of novel specific inhibitors is attracting considerable interest.  The 8-cyclopropyl-2(pyridin-3-yl)thiazolo[5,4-f]quinazolin-9(8H)-one (also called FC162) was found to be a promising inhibitor of DYRK1A and was characterized in biol. expts., by western transfer and flow cytometry on SH-SY5Y and pre-B cells.  Here, the results obtained with FC162 are compared to well-characterized known DYRK1A inhibitors (e.g., Leucettine L41 and EHT1610).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMCVXeZbYtAbVg90H21EOLACvtfcHk0lgeReNN7xNg3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslalt7%252FK&md5=49a63babbde0c887e44f48df3c167beb</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.3390%2Fph12040185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fph12040185%26sid%3Dliteratum%253Aachs%26aulast%3DFruit%26aufirst%3DC.%26aulast%3DCouly%26aufirst%3DF.%26aulast%3DBhansali%26aufirst%3DR.%26aulast%3DRammohan%26aufirst%3DM.%26aulast%3DLindberg%26aufirst%3DM.%2BF.%26aulast%3DCrispino%26aufirst%3DJ.%2BD.%26aulast%3DMeijer%26aufirst%3DL.%26aulast%3DBesson%26aufirst%3DT.%26atitle%3DBiological%2520characterization%2520of%25208-cyclopropyl-2-%2528pyridin-3-yl%2529thiazolo%255B5%252C4-f%255Dquinazolin-9%25288h%2529-one%252C%2520a%2520promising%2520inhibitor%2520of%2520dyrk1a%26jtitle%3DPharmaceuticals%26date%3D2019%26volume%3D12%26spage%3D185%26doi%3D10.3390%2Fph12040185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rothweiler, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stensen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandsdal, B. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isaksson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engh, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svendsen, J. S. M.</span></span> <span> </span><span class="NLM_article-title">Probing the atp-binding pocket of protein kinase dyrk1a with benzothiazole fragment molecules</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9814</span>– <span class="NLM_lpage">9824</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01086</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01086" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1yksr7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9814-9824&author=U.+Rothweilerauthor=W.+Stensenauthor=B.+O.+Brandsdalauthor=J.+Isakssonauthor=F.+A.+Leesonauthor=R.+A.+Enghauthor=J.+S.+M.+Svendsen&title=Probing+the+atp-binding+pocket+of+protein+kinase+dyrk1a+with+benzothiazole+fragment+molecules&doi=10.1021%2Facs.jmedchem.6b01086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the ATP-Binding Pocket of Protein Kinase DYRK1A with Benzothiazole Fragment Molecules</span></div><div class="casAuthors">Rothweiler, Ulli; Stensen, Wenche; Brandsdal, Bjoern Olav; Isaksson, Johan; Leeson, Frederick Alan; Engh, Richard Alan; Svendsen, John S. Mjoeen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9814-9824</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DYRK1A has emerged as a potential target for therapies of Alzheimer's disease using small mols.  Based on the observation of selective DYRK1A inhibition by firefly D-luciferin, we have explored static and dynamic structural properties of fragment sized variants of the benzothiazole scaffold with respect to DYRK1A using X-ray crystallog. and NMR techniques.  The compds. have excellent ligand efficiencies and show a remarkable diversity of binding modes in dynamic equil.  Binding geometries are detd. in part by interactions often considered "weak", including "orthogonal multipolar" types represented by e.g. F-CO, sulfur-aromat, and halogen-aromat interactions, together with hydrogen bonds that are modulated by variation of electron withdrawing groups.  These studies show how the benzothiazole scaffold is highly promising for the development of therapeutic DYRK1A inhibitors.  In addn., the subtleties of the binding interactions, including dynamics, show how full structural studies are required to fully interpret the essential phys. determinants of binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYSQe1_5bEjLVg90H21EOLACvtfcHk0lgeReNN7xNg3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1yksr7P&md5=edb850b232f53b3e161a5023915972e6</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01086%26sid%3Dliteratum%253Aachs%26aulast%3DRothweiler%26aufirst%3DU.%26aulast%3DStensen%26aufirst%3DW.%26aulast%3DBrandsdal%26aufirst%3DB.%2BO.%26aulast%3DIsaksson%26aufirst%3DJ.%26aulast%3DLeeson%26aufirst%3DF.%2BA.%26aulast%3DEngh%26aufirst%3DR.%2BA.%26aulast%3DSvendsen%26aufirst%3DJ.%2BS.%2BM.%26atitle%3DProbing%2520the%2520atp-binding%2520pocket%2520of%2520protein%2520kinase%2520dyrk1a%2520with%2520benzothiazole%2520fragment%2520molecules%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9814%26epage%3D9824%26doi%3D10.1021%2Facs.jmedchem.6b01086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span>; <span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Denzo and scalepack</span>. In  <i>International tables for crystallography volume f: Crystallography of biological macromolecules</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossmann, M. G.</span>, <span class="NLM_string-name">Arnold, E.</span></span>, Eds.; <span class="NLM_publisher-name">Springer Netherlands</span>: <span class="NLM_publisher-loc">Dordrecht</span>, <span class="NLM_year">2001</span>; pp  <span class="NLM_fpage">226</span>– <span class="NLM_lpage">235</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2001&pages=226-235&author=Z.+Otwinowski&author=W.+Minorauthor=M.+G.+Rossmann&author=E.+Arnold&title=International+tables+for+crystallography+volume+f%3A+Crystallography+of+biological+macromolecules"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26atitle%3DDenzo%2520and%2520scalepack%26btitle%3DInternational%2520tables%2520for%2520crystallography%2520volume%2520f%253A%2520Crystallography%2520of%2520biological%2520macromolecules%26aulast%3DRossmann%26aufirst%3DM.%2BG.%26pub%3DSpringer%2520Netherlands%26date%3D2001%26spage%3D226%26epage%3D235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Xds</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lgeReNN7xNg3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">Molecular replacement withMOLREP</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1107/s0907444909042589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1107%2FS0907444909042589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=20057045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=22-25&author=A.+Vaginauthor=A.+Teplyakov&title=Molecular+replacement+withMOLREP&doi=10.1107%2Fs0907444909042589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular replacement with MOLREP</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-25</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement that utilizes a no. of original approaches to rotational and translational search and data prepn.  Since the first publication describing the program, MOLREP has acquired a variety of features that include weighting of the X-ray data and search models, multi-copy search, fitting the model into electron d., structural superposition of two models and rigid-body refinement.  The program can run in a fully automatic mode using optimized parameters calcd. from the input data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzj5SzYvyu_7Vg90H21EOLACvtfcHk0lgeReNN7xNg3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktw%253D%253D&md5=820d114719aca209994ffb0403e3b20d</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042589%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMolecular%2520replacement%2520withMOLREP%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D22%26epage%3D25%26doi%3D10.1107%2Fs0907444909042589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/s0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2Fs0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lh7h3b6UVB6cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2Fs0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of theCCP4 suite and current developments</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/s0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+theCCP4+suite+and+current+developments&doi=10.1107%2Fs0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lh7h3b6UVB6cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520theCCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2Fs0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/s0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2Fs0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lh7h3b6UVB6cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2Fs0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schüttelkopf, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Aalten, D. M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prodrg, D. M. F.</span></span> <span> </span><span class="NLM_article-title">PRODRG: a tool for high-throughput crystallography of protein-ligand complexes</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1355</span>– <span class="NLM_lpage">1363</span>, <span class="refDoi"> DOI: 10.1107/s0907444904011679</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1107%2FS0907444904011679" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=15272157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvVegtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=1355-1363&author=A.+W.+Sch%C3%BCttelkopfauthor=D.+M.+F.+van+Aaltenauthor=D.+M.+F.+Prodrg&title=PRODRG%3A+a+tool+for+high-throughput+crystallography+of+protein-ligand+complexes&doi=10.1107%2Fs0907444904011679"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">PRODRG: a tool for high-throughput crystallography of protein-ligand complexes</span></div><div class="casAuthors">Schuettelkopf, Alexander W.; van Aalten, Daan M. F.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1355-1363</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The small-mol. topol. generator PRODRG is described, which takes input from existing coordinates or various two-dimensional formats and automatically generates coordinates and mol. topologies suitable for x-ray refinement of protein-ligand complexes.  Test results are described for automatic generation of topologies followed by energy minimization for a subset of compds. from the Cambridge Structural Database, which shows that, within the limits of the empirical GROMOS87 force field used, structures with good geometries are generated.  X-ray refinement in X-PLOR/CNS, REFMAC and SHELX using PRODRG-generated topologies produces results comparable to refinement with topologies from the std. libraries.  However, tests with distorted starting coordinates show that PRODRG topologies perform better, both in terms of ligand geometry and of crystallog. R factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbflrch0bs97Vg90H21EOLACvtfcHk0lhTyx7Txf54NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvVegtbY%253D&md5=f9bfdb14be7b04ea5c0db67feb39f72a</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444904011679&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904011679%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25BCttelkopf%26aufirst%3DA.%2BW.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%2BM.%2BF.%26aulast%3DProdrg%26aufirst%3DD.%2BM.%2BF.%26atitle%3DPRODRG%253A%2520a%2520tool%2520for%2520high-throughput%2520crystallography%2520of%2520protein-ligand%2520complexes%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D1355%26epage%3D1363%26doi%3D10.1107%2Fs0907444904011679" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0025.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (a) Ribbon structure of Dyrk1a, highlighting hinge (red) and ADPNP bound in the stick (monomer B in the crystal structure, PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A4O">7A4O</a>) and the portion of the structure (light blue) removed in all subsequent figures (residues E160 to D178 which include the glycine loop). (b) Closeup of the ATP binding site showing side by side ADPNP bound to Dyrk1a (main chain white) and PAK4 (main chain pink) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FIG">4FIG</a>). (c) Standard orientation and convention (unless stated) used for figures of compounds (element coloring with purple carbons) binding to Dyrk1a—remove the glycine loop from E160 to D178 to view compound binding, peptide backbone as a ribbon structure for the remainder of the protein except for Dyrk1a F238 to N244 (hinge region) and V306 to F308 with additional selected side chains and water molecules (none in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) colored on the element in stick (gray carbons) and protein backbone and side chain hydrogen bonds as blue and green dashed lines with nonstandard interactions in orange dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Detailed comparison of the binding mode of literature compounds and selected fragments to Dyrk1a showing the protein structure (white ribbon), selected waters, and key interactions with the fragment (a) harmine (<b>9</b>—green carbons, PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ANR">3ANR</a>) and fragment <b>3</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A4W">7A4W</a>); (b) meriolin-3 (<b>10</b>—green carbons, PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A5B">7A5B</a>) and fragment <b>1</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A4R">7A4R</a>); and (c) fragment <b>5</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A51">7A51</a>). See the legend to <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> for conventions used in the representation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structure of <b>16</b> bound to Dyrk1a (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A5D">7A5D</a>). See the legend to <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> for conventions used in the representation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structure of <b>24</b> bound to Dyrk1a (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A5L">7A5L</a>). See the legend to <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> for conventions used in the representation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structure of <b>34</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A5N">7A5N</a>) bound to Dyrk1a. (a) See the legend in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> for conventions used in the representation. (b). Alternate view to show hydrophobic interactions with underside of the glycine loop.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compound <b>34</b> inhibits the growth of U87MG glioblastoma xenografts. Nude mice with established U87MG subcutaneous tumor xenografts were dosed with either 100 or 200 mg/kg PO <b>34</b> daily for 21 days (qdx21). Tumor volume (<b>A</b>) was determined by twice weekly caliper measurement, and body weight (<b>B</b>) was recorded.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. <b>34</b> and representatives of different classes of DYRK1A inhibitors drawn in approximately the relative orientation seen in the available crystal structures for binding to Dyrk1a. Shading highlights equivalent features: green—equivalent to pyrollopyrimidine hinge binding; gray—equivalent to the methyl of <b>34</b>, binding in the small hydrophobic pocket adjacent to carbonyl of L241; yellow—equivalent pyrimidine; blue–equivalent interactions to the hydrophobic pocket; and pink—interaction with K188. See the text for references.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of Compound <b>16</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 33% HBr in acetic acid, acetone, −10 to −5 °C, 3 h; (b) triethyl orthoacetate, MeCN, reflux, yield 63%; (c) 7 N NH<sub>3</sub> in MeOH, sealed vessel, 105 °C, 6 h, yield 59%; (d) NaH, SEM-Cl, DMF, yield 57%, yield 48%; (e) pyridine-3-boronic acid, K<sub>2</sub>CO<sub>3</sub>, THF, water [1,1_-Bis(diphenylphosphino)ferrocene]dichloropalladium(II) (10 mol %), 130 °C; and (e) ethylenediamine, TBAF (1 M in THF), 120 °C microwave, 30 min, yield 38%.</p></p></figure><figure data-id="sch2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>19</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) SEM-Cl, NaH, DMF, 0 °C, 2 h, yield quant; (b) NBS, DMF, rt, yield 72%; (c) NaH, MeOH, 0 °C, yield quant; (d) (2-aminopyridin-4-yl)boronic acid, THF, water, K<sub>2</sub>CO<sub>3</sub>, cat [1,1′-Bis(diphenylphosphino)ferrocene]dichloropalladium(II), 1 h microwave, 140 °C, yield 32%; and (e) TBAF (1 M in THF), ethylenediamine, THF, yield 43%.</p></p></figure><figure data-id="sch3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <b>34</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 2,6-difluorobenzylamine, TEA, EtOH, 80 °C, 16 h, yield 87%; (b) 2-aminopyridine-4-boronic acid pinacol ester, K<sub>2</sub>CO<sub>3</sub>, Pd-118 (cat), THF, water, 60 °C, 16 h, yield: 76%; and (c) TBAF (1 M in THF) ethylenediamine, THF, 80 °C, 14 h, yield 59%.</p></p></figure><figure data-id="sch4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of <b>35</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) benzene sulfonyl chloride, NaH, DMF, 0 °C, 16 h, yield 91%; (b) 2,6-difluorobenzylamine, TEA, ethanol, 80 °C, 16 h, yield 93%; (c) bis(pinacolato)diboron, KOAc, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, THF, 140 °C, 1 h, yield 61%; (d) 2-amino-4-chloropyrimidine, K<sub>2</sub>CO<sub>3</sub>, Pd118, THF, 90 °C, 2 h, yield 64%; and (e) K<sub>2</sub>CO<sub>3</sub>, methanol, RT, 16 h, yield 40%.</p></p></figure><figure data-id="sch5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/medium/jm1c00024_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0013.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of <b>37</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/acs.jmedchem.1c00024/20210629/images/large/jm1c00024_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00024&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 3,4-(methylenedioxy)-phenyl boronic acid, Cs<sub>2</sub>CO<sub>3</sub>, palladium(dppf)Cl<sub>2</sub>, THF, water, 130 °C, 45 min, yield quant; (b) NBS, DMF 0 °C, 2 h, yield 54%; (c) boc-anhydride, DMAP, Et<sub>3</sub>N; (d), boc-2-aminopyridine-4-boronic acid pinacol ester, K<sub>2</sub>CO<sub>3</sub>, Pd118, THF, water, 65 °C, 16 h; and (e) HCl in MeOH, microwave 90 °C, 1 h, yield 27% (over two steps).</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i98">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08248" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08248" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 55 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altafaj, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aranda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de la Luna, S.</span></span> <span> </span><span class="NLM_article-title">Dyrk1a autophosphorylation on serine residue 520 modulates its kinase activity via 14-3-3 binding</span>. <i>Mol. Biol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1167</span>– <span class="NLM_lpage">1178</span>, <span class="refDoi"> DOI: 10.1091/mbc.e06-08-0668</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1091%2Fmbc.e06-08-0668" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=17229891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2sXktVSmtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2007&pages=1167-1178&author=M.+Alvarezauthor=X.+Altafajauthor=S.+Arandaauthor=S.+de+la+Luna&title=Dyrk1a+autophosphorylation+on+serine+residue+520+modulates+its+kinase+activity+via+14-3-3+binding&doi=10.1091%2Fmbc.e06-08-0668"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">DYRK1A autophosphorylation on serine residue 520 modulates its kinase activity via 14-3-3 binding</span></div><div class="casAuthors">Alvarez, Monica; Altafaj, Xavier; Aranda, Sergi; de la Luna, Susana</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology of the Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1167-1178</span>CODEN:
                <span class="NLM_cas:coden">MBCEEV</span>;
        ISSN:<span class="NLM_cas:issn">1059-1524</span>.
    
            (<span class="NLM_cas:orgname">American Society for Cell Biology</span>)
        </div><div class="casAbstract">Dual-specificity tyrosine-phosphorylated and regulated kinase (DYRK) proteins are an evolutionarily conserved family of protein kinases, with members identified from yeast to humans, that participate in a variety of cellular processes.  DYRKs are serine/threonine protein kinases that are activated by autophosphorylation on a tyrosine residue in the activation loop.  The family member DYRK1A has been shown to phosphorylate several cytosolic proteins and a no. of splicing and transcription factors, including members of the nuclear factor of activated T cells family.  In the present study, we show that DYRK1A autophosphorylates, via an intramol. mechanism, on Ser-520, in the PEST domain of the protein.  We also show that phosphorylation of this residue, which we show is subjected to dynamic changes in vivo, mediates the interaction of DYRK1A with 14-3-3β.  A second 14-3-3 binding site is present within the N-terminal of the protein.  In the context of the DYRK1A mol., neither site can act independently of the other.  Bacterially produced DYRK1A and the mutant DYRK1A/S520A have similar kinase activities, suggesting that Ser-520 phosphorylation does not affect the intrinsic kinase activity on its own.  Instead, we demonstrate that this phosphorylation allows the binding of 14-3-3β, which in turn stimulates the catalytic activity of DYRK1A.  These findings provide evidence for a novel mechanism for the regulation of DYRK1A kinase activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3iiUoSe8NbbVg90H21EOLACvtfcHk0lgu7f7Sg2CsnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXktVSmtrc%253D&md5=7ed3a1f45bc539f97aaeea6972f9e78c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1091%2Fmbc.e06-08-0668&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1091%252Fmbc.e06-08-0668%26sid%3Dliteratum%253Aachs%26aulast%3DAlvarez%26aufirst%3DM.%26aulast%3DAltafaj%26aufirst%3DX.%26aulast%3DAranda%26aufirst%3DS.%26aulast%3Dde%2Bla%2BLuna%26aufirst%3DS.%26atitle%3DDyrk1a%2520autophosphorylation%2520on%2520serine%2520residue%2520520%2520modulates%2520its%2520kinase%2520activity%2520via%252014-3-3%2520binding%26jtitle%3DMol.%2520Biol.%2520Cell%26date%3D2007%26volume%3D18%26spage%3D1167%26epage%3D1178%26doi%3D10.1091%2Fmbc.e06-08-0668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mercer, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewton, D. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naqvi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, E.</span></span> <span> </span><span class="NLM_article-title">Mirk/dyrk1b mediates cell survival in rhabdomyosarcomas</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">5143</span>– <span class="NLM_lpage">5150</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-05-1539</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1158%2F0008-5472.CAN-05-1539" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=16707437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD28XksF2isrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=5143-5150&author=S.+E.+Mercerauthor=D.+Z.+Ewtonauthor=S.+Shahauthor=A.+Naqviauthor=E.+Friedman&title=Mirk%2Fdyrk1b+mediates+cell+survival+in+rhabdomyosarcomas&doi=10.1158%2F0008-5472.can-05-1539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Mirk/Dyrk1b Mediates Cell Survival in rhabdomyosarcomas</span></div><div class="casAuthors">Mercer, Stephen E.; Ewton, Daina Z.; Shah, Sejal; Naqvi, Asghar; Friedman, Eileen</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5143-5150</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Rhabdomyosarcoma is the most common sarcoma in children and is difficult to treat if the primary tumor is non-resectable or if the disease presents with metastases.  The function of the serine/threonine kinase Mirk was investigated in this cancer.  Mirk has both growth arrest and survival functions in terminally differentiating skeletal myoblasts.  Maintenance of Mirk growth arrest properties would cause down-regulation of Mirk in transformed myoblasts.  Alternatively, Mirk expression would be retained if rhabdomyosarcoma cells used Mirk survival capability.  Mirk expression was significant in 12 of 16 clin. cases of rhabdomyosarcoma.  Mirk was detected in each rhabdomyosarcoma cell line examd.  Mirk was a functional kinase in each of three rhabdomyosarcoma cell lines, where it proved to be more active than in C2C12 skeletal myoblasts.  Mirk mediated survival of the majority of clonogenic rhabdomyosarcoma cells.  Knockdown of Mirk by RNA interference reduced the fraction of RD and of Rh30 rhabdomyosarcoma cells capable of colony formation 3- to 4-fold in multiple expts.  Depletion of Mirk induced cell death by apoptosis, as shown by increased nos. of terminal deoxynucleotidyl transferase-mediated nick-end labeling-pos. cells and by increased binding of Annexin V.  Mirk is a stress-activated kinase that mediates expression of contractile proteins in differentiating myoblasts, but Mirk is not essential for muscle formation in the embryo.  It is likely that Mirk also facilitates survival of satellite cell-derived rhabdomyoblasts in regenerating skeletal muscle and aids their differentiation.  This survival function is maintained in rhabdomyosarcoma, where Mirk may be a novel therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXnJZi1ESdPLVg90H21EOLACvtfcHk0lgu7f7Sg2CsnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XksF2isrY%253D&md5=adf0683dd6a03347c2574d66f4b53451</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-05-1539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-05-1539%26sid%3Dliteratum%253Aachs%26aulast%3DMercer%26aufirst%3DS.%2BE.%26aulast%3DEwton%26aufirst%3DD.%2BZ.%26aulast%3DShah%26aufirst%3DS.%26aulast%3DNaqvi%26aufirst%3DA.%26aulast%3DFriedman%26aufirst%3DE.%26atitle%3DMirk%252Fdyrk1b%2520mediates%2520cell%2520survival%2520in%2520rhabdomyosarcomas%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D5143%26epage%3D5150%26doi%3D10.1158%2F0008-5472.can-05-1539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstein, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hornicek, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mankin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z.</span></span> <span> </span><span class="NLM_article-title">The kinase mirk is a potential therapeutic target in osteosarcoma</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">552</span>– <span class="NLM_lpage">558</span>, <span class="refDoi"> DOI: 10.1093/carcin/bgp330</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1093%2Fcarcin%2Fbgp330" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=20042639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkt1OlsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=552-558&author=C.+Yangauthor=D.+Jiauthor=E.+J.+Weinsteinauthor=E.+Choyauthor=F.+J.+Hornicekauthor=K.+B.+Woodauthor=X.+Liuauthor=H.+Mankinauthor=Z.+Duan&title=The+kinase+mirk+is+a+potential+therapeutic+target+in+osteosarcoma&doi=10.1093%2Fcarcin%2Fbgp330"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The kinase Mirk is a potential therapeutic target in osteosarcoma</span></div><div class="casAuthors">Yang, Cao; Ji, Diana; Weinstein, Edward J.; Choy, Edwin; Hornicek, Francis J.; Wood, Kirkham B.; Liu, Xianzhe; Mankin, Henry; Duan, Zhenfeng</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">552-558</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Osteosarcoma is the most common primary malignant bone tumor affecting children and adolescents.  The majority of patients are treated by surgery and chemotherapy but have limited alternative therapeutic options.  Kinases play an important role in the growth and survival of tumor cells.  We aim to identify specific kinases to be vital in the survival of osteosarcoma cells and thus may be a key target in creating novel anticancer therapies.  A lentiviral short hairpin RNA kinase library, screened osteosarcoma cells, identified kinase minibrain-related kinase (Mirk) (Dyrk1B) as a potential target.  Knockdown Mirk expression could inhibit cell growth and induce apoptosis.  Chem. synthetic small interfering RNA knockdown and complementary DNA rescue assay further confirmed the results from the decrease of Mirk gene expression.  The relationship between Mirk gene expression and the clin. characteristics of patients with osteosarcoma was investigated using tissue microarray and immunohistochem. anal.  The data indicate that the overall survival rate of patients with Mirk high staining (high levels of Mirk protein expression) is significantly shorter than those with Mirk low staining and moderate staining.  This highlights Mirk's potential to serve as a promising target for mol. therapy in the treatment of osteosarcoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtj9Sf-jca27Vg90H21EOLACvtfcHk0ljet9ixAaxxEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkt1OlsL0%253D&md5=7c933da4b2b727f81e141036f2b26acf</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp330%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.%26aulast%3DJi%26aufirst%3DD.%26aulast%3DWeinstein%26aufirst%3DE.%2BJ.%26aulast%3DChoy%26aufirst%3DE.%26aulast%3DHornicek%26aufirst%3DF.%2BJ.%26aulast%3DWood%26aufirst%3DK.%2BB.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DMankin%26aufirst%3DH.%26aulast%3DDuan%26aufirst%3DZ.%26atitle%3DThe%2520kinase%2520mirk%2520is%2520a%2520potential%2520therapeutic%2520target%2520in%2520osteosarcoma%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26spage%3D552%26epage%3D558%26doi%3D10.1093%2Fcarcin%2Fbgp330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schell, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bepler, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haura, E. B.</span></span> <span> </span><span class="NLM_article-title">Mirk/dyrk1b, a novel therapeutic target, mediates cell survival in non-small cell lung cancer cells</span>. <i>Canc. Biol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1671</span>– <span class="NLM_lpage">1679</span>, <span class="refDoi"> DOI: 10.4161/cbt.8.17.9322</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.4161%2Fcbt.8.17.9322" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=19633423" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlslCiu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=1671-1679&author=J.+Gaoauthor=Z.+Zhengauthor=B.+Rawalauthor=M.+J.+Schellauthor=G.+Beplerauthor=E.+B.+Haura&title=Mirk%2Fdyrk1b%2C+a+novel+therapeutic+target%2C+mediates+cell+survival+in+non-small+cell+lung+cancer+cells&doi=10.4161%2Fcbt.8.17.9322"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Mirk/Dyrk1B, a novel therapeutic target, mediates cell survival in non-small cell lung cancer cells</span></div><div class="casAuthors">Gao, Jingchun; Zheng, Zhong; Rawal, Bhupendra; Schell, Michael J.; Bepler, Gerold; Haura, Eric B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Biology & Therapy</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">1671-1679</span>CODEN:
                <span class="NLM_cas:coden">CBTAAO</span>;
        ISSN:<span class="NLM_cas:issn">1538-4047</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Minibrain-related kinase (Mirk) is a member of the dual specificity tyrosine-phosphorylation-regulated kinase (Dyrk)/minibrain family of dual-specificity protein kinases and is identical to Dyrk1B.  Mirk/Dyrk1B is a serine/threonine kinase that has been found to be upregulated in solid tumors and mediates cell survival in colon cancer, pancreatic ductal adenocarcinoma and rhabdomyosarcomas.  There is little known about Mirk in lung cancer.  In the present study, we showed that Mirk protein was widely overexpressed in 13 of 19 NSCLC cell lines.  Mirk immunopptn. coupled with anti-phosphotyrosine western blotting confirmed tyrosine phosphorylation of Mirk in NSCLC cells.  Mirk knockdown by small interfering RNA induced cell apoptosis concomitant with upregulation of Bak, a Bcl-2 family member, and downregulation of signal transducers and activators of transcription 3 (STAT3) tyrosine phosphorylation.  Mirk knockdown led to decreased cell colony formation in vitro as well as delayed tumor growth in an orthotopic mouse model and sensitized cells to cisplatin-induced apoptosis.  Using automated quant. detn. of the Mirk protein in tumor specimens of patients with early-stage lung cancer, overexpression of Mirk was found in nearly 90% of tumor specimens in both the cytoplasm and nucleus.  These results suggest that Mirk is overexpressed in lung cancer, acts as a survival factor in lung cancer cells and may be a novel therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1x545JhuAN7Vg90H21EOLACvtfcHk0ljet9ixAaxxEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlslCiu7s%253D&md5=ff77dbf7fd354ccdd3723a86c3cd79f2</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.4161%2Fcbt.8.17.9322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcbt.8.17.9322%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DZ.%26aulast%3DRawal%26aufirst%3DB.%26aulast%3DSchell%26aufirst%3DM.%2BJ.%26aulast%3DBepler%26aufirst%3DG.%26aulast%3DHaura%26aufirst%3DE.%2BB.%26atitle%3DMirk%252Fdyrk1b%252C%2520a%2520novel%2520therapeutic%2520target%252C%2520mediates%2520cell%2520survival%2520in%2520non-small%2520cell%2520lung%2520cancer%2520cells%26jtitle%3DCanc.%2520Biol.%2520Ther.%26date%3D2009%26volume%3D8%26spage%3D1671%26epage%3D1679%26doi%3D10.4161%2Fcbt.8.17.9322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewton, D. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naqvi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercer, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, E.</span></span> <span> </span><span class="NLM_article-title">The kinase mirk/dyrk1b mediates cell survival in pancreatic ductal adenocarcinoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">4149</span>– <span class="NLM_lpage">4158</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-05-3089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1158%2F0008-5472.CAN-05-3089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=16618736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsFSlt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=4149-4158&author=X.+Dengauthor=D.+Z.+Ewtonauthor=S.+Liauthor=A.+Naqviauthor=S.+E.+Mercerauthor=S.+Landasauthor=E.+Friedman&title=The+kinase+mirk%2Fdyrk1b+mediates+cell+survival+in+pancreatic+ductal+adenocarcinoma&doi=10.1158%2F0008-5472.can-05-3089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The kinase Mirk/Dyrk1B mediates cell survival in pancreatic ductal adenocarcinoma</span></div><div class="casAuthors">Deng, Xiaobing; Ewton, Daina Z.; Li, Sheena; Naqvi, Asghar; Mercer, Stephen E.; Landas, Steve; Friedman, Eileen</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4149-4158</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Ductal adenocarcinoma of the pancreas is almost uniformly lethal as this cancer is invariably detected at an advanced stage and is resistant to treatment.  The serine/threonine kinase Mirk/Dyrk1B has been shown to be antiapoptotic in rhabdomyosarcomas.  We have now investigated whether Mirk might mediate survival in another cancer in which Mirk is widely expressed, pancreatic ductal adenocarcinoma.  Mirk was an active kinase in each pancreatic cancer cell line where it was detected.  Mirk knockdown by RNA interference (RNAi) reduced the clonogenicity of Panc1 pancreatic cancer cells 4-fold and decreased tumor cell no., showing that Mirk mediates survival in these cells.  Mirk knockdown by synthetic duplex RNAis in Panc1, AsPc1, and SU86.86 pancreatic cancer cells induced apoptosis and enhanced the apoptosis induced by gemcitibine.  Mirk knockdown did not increase the abundance or activation of Akt.  However, four of five pancreatic carcinoma cell lines exhibited either elevated Mirk activity or elevated Akt activity, suggesting that pancreatic cancer cells primarily rely on Mirk or Akt for survival signaling.  Mirk protein was detected by immunohistochem. in 25 of 28 cases (89%) of pancreatic ductal adenocarcinoma, with elevated expression in 11 cases (39%).  Increased expression of Mirk was seen in pancreatic carcinomas compared with primary cultures of normal ductal epithelium by serial anal. of gene expression and by immunohistochem.  Thus, Mirk is a survival factor for pancreatic ductal adenocarcinoma.  Because knockout of Mirk does not cause embryonic lethality, Mirk is not essential for normal cell growth and may represent a novel therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTvhrZzfM4ebVg90H21EOLACvtfcHk0ljK6-mhWuihsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsFSlt7w%253D&md5=317c3012cae9140c4c56d7eef07ab6dc</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-05-3089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-05-3089%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DEwton%26aufirst%3DD.%2BZ.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DNaqvi%26aufirst%3DA.%26aulast%3DMercer%26aufirst%3DS.%2BE.%26aulast%3DLandas%26aufirst%3DS.%26aulast%3DFriedman%26aufirst%3DE.%26atitle%3DThe%2520kinase%2520mirk%252Fdyrk1b%2520mediates%2520cell%2520survival%2520in%2520pancreatic%2520ductal%2520adenocarcinoma%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D4149%26epage%3D4158%26doi%3D10.1158%2F0008-5472.can-05-3089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, E. A.</span></span> <span> </span><span class="NLM_article-title">Ovarian cancer cells, not normal cells, are damaged by mirk/dyrk1b kinase inhibition</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">2258</span>– <span class="NLM_lpage">2269</span>, <span class="refDoi"> DOI: 10.1002/ijc.27917</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1002%2Fijc.27917" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=23114871" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs12rsbfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2013&pages=2258-2269&author=J.+Huauthor=H.+Dengauthor=E.+A.+Friedman&title=Ovarian+cancer+cells%2C+not+normal+cells%2C+are+damaged+by+mirk%2Fdyrk1b+kinase+inhibition&doi=10.1002%2Fijc.27917"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Ovarian cancer cells, not normal cells, are damaged by Mirk/Dyrk1B kinase inhibition</span></div><div class="casAuthors">Hu, Jing; Deng, Holly; Friedman, Eileen A.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2258-2269</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Prior studies had shown that the Mirk/dyrk1B gene is amplified/upregulated in about 75% of ovarian cancers, that protein levels of this kinase are elevated in quiescent G0 cells and that Mirk maintains tumor cells in quiescence by initiating rapid degrdn. of cyclin D isoforms and by phosphorylation of a member of the DREAM complex.  Depletion of Mirk/dyrk1B led to increased cyclin D levels, an elevated reactive oxygen species (ROS) content and loss of viability.  However, many normal cells in vivo are quiescent, and therefore, targeting a kinase found in quiescent cells might be problematic.  In our study, Mirk kinase activity was found to be higher in ovarian cancer cells than in normal cells.  Pharmacol. inhibition of Mirk/dyrk1B kinase increased cyclin D levels both in quiescent normal diploid cells and in quiescent CDKN2A-neg. ovarian cancer cells, but led to more active CDK4/cyclin D complexes in quiescent ovarian cancer cells, allowing them to escape G0/G1 quiescence, enter cycle with high ROS levels and undergo apoptosis.  The ROS scavenger N-acetyl cysteine reduced both the amt. of cleaved poly(ADP-ribose) polymerase (PARP) and the extent of cancer cell loss.  In contrast, normal cells were spared because of their expression of cyclin directed kinase (CDK) inhibitors that blocked unregulated cycling.  Quiescent early passage normal ovarian epithelial cells and two strains of quiescent normal diploid fibroblasts remained viable after the inhibition of Mirk/dyrk1B kinase, and the few cells that left G0/G1 quiescence were accumulated in G2+M.  Thus, inhibition of Mirk kinase targeted quiescent ovarian cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWFUPgfQBJiLVg90H21EOLACvtfcHk0ljK6-mhWuihsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs12rsbfF&md5=e0cf1c67ac4a66f2e12055bb0df0df0b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fijc.27917&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.27917%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DFriedman%26aufirst%3DE.%2BA.%26atitle%3DOvarian%2520cancer%2520cells%252C%2520not%2520normal%2520cells%252C%2520are%2520damaged%2520by%2520mirk%252Fdyrk1b%2520kinase%2520inhibition%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2013%26volume%3D132%26spage%3D2258%26epage%3D2269%26doi%3D10.1002%2Fijc.27917" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Litovchick, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florens, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Washburn, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeCaprio, J. A.</span></span> <span> </span><span class="NLM_article-title">Dyrk1a protein kinase promotes quiescence and senescence through dream complex assembly</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">801</span>– <span class="NLM_lpage">813</span>, <span class="refDoi"> DOI: 10.1101/gad.2034211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1101%2Fgad.2034211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=21498570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltlKntrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=801-813&author=L.+Litovchickauthor=L.+A.+Florensauthor=S.+K.+Swansonauthor=M.+P.+Washburnauthor=J.+A.+DeCaprio&title=Dyrk1a+protein+kinase+promotes+quiescence+and+senescence+through+dream+complex+assembly&doi=10.1101%2Fgad.2034211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">DYRK1A protein kinase promotes quiescence and senescence through DREAM complex assembly</span></div><div class="casAuthors">Litovchick, Larisa; Florens, Laurence A.; Swanson, Selene K.; Washburn, Michael P.; DeCaprio, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">801-813</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">In the absence of growth signals, cells exit the cell cycle and enter into G0 or quiescence.  Alternatively, cells enter senescence in response to inappropriate growth signals such as oncogene expression.  The mol. mechanisms required for cell cycle exit into quiescence or senescence are poorly understood.  The DREAM (DP, RB [retinoblastoma], E2F, and MuvB) complex represses cell cycle-dependent genes during quiescence.  DREAM contains p130, E2F4, DP1, and a stable core complex of five MuvB-like proteins: LIN9, LIN37, LIN52, LIN54, and RBBP4.  In mammalian cells, the MuvB core dissocs. from p130 upon entry into the cell cycle and binds to BMYB during S phase to activate the transcription of genes expressed late in the cell cycle.  We used mass spectroscopic anal. to identify phosphorylation sites that regulate the switch of the MuvB core from BMYB to DREAM.  Here we report that DYRK1A can specifically phosphorylate LIN52 on serine residue 28, and that this phosphorylation is required for DREAM assembly.  Inhibiting DYRK1A activity or point mutation of LIN52 disrupts DREAM assembly and reduces the ability of cells to enter quiescence or undergo Ras-induced senescence.  These data reveal an important role for DYRK1A in the regulation of DREAM activity and entry into quiescence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptXCB-wHS9bLVg90H21EOLACvtfcHk0ljK6-mhWuihsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltlKntrc%253D&md5=0d3048d4ea6a370a7a230b8f250e3867</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1101%2Fgad.2034211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.2034211%26sid%3Dliteratum%253Aachs%26aulast%3DLitovchick%26aufirst%3DL.%26aulast%3DFlorens%26aufirst%3DL.%2BA.%26aulast%3DSwanson%26aufirst%3DS.%2BK.%26aulast%3DWashburn%26aufirst%3DM.%2BP.%26aulast%3DDeCaprio%26aufirst%3DJ.%2BA.%26atitle%3DDyrk1a%2520protein%2520kinase%2520promotes%2520quiescence%2520and%2520senescence%2520through%2520dream%2520complex%2520assembly%26jtitle%3DGenes%2520Dev.%26date%3D2011%26volume%3D25%26spage%3D801%26epage%3D813%26doi%3D10.1101%2Fgad.2034211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewton, D. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, E.</span></span> <span> </span><span class="NLM_article-title">Mirk regulates the exit of colon cancer cells from quiescence</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">22916</span>– <span class="NLM_lpage">22925</span>, <span class="refDoi"> DOI: 10.1074/jbc.m109.035519</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1074%2Fjbc.M109.035519" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=19542220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpvFKitbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=22916-22925&author=K.+Jinauthor=D.+Z.+Ewtonauthor=S.+Parkauthor=J.+Huauthor=E.+Friedman&title=Mirk+regulates+the+exit+of+colon+cancer+cells+from+quiescence&doi=10.1074%2Fjbc.m109.035519"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Mirk Regulates the Exit of Colon Cancer Cells from Quiescence</span></div><div class="casAuthors">Jin, Kideok; Ewton, Daina Z.; Park, Sunju; Hu, Jing; Friedman, Eileen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">22916-22925</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Mirk/Dyrk1B is a serine/threonine kinase widely expressed in colon cancers.  Serum starvation induced HD6 colon carcinoma cells to enter a quiescent G0 state, characterized by a 2N DNA content and a lower RNA content than G1 cells.  Compared with cycling cells, quiescent cells exhibited 16-fold higher levels of the retinoblastoma protein p130/Rb2, which sequesters E2F4 to block entry into G1, 10-fold elevated levels of the CDK inhibitor p27kip1, and 10-fold higher levels of Mirk.  However, depletion of Mirk did not prevent entry into G0, but enabled quiescent HD6, SW480, and colo320 colon carcinoma cells to acquire some biochem. characteristics of G1 cells, including increased levels of cyclin D1 and cyclin D3 because of slower turnover, increased activity of their CDK4/cyclin D complexes, and increased phosphorylation and decreased E2F4 sequestering ability of the CDK4 target, p130/Rb2.  As a result, depletion of Mirk allowed some cells to escape quiescence and enabled cells released from quiescence to traverse G1 more quickly.  The kinase activity of Mirk was increased by the chemotherapeutic drug 5-fluorouracil (5-FU).  Treatment of p53 mutant colon cancer cells with 5-FU led to an elongated G1 in a Mirk-dependent manner, as G1 was shortened by ectopic overexpression of cyclin D1 mutated at the Mirk phosphorylation site (T288A), but not by wild-type cyclin D1.  Mirk, through regulating cyclin D turnover, and the CDK inhibitor p27, as shown by depletion studies, functioned independently and additively to regulate the exit of tumor cells from quiescence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaIbbM7RpsrbVg90H21EOLACvtfcHk0lh4-kM3tnhKJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpvFKitbg%253D&md5=98ffa71de12dddd61f2d4109e706ce07</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.035519&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.035519%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DK.%26aulast%3DEwton%26aufirst%3DD.%2BZ.%26aulast%3DPark%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DFriedman%26aufirst%3DE.%26atitle%3DMirk%2520regulates%2520the%2520exit%2520of%2520colon%2520cancer%2520cells%2520from%2520quiescence%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D22916%26epage%3D22925%26doi%3D10.1074%2Fjbc.m109.035519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boichuk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makielski, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litovchick, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baron, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zewe, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wozniak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehalek, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korzeniewski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seneviratne, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schöffski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debiec-Rychter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeCaprio, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duensing, A.</span></span> <span> </span><span class="NLM_article-title">The dream complex mediates gist cell quiescence and is a novel therapeutic target to enhance imatinib-induced apoptosis</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">5120</span>– <span class="NLM_lpage">5129</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-13-0579</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1158%2F0008-5472.CAN-13-0579" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=23786773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1OisrfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=5120-5129&author=S.+Boichukauthor=J.+A.+Parryauthor=K.+R.+Makielskiauthor=L.+Litovchickauthor=J.+L.+Baronauthor=J.+P.+Zeweauthor=A.+Wozniakauthor=K.+R.+Mehalekauthor=N.+Korzeniewskiauthor=D.+S.+Seneviratneauthor=P.+Sch%C3%B6ffskiauthor=M.+Debiec-Rychterauthor=J.+A.+DeCaprioauthor=A.+Duensing&title=The+dream+complex+mediates+gist+cell+quiescence+and+is+a+novel+therapeutic+target+to+enhance+imatinib-induced+apoptosis&doi=10.1158%2F0008-5472.can-13-0579"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The DREAM Complex Mediates GIST Cell Quiescence and Is a Novel Therapeutic Target to Enhance Imatinib-Induced Apoptosis</span></div><div class="casAuthors">Boichuk, Sergei; Parry, Joshua A.; Makielski, Kathleen R.; Litovchick, Larisa; Baron, Julianne L.; Zewe, James P.; Wozniak, Agnieszka; Mehalek, Keith R.; Korzeniewski, Nina; Seneviratne, Danushka S.; Schoeffski, Patrick; Debiec-Rychter, Maria; DeCaprio, James A.; Duensing, Anette</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5120-5129</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Gastrointestinal stromal tumors (GIST) can be successfully treated with imatinib mesylate (Gleevec); however, complete remissions are rare and patients frequently achieve disease stabilization in the presence of residual tumor masses.  The clin. observation that discontinuation of treatment can lead to tumor progression suggests that residual tumor cells are, in fact, quiescent and, therefore, able to re-enter the cell-division cycle.  In line with this notion, we have previously shown that imatinib induces GIST cell quiescence in vitro through the APCCDH1-SKP2-p27Klp1 signaling axis.  Here, we provide evidence that imatinib induces GIST cell quiescence in vivo and that this process also involves the DREAM complex, a multisubunit complex that has recently been identified as an addnl. key regulator of quiescence.  Importantly, inhibition of DREAM complex formation by depletion of the DREAM regulatory kinase DYRK1A or its target L1N52 was found to enhance imatinib-induced cell death.  Our results show that imatinib induces apoptosis in a fraction of GIST cells while, at the same time, a subset of cells undergoes quiescence involving the DREAM complex.  Inhibition of this process enhances imatinib-induced apoptosis, which opens the opportunity for future therapeutic interventions to target the DREAM complex for more efficient imatinib responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDU6EXoSaNMrVg90H21EOLACvtfcHk0lh4-kM3tnhKJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1OisrfL&md5=49f454ebe6457a4613dad2f5b289bcb9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-0579&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-0579%26sid%3Dliteratum%253Aachs%26aulast%3DBoichuk%26aufirst%3DS.%26aulast%3DParry%26aufirst%3DJ.%2BA.%26aulast%3DMakielski%26aufirst%3DK.%2BR.%26aulast%3DLitovchick%26aufirst%3DL.%26aulast%3DBaron%26aufirst%3DJ.%2BL.%26aulast%3DZewe%26aufirst%3DJ.%2BP.%26aulast%3DWozniak%26aufirst%3DA.%26aulast%3DMehalek%26aufirst%3DK.%2BR.%26aulast%3DKorzeniewski%26aufirst%3DN.%26aulast%3DSeneviratne%26aufirst%3DD.%2BS.%26aulast%3DSch%25C3%25B6ffski%26aufirst%3DP.%26aulast%3DDebiec-Rychter%26aufirst%3DM.%26aulast%3DDeCaprio%26aufirst%3DJ.%2BA.%26aulast%3DDuensing%26aufirst%3DA.%26atitle%3DThe%2520dream%2520complex%2520mediates%2520gist%2520cell%2520quiescence%2520and%2520is%2520a%2520novel%2520therapeutic%2520target%2520to%2520enhance%2520imatinib-induced%2520apoptosis%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D5120%26epage%3D5129%26doi%3D10.1158%2F0008-5472.can-13-0579" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Becker, W.</span></span> <span> </span><span class="NLM_article-title">A wake-up call to quiescent cancer cells - potential use ofDYRK1B inhibitors in cancer therapy</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">1203</span>– <span class="NLM_lpage">1211</span>, <span class="refDoi"> DOI: 10.1111/febs.14347</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1111%2Ffebs.14347" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=29193696" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFKhurnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2018&pages=1203-1211&author=W.+Becker&title=A+wake-up+call+to+quiescent+cancer+cells+-+potential+use+ofDYRK1B+inhibitors+in+cancer+therapy&doi=10.1111%2Ffebs.14347"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A wake-up call to quiescent cancer cells - potential use of DYRK1B inhibitors in cancer therapy</span></div><div class="casAuthors">Becker, Walter</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1203-1211</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Nondividing cancer cells are relatively resistant to chemotherapeutic drugs and environmental stress factors.  Promoting cell cycle re-entry of quiescent cancer cells is a potential strategy to enhance the cytotoxicity of agents that target cycling cells.  It is therefore important to elucidate the mechanisms by which these cells are maintained in the quiescent state.  The protein kinase dual specificity tyrosine phosphorylation-regulated kinase 1B (DYRK1B) is overexpressed in a subset of cancers and maintains cellular quiescence by counteracting G0/G1-S phase transition.  Specifically, DYRK1B controls the S phase checkpoint by stabilizing the cyclin-dependent kinase (CDK) inhibitor p27Kip1 and inducing the degrdn. of cyclin D.  DYRK1B also stabilizes the DREAM complex that represses cell cycle gene expression in G0 arrested cells.  In addn., DYRK1B enhances cell survival by upregulating antioxidant gene expression and reducing intracellular levels of reactive oxygen species (ROS).  Substantial evidence indicates that depletion or inhibition of DYRK1B drives cell cycle re-entry and enhances apoptosis of those quiescent cancer cells with high expression of DYRK1B.  Furthermore, small mol. DYRK1B inhibitors sensitize cells to the cytotoxic effects of anticancer drugs that target proliferating cells.  These encouraging findings justify continued efforts to investigate the use of DYRK1B inhibitors to disrupt the quiescent state and overturn chemoresistance of noncycling cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPcdYql-muXLVg90H21EOLACvtfcHk0lh4-kM3tnhKJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFKhurnM&md5=67216b30cc75747e7122118bcc7f1f53</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1111%2Ffebs.14347&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.14347%26sid%3Dliteratum%253Aachs%26aulast%3DBecker%26aufirst%3DW.%26atitle%3DA%2520wake-up%2520call%2520to%2520quiescent%2520cancer%2520cells%2520-%2520potential%2520use%2520ofDYRK1B%2520inhibitors%2520in%2520cancer%2520therapy%26jtitle%3DFEBS%2520J.%26date%3D2018%26volume%3D285%26spage%3D1203%26epage%3D1211%26doi%3D10.1111%2Ffebs.14347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewton, D. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercer, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, E.</span></span> <span> </span><span class="NLM_article-title">Mirk/dyrk1b kinase destabilizes cyclin d1 by phosphorylation at threonine 288</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">27790</span>– <span class="NLM_lpage">27798</span>, <span class="refDoi"> DOI: 10.1074/jbc.m403042200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1074%2Fjbc.M403042200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=15075324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2cXkvFGnu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=27790-27798&author=Y.+Zouauthor=D.+Z.+Ewtonauthor=X.+Dengauthor=S.+E.+Mercerauthor=E.+Friedman&title=Mirk%2Fdyrk1b+kinase+destabilizes+cyclin+d1+by+phosphorylation+at+threonine+288&doi=10.1074%2Fjbc.m403042200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Mirk/dyrk1B Kinase Destabilizes Cyclin D1 by Phosphorylation at Threonine 288</span></div><div class="casAuthors">Zou, Yonglong; Ewton, Daina Z.; Deng, Xiaobing; Mercer, Stephen E.; Friedman, Eileen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">27790-27798</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The phosphorylation of cyclin D1 at threonine 286 by glycogen synthase kinase 3β (GSK3β) has been shown to be required for the ubiquitination and nuclear export of cyclin D1 and its subsequent degrdn. in the proteasome.  The mutation of the nearby threonine 288 residue to nonphosphorylatable alanine has also been shown to reduce the ubiquitination of cyclin D1, suggesting that phosphorylation at threonine 288 may also lead to degrdn. of cyclin D1.  We now demonstrate that the G0/G1-active arginine-directed protein kinase Mirk/dyrk1B binds to cyclin D1 and phosphorylates cyclin D1 at threonine 288 in vivo and that the cyclin D1-T288A construct is more stable than wild-type cyclin D1.  Transient overexpression of Mirk in nontransformed Mv1Lu lung epithelial cells blocked cells in G0/G1.  Depletion of endogenous Mirk by RNA interference increased cyclin D1 protein levels but not mRNA levels, indicating that Mirk destabilizes cyclin D1 protein.  Destabilization was confirmed by induction of a stable Mirk transfectant of Mv1Lu cells, which blocked cell migration (Zou, Y., Lim, S., Lee, K., Deng, X., and Friedman, E. (2003) J. Biol. Chem. 278, 49573-49581), and caused a decrease in the half-life of endogenous cyclin D1, concomitant with an increase in Mirk expression.  In vitro cyclin D1 was phosphorylated in an additive fashion by Mirk and GSK3β.  Mirk-phosphorylated cyclin D1 mutated at the GSK3β phosphorylation site and was capable of phosphorylating cyclin D1 in the presence of the GSK3β inhibitor LiCl.  Mirk may function together with GSK3β to assist cell arrest in G0/G1 by destabilizing cyclin D1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppbNHWpluCW7Vg90H21EOLACvtfcHk0ljm6ae-q_h8gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXkvFGnu7Y%253D&md5=36ad27a9054ad5d2fabc4c5d7c52926f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M403042200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M403042200%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DY.%26aulast%3DEwton%26aufirst%3DD.%2BZ.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DMercer%26aufirst%3DS.%2BE.%26aulast%3DFriedman%26aufirst%3DE.%26atitle%3DMirk%252Fdyrk1b%2520kinase%2520destabilizes%2520cyclin%2520d1%2520by%2520phosphorylation%2520at%2520threonine%2520288%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D27790%26epage%3D27798%26doi%3D10.1074%2Fjbc.m403042200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ashford, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oxley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kettle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guichard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lochhead, P. A.</span></span> <span> </span><span class="NLM_article-title">A novel DYRK1B inhibitor AZ191 demonstrates that DYRK1B acts independently of GSK3β to phosphorylate cyclin D1 at Thr286, not Thr288</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>457</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1042/bj20130461</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1042%2FBJ20130461" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=24134204" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFSltbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=457&publication_year=2014&pages=43-56&author=A.+L.+Ashfordauthor=D.+Oxleyauthor=J.+Kettleauthor=K.+Hudsonauthor=S.+Guichardauthor=S.+J.+Cookauthor=P.+A.+Lochhead&title=A+novel+DYRK1B+inhibitor+AZ191+demonstrates+that+DYRK1B+acts+independently+of+GSK3%CE%B2+to+phosphorylate+cyclin+D1+at+Thr286%2C+not+Thr288&doi=10.1042%2Fbj20130461"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A novel DYRK1B inhibitor AZ191 demonstrates that DYRK1B acts independently of GSK3β to phosphorylate cyclin D1 at Thr286, not Thr288</span></div><div class="casAuthors">Ashford, Anne L.; Oxley, David; Kettle, Jason; Hudson, Kevin; Guichard, Sylvie; Cook, Simon J.; Lochhead, Pamela A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">457</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-56</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">DYRK1B (dual-specificity tyrosine phosphorylation-regulated kinase 1B) is amplified in certain cancers and may be an oncogene; however, our knowledge of DYRK1B has been limited by the lack of selective inhibitors.  In the present study we describe AZ191, a potent small mol. inhibitor that selectively inhibits DYRK1B in vitro and in cells.  CCND1 (cyclin D1), a key regulator of the mammalian G1-S-phase transition, is phosphorylated on Thr286 by GSK3β (glycogen synthase kinase 3β) to promote its degrdn.  DYRK1B has also been proposed to promote CCND1 turnover, but was reported to phosphorylate Thr288 rather than Thr286.  Using in vitro kinase assays, phospho-specific immunoblot anal. and MS in conjunction with AZ191 we now show that DYRK1B phosphorylates CCND1 at Thr286, not Thr288, in vitro and in cells.  In HEK (human embryonic kidney)-293 and PANC-1 cells (which exhibit DYRK1B amplification) DYRK1B drives Thr286 phosphorylation and proteasome-dependent turnover of CCND1 and this is abolished by AZ191 or DYRK1B RNAi, but not by GSK3β inhibitors or GSK3β RNAi.  DYRK1B expression causes a G1-phase cell-cycle arrest, but overexpression of CCND1 (wild-type or T286A) fails to overcome this; indeed, DYRK1B also promotes the expression of p21CIP1 (21 kDa CDK-interacting protein 1) and p27KIP1 (CDK-inhibitory protein 1).  The results of the present study demonstrate for the first time that DYRK1B is a novel Thr286-CCND1 kinase that acts independently of GSK3β to promote CCND1 degrdn.  Furthermore, we anticipate that AZ191 may prove useful in defining further substrates and biol. functions of DYRK1B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryxVa5ahY3ArVg90H21EOLACvtfcHk0ljm6ae-q_h8gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFSltbbF&md5=1951b0ef61c2fdaed0a15de224dec3dc</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1042%2FBJ20130461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20130461%26sid%3Dliteratum%253Aachs%26aulast%3DAshford%26aufirst%3DA.%2BL.%26aulast%3DOxley%26aufirst%3DD.%26aulast%3DKettle%26aufirst%3DJ.%26aulast%3DHudson%26aufirst%3DK.%26aulast%3DGuichard%26aufirst%3DS.%26aulast%3DCook%26aufirst%3DS.%2BJ.%26aulast%3DLochhead%26aufirst%3DP.%2BA.%26atitle%3DA%2520novel%2520DYRK1B%2520inhibitor%2520AZ191%2520demonstrates%2520that%2520DYRK1B%2520acts%2520independently%2520of%2520GSK3%25CE%25B2%2520to%2520phosphorylate%2520cyclin%2520D1%2520at%2520Thr286%252C%2520not%2520Thr288%26jtitle%3DBiochem.%2520J.%26date%3D2014%26volume%3D457%26spage%3D43%26epage%3D56%26doi%3D10.1042%2Fbj20130461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercer, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewton, D. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, E.</span></span> <span> </span><span class="NLM_article-title">The Cyclin-dependent Kinase Inhibitor p27Kip1 Is Stabilized in G0 by Mirk/dyrk1B Kinase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">22498</span>– <span class="NLM_lpage">22504</span>, <span class="refDoi"> DOI: 10.1074/jbc.m400479200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1074%2Fjbc.M400479200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=15010468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjvF2rtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=22498-22504&author=X.+Dengauthor=S.+E.+Mercerauthor=S.+Shahauthor=D.+Z.+Ewtonauthor=E.+Friedman&title=The+Cyclin-dependent+Kinase+Inhibitor+p27Kip1+Is+Stabilized+in+G0+by+Mirk%2Fdyrk1B+Kinase&doi=10.1074%2Fjbc.m400479200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The Cyclin-dependent Kinase Inhibitor p27Kip1 Is Stabilized in G0 by Mirk/dyrk1B Kinase</span></div><div class="casAuthors">Deng, Xiaobing; Mercer, Stephen E.; Shah, Sejal; Ewton, Daina Z.; Friedman, Eileen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">22498-22504</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Elevated levels of the cyclin-dependent kinase (CDK) inhibitor p27 block the cell in G0/G1 until mitogenic signals activate G1 cyclins and initiate proliferation.  Post-translational regulation of p27 by different phosphorylation events is crit. in allowing cells to proceed through the cell cycle.  We now demonstrate that the arginine-directed kinase, Mirk/dyrk1B, is maximally active in G0 in NIH3T3 cells, when it stabilizes p27 by phosphorylating it at Ser-10.  The phospho-mimetic mutant p27-S10D was more stable, and the non-phosphorylatable mutant p27-S10A was less stable than wild-type when expressed in G0-arrested cells.  Following phosphorylation by Mirk, p27 remains a functional CDK inhibitor, capable of binding to CDK2.  Mirk did not induce the translocation of p27 from the nucleus in G0, but instead co-localized with nuclear p27.  Depletion of Mirk by RNA interference decreased the phosphorylation of p27 at Ser-10 and the stability of endogenous p27.  RNAi to Mirk increased cell entry from G0 into G1 as shown by increased expression of proliferating cell nuclear antigen and decreased expression of p27.  These data suggest a model in which Mirk increases the amt. of nuclear p27 by stabilizing it during G0 when Mirk is most abundant.  Mitogen stimulation then causes cells to enter G1, reduces Mirk levels (Deng, X., Ewton, D., Pawlikowski, B., Maimone, M., and Friedman, E. (2003) J. Biol. Chem. 278, 41347-41354), and initiates the translocation of p27 to the cytoplasm.  In addn., depletion of Mirk by RNAi in postmitotic C2C12 myoblasts decreased protein but not mRNA levels of p27, suggesting that stabilization of p27 by Mirk also occurs during differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpx5aOLqkCRELVg90H21EOLACvtfcHk0ljm6ae-q_h8gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjvF2rtL0%253D&md5=4847b1a2365a17e44de4b99e10ec02ec</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M400479200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M400479200%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DMercer%26aufirst%3DS.%2BE.%26aulast%3DShah%26aufirst%3DS.%26aulast%3DEwton%26aufirst%3DD.%2BZ.%26aulast%3DFriedman%26aufirst%3DE.%26atitle%3DThe%2520Cyclin-dependent%2520Kinase%2520Inhibitor%2520p27Kip1%2520Is%2520Stabilized%2520in%2520G0%2520by%2520Mirk%252Fdyrk1B%2520Kinase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D22498%26epage%3D22504%26doi%3D10.1074%2Fjbc.m400479200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janumyan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sansam, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, E.</span></span> <span> </span><span class="NLM_article-title">G0 function of bcl2 and bcl-xl requires bax, bak, and p27 phosphorylation by mirk, revealing a novel role of bax and bak in quiescence regulation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">34108</span>– <span class="NLM_lpage">34120</span>, <span class="refDoi"> DOI: 10.1074/jbc.m806294200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1074%2Fjbc.M806294200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=18818203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVelu73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=34108-34120&author=Y.+Janumyanauthor=Q.+Cuiauthor=L.+Yanauthor=C.+G.+Sansamauthor=M.+Valentinauthor=E.+Yang&title=G0+function+of+bcl2+and+bcl-xl+requires+bax%2C+bak%2C+and+p27+phosphorylation+by+mirk%2C+revealing+a+novel+role+of+bax+and+bak+in+quiescence+regulation&doi=10.1074%2Fjbc.m806294200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">G0 Function of BCL2 and BCL-xL Requires BAX, BAK, and p27 Phosphorylation by Mirk, Revealing a Novel Role of BAX and BAK in Quiescence Regulation</span></div><div class="casAuthors">Janumyan, Yelena; Cui, Qinghua; Yan, Ling; Sansam, Courtney G.; Valentin, Mayda; Yang, Elizabeth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">34108-34120</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">BCL2 and BCL-xL facilitate G0 quiescence by decreasing RNA content and cell size and up-regulating p27 protein, but the precise mechanism is not understood.  We investigated the relationship between cell cycle regulation and the anti-apoptosis function of BCL2 and BCL-xL.  Neither caspase inhibition nor abrogation of mitochondria-dependent apoptosis by BAX and BAK deletion fully recapitulated the G0 effects of BCL2 or BCL-xL, suggesting that mechanisms in addn. to anti-apoptosis are involved in the cell cycle arrest function of BCL2 or BCL-xL.  We found that BCL2 and BCL-xL expression in bax-/- bak-/- cells did not confer cell cycle effects, consistent with the G0 function of BCL2 and BCL-xL being mediated through BAX or BAK.  Stabilization of p27 in G0 in BCL2 or BCL-xL cells was due to phosphorylation of p27 at Ser10 by the kinase Mirk.  In bax-/- bak-/- cells, total p27 and p27 phosphorylated at Ser10 were elevated.  Re-expression of BAX in bax-/- bak-/- cells and silencing of BAX and BAK in wild type cells confirmed that endogenous BAX and BAK modulated p27.  These data revealed a novel role for BAX and BAK in the regulation of G0 quiescence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAPrDVuZaTJ7Vg90H21EOLACvtfcHk0ljC_Pon4VtWtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVelu73O&md5=544796b9d792884a8df39132c01a39cc</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M806294200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M806294200%26sid%3Dliteratum%253Aachs%26aulast%3DJanumyan%26aufirst%3DY.%26aulast%3DCui%26aufirst%3DQ.%26aulast%3DYan%26aufirst%3DL.%26aulast%3DSansam%26aufirst%3DC.%2BG.%26aulast%3DValentin%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DE.%26atitle%3DG0%2520function%2520of%2520bcl2%2520and%2520bcl-xl%2520requires%2520bax%252C%2520bak%252C%2520and%2520p27%2520phosphorylation%2520by%2520mirk%252C%2520revealing%2520a%2520novel%2520role%2520of%2520bax%2520and%2520bak%2520in%2520quiescence%2520regulation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D34108%26epage%3D34120%26doi%3D10.1074%2Fjbc.m806294200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pozo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahonero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liñares, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayuso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagiwara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricoy, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernández-Laín, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sepúlveda, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Gómez, P.</span></span> <span> </span><span class="NLM_article-title">Inhibition of dyrk1a destabilizes egfr and reduces egfr-dependent glioblastoma growth</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">2475</span>– <span class="NLM_lpage">2487</span>, <span class="refDoi"> DOI: 10.1172/jci63623</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1172%2FJCI63623" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=23635774" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpsFWktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2013&pages=2475-2487&author=N.+Pozoauthor=C.+Zahoneroauthor=P.+Fernandezauthor=J.+M.+Li%C3%B1aresauthor=A.+Ayusoauthor=M.+Hagiwaraauthor=A.+P%C3%A9rezauthor=J.+R.+Ricoyauthor=A.+Hern%C3%A1ndez-La%C3%ADnauthor=J.+M.+Sep%C3%BAlvedaauthor=P.+S%C3%A1nchez-G%C3%B3mez&title=Inhibition+of+dyrk1a+destabilizes+egfr+and+reduces+egfr-dependent+glioblastoma+growth&doi=10.1172%2Fjci63623"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of DYRK1A destabilizes EGFR and reduces EGFR-dependent glioblastoma growth</span></div><div class="casAuthors">Pozo, Natividad; Zahonero, Cristina; Fernandez, Paloma; Linares, Jose M.; Ayuso, Angel; Hagiwara, Masatoshi; Perez, Angel; Ricoy, Jose R.; Hernandez-Lain, Aurelio; Sepulveda, Juan M.; Sanchez-Gomez, Pilar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2475-2487</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">Glioblastomas (GBMs) are very aggressive tumors that are resistant to conventional chemo- and radiotherapy.  New mol. therapeutic strategies are required to effectively eliminate the subpopulation of GBM tumor-initiating cells that are responsible for relapse.  Since EGFR is altered in 50% of GBMs, it represents one of the most promising targets; however, EGFR kinase inhibitors have produced poor results in clin. assays, with no clear explanation for the obsd. resistance.  We uncovered a fundamental role for the dual-specificity tyrosine phosphorylation-regulated kinase, DYRK1A, in regulating EGFR in GBMs.  We found that DYRK1A was highly expressed in these tumors and that its expression was correlated with that of EGFR.  Moreover, DYRK1A inhibition promoted EGFR degrdn. in primary GBM cell lines and neural progenitor cells, sharply reducing the self-renewal capacity of normal and tumorigenic cells.  Most importantly, our data suggest that a subset of GBMs depends on high surface EGFR levels, as DYRK1A inhibition compromised their survival and produced a profound decrease in tumor burden.  We propose that the recovery of EGFR stability is a key oncogenic event in a large proportion of gliomas and that pharmacol. inhibition of DYRK1A could represent a promising therapeutic intervention for EGFR-dependent GBMs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFv6XKpg04mLVg90H21EOLACvtfcHk0ljC_Pon4VtWtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpsFWktr8%253D&md5=41c1ef0531890b30dc6357a85fd6655c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1172%2FJCI63623&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI63623%26sid%3Dliteratum%253Aachs%26aulast%3DPozo%26aufirst%3DN.%26aulast%3DZahonero%26aufirst%3DC.%26aulast%3DFernandez%26aufirst%3DP.%26aulast%3DLi%25C3%25B1ares%26aufirst%3DJ.%2BM.%26aulast%3DAyuso%26aufirst%3DA.%26aulast%3DHagiwara%26aufirst%3DM.%26aulast%3DP%25C3%25A9rez%26aufirst%3DA.%26aulast%3DRicoy%26aufirst%3DJ.%2BR.%26aulast%3DHern%25C3%25A1ndez-La%25C3%25ADn%26aufirst%3DA.%26aulast%3DSep%25C3%25BAlveda%26aufirst%3DJ.%2BM.%26aulast%3DS%25C3%25A1nchez-G%25C3%25B3mez%26aufirst%3DP.%26atitle%3DInhibition%2520of%2520dyrk1a%2520destabilizes%2520egfr%2520and%2520reduces%2520egfr-dependent%2520glioblastoma%2520growth%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2013%26volume%3D123%26spage%3D2475%26epage%3D2487%26doi%3D10.1172%2Fjci63623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maye, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogerman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tejedor, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toftgard, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span> <span> </span><span class="NLM_article-title">Regulation of gli1 transcriptional activity in the nucleus by dyrk1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">35156</span>– <span class="NLM_lpage">35161</span>, <span class="refDoi"> DOI: 10.1074/jbc.m206743200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1074%2Fjbc.M206743200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=12138125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD38Xnt1Wgu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=35156-35161&author=J.+Maoauthor=P.+Mayeauthor=P.+Kogermanauthor=F.+J.+Tejedorauthor=R.+Toftgardauthor=W.+Xieauthor=G.+Wuauthor=D.+Wu&title=Regulation+of+gli1+transcriptional+activity+in+the+nucleus+by+dyrk1&doi=10.1074%2Fjbc.m206743200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of Gli1 Transcriptional Activity in the Nucleus by Dyrk1</span></div><div class="casAuthors">Mao, Junhao; Maye, Peter; Kogerman, Priit; Tejedor, Francisco J.; Toftgard, Rune; Xie, Wei; Wu, Guanqing; Wu, Dianqing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">35156-35161</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">To investigate the cellular role of dual specificity Yak1-related kinase (Dyrk) 1, a nuclear localized dual specificity protein kinase, we examd. its effect on transcriptional regulation using reporter gene assays.  We found that Dyrk1 can substantially enhance Gli1-dependent, but not LEF-1-, c-Jun-, or Elk-dependent, gene transcription.  In part, Dyrk1 does this through retaining Gli1 in the nucleus.  However, we also demonstrate that Dyrk1 can enhance the transcriptional activity of Gli1-AHA, a nuclear export mutant, suggesting that Dyrk1 may be more directly involved in regulating the transcriptional activity of Gli1.  In addn., Dyrk1 acted synergistically with Sonic hedgehog (Shh) to induce gene transcription and differentiation in mouse C3H10T1/2 cells.  The failure of Shh to stimulate Dyrk1 kinase activity suggests that Dyrk1 may not be directly regulated by the Shh signaling pathway but functionally interacts with it.  Thus, Gli1 transcriptional activity may be subjected to further regulation in the cell nucleus by a pathway distinct from Shh signaling, one mediated by Dyrk1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQU-enyGCh_bVg90H21EOLACvtfcHk0ljC_Pon4VtWtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xnt1Wgu7c%253D&md5=2a2702c9b9f3acc13204176735ffc62f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M206743200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M206743200%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DJ.%26aulast%3DMaye%26aufirst%3DP.%26aulast%3DKogerman%26aufirst%3DP.%26aulast%3DTejedor%26aufirst%3DF.%2BJ.%26aulast%3DToftgard%26aufirst%3DR.%26aulast%3DXie%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DD.%26atitle%3DRegulation%2520of%2520gli1%2520transcriptional%2520activity%2520in%2520the%2520nucleus%2520by%2520dyrk1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D35156%26epage%3D35161%26doi%3D10.1074%2Fjbc.m206743200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gruber, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutzinger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmer, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parigger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sternberg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cegielkowski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaja, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leban, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamm, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aberger, F.</span></span> <span> </span><span class="NLM_article-title">Dyrk1b as therapeutic target in hedgehog/gli-dependent cancer cells with smoothened inhibitor resistance</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">7134</span>– <span class="NLM_lpage">7148</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.6910</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.18632%2Foncotarget.6910" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=26784250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A280%3ADC%252BC28nhslSksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=7134-7148&author=W.+Gruberauthor=M.+Hutzingerauthor=D.+P.+Elmerauthor=T.+Pariggerauthor=C.+Sternbergauthor=L.+Cegielkowskiauthor=M.+Zajaauthor=J.+Lebanauthor=S.+Michelauthor=S.+Hammauthor=D.+Vittauthor=F.+Aberger&title=Dyrk1b+as+therapeutic+target+in+hedgehog%2Fgli-dependent+cancer+cells+with+smoothened+inhibitor+resistance&doi=10.18632%2Foncotarget.6910"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">DYRK1B as therapeutic target in Hedgehog/GLI-dependent cancer cells with Smoothened inhibitor resistance</span></div><div class="casAuthors">Gruber Wolfgang; Hutzinger Martin; Elmer Dominik Patrick; Parigger Thomas; Sternberg Christina; Cegielkowski Lukasz; Aberger Fritz; Zaja Mirko; Michel Susanne; Hamm Svetlana; Vitt Daniel; Leban Johann; Vitt Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">7134-48</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A wide range of human malignancies displays aberrant activation of Hedgehog (HH)/GLI signaling, including cancers of the skin, brain, gastrointestinal tract and hematopoietic system.  Targeting oncogenic HH/GLI signaling with small molecule inhibitors of the essential pathway effector Smoothened (SMO) has shown remarkable therapeutic effects in patients with advanced and metastatic basal cell carcinoma.  However, acquired and de novo resistance to SMO inhibitors poses severe limitations to the use of SMO antagonists and urgently calls for the identification of novel targets and compounds.Here we report on the identification of the Dual-Specificity-Tyrosine-Phosphorylation-Regulated Kinase 1B (DYRK1B) as critical positive regulator of HH/GLI signaling downstream of SMO.  Genetic and chemical inhibition of DYRK1B in human and mouse cancer cells resulted in marked repression of HH signaling and GLI1 expression, respectively.  Importantly, DYRK1B inhibition profoundly impaired GLI1 expression in both SMO-inhibitor sensitive and resistant settings.  We further introduce a novel small molecule DYRK1B inhibitor, DYRKi, with suitable pharmacologic properties to impair SMO-dependent and SMO-independent oncogenic GLI activity.  The results support the use of DYRK1B antagonists for the treatment of HH/GLI-associated cancers where SMO inhibitors fail to demonstrate therapeutic efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSFmF3n8no4EZw7prLlxLOzfW6udTcc2ebbm5E34fljirntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28nhslSksw%253D%253D&md5=166f2cf11d5cbc8b78bfd91cd5a28b1b</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.6910&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.6910%26sid%3Dliteratum%253Aachs%26aulast%3DGruber%26aufirst%3DW.%26aulast%3DHutzinger%26aufirst%3DM.%26aulast%3DElmer%26aufirst%3DD.%2BP.%26aulast%3DParigger%26aufirst%3DT.%26aulast%3DSternberg%26aufirst%3DC.%26aulast%3DCegielkowski%26aufirst%3DL.%26aulast%3DZaja%26aufirst%3DM.%26aulast%3DLeban%26aufirst%3DJ.%26aulast%3DMichel%26aufirst%3DS.%26aulast%3DHamm%26aufirst%3DS.%26aulast%3DVitt%26aufirst%3DD.%26aulast%3DAberger%26aufirst%3DF.%26atitle%3DDyrk1b%2520as%2520therapeutic%2520target%2520in%2520hedgehog%252Fgli-dependent%2520cancer%2520cells%2520with%2520smoothened%2520inhibitor%2520resistance%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D7134%26epage%3D7148%26doi%3D10.18632%2Foncotarget.6910" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, N.</span></span> <span> </span><span class="NLM_article-title">Dyrk1a suppression restrains mcl-1 expression and sensitizes nsclc cells to bcl-2 inhibitors</span>. <i>Cancer Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">387</span>– <span class="NLM_lpage">400</span>, <span class="refDoi"> DOI: 10.20892/j.issn.2095-3941.2019.0380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.20892%2Fj.issn.2095-3941.2019.0380" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=32587776" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitlalsb7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2020&pages=387-400&author=Y.+Liauthor=D.+Zhouauthor=S.+Xuauthor=M.+Raoauthor=Z.+Zhangauthor=L.+Wuauthor=C.+Zhangauthor=N.+Lin&title=Dyrk1a+suppression+restrains+mcl-1+expression+and+sensitizes+nsclc+cells+to+bcl-2+inhibitors&doi=10.20892%2Fj.issn.2095-3941.2019.0380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">DYRK1A suppression restrains Mcl-1 expression and sensitizes NSCLC cells to Bcl-2 inhibitors</span></div><div class="casAuthors">Li, Yangling; Zhou, Dongmei; Xu, Shuang; Rao, Mingjun; Zhang, Zuoyan; Wu, Linwen; Zhang, Chong; Lin, Nengming</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Biology & Medicine</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">387-400</span>CODEN:
                <span class="NLM_cas:coden">CBMADQ</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Tianjin Medical University Cancer Institute and Hospital</span>)
        </div><div class="casAbstract">Objective: Mcl-1 overexpression confers acquired resistance to Bcl-2 inhibitors in non-small cell lung cancer (NSCLC), but no direct Mcl-1 inhibitor is currently available for clin. use.  Thus, novel therapeutic strategies are urgently needed to target Mcl-1 and sensitize the anti-NSCLC activity of Bcl-2 inhibitors.  Methods: Cell proliferation was measured using sulforhodamine B and colony formation assays, and apoptosis was detected with Annexin V-FITC staining.  Gene expression was manipulated using siRNAs and plasmids.  Real-time PCR and Western blot were used to measure mRNA and protein levels.  Immunopptn. and immunofluorescence were used to analyze co-localization of dual specificity tyrosine-phosphorylation-regulated kinase 1A (DYRK1A) and Mcl-1.  Results: Suppression of DYRK1A resulted in reduced Mcl-1 expression in NSCLC cells, whereas overexpression of DYRK1A significantly increased Mcl-1 expression.  Suppression of DYRK1A did not alter Mcl-1 mRNA levels, but did result in an accelerated degrdn. of Mcl-1 protein in NSCLC cells.  Furthermore, DYRK1A mediated proteasome-dependent degrdn. of Mcl-1 in NSCLC cells, and DYRK1A co-localized with Mcl-1 in NSCLC cells and was co-expressed with Mcl-1 in tumor samples from lung cancer patients, suggesting that Mcl-1 may be a novel DYRK1A substrate.  We showed that combined therapy with harmine and Bcl-2 antagonists significantly inhibited cell proliferation and induced apoptosis in NSCLC cell lines as well as primary NSCLC cells.  Conclusions: Mcl-1 is a novel DYRK1A substrate, and the role of DYRK1A in promoting Mcl-1 stability makes it an attractive target for decreasing Bcl-2 inhibitor resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-Vu4TyDqLd7Vg90H21EOLACvtfcHk0lirkCL2l6asfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitlalsb7O&md5=1958de0db80c59bb12e2bda8281b98e7</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.20892%2Fj.issn.2095-3941.2019.0380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.20892%252Fj.issn.2095-3941.2019.0380%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DRao%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DN.%26atitle%3DDyrk1a%2520suppression%2520restrains%2520mcl-1%2520expression%2520and%2520sensitizes%2520nsclc%2520cells%2520to%2520bcl-2%2520inhibitors%26jtitle%3DCancer%2520Biol.%2520Med.%26date%3D2020%26volume%3D17%26spage%3D387%26epage%3D400%26doi%3D10.20892%2Fj.issn.2095-3941.2019.0380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, K.</span></span> <span> </span><span class="NLM_article-title">Multiple functions of dyrk2 in cancer and tissue development</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>593</i></span>,  <span class="NLM_fpage">2953</span>– <span class="NLM_lpage">2965</span>, <span class="refDoi"> DOI: 10.1002/1873-3468.13601</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1002%2F1873-3468.13601" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=31505048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslynsrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=593&publication_year=2019&pages=2953-2965&author=S.+Yoshidaauthor=K.+Yoshida&title=Multiple+functions+of+dyrk2+in+cancer+and+tissue+development&doi=10.1002%2F1873-3468.13601"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple functions of DYRK2 in cancer and tissue development</span></div><div class="casAuthors">Yoshida, Saishu; Yoshida, Kiyotsugu</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">593</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2953-2965</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Dual-specificity tyrosine-regulated kinases (DYRKs) are evolutionarily conserved from yeast to mammals.  Accumulating studies have revealed that DYRKs have important roles in regulation of the cell cycle and survival.  DYRK2, a member of the class II DYRK family protein, is a key regulator of p53, and phosphorylates it at Ser46 to induce apoptosis in response to DNA damage.  Moreover, recent studies have uncovered that DYRK2 regulates G1/S transition, epithelial-mesenchymal-transition, and stemness in human cancer cells.  DYRK2 also appears to have roles in tissue development in lower eukaryotes.  Thus, the elucidation of mechanisms for DYRK2 during mammalian tissue development will promote the understanding of cell differentiation, tissue homeostasis, and congenital diseases as well as cancer.  In this review, we discuss the roles of DYRK2 in tumor cells.  Moreover, we focus on DYRK2-dependent developmental mechanisms in several species including fly (Drosophila), worm (Caenorhabditis elegans), zebrafish (Danio rerio), and mammals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrypGm-jUS2DLVg90H21EOLACvtfcHk0lirkCL2l6asfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslynsrbJ&md5=1d8ccb464ded9d099472ad6da211f216</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2F1873-3468.13601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1873-3468.13601%26sid%3Dliteratum%253Aachs%26aulast%3DYoshida%26aufirst%3DS.%26aulast%3DYoshida%26aufirst%3DK.%26atitle%3DMultiple%2520functions%2520of%2520dyrk2%2520in%2520cancer%2520and%2520tissue%2520development%26jtitle%3DFEBS%2520Lett.%26date%3D2019%26volume%3D593%26spage%3D2953%26epage%3D2965%26doi%3D10.1002%2F1873-3468.13601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Correa-Saez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimenez-Izquierdo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrido-Rodriguez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrugares, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munoz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calzado, M. A.</span></span> <span> </span><span class="NLM_article-title">Updating dual-specificity tyrosine-phosphorylation-regulated kinase 2 (dyrk2): Molecular basis, functions and role in diseases</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">4747</span>– <span class="NLM_lpage">4763</span>, <span class="refDoi"> DOI: 10.1007/s00018-020-03556-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1007%2Fs00018-020-03556-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=32462403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVantb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2020&pages=4747-4763&author=A.+Correa-Saezauthor=R.+Jimenez-Izquierdoauthor=M.+Garrido-Rodriguezauthor=R.+Morrugaresauthor=E.+Munozauthor=M.+A.+Calzado&title=Updating+dual-specificity+tyrosine-phosphorylation-regulated+kinase+2+%28dyrk2%29%3A+Molecular+basis%2C+functions+and+role+in+diseases&doi=10.1007%2Fs00018-020-03556-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Updating dual-specificity tyrosine-phosphorylation-regulated kinase 2 (DYRK2): molecular basis, functions and role in diseases</span></div><div class="casAuthors">Correa-Saez, Alejandro; Jimenez-Izquierdo, Rafael; Garrido-Rodriguez, Martin; Morrugares, Rosario; Munoz, Eduardo; Calzado, Marco A.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">4747-4763</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">A review.  Abstr.: Members of the dual-specificity tyrosine-regulated kinase (DYRKs) subfamily possess a distinctive capacity to phosphorylate tyrosine, serine, and threonine residues.  Among the DYRK class II members, DYRK2 is considered a unique protein due to its role in disease.  According to the post-transcriptional and post-translational modifications, DYRK2 expression greatly differs among human tissues.  Regarding its mechanism of action, this kinase performs direct phosphorylation on its substrates or acts as a priming kinase, enabling subsequent substrate phosphorylation by GSK3β.  Moreover, DYRK2 acts as a scaffold for the EDVP E3 ligase complex during the G2/M phase of cell cycle.  DYRK2 functions such as cell survival, cell development, cell differentiation, proteasome regulation, and microtubules were studied in complete detail in this review.  We have also gathered available information from different bioinformatic resources to show DYRK2 interactome, normal and tumoral tissue expression, and recurrent cancer mutations.  Then, here we present an innovative approach to clarify DYRK2 functionality and importance.  DYRK2 roles in diseases have been studied in detail, highlighting this kinase as a key protein in cancer development.  First, DYRK2 regulation of c-Jun, c-Myc, Rpt3, TERT, and katanin p60 reveals the implication of this kinase in cell-cycle-mediated cancer development.  Addnl., depletion of this kinase correlated with reduced apoptosis, with consequences on cancer patient response to chemotherapy.  Other functions like cancer stem cell formation and epithelial-mesenchymal transition regulation are also controlled by DYRK2.  Furthermore, the pharmacol. modulation of this protein by different inhibitors (harmine, curcumine, LDN192960, and ID-8) has enabled to clarify DYRK2 functionality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-K55FlWvWvbVg90H21EOLACvtfcHk0lirkCL2l6asfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVantb3K&md5=d0f06b890a649b09cb386e74140c2b10</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2Fs00018-020-03556-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-020-03556-1%26sid%3Dliteratum%253Aachs%26aulast%3DCorrea-Saez%26aufirst%3DA.%26aulast%3DJimenez-Izquierdo%26aufirst%3DR.%26aulast%3DGarrido-Rodriguez%26aufirst%3DM.%26aulast%3DMorrugares%26aufirst%3DR.%26aulast%3DMunoz%26aufirst%3DE.%26aulast%3DCalzado%26aufirst%3DM.%2BA.%26atitle%3DUpdating%2520dual-specificity%2520tyrosine-phosphorylation-regulated%2520kinase%25202%2520%2528dyrk2%2529%253A%2520Molecular%2520basis%252C%2520functions%2520and%2520role%2520in%2520diseases%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2020%26volume%3D77%26spage%3D4747%26epage%3D4763%26doi%3D10.1007%2Fs00018-020-03556-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stringer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodlett, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roper, R. J.</span></span> <span> </span><span class="NLM_article-title">Targeting trisomic treatments: Optimizing dyrk1a inhibition to improve down syndrome deficits</span>. <i>Mol. Genet. Genomic Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">451</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.1002/mgg3.334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1002%2Fmgg3.334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=28944229" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOntL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2017&pages=451-465&author=M.+Stringerauthor=C.+R.+Goodlettauthor=R.+J.+Roper&title=Targeting+trisomic+treatments%3A+Optimizing+dyrk1a+inhibition+to+improve+down+syndrome+deficits&doi=10.1002%2Fmgg3.334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting trisomic treatments: optimizing Dyrk1a inhibition to improve Down syndrome deficits</span></div><div class="casAuthors">Stringer, Megan; Goodlett, Charles R.; Roper, Randall J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Genetics & Genomic Medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">451-465</span>CODEN:
                <span class="NLM_cas:coden">MGGMC6</span>;
        ISSN:<span class="NLM_cas:issn">2324-9269</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Overexpression of Dual-specificity tyrosine-phosphorylated regulated kinase 1A (DYRK1A), located on human chromosome 21, may alter mol. processes linked to developmental deficits in Down syndrome (DS).  Trisomic DYRK1A is a rational therapeutic target, and although redns. in Dyrk1a genetic dosage have shown improvements in trisomic mouse models, attempts to reduce Dyrk1a activity by pharmacol. mechanisms and correct these DS-assocd. phenotypes have been largely unsuccessful.  Epigallocatechin-3-gallate (EGCG) inhibits DYRK1A activity in vitro and this action has been postulated to account for improvement of some DS-assocd. phenotypes that have been reported in preclin. studies and clin. trials.  However, the beneficial effects of EGCG are inconsistent and there is no direct evidence that any obsd. improvement actually occurs through Dyrk1a inhibition.  Inconclusive outcomes likely reflect a lack of knowledge about the tissue-specific patterns of spatial and temporal overexpression and elevated activity of Dyrk1a that may contribute to emerging DS traits during development.  Emerging evidence indicates that Dyrk1a expression varies over the life span in DS mouse models, yet preclin. therapeutic treatments targeting Dyrk1a have largely not considered these developmental changes.  Therapies intended to improve DS phenotypes through normalizing trisomic Dyrk1a need to optimize the timing and dose of treatment to match the spatiotemporal patterning of excessive Dyrk1a activity in relevant tissues.  This will require more precise identification of developmental periods of vulnerability to enduring adverse effects of elevated Dyrk1a, representing the concurrence of increased Dyrk1a expression together with hypothesized tissue-specific-sensitive periods when Dyrk1a regulates cellular processes that shape the long-term functional properties of the tissue.  Future efforts targeting inhibition of trisomic Dyrk1a should identify these putative spatiotemporally specific developmental sensitive periods and det. whether normalizing Dyrk1a activity then can lead to improved outcomes in DS phenotypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7hKbmonKy97Vg90H21EOLACvtfcHk0ljxlTDW_-ma8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOntL%252FN&md5=5a99b154e1a12b6f8af4d73e391bb966</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fmgg3.334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmgg3.334%26sid%3Dliteratum%253Aachs%26aulast%3DStringer%26aufirst%3DM.%26aulast%3DGoodlett%26aufirst%3DC.%2BR.%26aulast%3DRoper%26aufirst%3DR.%2BJ.%26atitle%3DTargeting%2520trisomic%2520treatments%253A%2520Optimizing%2520dyrk1a%2520inhibition%2520to%2520improve%2520down%2520syndrome%2520deficits%26jtitle%3DMol.%2520Genet.%2520Genomic%2520Med.%26date%3D2017%26volume%3D5%26spage%3D451%26epage%3D465%26doi%3D10.1002%2Fmgg3.334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goodlett, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stringer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaCombe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roper, R. J.</span></span> <span> </span><span class="NLM_article-title">Evaluation of the therapeutic potential of epigallocatechin-3-gallate (egcg) via oral gavage in young adult down syndrome mice</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">10426</span>, <span class="refDoi"> DOI: 10.1038/s41598-020-67133-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1038%2Fs41598-020-67133-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=32591597" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlSitrbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=10426&author=C.+R.+Goodlettauthor=M.+Stringerauthor=J.+LaCombeauthor=R.+Patelauthor=J.+M.+Wallaceauthor=R.+J.+Roper&title=Evaluation+of+the+therapeutic+potential+of+epigallocatechin-3-gallate+%28egcg%29+via+oral+gavage+in+young+adult+down+syndrome+mice&doi=10.1038%2Fs41598-020-67133-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of the therapeutic potential of Epigallocatechin-3-gallate (EGCG) via oral gavage in young adult Down syndrome mice</span></div><div class="casAuthors">Goodlett, Charles R.; Stringer, Megan; LaCombe, Jonathan; Patel, Roshni; Wallace, Joseph M.; Roper, Randall J.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">10426</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Epigallocatechin-3-gallate (EGCG) is a candidate therapeutic for Down syndrome (DS) phenotypes based on in vitro inhibition of DYRK1A, a triplicated gene product of Trisomy 21 (Ts21).  Consumption of green tea exts. contg. EGCG improved some cognitive and behavioral outcomes in DS mouse models and in humans with Ts21.  In contrast, treatment with pure EGCG in DS mouse models did not improve neurobehavioral phenotypes.  This study tested the hypothesis that 200 mg/kg/day of pure EGCG, given via oral gavage, would improve neurobehavioral and skeletal phenotypes in the Ts65Dn DS mouse model.  Serum EGCG levels post-gavage were significantly higher in trisomic mice than in euploid mice.  Daily EGCG gavage treatments over three weeks resulted in growth deficits in both euploid and trisomic mice.  Compared to vehicle treatment, EGCG did not significantly improve behavioral performance of Ts65Dn mice in the multivariate concentric square field, balance beam, or Morris water maze tasks, but reduced swimming speed.  Furthermore, EGCG resulted in reduced cortical bone structure and strength in Ts65Dn mice.  These outcomes failed to support the therapeutic potential of EGCG, and the deleterious effects on growth and skeletal phenotypes underscore the need for caution in high-dose EGCG supplements as an intervention in DS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolMr6fUwdoLrVg90H21EOLACvtfcHk0ljxlTDW_-ma8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlSitrbM&md5=ce6e23ab184a700a54f3c403c401e4f4</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fs41598-020-67133-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-020-67133-z%26sid%3Dliteratum%253Aachs%26aulast%3DGoodlett%26aufirst%3DC.%2BR.%26aulast%3DStringer%26aufirst%3DM.%26aulast%3DLaCombe%26aufirst%3DJ.%26aulast%3DPatel%26aufirst%3DR.%26aulast%3DWallace%26aufirst%3DJ.%2BM.%26aulast%3DRoper%26aufirst%3DR.%2BJ.%26atitle%3DEvaluation%2520of%2520the%2520therapeutic%2520potential%2520of%2520epigallocatechin-3-gallate%2520%2528egcg%2529%2520via%2520oral%2520gavage%2520in%2520young%2520adult%2520down%2520syndrome%2520mice%26jtitle%3DSci.%2520Rep.%26date%3D2020%26volume%3D10%26spage%3D10426%26doi%3D10.1038%2Fs41598-020-67133-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryoo, S.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radnaabazar, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, J.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, I.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheon, Y.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, W.-J.</span></span> <span> </span><span class="NLM_article-title">DYRK1A-mediated Hyperphosphorylation of Tau</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">34850</span>– <span class="NLM_lpage">34857</span>, <span class="refDoi"> DOI: 10.1074/jbc.m707358200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1074%2Fjbc.M707358200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=17906291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlCrt77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=34850-34857&author=S.-R.+Ryooauthor=H.+K.+Jeongauthor=C.+Radnaabazarauthor=J.-J.+Yooauthor=H.-J.+Choauthor=H.-W.+Leeauthor=I.-S.+Kimauthor=Y.-H.+Cheonauthor=Y.+S.+Ahnauthor=S.-H.+Chungauthor=W.-J.+Song&title=DYRK1A-mediated+Hyperphosphorylation+of+Tau&doi=10.1074%2Fjbc.m707358200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">DYRK1A-mediated hyperphosphorylation of Tau: A functional link between Down syndrome and Alzheimer disease</span></div><div class="casAuthors">Ryoo, Soo-Ryoon; Jeong, Hey Kyeong; Radnaabazar, Chinzorig; Yoo, Jin-Ju; Cho, Hyun-Jeong; Lee, Hye-Won; Kim, In-Sook; Cheon, Young-Hee; Ahn, Young Soo; Chung, Sul-Hee; Song, Woo-Joo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">34850-34857</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Most individuals with Down syndrome show early onset of Alzheimer disease (AD), resulting from the extra copy of chromosome 21.  Located on this chromosome is a gene that encodes the dual specificity tyrosine phosphorylation-regulated kinase 1A (DYRK1A).  One of the pathol. hallmarks in AD is the presence of neurofibrillary tangles (NFTs), which are insol. deposits that consist of abnormally hyperphosphorylated Tau.  Previously it was reported that Tau at the Thr-212 residue was phosphorylated by Dyrk1A in vitro.  To det. the physiol. significance of this phosphorylation, an anal. was made of the amt. of phospho-Thr-212-Tau (pT212) in the brain of transgenic mice that overexpress the human DYRK1A protein (DYRK1A TG mice) that we recently generated.  A significant increase in the amt. of pT212 was found in the brains of DYRK1A transgenic mice when compared with age-matched littermate controls.  We further examd. whether Dyrk1A phosphorylates other Tau residues that are implicated in NFTs.  We found that Dyrk1A also phosphorylates Tau at Ser-202 and Ser-404 in vitro.  Phosphorylation by Dyrk1A strongly inhibited the ability of Tau to promote microtubule assembly.  Following this, using mammalian cells and DYRK1A TG mouse brains, it was demonstrated that the amts. of phospho-Ser-202-Tau and phospho-Ser-404-Tau are enhanced when DYRK1A amts. are high.  These results provide the first in vivo evidence for a physiol. role of DYRK1A in the hyperphosphorylation of Tau and suggest that the extra copy of the DYRK1A gene contributes to the early onset of AD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTPRnoJf4Ux7Vg90H21EOLACvtfcHk0ljxlTDW_-ma8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlCrt77K&md5=8105e5b9e3c9134aafebbe7af1e5cfec</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M707358200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M707358200%26sid%3Dliteratum%253Aachs%26aulast%3DRyoo%26aufirst%3DS.-R.%26aulast%3DJeong%26aufirst%3DH.%2BK.%26aulast%3DRadnaabazar%26aufirst%3DC.%26aulast%3DYoo%26aufirst%3DJ.-J.%26aulast%3DCho%26aufirst%3DH.-J.%26aulast%3DLee%26aufirst%3DH.-W.%26aulast%3DKim%26aufirst%3DI.-S.%26aulast%3DCheon%26aufirst%3DY.-H.%26aulast%3DAhn%26aufirst%3DY.%2BS.%26aulast%3DChung%26aufirst%3DS.-H.%26aulast%3DSong%26aufirst%3DW.-J.%26atitle%3DDYRK1A-mediated%2520Hyperphosphorylation%2520of%2520Tau%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D34850%26epage%3D34857%26doi%3D10.1074%2Fjbc.m707358200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferrer, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrachina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puig, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez de Lagrán, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martí, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avila, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dierssen, M.</span></span> <span> </span><span class="NLM_article-title">Constitutive dyrk1a is abnormally expressed in alzheimer disease, down syndrome, pick disease, and related transgenic models</span>. <i>Neurobiol. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">392</span>– <span class="NLM_lpage">400</span>, <span class="refDoi"> DOI: 10.1016/j.nbd.2005.03.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1016%2Fj.nbd.2005.03.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=16242644" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFCjs7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2005&pages=392-400&author=I.+Ferrerauthor=M.+Barrachinaauthor=B.+Puigauthor=M.+Mart%C3%ADnez+de+Lagr%C3%A1nauthor=E.+Mart%C3%ADauthor=J.+Avilaauthor=M.+Dierssen&title=Constitutive+dyrk1a+is+abnormally+expressed+in+alzheimer+disease%2C+down+syndrome%2C+pick+disease%2C+and+related+transgenic+models&doi=10.1016%2Fj.nbd.2005.03.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Constitutive Dyrk1A is abnormally expressed in Alzheimer disease, Down syndrome, Pick disease, and related transgenic models</span></div><div class="casAuthors">Ferrer, Isidro; Barrachina, Marta; Puig, Berta; Martinez de Lagran, M.; Marti, Eulalia; Avila, Jesus; Dierssen, Mara</div><div class="citationInfo"><span class="NLM_cas:title">Neurobiology of Disease</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">392-400</span>CODEN:
                <span class="NLM_cas:coden">NUDIEM</span>;
        ISSN:<span class="NLM_cas:issn">0969-9961</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">DYRK1A, dual-specificity tyrosine-regulated kinase 1A, maps to human chromosome 21 within the Down syndrome (DS) crit. region.  Dyrk1 phosphorylates the human microtubule-assocd. protein tau at Thr212 in vitro, a residue that is phosphorylated in fetal tau and hyper-phosphorylated in Alzheimer disease (AD) and tauopathies, including Pick disease (PiD).  Furthermore, phosphorylation of Thr212 primes tau for phosphorylation by glycogen synthase kinase 3 (GSK-3).  The present study examines Dyrk1A in the cerebral cortex of sporadic AD, adult DS with assocd. AD, and PiD.  Increased Dyrk1A immunoreactivity has been found in the cytoplasm and nuclei of scattered neurons of the neocortex, entorhinal cortex, and hippocampus in AD, DS, and PiD.  Dyrk1A is found in sarkosyl-insol. fractions which are enriched in phosphorylated tau in AD brains, thus suggesting a possible assocn. of Dyrk1A with neurofibrillary tangle pathol.  Yet, no clear relationship has been obsd. between tau phosphorylation at Thr212, and GSK-3 and Dyrk1A expression in diseased brains.  Transgenic mice bearing a triple tau mutation (G272V, P301L, and R406W) and expressing hyper-phosphoyrylated tau in neurons of the entorhinal cortex, hippocampus, and cerebral neocortex show increased expression of Dyrk1A in individual neurons in the same regions.  However, transgenic mice over-expressing Dyrk1A do not show increased phosphorylation of tau at Thr212, thus suggesting that Dyrk1A over-expression does not trigger per se hyper-phosphorylation of tau at Thr212 in vivo.  The present observations indicate modifications in the expression of constitutive Dyrk1A in the cytoplasm and nuclei of neurons in various neurodegenerative diseases assocd. with tau phosphorylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpW8vHkFXvWMLVg90H21EOLACvtfcHk0lh67dvna3ezoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFCjs7bN&md5=dd69cd9e3cdb10618294417873670311</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.nbd.2005.03.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.nbd.2005.03.020%26sid%3Dliteratum%253Aachs%26aulast%3DFerrer%26aufirst%3DI.%26aulast%3DBarrachina%26aufirst%3DM.%26aulast%3DPuig%26aufirst%3DB.%26aulast%3DMart%25C3%25ADnez%2Bde%2BLagr%25C3%25A1n%26aufirst%3DM.%26aulast%3DMart%25C3%25AD%26aufirst%3DE.%26aulast%3DAvila%26aufirst%3DJ.%26aulast%3DDierssen%26aufirst%3DM.%26atitle%3DConstitutive%2520dyrk1a%2520is%2520abnormally%2520expressed%2520in%2520alzheimer%2520disease%252C%2520down%2520syndrome%252C%2520pick%2520disease%252C%2520and%2520related%2520transgenic%2520models%26jtitle%3DNeurobiol.%2520Dis.%26date%3D2005%26volume%3D20%26spage%3D392%26epage%3D400%26doi%3D10.1016%2Fj.nbd.2005.03.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamino, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuripa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kudo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamagata, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akatsu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funakoshi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishitomi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakaguchi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uema, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, M.</span></span> <span> </span><span class="NLM_article-title">The DYRK1A gene, encoded in chromosome 21 Down syndrome critical region, bridges between β-amyloid production and tau phosphorylation in Alzheimer disease</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.1093/hmg/ddl437</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1093%2Fhmg%2Fddl437" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=17135279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotVWksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2007&pages=15-23&author=R.+Kimuraauthor=K.+Kaminoauthor=M.+Yamamotoauthor=A.+Nuripaauthor=T.+Kidaauthor=H.+Kazuiauthor=R.+Hashimotoauthor=T.+Tanakaauthor=T.+Kudoauthor=H.+Yamagataauthor=Y.+Tabaraauthor=T.+Mikiauthor=H.+Akatsuauthor=K.+Kosakaauthor=E.+Funakoshiauthor=K.+Nishitomiauthor=G.+Sakaguchiauthor=A.+Katoauthor=H.+Hattoriauthor=T.+Uemaauthor=M.+Takeda&title=The+DYRK1A+gene%2C+encoded+in+chromosome+21+Down+syndrome+critical+region%2C+bridges+between+%CE%B2-amyloid+production+and+tau+phosphorylation+in+Alzheimer+disease&doi=10.1093%2Fhmg%2Fddl437"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The DYRK1A gene, encoded in chromosome 21 Down syndrome critical region, bridges between β-amyloid production and tau phosphorylation in Alzheimer disease</span></div><div class="casAuthors">Kimura, Ryo; Kamino, Kouzin; Yamamoto, Mitsuko; Nuripa, Aidaralieva; Kida, Tomoyuki; Kazui, Hiroaki; Hashimoto, Ryota; Tanaka, Toshihisa; Kudo, Takashi; Yamagata, Hidehisa; Tabara, Yasuharu; Miki, Tetsuro; Akatsu, Hiroyasu; Kosaka, Kenji; Funakoshi, Eishi; Nishitomi, Kouhei; Sakaguchi, Gaku; Kato, Akira; Hattori, Hideyuki; Uema, Takeshi; Takeda, Masatoshi</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-23</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">We scanned throughout chromosome 21 to assess genetic assocns. with late-onset Alzheimer disease (AD) using 374 Japanese patients and 375 population-based controls, because trisomy 21 is known to be assocd. with early deposition of β-amyloid (Aβ) in the brain.  Among 417 markers spanning 33 Mb, 22 markers showed assocns. with either the allele or the genotype frequency (P < 0.05).  Logistic regression anal. with age, sex and apolipoprotein E (APOE)-ε4 dose supported genetic risk of 17 markers, of which eight markers were linked to the SAMSN1, PRSS7, NCAM2, RUNX1, DYRK1A and KCNJ6 genes.  In logistic regression, the DYRK1A (dual-specificity tyrosine-regulated kinase 1A) gene, located in the Down syndrome crit. region, showed the highest significance [OR = 2.99 (95% CI: 1.72-5.19), P = 0.001], whereas the RUNX1 gene showed a high odds ratio [OR = 23.3 (95% CI: 2.76-196.5), P = 0.038].  DYRK1A mRNA level in the hippocampus was significantly elevated in patients with AD when compared with pathol. controls (P < 0.01).  DYRK1A mRNA level was upregulated along with an increase in the Aβ-level in the brain of transgenic mice, overproducing Aβ at 9 mo of age.  In neuroblastoma cells, Aβ induced an increase in the DYRK1A transcript, which also led to tau phosphorylation at Thr212 under the overexpression of tau.  Therefore, the upregulation of DYRK1A transcription results from Aβ loading, further leading to tau phosphorylation.  These result indicates that DYRK1A could be a key mol. bridging between β-amyloid prodn. and tau phosphorylation in AD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpETsvyVeafabVg90H21EOLACvtfcHk0lh67dvna3ezoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotVWksA%253D%253D&md5=5aa7f15a92ce6328d33cb5cb179b2be2</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2Fddl437&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252Fddl437%26sid%3Dliteratum%253Aachs%26aulast%3DKimura%26aufirst%3DR.%26aulast%3DKamino%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DNuripa%26aufirst%3DA.%26aulast%3DKida%26aufirst%3DT.%26aulast%3DKazui%26aufirst%3DH.%26aulast%3DHashimoto%26aufirst%3DR.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DKudo%26aufirst%3DT.%26aulast%3DYamagata%26aufirst%3DH.%26aulast%3DTabara%26aufirst%3DY.%26aulast%3DMiki%26aufirst%3DT.%26aulast%3DAkatsu%26aufirst%3DH.%26aulast%3DKosaka%26aufirst%3DK.%26aulast%3DFunakoshi%26aufirst%3DE.%26aulast%3DNishitomi%26aufirst%3DK.%26aulast%3DSakaguchi%26aufirst%3DG.%26aulast%3DKato%26aufirst%3DA.%26aulast%3DHattori%26aufirst%3DH.%26aulast%3DUema%26aufirst%3DT.%26aulast%3DTakeda%26aufirst%3DM.%26atitle%3DThe%2520DYRK1A%2520gene%252C%2520encoded%2520in%2520chromosome%252021%2520Down%2520syndrome%2520critical%2520region%252C%2520bridges%2520between%2520%25CE%25B2-amyloid%2520production%2520and%2520tau%2520phosphorylation%2520in%2520Alzheimer%2520disease%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2007%26volume%3D16%26spage%3D15%26epage%3D23%26doi%3D10.1093%2Fhmg%2Fddl437" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barallobre, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bové, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laguna, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delabar, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vila, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arbonés, M. L.</span></span> <span> </span><span class="NLM_article-title">Dyrk1a promotes dopaminergic neuron survival in the developing brain and in a mouse model of parkinson’s disease</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e1289</span> <span class="refDoi"> DOI: 10.1038/cddis.2014.253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1038%2Fcddis.2014.253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=24922073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpsVGqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&author=M.+J.+Barallobreauthor=C.+Perierauthor=J.+Bov%C3%A9author=A.+Lagunaauthor=J.+M.+Delabarauthor=M.+Vilaauthor=M.+L.+Arbon%C3%A9s&title=Dyrk1a+promotes+dopaminergic+neuron+survival+in+the+developing+brain+and+in+a+mouse+model+of+parkinson%E2%80%99s+disease&doi=10.1038%2Fcddis.2014.253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">DYRK1A promotes dopaminergic neuron survival in the developing brain and in a mouse model of Parkinson's disease</span></div><div class="casAuthors">Barallobre, M. J.; Perier, C.; Bove, J.; Laguna, A.; Delabar, J. M.; Vila, M.; Arbones, M. L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Disease</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e1289</span>CODEN:
                <span class="NLM_cas:coden">CDDEA4</span>;
        ISSN:<span class="NLM_cas:issn">2041-4889</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In the brain, programmed cell death (PCD) serves to adjust the nos. of the different types of neurons during development, and its pathol. reactivation in the adult leads to neurodegeneration.  Dual-specificity tyrosine-(Y)-phosphorylation regulated kinase 1A (DYRK1A) is a pleiotropic kinase involved in neural proliferation and cell death, and its role during brain growth is evolutionarily conserved.  Human DYRK1A lies in the Down syndrome crit. region on chromosome 21, and heterozygous mutations in the gene cause microcephaly and neurol. dysfunction.  The mouse model for DYRK1A haploinsufficiency (the Dyrk1a+/- mouse) presents neuronal deficits in specific regions of the adult brain, including the substantia nigra (SN), although the mechanisms underlying these pathogenic effects remain unclear.  Here we study the effect of DYRK1A copy no. variation on dopaminergic cell homeostasis.  We show that mesencephalic DA (mDA) neurons are generated in the embryo at normal rates in the Dyrk1a haploinsufficient model and in a model (the mBACtgDyrk1a mouse) that carries three copies of Dyrk1a.  We also show that the no. of mDA cells diminishes in postnatal Dyrk1a+/- mice and increases in mBACtgDyrk1a mice due to an abnormal activity of the mitochondrial caspase9 (Casp9)-dependent apoptotic pathway during the main wave of PCD that affects these neurons.  In addn., we show that the cell death induced by 1-methyl-4-phenyl-1,2,3,6 tetrahydropyridine (MPTP), a toxin that activates Casp9-dependent apoptosis in mDA neurons, is attenuated in adult mBACtgDyrk1a mice, leading to an increased survival of SN DA neurons 21 days after MPTP intoxication.  Finally, we present data indicating that Dyrk1a phosphorylation of Casp9 at the Thr125 residue is the mechanism by which this kinase hinders both physiol. and pathol. PCD in mDA neurons.  These data provide new insight into the mechanisms that control cell death in brain DA neurons and they show that deregulation of developmental apoptosis may contribute to the phenotype of patients with imbalanced DYRK1A gene dosage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2sgRdeNjB2rVg90H21EOLACvtfcHk0lh67dvna3ezoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpsVGqt70%253D&md5=4b0f7afaa86266fa556f74c6f12960b2</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fcddis.2014.253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddis.2014.253%26sid%3Dliteratum%253Aachs%26aulast%3DBarallobre%26aufirst%3DM.%2BJ.%26aulast%3DPerier%26aufirst%3DC.%26aulast%3DBov%25C3%25A9%26aufirst%3DJ.%26aulast%3DLaguna%26aufirst%3DA.%26aulast%3DDelabar%26aufirst%3DJ.%2BM.%26aulast%3DVila%26aufirst%3DM.%26aulast%3DArbon%25C3%25A9s%26aufirst%3DM.%2BL.%26atitle%3DDyrk1a%2520promotes%2520dopaminergic%2520neuron%2520survival%2520in%2520the%2520developing%2520brain%2520and%2520in%2520a%2520mouse%2520model%2520of%2520parkinson%25E2%2580%2599s%2520disease%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2014%26volume%3D5%26doi%3D10.1038%2Fcddis.2014.253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwatsubo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paik, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, K. C.</span></span> <span> </span><span class="NLM_article-title">Dyrk1A Phosphorylates α-Synuclein and Enhances Intracellular Inclusion Formation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">33250</span>– <span class="NLM_lpage">33257</span>, <span class="refDoi"> DOI: 10.1074/jbc.m606147200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1074%2Fjbc.M606147200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=16959772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFegsb7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=33250-33257&author=E.+J.+Kimauthor=J.+Y.+Sungauthor=H.+J.+Leeauthor=H.+Rhimauthor=M.+Hasegawaauthor=T.+Iwatsuboauthor=D.+S.+Minauthor=J.+Kimauthor=S.+R.+Paikauthor=K.+C.+Chung&title=Dyrk1A+Phosphorylates+%CE%B1-Synuclein+and+Enhances+Intracellular+Inclusion+Formation&doi=10.1074%2Fjbc.m606147200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Dyrk1A phosphorylates α-Synuclein and enhances intracellular inclusion formation</span></div><div class="casAuthors">Kim, Eun Joo; Sung, Jee Young; Lee, Hyun Jung; Rhim, Hyewhon; Hasegawa, Masato; Iwatsubo, Takeshi; Min, Do Sik; Kim, Jongsun; Paik, Seung R.; Chung, Kwang Chul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">33250-33257</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Lewy bodies (LBs) are pathol. hallmarks of Parkinson disease (PD) but also occur in Alzheimer disease (AD) and dementia of LBs. α-Synuclein, the major component of LBs, is obsd. in the brain of Down syndrome (DS) patients with AD.  Dyrk1A, a dual specificity tyrosine-regulated kinase (Dyrk) family member, is the mammalian ortholog of the Drosophila minibrain (Mnb) gene, essential for normal postembryonic neurogenesis.  The Dyrk1A gene resides in the human chromosome 21q22.2 region, which is assocd. with DS anomalies, including mental retardation.  In this study, we examd. whether Dyrk1A interacts with α-synuclein and subsequently affects intracellular α-synuclein inclusion formation in immortalized hippocampal neuronal (H19-7) cells.  Dyrk1A selectively binds to α-synuclein in transformed and primary neuronal cells.  α-Synuclein overexpression, followed by basic fibroblast growth factor-induced neuronal differentiation, resulted in cell death.  We obsd. that accompanying cell death was increased α-synuclein phosphorylation and intracytoplasmic aggregation.  In addn., the transfection of kinase-inactive Dyrk1A or Dyrk1A small interfering RNA blocked α-synuclein phosphorylation and aggregate formation.  In vitro kinase assay of anti-Dyrk1A immunocomplexes demonstrated that Dyrk1A could phosphorylate α-synuclein at Ser-87.  Furthermore, aggregates formed by phosphorylated α-synuclein have a distinct morphol. and are more neurotoxic compared with aggregates composed of unmodified wild type α-synuclein.  These findings suggest α-synuclein inclusion formation regulated by Dyrk1A, potentially affecting neuronal cell viability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgp1DXYX4SP7Vg90H21EOLACvtfcHk0li72o7im1QXnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFegsb7P&md5=c3b2c23a59da54a7f9444a7c6d01c318</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M606147200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M606147200%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DE.%2BJ.%26aulast%3DSung%26aufirst%3DJ.%2BY.%26aulast%3DLee%26aufirst%3DH.%2BJ.%26aulast%3DRhim%26aufirst%3DH.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DIwatsubo%26aufirst%3DT.%26aulast%3DMin%26aufirst%3DD.%2BS.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DPaik%26aufirst%3DS.%2BR.%26aulast%3DChung%26aufirst%3DK.%2BC.%26atitle%3DDyrk1A%2520Phosphorylates%2520%25CE%25B1-Synuclein%2520and%2520Enhances%2520Intracellular%2520Inclusion%2520Formation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D33250%26epage%3D33257%26doi%3D10.1074%2Fjbc.m606147200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dirice, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walpita, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetere, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahraman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dančík, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, B. K.</span></span> <span> </span><span class="NLM_article-title">Inhibition of DYRK1A Stimulates Human β-Cell Proliferation</span>. <i>Diabetes</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1660</span>– <span class="NLM_lpage">1671</span>, <span class="refDoi"> DOI: 10.2337/db15-1127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.2337%2Fdb15-1127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=26953159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWns7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2016&pages=1660-1671&author=E.+Diriceauthor=D.+Walpitaauthor=A.+Vetereauthor=B.+C.+Meierauthor=S.+Kahramanauthor=J.+Huauthor=V.+Dan%C4%8D%C3%ADkauthor=S.+M.+Burnsauthor=T.+J.+Gilbertauthor=D.+E.+Olsonauthor=P.+A.+Clemonsauthor=R.+N.+Kulkarniauthor=B.+K.+Wagner&title=Inhibition+of+DYRK1A+Stimulates+Human+%CE%B2-Cell+Proliferation&doi=10.2337%2Fdb15-1127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of DYRK1A stimulates human beta-cell proliferation</span></div><div class="casAuthors">Dirice, Ercument; Walpita, Deepika; Vetere, Amedeo; Meier, Bennett C.; Kahraman, Sevim; Hu, Jiang; Dancik, Vlado; Burns, Sean M.; Gilbert, Tamara J.; Olson, David E.; Clemons, Paul A.; Kulkarni, Rohit N.; Wagner, Bridget K.</div><div class="citationInfo"><span class="NLM_cas:title">Diabetes</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1660-1671</span>CODEN:
                <span class="NLM_cas:coden">DIAEAZ</span>;
        ISSN:<span class="NLM_cas:issn">0012-1797</span>.
    
            (<span class="NLM_cas:orgname">American Diabetes Association, Inc.</span>)
        </div><div class="casAbstract">Restoring functional β-cell mass is an important therapeutic goal for both type 1 and type 2 diabetes (1).  While proliferation of existing β-cells is the primary means of β-cell replacement in rodents (2), it is unclear whether a similar principle applies to humans, as human β-cells are remarkably resistant to stimulation of division (3,4).  Here, we show that 5-iodotubercidin (5-IT), an annotated adenosine kinase inhibitor previously reported to increase proliferation in rodent and porcine islets (5), strongly and selectively increases human β-cell proliferation in vitro and in vivo.  Remarkably, 5-IT also increased glucose-dependent insulin secretion after prolonged treatment.  Kinome profiling revealed 5-IT to be a potent and selective inhibitor of the dual-specificity tyrosine phosphorylation-regulated kinase (DYRK) and cell division cycle-like kinase families.  Induction of β-cell proliferation by either 5-IT or harmine, another natural product DYRK1A inhibitor, was suppressed by coincubation with the calcineurin inhibitor FK506, suggesting involvement of DYRK1A and nuclear factor of activated T cells signaling.  Gene expression profiling in whole islets treated with 5-IT revealed induction of proliferation- and cell cycle-related genes, suggesting that true proliferation is induced by 5-IT.  Furthermore, 5-IT promotes β-cell proliferation in human islets grafted under the kidney capsule of NOD-scid IL2Rgnull mice.  These results point to inhibition of DYRK1A as a therapeutic strategy to increase human β-cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqLKPVi2wkdrVg90H21EOLACvtfcHk0li72o7im1QXnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWns7vP&md5=c383c8d1f2c1739252ad3cfe4361585c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.2337%2Fdb15-1127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2337%252Fdb15-1127%26sid%3Dliteratum%253Aachs%26aulast%3DDirice%26aufirst%3DE.%26aulast%3DWalpita%26aufirst%3DD.%26aulast%3DVetere%26aufirst%3DA.%26aulast%3DMeier%26aufirst%3DB.%2BC.%26aulast%3DKahraman%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DDan%25C4%258D%25C3%25ADk%26aufirst%3DV.%26aulast%3DBurns%26aufirst%3DS.%2BM.%26aulast%3DGilbert%26aufirst%3DT.%2BJ.%26aulast%3DOlson%26aufirst%3DD.%2BE.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DKulkarni%26aufirst%3DR.%2BN.%26aulast%3DWagner%26aufirst%3DB.%2BK.%26atitle%3DInhibition%2520of%2520DYRK1A%2520Stimulates%2520Human%2520%25CE%25B2-Cell%2520Proliferation%26jtitle%3DDiabetes%26date%3D2016%26volume%3D65%26spage%3D1660%26epage%3D1671%26doi%3D10.2337%2Fdb15-1127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen-Tran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meeusen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamireddy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swafford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bursalaya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiDonato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harb, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippi, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deaton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suarez-Pinzon, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herman, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hering, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin Seidel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNamara, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glynne, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laffitte, B.</span></span> <span> </span><span class="NLM_article-title">Inhibition of dyrk1a and gsk3b induces human beta-cell proliferation</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">8372</span>, <span class="refDoi"> DOI: 10.1038/ncomms9372</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1038%2Fncomms9372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=26496802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslegu7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8372&author=W.+Shenauthor=B.+Taylorauthor=Q.+Jinauthor=V.+Nguyen-Tranauthor=S.+Meeusenauthor=Y.+Q.+Zhangauthor=A.+Kamireddyauthor=A.+Swaffordauthor=A.+F.+Powersauthor=J.+Walkerauthor=J.+Lambauthor=B.+Bursalayaauthor=M.+DiDonatoauthor=G.+Harbauthor=M.+Qiuauthor=C.+M.+Filippiauthor=L.+Deatonauthor=C.+N.+Turkauthor=W.+L.+Suarez-Pinzonauthor=Y.+Liuauthor=X.+Haoauthor=T.+Moauthor=S.+Yanauthor=J.+Liauthor=A.+E.+Hermanauthor=B.+J.+Heringauthor=T.+Wuauthor=H.+Martin+Seidelauthor=P.+McNamaraauthor=R.+Glynneauthor=B.+Laffitte&title=Inhibition+of+dyrk1a+and+gsk3b+induces+human+beta-cell+proliferation&doi=10.1038%2Fncomms9372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of DYRK1A and GSK3B induces human β-cell proliferation</span></div><div class="casAuthors">Shen, Weijun; Taylor, Brandon; Jin, Qihui; Nguyen-Tran, Van; Meeusen, Shelly; Zhang, You-Qing; Kamireddy, Anwesh; Swafford, Austin; Powers, Andrew F.; Walker, John; Lamb, John; Bursalaya, Badry; DiDonato, Michael; Harb, George; Qiu, Minhua; Filippi, Christophe M.; Deaton, Lisa; Turk, Carolina N.; Suarez-Pinzon, Wilma L.; Liu, Yahu; Hao, Xueshi; Mo, Tingting; Yan, Shanshan; Li, Jing; Herman, Ann E.; Hering, Bernhard J.; Wu, Tom; Seidel, H. Martin; McNamara, Peter; Glynne, Richard; Laffitte, Bryan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8372</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Insufficient pancreatic β-cell mass or function results in diabetes mellitus.  While significant progress has been made in regulating insulin secretion from β-cells in diabetic patients, no pharmacol. agents have been described that increase β-cell replication in humans.  Here we report aminopyrazine compds. that stimulate robust β-cell proliferation in adult primary islets, most likely as a result of combined inhibition of DYRK1A and GSK3B.  Aminopyrazine-treated human islets retain functionality in vitro and after transplantation into diabetic mice.  Oral dosing of these compds. in diabetic mice induces β-cell proliferation, increases β-cell mass and insulin content, and improves glycemic control.  Biochem., genetic and cell biol. data point to Dyrk1a as the key mol. target.  This study supports the feasibility of treating diabetes with an oral therapy to restore β-cell mass, and highlights a tractable pathway for future drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVHzP-VkH5nrVg90H21EOLACvtfcHk0li72o7im1QXnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslegu7%252FK&md5=5ef960af3bfd2795b9fdce52876e5d59</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fncomms9372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms9372%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DW.%26aulast%3DTaylor%26aufirst%3DB.%26aulast%3DJin%26aufirst%3DQ.%26aulast%3DNguyen-Tran%26aufirst%3DV.%26aulast%3DMeeusen%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.%2BQ.%26aulast%3DKamireddy%26aufirst%3DA.%26aulast%3DSwafford%26aufirst%3DA.%26aulast%3DPowers%26aufirst%3DA.%2BF.%26aulast%3DWalker%26aufirst%3DJ.%26aulast%3DLamb%26aufirst%3DJ.%26aulast%3DBursalaya%26aufirst%3DB.%26aulast%3DDiDonato%26aufirst%3DM.%26aulast%3DHarb%26aufirst%3DG.%26aulast%3DQiu%26aufirst%3DM.%26aulast%3DFilippi%26aufirst%3DC.%2BM.%26aulast%3DDeaton%26aufirst%3DL.%26aulast%3DTurk%26aufirst%3DC.%2BN.%26aulast%3DSuarez-Pinzon%26aufirst%3DW.%2BL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHao%26aufirst%3DX.%26aulast%3DMo%26aufirst%3DT.%26aulast%3DYan%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DHerman%26aufirst%3DA.%2BE.%26aulast%3DHering%26aufirst%3DB.%2BJ.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DMartin%2BSeidel%26aufirst%3DH.%26aulast%3DMcNamara%26aufirst%3DP.%26aulast%3DGlynne%26aufirst%3DR.%26aulast%3DLaffitte%26aufirst%3DB.%26atitle%3DInhibition%2520of%2520dyrk1a%2520and%2520gsk3b%2520induces%2520human%2520beta-cell%2520proliferation%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D8372%26doi%3D10.1038%2Fncomms9372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rachdi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kariyawasam, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aïello, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herault, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delabar, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polak, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scharfmann, R.</span></span> <span> </span><span class="NLM_article-title">Dyrk1A induces pancreatic β cell mass expansion and improves glucose tolerance</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2221</span>– <span class="NLM_lpage">2229</span>, <span class="refDoi"> DOI: 10.4161/cc.29250</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.4161%2Fcc.29250" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=24870561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1WisLzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=2221-2229&author=L.+Rachdiauthor=D.+Kariyawasamauthor=V.+A%C3%AFelloauthor=Y.+Heraultauthor=N.+Janelauthor=J.-M.+Delabarauthor=M.+Polakauthor=R.+Scharfmann&title=Dyrk1A+induces+pancreatic+%CE%B2+cell+mass+expansion+and+improves+glucose+tolerance&doi=10.4161%2Fcc.29250"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Dyrk1A induces pancreatic β cell mass expansion and improves glucose tolerance</span></div><div class="casAuthors">Rachdi, Latif; Kariyawasam, Dulanjalee; Aiello, Virginie; Herault, Yann; Janel, Nathalie; Delabar, Jean-Maurice; Polak, Michel; Scharfmann, Raphael</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2221-2229</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Type 2 diabetes is caused by a limited capacity of insulin-producing pancreatic β cells to increase their mass and function in response to insulin resistance.  The signaling pathways that pos. regulate functional β cell mass have not been fully elucidated.  DYRK1A (also called minibrain/MNB) is a member of the dual-specificity tyrosine phosphorylation-regulated kinase (DYRK) family.  A significant amt. of data implicates DYRK1A in brain growth and Down syndrome, and recent data indicate that Dyrk1A haploinsufficient mice have a low functional β cell mass.  Here we ask whether Dyrk1A upregulation could be a way to increase functional β cell mass.  We used mice overexpressing Dyrk1A under the control of its own regulatory sequences (mBACTgDyrk1A).  These mice exhibit decreased glucose levels and hyperinsulinemia in the fasting state.  Improved glucose tolerance is obsd. in these mice as early as 4 wk of age.  Upregulation of Dyrk1A in β cells induces expansion of β cell mass through increased proliferation and cell size.  Importantly, mBACTgDyrk1A mice are protected against high-fat-diet-induced β cell failure through increase in β cell mass and insulin sensitivity.  These studies show the crucial role of the DYRK1A pathway in the regulation of β cell mass and carbohydrate metab. in vivo.  Activating the DYRK1A pathway could thus represent an innovative way to increase functional β cell mass.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre5nXGBuindrVg90H21EOLACvtfcHk0lgfZTE4OXIv9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1WisLzP&md5=92dc7241ee0f258a91351b6bffd941d8</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.4161%2Fcc.29250&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.29250%26sid%3Dliteratum%253Aachs%26aulast%3DRachdi%26aufirst%3DL.%26aulast%3DKariyawasam%26aufirst%3DD.%26aulast%3DA%25C3%25AFello%26aufirst%3DV.%26aulast%3DHerault%26aufirst%3DY.%26aulast%3DJanel%26aufirst%3DN.%26aulast%3DDelabar%26aufirst%3DJ.-M.%26aulast%3DPolak%26aufirst%3DM.%26aulast%3DScharfmann%26aufirst%3DR.%26atitle%3DDyrk1A%2520induces%2520pancreatic%2520%25CE%25B2%2520cell%2520mass%2520expansion%2520and%2520improves%2520glucose%2520tolerance%26jtitle%3DCell%2520Cycle%26date%3D2014%26volume%3D13%26spage%3D2221%26epage%3D2229%26doi%3D10.4161%2Fcc.29250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soundararajan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roos, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kettenbach, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerber, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eswaran, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span> <span> </span><span class="NLM_article-title">Structures of down syndrome kinases, dyrks, reveal mechanisms of kinase activation and substrate recognition</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">986</span>– <span class="NLM_lpage">996</span>, <span class="refDoi"> DOI: 10.1016/j.str.2013.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1016%2Fj.str.2013.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=23665168" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsVygtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=986-996&author=M.+Soundararajanauthor=A.+K.+Roosauthor=P.+Savitskyauthor=P.+Filippakopoulosauthor=A.+N.+Kettenbachauthor=J.+V.+Olsenauthor=S.+A.+Gerberauthor=J.+Eswaranauthor=S.+Knappauthor=J.+M.+Elkins&title=Structures+of+down+syndrome+kinases%2C+dyrks%2C+reveal+mechanisms+of+kinase+activation+and+substrate+recognition&doi=10.1016%2Fj.str.2013.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of Down Syndrome Kinases, DYRKs, Reveal Mechanisms of Kinase Activation and Substrate Recognition</span></div><div class="casAuthors">Soundararajan, Meera; Roos, Annette K.; Savitsky, Pavel; Filippakopoulos, Panagis; Kettenbach, Arminja N.; Olsen, Jesper V.; Gerber, Scott A.; Eswaran, Jeyanthy; Knapp, Stefan; Elkins, Jonathan M.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">986-996</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Dual-specificity tyrosine-(Y)-phosphorylation-regulated kinases (DYRKs) play key roles in brain development, regulation of splicing, and apoptosis, and are potential drug targets for neurodegenerative diseases and cancer.  We present crystal structures of one representative member of each DYRK subfamily: DYRK1A with an ATP-mimetic inhibitor and consensus peptide, and DYRK2 including NAPA and DH (DYRK homol.) box regions.  The current activation model suggests that DYRKs are Ser/Thr kinases that only autophosphorylate the second tyrosine of the activation loop YxY motif during protein translation.  The structures explain the roles of this tyrosine and of the DH box in DYRK activation and provide a structural model for DYRK substrate recognition.  Phosphorylation of a library of naturally occurring peptides identified substrate motifs that lack proline in the P+1 position, suggesting that DYRK1A is not a strictly proline-directed kinase.  Our data also show that DYRK1A wild-type and Y321F mutant retain tyrosine autophosphorylation activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDShaymgKcfLVg90H21EOLACvtfcHk0lgfZTE4OXIv9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsVygtL0%253D&md5=f101707a987083c7d3d637b39332db35</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2013.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2013.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DSoundararajan%26aufirst%3DM.%26aulast%3DRoos%26aufirst%3DA.%2BK.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKettenbach%26aufirst%3DA.%2BN.%26aulast%3DOlsen%26aufirst%3DJ.%2BV.%26aulast%3DGerber%26aufirst%3DS.%2BA.%26aulast%3DEswaran%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26atitle%3DStructures%2520of%2520down%2520syndrome%2520kinases%252C%2520dyrks%252C%2520reveal%2520mechanisms%2520of%2520kinase%2520activation%2520and%2520substrate%2520recognition%26jtitle%3DStructure%26date%3D2013%26volume%3D21%26spage%3D986%26epage%3D996%26doi%3D10.1016%2Fj.str.2013.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weber, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sipos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paczal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balint, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kun, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foloppe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dokurno, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massey, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walmsley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benwell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edmonds, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demarles, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruno, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burbridge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruzalegui, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotschy, A.</span></span> <span> </span><span class="NLM_article-title">Structure-Guided Discovery of Potent and Selective DYRK1A Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.1c00023</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.1c00023" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2021&author=C.+Weberauthor=M.+Siposauthor=A.+Paczalauthor=B.+Balintauthor=V.+Kunauthor=N.+Foloppeauthor=P.+Dokurnoauthor=A.+J.+Masseyauthor=D.+L.+Walmsleyauthor=R.+E.+Hubbardauthor=J.+Murrayauthor=K.+Benwellauthor=T.+Edmondsauthor=D.+Demarlesauthor=A.+Brunoauthor=M.+Burbridgeauthor=F.+Cruzaleguiauthor=A.+Kotschy&title=Structure-Guided+Discovery+of+Potent+and+Selective+DYRK1A+Inhibitors&doi=10.1021%2Facs.jmedchem.1c00023"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.1c00023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.1c00023%26sid%3Dliteratum%253Aachs%26aulast%3DWeber%26aufirst%3DC.%26aulast%3DSipos%26aufirst%3DM.%26aulast%3DPaczal%26aufirst%3DA.%26aulast%3DBalint%26aufirst%3DB.%26aulast%3DKun%26aufirst%3DV.%26aulast%3DFoloppe%26aufirst%3DN.%26aulast%3DDokurno%26aufirst%3DP.%26aulast%3DMassey%26aufirst%3DA.%2BJ.%26aulast%3DWalmsley%26aufirst%3DD.%2BL.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DBenwell%26aufirst%3DK.%26aulast%3DEdmonds%26aufirst%3DT.%26aulast%3DDemarles%26aufirst%3DD.%26aulast%3DBruno%26aufirst%3DA.%26aulast%3DBurbridge%26aufirst%3DM.%26aulast%3DCruzalegui%26aufirst%3DF.%26aulast%3DKotschy%26aufirst%3DA.%26atitle%3DStructure-Guided%2520Discovery%2520of%2520Potent%2520and%2520Selective%2520DYRK1A%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2021%26doi%3D10.1021%2Facs.jmedchem.1c00023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ha, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krauthammer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halaban, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boggon, T. J.</span></span> <span> </span><span class="NLM_article-title">Type ii p21-activated kinases (paks) are regulated by an autoinhibitory pseudosubstrate</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">16107</span>– <span class="NLM_lpage">16112</span>, <span class="refDoi"> DOI: 10.1073/pnas.1214447109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1073%2Fpnas.1214447109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=22988085" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGhsL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=16107-16112&author=B.+H.+Haauthor=M.+J.+Davisauthor=C.+Chenauthor=H.+J.+Louauthor=J.+Gaoauthor=R.+Zhangauthor=M.+Krauthammerauthor=R.+Halabanauthor=J.+Schlessingerauthor=B.+E.+Turkauthor=T.+J.+Boggon&title=Type+ii+p21-activated+kinases+%28paks%29+are+regulated+by+an+autoinhibitory+pseudosubstrate&doi=10.1073%2Fpnas.1214447109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Type II p21-activated kinases (PAKs) are regulated by an autoinhibitory pseudosubstrate</span></div><div class="casAuthors">Ha, Byung Hak; Davis, Matthew J.; Chen, Catherine; Lou, Hua Jane; Gao, Jia; Zhang, Rong; Krauthammer, Michael; Halaban, Ruth; Schlessinger, Joseph; Turk, Benjamin E.; Boggon, Titus J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">16107-16112, S16107/1-S16107/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The type II p21-activated kinases (PAKs) are key effectors of RHO-family GTPases involved in cell motility, survival, and proliferation.  Using a structure-guided approach, we discovered that type II PAKs are regulated by an N-terminal autoinhibitory pseudosubstrate motif centered on a crit. proline residue, and that this regulation occurs independently of activation loop phosphorylation.  We detd. six X-ray crystal structures of either full-length PAK4 or its catalytic domain, that demonstrate the mol. basis for pseudosubstrate binding to the active state with phosphorylated activation loop.  We show that full-length PAK4 is constitutively autoinhibited, but mutation of the pseudo-substrate releases this inhibition and causes increased phosphorylation of the apoptotic regulation protein Bcl-2/Bcl-XL antagonist causing cell death and cellular morphol. changes.  We also find that PAK6 is regulated by the pseudosubstrate region, indicating a common type II PAK autoregulatory mechanism.  Finally, we find Src SH3, but not β-PIX SH3, can activate PAK4.  We provide a unique understanding for type II PAK regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreLSMEq29IlrVg90H21EOLACvtfcHk0lioqeOQZ19WrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGhsL3J&md5=65e13a76626af2ecb01509f70d36711e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1214447109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1214447109%26sid%3Dliteratum%253Aachs%26aulast%3DHa%26aufirst%3DB.%2BH.%26aulast%3DDavis%26aufirst%3DM.%2BJ.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DLou%26aufirst%3DH.%2BJ.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DKrauthammer%26aufirst%3DM.%26aulast%3DHalaban%26aufirst%3DR.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DTurk%26aufirst%3DB.%2BE.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26atitle%3DType%2520ii%2520p21-activated%2520kinases%2520%2528paks%2529%2520are%2520regulated%2520by%2520an%2520autoinhibitory%2520pseudosubstrate%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D16107%26epage%3D16112%26doi%3D10.1073%2Fpnas.1214447109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baurin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aboul-Ela, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barril, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drysdale, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dymock, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finch, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fromont, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmonite, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">Design and characterization of libraries of molecular fragments for use in nmr screening against protein targets</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">2157</span>– <span class="NLM_lpage">2166</span>, <span class="refDoi"> DOI: 10.1021/ci049806z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049806z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotlaru78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=2157-2166&author=N.+Baurinauthor=F.+Aboul-Elaauthor=X.+Barrilauthor=B.+Davisauthor=M.+Drysdaleauthor=B.+Dymockauthor=H.+Finchauthor=C.+Fromontauthor=C.+Richardsonauthor=H.+Simmoniteauthor=R.+E.+Hubbard&title=Design+and+characterization+of+libraries+of+molecular+fragments+for+use+in+nmr+screening+against+protein+targets&doi=10.1021%2Fci049806z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Characterization of Libraries of Molecular Fragments for Use in NMR Screening against Protein Targets</span></div><div class="casAuthors">Baurin, Nicolas; Aboul-Ela, Fareed; Barril, Xavier; Davis, Ben; Drysdale, Martin; Dymock, Brian; Finch, Harry; Fromont, Christophe; Richardson, Christine; Simmonite, Heather; Hubbard, Roderick E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2157-2166</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have designed four generations of a low mol. wt. fragment library for use in NMR-based screening against protein targets.  The library initially contained 723 fragments which were selected manually from the Available Chems. Directory.  A series of in silico filters and property calcns. were developed to automate the selection process, allowing a larger database of 1.79M available compds. to be searched for a further 357 compds. that were added to the library.  A kinase binding pharmacophore was then derived to select 174 kinase-focused fragments.  Finally, an addnl. 61 fragments were selected to increase the no. of different pharmacophores represented within the library.  All of the fragments added to the library passed quality checks to ensure they were suitable for the screening protocol, with appropriate soly., purity, chem. stability, and unambiguous NMR spectrum.  The successive generations of libraries have been characterized through anal. of structural properties (mol. wt., lipophilicity, polar surface area, no. of rotatable bonds, and hydrogen-bonding potential) and by analyzing their pharmacophoric complexity.  These calcns. have been used to compare the fragment libraries with a drug-like ref. set of compds. and a set of mols. that bind to protein active sites.  In addn., an anal. of the overall results of screening the library against the ATP binding site of two protein targets (HSP90 and CDK2) reveals different patterns of fragment binding, demonstrating that the approach can find selective compds. that discriminate between related binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo652Eyyn5Lc7Vg90H21EOLACvtfcHk0lioqeOQZ19WrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotlaru78%253D&md5=7376be270294dae6997609448513c5d7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fci049806z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049806z%26sid%3Dliteratum%253Aachs%26aulast%3DBaurin%26aufirst%3DN.%26aulast%3DAboul-Ela%26aufirst%3DF.%26aulast%3DBarril%26aufirst%3DX.%26aulast%3DDavis%26aufirst%3DB.%26aulast%3DDrysdale%26aufirst%3DM.%26aulast%3DDymock%26aufirst%3DB.%26aulast%3DFinch%26aufirst%3DH.%26aulast%3DFromont%26aufirst%3DC.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DSimmonite%26aufirst%3DH.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DDesign%2520and%2520characterization%2520of%2520libraries%2520of%2520molecular%2520fragments%2520for%2520use%2520in%2520nmr%2520screening%2520against%2520protein%2520targets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2004%26volume%3D44%26spage%3D2157%26epage%3D2166%26doi%3D10.1021%2Fci049806z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. B.</span></span> <span> </span><span class="NLM_article-title">Experiences in fragment-based lead discovery</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>493</i></span>,  <span class="NLM_fpage">509</span>– <span class="NLM_lpage">531</span>, <span class="refDoi"> DOI: 10.1016/b978-0-12-381274-2.00020-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1016%2FB978-0-12-381274-2.00020-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=21371604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslers70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=493&publication_year=2011&pages=509-531&author=R.+E.+Hubbardauthor=J.+B.+Murray&title=Experiences+in+fragment-based+lead+discovery&doi=10.1016%2Fb978-0-12-381274-2.00020-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Experiences in fragment-based lead discovery</span></div><div class="casAuthors">Hubbard, Roderick E.; Murray, James B.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">493</span>
        (<span class="NLM_cas:issue">Fragment-Based Drug Design</span>),
    <span class="NLM_cas:pages">509-531</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  This chapter summarizes the experience at Vernalis over the past decade in developing and applying fragment-based discovery methods across a range of different targets.  The emphasis will be on the practical aspects of the different biophys. techniques (surface plasmon resonance (SPR), differential scanning fluorimetry (DSF), isothermal titrn. calorimetry, NMR, and X-ray crystallog.) that can be used to identify fragments that bind to targets and a discussion of the criteria and strategies for selecting and evolving fragments to lead compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbWFWP7PylG7Vg90H21EOLACvtfcHk0lioqeOQZ19WrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslers70%253D&md5=358abcb0648a3482c6be40f5b3d04ff9</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-381274-2.00020-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-381274-2.00020-0%26sid%3Dliteratum%253Aachs%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DMurray%26aufirst%3DJ.%2BB.%26atitle%3DExperiences%2520in%2520fragment-based%2520lead%2520discovery%26jtitle%3DMethods%2520Enzymol.%26date%3D2011%26volume%3D493%26spage%3D509%26epage%3D531%26doi%3D10.1016%2Fb978-0-12-381274-2.00020-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drysdale, M.</span></span> <span> </span><span class="NLM_article-title">The seeds approach: Integrating fragments into drug discovery</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1568</span>– <span class="NLM_lpage">1581</span>, <span class="refDoi"> DOI: 10.2174/156802607782341109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.2174%2F156802607782341109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=17979768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVOrur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=1568-1581&author=R.+Hubbardauthor=B.+Davisauthor=I.+Chenauthor=M.+Drysdale&title=The+seeds+approach%3A+Integrating+fragments+into+drug+discovery&doi=10.2174%2F156802607782341109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The SeeDs approach: integrating fragments into drug discovery</span></div><div class="casAuthors">Hubbard, Roderick E.; Davis, Ben; Chen, Ijen; Drysdale, Martin J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1568-1581</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Finding novel compds. as starting points for optimization is a major challenge in drug discovery research.  Fragment-based methods have emerged in the past ten years as an effective way to sample chem. diversity with a limited no. of low mol. wt. compds.  The structures of the fragments(s) binding to the protein can then be used to design new compds. with increased affinity, specificity and novelty.  This article describes the Vernalis approach to fragment based drug discovery, called SeeDs (Structural exploitation of exptl. Drug startpoints).  The approach includes the design of a fragment library, identification of fragments that bind competitively to a target by ligand-based NMR techniques and protein crystal structures to characterize binding.  Fragments that bind are then evolved to hits, either by growing the fragment or by combining structural features from a no. of compds.  The process is illustrated with examples from recent medicinal chem. programs to discover compds. against the oncol. targets Hsp90 and PDK1.  In addn., we summarize our experience with using mol. docking calcns. to predict fragment binding and anecdotes on the selectivity and binding modes for fragments seen against a range of targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox6z6oFlEi67Vg90H21EOLACvtfcHk0ljurZoR-o0lqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVOrur7J&md5=6bfa0109e33c2c23cc4c5a5bcd1ed12f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.2174%2F156802607782341109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802607782341109%26sid%3Dliteratum%253Aachs%26aulast%3DHubbard%26aufirst%3DR.%26aulast%3DDavis%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DI.%26aulast%3DDrysdale%26aufirst%3DM.%26atitle%3DThe%2520seeds%2520approach%253A%2520Integrating%2520fragments%2520into%2520drug%2520discovery%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2007%26volume%3D7%26spage%3D1568%26epage%3D1581%26doi%3D10.2174%2F156802607782341109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alex, A.</span></span> <span> </span><span class="NLM_article-title">Ligand efficiency: A useful metric for lead selection</span>. <i>Drug Discov. Today</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">430</span>– <span class="NLM_lpage">431</span>, <span class="refDoi"> DOI: 10.1016/s1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+A+useful+metric+for+lead+selection&doi=10.1016%2Fs1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2ebskXFeozxHJrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520A%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discov.%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2Fs1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">Lessons for fragment library design: Analysis of output from multiple screening campaigns</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">620</span>, <span class="refDoi"> DOI: 10.1007/s10822-009-9280-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1007%2Fs10822-009-9280-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=19495994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD1MXot1Gmtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=603-620&author=I.-J.+Chenauthor=R.+E.+Hubbard&title=Lessons+for+fragment+library+design%3A+Analysis+of+output+from+multiple+screening+campaigns&doi=10.1007%2Fs10822-009-9280-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons for fragment library design: analysis of output from multiple screening campaigns</span></div><div class="casAuthors">Chen, I.-Jen; Hubbard, Roderick E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">603-620</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Over the past 8 years, we have developed, refined and applied a fragment based discovery approach to a range of protein targets.  Here we report computational analyses of various aspects of our fragment library and the results obtained for fragment screening.  We reinforce the finding of others that the exptl. obsd. hit rate for screening fragments can be related to a computationally defined druggability index for the target.  In general, the physicochem. properties of the fragment hits display the same profile as the library, as is expected for a truly diverse library which probes the relevant chem. space.  An anal. of the fragment hits against various protein classes has shown that the physicochem. properties of the fragments are complementary to the properties of the target binding site.  The effectiveness of some fragments appears to be achieved by an appropriate mix of pharmacophore features and enhanced aromaticity, with hydrophobic interactions playing an important role.  The anal. emphasizes that it is possible to identify small fragments that are specific for different binding sites.  To conclude, we discuss how the results could inform further development and improvement of our fragment library.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBo4yzAG3RSbVg90H21EOLACvtfcHk0ljurZoR-o0lqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXot1Gmtbc%253D&md5=f544180cae108c2d7365037388df0262</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1007%2Fs10822-009-9280-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-009-9280-5%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DI.-J.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DLessons%2520for%2520fragment%2520library%2520design%253A%2520Analysis%2520of%2520output%2520from%2520multiple%2520screening%2520campaigns%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2009%26volume%3D23%26spage%3D603%26epage%3D620%26doi%3D10.1007%2Fs10822-009-9280-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hajduk, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huth, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Druggability indices for protein targets derived from nmr-based screening data</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">2518</span>– <span class="NLM_lpage">2525</span>, <span class="refDoi"> DOI: 10.1021/jm049131r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049131r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhslajtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2518-2525&author=P.+J.+Hajdukauthor=J.+R.+Huthauthor=S.+W.+Fesik&title=Druggability+indices+for+protein+targets+derived+from+nmr-based+screening+data&doi=10.1021%2Fjm049131r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Indices for Protein Targets Derived from NMR-Based Screening Data</span></div><div class="casAuthors">Hajduk, Philip J.; Huth, Jeffrey R.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2518-2525</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An anal. of heteronuclear-NMR-based screening data is used to derive relationships between the ability of small mols. to bind to a protein and various parameters that describe the protein binding site.  It is found that a simple model including terms for polar and apolar surface area, surface complexity, and pocket dimensions accurately predicts the exptl. screening hit rates with an R2 of 0.72, an adjusted R2 of 0.65, and a leave-one-out Q2 of 0.56.  Application of the model to predict the druggability of protein targets not used in the training set correctly classified 94% of the proteins for which high-affinity, noncovalent, druglike leads have been reported.  In addn. to understanding the pocket characteristics that contribute to high-affinity binding, the relationships that have been defined allow for quant. comparative analyses of protein binding sites for use in target assessment and validation, virtual ligand screening, and structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0d3V84DZMK7Vg90H21EOLACvtfcHk0ljurZoR-o0lqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhslajtr0%253D&md5=134793e571e8ef1858a734c1a15ce343</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm049131r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049131r%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DHuth%26aufirst%3DJ.%2BR.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDruggability%2520indices%2520for%2520protein%2520targets%2520derived%2520from%2520nmr-based%2520screening%2520data%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2518%26epage%3D2525%26doi%3D10.1021%2Fjm049131r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span> <span> </span><span class="NLM_article-title">Identifying and characterizing binding sites and assessing druggability</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1021/ci800324m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800324m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=377-389&author=T.+A.+Halgren&title=Identifying+and+characterizing+binding+sites+and+assessing+druggability&doi=10.1021%2Fci800324m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying and Characterizing Binding Sites and Assessing Druggability</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">377-389</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Identification and characterization of binding sites is key in the process of structure-based drug design.  In some cases there may not be any information about the binding site for a target of interest.  In other cases, a putative binding site has been identified by computational or exptl. means, but the druggability of the target is not known.  Even when a site for a given target is known, it may be desirable to find addnl. sites whose targeting could produce a desired biol. response.  A new program, called SiteMap, is presented for identifying and analyzing binding sites and for predicting target druggability.  In a large-scale validation, SiteMap correctly identifies the known binding site as the top-ranked site in 86% of the cases, with best results (>98%) coming for sites that bind ligands with subnanomolar affinity.  In addn., a modified version of the score employed for binding-site identification allows SiteMap to accurately classify the druggability of proteins as measured by their ability to bind passively absorbed small mols. tightly.  In characterizing binding sites, SiteMap provides quant. and graphical information that can help guide efforts to critically assess virtual hits in a lead-discovery application or to modify ligand structure to enhance potency or improve phys. properties in a lead-optimization context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfAt0-gvcXDLVg90H21EOLACvtfcHk0lhW_MZkf4FCQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D&md5=f03c7b27e30ae03d10e4127ed1f1ed3c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fci800324m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800324m%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DIdentifying%2520and%2520characterizing%2520binding%2520sites%2520and%2520assessing%2520druggability%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D377%26epage%3D389%26doi%3D10.1021%2Fci800324m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ogawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nonaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohnishi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiramatsu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kii, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onogi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibuya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosoya, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagiwara, M.</span></span> <span> </span><span class="NLM_article-title">Development of a novel selective inhibitor of the down syndrome-related kinase dyrk1a</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">86</span>, <span class="refDoi"> DOI: 10.1038/ncomms1090</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1038%2Fncomms1090" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=20981014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A280%3ADC%252BC3cbislehtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=86&author=Y.+Ogawaauthor=Y.+Nonakaauthor=T.+Gotoauthor=E.+Ohnishiauthor=T.+Hiramatsuauthor=I.+Kiiauthor=M.+Yoshidaauthor=T.+Ikuraauthor=H.+Onogiauthor=H.+Shibuyaauthor=T.+Hosoyaauthor=N.+Itoauthor=M.+Hagiwara&title=Development+of+a+novel+selective+inhibitor+of+the+down+syndrome-related+kinase+dyrk1a&doi=10.1038%2Fncomms1090"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a novel selective inhibitor of the Down syndrome-related kinase Dyrk1A</span></div><div class="casAuthors">Ogawa Yasushi; Nonaka Yosuke; Goto Toshiyasu; Ohnishi Eriko; Hiramatsu Toshiyuki; Kii Isao; Yoshida Miyo; Ikura Teikichi; Onogi Hiroshi; Shibuya Hiroshi; Hosoya Takamitsu; Ito Nobutoshi; Hagiwara Masatoshi</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">86</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Dyrk1A (dual-specificity tyrosine-(Y)-phosphorylation-regulated kinase 1A) is a serine/threonine kinase essential for brain development and function, and its excessive activity is considered a pathogenic factor in Down syndrome.  The development of potent, selective inhibitors of Dyrk1A would help to elucidate the molecular mechanisms of normal and diseased brains, and may provide a new lead compound for molecular-targeted drug discovery.  Here, we report a novel Dyrk1A inhibitor, INDY, a benzothiazole derivative showing a potent ATP-competitive inhibitory effect with IC(50) and K(i) values of 0.24 and 0.18 μM, respectively.  X-ray crystallography of the Dyrk1A/INDY complex revealed the binding of INDY in the ATP pocket of the enzyme.  INDY effectively reversed the aberrant tau-phosphorylation and rescued the repressed NFAT (nuclear factor of activated T cell) signalling induced by Dyrk1A overexpression.  Importantly, proINDY, a prodrug of INDY, effectively recovered Xenopus embryos from head malformation induced by Dyrk1A overexpression, resulting in normally developed embryos and demonstrating the utility of proINDY in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTOCqf6rsXbPtenUz0lpNqnfW6udTcc2eamD-Q5-YajNrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cbislehtw%253D%253D&md5=ac1d5972bec4085de101bb86194babd1</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fncomms1090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms1090%26sid%3Dliteratum%253Aachs%26aulast%3DOgawa%26aufirst%3DY.%26aulast%3DNonaka%26aufirst%3DY.%26aulast%3DGoto%26aufirst%3DT.%26aulast%3DOhnishi%26aufirst%3DE.%26aulast%3DHiramatsu%26aufirst%3DT.%26aulast%3DKii%26aufirst%3DI.%26aulast%3DYoshida%26aufirst%3DM.%26aulast%3DIkura%26aufirst%3DT.%26aulast%3DOnogi%26aufirst%3DH.%26aulast%3DShibuya%26aufirst%3DH.%26aulast%3DHosoya%26aufirst%3DT.%26aulast%3DIto%26aufirst%3DN.%26aulast%3DHagiwara%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520a%2520novel%2520selective%2520inhibitor%2520of%2520the%2520down%2520syndrome-related%2520kinase%2520dyrk1a%26jtitle%3DNat.%2520Commun.%26date%3D2010%26volume%3D1%26spage%3D86%26doi%3D10.1038%2Fncomms1090" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akue-Gedu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debiton, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferandin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meijer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prudhomme, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anizon, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreau, P.</span></span> <span> </span><span class="NLM_article-title">Synthesis and biological activities of aminopyrimidyl-indoles structurally related to meridianins</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">4420</span>– <span class="NLM_lpage">4424</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2009.05.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1016%2Fj.bmc.2009.05.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=19477650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1Gru7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=4420-4424&author=R.+Akue-Geduauthor=E.+Debitonauthor=Y.+Ferandinauthor=L.+Meijerauthor=M.+Prudhommeauthor=F.+Anizonauthor=P.+Moreau&title=Synthesis+and+biological+activities+of+aminopyrimidyl-indoles+structurally+related+to+meridianins&doi=10.1016%2Fj.bmc.2009.05.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and biological activities of aminopyrimidyl-indoles structurally related to meridianins</span></div><div class="casAuthors">Akue-Gedu, Rufine; Debiton, Eric; Ferandin, Yoan; Meijer, Laurent; Prudhomme, Michelle; Anizon, Fabrice; Moreau, Pascale</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4420-4424</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The synthesis of new meridianin derivs. substituted at the C-5 position of the 2-aminopyrimidine ring by various aryl groups and substituted or not by a Me group on the indole nitrogen is described.  These compds. were tested for their kinase inhibitory potencies toward five kinases (CDK5/p25, CK1δ/ε, GSK-3α/β, Dyrk1A and Erk2) as well as their in vitro antiproliferative activities toward a human fibroblast primary culture and two human solid cancer cell lines (MCF-7 and PA 1).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYufDKqEQZNrVg90H21EOLACvtfcHk0lhW_MZkf4FCQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1Gru7c%253D&md5=f14ea49d5c54c32f5a6be1d9214d4bc7</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2009.05.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2009.05.017%26sid%3Dliteratum%253Aachs%26aulast%3DAkue-Gedu%26aufirst%3DR.%26aulast%3DDebiton%26aufirst%3DE.%26aulast%3DFerandin%26aufirst%3DY.%26aulast%3DMeijer%26aufirst%3DL.%26aulast%3DPrudhomme%26aufirst%3DM.%26aulast%3DAnizon%26aufirst%3DF.%26aulast%3DMoreau%26aufirst%3DP.%26atitle%3DSynthesis%2520and%2520biological%2520activities%2520of%2520aminopyrimidyl-indoles%2520structurally%2520related%2520to%2520meridianins%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2009%26volume%3D17%26spage%3D4420%26epage%3D4424%26doi%3D10.1016%2Fj.bmc.2009.05.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tazarki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeinyeh, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esvan, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schröder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joerger, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khiari, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Josselin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baratte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bach, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruchaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anizon, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giraud, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreau, P.</span></span> <span> </span><span class="NLM_article-title">New pyrido[3,4-g]quinazoline derivatives as clk1 and dyrk1a inhibitors: Synthesis, biological evaluation and binding mode analysis</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>166</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.01.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1016%2Fj.ejmech.2019.01.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=30731399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC1MXis1aqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2019&pages=304-317&author=H.+Tazarkiauthor=W.+Zeinyehauthor=Y.+J.+Esvanauthor=S.+Knappauthor=D.+Chatterjeeauthor=M.+Schr%C3%B6derauthor=A.+C.+Joergerauthor=J.+Khiariauthor=B.+Josselinauthor=B.+Baratteauthor=S.+Bachauthor=S.+Ruchaudauthor=F.+Anizonauthor=F.+Giraudauthor=P.+Moreau&title=New+pyrido%5B3%2C4-g%5Dquinazoline+derivatives+as+clk1+and+dyrk1a+inhibitors%3A+Synthesis%2C+biological+evaluation+and+binding+mode+analysis&doi=10.1016%2Fj.ejmech.2019.01.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">New pyrido[3,4-g]quinazoline derivatives as CLK1 and DYRK1A inhibitors: synthesis, biological evaluation and binding mode analysis</span></div><div class="casAuthors">Tazarki, Helmi; Zeinyeh, Wael; Esvan, Yannick J.; Knapp, Stefan; Chatterjee, Deep; Schroder, Martin; Joerger, Andreas C.; Khiari, Jameleddine; Josselin, Beatrice; Baratte, Blandine; Bach, Stephane; Ruchaud, Sandrine; Anizon, Fabrice; Giraud, Francis; Moreau, Pascale</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">166</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-317</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Synthesis of new pyrido[3,4-g]quinazoline derivs. I [R = (CH2)2-NMe2, (CH2)2-morpholin-1-yl, (CH2)3-NMe2, etc.; R1 = NH2, NO2] and the evaluation of the inhibitory potencies of these compds. toward CDK5, CK1, GSK3, CLK1 and DYRK1A were described.  Introduction of aminoalkylamino groups at the 2-position resulted in several compds. with low nanomolar affinity and selective inhibition of CLK1 and/or DYRK1A.  Their evaluation on several immortalized or cancerous cell lines showed varying degree of cell viability redn.  Co-crystal structures of CLK1 with two of the most potent compds. I [R = Me, R1 = NO2; R = (CH2)3-morpholin-1-yl, R2 = NH2] revealed two alternative binding modes of the pyrido[3,4-g]quinazoline scaffold that can be exploited for future inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK9VnCqI3-v7Vg90H21EOLACvtfcHk0ljkbu3Bwq-QRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXis1aqsLk%253D&md5=d572d06711c1ecdf401051af38655449</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.01.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.01.052%26sid%3Dliteratum%253Aachs%26aulast%3DTazarki%26aufirst%3DH.%26aulast%3DZeinyeh%26aufirst%3DW.%26aulast%3DEsvan%26aufirst%3DY.%2BJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DChatterjee%26aufirst%3DD.%26aulast%3DSchr%25C3%25B6der%26aufirst%3DM.%26aulast%3DJoerger%26aufirst%3DA.%2BC.%26aulast%3DKhiari%26aufirst%3DJ.%26aulast%3DJosselin%26aufirst%3DB.%26aulast%3DBaratte%26aufirst%3DB.%26aulast%3DBach%26aufirst%3DS.%26aulast%3DRuchaud%26aufirst%3DS.%26aulast%3DAnizon%26aufirst%3DF.%26aulast%3DGiraud%26aufirst%3DF.%26aulast%3DMoreau%26aufirst%3DP.%26atitle%3DNew%2520pyrido%255B3%252C4-g%255Dquinazoline%2520derivatives%2520as%2520clk1%2520and%2520dyrk1a%2520inhibitors%253A%2520Synthesis%252C%2520biological%2520evaluation%2520and%2520binding%2520mode%2520analysis%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D166%26spage%3D304%26epage%3D317%26doi%3D10.1016%2Fj.ejmech.2019.01.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tahtouh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soundararajan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durieu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delhommel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oberholzer, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearl, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carreaux, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bazureau, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meijer, L.</span></span> <span> </span><span class="NLM_article-title">Selectivity, cocrystal structures, and neuroprotective properties of leucettines, a family of protein kinase inhibitors derived from the marine sponge alkaloid leucettamine b</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9312</span>– <span class="NLM_lpage">9330</span>, <span class="refDoi"> DOI: 10.1021/jm301034u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301034u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2mu7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9312-9330&author=T.+Tahtouhauthor=J.+M.+Elkinsauthor=P.+Filippakopoulosauthor=M.+Soundararajanauthor=G.+Burgyauthor=E.+Durieuauthor=C.+Cochetauthor=R.+S.+Schmidauthor=D.+C.+Loauthor=F.+Delhommelauthor=A.+E.+Oberholzerauthor=L.+H.+Pearlauthor=F.+Carreauxauthor=J.-P.+Bazureauauthor=S.+Knappauthor=L.+Meijer&title=Selectivity%2C+cocrystal+structures%2C+and+neuroprotective+properties+of+leucettines%2C+a+family+of+protein+kinase+inhibitors+derived+from+the+marine+sponge+alkaloid+leucettamine+b&doi=10.1021%2Fjm301034u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Selectivity, Cocrystal Structures, and Neuroprotective Properties of Leucettines, a Family of Protein Kinase Inhibitors Derived from the Marine Sponge Alkaloid Leucettamine B</span></div><div class="casAuthors">Tahtouh, Tania; Elkins, Jonathan M.; Filippakopoulos, Panagis; Soundararajan, Meera; Burgy, Guillaume; Durieu, Emilie; Cochet, Claude; Schmid, Ralf S.; Lo, Donald C.; Delhommel, Florent; Oberholzer, Anselm E.; Pearl, Laurence H.; Carreaux, Francois; Bazureau, Jean-Pierre; Knapp, Stefan; Meijer, Laurent</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9312-9330</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DYRKs (dual specificity, tyrosine phosphorylation regulated kinases) and CLKs (cdc2-like kinases) are implicated in the onset and development of Alzheimer's disease and Down syndrome.  The marine sponge alkaloid leucettamine B was recently identified as an inhibitor of DYRKs/CLKs.  Synthesis of analogs (leucettines) led to an optimized product, leucettine L41.  Leucettines were cocrystd. with DYRK1A, DYRK2, CLK3, PIM1, and GSK-3β.  The selectivity of L41 was studied by activity and interaction assays of recombinant kinases and affinity chromatog. and competition affinity assays.  These approaches revealed unexpected potential secondary targets such as CK2, SLK, and the lipid kinase PIKfyve/Vac14.  L41 displayed neuroprotective effects on glutamate-induced HT22 cell death.  L41 also reduced amyloid precursor protein-induced cell death in cultured rat brain slices.  The unusual multitarget selectivity of leucettines may account for their neuroprotective effects.  This family of kinase inhibitors deserves further optimization as potential therapeutics against neurodegenerative diseases such as Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOhwr44JiuCrVg90H21EOLACvtfcHk0ljkbu3Bwq-QRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2mu7vF&md5=a1749c347c2e8e0409d8fdd475dc70f9</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm301034u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301034u%26sid%3Dliteratum%253Aachs%26aulast%3DTahtouh%26aufirst%3DT.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DSoundararajan%26aufirst%3DM.%26aulast%3DBurgy%26aufirst%3DG.%26aulast%3DDurieu%26aufirst%3DE.%26aulast%3DCochet%26aufirst%3DC.%26aulast%3DSchmid%26aufirst%3DR.%2BS.%26aulast%3DLo%26aufirst%3DD.%2BC.%26aulast%3DDelhommel%26aufirst%3DF.%26aulast%3DOberholzer%26aufirst%3DA.%2BE.%26aulast%3DPearl%26aufirst%3DL.%2BH.%26aulast%3DCarreaux%26aufirst%3DF.%26aulast%3DBazureau%26aufirst%3DJ.-P.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMeijer%26aufirst%3DL.%26atitle%3DSelectivity%252C%2520cocrystal%2520structures%252C%2520and%2520neuroprotective%2520properties%2520of%2520leucettines%252C%2520a%2520family%2520of%2520protein%2520kinase%2520inhibitors%2520derived%2520from%2520the%2520marine%2520sponge%2520alkaloid%2520leucettamine%2520b%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9312%26epage%3D9330%26doi%3D10.1021%2Fjm301034u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Falke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loaëc, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lozach, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abu Jhaisha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meijer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunick, C.</span></span> <span> </span><span class="NLM_article-title">10-iodo-11h-indolo[3,2-c]quinoline-6-carboxylic acids are selective inhibitors of dyrk1a</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">3131</span>– <span class="NLM_lpage">3143</span>, <span class="refDoi"> DOI: 10.1021/jm501994d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501994d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjs1GhtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=3131-3143&author=H.+Falkeauthor=A.+Chaikuadauthor=A.+Beckerauthor=N.+Loa%C3%ABcauthor=O.+Lozachauthor=S.+Abu+Jhaishaauthor=W.+Beckerauthor=P.+G.+Jonesauthor=L.+Preuauthor=K.+Baumannauthor=S.+Knappauthor=L.+Meijerauthor=C.+Kunick&title=10-iodo-11h-indolo%5B3%2C2-c%5Dquinoline-6-carboxylic+acids+are+selective+inhibitors+of+dyrk1a&doi=10.1021%2Fjm501994d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">10-Iodo-11H-indolo[3,2-c]quinoline-6-carboxylic Acids Are Selective Inhibitors of DYRK1A</span></div><div class="casAuthors">Falke, Hannes; Chaikuad, Apirat; Becker, Anja; Loaec, Nadege; Lozach, Olivier; Abu Jhaisha, Samira; Becker, Walter; Jones, Peter G.; Preu, Lutz; Baumann, Knut; Knapp, Stefan; Meijer, Laurent; Kunick, Conrad</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3131-3143</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The protein kinase DYRK1A has been suggested to act as one of the intracellular regulators contributing to neurol. alterations found in individuals with Down syndrome.  For an assessment of the role of DYRK1A, selective synthetic inhibitors are valuable pharmacol. tools.  However, the DYRK1A inhibitors described in the literature so far either are not sufficiently selective or have not been tested against closely related kinases from the DYRK and the CLK protein kinase families.  The aim of this study was the identification of DYRK1A inhibitors exhibiting selectivity vs. the structurally and functionally closely related DYRK and CLK isoforms.  Structure modification of the screening hit 11H-indolo[3,2-c]quinoline-6-carboxylic acid revealed structure-activity relationships for kinase inhibition and enabled the design of 10-iodo-substituted derivs. as very potent DYRK1A inhibitors with considerable selectivity against CLKs.  X-ray structure detn. of three 11H-indolo[3,2-c]quinoline-6-carboxylic acids cocrystd. with DYRK1A confirmed the predicted binding mode within the ATP binding site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpe4rHbgYJUsbVg90H21EOLACvtfcHk0ljkbu3Bwq-QRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjs1GhtrY%253D&md5=70efbb6ad590a1240764ccc78af019ec</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm501994d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501994d%26sid%3Dliteratum%253Aachs%26aulast%3DFalke%26aufirst%3DH.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DLoa%25C3%25ABc%26aufirst%3DN.%26aulast%3DLozach%26aufirst%3DO.%26aulast%3DAbu%2BJhaisha%26aufirst%3DS.%26aulast%3DBecker%26aufirst%3DW.%26aulast%3DJones%26aufirst%3DP.%2BG.%26aulast%3DPreu%26aufirst%3DL.%26aulast%3DBaumann%26aufirst%3DK.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMeijer%26aufirst%3DL.%26aulast%3DKunick%26aufirst%3DC.%26atitle%3D10-iodo-11h-indolo%255B3%252C2-c%255Dquinoline-6-carboxylic%2520acids%2520are%2520selective%2520inhibitors%2520of%2520dyrk1a%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D3131%26epage%3D3143%26doi%3D10.1021%2Fjm501994d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Czarna, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zelencova, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kildalsen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seternes, O. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engh, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothweiler, U.</span></span> <span> </span><span class="NLM_article-title">Novel scaffolds for dual specificity tyrosine-phosphorylation-regulated kinase (dyrk1a) inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">7560</span>– <span class="NLM_lpage">7572</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01847</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01847" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVygtrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=7560-7572&author=A.+Czarnaauthor=J.+Wangauthor=D.+Zelencovaauthor=Y.+Liuauthor=X.+Dengauthor=H.+G.+Choiauthor=T.+Zhangauthor=W.+Zhouauthor=J.+W.+Changauthor=H.+Kildalsenauthor=O.+M.+Seternesauthor=N.+S.+Grayauthor=R.+A.+Enghauthor=U.+Rothweiler&title=Novel+scaffolds+for+dual+specificity+tyrosine-phosphorylation-regulated+kinase+%28dyrk1a%29+inhibitors&doi=10.1021%2Facs.jmedchem.7b01847"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Scaffolds for Dual Specificity Tyrosine-Phosphorylation-Regulated Kinase (DYRK1A) Inhibitors</span></div><div class="casAuthors">Czarna, Anna; Wang, Jinhua; Zelencova, Diana; Liu, Yao; Deng, Xianming; Choi, Hwan Geun; Zhang, Tinghu; Zhou, Wenjun; Chang, Jae Won; Kildalsen, Hanne; Seternes, Ole Morten; Gray, Nathanael S.; Engh, Richard A.; Rothweiler, Ulli</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7560-7572</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DYRK1A is one of five members of the dual-specificity tyrosine (Y) phosphorylation-regulated kinase (DYRK) family.  The DYRK1A gene is located in the Down syndrome crit. region and regulates cellular processes related to proliferation and differentiation of neuronal progenitor cells during early development.  This has focused research on its role in neuronal degenerative diseases, including Alzheimer's and Down syndrome.  Recent studies have also shown a possible role of DYRK1A in diabetes.  Here we report a variety of scaffolds not generally known for DYRK1A inhibition, demonstrating their effects in in vitro assays and also in cell cultures.  These inhibitors effectively block the tau phosphorylation that is a hallmark of Alzheimer's disease.  The crystal structures of these inhibitors support the design of optimized and novel therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqC4Uq_4mv-grVg90H21EOLACvtfcHk0lgVE40m51wQCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVygtrjF&md5=c4b55fc7510b9f9d80726933b32be508</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01847%26sid%3Dliteratum%253Aachs%26aulast%3DCzarna%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZelencova%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DChang%26aufirst%3DJ.%2BW.%26aulast%3DKildalsen%26aufirst%3DH.%26aulast%3DSeternes%26aufirst%3DO.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DEngh%26aufirst%3DR.%2BA.%26aulast%3DRothweiler%26aufirst%3DU.%26atitle%3DNovel%2520scaffolds%2520for%2520dual%2520specificity%2520tyrosine-phosphorylation-regulated%2520kinase%2520%2528dyrk1a%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D7560%26epage%3D7572%26doi%3D10.1021%2Facs.jmedchem.7b01847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fruit, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couly, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhansali, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rammohan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindberg, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crispino, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meijer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besson, T.</span></span> <span> </span><span class="NLM_article-title">Biological characterization of 8-cyclopropyl-2-(pyridin-3-yl)thiazolo[5,4-f]quinazolin-9(8h)-one, a promising inhibitor of dyrk1a</span>. <i>Pharmaceuticals</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">185</span>, <span class="refDoi"> DOI: 10.3390/ph12040185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.3390%2Fph12040185" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslalt7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2019&pages=185&author=C.+Fruitauthor=F.+Coulyauthor=R.+Bhansaliauthor=M.+Rammohanauthor=M.+F.+Lindbergauthor=J.+D.+Crispinoauthor=L.+Meijerauthor=T.+Besson&title=Biological+characterization+of+8-cyclopropyl-2-%28pyridin-3-yl%29thiazolo%5B5%2C4-f%5Dquinazolin-9%288h%29-one%2C+a+promising+inhibitor+of+dyrk1a&doi=10.3390%2Fph12040185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Biological characterization of 8-cyclopropyl-2-(pyridin-3-yl)thiazolo[5,4-f]quinazolin-9(8H)-one, a promising inhibitor of DYRK1A</span></div><div class="casAuthors">Fruit, Corinne; Couly, Florence; Bhansali, Rahul; Rammohan, Malini; Lindberg, Mattias F.; Crispino, John D.; Meijer, Laurent; Besson, Thierry</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceuticals</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">185</span>CODEN:
                <span class="NLM_cas:coden">PHARH2</span>;
        ISSN:<span class="NLM_cas:issn">1424-8247</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Dual-specificity tyrosine phosphorylation-regulated kinases (DYRKs) hyperactivity has been linked to the development of a no. of human malignancies.  DYRK1A is the most studied family member, and the discovery of novel specific inhibitors is attracting considerable interest.  The 8-cyclopropyl-2(pyridin-3-yl)thiazolo[5,4-f]quinazolin-9(8H)-one (also called FC162) was found to be a promising inhibitor of DYRK1A and was characterized in biol. expts., by western transfer and flow cytometry on SH-SY5Y and pre-B cells.  Here, the results obtained with FC162 are compared to well-characterized known DYRK1A inhibitors (e.g., Leucettine L41 and EHT1610).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMCVXeZbYtAbVg90H21EOLACvtfcHk0lgVE40m51wQCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslalt7%252FK&md5=49a63babbde0c887e44f48df3c167beb</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.3390%2Fph12040185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fph12040185%26sid%3Dliteratum%253Aachs%26aulast%3DFruit%26aufirst%3DC.%26aulast%3DCouly%26aufirst%3DF.%26aulast%3DBhansali%26aufirst%3DR.%26aulast%3DRammohan%26aufirst%3DM.%26aulast%3DLindberg%26aufirst%3DM.%2BF.%26aulast%3DCrispino%26aufirst%3DJ.%2BD.%26aulast%3DMeijer%26aufirst%3DL.%26aulast%3DBesson%26aufirst%3DT.%26atitle%3DBiological%2520characterization%2520of%25208-cyclopropyl-2-%2528pyridin-3-yl%2529thiazolo%255B5%252C4-f%255Dquinazolin-9%25288h%2529-one%252C%2520a%2520promising%2520inhibitor%2520of%2520dyrk1a%26jtitle%3DPharmaceuticals%26date%3D2019%26volume%3D12%26spage%3D185%26doi%3D10.3390%2Fph12040185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rothweiler, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stensen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandsdal, B. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isaksson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engh, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svendsen, J. S. M.</span></span> <span> </span><span class="NLM_article-title">Probing the atp-binding pocket of protein kinase dyrk1a with benzothiazole fragment molecules</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9814</span>– <span class="NLM_lpage">9824</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01086</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01086" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1yksr7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9814-9824&author=U.+Rothweilerauthor=W.+Stensenauthor=B.+O.+Brandsdalauthor=J.+Isakssonauthor=F.+A.+Leesonauthor=R.+A.+Enghauthor=J.+S.+M.+Svendsen&title=Probing+the+atp-binding+pocket+of+protein+kinase+dyrk1a+with+benzothiazole+fragment+molecules&doi=10.1021%2Facs.jmedchem.6b01086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the ATP-Binding Pocket of Protein Kinase DYRK1A with Benzothiazole Fragment Molecules</span></div><div class="casAuthors">Rothweiler, Ulli; Stensen, Wenche; Brandsdal, Bjoern Olav; Isaksson, Johan; Leeson, Frederick Alan; Engh, Richard Alan; Svendsen, John S. Mjoeen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9814-9824</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DYRK1A has emerged as a potential target for therapies of Alzheimer's disease using small mols.  Based on the observation of selective DYRK1A inhibition by firefly D-luciferin, we have explored static and dynamic structural properties of fragment sized variants of the benzothiazole scaffold with respect to DYRK1A using X-ray crystallog. and NMR techniques.  The compds. have excellent ligand efficiencies and show a remarkable diversity of binding modes in dynamic equil.  Binding geometries are detd. in part by interactions often considered "weak", including "orthogonal multipolar" types represented by e.g. F-CO, sulfur-aromat, and halogen-aromat interactions, together with hydrogen bonds that are modulated by variation of electron withdrawing groups.  These studies show how the benzothiazole scaffold is highly promising for the development of therapeutic DYRK1A inhibitors.  In addn., the subtleties of the binding interactions, including dynamics, show how full structural studies are required to fully interpret the essential phys. determinants of binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYSQe1_5bEjLVg90H21EOLACvtfcHk0lgVE40m51wQCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1yksr7P&md5=edb850b232f53b3e161a5023915972e6</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01086%26sid%3Dliteratum%253Aachs%26aulast%3DRothweiler%26aufirst%3DU.%26aulast%3DStensen%26aufirst%3DW.%26aulast%3DBrandsdal%26aufirst%3DB.%2BO.%26aulast%3DIsaksson%26aufirst%3DJ.%26aulast%3DLeeson%26aufirst%3DF.%2BA.%26aulast%3DEngh%26aufirst%3DR.%2BA.%26aulast%3DSvendsen%26aufirst%3DJ.%2BS.%2BM.%26atitle%3DProbing%2520the%2520atp-binding%2520pocket%2520of%2520protein%2520kinase%2520dyrk1a%2520with%2520benzothiazole%2520fragment%2520molecules%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9814%26epage%3D9824%26doi%3D10.1021%2Facs.jmedchem.6b01086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span>; <span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Denzo and scalepack</span>. In  <i>International tables for crystallography volume f: Crystallography of biological macromolecules</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossmann, M. G.</span>, <span class="NLM_string-name">Arnold, E.</span></span>, Eds.; <span class="NLM_publisher-name">Springer Netherlands</span>: <span class="NLM_publisher-loc">Dordrecht</span>, <span class="NLM_year">2001</span>; pp  <span class="NLM_fpage">226</span>– <span class="NLM_lpage">235</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2001&pages=226-235&author=Z.+Otwinowski&author=W.+Minorauthor=M.+G.+Rossmann&author=E.+Arnold&title=International+tables+for+crystallography+volume+f%3A+Crystallography+of+biological+macromolecules"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26atitle%3DDenzo%2520and%2520scalepack%26btitle%3DInternational%2520tables%2520for%2520crystallography%2520volume%2520f%253A%2520Crystallography%2520of%2520biological%2520macromolecules%26aulast%3DRossmann%26aufirst%3DM.%2BG.%26pub%3DSpringer%2520Netherlands%26date%3D2001%26spage%3D226%26epage%3D235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Xds</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lghIOmDbGt15g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">Molecular replacement withMOLREP</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1107/s0907444909042589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1107%2FS0907444909042589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=20057045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=22-25&author=A.+Vaginauthor=A.+Teplyakov&title=Molecular+replacement+withMOLREP&doi=10.1107%2Fs0907444909042589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular replacement with MOLREP</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-25</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement that utilizes a no. of original approaches to rotational and translational search and data prepn.  Since the first publication describing the program, MOLREP has acquired a variety of features that include weighting of the X-ray data and search models, multi-copy search, fitting the model into electron d., structural superposition of two models and rigid-body refinement.  The program can run in a fully automatic mode using optimized parameters calcd. from the input data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzj5SzYvyu_7Vg90H21EOLACvtfcHk0lghIOmDbGt15g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktw%253D%253D&md5=820d114719aca209994ffb0403e3b20d</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042589%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMolecular%2520replacement%2520withMOLREP%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D22%26epage%3D25%26doi%3D10.1107%2Fs0907444909042589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/s0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2Fs0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lghIOmDbGt15g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2Fs0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of theCCP4 suite and current developments</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/s0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+theCCP4+suite+and+current+developments&doi=10.1107%2Fs0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0li8YcS4JvjlGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520theCCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2Fs0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/s0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2Fs0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0li8YcS4JvjlGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2Fs0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schüttelkopf, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Aalten, D. M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prodrg, D. M. F.</span></span> <span> </span><span class="NLM_article-title">PRODRG: a tool for high-throughput crystallography of protein-ligand complexes</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1355</span>– <span class="NLM_lpage">1363</span>, <span class="refDoi"> DOI: 10.1107/s0907444904011679</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=10.1107%2FS0907444904011679" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=15272157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvVegtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=1355-1363&author=A.+W.+Sch%C3%BCttelkopfauthor=D.+M.+F.+van+Aaltenauthor=D.+M.+F.+Prodrg&title=PRODRG%3A+a+tool+for+high-throughput+crystallography+of+protein-ligand+complexes&doi=10.1107%2Fs0907444904011679"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">PRODRG: a tool for high-throughput crystallography of protein-ligand complexes</span></div><div class="casAuthors">Schuettelkopf, Alexander W.; van Aalten, Daan M. F.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1355-1363</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The small-mol. topol. generator PRODRG is described, which takes input from existing coordinates or various two-dimensional formats and automatically generates coordinates and mol. topologies suitable for x-ray refinement of protein-ligand complexes.  Test results are described for automatic generation of topologies followed by energy minimization for a subset of compds. from the Cambridge Structural Database, which shows that, within the limits of the empirical GROMOS87 force field used, structures with good geometries are generated.  X-ray refinement in X-PLOR/CNS, REFMAC and SHELX using PRODRG-generated topologies produces results comparable to refinement with topologies from the std. libraries.  However, tests with distorted starting coordinates show that PRODRG topologies perform better, both in terms of ligand geometry and of crystallog. R factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbflrch0bs97Vg90H21EOLACvtfcHk0li8YcS4JvjlGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvVegtbY%253D&md5=f9bfdb14be7b04ea5c0db67feb39f72a</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444904011679&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904011679%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25BCttelkopf%26aufirst%3DA.%2BW.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%2BM.%2BF.%26aulast%3DProdrg%26aufirst%3DD.%2BM.%2BF.%26atitle%3DPRODRG%253A%2520a%2520tool%2520for%2520high-throughput%2520crystallography%2520of%2520protein-ligand%2520complexes%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D1355%26epage%3D1363%26doi%3D10.1107%2Fs0907444904011679" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VX3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VX3','PDB','2VX3'); return false;">PDB: 2VX3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2vx3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2vx3','PDB','2vx3'); return false;">PDB: 2vx3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A4O" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A4O','PDB','7A4O'); return false;">PDB: 7A4O</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FIG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FIG','PDB','4FIG'); return false;">PDB: 4FIG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ANR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ANR','PDB','3ANR'); return false;">PDB: 3ANR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A4W" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A4W','PDB','7A4W'); return false;">PDB: 7A4W</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A5B" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A5B','PDB','7A5B'); return false;">PDB: 7A5B</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A4R" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A4R','PDB','7A4R'); return false;">PDB: 7A4R</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A51" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A51','PDB','7A51'); return false;">PDB: 7A51</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A5D" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A5D','PDB','7A5D'); return false;">PDB: 7A5D</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A5L" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A5L','PDB','7A5L'); return false;">PDB: 7A5L</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A5N" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A5N','PDB','7A5N'); return false;">PDB: 7A5N</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q8K" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q8K','PDB','6Q8K'); return false;">PDB: 6Q8K</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EIF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EIF','PDB','6EIF'); return false;">PDB: 6EIF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S1H" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S1H','PDB','6S1H'); return false;">PDB: 6S1H</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EIL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EIL','PDB','6EIL'); return false;">PDB: 6EIL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QU2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QU2','PDB','6QU2'); return false;">PDB: 6QU2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AZE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AZE','PDB','4AZE'); return false;">PDB: 4AZE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A4L" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A4L','PDB','5A4L'); return false;">PDB: 5A4L</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YLL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YLL','PDB','4YLL'); return false;">PDB: 4YLL</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i92"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00024">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_04766"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00024?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00024</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Supporting figures, tables and data: Compound <b>34</b> target engagement with DYRK1A demonstrated in nano-BRET assay; comparison of the conformation of equivalent leucine residues in the kinase hinge of CLK1 and Dyrk1a structures; in vivo PK characterisation of compound <b>34</b> in mice; compound <b>34</b> exhibits activity against U87MG glioblastoma cells in vitro; comparison of binding pose of <b>34</b> with recently published inhibitors of DYRK1A; crystallographic data collection; refinement statistics; final concentrations of reagents in the kinase inhibition assays; kinase profiling of <b>34</b>; brain penetrance of <b>34</b>; in vivo PK; complete list of kinome screen data; LC chromatograms, and 1H NMR spectra of key compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_001.pdf">jm1c00024_si_001.pdf (2.59 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00024/suppl_file/jm1c00024_si_002.csv">jm1c00024_si_002.csv (2.19 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00024&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00024%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-13%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00024" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                10MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677aff5c79ec2332","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
