
<html lang="en"     class="pb-page"  data-request-id="23056d68-6911-4334-a7cc-402d2c74cd33"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.0c02118"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Illuminating a Dark Kinase: Structure-Guided Design, Synthesis, and Evaluation of a Potent Nek1 Inhibitor and Its Effects on the Embryonic Zebrafish Pronephros" /></meta><meta name="dc.Creator" content="Georg  Baumann" /></meta><meta name="dc.Creator" content="Tobias  Meckel" /></meta><meta name="dc.Creator" content="Kevin  Böhm" /></meta><meta name="dc.Creator" content="Yung-Hsin  Shih" /></meta><meta name="dc.Creator" content="Mirco  Dickhaut" /></meta><meta name="dc.Creator" content="Torben  Reichardt" /></meta><meta name="dc.Creator" content="Johannes  Pilakowski" /></meta><meta name="dc.Creator" content="Ulrich  Pehl" /></meta><meta name="dc.Creator" content="Boris  Schmidt" /></meta><meta name="dc.Description" content="NIMA-related kinase 1 (Nek1) has lately garnered attention for its widespread function in ciliogenesis, apoptosis, and the DNA-damage response. Despite its involvement in various diseases and its p..." /></meta><meta name="Description" content="NIMA-related kinase 1 (Nek1) has lately garnered attention for its widespread function in ciliogenesis, apoptosis, and the DNA-damage response. Despite its involvement in various diseases and its p..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 16, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02118" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 The Authors. Published by American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02118" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02118" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02118" /></link>
        
    
    

<title>Illuminating a Dark Kinase: Structure-Guided Design, Synthesis, and Evaluation of a Potent Nek1 Inhibitor and Its Effects on the Embryonic Zebrafish Pronephros | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02118" /></meta><meta property="og:title" content="Illuminating a Dark Kinase: Structure-Guided Design, Synthesis, and Evaluation of a Potent Nek1 Inhibitor and Its Effects on the Embryonic Zebrafish Pronephros" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0012.jpeg" /></meta><meta property="og:description" content="NIMA-related kinase 1 (Nek1) has lately garnered attention for its widespread function in ciliogenesis, apoptosis, and the DNA-damage response. Despite its involvement in various diseases and its potential as a cancer drug target, no directed medicinal chemistry efforts toward inhibitors against this dark kinase are published. Here, we report the structure-guided design of a potent small-molecule Nek1 inhibitor, starting from a scaffold identified by kinase cross-screening analysis. Seven lead compounds were identified in silico and evaluated for their inhibitory activity. The top compound, 10f, was further profiled for efficacy, toxicity, and bioavailability in a zebrafish polycystic kidney disease model. Administration of 10f caused the expansion of fluorescence-labeled proximal convoluted tubules, supporting our hypothesis that Nek1-inhibition causes cystic kidneys in zebrafish embryos. Compound 10f displayed insignificant inhibition in 48 of 50 kinases in a selectivity test panel. The findings provide a powerful tool to further elucidate the function and pharmacology of this neglected kinase." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02118"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02118">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02118&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02118&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02118&amp;href=/doi/10.1021/acs.jmedchem.0c02118" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, XXXX</span><span class="cit-fg-issue">, XXX</span><span class="cit-fg-pageRange">, XXX-XXX</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/0/0" title="Return to Articles ASAP" id="returnToASAP" class="content-navigation__btn--return"><span>RETURN TO ARTICLES ASAP</span></a><a href="/doi/10.1021/acs.jmedchem.1c00311" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c02111" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Illuminating a Dark Kinase: Structure-Guided Design, Synthesis, and Evaluation of a Potent Nek1 Inhibitor and Its Effects on the Embryonic Zebrafish Pronephros</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Georg Baumann</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Georg Baumann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Clemens Schöpf−Institute of Organic Chemistry and Biochemistry, Technische Universität Darmstadt, 64287 Darmstadt, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#e384868c9184cd8182968e828d8da390979687cd9796ce8782918e9097828797cd8786"><span class="__cf_email__" data-cfemail="82e5e7edf0e5ace0e3f7efe3ececc2f1f6f7e6acf6f7afe6e3f0eff1f6e3e6f6ace6e7">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Georg++Baumann">Georg Baumann</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tobias Meckel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tobias Meckel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Clemens Schöpf−Institute of Organic Chemistry and Biochemistry, Technische Universität Darmstadt, 64287 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tobias++Meckel">Tobias Meckel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kevin Böhm</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kevin Böhm</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Clemens Schöpf−Institute of Organic Chemistry and Biochemistry, Technische Universität Darmstadt, 64287 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kevin++B%C3%B6hm">Kevin Böhm</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yung-Hsin Shih</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yung-Hsin Shih</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Clemens Schöpf−Institute of Organic Chemistry and Biochemistry, Technische Universität Darmstadt, 64287 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yung-Hsin++Shih">Yung-Hsin Shih</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mirco Dickhaut</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mirco Dickhaut</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Clemens Schöpf−Institute of Organic Chemistry and Biochemistry, Technische Universität Darmstadt, 64287 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mirco++Dickhaut">Mirco Dickhaut</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Torben Reichardt</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Torben Reichardt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Clemens Schöpf−Institute of Organic Chemistry and Biochemistry, Technische Universität Darmstadt, 64287 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Torben++Reichardt">Torben Reichardt</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Johannes Pilakowski</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Johannes Pilakowski</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Clemens Schöpf−Institute of Organic Chemistry and Biochemistry, Technische Universität Darmstadt, 64287 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Johannes++Pilakowski">Johannes Pilakowski</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ulrich Pehl</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ulrich Pehl</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Merck Healthcare KGaA, Biopharma R&D, Discovery and Development Technologies, 64293 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ulrich++Pehl">Ulrich Pehl</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Boris Schmidt</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Boris Schmidt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Clemens Schöpf−Institute of Organic Chemistry and Biochemistry, Technische Universität Darmstadt, 64287 Darmstadt, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: +496151-1623745. Fax: +496151-163278. E-mail: <a href="/cdn-cgi/l/email-protection#2b58484346424f5f7449445942586b5f0644454742454e054f4e"><span class="__cf_email__" data-cfemail="95e6f6fdf8fcf1e1caf7fae7fce6d5e1b8fafbf9fcfbf0bbf1f0">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Boris++Schmidt">Boris Schmidt</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1662-2392" title="Orcid link">http://orcid.org/0000-0003-1662-2392</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02118&amp;href=/doi/10.1021%2Facs.jmedchem.0c02118" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, XXXX</span><span class="cit-issue">, XXX</span><span class="cit-pageRange">, XXX-XXX</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 16, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>8 December 2020</li><li><span class="item_label"><b>Published</b> online</span>16 March 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02118" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02118</a></div><div class="article_header-article-copyright"><strong>© 2021 The Authors. Published by American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DGeorg%2BBaumann%252C%2BTobias%2BMeckel%252C%2BKevin%2BB%25C3%25B6hm%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D0%26issueNum%3D0%26contentID%3Dacs.jmedchem.0c02118%26title%3DIlluminating%2Ba%2BDark%2BKinase%253A%2BStructure-Guided%2BDesign%252C%2BSynthesis%252C%2Band%2BEvaluation%2Bof%2Ba%2BPotent%2BNek1%2BInhibitor%2Band%2BIts%2BEffects%2Bon%2Bthe%2BEmbryonic%2BZebrafish%2BPronephros%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D18%26publicationDate%3DMarch%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02118"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">705</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02118" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Illuminating a Dark Kinase: Structure-Guided Design, Synthesis, and Evaluation of a Potent Nek1 Inhibitor and Its Effects on the Embryonic Zebrafish Pronephros&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Georg&quot;,&quot;last_name&quot;:&quot;Baumann&quot;},{&quot;first_name&quot;:&quot;Tobias&quot;,&quot;last_name&quot;:&quot;Meckel&quot;},{&quot;first_name&quot;:&quot;Kevin&quot;,&quot;last_name&quot;:&quot;Böhm&quot;},{&quot;first_name&quot;:&quot;Yung-Hsin&quot;,&quot;last_name&quot;:&quot;Shih&quot;},{&quot;first_name&quot;:&quot;Mirco&quot;,&quot;last_name&quot;:&quot;Dickhaut&quot;},{&quot;first_name&quot;:&quot;Torben&quot;,&quot;last_name&quot;:&quot;Reichardt&quot;},{&quot;first_name&quot;:&quot;Johannes&quot;,&quot;last_name&quot;:&quot;Pilakowski&quot;},{&quot;first_name&quot;:&quot;Ulrich&quot;,&quot;last_name&quot;:&quot;Pehl&quot;},{&quot;first_name&quot;:&quot;Boris&quot;,&quot;last_name&quot;:&quot;Schmidt&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;16&quot;,&quot;issue&quot;:&quot;0&quot;,&quot;volume&quot;:&quot;0&quot;,&quot;pages&quot;:&quot;&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02118&quot;},&quot;abstract&quot;:&quot;NIMA-related kinase 1 (Nek1) has lately garnered attention for its widespread function in ciliogenesis, apoptosis, and the DNA-damage response. Despite its involvement in various diseases and its potential as a cancer drug target, no directed medicinal chemistry efforts toward inhibitors against this dark kinase are published. Here, we report the structure-guided design of a potent small-molecule Nek1 inhibitor, starting from a scaffold identified by kinase cross-screening analysis. Seven lead compounds were identified in silico and evaluated for their inhibitory activity. The top compound, 10f, was further profiled for efficacy, toxicity, and bioavailability in a zebrafish polycystic kidney disease model. Administration of 10f caused the expansion of fluorescence-labeled proximal convoluted tubules, supporting our hypothesis that Nek1-inhibition causes cystic kidneys in zebrafish embryos. Compound 10f displayed insignificant inhibition in 48 of 50 kinases in a selectivity test panel. The findings provide a p&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02118&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02118" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02118&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02118" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02118&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02118" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02118&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02118&amp;href=/doi/10.1021/acs.jmedchem.0c02118" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02118" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02118" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02118%26sid%3Dliteratum%253Aachs%26genre%3Darticle%26aulast%3DBaumann%26date%3D2021%26atitle%3DIlluminating%2Ba%2BDark%2BKinase%253A%2BStructure-Guided%2BDesign%252C%2BSynthesis%252C%2Band%2BEvaluation%2Bof%2Ba%2BPotent%2BNek1%2BInhibitor%2Band%2BIts%2BEffects%2Bon%2Bthe%2BEmbryonic%2BZebrafish%2BPronephros%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D0%26issue%3D0%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291257" title="Zebrafish">Zebrafish</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/0/0" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/medium/jm0c02118_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02118&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">NIMA-related kinase 1 (Nek1) has lately garnered attention for its widespread function in ciliogenesis, apoptosis, and the DNA-damage response. Despite its involvement in various diseases and its potential as a cancer drug target, no directed medicinal chemistry efforts toward inhibitors against this dark kinase are published. Here, we report the structure-guided design of a potent small-molecule Nek1 inhibitor, starting from a scaffold identified by kinase cross-screening analysis. Seven lead compounds were identified <i>in silico</i> and evaluated for their inhibitory activity. The top compound, <b>10f</b>, was further profiled for efficacy, toxicity, and bioavailability in a zebrafish polycystic kidney disease model. Administration of <b>10f</b> caused the expansion of fluorescence-labeled proximal convoluted tubules, supporting our hypothesis that Nek1-inhibition causes cystic kidneys in zebrafish embryos. Compound <b>10f</b> displayed insignificant inhibition in 48 of 50 kinases in a selectivity test panel. The findings provide a powerful tool to further elucidate the function and pharmacology of this neglected kinase.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar"><issue-title>New Horizons in Drug Discovery - Understanding and Advancing Kinase Inhibitors</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82403" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82403" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Nek1 is a dual serine/threonine protein kinase that also possesses tyrosine kinase activity, albeit to a lesser degree.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Since its discovery in a murine cDNA expression library screen, this mammalian homologue of the fungal NIMA cell cycle regulator has attracted widespread attention due to its pleiotropic function and its association with a number of diseases in humans.<a onclick="showRef(event, 'ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4">(2−4)</a> The ever-growing interest in this remarkable kinase was first sparked by findings that linked Nek1 to two independent mutant alleles (<i>kat</i> and <i>kat</i><sup>2J</sup>), which spontaneously arose from two inbred mouse strains, causing various effects such as male sterility, dwarfing, facial dysmorphism, anemia, and a progressive polycystic kidney disease (PKD).<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a> PKD is the most common genetic cause for renal failure in humans and is characterized by the relentless growth of large fluid-filled cysts and the accompanying destruction of the adjacent renal parenchyma.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> The evidence for the causative relation between Nek1 depletion and PKD was further strengthened by an interactomic study, which indicated strong interactions between the PKD-associated proteins KIF3A, tuberin, and α-catulin and the central coiled-coil region of Nek1.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Moreover, Nek1 expression levels are prominently elevated in embryonic proximal tubule and podocyte precursor cells  and decrease with kidney maturation, supporting the observation that Nek1 is of functional importance for the etiology of PKD.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></div><div class="NLM_p">Interestingly, NEK1 mutations have been linked not only to defects of ciliogenesis, like PKD and short-rib polydactyly syndrome type Majewski but also to other inheritable diseases such as Mohr–Laussen syndrome or amyotrophic lateral sclerosis (ALS), a neurodegenerative disorder that causes the selective death of motor neurons and is associated with an altered DNA damage response (DDR).<a onclick="showRef(event, 'ref10 ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12 ref13">(10−13)</a> Like most of the 11 known members of the human Nek family, Nek1 plays a crucial role in cell cycle control and in particular the DDR.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Nek1 expression is upregulated upon chemical or radiative induction of DNA double-strand breaks (DSBs) and acts as a regulator of DNA repair by homologous recombination (HR).<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> Cell cycle phase-specific phosphorylation of Rad54 by Nek1 either permits HR, by way of Rad51 removal from chromatin in G2-phase, or ensures replication fork stability in S-phase.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Moreover, functional Nek1 is indispensable for proper cell cycle checkpoint activation through phosphorylation of checkpoint kinases 1 and 2 (Chk1/Chk2) in response to radiative stress.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Eventually, Nek1-deficient cells accumulate unrepaired DSBs leading to genomic instability, chromosomal damage, and finally cell death.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> A remarkable feature of Nek1 is its ability to also limit apoptotic cell death even after severe damage has occurred already. Direct phosphorylation of voltage-dependent anion channel 1 (VDAC1) by Nek1 stabilizes the mitochondrial membrane potential thus limiting the caspase-mediated killing of cells.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> More recent evidence suggests that an activating phosphorylation of Nek1 by tousled-like kinase 1 (Tlk1) contributes to the phosphorylation of VDAC1 and that inhibition of the Tlk1/Nek1/VDAC1 axis in combination with DNA damaging agents can sensitize prostate cancer cells to apoptotic killing.<a onclick="showRef(event, 'ref22 ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24 ref25">(22−25)</a> Elevated levels of Nek1 expression have also been identified in other cancer types like thyroid cancer, human gliomas, or renal cell carcinoma (RCC).<a onclick="showRef(event, 'ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28">(26−28)</a> In the latter case, a decreased sensitivity to genotoxic treatment and ionizing radiation was associated with high levels of Nek1, emphasizing the importance of Nek1 inhibition as a therapeutic strategy for drug development in the treatment of RCC.<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> More recently, Freund et al. have shown that high Nek1 expression levels associate with impaired clinical outcome in cervical cancer patients following chemo-radiotherapy or brachytherapy and that a Nek1 knockdown impacts the 3D clonogenic survival of HeLa cervical and HCT-15 colorectal carcinoma cells. Furthermore, the knockdown sensitized those cells to ionizing radiation, underlining the potential of Nek1 inhibitors to increase the radiation or chemotherapy response in anticancer treatment.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></div><div class="NLM_p">Despite the crucial role of Nek1 in a broad variety of diseases and its outstanding potential as a target for anticancer therapeutics, no directed medicinal chemistry efforts toward potent and selective Nek1 inhibitors have been published in the past.<a onclick="showRef(event, 'ref4 ref31'); return false;" href="javascript:void(0);" class="ref ref4 ref31">(4,31)</a> In a first attempt to pinpoint structural elements that influence Nek family inhibitor specificity, Moraes et al. identified a c-Jun N-terminal kinase (JNK) inhibitor with limited inhibitory activity against a truncated Nek1 activation loop mutant in a thermal shift assay.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Shortly thereafter, the same group reported the crystal structure analysis of the human Nek1 kinase domain both in its apo form and in complex with an ATP-competitive CDK inhibitor.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Until today, only three members of the human Nek family, namely, Nek1, -2, and -7, have been structurally solved. More recently, a pyridin-2(5<i>H</i>)-one based inhibitor with a low micromolar IC<sub>50</sub> value against Nek6 was also shown to decrease Nek1 activity to a comparable degree.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> In the framework of developing a comprehensive kinase chemogenomic set, Wells et al. compiled chemical starting points for Nek-family inhibitors by performing an intricate analysis of kinome cross-screening data.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> In the case of Nek1, two series of compounds with promising activities were identified from the profiling data of a large inhibitor set (Published Kinase Inhibitor Set 2, PKIS2).<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The most potent candidate (UNC5078) of a series of 4-thiophene-7-azaindoles exhibited 91.9% Nek1 inhibition at 1 μM.<a onclick="showRef(event, 'ref34 ref37'); return false;" href="javascript:void(0);" class="ref ref34 ref37">(34,37)</a> Strikingly, the 4-aryl-7-azaindole derivative UNC5452 (<b>3a</b>), which was originally intended as an IκB kinase β (IKKβ) inhibitor, exhibited even higher Nek1 activity (99.8% inhibition at 1 μM) while only inhibiting 18 off-target kinases ≥90% at 1 μM (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref34 ref38'); return false;" href="javascript:void(0);" class="ref ref34 ref38">(34,38)</a> Nevertheless, finding suitable inhibitors for Nek1 is not trivial, as indicated by the low hit rates of previously reported screenings.<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/medium/jm0c02118_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical starting points for the development of Nek1 inhibitors, as identified by Wells et al. in an analysis of kinome cross-screening data.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02118&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In this work, we report the structure-guided design and synthesis of several novel Nek1 inhibitors. The 4-phenyl-7-azaindole moiety of UNC5452 (<b>3a</b>) served as a starting point and was expanded upon by <i>in silico</i> analysis, molecular docking simulations, and a virtual chemical library screen in P-loop reconstructed Nek1 crystal structures (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4APC">4APC</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4B9D">4B9D</a>) as well as a Nek1/Nek2 homology model. The most potent compound <b>10f</b> was further profiled in a kinase selectivity panel, a Caco-2 permeability assay, and a zebrafish developmental toxicity assay. <i>In vivo</i> efficacy of <b>10f</b> was evaluated by fluorescence microscopy imaging of zebrafish pronephroi after inhibitor treatment, which is an established model for cystic kidney diseases.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14004" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14004" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The synthesis of UNC5452 (<b>3a</b>) and its derivatives commenced with the amide formation between sulfonyl chloride <b>1</b> and the respective primary or secondary amines to afford sulfonamides <b>2a</b>–<b>h</b> and <b>2j</b>–<b>l</b>. Acetyloxyester <b>2i</b> was prepared from alcohol <b>2f</b> by acetylation with acetic anhydride. Miyaura borylation of aryl halides <b>2a</b>–<b>l</b> gave the corresponding pinacol boronic acid esters, which were fused with 4-bromo-7-azaindole by palladium catalyzed Suzuki cross-coupling forthwith, yielding 4-phenyl-7-azaindoles <b>3a</b>–<b>l</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> In an attempt to synthesize the carboxylic amide analogue <b>6</b> of sulfonamide <b>2f</b> under identical conditions, 4-bromobenzoyl chloride (<b>4</b>) was transformed to the undesired ester <b>5</b>. However, alkaline hydrolysis of <b>5</b> provided benzamide <b>6</b>, which was coupled to 4-bromo-7-azaindole, giving azaindole <b>7</b> in the final Suzuki cross-coupling step. (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). The amino alcohols <b>8a</b>–<b>f</b> were converted to final compounds <b>10a</b>–<b>f</b> through a similar sequence described in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Compounds <b>8a</b>–<b>c</b> were obtained commercially, while <b>8d</b> was prepared by reduction of <span class="smallcaps smallerCapital">d</span>-tyrosine methyl ester with sodium borohydride and <b>8e</b> and <b>8f</b> were prepared by reduction of <span class="smallcaps smallerCapital">l</span>-tryptophan and <span class="smallcaps smallerCapital">d</span>-tryptophan, following a procedure by Meyers et al.<a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41,42)</a> Amide formation between 4-bromosulfonyl chloride (<b>1</b>) and amino alcohols <b>8a</b>–<b>f</b> provided aryl halides <b>9a</b>–<b>f</b>, which again underwent Miyaura borylation and subsequent Suzuki cross-coupling to afford azaindoles <b>10a</b>–<b>f</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>).</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/medium/jm0c02118_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02118&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) for <b>2a</b>–<b>h</b>, DCM, amine, NEt<sub>3</sub>, 0 °C–rt, 74–95%; (b) DMF, bis(pinacolato)diboron, KOAc, PdCl<sub>2</sub>(dppf), 100 °C, not isolated; (c) DMF, 4-bromo-7-azaindole, NaHCO<sub>3</sub> (aq), Pd(PPh<sub>3</sub>)<sub>4</sub>, 100 °C, 11–69%; (d) for <b>2j</b>, DCM, amine, NEt<sub>3</sub>, 0 °C–rt, 64%; for <b>2k</b>, THF, 1-methylpiperazine, rt, 75%; for <b>2l</b>, H<sub>2</sub>O/acetone, <span class="smallcaps smallerCapital">L</span>-phenylalanine, NaHCO<sub>3</sub>, rt, 49%. *Compound <b>2i</b> was prepared from <b>2f</b> by acetylation with acetic anhydride and NEt<sub>3</sub> in DCM, 82%; see <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for more details.</p></p></figure><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/medium/jm0c02118_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02118&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DCM, monoethanolamine, NEt<sub>3</sub>, 0 °C–rt, 48%; (b) THF, 50% w/v KOH (aq), 60 °C, 92%; (c) DMF, bis(pinacolato)diboron, KOAc, PdCl<sub>2</sub>(dppf), 100 °C, not isolated; (d) DMF, 4-bromo-7-azaindole, NaHCO<sub>3</sub> (aq), Pd(PPh<sub>3</sub>)<sub>4</sub>, 100 °C, 43%; see <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for more details.</p></p></figure><figure id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/medium/jm0c02118_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02118&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) for <b>9a</b>,<b>b</b>, DCM, amine, NEt<sub>3</sub>, 0 °C–rt, 98–99%; for <b>9c</b>,<b>d</b>, H<sub>2</sub>O/acetone, <span class="smallcaps smallerCapital">L</span>-phenylalanine, NaHCO<sub>3</sub>, rt, 60–70%; for <b>9e</b>,<b>f</b>, DCM/acetonitrile, <i>N</i>,<i>N</i>-diisopropylethylamine, 0 °C–rt, 30–43%; (b) DMF, bis(pinacolato)diboron, KOAc, PdCl<sub>2</sub>(dppf), 100 °C, not isolated; (c) DMF, 4-bromo-7-azaindole, NaHCO<sub>3</sub> (aq), Pd(PPh<sub>3</sub>)<sub>4</sub>, 100 °C, 26–62%. <b>*</b>Compounds <b>8a</b>–<b>c</b> were obtained commercially; Compound <b>8d</b> was prepared from <span class="smallcaps smallerCapital">d</span>-tyrosine methyl ester hydrochloride by freeing the amine with KOAc in acetonitrile and subsequent reduction with NaBH<sub>4</sub> in methanol, 61%; compounds <b>8e</b>,<b>f</b> were prepared by reduction of the corresponding tryptophan isomer using the method described by Meyers et al.,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> 72–84%; see <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for more details.</p></p></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21823" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21823" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Molecular Modeling</h3><div class="NLM_p">As an initial step of the structure-guided design, the available Nek1 kinase domain crystal structures were analyzed in detail. Both the apo (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4APC">4APC</a>) and the CDK-inhibitor bound 3D structures (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4B9D">4B9D</a>) were obtained from the same inactivating T162A activation loop mutant and expectedly constitute an inactive “DFG-out/C-helix-out” conformation. The activation loop forms an ordered α helix, which is speculated to unfold upon autophosphorylation, thus adopting a disordered conformation more commonly found in active kinases.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> This activation mechanism might to some degree be conserved in the Nek-family, as a similar α-helical activation loop was observed in Nek2 structures but not in Nek7, which is activated through a different mechanism that involves binding of the noncatalytic C-terminal domain of Nek9.<a onclick="showRef(event, 'ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref43 ref44">(43,44)</a> Moreover, the Nek1 structures both crystallized as disulfide bridged dimers.</div><div class="NLM_p">In an attempt to identify areas of conformational mobility in the active site, we isolated and superimposed the monomeric kinase domains using the Molecular Operating Environment (MOE) software platform (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Figure S1</a>). Interestingly, the DFG-motif aspartate (D146) and the gatekeeper (M80) formed hydrogen bonds with an internal water molecule in the ATP binding site in all but one of the isolated chains (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4APC">4APC</a>.A). In this particular case, the side chains of D146 and M80 extend outward in absence of the water molecule, thus opening a hydrophobic back-pocket toward the DFG-motif. This conformational mobility might be exploited by locking Nek1 in an inactive conformation with a small molecule inhibitor. In order to assess the ligand–protein interactions and the amenability of the back-pocket, we docked UNC5452 (<b>3a</b>) into the Nek1 ATP-binding site (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4APC">4APC</a>.A, <a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Figures S2 and S3</a>).</div><div class="NLM_p">The docking experiments suggest that the 7-azaindole moiety positions itself between V31 and F135 side chains and forms two hydrogen bonds with the backbone heteroatoms of the hinge region C83. The pyrrole ring is packed tightly against the side chain of Y82 and directed toward the solvent exposed area of the ATP-binding site, thus adopting the so-called “flipped” binding mode. The “normal” binding mode, where the pyrrole ring points toward the gatekeeper M80, is also conceivable but was not favored in our docking experiments.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> From the hinge-binding motif, the phenyl ring of UNC5452 protrudes toward the ribose-binding region of Nek1 through a narrow channel formed by the side chains of F135, A18, I10, and the catalytic lysine (K33). Unfortunately, the terminal nitrogen atom of K33 is not structurally resolved, but it appears that its side-chain flexibility and the close proximity to the sulfonamide oxygen allow for hydrogen bonding. The proposed binding mode is supported by the 9-fold reduction of activity of carboxylic acid <b>7</b> against Nek1, when compared with sulfonamide <b>3f</b>. It appears that the tetrahedral geometry of the sulfonamide group is crucial for proper orientation of the ligand. Consequently, the sulfonamide nitrogen is positioned downward, toward the catalytic loop. As proposed by the docking experiment, the sulfonamide NH is not acting as a hydrogen bond donor, as the methyl substitution of the secondary sulfonamide hydrogen from <b>3f</b> to <b>3j</b> slightly increased potency.</div><div class="NLM_p">As expected, the highly flexible dimethylaminoethyl fragment was able to adopt multiple conformations. Unfortunately, our experiments led to the conclusion that an entrance into the hydrophobic back pocket from the 4-phenyl-7-azaindole scaffold is not trivial. Due to the intrusion of the F135 and Y82 side chains into the hinge-binding region, the 7-azaindole moiety is angled in a way that hypothetically prevents access to the back-pocket from the 5-position in the “flipped” binding mode. In the disfavored “normal” binding mode, the same problem arises for the 3-position and an entry from the 2-position is presumably blocked by the gatekeeper (M80). Moreover, the docking shows that an approach from the ortho position of the scaffold’s phenyl ring is likely also blocked by M80 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Figure S4</a>). It remains unclear if scaffold hopping to a five membered aromatic ring as in UNC5078 would sufficiently widen the C–C–C bond angle to allow bypassing of the gatekeeper. Laufer and colleagues have successfully applied this strategy in the development of inhibitors for EGFR Met gatekeeper mutants in the past.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></div><div class="NLM_p">As a consequence of these findings, we shifted our attention to the derivatization of the dimethylaminoethyl fragment with means to optimize existing and to establish new interactions in the ribose-binding region. Depending on the protonation state of the tertiary amine, interactions with the side chains of the catalytic loop residues D128 or Q132 or the unresolved side chains of the glycine-rich loop (P-Loop) residues E12 and S14 are conceivable. In order to establish a structure–activity relationship (SAR), we synthesized and evaluated compounds <b>3b</b>–<b>k</b>, with a range of steric demands as well as varying basicity and hydrogen bond capabilities. The most potent compound of the series, alcohol <b>3f</b>, served as the starting point for an <i>in silico</i> virtual library docking experiment to identify virtual hit compounds (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). We reasoned that the structural relation of the sulfonamide linked side chain to <i>N</i>-substituted α-amino acids and their corresponding alcohols allows us to make use of nature’s own toolkit for protein binding. Moreover, this strategy ensured the convenient access to stereochemically pure starting materials and a straightforward synthetic approach.</div><figure id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/medium/jm0c02118_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. <i>In silico</i> chemical library screening to identify hit compounds. See <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for more details.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02118&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The virtual library included a total of 80 compounds that correspond to both isomers of the 20 canonical amino acids and their respective amino alcohols. Molecular docking of the virtual library into the Nek1 kinase domain required reconstruction of the P-loop, which was performed using two independent methods. Starting from the ligand-bound B-chain (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4B9D">4B9D</a>), which was chosen because of its shorter unresolved gap, we first reconstructed the loop using the SuperLooper2 web application.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Second, a homology model of Nek1 was created using the X-ray structure of Nek2 bound to an aminopurine inhibitor (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Figure S7</a>).<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Seven virtual hit compounds were identified by analysis of the docking results. The selection was based on docking score (<i>S</i>-Score), topological polar surface area (TPSA), ligand efficiency, and visual inspection of the docked conformations. Both methods provided similar results, favoring aromatic amino alcohol side chains (<b>10a</b>–<b>f</b>) and an aromatic amino acid side chain (<b>3l</b>).</div><div class="NLM_p">Interestingly, the structurally related Nek1 kinase cross screening hit UNC5078 also possesses an equivalent benzyl group, although the stereoinformation is missing.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> The putative binding mode of the series most potent compound <b>10f</b> was determined from molecular docking into the reconstructed X-ray structure of Nek1 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). In similar fashion to <b>3a</b>, the 7-azaindole moiety is tightly packed between V31 and F135 in the “flipped” binding mode and forms two hydrogen bonds with the hinge region C83. Furthermore, the docking experiment indicates C–H−π interactions of the heterocyclic rings with G86 and V31 side chains as well as a contingent parallel-displaced π–π stacking interaction with F135. The phenyl ring of <b>10f</b> once again protrudes toward the ribose binding region and positions the sulfonamide moiety in close proximity to the catalytic lysine (K33), forming a hydrogen bond. Resulting from this interaction, the hydroxyl group of <b>10f</b> acts as a hydrogen donor toward the backbone carbonyl oxygen of I10, while the indole moiety fits tightly between K17, Q132, and R161. From here, the indole NH forms a hydrogen bond with the side chain of D128, while the aromatic system establishes a C–H−π interaction with the T166 backbone NH. Judging by these results, <b>10f</b> can be classified as a type IIB inhibitor that binds to an inactive DFG-out conformation without occupying the back cleft.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a></div><figure id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/medium/jm0c02118_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Molecular docking of compound <b>10f</b> into X-ray structure of Nek1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4B9D">4B9D</a>.B). P-loop reconstruction was performed using the SuperLooper2 web application.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> See <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Supporting Information</a> for more details (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Figures S5 and S6</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02118&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Biological Assays and Structure–Activity Relationship (SAR) Studies</h3><div class="NLM_p">As a first step, we synthesized UNC5452 (<b>3a</b>) in order to benchmark a commercial <i>in vitro</i> Nek1 activity assay by Eurofins Discovery (KinaseProfiler Item 15-020 KP10). Compound <b>3a</b> inhibited Nek1 activity by only 63% at 1 μM in this ATP-dependent radiometric assay, compared to the 99.8% inhibitory activity reported by Wells et al.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> We hypothesize that this difference results from the fact that screening of the parent PKIS2 data set was performed using an ATP-independent competition-binding assay (DiscoverX KINOME-scan) that instead measures the ability of a small molecule to compete for the active site with an immobilized inhibitor.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></div><div class="NLM_p">The molecular modeling suggested interactions of <b>3a</b> with the Nek1 hinge C83, catalytic K33, and the catalytic loop residues D128 and Q132 to be essential for potent inhibition. Since our primary aim was the optimization of the dimethylaminoethyl fragment, we first opted for the improvement of the terminal dimethylamino group. In the light of our docking experiments and the significant drop in activity upon removal of the two methyl groups, from <b>3a</b> to primary amine UNC5023 in the PKIS2 data set,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> we investigated the influence of the amine p<i>K</i><sub>a</sub> by synthesis and evaluation of compounds <b>3b</b>–<b>e</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Overall, the radiometric Nek1 assay revealed no clear trend between the amine basicity and kinase binding. Compounds <b>3b</b>, <b>3d</b>, and <b>3e</b> displayed decreased activity in comparison to <b>3a</b>, while <b>3c</b> showed comparable activity. The results were confirmed by the determination of IC<sub>50</sub> values for <b>3a</b>–<b>e</b> in a TR-FRET based Nek1 activity assay (LANCE Ultra TR-FRET, <i>K</i><sub>m,ATP</sub>, PerkinElmer). Strikingly, no effect on Nek1 activity was herein observed at the maximum concentration (30 μM) for <b>3e</b>, which reduced Nek1 activity to 49% in the radiometric assay. Furthermore, compound <b>3d</b> displayed a significant decrease in inhibitory activity compared to <b>3b</b>, in contrast to the previous findings. Apart from these discrepancies, the results confirm that a variation of the terminal amine’s basicity did not improve inhibitory activity against Nek1. Rigidification of the flexible dimethylamino-ethyl fragment in compound <b>3k</b> also significantly decreased inhibitory activity against Nek1 to over 10 μM from 7.1 μM in <b>3a</b>. As a result, we reasoned that the attempted optimization of interactions of <b>3a</b> with the catalytic loop residues D128 and Q132, as suggested by the docking experiment, is elusive and that the introduction of an alternative hydrogen bond donor might improve inhibitory activity. Synthesis and evaluation of alcohol <b>3f</b> revealed a significant decrease in residual Nek1 activity to 12% as well as a more than 2-fold decreased IC<sub>50</sub> of 3.2 μM, compared to <b>3a</b>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Inhibitory Activity of Compounds <b>3a</b>–<b>f</b> and <b>3k</b> against Nek1 and Nek2</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/medium/jm0c02118_0009.gif" alt="" id="gr9" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Eurofins Discovery KinaseProfiler service at 10 μM ATP.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">LANCE Ultra TR-FRET kinase assay at <i>K</i><sub>m,ATP</sub>. <sup>#</sup>A relative IC<sub>50</sub> value was provided because the determined IC<sub>50</sub> value exceeded the employed maximum test concentration (10 μM). See <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for more details. See <a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Supporting Information</a> for raw data. CI, confidence interval; nd, not determined; ne, no effect at maximum concentration (30 μM).</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Eurofins Discovery IC50Profiler service at <i>K</i><sub>m,ATP</sub>.</p></div></div><div></div></div><div class="NLM_p">In order to evaluate the hydrogen bond characteristics of the hydroxy group in <b>3f</b>, methyl ether <b>3g</b> and acetyl ester <b>3i</b> analogues were prepared and tested (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Both analogues showed decreased inhibitory activity compared to <b>3f</b>, with an IC<sub>50</sub> larger than 30 μM for ether <b>3g</b> and 8.8 μM for ester <b>3i</b>. The results clearly indicate that the hydroxy group in <b>3f</b> does indeed act as a hydrogen bond donor and not as an acceptor, as the reduced acceptor capability in <b>3i</b> did not further reduce inhibitory activity. It remains unclear if the carbonyl oxygen in <b>3i</b> acts as an alternative hydrogen bond acceptor but it is unlikely to interact with the same region, due to its different position relative to the rest of the molecule. Moreover, the introduction of a bulky tertiary alcohol in <b>3j</b> led to a significantly decreased IC<sub>50</sub> of 5.8 μM. With the improved inhibitory activity of <b>3f</b> in hand, we switched our focus to the sulfonamide moiety (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). As previously suggested by docking of <b>3a</b> into the Nek1 ATP binding site, the methyl replacement of the secondary sulfonamide hydrogen from <b>3f</b> to <b>3h</b> did not influence Nek1 inhibition significantly. However, substitution of the tetrahedral sulfonamide group with a planar carboxamide in <b>7</b> led to a 9-fold increased IC<sub>50</sub> compared to <b>3f</b>, providing evidence for the proposed importance of the sulfonamide geometry in hydrogen bonding to K33. In addition, compounds <b>3a</b>–<b>k</b> and <b>7</b> were profiled for their Nek2 inhibitory activity using the same TR-FRET based assay platform. In concordance with the Nek-family selectivity data for <b>3a</b> reported by Wells et al.,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> none of the tested compounds exerted an observable effect on the kinase activity of Nek2.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Inhibitory Activity of Compounds <b>3g</b>–<b>j</b> and <b>7</b> against Nek1 and Nek2</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/medium/jm0c02118_0010.gif" alt="" id="gr10" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Eurofins Discovery KinaseProfiler service at 10 μM ATP.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">LANCE Ultra TR-FRET kinase assay at <i>K</i><sub>m,ATP</sub>. <sup>#</sup>A relative IC<sub>50</sub> value was provided because the determined IC<sub>50</sub> value exceeded the employed maximum test concentration (30 μM). See <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for more details. See <a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Supporting Information</a> for raw data. CI, confidence interval; nd, not determined; ne, no effect at maximum concentration (30 μM).</p></div></div><div></div></div><div class="NLM_p">As described in the molecular modeling section, seven virtual hit compounds (<b>3l</b> and <b>10a</b>–<b>f</b>) were identified in an <i>in silico</i> screening from a library of canonical <i>N</i>-substituted α-amino acid derivatives, and their corresponding alcohols, structurally based on <b>3f</b>. Accordingly, phenylalanine derivative <b>3l</b> and phenylalaninol, tyrosinol, and tryptophanol derivatives <b>10a</b>–<b>f</b> were synthesized and evaluated (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). The <span class="smallcaps smallerCapital">l</span>-phenylalanine derivative <b>3l</b> did not notably decrease Nek1 activity, while the corresponding phenylalaninol derived enantiomers <b>10a</b> and <b>10b</b> displayed decreased Nek1 IC<sub>50</sub> values of 1.9 μM and 1.3 μM, respectively. The tyrosinol derivatives <b>10c</b> and <b>10d</b> performed slightly worse, while the <span class="smallcaps smallerCapital">l</span>-tryptophanol derivative <b>10e</b> exhibited a decreased IC<sub>50</sub> compared to <b>10a</b> (0.99 μM). Strikingly, the <span class="smallcaps smallerCapital">d</span>-tryptophanol derivative <b>10f</b> showed 6-fold decreased IC<sub>50</sub> of 0.33 μM against Nek1 compared to <b>3f</b>. Our docking experiments suggest that this decrease results from the generally favored orientation of the (<i>R</i>)-enantiomers, which in this particular case gives rise to an additional hydrogen bond interaction of the indole NH with the D128 side chain as well as a C–H−π interaction of the aromatic system with the T166 backbone NH (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Inhibitory Activity of Compounds <b>3l</b> and <b>10a</b>–<b>f</b> against Nek1</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/medium/jm0c02118_0011.gif" alt="" id="gr11" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Eurofins Discovery KinaseProfiler service at 10 μM ATP.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Eurofins Discovery IC50Profiler service at <i>K</i><sub>m,ATP</sub>. See <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for more details. See <a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Supporting Information</a> for raw data. CI, confidence interval; nd, not determined; ne, no effect at maximum concentration (10 μM).</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">Eurofins Discovery IC50Profiler service at 10 μM ATP.</p></div></div><div></div></div><div class="NLM_p">Kinase selectivity profiling of <b>10f</b> against 50 kinases was carried out with ATP concentrations set within 15 μM of the apparent <i>K</i><sub>m</sub> for ATP for each kinase tested, so that the resulting selectivity profile directly reflects the intrinsic affinities of <b>10f</b>.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Of the 50 kinases in this focused panel, including several members of the Nek family, 47 showed activity higher than 75%, whereas Nek1 displayed a residual activity of only 49.7%. The only kinase that was also significantly inhibited by <b>10f</b> was IKKβ with a residual activity of 36.1% (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). However, given that the selectivity profile is based on a limited number of kinases (approximately 10% of the kinome) and does not extend to enzymes beyond the kinome, caution must be exercised in deducing general selectivity from the data.</div><figure id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/medium/jm0c02118_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Screening of Compound <b>10f</b> against a panel of human protein kinases. Each bar represents the activity of one individual protein kinase. Compound <b>10f</b> was tested at a concentration of 1 μM. The ATP concentration was set to the approximate <i>K</i><sub>m,ATP</sub> for each kinase tested. See <a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Supporting Information</a> for more details.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02118&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Zebrafish Developmental Toxicity Assay and Fluorescence Microscopy Imaging of Zebrafish Pronephroi</h3><div class="NLM_p">In order to evaluate whether its <i>in vitro</i> inhibitory activity translates into predictable effects <i>in vivo</i>, compound <b>10f</b> was further profiled for its activity, safety, and toxicity in wild-type zebrafish (<i>Danio rerio</i>). For this purpose, we established a PKD model based on a previously mentioned report by Chen et al., which indicates that Nek1 deficiency results in renal cyst formation as a consequence of disordered kidney maturation in <i>kat</i><sup>2j</sup> mice.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Indeed, the zebrafish has proven itself as an established model system for the evaluation of mammalian kidney morphology, because of its simple pronephric structure, which consists of two nephroi that share major anatomical and functional features with the mammalian metanephros. Its usefulness as a model for cystic kidney diseases has recently been reviewed.<a onclick="showRef(event, 'ref39 ref51'); return false;" href="javascript:void(0);" class="ref ref39 ref51">(39,51)</a> Furthermore, it has been shown that the majority of human kinases display high degrees of identity with their zebrafish homologues. Human Nek1, for example, shares 89% identity to zebrafish Nek1 for the catalytically active kinase domain, according to a study by Wlodarchak et al.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Moreover, all residues of the active site appear unaltered in sequence analysis (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Figures S8 and S9</a>). We therefore hypothesized that treatment with <b>10f</b>, during early stages of embryonic organ development, leads to the formation of renal cysts, thus indicating efficient absorption and <i>in vivo</i> efficacy of <b>10f</b>. It is important to note here that our model is based on the necessary assumption that Nek1 inhibition results in the same phenotype that is associated with its depletion, as the correlation between small-molecule inhibition and loss-of-function phenotypes is not always straightforward.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> However, our approach is substantiated by the reported link of aberrant cystic growth to the absence of Nek1, as well as to its overexpression.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Hence, the authors conclude that healthy ciliogenesis requires a delicate spatiotemporal activity of Nek1, which is likely to be substantially disturbed by inhibition and depletion alike.</div><div class="NLM_p">In order to visualize the influence of <b>10f</b> on larval zebrafish nephrogenesis, we implemented a fluorescence-microscopy based imaging pipeline developed by Westhoff et al. (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> As a means of avoiding dependence on the transgenic <i>Tg</i>(<i>wt1b:EGFP</i>) zebrafish line, we instead opted for the use of PT-Yellow (BDNCA3-D2), a fluorescent probe that selectively labels the renal proximal tubules in wild-type zebrafish.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> This approach allowed for zebrafish maintenance and husbandry in a standard laboratory, and it was furthermore reasoned that the ingestion of an external probe is better suited to indicate the impaired renal clearance associated with fluid-filled cysts.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> As a first step, a bioavailability profile of compound <b>10f</b> was evaluated. The aqueous solubility of <b>10f</b> in E3-medium (13.9 ± 0.1 μg/mL, <a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Figure S10</a>) was determined in a shake-flask solubility assay, and apical to basolateral transport was determined in a Caco-2 permeability assay (3.4 × 10<sup>–6</sup> cm/s, 45% recovery, <a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Table S3</a>). We then determined the minimal lethal concentrations for compound <b>10f</b> in a zebrafish developmental toxicity assay over a duration of 120 h. Compound <b>10f</b> was added to 6 h postfertilization (hpf) embryos in concentrations ranging from 0.05 μM to 30 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Figure S11</a>). Full lethality was only observed close to the solubility limit for a concentration of 30 μM after 72 h. Treatment with <b>10f</b> did not induce obvious phenotypes visible to the eye. The treated embryos exhibited normal eye development and body curvature and an ordinary heartbeat.</div><figure id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/medium/jm0c02118_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Microscopic imaging of fluorescence-labeled (PT-Yellow) zebrafish pronephroi after treatment with Nek1-inhibitor <b>10f</b>. (A) Schematic representation of the workflow, entailing embryo selection, compound treatment, and image acquisition. (B) Schematic representation of the pronephros in the developing zebrafish (28-somite stage). Reprinted with permission from ref <a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a>. Copyright 2018 Gehrig, Pandey and Westhoff. (C) Exemplary maximum intensity projections of deconvolved <i>Z</i>-stacks (15× magnification; 33 <i>Z</i>-slices; d<i>Z</i> = 15 μm). The dotted line indicates the cropped region of interest. (C, i) negative control E3-medium, (C, ii) negative control 0.2% DMSO, (C, iii) 10 μM <b>10f</b>, and (C, iv) 20 μM <b>10f</b>. (D) Quantification of fluorescence intensity based on summed image projections of deconvolved <i>Z</i>-stacks using ImageJ. <i>n</i> = 3 for negative control, 0.2% DMSO, and 10 μM <b>10f</b>; <i>n</i> = 5 for 20 μM <b>10f</b> (<sup>#</sup><i>P</i> = 0.0516; *<i>P</i> < 0.05). (E) Quantification of pronephroi size by summed pixel area determination based on summed image projections of deconvolved <i>Z</i>-stacks using ImageJ. <i>n</i> = 3 for negative control, 0.2% DMSO, and 10 μM <b>10f</b>; <i>n</i> = 5 for 20 μM <b>10f</b> (*<i>P</i> < 0.05; **<i>P</i> < 0.01). Abbreviations: au, arbitrary unit; C, Cloaca; CS, corpuscles of Stannius; DE, distal early tubule; DL, distal late tubule; hpf, hours post fertilization; N, neck region; P, podocytes of the glomerulus; PCT, proximal convoluted tubule; PD, pronephric duct; PST, Proximal straight tubule.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02118&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Based on these results, zebrafish embryos were incubated with (i) the negative control (E3-medium), (ii) 0.2% DMSO, (iii) 10 μM <b>10f</b>, and (iv) 20 μM <b>10f</b> in E3-medium containing 75 μM 1-phenyl-2-thiourea (PTU) as a melanogenesis inhibitor at 6 hpf. To ensure the stability of <b>10f</b> against PTU and PT-Yellow over the duration of the experiment, a stability-indicating HPLC assay was performed in E3-medium (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Figure S12</a>). After 48 h, all embryos were treated with 100 nM PT-Yellow for 20 min, thoroughly washed with E3-medium, and returned to the respective compound solution for a total incubation time of 96 h. Finally, the embryos were anesthetized and transferred to agarose molds, which were prepared in 96-well plates according to a published procedure, for fluorescent imaging in a widefield microscope (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Figure S13</a>).<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> Administration of <b>10f</b> caused a substantial increase in fluorescence intensity (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D) and expansion (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E) of the proximal convoluted tubules, thus indicating the growth of fluid-filled cysts, alongside an impaired renal clearance. In addition, our findings provide further evidence for the adequate absorption and bioavailability of <b>10f</b>. As hypothesized, our experiments thus demonstrate that Nek1 inhibition by <b>10f</b> during early organ development causes cystic kidney growth in zebrafish embryos, despite our findings that <b>10f</b> is unselective against IKKβ. Park et al. have shown that levels of phosphorylated-IKKα/β and its downstream factor NF-κB are upregulated in the renal tissue of <i>PKD2</i> transgenic mice compared to wild-type controls.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a></div><div class="NLM_p last">Moreover, Booij et al. demonstrated that cystic growth was in fact reduced by IKKα/β-inhibitors in a murine 3D cell culture based screening platform.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> It therefore appears that Nek1 inhibition overturns any presumed protective effects of IKKβ inhibition on cystogenesis in our experiments. It should be noted here that, unlike in murine models, zebrafish IKKα is suspected to negatively regulate NF-κB activity. IKKα<sup>–/–</sup> zebrafish embryos displayed severe head and tail malformations, as well as an upregulated NF-κB response and increased NF-κB-dependent apoptosis, in a study by Correa et al.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> In accordance with the low IKKα-activity of <b>10f</b> (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Table S1</a>), this phenotype was not observed in our experiments. Overall, this leads to the conclusion that the aberrant cystic growth in <b>10f</b>-treated zebrafish embryos is unlikely to be mediated by IKK cross-reactivity. However, a causative relation to hitherto unidentified cross-reactivity cannot be ruled out entirely, due to the aforementioned limitations of the selectivity test panel and the fact that the complex PKD-related molecular pathways and regulatory networks are not yet fully understood.<a onclick="showRef(event, 'ref61 ref62 ref63'); return false;" href="javascript:void(0);" class="ref ref61 ref62 ref63">(61−63)</a></div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44147" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44147" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Based on a published chemical starting point, we generated hypotheses for improved interactions with the Nek1 ATP-binding site by molecular docking into the available crystal structures. These hypotheses were challenged by a series of structurally related inhibitors that were based on the same 4-aryl-7-azaindole scaffold. The most potent compound of this series, <b>3f</b>, served as the structural basis of a virtual structure library that encompassed 80 <i>N</i>-substituted amino acids and their corresponding alcohols. Seven virtual hits were thereupon identified in an <i>in silico</i> virtual library screen in two independent P-loop reconstructed Nek1 3D-structures. This natural product based approach allowed for convenient access to enantiomerically pure starting materials and a straightforward synthetic route and provided the opportunity to exploit nature’s own toolkit for protein binding. Our efforts culminated in the ATP-competitive Nek1 inhibitor <b>10f</b> that displayed IC<sub>50</sub> values in the mid to low nanomolar range. Moreover, 48 out of 50 kinases were not substantially inhibited by <b>10f</b> in a kinase selectivity panel (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Ample bioavailability of <b>10f</b> was confirmed in a Caco-2 permeability assay, and its toxicity was investigated in wild-type zebrafish embryos. Treatment with <b>10f</b> was well tolerated, as full lethality was only observed close to the solubility limit for a concentration of 30 μM. In order to evaluate the <i>in vivo</i> efficacy of <b>10f</b>, we modified a published procedure for fluorescence microscopy imaging of larval zebrafish pronephroi. As hypothesized, our experiments demonstrated that Nek1 inhibition by <b>10f</b> during early organ development causes cystic kidney growth in zebrafish embryos. Our study thus provides the first directed tool compound for a mostly neglected kinase that has been shown to be a key player in numerous forms of cancer and other illnesses, thereby helping to establish Nek1 as a therapeutic target and providing a powerful tool to further elucidate its biological function.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84676" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84676" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> General Information</h3><div class="NLM_p last">Unless otherwise noted, starting materials and reagents were purchased from commercial sources and used without further purification. All reactions employing anhydrous conditions were performed in dried glassware with dry solvents under argon atmosphere unless otherwise noted. All cross-coupling reactions were performed with degassed solvents. NMR spectra were recorded on a Bruker Avance II (300 MHz for <sup>1</sup>H and 75 MHz for <sup>13</sup>C NMR), a Bruker Avance III spectrometer (300 MHz for <sup>1</sup>H and 75 MHz for <sup>13</sup>C NMR), or a Bruker DRX 500 (500 MHz for <sup>1</sup>H and 126 MHz for <sup>13</sup>C NMR) spectrometer. Chemical shifts are reported as ppm by frequency downfield of TMS. Coupling constants are reported in Hertz (Hz). Mass spectrometry was performed on a Bruker Daltonik Impact II quadrupol-TOF spectrometer for atmospheric pressure chemical ionization (APCI) and ESI experiments. EI experiments were performed on either a Finnigan MAT95 sector field or a Fisons MD 800 quadrupol spectrometer. HPLC was carried out on an Agilent 1100 system using a Phenomenex Synergi Polar-RP reversed-phase column (4 μm particle size, 150 mm × 3.0 mm, pore size 80 Å) connected to a variable wavelength detector (VWD) or a Phenomenex Synergi Hydro-RP reversed-phase column (4 μm particle size, 150 mm × 3.0 mm, pore size 80 Å) connected to a diode array detector (DAD). Solvent gradient = 30% A for 1 min, linear gradient to 10% A for 10 min, 10% A for 1 min; solvent A = 0.1% TFA in water; solvent B = acetonitrile; flow rate 1.0 mL/min. TLC was performed on precoated 0.2 mm silica gel 60 F254 aluminum sheets (Merck) with detection by UV light (254 and 365 nm) or potassium permanganate staining. Flash chromatography was carried out using the specified solvent systems on 40–63 μm NORMASIL 60 silica gel (VWR). All compounds used in biological or biochemical assays had >95% purity as determined in the HPLC method described above.</div></div><div id="sec5_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> General Procedure A: Synthesis of Phenylsulfonamides</h3><div class="NLM_p last">Procedure A is exemplified by the triethylamine-mediated sulfonamide formation of compound <b>2a</b> from 4-bromobenzenesulfonyl chloride <b>1</b> and <i>N</i>,<i>N</i>-dimethylethylenediamine. Compounds <b>2b–h</b>, <b>2j</b>, <b>9a</b>, and <b>9b</b> were prepared in accordance with procedure A starting from 4-bromobenzenesulfonyl chloride <b>1</b> and the corresponding amine.</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> General Procedure B: Suzuki–Miyaura Coupling of Aromatic Rings<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></h3><div class="NLM_p last">Procedure B is exemplified by the Suzuki–Miyaura coupling of 4-bromo-7-azaindole and 4-bromo-<i>N</i>-(2-(dimethylamino)-ethyl)benzenesulfonamide (<b>2a</b>) to give target compound <b>3a</b>. Compounds <b>3b</b>–<b>k</b>, <b>10a</b>–<b>f</b>, and <b>7</b> were prepared in accordance with procedure B starting from 4-bromo-7-azaindole and phenylsulfonamides <b>2b</b>–<b>k</b> and <b>9a</b>–<b>f</b> and benzamide <b>6</b>, respectively.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> 4-Bromo-<i>N</i>-(2-(dimethylamino)ethyl)benzenesulfonamide (<b>2a</b>)</h3><div class="NLM_p last">To a stirred solution of 4-bromobenzene-sulfonyl chloride (<b>1</b>, 1.00 g, 3.91 mmol) in 10 mL DCM were added <i>N</i>,<i>N</i>-dimethylethylenediamine (414 mg, 4.70 mmol) and triethylamine (774 mg, 9.78 mmol) at 0 °C under cooling in an ice bath. The cooling bath was removed, and the solution was stirred for 1 h at ambient temperature. The organic layer was washed with water (3 × 5 mL) and brine (5 mL) and dried over MgSO<sub>4</sub>. The solvent was removed under reduced pressure to give <b>2a</b> (1.14 g, 95%) as a light tan solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 300 MHz): δ = 7.78–7.71 (m, 2H), 7.69–7.62 (m, 2H), 3.01–2.95 (m, 2H), 2.40–2.33 (m, 2H), 2.11 (s, 6H).</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> 4-Bromo-<i>N</i>-(2-(diethylamino)ethyl)benzenesulfonamide (<b>2b</b>)</h3><div class="NLM_p last">Synthesis of <b>2b</b> was performed by following general procedure A using <i>N</i>,<i>N</i>-dimethylethylenediamine. Yield 74%, yellow oil. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 300 MHz,): δ = 7.77–7.60 (m, 4H), 4.74 (s, 1H), 2.99–2.88 (m, 2H), 2.53–2.45 (m, 2H), 2.39 (q, <i>J</i> = 7.1 Hz, 4H), 0.92 (t, <i>J</i> = 7.1 Hz, 6H). <sup>13</sup>C NMR (CDCl<sub>3</sub>, 75 MHz,): δ = 139.0, 132.4, 128.8, 127.6, 51.1, 46.5, 40.3, 11.7. <i>m</i>/<i>z</i> (EI): 335 ([M]<sup>+</sup>).</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> 4-Bromo-<i>N</i>-(2-(piperidin-1-yl)ethyl)benzenesulfonamide (<b>2c</b>)</h3><div class="NLM_p last">Synthesis of <b>2c</b> was performed by following general procedure A using 1-(2-aminoethyl)piperidine. Yield 91%, orange solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 300 MHz): δ = 7.78–7.67 (m, 2H), 7.69–7.58 (m, 2H), 5.31 (s, 1H), 2.95 (dd, <i>J</i> = 6.6, 5.1 Hz, 2H), 2.34 (dd, <i>J</i> = 6.6, 5.1 Hz, 2H), 2.19 (t, <i>J</i> = 5.1 Hz, 4H), 1.43 (dp, <i>J</i> = 22.4, 5.7 Hz, 7H). <sup>13</sup>C NMR (CDCl<sub>3,</sub>75 MHz,) δ = 138.9, 132.4, 128.8, 127.5, 56.2, 54.0, 39.4, 25.9, 24.3. <i>m</i>/<i>z</i> (APCI): 347.05 (M + H)<sup>+</sup>.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> 4-Bromo-<i>N</i>-(2-morpholinoethyl)benzenesulfonamide (<b>2d</b>)</h3><div class="NLM_p last">Synthesis of <b>2d</b> was performed by following general procedure A using 4-(2-aminoethyl)morpholine. Yield 87%, colorless solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 300 MHz): δ = 7.78–7.70 (m, 2H), 7.70–7.62 (m, 2H), 5.29 (s, 1H), 3.67–3.60 (m, 4H), 3.02 (dd, <i>J</i> = 6.7, 4.9 Hz, 2H), 2.46–2.40 (m, 2H), 2.33–2.27 (m, 4H). <sup>13</sup>C NMR (CDCl<sub>3</sub>, 75 MHz): δ = 138.8, 132.4, 128.6, 127.6, 66.7, 56.3, 53.0, 38.9. <i>m</i>/<i>z</i> (APCI): 349.03 (M + H)<sup>+</sup>.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> <i>tert</i>-Butyl (2-(4-Bromophenyl)sulfonamido)ethyl)carbamate (<b>2e</b>)</h3><div class="NLM_p last">Synthesis of <b>2e</b> was performed by following general procedure A using <i>N</i>-Boc-ethylenediamine. Yield 78%, colorless solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 300 MHz): δ = 7.76–7.59 (m, 4H), 5.05 (s, 1H), 3.24–3.18 (m, 2H), 3.04 (dd, <i>J</i> = 6.3, 4.8 Hz, 2H), 1.41 (s, 9H). <sup>13</sup>C NMR (CDCl<sub>3</sub>, 75 MHz,): δ = 156.8, 139.2, 132.5, 128.7, 127.6, 52.6, 46.3, 43.9, 42.0, 40.4, 28.54, 28.47, 10.7. <i>m</i>/<i>z</i> (EI): 380 ([M<sup>+</sup>]).</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> 4-Bromo-<i>N</i>-(2-hydroxyethyl)benzenesulfonamide (<b>2f</b>)</h3><div class="NLM_p last">Synthesis of <b>2f</b> was performed by following general procedure A using 2-aminoethanol. Yield 81%, colorless solid. <sup>1</sup>H NMR (DMSO-<i>d</i>6, 300 MHz): δ = 7.84–7.70 (m, 4H), 4.67 (t, <i>J</i> = 5.5 Hz, 1H), 3.40–3.32 (m, 2H), 2.80 (q, <i>J</i> = 6.1 Hz, 2H). <i>m</i>/<i>z</i> (APCI): 279.96 ([M + H]<sup>+</sup>).</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> 4-Bromo-<i>N</i>-(2-methoxyethyl)benzenesulfonamide (<b>2g</b>)</h3><div class="NLM_p last">Synthesis of <b>2g</b> was performed by following general procedure A using 2-methoxyethylamine. Yield 78%, yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz,): δ = 7.82–7.78 (m, 2H), 7.75–7.69 (m, 2H), 3.32 (s, 3H), 3.29 (t, <i>J</i> = 5.6 Hz, 2H), 2.92 (q, <i>J</i> = 5.8 Hz, 2H).</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> 4-Bromo-<i>N</i>-(2-hydroxyethyl)-<i>N</i>-methylbenzenesulfonamide (<b>2h</b>)</h3><div class="NLM_p last">Synthesis of <b>2h</b> was performed by following general procedure A using <i>N</i>-methylethanolamine. Yield 90%, colorless oil. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 300 MHz): δ = 7.68 (s, 4H), 3.78 (dd, <i>J</i> = 5.6, 4.9 Hz, 2H), 3.18 (t, <i>J</i> = 5.3 Hz, 2H), 2.85 (s, 3H). <i>m</i>/<i>z</i> (APCI): 293.98 ([M + H]<sup>+</sup>).</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 2-((4-Bromophenyl)sulfonamido)ethyl acetate (<b>2i</b>)</h3><div class="NLM_p last">To a stirred solution of <b>2f</b> (0.77 g, 2.73 mmol) and triethylamine (0.33 g, 3.28 mmol) in 5 mL of DCM, acetanhydride (0.34 g, 3.28 mmol) was added dropwise at 0 °C under cooling in an ice bath. The cooling bath was removed, and the solution was stirred for 72 h at ambient temperature. The reaction mixture was diluted with 20 mL of DCM and quenched by bringing the pH to 9 through addition of saturated NaHCO<sub>3</sub> (aq). The organic layer was washed with saturated NaHCO<sub>3</sub> (aq, 3 × 5 mL) and brine (5 mL) and dried over MgSO<sub>4</sub>. The solvent was removed under reduced pressure to give <b>2i</b> (0.73 g, 82%) as a yellow oil that was used without further purification. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ = 7.85–7.80 (m, 2H), 7.75–7.69 (m, 2H), 3.95 (t, <i>J</i> = 5.6 Hz, 2H), 3.03 (q, <i>J</i> = 5.6 Hz, 2H), 1.93 (s, 3H). <i>m</i>/<i>z</i> (ESI): 343.96 ([M + Na]<sup>+</sup>).</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 4-Bromo-<i>N</i>-(2-hydroxy-2-methylpropyl)benzenesulfonamide (<b>2j</b>)</h3><div class="NLM_p last">Synthesis of <b>2j</b> was performed by following general procedure A using 1-amino-2-methyl-2-propanol. Yield 64%, yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 7.83–7.78 (m, 2H), 7.75–7.71 (m, 2H), 7.59 (t, <i>J</i> = 6.5 Hz, 1H), 4.40 (s, 1H), 2.62 (d, <i>J</i> = 6.5 Hz, 2H), 1.04 (s, 6H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz): δ = 140.1, 132.1, 128.5, 125.9, 68.7, 53.6, 27.0. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz, DEPT): δ = 132.1, 128.6, 53.6, 45.5, 27.1, 8.5. <i>m</i>/<i>z</i> (APCI): 307.98 ([M + H]<sup>+</sup>).</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 1-((4-Bromophenyl)sulfonyl)-4-methylpiperazine (<b>2k</b>)</h3><div class="NLM_p last">To a solution of 4-bromobenzene-sulfonyl chloride (0.40 g, 1.47 mmol) in 9 mL of dry THF, 1-methyl-piperazine (1.41 g, 14.09 mmol) was added, and the mixture was stirred for 16 h at ambient temperature. The solvent was removed under reduced pressure, and the resulting residue was dissolved in 10 mL of DCM. The organic layer was washed with saturated NaHCO<sub>3</sub> (aq, 3 × 2.5 mL) and dried over MgSO<sub>4</sub>. The solvent was removed under reduced pressure to give <b>2k</b> (0.37 g, 75%) as a yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 500 MHz): δ = 7.69–7.63 (m, 2H), 7.62–7.58 (m, 2H), 3.03 (t, <i>J</i> = 4.9 Hz, 4H), 2.47 (t, <i>J</i> = 5.0 Hz, 4H), 2.26 (s, 3H). <sup>13</sup>C NMR (CDCl<sub>3,</sub>126 MHz): δ = 134.7, 132.4, 129.4, 128.0, 54.1, 46.0, 45.8. <i>m</i>/<i>z</i> (EI): 380 ([M]<sup>+</sup>).</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> ((4-Bromophenyl)sulfonyl)-<span class="smallcaps smallerCapital">l</span>-phenylalanine (<b>2l</b>)</h3><div class="NLM_p last">To a stirred solution of <span class="smallcaps smallerCapital">l</span>-phenylalanine (194 mg, 1.17 mmol) in 4 mL of water was added sodium bicarbonate (296 mg, 3.52 mmol). The mixture was stirred at ambient temperature for 15 min, and a solution of 4-bromobenzenesulfonyl chloride (<b>1</b>, 300 mg, 1.17 mmol) in 4 mL of acetone was added at 0 °C under cooling in an ice bath. The cooling bath was removed, and the solution was stirred at ambient temperature for 20 h. After evaporation of the volatile solvent under reduced pressure, the product was precipitated by bringing the pH to 7 using hydrochloric acid (1 N). Filtration and drying under reduced pressure gave <b>2l</b> (220 mg, 49%) as a yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub><i>6</i></sub>, 500 MHz): δ = 7.72–7.69 (m, 2H), 7.65–7.61 (m, 2H), 7.18–7.11 (m, 5H), 3.35 (t, <i>J</i> = 5.1 Hz, 1H), 2.98 (dd, <i>J</i> = 13.4, 5.0 Hz, 1H), 2.90 (dd, <i>J</i> = 13.4, 5.2 Hz, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub><i>6</i></sub>, 126 MHz): δ = 170.8, 139.7, 138.7, 132.0, 129.9, 128.7, 127.5, 125.9, 125.6, 58.3, 38.5. <i>m</i>/<i>z</i> (ESI): 788.95 ([2M + Na]<sup>+</sup>), 405.97 ([M + Na]<sup>+</sup>), 383.99 ([M + H]<sup>+</sup>).</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> <i>N</i>-(2-(Dimethylamino)ethyl)-4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)benzenesulfonamide (<b>3a</b>)</h3><div class="NLM_p last">To a solution of sulfonamide <b>2a</b> (100 mg, 0.33 mmol) in 1.5 mL of DMF were added potassium acetate (97 mg, 0.99 mmol), bis(pinacolato)diboron (100 mg, 0.40 mmol), and PdCl<sub>2</sub>(dppf) (7 mg, 10 μmol), under argon atmosphere. The reaction mixture was stirred at 100 °C for 3 h. After cooling to ambient temperature, 15 mL of ethyl acetate was added, the solution was filtered through Celite, and the organic layer was washed with brine (5 × 15 mL) before drying over NaSO<sub>4</sub>. The solvent was removed under reduced pressure to give the corresponding boronic acid pinacolester as a crude solid that was dissolved in 1.5 mL of DMF and treated with 4-bromo-7-azaindole (65 mg, 0.33 mmol), 0.5 mL of saturated NaHCO<sub>3</sub> (aq) and Pd(PPh<sub>3</sub>)<sub>4</sub> (38 mg, 36 μmol) under argon atmosphere. The mixture was stirred at 100 °C for 18 h, cooled to ambient temperature, and diluted with 20 mL of EtOAc. After filtration through Celite, the organic layer was washed with brine (5 × 20 mL) and dried over MgSO<sub>4</sub>, and the solvent was removed under reduced pressure. Purification was performed by column chromatography using a 10:1:0.1 mixture of DCM/MeOH/NH<sub>3</sub> in MeOH (7 N) to give <b>3a</b> (47 mg, 42%) as a colorless solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ = 11.89 (s, 1H), 8.33 (d, <i>J</i> = 4.9 Hz, 1H), 7.98 (s, 4H), 7.60 (d, <i>J</i> = 3.5 Hz, 1H), 7.25 (d, <i>J</i> = 4.9 Hz, 1H), 6.64 (d, <i>J</i> = 3.5 Hz, 1H), 2.91 (t, <i>J</i> = 6.8 Hz, 2H), 2.28 (t, <i>J</i> = 6.8 Hz, 2H), 2.07 (s, 6H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz,): δ = 149.2, 142.9, 142.2, 140.3, 138.5, 128.9, 127.2, 127.2, 117.1, 114.4, 98.8, 58.1, 45.0, 40.7. <i>m</i>/<i>z</i> (APCI): 345.13 ([M + H]<sup>+</sup>).</div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> <i>N</i>-(2-(Diethylamino)ethyl)-4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)benzenesulfonamide (<b>3b</b>)</h3><div class="NLM_p last">Starting from <b>2b</b>, the synthesis of <b>3b</b> was performed by following general procedure B. Purification was performed by column chromatography using a 10:1:0.1 mixture of DCM/MeOH/NH<sub>3</sub> in MeOH (7 N). Yield 41%, beige semisolid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 11.88 (s, 1H), 8.33 (d, <i>J</i> = 4.9 Hz, 1H), 8.04–7.90 (m, 4H), 7.60 (d, <i>J</i> = 3.5 Hz, 1H), 7.25 (d, <i>J</i> = 4.9 Hz, 1H), 6.63 (d, <i>J</i> = 3.5 Hz, 1H), 2.88 (dd, <i>J</i> = 8.1, 6.3 Hz, 2H), 2.43 (t, <i>J</i> = 7.2 Hz, 2H), 2.39 (q, <i>J</i> = 7.1 Hz, 4H), 0.86 (t, <i>J</i> = 7.1 Hz, 6H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz): δ = 149.2, 142.9, 142.2, 140.3, 138.5, 128.9, 127.2, 127.1, 117.1, 114.3, 98.7, 51.8, 46.5, 41.0, 11.6. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, DEPT) δ = 142.9, 128.9, 127.1, 114.36, 98.7, 51.8, 46.5, 41.0, 11.6. <i>m</i>/<i>z</i> (ESI): 373.18 ([M + H]<sup>+</sup>), 187.09 ([M + 2H]<sup>2+</sup>). Mp: 143 °C.</div></div><div id="sec5_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> <i>N</i>-(2-(Piperidin-1-yl)ethyl)-4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)benzenesulfonamide (<b>3c</b>)</h3><div class="NLM_p last">Starting from <b>2c</b>, the synthesis of <b>3c</b> was performed by following general procedure B. Purification was performed by column chromatography using a 15:1:0.1 mixture of DCM/MeOH/NH<sub>3</sub> in MeOH (7 N). Yield 58%, beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ = 11.88 (s, 1H), 8.32 (d, <i>J</i> = 4.8 Hz, 1H), 7.97 (s, 4H), 7.60 (d, <i>J</i> = 3.4 Hz, 1H), 7.25 (d, <i>J</i> = 4.8 Hz, 1H), 6.63 (d, <i>J</i> = 3.4 Hz, 1H), 2.92 (t, <i>J</i> = 6.9 Hz, 2H), 2.29 (t, <i>J</i> = 6.9 Hz, 2H), 2.23 (t, <i>J</i> = 5.0 Hz, 4H), 1.45–1.21 (m, 6H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz,): δ = 149.2, 142.9, 142.1, 140.3, 138.5, 128.9, 127.2, 127.1, 117.1, 114.4, 98.7, 57.6, 53.9, 40.3, 25.4, 23.9. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz, DEPT): δ = 142.9, 128.9, 127.2, 127.1, 114.4, 98.7, 57.6, 53.9, 40.2, 25.34, 23.9. <i>m</i>/<i>z</i> (APCI): 385.18 ([M + H]<sup>+</sup>). Mp: 204 °C.</div></div><div id="sec5_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> <i>N</i>-(2-Morpholinoethyl)-4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)benzenesulfonamide (<b>3d</b>)</h3><div class="NLM_p last">Starting from <b>2d</b>, the synthesis of <b>3d</b> was performed by following general procedure B. Purification was performed by column chromatography using a 50:1:0.5 mixture of DCM/MeOH/NH<sub>3</sub> in MeOH (7 N). Yield 69%, beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz,): δ = 11.88 (s, 1H), 8.33 (d, <i>J</i> = 4.9 Hz, 1H), 7.98 (s, 4H), 7.65 (t, <i>J</i> = 5.9 Hz, 1H), 7.60 (dd, <i>J</i> = 3.5, 2.5 Hz, 1H), 7.25 (d, <i>J</i> = 4.9 Hz, 1H), 6.64 (dd, <i>J</i> = 3.5, 1.8 Hz, 1H), 3.52–3.46 (m, 4H), 2.95 (q, <i>J</i> = 6.3 Hz, 2H), 2.33 (t, <i>J</i> = 6.8 Hz, 2H), 2.27 (t, <i>J</i> = 4.7 Hz, 4H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz) δ = 149.2, 142.9, 142.1, 140.3, 138.46, 128.9, 127.2, 127.1, 117.1, 114.4, 98.7, 66.0, 57.2, 53.1, 39.9. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz, DEPT) δ = 142.9, 128.9, 127.2, 127.1, 114.4, 98.7, 66.0, 57.2, 53.1, 39.8. <i>m</i>/<i>z</i> (APCI): 387.16 ([M + H]<sup>+</sup>). Mp: 201 °C.</div></div><div id="sec5_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> <i>tert</i>-Butyl(2-((4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)phenyl)-sulfonamido)ethyl)carbamate (<b>3e</b>)</h3><div class="NLM_p last">Starting from <b>2e</b>, the synthesis of <b>3e</b> was performed by following general procedure B. Purification was performed by column chromatography using a 20:1:0.1 mixture of DCM/MeOH/NH<sub>3</sub> in MeOH (7 N). Yield 60%, colorless solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>,500 MHz,): δ = 11.88 (s, 1H), 8.33 (d, <i>J</i> = 4.9 Hz, 1H), 8.01–7.93 (m, 4H), 7.76 (t, <i>J</i> = 6.0 Hz, 1H), 7.60 (dd, <i>J</i> = 3.5, 2.6 Hz, 1H), 7.26 (d, <i>J</i> = 4.9 Hz, 1H), 6.78 (t, <i>J</i> = 5.9 Hz, 1H), 6.66 (dd, <i>J</i> = 3.5, 1.8 Hz, 1H), 3.01 (q, <i>J</i> = 6.6 Hz, 2H), 2.84 (q, <i>J</i> = 6.4 Hz, 2H), 1.34 (s, 9H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz): δ = 155.5, 149.2, 142.9, 142.2, 140.0, 138.4, 129.0, 127.2, 127.1, 117.1, 114.4, 98.8, 77.8, 42.3, 28.1. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz, DEPT): δ = 142.9, 129.0, 127.2, 127.1, 114.4, 98.8, 42.3, 39.8, 28.2. <i>m</i>/<i>z</i> (APCI): 417.16 ([M + H]<sup>+</sup>).</div></div><div id="sec5_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> <i>N</i>-(2-Hydroxyethyl)-4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)ben-zenesulfonamide (<b>3f</b>)</h3><div class="NLM_p last">Starting from <b>2f</b>, the synthesis of <b>3f</b> was performed by following general procedure B. Purification was performed by column chromatography using a 10:1:0.1 mixture of DCM/MeOH/NH<sub>3</sub> in MeOH (7 N). Yield 66%, colorless solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 11.88 (s, 1H), 8.33 (d, <i>J</i> = 4.9 Hz, 1H), 8.01–7.94 (m, 4H), 7.71 (t, <i>J</i> = 5.9 Hz, 1H), 7.60 (t, <i>J</i> = 3.0 Hz, 1H), 7.26 (d, <i>J</i> = 5.0 Hz, 1H), 6.65 (dd, <i>J</i> = 3.5, 1.8 Hz, 1H), 4.70 (t, <i>J</i> = 5.6 Hz, 1H), 3.42 (q, <i>J</i> = 6.0 Hz, 2H), 2.88 (q, <i>J</i> = 6.2 Hz, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz): δ = 149.2, 142.9, 142.1, 140.2, 138.5, 128.9, 127.2, 127.12, 117.1, 114.4, 98.8, 59.9, 45.1. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz, DEPT): δ = 142.9, 128.9, 127.2, 127.1, 114.4, 98.8, 60.0, 45.1. <i>m</i>/<i>z</i> (ESI): 318.09 ([M + H]<sup>+</sup>).</div></div><div id="sec5_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> <i>N</i>-(2-Methoxyethyl)-4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)benzenesulfonamide (<b>3g</b>)</h3><div class="NLM_p last">Starting from <b>2g</b>, the synthesis of <b>3g</b> was performed by following general procedure B. Purification was performed by column chromatography using a 20:1:0.1 mixture of DCM/MeOH/NH<sub>3</sub> in MeOH (7 N). Yield 65%, colorless solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 11.88 (s, 1H), 8.33 (d, <i>J</i> = 4.9 Hz, 1H), 8.02–7.91 (m, 4H), 7.83 (t, <i>J</i> = 6.0 Hz, 1H), 7.60 (dd, <i>J</i> = 3.5, 2.5 Hz, 1H), 7.25 (d, <i>J</i> = 4.9 Hz, 1H), 6.65 (dd, <i>J</i> = 3.6, 1.8 Hz, 1H), 3.34 (t, <i>J</i> = 5.7 Hz, 2H), 2.99 (q, <i>J</i> = 5.8 Hz, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz): δ = 149.17, 142.92, 142.15, 140.35, 138.49, 128.87, 127.19, 127.10, 117.13, 114.38, 98.76, 70.56, 57.83, 42.24. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz, DEPT): δ = 142.92, 128.87, 127.19, 127.10, 114.38, 98.76, 70.56, 57.83, 42.24. <i>m</i>/<i>z</i> (APCI): 332.10 ([M + H]<sup>+</sup>). Mp: 215 °C.</div></div><div id="sec5_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> <i>N</i>-(2-Hydroxyethyl)-<i>N</i>-methyl-4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)benzenesulfonamide (<b>3h</b>)</h3><div class="NLM_p last">Starting from <b>2h</b>, the synthesis of <b>3h</b> was performed by following general procedure B. Purification was performed by column chromatography using a 20:1:0.1 mixture of DCM/MeOH/NH<sub>3</sub> in MeOH (7 N). Yield 42%, colorless solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 11.89 (s, 1H), 8.33 (d, <i>J</i> = 4.9 Hz, 1H), 8.01 (d, <i>J</i> = 8.4 Hz, 2H), 7.93 (d, <i>J</i> = 8.4 Hz, 2H), 7.60 (t, <i>J</i> = 3.0 Hz, 1H), 7.27 (d, <i>J</i> = 5.0 Hz, 1H), 6.66 (dd, <i>J</i> = 3.6, 1.8 Hz, 1H), 4.81 (td, <i>J</i> = 5.5, 1.3 Hz, 1H), 3.56 (q, <i>J</i> = 5.8 Hz, 2H), 3.10 (t, <i>J</i> = 6.1 Hz, 2H), 2.80 (s, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz): δ = 149.2, 142.9, 142.6, 138.25, 136.82, 129.03, 127.71, 127.24, 117.07, 114.36, 98.7, 59.1, 51.94 35.6. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz, DEPT): δ = 142.9, 142.7, 129.0, 127.7, 127.2, 114.4, 98.7, 59.1, 51.94, 35.6. <i>m</i>/<i>z</i> (APCI): 332.11 ([M + H]<sup>+</sup>).</div></div><div id="sec5_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 2-((4-(1<i>H</i>-Pyrrolo[2,3-<i>b</i>]pyridin-4-yl)phenyl)sulfonamido)-ethyl Acetate (<b>3i</b>)</h3><div class="NLM_p last">Starting from <b>2i</b>, the synthesis of <b>3i</b> was performed by following general procedure B. Purification was performed by column chromatography using a 15:1:0.1 mixture of DCM/MeOH/NH<sub>3</sub> in MeOH (7 N). Yield 23%, colorless solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 11.90 (s, 1H), 8.35 (d, <i>J</i> = 5.0 Hz, 1H), 8.03–7.96 (m, 5H), 7.62 (dd, <i>J</i> = 3.5, 2.5 Hz, 1H), 7.28 (d, <i>J</i> = 5.0 Hz, 1H), 6.67 (dd, <i>J</i> = 3.5, 1.9 Hz, 1H), 4.02 (t, <i>J</i> = 5.6 Hz, 2H), 3.12 (q, <i>J</i> = 5.7 Hz, 2H), 1.96 (s, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz): δ = 170.2, 149.18, 142.94, 142.28, 140.22, 138.45, 128.98, 127.23, 127.1, 117.1, 114.4, 98.8, 62.5, 41.4, 20.6. <i>m</i>/<i>z</i> (APCI): 360.10 ([M + H]<sup>+</sup>).</div></div><div id="sec5_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> <i>N</i>-(2-Hydroxy-2-methylpropyl)-4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)benzenesulfonamide (<b>3j</b>)</h3><div class="NLM_p last">Starting from <b>2j</b>, the synthesis of <b>3j</b> was performed by following general procedure B. Purification was performed by column chromatography using a 13:1:0.1 mixture of DCM/MeOH/NH<sub>3</sub> in MeOH (7 N). Yield 11%, colorless solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 11.87 (s, 1H), 8.33 (d, <i>J</i> = 4.9 Hz, 1H), 8.01–7.94 (m, 4H), 7.62–7.54 (m, 2H), 7.26 (d, <i>J</i> = 4.9 Hz, 1H), 6.65 (dd, <i>J</i> = 3.5, 1.9 Hz, 1H), 4.42 (s, 1H), 2.70 (d, <i>J</i> = 6.6 Hz, 2H), 1.08 (s, 6H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz): δ = 149.2, 142.9, 142.1, 140.4, 138.5, 128.9, 127.18, 127.16, 117.1, 114.4, 98.8, 68.8, 53.7, 27.1. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz, DEPT): δ = 142.9, 128.9, 127.18, 127.16, 114.4, 98.8, 53.7, 27.1. <i>m</i>/<i>z</i> (EI): 345 ([M]<sup>+</sup>).</div></div><div id="sec5_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 4-(4-((4-Methylpiperazin-1-yl)sulfonyl)phenyl)-1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridine (<b>3k</b>)</h3><div class="NLM_p last">Starting from <b>2k</b>, the synthesis of <b>3k</b> was performed by following general procedure B. Purification was performed by column chromatography using a 20:1:0.1 mixture of DCM/MeOH/NH<sub>3</sub> in MeOH (7 N). Yield 58%, colorless solid. <sup>1</sup>H NMR (D<sub>2</sub>O, as a TFA salt, 500 MHz): δ = 8.39 (d, <i>J</i> = 6.0 Hz, 1H), 8.00–7.87 (m, 4H), 7.72 (d, <i>J</i> = 3.6 Hz, 1H), 7.54 (d, <i>J</i> = 6.0 Hz, 1H), 6.82 (d, <i>J</i> = 3.6 Hz, 1H), 4.02 (d, <i>J</i> = 13.5 Hz, 2H), 3.68 (d, <i>J</i> = 12.7 Hz, 2H), 3.33 (td, <i>J</i> = 12.5, 3.3 Hz, 2H), 2.98 (s, 3H), 2.95–2.86 (m, 2H). <sup>13</sup>C NMR (D<sub>2</sub>O, as a TFA salt, 126 MHz,): δ = 146.8, 141.2, 139.6, 135.1, 133.6, 130.2, 130.0, 128.1, 122.9, 114.9, 101.7, 52.7, 43.2, 42.8. <sup>13</sup>C NMR (D<sub>2</sub>O, as a TFA salt, 126 MHz, DEPT): δ = 133.5, 130.2, 130.0, 128.1, 114.9, 101.7, 52.7, 43.3, 42.8. <i>m</i>/<i>z</i> (APCI): 357.15 ([M + H]<sup>+</sup>).</div></div><div id="sec5_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> ((4-(1<i>H</i>-Pyrrolo[2,3-<i>b</i>]pyridin-4-yl)phenyl)sulfonyl)-<span class="smallcaps smallerCapital">l</span>-phenylalanine (<b>3l</b>)</h3><div class="NLM_p last">To a solution of sulfonamide <b>2l</b> (200 mg, 0.52 mmol) in 3 mL of DMF were added potassium acetate (153 mg, 1.56 mmol), bis(pinacolato)diboron (159 mg, 0.62 mmol), and PdCl<sub>2</sub>(dppf) (13 mg, 15 μmol) under argon atmosphere. The reaction mixture was stirred at 100 °C for 3 h. After cooling to ambient temperature 20 mL of EtOAc was added, the solution was filtered through Celite, and the organic layer was washed with brine (5 × 20 mL) before drying over NaSO<sub>4</sub>. The solvent was removed under reduced pressure to give the corresponding boronic acid pinacolester as a crude solid that was dissolved in 3 mL of DMF and treated with 4-bromo-7-azaindole (103 mg, 0.52 mmol), 1 mL of saturated NaHCO<sub>3</sub> (aq), and Pd(PPh<sub>3</sub>)<sub>4</sub> (60 mg, 52 μmol) under argon atmosphere. The mixture was stirred at 100 °C for 17 h, cooled to ambient temperature, and diluted with 20 mL of EtOAc. After filtration through Celite, the organic layer was washed with brine (5 × 20 mL), and the product was precipitated by setting the pH to 4 using hydrochloric acid (0.1 N). Filtration and drying under reduced pressure gave <b>3l</b> (55 mg, 25%) as a colorless solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 11.89 (s, 1H), 8.41 (d, <i>J</i> = 9.0 Hz, 1H), 8.33 (d, <i>J</i> = 4.9 Hz, 1H), 7.80 (d, <i>J</i> = 8.1 Hz, 2H), 7.70 (d, <i>J</i> = 8.3 Hz, 2H), 7.61 (t, <i>J</i> = 3.0 Hz, 1H), 7.22 (d, <i>J</i> = 4.9 Hz, 1H), 7.20–7.10 (m, 5H), 6.62 (dd, <i>J</i> = 3.5, 1.7 Hz, 1H), 3.95 (dt, <i>J</i> = 8.8, 4.0 Hz, 1H), 2.99 (dd, <i>J</i> = 13.8, 5.5 Hz, 1H), 2.79–2.72 (m, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz): δ = 172.4, 149.2, 142.9, 141.9, 140.6, 138.6, 136.8, 129.1, 128.5, 128.1, 127.2, 126.8, 126.4, 117.1, 114.3, 98.8, 57.5, 57.5, 37.8. <i>m</i>/<i>z</i> (APCI): 422.12 ([M + H]<sup>+</sup>).</div></div><div id="sec5_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 2-(4-Bromobenzamido)ethyl 4-Bromobenzoate (<b>5</b>)</h3><div class="NLM_p last">A solution of 4-bromobenzoyl chloride (<b>4</b>, 2.00 g, 9.11 mmol) in 10 mL of DCM was dropwise added to a solution of monoethanolamine (0.67 g, 10.94 mmol) and triethylamine (1.08 g, 6.78 mmol) in 10 mL of DCM over the course of 20 min. The mixture was stirred at ambient temperature for 24 h, diluted with 10 mL DCM, and washed with saturated NaHCO<sub>3</sub> (aq, 3 × 5 mL). The organic layer was dried over MgSO<sub>4</sub>, and the solvent was removed to give a colorless residue that was recrystallized from EtOH to give <b>5</b> (0.60 g, 48%) as a colorless solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 8.76 (t, <i>J</i> = 5.6 Hz, 1H), 7.92–7.87 (m, 2H), 7.81–7.76 (m, 2H), 7.75–7.70 (m, 2H), 7.70–7.64 (m, 2H), 4.40 (t, <i>J</i> = 5.5 Hz, 2H), 3.64 (q, <i>J</i> = 5.6 Hz, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz): δ = 165.7, 165.1, 133.4, 131.8, 131.3, 131.2, 129.3, 128.9, 127.3, 124.9, 63.6, 38.4. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz, DEPT): δ = 131.8, 131.3, 131.2, 129.3, 63.6, 38.4. <i>m</i>/<i>z</i> (ESI): 425.93 ([M + H]<sup>+</sup>).</div></div><div id="sec5_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 4-Bromo-<i>N</i>-(2-hydroxyethyl)benzamide (<b>6</b>)</h3><div class="NLM_p last">To a solution of ester <b>5</b> (0.50 g, 1.17 mmol) in 20 mL of THF was added 2 mL of 50% w/v aq. KOH solution, and the mixture was stirred at 60 °C for 16 h. The solvent was removed under reduced pressure, and the resulting white residue was taken up in 20 mL of EtOAc and washed with water (3 × 5 mL) and brine (5 mL). Purification was performed by column chromatography using a 15:1 mixture of DCM/MeOH to give <b>6</b> (262 mg, 92%) as a colorless solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 8.49 (t, <i>J</i> = 5.7 Hz, 1H), 7.82–7.77 (m, 2H), 7.68–7.65 (m, 2H), 4.70 (t, <i>J</i> = 5.6 Hz, 1H), 3.51 (q, <i>J</i> = 5.9 Hz, 2H), 3.32 (q, <i>J</i> = 6.1 Hz, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz): δ = 165.4, 133.7, 131.2, 129.3, 124.73, 59.6, 42.2. <i>m</i>/<i>z</i> (EI): 243 ([M]<sup>+</sup>).</div></div><div id="sec5_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> <i>N</i>-(2-Hydroxyethyl)-4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)benzamide (<b>7</b>)</h3><div class="NLM_p last">Starting from <b>6</b>, the synthesis of <b>7</b> was performed by following general procedure B. Purification was performed by column chromatography using a stepwise gradient from a 15:1:0.1 to a 10:1:0.1 mixture of DCM/MeOH/10% aq. NH<sub>4</sub>OH (25%) in MeOH. Yield 43%, colorless solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 11.83 (s, 1H), 8.53 (t, <i>J</i> = 5.6 Hz, 1H), 8.31 (d, <i>J</i> = 4.9 Hz, 1H), 8.07–8.00 (m, 2H), 7.89–7.82 (m, 2H), 7.60–7.55 (m, 1H), 7.23 (d, <i>J</i> = 4.9 Hz, 1H), 6.63 (dd, <i>J</i> = 3.6, 1.8 Hz, 1H), 4.75 (t, <i>J</i> = 5.6 Hz, 1H), 3.56 (q, <i>J</i> = 6.0 Hz, 2H), 3.39 (q, <i>J</i> = 6.1 Hz, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz): δ = 165.9, 149.2, 142.9, 141.0, 139.3, 134.2, 128.0, 127.8, 126.9, 117.2, 114.3, 99.0, 59.8, 42.2. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz, DEPT): δ = 142.9, 128.0, 127.9, 126.9, 114.3, 98.9, 59.8, 42.3. <i>m</i>/<i>z</i> (ESI): 282.12 ([M + H]<sup>+</sup>).</div></div><div id="sec5_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> (<i>R</i>)-4-(2-Amino-3-hydroxypropyl)phenol (<b>8d</b>)</h3><div class="NLM_p last">To a stirred solution of <span class="smallcaps smallerCapital">d</span>-tyrosine methyl ester hydrochloride (2.00 g, 8.63 mmol) in 20 mL of acetonitrile was added potassium carbonate (2.98 g, 21.6 mmol), and the solution was heated to 50 °C for 16 h. Filtration of the formed salt and evaporation of the solvent under reduced pressure gave an orange oil that was taken up in 20 mL of methanol. Under cooling to 0 °C in an ice bath, NaBH<sub>4</sub> (491 mg, 12.99 mmol) was added in small portions over 30 min. The cooling bath was removed, and the solution was stirred at ambient temperature for 5 h. The solvent was removed under reduced pressure, the resulting residue was taken up in 50 mL of water, and the aqueous layer was extracted with ethyl acetate (5 × 50 mL). The combined organic layers were dried over MgSO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to give <b>8d</b> (878 mg, 61%) as a crude yellow solid that was used for the next step without further purification. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 6.68–6.65 (m, 2H), 6.37–6.34 (m, 2H), 2.97 (dd, <i>J</i> = 10.3, 4.5 Hz, 1H), 2.87–2.83 (m, 1H), 2.24 (dd, <i>J</i> = 13.4, 6.0 Hz, 1H), 2.20–2.09 (m, 1H), 2.02 (dd, <i>J</i> = 13.4, 7.5 Hz, 1H), 1.44 (s, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 155.5, 130.0, 129.5, 115.0, 65.5, 54.6, 39.0. <i>m</i>/<i>z</i> (EI): 167 ([M]<sup>+</sup>).</div></div><div id="sec5_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> (<i>S</i>)-2-Amino-3-(1<i>H</i>-indol-3-yl)propan-1-ol (<b>8e</b>)</h3><div class="NLM_p last">An oven-dried three-neck flask was charged with <span class="smallcaps smallerCapital">l</span>-tryptophan (2.04 g, 10.0 mmol), sodium borohydride (757 mg, 20.0 mmol), and 20 mL of dry THF and equipped with a dropping funnel and a reflux condenser under argon atmosphere. A solution of iodine (2.54 g, 10.0 mmol) in 10 mL dry THF was added dropwise over the course of 30 min at 0 °C under cooling in an ice bath. Once the evolution of gas had stopped, the ice bath was removed, and the mixture was heated to reflux for 20 h. After cooling to ambient temperature, methanol was slowly added under vigorous stirring until the solution became clear. After stirring for another 15 min, removal of the solvents under reduced pressure gave a white residue that was dissolved in 25 mL of aq. KOH solution (20% w/v) and stirred for 16 h at ambient temperature. Adjustment to pH 11 by addition of hydrochloric acid (2 M) was followed by extraction of the aqueous layer with DCM (3 × 25 mL) and a 1:4 mixture of ethanol/chloroform (5 × 25 mL). The combined organic layers were washed with water (3 × 25 mL) and brine (25 mL) and dried over MgSO<sub>4</sub>. The solvent was removed under reduced pressure to give <b>8e</b> (1.36 g, 72%) as a crude yellow oil that was used without further purification. <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ = 136.5, 127.7, 122.9, 122.1, 119.4, 118.9, 112.4, 111.4, 66.6, 53.1, 30.1. <i>m</i>/<i>z</i> (ESI): 191.12 ([M + H]<sup>+</sup>).</div></div><div id="sec5_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> (<i>R</i>)-2-Amino-3-(1<i>H</i>-indol-3-yl)propan-1-ol (<b>8f</b>)</h3><div class="NLM_p last">An oven-dried three-neck flask was charged with <span class="smallcaps smallerCapital">d</span>-tryptophan (1.57 g, 7.69 mmol), sodium borohydride (640 mg, 16.91 mmol), and 15 mL of dry THF and equipped with a dropping funnel and a reflux condenser under argon atmosphere. A solution of iodine (1.95 g, 7.69 mmol) in 8 mL of dry THF was added dropwise over the course of 30 min at 0 °C under cooling in an ice bath. Once the evolution of gas had stopped, the ice bath was removed, and the mixture was heated to reflux for 19 h. After the mixture was cooled to ambient temperature, methanol was slowly added under vigorous stirring until the solution became clear. After stirring for another 15 min, removal of the solvents under reduced pressure gave a white residue that was dissolved in 20 mL of aq. KOH solution (20% w/v) and stirred for 15 h at ambient temperature. Adjustment to pH 11 by addition of hydrochloric acid (2 M) was followed by extraction of the aqueous layer with DCM (3 × 25 mL) and a 1:4 mixture of ethanol/chloroform (5 × 25 mL). The combined organic layers were washed with water (3 × 25 mL) and brine (25 mL) and dried over MgSO<sub>4</sub>. The solvent was removed under reduced pressure to give <b>8f</b> (1.22 g, 84%) as a crude yellow oil that was used for the next step without further purification. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 10.83 (s, 1H), 7.54 (d, <i>J</i> = 7.9 Hz, 1H), 7.34 (d, <i>J</i> = 8.0 Hz, 1H), 7.15 (d, <i>J</i> = 1.8 Hz, 1H), 7.06 (ddt, <i>J</i> = 8.2, 5.0, 1.4 Hz, 1H), 6.99–6.94 (m, 1H), 3.41–3.35 (m, 1H), 3.24 (m, 1H), 3.01 (q, <i>J</i> = 6.1 Hz, 1H), 2.81 (dd, <i>J</i> = 14.2, 6.1 Hz, 1H), 2.60 (dd, <i>J</i> = 14.2, 7.1 Hz, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz): δ = 136.3, 127.58, 123.3, 120.8, 118.4, 118.1, 111.6, 111.3, 65.7, 53.6, 29.4. <i>m</i>/<i>z</i> (ESI): 191.12 ([M + H]<sup>+</sup>).</div></div><div id="sec5_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> (<i>S</i>)-4-Bromo-<i>N</i>-(1-hydroxy-3-phenylpropan-2-yl)benzene-sulfonamide (<b>9a</b>)</h3><div class="NLM_p last">Synthesis of <b>9a</b> was performed by following general procedure A using <span class="smallcaps smallerCapital">l</span>-phenylalaninol (<b>8a</b>). Yield 98%, colorless solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz) δ = 7.60–7.56 (m, 2H), 7.50–7.46 (m, 2H), 7.12 (dd, <i>J</i> = 5.2, 1.9 Hz, 3H), 7.06–7.00 (m, 2H), 4.80 (s, 1H), 3.33 (dd, <i>J</i> = 10.1, 4.2 Hz, 1H), 3.30–3.24 (m, 1H), 3.21 (dd, <i>J</i> = 9.9, 6.6 Hz, 1H), 2.83 (dd, <i>J</i> = 13.7, 5.3 Hz, 1H), 2.46 (dd, <i>J</i> = 13.7, 8.0 Hz, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz) δ = 140.9, 138.3, 131.8, 129.1, 128.0, 128.0, 125.7, 125.5, 63.2, 57.3, 37.0. <i>m</i>/<i>z</i> (APCI): 370.01 ([M + H]<sup>+</sup>).</div></div><div id="sec5_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> (<i>R</i>)-4-Bromo-<i>N</i>-(1-hydroxy-3-phenylpropan-2-yl)benzene-sulfonamide (<b>9b</b>)</h3><div class="NLM_p last">Synthesis of <b>9b</b> was performed by following general procedure A using <span class="smallcaps smallerCapital">d</span>-phenylalaninol (<b>8b</b>). Yield 99%, colorless solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 300 MHz): δ = 7.48 (s, 4H), 7.21–7.13 (m, 3H), 7.00–6.93 (m, 2H), 3.69 (dd, <i>J</i> = 11.1, 4.1 Hz, 1H), 3.59 (dd, <i>J</i> = 11.1, 4.6 Hz, 1H), 3.53–3.39 (m, 1H), 2.83 (dd, <i>J</i> = 13.9, 6.3 Hz, 1H), 2.68 (dd, <i>J</i> = 14.0, 8.2 Hz, 1H). <i>m</i>/<i>z</i> (ESI): 370.01 ([M + H]<sup>+</sup>).</div></div><div id="sec5_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> (<i>S</i>)-4-Bromo-<i>N</i>-(1-hydroxy-3-(4-hydroxyphenyl)propan-2-yl)benzenesulfonamide (<b>9c</b>)</h3><div class="NLM_p last">To a solution of <span class="smallcaps smallerCapital">l</span>-tyrosinol (469 mg, 2.15 mmol) in 10 mL of a 1:1 mixture of water/acetone was added sodium bicarbonate (658 mg, 7.83 mmol), and the mixture was stirred at ambient temperature for 15 min. 4-Bromobenzene-sulfonyl chloride (<b>1</b>, 500 mg, 1.96 mmol) was added, and stirring was continued for 17 h. After evaporation of the volatile solvent under reduced pressure, 10 mL of ethyl acetate was added, and the layers were separated. The aqueous layer was extracted with ethyl acetate (3 × 5 mL), and the combined organic layers were washed with saturated NaHCO<sub>3</sub> (aq, 3 × 5 mL) and dried over MgSO<sub>4</sub>. The solvent was removed under reduced pressure to give crude crystals that were recrystallized from methanol to give <b>9c</b> (455 mg, 60%) as colorless crystals. <sup>1</sup>H NMR (DMSO-<i>d</i><sub><i>6</i></sub>, 500 MHz): δ = 9.09 (s, 1H), 7.67 (d, <i>J</i> = 7.3 Hz, 2H), 7.67–7.60 (m, 3H), 7.55–7.49 (m, 2H), 6.87–6.80 (m, 2H), 6.59–6.52 (m, 2H), 4.69 (t, <i>J</i> = 5.1 Hz, 1H), 3.30–3.25 (m, 2H), 3.22–3.14 (m, 2H), 2.69 (dd, <i>J</i> = 13.8, 5.4 Hz, 1H), 2.35 (dd, <i>J</i> = 13.8, 6.9 Hz, 1H). <i>m</i>/<i>z</i> (ESI): 386.01 ([M + H]<sup>+</sup>).</div></div><div id="sec5_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> (<i>R</i>)-4-Bromo-<i>N</i>-(1-hydroxy-3-(4-hydroxyphenyl)propan-2-yl)benzenesulfonamide (<b>9d</b>)</h3><div class="NLM_p last">To a solution of <span class="smallcaps smallerCapital">d</span>-tyrosinol (<b>8d</b>, 229 mg, 1.37 mmol) in 7 mL of a 1:1 mixture of water/acetone was added sodium bicarbonate (460 mg, 5.48 mmol), and the mixture was stirred at ambient temperature for 15 min. 4-Bromobenzenesulfonyl chloride (<b>1</b>, 350 mg, 1.37 mmol) was added, and the solution was heated to 40 °C for 4 h. After evaporation of the volatile solvent under reduced pressure, 7 mL of ethyl acetate was added, and the layers were separated. The aqueous layer was extracted with ethyl acetate (3 × 5 mL), and the combined organic layers were washed with saturated NaHCO<sub>3</sub> (aq, 3 × 5 mL) and dried over MgSO<sub>4</sub>. The solvent was removed under reduced pressure to give <b>9d</b> (371 mg, 70%) as a crude yellow solid that was used in the next step without further purification. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz): δ = 7.68–7.58 (m, 2H), 7.52 (d, <i>J</i> = 8.5 Hz, 2H), 6.88–6.78 (m, 2H), 6.60–6.51 (m, 2H), 4.71 (s, 1H), 3.25–3.12 (m, 4H), 2.69 (dd, <i>J</i> = 13.7, 5.5 Hz, 1H), 2.35 (dd, <i>J</i> = 13.9, 6.9 Hz, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 75 MHz) δ = 155.6, 141.0, 132.9, 131.8, 130.1, 130.0, 128.2, 114.8, 62.8, 57.4, 36.2. <i>m</i>/<i>z</i> (ESI): 386.01 ([M + H]<sup>+</sup>).</div></div><div id="sec5_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> (<i>S</i>)-4-Bromo-<i>N</i>-(1-hydroxy-3-(1<i>H</i>-indol-3-yl)propan-2-yl)-benzenesulfonamide (<b>9e</b>)</h3><div class="NLM_p last">To a stirred solution of <span class="smallcaps smallerCapital">l</span>-tryptophanol (<b>8e</b>, 400 mg, 2.10 mmol) in 40 mL of a 1:2 mixture of DCM/acetonitrile was added <i>N</i>,<i>N</i>′-diisopropylethylamine (326 mg, 2.52 mmol). The mixture was stirred at ambient temperature for 1 h, and 4-bromobenzene-sulfonyl chloride (<b>1</b>, 537 mg, 2.10 mmol) was added in small portions over 30 min at 0 °C under cooling in an ice bath. The cooling bath was removed, and the solution was stirred at ambient temperature for 24 h. Removal of the solvent under reduced pressure gave a brown residue that was taken up in 30 mL of DCM, washed with hydrochloric acid (1 N, 3 × 5 mL), water (3 × 5 mL), and brine (5 mL) and dried over MgSO<sub>4</sub>. The solvent was removed under reduced pressure to give a crude brown oil that was purified by column chromatography using a 1:1 mixture of ethyl acetate/cyclohexane yielding <b>9e</b> (257 mg, 30%) as a yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 10.70 (s, 1H), 7.73 (d, <i>J</i> = 6.9 Hz, 1H), 7.50–7.43 (m, 4H), 7.28 (dt, <i>J</i> = 7.8, 1.0 Hz, 2H), 7.07–7.01 (m, 2H), 6.91 (tt, <i>J</i> = 6.9, 0.9 Hz, 1H), 4.79 (t, <i>J</i> = 5.3 Hz, 1H), 3.42–3.39 (m, 1H), 3.33–3.25 (m, 2H), 2.95 (dd, <i>J</i> = 14.3, 6.0 Hz, 1H), 2.61 (dd, <i>J</i> = 14.3, 7.0 Hz, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz): δ = 140.6, 136.1, 131.4, 127.9, 127.0, 125.4, 123.7, 120.6, 118.1, 117.9, 111.4, 110.3, 63.2, 56.1, 27.0. <i>m</i>/<i>z</i> (ESI): 409.02 ([M + H]<sup>+</sup>).</div></div><div id="sec5_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> (<i>R</i>)-4-Bromo-<i>N</i>-(1-hydroxy-3-(1<i>H</i>-indol-3-yl)propan-2-yl)benzenesulfonamide (<b>9f</b>)</h3><div class="NLM_p last">To a stirred solution of <span class="smallcaps smallerCapital">d</span>-tryptophanol (<b>8f</b>, 625 mg, 3.29 mmol) in 60 mL of a 1:2 mixture of DCM/acetonitrile was added <i>N</i>,<i>N</i>′-diisopropylethylamine (510 mg, 3.94 mmol). The mixture was stirred at ambient temperature for 1 h, and 4-bromobenzene-sulfonyl chloride (<b>1</b>, 839 mg, 3.29 mmol) was added in small portions over 30 min at 0 °C under cooling in an ice bath. The cooling bath was removed, and the solution was stirred at ambient temperature for 24 h. Removal of the solvent under reduced pressure gave a brown residue that was taken up in 40 mL of DCM, washed with hydrochloric acid (1 N, 3 × 7 mL), water (3 × 7 mL), and brine (7 mL) and dried over MgSO<sub>4</sub>. The solvent was removed under reduced pressure to give a crude brown oil that was purified by column chromatography using a 1:1 mixture of ethyl acetate/cyclohexane yielding <b>9f</b> (579 mg, 43%) as a yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 10.66 (s, 1H), 7.69 (d, <i>J</i> = 7.0 Hz, 1H), 7.50–7.41 (m, 4H), 7.30–7.23 (m, 2H), 7.05–6.98 (m, 2H), 6.93–6.85 (m, 1H), 4.75 (t, <i>J</i> = 5.4 Hz, 1H), 3.40–3.36 (m, 1H), 3.33–3.24 (m, 2H), 2.94 (dd, <i>J</i> = 14.4, 6.0 Hz, 1H), 2.60 (dd, <i>J</i> = 14.3, 7.0 Hz, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz): δ = 140.6, 136.1, 131.4, 127.9, 127.0, 125.4, 123.7, 120.6, 118.1, 117.9, 111.4, 110.3, 63.2, 56.1, 27.0. <i>m</i>/<i>z</i> (ESI): 409.02 ([M + H]<sup>+</sup>).</div></div><div id="sec5_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> (<i>S</i>)-<i>N</i>-(1-Hydroxy-3-phenylpropan-2-yl)-4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)benzenesulfonamide (<b>10a</b>)</h3><div class="NLM_p last">Starting from <b>9a</b>, the synthesis of <b>10a</b> was performed by following general procedure B. Purification was performed by column chromatography using a 20:1:0.1 mixture of DCM/MeOH/10% aq. NH<sub>4</sub>OH (25%) in MeOH. Yield 42%, beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 11.90 (s, 1H), 8.35 (d, <i>J</i> = 14.7, 5.0 Hz, 1H), 7.86–7.67 (m, 4H), 7.59 (t, <i>J</i> = 22.3, 3.1 Hz, 1H), 7.28 (d, <i>J</i> = 50.0, 4.9 Hz, 1H), 7.16–6.99 (m, 4H), 6.58 (t, 1H), 4.87 (t, <i>J</i> = 5.5 Hz, 1H), 3.46–3.40 (m, 1H), 3.35–3.31 (m, 1H), 3.31–3.20 (m, 1H), 2.87 (dd, <i>J</i> = 13.6, 5.5 Hz, 1H), 2.47 (d, <i>J</i> = 8.4 Hz, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz): δ = 149.2, 142.9, 141.6, 141.2, 138.6, 138.4, 129.1, 128.6, 128.0, 127.2, 126.8, 125.9, 117.1, 114.31, 98.9, 63.3, 57.3, 37.0. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz, DEPT): δ = 142.9, 129.1, 128.6, 128.0, 127.1, 126.8, 125.8, 114.3, 98.8, 63.3, 57.3, 37.1. <i>m</i>/<i>z</i> (ESI): 408.14 ([M + H]<sup>+</sup>). <i>m</i>/<i>z</i> (ESI-HRMS): 408.1378 ([M + H]<sup>+</sup>).</div></div><div id="sec5_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> (<i>R</i>)-<i>N</i>-(1-Hydroxy-3-phenylpropan-2-yl)-4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)benzenesulfonamide (<b>10b</b>)</h3><div class="NLM_p last">Starting from <b>9b</b>, the synthesis of <b>10b</b> was performed by following general procedure B. Purification was performed by column chromatography using a 20:1:0.1 mixture of DCM/MeOH/10% aq. NH<sub>4</sub>OH (25%) in MeOH. Yield 56%, beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 11.88 (s, 1H), 8.34 (d, <i>J</i> = 4.9 Hz, 1H), 7.80–7.72 (m, 4H), 7.60 (t, <i>J</i> = 2.9 Hz, 1H), 7.22 (dd, <i>J</i> = 4.9, 1.1 Hz, 1H), 7.15–6.98 (m, 5H), 6.72–6.54 (m, 1H), 4.86 (t, <i>J</i> = 5.4 Hz, 1H), 3.44–3.40 (m, 1H), 3.37–3.32 (m, 1H), 3.28 (dt, <i>J</i> = 11.1, 6.1 Hz, 1H), 2.88 (dd, <i>J</i> = 13.7, 5.4 Hz, 1H), 2.50 (dd, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz): δ = 149.15, 142.95, 141.64, 141.22, 138.63, 138.37, 129.14, 128.59, 127.97, 127.14, 126.76, 125.85, 117.13, 114.32, 98.80, 63.27, 57.28, 37.07. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz, DEPT): δ = 143.0, 129.2, 128.6, 128.0, 127.2, 126.8, 125.9, 114.4, 98.8, 63.3, 57.3, 37.1. <i>m</i>/<i>z</i> (ESI): 815.09 ([2M + H]<sup>+</sup>), 408.14 ([M + H]<sup>+</sup>). <i>m</i>/<i>z</i> (ESI-HRMS): 408.1374 ([M + H]<sup>+</sup>).</div></div><div id="sec5_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> (<i>S</i>)-<i>N</i>-(1-Hydroxy-3-(4-hydroxyphenyl)propan-2-yl)-4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)benzenesulfonamide (<b>10c</b>)</h3><div class="NLM_p last">Starting from <b>9c</b>, the synthesis of <b>10c</b> was performed by following general procedure B. Purification was performed by column chromatography using a 10:1:0.1 mixture of DCM/MeOH/10% aq. NH<sub>4</sub>OH (25%) in MeOH. Yield 51%, colorless solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 11.88 (s, 1H), 9.14 (s, 1H), 8.33 (d, <i>J</i> = 4.9 Hz, 1H), 7.86–7.74 (m, 4H), 7.71 (d, <i>J</i> = 7.3 Hz, 1H), 7.59 (t, <i>J</i> = 3.0 Hz, 1H), 7.26 (d, <i>J</i> = 4.9 Hz, 1H), 6.88 (d, <i>J</i> = 8.5 Hz, 2H), 6.63 (dd, <i>J</i> = 3.6, 1.8 Hz, 1H), 6.58 (d, <i>J</i> = 8.4 Hz, 2H), 4.79 (t, <i>J</i> = 5.3 Hz, 1H), 3.38–3.33 (m, 1H), 3.25 (p, <i>J</i> = 6.1 Hz, 2H), 2.75 (dd, <i>J</i> = 13.7, 5.5 Hz, 1H), 2.40 (dd, <i>J</i> = 13.7, 6.9 Hz, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz): δ = 155.7, 149.2, 143.0, 141.8, 141.4, 138.7, 130.1, 128.6, 128.4, 127.2, 126.9, 117.2, 114.9, 114.48, 98.9, 62.9, 57.5, 36.3. <i>m</i>/<i>z</i> (ESI): 424.13 ([M + H]<sup>+</sup>). <i>m</i>/<i>z</i> (ESI-HRMS): 424.1326 ([M + H]<sup>+</sup>).</div></div><div id="sec5_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> (<i>R</i>)-<i>N</i>-(1-Hydroxy-3-(4-hydroxyphenyl)propan-2-yl)-4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)benzenesulfonamide (<b>10d</b>)</h3><div class="NLM_p last">Starting from <b>9d</b>, the synthesis of <b>10d</b> was performed by following general procedure B. Purification was performed by column chromatography using a 10:1:0.1 mixture of DCM/MeOH/10% aq. NH<sub>4</sub>OH (25%) in MeOH. Yield 26%, colorless solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 11.87 (s, 1H), 9.10 (d, <i>J</i> = 1.3 Hz, 1H), 8.33 (d, <i>J</i> = 4.9 Hz, 1H), 7.86–7.80 (m, 2H), 7.79–7.73 (m, 2H), 7.67 (d, <i>J</i> = 7.0 Hz, 1H), 7.59 (t, <i>J</i> = 2.9 Hz, 1H), 7.26 (dd, <i>J</i> = 5.0, 1.4 Hz, 1H), 6.87 (d, <i>J</i> = 7.1 Hz, 2H), 6.63 (q, <i>J</i> = 3.5, 1.6 Hz, 1H), 6.57 (d, <i>J</i> = 8.4 Hz, 2H), 4.75 (t, <i>J</i> = 5.2 Hz, 1H), 3.31–3.15 (m, 3H), 2.74 (dd, <i>J</i> = 13.7, 5.4 Hz, 1H), 2.40 (dd, <i>J</i> = 13.8, 6.8 Hz, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz): δ = 155.6, 149.1, 142.9, 141.7, 141.3, 138.6, 130.1, 128.6, 128.4, 127.1, 126.9, 117.2, 114.9, 114.4, 98.8, 62.8, 57.4, 36.3. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz, DEPT): δ = 142.9, 130.1, 128.6, 127.1, 126.9, 114.9, 114.4, 98.8, 62.8, 36.3. <i>m</i>/<i>z</i> (ESI): 424.13 ([M + H]<sup>+</sup>). <i>m</i>/<i>z</i> (ESI-HRMS): 424.1328 ([M + H]<sup>+</sup>).</div></div><div id="sec5_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> (<i>S</i>)-<i>N</i>-(1-Hydroxy-3-(1<i>H</i>-indol-3-yl)propan-2-yl)-4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)benzenesulfonamide (<b>10e</b>)</h3><div class="NLM_p last">Starting from <b>9e</b>, the synthesis of <b>10e</b> was performed by following general procedure B. Purification was performed by column chromatography using a 20:1:0.1 mixture of DCM/MeOH/10% aq. NH<sub>4</sub>OH (25%) in MeOH. Yield 60%, beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz,): δ = 11.85 (s, 1H), 10.69 (s, 1H), 8.32 (d, <i>J</i> = 4.9 Hz, 1H), 7.79–7.73 (m, 4H), 7.70 (d, <i>J</i> = 6.5 Hz, 1H), 7.57 (t, <i>J</i> = 2.9 Hz, 1H), 7.31 (d, <i>J</i> = 7.9 Hz, 1H), 7.20 (dd, <i>J</i> = 7.0, 5.3 Hz, 2H), 7.07 (d, <i>J</i> = 1.8 Hz, 1H), 6.93 (td, <i>J</i> = 8.1, 1.3 Hz, 1H), 6.86 (t, <i>J</i> = 7.4 Hz, 1H), 6.59 (dd, <i>J</i> = 3.4, 1.7 Hz, 1H), 4.74 (s, 1H), 3.42–3.38 (m, 2H), 3.37–3.26 (m, 2H), 2.97 (dd, <i>J</i> = 14.3, 6.4 Hz, 1H), 2.67 (dd, <i>J</i> = 14.3, 6.0 Hz, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz): δ = 149.1, 142.8, 141.6, 141.2, 138.6, 136.1, 128.4, 127.2, 127.1, 126.8, 123.8, 120.7, 118.1, 118.0, 117.1, 114.40, 111.3, 110.4, 98.9, 62.9, 56.2, 27.1. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz, DEPT): δ = 142.9, 128.4, 127.1, 126.8, 123.8, 120.7, 118.1, 118.0, 114.4, 111.3, 98.9, 62.9, 56.2, 27.1. <i>m</i>/<i>z</i> (ESI): 447.15 ([M + H]<sup>+</sup>). <i>m</i>/<i>z</i> (ESI-HRMS): 447.1484 ([M + H]<sup>+</sup>).</div></div><div id="sec5_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> (<i>R</i>)-<i>N</i>-(1-Hydroxy-3-(1<i>H</i>-indol-3-yl)propan-2-yl)-4-(1<i>H</i>-pyrrolo[2,3-<i>b</i>]pyridin-4-yl)benzenesulfonamide (<b>10f</b>)</h3><div class="NLM_p last">Starting from <b>9f</b>, the synthesis of <b>10f</b> was performed by following general procedure B. Purification was performed by column chromatography using a 20:1:0.1 mixture of DCM/MeOH/10% aq. NH<sub>4</sub>OH (25%) in MeOH. Yield 62%, beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ = 11.86 (s, 1H), 10.71 (s, 1H), 8.33 (d, <i>J</i> = 4.9 Hz, 1H), 7.79–7.74 (m, 4H), 7.71 (d, <i>J</i> = 7.0 Hz, 1H), 7.58 (dd, <i>J</i> = 3.5, 2.6 Hz, 1H), 7.32 (d, <i>J</i> = 7.0 Hz, 1H), 7.22 (d, <i>J</i> = 4.9 Hz, 1H), 7.20 (d, <i>J</i> = 8.2 Hz, 1H), 7.08 (d, <i>J</i> = 2.3 Hz, 1H), 6.94 (ddd, <i>J</i> = 8.1, 7.0, 1.1 Hz, 1H), 6.87 (ddd, <i>J</i> = 7.9, 7.0, 1.0 Hz, 1H), 6.60 (dd, <i>J</i> = 3.5, 1.9 Hz, 1H), 4.76 (t, <i>J</i> = 5.4 Hz, 1H), 3.41–3.37 (m, 2H), 3.32–3.29 (m, 1H), 2.98 (dd, <i>J</i> = 14.3, 6.5 Hz, 1H), 2.67 (dd, <i>J</i> = 14.3, 6.2 Hz, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 126 MHz): δ = 149.2, 142.9, 141.6, 141.2, 138.6, 136.1, 128.4, 127.2, 127.1, 126.8, 123.8, 120.7, 118.1, 118.0, 117.1, 114.4, 111.3, 110.4, 98.9, 62.9, 56.2, 27.0. <i>m</i>/<i>z</i> (ESI): 893.29 ([2M + H]<sup>+</sup>), 447.15 ([M + H]<sup>+</sup>). <i>m</i>/<i>z</i> (ESI-HRMS): 447.1485 ([M + H]<sup>+</sup>).</div></div><div id="sec5_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> Molecular Modeling</h3><div class="NLM_p">Molecular modeling was performed using the Molecular Operating Environment (Molecular Operating Environment MOE, 2016.0802; Chemical Computing Group ULC, 1010 Sherbooke St. West, Suite #910, Montreal, QC, Canada, H3A 2R7, 2021) software package, employing an “Amber10” force field.</div><div class="NLM_p">Isolation of the monomeric kinase domains was achieved by manual deletion of the bridged chain using the “sequence editor”. Superposition of the isolated chains was performed with the “align/superpose” tool of the “sequence editor”.</div><div class="NLM_p">Reconstruction of the glycine-rich loop (P-Loop) was performed using the “SuperLooper2” web server.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Starting from the crystal structure of Nek1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4B9D">4B9D</a>.B), the missing loop sequence EGSFG (between G11 and K17) was reconstructed by using the AGPAG loop (between P311 and R317) of the <i>Mycobacterium tuberculosis</i> probable periplasmic sugar-binding lipoprotein UspC (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K2Y">5K2Y</a>) crystal structure as a template.</div><div class="NLM_p">The Nek1 kinase domain homology model was created using the “homology model” function and the crystal structure of Nek2 bound to an aminopurine inhibitor (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5M53">5M53</a>) as the template.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></div><div class="NLM_p">For all docking experiments, residual water molecules were deleted, and the receptors were prepared using the “QuickPrep” function without “automated structure preparation”. Using the respective functions of the “MOE Database Viewer”, all ligands were “washed” before partial charges were determined and the structures underwent energy minimization. The binding sites were designated from the cocrystallized ligand (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4B9D">4B9D</a>), which was placed in the binding site by superposition in the case of the apo structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4APC">4APC</a>) and the homology model. Docking poses were placed using the “Triangle Matcher” method and scored by a “London dG” function. Refinements of the poses were performed by using the “Rigid Receptor” method and were finally scored using a “GBVI/WSA dG” function. For the docking of <b>3a</b> into the apo structure, a hinge binding pharmacophore was defined, based on the cocrystallized ligand.</div><div class="NLM_p">The sequence homology analysis between the human and zebrafish Nek1 kinase domains was performed using the “MOE protein similarity monitor”. Identity and similarity values are equal to the number of matches divided by the number of amino acids in the chain corresponding to the cell column. Residues are considered similar if their BLOSUM62 scores are greater than zero.</div><div class="NLM_p last">In accordance with the author guidelines, all virtual screening hits were filtered for PAINS using the PAINS-remover online application.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> All compounds passed the filter.</div></div><div id="sec5_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> <i>In Vitro</i> Kinase Activity Assays</h3><div class="NLM_p">Residual kinase inhibition was determined with the commercial Eurofins Discovery KinaseProfiler service at 1 μM compound concentration and either 10 μM ATP or an ATP concentration within 15 μM of the apparent <i>K</i><sub>m</sub> for ATP (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Supporting Information</a> for <i>K</i><sub>m</sub> values). In the case of Nek1 (KinaseProfiler ITEM 15-020KP10 and ITEM 15-020KP), recombinant human Nek1 (1–505) was incubated with 8 mM MOPS (pH 7.0), 0.2 mM EDTA, 250 μM RLGRDKYKTLRQIRQ, 10 mM magnesium acetate, and [γ-<sup>33</sup>P-ATP]. The reaction was initiated by the addition of the Mg/ATP mix. After incubation for 40 min at room temperature, the reaction was stopped by the addition of phosphoric acid to a concentration of 0.5%. Ten microliters of the reaction was then spotted onto a P30 filtermat and washed four times for 4 min in 0.425% phosphoric acid and once in methanol prior to drying and scintillation counting. Values are the mean from two independent experiments.</div><div class="NLM_p">IC<sub>50</sub> values for Nek1 and Nek2 were determined in a TR-FRET based activity assays (LANCE Ultra TR-FRET, PerkinElmer) at an ATP concentration within 15 μM of the apparent <i>K</i><sub>m</sub> for ATP in the respective assay format (60 μM for both Nek1 and Nek2). The IC<sub>50</sub> curves consist of 10 test compound concentrations tested at half-log dilutions from 30 μM to 1 nM with vehicle and inhibitory control wells. For compounds <b>3j</b> and <b>3k</b>, the maximum concentration was set to 10 μM as a consequence of solubility issues at 30 μM, and the respective IC<sub>50</sub> curves consist of 9 test compound concentrations tested at half-log dilutions from 10 μM to 1 nM. Values are the mean from two independent experiments. Recombinant human Nek1 (1–505; 0.02 nM) or Nek2 (full length; 0.2 nM) (both from Thermo Fisher Scientific) was incubated with 50 mM HEPES (pH 7.5), 1 mM EGTA, 10 mM MgCl<sub>2</sub>, 0.2 mM DTT, 0.01% Tween20, and 50 nM ULight-p70 S6K (Thr389) peptide (sequence LGFTYVAP; PerkinElmer). The reaction was initiated by the addition of the p70 S6K peptide/ATP mix. After incubation for 30 min at 22 °C, the reaction was stopped by the addition of 10 mM EDTA and 0.3 nM europium-antiphospho-p70 S6K (Thr389) antibody in CR97 detection buffer (both PerkinElmer). After incubation for 1 h at 22 °C for signal development, the plates were analyzed in a plate reader (PHERAstar FSX, BMG LABTECH). For compounds <b>3g</b> and <b>3k</b>, relative IC<sub>50</sub> values were provided instead of absolute IC<sub>50</sub>, as the determined IC<sub>50</sub> values exceeded the employed maximum test concentrations (30 μM for <b>3g</b>, 10 μM for <b>3k</b>).</div><div class="NLM_p">For compounds <b>3a</b>, <b>3f</b>, <b>10a</b>–<b>f</b>, IC<sub>50</sub> values were also determined with the Eurofins Discovery IC50Profiler service at either 10 μM (ITEM 15-020KP10) or an ATP concentration within 15 μM of the apparent <i>K</i><sub>m</sub> for ATP (90 μM, ITEM 15-020KP). The IC<sub>50</sub> curves consist of 9 test compound concentrations tested at half-log dilutions from 10 μM to 1 nM with vehicle control wells. Values are the mean from two independent experiments.</div><div class="NLM_p last">IC<sub>50</sub> values and 95% confidence intervals were determined from the raw data provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Supporting Information</a> by nonlinear regression (log_inhibitor vs response_variable slope_four parameters) using GraphPad Prism, version 8.4.3 (471) for OS X, (GraphPad Software, San Diego, California USA, <a href="http://www.graphpad.com" class="extLink">www.graphpad.com</a>). For compounds <b>3j</b> and <b>3k</b>, the maximum concentration data points were dropped, as a consequence of solubility issues at 30 μM.</div></div><div id="sec5_46" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> Caco-2 Cell Permeability Assay</h3><div class="NLM_p last">A–B permeability (Caco-2, pH 6.5/7.4) and percent recovery were determined by HPLC-MS/MS detection of the peak area response in a commercial assay by Eurofins Discovery Services (Item 3318). Propranolol, labetalol, colchicine, and ranitidine were tested concurrently as reference compounds.</div></div><div id="sec5_47" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> Fish Maintenance and Husbandry Protocols</h3><div class="NLM_p last">Fish maintenance and husbandry protocols were documented and approved by the Darmstadt administrative authority. All animals were treated humanely in accordance with the German animal protection standards and the EU Directive 2010/63/EU of the European parliament and of the council. Adult wild-type zebrafish (<i>Danio rerio</i>) were maintained in 60 L fish tanks in a laboratory with no daylight and a constant temperature of 28 °C. Room light was programmed to a 12 h dark/12 h light cycle. Adult zebrafish were bred in mating containers (2 animals/1 L) or mating tanks (6–8 animals/20 L) that were equipped with plastic grass and a removable sieve at the bottom of the tank. Embryos were collected 2–3 h after a light cycle started, rinsed with E3-medium, and isolated in Petri dishes containing E3-medium. E3-medium was prepared according to the Cold Spring Harbor Protocols recipe for E3 medium for zebrafish embryos (doi: 10.1101/pdb.rec066449).</div></div><div id="sec5_48" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Zebrafish Developmental Toxicity Assay</h3><div class="NLM_p">The aqueous solubility of <b>10f</b> was determined in a shake-flask HPLC solubility assay using the previously described HPLC system connected to a diode array detector (DAD, see <a class="ref internalNav" href="#sec5_1" aria-label="General Information">General Information</a>). Absorption was measured at 280 nm, and all measurements were performed in triplicate. A saturated solution of <b>10f</b> in E3-medium was prepared by shaking for 24 h at 25 °C. A 10 mM stock solution of <b>10f</b> in DMSO was diluted to 1, 5, 10, 50, and 100 μM solutions in acetonitrile, and the aqueous solubility of <b>10f</b> was determined by interpolation using Prism 8 by GraphPad Software, LLC.</div><div class="NLM_p">The zebrafish developmental toxicity screen was performed in accordance to guidelines published by M. Haldi et al. in <i>Zebrafish: Methods for Assessing Drug Saftey and Toxicity</i>.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a></div><div class="NLM_p last">At 4 hpf (hours postfertilization), 15 embryos per group (0.05, 0.1, 0.5, 1.5, 2.5, 5, 10, 15, 20, 30 μM) that exhibited intact chorion membranes were selected and reexamined after 2 h. A stock solution (10 mM in DMSO) of the test compound was diluted to 10× final concentrations, and the embryos were sorted in 180 μL of E3 medium. At 6 hpf, 20 μL of the 10× solution was added to give the final concentrations. Survival was monitored every 24 h by observation of the heartbeat. All survival rates were determined at least in triplicate with the controls and the survival rate at 30 μM being determined in 5 replicates.</div></div><div id="sec5_49" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> Stability-Indicating HPLC Assay</h3><div class="NLM_p last">The stability-indicating HPLC assay was performed using the previously described HPLC system connected to a diode array detector (DAD, see <a class="ref internalNav" href="#sec5_1" aria-label="General Information">General Information</a>). A 15 μM solution of <b>10f</b> in E3-medium containing 75 μM 2-phenylthiourea (PTU) and a 15 μM solution of <b>10f</b> containing 75 μM PTU and 2.5 μM PT-Yellow (BDNCA3-D2) were prepared for HPLC injection. Absorptions were measured every 24 h at 280 nm for compound <b>10f</b> and PTU and 548 nm for PT-Yellow over 48 or 120 h, respectively. All measurements were performed in triplicate.</div></div><div id="sec5_50" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> Fluorescence Microscopy Imaging of Zebrafish Pronephroi</h3><div class="NLM_p last">The 96-well plate molding tool was manufactured by selective laser melting on an EOS M290 (EOS GmbH Electro Optical Systems, Krailing, Germany) metal 3D-printing system using EOS StainlessSteel 316L as the material with a layer height of 20 μM. The tool consists of a base plate with 96 perpendicular pins that match the positions of the wells of a microtiter plate. The agarose molds were prepared using 1% NEEO ultraquality agarose (Carl Roth, Karlsruhe, Germany, Art. No. 2267.1) in black-walled 96-well Nunc polymer optical bottom plates (Thermo Fisher Scientific, Waltham, Massachusetts, USA). The specifications of the molding tool and the preparation of the agarose molds in 96-well microtiter plates followed the procedure published by Westhoff et al.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a></div></div><div id="sec5_51" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> Image Acquisition</h3><div class="NLM_p last">Imaging of fluorescently labeled (PT-Yellow) zebrafish embryos was performed on an Olympus IX-81 widefield microscope (Olympus, Shinjuku, Tokyo, Japan), using an Olympus CPlanFL N, 10×, NA 0.3 objective in combination with a 1.5× tube lens magnification, resulting in a total magnification of 15×. Fluorescence was excited using a CoolLED pE300 Ultra (CoolLED Ltd., Andover, NY, USA) and detected on a pco.edge 5.5 (PCO AG, Kelheim, Germany) using appropriate filter sets. The whole setup was controlled using Micromanager 1.4.22.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> to record 33 <i>Z</i>-slices with a <i>Z</i>-distance of 15 μm per embryo after manual orientation in the previously described agarose molds.</div></div><div id="sec5_52" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> Image Processing and Deconvolution</h3><div class="NLM_p last">Image processing was performed using ImageJ, version 2.0.0.-rc-69/1.52p. Deconvolution was performed using the DeconvolutionLab2 plugin and a point spread function that was simulated in the PSF Generator plugin. The image stacks are presented as colored maximum <i>Z</i>-projections using the lookup table <i>blue orange icb</i> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Figure S14</a>). As it was not possible to remove all out of focus blur with the deconvolution procedure, the sharpest 5 consecutive <i>Z</i>-slices of labeled pronephrons were manually selected in each stack (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Figures S15 and S16</a>). Subsequently, the following image analysis steps were applied using ImageJ: (1) subtract background using a 150 pixel rolling ball, (2) sum all 5 images into one, (3) threshold images above 100, (4) convert background to 8-bit mask for the intensity quantification or convert background to NaN for the quantification of the pronephron areas. The pronephron area in the summed stacks was used as the quantity to estimate the size of the pronephroi.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i67"><a href="/doi/suppl/10.1021/acs.jmedchem.0c02118" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86025" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86025" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c02118?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02118</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular modeling, homology model coordinates, <i>in vitro</i> pharmacology, NMR data, lead compound HPLC traces, bioavailability profile of compound <b>10f</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_003.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Nek1 homology model (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_004.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Superlooper2 reconstruction model (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_005.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf">jm0c02118_si_001.pdf (7.03 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_003.csv">jm0c02118_si_003.csv (2.16 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_004.pdb">jm0c02118_si_004.pdb (346.55 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_005.pdb">jm0c02118_si_005.pdb (387.46 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c02118" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65422" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65422" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Georg Baumann</span> - <span class="hlFld-Affiliation affiliation">Clemens
Schöpf−Institute of Organic Chemistry and Biochemistry, Technische Universität Darmstadt, 64287 Darmstadt, Germany</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#97f0f2f8e5f0b9f5f6e2faf6f9f9d7e4e3e2f3b9e3e2baf3f6e5fae4e3f6f3e3b9f3f2"><span class="__cf_email__" data-cfemail="b7d0d2d8c5d099d5d6c2dad6d9d9f7c4c3c2d399c3c29ad3d6c5dac4c3d6d3c399d3d2">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Boris Schmidt</span> - <span class="hlFld-Affiliation affiliation">Clemens
Schöpf−Institute of Organic Chemistry and Biochemistry, Technische Universität Darmstadt, 64287 Darmstadt, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1662-2392" title="Orcid link">http://orcid.org/0000-0003-1662-2392</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#2d5e4e4540444959724f425f445e6d5900424341444348034948"><span class="__cf_email__" data-cfemail="1f6c7c7772767b6b407d706d766c5f6b3270717376717a317b7a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tobias Meckel</span> - <span class="hlFld-Affiliation affiliation">Clemens
Schöpf−Institute of Organic Chemistry and Biochemistry, Technische Universität Darmstadt, 64287 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kevin Böhm</span> - <span class="hlFld-Affiliation affiliation">Clemens
Schöpf−Institute of Organic Chemistry and Biochemistry, Technische Universität Darmstadt, 64287 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yung-Hsin Shih</span> - <span class="hlFld-Affiliation affiliation">Clemens
Schöpf−Institute of Organic Chemistry and Biochemistry, Technische Universität Darmstadt, 64287 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mirco Dickhaut</span> - <span class="hlFld-Affiliation affiliation">Clemens
Schöpf−Institute of Organic Chemistry and Biochemistry, Technische Universität Darmstadt, 64287 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Torben Reichardt</span> - <span class="hlFld-Affiliation affiliation">Clemens
Schöpf−Institute of Organic Chemistry and Biochemistry, Technische Universität Darmstadt, 64287 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Johannes Pilakowski</span> - <span class="hlFld-Affiliation affiliation">Clemens
Schöpf−Institute of Organic Chemistry and Biochemistry, Technische Universität Darmstadt, 64287 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ulrich Pehl</span> - <span class="hlFld-Affiliation affiliation">Merck
Healthcare KGaA, Biopharma R&D, Discovery
and Development Technologies, 64293 Darmstadt, Germany</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>G.B., B.S., and T.M. conceived and designed the experiments. G.B., T.M., and B.S. wrote the manuscript. G.B., K.B., Y.S., M.D., U.D., and T.R. performed the experiments. G.B., B.S., U.P., and T.M. analyzed the data. G.B., B.S., T.M., and J.P. gave scientific advice. All authors contributed to the article and approved the submitted version.</p></li><li><span class="author-information-subsection-header">Funding</span><p>The authors declare that this study received funding from the German Research Foundation (DFG, GRK1657, Project number: 161030019). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i69">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20723" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20723" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Holger Hermann Merschroth for the design and production of the 96-well plate molding tool and Nadine Herget and Friedrich Baumgärtner for performing the TR-FRET <i>in vitro</i> kinase activity assays. We furthermore thank Prof. Markus Löbrich, Julian Spies, and Michael Ensminger for discussions on the biological role of Nek1.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ALS</td><td class="NLM_def"><p class="first last">amyotrophic lateral sclerosis</p></td></tr><tr><td class="NLM_term">CDK</td><td class="NLM_def"><p class="first last">cyclin-dependent kinase</p></td></tr><tr><td class="NLM_term">Chk1</td><td class="NLM_def"><p class="first last">checkpoint kinase 1</p></td></tr><tr><td class="NLM_term">Chk2</td><td class="NLM_def"><p class="first last">checkpoint kinase 2</p></td></tr><tr><td class="NLM_term">cDNA</td><td class="NLM_def"><p class="first last">complementary DNA</p></td></tr><tr><td class="NLM_term">DDR</td><td class="NLM_def"><p class="first last">DNA damage response</p></td></tr><tr><td class="NLM_term">HR</td><td class="NLM_def"><p class="first last">homologous recombination</p></td></tr><tr><td class="NLM_term">IKK</td><td class="NLM_def"><p class="first last">IκB kinase</p></td></tr><tr><td class="NLM_term">JNK</td><td class="NLM_def"><p class="first last">c-Jun N-terminal kinase</p></td></tr><tr><td class="NLM_term">DSB</td><td class="NLM_def"><p class="first last">double strand break</p></td></tr><tr><td class="NLM_term">HR</td><td class="NLM_def"><p class="first last">homologous recombination</p></td></tr><tr><td class="NLM_term">Nek1</td><td class="NLM_def"><p class="first last">NIMA-related kinase 1</p></td></tr><tr><td class="NLM_term">NF-κB</td><td class="NLM_def"><p class="first last">nuclear factor kappa-light-chain-enhancer of activated B-cells</p></td></tr><tr><td class="NLM_term">MOE</td><td class="NLM_def"><p class="first last">Molecular Operating Environment</p></td></tr><tr><td class="NLM_term">NIMA</td><td class="NLM_def"><p class="first last">never-in-mitosis gene A</p></td></tr><tr><td class="NLM_term">PKD</td><td class="NLM_def"><p class="first last">polycystic kidney disease</p></td></tr><tr><td class="NLM_term">PSF</td><td class="NLM_def"><p class="first last">point square function</p></td></tr><tr><td class="NLM_term">RCC</td><td class="NLM_def"><p class="first last">renal cell carcinoma</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin layer chromatography</p></td></tr><tr><td class="NLM_term">TPSA</td><td class="NLM_def"><p class="first last">topological polar surface area</p></td></tr><tr><td class="NLM_term">Tlk1</td><td class="NLM_def"><p class="first last">tousled-like kinase 1</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i71">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34382" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34382" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 66 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
de Kooij, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creixell, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vlimmeren, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joughin, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haider, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linding, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stambolic, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaffe, M. B.</span></span> <span> </span><span class="NLM_article-title">Comprehensive substrate specificity profiling of the human Nek kinome reveals unexpected signaling outputs</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e44635</span> <span class="refDoi"> DOI: 10.7554/eLife.44635</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.7554%2FeLife.44635" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=31124786" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Crtr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&author=B.+van%0Ade+Kooijauthor=P.+Creixellauthor=A.+van+Vlimmerenauthor=B.+A.+Joughinauthor=C.+J.+Millerauthor=N.+Haiderauthor=C.+D.+Simpsonauthor=R.+Lindingauthor=V.+Stambolicauthor=B.+E.+Turkauthor=M.+B.+Yaffe&title=Comprehensive+substrate+specificity+profiling+of+the+human+Nek+kinome+reveals+unexpected+signaling+outputs&doi=10.7554%2FeLife.44635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive substrate specificity profiling of the human Nek kinome reveals unexpected signaling outputs</span></div><div class="casAuthors">van de Kooij, Bert; Creixell, Pau; Van Vlimmeren, Anne; Joughin, Brian A.; Miller, Chad J.; Haider, Nasir; Simpson, Craig D.; Linding, Rune; Stambolic, Vuk; Turk, Benjamin E.; Yaffe, Michael B.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e44635</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Human NimA-related kinases (Neks) have multiple mitotic and non-mitotic functions, but few substrates are known.  We systematically detd. the phosphorylation-site motifs for the entire Nek kinase family, except for Nek11.  While all Nek kinases strongly select for hydrophobic residues in the 3 position, the family separates into four distinct groups based on specificity for a serine vs. threonine phospho-acceptor, and preference for basic or acidic residues in other positions.  Unlike Nek1-Nek9, Nek10 is a dual-specificity kinase that efficiently phosphorylates itself and peptide substrates on serine and tyrosine, and its activity is enhanced by tyrosine auto-phosphorylation.  Nek10 dual-specificity depends on residues in the HRD+2 and APE4 positions that are uncommon in either serine/threonine or tyrosine kinases.  Finally, we show that the phosphorylation-site motifs for the mitotic kinases Nek6, Nek7 and Nek9 are essentially identical to that of their upstream activator Plk1, suggesting that Nek6/7/9 function as phosphomotif amplifiers of Plk1 signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqk3K-2RlBb-7Vg90H21EOLACvtfcHk0lghGXLO-d7ktQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Crtr%252FE&md5=3ca0d00e517849a33a3565594a72f7fa</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.7554%2FeLife.44635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.44635%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bde%2BKooij%26aufirst%3DB.%26aulast%3DCreixell%26aufirst%3DP.%26aulast%3Dvan%2BVlimmeren%26aufirst%3DA.%26aulast%3DJoughin%26aufirst%3DB.%2BA.%26aulast%3DMiller%26aufirst%3DC.%2BJ.%26aulast%3DHaider%26aufirst%3DN.%26aulast%3DSimpson%26aufirst%3DC.%2BD.%26aulast%3DLinding%26aufirst%3DR.%26aulast%3DStambolic%26aufirst%3DV.%26aulast%3DTurk%26aufirst%3DB.%2BE.%26aulast%3DYaffe%26aufirst%3DM.%2BB.%26atitle%3DComprehensive%2520substrate%2520specificity%2520profiling%2520of%2520the%2520human%2520Nek%2520kinome%2520reveals%2520unexpected%2520signaling%2520outputs%26jtitle%3DeLife%26date%3D2019%26volume%3D8%26doi%3D10.7554%2FeLife.44635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Letwin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizzen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motro, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-David, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernstein, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span> <span> </span><span class="NLM_article-title">A mammalian dual specificity protein kinase, Nek1, is related to the NIMA cell cycle regulator and highly expressed in meiotic germ cells</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3521</span>– <span class="NLM_lpage">3531</span>, <span class="refDoi"> DOI: 10.1002/j.1460-2075.1992.tb05435.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1002%2Fj.1460-2075.1992.tb05435.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1382974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADyaK3sXkvV2rtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1992&pages=3521-3531&author=K.+Letwinauthor=L.+Mizzenauthor=B.+Motroauthor=Y.+Ben-Davidauthor=A.+Bernsteinauthor=T.+Pawson&title=A+mammalian+dual+specificity+protein+kinase%2C+Nek1%2C+is+related+to+the+NIMA+cell+cycle+regulator+and+highly+expressed+in+meiotic+germ+cells&doi=10.1002%2Fj.1460-2075.1992.tb05435.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">A mammalian dual specificity protein kinase, Nek1, is related to the NIMA cell cycle regulator and highly expressed in meiotic germ cells</span></div><div class="casAuthors">Letwin, Kenneth; Mizzen, Lee; Motro, Benny; Ben-David, Yaacov; Bernstein, Alan; Pawson, Tony</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3521-31</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    </div><div class="casAbstract">Screening of mouse cDNA expression libraries with antibodies to phosphotyrosine resulted in repeated isolation of cDNAs that encode a novel mammalian protein kinase of 774 amino acids termed Nek1.  Nek1 contains an N-terminal protein kinase domain which is most similar (42% identity) to the catalytic domain of NIMA, a protein kinase which controls initiation of mitosis in Aspergillus nidulans.  In addn., both Nek1 and NIMA have a long, basic C-terminal extension, and are therefore similar in overall structure.  Despite its identification with anti-phosphotyrosine antibodies, Nek1 contains sequence motifs characteristic of protein serine/threonine kinases.  The Nek1 kinase domain, when expressed in bacteria, phosphorylated exogenous substrates primarily on serine/threonine, but also on tyrosine, indicating that Nek1 is a dual-specificity kinase with the capacity to phosphorylate all 3 hydroxyamino acids.  Like NIMA, Nek1 preferentially phosphorylated β-casein in vitro.  In situ RNA anal. of nek1 expression in mouse gonads revealed a high level of expression in both male and female germ cells, with a distribution consistent with a role in meiosis.  Apparently, Nek1 is a mammalian relative of the fungal NIMA cell cycle regulator.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBIGgayMb1BrVg90H21EOLACvtfcHk0lghGXLO-d7ktQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXkvV2rtLs%253D&md5=119e2b72c4532c74e70eb7d92374f980</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2Fj.1460-2075.1992.tb05435.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fj.1460-2075.1992.tb05435.x%26sid%3Dliteratum%253Aachs%26aulast%3DLetwin%26aufirst%3DK.%26aulast%3DMizzen%26aufirst%3DL.%26aulast%3DMotro%26aufirst%3DB.%26aulast%3DBen-David%26aufirst%3DY.%26aulast%3DBernstein%26aufirst%3DA.%26aulast%3DPawson%26aufirst%3DT.%26atitle%3DA%2520mammalian%2520dual%2520specificity%2520protein%2520kinase%252C%2520Nek1%252C%2520is%2520related%2520to%2520the%2520NIMA%2520cell%2520cycle%2520regulator%2520and%2520highly%2520expressed%2520in%2520meiotic%2520germ%2520cells%26jtitle%3DEMBO%2520J.%26date%3D1992%26volume%3D11%26spage%3D3521%26epage%3D3531%26doi%3D10.1002%2Fj.1460-2075.1992.tb05435.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meirelles, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basei, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papa, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melo Hanchuk, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardoso, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobarg, J.</span></span> <span> </span><span class="NLM_article-title">Stop Ne(c)king around“: How interactomics contributes to functionally characterize Nek family kinases</span>. <i>World journal of biological chemistry</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">160</span>, <span class="refDoi"> DOI: 10.4331/wjbc.v5.i2.141</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.4331%2Fwjbc.v5.i2.141" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=24921005" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A280%3ADC%252BC2cflvF2jsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=141-160&author=G.+V.+Meirellesauthor=A.+M.+Perezauthor=E.+E.+Souzaauthor=F.+L.+Baseiauthor=P.+F.+Papaauthor=T.+D.+Melo+Hanchukauthor=V.+B.+Cardosoauthor=J.+Kobarg&title=Stop+Ne%28c%29king+around%E2%80%9C%3A+How+interactomics+contributes+to+functionally+characterize+Nek+family+kinases&doi=10.4331%2Fwjbc.v5.i2.141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">"Stop Ne(c)king around": How interactomics contributes to functionally characterize Nek family kinases</span></div><div class="casAuthors">Meirelles Gabriela Vaz; Perez Arina Marina; de Souza Edmarcia Elisa; Basei Fernanda Luisa; Papa Priscila Ferreira; Melo Hanchuk Talita Diniz; Cardoso Vanessa Bomfim; Kobarg Jorg</div><div class="citationInfo"><span class="NLM_cas:title">World journal of biological chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">141-60</span>
        ISSN:<span class="NLM_cas:issn">1949-8454</span>.
    </div><div class="casAbstract">Aside from Polo and Aurora, a third but less studied kinase family involved in mitosis regulation is the never in mitosis-gene A (NIMA)-related kinases (Neks).  The founding member of this family is the sole member NIMA of Aspergillus nidulans, which is crucial for the initiation of mitosis in that organism.  All 11 human Neks have been functionally assigned to one of the three core functions established for this family in mammals: (1) centrioles/mitosis; (2) primary ciliary function/ciliopathies; and (3) DNA damage response (DDR).  Recent findings, especially on Nek 1 and 8, showed however, that several Neks participate in parallel in at least two of these contexts: primary ciliary function and DDR.  In the core section of this in-depth review, we report the current detailed functional knowledge on each of the 11 Neks.  In the discussion, we return to the cross-connections among Neks and point out how our and other groups' functional and interactomics studies revealed that most Neks interact with protein partners associated with two if not all three of the functional contexts.  We then raise the hypothesis that Neks may be the connecting regulatory elements that allow the cell to fine tune and synchronize the cellular events associated with these three core functions.  The new and exciting findings on the Nek family open new perspectives and should allow the Neks to finally claim the attention they deserve in the field of kinases and cell cycle biology.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTQeWflPetSDcRAV4D_Ins_fW6udTcc2eZFZKXbhZ1OULntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cflvF2jsg%253D%253D&md5=26d9c7e0cc2d51f47a686efe07058290</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.4331%2Fwjbc.v5.i2.141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4331%252Fwjbc.v5.i2.141%26sid%3Dliteratum%253Aachs%26aulast%3DMeirelles%26aufirst%3DG.%2BV.%26aulast%3DPerez%26aufirst%3DA.%2BM.%26aulast%3DSouza%26aufirst%3DE.%2BE.%26aulast%3DBasei%26aufirst%3DF.%2BL.%26aulast%3DPapa%26aufirst%3DP.%2BF.%26aulast%3DMelo%2BHanchuk%26aufirst%3DT.%2BD.%26aulast%3DCardoso%26aufirst%3DV.%2BB.%26aulast%3DKobarg%26aufirst%3DJ.%26atitle%3DStop%2520Ne%2528c%2529king%2520around%25E2%2580%259C%253A%2520How%2520interactomics%2520contributes%2520to%2520functionally%2520characterize%2520Nek%2520family%2520kinases%26jtitle%3DWorld%2520journal%2520of%2520biological%2520chemistry%26date%3D2014%26volume%3D5%26spage%3D141%26epage%3D160%26doi%3D10.4331%2Fwjbc.v5.i2.141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peres
de Oliveira, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazuo Issayama, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betim Pavan, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riback Silva, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diniz Melo-Hanchuk, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreira Simabuco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobarg, J.</span></span> <span> </span><span class="NLM_article-title">Checking NEKs: Overcoming a Bottleneck in Human Diseases</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1778</span>, <span class="refDoi"> DOI: 10.3390/molecules25081778</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.3390%2Fmolecules25081778" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=1778&author=A.+Peres%0Ade+Oliveiraauthor=L.+Kazuo+Issayamaauthor=I.+C.+Betim+Pavanauthor=F.+Riback+Silvaauthor=T.+Diniz+Melo-Hanchukauthor=F.+Moreira+Simabucoauthor=J.+Kobarg&title=Checking+NEKs%3A+Overcoming+a+Bottleneck+in+Human+Diseases&doi=10.3390%2Fmolecules25081778"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3390%2Fmolecules25081778&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules25081778%26sid%3Dliteratum%253Aachs%26aulast%3DPeres%2Bde%2BOliveira%26aufirst%3DA.%26aulast%3DKazuo%2BIssayama%26aufirst%3DL.%26aulast%3DBetim%2BPavan%26aufirst%3DI.%2BC.%26aulast%3DRiback%2BSilva%26aufirst%3DF.%26aulast%3DDiniz%2BMelo-Hanchuk%26aufirst%3DT.%26aulast%3DMoreira%2BSimabuco%26aufirst%3DF.%26aulast%3DKobarg%26aufirst%3DJ.%26atitle%3DChecking%2520NEKs%253A%2520Overcoming%2520a%2520Bottleneck%2520in%2520Human%2520Diseases%26jtitle%3DMolecules%26date%3D2020%26volume%3D25%26spage%3D1778%26doi%3D10.3390%2Fmolecules25081778" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janaswami, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birkenmeier, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowe, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davisson, M. T.</span></span> <span> </span><span class="NLM_article-title">Identification and genetic mapping of a new polycystic kidney disease on mouse chromosome 8</span>. <i>Genomics</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">107</span>, <span class="refDoi"> DOI: 10.1006/geno.1996.4567</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1006%2Fgeno.1996.4567" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=9070925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADyaK2sXhtlSgt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1997&pages=101-107&author=P.+M.+Janaswamiauthor=E.+H.+Birkenmeierauthor=S.+A.+Cookauthor=L.+B.+Roweauthor=R.+T.+Bronsonauthor=M.+T.+Davisson&title=Identification+and+genetic+mapping+of+a+new+polycystic+kidney+disease+on+mouse+chromosome+8&doi=10.1006%2Fgeno.1996.4567"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and genetic mapping of a new polycystic kidney diseases on mouse chromosome 8</span></div><div class="casAuthors">Janaswami, Poornima M.; Birkenmeier, Edward H.; Cook, Susan A.; Rowe, Lucy B.; Bronson, Roderick T.; Davisson, Muriel T.</div><div class="citationInfo"><span class="NLM_cas:title">Genomics</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">101-107</span>CODEN:
                <span class="NLM_cas:coden">GNMCEP</span>;
        ISSN:<span class="NLM_cas:issn">0888-7543</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">We report here a new mouse mutation, kat, that causes pleiotropic effects including facial dysmorphism, dwarfing, male sterility, anemia, and progressive polycystic kidney disease.  Kat (kidney anemia and testis) and a second allele, kat2J, that occurred on C57BL/6J were mapped to mouse chromosome (Chr) 8 using intra- and intersubspecific intercrosses.  A high-resoln. map for kat2J on Chr 8 was constructed using the F2 progeny from a cross between C57BL/6J-kat2J/+ and an inbred strain of Mus musculus castaneus (CAST/Ei).  The kat2J mutation was localized between D8Mit129 and D8Mit128 with the gene order centromere-D8Mit100-(1.2±0.26 cM)-D8Mit231-(0.17±0.09 cM)-D8Mit129-(0.28±0.12 cM)-D8Mit128-(0.98±0.23 cM)-D8Mit25/D8Mit8.  This segment is homologous to human Chr 19 p.  The two mutations at this locus that have occurred at The Jackson Lab. will be invaluable for positional cloning and subsequent functional anal. of the mutated gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV1SQW7cqpALVg90H21EOLACvtfcHk0ljZm_k6FdqsaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhtlSgt70%253D&md5=9dd2b1d346aa007cec311cd201cbe751</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1006%2Fgeno.1996.4567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fgeno.1996.4567%26sid%3Dliteratum%253Aachs%26aulast%3DJanaswami%26aufirst%3DP.%2BM.%26aulast%3DBirkenmeier%26aufirst%3DE.%2BH.%26aulast%3DCook%26aufirst%3DS.%2BA.%26aulast%3DRowe%26aufirst%3DL.%2BB.%26aulast%3DBronson%26aufirst%3DR.%2BT.%26aulast%3DDavisson%26aufirst%3DM.%2BT.%26atitle%3DIdentification%2520and%2520genetic%2520mapping%2520of%2520a%2520new%2520polycystic%2520kidney%2520disease%2520on%2520mouse%2520chromosome%25208%26jtitle%3DGenomics%26date%3D1997%26volume%3D40%26spage%3D101%26epage%3D107%26doi%3D10.1006%2Fgeno.1996.4567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Upadhya, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birkenmeier, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birkenmeier, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, J. E.</span></span> <span> </span><span class="NLM_article-title">Mutations in a NIMA-related kinase gene cause pleiotropic effects including a progressive polycystic kidney disease in mice</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">217</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1073/pnas.97.1.217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1073%2Fpnas.97.1.217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10618398" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjvVCqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2000&pages=217-221&author=P.+Upadhyaauthor=E.+H.+Birkenmeierauthor=C.+S.+Birkenmeierauthor=J.+E.+Barker&title=Mutations+in+a+NIMA-related+kinase+gene+cause+pleiotropic+effects+including+a+progressive+polycystic+kidney+disease+in+mice&doi=10.1073%2Fpnas.97.1.217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in a NIMA-related kinase gene, Nek1, cause pleiotropic effects including a progressive polycystic kidney disease in mice</span></div><div class="casAuthors">Upadhya, Poornima; Birkenmeier, Edward H.; Birkenmeier, Connie S.; Barker, Jane E.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">217-221</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We previously have described a mouse model for polycystic kidney disease (PKD) caused by either of two mutations, kat or kat2J, that map to the same locus on chromosome 8.  The homozygous mutant animals have a latent onset, slowly progressing form of PKD with renal pathol. similar to the human autosomal-dominant PKD.  In addn., the mutant animals show pleiotropic effects that include facial dysmorphism, dwarfing, male sterility, anemia, and cystic choroid plexus.  We previously fine-mapped the kat2J mutation to a genetic distance of 0.28 centimorgan between D8Mit128 and D8Mit129.  To identify the underlying mol. defect in this locus, we constructed an integrated genetic and phys. map of the crit. region surrounding the kat2J mutation.  Cloning and expression anal. of the transcribed sequences from this region identified Nek1, a NIMA (never in mitosis A)-related kinase as a candidate gene.  Further anal. of the Nek1 gene from both kat/kat and kat2J/kat2J mutant animals identified a partial internal deletion and a single-base insertion as the mol. basis for these mutations.  The complex pleiotropic phenotypes seen in the homozygous mutant animals suggest that the NEK1 protein participates in different signaling pathways to regulate diverse cellular processes.  Our findings identify a previously unsuspected role for Nek1 in the kidney and open a new avenue for studying cystogenesis and identifying possible modes of therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqpjqA-TEUFbVg90H21EOLACvtfcHk0ljZm_k6FdqsaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjvVCqtQ%253D%253D&md5=d72b1ade4c33aa220b25a6c7c329fef8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1073%2Fpnas.97.1.217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.97.1.217%26sid%3Dliteratum%253Aachs%26aulast%3DUpadhya%26aufirst%3DP.%26aulast%3DBirkenmeier%26aufirst%3DE.%2BH.%26aulast%3DBirkenmeier%26aufirst%3DC.%2BS.%26aulast%3DBarker%26aufirst%3DJ.%2BE.%26atitle%3DMutations%2520in%2520a%2520NIMA-related%2520kinase%2520gene%2520cause%2520pleiotropic%2520effects%2520including%2520a%2520progressive%2520polycystic%2520kidney%2520disease%2520in%2520mice%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2000%26volume%3D97%26spage%3D217%26epage%3D221%26doi%3D10.1073%2Fpnas.97.1.217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chebib, F. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torres, V. E.</span></span> <span> </span><span class="NLM_article-title">Recent advances in the management of autosomal dominant polycystic kidney disease</span>. <i>Clin. J. Am. Soc. Nephrol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1765</span>– <span class="NLM_lpage">1776</span>, <span class="refDoi"> DOI: 10.2215/CJN.03960318</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.2215%2FCJN.03960318" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=30049849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktVOjsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=1765-1776&author=F.+T.+Chebibauthor=V.+E.+Torres&title=Recent+advances+in+the+management+of+autosomal+dominant+polycystic+kidney+disease&doi=10.2215%2FCJN.03960318"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in the management of autosomal dominant polycystic kidney disease</span></div><div class="casAuthors">Chebib, Fouad T.; Torres, Vicente E.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Journal of the American Society of Nephrology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1765-1776</span>CODEN:
                <span class="NLM_cas:coden">CJASC7</span>;
        ISSN:<span class="NLM_cas:issn">1555-9041</span>.
    
            (<span class="NLM_cas:orgname">American Society of Nephrology</span>)
        </div><div class="casAbstract">A review.  Autosomal dominant polycystic kidney disease (ADPKD), the most common monogenic cause of ESKD, is characterized by relentless development of kidney cysts, hypertension, and destruction of the kidney parenchyma.  Over the past few years, major advancements in diagnosing, prognosticating, and understanding the pathogenesis and natural course of the disease have been made.  Currently, no kidney disease is more suitable for nephron-protective strategies.  Early nephrol. referral and implementation of these strategies may have a substantial effect.  Total kidney vol. is a good prognostication marker and allows stratification of patients into slow or rapid progressing disease, with implications for their management.  Measurement of total kidney vol., disease stratification, and prognostication are possible using readily available tools.  Although some patients require only monitoring and basic optimized kidney protective measures, such as rigorous BP control and various lifestyle and dietary changes, others will benefit from disease-modifying treatments.  Vasopressin V2 receptor antagonists, a likely disease-modifying treatment, has been approved in several countries and recently by the US Food and Drug Administration; other therapies, such as somatostatin analogs and other novel agents, are currently in clin. trials.  The purpose of this article is to present our views on the optimal management to delay kidney disease progression in ADPKD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZD-kWSfMOibVg90H21EOLACvtfcHk0lhdh8BFENNoDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktVOjsr8%253D&md5=bbe016f282f906bc86af5061823cc565</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2215%2FCJN.03960318&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2215%252FCJN.03960318%26sid%3Dliteratum%253Aachs%26aulast%3DChebib%26aufirst%3DF.%2BT.%26aulast%3DTorres%26aufirst%3DV.%2BE.%26atitle%3DRecent%2520advances%2520in%2520the%2520management%2520of%2520autosomal%2520dominant%2520polycystic%2520kidney%2520disease%26jtitle%3DClin.%2520J.%2520Am.%2520Soc.%2520Nephrol.%26date%3D2018%26volume%3D13%26spage%3D1765%26epage%3D1776%26doi%3D10.2215%2FCJN.03960318" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Surpili, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delben, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobarg, J.</span></span> <span> </span><span class="NLM_article-title">Identification of proteins that interact with the central coiled-coil region of the human protein kinase NEK1</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">15369</span>– <span class="NLM_lpage">15376</span>, <span class="refDoi"> DOI: 10.1021/bi034575v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi034575v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD3sXptlOks7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=15369-15376&author=M.+J.+Surpiliauthor=T.+M.+Delbenauthor=J.+Kobarg&title=Identification+of+proteins+that+interact+with+the+central+coiled-coil+region+of+the+human+protein+kinase+NEK1&doi=10.1021%2Fbi034575v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Proteins That Interact with the Central Coiled-Coil Region of the Human Protein Kinase NEK1</span></div><div class="casAuthors">Surpili, Marcelo J.; Delben, Tatiana M.; Kobarg, Joerg</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">15369-15376</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">NEK protein kinases are evolutionarily conserved kinases structurally related to the Aspergillus nidulans mitotic regulator NIMA.  At least nine members of the NEK family in vertebrates have been described to date, but for most of them the interacting protein partners are unknown.  The pleiotropic deleterious effects and the formation of kidney cysts caused by NEK1 mutation in mice emphasize its involvement in the regulation of diverse cellular processes and in the etiol. of polycystic kidney disease (PKD), resp.  Here the authors report the identification of proteins that interacted with the human NEK1 protein kinase in a yeast two-hybrid screen of a human fetal brain cDNA library, using the catalytic and regulatory domains of NEK1 sep. as baits.  These proteins are known to take part either in the development of PKD, in the double-strand DNA break repair at the G2/M transition phase of the cell cycle, or in neural cell development.  The proteins involved in PKD include the motor protein KIF3A and the proteins tuberin and α-catulin.  Mapping studies of the human NEK1 regulatory domain (NRD) indicated a strong interaction of most of the proteins retrieved from the library with putative coiled coils located in the central region of NRD.  The results give further support to the previous observation that NEK1 is of functional importance for the etiol. of PKD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfVzTlZD29prVg90H21EOLACvtfcHk0lhdh8BFENNoDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXptlOks7c%253D&md5=2b87703316b69af79310f065d1015caf</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fbi034575v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi034575v%26sid%3Dliteratum%253Aachs%26aulast%3DSurpili%26aufirst%3DM.%2BJ.%26aulast%3DDelben%26aufirst%3DT.%2BM.%26aulast%3DKobarg%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520proteins%2520that%2520interact%2520with%2520the%2520central%2520coiled-coil%2520region%2520of%2520the%2520human%2520protein%2520kinase%2520NEK1%26jtitle%3DBiochemistry%26date%3D2003%26volume%3D42%26spage%3D15369%26epage%3D15376%26doi%3D10.1021%2Fbi034575v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, H.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litchfield, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pena, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D. J.</span></span> <span> </span><span class="NLM_article-title">Expression of Nek1 during kidney development and cyst formation in multiple nephron segments in the Nek1-deficient kat2J mouse model of polycystic kidney disease</span>. <i>J. Biomed. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">65</span>, <span class="refDoi"> DOI: 10.1186/s12929-014-0063-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1186%2Fs12929-014-0063-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=25030234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvV2isbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=63-65&author=Y.+Chenauthor=H.-C.+Chiangauthor=P.+Litchfieldauthor=M.+Penaauthor=C.+Juangauthor=D.+J.+Riley&title=Expression+of+Nek1+during+kidney+development+and+cyst+formation+in+multiple+nephron+segments+in+the+Nek1-deficient+kat2J+mouse+model+of+polycystic+kidney+disease&doi=10.1186%2Fs12929-014-0063-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of Nek1 during kidney development and cyst formation in multiple nephron segments in the Nek1-deficient kat2J mouse model of polycystic kidney disease</span></div><div class="casAuthors">Chen, Yumay; Chiang, Huai-Chin; Litchfield, Patricia; Pena, Michelle; Juang, Charity; Riley, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomedical Science (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">63/1-63/14, 14 pp.</span>CODEN:
                <span class="NLM_cas:coden">JBCIEA</span>;
        ISSN:<span class="NLM_cas:issn">1423-0127</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Neks, mammalian orthologs of the fungal protein kinase never-in-mitosis A, have been implicated in the pathogenesis of polycystic kidney disease.  Among them, Nek1 is the primary protein inactivated in kat2J mouse models of PKD.  Result: We report the expression pattern of Nek1 and characterize the renal cysts that develop in kat2J mice.  Nek1 is detectable in all murine tissues but its expression in wild type and kat2J heterozygous kidneys decrease as the kidneys mature, esp. in tubular epithelial cells.  In the embryonic kidney, Nek1 expression is most prominent in cells that will become podocytes and proximal tubules.  Kidney development in kat2J homozygous mice is aberrant early, before the appearance of gross cysts: developing cortical zones are thin, populated by immature glomeruli, and characterized by excessive apoptosis of several cell types.  Cysts in kat2J homozygous mice form postnatally in Bowman's space as well as different tubular subtypes.  Late in life, kat2J heterozygous mice form renal cysts and the cells lining these cysts lack staining for Nek1.  The primary cilia of cells lining cysts in kat2J homozygous mice are morphol. diverse: in some cells they are unusually long and in others there are multiple cilia of varying lengths.  Conclusion: Our studies indicate that Nek1 deficiency leads to disordered kidney maturation, and cysts throughout the nephron.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs27DDJl2RnLVg90H21EOLACvtfcHk0ljVFc5iTVG6Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvV2isbrN&md5=6cf62ec0884bc265b3d5648452d74a81</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1186%2Fs12929-014-0063-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12929-014-0063-5%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DChiang%26aufirst%3DH.-C.%26aulast%3DLitchfield%26aufirst%3DP.%26aulast%3DPena%26aufirst%3DM.%26aulast%3DJuang%26aufirst%3DC.%26aulast%3DRiley%26aufirst%3DD.%2BJ.%26atitle%3DExpression%2520of%2520Nek1%2520during%2520kidney%2520development%2520and%2520cyst%2520formation%2520in%2520multiple%2520nephron%2520segments%2520in%2520the%2520Nek1-deficient%2520kat2J%2520mouse%2520model%2520of%2520polycystic%2520kidney%2520disease%26jtitle%3DJ.%2520Biomed.%2520Sci.%26date%3D2014%26volume%3D21%26spage%3D63%26epage%3D65%26doi%3D10.1186%2Fs12929-014-0063-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giessl, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimmler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shalev, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von der Haar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zenker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahnleiter, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stöss, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abou Jamra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekici, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schröder-Kreß, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aigner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirchner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandstätter, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauch, A.</span></span> <span> </span><span class="NLM_article-title">NEK1 mutations cause short-rib polydactyly syndrome type Majewski</span>. <i>Am. J. Hum. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">106</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1016/j.ajhg.2010.12.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.ajhg.2010.12.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=21211617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A280%3ADC%252BC3M7jtVemtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2011&pages=106-114&author=C.+Thielauthor=K.+Kesslerauthor=A.+Giesslauthor=A.+Dimmlerauthor=S.+A.+Shalevauthor=S.+von+der+Haarauthor=M.+Zenkerauthor=D.+Zahnleiterauthor=H.+St%C3%B6ssauthor=E.+Beinderauthor=R.+Abou+Jamraauthor=A.+B.+Ekiciauthor=N.+Schr%C3%B6der-Kre%C3%9Fauthor=T.+Aignerauthor=T.+Kirchnerauthor=A.+Reisauthor=J.+H.+Brandst%C3%A4tterauthor=A.+Rauch&title=NEK1+mutations+cause+short-rib+polydactyly+syndrome+type+Majewski&doi=10.1016%2Fj.ajhg.2010.12.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">NEK1 mutations cause short-rib polydactyly syndrome type majewski</span></div><div class="casAuthors">Thiel Christian; Kessler Kristin; Giessl Andreas; Dimmler Arno; Shalev Stavit A; von der Haar Sigrun; Zenker Martin; Zahnleiter Diana; Stoss Hartmut; Beinder Ernst; Abou Jamra Rami; Ekici Arif B; Schroder-Kress Nadja; Aigner Thomas; Kirchner Thomas; Reis Andre; Brandstatter Johann H; Rauch Anita</div><div class="citationInfo"><span class="NLM_cas:title">American journal of human genetics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">106-14</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Defects of ciliogenesis have been implicated in a wide range of human phenotypes and play a crucial role in signal transduction and cell-cycle coordination.  We used homozygosity mapping in two families with autosomal-recessive short-rib polydactyly syndrome Majewski type to identify mutations in NEK1 as an underlying cause of this lethal osteochondrodysplasia.  NEK1 encodes a serine/threonine kinase with proposed function in DNA double-strand repair, neuronal development, and coordination of cell-cycle-associated ciliogenesis.  We found that absence of functional full-length NEK1 severely reduces cilia number and alters ciliar morphology in vivo.  We further substantiate a proposed digenic diallelic inheritance of ciliopathies by the identification of heterozygous mutations in NEK1 and DYNC2H1 in an additional family.  Notably, these findings not only increase the broad spectrum of ciliar disorders, but suggest a correlation between the degree of defective microtubule or centriole elongation and organization and the severity of the resulting phenotype.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcROyYQ7SrTmgx0LsK7uFuY7fW6udTcc2eYQzvVDBlt4gbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M7jtVemtQ%253D%253D&md5=cc069666f1d18413269e186ed574442a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.ajhg.2010.12.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ajhg.2010.12.004%26sid%3Dliteratum%253Aachs%26aulast%3DThiel%26aufirst%3DC.%26aulast%3DKessler%26aufirst%3DK.%26aulast%3DGiessl%26aufirst%3DA.%26aulast%3DDimmler%26aufirst%3DA.%26aulast%3DShalev%26aufirst%3DS.%2BA.%26aulast%3Dvon%2Bder%2BHaar%26aufirst%3DS.%26aulast%3DZenker%26aufirst%3DM.%26aulast%3DZahnleiter%26aufirst%3DD.%26aulast%3DSt%25C3%25B6ss%26aufirst%3DH.%26aulast%3DBeinder%26aufirst%3DE.%26aulast%3DAbou%2BJamra%26aufirst%3DR.%26aulast%3DEkici%26aufirst%3DA.%2BB.%26aulast%3DSchr%25C3%25B6der-Kre%25C3%259F%26aufirst%3DN.%26aulast%3DAigner%26aufirst%3DT.%26aulast%3DKirchner%26aufirst%3DT.%26aulast%3DReis%26aufirst%3DA.%26aulast%3DBrandst%25C3%25A4tter%26aufirst%3DJ.%2BH.%26aulast%3DRauch%26aufirst%3DA.%26atitle%3DNEK1%2520mutations%2520cause%2520short-rib%2520polydactyly%2520syndrome%2520type%2520Majewski%26jtitle%3DAm.%2520J.%2520Hum.%2520Genet.%26date%3D2011%26volume%3D88%26spage%3D106%26epage%3D114%26doi%3D10.1016%2Fj.ajhg.2010.12.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monroe, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kappen, I. F. P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokman, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terhal, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den
Boogaard, M.-J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savelberg, S. M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Veken, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Es, R. J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lens, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hengeveld, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creton, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janssen, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mink van der Molen, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebbeling, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giles, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoers, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Haaften, G.</span></span> <span> </span><span class="NLM_article-title">Compound heterozygous NEK1 variants in two siblings with oral-facial-digital syndrome type II (Mohr syndrome)</span>. <i>Eur. J. Hum. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1752</span>– <span class="NLM_lpage">1760</span>, <span class="refDoi"> DOI: 10.1038/ejhg.2016.103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1038%2Fejhg.2016.103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=27530628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtl2mt7nF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=1752-1760&author=G.+R.+Monroeauthor=I.+F.+P.+M.+Kappenauthor=M.+F.+Stokmanauthor=P.+A.+Terhalauthor=M.-J.+H.+van+den%0ABoogaardauthor=S.+M.+C.+Savelbergauthor=L.+T.+van+der+Vekenauthor=R.+J.+J.+van+Esauthor=S.+M.+Lensauthor=R.+C.+Hengeveldauthor=M.+A.+Cretonauthor=N.+G.+Janssenauthor=A.+B.+Mink+van+der+Molenauthor=M.+B.+Ebbelingauthor=R.+H.+Gilesauthor=N.+V.+Knoersauthor=G.+van+Haaften&title=Compound+heterozygous+NEK1+variants+in+two+siblings+with+oral-facial-digital+syndrome+type+II+%28Mohr+syndrome%29&doi=10.1038%2Fejhg.2016.103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Compound heterozygous NEK1 variants in two siblings with oral-facial-digital syndrome type II (Mohr syndrome)</span></div><div class="casAuthors">Monroe, Glen R.; Kappen, Isabelle F. P. M.; Stokman, Marijn F.; Terhal, Paulien A.; van den Boogaard, Marie-Jose H.; Savelberg, Sanne M. C.; van der Veken, Lars T.; van Es, Robert J. J.; Lens, Susanne M.; Hengeveld, Rutger C.; Creton, Marijn A.; Janssen, Nard G.; Mink van der Molen, Aebele B.; Ebbeling, Michelle B.; Giles, Rachel H.; Knoers, Nine V.; van Haaften, Gijs</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Human Genetics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1752-1760</span>CODEN:
                <span class="NLM_cas:coden">EJHGEU</span>;
        ISSN:<span class="NLM_cas:issn">1018-4813</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The oral-facial-digital (OFD) syndromes comprise a group of related disorders with a combination of oral, facial and digital anomalies.  Variants in several ciliary genes have been assocd. with subtypes of OFD syndrome, yet in most OFD patients the underlying cause remains unknown.  We investigated the mol. basis of disease in two brothers with OFD type II, Mohr syndrome, by performing single-nucleotide polymorphism (SNP)-array anal. on the brothers and their healthy parents to identify homozygous regions and candidate genes.  Subsequently, we performed whole-exome sequencing (WES) on the family.  Using WES, we identified compd. heterozygous variants c.[464G>C];[1226G>A] in NIMA (Never in Mitosis Gene A)-Related Kinase 1 (NEK1).  The novel variant c.464G>C disturbs normal splicing in an essential region of the kinase domain.  The nonsense variant c.1226G>A, p.  (Trp409*), results in nonsense-assocd. alternative splicing, removing the first coiled-coil domain of NEK1.  Candidate variants were confirmed with Sanger sequencing and alternative splicing assessed with cDNA anal.  Immunocytochem. was used to assess cilia no. and length.  Patient-derived fibroblasts showed severely reduced ciliation compared with control fibroblasts (18.0 vs 48.9%, P<0.0001), but showed no significant difference in cilia length.  In conclusion, we identified compd. heterozygous deleterious variants in NEK1 in two brothers with Mohr syndrome.  Ciliation in patient fibroblasts is drastically reduced, consistent with a ciliary defect pathogenesis.  Our results establish NEK1 variants involved in the etiol. of a subset of patients with OFD syndrome type II and support the consideration of including (routine) NEK1 anal. in patients suspected of OFD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-Su__4biVULVg90H21EOLACvtfcHk0lhKubRgAMHeyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtl2mt7nF&md5=09709846bb36064895492990c81a7848</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fejhg.2016.103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fejhg.2016.103%26sid%3Dliteratum%253Aachs%26aulast%3DMonroe%26aufirst%3DG.%2BR.%26aulast%3DKappen%26aufirst%3DI.%2BF.%2BP.%2BM.%26aulast%3DStokman%26aufirst%3DM.%2BF.%26aulast%3DTerhal%26aufirst%3DP.%2BA.%26aulast%3Dvan%2Bden%2BBoogaard%26aufirst%3DM.-J.%2BH.%26aulast%3DSavelberg%26aufirst%3DS.%2BM.%2BC.%26aulast%3Dvan%2Bder%2BVeken%26aufirst%3DL.%2BT.%26aulast%3Dvan%2BEs%26aufirst%3DR.%2BJ.%2BJ.%26aulast%3DLens%26aufirst%3DS.%2BM.%26aulast%3DHengeveld%26aufirst%3DR.%2BC.%26aulast%3DCreton%26aufirst%3DM.%2BA.%26aulast%3DJanssen%26aufirst%3DN.%2BG.%26aulast%3DMink%2Bvan%2Bder%2BMolen%26aufirst%3DA.%2BB.%26aulast%3DEbbeling%26aufirst%3DM.%2BB.%26aulast%3DGiles%26aufirst%3DR.%2BH.%26aulast%3DKnoers%26aufirst%3DN.%2BV.%26aulast%3Dvan%2BHaaften%26aufirst%3DG.%26atitle%3DCompound%2520heterozygous%2520NEK1%2520variants%2520in%2520two%2520siblings%2520with%2520oral-facial-digital%2520syndrome%2520type%2520II%2520%2528Mohr%2520syndrome%2529%26jtitle%3DEur.%2520J.%2520Hum.%2520Genet.%26date%3D2016%26volume%3D24%26spage%3D1752%26epage%3D1760%26doi%3D10.1038%2Fejhg.2016.103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brenner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieland, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weydt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Böhm, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lulé, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hübers, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuwirth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borck, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wahlqvist, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danzer, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volk, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meitinger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strom, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kassubek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludolph, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weishaupt, J. H.</span></span> <span> </span><span class="NLM_article-title">NEK1 mutations in familial amyotrophic lateral sclerosis</span>. <i>Brain</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>139</i></span>, <span class="NLM_elocation-id">e28</span> <span class="refDoi"> DOI: 10.1093/brain/aww033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1093%2Fbrain%2Faww033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=26945885" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2016&author=D.+Brennerauthor=K.+M%C3%BCllerauthor=T.+Wielandauthor=P.+Weydtauthor=S.+B%C3%B6hmauthor=D.+Lul%C3%A9author=A.+H%C3%BCbersauthor=C.+Neuwirthauthor=M.+Weberauthor=G.+Borckauthor=M.+Wahlqvistauthor=K.+M.+Danzerauthor=A.+E.+Volkauthor=T.+Meitingerauthor=T.+M.+Stromauthor=M.+Ottoauthor=J.+Kassubekauthor=A.+C.+Ludolphauthor=P.+M.+Andersenauthor=J.+H.+Weishaupt&title=NEK1+mutations+in+familial+amyotrophic+lateral+sclerosis&doi=10.1093%2Fbrain%2Faww033"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Fbrain%2Faww033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbrain%252Faww033%26sid%3Dliteratum%253Aachs%26aulast%3DBrenner%26aufirst%3DD.%26aulast%3DM%25C3%25BCller%26aufirst%3DK.%26aulast%3DWieland%26aufirst%3DT.%26aulast%3DWeydt%26aufirst%3DP.%26aulast%3DB%25C3%25B6hm%26aufirst%3DS.%26aulast%3DLul%25C3%25A9%26aufirst%3DD.%26aulast%3DH%25C3%25BCbers%26aufirst%3DA.%26aulast%3DNeuwirth%26aufirst%3DC.%26aulast%3DWeber%26aufirst%3DM.%26aulast%3DBorck%26aufirst%3DG.%26aulast%3DWahlqvist%26aufirst%3DM.%26aulast%3DDanzer%26aufirst%3DK.%2BM.%26aulast%3DVolk%26aufirst%3DA.%2BE.%26aulast%3DMeitinger%26aufirst%3DT.%26aulast%3DStrom%26aufirst%3DT.%2BM.%26aulast%3DOtto%26aufirst%3DM.%26aulast%3DKassubek%26aufirst%3DJ.%26aulast%3DLudolph%26aufirst%3DA.%2BC.%26aulast%3DAndersen%26aufirst%3DP.%2BM.%26aulast%3DWeishaupt%26aufirst%3DJ.%2BH.%26atitle%3DNEK1%2520mutations%2520in%2520familial%2520amyotrophic%2520lateral%2520sclerosis%26jtitle%3DBrain%26date%3D2016%26volume%3D139%26doi%3D10.1093%2Fbrain%2Faww033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Higelin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catanese, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semelink-Sedlacek, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeztuerk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lutz, A.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bausinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speit, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludolph, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demestre, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeckers, T. M.</span></span> <span> </span><span class="NLM_article-title">NEK1 loss-of-function mutation induces DNA damage accumulation in ALS patient-derived motoneurons</span>. <i>Stem Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">150</span>– <span class="NLM_lpage">162</span>, <span class="refDoi"> DOI: 10.1016/j.scr.2018.06.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.scr.2018.06.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=29929116" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFKgsLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2018&pages=150-162&author=J.+Higelinauthor=A.+Cataneseauthor=L.+L.+Semelink-Sedlacekauthor=S.+Oeztuerkauthor=A.-K.+Lutzauthor=J.+Bausingerauthor=G.+Barbiauthor=G.+Speitauthor=P.+M.+Andersenauthor=A.+C.+Ludolphauthor=M.+Demestreauthor=T.+M.+Boeckers&title=NEK1+loss-of-function+mutation+induces+DNA+damage+accumulation+in+ALS+patient-derived+motoneurons&doi=10.1016%2Fj.scr.2018.06.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">NEK1 loss-of-function mutation induces DNA damage accumulation in ALS patient-derived motoneurons</span></div><div class="casAuthors">Higelin, Julia; Catanese, Alberto; Semelink-Sedlacek, Lena Luisa; Oeztuerk, Sertap; Lutz, Anne-Kathrin; Bausinger, Julia; Barbi, Gotthold; Speit, Guenter; Andersen, Peter M.; Ludolph, Albert C.; Demestre, Maria; Boeckers, Tobias M.</div><div class="citationInfo"><span class="NLM_cas:title">Stem Cell Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">150-162</span>CODEN:
                <span class="NLM_cas:coden">SCRTDJ</span>;
        ISSN:<span class="NLM_cas:issn">1873-5061</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Mutations in genes coding for proteins involved in DNA damage response (DDR) and repair, such as C9orf72 and FUS (Fused in Sarcoma), are assocd. with neurodegenerative diseases and lead to amyotrophic lateral sclerosis (ALS).  Heterozygous loss-of-function mutations in NEK1 (NIMA-related kinase 1) have also been recently found to cause ALS.  NEK1 codes for a multifunctional protein, crucially involved in mitotic checkpoint control and DDR.  To resolve pathol. alterations assocd. with NEK1 mutation, we compared hiPSC-derived motoneurons carrying a NEK1 mutation with mutant C9orf72 and wild type neurons at basal level and after DNA damage induction.  Motoneurons carrying a C9orf72 mutation exhibited cell specific signs of increased DNA damage.  This phenotype was even more severe in NEK1c.2434A gT neurons that showed significantly increased DNA damage at basal level and impaired DDR after induction of DNA damage in an maturation-dependent manner.  Our results provide first mechanistic insight in pathophysiol. alterations induced by NEK1 mutations and point to a converging pathomechanism of different gene mutations causative for ALS.  Therefore, our study contributes to the development of novel therapeutic strategies to reduce DNA damage accumulation in neurodegenerative diseases and ALS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOEC_t3JzYX7Vg90H21EOLACvtfcHk0ljh72j71SOlVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFKgsLbL&md5=36ef39e4c9867350a97f592763cce762</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.scr.2018.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.scr.2018.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DHigelin%26aufirst%3DJ.%26aulast%3DCatanese%26aufirst%3DA.%26aulast%3DSemelink-Sedlacek%26aufirst%3DL.%2BL.%26aulast%3DOeztuerk%26aufirst%3DS.%26aulast%3DLutz%26aufirst%3DA.-K.%26aulast%3DBausinger%26aufirst%3DJ.%26aulast%3DBarbi%26aufirst%3DG.%26aulast%3DSpeit%26aufirst%3DG.%26aulast%3DAndersen%26aufirst%3DP.%2BM.%26aulast%3DLudolph%26aufirst%3DA.%2BC.%26aulast%3DDemestre%26aufirst%3DM.%26aulast%3DBoeckers%26aufirst%3DT.%2BM.%26atitle%3DNEK1%2520loss-of-function%2520mutation%2520induces%2520DNA%2520damage%2520accumulation%2520in%2520ALS%2520patient-derived%2520motoneurons%26jtitle%3DStem%2520Cell%2520Res.%26date%3D2018%26volume%3D30%26spage%3D150%26epage%3D162%26doi%3D10.1016%2Fj.scr.2018.06.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fry, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O'Regan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabir, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span> <span> </span><span class="NLM_article-title">Cell cycle regulation by the NEK family of protein kinases</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">4423</span>– <span class="NLM_lpage">4433</span>, <span class="refDoi"> DOI: 10.1242/jcs.111195</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1242%2Fjcs.111195" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=23132929" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjsl2gtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2012&pages=4423-4433&author=A.+M.+Fryauthor=L.+O%27Reganauthor=S.+R.+Sabirauthor=R.+Bayliss&title=Cell+cycle+regulation+by+the+NEK+family+of+protein+kinases&doi=10.1242%2Fjcs.111195"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Cell cycle regulation by the NEK family of protein kinases</span></div><div class="casAuthors">Fry, Andrew M.; O'Regan, Laura; Sabir, Sarah R.; Bayliss, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4423-4433</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">A review.  Genetic screens for cell division cycle mutants in the filamentous fungus Aspergillus nidulans led to the discovery of never-in-mitosis A (NIMA) kinase, a serine/threonine kinase that is required for mitotic entry.  Since that discovery, NIMA-related kinases, or NEKs, have been identified in most eukaryotes, including humans where 11 genetically distinct proteins named NEK1 to NEK11 are expressed.  Although there is no evidence that human NEKs are essential for mitotic entry, it is clear that several NEK family members play important roles in cell cycle control.  In particular, NEK2, NEK6, NEK7, and NEK9 contribute to the establishment of the microtubule-based mitotic spindle, whereas NEK1, NEK10, and NEK11 have been implicated in the DNA damage response.  Roles for NEKs in other aspects of mitotic progression, such as chromatin condensation, nuclear envelope breakdown, spindle assembly checkpoint signaling, and cytokinesis have also been proposed.  Interestingly, NEK1 and NEK8 also function within cilia, the microtubule-based structures that are nucleated from basal bodies.  This has led to the current hypothesis that NEKs have evolved to coordinate microtubule-dependent processes in both dividing and non-dividing cells.  Here, the authors review the functions of human NEKs, with particular emphasis on those family members that are involved in cell cycle control, and consider their potential as therapeutic targets in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKCk9miADRfLVg90H21EOLACvtfcHk0lgUVvI8yY287w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjsl2gtA%253D%253D&md5=6965616ca443eeeb40fb320bea94bd36</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1242%2Fjcs.111195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.111195%26sid%3Dliteratum%253Aachs%26aulast%3DFry%26aufirst%3DA.%2BM.%26aulast%3DO%2527Regan%26aufirst%3DL.%26aulast%3DSabir%26aufirst%3DS.%2BR.%26aulast%3DBayliss%26aufirst%3DR.%26atitle%3DCell%2520cycle%2520regulation%2520by%2520the%2520NEK%2520family%2520of%2520protein%2520kinases%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2012%26volume%3D125%26spage%3D4423%26epage%3D4433%26doi%3D10.1242%2Fjcs.111195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Polci, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">NIMA-related protein kinase 1 is involved early in the ionizing radiation-induced DNA damage response</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">8800</span>– <span class="NLM_lpage">8803</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-04-2243</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1158%2F0008-5472.CAN-04-2243" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=15604234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOms7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2004&pages=8800-8803&author=R.+Polciauthor=A.+Pengauthor=P.-L.+Chenauthor=D.+J.+Rileyauthor=Y.+Chen&title=NIMA-related+protein+kinase+1+is+involved+early+in+the+ionizing+radiation-induced+DNA+damage+response&doi=10.1158%2F0008-5472.CAN-04-2243"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">NIMA-Related Protein Kinase 1 Is Involved Early in the Ionizing Radiation-Induced DNA Damage Response</span></div><div class="casAuthors">Polci, Rosaria; Peng, Aimin; Chen, Phang-Lang; Riley, Daniel J.; Chen, Yumay</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8800-8803</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Cellular functions of the NimA-related mammalian kinase Nek1 have not been demonstrated to date.  Here we show that Nek1 is involved early in the DNA damage response induced by ionizing radiation (IR) and that Nek1 is important for cells to repair and recover from DNA damage.  When primary or transformed cells are exposed to IR, Nek1 kinase activity is increased within 4 min, and Nek1 expression is up-regulated shortly thereafter and sustained for hours.  At the same early time frame after IR that its kinase activity is highest, a portion of Nek1 redistributes in cells from cytoplasm to discrete nuclear foci at sites of DNA double-strand breaks.  There it colocalizes with γ-H2AX and NFBD1/MDC1, two key proteins involved very early in the response to IR-induced DNA double-strand breaks.  Finally, Nek1-deficient fibroblasts are much more sensitive to the effects of IR-induced DNA damage than otherwise identical fibroblasts expressing Nek1.  These results suggest that Nek1 may function as a kinase early in the DNA damage response pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrq1l6-l3mMqrVg90H21EOLACvtfcHk0lgUVvI8yY287w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOms7bL&md5=efa8478012fcbe045e6cf22bf9d394b6</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-04-2243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-04-2243%26sid%3Dliteratum%253Aachs%26aulast%3DPolci%26aufirst%3DR.%26aulast%3DPeng%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DP.-L.%26aulast%3DRiley%26aufirst%3DD.%2BJ.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DNIMA-related%2520protein%2520kinase%25201%2520is%2520involved%2520early%2520in%2520the%2520ionizing%2520radiation-induced%2520DNA%2520damage%2520response%26jtitle%3DCancer%2520Res.%26date%3D2004%26volume%3D64%26spage%3D8800%26epage%3D8803%26doi%3D10.1158%2F0008-5472.CAN-04-2243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grudzenski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raths, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conrad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rübe, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Löbrich, M.</span></span> <span> </span><span class="NLM_article-title">Inducible response required for repair of low-dose radiation damage in human fibroblasts</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">14205</span>– <span class="NLM_lpage">14210</span>, <span class="refDoi"> DOI: 10.1073/pnas.1002213107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1073%2Fpnas.1002213107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=20660770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVCkt7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=14205-14210&author=S.+Grudzenskiauthor=A.+Rathsauthor=S.+Conradauthor=C.+E.+R%C3%BCbeauthor=M.+L%C3%B6brich&title=Inducible+response+required+for+repair+of+low-dose+radiation+damage+in+human+fibroblasts&doi=10.1073%2Fpnas.1002213107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Inducible response required for repair of low-dose radiation damage in human fibroblasts</span></div><div class="casAuthors">Grudzenski, Saskia; Raths, Antonia; Conrad, Sandro; Rube, Claudia E.; Lobrich, Markus</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">14205-14210, S14205/1-S14205/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Ionizing radiation (IR) induces a variety of DNA lesions among which DNA double-strand breaks (DSBs) are the biol. most significant.  It is currently unclear if DSB repair is equally efficient after low and high doses.  Here, we use γ-H2AX, phospho-ATM (pATM), and 53BP1 foci anal. to monitor DSB repair.  We show, consistent with a previous study, that the kinetics of γ-H2AX and pATM foci loss in confluent primary human fibroblasts are substantially compromised after doses of 10 mGy and lower.  Following 2.5 mGy, cells fail to show any foci loss.  Strikingly, cells pretreated with 10 μM H2O2 efficiently remove all γ-H2AX foci induced by 10 mGy.  At the concn. used, H2O2 produces single-strand breaks and base damages via the generation of oxygen radicals but no DSBs.  Moreover, 10 μM H2O2 up-regulates a set of genes that is also up-regulated after high (200 mGy) but not after low (10 mGy) radiation doses.  This suggests that low radical levels induce a response that is required for the repair of radiation-induced DSBs when the radiation damage is too low to cause the induction itself.  To address the in vivo significance of this finding, we established γ-H2AX and 53BP1 foci anal. in various mouse tissues.  Although mice irradiated with 100 mGy or 1 Gy show efficient γ-H2AX and 53BP1 foci removal during 24 h post-IR, barely any foci loss was obsd. after 10 mGy.  Our data suggest that the cellular response to DSBs is substantially different for low vs. high radiation doses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVaHxBRlpDFbVg90H21EOLACvtfcHk0lggcSFlEZ7HqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVCkt7%252FP&md5=c1cf547306699f8cea631a43a84e0756</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1002213107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1002213107%26sid%3Dliteratum%253Aachs%26aulast%3DGrudzenski%26aufirst%3DS.%26aulast%3DRaths%26aufirst%3DA.%26aulast%3DConrad%26aufirst%3DS.%26aulast%3DR%25C3%25BCbe%26aufirst%3DC.%2BE.%26aulast%3DL%25C3%25B6brich%26aufirst%3DM.%26atitle%3DInducible%2520response%2520required%2520for%2520repair%2520of%2520low-dose%2520radiation%2520damage%2520in%2520human%2520fibroblasts%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D14205%26epage%3D14210%26doi%3D10.1073%2Fpnas.1002213107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spies, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waizenegger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heyer, W.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobrich, M.</span></span> <span> </span><span class="NLM_article-title">Nek1 regulates Rad54 to orchestrate homologous recombination and replication fork stability</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">903</span>– <span class="NLM_lpage">917</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2016.04.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.molcel.2016.04.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=27264870" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC28Xpt1entbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2016&pages=903-917&author=J.+Spiesauthor=A.+Waizeneggerauthor=O.+Bartonauthor=M.+Surderauthor=W.+D.+Wrightauthor=W.-D.+Heyerauthor=M.+Lobrich&title=Nek1+regulates+Rad54+to+orchestrate+homologous+recombination+and+replication+fork+stability&doi=10.1016%2Fj.molcel.2016.04.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Nek1 Regulates Rad54 to Orchestrate Homologous Recombination and Replication Fork Stability</span></div><div class="casAuthors">Spies, Julian; Waizenegger, Anja; Barton, Olivia; Suerder, Michael; Wright, William D.; Heyer, Wolf-Dietrich; Loebrich, Markus</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">903-917</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Never-in-mitosis A-related kinase 1 (Nek1) has established roles in apoptosis and cell cycle regulation.  We show that human Nek1 regulates homologous recombination (HR) by phosphorylating Rad54 at Ser572 in late G2 phase.  Nek1 deficiency as well as expression of unphosphorylatable Rad54 (Rad54-S572A) cause unresolved Rad51 foci and confer a defect in HR.  Phospho-mimic Rad54 (Rad54-S572E), in contrast, promotes HR and rescues the HR defect assocd. with Nek1 loss.  Although expression of phospho-mimic Rad54 is beneficial for HR, it causes Rad51 removal from chromatin and degrdn. of stalled replication forks in S phase.  Thus, G2-specific phosphorylation of Rad54 by Nek1 promotes Rad51 chromatin removal during HR in G2 phase, and its absence in S phase is required for replication fork stability.  In summary, Nek1 regulates Rad51 removal to orchestrate HR and replication fork stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUj-FGAj-B6rVg90H21EOLACvtfcHk0lggcSFlEZ7HqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xpt1entbY%253D&md5=66566886a3cc28d7737dbaf963e4da13</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2016.04.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2016.04.032%26sid%3Dliteratum%253Aachs%26aulast%3DSpies%26aufirst%3DJ.%26aulast%3DWaizenegger%26aufirst%3DA.%26aulast%3DBarton%26aufirst%3DO.%26aulast%3DSurder%26aufirst%3DM.%26aulast%3DWright%26aufirst%3DW.%2BD.%26aulast%3DHeyer%26aufirst%3DW.-D.%26aulast%3DLobrich%26aufirst%3DM.%26atitle%3DNek1%2520regulates%2520Rad54%2520to%2520orchestrate%2520homologous%2520recombination%2520and%2520replication%2520fork%2520stability%26jtitle%3DMol.%2520Cell%26date%3D2016%26volume%3D62%26spage%3D903%26epage%3D917%26doi%3D10.1016%2Fj.molcel.2016.04.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D. J.</span></span> <span> </span><span class="NLM_article-title">Never-in-mitosis related kinase 1 functions in DNA damage response and checkpoint control</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">3194</span>– <span class="NLM_lpage">3201</span>, <span class="refDoi"> DOI: 10.4161/cc.7.20.6815</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.4161%2Fcc.7.20.6815" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=18843199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsV2mu7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=3194-3201&author=Y.+Chenauthor=P.-L.+Chenauthor=C.-F.+Chenauthor=X.+Jiangauthor=D.+J.+Riley&title=Never-in-mitosis+related+kinase+1+functions+in+DNA+damage+response+and+checkpoint+control&doi=10.4161%2Fcc.7.20.6815"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Never-in-mitosis related kinase 1 functions in DNA damage response and checkpoint control</span></div><div class="casAuthors">Chen, Yumay; Chen, Phang-Lang; Chen, Chi-Fen; Jiong, Xianzhi; Riley, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3194-3201</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Nek1, the 1st mammalian ortholog of the fungal protein kinase never in mitosis A, is involved early in the DNA damage sensing/repair pathway after ionizing radiation (IR).  Here we extend this finding by showing that Nek1 localizes to nuclear foci of DNA damage in response to many different types of damage in addn. to IR.  Untransformed cells established from kat2J/Nek1-/- mice fail to arrest properly at G1/S and M-phase checkpoints in response to DNA damage.  G1-S-phase checkpoint control can be rescued by ectopically overexpressing wild-type Nek1.  In Nek1-/- murine cells and in human cells with Nek1 expression silenced by siRNA, the checkpoint kinases Chk1 and Chk2 fail to be activated properly in response to ionizing or UV radiation.  In cells without functional Nek1, DNA is not repaired properly, double-stranded DNA breaks persist long after low dose IR, and excessive nos. of chromosome breaks are obsd.  These data show that Nek1 is important for efficient DNA damage checkpoint control and for proper DNA damage repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqBuaNvqTOXbVg90H21EOLACvtfcHk0lggcSFlEZ7HqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsV2mu7fM&md5=5b37e281a263c733fed1de3aa86cb1f6</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.4161%2Fcc.7.20.6815&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.7.20.6815%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DP.-L.%26aulast%3DChen%26aufirst%3DC.-F.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DRiley%26aufirst%3DD.%2BJ.%26atitle%3DNever-in-mitosis%2520related%2520kinase%25201%2520functions%2520in%2520DNA%2520damage%2520response%2520and%2520checkpoint%2520control%26jtitle%3DCell%2520Cycle%26date%3D2008%26volume%3D7%26spage%3D3194%26epage%3D3201%26doi%3D10.4161%2Fcc.7.20.6815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.-L.</span></span> <span> </span><span class="NLM_article-title">Nek1 kinase functions in DNA damage response and checkpoint control through a pathway independent of ATM and ATR</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">655</span>– <span class="NLM_lpage">663</span>, <span class="refDoi"> DOI: 10.4161/cc.10.4.14814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.4161%2Fcc.10.4.14814" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=21301226" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkt1CksL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=655-663&author=Y.+Chenauthor=C.-F.+Chenauthor=D.+J.+Rileyauthor=P.-L.+Chen&title=Nek1+kinase+functions+in+DNA+damage+response+and+checkpoint+control+through+a+pathway+independent+of+ATM+and+ATR&doi=10.4161%2Fcc.10.4.14814"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Nek1 kinase functions in DNA damage response and checkpoint control through a pathway independent of ATM and ATR</span></div><div class="casAuthors">Chen, Yumay; Chen, Chi-Fen; Riley, Daniel J.; Chen, Phang-Lang</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">655-663</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Never-in-mitosis A related protein kinase 1 (Nek1) is involved early in a DNA damage sensing/repair pathway.  We have previously shown that cells without functional Nek1 fail to activate the more distal kinases Chk1 and Chk2 and fail to arrest properly at G1/S or M-phase checkpoints in response to DNA damage.  As a consequence, foci of damaged DNA in Nek1 null cells persist long after the instigating insult, and Nek1 null cells develop unstable chromosomes at a rate much higher than identically cultured wild-type cells.  Here we show that Nek1 functions independently of canonical DNA damage responses requiring the PI3 kinase-like proteins ATM and ATR.  Chem. inhibitors of ATM/ATR or mutation of the genes that encode them fail to alter the kinase activity of Nek1 or its localization to nuclear foci of DNA damage.  Moreover ATM and ATR activities, including the localization of the proteins to DNA damage sites and phosphorylation of early DNA damage response substrates, are intact in Nek1-/- murine cells and in human cells with Nek1 expression silenced by siRNA.  Our results demonstrate that Nek1 is important for proper checkpoint control and characterize for the first time a DNA damage response that does not directly involve one of the known upstream mediator kinases, ATM or ATR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohGiyVtPj3QbVg90H21EOLACvtfcHk0ljcEFV-WI1nGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkt1CksL4%253D&md5=2a14716ba3eb13e83a24fab6ce4dc8ea</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.4161%2Fcc.10.4.14814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.10.4.14814%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DC.-F.%26aulast%3DRiley%26aufirst%3DD.%2BJ.%26aulast%3DChen%26aufirst%3DP.-L.%26atitle%3DNek1%2520kinase%2520functions%2520in%2520DNA%2520damage%2520response%2520and%2520checkpoint%2520control%2520through%2520a%2520pathway%2520independent%2520of%2520ATM%2520and%2520ATR%26jtitle%3DCell%2520Cycle%26date%3D2011%26volume%3D10%26spage%3D655%26epage%3D663%26doi%3D10.4161%2Fcc.10.4.14814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craigen, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D. J.</span></span> <span> </span><span class="NLM_article-title">Nek1 regulates cell death and mitochondrial membrane permeability through phosphorylation of VDAC1</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">267</span>, <span class="refDoi"> DOI: 10.4161/cc.8.2.7551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.4161%2Fcc.8.2.7551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=19158487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvFSqtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=257-267&author=Y.+Chenauthor=W.+J.+Craigenauthor=D.+J.+Riley&title=Nek1+regulates+cell+death+and+mitochondrial+membrane+permeability+through+phosphorylation+of+VDAC1&doi=10.4161%2Fcc.8.2.7551"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Nek1 regulates cell death and mitochondrial membrane permeability through phosphorylation of VDAC1</span></div><div class="casAuthors">Chen, Yumay; Craigen, William J.; Riley, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">257-267</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Mammalian NIMA-related protein kinase 1 (Nek1) is important for keeping cells alive after DNA damage, but the mechanism by which injured cells die without functional Nek1 has not yet been demonstrated.  Here, the authors show that Nek1 regulates the pathway to mitochondrial cell death through phosphorylation of voltage-dependent anion channel 1 (VDAC1) on its Ser-193 residue.  Nek1 assocs. with VDAC1 in a yeast 2-hybrid system, as well as by glutathione S-transferase pull-down assays and by reciprocal immunopptn.  A portion of Nek1 in cells also localizes at mitochondria.  Ectopic expression of a kinase-dead Nek1 mutant resulted in cell death, which was immediately preceded by loss of Nek1-dependent VDAC1-Ser-193 phosphorylation.  UV irradn. of Nek1-deficient cells or silencing of endogenous Nek1 expression similarly resulted in the loss of the specific Ser-193 phosphorylation before cells died.  Nek1-deficient cells were characterized by exaggerated mitochondrial membrane permeability (MMP) and accelerated cell death.  Ectopic expression of a VDAC1-S193A mutant, which could not be phosphorylated by Nek1, also resulted in cell death.  A VDAC1-S193E mutant, designed to mimic constitutive phosphorylation by Nek1, rescued exaggerated MMP and kept cells alive after DNA damaging injury, but only transiently.  Thus, the direct interaction between Nek1 and VDAC1 provides a mechanism to explain how Nek1 prevents excessive cell death, as well as the 1st direct evidence that a specific kinase regulates VDAC1 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqecVvyQbM_BbVg90H21EOLACvtfcHk0ljcEFV-WI1nGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvFSqtbg%253D&md5=06a242fcfc11ddb9607433b86a157a43</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.4161%2Fcc.8.2.7551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.8.2.7551%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DCraigen%26aufirst%3DW.%2BJ.%26aulast%3DRiley%26aufirst%3DD.%2BJ.%26atitle%3DNek1%2520regulates%2520cell%2520death%2520and%2520mitochondrial%2520membrane%2520permeability%2520through%2520phosphorylation%2520of%2520VDAC1%26jtitle%3DCell%2520Cycle%26date%3D2009%26volume%3D8%26spage%3D257%26epage%3D267%26doi%3D10.4161%2Fcc.8.2.7551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaczynska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osmulski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polci, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D. J.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation by Nek1 regulates opening and closing of voltage dependent anion channel 1</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>394</i></span>,  <span class="NLM_fpage">798</span>– <span class="NLM_lpage">803</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2010.03.077</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.bbrc.2010.03.077" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=20230784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC3cXks1Kitrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=394&publication_year=2010&pages=798-803&author=Y.+Chenauthor=M.+Gaczynskaauthor=P.+Osmulskiauthor=R.+Polciauthor=D.+J.+Riley&title=Phosphorylation+by+Nek1+regulates+opening+and+closing+of+voltage+dependent+anion+channel+1&doi=10.1016%2Fj.bbrc.2010.03.077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation by Nek1 regulates opening and closing of voltage dependent anion channel 1</span></div><div class="casAuthors">Chen, Yumay; Gaczynska, Maria; Osmulski, Pawel; Polci, Rosaria; Riley, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">394</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">798-803</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Voltage-dependent anion channel 1 (VDAC1) is a key component of the mitochondrial permeability transition pore.  To initiate apoptosis and certain other forms of cell death, mitochondria become permeable such that cytochrome c and other pre-apoptotic mols. resident inside the mitochondria enter the cytosol and activate apoptotic cascades.  The authors showed recently that VDAC1 interacts directly with never-in-mitosis A-related kinase 1 (Nek1), and that Nek1 phosphorylates VDAC1 on Ser-193 to prevent excessive cell death after injury.  How this phosphorylation regulates the activity of VDAC1, however, has not yet been reported.  Here, the authors used at. force microscopy (AFM) and cytochrome c conductance studies to examine the configuration of VDAC1 before and after phosphorylation by Nek1.  Wild-type VDAC1 assumes an open configuration, but closes and prevents cytochrome c efflux when phosphorylated by Nek1.  A VDAC1-S193A mutant, which could not be phosphorylated by Nek1 under identical conditions, remained open and constitutively allowed cytochrome c efflux.  Conversely, a VDAC1-S193E mutant, which mimics constitutive phosphorylation by Nek1, remained closed by AFM and prevented cytochrome c leakage in the same liposome assays.  Thus, these data provide a mechanism to explain how Nek1 regulates cell death by affecting the opening and closing of VDAC1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfr0zRey7Y-bVg90H21EOLACvtfcHk0ljF7xu3g3Nnag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXks1Kitrg%253D&md5=34c91949652c63004253b0eccebb5def</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2010.03.077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2010.03.077%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DGaczynska%26aufirst%3DM.%26aulast%3DOsmulski%26aufirst%3DP.%26aulast%3DPolci%26aufirst%3DR.%26aulast%3DRiley%26aufirst%3DD.%2BJ.%26atitle%3DPhosphorylation%2520by%2520Nek1%2520regulates%2520opening%2520and%2520closing%2520of%2520voltage%2520dependent%2520anion%2520channel%25201%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2010%26volume%3D394%26spage%3D798%26epage%3D803%26doi%3D10.1016%2Fj.bbrc.2010.03.077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khalil, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Benedetti, A.</span></span> <span> </span><span class="NLM_article-title">The TLK1/Nek1 axis contributes to mitochondrial integrity and apoptosis prevention via phosphorylation of VDAC1</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">363</span>– <span class="NLM_lpage">375</span>, <span class="refDoi"> DOI: 10.1080/15384101.2019.1711317</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1080%2F15384101.2019.1711317" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=31914854" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlsVKitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2020&pages=363-375&author=V.+Singhauthor=M.+I.+Khalilauthor=A.+De+Benedetti&title=The+TLK1%2FNek1+axis+contributes+to+mitochondrial+integrity+and+apoptosis+prevention+via+phosphorylation+of+VDAC1&doi=10.1080%2F15384101.2019.1711317"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The TLK1/Nek1 axis contributes to mitochondrial integrity and apoptosis prevention via phosphorylation of VDAC1</span></div><div class="casAuthors">Singh, Vibha; Khalil, Md. Imtiaz; De Benedetti, Arrigo</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">363-375</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1551-4005</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">The TLK1/Nek1 axis contributes to cell cycle arrest and implementation of the DDR to mediate survival upon DNA damage.  However, when the damage is too severe, the cells typically are forced into apoptosis, and the contribution of TLKs in this process has not been investigated.  In contrast, it is known that Nek1 may play a role by phosphorylating VDAC1 maintaining proper opening and closure of the channel and thus mitochondrial integrity.  We now show that the activating phosphorylation of Nek1-T141 by TLK1 contributes to the phosphorylation and stability of VDAC1 and thereby to mitochondrial permeability and integrity.  Treatment of three different cell lines model that overexpress Nek1-T141A mutant with doxorubicin showed exquisite sensitivity to the drug, with implementation of rapid accumulation of cells with subG1 DNA content (apoptotic) and other alterations in the cell cycle.  In addn., these cells displayed reduced oxygen consumption under normal conditions and less reliance on mitochondria and more dependence on glycolysis for energy prodn.  Consistent with greater apoptosis, upon treatment with low doses of doxorubicin, cells overexpressing Nek1-T141A displayed leakage of Cyt-C into the cytoplasmic fraction.  This suggests that inhibiting the TLK1/Nek1/VDAC1 nexus could sensitize cancer cells to apoptotic killing in combination with an appropriate DNA damaging agent.  We in fact have previously reported that Nek1 expression is elevated in advanced Prostate Cancer (PCa) and we now report that VDAC1 expression is elevated and correlated with disease stage, thereby making the TLK1/Nek1/VDAC1 nexus a very attractive target for PCa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxEwQEOU6NULVg90H21EOLACvtfcHk0ljF7xu3g3Nnag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlsVKitw%253D%253D&md5=d2f7766566b1c42a61a9ed1bf2fcf20d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1080%2F15384101.2019.1711317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15384101.2019.1711317%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DV.%26aulast%3DKhalil%26aufirst%3DM.%2BI.%26aulast%3DDe%2BBenedetti%26aufirst%3DA.%26atitle%3DThe%2520TLK1%252FNek1%2520axis%2520contributes%2520to%2520mitochondrial%2520integrity%2520and%2520apoptosis%2520prevention%2520via%2520phosphorylation%2520of%2520VDAC1%26jtitle%3DCell%2520Cycle%26date%3D2020%26volume%3D19%26spage%3D363%26epage%3D375%26doi%3D10.1080%2F15384101.2019.1711317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaiswal, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koul, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Benedetti, A.</span></span> <span> </span><span class="NLM_article-title">The TLK1-Nek1 axis promotes prostate cancer progression</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>453</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">141</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2019.03.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.canlet.2019.03.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=30928383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmvFCgsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=453&publication_year=2019&pages=131-141&author=V.+Singhauthor=P.+K.+Jaiswalauthor=I.+Ghoshauthor=H.+K.+Koulauthor=X.+Yuauthor=A.+De+Benedetti&title=The+TLK1-Nek1+axis+promotes+prostate+cancer+progression&doi=10.1016%2Fj.canlet.2019.03.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The TLK1-Nek1 axis promotes prostate cancer progression</span></div><div class="casAuthors">Singh, Vibha; Jaiswal, Praveen Kumar; Ghosh, Ishita; Koul, Hari K.; Yu, Xiuping; De Benedetti, Arrigo</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">453</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">131-141</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">We recently uncovered the crit. TLK1>NEK1>ATR > Chk1 axis in mediating the DDR and cell cycle checkpoint while transiting from Androgen Sensitive to Insensitive growth for LNCaP and TRAMP-C2 cells.  However, we did not know the generality of this pathway in PCa progression since there are few cell lines where the transition has been studied.  Furthermore, the identification of Nek1, and more importantly the TLK-mediated phosphorylation of T141, has never been studied in PCa biopsies.  We now report the first study of a PCa TMA of p-Nek1-T141 and correlation to the Gleason score.  In addn. we found that TRAMP mice treated with the TLK inhibitor, thioridazine (THD), following castration did not recover cancerous growth of their prostates.  Moreover, we recapitulated the process of translational increase in TLK1B expression in a naive PDX model that was established from an AR + adenocarcinoma.  Therefore, we believe that this TLK1-Nek1 mediated DDR axis is likely to be a common adaptive response during the transition of PCa cells toward androgen-insensitive growth, and hence CRPC progression, which has the potential to be targeted with THD and other TLK or Nek1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUOEFnPux-zbVg90H21EOLACvtfcHk0lj2-Mr3Si4ETw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmvFCgsLw%253D&md5=48556b830b64b1c1e19cbde42947d3d4</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2019.03.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2019.03.041%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DV.%26aulast%3DJaiswal%26aufirst%3DP.%2BK.%26aulast%3DGhosh%26aufirst%3DI.%26aulast%3DKoul%26aufirst%3DH.%2BK.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DDe%2BBenedetti%26aufirst%3DA.%26atitle%3DThe%2520TLK1-Nek1%2520axis%2520promotes%2520prostate%2520cancer%2520progression%26jtitle%3DCancer%2520Lett.%26date%3D2019%26volume%3D453%26spage%3D131%26epage%3D141%26doi%3D10.1016%2Fj.canlet.2019.03.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaiswal, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koul, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Benedetti, A.</span></span> <span> </span><span class="NLM_article-title">Targeting the TLK1/NEK1 DDR axis with Thioridazine suppresses outgrowth of androgen independent prostate tumors</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>145</i></span>,  <span class="NLM_fpage">1055</span>– <span class="NLM_lpage">1067</span>, <span class="refDoi"> DOI: 10.1002/ijc.32200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1002%2Fijc.32200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=30737777" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjsFenu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=145&publication_year=2019&pages=1055-1067&author=V.+Singhauthor=P.+K.+Jaiswalauthor=I.+Ghoshauthor=H.+K.+Koulauthor=X.+Yuauthor=A.+De+Benedetti&title=Targeting+the+TLK1%2FNEK1+DDR+axis+with+Thioridazine+suppresses+outgrowth+of+androgen+independent+prostate+tumors&doi=10.1002%2Fijc.32200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the TLK1/NEK1 DDR axis with Thioridazine suppresses outgrowth of androgen independent prostate tumors</span></div><div class="casAuthors">Singh, Vibha; Jaiswal, Praveen Kumar; Ghosh, Ishita; Koul, Hari K.; Yu, Xiuping; De Benedetti, Arrigo</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">145</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1055-1067</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We now show that androgen deprivation therapy (ADT) of LNCaP cells results in increased expression of TLK1B, a key kinase upstream of NEK1 and ATR and mediating the DDR that typically results in a temporary cell cycle arrest of androgen responsive PCa cells.  Following DNA damage, addn. of the TLK specific inhibitor, thioridazine (THD), impairs ATR and Chk1 activation, establishing the existence of a ADT > TLK1 > NEK1 > ATR > Chk1, DDR pathway, while its abrogation leads to apoptosis.  Treatment with THD suppressed the outgrowth of androgen-independent (AI) colonies of LNCaP and TRAMP-C2 cells cultured with bicalutamide.  Moreover, THD significantly inhibited the growth of several PCa cells in vitro (including AI lines).  Administration of THD or bicalutamide was not effective at inhibiting long-term tumor growth of LNCaP xenografts.  In contrast, combination therapy remarkably inhibited tumor growth via bypass of the DDR.  Moreover, xenografts of LNCaP cells overexpressing a NEK1-T141A mutant were durably suppressed with bicalutamide.  Collectively, these results suggest that targeting the TLK1/NEK1 axis might be a novel therapy for PCa in combination with std. of care (ADT).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkKHEgGNvGzbVg90H21EOLACvtfcHk0lj2-Mr3Si4ETw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjsFenu7o%253D&md5=417328fe7127cf68127e9b9191c7cd8d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fijc.32200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.32200%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DV.%26aulast%3DJaiswal%26aufirst%3DP.%2BK.%26aulast%3DGhosh%26aufirst%3DI.%26aulast%3DKoul%26aufirst%3DH.%2BK.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DDe%2BBenedetti%26aufirst%3DA.%26atitle%3DTargeting%2520the%2520TLK1%252FNEK1%2520DDR%2520axis%2520with%2520Thioridazine%2520suppresses%2520outgrowth%2520of%2520androgen%2520independent%2520prostate%2520tumors%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2019%26volume%3D145%26spage%3D1055%26epage%3D1067%26doi%3D10.1002%2Fijc.32200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Benedetti, A.</span></span> <span> </span><span class="NLM_article-title">Identification of the proteome complement of humanTLK1 reveals it binds and phosphorylates NEK1 regulating its activity</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">915</span>– <span class="NLM_lpage">926</span>, <span class="refDoi"> DOI: 10.1080/15384101.2017.1314421</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1080%2F15384101.2017.1314421" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=28426283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvVChurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=915-926&author=V.+Singhauthor=Z.+M.+Connellyauthor=X.+Shenauthor=A.+De+Benedetti&title=Identification+of+the+proteome+complement+of+humanTLK1+reveals+it+binds+and+phosphorylates+NEK1+regulating+its+activity&doi=10.1080%2F15384101.2017.1314421"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of the proteome complement of human TLK1 reveals it binds and phosphorylates NEK1 regulating its activity</span></div><div class="casAuthors">Singh, Vibha; Connelly, Zachary M.; Shen, Xinggui; De Benedetti, Arrigo</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">915-926</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1551-4005</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">The Tousled Like kinases (TLKs) are involved in numerous cellular functions, including the DNA Damage Response (DDR), but only a handful of substrates have been identified thus far.  Through a novel proteomic screen, we have now identified 165 human proteins interacting with TLK1, and we have focused this work on NEK1 because of its known role in the DDR, upstream of ATR and Chk1.  TLK1 and NEK1 were found to interact by coIP, and their binding is strengthened following exposure of cells to H2O2.  Following incubation with doxorubicin, TLK1 and NEK1 relocalize with nuclear repair foci along with γH2AX.  TLK1 phosphorylated NEK1 at T141, which lies in the kinase domain, and caused an increase in its activity.  Following DNA damage, addn. of the TLK1 inhibitor, THD, or overexpression of NEK1-T141A mutant impaired ATR and Chk1 activation, indicating the existence of a TLK1>NEK1>ATR>Chk1 pathway.  Indeed, overexpression of the NEK1-T141A mutant resulted in an altered cell cycle response after exposure of cells to oxidative stress, including bypass of G1 arrest and implementation of an intra S-phase checkpoint.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnk8vGEabv-bVg90H21EOLACvtfcHk0lj2-Mr3Si4ETw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvVChurg%253D&md5=c5ff4681869a38739a0d7e0d27523efe</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1080%2F15384101.2017.1314421&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15384101.2017.1314421%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DV.%26aulast%3DConnelly%26aufirst%3DZ.%2BM.%26aulast%3DShen%26aufirst%3DX.%26aulast%3DDe%2BBenedetti%26aufirst%3DA.%26atitle%3DIdentification%2520of%2520the%2520proteome%2520complement%2520of%2520humanTLK1%2520reveals%2520it%2520binds%2520and%2520phosphorylates%2520NEK1%2520regulating%2520its%2520activity%26jtitle%3DCell%2520Cycle%26date%3D2017%26volume%3D16%26spage%3D915%26epage%3D926%26doi%3D10.1080%2F15384101.2017.1314421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Melo-Hanchuk, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martins, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunha, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassallo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobarg, J.</span></span> <span> </span><span class="NLM_article-title">Expression of the NEK family in normal and cancer tissue: an immunohistochemical study</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">23</span>, <span class="refDoi"> DOI: 10.1186/s12885-019-6408-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1186%2Fs12885-019-6408-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=31906878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjsl2htw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2020&pages=23&author=T.+D.+Melo-Hanchukauthor=M.+B.+Martinsauthor=L.+L.+Cunhaauthor=F.+A.+Soaresauthor=L.+S.+Wardauthor=J.+Vassalloauthor=J.+Kobarg&title=Expression+of+the+NEK+family+in+normal+and+cancer+tissue%3A+an+immunohistochemical+study&doi=10.1186%2Fs12885-019-6408-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of the NEK family in normal and cancer tissue: an immunohistochemical study</span></div><div class="casAuthors">Melo-Hanchuk, Talita Diniz; Martins, Mariana Bonjiorno; Cunha, Lucas Leite; Soares, Fernando Augusto; Ward, Laura Sterian; Vassallo, Jose; Kobarg, Jorg</div><div class="citationInfo"><span class="NLM_cas:title">BMC Cancer</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23</span>CODEN:
                <span class="NLM_cas:coden">BCMACL</span>;
        ISSN:<span class="NLM_cas:issn">1471-2407</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">The NEK serine/threonine protein kinases are involved in cell cycle checkpoints, DNA damage repair, and apoptosis.  Alterations in these pathways are frequently assocd. with cell malignant cellular transformations.  Thyroid cancer is the most common malignant tumor in the endocrine system.  Despite good treatment methods, the no. of cases has increased significantly in recent years.  Here, we studied the expression of NEK1, NEK2, NEK3, and NEK5 in different types of normal and malignant tissues, using tissue microarray anal., and identified NEKs as potential markers in thyroid malignancy.  The studied cases comprised multiple cancer tissue microarrays, including breast, colon, esophagus, kidney, lung, pancreas, prostate, stomach, thyroid and uterine cervix, as well as 281 patients who underwent thyroid resection for thyroid cancer or thyroid nodules.  The expression of NEK1, NEK2, NEK3, and NEK5 was analyzed by immunohistochem.  The expression pattern was evaluated in terms of intensity by two methods, semiquant. and quant., and was compared between normal and cancer tissue.  We analyzed the expression of each member of the NEK family in a tissue-dependent manner.  Compared to normal tissue, most of the evaluated proteins showed lower expression in lung tumor.  However, in the thyroid, the expression was higher in malignant tissue, esp. for NEK 1, NEK3 and NEK5.  Concerning characteristics of the thyroid tumor, such as aggressiveness, NEK1 expression was higher in tumors with multifocality and in patients with lymph node metastasis.  NEK3 expression was stronger in patients with stage II, that involved metastasis.  NEK5, on the other hand, showed high expression in patients with invasion and metastasis and in patients with tumor size > 4 cm.  Furthermore, this work, demonstrated for the first time a high specificity and sensitivity of over-expression of NEK1 in classical and follicular variants of papillary thyroid cancer and NEK3 in tall-cell papillary thyroid cancer.  Taken together, the NEK protein kinases emerge as important proteins in thyroid cancer development and may help to identify malignancy and aggressiveness features during diagnosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7IByQd0eQfLVg90H21EOLACvtfcHk0lhtkFDGJ06vbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjsl2htw%253D%253D&md5=dce11d760e07b7d9be7a2936496d79bb</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1186%2Fs12885-019-6408-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12885-019-6408-4%26sid%3Dliteratum%253Aachs%26aulast%3DMelo-Hanchuk%26aufirst%3DT.%2BD.%26aulast%3DMartins%26aufirst%3DM.%2BB.%26aulast%3DCunha%26aufirst%3DL.%2BL.%26aulast%3DSoares%26aufirst%3DF.%2BA.%26aulast%3DWard%26aufirst%3DL.%2BS.%26aulast%3DVassallo%26aufirst%3DJ.%26aulast%3DKobarg%26aufirst%3DJ.%26atitle%3DExpression%2520of%2520the%2520NEK%2520family%2520in%2520normal%2520and%2520cancer%2520tissue%253A%2520an%2520immunohistochemical%2520study%26jtitle%3DBMC%2520Cancer%26date%3D2020%26volume%3D20%26spage%3D23%26doi%3D10.1186%2Fs12885-019-6408-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span> <span> </span><span class="NLM_article-title">Frequent Nek1 overexpression in human gliomas</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>476</i></span>,  <span class="NLM_fpage">522</span>– <span class="NLM_lpage">527</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2016.05.156</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.bbrc.2016.05.156" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=27251576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslWqtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=476&publication_year=2016&pages=522-527&author=J.+Zhuauthor=Y.+Caiauthor=P.+Liuauthor=W.+Zhao&title=Frequent+Nek1+overexpression+in+human+gliomas&doi=10.1016%2Fj.bbrc.2016.05.156"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Frequent Nek1 overexpression in human gliomas</span></div><div class="casAuthors">Zhu, Jun; Cai, Yu; Liu, Pin; Zhao, Weiguo</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">476</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">522-527</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Never in mitosis A (NIMA)-related kinase 1 (Nek1) regulates cell cycle progression to mitosis.  Its expression and potential functions in human gliomas have not been studied.  Here, our immunohistochem. (IHC) assay and Western blot assay results showed that Nek1 expression was significantly upregulated in fresh and paraffin-embedded human glioma tissues.  Its level in normal brain tissues was low.  Nek1 overexpression in human gliomas was correlated with the proliferation marker (Ki-67), tumor grade, Karnofsky performance scale (KPS) and more importantly, patients' poor survival.  Further studies showed that Nek1 expression level was also increased in multiple human glioma cell lines (U251-MG, U87-MG, U118, H4 and U373).  Significantly, siRNA-mediated knockdown of Nek1 inhibited glioma cell (U87-MG/U251-MG) growth.  Nek1 siRNA also sensitized U87-MG/U251-MG cells to temozolomide (TMZ), causing a profound apoptosis induction and growth inhibition.  The current study indicates Nek1 might be a novel and valuable oncotarget of glioma, it is important for glioma cell growth and TMZ-resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6Hw3K3vHISLVg90H21EOLACvtfcHk0lhtkFDGJ06vbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslWqtbc%253D&md5=52b7149201a0891a0f95b21499a23edd</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2016.05.156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2016.05.156%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DZhao%26aufirst%3DW.%26atitle%3DFrequent%2520Nek1%2520overexpression%2520in%2520human%2520gliomas%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2016%26volume%3D476%26spage%3D522%26epage%3D527%26doi%3D10.1016%2Fj.bbrc.2016.05.156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polci, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.-L.</span></span> <span> </span><span class="NLM_article-title">Increased Nek1 expression in renal cell carcinoma cells is associated with decreased sensitivity to DNA-damaging treatment</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">4283</span>– <span class="NLM_lpage">4294</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.18632%2Foncotarget.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=24970796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A280%3ADC%252BC2cfntFSnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=4283-4294&author=Y.+Chenauthor=C.-F.+Chenauthor=R.+Polciauthor=R.+Weiauthor=D.+J.+Rileyauthor=P.-L.+Chen&title=Increased+Nek1+expression+in+renal+cell+carcinoma+cells+is+associated+with+decreased+sensitivity+to+DNA-damaging+treatment&doi=10.18632%2Foncotarget.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Increased Nek1 expression in renal cell carcinoma cells is associated with decreased sensitivity to DNA-damaging treatment</span></div><div class="casAuthors">Chen Yumay; Chen Chi-Fen; Wei Randy; Chen Phang-Lang; Polci Rosaria; Riley Daniel J</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4283-94</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Renal cell carcinoma (RCC) is a heterogeneous disease with resistance to systemic chemotherapy.  Elevated expression of multiple drug resistance (MDR) has been suggested to be one of the mechanisms for this resistance.  Here, we provide an alternative mechanism to explain RCC's resistance to chemotherapy-induced apoptosis.  Never-in mitosis A-related protein kinase 1 (Nek1) plays an important role in DNA damage response and proper checkpoint activation.  The association of Nek1 with the voltage-dependent anion channel (VDAC1) is a critical determinant of cell survival following DNA-damaging treatment.  We report here that Nek1 is highly expressed in RCC tumor and cultured RCC cells compared to that of normal renal tubular epithelial cells (RTE).  The association between Nek1 and VDAC1 is genotoxic dependent: prolonged Nek1/VDAC1 dissociation will lead to VDAC1 dephosphorylation and initiate apoptosis.  Down-regulation of Nek1 expression in RCC cells enhanced their sensitivity to DNA-damaging treatment.  Collectively, these results suggest that the increased Nek1 expression in RCC cells maintain persistent VDAC1 phosphorylation, closing its channel and preventing the onset of apoptosis under genotoxic insults.  Based on these results, we believe that Nek1 can serve as a potential therapeutic target for drug development in the treatment of RCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTkcXTsKbMaOIIRAdBLKY41fW6udTcc2eYlem-gYruYFLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cfntFSnsw%253D%253D&md5=82dcd4a0048660d34dcf8d6c695bd367</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.2005%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DC.-F.%26aulast%3DPolci%26aufirst%3DR.%26aulast%3DWei%26aufirst%3DR.%26aulast%3DRiley%26aufirst%3DD.%2BJ.%26aulast%3DChen%26aufirst%3DP.-L.%26atitle%3DIncreased%2520Nek1%2520expression%2520in%2520renal%2520cell%2520carcinoma%2520cells%2520is%2520associated%2520with%2520decreased%2520sensitivity%2520to%2520DNA-damaging%2520treatment%26jtitle%3DOncotarget%26date%3D2014%26volume%3D5%26spage%3D4283%26epage%3D4294%26doi%3D10.18632%2Foncotarget.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">Overexpression of Nek1 in renal cell carcinoma protects against cell death from ionizing radiation</span>. <i>Int. J. Radiat. Oncol., Biol., Phys.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">S712</span>, <span class="refDoi"> DOI: 10.1016/j.ijrobp.2011.06.1364</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.ijrobp.2011.06.1364" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=20932667" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2011&pages=S712&author=R.+L.+Weiauthor=Y.+Chen&title=Overexpression+of+Nek1+in+renal+cell+carcinoma+protects+against+cell+death+from+ionizing+radiation&doi=10.1016%2Fj.ijrobp.2011.06.1364"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.ijrobp.2011.06.1364&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijrobp.2011.06.1364%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DR.%2BL.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DOverexpression%2520of%2520Nek1%2520in%2520renal%2520cell%2520carcinoma%2520protects%2520against%2520cell%2520death%2520from%2520ionizing%2520radiation%26jtitle%3DInt.%2520J.%2520Radiat.%2520Oncol.%252C%2520Biol.%252C%2520Phys.%26date%3D2011%26volume%3D81%26spage%3DS712%26doi%3D10.1016%2Fj.ijrobp.2011.06.1364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Freund, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hehlgans, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ensminger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fokas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rödel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Löbrich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rödel, F.</span></span> <span> </span><span class="NLM_article-title">Fractionation-dependent radiosensitization by molecular targeting of Nek1</span>. <i>Cells</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1235</span>, <span class="refDoi"> DOI: 10.3390/cells9051235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.3390%2Fcells9051235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1Sku7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&pages=1235&author=I.+Freundauthor=S.+Hehlgansauthor=D.+Martinauthor=M.+Ensmingerauthor=E.+Fokasauthor=C.+R%C3%B6delauthor=M.+L%C3%B6brichauthor=F.+R%C3%B6del&title=Fractionation-dependent+radiosensitization+by+molecular+targeting+of+Nek1&doi=10.3390%2Fcells9051235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Fractionation-dependent radiosensitization by molecular targeting of Nek1</span></div><div class="casAuthors">Freund, Isabel; Hehlgans, Stephanie; Martin, Daniel; Ensminger, Michael; Fokas, Emmanouil; Roedel, Claus; Loebrich, Markus; Roedel, Franz</div><div class="citationInfo"><span class="NLM_cas:title">Cells</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1235</span>CODEN:
                <span class="NLM_cas:coden">CELLC6</span>;
        ISSN:<span class="NLM_cas:issn">2073-4409</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">NIMA (never-in-mitosis gene A)-related kinase 1 (Nek1) is shown to impact on different cellular pathways such as DNA repair, checkpoint activation, and apoptosis.  Its role as a mol. target for radiation sensitization of malignant cells, however, remains elusive.  Stably transduced doxycycline (Dox)-inducible Nek1 shRNA HeLa cervix and siRNA-transfected HCT-15 colorectal carcinoma cells were irradiated in vitro and 3D clonogenic radiation survival, residual DNA damage, cell cycle distribution, and apoptosis were analyzed.  Nek1 knockdown (KD) sensitized both cell lines to ionizing radiation following a single dose irradn. and more pronounced in combination with a 6 h fractionation (3 x 2 Gy) regime.  For preclin. analyses we focused on cervical cancer.  Nek1 shRNA HeLa cells were grafted into NOD/SCID/IL-2Rγc-/- (NSG) mice and Nek1 KD was induced by Dox-infused drinking water resulting in a significant cytostatic effect if combined with a 6 h fractionation (3 x 2 Gy) regime.  In addn., we correlated Nek1 expression in biopsies of patients with cervical cancer with histopathol. parameters and clin. follow-up.  Our results indicate that elevated levels of Nek1 were assocd. with an increased rate of local or distant failure, as well as with impaired cancer-specific and overall survival in univariate analyses and for most endpoints in multivariable analyses.  Finally, findings from The Cancer Genome Atlas (TCGA) validation cohort confirmed a significant assocn. of high Nek1 expression with a reduced disease-free survival.  In conclusion, we consider Nek1 to represent a novel biomarker and potential therapeutic target for drug development in the context of optimized fractionation intervals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH2xPGDJXPObVg90H21EOLACvtfcHk0li3hmupcPNcwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1Sku7jO&md5=292b4ae41d1a19d25b24418cddca3980</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.3390%2Fcells9051235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcells9051235%26sid%3Dliteratum%253Aachs%26aulast%3DFreund%26aufirst%3DI.%26aulast%3DHehlgans%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DD.%26aulast%3DEnsminger%26aufirst%3DM.%26aulast%3DFokas%26aufirst%3DE.%26aulast%3DR%25C3%25B6del%26aufirst%3DC.%26aulast%3DL%25C3%25B6brich%26aufirst%3DM.%26aulast%3DR%25C3%25B6del%26aufirst%3DF.%26atitle%3DFractionation-dependent%2520radiosensitization%2520by%2520molecular%2520targeting%2520of%2520Nek1%26jtitle%3DCells%26date%3D2020%26volume%3D9%26spage%3D1235%26doi%3D10.3390%2Fcells9051235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">NEK1 protein kinase as a target for anticancer therapeutics</span>. <i>Chemotherapy: Open Access</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">e118</span>, <span class="refDoi"> DOI: 10.4172/2167-7700.1000e118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.4172%2F2167-7700.1000e118" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2012&pages=e118&author=Y.+Chen&title=NEK1+protein+kinase+as+a+target+for+anticancer+therapeutics&doi=10.4172%2F2167-7700.1000e118"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.4172%2F2167-7700.1000e118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4172%252F2167-7700.1000e118%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26atitle%3DNEK1%2520protein%2520kinase%2520as%2520a%2520target%2520for%2520anticancer%2520therapeutics%26jtitle%3DChemotherapy%253A%2520Open%2520Access%26date%3D2012%26volume%3D1%26spage%3De118%26doi%3D10.4172%2F2167-7700.1000e118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moraes, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meirelles, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honorato, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Souza, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Souza, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murakami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Oliveira, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobarg, J.</span></span> <span> </span><span class="NLM_article-title">Kinase inhibitor profile for human nek1, nek6, and nek7 and analysis of the structural basis for inhibitor specificity</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1176</span>– <span class="NLM_lpage">1191</span>, <span class="refDoi"> DOI: 10.3390/molecules20011176</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.3390%2Fmolecules20011176" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=25591119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVyhu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=1176-1191&author=E.+Moraesauthor=G.+Meirellesauthor=R.+Honoratoauthor=T.+de+Souzaauthor=E.+de+Souzaauthor=M.+Murakamiauthor=P.+de+Oliveiraauthor=J.+Kobarg&title=Kinase+inhibitor+profile+for+human+nek1%2C+nek6%2C+and+nek7+and+analysis+of+the+structural+basis+for+inhibitor+specificity&doi=10.3390%2Fmolecules20011176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitor profile for human Nek1, Nek6, and Nek7 and analysis of the structural basis for inhibitor specificity</span></div><div class="casAuthors">Moraes, Eduardo Cruz; Meirelles, Gabriela Vaz; Honorato, Rodrigo Vargas; Brasil de Souza, Tatiana de Arruda Campos; Elisa de Souza, Edmarcia; Murakami, Mario Tyago; Lopes de Oliveira, Paulo Sergio; Kobarg, Jorg</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1176-1191, 16 pp.</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Human Neks are a conserved protein kinase family related to cell cycle progression and cell division and are considered potential drug targets for the treatment of cancer and other pathologies.  We screened the activation loop mutant kinases hNek1 and hNek2, wild-type hNek7, and five hNek6 variants in different activation/phosphorylation statesand compared them against 85 compds. using thermal shift denaturation.  We identified three compds. with significant Tm shifts: JNK Inhibitor II for hNek1(Δ262-1258)-(T162A), Isogranulatimide for hNek6(S206A), andGSK-3 Inhibitor XIII for hNek7wt.  Each one of these compds. was also validated by reducing the kinases activity by at least 25%.  The binding sites for these compds. were identified by in silico docking at the ATP-binding site of the resp. hNeks.  Potential inhibitors were first screened by thermal shift assays, had their efficiency tested by a kinase assay, and were finally analyzed by mol. docking.  Our findings corroborate the idea of ATP-competitive inhibition for hNek1 and hNek6 and suggest a novel non-competitive inhibition for hNek7 in regard to GSK-3 Inhibitor XIII.  Our results demonstrate that our approach is useful for finding promising general and specific hNekscandidate inhibitors, which may also function as scaffolds to design more potent and selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpx1iPvmNQxk7Vg90H21EOLACvtfcHk0li3hmupcPNcwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVyhu7o%253D&md5=97e48f6686c84f7171b094981f591810</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.3390%2Fmolecules20011176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules20011176%26sid%3Dliteratum%253Aachs%26aulast%3DMoraes%26aufirst%3DE.%26aulast%3DMeirelles%26aufirst%3DG.%26aulast%3DHonorato%26aufirst%3DR.%26aulast%3Dde%2BSouza%26aufirst%3DT.%26aulast%3Dde%2BSouza%26aufirst%3DE.%26aulast%3DMurakami%26aufirst%3DM.%26aulast%3Dde%2BOliveira%26aufirst%3DP.%26aulast%3DKobarg%26aufirst%3DJ.%26atitle%3DKinase%2520inhibitor%2520profile%2520for%2520human%2520nek1%252C%2520nek6%252C%2520and%2520nek7%2520and%2520analysis%2520of%2520the%2520structural%2520basis%2520for%2520inhibitor%2520specificity%26jtitle%3DMolecules%26date%3D2015%26volume%3D20%26spage%3D1176%26epage%3D1191%26doi%3D10.3390%2Fmolecules20011176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Melo-Hanchuk, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slepicka, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meirelles, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basei, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovato, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granato, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauletti, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Domingues, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leme, A. F. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelegrini, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobarg, J.</span></span> <span> </span><span class="NLM_article-title">NEK1 kinase domain structure and its dynamic protein interactome after exposure to Cisplatin</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">5445</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-05325-w</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1038%2Fs41598-017-05325-w" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=28710492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A280%3ADC%252BC1cjnvVCktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=5445&author=T.+D.+Melo-Hanchukauthor=P.+F.+Slepickaauthor=G.+V.+Meirellesauthor=F.+L.+Baseiauthor=D.+V.+Lovatoauthor=D.+C.+Granatoauthor=B.+A.+Paulettiauthor=R.+R.+Dominguesauthor=A.+F.+P.+Lemeauthor=A.+L.+Pelegriniauthor=G.+Lenzauthor=S.+Knappauthor=J.+M.+Elkinsauthor=J.+Kobarg&title=NEK1+kinase+domain+structure+and+its+dynamic+protein+interactome+after+exposure+to+Cisplatin&doi=10.1038%2Fs41598-017-05325-w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">NEK1 kinase domain structure and its dynamic protein interactome after exposure to Cisplatin</span></div><div class="casAuthors">Melo-Hanchuk Talita D; Meirelles Gabriela Vaz; Basei Fernanda Luisa; Kobarg Jorg; Slepicka Priscila Ferreira; Lovato Diogo Ventura; Granato Daniela Campos; Pauletti Bianca Alves; Domingues Romenia Ramos; Leme Adriana Franco Paes; Pelegrini Alessandra Luiza; Lenz Guido; Knapp Stefan; Elkins Jonathan M; Knapp Stefan; Elkins Jonathan M; Kobarg Jorg</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5445</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">NEK family kinases are serine/threonine kinases that have been functionally implicated in the regulation of the disjunction of the centrosome, the assembly of the mitotic spindle, the function of the primary cilium and the DNA damage response.  NEK1 shows pleiotropic functions and has been found to be mutated in cancer cells, ciliopathies such as the polycystic kidney disease, as well as in the genetic diseases short-rib thoracic dysplasia, Mohr-syndrome and amyotrophic lateral sclerosis.  NEK1 is essential for the ionizing radiation DNA damage response and priming of the ATR kinase and of Rad54 through phosphorylation.  Here we report on the structure of the kinase domain of human NEK1 in its apo- and ATP-mimetic inhibitor bound forms.  The inhibitor bound structure may allow the design of NEK specific chemo-sensitizing agents to act in conjunction with chemo- or radiation therapy of cancer cells.  Furthermore, we characterized the dynamic protein interactome of NEK1 after DNA damage challenge with cisplatin.  Our data suggest that NEK1 and its interaction partners trigger the DNA damage pathways responsible for correcting DNA crosslinks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQENCCRhZkgKDToTC3aBaesfW6udTcc2eZEDpjNLA-N-bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cjnvVCktA%253D%253D&md5=c36569b4a002cb1bdacd0dae93622d38</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-05325-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-05325-w%26sid%3Dliteratum%253Aachs%26aulast%3DMelo-Hanchuk%26aufirst%3DT.%2BD.%26aulast%3DSlepicka%26aufirst%3DP.%2BF.%26aulast%3DMeirelles%26aufirst%3DG.%2BV.%26aulast%3DBasei%26aufirst%3DF.%2BL.%26aulast%3DLovato%26aufirst%3DD.%2BV.%26aulast%3DGranato%26aufirst%3DD.%2BC.%26aulast%3DPauletti%26aufirst%3DB.%2BA.%26aulast%3DDomingues%26aufirst%3DR.%2BR.%26aulast%3DLeme%26aufirst%3DA.%2BF.%2BP.%26aulast%3DPelegrini%26aufirst%3DA.%2BL.%26aulast%3DLenz%26aufirst%3DG.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DKobarg%26aufirst%3DJ.%26atitle%3DNEK1%2520kinase%2520domain%2520structure%2520and%2520its%2520dynamic%2520protein%2520interactome%2520after%2520exposure%2520to%2520Cisplatin%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D5445%26doi%3D10.1038%2Fs41598-017-05325-w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapadia, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Counago, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span> <span> </span><span class="NLM_article-title">In depth analysis of kinase cross screening data to identify chemical starting points for inhibition of the Nek family of kinases</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">44</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1039/C7MD00510E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1039%2FC7MD00510E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=30108900" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFersbzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=44-66&author=C.+I.+Wellsauthor=N.+R.+Kapadiaauthor=R.+M.+Counagoauthor=D.+H.+Drewry&title=In+depth+analysis+of+kinase+cross+screening+data+to+identify+chemical+starting+points+for+inhibition+of+the+Nek+family+of+kinases&doi=10.1039%2FC7MD00510E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">In depth analysis of kinase cross screening data to identify chemical starting points for inhibition of the Nek family of kinases</span></div><div class="casAuthors">Wells, C. I.; Kapadia, N. R.; Counago, R. M.; Drewry, D. H.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">44-66</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Potent, selective, and cell active small mol. kinase inhibitors are useful tools to help unravel the complexities of kinase signaling.  As the biol. functions of individual kinases become better understood, they can become targets of drug discovery efforts.  The small mols. used to shed light on function can also then serve as chem. starting points in these drug discovery efforts.  The Nek family of kinases has received very little attention, as judged by no. of citations in PubMed, yet they appear to play many key roles and have been implicated in disease.  Here we present our work to identify high quality chem. starting points that have emerged due to the increased incidence of broad kinome screening.  We anticipate that this anal. will allow the community to progress towards the generation of chem. probes and eventually drugs that target members of the Nek family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBzprTGDD0hbVg90H21EOLACvtfcHk0ljcuSn72XiVyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFersbzJ&md5=0daf63621e95f51f04cb119fa98d69d9</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1039%2FC7MD00510E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7MD00510E%26sid%3Dliteratum%253Aachs%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DKapadia%26aufirst%3DN.%2BR.%26aulast%3DCounago%26aufirst%3DR.%2BM.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26atitle%3DIn%2520depth%2520analysis%2520of%2520kinase%2520cross%2520screening%2520data%2520to%2520identify%2520chemical%2520starting%2520points%2520for%2520inhibition%2520of%2520the%2520Nek%2520family%2520of%2520kinases%26jtitle%3DMedChemComm%26date%3D2018%26volume%3D9%26spage%3D44%26epage%3D66%26doi%3D10.1039%2FC7MD00510E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Donato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Righino, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippetti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaglia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrillo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pirolli, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scambia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Rosa, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallo, D.</span></span> <span> </span><span class="NLM_article-title">Identification and antitumor activity of a novel inhibitor of the NIMA-related kinase NEK6</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">16047</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-34471-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1038%2Fs41598-018-34471-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=30375481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A280%3ADC%252BB3cvkslCmtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=16047&author=M.+De%0ADonatoauthor=B.+Righinoauthor=F.+Filippettiauthor=A.+Battagliaauthor=M.+Petrilloauthor=D.+Pirolliauthor=G.+Scambiaauthor=M.+C.+De+Rosaauthor=D.+Gallo&title=Identification+and+antitumor+activity+of+a+novel+inhibitor+of+the+NIMA-related+kinase+NEK6&doi=10.1038%2Fs41598-018-34471-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and antitumor activity of a novel inhibitor of the NIMA-related kinase NEK6</span></div><div class="casAuthors">De Donato Marta; Filippetti Flavia; Battaglia Alessandra; Petrillo Marco; Scambia Giovanni; Gallo Daniela; De Donato Marta; Filippetti Flavia; Scambia Giovanni; Gallo Daniela; Righino Benedetta; Petrillo Marco; Pirolli Davide; De Rosa Maria Cristina</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">16047</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The NIMA (never in mitosis, gene A)-related kinase-6 (NEK6), which is implicated in cell cycle control and plays significant roles in tumorigenesis, is an attractive target for the development of novel anti-cancer drugs.  Here we describe the discovery of a potent ATP site-directed inhibitor of NEK6 identified by virtual screening, adopting both structure- and ligand-based techniques.  Using a homology-built model of NEK6 as well as the pharmacophoric features of known NEK6 inhibitors we identified novel binding scaffolds.  Twenty-five compounds from the top ranking hits were subjected to in vitro kinase assays.  The best compound, i.e. compound 8 ((5Z)-2-hydroxy-4-methyl-6-oxo-5-[(5-phenylfuran-2-yl)methylidene]-5,6-dihydropyridine-3-carbonitrile), was able to inhibit NEK6 with low micromolar IC50 value, also displaying antiproliferative activity against a panel of human cancer cell lines.  Our results suggest that the identified inhibitor can be used as lead candidate for the development of novel anti-cancer agents, thus opening the possibility of new therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRglXfZMl481XthwrmtH4OBfW6udTcc2eZEDpjNLA-N-bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cvkslCmtw%253D%253D&md5=7660b56cc7704766fc7dd27893409f9b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-34471-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-34471-y%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BDonato%26aufirst%3DM.%26aulast%3DRighino%26aufirst%3DB.%26aulast%3DFilippetti%26aufirst%3DF.%26aulast%3DBattaglia%26aufirst%3DA.%26aulast%3DPetrillo%26aufirst%3DM.%26aulast%3DPirolli%26aufirst%3DD.%26aulast%3DScambia%26aufirst%3DG.%26aulast%3DDe%2BRosa%26aufirst%3DM.%2BC.%26aulast%3DGallo%26aufirst%3DD.%26atitle%3DIdentification%2520and%2520antitumor%2520activity%2520of%2520a%2520novel%2520inhibitor%2520of%2520the%2520NIMA-related%2520kinase%2520NEK6%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D16047%26doi%3D10.1038%2Fs41598-018-34471-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Ali, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axtman, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capuzzi, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederiksen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hooper, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luecking, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelides, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saikatendu, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span> <span> </span><span class="NLM_article-title">Progress towards a public chemogenomic set for protein kinases and a call for contributions</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0181585</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0181585</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1371%2Fjournal.pone.0181585" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=28767711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFWktL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=D.+H.+Drewryauthor=C.+I.+Wellsauthor=D.+M.+Andrewsauthor=R.+Angellauthor=H.+Al-Aliauthor=A.+D.+Axtmanauthor=S.+J.+Capuzziauthor=J.+M.+Elkinsauthor=P.+Ettmayerauthor=M.+Frederiksenauthor=O.+Gileadiauthor=N.+Grayauthor=A.+Hooperauthor=S.+Knappauthor=S.+Lauferauthor=U.+Lueckingauthor=M.+Michaelidesauthor=S.+M%C3%BCllerauthor=E.+Muratovauthor=R.+A.+Dennyauthor=K.+S.+Saikatenduauthor=D.+K.+Treiberauthor=W.+J.+Zuercherauthor=T.+M.+Willson&title=Progress+towards+a+public+chemogenomic+set+for+protein+kinases+and+a+call+for+contributions&doi=10.1371%2Fjournal.pone.0181585"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Progress towards a public chemogenomic set for protein kinases and a call for contributions</span></div><div class="casAuthors">Drewry, David H.; Wells, Carrow I.; Andrews, David M.; Angell, Richard; Al-Ali, Hassan; Axtman, Alison D.; Capuzzi, Stephen J.; Elkins, Jonathan M.; Ettmayer, Peter; Frederiksen, Mathias; Gileadi, Opher; Gray, Nathanael; Hooper, Alice; Knapp, Stefan; Laufer, Stefan; Luecking, Ulrich; Michaelides, Michael; Muller, Susanne; Muratov, Eugene; Aldrin, Denny R.; Saikatendu, Kumar S.; Treiber, Daniel K.; Zuercher, William J.; Willson, Timothy M.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e0181585/1-e0181585/20</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein kinases are highly tractable targets for drug discovery.  However, the biol. function and therapeutic potential of the majority of the 500+ human protein kinases remains unknown.  We have developed phys. and virtual collections of small mol. inhibitors, which we call chemogenomic sets, that are designed to inhibit the catalytic function of almost half the human protein kinases.  In this manuscript we share our progress towards generation of a comprehensive kinase chemogenomic set (KCGS), release kinome profiling data of a large inhibitor set (Published Kinase Inhibitor Set 2 (PKIS2)), and outline a process through which the community can openly collaborate to create a KCGS that probes the full complement of human protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpozOiNZl51XbVg90H21EOLACvtfcHk0lhJ-JNZnnRw9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFWktL0%253D&md5=9c27ba5cd21da43d30ce485ffa0e1675</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0181585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0181585%26sid%3Dliteratum%253Aachs%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DAndrews%26aufirst%3DD.%2BM.%26aulast%3DAngell%26aufirst%3DR.%26aulast%3DAl-Ali%26aufirst%3DH.%26aulast%3DAxtman%26aufirst%3DA.%2BD.%26aulast%3DCapuzzi%26aufirst%3DS.%2BJ.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DEttmayer%26aufirst%3DP.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DHooper%26aufirst%3DA.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DLaufer%26aufirst%3DS.%26aulast%3DLuecking%26aufirst%3DU.%26aulast%3DMichaelides%26aufirst%3DM.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DMuratov%26aufirst%3DE.%26aulast%3DDenny%26aufirst%3DR.%2BA.%26aulast%3DSaikatendu%26aufirst%3DK.%2BS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26atitle%3DProgress%2520towards%2520a%2520public%2520chemogenomic%2520set%2520for%2520protein%2520kinases%2520and%2520a%2520call%2520for%2520contributions%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0181585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seefeld, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouse, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNulty, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luengo, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minthorn, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Concha, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heerding, D. A.</span></span> <span> </span><span class="NLM_article-title">Discovery of 5-pyrrolopyridinyl-2-thiophenecarboxamides as potent AKT kinase inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2244</span>– <span class="NLM_lpage">2248</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2009.02.094</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.bmcl.2009.02.094" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=19285393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkt1Sjt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=2244-2248&author=M.+A.+Seefeldauthor=M.+B.+Rouseauthor=K.+C.+McNultyauthor=L.+Sunauthor=J.+Wangauthor=D.+S.+Yamashitaauthor=J.+I.+Luengoauthor=S.+Zhangauthor=E.+A.+Minthornauthor=N.+O.+Conchaauthor=D.+A.+Heerding&title=Discovery+of+5-pyrrolopyridinyl-2-thiophenecarboxamides+as+potent+AKT+kinase+inhibitors&doi=10.1016%2Fj.bmcl.2009.02.094"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 5-pyrrolopyridinyl-2-thiophenecarboxamides as potent AKT kinase inhibitors</span></div><div class="casAuthors">Seefeld, Mark A.; Rouse, Meagan B.; McNulty, Kenneth C.; Sun, Lihui; Wang, Jizhou; Yamashita, Dennis S.; Luengo, Juan I.; Zhang, ShuYun; Minthorn, Elisabeth A.; Concha, Nestor O.; Heerding, Dirk A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2244-2248</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A pyrrolopyridinyl thiophenecarboxamide I was discovered as a tractable starting point for a lead optimization effort in an AKT kinase inhibition program.  SAR studies aided by a co-crystal structure of I in AKT2 led to the identification of AKT inhibitors with subnanomolar potency.  Representative compds. showed antiproliferative activity as well as inhibition of phosphorylation of the downstream target GSK3β.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_nx0I3_W9-7Vg90H21EOLACvtfcHk0lhJ-JNZnnRw9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkt1Sjt7s%253D&md5=7d34026abcf524ffced43f2d0e4629ec</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.02.094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.02.094%26sid%3Dliteratum%253Aachs%26aulast%3DSeefeld%26aufirst%3DM.%2BA.%26aulast%3DRouse%26aufirst%3DM.%2BB.%26aulast%3DMcNulty%26aufirst%3DK.%2BC.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DYamashita%26aufirst%3DD.%2BS.%26aulast%3DLuengo%26aufirst%3DJ.%2BI.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DMinthorn%26aufirst%3DE.%2BA.%26aulast%3DConcha%26aufirst%3DN.%2BO.%26aulast%3DHeerding%26aufirst%3DD.%2BA.%26atitle%3DDiscovery%2520of%25205-pyrrolopyridinyl-2-thiophenecarboxamides%2520as%2520potent%2520AKT%2520kinase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D2244%26epage%3D2248%26doi%3D10.1016%2Fj.bmcl.2009.02.094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liddle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cousins, R. P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobbs, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioannou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellor, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payne, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pritchard, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tape, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, R. A.</span></span> <span> </span><span class="NLM_article-title">4-Phenyl-7-azaindoles as potent and selective IKK2 inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2504</span>– <span class="NLM_lpage">2508</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2009.03.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.bmcl.2009.03.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=19349179" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkslyitbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=2504-2508&author=J.+Liddleauthor=P.+Bamboroughauthor=M.+D.+Barkerauthor=S.+Camposauthor=R.+P.+C.+Cousinsauthor=G.+J.+Cutlerauthor=H.+Hobbsauthor=D.+S.+Holmesauthor=C.+Ioannouauthor=G.+W.+Mellorauthor=M.+A.+Morseauthor=J.+J.+Payneauthor=J.+M.+Pritchardauthor=K.+J.+Smithauthor=D.+T.+Tapeauthor=C.+Whitworthauthor=R.+A.+Williamson&title=4-Phenyl-7-azaindoles+as+potent+and+selective+IKK2+inhibitors&doi=10.1016%2Fj.bmcl.2009.03.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">4-Phenyl-7-azaindoles as potent and selective IKK2 inhibitors</span></div><div class="casAuthors">Liddle, John; Bamborough, Paul; Barker, Michael D.; Campos, Sebastien; Cousins, Rick P. C.; Cutler, Geoffrey J.; Hobbs, Heather; Holmes, Duncan S.; Ioannou, Chris; Mellor, Geoff W.; Morse, Mary A.; Payne, Jeremy J.; Pritchard, John M.; Smith, Kathryn J.; Tape, Daniel T.; Whitworth, Caroline; Williamson, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2504-2508</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The synthesis and SAR of a novel series of IKK2 inhibitors are described.  Modification around the hinge binding region of the 7-azaindole led to a series of potent and selective inhibitors with good cellular activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqav8UP5rEwSbVg90H21EOLACvtfcHk0lgF7WBR7SsINg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkslyitbk%253D&md5=e53c6eda9da4d535410c96c460de511b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.03.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.03.034%26sid%3Dliteratum%253Aachs%26aulast%3DLiddle%26aufirst%3DJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBarker%26aufirst%3DM.%2BD.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DCousins%26aufirst%3DR.%2BP.%2BC.%26aulast%3DCutler%26aufirst%3DG.%2BJ.%26aulast%3DHobbs%26aufirst%3DH.%26aulast%3DHolmes%26aufirst%3DD.%2BS.%26aulast%3DIoannou%26aufirst%3DC.%26aulast%3DMellor%26aufirst%3DG.%2BW.%26aulast%3DMorse%26aufirst%3DM.%2BA.%26aulast%3DPayne%26aufirst%3DJ.%2BJ.%26aulast%3DPritchard%26aufirst%3DJ.%2BM.%26aulast%3DSmith%26aufirst%3DK.%2BJ.%26aulast%3DTape%26aufirst%3DD.%2BT.%26aulast%3DWhitworth%26aufirst%3DC.%26aulast%3DWilliamson%26aufirst%3DR.%2BA.%26atitle%3D4-Phenyl-7-azaindoles%2520as%2520potent%2520and%2520selective%2520IKK2%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D2504%26epage%3D2508%26doi%3D10.1016%2Fj.bmcl.2009.03.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gehrig, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westhoff, J. H.</span></span> <span> </span><span class="NLM_article-title">Zebrafish as a model for drug screening in genetic kidney diseases</span>. <i>Frontiers in Pediatrics</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">183</span>, <span class="refDoi"> DOI: 10.3389/fped.2018.00183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.3389%2Ffped.2018.00183" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=30003073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252Fns12jtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=183&author=J.+Gehrigauthor=G.+Pandeyauthor=J.+H.+Westhoff&title=Zebrafish+as+a+model+for+drug+screening+in+genetic+kidney+diseases&doi=10.3389%2Ffped.2018.00183"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Zebrafish as a Model for Drug Screening in Genetic Kidney Diseases</span></div><div class="casAuthors">Gehrig Jochen; Pandey Gunjan; Pandey Gunjan; Westhoff Jens H</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in pediatrics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">183</span>
        ISSN:<span class="NLM_cas:issn">2296-2360</span>.
    </div><div class="casAbstract">Genetic disorders account for a wide range of renal diseases emerging during childhood and adolescence.  Due to the utilization of modern biochemical and biomedical techniques, the number of identified disease-associated genes is increasing rapidly.  Modeling of congenital human disease in animals is key to our understanding of the biological mechanism underlying pathological processes and thus developing novel potential treatment options.  The zebrafish (Danio rerio) has been established as a versatile small vertebrate organism that is widely used for studying human inherited diseases.  Genetic accessibility in combination with elegant experimental methods in zebrafish permit modeling of human genetic diseases and dissecting the perturbation of underlying cellular networks and physiological processes.  Beyond its utility for genetic analysis and pathophysiological and mechanistic studies, zebrafish embryos, and larvae are amenable for phenotypic screening approaches employing high-content and high-throughput experiments using automated microscopy.  This includes large-scale chemical screening experiments using genetic models for searching for disease-modulating compounds.  Phenotype-based approaches of drug discovery have been successfully performed in diverse zebrafish-based screening applications with various phenotypic readouts.  As a result, these can lead to the identification of candidate substances that are further examined in preclinical and clinical trials.  In this review, we discuss zebrafish models for inherited kidney disease as well as requirements and considerations for the technical realization of drug screening experiments in zebrafish.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTYV2FmQJ5uRVUIs6wFK5xJfW6udTcc2eZ2rkt7IVzOz7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252Fns12jtA%253D%253D&md5=7c9e5b3158f7042e9241f2a0887476d6</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.3389%2Ffped.2018.00183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffped.2018.00183%26sid%3Dliteratum%253Aachs%26aulast%3DGehrig%26aufirst%3DJ.%26aulast%3DPandey%26aufirst%3DG.%26aulast%3DWesthoff%26aufirst%3DJ.%2BH.%26atitle%3DZebrafish%2520as%2520a%2520model%2520for%2520drug%2520screening%2520in%2520genetic%2520kidney%2520diseases%26jtitle%3DFrontiers%2520in%2520Pediatrics%26date%3D2018%26volume%3D6%26spage%3D183%26doi%3D10.3389%2Ffped.2018.00183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span>; <span class="NLM_string-name">Barker, M. D.</span>; <span class="NLM_string-name">Campos, S. A.</span>; <span class="NLM_string-name">Cousins, R. P. C.</span>; <span class="NLM_string-name">Faulder, P.</span>; <span class="NLM_string-name">Hobbs, H.</span>; <span class="NLM_string-name">Holmes, D. S.</span>; <span class="NLM_string-name">Johnston, M. J.</span>; <span class="NLM_string-name">Liddle, J.</span>; <span class="NLM_string-name">Payne, J. J.</span>; <span class="NLM_string-name">Pritchard, J. M.</span>; <span class="NLM_string-name">Whitworth, C.</span></span> <span> </span><span class="NLM_article-title">Pyrrolo[2,3-b]pyridin-4-yl-benzenesulfonamide compounds as IKK2 inhibitors and their preparation, pharmaceutical compositions and use in the treatment of diseases</span>. <span class="NLM_patent">WO2008034860A1</span>, <span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=P.+Bamborough&author=M.+D.+Barker&author=S.+A.+Campos&author=R.+P.+C.+Cousins&author=P.+Faulder&author=H.+Hobbs&author=D.+S.+Holmes&author=M.+J.+Johnston&author=J.+Liddle&author=J.+J.+Payne&author=J.+M.+Pritchard&author=C.+Whitworth&title=Pyrrolo%5B2%2C3-b%5Dpyridin-4-yl-benzenesulfonamide+compounds+as+IKK2+inhibitors+and+their+preparation%2C+pharmaceutical+compositions+and+use+in+the+treatment+of+diseases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26atitle%3DPyrrolo%255B2%252C3-b%255Dpyridin-4-yl-benzenesulfonamide%2520compounds%2520as%2520IKK2%2520inhibitors%2520and%2520their%2520preparation%252C%2520pharmaceutical%2520compositions%2520and%2520use%2520in%2520the%2520treatment%2520of%2520diseases%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McKennon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyers, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drauz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarm, M.</span></span> <span> </span><span class="NLM_article-title">A convenient reduction of amino acids and their derivatives</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">3568</span>– <span class="NLM_lpage">3571</span>, <span class="refDoi"> DOI: 10.1021/jo00065a020</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00065a020" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADyaK3sXksV2rtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=1993&pages=3568-3571&author=M.+J.+McKennonauthor=A.+I.+Meyersauthor=K.+Drauzauthor=M.+Schwarm&title=A+convenient+reduction+of+amino+acids+and+their+derivatives&doi=10.1021%2Fjo00065a020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A convenient reduction of amino acids and their derivatives</span></div><div class="casAuthors">McKennon, Marc J.; Meyers, A. I.; Drauz, Karlheinz; Schwarm, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3568-71</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">Amino acids were reduced by NaBH4 in the presence of iodine to give the corresponding amino alcs.  Thus, amino acids I (R = CMe3, CHMe2, CH2Ph, CHEtMe, CH2CH2SMe, C6H4OH-4) were reduced by NaBH4-iodine in THF to give amino alcs. II.  N-acyl amino acids III [R = CH2Ph, R1 = H, R2 = H, Me; RR1 = (CH2)3 R2 = Me; R = R1 = H, R2 = Ph] were also reduced by NaBH4-iodine in THF to give N-acyl amino alcs. IV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnnqE3R204brVg90H21EOLACvtfcHk0lhZjxpaQvsvTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXksV2rtL8%253D&md5=d79a74af1c20e79ceee5c61ed977bccb</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjo00065a020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00065a020%26sid%3Dliteratum%253Aachs%26aulast%3DMcKennon%26aufirst%3DM.%2BJ.%26aulast%3DMeyers%26aufirst%3DA.%2BI.%26aulast%3DDrauz%26aufirst%3DK.%26aulast%3DSchwarm%26aufirst%3DM.%26atitle%3DA%2520convenient%2520reduction%2520of%2520amino%2520acids%2520and%2520their%2520derivatives%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1993%26volume%3D58%26spage%3D3568%26epage%3D3571%26doi%3D10.1021%2Fjo00065a020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chong, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dadwal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">Efficient synthesis of functionalized aziridinium salts</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">219</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1021/jo901893n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo901893n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsV2nurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2010&pages=219-221&author=H.-S.+Chongauthor=H.+A.+Songauthor=M.+Dadwalauthor=X.+Sunauthor=I.+Sinauthor=Y.+Chen&title=Efficient+synthesis+of+functionalized+aziridinium+salts&doi=10.1021%2Fjo901893n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient synthesis of functionalized aziridinium salts</span></div><div class="casAuthors">Chong, Hyun-Soon; Song, Hyun A.; Dadwal, Mamta; Sun, Xiang; Sin, Inseok; Chen, Yunwei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">219-221</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Various aziridinium salts were efficiently prepd. from bromination of a series of backbone substituted N,N-disubstituted β-amino alcs. and isolated via flash column chromatog.  The effect of C-substitution, N-substitution, solvent, leaving group, and counteranions on formation of the isolable aziridinium salts was investigated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBaN2nKqK0DrVg90H21EOLACvtfcHk0lhFK_9jbnnuJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsV2nurrJ&md5=a326087f310c15dc390be8baf09a0829</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjo901893n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo901893n%26sid%3Dliteratum%253Aachs%26aulast%3DChong%26aufirst%3DH.-S.%26aulast%3DSong%26aufirst%3DH.%2BA.%26aulast%3DDadwal%26aufirst%3DM.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DSin%26aufirst%3DI.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DEfficient%2520synthesis%2520of%2520functionalized%2520aziridinium%2520salts%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2010%26volume%3D75%26spage%3D219%26epage%3D221%26doi%3D10.1021%2Fjo901893n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Westwood, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheary, D.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baxter, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Montfort, R. L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span> <span> </span><span class="NLM_article-title">Insights into the conformational variability and regulation of human Nek2 kinase</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>386</i></span>,  <span class="NLM_fpage">476</span>– <span class="NLM_lpage">485</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2008.12.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.jmb.2008.12.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=19124027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhs1ars7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=386&publication_year=2009&pages=476-485&author=I.+Westwoodauthor=D.-M.+Chearyauthor=J.+E.+Baxterauthor=M.+W.+Richardsauthor=R.+L.+M.+van+Montfortauthor=A.+M.+Fryauthor=R.+Bayliss&title=Insights+into+the+conformational+variability+and+regulation+of+human+Nek2+kinase&doi=10.1016%2Fj.jmb.2008.12.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the Conformational Variability and Regulation of Human Nek2 Kinase</span></div><div class="casAuthors">Westwood, Isaac; Cheary, Donna-Marie; Baxter, Joanne E.; Richards, Mark W.; van Montfort, Rob L. M.; Fry, Andrew M.; Bayliss, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">386</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">476-485</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Nek family of serine/threonine kinases regulates centrosome and cilia function; in addn., several of its members are potential targets for drug discovery.  Nek2 is dimeric, is cell cycle regulated and functions in the sepn. of centrosomes at G2/M.  Here, we report the crystal structures of wild-type human Nek2 kinase domain bound to ADP at 1.55-Å resoln. and T175A mutant in apo form as well as that bound to a non-hydrolyzable ATP analog.  These show that regions of the Nek2 structure around the nucleotide-binding site can adopt several different but well-defined conformations.  None of the conformations was the same as that obsd. for the previously reported inhibitor-bound structure, and the two nucleotides stabilized two conformations.  The structures suggest mechanisms for the auto-inhibition of Nek2 that we have tested by mutagenesis.  Comparison of the structures with Aurora-A and Cdk2 gives insight into the structural mechanism of Nek2 activation.  The prodn. of specific inhibitors that target individual kinases of the human genome is an urgent challenge in drug discovery, and Nek2 is esp. promising as a cancer target.  We not only identify potential challenges to the task of producing Nek2 inhibitors but also propose that the conformational variability provides an opportunity for the design of Nek2 selective inhibitors because one of the conformations may provide a unique target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyfL7jAcF9TbVg90H21EOLACvtfcHk0ljYwdUen-_vIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhs1ars7o%253D&md5=e51d49845a03d605cdc6f7d2ba244fce</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2008.12.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2008.12.033%26sid%3Dliteratum%253Aachs%26aulast%3DWestwood%26aufirst%3DI.%26aulast%3DCheary%26aufirst%3DD.-M.%26aulast%3DBaxter%26aufirst%3DJ.%2BE.%26aulast%3DRichards%26aufirst%3DM.%2BW.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%2BL.%2BM.%26aulast%3DFry%26aufirst%3DA.%2BM.%26aulast%3DBayliss%26aufirst%3DR.%26atitle%3DInsights%2520into%2520the%2520conformational%2520variability%2520and%2520regulation%2520of%2520human%2520Nek2%2520kinase%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D386%26spage%3D476%26epage%3D485%26doi%3D10.1016%2Fj.jmb.2008.12.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haq, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallego, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeoh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Regan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reverter, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roig, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span> <span> </span><span class="NLM_article-title">Mechanistic basis of Nek7 activation through Nek9 binding and induced dimerization</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">8771</span>, <span class="refDoi"> DOI: 10.1038/ncomms9771</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1038%2Fncomms9771" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=26522158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslOgtbrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8771&author=T.+Haqauthor=M.+W.+Richardsauthor=S.+G.+Burgessauthor=P.+Gallegoauthor=S.+Yeohauthor=L.+O%E2%80%99Reganauthor=D.+Reverterauthor=J.+Roigauthor=A.+M.+Fryauthor=R.+Bayliss&title=Mechanistic+basis+of+Nek7+activation+through+Nek9+binding+and+induced+dimerization&doi=10.1038%2Fncomms9771"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic basis of Nek7 activation through Nek9 binding and induced dimerization</span></div><div class="casAuthors">Haq, Tamanna; Richards, Mark W.; Burgess, Selena G.; Gallego, Pablo; Yeoh, Sharon; O'Regan, Laura; Reverter, David; Roig, Joan; Fry, Andrew M.; Bayliss, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8771</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mitotic spindle assembly requires the regulated activities of protein kinases such as Nek7 and Nek9.  Nek7 is autoinhibited by the protrusion of Tyr97 into the active site and activated by the Nek9 non-catalytic C-terminal domain (CTD).  CTD binding apparently releases autoinhibition because mutation of Tyr97 to phenylalanine increases Nek7 activity independently of Nek9.  Here the authors find that self-assocn. of the Nek9-CTD is needed for Nek7 activation.  The authors map the minimal Nek7 binding region of Nek9 to residues 810-828.  A crystal structure of Nek7Y97F bound to Nek9810-828 reveals a binding site on the C-lobe of the Nek7 kinase domain.  Nek7Y97F crystallizes as a back-to-back dimer between kinase domain N-lobes, in which the specific contacts within the interface are coupled to the conformation of residue 97.  Hence, the authors propose that the Nek9-CTD activates Nek7 through promoting back-to-back dimerization that releases the autoinhibitory tyrosine residue, a mechanism conserved in unrelated kinase families.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7eBzbywmhnbVg90H21EOLACvtfcHk0ljYwdUen-_vIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslOgtbrP&md5=47f92bb0b477c6ee53cd7d7d46f954c8</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fncomms9771&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms9771%26sid%3Dliteratum%253Aachs%26aulast%3DHaq%26aufirst%3DT.%26aulast%3DRichards%26aufirst%3DM.%2BW.%26aulast%3DBurgess%26aufirst%3DS.%2BG.%26aulast%3DGallego%26aufirst%3DP.%26aulast%3DYeoh%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Regan%26aufirst%3DL.%26aulast%3DReverter%26aufirst%3DD.%26aulast%3DRoig%26aufirst%3DJ.%26aulast%3DFry%26aufirst%3DA.%2BM.%26aulast%3DBayliss%26aufirst%3DR.%26atitle%3DMechanistic%2520basis%2520of%2520Nek7%2520activation%2520through%2520Nek9%2520binding%2520and%2520induced%2520dimerization%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D8771%26doi%3D10.1038%2Fncomms9771" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawa, M.</span></span> <span> </span><span class="NLM_article-title">7-Azaindole: A versatile scaffold for developing kinase inhibitors</span>. <i>Chem. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">36</span>, <span class="refDoi"> DOI: 10.1248/cpb.c17-00380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1248%2Fcpb.c17-00380" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=29311509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslOntr%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2018&pages=29-36&author=T.+Irieauthor=M.+Sawa&title=7-Azaindole%3A+A+versatile+scaffold+for+developing+kinase+inhibitors&doi=10.1248%2Fcpb.c17-00380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">7-Azaindole: a versatile scaffold for developing kinase inhibitors</span></div><div class="casAuthors">Irie, Takayuki; Sawa, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-36</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">A review.  This review summarized recent examples of 7-azaindole-based kinase inhibitors and discusses their binding interactions with the kinase hinge regions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9QuvBOqQ8GrVg90H21EOLACvtfcHk0lhn_QjWzgjfVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslOntr%252FK&md5=69a5097cb54f2598d2e02414e74b3e6b</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1248%2Fcpb.c17-00380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.c17-00380%26sid%3Dliteratum%253Aachs%26aulast%3DIrie%26aufirst%3DT.%26aulast%3DSawa%26aufirst%3DM.%26atitle%3D7-Azaindole%253A%2520A%2520versatile%2520scaffold%2520for%2520developing%2520kinase%2520inhibitors%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D2018%26volume%3D66%26spage%3D29%26epage%3D36%26doi%3D10.1248%2Fcpb.c17-00380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Juchum, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günther, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Döring, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sievers-Engler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lämmerhofer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S.</span></span> <span> </span><span class="NLM_article-title">Trisubstituted imidazoles with a rigidized hinge binding motif act as single digit nM inhibitors of clinically relevant EGFR L858R/T790M and L858R/T790M/C797S mutants: An example of target hopping</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4636</span>– <span class="NLM_lpage">4656</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00178</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00178" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnt1Ciu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4636-4656&author=M.+Juchumauthor=M.+G%C3%BCntherauthor=E.+D%C3%B6ringauthor=A.+Sievers-Englerauthor=M.+L%C3%A4mmerhoferauthor=S.+Laufer&title=Trisubstituted+imidazoles+with+a+rigidized+hinge+binding+motif+act+as+single+digit+nM+inhibitors+of+clinically+relevant+EGFR+L858R%2FT790M+and+L858R%2FT790M%2FC797S+mutants%3A+An+example+of+target+hopping&doi=10.1021%2Facs.jmedchem.7b00178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Trisubstituted Imidazoles with a Rigidized Hinge Binding Motif Act As Single Digit nM Inhibitors of Clinically Relevant EGFR L858R/T790M and L858R/T790M/C797S Mutants: An Example of Target Hopping</span></div><div class="casAuthors">Juchum, Michael; Guenther, Marcel; Doering, Eva; Sievers-Engler, Adrian; Laemmerhofer, Michael; Laufer, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4636-4656</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Trisubstituted imidazoles with a rigidized 7-azaindole hinge binding motif such as I were prepd. as inhibitors of EGFR tyrosine kinase with L858R/T790M/C797S mutations (against which current EGFR inhibitors are ineffective) for potential use in the treatment of non-small-cell lung cancer.  II [X = (CH2)2, (CH2)4] reversibly inhibited the clin. challenging EGFR tyrosine kinase triple mutant L858R/T790M/C797S with IC50 values in the low nanomolar range.  I inhibited EGFR tyrosine kinase irreversibly with IC50 value of 1 nM against the EGFR L858R/T790M double mutant; I was selective for EGFR tyrosine kinase over other related kinases except for MAP4K4, MAP4K5, and MINK.  The kinetics of target inhibition, aq. soly, and metabolites were detd. for I and for selected compds.; mol. docking calcns. were also performed for selected compds. with EGFR isoforms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzeDYy9aSCrLVg90H21EOLACvtfcHk0liqrlJlujksIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnt1Ciu78%253D&md5=a7003c8160b506ce964edce6adf7c3ff</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00178%26sid%3Dliteratum%253Aachs%26aulast%3DJuchum%26aufirst%3DM.%26aulast%3DG%25C3%25BCnther%26aufirst%3DM.%26aulast%3DD%25C3%25B6ring%26aufirst%3DE.%26aulast%3DSievers-Engler%26aufirst%3DA.%26aulast%3DL%25C3%25A4mmerhofer%26aufirst%3DM.%26aulast%3DLaufer%26aufirst%3DS.%26atitle%3DTrisubstituted%2520imidazoles%2520with%2520a%2520rigidized%2520hinge%2520binding%2520motif%2520act%2520as%2520single%2520digit%2520nM%2520inhibitors%2520of%2520clinically%2520relevant%2520EGFR%2520L858R%252FT790M%2520and%2520L858R%252FT790M%252FC797S%2520mutants%253A%2520An%2520example%2520of%2520target%2520hopping%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4636%26epage%3D4656%26doi%3D10.1021%2Facs.jmedchem.7b00178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ismer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiemann, J. K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goede, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preissner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hildebrand, P. W.</span></span> <span> </span><span class="NLM_article-title">SL2: an interactive webtool for modeling of missing segments in proteins</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">W390</span>– <span class="NLM_lpage">W394</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw297</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1093%2Fnar%2Fgkw297" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=27105847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2itrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=W390-W394&author=J.+Ismerauthor=A.+S.+Roseauthor=J.+K.+S.+Tiemannauthor=A.+Goedeauthor=R.+Preissnerauthor=P.+W.+Hildebrand&title=SL2%3A+an+interactive+webtool+for+modeling+of+missing+segments+in+proteins&doi=10.1093%2Fnar%2Fgkw297"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">SL2: an interactive webtool for modeling of missing segments in proteins</span></div><div class="casAuthors">Ismer, Jochen; Rose, Alexander S.; Tiemann, Johanna K. S.; Goede, Andrean; Preissner, Robert; Hildebrand, Peter W.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W390-W394</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">SuperLooper2 (SL2) is the updated version of our previous webserver SuperLooper, a fragment based tool for the prediction and interactive placement of loop structures into globular and helical membrane proteins.  In comparison to our previous version, SL2 benefits from both a considerably enlarged database of fragments derived from high-resoln. 3D protein structures of globular and helical membrane proteins, and the integration of a new protein viewer.  The database, now with double the content, significantly improved the coverage of fragment conformations and prediction quality.  The employment of the NGL viewer for visualization of the protein under investigation and interactive selection of appropriate loops makes SL2 independent of third-party plug-ins and addnl. installations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWfroRDMUuZbVg90H21EOLACvtfcHk0liqrlJlujksIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2itrfE&md5=1df0214bfe56ba79794ef82751ff8630</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw297%26sid%3Dliteratum%253Aachs%26aulast%3DIsmer%26aufirst%3DJ.%26aulast%3DRose%26aufirst%3DA.%2BS.%26aulast%3DTiemann%26aufirst%3DJ.%2BK.%2BS.%26aulast%3DGoede%26aufirst%3DA.%26aulast%3DPreissner%26aufirst%3DR.%26aulast%3DHildebrand%26aufirst%3DP.%2BW.%26atitle%3DSL2%253A%2520an%2520interactive%2520webtool%2520for%2520modeling%2520of%2520missing%2520segments%2520in%2520proteins%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DW390%26epage%3DW394%26doi%3D10.1093%2Fnar%2Fgkw297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coxon, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxall, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardcastle, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matheson, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newell, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivaprakasam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeoh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golding, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cano, C.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design of purine-based probes for selective Nek2 inhibition</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">19089</span>– <span class="NLM_lpage">19124</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.13249</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.18632%2Foncotarget.13249" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=27833088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A280%3ADC%252BC2snkvF2jsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=19089-19124&author=C.+R.+Coxonauthor=C.+Wongauthor=R.+Baylissauthor=K.+Boxallauthor=K.+H.+Carrauthor=A.+M.+Fryauthor=I.+R.+Hardcastleauthor=C.+J.+Mathesonauthor=D.+R.+Newellauthor=M.+Sivaprakasamauthor=H.+Thomasauthor=D.+Turnerauthor=S.+Yeohauthor=L.+Z.+Wangauthor=R.+J.+Griffinauthor=B.+T.+Goldingauthor=C.+Cano&title=Structure-guided+design+of+purine-based+probes+for+selective+Nek2+inhibition&doi=10.18632%2Foncotarget.13249"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-guided design of purine-based probes for selective Nek2 inhibition</span></div><div class="casAuthors">Coxon Christopher R; Wong Christopher; Hardcastle Ian R; Matheson Christopher J; Sivaprakasam Mangaleswaran; Turner David; Griffin Roger J; Golding Bernard T; Cano Celine; Bayliss Richard; Carr Katherine H; Fry Andrew M; Yeoh Sharon; Boxall Kathy; Newell David R; Thomas Huw; Wang Lan Z</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">19089-19124</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Nek2 (NIMA-related kinase 2) is a cell cycle-dependent serine/threonine protein kinase that regulates centrosome separation at the onset of mitosis.  Overexpression of Nek2 is common in human cancers and suppression can restrict tumor cell growth and promote apoptosis.  Nek2 inhibition with small molecules, therefore, offers the prospect of a new therapy for cancer.  To achieve this goal, a better understanding of the requirements for selective-inhibition of Nek2 is required. 6-Alkoxypurines were identified as ATP-competitive inhibitors of Nek2 and CDK2.  Comparison with CDK2-inhibitor structures indicated that judicious modification of the 6-alkoxy and 2-arylamino substituents could achieve discrimination between Nek2 and CDK2.  In this study, a library of 6-cyclohexylmethoxy-2-arylaminopurines bearing carboxamide, sulfonamide and urea substituents on the 2-arylamino ring was synthesized.  Few of these compounds were selective for Nek2 over CDK2, with the best result being obtained for 3-((6-(cyclohexylmethoxy)-9H-purin-2-yl)amino)-N,N-dimethylbenzamide (CDK2 IC50 = 7.0 μM; Nek2 IC50 = 0.62 μM) with >10-fold selectivity.  Deletion of the 6-substituent abrogated activity against both Nek2 and CDK2.  Nine compounds containing an (E)-dialkylaminovinyl substituent at C-6, all showed selectivity for Nek2, e.g. (E)-6-(2-(azepan-1-yl)vinyl)-N-phenyl-9H-purin-2-amine (CDK2 IC50 = 2.70 μM; Nek2 IC50 = 0.27 μM).  Structural biology of selected compounds enabled a partial rationalization of the observed structure activity relationships and mechanism of Nek2 activation.  This showed that carboxamide 11 is the first reported inhibitor of Nek2 in the DFG-in conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQRxhuJ4vkuqOLt9HypzAgQfW6udTcc2eZLhM66b2sqx7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2snkvF2jsw%253D%253D&md5=b8d7777e4bfb6c375d7ad122a9304e2a</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.13249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.13249%26sid%3Dliteratum%253Aachs%26aulast%3DCoxon%26aufirst%3DC.%2BR.%26aulast%3DWong%26aufirst%3DC.%26aulast%3DBayliss%26aufirst%3DR.%26aulast%3DBoxall%26aufirst%3DK.%26aulast%3DCarr%26aufirst%3DK.%2BH.%26aulast%3DFry%26aufirst%3DA.%2BM.%26aulast%3DHardcastle%26aufirst%3DI.%2BR.%26aulast%3DMatheson%26aufirst%3DC.%2BJ.%26aulast%3DNewell%26aufirst%3DD.%2BR.%26aulast%3DSivaprakasam%26aufirst%3DM.%26aulast%3DThomas%26aufirst%3DH.%26aulast%3DTurner%26aufirst%3DD.%26aulast%3DYeoh%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DL.%2BZ.%26aulast%3DGriffin%26aufirst%3DR.%2BJ.%26aulast%3DGolding%26aufirst%3DB.%2BT.%26aulast%3DCano%26aufirst%3DC.%26atitle%3DStructure-guided%2520design%2520of%2520purine-based%2520probes%2520for%2520selective%2520Nek2%2520inhibition%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D19089%26epage%3D19124%26doi%3D10.18632%2Foncotarget.13249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.</span></span> <span> </span><span class="NLM_article-title">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.phrs.2015.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=26529477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2016&pages=26-48&author=R.+Roskoski&title=Classification+of+small+molecule+protein+kinase+inhibitors+based+upon+the+structures+of+their+drug-enzyme+complexes&doi=10.1016%2Fj.phrs.2015.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span></div><div class="casAuthors">Roskoski, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26-48</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Because dysregulation and mutations of protein kinases play causal roles in human disease, this family of enzymes has become one of the most important drug targets over the past two decades.  The X-ray crystal structures of 21 of the 27 FDA-approved small mol. inhibitors bound to their target protein kinases are depicted in this paper.  The structure of the enzyme-bound antagonist complex is used in the classification of these inhibitors.  Type I inhibitors bind to the active protein kinase conformation (DFG-Asp in, αC-helix in).  Type I1/2 inhibitors bind to a DFG-Asp in inactive conformation while Type II inhibitors bind to a DFG-Asp out inactive conformation.  Type I, I1/2, and type II inhibitors occupy part of the adenine binding pocket and form hydrogen bonds with the hinge region connecting the small and large lobes of the enzyme.  Type III inhibitors bind next to the ATP-binding pocket and type IV inhibitors do not bind to the ATP or peptide substrate binding sites.  Type III and IV inhibitors are allosteric in nature.  Type V inhibitors bind to two different regions of the protein kinase domain and are therefore bivalent inhibitors.  The type I-V inhibitors are reversible.  In contrast, type VI inhibitors bind covalently to their target enzyme.  Type I, I1/2, and II inhibitors are divided into A and B subtypes.  The type A inhibitors bind in the front cleft, the back cleft, and near the gatekeeper residue, all of which occur within the region sepg. the small and large lobes of the protein kinase.  The type B inhibitors bind in the front cleft and gate area but do not extend into the back cleft.  An anal. of the limited available data indicates that type A inhibitors have a long residence time (minutes to hours) while the type B inhibitors have a short residence time (seconds to minutes).  The catalytic spine includes residues from the small and large lobes and interacts with the adenine ring of ATP.  Nearly all of the approved protein kinase inhibitors occupy the adenine-binding pocket; thus it is not surprising that these inhibitors interact with nearby catalytic spine (CS) residues.  Moreover, a significant no. of approved drugs also interact with regulatory spine (RS) residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH0COwqC0CabVg90H21EOLACvtfcHk0lgZJAFp7rk2sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN&md5=e6b45c8644e322f30f898e763780c64b</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DClassification%2520of%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%2520based%2520upon%2520the%2520structures%2520of%2520their%2520drug-enzyme%2520complexes%26jtitle%3DPharmacol.%2520Res.%26date%3D2016%26volume%3D103%26spage%3D26%26epage%3D48%26doi%3D10.1016%2Fj.phrs.2015.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smyth, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span> <span> </span><span class="NLM_article-title">Measuring and interpreting the selectivity of protein kinase inhibitors</span>. <i>Journal of Chemical Biology</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">151</span>, <span class="refDoi"> DOI: 10.1007/s12154-009-0023-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1007%2Fs12154-009-0023-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=19568781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A280%3ADC%252BD1MrkslGqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2009&pages=131-151&author=L.+A.+Smythauthor=I.+Collins&title=Measuring+and+interpreting+the+selectivity+of+protein+kinase+inhibitors&doi=10.1007%2Fs12154-009-0023-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Measuring and interpreting the selectivity of protein kinase inhibitors</span></div><div class="casAuthors">Smyth Lynette A; Collins Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">131-51</span>
        ISSN:<span class="NLM_cas:issn">1864-6158</span>.
    </div><div class="casAbstract">Protein kinase inhibitors are a well-established class of clinically useful drugs, particularly for the treatment of cancer.  Achieving inhibitor selectivity for particular protein kinases often remains a significant challenge in the development of new small molecules as drugs or as tools for chemical biology research.  This review summarises the methodologies available for measuring kinase inhibitor selectivity, both in vitro and in cells.  The interpretation of kinase inhibitor selectivity data is discussed, particularly with reference to the structural biology of the protein targets.  Measurement and prediction of kinase inhibitor selectivity will be important for the development of new multi-targeted kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQzU9rA5WZcz9FxYDaSSP8XfW6udTcc2ebFtxs642DziLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MrkslGqtA%253D%253D&md5=f78ff1202a949a71ffd65e804298c37f</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1007%2Fs12154-009-0023-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12154-009-0023-9%26sid%3Dliteratum%253Aachs%26aulast%3DSmyth%26aufirst%3DL.%2BA.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMeasuring%2520and%2520interpreting%2520the%2520selectivity%2520of%2520protein%2520kinase%2520inhibitors%26jtitle%3DJournal%2520of%2520Chemical%2520Biology%26date%3D2009%26volume%3D2%26spage%3D131%26epage%3D151%26doi%3D10.1007%2Fs12154-009-0023-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Outtandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleta, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bockenhauer, D.</span></span> <span> </span><span class="NLM_article-title">Zebrafish as a model for kidney function and disease</span>. <i>Pediatr. Nephrol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">751</span>– <span class="NLM_lpage">762</span>, <span class="refDoi"> DOI: 10.1007/s00467-018-3921-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1007%2Fs00467-018-3921-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=29502161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A280%3ADC%252BC1MrnsVCisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2019&pages=751-762&author=P.+Outtandyauthor=C.+Russellauthor=R.+Kletaauthor=D.+Bockenhauer&title=Zebrafish+as+a+model+for+kidney+function+and+disease&doi=10.1007%2Fs00467-018-3921-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Zebrafish as a model for kidney function and disease</span></div><div class="casAuthors">Outtandy Priya; Kleta Robert; Bockenhauer Detlef; Outtandy Priya; Russell Claire</div><div class="citationInfo"><span class="NLM_cas:title">Pediatric nephrology (Berlin, Germany)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">751-762</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Kidney disease is a global problem with around three million people diagnosed in the UK alone and the incidence is rising.  Research is critical to develop better treatments.  Animal models can help to better understand the pathophysiology behind the various kidney diseases and to screen for therapeutic compounds, but the use especially of mammalian models should be minimised in the interest of animal welfare.  Zebrafish are increasingly used, as they are genetically tractable and have a basic renal anatomy comparable to mammalian kidneys with glomerular filtration and tubular filtration processing.  Here, we discuss how zebrafish have advanced the study of nephrology and the mechanisms underlying kidney disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQyE-WYqwvzUrt7lRqEFUJVfW6udTcc2eaQ14xOcwrck7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MrnsVCisw%253D%253D&md5=bea6e7eaef05d9690f90d0bb38a0b36d</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1007%2Fs00467-018-3921-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00467-018-3921-7%26sid%3Dliteratum%253Aachs%26aulast%3DOuttandy%26aufirst%3DP.%26aulast%3DRussell%26aufirst%3DC.%26aulast%3DKleta%26aufirst%3DR.%26aulast%3DBockenhauer%26aufirst%3DD.%26atitle%3DZebrafish%2520as%2520a%2520model%2520for%2520kidney%2520function%2520and%2520disease%26jtitle%3DPediatr.%2520Nephrol.%26date%3D2019%26volume%3D34%26spage%3D751%26epage%3D762%26doi%3D10.1007%2Fs00467-018-3921-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wlodarchak, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tariq, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Striker, R.</span></span> <span> </span><span class="NLM_article-title">Comparative analysis of the human and zebrafish kinomes: focus on the development of kinase inhibitors</span>. <i>Trends Cell Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">75</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=27011661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A280%3ADC%252BC2srovVGktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=49-75&author=N.+Wlodarchakauthor=R.+Tariqauthor=R.+Striker&title=Comparative+analysis+of+the+human+and+zebrafish+kinomes%3A+focus+on+the+development+of+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative analysis of the human and zebrafish kinomes: focus on the development of kinase inhibitors</span></div><div class="casAuthors">Wlodarchak Nathan; Tariq Rehan; Striker Rob</div><div class="citationInfo"><span class="NLM_cas:title">Trends in cell & molecular biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">49-75</span>
        ISSN:<span class="NLM_cas:issn">0972-8449</span>.
    </div><div class="casAbstract">Targeting kinases with semi-selective kinase inhibitors is one of the most successful drug development strategies of the 21(st) century.  Zebrafish have become an increasingly useful model for pharmaceutical development.  Water-soluble compounds can be screened for zebrafish phenotypes in a high throughput format against a living vertebrate, and cell-signaling events can be imaged in transparent living fish.  Despite zebrafish being a more relevant model than more distantly related systems such as the well-annotated kinome of yeast and drosophila, there is no comparative analysis of the human and zebrafish kinome.  Furthermore most approved kinase inhibitors, often called 'DFG in' ATP competitive inhibitors, act on conserved active site residues in the kinase.  Since the active site residues can be identified by examining the primary sequence, primary sequence identity can be a rough guide as to whether a particular inhibitor will have activity against another kinase.  There is a need to evaluate the utility of zebrafish as a drug development model for active site inhibitors of kinases.  Here we offer a systematic comparison of the catalytic domains of classical human kinases with the catalytic domains of all annotated zebrafish kinases.  We found a high degree of identity between the catalytic domains of most human kinases and their zebrafish homologs, and we ranked 504 human kinase catalytic domains by order of similarity.  We found only 23 human kinases with no easily recognizable homologous zebrafish catalytic domain.  On the other hand we found 78 zebrafish kinase catalytic domains with no close human counterpart.  These 'additional kinase active sites' could represent potential mediators of zebrafish toxicity that may not be relevant to human kinase inhibitors.  We used two clinically approved human kinase inhibitors, one targeting a highly homologous target and one targeting a lesser homologous target, and we compared the known human kinase target structures with modeled zebrafish target structures.  As expected, the homologous target had high structural identity, but even the less homologous target had high structural identity in residues contacted by the inhibitor.  Overall this analysis should help guide researchers interested in studying human kinases and their inhibitors in more tractable systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRJjr-hcquXE8ve_zCu79I8fW6udTcc2ea4YT5cZQtjlbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srovVGktw%253D%253D&md5=bae46c964ee9db2e7e185b38310e4000</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWlodarchak%26aufirst%3DN.%26aulast%3DTariq%26aufirst%3DR.%26aulast%3DStriker%26aufirst%3DR.%26atitle%3DComparative%2520analysis%2520of%2520the%2520human%2520and%2520zebrafish%2520kinomes%253A%2520focus%2520on%2520the%2520development%2520of%2520kinase%2520inhibitors%26jtitle%3DTrends%2520Cell%2520Mol.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D49%26epage%3D75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knight, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">Chemical Genetics: Where Genetics and Pharmacology Meet</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">425</span>– <span class="NLM_lpage">430</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.01.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.cell.2007.01.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=17289560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit1Wlt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=425-430&author=Z.+A.+Knightauthor=K.+M.+Shokat&title=Chemical+Genetics%3A+Where+Genetics+and+Pharmacology+Meet&doi=10.1016%2Fj.cell.2007.01.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical genetics: where genetics and pharmacology meet</span></div><div class="casAuthors">Knight, Zachary A.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">425-430</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Genetics and pharmacol. can elicit surprisingly different phenotypes despite targeting the same protein.  This Essay explores these unexpected differences and their implications for biol. and medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq056BEgxFUHrVg90H21EOLACvtfcHk0lgDCBEQph_Dxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit1Wlt74%253D&md5=8b5d8733dc3b3f699907709cd0ba6f62</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.01.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.01.021%26sid%3Dliteratum%253Aachs%26aulast%3DKnight%26aufirst%3DZ.%2BA.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DChemical%2520Genetics%253A%2520Where%2520Genetics%2520and%2520Pharmacology%2520Meet%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D425%26epage%3D430%26doi%3D10.1016%2Fj.cell.2007.01.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shalom, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shalva, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altschuler, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motro, B.</span></span> <span> </span><span class="NLM_article-title">The mammalian Nek1 kinase is involved in primary cilium formation</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>582</i></span>,  <span class="NLM_fpage">1465</span>– <span class="NLM_lpage">1470</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2008.03.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.febslet.2008.03.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=18387364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvVOiurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=582&publication_year=2008&pages=1465-1470&author=O.+Shalomauthor=N.+Shalvaauthor=Y.+Altschulerauthor=B.+Motro&title=The+mammalian+Nek1+kinase+is+involved+in+primary+cilium+formation&doi=10.1016%2Fj.febslet.2008.03.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The mammalian Nek1 kinase is involved in primary cilium formation</span></div><div class="casAuthors">Shalom, Ohad; Shalva, Nechama; Altschuler, Yoram; Motro, Benny</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">582</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1465-1470</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Recent studies implicate primary cilium (PC) proteins in the etiologies of various polycystic kidney diseases (PKD).  NIMA-related kinases (NRKs) are conserved serine/threonine kinases, which are usually defined as 'mitotic kinases'.  Murine mutants for the NRKs, nek1 (kat mice) suffer from PKD, suggesting that it may be involved in cilium control.  We demonstrated herein that Nek1 is localized to basal body region and that Nek1 overexpression inhibits ciliogenesis in Madin-Darby canine kidney epithelial cells.  The no. of primary cilia is dramatically reduced in kat2J mouse embryonic fibroblasts culture.  It is thus hypothesized that Nek1 links cell cycle progression and the PC cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxogIIq-Yl7rVg90H21EOLACvtfcHk0lgDCBEQph_Dxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvVOiurc%253D&md5=af63d67e0595f71b959f8d0353efc403</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2008.03.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2008.03.036%26sid%3Dliteratum%253Aachs%26aulast%3DShalom%26aufirst%3DO.%26aulast%3DShalva%26aufirst%3DN.%26aulast%3DAltschuler%26aufirst%3DY.%26aulast%3DMotro%26aufirst%3DB.%26atitle%3DThe%2520mammalian%2520Nek1%2520kinase%2520is%2520involved%2520in%2520primary%2520cilium%2520formation%26jtitle%3DFEBS%2520Lett.%26date%3D2008%26volume%3D582%26spage%3D1465%26epage%3D1470%26doi%3D10.1016%2Fj.febslet.2008.03.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Westhoff, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giselbrecht, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidts, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schindler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beales, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tönshoff, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liebel, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehrig, J.</span></span> <span> </span><span class="NLM_article-title">Development of an automated imaging pipeline for the analysis of the zebrafish larval kidney</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e82137</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0082137</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1371%2Fjournal.pone.0082137" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=24324758" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFeku70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=J.+H.+Westhoffauthor=S.+Giselbrechtauthor=M.+Schmidtsauthor=S.+Schindlerauthor=P.+L.+Bealesauthor=B.+T%C3%B6nshoffauthor=U.+Liebelauthor=J.+Gehrig&title=Development+of+an+automated+imaging+pipeline+for+the+analysis+of+the+zebrafish+larval+kidney&doi=10.1371%2Fjournal.pone.0082137"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Development of an automated imaging pipeline for the analysis of the zebrafish larval kidney</span></div><div class="casAuthors">Westhoff, Jens H.; Giselbrecht, Stefan; Schmidts, Miriam; Schindler, Sebastian; Beales, Philip L.; Toenshoff, Burkhard; Liebel, Urban; Gehrig, Jochen</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e82137/1-e82137/13, 13 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The anal. of kidney malformation caused by environmental influences during nephrogenesis or by hereditary nephropathies requires animal models allowing the in vivo observation of developmental processes.  The zebrafish has emerged as a useful model system for the anal. of vertebrate organ development and function, and it is suitable for the identification of organotoxic or disease-modulating compds. on a larger scale.  However, to fully exploit its potential in high content screening applications, dedicated protocols are required allowing the consistent visualization of inner organs such as the embryonic kidney.  To this end, we developed a high content screening compatible pipeline for the automated imaging of standardized views of the developing pronephros in zebrafish larvae.  Using a custom designed tool, cavities were generated in agarose coated microtiter plates allowing for accurate positioning and orientation of zebrafish larvae.  This enabled the subsequent automated acquisition of stable and consistent dorsal views of pronephric kidneys.  The established pipeline was applied in a pilot screen for the anal. of the impact of potentially nephrotoxic drugs on zebrafish pronephros development in the Tg(wt1b:EGFP) transgenic line in which the developing pronephros is highlighted by GFP expression.  The consistent image data that was acquired allowed for quantification of gross morphol. pronephric phenotypes, revealing concn. dependent effects of several compds. on nephrogenesis.  In addn., applicability of the imaging pipeline was further confirmed in a morpholino based model for cilia-assocd. human genetic disorders assocd. with different intraflagellar transport genes.  The developed tools and pipeline can be used to study various aspects in zebrafish kidney research, and can be readily adapted for the anal. of other organ systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrY3u0Z4CTYw7Vg90H21EOLACvtfcHk0lgmQEEDomC1XQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFeku70%253D&md5=6b48f06d579844ff51bc296eb45171b5</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0082137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0082137%26sid%3Dliteratum%253Aachs%26aulast%3DWesthoff%26aufirst%3DJ.%2BH.%26aulast%3DGiselbrecht%26aufirst%3DS.%26aulast%3DSchmidts%26aufirst%3DM.%26aulast%3DSchindler%26aufirst%3DS.%26aulast%3DBeales%26aufirst%3DP.%2BL.%26aulast%3DT%25C3%25B6nshoff%26aufirst%3DB.%26aulast%3DLiebel%26aufirst%3DU.%26aulast%3DGehrig%26aufirst%3DJ.%26atitle%3DDevelopment%2520of%2520an%2520automated%2520imaging%2520pipeline%2520for%2520the%2520analysis%2520of%2520the%2520zebrafish%2520larval%2520kidney%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0082137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sander, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teoh, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, A. J.</span></span> <span> </span><span class="NLM_article-title">The small molecule probe PT-Yellow labels the renal proximal tubules in zebrafish</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">395</span>– <span class="NLM_lpage">398</span>, <span class="refDoi"> DOI: 10.1039/C4CC08075K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1039%2FC4CC08075K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=25407666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFSmu7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2015&pages=395-398&author=V.+Sanderauthor=S.+Patkeauthor=S.+Sahuauthor=C.+L.+Teohauthor=Z.+Pengauthor=Y.-T.+Changauthor=A.+J.+Davidson&title=The+small+molecule+probe+PT-Yellow+labels+the+renal+proximal+tubules+in+zebrafish&doi=10.1039%2FC4CC08075K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">The small molecule probe PT-Yellow labels the renal proximal tubules in zebrafish</span></div><div class="casAuthors">Sander, Veronika; Patke, Shantanu; Sahu, Srikanta; Teoh, Chai Lean; Peng, Zhenzhen; Chang, Young-Tae; Davidson, Alan J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">395-398</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">We report the development of a small fluorescent mol., BDNCA3-D2, herein referred to as PT-Yellow.  Soaking zebrafish embryos in PT-Yellow or i.p. injection into adults results in non-toxic in vivo fluorescent labeling of the renal proximal tubules, the major site of blood filtrate reabsorption and a common target of injury in acute kidney injury.  We demonstrate the applicability of this new compd. as a rapid and simple readout for zebrafish kidney filtration and proximal tubule reabsorption function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobqD-zCxFwrrVg90H21EOLACvtfcHk0lgmQEEDomC1XQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFSmu7%252FI&md5=623b714263266da461d427e66195018b</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1039%2FC4CC08075K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4CC08075K%26sid%3Dliteratum%253Aachs%26aulast%3DSander%26aufirst%3DV.%26aulast%3DPatke%26aufirst%3DS.%26aulast%3DSahu%26aufirst%3DS.%26aulast%3DTeoh%26aufirst%3DC.%2BL.%26aulast%3DPeng%26aufirst%3DZ.%26aulast%3DChang%26aufirst%3DY.-T.%26aulast%3DDavidson%26aufirst%3DA.%2BJ.%26atitle%3DThe%2520small%2520molecule%2520probe%2520PT-Yellow%2520labels%2520the%2520renal%2520proximal%2520tubules%2520in%2520zebrafish%26jtitle%3DChem.%2520Commun.%26date%3D2015%26volume%3D51%26spage%3D395%26epage%3D398%26doi%3D10.1039%2FC4CC08075K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Christou-Savina, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beales, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborn, D. P. S.</span></span> <span> </span><span class="NLM_article-title">Evaluation of Zebrafish Kidney Function Using a Fluorescent Clearance Assay</span>. <i>JoVE (Journal of Visualized Experiments)</i> <span class="NLM_year" style="font-weight: bold;">2015</span>, <span class="NLM_elocation-id">e52540</span> <span class="refDoi"> DOI: 10.3791/52540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.3791%2F52540" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=S.+Christou-Savinaauthor=P.+L.+Bealesauthor=D.+P.+S.+Osborn&title=Evaluation+of+Zebrafish+Kidney+Function+Using+a+Fluorescent+Clearance+Assay&doi=10.3791%2F52540"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.3791%2F52540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3791%252F52540%26sid%3Dliteratum%253Aachs%26aulast%3DChristou-Savina%26aufirst%3DS.%26aulast%3DBeales%26aufirst%3DP.%2BL.%26aulast%3DOsborn%26aufirst%3DD.%2BP.%2BS.%26atitle%3DEvaluation%2520of%2520Zebrafish%2520Kidney%2520Function%2520Using%2520a%2520Fluorescent%2520Clearance%2520Assay%26jtitle%3DJoVE%2520%2528Journal%2520of%2520Visualized%2520Experiments%2529%26date%3D2015%26doi%3D10.3791%2F52540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. H.</span></span> <span> </span><span class="NLM_article-title">Effects of specific genes activating RAGE on polycystic kidney disease</span>. <i>Am. J. Nephrol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">178</span>, <span class="refDoi"> DOI: 10.1159/000315859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1159%2F000315859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=20606421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVaqsbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2010&pages=169-178&author=E.+Y.+Parkauthor=M.+J.+Seoauthor=J.+H.+Park&title=Effects+of+specific+genes+activating+RAGE+on+polycystic+kidney+disease&doi=10.1159%2F000315859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of specific genes activating RAGE on polycystic kidney disease</span></div><div class="casAuthors">Park, Eun Young; Seo, Min Ji; Park, Jong Hoon</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Nephrology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">169-178</span>CODEN:
                <span class="NLM_cas:coden">AJNED9</span>;
        ISSN:<span class="NLM_cas:issn">0250-8095</span>.
    
            (<span class="NLM_cas:orgname">S. Karger AG</span>)
        </div><div class="casAbstract">Background: Autosomal-dominant polycystic kidney disease (ADPKD) is characterized by progressive cyst formation and secretion of fluid and is assocd. with interstitial inflammation and fibrosis, resulting in the loss of renal function.  We previously generated mice overexpressing PKD2, causing progressive cyst development with an inflammatory and fibrotic phenotype in the kidneys.  Methods: To profile the gene expression related to inflammation and cystogenesis, microarray anal. was performed with kidney tissue from 6-, 12- and 18-mo-old mice.  Subsequently, levels and related mechanisms of selected genes, s100a8 and s100a9, were evaluated.  Results: S100a8 and s100a9 was upregulated more than 2-fold and differently expressed in the cystic region.  Receptor of advanced glycation end product (RAGE) is a putative cell surface receptor for s100a8/a9.  It was expressed in cyst-lining cells and up-regulated pro-inflammatory transcription factor NF-κB in transgenic mice.  We also confirmed RAGE expression in ADPKD patient kidneys.  It was suggested that the signaling related to proliferative cystogenesis through previous reports; therefore, we confirmed that phosphorylated-ERK and cyst formation was reduced by treatment of RAGE-siRNA.  Conclusions: The results may provide important information for the expression of s100a8/a9 and RAGE, linking progressive cystogenesis with inflammation in cystic kidney.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbd5fZHIZl77Vg90H21EOLACvtfcHk0lhz9YhxYkVe3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVaqsbjL&md5=5c4547473bdff7297face14b319421a7</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1159%2F000315859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1159%252F000315859%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DE.%2BY.%26aulast%3DSeo%26aufirst%3DM.%2BJ.%26aulast%3DPark%26aufirst%3DJ.%2BH.%26atitle%3DEffects%2520of%2520specific%2520genes%2520activating%2520RAGE%2520on%2520polycystic%2520kidney%2520disease%26jtitle%3DAm.%2520J.%2520Nephrol.%26date%3D2010%26volume%3D32%26spage%3D169%26epage%3D178%26doi%3D10.1159%2F000315859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Booij, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bange, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonhard, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fokkelman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunnen, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dauwerse, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van de Water, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Westen, G. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, D. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, L. S.</span></span> <span> </span><span class="NLM_article-title">High-throughput phenotypic screening of kinase inhibitors to identify drug targets for polycystic kidney disease</span>. <i>SLAS Discov</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">974</span>– <span class="NLM_lpage">984</span>, <span class="refDoi"> DOI: 10.1177/2472555217716056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1177%2F2472555217716056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=28644734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKku7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=974-984&author=T.+H.+Booijauthor=H.+Bangeauthor=W.+N.+Leonhardauthor=K.+Yanauthor=M.+Fokkelmanauthor=S.+J.+Kunnenauthor=J.+G.+Dauwerseauthor=Y.+Qinauthor=B.+van+de+Waterauthor=G.+J.+P.+van+Westenauthor=D.+J.+M.+Petersauthor=L.+S.+Price&title=High-throughput+phenotypic+screening+of+kinase+inhibitors+to+identify+drug+targets+for+polycystic+kidney+disease&doi=10.1177%2F2472555217716056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Phenotypic Screening of Kinase Inhibitors to Identify Drug Targets for Polycystic Kidney Disease</span></div><div class="casAuthors">Booij, Tijmen H.; Bange, Hester; Leonhard, Wouter N.; Yan, Kuan; Fokkelman, Michiel; Kunnen, Steven J.; Dauwerse, Johannes G.; Qin, Yu; van de Water, Bob; van Westen, Gerard J. P.; Peters, Dorien J. M.; Price, Leo S.</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">974-984</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Polycystic kidney disease (PKD) is a prevalent disorder characterized by renal cysts that lead to kidney failure.  Various signaling pathways have been targeted to stop disease progression, but most interventions still focus on alleviating PKD-assocd. symptoms.  The mechanistic complexity of the disease, as well as the lack of functional in vitro assays for compd. testing, has made drug discovery for PKD challenging.  To identify modulators of PKD, Pkd1-/- kidney tubule epithelial cells were applied to a scalable and automated 3D cyst culture model for compd. screening, followed by phenotypic profiling to det. compd. efficacy.  We used this screening platform to screen a library of 273 kinase inhibitors to probe various signaling pathways involved in cyst growth.  We show that inhibition of several targets, including aurora kinase, CDK, Chk, IGF-1R, Syk, and mTOR, but, surprisingly, not PI3K, prevented forskolin-induced cyst swelling.  Addnl., we show that multiparametric phenotypic classification discriminated potentially undesirable (i.e., cytotoxic) compds. from mols. inducing the desired phenotypic change, greatly facilitating hit selection and validation.  Our findings show that a pathophysiol. relevant 3D cyst culture model of PKD coupled to phenotypic profiling can be used to identify potentially therapeutic compds. and predict and validate mol. targets for PKD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5seGssaoSALVg90H21EOLACvtfcHk0lhz9YhxYkVe3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFKku7Y%253D&md5=dcd2e6d399d403c033b8bc10c26aa823</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1177%2F2472555217716056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555217716056%26sid%3Dliteratum%253Aachs%26aulast%3DBooij%26aufirst%3DT.%2BH.%26aulast%3DBange%26aufirst%3DH.%26aulast%3DLeonhard%26aufirst%3DW.%2BN.%26aulast%3DYan%26aufirst%3DK.%26aulast%3DFokkelman%26aufirst%3DM.%26aulast%3DKunnen%26aufirst%3DS.%2BJ.%26aulast%3DDauwerse%26aufirst%3DJ.%2BG.%26aulast%3DQin%26aufirst%3DY.%26aulast%3Dvan%2Bde%2BWater%26aufirst%3DB.%26aulast%3Dvan%2BWesten%26aufirst%3DG.%2BJ.%2BP.%26aulast%3DPeters%26aufirst%3DD.%2BJ.%2BM.%26aulast%3DPrice%26aufirst%3DL.%2BS.%26atitle%3DHigh-throughput%2520phenotypic%2520screening%2520of%2520kinase%2520inhibitors%2520to%2520identify%2520drug%2520targets%2520for%2520polycystic%2520kidney%2520disease%26jtitle%3DSLAS%2520Discov%26date%3D2017%26volume%3D22%26spage%3D974%26epage%3D984%26doi%3D10.1177%2F2472555217716056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Correa, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsui, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tergaonkar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Esteban, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izpisua-Belmonte, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, I. M.</span></span> <span> </span><span class="NLM_article-title">Zebrafish IκB Kinase 1 Negatively Regulates NF-κB Activity</span>. <i>Curr. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1291</span>– <span class="NLM_lpage">1295</span>, <span class="refDoi"> DOI: 10.1016/j.cub.2005.06.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.cub.2005.06.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=16051172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvVKgsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2005&pages=1291-1295&author=R.+G.+Correaauthor=T.+Matsuiauthor=V.+Tergaonkarauthor=C.+Rodriguez-Estebanauthor=J.+C.+Izpisua-Belmonteauthor=I.+M.+Verma&title=Zebrafish+I%CE%BAB+Kinase+1+Negatively+Regulates+NF-%CE%BAB+Activity&doi=10.1016%2Fj.cub.2005.06.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Zebrafish IκB Kinase 1 Negatively Regulates NF-κB Activity</span></div><div class="casAuthors">Correa, Ricardo G.; Matsui, Takaaki; Tergaonkar, Vinay; Rodriguez-Esteban, Concepcion; Izpisua-Belmonte, Juan Carlos; Verma, Inder M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1291-1295</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The IκB kinase (IKK) activity is crit. for processing IκB inhibitory proteins and activating the NF-κB signaling, which is involved in a series of physiol. and developmental steps in vertebrates.  The IKK activity resides in two catalytic subunits, IKK1 and IKK2, and two regulatory subunits, NEMO and ELKS.  IKK2 is the major cytokine-responsive IκB kinase because depletion of IKK1 does not interfere with the IKK activity.  In fact, IKK1-/- mice display morphol. abnormalities that are independent of its kinase activity and NF-κB activation.  Hence, using zebrafish (Danio rerio) as a model, the authors examd. the evolutionary role of IKK1 in modulating NF-κB.  Ikk1-/- zebrafish embryos present head and tail malformations and, surprisingly, show upregulation of NF-κB-responsive genes and increased NF-κB-dependent apoptosis.  Overexpression of ikk1 leads to midline structure defects that resemble NF-κB blockage in vivo.  Zebrafish Ikk1 forms complexes with NEMO that represses NF-κB in vertebrate cells.  Indeed, truncation of its NEMO binding domain (NBD) restores NF-κB-dependent transcriptional activity and, consequently, the ikk1-overexpressing phenotype.  Here, the authors report that Ikk1 neg. regulates NF-κB by sequestering NEMO from active IKK complexes, indicating that IKK1 can function as a repressor of NF-κB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokL64ZlJ8V6rVg90H21EOLACvtfcHk0ljBqfvnT5HqSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmvVKgsrs%253D&md5=d69d6fcf5a97936335bda74f6f058835</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.cub.2005.06.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cub.2005.06.023%26sid%3Dliteratum%253Aachs%26aulast%3DCorrea%26aufirst%3DR.%2BG.%26aulast%3DMatsui%26aufirst%3DT.%26aulast%3DTergaonkar%26aufirst%3DV.%26aulast%3DRodriguez-Esteban%26aufirst%3DC.%26aulast%3DIzpisua-Belmonte%26aufirst%3DJ.%2BC.%26aulast%3DVerma%26aufirst%3DI.%2BM.%26atitle%3DZebrafish%2520I%25CE%25BAB%2520Kinase%25201%2520Negatively%2520Regulates%2520NF-%25CE%25BAB%2520Activity%26jtitle%3DCurr.%2520Biol.%26date%3D2005%26volume%3D15%26spage%3D1291%26epage%3D1295%26doi%3D10.1016%2Fj.cub.2005.06.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span> <span> </span><span class="NLM_article-title">Epigenetics and cell cycle regulation in cystogenesis</span>. <i>Cell. Signalling</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">109509</span>, <span class="refDoi"> DOI: 10.1016/j.cellsig.2019.109509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.cellsig.2019.109509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=31874209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVeksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2020&pages=109509&author=X.+Li&title=Epigenetics+and+cell+cycle+regulation+in+cystogenesis&doi=10.1016%2Fj.cellsig.2019.109509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetics and cell cycle regulation in cystogenesis</span></div><div class="casAuthors">Li, Xiaogang</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">109509</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  The role of genetic mutations in the development of polycystic kidney disease (PKD), such as alterations in PKD1 and PKD2 genes in autosomal dominant PKD (ADPKD), is well understood.  However, the significance of epigenetic mechanisms in the progression of PKD remains unclear and is increasingly being investigated.  The term of epigenetics describes a range of mechanisms in genome function that do not solely result from the DNA sequence itself.  Epigenetic information can be inherited during mammalian cell division to sustain phenotype specifically and physiol. responsive gene expression in the progeny cells.  A multitude of functional studies of epigenetic modifiers and systematic genome-wide mapping of epigenetic marks reveal the importance of epigenomic mechanisms, including DNA methylation, histone/chromatin modifications and non-coding RNAs, in PKD pathologies.  Deregulated proliferation is a characteristic feature of cystic renal epithelial cells.  Moreover, defects in many of the mols. that regulate the cell cycle have been implicated in cyst formation and progression.  Recent evidence suggests that alterations of DNA methylation and histone modifications on specific genes and the whole genome involved in cell cycle regulation and contribute to the pathogenesis of PKD.  This review summarizes the recent advances of epigenetic mechanisms in PKD, which helps us to define the term of "PKD epigenetics" and group PKD epigenetic changes in three categories.  In particularly, this review focuses on the interplay of epigenetic mechanisms with cell cycle regulation during normal cell cycle progression and cystic cell proliferation, and discusses the potential to detect and quantify DNA methylation from body fluids as diagnostic/prognostic biomarkers.  Collectively, this review provides concepts and examples of epigenetics in cell cycle regulation to reveal a broad view of different aspects of epigenetics in biol. and PKD, which may facilitate to identify possible novel therapeutic intervention points and to explore epigenetic biomarkers in PKD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoN6h7oTe7ci7Vg90H21EOLACvtfcHk0ljBqfvnT5HqSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVeksLs%253D&md5=556e6f66521bf8ca404abe67a43fc7b0</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2019.109509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2019.109509%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26atitle%3DEpigenetics%2520and%2520cell%2520cycle%2520regulation%2520in%2520cystogenesis%26jtitle%3DCell.%2520Signalling%26date%3D2020%26volume%3D68%26spage%3D109509%26doi%3D10.1016%2Fj.cellsig.2019.109509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saigusa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, P. D.</span></span> <span> </span><span class="NLM_article-title">Molecular pathways and therapies in autosomal-dominant polycystic kidney disease</span>. <i>Physiology</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">207</span>, <span class="refDoi"> DOI: 10.1152/physiol.00032.2014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1152%2Fphysiol.00032.2014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=25933820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVOju7nO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2015&pages=195-207&author=T.+Saigusaauthor=P.+D.+Bell&title=Molecular+pathways+and+therapies+in+autosomal-dominant+polycystic+kidney+disease&doi=10.1152%2Fphysiol.00032.2014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular pathways and therapies in autosomal-dominant polycystic kidney disease</span></div><div class="casAuthors">Saigusa, Takamitsu; Bell, P. Darwin</div><div class="citationInfo"><span class="NLM_cas:title">Physiology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">May</span>),
    <span class="NLM_cas:pages">195-207</span>CODEN:
                <span class="NLM_cas:coden">PHYSCI</span>;
        ISSN:<span class="NLM_cas:issn">1548-9213</span>.
    
            (<span class="NLM_cas:orgname">International Union of Physiological Sciences</span>)
        </div><div class="casAbstract">A review.  Autosomal-dominant polycystic kidney disease (ADPKD) is the most prevalent inherited renal disease, characterized by multiple cysts that can eventually lead to kidney failure.  Studies investigating the role of primary cilia and polycystins have significantly advanced our understanding of the pathogenesis of PKD.  This review will present clin. and basic aspects of ADPKD, review current concepts of PKD pathogenesis, evaluate potential therapeutic targets, and highlight challenges for future clin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxW9aaMg23f7Vg90H21EOLACvtfcHk0ljBqfvnT5HqSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVOju7nO&md5=b999675a2119ed75898b32515288fe5a</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1152%2Fphysiol.00032.2014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fphysiol.00032.2014%26sid%3Dliteratum%253Aachs%26aulast%3DSaigusa%26aufirst%3DT.%26aulast%3DBell%26aufirst%3DP.%2BD.%26atitle%3DMolecular%2520pathways%2520and%2520therapies%2520in%2520autosomal-dominant%2520polycystic%2520kidney%2520disease%26jtitle%3DPhysiology%26date%3D2015%26volume%3D30%26spage%3D195%26epage%3D207%26doi%3D10.1152%2Fphysiol.00032.2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Giovanni, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingram, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenblum, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, Y.</span></span> <span> </span><span class="NLM_article-title">Systems biology of autosomal dominant polycystic kidney disease (ADPKD): computational identification of gene expression pathways and integrated regulatory networks</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2328</span>– <span class="NLM_lpage">2343</span>, <span class="refDoi"> DOI: 10.1093/hmg/ddp165</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1093%2Fhmg%2Fddp165" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=19346236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1ehtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2009&pages=2328-2343&author=X.+Songauthor=V.+Di+Giovanniauthor=N.+Heauthor=K.+Wangauthor=A.+Ingramauthor=N.+D.+Rosenblumauthor=Y.+Pei&title=Systems+biology+of+autosomal+dominant+polycystic+kidney+disease+%28ADPKD%29%3A+computational+identification+of+gene+expression+pathways+and+integrated+regulatory+networks&doi=10.1093%2Fhmg%2Fddp165"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Systems biology of autosomal dominant polycystic kidney disease (ADPKD): computational identification of gene expression pathways and integrated regulatory networks</span></div><div class="casAuthors">Song, Xuewen; Di Giovanni, Valeria; He, Ning; Wang, Kairong; Ingram, Alistair; Rosenblum, Norman D.; Pei, York</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2328-2343</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">To elucidate the mol. pathways that modulate renal cyst growth in ADPKD, we performed global gene profiling on cysts of different size (<1 mL, n = 5; 10-20 mL, n = 5; >50 mL, n = 3) and minimally cystic tissue (MCT, n = 5) from five PKD1 human polycystic kidneys using Affymetrix HG-U133 Plus 2.0 arrays.  We used gene set enrichment anal. to identify overrepresented signaling pathways and key transcription factors (TFs) between cysts and MCT.  We found down-regulation of kidney epithelial restricted genes (e.g. nephron segment-specific markers and cilia-assocd. cystic genes such as HNF1B, PKHD1, IFT88 and CYS1) in the renal cysts.  On the other hand, PKD1 cysts displayed a rich profile of gene sets assocd. with renal development, mitogen-mediated proliferation, cell cycle progression, epithelial-mesenchymal transition, hypoxia, aging and immune/inflammatory responses.  Notably, our data suggest that up-regulation of Wnt/beta-catenin, pleiotropic growth factor/receptor tyrosine kinase (e.g. IGF/IGF1R, FGF/FGFR, EGF/EGFR, VEGF/VEGFR), G-protein-coupled receptor (e.g. PTGER2) signaling was assocd. with renal cystic growth.  By integrating these pathways with a no. of dysregulated networks of TFs (e.g. SRF, MYC, E2F1, CREB1, LEF1, TCF7, HNF1B/ HNF1A and HNF4A), our data suggest that epithelial dedifferentiation accompanied by aberrant activation and cross-talk of specific signaling pathways may be required for PKD1 cyst growth and disease progression.  Pharmacol. modulation of some of these signaling pathways may provide a potential therapeutic strategy for ADPKD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8tdwJNQDsg7Vg90H21EOLACvtfcHk0lh9Hff0q2_hKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1ehtLs%253D&md5=6a0af3a16ffd7d70714cd4cefd3e2b92</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2Fddp165&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252Fddp165%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DX.%26aulast%3DDi%2BGiovanni%26aufirst%3DV.%26aulast%3DHe%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DIngram%26aufirst%3DA.%26aulast%3DRosenblum%26aufirst%3DN.%2BD.%26aulast%3DPei%26aufirst%3DY.%26atitle%3DSystems%2520biology%2520of%2520autosomal%2520dominant%2520polycystic%2520kidney%2520disease%2520%2528ADPKD%2529%253A%2520computational%2520identification%2520of%2520gene%2520expression%2520pathways%2520and%2520integrated%2520regulatory%2520networks%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2009%26volume%3D18%26spage%3D2328%26epage%3D2343%26doi%3D10.1093%2Fhmg%2Fddp165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lh9Hff0q2_hKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haldi, M.</span>; <span class="NLM_string-name">Harden, M.</span>; <span class="NLM_string-name">D’Amico, L.</span>; <span class="NLM_string-name">DeLise, A.</span>; <span class="NLM_string-name">Seng, W. L.</span></span> <span> </span><span class="NLM_article-title">Developmental Toxicity Assessment in Zebrafish</span>. In  <i>Zebrafish: Methods for Assessing Drug Safety and Toxicity</i>, <span class="NLM_contrib-group"><span class="NLM_string-name">McGrath, P.</span></span>, Ed.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>, <span class="NLM_year">2012</span>; Chapter 2, pp  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">17</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=15-17&author=M.+Haldi&author=M.+Harden&author=L.+D%E2%80%99Amico&author=A.+DeLise&author=W.+L.+Sengauthor=P.+McGrath&title=Zebrafish%3A+Methods+for+Assessing+Drug+Safety+and+Toxicity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DHaldi%26aufirst%3DM.%26atitle%3DDevelopmental%2520Toxicity%2520Assessment%2520in%2520Zebrafish%26btitle%3DZebrafish%253A%2520Methods%2520for%2520Assessing%2520Drug%2520Safety%2520and%2520Toxicity%26aulast%3DMcGrath%26aufirst%3DP.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2012%26spage%3D15%26epage%3D17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edelstein, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchida, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amodaj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinkard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vale, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuurman, N.</span></span> <span> </span><span class="NLM_article-title">Advanced methods of microscope control using μManager software</span>. <i>Journal of Biological Methods</i> <span class="NLM_year" style="font-weight: bold;">2014</span>, <span class="NLM_elocation-id">e10</span> <span class="refDoi"> DOI: 10.14440/jbm.2014.36</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.14440%2Fjbm.2014.36" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=25606571" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=A.+D.+Edelsteinauthor=M.+A.+Tsuchidaauthor=N.+Amodajauthor=H.+Pinkardauthor=R.+D.+Valeauthor=N.+Stuurman&title=Advanced+methods+of+microscope+control+using+%CE%BCManager+software&doi=10.14440%2Fjbm.2014.36"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.14440%2Fjbm.2014.36&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.14440%252Fjbm.2014.36%26sid%3Dliteratum%253Aachs%26aulast%3DEdelstein%26aufirst%3DA.%2BD.%26aulast%3DTsuchida%26aufirst%3DM.%2BA.%26aulast%3DAmodaj%26aufirst%3DN.%26aulast%3DPinkard%26aufirst%3DH.%26aulast%3DVale%26aufirst%3DR.%2BD.%26aulast%3DStuurman%26aufirst%3DN.%26atitle%3DAdvanced%2520methods%2520of%2520microscope%2520control%2520using%2520%25CE%25BCManager%2520software%26jtitle%3DJournal%2520of%2520Biological%2520Methods%26date%3D2014%26doi%3D10.14440%2Fjbm.2014.36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/medium/jm0c02118_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02118&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/medium/jm0c02118_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical starting points for the development of Nek1 inhibitors, as identified by Wells et al. in an analysis of kinome cross-screening data.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02118&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/medium/jm0c02118_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02118&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) for <b>2a</b>–<b>h</b>, DCM, amine, NEt<sub>3</sub>, 0 °C–rt, 74–95%; (b) DMF, bis(pinacolato)diboron, KOAc, PdCl<sub>2</sub>(dppf), 100 °C, not isolated; (c) DMF, 4-bromo-7-azaindole, NaHCO<sub>3</sub> (aq), Pd(PPh<sub>3</sub>)<sub>4</sub>, 100 °C, 11–69%; (d) for <b>2j</b>, DCM, amine, NEt<sub>3</sub>, 0 °C–rt, 64%; for <b>2k</b>, THF, 1-methylpiperazine, rt, 75%; for <b>2l</b>, H<sub>2</sub>O/acetone, <span class="smallcaps smallerCapital">L</span>-phenylalanine, NaHCO<sub>3</sub>, rt, 49%. *Compound <b>2i</b> was prepared from <b>2f</b> by acetylation with acetic anhydride and NEt<sub>3</sub> in DCM, 82%; see <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for more details.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/medium/jm0c02118_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02118&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DCM, monoethanolamine, NEt<sub>3</sub>, 0 °C–rt, 48%; (b) THF, 50% w/v KOH (aq), 60 °C, 92%; (c) DMF, bis(pinacolato)diboron, KOAc, PdCl<sub>2</sub>(dppf), 100 °C, not isolated; (d) DMF, 4-bromo-7-azaindole, NaHCO<sub>3</sub> (aq), Pd(PPh<sub>3</sub>)<sub>4</sub>, 100 °C, 43%; see <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for more details.</p></p></figure><figure data-id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/medium/jm0c02118_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02118&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) for <b>9a</b>,<b>b</b>, DCM, amine, NEt<sub>3</sub>, 0 °C–rt, 98–99%; for <b>9c</b>,<b>d</b>, H<sub>2</sub>O/acetone, <span class="smallcaps smallerCapital">L</span>-phenylalanine, NaHCO<sub>3</sub>, rt, 60–70%; for <b>9e</b>,<b>f</b>, DCM/acetonitrile, <i>N</i>,<i>N</i>-diisopropylethylamine, 0 °C–rt, 30–43%; (b) DMF, bis(pinacolato)diboron, KOAc, PdCl<sub>2</sub>(dppf), 100 °C, not isolated; (c) DMF, 4-bromo-7-azaindole, NaHCO<sub>3</sub> (aq), Pd(PPh<sub>3</sub>)<sub>4</sub>, 100 °C, 26–62%. <b>*</b>Compounds <b>8a</b>–<b>c</b> were obtained commercially; Compound <b>8d</b> was prepared from <span class="smallcaps smallerCapital">d</span>-tyrosine methyl ester hydrochloride by freeing the amine with KOAc in acetonitrile and subsequent reduction with NaBH<sub>4</sub> in methanol, 61%; compounds <b>8e</b>,<b>f</b> were prepared by reduction of the corresponding tryptophan isomer using the method described by Meyers et al.,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> 72–84%; see <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for more details.</p></p></figure><figure data-id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/medium/jm0c02118_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. <i>In silico</i> chemical library screening to identify hit compounds. See <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for more details.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02118&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/medium/jm0c02118_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Molecular docking of compound <b>10f</b> into X-ray structure of Nek1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4B9D">4B9D</a>.B). P-loop reconstruction was performed using the SuperLooper2 web application.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> See <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Supporting Information</a> for more details (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Figures S5 and S6</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02118&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/medium/jm0c02118_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Screening of Compound <b>10f</b> against a panel of human protein kinases. Each bar represents the activity of one individual protein kinase. Compound <b>10f</b> was tested at a concentration of 1 μM. The ATP concentration was set to the approximate <i>K</i><sub>m,ATP</sub> for each kinase tested. See <a href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf" class="ext-link">Supporting Information</a> for more details.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02118&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/medium/jm0c02118_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Microscopic imaging of fluorescence-labeled (PT-Yellow) zebrafish pronephroi after treatment with Nek1-inhibitor <b>10f</b>. (A) Schematic representation of the workflow, entailing embryo selection, compound treatment, and image acquisition. (B) Schematic representation of the pronephros in the developing zebrafish (28-somite stage). Reprinted with permission from ref <a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a>. Copyright 2018 Gehrig, Pandey and Westhoff. (C) Exemplary maximum intensity projections of deconvolved <i>Z</i>-stacks (15× magnification; 33 <i>Z</i>-slices; d<i>Z</i> = 15 μm). The dotted line indicates the cropped region of interest. (C, i) negative control E3-medium, (C, ii) negative control 0.2% DMSO, (C, iii) 10 μM <b>10f</b>, and (C, iv) 20 μM <b>10f</b>. (D) Quantification of fluorescence intensity based on summed image projections of deconvolved <i>Z</i>-stacks using ImageJ. <i>n</i> = 3 for negative control, 0.2% DMSO, and 10 μM <b>10f</b>; <i>n</i> = 5 for 20 μM <b>10f</b> (<sup>#</sup><i>P</i> = 0.0516; *<i>P</i> < 0.05). (E) Quantification of pronephroi size by summed pixel area determination based on summed image projections of deconvolved <i>Z</i>-stacks using ImageJ. <i>n</i> = 3 for negative control, 0.2% DMSO, and 10 μM <b>10f</b>; <i>n</i> = 5 for 20 μM <b>10f</b> (*<i>P</i> < 0.05; **<i>P</i> < 0.01). Abbreviations: au, arbitrary unit; C, Cloaca; CS, corpuscles of Stannius; DE, distal early tubule; DL, distal late tubule; hpf, hours post fertilization; N, neck region; P, podocytes of the glomerulus; PCT, proximal convoluted tubule; PD, pronephric duct; PST, Proximal straight tubule.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02118/20210316/images/large/jm0c02118_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02118&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i71">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67083" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67083" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 66 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
de Kooij, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creixell, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vlimmeren, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joughin, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haider, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linding, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stambolic, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaffe, M. B.</span></span> <span> </span><span class="NLM_article-title">Comprehensive substrate specificity profiling of the human Nek kinome reveals unexpected signaling outputs</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e44635</span> <span class="refDoi"> DOI: 10.7554/eLife.44635</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.7554%2FeLife.44635" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=31124786" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Crtr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&author=B.+van%0Ade+Kooijauthor=P.+Creixellauthor=A.+van+Vlimmerenauthor=B.+A.+Joughinauthor=C.+J.+Millerauthor=N.+Haiderauthor=C.+D.+Simpsonauthor=R.+Lindingauthor=V.+Stambolicauthor=B.+E.+Turkauthor=M.+B.+Yaffe&title=Comprehensive+substrate+specificity+profiling+of+the+human+Nek+kinome+reveals+unexpected+signaling+outputs&doi=10.7554%2FeLife.44635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive substrate specificity profiling of the human Nek kinome reveals unexpected signaling outputs</span></div><div class="casAuthors">van de Kooij, Bert; Creixell, Pau; Van Vlimmeren, Anne; Joughin, Brian A.; Miller, Chad J.; Haider, Nasir; Simpson, Craig D.; Linding, Rune; Stambolic, Vuk; Turk, Benjamin E.; Yaffe, Michael B.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e44635</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Human NimA-related kinases (Neks) have multiple mitotic and non-mitotic functions, but few substrates are known.  We systematically detd. the phosphorylation-site motifs for the entire Nek kinase family, except for Nek11.  While all Nek kinases strongly select for hydrophobic residues in the 3 position, the family separates into four distinct groups based on specificity for a serine vs. threonine phospho-acceptor, and preference for basic or acidic residues in other positions.  Unlike Nek1-Nek9, Nek10 is a dual-specificity kinase that efficiently phosphorylates itself and peptide substrates on serine and tyrosine, and its activity is enhanced by tyrosine auto-phosphorylation.  Nek10 dual-specificity depends on residues in the HRD+2 and APE4 positions that are uncommon in either serine/threonine or tyrosine kinases.  Finally, we show that the phosphorylation-site motifs for the mitotic kinases Nek6, Nek7 and Nek9 are essentially identical to that of their upstream activator Plk1, suggesting that Nek6/7/9 function as phosphomotif amplifiers of Plk1 signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqk3K-2RlBb-7Vg90H21EOLACvtfcHk0lizjBVuo2kmig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Crtr%252FE&md5=3ca0d00e517849a33a3565594a72f7fa</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.7554%2FeLife.44635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.44635%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bde%2BKooij%26aufirst%3DB.%26aulast%3DCreixell%26aufirst%3DP.%26aulast%3Dvan%2BVlimmeren%26aufirst%3DA.%26aulast%3DJoughin%26aufirst%3DB.%2BA.%26aulast%3DMiller%26aufirst%3DC.%2BJ.%26aulast%3DHaider%26aufirst%3DN.%26aulast%3DSimpson%26aufirst%3DC.%2BD.%26aulast%3DLinding%26aufirst%3DR.%26aulast%3DStambolic%26aufirst%3DV.%26aulast%3DTurk%26aufirst%3DB.%2BE.%26aulast%3DYaffe%26aufirst%3DM.%2BB.%26atitle%3DComprehensive%2520substrate%2520specificity%2520profiling%2520of%2520the%2520human%2520Nek%2520kinome%2520reveals%2520unexpected%2520signaling%2520outputs%26jtitle%3DeLife%26date%3D2019%26volume%3D8%26doi%3D10.7554%2FeLife.44635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Letwin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizzen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motro, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-David, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernstein, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span> <span> </span><span class="NLM_article-title">A mammalian dual specificity protein kinase, Nek1, is related to the NIMA cell cycle regulator and highly expressed in meiotic germ cells</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3521</span>– <span class="NLM_lpage">3531</span>, <span class="refDoi"> DOI: 10.1002/j.1460-2075.1992.tb05435.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1002%2Fj.1460-2075.1992.tb05435.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1382974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADyaK3sXkvV2rtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1992&pages=3521-3531&author=K.+Letwinauthor=L.+Mizzenauthor=B.+Motroauthor=Y.+Ben-Davidauthor=A.+Bernsteinauthor=T.+Pawson&title=A+mammalian+dual+specificity+protein+kinase%2C+Nek1%2C+is+related+to+the+NIMA+cell+cycle+regulator+and+highly+expressed+in+meiotic+germ+cells&doi=10.1002%2Fj.1460-2075.1992.tb05435.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">A mammalian dual specificity protein kinase, Nek1, is related to the NIMA cell cycle regulator and highly expressed in meiotic germ cells</span></div><div class="casAuthors">Letwin, Kenneth; Mizzen, Lee; Motro, Benny; Ben-David, Yaacov; Bernstein, Alan; Pawson, Tony</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3521-31</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    </div><div class="casAbstract">Screening of mouse cDNA expression libraries with antibodies to phosphotyrosine resulted in repeated isolation of cDNAs that encode a novel mammalian protein kinase of 774 amino acids termed Nek1.  Nek1 contains an N-terminal protein kinase domain which is most similar (42% identity) to the catalytic domain of NIMA, a protein kinase which controls initiation of mitosis in Aspergillus nidulans.  In addn., both Nek1 and NIMA have a long, basic C-terminal extension, and are therefore similar in overall structure.  Despite its identification with anti-phosphotyrosine antibodies, Nek1 contains sequence motifs characteristic of protein serine/threonine kinases.  The Nek1 kinase domain, when expressed in bacteria, phosphorylated exogenous substrates primarily on serine/threonine, but also on tyrosine, indicating that Nek1 is a dual-specificity kinase with the capacity to phosphorylate all 3 hydroxyamino acids.  Like NIMA, Nek1 preferentially phosphorylated β-casein in vitro.  In situ RNA anal. of nek1 expression in mouse gonads revealed a high level of expression in both male and female germ cells, with a distribution consistent with a role in meiosis.  Apparently, Nek1 is a mammalian relative of the fungal NIMA cell cycle regulator.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBIGgayMb1BrVg90H21EOLACvtfcHk0liiJ5HhB6132A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXkvV2rtLs%253D&md5=119e2b72c4532c74e70eb7d92374f980</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2Fj.1460-2075.1992.tb05435.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fj.1460-2075.1992.tb05435.x%26sid%3Dliteratum%253Aachs%26aulast%3DLetwin%26aufirst%3DK.%26aulast%3DMizzen%26aufirst%3DL.%26aulast%3DMotro%26aufirst%3DB.%26aulast%3DBen-David%26aufirst%3DY.%26aulast%3DBernstein%26aufirst%3DA.%26aulast%3DPawson%26aufirst%3DT.%26atitle%3DA%2520mammalian%2520dual%2520specificity%2520protein%2520kinase%252C%2520Nek1%252C%2520is%2520related%2520to%2520the%2520NIMA%2520cell%2520cycle%2520regulator%2520and%2520highly%2520expressed%2520in%2520meiotic%2520germ%2520cells%26jtitle%3DEMBO%2520J.%26date%3D1992%26volume%3D11%26spage%3D3521%26epage%3D3531%26doi%3D10.1002%2Fj.1460-2075.1992.tb05435.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meirelles, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basei, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papa, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melo Hanchuk, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardoso, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobarg, J.</span></span> <span> </span><span class="NLM_article-title">Stop Ne(c)king around“: How interactomics contributes to functionally characterize Nek family kinases</span>. <i>World journal of biological chemistry</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">160</span>, <span class="refDoi"> DOI: 10.4331/wjbc.v5.i2.141</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.4331%2Fwjbc.v5.i2.141" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=24921005" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A280%3ADC%252BC2cflvF2jsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=141-160&author=G.+V.+Meirellesauthor=A.+M.+Perezauthor=E.+E.+Souzaauthor=F.+L.+Baseiauthor=P.+F.+Papaauthor=T.+D.+Melo+Hanchukauthor=V.+B.+Cardosoauthor=J.+Kobarg&title=Stop+Ne%28c%29king+around%E2%80%9C%3A+How+interactomics+contributes+to+functionally+characterize+Nek+family+kinases&doi=10.4331%2Fwjbc.v5.i2.141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">"Stop Ne(c)king around": How interactomics contributes to functionally characterize Nek family kinases</span></div><div class="casAuthors">Meirelles Gabriela Vaz; Perez Arina Marina; de Souza Edmarcia Elisa; Basei Fernanda Luisa; Papa Priscila Ferreira; Melo Hanchuk Talita Diniz; Cardoso Vanessa Bomfim; Kobarg Jorg</div><div class="citationInfo"><span class="NLM_cas:title">World journal of biological chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">141-60</span>
        ISSN:<span class="NLM_cas:issn">1949-8454</span>.
    </div><div class="casAbstract">Aside from Polo and Aurora, a third but less studied kinase family involved in mitosis regulation is the never in mitosis-gene A (NIMA)-related kinases (Neks).  The founding member of this family is the sole member NIMA of Aspergillus nidulans, which is crucial for the initiation of mitosis in that organism.  All 11 human Neks have been functionally assigned to one of the three core functions established for this family in mammals: (1) centrioles/mitosis; (2) primary ciliary function/ciliopathies; and (3) DNA damage response (DDR).  Recent findings, especially on Nek 1 and 8, showed however, that several Neks participate in parallel in at least two of these contexts: primary ciliary function and DDR.  In the core section of this in-depth review, we report the current detailed functional knowledge on each of the 11 Neks.  In the discussion, we return to the cross-connections among Neks and point out how our and other groups' functional and interactomics studies revealed that most Neks interact with protein partners associated with two if not all three of the functional contexts.  We then raise the hypothesis that Neks may be the connecting regulatory elements that allow the cell to fine tune and synchronize the cellular events associated with these three core functions.  The new and exciting findings on the Nek family open new perspectives and should allow the Neks to finally claim the attention they deserve in the field of kinases and cell cycle biology.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTQeWflPetSDcRAV4D_Ins_fW6udTcc2eaJR09rfc1OuLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cflvF2jsg%253D%253D&md5=26d9c7e0cc2d51f47a686efe07058290</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.4331%2Fwjbc.v5.i2.141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4331%252Fwjbc.v5.i2.141%26sid%3Dliteratum%253Aachs%26aulast%3DMeirelles%26aufirst%3DG.%2BV.%26aulast%3DPerez%26aufirst%3DA.%2BM.%26aulast%3DSouza%26aufirst%3DE.%2BE.%26aulast%3DBasei%26aufirst%3DF.%2BL.%26aulast%3DPapa%26aufirst%3DP.%2BF.%26aulast%3DMelo%2BHanchuk%26aufirst%3DT.%2BD.%26aulast%3DCardoso%26aufirst%3DV.%2BB.%26aulast%3DKobarg%26aufirst%3DJ.%26atitle%3DStop%2520Ne%2528c%2529king%2520around%25E2%2580%259C%253A%2520How%2520interactomics%2520contributes%2520to%2520functionally%2520characterize%2520Nek%2520family%2520kinases%26jtitle%3DWorld%2520journal%2520of%2520biological%2520chemistry%26date%3D2014%26volume%3D5%26spage%3D141%26epage%3D160%26doi%3D10.4331%2Fwjbc.v5.i2.141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peres
de Oliveira, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazuo Issayama, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betim Pavan, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riback Silva, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diniz Melo-Hanchuk, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreira Simabuco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobarg, J.</span></span> <span> </span><span class="NLM_article-title">Checking NEKs: Overcoming a Bottleneck in Human Diseases</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1778</span>, <span class="refDoi"> DOI: 10.3390/molecules25081778</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.3390%2Fmolecules25081778" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=1778&author=A.+Peres%0Ade+Oliveiraauthor=L.+Kazuo+Issayamaauthor=I.+C.+Betim+Pavanauthor=F.+Riback+Silvaauthor=T.+Diniz+Melo-Hanchukauthor=F.+Moreira+Simabucoauthor=J.+Kobarg&title=Checking+NEKs%3A+Overcoming+a+Bottleneck+in+Human+Diseases&doi=10.3390%2Fmolecules25081778"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3390%2Fmolecules25081778&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules25081778%26sid%3Dliteratum%253Aachs%26aulast%3DPeres%2Bde%2BOliveira%26aufirst%3DA.%26aulast%3DKazuo%2BIssayama%26aufirst%3DL.%26aulast%3DBetim%2BPavan%26aufirst%3DI.%2BC.%26aulast%3DRiback%2BSilva%26aufirst%3DF.%26aulast%3DDiniz%2BMelo-Hanchuk%26aufirst%3DT.%26aulast%3DMoreira%2BSimabuco%26aufirst%3DF.%26aulast%3DKobarg%26aufirst%3DJ.%26atitle%3DChecking%2520NEKs%253A%2520Overcoming%2520a%2520Bottleneck%2520in%2520Human%2520Diseases%26jtitle%3DMolecules%26date%3D2020%26volume%3D25%26spage%3D1778%26doi%3D10.3390%2Fmolecules25081778" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janaswami, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birkenmeier, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowe, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davisson, M. T.</span></span> <span> </span><span class="NLM_article-title">Identification and genetic mapping of a new polycystic kidney disease on mouse chromosome 8</span>. <i>Genomics</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">107</span>, <span class="refDoi"> DOI: 10.1006/geno.1996.4567</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1006%2Fgeno.1996.4567" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=9070925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADyaK2sXhtlSgt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1997&pages=101-107&author=P.+M.+Janaswamiauthor=E.+H.+Birkenmeierauthor=S.+A.+Cookauthor=L.+B.+Roweauthor=R.+T.+Bronsonauthor=M.+T.+Davisson&title=Identification+and+genetic+mapping+of+a+new+polycystic+kidney+disease+on+mouse+chromosome+8&doi=10.1006%2Fgeno.1996.4567"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and genetic mapping of a new polycystic kidney diseases on mouse chromosome 8</span></div><div class="casAuthors">Janaswami, Poornima M.; Birkenmeier, Edward H.; Cook, Susan A.; Rowe, Lucy B.; Bronson, Roderick T.; Davisson, Muriel T.</div><div class="citationInfo"><span class="NLM_cas:title">Genomics</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">101-107</span>CODEN:
                <span class="NLM_cas:coden">GNMCEP</span>;
        ISSN:<span class="NLM_cas:issn">0888-7543</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">We report here a new mouse mutation, kat, that causes pleiotropic effects including facial dysmorphism, dwarfing, male sterility, anemia, and progressive polycystic kidney disease.  Kat (kidney anemia and testis) and a second allele, kat2J, that occurred on C57BL/6J were mapped to mouse chromosome (Chr) 8 using intra- and intersubspecific intercrosses.  A high-resoln. map for kat2J on Chr 8 was constructed using the F2 progeny from a cross between C57BL/6J-kat2J/+ and an inbred strain of Mus musculus castaneus (CAST/Ei).  The kat2J mutation was localized between D8Mit129 and D8Mit128 with the gene order centromere-D8Mit100-(1.2±0.26 cM)-D8Mit231-(0.17±0.09 cM)-D8Mit129-(0.28±0.12 cM)-D8Mit128-(0.98±0.23 cM)-D8Mit25/D8Mit8.  This segment is homologous to human Chr 19 p.  The two mutations at this locus that have occurred at The Jackson Lab. will be invaluable for positional cloning and subsequent functional anal. of the mutated gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV1SQW7cqpALVg90H21EOLACvtfcHk0lgEjLgRfTz2vQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhtlSgt70%253D&md5=9dd2b1d346aa007cec311cd201cbe751</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1006%2Fgeno.1996.4567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fgeno.1996.4567%26sid%3Dliteratum%253Aachs%26aulast%3DJanaswami%26aufirst%3DP.%2BM.%26aulast%3DBirkenmeier%26aufirst%3DE.%2BH.%26aulast%3DCook%26aufirst%3DS.%2BA.%26aulast%3DRowe%26aufirst%3DL.%2BB.%26aulast%3DBronson%26aufirst%3DR.%2BT.%26aulast%3DDavisson%26aufirst%3DM.%2BT.%26atitle%3DIdentification%2520and%2520genetic%2520mapping%2520of%2520a%2520new%2520polycystic%2520kidney%2520disease%2520on%2520mouse%2520chromosome%25208%26jtitle%3DGenomics%26date%3D1997%26volume%3D40%26spage%3D101%26epage%3D107%26doi%3D10.1006%2Fgeno.1996.4567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Upadhya, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birkenmeier, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birkenmeier, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, J. E.</span></span> <span> </span><span class="NLM_article-title">Mutations in a NIMA-related kinase gene cause pleiotropic effects including a progressive polycystic kidney disease in mice</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">217</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1073/pnas.97.1.217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1073%2Fpnas.97.1.217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10618398" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjvVCqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2000&pages=217-221&author=P.+Upadhyaauthor=E.+H.+Birkenmeierauthor=C.+S.+Birkenmeierauthor=J.+E.+Barker&title=Mutations+in+a+NIMA-related+kinase+gene+cause+pleiotropic+effects+including+a+progressive+polycystic+kidney+disease+in+mice&doi=10.1073%2Fpnas.97.1.217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in a NIMA-related kinase gene, Nek1, cause pleiotropic effects including a progressive polycystic kidney disease in mice</span></div><div class="casAuthors">Upadhya, Poornima; Birkenmeier, Edward H.; Birkenmeier, Connie S.; Barker, Jane E.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">217-221</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We previously have described a mouse model for polycystic kidney disease (PKD) caused by either of two mutations, kat or kat2J, that map to the same locus on chromosome 8.  The homozygous mutant animals have a latent onset, slowly progressing form of PKD with renal pathol. similar to the human autosomal-dominant PKD.  In addn., the mutant animals show pleiotropic effects that include facial dysmorphism, dwarfing, male sterility, anemia, and cystic choroid plexus.  We previously fine-mapped the kat2J mutation to a genetic distance of 0.28 centimorgan between D8Mit128 and D8Mit129.  To identify the underlying mol. defect in this locus, we constructed an integrated genetic and phys. map of the crit. region surrounding the kat2J mutation.  Cloning and expression anal. of the transcribed sequences from this region identified Nek1, a NIMA (never in mitosis A)-related kinase as a candidate gene.  Further anal. of the Nek1 gene from both kat/kat and kat2J/kat2J mutant animals identified a partial internal deletion and a single-base insertion as the mol. basis for these mutations.  The complex pleiotropic phenotypes seen in the homozygous mutant animals suggest that the NEK1 protein participates in different signaling pathways to regulate diverse cellular processes.  Our findings identify a previously unsuspected role for Nek1 in the kidney and open a new avenue for studying cystogenesis and identifying possible modes of therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqpjqA-TEUFbVg90H21EOLACvtfcHk0lgEjLgRfTz2vQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjvVCqtQ%253D%253D&md5=d72b1ade4c33aa220b25a6c7c329fef8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1073%2Fpnas.97.1.217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.97.1.217%26sid%3Dliteratum%253Aachs%26aulast%3DUpadhya%26aufirst%3DP.%26aulast%3DBirkenmeier%26aufirst%3DE.%2BH.%26aulast%3DBirkenmeier%26aufirst%3DC.%2BS.%26aulast%3DBarker%26aufirst%3DJ.%2BE.%26atitle%3DMutations%2520in%2520a%2520NIMA-related%2520kinase%2520gene%2520cause%2520pleiotropic%2520effects%2520including%2520a%2520progressive%2520polycystic%2520kidney%2520disease%2520in%2520mice%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2000%26volume%3D97%26spage%3D217%26epage%3D221%26doi%3D10.1073%2Fpnas.97.1.217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chebib, F. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torres, V. E.</span></span> <span> </span><span class="NLM_article-title">Recent advances in the management of autosomal dominant polycystic kidney disease</span>. <i>Clin. J. Am. Soc. Nephrol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1765</span>– <span class="NLM_lpage">1776</span>, <span class="refDoi"> DOI: 10.2215/CJN.03960318</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.2215%2FCJN.03960318" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=30049849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktVOjsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=1765-1776&author=F.+T.+Chebibauthor=V.+E.+Torres&title=Recent+advances+in+the+management+of+autosomal+dominant+polycystic+kidney+disease&doi=10.2215%2FCJN.03960318"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in the management of autosomal dominant polycystic kidney disease</span></div><div class="casAuthors">Chebib, Fouad T.; Torres, Vicente E.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Journal of the American Society of Nephrology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1765-1776</span>CODEN:
                <span class="NLM_cas:coden">CJASC7</span>;
        ISSN:<span class="NLM_cas:issn">1555-9041</span>.
    
            (<span class="NLM_cas:orgname">American Society of Nephrology</span>)
        </div><div class="casAbstract">A review.  Autosomal dominant polycystic kidney disease (ADPKD), the most common monogenic cause of ESKD, is characterized by relentless development of kidney cysts, hypertension, and destruction of the kidney parenchyma.  Over the past few years, major advancements in diagnosing, prognosticating, and understanding the pathogenesis and natural course of the disease have been made.  Currently, no kidney disease is more suitable for nephron-protective strategies.  Early nephrol. referral and implementation of these strategies may have a substantial effect.  Total kidney vol. is a good prognostication marker and allows stratification of patients into slow or rapid progressing disease, with implications for their management.  Measurement of total kidney vol., disease stratification, and prognostication are possible using readily available tools.  Although some patients require only monitoring and basic optimized kidney protective measures, such as rigorous BP control and various lifestyle and dietary changes, others will benefit from disease-modifying treatments.  Vasopressin V2 receptor antagonists, a likely disease-modifying treatment, has been approved in several countries and recently by the US Food and Drug Administration; other therapies, such as somatostatin analogs and other novel agents, are currently in clin. trials.  The purpose of this article is to present our views on the optimal management to delay kidney disease progression in ADPKD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZD-kWSfMOibVg90H21EOLACvtfcHk0lhTY1O02h2Zgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktVOjsr8%253D&md5=bbe016f282f906bc86af5061823cc565</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2215%2FCJN.03960318&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2215%252FCJN.03960318%26sid%3Dliteratum%253Aachs%26aulast%3DChebib%26aufirst%3DF.%2BT.%26aulast%3DTorres%26aufirst%3DV.%2BE.%26atitle%3DRecent%2520advances%2520in%2520the%2520management%2520of%2520autosomal%2520dominant%2520polycystic%2520kidney%2520disease%26jtitle%3DClin.%2520J.%2520Am.%2520Soc.%2520Nephrol.%26date%3D2018%26volume%3D13%26spage%3D1765%26epage%3D1776%26doi%3D10.2215%2FCJN.03960318" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Surpili, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delben, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobarg, J.</span></span> <span> </span><span class="NLM_article-title">Identification of proteins that interact with the central coiled-coil region of the human protein kinase NEK1</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">15369</span>– <span class="NLM_lpage">15376</span>, <span class="refDoi"> DOI: 10.1021/bi034575v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi034575v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD3sXptlOks7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=15369-15376&author=M.+J.+Surpiliauthor=T.+M.+Delbenauthor=J.+Kobarg&title=Identification+of+proteins+that+interact+with+the+central+coiled-coil+region+of+the+human+protein+kinase+NEK1&doi=10.1021%2Fbi034575v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Proteins That Interact with the Central Coiled-Coil Region of the Human Protein Kinase NEK1</span></div><div class="casAuthors">Surpili, Marcelo J.; Delben, Tatiana M.; Kobarg, Joerg</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">15369-15376</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">NEK protein kinases are evolutionarily conserved kinases structurally related to the Aspergillus nidulans mitotic regulator NIMA.  At least nine members of the NEK family in vertebrates have been described to date, but for most of them the interacting protein partners are unknown.  The pleiotropic deleterious effects and the formation of kidney cysts caused by NEK1 mutation in mice emphasize its involvement in the regulation of diverse cellular processes and in the etiol. of polycystic kidney disease (PKD), resp.  Here the authors report the identification of proteins that interacted with the human NEK1 protein kinase in a yeast two-hybrid screen of a human fetal brain cDNA library, using the catalytic and regulatory domains of NEK1 sep. as baits.  These proteins are known to take part either in the development of PKD, in the double-strand DNA break repair at the G2/M transition phase of the cell cycle, or in neural cell development.  The proteins involved in PKD include the motor protein KIF3A and the proteins tuberin and α-catulin.  Mapping studies of the human NEK1 regulatory domain (NRD) indicated a strong interaction of most of the proteins retrieved from the library with putative coiled coils located in the central region of NRD.  The results give further support to the previous observation that NEK1 is of functional importance for the etiol. of PKD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfVzTlZD29prVg90H21EOLACvtfcHk0lhTY1O02h2Zgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXptlOks7c%253D&md5=2b87703316b69af79310f065d1015caf</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fbi034575v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi034575v%26sid%3Dliteratum%253Aachs%26aulast%3DSurpili%26aufirst%3DM.%2BJ.%26aulast%3DDelben%26aufirst%3DT.%2BM.%26aulast%3DKobarg%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520proteins%2520that%2520interact%2520with%2520the%2520central%2520coiled-coil%2520region%2520of%2520the%2520human%2520protein%2520kinase%2520NEK1%26jtitle%3DBiochemistry%26date%3D2003%26volume%3D42%26spage%3D15369%26epage%3D15376%26doi%3D10.1021%2Fbi034575v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, H.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litchfield, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pena, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D. J.</span></span> <span> </span><span class="NLM_article-title">Expression of Nek1 during kidney development and cyst formation in multiple nephron segments in the Nek1-deficient kat2J mouse model of polycystic kidney disease</span>. <i>J. Biomed. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">65</span>, <span class="refDoi"> DOI: 10.1186/s12929-014-0063-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1186%2Fs12929-014-0063-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=25030234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvV2isbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=63-65&author=Y.+Chenauthor=H.-C.+Chiangauthor=P.+Litchfieldauthor=M.+Penaauthor=C.+Juangauthor=D.+J.+Riley&title=Expression+of+Nek1+during+kidney+development+and+cyst+formation+in+multiple+nephron+segments+in+the+Nek1-deficient+kat2J+mouse+model+of+polycystic+kidney+disease&doi=10.1186%2Fs12929-014-0063-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of Nek1 during kidney development and cyst formation in multiple nephron segments in the Nek1-deficient kat2J mouse model of polycystic kidney disease</span></div><div class="casAuthors">Chen, Yumay; Chiang, Huai-Chin; Litchfield, Patricia; Pena, Michelle; Juang, Charity; Riley, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomedical Science (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">63/1-63/14, 14 pp.</span>CODEN:
                <span class="NLM_cas:coden">JBCIEA</span>;
        ISSN:<span class="NLM_cas:issn">1423-0127</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Neks, mammalian orthologs of the fungal protein kinase never-in-mitosis A, have been implicated in the pathogenesis of polycystic kidney disease.  Among them, Nek1 is the primary protein inactivated in kat2J mouse models of PKD.  Result: We report the expression pattern of Nek1 and characterize the renal cysts that develop in kat2J mice.  Nek1 is detectable in all murine tissues but its expression in wild type and kat2J heterozygous kidneys decrease as the kidneys mature, esp. in tubular epithelial cells.  In the embryonic kidney, Nek1 expression is most prominent in cells that will become podocytes and proximal tubules.  Kidney development in kat2J homozygous mice is aberrant early, before the appearance of gross cysts: developing cortical zones are thin, populated by immature glomeruli, and characterized by excessive apoptosis of several cell types.  Cysts in kat2J homozygous mice form postnatally in Bowman's space as well as different tubular subtypes.  Late in life, kat2J heterozygous mice form renal cysts and the cells lining these cysts lack staining for Nek1.  The primary cilia of cells lining cysts in kat2J homozygous mice are morphol. diverse: in some cells they are unusually long and in others there are multiple cilia of varying lengths.  Conclusion: Our studies indicate that Nek1 deficiency leads to disordered kidney maturation, and cysts throughout the nephron.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs27DDJl2RnLVg90H21EOLACvtfcHk0lj6yGmm64y4mQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvV2isbrN&md5=6cf62ec0884bc265b3d5648452d74a81</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1186%2Fs12929-014-0063-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12929-014-0063-5%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DChiang%26aufirst%3DH.-C.%26aulast%3DLitchfield%26aufirst%3DP.%26aulast%3DPena%26aufirst%3DM.%26aulast%3DJuang%26aufirst%3DC.%26aulast%3DRiley%26aufirst%3DD.%2BJ.%26atitle%3DExpression%2520of%2520Nek1%2520during%2520kidney%2520development%2520and%2520cyst%2520formation%2520in%2520multiple%2520nephron%2520segments%2520in%2520the%2520Nek1-deficient%2520kat2J%2520mouse%2520model%2520of%2520polycystic%2520kidney%2520disease%26jtitle%3DJ.%2520Biomed.%2520Sci.%26date%3D2014%26volume%3D21%26spage%3D63%26epage%3D65%26doi%3D10.1186%2Fs12929-014-0063-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giessl, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimmler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shalev, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von der Haar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zenker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahnleiter, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stöss, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abou Jamra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekici, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schröder-Kreß, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aigner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirchner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandstätter, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauch, A.</span></span> <span> </span><span class="NLM_article-title">NEK1 mutations cause short-rib polydactyly syndrome type Majewski</span>. <i>Am. J. Hum. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">106</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1016/j.ajhg.2010.12.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.ajhg.2010.12.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=21211617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A280%3ADC%252BC3M7jtVemtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2011&pages=106-114&author=C.+Thielauthor=K.+Kesslerauthor=A.+Giesslauthor=A.+Dimmlerauthor=S.+A.+Shalevauthor=S.+von+der+Haarauthor=M.+Zenkerauthor=D.+Zahnleiterauthor=H.+St%C3%B6ssauthor=E.+Beinderauthor=R.+Abou+Jamraauthor=A.+B.+Ekiciauthor=N.+Schr%C3%B6der-Kre%C3%9Fauthor=T.+Aignerauthor=T.+Kirchnerauthor=A.+Reisauthor=J.+H.+Brandst%C3%A4tterauthor=A.+Rauch&title=NEK1+mutations+cause+short-rib+polydactyly+syndrome+type+Majewski&doi=10.1016%2Fj.ajhg.2010.12.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">NEK1 mutations cause short-rib polydactyly syndrome type majewski</span></div><div class="casAuthors">Thiel Christian; Kessler Kristin; Giessl Andreas; Dimmler Arno; Shalev Stavit A; von der Haar Sigrun; Zenker Martin; Zahnleiter Diana; Stoss Hartmut; Beinder Ernst; Abou Jamra Rami; Ekici Arif B; Schroder-Kress Nadja; Aigner Thomas; Kirchner Thomas; Reis Andre; Brandstatter Johann H; Rauch Anita</div><div class="citationInfo"><span class="NLM_cas:title">American journal of human genetics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">106-14</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Defects of ciliogenesis have been implicated in a wide range of human phenotypes and play a crucial role in signal transduction and cell-cycle coordination.  We used homozygosity mapping in two families with autosomal-recessive short-rib polydactyly syndrome Majewski type to identify mutations in NEK1 as an underlying cause of this lethal osteochondrodysplasia.  NEK1 encodes a serine/threonine kinase with proposed function in DNA double-strand repair, neuronal development, and coordination of cell-cycle-associated ciliogenesis.  We found that absence of functional full-length NEK1 severely reduces cilia number and alters ciliar morphology in vivo.  We further substantiate a proposed digenic diallelic inheritance of ciliopathies by the identification of heterozygous mutations in NEK1 and DYNC2H1 in an additional family.  Notably, these findings not only increase the broad spectrum of ciliar disorders, but suggest a correlation between the degree of defective microtubule or centriole elongation and organization and the severity of the resulting phenotype.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcROyYQ7SrTmgx0LsK7uFuY7fW6udTcc2eaHgwmQ_G_gZbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M7jtVemtQ%253D%253D&md5=cc069666f1d18413269e186ed574442a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.ajhg.2010.12.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ajhg.2010.12.004%26sid%3Dliteratum%253Aachs%26aulast%3DThiel%26aufirst%3DC.%26aulast%3DKessler%26aufirst%3DK.%26aulast%3DGiessl%26aufirst%3DA.%26aulast%3DDimmler%26aufirst%3DA.%26aulast%3DShalev%26aufirst%3DS.%2BA.%26aulast%3Dvon%2Bder%2BHaar%26aufirst%3DS.%26aulast%3DZenker%26aufirst%3DM.%26aulast%3DZahnleiter%26aufirst%3DD.%26aulast%3DSt%25C3%25B6ss%26aufirst%3DH.%26aulast%3DBeinder%26aufirst%3DE.%26aulast%3DAbou%2BJamra%26aufirst%3DR.%26aulast%3DEkici%26aufirst%3DA.%2BB.%26aulast%3DSchr%25C3%25B6der-Kre%25C3%259F%26aufirst%3DN.%26aulast%3DAigner%26aufirst%3DT.%26aulast%3DKirchner%26aufirst%3DT.%26aulast%3DReis%26aufirst%3DA.%26aulast%3DBrandst%25C3%25A4tter%26aufirst%3DJ.%2BH.%26aulast%3DRauch%26aufirst%3DA.%26atitle%3DNEK1%2520mutations%2520cause%2520short-rib%2520polydactyly%2520syndrome%2520type%2520Majewski%26jtitle%3DAm.%2520J.%2520Hum.%2520Genet.%26date%3D2011%26volume%3D88%26spage%3D106%26epage%3D114%26doi%3D10.1016%2Fj.ajhg.2010.12.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monroe, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kappen, I. F. P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokman, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terhal, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den
Boogaard, M.-J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savelberg, S. M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Veken, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Es, R. J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lens, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hengeveld, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creton, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janssen, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mink van der Molen, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebbeling, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giles, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoers, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Haaften, G.</span></span> <span> </span><span class="NLM_article-title">Compound heterozygous NEK1 variants in two siblings with oral-facial-digital syndrome type II (Mohr syndrome)</span>. <i>Eur. J. Hum. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1752</span>– <span class="NLM_lpage">1760</span>, <span class="refDoi"> DOI: 10.1038/ejhg.2016.103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1038%2Fejhg.2016.103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=27530628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtl2mt7nF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=1752-1760&author=G.+R.+Monroeauthor=I.+F.+P.+M.+Kappenauthor=M.+F.+Stokmanauthor=P.+A.+Terhalauthor=M.-J.+H.+van+den%0ABoogaardauthor=S.+M.+C.+Savelbergauthor=L.+T.+van+der+Vekenauthor=R.+J.+J.+van+Esauthor=S.+M.+Lensauthor=R.+C.+Hengeveldauthor=M.+A.+Cretonauthor=N.+G.+Janssenauthor=A.+B.+Mink+van+der+Molenauthor=M.+B.+Ebbelingauthor=R.+H.+Gilesauthor=N.+V.+Knoersauthor=G.+van+Haaften&title=Compound+heterozygous+NEK1+variants+in+two+siblings+with+oral-facial-digital+syndrome+type+II+%28Mohr+syndrome%29&doi=10.1038%2Fejhg.2016.103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Compound heterozygous NEK1 variants in two siblings with oral-facial-digital syndrome type II (Mohr syndrome)</span></div><div class="casAuthors">Monroe, Glen R.; Kappen, Isabelle F. P. M.; Stokman, Marijn F.; Terhal, Paulien A.; van den Boogaard, Marie-Jose H.; Savelberg, Sanne M. C.; van der Veken, Lars T.; van Es, Robert J. J.; Lens, Susanne M.; Hengeveld, Rutger C.; Creton, Marijn A.; Janssen, Nard G.; Mink van der Molen, Aebele B.; Ebbeling, Michelle B.; Giles, Rachel H.; Knoers, Nine V.; van Haaften, Gijs</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Human Genetics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1752-1760</span>CODEN:
                <span class="NLM_cas:coden">EJHGEU</span>;
        ISSN:<span class="NLM_cas:issn">1018-4813</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The oral-facial-digital (OFD) syndromes comprise a group of related disorders with a combination of oral, facial and digital anomalies.  Variants in several ciliary genes have been assocd. with subtypes of OFD syndrome, yet in most OFD patients the underlying cause remains unknown.  We investigated the mol. basis of disease in two brothers with OFD type II, Mohr syndrome, by performing single-nucleotide polymorphism (SNP)-array anal. on the brothers and their healthy parents to identify homozygous regions and candidate genes.  Subsequently, we performed whole-exome sequencing (WES) on the family.  Using WES, we identified compd. heterozygous variants c.[464G>C];[1226G>A] in NIMA (Never in Mitosis Gene A)-Related Kinase 1 (NEK1).  The novel variant c.464G>C disturbs normal splicing in an essential region of the kinase domain.  The nonsense variant c.1226G>A, p.  (Trp409*), results in nonsense-assocd. alternative splicing, removing the first coiled-coil domain of NEK1.  Candidate variants were confirmed with Sanger sequencing and alternative splicing assessed with cDNA anal.  Immunocytochem. was used to assess cilia no. and length.  Patient-derived fibroblasts showed severely reduced ciliation compared with control fibroblasts (18.0 vs 48.9%, P<0.0001), but showed no significant difference in cilia length.  In conclusion, we identified compd. heterozygous deleterious variants in NEK1 in two brothers with Mohr syndrome.  Ciliation in patient fibroblasts is drastically reduced, consistent with a ciliary defect pathogenesis.  Our results establish NEK1 variants involved in the etiol. of a subset of patients with OFD syndrome type II and support the consideration of including (routine) NEK1 anal. in patients suspected of OFD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-Su__4biVULVg90H21EOLACvtfcHk0lgqodpUvaft1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtl2mt7nF&md5=09709846bb36064895492990c81a7848</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fejhg.2016.103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fejhg.2016.103%26sid%3Dliteratum%253Aachs%26aulast%3DMonroe%26aufirst%3DG.%2BR.%26aulast%3DKappen%26aufirst%3DI.%2BF.%2BP.%2BM.%26aulast%3DStokman%26aufirst%3DM.%2BF.%26aulast%3DTerhal%26aufirst%3DP.%2BA.%26aulast%3Dvan%2Bden%2BBoogaard%26aufirst%3DM.-J.%2BH.%26aulast%3DSavelberg%26aufirst%3DS.%2BM.%2BC.%26aulast%3Dvan%2Bder%2BVeken%26aufirst%3DL.%2BT.%26aulast%3Dvan%2BEs%26aufirst%3DR.%2BJ.%2BJ.%26aulast%3DLens%26aufirst%3DS.%2BM.%26aulast%3DHengeveld%26aufirst%3DR.%2BC.%26aulast%3DCreton%26aufirst%3DM.%2BA.%26aulast%3DJanssen%26aufirst%3DN.%2BG.%26aulast%3DMink%2Bvan%2Bder%2BMolen%26aufirst%3DA.%2BB.%26aulast%3DEbbeling%26aufirst%3DM.%2BB.%26aulast%3DGiles%26aufirst%3DR.%2BH.%26aulast%3DKnoers%26aufirst%3DN.%2BV.%26aulast%3Dvan%2BHaaften%26aufirst%3DG.%26atitle%3DCompound%2520heterozygous%2520NEK1%2520variants%2520in%2520two%2520siblings%2520with%2520oral-facial-digital%2520syndrome%2520type%2520II%2520%2528Mohr%2520syndrome%2529%26jtitle%3DEur.%2520J.%2520Hum.%2520Genet.%26date%3D2016%26volume%3D24%26spage%3D1752%26epage%3D1760%26doi%3D10.1038%2Fejhg.2016.103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brenner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieland, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weydt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Böhm, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lulé, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hübers, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuwirth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borck, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wahlqvist, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danzer, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volk, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meitinger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strom, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kassubek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludolph, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weishaupt, J. H.</span></span> <span> </span><span class="NLM_article-title">NEK1 mutations in familial amyotrophic lateral sclerosis</span>. <i>Brain</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>139</i></span>, <span class="NLM_elocation-id">e28</span> <span class="refDoi"> DOI: 10.1093/brain/aww033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1093%2Fbrain%2Faww033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=26945885" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2016&author=D.+Brennerauthor=K.+M%C3%BCllerauthor=T.+Wielandauthor=P.+Weydtauthor=S.+B%C3%B6hmauthor=D.+Lul%C3%A9author=A.+H%C3%BCbersauthor=C.+Neuwirthauthor=M.+Weberauthor=G.+Borckauthor=M.+Wahlqvistauthor=K.+M.+Danzerauthor=A.+E.+Volkauthor=T.+Meitingerauthor=T.+M.+Stromauthor=M.+Ottoauthor=J.+Kassubekauthor=A.+C.+Ludolphauthor=P.+M.+Andersenauthor=J.+H.+Weishaupt&title=NEK1+mutations+in+familial+amyotrophic+lateral+sclerosis&doi=10.1093%2Fbrain%2Faww033"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Fbrain%2Faww033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbrain%252Faww033%26sid%3Dliteratum%253Aachs%26aulast%3DBrenner%26aufirst%3DD.%26aulast%3DM%25C3%25BCller%26aufirst%3DK.%26aulast%3DWieland%26aufirst%3DT.%26aulast%3DWeydt%26aufirst%3DP.%26aulast%3DB%25C3%25B6hm%26aufirst%3DS.%26aulast%3DLul%25C3%25A9%26aufirst%3DD.%26aulast%3DH%25C3%25BCbers%26aufirst%3DA.%26aulast%3DNeuwirth%26aufirst%3DC.%26aulast%3DWeber%26aufirst%3DM.%26aulast%3DBorck%26aufirst%3DG.%26aulast%3DWahlqvist%26aufirst%3DM.%26aulast%3DDanzer%26aufirst%3DK.%2BM.%26aulast%3DVolk%26aufirst%3DA.%2BE.%26aulast%3DMeitinger%26aufirst%3DT.%26aulast%3DStrom%26aufirst%3DT.%2BM.%26aulast%3DOtto%26aufirst%3DM.%26aulast%3DKassubek%26aufirst%3DJ.%26aulast%3DLudolph%26aufirst%3DA.%2BC.%26aulast%3DAndersen%26aufirst%3DP.%2BM.%26aulast%3DWeishaupt%26aufirst%3DJ.%2BH.%26atitle%3DNEK1%2520mutations%2520in%2520familial%2520amyotrophic%2520lateral%2520sclerosis%26jtitle%3DBrain%26date%3D2016%26volume%3D139%26doi%3D10.1093%2Fbrain%2Faww033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Higelin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catanese, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semelink-Sedlacek, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeztuerk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lutz, A.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bausinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speit, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludolph, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demestre, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeckers, T. M.</span></span> <span> </span><span class="NLM_article-title">NEK1 loss-of-function mutation induces DNA damage accumulation in ALS patient-derived motoneurons</span>. <i>Stem Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">150</span>– <span class="NLM_lpage">162</span>, <span class="refDoi"> DOI: 10.1016/j.scr.2018.06.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.scr.2018.06.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=29929116" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFKgsLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2018&pages=150-162&author=J.+Higelinauthor=A.+Cataneseauthor=L.+L.+Semelink-Sedlacekauthor=S.+Oeztuerkauthor=A.-K.+Lutzauthor=J.+Bausingerauthor=G.+Barbiauthor=G.+Speitauthor=P.+M.+Andersenauthor=A.+C.+Ludolphauthor=M.+Demestreauthor=T.+M.+Boeckers&title=NEK1+loss-of-function+mutation+induces+DNA+damage+accumulation+in+ALS+patient-derived+motoneurons&doi=10.1016%2Fj.scr.2018.06.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">NEK1 loss-of-function mutation induces DNA damage accumulation in ALS patient-derived motoneurons</span></div><div class="casAuthors">Higelin, Julia; Catanese, Alberto; Semelink-Sedlacek, Lena Luisa; Oeztuerk, Sertap; Lutz, Anne-Kathrin; Bausinger, Julia; Barbi, Gotthold; Speit, Guenter; Andersen, Peter M.; Ludolph, Albert C.; Demestre, Maria; Boeckers, Tobias M.</div><div class="citationInfo"><span class="NLM_cas:title">Stem Cell Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">150-162</span>CODEN:
                <span class="NLM_cas:coden">SCRTDJ</span>;
        ISSN:<span class="NLM_cas:issn">1873-5061</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Mutations in genes coding for proteins involved in DNA damage response (DDR) and repair, such as C9orf72 and FUS (Fused in Sarcoma), are assocd. with neurodegenerative diseases and lead to amyotrophic lateral sclerosis (ALS).  Heterozygous loss-of-function mutations in NEK1 (NIMA-related kinase 1) have also been recently found to cause ALS.  NEK1 codes for a multifunctional protein, crucially involved in mitotic checkpoint control and DDR.  To resolve pathol. alterations assocd. with NEK1 mutation, we compared hiPSC-derived motoneurons carrying a NEK1 mutation with mutant C9orf72 and wild type neurons at basal level and after DNA damage induction.  Motoneurons carrying a C9orf72 mutation exhibited cell specific signs of increased DNA damage.  This phenotype was even more severe in NEK1c.2434A gT neurons that showed significantly increased DNA damage at basal level and impaired DDR after induction of DNA damage in an maturation-dependent manner.  Our results provide first mechanistic insight in pathophysiol. alterations induced by NEK1 mutations and point to a converging pathomechanism of different gene mutations causative for ALS.  Therefore, our study contributes to the development of novel therapeutic strategies to reduce DNA damage accumulation in neurodegenerative diseases and ALS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOEC_t3JzYX7Vg90H21EOLACvtfcHk0lgqodpUvaft1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFKgsLbL&md5=36ef39e4c9867350a97f592763cce762</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.scr.2018.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.scr.2018.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DHigelin%26aufirst%3DJ.%26aulast%3DCatanese%26aufirst%3DA.%26aulast%3DSemelink-Sedlacek%26aufirst%3DL.%2BL.%26aulast%3DOeztuerk%26aufirst%3DS.%26aulast%3DLutz%26aufirst%3DA.-K.%26aulast%3DBausinger%26aufirst%3DJ.%26aulast%3DBarbi%26aufirst%3DG.%26aulast%3DSpeit%26aufirst%3DG.%26aulast%3DAndersen%26aufirst%3DP.%2BM.%26aulast%3DLudolph%26aufirst%3DA.%2BC.%26aulast%3DDemestre%26aufirst%3DM.%26aulast%3DBoeckers%26aufirst%3DT.%2BM.%26atitle%3DNEK1%2520loss-of-function%2520mutation%2520induces%2520DNA%2520damage%2520accumulation%2520in%2520ALS%2520patient-derived%2520motoneurons%26jtitle%3DStem%2520Cell%2520Res.%26date%3D2018%26volume%3D30%26spage%3D150%26epage%3D162%26doi%3D10.1016%2Fj.scr.2018.06.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fry, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O'Regan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabir, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span> <span> </span><span class="NLM_article-title">Cell cycle regulation by the NEK family of protein kinases</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">4423</span>– <span class="NLM_lpage">4433</span>, <span class="refDoi"> DOI: 10.1242/jcs.111195</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1242%2Fjcs.111195" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=23132929" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjsl2gtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2012&pages=4423-4433&author=A.+M.+Fryauthor=L.+O%27Reganauthor=S.+R.+Sabirauthor=R.+Bayliss&title=Cell+cycle+regulation+by+the+NEK+family+of+protein+kinases&doi=10.1242%2Fjcs.111195"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Cell cycle regulation by the NEK family of protein kinases</span></div><div class="casAuthors">Fry, Andrew M.; O'Regan, Laura; Sabir, Sarah R.; Bayliss, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4423-4433</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">A review.  Genetic screens for cell division cycle mutants in the filamentous fungus Aspergillus nidulans led to the discovery of never-in-mitosis A (NIMA) kinase, a serine/threonine kinase that is required for mitotic entry.  Since that discovery, NIMA-related kinases, or NEKs, have been identified in most eukaryotes, including humans where 11 genetically distinct proteins named NEK1 to NEK11 are expressed.  Although there is no evidence that human NEKs are essential for mitotic entry, it is clear that several NEK family members play important roles in cell cycle control.  In particular, NEK2, NEK6, NEK7, and NEK9 contribute to the establishment of the microtubule-based mitotic spindle, whereas NEK1, NEK10, and NEK11 have been implicated in the DNA damage response.  Roles for NEKs in other aspects of mitotic progression, such as chromatin condensation, nuclear envelope breakdown, spindle assembly checkpoint signaling, and cytokinesis have also been proposed.  Interestingly, NEK1 and NEK8 also function within cilia, the microtubule-based structures that are nucleated from basal bodies.  This has led to the current hypothesis that NEKs have evolved to coordinate microtubule-dependent processes in both dividing and non-dividing cells.  Here, the authors review the functions of human NEKs, with particular emphasis on those family members that are involved in cell cycle control, and consider their potential as therapeutic targets in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKCk9miADRfLVg90H21EOLACvtfcHk0ljptvnUV7TElw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjsl2gtA%253D%253D&md5=6965616ca443eeeb40fb320bea94bd36</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1242%2Fjcs.111195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.111195%26sid%3Dliteratum%253Aachs%26aulast%3DFry%26aufirst%3DA.%2BM.%26aulast%3DO%2527Regan%26aufirst%3DL.%26aulast%3DSabir%26aufirst%3DS.%2BR.%26aulast%3DBayliss%26aufirst%3DR.%26atitle%3DCell%2520cycle%2520regulation%2520by%2520the%2520NEK%2520family%2520of%2520protein%2520kinases%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2012%26volume%3D125%26spage%3D4423%26epage%3D4433%26doi%3D10.1242%2Fjcs.111195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Polci, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">NIMA-related protein kinase 1 is involved early in the ionizing radiation-induced DNA damage response</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">8800</span>– <span class="NLM_lpage">8803</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-04-2243</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1158%2F0008-5472.CAN-04-2243" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=15604234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOms7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2004&pages=8800-8803&author=R.+Polciauthor=A.+Pengauthor=P.-L.+Chenauthor=D.+J.+Rileyauthor=Y.+Chen&title=NIMA-related+protein+kinase+1+is+involved+early+in+the+ionizing+radiation-induced+DNA+damage+response&doi=10.1158%2F0008-5472.CAN-04-2243"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">NIMA-Related Protein Kinase 1 Is Involved Early in the Ionizing Radiation-Induced DNA Damage Response</span></div><div class="casAuthors">Polci, Rosaria; Peng, Aimin; Chen, Phang-Lang; Riley, Daniel J.; Chen, Yumay</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8800-8803</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Cellular functions of the NimA-related mammalian kinase Nek1 have not been demonstrated to date.  Here we show that Nek1 is involved early in the DNA damage response induced by ionizing radiation (IR) and that Nek1 is important for cells to repair and recover from DNA damage.  When primary or transformed cells are exposed to IR, Nek1 kinase activity is increased within 4 min, and Nek1 expression is up-regulated shortly thereafter and sustained for hours.  At the same early time frame after IR that its kinase activity is highest, a portion of Nek1 redistributes in cells from cytoplasm to discrete nuclear foci at sites of DNA double-strand breaks.  There it colocalizes with γ-H2AX and NFBD1/MDC1, two key proteins involved very early in the response to IR-induced DNA double-strand breaks.  Finally, Nek1-deficient fibroblasts are much more sensitive to the effects of IR-induced DNA damage than otherwise identical fibroblasts expressing Nek1.  These results suggest that Nek1 may function as a kinase early in the DNA damage response pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrq1l6-l3mMqrVg90H21EOLACvtfcHk0ljptvnUV7TElw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOms7bL&md5=efa8478012fcbe045e6cf22bf9d394b6</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-04-2243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-04-2243%26sid%3Dliteratum%253Aachs%26aulast%3DPolci%26aufirst%3DR.%26aulast%3DPeng%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DP.-L.%26aulast%3DRiley%26aufirst%3DD.%2BJ.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DNIMA-related%2520protein%2520kinase%25201%2520is%2520involved%2520early%2520in%2520the%2520ionizing%2520radiation-induced%2520DNA%2520damage%2520response%26jtitle%3DCancer%2520Res.%26date%3D2004%26volume%3D64%26spage%3D8800%26epage%3D8803%26doi%3D10.1158%2F0008-5472.CAN-04-2243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grudzenski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raths, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conrad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rübe, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Löbrich, M.</span></span> <span> </span><span class="NLM_article-title">Inducible response required for repair of low-dose radiation damage in human fibroblasts</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">14205</span>– <span class="NLM_lpage">14210</span>, <span class="refDoi"> DOI: 10.1073/pnas.1002213107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1073%2Fpnas.1002213107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=20660770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVCkt7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=14205-14210&author=S.+Grudzenskiauthor=A.+Rathsauthor=S.+Conradauthor=C.+E.+R%C3%BCbeauthor=M.+L%C3%B6brich&title=Inducible+response+required+for+repair+of+low-dose+radiation+damage+in+human+fibroblasts&doi=10.1073%2Fpnas.1002213107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Inducible response required for repair of low-dose radiation damage in human fibroblasts</span></div><div class="casAuthors">Grudzenski, Saskia; Raths, Antonia; Conrad, Sandro; Rube, Claudia E.; Lobrich, Markus</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">14205-14210, S14205/1-S14205/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Ionizing radiation (IR) induces a variety of DNA lesions among which DNA double-strand breaks (DSBs) are the biol. most significant.  It is currently unclear if DSB repair is equally efficient after low and high doses.  Here, we use γ-H2AX, phospho-ATM (pATM), and 53BP1 foci anal. to monitor DSB repair.  We show, consistent with a previous study, that the kinetics of γ-H2AX and pATM foci loss in confluent primary human fibroblasts are substantially compromised after doses of 10 mGy and lower.  Following 2.5 mGy, cells fail to show any foci loss.  Strikingly, cells pretreated with 10 μM H2O2 efficiently remove all γ-H2AX foci induced by 10 mGy.  At the concn. used, H2O2 produces single-strand breaks and base damages via the generation of oxygen radicals but no DSBs.  Moreover, 10 μM H2O2 up-regulates a set of genes that is also up-regulated after high (200 mGy) but not after low (10 mGy) radiation doses.  This suggests that low radical levels induce a response that is required for the repair of radiation-induced DSBs when the radiation damage is too low to cause the induction itself.  To address the in vivo significance of this finding, we established γ-H2AX and 53BP1 foci anal. in various mouse tissues.  Although mice irradiated with 100 mGy or 1 Gy show efficient γ-H2AX and 53BP1 foci removal during 24 h post-IR, barely any foci loss was obsd. after 10 mGy.  Our data suggest that the cellular response to DSBs is substantially different for low vs. high radiation doses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVaHxBRlpDFbVg90H21EOLACvtfcHk0li2TiPdIGL42g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVCkt7%252FP&md5=c1cf547306699f8cea631a43a84e0756</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1002213107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1002213107%26sid%3Dliteratum%253Aachs%26aulast%3DGrudzenski%26aufirst%3DS.%26aulast%3DRaths%26aufirst%3DA.%26aulast%3DConrad%26aufirst%3DS.%26aulast%3DR%25C3%25BCbe%26aufirst%3DC.%2BE.%26aulast%3DL%25C3%25B6brich%26aufirst%3DM.%26atitle%3DInducible%2520response%2520required%2520for%2520repair%2520of%2520low-dose%2520radiation%2520damage%2520in%2520human%2520fibroblasts%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D14205%26epage%3D14210%26doi%3D10.1073%2Fpnas.1002213107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spies, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waizenegger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heyer, W.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobrich, M.</span></span> <span> </span><span class="NLM_article-title">Nek1 regulates Rad54 to orchestrate homologous recombination and replication fork stability</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">903</span>– <span class="NLM_lpage">917</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2016.04.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.molcel.2016.04.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=27264870" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC28Xpt1entbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2016&pages=903-917&author=J.+Spiesauthor=A.+Waizeneggerauthor=O.+Bartonauthor=M.+Surderauthor=W.+D.+Wrightauthor=W.-D.+Heyerauthor=M.+Lobrich&title=Nek1+regulates+Rad54+to+orchestrate+homologous+recombination+and+replication+fork+stability&doi=10.1016%2Fj.molcel.2016.04.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Nek1 Regulates Rad54 to Orchestrate Homologous Recombination and Replication Fork Stability</span></div><div class="casAuthors">Spies, Julian; Waizenegger, Anja; Barton, Olivia; Suerder, Michael; Wright, William D.; Heyer, Wolf-Dietrich; Loebrich, Markus</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">903-917</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Never-in-mitosis A-related kinase 1 (Nek1) has established roles in apoptosis and cell cycle regulation.  We show that human Nek1 regulates homologous recombination (HR) by phosphorylating Rad54 at Ser572 in late G2 phase.  Nek1 deficiency as well as expression of unphosphorylatable Rad54 (Rad54-S572A) cause unresolved Rad51 foci and confer a defect in HR.  Phospho-mimic Rad54 (Rad54-S572E), in contrast, promotes HR and rescues the HR defect assocd. with Nek1 loss.  Although expression of phospho-mimic Rad54 is beneficial for HR, it causes Rad51 removal from chromatin and degrdn. of stalled replication forks in S phase.  Thus, G2-specific phosphorylation of Rad54 by Nek1 promotes Rad51 chromatin removal during HR in G2 phase, and its absence in S phase is required for replication fork stability.  In summary, Nek1 regulates Rad51 removal to orchestrate HR and replication fork stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUj-FGAj-B6rVg90H21EOLACvtfcHk0li2TiPdIGL42g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xpt1entbY%253D&md5=66566886a3cc28d7737dbaf963e4da13</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2016.04.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2016.04.032%26sid%3Dliteratum%253Aachs%26aulast%3DSpies%26aufirst%3DJ.%26aulast%3DWaizenegger%26aufirst%3DA.%26aulast%3DBarton%26aufirst%3DO.%26aulast%3DSurder%26aufirst%3DM.%26aulast%3DWright%26aufirst%3DW.%2BD.%26aulast%3DHeyer%26aufirst%3DW.-D.%26aulast%3DLobrich%26aufirst%3DM.%26atitle%3DNek1%2520regulates%2520Rad54%2520to%2520orchestrate%2520homologous%2520recombination%2520and%2520replication%2520fork%2520stability%26jtitle%3DMol.%2520Cell%26date%3D2016%26volume%3D62%26spage%3D903%26epage%3D917%26doi%3D10.1016%2Fj.molcel.2016.04.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D. J.</span></span> <span> </span><span class="NLM_article-title">Never-in-mitosis related kinase 1 functions in DNA damage response and checkpoint control</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">3194</span>– <span class="NLM_lpage">3201</span>, <span class="refDoi"> DOI: 10.4161/cc.7.20.6815</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.4161%2Fcc.7.20.6815" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=18843199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsV2mu7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=3194-3201&author=Y.+Chenauthor=P.-L.+Chenauthor=C.-F.+Chenauthor=X.+Jiangauthor=D.+J.+Riley&title=Never-in-mitosis+related+kinase+1+functions+in+DNA+damage+response+and+checkpoint+control&doi=10.4161%2Fcc.7.20.6815"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Never-in-mitosis related kinase 1 functions in DNA damage response and checkpoint control</span></div><div class="casAuthors">Chen, Yumay; Chen, Phang-Lang; Chen, Chi-Fen; Jiong, Xianzhi; Riley, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3194-3201</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Nek1, the 1st mammalian ortholog of the fungal protein kinase never in mitosis A, is involved early in the DNA damage sensing/repair pathway after ionizing radiation (IR).  Here we extend this finding by showing that Nek1 localizes to nuclear foci of DNA damage in response to many different types of damage in addn. to IR.  Untransformed cells established from kat2J/Nek1-/- mice fail to arrest properly at G1/S and M-phase checkpoints in response to DNA damage.  G1-S-phase checkpoint control can be rescued by ectopically overexpressing wild-type Nek1.  In Nek1-/- murine cells and in human cells with Nek1 expression silenced by siRNA, the checkpoint kinases Chk1 and Chk2 fail to be activated properly in response to ionizing or UV radiation.  In cells without functional Nek1, DNA is not repaired properly, double-stranded DNA breaks persist long after low dose IR, and excessive nos. of chromosome breaks are obsd.  These data show that Nek1 is important for efficient DNA damage checkpoint control and for proper DNA damage repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqBuaNvqTOXbVg90H21EOLACvtfcHk0li2TiPdIGL42g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsV2mu7fM&md5=5b37e281a263c733fed1de3aa86cb1f6</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.4161%2Fcc.7.20.6815&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.7.20.6815%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DP.-L.%26aulast%3DChen%26aufirst%3DC.-F.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DRiley%26aufirst%3DD.%2BJ.%26atitle%3DNever-in-mitosis%2520related%2520kinase%25201%2520functions%2520in%2520DNA%2520damage%2520response%2520and%2520checkpoint%2520control%26jtitle%3DCell%2520Cycle%26date%3D2008%26volume%3D7%26spage%3D3194%26epage%3D3201%26doi%3D10.4161%2Fcc.7.20.6815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.-L.</span></span> <span> </span><span class="NLM_article-title">Nek1 kinase functions in DNA damage response and checkpoint control through a pathway independent of ATM and ATR</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">655</span>– <span class="NLM_lpage">663</span>, <span class="refDoi"> DOI: 10.4161/cc.10.4.14814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.4161%2Fcc.10.4.14814" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=21301226" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkt1CksL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=655-663&author=Y.+Chenauthor=C.-F.+Chenauthor=D.+J.+Rileyauthor=P.-L.+Chen&title=Nek1+kinase+functions+in+DNA+damage+response+and+checkpoint+control+through+a+pathway+independent+of+ATM+and+ATR&doi=10.4161%2Fcc.10.4.14814"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Nek1 kinase functions in DNA damage response and checkpoint control through a pathway independent of ATM and ATR</span></div><div class="casAuthors">Chen, Yumay; Chen, Chi-Fen; Riley, Daniel J.; Chen, Phang-Lang</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">655-663</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Never-in-mitosis A related protein kinase 1 (Nek1) is involved early in a DNA damage sensing/repair pathway.  We have previously shown that cells without functional Nek1 fail to activate the more distal kinases Chk1 and Chk2 and fail to arrest properly at G1/S or M-phase checkpoints in response to DNA damage.  As a consequence, foci of damaged DNA in Nek1 null cells persist long after the instigating insult, and Nek1 null cells develop unstable chromosomes at a rate much higher than identically cultured wild-type cells.  Here we show that Nek1 functions independently of canonical DNA damage responses requiring the PI3 kinase-like proteins ATM and ATR.  Chem. inhibitors of ATM/ATR or mutation of the genes that encode them fail to alter the kinase activity of Nek1 or its localization to nuclear foci of DNA damage.  Moreover ATM and ATR activities, including the localization of the proteins to DNA damage sites and phosphorylation of early DNA damage response substrates, are intact in Nek1-/- murine cells and in human cells with Nek1 expression silenced by siRNA.  Our results demonstrate that Nek1 is important for proper checkpoint control and characterize for the first time a DNA damage response that does not directly involve one of the known upstream mediator kinases, ATM or ATR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohGiyVtPj3QbVg90H21EOLACvtfcHk0li0YuQ6uCzxHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkt1CksL4%253D&md5=2a14716ba3eb13e83a24fab6ce4dc8ea</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.4161%2Fcc.10.4.14814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.10.4.14814%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DC.-F.%26aulast%3DRiley%26aufirst%3DD.%2BJ.%26aulast%3DChen%26aufirst%3DP.-L.%26atitle%3DNek1%2520kinase%2520functions%2520in%2520DNA%2520damage%2520response%2520and%2520checkpoint%2520control%2520through%2520a%2520pathway%2520independent%2520of%2520ATM%2520and%2520ATR%26jtitle%3DCell%2520Cycle%26date%3D2011%26volume%3D10%26spage%3D655%26epage%3D663%26doi%3D10.4161%2Fcc.10.4.14814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craigen, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D. J.</span></span> <span> </span><span class="NLM_article-title">Nek1 regulates cell death and mitochondrial membrane permeability through phosphorylation of VDAC1</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">267</span>, <span class="refDoi"> DOI: 10.4161/cc.8.2.7551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.4161%2Fcc.8.2.7551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=19158487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvFSqtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=257-267&author=Y.+Chenauthor=W.+J.+Craigenauthor=D.+J.+Riley&title=Nek1+regulates+cell+death+and+mitochondrial+membrane+permeability+through+phosphorylation+of+VDAC1&doi=10.4161%2Fcc.8.2.7551"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Nek1 regulates cell death and mitochondrial membrane permeability through phosphorylation of VDAC1</span></div><div class="casAuthors">Chen, Yumay; Craigen, William J.; Riley, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">257-267</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Mammalian NIMA-related protein kinase 1 (Nek1) is important for keeping cells alive after DNA damage, but the mechanism by which injured cells die without functional Nek1 has not yet been demonstrated.  Here, the authors show that Nek1 regulates the pathway to mitochondrial cell death through phosphorylation of voltage-dependent anion channel 1 (VDAC1) on its Ser-193 residue.  Nek1 assocs. with VDAC1 in a yeast 2-hybrid system, as well as by glutathione S-transferase pull-down assays and by reciprocal immunopptn.  A portion of Nek1 in cells also localizes at mitochondria.  Ectopic expression of a kinase-dead Nek1 mutant resulted in cell death, which was immediately preceded by loss of Nek1-dependent VDAC1-Ser-193 phosphorylation.  UV irradn. of Nek1-deficient cells or silencing of endogenous Nek1 expression similarly resulted in the loss of the specific Ser-193 phosphorylation before cells died.  Nek1-deficient cells were characterized by exaggerated mitochondrial membrane permeability (MMP) and accelerated cell death.  Ectopic expression of a VDAC1-S193A mutant, which could not be phosphorylated by Nek1, also resulted in cell death.  A VDAC1-S193E mutant, designed to mimic constitutive phosphorylation by Nek1, rescued exaggerated MMP and kept cells alive after DNA damaging injury, but only transiently.  Thus, the direct interaction between Nek1 and VDAC1 provides a mechanism to explain how Nek1 prevents excessive cell death, as well as the 1st direct evidence that a specific kinase regulates VDAC1 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqecVvyQbM_BbVg90H21EOLACvtfcHk0li0YuQ6uCzxHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvFSqtbg%253D&md5=06a242fcfc11ddb9607433b86a157a43</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.4161%2Fcc.8.2.7551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.8.2.7551%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DCraigen%26aufirst%3DW.%2BJ.%26aulast%3DRiley%26aufirst%3DD.%2BJ.%26atitle%3DNek1%2520regulates%2520cell%2520death%2520and%2520mitochondrial%2520membrane%2520permeability%2520through%2520phosphorylation%2520of%2520VDAC1%26jtitle%3DCell%2520Cycle%26date%3D2009%26volume%3D8%26spage%3D257%26epage%3D267%26doi%3D10.4161%2Fcc.8.2.7551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaczynska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osmulski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polci, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D. J.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation by Nek1 regulates opening and closing of voltage dependent anion channel 1</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>394</i></span>,  <span class="NLM_fpage">798</span>– <span class="NLM_lpage">803</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2010.03.077</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.bbrc.2010.03.077" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=20230784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC3cXks1Kitrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=394&publication_year=2010&pages=798-803&author=Y.+Chenauthor=M.+Gaczynskaauthor=P.+Osmulskiauthor=R.+Polciauthor=D.+J.+Riley&title=Phosphorylation+by+Nek1+regulates+opening+and+closing+of+voltage+dependent+anion+channel+1&doi=10.1016%2Fj.bbrc.2010.03.077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation by Nek1 regulates opening and closing of voltage dependent anion channel 1</span></div><div class="casAuthors">Chen, Yumay; Gaczynska, Maria; Osmulski, Pawel; Polci, Rosaria; Riley, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">394</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">798-803</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Voltage-dependent anion channel 1 (VDAC1) is a key component of the mitochondrial permeability transition pore.  To initiate apoptosis and certain other forms of cell death, mitochondria become permeable such that cytochrome c and other pre-apoptotic mols. resident inside the mitochondria enter the cytosol and activate apoptotic cascades.  The authors showed recently that VDAC1 interacts directly with never-in-mitosis A-related kinase 1 (Nek1), and that Nek1 phosphorylates VDAC1 on Ser-193 to prevent excessive cell death after injury.  How this phosphorylation regulates the activity of VDAC1, however, has not yet been reported.  Here, the authors used at. force microscopy (AFM) and cytochrome c conductance studies to examine the configuration of VDAC1 before and after phosphorylation by Nek1.  Wild-type VDAC1 assumes an open configuration, but closes and prevents cytochrome c efflux when phosphorylated by Nek1.  A VDAC1-S193A mutant, which could not be phosphorylated by Nek1 under identical conditions, remained open and constitutively allowed cytochrome c efflux.  Conversely, a VDAC1-S193E mutant, which mimics constitutive phosphorylation by Nek1, remained closed by AFM and prevented cytochrome c leakage in the same liposome assays.  Thus, these data provide a mechanism to explain how Nek1 regulates cell death by affecting the opening and closing of VDAC1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfr0zRey7Y-bVg90H21EOLACvtfcHk0li0YuQ6uCzxHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXks1Kitrg%253D&md5=34c91949652c63004253b0eccebb5def</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2010.03.077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2010.03.077%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DGaczynska%26aufirst%3DM.%26aulast%3DOsmulski%26aufirst%3DP.%26aulast%3DPolci%26aufirst%3DR.%26aulast%3DRiley%26aufirst%3DD.%2BJ.%26atitle%3DPhosphorylation%2520by%2520Nek1%2520regulates%2520opening%2520and%2520closing%2520of%2520voltage%2520dependent%2520anion%2520channel%25201%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2010%26volume%3D394%26spage%3D798%26epage%3D803%26doi%3D10.1016%2Fj.bbrc.2010.03.077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khalil, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Benedetti, A.</span></span> <span> </span><span class="NLM_article-title">The TLK1/Nek1 axis contributes to mitochondrial integrity and apoptosis prevention via phosphorylation of VDAC1</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">363</span>– <span class="NLM_lpage">375</span>, <span class="refDoi"> DOI: 10.1080/15384101.2019.1711317</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1080%2F15384101.2019.1711317" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=31914854" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlsVKitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2020&pages=363-375&author=V.+Singhauthor=M.+I.+Khalilauthor=A.+De+Benedetti&title=The+TLK1%2FNek1+axis+contributes+to+mitochondrial+integrity+and+apoptosis+prevention+via+phosphorylation+of+VDAC1&doi=10.1080%2F15384101.2019.1711317"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The TLK1/Nek1 axis contributes to mitochondrial integrity and apoptosis prevention via phosphorylation of VDAC1</span></div><div class="casAuthors">Singh, Vibha; Khalil, Md. Imtiaz; De Benedetti, Arrigo</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">363-375</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1551-4005</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">The TLK1/Nek1 axis contributes to cell cycle arrest and implementation of the DDR to mediate survival upon DNA damage.  However, when the damage is too severe, the cells typically are forced into apoptosis, and the contribution of TLKs in this process has not been investigated.  In contrast, it is known that Nek1 may play a role by phosphorylating VDAC1 maintaining proper opening and closure of the channel and thus mitochondrial integrity.  We now show that the activating phosphorylation of Nek1-T141 by TLK1 contributes to the phosphorylation and stability of VDAC1 and thereby to mitochondrial permeability and integrity.  Treatment of three different cell lines model that overexpress Nek1-T141A mutant with doxorubicin showed exquisite sensitivity to the drug, with implementation of rapid accumulation of cells with subG1 DNA content (apoptotic) and other alterations in the cell cycle.  In addn., these cells displayed reduced oxygen consumption under normal conditions and less reliance on mitochondria and more dependence on glycolysis for energy prodn.  Consistent with greater apoptosis, upon treatment with low doses of doxorubicin, cells overexpressing Nek1-T141A displayed leakage of Cyt-C into the cytoplasmic fraction.  This suggests that inhibiting the TLK1/Nek1/VDAC1 nexus could sensitize cancer cells to apoptotic killing in combination with an appropriate DNA damaging agent.  We in fact have previously reported that Nek1 expression is elevated in advanced Prostate Cancer (PCa) and we now report that VDAC1 expression is elevated and correlated with disease stage, thereby making the TLK1/Nek1/VDAC1 nexus a very attractive target for PCa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxEwQEOU6NULVg90H21EOLACvtfcHk0liTSYOyAWbtPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlsVKitw%253D%253D&md5=d2f7766566b1c42a61a9ed1bf2fcf20d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1080%2F15384101.2019.1711317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15384101.2019.1711317%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DV.%26aulast%3DKhalil%26aufirst%3DM.%2BI.%26aulast%3DDe%2BBenedetti%26aufirst%3DA.%26atitle%3DThe%2520TLK1%252FNek1%2520axis%2520contributes%2520to%2520mitochondrial%2520integrity%2520and%2520apoptosis%2520prevention%2520via%2520phosphorylation%2520of%2520VDAC1%26jtitle%3DCell%2520Cycle%26date%3D2020%26volume%3D19%26spage%3D363%26epage%3D375%26doi%3D10.1080%2F15384101.2019.1711317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaiswal, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koul, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Benedetti, A.</span></span> <span> </span><span class="NLM_article-title">The TLK1-Nek1 axis promotes prostate cancer progression</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>453</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">141</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2019.03.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.canlet.2019.03.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=30928383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmvFCgsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=453&publication_year=2019&pages=131-141&author=V.+Singhauthor=P.+K.+Jaiswalauthor=I.+Ghoshauthor=H.+K.+Koulauthor=X.+Yuauthor=A.+De+Benedetti&title=The+TLK1-Nek1+axis+promotes+prostate+cancer+progression&doi=10.1016%2Fj.canlet.2019.03.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The TLK1-Nek1 axis promotes prostate cancer progression</span></div><div class="casAuthors">Singh, Vibha; Jaiswal, Praveen Kumar; Ghosh, Ishita; Koul, Hari K.; Yu, Xiuping; De Benedetti, Arrigo</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">453</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">131-141</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">We recently uncovered the crit. TLK1>NEK1>ATR > Chk1 axis in mediating the DDR and cell cycle checkpoint while transiting from Androgen Sensitive to Insensitive growth for LNCaP and TRAMP-C2 cells.  However, we did not know the generality of this pathway in PCa progression since there are few cell lines where the transition has been studied.  Furthermore, the identification of Nek1, and more importantly the TLK-mediated phosphorylation of T141, has never been studied in PCa biopsies.  We now report the first study of a PCa TMA of p-Nek1-T141 and correlation to the Gleason score.  In addn. we found that TRAMP mice treated with the TLK inhibitor, thioridazine (THD), following castration did not recover cancerous growth of their prostates.  Moreover, we recapitulated the process of translational increase in TLK1B expression in a naive PDX model that was established from an AR + adenocarcinoma.  Therefore, we believe that this TLK1-Nek1 mediated DDR axis is likely to be a common adaptive response during the transition of PCa cells toward androgen-insensitive growth, and hence CRPC progression, which has the potential to be targeted with THD and other TLK or Nek1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUOEFnPux-zbVg90H21EOLACvtfcHk0liTSYOyAWbtPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmvFCgsLw%253D&md5=48556b830b64b1c1e19cbde42947d3d4</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2019.03.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2019.03.041%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DV.%26aulast%3DJaiswal%26aufirst%3DP.%2BK.%26aulast%3DGhosh%26aufirst%3DI.%26aulast%3DKoul%26aufirst%3DH.%2BK.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DDe%2BBenedetti%26aufirst%3DA.%26atitle%3DThe%2520TLK1-Nek1%2520axis%2520promotes%2520prostate%2520cancer%2520progression%26jtitle%3DCancer%2520Lett.%26date%3D2019%26volume%3D453%26spage%3D131%26epage%3D141%26doi%3D10.1016%2Fj.canlet.2019.03.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaiswal, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koul, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Benedetti, A.</span></span> <span> </span><span class="NLM_article-title">Targeting the TLK1/NEK1 DDR axis with Thioridazine suppresses outgrowth of androgen independent prostate tumors</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>145</i></span>,  <span class="NLM_fpage">1055</span>– <span class="NLM_lpage">1067</span>, <span class="refDoi"> DOI: 10.1002/ijc.32200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1002%2Fijc.32200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=30737777" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjsFenu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=145&publication_year=2019&pages=1055-1067&author=V.+Singhauthor=P.+K.+Jaiswalauthor=I.+Ghoshauthor=H.+K.+Koulauthor=X.+Yuauthor=A.+De+Benedetti&title=Targeting+the+TLK1%2FNEK1+DDR+axis+with+Thioridazine+suppresses+outgrowth+of+androgen+independent+prostate+tumors&doi=10.1002%2Fijc.32200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the TLK1/NEK1 DDR axis with Thioridazine suppresses outgrowth of androgen independent prostate tumors</span></div><div class="casAuthors">Singh, Vibha; Jaiswal, Praveen Kumar; Ghosh, Ishita; Koul, Hari K.; Yu, Xiuping; De Benedetti, Arrigo</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">145</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1055-1067</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We now show that androgen deprivation therapy (ADT) of LNCaP cells results in increased expression of TLK1B, a key kinase upstream of NEK1 and ATR and mediating the DDR that typically results in a temporary cell cycle arrest of androgen responsive PCa cells.  Following DNA damage, addn. of the TLK specific inhibitor, thioridazine (THD), impairs ATR and Chk1 activation, establishing the existence of a ADT > TLK1 > NEK1 > ATR > Chk1, DDR pathway, while its abrogation leads to apoptosis.  Treatment with THD suppressed the outgrowth of androgen-independent (AI) colonies of LNCaP and TRAMP-C2 cells cultured with bicalutamide.  Moreover, THD significantly inhibited the growth of several PCa cells in vitro (including AI lines).  Administration of THD or bicalutamide was not effective at inhibiting long-term tumor growth of LNCaP xenografts.  In contrast, combination therapy remarkably inhibited tumor growth via bypass of the DDR.  Moreover, xenografts of LNCaP cells overexpressing a NEK1-T141A mutant were durably suppressed with bicalutamide.  Collectively, these results suggest that targeting the TLK1/NEK1 axis might be a novel therapy for PCa in combination with std. of care (ADT).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkKHEgGNvGzbVg90H21EOLACvtfcHk0liSuWzPQQ6ffg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjsFenu7o%253D&md5=417328fe7127cf68127e9b9191c7cd8d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fijc.32200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.32200%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DV.%26aulast%3DJaiswal%26aufirst%3DP.%2BK.%26aulast%3DGhosh%26aufirst%3DI.%26aulast%3DKoul%26aufirst%3DH.%2BK.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DDe%2BBenedetti%26aufirst%3DA.%26atitle%3DTargeting%2520the%2520TLK1%252FNEK1%2520DDR%2520axis%2520with%2520Thioridazine%2520suppresses%2520outgrowth%2520of%2520androgen%2520independent%2520prostate%2520tumors%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2019%26volume%3D145%26spage%3D1055%26epage%3D1067%26doi%3D10.1002%2Fijc.32200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Benedetti, A.</span></span> <span> </span><span class="NLM_article-title">Identification of the proteome complement of humanTLK1 reveals it binds and phosphorylates NEK1 regulating its activity</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">915</span>– <span class="NLM_lpage">926</span>, <span class="refDoi"> DOI: 10.1080/15384101.2017.1314421</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1080%2F15384101.2017.1314421" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=28426283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvVChurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=915-926&author=V.+Singhauthor=Z.+M.+Connellyauthor=X.+Shenauthor=A.+De+Benedetti&title=Identification+of+the+proteome+complement+of+humanTLK1+reveals+it+binds+and+phosphorylates+NEK1+regulating+its+activity&doi=10.1080%2F15384101.2017.1314421"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of the proteome complement of human TLK1 reveals it binds and phosphorylates NEK1 regulating its activity</span></div><div class="casAuthors">Singh, Vibha; Connelly, Zachary M.; Shen, Xinggui; De Benedetti, Arrigo</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">915-926</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1551-4005</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">The Tousled Like kinases (TLKs) are involved in numerous cellular functions, including the DNA Damage Response (DDR), but only a handful of substrates have been identified thus far.  Through a novel proteomic screen, we have now identified 165 human proteins interacting with TLK1, and we have focused this work on NEK1 because of its known role in the DDR, upstream of ATR and Chk1.  TLK1 and NEK1 were found to interact by coIP, and their binding is strengthened following exposure of cells to H2O2.  Following incubation with doxorubicin, TLK1 and NEK1 relocalize with nuclear repair foci along with γH2AX.  TLK1 phosphorylated NEK1 at T141, which lies in the kinase domain, and caused an increase in its activity.  Following DNA damage, addn. of the TLK1 inhibitor, THD, or overexpression of NEK1-T141A mutant impaired ATR and Chk1 activation, indicating the existence of a TLK1>NEK1>ATR>Chk1 pathway.  Indeed, overexpression of the NEK1-T141A mutant resulted in an altered cell cycle response after exposure of cells to oxidative stress, including bypass of G1 arrest and implementation of an intra S-phase checkpoint.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnk8vGEabv-bVg90H21EOLACvtfcHk0liSuWzPQQ6ffg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvVChurg%253D&md5=c5ff4681869a38739a0d7e0d27523efe</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1080%2F15384101.2017.1314421&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15384101.2017.1314421%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DV.%26aulast%3DConnelly%26aufirst%3DZ.%2BM.%26aulast%3DShen%26aufirst%3DX.%26aulast%3DDe%2BBenedetti%26aufirst%3DA.%26atitle%3DIdentification%2520of%2520the%2520proteome%2520complement%2520of%2520humanTLK1%2520reveals%2520it%2520binds%2520and%2520phosphorylates%2520NEK1%2520regulating%2520its%2520activity%26jtitle%3DCell%2520Cycle%26date%3D2017%26volume%3D16%26spage%3D915%26epage%3D926%26doi%3D10.1080%2F15384101.2017.1314421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Melo-Hanchuk, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martins, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunha, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassallo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobarg, J.</span></span> <span> </span><span class="NLM_article-title">Expression of the NEK family in normal and cancer tissue: an immunohistochemical study</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">23</span>, <span class="refDoi"> DOI: 10.1186/s12885-019-6408-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1186%2Fs12885-019-6408-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=31906878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjsl2htw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2020&pages=23&author=T.+D.+Melo-Hanchukauthor=M.+B.+Martinsauthor=L.+L.+Cunhaauthor=F.+A.+Soaresauthor=L.+S.+Wardauthor=J.+Vassalloauthor=J.+Kobarg&title=Expression+of+the+NEK+family+in+normal+and+cancer+tissue%3A+an+immunohistochemical+study&doi=10.1186%2Fs12885-019-6408-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of the NEK family in normal and cancer tissue: an immunohistochemical study</span></div><div class="casAuthors">Melo-Hanchuk, Talita Diniz; Martins, Mariana Bonjiorno; Cunha, Lucas Leite; Soares, Fernando Augusto; Ward, Laura Sterian; Vassallo, Jose; Kobarg, Jorg</div><div class="citationInfo"><span class="NLM_cas:title">BMC Cancer</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23</span>CODEN:
                <span class="NLM_cas:coden">BCMACL</span>;
        ISSN:<span class="NLM_cas:issn">1471-2407</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">The NEK serine/threonine protein kinases are involved in cell cycle checkpoints, DNA damage repair, and apoptosis.  Alterations in these pathways are frequently assocd. with cell malignant cellular transformations.  Thyroid cancer is the most common malignant tumor in the endocrine system.  Despite good treatment methods, the no. of cases has increased significantly in recent years.  Here, we studied the expression of NEK1, NEK2, NEK3, and NEK5 in different types of normal and malignant tissues, using tissue microarray anal., and identified NEKs as potential markers in thyroid malignancy.  The studied cases comprised multiple cancer tissue microarrays, including breast, colon, esophagus, kidney, lung, pancreas, prostate, stomach, thyroid and uterine cervix, as well as 281 patients who underwent thyroid resection for thyroid cancer or thyroid nodules.  The expression of NEK1, NEK2, NEK3, and NEK5 was analyzed by immunohistochem.  The expression pattern was evaluated in terms of intensity by two methods, semiquant. and quant., and was compared between normal and cancer tissue.  We analyzed the expression of each member of the NEK family in a tissue-dependent manner.  Compared to normal tissue, most of the evaluated proteins showed lower expression in lung tumor.  However, in the thyroid, the expression was higher in malignant tissue, esp. for NEK 1, NEK3 and NEK5.  Concerning characteristics of the thyroid tumor, such as aggressiveness, NEK1 expression was higher in tumors with multifocality and in patients with lymph node metastasis.  NEK3 expression was stronger in patients with stage II, that involved metastasis.  NEK5, on the other hand, showed high expression in patients with invasion and metastasis and in patients with tumor size > 4 cm.  Furthermore, this work, demonstrated for the first time a high specificity and sensitivity of over-expression of NEK1 in classical and follicular variants of papillary thyroid cancer and NEK3 in tall-cell papillary thyroid cancer.  Taken together, the NEK protein kinases emerge as important proteins in thyroid cancer development and may help to identify malignancy and aggressiveness features during diagnosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7IByQd0eQfLVg90H21EOLACvtfcHk0liSuWzPQQ6ffg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjsl2htw%253D%253D&md5=dce11d760e07b7d9be7a2936496d79bb</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1186%2Fs12885-019-6408-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12885-019-6408-4%26sid%3Dliteratum%253Aachs%26aulast%3DMelo-Hanchuk%26aufirst%3DT.%2BD.%26aulast%3DMartins%26aufirst%3DM.%2BB.%26aulast%3DCunha%26aufirst%3DL.%2BL.%26aulast%3DSoares%26aufirst%3DF.%2BA.%26aulast%3DWard%26aufirst%3DL.%2BS.%26aulast%3DVassallo%26aufirst%3DJ.%26aulast%3DKobarg%26aufirst%3DJ.%26atitle%3DExpression%2520of%2520the%2520NEK%2520family%2520in%2520normal%2520and%2520cancer%2520tissue%253A%2520an%2520immunohistochemical%2520study%26jtitle%3DBMC%2520Cancer%26date%3D2020%26volume%3D20%26spage%3D23%26doi%3D10.1186%2Fs12885-019-6408-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span> <span> </span><span class="NLM_article-title">Frequent Nek1 overexpression in human gliomas</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>476</i></span>,  <span class="NLM_fpage">522</span>– <span class="NLM_lpage">527</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2016.05.156</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.bbrc.2016.05.156" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=27251576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslWqtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=476&publication_year=2016&pages=522-527&author=J.+Zhuauthor=Y.+Caiauthor=P.+Liuauthor=W.+Zhao&title=Frequent+Nek1+overexpression+in+human+gliomas&doi=10.1016%2Fj.bbrc.2016.05.156"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Frequent Nek1 overexpression in human gliomas</span></div><div class="casAuthors">Zhu, Jun; Cai, Yu; Liu, Pin; Zhao, Weiguo</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">476</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">522-527</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Never in mitosis A (NIMA)-related kinase 1 (Nek1) regulates cell cycle progression to mitosis.  Its expression and potential functions in human gliomas have not been studied.  Here, our immunohistochem. (IHC) assay and Western blot assay results showed that Nek1 expression was significantly upregulated in fresh and paraffin-embedded human glioma tissues.  Its level in normal brain tissues was low.  Nek1 overexpression in human gliomas was correlated with the proliferation marker (Ki-67), tumor grade, Karnofsky performance scale (KPS) and more importantly, patients' poor survival.  Further studies showed that Nek1 expression level was also increased in multiple human glioma cell lines (U251-MG, U87-MG, U118, H4 and U373).  Significantly, siRNA-mediated knockdown of Nek1 inhibited glioma cell (U87-MG/U251-MG) growth.  Nek1 siRNA also sensitized U87-MG/U251-MG cells to temozolomide (TMZ), causing a profound apoptosis induction and growth inhibition.  The current study indicates Nek1 might be a novel and valuable oncotarget of glioma, it is important for glioma cell growth and TMZ-resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6Hw3K3vHISLVg90H21EOLACvtfcHk0lgtD6gdz0PtoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslWqtbc%253D&md5=52b7149201a0891a0f95b21499a23edd</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2016.05.156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2016.05.156%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DZhao%26aufirst%3DW.%26atitle%3DFrequent%2520Nek1%2520overexpression%2520in%2520human%2520gliomas%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2016%26volume%3D476%26spage%3D522%26epage%3D527%26doi%3D10.1016%2Fj.bbrc.2016.05.156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polci, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.-L.</span></span> <span> </span><span class="NLM_article-title">Increased Nek1 expression in renal cell carcinoma cells is associated with decreased sensitivity to DNA-damaging treatment</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">4283</span>– <span class="NLM_lpage">4294</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.18632%2Foncotarget.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=24970796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A280%3ADC%252BC2cfntFSnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=4283-4294&author=Y.+Chenauthor=C.-F.+Chenauthor=R.+Polciauthor=R.+Weiauthor=D.+J.+Rileyauthor=P.-L.+Chen&title=Increased+Nek1+expression+in+renal+cell+carcinoma+cells+is+associated+with+decreased+sensitivity+to+DNA-damaging+treatment&doi=10.18632%2Foncotarget.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Increased Nek1 expression in renal cell carcinoma cells is associated with decreased sensitivity to DNA-damaging treatment</span></div><div class="casAuthors">Chen Yumay; Chen Chi-Fen; Wei Randy; Chen Phang-Lang; Polci Rosaria; Riley Daniel J</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4283-94</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Renal cell carcinoma (RCC) is a heterogeneous disease with resistance to systemic chemotherapy.  Elevated expression of multiple drug resistance (MDR) has been suggested to be one of the mechanisms for this resistance.  Here, we provide an alternative mechanism to explain RCC's resistance to chemotherapy-induced apoptosis.  Never-in mitosis A-related protein kinase 1 (Nek1) plays an important role in DNA damage response and proper checkpoint activation.  The association of Nek1 with the voltage-dependent anion channel (VDAC1) is a critical determinant of cell survival following DNA-damaging treatment.  We report here that Nek1 is highly expressed in RCC tumor and cultured RCC cells compared to that of normal renal tubular epithelial cells (RTE).  The association between Nek1 and VDAC1 is genotoxic dependent: prolonged Nek1/VDAC1 dissociation will lead to VDAC1 dephosphorylation and initiate apoptosis.  Down-regulation of Nek1 expression in RCC cells enhanced their sensitivity to DNA-damaging treatment.  Collectively, these results suggest that the increased Nek1 expression in RCC cells maintain persistent VDAC1 phosphorylation, closing its channel and preventing the onset of apoptosis under genotoxic insults.  Based on these results, we believe that Nek1 can serve as a potential therapeutic target for drug development in the treatment of RCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTkcXTsKbMaOIIRAdBLKY41fW6udTcc2eZy57CWFpn-37ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cfntFSnsw%253D%253D&md5=82dcd4a0048660d34dcf8d6c695bd367</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.2005%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DC.-F.%26aulast%3DPolci%26aufirst%3DR.%26aulast%3DWei%26aufirst%3DR.%26aulast%3DRiley%26aufirst%3DD.%2BJ.%26aulast%3DChen%26aufirst%3DP.-L.%26atitle%3DIncreased%2520Nek1%2520expression%2520in%2520renal%2520cell%2520carcinoma%2520cells%2520is%2520associated%2520with%2520decreased%2520sensitivity%2520to%2520DNA-damaging%2520treatment%26jtitle%3DOncotarget%26date%3D2014%26volume%3D5%26spage%3D4283%26epage%3D4294%26doi%3D10.18632%2Foncotarget.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">Overexpression of Nek1 in renal cell carcinoma protects against cell death from ionizing radiation</span>. <i>Int. J. Radiat. Oncol., Biol., Phys.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">S712</span>, <span class="refDoi"> DOI: 10.1016/j.ijrobp.2011.06.1364</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.ijrobp.2011.06.1364" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=20932667" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2011&pages=S712&author=R.+L.+Weiauthor=Y.+Chen&title=Overexpression+of+Nek1+in+renal+cell+carcinoma+protects+against+cell+death+from+ionizing+radiation&doi=10.1016%2Fj.ijrobp.2011.06.1364"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.ijrobp.2011.06.1364&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijrobp.2011.06.1364%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DR.%2BL.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DOverexpression%2520of%2520Nek1%2520in%2520renal%2520cell%2520carcinoma%2520protects%2520against%2520cell%2520death%2520from%2520ionizing%2520radiation%26jtitle%3DInt.%2520J.%2520Radiat.%2520Oncol.%252C%2520Biol.%252C%2520Phys.%26date%3D2011%26volume%3D81%26spage%3DS712%26doi%3D10.1016%2Fj.ijrobp.2011.06.1364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Freund, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hehlgans, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ensminger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fokas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rödel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Löbrich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rödel, F.</span></span> <span> </span><span class="NLM_article-title">Fractionation-dependent radiosensitization by molecular targeting of Nek1</span>. <i>Cells</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1235</span>, <span class="refDoi"> DOI: 10.3390/cells9051235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.3390%2Fcells9051235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1Sku7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&pages=1235&author=I.+Freundauthor=S.+Hehlgansauthor=D.+Martinauthor=M.+Ensmingerauthor=E.+Fokasauthor=C.+R%C3%B6delauthor=M.+L%C3%B6brichauthor=F.+R%C3%B6del&title=Fractionation-dependent+radiosensitization+by+molecular+targeting+of+Nek1&doi=10.3390%2Fcells9051235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Fractionation-dependent radiosensitization by molecular targeting of Nek1</span></div><div class="casAuthors">Freund, Isabel; Hehlgans, Stephanie; Martin, Daniel; Ensminger, Michael; Fokas, Emmanouil; Roedel, Claus; Loebrich, Markus; Roedel, Franz</div><div class="citationInfo"><span class="NLM_cas:title">Cells</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1235</span>CODEN:
                <span class="NLM_cas:coden">CELLC6</span>;
        ISSN:<span class="NLM_cas:issn">2073-4409</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">NIMA (never-in-mitosis gene A)-related kinase 1 (Nek1) is shown to impact on different cellular pathways such as DNA repair, checkpoint activation, and apoptosis.  Its role as a mol. target for radiation sensitization of malignant cells, however, remains elusive.  Stably transduced doxycycline (Dox)-inducible Nek1 shRNA HeLa cervix and siRNA-transfected HCT-15 colorectal carcinoma cells were irradiated in vitro and 3D clonogenic radiation survival, residual DNA damage, cell cycle distribution, and apoptosis were analyzed.  Nek1 knockdown (KD) sensitized both cell lines to ionizing radiation following a single dose irradn. and more pronounced in combination with a 6 h fractionation (3 x 2 Gy) regime.  For preclin. analyses we focused on cervical cancer.  Nek1 shRNA HeLa cells were grafted into NOD/SCID/IL-2Rγc-/- (NSG) mice and Nek1 KD was induced by Dox-infused drinking water resulting in a significant cytostatic effect if combined with a 6 h fractionation (3 x 2 Gy) regime.  In addn., we correlated Nek1 expression in biopsies of patients with cervical cancer with histopathol. parameters and clin. follow-up.  Our results indicate that elevated levels of Nek1 were assocd. with an increased rate of local or distant failure, as well as with impaired cancer-specific and overall survival in univariate analyses and for most endpoints in multivariable analyses.  Finally, findings from The Cancer Genome Atlas (TCGA) validation cohort confirmed a significant assocn. of high Nek1 expression with a reduced disease-free survival.  In conclusion, we consider Nek1 to represent a novel biomarker and potential therapeutic target for drug development in the context of optimized fractionation intervals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH2xPGDJXPObVg90H21EOLACvtfcHk0lge6d66zuoFjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1Sku7jO&md5=292b4ae41d1a19d25b24418cddca3980</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.3390%2Fcells9051235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcells9051235%26sid%3Dliteratum%253Aachs%26aulast%3DFreund%26aufirst%3DI.%26aulast%3DHehlgans%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DD.%26aulast%3DEnsminger%26aufirst%3DM.%26aulast%3DFokas%26aufirst%3DE.%26aulast%3DR%25C3%25B6del%26aufirst%3DC.%26aulast%3DL%25C3%25B6brich%26aufirst%3DM.%26aulast%3DR%25C3%25B6del%26aufirst%3DF.%26atitle%3DFractionation-dependent%2520radiosensitization%2520by%2520molecular%2520targeting%2520of%2520Nek1%26jtitle%3DCells%26date%3D2020%26volume%3D9%26spage%3D1235%26doi%3D10.3390%2Fcells9051235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">NEK1 protein kinase as a target for anticancer therapeutics</span>. <i>Chemotherapy: Open Access</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">e118</span>, <span class="refDoi"> DOI: 10.4172/2167-7700.1000e118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.4172%2F2167-7700.1000e118" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2012&pages=e118&author=Y.+Chen&title=NEK1+protein+kinase+as+a+target+for+anticancer+therapeutics&doi=10.4172%2F2167-7700.1000e118"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.4172%2F2167-7700.1000e118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4172%252F2167-7700.1000e118%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26atitle%3DNEK1%2520protein%2520kinase%2520as%2520a%2520target%2520for%2520anticancer%2520therapeutics%26jtitle%3DChemotherapy%253A%2520Open%2520Access%26date%3D2012%26volume%3D1%26spage%3De118%26doi%3D10.4172%2F2167-7700.1000e118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moraes, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meirelles, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honorato, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Souza, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Souza, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murakami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Oliveira, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobarg, J.</span></span> <span> </span><span class="NLM_article-title">Kinase inhibitor profile for human nek1, nek6, and nek7 and analysis of the structural basis for inhibitor specificity</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1176</span>– <span class="NLM_lpage">1191</span>, <span class="refDoi"> DOI: 10.3390/molecules20011176</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.3390%2Fmolecules20011176" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=25591119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVyhu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=1176-1191&author=E.+Moraesauthor=G.+Meirellesauthor=R.+Honoratoauthor=T.+de+Souzaauthor=E.+de+Souzaauthor=M.+Murakamiauthor=P.+de+Oliveiraauthor=J.+Kobarg&title=Kinase+inhibitor+profile+for+human+nek1%2C+nek6%2C+and+nek7+and+analysis+of+the+structural+basis+for+inhibitor+specificity&doi=10.3390%2Fmolecules20011176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitor profile for human Nek1, Nek6, and Nek7 and analysis of the structural basis for inhibitor specificity</span></div><div class="casAuthors">Moraes, Eduardo Cruz; Meirelles, Gabriela Vaz; Honorato, Rodrigo Vargas; Brasil de Souza, Tatiana de Arruda Campos; Elisa de Souza, Edmarcia; Murakami, Mario Tyago; Lopes de Oliveira, Paulo Sergio; Kobarg, Jorg</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1176-1191, 16 pp.</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Human Neks are a conserved protein kinase family related to cell cycle progression and cell division and are considered potential drug targets for the treatment of cancer and other pathologies.  We screened the activation loop mutant kinases hNek1 and hNek2, wild-type hNek7, and five hNek6 variants in different activation/phosphorylation statesand compared them against 85 compds. using thermal shift denaturation.  We identified three compds. with significant Tm shifts: JNK Inhibitor II for hNek1(Δ262-1258)-(T162A), Isogranulatimide for hNek6(S206A), andGSK-3 Inhibitor XIII for hNek7wt.  Each one of these compds. was also validated by reducing the kinases activity by at least 25%.  The binding sites for these compds. were identified by in silico docking at the ATP-binding site of the resp. hNeks.  Potential inhibitors were first screened by thermal shift assays, had their efficiency tested by a kinase assay, and were finally analyzed by mol. docking.  Our findings corroborate the idea of ATP-competitive inhibition for hNek1 and hNek6 and suggest a novel non-competitive inhibition for hNek7 in regard to GSK-3 Inhibitor XIII.  Our results demonstrate that our approach is useful for finding promising general and specific hNekscandidate inhibitors, which may also function as scaffolds to design more potent and selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpx1iPvmNQxk7Vg90H21EOLACvtfcHk0lge6d66zuoFjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVyhu7o%253D&md5=97e48f6686c84f7171b094981f591810</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.3390%2Fmolecules20011176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules20011176%26sid%3Dliteratum%253Aachs%26aulast%3DMoraes%26aufirst%3DE.%26aulast%3DMeirelles%26aufirst%3DG.%26aulast%3DHonorato%26aufirst%3DR.%26aulast%3Dde%2BSouza%26aufirst%3DT.%26aulast%3Dde%2BSouza%26aufirst%3DE.%26aulast%3DMurakami%26aufirst%3DM.%26aulast%3Dde%2BOliveira%26aufirst%3DP.%26aulast%3DKobarg%26aufirst%3DJ.%26atitle%3DKinase%2520inhibitor%2520profile%2520for%2520human%2520nek1%252C%2520nek6%252C%2520and%2520nek7%2520and%2520analysis%2520of%2520the%2520structural%2520basis%2520for%2520inhibitor%2520specificity%26jtitle%3DMolecules%26date%3D2015%26volume%3D20%26spage%3D1176%26epage%3D1191%26doi%3D10.3390%2Fmolecules20011176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Melo-Hanchuk, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slepicka, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meirelles, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basei, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovato, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granato, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauletti, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Domingues, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leme, A. F. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelegrini, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobarg, J.</span></span> <span> </span><span class="NLM_article-title">NEK1 kinase domain structure and its dynamic protein interactome after exposure to Cisplatin</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">5445</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-05325-w</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1038%2Fs41598-017-05325-w" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=28710492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A280%3ADC%252BC1cjnvVCktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=5445&author=T.+D.+Melo-Hanchukauthor=P.+F.+Slepickaauthor=G.+V.+Meirellesauthor=F.+L.+Baseiauthor=D.+V.+Lovatoauthor=D.+C.+Granatoauthor=B.+A.+Paulettiauthor=R.+R.+Dominguesauthor=A.+F.+P.+Lemeauthor=A.+L.+Pelegriniauthor=G.+Lenzauthor=S.+Knappauthor=J.+M.+Elkinsauthor=J.+Kobarg&title=NEK1+kinase+domain+structure+and+its+dynamic+protein+interactome+after+exposure+to+Cisplatin&doi=10.1038%2Fs41598-017-05325-w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">NEK1 kinase domain structure and its dynamic protein interactome after exposure to Cisplatin</span></div><div class="casAuthors">Melo-Hanchuk Talita D; Meirelles Gabriela Vaz; Basei Fernanda Luisa; Kobarg Jorg; Slepicka Priscila Ferreira; Lovato Diogo Ventura; Granato Daniela Campos; Pauletti Bianca Alves; Domingues Romenia Ramos; Leme Adriana Franco Paes; Pelegrini Alessandra Luiza; Lenz Guido; Knapp Stefan; Elkins Jonathan M; Knapp Stefan; Elkins Jonathan M; Kobarg Jorg</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5445</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">NEK family kinases are serine/threonine kinases that have been functionally implicated in the regulation of the disjunction of the centrosome, the assembly of the mitotic spindle, the function of the primary cilium and the DNA damage response.  NEK1 shows pleiotropic functions and has been found to be mutated in cancer cells, ciliopathies such as the polycystic kidney disease, as well as in the genetic diseases short-rib thoracic dysplasia, Mohr-syndrome and amyotrophic lateral sclerosis.  NEK1 is essential for the ionizing radiation DNA damage response and priming of the ATR kinase and of Rad54 through phosphorylation.  Here we report on the structure of the kinase domain of human NEK1 in its apo- and ATP-mimetic inhibitor bound forms.  The inhibitor bound structure may allow the design of NEK specific chemo-sensitizing agents to act in conjunction with chemo- or radiation therapy of cancer cells.  Furthermore, we characterized the dynamic protein interactome of NEK1 after DNA damage challenge with cisplatin.  Our data suggest that NEK1 and its interaction partners trigger the DNA damage pathways responsible for correcting DNA crosslinks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQENCCRhZkgKDToTC3aBaesfW6udTcc2eahxOk2jo4H67ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cjnvVCktA%253D%253D&md5=c36569b4a002cb1bdacd0dae93622d38</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-05325-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-05325-w%26sid%3Dliteratum%253Aachs%26aulast%3DMelo-Hanchuk%26aufirst%3DT.%2BD.%26aulast%3DSlepicka%26aufirst%3DP.%2BF.%26aulast%3DMeirelles%26aufirst%3DG.%2BV.%26aulast%3DBasei%26aufirst%3DF.%2BL.%26aulast%3DLovato%26aufirst%3DD.%2BV.%26aulast%3DGranato%26aufirst%3DD.%2BC.%26aulast%3DPauletti%26aufirst%3DB.%2BA.%26aulast%3DDomingues%26aufirst%3DR.%2BR.%26aulast%3DLeme%26aufirst%3DA.%2BF.%2BP.%26aulast%3DPelegrini%26aufirst%3DA.%2BL.%26aulast%3DLenz%26aufirst%3DG.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DKobarg%26aufirst%3DJ.%26atitle%3DNEK1%2520kinase%2520domain%2520structure%2520and%2520its%2520dynamic%2520protein%2520interactome%2520after%2520exposure%2520to%2520Cisplatin%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D5445%26doi%3D10.1038%2Fs41598-017-05325-w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapadia, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Counago, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span> <span> </span><span class="NLM_article-title">In depth analysis of kinase cross screening data to identify chemical starting points for inhibition of the Nek family of kinases</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">44</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1039/C7MD00510E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1039%2FC7MD00510E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=30108900" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFersbzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=44-66&author=C.+I.+Wellsauthor=N.+R.+Kapadiaauthor=R.+M.+Counagoauthor=D.+H.+Drewry&title=In+depth+analysis+of+kinase+cross+screening+data+to+identify+chemical+starting+points+for+inhibition+of+the+Nek+family+of+kinases&doi=10.1039%2FC7MD00510E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">In depth analysis of kinase cross screening data to identify chemical starting points for inhibition of the Nek family of kinases</span></div><div class="casAuthors">Wells, C. I.; Kapadia, N. R.; Counago, R. M.; Drewry, D. H.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">44-66</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Potent, selective, and cell active small mol. kinase inhibitors are useful tools to help unravel the complexities of kinase signaling.  As the biol. functions of individual kinases become better understood, they can become targets of drug discovery efforts.  The small mols. used to shed light on function can also then serve as chem. starting points in these drug discovery efforts.  The Nek family of kinases has received very little attention, as judged by no. of citations in PubMed, yet they appear to play many key roles and have been implicated in disease.  Here we present our work to identify high quality chem. starting points that have emerged due to the increased incidence of broad kinome screening.  We anticipate that this anal. will allow the community to progress towards the generation of chem. probes and eventually drugs that target members of the Nek family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBzprTGDD0hbVg90H21EOLACvtfcHk0lihFnJ4RXTVmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFersbzJ&md5=0daf63621e95f51f04cb119fa98d69d9</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1039%2FC7MD00510E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7MD00510E%26sid%3Dliteratum%253Aachs%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DKapadia%26aufirst%3DN.%2BR.%26aulast%3DCounago%26aufirst%3DR.%2BM.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26atitle%3DIn%2520depth%2520analysis%2520of%2520kinase%2520cross%2520screening%2520data%2520to%2520identify%2520chemical%2520starting%2520points%2520for%2520inhibition%2520of%2520the%2520Nek%2520family%2520of%2520kinases%26jtitle%3DMedChemComm%26date%3D2018%26volume%3D9%26spage%3D44%26epage%3D66%26doi%3D10.1039%2FC7MD00510E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Donato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Righino, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippetti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaglia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrillo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pirolli, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scambia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Rosa, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallo, D.</span></span> <span> </span><span class="NLM_article-title">Identification and antitumor activity of a novel inhibitor of the NIMA-related kinase NEK6</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">16047</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-34471-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1038%2Fs41598-018-34471-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=30375481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A280%3ADC%252BB3cvkslCmtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=16047&author=M.+De%0ADonatoauthor=B.+Righinoauthor=F.+Filippettiauthor=A.+Battagliaauthor=M.+Petrilloauthor=D.+Pirolliauthor=G.+Scambiaauthor=M.+C.+De+Rosaauthor=D.+Gallo&title=Identification+and+antitumor+activity+of+a+novel+inhibitor+of+the+NIMA-related+kinase+NEK6&doi=10.1038%2Fs41598-018-34471-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and antitumor activity of a novel inhibitor of the NIMA-related kinase NEK6</span></div><div class="casAuthors">De Donato Marta; Filippetti Flavia; Battaglia Alessandra; Petrillo Marco; Scambia Giovanni; Gallo Daniela; De Donato Marta; Filippetti Flavia; Scambia Giovanni; Gallo Daniela; Righino Benedetta; Petrillo Marco; Pirolli Davide; De Rosa Maria Cristina</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">16047</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The NIMA (never in mitosis, gene A)-related kinase-6 (NEK6), which is implicated in cell cycle control and plays significant roles in tumorigenesis, is an attractive target for the development of novel anti-cancer drugs.  Here we describe the discovery of a potent ATP site-directed inhibitor of NEK6 identified by virtual screening, adopting both structure- and ligand-based techniques.  Using a homology-built model of NEK6 as well as the pharmacophoric features of known NEK6 inhibitors we identified novel binding scaffolds.  Twenty-five compounds from the top ranking hits were subjected to in vitro kinase assays.  The best compound, i.e. compound 8 ((5Z)-2-hydroxy-4-methyl-6-oxo-5-[(5-phenylfuran-2-yl)methylidene]-5,6-dihydropyridine-3-carbonitrile), was able to inhibit NEK6 with low micromolar IC50 value, also displaying antiproliferative activity against a panel of human cancer cell lines.  Our results suggest that the identified inhibitor can be used as lead candidate for the development of novel anti-cancer agents, thus opening the possibility of new therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRglXfZMl481XthwrmtH4OBfW6udTcc2eaC42ASV-WCI7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cvkslCmtw%253D%253D&md5=7660b56cc7704766fc7dd27893409f9b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-34471-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-34471-y%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BDonato%26aufirst%3DM.%26aulast%3DRighino%26aufirst%3DB.%26aulast%3DFilippetti%26aufirst%3DF.%26aulast%3DBattaglia%26aufirst%3DA.%26aulast%3DPetrillo%26aufirst%3DM.%26aulast%3DPirolli%26aufirst%3DD.%26aulast%3DScambia%26aufirst%3DG.%26aulast%3DDe%2BRosa%26aufirst%3DM.%2BC.%26aulast%3DGallo%26aufirst%3DD.%26atitle%3DIdentification%2520and%2520antitumor%2520activity%2520of%2520a%2520novel%2520inhibitor%2520of%2520the%2520NIMA-related%2520kinase%2520NEK6%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D16047%26doi%3D10.1038%2Fs41598-018-34471-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Ali, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axtman, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capuzzi, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederiksen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hooper, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luecking, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelides, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saikatendu, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span> <span> </span><span class="NLM_article-title">Progress towards a public chemogenomic set for protein kinases and a call for contributions</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0181585</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0181585</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1371%2Fjournal.pone.0181585" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=28767711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFWktL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=D.+H.+Drewryauthor=C.+I.+Wellsauthor=D.+M.+Andrewsauthor=R.+Angellauthor=H.+Al-Aliauthor=A.+D.+Axtmanauthor=S.+J.+Capuzziauthor=J.+M.+Elkinsauthor=P.+Ettmayerauthor=M.+Frederiksenauthor=O.+Gileadiauthor=N.+Grayauthor=A.+Hooperauthor=S.+Knappauthor=S.+Lauferauthor=U.+Lueckingauthor=M.+Michaelidesauthor=S.+M%C3%BCllerauthor=E.+Muratovauthor=R.+A.+Dennyauthor=K.+S.+Saikatenduauthor=D.+K.+Treiberauthor=W.+J.+Zuercherauthor=T.+M.+Willson&title=Progress+towards+a+public+chemogenomic+set+for+protein+kinases+and+a+call+for+contributions&doi=10.1371%2Fjournal.pone.0181585"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Progress towards a public chemogenomic set for protein kinases and a call for contributions</span></div><div class="casAuthors">Drewry, David H.; Wells, Carrow I.; Andrews, David M.; Angell, Richard; Al-Ali, Hassan; Axtman, Alison D.; Capuzzi, Stephen J.; Elkins, Jonathan M.; Ettmayer, Peter; Frederiksen, Mathias; Gileadi, Opher; Gray, Nathanael; Hooper, Alice; Knapp, Stefan; Laufer, Stefan; Luecking, Ulrich; Michaelides, Michael; Muller, Susanne; Muratov, Eugene; Aldrin, Denny R.; Saikatendu, Kumar S.; Treiber, Daniel K.; Zuercher, William J.; Willson, Timothy M.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e0181585/1-e0181585/20</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein kinases are highly tractable targets for drug discovery.  However, the biol. function and therapeutic potential of the majority of the 500+ human protein kinases remains unknown.  We have developed phys. and virtual collections of small mol. inhibitors, which we call chemogenomic sets, that are designed to inhibit the catalytic function of almost half the human protein kinases.  In this manuscript we share our progress towards generation of a comprehensive kinase chemogenomic set (KCGS), release kinome profiling data of a large inhibitor set (Published Kinase Inhibitor Set 2 (PKIS2)), and outline a process through which the community can openly collaborate to create a KCGS that probes the full complement of human protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpozOiNZl51XbVg90H21EOLACvtfcHk0lgLZlB_PJ-qWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFWktL0%253D&md5=9c27ba5cd21da43d30ce485ffa0e1675</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0181585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0181585%26sid%3Dliteratum%253Aachs%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DAndrews%26aufirst%3DD.%2BM.%26aulast%3DAngell%26aufirst%3DR.%26aulast%3DAl-Ali%26aufirst%3DH.%26aulast%3DAxtman%26aufirst%3DA.%2BD.%26aulast%3DCapuzzi%26aufirst%3DS.%2BJ.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DEttmayer%26aufirst%3DP.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DHooper%26aufirst%3DA.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DLaufer%26aufirst%3DS.%26aulast%3DLuecking%26aufirst%3DU.%26aulast%3DMichaelides%26aufirst%3DM.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DMuratov%26aufirst%3DE.%26aulast%3DDenny%26aufirst%3DR.%2BA.%26aulast%3DSaikatendu%26aufirst%3DK.%2BS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26atitle%3DProgress%2520towards%2520a%2520public%2520chemogenomic%2520set%2520for%2520protein%2520kinases%2520and%2520a%2520call%2520for%2520contributions%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0181585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seefeld, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouse, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNulty, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luengo, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minthorn, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Concha, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heerding, D. A.</span></span> <span> </span><span class="NLM_article-title">Discovery of 5-pyrrolopyridinyl-2-thiophenecarboxamides as potent AKT kinase inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2244</span>– <span class="NLM_lpage">2248</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2009.02.094</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.bmcl.2009.02.094" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=19285393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkt1Sjt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=2244-2248&author=M.+A.+Seefeldauthor=M.+B.+Rouseauthor=K.+C.+McNultyauthor=L.+Sunauthor=J.+Wangauthor=D.+S.+Yamashitaauthor=J.+I.+Luengoauthor=S.+Zhangauthor=E.+A.+Minthornauthor=N.+O.+Conchaauthor=D.+A.+Heerding&title=Discovery+of+5-pyrrolopyridinyl-2-thiophenecarboxamides+as+potent+AKT+kinase+inhibitors&doi=10.1016%2Fj.bmcl.2009.02.094"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 5-pyrrolopyridinyl-2-thiophenecarboxamides as potent AKT kinase inhibitors</span></div><div class="casAuthors">Seefeld, Mark A.; Rouse, Meagan B.; McNulty, Kenneth C.; Sun, Lihui; Wang, Jizhou; Yamashita, Dennis S.; Luengo, Juan I.; Zhang, ShuYun; Minthorn, Elisabeth A.; Concha, Nestor O.; Heerding, Dirk A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2244-2248</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A pyrrolopyridinyl thiophenecarboxamide I was discovered as a tractable starting point for a lead optimization effort in an AKT kinase inhibition program.  SAR studies aided by a co-crystal structure of I in AKT2 led to the identification of AKT inhibitors with subnanomolar potency.  Representative compds. showed antiproliferative activity as well as inhibition of phosphorylation of the downstream target GSK3β.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_nx0I3_W9-7Vg90H21EOLACvtfcHk0lgLZlB_PJ-qWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkt1Sjt7s%253D&md5=7d34026abcf524ffced43f2d0e4629ec</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.02.094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.02.094%26sid%3Dliteratum%253Aachs%26aulast%3DSeefeld%26aufirst%3DM.%2BA.%26aulast%3DRouse%26aufirst%3DM.%2BB.%26aulast%3DMcNulty%26aufirst%3DK.%2BC.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DYamashita%26aufirst%3DD.%2BS.%26aulast%3DLuengo%26aufirst%3DJ.%2BI.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DMinthorn%26aufirst%3DE.%2BA.%26aulast%3DConcha%26aufirst%3DN.%2BO.%26aulast%3DHeerding%26aufirst%3DD.%2BA.%26atitle%3DDiscovery%2520of%25205-pyrrolopyridinyl-2-thiophenecarboxamides%2520as%2520potent%2520AKT%2520kinase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D2244%26epage%3D2248%26doi%3D10.1016%2Fj.bmcl.2009.02.094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liddle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cousins, R. P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobbs, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioannou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellor, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payne, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pritchard, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tape, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, R. A.</span></span> <span> </span><span class="NLM_article-title">4-Phenyl-7-azaindoles as potent and selective IKK2 inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2504</span>– <span class="NLM_lpage">2508</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2009.03.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.bmcl.2009.03.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=19349179" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkslyitbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=2504-2508&author=J.+Liddleauthor=P.+Bamboroughauthor=M.+D.+Barkerauthor=S.+Camposauthor=R.+P.+C.+Cousinsauthor=G.+J.+Cutlerauthor=H.+Hobbsauthor=D.+S.+Holmesauthor=C.+Ioannouauthor=G.+W.+Mellorauthor=M.+A.+Morseauthor=J.+J.+Payneauthor=J.+M.+Pritchardauthor=K.+J.+Smithauthor=D.+T.+Tapeauthor=C.+Whitworthauthor=R.+A.+Williamson&title=4-Phenyl-7-azaindoles+as+potent+and+selective+IKK2+inhibitors&doi=10.1016%2Fj.bmcl.2009.03.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">4-Phenyl-7-azaindoles as potent and selective IKK2 inhibitors</span></div><div class="casAuthors">Liddle, John; Bamborough, Paul; Barker, Michael D.; Campos, Sebastien; Cousins, Rick P. C.; Cutler, Geoffrey J.; Hobbs, Heather; Holmes, Duncan S.; Ioannou, Chris; Mellor, Geoff W.; Morse, Mary A.; Payne, Jeremy J.; Pritchard, John M.; Smith, Kathryn J.; Tape, Daniel T.; Whitworth, Caroline; Williamson, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2504-2508</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The synthesis and SAR of a novel series of IKK2 inhibitors are described.  Modification around the hinge binding region of the 7-azaindole led to a series of potent and selective inhibitors with good cellular activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqav8UP5rEwSbVg90H21EOLACvtfcHk0lgbDqmOwwM98w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkslyitbk%253D&md5=e53c6eda9da4d535410c96c460de511b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.03.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.03.034%26sid%3Dliteratum%253Aachs%26aulast%3DLiddle%26aufirst%3DJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBarker%26aufirst%3DM.%2BD.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DCousins%26aufirst%3DR.%2BP.%2BC.%26aulast%3DCutler%26aufirst%3DG.%2BJ.%26aulast%3DHobbs%26aufirst%3DH.%26aulast%3DHolmes%26aufirst%3DD.%2BS.%26aulast%3DIoannou%26aufirst%3DC.%26aulast%3DMellor%26aufirst%3DG.%2BW.%26aulast%3DMorse%26aufirst%3DM.%2BA.%26aulast%3DPayne%26aufirst%3DJ.%2BJ.%26aulast%3DPritchard%26aufirst%3DJ.%2BM.%26aulast%3DSmith%26aufirst%3DK.%2BJ.%26aulast%3DTape%26aufirst%3DD.%2BT.%26aulast%3DWhitworth%26aufirst%3DC.%26aulast%3DWilliamson%26aufirst%3DR.%2BA.%26atitle%3D4-Phenyl-7-azaindoles%2520as%2520potent%2520and%2520selective%2520IKK2%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D2504%26epage%3D2508%26doi%3D10.1016%2Fj.bmcl.2009.03.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gehrig, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westhoff, J. H.</span></span> <span> </span><span class="NLM_article-title">Zebrafish as a model for drug screening in genetic kidney diseases</span>. <i>Frontiers in Pediatrics</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">183</span>, <span class="refDoi"> DOI: 10.3389/fped.2018.00183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.3389%2Ffped.2018.00183" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=30003073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252Fns12jtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=183&author=J.+Gehrigauthor=G.+Pandeyauthor=J.+H.+Westhoff&title=Zebrafish+as+a+model+for+drug+screening+in+genetic+kidney+diseases&doi=10.3389%2Ffped.2018.00183"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Zebrafish as a Model for Drug Screening in Genetic Kidney Diseases</span></div><div class="casAuthors">Gehrig Jochen; Pandey Gunjan; Pandey Gunjan; Westhoff Jens H</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in pediatrics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">183</span>
        ISSN:<span class="NLM_cas:issn">2296-2360</span>.
    </div><div class="casAbstract">Genetic disorders account for a wide range of renal diseases emerging during childhood and adolescence.  Due to the utilization of modern biochemical and biomedical techniques, the number of identified disease-associated genes is increasing rapidly.  Modeling of congenital human disease in animals is key to our understanding of the biological mechanism underlying pathological processes and thus developing novel potential treatment options.  The zebrafish (Danio rerio) has been established as a versatile small vertebrate organism that is widely used for studying human inherited diseases.  Genetic accessibility in combination with elegant experimental methods in zebrafish permit modeling of human genetic diseases and dissecting the perturbation of underlying cellular networks and physiological processes.  Beyond its utility for genetic analysis and pathophysiological and mechanistic studies, zebrafish embryos, and larvae are amenable for phenotypic screening approaches employing high-content and high-throughput experiments using automated microscopy.  This includes large-scale chemical screening experiments using genetic models for searching for disease-modulating compounds.  Phenotype-based approaches of drug discovery have been successfully performed in diverse zebrafish-based screening applications with various phenotypic readouts.  As a result, these can lead to the identification of candidate substances that are further examined in preclinical and clinical trials.  In this review, we discuss zebrafish models for inherited kidney disease as well as requirements and considerations for the technical realization of drug screening experiments in zebrafish.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTYV2FmQJ5uRVUIs6wFK5xJfW6udTcc2ebAX-AatEmQL7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252Fns12jtA%253D%253D&md5=7c9e5b3158f7042e9241f2a0887476d6</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.3389%2Ffped.2018.00183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffped.2018.00183%26sid%3Dliteratum%253Aachs%26aulast%3DGehrig%26aufirst%3DJ.%26aulast%3DPandey%26aufirst%3DG.%26aulast%3DWesthoff%26aufirst%3DJ.%2BH.%26atitle%3DZebrafish%2520as%2520a%2520model%2520for%2520drug%2520screening%2520in%2520genetic%2520kidney%2520diseases%26jtitle%3DFrontiers%2520in%2520Pediatrics%26date%3D2018%26volume%3D6%26spage%3D183%26doi%3D10.3389%2Ffped.2018.00183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span>; <span class="NLM_string-name">Barker, M. D.</span>; <span class="NLM_string-name">Campos, S. A.</span>; <span class="NLM_string-name">Cousins, R. P. C.</span>; <span class="NLM_string-name">Faulder, P.</span>; <span class="NLM_string-name">Hobbs, H.</span>; <span class="NLM_string-name">Holmes, D. S.</span>; <span class="NLM_string-name">Johnston, M. J.</span>; <span class="NLM_string-name">Liddle, J.</span>; <span class="NLM_string-name">Payne, J. J.</span>; <span class="NLM_string-name">Pritchard, J. M.</span>; <span class="NLM_string-name">Whitworth, C.</span></span> <span> </span><span class="NLM_article-title">Pyrrolo[2,3-b]pyridin-4-yl-benzenesulfonamide compounds as IKK2 inhibitors and their preparation, pharmaceutical compositions and use in the treatment of diseases</span>. <span class="NLM_patent">WO2008034860A1</span>, <span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=P.+Bamborough&author=M.+D.+Barker&author=S.+A.+Campos&author=R.+P.+C.+Cousins&author=P.+Faulder&author=H.+Hobbs&author=D.+S.+Holmes&author=M.+J.+Johnston&author=J.+Liddle&author=J.+J.+Payne&author=J.+M.+Pritchard&author=C.+Whitworth&title=Pyrrolo%5B2%2C3-b%5Dpyridin-4-yl-benzenesulfonamide+compounds+as+IKK2+inhibitors+and+their+preparation%2C+pharmaceutical+compositions+and+use+in+the+treatment+of+diseases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26atitle%3DPyrrolo%255B2%252C3-b%255Dpyridin-4-yl-benzenesulfonamide%2520compounds%2520as%2520IKK2%2520inhibitors%2520and%2520their%2520preparation%252C%2520pharmaceutical%2520compositions%2520and%2520use%2520in%2520the%2520treatment%2520of%2520diseases%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McKennon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyers, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drauz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarm, M.</span></span> <span> </span><span class="NLM_article-title">A convenient reduction of amino acids and their derivatives</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">3568</span>– <span class="NLM_lpage">3571</span>, <span class="refDoi"> DOI: 10.1021/jo00065a020</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00065a020" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADyaK3sXksV2rtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=1993&pages=3568-3571&author=M.+J.+McKennonauthor=A.+I.+Meyersauthor=K.+Drauzauthor=M.+Schwarm&title=A+convenient+reduction+of+amino+acids+and+their+derivatives&doi=10.1021%2Fjo00065a020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A convenient reduction of amino acids and their derivatives</span></div><div class="casAuthors">McKennon, Marc J.; Meyers, A. I.; Drauz, Karlheinz; Schwarm, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3568-71</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">Amino acids were reduced by NaBH4 in the presence of iodine to give the corresponding amino alcs.  Thus, amino acids I (R = CMe3, CHMe2, CH2Ph, CHEtMe, CH2CH2SMe, C6H4OH-4) were reduced by NaBH4-iodine in THF to give amino alcs. II.  N-acyl amino acids III [R = CH2Ph, R1 = H, R2 = H, Me; RR1 = (CH2)3 R2 = Me; R = R1 = H, R2 = Ph] were also reduced by NaBH4-iodine in THF to give N-acyl amino alcs. IV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnnqE3R204brVg90H21EOLACvtfcHk0lgXtkI9QKn5iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXksV2rtL8%253D&md5=d79a74af1c20e79ceee5c61ed977bccb</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjo00065a020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00065a020%26sid%3Dliteratum%253Aachs%26aulast%3DMcKennon%26aufirst%3DM.%2BJ.%26aulast%3DMeyers%26aufirst%3DA.%2BI.%26aulast%3DDrauz%26aufirst%3DK.%26aulast%3DSchwarm%26aufirst%3DM.%26atitle%3DA%2520convenient%2520reduction%2520of%2520amino%2520acids%2520and%2520their%2520derivatives%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1993%26volume%3D58%26spage%3D3568%26epage%3D3571%26doi%3D10.1021%2Fjo00065a020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chong, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dadwal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">Efficient synthesis of functionalized aziridinium salts</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">219</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1021/jo901893n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo901893n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsV2nurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2010&pages=219-221&author=H.-S.+Chongauthor=H.+A.+Songauthor=M.+Dadwalauthor=X.+Sunauthor=I.+Sinauthor=Y.+Chen&title=Efficient+synthesis+of+functionalized+aziridinium+salts&doi=10.1021%2Fjo901893n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient synthesis of functionalized aziridinium salts</span></div><div class="casAuthors">Chong, Hyun-Soon; Song, Hyun A.; Dadwal, Mamta; Sun, Xiang; Sin, Inseok; Chen, Yunwei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">219-221</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Various aziridinium salts were efficiently prepd. from bromination of a series of backbone substituted N,N-disubstituted β-amino alcs. and isolated via flash column chromatog.  The effect of C-substitution, N-substitution, solvent, leaving group, and counteranions on formation of the isolable aziridinium salts was investigated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBaN2nKqK0DrVg90H21EOLACvtfcHk0lgXtkI9QKn5iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsV2nurrJ&md5=a326087f310c15dc390be8baf09a0829</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjo901893n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo901893n%26sid%3Dliteratum%253Aachs%26aulast%3DChong%26aufirst%3DH.-S.%26aulast%3DSong%26aufirst%3DH.%2BA.%26aulast%3DDadwal%26aufirst%3DM.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DSin%26aufirst%3DI.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DEfficient%2520synthesis%2520of%2520functionalized%2520aziridinium%2520salts%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2010%26volume%3D75%26spage%3D219%26epage%3D221%26doi%3D10.1021%2Fjo901893n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Westwood, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheary, D.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baxter, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Montfort, R. L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span> <span> </span><span class="NLM_article-title">Insights into the conformational variability and regulation of human Nek2 kinase</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>386</i></span>,  <span class="NLM_fpage">476</span>– <span class="NLM_lpage">485</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2008.12.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.jmb.2008.12.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=19124027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhs1ars7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=386&publication_year=2009&pages=476-485&author=I.+Westwoodauthor=D.-M.+Chearyauthor=J.+E.+Baxterauthor=M.+W.+Richardsauthor=R.+L.+M.+van+Montfortauthor=A.+M.+Fryauthor=R.+Bayliss&title=Insights+into+the+conformational+variability+and+regulation+of+human+Nek2+kinase&doi=10.1016%2Fj.jmb.2008.12.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the Conformational Variability and Regulation of Human Nek2 Kinase</span></div><div class="casAuthors">Westwood, Isaac; Cheary, Donna-Marie; Baxter, Joanne E.; Richards, Mark W.; van Montfort, Rob L. M.; Fry, Andrew M.; Bayliss, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">386</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">476-485</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Nek family of serine/threonine kinases regulates centrosome and cilia function; in addn., several of its members are potential targets for drug discovery.  Nek2 is dimeric, is cell cycle regulated and functions in the sepn. of centrosomes at G2/M.  Here, we report the crystal structures of wild-type human Nek2 kinase domain bound to ADP at 1.55-Å resoln. and T175A mutant in apo form as well as that bound to a non-hydrolyzable ATP analog.  These show that regions of the Nek2 structure around the nucleotide-binding site can adopt several different but well-defined conformations.  None of the conformations was the same as that obsd. for the previously reported inhibitor-bound structure, and the two nucleotides stabilized two conformations.  The structures suggest mechanisms for the auto-inhibition of Nek2 that we have tested by mutagenesis.  Comparison of the structures with Aurora-A and Cdk2 gives insight into the structural mechanism of Nek2 activation.  The prodn. of specific inhibitors that target individual kinases of the human genome is an urgent challenge in drug discovery, and Nek2 is esp. promising as a cancer target.  We not only identify potential challenges to the task of producing Nek2 inhibitors but also propose that the conformational variability provides an opportunity for the design of Nek2 selective inhibitors because one of the conformations may provide a unique target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyfL7jAcF9TbVg90H21EOLACvtfcHk0lgXtkI9QKn5iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhs1ars7o%253D&md5=e51d49845a03d605cdc6f7d2ba244fce</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2008.12.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2008.12.033%26sid%3Dliteratum%253Aachs%26aulast%3DWestwood%26aufirst%3DI.%26aulast%3DCheary%26aufirst%3DD.-M.%26aulast%3DBaxter%26aufirst%3DJ.%2BE.%26aulast%3DRichards%26aufirst%3DM.%2BW.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%2BL.%2BM.%26aulast%3DFry%26aufirst%3DA.%2BM.%26aulast%3DBayliss%26aufirst%3DR.%26atitle%3DInsights%2520into%2520the%2520conformational%2520variability%2520and%2520regulation%2520of%2520human%2520Nek2%2520kinase%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D386%26spage%3D476%26epage%3D485%26doi%3D10.1016%2Fj.jmb.2008.12.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haq, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallego, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeoh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Regan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reverter, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roig, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span> <span> </span><span class="NLM_article-title">Mechanistic basis of Nek7 activation through Nek9 binding and induced dimerization</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">8771</span>, <span class="refDoi"> DOI: 10.1038/ncomms9771</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1038%2Fncomms9771" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=26522158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslOgtbrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8771&author=T.+Haqauthor=M.+W.+Richardsauthor=S.+G.+Burgessauthor=P.+Gallegoauthor=S.+Yeohauthor=L.+O%E2%80%99Reganauthor=D.+Reverterauthor=J.+Roigauthor=A.+M.+Fryauthor=R.+Bayliss&title=Mechanistic+basis+of+Nek7+activation+through+Nek9+binding+and+induced+dimerization&doi=10.1038%2Fncomms9771"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic basis of Nek7 activation through Nek9 binding and induced dimerization</span></div><div class="casAuthors">Haq, Tamanna; Richards, Mark W.; Burgess, Selena G.; Gallego, Pablo; Yeoh, Sharon; O'Regan, Laura; Reverter, David; Roig, Joan; Fry, Andrew M.; Bayliss, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8771</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mitotic spindle assembly requires the regulated activities of protein kinases such as Nek7 and Nek9.  Nek7 is autoinhibited by the protrusion of Tyr97 into the active site and activated by the Nek9 non-catalytic C-terminal domain (CTD).  CTD binding apparently releases autoinhibition because mutation of Tyr97 to phenylalanine increases Nek7 activity independently of Nek9.  Here the authors find that self-assocn. of the Nek9-CTD is needed for Nek7 activation.  The authors map the minimal Nek7 binding region of Nek9 to residues 810-828.  A crystal structure of Nek7Y97F bound to Nek9810-828 reveals a binding site on the C-lobe of the Nek7 kinase domain.  Nek7Y97F crystallizes as a back-to-back dimer between kinase domain N-lobes, in which the specific contacts within the interface are coupled to the conformation of residue 97.  Hence, the authors propose that the Nek9-CTD activates Nek7 through promoting back-to-back dimerization that releases the autoinhibitory tyrosine residue, a mechanism conserved in unrelated kinase families.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7eBzbywmhnbVg90H21EOLACvtfcHk0lgMih8ZlBq_rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslOgtbrP&md5=47f92bb0b477c6ee53cd7d7d46f954c8</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fncomms9771&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms9771%26sid%3Dliteratum%253Aachs%26aulast%3DHaq%26aufirst%3DT.%26aulast%3DRichards%26aufirst%3DM.%2BW.%26aulast%3DBurgess%26aufirst%3DS.%2BG.%26aulast%3DGallego%26aufirst%3DP.%26aulast%3DYeoh%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Regan%26aufirst%3DL.%26aulast%3DReverter%26aufirst%3DD.%26aulast%3DRoig%26aufirst%3DJ.%26aulast%3DFry%26aufirst%3DA.%2BM.%26aulast%3DBayliss%26aufirst%3DR.%26atitle%3DMechanistic%2520basis%2520of%2520Nek7%2520activation%2520through%2520Nek9%2520binding%2520and%2520induced%2520dimerization%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D8771%26doi%3D10.1038%2Fncomms9771" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawa, M.</span></span> <span> </span><span class="NLM_article-title">7-Azaindole: A versatile scaffold for developing kinase inhibitors</span>. <i>Chem. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">36</span>, <span class="refDoi"> DOI: 10.1248/cpb.c17-00380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1248%2Fcpb.c17-00380" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=29311509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslOntr%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2018&pages=29-36&author=T.+Irieauthor=M.+Sawa&title=7-Azaindole%3A+A+versatile+scaffold+for+developing+kinase+inhibitors&doi=10.1248%2Fcpb.c17-00380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">7-Azaindole: a versatile scaffold for developing kinase inhibitors</span></div><div class="casAuthors">Irie, Takayuki; Sawa, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-36</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">A review.  This review summarized recent examples of 7-azaindole-based kinase inhibitors and discusses their binding interactions with the kinase hinge regions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9QuvBOqQ8GrVg90H21EOLACvtfcHk0lgMih8ZlBq_rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslOntr%252FK&md5=69a5097cb54f2598d2e02414e74b3e6b</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1248%2Fcpb.c17-00380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.c17-00380%26sid%3Dliteratum%253Aachs%26aulast%3DIrie%26aufirst%3DT.%26aulast%3DSawa%26aufirst%3DM.%26atitle%3D7-Azaindole%253A%2520A%2520versatile%2520scaffold%2520for%2520developing%2520kinase%2520inhibitors%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D2018%26volume%3D66%26spage%3D29%26epage%3D36%26doi%3D10.1248%2Fcpb.c17-00380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Juchum, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günther, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Döring, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sievers-Engler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lämmerhofer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S.</span></span> <span> </span><span class="NLM_article-title">Trisubstituted imidazoles with a rigidized hinge binding motif act as single digit nM inhibitors of clinically relevant EGFR L858R/T790M and L858R/T790M/C797S mutants: An example of target hopping</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4636</span>– <span class="NLM_lpage">4656</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00178</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00178" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnt1Ciu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4636-4656&author=M.+Juchumauthor=M.+G%C3%BCntherauthor=E.+D%C3%B6ringauthor=A.+Sievers-Englerauthor=M.+L%C3%A4mmerhoferauthor=S.+Laufer&title=Trisubstituted+imidazoles+with+a+rigidized+hinge+binding+motif+act+as+single+digit+nM+inhibitors+of+clinically+relevant+EGFR+L858R%2FT790M+and+L858R%2FT790M%2FC797S+mutants%3A+An+example+of+target+hopping&doi=10.1021%2Facs.jmedchem.7b00178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Trisubstituted Imidazoles with a Rigidized Hinge Binding Motif Act As Single Digit nM Inhibitors of Clinically Relevant EGFR L858R/T790M and L858R/T790M/C797S Mutants: An Example of Target Hopping</span></div><div class="casAuthors">Juchum, Michael; Guenther, Marcel; Doering, Eva; Sievers-Engler, Adrian; Laemmerhofer, Michael; Laufer, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4636-4656</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Trisubstituted imidazoles with a rigidized 7-azaindole hinge binding motif such as I were prepd. as inhibitors of EGFR tyrosine kinase with L858R/T790M/C797S mutations (against which current EGFR inhibitors are ineffective) for potential use in the treatment of non-small-cell lung cancer.  II [X = (CH2)2, (CH2)4] reversibly inhibited the clin. challenging EGFR tyrosine kinase triple mutant L858R/T790M/C797S with IC50 values in the low nanomolar range.  I inhibited EGFR tyrosine kinase irreversibly with IC50 value of 1 nM against the EGFR L858R/T790M double mutant; I was selective for EGFR tyrosine kinase over other related kinases except for MAP4K4, MAP4K5, and MINK.  The kinetics of target inhibition, aq. soly, and metabolites were detd. for I and for selected compds.; mol. docking calcns. were also performed for selected compds. with EGFR isoforms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzeDYy9aSCrLVg90H21EOLACvtfcHk0lgMih8ZlBq_rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnt1Ciu78%253D&md5=a7003c8160b506ce964edce6adf7c3ff</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00178%26sid%3Dliteratum%253Aachs%26aulast%3DJuchum%26aufirst%3DM.%26aulast%3DG%25C3%25BCnther%26aufirst%3DM.%26aulast%3DD%25C3%25B6ring%26aufirst%3DE.%26aulast%3DSievers-Engler%26aufirst%3DA.%26aulast%3DL%25C3%25A4mmerhofer%26aufirst%3DM.%26aulast%3DLaufer%26aufirst%3DS.%26atitle%3DTrisubstituted%2520imidazoles%2520with%2520a%2520rigidized%2520hinge%2520binding%2520motif%2520act%2520as%2520single%2520digit%2520nM%2520inhibitors%2520of%2520clinically%2520relevant%2520EGFR%2520L858R%252FT790M%2520and%2520L858R%252FT790M%252FC797S%2520mutants%253A%2520An%2520example%2520of%2520target%2520hopping%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4636%26epage%3D4656%26doi%3D10.1021%2Facs.jmedchem.7b00178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ismer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiemann, J. K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goede, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preissner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hildebrand, P. W.</span></span> <span> </span><span class="NLM_article-title">SL2: an interactive webtool for modeling of missing segments in proteins</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">W390</span>– <span class="NLM_lpage">W394</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw297</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1093%2Fnar%2Fgkw297" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=27105847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2itrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=W390-W394&author=J.+Ismerauthor=A.+S.+Roseauthor=J.+K.+S.+Tiemannauthor=A.+Goedeauthor=R.+Preissnerauthor=P.+W.+Hildebrand&title=SL2%3A+an+interactive+webtool+for+modeling+of+missing+segments+in+proteins&doi=10.1093%2Fnar%2Fgkw297"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">SL2: an interactive webtool for modeling of missing segments in proteins</span></div><div class="casAuthors">Ismer, Jochen; Rose, Alexander S.; Tiemann, Johanna K. S.; Goede, Andrean; Preissner, Robert; Hildebrand, Peter W.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W390-W394</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">SuperLooper2 (SL2) is the updated version of our previous webserver SuperLooper, a fragment based tool for the prediction and interactive placement of loop structures into globular and helical membrane proteins.  In comparison to our previous version, SL2 benefits from both a considerably enlarged database of fragments derived from high-resoln. 3D protein structures of globular and helical membrane proteins, and the integration of a new protein viewer.  The database, now with double the content, significantly improved the coverage of fragment conformations and prediction quality.  The employment of the NGL viewer for visualization of the protein under investigation and interactive selection of appropriate loops makes SL2 independent of third-party plug-ins and addnl. installations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWfroRDMUuZbVg90H21EOLACvtfcHk0li8GuFTOwvr-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2itrfE&md5=1df0214bfe56ba79794ef82751ff8630</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw297%26sid%3Dliteratum%253Aachs%26aulast%3DIsmer%26aufirst%3DJ.%26aulast%3DRose%26aufirst%3DA.%2BS.%26aulast%3DTiemann%26aufirst%3DJ.%2BK.%2BS.%26aulast%3DGoede%26aufirst%3DA.%26aulast%3DPreissner%26aufirst%3DR.%26aulast%3DHildebrand%26aufirst%3DP.%2BW.%26atitle%3DSL2%253A%2520an%2520interactive%2520webtool%2520for%2520modeling%2520of%2520missing%2520segments%2520in%2520proteins%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DW390%26epage%3DW394%26doi%3D10.1093%2Fnar%2Fgkw297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coxon, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxall, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardcastle, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matheson, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newell, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivaprakasam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeoh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golding, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cano, C.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design of purine-based probes for selective Nek2 inhibition</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">19089</span>– <span class="NLM_lpage">19124</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.13249</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.18632%2Foncotarget.13249" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=27833088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A280%3ADC%252BC2snkvF2jsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=19089-19124&author=C.+R.+Coxonauthor=C.+Wongauthor=R.+Baylissauthor=K.+Boxallauthor=K.+H.+Carrauthor=A.+M.+Fryauthor=I.+R.+Hardcastleauthor=C.+J.+Mathesonauthor=D.+R.+Newellauthor=M.+Sivaprakasamauthor=H.+Thomasauthor=D.+Turnerauthor=S.+Yeohauthor=L.+Z.+Wangauthor=R.+J.+Griffinauthor=B.+T.+Goldingauthor=C.+Cano&title=Structure-guided+design+of+purine-based+probes+for+selective+Nek2+inhibition&doi=10.18632%2Foncotarget.13249"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-guided design of purine-based probes for selective Nek2 inhibition</span></div><div class="casAuthors">Coxon Christopher R; Wong Christopher; Hardcastle Ian R; Matheson Christopher J; Sivaprakasam Mangaleswaran; Turner David; Griffin Roger J; Golding Bernard T; Cano Celine; Bayliss Richard; Carr Katherine H; Fry Andrew M; Yeoh Sharon; Boxall Kathy; Newell David R; Thomas Huw; Wang Lan Z</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">19089-19124</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Nek2 (NIMA-related kinase 2) is a cell cycle-dependent serine/threonine protein kinase that regulates centrosome separation at the onset of mitosis.  Overexpression of Nek2 is common in human cancers and suppression can restrict tumor cell growth and promote apoptosis.  Nek2 inhibition with small molecules, therefore, offers the prospect of a new therapy for cancer.  To achieve this goal, a better understanding of the requirements for selective-inhibition of Nek2 is required. 6-Alkoxypurines were identified as ATP-competitive inhibitors of Nek2 and CDK2.  Comparison with CDK2-inhibitor structures indicated that judicious modification of the 6-alkoxy and 2-arylamino substituents could achieve discrimination between Nek2 and CDK2.  In this study, a library of 6-cyclohexylmethoxy-2-arylaminopurines bearing carboxamide, sulfonamide and urea substituents on the 2-arylamino ring was synthesized.  Few of these compounds were selective for Nek2 over CDK2, with the best result being obtained for 3-((6-(cyclohexylmethoxy)-9H-purin-2-yl)amino)-N,N-dimethylbenzamide (CDK2 IC50 = 7.0 μM; Nek2 IC50 = 0.62 μM) with >10-fold selectivity.  Deletion of the 6-substituent abrogated activity against both Nek2 and CDK2.  Nine compounds containing an (E)-dialkylaminovinyl substituent at C-6, all showed selectivity for Nek2, e.g. (E)-6-(2-(azepan-1-yl)vinyl)-N-phenyl-9H-purin-2-amine (CDK2 IC50 = 2.70 μM; Nek2 IC50 = 0.27 μM).  Structural biology of selected compounds enabled a partial rationalization of the observed structure activity relationships and mechanism of Nek2 activation.  This showed that carboxamide 11 is the first reported inhibitor of Nek2 in the DFG-in conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQRxhuJ4vkuqOLt9HypzAgQfW6udTcc2eYp2yvtXyWX27ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2snkvF2jsw%253D%253D&md5=b8d7777e4bfb6c375d7ad122a9304e2a</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.13249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.13249%26sid%3Dliteratum%253Aachs%26aulast%3DCoxon%26aufirst%3DC.%2BR.%26aulast%3DWong%26aufirst%3DC.%26aulast%3DBayliss%26aufirst%3DR.%26aulast%3DBoxall%26aufirst%3DK.%26aulast%3DCarr%26aufirst%3DK.%2BH.%26aulast%3DFry%26aufirst%3DA.%2BM.%26aulast%3DHardcastle%26aufirst%3DI.%2BR.%26aulast%3DMatheson%26aufirst%3DC.%2BJ.%26aulast%3DNewell%26aufirst%3DD.%2BR.%26aulast%3DSivaprakasam%26aufirst%3DM.%26aulast%3DThomas%26aufirst%3DH.%26aulast%3DTurner%26aufirst%3DD.%26aulast%3DYeoh%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DL.%2BZ.%26aulast%3DGriffin%26aufirst%3DR.%2BJ.%26aulast%3DGolding%26aufirst%3DB.%2BT.%26aulast%3DCano%26aufirst%3DC.%26atitle%3DStructure-guided%2520design%2520of%2520purine-based%2520probes%2520for%2520selective%2520Nek2%2520inhibition%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D19089%26epage%3D19124%26doi%3D10.18632%2Foncotarget.13249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.</span></span> <span> </span><span class="NLM_article-title">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.phrs.2015.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=26529477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2016&pages=26-48&author=R.+Roskoski&title=Classification+of+small+molecule+protein+kinase+inhibitors+based+upon+the+structures+of+their+drug-enzyme+complexes&doi=10.1016%2Fj.phrs.2015.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span></div><div class="casAuthors">Roskoski, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26-48</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Because dysregulation and mutations of protein kinases play causal roles in human disease, this family of enzymes has become one of the most important drug targets over the past two decades.  The X-ray crystal structures of 21 of the 27 FDA-approved small mol. inhibitors bound to their target protein kinases are depicted in this paper.  The structure of the enzyme-bound antagonist complex is used in the classification of these inhibitors.  Type I inhibitors bind to the active protein kinase conformation (DFG-Asp in, αC-helix in).  Type I1/2 inhibitors bind to a DFG-Asp in inactive conformation while Type II inhibitors bind to a DFG-Asp out inactive conformation.  Type I, I1/2, and type II inhibitors occupy part of the adenine binding pocket and form hydrogen bonds with the hinge region connecting the small and large lobes of the enzyme.  Type III inhibitors bind next to the ATP-binding pocket and type IV inhibitors do not bind to the ATP or peptide substrate binding sites.  Type III and IV inhibitors are allosteric in nature.  Type V inhibitors bind to two different regions of the protein kinase domain and are therefore bivalent inhibitors.  The type I-V inhibitors are reversible.  In contrast, type VI inhibitors bind covalently to their target enzyme.  Type I, I1/2, and II inhibitors are divided into A and B subtypes.  The type A inhibitors bind in the front cleft, the back cleft, and near the gatekeeper residue, all of which occur within the region sepg. the small and large lobes of the protein kinase.  The type B inhibitors bind in the front cleft and gate area but do not extend into the back cleft.  An anal. of the limited available data indicates that type A inhibitors have a long residence time (minutes to hours) while the type B inhibitors have a short residence time (seconds to minutes).  The catalytic spine includes residues from the small and large lobes and interacts with the adenine ring of ATP.  Nearly all of the approved protein kinase inhibitors occupy the adenine-binding pocket; thus it is not surprising that these inhibitors interact with nearby catalytic spine (CS) residues.  Moreover, a significant no. of approved drugs also interact with regulatory spine (RS) residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH0COwqC0CabVg90H21EOLACvtfcHk0liha4zWJmy6Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN&md5=e6b45c8644e322f30f898e763780c64b</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DClassification%2520of%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%2520based%2520upon%2520the%2520structures%2520of%2520their%2520drug-enzyme%2520complexes%26jtitle%3DPharmacol.%2520Res.%26date%3D2016%26volume%3D103%26spage%3D26%26epage%3D48%26doi%3D10.1016%2Fj.phrs.2015.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smyth, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span> <span> </span><span class="NLM_article-title">Measuring and interpreting the selectivity of protein kinase inhibitors</span>. <i>Journal of Chemical Biology</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">151</span>, <span class="refDoi"> DOI: 10.1007/s12154-009-0023-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1007%2Fs12154-009-0023-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=19568781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A280%3ADC%252BD1MrkslGqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2009&pages=131-151&author=L.+A.+Smythauthor=I.+Collins&title=Measuring+and+interpreting+the+selectivity+of+protein+kinase+inhibitors&doi=10.1007%2Fs12154-009-0023-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Measuring and interpreting the selectivity of protein kinase inhibitors</span></div><div class="casAuthors">Smyth Lynette A; Collins Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">131-51</span>
        ISSN:<span class="NLM_cas:issn">1864-6158</span>.
    </div><div class="casAbstract">Protein kinase inhibitors are a well-established class of clinically useful drugs, particularly for the treatment of cancer.  Achieving inhibitor selectivity for particular protein kinases often remains a significant challenge in the development of new small molecules as drugs or as tools for chemical biology research.  This review summarises the methodologies available for measuring kinase inhibitor selectivity, both in vitro and in cells.  The interpretation of kinase inhibitor selectivity data is discussed, particularly with reference to the structural biology of the protein targets.  Measurement and prediction of kinase inhibitor selectivity will be important for the development of new multi-targeted kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQzU9rA5WZcz9FxYDaSSP8XfW6udTcc2eb-ve-GWUR0KLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MrkslGqtA%253D%253D&md5=f78ff1202a949a71ffd65e804298c37f</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1007%2Fs12154-009-0023-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12154-009-0023-9%26sid%3Dliteratum%253Aachs%26aulast%3DSmyth%26aufirst%3DL.%2BA.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMeasuring%2520and%2520interpreting%2520the%2520selectivity%2520of%2520protein%2520kinase%2520inhibitors%26jtitle%3DJournal%2520of%2520Chemical%2520Biology%26date%3D2009%26volume%3D2%26spage%3D131%26epage%3D151%26doi%3D10.1007%2Fs12154-009-0023-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Outtandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleta, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bockenhauer, D.</span></span> <span> </span><span class="NLM_article-title">Zebrafish as a model for kidney function and disease</span>. <i>Pediatr. Nephrol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">751</span>– <span class="NLM_lpage">762</span>, <span class="refDoi"> DOI: 10.1007/s00467-018-3921-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1007%2Fs00467-018-3921-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=29502161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A280%3ADC%252BC1MrnsVCisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2019&pages=751-762&author=P.+Outtandyauthor=C.+Russellauthor=R.+Kletaauthor=D.+Bockenhauer&title=Zebrafish+as+a+model+for+kidney+function+and+disease&doi=10.1007%2Fs00467-018-3921-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Zebrafish as a model for kidney function and disease</span></div><div class="casAuthors">Outtandy Priya; Kleta Robert; Bockenhauer Detlef; Outtandy Priya; Russell Claire</div><div class="citationInfo"><span class="NLM_cas:title">Pediatric nephrology (Berlin, Germany)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">751-762</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Kidney disease is a global problem with around three million people diagnosed in the UK alone and the incidence is rising.  Research is critical to develop better treatments.  Animal models can help to better understand the pathophysiology behind the various kidney diseases and to screen for therapeutic compounds, but the use especially of mammalian models should be minimised in the interest of animal welfare.  Zebrafish are increasingly used, as they are genetically tractable and have a basic renal anatomy comparable to mammalian kidneys with glomerular filtration and tubular filtration processing.  Here, we discuss how zebrafish have advanced the study of nephrology and the mechanisms underlying kidney disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQyE-WYqwvzUrt7lRqEFUJVfW6udTcc2eb-ve-GWUR0KLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MrnsVCisw%253D%253D&md5=bea6e7eaef05d9690f90d0bb38a0b36d</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1007%2Fs00467-018-3921-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00467-018-3921-7%26sid%3Dliteratum%253Aachs%26aulast%3DOuttandy%26aufirst%3DP.%26aulast%3DRussell%26aufirst%3DC.%26aulast%3DKleta%26aufirst%3DR.%26aulast%3DBockenhauer%26aufirst%3DD.%26atitle%3DZebrafish%2520as%2520a%2520model%2520for%2520kidney%2520function%2520and%2520disease%26jtitle%3DPediatr.%2520Nephrol.%26date%3D2019%26volume%3D34%26spage%3D751%26epage%3D762%26doi%3D10.1007%2Fs00467-018-3921-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wlodarchak, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tariq, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Striker, R.</span></span> <span> </span><span class="NLM_article-title">Comparative analysis of the human and zebrafish kinomes: focus on the development of kinase inhibitors</span>. <i>Trends Cell Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">75</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=27011661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A280%3ADC%252BC2srovVGktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=49-75&author=N.+Wlodarchakauthor=R.+Tariqauthor=R.+Striker&title=Comparative+analysis+of+the+human+and+zebrafish+kinomes%3A+focus+on+the+development+of+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative analysis of the human and zebrafish kinomes: focus on the development of kinase inhibitors</span></div><div class="casAuthors">Wlodarchak Nathan; Tariq Rehan; Striker Rob</div><div class="citationInfo"><span class="NLM_cas:title">Trends in cell & molecular biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">49-75</span>
        ISSN:<span class="NLM_cas:issn">0972-8449</span>.
    </div><div class="casAbstract">Targeting kinases with semi-selective kinase inhibitors is one of the most successful drug development strategies of the 21(st) century.  Zebrafish have become an increasingly useful model for pharmaceutical development.  Water-soluble compounds can be screened for zebrafish phenotypes in a high throughput format against a living vertebrate, and cell-signaling events can be imaged in transparent living fish.  Despite zebrafish being a more relevant model than more distantly related systems such as the well-annotated kinome of yeast and drosophila, there is no comparative analysis of the human and zebrafish kinome.  Furthermore most approved kinase inhibitors, often called 'DFG in' ATP competitive inhibitors, act on conserved active site residues in the kinase.  Since the active site residues can be identified by examining the primary sequence, primary sequence identity can be a rough guide as to whether a particular inhibitor will have activity against another kinase.  There is a need to evaluate the utility of zebrafish as a drug development model for active site inhibitors of kinases.  Here we offer a systematic comparison of the catalytic domains of classical human kinases with the catalytic domains of all annotated zebrafish kinases.  We found a high degree of identity between the catalytic domains of most human kinases and their zebrafish homologs, and we ranked 504 human kinase catalytic domains by order of similarity.  We found only 23 human kinases with no easily recognizable homologous zebrafish catalytic domain.  On the other hand we found 78 zebrafish kinase catalytic domains with no close human counterpart.  These 'additional kinase active sites' could represent potential mediators of zebrafish toxicity that may not be relevant to human kinase inhibitors.  We used two clinically approved human kinase inhibitors, one targeting a highly homologous target and one targeting a lesser homologous target, and we compared the known human kinase target structures with modeled zebrafish target structures.  As expected, the homologous target had high structural identity, but even the less homologous target had high structural identity in residues contacted by the inhibitor.  Overall this analysis should help guide researchers interested in studying human kinases and their inhibitors in more tractable systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRJjr-hcquXE8ve_zCu79I8fW6udTcc2eZHRbrecMiPrbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srovVGktw%253D%253D&md5=bae46c964ee9db2e7e185b38310e4000</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWlodarchak%26aufirst%3DN.%26aulast%3DTariq%26aufirst%3DR.%26aulast%3DStriker%26aufirst%3DR.%26atitle%3DComparative%2520analysis%2520of%2520the%2520human%2520and%2520zebrafish%2520kinomes%253A%2520focus%2520on%2520the%2520development%2520of%2520kinase%2520inhibitors%26jtitle%3DTrends%2520Cell%2520Mol.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D49%26epage%3D75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knight, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">Chemical Genetics: Where Genetics and Pharmacology Meet</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">425</span>– <span class="NLM_lpage">430</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.01.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.cell.2007.01.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=17289560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit1Wlt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=425-430&author=Z.+A.+Knightauthor=K.+M.+Shokat&title=Chemical+Genetics%3A+Where+Genetics+and+Pharmacology+Meet&doi=10.1016%2Fj.cell.2007.01.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical genetics: where genetics and pharmacology meet</span></div><div class="casAuthors">Knight, Zachary A.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">425-430</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Genetics and pharmacol. can elicit surprisingly different phenotypes despite targeting the same protein.  This Essay explores these unexpected differences and their implications for biol. and medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq056BEgxFUHrVg90H21EOLACvtfcHk0lid77XlNdGRJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit1Wlt74%253D&md5=8b5d8733dc3b3f699907709cd0ba6f62</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.01.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.01.021%26sid%3Dliteratum%253Aachs%26aulast%3DKnight%26aufirst%3DZ.%2BA.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DChemical%2520Genetics%253A%2520Where%2520Genetics%2520and%2520Pharmacology%2520Meet%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D425%26epage%3D430%26doi%3D10.1016%2Fj.cell.2007.01.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shalom, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shalva, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altschuler, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motro, B.</span></span> <span> </span><span class="NLM_article-title">The mammalian Nek1 kinase is involved in primary cilium formation</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>582</i></span>,  <span class="NLM_fpage">1465</span>– <span class="NLM_lpage">1470</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2008.03.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.febslet.2008.03.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=18387364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvVOiurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=582&publication_year=2008&pages=1465-1470&author=O.+Shalomauthor=N.+Shalvaauthor=Y.+Altschulerauthor=B.+Motro&title=The+mammalian+Nek1+kinase+is+involved+in+primary+cilium+formation&doi=10.1016%2Fj.febslet.2008.03.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The mammalian Nek1 kinase is involved in primary cilium formation</span></div><div class="casAuthors">Shalom, Ohad; Shalva, Nechama; Altschuler, Yoram; Motro, Benny</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">582</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1465-1470</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Recent studies implicate primary cilium (PC) proteins in the etiologies of various polycystic kidney diseases (PKD).  NIMA-related kinases (NRKs) are conserved serine/threonine kinases, which are usually defined as 'mitotic kinases'.  Murine mutants for the NRKs, nek1 (kat mice) suffer from PKD, suggesting that it may be involved in cilium control.  We demonstrated herein that Nek1 is localized to basal body region and that Nek1 overexpression inhibits ciliogenesis in Madin-Darby canine kidney epithelial cells.  The no. of primary cilia is dramatically reduced in kat2J mouse embryonic fibroblasts culture.  It is thus hypothesized that Nek1 links cell cycle progression and the PC cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxogIIq-Yl7rVg90H21EOLACvtfcHk0lid77XlNdGRJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvVOiurc%253D&md5=af63d67e0595f71b959f8d0353efc403</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2008.03.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2008.03.036%26sid%3Dliteratum%253Aachs%26aulast%3DShalom%26aufirst%3DO.%26aulast%3DShalva%26aufirst%3DN.%26aulast%3DAltschuler%26aufirst%3DY.%26aulast%3DMotro%26aufirst%3DB.%26atitle%3DThe%2520mammalian%2520Nek1%2520kinase%2520is%2520involved%2520in%2520primary%2520cilium%2520formation%26jtitle%3DFEBS%2520Lett.%26date%3D2008%26volume%3D582%26spage%3D1465%26epage%3D1470%26doi%3D10.1016%2Fj.febslet.2008.03.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Westhoff, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giselbrecht, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidts, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schindler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beales, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tönshoff, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liebel, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehrig, J.</span></span> <span> </span><span class="NLM_article-title">Development of an automated imaging pipeline for the analysis of the zebrafish larval kidney</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e82137</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0082137</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1371%2Fjournal.pone.0082137" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=24324758" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFeku70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=J.+H.+Westhoffauthor=S.+Giselbrechtauthor=M.+Schmidtsauthor=S.+Schindlerauthor=P.+L.+Bealesauthor=B.+T%C3%B6nshoffauthor=U.+Liebelauthor=J.+Gehrig&title=Development+of+an+automated+imaging+pipeline+for+the+analysis+of+the+zebrafish+larval+kidney&doi=10.1371%2Fjournal.pone.0082137"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Development of an automated imaging pipeline for the analysis of the zebrafish larval kidney</span></div><div class="casAuthors">Westhoff, Jens H.; Giselbrecht, Stefan; Schmidts, Miriam; Schindler, Sebastian; Beales, Philip L.; Toenshoff, Burkhard; Liebel, Urban; Gehrig, Jochen</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e82137/1-e82137/13, 13 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The anal. of kidney malformation caused by environmental influences during nephrogenesis or by hereditary nephropathies requires animal models allowing the in vivo observation of developmental processes.  The zebrafish has emerged as a useful model system for the anal. of vertebrate organ development and function, and it is suitable for the identification of organotoxic or disease-modulating compds. on a larger scale.  However, to fully exploit its potential in high content screening applications, dedicated protocols are required allowing the consistent visualization of inner organs such as the embryonic kidney.  To this end, we developed a high content screening compatible pipeline for the automated imaging of standardized views of the developing pronephros in zebrafish larvae.  Using a custom designed tool, cavities were generated in agarose coated microtiter plates allowing for accurate positioning and orientation of zebrafish larvae.  This enabled the subsequent automated acquisition of stable and consistent dorsal views of pronephric kidneys.  The established pipeline was applied in a pilot screen for the anal. of the impact of potentially nephrotoxic drugs on zebrafish pronephros development in the Tg(wt1b:EGFP) transgenic line in which the developing pronephros is highlighted by GFP expression.  The consistent image data that was acquired allowed for quantification of gross morphol. pronephric phenotypes, revealing concn. dependent effects of several compds. on nephrogenesis.  In addn., applicability of the imaging pipeline was further confirmed in a morpholino based model for cilia-assocd. human genetic disorders assocd. with different intraflagellar transport genes.  The developed tools and pipeline can be used to study various aspects in zebrafish kidney research, and can be readily adapted for the anal. of other organ systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrY3u0Z4CTYw7Vg90H21EOLACvtfcHk0lgFhubW7cApNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFeku70%253D&md5=6b48f06d579844ff51bc296eb45171b5</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0082137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0082137%26sid%3Dliteratum%253Aachs%26aulast%3DWesthoff%26aufirst%3DJ.%2BH.%26aulast%3DGiselbrecht%26aufirst%3DS.%26aulast%3DSchmidts%26aufirst%3DM.%26aulast%3DSchindler%26aufirst%3DS.%26aulast%3DBeales%26aufirst%3DP.%2BL.%26aulast%3DT%25C3%25B6nshoff%26aufirst%3DB.%26aulast%3DLiebel%26aufirst%3DU.%26aulast%3DGehrig%26aufirst%3DJ.%26atitle%3DDevelopment%2520of%2520an%2520automated%2520imaging%2520pipeline%2520for%2520the%2520analysis%2520of%2520the%2520zebrafish%2520larval%2520kidney%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0082137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sander, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teoh, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, A. J.</span></span> <span> </span><span class="NLM_article-title">The small molecule probe PT-Yellow labels the renal proximal tubules in zebrafish</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">395</span>– <span class="NLM_lpage">398</span>, <span class="refDoi"> DOI: 10.1039/C4CC08075K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1039%2FC4CC08075K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=25407666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFSmu7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2015&pages=395-398&author=V.+Sanderauthor=S.+Patkeauthor=S.+Sahuauthor=C.+L.+Teohauthor=Z.+Pengauthor=Y.-T.+Changauthor=A.+J.+Davidson&title=The+small+molecule+probe+PT-Yellow+labels+the+renal+proximal+tubules+in+zebrafish&doi=10.1039%2FC4CC08075K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">The small molecule probe PT-Yellow labels the renal proximal tubules in zebrafish</span></div><div class="casAuthors">Sander, Veronika; Patke, Shantanu; Sahu, Srikanta; Teoh, Chai Lean; Peng, Zhenzhen; Chang, Young-Tae; Davidson, Alan J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">395-398</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">We report the development of a small fluorescent mol., BDNCA3-D2, herein referred to as PT-Yellow.  Soaking zebrafish embryos in PT-Yellow or i.p. injection into adults results in non-toxic in vivo fluorescent labeling of the renal proximal tubules, the major site of blood filtrate reabsorption and a common target of injury in acute kidney injury.  We demonstrate the applicability of this new compd. as a rapid and simple readout for zebrafish kidney filtration and proximal tubule reabsorption function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobqD-zCxFwrrVg90H21EOLACvtfcHk0lgFhubW7cApNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFSmu7%252FI&md5=623b714263266da461d427e66195018b</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1039%2FC4CC08075K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4CC08075K%26sid%3Dliteratum%253Aachs%26aulast%3DSander%26aufirst%3DV.%26aulast%3DPatke%26aufirst%3DS.%26aulast%3DSahu%26aufirst%3DS.%26aulast%3DTeoh%26aufirst%3DC.%2BL.%26aulast%3DPeng%26aufirst%3DZ.%26aulast%3DChang%26aufirst%3DY.-T.%26aulast%3DDavidson%26aufirst%3DA.%2BJ.%26atitle%3DThe%2520small%2520molecule%2520probe%2520PT-Yellow%2520labels%2520the%2520renal%2520proximal%2520tubules%2520in%2520zebrafish%26jtitle%3DChem.%2520Commun.%26date%3D2015%26volume%3D51%26spage%3D395%26epage%3D398%26doi%3D10.1039%2FC4CC08075K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Christou-Savina, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beales, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborn, D. P. S.</span></span> <span> </span><span class="NLM_article-title">Evaluation of Zebrafish Kidney Function Using a Fluorescent Clearance Assay</span>. <i>JoVE (Journal of Visualized Experiments)</i> <span class="NLM_year" style="font-weight: bold;">2015</span>, <span class="NLM_elocation-id">e52540</span> <span class="refDoi"> DOI: 10.3791/52540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.3791%2F52540" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=S.+Christou-Savinaauthor=P.+L.+Bealesauthor=D.+P.+S.+Osborn&title=Evaluation+of+Zebrafish+Kidney+Function+Using+a+Fluorescent+Clearance+Assay&doi=10.3791%2F52540"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.3791%2F52540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3791%252F52540%26sid%3Dliteratum%253Aachs%26aulast%3DChristou-Savina%26aufirst%3DS.%26aulast%3DBeales%26aufirst%3DP.%2BL.%26aulast%3DOsborn%26aufirst%3DD.%2BP.%2BS.%26atitle%3DEvaluation%2520of%2520Zebrafish%2520Kidney%2520Function%2520Using%2520a%2520Fluorescent%2520Clearance%2520Assay%26jtitle%3DJoVE%2520%2528Journal%2520of%2520Visualized%2520Experiments%2529%26date%3D2015%26doi%3D10.3791%2F52540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. H.</span></span> <span> </span><span class="NLM_article-title">Effects of specific genes activating RAGE on polycystic kidney disease</span>. <i>Am. J. Nephrol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">178</span>, <span class="refDoi"> DOI: 10.1159/000315859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1159%2F000315859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=20606421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVaqsbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2010&pages=169-178&author=E.+Y.+Parkauthor=M.+J.+Seoauthor=J.+H.+Park&title=Effects+of+specific+genes+activating+RAGE+on+polycystic+kidney+disease&doi=10.1159%2F000315859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of specific genes activating RAGE on polycystic kidney disease</span></div><div class="casAuthors">Park, Eun Young; Seo, Min Ji; Park, Jong Hoon</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Nephrology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">169-178</span>CODEN:
                <span class="NLM_cas:coden">AJNED9</span>;
        ISSN:<span class="NLM_cas:issn">0250-8095</span>.
    
            (<span class="NLM_cas:orgname">S. Karger AG</span>)
        </div><div class="casAbstract">Background: Autosomal-dominant polycystic kidney disease (ADPKD) is characterized by progressive cyst formation and secretion of fluid and is assocd. with interstitial inflammation and fibrosis, resulting in the loss of renal function.  We previously generated mice overexpressing PKD2, causing progressive cyst development with an inflammatory and fibrotic phenotype in the kidneys.  Methods: To profile the gene expression related to inflammation and cystogenesis, microarray anal. was performed with kidney tissue from 6-, 12- and 18-mo-old mice.  Subsequently, levels and related mechanisms of selected genes, s100a8 and s100a9, were evaluated.  Results: S100a8 and s100a9 was upregulated more than 2-fold and differently expressed in the cystic region.  Receptor of advanced glycation end product (RAGE) is a putative cell surface receptor for s100a8/a9.  It was expressed in cyst-lining cells and up-regulated pro-inflammatory transcription factor NF-κB in transgenic mice.  We also confirmed RAGE expression in ADPKD patient kidneys.  It was suggested that the signaling related to proliferative cystogenesis through previous reports; therefore, we confirmed that phosphorylated-ERK and cyst formation was reduced by treatment of RAGE-siRNA.  Conclusions: The results may provide important information for the expression of s100a8/a9 and RAGE, linking progressive cystogenesis with inflammation in cystic kidney.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbd5fZHIZl77Vg90H21EOLACvtfcHk0lgFhubW7cApNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVaqsbjL&md5=5c4547473bdff7297face14b319421a7</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1159%2F000315859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1159%252F000315859%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DE.%2BY.%26aulast%3DSeo%26aufirst%3DM.%2BJ.%26aulast%3DPark%26aufirst%3DJ.%2BH.%26atitle%3DEffects%2520of%2520specific%2520genes%2520activating%2520RAGE%2520on%2520polycystic%2520kidney%2520disease%26jtitle%3DAm.%2520J.%2520Nephrol.%26date%3D2010%26volume%3D32%26spage%3D169%26epage%3D178%26doi%3D10.1159%2F000315859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Booij, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bange, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonhard, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fokkelman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunnen, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dauwerse, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van de Water, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Westen, G. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, D. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, L. S.</span></span> <span> </span><span class="NLM_article-title">High-throughput phenotypic screening of kinase inhibitors to identify drug targets for polycystic kidney disease</span>. <i>SLAS Discov</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">974</span>– <span class="NLM_lpage">984</span>, <span class="refDoi"> DOI: 10.1177/2472555217716056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1177%2F2472555217716056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=28644734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKku7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=974-984&author=T.+H.+Booijauthor=H.+Bangeauthor=W.+N.+Leonhardauthor=K.+Yanauthor=M.+Fokkelmanauthor=S.+J.+Kunnenauthor=J.+G.+Dauwerseauthor=Y.+Qinauthor=B.+van+de+Waterauthor=G.+J.+P.+van+Westenauthor=D.+J.+M.+Petersauthor=L.+S.+Price&title=High-throughput+phenotypic+screening+of+kinase+inhibitors+to+identify+drug+targets+for+polycystic+kidney+disease&doi=10.1177%2F2472555217716056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Phenotypic Screening of Kinase Inhibitors to Identify Drug Targets for Polycystic Kidney Disease</span></div><div class="casAuthors">Booij, Tijmen H.; Bange, Hester; Leonhard, Wouter N.; Yan, Kuan; Fokkelman, Michiel; Kunnen, Steven J.; Dauwerse, Johannes G.; Qin, Yu; van de Water, Bob; van Westen, Gerard J. P.; Peters, Dorien J. M.; Price, Leo S.</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">974-984</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Polycystic kidney disease (PKD) is a prevalent disorder characterized by renal cysts that lead to kidney failure.  Various signaling pathways have been targeted to stop disease progression, but most interventions still focus on alleviating PKD-assocd. symptoms.  The mechanistic complexity of the disease, as well as the lack of functional in vitro assays for compd. testing, has made drug discovery for PKD challenging.  To identify modulators of PKD, Pkd1-/- kidney tubule epithelial cells were applied to a scalable and automated 3D cyst culture model for compd. screening, followed by phenotypic profiling to det. compd. efficacy.  We used this screening platform to screen a library of 273 kinase inhibitors to probe various signaling pathways involved in cyst growth.  We show that inhibition of several targets, including aurora kinase, CDK, Chk, IGF-1R, Syk, and mTOR, but, surprisingly, not PI3K, prevented forskolin-induced cyst swelling.  Addnl., we show that multiparametric phenotypic classification discriminated potentially undesirable (i.e., cytotoxic) compds. from mols. inducing the desired phenotypic change, greatly facilitating hit selection and validation.  Our findings show that a pathophysiol. relevant 3D cyst culture model of PKD coupled to phenotypic profiling can be used to identify potentially therapeutic compds. and predict and validate mol. targets for PKD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5seGssaoSALVg90H21EOLACvtfcHk0lh1eT9Su123Vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFKku7Y%253D&md5=dcd2e6d399d403c033b8bc10c26aa823</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1177%2F2472555217716056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555217716056%26sid%3Dliteratum%253Aachs%26aulast%3DBooij%26aufirst%3DT.%2BH.%26aulast%3DBange%26aufirst%3DH.%26aulast%3DLeonhard%26aufirst%3DW.%2BN.%26aulast%3DYan%26aufirst%3DK.%26aulast%3DFokkelman%26aufirst%3DM.%26aulast%3DKunnen%26aufirst%3DS.%2BJ.%26aulast%3DDauwerse%26aufirst%3DJ.%2BG.%26aulast%3DQin%26aufirst%3DY.%26aulast%3Dvan%2Bde%2BWater%26aufirst%3DB.%26aulast%3Dvan%2BWesten%26aufirst%3DG.%2BJ.%2BP.%26aulast%3DPeters%26aufirst%3DD.%2BJ.%2BM.%26aulast%3DPrice%26aufirst%3DL.%2BS.%26atitle%3DHigh-throughput%2520phenotypic%2520screening%2520of%2520kinase%2520inhibitors%2520to%2520identify%2520drug%2520targets%2520for%2520polycystic%2520kidney%2520disease%26jtitle%3DSLAS%2520Discov%26date%3D2017%26volume%3D22%26spage%3D974%26epage%3D984%26doi%3D10.1177%2F2472555217716056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Correa, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsui, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tergaonkar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Esteban, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izpisua-Belmonte, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, I. M.</span></span> <span> </span><span class="NLM_article-title">Zebrafish IκB Kinase 1 Negatively Regulates NF-κB Activity</span>. <i>Curr. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1291</span>– <span class="NLM_lpage">1295</span>, <span class="refDoi"> DOI: 10.1016/j.cub.2005.06.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.cub.2005.06.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=16051172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvVKgsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2005&pages=1291-1295&author=R.+G.+Correaauthor=T.+Matsuiauthor=V.+Tergaonkarauthor=C.+Rodriguez-Estebanauthor=J.+C.+Izpisua-Belmonteauthor=I.+M.+Verma&title=Zebrafish+I%CE%BAB+Kinase+1+Negatively+Regulates+NF-%CE%BAB+Activity&doi=10.1016%2Fj.cub.2005.06.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Zebrafish IκB Kinase 1 Negatively Regulates NF-κB Activity</span></div><div class="casAuthors">Correa, Ricardo G.; Matsui, Takaaki; Tergaonkar, Vinay; Rodriguez-Esteban, Concepcion; Izpisua-Belmonte, Juan Carlos; Verma, Inder M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1291-1295</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The IκB kinase (IKK) activity is crit. for processing IκB inhibitory proteins and activating the NF-κB signaling, which is involved in a series of physiol. and developmental steps in vertebrates.  The IKK activity resides in two catalytic subunits, IKK1 and IKK2, and two regulatory subunits, NEMO and ELKS.  IKK2 is the major cytokine-responsive IκB kinase because depletion of IKK1 does not interfere with the IKK activity.  In fact, IKK1-/- mice display morphol. abnormalities that are independent of its kinase activity and NF-κB activation.  Hence, using zebrafish (Danio rerio) as a model, the authors examd. the evolutionary role of IKK1 in modulating NF-κB.  Ikk1-/- zebrafish embryos present head and tail malformations and, surprisingly, show upregulation of NF-κB-responsive genes and increased NF-κB-dependent apoptosis.  Overexpression of ikk1 leads to midline structure defects that resemble NF-κB blockage in vivo.  Zebrafish Ikk1 forms complexes with NEMO that represses NF-κB in vertebrate cells.  Indeed, truncation of its NEMO binding domain (NBD) restores NF-κB-dependent transcriptional activity and, consequently, the ikk1-overexpressing phenotype.  Here, the authors report that Ikk1 neg. regulates NF-κB by sequestering NEMO from active IKK complexes, indicating that IKK1 can function as a repressor of NF-κB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokL64ZlJ8V6rVg90H21EOLACvtfcHk0lh1eT9Su123Vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmvVKgsrs%253D&md5=d69d6fcf5a97936335bda74f6f058835</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.cub.2005.06.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cub.2005.06.023%26sid%3Dliteratum%253Aachs%26aulast%3DCorrea%26aufirst%3DR.%2BG.%26aulast%3DMatsui%26aufirst%3DT.%26aulast%3DTergaonkar%26aufirst%3DV.%26aulast%3DRodriguez-Esteban%26aufirst%3DC.%26aulast%3DIzpisua-Belmonte%26aufirst%3DJ.%2BC.%26aulast%3DVerma%26aufirst%3DI.%2BM.%26atitle%3DZebrafish%2520I%25CE%25BAB%2520Kinase%25201%2520Negatively%2520Regulates%2520NF-%25CE%25BAB%2520Activity%26jtitle%3DCurr.%2520Biol.%26date%3D2005%26volume%3D15%26spage%3D1291%26epage%3D1295%26doi%3D10.1016%2Fj.cub.2005.06.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span> <span> </span><span class="NLM_article-title">Epigenetics and cell cycle regulation in cystogenesis</span>. <i>Cell. Signalling</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">109509</span>, <span class="refDoi"> DOI: 10.1016/j.cellsig.2019.109509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1016%2Fj.cellsig.2019.109509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=31874209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVeksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2020&pages=109509&author=X.+Li&title=Epigenetics+and+cell+cycle+regulation+in+cystogenesis&doi=10.1016%2Fj.cellsig.2019.109509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetics and cell cycle regulation in cystogenesis</span></div><div class="casAuthors">Li, Xiaogang</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">109509</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  The role of genetic mutations in the development of polycystic kidney disease (PKD), such as alterations in PKD1 and PKD2 genes in autosomal dominant PKD (ADPKD), is well understood.  However, the significance of epigenetic mechanisms in the progression of PKD remains unclear and is increasingly being investigated.  The term of epigenetics describes a range of mechanisms in genome function that do not solely result from the DNA sequence itself.  Epigenetic information can be inherited during mammalian cell division to sustain phenotype specifically and physiol. responsive gene expression in the progeny cells.  A multitude of functional studies of epigenetic modifiers and systematic genome-wide mapping of epigenetic marks reveal the importance of epigenomic mechanisms, including DNA methylation, histone/chromatin modifications and non-coding RNAs, in PKD pathologies.  Deregulated proliferation is a characteristic feature of cystic renal epithelial cells.  Moreover, defects in many of the mols. that regulate the cell cycle have been implicated in cyst formation and progression.  Recent evidence suggests that alterations of DNA methylation and histone modifications on specific genes and the whole genome involved in cell cycle regulation and contribute to the pathogenesis of PKD.  This review summarizes the recent advances of epigenetic mechanisms in PKD, which helps us to define the term of "PKD epigenetics" and group PKD epigenetic changes in three categories.  In particularly, this review focuses on the interplay of epigenetic mechanisms with cell cycle regulation during normal cell cycle progression and cystic cell proliferation, and discusses the potential to detect and quantify DNA methylation from body fluids as diagnostic/prognostic biomarkers.  Collectively, this review provides concepts and examples of epigenetics in cell cycle regulation to reveal a broad view of different aspects of epigenetics in biol. and PKD, which may facilitate to identify possible novel therapeutic intervention points and to explore epigenetic biomarkers in PKD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoN6h7oTe7ci7Vg90H21EOLACvtfcHk0lgZ_m1JkNNL6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVeksLs%253D&md5=556e6f66521bf8ca404abe67a43fc7b0</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2019.109509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2019.109509%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26atitle%3DEpigenetics%2520and%2520cell%2520cycle%2520regulation%2520in%2520cystogenesis%26jtitle%3DCell.%2520Signalling%26date%3D2020%26volume%3D68%26spage%3D109509%26doi%3D10.1016%2Fj.cellsig.2019.109509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saigusa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, P. D.</span></span> <span> </span><span class="NLM_article-title">Molecular pathways and therapies in autosomal-dominant polycystic kidney disease</span>. <i>Physiology</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">207</span>, <span class="refDoi"> DOI: 10.1152/physiol.00032.2014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1152%2Fphysiol.00032.2014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=25933820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVOju7nO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2015&pages=195-207&author=T.+Saigusaauthor=P.+D.+Bell&title=Molecular+pathways+and+therapies+in+autosomal-dominant+polycystic+kidney+disease&doi=10.1152%2Fphysiol.00032.2014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular pathways and therapies in autosomal-dominant polycystic kidney disease</span></div><div class="casAuthors">Saigusa, Takamitsu; Bell, P. Darwin</div><div class="citationInfo"><span class="NLM_cas:title">Physiology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">May</span>),
    <span class="NLM_cas:pages">195-207</span>CODEN:
                <span class="NLM_cas:coden">PHYSCI</span>;
        ISSN:<span class="NLM_cas:issn">1548-9213</span>.
    
            (<span class="NLM_cas:orgname">International Union of Physiological Sciences</span>)
        </div><div class="casAbstract">A review.  Autosomal-dominant polycystic kidney disease (ADPKD) is the most prevalent inherited renal disease, characterized by multiple cysts that can eventually lead to kidney failure.  Studies investigating the role of primary cilia and polycystins have significantly advanced our understanding of the pathogenesis of PKD.  This review will present clin. and basic aspects of ADPKD, review current concepts of PKD pathogenesis, evaluate potential therapeutic targets, and highlight challenges for future clin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxW9aaMg23f7Vg90H21EOLACvtfcHk0lgZ_m1JkNNL6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVOju7nO&md5=b999675a2119ed75898b32515288fe5a</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1152%2Fphysiol.00032.2014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fphysiol.00032.2014%26sid%3Dliteratum%253Aachs%26aulast%3DSaigusa%26aufirst%3DT.%26aulast%3DBell%26aufirst%3DP.%2BD.%26atitle%3DMolecular%2520pathways%2520and%2520therapies%2520in%2520autosomal-dominant%2520polycystic%2520kidney%2520disease%26jtitle%3DPhysiology%26date%3D2015%26volume%3D30%26spage%3D195%26epage%3D207%26doi%3D10.1152%2Fphysiol.00032.2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Giovanni, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingram, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenblum, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, Y.</span></span> <span> </span><span class="NLM_article-title">Systems biology of autosomal dominant polycystic kidney disease (ADPKD): computational identification of gene expression pathways and integrated regulatory networks</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2328</span>– <span class="NLM_lpage">2343</span>, <span class="refDoi"> DOI: 10.1093/hmg/ddp165</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.1093%2Fhmg%2Fddp165" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=19346236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1ehtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2009&pages=2328-2343&author=X.+Songauthor=V.+Di+Giovanniauthor=N.+Heauthor=K.+Wangauthor=A.+Ingramauthor=N.+D.+Rosenblumauthor=Y.+Pei&title=Systems+biology+of+autosomal+dominant+polycystic+kidney+disease+%28ADPKD%29%3A+computational+identification+of+gene+expression+pathways+and+integrated+regulatory+networks&doi=10.1093%2Fhmg%2Fddp165"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Systems biology of autosomal dominant polycystic kidney disease (ADPKD): computational identification of gene expression pathways and integrated regulatory networks</span></div><div class="casAuthors">Song, Xuewen; Di Giovanni, Valeria; He, Ning; Wang, Kairong; Ingram, Alistair; Rosenblum, Norman D.; Pei, York</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2328-2343</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">To elucidate the mol. pathways that modulate renal cyst growth in ADPKD, we performed global gene profiling on cysts of different size (<1 mL, n = 5; 10-20 mL, n = 5; >50 mL, n = 3) and minimally cystic tissue (MCT, n = 5) from five PKD1 human polycystic kidneys using Affymetrix HG-U133 Plus 2.0 arrays.  We used gene set enrichment anal. to identify overrepresented signaling pathways and key transcription factors (TFs) between cysts and MCT.  We found down-regulation of kidney epithelial restricted genes (e.g. nephron segment-specific markers and cilia-assocd. cystic genes such as HNF1B, PKHD1, IFT88 and CYS1) in the renal cysts.  On the other hand, PKD1 cysts displayed a rich profile of gene sets assocd. with renal development, mitogen-mediated proliferation, cell cycle progression, epithelial-mesenchymal transition, hypoxia, aging and immune/inflammatory responses.  Notably, our data suggest that up-regulation of Wnt/beta-catenin, pleiotropic growth factor/receptor tyrosine kinase (e.g. IGF/IGF1R, FGF/FGFR, EGF/EGFR, VEGF/VEGFR), G-protein-coupled receptor (e.g. PTGER2) signaling was assocd. with renal cystic growth.  By integrating these pathways with a no. of dysregulated networks of TFs (e.g. SRF, MYC, E2F1, CREB1, LEF1, TCF7, HNF1B/ HNF1A and HNF4A), our data suggest that epithelial dedifferentiation accompanied by aberrant activation and cross-talk of specific signaling pathways may be required for PKD1 cyst growth and disease progression.  Pharmacol. modulation of some of these signaling pathways may provide a potential therapeutic strategy for ADPKD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8tdwJNQDsg7Vg90H21EOLACvtfcHk0lgZ_m1JkNNL6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1ehtLs%253D&md5=6a0af3a16ffd7d70714cd4cefd3e2b92</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2Fddp165&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252Fddp165%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DX.%26aulast%3DDi%2BGiovanni%26aufirst%3DV.%26aulast%3DHe%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DIngram%26aufirst%3DA.%26aulast%3DRosenblum%26aufirst%3DN.%2BD.%26aulast%3DPei%26aufirst%3DY.%26atitle%3DSystems%2520biology%2520of%2520autosomal%2520dominant%2520polycystic%2520kidney%2520disease%2520%2528ADPKD%2529%253A%2520computational%2520identification%2520of%2520gene%2520expression%2520pathways%2520and%2520integrated%2520regulatory%2520networks%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2009%26volume%3D18%26spage%3D2328%26epage%3D2343%26doi%3D10.1093%2Fhmg%2Fddp165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0ljnfvAm1J-_oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haldi, M.</span>; <span class="NLM_string-name">Harden, M.</span>; <span class="NLM_string-name">D’Amico, L.</span>; <span class="NLM_string-name">DeLise, A.</span>; <span class="NLM_string-name">Seng, W. L.</span></span> <span> </span><span class="NLM_article-title">Developmental Toxicity Assessment in Zebrafish</span>. In  <i>Zebrafish: Methods for Assessing Drug Safety and Toxicity</i>, <span class="NLM_contrib-group"><span class="NLM_string-name">McGrath, P.</span></span>, Ed.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>, <span class="NLM_year">2012</span>; Chapter 2, pp  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">17</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=15-17&author=M.+Haldi&author=M.+Harden&author=L.+D%E2%80%99Amico&author=A.+DeLise&author=W.+L.+Sengauthor=P.+McGrath&title=Zebrafish%3A+Methods+for+Assessing+Drug+Safety+and+Toxicity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DHaldi%26aufirst%3DM.%26atitle%3DDevelopmental%2520Toxicity%2520Assessment%2520in%2520Zebrafish%26btitle%3DZebrafish%253A%2520Methods%2520for%2520Assessing%2520Drug%2520Safety%2520and%2520Toxicity%26aulast%3DMcGrath%26aufirst%3DP.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2012%26spage%3D15%26epage%3D17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edelstein, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchida, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amodaj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinkard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vale, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuurman, N.</span></span> <span> </span><span class="NLM_article-title">Advanced methods of microscope control using μManager software</span>. <i>Journal of Biological Methods</i> <span class="NLM_year" style="font-weight: bold;">2014</span>, <span class="NLM_elocation-id">e10</span> <span class="refDoi"> DOI: 10.14440/jbm.2014.36</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=10.14440%2Fjbm.2014.36" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;key=25606571" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=A.+D.+Edelsteinauthor=M.+A.+Tsuchidaauthor=N.+Amodajauthor=H.+Pinkardauthor=R.+D.+Valeauthor=N.+Stuurman&title=Advanced+methods+of+microscope+control+using+%CE%BCManager+software&doi=10.14440%2Fjbm.2014.36"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.14440%2Fjbm.2014.36&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.14440%252Fjbm.2014.36%26sid%3Dliteratum%253Aachs%26aulast%3DEdelstein%26aufirst%3DA.%2BD.%26aulast%3DTsuchida%26aufirst%3DM.%2BA.%26aulast%3DAmodaj%26aufirst%3DN.%26aulast%3DPinkard%26aufirst%3DH.%26aulast%3DVale%26aufirst%3DR.%2BD.%26aulast%3DStuurman%26aufirst%3DN.%26atitle%3DAdvanced%2520methods%2520of%2520microscope%2520control%2520using%2520%25CE%25BCManager%2520software%26jtitle%3DJournal%2520of%2520Biological%2520Methods%26date%3D2014%26doi%3D10.14440%2Fjbm.2014.36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4APC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4APC','PDB','4APC'); return false;">PDB: 4APC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4B9D" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4B9D','PDB','4B9D'); return false;">PDB: 4B9D</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K2Y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5K2Y','PDB','5K2Y'); return false;">PDB: 5K2Y</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5M53" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5M53','PDB','5M53'); return false;">PDB: 5M53</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i67"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02118">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_58703"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c02118?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02118</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular modeling, homology model coordinates, <i>in vitro</i> pharmacology, NMR data, lead compound HPLC traces, bioavailability profile of compound <b>10f</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_003.csv">CSV</a>)</p></li><li><p class="inline">Nek1 homology model (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_004.pdb">PDB</a>)</p></li><li><p class="inline">Superlooper2 reconstruction model (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_005.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_001.pdf">jm0c02118_si_001.pdf (7.03 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_003.csv">jm0c02118_si_003.csv (2.16 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_004.pdb">jm0c02118_si_004.pdb (346.55 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02118/suppl_file/jm0c02118_si_005.pdb">jm0c02118_si_005.pdb (387.46 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02118&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02118" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02118" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677b6044fc673c4c","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
